<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-14 09:28:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>31</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>67</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>108</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>322</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>281</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6556f74989cd6bd7e813ef9e417ef3b7531711e1" target='_blank'>
              Total whole-arm chromosome losses predict malignancy in human cancer
              </a>
            </td>
          <td>
            Ye Zheng, K. Ahmad, S. Henikoff
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="As a well-known cancer risk region, the 8q24 locus is frequently amplified in a variety of solid tumors. Here we identify a pseudogene-derived oncogenic lncRNA, ZNF252P, which is upregulated in a variety of cancer types by copy number gain as well as c-Myc-mediated transcriptional activation. Mechanistically, ZNF252P binds and drives “phase separation” of HNRNPK and ILF3 protein in the nucleus and cytoplasm, respectively, to transcriptionally and posttranscriptionally activate c-Myc, thus forming a c-Myc/ZNF252P/c-Myc positive feedback loop. These findings expand the understanding of the relationship between genomic instability in the 8q24 region and tumorigenesis and clarify a regulatory mechanism involved in transcription and posttranscription from the perspective of RNA-mediated nuclear and cytoplasmic protein phase separation, which sheds light on the dialogue with the driver oncogene c-Myc. The pivotal regulatory axis of ZNF252P/c-Myc has potential as a promising biomarker and therapeutic target in cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad9d941478590c32e5cf0763f3872cb1fd2c287" target='_blank'>
              Satellite DNA Amplification in Advanced Prostate Cancer Is Largely Independent From Euchromatic and Oncogene Amplicons.
              </a>
            </td>
          <td>
            Anja Weise, A. Ornellas, Gilda Alves, Constanze Pentzold, Jenny Holler, Melanie Wolter, Elena Jamali, Bernhard Theis, Thomas Liehr
          </td>
          <td>2025-03-17</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Despite the development of HER2‐targeting drugs such as trastuzumab and T‐DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2‐positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti‐HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant‐targeted therapy cohorts and a patient‐derived organoid in vitro treatment model to uncover the potential targetable drivers of anti‐HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2‐positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing‐Induced Chromosomal Instability in Breast Cancer
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromosome territories (CTs) are intricately organized and regulated within the nucleus. Despite remarkable advances in our understanding of genome packaging and gene expression, the interplay among CTs, pairing of parental homologous chromosomes, and genome function during development remains elusive. Here, we employ an Oligopaints-based high-resolution imaging approach to examine variable CT organization in single nuclei during the developmental process of zygotic genome activation. We reveal large-scale chromosome changes with extensive homolog pairing at the whole-chromosome level that decreases locally due to spatial variability in chromosome conformations. In the absence of one homolog copy, the dynamics of CT compaction and RNA polymerase II recruitment are supported by transcriptional changes in haploid embryos. Finally, global inhibition of transcription results in decreased CT opening and no significant impact on CT pairing levels. These findings enhance our understanding of parental genome folding and regulation, which may inform strategies for chromosome-based diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461f9f862c0bc9f9b6c4530eae22c13287308545" target='_blank'>
              The functional organization of chromosome territories in single nuclei during zygotic genome activation
              </a>
            </td>
          <td>
            Akshada Shankar Ganesh, Taylor M. Orban, Romir Raj, Peter I. Fatzinger, Anna Johnson, Sean M. Riccard, Akhmed Zhanaidarov, Mayu Inaba, Jelena Erceg
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9124f513a579068358157bc6335b2150cb3dde2" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma
              </a>
            </td>
          <td>
            Lays Martin Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="NOTCH2NL (NOTCH2-N-terminus-like) genes arose from incomplete, recent chromosome 1 segmental duplications implicated in human brain cortical expansion. Genetic characterization of these loci and their regulation is complicated by the fact they are embedded in large, nearly identical duplications that predispose to recurrent microdeletion syndromes. Using nearly complete long-read assemblies generated from 67 human and 12 ape haploid genomes, we show independent recurrent duplication among apes with functional copies emerging in humans ∼2.1 million years ago. We distinguish NOTCH2NL paralogs present in every human haplotype (NOTCH2NLA) from copy number variable ones. We also characterize large-scale structural variation, including gene conversion, for 28% of haplotypes leading to a previously undescribed paralog, NOTCH2tv. Finally, we apply Fiber-seq and long-read transcript sequencing to human cortical neurospheres to characterize the regulatory landscape and find that the most fixed paralogs, NOTCH2 and NOTCH2NLA, harbor the greatest number of paralog-specific elements potentially driving their regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57156ab124ecd643544fdfa8cdaaa9cc74e49805" target='_blank'>
              Genetic diversity and regulatory features of human-specific NOTCH2NL duplications
              </a>
            </td>
          <td>
            Taylor Real, Prajna Hebbar, DongAhn Yoo, F. Antonacci, Ivana Pačar, M. Diekhans, Gregory J. Mikol, Oyeronke G. Popoola, Ben Mallory, Mitchell R. Vollger, Philip C. Dishuck, Xavi Guitart, Allison N. Rozanski, Katherine M. Munson, Kendra Hoekzema, Jane Ranchalis, Shane Neph, Adriana E. Sedeño-Cortés, B. Paten, Sofie R. Salama, A. Stergachis, E. Eichler
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Exonic enhancers (EEs) occupy an under-appreciated niche in gene regulation. By integrating transcription factor binding, chromatin accessibility, and high-throughput enhancer-reporter assays, we demonstrate that many protein-coding exons possess enhancer activity across species. These EEs exhibit characteristic epigenomic signatures, form long-range interactions with gene promoters, and can be altered by both nonsynonymous and synonymous variants. CRISPR–mediated inactivation demonstrated the involvement of EEs in the cis-regulation of host and distal gene expression. Through large-scale cancer genome analyses, we reveal that EE mutations correlate with dysregulated target-gene expression and clinical outcomes, highlighting their potential relevance in disease. Evolutionary comparisons show that EEs exhibit both strong sequence constraint and lineage-specific plasticity, suggesting that they serve ancient regulatory functions while also contributing to species divergence. Our findings redefine the landscape of functional elements by establishing EEs as a component of gene regulation, while revealing how coding regions can simultaneously fulfil both protein-coding and cis-regulatory roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/163da6e03d81a31e9fe14936d274ce9d2b480af7" target='_blank'>
              Exonic enhancers are a widespread class of dual-function regulatory elements
              </a>
            </td>
          <td>
            Jean-Christophe Mouren, Magali Torres, Antoinette F. van Ouwerkerk, Iris Manosalva, F. Gallardo, Salvatore Spicuglia, Benoît Ballester
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8033a50256e64b94ba359a06046d4ac24e218fe" target='_blank'>
              Targeting SUV4-20H epigenetic enzymes as therapeutic strategy for enhancing topoisomerase II poisoning in prostate cancer
              </a>
            </td>
          <td>
            Fatima Alhourani, Marine Tauziet, Julie Patouillard, Benoit Miotto, Véronique Baldin, Aurélie Gennetier, Simon George, Xavier Miahle, Cyril Ribeyre, Philippe Pourquier, Florence M. Cammas, A. Torro, Diego Tosi, Céline Gongora, Mona Dergham, Raghida Abou-Merhi, Eric Julien
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Identification of new therapeutic targets in hepatocellular carcinoma (HCC) remains critical. Chromatin regulating complexes are frequently mutated or aberrantly expressed in HCC, suggesting dysregulation of chromatin environments is a key feature driving liver cancer. To investigate whether the altered chromatin state in HCC cells could be targeted, we designed and utilized an epigenome-focused CRISPR library that targets genes involved in chromatin regulation. This focused approach allowed us to test multiple HCC cell lines in both 2D and 3D growth conditions, which revealed striking differences in the essentiality of genes involved in ubiquitination and multiple chromatin regulators vital for HCC cell survival in 2D but whose loss promoted growth in 3D. We found the core subunits of the menin-MLL1 complex among the strongest essential genes for HCC survival in all screens and thoroughly characterized the mechanism through which the menin-MLL1 complex promotes HCC cell growth. Inhibition of the menin-MLL1 interaction led to global changes in occupancy of the complex with concomitant decreases in H3K4me3 and expression of genes involved in PI3K/AKT/mTOR signaling pathway. Menin inhibition affected chromatin accessibility in HCC cells, revealing that increased chromatin accessibility at sites not bound by menin-MLL1 was associated with the recruitment of the pioneer transcription factor complex NF-Y. A CRISPR/Cas9 screen of chromatin regulators in the presence of menin inhibitor SNDX-5613 revealed a significantly increased cell death when combined with NFYB knockout. Together these data show that menin-MLL1 is necessary for HCC cell survival and cooperates with NF-Y to regulate oncogenic gene transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb9b8fe99339ff834a043435b391f2d743bc86f" target='_blank'>
              Menin-MLL1 complex cooperates with NF-Y to promote HCC survival
              </a>
            </td>
          <td>
            Margarita Dzama-Karels, Peyton C. Kuhlers, Mallory Sokolowski, Jacqueline A. Brinkman, John P. Morris, Jesse R. Raab
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, A. Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Allele-specific DNA methylation (ASM) provides critical insights into the complex genetic and epigenetic mechanisms regulating gene transcription. Emerging evidence suggests that ASM is particularly enriched in gene enhancer regions, and recent studies have demonstrated that ASM is increased in cancer tissues compared with normal tissues. Despite the increasing recognition of ASM as a potential biomarker in tumorigenesis, systematic resources dedicated to identifying and annotating ASMs in cancer contexts remain limited. In this study, we developed CanASM (https://bioinfor.nefu.edu.cn/CanASM/), the first comprehensive database specifically designed to identify and annotate ASM in cancer. In CanASM, ASM sites identified from bisulfite sequencing data across 31 cancer types and their matched normal tissue samples are cataloged. Importantly, CanASM includes extensive regulatory annotations for ASMs, including associated genes, cis-regulatory elements and transcription factor binding colocalizations, transcription factor affinity changes, etc. Users can query and explore ASMs using various parameters, such as single-nucleotide variations (SNVs), chromosomal coordinates, and gene names. The current version of CanASM includes 5,003,877 unique SNV–CpG pairs, including 3,056,776 index SNVs, of which 2,634,406 are single-nucleotide polymorphisms (SNPs), and 4,157,508 CpGs. With an intuitive interface for browsing, querying, analyzing, and downloading, CanASM serves as a valuable resource for researchers investigating cancer-associated genetic variations and epigenetic regulation in cancer. Author summary Identifying target genes and regulatory mechanisms of cancer-associated genetic variants remains a major challenge in the post-GWAS era. Existing databases like Pancan-meQTL and ASMdb have contributed to uncovering the role of genetic variants in cancer by linking DNA variations to methylation. However, they lack comprehensive regulatory annotations for such sites and rely solely on genomic proximity for target gene prediction, overlooking distal interactions. Our analysis using 3D interaction data reveals that over 94.8% of ASMs exhibit distal regulatory relationships, emphasizing their critical role in cancer epigenetics. Furthermore, these databases cover a limited number of cancer types (Pancan-meQTL: 23, ASMdb: 8). To address these limitations, we developed CanASM, a database that provides ASMs identified from BS-seq data for 31 human cancers, along with multidimensional regulatory annotations for ASM sites. This includes target genes, cis-regulatory elements (enhancers, ATAC-seq regions, insulators, 3D interactions, CpG islands), TFBSs, and GWAS cancer trait associations. Additionally, CanASM offers three interactive tools for analyzing TF binding affinity changes, predicting ASM target genes through 3D interactions, and identifying cell marker variations. As a comprehensive resource for SNV-driven epigenetic regulation in cancer, CanASM is helpful for in-depth research in cancer genetics and epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0525da13a9172e31ed829ad891d330567e188ba3" target='_blank'>
              CanASM: A comprehensive database for genome-wide allele-specific DNA methylation identification and annotation in cancer
              </a>
            </td>
          <td>
            Jianmei zhao, Zeyu zhao, Hongfei Li, Haojie Yu, Hao Lin, Hanqi Chen, Xuecang Li, Di Liu, Yiming Wang, Guohua Wang
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, Victor Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, driving oncogenic mutations that enhance tumor aggressiveness and drug resistance. MYC, a master transcription factor that is deregulated in nearly all human tumors, paradoxically induces replication stress and associated DNA damage while also increasing expression of DNA repair factors and mediating resistance to DNA-damaging therapies. Emerging evidence supports a non-transcriptional role for MYC in preserving genomic integrity at sites of active transcription and protecting stalled replication forks under stress. Understanding how MYC’s genotoxic and genoprotective functions diverge may reveal new therapeutic strategies for MYC-driven cancers. Here, we identify a non-canonical role of MYC in DNA damage response (DDR) through its direct association with DNA breaks. We show that phosphorylation at serine 62 (pS62-MYC) is crucial for the efficient recruitment of MYC to damage sites, its interaction with repair factors BRCA1 and RAD51, and effective DNA repair to support cell survival under stress. Mass spectrometry analysis with MYC-BioID2 during replication stress reveals a shift in MYC’s interactome, maintaining DDR associations while losing transcriptional regulators. These findings establish pS62-MYC as a key regulator of genomic stability and a potential therapeutic target in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73224953c4cbe478fbdd30504471b72d2f0b5d71" target='_blank'>
              MYC Serine 62 phosphorylation promotes its binding to DNA double strand breaks to facilitate repair and cell survival under genotoxic stress
              </a>
            </td>
          <td>
            Gabriel M. Cohn, Colin J. Daniel, Jenny Eng, Xiao-Xin Sun, C. Pelz, Koei Chin, Alexander Smith, Charles D. Lopez, Jonathan R. Brody, M. Dai, Rosalie C. Sears
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT The non‐coding genome, constituting 98% of human DNA, remains largely unexplored, yet holds potential for identifying new biomarkers and therapeutic targets in acute lymphoblastic leukemia (ALL). In this study, we conducted a systematic analysis of recurrent somatic non‐coding single nucleotide variants (SNVs) in pediatric B‐cell precursor (BCP) ALL. We leveraged whole genome sequencing (WGS) data from 345 pediatric BCP ALL cases, representing all major genetic subtypes and identified 346 mutational hotspots that harbored somatic SNVs in at least three cases. Through the integration of paired RNA sequencing along with published ChIP‐seq and ATAC‐seq data, we found 128 non‐coding hotspots associated with differentially expressed genes nearby, which were enriched for cis‐regulatory elements, demonstrating the effectiveness of multi‐omics integration in distinguishing pathogenic mutations from passengers. We identified one mutational hotspot that was associated with increased expression of the leukemia‐associated gene NRAS in three primary ALLs. Micro‐C analysis in the leukemia cell line demonstrated interactions between the hotspot region and NRAS regulatory elements. Dual luciferase assays indicated that the mutations disrupted regulatory interactions and CRISPR‐mediated deletion of the region significantly upregulated NRAS, confirming the hypothesized regulatory link. Altogether, we provide new insights into the functional roles of non‐coding mutations in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6089d599cb2b181049865efd98f322d1bb52759" target='_blank'>
              Discovery of Cis‐Regulatory Mechanisms via Non‐Coding Mutations in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Efe Aydın, Eleanor L Woodward, Gladys Telliam Dushime, Rebeqa Gunnarsson, H. Lilljebjörn, L. Moura-Castro, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Prior studies showed that structural loops collapse upon acute cohesin depletion, while regulatory enhancer-promoter (E-P) loops largely persist, consistent with minimal transcriptional changes. However, these studies, conducted in asynchronous cells, could not resolve whether cohesin is required for the establishment of regulatory interactions and transcriptional activation during cell division or cell state transitions. To address this gap, we degraded RAD21, a core cohesin subunit, in naïve mouse embryonic stem cells (ESCs) transitioning from mitosis to G1 either in self-renewal condition or during differentiation toward formative pluripotency. Although most structural loops failed to be re-established without cohesin, about 35% of regulatory loops reformed at normal or higher frequencies. Cohesin-independent loops showed characteristics of strong active enhancers and promoters and a significant association with H3K27ac mitotic bookmarks. However, inhibition of CBP/p300 during mitotic exit did not impact these cohesin-independent interactions, suggesting the presence of complex compensatory mechanisms. At the transcriptional level, cohesin depletion induced only minor changes, supporting that post-mitotic transcriptional reactivation is largely independent of cohesin. The few genes with impaired reactivation were directly bound by RAD21 at their promoters, engaged in many structural loops, and located within strongly insulated TADs with low gene density. Importantly, degrading cohesin during the M-to-G1 transition in the presence of EpiLC differentiation signals revealed a larger group of susceptible genes, including key signature genes and transcription factors. Impaired activation of these genes was partly due to the failure to establish de novo EpiLC-specific interactions in the absence of cohesin. These experiments revealed locus-specific and context-specific dependencies between cohesin, E-P interactions, and transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3acd22c348292a09acc57cb36b9467e0afd8f399" target='_blank'>
              Post-mitotic transcriptional activation and 3D regulatory interactions show locus- and differentiation-specific sensitivity to cohesin depletion
              </a>
            </td>
          <td>
            UkJin Lee, Alejandra Laguillo-Diego, Wilfred Wong, Zhangli Ni, Lingling Cheng, Jieru Li, B. Pelham-Webb, A. Pertsinidis, Christina S. Leslie, E. Apostolou
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="In many organisms, meiotic crossover recombination is suppressed near the extreme ends of chromosomes. Here, we identified two chromatin modifiers, the histone methyltransferase Dot1 and the Sir silencing complex, as regulators of this process in Saccharomyces cerevisiae. We show that the recombination-promoting axis proteins Red1 and Hop1, but not the axis-associated cohesin Rec8, are significantly reduced within 20 kb of telomeres compared to the chromosome interior. Dot1, which preferentially methylates histones in the chromosome interior, is required for this pattern by directing Red1 binding toward the chromosome interior. In parallel, the Sir complex suppresses the induction of meiotic DNA double-strand breaks (DSBs) at chromosome ends. Sir-dependent DSB suppression is independent of axis deposition and occurs in a chromosome end-specific manner that mirrors the spreading and transcriptional silencing activity of the complex, suggesting that the Sir complex suppresses DSB formation by limiting the openness of promoters, the preferred sites of meiotic DSB formation. We conclude that multiple chromatin-based mechanisms collaborate to achieve a robust reduction of meiotic recombination near chromosome ends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091c445669f4b199e48685b922281d0349f41b22" target='_blank'>
              Distinct chromatin regulators downmodulate meiotic axis formation and DNA break induction at chromosome ends
              </a>
            </td>
          <td>
            A. Raghavan, Kieron May, Vijayalakshmi V. Subramanian, Hannah G. Blitzblau, Neem J. Patel, Jon Houseley, Andreas Hochwagen
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22caf0363586789d3ef13130aa7bdb69bf876e9" target='_blank'>
              Nuclear and genome dynamics underlying DNA double-strand break repair.
              </a>
            </td>
          <td>
            I. Chiolo, Matthias Altmeyer, G. Legube, Karim Mekhail
          </td>
          <td>2025-03-17</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a deadly cancer with an aggressive nature, and how CRC tumor cells manage to translocate and proliferate in a new tissue environment remains not fully understood. Recently, higher-order chromatin structures and spatial genome organization are increasingly implicated in diseases including cancer, but in-depth studies of three-dimensional genome (3D genome) of metastatic cancer are currently lacking, preventing the understanding of the roles of genome organization during metastasis. Here we perform multi-omics profiling of matched normal colon, primary tumor, lymph node metastasis, liver metastasis and normal liver tissue from CRC patients using Hi-C, ATAC-seq and RNA-seq technologies. We find that widespread alteration of 3D chromatin structure is accompanied by dysregulation of genes including SPP1 during the tumorigenesis or metastasis of CRC. Remarkably, the hierarchy of topological associating domain (TAD) changes dynamically, which challenges the traditional view that the TAD structure between tumor and normal tissue is conservative. In addition, we define compartment stability score to measure large-scale alteration in metastatic tumors. To integrate multi-omics data and recognize candidate genes driving cancer metastasis, a pipeline is developed based on Hi-C, RNA-seq and ATAC-seq data. And three candidate genes ARL4C, FLNA, and RGCC are validated to be associated with CRC cell migration and invasion using in vitro knockout experiments. Overall, these data resources and results offer new insights into the involvement of 3D genome in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Regulator of telomere length 1 (RTEL1) helicase facilitates telomere replication by disassembling DNA secondary structures, such as G-quadruplexes and telomeric loops (t-loops), at the ends of the chromosomes. The recruitment of RTEL1 to telomeres occurs during the S-phase of the cell cycle, but the dynamics of the process has not been studied. Here, we utilized CRISPR genome editing and single-molecule imaging to monitor RTEL1 movement within human cell nuclei. RTEL1 utilizes rapid three-dimensional diffusion to search for telomeres and other nuclear targets. Only 5% of the chromatin-bound RTEL1 is associated with telomeres at any time in the S-phase, but the telomere-bound RTEL1 has much more extended associations. This binding is enhanced by the interaction between RTEL1 and the telomeric protein TRF2 but is largely independent of RTEL1 ATPase activity. The absence of RTEL1 catalytic activity leads to severe defects in cell proliferation, slow progression out of S-phase, and chromosome end-to-end fusion events. We propose that the rapid diffusion of RTEL1 allows this low-abundance protein to explore the nucleus, bind TRF2, and be recruited to telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d521287870dd985b41001fde2641677325213069" target='_blank'>
              Rapid dynamics allow the low-abundance RTEL1 helicase to promote telomere replication
              </a>
            </td>
          <td>
            Guanhui Wu, Erin Taylor, Daniel T. Youmans, Nausica Arnoult, Thomas R. Cech
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Helicobacter pylori (H. pylori) infection is a significant risk factor for gastric cancer (GC) development. A growing body of evidence suggests a causal link between infection with H. pylori and increased DNA breakage in the host cells. While several mechanisms have been proposed for this damage, their relative impact on the overall bacterial genotoxicity is unknown. Moreover, the link between the formation of DNA damage following infection and the emergence of cancerous structural variants (SV) in the genome of infected cells remained unexplored. Methods We constructed a high-resolution map of genomic H. pylori-induced recurrent break sites using the END-seq method on AGS human gastric cells before and after infection. We next applied END-seq to cycling and arrested cells to identify the role of DNA replication on break formation. Recurrent H. pylori-mediated break sites were further characterized by analyzing published RNA-seq, DRIP-seq, and GRO-seq data at these sites. γH2AX staining and comet assay were used for DNA breakage quantification. Liquid chromatography-mass spectrometry (LC–MS) assay was used to quantify cellular concentrations of dNTPs. Results Our data indicated that sites of recurrent H. pylori-mediated DNA breaks are ubiquitous across cell types, localized at replication-related fragile sites, and their breakage is dependent on replication. Consistent with that, we found that H. pylori inflicts nucleotide depletion, and that rescuing the cellular nucleotide pool largely reduced H. pylori-induced DNA breaks. Intriguingly, we found that this genotoxic mechanism operates independently of H. pylori cag pathogenicity island (CagPAI) that encodes for the bacterial type 4 secretion system (T4SS), and its virulence factor, CagA, which was previously implicated in increasing DNA damage by downregulating the DNA damage response. Finally, we show that sites of recurrent H. pylori-mediated breaks coincide with chromosomal deletions observed in patients with intestinal-type GC and that this link potentially elucidates the persistent transcriptional alterations observed in cancer driver genes. Conclusions Our findings indicate that dNTP depletion by H. pylori is a key component of its genotoxicity and suggest a link between H. pylori genotoxicity and its oncogenic potential. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01439-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab99623f5d226080b2ad6d750b3bbe647805883" target='_blank'>
              The landscape of Helicobacter pylori-mediated DNA breaks links bacterial genotoxicity to its oncogenic potential
              </a>
            </td>
          <td>
            Hadas Sibony-Benyamini, Rose Jbara, Tania Shubash Napso, Layan Abu-Rahmoun, Daniel Vizenblit, Michal Easton-Mor, Shira Perez, Alexander Brandis, Tamar Leshem, Avi Peretz, Yaakov Maman
          </td>
          <td>2025-02-24</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background B chromosomes contribute to the genetic variation in numerous eukaryotes. Yet their genetic and epigenetic characteristics, as well as their effects on the host genome remain poorly understood. Results Here, we present a comprehensive genome assembly of diploid maize B73 with two copies of B chromosomes using long-read sequencing. We annotate a total of 1124 high-confidence protein-coding genes and 119,579,190 bp repeat elements representing 88.55% of the B chromosome assembly. Using CENH3 ChIP-seq data, we accurately determined the position of the B chromosome centromere, which features a unique monomer-composed satellite array distinct from that found on the chromosome arms. Our research provides detailed genetic and epigenetic maps of the B chromosome, shedding light on its molecular landscape, including DNA sequence composition, DNA methylation patterns, histone modifications, and R-loop distributions across various chromatin regions. Consistent with the cytological morphology of the B chromosome, the less condensed euchromatin regions displayed high levels of H3K4me3, H3K9ac, gene expression, and dense R-loop distributions. DNA methylation on the B chromosome was primarily observed at CG sites. The centromeric region is notably enriched with H3K4me3 and H3K9ac histone modifications and has lower CHG methylation compared to the pericentromeric regions. Moreover, our findings reveal that B chromosome accumulation affects R-loop formation on A chromosomes, and exerts tissue-specific influences on A chromosome gene expression. Conclusions The accurate assembly and detailed epigenetic maps of the maize B chromosome will help understand the drive mechanism, reveal its conflict with the host genome, and accelerate the construction of artificial chromosomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03517-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07c5cd6707342f2b03c56ee98d0f9642c61426f7" target='_blank'>
              Genome assembly of the maize B chromosome provides insight into its epigenetic characteristics and effects on the host genome
              </a>
            </td>
          <td>
            Qian Liu, Yang Liu, Congyang Yi, Zhi Gao, Zeyan Zhang, Congle Zhu, J. Birchler, Fang Han
          </td>
          <td>2025-03-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548772b9695227d26aad36aaec8b4ab6abb6678b" target='_blank'>
              The Chromosome 19 miRNA Cluster Guards Trophoblasts Against Overacting Innate Immunity
              </a>
            </td>
          <td>
            J. Mouillet, Yingshi Ouyang, E. Sadovsky, Vinoth K. Kothnadan, Heather L. Sorenson, Logan J. Badeau, S. Sarkar, Tianjiao Chu, Alexander Sorkin, Y. Sadovsky
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Replicative stress (RS) is emerging as a promising therapeutic target in oncology, yet full exploitation of its potential requires a detailed understanding of the mechanisms and genes involved. Here, we investigated the RNA helicase Senataxin (SETX), an enzyme that resolves RNA-DNA hybrids and R-loops, to address its role in preventing RS by oncogenic Myc. Upon Myc activation, silencing of SETX led to selective engagement of the DNA damage response (DDR) and robust cytotoxicity. Pharmacological dissection of the upstream kinases regulating the DDR uncovered a protective role of the ATR pathway, that once inhibited, boosted SETX driven-DDR. While SETX loss did not lead to a genome-wide increase of R-loops, mechanistic analyses revealed enhanced R-loops localized at DDR-foci and newly replicated genomic loci, compatible with a selective role of SETX in resolving RNA-DNA hybrids to alleviate Myc-induced RS. Genome-wide mapping of DNA double-strand breaks confirmed that SETX silencing exacerbated DNA damage at transcription-replication conflict (TRC) regions at early replicated sites. We propose that SETX prevents Myc-induced TRCs by resolving transcription-associated R-loops that encounter the replisome. The identification of SETX as a genetic liability of oncogenic Myc opens up new therapeutic options against aggressive Myc-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer exhibits altered metabolism, influencing its response to radiation. To investigate the metabolic regulation of radiation response, we conducted a comprehensive, metabolic-wide CRISPR-Cas9 loss-of-function screen using radiation as selection pressure in human non–small cell lung cancer. Lipoylation emerged as a key metabolic target for radiosensitization, with lipoyltransferase 1 (LIPT1) identified as a top hit. LIPT1 covalently conjugates mitochondrial 2-ketoacid dehydrogenases with lipoic acid, facilitating enzymatic functions involved in the tricarboxylic acid cycle. Inhibiting lipoylation, either through genetic LIPT1 knockout or a lipoylation inhibitor (CPI-613), enhanced tumor control by radiation. Mechanistically, lipoylation inhibition increased 2-hydroxyglutarate, leading to H3K9 trimethylation, disrupting TIP60 recruitment and ataxia telangiectasia mutated (ATM)–mediated DNA damage repair signaling, impairing homologous recombination repair. In summary, our findings reveal a critical role of LIPT1 in regulating DNA damage and chromosome stability and may suggest a means to enhance therapeutic outcomes with DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c8836655ab7f9cea8b4528bffe4e53726a560" target='_blank'>
              Lipoylation inhibition enhances radiation control of lung cancer by suppressing homologous recombination DNA damage repair
              </a>
            </td>
          <td>
            Jui-Chung Chiang, Zeng Shang, Tracy Rosales, Ling Cai, Wei-Min Chen, Feng Cai, Hieu S Vu, John D Minna, Min Ni, Anthony J. Davis, Robert D Timmerman, Ralph J DeBerardinis, Yuanyuan Zhang
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1afe205564d682ee54296311b108e6d8a61e8b1" target='_blank'>
              Spatial mapping of DNA synthesis reveals dynamics and geometry of human replication nanostructures
              </a>
            </td>
          <td>
            Michael Hawgood, Bruno Urién González, Ana Agostinho, Praghadhesh Thiagarajan, Yiqiu Yang, Xue Zhang, Giovanni Giglio, Gemma Quijada, Matilde Fonseca, J. Bartek, Hans Blom, B. Lemmens
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="YTHDC1, a key protein in the m6A-related regulatory network within cells, is involved in multiple cellular processes such as chromatin-related regulation, RNA splicing, and nuclear export. Understanding its role in colorectal cancer (CRC) development and DNA damage repair is critical for the advancement of treatment strategies. Our study found that YTHDC1 was highly expressed in high-malignancy CRC tissues compared with low-malignancy ones. Upon silencing YTHDC1, we observed a pronounced suppression of the proliferation of CRC cell lines, accompanied by a substantial increase in cell apoptosis. Furthermore, we identified RAD51 as a crucial downstream target of YTHDC1. Knocking down YTHDC1 led to a notable decrease in RAD51 protein levels, and silencing RAD51 also inhibited cancer cell proliferation. Interestingly, RNA-sequencing data indicated that the YTHDC1 deletion did not affect RAD51 transcription. However, Western blot revealed that this deletion increased the ubiquitination of RAD51, likely due to the upregulated E3 ligase UBE3A. Ubiquitination experiments subsequently confirmed that RAD51 is indeed one of the substrates of UBE3A. In summary, our study provides novel insights into how YTHDC1 modulates the expression of RAD51 through post-translational modifications. These findings offer valuable information that may potentially contribute to the development of more effective therapeutic strategies for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92925-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cf4d3cb02cc31b95af37c1defc2a6fa9b119aa" target='_blank'>
              YTHDC1 negatively regulates UBE3A to influence RAD51 ubiquitination and inhibit apoptosis in colorectal cancer cells
              </a>
            </td>
          <td>
            Mingwei Gao, Yueguang Wu, Li Zhang, Yan Zhou, Huijuan Liu, Weimin Zhang, Shubin Wang, Yongping Cui
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6227b696ba91084606d106cac80b2ab83ed05f1" target='_blank'>
              Extracellular alpha satellite DNA in human plasma as a novel molecular biomarker for early diagnosis of bladder cancer
              </a>
            </td>
          <td>
            I. Feliciello, D. Dermic
          </td>
          <td>2025-02-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A string of nucleosomes, where genomic DNA is wrapped around histones, is organized in the cell as chromatin, ranging from euchromatin to heterochromatin, with distinct genome functions. Understanding physical differences between euchromatin and heterochromatin is crucial, yet specific labeling methods in living cells remain limited. Here, we have developed replication-dependent histone (Repli-Histo) labeling to mark nucleosomes in euchromatin and heterochromatin based on DNA replication timing. Using this approach, we investigated local nucleosome motion in the four known chromatin classes, from euchromatin to heterochromatin, of living human and mouse cells. The more euchromatic (earlier-replicated) and more heterochromatic (later-replicated) regions exhibit greater and lesser nucleosome motions, respectively. Notably, the motion profile in each chromatin class persists throughout interphase. Genome chromatin is essentially replicated from regions with greater nucleosome motions, although the replication timing is perturbed. Our findings, combined with computational modeling, suggest that earlier-replicated regions have more accessibility, and local chromatin motion can be a major determinant of genome-wide replication timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2b9973e20e3b9a1e91f7d1257dd38b93d96cad" target='_blank'>
              Replication-dependent histone labeling dissects the physical properties of euchromatin/heterochromatin in living human cells
              </a>
            </td>
          <td>
            Katsuhiko Minami, Kako Nakazato, Satoru Ide, Kazunari Kaizu, Koichi Higashi, Sachiko Tamura, Atsushi Toyoda, Koichi Takahashi, Ken Kurokawa, K. Maeshima
          </td>
          <td>2025-03-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The apicoplast organelle of Plasmodium falciparum is essential for parasite’s replication, however the details of its biogenesis, inheritance and regulation throughout the cell cycle are unknown. Here, we report the development of a dynamic imaging platform coupled with an analytical pipeline that enables us to follow and measure subcellular structures throughout the 48-hour cell cycle of live parasites. We reveal a predetermined sequence of four discrete morphological steps in organelle development, which are tightly correlated with nuclear replication. We show that one of these steps, which we term the Crown morphology, is required for nucleus-apicoplast attachment. During Crown, apicoplast is stretched over multiple nuclei, fastened by centriolar tubulin. A complementary molecular approach was used to discover the basic ploidy of apicoplast and mitochondrial genomes, their replication rates and association with nuclear DNA replication. We inhibited nuclear DNA replication and found that it completely blocks apicoplast biogenesis in its most initial stages, demonstrating dependency on S-phase initiation. Conversely, specific inhibition of apicoplast genome replication resulted in an almost-undisturbed organelle development and division. However, it affected the Crown step, preventing association to tubulin-nuclear structures, leading to failure in accurate organelle sorting into daughter cells. Collectively, these experiments reveal a central cellular pathway linking apicoplast development to the parasite’s cell cycle, and a second independent organellar mechanism responsible for segregation into daughter cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bfca923aa81dca1cabb89b41f451971dd56d89" target='_blank'>
              Organelle Development and Inheritance are Driven by Independent Nuclear and Organellar Mechanisms in Malaria Parasites
              </a>
            </td>
          <td>
            Shahar Michal, Qasem Alia, Freedman Eshkar, Maron Yariv, Tissawak Amanda, Florentin Anat
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, P. Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2be5d83a44f2113bcc7ca92950bf19d6878d25c9" target='_blank'>
              Functional landscape of circular RNAs in human cancer cells
              </a>
            </td>
          <td>
            Peter Hyunwuk Her, Tiantian Li, Ziwei Huang, Xin Xu, Weining Yang, Moliang Chen, Mona Teng, Sujun Chen, Yong Zeng, Stanley Liu, Benjamin Haibe-Kains, Fraser Soares, Jie Ming, H. H. He
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cff48bb1e4b4c69ed649cac63da394d5d1ac2d2" target='_blank'>
              Developmental regulation of Drosophila dosage compensation in a 3D genome context
              </a>
            </td>
          <td>
            Lubna Younas, Mujahid Ali, Xinpei Zhang, C. Regnard, Qi Zhou
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatocellular carcinoma is the most prevalent primary liver cancer and arises from hepatocytes, which are very metabolically active and contain abundant mitochondria. Despite extensive characterisation of nuclear driver genes and pathways, the role of crosstalk between the mitochondrial and nuclear genomes in mitochondrial mutagenesis remains poorly understood. To address this, we leveraged a chemical carcinogenesis model of liver cancer in four mouse strains. We analysed hundreds of tumours with paired whole-genome sequencing and total RNA sequencing to comprehensively characterise mitochondrial mutations and expression. Following de novo assembly of three new mitochondrial genomes, we devised an accurate and efficient heteroplasmy detection approach and analysed mitochondrial DNA (mtDNA) content and transcription. Across strains, there was high concordance of the number of heteroplasmies, mutation signatures, and variant allele frequencies. In contrast, the heteroplasmy rate and genic locations differed between strains, suggesting strain-specificity in DNA repair mechanisms in early tumour development. Independent of age or environmental factors, tumours had lower mtDNA content than adjacent normal liver, suggesting mitochondrial loss during tumourigenesis. Additionally, within individual strains, mtDNA content differences were associated with nuclear driver gene choice. Our comprehensive characterisation of mitochondrial genomes and expression patterns demonstrates that both germline and somatic genetic variation influence tumour mtDNA content, mutation patterns, and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7507e40e7369ffa2dd7aab3b39dffb6074ecd240" target='_blank'>
              Germline and somatic variation influence nuclear-mitochondrial crosstalk in tumourigenesis
              </a>
            </td>
          <td>
            Maëlle Daunesse, Vasavi Sundaram, Elissavet Kentepozidou, John Connelly, Craig J. Anderson, Núria López-Bigas, Colin A. Semple, D. T. Odom, Martin S. Taylor, Maša Roller, Sarah J. Aitken, Paul Flicek
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. The ribosomal DNA (rDNA) region is particularly vulnerable to such changes due to its repetitive nature. Here, we demonstrate that Rad27/FEN-1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining rDNA stability. The production of extrachromosomal rDNA circles and severe chromosomal rDNA instability are observed in the rad27Δ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates unprocessed Okazaki fragments in the rDNA region, without inducing DSB formation. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, we show that Exonuclease 1 and PCNA can compensate for the loss of Rad27 function in the rDNA stabilization. These findings highlight the importance of proper Okazaki fragment processing in preventing non-DSB-induced rDNA copy number changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35141ab23e752c4a99955ef6952f7857bccfbbe" target='_blank'>
              Rad27/FEN1 prevents accumulation of unprocessed Okazaki fragments and ribosomal DNA copy number changes
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1df029231423cb536b06a558936a00ba5d9246d" target='_blank'>
              PRAME Epitopes are T-Cell Immunovulnerabilities in BRD4::NUTM1 Initiated NUT Carcinoma
              </a>
            </td>
          <td>
            Jeffrey L. Jensen, Sara K. Peterson, Shawn Yu, Tomoaki Kinjo, Brandon A. Price, Maria Sambade, Steven Vesko, Johnathan D. DeBetta, Julie Geyer, Kwang P Nickel, Randall J. Kimple, R. Kotecha, Ian J. Davis, Jeremy R. Wang, Christopher A. French, Brian Kuhlman, Alex Rubinsteyn, Jared Weiss, Benjamin G. Vincent
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic instability and inflammation are distinct hallmarks of aging, but the connection between them is poorly understood. Here we report a mechanism directly linking genomic instability and inflammation in senescent cells through a mitochondria-regulated molecular circuit involving p53 and cytoplasmic chromatin fragments (CCF) that are enriched for DNA damage signaling marker γH2A.X. We show that p53 suppresses CCF accumulation and its downstream inflammatory phenotype. p53 activation suppresses CCF formation linked to enhanced DNA repair and genome integrity. Activation of p53 in aged mice by pharmacological inhibition of MDM2 reverses transcriptomic signatures of aging and age-associated accumulation of monocytes and macrophages in liver. Mitochondrial ablation in senescent cells suppresses CCF formation and activates p53 in an ATM-dependent manner, suggesting that mitochondria-dependent formation of γH2A.X + CCF dampens nuclear DNA damage signaling and p53 activity. These data provide evidence for a mitochondria-regulated p53 signaling circuit in senescent cells that controls DNA repair, genome integrity, and senescence- and age-associated inflammation, with relevance to therapeutic targeting of age-associated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4941baca9720335671879d8e4ca5f2fe5568326" target='_blank'>
              p53 enhances DNA repair and suppresses cytoplasmic chromatin fragments and inflammation in senescent cells
              </a>
            </td>
          <td>
            Karl N. Miller, Brightany Li, Hannah R. Pierce-Hoffman, Shreeya Patel, Xue Lei, Adarsh Rajesh, M. G. Teneche, Aaron Havas, Armin Gandhi, Carolina Cano Macip, Jun Lyu, Stella G. Victorelli, Seung-Hwa Woo, A. Lagnado, Michael A LaPorta, Tianhui Liu, Nirmalya Dasgupta, Sha Li, Andrew Davis, Anatoly Korotkov, Erik Hultenius, Zichen Gao, Yoav Altman, Rebecca A Porritt, Guillermina Garcia, C. Mogler, A. Seluanov, Vera Gorbunova, S. Kaech, Xiao Tian, Zhixun Dou, Chongyi Chen, João F. Passos, P. Adams
          </td>
          <td>2025-03-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="FANCM is branchpoint DNA translocase essential for cellular response to replication stress. Here, we show that replication stress stimulates FANCM and the TONSL-MMS22L heterodimer bound to histones H3-H4 to form an interdependent complex on chromatin. TONSL-MMS22L recruits FANCM and Fanconi anemia (FA) core complex to stalled and collapsed forks, maintains FANCM on replication-stressed chromatin, promotes FANCD2 monoubiquitination, facilitates both repair and replication traverse of DNA interstrand crosslinks (ICLs), and suppresses sister chromatid exchanges, through its interactions with FANCM and H3-H4. Reciprocally, both DNA translocase activity and phosphorylation of FANCM facilitate recruitment of TONSL-MMS22L and RAD51 to perturbed forks. Moreover, TONSL-MMS22L and FANCM function together to promote activation of the FA pathway, ICL repair, homologous recombination and replication traverse. Cancer patients with tumors with wildtype FANCM and low expression of TONSL-MMS22L have a more favorable prognosis than those with high expression. Thus, FANCM-TONSL-MMS22L acts coordinately as a complex on chromatin that resolves replication stress, and this complex may present a therapeutic target for wildtype FANCM-linked cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6a7b1f902e81794874e5ad4e122751d4fecd95" target='_blank'>
              The TONSL-MMS22L complex and FANCM form an interdependent complex on chromatin to counter replication stress
              </a>
            </td>
          <td>
            Haixia Zhou, Jiaoyan Yan, Xinlei Cao, Jingfei Zhan, Chen Ling, Yunhui Luo, Zhuang Ma, Yinyying Sun, Peining Song, Weiwei Liu, Lizhu Wang, Jiahuan Li, Althaf Shaik, M. Bellani, Linqian Wang, Yun Xie, Jing Zhang, Xiangyang Xue, Xian Shen, Michael M. Seidman, Weidong Wang, Zhijiang Yan
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background CTCF is considered as the most essential transcription factor regulating chromatin architecture and gene expression. However, genome-wide impact of CTCF on erythropoiesis has not been extensively investigated. Results Using a state-of-the-art human erythroid progenitor cell model (HUDEP-2 and HEL cell lines), we systematically investigate the effects of acute CTCF loss by an auxin-inducible degron system on transcriptional programs, chromatin accessibility, CTCF genome occupancy, and genome architecture. By integrating multi-omics datasets, we reveal that acute CTCF loss notably disrupts genome-wide chromatin accessibility and the transcription network. We detect over thousands of decreased chromatin accessibility regions but only a few hundred increased regions after CTCF depletion in HUDEP-2 and HEL lines, suggesting the role of CTCF in maintaining proper chromatin openness in the erythroid lineage. CTCF depletion in the erythroid context notably disrupts the boundary integrity of topologically associating domains and chromatin loops but does not affect nuclear compartmentalization. We find erythroid lineage-specific genes, including some metabolism-related genes, are suppressed at immature and mature stages. Notably, we find a subset of genes whose transcriptional levels increase upon CTCF depletion, accompanied by decreased chromatin accessibility regions enriched with the GATA motif. We further decipher the molecular mechanism underlying the CTCF/GATA2 repression axis through distal non-coding chromatin regions. These results suggest a suppressive role of CTCF in gene expression during erythroid lineage specification. Conclusions Our study reveals a novel role of CTCF in regulating erythroid differentiation by maintaining its proper chromatin openness and gene expression network, which extends our understanding of CTCF biology. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03510-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861d404b3fd5272bc30487378b974c5ae16b843d" target='_blank'>
              CTCF is selectively required for maintaining chromatin accessibility and gene expression in human erythropoiesis
              </a>
            </td>
          <td>
            Xue Yang, Li Cheng, Ye Xin, Jianxiang Zhang, Xinfeng Chen, Jinchao Xu, Mengli Zhang, Ruopeng Feng, Judith Hyle, Wenjie Qi, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li, Peng Xu
          </td>
          <td>2025-02-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a58a1f7cb1b444929655b075aa93afb10f16c55" target='_blank'>
              Evolutionary hotspots of structural variation drive inter-individual differences in the expression of fusion transcripts in the human brain
              </a>
            </td>
          <td>
            Colette Moses, Angelica Vanini, Maria Chiara Bocchi, Luca Wagner, Isis van Loenen, Brooke L. Latour, N. N. Kasri, F. Jacobs
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The ability to replicate and pass information to descendants is a fundamental requirement for life. In addition to the DNA-based genetic information, modifications of the DNA or DNA-associated proteins can create patterns of heritable gene regulation. Such epigenetic inheritance allows for adaptation without mutation, but its limits and regulation are incompletely understood. Here we developed a C. elegans system to study the transgenerational epigenetic inheritance of H3K27me3, a conserved histone posttranslational modification associated with gene repression. We find that induced alterations of the genome-wide H3K27me3 landscape and the associated fertility defects persist for many generations in genetically wildtype descendants under selective pressure. We uncover that the inheritance of the altered H3K27me3 landscape is regulated by two chromodomain proteins with antagonizing functions, and provide mechanistic insight into how this molecular memory is initiated and maintained. Our results demonstrate that epigenetic inheritance can act as a mutation-independent, heritable mechanism of adaptation. In Brief Özdemir et al. demonstrate that an altered genomic distribution of the histone modification H3K27me3 can be epigenetically inherited across many generations through the activity of HERI-1/SET-32/MES-4, which is antagonized by CEC-6/PRC2 in C. elegans. Highlights Altered H3K27me3 landscapes can be inherited for at least 15 generations in C. elegans. The chromodomain proteins CEC-6 and HERI-1 have opposite roles in antagonizing or promoting the maintenance of the altered H3K27me3 landscape. H3K23me3 and H3K36me3 replace H3K27me3 to promote the intergenerational and transgenerational inheritance of the altered epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9477951c95e1934dfca6ddb054dc1db83139c5" target='_blank'>
              Transgenerational maintenance of H3K27me3 heterochromatin is balanced by chromodomain proteins in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Isa Özdemir, Anna Höfler, Kamila Delaney, Joanna M. Wenda, Chengyin Li, Arneet L. Saltzman, Andreas Boland, Florian A. Steiner
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1ae2ca598b5cdc041c4c163b222a0b0f6d55027" target='_blank'>
              Tumor suppressor collateral damage screens reveal mRNA homeostasis protein HBS1L as a novel vulnerability in ch9p21 driven FOCAD deleted cancer
              </a>
            </td>
          <td>
            Hongxiang Zhang, M. Tonini, Lauren Martires, Charlotte B Pratt, Helena N. Jenkins, Eden N. Gordon, Shanchuan Zhao, Ashley H Choi, Samuel R. Meier, Tenzing Khendu, Shangtao Liu, B. Shen, Hannah Stowe, Katerina Pashiardis, Xuewen Pan, Madhavi Bandi, Minjie Zhang, Yi Yu, Chengyin Min, A. Huang, Jannik N Andersen, Hilary E. Nicholson, T. Teng
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The whole genome karyotype refers to the sequence of large chromosomal segments that make up an individual genotype. karyotype analysis, which includes descriptions of aneuploidies and other rearrangements is crucial for understanding genetic risk factors, for diagnosis, treatment decisions, and genetic counseling linked to constitutional disorders. The current karyotyping standard is based on microscopic examination of chromosomes, a complex process that requires high expertise and offers Mb scale resolution. Optical Genome Mapping (OGM) technology can identify large DNA lesions in a cost-effective manner. In this paper, we developed OMKar, a method that uses OGM data to create a virtual karyotype. OMKar processes Structural (SV) and Copy Number (CN) Variants as inputs and encodes them into a compact breakpoint graph. It recomputes copy numbers using Integer Linear Programming to maintain CN balance and then identifies constrained Eulerian paths representing entire donor chromosomes. In tests using 38 whole genome simulations of constitutional disorders, OMKar reconstructed the karyotype with 88% precision and 95% recall on SV concordance and 95% Jaccard score on CN concordance. We applied OMKar to 50 prenatal, 41 postnatal, and 63 parental samples from ten different sites. OMKar reconstructed the correct karyotype in 144 out of 154 samples, covering 25 of 25 aneuploidies, 32 of 32 balanced translocations, and 72 of 82 unbalanced variations. Detected constitutional disorders included Cri-du-chat, Wolf-Hirschhorn, Prader-Willi deletions, Down, and Turner syndromes. Importantly, it identified a plausible genetic mechanism for five cases of constitutional disorder that were not detected by other technologies. Together, these results demonstrate the robustness of OMKar for OGM-based karyotyping. OMKar is publicly available at https://github.com/siavashre/OMKar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f601f0712cbf1fcd9d4ef4e675b52652a6476c" target='_blank'>
              OMKar: optical map based automated karyotyping of genomes to identify constitutional abnormalities
              </a>
            </td>
          <td>
            S. R. Dehkordi, Zhaoyang Jia, Joey Estabrook, Jen Hauenstein, Neil Miller, Naz Güleray-Lafci, Jürgen Neesen, Alex Hastie, A. Chaubey, Andy Wing Chun Pang, Paul Dremsek, V. Bafna
          </td>
          <td>2025-02-14</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1bc086a943a7e73ef25816e4af3f165ae6d1" target='_blank'>
              PARP inhibitors affect the replisome and replication fork progression during a single S phase
              </a>
            </td>
          <td>
            Lina-Marie Briu, C. Maric, Nicolas Valentin, Nicolas Panara, Guillaume Chevreux, Giuseppe Baldacci, Jean-Charles Cadoret
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b525ba6493ef6d26b4e42891bed37752a529ff9f" target='_blank'>
              Rewiring the fusion oncoprotein EWS/FLI1 in Ewing sarcoma with bivalent small molecules
              </a>
            </td>
          <td>
            Michael J. Bond, Ryan P Golden, Giulia Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, N. Gray, K. Stegmaier
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3087cc8f484e07e82cdb51614296a13d7efc4fb4" target='_blank'>
              Colorectal cancer progression to metastasis is associated with dynamic genome-wide biphasic 5-hydroxymethylcytosine accumulation
              </a>
            </td>
          <td>
            Ben Murcott, Floris Honig, Dominic Oliver Halliwell, Yuan Tian, J. L. Robson, Piotr J. Manasterski, Jennifer Pinnell, Thérèse Dix-Peek, Santiago Uribe-Lewis, Ashraf EK Ibrahim, Julia Sero, David Gurovich, Nikolas Nikolai, Adele Murrell
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D. Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94444bc68fc048c7923ba3b90250650c889cfa29" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer that drives metastasis, immune evasion and treatment resistance. CIN may result from chromosome mis-segregation errors and excessive chromatin is frequently packaged in micronuclei (MN), which can be enumerated to quantify CIN. The assessment of CIN remains a predominantly manual and time-consuming task. Here, we present micronuclAI, a pipeline for automated and reliable quantification of MN of varying size and morphology in cells stained only for DNA. micronuclAI can achieve close to human-level performance on various human and murine cancer cell line datasets. The pipeline achieved a Pearson’s correlation of 0.9278 on images obtained at 10X magnification. We tested the approach in otherwise isogenic cell lines in which we genetically dialed up or down CIN rates, and on several publicly available image datasets where we achieved a Pearson’s correlation of 0.9620. Given the increasing interest in developing therapies for CIN-driven cancers, this method provides an important, scalable, and rapid approach to quantifying CIN on images that are routinely obtained for research purposes. We release a GUI-implementation for easy access and utilization of the pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad501d60562e0f7baf32d920b40de21a7a01759" target='_blank'>
              micronuclAI enables automated quantification of micronuclei for assessment of chromosomal instability
              </a>
            </td>
          <td>
            Miguel A. Ibarra-Arellano, Lindsay A. Caprio, Aroj Hada, Niklas Stotzem, Luke L Cai, Shivem B Shah, Zachary H. Walsh, Johannes C. Melms, Florian Wünneman, K. Bestak, Ibrahim Mansaray, Benjamin Izar, Denis Schapiro
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Trypanosoma cruzi possesses hundreds of genes associated with its pathogenesis. The extent and organization of this diverse gene repertoire, expression, and role in infection remain unclear. Using accurate long-read sequencing and chromatin conformation capture, T. cruzi chromosomes were assembled from telomere-to-telomere. The genome revealed multigene families of virulence genes accounting for ∼70% of some chromosomes, organized in clusters or scattered through housekeeping genes. Quantitative proteomics identified stage-specific proteins and numerous trans-sialidases upregulated in trypomastigotes. Notably, the expression of virulence gene families changed stochastically in trypomastigotes, conferring T. cruzi fitness to cardiomyocyte infection while diversifying the invasion to multiple tissues. A T. cruzi genome-wide yeast surface display screen against Chagas disease patients’ antibodies revealed virulence genes expressed during human infections. However, limited conservation in their antibody-binding sites suggests their sequence diversity and variation help parasites avert antibody recognition. The data point to a role for virulence multigene families in infection persistence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7300cb53bb30ef4e8c826b39532820f2001e0681" target='_blank'>
              Stochastic variation in surface protein expression diversifies Trypanosoma cruzi infection
              </a>
            </td>
          <td>
            Lissa Cruz Saavedra, Mira Loock, Luiza Berenguer Antunes, I. Cestari
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93a4683d0794daa101a332796e1ced942dd7fd5" target='_blank'>
              A Novel De Novo Exceptional Complex Chromosomal Rearrangement Involving 5 Chromosomes Resulting in Neurodevelopmental Delay and Dysmorphism
              </a>
            </td>
          <td>
            Sabri Aynacı, Sinem Kocagil, C. Yarar, Efsun Tosumoğlu, Ebru Erzurumluoğlu Gökalp, Mehmet Burak Mutlu, S. Artan
          </td>
          <td>2025-03-25</td>
          <td>Molecular Syndromology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Research on chromosome organization and cell cycle progression in spherical bacteria, particularly Staphylococcus aureus, remains limited and fragmented. In this study, we established a working model to investigate chromosome dynamics in S. aureus using a Fluorescent Repressor-Operator System (FROS), which enabled precise localization of specific chromosomal loci. This approach revealed that the S. aureus cell cycle and chromosome replication cycle are not coupled, with cells exhibiting two segregated origins of replication at the start of the cell cycle. The chromosome has a specific origin-terminus-origin conformation, with origins localizing near the membrane, towards the tip of each hemisphere, or the “cell poles”. We further used this system to assess the role of various proteins with a role in S. aureus chromosome biology, focusing on the ParB-parS and SMC-ScpAB systems. Our results demonstrate that ParB binds five parS chromosomal sequences and the resulting complexes influence chromosome conformation, but play a minor role in chromosome compaction and segregation. In contrast, the SMC-ScpAB complex plays a key role in S. aureus chromosome biology, contributing to chromosome compaction, segregation and spatial organization. Additionally, we systematically assessed and compared the impact of proteins linking chromosome segregation to cell division—Noc, FtsK, SpoIIIE and XerC—on origin and terminus number and positioning. This work provides a comprehensive study of the factors governing chromosome dynamics and organization in S. aureus, contributing to our knowledge on chromosome biology of spherical bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbeb0ffed138b49203f6a896929817f3b337f1fa" target='_blank'>
              Chromosome segregation dynamics during the cell cycle of Staphylococcus aureus
              </a>
            </td>
          <td>
            Adrian Izquierdo-Martinez, Simon Schäper, António D. Brito, Qinchan Liao, Coralie Tesseur, Moritz Sorg, Daniela S. Botinas, Xindan Wang, M. G. Pinho
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, Lodewyk F. A. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Genes that escape X-chromosome inactivation (XCI) in female somatic cells vary in number and levels of escape among mammalian species and tissues, potentially contributing to species- and tissue-specific sex differences. CTCF, a master chromatin conformation regulator, is enriched at escape regions and may play an important role in regulating escape, but the molecular mechanisms remain elusive. Results CTCF binding profiles and epigenetic features were systematically examined at escape genes (escapees) using mouse allelic systems with skewed XCI to distinguish the inactive X (Xi) and active X (Xa) chromosomes. We found that six constitutive and two facultative escapees are located inside 30-800 kb domains marked by convergent arrays of CTCF binding sites, consistent with the formation of chromatin loops. Facultative escapees show clear differences in CTCF binding depending on their XCI status in specific cell types/tissues. In addition, sets of strong and in some cases divergent CTCF binding sites located at the boundary between an escapee and its adjacent neighbors subject to XCI would also help insulate domains. Indeed, deletion but not inversion of a CTCF binding site at the boundary between the facultative escapee Car5b and its silent neighbor Siah1b results in a dramatic reduction of Car5b escape. This is associated with reduced CTCF and cohesin binding, which indicates loss of looping and insulation and is supported by 3C combined with Hi-C analysis. In addition, enrichment in the repressive mark H3K27me3 invades the Car5b domain in deleted cells, consistent with loss of expression from the Xi. In contrast, cells with an inversion of the CTCF binding site retain CTCF and cohesin binding, as well as looping, in line with persistence of escape. Interestingly, the levels of escape increase in cells with deletion of either Dxz4, which disrupts the Xi-specific compact 3D structure, or Firre, which results in lower H3K27me3 enrichment on the Xi, indicating that the structural and epigenetic features of the Xi constrain escape from XCI in wild type conditions. Conclusions Taken together, our findings support the idea that escape from XCI in female somatic cells is modulated by both the topological insulation of domains via CTCF binding and the surrounding heterochromatin environment. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02137-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fde8cc47c2c193748a3f06ce03baac9ac12fe7" target='_blank'>
              CTCF-mediated insulation and chromatin environment modulate Car5b escape from X inactivation
              </a>
            </td>
          <td>
            H. Fang, A. Tronco, G. Bonora, Truong Nguyen, Jitendra Thakur, J. B. Berletch, Galina N. Filippova, S. Henikoff, Jay Shendure, W. S. Noble, Zhijun Duan, Christine M. Disteche, Xinxian Deng
          </td>
          <td>2025-03-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38eeccd7f5dc5d4b304bd2c0df97ce58a72f338a" target='_blank'>
              RAmbler resolves complex repeats in human Chromosomes 8, 19, and X.
              </a>
            </td>
          <td>
            Sakshar Chakravarty, Glennis Logsdon, Stefano Lonardi
          </td>
          <td>2025-03-04</td>
          <td>Genome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Antisense oligonucleotides (ASOs) and small interfering RNA (siRNA) therapeutics highlight the power of oligonucleotides in silencing disease-causing messenger RNAs (mRNAs). Another promising class of gene-silencing oligonucleotides is RNA-cleaving nucleic acid enzymes, which offer the potential for allele-specific RNA inhibition with greater precision than ASOs and siRNAs. Herein, we chemically evolved the nucleolytic DNA enzyme (DNAzyme) 10–23, by incorporating the modifications that are essential to the success of ASO drugs, including 2′-fluoro, 2′-O-methyl, and 2′-O-methoxyethyl RNA analogues, and backbone phosphorothioate, to enhance catalytic efficiency by promoting RNA substrate binding and preventing dimerization of 10–23. These ASO-like DNAzymes cleaved structured RNA targets in long transcripts, showed prolonged intracellular stability, and downregulated mRNA and protein levels of both exogenously transfected eGFP and endogenously elevated oncogenic c-MYC. In colon cancer HCT116 cells, the downregulation of oncogenic c-MYC RNA resulted in cell cycle arrest, reduced proliferation, and increased apoptosis. RACE (rapid amplification of cDNA ends) polymerase chain reaction and Sanger sequencing confirmed precise, site-specific mRNA transcript cleavage with minimal RNase H activation in cells. By merging ASO structural and pharmacokinetic advantages with DNAzyme catalytic versatility, these ASO-like 10–23 variants offer a promising new class of potent gene-silencing agents, representing a significant step toward therapeutic DNAzyme development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0676a1fc6fbd741743702722cb6c5dbd58ba89e" target='_blank'>
              Chemical evolution of ASO-like DNAzymes for effective and extended gene silencing in cells
              </a>
            </td>
          <td>
            Yingyu Liu, Sheyu Zhang, Meiqi Zhang, Xin Liu, Yashu Wu, Qin Wu, John C Chaput, Yajun Wang
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be267d81c51ec120632413ac620f868fe9f5634" target='_blank'>
              Rapid delivery of RNase H1 enables replication fork progression under stress independently of R-loop resolution
              </a>
            </td>
          <td>
            Amandine Basille, Yea-Lih Lin, Connor Kirke, D. Cluet, Yara Bawadi, Laetitia Vachez, Virgile Dervieux, Laura Guiguettaz, Pascal Bernard, Stephan Hamperl, Philippe Pasero, Emiliano P Ricci, Vincent Vanoosthuyse
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, L. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) shape human genome functionality and influence disease pathogenesis, including cancer. ERVK-7, a significant ERV, acts as an immune modulator and prognostic marker in lung adenocarcinoma (LUAD). Although ERVK-7 overexpression has been linked to the amplification of the 1q22 locus in approximately 10% of LUAD cases, it predominantly arises from alternative regulatory mechanisms. Our findings indicate that the canonical 5’ long terminal repeat (LTR) of ERVK-7 is methylated and inactive, necessitating the use of alternative upstream promoters. We identified two novel transcripts, ERVK-7.long and ERVK-7.short, arising from distinct promoters located 2.8 kb and 13.8 kb upstream of the 5’LTR of ERVK-7, respectively. ERVK-7.long is predominantly overexpressed in LUAD. Through comprehensive epigenetic mapping and single-cell transcriptomics, we demonstrate that ERVK-7.long activation is predetermined by cell lineage, specifically in small airway epithelial cells (SAECs), where its promoter displays tumor-specific H3K4me3 modifications. Single-cell RNA sequencing further reveals a distinct enrichment of ERVK-7.long in LUAD tumor cells and alveolar type 2 epithelial cells, underscoring a cell-type-specific origin. Additionally, inflammatory signaling significantly influences ERVK-7 expression; TNF-α enhances ERVK-7.long, while interferon signaling preferentially augments ERVK-7.short by differential recruitment of NF-κB/RELA and IRF to their respective promoters. This differential regulation clarifies the elevated ERVK-7 expression in LUAD compared to lung squamous cell carcinoma (LUSC). Our study elucidates the complex regulatory mechanisms governing ERVK-7 in LUAD and proposes these transcripts as potential biomarkers and therapeutic targets, offering new avenues to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86f3682460f118255d9b277164814fad089c123" target='_blank'>
              A tumor necrosis factor-α responsive cryptic promoter drives overexpression of the human endogenous retrovirus ERVK-7
              </a>
            </td>
          <td>
            Sojung Lee, Yin Yee Ho, Suyu Hao, Yingqi Ouyang, Ling Liew, Ashish Goyal, Stephen Li, J. Barbour, Mu He, Yuanhua Huang, J. W. Wong
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Monoallelic deletion of the short arm of chromosome 17 (del17p) is a recurrent abnormality in cancers with poor outcomes. Best studied in relation to haematological malignancies, associated functional outcomes are attributed mainly to loss and/or dysfunction of TP53, which is located at 17p13.1, but the wider impact of deletion of other genes located on 17p is poorly understood. 17p is one of the most gene-dense regions of the genome and includes tumour suppressor genes additional to TP53, genes essential for cell survival and proliferation, as well as small and long non-coding RNAs. In this review we utilise a data-driven approach to demarcate the extent of 17p deletion in multiple cancers and identify a common loss-of-function gene signature. We discuss how the resultant loss of heterozygosity (LOH) and haploinsufficiency may influence cell behaviour but also identify vulnerabilities that can potentially be exploited therapeutically. Finally, we highlight how emerging animal and isogenic cell line models of del17p can provide critical biological insights for cancer cell behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Accurate sister chromatid segregation requires remodeling chromosome architecture, decatenation, and attachment to the mitotic spindle. Some of these events are initiated during S-phase, but they accelerate and conclude during mitosis. Here we describe SRBD1 as a histone and nucleic acid binding protein that prevents DNA damage in interphase cells, localizes to nascent DNA during replication and the chromosome scaffold in mitosis, and is required for chromosome segregation. SRBD1 inactivation causes micronuclei, chromatin bridges, and cell death. Inactivating SRBD1 immediately prior to mitotic entry causes anaphase failure, with a reduction in topoisomerase IIα localization to mitotic chromosomes and defects in properly condensing and decatenating chromosomes. In contrast, SRBD1 is not required to complete cell division after chromosomes are condensed. Strikingly, depleting condensin II reduces the severity of the anaphase defects in SRBD1-deficient cells by restoring topoisomerase IIα localization. Thus, SRBD1 is an essential genome maintenance protein required for mitotic chromosome organization and segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966c52bd12100322ff3d9cac1b57f3d49dccb625" target='_blank'>
              SRBD1 facilitates chromosome segregation by promoting topoisomerase IIα localization to mitotic chromosomes
              </a>
            </td>
          <td>
            Courtney A. Lovejoy, Sarah R. Wessel, Rahul Bhowmick, Yuki Hatoyama, Masato T. Kanemaki, Runxiang Zhao, David Cortez
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This study explores the multifaceted role of p21 in mediating cellular responses to DNA-damaging agents, with a focus on doxorubicin treatment in HCT116 colon carcinoma cells. We investigated how different doses of doxorubicin affect cells with varied p21 and p53 statuses, revealing distinct roles for p21 depending on the drug dosage. At high doses (HD), p21 is more critical than p53 in mediating apoptosis, whereas at low doses (LD), p21 is essential for preventing mitotic defects and multinucleation. Notably, reintroducing p21 or pharmacologically inhibiting CDK1 reduced multinucleation. The absence of p21 upon LD doxorubicin exposure led to aberrant chromosome segregation, persistent DNA damage response (DDR) activation, and increased non-homologous end-joining (NHEJ) activity, resulting in unrepaired DNA accumulation and multinucleation. Additionally, mitotic defects in p21-deficient cells were associated with mislocalization of key mitotic regulators, Aurora B and mitotic kinesin-like protein 1 (MKLP1), exacerbating defective mitosis. In summary, p21 functions as a dual regulator in response to DNA damage, promoting apoptosis at HD and preventing mitotic failure at LD. These insights have significant implications for cancer therapy, highlighting the potential of targeting the p21 to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62798e3e091c52a2538fbed6da27e2c926098463" target='_blank'>
              Dual role of p21 in regulating apoptosis and mitotic integrity in response to doxorubicin in colon cancer cells
              </a>
            </td>
          <td>
            Heeyeon Kim, Haein Kim, Eunjung Jang, Young-Woo Eom, Gyesoon Yoon, Kyeong Sook Choi, Eunhee Kim
          </td>
          <td>2025-04-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567b69c3e9b7e6f973b80fe210ce55ed16d03194" target='_blank'>
              Ancestral origin and structural characteristics of non-syntenic homologous chromosomes in abalones
              </a>
            </td>
          <td>
            S. Hirase, Takashi Makino, Takeshi Takeuchi, Mitsutaka Kadota, Shigehiro Kuraku, Kiyoshi Kikuchi
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks arising from base excision repair (BER) during the misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination–deficient (HRD) cells. At templated AP sites, cross-links between the DNA and 5-hydroxymethylcytosine binding, embryonic stem cell–specific (HMCES) regulate replication fork speed, avoiding cytotoxic double-strand breaks. While the role of HMCES at the template DNA strand is well studied, its effects on nascent DNA are not. We provide evidence that HMCES–DNA-protein cross-links (DPCs) are detrimental to the BER-mediated removal of 5-hydroxymethyl-2′-deoxycytidine (5hmdC)–derived 5-hydroxymethyl-2′-deoxyuridine from replication forks. HRD cells have heightened HMCES-DPCs, which increase further upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP sites. HMCES depletion substantially suppresses 5hmdC-mediated replication fork defects, chromosomal aberrations, and cell death in HRD cells. This reveals that HMCES-DPCs are a source of BER-initiated single-stranded DNA gaps, which indicates that endogenous DPCs contribute to genomic instability in HRD tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622156d932ecc5d7d331e4371a1b29a1f41c3ef" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination–deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martín, Marta del Rio Oliva, Yodhara Wijesekara Hanthi, Sara Berrada, Raimundo Freire, Jean-Yves Masson, José Carlos Reyes, Vincenzo Costanzo, Iván V. Rosado
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Transposable elements (TEs), which constitute nearly half of the human genome, have long been regarded as genomic "dark matter". However, their reactivation in tumor cells, resulting in the production of TE-chimeric transcripts (TCTs), has emerged as a potential driver of cancer progression. The complexity and full extent of these transcripts remain elusive, largely due to the limitations of short-read next-generation sequencing technologies. These methods have struggled to comprehensively capture the diversity and structure of TCTs, particularly those involving short interspersed nuclear elements (SINEs) or closely co-transcribed TEs. Methods Leveraging full-length cDNA sequencing technology based on nanopore sequencing platform, we developed a customized pipeline for identifying and quantifying TCTs in 19 lung adenocarcinoma (LUAD) cell lines. The short-read RNA-seq dataset from a LUAD corhort (~ 200 tumor samples) was employed to validate the identified TCTs and explore their association with tumor progression. To assess the functional roles of a specific TCTs, cell migration and cell proliferation assays were performed. Results We uncovered 208 unique TCT candidates in the LUAD cell lines. Our approach allowed for the identification of cryptic promoters and terminators within non-transposing TEs. Notably, we identified a chimeric transcript involving MIR_HKDC1, which appears to play a significant role in the progression of LUAD. Furthermore, the expression of these TCTs were associated with poor clinical outcomes in a cohort of LUAD patients, suggesting their potential as novel biomarkers for both LUAD progression and prognosis. Conclusions Our study underscores the application of long-read sequencing to unravel the complex landscape of TCTs in LUAD. We provide a comprehensive characterization of TCTs in LUAD, exploring their potential regulatory roles in cancer progression. These findings contribute to a deeper understanding of the genomic intricacies underlying cancer, and offer new directions for the development of targeted therapies and personalized treatment strategies for LUAD. This research highlights the potential of TCTs as both biomarkers and therapeutic targets in the oncogenesis, offering new insights into the interplay between transposable elements and gene regulation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13888-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers that arise from germline mutations of breast cancer associated gene 1 (BRCA1), which is a crucial player in homologous recombination (HR) DNA repair, are vulnerable to DNA-damaging agents such as platinum and PARP inhibitors (PARPis). Increasing evidence suggests that BRCA1 is an essential driver of all phases of the cell cycle, thereby maintaining orderly steps during cell cycle progression. Specifically, loss of BRCA1 activity causes the S-phase, G2/M, spindle checkpoints, and centrosome duplication to be dysregulated, thereby blocking cell proliferation and inducing apoptosis. In vertebrates, loss of HR genes such as BRCA1 and/or BRCA2 is lethal, since HR is a prerequisite for genome integrity. Thus, cancer cells utilize alternative DNA repair pathways such as non-homologous end joining (NHEJ) to cope with the loss of BRCA1 function. In this review, we attempt to update and discuss how these novel components are crucial for regulating DNA damage repair (DDR) in BRCA1-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca96851a56316ea96ea1fc73475ff478d721c03" target='_blank'>
              Overview of Roles of Novel Components in the Regulation of DNA Damage Repair in BRCA1-Deficient Cancers: An Update
              </a>
            </td>
          <td>
            Nhat Nguyen, Dominic Arris, M. T. Tran
          </td>
          <td>2025-04-01</td>
          <td>DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The reactivation of developmental silenced γ-globin genes (HBG1/2) has shown promise as a therapeutic strategy for improving symptoms of β-hemoglobinopathies. Currently, the focus of therapeutic targets is primarily on the major fetal hemoglobin suppressors, such as BCL11A and ZBTB7A and of their binding sites on the proximal HBG promoter. However, the role of the distal HBG promoter in regulating gene expression remains to be explored. Methods We used CRISPR/Cas9 system to edit the distal HBG promoter. In vitro and in vivo assays, as well as engrafted NCG-Kit-V831M mice, were used for functional validation and mechanistic studies. Results We discovered an insertion of nucleotide A (insA) between − 1368 and − 1369 bp upstream of the TSS in HBG2 resulting in remarkable increase in γ-globin expression in HUDEP-2 cells. We also observed elevated γ-globin expression in human CD34+ erythroid progenitor cells from healthy individuals and those with β-thalassemia when introducing insA mutation. Similarly, engrafted NCG-Kit-V831M mice showed increased γ-globin expression. Importantly, neither did insA have any off-target effects nor did it affect the maturation of erythroid cells. Furthermore, we found that the insA mutation created a binding site for the transcription activator FOXO3, which was activated by AMPK. Additionally, introducing insA specifically demethylated the − 162 CpG site on HBG promoter by reducing the enrichment of DNA methyltransferase 3 A (DNMT3A). At the same time, it activated histone modifications and RNA polymerase II (Pol II) in both distal and proximal HBG promoter and might inhibit the binding of BCL11A and ZBTB7A on -115 and − 200 sites on the HBG promoter respectively. In addition, combination of insA and the − 115 or -200 editing targets resulted in an amplify effect in reactivating γ-globin genes expression. Conclusions Overall, we presented the preclinical data to support the role of insA on regulating γ-globin expression using CD34+ HSPC cells derived from healthy donors or patients with β-thalassemia, and subsequently engrafted mice. Our study suggests that introducing insA mutation leads to significantly boosted fetal globin levels and uncovers new safe therapeutic target or strategy for β-hemoglobinopathies. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00626-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e602d03d7e5d834d0278abcf5359fc05d4a105ed" target='_blank'>
              A one-base therapeutic insertion in the HBG2 distal promoter reactivates γ-globin expression
              </a>
            </td>
          <td>
            Xiuqin Bao, Yuanyi Gao, Xiaoyi Chen, Zhongju Wang, Xiaoqin Feng, Liren Wang, Jing Du, Yuhua Ye, Feijing Chen, Li Du, Aihua Yin, Xiangmin Xu
          </td>
          <td>2025-03-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3389d87da3fd17d24b405d1383fde338ac5299e" target='_blank'>
              MYC in cancer: from undruggable target to clinical trials.
              </a>
            </td>
          <td>
            Jonathan R Whitfield, Laura Soucek
          </td>
          <td>2025-02-19</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1324a05392de89320b697d282b9ec5e482ee4e56" target='_blank'>
              γ-tubulin mediates DNA double-strand break repair.
              </a>
            </td>
          <td>
            Abhishikt David Solomon, Odjo G. Gouttia, Ling Wang, Songli Zhu, Feifei Wang, Yanqui Li, Mohammadjavad Paydar, T. Bessho, Benjamin H Kwok, A. Peng
          </td>
          <td>2025-03-15</td>
          <td>Journal of cell science</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c305dc46bcd16adb030553a33a9a8b7556cef2ef" target='_blank'>
              Trypanosoma cruzi Has 32 Chromosomes: A Telomere-to-Telomere Assembly Defines Its Karyotype
              </a>
            </td>
          <td>
            G. Greif, F. Díaz-Viraqué, M. L. Chiribao, Carlos E. Sanz-Rodriguez, C. Robello
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Perturbed DNA replication can lead to incompletely replicated DNA when cells enter mitosis and can interfere with chromosome segregation. Cells therefore require mechanisms to resolve these lesions during mitosis. The CIP2A-TOPBP1 complex is described to function as a molecular tether that connects fragmented DNA molecules. However, whether CIP2A also functions in processing of incompletely replicated DNA remained unclear. We show that CIP2A-TOPBP1 forms large filamentous structures at sites of incomplete DNA replication during mitosis, which recruit the SMX tri-nuclease complex members SLX4, MUS81 and ERCC1/XPF. These structures form in proximity to sites of mitotic DNA synthesis, although CIP2A is not required for mitotic DNA synthesis. In addition to its globular and coiled-coil domain, the unstructured C-terminal domain of CIP2A is essential for CIP2A-TOPBP1 filamentous structure formation and recruitment of the SMX complex. BRCA1-/- and BRCA2-/- cells have increased mitotic DNA lesions that recruit CIP2A and SLX4. We show that the C-terminal part of CIP2A is required for survival of BRCA2-/- cells. Moreover, SLX4 is crucial for genome stability in BRCA2-/- cells. Combined, we demonstrate that CIP2A-TOPBP1 recruits the SMX complex during mitosis, which is required to resolve mitotic DNA lesions, allows faithful chromosome segregation and maintain viability of BRCA2-/- cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b232ddba76a37f75a5a20189f87cbdb729879f8" target='_blank'>
              CIP2A is required for mitotic recruitment of the SLX1/XPF/MUS81 tri-nuclease complex to replication stress-induced DNA lesions to maintain genome integrity
              </a>
            </td>
          <td>
            Lauren de Haan, Sietse J Dijt, Alejandro García López, Dan Ruan, Femke J. Bakker, M. Everts, Harry M Warner, Frank N. Mol, H. R. de Boer, Pim J. Huis in ’t Veld, Bert van de Kooij, R. Vlijm, M. V. van Vugt
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV, HHV-8) is associated with several human malignancies. During latency, the viral genomes reside in the nucleus of infected cells as large non-integrated plasmids, known as episomes. To ensure episome maintenance, the latency protein LANA tethers the viral episomes to the cell chromosomes during cell division. Directional recruitment of protein complexes is critical for the proper function of many nuclear processes. To test for recruitment directionality between LANA and cellular proteins, we directed LANA via catalytically inactive Cas9 (dCas9) to a repeat sequence to obtain easily detectable dots. Then, the recruitment of nuclear proteins to these dots can be evaluated. We termed this assay CRISPR-PITA for Protein Interaction and Telomere Recruitment Assay. Using this protein recruitment assay, we found that LANA recruits its known interactors ORC2 and SIN3A. Interestingly, LANA was unable to recruit MeCP2, but MeCP2 recruited LANA. Both LANA and histone deacetylase 1 (HDAC1) interact with the transcriptional-repression domain (TRD) and the methyl-CpG-binding domain (MBD) of MeCP2. Similar to LANA, HDAC1 was unable to recruit MeCP2. While heterochromatin protein 1 (HP1), which interacts with the N-terminal of MeCP2, can recruit MeCP2. We propose that available interacting domains force this recruitment directionality. We hypothesized that the tandem repeats in the SunTag may force MeCP2 dimerization and mimic the form of DNA-bound MeCP2. Indeed, providing only the tandem epitopes of SunTag allows LANA to recruit MeCP2 in infected cells. Therefore, CRISPR-PITA revealed the rules of unidirectional recruitment and allowed us to break this directionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b4f351e09d3750518b9d66bb9ef783bc4bf147" target='_blank'>
              Unidirectional recruitment between MeCP2 and KSHV-encoded LANA revealed by CRISPR/Cas9 recruitment assay
              </a>
            </td>
          <td>
            Ido Lavi, Supriya Bhattacharya, Ankita Awase, Ola Orgil, Nir Avital, Guy Journo, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-03-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Intratumor transcriptional heterogeneity (ITTH) presents a major challenge in cancer treatment, particularly due to limited understanding of the diverse malignant cell populations and their relationship to therapy resistance. While single-cell sequencing has provided valuable insights into tumor composition, its high cost and technical complexity limits its use for large-scale tumor screening. In contrast, several databases collecting bulk RNA sequencing (RNA-seq) data from multiple samples across various cancer types are available and could be used to profile ITTH. Several deconvolution approaches have been developed to infer cellular composition from such data. However, most of these methods rely on predefined markers or single cell reference datasets, limiting the performance of such methods by the quality of used reference data. Although unsupervised approaches do not face such limitations, existing methods have not been specifically adapted to characterize malignant cell states, and focus instead on general cell types. To address these gaps, we introduce CDState, an unsupervised method for inferring malignant cell subpopulations from bulk RNA-seq data. CDState utilizes a Nonnegative Matrix Factorization model improved with sum-to-one constraints and a cosine similarity-based optimization to deconvolve bulk gene expression into distinct cell state-specific profiles, and estimate the abundance of each state across tumor samples. We validate CDState using bulkified single-cell RNA-seq data from five cancer types, showing that it outperforms existing unsupervised deconvolution methods in both cell state proportions and gene expression estimation. Applying CDState to 33 cancer types from TCGA, we identified recurrent malignant cell programs, with epithelial-mesenchymal transition as the main driver of tumor transcriptional heterogeneity. We further link the identified malignant states to patient clinical features, revealing states associated with worse patient prognosis. We show that ITTH is linked with patient survival and clinical features, as well as is associated with varied responses to therapy. Finally, we identified potential genetic drivers of ITTH and malignant cell states, including TP53, KRAS, and PIK3CA, known oncogenic genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16cea30bc9f4619225da6166ac6508ace6b994e" target='_blank'>
              CDState: an unsupervised approach to predict malignant cell heterogeneity in tumor bulk RNA-sequencing data
              </a>
            </td>
          <td>
            Agnieszka Kraft, Josephine Yates, Florian Barkmann, Valentina Boeva
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Structural variants (SVs) are critical for plant genomic diversity and phenotypic variation. This study investigates a large, 9.7 Mbp highly repetitive segment on chromosome 12 of TMEB117, a region not previously characterized in cassava (Manihot esculenta Crantz). We aim to explore its presence and variability across multiple cassava landraces, providing insights into its genomic significance and potential implications. Results We validated the presence of the 9.7 Mbp segment in the TMEB117 genome, distinguishing it from other published cassava genome assemblies. By mapping short-read sequencing data from 16 cassava landraces to TMEB117 chromosome 12, we observed variability in read mapping, suggesting that while all genotypes contain the insertion region, some exhibit missing segments or sequence differences. Further analysis revealed two unique genes associated with deacetylase activity, HDA14 and SRT2, within the insertion. Additionally, the MUDR-Mutator transposable element was significantly overrepresented in this region. Conclusions This study uncovers a large structural variant in the TMEB117 cassava genome, highlighting its variability among different genotypes. The enrichment of HDA14 and SRT2 genes and the MUDR-Mutator elements within the insertion suggests potential functional significance, though further research is needed to explore this. These findings provide important insights into the role of structural variations in shaping cassava genomic diversity. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11523-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81f6439804e83d40ef072194b5300e584bbf3322" target='_blank'>
              Genome-wide comparison reveals large structural variants in cassava landraces
              </a>
            </td>
          <td>
            Michael Landi, A. V. Carluccio, Trushar Shah, Adnan Niazi, Livia Stavolone, Laurent Falquet, A. Gisel, Erik Bongcam-Rudloff
          </td>
          <td>2025-04-10</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica D. Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y W T Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/230adcbb775deb02eb0452f0dd751e2a887a0270" target='_blank'>
              KRAB zinc-finger proteins regulate endogenous retroviruses to sculpt germline transcriptomes and genome evolution.
              </a>
            </td>
          <td>
            Kai Otsuka, A. Sakashita, So Maezawa, Richard M Schultz, S. Namekawa
          </td>
          <td>2025-03-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Aberrant alternative splicing, prevalent in cancer, impacts various cancer hallmarks involving proliferation, angiogenesis, and invasion. Splicing disruption often results from somatic point mutations rewiring functional pathways to support cancer cell survival. We introduce iSoMAs (iSoform expression and somatic Mutation Association), an efficient computational pipeline leveraging principal component analysis technique, to explore how somatic mutations influence transcriptome-wide gene expression at the isoform level. Applying iSoMAs to 33 cancer types comprising 9,738 tumor samples in The Cancer Genome Atlas, we identified 908 somatically mutated genes significantly associated with altered isoform expression across three or more cancer types. Mutations linked to differential isoform expression occurred through both cis- and trans-acting mechanisms, involving well-known oncogenes/suppressor genes, RNA binding protein and splicing factor genes. With wet-lab experiments, we verified direct association between TP53 mutations and differential isoform expression in cell cycle genes. Additional iSoMAs genes have been validated in the literature with independent cohorts and/or methods. Despite the complexity of cancer, iSoMAs attains computational efficiency via dimension reduction strategy and reveals critical associations between regulatory factors and transcriptional landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1e96c7e45926632d80fe6cd693fccd8a255dd7" target='_blank'>
              iSoMAs: Finding isoform expression and somatic mutation associations in human cancers.
              </a>
            </td>
          <td>
            Hua Tan, Valer Gotea, S. Jaiswal, Nancy E Seidel, David O. Holland, Kevin Fedkenheuer, Abdel Elkahloun, Sara R. Bang-Christensen, L. Elnitski
          </td>
          <td>2025-03-07</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2b2e41c178b6dbd66cbfc9ba10d448bf6866f2" target='_blank'>
              Spatial multi-omics reveals cell-type-specific nuclear compartments.
              </a>
            </td>
          <td>
            Yodai Takei, Yujing Yang, Jonathan White, Isabel N. Goronzy, Jina Yun, Meera Prasad, Lincoln J Ombelets, Simon Schindler, Prashant Bhat, Mitchell Guttman, L. Cai
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbddfb38eb3b46fe2a2daba20fe0e51c20e3f4a" target='_blank'>
              βTrCP is involved in the localization of the MRN complex on chromatin to enhance DNA damage repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a highly heterogeneous malignancy, often characterized by complex genomic alterations that drive tumor progression and therapy resistance. In this paper, we report a novel de novo BRCA2 germline variant NM_000059.3:c.(8693_8695delinsGT) associated with early-onset OC that featured two regions with differential MMR (Mismatch Repair) gene expression. To date, only six cases of de novo BRCA2 variants have been reported, none of which were associated with early-onset high-grade serous OC. The immunohistochemical analysis of MMR genes revealed two distinct tumor areas, separated by a clear topographic boundary, with the heterogeneous expression of MLH1 and PMS2 proteins. Seventy-five percent of the tumor tissue showed positivity, while the remaining 25% exhibited a complete absence of expression, underscoring the spatial variability in MMR gene expression within the tumor. Integrated comparative spatial genomic profiling identified several tumor features associated with the genetic variant as regions of loss of heterozygosity (LOH) that involved BRCA2 and MLH1 genes, along with a significantly higher mutational tumor burden in the tumor area that lacked MLH1 and PMS2 expression, indicating its further molecular evolution. The following variants were acquired: c.6572C>T in NOTCH2, c.1852C>T in BCL6, c.191A>T in INHBA, c.749C>T in CUX1, c.898C>A in FANCG, and c.1712G>C in KDM6A. Integrated comparative spatial proteomic profiles revealed defects in the DNA repair pathways, as well as significant alterations in the extracellular matrix (ECM). The differential expression of proteins involved in DNA repair, particularly those associated with MMR and Base Excision Repair (BER), highlights the critical role of defective repair mechanisms in driving genomic instability. Furthermore, ECM components, such as collagen isoforms, Fibrillin-1, EMILIN-1, Prolargin, and Lumican, were found to be highly expressed in the MLH1/PMS2-deficient tumor area, suggesting a connection between DNA repair deficiencies, ECM remodeling, and tumor progression. Thus, the identification of the BRCA2 variant sheds light on the poorly understood interplay between DNA repair deficiencies and ECM remodeling in OC, providing new insights into their dual role in shaping tumor evolution and suggesting potential targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3d22f654834c47296af7f38cdf9042ad40753" target='_blank'>
              Novel De Novo BRCA2 Variant in an Early-Onset Ovarian Cancer Reveals a Unique Tumor Evolution Pathway
              </a>
            </td>
          <td>
            G. Miolo, Giovanni Canil, Maurizio Polano, M. Dal Bo, Alessia Mondello, Antonio Palumbo, F. Puglisi, Giuseppe Corona
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="To counteract the damaging effects of DNA interstrand crosslinks (ICLs), cells have evolved various specialized ICL repair pathways. However, how ICL repair impacts genetic integrity remains incompletely understood. Here, we determined the mutagenic consequences of psoralen ICL repair in the animal model C. elegans and identify two mutagenic repair mechanisms: (i) translesion synthesis through POLH and REV1/3-mediated bypass, leading to single nucleotide polymorphisms (SNVs), and (ii) end joining via POLQ or HELQ action resulting in deletions. While we found no role for the Fanconi anemia genes FANCD2 and FANCI, disruption of TRAIP, which triggers unloading of the CMG helicase at sites of blocked replication, led to a strikingly altered repair profile, suggesting a role for DNA replication in the etiology of ICL-induced deletions. TRAIP deficiency did not affect SNV formation; instead, we found these SNVs to depend on the functionality of the Fanconi anemia-associated nuclease FAN1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c8e44dc0dd119d75905686eba6c875d7b3f783" target='_blank'>
              FAN1-mediated translesion synthesis and POLQ/HELQ-mediated end joining generate interstrand crosslink-induced mutations
              </a>
            </td>
          <td>
            Jip Verschuren, R. van Schendel, Ivo van Bostelen, Alex E E Verkennis, P. Knipscheer, M. Tijsterman
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Variants of uncertain significance (VUS) limit the actionability of genetic testing. A prominent example is MUTYH, a base excision repair factor associated with polyposis and colorectal cancer, which has a pathogenic variant carrier rate approaching 1 in 50 individuals in some populations. To systematically interrogate variant function in MUTYH, we coupled deep mutational scanning with a DNA repair reporter containing its lesion substrate, 8OG:A. Our variant-to-function map covers >97% of all possible MUTYH point variants (n=10,941) and achieves 100% accuracy classifying the pathogenicity of known clinical variants (n=247). Leveraging a large clinical registry, we observe significant associations with colorectal polyps and cancer, with more severely impaired missense variants conferring greater risk. We recapitulate known functional differences between pathogenic founder alleles, and highlight sites of complete missense intolerance, including residues that intercalate DNA and coordinate essential Zn2+ or Fe-S clusters. This map provides a resource to resolve the 1,032 existing missense VUS and 90 variants with conflicting interpretations in MUTYH, and demonstrates a scalable strategy to interrogate other clinically relevant DNA repair factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ddb79d358a7742980d1fc41247ef4a3c39e8c" target='_blank'>
              Saturation mapping of MUTYH variant effects using DNA repair reporters
              </a>
            </td>
          <td>
            Shelby L. Hemker, Ashley Marsh, Felicia Hernandez, Elena Glick, Grace Clark, Alyssa Bashir, Krystal Jiang, Jacob O. Kitzman
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a86ef1aaa24e6b5fb3e115c100f92a87dac07208" target='_blank'>
              EPIraction - an atlas of candidate enhancer-gene interactions in human tissues and cell lines
              </a>
            </td>
          <td>
            Ramil N. Nurtdinov, R. Guigó
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8904feaf495af004b999f1283404430b8205d7d8" target='_blank'>
              p53-deficient cancer cells hyperactivate DNA double-strand break repair pathways to overcome chemotherapeutic damage and augment survival.
              </a>
            </td>
          <td>
            Rebecca Dsouza, Meghna Jain, Ekta Khattar
          </td>
          <td>2025-03-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0b8e5181c88e09c80086ca81b1bb518a1efc54" target='_blank'>
              PEAS: Detection of Clustered Differences in Genomic Data
              </a>
            </td>
          <td>
            Dylan Skola, Christopher K. Glass
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differential alternative splicing transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes including vFLIP, vCyclin and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells, and significantly inhibited KSHV-driven proliferation, cellular transformation and tumorigenesis. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. Importance Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1dd5dbd2f3302db90bfa70c7f9ae17b8e6ba3f" target='_blank'>
              KSHV Reprograms Host RNA Splicing via FAM50A to Activate STAT3 and Drive Oncogenic Cellular Transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Karla Paniagua, Ling Ding, Xian Wang, Yufei Huang, Mario A Flores, Shou‐Jiang Gao
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA-dependent protein kinase catalytic subunit (DNA-PKcs) is a key kinase in the DNA repair process that responds to DNA damage caused by various factors and maintains genomic stability. However, DNA-PKcs is overexpressed in some solid tumors and is frequently associated with poor prognosis. DNA-PKcs was initially identified as a part of the transcription complex. In recent years, many studies have focused on its nonclassical functions, including transcriptional regulation, metabolism, innate immunity, and inflammatory response. Given the pleiotropic roles of DNA-PKcs in tumors, pharmacological inhibition of DNA-PK can exert antitumor effects and may serve as a potential target for tumor therapy in the future. This review summarizes several aspects of DNA-PKcs regulation of RNA metabolism, including its impact on transcriptional machinery, alternative splicing, and interaction with noncoding RNAs, and provides insights into DNA-PKcs beyond its DNA damage repair function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f296dc7b10466d316234339cf9617d0c3278a601" target='_blank'>
              DNA-PKcs, a player winding and dancing with RNA metabolism and diseases
              </a>
            </td>
          <td>
            Jiabao Hou, Mingjun Lu, Jingwei Guo, Jinghong Wu, Chenyang Wang, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2025-03-04</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac21d254e59fc1b3c318f6ce0feabf0e67df48b5" target='_blank'>
              Complete sequencing of ape genomes.
              </a>
            </td>
          <td>
            DongAhn Yoo, A. Rhie, Prajna Hebbar, F. Antonacci, Glennis A. Logsdon, Steven J. Solar, D. Antipov, Brandon D. Pickett, Yana Safonova, Francesco Montinaro, Yanting Luo, Joanna Malukiewicz, Jessica M. Storer, Jiadong Lin, Abigail N Sequeira, Riley J. Mangan, G. Hickey, Graciela Monfort Anez, Parithi Balachandran, Anton Bankevich, Christine R. Beck, Arjun Biddanda, M. Borchers, G. Bouffard, Emry Brannan, Shelise Y. Brooks, Lucia Carbone, Laura Carrel, Agnes P Chan, Juyun Crawford, M. Diekhans, Eric Engelbrecht, Cédric Feschotte, G. Formenti, Gage H Garcia, Luciana de Gennaro, David M. Gilbert, Richard E. Green, A. Guarracino, Ishaan Gupta, Diana Haddad, Junmin Han, Robert S. Harris, Gabrielle Hartley, William T. Harvey, Michael Hiller, Kendra Hoekzema, M. Houck, Hyeonsoo Jeong, Kaivan Kamali, M. Kellis, Bryce Kille, Chul Lee, Youngho Lee, William Lees, Alexandra P. Lewis, Qiuhui Li, Mark Loftus, Yong-Hwee E Loh, Hailey Loucks, Jian Ma, Yafei Mao, J. F. I. Martinez, Patrick Masterson, R. McCoy, Barbara McGrath, S. McKinney, Britta S. Meyer, K. Miga, S. K. Mohanty, Katherine M. Munson, Karol Pál, Matt Pennell, Pavel A. Pevzner, David Porubsky, T. Potapova, F. R. Ringeling, Joana L. Rocha, Oliver A. Ryder, Samuel Sacco, Swati Saha, Takayo Sasaki, Michael C. Schatz, NJ Schork, Cole Shanks, Linnéa Smeds, Dongmin R Son, Cynthia Steiner, Alex Sweeten, Michael G. Tassia, F. Thibaud-Nissen, Edmundo Torres-González, Mihir Trivedi, Wenjie Wei, Julie Wertz, Muyu Yang, Panpan Zhang, Shilong Zhang, Yang Zhang, Zhenmiao Zhang, Sarah A. Zhao, Yixin Zhu, Erich D. Jarvis, J. Gerton, Iker Rivas-González, B. Paten, Zachary A. Szpiech, C. Huber, Tobias L Lenz, Miriam K. Konkel, Soojin V Yi, Stefan Canzar, Corey T. Watson, Peter H. Sudmant, Erin K. Molloy, Erik Garrison, Craig B. Lowe, Mario Ventura, Rachel J. O’Neill, S. Koren, Kateryna D. Makova, A. Phillippy, E. Eichler
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>2</td>
          <td>78</td>
        </tr>

        <tr id="Abstract During meiosis, programmed DNA double-strand breaks (DSBs) are formed at hotspots to initiate homologous recombination, which is vital for reassorting genetic material. In fission yeast, the linear element (LinE) proteins Mug20, Rec25, and Rec27 interdependently bind chromosomal hotspots with high specificity and are necessary for high-level DSB formation. However, their mechanistic role in regulating the meiotic DSB machinery remains unknown. Here, using purified Mug20–Rec25–Rec27 (MRR) complex and functional intracellular analyses, we reveal that the MRR–DNA nucleoprotein complex assembles phase-separated condensates that compact the DNA. Notably, MRR complex formation is a prerequisite for DNA binding and condensate assembly, with Rec27 playing a pivotal role in directly binding DNA. Consistent with this finding, failure to form MRR–DNA condensates results in defective intracellular meiotic DSB formation and recombination. Our results provide mechanistic insights into how LinEs enhance meiotic DSB formation and provide a paradigm for studies in other species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2043e56114bc07be8ef392e7f7e4ffa5372e0c1" target='_blank'>
              Mug20–Rec25–Rec27 binds DNA and enhances meiotic DNA break formation via phase-separated condensates
              </a>
            </td>
          <td>
            Max F Wang, Meng-Yun Li, Ya-Ching Yang, Yu-Chien Chuang, Chieh-Yu Tsai, Mai-Chi Nguyen Binder, Lijuan Ma, Sheng-Wei Lin, Hung-Wen Li, Gerald R Smith, Peter Chi
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Garra rufa, commonly known as doctor fish, is a small freshwater cyprinid fish recognized for its high-temperature tolerance. G. rufa is primarily known for its role in ichthyotherapy, in which it helps remove keratinized skin from human hands and feet. However, in recent years, its high-temperature tolerance has attracted attention owing to its potential use as a model for human disease research, including infectious diseases and cancer xenograft models. Despite this interest, the genomic basis underlying high-temperature tolerance remains largely unexplored, primarily because of the limited availability of genomic resources, which hinders its development as an experimental model. In the present study, a high-quality chromosome-level genome assembly of G. rufa was generated using a combination of PacBio HiFi long-read sequencing and Hi-C technology. We generated a chromosome-level genome assembly with 25 chromosomes, 1.38 Gb in total length, and scaffold N50 of 49.3 Mb. Approximately 59% of the genome comprises repetitive elements, with DNA and LTR elements being particularly abundant. In total, 27,352 protein-coding genes were annotated, of which 26,900 genes (98.3%) were functionally annotated. Benchmarking Universal Single-Copy Orthologs (BUSCO) benchmark for genome assembly and gene annotation demonstrated 94.5% and 94.7% of complete BUSCOs, respectively. We identified two heat shock transcription factor (HSF) and 239 heat shock protein (HSP)-related genes. Heat shock elements, which are HSF-binding motifs, were present within 3 kb upstream of 944 genes, with statistically significant enrichment of HSP-related genes in this set. Furthermore, molecular phylogenetic analysis and whole-genome comparisons revealed that G. rufa is evolutionarily closely related to species in the Labeoninae subfamily of the Cyprinidae family. This chromosome-level reference genome provides a valuable resource for future research aimed at elucidating the molecular mechanisms underlying high-temperature tolerance in G. rufa and establishing it as a model organism for human biomedical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba084e3b18995a64de9db1867a75cd406732e5f" target='_blank'>
              Chromosome-level genome assembly of the doctor fish (Garra rufa)
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Kiki Syaputri Handayani, Liqing Zang, Fahrurrozi, Oleg Simakov, Victor David, Nico Gultom, Yasuhito Shimada
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Ribosome biogenesis is one of the most fundamental and energetically demanding cellular processes. In humans, the ribosomal DNA (rDNA) repeats span a large region of DNA and comprise 200 to 600 copies of a ~43 kb unit spread over five different chromosomes. Control over ribosome biogenesis is closely tied to the regulation of the chromatin environment of this large genomic region. The proportion of rDNA loci which are active or silent is altered depending on the proliferative or metabolic state of the cell. Repeat silencing is driven by epigenetic changes culminating in a repressive heterochromatin environment. One group of proteins facilitating these epigenetic changes in response to growth or metabolic demands are ATP-dependent chromatin remodeling protein complexes that use ATP hydrolysis to reposition nucleosomes. Indeed, some chromatin remodelers are known to have indispensable roles in regulating the chromatin environment of rDNA. In this review, we highlight these proteins and their complexes and describe their mechanistic roles at rDNA. We also introduce the developmental disorders arising from the dysfunction of these proteins and discuss how the consequent dysregulation of rDNA loci may be reflected in the phenotypes observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b44d8b12e96514a790a07032e2485fc056c2a29" target='_blank'>
              It Takes a Village of Chromatin Remodelers to Regulate rDNA Expression
              </a>
            </td>
          <td>
            Mathieu G. Levesque, D. Picketts
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8774da0a06c116c814c2652572a737b8d86a0c02" target='_blank'>
              Genome-wide ribonucleotide detection in Archaea
              </a>
            </td>
          <td>
            Y. Moalic, Maurane Reveil, Deepali L. Kundnani, S. Balachander, Taehwan Yang, Alli L. Gombolay, Farahnaz Ranjbarian, Raphaël Brizard, Patrick Durand, H. Myllykallio, Mohamed Jebbar, Anders Hofer, F. Storici, Ghislaine Henneke
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2140165b325769cf3794c12431eadebdd635d22" target='_blank'>
              Chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Nina Trautwein, Bodo Laube, Christian Dietz, Robert W. Stark, M. C. Cardoso
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a03b89d9e1c30e4ab6bb1a8b31d9bee97c5524d" target='_blank'>
              Replisomes restrict SMC-mediated DNA-loop extrusion in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Lung cancer is a serious threat to human life and health, but effective screening and treatment methods are lacking. Circular RNAs (circRNAs) have important biological functions and are closely related to tumour development. Some studies have shown that the 8-oxo-7,8-dihydroguanosine (o8G) modification plays a key role in the disease process, but the effect of the o8G modification on circRNAs has not been elucidated. Moreover, cuproptosis is a novel mode of cell death in which copper ions directly promote protein aggregation and the disruption of cellular metabolic pathways. The present study revealed that the o8G modification of circKIAA1797 occurs and promotes lung cancer development by inhibiting cuproptosis, which provides new perspectives for epitranscriptomic studies and the development of novel therapeutic approaches for lung cancer. Methods circRNA differential expression profiles in lung cancer were revealed via RNA high-throughput sequencing, and circKIAA1797 expression in lung cancer cell lines and tissues was detected using qPCR. Experiments such as o8G RNA immunoprecipitation (o8G RIP) and crosslinking immunoprecipitation (CLIP) were performed to explore the presence of o8G on circKIAA1797. The regulation of circKIAA1797 by the o8G reader Y-box binding protein 1 (YBX1) was explored using nuclear–cytoplasmic fractionation, actinomycin D (Act D) stability experiments and other experiments. circKIAA1797 silencing and overexpression systems were constructed for in vivo and in vitro experiments to study the role of circKIAA1797 in lung cancer development. Tagged RNA affinity purification (TRAP), RNA immunoprecipitation (RIP), coimmunoprecipitation (Co-IP), and immunofluorescence (IF) staining were subsequently conducted to reveal the molecular mechanism by which circKIAA1797 regulates cuproptosis and promotes lung cancer development. Results This study is the first to reveal the presence of o8G on circKIAA1797 and that YBX1 is a reader that recognises ROS-induced circKIAA1797 o8G modifications and increases the stability and cytoplasmic expression of circKIAA1797. circKIAA1797, which is associated with the tumour stage and prognosis, has been shown to significantly promote the biological function of lung cancer development both in vivo and in vitro. This study revealed that circKIAA1797 inhibits intracellular cuproptosis by binding to the ferredoxin 1 (FDX1) mRNA, decreasing FDX1 mRNA stability, inhibiting FDX1 expression, and binding to the signal transducer and activator of transcription 1 (STAT1) protein and inhibiting lipoyltransferase 1 (LIPT1) transcription; moreover, circKIAA1797 promotes the closure of the mitochondrial permeability transition pore (mPTP), inhibits cuproptosis, and ultimately promotes lung cancer development. Conclusions This study revealed the presence of the o8G modification in circKIAA1797, which plays an important role in the development of lung cancer. circKIAA1797 can inhibit cuproptosis by inhibiting key cuproptosis proteins and promoting mPTP closure, ultimately promoting the development of lung cancer. This study provides not only a new theoretical basis for an in-depth understanding of the molecular mechanisms of lung cancer development but also a potential target for lung cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03365-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2b06e7d50618fe13a63997ac85d779bfa1573" target='_blank'>
              o8G-modified circKIAA1797 promotes lung cancer development by inhibiting cuproptosis
              </a>
            </td>
          <td>
            Haotian Xu, Qingyun Zhao, Dunyu Cai, Xingcai Chen, Xiaodong Zhou, Yihong Gao, Jiaxi Wu, Shengyi Yuan, Deqing Li, RuiRui Zhang, Wenyi Peng, Gang Li, Aruo Nan
          </td>
          <td>2025-04-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a494e730aa784cad617baf97d8efcc5bc5c5345" target='_blank'>
              Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In the tumor context, platelets play a significant role in primary tumor progression, dissemination and metastasis. Analysis of this interaction in various cancers, such as non-small cell lung cancer (NSCLC), demonstrate that platelets can both transfer and receive biomolecules (e.g. RNA and proteins) to and from the tumor at different stages, becoming tumor-educated platelets. To investigate how platelets are able to transfer oncogenic material, we developed in vitro platelet-like particles (PLPs), from a differentiated MEG-01 cell line, that stably carry RNA and protein of the KRASG12D oncogene in fusion with GFP. We co-cultured these PLPs with NSCLC H1975 tumor cells to assess their ability to transfer this material. We observed that the generated platelets were capable of stably expressing the oncogene and transferring both its RNA and protein forms to tumor cells using qPCR and imaging techniques. Additionally, we found that coculturing PLPs loaded with GFP-KRASG12D with tumor cells increased their proliferative capacity at specific PLP concentrations. In conclusion, our study successfully engineered an MEG-01 cell line to produce PLPs carrying oncogenic GFP-KRASG12D simulating the tumor microenvironment, demonstrating the efficient transfer of this oncogene to tumor cells and its significant impact on enhancing proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b51d4673aaea8a53878437c3a918ca1f1fbfbf" target='_blank'>
              Oncogenic KRASG12D Transfer from Platelet-like Particles Enhances Proliferation and Survival in Non-Small Cell Lung Cancer Cells
              </a>
            </td>
          <td>
            J. Ceron-Hernandez, Gonzalo Martínez-Navajas, Jose Manuel Sanchez-Manas, María Pilar Molina, Jiajun Xie, Inés Aznar-Peralta, Abel Garcia-Diaz, Sonia Perales, Carolina Torres, M. J. Serrano, Pedro J. Real
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8859cb309a43420dfdd0f2da144b37f5d23d1c73" target='_blank'>
              Epigenetic Modulation, Intra-tumoral Microbiome and Immunity in Early Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Ning Jin, R. Hoyd, Ayse Selen Yilmaz, Jian Zhu, YunZhou Liu, M. J. Singh, Dennis Grencewicz, Xiaokui Mo, Matthew Kalady, Daniel Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos HF Chan, Michelle L Churchman, N. Denko, Frances Di Clemente, Rebecca D Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, A. Ikeguchi, Anjun Ma, Qin Ma, Martin D McCarter, A. Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, Aik Choon Tan, Daniel J. Spakowicz
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Natural plasmids are common in prokaryotes, but few have been documented in eukaryotes. The natural 2µ plasmid present in the yeast Saccharomyces cerevisiae is one of these best-characterized exceptions. This highly stable genetic element has coexisted with its host for millions of years, faithfully segregating at each cell division through a mechanism that remains unclear. Using proximity ligation methods (such as Hi-C, Micro-C) to map the contacts between 2µ plasmid and yeast chromosomes under dozens of different biological conditions, we found that the plasmid is tethered preferentially to regions with low transcriptional activity, often corresponding to long, inactive genes. These contacts do not depend on common chromosome-structuring factors, such as members of the structural maintenance of chromosome complexes (SMC) but depend on a nucleosome-encoded signal associated with RNA Pol II depletion. They appear stable throughout the cell cycle and can be established within minutes. This chromosome hitchhiking strategy may extend beyond the 2µ plasmid/S. cerevisiae pair, as suggested by the binding pattern of the natural eukaryotic plasmid Ddp5 along silent chromosome regions of the amoeba Dictyostelium discoideum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60d701689f8f3be285cb82707065b4218a7a99" target='_blank'>
              Parasitic plasmids are anchored to inactive regions of eukaryotic chromosomes through a nucleosome signal
              </a>
            </td>
          <td>
            Fabien Girard, Antoine Even, Agnès Thierry, Myriam Ruault, Léa Meneu, Pauline Larrous, Mickael Garnier, Sandrine Adiba, A. Taddei, Romain Koszul, A. Cournac
          </td>
          <td>2025-02-27</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e847c35fd0feaa6983cfa58c60c7da74934800f2" target='_blank'>
              Moss BRCA2 lacking canonical DNA binding domain promotes efficient homologous recombination and binds to DNA
              </a>
            </td>
          <td>
            Alice Chanteau, Suliane Quilleré, Arthur Crouset, Sreejith Allipra, Ulysse Tuquoi, P. Perroud, Simona Miron, P. Dupaigne, Sophie Zinn-Justin, Fabien Nogué, Rajeev Kumar
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Eukaryotic somatic cells undergo continuous telomere shortening because of end-replication problems. Approximately 10%~15% of human cancers rely on alternative lengthening of telomeres (ALT) to overcome telomere shortening. ALT cells are characterized by persistent telomere DNA replication stress and rely on recombination-based DNA repair pathways for telomere elongation. The Bloom syndrome (BLM) helicase is a member of the RecQ family, which has been implicated as a key regulator of the ALT mechanism as it is required for either telomere length maintenance or telomere clustering in ALT-associated promyelocytic leukemia bodies (APBs). Here, we summarize recent evidence detailing the role of BLM in the activation and maintenance of ALT. We propose that the role of BLM-dependent recombination and its interacting proteins remains a crucial question for future research in dissecting the molecular mechanisms of ALT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecfa6b604141f203a1f4a0229bf2941af19cc8c" target='_blank'>
              Multiple functions of the ALT favorite helicase, BLM
              </a>
            </td>
          <td>
            Shun Chang, Jiang Tan, Ren Bao, Yanduo Zhang, Jinkai Tong, Tongxin Jia, Jing Liu, Juhua Dan, Shuting Jia
          </td>
          <td>2025-03-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76844769f269400baf440ee5c0922f5fb7d1d20e" target='_blank'>
              H3K27M Mutation Reshapes the Antigenic Landscape in Diffuse Midline Glioma
              </a>
            </td>
          <td>
            Tima Shamekhi, Bijun Zeng, Claire Xin Sun, Paul Daniel, Terry C.C Lim Kam Sian, Gabriel Goncalves, Grace Huang, Farnaz Fahimi, Nivedhitha Selvakumar, Erwin Tanuwidjaya, Isaac Woodhouse, Bettina Kritzer, Ralf B. Schittenhelm, Roberta Mazzieri, Jason E. Cain, Javad Nazarin, Jordan R Hansford, Ron Firestein, Riccardo Dolcetti, Pouya Faridi
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract A Bhas42 cell transformation assay is a method used to detect the tumour-promoting activities of chemicals. However, the mechanisms underlying tumour transformations mediated by non-genotoxic carcinogens (NGCs) are poorly understood. This study aimed to examine the correlation between 12-O-tetradecanoylphorbol 13-acetate (TPA) or mezerein and the initiation of tumourous transformations by epigenetic regulation in Bhas42 cells. We found that TPA and mezerein prompted tumourous transformations by stimulating cell proliferation and migration in Bhas42 cells. Furthermore, we observed alterations in the expression levels of 134 genes, with 87 genes being upregulated and 47 genes being downregulated, following exposure to either TPA or mezerein. Among the differentially regulated genes, we identified 17 upregulated genes and 8 downregulated genes corresponding to differentially expressed genes in TNM [primary tumour (T), regional nodes (N), and metastasis (M)]. Importantly, we found that TPA and mezerein triggered the expression of Hmga2 and Ezh2 by loss of miRNA let-7 (miR let-7) in Bhas42 cells. Finally, the microRNA (miRNA) mimic of let-7 prevented the TPA- and mezerein-induced activation of Hmga2 and Ezh2 in Bhas42 cells. Our findings reveal a connection between tumourous transformations and the epigenetic regulator miR let-7 in NGCs, such as TPA and mezerein in Bhas42 cells. This highlights miR let-7 as a promising therapeutic target for mitigating tumourous transformations induced by NGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a471b7aa3bb5ea9d6651fe6b077bdee22a1d6f" target='_blank'>
              Non-genotoxic carcinogens (TPA and mezerein) activate tumourous transformation through miR let-7-mediated Hmga2 expression in Bhas42 cells
              </a>
            </td>
          <td>
            Moon Yi Ko, Euijun Min, Minjeong Kim, Heejin Park, Sumi Jang, Younhee Kim, Byoung-Seok Lee, Sung-Ae Hyun, Minhan Ka
          </td>
          <td>2025-03-03</td>
          <td>Environmental Epigenetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, Kun Liang, Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c650c9b7245da58253a96046c65fd7d628305a" target='_blank'>
              Understanding DNA and PARP in Cancer: Tackling Inhibitor Resistance
              </a>
            </td>
          <td>
            Hardha Balachandran, Gowramma Byran, V. Karri, Senthil Kumar Murugesan, K. Rajagopal
          </td>
          <td>2025-03-13</td>
          <td>Drug Metabolism and Bioanalysis Letters</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Our research aims to test the suitability of ddPCR for the detection of tumor-related genomic imbalances (CNVs) in ovarian cancer (OC) in the peripheral blood system. In principle, in an artificial CNV simulation, we can show that CNVs can be detected even in larger dilutions in diploid DNA using ddPCR. Furthermore, we identified ovarian cancer CNVs by using aCGH and used this information to produce suitable ddPCR tests. These ddPCR tests are based on the principle of two differently colored comparative DNA loci that mark both very common genome imbalances in OC and regions that hardly indicate genome imbalances in OC. We were able to generate six ddPCR tests that indicate OC by identifying cell-free tumor DNA with a sensitivity of 0.8 and a specificity of 0.8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a29b17cdaeb68fcde9e2c47b4a309aeadaf32f5" target='_blank'>
              Detection of Genomic Copy Number Variations in Ovarian Cancer in the Peripheral Blood System
              </a>
            </td>
          <td>
            Lena Wahl, Ulyana Hliabtsova, Xueqian Qian, Anastasia Klopf, N. Hedemann, Inken Flörkemeier, C. Rogmans, Michael Kalab, Astrid Dempfle, Nicolai Maass, D. Bauerschlag, M. V. van Mackelenbergh, J. P. Weimer, Huy Duc Le
          </td>
          <td>2025-02-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f33e9da345df8224f97295c4e7efc05d28c012" target='_blank'>
              Crosstalk Between Plk1 and PTEN in Mitosis Affects Chromosomal Stability.
              </a>
            </td>
          <td>
            Wei Li, Xianning Wang, Jiannan Liu, Bing Liu, Yongjian Hao
          </td>
          <td>2025-03-21</td>
          <td>DNA and cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Plastids have highly conserved genomes in most land plants. However, in several families, plastid genomes exhibit high rates of nucleotide substitution and structural rearrangements among species. This elevated rate of evolution has been posited to lead to increased rates of plastid-nuclear incompatibilities (PNI), potentially acting as a driver of speciation. However, the extent to which plastid structural variation exists within a species is unknown. This study investigates whether plastid structural variation, observed at the interspecific level in Campanulaceae, also occurs within Campanula americana, a species with strong intraspecific PNI. We assembled multiple plastid genomes from three lineages of C. americana that exhibit varying levels of PNI when crossed. We then investigated the structural variation and repetitive DNA content among these lineages and compared the repetitive DNA content with that of other species within the family. Results We found significant variation in plastid genome size among the lineages of C. americana (188,309–201,788 bp). This variation was due in part to multiple gene duplications in the inverted repeat region. Lineages also varied in their repetitive DNA content, with the Appalachian lineage displaying the highest proportion of tandem repeats (~ 10%) compared to the Eastern and Western lineages (~ 6%). In addition, genes involved in transcription and protein transport showed elevated sequence divergence between lineages, and a strong correlation was observed between genome size and repetitive DNA content. Campanula americana was found to have one of the most repetitive plastid genomes within Campanulaceae. Conclusions These findings challenge the conventional view of plastid genome conservation within a species and suggest that structural variation, differences in repetitive DNA content, and divergence of key genes involved in transcription and protein transport may play a role in PNI. This study highlights the need for further research into the genetic mechanisms underlying PNI, a key process in the early stages of speciation. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11525-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d557865d1a1b45ab5236860da2c46dd5c6d3525" target='_blank'>
              Substantial structural variation and repetitive DNA content contribute to intraspecific plastid genome evolution
              </a>
            </td>
          <td>
            Alfredo López-Caamal, Tyler Gandee, Laura F. Galloway, Karen B. Barnard-Kubow
          </td>
          <td>2025-04-04</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Homologous recombination (HR) is a major pathway to repair DNA double-strand breaks. Hereditary breast and ovarian cancer syndrome (HBOC) is caused by germline pathogenic variants of HR-related genes, such as BRCA1 and BRCA2 (BRCA1/2). Cancer cells with HR deficiency are sensitive to poly(ADP-ribose) polymerase (PARP) inhibitors. Therefore, accurate evaluation of HR activity is helpful to diagnose HBOC and predict the effects of PARP inhibitors. The direct-repeat GFP (DR-GFP) assay has been utilized to evaluate cellular HR activity. However, evaluation by the DR-GFP assay tends to be qualitative and requires the establishment of stable cell lines. Therefore, we developed an assay to quantitatively measure HR activity called Assay for Site-Specific HR Activity (ASHRA), which can be performed by transiently transfecting two plasmids. In ASHRA, we use Cas9 endonuclease to create DNA double-strand breaks at specific sites in the genome, enabling the targeting of any endogenous loci. Quantification of HR products by real-time PCR using genomic DNA allows HR activity evaluated at the DNA level. Thus, ASHRA is an easy and quantitative method to evaluate HR activity at any genomic locus in various samples. Here, we present the protocols for adherent cells, suspension cells, and tumor tissues. Key features • This assay quantitatively evaluates homologous recombination (HR) activity. • This assay can measure HR activity in adherent cells, suspension cells, and tumor tissues. • This real-time PCR-based assay does not require a flow cytometer or next-generation sequencer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6041469126e683215dc13399e4746d11c09375" target='_blank'>
              Assay for Site-Specific Homologous Recombination Activity in Adherent Cells, Suspension Cells, and Tumor Tissues
              </a>
            </td>
          <td>
            Yuki Yoshino, Shin Kikuta, N. Chiba
          </td>
          <td>2025-04-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Extrahepatic cholangiocarcinoma (eCCA) is a rare but refractory cancer with dense desmoplasia. Prognosis-associated stromal cells in eCCA remain poorly characterized. Here, we profiled the tumor cellular composition and identified prognosis-related stromal signatures by single-cell RNA sequencing (scRNA-seq) in eCCA. ECCA patients were further stratified into different categories based on identified stromal signatures. Methods Using scRNA-seq, we profiled the transcriptomes of 37,498 individual cells from eight eCCA biopsies, including five tumor tissues and three paired adjacent normal tissues. Bulk RNA sequencing (bRNA-seq) was also performed on 43 eCCA tumor tissues. Stromal cell composition and heterogeneity were examined through differential gene expression and gene set enrichment analyses. By assessing the expression levels of marker genes in bRNA-seq data, the correlation of stromal cell clusters with survival was explored. The GSVA scores of the cell-specific signature genes of the prognosis-related stromal cell subtypes were calculated and used to stratify eCCA patients. Results The results revealed that tumor stroma in eCCA were composed of hematopoietic progenitor-like cells (HPLCs), fibroblasts (Fb), Schwann cells (Sch), endothelial cells and immune cells. Prognosis-associated stromal cell subpopulations included MKI67 + HPLC, TMEM158 + C3-Fb, FOXP3 + regulatory T cells (Treg), SLIT2 + Sch, TPSD1 + C2-mast cells (MC) and CTSG + C3-MC. Based on these stromal signatures, the eCCA tumors were categorized into three classes: proliferative Group 1 with enrichment of MKI67 + HPLC, inflammatory and fibrotic Group 2 with enrichment of TPSD1 + C2- MC, FOXP3 + Treg and TMEM158 + C3-Fb, and neuronal Group 3 with enrichment of SLIT2 + Sch and CTSG + C3-MC. ECCA patients in Group 3 had a better prognosis when compared to Group 1 and 2, reflecting different impact of stromal subtypes on tumor progression. Conclusion Single-cell transcriptomic analysis reveals prognosis-related stromal signatures that potentiate the stratification of eCCA into proliferative, inflammatory and fibrotic, and neuronal phenotypes, which has important implications on molecular classification and exploring therapeutic targets in eCCA. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-03829-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cc06851451f803822f912fc1af35b083131a67" target='_blank'>
              Single-cell transcriptomic analysis reveals prognosis-related stromal signatures that potentiate stratification of patients with extrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Chen Ni, Rulin Hua, Yuanyuan Yang, Jialu Liang, Wentao Liu, Linlin Wang, Xiaohan Yao, Anqi Li, Long Yu, Ruo Feng, Dekang Lv, Zhihai Qin, Wenlong Zhai
          </td>
          <td>2025-04-09</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fca76d81501556f6f871b8db3defb5ed37bf0b72" target='_blank'>
              Loss of BCL7A Permits IRF4 Transcriptional Activity and Cellular Growth in Multiple Myeloma.
              </a>
            </td>
          <td>
            C. Chakraborty, S. Talluri, Moritz Binder, E. Morelli, Jessica Encinas Mayoral, Sanika Derebail, Anil Aktas Samur, Charles B Epstein, Kenneth C Anderson, M. Shammas, M. Samur, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5821f61b692c344ab6ba3f9c1abecff3caf90b" target='_blank'>
              Multisite long-read sequencing reveals the early contribution of somatic structural variations to HBV-related hepatocellular carcinoma tumorigenesis.
              </a>
            </td>
          <td>
            Tianfu Zeng, Haotian Liao, Lin Xia, Siyao You, Yan Huang, Jiaxun Zhang, Yahui Liu, Xuyan Liu, Dan Xie
          </td>
          <td>2025-03-04</td>
          <td>Genome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4b38200a1231a3f3c1164315cc7d10135fa36f" target='_blank'>
              Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system
              </a>
            </td>
          <td>
            Stefanie Egetemaier, H. Chauvistré, R. Váraljai, Yichao Hua, Smiths S. Lueong, Samira Makhzami, Nalini Srinivas, Jan Forster, V. Ullrich, Simone Stupia, Valéria Schroeder, Sarah Scharfenberg, A. Hoewner, B. Shannan, J. Siveke, M. Baietti, Eleonora Leucci, J. Marine, A. Paschen, Bjoern Scheffler, D. R. Engel, Lisa M. Becker, Felix Nensa, Johannes Koester, Barbara Grünwald, S. Poepsel, Sabrina Ninck, F. Kaschani, Dirk Schadendorf, Juergen C. Becker, Alpaslan Tasdogan, Florian Rambow, Alexander Roesch
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, Min-Jung Lee, Nahoko Sato, Shraddha Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, Europa Azucena González, Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors.
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e5a35f02b297af4af1d7c2312aeb3f7e720c6f2" target='_blank'>
              p53-dependent chromatin relaxation is required for DNA double-strand break repair.
              </a>
            </td>
          <td>
            Hongyu Chen, Jin Shan, Wenjing Qi, Lili Chen, Xianlu Zeng
          </td>
          <td>2025-02-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC), a highly heterogeneous disease, shows genomic instability and a high mutation rate, making it difficult to treat. Recent studies revealed that cancer stem cells (CSCs) play a critical role in MIBC frequent recurrence and high morbidity. Previous research has shown that Cyclooxygenases-2 (COX-2) is particularly highly expressed in bladder cancer cells. In recent years, the development of oncolytic adenoviruses and their use in clinical trials have gained increased attention. In this study, we composed a conditionally replicative adenovirus vector (CRAd-synNotch) that carries the COX-2 promotor driving adenoviral E1 gene, the synNotch receptor therapeutic gene, and the Ad5/35 fiber gene. Activation of the COX-2 promoter gene causes replication only within COX-2 expressing cancer cells, thereby leading to tumor oncolysis. Also, CD44 and HIF signals contribute to cancer stemness and maintaining CSCs in bladder cancer, and the transduced synNotch receptor inhibits both CD44 and HIF signals simultaneously. We performed an in vivo study using a mouse xenograft model of T24 human MIBC cells and confirmed the significant antitumor activity of CRAd-synNotch. Our findings in this study warrant the further development of CRAd-synNotch for treating patients with MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eee590b5a2cfc96e3c5fb66879d16597ce9245" target='_blank'>
              A conditionally replicative adenovirus vector containing the synNotch receptor gene for the treatment of muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Ruhan A, H. Ueki, Shunya Nishioka, Rion Yamazaki, Marina Maekawa, Koichi Kitagawa, Hideaki Miyake, Toshiro Shirakawa
          </td>
          <td>2025-02-26</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long interspersed element 1 (LINE-1) retrotransposons are repetitive sequences that can move within the genome by an autonomous mechanism. To limit their mutagenic potential, benign cells restrict LINE-1 expression through molecular mechanisms such as DNA methylation and histone modification, but these mechanisms are usually impaired in cancer. Clear cell ovarian carcinoma (CCOC) represents 5–10% of ovarian cancers and is thought to arise from endometriosis. Women with advanced CCOC face poor prognoses, highlighting the importance of understanding early disease pathogenesis. In our study, 33 of 40 cases (over 82%) of CCOC tumors express ORF1p, a LINE-1-encoded protein. We found that LINE-1 de-repression is an early event in CCOC, as ORF1p is enhanced during the transition from typical to atypical endometriosis and persists in invasive cancer. Finally, using single-molecule array (Simoa) assays, we detected ORF1p in patient blood, suggesting it as a potential minimally invasive biomarker for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: KMT2A rearrangements occur in ~10% of acute myeloid leukemia (AML) cases and are critical for classification, risk stratification, and use of targeted therapy. However, insertions involving the KMT2A gene can evade detection using chromosomal analysis and/or fluorescence in situ hybridization (FISH). Methods: We present a case of a 22-year-old woman with acute monoblastic leukemia harboring a cryptic KMT2A::AFDN fusion identified by RNA sequencing. Initial FISH showed a 3′ KMT2A deletion, while conventional karyotyping and the automated bioinformatic pipeline for optical genome mapping (OGM) did not identify the canonical translocation. Results: To resolve these discrepancies, metaphase KMT2A FISH (break-apart fusion probe) was performed to assess whether KMT2A was translocated to another chromosome. However, the results did not support this possibility. As the fusion signal remained on the normal chromosome 11, with the 5′ KMT2A signal localized to the derivative chromosome 11. A subsequent manual review of the OGM data revealed a cryptic ~300 kb insertion of AFDN into the 3′ region of KMT2A, reconciling the discrepancies between chromosomal analysis, FISH, and RNA fusion results. Conclusions: This case highlights the importance of integrating multiple testing modalities with expert review when there is a discrepancy. Our findings emphasize the need for a comprehensive approach to genomic assessment to enhance diagnostic accuracy and guide therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b71c31c191c38bbe63bd8eb6331d6c11375f1" target='_blank'>
              Cryptic KMT2A::AFDN Fusion Due to AFDN Insertion into KMT2A in a Patient with Acute Monoblastic Leukemia
              </a>
            </td>
          <td>
            Qing Wei, Gokce A. Toruner, B. Thakral, Keyur P. Patel, N. Pemmaraju, Sa A. Wang, R. Kanagal-Shamanna, Guilin Tang, G. Issa, S. Loghavi, L. J. Medeiros, Courtney D. DiNardo
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-05-01</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Ultraviolet B (UV-B) is a significant risk factor for skin damage, as it induces cyclobutane pyrimidine dimers (CPD), which suppress DNA replication and transcription. Photolyase (PHR) is a blue light-dependent enzyme that repairs DNA damage caused by UV irradiation. While it is absent in human, it plays a crucial role in repairing CPD in other organisms. Acerola (Malpighia emarginata DC), a fruit with high antioxidant content, is widely consumed for health benefits. This study aimed to identify a novel PHR in acerola and evaluate its photorepair activity. Methods: Using RNA-seq data, we cloned the full-length sequence of the acerola PHR gene and constructed an expression vector. A stable transfected HEK293 cell line (HEK293/acPHR) was established. CPD repair activity was analyzed under blue light in these cells, as well as in normal human dermal fibroblasts (NHDFs) supplemented with extracellular vesicles (EVs) from HEK293/acPHR cells and extracellular vesicle-like nanoparticles derived from acerola extract. Results: Blue light-dependent CPD reduction was observed in HEK293/acPHR cells compared to control cells following UV-B irradiation. Additionally, CPD repair activity was demonstrated in NHDFs and HEK293 cells treated with EVs from HEK293/acPHR cells and nanoparticles from acerola extract. Conclusions: Acerola-derived PHR exhibits the potential to repair UV-induced DNA damage in human cells. Furthermore, EV-mediated delivery of PHR provides a promising avenue for extending photorepair capabilities to other cells. These findings highlight the potential applications of acerola PHR in the prevention and treatment of UV-induced skin damage and related conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb666706d3fe84c4a2f13c15b7b58624bee3eba8" target='_blank'>
              Acerola-Derived Photorepair System for Eliminating Ultraviolet-Induced Pyrimidine Dimers in Human Cells
              </a>
            </td>
          <td>
            Mamoru Yanagimachi, Tomohiro Umezu, M. Takanashi, Yoshiki Murakami, Takahiro Ochiya, Masahiko Kuroda
          </td>
          <td>2025-02-25</td>
          <td>Nutrients</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af5f4741557d24f0a2d8105bb65cd5398d4c085" target='_blank'>
              Alectinib enhances response to RBM39 degradation via SRPK1 inhibition
              </a>
            </td>
          <td>
            Yurui Ma, E. Poon, Chenchen Jin, Barbara Martins Da Costa, Yuewei Xu, Sadiya Quazi, Nikolaos Zourdoumis, Chiharu Wickremesinghe, L. Chesler, Hector C. Keun, Anke Nijhuis
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Virion Infectivity Factor (Vif) of the Human Immunodeficiency Virus type 1 (HIV-1) targets and degrades cellular APOBEC3 proteins, key regulators of intrinsic and innate antiretroviral immune responses, thereby facilitating HIV-1 infection. While Vif’s role in degrading APOBEC3G is well-studied, Vif is also known to cause cell cycle arrest, but the detailed nature of Vif’s effects on the cell cycle has yet to be delineated. In this study, we employed high-temporal resolution single-cell live imaging and super-resolution microscopy to monitor individual cells during Vif-induced cell cycle arrest. Our findings reveal that Vif does not affect the G2/M boundary as previously thought. Instead, Vif triggers a unique and robust pseudo-metaphase arrest, distinct from the mild prometaphase arrest induced by Vpr. During this arrest, chromosomes align properly and form the metaphase plate, but later lose alignment, resulting in polar chromosomes. Notably, Vif, unlike Vpr, significantly reduces the levels of both Protein Phosphatase 1 (PP1) and 2 A (PP2A) at kinetochores, which regulate chromosome-microtubule interactions. These results unveil a novel role for Vif in kinetochore regulation that governs the spatial organization of chromosomes during mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b84661d7085ab158b0fb68712a20f55d836cb3e" target='_blank'>
              HIV-1 Vif disrupts phosphatase feedback regulation at the kinetochore, leading to a pronounced pseudo-metaphase arrest
              </a>
            </td>
          <td>
            Dhaval Ghone, Edward L. Evans, Madison Bandini, Kaelyn G. Stephenson, N. Sherer, Aussie Suzuki
          </td>
          <td>2025-03-13</td>
          <td>eLife</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="DNA–RNA hybrids triggered by double-strand breaks (DSBs) are crucial intermediates during DSB repair, and their timely resolution requires numbers of RNA helicases, including DEAD box 1 (DDX1). However, how these helicases are recruited to DSB-induced hybrids in time remains largely unclear. Here, we revealed that squamous cell carcinoma antigen recognized by T cells 3 (SART3) promotes DDX1 binding to DNA–RNA hybrids at DSBs for optimal homologous recombination (HR) repair. SART3 itself associates with DNA–RNA hybrids and PAR chains and accumulates at DSBs in both PARylation- and DNA–RNA hybrids-dependent fashion. SART3 also associates with DDX1 and is necessary for DDX1 enrichment at DSBs. The defective SART3-DDX1 association observed in cells expressing the cancer-associated variant SART3-R836W impairs not only the accumulation of DDX1, but also hybrid removal and HR efficiency. Moreover, SART3 promotes DNA end resection through enhancing USP15-BARD1 association and BRCA1-BARD1 retention. Together, our study reveals an role of SART3 in DSB repair, rendering SART3 a promising target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0676fb55661a5af6ba07cbca79e578a0d145a7" target='_blank'>
              SART3 promotes homologous recombination repair by stimulating DNA-RNA hybrids removal and DNA end resection
              </a>
            </td>
          <td>
            Hui Fu, Min Huang, Honglin Wu, Hui Zheng, Yifei Gong, Lingyu Xing, Juanjuan Gong, Ruiyuan An, Qian Li, Xinyu Jie, Xiaolu Ma, Tie-Shan Tang, Caixia Guo
          </td>
          <td>2025-03-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028c06d1c8028cbd2ed32b765be45005f4a5730a" target='_blank'>
              Unveiling Candidate Drivers in Cancer Progression Using Somatic-IV Analysis
              </a>
            </td>
          <td>
            Wei Zhang, Saba Ghaffari, Andrew Browne, Carlos Macintosh, Xiaoyu Lu, Ha Dang, Gonzalo Lopez, Kai Wang
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cohesin and condensin, two related protein complexes, play essential roles in ensuring the accurate segregation of the genome into daughter cells during cell division. However, the interaction between cohesin and condensin in embryonic stem cells remains unclear, as does the specific function of the meiosis-specific cohesin complex. Cohesin maintains the cohesion of replicated sister chromatids until their separation at anaphase, whereas condensin facilitates the reorganization of chromosomes into a highly compact structure characteristic of mitosis. First, we found via ChIP-seq analysis that cohesins (SMC3, RAD21, and REC8) and condensin (SMC4) share DNA binding sites in close proximity and directly interact with the insulator protein CTCF. Second, siRNA-regulated SMC3 depletion led to nuclear accumulation of SMC4. Third, embryonic stem (ES) cells uniquely harbor cohesin complexes containing the meiotic kleisin subunit REC8. RAD21 knockdown increased the proportion of SMC3–REC8 complexes. Our findings indicate that cohesin and condensin make important contributions to the functions of the chromosomal organization, and that meiotic cohesin may be specifically required for the mitotic program in ES cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94533-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52632683a10ffe4a0eee9d00044e285b23727556" target='_blank'>
              Cohesin and condensin regulate chromosome topology and play an essential role in maintaining pluripotency in embryonic stem cells
              </a>
            </td>
          <td>
            Eui-Hwan Choi, Keun P. Kim
          </td>
          <td>2025-03-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Progressing transcription and replication machineries profoundly impact their underlying chromatin template. Consequently, transcription-replication conflict (TRC) sites are vulnerable to chromatin and epigenome alterations, provoking genome instability. Here, we engineered an inducible TRC reporter system using a genome-integrated R-loop-prone sequence and characterized the dynamic changes of the local chromatin structure inflicted by TRCs, leading to reduced nucleosome occupancy and replication fork blockage. Strikingly, inducing a small number of TRCs on the genome results in a measurable global replication stress response. Furthermore, we find a TRC-dependent increase in H3K79 methylation specifically at the R-loop forming TRC site. Accordingly, inhibition of the H3K79 methyltransferase DOT1L leads to reduced transcriptional output and an exacerbated DNA damage response, suggesting that deposition of this mark is required for effective transcription recovery and resolution of TRCs. Our work shows the molecular dynamics and reveals a specific epigenetic modifier bookmarking TRC sites, relevant to cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2c4476d7a403d74c2111b85e2186854b6c2af7f" target='_blank'>
              Transcription-replication conflicts drive R-loop-dependent nucleosome eviction and require DOT1L activity for transcription recovery
              </a>
            </td>
          <td>
            Marcel Werner, Manuel Trauner, Tamas Schauer, Henning Ummethum, Elizabeth Márquez-Gómez, Maxime Lalonde, Clare S. K. Lee, Ioannis Tsirkas, Atiqa Sajid, Augusto C Murriello, Gernot Längst, Stephan Hamperl
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Background Preimplantation genetic testing for aneuploidy and for chromosomal structural rearrangement (PGT-A/-SR) can improve clinical pregnancy rates and live birth rates, and shorten the time to pregnancy. The large-scale statistics on their efficacy and accuracy across different centres, as well as the frequency of abnormalities for each chromosome, will provide a valuable supplement to previous research. Methods Patients who had PGT-A or -SR procedures at five reproductive centres from 2018 to 2022 were recruited based on PGT-A/-SR indications. ChromInst and next-generation sequencing (NGS)-based PGT technology were utilised to detect copy number variations in embryos. Sequencing data metrics such as median absolute pairwise difference (MAPD) and detection success rate were analysed to evaluate the robustness of ChromInst. To assess ChromInst’s accuracy, the chromosomal results from amniocentesis, abortions, and neonatal blood was as the gold standard for negative PGT results; the fluorescence in situ hybridisation (FISH), which was performed on embryos that identified as aneuploid through PGT was as the gold standard for positive PGT results. The frequency of abnormalities in each chromosome was also explored in aneuploid embryos. Results A total of 5,730 embryos were tested from 1,015 patients in the study, 391 of whom had PGT-A and 624 of whom had PGT-SR. 99.5% (5,699/5,730) of the embryos had an NGS sequencing MAPD value < 0.25, and 99.3% (5,689/5,730) of the embryos achieved successful PGT-A/-SR detection. Compared with the gold standard, the concordance of negative PGT-A/-SR results was 99.8% (506/507), and that of positive results was 99.8% (1,123/1,125). The euploidy rate in the PGT-A population was 45.9% (981/2,135). The proportion of euploid + balanced embryos was highest among couples with non-polymorphic inversions (44.6%, 152/341), followed by those with Robertsonian translocations (39.0%, 293/752), and lowest among those with reciprocal translocations (22.5%, 483/2,143). Chromosomes 16, 22, and 15 had the highest frequency of autosomal trisomies among the embryos from PGT-A patients, while chromosomes 16, 22, and 21 had the highest frequency of monosomies. High-frequency chromosomes with de novo chromosomal abnormalities for trisomies and monosomies were similar in the PGT-SR patients to those in the PGT-A patients. Conclusions ChromInst-based PGT-A/-SR could accommodate operational variations among different clinical centres, ensuring accurate results through robust and stable detection performance. Prior to PGT-A/-SR, more trustworthy data could be provided to support the genetic counselling. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06242-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27907d7fcef7fbc30355e3fd09a1e1e55996153" target='_blank'>
              ChromInst: a multicentre evaluation of robustness in aneuploidy and structural rearrangement testing
              </a>
            </td>
          <td>
            W. Niu, Shanjun Dai, Linli Hu, Yao He, Xiqian Zhang, Xia Xue, Li Wu, Haixia Jin, Dun Liu, Keya Tong, Senlin Shi, L. Tian, Yifan Zhou, Guidong Yao, Sijia Lu, Ya-xin Yao, Dunmei Zhao, Lei Jin, H. Bai, Fenghua Liu, Dongyun Liu, Yingpu Sun
          </td>
          <td>2025-02-26</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many antibiotics indirectly generate reactive oxygen species (ROS) that can damage bacterial genomes. Oxidised nucleobases become genotoxic when they are targeted for repair through excision, generating a single-strand discontinuity that can be converted to a double-strand break (DSB) by an oncoming replication fork. Because the genomic location of nucleobase oxidation is stochastic, investigating the fate of DNA replication machinery (replisome) at single-strand discontinuities has been limited. Here we have addressed this issue by expressing Cas9 nickases in Bacillus subtilis to create site specific single-strand discontinuities in a bacterial chromosome. We find that nicks in either leading or lagging strand arrest bacterial replication fork progression and generate a DSB that requires repair using homologous recombination to allow replication restart. These discoveries provoke reassessment of the fundamental mechanism of bacterial homologous recombination and provide insights to the development of alternative antimicrobials by identifying a specific pathway that can potentiate ROS-dependent bacterial killing. HIGHLIGHTS Single-strand discontinuities inactivate the bacterial replisome Nicks on the leading or lagging strand template differentially affect fate of the helicase Double-strand breaks are repaired via homologous recombination and elicit the replication restart pathway Resection often generates a DNA strand with a 5′-end that must be processed to ensure productive helicase reloading">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733169863ada127f78f4c42d1d0de7e45edaadc" target='_blank'>
              Rescuing bacterial genome replication: essential functions to repair a double-strand break and restart DNA synthesis
              </a>
            </td>
          <td>
            Charles Winterhalter, Stepan Fenyk, H. Murray
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0818d31f654dc2a6f66d1620f983e96d2fa6b573" target='_blank'>
              Genomic and fragmentomic landscapes of cell-free DNA for early cancer detection.
              </a>
            </td>
          <td>
            D. C. Bruhm, Nicholas A. Vulpescu, Zachariah H. Foda, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2025-03-04</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed4652faa332d57f89c15c04ea896c21df38766" target='_blank'>
              Individualized pseudogenes networks for survival prognosis in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Arturo Kenzuke Nakamura-García, M. Kuijjer, J. Espinal-Enríquez
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Uniparental disomy is a type of chromosomal variation leading to in which both homologous chromosomes or chromosomal regions are inherited from one parent. Such variations have been detected for all chromosomes. The frequency of uniparental disomies is estimated at 1 case per 2000 births. The causes of uniparental disomies include errors during meiosis, postzygotic errors, Robertsonian and reciprocal translocations. Clinical manifestations are associated with loss of heterozygosity for pathogenic genetic variants and defects in genomic imprinting.Currently, the diagnosis of uniparental disomy is performed using methods such as microsatellite analysis, chromosomal microarray analysis, methyl-sensitive PCR, methyl-specific amplification of a probe dependent on multiplex ligation and next-generation sequencing (NGS). The methods used nowadays separately do not allow for a definitive diagnosis of uniparental disomy. A combination of NGS method that simultaneously assesses the DNA methylation status and regions of loss of heterozygosity, in particular those based on fragmentation of genomic DNA by methyl-dependent restriction enzymes, with classical approaches such as methyl-sensitive PCR and microsatellite testing will enable rapid and accurate diagnosis of uniparental disomies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26da94ff26872e12435f3b4e40259e455c4432b4" target='_blank'>
              Whole-genome sequencing for the identification of uniparental disomy
              </a>
            </td>
          <td>
            P. A. Suchko, A. Danilova, E. V. Pinkovskaya, O. Glotov, L. Danilov
          </td>
          <td>2025-03-05</td>
          <td>Rossiyskiy Vestnik Perinatologii i Pediatrii (Russian Bulletin of Perinatology and Pediatrics)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Stem cells display asymmetric histone inheritance, while nonstem progenitor cells exhibit symmetric patterns in the Drosophila male germ line. Here, we report that components involved in lagging strand synthesis, DNA polymerases α and δ, have substantially reduced levels in stem cells compared to progenitor cells, and this promotes local asymmetry of parental histone incorporation at the replication fork. Compromising Polα genetically induces the local replication-coupled histone incorporation pattern in progenitor cells to resemble that in stem cells, seen by both nuclear localization patterns and chromatin fibers. This is recapitulated using a Polα inhibitor in a concentration-dependent manner. The local old versus new histone asymmetry is comparable between stem cells and progenitor cells at both S phase and M phase. Together, these results indicate that developmentally programmed expression of key DNA replication components is important to shape stem cell chromatin. Furthermore, manipulating one crucial DNA replication component can induce replication-coupled histone dynamics in nonstem cells to resemble those in stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8c0df555eddd954cf53afad22d00f47394127d" target='_blank'>
              Increased levels of lagging strand polymerase α in an adult stem cell lineage affect replication-coupled histone incorporation
              </a>
            </td>
          <td>
            Brendon E. M. Davis, Jonathan C. Snedeker, R. Ranjan, Matthew Wooten, Savannah Sáde Barton, Joshua Blundon, Xin Chen
          </td>
          <td>2025-02-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S-adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient-deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti-Lat4 small interfering RNA complex nanoregulator (AS-F-NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor-infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation.
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Cancer is a disease of multicellularity, observed across the tree of life. In principle, animals with larger body sizes and longer lifespans should be at increased risk of developing cancer. However, there is no strong relationship between these traits and cancer across mammals. Previous studies have proposed that increased copy number of cancer-related genes may enhance the robustness of cancer suppression pathways in long-lived mammals, but these studies have not extended beyond known cancer-related genes. In this study, we conducted a phylogenetic generalized least squares analysis to test for associations between copy number of all protein-coding genes and longevity, body size, and cancer prevalence across 94 species of mammals. In addition to investigating the copy number of individual genes, we tested sets of related genes for a relationship between the aggregated gene copy number of the set and these traits. We did not find strong evidence to support the hypothesis that adaptive changes in gene copy number contribute to the lack of correlation between cancer prevalence and body size or lifespan. However, we found several biological processes where aggregate copy number was associated with malignancy rate. The strongest association was for the gene set relating to transforming growth factor beta, a cytokine that plays a role in cancer progression. Overall, this study provides a comprehensive evaluation of the role of gene copy number in adaptation to body size and lifespan and sheds light on the contribution of gene copy number to variation in cancer prevalence across mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a5893a996e91a392fac375709075fef182cddc" target='_blank'>
              Variable Gene Copy Number in Cancer-Related Pathways Is Associated With Cancer Prevalence Across Mammals
              </a>
            </td>
          <td>
            Sophie Matthews, Vahid Nikoonejad Fard, Marc Tollis, Cathal Seoighe
          </td>
          <td>2025-03-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the single nucleotide polymorphism (SNP) rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC (OR = 0.78, P = 0.015). Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling.
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is a rising issue worldwide, and numerous studies have focused on understanding the underlying reasons for its occurrence and finding proper ways to defeat it. By applying technological advances, researchers are continuously uncovering and updating treatments in cancer therapy. Their vast functions in the regulation of cell growth and proliferation and their significant role in the progression of diseases, including cancer. This review provides a comprehensive analysis of ncRNAs in breast cancer, focusing on long non-coding RNAs such as HOTAIR, MALAT1, and NEAT1, as well as microRNAs such as miR-21, miR-221/222, and miR-155. These ncRNAs are pivotal in regulating cell proliferation, metastasis, drug resistance, and apoptosis. Additionally, we discuss experimental approaches that are useful for studying them and highlight the advantages and challenges of each method. We then explain the results of these clinical trials and offer insights for future studies by discussing major existing gaps. On the basis of an extensive number of studies, this review provides valuable insights into the potential of ncRNAs in cancer therapy. Key findings show that even though the functions of ncRNAs are vast and undeniable in cancer, there are still complications associated with their therapeutic use. Moreover, there is an absence of sufficient experiments regarding their application in mouse models, which is an area to work on. By emphasizing the crucial role of ncRNAs, this review underscores the need for innovative approaches and further studies to explore their potential in cancer therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d581bb8c9e5870e5b44e989456c39919368fe9" target='_blank'>
              Non-coding RNAs, a double-edged sword in breast cancer prognosis
              </a>
            </td>
          <td>
            Maryam Solaimani, Sahar Hosseinzadeh, M. Abasi
          </td>
          <td>2025-04-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, Tilmann M. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47bc580b247bccc66bfbe3ed13daf3da5e31eb79" target='_blank'>
              Eukaryotes evade information storage-replication rate trade-off with endosymbiont assistance leading to larger genomes
              </a>
            </td>
          <td>
            Parthasarathi Sahu, Sashikanta Barik, Koushik Ghosh, Hemachander Subramanian
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce2dfa193c78b50465920b8e77584f1d48b10014" target='_blank'>
              SCFRMF-dependent degradation of the nuclear lamina releases the somatic chromatin mobility restriction for meiotic recombination.
              </a>
            </td>
          <td>
            Xinjie Yuan, Bowei Cai, Yuki Hamamura, A. Schnittger, Chao Yang
          </td>
          <td>2025-02-21</td>
          <td>Science advances</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Birt-Hogg-Dubé syndrome (BHD) patients are uniquely susceptible to all renal tumor subtypes. However, the underlying mechanism of carcinogenesis is unclear. To study cancer development in BHD, we used human proximal kidney (HK2) cells and found that long-term folliculin (FLCN) knockdown was required to increase the tumorigenic potential of these cells, as evidenced by the formation of larger spheroids under nonadherent conditions. Transcriptomic and proteomic analyses revealed links between the FLCN, cell cycle control and DNA damage response (DDR) machinery. In addition, HK2 cells lacking FLCN had an altered transcriptome profile and enriched cell cycle control genes. G1/S cell cycle checkpoint signaling was compromised by increased protein levels of cyclin D1 (CCND1) and hyperphosphorylation of retinoblastoma 1 (RB1). A FLCN interactome screen revealed that FLCN binds to DNA-dependent protein kinase (DNA-PK). This novel interaction was reversed in an irradiation-responsive manner. Knockdown of FLCN in HK2 cells caused a marked increase in γH2AX and RB1 phosphorylation. The levels of both CCND1 and phosphorylated RB1 remained high during DNA damage, which was associated with defective cell cycle control caused by FLCN knockdown. Furthermore, Flcn-knockdown C. elegans were defective in cell cycle arrest caused by DNA damage. This work revealed that long-term FLCN loss and associated cell cycle defects in BHD patients could contribute to their increased risk of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce72e1df530d7f8cfbe56a7a8e356852f9f7441" target='_blank'>
              Characterizing the tumor suppressor activity of FLCN in Birt-Hogg-Dubé syndrome cell models through transcriptomic and proteomic analysis.
              </a>
            </td>
          <td>
            Rachel-Ann Jones, E. Dunlop, Jesse D Champion, Peter F Doubleday, T. Claessens, Zahra Jalali, Sara Seifan, Iain Perry, Peter J. Giles, Oliver Harrison, Barry J. Coull, A. Houweling, A. Pause, Bryan A Ballif, Andrew R. Tee
          </td>
          <td>2025-03-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8876255590dea7ef4a31a07f23a3590537b155f1" target='_blank'>
              Immunogenomic cancer evolution: A framework to understand cancer immunosuppression.
              </a>
            </td>
          <td>
            S. Kumagai, Yusaku Momoi, H. Nishikawa
          </td>
          <td>2025-03-28</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oscillatory p53 expression occurs in individual cells responding to DNA breaks. While the majority of cells exhibit the same qualitative response, quantitative features of the oscillations (e.g., amplitude or period) can be highly variable between cells, generating heterogeneity in downstream cell fate responses. Since heterogeneity can be detrimental to therapies based on DNA damage, methods to induce synchronization of p53 oscillations across cells in a population have the potential to generate more predictable responses to DNA-damaging treatments. Using mathematical modeling and time-lapse microscopy, we demonstrated that p53 oscillations can be synchronized through the phenomenon of phase resetting. Surprisingly, p53 oscillations were synchronized over a wider range of damage-induction frequencies than predicted computationally. Recapitulating the range of synchronizing frequencies required, non-intuitively, a less robust oscillator. We showed that p53 phase resetting altered the expression of downstream targets responsible for cell fate depending on target mRNA stability. This study demonstrates that p53 oscillations can be phase reset and highlights the potential of driving p53 dynamics to reduce cellular variability and synchronize cell fate responses to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/372ff4f36befbd056cd96855dd2d6817d3ff38d6" target='_blank'>
              Pulsed stimuli enable p53 phase resetting to synchronize single cells and modulate cell fate
              </a>
            </td>
          <td>
            Harish Venkatachalapathy, Samuel Dallon, Zhilin Yang, Samira M. Azarin, Casim A. Sarkar, Eric Batchelor
          </td>
          <td>2025-03-03</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc6f21df46de0763db659ebbc92b87b44e41294" target='_blank'>
              Centrophilic Retrotransposons of Plant Genomes.
              </a>
            </td>
          <td>
            Alexandros Bousios, T. Kakutani, Ian R. Henderson
          </td>
          <td>2025-02-14</td>
          <td>Annual review of plant biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff664ee6fe6a3f28afa5b7cbe6650ff21d793513" target='_blank'>
              Fetal context conveys heritable protection against MLL-rearranged leukemia that depends on MLL3
              </a>
            </td>
          <td>
            Jonny Mendoza-Castrejon, Wei Yang, Elisabeth D Denby, Helen Wang, Emily B. Casey, Rohini Muthukumar, Riddhi M Patel, Jihye Yoon, Yanan Li, J. M. White, Ran Chen, Luis Batista, Jeffrey A. Magee
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Over the past decade, circular RNAs (circRNAs) have gained recognition as a novel class of genetic molecules, many of which are implicated in cancer pathogenesis via different mechanisms, including drug resistance, immune escape, and radio-resistance. ExosomalcircRNAs, in particular, facilitatecommunication between tumour cells and micro-environmental cells, including immune cells, fibroblasts, and other components. Notably, micro-environmental cells can reportedly influence tumour progression and treatment resistance by releasing exosomalcircRNAs. circRNAs often exhibit tissue- and cancer-specific expression patterns, and growing evidence highlights their potential clinical relevance and utility. These molecules show strong promise as potential biomarkers and therapeutic targets for cancer diagnosis and treatment. Therefore, this review aimed to briefly discuss the latest findings on the roles and resistance mechanisms of key circRNAs in the treatment of various malignancies, including lung, breast, liver, colorectal, and gastric cancers, as well as haematological malignancies and neuroblastoma.This review will contribute to the identification of new circRNA biomarkers for the early diagnosis as well as therapeutic targets for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/591c39d8ca29bfec5f8e19848bdc01b09dbd47c9" target='_blank'>
              Lake Malawi cichlid pangenome graph reveals extensive structural variation driven by transposable elements.
              </a>
            </td>
          <td>
            Fu Xiang Quah, Miguel Vasconcelos Almeida, Moritz Blumer, Chengwei Ulrika Yuan, Bettina Fischer, Kirsten See, Ben Jackson, R. Zatha, B. Rusuwa, George F. Turner, M. E. Santos, H. Svardal, M. Hemberg, R. Durbin, E. Miska
          </td>
          <td>2025-04-10</td>
          <td>Genome research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality in homologous recombination (HR)-deficient cancers caused by Poly (ADP-ribose) polymerase inhibitors (PARPi) has been classically attributed to its role in DNA repair. The mode of action of PARPi and resistance thereof are now believed to be predominantly replication associated. Therefore, effective combinatorial approaches of targeting replication fork processing along with HR-downregulation to target HR-proficient and possibly PARPi-resistant tumors are warranted. Stilbenes are a privileged class of molecules, which include resveratrol, pterostilbene, piceatannol, etc, that modulate both replication processes and RAD51-expression. In this investigation, by screening a small library of stilbenes, including in-house synthesized molecules, trans-4,4'-dihydroxystilbene (DHS) was discovered as a potent natural agent, which downregulates RAD51 expression and HR repair (GFP-reporter assay). DHS induces extensive synergistic cell death in ovarian cancers when combined with talazoparib (PARPi). Mechanistically, DHS elicits replication-stress through severely impeding replication fork progress, speed, and inducing fork-asymmetry. This leads to robust induction of single stranded DNA (ssDNA) gaps and poly-ADP-ribosylation (PARylation) in S-phase cells, signifying issues related to lagging (Okazaki) strand synthesis. PARPi, which abrogates PARylation, potentiates DHS induced ssDNA gaps, and their conversion into lethal double strand breaks through MRE11 action. Furthermore, the combination is highly effective in mitigating ovarian tumor xenograft growth in SCID mice and exhibited a good therapeutic-index with no/minimal tissue-toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e984c0229d51f87822834c928220d9d0bc66005" target='_blank'>
              Targeting Replication Fork Processing Synergizes with PARP Inhibition to Potentiate Lethality in Homologous Recombination Proficient Ovarian Cancers.
              </a>
            </td>
          <td>
            Ganesh Pai Bellare, Kshama Kundu, Papiya Dey, Krupa Thankam Philip, Nitish Chauhan, Muskan Sharma, Sankarsingh Kesharsingh Rajput, B. Patro
          </td>
          <td>2025-03-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd93a55f84ce2095fede9891e98cc26d5f84458d" target='_blank'>
              Doxorubicin, a DNA intercalator, inhibits transcription elongation.
              </a>
            </td>
          <td>
            Mathew Tempel, Kari Green, Dhanvi Prajapati, Angela Duaqui, Mahboobeh Norouzi, Hedieh Sattarifard, Ahmed Ashraf, Elly Wu, Athanasios Zovoilis, Ted M Lakowski, James R Davie
          </td>
          <td>2025-02-14</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56b53109651f083205e0fc320c14b9f0267e185" target='_blank'>
              Mining metagenomes and metatranscriptomes unveils viruses associated with cutaneous squamous cell carcinoma in hematopoietic stem cell transplant recipients.
              </a>
            </td>
          <td>
            Jana K Dickter, Yuqi Zhao, Vishwas Parekh, Huiyan Ma, Badri G Modi, Wai-Yee Li, S. Armenian, Xiwei Wu, Farah R Abdulla
          </td>
          <td>2025-03-31</td>
          <td>Microbiology spectrum</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Skin cancer is the most common malignancy worldwide, comprising three major types: melanoma, basal cell carcinoma (BCC), and squamous cell carcinoma (SCC). A critical goal in advancing our understanding of skin cancer is to identify the shared and distinct biological mechanisms that drive the risk profiles for each skin cancer type. In this study, we integrated tissue-specific chromatin conformation and expression quantitative trait loci data (spatial eQTL) to construct gene regulatory networks (GRNs) for melanocytes, sun-exposed skin, not sun-exposed skin, and blood. These GRNs, which capture spatially regulated gene expressions, were used as instrumental variables along with GWAS summary statistics for melanoma, BCC and SCC, to infer causal relationships between specific gene expression changes and each type of skin cancer. These Mendelian randomization analyses identified 82, 62, and 125 causal genes for melanoma, BCC, and SCC, respectively, with many of these genes not evident from the GWAS data alone. Our analyses revealed distinct mechanisms for each skin cancer type: telomere maintenance and nevus pathways in melanoma, inherited immune traits in BCC, and p53 dysfunction in SCC. Notably, apoptosis and pigmentation emerged as shared biological processes across skin cancers. These findings provide new insights into the genetic drivers of three distinct skin cancers, highlighting new gene targets which can be used to increase diagnostic precision, as well as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1a0a87d4f015e13185fa2728ac9fd439bf3488" target='_blank'>
              Mendelian randomizations with spatial gene networks reveal shared and distinct drivers of risk in major skin cancer types
              </a>
            </td>
          <td>
            Michael Pudjihartono, J. O’Sullivan, W. Schierding
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutator phenotypes are short-lived due to the rapid accumulation of deleterious mutations. Yet, recent observations reveal that certain fungi can undergo prolonged accelerated evolution after losing DNA repair genes. Here, we surveyed 1,154 yeast genomes representing nearly all known yeast species of the subphylum Saccharomycotina to examine the relationship between reduced DNA repair repertoires and elevated evolutionary rates. We identified three distantly related lineages—encompassing 12% of species—with substantially reduced sets of DNA repair genes and the highest evolutionary rates in the entire subphylum. Two of these “faster-evolving lineages” (FELs)—a subclade within the order Pichiales and the Wickerhamiella/Starmerella (W/S) clade (order Dipodascales)—are described here for the first time, while the third corresponds to a previously documented Hanseniaspora FEL. Examination of DNA repair gene repertoires revealed a set of genes predominantly absent in these three FELs, suggesting a potential role in the observed acceleration of evolutionary rates. Genomic signatures in the W/S clade are consistent with a substantial mutational burden, including pronounced A|T bias and signatures of endogenous DNA damage. The W/S clade appears to mitigate UV-induced damage through horizontal acquisition of a bacterial photolyase gene, underscoring how gene loss may be offset by nonvertical evolution. These findings highlight how the loss of DNA repair genes gave rise to hypermutators that persist across macroevolutionary timescales, with horizontal gene transfer as an avenue for partial functional compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e78c4fc2b6cc810f590c3dd41520772da92ce0ea" target='_blank'>
              Stable hypermutators revealed by the genomic landscape of DNA repair genes among yeast species
              </a>
            </td>
          <td>
            Carla Gonçalves, Jacob L. Steenwyk, D. Rinker, D. A. Opulente, A. Labella, Marie-Claire Harrison, John F. Wolters, Xiaofan Zhou, Xing‐Xing Shen, S. Covo, M. Groenewald, C. T. Hittinger, A. Rokas
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e24bbb19a6e12af52d533253b1371130f86db3" target='_blank'>
              Integrating genomic, transcriptomic and epigenetic data to identify phenotypically impactful driver pathways in glioblastoma
              </a>
            </td>
          <td>
            Junhua Zhang
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/735584e14c88c91d3e142e0b8a6d20e3a98ec51e" target='_blank'>
              C-terminal fusion partner activity contributes to the oncogenic functions of YAP1::TFE3
              </a>
            </td>
          <td>
            Patrick J Cimino, Dylan J Keiser, Abigail G Parrish, Eric C. Holland, Frank Szulzewsky
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e899cf4a3d1394fc8fe742ffd50bbb8e81d067" target='_blank'>
              Extensive mutual influences of SMC complexes shape 3D genome folding.
              </a>
            </td>
          <td>
            Han Zhao, Lirong Shu, Shiyi Qin, Fangxuan Lyu, Fuhai Liu, En Lin, Sijian Xia, Baiyue Wang, Manzhu Wang, Fengnian Shan, Yinzhi Lin, Lin Zhang, Yufei Gu, Gerd A. Blobel, Kai Huang, Haoyue Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaaa908df3e4b28207f7ab8e87d00d4a0e587cfc" target='_blank'>
              Integration Analysis of Circle-Sequencing and Transcriptome Reveals Extrachromosomal Circular DNA Is Involved in the Regulation of Vascular Cambium Annual Cycle in Chinese Pine.
              </a>
            </td>
          <td>
            Yushuang Song, Keji Yu, Yayu Guo, Yuan Cao, Changwen Xu, Lingyu Ma, Zhun Zhang, Yiqun Liu, Jinhuan Yin, Zijian Hu, Shiya Shen, Huimin Xu, Xi Zhang, Shihui Niu, Yaning Cui, Jinxing Lin
          </td>
          <td>2025-04-09</td>
          <td>Plant, cell & environment</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ac4bb2802041c240d28204f63b19e18c077194" target='_blank'>
              Targeting Regulatory Noncoding RNAs in Human Cancer: The State of the Art in Clinical Trials
              </a>
            </td>
          <td>
            R. Piergentili, S. Sechi
          </td>
          <td>2025-04-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="FOXP3+ natural regulatory T cells (nTregs) promote resolution of inflammation and repair of epithelial damage following viral pneumonia-induced lung injury, thus representing a cellular therapy for patients with acute respiratory distress syndrome (ARDS). Whether in vitro induced Tregs (iTregs), which can be rapidly generated in substantial numbers from conventional T cells, also promote lung recovery is unknown. nTregs require specific DNA methylation patterns maintained by the epigenetic regulator, ubiquitin-like with PHD and RING finger domains 1 (UHRF1). Here, we tested whether iTregs promote recovery following viral pneumonia and whether iTregs require UHRF1 for their pro-recovery function. We found that adoptive transfer of iTregs to mice with influenza virus pneumonia promotes lung recovery and that loss of UHRF1-mediated maintenance DNA methylation in iTregs leads to reduced engraftment and a delayed repair response. Transcriptional and DNA methylation profiling of adoptively transferred UHRF1-deficient iTregs that had trafficked to influenza-injured lungs demonstrated transcriptional instability with gain of effector T cell lineage-defining transcription factors. Strategies to promote the stability of iTregs could be leveraged to further augment their pro-recovery function during viral pneumonia and other causes of ARDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8259d3d12170176131ead01b0d7705b88e45403" target='_blank'>
              Maintenance DNA methylation is required for induced regulatory T cell reparative function following viral pneumonia
              </a>
            </td>
          <td>
            A. Joudi, J. Gurkan, Qianli Liu, Manuel A. Torres Acosta, K. Helmin, Luisa Morales-Nebreda, N. Mambetsariev, Carla P. Reyes Flores, H. Abdala-Valencia, Elizabeth M. Steinert, S. Weinberg, Benjamin D. Singer
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="eccDNA is a driver of many cancers and a potential intermediate in other age-related disorders. However, little is known about the mechanisms underlying eccDNA formation in healthy tissue and how aging affects these processes. Here, we present an atlas of eccDNA across seven tissues of male mice spanning four ages. EccDNA correlates with open chromatin characterized by signatures of H3K27ac and H3K4me1. Additionally, the mutational load of eccDNA on genes correlates with tissue-specific transcription and increases logarithmically as a function of transcript level. Still, a population of intron-dense genes with many splice forms remains sheltered from eccDNA formation. We also find that the total number of eccDNA molecules does not increase as mice age, unlike other types of mutations. Our data reveal a link between eccDNA formation and transcript level that may drive gene architecture in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent research has shifted the focus from the genetic code of DNA to its structural variations, which significantly impact cancer, genetic diseases, and gene regulation. Structural changes, such as the transition from B-DNA to A-DNA, influence DNA stability and flexibility and are affected by factors like DNA methylation and sugar puckering. This study is the first to investigate the relationship between DNA conformational changes and lifespan in two rodent species. The analysis focused on long-lived Nannospalax xanthodon and shorter-lived Rattus rattus, utilizing infrared spectroscopy and principal component analysis (PCA) to examine liver DNA. Results indicated that transition from B-form to A- and Z-forms were more prevalent in N. xanthodon than in R. rattus. However, the dominant DNA conformations in both species are in B-form. Additionally, N-type sugar puckers (C3-endo conformation), associated with these DNA forms, were more prominent in N. xanthodon. In contrast, S-type sugar puckers (C2-endo conformation), characteristic of B-DNA, were found at lower levels in N. xanthodon. Furthermore, the variations in methylation-specific structural modifications of nucleobases were quantitatively assessed among these species. The study proposes a significant connection between the long lifespan of N. xanthodon, which live underground, and their unique DNA structure, offering insights into how different DNA forms, as well as the conformations of their backbone and sugar-base components, may affect longevity, highlighting potential research avenues regarding the biomolecular aspects of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e194ada8d97ff0f6ac7375a7dc3c14a580e955" target='_blank'>
              Structural features of DNA and their potential contribution to blind mole rat (Nannospalax xanthodon) longevity
              </a>
            </td>
          <td>
            Gamzenur Sönmez, Tuba Yağcı Gurbanov
          </td>
          <td>2025-03-25</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0e4ce978bcbd93656c4cbcf8bd876d6363d193" target='_blank'>
              Analysis of the role of cellular autosomal genetic banding position subtyping on clinical prognosis and immune infiltration in colorectal cancer
              </a>
            </td>
          <td>
            Jiansong Wang, Xin Mao
          </td>
          <td>2025-03-27</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The RAD51-BRCA2 interaction is central to DNA repair through homologous recombination. Emerging evidence indicates RAD51 overexpression and its correlation with chemoresistance in various cancers, suggesting RAD51-BRCA2 inhibition as a compelling avenue for intervention. We previously showed that combining olaparib (a PARP inhibitor (PARPi)) with RS-35d (a BRCA2-RAD51 inhibitor) was efficient in killing pancreatic ductal adenocarcinoma (PDAC) cells. However, RS-35d impaired cell viability even when administered alone, suggesting potential off-target effects. Here, through multiple, integrated orthogonal biological approaches in different 2D and 3D PDAC cultures, we characterised RS-35d enantiomers, in terms of mode of action and single contributions. By differentially inhibiting both RAD51-BRCA2 interaction and sensor kinases ATM, ATR and DNA-PK, RS-35d enantiomers exhibit a ‘within-pathway synthetic lethality’ profile. To the best of our knowledge, this is the first reported proof-of-concept single small molecule capable of demonstrating this built-in synergism. In addition, RS-35d effect on BRCA2-mutated, olaparib-resistant PDAC cells suggests that this compound may be effective as an anticancer agent possibly capable of overcoming PARPi resistance. Our results demonstrate the potential of synthetic lethality, with its diversified applications, to propose new and concrete opportunities to effectively kill cancer cells while limiting side effects and potentially overcoming emerging drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d347103c381bd6fd50aa622f9e2cee5595e830" target='_blank'>
              Investigating synthetic lethality and PARP inhibitor resistance in pancreatic cancer through enantiomer differential activity
              </a>
            </td>
          <td>
            Mirco Masi, Laura Poppi, Viola Previtali, Shannon R. Nelson, Kieran Wynne, Giulia Varignani, Federico Falchi, Marina Veronesi, E. Albanesi, Daniele Tedesco, Francesca De Franco, Andrea Ciamarone, Samuel H. Myers, José Antonio Ortega, Greta Bagnolini, Giovanni Ferrandi, Fulvia Farabegoli, Nicola Tirelli, Giuseppina Di Stefano, G. Oliviero, Naomi Walsh, M. Roberti, S. Girotto, Andrea Cavalli
          </td>
          <td>2025-03-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78eb760dac971fc0c9d9c921e3630cb4941aa8c4" target='_blank'>
              Tumor-colonizing Pseudoalteromonas elyakovii metabolically reprograms the tumor microenvironment and promotes breast ductal carcinoma.
              </a>
            </td>
          <td>
            Shuyan Liu, Youpeng Pan, Chaopeng Zheng, Qinghui Zheng, Yaoqiang Du, Yajuan Zheng, Hongchao Tang, Xiaozhen Liu, Jiancheng Mou, Xin Zeng, Zhuotao Yang, Wenjuan Gui, Yuning Tang, Mingxing Xu, Zhihao Ye, Haotian Su, Qiuran Xu, Xuli Meng
          </td>
          <td>2025-04-07</td>
          <td>mBio</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is among the most aggressive cancers, with rising incidence and limited responsiveness to immunotherapy due to its highly suppressive tumor microenvironment (TME). Tertiary lymphoid structures (TLS), ectopic formation structures of immune cells, are linked to better prognosis and improved immunotherapy responses in PDAC. Understanding TLS’s role in PDAC could enhance immunotherapy effectiveness. Methods This study integrated transcriptomic and clinical data from 310 PDAC patients in GEO database. We performed consensus clustering using tumor-associated TLS (TA-TLS) genes, identifying three distinct molecular subtypes. Single-sample gene set enrichment analysis (ssGSEA) was then employed to calculate a TLS score for each patient, allowing for TLS-based evaluation. Key prognostic genes were identified using an iterative LASSO method, leading to the construction of a risk assessment model, which was validated across independent cohorts. We further analyzed the TLS score using single-cell RNA sequencing (scRNA-seq), visualized key gene expression, and validated protein expression through immunohistochemistry (IHC). Additionally, we explored the effects of DNASE1L3 on cell proliferation and migration, and its immune-related functions using Gene Set Enrichment Analysis (GSEA) and multiplex cytokine analysis. Results Consensus clustering revealed three PDAC molecular subtypes with significant differences in prognosis, TA-TLS gene expression, and TME features. The TLS score effectively stratified patients into high and low groups, correlating with survival outcomes and TME characteristics. Our risk model, validated across cohorts, reliably predicted patient outcomes. Validation studies showed lower expression of DNASE1L3 and IL33 in tumor tissues. scRNA-seq confirmed TLS score associations with immune cells. DNASE1L3 overexpression inhibited PDAC cell proliferation and migration, with cytokine analysis indicating increased immune activity. Conclusions This study elucidated the expression profile of TA-TLS genes in PDAC, constructed a TLS gene-based scoring system, and developed a related risk model. We also explored the functions and potential antitumor mechanisms of key genes, providing evidence and new insights for enhancing TLS-targeted immunotherapy strategies in PDAC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06152-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad4f13e415f9357404f99d30b4979256aef4341" target='_blank'>
              Prognostic implications and characterization of tumor-associated tertiary lymphoid structures genes in pancreatic cancer
              </a>
            </td>
          <td>
            Enkui Zhang, Yongsu Ma, Zonghao Liu, Jixin Zhang, Weikang Liu, Yiran Chen, Guangnian Liu, Xinxin Liu, Fusheng Zhang, Yu Zhu, Yinmo Yang, Xiaodong Tian
          </td>
          <td>2025-03-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Inflammation is a multifaceted defense mechanism of the immune system against infection. Chronic inflammation is intricately linked to all stages of tumorigenesis and is therefore associated with an elevated risk of developing serious cancers. Epigenetic mechanisms have the capacity to trigger inflammation as well as facilitate tumor development and transformation within an inflammatory context. They achieve this by dynamically modulating the expression of both pro—inflammatory and anti—inflammatory cytokines, which in turn sustains chronic inflammation. The aberrant epigenetic landscape reconfigures the transcriptional programs of inflammatory and oncogenic genes. This reconfiguration is pivotal in dictating the biological functions of both tumor cells and immune cells. Aberrant histone H3 lysine 27 site (H3K27) methylation has been shown to be involved in biological behaviors such as inflammation development, tumor progression, and immune response. The establishment and maintenance of this repressive epigenetic mark is dependent on the involvement of the responsible histone modifying enzymes enhancer of zeste homologue 2 (EZH2), jumonji domain containing 3 (JMJD3) and ubiquitously transcribed tetratricopeptide repeat gene X (UTX) as well as multiple cofactors. In addition, specific pharmacological agents have been shown to modulate H3K27 methylation levels, thereby modulating inflammation and carcinogenesis. This review comprehensively summarises the current characteristics and clinical significance of epigenetic regulation of H3K27 methylation in the context of inflammatory response and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a333da9e2131a9a75deaa130f4cc548713bf46a8" target='_blank'>
              Regulation of histone H3K27 methylation in inflammation and cancer
              </a>
            </td>
          <td>
            Jing Ma, Yalin Zhang, Jingyuan Li, Y. Dang, Dan Hu
          </td>
          <td>2025-03-05</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead71d789cc48679b7887d342b68b10c1657a0d8" target='_blank'>
              Glycolysis Induces Abnormal Transcription Through Histone Lactylation in T-lineage Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Wenyan Wu, Jingyi Zhang, Huiying Sun, Xiaoyu Wu, Han Wang, Bowen Cui, Shuang Zhao, Kefei Wu, Yanjun Pan, Rongrong Fan, Ying Zhong, Xiang Wang, Ying Wang, Xiaoxiao Chen, Jianan Rao, Ronghua Wang, Kai Luo, Xinrong Liu, Liang Zheng, Shuhong Shen, Meng Yin, Yangyang Xie, Yu Liu
          </td>
          <td>2025-04-07</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0f5dce3feb58cd819120465c413a2814ac1e2" target='_blank'>
              Path2Omics: Enhanced transcriptomic and methylation prediction accuracy from tumor histopathology
              </a>
            </td>
          <td>
            Danh-Tai Hoang, E. Shulman, S. R. Dhruba, N. Nair, R. K. Barman, H. Lalchungnunga, Omkar Singh, MacLean P Nasrallah, Eric A Stone, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is responsible for over 900,000 annual deaths worldwide. Emerging evidence supports pro‐carcinogenic bacteria in the colonic microbiome are at least promotional in CRC development and may be causal. We previously showed toxigenic C. difficile from human CRC‐associated bacterial biofilms accelerates tumorigenesis in Apc Min/+ mice, both in specific pathogen‐free mice and in gnotobiotic mice colonized with a defined consortium of bacteria. To further understand host–microbe interactions during colonic tumorigenesis, we combined single‐cell RNA‐sequencing (scRNA‐seq), spatial transcriptomics, and immunofluorescence to define the molecular spatial organization of colonic dysplasia in our consortium model with or without C. difficile. Our data show a striking bipartite regulation of Deleted in Malignant Brain Tumors 1 (DMBT1) in the inflamed versus dysplastic colon. From scRNA‐seq, differential gene expression analysis of normal absorptive colonocytes at 2 weeks postinoculation showed DMBT1 upregulated by C. difficile compared to colonocytes from mice without C. difficile exposure. In contrast, our spatial transcriptomic analysis showed DMBT1 dramatically downregulated in dysplastic foci compared with normal‐adjacent tissue. We further integrated our datasets to generate custom colonic dysplasia scores and ligand‐receptor mapping. Validation with immunofluorescence showed DMBT1 protein downregulated in dysplastic foci from three mouse models of colonic tumorigenesis and in adenomatous dysplasia from human samples. Finally, we used mouse and human organoids to implicate WNT signaling in the downregulation of DMBT1 mRNA and protein. Together, our data reveal cell type‐specific regulation of DMBT1, a potential mechanistic link between bacteria and colonic tumorigenesis. Published 2025. This article is a U.S. Government work and is in the public domain in the USA. The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7def78e585a3b57bba506b3445109c092c108f8" target='_blank'>
              Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia
              </a>
            </td>
          <td>
            Emily H Green, Subhag R. Kotrannavar, Megan E. Rutherford, Hannah M. Lunnemann, Harsimran Kaur, Cody N. Heiser, Hua Ding, A. Simmons, Xiao Liu, Borden Lacy, Kay Washington, M. Shrubsole, Qi Liu, Ken S. Lau, Cynthia L Sears, Robert J. Coffey, Julia L. Drewes, Nicholas O Markham
          </td>
          <td>2025-03-03</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ffb5861f3091086b8e6d39be2fc47951f44015b" target='_blank'>
              Cohesins: Crossroad Between Cornelia de Lange Spectrum and Cancer Predisposition.
              </a>
            </td>
          <td>
            Laura Rigotti, Stefano Rebellato, Antonella Lettieri, Silvia Castiglioni, M. Mariani, Simona Totaro, C. Saitta, C. Gervasini, G. Fazio, Valentina Massa, Giovanni Cazzaniga, Angelo Selicorni
          </td>
          <td>2025-04-05</td>
          <td>American journal of medical genetics. Part A</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7749502012ae50efd549b8d9691628c7f5bce04c" target='_blank'>
              Epigenetics of Human Telomeres.
              </a>
            </td>
          <td>
            Nicole Bettin, Mélina Vaurs, Anabelle Decottignies
          </td>
          <td>2025-03-17</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The specialised structure of the centromere is critical for effective chromosome segregation, but its repetitive nature makes it vulnerable to rearrangements. Centromere fragility can drive tumorigenesis, but protective mechanisms preventing fragility are still not fully understood. The PBAF chromatin remodelling complex is frequently misregulated in cancer, but its role in cancer is incompletely characterized. Here, we identify PBAF as a protector of centromere and pericentromere structure with profound consequences for genome stability. A conserved feature of isogenic cell lines lacking PBRM1, a subunit of PBAF, is compromised centromere and pericentromere integrity. PBAF is present at these regions, and binding patterns of PBAF and H3K9 methylation change when PBRM1 is absent. PBRM1 loss creates a dependence on the spindle assembly checkpoint, which represents a therapeutic vulnerability. Importantly, we find that even in the absence of any perturbations, PBRM1 loss leads to centromere fragility, thus identifying a key player in centromere protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4067eaba6eda6efd31e27843f706f972ee2e6e" target='_blank'>
              PBRM1 directs PBAF to pericentromeres and protects centromere integrity
              </a>
            </td>
          <td>
            Karen A. Lane, Alison Harrod, Lillian Wu, T. I. Roumeliotis, Hugang Feng, Shane Foo, Katheryn A. G. Begg, Federica Schiavoni, Noa Amin, Frank T Zenke, Alan A. Melcher, J. Choudhary, Jessica A Downs
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Dysbiosis of intratumoral fungal and bacterial communities is associated with poor prognosis in various cancers. However, the mycobiome characteristics in laryngeal squamous cell carcinoma (LSCC) and its correlation with recurrence have not yet been investigated. The mycobiome in 80 LSCC samples was characterized using internal transcribed spacer sequencing, encompassing both tumor tissues and their matched para-cancerous tissues. The intratumoral bacteriome was further identified using 16S rRNA sequencing. These two microbial communities were analyzed using bioinformatics and statistical methods to determine its potential correlation with LSCC recurrence. The fungal alpha diversity in tumors was higher compared with that in para-cancerous tissues (P < 0.001). A significant difference in the overall fungal community patterns between tumor tissues and para-cancerous tissues was observed based on Bray-Curtis dissimilarity (P < 0.001). The presence of Alloprevotella, Porphyromonas, Candida, and Fusarium in tumors exhibited a correlation with alcohol consumption. The relative abundance of Penicillium, Exophiala, and Aspergillus in the mycobiome, as well as that of Alloprevotella, Porphyromonas, and Peptostreptococcus in the bacteriome significantly increased the risk of LSCC recurrence (P < 0.05). These six microorganisms can combine to form a microbial complex, which may independently contribute to recurrence risk in patients with LSCC when enriched within the tumor (hazard ratio = 6.844, P < 0.01). Intratumoral fungi and bacteria can be valuable indicators for assessing recurrence in patients with LSCC, indicating their potential as valuable targets for therapeutic intervention. IMPORTANCE Our results revealed that dysbiosis of intratumoral microbiota, including increased fungal community diversity and overgrowth of several fungal or bacterial organisms, is substantially linked to the recurrence of LSCC. Drinking habits might alter the laryngeal microbiota to influence the recurrence of LSCC. We also explored a method to potentially predict the recurrence of LSCC from a novel perspective. These findings could offer insights into the etiology of LSCC and pave way to prevent and treat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc011a9eb62b9ac278bb5cc6ef5e19d7054ea20" target='_blank'>
              Complex of intratumoral mycobiome and bacteriome predicts the recurrence of laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xinhui Mao, Hui-ying Huang, Limin Zhao, Feiran Li, Zhenwei Wang, Xiaohui Yuan, Hui-Ching Lau, Chi-Yao Hsueh, Ming Zhang
          </td>
          <td>2025-02-21</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb646ade7879700ef0b4267a1e8d7990cd0524fa" target='_blank'>
              Abstract IA018: The evolutionary origins of cancer lethality
              </a>
            </td>
          <td>
            P. Boutros
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>