<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-19 09:44:23 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>47</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>91</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>125</td>
          <td>147</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>404</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>343</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D.
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, Jessica Alsing, J. Luebeck, Maria E. Stefanova, Owen S Chapman, Katerina Kraft, Shu Zhang, Jun Yi Stanley Lim, Yipeng Xie, Yoon Jung Kim, Sihan Wu, Lukas Chavez, Guy Nir, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, , J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract In this Perspective article, we follow the journey of a gene that breaks free from its stringent chromosomal transmission dynamics to become an independently multiplying agent on so-called extrachromosomal circular DNA (ecDNA/eccDNA). We discuss how the release of a gene from its chromosomal anchor can affect its dosage, regulatory context, and potential evolutionary trajectory before examining the phenotypic implications for unicellular and multicellular eukaryotes that transmit genes on circular DNA. We also briefly explore the fundamental difference between circularized genes in flowering plants (angiosperms) and mammals (primarily cancer) concerning somatic and germline inheritance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2 The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="Abstract CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db26ccc6dc7e76ac6a63d600396d97f0ca6b89f2" target='_blank'>
              Passenger mutations link cellular origin and transcriptional identity in human lung adenocarcinomas
              </a>
            </td>
          <td>
            Sukanya Panja, Padmaja Mantri, K. E. Johnson, J. Andrade-Martínez, S. Yang, Aditya Deshpande, Huasong Tian, Shaham Beg, Kentaro Ohara, Alessandro Leal, Joel Rosiene, Marlon Stoeckius, Peter Smibert, William D. Travis, J. Mosquera, P. Polak, Marcin Imieliński
          </td>
          <td>2025-11-27</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent mini-chromatin. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template is mediated by decreased efficacy of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS with LANA as similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA TR binding establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memories from prior activation, determines the degree of KSHV lytic replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43595752cf1e93615641d0b346a78e9c25448ab3" target='_blank'>
              KSHV TR deletion episomes uncover enhancer–promoter dynamics in gene regulation
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, J. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-12-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Simple Summary This study investigates the genomic alterations of cleavage and polyadenylation factors and their clinical relevance in cancer. We show that mutations in CPA genes may not be clinically relevant. However, copy number variations, particularly amplification of CPSF1, demonstrate significant impact on patient prognosis. We show that CPSF1 amplification, although co-occurring with MYC amplification, can independently change alternative polyadenylation (APA) patterns and dysregulate cancer-associated gene signatures. These results highlight the crucial impact of CPA gene copy number variation on biological processes and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common form of primary liver cancer and has limited therapeutic options. Epigenetic dysregulation plays a crucial role in hepatocarcinogenesis; however, its impact on cis-regulatory elements (CREs) and retrotransposons remains underexplored. Here, we investigated the epigenetic changes underlying the aberrant CRE and retrotransposon activity in HCC. We show that focal DNA hypomethylation of these elements is associated with transcriptional reprogramming. Notably, we uncover a dual regulatory mechanism for GPC3, a key diagnostic biomarker and immunotherapeutic target in HCC. This requires the concomitant reactivation of a fetal liver super-enhancer (SE) and DNA hypomethylation of CpG islands. Furthermore, DNA methylation loss drives cryptic activation of retrotransposons, some of which exhibit prognostic potential. Intriguingly, we identify a HERVE-int-derived long non-coding RNA that shows higher expression in patients with more aggressive tumors, poorer disease outcomes, and is correlated with molecular signatures associated with improved response to immunotherapy. Collectively, our findings reveal widespread epigenomic dysregulation of CREs and retrotransposons in HCC, highlighting new potential therapeutic strategies and biomarkers. Integrative epigenomic profiling of hepatocellular carcinoma reveals dual regulation of the immunotherapeutic target GPC3 and identifies retrotransposons as potential biomarkers for prognosis and immunotherapy response prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed2d6556ed99cbdaa0f49af5ea87b73a7cb6583" target='_blank'>
              Multi-omic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potential
              </a>
            </td>
          <td>
            Clooney C Y Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y J Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B S Lai, K. Yip, Alfred S Cheng, Danny Leung
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Copy number variants at genomic loci evolve at a high rate, are linked to many different diseases, and play a role in adaptive evolution in humans and other organisms. Here we show that stickleback fish from freshwater environments have rapidly and repeatedly evolved an expanded number of copies of a gene family involved in muscle development, Myosin Heavy Chain 3 Cluster C (MYH3C), compared to marine populations. Differences in copy number between marine and freshwater fish are maintained even in the presence of gene flow, suggesting that MYH3C changes represent adaptive divergence between ecotypes. Copy number expansion occurs by tandem duplication of MYH3C coding and regulatory regions on the stickleback sex chromosome. We identify a muscle regulatory enhancer within the expanded MYH3C region and show that elevated copy number is associated with developmental and tissue-specific increases in corresponding mRNA expression levels in skeletal muscle. Common MYH3C clusters include 3-, 4-, 5-, and 6-copy variants that likely evolved through a combination of microhomology-mediated break repair and non-allelic homologous recombination. Our results provide a new example of copy number changes in a wild species and identify CNVs as potential “hotspots” of repeated adaptive evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f303c1abd69915406b352bedd1f516532851ab" target='_blank'>
              Rapid and repeated evolution of myosin copy number in threespine stickleback
              </a>
            </td>
          <td>
            Alyssa M. Yoxsimer, Rhea R. Daugherty, Emily E. Hare, Y. F. Chan, Felicity C. Jones, Garrett A. Roberts Kingman, Emma G. Offenberg, Timothy R. Howes, Haili Zhang, Alex A. Pollen, S. Brady, Kathleen T. Xie, Heidi I. Chen, Craig B. Lowe, Eric H. Au, J. Grimwood, Jeremy Schmutz, Richard M. Myers, D. Schluter, D. Heins, Miguel L. Reyes, John A. Baker, Bjarni Jónsson, T. Reimchen, Michael A. Bell, David M. Kingsley
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Structural variations in cancer predisposition genes significantly contribute to the pool of pathogenic variants underlying heritable cancer susceptibility. In this report, we describe and functionally characterize a novel transposon-mediated germline pathogenic insertion identified within the coding region of the BRCA1 gene. The inserted sequence comprises the entire processed transcript of RPL18A, a ribosomal protein-coding gene, and may have been generated through a target-primed reverse transcription event. Robust molecular testing confirmed the heritability of the variant and provided evidence of its correlation with the observed phenotype. This is the first documented case of a germline cancer susceptibility variant arising through this unique mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab6faf5664bd66e2e317ebc80d8ac230de46d4a" target='_blank'>
              Processed Transcript Insertion as a Novel Germline Mutational Mechanism in BRCA1-Associated Hereditary Breast Cancer
              </a>
            </td>
          <td>
            A. Bozsik, H. Butz, V. Grolmusz, Petra Nagy, T. Pócza, Erika Tóth, E. Csernák, Attila Patócs, János Papp
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277dcab08835ae1612852095fa2fa4e5142ac6cf" target='_blank'>
              Proximity of pre-mRNA 3′ end processing and transcription termination predicts enhanced gene expression
              </a>
            </td>
          <td>
            Agata Stępień, Deepshika Pulimamidi, Martyna Plens-Gałąska, Maxime Lalonde, Magda Kopczyńska, Hiroshi Kimura, S. Irshad, M. Gdula, Stephan Hamperl, Kinga Kamieniarz-Gdula
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte‐rich malignant tumours with few somatic mutations in protein‐coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte‐rich tissues, and then performed genome‐wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis‐regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour‐specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D‐loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour‐specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome‐wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Double-strand breaks (DSBs) pose a significant threat to chromosome stability and, if left unrepaired, can result in chromosome rearrangements. Canonical DNA repair pathways mitigate these risks. However, if these repair mechanisms fail to repair the DSB, alternative repair pathways, such as break-induced replication, single-strand annealing, and de novo telomere addition (dnTA), can be utilized. Yeast subtelomeric regions are hotspots of recombination, while interstitial telomere-like sites can promote dnTA. In yeast, dnTA sites, termed SiRTAs (sites of repair-associated telomere addition), require Cdc13 association. We identified the ubiquitin protease Ubp10 as a positive regulator of dnTA at SiRTAs. Loss of UBP10 reduces dnTA frequency but increases the frequency of other chromosomal rearrangements at SiRTAs. SiRTAs utilize the repetitive subtelomeric regions of donor chromosomes to facilitate rearrangements, with a fraction occurring independently of RAD51 and requiring Sir4 and Sir2 components of the SIR complex. A DNA sequence capable of binding Cdc13 is necessary and sufficient to stimulate translocations in the absence of UBP10. This study highlights the diversity of DNA repair mechanisms at SiRTAs, advancing our understanding of telomere maintenance and chromosomal rearrangement formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a39def95be817353ddcb804972d341e87042282" target='_blank'>
              The ubiquitin protease Ubp10 suppresses the formation of translocations at interstitial telomere-like sites
              </a>
            </td>
          <td>
            David I. Gonzalez, Allison R. Westerbeek, Esther A. Epum, K. Friedman
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Large-scale genomic rearrangements in cancer cells can lead to the creation of fusion genes which alter the expression or function of the partner genes. Some fusion genes act as tumour drivers and the use of kinase inhibitors in patients with specific fusions have led to breakthroughs in cancer therapy. The large clinical and scientific interest in fusions has led to the development of software that use RNA sequencing data to identify fusion transcripts. Unfortunately, fusion transcript callers output many predictions which lack underlying genomic rearrangements and large datasets with validated fusions are scarce. This paper and the accompanying Fusions4U web application present a resource of validated and annotated gene fusions for 328 cell lines from the Cancer Cell Line Encyclopedia. Predicted fusion transcripts from Arriba and STAR-Fusion were analysed with our published validation pipeline that uses matched whole-genome sequencing data to identify discordantly mapped read pairs and candidate genomic breakpoints that support genuine fusion events. This resulted in 8,753 and 2,244 validated fusion transcripts for Arriba and STAR-Fusion predictions, respectively, with 1,596 fusions common to both. Additional layers of annotation include alternative splicing of fusion transcripts, kinases, microRNA host genes, genes in the COSMIC Cancer Gene Census, as well as known fusion gene pairs from the Mitelman Database of Chromosome Aberrations and Gene Fusions in Cancer and the TumorFusions dataset. We furthermore analysed information about fusion genes together with cell line data from the PRISM drug repurposing screening as an example of how this dataset can be used. This resource can be used to design experiments for functional studies and drug development, alone or in combination with publicly available information for the Cancer Cell Line Encyclopedia cell lines. The large collection of validated fusion transcripts with candidate genomic breakpoints can also be used in development and evaluation of bioinformatic tools for fusion transcript detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1751293449b5956386c8f2f0dbda4aebaafaddec" target='_blank'>
              Fusions4U: a resource of validated and annotated gene fusions in 328 cancer cell lines
              </a>
            </td>
          <td>
            Arianna Alamshahi, Helena Persson
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfde1f929fb1d5e109604b5adc00a4ea625430b8" target='_blank'>
              Localised activity of reverse gyrase at gene regulatory elements
              </a>
            </td>
          <td>
            Paul Villain, Vladislav Kuzin, Augustin Darennes-Degaugue, Florence Lorieux, Antoine Hocher, Romain Le Bars, Tobias Warnecke, Laura Baranello, Tamara Basta
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The dynamic three-dimensional (3D) organization of the human genome (the 4D nucleome) is linked to genome function. Here we describe efforts by the 4D Nucleome Project1 to map and analyse the 4D nucleome in widely used H1 human embryonic stem cells and immortalized fibroblasts (HFFc6). We produced and integrated diverse genomic datasets of the 4D nucleome, each contributing unique observations, which enabled us to assemble extensive catalogues of more than 140,000 looping interactions per cell type, to generate detailed classifications and annotations of chromosomal domain types and their subnuclear positions, and to obtain single-cell 3D models of the nuclear environment of all genes including their long-range interactions with distal elements. Through extensive benchmarking, we describe the unique strengths of different genomic assays for studying the 4D nucleome, providing guidelines for future studies. Three-dimensional models of population-based and individual cell-to-cell variation in genome structure showed connections between chromosome folding, nuclear organization, chromatin looping, gene transcription and DNA replication. Finally, we demonstrate the use of computational methods to predict genome folding from DNA sequence, which will facilitate the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function. The 4D Nucleome Project demonstrates the use of genomic assays and computational methods to measure genome folding and then predict genomic structure from DNA sequence, facilitating the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Summary SISTEM is a software package and mathematical framework for simulating tumor evolution and cell migrations at single-cell resolution. Unlike existing frameworks which simulate cancer cell populations under the neutral coalescent or using simple birth–death models, SISTEM simulates tumor populations under somatic clonal selection using an agent-based framework. SISTEM can generate mutation profiles, read counts, and DNA sequencing reads along with ground truth cell lineages and migration graphs under a number of easily customizable mutation and selection models. For improved realism, SISTEM allows for cell fitness to be driven by genomic events of various scales including single nucleotide variants, segmental gains and losses, whole-chromosomal and chromosome-arm aberrations, and whole-genome duplications. SISTEM also includes numerous migration models to simulate metastatic cancers, facilitating the exploration and evaluation of diverse migration patterns. Availability and implementation SISTEM is written in Python and is freely available open-source under GNU GPLv3 from: https://github.com/samsonweiner/sistem">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6ea45c6bd4d361ab9e67581ccc6b02aecffc37" target='_blank'>
              SISTEM: simulation of tumor evolution, metastasis, and DNA-seq data under genotype-driven selection
              </a>
            </td>
          <td>
            Samson Weiner, M. Bansal
          </td>
          <td>2025-11-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background/Objectives To address the unelucidated mechanisms of breakpoint formation in TFE3-rearranged renal cell carcinoma (TFE3-rRCC), this study characterizes breakpoint distribution within the TFE3 gene. We further explore how non-canonical DNA structures and their thermodynamic stability fluctuation may act as predisposing factors for the genomic instability driving these characteristic translocations. Methods TFE3 breakpoints were identified in a cohort of 31 TFE3-rRCC tumor samples. The chi-square test was used to assess the statistical significance of breakpoint clustering. To investigate potential structural determinants, we predicted the distribution of G-quadruplex-forming sequences and palindromic motifs. Moving beyond simple motif density, we calculated the local Gibbs free energy changes (ΔG) associated with DNA secondary structures using Mfold and RNAfold to model thermodynamic stability across the TFE3 gene. This thermodynamic stability fluctuation was quantified as the maximum absolute local change in folding free energy (|dΔG|). Finally, this correlation between thermodynamic stability fluctuation and breakpoint location was validated by analyzing the 13 most frequently rearranged genes reported in the COSMIC database. Results A significant breakpoint cluster was identified within intron 5 of TFE3, containing 23 of 31 breakpoints (74.19%; chi-square test, P < 0.05). While the simple density of G-quadruplex or palindromic motifs did not directly correlate with breakpoint locations, a strong association with local thermodynamic stability fluctuation was observed. The region within intron 5 exhibited the highest thermodynamic stability fluctuation. This result suggests that regions of high thermodynamic stability fluctuation are correlated with increased susceptibility to DNA breakage. This finding was corroborated in the COSMIC dataset, where breakpoints in 12 of the 13 most frequently rearranged genes were similarly located near peaks of high |dΔG|. Conclusion Our findings indicate that breakpoint events in TFE3-rRCC are non-randomly clustered within intron 5. This clustering correlates strongly with regions characterized by high thermodynamic stability fluctuation (|dΔG|) of potential non-canonical DNA secondary structures. The principle that elevated local thermodynamic stability fluctuation is a feature of breakpoint locations was supported by analysis of a broader set of oncogenes, suggesting that high local thermodynamic stability fluctuation is a common feature of translocation-prone regions in cancer, representing a plausible, though not proven, contributor to genomic fragility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba18c1a28435183414957442b6594dd9961a8358" target='_blank'>
              Mechanistic study of TFE3 breakage in TFE3-rearranged renal cell carcinoma: the perspective of non-canonical DNA structures and their stability
              </a>
            </td>
          <td>
            Xiaopo Zhang, Qinshuang Dang, Xinghe Pan, Zhenggen Deng, Yanhao Xu, W. Gan, Hongqian Guo
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The ciliate Oxytricha undergoes massive genome rearrangements during development to produce a functional nucleus from an encrypted zygotic genome. PIWI-interacting small RNAs protect DNA regions against deletion, but how that protective mark is established has been a mystery. Recently our lab discovered MTA1, a methyltransferase that catalyzes DNA N6-adenine (6mA) methylation. Both MTA1 and the Oxytricha Piwi protein, Otiwi1, are required for development, and Otiwi1 mutation eliminates 6mA signal. To examine the role of 6mA, we analyzed its genome-wide distribution across development in wild-type and MTA1 mutant backcrossed cells. We find specific and abundant enrichment on retained sequences, suggesting a protective role for this epigenetic mark. Furthermore, programmed retention of a DNA region that is normally deleted leads to accumulation of new 6mA marks on the ectopically retained DNA sequence. Together, these results suggest that piRNA-guided 6mA DNA methylation leads to protection of DNA sequences against deletion during nuclear differentiation. Highlights DNA N6-methyladenine accumulates on retained DNA regions during Oxytricha development. mta1 mutant backcrosses have disrupted DNA methylation and a developmental delay. DNA N6-adenine methylation during genome rearrangement requires the presence of Otiwi1. Programmed retention of a germline-limited region leads to developmental methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecfb36caac2df07b165f21a1abfc41a4ec2bd5d" target='_blank'>
              A PIWI protein-dependent DNA N6-adenine methylation pathway in Oxytricha protects genomic sequences from deletion
              </a>
            </td>
          <td>
            Margarita T. Angelova, Yi Feng, D. Villano, Erhan Aslan, , P. Esteve, Sagnik Sen, William E. Jack, S. Pradhan, L. Landweber
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, C. Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs) are repetitive DNA sequences typically silenced in normal tissues. Their dysregulation in cancer can significantly impact oncogene activation and tumorigenesis. However, due to their repetitive nature, TEs are often excluded from gene-centric single-cell analyses. Although recent advances have enabled single-cell TE quantification, a systematic resource for exploring single-cell TE dynamics in cancer has remained lacking. To address this, we developed TE-SCALE (https://ngdc.cncb.ac.cn/te-scale/), a single-cell database for integrative analysis and visualization of TE expression across human cancers. It incorporates over 1.3 million cells from 330 samples across 20 cancer types and 12 tissue origins. Built on our in-house pipeline scTEfinder, TE-SCALE provides a comprehensive pan-cancer TE expression atlas, enabling multi-scale exploration from tissue to cell population, supported by well-curated TE annotations. The platform offers three key analytical modules: differential TE expression, TE-gene co-expression network, and functional enrichment analysis. A user-friendly web interface supports flexible browsing, searching, analysis, and data download. Notably, TE-SCALE identifies tumor-specific TEs preferentially expressed in particular cancer types or disease states, underscoring their potential as biomarkers for diagnosis, monitoring, and immunotherapeutic targeting. Collectively, TE-SCALE provides an essential resource for decoding TE biology in cancer and expedites its translation into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a20c719f57adedc7ed801a6aba2dc3cd3ae0ec" target='_blank'>
              TE-SCALE: a comprehensive database for exploring transposable element expression across human cancers at single-cell resolution
              </a>
            </td>
          <td>
            Xini Meng, Zhi Nie, Qifei Wang, Yiwen Hu, Yulan Deng, Na Ai, Zheng Huang, Yun Li, Yang Yuan, Jingfa Xiao, Jingyao Zeng, Guochao Li, Lan Jiang
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, often arising from mutations to DNA damage repair and response (DDR) genes. Classical genetic, biochemical, and structural approaches elucidated the foundational mechanisms of DDR pathways and provided a scientific understanding of their involvement in repair of lesions induced by broad classes of DNA-damaging agents (DDAs). However, given the chemical diversity of DDAs and resultant DNA lesions, along with the multitude of interconnected DDR factors, the chemogenomic landscape of DDA–DDR interactions remains incompletely mapped. To this end, we developed a DDR-targeted, CRISPR knockout screening approach and assessed relationships amongst 353 DNA repair genes and 15 DDAs in LN229 glioma cells. Within this dataset of 5295 DDR-related chemogenomic interactions, we identified many established interactions and discovered novel ones. For example, we observed a specific role of transcription-coupled nucleotide excision repair in the repair of adducts generated by monofunctional alkylating agents, a role for the Fanconi anemia pathway in addressing methyl lesions, overt differences in DSB repair following treatment with topoisomerase I versus II poisons, and repair dependencies associated with the imidazotetrazines temozolomide, mitozolomide, and KL-50. Future directions will continue to investigate the mechanisms of novel chemogenomic interactions that we have uncovered as well as work to identify chemogenomic interactions amenable to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018770119a2496ee321bc85a8d0504634bbf54c9" target='_blank'>
              Targeted CRISPR knockout screening identifies known and novel chemogenomic interactions between DNA damaging agents and DNA repair genes
              </a>
            </td>
          <td>
            Collin D. Heer, James L. Elia, Vijay Menon, Spenser S Johnson, Sofia R. Arbelaez, Sam Friedman, Francesc Lopez-Giraldez, R. Sundaram, Seth B Herzon, R. Bindra, Susan E. Gueble
          </td>
          <td>2026-01-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a1d0df44d9ce5038b1336c99cf98a4e708b134" target='_blank'>
              The alternative lengthening of telomeres pathway through a DNA repair lens: mechanism and therapeutic opportunities.
              </a>
            </td>
          <td>
            D. Muoio, Elise Fouquerel
          </td>
          <td>2025-12-01</td>
          <td>NAR cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Rationale: Neuroblastoma (NB) is a predominant extra-cranial malignancy in childhood, while molecular drivers of its progression and effective treatment strategies have yet to be clarified. Methods: RNA sequencing was performed to identify transcriptional regulators and corresponding target genes. To explore the biological effects and underlying mechanisms of these regulators, a comprehensive methodology was utilized, encompassing chromatin immunoprecipitation, dual-luciferase reporter assay, qRT-PCR, western blot, alongside gene over-expression and silencing techniques, co-immunoprecipitation, and mass spectrometry. The MTT assay, soft agar colony formation, Matrigel invasion, and nude mouse xenograft models were applied to assess oncogenic properties. Patient survival was analyzed using the log-rank test. Results: Armadillo repeat containing 12 (ARMC12) was identified as a MYC-interacting modulator within liquid condensates to up-regulate critical nucleoporin-encoding targets (NUP62/NUP93/NUP98), which promoted nuclear pore complex (NPC) biogenesis to facilitate nuclear trafficking of oncogenic effectors, thereby enhancing invasion and metastasis of NB. As a protein within extracellular vesicles of Malassezia globosa colonizing NB tissues, MGL_0381 also facilitated MYC transactivation via physical interaction to accelerate NPC biogenesis and NB progression. Tioconazole (TCZ) and UU-T02 were identified as efficient inhibitors blocking ARMC12-MYC and MGL_0381-MYC interaction, and synergistically reduced NPC number and aggressive features of NB. High ARMC12, MYC, NUP62, NUP93, or NUP98 levels served as markers of unfavorable patient outcomes in clinical cohorts. Conclusions: These findings collectively demonstrate that dual targeting of AMRC12 and Malassezia globosa disrupts MYC liquid condensates-driven NPC biogenesis during NB progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d399122edcdb055994dc03e897307f03c8f832ff" target='_blank'>
              Dual targeting of AMRC12 and Malassezia globosa disrupts MYC liquid condensates-driven nuclear pore complex biogenesis in neuroblastoma
              </a>
            </td>
          <td>
            A. Hu, Chunhui Yang, Zhijie Wang, Xiaolin Wang, Xinyue Li, Shunchen Zhou, Bosen Zhao, Jiaying Qu, Xiaojing Wang, Liduan Zheng, Q. Tong
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Achieving precise and efficient integration of gene-sized DNA sequences into the human genome remains a major obstacle to gene therapy. Existing approaches depend on double-strand DNA breaks, which can lead to unintended genome alterations. Many monogenic diseases arise from diverse patient-specific mutations, making individualized correction impractical and underscoring the need for universal full-gene replacement strategies. We developed INsertion by Targeted Anchoring and Conditional Transposition (INTACT) to enable targeted insertion at genomic safe harbor loci. We engineered a mammalian transposase with mutations in its DNA-binding domain to reduce off-target integration. Site specificity was then restored by linking programmable sequence-specific DNA-binding proteins to the transposase. Systematic optimization of INTACT revealed key determinants of precision, including noncovalent linkage between the transposase and DNA-binding protein, strict spacing between the binding site and the TTAA insertion sequence, and linkage of the DNA-binding protein to an internal position within the transposase. On-target insertion was achieved across multiple loci, with optimized INTACT averaging 1.2 targeted insertions per cell. An off-target assay confirmed that DNA-binding domain mutations substantially reduced unwanted integration events to near-background levels. Our site-directed transposase enables precise, efficient genomic insertion of >4 kb DNA without double-strand breaks, offering a powerful new tool for genome engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dab1f9e5fe9c69a3e2717cbaf0dfc3ffa7c602b3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E. Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T. Tran, Clara Errard, Brian E. Hew, Brandon E. Johnson, David F. Waller, Shanfu Xie, Joseph J. Higgins, Raymond Tabibiazar, Jesse B. Owens
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="53BP1 accumulates at DNA double strand breaks (DSBs) and is implicated in non-homologous end joining (NHEJ), but the genetic interplay of 53BP1 with the NHEJ pathway (e.g., DNA-PKcs) is poorly understood. We examine blunt DSB NHEJ of Cas9 DSBs, which is dependent on core NHEJ factors, and find that loss of 53BP1 does not affect such repair but causes a reduction when combined with DNA-PKcs disruption. In contrast, disrupting 53BP1 and DNA-PKcs, alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We find similar effects with RIF1, such that 53BP1-RIF1 appear to play a backup role for DNA-PKcs during blunt DSB NHEJ, but function in the same pathway to suppress microhomology deletions. Finally, DNA-PKcs kinase inhibition causes increased radiosensitivity and homology-directed repair that is not additive with loss of 53BP1. Altogether, 53BP1-RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes. DNA double strand breaks can be repaired by several pathways leading to different genetic outcomes. Here, the authors define the interplay between the DNA damage response factors 53BP1-RIF1 and core non-homologous end joining factors on diverse repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6dc22c93555e7812dfc2b85eba7a6ca2131899" target='_blank'>
              53BP1-RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40038069867c0720d29003167b0971829e037042" target='_blank'>
              Nanopore whole-genome sequencing reveals conserved chromosome-specific telomere architecture across tissues and populations
              </a>
            </td>
          <td>
            Niklas L. Engel, B. Brors, L. Feuerbach, Peter J. Park
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a25c976684f63347d1fe095c70af1dbd2cc6459" target='_blank'>
              A molecular framework of chromatin extrusion in plants
              </a>
            </td>
          <td>
            Sofia Tzourtzou, Zhidan Wang, Sarah Leuchtenberg, D. Latrasse, M. Benhamed, Chang Liu
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Transposable elements (TEs) are indispensable components of eukaryotic genomes, mechanistically linked to carcinogenesis, aging and other degenerative diseases. The ability of TEs to self-propagate and cause deletions, inversions or insertions within the genome poses a real threat to the fidelity of genomic integrity. This review discusses the fundamental properties of TEs, with a focus on cellular interactions associated with mechanisms involved in recombination, replication, and DNA repair. Since mobilization of TEs induces double-strand breaks (DSBs), faulty repair mechanisms could lead to cellular dysfunction, pathology and death. The TE-induced DNA DSB repair cascade follows either homologous recombination (HR) or non-homologous end-joining (NHEJ) pathways. Importantly, epigenetic regulatory mechanisms including DNA methylation and histone acetylation provide additional control in ensuring accurate DNA repair and could prove to be key targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65023eda36836bf677b82b04598ca5b47b85bce0" target='_blank'>
              Impact of Transposable Elements on DNA Double-Strand Break Repair and Genomic Stability.
              </a>
            </td>
          <td>
            Mohd Iqbal Bhat, R. Pandita, Arjamand Mushtaq, U. S. Mir, Najumu Saqib, Parthas S. Sarkar, Audesh Bhat, Kenneth S. Ramos, Tej K. Pandita, Mohammad Altaf
          </td>
          <td>2026-01-11</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Low-pass whole-genome sequencing (LP-WGS) of circulating tumor DNA (ctDNA) is increasingly recognized for its utility in identifying somatic copy-number aberration (CNA). In this study, we analyzed LP-WGS ctDNA data from 73 pediatric patients with neuroblastic tumor and 11 healthy controls to explore diagnostic value of ctDNA CNA burden (including the genotypings) with a customized bioinformatics workflow. We found that a high baseline ctDNA CNA burden [tumor DNA fraction (TFx) ≥0.2%] was present in 36 of 41 patients (87.80%) with neuroblastoma, six of 22 patients (27.27%) with ganglioneuroblastoma, and three of 10 patients (30%) with ganglioneuroma. High baseline ctDNA CNA burden could predict high-risk neuroblastic tumors with an area under curve (AUC) of 0.95, sensitivity of 94.12%, and specificity of 100%. Frequent chromosomal copy-number changes, including chr17q gain, chr7 gain, chr3p loss, and chr11q loss, were found in ctDNA. Gain of chr17q demonstrated the highest diagnostic value with an AUC of 0.92, indicating strong sensitivity and specificity for detecting high-risk neuroblastic tumors. The homologous recombination deficiency score in the high- and intermediate-risk groups was significantly elevated compared with those in the low-/very low–risk group. The TFx levels and segmental alterations significantly decreased in patients with neuroblastic tumor who underwent chemotherapy, from median TFx = 13.82% before treatment to 0.24% after treatment (P < 0.0001). Our findings highlight the effectiveness of LP-WGS ctDNA CNA analysis as a promising approach for diagnosis and risk stratification of pediatric neuroblastic tumors and for monitoring chemotherapy response. Particularly, ctDNA analysis is minimally invasive, rapid, and cost-effective, which could bring additional benefits in pediatric practices. Significance: Our results support the development of ctDNA CNA analysis as a robust and minimally invasive approach for early detection, molecular diagnosis, and risk stratification of peripheral neuroblastic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01256a37f8920b62f2d81f1a7bbc4542d1a51191" target='_blank'>
              Copy-Number Aberrations in Circulating Tumor DNA Enable Diagnosis and Risk Stratification of Pediatric Neuroblastic Tumors
              </a>
            </td>
          <td>
            Ting Tao, Jiabin Cai, Yinbing Tang, Dong-Fang Lu, Lifeng Zhang, Jinkai Peng, Yilong Wang, Weizhong Gu, Shouhua Zhang, Jinhu Wang
          </td>
          <td>2025-12-03</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA:RNA hybrids are unusual structures found throughout the genomes of many species, including yeast and mammals. While DNA:RNA hybrids may promote various cellular functions, persistent hybrids lead to double strand breaks, resulting in genomic instability. DNA:RNA hybrid formation and removal are therefore highly regulated, including by enzymes that either degrade or unwind RNA from the hybrid. Meiosis is the specialized cell division that creates haploid gametes for sexual reproduction. Previous work in yeast and mammals showed that elimination of DNA:RNA hybrids by RNase H facilitates meiotic recombination. This work demonstrates that the conserved Sen1 DNA/RNA helicase functions during three temporally distinct processes during yeast meiosis. First, SEN1 allows meiosis-specific genes to be expressed at the proper time to allow entry into meiosis. Second, SEN1 prevents the accumulation of hybrids during premeiotic DNA replication. Third, SEN1 promotes the repair of programmed meiotic double strand breaks that are necessary to form crossovers between homologous chromosomes to allow their proper segregation at the first meiotic division. Given the evolutionary conservation of Sen1 with its mammalian counterpart, Senataxin, studies of Sen1 function in yeast are likely to be informative about the regulation of DNA:RNA hybrids during human meiosis as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8231ab0fa8d6701759423756235a545680a4b00a" target='_blank'>
              The conserved SEN1 DNA/RNA helicase has multiple functions during yeast meiosis
              </a>
            </td>
          <td>
            Robert Gaglione, Leonidas Pierrakeas, Lihong Wan, Jonathan Caradonna, A. MacQueen, Ed Luk, N. M. Hollingsworth
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2ceccde009f3d35721416ae6a0441554cd9fa0" target='_blank'>
              The mitotic stopwatch synergizes with mild p53 activation to halt cell proliferation
              </a>
            </td>
          <td>
            Beata E. Mierzwa, Franz Meitinger, A. Desai, K. Oegema
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c06b0dfa9d58dc239114dc3e3b926c74b839c6" target='_blank'>
              Genomic analysis of PLNTY-like tumor progression into epithelioid glioblastoma: a case report.
              </a>
            </td>
          <td>
            S. Mäntylä, Anssi Nurminen, Sanna Huovinen, S. Jaatinen, T. Haapaniemi, R. Nurminen, Ismaïl Hermelo, Stefanie Volz, K. Maass, K. Pajtler, K. Nordfors, H. Haapasalo, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2026-01-08</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0bf2ebc24868e52a4bf9e9e5a2567d97f5b5a0" target='_blank'>
              Integrated proteogenomics uncovers ancestry-specific and shared molecular drivers in localized prostate cancer
              </a>
            </td>
          <td>
            Cara C. Schafer, T. Abulez, Xijun Zhang, Kun-Lin Ho, Jiji Jiang, D. Young, Jesse M. Fox, K. Conrads, B. Hood, G. Sukumar, Darryl Nousome, Praveen-Kumar Raj-Kumar, Mariano Russo, Ayesha A. Shafi, Xiaofeng A. Su, A. Dobi, Amina Ali, S. Elsamanoudi, J. Cullen, W. Figg, G. Petrovics, C. Dalgard, M. D. Wilkerson, N. Bateman, T. Conrads, I. Sesterhenn, Leigh Ellis, Craig D. Shriver, Gregory T. Chesnut, Shyh-Han Tan
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ab72daae241f6f66396542f56383a72c207669" target='_blank'>
              Rescue from pyridostatin-stabilized telomeric G-quadruplexes by DNA cleavage and microhomology-mediated end joining
              </a>
            </td>
          <td>
            Samah Matmati, Satyajeet Rao, Joachim Lingner
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The T‐cell Leukemia Homeobox 1 (TLX1) oncogene is frequently deregulated in T‐cell acute lymphoblastic leukemia (T‐ALL) and T‐cell lymphoblastic lymphoma (T‐LBL). In most instances, TLX1 overexpression results from its juxtaposition with a T‐cell receptor (TCR) locus caused by interchromosomal translocations. However, in the subset of non‐TCR‐translocated TLX1‐positive (non‐TCR TLX1+) T‐ALL cases, the underlying mechanisms driving TLX1 overexpression and their clinico‐biological implications remain unclear. By integrating high‐resolution data from next‐generation sequencing, fluorescence in situ hybridization, karyotyping, optical genome mapping, and long‐read whole‐genome sequencing across a cohort of 1122 adult and pediatric T‐ALL/T‐LBL cases, we identified TLX1 overexpression in 11% of T‐ALL/T‐LBL cases, with an unexpected 14% incidence of non‐TCR TLX1+ cases among TLX1+ cases. Non‐TCR TLX1 + T‐ALL cases show distinct clinico‐biological features, including a lower frequency of cortical phenotype compared to the TCR‐translocated TLX1+ (TCR TLX1+) subgroup, regardless of TLX1 expression levels. In non‐TCR TLX1⁺ cases, TLX1 deregulation results mostly (83%) from the hijacking of the ANKRD1 and PCGF5 enhancer at 10q23.31 through various chromosomal aberrations. Genomic analyses revealed that these aberrations juxtapose the ANKRD1 and PCGF5 enhancer with TLX1, driving its overexpression. Importantly, survival analysis revealed that non‐TCR TLX1+ patients had significantly poorer outcomes compared to their TCR TLX1⁺ counterparts (3y‐OS: 55% vs. 90%, P < 0.001 and 3y‐DFS: 45% vs. 75%, P = 0.02). These findings, consistent with our previous observations in TAL1‐driven T‐ALL, confirm that the clinico‐biological impact of an oncogene is influenced by the specific mechanism underlying its deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd726bac8a093d55c346f606656874776ef2007c" target='_blank'>
              TLX1 translocation variants and enhancer hijacking influence clinical outcome in T‐cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            E. Balducci, T. Steimlé, A. Cieslak, G. Charbonnier, A. Pinton, M. Simonin, Charlotte Smith, Valentine Louis, M. Féroul, Manon Delafoy, Sophie Kaltenbach, Audrey Bidet, Béatrice Grange, Lauren Rigollet, P. Villarese, A. Touzart, Hervé Dombret, André Baruchel, N. Boissel, Vahid Asnafi
          </td>
          <td>2025-12-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Manipulating the mitochondrial genome remains a major challenge in genetic engineering, largely due to the mitochondrial double-membrane structure. While recent advances have expanded the genetic toolkit for nuclear and cytoplasmic targets, precise editing of mtDNA has remained elusive. Here we report the first successful mitochondrial import of a functional RNA-guided prokaryotic Argonaute protein from the mesophilic bacterium Alteromonas macleodii (AmAgo). By directing AmAgo to the single-stranded D- or R-loop region of mtDNA using synthetic RNA guides, we observed a nearly threefold reduction in mtDNA copy number in human cell lines. This proof of concept study demonstrates that a bacterial Argonaute can remain active within the mitochondrial environment and influence mtDNA levels. These findings lay the groundwork for further development of programmable systems for mitochondrial genome manipulation. Highlights AmAgo targeted to mitochondria significantly reduces human mtDNA copy number Mitochondrial localization of AmAgo is essential for changes in the mtDNA content AmAgo effects on mtDNA are modulated by exogenous guide RNAs Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c5e33a1532b92b0810287a27ea33b2ef13bfc31" target='_blank'>
              Targeting of human mitochondrial DNA with programmable pAgo nuclease
              </a>
            </td>
          <td>
            B. Rimskaya, E. Kropocheva, Elza Shchukina, Egor Ulashchik, Daria Gelfenbein, L. Lisitskaya, Vadim Shmanai, Svetlana Smirnikhina, A. Kulbachinskiy, I. Mazunin
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, Chongyang Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC.
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Double-stranded DNA ends arise from external agents or cellular processes like transcription–replication collisions (TRCs), threatening genome stability. Here, we performed genomic CRISPRi screens to uncover DNA end formation factors in Escherichia coli. We discovered that translation–transcription decoupling causes DNA end formation through a TRC-dependent pathway, which is lethal when DNA end processing by RecBCD is disrupted, but not when recombination is disrupted. We find that TRCs cause replisome stalling followed by “rear-ending” from trailing replisomes which generates free DNA ends, rather than strand breaks. Surprisingly, these DNA ends are resolved through a process we call “replication reset”, where the stalled replicore is degraded, without triggering recombination, the DNA damage response, or mutagenesis. This hidden replicore-degradation resets the replication cycle without consequence for the genome. This discovery reveals a novel DNA safeguard mechanism for preserving genome stability when replication is disturbed and challenges the notion that TRCs necessarily cause genome instability in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92eb459708e432dd34fa580ebf7ed6d57b5e4227" target='_blank'>
              Transcription–replication collisions trigger high-fidelity replication reset
              </a>
            </td>
          <td>
            Matthew B. Cooke, Kobie T Welch, Laura Deus Ramirez, Katelin M Hagstrom, Alice X. Wen, J. A. Halliday, Susan M. Rosenberg, Christophe Herman
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0453b8fd3ee789a5730a5cb95cb52509da58c9" target='_blank'>
              Unraveling miRNA-Driven DNA Damage Response Networks in Pancreatic Adenocarcinoma: A Multi-Omics and Machine Learning Approach
              </a>
            </td>
          <td>
            Shayori Bose, Paramita Basak Upama, S. Ahamed
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, with the most consequential mutations occurring in coding exons and regulatory regions. However, the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a mutational hotspot at transcription start sites (TSSs) in the human germline, spanning several hundred base pairs in both directions. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants, many of which are excluded from de novo mutation calls, indicating that the hotspot partly arises during early embryogenesis. We associate the TSS mutational hotspot with divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, suggesting a recombination-independent mechanism distinct from known processes. Our findings are reinforced by mutational signature analysis, which highlights alternative double-strand break repair and transcription-associated mutagenesis. These insights reveal a germline mutational phenomenon with evolutionary and biomedical implications, particularly affecting genes linked to cancer and developmental phenotypes. The impact of transcription on germline mutagenesis remains poorly understood. Here, the authors identify a mutational hotspot at transcription start sites in the human germline that is significantly enriched with early mosaic variants; they also analyse the molecular mechanisms that could cause it and explore its potential consequences for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de526e07a78fe794f2e553a66798f6d31a5db32" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fueled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, Vladimir Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The Golden Mussel,
 Limnoperna fortunei
 , represents one of the most successful aquatic invaders in South America, causing significant ecological and economic impacts. Repetitive DNA sequences, particularly satellite DNAs (satDNAs), play crucial roles in genome architecture and evolution, yet the satDNA landscape of this invasive species remains largely uncharacterized. This study aimed to comprehensively analyze the satellitome of
 L. fortunei
 using integrated computational and cytogenetic approaches.





 We employed a read-clustering approach (RepeatExplorer2) to identify satDNA families from short-read genomic data. The identified families were then mapped onto the chromosome-level reference genome
 in silico
 to determine their chromosomal distribution. The physical localization of two representative satDNA families with contrasting distribution patterns was validated through fluorescence
 in situ
 hybridization (FISH) on meiotic chromosomes.





 Our analysis identified 129 distinct satDNA families, which were estimated to comprise approximately 9.1% of the genome based on read clustering. Subsequent
 in silico
 mapping successfully localized 126 of these families to the reference genome, where they collectively represented approximately 5.3% of the assembled chromosomes. Most families showed low divergence levels (0%–5% Kimura substitutions), suggesting recent amplification events. While most satDNAs were distributed across all 15 chromosomes, FISH analysis of two distinct families revealed contrasting distribution patterns: LfoSat025 showed preferential localization to pericentromeric regions, while LfoSat004 exhibited localized enrichment in specific chromosomal regions, demonstrating diverse organizational strategies within the satellitome. Meiotic analysis revealed normal chromosome pairing (15 bivalents), with no evidence of differentiated sex chromosomes, consistent with the species’ predominantly gonochoristic reproductive mode.





 This study provides the first comprehensive characterization of the
 L. fortunei
 satellitome, revealing a diverse and recently amplified repetitive landscape. The discrepancy between the genome-wide abundance (9.1%) and the mapped abundance (5.3%) highlights the challenges of assembling repetitive regions and underscores the value of using complementary methodologies. The identification of chromosome-specific satDNA markers establishes a foundation for developing molecular tools for invasion monitoring, including population genetic analyses to trace invasion routes and identify source populations. These findings contribute to understanding the role of repetitive DNA in genome evolution and the adaptive success of invasive species.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecafaaf22baf704516b16401c19167437d585cf" target='_blank'>
              A diverse satellite DNA repertoire in Limnoperna fortunei: insights into genome structure and chromosomal organization
              </a>
            </td>
          <td>
            J. P. Castro, T. M. Degrandi, Allan Oliveira Leal, Augusto Luiz Ferreira Júnior, M. C. Almeida, R. Artoni
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="To delineate the genomic and cellular features of CCNE1 amplification across gynecologic cancer types using large-scale multi-omics data and assess its relevance for patient stratification and therapeutic targeting. We analyzed genomic and transcriptomic data from 12,845 patients with gynecological cancers curated from the cBioPortal database, including ovarian, endometrial, and cervical carcinomas. CCNE1 amplification rates, cooccurring genomic alterations, and pathway enrichment were evaluated. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic data from cervical, endometrial, and ovarian tumors were used to investigate CCNE1 expression patterns, cell type specificity, and microenvironmental interactions. CCNE1 amplification was detected in 8.40% of all patients, with the highest rates in patients with ovarian cancer (8.01%) and endometrial cancer (5.06%). Among the ovarian subtypes, carcinosarcoma showed the highest amplification frequency (12.46%), followed by high-grade serous carcinoma (9.84%). Amplified tumors are frequently co-altered by TP53, PIK3CA, and PI3K-AKT pathway genes. Single-cell analyses revealed that CCNE1 expression was largely confined to epithelial cells and was associated with proliferative and structural remodeling pathways, with limited immune engagement across tumor types. Spatial transcriptomics confirmed the localization of CCNE1 in epithelial-rich regions and its correlation with adhesion- and motility-related genes in clear cell and high-grade serous ovarian carcinomas. CCNE1 amplification is enriched in aggressive histologic subtypes of gynecological cancers and is linked to epithelial-specific expression and immune exclusion. These findings indicate that CCNE1 represents a potential molecular marker for tumor aggressiveness, warranting further investigation in the context of patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07c358e69444ecc527d46e34a80b43d34d202949" target='_blank'>
              Multiomic analysis of CCNE1 amplification associated molecular and immune features in gynecological cancers
              </a>
            </td>
          <td>
            Linlin Liu, Wenjin Zheng, Liang Wen, Zhe Zhang, Yuanguang Meng
          </td>
          <td>2025-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Brain cancers are the most common solid tumors in children and adolescents and remain the leading cause of disease-related death in these populations. Despite their clinical importance, therapeutic progress has been limited by extensive intra- and inter-tumor heterogeneity. In particular, the clonal architecture and evolutionary dynamics of pediatric brain tumors remain poorly understood. Addressing this knowledge gap is critical, as genetically distinct subclones can shape disease progression, treatment resistance, and clinical outcome. To investigate clonal evolution at the population level, we analyzed longitudinal whole-genome sequencing data from 1,742 tumors across 27 brain cancer types in 1,479 pediatric patients from the Children’s Brain Tumor Network. We inferred clonal composition and phylogenetic relationships for each patient and reconstructed the space of tumor phylogenies for the entire cohort using a regularized Gromov-Hausdorff distance. Tumor phylogenetic structure was significantly associated with mutational burden, mutational signatures, cancer classification, and patient's survival status. In longitudinal samples, survival status remained significantly associated with tumor clonal architecture after adjusting for mutational burden, tumor subtype, and anatomical location, underscoring the clinical value of this readout. In contrast, in primary tumors, mutational signatures remained significant after adjusting for mutation count, tumor subtype, anatomical location, and purity, while survival status associations were no longer significant. Mutational signature timing analysis revealed that pediatric brain tumor genetic architecture is shaped by distinct mutational processes acting at different stages of tumor evolution. Truncal somatic mutations were enriched for age-related, defective proofreading, and homologous recombination deficiency signatures, whereas subclonal mutations showed significant contributions from DNA mismatch repair, chemotherapy-related, temozolomide-related, and somatic hypermutation activity signatures. Comparisons of matched longitudinal and primary phylogenetic trees further revealed that deceased patients and those with recurrent or metastatic disease exhibited significantly greater evolutionary divergence, suggesting that primary tumors are less predictive of complete evolutionary trajectories in poor-outcome cases. Together, these findings suggest that pediatric brain tumor evolution is shaped by cancer type and various mutational processes, and that patients with adverse outcomes follow distinct evolutionary trajectories independently of subtype. Our study provides a population-level view of tumor evolution in pediatric brain cancers and highlights the value of genetic clonal dynamics for understanding tumor etiology and progression.



 Minh A. Nguyen, Pablo G. Cámara. Mapping clonal architecture and evolution in pediatric brain cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d01cdf935c0ff4316725768de123e5059b9b4ae1" target='_blank'>
              Abstract B008: Mapping clonal architecture and evolution in pediatric brain cancers
              </a>
            </td>
          <td>
            Minh A. Nguyen, Pablo G. Cámara
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Comprehensive genomic analysis is essential for advancing our understanding of human genetics and disease. However, short-read sequencing technologies are inherently limited in their ability to resolve highly repetitive, structurally complex, and low-mappability genomic regions, previously coined as “dark” regions. Long-read sequencing technologies, such as PacBio and Oxford Nanopore Technologies (ONT), offer improved resolution of these regions, yet they are not perfect. With the advent of the new Telomere-to-Telomere (T2T) CHM13 reference genome, exploring its effect on dark regions is prudent. In this study, we systematically analyze dark regions across four human genome references—HG19, HG38 (with and without alternate contigs), and CHM13—using both short- and long-read sequencing data. We found that dark regions increase as the reference becomes more complete, especially dark-by-MAPQ regions, but that long-read sequencing significantly reduces the number of dark regions in the genome, particularly within gene bodies. However, we identify potential alignment challenges in long-read data, such as centromeric regions. These findings highlight the importance of both reference genome selection and sequencing technology choice in achieving a truly comprehensive genomic analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0af39dc3f40d9c2bc549d4b7b8ccbbcdca891e97" target='_blank'>
              Dark and camouflaged genomic regions remain challenging in CHM13
              </a>
            </td>
          <td>
            Mark E. Wadsworth, Madeline L. Page, B. A. Heberle, Justin B. Miller, Cody J. Steely, M. Ebbert
          </td>
          <td>2026-01-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd31662e57a3a92ec93af3c5d7b5f977fa061eb3" target='_blank'>
              Designing synthetic regulatory elements using the generative AI framework DNA-Diffusion
              </a>
            </td>
          <td>
            Lucas Ferreira DaSilva, Simon Senan, Judith F. Kribelbauer-Swietek, Z. M. Patel, L. Louis, Aniketh Janardhan Reddy, Sameer Gabbita, Jonathan D. Rosen, Zach Nussbaum, César Miguel Valdez Córdova, Aaron Wenteler, Noah Weber, Tin M. Tunjic, Martino Mansoldo, Talha Ahmad Khan, Gue-Ho Hwang, V. Gardeux, David T. Humphreys, Cameron Smith, Matei Bejan, Peter Bromley, Will Connell, Bart Deplancke, Michael I Love, Emily S. Wong, Wouter Meuleman, L. Pinello
          </td>
          <td>2025-12-23</td>
          <td>Nature Genetics</td>
          <td>2</td>
          <td>21</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract DNA replication-coupled chromatin assembly is crucial to maintain genome integrity. Here, we demonstrate that the absence of the budding yeast histone chaperone CAF-1 induces the production of extrachromosomal ribosomal RNA gene (rDNA) circles (ERCs), accompanied by chromosomal rDNA copy number changes, in a manner dependent on Fob1-mediated DNA replication fork arrest in the rDNA, the homologous recombination (HR) protein Rad52, and its interaction with proliferating cell nuclear antigen. In the caf-1 mutant, ERC production is triggered partly by increased transcription from the regulatory promoter E-pro, but is also affected by defects independent of E-pro misregulation. Absence of CAF-1 accumulates resected DNA double-strand breaks (DSBs) formed at arrested replication forks, repair of which leads to enhanced ERC formation. CAF-1 deficiency causes partial defects in lagging strand synthesis coupled to nucleosome spacing in the rDNA. Our findings suggest that CAF-1 suppresses HR-mediated rDNA instability during repair of replication-coupled DSBs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d973d822929747e684db2962d721228aced82ad" target='_blank'>
              The histone chaperone CAF-1 prevents homologous recombination-mediated instability of the budding yeast ribosomal DNA during replication-coupled DNA double-strand break repair
              </a>
            </td>
          <td>
            Hajime Futami, Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mio Nagura, Takehiko Kobayashi, Mariko Sasaki
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, Gang Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Mutations in the global transcriptional activator EP300/KAT3B are being reported in aggressive malignancies. However, the mechanistic contribution of EP300 dysregulation to cancer is currently unknown. While EP300 has been implicated in regulating cell cycle and DNA replication, the role of EP300 in maintaining replication fork integrity has not been studied. Here, using EP300-mutated adult T-cell leukemia/lymphoma cells and an EP300-selective degrader, we reveal that EP300 loss leads to pronounced dysregulations in DNA replication dynamics and persistent genomic instability. Aberrant DNA replication in EP300-mutated cells is characterized by elevated replication origin firing due to replisome pausing. EP300 deficiency results in a prominent defect in fork protection resulting in the accumulation of single-stranded DNA gaps. Importantly, we find that the loss of EP300 results in decreased expression of BRCA2 protein leading to sensitivity to treatments that are cytotoxic to BRCA-deficient cancers. Overall, we demonstrate that EP300-mutated cells recapitulate features of BRCA-deficient cancers. Loss-of-function mutations in EP300/KAT3B and CBP/KAT3A have been implicated in the pathogenesis of cancer. Here, the authors reveal that the EP300 protein has a role in mediating replication fork protection at sites of replication stalling and show that EP300-mutated cells recapitulate features of BRCA-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c1df38e22e1ee22880d25154078c1bc6f87527f" target='_blank'>
              EP300 deficiency leads to chronic replication stress mediated by defective replication fork protection
              </a>
            </td>
          <td>
            Angelica Barreto-Galvez, Mrunmai Niljikar, J. Gagliardi, Carolina Plasencia Guzman, Ranran Zhang, Vasudha Kumar, Aastha Juwarwala, Archana Pradeep, Ankit Saxena, Cristina Montagna, Priya Mittal, Jeannine Gerhardt, Bing Xia, Jian Cao, Keisuke Kataoka, A. Durbin, Jun Qi, B. H. Ye, Advaitha Madireddy
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract V(D)J recombination, essential for adaptive immunity, is initiated by RAG1/2, which recognizes recombination signal sequences (RSSs) flanking gene segments at antigen receptor loci. RSSs comprise conserved heptamer and nonamer motifs separated by a 12/23-base spacer. While the first three heptamer nucleotides (5′-CAC) are strictly conserved, RSS promiscuity enables RAG “off-target” activity at RSS-like sequences, termed “cryptic RSSs” (cRSSs), contributing to chromosomal deletions and lymphoid tumorigenesis. Notably, the nonamer exhibits substantial sequence variability across physiological RSSs. In addition, many cRSSs lack a discernible nonamer-like sequence, limiting the ability to predict RAG targets based on RSS consensus. Using a high-throughput approach, we characterized here the nonamer properties supporting RAG-mediated recombination. While the consensus nonamer (5′-ACAAAAACC) exhibited strong functionality, numerous sequences significantly different from it promoted a high recombination rate. These functional nonamers balance two, somewhat opposing, features: affinity for the RAG nonamer-binding domain—primarily via a shared purine–A/T–purine motif at positions 4–6—and nucleosome repulsion. Moreover, nonamers of genomic cRSSs mimic canonical nonamers mainly through nucleosome-repelling sequences. This study provides a model for both physiological RAG activity and its off-target effects, enhancing our understanding of immune repertoire formation and the genetic basis of lymphoid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3be51b4ff1033c4b22dd2a4e43eac3aedc8d967" target='_blank'>
              The nonamer code for RAG-mediated recombination in vivo
              </a>
            </td>
          <td>
            Wessal Hanout-Chuiko, Sunil Pachakar, Rawan Foqara, Hadas Sibony-Benyamini, Shira Peretz, Nurit Edri, Moshe Dessau, Yaakov Maman
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, T. F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03533d5a535f8136a5f4c27e74c671f6ec467639" target='_blank'>
              cGAS restricts PARP1-mediated microhomology-mediated end joining by suppressing poly-ADP-ribosylation.
              </a>
            </td>
          <td>
            Haibo Yang, Boya Gao, Bo Wu, Wenjing Li, Xueping Zhu, Laiyee Phoon, Jing Luo, Fu Gui, Weixing Zhao, Li Jia, Li Lan
          </td>
          <td>2025-12-17</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism.
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Break-induced replication (BIR) is a primary homologous recombination pathway for repairing one-ended double-strand DNA breaks, including those arising from collapsed replication forks and eroded telomeres. BIR frequently leads to loss of heterozygosity, genetic mutations, and gross chromosomal rearrangements, all hallmarks of cancer. Here, we conducted a genome-wide screen that allowed us to identify and validate the involvement of 33 novel yeast genes in BIR. We report that, while DNA damage and spindle checkpoint machineries are both required to delay nuclear division and provide adequate time for BIR to complete, the spindle position checkpoint is required to coordinate between nuclear division and cytokinesis. Furthermore, we show that two nucleopore proteins play a sequential role during BIR: Nup84 acts before DNA synthesis, while Nup188 functions later to support repair completion. Given the conservation of BIR between yeast and humans and the role of BIR in cancer development, human homologs of the identified BIR proteins may represent promising targets for anti-cancer therapeutics. Break-induced replication (BIR) repairs broken DNA but can also destabilize genomes. The authors identify 33 new genes controlling BIR completion, showing that spindle assembly and spindle positioning checkpoints coordinate repair, and that nuclear pore proteins regulate BIR at multiple steps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, D. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Purpose BEND3 is implicated in various physiological processes, including chromatin regulation, cell cycle regulation, etc.; nonetheless, its function in cancer is not well comprehended. Methods Using public databases including UCSC Xena, TCGA, GTEx, and GEO, we conducted a comprehensive pan-cancer analysis to evaluate the clinical relevance of BEND3 across 33 cancer types. We analyzed genetic alterations, copy number variations (CNVs), and methylation profiles of BEND3, and explored its associations with tumor mutational burden (TMB), microsatellite instability (MSI), immune checkpoint molecules, and the tumor immune microenvironment. A protein-protein interaction (PPI) network and functional enrichment analysis were performed to investigate potential molecular mechanisms. In vitro, BEND3 expression was assessed by immunohistochemistry (IHC), Western blot (WB), and qRT-PCR. Cell Counting Kit-8 (CCK-8), colony formation, and wound healing assays were performed to validate BEND3’s oncogenic role in breast cancer cell lines. Results Our results show that BEND3 is frequently overexpressed in multiple cancers, including breast, liver, lung, thyroid, and gastric cancers, among others, and is associated with poor prognosis. Its expression correlates with TMB, MSI, immune checkpoint molecules, and immunoinfiltration, suggesting a role in tumor immunity. Functional analysis indicates involvement in key cancer-related pathways. In breast cancer, BEND3 was upregulated in clinical tissues and cell lines, and in vitro experiments demonstrated that BEND3 overexpression promoted the proliferation, migration, and invasion of breast cancer cells, whereas its knockdown suppressed these phenotypes. Conclusion Our findings suggest BEND3 is a potential prognostic and immune-related cancer biomarker. Its overexpression in multiple cancers, association with poor survival, and role in tumor immunity support its oncogenic function. Functional analysis indicates it may regulate key cancer pathways. Importantly, in vitro experiments confirm its tumor-promoting effects in breast cancer, providing a foundation for further study of BEND3’s mechanisms and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3fc8a61c1cbaa61261fe78f5975a9b571d5adf" target='_blank'>
              Integrated Pan-Cancer Profiling and Breast Cancer Validation Identify BEND3 as a Potential Prognostic and Immune Biomarker
              </a>
            </td>
          <td>
            Yuting Gou, Yongxiang Li, Yiyang Wang, Haotian Ma, Haocong Li, Reyima Geni, Dilimulati Ismtula, Chenming Guo
          </td>
          <td>2025-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Transposable elements (TEs) are often epigenetically repressed in eukaryotic cells, but still affect the molecular state of the cell in certain contexts. A flurry of recent studies have elucidated new effects of TE sequences in eukaryotic cells. We review these emerging molecular effects of TEs, including a variety of new mechanisms by which TE sequences affect the cell, including pre- and post-transcriptional regulation of gene expression; cell-to-cell transmission of genes within a multicellular organism through virus-like activity; and RNA-guided DNA insertion. Recent demonstration of TE-guided genome editing underscores the importance of these investigations for both basic and translational research. Future work is needed to continue to unravel the molecular effects of TE sequences across developmental stages, across cell types, and in various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3445e991f8e7a3fbd17d3feddadbe0f7c8e2d56" target='_blank'>
              Molecular effects of transposable element sequences in mammalian cells
              </a>
            </td>
          <td>
            Ming-Ching C. Wen, Joshua D. Welch
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6136e0a41212ee02eab7d57de4a28f7d297bef9a" target='_blank'>
              Sensing of DNA double-strand breaks by the NHEJ system stabilizes RORγt transcriptional activity and shapes Th17 pathogenicity in autoimmunity.
              </a>
            </td>
          <td>
            Guan-Yu Chen, Wen-Jie Zhu, Zhuang Li, Yun-Wei Hu, Xiao-Shuang Luo, Zhi-Qing Mai, Yuan Pan, Yu-Xun Shi, Zuo-Yi Li, Jun Huang, Pei-Dong Yuan, Zhi-Qiang Xiao, Qian Chen, Yan-Yan Xie, Hai-Xiang Huang, Yu-Xi Chen, Yao Lu, Min-Zhen Wang, Yi-Wen Xia, Xiao-Qing Chen, Dong-Ming Kuang, Dan Liang
          </td>
          <td>2026-01-07</td>
          <td>Cell research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="RecQ family of DNA helicases play pivotal roles in DNA replication, repair and responses to DNA damage or replication stress. Several human RecQ helicases are defective in diseases associated with chromosomal instability, premature aging and cancer. We recently discovered novel mutations in the RECQL4 gene in glioblastoma (GBM), the most malignant brain tumor in adults. Transcriptomic profiles of GBMs with REQCL4 mutations resembled those in REQCL4 KO glioma cells. We employ structural modelling and biochemical approaches to elucidate impacts of novel mutations on RECQL4 helicase activities. Using recombinant RECQL4P532S and RECQL4R766Q proteins we demonstrate that P532S substitution reduces the RECQL4 ability to unwind DNA and disrupts DNA-coupled ATP-hydrolysis activity. WT and mutated RECQL4 were overexpressed in RECQL4 KO glioma cells to study interactions with BLM helicases, cell viability and specific responses to UVC- and chemotherapy-induced DNA damage/repair. Overexpression of RECQL4P532S or RECQL4R766Q variants affected DNA repair and responses to chemotherapeutics in glioma cells, and RECQL4R766Q disturbed interactions with the BLM helicase. Our results reveal deleterious consequences of novel RECQL4 mutations in GBMs. The newly identified RECQL4 mutations affect RECQL4 helicases and their interactions with BLM contributing to glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650345dd7db490bb44b3ecdcc3b0ad865cd59a8" target='_blank'>
              Novel mutations in the RECQL4 gene affect its helicase functions, interactions with the BLM helicase and chemotherapeutics-induced cell death
              </a>
            </td>
          <td>
            Agnieszka Kaczmarczyk, Mikołaj Sokołowski, K. Wojnicki, Marta Pabiś, B. Wojtaś, I. Ciechomska, K. Poleszak, B. Gielniewski, S. K. Król, Matthew Guille, Sebastian Glatt, B. Kaminska
          </td>
          <td>2025-12-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Spindle cell lipomas (SCL) and pleomorphic lipomas (PL) are today considered a single tumor entity (SCLPL). Atypical SCLPL, in contrast, represents a recently recognized related but distinct entity. Here we explored the correlation between genomic features and morphological aspects of SCLPL and atypical SCLPL and the role of the RB1 gene in tumor development. Seventy‐one samples from 68 patients with SCLPL or atypical SCLPL, as well as a lipomatous tumor from a retinoblastoma patient with a germline pathogenic variant in the RB1 gene, and two pleomorphic liposarcomas were analyzed using chromosome banding, high‐resolution genomic arrays (SNP array), whole exome sequencing (WES), and/or RNA sequencing (RNA‐seq). Common for all tumors was involvement of 13q; other recurring variants were deletion of 16q, 6q, and 17p. A minimally deleted region that only contained RB1 was found on 13q. A distinction was seen between SCL on the one hand and PL and atypical SCLPL on the other; SCL had fewer copy number aberrations in general, and loss of 17p/TP53 gene or a SNV affecting TP53 was only rarely detected in SCL but seen in the vast majority of PL and atypical SCLPL tumors. Thus, at the molecular level, SCL is different from PL/atypical SCLPL. Furthermore, the finding of the same copy number changes (loss of 13q, 6q, 16q, and 17p) in some pleomorphic liposarcomas raises the possibility that a subset of SCLPL/atypical SCLPL have the potential for malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b1988ef8c66c3a7438406d89e232e1e91d6e09" target='_blank'>
              The Role of RB1 and Secondary Genomic Changes in the Development of Spindle Cell and Pleomorphic Lipomas
              </a>
            </td>
          <td>
            Maria Hellberg, V. Difilippo, Emilia Gottberg, A. Hesla, Felix Haglund de Flon, J. Nilsson, L. Magnusson, Saskia Sydow, Paul Piccinelli, Jakob Hofvander, Fredrik Mertens
          </td>
          <td>2025-12-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Retinoblastoma is the most common intraocular malignancy of childhood, yet its genomic landscape remains incompletely defined, particularly in understudied populations. Beyond RB1 loss, the contribution of additional somatic and germline alterations to disease heterogeneity and clinical behavior is unclear.



 We performed whole-exome sequencing of 166 retinoblastoma samples from 166 patients with matched germline DNA, representing the largest cohort analyzed to date. Clinical data were available for 160 patients. Variant calling, copy number alteration (CNA) profiling, and integrative analyses were performed to characterize genetic drivers and their associations with clinical features.



 Pathogenic RB1 variants were identified in 120 patients, and MYCN amplification in six patients. Additional recurrent alterations involved BCOR, CCND3, ERBB2, and PDGFRB. Copy number gains of 6p (41.3%) and 17q (8.1%) were significantly associated with high-risk features including rubeosis, subretinal seeding, and tumor extension beyond the lamina cribrosa. Germline ERBB2 variants correlated with orbital invasion, while germline PDGFRB variants were associated with second primary cancers. Together, these findings underscore the genetic heterogeneity of retinoblastoma and reveal novel genotype–phenotype correlations.



 This study provides the most comprehensive genomic characterization of retinoblastoma to date, expands the known mutational spectrum, and identifies biomarkers with direct clinical relevance. These insights have potential to refine risk stratification, inform precision therapeutic strategies, and improve long-term outcomes for children with retinoblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca1a82a307761a6281f02514b112932e6b6ae23" target='_blank'>
              Genomic landscape of retinoblastoma: Insights into risk stratification and precision pediatric Neuro-Oncology
              </a>
            </td>
          <td>
            A. Maktabi, Yang Liu, Saleh A. Almesfer, M. Abdelhakim, Hind Aldakhil, Maxat Kulmanov, Deepak P. Edward, R. Hoehndorf, M. Abedalthagafi
          </td>
          <td>2025-12-12</td>
          <td>Neuro-Oncology Pediatrics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5 A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The single-molecule DNA fiber spreading assay is widely used to study DNA replication fork dynamics through pulse labeling of nascent DNA with distinct thymidine analogs. However, tight sister chromatid cohesion during spreading causes leading and lagging strands to appear as a single fiber, masking strand-specific replication changes that are increasingly recognized as biologically and therapeutically important. Here, we report the unexpected finding that lentiviral infection of human cell lines prior to DNA spreading enables visualization of lagging strand–specific single-stranded DNA gaps arising from defects in Okazaki fragment maturation. Our results suggest that such an effect is transient and independent of viral vectors, encoded sequences, or genome integration. Mechanistically, lentivirus reduces cohesin level on nascent DNA, indicating that transient loosening of sister chromatid cohesion allows daughter strand separation. We propose that pre-exposure to control lentiviruses can be a simple, effective modification to enable the DNA fiber spreading assay to detect strand-specific DNA replication changes and overcome its limitation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ef2ffbae1ef8504f554c5b27139d4fac6d3c93" target='_blank'>
              Lentivirus Enables the Detection of Strand-Specific ssDNA Gaps by the DNA Fiber Spreading Assay
              </a>
            </td>
          <td>
            Jieya Shao, Melisande Wong, Xupei Ou, Evan Brown Ton
          </td>
          <td>2025-12-12</td>
          <td>Research Square</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background and objectives Somatic mutation patterns in cancer remain largely unexplored outside humans, despite their significance for aging and oncogenesis. Chimpanzees (Pan troglodytes), sharing >98% genomic similarity with humans, display markedly different cancer spectra. To gain comparative insights into cancer susceptibility and resistance, we sequenced chimpanzee hepatocellular carcinoma (HCC) genomes and analyzed their mutational profiles alongside human counterparts. Methodology HCC and matched non-cancerous tissues from five chimpanzees were examined using histopathology, immunohistochemistry (β-catenin, ARID1A, TSC2, FAP, vimentin, TGF-β), whole-genome sequencing (one pair), and whole-exome sequencing (four pairs). Somatic variants were identified with GATK MuTect2, annotated with Ensembl VEP, and analyzed for functional enrichment. Comparative analyses were performed with subsets of human HCC datasets (TCGA, ICGC) including TSC2-positive and TSC2-negative cases. Results Chimpanzee HCCs exhibited histological and immunohistochemical features similar to human tumors but displayed sharply divergent genomic landscapes. Chimpanzee tumors carried significantly higher coding mutation loads (mean 5632 per sample vs. 96–275 in humans). Non-synonymous TSC2 mutations occurred in 80% of chimpanzees, versus ~7% in human HCC, suggesting a species-specific oncogenic pathway linked to the scirrhous subtype. Additional recurrently mutated genes included ARID1A, FAT1–4, TP53, and FGA. Despite greater heterogeneity in chimpanzee tumors, humans showed stronger enrichment of non-synonymous single nucleotide variants, implying more intense positive selection. Shared alterations across species involved canonical drivers such as TP53, CTNNB1, FAT4, and TTN. Conclusions and implications Chimpanzee HCCs are defined by high mutational burden and frequent TSC2 alterations, contrasting with the more selectively constrained mutation spectrum of human HCC. Divergent evolutionary patterns highlight species-specific oncogenic routes while underscoring conserved pathways. Comparative primate cancer genomics offers novel insights into cancer evolution, biomarkers, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7fdead0d4830205015111b5a2fbfd3bc9dc0ec" target='_blank'>
              Divergent evolution of hepatocellular carcinoma genomes in chimpanzees and humans
              </a>
            </td>
          <td>
            Lin Kang, K. Michalak, Robin T. Varghese, R. Anandakrishnan, Edward J Dick, Z. A. Abd Elmageed, Pawel Michalak
          </td>
          <td>2025-12-15</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Ewing sarcoma is driven by the EWSR1–FLI1 fusion oncoprotein, a transcription factor created by chromosomal translocation in otherwise low-mutation tumors. EWSR1–FLI1 recruits chromatin-remodeling complexes, opens chromatin, and establishes broad H3K27ac/H3K4me1 domains. Uniquely, it binds normally silent GGAA microsatellites dispersed across the genome and converts them into de novo enhancers, elevating transcription of downstream targets. We hypothesize that EWSR1–FLI1 engagement of GGAA repeats globally destabilizes chromatin and enforces a differentiation block, and that direct inhibition of its DNA binding will restore transcriptional control. To test this, we established a propidium iodide (PI) uptake assay after exposure to the chromatin-damaging agent CBL0137 as a surrogate for chromatin accessibility, benchmarking ES against non-ES tumor and non-tumor cells, and in parallel engineered a DNA-binding–dependent reporter to screen small molecules that disrupt fusion–DNA interactions. ES cells showed greater PI uptake following CBL0137 than controls, consistent with globally relaxed chromatin. The reporter screen identified candidate compounds that decreased fusion-dependent activity in a dose-responsive manner; orthogonal luciferase assays supported potential on-target inhibition, and viability assays revealed selective toxicity in ES relative to fusion-negative sarcomas. These findings support a model in which fusion oncoproteins promote chromatin relaxation and establish a complementary screening platform that nominates direct fusion inhibitors, providing mechanistic insight and therapeutic starting points for pediatric sarcomas.



 Ayah Salameh. GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c2a09d1dd63318d72cdbf406bfb1ca4ece465c" target='_blank'>
              Abstract A010: GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform
              </a>
            </td>
          <td>
            Ayah Salameh
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5f6a9b5974a2c0b0f8a40b9e9b46d7609aa730" target='_blank'>
              DNA sequence quantitatively encodes CTCF-binding affinity at genome scale
              </a>
            </td>
          <td>
            Zihang Yin, Yefan Wang, Guojun Hou, Yuqiao Chen, Huaijin Feng, Jin Zhu, Yufan Xie, Ruoxi Wang, Zhiyu Zhang, Tao Wu, Haiyan Huang, Sheila Q. Xie, Wuming Wang, Wenchao Gu, Qiang Wu, Nan Shen, Ya Guo
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771a8f2444012ff6a011f16c283fdc97cba129d" target='_blank'>
              CDC7 and CDK8 kinases cooperate to support DNA replication origin firing in human cells
              </a>
            </td>
          <td>
            M. Rainey, Stefanus Bernard, Aikaterini Pravi, Liadhan Farrell, Dominik Boos, Colm J. Ryan, Corrado Santocanale
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer. Krepelova and colleagues identify an aging- and colon cancer-associated DNA methylation drift in healthy and cancerous human colon samples, echoed in the mouse intestinal epithelium. They show it expands via intestinal cell clonality and is driven by dysregulated iron metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) can be induced by cellular byproducts or genotoxic agents. Improper processing of these lesions leads to increased genome instability, which constitutes a hallmark of pathological conditions and fuels carcinogenesis. DSBs are primarily repaired by homologous recombination (HR) and non-homologous end joining (NHEJ) and the proper balance between these two pathways is finely modulated by specific molecular events. Here, we report that the histone chaperone DAXX plays a fundamental role in the response to DSBs. Indeed, in human cells, DSBs induce ATM/ATR-dependent phosphorylation of DAXX on serine 424 and 712 and promote its binding to chromatin and the deposition of the histone variant H3.3 in proximity to DNA breaks. Enrichment of H3.3 at DSBs promotes 53BP1 recruitment to these lesions and the repair of DNA breaks by HR pathways. Moreover, H3.3-specific post translational modifications, particularly K36 tri-methylation, play a key role in these processes. Altogether, these findings indicate that DAXX and H3.3 mutations may contribute to tumorigenesis-enhancing genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/009a4d5a2fca64469e1daafccf7d54c8bc81a3b2" target='_blank'>
              Daxx-Dependent H3.3 Deposition Promotes Double-Strand Breaks Repair by Homologous Recombination
              </a>
            </td>
          <td>
            L. Zannini, Simona Aliprandi, Domenico Delia, G. Buscemi
          </td>
          <td>2026-01-16</td>
          <td>Cells</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract Z-DNA is a left-handed DNA conformation linked to gene regulation, chromatin dynamics, and immunity. Despite its importance, genome-wide prediction of Z-DNA forming sequences (ZFS) remains limited by the absence of fast and accessible tools. Here, we present Z-DNA Hunter, a user-friendly web server for genome-scale ZFS prediction utility. The algorithm employs a pattern-based approach optimized for canonical motifs such as (GC)n and (CA)n repeats, with adjustable parameters for detection stringency. Compared with existing methods, Z-DNA Hunter achieves similar or higher accuracy while reducing runtime from hours to seconds, making large-scale analyses feasible. To demonstrate its application, we analyzed the Drosophila melanogaster genome and uncovered a pronounced enrichment of long ZFS on the X chromosome, contrasting with their near absence on the satellite repeat- and transposable element-rich Y chromosome. These findings illustrate both the scalability of Z-DNA Hunter and its potential to reveal biologically meaningful patterns of non-B-DNA. The tool provides direct visualization and export options (e.g. BedGraph for UCSC Genome Browser) and is freely available at https://bioinformatics.ibp.cz/#/analyse/zdna.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782d54fb0da840e7a91c37526570fd49fc670b36" target='_blank'>
              Z-DNA Hunter tool for straightforward detection of Z-DNA forming regions and a case study in Drosophila
              </a>
            </td>
          <td>
            Michal Petrovič, M. Bartas, Alistair N Garratt, P. Pečinka, Michaela Dobrovolná, Klára Koňaříková, O. Trenz, Václav Brázda, Jiří Šťastný
          </td>
          <td>2025-10-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="The APOBEC3 family of RNA and single stranded DNA cytidine deaminases contribute prominently to the mutagenesis of certain cancers including urothelial carcinoma of the bladder (UC). Remarkably, up to 70% of mutations in UC are attributable to the mutagenic activity of the APOBEC3 deaminases. Despite this strong association, few functional studies have investigated APOBEC3’s role in bladder cancer. We report a genetically engineered murine model with conditional knock out of Pten and Trp53 in addition to overexpression of mouse Apobec3 (UPPA). Analysis of bladder tumors from UPPA mice demonstrates that mA3 promotes tumor progression and squamous trans-differentiation. We establish that APOBEC3 promotes squamous differentiation through IL-1α and downstream activation of the AP-1 transcription factor. Bulk RNA-sequencing from human UC shows APOBEC3A as the only human APOBEC3 family member to correlate with squamous differentiation. Furthermore, single cell and spatial transcriptomics reinforces the role of APOBEC3A in fostering squamous trans-differentiation and promoting the emergence of a subpopulation of highly squamous epithelial cells. Our results demonstrate that mouse Apobec3 and human APOBEC3A promote squamous differentiation in urothelial carcinoma and that this trans-differentiation phenotype is mediated through IL-1α signaling, a target of FDA approved therapies for rheumatologic disease. Deaminases of the APOBEC3 family contribute to the mutagenesis of various cancers, including urothelial carcinoma (UC) of the bladder. Here, the authors use functional studies and transcriptomics to demonstrate that APOBEC3 promotes tumour progression and squamous trans-differentiation in UC through IL-1α and downstream activation of the AP-1 transcription factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14e67774e5f26f7adb282a687c97e8843cc0470" target='_blank'>
              APOBEC3 promotes squamous differentiation via IL-1A/AP-1 signaling
              </a>
            </td>
          <td>
            Michael S. Sturdivant, Andrew Truong, Mi Zhou, E. D. Toomer, Wolfgang Beckabir, John D. Raupp, Ujjawal Manocha, Ibardo A. Zambrano, Hung-Jui Tan, M. Bjurlin, Angela B Smith, Tracy L Rose, Matthew I. Milowsky, Sara E. Wobker, Kathryn H. Gessner, J. Damrauer, William Y. Kim
          </td>
          <td>2025-12-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f26e5c36cf6fc942909dc97f82cb84d88175870d" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes.
              </a>
            </td>
          <td>
            Jiawen Yang, John M. Ryniawec, Diogo de Oliveira Pessoa, Matthew R Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Megha Padi, Gregory C Rogers
          </td>
          <td>2026-01-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Colibactin is a secondary metabolite produced by bacteria present in the human gut and is implicated in the development of colorectal cancer. This genotoxin alkylates deoxyadenosines on opposite strands of host cell DNA to produce DNA interstrand cross-links. While cells have evolved multiple mechanisms to resolve (“unhook”) interstrand cross-links, little is known about which of these pathways promote resistance to colibactin. Here, we use Xenopus egg extracts to investigate replication-coupled repair of colibactin-induced interstrand cross-links. We show that replication fork stalling at a colibactin-induced interstrand cross-link activates the Fanconi anemia interstrand cross-link repair pathway, which unhooks the interstrand cross-link through nucleolytic incisions. These incisions generate a DNA double-strand break intermediate in one sister chromatid, which can be repaired by homologous recombination, and a monoadduct (“interstrand cross-link remnant”) in the other. Translesion synthesis past the colibactin-induced interstrand cross-link remnant depends on Pol η and the Pol κ-REV1-Pol ζ polymerase complex and introduces predominantly T>A point mutations at the sites of colibactin alkylation. Taken together, our work provides a molecular framework for understanding how cells tolerate a naturally occurring and clinically relevant interstrand cross-link. The bacterial genotoxin colibactin induces DNA interstrand cross-links which pose a barrier to DNA replication. Here, the authors use Xenopus egg extracts to show that the Fanconi anemia pathway is responsible for repairing these cross-links.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af9af8aec073ccec4ec8bcc56779d2d58c22a2a" target='_blank'>
              The Fanconi anemia pathway repairs colibactin-induced DNA interstrand cross-links
              </a>
            </td>
          <td>
            Maria Altshuller, Xu He, Elliot J. MacKrell, Kevin M. Wernke, Yougant Airan, Joel W. H. Wong, Selene Sellés-Baiget, T. Wang, Tsui-Fen Chou, Julien P. Duxin, E. Balskus, Seth B. Herzon, Daniel R. Semlow
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is one of the most common cancers and a leading cause of cancer-related mortality worldwide, highlighting the need for novel therapeutic strategies. Proteasome 26S Subunit, Non-ATPase 12 (PSMD12), a component of the proteasomal 19S regulatory particle, is associated with tumorigenesis; however, its role in LUAD remains poorly understood. Integrative bioinformatic analysis of The Cancer Genome Atlas (TCGA) and other publicly available LUAD datasets identified PSMD12 as a candidate driver gene on chromosome 17q, a region frequently amplified in LUAD. Clinicopathological and prognostic analyses revealed that PSMD12 was significantly upregulated in tumor tissues because of DNA copy number gain. High PSMD12 expression was associated with poor prognosis and advanced pathological stages. Gene set enrichment analysis of TCGA LUAD dataset demonstrated that samples with high PSMD12 expression were enriched for cell cycle-related pathways. Using CRISPR-Cas9-mediated PSMD12 knockout and lentivirus-mediated overexpression models, we demonstrated that PSMD12 promoted tumor cell proliferation by accelerating the G2/M cell cycle transition in vitro, and xenograft experiments confirmed its tumor-promoting effect in vivo. Mechanistically, PSMD12 overexpression reduced the ubiquitination of CDK1, a key regulator of mitotic entry. Cycloheximide chase and MG132 assays confirmed that PSMD12 stabilized CDK1 by inhibiting proteasome-mediated degradation. In conclusion, we identified PSMD12 as a novel driver gene and prognostic biomarker of LUAD. PSMD12 promoted LUAD progression by modulating CDK1 ubiquitination and enhancing cell cycle progression. These findings suggest that PSMD12 is a promising molecular target for future LUAD therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb3bc3b62a546392f132deef63cdce1ec580a2a" target='_blank'>
              PSMD12 Overexpression Promotes Lung Adenocarcinoma Progression via Ubiquitin-Proteasome Pathway Dysregulation.
              </a>
            </td>
          <td>
            Yuya Ono, Hajime Otsu, Takaaki Masuda, K. Kosai, Shohei Shibuta, Kosuke Hirose, Takashi Ofuchi, Yuki Ando, Koto Kawata, Yasuo Tsuda, Yusuke Yonemura, Taro Tobo, T. Takenaka, T. Yoshizumi, Koshi Mimori
          </td>
          <td>2026-01-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Identifying viruses in tumor transcriptome helps to unravel the potential role of viruses in oncogenesis and tumor progression. Most of the current tools for virus identification in RNA-Seq data rely on sequence alignment, whose performance is constrained by fast mutations, large divergence, and the incompleteness of viral genomes. In this study, we develop ViTrace to detect viral sequences in human transcriptomic data by a hybrid language representation learning model, which integrates DNA contexts, position relationships and amino acid coding information. Although ViTrace is only trained on 13 species from 7 genera, it achieves 86.39% recall in 1179 absent-in-train virus strains of 935 species belonging to 167 genera across 10 phyla. Applied to single-cell RNA-seq data from esophageal and oropharyngeal squamous cell carcinomas, the model reveals tumor-, cell-, and patient-specific viral colonization patterns, uncovering both known and previously unreported viruses. Overall, ViTrace provides a scalable framework for guiding precision oncology and facilitating future discoveries of previously uncharacterized tumor-associated viruses. Hybrid language model ViTrace identifies viral sequences from human transcriptomes, revealing tumor-, cell-, and patient-specific viral colonization patterns and uncovering novel tumor-associated viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235fa47ea6e35bee83f6f73e3fb62ee13668fc7d" target='_blank'>
              ViTrace detects viral signatures in tumor transcriptomes using a hybrid language model
              </a>
            </td>
          <td>
            Feng Zhou, Yushuang He, Fan Yang, Jin Gu, Fengzhu Sun, Shunzhi Zhu, Xiaobing Huang, Ying Wang
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Following cell entry, HIV-1 capsids enter the nucleus by passage through nuclear pores and reach nuclear speckles with subsequent uncoating of the reverse-transcribed genome and its integration into speckle-associated chromatin domains. Here, we characterized the ultrastructure of HIV-1 subviral complexes in nuclei of primary monocyte-derived macrophages and cell lines using live-cell imaging, super-resolution microscopy, and correlative light and electron tomography in the absence and presence of capsid-targeting inhibitors Lenacapavir and PF74. Capsid-like structures containing viral DNA, as well as broken capsids, clustered in nuclear speckles and were displaced from speckles by drug treatment. This was accompanied by alteration of the nuclear capsid structure, with electron-dense protrusions emanating from the narrow end of capsid cones and exposure of integration-competent genomic HIV-1 DNA. Our data indicate that synthesis of genomic dsDNA can be completed inside the closed HIV-1 capsid, and speckle-associated factors could regulate genome uncoating. This may ensure that genome uncoating occurs at optimal sites for integration into transcriptionally active chromatin. The results also shed further light on the mechanism of action of Lenacapavir. The mechanism of HIV-1 genome uncoating is currently not well understood. This work shows that capsid-targeting drugs lay bare integration-competent genomes enclosed in nuclear HIV-1 capsids, indicating that reverse transcription can be functionally completed inside the closed capsid. Conical and broken capsids cluster in nuclear speckles of primary monocyte-derived macrophages. Lenacapavir or PF74 displace CPSF6 from nuclear capsids, resulting in rapid exit of subviral structures from nuclear speckles. Lenacapavir or PF74 also expose previously hidden genomes, indicating that reverse transcription by itself is not sufficient to induce immediate uncoating. Brief Lenacapavir pulses increase the amount of integrated provirus in macrophages, indicating integration competence of the exposed genomes. CLEM-ET reveals Lenacapavir-induced bifurcated protrusions at the narrow end and flattened lattice remnants. Conical and broken capsids cluster in nuclear speckles of primary monocyte-derived macrophages. Lenacapavir or PF74 displace CPSF6 from nuclear capsids, resulting in rapid exit of subviral structures from nuclear speckles. Lenacapavir or PF74 also expose previously hidden genomes, indicating that reverse transcription by itself is not sufficient to induce immediate uncoating. Brief Lenacapavir pulses increase the amount of integrated provirus in macrophages, indicating integration competence of the exposed genomes. CLEM-ET reveals Lenacapavir-induced bifurcated protrusions at the narrow end and flattened lattice remnants. Exposure of integration-competent genomes upon treatment with capsid-targeting drugs reveals that HIV-1 reverse transcription can be completed in closed viral capsids without automatically inducing uncoating.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f8abb290a62c5b3aa53581143c941a3ce167f6" target='_blank'>
              Lenacapavir-induced capsid damage uncovers HIV-1 genomes emanating from nuclear speckles
              </a>
            </td>
          <td>
            Thorsten G. Müller, Severina Klaus, V. Zila, Bojana Lucic, C. Penzo, Svenja L. Nopper, Gonen Golani, Maria Anders-Össwein, Vera Sonntag-Buck, A. Heuser, U. Schwarz, V. Laketa, M. Lusic, Barbara Müller, H. Kräusslich
          </td>
          <td>2025-12-01</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) is a powerful isothermal strategy for nucleic acid detection, but its linear kinetics and dependence on externally supplied primers limit its sensitivity and programmability. Here, we report an exponential RCA (E-RCA) platform that integrates primer regeneration and signal amplification into a single DNA-encoded system. The design uses a circular DNA template encoding the I-R3 self-cleaving DNAzyme sequence; during RCA, tandem I-R3 units are generated within the DNA amplicons, which then catalyze site-specific cleavage to release new primers. This self-sustained (RCA ↔ DNAzyme) amplification circuit enables robust exponential signal growth using only a circular DNA probe and a DNA polymerase without requiring external primers or protein enzymes. We elucidate the mechanism through biochemical experiments and kinetic modeling and validate the system in multiplexed intracellular microRNA imaging and quantitative dual-marker profiling of clinical breast cancer tissues. The E-RCA strategy achieved high diagnostic accuracy (AUC = 0.914; specificity = 100%; sensitivity = 81.3%), demonstrating its potential for sensitive, programmable, and autonomous molecular analysis in both biological and clinical contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779a0bfc06573a96632b1849e24d829296851cc1" target='_blank'>
              Autonomous Exponential Amplification via Rolling Circle-DNAzyme Feedback Programming.
              </a>
            </td>
          <td>
            Jinhua Shang, Mengdi Yu, Yifei Wang, Shanshan Yu, Xianzheng Zhang, Fuan Wang, Yingfu Li
          </td>
          <td>2025-11-26</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf9f556241d0007b43f3963aee0e9a310306573" target='_blank'>
              lncRNA EGFR-AS1 promotes DNA damage repair by enhancing PARP1-mediated PARylation.
              </a>
            </td>
          <td>
            Kun Gao, Ruiya Shi, Chenying Xu, Zhaoyang Mao, Changying Guo, Liang Jin
          </td>
          <td>2026-01-05</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Micronuclei are formed during cell division when acentric fragments or lagging chromosomes cannot be incorporated into the primary nucleus. Macroautophagy/autophagy may reduce chromosomal instability (CIN) by clearing isolated, atypical micronuclei. Other studies implicate that the loss of autophagy disrupts DNA repair pathways. However, whether aberrant mitosis contributing to CIN occurs when autophagy is inhibited has yet to be evaluated. We found impaired autophagy initiation contributes to CIN and facilitates the formation of micronuclei and other abnormal nuclear phenotypes either by genetic or pharmacological manipulation in multiple cell lines. We also found that loss of the integral autophagy protein ATG9A resulted in various types of mitotic errors that can contribute to the formation of micronuclei. ATG9A also localizes to centrosomes and midbody during cell division. Autophagy inhibition causes the overactivation and mislocalization of TBK1 (TANK binding kinase 1) into cytoplasmic, punctate structures that colocalize with SQSTM1/p62. This overactivation interferes with its function in cell division as a mitotic kinase and its role at the centrosome. These results indicate that loss of autophagy contributes to genomic instability from multiple angles, one of which being aberrant cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7652d445dff3bc9d16db0af8750b841d562ae2" target='_blank'>
              Disrupted autophagy overactivates TBK1 and results in mitotic defects promoting chromosomal instability.
              </a>
            </td>
          <td>
            Swagatika Paul, Porter L Tomsick, Julia P. Milner, Sahitya Ranjan Biswas, Samantha Brindley, Nicole DeFoor, Leila Zavar, Grace Wright, Yairis Soto, A. Pickrell
          </td>
          <td>2026-01-16</td>
          <td>Autophagy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Apurinic/apyrimidinic endonucleases - APE1 and APE2 are central to genome maintenance and the cellular DNA damage response, with expanding relevance in cancer biology. APE1 is the primary endonuclease in base excision repair and functions as a redox coactivator of transcription factors. In contrast, APE2 exhibits PCNA dependent 3′–5′ exonuclease and 3′-phosphodiesterase activities, contributing to microhomology-mediated end joining, ATR-Chk1 activation, and immunoglobulin diversification. Both enzymes are often deregulated in cancer: APE1 is frequently overexpressed, drives tumor progression and chemoresistance, while APE2 is similarly upregulated in multiple malignancies. APE1 can be targeted by redox-specific or endonuclease inhibitors, with early clinical evidence of biological activity and tolerability. Although APE2-specific inhibitors remain in early development, emerging synthetic lethality data and preclinical studies highlight APE2 as a novel clinical target in breast cancer type 1/2 susceptibility (BRCA)-mutated cancers. This review discusses the structural and functional roles of APE1 and APE2, their contributions to cancer biology and therapeutics, recent advances in inhibitor development, and future strategies for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42492162323d747cbf3ed4c8e187621438e9d88" target='_blank'>
              Contrasting roles of APE1 and APE2 in genome maintenance, cancer development, and therapeutic targeting
              </a>
            </td>
          <td>
            Aman Sharma, Helen E. Grimsley, Katharine Courtemanche, Simon N. Powell
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Dysregulated transcription is a defining hallmark of cancer. Recently, novel chemically induced proximity approaches have enabled the rewiring of transcriptional machinery to drive expression of pro-apoptotic genes using bivalent small molecules. In this work, we demonstrate that this strategy is amenable to relocalizing DNA bound transcriptional machinery, such as fusion transcription factors that commonly drive pediatric malignancies. Targeting fusion transcription factors, such as EWSR1::FLI1 in Ewing sarcoma, with these bivalent compounds may open new therapeutic avenues. Here, we develop a small molecule, EB-TCIP, that recruits FKBP12F36V-tagged EWSR1::FLI1 to DNA sites bound by the transcriptional regulator BCL6, leading to rapid chromatin remodeling and expression of BCL6 target genes. This proof-of-concept study demonstrates that DNA binding proteins with pioneering transcription factor activity, such as EWSR1::FLI1, can be relocalized on chromatin to induce expression of repressed genes. Insights herein will guide the development of future bivalent molecules that rewire DNA binding transcriptional machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f7b6214e9ef05ce0f8a706c26c1ffab5a52705" target='_blank'>
              Rewiring the Fusion Oncoprotein EWSR1::FLI1 in Ewing Sarcoma with Bivalent Small Molecules.
              </a>
            </td>
          <td>
            Michael J Bond, Ryan P. Golden, G. Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, Hannah M Jones, Brendan G. Dwyer, N. Gray, K. Stegmaier
          </td>
          <td>2025-11-27</td>
          <td>Journal of the American Chemical Society</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d56a6de92ca30529df87362f7d733257748de198" target='_blank'>
              Fibrillarin and fibrillarin-like their role in cancer progression: new approaches and perspectives
              </a>
            </td>
          <td>
            Jaime Abelardo Ceja-López, Jany Valdes, Ana Miladinović, Ludovica Antiga, L. Rodríguez-Zapata, Pavel Hozak, Enrique Castaño
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cellular differentiation is driven by epigenetic modifiers and readers, including the methyl CpG binding protein 2 (MeCP2), whose level and mutations cause the neurological disorder Rett syndrome. During differentiation, most of the genome gets densely packed into heterochromatin, whose function has been simplistically viewed as gene silencing. However, gene expression changes reported in mutations leading to Rett syndrome have failed to be a predictor of disease severity. Here we show that MeCP2 increases nuclear stiffness in a concentration-dependent manner and dependent on its ability to cluster heterochromatin during differentiation. MeCP2-dependent stiffness increase could not be explained by changes in the expression of mechanobiology-related genes, but we found that it is disrupted by Rett syndrome mutations and correlated with disease severity. Our results highlight the impact of chromatin organization on the mechanical properties of the cell as an alternative or complementary mechanism to changes in cytoskeleton components. A correlative AFM–fluorescence approach elucidates how epigenetic control of chromatin organization contributes to nuclear stiffness and its dysregulation in Rett syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d330823c212a15c08be21ea0eb068997201f78c8" target='_blank'>
              MeCP2-driven chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Andreas Zhadan, Marah Mahmoud, Nina Trautwein, Bodo Laube, Christian Dietz, R. Stark, Cristina Cardoso
          </td>
          <td>2025-12-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gene expression regulation is a stochastic process that can be modified by mutations not only in a deterministic manner but also in a probabilistic way, for example by changing the extent of cell-to-cell variability in gene expression (also called “gene expression noise”) without necessarily changing the expected (mean) expression level. Such mutations can act either in cis (perturbing expression noise at their own locus) or in trans (perturbing expression noise of a gene located elsewhere in the genome). Although systematic studies have successfully analyzed the properties of cis-acting mutations modulating gene expression noise, less is known about the type of genetic changes that may alter expression noise in trans. Here, we applied genetic mapping on yeast strains (Saccharomyces cerevisiae) generated by random mutagenesis and identified three mutations (in chs1, yme2 and msh1 genes) that changed the expression noise of a reporter gene in the nuclear genome regulated by the yeast TDH3 promoter. Each of these mutations affected either extrinsic noise (variability due to cell-specific factors), intrinsic noise (inherent variability within each cell), or both. Surprisingly, all three mutations targeted proteins located outside of the nucleus: Yme2 and Msh1 being involved in the maintenance of mitochondrial genome integrity, and Chs1 being necessary for the repair of cell-wall defects in freshly-born daughter cells. Our results reveal that mitochondrial state can modulate the extent of intrinsic expression noise of eukaryotic nuclear genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648b0095152d2b4dfed796867db1f69280813729" target='_blank'>
              Trans-acting mutations reveal non-nuclear modulators of both intrinsic and extrinsic gene expression noise in a eukaryote
              </a>
            </td>
          <td>
            Nena Martin, Julie Kleine-Schultjann, Quentin Boussau, Agnès Dumont, Hélène Duplus-Bottin, Laurent Modolo, G. Yvert, P. Wittkopp, F. Duveau
          </td>
          <td>2025-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="N4BP1 specifically degrades a subset of mRNA targets through their coding sequences and functions as a negative regulator of inflammation; however, its role in cancer development remains undefined. N4BP1 exhibits the highest expression in head and neck squamous cell carcinoma among all analyzed cancer types. Unlike wild-type mice, N4bp1−/− mice did not develop visible tongue tumor masses in a 4-NQO-induced oral carcinogenesis model. Furthermore, N4bp1−/− mice (86% vs 0%) exhibited significantly prolonged survival compared to wild-type mice within 26 weeks in 4-NQO-induced oral carcinogenesis model. Single-cell profiling demonstrated that N4BP1-deficient epithelial cells arrest at an early stage of cancerous transformation, while wild-type epithelial cells efficiently progress to an advanced stage of cancer. In established human cancer cell lines, N4BP1 also plays a crucial role in proliferation, migration, colony formation, and in vivo growth. Transcriptome profiling identified CCL2 and GM-CSF as downstream targets of N4BP1 in oral cancer. Apart from its intrinsic role in cancer cells, N4BP1-deficient cancer cells induce the differentiation of macrophages into the M1 phenotype. In N4BP1-deficient tissues, CCL2 and GM-CSF were significantly increased, accompanied by the accumulation of M1 macrophages and neutrophils. Our results demonstrate that N4BP1 is an essential gene in tongue cancer development. N4BP1 not only drives cancer cell evolution but also establishes an immune-suppressive microenvironment. N4BP1 is an endoribonuclease that specifically regulates a subset of mRNA targets (including CCL2 and GM-CSF) and plays an essential role in oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/453fe4fbe3e2a6087004d4f6f19727a6b189bb9b" target='_blank'>
              N4BP1 is essential for the development of oral cancer via controlling both cancer cells and immune microenvironment
              </a>
            </td>
          <td>
            Yihua Song, Rong Sun, Jie Ji, Wen Zheng, Yanli Li, Xiaohong Guo, Liuting Chen, Yuanyuan Wu, Miaomiao Chen, Xingmei Feng, Mingbing Xiao, Renfang Mao, Yihui Fan
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Due to its influence in transcriptional potential of genes, genetic regulation by means of epigenetic  mechanisms  is  essential for normal growth and development. In mammary cancer, epigenetic modifications play a key role for its development and progression. In early carcinogenesis stages, due to genetic alterations or environmental factors, chromatin structure alterations due to DNA methylation and post-translational modifications of DNA-bound proteins may appear. As with other types of tumor, genome-wide hypomethylation and hypermethylation of specific genes, particularly in CpG islands that normally are not methylated, are observed. In order to compare global DNA methylation levels between tumor tissue and normal mammary tissue, we studied 11 intact female dogs with mammary tumors. Both types of tissue were collected during surgery, with subsequent clinical staging and histopathological classification of tumors. For each animal, DNA was extracted from paired samples of tumor tissue and normal mammary tissue. Global genomic methylation levels were calculated by relative quantitation of 5-methyl-2’-deoxycytidine (5mdC) with HPLC. Results showed that tumoral tissue had a global DNA hypomethylation when compared with normal mammary tissue (P < 0.05). This difference was greater in high histopathological grade tumors, characterized by their aggressive clinical behavior and high metastatic rate. These findings underscore the importance of additional studies in this line of research, with greater sample sizes. In the future, global DNA methylation may be used as a prognosis biomarker for mammary cancer in dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927c93e449341b53ac59cfa7b6470adfd1ef1f9b" target='_blank'>
              Global DNA hypomethylation in canine mammary tumors
              </a>
            </td>
          <td>
            Alicia Decuadro, María del Carmen Montenegro, Silvia Llambí, N. Balemian, M. Cappetta
          </td>
          <td>2025-12-04</td>
          <td>Revista Científica de la Facultad de Ciencias Veterinarias</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Sequestosome 1 (p62/SQSTM1) is a multifunctional adaptor protein whose dysregulation promotes tumorigenesis through autophagy, metabolic reprogramming, and immune modulation. However, its role across cancer types and impact on the tumor immune microenvironment remain poorly defined. Here, we performed a comprehensive pan-cancer analysis to delineate the molecular and immunological landscape of p62 across human malignancies. TCGA analysis revealed rare mutations and limited prognostic impact of genomic alterations but marked transcriptomic upregulation in LIHC, LUAD, BRCA, KIRP, KICH, KIRC, and READ, correlating with poor survival, advanced stage, and higher T classification, particularly in BRCA and LUAD. Pathway analysis showed strong positive associations between p62 and metabolic adaptation and stress tolerance pathways, including oxidative phosphorylation, reactive oxygen species, and DNA repair, in most cancers. Notably, high p62 expression inversely correlated with immune cell infiltration in most epithelial cancers, such as BRCA, COAD, ESCA, HNSC, KIRC, LIHC, LUAD, LUSC, PAAD, PRAD, READ, THCA, while coinciding with elevated expression of immunosuppressive checkpoints such as PD-L1, B7-H3, EBAG9, PVR, and TGFB1, supporting a link between p62-driven metabolic remodeling and an immune-excluded tumor microenvironment. In contrast, GBM, LGG, OV, SARC, and TGCT showed positive correlations between p62 and immunoscore, with enrichment of interferon and pro-inflammatory pathways, reflecting a distinct immune-activated phenotype. Collectively, these findings identify p62 as a central regulator of tumor metabolism and immunity, suggesting that its context-dependent activity may dictate the balance between immune suppression and activation across cancers, and highlight two natural-product-derived PB1 inhibitors (ZINC70669789 and ZINC08877690) as promising candidates for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716e845c24cbd3bb41e4b234ab94e5c67dc1b5b3" target='_blank'>
              Comprehensive pan-cancer analysis of p62 reveals its contribution to shaping tumor microenvironment and anti-tumor immunity
              </a>
            </td>
          <td>
            Zahra Nayeri, Elahe Tavakol, M. Rahmati, Marco Cordani, M. Djavaheri-Mergny, Vahid Shariati, M. Moosavi
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>34</td>
        </tr>

        <tr id="DNA oxidation is one of the main types of damage to the genetic material of living organisms. Of the many dozens of oxidative lesions, the most abundant is 8-oxoguanine (8-oxoG), a premutagenic base that leads to G→T transversions during replication. Double-stranded DNA can conduct holes through the π system of stacked nucleobases. Such electron vacancies are ultimately localized at the 5’-terminal nucleotides of polyguanine runs (G-runs), making these positions characteristic sites of 8-oxoG formation. While such properties of G-runs have been studied in vitro at the level of chemical reactivity, the extent to which they can influence mutagenesis spectra in vivo remains unclear. Here, we have analyzed the nucleotide context of G-runs in a representative set of 62 high-quality prokaryotic genomes and in the human telomere-to-telomere genome. G-runs were, on average, shorter than polyadenine runs (A- runs), and the probability of a G-run being elongated by one nucleotide is lower than in the case of A-runs. The representation of T in the position 5’-flanking G-runs is increased, especially in organisms with aerobic metabolism, which is consistent with the model of preferential G→T substitutions at the 5’-position with 8-oxoG as a precursor. Conversely, the frequency of G and C is increased and the frequency of T is decreased in the position 5’-flanking A- runs. A biphasic pattern of G-run expansion is observed in the human genome: the probability of sequences longer than 8–9 nucleotides being elongated by one nucleotide increases significantly. An increased representation of C in the 5’-flanking position to long G-runs was found, together with an elevated frequency of 5’-G→A substitutions in telomere repeats. This may indicate the existence of mutagenic processes whose mechanism has not yet been characterized but may be associated with DNA polymerase errors during replication of the products of further oxidation of 8-oxoG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83075f585c67b259c09dcc5cab31e9fb64185e60" target='_blank'>
              DNA damage reflected in the evolution of G-runs in genomes
              </a>
            </td>
          <td>
            I. Grin, D. Zharkov
          </td>
          <td>2025-12-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that dictate complex transcriptional repertoire. Herpes viruses exhibit programmed latent and lytic gene expression depending on the infected tissue and physiological state. Previously, using a systematic functional assay, we identified six enhancers within the genome of Kaposi’s sarcoma-associated herpesvirus (KSHV). In this study, we present a natural language processing model (NLP)-based tool, ENHAvir, that is trained with these six enhancers and non-enhancer control sequences from the KSHV genome. ENHAvir identifies known enhancers and predicted novel enhancer elements in human herpesviruses. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV is confirmed in an enhancer reporter assay. The terminal repeats of all the herpes viruses also serve as a strong enhancer. Comparing herpesvirus enhancers with human enhancers reveals conserved enhancer signatures and the involvement of Alu elements. Here, we present an AI tool that successfully predicts enhancers in both viral and human genomes. Artificial intelligence (AI) can detect and predict patterns that are hidden from the human eye and from conventional homology-detection tools. Enhancers are distal DNA cis-regulatory elements that regulate complex transcriptional repertoire. A natural language processing (NLP) model was trained using only six enhancer sequences from the Kaposi’s sarcoma-associated herpesvirus (KSHV) genome. This tool, termed ENHAvir, can identify known enhancers and predict novel enhancer elements in other herpesviruses, different viruses, and the human genome. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV was confirmed experimentally, enabling the creation of a comprehensive enhancer map of human herpesviruses. All human herpesviruses contain terminal repeats (TRs), which play important roles in cleaving the viral genome into genome-size units, genome encapsidation, and genome circularization following entry into the nucleus of a newly infected cell. This study adds another role for the TR of all human herpesviruses, a strong enhancer with the features of a “viral super enhancer”. Comparing herpesvirus enhancers with human enhancers revealed conserved enhancer signatures and the involvement of Alu elements. Here, an AI tool is presented that successfully predicts enhancers in both viral and human genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980482a475765af78491bb69bee337aba4e9cbe1" target='_blank'>
              Discovering the complete enhancer map of human herpesviruses using a natural language processing model
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Deepanway Ghosal, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-12-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures comprising an RNA/DNA hybrid and a displaced single-stranded DNA. While transient R-loop formation is essential for various physiological processes, their persistent accumulation leads to genomic instability. Cancer cells exhibit elevated R-loop levels due to hypertranscription, replication stress, and impaired DNA repair pathways. In this review, we provide a comprehensive overview of the molecular machinery that resolves R-loops, including chromatin remodelers, transcriptional regulators, nucleases, and helicases. We also highlight the emerging roles of long noncoding RNAs (lncRNAs) in modulating R-loop dynamics and explore how these RNA-based mechanisms cooperate with canonical resolution pathways. Finally, we explore the potential of targeting R-loop regulatory networks as a novel therapeutic strategy in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7591e333b5ca8a5feff5d45b0c90d90e304373" target='_blank'>
              Regulation of R-Loop Dynamics by Proteins and Long Noncoding RNAs: An Emerging Paradigm for Cancer Treatment.
              </a>
            </td>
          <td>
            Miho M. Suzuki, Keiko Shinjo, Tatsunori Nishimura, Yutaka Kondo
          </td>
          <td>2025-11-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs), constituting approximately 8% of the human genome, represent genomic remnants of ancestral retroviral infections that colonized the germline through evolutionary processes. While most HERVs remain epigenetically silenced, their reactivation through environmental stimuli or epigenetic dysregulation enables participation in oncogenesis via viral mimicry, immunomodulation, and insertional mutagenesis. Substantial evidence now implicates aberrant HERVs activity across urologic malignancies—including prostate cancer, renal cell carcinoma (RCC), bladder cancer, and testicular germ cell tumors—where cancer-type-specific mechanisms drive tumor development and progression. These encompass androgen-responsive HERV-K activation in prostate malignancies, hypoxia-inducible factor-mediated ERV immunogenicity in RCC, HERV-derived microRNA silencing of tumor suppressors in bladder cancer, and DNA hypomethylation-associated HERV expression in testicular germ cell tumors. This review synthesizes fundamental HERV biology with recent advances in their diagnostic and therapeutic applications for urologic neoplasms. Key clinical translations include ERV-based stratification models predicting immune checkpoint inhibitor response in metastatic RCC, HERV-E-targeted adoptive T cell therapies, and noncoding RNA biomarkers for early bladder cancer detection. We further discuss unresolved mechanistic paradoxes such as contradictory prognostic associations between HERV superfamily expression and PBRM1 inactivation in RCC, concluding with priorities for future research: validating HERV-derived neoantigens in immunotherapy platforms, optimizing epigenetic priming strategies to enhance viral mimicry effects, and establishing standardized HERV signatures as clinical biomarkers through multi-institutional cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38383e2f180ac1b254c2d12832458ab1cd318585" target='_blank'>
              The expression, regulation, and function of human endogenous retroviruses in genitourinary cancers
              </a>
            </td>
          <td>
            Wenjie Ma, Chencheng Ji, Abudukelimu Abudushataer, Ning Liu, Tao Xu, Kunlun Zhao, Yiguan Qian, Paerhati Tuerxun, Xiaotian Jiang, Zhongli Xiong, Min Wang, Ruipeng Jia, Zheng Xu, Yang Li, Yu-Zheng Ge
          </td>
          <td>2025-11-28</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Meningiomas are the most common primary tumors of the central nervous system and are typically treated with surgery or radiation, as targeted therapies remain limited. Despite extensive study, seventeen percent of meningiomas lack known genetic drivers. Our analysis of meningiomas without driver mutations or major chromosomal alterations identifies a subset with recurrent genomic rearrangements involving the FOS and FOSB genes. These tumors exhibit elevated FOS/FOSB protein levels and retain meningothelial lineage. Here we show that FOS/FOSB fusion-positive meningiomas represent a distinct molecular subgroup, defined by unique gene expression patterns, including activation of AP-1 target genes and signatures resembling preadipocyte-like and mast cell–associated profiles. Clinically, these tumors display low-grade behavior and DNA methylation profiles consistent with benign subtypes. Our findings identify a meningioma subgroup with distinct genetic, transcriptomic, and clinical features, expanding the molecular classification of meningiomas and opening new avenues for targeted treatment strategies. Approximately 17% of meningiomas remain genomically uncharacterized. Here, the authors analyze 105 meningiomas without known driver mutations or significant copy number alterations and identify a subgroup of meningiomas, defined by FOS/FOSB gene fusions with distinctive transcriptomic and histopathological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/848d67ce829255f08e0caafaf14bc914e2cca146" target='_blank'>
              Subgroup of meningiomas involving FOS and FOSB gene fusions
              </a>
            </td>
          <td>
            K. Yalcin, Hasan Alanya, B. Gultekin, Diego Samper Figuera, Tanyeri Barak, D. Miyagishima, Mark W. Youngblood, Mateo Gomez Hjerthen, Amos Brooks, Natalia Samuel Lopez, Joseph O’Brien, M. M. Chavez, Alper Dincer, Hao Wu, S. Omay, Katsuhito Yasuno, K. Bilguvar, A. Ercan-Sencicek, Ketu Mishra-Gorur, D. McGuone, Jennifer Moliterno, Murat Gunel, E. Erson-Omay
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W Snyder, Katherine Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c6e2a39a6240d55a8cd9334672f5c4459bbc0c" target='_blank'>
              A compendium of chromatin interaction maps in the Giant Panda genome.
              </a>
            </td>
          <td>
            Pengliang Liu, Jiaman Zhang, Kailai Cai, Juan Wang, Hong Liu, Liang Zhang, Wei Xu, Yuliang Liu, Fujun Shen, Rong Hou, Yan Li
          </td>
          <td>2026-01-13</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Complex chromosomal rearrangement (CCR) refers to a structural rearrangement involving at least two chromosomes or a minimum of three breakpoints. CCR may lead to intellectual disability, structural anomalies, infertility, and recurrent miscarriages. Chromosome karyotyping and chromosomal microarray analysis (CMA) are unable to detect complex chromosomal rearrangements. As multiple diagnostic approaches are available in clinical practice for detecting chromosomal structural abnormalities and copy number variations—each with its own advantages and limitations—selecting the appropriate testing method is crucial for effective clinical management. Optical genome mapping (OGM) is an advanced genomic technology that utilizes ultra-long single-molecule analysis to comprehensively detect chromosomal aberrations and structural variants at high resolution. Amniocentesis was performed for a 36-year-old multipara (advanced maternal age), with subsequent comprehensive fetal genetic analysis including chromosome karyotyping, CMA, and OGM. Family members underwent peripheral blood karyotyping and OGM. The fetal karyotype derived from amniotic fluid was 46,XN,?ins(18)(q21.2;p11.31p11.2). CMA demonstrated duplications of four segments and a deletion of one segment on chromosome 18. Therefore, OGM was performed on the fetal and family members to further elucidate the chromosomal structure. The fetus has derived CCRs on chromosome 18 of maternal origin. In contrast, both the mother and the second daughter, who carried the identical CCRs, were phenotypically normal. OGM is of significant importance in the diagnosis and characterization of CCRs. OGM plays a critical role in diagnosing complex chromosomal rearrangements and has proven to be invaluable in clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a05870664d5ee26da7943564ddcc7debed72e97b" target='_blank'>
              Optical genome mapping of a complex structural rearrangement family line on chromosome 18
              </a>
            </td>
          <td>
            Liyi Cai, Yuying Jiang, Na Zhang, Xinying Chen
          </td>
          <td>2025-12-01</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 In Non-Small Cell Lung Cancer (NSCLC), low-fidelity transcription mechanisms such as ecDNA (extra-chromosomal DNA) are an increasingly relevant aspect of progression and prognosis, warranting comprehensive investigation into stability and prevalence of chimeric RNA transcripts. Here, we describe global distribution of fusion transcripts in encounter-matched primary tumor and lymph node metastases and their prognostic implications.



 RNAseq was performed on resected FFPE (encounter-matched primary and lymph nodes) from 121 stage IIB-IIIA (T1-4N1-2M0) NSCLC patients, in addition to 693 Stage IA1-IB cases (T1a-2aN0M0). STAR and STAR-Fusion were used to align data to the GRCh37 genome. Differential gene expression was based on specimen source (primary or lymph node). Aggregate analyses were also performed, including determination of a fusion burden metric, assessing the relationship between primary tumors, lymph node metastases, and recurrence. Deconvolution of Tumor Microenvironment was performed via Quantiseq.



 Comparing primary tumors to paired lymph node metastases, 7,930 ensembl IDs were differentially-expressed at FDR < 0.001 (n=101 pairs). A total of 613 fusions (685 unique transcripts) were detected from paired primaries and lymph node metastases (n = 242). Frequency of specific gene fusions in primaries ranged from 0-0.963 (median 0.019), compared to 0-0.870 (median 0.0093) in LN metastases (p=1.87×10-4). We observed fusion of TULP4-RP11-732M18.3 as pervasive in NSCLC, occurring in 84-87% of primary tumors and lymph nodes, with a 91% retention rate. Conversely, fusions involving CCDC7 or SEPT14 were preferentially observed in metastases, where 82% of CCDC7 fusions and 75% of SEPT14 fusions were not observed in the primary tumor despite occurring at high frequency (66% and 18.5%, respectively) among LN metastases. Interestingly, median fusion burden (unique gene-gene fusions) for primaries was 14 compared to 11 (p=2.06×10-7) in LN metastases (median change -4). Among N0 cases that did not receive adjuvant therapy, median fusion burden in non-recurrent tumors was 15 (n=570), compared to 13 in those that recurred over 3-60 months (n=123; p = 0.04). Interestingly, there was no significant difference in fusion burden between recurrent cases and primary tumors with LN metastases (n = 108). For both recurrent cases and LN-paired primaries, the difference in fusion burden compared to LN metastases (n = 110) was significant (p=2.10×10-4 and 2.06×10-7). A subset of LN metastases (n=27) increased in fusion burden relative to the primary tumor; these primaries uniquely exhibited 33% decrease in regulatory T cell content (p=0.008), 20% decrease in M2 macrophage content (p = 0.01 with M1 macrophages static), and 40% decrease in B-cell content (p=0.05).



 Fusion transcripts are pervasive in RNAseq data, suggesting a broader and more variable landscape than accounted for by primary tumors alone. Concurrent ongoing studies explore the role of ecDNA in fusion burden changes related to NSCLC progression and metastasis.



 Kelly M. Cagin, Keri L. Denson, Sierra R. Broad, Wara M. Naeem, Arsalan M. Khan, Michael M. Liptay, Christopher W. Seder, Jeffrey A. Borgia. A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abeb8dbe1ed35e15732a9cc5bda99a8030ed67" target='_blank'>
              Abstract B027: A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs
              </a>
            </td>
          <td>
            K. Cagin, Keri L. Denson, Sierra R. Broad, Wara Naeem, Arsalan M. Khan, Michael M. Liptay, C.W. Seder, Jeffrey Borgia
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0bbb633caa4247cf44deccb24c197d220a3f9fb" target='_blank'>
              Mechanics-dependent Global Nuclear Eviction and Site-Specific Recruitment of YAP Regulates DNA Damage Responses
              </a>
            </td>
          <td>
            Shantam Yagnik, Aprotim Mazumder
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="How cells repair oxidative damage to DNA has been studied for over 60 years. Recent evidence confirms that the base excision repair (BER) machinery not only acts to restore an intact double DNA helix by replacing oxidized bases, but under some circumstances, BER goes awry, generating double‐strand breaks and provoking chromosome fragmentation. This fragmentation can lead to extensive genomic rearrangements that correlate with oncogenesis. Whether the BER factors suppress or promote DNA damage depends on multiple parameters: the nature of the damage, the clustering of modified bases, the pathway of BER chosen, and chromatin remodelers. Recent data leading to this unexpected role for BER are reviewed here.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20c4fda434c580a7e6fa7184e3c278d6397c746" target='_blank'>
              The Double Face of Base Excision Repair: Preventing and Triggering Double‐Strand Breaks
              </a>
            </td>
          <td>
            Susan M. Gasser
          </td>
          <td>2025-11-21</td>
          <td>Bioessays</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ABSTRACT Colon cancer (CC) is a malignancy with high global incidence and mortality, and elucidating its underlying molecular mechanisms is critical for improving prognostic assessment and therapeutic strategies. In this study, transcriptomic data from a large cohort of CC samples and a limited number of normal controls from the TCGA database were used to construct a multigene prognostic risk model using univariate Cox and least absolute shrinkage and selection operator (LASSO) regression analyses. The expression of key prognostic genes, including immunoglobulin superfamily member 9 (IGSF9), was further validated in CC tissues by PCR and Western blotting. Functional assays were performed in HCT116 cells to investigate the biological effects of IGSF9 overexpression and its regulatory relationship with p53. The prognostic model identified IGSF9 as a gene significantly associated with patient survival. Although IGSF9 expression was reduced at both the mRNA and protein levels in CC tissues, its overexpression in vitro markedly promoted apoptosis, alleviated DNA damage, and suppressed cell migration and invasion. Notably, silencing of p53 partially reversed the tumor‐suppressive effects induced by IGSF9 overexpression, indicating that IGSF9 exerts its biological functions in a p53‐dependent manner. Collectively, these findings demonstrate that IGSF9 acts as a tumor suppressor in colorectal cancer and regulates DNA damage responses and apoptosis through a mechanism partially dependent on p53, highlighting its potential value as a prognostic biomarker and therapeutic target in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ff548ce8387a244b0397236c92065d62ddf1f1" target='_blank'>
              P53 Inhibition Diminishes IGSF9 Gene Activity to Promote DNA Repair and Exacerbate the Progression of Colon Cancer
              </a>
            </td>
          <td>
            Huan-Yu Zhang, Dan Tian, Wan-Fu Zhang, Ying-Hui Zhang, Jia-Li Feng, Juan Sheng, Xue-Qin Shang
          </td>
          <td>2026-01-01</td>
          <td>Journal of Biochemical and Molecular Toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898be89e91659bf810fb7787929dbf8540600811" target='_blank'>
              Advances in functional mechanisms of genomic G-quadruplex structures in transcriptional regulation.
              </a>
            </td>
          <td>
            Zhen-Zhen He, Xiao-Feng Chen, Yue Hou, Tie-Lin Yang, Bo Yang, Yan Guo
          </td>
          <td>2025-12-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA methylation is a most heritable epigenetic modification. Being a major vegetable crop, pepper (Capsicum spp.) possesses an over 3 Gb genome populated with TEs. This indicates a rich reservoir of epigenetic regulatory mechanisms. However, this large and complex genome renders the study of DNA methylation unaffordable and technically challenging. In this study, we analyzed DNA methylome in Capsicum spp., with a focus on C. annuum ST-8. We found that the genomes of Capsicum spp. are heavily methylated, particularly in the non-CG contexts. This is true when comparing to wheat, whose genome is over 16 Gb, containing over 80% TEs and repeats. Interestingly, we observed genic non-CG methylation and found that it is likely maintained by the CMTs, instead of RdDM. Overall, there is a negative relationship between gene expression and H3K9me2, and a positive relationship between genic non-CG methylation and H3K9me2, despite that genes without genic CHH methylation also possess some H3K9me2. Finally, we performed salt stress treatment with and without priming, and profiled active chromatin features as well as transcriptomes. We found that regardless of the environmental stimuli and developmental stages, the overall negative relationship between transcription and H3K9me2 is stably maintained. Altogether, our study revealed features of DNA methylation in ST-8 and we suggest that these features are likely common in Capsicum spp.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee5cf6576edac68d63b332c1994530f8d8f03ba" target='_blank'>
              Features of DNA methylome in Capsicum spp.
              </a>
            </td>
          <td>
            Ye Liu, Yingjie Mi, Manru Song, Hongji Yang, Lulu Liu, Qian Yin, Xueyan Yao, Guorong Yu, Ying Fu, Guochen Qin, Feng Liu, Xuan Huang, Yan Xue
          </td>
          <td>2025-12-10</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Fusion-positive rhabdomyosarcoma (FP-RMS), an aggressive pediatric soft tissue cancer, is driven by the oncogenic fusion transcription factors PAX3::FOXO1 (P3F) or PAX7::FOXO1. In a subset of FP-RMS cases, amplification of the MIR17HG locus leads to overexpression of the oncogenic miR-17–92 cluster. However, miR-17–92 is also upregulated in a subset of FP-RMS tumors lacking this amplification, suggesting the involvement of alternative regulatory mechanisms. Here, we show that P3F and MYCN cooperatively activate MIR17HG transcription in FP-RMS. CRISPR/Cas9-mediated knockout of P3F or MYCN in FP-RMS cell lines significantly reduced miR-17–92 expression. In a human myoblast model with inducible P3F expression, MYCN alone induced modest MIR17HG transcription, whereas co-expression with P3F led to robust activation of the cluster and acquisition of oncogenic phenotypes. Chromatin immunoprecipitation sequencing (ChIP-seq) revealed a P3F binding site located 1.84 Mb upstream of the MIR17HG promoter. CRISPR-mediated deletion of this site markedly reduced MIR17HG expression and impaired transformation in the myoblast system. Functional inhibition of mature miR-17–92 members using miRNA sponges also suppressed proliferation and transformation in FP-RMS cells. In vitro studies showed that miR-17–92 is necessary but not sufficient for transformation, highlighting its role as a key downstream effector of P3F and MYCN. Together, these findings define a cooperative transcriptional axis where P3F and MYCN activate MIR17HG through a distal DNA binding element, thereby revealing a novel therapeutic vulnerability in FP-RMS.



 Shabir Zargar, Pawam Raut, Hana Kim, Rachel Hoffman, Benjamin Z. Stanton, Frederic G. Barr. Oncogenic MIR17HG Expression Is Transcriptionally Regulated by PAX3::FOXO1 and MYCN in Fusion-Positive Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cec2c5fd391552299369bfa86ed2e2816c5ee0" target='_blank'>
              Abstract B029: Oncogenic MIR17HG Expression Is Transcriptionally Regulated by PAX3::FOXO1 and MYCN in Fusion-Positive Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Shabir Zargar, Pawam Raut, Hana Kim, Rachel A. Hoffman, Benjamin Z. Stanton, Frederic G. Barr
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Three-dimensional genome architecture is essential for gene regulation and cellular homeostasis. Its perturbation underlies pathologies. Advances in chromosome conformation capture technologies, such as Hi-C [1], have enabled researchers to study genome architecture at high resolution, yet the resulting matrices are difficult to explore visually, especially when one wishes to overlay genomic annotations or examine inter-chromosomal interactions. We introduce gghic, an R package that extends the ggplot2 grammar for the publication-quality visualization of chromatin-interaction data. Gghic supplies layers that draw (i) triangular Hi-C heatmaps, (ii) chromatin loops, topologically associating domains, (iii) gene or transcript models, (iv) one-dimensional signal tracks, (v) and multi-way contacts. The package supports Bioconductor classes, including HiCExperiment [2] and GInteractions [3]. It handles data from multiple chromosomes, enabling inspection of inter-chromosomal contacts. We illustrate the utility of gghic with acute lymphoblastic leukemia Hi-C datasets, producing publication-ready figures that reveal translocation breakpoints and higher-order chromatin hubs. By coupling advanced genome-structure analyses with the ggplot2 syntax, gghic makes the exploration and communication of 3D genomics data both intuitive and reproducible. References 1. Belton JM, McCord RP, Gibcus JH et al. ‘Hi-C: a comprehensive technique to capture the conformation of genomes’ Methods 2012;58:268–276. 2. Serizay J, Matthey-Doret C, Bignaud A et al. ‘Orchestrating chromosome conformation capture analysis with Bioconductor’ Nat Commun 2024;15:1072. 3. Lun AT, Perry M, Ing-Simmons E. ‘Infrastructure for genomic interactions: Bioconductor classes for Hi-C, ChIA-PET and related experiments’ F1000Res 2016;5:950.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbff5aed0fd8c28e8521d832d596353743e33b7" target='_blank'>
              Gghic: a versatile R package for exploring and visualizing 3D genome organization
              </a>
            </td>
          <td>
            Minghao Jiang, Duohui Jing, Jason W.H. Wong
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="PURPOSE
HPV integration (HPVint) is associated with carcinogenesis and tumor progression in HPV-associated cancers, including head and neck squamous cell carcinomas (HNSCC). While its impact on human DNA has been well characterized, its relationship with clinical outcomes remains unconfirmed.


EXPERIMENTAL DESIGN
We analyzed HPVint events from 261 HPV-associated HNSCC bulk and single-cell RNA-seq samples from five cohorts, including 62 from a new University of Michigan cohort, and DNA HPVint events from 102 HPV(+) HNSCC participants in two of the cohorts. We investigated the consequences of HPVint both with respect to human and HPV gene expression and clinical outcomes (recurrence and overall survival).


RESULTS
By leveraging this large meta-cohort of HNSCC, we first reveal an oncogenic gene network based on the recurrent HPV integration locations in the human genome and gene expression alterations, highlighting key recurrent and overexpressed genes including NR4A2, CD274, CCER1 and genes from the CAMK and KLF families. We then stratify HPVint-positive participants by risk using HPV RNA features, specifically spliced HPV-human fusion transcripts (E1* integration) and HPV gene expression ratios (HGER), showing that subsets of participants have worse clinical outcomes based on these two candidate biomarkers.


CONCLUSIONS
By focusing on RNA instead of DNA, we expand our understanding of the carcinogenic mechanisms of HPVint, in part addressing the conflicting findings of whether HPVint is associated with aggressive phenotypes and worse clinical consequences and provide potential biomarkers to advance precision oncology in HPV-associated HNSCC. Newly identified genes with recurrent integration events may serve as candidates for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/737e8ec9fc9860af7b63c3f955141de9c82f430b" target='_blank'>
              HPV integration in head and neck cancer: downstream splicing events and expression ratios linked with poor outcomes.
              </a>
            </td>
          <td>
            Shiting Li, Shaomiao Xia, Maria Lawas, Aishani Kulshreshtha, Bailey F. Garb, Sarah E. Soppe, Chamila Perera, Chen Li, Min Liu, Y. X. Lim, Tingting Qin, Joshua D. Welch, N. D’Silva, Laura S Rozek, Maureen A. Sartor
          </td>
          <td>2025-11-21</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Background/Objectives: Despite extensive research on hepatitis B virus (HBV), its post-transcriptional regulatory mechanisms remain incompletely characterized, particularly regarding epitranscriptomic modifications. This study aims to systematically profile the transcriptomic complexity and RNA modification landscape of HBV in hepatocellular carcinoma models. Methods: We transfected PLC/PRF/5 and Huh7 cells with the HBV 1.3-mer WT replicon plasmid, followed by qPCR measurement of viral load. Total nucleic acids extracted from transfected cells underwent nanopore direct RNA sequencing. The complete HBV transcriptome was then analyzed in two established hepatocellular carcinoma cell lines (PLC/PRF/5 and Huh7), with alternative splicing, polyadenylation, and RNA modifications identified through comprehensive bioinformatics analysis. Results: Our analysis revealed substantial transcriptomic diversity, identifying 34 distinct splice variants—including 14 previously unreported isoforms—with cell-type-specific expression patterns. Additionally, we detected 30 high-confidence RNA modification sites across HBV transcripts, with 93% (28 sites) conserved between both cellular environments. Notably, we observed significant intercellular heterogeneity in poly(A) tail length distributions. Conclusions: A comparison of the post-transcriptional processing modifications of HBV in PLC/PRF/5 and Huh7 cells reveals that the former may be better able to mimic the immune evasion mechanisms of chronic HBV infection. In contrast, the longer poly(A) tails present in Huh7 cells facilitate efficient replication, rendering these cells more amenable to the study of HBV transcription and replication mechanisms. These findings comprehensively elucidate the post-transcriptional regulatory mechanisms of hepatitis B virus in different hepatocellular carcinoma cell lines, establishing a critical benchmark for selecting appropriate experimental models in virology research. The identified transcriptomic features may provide new insights for developing antiviral strategies targeting the viral epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fad59ef49abbe4a1a832a5a2fad0a73e6b154a" target='_blank'>
              Comprehensive Transcriptomic and Epitranscriptomic Profiling of Hepatitis B Virus Transcripts in Two Hepatocellular Carcinoma Cell Lines
              </a>
            </td>
          <td>
            Qinan Zhang, Bohan Zhang, Lei Wang, Yongjian Liu, Jingwan Han, Lei Jia, Hanping Li, Xiao-lin Wang, Jingyun Li, Chang-Qing Yu, Lin Li
          </td>
          <td>2025-11-21</td>
          <td>Genes</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DExD-box (DDX) proteins are essential for RNA metabolism and are targets for treatment of cancers and neurodevelopmental disorders. The biochemical mechanisms of many DDX proteins remain unclear, including human DDX52. DDX52 is essential for cell survival and is an emerging biomarker for the onset of metastatic melanoma. In this work, we identified that human DDX52 is an ATP-dependent translocase with 3'-5' polarity, which can unwind DNA duplexes and DNA/RNA hybrids. Further, DDX52 is a nucleic acid annealase, an activity that requires an N-terminal intrinsically disordered protein region. DDX52 becomes hyperactive at DNA annealing if DDX52 helicase activity is inactivated by mutagenesis. Using CRISPR-Cas9 genetic editing, we generated U2OS cell lines heterozygous for DDX52 (DDX52+/-), which exhibit growth defects and impaired cell migration, providing direct support for previous suggestions that DDX52 may promote cancer cell metastasis and C-myc regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f186703aec8952740844b8d3f7ea360a94084d2a" target='_blank'>
              The human DDX52 protein is a nucleic acid helicase and strand annealase that promotes cell migration.
              </a>
            </td>
          <td>
            Ashley J Parkes, Philipp J Springer, E. Bolt
          </td>
          <td>2026-01-01</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Meiotic recombination is initiated by DNA double-strand breaks (DSBs); factors that control DSB frequencies are important to produce viable progeny. In many organisms, the ATM (Tel1) protein kinase prevents excessive meiotic DSBs, especially nearby DSBs on the same chromatid. Normally, two close DSBs are less frequent than expected from independence, a feature called DSB interference, which is lost in tel1Δ mutants. In the fission yeast Schizosaccharomyces pombe, high-level DSB formation depends on linear elements, Hop1, and meiotic cohesin complexes; we show here that these complexes impart competition between nearby DSB sites. When these complexes are impaired, Tel1 substantially represses DSB formation, and in its absence, two close DSBs on the same chromatid occur frequently and manifest high negative interference. After mitotic DNA damage, the conserved Mre11–Rad50–Nbs1 (MRN) complex is required for DNA resection, and the Tel1 kinase activity is needed to complete DSB repair. We found that during meiosis mre11Δ and rad50Δ mutants, like tel1Δ mutants, lack DSB interference and display highly negative DSB interference in meiotic complex mutants. Thus, MRN at a DSB site appears critical for Tel1 function in meiosis and reveals a complex interplay of positive and negative factors controlling meiotic DSB formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee1a2d4929cc57ee0b0191527c1dc8fe91c21069" target='_blank'>
              Mutual, spatially limited control of meiotic DNA break formation by Mre11–Rad50–Nbs1 DNA repair complex and Tel1 (ATM) protein kinase
              </a>
            </td>
          <td>
            Randy W. Hyppa, Gerald R. Smith
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c2bcfd8cc843d79fc1979b529c65ef07bebf80" target='_blank'>
              Targeted suppression of tumor growth by CREPT promoter-driven diphtheria toxin fragment A
              </a>
            </td>
          <td>
            Jingya Li, Mengdi Li, Wenchen Wang, Guancheng Jiang, Yinyin Wang, Jianqiu Sheng, Zhijie Chang, Jianqiu Sheng, Ming-yang Li, Fangli Ren
          </td>
          <td>2025-11-22</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The maintenance of genome stability requires efficient leading strand synthesis by DNA Polymerase Epsilon (Polε). By performing CRISPR genetic screens in cells lacking the POLE4 subunit of Polε we define a genetic map of the factors required to support Polε function in the absence of its accessory subunits. A set of genes involved in iron metabolism emerge as required to sustain Iron Sulphur Cluster (ISC)-dependent Polε activity. We then dissect a synthetic lethal interaction between POLE3-POLE4 and the CHTF18-RFC2/5 complex. By combining cell biology, structural modelling and biochemistry, we define the existence of two tiers of regulation of Polε processivity: leading strand-specific loading of PCNA by CHTF18-RFC2/5 and “gripping” of newly synthesised dsDNA by POLE3-POLE4. The combined loss of these functions is incompatible with leading strand synthesis and viability. In summary, we describe the biochemical basis of human leading strand synthesis and the consequence of its dysfunction in genome stability. How DNA Polymerase Epsilon accomplishes continuous leading strand synthesis during DNA replication is not understood. Here, the authors describe a two tiers mechanism required to sustain Pol Epsilon processivity: CHTF18-dependent loading of PCNA at leading strand and dsDNA binding by its POLE3-POLE4 subunits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9c13b963d3208ae6cad837c1eaa9236a2286b16" target='_blank'>
              The genetic and biochemical basis of human leading strand synthesis
              </a>
            </td>
          <td>
            Alessandro Agnarelli, Lauryn Buckley-Benbow, Meryem Ozgencil, Melanie Lad, K. K. Ampah, Alex Kalinka, Ondrej Belan, Sarah Maslen, Mark Skehel, David Walter, Matthew Day, Roberto Bellelli
          </td>
          <td>2025-12-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background and Objectives: Cervical adenocarcinoma (CAC) is a histologically distinct subtype of cervical cancer with a rising incidence in many regions. While the roles of key driver mutations are known, a comprehensive understanding of its genomic landscape, particularly variations across different populations and tumor stages, remains incomplete. This study aims to characterize the somatic genomic landscape of CAC by identifying recurrent mutations, copy number alterations (CNAs), and patterns of co-occurrence, with a focus on variations across racial groups and between primary and metastatic tumors. Materials and Methods: We conducted a comprehensive genomic analysis of 102 tumor samples from 99 patients diagnosed with cervical adenocarcinoma using data from the American Association for Cancer Research (AACR) Project Genomics Evidence Neoplasia Information Exchange (GENIE) database. Results: The most frequently mutated genes were PIK3CA (25.5%), TP53 (21.6%), ARID1A (20.6%), and KRAS (16.7%). Significant amplification of ERBB2 was also observed (n = 3; 4.83%). Our analysis revealed notable genomic disparities across racial groups, with TP53 mutations being significantly more frequent in White patients compared to Asian and Black patients (p = 0.0236). Furthermore, we identified significant co-occurrence between mutations in KRAS and MSH2 (p = 0.011) as well as ATM and STK11 (p = 0.037). In comparing tumor types, mutations in BCL6 were found to be significantly enriched in metastatic samples. Conclusions: This study validates the primary drivers of cervical adenocarcinoma and reveals novel findings, including notable racial disparities in TP53 mutation frequency and unique patterns of co-occurring mutations. These findings highlight the genomic heterogeneity of the disease and suggest that ancestry and tumor evolution may influence its molecular pathogenesis, offering potential avenues for the development of targeted therapies and personalized biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3066f0da5ff1e91963618a1014e74836869aa1ba" target='_blank'>
              Comprehensive Genomic Characterization of 102 Cervical Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Gejla Toromani, Grace S. Saglimbeni, Bhanu Surabi Upadhyayula, Eugene Manu, Tyson J. Morris, Beau Hsia, A. Tauseef
          </td>
          <td>2026-01-07</td>
          <td>Medicina</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Retroviruses are exclusive group of positive-sense RNA viruses defined by their ability to reverse transcribe their RNA genome and integrate it into the host’s chromosomal DNA. This distinctive replication strategy enables persistent infection and has profoundly shaped our understanding of molecular biology, gene regulation, and evolution. Retroviruses have contributed to landmark discoveries, including the identification of oncogenes, mechanisms of transcriptional control, and the development of gene therapy vectors. This review provides an updated overview of retroviral molecular biology, emphasizing the coordinated steps of the viral life cycle and emerging insights that are reshaping classical models. It explores virion structure, genome organization, and the interplay of cis-acting sequences and trans-acting factors that govern replication. Special focus is given to recent advances in understanding nuclear trafficking of capsids, spatial dynamics of reverse transcription and integration leading to provirus formation, RNA nuclear export, and selective genome packaging. The structural and functional roles of viral proteins, particularly Gag, are discussed in the context of assembly and maturation. By integrating foundational concepts with new discoveries, this review highlights the molecular sophistication of retroviral replication and identifies outstanding questions that guide future research, with implications extending to antiviral strategies, gene therapy, cancer biology, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2158d0cf0754098c7826e6edf394a24a8aa054" target='_blank'>
              Beyond reverse transcription: molecular mechanisms and emerging paradigms in retroviral replication
              </a>
            </td>
          <td>
            Mohammad Abdullah Jehad, L. Ali, Vineeta N Pillai, Suresha G. Prabhu, Farah Mustafa, Tahir A. Rizvi
          </td>
          <td>2025-12-26</td>
          <td>FEMS Microbiology Reviews</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal liver metastasis (CRLM) occurs frequently in patients with colorectal cancer (CRC). Methionine adenosyltransferase (MAT) catalyzes the formation of S-adenosylmethionine, the principal methyl donor. MAT1A (encodes MATα1) is expressed mainly in normal adult liver, whereas MAT2A (encodes MATα2) is expressed in all extrahepatic tissues. MAT1A is a major defense against CRLM as loss of Mat1a sensitizes the liver to CRLM. In contrast, MAT2A is overexpressed in CRC and promotes oncogenicity. Here, we sought to determine if CRCs secrete MATα2 and if this influences CRLM. Our study included human hepatocytes, human CRC cells, extracellular vesicle (EV) isolation, chromatin immunoprecipitation (ChIP), ChIP-seq, promoter activity assays, proliferation, migration, and invasion assays, western blotting, immunohistochemistry and immunofluorescence. We confirmed some of the findings using human hepatocyte spheroids, CRLM and normal liver tissue array, and plasma samples. CRCs secrete MATα2 in free but truncated form (MATα2-t) and intact within EVs (EV-MATα2). EV-MATα2 can be internalized by human hepatocytes and CRCs, found within the nucleus, which then binds to MAT1A and MAT2A promoters on ChIP to lower and increase MAT1A and MAT2A promoter activities, respectively. In human CRLM samples, hepatocytes in nontumor regions express lower MATα1 but higher MATα2 as compared to normal liver. Treating RKO cells with EVs released from RKO cells overexpressing MAT2A promoted cell proliferation, migration, and invasion. MATα2-t was detected at a higher level in media from colon, pancreatic, and prostate cancer cell lines than corresponding normal epithelial cells as well as in the plasma of CRC patients as compared to healthy controls. RKO cells treated with MATα2-t activated focal adhesion kinase (FAK), an important kinase for cancer cell evasion of apoptosis. Conversely, treatment with MATα2 neutralizing antibody inhibited FAK and induced apoptosis. CRC cells secrete both MATα2 within EVs and free MATα2-t. EV-MATα2 can be internalized and act as a transcription factor to lower hepatocytes’ MAT1A, the major defense against CRLM, while promoting CRC oncogenicity. Freely released MATα2-t acts as a ligand in an autocrine fashion to activate FAK, which is essential for CRC survival. Taken together, secreted MATα2 plays an essential role in promoting CRLM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d717a5a8641241a8445c43509a6ec64d1a59947" target='_blank'>
              A novel role of secreted methionine adenosyltransferase α2 in colorectal liver metastases
              </a>
            </td>
          <td>
            Monica Justo, Youngyi Lim, Heping Yang, Andrea Floris, Swati Chandla, Manisha Dagar, Alexandra Gangi, E. Posadas, M. Edderkaoui, Stephen J. Pandol, Neil A. Bhowmick, M. Tomasi, Shelly C. Lu
          </td>
          <td>2025-12-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Aside from standard surgical resection followed by chemo/radiation therapy, there is a dearth of effective therapies for tumors of the central nervous system (CNS) despite increasing understanding of the molecular mechanisms driving oncogenesis and progression. Antisense oligonucleotides (ASOs) have emerged as a powerful method to target disease-causing transcripts through sequence-specific translational blocking or degradation. However, while it is known that oncogenic fusions drive some CNS tumors, few studies have investigated and characterized the landscape of oncogenic fusions and variation in breakpoints, pertinent for the translation of ASOs into a viable therapy.



 We performed retrospective analysis of CNS tumors from 1690 patients profiled at the Children’s Hospital of Philadelphia as part of routine clinical molecular workup. Additionally, we have developed an ASO-based therapeutic platform to design ASOs which target the fusion junctions and validate their efficacy in eliminating the targeted fusion transcripts.



 Our cohort spanned both pediatric (980/1690) and adult (710/1690) ages (range = 0 - 91) and histologic subtypes. In total, 21% of CNS tumor patients (354/1690) harbored an oncogenic fusion; Grade 1 = 38% (218/567), Grade 2 = 6% (12/211), Grade 3 = 12% (22/176), and Grade 4 = 13% (75/570). Fusions were most common in glial neoplasms (27%, 328/1213) followed by ependymal tumors driven by ZFTA rearranged supratentorial ependymoma (13%, 20/150), and mesenchymal tumors (28%, 5/18). The most common fusions included BRAF in Grade 1 tumors and MET and EGFR in Grade 4 tumors. NTRK1/2/3 fusions, which have promiscuous partners spanned all grades, and FGFR1/2/3 fusions were only found in Grade 1 or 4 tumors, most often fused to TACC1 or TACC3. BRAF, NTRK2, and MET fusions had the highest number of unique partners. Investigating unique transcripts, nearly all fusions had conserved breakpoints except for KIAA1549::BRAF fusions, which had 15 unique transcripts. Interestingly, while ZFTA::RELA fusions had relatively conserved breakpoints, all tumors expressed at least 2 unique transcripts (1 major), suggesting co-existence of different chimeric isoforms. MYB::QKI fusions, identified in low-grade gliomas, was selected as a pilot target to evaluate the ASO-based therapeutic platform. A list of five ASOs were tested in mammalian cells expressing MYB::QKI fusion reporters. The most effective ASO was found to reduce MYB-QKI reporter fluorescent signal by ∼83% and fusion transcript level by ∼63% compared to the negative ASO, as measured by cellular imaging and RT-qPCR, respectively.



 In the current landscape, where therapeutic options for fusion-driven tumors in the CNS are limited, this ASO-based therapeutic platform presents a new avenue for developing personalized nucleic acid-based therapies aimed at improving clinical outcomes for individuals with CNS tumors.



 Hyojeong Hwang, Jinhua Wu, Jeffery Schubert, Meha Patel, Natali Naveh, Netta Golenberg, Minjie Luo, Yiming Zhong, Lea F Surrey, Marilyn M Li, Monica Pomaville, Kathy Fange Liu, Derek Wong. Transcriptomic landscape of fusion-driven CNS tumors for the design and validation of junction-specific antisense oligonucleotide-mediated targeting of fusion oncogenes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5efa3b53836a0546236feafdc56a86dd9579377" target='_blank'>
              Abstract A006: Transcriptomic landscape of fusion-driven CNS tumors for the design and validation of junction-specific antisense oligonucleotide-mediated targeting of fusion oncogenes
              </a>
            </td>
          <td>
            Hyojeong Hwang, Jinhua Wu, Jeffery Schubert, Meha Patel, N. Naveh, Netta Golenberg, Minjie Luo, Yiming Zhong, Lea F. Surrey, Marilyn M. Li, Monica Pomaville, Kathy Fange Liu, Derek Wong
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Maintaining genome integrity in germ cells is crucial for fertility and species survival. However, the DNA repair mechanisms that sustain genome integrity in primordial germ cells (PGCs), which cope with high levels of replication stress, remain largely unknown. While the TatD family of proteins, evolutionarily conserved nucleases, has been found to play roles in various DNA-related processes, their in vivo functions in vertebrates have yet to be fully elucidated. TATDN2 has recently been implicated in resolving R-loops and participating in the replication stress response in BRCA1-deficient cancer cells. Here we found that tatdn2 exhibits conserved expression in mitotic and early meiotic germ cells across teleosts and mammals. Using medaka fish as a model, we then showed that loss of tatdn2 leads to all-phenotypically male adults and infertility due to PGC depletion during mitotic proliferation. We further demonstrated that knockout of tatdn2 increases R-loop accumulation and DNA damage, subsequently triggering apoptosis in PGCs. These findings indicate that tatdn2 plays a critical role in DNA damage repair associated with R-loop resolution in mitotic PGCs. Our study provides novel insights into the physiological function of TATDN2 and the mechanisms of genome maintenance in PGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aba7e41b7a154253da5adb67bf89677856b6ff" target='_blank'>
              Tatdn2 is required for DNA repair to safeguard genome stability in primordial germ cells
              </a>
            </td>
          <td>
            Hanqiao Shang, Puxuan Jiang, Zehui Wang, Zhaojun Shan, Yuze Chen, Ting Zhang, Yingshu Li, Qiang Tu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Drug tolerant persister cells (DTPs) refer to a transient drug-tolerance sub-population of cancer cells characteristics of phenotype plasticity and heterogeneity. This adaptive cell state is a critical transitional phase, standing on the crossroad that cancer cells reacquire drug sensitivity or enter into the permanent drug resistance. Emerging evidences indicate the epitranscriptomic regulations, particularly RNA methylations are the important mechanism underline post-transcriptional regulations of genes expression across all RNA species. RNA is integral to gene expression as messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA), which play roles in transmitting information from DNA to the synthesis of functional proteins. Methylation modifications on these RNAs are prevalent and represent a well-recognized non-genetic mechanism, exerting multifaceted regulatory effects on nucleic acid metabolism, such as nucleotide precursor availability, RNA processing dynamics, sub-cellular localization, transcript stability and translational fidelity/ efficiency. This review systematically sorts out the relevant references, demonstrating recent advances on the knowledge of the patterns of methylation modifications on mRNA, tRNA and rRNA, and how these modifications drive the generation and development of DTPs, which hallmarks of epithelial-mesenchymal transition, metabolism shift and immune escape. And then clinical strategies are delineated, leveraging pharmacological modulators of RNA-modifying enzymes alongside non-pharmaceutical lifestyle advice, for the development of therapy strategies preventing DTPs-rooted tumor relapse in this anti-tumor armamentarium with cytotoxic reagents, targeted therapies and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098e2b3a70bd6f86f19a2d17ea69012683c3e930" target='_blank'>
              Therapy-induced mRNA, rRNA and tRNA methylation alterations confer tolerance phenotype in tumor cells: mechanism and implications
              </a>
            </td>
          <td>
            Anfeng Jiang, Shujie Liu, Zhiyuan Li, Xiongzhou Zhang, Minghao Duan, Bin Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4efb8218ff34b8674a51cd71fbecd16898befa89" target='_blank'>
              Developing Topics.
              </a>
            </td>
          <td>
            Morgan Elizabeth Lambert, Paulino Ramirez, Wenyan Sun, Bess Frost
          </td>
          <td>2025-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Lysine specific demethylase 1 (LSD1), encoded by the gene KDM1A, is overexpressed and correlates with poor patient prognosis in Ewing sarcoma. LSD1 and the pathognomonic fusion oncoprotein, EWSR1::FLI1, colocalize throughout the genome, suggesting LSD1 is a critical co-regulator driving the progression of Ewing sarcoma. However, therapeutic targeting of LSD1 by competitive and noncompetitive inhibitors has yielded mixed results. Irreversible, enzymatic inhibition seems ineffective, but reversible noncompetitive inhibition has predominant off target mechanisms, leaving open the question of LSD1 function in Ewing sarcoma. Here we take a robust approach through multiple methods of depletion in multiple EwS cell lines to define enzymatic and nonenzymatic contributions of LSD1 to transcriptional regulation. We define a core set of 22 genes that are commonly repressed by LSD1 in all cell lines, and that repression of these genes downregulates synapse functioning and e-cadherin target genes. Derepression of these genes with LSD1 loss is an early and sustained genotype in all cell lines tested. We further define distinct gene sets in each cell line that are regulated by enzymatic and nonenzymatic LSD1 activity and find repression of e-cadherin target genes to be nonenzymatically regulated. This finding supports the growing body of evidence that in addition to their canonical catalytic activity, chromatin regulatory enzymes serve essential noncanonical roles as well. Furthermore, we uncovered evidence through use of the irreversible inhibitor OG-L002 that 2D cytotoxicity and proliferation assays may be insufficient to determine Ewing sarcoma response to LSD1 inhibition. SIGNIFICANCE Here we address a long-standing question in the field surrounding LSD1 and define the distinct enzymatic and nonenzymatic functions of LSD1 in Ewing sarcoma. In doing so, we have created a robust data set using genetic and pharmacological techniques in multiple models to thoroughly characterize LSD1 function in Ewing sarcoma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa31c55d8000608b4f5b4507892563a7a74a977c" target='_blank'>
              LSD1 Performs Demethylase-Independent and Context-Specific Roles in Ewing Sarcoma
              </a>
            </td>
          <td>
            Rachel D. Dreher, C. Taslim, Ira Miller, John W. Sherman, Ariunaa Bayanjargal, Emily R. Theisen
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genomic studies have revealed elevated damage and mutation rates in active transcription factor (TF) binding sites in UV-linked cancers. Previous investigations into the relationship between TF activity and UV DNA damage have primarily focused on select TFs or been done in aggregate across large cohorts of TFs at kilobase resolution. While collectively, there is evidence that TFs contribute to UV-induced mutagenesis by both enhancing initial damage formation and attenuating repair, there has yet to be a comprehensive characterization of these mechanisms on a per-TF basis. Using genome-wide maps of UV damage from human skin fibroblasts, we developed a scalable statistical framework to analyze TF-mediated mutagenic mechanisms across hundreds of TFs. We identify numerous previously unreported TFs that significantly enhance and / or inhibit damage formation in their binding sites. A systematic survey of TF-DNA complexes further revealed that positions of UV damage modulation coincide with TF-induced structural distortions that either protect or predispose DNA to photodimer formation. Additionally, we analyzed repair efficiency in TF binding sites with unprecedented resolution, identifying specific TFs and binding site positions likely to compete with repair factors. By comparing these results with skin cancer mutations, we distinguish mutation peaks driven by increased damage susceptibility versus attenuated repair, illustrating that TF-mediated mutagenesis is highly contextual and dependent on the TF, binding site position, and sequence context of the damaged locus. Our approach provides a robust statistical framework for elucidating mechanisms of mutagenic TF-binding and offers novel insights into the complex interplay between protein interactions, DNA damage, and repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87aa924971352d37048c9215f6b18e4948ab222" target='_blank'>
              High-throughput characterization of transcription factors that modulate UV damage formation and repair at single-nucleotide resolution
              </a>
            </td>
          <td>
            Raluca Gordân, Hana Wasserman, Bo Chi, Kaitlynne A. Bohm, M. Duan, Harshit Sahay, Alexias Safi, Gregory Crawford, Peng Mao, John J. Wyrick, Miles Pufall
          </td>
          <td>2025-12-10</td>
          <td>Research Square</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Liquid biopsies containing circulating tumor DNA (ctDNA) are important biomarkers across several forms of cancer. The detection of mutations in cell-free DNA (cfDNA) indicates the presence of ctDNA. However, unsatisfactory ctDNA mutation sensitivities, issues with sequencing errors, and clonal hematopoiesis variants have limited the clinical utility of mutation-based ctDNA assays. Recently, a new avenue of cfDNA assays has been developed, focusing on cfDNA epigenetics. Here, we outline the recent advancements in cfDNA epigenetics, focusing on cfDNA methylation, fragmentomics, and post-translational modifications (PTMs) of circulating nucleosomes. We present various methylation strategies concerning ctDNA detection and tissue of origin (TOO) analyses. cfDNA fragmentomics focuses on cfDNA fragment lengths, fragment end motifs, and nucleosome positioning to infer gene expression and estimate the ctDNA fraction. Lastly, we discuss the development of cell-free chromatin immunoprecipitation of circulating nucleosomes with PTMs. This method has been implemented to detect tumor gene expression, TOO, and treatment resistance. Combining the epigenetic features of cfDNA will expand the utility of liquid biopsies to give a more comprehensive insight into tumor biology, treatment response, and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0025e7aa342389077db174680eb853f811df7b03" target='_blank'>
              Liquid biopsy epigenetics: establishing a molecular profile based on cell-free DNA.
              </a>
            </td>
          <td>
            C. T. Maansson, A. L. Nielsen, B.S. Sørensen
          </td>
          <td>2025-11-21</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f741cfc314db9bdb828e55bb38f137b5fb40cdaa" target='_blank'>
              Plasmid copy number affects the DNA methylation-driven expression dynamics of the CfrBI restriction-modification system and impacts phage restriction
              </a>
            </td>
          <td>
            Johannes Gibhardt, Trung Duc Hoang, K. Severinov, Mikhail A. Khodorkovskii, N. Morozova
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Despite advancements in treatment, 5-year survival for medulloblastoma (MB) remains 60-70%. Human cytomegalovirus(HCMV) has been implicated as an oncomodulator in MB, but its impact on survival has not been explored. This study evaluates HCMV expression in pediatric MB tissue and its association with molecular risk groups and survival.



 A retrospective study of pediatric MB cases (≤19 years) from 2007-2023 was conducted using the WHO 2021 classification, the Northcott brain-tumor classifier, and the SIOP-Europe/ERN PaedCan risk stratification. HCMV immediate-early (HCMV-IE) and late-antigen (HCMV-LA) expression were assessed by optimized immunohistochemistry, and whole-genome sequencing (WGS) data from 20 tumors were analyzed for viral gene expression.



 Forty-five patients (mean age 8.2 years; 22% ≤3 years) were included: WNT (18%), SHH TP53-wildtype (18%), and non-WNT/non-SHH (64%). 20% of MB belonged to low-risk, 33% to standard-risk, and 47% to high-risk (HR-MB) groups. Four tumors exhibited MYC/N amplification. HCMV-LA positivity was found in 84% of cases, with 53% showing high expression (≥25% of cells). Cox-regression identified HR-MB (HR = 4.197, p = 0.021) and high HCMV-LA (HR = 4.334, p = 0.027) as independent predictors of poor outcomes. WGS revealed UL88 as the most abundantly expressed HCMV gene(log2[TPM+1] ≈14–15), with maximal expression in Group 3 MB.



 This study provides the first evidence linking high HCMV-LA expression to adverse outcomes and high-risk molecular features in pediatric MB. UL88 emerged as the most strongly expressed HCMV gene across MB samples. These findings suggest a potential prognostic and therapeutic role for HCMV in MB, warranting validation in larger, prospective cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60feabdb3f3fd823a43f478c57c8aa48c2982d72" target='_blank'>
              High Human Cytomegalovirus antigen Expression in pediatric medulloblastoma tissue is associated with poor event-free survival
              </a>
            </td>
          <td>
            Maria F De la Cerda-Vargas, J. Schittenhelm, Martin Ebinger, J. Zipfel, R. Beschorner, Ghazaleh Tabatabai, Martin U Schuhmann
          </td>
          <td>2025-12-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Kaposi’s sarcoma–associated herpesvirus (KSHV) causes cancers for which no vaccine exists, and current treatments are often imperfect. Recent work shows that the virus and infected cells produce circular RNAs, which are stable RNA loops that can influence gene activity and build up during active infection. Because these molecules resist degradation and may travel inside virus particles, they can help track disease or shape how the virus behaves. This review explains how viral circular RNAs are produced, how researchers detect and assay them, and what is known about their roles during quiet and active stages of infection in the context of KSHV. It also outlines experimental tools to test what these RNAs do and how to target them. By organizing current knowledge and practical methods, this review aims to guide future studies and speed progress toward better diagnostics and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c49ebc6839a316e30284da0f7440e889a77090bc" target='_blank'>
              Recent Studies on Kaposi’s Sarcoma-Associated Herpesvirus Circular RNAs
              </a>
            </td>
          <td>
            Cristian J. Pagtalunan, Isadora Zhang, Ariella Turley, Fenyong Liu
          </td>
          <td>2025-11-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Transposable elements (TEs) are mobile genetic elements that are present in great numbers in the majority of eukaryotic genomes. They are major drivers of genome evolution as they can facilitate chromosome rearrangements, provide mechanisms for genomic shuffling, and contribute to genome expansion, thereby altering genome architecture. Additionally, TEs can also modify gene expression by disrupting regulatory sequences, impairing genes, and promoting the emergence of new sequences. Despite their abundance, identifying TEs is challenging and time-consuming due to their extreme diversity in DNA sequence. Many TE families are ancient, and most of their sequences have become inactive due to accumulated mutations and fragmentation. Consequently, different copies of the same TE can differ greatly from each other, which makes identifying decayed copies particularly difficult. In this work, we employ transformer architecture for TE classification. TEclass2 is an integrated classifier that rapidly predicts TE orders and superfamilies using models built on this advanced machine learning approach. The software is available through a web interface, allowing users to classify sequences into sixteen superfamilies according to the Wicker classification system, or alternatively, users can download the source code to train and build custom classification models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf277b82c7226a65ad3cdcbe7a255c99eb31947" target='_blank'>
              Transformer-Based Classification of Transposable Element Consensus Sequences with TEclass2
              </a>
            </td>
          <td>
            Lucas Bickmann, Matias Rodriguez, Xiaoyi Jiang, Wojciech Makałowski
          </td>
          <td>2025-12-29</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, Hyeonjin Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, Daniel Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Intestinal metaplasia (IM) is a premalignant condition associated with increased risk of gastric cancer-a deadly malignancy with varying geographic incidence. High-depth targeted sequencing of more than 1,500 IM samples from six countries identified 47 significantly mutated genes, including driver genes associated with high-risk populations and worse prognosis (ARID1A), KRAS/MAPK signaling (KRAS, BRAF, MAP2K1, MAP3K1, and MAP2K4), and altered mucosal immunity (PIGR). IM whole-genome sequencing and DNA methylation analysis revealed SBS17 as a specific mutational signature separating IMs from normal gastric tissues, associated with late DNA replication, genomic hypomethylation, and tobacco exposure. Beyond epithelial-derived somatic mutations, we observed elevated clonal hematopoiesis (CH) in patients with IM associated with age, smoking, and enhanced risk of progressing to gastric cancer. Patients with CH expansions exhibited co-occurring IM PIGR truncating mutations and greater colonization of the IM microenvironment by orally derived bacteria, suggesting that CH may promote IM progression by modulating host-microbe mucosal immunity.


SIGNIFICANCE
This international study identifies recurrent IM driver genes, IM-specific mutational signatures, and alterations in IM-associated immune landscapes and microbiomes. Our results highlight a role for nonepithelial somatic alterations (CH) in IM progression to gastric cancer, offering new translational opportunities for early cancer detection and interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f6a437b6939f544182c7d61adabe509b3b8f5bd" target='_blank'>
              Mutational Signatures and Clonal Hematopoiesis in Intestinal Metaplasia across Countries with Varying Stomach Cancer Incidence.
              </a>
            </td>
          <td>
            Kiekyong Huang, T. Hagihara, Benedict Shi Xiang Lian, Zhi Xuan Ong, Shen Kiat Lim, R. Chong, Supriya Srivastava, Jason Xing Kang, May Yin Lee, A. Tan, Minghui Lee, Shamaine Wei Ting Ho, Siti Aishah Binte Abdul Ghani, Clara Shi Ya Ng, Ruanyi Liang, Lin Liu, S. Tay, Xuewen Ong, Feng Zhu, Hui Chen, Zhen Li, T. Ang, Takuji Gotoda, Robert J. Huang, Christopher J L Khor, Hyun-Soo Kim, L. H. Lau, Yi-Chia Lee, Ayaka Takasu, Ming Teh, M. Thian, Wai Leong Tam, Xin Lu, Sunny H Wong, J. B. So, Hyunsoo Chung, J. Lee, K. Yeoh, Patrick Tan
          </td>
          <td>2026-01-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d6b8d33286dfd47e06aa024a1a7a5fb5466454" target='_blank'>
              Deciphering APOBEC1 in Avians: Unravelling loss events and functional insights
              </a>
            </td>
          <td>
            Aswin S Soman, N. Vijay
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability.
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="We have compared genome-wide patterns of RNA 2’-O-methylation (Nm) between two isogenic pairs of neurons. Each pair includes one line harboring a small deletion of orphan box C/D snoRNAs (SNORD116s) from the paternal chr15q11-q13 region. One isogenic pair also differs in expression of SNORD113/114 snoRNAs from chr14q32.2. Wild-type and modified cells were differentiated into cortical neurons, and genome-wide patterns of Nm identified. Neurons display a distinctive signature of rRNA modification compared to undifferentiated stem cells. We further identified thousands of shared Nm sites in mRNAs, lncRNAs and small RNAs. Most sites do not exhibit canonical complementarity to snoRNAs, but a number exhibit strong complementarity to U3 snoRNA, not previously shown to direct Nm. Evidence from cross-linking and sequencing of hybrids (CLASH) suggests that U3 is proximally associated with a subset of 2’-O-methylation events. Finally, we identify a number of apparent canonical targets of SNORD113, SNORD114 and SNORD116 snoRNAs. These data present a comprehensive characterization of the Nm landscape in neurons and, for the first time, allow the assignment of Nm sites targeted by specific orphan snoRNAs associated with neurodevelopmental and other disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/406a3c32baf26576f3b4fe5322db466adc1facf5" target='_blank'>
              Genome-wide profiling of RNA 2’-O-methylation in neurons and identification of orphan snoRNA targets
              </a>
            </td>
          <td>
            Xuan Ye, Yaling Liu, Yinzhou Zhu, Sarah A Alshawi, B. A. Elliott, Christopher L. Holley, Jean-Denis Beaudoin, B. Graveley, Gordon G. Carmichael
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Nonhomologous end-joining repairs chromosomal double strand breaks, but it is unknown whether both strands are repaired by this pathway, and if one strand break’s repair path impacts the other. Here, we show that nonhomologous end-joining employs both of two a priori possible strategies. Strand breaks that can be directly ligated are joined near-simultaneously, with no effect of one strand break’s repair path on the other. More complex end structures require obligatorily ordered repair. The first strand to be repaired is used as template for repair of the opposite/second strand break, with the latter repair reaction occurring fastest when also coupled to nonhomologous end-joining. Enforced asymmetry in repair of each strand break can extend to the gap-filling polymerase employed, and whether the polymerases incorporate RNA or DNA. Our results resolve questions about pathway mechanism and identify a requirement for flexibility of the nonhomologous end-joining machinery for efficient repair of both strand breaks within diverse cellular double strand breaks. Breakage of both chromosomal DNA strands creates unique problems for DNA repair. Here, Luthman et al. show that for some broken ends, the two strand breaks are repaired in parallel, while for other ends one strand must be repaired before the other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/373066512ad478c3f6bdf424666dac921cd13ebb" target='_blank'>
              Nonhomologous end-joining uses distinct mechanisms to repair each strand of a double strand break
              </a>
            </td>
          <td>
            Adam J. Luthman, K. K. Chiruvella, A. M. Kaminski, Thomas A. Kunkel, Lars C. Pedersen, Dale A. Ramsden
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Multiplexed DNA FISH (also known as FISH Omics) is a powerful imaging technology to map three-dimensional (3D) genome organization in thousands of single cells at kilobase resolution. However, existing computational approaches to identify 3D genome features, such as A/B compartments, topologically associating domains (TADs) and chromatin loops, have been mainly adapted from Hi-C and do not account for the distinct measurement errors across the three XYZ physical axes. Here we present ArcFISH (accurate and robust chromatin analysis from FISH data), which explicitly models axis-specific measurement errors in a unified statistical framework to detect multi-level 3D genome features. Comprehensive evaluations using multiple simulated as well as experimental datasets show that ArcFISH reliably identifies chromatin loops under heterogeneous measurement errors, detects TAD boundaries highly enriched for active histone marks H3K4me3 and H3K36me3, and requires as few as 30 chromatin traces to faithfully assign A/B compartments. Applying ArcFISH to multiplexed DNA FISH data from mouse embryonic stem cells and mouse excitatory neurons reveals cell-type-specific chromatin loops and TADs, overlapping with differentially expressed genes. ArcFISH is freely available at https://hyuyu104.github.io/ArcFISH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9edbaba3bc8b0ff6aa7fa6a3e2e3e50014db57c6" target='_blank'>
              Accurate and robust 3D genome feature discovery from multiplexed DNA FISH
              </a>
            </td>
          <td>
            Hongyu Yu, Lingbo Zhou, Liangqi Xie, C. Strambio‐De‐Castillia, Ming Hu, Yun Li
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="The dynamics of colorectal epigenetics within the adenoma stages of oncogenesis remain undocumented. In this study, we investigated DNA methylation dynamics in colorectal cancer oncogenesis from non-tumor colon tissue to low-grade, high-grade adenoma and adenocarcinoma. The methylome of 12 low-grade and 19 high-grade colorectal adenomas was determined via the EPIC v1 Human Methylation BeadChip. These methylation profiles were complemented with the methylomes of 206 non-tumor colon and 22 colon adenocarcinoma samples from the GEO and TCGA databases. Differentially methylated CpGs were identified via Student’s t test and used to monitor the evolution of the colon methylome during oncogenesis. The differentially methylated promoters were used to infer the associated biological process via gene ontology and the evolution of the methylation of 34 described colorectal cancer DNA methylation biomarkers was explored. A total of 11.9% of the colon methylome was significantly altered (q < 10− 4) during oncogenesis, with half corresponding to DNA demethylation. Of which, 67.4% occurred during the transition from non-tumor colon tissue to low-grade adenoma. A total of 9% of the DNA methylation changes were specific to low-grade and/or high-grade adenomas. The biological pathways related to the sensory perception of odor and stimulus were hypomethylated early, nucleic acid metabolic process were methylated early, post-transcriptional regulation were transiently hypomethylated and mitotic cell cycle were transiently methylated. Twenty-one out of 34 the biomarkers were methylated in low-grade adenomas and 11 out of 34 in high-grade adenomas. This suggests that they could be used to distinguish stages of oncogenesis. This study provides insight into the dynamics of colonic epigenetics during oncogenesis, with early DNA methylation changes in low-grade adenomas associated with transient DNA methylation changes. However, the causality of these changes remains to be elucidated. This study also explores the evolution of known biomarkers and their clinical applications for indirectly asserting the tumor’s stage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693aec5fb4af242001a18947727bde5f96cd3844" target='_blank'>
              Dynamic DNA methylation changes during colorectal oncogenesis with insights from adenoma stages
              </a>
            </td>
          <td>
            A. Overs, C. Molimard, Jules Durand, F. Bibeau, Laurent Arnould, F. Monnien, Claire Clavier, Christophe Borg, M. Guittaut, Jean‐Paul Feugeas, E. Hervouet, Paul Peixoto, Z. Selmani
          </td>
          <td>2025-11-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Cutavirus (CuV), the newest human protoparvovirus (PPV), has gained attention due to its significant association with cutaneous T-cell lymphoma (CTCL) and its precursor, parapsoriasis, whereas the other human PPVs, bufavirus and tusavirus, show no such link. Given this association, it is important to investigate the prevalence of CuV DNA in other tissues, particularly those affected by malignancy or inflammation. This study assessed, by multiplex quantitative PCR, the genoprevalences of all three PPVs in 427 fresh-frozen intestinal biopsies from inflammatory bowel disease (IBD), colorectal cancer, adenomas or healthy mucosa of 185 individuals, as well as in 94 formalin-fixed paraffin-embedded (FFPE) biopsies from malignant and non-malignant breast conditions of 85 patients. The study also compared the DNA prevalences of human herpesvirus (HHV)-6A, -6B and -7 in the breast tissues. CuV mRNA was assayed with reverse-transcription PCR, and corresponding FFPE sections underwent in situ hybridization. CuV DNA was detected in intestinal IBD or healthy mucosa from 6/185 (3.2%) subjects, but no CuV mRNA or in situ signals were detected. In breast biopsies, HHV-6B and HHV-7 DNAs were present in 20.3 and 5.1%, respectively, while all PPVs and HHV-6A were absent. Overall, CuV was absent in all 70 cancer tissues, underscoring its association with CTCL. The low CuV DNA loads and prevalences in intestinal and breast morbidities, and lack of activity, suggest that CuV is unlikely to play a role in these malignancies or inflammatory conditions. In contrast, HHV-6B may be more relevant to breast pathology, even though it is also widely detected in healthy tissues. Nevertheless, our study provides insight into persistent DNA viruses implicated in cancer and highlights their occurrence across various disease manifestations, laying a foundation for future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d6897679f19828620cb2c9e6fa42182992a9360" target='_blank'>
              Genoprevalence of cutavirus in benign and malignant intestinal and breast tissues
              </a>
            </td>
          <td>
            Irini M Assimakopoulou, Ushanandini Mohanraj, Taina Sipponen, Anna H. Lepistö, Dalia Y Kadry, R. Hamdy, Mahmoud M. Kamel, Heba M El-Batal, A. Abdel-Moneim, M. Söderlund-Venermo
          </td>
          <td>2025-12-01</td>
          <td>The Journal of General Virology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The DNA Damage Repair (DDR) signaling pathway serves as a crucial molecular hub that regulates chemotherapy efficacy, offering significant translational value in the field of precision oncology. This review systematically analyzes the molecular mechanisms of five core DDR pathways (Homologous Recombination Repair, Non‐Homologous End Joining, Base Excision Repair, Nucleotide Excision Repair, and Mismatch Repair) in mediating chemotherapy resistance in tumors, and thoroughly elucidates the correlation between key molecular events—such as BRCA1/2 deficiency, MMR functional abnormalities, and Ataxia Telangiectasia Mutated/Ataxia Telangiectasia and Rad3‐related (ATM/ATR) signaling pathway dysregulation—and chemotherapy sensitivity. The DDR deficiency biomarker system established through the integration of multi‐omics data provides molecular classification tools for predicting the efficacy of platinum‐based drugs. This study focuses on the mechanism by which Poly ADP‐Ribose Polymerase inhibitors reverse Homologous Recombination‐Deficient tumor resistance through “synthetic lethality” effects while also revealing the synergistic anti‐tumor effects of ATM/ATR inhibitors in combination with chemotherapeutic agents. The research presents an innovative molecular synergy model between DDR regulation and Immune Checkpoint Blockade, confirming that tumor neoantigen release induced by DDR deficiency can enhance immunotherapy responses. This article also provides perspectives on multidimensional intervention strategies based on the DDR network, including the development of inhibitors targeting novel DDR targets, the establishment of DDR pathway functional assessment systems based on multidimensional biomarkers, and the investigation of synergistic paradigms between DDR and novel therapeutic modalities. Additionally, we explore the dynamic evolution mechanisms of DDR‐mediated chemotherapy resistance by analyzing the interactions between DDR and metabolic reprogramming, as well as other related processes. These breakthrough advances provide theoretical foundations and innovative directions for overcoming chemotherapy resistance and advancing personalized treatment, marking a new era in cancer therapy characterized by precision targeting of DDR pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/944e141ebbf24071dde5281d6930372251bae23d" target='_blank'>
              Translational Aspects of DNA Damage Repair in Optimizing Cancer Chemotherapy
              </a>
            </td>
          <td>
            Anqi Lin, Jinyue He, Aimin Jiang, Jian Zhang, Quan Cheng, Hengguo Zhang, Wenjun Mao, Peng Luo
          </td>
          <td>2025-12-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bbd34479581895296f8a2acf6b6977e90cd1df9" target='_blank'>
              Mechanisms of Intracellular Selection of Mitochondrial DNA
              </a>
            </td>
          <td>
            Georgii Muravyov, Dmitry A. Knorre
          </td>
          <td>2025-12-01</td>
          <td>Biochemistry (Moscow)</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf0ad87ffc8bb1692d847c310099f651a0e6259" target='_blank'>
              Genome organizer SATB1 restrains the BACH1-cMAF axis that drives autoimmunity
              </a>
            </td>
          <td>
            Dionysios-Alexandros Papamatheakis, Petros Tzerpos, Despina Tsoukatou, Eleftherios Morres, Manouela Kapsetaki, Dezso Balázs, Kazuhiro Igarashi, Charalampos G Spilianakis
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: Systemic sclerosis (SSc) is a rare, complex autoimmune disease characterized by fibrosis of the skin and internal organs. While its pathogenesis is not fully understood, chromosomal instability and telomere attrition have emerged as significant areas of investigation. Methods: This review provides a historical narrative perspective and synthesizes current findings on the role of these genomic anomalies in SSc pathogenesis. We synthesized findings from foundational and recent research articles investigating genotoxic factors, chromosomal aberrations, and telomere biology in SSc. Results: There is a strong historical basis for chromosomal instability in SSc, manifesting as micronuclei, translocations, and breaks. This instability is driven by clastogenic factors and oxidative stress. SSc-specific autoantibodies are implicated; anti-centromere antibodies correlate with aneuploidy and micronuclei, while anti-topoisomerase I may inhibit DNA repair. SSc is also characterized by significant telomere attrition, first reported in 1996 and now confirmed by additional genetic studies. This telomere loss is associated with reduced telomerase activity and the presence of autoantibodies against telomere-associated proteins, including shelterin components. Conclusions: We conclude that inflammation, telomere attrition, and chromosomal instability are linked in a self-perpetuating cycle that drives SSc pathogenesis. We propose that an initial inflammatory stimulus leads to reactive oxygen species production, causing telomere damage and attrition. Critically short telomeres trigger faulty DNA repair mechanisms, such as breakage–fusion–bridge cycles, resulting in chromosomal instability. This genomic damage, in turn, acts as a danger signal, further activating inflammatory pathways and creating a feedback loop that perpetuates fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a5b157843adfebc89c062c9bea135fbbb56036" target='_blank'>
              Chromosomal Instability and Telomere Attrition in Systemic Sclerosis: A Historical Perspective
              </a>
            </td>
          <td>
            C. M. Artlett
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mini-Chromosome Maintenance 10 (MCM10) is essential for maintaining genome stability by facilitating DNA replication and repair across various organisms. While the role of MCM10 in DNA replication is well-established, its mechanism in DNA repair remains less understood. In this study, we demonstrate that loss of AtMCM10 function leads to increased DNA damage under genotoxic or salinity stress in Arabidopsis thaliana. Detailed analysis reveals that AtMCM10 works primarily downstream of ATM and is crucial for intermolecular homologous recombination (HR) mediated by synthesis-dependent strand annealing (SDSA) in response to DNA damage. Further cytological and biochemical analyses reveal that AtMCM10 possesses DNA annealing activity, colocalizes with the double-strand break (DSB) sites, and undergoes liquid-liquid phase separation (LLPS) upon DNA damage, facilitated by single strand DNA (ssDNA) in vitro. Altogether, our findings indicate that AtMCM10 acts as a single-strand DNA (ssDNA) annealing protein to promote SDSA-mediated intermolecular HR repair via LLPS in somatic cells upon DNA damage, providing new insights into the HR repair mechanisms. Zhao et al. demonstrate that AtMCM10 repairs DNA double-strand breaks by forming liquid-like condensates at damage sites, where it directly anneals single–stranded DNA to promote synthesis–dependent strand annealing–mediated homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa34d1b94aca30e8567b05e2f87bf9eafb307a0" target='_blank'>
              AtMCM10 facilitates SDSA-mediated intermolecular homologous recombination repair via liquid-liquid phase separation in DNA damage response
              </a>
            </td>
          <td>
            Xinjie Zhao, Tianren Zhang, Dan Jin, Hui Chen, Shiqi Xia, Li Bai, Xiaohua Hao, Liqun Huang, Lianfu Tian, Yan Guo, Yusheng Zhao, Dongping Li, Zhizhong Gong
          </td>
          <td>2025-11-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2967424876b05ad4fc08b0dd3adaa2e4feebc8f7" target='_blank'>
              Deep meta-analysis of human pangenome data reveals triplication of the IGH constant gene locus
              </a>
            </td>
          <td>
            Pietro D’Addabbo, Rossella Cianci, Domenico Frezza
          </td>
          <td>2026-01-17</td>
          <td>Discover Immunity</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is an aggressive pediatric soft tissue cancer, with approximately 350 cases per year in the US. The highly aggressive tumor subtype, Fusion-Positive Rhabdomyosarcoma (FP-RMS) is characterized by chromosomal translocations yielding oncogenic fusion transcription factors (TFs). The most common RMS fusion (PAX3::FOXO1) forms an in-frame chimera of the DNA-binding domain of PAX3 with the activator domain of FOXO1. We hypothesize that understanding the precise mechanisms of PAX3::FOXO1’s chromatin engagement and nucleosome invasion would enable precision therapy to target these mechanisms to ultimately improve patient outcomes. Our studies are providing evidence that PAX3::FOXO1 has pioneer activity, including nucleosome engagement, and an ability to recognize multiple chromatin states including both open and repressive chromatin regions. We are investigating the biochemical regulatory functions of PAX3::FOXO1 binding to specific DNA motifs embedded within chromatin or nucleosomes, developing synthetic epigenetics to investigate motifs from our genomics-scale studies. We are interested in the preservation of naturally occurring post-translational modifications from the mammalian cellular context, and have sought to build a highly versatile system for investigation of TF/nucleosome targeting from cell extracts. In our studies, we are finding new evidence for PAX3::FOXO1’s targeting of homeodomain and paired domain DNA motifs in free DNA and also in a nucleosome context. We are validating these findings through parallel approaches including coupling electrophoretic mobility shift assays (EMSAs) with westerns and antibody-targeting of complexes within EMSA experiments. We are finding that PAX3::FOXO1’s binding to high-complexity motifs, conjoining paired domain and homeodomain sequences, occurs during nucleosome invasion. Through iterative comparisons, custom unmodified mono-nucleosomes containing three major classes of DNA binding motifs reveal PAX3::FOXO1’s binding to these nucleosomes without a strong motif preference within the categories examined in our series. These findings support a mechanistic hypothesis where PAX3::FOXO1 may invade nucleosomes, altering their local positioning prior to generation of chromatin accessibility. Additional investigations are underway to unravel these mechanisms further and probe the contexts for motif specificity or generality in nucleosome engagement, and to understand conformational changes in the target nucleosomes. The mechanisms we are uncovering will contribute to the search for new therapeutic vulnerabilities as our community progresses toward targeted therapies for FP-RMS.



 Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic Barr, Michael G. Poirier, Benjamin Stanton. Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbbbd914885e1df8de9eefd492cd42d843d809e" target='_blank'>
              Abstract B016: Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic G. Barr, Michael G. Poirier, Benjamin Z. Stanton
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In the 40 years since the discovery of the CENP proteins, many studies have examined the role of these proteins and their interactions with other chromosomal proteins of the centromere and beyond. Together, these studies have yielded vast amounts of sequencing and proteomics data. Typically, each study has focused on a single question and the majority of each dataset remains largely unexplored. Often the interesting details of publicly deposited data are left behind, buried in archives online, while more and more new data are generated. Reanalysing these databases can represent a new paradigm for investigating diverse biological pathways in unprecedented detail. Here, we explore two publicly available pan-cancer proteomic datasets to compare proteins whose abundance correlates with CENP proteins, with a particular focus on CENP-C. Our analysis confirms an expected link between CENP-C and cohesin levels but reveals a surprising and unexpected correlation between CENP-C and proteins of the inner nuclear membrane and the NuMA protein. This guilt-by-association analysis has the potential to identify proteins that act in common pathways but never associate or colocalize and may not even be expressed at the same time in cells. As an example, we show here that it can reveal unexpected links that expand our conception of centromeric chromatin beyond chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b131b447bc4a00bf7e5247f66ecd03a10d2ff6" target='_blank'>
              Sustainable integrative cell biology: CENP-C is guilty by association
              </a>
            </td>
          <td>
            N. Y. Kochanova, I. Samejima, William C. Earnshaw
          </td>
          <td>2025-11-25</td>
          <td>Chromosome Research</td>
          <td>2</td>
          <td>21</td>
        </tr>

        <tr id="The preservation and faithful propagation of genetic information is essential for all life forms and depends on cellular pathways that enable replication, recombination, and repair of DNA. The multifunctional XPF-ERCC1 DNA endonuclease complex acts in several DNA repair pathways and interacts with numerous partner proteins and large DNA repair assemblies, including the nucleotide excision repair machinery and the SMX tri-endonuclease complex. Here, we report structures of XPF-ERCC1 in complex with the DNA repair factors SLX4 and SLX4IP, thereby identifying key residues responsible for direct interactions with XPF-ERCC1. When introduced into human cells, point mutations in these interfaces impair the interactions between XPF-ERCC1 and SLX4 or SLX4IP, and disruption of the XPF-SLX4IP interface leads to cis-platin sensitivity. Furthermore, our data reveal the structure of the human XPF-ERCC1-SLX4IP-SLX4330-555 complex with DNA bound at its active site, and they complete the structural characterisation of molecular interactions required to assemble the SMX complex. Structure-specific endonucleases play an important role in several DNA repair pathways. Here the authors present structures of the endonuclease XPF-ERCC1 in complex with SLX4, SLX4IP, and DNA. Combined with functional analysis, these results provide insight into the mechanisms of XPF-ERCC1 recruitment and activation during DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff06e1b310b6d6c04354cb0699bf090be746399" target='_blank'>
              Molecular basis of XPF-ERCC1 targeting to SLX4-dependent DNA repair pathways
              </a>
            </td>
          <td>
            Junjie Feng, Peter R. Martin, Szymon Kowalski, Maxime Lecot, Nora B Cronin, T. Matthews-Palmer, Wojciech Niedzwiedz, Basil J. Greber
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 597


 Background:
 HER3, a tyrosine kinase receptor encoded by
 ERBB3
 lacks intrinsic catalytic activity but drives tumorigenesis and drug resistance when heterodimerized with other HER and non-HER receptors. The prevalence, spectrum, and clinical impact of
 ERBB3
 alterations in biliary tract cancers (BTCs) remain poorly defined yet may inform resistance mechanisms and opportunities for HER2/HER3-directed therapies.
 Methods:
 We interrogated the following cohorts for intrahepatic cholangiocarcinoma (iCCA) cases: (a) the FoundationCORE database (b) the cBioPortal, and (c) a single-institution retrospective BTC cohort with
 ERBB3
 alterations (IRB-approved). Analyses included alteration frequency, co-mutations, immune biomarkers, treatment outcomes, and survival.
 Results:
 149 (2.02%) cases from the (a) CORE database (n=7380) harbored an
 ERBB3
 alteration of which 48.5% were short variant base substitution mutations (SV) + 54.2% gene amplifications (6 copies or greater). The most commonly occurring co-mutations in the
 ERBB3
 mutated group were
 TP53
 (45.6%),
 CDKN2A
 (34.9%),
 CDK4
 (33.6%), and
 ARID1A
 (18.1%),
 ERBB2
 (20.8% ) and MDM2 (
 28.9
 % ). Classic iCCA drivers were less common (
 IDH1
 12.7% vs 13.7%;
 FGFR2
 6.7% vs 11.9%). The median TMB was higher (3.6 vs 1.3) and MSI-High status was more common (7.6% vs 1.4%) in
 ERBB3
 altered patients. Similar trends were seen among (b) iCCA in cBioPortal (n=727), with 15 (2.06%) harboring ERBB3 alterations with 53.3% SV and 46.6% gene amplifications. In the institutional BTC cohort (c) (n=21), the median age was 63 years, 57% male. SV predominated (89.5%), with co-alterations in
 TP53
 (38.1%),
 CDKN2A
 (21.0%),
 ARID1A
 (26.3%), and
 ERBB2
 (15.8%). First-line therapy included gemcitabine/cisplatin (28%), gemcitabine/cisplatin/durvalumab (23%), and capecitabine ± gemcitabine (19%). Median time to progression was 4.7 months (range 1.4–7.7); median overall survival for the cohort was 27.3 months (95% CI, 5.7–90+ months), with 63% alive at last follow-up. The median survival of patients with metastatic disease was 9.5 months (range: 1.7-97.6 months).
 Conclusions:
 ERBB3
 alterations occur in ~2% of iCCA and are associated with distinct genomic and immune features, including higher TMB, MSI-H, and enrichment of
 ERBB2
 co-mutations. These findings suggest potential therapeutic vulnerability to HER2/HER3-targeted strategies. Prospective studies focusing on structural variants and co-alteration contexts are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca7fb522ae81eb627d113bd80428112192496d4" target='_blank'>
              ERBB3
 driver genomic alterations in intrahepatic cholangiocarcinoma: A genomic landscape and clinical outcome study.
              </a>
            </td>
          <td>
            Mridula Nadamuni, Q. Kimana, F. Saj, D. Pavlick, Felicity K. Namayanja, , M. Javle
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181f022b28db4f3ed7a11dbea0e2ef041b80de4b" target='_blank'>
              Discovery of the TGCB microprotein translated from the lncRNA CHASERR
              </a>
            </td>
          <td>
            Bernardo Bonilauri, Xiaochun Yang
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) infects mainly CD4+ T lymphocytes and causes both malignant and inflammatory diseases: the aggressive malignancy known as Adult T-cell leukemia/lymphoma (ATL) and several chronic inflammatory syndromes. HTLV-1 infection is established by integration of proviral DNA (~9000 bp) into the host genome. In HTLV-1, two viral genes, tax and HBZ, play critical roles in viral transcription and promotion of T-cell proliferation, respectively. The present study was undertaken to test the hypothesis that the higher-order structure of proviral chromatin regulates its transcription on both the plus strand and the minus strand. ATAC-seq analysis identified an open chromatin region in the pol gene of proviral DNA, which we name IPOR, in many ATL cases. Using reporter assays, it was found that the sequence of IPOR suppresses the transcription of the plus strand and activates that of the minus strand. Binding motifs of Eomes and TEAD proteins were predicted in this region, and we confirmed recruitment of the transcription factors to their respective motifs by ChIP-qPCR. A mutant of IPOR which cannot bind the transcription factors weakened the transcriptional activating effects compared with the wild type, suggesting that the IPOR and those transcription factors suppress the 5' long terminal repeat (LTR) but activate the 3'LTR. In addition, a mutant HTLV-1 molecular clone, which possesses the IPOR mutant, produced a higher titer of virus than the wild type. RNA-seq analysis of HTLV-1-infected cell lines, in which Tax expression can be traced after induction, revealed that EOMES expression decreases during the tax transcriptional burst and resumes following termination of the burst. These findings suggested that the expression dynamics of Eomes affect the transient expression of Tax. The IPOR sequence appears to regulate the transcription from both LTRs, suppressing the 5'LTR but activating the 3'LTR. Recruitment of Eomes to the IPOR is likely to influence the expression of Tax and HBZ. Since both viral genes are involved in diverse mechanisms for viral replication, cellular proliferation, and immune regulation, the IPOR may play a role in fine-tuning the modes of viral persistence in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2f1e6ae01e0a8f9b5438f95ad7675208ffd69" target='_blank'>
              Intra-pol proviral open region of HTLV-1 controls the transcription from both long terminal repeats
              </a>
            </td>
          <td>
            Miyu Sonoda, Azusa Tanaka, Xueda Chen, Wenyi Zhang, Helen Kiik, S. Ramanayake, K. Nosaka, N. Takenouchi, Masanori Nakagawa, Akihiro Fujimoto, C. Bangham, Masao Matsuoka, Jun-ichirou Yasunaga
          </td>
          <td>2025-12-08</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d96eca4dc76fb7d7dcfd02bd48a5d396e340f16" target='_blank'>
              Role of circAGFG1 as an oncogene in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wei Zhang, Lizhe Zhu, Xiaoqian Li, Jingyi Yang, Yu Ren, Jianjun He, Jing Xu
          </td>
          <td>2026-01-01</td>
          <td>Apoptosis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor that occurs in the mucosal epithelial cells of the colon or rectum, with high mortality and morbidity. Therefore, the study of its molecular principle is more important. In recent years, more attention has been paid to the regulation of CRC by epigenetic factors, especially the regulation of non-coding RNAs (ncRNAs) and RNA modifications. ncRNAs include miRNA, circRNA, and lncRNA. Among them, miRNAs regulate MAPKs, Wnt, and TGF-β signaling pathways and are involved in the tumor process. CircRNAs, as regulators of gene expression, can participate in various biological processes. lncRNA interacts with DNA, RNA, and proteins to regulate signaling pathways in various ways, potentially acting as biomarkers in non-invasive examinations. RNA modifications, especially N6-methyladenosine (m6A), affect CRC development. m6A consists of three proteins, writers, erasers, and readers, which maintain the stability of RNA modifications. Different levels of expression of these three proteins in CRC can lead to changes in the tumor microenvironment, gene expression, and other factors, and thus play an important role in CRC. ncRNAs and RNA modifications comprise the CRC epigenetic regulatory network, providing a new direction for the diagnosis and treatment of CRC in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d5caa5bf293c7752dde20eabcef8521208dcce" target='_blank'>
              Center Molecular Mechanisms of Non-coding RNAs and RNA Modifications in Colorectal Cancer
              </a>
            </td>
          <td>
            Tongyue Yu
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [20, 2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>