<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-12 09:29:51 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>34</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>69</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>111</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>331</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>291</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a common source of oncogene amplification across many types of cancer. The non-Mendelian inheritance of ecDNA contributes to heterogeneous tumour genomes that rapidly evolve to resist treatment. Here, using single-cell and live-cell imaging, single-micronucleus sequencing, and computational modelling, we demonstrate that elevated levels of ecDNA predisposes cells to micronucleation. Damage on ecDNA, commonly arising from replication stress, detaches ecDNA from the chromosomes upon which they hitchhike during cell division, thereby causing micronucleus formation in daughter cells. Clusters of oncogene-containing, CIP2A-TOPBP1-associated ecDNA molecules form, and asymmetrically segregate into daughter cell micronuclei during cell division. ecDNA chromatin remains highly active during mitosis, but upon micronucleation, it undergoes suppressive chromatin remodeling, largely ceasing oncogene transcription. These studies provide insight into the fate of damaged ecDNA during cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f629203ff4e6d119bd3d8d7b5f0a9d082c7f87e7" target='_blank'>
              Abstract 5046: Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, W.J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7372198dd7ad93e0ac09354eea55bbe4096844" target='_blank'>
              Human Papillomavirus Genomes Associate with Active Host Chromatin during Persistent Viral Infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7809d316520a84fae49c8004779285724bae28b" target='_blank'>
              Analysis of regulatory mechanisms of enhancers in gastric cancer with double minute chromosomes based on bioinformatics.
              </a>
            </td>
          <td>
            Meng-Ting An, Guan-Lin Guo, Jie Wu, Wen-Jing Sun, 
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, P. Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John P. McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J. Colón-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad9d941478590c32e5cf0763f3872cb1fd2c287" target='_blank'>
              Satellite DNA Amplification in Advanced Prostate Cancer Is Largely Independent From Euchromatic and Oncogene Amplicons.
              </a>
            </td>
          <td>
            Anja Weise, A. Ornellas, Gilda Alves, Constanze Pentzold, Jenny Holler, Melanie Wolter, Elena Jamali, Bernhard Theis, Thomas Liehr
          </td>
          <td>2025-03-17</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, Katie Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, Markus Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M. Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="NOTCH2NL (NOTCH2-N-terminus-like) genes arose from incomplete, recent chromosome 1 segmental duplications implicated in human brain cortical expansion. Genetic characterization of these loci and their regulation is complicated by the fact they are embedded in large, nearly identical duplications that predispose to recurrent microdeletion syndromes. Using nearly complete long-read assemblies generated from 67 human and 12 ape haploid genomes, we show independent recurrent duplication among apes with functional copies emerging in humans ∼2.1 million years ago. We distinguish NOTCH2NL paralogs present in every human haplotype (NOTCH2NLA) from copy number variable ones. We also characterize large-scale structural variation, including gene conversion, for 28% of haplotypes leading to a previously undescribed paralog, NOTCH2tv. Finally, we apply Fiber-seq and long-read transcript sequencing to human cortical neurospheres to characterize the regulatory landscape and find that the most fixed paralogs, NOTCH2 and NOTCH2NLA, harbor the greatest number of paralog-specific elements potentially driving their regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57156ab124ecd643544fdfa8cdaaa9cc74e49805" target='_blank'>
              Genetic diversity and regulatory features of human-specific NOTCH2NL duplications
              </a>
            </td>
          <td>
            Taylor Real, Prajna Hebbar, DongAhn Yoo, F. Antonacci, Ivana Pačar, M. Diekhans, Gregory J. Mikol, Oyeronke G. Popoola, Ben Mallory, Mitchell R. Vollger, Philip C. Dishuck, Xavi Guitart, Allison N. Rozanski, Katherine M. Munson, Kendra Hoekzema, Jane Ranchalis, Shane Neph, Adriana E. Sedeño-Cortés, B. Paten, Sofie R. Salama, A. Stergachis, E. Eichler
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8275563011d6ab6090a01c6b82a9c990741f5" target='_blank'>
              Z-DNA formation induces the totipotent-like state and primes Zscan4-dependent chromatin compartmentalization
              </a>
            </td>
          <td>
            Shireen Shajahan, Yann Loe-Mie, Marion Salmon-Legagneur, Tatiana Traboulsi, Agnès Thierry, Anne Dejean, J. Cossec
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer is fundamentally a disease of the genome, characterized by extensive genomic, transcriptomic, and epigenomic alterations. Most current studies predominantly use short-read sequencing, gene panels, or microarrays to explore these alterations; however, these technologies can systematically miss or misrepresent certain types of alterations, especially structural variants, complex rearrangements, and alterations within repetitive regions. Long-read sequencing is rapidly emerging as a transformative technology for cancer research by providing a comprehensive view across the genome, transcriptome, and epigenome, including the ability to detect alterations that previous technologies have overlooked. In this Perspective, we explore the current applications of long-read sequencing for both germline and somatic cancer analysis. We provide an overview of the computational methodologies tailored to long-read data and highlight key discoveries and resources within cancer genomics that were previously inaccessible with prior technologies. We also address future opportunities and persistent challenges, including the experimental and computational requirements needed to scale to larger sample sizes, the hurdles in sequencing and analyzing complex cancer genomes, and opportunities for leveraging machine learning and artificial intelligence technologies for cancer informatics. We further discuss how the telomere-to-telomere genome and the emerging human pangenome could enhance the resolution of cancer genome analysis, potentially revolutionizing early detection and disease monitoring in patients. Finally, we outline strategies for transitioning long-read sequencing from research applications to routine clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing
              </a>
            </td>
          <td>
            Qiuhui Li, A. Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome Research</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b54c36ea8dc8ae6d831b9cec1f08634da1f520c" target='_blank'>
              Abstract 1152: The landscape of copy number variations following WGD in 5,980 colorectal cancers
              </a>
            </td>
          <td>
            Jierui Xu, H. Walch, Paolo Manca, Maria A Perry, Christopher Fong, Justin Jee, K. Pichotta, E. Vakiani, Neil H. Segal, Adam Price, Allison L Richards, Chai Bandlamudi, M. Donoghue, A. Cercek, R. Yaeger, N. Schultz, W. Chatila
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b6b876ae54e45db144a86ddc16615e044025e57" target='_blank'>
              Abstract 5049: Haplotype-specific copy number profiling from long reads sequencing data
              </a>
            </td>
          <td>
            Tanveer Ahmad, M. Kolmogorov, A. Keskus, Ataberk Donmez, M. Farooqi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8be723b80118653ca09b51446fe4768c5505b8" target='_blank'>
              A Guide to Extrachromosomal DNA: Cancer's Dynamic Circular Genome.
              </a>
            </td>
          <td>
            Natasha E Weiser, Thomas B K Watkins, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Chromosomal rearrangements involving additional material in individuals with phenotypic alterations usually result in partial trisomy, often accompanied by partial monosomy. To characterize chromosomal rearrangements and analyze genomic characteristics in the breakpoint regions in 31 patients with additional material on an autosomal chromosome. Different tests were performed to characterize these patients, including karyotyping, chromosomal microarray analysis (CMA), and fluorescent in situ hybridization (FISH). In silico analyses evaluated A/B chromosomal compartments, segmental duplications, and repetitive elements at breakpoints. The 31 rearrangements resulted in 47 copy number variations (CNVs) and a range of structural aberrations were identified, including six tandem duplications, 19 derivative chromosomes, two intrachromosomal rearrangements, one recombinant, two dicentric chromosomes, and one triplication. A deleted segment was associated with the duplication in 16 of the 19 patients with derivative chromosomes from translocation. Among the trios whose chromosome rearrangement origin could be investigated, 54,5% were de novo, 31,9% were maternally inherited, and 13,6% were paternally inherited from balanced translocations or inversion. Breakpoint analysis revealed that 22 were in the A compartment (euchromatin), 25 were in the B compartment (heterochromatin), and five were in an undefined compartment. Additionally, 14 patients had breakpoints in regions of segmental duplications and repeat elements. Our study found that a deletion accompanied by additional genetic material was present in 51.6% of the patients, uncovering the underlying genetic imbalances. Statistical analyses revealed a positive correlation between chromosome size and the occurrence of CNVs in the rearrangements. Furthermore, no preference was observed for breakpoints occurring in compartments A and B, repetitive elements, or segmental duplications. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97077-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4680fc424cbc00a4561d2d03671779c2ded30f6" target='_blank'>
              Cytogenomic characterization of karyotypes with additional autosomal material
              </a>
            </td>
          <td>
            Priscila Soares Rodrigues, Bruna Burssed, F. Bellucco, Débora Cristina Batista Rosolen, Chong Ae Kim, M. Melaragno
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) are circular, double-stranded DNA molecules ubiquitously present across various organisms, playing a critical role in tumorigenesis and tumor progression. However, their precise contribution to prostate cancer (PCa) remains incompletely understood. To elucidate the function of eccDNAs in PCa, eccDNA sequencing and annotation were performed in PCa tissues and cell lines using Circle-seq. Amplified genes on eccDNAs were identified by cross-referencing annotated eccDNA-associated genes with those overexpressed in PCa based on TCGA data. Furthermore, eccDNA profiles were compared between Enzalutamide-sensitive and -resistant cell lines to investigate their role in resistance mechanisms. Notably, VAV2 was detected on both linear and circular DNA, as confirmed by PCR and Sanger sequencing. Functional analyses demonstrated that VAV2 overexpression promotes PCa proliferation and metastasis by activating the PAK1/AKT signaling pathway through PAK1 phosphorylation. Additionally, VAV2 contributes to Enzalutamide resistance by enhancing AR/ARv7 protein stability via reduced ubiquitination, mediated through the recruitment of the deubiquitinating enzyme USP48. These findings establish VAV2, identified through eccDNA sequencing, as a potential oncogene and a promising biomarker for PCa diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin’an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1dc55203f4f61b62677fd9af379d34ddca08f1" target='_blank'>
              Abstract 1479: Defining the interplay between DNAPKcs and shieldin during chromosomal break repair
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef8310a64d3e549ee7807fb7e90ca8640dfc591" target='_blank'>
              Programmed DNA breaks drive chromatin reconfiguration and facilitate transcriptional potentiation at neuronal early response genes
              </a>
            </td>
          <td>
            Lance Heady, Richard Rueda, Amir Segev, Krystal G. Morton, Ram Madabhushi
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22caf0363586789d3ef13130aa7bdb69bf876e9" target='_blank'>
              Nuclear and genome dynamics underlying DNA double-strand break repair.
              </a>
            </td>
          <td>
            I. Chiolo, Matthias Altmeyer, G. Legube, Karim Mekhail
          </td>
          <td>2025-03-17</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy J. Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6be55a5ef5db321d17ba7bfe59475decf1a2ca" target='_blank'>
              Abstract 2855: CDK12 inhibition remodels DNA replication to drive selective gene amplification
              </a>
            </td>
          <td>
            Shivani Yalala, Julia Kutz, Helmut Pospiech, Harri M Itkonen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac35aaec3d88c6b3cd4be700d6af5bf25aaebdef" target='_blank'>
              Drosophila SA1 expression prevents brain tumorigenesis and PARP-mediated cell elimination
              </a>
            </td>
          <td>
            Simona Totaro, Antonella Lettieri, Silvia Castiglioni, Francesco Lavezzari, C. Gervasini, Valentina Massa, Thomas Vaccari
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3537c6c5299b945e50a679b1495982f083cd02" target='_blank'>
              Abstract 5359: Multi-omics characterization of copy number variation in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yuxin Jin, Jing Qian, Michelle G. Webb, David W. Craig, Lynda D. Roman, John D. Carpten, R. Bassiouni
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, P. Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0036830d7ca0d3dd2d206e6880d31ee221a0279" target='_blank'>
              Advances in single-cell DNA sequencing enable insights into human somatic mosaicism.
              </a>
            </td>
          <td>
            Diane D. Shao, Andrea J. Kriz, Daniel A. Snellings, Zinan Zhou, Yifan Zhao, Liz Enyenihi, Christopher A. Walsh
          </td>
          <td>2025-04-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a4380bb34311ba8e2b2e583f5eb6ff7d05c33e" target='_blank'>
              Abstract 5050: Comparative analysis of somatic copy number variations and allelic imbalance in whole-genome bisulfite sequencing: challenges and insights
              </a>
            </td>
          <td>
            Yiru Gu, Jian Gu, P. Scheet, Justin Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68e6681ead255204363cfd6f85f96a1a0227012" target='_blank'>
              Genome-wide modeling of DNA replication in space and time confirms the emergence of replication specific patterns in vivo in eukaryotes
              </a>
            </td>
          <td>
            Dario D’Asaro, J. Arbona, Vinciane Piveteau, Aurèle Piazza, Cédric Vaillant, Daniel Jost
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96f5ae7db10ebf2238c71827c45c782f6d159f1b" target='_blank'>
              New insights into chromosomal regions 15p11.2 to 15q11.2 by studying submicroscopic variations using molecular cytogenetics.
              </a>
            </td>
          <td>
            Maria N Correia, Stefanie Kankel, I. Carreira, Joana B. Melo, Thomas Liehr
          </td>
          <td>2025-04-17</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C-and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce720a5b79eb9fb996bed883c2b0fde9438b7fa" target='_blank'>
              Genome evolution in parthenogenetic nematodes shaped by chromosome rearrangements and rare sex
              </a>
            </td>
          <td>
            George Chung, Lewis Stevens, Xiaoguang Dai, Scott Hickey, K. Kiontke, Priyesh Rughani, M. Kieninger, Joanna C. Collins, S. Juul, Fabio Piano, David H.A. Fitch, Mark Blaxter, Kristin C. Gunsalus
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The study examines Opisthorchis viverrini (OV)-related cholangiocarcinoma (CCA), a serious malignancy common in Southeast Asia. Through multi-regional whole-exome sequencing of 52 tumor samples and 13 adjacent tissues from 13 patients, significant intratumoral heterogeneity (ITH) and inter-patient heterogeneity are shown. Chronic liver fluke infection induces a distinct mutational landscape, with 48–90% of mutations concentrated in each region of the tumor. The average mutation burden is 95 non-synonymous mutations per area, exceeding previous CCA investigations. Critical driver mutations in TP53, SMAD4, and other genes underscore their significance in pathogenesis. Mutational markers elucidate mechanisms including spontaneous deamination and impaired DNA repair. Unique mutation patterns distinguish OV-associated CCA from other variants. Chromosomal instability in patient K110 signifies aggressive tumor behavior and unfavorable prognosis. Targetable mutations such as ERBB2 underscore the possibility for personalized therapeutics. These findings underscore the necessity for personalized strategies for treatment that target both trunk and branch mutations in endemic areas. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95142-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48c940bd80f998f283a697250c46b3404991cb4" target='_blank'>
              Whole exome sequencing of multi-regions reveals tumor heterogeneity in Opisthorchis viverrini-associated cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, A. Jusakul, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Bin Tean Teh, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In vivo genetic diversifiers have previously enabled efficient searches of genetic variant fitness landscapes for continuous directed evolution. However, existing genomic diversification modalities for mammalian genomic loci exclusively rely on deaminases to generate transition mutations within target loci, forfeiting access to most missense mutations. Here, we engineer CRISPR-guided error-prone DNA polymerases (EvolvR) to diversify all four nucleotides within genomic loci in mammalian cells. We demonstrate that EvolvR generates both transition and transversion mutations throughout a mutation window of at least 40 bp and implement EvolvR to evolve previously unreported drug-resistant MAP2K1 variants via substitutions not achievable with deaminases. Moreover, we discover that the nickase’s mismatch tolerance limits EvolvR’s mutation window and substitution biases in a gRNA-specific fashion. To compensate for gRNA-to-gRNA variability in mutagenesis, we maximize the number of gRNA target sequences by incorporating a PAM-flexible nickase into EvolvR. Finally, we find a strong correlation between predicted free energy changes underlying R-loop formation and EvolvR’s performance using a given gRNA. The EvolvR system diversifies all four nucleotides to enable the evolution of mammalian cells, while nuclease and gRNA-specific properties underlying nickase fidelity can be engineered to further enhance EvolvR’s mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4485500132f714892fcf79f15fa49a1b187bbe0" target='_blank'>
              Nickase fidelity drives EvolvR-mediated diversification in mammalian cells
              </a>
            </td>
          <td>
            Juan E Hurtado, Adam J. Schieferecke, Shakked O Halperin, John Guan, Dylan Aidlen, David V. Schaffer, J. Dueber
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb48bc35dc30e8efcbd8268b117fd6382f30da8" target='_blank'>
              Abstract 2395: The role of exonized mobile elements in chimeric transcript emergence in glioblastoma
              </a>
            </td>
          <td>
            Rafael L. V. Mercuri, T. L. Miller, Filipe F. dos Santos, M. D. de Lima, Aline Rangel-Pozzo, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3738bec14783d53b32ea0221a11feadb11d7ceea" target='_blank'>
              Abstract 176: Simultaneous spatial characterisation of copy number change and immune cell profiling using DNA FISH and multiplexed immunofluorescence
              </a>
            </td>
          <td>
            Eleanor M O'Roberts, Pranauti Panshikar, Xiaoze Li-Wang, Quentin Verron, S. S. Fard, Magda Bienko, Charlotte Stadler
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94444bc68fc048c7923ba3b90250650c889cfa29" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cc40d7bb5bed72dbfa57e9bb607d9efbd812f2" target='_blank'>
              Abstract 6659: Defective lipid homeostasis induced by QKI loss is a mutator on gliomagenesis
              </a>
            </td>
          <td>
            Yinglu Guan, Takashi Shingu, Kaylene Lu, Ece Kilinc, Traver G. Hart, Jianhua Zhang, Yiwen Chen, Betty Y. S. Kim, Jian Hu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chemo-/radioresistance of malignant tumors hampers cancer control and increases patient mortality. Efficient repair of damaged DNA is critical for the maintenance of genomic integrity and fidelity of genetic information. In reverse, increased DNA repair capability in cancer cells contributes to chemo-/radioresistance of malignant tumors. DNA double-strand break (DSB) is the most serious DNA damage and is also the principal molecular basis of radiotherapy. Upon DNA damage, the Ku80 is recruited and forms a critical DNA-PK complex at the DSB sites with Ku70 and the catalytic subunit (DNA-PKcs) to initiate DNA repair. How DNA-PK is assembled and activated is not fully understood. Based on the identification of radiation-reduced Ku80 K568 crotonylation through quantitative global lysine crotonylome analysis, we reveal that Ku80 K568 is crotonylated by p300-CBP-associated factor (PCAF). Upon DNA damage, the K568cr is decrotonylated by HDAC8 (Histone deacetylase 8). Decrotonylation of K568cr empties this site for the subsequent SUMOylation of Ku80 by CBX4. The conversion of Ku80 from K568 crotonylation to SUMOylation facilitates the assembly of DNA-PK complex and autophosphorylation of DNA-PKcs S2056, consequently activating the DSB repair. Moreover, mutation disrupting the post-translational modification (PTM) of Ku80 K568 site sensitizes cancer cells to radiotherapy in tumor-bearing nude mice models. This study elucidates the conversion model between two different forms of PTMs in the regulation of DNA-PK complex assembly and DSB repair, highlighting this model’s potential in controlling chemo-/radioresistance of malignant tumors, as well as expands the atlas of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e28a599caae10368a0a56703d29a0e46b8a2e" target='_blank'>
              Conversion of Ku80 K568 crotonylation to SUMOylation facilitates DNA non-homologous end joining and cancer radioresistance
              </a>
            </td>
          <td>
            Hongling Zhao, Shan-shan Gao, Yang Han, Dafei Xie, Lihui Xuan, Xin Huang, Jinhua Luo, Q. Ran, Gang Li, Hejiang Guo, Weixiang Hu, Jin Jia, Xiaochang Liu, Yuhao Liu, Jinpeng Tan, Chenjun Bai, Yongqing Gu, Teng Ma, Zhongjun Li, Hua Guan, Ruixue Huang, Ping-Kun Zhou
          </td>
          <td>2025-04-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4868141c39dc560c925882ee816da6d52c4deb" target='_blank'>
              Uev1A counteracts oncogenic Ras stimuli in both polyploid and diploid cells
              </a>
            </td>
          <td>
            Qi Zhang, Yunfeng Wang, Xueli Fu, Ziguang Wang, Yang Zhang, Lizhong Yan, Yuejia Wang, Muhan Yang, Dongze Song, Ruixing Zhang, Hongru Zhang, Shian Wu, Shaowei Zhao
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct “fragmentomic” characteristics that are related with their cellular origins. Despite the wealth of low-pass whole genome sequencing data from recent cancer liquid biopsy studies, somatic variants within this data have often been overlooked without detailed characterization. Here, we reported that in low-pass sequencing data, somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, our investigation into cfDNA fragmentomics showed that even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like traits, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilized artificial intelligence to integrate genomic, fragmentomic, and epigenomic features, developing diagnostic models named FreeSV (Fragmentomic and epigenomic examinations of Somatic Variants in cfDNA) and FreeSV-m. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features were also included, the AUCs of FreeSV-m model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64ed6f9ef46f8ac541b4cf9a70df9227b7d0f90" target='_blank'>
              Cancer-like fragmentomic characteristics of somatic variants in cell-free DNA
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Wanqiu Wang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93a4683d0794daa101a332796e1ced942dd7fd5" target='_blank'>
              A Novel de novo Exceptional Complex Chromosomal Rearrangement Involving 5 Chromosomes Resulting in Neurodevelopmental Delay and Dysmorphism.
              </a>
            </td>
          <td>
            Sabri Aynacı, Sinem Kocagil, C. Yarar, Efsun Tosumoğlu, Ebru Erzurumluoğlu Gökalp, Mehmet Burak Mutlu, S. Artan
          </td>
          <td>2025-03-25</td>
          <td>Molecular syndromology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d143808abe3792603e16e2ab02c9eaad5e2640" target='_blank'>
              Abstract 2409: Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways
              </a>
            </td>
          <td>
            Joel Pardo, N. Temiz, Douglas Yee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cc2e0fd7fc6d0c80101963194f461583f9f563a" target='_blank'>
              Adeno-Associated Virus 2 (AAV2) - induced RPA exhaustion generates cellular DNA damage and restricts viral gene expression
              </a>
            </td>
          <td>
            Monnette F. Summers, MegAnn K. Haubold, Marcel Morgenstern, Phoenix Shepherd, Clairine I. S. Larsen, Ava E. Bartz, Gopishankar Thirumoorthy, R. N. Kirchdoerfer, Joshua J. Coon, Kavi P M Mehta, Kinjal Majumder
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks arising from base excision repair (BER) during the misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination–deficient (HRD) cells. At templated AP sites, cross-links between the DNA and 5-hydroxymethylcytosine binding, embryonic stem cell–specific (HMCES) regulate replication fork speed, avoiding cytotoxic double-strand breaks. While the role of HMCES at the template DNA strand is well studied, its effects on nascent DNA are not. We provide evidence that HMCES–DNA-protein cross-links (DPCs) are detrimental to the BER-mediated removal of 5-hydroxymethyl-2′-deoxycytidine (5hmdC)–derived 5-hydroxymethyl-2′-deoxyuridine from replication forks. HRD cells have heightened HMCES-DPCs, which increase further upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP sites. HMCES depletion substantially suppresses 5hmdC-mediated replication fork defects, chromosomal aberrations, and cell death in HRD cells. This reveals that HMCES-DPCs are a source of BER-initiated single-stranded DNA gaps, which indicates that endogenous DPCs contribute to genomic instability in HRD tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622156d932ecc5d7d331e4371a1b29a1f41c3ef" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination–deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martín, Marta del Rio Oliva, Yodhara Wijesekara Hanthi, Sara Berrada, Raimundo Freire, Jean-Yves Masson, José Carlos Reyes, Vincenzo Costanzo, Iván V. Rosado
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa24230eaf15e64077b2640e42b580f6dcef4855" target='_blank'>
              Abstract 2838: Modeling putative oncogenic variants of human topoisomerase II by genome editing of a near haploid cell line
              </a>
            </td>
          <td>
            K. C. Nitiss, Brigid Conroy, Maureen N. McCoy, J. Nitiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Homology, or relationship among characters by common descent, has been notoriously difficult to assess for many morphological features, and cell types in particular. The ontogenetic origin of morphological traits means that the only physically inherited information is encoded in the genomes. However, the complexity of the underlying gene regulatory network and often miniscule changes that can impact gene expression, make it practically impossible to postulate a clear demarcation line for what molecular signature should "define" a homologous cell type between two deeply branching animals. In this Hypothesis article, we propose the use of the recently characterized irreversible genomic states, that occur after chromosomal and sub-chromosomal mixing of genes and regulatory elements, to dissect regulatory signatures of each cell type into irreversible and reversible configurations. While many of such states will be non-functional, some may permanently impact gene expression in a given cell type. Our proposal is that such evolutionarily irreversible, and thus synapomorphic, functional genomic states can constitute a criterion for the timing of the origin of deep evolutionary cell type homologies. Our proposal thus aims to close the gap between the clearly defined homology of the individual genomic characters and their genomic states to the homology at the phenotypic level through the identification of the underlying evolutionarily irreversible and regulatory linked states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1c7265ffd0750ec9e349f309b4f00f20641d114" target='_blank'>
              The application of irreversible genomic states to define and trace ancient cell type homologies
              </a>
            </td>
          <td>
            Oleg Simakov, Günter P. Wagner
          </td>
          <td>2025-05-03</td>
          <td>EvoDevo</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54ccd0aa3ae124063240ed82d6af46f8d425d72" target='_blank'>
              Aneuploidy generates enhanced nucleotide dependency and sensitivity to metabolic perturbation.
              </a>
            </td>
          <td>
            R. Magesh, Arshia N Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, Jessica B Spinelli, S. Elledge, E. Watson
          </td>
          <td>2025-05-05</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Addiction to oncogene-rewired transcriptional networks is a therapeutic vulnerability in cancer cells, underscoring a need to better understand mechanisms that relay oncogene signals to the transcriptional machinery. Here, using human and mouse T cell acute lymphoblastic leukemia (T-ALL) models, we identify an essential requirement for the endosomal sorting complex required for transport protein CHMP5 in T-ALL epigenetic and transcriptional programming. CHMP5 is highly expressed in T-ALL cells where it mediates recruitment of the coactivator BRD4 and the histone acetyl transferase p300 to enhancers and super-enhancers that enable transcription of T-ALL genes. Consequently, CHMP5 depletion causes severe downregulation of critical T-ALL genes, mitigates chemoresistance and impairs T-ALL initiation by oncogenic NOTCH1 in vivo. Altogether, our findings uncover a non-oncogene dependency on CHMP5 that enables T-ALL initiation and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768bb462ad3f5f7bfdc0d91a1f7eb42525d3db0" target='_blank'>
              The ESCRT protein CHMP5 promotes T cell leukemia by enabling BRD4-p300-dependent transcription
              </a>
            </td>
          <td>
            Katharine Umphred-Wilson, Shashikala Ratnayake, Qianzi Tang, Rui Wang, S. Chaudhary, Devaiah N Ballachanda, Josephine Trichka, Jan Wisniewski, Lan Zhou, Qing-rong Chen, D. Meerzaman, Dinah S. Singer, S. Adoro
          </td>
          <td>2025-05-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genotoxic therapies such as ionizing radiation eliminate cancer cells by inducing extensive DNA damage but often cause normal tissue toxicity, including cutaneous injury. Extrachromosomal circular DNA (eccDNA) refers to circular DNA fragments outside the chromosomal context, with their formation and persistence linked to DNA damage repair and genomic instability. Despite growing recognition of eccDNA in oncogenesis, its role under genotoxic stress in normal tissues remains poorly understood. Here, eccDNA is profiled in irradiated rat skin using Circle-seq, identifying alterations in eccDNA number and composition. Specifically, radiation induced circle17:44148731-48208624, in which vacuolar protein sorting 41 homolog (VPS41) is the sole radiation-induced amplification gene by semiquantitative PCR and gel electrophoresis. The findings show that eccDNA or VPS41 overexpression reduces radiation-induced skin injury (RISI) in vitro and in vivo. Proteomic and interaction analyses identified metastasis suppressor kangai-1 (KAI1) as a VPS41-interacting partner. Notably, VPS41 overexpression promotes KAI1 lysosomal degradation, protecting against radiation-induced apoptotic cell death. Peptide array analysis pinpoints the VPS41-KAI1 interaction through the K263 residue, consistent with AlphaFold prediction. The findings uncover a novel mechanism in which radiation-induced eccDNA, specifically VPS41, mitigates skin injury by modulating KAI1 degradation. This study highlights the role of eccDNA in cellular defense, providing strategies to enhance tissue resilience to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821c77d7a2a02d4160cf107b29b6628f9d6cb1ed" target='_blank'>
              EccDNA-Driven VPS41 Amplification Alleviates Genotoxic Stress via Lysosomal KAI1 Degradation.
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Jinchen He, Bin Song, Hao Bai, Fengdi Jiang, Yining Zhang, Qian Li, Tao Yan, Wenling Tu, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability.
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) occupy 8% of the human genome. Although most HERV integrations are severely degenerated by mutations, the most recently integrated proviruses, such as members of the HERV-K HML-2 subfamily, partially retain regulatory and protein-coding capacity. The precise number of HML-2 proviral copies in the modern human population is constantly changing in literature, as new integrations are being uncovered. The first comprehensive list of HML-2 proviral loci was compiled in 2011, including a total of 91 proviruses. Since then, multiple articles published additions and modifications to that list, mainly in the form of new polymorphic proviral sites, updated chromosomal band characterizations or the correspondence of coordinates in the new version of the published human reference genome. In the present study, we systematically searched the literature for lists of HML-2 proviruses and their coordinates and cross-examined every proviral locus information, also against the human genome. We gathered all available data about all HML-2 proviral integrations identified to date and updated, corrected and refined the coordinates in both human genome assemblies currently used in research, to incorporate the whole provirus in each case. Thereby we present an exhaustive (to date) catalogue of all known HML-2 proviruses and their respective coordinates, as a powerful tool for studies aiming to decipher HERV role in health and disease, especially for high-throughput data analyses, which could lead to the discovery of links between specific HERV integrations and biological mechanisms or medical disorders. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00359-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c87290cae809a847d576ad6894bc21b0d56f20b2" target='_blank'>
              Compilation of all known HERV-K HML-2 proviral integrations
              </a>
            </td>
          <td>
            Eleni Kyriakou, G. Magiorkinis
          </td>
          <td>2025-05-07</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are widely existing stable DNA secondary structures in mammalian cells. A long-standing hypothesis is that timely resolution of G4s is needed for efficient and faithful DNA replication. In vitro, G4s may be unwound by helicases or alternatively resolved via DNA2 nuclease mediated G4 cleavage. However, little is known about the biological significance and regulatory mechanism of the DNA2-mediated G4 removal pathway. Here, we report that DNA2 deficiency or its chemical inhibition leads to a significant accumulation of G4s and stalled replication forks at telomeres, which is demonstrated by a high-resolution technology: Single molecular analysis of replicating DNA (SMARD). We further identify that the DNA repair complex MutSα (MSH2-MSH6) binds G4s and stimulates G4 resolution via DNA2-mediated G4 excision. MSH2 deficiency, like DNA2 deficiency or inhibition, causes G4 accumulation and defective telomere replication. Meanwhile, G4-stabilizing environmental compounds block G4 unwinding by helicases but not G4 cleavage by DNA2. Consequently, G4 stabilizers impair telomere replication and cause telomere instabilities, especially in cells deficient in DNA2 or MSH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding evolutionary dynamics is critical for unraveling the complex progression of diseases such as cancer. Cancer evolution is inherently a temporal process driven by the accumulation of mutations and clonal expansions over time. Traditional phylogenetic methods often rely solely on static, cross-sectional data, limiting their ability to infer the timing of key evolutionary events. To address this challenge, we developed NestedBD-Long, a novel method that integrates temporal data from longitudinal sampling into phylogenetic analyses using the birthdeath evolutionary model on copy numbers. This approach allows for the direct mapping of real-world time onto inferred evolutionary trees, providing a clearer and more accurate representation of cancer’s evolutionary trajectory. Evaluations demonstrate that NestedBD-Long outperforms traditional approaches, with accuracy improving as the number of temporal sampling points increases. This advancement provides a powerful framework for studying tumor progression, treatment resistance, and metastatic spread by capturing the dynamics between evolutionary events and real-world timelines. NestedBD-Long is available at https://github.com/Androstane/NestedBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ed9e4b0ce33758b268836e54dd2099f9eeb3c4" target='_blank'>
              Inferring Phylogenetic Trees of Cancer Evolution from Longitudinal Single-Cell Copy Number Profiles
              </a>
            </td>
          <td>
            Yushu Liu, L. Nakhleh
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="A prominent feature of eukaryotic chromosomes are centromeres, which are specialized regions of repetitive DNA required for faithful chromosome segregation during cell division. In interphase cells, centromeres are non-randomly positioned in the three-dimensional space of the nucleus in a cell type-specific manner. The functional relevance and the cellular mechanisms underlying this localization are unknown, and quantitative methods to measure distribution patterns of centromeres in 3D space are needed. Here, we developed an analytical framework that combines sensitive clustering metrics and advanced modeling techniques for the quantitative analysis of centromere distributions at the single-cell level. To identify a robust quantitative measure for centromere clustering, we benchmarked six metrics for their ability to sensitively detect changes in centromere distribution patterns from high-throughput imaging data of human cells, both under normal conditions and upon experimental perturbation of centromere distribution. We found that Ripley’s K function has the highest accuracy with minimal sensitivity to variations in the number of centromeres, making it the most suitable metric for measuring centromere distributions. As a complementary approach, we also developed and validated spatial models to replicate centromere distribution patterns, and we show that a radially shifted Gaussian distribution best represents the centromere patterns seen in human cells. Our approach creates tools for the quantitative characterization of spatial centromere distributions with applications in both targeted studies of centromere organization and unbiased screening approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174a17b1ef834044537c7e22b9436da4a482c609" target='_blank'>
              Simulation and Quantitative Analysis of Spatial Centromere Distribution Patterns
              </a>
            </td>
          <td>
            Adib Keikhosravi, Krishnendu Guin, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-03-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea8da16c37659abb745c723fb795dcd02f7d43b2" target='_blank'>
              Quantitative AI-based DNA fiber workflow to study replication stress
              </a>
            </td>
          <td>
            Paolo Fagherazzi, Timo Diekmann, Alessandra Ardizzoia, Zuzana Machacova, Simran Negi, A. K. Yadav, Pavel Moudry, Vincenzo Costanzo, Hana Polasek-Sedlackova
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05f154cb832f6f7744c5bac28235208f5d152b7" target='_blank'>
              Mitochondrial DNA copy numbers in gastric cancer tissues: a possible biomarker for estimating cancer progression.
              </a>
            </td>
          <td>
            Hisanori Fukunaga, Mayuko Fukunaga
          </td>
          <td>2025-04-23</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In the course of antibody affinity maturation, germinal centre (GC) B cells mutate their immunoglobulin heavy- and light-chain genes in a process known as somatic hypermutation (SHM)1–4. Panels of mutant B cells with different binding affinities for antigens are then selected in a Darwinian manner, which leads to a progressive increase in affinity among the population5. As with any Darwinian process, rare gain-of-fitness mutations must be identified and common loss-of-fitness mutations avoided6. Progressive acquisition of mutations therefore poses a risk during large proliferative bursts7, when GC B cells undergo several cell cycles in the absence of affinity-based selection8–13. Using a combination of in vivo mouse experiments and mathematical modelling, here we show that GCs achieve this balance by strongly suppressing SHM during clonal-burst-type expansion, so that a large fraction of the progeny generated by these bursts does not deviate from their ancestral genotype. Intravital imaging and image-based cell sorting of a mouse strain carrying a reporter of cyclin-dependent kinase 2 (CDK2) activity showed that B cells that are actively undergoing proliferative bursts lack the transient CDK2low ‘G0-like’ phase of the cell cycle in which SHM takes place. We propose a model in which inertially cycling B cells mostly delay SHM until the G0-like phase that follows their final round of division in the GC dark zone, thus maintaining affinity as they clonally expand in the absence of selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029979c8692579fbda44fe3f3e73b853b5d3c6f1" target='_blank'>
              Transient silencing of hypermutation preserves B cell affinity during clonal bursting
              </a>
            </td>
          <td>
            Juhee Pae, Niklas Schwan, Bertrand Ottino-Loffler, William S DeWitt, Amar Garg, Juliana Bortolatto, Ashni Vora, Jin-Jie Shen, Alvaro Hobbs, Tiago B R Castro, L. Mesin, Frederick A. Matsen, Michael Meyer-Hermann, G. Victora
          </td>
          <td>2025-03-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differential alternative splicing transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes including vFLIP, vCyclin and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells, and significantly inhibited KSHV-driven proliferation, cellular transformation and tumorigenesis. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. Importance Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1dd5dbd2f3302db90bfa70c7f9ae17b8e6ba3f" target='_blank'>
              KSHV Reprograms Host RNA Splicing via FAM50A to Activate STAT3 and Drive Oncogenic Cellular Transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Karla Paniagua, Ling Ding, Xian Wang, Yufei Huang, Mario A Flores, Shou-Jiang Gao
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT We investigated the presence of viral DNA and RNA in cutaneous squamous cell carcinoma (cSCC) tumor and normal tissues from nine individuals with a history of hematopoietic stem cell transplantation (HCT). Microbiome quantification through DNA and RNA sequencing (RNA-seq) revealed the presence of 18 viruses in both tumor and normal tissues. DNA sequencing (DNA-seq) identified Torque teno virus, Saimiriine herpesvirus 1, Merkel cell polyomavirus, Human parvovirus B19, Human gammaherpesvirus-4, Human herpesvirus-6, and others. RNA-seq revealed additional viruses such as Tobamovirus, Pinus nigra virus, Orthohepadnavirus, Human papillomavirus-5, Human herpesvirus-7, Human gammaherpesvirus-4, Gammaretrovirus, and others. Notably, DNA-seq indicated that tumor samples exhibited low levels of Escherichia virus in three out of nine subjects and elevated levels of Human gammaherpesvirus-4 in one subject, while normal samples frequently contained Gammaretrovirus and occasionally Escherichia virus. A comparative analysis using both DNA- and RNA-seq captured three common viruses: Abelson murine leukemia virus, Murine type C retrovirus, and Human gammaherpesvirus-4. These findings were corroborated by an independent data set, supporting the reliability of the viral detection methods utilized. The study provides insights into the viral landscape in post-HCT patients, emphasizing the need for comprehensive viral monitoring in this vulnerable population. IMPORTANCE This study is important because it explores the potential role of viruses in the development of cSCC in individuals who have undergone allogeneic HCT. cSCC is common in this population, particularly in those with chronic graft-versus-host disease on long-term immunosuppression. By using advanced metagenomic and metatranscriptomic next-generation sequencing, we aimed to identify viral pathogens present in tumor and adjacent normal tissue. The results could lead to targeted preventive or therapeutic interventions for these high-risk people, potentially improving their outcomes and management of cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56b53109651f083205e0fc320c14b9f0267e185" target='_blank'>
              Mining metagenomes and metatranscriptomes unveils viruses associated with cutaneous squamous cell carcinoma in hematopoietic stem cell transplant recipients
              </a>
            </td>
          <td>
            Jana K Dickter, Yuqi Zhao, Vishwas Parekh, Huiyan Ma, Badri G Modi, Wai-Yee Li, S. Armenian, Xiwei Wu, Farah R Abdulla
          </td>
          <td>2025-03-31</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5545243aab210759476cd520384aa54505acdf5" target='_blank'>
              SAUNA - Simulated Annealing for Unique Nucleosome Arrangements from Cell-Free DNA-Sequencing Data
              </a>
            </td>
          <td>
            Maria Rafaela Lambrinos, Tara Rahimi, Tarun Mehra, Michael P Schmid, Nicola Miglino, A. Romanens, Kirsten D. Mertz, Michael Krauthammer, Ronny Nienhold, Andreas Wicki, Zsolt Balázs
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome replication in eukaryotic cells is surveyed by the S-phase checkpoint, which orchestrates sequential replication origin activation to avoid exhaustion of hitherto poorly defined rate-limiting replisome components. Here, we find that excessive activation of replication origins depletes chromatin-bound PCNA and lagging strand components, thereby limiting additional PCNA loading at new origins when checkpoint control is disrupted. PAF15 (PCNA-associated factor 15) emerges as a dosage-sensitive regulator of PCNA, delineating the dynamic range of global genome duplication and defining distinct roles for PCNA on the leading and lagging strands. Through its high-affinity PIP motif and interaction within the DNA encircling channel of PCNA, PAF15 stabilizes PCNA exclusively on the lagging strand, optimizing and rate-limiting lagging strand processing. On the other hand, misregulation of PAF15—whether by overexpression or mislocalization to the leading strand—impairs replication fork progression and leads to cell death. These defects are mitigated by TIMELESS and CLASPIN, which restrain PAF15-PCNA interactions beyond the lagging strand. E2F4-mediated repression orchestrates PAF15 expression in normal and cancer cells, maintaining its optimal dosage for lagging strand-specific interactions with PCNA. Thus, the S-phase checkpoint functions in concert to restrict origin activation when lagging strand PAF15-PCNA assembly is exhausted, linking a previously concealed strand-specific rate limitation to overall replication dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe32e8db1ccfcee1501ea6b48ca4b9c67236d177" target='_blank'>
              PAF15-PCNA assembly exhaustion governs lagging strand replication and replisome integrity
              </a>
            </td>
          <td>
            Gita Chhetri, S. Badugu, Narcis-Adrian Petriman, Mikkel Bo Petersen, Ganesha P. Pitchai, Aylin Seren Güller, Jan Novotný, Barath Balarasa, Morten Frendø Ebbesen, Frederik Tibert Larsen, Tina Ravnsborg, A. K. Yadav, Anita Lunding, Hana Polasek-Sedlackova, Ole Nørregaard Jensen, Kim Ravnskjaer, Jonathan R. Brewer, Jesper G.S. Madsen, Jens S. Andersen, Kumar Somyajit
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, J. Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e00eb9adcac5876d77799de1f3106951576868" target='_blank'>
              Unveiling the BRAF fusion structure variations through DNA and RNA sequencing.
              </a>
            </td>
          <td>
            Qiong Yang, Baoming Wang, Xuli Meng, Tonghui Ma, Zhenyuan Qian, Xufan Cai, Xiaochen Sun, Yan Xu, Qifeng Luo, Tianyao Yang, H. Tao, Yangyang Qian, Jianzhong Shao, Zaiyuan Ye
          </td>
          <td>2025-04-19</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8774da0a06c116c814c2652572a737b8d86a0c02" target='_blank'>
              Genome-wide ribonucleotide detection in Archaea
              </a>
            </td>
          <td>
            Y. Moalic, Maurane Reveil, Deepali L. Kundnani, S. Balachander, Taehwan Yang, Alli L. Gombolay, Farahnaz Ranjbarian, Raphaël Brizard, Patrick Durand, H. Myllykallio, Mohamed Jebbar, Anders Hofer, F. Storici, Ghislaine Henneke
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0c142c6c3dbc24f6010a9b80df6aa28614056f" target='_blank'>
              The Spatial Location of Chromosomes in Dividing Cells and the Relative Stability of Chromosome Spatial Structure
              </a>
            </td>
          <td>
            Chunxiao Wu, Dongyun Jiang, Tao Jiang, Luxia Xu, Qian Xu, Meng Zhao, Qin Zhu, Zhigang Guo, Jinlan Pan, Suning Chen
          </td>
          <td>2025-04-02</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Bleomycin (Bleo) is a glycopeptide with potent antitumor activity that induces DNA double-strand breaks (DSBs) through free radical generation, similar to ionizing radiation (IR). Therefore, Bleo is considered a radiomimetic drug. However, differences in DNA repair mechanisms between IR- and Bleo-induced DNA damage have not been fully elucidated. Therefore, in the present study, we examined a panel of repair-deficient human TK6 cell lines to elucidate the relative contributions of individual repair factors. Results Our comprehensive profiling indicated that both non-homologous end joining (NHEJ) and homologous recombination (HR) contributed to DSB repair induced by X-rays and Bleo. Furthermore, tyrosyl-DNA phosphodiesterase (TDP)-related repair was a significant factor for cellular sensitivity to Bleo treatment. TDP1−/−/TDP2−/− cells exhibited greater sensitivity to Bleo than TDP1−/− or TDP2−/− cells, but not to X-rays. In addition, we determined whether TDP2 is involved in the repair of Bleo-induced DSBs using a neutral comet assay. In TDP1-deficient cells, knockout of TDP2 resulted in a significant delay in the repair kinetics of DSBs induced by Bleo, but not by X-rays. Conclusions The contribution of the TDP-related pathway to DSB repair significantly differed between IR and radiomimetic drugs. The discovery of this novel TDP2-dependent repair of DSBs resulting from radiomimetic drug exposure indicates that TDP1 and TDP2 inhibition in combination with radiomimetic drugs represents a strategy for cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00329-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b896deb7b39c22219dcb649c924ec1a29423e547" target='_blank'>
              Role of TDP2 in the repair of DNA damage induced by the radiomimetic drug Bleomycin
              </a>
            </td>
          <td>
            N. Shimizu, Kazuki Izawa, Mubasshir Washif, Ryosuke Morozumi, Kouji Hirota, Masataka Tsuda
          </td>
          <td>2025-03-28</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="ABSTRACT Double-strand breaks (DSBs) in DNA pose a critical threat to genomic integrity, potentially leading to the onset and progression of various diseases, including cancer. Cellular responses to such lesions entail sophisticated repair mechanisms primarily mediated by non-homologous end joining (NHEJ) and homologous recombination (HR). Interestingly, the efficient recruitment of repair proteins and completion of DSB repair likely involve complex, inter-organelle communication and coordination of cellular components. In this study, we report a role of γ-tubulin in DSB repair. γ-tubulin is a major microtubule nucleation factor governing microtubule dynamics. We show that γ-tubulin is recruited to the site of DNA damage and is required for efficient DSB repair via both NHEJ and HR. Suppression of γ-tubulin impedes DNA repair and exacerbates DNA damage accumulation. Furthermore, γ-tubulin mediates the mobilization and formation of DNA damage foci, which serve as repair centers, thereby facilitating the recruitment of HR and NHEJ repair proteins on damaged chromatin. Finally, pharmacological inhibition of γ-tubulin enhances the cytotoxic effect of DNA-damaging agents, consistent with the DNA repair function of γ-tubulin, and underscoring the potential of its therapeutic intervention in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1324a05392de89320b697d282b9ec5e482ee4e56" target='_blank'>
              γ-tubulin mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Abhishikt David Solomon, Odjo G. Gouttia, Ling Wang, Songli Zhu, Feifei Wang, Yanqui Li, Mohammadjavad Paydar, T. Bessho, Benjamin H Kwok, A. Peng
          </td>
          <td>2025-03-15</td>
          <td>Journal of Cell Science</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Inverted repeats are repetitive elements that can form hairpin and cruciform structures. They are linked to genomic instability; however, they also have various biological functions. Their distribution differs markedly across taxonomic groups in the tree of life, and they exhibit high polymorphism due to their inherent genomic instability. Advances in sequencing technologies and declined costs have enabled the generation of an ever-growing number of complete genomes for organisms across taxonomic groups in the tree of life. However, a comprehensive database encompassing inverted repeats across diverse organismal genomes has been lacking. We present invertiaDB, the first comprehensive database of inverted repeats spanning multiple taxa, featuring repeats identified in the genomes of 118 101 organisms across all major taxonomic groups. For each organism, we derived inverted repeats with arm lengths of at least 10 bp, spacer lengths up to 8 bp, and no mismatches in the arms. The database currently hosts 34 330 450 inverted repeat sequences, serving as a centralized, user-friendly repository to perform searches and interactive visualizations, and download existing inverted repeat data for independent analysis. invertiaDB is implemented as a web portal for browsing, analyzing, and downloading inverted repeat data. invertiaDB is publicly available at https://invertiadb.netlify.app/homepage.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c7a86ee0357787552c41e04bfb3755f44fbabf" target='_blank'>
              invertiaDB: a database of inverted repeats across organismal genomes
              </a>
            </td>
          <td>
            Kimonas Provatas, Nikol Chantzi, Nafsika Amptazi, Michail Patsakis, Akshatha Nayak, I. Mouratidis, A. Zaravinos, Georgos A. Pavlopoulos, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Centromeres are pivotal for accurate chromosome segregation, yet their regulation and evolutionary dynamics remain poorly understood. Here, we investigate centromeres of the diploid species Gossypium anomalum (Ga, B-genome) that were transferred into tetraploid cotton G. hirsutum (Gh, AD-genome) as either an additional or integrated chromosome, as well as in synthetic allohexaploid (AABBDD) lines. We demonstrate consistent size reduction for all Ga centromeres in the Gh background. Histone modification profiling across 10 marks revealed heightened levels of both active and repressive chromatin marks within the Ga centromeres when transferred into the Gh background, particularly for H3K36me2. The centromeric histone modification perturbation extended into pericentromeric regions, with variable CENH3-binding domains consistently exhibiting a more pronounced increase in histone modification levels compared to stable centromere regions, highlighting the role of histone modification elevation in centromere dynamics. In addition, we observed enhanced chromatin accessibility and the presence of non-B-form DNA motifs, such as A-phased DNA repeats within stable centromere domains that are correlated with centromere stability. Hi-C analysis reveals a reorganized 3D chromatin architecture within the introgression line centromeres, including the formation of new topologically associating domains linked to H3K36me2 dynamics, emphasizing the importance of H3K36me2 in centromere organization. Together, these findings elucidate epigenetic mechanisms underlying centromere composition following intergenomic hybridization and allopolyploid formation, offering insights into centromere evolution in plants and its myriad epigenetic and potentially functional dimensions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cd7b66a7f1ffe44365de20596b44e77ed523aee" target='_blank'>
              Centromere-size reduction and chromatin state dynamics following intergenomic hybridization in cotton.
              </a>
            </td>
          <td>
            Jinlei Han, Guanjing Hu, Yan Dai, Xin Zhang, Jingjing Tian, Jialiang Zhou, Xinqi Xu, Qi Chen, Xiaobing Kou, Lei Xu, Xinyu Wu, Ziying Sun, Jiahui Geng, Lin Li, Chenyu Qiu, Teame Gereziher Mehari, Baohua Wang, Hui Zhang, Xinlian Shen, Zhenzhen Xu, Jonathan F. Wendel, Kai Wang
          </td>
          <td>2025-05-02</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c91a033b11a5d910b45df1fe7d4556a0c6c8b6e" target='_blank'>
              Abstract 5585: DNA methylation and transcription factors regulate overexpression of PVT1 in prostate cancer
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Colin Finnegan, Murtaza Barkarar, Oluwabusola Ajagbe, Rachel E. Bonacci, O. Ogunwobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d51acfd255e9f2c3851395fb88bf941b1c544b" target='_blank'>
              Abstract 1735: The discovery of PKMYT1 inhibitors to exploit synthetic lethality caused by replication stress
              </a>
            </td>
          <td>
            R. Marais, J. Grabarek, M. Ioannou, L. Leung, A. Raoof, Mike Brown, D. Smithen, C. Fitzpatrick, C. Chester, J. Leigh, M. Aljarah, M. Lisauskaite, O. Grytsai, R. Burgin, D. Niculescu-Duvaz, G. Thomson, C. Springer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d08007dfb89c4d2387e3dfb8d798320475546d" target='_blank'>
              RF-SIRF defining reversed DNA replication forks with single-cell and spatio-temporal resolution reveals a replication stress specific epigenetic code
              </a>
            </td>
          <td>
            Sunetra Roy, Morgan Fimreite, Yue Chen, Francesca Citron, Giulio Draetta, K. Schlacher
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Archaea, often thriving in extreme habitats, are believed to have evolved efficient DNA repair pathways to cope with constant insults to their genomes. However, how these organisms repair DNA double-strand breaks (DSBs), the most lethal DNA lesions, remains unclear. Here, we show that replicative primase consisting of the catalytic subunit PriS and the noncatalytic subunits PriL and PriX from the hyperthermophilic archaeon Saccharolobus islandicus is involved in DSB repair. We show that the overproduction or knockdown of PriL increases or decreases, respectively, the rate of survival and mutation frequency of S. islandicus cells following treatment with a DNA damaging agent. The increase in mutation is attributed primarily to an increase in small insertions or deletions. Further, overproduction of PriL enhances the repair of CRISPR-generated DSBs in vivo. These results are consistent with the extraordinary ability of PriSL to promote annealing between DNA strands sharing microhomology in addition to the activity of the heterodimer in terminal transfer and primer extension. The primase-mediated DSB repair is cell-cycle dependent since PriL is barely detectable during the S/G2 transition. Our data demonstrate that replicative primase is involved in DSB repair through microhomology-mediated end joining in Archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41c802b392e2aa09bc0c77d38347ce453e14aa3" target='_blank'>
              Archaeal replicative primase mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Daijiang Xiong, Zhimeng Li, Wen Qi, Shaoying Wang, Junkai Huang, Ningning Zhang, Zhenfeng Zhang, Li Huang
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination (HR) is a major pathway to repair DNA double-strand breaks. Hereditary breast and ovarian cancer syndrome (HBOC) is caused by germline pathogenic variants of HR-related genes, such as BRCA1 and BRCA2 (BRCA1/2). Cancer cells with HR deficiency are sensitive to poly(ADP-ribose) polymerase (PARP) inhibitors. Therefore, accurate evaluation of HR activity is helpful to diagnose HBOC and predict the effects of PARP inhibitors. The direct-repeat GFP (DR-GFP) assay has been utilized to evaluate cellular HR activity. However, evaluation by the DR-GFP assay tends to be qualitative and requires the establishment of stable cell lines. Therefore, we developed an assay to quantitatively measure HR activity called Assay for Site-Specific HR Activity (ASHRA), which can be performed by transiently transfecting two plasmids. In ASHRA, we use Cas9 endonuclease to create DNA double-strand breaks at specific sites in the genome, enabling the targeting of any endogenous loci. Quantification of HR products by real-time PCR using genomic DNA allows HR activity evaluated at the DNA level. Thus, ASHRA is an easy and quantitative method to evaluate HR activity at any genomic locus in various samples. Here, we present the protocols for adherent cells, suspension cells, and tumor tissues. Key features • This assay quantitatively evaluates homologous recombination (HR) activity. • This assay can measure HR activity in adherent cells, suspension cells, and tumor tissues. • This real-time PCR-based assay does not require a flow cytometer or next-generation sequencer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6041469126e683215dc13399e4746d11c09375" target='_blank'>
              Assay for Site-Specific Homologous Recombination Activity in Adherent Cells, Suspension Cells, and Tumor Tissues
              </a>
            </td>
          <td>
            Yuki Yoshino, Shin Kikuta, N. Chiba
          </td>
          <td>2025-04-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The development of hematopoietic cell lineages is a highly complex process governed by a delicate interplay of various transcription factors. The expression of these factors is influenced, in part, by epigenetic signatures that define each stage of cell differentiation. In particular, the formation of B lymphocytes depends on the sequential silencing of stemness genes and the balanced expression of interdependent transcription factors, along with DNA rearrangement. We have investigated the impact of Dido3 deficiency, a protein involved in chromatin status readout, on B cell differentiation within the hematopoietic compartment of mice. Our findings revealed significant impairments in the successive stages of B cell development. The absence of Dido3 resulted in remarkable alterations in the expression of essential transcription factors and differentiation markers, which are crucial for orchestrating the differentiation process. Additionally, the somatic recombination process, responsible for generation of antigen receptor diversity, was also adversely affected. These observations highlight the vital role of epigenetic regulation, particularly the involvement of Dido3, in ensuring proper B cell differentiation. This study reveals new mechanisms underlying disruptive alterations, deepening our understanding of hematopoiesis and may potentially lead to insights that aid in the development of therapeutic interventions for disorders involving aberrant B cell development. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01394-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba226143005da64aa14cb154db38e86341f77b0e" target='_blank'>
              The chromatin reader Dido3 is a regulator of the gene network that controls B cell differentiation
              </a>
            </td>
          <td>
            Fernando Gutiérrez del Burgo, María Ángeles García-López, Tirso Pons, Enrique Vázquez de Luis, Carlos Martínez-A, R. Villares
          </td>
          <td>2025-04-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6032320cee71742e5a132f4662a3a337579e09" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe.
              </a>
            </td>
          <td>
            Joshua M Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-05-03</td>
          <td>Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7d904ab0bb7bfa986bb2bea4c6f19edb348725" target='_blank'>
              Dysregulation of Locus-Specific Repetitive Elements in TCGA Pan-Cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Inaccurate chromosome segregation can lead to the formation of aneuploid cells that harbor an imbalanced complement of chromosomes. Several fungal species are not only able to tolerate the detrimental effects of aneuploidy but can use it to adapt to environmental pressures. The fungal pathobiont Candida albicans frequently acquires supernumerary chromosomes that enable growth in the presence of antifungal drugs or in specific host niches, yet the transcriptional changes associated with aneuploidy are not globally defined. Here, a karyotypically diverse set of C. albicans strains revealed that expression generally correlated with gene copy number regardless of the strain karyotype. Unexpectedly, aneuploid strains shared a characteristic transcriptional profile that was distinct from a generalized environmental stress response previously defined in aneuploid yeast cells. This aneuploid transcriptional response led to altered growth and oxidative balances relative to euploid control strains. The increased expression of reactive oxygen species (ROS) mitigating enzymes in aneuploid cells reduced the levels of ROS but caused an acute sensitivity to both internal and external sources of oxidative stress. Taken together, our work demonstrates common transcriptional and phenotypic features of aneuploid C. albicans cells with consequences for infection of different host niches and susceptibility to environmental stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e3ff6416e6e4c37bdd4902bf219f1aee84a69b" target='_blank'>
              Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans
              </a>
            </td>
          <td>
            Anna I Mackey, Robert J. Fillinger, P. S. Hendricks, Gregory J. Thomson, Christina A. Cuomo, Richard J. Bennett, Matthew Z. Anderson
          </td>
          <td>2025-04-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, Moritz J Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de660154998a356e9b85c8c4e7d59ed34b5f5dd0" target='_blank'>
              Abstract 4668: Prognostic significance and therapeutic potential of TOP2A expression in rare cancers
              </a>
            </td>
          <td>
            Jifa Li, Hua Ye, Xiaoqiong Bao, Wenwen Yu, Xiangxiang Chen, Yangyang Ni, Jincong Wang, Yubo Shi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Summary: The term “anaplasia” was coined in 1890 to describe chromosomal changes common to primordial cancer cells, but ever since the mechanisms whereby a cell becomes anaplastic has been the subject of much speculation. Recent results based on genomic and epigenomic profiles of cancer patient samples provide a glimpse into early events that lead to aneuploidy, the original defining feature of an anaplastic cell. We propose that the anaplastic cell is one in which RNA Polymerase II hypertranscribes S-phase-dependent histone genes, and the resulting histone excess facilitates DNA replication while competing for CENP-A, causing centromere breaks that initiate whole-arm aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e327392e26ad44de92c835846a2ea7101b5c4f9e" target='_blank'>
              Hansemann’s anaplastic theory of cancer after 135 years
              </a>
            </td>
          <td>
            S. Henikoff, K. Ahmad
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, C. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3da42eb5cba4f866717a7ca7075985f83d3c3fb" target='_blank'>
              Review Article
‘An Overview Genetics of Inherited Neurological and Non-Neurological Disorders in India’
              </a>
            </td>
          <td>
            Male D. N., Amle V. S., Ade R. P., Vyvahare S. S., Pawar V. K.
          </td>
          <td>2025-04-05</td>
          <td>International Journal of Advanced Research in Science, Communication and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4f3702444f966febda408aa09a277d286c200" target='_blank'>
              Abstract 1409: Reprogramming the alternative polyadenylation of an oncogenic transcription factor as a therapeutic strategy for prostate cancer
              </a>
            </td>
          <td>
            Giulia Pagani, Chiara Casirati, Stefania Fornezza, Roberta Cacioppo, P. Gandellini
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The D4Z4 locus is a macrosatellite array on chromosome 4q that normally comprises 8 to >100 3.3-kb repeat units. Its size and repetitiveness render it refractory to most sequencing technologies; consequently its genetic and epigenetic architectures remain incompletely understood despite their relevance to human health, in particular facioscapulohumeral muscular dystrophy (FSHD). Molecular diagnosis for FSHD following clinical description currently involves complex, multi-step and low resolution assays, aiming at identifying contractions on permissive haplotypes (FSHD type 1) or epigenetic reactivation (FSHD type 2) due to pathogenic variants in the epigenetic machinery (most often in SMCHD1).Here we leverage ultra-long whole-genome and Cas9-targeted sequencing to develop a fast and accurate workflow, D4Z4End2End, to comprehensively charactere the genetics and methylation of D4Z4 alleles. We apply it to samples from patients affected by FSHD1, FSHD2, and another disease caused by SMCHD1 variants, Bosma arhinia microphthalmia syndrome (BAMS), as well as publicly-available data from the 1000 Genomes Project and Human Pangenome Reference Consortium. We attain high read depth sequencing of full-length D4Z4 arrays of up to 40 repeat units (~132 kb), accurately capture contracted arrays, genetic mosaicism, and pathogenic SMCHD1 variants, and generate accurate consensus sequences of the full set of D4Z4 alleles for variant analysis. Moreover, we identify new allelic variants, analyse complex D4Z4 rearrangements including in-cis duplications, and reveal striking length- and SMCHD1 status-dependent methylation patterns across the D4Z4 array. Our findings provide new insights into human macrosatellite genetics and epigenetics, and demonstrate the potential of long-read nanopore sequencing to accelerate FSHD research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a67b65544abaa38aef2d4ff63b752b6fdbd016b" target='_blank'>
              D4Z4End2End: complete genetic and epigenetic architecture of D4Z4 macrosatellites in FSHD, BAMS and reference cohorts
              </a>
            </td>
          <td>
            Lucinda C. Xiao, Ayush Semwal, Brianna St, Kathleen Zeglinski, Shian Su, James Lancaster, Shifeng Xue, B. Reversade, Matthew E. Ritchie, Frédérique Magdinier, M. Blewitt, Quentin Gouil
          </td>
          <td>2025-04-26</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, Shotaro Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561cdb775ea226e1249d7f24f55971f03ff3eed2" target='_blank'>
              Functions and mechanisms of eukaryotic RNA-guided programmed DNA elimination.
              </a>
            </td>
          <td>
            Bozhidar-Adrian Stefanov, M. Nowacki
          </td>
          <td>2025-04-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e879216882ba777a52dac02a65cb65b0e331d4" target='_blank'>
              Abstract 5361: Clinical implications of The Cancer Genome Atlas molecular classification system in esophagogastric cancer
              </a>
            </td>
          <td>
            H. Walch, Raktim Borpatragohain, Justin Jee, W. Chatila, Christopher Fong, S. Maron, Geoffrey Y Ku, D. Ilson, Yelena Y Janjigian, Abraham J Wu, Pari Shah, Dan Coit, M. Bains, Valerie W Rusch, B. Park, M. Bott, Katherine Gray, David R. Jones, Michael F. Berger, N. Schultz, V. Strong, D. Molena, S. Sihag
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bde3f65de807bb9cec86aaa0dfc6c09be8467d" target='_blank'>
              Abstract 1942: Copy number aberrations in the ctDNA uncover clinical and biological heterogeneity in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Maare Arffman, L. Meriranta, J. Jørgensen, M. Karjalainen-Lindsberg, K. Beiske, Mette Pedersen, Kristina DrottØystein Fluge, S. Jyrkkiö, Peter Brown, Harald Holte, S. Leppä
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae14464f2ce77e42ffad40f3c6898b5621cece3e" target='_blank'>
              PLK4: Master Regulator of Centriole Duplication and Its Therapeutic Potential.
              </a>
            </td>
          <td>
            Muhammad Hamzah, Franz Meitinger, Midori Ohta
          </td>
          <td>2025-04-21</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="KRAS is a proto-oncogene that contains activating mutations in up to 30% of tumors. Many conventional therapies inhibit both cancerous and normal cells, which may cause toxicity. Thus, programmable mutant-selective targeted inhibitors are needed. Peptide nucleic acids (PNAs) incorporate base sequences analogous to DNA, with modified peptide backbones instead of ribose-phosphate backbones, allowing PNAs to hybridize with DNA with high avidity to suppress transcription. Here, we developed KRAS G12D-selective PNA oligomers with novel cell-penetrating flanking regions. Fluorescein-labeled PNA oligomers displayed high uptake rates in cells and nuclei. Exposure to PNA-delivery peptide conjugates resulted in repression of KRAS G12D mRNA and protein expression within 2 hours and lasting up to 48 hours. Varying cell-penetrating peptide (CPP) compositions and lengths of complementary KRAS sequences were tested using dose-response cell viability assays. These experiments identified configurations that were effective at selectively preventing growth of on-target KRAS G12D cells, while relatively sparing off-target KRAS G12C cells. Electrophoretic mobility shift assays demonstrated in vitro binding and selectivity for KRAS G12D DNA sequences. CPP-PNA-G12D-1 was effective against a panel of pancreatic ductal adenocarcinoma cell lines and patient-derived xenografts in vivo. These results show promise for an enhanced PNA-delivery peptide conjugate strategy as both a tool for studying tumors driven by oncogenic point mutations and as a potential therapeutic strategy to selectively target mutant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2065faaa8c8b01822bceb610976fd9fb9507b1" target='_blank'>
              Mechanism of DNA replication fork breakage and PARP1 hyperactivation during replication catastrophe.
              </a>
            </td>
          <td>
            Pedro Ortega, Elodie Bournique, Junyi Li, Ambrocio Sanchez, Gisselle Santiago, Brooke R. Harris, Josefine Striepen, J. Maciejowski, Abby M. Green, Rémi Buisson
          </td>
          <td>2025-04-18</td>
          <td>Science advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Toxoplasma gondii is an obligate intracellular parasite with a high replication rate that can lead to DNA replicative stress, in turn associated with the generation of DNA double-strand breaks (DSBs). Cells have two main pathways to repair DSBs: non-homologous end joining and homologous recombination repair (NHEJ and HRR respectively). RAD51 is the key recombinase in the HRR pathway. In this work, we achieved endogenous tagging of the RAD51 gene using the Auxin Inducible Degron (AID) system, to generate the clonal line RH RAD51HA-AID. Here we demonstrate that RAD51 is expressed in replicative tachyzoites and establishes damage foci. Auxin-induced knock-down (KD) affects the correct replication of tachyzoites which show loss of synchronization. The use of the RAD51 inhibitor B02 also affects parasite growth, with an IC50 of 4.8 µM. B02 produced alterations in tachyzoite replication and arrest in the S phase of the cell cycle. Additionally, B02 induced tachyzoite to bradyzoite differentiation showing small cyst-like structures. In conclusion, the HRR pathway is necessary for maintaining proper tachyzoite replication under normal growth conditions, supporting that replicative stress occurs during the cell cycle. Our findings also suggest that DNA replication stress can induce bradyzoite differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a99ccd548e29dfb7eccfd9c686305c90fbab532" target='_blank'>
              Toxoplasma gondii RAD51 recombinase is required to overcome DNA replication stress and its inactivation leads to bradyzoite differentiation
              </a>
            </td>
          <td>
            Ana M. Saldarriaga Cartagena, Ayelén Aparicio Arias, C. Cristaldi, Agustina Ganuza, M. M. Gonzalez, María M. Corvi, William J. Sullivan, Laura Vanagas, Sergio O. Angel
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8572ed5aac9e75313380c3e2cd2f1a7e899e36b7" target='_blank'>
              Abstract 1903: Actionable mutational landscape in solid tumors: a case study of ovarian and colon cancer
              </a>
            </td>
          <td>
            Amit Kumar, Samir Agarwal, Pankaj Khanna, Shobit Gupta, Rama Chandran, Ananth S. Bommakanti
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284e8736a9d8069a665b78b16285eb6fd004904b" target='_blank'>
              Escape from X-chromosome inactivation at KDM5C is driven by promoter-proximal DNA elements and enhanced by domain context.
              </a>
            </td>
          <td>
            Samantha B Peeters, S. Baldry, Andrea J. Korecki, Aditi Srinivasan, W. Wasserman, Elizabeth M Simpson, Carolyn J Brown
          </td>
          <td>2025-04-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7defb0dc0b2dcfb00df9b73a0a0c72cd7d43d4" target='_blank'>
              Transposable elements as instructors of the immune system.
              </a>
            </td>
          <td>
            Lisa Schmidleithner, Philipp Stüve, Markus Feuerer
          </td>
          <td>2025-04-29</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Germinal centres are specialized microenvironments where B cells undergo affinity maturation. B cells expressing antibodies whose affinity is improved by somatic hypermutation are selected for expansion by limiting numbers of T follicular helper cells. Cell division is accompanied by mutation of the immunoglobulin genes, at what is believed to be a fixed rate of around 1 × 10−3 per base pair per cell division1. As mutagenesis is random, the probability of acquiring deleterious mutations outweighs the probability of acquiring affinity-enhancing mutations. This effect might be heightened, and even become counterproductive, in B cells that express high-affinity antibodies and undergo the greatest number of cell divisions2. Here we experimentally examine a theoretical model that explains how affinity maturation could be optimized by varying the rate of somatic hypermutation such that cells that express higher-affinity antibodies divide more but mutate less per division. Data obtained from mice immunized with SARS-CoV-2 vaccines or a model antigen align with the theoretical model and show that cells producing high-affinity antibodies shorten the G0/G1 phases of the cell cycle and reduce their mutation rates. We propose that these mechanisms safeguard high-affinity B cell lineages and enhance the outcomes of antibody affinity maturation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8839c8d13cd09c345fd7a32753ffff5bd295051" target='_blank'>
              Regulated somatic hypermutation enhances antibody affinity maturation
              </a>
            </td>
          <td>
            Julia Merkenschlager, Andrew G T Pyo, Gabriela S Silva Santos, Dennis J. Schaefer-Babajew, M. Cipolla, H. Hartweger, Alexander D. Gitlin, N. Wingreen, Michel C. Nussenzweig
          </td>
          <td>2025-03-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21340650146a334354ebce820c4c26cb9ab3ba1b" target='_blank'>
              Abstract LB386: Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer
              </a>
            </td>
          <td>
            Aashirwad Shahi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b042bad9ffec7e55a0de94ab786a57c1a323f2d" target='_blank'>
              Abstract 4568: Biochemical mitigation of technical artifacts improves performance of a ctDNA methyl sequencing assay
              </a>
            </td>
          <td>
            James Yu, Peter George, J. Close, Andy Eang, N. Peterman, Yu Sun, Nafisa Bulsara, Eileen Dimalanta, Alex Robertson, Yexun Wang, Nicole Lambert
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA topoisomerase 1 (TOP1) is essential for transcription, replication, and repair. Its function relies on two catalytic steps, DNA breakage and rejoining. Inhibitors of the second step prevent DNA rejoining and lead to persistent DNA breaks, acting as topoisomerase poisons, used as anticancer drugs. However, reliable inhibitors of the first step are not available. Here, we provide genetic and molecular evidence supporting that Patulin and, to a lesser extent, Xestoquinol inhibit the first catalytic step of TOP1 in vitro, in yeast and in human cells. Particularly, Patulin prevents the accumulation of TOP1 cleavage complexes caused by the TOP1 poison camptothecin (CPT) in human cells. Moreover, Patulin pretreatment of human or yeast cells reduces DNA damage and the accumulation of DNA breaks upon CPT exposure. Consistent with the protective role of TOP1 against harmful R-loops, Patulin treatment increases R-loops and R-loop-associated cytotoxicity, mimicking the effect of TOP1 silencing. Altogether our findings indicate that Patulin and Xestoquinol are nonpoisoning inhibitors of TOP1, which should potentiate new research approaches in molecular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c91b4af793ee46a1c397d0e749fad65c6dce6ba" target='_blank'>
              Patulin and Xestoquinol are inhibitors of DNA topoisomerase 1.
              </a>
            </td>
          <td>
            E. Tumini, R. Wellinger, Emilia Herrera-Moyano, Patricia Navarro-Cansino, María L. García-Rubio, Daniel Salas-Lloret, Alejandro Losada, M. Muñoz-Alonso, H. Gaillard, Rosa Luna, A. Aguilera
          </td>
          <td>2025-04-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b4705110ed5255e541024e217bd1b38f97bdc5" target='_blank'>
              Abstract 2802: Circular RNA variants of theNUMB gene modulate the oncogenicity of oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Chung-Hsien Chou, Pin-Yu Chen, Shu-Chun Lin, Kuo-Wei Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary Ductal adenocarcinoma of the pancreas is a cancer with exceedingly dismal prognosis as less than 5% of all patients survive more than 5 years. Therefore, targeted molecular therapies are urgently needed. MTAP deficiency is currently one of the most promising molecular targets for tumor therapies. MTAP deficiency—often caused by homozygous 9p21 deletion—is easily detectable by immunohistochemistry because it is characterized by a complete MTAP expression loss. It makes tumor cells more susceptible to PRMT5- and MAT2A-inhibiting drugs as well as to pemetrexed (antifolate therapy). However, the success of targeted therapies critically depends on the extent of heterogeneity of the targeted tumor cell trait in clinical tumors. In this study, intratumoral heterogeneity of MTAP deficiency was thus assessed in more than 200 ductal adenocarcinomas of the pancreas in a tissue microarray format and whole-section analysis was carried out by immunohistochemistry and fluorescence in situ hybridization. MTAP deficiency was found in almost 40% of all cancers, was always caused by homozygous deletion, and was rarely heterogenous in TMA and large-section analyses (less than 10%). Pancreatic adenocarcinomas may thus represent an ideal cancer type for studying new drugs targeting MTAP-deficient cancer cells in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa497701c09c398746a0513a639855972ce4cb6a" target='_blank'>
              Deficiency of MTAP Is Frequent and Mostly Homogeneous in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            N. Gorbokon, Katharina Teljuk, V. Reiswich, M. Lennartz, S. Minner, R. Simon, G. Sauter, W. Wilczak, T. Clauditz, Nina Schraps, T. Hackert, Faik G. Uzunoglu, M. Kluth, L. Bubendorf, Matthias Matter, F. Viehweger, Morton Freytag, Frank Jacobsen, Katharina Möller, S. Steurer
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86f33c29e42254bdc63ed12ff517dd771563902" target='_blank'>
              Abstract 3319: Evaluating and targeting DNA repair defects in pediatric high grade glioma
              </a>
            </td>
          <td>
            Evan Savage, Ke Xu, Miaomiao Li, Jason R Myers, E. Rampersaud, Rebekah DeVries, Kamran Mohammad Zahid, Krista Millican, Yawei Hui, Ti-Cheng Chang, Gang Wu, A. Shelat, Christopher L Tinkle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/879969a6652e4d8acc441456b7bcb05f2b8a551c" target='_blank'>
              Abstract 3345: Application of cGAS-immunopositive micronuclei as an indicator of tumor aggressiveness in liposarcoma
              </a>
            </td>
          <td>
            Kiyota Kato, Toru Motoi, Yu Naito, Masumi Ogawa, Chihiro Onda, Hideo Arai, Shin-ichiro Horiguchi, Toshihide Hirai, T. Hishima
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pharmacogenomics is revolutionizing precision medicine by enabling tailored therapeutic strategies based on an individual genetic and molecular profile. Circular RNAs (circRNAs), a distinct subclass of endogenous non-coding RNAs, have recently emerged as key regulators of drug resistance, tumor progression, and therapeutic responses. Their covalently closed circular structure provides exceptional stability and resistance to exonuclease degradation, positioning them as reliable biomarkers and novel therapeutic targets in cancer management. This review provides a comprehensive analysis of the interplay between circRNAs and pharmacogenomics, focusing on their role in modulating drug metabolism, therapeutic efficacy, and toxicity profiles. We examine how circRNA-mediated regulatory networks influence chemotherapy resistance, alter targeted therapy responses, and impact immunotherapy outcomes. Additionally, we discuss emerging experimental tools and bioinformatics techniques for studying circRNAs, including multi-omics integration, machine learning-driven biomarker discovery, and high-throughput sequencing technologies. Beyond their diagnostic potential, circRNAs are being actively explored as therapeutic agents and drug delivery vehicles. Recent advancements in circRNA-based vaccines, engineered CAR-T cells, and synthetic circRNA therapeutics highlight their transformative potential in oncology. Furthermore, we address the challenges of standardization, reproducibility, and clinical translation, emphasizing the need for rigorous biomarker validation and regulatory frameworks to facilitate their integration into clinical practice. By incorporating circRNA profiling into pharmacogenomic strategies, this review underscores a paradigm shift toward highly personalized cancer therapies. circRNAs hold immense potential to overcome drug resistance, enhance treatment efficacy, and optimize patient outcomes, marking a significant advancement in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c91f6707f728019195288d5d968764867ed64f" target='_blank'>
              Unveiling Pharmacogenomics Insights into Circular RNAs: Toward Precision Medicine in Cancer Therapy
              </a>
            </td>
          <td>
            Saud Alqahtani, T. Alqahtani, K. Venkatesan, Durgaramani Sivadasan, Rehab Ahmed, Hassabelrasoul Elfadil, Premalatha Paulsamy, Kalaiselvi Periannan
          </td>
          <td>2025-04-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The supernumerary B chromosome of maize has a drive mechanism to maintain itself in a population despite being dispensible. This involves nondisjunction of the B centromere at the second pollen mitosis that produces the two sperm followed by preferential fertilization of the egg by the B containing sperm during double fertilization. During an introgression of the supernumerary B chromosome into the inbred line B73, an unusually high frequency of trisomies for A chromosomes was observed. Due to parallels to the High Loss phenomenon in which three or more B chromosomes in a specific genetic background cause chromosomal breakage at heterochromatic knob sites during the second pollen mitosis as well as ploidy changes, this phenomenon was revisited. Examination of pollen of the High Loss line revealed a high frequency of single sperm in the presence of the B chromosomes, which was previously not realized. Crosses to tetraploid females confirmed that the single sperm were diploid and functional but also revealed the presence of diploids with their A chromosomes derived solely from the tetraploid parent indicating a "diploid induction". Collectively, the results reveal two backgrounds in which the B drive mechanism is not confined to this chromosome causing detrimental effects by adherence of heterochromatic knobs and apparently A centromeres at the mitosis preceding sperm development. In most genetic backgrounds this process is restricted to the B chromosome but in B73 and the High Loss line, there is spillover to the normal chromosomes in distinct ways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822f11adca9e71b6e0ff40e6fc00f85ee3a4109f" target='_blank'>
              Aneuploidy and ploidy variation conditioned by the B chromosome of maize.
              </a>
            </td>
          <td>
            Patrice S. Albert, Hua Yang, Zhi Gao, Cassidy DeVore, J. Birchler
          </td>
          <td>2025-04-25</td>
          <td>Heredity</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T. Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c16d5ebf6f1d5ea4dbccbf5a0178380f13bbf886" target='_blank'>
              Abstract 7429: Illuminating the dark genome in cancer using artificial intelligence
              </a>
            </td>
          <td>
            M. D. M. Álvarez-Torres, Xi Fu, Raul Rabadan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43e1e965f05b28c3994c4e8f0fb6a1f9448be497" target='_blank'>
              Abstract 2856: Pks+ E. coli trigger intestinal stem cell plasticity and early onset colorectal cancer
              </a>
            </td>
          <td>
            Stefanie Gerstberger, Melissa Lumish, S. Hartner, Farheen Shah, Seongmin Choi, Krystal Lum, Christopher Cowley, A. Luthra, Qingwen Jiang, Hyung Jun Woo, Ahmed Mahmoud, H. Walch, Asha Saxena, Tavis J. Reed, A. Cercek, R. Yaeger, Ileana Cristea, Andrew McPherson, F. Sánchez-Vega, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="We have recently demonstrated that genetic inactivation of EWSR1 : : FLI1 by CRISPR/Cas9, successfully blocks cell proliferation in a cell model of Ewing sarcoma. However, CRISPR/Cas9-mediated gene editing can exhibit off-target effects, and thus, precise regulation of Cas9 expression in target cells is essential to develop gene-editing strategies to inactivate EWSR1 : : FLI1 in Ewing sarcoma cells. In this study, we demonstrate that Cas9 can be specifically expressed in Ewing sarcoma cells when located downstream a promoter consisting of GGAA repeats and a consensus TATA box (GGAAprom). Under these conditions, Cas9 is selectively expressed in Ewing sarcoma cells that express EWSR1 : : FLI1 oncoproteins, but not in cells expressing wild-type FLI1. Consequently, Ewing sarcoma cells infected with GGAAprom>Cas9 and a specific gRNA designed to inactivate EWSR1 : : FLI1, showed successful EWSR1 : : FLI1 inactivation and the subsequent blockade of cell proliferation. Notably, GGAAprom>Cas9 can be efficiently delivered to Ewing sarcoma cells via adenoviral vectors both in vitro and in vivo, highlighting the potential of this approach for Ewing sarcoma treatment. Our results demonstrate that the CRISPR/Cas9 machinery is safe and specific for Ewing sarcoma cells when driven under a GGAAprom, paving the way for the development of cancer gene therapies based on the selective expression of genes with therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba0d95a65d0d8225a5257acbb9853ce632bbe70" target='_blank'>
              Targeted inactivation of EWSR1 : : FLI1 gene in Ewing sarcoma via CRISPR/Cas9 driven by an Ewing-specific GGAA promoter
              </a>
            </td>
          <td>
            Saint T Cervera, Selene Martínez, María Iranzo-Martínez, L. Notario, R. M. Melero-Fernández de Mera, Javier Alonso
          </td>
          <td>2025-03-15</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fe83552a51ec8685227ae6ed241734102c7207" target='_blank'>
              Differential interfacial tension between oncogenic and wild-type populations forms the mechanical basis of tissue-specific oncogenesis in epithelia
              </a>
            </td>
          <td>
            Amrapali Datta, Phanindra Dewan, Aswin Anto, Tanya Chhabra, Tanishq Tejaswi, Sindhu Muthukrishnan, Akshar Rao, Sumantra Sarkar, Medhavi Vishwakarma
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f68977168d050d5c5f7c3fa27b15c83e3b09b0a" target='_blank'>
              Abstract 1011: Transposable element patterns across genetic ancestries in prostate cancer patients
              </a>
            </td>
          <td>
            Nicholas Korvink, Lucas Ueta, Hannah Park, Vinay Kumar, Farah Rahmatpanah
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have been identified in various eukaryotic organisms and are known to play crucial roles in genomic plasticity. However, in crop plants, the role of eccDNAs in responses to environmental cues, particularly nutritional stresses, remains unexplored. Rice (Oryza sativa ssp. japonica), a vital crop for over half the world’s population and an excellent model plant for genomic studies, faces numerous environmental challenges during growth. Therefore, we conduct comprehensive studies investigating the distribution, sequence, and potential responses of rice eccDNAs to nutritional stresses. We describe the changes in the eccDNA landscape at various developmental stages of rice in optimal growth. We also identify eccDNAs overlapping with genes (ecGenes), transposable elements (ecTEs), and full-length repeat units (full-length ecRepeatUnits), whose prevalence responds to nitrogen (N) and phosphorus (P) deficiency. We analyze multiple-fragment eccDNAs and propose a potential TE-mediated homologous recombination mechanism as the origin of rice’s multiple-fragment eccDNAs. We provide evidence for the role of eccDNAs in the rice genome plasticity under nutritional stresses and underscore the significance of their abundance and specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4749bc544ae6bce267e8a661f0fd6d51c2fdcee2" target='_blank'>
              Adaptive dynamics of extrachromosomal circular DNA in rice under nutrient stress
              </a>
            </td>
          <td>
            Hanfang Ni, Lenin Yong-Villalobos, Mian Gu, D. López-Arredondo, Min Chen, Liyan Geng, Guohua Xu, L. Herrera-Estrella
          </td>
          <td>2025-05-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Recent research has shifted the focus from the genetic code of DNA to its structural variations, which significantly impact cancer, genetic diseases, and gene regulation. Structural changes, such as the transition from B-DNA to A-DNA, influence DNA stability and flexibility and are affected by factors like DNA methylation and sugar puckering. This study is the first to investigate the relationship between DNA conformational changes and lifespan in two rodent species. The analysis focused on long-lived Nannospalax xanthodon and shorter-lived Rattus rattus, utilizing infrared spectroscopy and principal component analysis (PCA) to examine liver DNA. Results indicated that transition from B-form to A- and Z-forms were more prevalent in N. xanthodon than in R. rattus. However, the dominant DNA conformations in both species are in B-form. Additionally, N-type sugar puckers (C3-endo conformation), associated with these DNA forms, were more prominent in N. xanthodon. In contrast, S-type sugar puckers (C2-endo conformation), characteristic of B-DNA, were found at lower levels in N. xanthodon. Furthermore, the variations in methylation-specific structural modifications of nucleobases were quantitatively assessed among these species. The study proposes a significant connection between the long lifespan of N. xanthodon, which live underground, and their unique DNA structure, offering insights into how different DNA forms, as well as the conformations of their backbone and sugar-base components, may affect longevity, highlighting potential research avenues regarding the biomolecular aspects of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e194ada8d97ff0f6ac7375a7dc3c14a580e955" target='_blank'>
              Structural features of DNA and their potential contribution to blind mole rat (Nannospalax xanthodon) longevity
              </a>
            </td>
          <td>
            Gamzenur Sönmez, Tuba Yağcı Gurbanov
          </td>
          <td>2025-03-25</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA repair is a hierarchically organized, spatially and temporally regulated process involving numerous repair factors that respond to various types of damage. Despite decades of research, the mechanisms by which these factors are recruited to and depart from repair sites have been a subject of intrigue. Recent advancements in the field have increasingly highlighted the role of phase separation as a critical facilitator of the efficiency of DNA repair. This review emphasizes how phase separation enhances the concentration and coordination of repair factors at damage sites, optimizing repair efficiency. Understanding how dysregulation of phase separation can impair DNA repair and alter nuclear organization, potentially leading to diseases such as cancer and neurodegenerative disorders, is crucial. This manuscript provides a comprehensive understanding of the pivotal role of phase separation in DNA repair, sheds light on the current research, and suggests potential future directions for research and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c72f1d4a9e08095f98c1ef277b37623b6195f2e8" target='_blank'>
              Phase separation in DNA repair: orchestrating the cellular response to genomic stability
              </a>
            </td>
          <td>
            Juxin Deng, Zhaoyang Du, Lei Li, Min Zhu, Hongchang Zhao
          </td>
          <td>2025-05-02</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28e364fda09dbd60182752672b8d265061c0b8b" target='_blank'>
              Re-annotating the EPICv2 manifest with genes, intragenic features, and regulatory elements
              </a>
            </td>
          <td>
            B. Mallabar-Rimmer, Philippa M Wells, Alice Franklin, J. Mill, Amy P Webster
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617223433e545a7cf3bf6bdc859c98bd5269831a" target='_blank'>
              InferPloidy: A fast ploidy inference tool accurately classifies cells with abnormal CNVs in large single-cell RNA-seq datasets
              </a>
            </td>
          <td>
            Wonjung Sung, Jaeyoung Chae, Jucheol Moon, Seokhyun Yoon
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a30e97ab73c3d9f10f09249e3cecf21c25bbdf3" target='_blank'>
              Establishment of SHON Gene Knockout Breast Cancer Cell Line using CRISPR-Cas9 Technology
              </a>
            </td>
          <td>
            Alyssa Clemas
          </td>
          <td>2025-03-26</td>
          <td>Rangahau Aranga: AUT Graduate Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Hepatocytes display notable ploidy diversity, varying both in the number of genomic copies and in the number of nuclei. Most mouse hepatocytes are tetraploid (4n), which can exist as either mononucleated (1×4n) or binucleated (2×2n) cells. Despite this distinction, these two cell types have traditionally been grouped and studied as a single population. One likely reason for this is that conventional ploidy-sorting methods classify hepatocytes based solely on total DNA content, without distinguishing between mononucleated and binucleated states. In this study, we developed a novel flow cytometry strategy to distinguish and isolate 1×4n and 2×2n hepatocytes. Our approach leverages Hoechst-Area to assess total ploidy and Hoechst-Height to differentiate mononucleated and binucleated hepatocytes. Transcriptome analysis comparing these two populations revealed that 1×4n hepatocytes exhibit a broader and more metabolically active gene expression profile. Importantly, this enhanced metabolic activity was independent of liver zonation, a well-known driver of metabolic heterogeneity in hepatocytes. Our findings uncover a previously underappreciated layer of functional diversity in the liver and provide a new framework for studying the physiological and pathological roles of nuclear configuration in hepatocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e794617423650ecf97e3450334b4c20dba16a6" target='_blank'>
              A novel sorting method reveals superior metabolic activity of mononucleated over binucleated tetraploid hepatocytes
              </a>
            </td>
          <td>
            Yusuke Watanabe, Jonathan H. Sussman, Tomoya Anami, Moena Hattori, Saaya Yamane, Masaki Nishikawa, Yasuyuki Sakai, Takeshi Katsuda
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2213c097bc0bf3842b91050d879626cc38ef3b6" target='_blank'>
              Abstract 1488: DNA polymerase kappa slows replication fork speed by promoting fork reversal in glioblastoma
              </a>
            </td>
          <td>
            Reham S. Sewilam, Megan R. Reed, R. Eoff
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d38aba034857c4c0fa7d93c806340bd3f91d6a" target='_blank'>
              Cytosine methylation by ScmA contributes to the fitness of Caulobacter crescentus cells naturally expressing a Vsr-like protein
              </a>
            </td>
          <td>
            Noémie Matthey, Giorgia Wennubst, Karolina Bojkowska, Julien Marquis, Justine Collier
          </td>
          <td>2025-03-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029a236a2721717e13973bdc5e5b49afdd737065" target='_blank'>
              Abstract 2624: Aneuploidy induction mechanism in Ewing sarcoma
              </a>
            </td>
          <td>
            Priyani H. Hapugaswatta, M. Azuma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/304d6e4c38ec9dd6d17c47e93a7fbe83d1abb83b" target='_blank'>
              OGT prevents DNA demethylation and suppresses the expression of transposable elements in heterochromatin by restraining TET activity genome-wide.
              </a>
            </td>
          <td>
            Hugo Sepulveda, Xiang Li, Leo J. Arteaga-Vazquez, Isaac F. López-Moyado, Melina Brunelli, Lot M Hernández-Espinosa, Xiaojing Yue, J. C. Angel, Caitlin Brown, Zhen Dong, Natasha Jansz, Fabio Puddu, Aurélie Modat, Jamie Scotcher, Páidí Creed, Patrick H Kennedy, Cindy Manriquez-Rodriguez, Samuel A Myers, Robert Crawford, Geoffrey J Faulkner, Anjana Rao
          </td>
          <td>2025-03-28</td>
          <td>Nature structural & molecular biology</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT The intricacies of tumour microenvironment, particularly the extracellular matrix (ECM), underscore its pivotal function in modulating tumour progression and drug resistance. Among the key regulators of ECM remodelling and homeostasis, the lysyl oxidases (LOXs) emerge as promising therapeutic targets of tumour treatment. Despite their significance, a holistic evaluation of the LOX family's genomics and clinical implications across diverse cancer types remains elusive. Herein, this study aimed to investigate the correlation between LOX family expression and patient outcomes, drug responsiveness and tumour microenvironment (TME) characteristics in a cohort of 33 tumours based on The Cancer Genome Atlas (TCGA) database. Notably, patients exhibiting elevated LOX family expression suffer from worse prognosis and resistance to a spectrum of antitumor therapies, encompassing chemotherapy, endocrine therapy, targeted therapy and immunotherapy, in contrast to counterparts with subdued LOX family expression levels. Furthermore, enrichment analysis indicated that the LOX family fosters tumour progression and drug resistance. These findings were further validated by multiplex immunofluorescence staining in breast, gastric and rectal cancer, as well as breast cancer organoids. Altogether, this study unravels the intricate association between the LOX family and tumour progression, alongside multidrug resistance. We have gained further insights into the roles of LOX family genes in various tumour types, offering a novel avenue for future research into the relationship between LOX family genes and tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de26c0c5ab2c00eb5a4adc28b824cfdafa9df26" target='_blank'>
              Systematic Characterisation and Analysis of Lysyl Oxidase Family Members as Drivers of Tumour Progression and Multiple Drug Resistance
              </a>
            </td>
          <td>
            Hongjin Liu, Xiaojiao Sun, Bingqi Dong, Jixin Zhang, Junling Zhang, Yanlun Gu, Lin Chen, Xiaocong Pang, Jingming Ye, Xin Wang, Zhuona Rong
          </td>
          <td>2025-04-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ffb5861f3091086b8e6d39be2fc47951f44015b" target='_blank'>
              Cohesins: Crossroad Between Cornelia de Lange Spectrum and Cancer Predisposition.
              </a>
            </td>
          <td>
            Laura Rigotti, Stefano Rebellato, Antonella Lettieri, Silvia Castiglioni, M. Mariani, Simona Totaro, C. Saitta, C. Gervasini, G. Fazio, Valentina Massa, G. Cazzaniga, Angelo Selicorni
          </td>
          <td>2025-04-05</td>
          <td>American journal of medical genetics. Part A</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f55512a8c15d7d2448765df1044f7457324d44a" target='_blank'>
              Abstract 2488: Employing RNA base editors to engineer personalized tumor-specific neoantigens
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Netanel Landesman, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ac2ba6fe27ed44bff66e7ee62b13701829278a" target='_blank'>
              Abstract 6437: Mutational patterns in normal tissue hold the key to understanding the cellular origins of cancer
              </a>
            </td>
          <td>
            Kirsten Kübler, R. Karlić, Mendy Miller, C. Stewart, William D. Foulkes, P. Polak, Peter F. Arndt, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc85444c3b3642b4f78c90f641ae8bb9081ac7d" target='_blank'>
              Abstract 4916: Co-occurring clonal hematopoiesis in UK Biobank participants with prior cancer
              </a>
            </td>
          <td>
            Kara M. Barnao, Aubrey K Hubbard, I. Chan, Weiyin Zhou, Corey D Young, Rebecca L. Kelly, Derek W. Brown, G. Genovese, Duc Tran, Yin Cao, S. Chanock, Kelly L Bolton, M. Machiela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ef3faa0e6c95091fda1daef8c22d574a2979f" target='_blank'>
              Training your genome
              </a>
            </td>
          <td>
            Alan Herbert
          </td>
          <td>2025-04-08</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5fda087600bb54eebbaa6751c7c0db16b631c2" target='_blank'>
              Abstract 4670: The expression of HERV-K subfamilies indicated improved prognosis in small cell lung cancer
              </a>
            </td>
          <td>
            Xingyuan Wu, Jingwei Zhang, Yan Ju, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Saccharomyces cerevisiae chromosomal architectural protein Hmo1 is categorized as an HMGB protein, as it contains two HMGB motifs that bind DNA in a structure-specific manner. However, Hmo1 has a basic C-terminal domain (CTD) that promotes DNA bending instead of an acidic one found in a canonical HMGB protein. Hmo1 has diverse functions in genome maintenance and gene regulation. It is implicated in DNA damage tolerance (DDT) that enables DNA replication to bypass lesions on the template. Hmo1 is believed to direct DNA lesions to the error-free template switching (TS) pathway of DDT and to aid in the formation of the key TS intermediate sister chromatid junction (SCJ), but the underlying mechanisms have yet to be resolved. In this work, we used genetic and molecular biology approaches to further investigate the role of Hmo1 in DDT. We found extensive functional interactions of Hmo1 with components of the genome integrity network in cellular response to the genotoxin methyl methanesulfonate (MMS), implicating Hmo1 in the execution or regulation of homology-directed DNA repair, replication-coupled chromatin assembly, and the DNA damage checkpoint. Notably, our data pointed to a role for Hmo1 in directing SCJ to the nuclease-mediated resolution pathway instead of the helicase/topoisomerase mediated dissolution pathway for processing/removal. They also suggested that Hmo1 modulates both the recycling of parental histones and the deposition of newly synthesized histones on nascent DNA at the replication fork to ensure proper chromatin formation. We found evidence that Hmo1 counteracts the function of histone H2A variant H2A.Z (Htz1 in yeast) in DDT possibly due to their opposing effects on DNA resection. We showed that Hmo1 promotes DNA negative supercoiling as a proxy of chromatin structure and MMS-induced DNA damage checkpoint signaling, which is independent of the CTD of Hmo1. Moreover, we obtained evidence indicating that whether the CTD of Hmo1 contributes to its function in DDT is dependent on the host’s genetic background. Taken together, our findings demonstrated that Hmo1 can contribute to, or regulate, multiple processes of DDT via different mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f18b39d5105aea6884a60aabbdfb120450e322" target='_blank'>
              The Yeast HMGB Protein Hmo1 Is a Multifaceted Regulator of DNA Damage Tolerance
              </a>
            </td>
          <td>
            Jinlong Huo, Anhui Wei, Na Guo, Ruotong Wang, Xin Bi
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Methylation is a major DNA modification contributing to the epigenetic regulation of nuclear gene expression and genome stability. DNA methyltransferases (DNMT) inhibitors are widely used in epigenetic and cancer research, but their biological effects and the mechanisms of their action are not well recognized in plants. This research focuses on comparing the effects of two DNMT inhibitors, namely 5-azacytidine (AZA) and zebularine (ZEB), on cellular processes, including organogenesis in vitro. Protoplasts are a unique single-cell system to analyze biological processes in plants; therefore in our study, both inhibitors were applied to protoplast culture medium or the medium used for the regeneration of protoplast-derived calluses. Results AZA induced a dose-dependent reduction in protoplast viability, delayed cell wall reconstruction, and reduced mitotic activity, while ZEB in low concentration (2.5 µM) promoted mitoses and stimulated protoplast-derived callus development. The higher effectiveness of shoot regeneration was observed when drugs were applied directly to protoplasts compared to protoplast-derived callus treatments. Our findings reveal that both drugs affected the genome stability of the obtained regenerants by inducing polyploidization. Both drugs induced hypomethylation and modulated the distribution patterns of methylated DNA in the protoplast-derived callus. Conclusion AZA was more toxic to plant protoplasts compared to ZEB. Both inhibitors affect the ploidy status of protoplast-derived regenerants. A comparison of the data on global methylation levels with the regeneration efficiency suggests that organogenesis in cabbage is partially controlled by variations in DNA methylation levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c4c507e348c1843b2f175ecdbdb90917744a3c1" target='_blank'>
              Demethylating drugs alter protoplast development, regeneration, and the genome stability of protoplast-derived regenerants of cabbage
              </a>
            </td>
          <td>
            A. Kiełkowska, Agnieszka Brąszewska
          </td>
          <td>2025-04-11</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675c67dd0133028e8c6f69ff63a5bc578aace569" target='_blank'>
              TOPO-seq reveals DNA topology-induced off-target activity by Cas9 and base editors.
              </a>
            </td>
          <td>
            Min Duan, Pan Gao, Yi-Zhou Zhang, Yu-Long Hu, Lei Zhou, Zhong-Chen Xu, Houyuan Qiu, Xiao-Han Tong, Rui-Jin Ji, Xinlin Lei, Hao Yin, Cun-Lan Guo, Ying Zhang
          </td>
          <td>2025-04-02</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81014681c316f4c01121749b9d1b735ce752673e" target='_blank'>
              Abstract 6947: CRISPR/Cas9 kinome screen identifies PNKP as a novel fitness gene in HPV-negative head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Gurleen Kaur Tung, Kate Chatfield-Reed, Masaru Miyagi, W. O'Neill, Quintin Pan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e9293691e68c8ed457caf9800f0b8e56bb89a9" target='_blank'>
              Abstract 3895: Clonal expansion in normal endometrium and its association with endometriosis-associated ovarian cancer
              </a>
            </td>
          <td>
            Koichi Watanabe, N. Kakiuchi, Kosuke Ieiri, Maeda Hirona, Tomonori Hirano, Mana Taki, Koji Yamanoi, R. Murakami, Ken Yamaguchi, J. Hamanishi, Masaki Mandai, Seishi Ogawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT Contagious cancers represent one of the least understood types of infections in wildlife. Devil Facial Tumor Disease (comprised of two different contagious cancers, DFT1 and DFT2) has led to an 80% decline in the Tasmanian devil ( Sarcophilus harrisii ) population at the regional level since it was first observed in 1996. There are currently no treatment options for the disease, and research efforts are focused on vaccine development. Although DFT1 is clonal, phylogenomic studies have identified different genetic variants of the pathogen. We postulated that different genetic strains may have different gene expression profiles and would therefore require different vaccine components. Here, we aimed to test this hypothesis by applying two types of unsupervised clustering (hierarchical and k‐means) to 35 DFT1 transcriptomes selected from the disease's four major phylogenetic clades. The two algorithms produced conflicting results, and there was low support for either method individually. Validation metrics, such as the Gap statistic method, the Elbow method, and the Silhouette method, were ambiguous, contradictory, or indicated that our dataset only consisted of a single cluster. Collectively, our results show that the different phylogenetic clades of DFT1 all have similar gene expression profiles. Previous studies have suggested that transcriptomic differences exist between tumours from different locations. However, our study differs in that it considers both tumor purity and genotypic clade when analysing differences between DFTD biopsies. These results have important implications for therapeutic development, as they indicate that a single vaccine or treatment approach has the potential to be effective for a large cross‐section of DFT1 tumors. As one of the largest studies to use transcriptomics to investigate phenotypic variation within a single contagious cancer, it also provides novel insight into this unique group of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cbe855a88fcb5a55c5c65ed83b5ede5e0203c9d" target='_blank'>
              No Evidence for Distinct Transcriptomic Subgroups of Devil Facial Tumor Disease (DFTD)
              </a>
            </td>
          <td>
            Cleopatra Petrohilos, E. Peel, Kimberley C Batley, S. Fox, C. Hogg, K. Belov
          </td>
          <td>2025-04-01</td>
          <td>Evolutionary Applications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a5d646d97b0662ff1d0b09f324907510c69f3b" target='_blank'>
              The RAD52 double-ring remodels replication forks restricting fork reversal.
              </a>
            </td>
          <td>
            Masayoshi Honda, M. Razzaghi, Paras Gaur, Eva Malacaria, Giorgia Marozzi, Ludovica Di Biagi, F. Aiello, Emeleeta A. Paintsil, Andrew J. Stanfield, Bailey J. Deppe, L. Gakhar, N. Schnicker, M. Spies, P. Pichierri, M. Spies
          </td>
          <td>2025-04-02</td>
          <td>Nature</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd630ef4621d438baadd64541aa5f505561cc528" target='_blank'>
              Gemini Molecular Assembly Colocalization (GOAL): Accurate and Efficient Fusion Genotyping for Chronic Myeloid Leukemia Intelligent Diagnosis.
              </a>
            </td>
          <td>
            Yuanyuan Huang, Zixin Zhang, Tiantian Yang, Yangli Zhang, Xiaoxue Cheng, Yuexi Kang, Yujie Guang, Yuting Zou, Xiaoying Zhang, Zewei Luo, Junman Chen, W. Cheng
          </td>
          <td>2025-04-14</td>
          <td>Small methods</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950a3c88211fd8f62863e7c5d3ab9f041ca13d73" target='_blank'>
              Abstract SY07-01: Gain-of-function small molecules to reprogram oncogenic transcription factors
              </a>
            </td>
          <td>
            S. Gourisankar
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da5c17d8f344b2651d2d03913e601f8e4a43a5a5" target='_blank'>
              Abstract 500: A molecular comparison reveals distinct transcriptomic differences between uterine carcinosarcoma and papillary serous carcinoma distinguishable by DNA damage
              </a>
            </td>
          <td>
            Kevin J. Lee, Tanvi Joshi, Sagar Chokshi, Mackenzie Cummings, Jennifer Scalici, Nathaniel Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d32b1ca7eec503a083b45ef28636d90a045aa" target='_blank'>
              Fork-barrier-independent roles of topoisomerase I in the ribosomal DNA.
              </a>
            </td>
          <td>
            Temistocles Molinar, D. Sultanov, Hannah Klein, Andreas Hochwagen
          </td>
          <td>2025-03-22</td>
          <td>Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The dissemination of transferable plasmids, a major type of mobile genetic elements (MGEs), is one main driver of antibiotic resistance outbreaks. While the plasmid persistence condition in well-mixed environments has been extensively studied, most microbiota in nature are spatially heterogeneous. However, our knowledge regarding how spatial landscape shapes plasmid maintenance and dissemination remains limited. Here we establish a theoretical framework describing plasmid spread over a metacommunity of multiple patches. By analyzing the gene flow dynamics on randomly generated landscapes, we show that plasmid survival and dispersal are dictated by a simple feature of the landscape, spatial entropy. Reducing entropy speeds up plasmid range expansion and allows the global maintenance of many plasmids that are predicted to be lost by classic theories. The entropy's effects are experimentally validated in E. coli metacommunities transferring a conjugative plasmid. We further examine a vast collection of prokaryotic genomes and show that prokaryotes from low-entropy environments indeed carry more abundant MGEs and antibiotic resistance genes. Our work provides critical insights into the management and control of antimicrobial resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522f2ce1689b867c692982cdb1f1bea2971d931d" target='_blank'>
              Spatial entropy drives the maintenance and dissemination of transferable plasmids.
              </a>
            </td>
          <td>
            Wenzhi Xue, Juken Hong, Runmeng Zhao, Huaxiong Yao, Yi Zhang, Zhuojun Dai, Teng Wang
          </td>
          <td>2025-04-29</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/023882aa8c2054e1031402cd73d0b72d7f3038c5" target='_blank'>
              Abstract 6263: Multi-transcriptomic analysis reveals tumor microenvironment differences associated with anti-EGFR therapy responsiveness between left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            Atsuki Taniguchi, S. Kagawa, Shohei Nogi, Nobuhiko Kanaya, Yoshihiko Kakiuchi, Satoru Kikuchi, Shinji Kuroda, H. Tazawa, Toshiyoshi Fujiwara
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Pre-malignant transformation of pancreatic acinar cells by oncogenic Kras is dependent upon stochastic emergence of metaplastic cell states, through unknown mechanisms. We reveal that an early, transcriptionally-mediated effect of Kras is sporadic failure of proteostatic ER-phagy. Genetically-altered mice deficient in ER-phagy demonstrate that this event co-operates with Kras to drive acinar-ductal metaplasia (ADM) and subsequent cancer. Mechanistically, proteomics and high-resolution imaging uncover pathologic aggregation of a subset of ER proteins, including the injury marker REG3B, resulting from failure to physically interact with the ER-phagy receptor CCPG1. Spatial transcriptomics demonstrate that the appearance of sporadic intracellular aggregates upon Kras activation marks rare acinar cells existing in an injured, ADM-primed state. Importantly, engineered mutants of REG3B establish that aggregate formation is sufficient to directly engender this epithelial cell state. Pancreatic cancer can thus arise from stochastic pathologic protein aggregates that are influenced by and co-operate with an oncogene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16e9fa357e12d1adfb63608544864c4d2c4b3e55" target='_blank'>
              Dysproteostasis primes pancreatic epithelial state changes in KRAS-mediated oncogenesis
              </a>
            </td>
          <td>
            Carla Salomo Coll, Marisa di Monaco, Jocelyn Holkham, Matt Smith, Morwenna T. Muir, Philippe Gautier, Xiaozhong Zheng, Roopesh Krishnankutty, Alain J. Kemp, Katie Winnington-Ingram, A. von Kriegsheim, Jennifer P. Morton, Natalia Jimenez-Moreno, Damian Mole, Simon Wilkinson
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff6876ea2ec52d332b9f4431701ca0782cf05c0a" target='_blank'>
              Abstract 5294: Prevalence of KDM6A deficiency in human cancer: a tissue microarray study on 14,814 cancers from 153 different tumor types
              </a>
            </td>
          <td>
            F. Viehweger, Paul Wirth, N. Gorbokon, M. Lennartz, V. Reiswich, T. Krech, C. Hube-Magg, E. Bady, Lisa Hornsteiner, G. Sauter, R. Simon, S. Steurer, M. Kluth, A. Marx, A. Hinsch, A. Luebke, A. Menz, Clara Marie von Bargen, Frank Jacobsen, Piero Caneve, Ria Schlichter, S. Rico, S. Kind, S. Weidemann, M. Tsourlakis, W. Wilczak, T. Clauditz, C. Bernreuther, S. Minner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Genetic variants can affect protein function by driving aberrant subcellular localization. However, comprehensive analysis of how mutations promote tumor progression by influencing nuclear localization is currently lacking. Here, we systematically characterize potential shuttling-attacking mutations (SAMs) across cancers through developing the deep learning model pSAM for the ab initio decoding of the sequence determinants of nucleocytoplasmic shuttling. Leveraging cancer mutations across 11 cancer types, we find that SAMs enrich functional genetic variations and critical genes in cancer. We experimentally validate a dozen SAMs, among which R14M in PTEN, P255L in CHFR, etc. are identified to disrupt the nuclear localization signals through interfering their interactions with importins. Further studies confirm that the nucleocytoplasmic shuttling altered by SAMs in PTEN and CHFR rewire the downstream signaling and eliminate their function of tumor suppression. Thus, this study will help to understand the molecular traits of nucleocytoplasmic shuttling and their dysfunctions mediated by genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47b5ee70b22223e164afab1c522ee489fff66db" target='_blank'>
              Deep learning prioritizes cancer mutations that alter protein nucleocytoplasmic shuttling to drive tumorigenesis
              </a>
            </td>
          <td>
            Yongqiang Zheng, Kai Yu, Jin-Fei Lin, Zhuoran Liang, Qingfeng Zhang, Junteng Li, Qi-Nian Wu, Cai-Yun He, Mei Lin, Qi Zhao, Zhi-Xiang Zuo, Huai-qiang Ju, Rui-Hua Xu, Ze-xian Liu
          </td>
          <td>2025-03-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Constitutional epimutations are epigenetic aberrations that arise in normal cells prenatally. Two major forms exist: secondary constitutional epimutations (SCEs), associated with cis-acting genetic aberrations, and primary constitutional epimutations (PCEs), for which no associated genetic aberrations were identified. Some SCEs have been associated with risk of cancer (MLH1 and MSH2 with colon or endometrial cancers, BRCA1 with familial breast and ovarian cancers), although such epimutations are rare, with a total of <100 cases reported. This contrasts recent findings for PCE, where low-level mosaic BRCA1 epimutations are recorded in 5%-10% of healthy females across all age groups, including newborns. BRCA1 PCEs predict an elevated risk of high-grade serous ovarian cancer and triple-negative breast cancer (TNBC) and are estimated to account for about 20% of all TNBCs. A similarly high population frequency is observed for mosaic constitutional epimutations in MGMT, occurring as PCE or SCE, but not in MLH1. Contrasting BRCA1 and MLH1, a potential association with cancer risk for MGMT epimutations is yet unclear. In this review, we provide a summary of findings linking constitutional epimutations to cancer risk with emphasis on PCE. We also highlight challenges in detection of PCE exemplified by low-level mosaic epimutations in BRCA1 and indicate the need for further studies, hypothesizing that improved knowledge about PCE may add significantly to our understanding of cancer risk, carcinogenesis, and potentially development of other diseases as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f575b14e21266bf913814992a38a5ee8cd2dfa1" target='_blank'>
              Constitutional Epimutations: From Rare Events Toward Major Cancer Risk Factors?
              </a>
            </td>
          <td>
            P. Lønning, O. Nikolaienko, S. Knappskog
          </td>
          <td>2025-04-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305b8e7bc2c7c8696a5122f2e68b6a176a4582a5" target='_blank'>
              Model-based inference of cell cycle dynamics captures alterations of the DNA replication programme
              </a>
            </td>
          <td>
            Adolfo Alsina, M. Fumasoni, Pablo Sartori
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476371ed64c35ea1249a75d4d3d698e19d9d61e9" target='_blank'>
              Hyperosmotic Stress of HL-60/S4 Cells: Gene Set Enrichment Analysis and Microscopic Changes
              </a>
            </td>
          <td>
            Ada L. Olins, D. M. Mark Welch, Igor Prudovsky, D. Olins
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560018e7c0fcf02fe82f350604281c6789453dd9" target='_blank'>
              Tumor-Selective Gene Therapy: Using Hairpin DNA Oligonucleotides to Trigger Cleavage of Target RNA by Endogenous flap endonuclease 1 (FEN 1) Highly Expressed in Tumor Cells.
              </a>
            </td>
          <td>
            Chunlu Wang, Chen Wang, Chenxin Xiao, Weijie Zhang, Yan Guo, Muqing Qu, Qinxin Song, Xiaole Qi, Bingjie Zou
          </td>
          <td>2025-03-28</td>
          <td>Small</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c5132c005dfce0cf64cc574af8d4383b034ce39" target='_blank'>
              Abstract 2325: Anatomical genomic and transcriptomic analyses of Helicobacter pylori-driven gastric transformation
              </a>
            </td>
          <td>
            Yosuke Tanaka, Fumishi Kishigami, Toshihide Ueno, S. Kojima, T. Hashimoto, Y. Yamagata, T. Yoshikawa, Shigeki Sekine, Hiroyuki Mano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Hexameric helicases are nucleotide-driven molecular machines that unwind DNA to initiate replication across all domains of life. Despite decades of intensive study, several critical aspects of their function remain unresolved1: the site and mechanism of DNA strand separation, the mechanics of unwinding propagation, and the dynamic relationship between nucleotide hydrolysis and DNA movement. Here, using cryo-electron microscopy (cryo-EM), we show that the simian virus 40 large tumour antigen (LTag) helicase assembles in the form of head-to-head hexamers at replication origins, melting DNA at two symmetrically positioned sites to establish bidirectional replication forks. Through continuous heterogeneity analysis2, we characterize the conformational landscape of LTag on forked DNA under catalytic conditions, demonstrating coordinated motions that drive DNA translocation and unwinding. We show that the helicase pulls the tracking strand through DNA-binding loops lining the central channel, while directing the non-tracking strand out of the rear, in a cyclic process. ATP hydrolysis functions as an ‘entropy switch’, removing blocks to translocation rather than directly powering DNA movement. Our structures show the allosteric couplings between nucleotide turnover and subunit motions that enable DNA unwinding while maintaining dedicated exit paths for the separated strands. These findings provide a comprehensive model for replication fork establishment and progression that extends from viral to eukaryotic systems. More broadly, they introduce fundamental principles of the mechanism by which ATP-dependent enzymes achieve efficient mechanical work through entropy-driven allostery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a629a0de8b0881af3cce2ed03cf65bc35d84c28" target='_blank'>
              Structural dynamics of DNA unwinding by a replicative helicase
              </a>
            </td>
          <td>
            Taha Shahid, A. U. Danazumi, M. Tehseen, Lubna Alhudhali, Alice R Clark, Christos G. Savva, Samir M. Hamdan, A. De Biasio
          </td>
          <td>2025-03-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Background DNA cytosine deaminase APOBEC3B (A3B) is one of the endogenous sources of somatic mutations in many types of human cancers and is associated with tumor progression rather than tumorigenesis. However, it remains uncertain whether APOBEC3B‐induced mutations accelerate tumor progression or not. In this paper, we established a mouse model with A3B overexpression and investigated whether the introduction of A3B overexpression accelerates tumor development in Tp53 hemizygous mice. Methods We established A3B transgenic mouse by microinjection and selected the mouse which has only one A3B transgene by genomic qPCR and southern blotting using the probe against the transgene. A3B expression was validated by qPCR, immunoblotting, immunohistochemistry and in vitro CDA assays using lysates of this transgenic mouse liver, spleen and bone marrow. We interbreed this transgenic mouse model with CAG‐Cre and Tp53 knockout mice and observed differences in tumor progression and survival between Tp53 hemizygous mice and Tp53 homozygous mice irrespective of A3B expression. Finally, comprehensive genomic mutation analysis was done using the developed tumors. Results We established A3B transgenic mouse which has only one transgene. A3B expression and its CDA activity were confirmed in liver cells and tumor tissues of mice overexpressing A3B. Tp53 hemizygous mice developed osteosarcomas, spindle and pleomorphic sarcomas, and squamous cell carcinomas, however we did not observe any difference in tumor development between the mice with or without A3B expression. The tumor with A3B expression has more high‐VAF mutations than the one without A3B, but these mutations are not APOBEC signature. Conclusion We developed a Cre inducible A3B transgenic mouse model bearing single copy of A3B gene. Although the introduction of A3B overexpression did not accelerate tumor development in Tp53 hemizygous mice, our mouse model with A3B overexpression is well‐validated and useful for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b2e1c29be3a5ab7793714efd1b22c6b4cc4521" target='_blank'>
              APOBEC3B Does Not Promote Tumor Progression in Tp53 Hemizygous Mice
              </a>
            </td>
          <td>
            Yoshihito Horisawa, Tadahiko Matsumoto, June Takeda, Yusuke Tashiro, Ryosuke Nomura, Suguru Takeuchi, Yugo Kawai, Yasuhiro Kazuma, Y. Konishi, H. Yamazaki, Hiroyuki Matsui, Kotaro Shirakawa, A. Takaori-Kondo
          </td>
          <td>2025-04-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The discovery of N6-methyladenine (6mA) in eukaryotic genomes, typically found in prokaryotic DNA, has revolutionized epigenetics. Here, we show that symmetric 6mA is essential in the early diverging fungus Rhizopus microsporus, as the absence of the MT-A70 complex (MTA1c) responsible for this modification results in a lethal phenotype. 6mA is present in 70% of the genes, correlating with the presence of H3K4me3 and H2A.Z in open euchromatic regions. This modification is found predominantly in nucleosome linker regions, influencing the nucleosome positioning around the transcription start sites of highly expressed genes. Controlled downregulation of MTA1c reduces symmetric 6mA sites affecting nucleosome positioning and histone modifications, leading to altered gene expression, which is likely the cause of the severe phenotypic changes observed. Our study highlights the indispensable role of the DNA 6mA in a multicellular organism and delineates the mechanisms through which this epigenetic mark regulates gene expression in a eukaryotic genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05202d5ed7a63bfb1ef94a4cd810b66ec80dda5e" target='_blank'>
              Symmetric adenine methylation is an essential DNA modification in the early-diverging fungus Rhizopus microsporus
              </a>
            </td>
          <td>
            Carlos Lax, S. Mondo, J. F. Martínez, A. Muszewska, Leo A. Baumgart, José A Pérez-Ruiz, Pablo Carrillo-Marín, K. Labutti, A. Lipzen, Yu Zhang, Jie Guo, Vivian Ng, Eusebio Navarro, T. Pawlowska, Igor V. Grigoriev, F. Nicolás, V. Garre
          </td>
          <td>2025-04-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4bf206646c787c811c3e18fa137c014843f6fee" target='_blank'>
              Emergence of coordinated cell division during the evolution of multicellularity
              </a>
            </td>
          <td>
            Luis F. Cedeño-Pérez, R. Pineau, Thomas C. Day, W. Ratcliff, Peter L. Conlin
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d74135fcdadea0bd26562069b9d6960930f7d34" target='_blank'>
              Molecular Profiling of Nasopharyngeal Carcinoma Using the AACR Project GENIE Repository
              </a>
            </td>
          <td>
            Beau Hsia, Asritha Sure, Roshan Dongre, Nicolas Jo, Julia Kuzniar, G. Bitar, Saif A. Alshaka, Jeeho D. Kim, Bastien A. Valencia-Sanchez, Michael G. Brandel, Mariko Sato, J. Crawford, Michael L. Levy, Sean P. Polster, Vijay A. Patel
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e12e4fc2e2ae45882770e50fef3981ce0d4239" target='_blank'>
              Abstract 2993: Loss of p53 sensitizes cancer cells to oxidative phosphorylation agonism
              </a>
            </td>
          <td>
            Jianhuang Lin, Dimitra Georgopoulou, Shuzhen Wu, Svaty Sachania, Rebecca Saunders, Biju Mangatt, Geeta Sharma, Euan Stronach, Anthony Mazurek, Fiona M. Behan, Benjamin Schwartz, Brendan A. Veeneman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d329440f52845fe5e04ed56db29d5167fcf05d" target='_blank'>
              Abstract 6199: Interlab study supporting hereditary cancer testing in an NGS-focused landscape
              </a>
            </td>
          <td>
            Dana J. Ruminski Lowe, Deepika Philkana, K. Nahlik, Melissa Berenger, Andrew Anfora, Yves Konigshofer, Catherine Huang, R. Garlick, B. Anekella
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641c07283b7fb2cdea66c7bdedd4e3eb2d81d73a" target='_blank'>
              Abstract 948: PAX3-FOXO1-driven oncolytic herpes virus designed for treatment of alveolar rhabdomyosarcoma
              </a>
            </td>
          <td>
            Cole W. Peters, Theodore Nowicki, Moe Kawakami, Connor Kidd, Analynn Lechein
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="During Bacillus subtilis spore germination/outgrowth, the rehydration of the spore core and activation of aerobic metabolism can generate reactive oxygen species (ROS)-promoted DNA lesions that are repaired via the base excision repair pathway (BER). Accordingly, spores deficient in the AP-endonucleases (APEs) Nfo and ExoA exhibit a delayed outgrowth that is suppressed following disruption of the checkpoint protein DisA. Here, we report that DisA-independent DNA damage checkpoints operate during B. subtilis spore outgrowth. Consistent with this notion, spores lacking Nfo, ExoA, and Nth, which functions as an APE, did not suppress delayed outgrowth following disA disruption. Furthermore, in reference to the ∆nfo ∆exoA ∆nth spores, spores deficient for these APEs and DisA displayed a significantly higher number of oxidative genetic lesions and failed to properly segregate its chromosome during the first round of replication in the outgrowth stage. Finally, we found that DisA promotes low-fidelity repair and replication events, as revealed by DNA-alkaline gel electrophoresis (AGE) as well as spontaneous and H2O2-promoted RifR mutagenesis. Overall, our results unveil the existence of DisA-independent DNA damage checkpoint(s) that are activated by genomic lesions of an oxidative nature during spore germination and outgrowth, ensuring a proper transition to vegetative growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5895166da94cd4b19c2f868bc8a81c945249be" target='_blank'>
              Germination and Outgrowth of Bacillus subtilis Spores Deficient in BER and DisA Unveil Alternative Genetic Checkpoints
              </a>
            </td>
          <td>
            Alejandra Rangel-Mendoza, Luz I. Valenzuela-García, E. Robleto, M. Pedraza-Reyes
          </td>
          <td>2025-04-01</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f986802170cb9c7742c1de57719ba8f1909e04" target='_blank'>
              Abstract 2767: Aneuploidy-induced VHL haploinsufficiency in lung squamous cell carcinoma and clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Joanna Kopko, Joan J. Castellano-Perez, Gayatri Bulsara, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa234ebfaf50e592837e263e4dd2aff2aed0c794" target='_blank'>
              Abstract 4368: Selective targeting of oncogenic KRAS G12D using complementary peptide nucleic acid oligomers
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, William Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac2dc7a55fea26667edfda1482feea25ad7c347" target='_blank'>
              Abstract 6438: Decoding the archetypes and ecotypes of triple-negative breast cancer in responses to chemotherapy
              </a>
            </td>
          <td>
            Yun Yan, Yiyun Lin, T. Kumar, S. Bai, A. Thennavan, Jianzhuo Li, Tuan M. Tran, Min Hu, Mitchell Rao, Anna K. Casasent, E. Ravenberg, G. Rauch, A. Clayborn, Debu Tripathy, Alastair M. Thompson, Bora Lim, Lei Huo, S. Moulder, C. Yam, Nicholas Navin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The human placenta, a unique tumor-like organ, is thought to exhibit rare aneuploidy associated with adverse pregnancy outcomes. Discrepancies in reported aneuploidy prevalence in placentas stem from limitations in modeling and detection methods. Here, we use isogenic trophoblast stem cells (TSCs) derived from both naïve and primed human pluripotent stem cells (hPSCs) to reveal the spontaneous occurrence of aneuploidy, suggesting chromosomal instability (CIN) as an inherent feature of the trophoblast lineage. We identify potential pathways contributing to the occurrence and tolerance of CIN, such as autophagy, which may support the survival of aneuploid cells. Despite extensive chromosomal abnormalities, TSCs maintain their proliferative and differentiation capacities. These findings are further validated in placentas, where we observe a high prevalence of heterogeneous aneuploidy across trophoblasts, particularly in invasive extravillous trophoblasts. Our study challenges the traditional view of aneuploidy in the placenta and provides insights into the implications of CIN in placental function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6159f217a2759af79e2341d14a4ba7c6f9a70e77" target='_blank'>
              Chromosomal instability in human trophoblast stem cells and placentas
              </a>
            </td>
          <td>
            Danyang Wang, Andrew Cearlock, Katherine Lane, Chongchong Xu, Ian Jan, Stephen McCartney, Ian Glass, R. McCoy, Min Yang
          </td>
          <td>2025-04-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1582331698256f81dcbd70eb9ab64e969d89900e" target='_blank'>
              Abstract 5664: Identifying novel vulnerabilities in clear cell sarcoma through functional and genomic screening
              </a>
            </td>
          <td>
            Laura Leuenberger, Larissa Isenegger, Yanjiang Chen, Miles Burri, Sarah Kollar, Lara Planas-Paz, Peter Karl Bode, A. Wozniak, Sebastian Bauer, Patrick Schöffski, Chantal Pauli, Christian Britschgi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea648b38c8ec7739bccf80938ba1f0a86b09a8b" target='_blank'>
              Extraction and Identification of Micronuclei from Human Peripheral Blood Lymphocytes.
              </a>
            </td>
          <td>
            Ping Wu, Zhengxi Tian, Lei Xie, Jiang Yao, Qijie Li
          </td>
          <td>2025-03-28</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell division and chromosome replication are tightly coordinated in bacteria. The dipM gene in Caulobacter crescentus is known to be involved in cell division, but its role in chromosome replication remains unclear; previously, a dipM mutant was found to be capable of replication but not of maintaining a plasmid. This study investigates dipM’s function in both processes and aims to identify interacting genes. We hypothesize that dipM regulates chromosome replication through interactions with cell cycle proteins. To test this, a mutagenesis test will be conducted to isolate Caulobacter mutants with replication defects, and we expect to identify mutations that disrupt chromosome replication. Fluorescence-labeled replication reporters will be used to detect cell cycle abnormalities, likely revealing mislocalized replication proteins in dipM mutants. Genetic complementation will be used to determine whether introducing wild-type dipM can restore normal replication and division. We will use a whole-genome library alongside DNA sequencing to identify affected genes, predicting new regulatory components within this pathway. Finally, bioinformatics tools such as BLAST will be used to analyze these genes, potentially uncovering conserved mechanisms of bacterial cell cycle regulation. This study is expected to provide insight into bacterial cell cycle regulation; however, potential genetic redundancy and the need for further validation may pose challenges. Ultimately, this study has the potential to inform research on microbial genetics and antimicrobial development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c9022fd0b76bc447ed7b84669e30019d491b49" target='_blank'>
              Analysis of the Regulatory Pathway Between DNA Replication and Cell Division
              </a>
            </td>
          <td>
            Zeineb Haouari, Adrian Neulander, Gregory Marczynski
          </td>
          <td>2025-03-25</td>
          <td>McGill Science Undergraduate Research Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) binding proteins (ssBPs) are essential in eukaryotes to protect telomeres from nuclease activity. In Saccharomyces cerevisiae, the ssBP Cdc13 is an essential protein that acts as a central regulator of telomere length homeostasis and chromosome end protection, both alone and as part of the Cdc13-Stn1-Ten1 (CST) complex. Cdc13 has high binding affinity for telomeric ssDNA, with a very slow off-rate. Previously, we reported that despite this tight ssDNA binding, Cdc13 rapidly exchanges between bound and unbound telomeric ssDNA substrates, even at sub-stoichiometric concentrations of competitor ssDNA. This dynamic DNA exchange (DDE) is dependent on the presence and length of telomeric repeat sequence ssDNA and requires both Cdc13 DNA binding domains, OB1 and OB3. Here we investigated if Cdc13 dimerization is important for DDE by characterizing the dimerization mutant Cdc13-L91R. Using mass photometry, we confirmed that Cdc13-L91R fails to dimerize in solution, even in the presence of ssDNA. Gel-based DDE assays revealed that Cdc13-L91R fails to undergo ssDNA exchange compared to recombinant wild-type protein. Biolayer interferometry demonstrated that this effect was not due to differences in ssDNA binding kinetics. Thus, dimerization of Cdc13 is essential for DDE, and we model how this may impact telomere biology in vivo. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865c75282cc4f48f3c748b1be60316b7f2486048" target='_blank'>
              Dimerization of Cdc13 is essential for dynamic DNA exchange on telomeric DNA
              </a>
            </td>
          <td>
            D. Nickens, Spencer J Gray, Robert H. Simmons, Matthew L. Bochman
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7c4f2a96fbb0d3027c8d467c20c2ff1f764f3e" target='_blank'>
              Abstract 3541: A novel chimeric RNA as a source of neoantigens in CRC
              </a>
            </td>
          <td>
            Sarah Lynch, Amanda M. Lulu, Samir Lalani, Justin Elfman, C. Slingluff, Hui Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7582dcd1a7c773e7f2f2ada600a1280dbd165c3" target='_blank'>
              A DNA-Centric Mechanism for Protein Targeting in 6mA Methylation
              </a>
            </td>
          <td>
            Li Yang, Dongbo Wang
          </td>
          <td>2025-03-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aff6e4c1130ad77a048fffd67082d1b6b987410" target='_blank'>
              Abstract 1480: Unlocking DNA repair mechanisms with dynamic single-molecule analysis
              </a>
            </td>
          <td>
            Paul Pease, Jack O'Sullivan, Trey Simpson, Josephine Yeh, A. Candelli
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="A prime goal of regenerative medicine is to replace dysfunctional cells in the body. To design protocols for producing target cells in the laboratory, one may need to consider exponentially large combinations of culture components. Here, we investigated the potential of iteratively approximating the target phenotype by quantifying the distance between chromatin profiles (ATAC-seq) of differentiating cells in vitro and their in-vivo counterparts. We tested this approach on the well-studied generation of erythroblasts from haematopoietic stem cells, evaluating a fixed number of components over two sequential differentiation rounds (8×8 protocols). We found that the most erythroblast-like cells upon the first round yielded the most erythroblast-like cells at the second round, suggesting that greedy selection by chromatin approximation can be a viable optimisation strategy. Furthermore, by analysing regulatory sequences in incompletely reprogrammed chromatin regions, we uncovered transcriptional regulators linked to roadblocks in differentiation and made a data-driven selection of ligands that further improved erythropoiesis. In future, our methodology can help craft notoriously difficult cells in vitro, such as B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7b441570e9d78db0e22f24a21be3538d6a31df" target='_blank'>
              Directing stem cell differentiation by chromatin state approximation
              </a>
            </td>
          <td>
            L. F. Montaño-Gutierrez, Sophie Müller, A. Kutschat, Davide Seruggia, F. Halbritter
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c39ca98ea2816bab12e16bae857d1896574f49" target='_blank'>
              The multifaceted functions of SPC25 in cancer: from molecular pathways to targeted therapy
              </a>
            </td>
          <td>
            Yigang Jin, Minjie Chen, Fei Chen, Zhao-feng Gao, Xiaoping Li, Lingyu Hu, Dandan Cai, Siqi Zhao, Zhengwei Song
          </td>
          <td>2025-05-07</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c34b2c44db5e2413af90c06943f2d16b821f4c" target='_blank'>
              Abstract 6621: Multi-modal profiling reveals the role of somatic structural variations in renal cell carcinoma
              </a>
            </td>
          <td>
            Yuwei Zhang, Kapur B. Dhami, Yize Li, Cody Weimholt, Wagma Caravan, John M Herndon, R. Jayasinghe, Song Cao, Melissa A. Reimers, Russell Pachynski, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043b9ea73372c63fd71b5f71e5fcf3ae2c56736f" target='_blank'>
              Insights into gallbladder cancer pathogenesis from a living organoid gallbladder cholangiocyte biorepository
              </a>
            </td>
          <td>
            Ankita Dutta, Nandita Chowdhury, Akshaya Vijayan Selvarajan, Pritha Banerjee, Abhirupa Kar, Shinjini Chandra, Uma Sunderam, D. Ganguli, Trina Dutta, Dipjit Basak, Smrithi Jayashree Satheeshkumar, Anand Sagar Ragate, S. Krishnan, Saugata Sen, Manas Kumar Roy, Sudeep Banerjee, Rajgopal Srinivasan, P. Roy, Vaskar Saha, Anindita Dutta, Dwijit GuhaSarkar
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c0e4ffc625e622788f94fda65aab7a646ab736" target='_blank'>
              Abstract 6707: Direct in vivo CRISPR screen identifies BAP1 and FAT1 as potent tumor suppressors in sarcomagenesis
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Andrea R Daniel, Zhaoyu Sun, Nerissa T. Williams, Melissa J. Kasiewicz, Yaping Wu, Diana M. Cardona, B. Piening, John T. Welle, Wesley K. Rosales, Venkatesh Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, K. Oristian, Omar M. Lopez, Nicholas E. Sibinga, Rutulkumar Patel, Ziqiang Zhang, Alexander J. Lazar, Corinne M. Linardic, Brady Bernard, William L Redmond, Walter J. Urba, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ddf2b8f34051e75cce25b49af56dc93daa7e41a" target='_blank'>
              The Influence of Homologous Recombination Repair on Temozolomide Chemosensitivity in Gliomas.
              </a>
            </td>
          <td>
            Biyun Zeng, Hansen Shi, Tiancai Liu, Jinjing Tang, Juncheng Lin, Xiaocong Lin, Tao Zeng
          </td>
          <td>2025-03-22</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>