<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-29 09:30:30 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>81</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>362</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>316</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNA (eccDNA) has emerged as a critical driver of oncogenesis, yet its functional roles in high‐grade serous ovarian cancer (HGSOC) remain poorly characterized. This highlights the need for comprehensive investigations into the abundance, biogenesis, and functional implications of eccDNA in HGSOC. Methods To characterize eccDNA in HGSOC, we performed comprehensive Circle‐seq analysis to assess eccDNA abundance and genomic annotation in HGSOC tissues compared to normal ovarian tissue. For mechanistic validation of eccDNA biogenesis pathways, targeted knockdown experiments of microhomology‐mediated end‐joining (MMEJ) dependent on LIG3 and POLQ were conducted. Functional characterization of HGSOC‐specific eccDNA‐harboring precursor microRNAs (eccMIRs) included in vitro assays using HGSOC cells and in vivo tumor growth experiments. Results Circle‐seq analysis revealed a 13‐fold increase in eccDNA abundance in HGSOC compared to normal ovarian tissue, with significant enrichment in promoter and coding regions. The MMEJ pathway was identified as the predominant pathway for eccDNA biogenesis in HGSOC, supported by characteristic microhomologies at junction sites and validation via LIG3 and POLQ knockdown experiments. Notably, HGSOC‐specific eccDNA frequently contained functional eccMIRs (eccMIR3661, eccMIR618, and eccMIR2277), which generate oncogenic miRNAs. These miRNAs promote tumor progression by downregulating tumor suppressor genes and activating key oncogenic pathways. Functional assays confirmed that these eccMIRs significantly enhanced HGSOC cell proliferation, migration, and invasion in vitro and promoted tumor growth in vivo. Conclusions These results underscore eccDNA as an oncogenic driver in HGSOC through non‐coding RNA‐mediated regulatory mechanisms, revealing novel therapeutic opportunities for targeting eccDNA biogenesis in this aggressive malignancy. Key points This study revealed a 13‐fold increase of eccDNA in HGSOC compared to normal tissues, with significant enrichment in promoter and coding regions. eccDNA‐derived miRNAs (eccMIRs) were shown to enhance cancer cell proliferation, invasion, and tumor growth through the expression of oncogenic miRNA sequences. The study highlights the importance of the MMEJ pathway in eccDNA generation and proposes that targeting eccDNA biogenesis in this aggressive malignancy presents a novel therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21093a2da7863b736808acd2b5cbd885658bf0" target='_blank'>
              Extrachromosomal circular DNA expressing miRNA promotes ovarian cancer progression
              </a>
            </td>
          <td>
            Ning Wu, Ling Wei, Qiyu Liu, T. He, Cuiyu Huang, Yunpeng Jiang, Kailong Li, Hongyan Guo, Fengbiao Mao, Xiaolu Zhao
          </td>
          <td>2025-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Extrachromosomal DNA amplification is associated with poor cancer prognoses1. Large numbers of excised signal circles (ESCs) are produced as by-products of antigen receptor rearrangement during V(D)J recombination2,3. However, current dogma states that ESCs are progressively lost through cell division4. Here we show that ESCs replicate and persist through many cell generations and share many properties in common with circular extrachromosomal DNAs. Increased ESC copy numbers at diagnosis of B cell precursor acute lymphoblastic leukaemia were highly correlated with subsequent relapse. By taking advantage of the matching recombination footprint that is formed upon the generation of each ESC, we measured ESC persistence and replication and found increased ESC replication in patients who later relapsed. This increased replication is controlled by cell-intrinsic factors and corresponds to increased expression of DNA replication- and repair-associated genes. Consistent with high ESC levels having a role in disease progression, the number of mutations typical of those caused by the V(D)J recombinase–ESC complex was significantly increased at diagnosis in patients who later relapsed. The number of such mutations in genes associated with relapse increased between diagnosis and relapse, and corresponded to clonal expansion of cells with high ESC copy numbers. These data demonstrate that the by-product of V(D)J recombination, when increased in abundance, potently associates with the V(D)J recombinase to cause adverse disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>2</td>
          <td>18</td>
        </tr>

        <tr id="Oncogenes amplified on extrachromosomal DNA (ecDNA) contribute to treatment resistance and poor survival across cancers. Currently, the spatiotemporal evolution of ecDNA remains poorly understood. In this study, we integrate computational modeling with samples from 94 treatment-naive human glioblastomas (GBM) to investigate the spatiotemporal evolution of ecDNA. We observe oncogene-specific patterns of ecDNA spatial heterogeneity, emerging from random ecDNA segregation and differing fitness advantages. Unlike PDGFRA-ecDNAs, EGFR-ecDNAs often accumulate prior to clonal expansions, conferring strong fitness advantages and reaching high abundances. In corroboration, we observe pretumor ecDNA accumulation in vivo in genetically engineered mouse neural stem cells. Variant and wild-type EGFR-ecDNAs often coexist in GBM. Those variant EGFR-ecDNAs, most commonly EGFRvIII-ecDNA, always derive from preexisting wild-type EGFR-ecDNAs, occur early, and reach high abundance. Our results suggest that the ecDNA oncogenic makeup determines unique evolutionary trajectories. New concepts such as ecDNA clonality and heteroplasmy require a refined evolutionary interpretation of genomic data in a large subset of GBMs.


SIGNIFICANCE
We study spatial patterns of ecDNA-amplified oncogenes and their evolutionary properties in human GBM, revealing an ecDNA landscape and ecDNA oncogene-specific evolutionary histories. ecDNA accumulation can precede clonal expansion, facilitating the emergence of EGFR oncogenic variants, reframing our interpretation of genomic data in a large subset of GBMs. See related article by Korsah et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Chapus Fleur, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Recent evidence highlights the role of extrachromosomal circular DNAs (eccDNAs) in cancers. However, reports regarding its role in hepatocellular carcinoma (HCC) are infrequent. The abundance of eccDNAs from five HCC/adjacent tissue pairs was explored using Circle-Sequencing. eccDNA PAFAH1B3 was selected as one of the objects. The effect of eccDNA PAFAH1B3 on HCC progression was determined using EdU, Transwell, and apoptosis assays. Additionally, the expressions of eccDNA PAFAH1B3, mRNA PAFAH1B3, and epithelial–mesenchymal transition (EMT)-related markers were determined using RT-PCR and WB. A xenograft tumor model was established to explore the function of PAFAH1B3 in vivo, and EMT-related markers were detected using RT-PCR and IHC analyses. The abundance of eccDNA PAFAH1B3 was significantly increased in HCC cell lines after transfection with eccDNA PAFAH1B3, and promoted the proliferation, migration, and invasion of liver cells while inhibiting apoptosis. The levels of mRNA PAFAH1B3 were also upregulated. Furthermore, intratumoral injection of PAFAH1B3 inhibitor suppressed tumor growth, and PAFAH1B3 knockdown increased and decreased the levels of the E-cadherin and N-cadherin, respectively. Our study findings reveal that eccDNA PAFAH1B3 may promote the occurrence and development of HCC by enhancing the expression of PAFAH1B3 and regulating EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173a3d83ec58d1fbf7f0f7dcd22ef9f85da2bb94" target='_blank'>
              PAFAH1B3 Exists in Linear Chromosomal and Extrachromosomal Circular DNA and Promotes HCC Progression via EMT
              </a>
            </td>
          <td>
            Dandan Li, Huishan Sun, Yingjie Wang, Yicong Yin, Ying Zhu, Xia Qian, Shanshan Wang, Longhao Zhang, Haitao Zhao, Ling Qiu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Acral melanoma (AM) is an aggressive melanoma subtype with limited therapeutic options and poor outcomes. In non-European descent and admixed populations, like those residing in Latin America, AM accounts for a significant proportion of cutaneous melanoma cases. Here, we performed comprehensive genomic and functional profiling of AM from a uniquely diverse Brazilian cohort. Whole-exome and transcriptome sequencing revealed low mutation burden and predominance of copy number alterations, including high-amplitude focal amplifications termed hailstorms. These hailstorms frequently affected chromosomes 11, 5 and 22 and key oncogenes such as CCND1, GAB2, CDK4, and TERT. The presence of hailstorms in the long arms of chromosomes 11 and 22 was associated with higher focal copy number burden and loss of DNA damage response genes (ATM, CHEK1), suggesting a permissive genomic environment driving structural instability. To explore the unique genomic context of AM, we established a comprehensive collection of patient-derived xenograft (AM-PDX) models that faithfully retain the histopathological and genomic features of the original tumours. Functional exploration of AM-specific vulnerabilities through pharmacological and CRISPR/Cas9 knockout screenings identified strong sensitivity to targeting MAPK, CDK4/6, MDM2, and WEE1 pathways. Notably, the pan-RAS(ON) inhibitor RMC-7977 effectively reduced viability in NRAS-, KRAS-, and KIT-mutant AM cell lines. Finally, CRISPR screens revealed AM-enriched dependencies, including CRKL and SF3B4, highlighting previously unrecognized vulnerabilities. Our findings emphasize the distinct biology of AM compared to other subtypes of melanoma, provide a valuable resource of models reflective of Latin American ancestry, and identify potential drivers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0407eee77f76a83d31f10dc4bc4ede91f52ba5a" target='_blank'>
              Modelling Acral Melanoma in Admixed Brazilians Uncovers Genomic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Annie Cristhine Moraes Sousa-Squiavinato, Sara Santos Bernardes, Flavia C. Aguiar, Antonio C. Facciolo, Martín del Castillo Velasco Herrera, J. R. C. Wong-Ramirez, P. Basurto-Lozada, Aretha Brito Nobre, Geethanjali Annamalai, Rebecca Martins Cadimo do Nascimento, Jacqueline M Boccacino, Rafaela Fagundes, Pedro Sodré do R. Barros, Mariana de Moraes Pitombo, R. Olvera-León, Larissa Satiko Alcantara Sekimoto Matsuyama, J. Billington, I. Vermes, Irving Simonin-Wilmer, D. G. Carvalho, João Pedro Cavalcante Simoes, Priscila Valverde Fernandes, Luiz Fernando Nunes, Andreia Cristina de Melo, Jadivan Leite de Oliveira, M. Herlyn, Andrew E. Aplin, C. D. Robles-Espinoza, David J. Adams, P. Possik
          </td>
          <td>2025-08-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d343e029068c438cb11fe609f9a8f46f24c108" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Gliomas: Insights from TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter H. Rheinstein
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma.
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, Gregory Schuyler Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b7fddece1de57c4f545b96bbc40fa5c54a7ad7" target='_blank'>
              Enhancer regulation in cancer: from epigenetics to m6A RNA modification.
              </a>
            </td>
          <td>
            Arathy Vasukutty, Poshan Yugal Bhattarai, Hong Seok Choi
          </td>
          <td>2025-08-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e777b5adbe2904319c350a6fa32537ac2efa7f" target='_blank'>
              Expansion of satellite DNAs derived from transposable elements in beetles with reduced diploid numbers.
              </a>
            </td>
          <td>
            José M. Rico-Porras, P. Mora, Ana E. Gasparotto, V. B. Bardella, T. Palomeque, Pedro Lorite, D. C. Cabral-de-Mello
          </td>
          <td>2025-08-24</td>
          <td>Heredity</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Genome instability in induced pluripotent stem cells (IPSC) poses a significant challenge for their use in research and medicine. Cataloging and precisely describing all the identified aberrations that arise during cell reprogramming, expansion, and differentiation is essential for improving approaches to instability prevention and ensuring genetic quality control. We report the karyotypic analysis of 65 cell lines derived from skin fibroblasts, urinal sediment, and peripheral blood mononuclear cells of 33 individuals, 82% of whom suffer from monogenic genetic disorders not associated with genetic instability. Trisomy of chromosomes 20 and 8 was revealed recurrently, while the 1q arm was the most frequently affected region involved in interstitial duplications and unbalanced translocations with chromosomes 15 and 18. The localization of rearrangement breakpoints identified by SNP arrays within the large DCC gene and histone gene clusters links genetic instability in IPSCs to replication-stress-induced chromosome breakage at common and early replicating fragile sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4802c78d378e6001adc2d0cfa2a7308e860155" target='_blank'>
              Chromosomal Aberrations in Induced Pluripotent Stem Cells: Identification of Breakpoints in the Large DCC Gene and HIST2 Histone Gene Cluster
              </a>
            </td>
          <td>
            D. Zheglo, V. Pozhitnova, A. V. Kislova, Zh.G. Markova, Danila Kiselev, P.S. Sviridov, Valeria Sviridova, Lyajsan I. Gumerova, S. Smirnikhina, Almaqdad Alsalloum, S. V. Pylina, S. Kutsev, Ekaterina Sergeevna Voronina
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/952548d2d250db0e8e23c4ea5a20ae1aba4ce302" target='_blank'>
              Coregulators determine androgen receptor activity in prostate cancer.
              </a>
            </td>
          <td>
            Kerim Yavuz, Nathan A. Lack
          </td>
          <td>2025-08-20</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance This study highlights a specific source of endogenous DNA replication stress, G-quadruplexes (G4s), that can increase genome instability in BRCA2-deficient cerebellar granule cell progenitors (GCPs), leading to medulloblastoma. BRCA2 is necessary for maintenance of replication speed at G4s in GCPs, and its absence allows these structures to become hotspots of increased mutagenesis. The G4-unwinding PIF1 helicase was found to be upregulated in tumor cells as a potential way to cope with G4-related replication stress. Targeting PIF1 may represent a therapeutic strategy for BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0cd9b13650ad33751efc7afb07cdd2a585f9d5" target='_blank'>
              G-quadruplexes as a source of vulnerability in BRCA2-deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle L. Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, C. Sandoval-Garcia, Margaret L MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole genome sequencing to study the coding and noncoding genome (promoters, enhancers and noncoding-NOS) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region are preserved despite treatment-induced genetic bottlenecks. Compared to other noncoding regions enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer including KLF5 and TP63. Transcriptional expression based on RNA-seq data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared to cancers without enhancer mutations suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Non-Coding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions.
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma is the most common extracranial solid tumors in children, and patients with high-risk disease have poor survival despite intensive multimodal therapies. MYCN amplification is a defining feature of high-risk neuroblastoma and drives a transcriptional program that maintains an undifferentiated and proliferative tumor state. Although MYCN is a well-established oncogenic driver, it remains a difficult therapeutic target. In this study, we investigated the role of NIPBL, a cohesin loading factor, in regulating MYCN-driven transcriptional programs. We found that elevated NIPBL expression is associated with poorly differentiated tumor phenotypes and worse clinical outcomes. Importantly, NIPBL depletion led to downregulation of MYCN expression and induced transcriptional reprogramming consistent with neuronal differentiation. These findings suggest that NIPBL sustains the oncogenic activity of MYCN and may serve as a tractable therapeutic vulnerability in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876bbc7066492a73f26bd266ecac38324c99091c" target='_blank'>
              Cohesin Loading Factor NIPBL Is Essential for MYCN Expression and MYCN-Driven Oncogenic Transcription in Neuroblastoma
              </a>
            </td>
          <td>
            Jeeyoun Kang, Kaitlyn Tremble, Philip Homan, Carol J Thiele
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Histone proteins and their variants have been found to play crucial and specialized roles in chromatin organization and the regulation of downstream gene expression; however, the relationship between histone sequence and its effect on chromatin organization remains poorly understood, limiting our functional understanding of sequence variation between distinct subtypes and across evolution and frustrating efforts to rationally design synthetic histones that can be used to engineer specified cell states. Here, we make the first advance towards engineered histone-driven chromatin organization. By expressing libraries of sequence variants of core histones in human cells, we identify variants that dominantly modulate chromatin structure. We further interrogate variants using a combination of imaging, proteomics, and genomics to reveal both cis and trans-acting mechanisms of effect. Functional screening with transcription factor libraries identifies transcriptional programs that are facilitated by engineered histone expression. Double mutation screens combined with protein language models allow us to learn sequence-to-function patterns and design synthetic histone proteins optimized to drive specific chromatin states. This work establishes a foundation for the high-throughput evaluation and engineering of chromatin-associated proteins and positions histones as tunable nodes for understanding and modulating mesoscale chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753e8190a3afe5b70fb5e9e449b056c25b4d9c43" target='_blank'>
              Engineered histones reshape chromatin in human cells
              </a>
            </td>
          <td>
            Siddhartha G. Jena, Surya Nagaraja, Andrew S. Earl, Amalia R. Driller-Colangelo, Michael A. Quezada, Ena Oreskovic, Max A. Horlbeck, Ruochi Zhang, Wilson Gomarga, Jason D. Buenrostro
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking. Methods We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq. Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq. Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors. Results Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC. Conclusions Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14585-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667c56c3222da6ac371627cb03ea1bf3e1f96087" target='_blank'>
              Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing
              </a>
            </td>
          <td>
            Wen Chen, Jiawei Xu, Chengdong Yu, Meng Zhou, Yong Ai, Wenbing Rao, Qingchuan Wang, Siyi Xu, Lei Tang, Si Gong, Jun Rao
          </td>
          <td>2025-08-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved – specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4849479727b73a833dd0ff467a2ebd5a79ba5e8" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="This review considers fundamental issues related to the genomics of birds (Aves), including the special organization and evolution of their chromosomes. In particular, we address the capabilities of molecular genetic/genomic approaches to clarify aspects of their evolutionary history, including how they have adapted to multiple habitats. We contemplate general genomic organization, including the small size and typical number of micro/macrochromosomes. We discuss recent genome sequencing efforts and how this relates to cytogenomic studies. We consider the emergence of this unique organization ~245 million years ago, examining examples where the “norm” is not followed. We address the functional role of synteny disruptions, centromere repositioning, repetitive elements, evolutionary breakpoints, synteny blocks and the role of the unique ZW sex chromosome system. By analyzing the cytogenetic maps and chromosomal rearrangements of eight species, the possibility of successfully applying modern genomic methods/technologies to identify general and specific features of genomic organization and an in-depth understanding of the fundamental patterns of the evolution of avian genomes are demonstrated. An interpretation of the observed genomic “variadicity” and specific chromosomal rearrangements is subsequently proposed. We also present a mathematical assessment of cross-species bacterial artificial chromosome (BAC) hybridization during genomic mapping in the white-throated sparrow, a species considered a key model of avian behavior. Building on model species (e.g., chicken), avian cytogenomics now encompasses hundreds of genomes across nearly all families, revealing remarkable genomic conservation with many dynamic aspects. Combining classical cytogenetics, high-throughput sequencing and emerging technologies provides increasingly detailed insights into the structure, function and evolutionary organization of these remarkable genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d522936fb056b50f35fb472ce8bf96b8ff9560" target='_blank'>
              Avian Cytogenomics: Small Chromosomes, Long Evolutionary History
              </a>
            </td>
          <td>
            Darren K. Griffin, Rafael Kretschmer, D. Larkin, Kornsorn Srikulnath, Worapong Singchat, Valery G Narushin, R. O’Connor, Michael N. Romanov
          </td>
          <td>2025-08-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs) maintain their genomes as minichromosomes in the nuclei of infected keratinocytes. This study investigates the association of HPV31 genomes with host chromatin using both HiC and 4C-seq chromosome conformation capture techniques. We show that HPV31 genomes preferentially associate with transcriptionally active A compartments of host chromatin, regions of open chromatin defined by ATAC-seq, and super-enhancers defined by Brd4 and H3K27ac ChIP-seq. The viral genome association sites were also highly correlated with genomic loci previously identified as common HPV integration sites in cervical cancers. Recent studies have shown that transcriptionally active sites are prone to dsDNA breaks, and we find a strong correlation among dsBREAK datasets with transcriptionally active and open regions of host chromatin and the HPV31 genome association sites defined in our study. These findings suggest that HPV genomes associate with cellular transcriptional epicenters to maintain active viral gene expression during persistent infection, but also indicate that the susceptibility of these regions to dsDNA breaks could explain their propensity for viral DNA integration in HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ecc91def6b800389ed078be4c219004519589f" target='_blank'>
              Human papillomavirus genomes associate with active host chromatin during persistent viral infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic polyploid macronucleus is lost. The formation of a new macronucleus involves programmed genome rearrangements, introducing DNA DSBs at ∼45 000 loci. Paramecium tetraurelia employs a non-homologous end joining (NHEJ) mechanism for the faithful repair of these DSBs. There are four genes encoding DNA polymerases of family X in the genome, one of which was found recently to colocalize with other NHEJ proteins in the nucleus. Here we have characterized all four enzymes and shown that they are generally very faithful. They fall into two functional classes that may specialize in the distinct repair contexts encountered during DSB DNA repair. Biochemical assays, site-directed mutagenesis, and X-ray structures of mutants of human Polλ incorporating sequence determinants from P. tetraurelia PolX or metazoan Polβ are used to investigate the origin of their fidelity. Our findings suggest that Paramecium PolX enzymes may represent evolutionary intermediates between metazoan Polβ and Polλ. A general classification of DNA PolXs based on clustering methods indicates that our results can be generalized to plant DNA PolXs (Polλ-like) involved in DSB DNA repair generated by CRISPR–Cas9 engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e54e53d0ac06ac21d27fa1e914141a18179e83e" target='_blank'>
              Fidelity, specialization, and evolution of Paramecium PolX DNA polymerases involved in programmed double-strand break DNA repair
              </a>
            </td>
          <td>
            Nourisson Antonin, Missoury Sophia, Lucas-Staat Soizick, Haouz Ahmed, Delarue Marc
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Genomic organization requires an intricate balance between the compact storage of genetic material and the ability to finely tune gene regulation. Chromatin looping achieves this balance by organizing concordantly regulated groups of genes and their regulatory elements into loops while also condensing DNA to fit into the small volume of a nucleus. A number of DNA-binding and associated proteins, including CTCF and cohesin, act as chromatin looping factors that mediate this process. Given the tight association between chromatin looping and gene expression, disordered genomic organization has been linked to disease development, including cancer. Recurrent mutations in chromatin looping factors are common in cancer, in particular blood cancers such as leukemia and myelodysplastic syndromes. In this review, we describe the evolution of our understanding of the chromatin looping process in healthy and malignant hematopoiesis and discuss the therapeutic potential of targeting chromatin looping factors in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05682afd3dc6513232a3e809f87b0eed8c538362" target='_blank'>
              Role of Chromatin Looping Factors in Leukemia.
              </a>
            </td>
          <td>
            Shira G Glushakow-Smith, Zuzana Tothova
          </td>
          <td>2025-09-15</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The murine endogenous retrovirus MERVL is dynamically activated in a small population of in vitro cultured mouse embryonic stem cells (mESCs) exhibiting totipotent-like features. Yet, the relationship between MERVL activation and cell fate decisions of mESCs is incompletely understood. Through a genome-wide knockout screen, we discovered that MERVL activity is intrinsically linked to DNA damage response pathways. Loss of Ints7, a backbone subunit of the Integrator complex, increased DNA damage and triggered MERVL expression. Mechanistically, Ints7 depletion induced phosphorylation of Kap1, increased chromatin accessibility at MERVL loci, and activated the p53–Dux axis to drive MERVL transcription. Intriguingly, DNA damage-induced MERVL resurgence followed the cleavage of caspase-3, often accompanying a process known as anastasis—cell survival after transient apoptotic signaling. Collectively, our study uncovered that MERVL activation in mESCs is integrated into the cellular circuit for decision-making in response to DNA damage, suggesting that sublethal caspase activation can influence the developmental potential of stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9647f46def2d9561823487f197db2f3a1327b93a" target='_blank'>
              Ints7 deficiency activates DNA damage response to elicit resurgence of endogenous retrovirus MERVL and anastasis of embryonic stem cells
              </a>
            </td>
          <td>
            Yunfan Shen, Li Li, Huiling Ni, Hui Li, Mingrui Xu, Xiaoyang Tan, Zhangjie Li, Pishun Li, Fang Chen, Song Mao, Gongping Sun, Kai Yuan
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling.
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, R. Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54a3ce373321b31d8cc83feaca65eb77c05c889" target='_blank'>
              Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition
              </a>
            </td>
          <td>
            F. Ringnalda, Gijs J. F. van Son, Laurens H. G. Verweij, Seok-Young Kim, Vicky Amo-Addae, U. E. Flucke, L. Hiemcke-Jiwa, Karin P S Langenberg, Jos A. M. Bramer, L. Heimans, M. A. van de Sande, W. V. van Houdt, Max M. van Noesel, H. Kerstens, Marcel Santoso, Georg Seifert, O. Delattre, Katia Scotlandi, B. Geoerger, J. H. Merks, J. Molenaar, R. van Boxtel, Marc van de Wetering, K. Sanders, H. Clevers
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Over 95% of ependymomas that arise in the cortex are driven by a gene fusion involving the zinc finger translocation-associated (ZFTA) protein. Here, using super-resolution and lattice light-sheet microscopy, we demonstrate that the most frequent fusion variant, ZFTA–RELA (ZR), forms dynamic nuclear condensates that are required for oncogene expression and tumorigenesis. Mutagenesis studies of ZR reveal a key intrinsically disordered region (IDR) in RELA that governs condensate formation. Condensate-modulating IDR mutations introduced into ZR impaired its genomic occupancy at oncogenic loci and inhibited the recruitment of transcriptional effector proteins, such as MED1, BRD4 and RNA polymerase II. Using nuclear magnetic resonance spectroscopy, we examined the DNA-binding residues of the critical zinc finger (ZF1) found in ZR and characterized their significance for condensate formation, genomic binding and oncogene activation. We generated synthetic ZFTA fusion proteins where IDRs from known condensate-forming proteins were grafted into ZR. Synthetic ZFTA fusion oncoproteins utilizing IDRs from EWS and FUS restored condensate formation, oncogene transcription and tumour initiation in mice. These findings provide key insights into the oncogenic mechanism of ZR and the importance of IDR acquisition in fusion oncoproteins in brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d5d4e3905b6133ceff9f0311d9fcea6d57257e" target='_blank'>
              Synthetic ZFTA fusions pinpoint disordered protein domain acquisition as a mechanism of brain tumorigenesis
              </a>
            </td>
          <td>
            A. Arabzade, H. K. Shirnekhi, S. Varadharajan, S. M. Ippagunta, A. Phillips, N. Laboe, D. W. Baggett, Wahiduzzaman, M. Jo, T. Zheng, R. Pathak, D. Gee, D. Bhimsaria, H. Wu, X. Gao, J. Liu, E. Emanus, A. Bland, A. Kardian, A. Hancock, B. Holcomb, T. Wright, T. Bugbee, H. Sun, M. Zhai, E. Caesar, M. Park, S. Tripathi, A. Shirinifard, K. Lowe, A. Khalighifar, R. A. Petersen, S. King, D. Stabley, A. Pitre, G. Campbell, C-G Park, W. T. Freyaldenhoven, B. Chandra, Y. Xia, E. Bonten, A. Achari, S. Kandikonda, A. Carisey, S. Pounds, J. Xu, D. W. Ellison, B. Deneen, K. Bertrand, R. Kriwacki, S. C. Mack
          </td>
          <td>2025-08-27</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer arises from a convergence of genetic mutations and epigenetic alterations. Among the consequences of epigenetic deregulation is the aberrant activation of transposable elements, particularly retrotransposons, the most expansive and functionally impactful class. This review examines the paradoxical role of retrotransposons in cancer, emphasizing their dual capacity to support oncogenesis while simultaneously triggering innate and adaptive immune responses that may limit tumor growth. A central question is discussed: Is retrotransposon activation merely a stochastic byproduct of epigenetic instability, or could it represent an intrinsic layer of early tumor surveillance that inadvertently turns against the host as malignancy progresses? Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00368-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a2bf5f051e3bd6c0fa28e6e14cd088f5983472" target='_blank'>
              A plan or pandemonium? The conundrum of retrotransposon activation in cancer
              </a>
            </td>
          <td>
            Elina Zueva
          </td>
          <td>2025-08-09</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The cohesin complex structures the interphase genome by extruding loops and organizing topologically associating domains (TADs). While cohesin engages chromatin in context-dependent modes, the regulatory influence of chromatin state on these interactions remains unclear. Here, we show that histone hyperacetylation, induced by the histone deacetylase inhibitor trichostatin A (TSA), preferentially disrupts short-range interactions within TADs but spares CTCF-anchored loops, despite reduced cohesin occupancy at these sites. These findings point to two functionally distinct cohesin populations: a TSA-sensitive pool within TADs, likely representing extruding, non-topologically bound cohesin, and a TSA-resistant population at CTCF–CTCF anchors that maintains loops through topological entrapment. Using a semi–in vitro system with TEV-cleavable RAD21, we show that TSA-resistant cohesin at CTCF sites becomes TSA-sensitive after proteolytic cleavage that opens the cohesin ring, showing that it is the topological engagement with DNA that makes cohesin, and CTCF-CTCF loops, TSA-resistant. Notably, we also detect TSA-sensitive cohesin at CTCF sites, suggesting the presence of transient, non-encircling cohesin that either precedes conversion to the stable form or is halted by pre-existing encircling cohesin. Together, our results suggest that cohesin exists in distinct biochemical states: an extruding form found within TADs and at CTCF sites, that is sensitive to hyperacetylation, and a topologically bound form specifically at CTCF-CTCF loops that is insensitive. The former may allow dynamic changes in chromatin loops, while latter ensures robustness of CTCF-anchored loops in response to chromatin state changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d85a0b4ebd9ab0ccdb55e1e844bae65bba2393e" target='_blank'>
              Histone Acetylation Differentially Modulates CTCF-CTCF Loops and Intra-TAD Interactions
              </a>
            </td>
          <td>
            Rebecca G Smith, Yu Fu, Kathleen L Schiela, Madison Dautle, Ryan Williams, Hannah M Wilson, Chloe Azadegan, Johnathan R Whetstine, J. Dekker, Yu Liu
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic p53 mutations (Onc-p53) are frequent in lung and many other solid tumors often associated with chromosome aberrations. Why cells with Onc-p53 develop chromosomal aberrations and whether the abnormalities contribute to tumor growth remain elusive. Evidence in this communication demonstrate for the first time that replication stress induced by Onc-p53 triggers re-copying of DNA replication forks, which generates replication intermediates that cause persistent mitotic aberration and DNA segregation errors. Replication intermediates from re-copied replication forks induced by Onc-p53 activate ATM signaling, which stabilizes Onc-p53, reinforces its ability to upregulate replication factors for sustaining replication stress, thus generating a feedforward cycle accelerating tumor formation. In agreement with this observation our time lapse video microscopy show in real time that persistent mitotic aberration and DNA segregation errors induced by Onc-p53 confer selective growth advantage. Accordingly, human lung tumors with Onc-p53 show selection of cells with mitotic aberration during serial passages. Knock down of active replication forks reduces re-copied fork generation by Onc-p53 and specifically induces apoptotic death of lung cancer cells expressing Onc-p53 in xenograft lung tumors synergistically in cooperation with inhibitors of ATM activation, deselecting cells with Onc-p53 with mitotic errors. This communication reveals a novel mechanism which interconnects replication stress induced by Onc-p53 to its stabilization and ability to generate chromosomal aberration in lung cancer cells that both accelerate tumor growth and serve as a targetable therapeutic vulnerability. These findings will be extremely valuable for tumor-specific treatment of a high percentage of cancer patients with p53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77d562f308b7a32a1524622e9f1f8039e4924f9" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage
              </a>
            </td>
          <td>
            S. Deb, Shilpa Singh, Lilia Gheghiani, R. Frum, Steven Grossman, Brad Windle, S. Deb
          </td>
          <td>2025-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancerous transcriptome alterations in carcinoma cells could be originated from either genetic copy number changes or epigenetic reprogramming. Ovarian cancer (OV) is the most malignant gynecologic tumor, known for high aneuploidy with robust copy number alterations. However, low aneuploidy ovarian tumors are also frequently found, indicating an essential contribution of epigenetic factors during tumorigenesis and cancer development. Chromatin remodeling modulates the transcriptome epigenetically in a variety of cancer types, but its role in OV is still unclear. Hence, we investigated a cohort of 102 OV patients, analyzed transcriptomic and clinical data from public databases, and performed cellular experiments. We found that RUVBL2, a subunit of the INO80 complex, functions as the key oncogenic chromatin remodeler in OV. RUVBL2 is upregulated in tumors, particularly in low-aneuploidy cases, and is associated with poor prognosis. RUVBL2 drives nucleosome dynamics and elevates chromatin accessibility selectively at promoter regions. The landscape of RUVBL2-dependent modulation of chromatin accessibility and the transcriptome exhibits activation of various transcription factors, especially the AP-1 family, and upregulation of a series of key genes, including CDKN3, MYBL2, and ZNF144, resulting in mediation of cell cycle and Hippo signaling pathway to promote DNA synthesis and cell proliferation. Hence, RUVBL2-dependent chromatin remodeling plays a key role in oncogenic reprogramming of the transcriptome in OV. These findings provide novel insights into the molecular etiology of OV and disclose potential biomarkers and drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2407b3becd0c884fdbbc4e31bad36bc2c5c367" target='_blank'>
              Profiling of RUVBL2-Induced Transcriptome Alterations Highlights a Critical Role for Chromatin Remodeling in Ovarian Cancer.
              </a>
            </td>
          <td>
            Renhao Xue, Yingjie Wang, Xiaomei Luo, Hao Zhang, Dongcheng Guan, Shuo Shi, Yu Wang
          </td>
          <td>2025-07-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer genomes frequently carry APOBEC (apolipoprotein B mRNA editing catalytic polypeptide-like)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigated the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrated that transcriptional upregulation of APOBEC3A led to the acquisition of a massive number of genomic mutations in just a few cell cycles. By contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation did not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remained minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation revealed a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium.
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repetitive DNA sequences are abundant in genomes and can adopt alternative DNA structures (i.e., non-B DNA). One such structure, Z-DNA, has been shown to stimulate genetic instability in a variety of organisms, including human cells and mice. Z-DNA-forming sequences are enriched at mutation hotspots in human cancer genomes, implicating them in cancer etiology. Aging is a known risk factor for the development of cancer, and genetic instability is a hallmark of both aging and cancer. However, how aging affects the mutagenic potential of Z-DNA has not yet been investigated. Here, we explored the effects of aging on the mutagenic processing of Z-DNA using a transgenic mouse model. Surprisingly, Z-DNA-induced mutations decreased or remained unchanged with increasing age. Cleavage of Z-DNA was unaffected with increasing age, suggesting that downstream repair processing, such as double-strand break repair processes, may be involved in the age-related changes in Z-DNA-induced mutagenesis in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d857eba11e4fc492ce4ba5d13294865a2445fdd6" target='_blank'>
              Effects of Aging on Z-DNA-Induced Genetic Instability In Vivo
              </a>
            </td>
          <td>
            Tonia T. Li, Alexandra M D'Amico, Laura A. Christensen, Karen M. Vasquez
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gastric cancer (GC) is classified into four molecular subtypes according to the Epstein-Barr virus-positive status, microsatellite instability, genomic stability and chromosomal instability (CIN). The CIN subtype is characterized by a high frequency of gene amplifications in receptor tyrosine kinases (RTKs) and a poor prognosis. In addition, the CIN subtype often exhibits intratumoral heterogeneity and indicates insensitivity to targeted drugs. Elucidating the molecular mechanisms of CIN in GC is therapeutically crucial; however, the molecular mechanisms involved are not yet fully understood. The kinetochore-associated 1 (KNTC1) gene encodes kinetochore-associated protein 1 (KNTC1), a major component of the outer kinetochore. The downregulation of KNTC1 causes a high frequency of lagging chromosomes and consequent aneuploidy and CIN in Drosophila and Caenorhabditis elegans. However, the association between KNTC1 and CIN in GC has not yet been clarified. Therefore, the present study investigated the role of KNTC1 in GC CIN. It was found that GC cell lines with a high frequency of lagging chromosomes had a low KNTC1 mRNA expression. Notably, KNTC1 knockdown increased the frequency of lagging chromosomes in GC cell lines. In particular, GC cell lines with the amplification of RTK genes exhibited a significant increase in the frequency of lagging chromosomes. On the whole, the findings of the present study suggest that the suppression of KNTC1 expression may contribute to CIN in GC and may be involved in the generation of intratumoral genetic heterogeneity in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4f34e8d31ebf5a0db6caf271a09a3caf4a8164" target='_blank'>
              Downregulation of kinetochore-associated 1 gene increases lagging chromosomes and contributes to chromosomal instability in gastric cancer cells
              </a>
            </td>
          <td>
            Daiki Ohsaki, Kazuki Kanayama
          </td>
          <td>2025-08-04</td>
          <td>Medicine International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of DNA glycosylases as transcriptional regulators and their involvement in cognition remain unclear. Here, we combine transcriptomic and epigenomic analyses of the hippocampus in mice deficient in DNA glycosylases. We report that the combined deficiency of Ogg1 and Mutyh impairs spatial long-term memory. Mechanistically, Ogg1 and Mutyh modulate DNA methylation of polycomb repressive complex 2 (PRC2) target genes. PRC2 occupancy and associated histone post-translational modifications depend on Ogg1 and Mutyh in neurons and glia. These epigenetic changes correlate with cell-type specific differences in gene expression of PRC2 targets. Finally, human genetic data link DNA glycosylase-regulated genes to cognitive function, highlighting a novel role for Ogg1 and Mutyh in modulating the epigenome to control transcriptional responses relevant for brain-related diseases. Graphical Abstract Created in BioRender. Scheffler, K. (2025). https://BioRender.com/jwscej4. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05730-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788c473f610d3e71e5a7b1644717d7c2a8e6b1a6" target='_blank'>
              DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition
              </a>
            </td>
          <td>
            Andreas Abentung, Teri Sakshaug, Rabina Dumaru, N. Liabakk, Mingyi Yang, Junbai Wang, Magnar Bjørås, Katja Scheffler
          </td>
          <td>2025-08-08</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Centromeres are crucial for precise chromosome segregation and maintaining genome stability during cell division. However, their evolutionary dynamics, particularly in polyploid organisms with complex genomic architectures, remain largely enigmatic. Allopolyploid wheat, with its well-defined hierarchical ploidy series and recent polyploidization history, serves as an excellent model to explore centromere evolution. Results In this study, we perform a systematic comparative analysis of centromeres in common wheat and its corresponding ancestral species, utilizing the latest comprehensive reference genome assembly available. Our findings reveal that wheat centromeres predominantly consist of five types of centromeric-specific retrotransposon elements (CRWs), with CRW1 and CRW2 being the most prevalent. We identify distinct evolutionary trajectories in the functional centromeres of each subgenome, characterized by variations in copy number, insertion age, and CRW composition. By utilizing CENH3-ChIP data across various ploidy levels, we uncover a series of CRW invasion events that have shaped the evolution of AA subgenome centromeres. Conversely, the evolutionary process of the DD subgenome centromeres involves their expansion from diploid to hexaploid wheat, facilitating adaptation to a larger genomic context. Integration of complete einkorn centromere assemblies and Aegilops tauschii pan-genomes further revealed subgenome-specific centromere evolutionary trajectories. By inclusion of synthetic hexaploid from S2-S3 generations, alongside 2x/6 × natural accessions, we demonstrate that DD subgenome centromere expansion represents a gradual evolutionary process rather than an immediate response to polyploidization. Conclusions Our study provides a comprehensive landscape of centromere adaptation, evolution, and maturation, along with insights into how retrotransposon invasions drive centromere evolution in polyploid wheat. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03759-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f98dee703cbbec473e46d88966ab2318a6f43c" target='_blank'>
              Distinct evolutionary trajectories of subgenomic centromeres in polyploid wheat
              </a>
            </td>
          <td>
            Yuhong Huang, Yang Liu, Chang Liu, Congyang Yi, Jinsheng Lai, Hongqing Ling, H. Su, Fang Han
          </td>
          <td>2025-09-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Non-invasive urine tests for bladder cancer (BC) could reduce dependence on flexible cystoscopy for diagnosis and surveillance. Most recent developments in urine testing are based on targeted detection of genomic and/or epigenomic markers. We hypothesised that long-read whole-genome sequencing of urinary DNA with direct methylation profiling may allow accurate BC detection and insights into disease biology. However, the feasibility of such an approach has not yet been reported. Methods We applied long-read whole-genome sequencing with direct methylation detection to urine cell pellet DNA (ucpDNA) from 21 haematuria clinic patients: 13 BCs and 8 non-BCs. The modkit Hidden Markov Model algorithm was used to define differentially methylated regions across the genome. The ability to discriminate between BC and non-BC, and the cellular pathways affected were tested using PCA, h-clust and GSEA. Results We observed global hypomethylation and cancer-specific patterns of promoter hypermethylation in urine from BC patients. Sequencing of a single ucpDNA sample per flow cell yielded read depths of 18-34x; furthermore, BC methylation patterns were also evident with 2–5x multiplex sequencing. Copy number changes were also evident in ucpDNAs from BC patients. A limitation of the study is the small number of samples analysed; however, the detection of cancer-specific events demonstrates the feasibility of the approach, both in single and multiplexed flow-cell runs. Conclusions Even at low-read depths, genome-wide methylation patterns in urinary DNA reflect the presence of BC, potentially permitting rapid, non-invasive and cost-effective BC detection. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01946-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390cf4d4c35b389c0ed48f2ce2f3090aabaa5c3" target='_blank'>
              Detection of genome-wide methylation changes in bladder cancer by long-read sequencing of urinary DNA
              </a>
            </td>
          <td>
            A. Goel, B. Tura, Joanne D. Stockton, Nicholas Tovey, Luke Ames, Andrew D. Beggs, M. Zeegers, Nicholas D. James, K. Cheng, Richard T. Bryan, D. G. Ward, R. Arnold
          </td>
          <td>2025-08-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Overexpression of the proto-oncogene MYC occurs in >70% of cancers and is especially prevalent in breast cancer. Myc partners with transcription factor Max to bind to the E-box DNA response element. By patterning our frankenproteins on the basic region/helix-loop-helix/leucine zipper motif of Max, we designed MEF and MEF/C93 to bind to the E-box. In bacterial one-hybrid assays and quantitative electrophoretic mobility shift assay, both proteins bound specifically to the E-box with high sequence-specificity and affinity (Kd = 8 nM) rivaling native transcription factors. Quantitative PCR revealed that MEF and MEF/C93 selectively downregulated Myc target genes in Myc-dependent MDA-MB-231 breast cancer cells, but not in Myc-independent MCF-7 cells. Fluorescence colocalization demonstrated transport into cell nuclei. Our proteins displayed IC50 values of 1-2 μM in cell viability assays in MDA-MB-231, compared with ∼25 μM in MCF-7. These results demonstrate the specificity of targeting Myc-dependent cancer cells and mark significant progress toward protein-based therapies aimed at Myc-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cc39bbdc2499f24c52dd2589dd7f515be4ffe6" target='_blank'>
              Designer Frankenproteins That Halt the Proliferation of Myc-Driven Cancer Cells.
              </a>
            </td>
          <td>
            Maryam Ali, Raneem Akel, Francine He, Micheline Piquette-Miller, Jumi A Shin
          </td>
          <td>2025-09-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs), once considered genomic “junk”, are now recognized as critical regulators of genome function and human disease. These mobile genetic elements—including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons—are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages—such as zygotic genome activation and cell fate determination—where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs—and balancing their contributions to normal development and disease—is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The ongoing arms race between prokaryotes and mobile genetic elements (MGEs) gives rise to a myriad of host-defense systems that detect and degrade invading nucleic acids. However, it is yet unclear whether changing the mechanical properties of nucleic acids can impact plasmid/phage invasion. Here, we demonstrate that H-NS, an abundant nucleoid-associated protein (NAP), limits plasmid transfer by directly binding to and oligomerizing along with transforming double-stranded DNA. The constitutive defense function of H-NS can be complemented by convergently evolved NAPs from different phyla. H-NS proteins form intramolecular bridges between DNA duplexes within incoming plasmids. Different from other defense systems that exist at low levels prior to detecting MGEs, major NAPs exist in high abundance, which may moonlight as a constant plasmid surveillance agent. Our study implies that mechanical modification of nucleic acids may be an underexplored mechanism for prokaryotic immunity, which could help balance the advantages and disadvantages of MGEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992180cb487a041111c4f065e8e80d198a5eb5b" target='_blank'>
              Abundant bacterial nucleoid-associated protein H-NS limits plasmid transfer through mechanical modification of DNA
              </a>
            </td>
          <td>
            Mingyue Fei, Mengdie Fang, Qi Zhou, Ziyan Chen, Mengxin Gong, Fabai Wu, Changfu Tian, Dongchang Sun
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: a comprehensive and high-quality database for extrachromosomal circular DNA.
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cell division cycle 6 (CDC6) is a key licensing factor for DNA replication in the G1 and S phases. Besides initiating replication, CDC6 also helps establish and maintain the S-M checkpoint, ensuring genomic stability. Emerging evidence highlights its dysregulation in various cancers, implicating CDC6 in tumor progression and therapy resistance. However, a comprehensive pan-cancer analysis evaluating its diagnostic, prognostic and immunomodulatory potential remains lacking, underscoring the need for further investigation. Methods By integrating multi-omics datasets from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression Project (GTEx), cBioPortal, Human Protein Atlas (HPA), UALCAN and SangerBox, we employed systematic bioinformatics approaches to investigate the oncogenic role of CDC6 across multiple cancer types. Our analysis encompassed prognostic associations, mutational landscapes, tumor immune microenvironment (TIME) infiltration patterns and epigenetic regulation via DNA methylation, providing a pan-cancer perspective on CDC6’s potential role in tumorigenesis. In addition, CDC6’s role in melanoma cell proliferation, invasion and migration was experimentally assessed. Results Pan-cancer analysis demonstrated CDC6 as a consistently upregulated oncogene across multiple malignancies, exhibiting significantly elevated expression compared to normal tissues. Notably, CDC6 is closely associated with prognosis across various cancer types. Our investigation further revealed robust correlations between CDC6 expression and immune cell infiltration patterns. Epigenetic profiling identified significant associations between CDC6 expression and DNA methylation alterations in nine cancer types. Functional studies validated CDC6’s oncogenic role, where its overexpression significantly promoted cellular proliferation, migration and invasion in melanoma. Conclusions Our study demonstrates that CDC6 serves as a crucial oncogenic driver across diverse tumor types, establishing its dual utility as a diagnostic biomarker and independent prognostic indicator. Importantly, we identified a significant correlation between elevated CDC6 expression and specific immune microenvironment alterations, suggesting its potential as a predictive biomarker for immunotherapy response. These findings demonstrate CDC6’s dual role in cancer development and immune regulation, warranting further investigation into its mechanisms and therapeutic potential. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14782-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b86557cdc5c9f375fa0289acd10e271c4227fce8" target='_blank'>
              CDC6 as a pan-cancer immunological and prognostic biomarker and its role in suppressing melanoma malignancy
              </a>
            </td>
          <td>
            Lijun Mo, Min Jia, Qian Wu, Qingqing Xu, Z. Mai, Bo Fu, Zhengqiang He, Mingchang Huang, Yinyuan Lan, Qiaozhen Xu, Zhenmou Xie, Xiaoni He, Yaohua Xue
          </td>
          <td>2025-09-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Esophageal carcinoma (ESCA) is a common tumor in the digestive system, resulting in approximately 300,000 death worldwide every year. ALDH18A1, a potential RNA binding protein (RBP), is significantly overexpressed as in ESCA, while its functions and mechanism is unclear. In this study, we silenced ALDH18A1 in KYSE150 cells using small interfering RNA (siALDH18A1), and evaluated its effect on cell characteristics. Finally, we conducted whole transcriptome sequencing (RNA-seq) and RNA immunoprecipitation (iRIP-seq) experiments to explore the downstream targets of ALDH18A1, and verified their change patterns using quantitative polymerase chain reaction. We found that ALDH18A1 exhibited a high expression level in ESCA patients using online database. The proliferation, invasion, and migration abilities of KYSE150 cells were significantly inhibited in siALDH18A1 samples. Through the investigation of RNA-seq data, we found that ALDH18A1 globally regulates the expression and alternative splicing profiles of KYSE150 cells by identifying 437 differentially expressed genes (DEGs) and 1071 ALDH18A1-regulated AS events (RASEs), respectively. The functions of RASE genes (RASGs) were enriched in pathways such as DNA repair and gene expression. In addition, through the integration analysis of iRIP-seq data and DEGs, we found that ALDH18A1 bound to and regulated the expression of 59 DEGs, including LAMB3 and IL1B, which were verified by RT-qPCR experiment. In addition, we performed an overlap analysis between the genes bound by ALDH18A1 and RASGs and found 339 bound RASGs. The functional enrichment of these genes revealed pathways such as DNA repair, cell proliferation, and DNA damage response. Among them, we found significant alternative splicing events of MTA1, which were confirmed by PCR experiments. In summary, our study comprehensively explored the functions and downstream targets of ALDH18A1 in ESCA cells, highlighting the molecular mechanism of ALDH18A1 on the carcinogenic characteristics of ESCA. The identified molecular targets, especially DNA repair-related genes, can serve as potential therapeutic targets for future ESCA treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08006-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4696a54f3a85524e5b23e6278282dcbdfa70bd9a" target='_blank'>
              ALDH18A1 has carcinogenic functions and regulates alternative splicing events of DNA repair-related genes in esophageal carcinoma cells
              </a>
            </td>
          <td>
            Yongkang Wang, Maimaiti Yisireyili, Kelimu Abudureyimu, Atsushi Enomoto
          </td>
          <td>2025-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b074d01f6fb78ae72372c0d26a26df7f99fb224" target='_blank'>
              Fungi as models of centromere innovation: from DNA sequence to 3-dimensional arrangement.
              </a>
            </td>
          <td>
            Srijana Dutta, Krishna Bhat, Rashi Aggarwal, K. Sanyal
          </td>
          <td>2025-08-11</td>
          <td>Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Ribophorin 1 (RPN1), a key component of the oligosaccharyltransferase complex, is implicated in tumor progression through glycosylation-mediated pathways, yet its pan-cancer roles remain unexplored. This study presents a comprehensive multi-omics analysis of RPN1 across 33 cancers such as sarcoma (SARC), integrating genomic, transcriptomic, and proteomic data from TCGA, GTEx, and CPTAC. RPN1 was significantly overexpressed in 14 malignancies and correlated with advanced tumor stages and poor prognosis in glioblastoma (GBM), lower-grade glioma, SARC and hepatocellular carcinoma, validated in an independent glioma cohort (n = 151). Genomically, RPN1 amplification linked to homologous recombination deficiency and elevated tumor mutational burden, suggesting a role in genomic instability. Critically, multiplex immunofluorescence demonstrates RPN1 overexpression colocalizes with CD206+ M2 macrophages in tumor microenvironments, while in vitro coculture experiments confirm RPN1-dependent microglial recruitment and M2 polarization. RPN1 expression negatively correlates with CD8+ T cell infiltration and predicts resistance to chemotherapy (GBM, ovarian cancer) and immunotherapy (GBM, esophageal carcinoma), though it associates with PD-1 inhibitor sensitivity in bladder cancer. Functional validation shows RPN1 knockdown suppresses proliferation, migration, and invasion in GBM cells. Pathway enrichment connects RPN1 to endoplasmic reticulum stress, glycosylation, DNA repair, and immune checkpoint regulation. These findings position RPN1 as a multimodal oncogenic driver promoting genomic instability, immunosuppressive microenvironment remodeling, and context-dependent therapeutic vulnerabilities across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fad4cdee9a5624b27cf2ed12f95c27e148fc7b5" target='_blank'>
              RPN1 Is Associated With Immunosuppression in Pan-Cancer and Affects the Malignant Phenotype of Tumor.
              </a>
            </td>
          <td>
            Shaoru Lin, Changwu Wu, Qing Liu, Fushu Luo, Jun Tan
          </td>
          <td>2025-08-26</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC), an aggressive neuroendocrine tumor strongly associated with exposure to tobacco carcinogens, is characterized by early dissemination and dismal prognosis with a five-year overall survival of less than 7%. High-frequency gain-of-function mutations in oncogenes are rarely reported, and intratumor heterogeneity (ITH) remains to be determined in SCLC. Here, via multiomics analyses of 314 SCLCs, we found that the ASCL1+/MKI67+ and ASCL1+/CRIP2+ clusters accounted for 74.38% of the 190,313 SCLC cancer cells from 39 patients, with the ASCL1+SOX1+ stem-like cell cluster across SCLC subtypes. The major histocompatibility complex (MHC) class I molecules were expressed at low levels in six and high levels in five cancer cell clusters and were inversely associated with the KI67 expression level. Abnormal splicing of mRNAs was a feature of SCLC, with focal adhesion kinase (FAK) splicing variants identified in 119 (77.3%) of 154 patients. FAK variants exhibited elevated kinase activity, were associated with the worst prognosis, and were sensitive to FAK inhibitors in patient-derived organoids and xenograft models. Eleven high-frequency mutations were identified in addition to TP53 and RB1, and smoking status and tumor stage did not affect microbiota variance in SCLC. Taken together, our data further revealed the complicated ITH and discovered that FAK splicing variants represent high-frequency gain-of-function alterations in oncogene in SCLC and potential therapeutic targets for this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3bde80c19d7166cce5785be8ae3a4a5e971fbbd" target='_blank'>
              Characterization of the extrinsic and intrinsic signatures and therapeutic vulnerability of small cell lung cancers
              </a>
            </td>
          <td>
            Gui-zhen Wang, Zheng Wang, Shi-Hao Bai, Yun Tan, Wen-Zhao Zhong, Guo-Gui Sun, Yu-Tao Liu, B. Pan, Chen Huang, Di Wang, Beibei Sun, Dong-Ni Chen, Bin Zhang, Yong-Chun Zhou, Sheng Li, Xiang-Wei Zhang, Si-Chong Han, Fu-ying Yang, Xue-Yan Shi, Xiao-Liang Jie, Yu-Ke Shen, L. Liang, Zhe-Sheng Wen, Li Zhang, Mingkun Li, Na Wang, Jin-Song Liu, Ying Dong, Manli Wang, Yan Wang, Chang-Li Wang, Dawei Xie, Ze-Guang Han, Jian-Ming Ying, Chong Chen, Yunchao Huang, Hongbin Ji, Yuan-Yuan Zhang, Yan Yu, Guang-Biao Zhou
          </td>
          <td>2025-09-10</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA and limit the rate of HSV-1 genome decompaction. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral immediate-early (IE) transcription, genome expansion, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617d8ae459ea016f8dca2b2644f05eeab5725c80" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P. E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Mila Collados Rodriguez, Anna. R Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da656da84cf3e9dc1144dda4829e9c83aef2891" target='_blank'>
              CHAMP1 Complex Promotes Heterochromatin Assembly and Reduces Replication Stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background DNA methylation (DNAm) plays an important functional role and is influenced by genetic variants known as methylation QTLs (meQTLs). The majority of meQTL studies have been conducted in human blood. Despite its unique landscape, the genetic regulation of sperm DNAm remains largely unexplored. In this study, we leveraged DNAm measured in sperm from 405 Holstein bulls using reduced representation bisulfite sequencing (RRBS) and performed sequence-level genome-wide association studies for 166,985 variable CpGs (s.d. >5%). We reported heritability estimates and have mapped both cis-meQTLs and trans-meQTLs. Results Heritability estimates ranged from 0 to 1 and averaged 0.26 across all selected CpGs, with 76% of estimates above 0.1. The meQTL mapping revealed that 32.9% of the CpGs had a cis-meQTL, 3.6% had a trans-meQTL and 1.0% had both cis- and trans-meQTLs. The cis-CpGs were located on average 261 kb (absolute mean) from their cis-meQTL top SNPs (defined by the most significant association). MeQTLs were enriched in featured genomic annotations, including regions surrounding transcription start sites and ATAC-seq peaks. We also identified spurious trans-associations by analyzing data across multiple genome assemblies, including the construction of a partial pangenome. Additionally, eight trans-meQTL hotspots, defined as variants associated with at least 30 trans-CpGs, were identified and overlapped with genes involved in epigenetic regulation. Using peripheral blood mononuclear cell DNAm from 54 out of the 405 bulls, we did not observe a similar effect of the trans-meQTL hotspots to that one observed in sperm. Conclusions For the first time, meQTLs have been detected and characterized in bovine sperm, contributing to a better understanding of the transmission of paternally inherited DNAm marks. These findings provide useful information for further research aimed at integrating epigenetic information into the prediction of performance traits. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11934-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03d42a2f9879b2547c8e0d23b4ffde22fa6774da" target='_blank'>
              Genetic regulation of sperm DNA methylation in cattle through meQTL mapping
              </a>
            </td>
          <td>
            C. Fouéré, Valentin Costes, C. Hozé, Amrita Raja Ravi Shankar, Florian Besnard, G. Costa Monteiro Moreira, Valentin Sorin, C. Le Danvic, A. Chaulot-Talmon, Francesca Ali, M. Deloche, Aurélie Bonnet, E. Sellem, Hélène Jammes, Sébastien Fritz, M. Boussaha, D. Boichard, Hélène Kiefer, M. Sanchez
          </td>
          <td>2025-08-22</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and leukemogenic function remain unclear. A major challenge has been their large size, which has impeded preclinical model development and mechanistic studies. Results We employ a domain-focused truncation strategy to generate de novo murine models of K/C and K/P fusions, which faithfully recapitulate the morphological, immunophenotypic, and transcriptomic features of KAT6A-rearranged AML. Genomic profiling reveals that KAT6A fusions preferentially localize to H3K4me2/3-marked regions, while biochemical analyses uncover that KAT6A interacts with the Nucleosome Remodeling Factor (NURF), a key H3K4me2/3 reader. Disrupting NURF-chromatin interactions via depletion or small-molecule inhibition of its subunit, Bromodomain PHD Finger Transcription Factor (BPTF), impairs K/C recruitment and disrupts MLL/COMPASS-mediated H3K4me2 deposition, defining a functional epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS. Notably, CBP/P300 inhibition reduces histone acetylation and chromatin accessibility, further impairing the recruitment of this epigenetic module. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects observed when combined. Conclusions Our study identifies a self-reinforcing epigenetic module of histone modifiers and readers in KAT6A-rearranged AML, providing mechanistic insights into the genomic targeting of KAT6A chimeras and highlighting promising combinatorial therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03743-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b1939d769bfdab61413c545354658eef418fa8" target='_blank'>
              KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML
              </a>
            </td>
          <td>
            Junhui Lv, Zhinang Yin, Conghui Li, Honglin Wen, Jian Ni, Peiyuan Yang, Zemin Song, Ying Xiang, Honghong Wang, Rui Lu, Li Huang, Ying Zhou, Hai-Bing Zhou, Ruijing Xiao, Pingping Fang, Kaiwei Liang
          </td>
          <td>2025-08-19</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Invasive cervical cancer (ICC) is the leading cause of cancer-related deaths among women in low-income regions, with approximately 70% of cases caused by high-risk human papillomavirus (HPV) strains—primarily HPV 16 and HPV 18. HPV infection triggers the production of viral oncoproteins E6 and E7, which interfere with the tumor suppressor proteins p53 and pRb, respectively, leading to disrupted cell cycle control and uncontrolled cellular proliferation. Moreover, only a few targeted therapies are approved for treatment.



 To address this gap, we conducted exome sequencing on 400 ICC fresh frozen samples from Guatemala, the country with one of the highest incidences of cervical cancer globally. We also performed long-read whole genome sequencing (WGS) on 29 cervical cancer cell lines with a coverage depth ranging from 50 to 100x. Additionally, ICC data from public database like TCGA was included in the analysis.



 Our lab previously identified that YAP1 oncogene amplification (11%) is associated with earlier diagnosis (12 years earlier on average), poorer survival, and a novel aggressive cervical cancer subtype, more prevalent in minority populations. Treatment with the PIK3CA inhibitor Alpelisib reduced proliferation in YAP1 amplified cervical cancer cell lines, like CaSki, SNU1000, SNU1299, QG-U, C4I and CERV-215 but had no effect on PIK3CA wildtype cells. We also observed that STK11, a tumor suppressor gene, is frequently altered in ICC. Exome sequencing data from Guatemala revealed that 32% of tumors (25% in adenocarcinoma (ASC) and 7% in squamous cell carcinoma (SCC)) have STK11 alterations, consistent with our WGS data showing alterations in 35% of cervical cancer cell lines. These mutations are associated with a 10-year earlier age of diagnosis in minority populations. Furthermore, we noticed that STK11 and YAP1 are both functionally involved in the same Hippo signaling pathway. Interestingly, we found in TCGA that 24% of STK11 wild-type cases have YAP1 amplifications while only 8% of STK11 altered cases have YAP1 amplification (p=0.009), exhibiting a co-occurrence of STK11 alterations with YAP1 amplification.



 We propose a model that HPV E7 oncoprotein and STK11 alterations activate YAP1, promoting oncogenesis. Therefore, co-occurrence of STK11 alterations and HPV infection may synergize to promote more aggressive cancers through hyper-activation of YAP1. Presently, we are validating our observations in highly expressed STK11 cervical cancer cell lines (direct RNA sequencing data) through siRNA transfection and protein expression. This suggests that evaluating YAP1 expression in STK11-deleted cell lines could uncover additional opportunities for targeted therapies and screening invasive cervical cancer cases for STK11 and YAP1 gene alterations may help identify women who are more likely to respond favorably to targeted therapies.



 Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabele Rodriguez, Eduardo Gharzouzi, Enrique Alvirez, Michael Dean. Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A085.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f27666748c2b6097853c42aef3cfa545e01c9e3" target='_blank'>
              Abstract A085: Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer
              </a>
            </td>
          <td>
            Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabel Rodríguez, E. Gharzouzi, E. Alvirez, Michael Dean
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9490feaeec24c63eafa0bf22c93391f3883ea64" target='_blank'>
              [Exploration of ribosomal DNA copy number as an early biomarker for exposure to chemical carcinogens].
              </a>
            </td>
          <td>
            C. Ding, H. D. Xu, J. L. Lou
          </td>
          <td>2025-08-20</td>
          <td>Zhonghua lao dong wei sheng zhi ye bing za zhi = Zhonghua laodong weisheng zhiyebing zazhi = Chinese journal of industrial hygiene and occupational diseases</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations that initiate AML can cause clonal expansion without transformation ("clonal hematopoiesis"). Cooperating mutations, usually in signaling genes, are needed to cause overt disease, but these may require a specific "fitness state" to be tolerated. Here, we show that nearly all AMLs arising in a mouse model expressing two common AML initiating mutations (Dnmt3aR878H and Npm1cA) acquire a single copy amplification of chromosome 7, followed by activating mutations in signaling genes. We show that overexpression of a single gene on chromosome 7 (Gab2, which coordinates signaling pathways) is tolerated in the presence of the Npm1cA mutation, can accelerate the development of AML, and is important for survival of fully transformed AML cells. GAB2 is likewise overexpressed in many human AMLs with mutations in NPM1 and/or signaling genes, and also in Acute Promyelocytic Leukemia initiated by PML::RARA; the PML::RARA fusion protein may activate GAB2 by directly binding to its 5' flanking region. A similar pattern of GAB2 overexpression preceding mutations in signaling genes has been described in other human malignancies. GAB2 overexpression may represent an oncogene-driven adaptation that facilitates the action of signaling mutations, suggesting an important (and potentially targetable) "missing link" between the initiating and progression mutations associated with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6f3ad6142f75e1cb2d2a0d8414ccd605f8e467" target='_blank'>
              Overexpression of the signaling-coordinator GAB2 can play an important role in Acute Myeloid Leukemia progression.
              </a>
            </td>
          <td>
            Michael H. Kramer, Stephanie N Richardson, Yang Li, Tiankai Yin, Nichole M. Helton, Daniel R. George, Michelle Cai, S. M. Ramakrishnan, C. Katerndahl, Christopher A. Miller, T. Ley
          </td>
          <td>2025-08-07</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a malignant epithelial tumor. N4-acetylcytidine (ac4C) modification regulates mRNA stability and contributes to tumorigenesis. FOXD1, a crucial transcription factor, acts as a tumor-promoting factor in NPC. However, its molecular underpinnings are not fully elucidated. Methods Expression of mRNA and protein was quantified by quantitative PCR, immunohistochemistry, or immunoblotting. The effects on cell phenotypes were determined by assessing NPC cell proliferation, apoptosis, invasiveness, sphere formation, and TUVEC tube formation. The interaction between FOXD1 and N-acetyltransferase 10 (NAT10) was predicted by online computational methods and validated using RIP, mRNA stability, ChIP, and luciferase assays. Xenograft studies were performed to observe the in vivo effects. Results FOXD1 expression was increased in NPC clinical samples and cell lines. Functionally, FOXD1 depletion suppressed NPC cell growth, invasion, sphere formation ability, while promoting cell apoptosis and impairing HUVEC tube formation. Mechanistically, NAT10 stabilized FOXD1 mRNA by mediating its ac4C modification. FOXD1 upregulation counteracted NAT10 depletion-driven cellular phenotypic alterations. In turn, FOXD1 acted as a transcriptional activator of NAT10 in NPC cells. NAT10 reconstitution reversed FOXD1 depletion-driven cellular phenotypic alterations. Additionally, NAT10 knockdown inhibited NPC xenograft growth in vivo by reducing FOXD1 levels. Conclusion Our study demonstrates that a mutually reinforcing FOXD1/NAT10 positive feedback loop drives NPC progression, providing new therapeutic vulnerabilities for clinical intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00555-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31c4d75d857fdedce25a48b5ec00232b33297e48" target='_blank'>
              The FOXD1/NAT10 positive feedback loop drives nasopharyngeal carcinoma progression
              </a>
            </td>
          <td>
            Leifeng Liu, Qizhu Chen, Yiling Li, Weihao Wu, Feng Jiang, Haitao Qiu
          </td>
          <td>2025-09-25</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Transposable elements (TEs) have hijacked cellular machineries to replicate and spread throughout host genomes. TEs now make up a significant portion of eukaryotic genomes and play notable roles in genomic evolution, driving both speciation and providing raw material for genetic innovation. Barbara McClintock's pioneering work on these “jumping genes” laid the foundation for modern TE research; however, her paradigm‐shifting theories in which TEs act as “controlling elements” were initially rejected due to the long‐held belief that TEs were “junk” or parasitic DNA elements. Historically, the highly repetitive nature of TEs made it challenging to both identify and investigate functions. However, recent advances in genomics have greatly accelerated our understanding of TEs. Despite their potential to cause insertional mutagenesis and disease, many transposable elements have been co‐opted by host genomes to contribute to gene regulation and development. In contrast to protein‐coding genes that typically begin their journey as DNA, are transcribed into RNA, and reach their ultimate functional form as proteins, TEs can function as cis‐regulatory DNA, functional RNA, and in rare cases, domesticated proteins and fusion events between TE and host genes. Driven by rapidly advancing technologies, the roles of TEs in both development and disease are being uncovered faster than ever, making current and future work an exciting continuation of Barbara McClintock's groundbreaking legacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831a0252c52d2b92a8d6f28251facae6b2cf9eff" target='_blank'>
              From Junk DNA to Genomic Treasure: Impacts of Transposable Element DNA, RNA, and Protein in Mammalian Development and Disease
              </a>
            </td>
          <td>
            Ten D. Li, Katelyn Toohill, A. Modzelewski
          </td>
          <td>2025-07-01</td>
          <td>Wiley Interdisciplinary Reviews. RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc917a5b15125f8360c9f54b472a10a0463afbf2" target='_blank'>
              Oncogenomic profiling of cutaneous pericytic tumours reveals distinct drivers and shared biological processes
              </a>
            </td>
          <td>
            M. D. C. Velasco-Herrera, S. Cheema, K. Wong, J. Billington, I. Vermes, E. Anderson, Elizabeth Ferla, Paul W. Harms, N. D. S. Aubain, Emily L. Clarke, William Merchant, A. K. Alomari, Neil Rajan, Peter M Ferguson, Maximillian A. Weigelt, C. Monteagudo, Steven D. Billings, M. Arends, I. Ferreira, T. Brenn, L. V. D. Weyden, David J. Adams
          </td>
          <td>2025-08-08</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is the most frequent soft tissue sarcoma of childhood. The fusion positive (FP) subtype expressing the chimeric transcriptional factor PAX3-FOXO1, the driver of malignancy, has limited treatment options due to resistance to treatments. As a pediatric cancer, RMS has a low mutational burden with epigenetic transcriptional deregulations playing key roles. Thus, targeting the basic transcriptional machinery including catalytic effectors of transcription is an emerging anti-cancer approach. The transcriptional cyclin-dependent kinase CDK12 regulates the transcription elongation phase, particularly of the DNA damage response (DDR) genes, by phosphorylating Ser2 (pSer) on RNA PolII. In Ewing sarcoma, CDK12 inhibition leads to a DNA repair deficient phenotype and high sensitivity to DNA damaging agents. PAX3-FOXO1 RMS cells display elevated levels of replication stress highly relying to DDR for survival. Therefore, in this study we evaluate the impact of CDK12 inhibition on PAX3-FOXO1 RMS tumorigenic features. We found that CDK12 mRNA levels were highly up-regulated in RMS primary samples compared to healthy control muscles, with one of the highest expression among the adult and pediatric tumors analyzed. Then, we treated PAX3-FOXO1 cell lines with THZ531, an inhibitor of CDK12 and its paralog CDK13. Six hours of THZ531 treatment at nanomolar concentrations decreased pSer2 of RNA PolII and reduced PAX3-FOXO1 transcripts and protein levels. The drug also reduced cell survival in 2D and 3D culture conditions. THZ531-treated cells accumulated in G2/M phase and underwent PARP- and Casp 3/7-dependent apoptosis, increased markers of DNA damage associated to DNA double-strand breaks (neutral comet assay), and reduced the levels of the DDR players ATM, ATR, RAD51, BRCA1 and BRCA2. Similar results were obtained after treatment with another CDK12/13i with Cyclin K degrader ability, SR-4835, and with a CDK12-specific degrader, BSJ-4-116, suggesting on-target effects. The three drugs showed similar low nanomolar concentrations effective on PAX3-FOXO1 RMS organoids. Preliminary RNAseq data in two PAX3-FOXO1 RMS cell lines showed that THZ531 downregulated only a low number of genes, common to the two cell lines. The most downregulated gene families were those of DDR pathways. Moreover, a negative correlation between gene length and gene expression levels was found. CDK12 KO in two RMS cell lines affected cell survival in 2D and 3D conditions compared to control cells suggesting on-target activity of the drugs. Phosphoproteomics analysis after THZ531 treatment showed a significant decrease of a number of serines and threonine phosphopeptides also involving a subset on DDR proteins. THZ531 treatment resulted synergic in vitro with Olaparib (PARPi) and with JQ1 (BRD4i). Interestingly, BRD4 is needed for PAX3-FOXO1 activity on oncogenic super-enhancers. Analysis of nascent RNAs and mapping of RNA PolII to identify direct targets of CDK12 is ongoing. Supported by Associazione Italiana per la Ricerca sul Cancro (AIRC) IG #27794 grant to RR



 Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, Francesca Antonella Aiello, Matteo Cassandri, Silvia Pomella, Adam D. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T Meister, Jarno Drost, Concetta Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, Biagio De Angelis, Kristy R Stengel, Rossella Rota. CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd33df041a8224c723527c37b828d7c14637d9c" target='_blank'>
              Abstract B015: CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, F. Aiello, Matteo Cassandri, S. Pomella, A. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T. Meister, J. Drost, C. Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, B. De Angelis, Kristy Stengel, R. Rota
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, K. Cheng, Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xin-Yuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, A. W. Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract The presence of CDKN2A/B homozygous deletion (HD) denotes a higher WHO grade and more aggressive behavior in gliomas and meningiomas. The gold standard assays for detecting copy number variation (CNV) include chromosomal microarray, next generation sequencing, and fluorescence in situ hybridization (FISH). However, these methods are costly, time-consuming, and are limited in resource-sparse centers. Thus, an inexpensive and time-efficient assay for detecting CDKN2A/B HD is necessary for routine neuropathology cases. Chromogenic in situ hybridization (CISH) combines DNA hybridization techniques utilized in FISH with conventional peroxidase reactions utilized in immunohistochemistry to interrogate CNV with brightfield microscopy. We hypothesize that CISH is a sensitive and specific method for detecting CDKN2A/B HD in gliomas and meningiomas. We performed CDKN2A CISH on a retrospective cohort of whole slide tissue sections of 5 gliomas and 12 meningiomas using the ABNOVA CDKN2A/Centromere 9 (CEP9) CISH probes and kit. Two pathologists independently counted and scored 100 cells from each case showing any CEP9 (red) signals and no CDKN2A (green) signals as HD, and cells showing red and green signals as CDKN2A/B intact. The pathologists’ scores were averaged and compared to the CDKN2A/B status for each of the glioma and meningioma cases, acquired via FISH and methylation profiling, respectively. Our results demonstrate that detection of CDKN2A/B HD with CISH showed 100% concordance with CDKN2A/B locus interrogation with FISH and methylation profiling. CISH is therefore a sensitive and specific method for detecting CDKN2A/B HD in glioma and meningioma and is a practical alternative to traditional CNV detection methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee0c49f58caa417ea1856c56abdf8eb9dd1a185" target='_blank'>
              25 CHROMOGENIC IN SITU HYBRIDIZATION FOR DETECTION OF CDKN2A/B HOMOZYGOUS DELETION IN GLIOMA AND MENINGIOMA
              </a>
            </td>
          <td>
            K. C. Martin, Tara Spence, Janine Senz, Julia Naso, Andrew Churg, Stephen Yip
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The interdependence of chromatin states and transcription factor (TF) binding in eukaryotic genomes is critical for the proper regulation of gene expression. In this study, we explore the connection between TFs and chromatin states in the human malaria parasite, Plasmodium falciparum, throughout its 48-hour asexual intraerythrocytic developmental cycle (IDC). Most P. falciparum genes are expressed in a periodic manner during the IDC, accompanied by dynamic shifts in histone modifications and chromatin accessibility. Leveraging genome-wide profiles of chromatin accessibility, histone modifications, and Heterochromatin Protein 1 (HP1) occupancy, we characterize chromatin state dynamics during the IDC. Our results indicate that several chromatin states remain stable throughout the lifecycle, while others are dynamic and are linked to gene activation or repression. We further characterize chromatin state dynamics at the genome-wide DNA binding sites for a selection of Plasmodium TFs, allowing us to group TFs according to their chromatin preferences. By correlating changes in chromatin accessibility, histone modifications, and TF binding, we provide a global overview of the chromatin state dynamics that coordinate P. falciparum asexual blood stage development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b686ddbb8d5ef2212dff5e6a4c64a743f3b09bc" target='_blank'>
              Chromatin state dynamics during the Plasmodium falciparum intraerythrocytic development cycle
              </a>
            </td>
          <td>
            Alan S. Brown, Manuel Llinás, Shaun Mahony
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome transmission fidelity is vital for organism fitness. Yet, extrinsic and intrinsic changes can affect this process, leading to aneuploidy, the loss/gain of chromosomes, which is a hallmark of cancer. Here, using a haploid fission yeast Schizosaccharomyces pombe strain with a segmental aneuploidy, we assayed genome stability under different temperatures and altered gene dosage. We find that S. pombe genome stability is temperature-dependent and is unexpectedly modulated by intracellular levels of inorganic polyphosphate polymers (polyP). The vtc4+ gene, encoding a subunit of the polyP-generating VTC complex, is present twice due to the segmental aneuploidy resulting in a gene-dosage-coupled increase in polyP. Using strains with different amounts of polyP, we find a direct negative correlation between polyP and chromosome segregation fidelity. PolyP modulates the function of the conserved CCAN kinetochore subcomplex, as the abnormal growth phenotype caused by the mutant CCAN protein Fta2-291 was rescued in the absence of polyP, while extra polyP had the opposite effect. Importantly, this appears to occur in part by modulation of the nucleolin Gar2. Gar2 is the functional homolog of the Saccharomyces cerevisiae Nsr1 protein, whose function is modulated by posttranslational polyP-mediated polyphosphorylation. Thus, polyP modulates genome stability, linking cellular metabolism to chromosome transmission fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/073fe49e2d43274ed78a92bbb92ef640f4ef5dad" target='_blank'>
              Inorganic Polyphosphate Modulates Chromosome Transmission Fidelity in the Fission Yeast Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Sarune Bollé, Elisa Koc, Adolfo Saiardi, Lisa Juhran, Eva Walla, Ursula Fleig, Abel R Alcázar-Román
          </td>
          <td>2025-09-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd145cac2df335e27d4a9869d5f4c02876f7c50" target='_blank'>
              RNA reinforces condensate nucleation on chromatin to amplify oncogenic transcription.
              </a>
            </td>
          <td>
            Krista A Budinich, Xinyi Yao, Chujie Gong, Lele Song, Michelle C Lee, Qinglan Li, Sylvia Tang, Kaeli M Mathias, Yiman Liu, Son C Nguyen, Eric F. Joyce, Yuanyuan Li, Haitao Li, Liling Wan
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A substantial portion of lung cancer-associated genetic elements in East Asian populations remains unidentified, underscoring the need for large-scale genome-wide studies, particularly on non-coding regulation. We conducted a whole genome sequencing (WGS)-based genome-wide scan in 13,722 Chinese individuals to identify regulatory elements associated with lung cancer. We verified common-variant-based loci by meta-analysis across the available East Asian studies. Integrating a genome-transcriptome reference panel of lung tissue in 297 Chinese, we bridged the variant-lung cancer associations, highlighting genes including TP63 and DCBLD1. Implementing the STAAR pipeline for rare variant aggregate analysis, we identified and replicated novel genes, including PARPBP, PLA2G4C, and RITA1 in the context of non-coding regulation. Adapting a deep learning-based approach, potential upstream regulators such as TP53, MYC, ZEB1, and NFKB1 were revealed for the lung cancer-associated genes. These findings offered crucial insights into the non-coding regulation for the etiology of lung cancer, providing additional potential targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218dccec773052bdb16c39b8293ccbf37c8ec89f" target='_blank'>
              Non-coding genetic elements of lung cancer identified using whole genome sequencing in 13,722 Chinese
              </a>
            </td>
          <td>
            Dan Zhou, Ming Wu, Qilong Tan, Liyang Sun, Yuanxing Tu, Weifang Zheng, Yun Zhu, Min Yang, Kejia Hu, Fang Hu, Xiaohang Xu, Hanyi Zhou, Tian Luo, Fangming Yang, Fuqiang Li, Xin Jin, Huakang Tu, Wenyuan Li, Kui Wu, Xifeng Wu
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Insertion sequence (IS) elements are key drivers of bacterial genome plasticity, yet the overall regulation of their transposition remains poorly understood. This is especially true for the multiple-layer regulation at the donor site, which has been largely overlooked. Using multiple mutation assays, genetic manipulations and reporter genes, this study focuses on characterizing how endogenous DNA sequences, transcriptional and translational factors, and genomic context regulate IS1 transposition from its donor site. Out of six elements within the chromosome of E. coli strain BW25113, IS1A and IS1E (both with the consensus sequence) contribute to over 99.9% of the overall IS1 transposition within the genome while the other four elements without the non-consensus sequence are essentially incapable of transposing. Inducing a ribosomal -1 frameshift at the A6C motif increases transposition over 1000-fold, but this enhancement is largely reversed by restoring InsA-mediated transcriptional regulation. Strikingly, genomic sequences flanking IS1 elements appreciably modulate transposition by promoting transcription or facilitating formation of transpososomes, a phenomenon that remains under-studied. Finally, IS1 was confirmed to undergo replicative transposition intramolecularly, a mechanism shown here to be independent of transposase levels in the cell. These findings contribute to our understanding of mobile genetic element regulation and potentially offer strategies for mitigating their potentially harmful effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed2e184755f4f8d507b9afcfaae112ea127ba2c1" target='_blank'>
              Examining the Roles of Genomic Context and Endogenous Regulatory Elements on IS1 Transposition Within the Escherichia coli Genome
              </a>
            </td>
          <td>
            Sofia Smith, Zhongge Zhang, Allyson Ho, Tusha Karnani, Jack W Ord, Milton H. Saier
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The GATA3 noncoding variant rs3824662 has been implicated in the pathogenesis of Ph-like B-ALL, where it is associated with extensive chromatin reorganisation, resulting in the dysregulation of multiple genes, including CRLF2 overexpression. Given the altered chromatin landscape and increased accessibility of GATA3 binding regions associated with the rs3824662 variant, we investigated the potential role of enhancer RNAs (eRNAs) located near the GATA3 locus in regulating CRLF2 expression. We found that the expression of eRNA_G3, located at chr10:8,443,562–8,449,563, was positively correlated with CRLF2 expression. Notably, eRNA_G3 was significantly upregulated in Ph-like ALL cases carrying the GATA3 rs3824662 variant. These findings suggest that both rs3824662 and eRNAs may cooperatively contribute to the regulatory mechanisms governing GATA3 and CRLF2 expression in Ph-like ALL. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10262-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a6bfc4308f5a1eb04611b56714ff4964a37c1d8" target='_blank'>
              Enhancer RNAs contribute to genome reprogramming driven by a GATA3 noncoding variant in leukaemia
              </a>
            </td>
          <td>
            Thayana da Conceição Barbosa, C. P. Poubel, A.L.T. Maciel, C. B. Blunck, Rafael Lima Dos Santos, M. Boroni, M. B. Mansur, M. Emerenciano
          </td>
          <td>2025-08-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The most common cause of death in pediatric cancer patients is a treatment-resistant tumor, compounded by metastatic spread, making surgery, radiation therapy, and chemotherapy unfeasible as curative treatment options. However, the mechanisms behind metastatic spread in pediatric tumors remain largely unexplored. We conducted whole-genome copy number profiling on 171 primary tumor and metastases samples from 17 patients with neuroblastoma, Wilms tumor, or gonadal tumors, and performed targeted deep sequencing on a subset. Phylogenetic reconstruction enabled spatiotemporal tracking of subclones. In total, 11 of 17 patients displayed at least one metastasis arising earlier, defined as occurring before the most recent common ancestor in the primary tumor. In eight patients, metastatic spread was observed several times during tumor evolution, with different subclones from the same primary tumor having metastatic capability, even colonizing the same site. Strikingly, dissemination between metastases (intermetastatic spread) occurred in eight of nine patients with metastases in at least two different sites, indicating that this is a common phenomenon in pediatric malignancy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24f7bd63ad2744ced5ab2ba8c285ffb7719db78" target='_blank'>
              Tracing metastatic spread in pediatric solid tumors using copy number and targeted deep sequencing.
              </a>
            </td>
          <td>
            Natalie Andersson, Michele Ferro, Caroline Jansson, Subhayan Chattopadhyay, Jenny Karlsson, David Gisselsson
          </td>
          <td>2025-09-23</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In cellular genomes, CCCTC-binding factor (CTCF) insulators impact transcription over small distances in a one-dimensional manner and over much longer distances in a three-dimensional manner by maintaining chromatin loops. We have previously shown that the latent HSV-1 genome contains CTCF insulators that function to regulate lytic transcription of adjacent genes in a one-dimensional manner. Here, we test the hypothesis that HSV-1 CTCF insulators nucleate chromatin loops to regulate the expression of distance-separated gene regions through three-dimensional organization of viral genomes. We used 4C-seq methods to identify multiple long-range cis interactions in HSV-1 genomes that generate viral chromatin domains, including those nucleated by the viral CTCF insulator CTRL2. Deletion of the CTRL2 insulator disrupted these viral chromatin domains. Loop-nucleating interactions were quantitated with a novel approach (UMI-4C-seq) that utilizes unique molecular identifiers to label and count chromatin interactions associated with specific viewpoint primers. Cis-interaction peaks across four different viewpoints were quantified. Viral genomes lacking CTRL2 displayed more cis-interaction peaks and wider ranges of interaction lengths compared to wt virus, suggesting altered chromatin organization. Furthermore, differential looping analysis showed that viral genomes lacking CTRL2 displayed a more transcriptionally permissive chromatin environment. Thus, the CTRL2 insulator functions as a critical regulator of long-range chromatin interactions, and its deletion reshapes the viral chromatin landscape, leading to a more accessible and dynamic regulatory environment that may influence HSV-1 transcriptional programs and latency-associated chromatin states.IMPORTANCEHSV-1 is a significant lifelong human pathogen that infects 70% of adults worldwide. The latent HSV-1 genome is chromatinized and maintained in distinct chromatin structures that silence the virus, while reactivation is facilitated by transient reversal of host factors that maintain those chromatin domains. Understanding how this happens is critical for the development of novel therapeutics. It is becoming clear that CTCF insulators play a key role in the reversal that leads to reactivation. CTCF insulators are essential regulators of chromatin structure and gene expression in mammalian cells and play vital regulatory roles in transcriptional control of DNA viruses by organizing chromatin architecture during both latent and lytic stages of virus lifecycles. Here, we present the first report that latent HSV-1 genomes are organized into 3D structures to support latency yet allow the viral genome to reactivate, opening the door for future therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a535267a81e37204f0c265ca04fb430e6f4497" target='_blank'>
              Multiple long-range cis interactions generate CTCF insulator-dependent viral chromatin domains in quiescent HSV-1 genomes.
              </a>
            </td>
          <td>
            Alyssa Richman, Sophie Kogut, Terri G. Edwards, Joseph R. Boyd, Princess D. Rodriguez, Michael Mariani, Mason A. Shipley, Kayley A Manuel, Ziyun A Ye, David C. Bloom, Seth Frietze, Donna M Neumann
          </td>
          <td>2025-08-28</td>
          <td>mBio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="mRNA decapping enzyme 2 (DCP2) is a key regulator of mRNA degradation, influencing RNA metabolism and gene expression. While implicated in various diseases, its role in Hepatocellular carcinoma (HCC) remains unclear. This study explores DCP2 as a potential biomarker for HCC through bioinformatics and experimental analyses. Public databases, including The Cancer Genome Atlas (TCGA) and GEPIA2 online database, revealed significant DCP2 upregulation in HCC tissues, correlating with Tumor-Node-Metastasis (TNM) staging, histological grading, platelet-to-lymphocyte ratio (PLR), neutrophil-to-lymphocyte ratio (NLR), and alpha-fetoprotein (AFP) levels. Prognostic analysis confirmed high DCP2 expression as an indicator of poor overall survival (OS). Functional enrichment analyses linked DCP2 to cancer-associated pathways, including PI3K/Akt, Hippo signaling, cholesterol metabolism, and oxidative phosphorylation. Immune infiltration analysis showed significant correlations with tumor-associated macrophages (TAMs) and regulatory T cells (Tregs), suggesting a role in immune modulation. Experimental validation via western blot and immunohistochemistry (IHC) confirmed elevated DCP2 expression in HCC cell lines and tissues. These findings suggest that DCP2 may be involved in HCC progression by influencing mRNA degradation, cancer-related signaling, and the immune microenvironment. Although DCP2 shows potential as a diagnostic and prognostic biomarker, as well as a therapeutic target, further functional studies—such as gene knockdown and overexpression experiments—are needed to confirm its mechanistic role in the development of hepatocellular carcinoma (HCC). Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10308-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea65acb3c02d509d0fb354d65ba55dd81bb1cdf9" target='_blank'>
              mRNA decapping enzyme2 DCP2 expression correlates with progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Tianyi Huang, Rui Wang, Cuihua Lu, Jiaying Zhang
          </td>
          <td>2025-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gene regulation often entails a cooperative dynamic interplay among several protein molecules and several distinct DNA segments. Intersegment transfer of the Fis protein stimulates DNA inversion during DNA recombination. Individual DNA segments have been found to facilitate the dissociation of Fis proteins already bound to DNA and also allow for the transfer of the Fis between segments. Here, we use the hybrid coarse-grained AWSEM/3SPN.2C model to simulate the Fis protein intersegment transfer and explore its mechanism. We show that entropic effects within the Fis protein-DNA complex dictate the transfer pathway through specific structural configurations, involving specific grooves and consequent orientation constraints on the DNA segments. Multiple copies of the Fis protein facilitate intersegment transfer, explaining how changes in protein-DNA stoichiometry and concentration influence how the Fis-DNA complex architecture is established. This orientational dependence indicates that the assembly of the Fis-DNA complex mimics the interlocking of screws, functioning as a molecular machine that may couple to DNA supercoiling and torsional stress in the DNA generated by motor proteins, thus offering a potential regulatory mechanism for chromosomal organization and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a31cc6e4951420f062570d943b327e0539b5ca" target='_blank'>
              Intersegment Transfer and the Dynamical Architecture of Fis Protein-DNA Multimer Complexes.
              </a>
            </td>
          <td>
            Xun Chen, Shikai Jin, Cheng-Han Liu, Yaakov Levy, Min-Yeh Tsai, Peter G. Wolynes
          </td>
          <td>2025-08-05</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Diversification and selective propagation are the main driving forces of evolution, resulting in the emergence of organisms possessing various strategies. Here, we conducted in vitro evolution of the replication origin (oriC) under the pressure of an AT-rich mini-chromosome amplification using a reconstituted Escherichia coli replication-cycle reaction (RCR). Using next-generation sequencing, we identified that these evolved oriCs contain select mutations within the duplex unwinding element (DUE) and DnaA-binding site (DnaA box) regions, which are crucial for replication initiation. Real-time detection of RCR amplification (real-time RCR) revealed that the DUE mutations, which decreased the GC content, along with the introduction of a specific A/T sequence in DUE-M and a consecutive KAK (K = T or G) motif in DUE-R, enhanced the RCR amplification efficiency compared to the wild-type oriC (oriCwt). A competitive amplification assay also elucidated that the DnaA box mutations confer a competitive advantage over coexistent oriCwt. Although these DUE and DnaA box mutations were selected through the same amplification reaction, they exhibited distinct competitive amplification strategies. The DUE mutant represents a faster propagation strategy (r-strategy), while the DnaA box mutant represents an adaptive strategy with a competitive advantage (K-strategy), representing the r/K-selection theory in molecular evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d8adc8e642b9b4fca6ba9da0b2f823fd1341a1" target='_blank'>
              Adaptive evolution of oriC through in vitro propagation of a mini-chromosome in RCR
              </a>
            </td>
          <td>
            Shota Suzuki, M. Su’etsugu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary BRG1 protein expression or its normal functions are lost in many cancers, including ~10% of lung cancer cases, due to mutation. However, how these losses contribute to the development of lung cancer is not clearly known. This study used human lung epithelial cell-derived normal, pre-cancer, and cancer cell lines to investigate the initiation and progression of lung cancer. Our results show that BRG1 loss induces epigenetic (non-genetic) and expression changes in thousands of genes genome-wide that lead to the early transformation of normal cells into pre-cancer cells. In established cancer cells, BRG1 loss and its downstream epigenetic and gene expression changes make the cancer cells become sensitive to epigenetic drugs. Taken together, these results indicate that BRG1 loss contributes to early lung cancer development and could be exploited to improve the treatment of this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23222db0933f5999fd32623b0319bdf7442135af" target='_blank'>
              BRG1 Loss Is Frequent in Lung Cancer and Transforms Lung Epithelial Cells via Transcriptional and Epigenetic Reprograming
              </a>
            </td>
          <td>
            M. Tessema, Christin M. Yingling, Loryn M. Phillips, K. Do, M. Picchi, R. Willink, Steven A. Belinsky
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Most of the human genome is transcribed into noncoding RNAs (ncRNAs), which although do not encode proteins, are often involved in key cellular processes. Recent technological advances have accelerated the discovery of ncRNA as well as our understanding of ncRNAs as cancer biomarkers and targets. In this “In the Limelight” special issue of FEBS Open Bio, four review articles by leading experts provide an overview of the role of ncRNAs in cancer, from their association to liver cancer to their role as oncogenes in head and neck squamous cell carcinoma (HNSCC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d403967dd7059c9aa239237ba085b5894f7584" target='_blank'>
              Non‐coding RNAs as key player in cancer diagnosis and treatment
              </a>
            </td>
          <td>
            Marcin Majka
          </td>
          <td>2025-09-01</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Histones are the fundamental building blocks of chromatin and serve as pivotal regulators of gene expression. Differential expression and mutations of H3.1 and H3.3 genes have been implicated in the pathogenesis of various cancer types. Mutations in H3.3, especially lysine to methionine substitutions (K27M/K36M), are particularly prevalent. Moreover, genetic alterations such as G34R/W/V/L, as well as variations in H3F3A and H3F3B genes, have also been identified. Despite high similarity in amino acid sequences, H3.1 and H3.3 have discrete functions in cancer. In this review, we delve into the recent advances in elucidating the implications of canonical histone H3.1 and its variant H3.3 on chromatin structure and function. Additionally, we explore how potential enhancing factors such as PTEN, MLL5, GPR87 and histone chaperones influence H3.1/H3.3 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f17578aa817134a59d69c6b812cf15a821be1415" target='_blank'>
              Implications of canonical histone H3.1 and histone variant H3.3 in cancer
              </a>
            </td>
          <td>
            Peng Wu, Li Wang, Ting Wen, Qiao Yi Chen
          </td>
          <td>2025-09-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>24</td>
        </tr>

        <tr id="

 Prostate cancer is known to be caused by the androgen receptor (AR) axis and testosterone, emerging evidence highlights the significance of epigenetic modifications in mediating androgen-dependent tumorigenesis and therapy resistance.



 The literature search was conducted in Scopus, PubMed, and Google Scholar following PRISMA guidelines. Case-control, clinical and preclinical studies, and human clinical trials analyzed epigenetic modifications in testosterone-mediated prostate cancer were included. Studies that did not analyze epigenetic alterations in prostate cancer or focused on other cancer types were excluded.



 Prostate cancer is largely caused by DNA methylation, which controls gene expression. Genes are silenced when CpG islands are hypermethylated, and genomic instability is exacerbated by hypomethylation. Emerging therapeutic strategies, such as DNMT inhibitors and combination therapies with AR pathway inhibitors, are being explored to target aberrant DNA methylation in prostate cancer treatment. Prostate cancer development and androgen receptor signaling are influenced by chromatin remodeling and gene regulation, both of which are significantly impacted by histone changes.



 Epigenetic dysregulation drives prostate cancer progression by silencing tumor suppressors, activating oncogenes, and promoting therapy resistance. These changes have potential as therapeutic targets for better patient clinical care.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96426730ce63a2add9c14a2ff4ad289cf52f8109" target='_blank'>
              Epigenetic modifications and gene expression regulation in testosterone-mediated prostate cancer: A systematic review
              </a>
            </td>
          <td>
            V. K. Singh, Jigar Haria, Pothu Usha Kiran, Prithpal Singh Matreja
          </td>
          <td>2025-08-26</td>
          <td>Journal of Cellular Biotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent researches have revealed the potential utility of extrachromosomal circular DNAs (eccDNAs) as biomarkers in various diseases. However, the association between plasma eccDNAs and myocardial infarction (MI) remains unclear. In this study, we extracted plasma eccDNA from blood samples of individuals with acute MI and conducted Circle-Seq. We identified an MI-related eccDNA (MIRECD) with high expression levels in the plasma of patients with MI. Sanger sequencing validated its loop construction and sequence. Mechanistically, MIRECD could aggravate necroptosis via regulating the expression of mixed-lineage-kinase-domain-like pseudokinase (MLKL). Kaplan–Meier analysis demonstrated that the incidences of major adverse cardiac events (MACEs) and cardiovascular mortality were higher in individuals with elevated MIRECD levels. Univariate and multivariate Cox regression analyses indicated that MIRECD independently predicted the occurrence of MACEs in patients with MI. The addition of MIRECD enhanced the discrimination and reclassification ability compared with conventional risk factors. In conclusion, our study identified a novel eccDNA, MIRECD, which might regulate myocardial necroptosis through MLKL. MIRECD has the potential to serve as a reliable indicator for predicting the prognosis and stratifying the risk of MI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a8ed2aed986868beea935ead77f232c45fe1fb" target='_blank'>
              Extrachromosomal Circular DNA MIRECD Enhances Necroptosis and Predicts Prognosis of Myocardial Infarction
              </a>
            </td>
          <td>
            Yiheng Zhao, Yujia Zhou, Shuchen Zhang, Boyang Xiang, Xiang Zhou
          </td>
          <td>2025-08-08</td>
          <td>Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Variations in DNA methylation within the DNA damage response (DDR) mechanism could have significant implications for glioma prognosis and immune responses. This study aimed to explore the global DNA methylation landscape of DDR genes in gliomas and identify key epigenetically regulated genes influencing glioma biology and immunity. Methods This study incorporated a range of public and local glioma datasets. Multiple clinical, bioinformatic, and in vitro experimental analyses were conducted to explore clinical and biological aspects. Results Global DNA methylation variations in DDR genes correlated with distinct glioma prognoses, with five CpGs identified as potent predictors. Hierarchical clustering and a risk-score model based on these CpGs unveiled immune-related prognostic subgroups in glioblastomas (GBMs) and lower-grade gliomas (LGGs). NSUN5, epigenetically regulated by one of these CpGs, highlighted the biological significance of the DDR CpG panel. In vitro, NSUN5 displayed tumor-suppressor-like activities in GBM cells, but clinically, it was an unfavorable prognostic marker. Depletion of NSUN5 shifted cytosolic DNA sensing from a STING-dependent (cGAS-STING) pathway to a STING-independent (DNA-PK-HSPA8) pathway, leading to a delayed but more robust type I interferon (IFN) response in GBM cells and enhancing microglial M1 polarization and chemotaxis. This may partially account for the functional discrepancy of NSUN5 observed between experimental and clinical contexts. Conclusion This study highlights the complex interplay between DNA methylation, the DDR mechanism, cytosolic DNA sensing, and glioma immunity. These findings may inspire novel strategies for DNA sensing-based immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06360-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54757ac8e99a0979bc781abac88c0c6433dc38" target='_blank'>
              DNA methylation variations of DNA damage response in glioblastoma: NSUN5 modulates tumor-intrinsic cytosolic DNA-sensing and microglial behavior
              </a>
            </td>
          <td>
            An-An Yin, Yan-Qiu Yao, Yi-Feng Liu, Yu-Sha Ji, Yalong He, Tian-Chi Ma, Wen-Heng Guo, A. Etcheverry, M. Aubry, Jean Mosser, Wei Lin, Yuan-Ming Wu, Kun Chen
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are remnants of retroviral infections that have become stably integrated into host germline genomes. Far beyond passive genomic elements, ERVs actively shape host evolution through complex mechanisms involving genetic innovation, immune modulation, and species adaptation. This review provides a comprehensive synthesis of ERV biology, highlighting recent advances in their classification, amplification mechanisms, and epigenetic silencing. Particular emphasis is placed on the cross-talk between ERVs and exogenous retroviruses (XRVs), demonstrating how receptor competition, recombination, and immune evasion contribute to virus-host co-evolution. We explore ERVs as molecular markers for phylogenetic reconstruction, with case studies such as Koala retrovirus (KoRV) and HERV-K illustrating regional transmission dynamics and co-opted immune functions. Additionally, we discuss the functional domestication of ERVs into regulatory elements, non-coding RNAs, and envelope-derived fusion proteins that influence gene expression, antiviral defense, and placental development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7f347985d489007229fb4a5b8a3cd06f14b32" target='_blank'>
              Endogenous Retroviruses in Host-Virus Coevolution: From Genomic Domestication to Functional Innovation
              </a>
            </td>
          <td>
            Ruqi Jiang, Jingjun Zhou, Yue Liu, Guanjin Zhou, Dongdong Fan, Lixin Xiang, Ye Chen, Jianzhong Shao
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA topoisomerase IIα (TOP2A) plays a key role in resolving DNA topological stress during replication, transcription, and chromosome segregation. Yatein, a biogenic precursor of deoxypodophyllotoxin possesses antitumor effects. Herein, we explored its antitumor effect in breast cancer. Yatein significantly inhibited the proliferation of breast cancer cells by MTT, colony formation, EdU assay, and cell cycle analysis. Moreover, yatein suppressed the stemness of cancer cells, causing inhibition of migration and epithelial-mesenchymal transition. Mechanism study revealed that yatein inhibited TOP2A activity and stabilized TOP2A-DNA cleavage complexes, caused DNA damage and apoptosis, suggesting its potential in the treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d219f76fc877a179547f374e85d00ebe2c5225" target='_blank'>
              Yatein-induced DNA damage of breast cancer cells by interfering with DNA topoisomerase IIα.
              </a>
            </td>
          <td>
            Yu-Yao Liu, Ya Chen, Qi-Qi Wang, Xi Zhang, Xiao-Xia Ma, Xing-De Wu, Jia Su
          </td>
          <td>2025-08-25</td>
          <td>Journal of Asian natural products research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-associated fibroblasts (CAFs) regulate tumorigenesis, tumor cell proliferation, and metastasis via secreting related regulatory factors; however, the evidence for CAFs in regulating development of upper tract urothelial carcinoma (UTUC) remains unclear. Here, by utilizing single-cell RNA sequencing (scRNA-seq), single-nucleus RNA sequencing (snRNA-seq), SpaTial enhanced resolution omics-sequencing (Stereo-seq), and UTUC immunofluorescence chip cohort to construct the first comprehensive microenvironmental atlas of CAFs, we investigated the roles of CAFs in UTUC progression. Through hierarchical clustering and the copy number variation (CNV) scores of UTUC epithelial cells, we first classified tumor epithelial cells into high-malignant, medium-malignant, and low-malignant potential categories. We found that the myofibroblastic CAFs1 (myCAFs1) and myCAFs2 subclusters exhibited significant interaction signals with all three types of epithelial cells, among which high-malignant epithelial cells (HMECs) exhibited pronounced communication signals with CAFs, and FN1 and COL1A1 generated by CAFs played critical roles in this process, suggesting that the progression of UTUC may be attributed to the activation of tumor cells by CAFs. Both myCAFs1 and myCAFs2 simultaneously affect bladder urothelial carcinoma (BUC) prognosis, with the risk model showing good consistency across cohorts. The study constructs a multi-omics landscape of UTUC and identify common prognostic markers shared with BUC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281dda7d54e63559872a9e12f7b32f74d4425ae4" target='_blank'>
              The tumor-associated fibroblasts regulate urothelial carcinoma progression.
              </a>
            </td>
          <td>
            Yu Xiao, Junfeng Yang, Mengjie Sun, Yongfu Li, Qinyin Liu, Jinjun Leng, Maolin Yang, Jinrui Wang, Hongju Li, Caifeng Yang, Changfen Luo, Jiahong Li, Longli Kang, Fen Huang, Yanhong Yu, Chuanmao Zhang
          </td>
          <td>2025-09-19</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaa6c9f0d6e6a493ccdb8dd3165ee404adbcef4e" target='_blank'>
              Nuclear ribonucleoprotein condensates as platforms for gene expression regulation.
              </a>
            </td>
          <td>
            Sunkyung Choi, Kee K. Kim
          </td>
          <td>2025-08-04</td>
          <td>Genes & genomics</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Integrating conjugative elements (ICEs) are mobile genetic elements conferring a wide range of beneficial functions upon their bacterial hosts. Generally, they can be activated from their integrated states to undergo horizontal gene transfer via conjugation. In the case of the human gastric pathogen Helicobacter pylori, a paradigm for extensive genetic diversity, highly efficient natural transformation and recombination processes may superimpose canonical transfer of its two ICEs termed ICEHptfs3 and ICEHptfs4, and thus shape their composition substantially. Here, as a part of the Helicobacter pylori Genome Project (HpGP) initiative, we have analyzed high-quality genome sequences from 1011 clinical strains with respect to their ICE content and variability. We show that both elements are highly prevalent in all H. pylori populations, but have a strong tendency for gene erosion. ICE sequence variations reflect the population structure and show a clear signature of increased horizontal transfer. A detailed map of ICE integration sites revealed local preferences, but also how recombination processes result in hybrid elements or genome rearrangements. Population-specific differences in ICE cargo genes might reflect distinct requirements in the biological functions provided by these mobile elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573f2ef5afd055f208fa4bb18b449324318e3244" target='_blank'>
              Global diversity of integrating conjugative elements (ICEs) in Helicobacter pylori and their influence on genome architecture
              </a>
            </td>
          <td>
            A. Gutiérrez-Escobar, Mrinalini Srinivasan, Z. Muñoz-Ramirez, Filipa F. Vale, Difei Wang, Santiago Sandoval-Motta, J. P. Dekker, Kaisa Thorell, M. C. Camargo, Yoshio Yamaoka, Wolfgang Fischer
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aadf2778bea4e8d563036eda473ff737cd085e3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase.
              </a>
            </td>
          <td>
            James E Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T Tran, Clara Errard, Brian E Hew, Brandon E Johnson, David F Waller, Shanfu Xie, Joseph J Higgins, Raymond Tabibiazar, Jesse B Owens
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for DNA supercoil removal, the aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from excessive TOP1cc accumulation remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), the histone variant macroH2A1.1 facilitates TOP1cc repair factor recruitment and lesion turnover, thereby preventing DNA damage in response to transcription-associated topological stress. The alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. We propose macroH2A1 alternative splicing as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdec4fbd7a5287d669fa0b66871ebd9fb8a9c98" target='_blank'>
              Epigenetic control of topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Marina Farkas, Zhiyan Zhao, Vijayalalitha Ramanarayanan, Tongyu Wu, Tianyi Guan, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2025-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="ABSTRACT In addition to the main chromosome, approximately 10% bacterial genomes have one or more large secondary replicons, including a unique type of replicon known as “chromid,” which has plasmid-type replication and partition systems but carries core genes. Their nucleotide composition of chromids is very similar to that of their related chromosomes. However, the distribution, characteristics, functions, and origins of these chromids remain unclear. In this study, we established a workflow to identify chromids from complete bacterial genomes and screened 36,322 complete bacterial genomes, resulting in the identification of 1,104 bacterial genomes with chromids as secondary replicons. These chromid-carrying bacteria belong to eight phyla and 73 genera, exhibiting diversity and a wide global distribution. We analyzed the characteristics of chromids and found that their average size is larger than that of “megaplasmids” and that multi-chromid bacteria exist. Furthermore, chromids encode genes related to bacterial respiratory chain enzyme complexes and antiviral systems, expanding bacterial metabolic capabilities and enhancing their antiviral defenses. In addition, we developed an automated identification program, Chromid-Finder, for identifying chromid sequences in metagenomic data, which has demonstrated outstanding performance. To demonstrate its application, we analyzed 92,143 metagenome-assembled genomes (MAGs) from the human gut microbiome. We found that the distribution of chromid-carrying bacteria in the human gut is closely associated with host age, health status, and geographic location. Species with chromids exhibit unique functional capabilities, showing good separation at the phylum level. IMPORTANCE In this study, we have developed a workflow to identify chromids from complete bacterial genomes. We utilized this workflow to search for chromids in the latest NCBI RefSeq databases, to map the distribution of bacteria carrying chromids, to identify the characteristics of bacterial chromids, to discuss their origins, and to investigate their roles in bacterial life. To address the growing volume of metagenomic data, we developed a high-performance automated identification program, Chromid-Finder, designed to identify chromids and their corresponding bacterial main chromosomes within extensive metagenomic data sets. Using this tool, we analyzed 92,143 metagenome-assembled genomes (MAGs) from the human gut microbiome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1858682f6d07c026bc78fe000bacfa219ec68e47" target='_blank'>
              Unexplored diversity and potential functions of extra-chromosomal elements
              </a>
            </td>
          <td>
            Haoyu Liu, Jia Sun, JuanJuan Si, Yi Liao, Jiaqi Bai, Xia Li, Limin Wang, Kuojun Cai, Wei Ni, Ping Zhou, Shengwei Hu
          </td>
          <td>2025-08-19</td>
          <td>mSystems</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H.A. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human papillomavirus (HPV) plays a major role in the development of head and neck cancers (HNCs), particularly oropharyngeal squamous cell carcinoma. This review highlights the key molecular mechanisms of HPV-driven carcinogenesis, focusing on the oncogenic E6 and E7 proteins and their disruption of tumor suppressor pathways and epigenetic regulation. We discuss the rising prevalence of HPV-related HNCs, their distinct clinical features, and diagnostic approaches such as p16 immunohistochemistry and HPV DNA/RNA detection. HPV-positive tumors show better prognosis and response to treatment, prompting interest in therapy de-escalation. Emerging strategies including immune checkpoint inhibitors, therapeutic vaccines, CRISPR-based gene editing, and ctDNA monitoring are advancing precision oncology in this field. We also examine the preventive potential of HPV vaccination and ongoing research into its role across various HNC subtypes. A deeper understanding of HPV’s molecular impact may guide more effective, targeted, and less toxic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a80070351d0160df6a7102f80e58aa08b5c7fd1a" target='_blank'>
              Molecular Insights into HPV-Driven Head and Neck Cancers: From Viral Oncoproteins to Precision Therapeutics
              </a>
            </td>
          <td>
            Mustafa Ozdogan, Gizem Tutkun, M. O. Cakir, G. H. Ashrafi
          </td>
          <td>2025-09-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transposable elements (TEs), comprising over one-third of the human genome, play a crucial role in its evolution, serving as a significant source of regulatory sequences. Under normal circumstances, their activity is tightly controlled by DNA methylation mechanisms; however, the effectiveness of this suppression varies substantially across tissues. The placenta, characterized by global hypomethylation, represents a unique environment where retroviruses and retrotransposons, typically silenced in somatic cells, gain the opportunity for activation. This distinct epigenetic landscape of the placenta allows transposons to participate in the regulation of genomic activity, influencing processes ranging from early embryogenesis to postnatal development. DNA hypomethylation in the placenta not only promotes TE mobilization, but also opens the possibility of using their components as independent genes and regulatory elements – promoters, enhancers, and other functional modules. These elements are involved in key aspects of placental development, including syncytiotrophoblast formation, extravillous trophoblast invasion, spiral artery remodeling, and endometrial decidualization. Importantly, TEs can serve as sources of alternative promoters for neighboring genes, and ancient mammalian transposons contain multiple transcription factor binding sites, enabling coordinated regulation of genes sharing a common function. Despite the growing interest in the role of transposable elements in placental development and function, many questions remain unanswered. In particular, the mechanisms of non-long terminal repeat (non-LTR) retrotransposon function during pregnancy remain poorly understood. A deep understanding of these processes is necessary to elucidate regulatory disorders in the placenta associated with major obstetric syndromes. This review examines the contribution of transposable elements to the functioning of the human genome, particularly their impact on gene expression, in the context of pregnancy and placental development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe4a3e7ac959ca8083b328f8c6eeb28a76f946c" target='_blank'>
              Transposable elements as key regulators of placental development
              </a>
            </td>
          <td>
            M. A. Zhilkina, E. Tolmacheva, S. Vasilyev
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Genes encoding the RNA-binding proteins FUS, EWSR1, and TAF15 (FET proteins) are involved in chromosomal translocations in rare sarcomas. The encoded fusion oncoproteins typically include a DNA-binding domain and function as oncogenic transcription factors. FET-rearranged sarcomas are often aggressive malignancies affecting patients of all ages. Oncogenic fusion transcription factors are challenging to target directly, and new therapies are needed. Treatment with the small molecule SP-2509 results in reversal of the transcriptional activity of the FET fusion that causes Ewing sarcoma, EWSR1::FLI1. A similar molecule, seclidemstat (SP-2577), is currently in clinical trials for FET-rearranged sarcomas (NCT03600649), but its pharmacologic activity in FET fusion–positive sarcomas has not been demonstrated. We found potent seclidemstat cytotoxicity against both FET-rearranged and other fusion-positive sarcoma cell lines in vitro, including Ewing sarcoma, desmoplastic small round cell tumor, clear cell sarcoma, myxoid liposarcoma, and fusion-positive rhabdomyosarcoma. We also define transcriptomic effects of seclidemstat with bulk RNA sequencing. Seclidemstat recapitulated much of SP-2509 transcriptional activity in Ewing sarcoma. Widespread transcriptional changes were seen in all tested cell lines after seclidemstat treatment, regardless of the fusion protein expressed. This included reversal of FET fusion transcriptional signatures for EWSR1::WT1, EWSR1::ATF1, and EWSR1::ERG. Though novel inhibitors are unlikely to display single-agent efficacy in the clinic, these data suggest that seclidemstat remains a promising new treatment strategy for patients with FET-rearranged and other fusion-positive sarcomas. Significance: In this study, we show seclidemstat has in vitro activity in multiple rare and aggressive sarcomas caused by FET fusion proteins. With 13 RNA sequencing experiments, including multiple FET-rearranged sarcoma cell lines, this dataset is a rich resource for those studying FET-rearranged sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb78384576a9a5b2487874950835b687b754a22" target='_blank'>
              Seclidemstat (SP-2577) Induces Transcriptomic Reprogramming and Cytotoxicity in Multiple Fusion–Positive Sarcomas
              </a>
            </td>
          <td>
            G. Rask, C. Taslim, A. Bayanjargal, Rachel D. Dreher, Matthew V. Cannon, Julia Selich-Anderson, Jesse C. Crow, Aundrietta Duncan, Emily R. Theisen
          </td>
          <td>2025-08-25</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. H. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, Marwa A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The genomes of 43 distinct lactococcal strains were reconstructed by a combination of long- and short-read sequencing, resolving the plasmid complement and methylome of these strains. The genomes comprised 43 chromosomes of approximately 2.5 Mb each and 269 plasmids ranging from 2 to 211 kb (at an average occurrence of 6 per strain). A total of 953 antiphage genes representing 538 phage defence systems were identified in the 43 strains and were catalogued and cross-correlated with co-occurrent mobile elements, which indicated that almost 60% of these systems are predicted to be mobile. Detailed analysis established that restriction-modification (R-M) systems form a significant portion of this mobile phage defensome. As such, all detected Type I, II, and III-associated methylated motifs (46 of which were unique to this study) were matched to their corresponding methylating enzymes by homology detection or molecular cloning. The cumulative antiphage activity of selected systems and the ability of truncated R-M genes to contribute to methylation were demonstrated. This study reveals, for the first time, the dairy lactococcal plasmidome to be a rich reservoir of orphan HsdS-encoding genes, in a comprehensive survey of (mobile) phage defence systems in lactic acid bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6afd146a6e3d233d4624dbd25f3bf7a0d3905b" target='_blank'>
              Lactococcal mobile genetic elements harbour a diverse phage defensome rich in restriction-modification systems
              </a>
            </td>
          <td>
            Brian McDonnell, P. Kelleher, Alexey Fomenkov, Guillermo Ortiz Charneco, Keith Coughlan, Pascal Quénée, S. Kulakauskas, Christian Cambillau, Brian P. Anton, Paul P. de Waal, N. V. van Peij, Francesca Bottacini, J. Mahony, R. J. Roberts, D. van Sinderen
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objective: Prostate cancer, a leading cause of cancer deaths in men, involves genetic alterations and DNA damage. Oxidative stress–induced DNA damage is crucial in its development, necessitating efficient DNA repair mechanisms. Base excision repair (BER), a key pathway, addresses oxidative DNA damage with enzymes like Apurinic/Apyrimidinic Endonuclease 1 (APE1). Recent findings show that altered APE1 expression disrupts BER balance, contributing to genomic instability and cancer progression. This study examines APE1 expression in prostate cancer tissues to clarify its role in cancer development and therapeutic implications. Assessing APE1 expression in prostate cancer may elucidate its role in tumor progression, therapeutic resistance, and genomic instability, offering deeper insights into the mechanistic relevance of the BER pathway in oncogenesis. Methods: A total of 50 male patients diagnosed with prostate cancer were included in this study. Tumor and adjacent normal tissue samples were collected from consecutive male patients who underwent Tru-cut prostate biopsies due to clinical suspicion of prostate cancer, and gene expression levels were analyzed using real-time polymerase chain reaction. Results: The results showed no significant difference in APE1 expression between tumor and normal tissues (P > .05). Clinical factors like smoking, age, diabetes, hypertension, PIRADS, and Gleason scores did not significantly correlate with APE1 expression. However, a statistically significant correlation was observed between PSA levels and APE1 expression (r: 0.287, P = .04). Conclusion: The authors’ findings suggest that while no substantial changes in APE1 expression were observed in prostate cancer tissues, the enzyme’s role in DNA repair remains crucial. Abnormalities in APE1 expression and function can contribute to genomic instability and cancer progression. Future studies with larger sample sizes and standardized methodologies are necessary to better understand the potential role of APE1 as a biomarker and therapeutic target in prostate cancer.Cite this article as: Rouhi V, Habibi S, Baykal Koca S, Köseoğlu H, Güven M. Investigation of apurinic/apyrimidinic endonuclease 1 expression changes in prostate cancer. Cerrahpaşa Med J 2025, 49, 0009, doi: 10.5152/cjm.2025.25009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a4b0fb3242b36fc3de3889e7200ec9f3f9fad" target='_blank'>
              Investigation of Apurinic/Apyrimidinic Endonuclease 1 Expression Changes in Prostate Cancer
              </a>
            </td>
          <td>
            Vahid Rouhi, Sanaz Habibi, Sevim Baykal Koca, Hikmet Köseoğlu, Mehmet Güven
          </td>
          <td>2025-08-29</td>
          <td>Cerrahpaşa Medical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human oncogenic viruses contribute significantly to the global health burden and include seven types; Epstein-Barr virus, hepatitis B virus, human T-cell leukemia virus type 1, human papillomavirus, hepatitis C virus, Kaposi's sarcoma-associated herpesvirus, and Merkel cell polyomavirus. While the roles of latent or integrated viral genomes in cancer have been documented, emerging evidence highlights the contribution of defective viruses-those carrying intragenic deletions or loss-of-function mutations-in promoting viral oncogenesis. These altered genomes often lack genes essential for lytic replication or immune recognition, which enhances their persistence and immune evasion. In virus-associated diseases, specific patterns of gene retention and deletion suggest that host-driven selective pressures drive the emergence of these altered genomes. This review examines the generation, prevalence, and functional impact of these viruses, reframing them as active participants in disease development and progression. Recognizing their role offers new insights into viral tumor evolution and creates opportunities for applications in viral diagnostics and targeted intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389381b25524966b33d29c79924df9e5f4c7676f" target='_blank'>
              Defective But Tumorigenic: The Evolutionary and Functional Roles of Mutated Oncoviruses.
              </a>
            </td>
          <td>
            Yoshitaka Sato, Yusuke Okuno, Takayuki Murata, Hiroshi Kimura
          </td>
          <td>2025-09-26</td>
          <td>FEMS microbiology reviews</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor with high morbidity and mortality rates worldwide and only presents symptoms in later stage; no ideal biomarker is available for the early diagnosis of CRC. Therefore, it is important to explore novel molecules that significantly contribute to CRC progression. The cohort contains different stages of CRC were downloaded and comprehensive bioinformatics analyses were performed by Mfuzz, Protein–Protein Interaction (PPI), MCODE, ESTIMATE, and ssGSEA.The results revealed that Protein Kinase, Membrane Associated Tyrosine/Threonine 1 (PKMYT1) served as a functional hub gene and its high expression might be associated with an immunosuppressive microenvironment, therapeutic sensitivity and tumor progression. PKMYT1-related genes are linked to DNA replication, the cell cycle, and mismatch repair, indicating PKMYT1 functions as an oncogene and potential biomarker in CRC development. Moreover, in vitro experimental investigation was conducted and the data found that CRC tumor tissues and cells have elevated PKMYT1 expression. Knockdown of PKMYT1 by siRNAs significantly impaired the proliferation, cell cycling, migration, and invasion of CRC cells. In summary, this study demonstrated that PKMYT1 may be a promising target for therapeutic intervention and play a significant role in the development of CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18430-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c70064227e93e681809cf44b6b66e461f9dc4b0" target='_blank'>
              Exploring PKMYT1 as a potential marker for colorectal cancer progression through bioinformatics analyses and experimental validation
              </a>
            </td>
          <td>
            Yuqiang Zhang, Yingwei Chang, Yaorui Hu, Zhichao Ding, Zhihui Zhang, Yanbei Wei, Ben Liu, Minghao Yang, Wei Chen
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Medulloblastomas, which occurs most commonly in children, is defined as an embryonal tumor derived from cells in the early stages of development. However, it is still unclear whether these embryonal tumors arise from embryonic cells in the uterus or as a result of single oncogenic event in fetal cells that rapidly progress to genetic instability. Meanwhile, potential transcription factor ZNF536 is most abundant in brain, especially in developing central nervous system. ZNF536 has been reported to have an inhibitory effect on neuronal differentiation, but the detailed regulatory mechanism has not been revealed. In particular, its pathological function in tumor development or progression of tumors is unknown. In this study, spontaneous tumor formation was observed in Zfp536+/- heteromice about 1 year old, and knock-down of ZNF536 in normal neuronal cells led cancerous phenotypes, such as neurosphere formation. Additionally, when comparing RNA expression levels in molecular genetic subgroups of medulloblastoma, ZNF536 expression was observed to be significantly reduced in group3, which is known to have the most malignant medulloblastoma. Based on our observation that loss of ZNF536 expression led spontaneous cancer development, we investigated the effect on cancer phenotypes after ZNF536 modulation. Besides, by performing ChIP-sequencing and RNA-sequencing, we identified the several downstream candidate targets of ZNF536 and further demonstrated comprehensive mechanisms of ZNF536 involved in cancer development. In summary, we suggested new factors including ZNF536 as a regulator of medulloblastoma development and progress, also we demonstrated regulation mechanisms by ZNF536. These results will provide insights understanding medulloblastoma development and new targeting strategies.



 Eun-Jin Yun and Seung Tae Baek. DISCOVERY OF NEW FACTORS INVOLVED IN MEDULLOBLASTOMA DEVELOPMENT AND INVESTIGATION OF ITS REGULATORY MECHANISM [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P77.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2442789bfe7e730bd10bb94e1ebf6df3c2d39113" target='_blank'>
              Abstract P77: DISCOVERY OF NEW FACTORS INVOLVED IN MEDULLOBLASTOMA DEVELOPMENT AND INVESTIGATION OF ITS REGULATORY MECHANISM
              </a>
            </td>
          <td>
            Eun-Jin Yun, Seung Tae Baek
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b570c21348c29c543cb996a976f642797b0edb0" target='_blank'>
              A SETD2-CDK1-lamin axis maintains nuclear morphology and genome stability.
              </a>
            </td>
          <td>
            Abid Khan, Cheng Zhang, Phu G Nguyen, James M. Metts, Lucas C. Collins, Kanishk Jain, C. A. Mills, Logan Vlach, Kelin Li, Amanda L. Brademeyer, Brittany M. Bowman, M. Major, Jeffrey Aubé, Laura E. Herring, W. Rathmell, Frank M. Mason, Ian J. Davis, Qing Zhang, B. Strahl
          </td>
          <td>2025-08-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Histologic transformation of prostate cancer from adenocarcinoma to neuroendocrine prostate cancer (NEPC) is associated with aggressive disease and poor prognosis. This lineage transition is accompanied by Polycomb complex 2 (PRC2)-mediated epigenetic de-repression of cell-fate determining transcription factors, including prospero-homeobox 1 (PROX1). Here, we sought to functionally characterize the role of PROX1 in NEPC. An unbiased CRISPR screen in two NEPC patient-derived organoid models demonstrated high cellular dependency for PROX1. Knockout of PROX1 impeded tumor growth in NEPC models and overexpression of PROX1 promoted tumor growth and spontaneous metastasis in prostate adenocarcinoma. Transcriptomic and cistromic analyses across castration resistant adenocarcinoma and neuroendocrine models pointed to PROX1-mediated regulation of neuroendocrine-lineage transcriptional programs. Immunoprecipitation followed by mass spectrometry identified three phosphorylated sites in the DNA-binding domain of PROX1 that are critical for its stability and function. CHEK1 and CDK2 were predicted to be upstream kinases that phosphorylate PROX1, and treatment with a CHEK1 or CDK2 inhibitor reduced NEPC viability. Together, these results substantiate the role of PROX1 in NEPC and identify PROX1 phosphorylation in the DNA binding domain, which might represent a therapeutic target in NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504618274d4b5115df7a7c55b6cd6538afea49bd" target='_blank'>
              Epigenetic De-repression of PROX1 Promotes Neuroendocrine Prostate Cancer Progression.
              </a>
            </td>
          <td>
            V. B. Venkadakrishnan, Adam Presser, Nathaniel C. E. Voss, James Neiswender, Lisa Brenan, Keira Prenza. Sosa, Kenny Weng, Andres M. Acosta, Francisca Vazquez, H. Beltran
          </td>
          <td>2025-08-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Specialized DNA polymerases facilitate various cellular processes. Despite extensive research, the mutagenic effects of these error-prone enzymes on genomes are not fully understood. Here we show that Pol IV promotes genomic instability in Pseudomonas aeruginosa by misincorporating oxidized guanine nucleotides. This activity led to a distinctive mutational signature, characterized by A-to-C transversions occurring preferentially at AT sites flanked by a 5’G and/or 3’C. Furthermore, Pol IV preferentially targeted pathogenicity genes located at specific chromosomal locations near the replication termination region and rRNA-encoding operons. Half of the mutation events catalyzed by Pol IV impaired gene function. This can be attributed to the bias of Pol IV for mutating codons with its preferred sequence contexts, leading to substitutions to unreactive alanine and glycine residues. Remarkably, mutation signatures identified for Pol IV were found in clinical isolate genomes of P. aeruginosa, providing compelling evidence for its role in genetic diversification during pathogen adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f7c02761e12b8b73eebc31b5beda14fbe3ac0c" target='_blank'>
              The low-fidelity DNA Pol IV accelerates evolution of pathogenicity genes in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Sofía D. Castell, Consuelo M. Fernandez, Ignacio N. Tumas, L. M. Margara, María C. Miserendino, Danilo Ceschin, R. Pezza, Mariela R. Monti
          </td>
          <td>2025-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Bladder cancer pathogenesis is closely linked to epigenetic alterations, particularly DNA methylation and demethylation processes. Environmental carcinogens and persistent inflammatory stimuli—such as recurrent urinary tract infections—can induce aberrant DNA methylation, altering gene expression profiles and contributing to malignant transformation. This review synthesizes current evidence on the role of DNA methyltransferases (DNMT1, DNMT3a, DNMT3b) and the hypermethylation of key tumour suppressor genes, including A2BP1, NPTX2, SOX11, PENK, NKX6-2, DBC1, MYO3A, and CA10, in bladder cancer. It also evaluates the therapeutic application of DNA-demethylating agents such as 5-azacytidine and highlights the impact of chronic inflammation on epigenetic regulation. Promoter hypermethylation of tumour suppressor genes leads to transcriptional silencing and unchecked cell proliferation. Urine-based DNA methylation assays provide a sensitive and specific method for non-invasive early detection, with single-target approaches offering high diagnostic precision. Animal models are increasingly employed to validate these findings, allowing the study of methylation dynamics and gene–environment interactions in vivo. DNA methylation represents a key epigenetic mechanism in bladder cancer, with significant diagnostic, prognostic, and therapeutic implications. Integration of human and experimental data supports the use of methylation-based biomarkers for early detection and targeted treatment, paving the way for personalized approaches in bladder cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f706e4ed85cef4708cafccd425d482bfabf2666" target='_blank'>
              DNA Methylation in Bladder Cancer: Diagnostic and Therapeutic Perspectives—A Narrative Review
              </a>
            </td>
          <td>
            D. Puia, M. Ivănuță, C. Pricop
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In mammals, primordial germ cells (PGCs) undergo global erasure of DNA methylation with delayed demethylation of germline genes and selective retention of DNA methylation at evolutionarily young retrotransposons. However, the molecular mechanisms of persistent DNA methylation in PGCs remain unclear. Here we report that resistance to DNA methylation reprogramming in PGCs requires UHRF2, the paralog of the DNMT1 cofactor UHRF1. PGCs from Uhrf2 knock-out mice show loss of retrotransposon DNA methylation, while DNA methylation is unaffected in somatic cells. This is not associated with changes in the expression of retrotransposons in E13.5 PGCs, indicating that other mechanisms compensate for retrotransposon control at this stage. Furthermore, Uhrf2-deficient PGCs show precocious demethylation of germline genes and overexpress meiotic genes in females. Subsequently, Uhrf2-deficient mice show impaired oocyte development and female-specific reduced fertility, as well as incomplete remethylation of retrotransposons during spermatogenesis. These findings reveal a crucial function for the UHRF1 paralog UHRF2 in controlling DNA methylation in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea0f6cead9b7479f9deb5d526e8e54ad404a663" target='_blank'>
              UHRF2 mediates resistance to DNA methylation reprogramming in primordial germ cells
              </a>
            </td>
          <td>
            Ambre Bender, Marion Morel, Michael Dumas, M. Klopfenstein, Naël Osmani, Maxim V. C. Greenberg, Déborah Bourc'his, Norbert B Ghyselinck, Michael Weber
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood.


METHODS
Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells.


RESULTS
A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures.


CONCLUSIONS
MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties.
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, Hafiz Muhammad Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transfer RNA-derived fragments (tRFs) are a class of small non-coding RNAs that have exhibited several functions in cancer. Recent studies have shown that mutations in tRNA genes can lead to global changes in tRF expression levels and may affect tRF function, highlighting the need to further elucidate the regulation and functions of tRFs in cancer. Here, we conducted a pan-cancer analysis of tRF quantitative trait loci (tRFQTLs), encompassing 16,703 genetic variants associated with tRF expression across 31 cancer types. A joint analysis of GWAS data revealed that tRFQTLs were preferentially enriched in cancer risk loci and colocalized with 106 GWAS variants, explaining a substantial portion of cancer heritability. Moreover, tRFs regulated by tRFQTLs were enriched in cancer-related pathways and correlated with drug response and immune infiltration. Notably, polygenic risk score models incorporating tRFQTLs improved high-risk population identification. Investigation of large-scale population cohorts revealed a tRFQTL, rs9461276, associated with colorectal cancer (CRC) risk. In biological assays, the rs9461276-C allele increased tRF-18-HSQS52D2 expression, which suppressed CRC malignant phenotypes. Mechanistically, tRF-18-HSQS52D2 bound to the 3'UTR of POU2F1, destabilizing the oncogenic transcript. Integrated RNA sequencing and ChIP-seq assays indicated that POU2F1 enhanced CRC cell proliferation by activating various pathological pathways associated with oxidative and glycolytic metabolism, mitotic stability, and cell cycle regulation. Finally, a database (Cancer-tRFQTL) was generated as a resource to support investigation into tRF-mediated mechanisms and genetic basis of tRF expression in human cancers. Overall, this study helps advance the understanding of tRFs in cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/888d9f82216c90d7528814f40ba0327b18c81f36" target='_blank'>
              Genetic Control of tRNA-Derived Fragments Contributes to Cancer Risk.
              </a>
            </td>
          <td>
            Bin Li, Hanting Li, Yan Li, Zhirui Chen, Danyi Zou, Chunyi He, Songqiang Li, Kexin Hu, Caibo Ning, Yanmin Li, Hui Geng, Shuhui Yang, Chaoqun Huang, Xiaojun Yang, Qiuhong Wang, Xu Zhu, Xiangpan Li, Ying Zhu, Meng Jin, Xiaoping Miao, Jianbo Tian
          </td>
          <td>2025-08-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Multidrug resistance remains a major obstacle in the treatment of ovarian cancer (OC) patients. Recent research has underscored the critical role of extrachromosomal circular DNA (eccDNA) in tumor initiation and progression. However, there is limited comprehensive understanding of the role eccDNA plays in tumor resistance.


OBJECTIVES
This study investigates the involvement of WWP1-eccDNA in the resistance mechanisms of OC.


MATERIAL AND METHODS
Human OC cells (SKOV3 and cisplatin-resistant SKOV3/DDP) were cultured and high-throughput sequencing was performed, leading to the identification of eccDNA in SKOV3/DDP cells. Female BALB/cA-nu nude mice with SKOV3 and SKOV3/DDP xenografts received cisplatin (5.5 mg/kg), hydroxyurea (50 mg/kg) or saline for 14 days, followed by tumor weight assessment. Digital droplet polymerase chain reaction (ddPCR) and real-time quantitative polymerase chain reaction (qPCR) were used to quantify WWP1-eccDNA, evaluating their sensitivity and accuracy. Linear DNA removal and BsmI digestion were tested to improve eccDNA detection.


RESULTS
WWP1-eccDNA was among the top upregulated eccDNA in SKOV3/DDP cells. Both cisplatin and hydroxyurea reduced tumor growth in mice, with cisplatin showing limited efficacy in resistant tumors. The ddPCR outperformed RT-qPCR in sensitivity, and linear DNA removal improved WWP1-eccDNA detection. WWP1-eccDNA levels were significantly elevated in SKOV3/DDP tumors. Treatment with cisplatin further increased its expression, whereas hydroxyurea led to a reduction in WWP1-eccDNA levels.


CONCLUSIONS
WWP1-eccDNA is critical in OC resistance, with cisplatin treatment increasing WWP1-eccDNA levels, contributing to resistance. The ddPCR proves to be a superior method for eccDNA detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ff3cc0602dcb179f8683c613e50da47a26f84f2" target='_blank'>
              Cisplatin-induced WWP1-eccDNA expression contributes to ovarian cancer resistance.
              </a>
            </td>
          <td>
            Chenyang Lu, Li HAN, Xiaojuan Guo, Ruijuan Du, Hui Zhang, K. Guo, Yun-Fei Tu, Ruifang Li
          </td>
          <td>2025-09-04</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human Papillomavirus (HPV), particularly high-risk strains such as HPV16 and HPV18, is a leading cause of cervical cancer and a significant risk factor for several other epithelial malignancies. While the oncogenic mechanisms of viral proteins E6 and E7 are well characterized, the broader effects of HPV infection on host transcriptional regulation remain less clearly defined. This study explores the hypothesis that conserved genomic motifs within the HPV genome may act as molecular decoys, sequestering human transcription factors (TFs) and thereby disrupting normal gene regulation in host cells. Such interactions could contribute to oncogenesis by altering the transcriptional landscape and promoting malignant transformation.We conducted a computational analysis of the genomes of high-risk HPV types using MEME-ChIP for de novo motif discovery, followed by Tomtom for identifying matching human TFs. Protein–protein interactions among the predicted TFs were examined using STRING, and biological pathway enrichment was performed with Enrichr. The analysis identified conserved viral motifs with the potential to interact with host transcription factors (TFs), notably those from the FOX, HOX, and NFAT families, as well as various zinc finger proteins. Among these, SMARCA1, DUX4, and CDX1 were not previously associated with HPV-driven cell transformation. Pathway enrichment analysis revealed involvement in several key biological processes, including modulation of Wnt signaling pathways, transcriptional misregulation associated with cancer, and chromatin remodeling. These findings highlight the multifaceted strategies by which HPV may influence host cellular functions and contribute to pathogenesis. In this context, the study underscores the power of in silico approaches for elucidating viral–host interactions and reveals promising therapeutic targets in computationally predicted regulatory network changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019a01c1572bf807f787812d1b2ed2b11781bb4f" target='_blank'>
              HPV as a Molecular Hacker: Computational Exploration of HPV-Driven Changes in Host Regulatory Networks
              </a>
            </td>
          <td>
            M. Chetta, Alessandra Rosati, N. Bukvic
          </td>
          <td>2025-08-27</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 DNA damaging agents remain the mainstay for high grade serous ovarian cancer (HGSOC) therapy. While many HGSOCs are sensitized to DNA damaging agents via dysregulated DNA repair, those patients harboring tumors with amplification or overexpression of CCNE1 (encoding cyclin E1) respond poorly in part due to proficiency in the homologous recombination (HR) DNA repair pathway. Here, we put forth a novel paradigm that HR proficiency of CCNE1-driven HGSOCs is promoted by altered cellular metabolism. We found that alpha-ketoglutarate (aKG) is upregulated in cyclin E1-driven HGSOC cells, and suppression of aKG sensitized these cells to DNA damaging agents both in vitro and in vivo, demonstrating that aKG drives chemoresistance in these models. aKG is required for the activity of aKG-dependent dioxygenases (aKGDD). There are >60 known aKGDD, and prior work has almost exclusively focused on aKGDD-mediated histone demethylation. Through a targeted CRISPR knockout library, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for chemoresistance to DNA damaging agents in cyclin E1-driven cells. Unexpectedly, aKG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. These data point to a previously unrecognized role for aKG in acetylation reactions. The increase in histone acetylation via aKG-dependent carnitine synthesis promoted HR-mediated repair through site-specific histone acetylation. Additionally, analysis of HGSOC patient samples demonstrated that TMLHE positively correlates with histone acetylation, and both high tumor TMLHE or high serum acetylcarnitine are associated with worse progression free survival (PFS) in HGSOC patients treated with DNA damaging agents. Pharmacological targeting of carnitine synthesis via mildronate in vivo was sufficient to decrease tumor burden >30% in combination with cisplatin. Since mildronate is a non-toxic and extensively used drug in humans, this combination shows potential applicability of inhibition of this axis in humans. These data demonstrate for the first time that HR-proficiency in cyclin E1-driven HGSOCs is mediated through a metabolic axis that directly influences histone acetylation and highlight the translational potential of targeting this axis to induce HR deficiency and sensitize these tumors to DNA damaging agents.



 Apoorva Uboveja, Nathaniel W. Snyder, Katherine M. Aird. aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7139762235894a9ff9c39c95032bcf77d46290e" target='_blank'>
              Abstract B031: aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation
              </a>
            </td>
          <td>
            Apoorva Uboveja, Nathaniel W. Snyder, K. Aird
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Bladder cancer (BC) is a disease that predominantly affects older adults, with aging playing a critical role in its onset and progression. Age-associated phenomena, including immunosenescence and chronic inflammation, form a pro-tumor milieu, while genomic instability and epigenetic drift further increase cancer risk. The review highlights the dual role of DNA methylation in BC: global hypomethylation can activate transposable elements and oncogenes, whereas focal hypermethylation silences tumor-suppressor genes like CDKN2A, especially detrimental in older tissues that rely on these genes for senescence control. In parallel, frequent mutations in chromatin modifiers (e.g., KDM6A, KMT2D) and overexpression of histone-modifying enzymes (e.g., EZH2) alter the tumor epigenome to promote immune evasion and tumor aggressiveness. At the non-coding RNA level, dysregulated microRNAs (miRNAs) and long non-coding RNAs (lncRNAs) in BC contribute to aberrant proliferation, metastatic potential, and immune suppression, with aging-associated declines in miRNA processing further exacerbating these effects. Collectively, the accumulation of epigenetic alterations in older patients appears to facilitate both tumor progression and resistance to therapy. Looking forward, epigenetic biomarkers may improve early detection and risk stratification. Furthermore, “epigenetic therapies,” such as DNA methyltransferase inhibitors (DNMTi), EZH2 inhibitors (EZH2i), or histone deacetylases inhibitors (HDACi), hold promise to restore tumor-suppressor function and enhance immunogenicity, offering an attractive avenue for improving outcomes in older patients with BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bc5ece634826938f00d72c674db3193d520a4b" target='_blank'>
              Epigenetic regulation of bladder cancer in the context of aging
              </a>
            </td>
          <td>
            Xuewei Liu, Guofeng Ding, Yifan Liu, Xiaoli Yan, Yan Zhao, Hailin Lv, Xiaojuan Xu
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Proper genome organization is essential for genome function and stability. Disruptions to this organization can lead to detrimental effects and the transformation of cells into diseased states. Individual chromosomes and their subregions can move or rearrange during transcriptional activation, in response to DNA damage, and during terminal differentiation. Techniques such as fluorescence in situ hybridization (FISH) and chromosome conformation capture (e.g., 3C and Hi-C) have provided valuable insights into genome architecture. However, these techniques require cell fixation, limiting studies of the temporal evolution of chromatin organization in detail. Our understanding of the heterogeneity and dynamics of chromatin organization at the single-cell level is still emerging. To address this, clustered regularly interspaced short palindromic repeats (CRISPR)/dead Cas9 (dCas9) systems have been repurposed for precise live-cell imaging of genome dynamics. This protocol uses a system called CRISPRainbow, a powerful tool that allows simultaneous targeting of up to seven genomic loci and tracks their locations over time using spectrally distinct fluorescent markers to study real-time chromatin organization. Multiple single-guide RNA (sgRNA), carrying specific RNA aptamers for labeling, can be cloned into a single vector to improve transfection efficiency in human cells. The precise targeting of CRISPRainbow offers distinct advantages over previous techniques while also complementing them by validating findings in live cells. Key features • Simultaneous imaging of up to seven specific genomic loci in living cells. • Multicolor imaging using a single CRISPR system from Streptococcus pyogenes. • Signal amplification through targeting repetitive sequences. • Targeting endogenous DNA without the need for foreign DNA insertion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bbbccdb01c211556cafc9956628f40eb64de811" target='_blank'>
              Real-Time Imaging of Specific Genomic Loci With CRISPR/dCas9 in Human Cells Using CRISPRainbow
              </a>
            </td>
          <td>
            Thomas Versosky, Dilshodbek Nishonov, Li-Chun Tu
          </td>
          <td>2025-09-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Survival strategy of bacteria is expanded by extrachromosomal elements (ECEs). However, their genetic diversity and functional roles for adaptability are largely unknown. Here, we discover a novel family of intracellular ECEs using 56 saliva samples by developing an efficient microbial DNA extraction method coupled with long-read metagenomics assembly. Even though this ECE family was not hitherto identified, our global prevalence analysis using 476 salivary metagenomic datasets elucidates that these ECEs reside in 74% of the population. These ECEs, which we named, “Inocles”, are giant plasmid-like circular genomic elements of 395 kb in length, including Streptococcus as a host bacterium. Inocles encode a series of genes that contribute to intracellular stress tolerance, such as oxidative stress and DNA damage, and cell wall biosynthesis and modification involved in the interactions with oral epithelial cells. Moreover, Inocles exhibit significant positive correlations with immune cells and proteins responding to microbial infection in peripheral blood. Intriguingly, we examine and find their marked reductions among 68 patients of head and neck cancers and colorectal cancers, suggesting its potential usage for a novel biomarker of gastrointestinal cancers. Our results suggest that Inocles potentially boost the adaptive capacity of host bacteria against various stressors in the oral environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ef6b2491714273e3a5acd966b2b147c6dd3e3d" target='_blank'>
              Giant extrachromosomal element “Inocle” potentially expands the adaptive capacity of the human oral microbiome
              </a>
            </td>
          <td>
            Yuya Kiguchi, Nagisa Hamamoto, Y. Kashima, L. Runtuwene, A. Ishizaka, Yuta Kuze, T. Enokida, N. Tanaka, Makato Tahara, S. Kageyama, T. Fujisawa, Riu Yamashita, A. Kanai, Josef S. B. Tuda, Taketoshi Mizutani, Yutaka Suzuki
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma.
              </a>
            </td>
          <td>
            Sixuan Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, Arie Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Cervical cancer remains a significant public health burden in Sub-Saharan Africa, where high-risk human papillomavirus (HPV) infections are prevalent and health system inequities contribute to poor outcomes. Beyond viral oncogenes, HPV modulates host gene expression through alternative splicing, a process increasingly recognized as a driver of cancer progression. The serine/arginine-rich protein kinase 1 (SRPK1) regulates splicing via phosphorylation of SRSF1 and is emerging as a potential therapeutic target. However, the transcriptomic consequences of SRPK1 inhibition in HPV-associated cervical cancer, particularly in the African context, remain underexplored.



 This study investigates how SRPK1 inhibition alters splicing and gene expression in cervical cancer models relevant to the African disease landscape. SiHa (HPV16-positive) and C33A (HPV-negative) cell lines were treated with the SRPK1 inhibitor SPHINX31. Cellular phenotypes were assessed via resazurin viability assays, flow cytometric cell cycle analysis, and Annexin V/PI apoptosis assays. RNA sequencing was performed to evaluate global transcriptomic changes, including differential expression and alternative splicing events.



 SRPK1 inhibition did not significantly affect cell viability or apoptosis in either cell line. However, RNA-seq revealed distinct transcriptomic reprogramming between HPV-positive and HPV-negative models. In HPV-positive SiHa cells, proliferation-related genes were downregulated, accompanied by upregulation of metabolic and xenobiotic response pathways, indicating a unique adaptive response linked to viral oncogene presence. Conversely, HPV-negative C33A cells showed a broader downregulation of transcriptional regulators and increased expression of metabolic genes, reflecting alternative oncogenic pathways independent of HPV infection. Notably, splicing alterations also differed: SiHa cells exhibited splicing changes enriched in focal adhesion and endoplasmic reticulum targeting genes, while C33A cells showed splicing events associated with mitochondrial organization and transcriptional regulation.



 These findings suggest that SRPK1 inhibition influences RNA splicing and gene expression without altering immediate cellular phenotypes. Importantly, the identification of HPV-specific splicing signatures in models relevant to African cervical cancer supports SRPK1 as a potential modulator of RNA processing with therapeutic relevance. This study contributes to the molecular understanding of cervical cancer disparities and highlights the need for splicing-targeted approaches in precision oncology across underserved populations.



 Zodwa Dlamini, Afra T. Basera, David O. Bates, Rahaba Marima, Mohammed Alaouna, Janie Duvenhage. SRPK1-mediated distinct transcriptomic reprogramming in cervical cancer: Insights into splicing regulation in the African context [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ead886f9863839ac9fd2abd1d07ec66732be60" target='_blank'>
              Abstract A034: SRPK1-mediated distinct transcriptomic reprogramming in cervical cancer: Insights into splicing regulation in the African context
              </a>
            </td>
          <td>
            Zodwa Dlamini, Afra Basera, David O. Bates, R. Marima, Mohammed Alaouna, Janie Duvenhage
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive subtype of breast cancer with high metastatic potential, limited treatment options, and low patient survival rates. By combining functional proteomics and genomics approaches, we identified an oncogenic transcriptional network in mesenchymal and invasive TNBC involving the glucocorticoid receptor (GR), GATA6, MYC, and AP-1 transcription factors. Although these transcription factors bound extensively to shared enhancers, they utilized different enhancer repertoires from this shared enhancer pool to drive distinct downstream oncogenic pathways. The SWI/SNF chromatin remodeling complex was a common gatekeeper of chromatin access for these transcription factors, and SWI/SNF inhibition decommissioned the distinct enhancer networks and pro-tumorigenic gene programs they control. Consistently, SWI/SNF inhibition repressed TNBC proliferation, glucocorticoid-induced chemoresistance, and invasion in vitro and suppressed tumor growth and metastasis in vivo. Thus, these findings indicate that SWI/SNF-mediated chromatin remodeling drives TNBC progression, positioning SWI/SNF as a promising therapeutic target in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5aecd9fb0bc36400b799414dcc4be0d90b23c11" target='_blank'>
              SWI/SNF Functions as a Gatekeeper of Enhancer Chromatin Access to Control Progression of Mesenchymal Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Rikkemaja Isahbell J Møldrup, M. Terp, S. T. Jakobsen, Tina Ravnsborg, M. S. Madsen, Daniel D Brown, Laurits R Sloth, Michael C. Gorry, S. Oesterreich, Adrian V. Lee, Henrik J Ditzel, Ole N. Jensen, Rasmus Siersbæk
          </td>
          <td>2025-09-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Graphical Abstract Abstract Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a gene of unknown significance because no inherited disease had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drives oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanic calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanic calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic features of the methylosome protein MEP50 and its implications in hormone signaling and cancer
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45e0bd555bafab74ee26abfc65e0d5026258f51" target='_blank'>
              Acetic acid produced by Staphylococcus epidermidis remodels chromatin architecture and suppresses gene expression in Malassezia restricta.
              </a>
            </td>
          <td>
            Jae Min Lee, Hyun Oh Yang, Hideki Tanizawa, Ken-ichi Noma, Tae Kwon Lee, W. H. Jung, Yong-Joon Cho, Kyoung-Dong Kim
          </td>
          <td>2025-09-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Despite treatment advances, ovarian cancer (OC) remains the deadliest gynecological cancer, with a ∼70% mortality rate. OCs initially respond to platinum chemotherapy but often develop resistance. While PARP inhibitors improve relapse-free survival in responsive cases, they do not extend overall survival. This underscores the need for effective combination therapies beyond BRCA status. The lysosomal protein cathepsin L (CTSL) can traffic to the nucleus, where it can regulate DNA repair and serve as a chromatin modifier. We found that nucleoside analog clofarabine (CLF) induces CTSL nuclear trafficking in OC cells. Here we tested the hypothesis that accumulation nuclear CTSL (nCTSL) is a biomarker of sensitivity to DNA damaging therapy in OCs for improved therapeutic options.



 To determine the role of CLF induced nuclear CTSL in DNA damage and its synergy with PARP and CRM1 inhibitors in PARP inhibitor–sensitive and –resistant ovarian cancer (OC) cell lines, ex vivo cultures of ascites-derived patient OC cells (OVA-3D spheroids), primary ovarian tumors, and patient-derived xenograft (PDX) models and validated by colony formation assays. Nuclear CTSL localization was determined in subcellular fractionated lysates by western blots and immunofluorescence cytometry (IFC). Tumor regression was determined in vivo using ex vivo cultures of PDX derived cells.



 In ovarian cancer (OC) cell lines, CLF monotherapy induced nuclear CTSL (nCTSL) in a subset of models classified as CLF-responsive (CLF-r), sensitizing them to the PARP inhibitors olaparib and rucaparib. In CLF-non-responsive (CLF-nr) models, nCTSL induction and therapeutic synergy were observed only with the CLF+olaparib combination. Synergy was detected in 47% of CLF-r and 24% of CLF-nr patient-derived ascites samples (OVA). Mechanistically, CLF induced nuclear import of CTSL via KPNB1 in CLF-r cells, while this import was triggered by the CLF+olaparib combination in CLF-nr cells. CLF also downregulated the nuclear export protein CRM1 (XPO1) in both groups. Knockdown of CTSL in either CLF-r or CLF-nr cells abrogated sensitivity to the combination. Notably, in 29% of OVA samples classified as CLF-resistant (CLF-Res), CLF failed to suppress CRM1 expression. In these cases, CRM1 inhibition with KPT8602 restored CLF+PARP inhibitor synergy. In vivo, CLF-r and CLF-nr PDX models treated with the combination exhibited increased DNA damage, reduced tumor burden, and extended survival.



 These findings identify nuclear CTSL (nCTSL) induction as a potential biomarker of response to DNA-damaging therapies. The combination of CLF and olaparib represents a promising strategy to overcome drug resistance in ovarian cancer by promoting CTSL nuclear localization and enhancing DNA damage.



 Prabhu Thirusangu, Ling Jin, Upasana Ray, Anya Zhao, Julie Staub, Xinyan Wu, Xiaonan Hou, Jamison L. VanBlaricom, Syed Mohamed Musheer Aalam, Sun-Hee Lee, Ann Oberg, Nagarajan Kannan, John Weroha, Jeremy Chien, Scott H. Kaufmann, Jamie N. Bakkum-Gamez, Viji Shridhar. Role of drug induced nuclear CTSL (nCTSL) in DNA damage response in cancer- therapeutic implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f74528b6924a7e697a49ee1e41cc77afb923b83" target='_blank'>
              Abstract A050: Role of drug induced nuclear CTSL (nCTSL) in DNA damage response in cancer- therapeutic implications
              </a>
            </td>
          <td>
            P. Thirusangu, Ling Jin, Upasana Ray, Anya Zhao, J. Staub, Xinyan Wu, Xiaonan Hou, Ja Vanblaricom, Syed Mohammed Musheer Aalam, Sun-Hee Lee, A. Oberg, Nagarajan Kannan, John S Weroha, Jeremy Chien, Scott H. Kaufmann, Jamie N. Bakkum-Gamez, V. Shridhar
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma stem cells (GSCs), a stem-like tumorigenic subpopulation within glioblastoma (GBM), exhibit remarkable self-renewal capacity and therapeutic resistance. Zinc finger BED domain-containing protein 1 (ZBED1), a dual-function transcription factor and SUMO E3 ligase, has been implicated in oncogenic processes across malignancies, its functional role and regulatory mechanisms in GSCs remain enigmatic.


METHODS
Multimodal approaches including immunohistochemistry, immunoblotting, and immunofluorescence were employed to evaluate ZBED1 expression patterns in GSCs and clinical GBM specimens. Functional characterization utilized in vitro models (proliferation assays, tumor-sphere formation assays, limiting dilution analysis) complemented by in vivo orthotopic xenograft models. Mechanistic investigations integrated RNA sequencing, label-free proteomics, chromatin immunoprecipitation (ChIP), immunohistochemistry, and western blotting to delineate the EGFR/ZBED1 regulatory axis.


RESULTS
We demonstrated that ZBED1 was significantly upregulated in GSCs and linked to unfavorable prognosis. Genetic ablation of ZBED1 significantly impaired GSC proliferation and self-renewal capacity, while extending survival in xenograft models. Mechanistically, EGFR-mediated ZBED1 phosphorylation at tyrosine residues Y160/Y513 enhanced ZBED1-UBC9 interaction, promoting SUMOylation-dependent protein stabilization. Remarkably, ZBED1 reciprocally sustained EGFR expression through transcriptional repression of the E3 ubiquitin ligase PARK2, establishing a self-reinforcing EGFR/ZBED1/PARK2 signaling circuit critical for GSC maintenance.


CONCLUSIONS
Our findings elucidate a novel EGFR/ZBED1 positive feedback loop that drives GSC propagation and tumorigenesis, highlighting ZBED1 as an attractive candidate for therapeutic targeting in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5de1c18b91342ef1a1f3fd2247cd9768fb0fd3" target='_blank'>
              EGFR/ZBED1 reciprocal regulation promotes stemness and tumorigenesis in glioblastoma.
              </a>
            </td>
          <td>
            Nana Hou, Yutao Wang, Qiuxiang You, Lei Zhang, Wenjia Zhang, Xinyi Wang, Chunyan Deng, Jiachuan Yan, Saiyu Cheng, Jianwen Ji, Jianbing Hou, Hongjuan Cui, Yundong Zhang
          </td>
          <td>2025-09-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Extrachromosomal and mobile genetic elements, including plasmids, and accessory chromosomes, are prevalent in all life domains. Elements integrated into the host chromosome replicate and segregate via the host life cycle. In contrast, the persistence of autonomously replicating elements relies on their ability to remain within the host population. Here, we compare the evolutionary advantage of different persistence strategies found in prokaryotic plasmids. Through intracellular competitions between plasmid genotypes, we find that the combination of active partitioning during cell division with a toxin-antitoxin (TA) system for post-segregational killing increases plasmid fitness more than either strategy alone. Mathematical modeling of long-term plasmid evolution, calibrated with empirical plasmid loss dynamics, further supports these findings. A survey of enterobacterial genomes indicates that partitioning and TA systems are core features of large plasmids. Indeed, we confirm the presence of a previously unrecognized type I TA system in conjugative IncX3 plasmids, which serve as important vectors of antibiotic resistance in human pathogens. These findings suggest that large plasmids – including conjugative and mobilizable types – encode TA systems, some of which have yet to be identified. The combination of TA and partitioning systems emerges as the most effective strategy for the evolutionary success of low-copy extrachromosomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ddba67c058e73d4520b91af55962bc53698843f" target='_blank'>
              The combination of active partitioning and toxin-antitoxin systems is most advantageous for low-copy plasmid fitness
              </a>
            </td>
          <td>
            Johannes Effe, M. Santer, Yiqing Wang, Theresa E. Feenstra, Nils F. Hülter, Tal Dagan
          </td>
          <td>2025-08-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="We report here on the identification of a previously unrecognized property of MYOD as a repressor of gene expression via E-box-independent chromatin binding during the process of somatic cell trans-differentiation into skeletal muscle. When ectopically expressed in proliferating human fibroblasts or endogenously induced in activated muscle stem cells (MuSCs), MYOD was detected at accessible regulatory elements of expressed genes, invariably leading to reduced chromatin accessibility and gene repression. At variance with conventional E-box-driven increased chromatin accessibility and H3K27 acetylation at previously silent loci of MYOD-activated genes, MYOD-mediated chromatin compaction and repression of transcription was associated with high occurrence of non-E-box motifs and did not lead to reduced levels of H3K27ac but coincided with reduced levels of H4 acetyl-methyl lysine modification (Kacme). Using MYOD mutants, we dissected the molecular mechanism of MYOD-mediated repression, whereby repression of mitogen-responsive and growth factor-responsive genes occurred via promoter binding, which requires a conserved domain within the first helix; conversely, repression of cell of origin/alternative lineage genes occurred via binding and decommissioning of distal regulatory elements such as superenhancers (SEs), required either the N-terminal activation domain or the two chromatin remodeling domains, and coincided with reduced strength of CTCF-mediated chromatin interactions. These data extend MYOD biological properties beyond the current dogma that restricts MYOD function to a monotone transcriptional activator. They also reveal an unprecedented functional versatility arising from alternative chromatin recruitment through E-box or non-E-box sequences, whereby genetic determinants dictate differential usage of MYOD functional domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ca9bd0655d7bc05438f6e1c35c0aee46817463" target='_blank'>
              MYOD represses gene expression from non-E-box motifs.
              </a>
            </td>
          <td>
            C. Nicoletti, Jimmy Massenet, Andreas P. Pintado-Urbanc, Leah J. Connor, Monica Nicolau, Swetha Sundar, Mingzhi Xu, Anthony Schmitt, Wenxin Zhang, Zesen Fang, Tsz Ching Indigo Chan, Yu Xin Wang, Stephen J. Tapscott, Tom H. Cheung, Matthew D. Simon, Luca Caputo, Pier Lorenzo Puri
          </td>
          <td>2025-08-06</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Many recent lines of evidence from the human microbiome and other fields indicate bacterial involvement in various types of cancer. Helicobacter pylori has been recognized as the major cause of stomach cancer (gastric cancer), but the mechanism by which it destabilizes the human genome to cause cancer remains unclear. Our recent studies have identified a unique family of toxic restriction enzymes that excise a base (A: adenine) from their recognition sequence (5′-GTAC). At the resulting abasic sites (5′-GT_C), its inherent endonuclease activity or that of a separate endonuclease may yield atypical strand breaks that resist repair by ligation. Here, we present evidence demonstrating involvement of its H. pylori member, HpPabI, in stomach carcinogenesis: (i) Association of intact HpPabI gene with gastric cancer in the global H. pylori Genome Project and the open genomes; (ii) Frequent mutations at A in 5′-GTAC in the gastric cancer genomes as well as in H. pylori genomes; (iii) Its induction of chromosomal double-strand breaks in infected human cells and of mutagenesis in bacterial test systems. In addition, its unique regions that interact with DNA exhibit signs of diversifying selection. Our further analysis revealed similar oncogenic bacterium–restriction–enzyme pairs for other types of cancer. These results set another stage for cancer research and medicine around oncogenic restriction enzymes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f484817b57e3d6d7ad5b587893b556976204531a" target='_blank'>
              Helicobacter pylori base-excision restriction enzyme in stomach carcinogenesis
              </a>
            </td>
          <td>
            Masaki Fukuyo, Noriko Takahashi, Katsuhiro Hanada, Ken Ishikawa, eslovas Venclovas, K. Yahara, Hideo Yonezawa, Takeshi Terabayashi, Yukako Katsura, Naoki Osada, Atsushi Kaneda, M. C. Camargo, Charles S Rabkin, I. Uchiyama, Takako Osaki, Ichizo Kobayashi, Edited By, Asima Bhattacharyya
          </td>
          <td>2025-08-01</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Chromatin relaxation is a permissiven progress for DNA repair through enabling repair factors to access the damaged DNA. Linker histone H1 is important in maintaining chromatin compaction under physiological state. The recent evidence highlights the importance of H1 modifications in response to cellular stress. Following DNA double‐strand breaks, the metabolic enzyme phosphorylated CTP synthase 1 (CTPS1) functions as a deamidase, catalyzing the rapid conversion of H1 residues Asn76 and Asn77 into aspartate. This modification enables subsequent acetylation at Lys75 by the histone acetyltransferase p300, thereby reducing H1‐DNA affinity and promoting chromatin decompaction. This sequential modification‐H1 deamidation followed by acetylation‐facilitates the recruitment of repair factors involving both homologous recombination and non‐homologous end joining repair pathways, and consequently promoting DNA repair. Importantly, high expression of CTPS1 is associated with resistance to radiotherapy in mouse models and clinical cancer patients, suggesting that the CTPS1 may serve as a potential therapeutic target. While targeting CTPS1 may offer opportunities to enhance radiosensitivity of cancer patients, challenges related to specificity and off‐target effects require further studies. This article highlights an emerging role of H1 modification in the DNA damage repair and discusses the therapeutic potential of manipulating H1 deamidation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ef2febfbdf0354039134d0a6f7b0aac3792561" target='_blank'>
              H1N76/77 deamidation facilitates chromatin remodelling and genome stability during DNA damage repair
              </a>
            </td>
          <td>
            Tingting Feng, Mengyan Li, Chenmiao Hu, Yuan Tian, Wei-Guo Zhu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41c845deccf83a402934c97c50b9796b93645c5b" target='_blank'>
              Establishment of chromatin architecture interplays with embryo hypertranscription.
              </a>
            </td>
          <td>
            Guang Yu, Kai Xu, Weikun Xia, Ke Zhang, Qianhua Xu, Lijia Li, Zili Lin, Ling Liu, Bofeng Liu, Zhenhai Du, Xia Chen, Qiang Fan, Fangnong Lai, Wenying Wang, Lijuan Wang, Feng Kong, Chao Wang, Haiqiang Dai, Huili Wang, Wei Xie
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Nasopharyngeal Carcinoma (NPC) is a highly metastatic malignancy, often diagnosed at advanced stages with poor prognosis and survival rates. It is strongly associated with Epstein-Barr virus (EBV) exposure and Chinese/Asian ethnic backgrounds, leading to approximately 300 new cases annually in Singapore. EBV plays a critical role in NPC development, with both its latent and lytic phases contributing to tumorigenesis. The virus precisely controls its methylation pattern to regulate latent gene expression and lytic reactivation. In contrast, disruptions in DNA modification patterns in humans are frequently observed in tumorigenesis, suggesting transcriptional dysregulation. Currently, few studies have explored limited loci in human and EBV epigenomes, with most only profiling population-level average methylation status in NPC. These studies cannot distinguish between 5-hydroxymethylcytosine (5hmC) and 5-methylcytosine (5mC), which have opposing regulatory roles. Moreover, existing methods fail to access repetitive regions, which comprise significant portions of both the EBV and human genomes. To address these gaps, we generated whole-genome, single-molecule, single-base resolution profiles of 5mC and 5hmC modifications in EBV and human DNA from different NPC tumor samples and cell lines using direct Nanopore ultralong-read sequencing. This approach offers unique insights into the complexities of EBV and human DNA modifications during NPC development. We discovered that distinct EBV strains cause contrasting 5mC changes in both human and EBV epigenomes, with some inducing hypermethylation and others hypomethylation. While hypomethylation was previously linked to APOBEC activity, our findings suggest that EBV strain-specific sequences play a significant role. Notably, in EBV epigenome, 5mC differences are concentrated in repetitive regions, whereas 5hmC profiles are consistent across strains, with persistent 5hmC modifications at specific loci. Our study provides valuable epigenome knowledge to the field of NPC research. Further data and detailed mechanistic studies are crucial to deepening our understanding of disease onset, progression, and advancing early detection and therapeutic strategies.



 Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Liu Jianjun. Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P52.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e731f932485dff94f0e6e21fa72516a251e45df3" target='_blank'>
              Abstract P52: Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing
              </a>
            </td>
          <td>
            Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Jianjun Liu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Lynch syndrome (LS) and familial adenomatous polyposis (FAP) are hereditary cancer predisposing syndromes characterized by increased risk of especially early-onset colorectal cancer. Predisposition to LS is caused by germline mutations in DNA mismatch repair genes leading to elevated cancer progression and microsatellite instability. FAP is associated with germline mutations in APC promoting cancer initiation and chromosomal instability. DNA methylation is an important epigenetic mechanism in early tumorigenesis via, e.g., field defects in non-neoplastic colon. Our aim was to study genome-wide methylation changes in colorectal specimens (adenomas and carcinomas supplemented with paired normal colon) obtained during colonoscopy surveillance, and explore the role of such alterations in tumorigenesis, with a special focus on early changes. To our best knowledge, this study is the first one to compare altered DNA methylation genome-wide in LS and FAP-associated colorectal neoplasia. Results DNA methylation alterations were subtle in FAP adenomas, whereas in LS adenomas, changes were abundant when compared to their normal counterparts. When FAP normal and LS normal colon were compared, DNA methylation changes of FAP normal colon mirrored those occurring in LS tumors, suggesting that colorectal tumorigenesis-associated DNA methylation alterations take place already in FAP normal colon mucosa. DNA methylation age was more variable in LS than FAP normal colon, and in proximal than distal colon, when compared to individuals’ age at the time of sampling. In LS tumors, DNA methylation changes (hyper- and hypomethylation) were abundant even in adenomas with low-grade dysplasia and stable microsatellites and peaked in adenomas with high-grade dysplasia. LINE-1 hypomethylation was more prominent in LS adenomas than FAP adenomas, but normal colon of LS and FAP displayed similar levels of LINE-1 methylation. Conclusions Genome-wide DNA methylation changes are an integral part of FAP and LS-associated colorectal tumorigenesis. Occurrence at early stages, even in non-neoplastic colonic mucosa, and increased prevalence with progressive dysplasia suggest a role in tumor development. Overlap of many of the topmost DNA methylation alterations between LS and FAP, and previous reports of their occurrence in sporadic colorectal and other tumors as well, imply their broad biological relevance and possible biomarker potential for clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01940-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f732072d5015ea229362c0044955d5acfa518d6c" target='_blank'>
              Genome-wide DNA methylation profiles of colorectal tumors in Lynch syndrome and familial adenomatous polyposis
              </a>
            </td>
          <td>
            Satu Mäki-Nevala, Anni K Kauppinen, A. Olkinuora, Aleksi Laiho, Petri Törönen, L. Renkonen-Sinisalo, A. Lepistö, T. Seppälä, J. Mecklin, P. Peltomäki
          </td>
          <td>2025-08-02</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Plasmid fusion, the recombination of distinct plasmids into a single plasmid, constitutes a critical mechanism shaping bacterial evolution. This process facilitates genetic resource consolidation, enabling bacterial populations to rapidly adapt to selective pressures such as antibiotic exposure. The growing recognition of plasmid fusion as a widespread genomic recombination event necessitates a comprehensive analysis of its prerequisites, molecular mechanisms, and functional consequences. This review synthesizes current knowledge on the triggers, molecular mechanisms, and outcomes of plasmid fusion. We demonstrate that conjugative transfer and antibiotic selection pressure drive plasmid fusion via DNA double-strand breaks and SOS response activation, with formation mechanisms dominated by insertion sequence-mediated homologous recombination or transposon-mediated recombination. Fusion plasmids amplify antimicrobial resistance (AMR) dissemination by creating multidrug-resistant megaplasmids, enable cross-host transfer of non-conjugative plasmids, and foster virulence-resistance hybrid elements. Furthermore, we propose a novel framework for investigating, specifically addressing how fusion plasmids achieve compensatory mechanisms to balance functional redundancies among critical genetic modules. Elucidating the evolutionary drivers underlying the pervasive dissemination of fusion plasmids will not only advance our understanding of their ecological success but also identify critical intervention targets to disrupt their dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e76ccd851ef2713c4c63665c2a723bfdeba73e" target='_blank'>
              Molecular drivers of fusion plasmid: mechanistic insights and evolutionary implications.
              </a>
            </td>
          <td>
            Ziyi Liu, Ying Tang, Mingxing He, Chenggang Xu
          </td>
          <td>2025-08-20</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Targeted hypermutation tools are useful for engineering proteins and pathways, and exploring the evolutionary landscapes. However, existing targeted hypermutation tools for genomic loci mostly exhibit restricted mutation windows and limited mutational types. Here, by integrating mutagenic, high-processivity bacteriophage T5 or T7 DNA polymerases (DNAPs) with CRISPR-Cas9, the study develops an in vivo mutagenesis system that enables all possible types of nucleotide substitutions and an expanded mutation window of up to 2 kilobases, achieving a maximum mutation rate 1.1 × 106-fold higher than wild-type Escherichia coli. Through MS2-mediated recruitment of T5 or T7 DNAP for co-localization with nickase nCas9, off-target rate is reduced by up to 96.8% without compromising on-target rate. Further benefiting from the dTnpB-based transcriptional repression system, the mutagenesis process can be properly regulated during continuous evolution. Finally, the CRISPR-TDNAP-assisted targeted mutagenesis for regulable laboratory evolution (CTRLE) confers cellular triple-antibiotic resistance in 8 days, and enhances the efficiency of the twin-arginine translocation pathway by over threefold in 6 days. Furthermore, CTRLE proves effective in Bacillus subtilis and Kluyveromyces lactis, yielding targeted mutation rates 1.2 × 10⁵-fold and 5 × 10⁷-fold higher than host backgrounds, respectively. Collectively, CTRLE provides an efficient and universal way to accelerate the continuous evolution of different microbial cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/453cdc9bc3fd8236cad5dbabb388b99d7317237d" target='_blank'>
              CRISPR-DNA Polymerase Assisted Targeted Mutagenesis for Regulable Laboratory Evolution.
              </a>
            </td>
          <td>
            Shuaili Chen, Xiangdi Chen, Yifan Peng, Qinghua Li, Jingwen Zhou, Jianghua Li, Guocheng Du, Jian Chen, Guoqiang Zhang
          </td>
          <td>2025-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The remarkable advances in cancer therapies significantly enhance the survival rates and longevity of cancer patients. Among childhood, adolescent, and young adult female cancer survivors, however, anti-cancer agents frequently cause primary ovarian insufficiency (POI), early menopause, and infertility, primarily due to the depletion of the ovarian reserve. Oocytes, the female germ cells, exhibit a notable susceptibility to DNA damage, given that they remain in meiotic arrest of prophase I for prolonged durations from months to years, which increases the risks of accumulating DNA damage overtime. To counteract this, a tightly controlled DNA damage response (DDR) signaling ensures that only oocytes with an intact genome progress to ovulation, fertilization, and next generations. Chemotherapeutic anti-cancer agents, including doxorubicin, cisplatin, cyclophosphamide, along with irradiation, elicit DNA damage via various mechanisms, including DNA crosslinking, single and double-strand DNA breaks, and oxidative stress. The genotoxic insults activate DDR in the oocytes, which detect and repair DNA damage or initiate apoptosis to eliminate impaired oocytes. Although several protein molecules such as DNA damage-sensing kinases, checkpoint kinases, p53 family transcription factors, and pro-apoptotic molecules, have been discovered, the precise mechanisms of DDR in determining the fate of oocytes, particularly how they differ from those in somatic cells and cancer cells, remain poorly understood. From an oncofertility perspective, the current review analyzes the molecular mechanisms of anti-cancer agent-induced DDR in oocytes and discuss knowledge gaps and urgent future research directions for preserving the ovarian reserve, fertility, and endocrine functions of young female cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman (Sonia) Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy, with five-year survival rates below 50% largely due to late-stage diagnosis and the development of drug resistance. The advent of precision medicine offers new opportunities to improve patient outcomes. To date, molecular profiling has identified key therapeutic targets in ovarian cancer, including homologous recombination deficiency (HRD), BRCA mutations, and mismatch repair defects. PARP inhibitors have demonstrated significant clinical benefit in patients with HRD-positive tumors. Also, folate receptor alpha (FRa) targeting antibody drug conjugate, mirvetuximab soravtansine has demonstrated improved overall survival in platinum resistant ovarian cancer expressing high levels of FRa. To exploit DNA replication vulnerabilities in ovarian cancer cells, inhibitors of cell cycle regulators ATR and CHK1, have been studied in ovarian cancer preclinical and clinical settings. DHX9 helicase inhibition represents another novel approach, targeting RNA/DNA helicase activity critical for transcriptional regulation and DNA repair processes in cancer cells. Additional emerging targets include KIF18A that regulates mitotic progression and microtubule dynamics, which have demonstrated selective activity in chromosomally instable high-grade serous ovarian cancer cells. Novel therapeutic targets such as DHX9 helicase and KIF18A, offer new opportunities for patients with limited treatment options. Future directions include expanding biomarker testing, developing combination therapeutic strategies that exploit multiple vulnerabilities simultaneously, and addressing resistance mechanisms.



 Jung-Min Lee. Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c03eea4f891eb2bca142fb8a480f45ccc5bf2a" target='_blank'>
              Abstract IA018: Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="SUMMARY Land plants exhibit remarkable cellular plasticity, readily reprogramming differentiated cells into stem cells in response to internal and external stimuli. While chromatin remodeling is crucial for cellular reprogramming, its interplay with gene expression during reprogramming into stem cells remains elusive. In the moss Physcomitrium patens, wounding induces reprogramming of leaf cells facing wounded cells to change into chloronema apical stem cells through the activation of the AP2/ERF transcription factor STEMIN. In this study, we employed multimodal single‐nuclei RNA and ATAC sequencing to explore the interplay between gene expression and chromatin dynamics during STEMIN‐mediated reprogramming. Profiling 20 883 single‐nuclei from gametophores, protonemata, and cut leaves, we identified 11 distinct cell types including reprogramming leaf cells. Our analysis revealed that reprogramming leaf cells exhibit a partly relaxed chromatin landscape and STEMIN transcription factors selectively enhance accessibility at specific genomic loci essential for stem cell formation. Thus, our results indicate that wounding initiates a broad chromatin relaxation, creating a permissive environment and specific transcription factors act to refine this permissive state by specifically relaxing chromatin regions critical for reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e828665ed01a00e708d118690c03fb784def434e" target='_blank'>
              STEMIN transcription factor drives selective chromatin remodeling for gene activation within a relaxed chromatin during reprogramming in the moss Physcomitrium patens
              </a>
            </td>
          <td>
            Ruan Morné de Villiers, Gergő Pálfalvi, A. Kanai, Yutaka Suzuki, M. Hasebe, Masaki Ishikawa
          </td>
          <td>2025-08-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="


 Tumorigenesis is a multistep process involving germline and somatic alterations, also known as "drivers." Understanding how these alterations cooperate is crucial for assessing cancer risk and developing strategies to intercept malignant transformation.



 To investigate the interplay between germline pathogenic variants (GPVs) and somatic drivers in childhood cancers, and to compare tumor molecular profiles based on the origin of initiating alterations.



 We analyzed whole-genome (paired tumor-germline) and transcriptomic data from 2,397 children enrolled in the ZERO Childhood Cancer Program (2018–2025). A subset of 199 participants with autosomal dominant cancer predisposition syndromes consistent with their tumor type was selected for detailed analysis.



 GPVs were distributed across multiple cancer predisposition genes (CPGs), with TP53, NF1, SMARCB1, and DICER1 being the most prevalent. A second somatic hit in the same CPG was identified in 83% of cases (166/199), 56% resulting from loss of heterozygosity. RNA sequencing revealed additional variants (10% of cases) not detected in the DNA data. Additional somatic drivers were identified in 149 tumors (75%), 82 of which exhibited recurrent drivers for their cancer type (e.g., CDKN2A in NF1-driven malignant peripheral nerve sheath tumor). In 33% of them (27/82), no second hit was present in the CPG (e.g., NF1 GPV in low-grade glioma with somatic FGFR1 mutation), suggesting the GPV may act as a cooperating enabler. Of interest, 67 tumors harbored additional drivers atypical for their cancer type, with most (62/67) showing a second hit in the same CPG, suggesting a distinct molecular profile in this group of GPV-driven tumors (e.g. GPV in FBXW7 and Wilms tumor with somatic CDKN2A loss). In 49 cases, the only somatic event was a somatic second hit in the CPG, indicating that some GPVs may independently initiate tumorigenesis (e.g., SMARCB1, DICER1). This supports the hypothesis that GPVs can enable and promote tumour progression without the need for cooperation with other genomic drivers. Conversely, 25 cases lacked a second hit in the CPG but harbored other somatic drivers, including unexpected tumor alterations (e.g., GPV in RECQL4 and osteosarcoma without TP53 alteration), suggesting possible GPV haploinsufficiency in driving tumorigenesis through an alternative molecular pathway.



 Our study highlights the multifactorial nature of tumour initiation and promotion in GPV-driven tumorigenesis involving direct initiation and cooperation with somatic drivers. Genes like SMARCB1 can drive cancer alone, while others like NF1 need additional somatic alterations for malignancy. Notably, biallelic inactivation of CPG rarely leads directly to high-grade tumors, emphasizing gene and cancer-type specificity of this process. Future research, such as the study of induced pluripotent cancer stem cells, can help clarify the role of GPV and the sequence of multi-step events. This may guide strategies for preventing malignant transformation and enabling earlier detection.



 Noemi A Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, Paulette Barahona, Ashleigh Sullivan, Ann Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vaness Tyrrell, Michelle Haber, Katherine Tucker, Mark J Cowley, David S Ziegler, Mark Pinese, Loretta M S Lau. Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b9cb084afe69c38e9d831c9072216bcdeed64f" target='_blank'>
              Abstract A045: Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort
              </a>
            </td>
          <td>
            N. Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, P. Barahona, Ashleigh Sullivan, A. Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vanessa Tyrrell, Michelle Haber, Katherine Tucker, Mark J. Cowley, D. S. Ziegler, M. Pinese, L. Lau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db6a32592bb8422fc6525fc67d1f14503f9f5052" target='_blank'>
              Lung Adenocarcinoma Just Desserts: An Expanding Pie of Activating Oncogenes or a Layer Cake of Integrated Alterations.
              </a>
            </td>
          <td>
            Minjeong Kim, W. Lamlertthon, Heejoon Jo, Yan Cui, Hyo Young Choi, K. Hoadley, M. Smeltzer, M. Hayward, M. Wilkerson, Liza Makowski, D. Hayes
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Precision oncology is witnessing an increasing number of molecular targets fueled by the continuous improvement of cancer genomics and drug development. Tumor genomic profiling is nowadays (August 2025) part of routine cancer patient care, guiding therapeutic decisions day by day. Nevertheless, implementing and distilling the increasing number of potential gene targets and possible precision drugs into therapeutically relevant actions is a challenge. The availability of prescreening programs for clinical trials has expanded the description of the genomic landscape of gastrointestinal tumors. The selection of the genomic test to use in each clinical situation, the correct interpretation of the results, and ensuring clinically meaningful implications in the context of diverse geographical drug accessibility, economic cost, and access to clinical trials are daily challenges of personalized medicine. In this context, well-established negative predictive biomarkers, such as extended RAS extended mutations for anti-EGFR therapy in colorectal cancer, and positive predictive biomarkers, such as MSI status, BRAF p.V600E hotspot mutation, ERBB2 amplification, or even NTRK1, NTRK2, NTRK3, RET, and NRG1 fusions across gastrointestinal cancers, are mandatory to provide tailored clinical care, improve patient selection for treatment and clinical trials, maximize therapeutic benefit, and minimize unnecessary toxicity. In this review, we provide an updated overview of actionable genomic alterations in GI cancers and discuss their implications for clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a041af7c3fe90ad802a0f68558db067f344766d8" target='_blank'>
              Clinical Actionability of Genes in Gastrointestinal Tumors
              </a>
            </td>
          <td>
            Nadia Saoudi Gonzalez, Giorgio Patelli, G. Crisafulli
          </td>
          <td>2025-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Pleomorphic giant cell adenocarcinoma (PGCA) of the prostate is a rare, aggressive variant characterized by multinucleated giant cells, sarcomatoid features, and resistance to conventional therapies. Despite its recognition in the WHO 2016 guidelines, the molecular drivers and clinicopathological correlates of PGCA remain poorly characterized. This study presents the first integrative clinicogenomic profiling of PGCA, revealing a novel prognostic gene signature with direct implications for diagnosis and treatment. Methods We conducted comprehensive clinicopathological and genomic analyses of a treatment-refractory PGCA case using histology, immunohistochemistry (IHC), whole-exome sequencing (WES), clonal evolution modeling, and multicohort validation. IHC assessed key prostate cancer markers (AR, AMACR, KLK3, PTEN, NKX3-1, VIM), while WES compared somatic alterations in PGCA, adjacent adenocarcinoma, and stromal tissue. Public datasets (prostate_dkfz_2018, prad_tcga, prad_mcspc_mskcc_2020) were used for external validation. Results PGCA displayed profound pleomorphism, necrosis, and complete loss of luminal markers (AR/AMACR/KLK3), along with strong vimentin (VIM) expression, consistent with epithelial–mesenchymal transition. WES revealed PGCA-specific mutations enriched in cell-cycle and inflammatory response pathways, distinct from metabolic alterations in the adjacent adenocarcinoma. Clonal evolution analysis showed divergent progression from a shared ancestral clone. Importantly, mutations in ADAMTS7, CDH1, DRD5, MGAT5, and TP53 emerged as a robust five-gene signature predictive of biochemical recurrence, metastasis, and poor survival, validated across multiple independent cohorts. Conclusion Our study provides the first molecular roadmap of prostatic PGCA to date, establishing a novel five-gene prognostic signature and revealing fundamental insights into its pathogenesis through divergent evolution from conventional adenocarcinoma. These insights offer new opportunities for precise diagnosis, prognostic stratification, and targeted therapeutic strategies for this lethal prostate cancer variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fd48864bfffc9bb6aed51ef499e504db68fa6c" target='_blank'>
              Clonal divergence and genomic meltdown in prostatic pleomorphic giant cell adenocarcinoma
              </a>
            </td>
          <td>
            Xiaoshi Ma, Kun Chen, Jing Zhang, Liming Liu, Jiping Luo, Kaipeng Huang, Hongying Zhang, Danni Liu, Jizhou Gou, Changyin Feng, Xia Zhao, Wanying Li, Lipeng Chen, Li Yin, Xianlin Meng, Zhiqiang Cheng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94f6767c83b49c5f93547ae2e51f853756e7c466" target='_blank'>
              DNA methylation influences human centromere positioning and function.
              </a>
            </td>
          <td>
            Catalina Salinas-Luypaert, Danilo Dubocanin, Rosa Jooyoung Lee, Lorena Andrade Ruiz, Riccardo Gamba, Marine Grison, Léonid Velikovsky, Annapaola Angrisani, Andrea Scelfo, Yuan Xu, Marie Dumont, Viviana Barra, Therese Wilhelm, Guillaume Velasco, Marialucrezia Losito, René Wardenaar, C. Francastel, F. Foijer, G. Kops, K. Miga, Nicolas Altemose, D. Fachinetti
          </td>
          <td>2025-09-04</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Epigenetic regulation plays a key role in the progression, proliferation, and dedifferentiation of thyroid cancer. Epigenetic control occurs at multiple levels, including DNA methylation, RNA modification, histone modification, chromatin remodeling, and chromatin accessibility. Genetic alterations in chromatin regulators are commonly observed in thyroid cancer, which includes papillary thyroid carcinoma (PTC), medullary thyroid carcinoma (MTC), anaplastic thyroid carcinoma (ATC), and follicular thyroid carcinoma (FTC). These cancers exhibit distinct characteristics in terms of genetics, biology, and clinical presentation. Therefore, we review the disease biology driven by changes in chromatin pathways in thyroid cancer. Specifically, we summarize examples of epigenetic dysregulation at each level, with mechanisms involving alterations in enzymes regulating DNA methylation, RNA modification and posttranslational modifications of histones, as well as the loss or fusion of subunits involved in chromatin remodeling and chromatin accessibility. Finally, on the basis of clinical applications, we review the current and potential future approaches for thyroid cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b8e029502363c1309237f110ef671aea68cd77" target='_blank'>
              Epigenetic control in thyroid cancer: mechanisms and clinical perspective
              </a>
            </td>
          <td>
            Jiahui Zhang, Shengkai Zheng, Ruiwang Xie, Junsi Zhang, Xiangjin Chen, Sunwang Xu
          </td>
          <td>2025-08-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) is highly malignant with limited therapeutic targets. Prior studies indicate transglutaminase 2 (TGM2) regulates the tumor microenvironment, but its mechanisms in driving LUSC progression and immune evasion remain unclear.The prognostic value of TGM2 was analyzed using the TCGA LUSC cohort. Key genes were screened via random forest algorithm. Functional validation was performed in NCI-H520 and SK-MES-1 cell lines. Proteomics, GSEA, and TIMER2.0 assessed downstream pathways and immune infiltration. Transcriptional regulatory databases predicted the upstream transcription factors of TGM2, which was then validated by chromatin immunoprecipitation (ChIP)-qPCR.TGM2 was an independent prognostic factor for LUSC. High TGM2 expression correlated with reduced overall survival (OS, P = 0.00018) and disease-free survival (DFS, P = 0.00019). TGM2 promoted proliferation, migration, invasion, clonogenicity, and suppressed apoptosis in LUSC cells by activating p38 MAPK signaling. Elevated TGM2 levels were associated with an immunosuppressive microenvironment: decreased Th1 (R = -0.186, P < 0.0001) and NK cell infiltration (R = -0.116, P = 0.0092), and increased M2 macrophage (R = 0.164–0.528, P < 0.0001) and cancer-associated fibroblast infiltration (R = 0.469, P < 0.0001). NR3C1 was identified as a key transcription factor regulating TGM2. ChIP-qPCR analysis confirmed that NR3C1 binds to a specific site (921-935) within the TGM2 promoter, and their expression showed a strong positive correlation (R = 0.53, P < 0.0001).TGM2 drives LUSC progression via p38 MAPK activation and shapes an immunosuppressive microenvironment, which is transcriptionally regulated by NR3C1. This study supports TGM2 as a prognostic biomarker and suggests its potential as a therapeutic target, which may inform future combination immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac9e9e2ea294419d4bd204ae0bf8bb3759740e" target='_blank'>
              TGM2 regulated by transcription factor NR3C1 drives p38 MAPK-mediated tumor progression and immune evasion in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Chunlong Lin, Shangfu Li, Liang Yi, Hui Zhou, Zhiyi Xiao, Zhuo Yang, Qingping Chen, Xiangyang Peng, Kanao Li, Qing Wang, Wei Liu, Ni Li, Lun Li, Duanlin Du, Qi Xu, Lingge Yang
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background. Studies of comorbid (syntropic) and inversely comorbid (rarely occurring together, i.e., dystropic) diseases have focused on the search for molecular causes of this phenomenon. Materials. We investigated DNA methylation levels in regulatory regions of 23 apoptosis-associated genes as candidate loci associated with the “cancer–neurodegeneration” dystropy in patients with Huntington’s disease (HD) and patients with non–small cell lung cancer (LC). Results. Statistically significant differences in methylation levels between the HD and LC groups were found for 41 CpG sites in 16 genes. The results show that five genes (SETDB1, TWIST1, HDAC1, SP1, and GRIA2) are probably involved in the phenomenon of inverse comorbidity of these diseases. For these genes, the methylation levels of the studied CpG sites were altered in opposite directions in the two groups of patients, compared to the control group. Conclusions. For the SP1 gene, the above hypothesis is supported by our analysis of open-access data on gene expression in patients with the aforementioned diagnoses and fits a probable mechanism of the “HD–LC” dystropy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea115150dd05fa7cf3e3d8dd1a0ac32240561c27" target='_blank'>
              DNA Methylation Status of Regulatory Regions of Apoptosis-Associated Genes in Dystropy «Huntington’s Disease—Non-Small Cell Lung Cancer»
              </a>
            </td>
          <td>
            Nadezhda P. Babushkina, E. Bragina, D. Gomboeva, Iuliia A. Koroleva, S. Illarioshkin, S. Klyushnikov, N. Abramycheva, Maria A. Nikitina, Valentina M. Alifirova, N. Litviakov, M. K. Ibragimova, M. M. Tsyganov, I. A. Tsydenova, Aleksei A. Zarubin, I. Goncharova, M. Golubenko, R. R. Salakhov, A. A. Sleptcov, Aksana N. Kucher, M. S. Nazarenko, Valery P. Puzyrev
          </td>
          <td>2025-08-07</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Endometrial cancer (EC), a type of uterine cancer, is witnessing a global increase in incidence. Despite advancement in diagnosis and treatment, metastatic or recurrent EC often exhibits a poor prognosis, necessitating novel therapeutic strategies. DEAD-box helicase 17 (DDX17) is implicated in several cancers. Our study aimed to uncover the biological function and molecular mechanism of DDX17 in EC. Methods EC and matched adjacent normal tissues from 80 patients were analyzed; DDX17 mRNA/protein expression was quantified via RT-qPCR and immunoblotting in clinical specimens and cell lines (HEC-1A, HEC-1B, Ishikawa), with functional assays (proliferation/migration/invasion) performed following DDX17 overexpression in vitro, while xenograft modeling in BALB/c nude mice enabled in vivo validation through immunofluorescence and immunohistochemical staining; mechanistic studies employed RNA immunoprecipitation (RIP-PCR), m6A-specific RNA immunoprecipitation (MeRIP-PCR), and protein interaction analyses. Results DDX17 was significantly downregulated in EC tissues/cells, correlating with poor prognosis in clinical cohorts. Overexpression of DDX17 suppressed tumorigenesis both in vitro and in vivo through PI3K/AKT pathway inactivation. METTL3-mediated m6A modification stabilized DDX17 mRNA, with IGF2BP2 specifically recognizing m6A-modified transcripts. Critically, METTL3 ablation reversed DDX17 stabilization and abolished its tumor-suppressive effects, while PI3K inhibition (LY294002) phenocopied METTL3 restoration in rescuing DDX17 deficiency-induced oncogenicity. Conclusion METTL3-mediated m6A modification stabilizes DDX17 to suppress EC cell proliferation, migration, and invasion through an IGF2BP2-dependent mechanism by inactivating the PI3K/AKT pathway. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc74064b9143aa7c166a780986e41cb0e801d2f4" target='_blank'>
              METTL3 stabilizes DDX17 mRNA via IGF2BP2-mediated m6A modification to suppress endometrial cancer progression
              </a>
            </td>
          <td>
            Yuan Zhang, Mengjun Liu, Jing Liu, Jing Zhang
          </td>
          <td>2025-08-07</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Significant group-based variations in prostate cancer (PCa) outcomes represents a critical healthcare challenge in the USA. African American (AA) men face the highest incidence of PCa and are twice as likely to succumb to their disease compared to their Caucasian American (CA) counterparts. This disparity stems from a complex interplay involving social, environmental, and biological factors. While inequitable access to quality healthcare is a key contributor, even in settings with equal care, AA patients more often present with advanced disease at diagnosis, suggesting unique tumor biology. While some understanding of the epigenomic landscape has been recently identified, the full extent of global chromatin remodeling contribution towards differences in health outcomes across population groups remains poorly understood. To better elucidate how chromatin remodeling and gene regulation influences PCa development across CA and AA population groups, we employed single-cell spatial transposase-accessible chromatin technology followed by sequencing (scATAC-seq) and H3K27ac spatial-CUT&Tag using samples from men who underwent prostatectomy at the Center for Prostate Disease Research/Walter-Reed National Military Medical Center. Our current analysis revealed significantly greater chromatin accessibility in AA men compared to CA men. Leveraging a machine learning-based computational cell typing method called Cellcano to annotate scATACseq data, enabled the identification of unique cell types and distinct gene score signatures enriched in AA samples compared to CA counterparts. Furthermore, AA demonstrated significant lower accessible chromatin associated with CCCTC-binding factor (CTCF) which is a critical zinc finger protein crucial for regulating gene expression, epigenetic marks, and maintaining 3-dimensional genome structure. We have validated this finding by CTCF immunohistochemistry in an independent TMA that includes 99 patients with distribution of CA and AA prostatectomy samples. Moreover, integration of H3K27ac profiling revealed several differentially regulated enhancer regions near specific genes across samples. Collectively, our current findings provide critical insights into the divergent chromatin accessibility profiles between AA and CA men with localized PCa, which likely underlie distinct transcriptional responses that can determine therapeutic resistance and tumor progression. These data shed light on the complex epigenomic mechanisms driving the group-based variations in PCa outcomes and may inform the development of personalized treatment strategies to address this critical healthcare challenge.



 Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, Sally Elsamanoudi, Colin Ng, Gregory Chesnut, Leigh Ellis. Analysis of the chromatin accessibility landscape in patients with localized prostate cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f758d1a482aa1ee0457d64a79f2bbfa79c84e92e" target='_blank'>
              Abstract C087: Analysis of the chromatin accessibility landscape in patients with localized prostate cancer
              </a>
            </td>
          <td>
            Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, S. Elsamanoudi, Colin Ng, Gregory T. Chesnut, Leigh Ellis
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are promising biomarkers for cancer diagnosis and prognosis due to their ability to carry specific biomolecular cargo, including DNA. However, the clinical utility of DNA methylation-based liquid biopsies using EV-DNA remains underexplored. The low quantity and relatively long length of EV-DNA complicate whole-genome methylation profiling. To address this, we develop Tn5-assisted Enzymatic Methyl-sequencing with Post-conversion Tailing (TEMPT), a bisulfite-free whole-genome profiling method for EV-DNA. TEMPT employs single-adapter Tn5 tagmentation, enzymatic conversion of unmodified cytosines, and post-conversion tailing to generate high-depth whole-genome EV-DNA methylomes. We apply TEMPT to EV-DNA from 58 gastric cancer and polyp samples, generating methylomes from sub-nanogram inputs and identifying differentially methylated regions (DMRs) that distinguish cancer from controls. We identify potential cancer biomarkers through DMR-associated genes, highlighting the roles of EVs in cellular communication. Our findings suggest that immune cells may serve as an alternative source of EV-DNA. This approach holds significant promise for advancing EV-DNA research and its applications in early disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad67a59c8fc16bd26c5fe5c57db4a838137c7af1" target='_blank'>
              Whole-genome methylation profiling of extracellular vesicle DNA in gastric cancer identifies intercellular communication features
              </a>
            </td>
          <td>
            Bingqian Lin, Zhenna Jiao, Shouquan Dong, Weikai Yan, Jinting Jiang, Yanfang Du, Xiaocheng Weng, Hongling Wang, Zhiyuan Hu, Yibin Liu, Xiang Zhou
          </td>
          <td>2025-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway.
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) is the most common hematological malignancy. More than half of DLBCL patients achieve long‐term remission after treatment, but a third relapse after conventional Rituximab (R)‐based chemotherapy regimens, such as CHOP (cyclophosphamide, doxorubicin, vincristine and prednisone). Cancer cells are exposed to chronic replication stress, which impedes the duplication of their genome. Functional DNA repair pathways are therefore important for the survival of cancer cells. This dependence can be exploited therapeutically to hamper repair of the intrinsic DNA damage occurring during replication or to exacerbate DNA damage induced by chemotherapy. Using CRISPR‐Cas9 screening, we identified CHEK1, WEE1, ATR and RAD51 DNA repair factors as essential genes in DLBCL cells. According to these results, we investigated the effect of small molecules targeting DNA replication stress and DNA repair mechanisms, alone or in combination with the R‐CHOP genotoxic agents, cyclophosphamide and doxorubicin. Applying a standard threshold of 2 SDs below the IC50 of the genotoxic agent alone, a total of 3 synthetic lethal combinations have been identified including cyclophosphamide with CHK1/2 inhibitor, cyclophosphamide and ATR inhibitor and doxorubicin with DNAPK inhibitor. Co‐treatment with these molecules led to cell death, DNA damage induction and cell cycle arrest in DLBCL cells more efficiently than genotoxic agents alone. These data have been validated using primary DLBCL cells from patients. Our results open new perspectives for therapeutic approaches exploiting the synthetic lethality of genotoxic agents with DNA repair inhibitors to improve the therapeutic outcome of patients with DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ea8280a7e78b4aa4b06c11d4f0a4eaa409f0ab" target='_blank'>
              Synthetic Lethal Combinations of DNA Repair Inhibitors and Genotoxic Agents to Target High‐Risk Diffuse Large B Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Ovejero, Julie Devin, Laura Alibert, Camille Soun, Yea-Lih Lin, Laure Dutrieux, M. Abouladze, Elvira Garcia de Paco, Ouissem Karmous Gadacha, A. Constantinou, Guillaume Cartron, C. Herbaux, O. Elemento, Philippe Pasero, Sandrine Roulland, J. Moreaux, C. Bret
          </td>
          <td>2025-08-23</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb04dfc69e03ddcb4aa348ab8eef9592fa55873" target='_blank'>
              p300/CBP is an essential driver of pathogenic enhancer activity and gene expression in Ewing sarcoma.
              </a>
            </td>
          <td>
            Laura C Godfrey, Brandon Regalado, Sydney R Schweber, Charlie Hatton, Daniela V. Wenge, Yanhe Wen, Meaghan Boileau, Maria Wessels, Jun Qi, Christopher J Ott, K. Stegmaier, Miguel N Rivera, Scott A Armstrong
          </td>
          <td>2025-09-01</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) evolves as an aggressive phenotype during prolonged androgen deprivation therapy, lacking effective clinical management. Here, we elucidated a reciprocal metabolic-epigenetic mechanism involving a positive feedback loop between glycolysis and the transcription factor LHX2 that promotes PCa progression. Mechanistically, enzalutamide-mediated inhibition of the androgen receptor (AR) led to upregulation of key glycolytic enzymes. Elevated glycolysis resulted in lactate accumulation and subsequent histone lactylation, which in turn enhanced LHX2 expression. Reciprocally, LHX2 transcriptionally activated the lactate dehydrogenase LDHA, which further amplified lactate production. Furthermore, LHX2 augmented the expression of the lineage-determining enzyme DNMT1, potentiating neuroendocrine gene expression and tumor growth. Significantly, the antiviral agent paritaprevir could directly bind to LHX2, effectively suppressing neuroendocrine marker expression and tumor progression. These findings uncover a potential role for LHX2 in orchestrating NEPC development, offering putative therapeutic targets for treating this aggressive cancer phenotype and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef2fefc98e5fd3352706deab4254aa3b33bda2c" target='_blank'>
              LHX2 Rewires the Metabolic and Epigenetic Landscape to Drive Tumor Progression in Prostate Cancer.
              </a>
            </td>
          <td>
            Jun Jiang, Shaojie Liu, Chao Xu, Limin He, Hongji Li, Yike Zhou, Zhengxuan Li, Yu Li, Fa Yang, Yuchen Wei, Tong Lu, Keying Zhang, Jingliang Zhang, Pang Wang, Li Guo, Changhong Shi, Weihong Wen, Rui Zhang, Donghui Han, Weijun Qin
          </td>
          <td>2025-09-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although intragenic CpG dinucleotides are conserved during evolution, they are also sensitive to methylation-dependent mechanisms. Methylation status of the TP53 introns 1, 3 and 4 have been analysed in stage III ovarian carcinoma (OC). In the present study, the methylation of exon 4 (10 CpG pairs) was analysed in advanced-stage OC to investigate TP53 methylation and compare exon and intron 4 methylation patterns. A total of 80 samples from patients with advanced-stage OC and metastatic lesions were examined, along with 80 samples derived from healthy patients who had never been diagnosed with cancer. Methylation analysis of the human A2780 ovarian cancer cell line was also performed. Exon and intron 4 were methylated in OC, corresponding metastases and paired healthy tissue. The DNA from the human A2780 ovarian cancer cell line and the normal samples from healthy subjects was also methylated. The data indicate the existence of an intragenic mechanism of regulation of TP53 activity that involves demethylation/methylation processes. This mechanism provides the ability to alter the response from cell cycle arrest to apoptosis by manipulating only the expression of long or short p53 isoforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba65eba61b813922b785bd71652e055bf783b2a" target='_blank'>
              Exon 4 and intron 4 TP53 are both methylated in advanced-stage ovarian carcinomas
              </a>
            </td>
          <td>
            Wiktor Szewczuk, Oksana Szewczuk, Krzysztof Czajkowski, R. Gromadka, Maciej Waledziak, Marek Semczuk, Andrzej Semczuk
          </td>
          <td>2025-09-12</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Some selfish genetic elements drive at meiosis to achieve transmission distortion, breaking the rules of Mendelian segregation to enhance their own evolutionary success. It has been shown that enhancers of drive must act in cis in order to gain the selfish benefit of drive and that suppressors of drive will be selected at unlinked loci. Here, we model the evolution of an autosomal trans-acting gene (Distorter) that causes the Y chromosome (or even 0 chromosome) to drive without driving itself, a phenomenon we call 'remote-control meiotic drive'. We show that such a gene may spread in the population when linked to a second locus, Assister, whose alleles are transmitted at different frequencies through sperm as compared to eggs, for which we consider various scenarios, such as sexually antagonistic selection or sex-limited drive. Depending on the mechanistic details of sex-chromosome drive, Distorter's spread can additionally facilitate transitions between XY and X0 sex determination. Our results provide a proof of principle that stretches the current understanding of segregation modifier and sex allocation theory. Moreover, we identify alternative evolutionary trajectories that could also lead to remote control drive and discuss its potential applications in developing synthetic sex-ratio-distorting elements for use in pest management, for example.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e8563304f91f2028efd43ad9bed1b9ed15be83" target='_blank'>
              Autosomally-encoded segregation distortion of sex chromosomes.
              </a>
            </td>
          <td>
            Naomi L. Greenberg, M. Patten, M. Schenkel
          </td>
          <td>2025-08-01</td>
          <td>Proceedings. Biological sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="With the increase in dog life expectancy, canine mammary cancer has become the most critical and frequent neoplasm in dogs. As in humans, this type of cancer has spontaneous development and is influenced by several risk factors, such as age and hormonal exposure in addition to genetic and epigenetic factors. Thus, knowing that epigenetic patterns influence a variety of biological mechanisms, such development cancer, this study aimed to correlate the methylation pattern of LINE-1 elements in healthy and neoplastic canine mammary tissue, as well as in isolated cell lines of carcinoma canine with tumoral aggressiveness degree. To this end, a retrospective study was conducted in which genomic DNA was extracted from histological sections of biopsies and cell lines using the phenol/chloroform method. To determine the percentages, the genomic DNA was pretreated with methylation-sensitive endonucleases and then amplified by qPCR with primers specific for the L1td1 gene. The results revealed that both the tumor cell lines and the canine adenoma and inflammatory carcinoma samples are hypomethylated relative to normal tissue. However, it was not possible to confirm the relationship between the percentage of LINE-1 hypomethylation and tumor aggressiveness in canine breast cancer. Therefore, the combined analysis of our results allows us to conclude that the methylation pattern of LINE-1 elements can be used as a molecular biomarker to differentiate normal canine mammary tissue from neoplastic canine mammary tissue. However, to confirm whether there is a relationship between the degree of methylation of LINE-1 elements and tumor staging, further studies with canine samples are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3e474a09cd5273ac4d9d3cd5402967aa34da4d" target='_blank'>
              ASSOCIATION BETWEEN LINE-1 HYPOMETHYLATION IN CANINE MAMMARY TUMOR TISSUES: IMPLICATIONS FOR DIAGNOSIS AND RESEARCH
              </a>
            </td>
          <td>
            Patrícia A. dos Santos de Carvalho Joaquim, José Guilherme Xavier, M. Lallo, Rodrigo A. Silva
          </td>
          <td>2025-09-24</td>
          <td>Revista CPAQV - Centro de Pesquisas Avançadas em Qualidade de Vida</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f48f2bc782ec8e2f8bb67fc1e74e0cf830d802" target='_blank'>
              KPNA2 silencing sensitizes triple-negative breast cancer to chemotherapy by promoting multipolar division and suppressing DNA damage repair.
              </a>
            </td>
          <td>
            Yufan Cai, Jun-Ze Du, Haiyu Wang, Lei Shen, Zujing Xu, Yizhou Zhaoxiong, Zheng Gong, You Zhu, Chuxun Wu, Jialiang Cai, Peiling Zhang, Shiping Chen, Zhi Dai, Run Huang, Wei Zhu
          </td>
          <td>2025-09-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is the most common pediatric liver malignancy. However, its cellular origin and molecular drivers remain poorly defined. Using single-nuclear RNA sequencing (snRNA-seq), we identified a proliferative, hepatocyte-derived tumor cell population (cycling HepT) enriched for Enhancer of Zeste Homolog 2 (EZH2) expression, particularly in the aggressive embryonal subtype. Integrative genomic and transcriptomic profiling confirmed EZH2 overexpression. Disruption of the PRC2 complex was evident through mislocalization and reduced expression of SUZ12, a core component. EZH2 overexpression correlated with upregulation of mitotic regulators such as AURKB and Ki67 in human HB gene expression analysis as compared to background liver. Targeted sequencing identified variants of uncertain significance in EZH2 and SUZ12 in 11 of 11 patient tumors. Pharmacologic inhibition of EZH2 with EPZ-6438 reduced proliferation and sensitized HB cells to cisplatin through gene regulation, potentially modulating platinum accumulation both in vitro and in vivo. In summary, EZH2 promotes HB progression through epigenetic silencing and noncanonical signaling pathways. These findings support EZH2’s contribution to HB pathogenesis, therefore identifying it as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b140db08052390107c208976008407b428e0ef2" target='_blank'>
              Oncogenic Role of Aberrant EZH2 in Hepatoblastoma
              </a>
            </td>
          <td>
            Kathryn Glaser, Lara Berklite, Brian T Hickner, Priyanka Rao, R. Patel, Andrew A. Badachhape, Somak Roy, Emily J Schepers, Nikolai Timchenko, James I Geller, S. Ranganathan, Greg Tiao, Bruce J Aronow, Takanori Takebe, Sarah E. Woodfield, Sanjeev A. Vasudevan, E. DePasquale, Alex J Bondoc
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract While the majority of viruses falls within a handful of well-defined realms, many novel viruses and virus-like genetic elements remain to be discovered. Given the abundance and diversity of small, circular DNA genomes in animal gut viromes, we hypothesized that viruses of similar size might have been overlooked in other environments. Re-analyzing existing ocean virome datasets for small viruses, we discover a novel group of circular mobile genetic elements of ∼4.6kb in size, likely with double-stranded DNA, which we term Charybdis elements. Charybdis elements form a singular group based on cohesive gene content, encode around 10 hypothetical genes but apparently contain no virus, transposon or plasmid associated hallmark proteins. They are detected exclusively in marine datasets ranging from surface water to hydrothermal vents and sediments, from the Arctic to the Antarctic. Virome datasets separating virus-like particles by density gradients show that Charybdis elements accumulate in the fraction representing extracellular vesicles and small, tail-less viruses that are often associated with lipid membranes. One specific clade of Charybdis elements is shown to integrate into the genomes of the common marine group II archaeon candidate order Poseidoniales, the putative host of these elements. Analysis of thousands genomes shows a common genome architecture consisting of two modules: An accessory module containing a diverse range methyltransferases and other genes likely involved in host interactions, and a more conserved (putative) structural module. The structural module encodes proteins that are confidently predicted to form striking tri- or pentameric structures. These structures show stark resemblance and in some cases homology to protrusion or tail-elements on the capsids of viruses from the realms Varidnaviria and Duplodnaviria . Given these traits, Charybdis elements likely represent a hitherto undiscovered group of virus or virus-satellites, with at least one protein of a novel double-Greek-key fold serving as a potential capsid element. However, since their hosts remain unculturable, the true nature of these ubiquitous marine elements remains as of yet unresolvable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4b273a6552706a198592704ffd63db0684ca2fe" target='_blank'>
              A novel mobile genetic element with virus-like characteristics is widespread in the world’s oceans
              </a>
            </td>
          <td>
            P. Kirchberger, Antoni Luque
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Methylation of DNA CpG domains in cellular DNA is a key mechanism of epigenetic regulation. Disruptions in the processes maintaining DNA methylation can lead to diseases like cancer. The radiation response of certain cancer cells may be affected by their DNA methylation levels, which may have consequences in their response to radiotherapy. In this work, we utilized DNA origami nanotechnology to examine whether DNA methylation impacts DNA response to ionizing radiation in solution before biological processes come into play. Our findings reveal that a protective effect is achieved with just a few methylated CpG adducts. Both low-LET (electron) and high-LET (carbon ion) irradiation show a reduced lesion count in methylated DNA, as indicated by qPCR results. AFM single-molecule observations using DNA origami nanoframes suggest fewer double-strand breaks in methylated DNA after carbon ion irradiation. This radioprotective effect may contribute to the differential radiation response of cellular DNA and should be considered when predicting and evaluating DNA radiation damage yields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60446411a5ce1e0fa6ca09e48e25c2f668acc1a4" target='_blank'>
              CpG Methylation Protects DNA against Ionizing Radiation
              </a>
            </td>
          <td>
            Leo Sala, Tereza Zápotocká, Jana Šáchová, Václav Olšanský, David Chvatil, François Chevalier, Violaine Vizcaino, Alain Mery, J. Kočišek
          </td>
          <td>2025-08-20</td>
          <td>The Journal of Physical Chemistry. B</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Recombinases are essential enzymes in synthetic biology and biomedical research, enabling site-specific DNA modifications for applications such as gene therapy, the generation of transgenic models, and the construction of genetic circuits. Recombination efficiency depends on several factors, including intracellular recombinase concentration and the growth phase of the host cells. Although recombination is typically studied during exponential growth, the effects of stationary-phase dynamics on recombinase activity remain poorly understood. In this study, we examine how bacterial growth phase influences recombinase-mediated DNA modifications, using the serine recombinase Bxb1 as a model. We engineered a genetic system in Escherichia coli to quantify intracellular Bxb1 levels and to measure recombination efficiency across different growth phases. Our results reveal a quasi-linear relationship between recombinase concentration and recombination efficiency during exponential growth, up to a saturation point. Notably, recombination continues in the stationary phase following recombinase induction in exponential phase, despite the decline in plasmid gene expression. Cells that undergo recombination during the stationary phase show significantly higher recombination efficiencies upon re-entering exponential growth than those maintained in exponential phase throughout. These findings highlight the importance of induction timing in optimizing recombinase-based genetic modifications. Specifically, inducing recombinase expression just before the onset of stationary phase can enhance recombination efficiency while minimizing the need for high expression levels. Moreover, the observed quasi-linear relationship during exponential growth provides a framework for tuning gene expression with precision. Overall, this work offers new insights into leveraging bacterial growth dynamics to improve the design and control of synthetic genetic systems. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17024-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df10f0c8f4691b2c66e4226a8d26ecc0ab6dc10" target='_blank'>
              Characterization of recombinase activity across cellular growth phases
              </a>
            </td>
          <td>
            M. Gonzalez-Colell, J. Macía
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation, a key epigenetic modification, regulates gene expression and diverse cellular functions. Bisulfite sequencing (BS) remains the gold standard for methylation detection, while PacBio HiFi sequencing enables direct detection without chemical conversion. Although both technologies are increasingly used, few studies have directly compared their concordance, particularly in clinically relevant settings such as Down syndrome (DS). We performed a comparative analysis of DNA methylation profiles using whole-genome bisulfite sequencing (WGBS) and PacBio high-fidelity (HiFi) whole-genome sequencing (WGS) in a pair of monozygotic twins with DS. WGBS data were processed with two pipelines, wg-blimp and Bismark, while HiFi WGS data were analyzed using pb-CpG-tools. Our analysis focused on four key aspects: CpG site detection, genomic distribution of methylated CpGs (mCs), average methylation levels, and inter-platform concordance. HiFi WGS detected a greater number of mCs—particularly in repetitive elements and regions with low WGBS coverage—while WGBS reported higher average methylation levels than HiFi WGS. Both platforms exhibited methylation patterns consistent with known biological principles, such as low methylation in CpG islands, and the relative methylation patterns across genomic features were largely concordant. Pearson correlation coefficients indicated strong agreement between platforms (r ≈ 0.8), with higher concordance in GC-rich regions and at increased sequencing depths. Depth-matched comparisons and site-level down-sampling revealed that methylation concordance improves with increasing coverage, with stronger agreement observed beyond 20 × . Our findings support the reliability of HiFi WGS for methylation detection and highlight its advantages in regions that are challenging for bisulfite-based methods. This study demonstrates that HiFi WGS can serve as a robust alternative for genome-wide methylation profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c365104ac077b0a1dea7a1e5e6423a3a137296f2" target='_blank'>
              A comparison of DNA methylation detection between HiFi sequencing and whole genome bisulfite sequencing in monozygotic twins with Down syndrome
              </a>
            </td>
          <td>
            Kanyanee Promsawan, C. Srichomthong, M. Pongpanich, V. Shotelersuk
          </td>
          <td>2025-08-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Despite numerous studies investigating intratumoral microorganisms and their significant roles in cancer initiation, progression, and treatment efficacy, a systematic understanding of intratumoral microorganisms remains lacking. Herein, we conducted a study using 16S rDNA data on seven types of cancer, comprising a total of 783 samples. It’s worth noting that Pseudomonas, Streptococcus, and Prevotella were found to be shared with the microbial communities of the seven cancers, suggesting that these may be associated with the occurrence and development of cancers. We anticipate establishing a foundation for related research and exploring potential methods for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80274119fe0660855f0a9a793c84d8230ca38ecb" target='_blank'>
              Pan-cancer analysis reveals intratumoral microbial diversity in multiple cancers by amplicon technology
              </a>
            </td>
          <td>
            Xudong Liu, Yuteng Yao, Aoyi Xiao, Dingyan Cao, Jingheng Zhang, Yanan Shi, Qing Zhong, Zilong He, Wenming Wu
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Rad51 is a pivotal DNA repair protein that plays a central role in homologous recombination repair. Its significance in tumorigenesis and cancer treatment resistance has garnered increasing attention. In recent years, studies have shown that Rad51 not only plays an important role in the biological characteristics of tumor cells but is also closely related to tumor immunity. The complexity of the tumor immune microenvironment makes the action mechanism of Rad51 more complex and affects the immune system's ability to recognize and clear tumors by influencing the interaction between tumor cells and immune cells. Although preliminary studies have suggested a potential role of Rad51 in tumor immunity, the underlying molecular mechanisms require further elucidation. Furthermore, the research on Rad51 as a therapeutic target is also continuously deepening, indicating its application potential in tumor treatment. In summary, this review aims to comprehensively analyze the role of Rad51 in tumor immunity and its challenges and opportunities in future tumor treatment research and clinical application in order to provide new ideas for clinical tumor immunotherapy. However, the current understanding of Rad51's immunomodulatory functions remains incomplete, and further preclinical and clinical studies are needed to validate its therapeutic potential, address potential off‐target effects, and elucidate tumor‐type‐specific mechanisms and microenvironmental interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00a9c3b142aaabf70111312c22aa00050983254" target='_blank'>
              Rad51 at the Crossroads: Bridging DNA Repair, Tumorigenesis, and Immune Regulation
              </a>
            </td>
          <td>
            Zhendong Qin, Mingjun Lu, Jingwei Guo, Chenyang Wang, Teng Ma
          </td>
          <td>2025-09-24</td>
          <td>Med Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many pathogenic fungi display 'two-speed genome', with the fast-evolving genomic compartments enriched with repetitive sequences, particularly the transposons, which have been shown to drive the variation of pathogenicity-associated genes. Supernumerary chromosomes (SCs) are known to facilitate genomic variation in fungal pathogens, but their specific role in such processes remains understudied. In this study, we assessed the transferability of SCs between asexual Magnaporthe oryzae strains during co-culture and co-infection, and investigated their role in genome reconstruction through experimental evolution assays. We found that SCs could be horizontally transferred between M. oryzae strains and revealed frequent structural variations facilitated by SCs, including deletions, duplications, translocations, and SC-core chromosome recombinations during and after horizontal transfer. Remarkably, all observed intra- and inter-chromosome rearrangements were confined to core chromosome ends and SCs, indicating a robust role of SCs in facilitating genetic exchange within fast-evolving genomic compartments. Additionally, SC carrying the avirulence gene AvrPikE modulates M. oryzae virulence against Pikh rice through horizontal transfer, loss of whole SC, and segmental deletions. Our findings establish SCs as critical players in shaping the diversity and dynamics of the pathogenic fungal genomes, highlighting them as a cradle for the variation of pathogenicity-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fc7ba98841d16bfd287154023ca3410a86f26c" target='_blank'>
              Experimental insights into genome reconstruction driven by horizontal transfer of supernumerary chromosomes in Magnaporthe oryzae.
              </a>
            </td>
          <td>
            Zhenyu Fang, Yuyong Li, Jianqiang Huang, Meilian Chen, Xiali Chen, Xueting Mo, Z. Zhong, Xiuxiu Li, G. Lu, Guifang Lin, Zonghua Wang, Huakun Zheng
          </td>
          <td>2025-08-03</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary This study evaluated 479 advanced prostate cancer patients who had a baseline and at least one subsequent circulating tumor DNA (ctDNA) test performed. New and potentially actionable genomic alterations emerged in more than half of patients, revealing possible targets for on-label therapy, off-label therapy, and clinical trials. These findings underscore the potential of serial ctDNA profiling to guide personalized therapy and clinical trial selection by offering dynamic insights into genomic evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7379b6d0838988ee41b415fc0330595914905d" target='_blank'>
              Evolution of Potentially Actionable Genomic Alterations in Advanced Prostate Cancer: A Real-World Analysis of Serial Circulating Tumor DNA Testing
              </a>
            </td>
          <td>
            Miguel Muniz, L. J. Tsai, J. Orme, L. Bucheit, S. Basourakos, Nancy Wei, Regina M. Koch, Zachary Scharf, Sounak Gupta, Adam M. Kase, Rodrigo Rodrigues Pessoa, I. Riaz, Eugene D. Kwon, Jack R. Andrews, Daniel S. Childs
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) ranks as the fifth most prevalent malignancy worldwide. Disruptions in copper homeostasis adversely affect liver function. Cuproptosis, a recently defined form of regulated cell death triggered by intracellular copper accumulation, disrupts the tricarboxylic acid cycle and mitochondrial respiration. However, the specific roles and mechanisms of cuproptosis-related genes (CRGs) in HCC pathogenesis remain incompletely understood. Materials and methods We systematically evaluated the expression of 10 CRGs in HCC tissues versus adjacent normal tissues. Bioinformatics analyses included Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment, and Gene Set Enrichment Analysis (GSEA) were performed. Immune infiltration levels within the tumor microenvironment were assessed. The prognostic significance of CDKN2A was evaluated using Kaplan-Meier (KM) survival analysis and univariate/multivariate Cox proportional hazards regression. CDKN2A protein expression was validated using immunohistochemistry (IHC). Results CDKN2A was significantly overexpressed in HCC compared to normal tissues. Bioinformatics analyses implicated CDKN2A in DNA replication, organelle fission, and cell cycle checkpoint signaling. Immune-related analysis revealed that high CDKN2A expression correlated positively with dendritic cell (DC) and Th2 cell infiltration, but negatively with CD8 + T cell and natural killer (NK) cell infiltration. KM analysis demonstrated that high CDKN2A expression predicted significantly shorter overall survival in HCC patients. Univariate and multivariate Cox regression identified CDKN2A as an independent prognostic risk factor. Conclusions This study demonstrates that CDKN2A is significantly overexpressed in HCC and plays a role in immune microenvironment modulation. CDKN2A serves as a promising independent prognostic biomarker for HCC, associated with poorer patient survival. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03496-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e16599e64b7443664631899640a1dd84fe24a22" target='_blank'>
              Systematic analysis identifies CDKN2A as a prognostic biomarker for hepatocellular carcinoma
              </a>
            </td>
          <td>
            Shilin He, Yanling Zhang, Junjie Xu, Xiao Liang
          </td>
          <td>2025-08-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, Hisato Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Detection of cancer biomarkers at the earliest stages of disease progression is commonly assumed to extend the overall quality of life for cancer patients as the result of earlier clinical management of the disease. Therefore, there is an urgent need for the development of standardized, sensitive, robust, and commonly available screening and diagnostic tools for detecting the earliest signals of neoplastic pathology progression. Recently, a new paradigm of cancer control, known as multi-cancer detection (MCD), evolved, which measures the composition of cancer-related molecular analytes in the patient’s fluids using minimally invasive techniques. In this respect, the “Holy Grail” of cancer researchers and bioengineers for decades has been composing a repertoire or molecular sensing probes that would allow for the diagnosis, prognosis, and monitoring of cancer diseases via their interaction with cell-secreted and cell-associated cancer antigens and biomarkers. Therefore, the current trend in screening and detection of cancer-related pathologies is the development of portable biosensors for mobile laboratories and individual use. Phage display, since its conception by George Smith 40 years ago, has emerged as a premier tool for molecular evolution in molecular biology with widespread applications including identification and screening of cancer biomarkers, such as Circulating Cellular Communication Network Factor 1 (CCN1), an extracellular matrix-associated signaling protein responsible for a variety of cellular functions and has been shown to be overexpressed as part of the response to various pathologies including cancer. We hypothesize that CCN1 protein can be used as a soluble marker for the early detection of breast cancer in a multi-cancer detection (MCD) platform. However, validated probes have not been identified to date. Here, we screened the multi-billion clone landscape phage display library for phages interacting specifically with immobilized CCN1 protein. Through our study, we discovered a panel of 26 different phage-fused peptides interacting selectively with CCN1 protein that can serve for development of a novel phage-based diagnostic platform to monitor changes in CCN1 serum concentration by liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a4d4d24d30116defeafa45da681d3d2782a86b6" target='_blank'>
              Discovery of Landscape Phage Probes Against Cellular Communication Network Factor 1 (CCN1/Cyr61)
              </a>
            </td>
          <td>
            J. W. Gillespie, Valery A Petrenko
          </td>
          <td>2025-09-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis.
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objective DNA methylation is a key regulator of gene expression and plays a crucial role in cancer development. However, the mechanisms driving gene-specific methylation remain unclear. This study investigates the role of microRNAs (miRNAs) in regulating promoter methylation of specific genes, aiming to uncover miRNA-driven modulation of gene methylation in cancer. Methods We analyzed data from the Cancer Cell Line Encyclopedia (CCLE) database, comprising 813 cell lines. Spearman's rank correlation was performed between the expression levels of 734 miRNAs and the methylation levels of 20,587 genes, focusing on CpG islands in promoter regions. Linear regression analysis was used to validate the relationship between selected miRNAs and gene clusters. Bioinformatics screening identified statistically significant miRNA-gene pairs involved in promoter methylation. Results The analysis revealed 25 target genes whose promoter methylation was significantly associated with the expression of four miRNAs (hsa-miR-200a, hsa-miR-200b, hsa-miR-200c, and hsa-miR-141). These correlations were most pronounced in colorectal, gastric, lung, and ovarian cancers. Notably, cancer-related genes such as ST14, OVOL1, and EPCAM were identified as targets, supporting the hypothesis that miRNAs regulate promoter methylation in these genes. Conclusion Our findings suggest that specific miRNAs induce promoter methylation in cancer-related genes, thereby influencing gene expression. This study expands our understanding of the role of miRNAs in tumor development and highlights the potential of miRNA-based therapies in cancer treatment. As this is a computational study, further experimental validation is required to confirm the proposed regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb2407a7f42df96a4a85bf22be44148df4c8633" target='_blank'>
              Investigating miRNA-driven DNA methylation: Statistical evidence of gene-specific modulation
              </a>
            </td>
          <td>
            Seyeon Jeon, H. Jun, Ji-Young Lee, C. O. Sung, Sung-Min Chun
          </td>
          <td>2025-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition.
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="SOX2 is a potent oncodriver for various squamous cancers, but the underlying mechanism is largely unknown. Here we uncover a role of SOX2 in promoting global histone acetylation in esophageal squamous cancer cells (ESCCs). Mechanistic studies reveal that SOX2 promotes global histone acetylation in an AKT-independent manner, and does so by promoting histone acetylation at both SOX2 binding and non-SOX2 binding sites, and accounts for the formation of about half of the super-enhancers. Combined metabolic and transcriptional analyses reveal two mechanisms by which SOX2 enhances global histone acetylation: promoting the expression of multiple histone acetyltransferases and reducing acetyl-CoA consuming fatty acid synthesis in part by repressing the expression of ACSL5. Finally, SOX2 expression correlates negatively with ACSL5 and positively with histone acetylation in clinical esophageal squamous tumors. Altogether, our study uncovers a role of SOX2 in reprogramming lipid metabolism and driving histone hyperacetylation and super-enhancer function, providing mechanistic insights of SOX2 acting as a potent oncodriver.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddaae346de80ff35e3e8efa77cec7316c62e56fd" target='_blank'>
              SOX2 drives esophageal squamous carcinoma by reprogramming lipid metabolism and histone acetylation landscape
              </a>
            </td>
          <td>
            Zhen Wang, Ruofei Dai, Li Kang, Huan Yang, Zhaosu Chen, Jianzhong He, Lei Shu, Yiting Zhong, Yunfeng Zhang, Zhengyi Hua, Yuanyong Huang, Yuhan Jiang, Jiwen Li, Liyan Xu, Fei Lan, Shuhai Lin, Jeimin Wong
          </td>
          <td>2025-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Tumor drug resistance remains a significant barrier to effective cancer treatment, undermining the clinical efficacy of current therapies through adaptive and evolving mechanisms. Cuproptosis, a recently discovered form of programmed cell death, emerges as a promising tumor suppressor by targeting mitochondrial metabolic pathways, offering a novel strategy to combat drug resistance. However, targeting mitochondrial metabolism presents challenges due to the essential physiological roles of mitochondria in normal tissues, raising concerns about potential on-target toxicity. Recent studies have revealed the dynamic role of non-coding RNAs (ncRNAs) in modulating susceptibility to cuproptosis via epigenetic alterations, potentially serving as key factors in therapeutic response. In this review, we first summarize the complex interplay between chemotherapeutic resistance and cancer progression/relapse. Then, we outline the key molecular mechanisms underlying cuproptosis, its upstream epigenetic regulators, and the downstream effects on drug resistance. Finally, this review discusses the latest findings on how ncRNAs influence resistance to chemotherapy, immunotherapy, and targeted therapy by modulating cuproptosis. This review aims to untangle the regulatory mechanisms of ncRNA-mediated epigenetic modifications on cuproptosis, highlighting the critical role of the ncRNA-cuproptosis interaction network in tumor drug resistance progression. These insights provide a conceptual foundation for developing ncRNA-targeted therapies in conjunction with cuproptosis-inducing approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368bafe9b516f67cfc94213e12ae103aa111fe6a" target='_blank'>
              Epigenetic modification of cuproptosis by non-coding RNAs in cancer drug resistance
              </a>
            </td>
          <td>
            Fei Du, Lu Tang, Fang Wang, Xian Jiang, Junxin Li, Guojun Wang, Zhuo Zhang
          </td>
          <td>2025-09-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Xist long noncoding RNA is the master regulator of the X-Chromosome inactivation (XCI) process. Xist is expressed from the inactive X and coats the inactive X to facilitate XCI. Cis-regulation of Xist expression remains poorly understood in the context of maintenance of XCI. Here, we have explored the role of the Xist upstream sequences (∼6 kb) lying between Tsix and Jpx in the regulation of Xist and XCI in mouse extra-embryonic endoderm stem cells (XEN), which represent the maintenance phase of imprinted XCI. Here, we show that the deletion of this Xist upstream sequence in the inactive X leads to the upregulation of Xist expression accompanied by the dispersal of the Xist cloud. Notably, we find the loss of enrichment of repressive marks such as H3K27me3, H4K20me1, and MacroH2A, except that of H2AK119ub, in dispersed Xist nuclei. However, X-linked genes remain silent despite Xist dispersal and loss of enrichment of repressive marks. Notably, we find that many autosomal genes, including cohesin Rad21, are dysregulated in Xist-upstream-deleted cells. Additionally, we demonstrate that Xist-upstream deletion leads to alterations of topological contacts of the Xist locus with its upstream positive regulator Ftx and across the inactive X and autosomes. Finally, we show genome-wide alterations of the occupancy of architectural proteins CTCF/RAD21, including at many loci of the inactive X such as the Xist upstream regions and the Firre locus, which is critical for maintaining inactive X conformation. Taken together, we demonstrate that the Xist upstream sequence imparts a multifaceted role in genome regulation beyond the XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4dfe14919eb3bb81fdec07fae01c3bc92dc487" target='_blank'>
              Xist upstream deletion leads to dysregulation of Xist and autosomal gene expression.
              </a>
            </td>
          <td>
            Sudeshna Majumdar, Lakshmi Sowjanya Bammidi, H. C. Naik, Avinchal Manhas, R. Baro, A. Kalita, Amlan Jyoti Naskar, Sundarraj Nidharshan, G. S. Bariha, D. Notani, S. Gayen
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In this study, we delineated the distinct transcriptomic effects of VHL mutation and chromosome 3p (chr3p) loss, revealing that chr3p loss is specifically associated with immune suppression in clear cell renal cell carcinoma (ccRCC). Furthermore, we developed driver genomic aberration (DGA) gene signatures that demonstrate superior performance in predicting both patient prognosis and treatment response compared to traditional mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8edc71bb1913e06b7ed3c5e92c20979a6f59e6" target='_blank'>
              Unravelling the distinct effects of VHL mutations and chromosome 3p loss in clear cell renal cell carcinoma: Implications for prognosis and treatment
              </a>
            </td>
          <td>
            Xiang Wang, Jian-Rong Li, N. Chowdhury, Lang Wu, Cheng Chao
          </td>
          <td>2025-09-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 E1A binding protein (EP300) is a histone acetyltransferase, which modifies chromatin and drives the expression of oncogenes critical in several hematological malignancies and solid tumors. Current pharmacological inhibitors targeting EP300 are not very selective and result in adverse effects, such as thrombocytopenia. Therefore, the development of agents that selectively target EP300 is of high interest as it has the potential to broaden the therapeutic window. Using the targeted degradation approach, we have developed a potent and selective heterobifunctional degrader of EP300 with biological activity in multiple hematological malignancies, including diffuse large B-cell lymphoma, multiple myeloma, and follicular lymphoma. We observed excellent translation of in vitro sensitivities to profound efficacies in a series of xenograft-based in vivo studies. Importantly, an investigational safety study was performed in mice at efficacious doses, and no thrombocytopenia was observed. To further investigate the therapeutic potential of this novel class of agents, we also demonstrated synergistic effects of EP300 degraders with several standard-of-care agents (such as IMiDs, dexamethasone, and venetoclax) in several tumor indications. Collectively, leveraging targeted protein degradation to achieve selective EP300 degradation shows promise to be a well-tolerated and effective treatment strategy for multiple hematological malignancies.



 Gromek Smolen. Targeting chromatin modulators with therapeutic degraders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr IA001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae7067252d901d28b411389880459f57bdea733" target='_blank'>
              Abstract IA001: Targeting chromatin modulators with therapeutic degraders
              </a>
            </td>
          <td>
            Gromek Smolen
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="There is growing interest in understanding the mechanisms underlying differences in cancer incidence among species (comparative oncology). The naked mole-rat (NMR) is often referenced as "cancer-resistant" and prior studies focused on identifying mechanisms explaining this. However, efforts to assess this in vivo have been limited. Herein, we provide evidence that the NMR presents as a novel autochthonous model of lung tumor initiation, driven by an introduction of the oncogenic Eml4-Alk fusion protein using CRISPR-mediated genome editing. Whereas in mice the inversion alone is sufficient to drive tumorigenesis, the inversion alone was insufficient to drive tumorigenesis in the NMR lung and tumor development required additional losses of the tumor suppressors p53 and pRb. Our findings suggest that the proposed "resistance" of the NMR to the development of cancer may reflect that the genetic events leading to tumor initiation are likely to be comparable to those present in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a53eef9b5cb3d97ea0c88db9955706dc37ce138" target='_blank'>
              An Autochthonous Model of Lung Cancer Identifies Requirements for Cellular Transformation in the Naked Mole-Rat.
              </a>
            </td>
          <td>
            A. Shepard, Daniel K Lester, Scott Troutman, Sany Hoxha, Walid T Khaled, E. S. Smith, Thomas J. Park, Rochelle Buffenstein, Dongliang Du, Mingxiang Teng, C. Dengler-Crish, Kenneth Y Tsai, Elsa R Flores, Andrea Ventura, Joseph L. Kissil
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018e4ec4dd8ef81f5b174f65fbe205bcc992d85a" target='_blank'>
              The Impact of Polyploid Giant Cancer Cells: The Root of Stress Resilience.
              </a>
            </td>
          <td>
            Yuta Ogawa, Lydia Fisher, Tomonori Matsumoto
          </td>
          <td>2025-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8304854a46599698bac2a6e2d8f83da3bfb9d71b" target='_blank'>
              DNA2 enables growth by restricting recombination-restarted replication.
              </a>
            </td>
          <td>
            Jessica J R Hudson, Rowin Appanah, David Jones, Kathryn Davidson, Alice M Budden, Alina Vaitsiankova, Kok-Lung Chan, Keith W. Caldecott, Antony M. Carr, Ulrich Rass
          </td>
          <td>2025-09-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3614c9515947d983f3274b6eeb00370803c6fd8" target='_blank'>
              Circular RNAs in colorectal cancer: From molecular mechanisms to therapeutic applications (Review).
              </a>
            </td>
          <td>
            Yuxing He, Can Luo, Wei Du, Debei Liu, Lingshan Liao, Siqian Wang, Yanlin Cao
          </td>
          <td>2025-11-01</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Mediator complex subunit 10 (MED10) serves as a critical regulator of eukaryotic gene expression by facilitating RNA polymerase II activity. Our investigation aims to characterize MED10's functional contributions and underlying molecular pathways in hepatocellular carcinoma (HCC) development.


METHODS
MED10 expression patterns in HCC and their correlation with clinicopathological parameters and patient outcomes were examined using bioinformatics databases and immunohistochemistry. Subsequently, we systematically investigated the biological functions of MED10 in the malignant progression of HCC through comprehensive in vitro experiments, including assessments of cell migration (transwell and wound healing assays), proliferative capacity (cell counting kit-8, colony formation, and 5-Ethynyl-2'-deoxyuridine assays), and cell cycle progression (flow cytometry analysis). Furthermore, we elucidated the underlying molecular mechanisms using real-time quantitative PCR (RT-qPCR), western blotting, immunofluorescence staining, and public database analyses. Furthermore, an in vivo subcutaneous xenograft model was employed to validate MED10's impact on tumor growth.


RESULTS
The results revealed a marked increase in MED10 expression levels within HCC tissues, showing a strong association with unfavorable clinical outcomes. Mechanistically, MED10 induced the epithelial-mesenchymal transition (EMT) and enhanced HCC cell migration. Moreover, MED10 overexpression drives HCC cell cycle progression and proliferation by activating rapidly accelerated fibrosarcoma 1 (RAF1), a process potentially mediated through the mitogen-activated protein kinase (MEK)/extracellular signal-regulated kinase (ERK)/cellular myelocytomatosis oncogene (c-Myc) signaling axis.


CONCLUSION
MED10 promotes HCC cell migration and EMT but, more importantly, also drives cell cycle progression and proliferation via RAF1 activation, and is related to the MEK/ERK/c-Myc axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83554affa0ce8872e89c5a91b5147c43bb5338b7" target='_blank'>
              MED10 as a Novel Oncogenic Driver in HCC: Promoting Cell Cycle Progression and Proliferation Through RAF1 Activation.
              </a>
            </td>
          <td>
            Junhao Liu, Yongxue Lv, Kejun Liu, Zhengquan Li, Bendong Chen, Yang Bu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Gastric cancer (GC) is a leading cause of cancer-related deaths worldwide. While more recent incidence and mortality rates have dropped, GC remains a significant cause of health disparities for many underserved and under-resourced communities in the US, including racial and ethnic minorities. Despite a high minority cancer burden, few FDA-approved targeted therapies are available for GC. This can be partially explained by limited availability of cancer genomic data and patient-derived models from diverse populations, which hamper therapeutic target identification and drug efficacy studies. To address this, our group has spearheaded two patient-derived xenograft and organoid development and trial centers to generate and characterize preclinical models of GC that are representative of the patient population. Genomic sequencing of a Latino GC cohort revealed over 30% of tumors harbored pathogenic mutations in the tumor suppressor gene, ARID1A. This is comparable to frequencies observed in TCGA, suggesting that ARID1A may be a bona fide driver of GC. ARID1A functions within the SWI/SNF (BAF) chromatin remodeling complex to regulate chromatin accessibility. We performed Cut-and-Run and RNA-Seq to identify chromatin-mediated alterations in gene expression in Latino patient-derived organoids with ARID1A mutations vs those with wildtype ARID1A, and in normal gastric organoids that we engineered with ARID1A knockout mutations using CRISPR/Cas9 compared unedited normal gastric organoids. Analyses revealed significant alterations in multiple signaling pathways implicated in cancer proliferation and migration. To investigate sensitivity to inhibitors targeting altered pathways, we then performed drug response screens in these models with targeted inhibitors against PARP enzymes (PARPi) and PI3K/Akt (PI3Ki, AKTi) pathway. Results revealed varied vulnerability against individual targeted inhibitors, which we are actively testing in combination for greater efficacy. These studies address critical knowledge gaps in ARID1A-related tumor biology and may inform precision medicine approaches in an underserved and disproportionately affected population.



 Nicole Halmai, Jasmine Diaz, Alexa Morales Arana, Hongyong Zhang, Paul Lott, Ana Estrada-Florez, Luis Carvajal-Carmona. Understanding the role of tumor suppressor gene ARID1A in gastric cancer development and therapeutic response [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C088.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69dff454764e9f995ca490cd8623c309f465ba02" target='_blank'>
              Abstract C088: Understanding the role of tumor suppressor gene ARID1A in gastric cancer development and therapeutic response
              </a>
            </td>
          <td>
            Nicole B. Halmai, Jasmine Diaz, Alexa Morales Arana, Hongyong Zhang, Paul C. Lott, Ana P. Estrada-Florez, Luis G Carvajal-Carmona
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Platelets are anucleate blood cells vital for hemostasis and immunity. During cell death and aberrant mitosis, nucleated cells release DNA, resulting in "cell-free" DNA in plasma (cfDNA). An excess of cfDNA is deleterious. Given their ability to internalize pathogen-derived nucleic acids, we hypothesized that platelets may also clear endogenous cfDNA. We found that, despite lacking a nucleus, platelets contained a repertoire of DNA fragments mapping across the nuclear genome. We detected fetal DNA in maternal platelets and cancer-derived DNA in platelets from patients with premalignant and cancerous lesions. As current liquid biopsy approaches utilize platelet-depleted plasma, important genetic information contained within platelets is being missed. This study establishes a physiological role for platelets that has not previously been highlighted, with broad translational relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/581d373e8e79203d92ac9a7156f631a3659ced7b" target='_blank'>
              Platelets sequester extracellular DNA, capturing tumor-derived and free fetal DNA.
              </a>
            </td>
          <td>
            L. Murphy, Jeanne Inchauspé, Giampiero Valenzano, Pamela Holland, N. Sousos, Hayley L Belnoue-Davis, Rong Li, Natalie J Jooss, Camelia Benlabiod, Eleanor Murphy, Zohar Etzioni, Emelie Shepherd, Lucy Denly, Sujata Biswas, Lin Chen, Jennifer O'Sullivan, Michael P Rimmer, Abdullah O. Khan, C. S. Karali, N. Nasreddin, Ian S Hitchcock, Milka Koupenova, S. Kriaučionis, Jim R Hughes, Eric O’Neill, Manu Vatish, Paul Rees, Simon J Leedham, M. Desborough, Adam J Mead, Benjamin Schuster-Böckler, Christopher D Gregory, B. Psaila
          </td>
          <td>2025-08-14</td>
          <td>Science</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) remains a global health burden, with disproportionately high mortality in China’s Guangxi region, where endemic Clonorchis sinensis (C. sinensis) infection coincides with elevated HCC incidence. Preliminary single-cell sequencing revealed marked overexpression of cluster of differentiation 24 (CD24) in HCC tissues, suggesting its potential pathological role. This study aims to elucidate the oncogenic mechanisms of C. sinensis excretory-secretory products (CsESPs) and their link to CD24-mediated HCC progression. Methods We employed an integrated clinical and experimental approach. First, clinical cohort analysis assessed CD24 expression in C. sinensis-associated HCC cases. Multiplatform bioinformatics validation (GEPIA/UALCAN/TIMER) evaluated CD24’s prognostic significance and immune microenvironment modulation. Functional studies (quantitative polymerase chain reaction (qPCR), Western blotting, CCK-8 assays, flow cytometry) examined CsESPs’ effects on CD24 expression, cell proliferation, and apoptosis. Mechanistic investigations (chromatin immunoprecipitation, dual-luciferase reporter assays) identified E2F1-mediated transcriptional activation of CD24. siRNA-mediated CD24 knockdown validated its role in CsESPs-driven oncogenesis. Additionally, the expression of immune checkpoint (CTLA-4, LAG-3) was assessed in the co-cultures of peripheral blood mononuclear cells (PBMCs)–HCC cells. Results Clinical cohort analysis confirmed significant CD24 upregulation in HCC, particularly in C. sinensis-infected cases. Bioinformatic analyses linked high CD24 expression to poor prognosis and immune microenvironment alterations. Functional assays demonstrated that CsESPs enhance CD24 expression, promoting proliferation and inhibiting apoptosis. Mechanistically, E2F1 directly binds to CD24 promoter, driving its transcription upon CsESPs exposure. CD24 silencing reversed CsESPs-induced oncogenic effects. Furthermore, CsESPs upregulated immune checkpoints (CTLA-4, LAG-3) in the co-cultures of PBMC–HCC cells, an effect reversed by CD24 knockdown. Conclusions Our findings establish a novel parasitic carcinogenesis paradigm wherein C. sinensis promotes HCC development through E2F1-mediated transcriptional activation of CD24, simultaneously identifying prognostic biomarkers and therapeutic targets while suggesting combinatory immunotherapy strategies for parasite-associated HCC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13071-025-06979-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44a225526ec9b8357a014fc963659daa1b7f9c" target='_blank'>
              Clonorchis sinensis-driven hepatocarcinogenesis via E2F1-CD24 transcriptional axis: mechanistic and therapeutic implications
              </a>
            </td>
          <td>
            Wen-Min Lu, Jin Yan, Zhao-Ji Liu, Yong Wu, Qian-Ru Cui, Ji Feng, Yu Chen, Guangzhi Zhu, Tao Peng, Jing Zhou, Guo-Dong Lu
          </td>
          <td>2025-08-19</td>
          <td>Parasites & Vectors</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Promyelocytic leukemia nuclear bodies (PML-NBs) are dynamic membrane-less organelles (MLOs) located in the nucleus that serve as regulatory hubs for multiple cellular processes. This review examines current understanding of PML-NB structure, assembly mechanisms, and their diverse functional roles. We discuss how PML-NBs interact with chromatin to influence gene expression, regulate transcription factors, and participate in protein quality control. The review highlights their critical functions in tumor suppression, particularly in acute promyelocytic leukemia, and their role in intrinsic antiviral defense against various pathogens. Despite significant advances in the field, key questions remain regarding the mechanistic triggers of PML-NB formation and their common roles across different pathologies. Further elucidation of these aspects may provide valuable insights for developing therapeutic approaches targeting the PML-NB axis in disease treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/038aba7e169328e584cd868fae3cb74bfb23239c" target='_blank'>
              Structure and Function of PML Nuclear Bodies: A Brief Overview of Key Cellular Roles
              </a>
            </td>
          <td>
            Karolina Dorosz, Lidia Majewska, Jacek Kijowski
          </td>
          <td>2025-09-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ee589a5bd29544fb7fca88fdf42fdd21a2e628" target='_blank'>
              A subset of transposable elements as mechano-response enhancer elements in controlling human embryonic stem cell fate.
              </a>
            </td>
          <td>
            Tongyu Sun, Yueyuan Xu, Nicole Angel, Luna Chen, Kan Zhang, B. Hoffman, Jianhong Ou, Zhipeng Meng, Shyni Varghese, Yarui Diao
          </td>
          <td>2025-09-17</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Stem cells are essential for tissue maintenance, repair, and regeneration, yet their dysregulation gives rise to cancer stem cells (CSCs), which drive tumor progression, metastasis, and therapy resistance. Despite extensive research on stemness and oncogenesis, a critical gap remains in our understanding of how the transcriptomic landscapes of normal somatic stem cells (SSCs) diverge from those of CSCs to enable malignancy. This review synthesizes current knowledge of the key signaling pathways (Wnt, Notch, Hedgehog, TGF-β), transcription factors (Oct4, Sox2, Nanog, c-Myc, YAP/TAZ), and epigenetic mechanisms (chromatin remodeling, DNA methylation, microRNA regulation) that govern stemness in SSCs and are hijacked or dysregulated in CSCs. We highlight how context-specific modulation of these pathways distinguishes physiological regeneration from tumorigenesis. Importantly, we discuss the role of epithelial–mesenchymal transition (EMT), cellular plasticity, and microenvironmental cues in reprogramming and maintaining CSC phenotypes. By integrating transcriptomic and epigenetic insights across cancer biology and regenerative medicine, this review provides a framework for identifying vulnerabilities specific to CSCs while still preserving normal stem cell function. Understanding these distinctions is essential for the development of targeted therapies that minimize damage to healthy tissues and advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb9e90607e3b0ca792b0cb08f2440605b878350" target='_blank'>
              Transcriptomic Comparisons of Somatic and Cancer Stem Cells
              </a>
            </td>
          <td>
            Austin Drysch, Arun Ahuja, Dillan Prasad, Rishi Jain, Sharbel Romanos, A. Alwakeal, Christopher S. Ahuja
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters.Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis.We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1.These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b54093d228fc592b42597f1baffae61141e09d6b" target='_blank'>
              DNA Methylation in Ovarian and Endometrial Cancers: Predictive and Mechanistic Roles in PARP Inhibitor and ICI Response.
              </a>
            </td>
          <td>
            Shuhei Kitamura, Ayumi Taguchi, Kana Tamai, Yoko Yamamoto, Anh Quynh Duong, Daisuke Yoshimoto, Ayako Mori, Aya Ishizaka, Saki Tsuchimochi, K. Sone, Masahito Kawazu, K. Oda, Yasushi Hirota
          </td>
          <td>2025-09-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>28</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>