<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-03 09:32:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>40</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>84</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>119</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>374</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>324</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke‐positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single‐cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low‐pass whole‐genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke‐endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability‐associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer. Methods The article provides a comprehensive analysis of the mechanisms underlying the formation of ecDNA, highlighting factors such as aberrant DNA damage repair and chromosomal rearrangements. It further examines the biological roles of ecDNA in augmenting oncogene expression, fostering tumor heterogeneity, and facilitating immune evasion. Results Epidemiological studies indicate significant variability in the distribution of ecDNA across different breast cancer subtypes, with a notable association with invasive subtypes, such as triple‐negative breast cancer. The presence of ecDNA is linked to poor prognosis and treatment resistance in patients. Current detection technologies for ecDNA include molecular biology and imaging techniques, and its detectability in plasma underscores its potential as a biomarker for liquid biopsy. Conclusions The development of ecDNA‐targeted therapies and their integration with immunotherapy strategies offers promising new avenues for breast cancer treatment. Despite challenges such as incomplete elucidation of mechanisms, standardization of detection methods, and ethical considerations, ecDNA remains a valuable biomarker and therapeutic target in breast cancer. Future research is anticipated to advance its clinical transformation and application in individualized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-01</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f59455ffad8232534ff09a38ff5659981429a5" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes.
              </a>
            </td>
          <td>
            L. G. de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cryptococcus neoformans is a fungal pathogen of humans that causes life-threatening meningoencephalitis. During infection, enlarged, polyploid titan cells are produced that promote survival, evade immune cells, and generate diverse progeny. These titan cells divide to produce haploid, aneuploid or diploid daughter cells with enhanced stress tolerance; however, how ploidy reduction occurs is poorly understood. Here, we show that titan cells developed from diploid strains predominantly produce diploid daughter cells with haploid daughters observed infrequently. We further demonstrate that meiosis-specific genes, including DMC1 and SPO11, are critical for stable inheritance of a diploid genome in the daughter cells. Specifically, deletion of these genes in a heterozygous diploid background results in: 1) titan cells with a significantly reduced capacity to produce daughter cells; 2) increased phenotypic variation among daughter cells produced by the titan cells, including traits that could be relevant to cell growth and viability; and 3) daughter cells produced by the titan cells exhibiting high levels of loss of heterozygosity (LOH) and aneuploidy, suggested elevated genome instability. Taken together, these findings demonstrate the importance of meiosis-specific genes in the ploidy reduction process of titan cells derived from a heterozygous diploid background in an important human fungal pathogen. SIGNIFICANCE Polyploid titan cells are central to Cryptococcus neoformans persistence during infection, yet how they reduce ploidy to generate daughter cells was unclear. We show that the meiosis-specific factors Dmc1 and Spo11 are critical for titan cells to produce viable daughter cells specifically in a heterozygous diploid context, ensuring chromosome stability without introducing detectable canonical meiotic recombination. Loss of either gene compromises daughter cell production and causes genome instability (LOH, aneuploidy) and phenotypic heterogeneity. These findings reveal a “para-meiosis” safeguard that preserves genome balance during titan cell depolyploidization in C. neoformans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b707195ab1586edfb629b8392c64a813e56aa4" target='_blank'>
              Meiosis-specific genes play roles in ploidy reduction in Cryptococcus neoformans titan cells
              </a>
            </td>
          <td>
            Zhuyun Bian, Kayla Wilhoit, Julian Liber, Anushka Peer, Ziyan Xu, Paul M. Magwene, Sheng Sun, Joseph Heitman
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomally amplified oncogenes are shown to drive evolution in a large subset of IDH wild-type glioblastomas, and the spatial patterns of extrachromosomal DNA contain information on a tumor’s evolutionary history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA–Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="OncoDB was initially developed to advance cancer research by integrating RNA expression profiles, DNA methylation patterns, clinical annotations, and oncoviral signatures derived from the Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression datasets. We now present OncoDB 2.0, an extensively expanded platform that offers a more comprehensive and integrated view of cancer omics. The updated version includes an atlas of somatic mutations discovered from combined DNA and RNA sequencing, enabling in-depth investigation of mutation patterns across tumor types and their association with clinical features. Furthermore, we have integrated proteomic data from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and chromatin accessibility data from TCGA, offering new dimensions for oncogene regulation studies. OncoDB 2.0 also introduces advanced multi-omics analysis modules that facilitate the combined exploration of RNA expression, DNA methylation, and somatic mutations, allowing researchers to examine complex cross-omic relationships with greater depth and flexibility. Together, these enhancements make OncoDB 2.0 a robust and invaluable tool for the cancer research community. OncoDB 2.0 is freely available at https://oncodb.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2597afaecc32114e7746cc65fb5c9507bfe36bf0" target='_blank'>
              OncoDB 2.0: a comprehensive platform for integrated pan-cancer omics analysis.
              </a>
            </td>
          <td>
            Minsu Cho, Gongyu Tang, Charles S Rogers, Makayla Dove, Xinyi Liu, Yuanxiang Li, Xiaowei Wang
          </td>
          <td>2025-09-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc4a7f8e64bdd86a0871fb665e1084f20e21063" target='_blank'>
              A mobile circular DNA element drives memory-related processes in mice
              </a>
            </td>
          <td>
            Haobin Ren, Hao Gong, W. Liau, Qiongyi Zhao, Alexander P. Walsh, Mason R. B. Musgrove, J. Davies, Esmi L. Zajackowski, Sachithrani U. Madugalle, P. Marshall, Tian Lan, Jichun Shi, Jiazhi Jiang, Wei Wei, Xiang Li, T. Bredy
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histone variant macroH2A1 (mH2A1) has been widely recognized as a suppressor of gene expression. Recently, a cell cycle-dependent deposition of mH2A1 was discovered in mouse cells, but whether this process exists in human chromatin is unclear, which might be crucial for related diseases, particularly cancer. In this study, with native chromatin immunoprecipitation (nChIP-seq), we firstly demonstrate that dynamic mH2A1 domains occur in both normal and cancerous human cells and have conserved enrichment patterns across species. Our findings further provide new epigenetic insights into the role of mH2A1 in malignant proliferation, offering a novel perspective for future cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a188786f3e05b473ebb9d1416685a9aa16e12df" target='_blank'>
              Regional Conservation and Transcriptional Regulation of Tumor-Associated Genes by macroH2A1 Deposition in Mammalian Cells
              </a>
            </td>
          <td>
            Yongzhuo Deng, Zeqian Xu, Le Zhang, Bishan Ye, Zhifeng Shao, Xinhui Li
          </td>
          <td>2025-09-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b95d07f47eb2caf5e3c1769c6acf5b8c92ae573" target='_blank'>
              Enhancers mediate euchromatin hopping at chromatin contact points
              </a>
            </td>
          <td>
            Shanelle Mullany, Tiegh Taylor, Hangpeng Li, Yaqing Zhao, Alisha Suri, Tom Sexton, James O. J. Davies, 
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb77c980017aee37cb31324f45fa9fe45acb96" target='_blank'>
              Moonlighting Role of Meiotic SYCP1 in Breast Cancer: A Chromatin-Bound Regulator of DNA Repair, Transcription, and Drug Resistance
              </a>
            </td>
          <td>
            LC Brennan, OV Grinchuk, M. Pachon, IF Sou, C. Fawcett, CG Nogueira, M. Guthrie, AD Bates, M. Hine, A. Thomaz, Z. Hu, AB Fielding, OR Davies, W-W Tee, UL McClurg
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The genome is compacted in the nucleus through a hierarchical chromatin organization, ranging from chromosome territories to compartments, topologically associating domains (TADs), and individual nucleosomes. Nucleosome remodeling complexes hydrolyze ATP to translocate DNA and thereby mobilize histone proteins. While nucleosome remodeling complexes have been extensively studied for their roles in regulating nucleosome positioning and accessibility, their contributions to higher-order chromatin architecture remain less well understood. Here, we investigate the roles of two key nucleosome remodelers, esBAF and INO80C, in shaping 3D genome organization in mouse embryonic stem cells. Using Hi-C, we find that loss of either remodeler has minimal effects on global compartment or TAD structures. In contrast, subcompartment organization is notably altered, suggesting that esBAF and INO80C contribute to finer-scale chromatin topology. To overcome the limited resolution of Hi-C for detecting regulatory loops, we employed promoter capture Micro-C (PCMC), which revealed that the loss of esBAF or INO80C alters a subset of promoter anchored looping interactions. Although these changes occur at distinct genomic loci for each remodeler, the affected sites are commonly enriched for bivalent chromatin regions bound by OCT4, SOX2, and NANOG (OSN), as well as BRG1 and INO80 themselves. Together, our findings reveal that esBAF and INO80C selectively influence subcompartment identity and enhancer–promoter communication at key regulatory loci, highlighting a previously underappreciated role for nucleosome remodelers in higher-order chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e95264023995f85a35edad63b59e9eef19fae0c" target='_blank'>
              esBAF and INO80C fine-tune subcompartments and differentially regulate enhancer-promoter interactions
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd28764a1bb58dc6ea970c4ddb741f750a9e8bc" target='_blank'>
              Genome-wide DNA bridging by H-NS reshapes the stationary phase nucleoid and transcriptional landscape
              </a>
            </td>
          <td>
            Lindsey E. Way, Xiaofeng Dai, Emma E Wiesler, Lauren A McCarthy, Zhongqing Ren, Gail G Hardy, D. E. Fuller, Labeeb Hossain, Anne S. Meyer, E. Abbondanzieri, Julie S. Biteen, Xindan Wang
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. V. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated.The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development.Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation.Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The ataxia telangiectasia mutated (ATM) kinase orchestrates the early stages of DNA double-strand break repair by promoting hyperphosphorylation of CtIP, a key step in the initiation of DNA end resection. However, the regulatory mechanisms controlling resection extent remain incompletely understood. Here we identify ERCC6L2 as a key regulator of DNA end resection in response to ATM inhibition. ERCC6L2 undergoes liquid–liquid phase separation via its intrinsically disordered regions, forming dynamic nuclear condensates that regulate CtIP stability. Disruption of these condensates renders CtIP susceptible to RNF138-mediated ubiquitination and degradation, thereby mitigating the heightened chromatin recruitment of CtIP induced by ATM inhibition. Intriguingly, ERCC6L2 is frequently downregulated in multiple cancer types and correlates with resistance to ATM inhibitors in both in vitro and in vivo settings. Our findings unveil the crucial role of ERCC6L2–CtIP condensates in governing the extent of DNA end resection and underscore the potential significance of ERCC6L2 as a predictive biomarker for ATM inhibitor response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07119c6c29989e52a1007b94a87a49c0954c86a" target='_blank'>
              Phase separation of ERCC6L2–CtIP regulates the extent of DNA end resection
              </a>
            </td>
          <td>
            Yixin Yin, Jinlong Lin, Xiaoxia Cai, Runcong Nie, Jiliang Lin, Shuting Li, Z. Xiang, Yihong Ling, Yiyang Zhang, Jie Zhou, Jiewei Chen, Wenping Lin, Jinling Duan, Xueyi Zheng, Dan Xie, Mu-yan Cai
          </td>
          <td>2025-09-05</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Background/Objectives: Uterine leiomyomas (ULs) and breast fibroadenomas (FAs) are the most common benign tumors in women, both arising in hormone-responsive mesenchymal tissues and often co-occurring during reproductive years. Despite their shared hormonal sensitivity and frequent MED12 mutations, their downstream molecular evolution remains poorly characterized. This study aimed to investigate whether ULs and FAs, though initiated by similar genetic alterations, diverge in their oncogenic trajectories, thereby addressing the molecular basis for their distinct clinical behaviors. Methods: We performed whole-exome sequencing (WES) on 15 uterine leiomyomas and 7 publicly available fibroadenomas with matched normal controls to compare somatic mutations, copy number alterations (CNAs), and mutational signatures. All UL samples were derived from Korean patients who underwent surgical treatment at a tertiary hospital between 2017 and 2019. Somatic variants were analyzed using MuTect2 and Strelka2. FACETS was used to estimate copy number changes in individual samples, and GISTIC2 to identify recurrent and statistically significant copy number alterations across patient cohorts. Mutational processes were inferred using SigProfiler. Microsatellite instability status was determined with MSIsensor2. The study was approved by the Institutional Review Board (UC17SNSI0092). Results: Comparative whole-exome sequencing of 15 ULs and 7 FAs from matched tumor-normal samples in an East Asian cohort confirmed both tumor types harbor identical MED12 p.G44D mutations, establishing shared molecular initiation. However, post-initiation evolution diverged dramatically: ULs exhibited chromosomal instability with 15 copy number amplifications, 6 of which affected oncogenes, but relatively modest point mutations. In contrast, FAs remained chromosomally stable but hypermutated, harboring 1.47× more variants than ULs despite lower tumor purity. Notably, one histologically benign FA harbored multiple loss-of-function mutations plus an EGFR gain-of-function mutation typically associated with malignant breast cancer, challenging traditional benign-malignant classifications. Conclusions: Despite sharing a common initiating mutation in MED12, ULs and FAs evolve through fundamentally distinct genomic pathways. UL evolves through chromosomal instability, whereas FA evolves through a mutator phenotype, with important implications for understanding tumor biology and molecular-based risk stratification. These findings support a paradigm of tissue-specific oncogenic evolution and underscore the potential clinical utility of genomic profiling in distinguishing benign tumors with atypical molecular features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9889bd66df0b0830e6828b3b935a1992068cb7" target='_blank'>
              Tissue-Specific Genomic Evolution Despite Shared MED12 Mutations in Benign Tumors
              </a>
            </td>
          <td>
            Jeong Namkung, Sang Ho Park, Ayoung Hwang, Hae Seo, Jooyoung Park, Moonyoung Lee, Hyunkyung Kim, Jungmin Choi, Jaeyen Song
          </td>
          <td>2025-10-16</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73a0dfd6d309117f8f8d19fdfdd72a4ff9c34bc" target='_blank'>
              Long-read single-cell genome, transcriptome and open chromatin profiling links genotype to phenotypes
              </a>
            </td>
          <td>
            Alexandra Pančíková, Ruben Cools, Marios Eftychiou, Margo Aertgeerts, Joris Vande Velde, Heidi Segers, Jan Cools, L. Harbers, J. Demeulemeester
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f85b37be71af2ec4cbf186601f4b1c5fc232717" target='_blank'>
              Influence of cis-regulatory elements on regulatory divergence in human segmental duplications
              </a>
            </td>
          <td>
            Colin J. Shew, Gulhan Kaya, Sean P McGinty, Megan Y. Dennis
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e04c6f5ef9fa6ac101d2f55bfc644ec979072" target='_blank'>
              Pervasive suppressors halt the spread of selfish Segregation Distorter in a natural population
              </a>
            </td>
          <td>
            Ching-Ho Chang, Tyler Handler, Nicholas J. Fuda, Danielle Pascua, Taylar Mouton, A. Larracuente
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Wei-Ke Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f77d30c5b35d3bee66492e606ea00f82e5f1ceb" target='_blank'>
              Argonautes and small RNAs associated with nematode programmed DNA elimination
              </a>
            </td>
          <td>
            M. Zagoskin, Yuanyuan Kang, Richard E Davis, Jianbin Wang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d85f000ce66e3391a720ff68ea8a6722807028f" target='_blank'>
              TAOK1 regulates chemo- and radiosensitivity in BRCA1/2-deficient tumors
              </a>
            </td>
          <td>
            Lea Lingg, M. Mutlu, A. Naguleswaran, I. Klebic, E. Gogola, Dilara Akhoundova Sanoyan, Pascale Martine Schürch, Raviprasad Kuthethur, Carmen Fonseca, Simone de Brot, M. Everts, Martín González-Fernández, Saskia Hussung, E. Vassella, M. V. van Vugt, Arnab Ray Chaudhuri, Jos Jonkers, Sven Rottenberg, P. Francica
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab22d511d2948a185e5a280612430d61e3b4b0a" target='_blank'>
              Linking Germline Telomere Removal to Global Programmed DNA Elimination in Tetrahymena Genome Differentiation
              </a>
            </td>
          <td>
            Kohei Nagao, Kazufumi Mochizuki
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite shared genetic driver alterations and histology, the genomic fidelity of most mouse tumor models, including those genetically engineered (GEMM), to their human counterparts is unknown. Here, we developed MiMouse, a mouse comprehensive genomic profiling panel for high throughput credentialling applicable to routine FFPE tumors. Through simulation/validation, we focused on considerations for cross-species mutation prioritization, strain determination, and aneuploidy detection. Using MiMouse, we profiled >250 tumors from high-grade serous carcinoma (HGSC) GEMMs based on conditional inactivation of Brca1 (B), Trp53 (P), Pten (Pt), Rb1 (R), and/or Nf1 (N), and a colorectal carcinoma (CRC) GEMM based on conditional inactivation of Apc, Kras and/or P. We confirmed increased genomic instability in HGSC tumors, with BPPt cancers having both the shortest latency and the least genomic instability. In CRC, focusing on fidelity to human CRC aneuploidy events, our results highlighted the critical importance of synteny in transgenic studies, as not only was loss of mouse chromosome 18 (containing the tumor suppressor gene Smad4) a significant aneuploidy event (18%), additional tumors harbored focal Smad4 copy loss, potentially due to the mouse-specific proximity of Apc (mouse and human chromosomes 18 and 5, respectively). Likewise, mouse chromosome 5, the only significantly gained (46%) chromosome in our CRC models, has syntenic blocks from human chromosomes 7p, 7q and 13q, including Cdx2, which is both a lineage specific CRC oncogene and the CRC GEMM promoter source. Given the importance of mice to translational cancer research, this study highlights the considerations and utility of approaches for comprehensive genomic credentialling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc16f185cf42f303d66c6b3a8187a11fa08b90af" target='_blank'>
              Development and Application of MiMouse, a Comprehensive Genomic Profiling Panel for Credentialling Mouse Tumor Models.
              </a>
            </td>
          <td>
            Kevin Hu, C. Liu, Zhaoping Qin, Aaron M Udager, M. Cieslik, S. Tomlins
          </td>
          <td>2025-10-06</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b1867c4003902c154bf44ef7de285c3bf97bccb" target='_blank'>
              Selective mRNA translation determines adaptative mutability of melanoma cells to anti-BRAF/MEK combination therapy
              </a>
            </td>
          <td>
            Lucilla Fabbri, Lucie Lagadec, Eva Guérin, Hélène Lecourt, Dorothée Baille, Laetitia Besse, Cédric Messaoudi, Laurent Désaubry, Hussein Abou-Hamdan, Sévérine Roy, Bérangère Lombard, D. Loew, Jean Yves Scoazec, Caroline Robert, S. Vagner
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA methylation and histone modifications encode epigenetic information. Recently, major progress was made to measure either mark at a single-cell resolution; however, a method for simultaneous detection is lacking, preventing study of their interactions. Here, to bridge this gap, we developed scEpi2-seq. Our technique provides a readout of histone modifications and DNA methylation at the single-cell and single-molecule level. Application in a cell line with the FUCCI cell cycle reporter system reveals how DNA methylation maintenance is influenced by the local chromatin context. In addition, profiling of H3K27me3 and DNA methylation in the mouse intestine yields insights into epigenetic interactions during cell type specification. Differentially methylated regions also demonstrated independent cell-type regulation in addition to H3K27me3 regulation, which reinforces that CpG methylation acts as an additional layer of control in facultative heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1b5e8d6e33470f1946309c40b533c074bbdfc5" target='_blank'>
              Single-cell multi-omic detection of DNA methylation and histone modifications reconstructs the dynamics of epigenomic maintenance
              </a>
            </td>
          <td>
            Christoph Geisenberger, Jeroen van den Berg, Vincent van Batenburg, Buys Anton de Barbanson, Anna Lyubimova, Joe Verity-Legg, Xiufei Chen, Yibin Liu, Chun‐Xiao Song, J. de Ridder, Alexander van Oudenaarden
          </td>
          <td>2025-09-25</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d93b649d64497bfbe4009380267e67f2f32adc" target='_blank'>
              The limited effects of AAV2 vectors on host chromatin accessibility and nuclear architecture are consistent with a favorable safety profile
              </a>
            </td>
          <td>
            Joël W. Zöllig, Maija K. Pietilä, Kurt Tobler, C. Fraefel
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Whole genome doubling (WGD) triggers profound genomic and epigenetic reorganization, yet the functional dynamics of DNA methylation during this process remain incompletely resolved. Here, we integrate whole genome bisulfite sequencing (WGBS) and three-dimensional chromatin interaction data to display methylation landscapes in autotetraploid Arabidopsis thaliana. Our analysis reveals evolutionarily conserved spatial patterning of DNA methylation after WGD, with centromeric enrichment and telomeric depletion. Chromosome-level profiling identifies Chromosome 2 as the most highly methylated across CG, CHG, and CHH contexts, while Chromosome 1 shows the lowest methylation. Subcontext methylation analysis uncovers increases in methylation levels in autotetraploid Arabidopsis thaliana, most pronounced in the CHH context, yet global distribution patterns remain stable. Comparative methylation profiling around genes and transposable elements (TEs) reveals elevated CHH methylation in autotetraploid gene bodies and flanking regions, whereas TE bodies exhibit minimal changes despite minor flanking hypermethylation. Strikingly, 8% of chromatin compartments were restructured, and B-B interactions weakened in autotetraploid, while DNA methylation remained stable across shifting A/B compartments. Our findings suggest that DNA methylation serves as a resilient epigenetic modification during WGD, even if 3D chromatin architecture undergoes reorganization upon WGD in some degree.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c794ab17ab1c871d4bc77df6d82021f2b6c73a" target='_blank'>
              The Association Between DNA Methylation and Three-Dimensional Genome During Whole Genome Doubling in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Ranze Zhao, Zhongqiu Ni, Dingyu Zhang, Yuda Fang
          </td>
          <td>2025-09-24</td>
          <td>Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Chromosome conformation within the nucleus is essential for genome function. These have primarily been studied at the scale of loops and compartments, or at lower spatial resolution using traditional in situ hybridization in chemically fixed samples. However, the mesoscale organization of single chromosomes in vivo, shaped by the interplay between chromatin architectural proteins and histone modifications, remains partially understood. In this study, we interrogated the mesoscale conformations of interphase chromosomes in live human osteoblasts and transformed osteosarcoma cells, focusing on chromosome 19. Results Chromosome conformations were quantified by the aspect ratio of the principal axes of gyration tensors. In osteoblasts, approximately 81% of chromosome 19 are observed to consist of regions characterized by highly extended organizations, with aspect ratios approximately four times greater than those of spheres. In contrast, in osteosarcoma cells, the chromosome displays an extensively collapsed conformation, with aspect ratios more closely approximately that of a sphere. In both cell types, the chromosome’s conformation is bimodal and the balance between these two modes differs very significantly between the two cell types. While the mesoscopic conformation is considerably stable, it is superimposed on dynamic, smaller scale regions. Additional results reveal that this significant conformational shift is independent of the cell cycle but co-regulated by CTCF, cohesion, and H3K27 modifications. Conclusions Our findings provide new insights into the coordinated complex regulatory mechanisms governing mesoscale chromosome organization in normal and transformed osteogenic tissues. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03785-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f8561f1fa7ab9bbf02e738cec3deefcabedf878" target='_blank'>
              Differential regulation of mesoscale chromosome conformations in osteoblasts and osteosarcoma
              </a>
            </td>
          <td>
            Madhoolika Bisht, Yu-Chieh Chung, Siou-Luan He, Sydney Willey, Benjamin D. Sunkel, Meng Wang, Benjamin Z. Stanton, Li-Chun Tu
          </td>
          <td>2025-09-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genomic organization requires an intricate balance between the compact storage of genetic material and the ability to finely tune gene regulation. Chromatin looping achieves this balance by organizing concordantly regulated groups of genes and their regulatory elements into loops while also condensing DNA to fit into the small volume of a nucleus. A number of DNA-binding and associated proteins, including CTCF and cohesin, act as chromatin looping factors that mediate this process. Given the tight association between chromatin looping and gene expression, disordered genomic organization has been linked to disease development, including cancer. Recurrent mutations in chromatin looping factors are common in cancer, in particular blood cancers such as leukemia and myelodysplastic syndromes. In this review, we describe the evolution of our understanding of the chromatin looping process in healthy and malignant hematopoiesis and discuss the therapeutic potential of targeting chromatin looping factors in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05682afd3dc6513232a3e809f87b0eed8c538362" target='_blank'>
              Role of Chromatin Looping Factors in Leukemia.
              </a>
            </td>
          <td>
            Shira G Glushakow-Smith, Zuzana Tothova
          </td>
          <td>2025-09-15</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Balancing replication fork progression and origin usage is essential to maintain genome stability, but measuring replication fork progression rates and origin usage throughout the genome has been challenging. Here, we use nanopore sequencing combined with DNAscent to measure replication fork progression together with origin and termination site usage with single-molecule precision throughout the Drosophila genome with nearly full genome coverage. We find that replication fork progression rates are not uniform throughout the genome. Rather, fork progression is slowest in euchromatin, and this is not correlated with active transcription. Replication origins are also influenced by chromatin, but the exact position of initiation is highly variable and are often several kilobases away from ORC binding sites. Termination sites lack any chromatin or sequence motifs and appear nearly random throughout the genome. By measuring DNA replication dynamics at near full genome coverage, our work reveals key principles of metazoan replication dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0599dfed7162a33334c7070e7c9fe3430e8ac1ed" target='_blank'>
              Nanopore-based sequencing of active DNA replication reveals key principles of metazoan replication fork progression, origin and termination sites
              </a>
            </td>
          <td>
            Dongsheng Han, Cole Shepherd, Mary Lauren Benton, Jared T Nordman
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aadf2778bea4e8d563036eda473ff737cd085e3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T Tran, Clara Errard, Brian E Hew, Brandon E Johnson, David F Waller, Shanfu Xie, Joseph J Higgins, Raymond Tabibiazar, Jesse B Owens
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R‑loops, three‑stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single‑stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R‑loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R‑loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R‑loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R‑loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R‑loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R‑loop biology were examined, including small molecules that induce synthetic lethality, gene‑based interventions and combination approaches that exploit R‑loop vulnerabilities. Challenges in targeting R‑loops and future directions, including multi‑omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R‑loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R‑loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review).
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular medicine reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: Enhancer RNAs (eRNAs), a subclass of long non-coding RNAs transcribed from enhancer regions, have emerged as dynamic regulators of gene expression, tumor progression, and therapeutic response. In gliomas, their biological and clinical significance is only recently being elucidated. This systematic review aimed to synthesize current evidence regarding the role of eRNAs in gliomagenesis, chemoresistance, and prognosis. Methods: We conducted a systematic review following PRISMA 2020 guidelines. PubMed/MEDLINE and Scopus databases were searched on September 2025 using a predefined strategy. Eligible studies included clinical or pre-clinical analyses of eRNAs in gliomas, reporting associations with tumorigenicity, survival, or resistance to temozolomide (TMZ). Risk of bias was assessed using ROBINS-I (Version 2), and findings were qualitatively synthesized. Results: From 26 retrieved records, 10 studies were included, encompassing 22 unique eRNAs. Two studies demonstrated that TMZR1-eRNA and LINC02454* modulate TMZ sensitivity by regulating STAT3, SORBS2, and DDR1 pathways. Seven studies evaluated prognostic implications: 12 eRNAs (e.g., AC003092.1, CYP1B1-AS1, CRNDE) were consistently associated with poor survival, while seven (e.g., LINC00844, ENSR00000260547) correlated with favorable outcomes, particularly in low-grade gliomas. One mechanistic study showed that HOXDeRNA directly promotes gliomagenesis by displacing PRC2 repression at key transcription factor promoters and activating oncogenic super-enhancers. Conclusions: eRNAs are not passive transcriptional by-products but active modulators of glioma biology. They influence tumor initiation, therapeutic resistance, and survival outcomes, underscoring their potential as prognostic biomarkers and therapeutic targets. Future research should validate these findings in larger clinical cohorts and explore strategies for eRNA-directed therapies in precision neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc4bce08879520ae703d36596d7175b40d9d4f73" target='_blank'>
              Unveiling Enhancer RNAs in Gliomas: A Systematic Review and Qualitative Synthesis
              </a>
            </td>
          <td>
            Matteo Palermo, Giovanni Pennisi, Benedetta Burattini, P. Bruzzaniti, A. Talacchi, Alessandro Olivi, C. Sturiale
          </td>
          <td>2025-10-15</td>
          <td>Cancers</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3ca6346229918f35167d16749694c1e3b771aa" target='_blank'>
              STAG2 cohesin cooperates with DREAM to maintain quiescence and suppress tumourigenesis in the urothelium
              </a>
            </td>
          <td>
            Maria Ramal, E. Lapi, M. Kalisz, Jaime Martínez de Villarreal, Meritxell Novillo-Font, , E. Zarzuela, Marta Isasa, Osvaldo Graña-Castro, Jianming Xu, Ana Cayuela, François Radvanyi, Mireia Novell, Jorge Martínez-Torrecuadrada, M. Barallobre, I. Bernard-Pierrot, María Marti-Marimon, Maria L Arbonés, M. Marti-Renom, Francisco X. Real
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d535d363c465b65e62fbbcdfa920c1c51d8d6da" target='_blank'>
              Segregating DNA lesions point to high selective advantage of tumor initiating cells
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Maha Shady, Maria A Andrianova, Michael Spencer Chapman, E. V. Van Allen, S. Sunyaev
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a28c3b0eeec817792757ba08ffa47e832addea" target='_blank'>
              The mitotic stopwatch limits the proliferation of whole genome doubled cells
              </a>
            </td>
          <td>
            LA Allan, R. Foy, M. Keli, AT Saurin
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16cd8bbe2481e2a3b29d88e6d72cbcf077ea54e4" target='_blank'>
              Independent evolution of holocentric centromeres in an early branching apicomplexan parasite
              </a>
            </td>
          <td>
            Abigail Kimball, Wanyi Huang, Rui Xu, Melanie Key, Lisa J. Funkhouser-Jones, L. D. Sibley, Dr. L. David Sibley
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Double-strand break (DSB) repair occurs through non-homologous end joining (NHEJ) or homologous recombination (HR). To identify non-canonical factors that influence DSB repair outcomes, we parsed data from pooled genetic screens. Through this approach, we identified the splicing factor SFPQ, which has been previously reported to associate with DSBs and promote repair. Here, we show that SFPQ depletion alters DSB repair via HR. However, in contrast to other published work, we find that SFPQ does not localize to DSBs but instead stabilizes the expression of RAD51 and its paralogs independently of p53 activation or DNA damage. Our findings suggest that SFPQ contributes to constitutive DSB repair by maintaining RAD51 paralog mRNA stability rather than through direct interaction with DSBs or RAD51 protein and highlight indirect mechanisms by which RNA-binding proteins can influence genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1f2a926bfc3d7a9fcc9c4f329c1455e2b3d6c2b" target='_blank'>
              SFPQ Promotes Homologous Recombination via mRNA Stabilization of RAD51 and Its Paralogs
              </a>
            </td>
          <td>
            Sofia Gotthold, Keile R. Hansen, Andrew N. Brown, Soham P. Chowdhury, Hannah I. Ghasemi, Amanda C. Yoon, Christine M. Joyce, Julien Bacal, Brooke M. Gardner, Chris D. Richardson
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f693038dd949f6166800477057cd05b931fa2da" target='_blank'>
              SLFN11 restricts escape from telomere crisis to prevent alternative lengthening of telomeres
              </a>
            </td>
          <td>
            Sandra Segura-Bayona, V. Borel, Tyler H Stanage, Marija Maric, Melanie Walter, Rafaela A Oliveira, Shudong Li, Aurora I Idilli, Martina Peritore, Graeme Hewitt, M. Hekkelman, Daniel M. Snell, Sam T Jones, Chris Cheshire, Areda Elezi, Probir Chakravarty, Richard Mitter, Anna Mikolajczak, Harriet E Gee, E. Nye, Roderick J. O’Sullivan, A. Singhi, T. Brummelkamp, Anthony J. Cesare, S. Boulton
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres.
              </a>
            </td>
          <td>
            Bailey A Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c454f3d7e351cc169def09b1bb6a18830063a225" target='_blank'>
              PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES
              </a>
            </td>
          <td>
            Rubén Fernández, L. F. Lorenzo-Martín, Víctor Quesada, Xosé R. Bustelo
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf151a47356eecc8542d42d62be9db47d608e1" target='_blank'>
              Joint Mapping of Chromatin Accessibility and Targeted Proteomics in HER2-expressing Breast Cancer Systems
              </a>
            </td>
          <td>
            Anna Fomitcheva Khartchenko, Trinh Lam, Nadine Goldhammer, Marin Bont, Shruti Warhadpande, Jennifer M. Rosenbluth, Amy E. Herr
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic variation at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, has been statistically associated with gene expression and other molecular phenotypes in humans. However, the causality and regulatory mechanisms for most of these STRs remains unknown. Massively parallel reporter assays (MPRA) enable testing the regulatory activity of a large number of synthesized variants, but have not been applied to STRs due to experimental and computational challenges. Here, we optimized an MPRA framework based on random barcoding to study the impact of variation in repeat copy number on expression. We first performed an MPRA on sequences derived from 30,516 promoter-proximal STR loci along with up to 152bp of genomic context, testing 3-4 variants with differing repeat copy numbers for each locus in HEK293T cells. We identified 1,366 loci with significant associations between repeat copy number and expression, which were enriched for positive effect sizes (P=2.08e-110). We then designed a second MPRA in which we performed deeper perturbations, including systematic manipulation of the repeat unit sequence, orientation, and copy number, with 200-300 perturbations for each of the 300 loci with the strongest signals. Our results revealed that the repeat unit sequence is the primary driver of differences in the relationship between copy number and expression across loci, whereas orientation and flanking sequence have weaker effects, primarily for AT-rich repeat units. The high resolution of these perturbations enabled us to detect non-linear effects, most notably for AAAC/GTTT repeats, which emerge only beyond a certain copy number threshold. Finally, we observed that a subset of STRs in our library show expression levels that are tightly linked with predicted DNA secondary structure formation. We repeated our perturbation MPRA in HeLa S3 cells under wildtype and RNase H1 knockdown conditions, which, via reduction in RNase H1 activity, are expected to hinder resolution of R-loops. This demonstrated that associations between copy number and expression at G-quadruplex-forming CCCCG/CGGGG repeats are particularly sensitive to loss of RNase H1, providing support for an R-loop mediated mechanism for these repeats. Altogether, we establish STRs as a critical component of the non-coding regulatory grammar and provide a framework for understanding how this dynamic form of genetic variation shapes gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad23e384b94840a5506d6ec02839f2c2721fcd3" target='_blank'>
              Systematic evaluation of the impact of promoter proximal short tandem repeats on expression
              </a>
            </td>
          <td>
            Xuan Zhang, Lingzhi Zhang, Ellice Wang, Susan Benton, Eduardo Modolo, Mikhail Maksimov, Sharona Shleizer-Burko, Qingqing Gong, Catherine Wang, M. Lamkin, E. Mendenhall, Melissa Gymrek, A. Goren
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Human positive coactivator 4 (PC4) is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, autophagy, B-cell differentiation, neurogenesis, DNA repair, etc. Most PC4 is phosphorylated in cells, which interacts with core histones and the linker histone H1 to confer the compact heterochromatin state of the genome. Knocking down PC4 at both cellular and organismal levels leads to significant chromatin decondensation, altered epigenetic landscape, enhanced autophagy, and increased DNA damage susceptibility. Here, we report that besides p300, PC4 also gets acetylated by DNA repair, facilitating lysine acetyltransferase KAT5 (Tip60) at a specific lysine residue (PC4K80) when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells was substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (PC4K80R), defective in KAT5-mediated acetylation. High-resolution microscopy techniques, including transmission electron microscopy and atomic force microscopy, are employed to visualize chromatin structural changes in response to DNA damage and repair in a Tip60-mediated PC4 acetylation-dependent manner. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates access to the damaged DNA by altering chromatin structures at damage sites, thus promoting DNA repair. This process could be highly significant both in cancer and in aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c83d5690dbc70212ddc740cb45975fd69abb4f3" target='_blank'>
              KAT5-mediated acetylation of PC4 facilitates DNA repair by promoting chromatin reorganization
              </a>
            </td>
          <td>
            Aa yushi A gr a w al, eta Sikder, Siddharth Singh, Rakesh Kumar, Nikhil Pallaprolu, Kalyan Mitra, Sourav Mondal, Rupa Mukhopadhy, anta Sarkar, Ramaling am Per aman, Ra vic, Tapas K. Kundu
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5023d83ec2a5ceab2bac53b66c2b863c5e8ad9" target='_blank'>
              Non-centromeric CENP-A epigenetically regulates epithelial-mesenchymal plasticity and heterogeneity in human cells
              </a>
            </td>
          <td>
            Charlène Renaud-Pageot, Daniele Capocefalo, Sébastien Lemaire, Audrey Forest, Laura Cantini, G. Almouzni
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf8d0c3dbf667b289968c8f0d2992da2f2099762" target='_blank'>
              Distinct ATRX functions cooperate with 9-1-1 and CST complexes to safeguard replication and telomere integrity
              </a>
            </td>
          <td>
            Sandra Segura-Bayona, Marija Maric, Tohru Takaki, Zornitsa Manova, Shudong Li, Tyler H Stanage, Aurora I Idilli, Graeme Hewitt, P. Ruis, Rhona Millar, Harshil Patel, Steven Howell, Panagiotis Kotsantis, Michael Howell, S. Boulton
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Ovarian cancer (OVCA) is third most lethal gynecologic cancers and acquired chemoresistance is the key link in the high mortality rate of OVCA patients. Currently, there are no reliable methods to predict chemoresistance in OVCA. In our study, we identify genes, pathways and networks altered by DNA methylation in high-grade serous ovarian carcinoma (HGSC) cells that are associated with chemoresistance and prognosis of HGSC patients. We performed methylome-wide profiling using Illumina Infinium MethylationEPIC BeadChip (HM850K) methylation array on a set of HGSC chemoresistant and chemosensitive cell lines. Differentially Methylated CpG Probes (DMPs) were identified between the resistant and sensitive groups in HGSC. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) over-representation analyses were conducted to identify both common and unique pathways between resistant and sensitive cells. While the HM850K array was used for the discovery phase to identify differentially methylated probes and regions in HGSC cell lines, the publicly available The Cancer Genome Atlas ovarian cancer (TCGA-OV) dataset generated using the Illumina Infinium HumanMethylation27 BeadChip (27 K array) methylation array served as an independent validation cohort for downstream survival and drug sensitivity analyses. Machine learning methods were applied to our dataset to predict drug sensitivity in the TCGA-OV cohort and to investigate associations with overall survival and progression-free survival. Kaplan-Meier analysis was performed to assess the relationship between differentially methylated genes and patient survival outcomes. The overlapping CpG probes shared between the two Illumina platforms were used for machine learning and survival analyses. Data visualization was performed using various R/Bioconductor packages. Our analysis identified a total of 3,641 DMPs spanning 1,617 differentially methylated genes between chemoresistant and sensitive HGSC cells, whereas 80% of them were hypermethylated CpG sites associated with HGSC resistant cells. Approximately half of the DMPs were distributed on chromosomes 1–3, 6, 11–12 and 17 and top identified hypermethylated CpGs were cg21226224 (SOX17, ∆β = 79%, adj.P = 7.73E-03), cg02538901 (ATP1A1, ∆β = 75%, adj.P = 7.6E-03), and cg17032184 (CD58, ∆β = 64%, adj.P = 4.39E-02). Machine learning analysis identified significant association of global hypermethylation in the HGSC chemoresistant cells with poor overall and progression-free survival of HGSC patients. Further analysis identified four differentially methylated genes (CD58, SOX17, FOXA1, ETV1) that were also positively associated with poor prognosis of HGSC OC patients. Functional enrichment analysis showed enrichment of several cancer-related pathways, including phosphatidylinositol signaling, homologous recombination and ECM-receptor interaction pathways. This study supplements the current knowledge of the underlying mechanism behind acquired chemoresistance in OVCA. Four differentially methylated genes identified in this study may have the potential to serve as promising epigenetic clinical biomarkers for HGSC chemotherapy resistance. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20827-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488e6c0c8f72e40ef843295838b808f3ae35f44c" target='_blank'>
              Global DNA methylation signatures associated with chemoresistance and poor prognosis of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hafiza Bushra Manzoor, Melisa Acun, Hamed Yasavoli-Sharahi, Chantal Matar, A. Mer, Benjamin K. Tsang, Meshach Asare-Werehene
          </td>
          <td>2025-10-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f04f35397873ebcf0a05890d5a35b22af43957" target='_blank'>
              LncRNA LINC00941 Links Oncogenic KRAS Signaling to Aggressiveness and Chemoresistance in Pancreatic Cancer
              </a>
            </td>
          <td>
            Thalita Bueno Corrêa, Gabriel Lucas da Fonseca, Rauni Borges Marques, S. M. Gomes-Filho, Lutero Hasenkamp, Y. Magalhães, L. Russo, N. Hoch, F. Forti, Elena Levantini, E. M. Reis, D. S. Bassères
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49580bd801398c666e64f8508dfb8bf04f7a7c7d" target='_blank'>
              Epidrugs in cancer: mechanisms, applications, and future direction.
              </a>
            </td>
          <td>
            Roshni Bibi, Lakshmi Varshetha, Rakshit Kamal Lahary, Jahnavi Namburi, Fardeen Ahmed Laskar, Koustav Sarkar
          </td>
          <td>2025-10-08</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin "TE graveyards" can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA "recycling yards." Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards.
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA research : an international journal for rapid publication of reports on genes and genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Results Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. Conclusions We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03787-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1499e0bd4fc727a3ec07930d21e6808e9e69593" target='_blank'>
              Global Mapping of Combinatorial Chromatin Regulatory Events Using Hi-Plex CUT&Tag
              </a>
            </td>
          <td>
            Yuan Liao, Weiqiang Zhou, , Kuai Yu, Zixiao Jolene Wang, Thomas Drummond, Meng Ding, Yingming Zhao, Ignacio Pino, Sean D. Taverna, Hongkai Ji, Heng Zhu
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Macrophages, the central players of innate immunity, control invading microbes by encapsulating them inside the phagosome, a nutrient-poor, reactive oxidant species-rich organelle. Nevertheless, some microbes, including the opportunistic yeast pathogen Candida glabrata, noted for its karyotype diversity, rapid evolution of antifungal drug resistance, and lack of meiosis, can survive and even replicate inside macrophages. However, it is not fully understood how C. glabrata responds to macrophage engulfment, and it is unknown how this presumably DNA-damaging environment influences the pathogen’s genome stability. In this study, we used comparative transcriptomics to identify amino acid starvation and DNA damage as conditions eliciting C. glabrata responses most similar to macrophage engulfment. Consistent with this, we found that C. glabrata intra-macrophage survival and replication require master regulator of amino acid biosynthesis GCN4 and functional DNA double-strand break repair. Furthermore, comet assays provided the first direct evidence for increased DNA breaks in intra-macrophage yeast, and pulse-field gel electrophoresis showed that chromosomal alterations occur frequently in macrophage-passaged C. glabrata. Interestingly, these alterations could not be resolved by long read DNA sequencing, suggesting that they involved highly complex repetitive regions. Finally, we identified several point mutations emerging during macrophage passaging and showed that among them, a frameshift in RME1 (repressor of meiosis in Saccharomyces cerevisiae), increased C. glabrata intra-macrophage fitness. Together, these analyses point to amino acid deprivation, reveal elevated DNA breakage and chromosome instability, and raise intriguing questions about the role of meiotic gene orthologs in C. glabrata persisting and replicating within macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25e5e8836cf2996759f61e3d723daae16788359" target='_blank'>
              Candida glabrata replicating within macrophages experiences amino acid deprivation, DNA damage, and chromosome instability
              </a>
            </td>
          <td>
            Ariel A. Aptekmann, Nathaly Cabrera, Melody C. Hayman, Ruth Watson, Joseph A. Stewart, Christopher Quinteros, Chang Liu, Caroline E. Hayter, Mikhail V. Keniya, David S. Perlin, J. L. Argueso, Erika Shor
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background The advent of poly (ADP-ribose) polymerase inhibitors (PARPi) over the past decade has significantly altered the management of epithelial ovarian cancer (EOC). We proposed that the etiology of homologous recombination deficiency (HRD) might underlie the variable responses to PARPi observed across patient populations. Methods As part of the phase 2 study of the Chinese HRD Harmonization Project, we developed a genomic methylation sequencing (GM-seq) pipeline facilitated by the TET enzyme for the simultaneous identification of methylated modifications and genetic variations in EOC tumor samples, and compared with established DNA sequencing-based HRD assays. Results Somatic mutation and HRD scores were confounded by low tumor purity in our cohort of 98 locally advanced/advanced EOC patients. In samples with tumor purity ≥ 30% (n = 45), the GM-seq pipeline showed high consistency with DNA sequencing-based HRD assay, identifying genetic variations in homologous recombination repair (HRR) genes and HRD score with 92.6% (25/27) and 97.1% (33/34) consistency respectively, in addition to conducting methylation profiling. Moreover, different underlying mechanisms of HRD were associated with varying degrees of PARPi efficacy, with BRCA1/2 LOH group having the best efficacy (median PFS, undefined), followed by BRCA1 methylation group (median PFS, 23.4 months), and those with unknown etiology of HRD having the worst efficacy (median PFS, 8.8 months, p < 0.001). Conclusion Our findings underscore the importance of considering HRD etiology when evaluating PARPi efficacy in EOC patients. The GM-seq pipeline, represents a significant advancement in HRD detection, enabling more accurate predictions of PARPi response. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00843-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5213220b5e2f3e35a5d8562e34edf897da4714a0" target='_blank'>
              Comprehensive assessment of homologous recombination deficiency via simultaneous methylation and mutation analysis in epithelial ovarian cancer: implications for PARP inhibitors efficacy
              </a>
            </td>
          <td>
            Lin Dong, Huanwen Wu, Ning Li, Wenbin Li, Yan Song, Yu Xiong, Huan Yin, Huan Fang, Rongrong Chen, Xin Yi, Jie Huang, Jianming Ying
          </td>
          <td>2025-10-10</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Many gene fusions have been implicated in the development and progression of human cancers. Our group and others have identified specific gene fusions involving the MAML2 gene in human cancers, and we were interested in searching for additional novel MAML2 fusions. Our analysis with the Caris Life Sciences molecular database demonstrated that most of the novel fusions were associated with increased fusion burden and genomic loss of heterozygosity, which are potentially suggestive of genomic instability, except for ATXN3::MAML2 which warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa520a41486a278de5084f190cbe8e828a82921" target='_blank'>
              Novel MAML2 Fusions in Human Malignancy
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao H. Huang, A. Crymes, E. S. Antonarakis, Andrew Elliott, Matt Oberley, Mark G. Evans
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41521f18f4da65cbb988d3434d4d07459d2803e" target='_blank'>
              Epigenetic characterization of pseudogenes across human tissues
              </a>
            </td>
          <td>
            Yunzhe Jiang, Beatrice Borsari, Mark B. Gerstein
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Among them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed a synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2446ce02999836acb1c038122ee522f1878bf0c3" target='_blank'>
              TopBP1 biomolecular condensates as a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, B. Bordignon, C. Hassen-Khodja, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Laurent Chaloin, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2025-10-21</td>
          <td>eLife</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Human leukocyte antigen (HLA) polymorphism hinders off-the-shelf cell therapies. We developed REWRITE, a modular platform for iterative, scar-minimized genome writing of synthetic constructs >100 kb in human pluripotent stem cells (hPSCs). Using REWRITE, we deleted 105–209 kb of the HLA locus and installed synthetic 24 kb or 100 kb HLA haplotypes, and a 62 kb antigen-processing locus. This uncovered a persistent, heritable “epigenetic ghost” — an active state lingering despite genetic removal — whose resolution to a silenced default state is driven by native intergenic DNA. These loci restored inducible expression in key lineages, sparing cells from NK-mediated killing and establishing HLA-matched T-cell tolerance, enabling off-the-shelf cell therapies. REWRITE facilitates extensible programming of multigenic functions in allogeneic human cells — from immune design to genome architecture discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3662bdeb8a587004f28d0c02911c74aa245dde2b" target='_blank'>
              Human Genome REWRITE for Off-the-Shelf Stem Cells Reveals an “Epigenetic Ghost”
              </a>
            </td>
          <td>
            Serena F. Generoso, Sarah Levovitz, Susanna Jaramillo, Minjoo Kim, Sumanth Dara, Shean Fu Phen, Bryan Yi, Tomoki Yanagi, Thomas L. DesMarais, Neta Agmon, Megan S. Hogan, Leslie A. Mitchell, David M. Truong
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The genome stores and processes approximately 1.5 gigabytes of encoded information. In this article, we propose that the eukaryotic genome and its adaptable three-dimensional packing in the form of chromatin offer a valuable template for the system architecture of DNA-based digital computers. We examine embryonic and stem cells, which exhibit distinct chromatin-associated condensates enriched in transcription machinery. These dynamic biomolecular condensates facilitate the spatial association of genes, genomic control elements, and molecular machinery responsible for reading the genomic code. Drawing a compelling analogy to the von Neumann computer architecture-which integrates storage, processing, and memory in most electronic computers-we reflect on how the operational principles of these condensates could inspire the design of a similar architecture for future DNA computers. In particular, we describe how one could recreate such an architecture by exploiting the process of surface condensation, which underlies the formation of chromatin-associated condensates. We conclude by reviewing our initial steps of constructing synthetic DNA nanostructures that follow the same operational principles and enable programmable surface condensation. Finally, we outline how computational methods from accelerated materials design could further advance the development of DNA computer system architectures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193d2f579561bdfc3005e20f9a7dfc04b29d9f5f" target='_blank'>
              Chromatin-associated condensates as an inspiration for the system architecture of future DNA computers.
              </a>
            </td>
          <td>
            Lennart Hilbert, Aaron Gadzekpo, Simon Lo Vecchio, Mona Wellhäusser, Xenia Tschurikow, Roshan Prizak, Barbara Becker, Sandra Burghart, Ewa Anna Oprzeska-Zingrebe
          </td>
          <td>2025-09-05</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Formaldehyde (FA) is a highly reactive aldehyde that forms a variety of adducts with biomolecules, including DNA base adducts, interstrand crosslinks, intrastrand crosslinks, and DNA–protein crosslinks (DPCs). FA is produced from several exogenous and endogenous sources and has been linked to many cancer types. Persistent adducts are often bypassed by error-prone translesion synthesis (TLS) polymerases, causing mutations that contribute to the FA-induced mutational spectrum. However, it is unknown how DPC repair and TLS coordinate to bypass FA damage, and how bypass by different TLS polymerases alters FA’s mutational spectrum. Here, we use an established yeast mutational reporter system to confirm that FA preferentially mutagenizes guanine residues within single-stranded DNA (ssDNA). We find that functional TLS prevents chromosomal instability following FA exposure and that FA-induced chromosomal rearrangements are dependent on DPC repair. Finally, we find that the TLS polymerases Rev1 and Pol η function together to prevent FA mutagenesis. Overall, these results outline the mechanisms by which TLS and DPC repair coordinate to bypass FA-induced ssDNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b38028676b90bbc4fd75eb225bb7dbf166a301" target='_blank'>
              Translesion polymerases and DNA–protein crosslink repair shapes the cellular response to formaldehyde-induced DNA damage in ssDNA
              </a>
            </td>
          <td>
            Thomas Blouin, Elizabeth Ampolini, Kristina Stayer, Parameshwar Suresh, P. A. Mieczkowski, Natalie Saini
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a91328f52e5aebb03d09a37f45c4b88f435693" target='_blank'>
              DDIAS is a single-stranded DNA-binding effector of the TOPBP1-CIP2A complex in mitosis
              </a>
            </td>
          <td>
            Kaima Tsukada, Liudmyla Lototska, Aya Tsukada, Ipek Ilgin Gönenç, Charlotte A Sherlaw-Sturrock, Manil Kanade, Antony W. Oliver, , Satpal S. Jhujh, Julius Bannister, Antoine Pouzet, Jonathan Lévy, Asmat Ali, Muhammad Raza, Mathias Toft, Zafar Iqbal, Ambrin Fatima, Thomas C. R. Miller, Grant S. Stewart, Fena Ochs, Andrew N. Blackford
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are widely existing stable DNA secondary structures in mammalian cells. A long-standing hypothesis is that timely resolution of G4s is needed for efficient and faithful DNA replication. In vitro, G4s may be unwound by helicases or alternatively resolved via DNA2 nuclease mediated G4 cleavage. However, little is known about the biological significance and regulatory mechanism of the DNA2-mediated G4 removal pathway. Here, we report that DNA2 deficiency or its chemical inhibition leads to a significant accumulation of G4s and stalled replication forks at telomeres, which is demonstrated by a high-resolution technology: Single molecular analysis of replicating DNA (SMARD). We further identify that the DNA repair complex MutSα (MSH2-MSH6) binds G4s and stimulates G4 resolution via DNA2-mediated G4 excision. MSH2 deficiency, like DNA2 deficiency or inhibition, causes G4 accumulation and defective telomere replication. Meanwhile, G4-stabilizing environmental compounds block G4 unwinding by helicases but not G4 cleavage by DNA2. Consequently, G4 stabilizers impair telomere replication and cause telomere instabilities, especially in cells deficient in DNA2 or MSH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695dd3fcaf795cd574da62afbe86a1c81c4f33d6" target='_blank'>
              DNA2 and MSH2 cooperatively repair stabilized G4 and allow efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Yi Lei, Nian Liu, Steven Esworthy, Changxian Shen, Helen Liu, Jessica D Hess, Hang Yuan, Guojun Shi, Mian Zhou, Lei Shen, Sufang Zhang, Settapong T. Kosiyatrakul, Vikas Gaur, J. Sommers, Nityanand Srivastava, Winfried Edelmann, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="NUPTs are DNA sequences of plastid origin present in plant nuclear genomes to varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analysis from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are i) formed by both continuous and episodic mechanisms; ii) unevenly represented across the plastid genome; iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA and regulatory RNA genes; iv) differentially contributing to structural genes; and v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which do not only involve their continuous formation but also episodic, highlight their role as a major source of non-coding RNA genes and other genomic features and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and ongoing mechanisms drive plastid-derived nuclear DNA evolution in angiosperms.
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily L. Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Centromeres are fundamental chromosomal structures that ensure accurate chromosome segregation during cell division. Despite their conserved and essential role in maintaining genomic stability, centromeres are subject to rapid evolutionary change. At the heart of centromere identity is the histone H3 variant CENP-A, an epigenetic mark that defines and propagates active centromeres and is essential for their function. Recent evidence supports a rapid evolution of centromere DNA sequences but also suggests a certain degree of flexibility in CENP-A deposition and propagation. The phenomenon of centromere drift, recently observed in humans, highlights how the dynamic repositioning of CENP-A and associated epigenetic environment over time maintains a regulated equilibrium, ensuring centromere function despite positional variation. Understanding these processes is crucial for unraveling centromere dynamics and their broader implications for genome stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0e9c5778af501dce621770da4beeabba7210f5" target='_blank'>
              The dynamic centromere
              </a>
            </td>
          <td>
            Angela Enriquez, Yael Nechemia-Arbely
          </td>
          <td>2025-10-16</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer (BRCA) remains the most common malignancy and the leading cause of cancer-related death among women worldwide. Recent studies have highlighted dysregulated copper homeostasis as a contributor to tumorigenesis, with cuproptosis—a copper-dependent form of regulated cell death—emerging as a potential therapeutic target. In this study, we systematically evaluated the prognostic significance of cuproptosis-related genes (CRGs) in BRCA by integrating multi-omics data from TCGA and GEO cohorts. Through weighted gene co-expression network analysis (WGCNA), we identified four key CRGs (CCDC24, TMEM65, XPOT, and NUDCD1) to construct a prognostic signature. The resulting risk score effectively stratified patients into high- and low-risk groups, with the high-risk group showing significantly worse overall survival, higher TP53 mutation frequency, and features of an immunosuppressive tumor microenvironment (TME). Functional enrichment analyses further implicated these genes in immune evasion and metabolic reprogramming pathways. Single-cell RNA sequencing (scRNAseq) confirmed heterogeneous expression of the signature genes across distinct cell populations, supporting their involvement in both tumor biology and immune modulation. Clinically, this CRG-based model provides a promising tool for individualized risk assessment and treatment planning. High-risk patients may benefit from intensified therapies or immunomodulatory strategies, while low-risk individuals could be considered for treatment de-escalation. In summary, our findings suggest that cuproptosis may play a regulatory role in BRCA progression and offer a novel, clinically actionable framework for prognostic stratification. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03525-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/820ee732493deee1307b1b616f142ac8b963048d" target='_blank'>
              Integrating single-cell and bulk RNA sequencing data establishes a cuproptosis-related gene predictive signature in breast cancer
              </a>
            </td>
          <td>
            Rongrong Peng, Yu Li, Bo Xu
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7f9e82789ed7b8469f2663f27086e180c73bd6" target='_blank'>
              The mechanical code of DNA impacts its interaction with DNA gyrase
              </a>
            </td>
          <td>
            Bailey G. Forbes, Rosella Pinckney, Sisi Liu, Aditi Biswas, Heather Potter, Elizabeth R. Morris, Aakash Basu-Biswas
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetically encoded DNA recorders convert transient biological events into stable genomic mutations, offering a means to reconstruct past cellular states. However, current approaches to log historical events by modifying genomic DNA have limited capacity to record the magnitude of biological signals within individual cells. Here, we introduce MitoScribe, a mitochondrial DNA (mtDNA)-based recording platform that uses mtDNA base editors (DdCBEs) to write graded biological signals into mtDNA as neutral, single-nucleotide substitutions at a defined site. Taking advantage of the hundreds to thousands of mitochondrial genome copies per cell, we demonstrate MitoScribe enables reproducible, highly sensitive, non-destructive, durable, and high-throughput measurements of molecular signals, including hypoxia, NF-κB activity, BMP and Wnt signaling. We show multiple modes of operation, including multiplexed recordings of two independent signals, and coincidence detection of temporally overlapping signals. Coupling MitoScribe with single-cell RNA sequencing and mitochondrial transcript enrichment, we further reconstruct signaling dynamics at the single-cell transcriptome level. Applying this approach during the directed differentiation of human induced pluripotent stem cells (iPSCs) toward mesoderm, we show that early heterogeneity in response to a differentiation cue predicts the later cell state. Together, MitoScribe provides a scalable platform for high-resolution molecular recording in complex cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03af0fb96ee61d1661672c9f2ce3319d8f9ca337" target='_blank'>
              MitoScribe single-cell molecular recorder logs graded signaling dynamics into mitochondrial DNA
              </a>
            </td>
          <td>
            Linhan Wang, Nikolaos Poulis, Deepak Srivastava, Seth L Shipman
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ∼2.5×10-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G. Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Centromeres, characterized by their unique chromatin attributes, are indispensable for safeguarding genomic stability. Due to their intricate and fragile nature, centromeres are susceptible to chromosomal rearrangements. However, the mechanisms preserving their functional integrity and supporting nucleus homeostasis following breakages remained enigmatic. In this study, we use wheat ditelosomic stocks, which arise from centromere breakage, to explore the genetic and epigenetic alterations in damaged centromeres. Our investigations unveil novel chromosome end structures marked by de novo addition of telomeres, as well as localized chromosomal shattering, including segment deletions and duplications near centromere breakpoints. We reveal that the damaged centromeres possess a remarkable capacity for self-regulation, through employing structural modifications such as expansion, contraction, and neocentromere formation to maintain their functional integrity. Centromere breakage triggers nucleosome remodeling and is accompanied by local transcription changes and chromatin reorganization, and subsequently may contribute to the stabilization of broken chromosomes. Our findings highlight the resilience and adaptability of plant chromosomes in response to centromere breakage, and provide valuable insights into the stability of centromeres, thereby offering promising prospects to manipulate centromeres for targeted chromosomal innovation and crop genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c301bcc4744c9e967b82a2d0c48310497c9c67e9" target='_blank'>
              Adaptation of centromere to breakage through local genomic and epigenomic remodeling in wheat.
              </a>
            </td>
          <td>
            Jingwei Zhou, Yuhong Huang, Huan Ma, Yiqian Chen, Chuanye Chen, Fang Han, H. Su
          </td>
          <td>2025-09-30</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Targeting cancer cell plasticity through chromatin organization is an emerging research area, yet the molecular mechanisms that govern chromatin loop formation remain unclear. Here, we develop a CRISPR screen based on our engineered live-cell CTCF-cohesin contact reporters to identify regulators of chromatin loops. Our findings reveal that tousled-like kinase 2 (TLK2) functions as a key regulator of chromatin loop formation during the cancer stemness transition. Mechanistically, TLK2 phosphorylates DYNLL1, enhancing its interaction with CTCF to promote CTCF-cohesin hub formation at the KLF4 locus. Suppressing TLK2 impairs cancer stemness plasticity, sensitizes cancer cells to cytotoxic stress in vitro, and reduces lung metastases and enhances immunotherapy response in breast cancer mouse models. Clinically, elevated TLK2 expression correlates with poor prognosis in breast cancer patients. Collectively, these findings identify TLK2 as a potential therapeutic target for mitigating cancer stemness plasticity, highlighting chromatin loop-targeting therapy as a promising strategy to eradicate cancer stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040d9374168c648d367bfc7319dadc6870704688" target='_blank'>
              Chromatin looping-based CRISPR screen identifies TLK2 as chromatin loop formation regulator in cancer stemness plasticity
              </a>
            </td>
          <td>
            Zifeng Wang, Fang Liu, Nana Chen, Jingjing Wu, Xinhao Li, Mouxiang Fang, Min Yan, Ji Zhang, Bing Deng, Lulu Wang, Xuan Wang, Meiling Liu, Deshun Zeng, Zhengzhi Zou, Bo Wang, Songyang Zhou, Bin He, Quentin Liu
          </td>
          <td>2025-10-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a60db9a10da32fa1ae50df98fedcabbcf5befcb" target='_blank'>
              Mutation bias in driver genes reveals the distribution of effects of oncogenic mutations
              </a>
            </td>
          <td>
            Anastasia V. Stolyarova, G. Bazykin
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Chromosome organization is critical for the maintenance of genetic integrity. Most studies of bacterial nucleoids have focused on growing rod-shaped organisms. Studying nucleoid dynamics during Myxococcus xanthus development offers the unique opportunity to investigate the localization of two sister chromosomes as rod-shaped cells transition into round spores. During starvation-induced multicellular development, DNA replication is required for M. xanthus rods to transition into spores with two copies of the chromosome. Here, we report novel approaches using confocal fluorescence microscopy to observe the chromosome number and arrangement, and nucleoid localization in developing cells in situ. We discovered that sister chromosomes are present in some rods and transitioning cells (TCs) early in development. The arrangement of the two chromosomes in developing cells was novel compared to predivisional growing cells studied previously. We observed segregated nucleoids in ~40% of TCs and spores. The majority of TCs contained a crescent-shaped nucleoid along one side, perhaps due to ongoing chromosome segregation, whereas most spores appeared to have undergone nucleoid decondensation. During unicellular glycerol-induced sporulation of M. xanthus, we observed segregated nucleoids in only ~10%–20% of TCs and spores. In addition, early in starvation-induced development, we discovered a subpopulation of cells that may be spheroplasts destined for lysis, which is the fate of most cells under these conditions. Chromosome segregation in developing M. xanthus may be a bet-hedging strategy to increase survival under different conditions and/or an evolutionary remnant of ancestral events that included cell division to produce spores with one copy of the chromosome. IMPORTANCE The cell cycle normally involves DNA replication, chromosome segregation, and cell division. During starvation-induced Myxococcus xanthus development, DNA replication is necessary for progression to spore formation, which occurs without cell division, resulting in spores with two copies of the chromosome. The organization of sister chromosomes during the morphological change of rod-shaped cells into round spores was unknown. We discovered that the two nucleoids often segregate during the transition from rods to spores. Mature spores contained decondensed nucleoids. Our observations raise important questions about the mechanism of chromosome segregation during M. xanthus development and the reason for its existence. We also discovered a subpopulation of developing cells with characteristics suggesting they are spheroplasts on the verge of cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe53b6817f634726006f01b89cdd1037a4dfb40" target='_blank'>
              Segregation of sister chromosomes during the shape change of developing Myxococcus xanthus cells
              </a>
            </td>
          <td>
            Y. Hoang, Yann S Dufour, Lee Kroos
          </td>
          <td>2025-09-26</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20084-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Neural tube defects (NTDs) are one of the most common congenital malformations. Folic acid deficiency in pregnant women increases the risk of developing NTDs; however, the underlying etiology and mechanisms remain elusive. In this study, the role of DNA double-strand breaks (DSBs) in 3D genome organization in NTDs with folate deficiency is reported. The NTD mouse model is burdened with abundant DSBs associated with the disruption of 3D genome organization. DSBs occurring in active genes lead to the stalling of RNA polymerase II (Pol II) and formation of R-loops in the 3D genome. The DSB ratios of the genomic regions negatively correlated with the distance from the transcription start sites of the gene. The DSB ratios of the proximal and distal enhancers are significantly higher and induce the displacement of loops with busy anchors. Furthermore, DSB-associated dysregulation of chromatin loops occurs in neural tube closure-associated genes that are abnormally expressed in human NTDs. Taken together, excessive DSB-associated 3D genome organization disruption within NTDs with folate deficiency contributes to the dysregulation of neural tube closure-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8659be399a2187d7c9eb42445c50713cbee2a3dc" target='_blank'>
              Excessive DNA Double-Strand Breaks-Associated 3D Genome Reorganization Contributes to Neural Tube Defects with Folate Deficiency.
              </a>
            </td>
          <td>
            Ting Zhang, Lin Lin, Jianting Li, Caihua Li, Shengjun Liang, Xuemei Bai, Fang Wang, Y. Bao, Dan Guo, Xiaochen Bo, Hao Li, Hebing Chen, Qiu Xie
          </td>
          <td>2025-09-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background: Bladder cancer is rare in young women, and advanced presentations are exceptionally uncommon. We report a de-identified case of a previously healthy 31-year-old female who developed rapidly progressive stage IV bladder cancer within 12 months of completing a three-dose Moderna mRNA vaccination series (May 2021, June 2021, December 2021). Case Findings: Comprehensive multi-omic profiling was performed using PBIMA (Molecular Surveillance and Individualized Targeted Immunotherapy Peptide Editing) and REViSS (Spike-associated Transcription-al/Translational Instability Surveillance), incorporating analyses of plasma-derived circulating tumor DNA, whole-blood RNA, and urine exosome proteomics. Dysregulated gene expressions were identified across oncogenic driver genes (KRAS, ATM, MAPK1, NRAS, CHD4, PIK3CA, and SF3B1), auxiliary tumor-promoting signals (TOP1, PSIP1, and ERBB2), and broad evidence of genome instability with impaired DNA repair (ATM, MSH2). Within circulating tumor DNA, a host–vector chimeric read mapped to chr19:55,482,637–55,482,674 (GRCh38), in cytoband 19q13.42, positioned ~367 kb downstream of the canonical AAVS1 safe harbor and ~158 kb upstream of ZNF580 at the proximal edge of the zinc-finger (ZNF) gene cluster. This sequence aligned with perfect 20/20 bp identity to a segment (bases 5905–5924) within the Spike open reading frame (ORF) coding region (bases 3674–7480) of the Pfizer BNT162b2 DNA plasmid reference (GenBank accession OR134577.1), despite the patient only receiving Moderna vaccinations. This apparent paradox is best explained by shared Spike ORF sequences within the expression cassette across both vaccine platforms; because Moderna has not deposited its proprietary plasmid sequence in NCBI, BLAST defaults to Pfizer’s published reference as the nearest available match. The integration site was located outside the canonical AAVS1 “safe harbor” and within a gene-dense, recombination-prone regulatory region, raising concern for transcriptional disruption, fusion transcript formation, and oncogenic potential. The probability of a random 20-base sequence perfectly matching a predefined target is approximately 1 in a trillion, making this alignment statistically compelling and highly unlikely to be an incidental artifact. Conclusions: This sentinel case report provides the first documented evidence of genomic integration of mRNA vaccine-derived genetic material in a human subject, documenting a temporal association between COVID-19 mRNA vaccination and aggressive malignancy, reproducible multi-omic evidence of oncogenic signaling, and a non–safe harbor host–vector integration event. While causality cannot be established from a single case, the con-vergence of (i) close temporal proximity to vaccination, (ii) genomic integration of a vaccine plasmid–derived spike gene fragment, and (iii) consistent transcriptomic and proteomic instability across biospecimens represents a highly unusual and biologically plausible pattern. These findings highlight an urgent need for systematic genomic sur-veillance, orthogonal validation with long-read sequencing, and larger cohort studies to rigorously define the im-pact of synthetic mRNA vaccine platforms on genome integrity and cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8848f5c106afb2dfada322a3642c497ad8b443d" target='_blank'>
              Genomic Integration and Molecular Dysregulation in Aggressive Stage IV Bladder Cancer Following COVID-19 mRNA Vaccination
              </a>
            </td>
          <td>
            J. A. Catanzaro, Nicolas Hulscher, Peter A. McCullough
          </td>
          <td>2025-10-05</td>
          <td>International Journal of Innovative Research in Medical Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03020-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9f14cca351b80c4eaf8fa50610851b494729d1" target='_blank'>
              Noncanonical chromosomal-end-specific telomeric arrays in naturally telomerase-negative yeasts
              </a>
            </td>
          <td>
            Broňa Brejová, V. Hodorová, Hana Lichancová, A. Gafurov, Dominik Bujna, Filip Brázdovič, Filip Červenák, Tomáš Petrík, Eva Hegedűsová, M. Jakúbková, M. Neboháčová, Ľubomír Tomáška, Matthias Sipiczki, T. Vinař, J. Nosek
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DSCC1, a key subunit of the alternative RFC complex, is essential for DNA replication and genome stability. While its role in cancers like breast cancer is documented, its function in LIHC remains largely unexplored. This study comprehensively examines DSCC1's involvement in LIHC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6e78aa5a515dfa7feb5f0abf3158e965c6ce93d" target='_blank'>
              DSCC1 as a Novel Biomarker in Hepatocellular Carcinoma: Diagnostic Potential, Prognostic Impact, Correlation With Key Pathways, and Association With the Immune Landscape
              </a>
            </td>
          <td>
            Abdul Jamil Khan, Shad Man, Shahid Ullah Khan
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Gene Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1892593b4d09d954d617fd50a949c2f67aa1bce" target='_blank'>
              Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin
              </a>
            </td>
          <td>
            Agnisrota Mazumder, John Cooper, Can Goksal, J. S. Khanduja, Richard I. Joh, Johannes J. Groos, Rosario Y.J. Brockhausen, Junko Kanoh, Mohammad R. Motamedi, Ilya J. Finkelstein, Bassem Al-Sady, S. Braun
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1147c4ec4d5050e89e9fab52610e8f2d66fcc3cc" target='_blank'>
              Integrative multi-modal analysis reveals the contribution of noncoding RNAs to post-treatment progression of IDH-mutant astrocytomas
              </a>
            </td>
          <td>
            A. Hartewig, S. Jaatinen, M. Annala, J. Haapasalo, Wei Zhang, H. Haapasalo, M. Nykter, Xin Lai, K. Rautajoki
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Breast cancer (BCa) is a major contributor to female mortality worldwide. Treatment resistance and tumor heterogeneity contribute to the lack of effective therapeutic targets, posing a significant challenge in BCa management. CENPI, a core centromere protein involved in chromosome segregation, has emerging evidence implicating it in oncogenesis across diverse malignancies. However, its functional and molecular mechanisms in BCa remain unclear. Methods We analyzed CENPI expression and its clinical significance by using the BCa dataset from the Cancer Genome Atlas (TCGA) and immunohistochemical staining of 3 human BCa tissue samples. Cellular functional assays and mice xenograft models were utilized to assess the effects of CENPI on BCa growth. RNA sequencing combined with bioinformatics analysis was conducted to elucidate the molecular mechanisms underlying CENPI function, with further validation through Western blotting, immunofluorescence, and TOP/FOP flash assays. Results CENPI was aberrantly overexpressed in BCa, with elevated expression levels strongly associated with disease progression and poor prognosis. Functional assays demonstrated that CENPI significantly promoted breast carcinogenesis in both cellular and animal models. Mechanistically, CENPI increased BCa progression and malignant phenotypes by modulating the Wnt/β-catenin axis. Conclusions CENPI is a critical oncogene in BCa, driving tumorigenesis and disease progression via the Wnt/β-catenin axis, which represents a promising biomarker and therapeutic target for BCa. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04001-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33eef316f873dd58287f9f4d71241b7970f53004" target='_blank'>
              Centromere protein I facilitates breast cancer tumorigenesis and disease progression through modulation of Wnt/β-Catenin signaling
              </a>
            </td>
          <td>
            Chaoshen Wu, Yibing Zhou, Yuxiao Mu, Yijie Cheng, Wenqian Xu, Mengna Huang, Jingyuan Xu, Ming Li, Jianfeng Gu, Xuli Meng, Da Qian
          </td>
          <td>2025-10-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. Methods We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). Results We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Conclusions Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15053-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Pancreatic adenocarcinoma (PAAD) has a poor prognosis due to late diagnosis and limited treatment options. While protein-coding genes are known to contribute to disease pathogenesis, their specific roles require further investigation. Methods We analyzed RNA-seq and clinical data from 179 PAAD patients in TCGA to examine TICRR (TOPBP1 Interacting Checkpoint And Replication Regulator). Prognostic analyses included survival and risk assessment. Complementary cell-based experiments explored TICRR and associated non-coding RNAs (ncRNAs). Multi-omics profiling examined potential relationships between TICRR expression and tumor microenvironment features. Results Higher TICRR expression showed association with poorer survival outcomes. Experimental modulation of MIR659 and AC074099.1 expression appeared to influence malignant phenotypes in vitro. Multi-omics data suggested potential links between TICRR-associated molecular changes and features of immune evasion and genomic instability. Conclusion Our findings suggest possible connections between TICRR expression and PAAD progression. The observed phenotypic changes following ncRNA modulation may indicate their involvement in related oncogenic processes. These preliminary observations warrant further investigation into potential regulatory relationships between TICRR and associated ncRNAs, which could inform future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03803-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dcdb22943faec5ffead6df5f9d9a524741a08be" target='_blank'>
              Deciphering TICRR's oncogenic landscape in pancreatic adenocarcinoma: molecular insights and clinical implications
              </a>
            </td>
          <td>
            Ke‐Jie He, Haitao Wang
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28967441eb1f46f1a9e983b8df32cfb979b71b23" target='_blank'>
              Chromatin and gene-regulatory dynamics of human pulmogenesis by single cell multiomic sequencing
              </a>
            </td>
          <td>
            Samuel H. Kim, Soon Il Higashino, Lea C. Steffes, Li Li, Anca M. Pașca, Maya Kumar, Christin S. Kuo, Mark A. Krasnow, William J. Greenleaf
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Homology-directed DNA repair (HDR) is critical for genome stability and tumor suppression. HDR is initiated by the RAD51 single-stranded (ss)DNA nucleoprotein filament which conducts the homology search and invades a homologous DNA template, creating a displacement-loop (D-loop). The RAD51 filament is assisted in these processes by several proteins. One such protein is RAD51-Associated-Protein 1 (RAD51AP1) which binds DNA and RNA and directly interacts with RAD51. Of note, RAD51AP1 overexpression is associated with poor prognosis in several different cancer types. Here, we show that RAD51AP1 activity is regulated by phosphorylation. RAD51AP1 bearing S277/282A mutations is more proficient in the stimulation of D-loop formation than wild type RAD51AP1 or phosphomimetic RAD51AP1-S277/282D. In EMSAs, RAD51AP1 with S277/282A mutations more avidly binds ssDNA, double-stranded (ds)DNA, and the nucleosome core particle than wild type RAD51AP1 or RAD51AP1-S277/282D. In cells, RAD51AP1-S277/282A confers no rescue of RAD51AP1 deficiency in toxicity tests and DNA replication assays. In contrast, RAD51AP1-S277/282D fully rescues RAD51AP1 deficiency. We provide evidence that RAD51AP1-S277 is a CDK2 target and propose a model in which RAD51AP1-S277/282 phosphorylation ensures RAD51AP1 flexibility for dynamic engagement in consecutive steps of the HDR reaction. Our results provide new mechanistic insights into RAD51AP1 regulation by a CDK.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7032aca65f6e9576eb0613414324a154e6a31c" target='_blank'>
              Phosphoregulation of RAD51AP1 function in homology-directed repair
              </a>
            </td>
          <td>
            Neelam Sharma, Mollie E. Uhrig, Youngho Kwon, P. Sung, C. Wiese
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Telomeres protect chromosome ends and their length varies significantly between organisms. Because telomere length variation is associated with various biomedical and eco-evolutionary phenotypes, many biological fields are interested in understanding its biological significance. Here, we introduce Topsicle, a computational method that estimates telomere length from whole genome long-read sequencing data using k-mer and change-point detection analysis. Simulations show Topsicle is robust to sequencing errors and coverage. Application of Topsicle to plant and human cancer cells shows high accuracy and comparable results to direct telomere length measurements. We predict Topsicle will be a useful tool for studying telomere biology. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03783-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aff7ef6bf5d73645c58ce8d4d7e64383c77d646" target='_blank'>
              Topsicle: a method for estimating telomere length from whole genome long-read sequencing data
              </a>
            </td>
          <td>
            Linh Nguyen, Jae Young Choi
          </td>
          <td>2025-09-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="MYC is among the most frequently dysregulated oncogenes in human cancer, yet its direct targeting remains a significant challenge. Here, we present an in-silico integrative screening approach to identify compounds and combinations that can block MYC's oncogenic function by specifically disrupting its transcriptional regulatory function. Using a doxycycline (DOX)-inducible model, we established a MYC loss-of-function (LOF) gene signature that specifically captures the molecular consequences corresponding to the loss of MYC's ability in transcriptional regulation. By integrating large-scale post-perturbation transcriptomic profiling from the CMAP database, we screened over 8300 drug-induced profiles and identified 70 recurrent compounds that are predicted to antagonize MYC's transcriptional programs. To further enhance their therapeutic potential, we also developed an orthogonality analysis to pinpoint synergistic drug combinations that suppress MYC activity more effectively than single agents. Our scalable framework enables a rational and systematic identification of compounds with potential to antagonize MYC's oncogenic function by disrupting its transcriptional regulatory ability without necessarily decreasing its abundance. Our approach provides new insights on utilizing existing anticancer drugs to indirectly target MYC in MYC-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f19d2dd4a2e3e24bf8c740f6f2a3bb3a7026742" target='_blank'>
              An Integrative Drug-Induced Transcriptomic Analysis Identifies Novel MYC Antagonists and Potential Synergistic Drug Combinations.
              </a>
            </td>
          <td>
            Anthony Aceto, Yue Wang, Da Yang
          </td>
          <td>2025-09-30</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106cf167da77fcf718e98566d67aa1a48f009f0c" target='_blank'>
              Telomere-to-telomere gap-free genome assembly of a male donkey and the identification of novel SVs associated with functional genes
              </a>
            </td>
          <td>
            Tao Yang, Ran Yang, Yu Liu, Mo Feng, Yuanyuan Li, Hetong Zhang, Xinyu Wang, Ruyu Yao, Jiahui Wu, Wenhui Xing, Shiyu Qian, Chunjiang Zhao
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: The Structural Maintenance of Chromosomes (SMC) proteins form essential heterocomplexes for the preservation of DNA structure and its functions, and hence cell viability. The SMC5/6 dimer is assembled by direct interactions of ATP heads via the kleisin NSE4 bridge and by SMC hinges. The structure might be interrupted by a single point mutation within a conserved motif of the SMC6-hinge. We describe the phenomena associated with the impairment of the SMC5/6 complex with morphology, repair of DNA double strand breaks (DSB), mutagenesis, recombination and gene targeting (GT) in the moss Physcomitrium patens (P. patens). Methods: Using CRISPR/Cas9-directed oligonucleotide replacement, we have introduced two close G to R point mutations in the hinge domain of SMC6 of P. patens and show that both mutations are not toxic and allow viability of mutant lines. Results: The G514R mutation fully prevents the interaction of SMC6 not only with SMC5, but also with NSE5 and NSE6, while the mutation at G517R has no effect. The Ppsmc6_G514R line has aberrant morphology, spontaneous and bleomycin-induced mutagenesis, and maintenance of the number of rDNA copies. The most unique feature is the interference with gene targeting (GT), which is completely abolished. In contrast, the Ppsmc6_G517R line is close to WT in many aspects. Surprisingly, both mutations have no direct effect on the rate of DSB repair in dividing and differentiated cells. Conclusions: Abolished interactions of SMC6 with SMC5 and NSE5,6 partners, which allow DSB repair, but impair other repair and recombination functions, suggests also regulatory role for SMC6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644b9adc198c09e23fb04ece06d59bdd098a7d62" target='_blank'>
              The Phenotype of Physcomitrium patens SMC6 Mutant with Interrupted Hinge Interactions
              </a>
            </td>
          <td>
            K.J. Angelis, M. Holá, R. Vágnerová, Jitka Vaculíková, J. Palecek
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc35f325082fc81f070889e73b4bc3e46dd674e1" target='_blank'>
              Mitotic single-stranded DNA suppression by DDIAS
              </a>
            </td>
          <td>
            Yibo Xue, Faisal Bin Rashed, Daniel Y. L. Mao, K. Abe, Z. Lin, Dheva Setiaputra, Lisa Hoeg, Clara Bonnet, Frank Sicheri, A. Gingras, Daniel Durocher
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b870d32a021571695dc3e8913c757c0de67eb433" target='_blank'>
              Deciphering glioma susceptibility
              </a>
            </td>
          <td>
            A. Fratton, B. Bonev
          </td>
          <td>2025-10-01</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a4e0e41973b219958443ce76f542cfd1955a71" target='_blank'>
              Investigations into fission yeast chromosome size determinants
              </a>
            </td>
          <td>
            , T. Fallesen, Frank Uhlmann
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract BRCA2 is crucial for mediating homology-directed DNA repair (HDR) through its binding to single-stranded DNA (ssDNA) and the recombinases RAD51 and DMC1. Most BRCA2 orthologs have a canonical DNA-binding domain (DBD) with the exception of Drosophila melanogaster. It remains unclear whether such a noncanonical BRCA2 variant without DBD possesses a DNA-binding activity. Here, we identify a new noncanonical BRCA2 in the model plant Physcomitrium patens (PpBRCA2). We establish that PpBRCA2 is essential for genome integrity maintenance, somatic DNA double-strand break (DSB) repair, HDR-mediated gene targeting, and RAD51 foci recruitment at DNA break sites. PpBRCA2 is also critical for DSB repair during meiosis. Interestingly, PpBRCA2 interacts strongly with RAD51 but weakly with DMC1, suggesting a distinct meiotic function compared to other BRCA2 homologs. Despite lacking the canonical DBD, PpBRCA2 binds ssDNA through its disordered N-terminal region and efficiently promotes HDR. Our work highlights that the ssDNA binding capacity of BRCA2 homologs is conserved regardless of the presence of a canonical DBD and provides a deeper understanding of BRCA2’s functional diversity across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2557db1433754b9374705626a1590a3dc83de8b" target='_blank'>
              Moss BRCA2 lacking the canonical DNA-binding domain promotes homologous recombination and binds to DNA
              </a>
            </td>
          <td>
            Alice Chanteau, Suliane Quilleré, Arthur Crouset, Sreejith Allipra, Ulysse Tuquoi, P. Perroud, Simona Miron, P. Dupaigne, Sophie Zinn-Justin, Fabien Nogué, Rajeev Kumar
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract The O6-methylguanine-DNA methyltransferase (MGMT) plays a significant role in the pathogenesis and progression of glioma. Numerous enhancer variants, including those within the MGMT gene region and adjacent gene regions, have been found to be associated with cancer development and progression. We investigated the significance of enhancer variants located in the intergenic spacer far from the MGMT gene in relation to glioma susceptibility and progression. We recruited 402 glioma patients and 654 controls for this investigation using Sequenom MassARRAY genotyping. We identified a significantly elevated risk of glioma among carriers with the rs11016629 TG genotype compared to those with the GG genotype (OR = 1.41, 95% CI 1.03–1.93; P = 0.034). Subgroup analyses revealed that rs11016629 was significantly associated with glioma risk in subjects with WHO grade IV tumor (OR = 1.59, 95% CI 1.07–2.38; P = 0.023) and high-grade glioma (OR = 1.57, 95% CI 1.11–2.21; P = 0.011). Patients who underwent gross total resection with TG/TT genotypes exhibited a 2.66-fold higher risk of disease progression than GG carriers (HR = 2.66, 95% CI 1.23–5.79; P = 0.014). The study demonstrates that a MGMT enhancer variant rs11016629 contributes to both glioma susceptibility and progression. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s10571-025-01605-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ffd3767daacb020b869e9275c5c5550f74b90c" target='_blank'>
              A MGMT Enhancer Variant is Associated with Glioma Susceptibility and Progression
              </a>
            </td>
          <td>
            Shu Zhang, Yiji Chen, Wenshen Xu, Danfang Yan, Meiqin Chen, Jiayi Zheng, Liming Huang
          </td>
          <td>2025-10-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a83ebfb77b99a082f33dfac7fdc4e834369069a" target='_blank'>
              Gastric epithelium from BRCA1 and BRCA2 carriers harbors increased double-stranded DNA damage and augmented growth
              </a>
            </td>
          <td>
            Kole H. Buckley, Michaela Dungan, Kevin Dinh, Gregory M. Kelly, Ryan Hausler, Kate E. Bennett, Daniel Clay, Julia E. Youngman, Ariana D. Majer, Keely Beyries, Blake A. Niccum, Sydney M. Shaffer, Tatiana A. Karakasheva, Kathryn E. Hamilton, M. Kochman, Gregory G Ginsberg, N. Ahmad, Kara N Maxwell, Bryson W. Katona
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b73f806f958dd1db9fdcf8f058c71382d2407" target='_blank'>
              Antagonistic histone H2A variants and autonomous heterochromatin formation shape epigenomic patterns in Arabidopsis
              </a>
            </td>
          <td>
            Shoko Oda, Sayaka Tominaga, Shumpei Takeuchi, A. Osakabe, Akira Kawabe, Frédéric Berger, T. Kakutani, T. To
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Waleed K. Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a2eaeac8d39964b8ce29ff957548c6f095cc6b" target='_blank'>
              Beckwith-Wiedemann syndrome multiomic analysis of hepatoblastoma uncovers unique tumor heterogeneity and cellular landscapes, including transition cells leading to tumor formation
              </a>
            </td>
          <td>
            Snehal Nirgude, Elisia D Tichy, Yuanchao Zhang, Rose D Pradieu, Michael Xie, Kathrin M. Bernt, Suzanne Macfarland, J. Kalish
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Loss of DNA mismatch repair (MMR) increases mutagenesis and tumorigenesis. mutS homolog 2 (MSH2), a central component of the MMR pathway, is essential for correcting base–base mismatches and insertion/deletion loops during DNA replication. To investigate how Msh2 deficiency cooperates with oxidative stress to drive mutagenesis and tumorigenesis, we employed an rpsL reporter gene assay using normal tissues before tumor development following treatment with an oxidizing agent. Results The background mutation frequency in the small intestines of Msh2-/- mice was over 20-fold higher than that of wild-type mice. In addition to G > A base substitutions, frequent 1-bp deletions in adenine mononucleotide repeats ((A)n) in the rpsL gene were observed. Potassium bromate treatment further increased the mutation frequency, particularly insertion-deletion mutations (indel), in the normal small intestinal epithelium of Msh2-/- mice before tumor development. Mutation signature analysis from next-generation sequencing data revealed that signatures associated with MMR deficiency (SBS15, SBS44, and ID2) and clock-like processes (SBS1 and SBS5) were consistently detected across all Msh2-/- tumors, similar to those observed in human MMR-deficient cancers. ID2, which involves 1-base deletions occurring in (A/T)n tracts of six bases or longer, supports the findings of the rpsL assay. Microsatellite instability (MSI) analysis showed that indel mutations at (A)n loci detected using the rpsL assay reflect genome-wide MSI. Msh2-/- tumors frequently harbored driver mutations, such as frameshift mutations in short tandem repeats within Apc and G > A substitutions in Ctnnb1, both of which activate the Wnt signaling pathway. Oxidative stress further accelerated these mutational processes. Conclusion Oxidative stress promotes repeat-associated mutagenesis, which manifests as MSI and base substitutions in MMR-deficient intestinal tissues, thereby enhancing the mutator phenotype and increasing the overall mutation burden. This process can be sensitively captured using our rpsL assay, which serves as a functional indicator of MMR deficiency and replication instability in normal tissues before tumor formation. This increases the likelihood of driver mutations in oncogenes and tumor suppressor genes, ultimately accelerating early tumorigenesis. This study demonstrated that MSH2 is essential for maintaining genome stability under oxidative conditions and functions as a key suppressor of oxidative stress–induced tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00342-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c307ffc3ea1fca1fb4bfcae167ad12e85c5a8ce" target='_blank'>
              Oxidative stress accelerates repeat sequence instability and base substitutions promoting gastrointestinal driver mutations in MSH2 deficient mice
              </a>
            </td>
          <td>
            M. Ohno, Noriko Takano, Kyoko Hidaka, Fumiko Sasaki, Yasunobu Aoki, Takehiko Nohmi, Teruhisa Tsuzuki
          </td>
          <td>2025-10-23</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-18</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The submandibular glands (SMGs), along with the parotid and sublingual glands, generate the majority of saliva and play critical roles in maintaining oral and systemic health. Despite their physiological importance, long-term therapeutic options for salivary gland dysfunction remain limited, highlighting the need for a deeper molecular understanding of SMG biology, particularly in humans. To address this knowledge gap, we have performed transcriptomic- and epigenomic-based analyses and molecular characterization of the human SMG. Our integrated analysis of multiorgan RNA-sequencing datasets has identified an SMG-enriched gene expression signature comprising 289 protein-coding and 75 long non-coding RNA (lncRNA) genes that include both known regulators of salivary gland function and several novel candidates ripe for future exploration. To complement these transcriptomic studies, we have generated chromatin immunoprecipitation sequencing (ChIP-seq) datasets of key histone modifications on human SMGs. Our epigenomic analyses have allowed us to identify genome-wide enhancers and super-enhancers that are likely to drive genes and regulatory pathways that are important in human SMG biology. Finally, comparative analysis with mouse and human SMG and other tissue datasets reveals evolutionary conserved gene and regulatory networks, underscoring fundamental mechanisms of salivary gland biology. Collectively, this study offers a valuable knowledge-based resource that can facilitate targeted research on salivary gland dysfunction in human patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d76e2d3fba6c49c8bced153f47a39c20dda0dbf" target='_blank'>
              Multimodal Exploration Offers Novel Insights into the Transcriptomic and Epigenomic Landscape of the Human Submandibular Glands
              </a>
            </td>
          <td>
            E. Horeth, T. Wrynn, Jason M. Osinski, Alexandra Glathar, J. Bard, Mark S. Burke, Saurin Popat, Thom Loree, Michael Nagai, Robert Phillips, J.L. Tapia, Jennifer Frustino, Jill M Kramer, Satrajit Sinha, R. Romano
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554ff6de17aca3fc14055b3c5a6203467aaf1fe3" target='_blank'>
              A gap-free telomere-to-telomere pig reference genome provides insight in centromere evolution and breed-specific selection
              </a>
            </td>
          <td>
            Dong Li, Yulong Wang, Tiantian Yuan, Xiang Li, Yu Quan, Wansheng Liu, K. Schachtschneider, Lawrence B. Schook, R. Crooijmans, M. Groenen, M. Derks, Taiyong Yu
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="


 Centrosome amplification (CA) is a hallmark of aggressive cancers, strongly linked to tumor progression, invasion, and poor clinical outcomes. While preclinical studies suggest CA plays a direct role in tumor initiation and metastatic spread, therapeutic strategies targeting CA remain limited. We previously identified Transforming Acidic Coiled-Coil Containing Protein 3 (TACC3) as a key regulator overexpressed in CA-positive tumors. TACC3 governs spindle integrity, clusters supernumerary centrosomes, and modulates transcriptional programs critical for the survival of CA-bearing cancer cells. However, the molecular mechanisms by which TACC3 drives invasion and metastasis remain largely unknown. In this study, we uncover TACC3 as a central mediator of metastatic progression in breast cancers with CA. Clinically, elevated TACC3 expression correlates with significantly worse metastasis-free survival in patients harboring CA-positive tumors. Functionally, using in vitro assays, transcriptomics, proximity labeling (APEX2-TACC3), and mass spectrometry, we show that TACC3 orchestrates cancer cell migration and invasion through novel interactions with cytoskeletal, vesicle trafficking, and leading-edge proteins. Importantly, pharmacologic inhibition with AO252, clinical grade TACC3 inhibitor or CRISPR-Cas9–mediated knockout of TACC3 disrupts these interactions and significantly impairs invasive behavior. In vivo, TACC3 inhibition markedly suppresses local invasion, circulating tumor cells, and spontaneous lung metastasis in both MMTV-PyMT transgenic mice and lung-tropic breast cancer xenografts, without inducing systemic toxicity. Together, our findings establish TACC3 as a druggable driver of metastasis in centrosome-amplified breast cancer and offer new mechanistic insights. This work highlights a novel therapeutic strategy for halting metastasis in a highly aggressive breast cancer subtype.



 Ozge Saatci, Kubra Calisir Unsal, Baris Cerci, Mustafa Emre. Gedik, Metin Cetin, Ozlem Sener Sahin, Rony Hull, Lauren Ball, Elizabeth Hill, Ioulia Chatzistamou, Chaemin Lim, Sridhar Vempati, Burcu Caliskan, Erden Banoglu, Ozgur Sahin. TACC3 is a driver of metastasis in centrosome-amplified breast cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a14c1abcdad2f3f277741c48118314eef415f5b6" target='_blank'>
              Abstract C120: TACC3 is a driver of metastasis in centrosome-amplified breast cancer
              </a>
            </td>
          <td>
            Ozge Saatci, Kubra Calisir Unsal, B. Çerçi, M. E. Gedik, Metin Cetin, Ozlem Sener Sahin, Rony Hull, Lauren Ball, Elizabeth Hill, I. Chatzistamou, Chaemin Lim, Sridhar Vempati, B. Çalışkan, E. Banoglu, Ozgur Sahin
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06e0ec6da10ec76bbd69b6353780f9b000cfc6a" target='_blank'>
              Overexpression of the TPX2 gene is associated with enhanced tumor malignancy in lung adenocarcinoma and identification of marine natural inhibitors of the Aurora kinase A-TPX2 protein complex
              </a>
            </td>
          <td>
            Md Imtiaz, Partha Biswas, Md. Mohaimenul Islam Tareq, Md Hasan Jafre Shovon, Sadia Jannat Tauhida, Md. Nazmul Hasan Zilani, Md Nazmul Hasan
          </td>
          <td>2025-10-17</td>
          <td>In Silico Pharmacology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b04efe93fb72cdef3529bf2ec79fd71db767d8a" target='_blank'>
              DNASE1L3 surveils mitochondrial DNA on the surface of distinct mammalian cells
              </a>
            </td>
          <td>
            Jennifer Porat, Valentina Poli, Karim Almahayni, Benson M. George, F. N. de Luna Vitorino, Li Yi, Carolina Bras Costa, Yitong Zhou, Andrew Crompton, Hongxiu Wen, Leon Zheng, Peter K. Gregersen, Suneet Agarwal, Benjamin A. Garcia, Yixuan Xie, Leonhard Möckl, Ivan Zanoni, K. Simpfendorfer, Ryan A. Flynn
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13.
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell death discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/167077c6c722e64c238ff049d56bccfcac548091" target='_blank'>
              Acytota and the evolution of complexity.
              </a>
            </td>
          <td>
            E. Kejnovský, Z. Kubat, J. Šponer
          </td>
          <td>2025-10-27</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epigenetic regulation, encompassing DNA methylation, histone modifications, and non-coding RNA activities, is a crucial mechanism through which gene expression is modified without corresponding changes in genomic DNA sequences. Dysregulation of these mechanisms can lead to histone and DNA modifications that either suppress or enhance the expression of disease progression-related genes. Among these regulatory processes, histone modifications are particularly significant, as they contribute to genomic stability, DNA repair, and chromatin dynamics, all of which influence breast cancer initiation and progression. This review offers a detailed analysis of the current state of research centered on epigenetic regulatory factors, with a particular focus on the role that histone modifications play in the treatment of breast cancer. It also examines the interplay between epigenetic modifications and the effectiveness of radiotherapy and chemotherapy when treating breast cancer. Lastly, this article explores the potential of epigenetic regulatory factors as viable targets for the future design of new anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379da6773863c74043489d2ccffa9dd0b5e23c2c" target='_blank'>
              Epigenetic regulation espeically histone modifications in breast cancer: A viable and emerging targeted therapeutic strategy
              </a>
            </td>
          <td>
            Yibing Zhou, Haotian Liu, Weimin Hong, Haotian Su, Yuxiao Mu, Yijie Cheng, Chaoshen Wu, Xuli Meng, Da Qian
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-15</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793bb57e9ee8be0a292f9fe822d8b2b3a9781a8c" target='_blank'>
              Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq
              </a>
            </td>
          <td>
            Wangjie Liu, Wouter Saelens, Pernille Y Rainer, Marjan Biočanin, V. Gardeux, Antoni J. Gralak, Guido van Mierlo, Angelika Gebhart, Julie Russeil, Tingdang Liu, Wanze Chen, Bart Deplancke
          </td>
          <td>2025-10-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44901dfbd5176301295029fab9bb4b40320a082" target='_blank'>
              AAV Kills Dividing Cells by Depleting PARP1 and Other DNA Damage Response Proteins
              </a>
            </td>
          <td>
            Sasha Friese, Junjie Zai, Grace Luzbetak, Nyonika Khanna, Johanna Gesperger, Chien-Hung Liu, Wynand P. Roos, Rasha Q. Al-Rahahleh, Matisse Willardson, Feng Yang, Duoduo Fu, Yiming Han, Nadia Lintag, Jaiden Saykham, Joshua Le, Emeen Al-Delaimy, Nolan Soutipan, Ellen Duong, Jeremy Rich, Maria Carolina Marchetto, Michael Rosenfeld, Robert W. Sobol, Matthew Shtrahman
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba22ede5cb07736eac6d115a03c89fcfe2e86a6e" target='_blank'>
              Evolutionary Variation of Poxvirus Genome Architecture
              </a>
            </td>
          <td>
            J. Enow, Arkaidy Garber, Betram Jacobs, G. McFadden, Masmudur M. Rahman
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Method Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Results Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). Conclusion This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c809269d9254558fb4308ec6c19a38c6246f527" target='_blank'>
              STING activation reshapes the tumor microenvironment leading to tumor regression in osteosarcoma
              </a>
            </td>
          <td>
            Elizabeth P Young, , Alex G Lee, Douglass Morris, Jessica Tsui, Marcus R. Breese, Juan Antonio, Camara Serrano, Courtney R Schott, Clare L. Abram, Stanley G. Leung, L. Sayles, Aafrin M. Pettiwala, Vanessa S Gutierrez Vera, Clifford A. Lowell, Alexis J. Combes, T. A. McEachron, A. Sweet-Cordero
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c93dca3eb5be61d1472406ead880481fa79563f" target='_blank'>
              Genetic diversity and evolution of rice centromeres.
              </a>
            </td>
          <td>
            Lingjuan Xie, Yujie Huang, Wei Huang, L. Shang, Yanqing Sun, Quanyu Chen, Shuangtian Bi, Mingyu Suo, Shiyu Zhang, Chentao Yang, Xiaoming Zheng, Weiwei Jin, Qian Qian, Longjiang Fan, Dongya Wu
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains a formidable challenge in clinical settings due to limited treatments available. The surface protein CD133 marks glioblastoma stem cells (GSCs), cells capable of overcoming therapeutic pressures and correlate with more aggressiveness tumor phenotypes. In this study, we employed a CRISPR-Cas9 functional screen to deconvolute CD133 dynamics in tumors. This led us to establish that SOX2 is a key player in controlling the PROM1 gene, which in turn influences how cells react to stress factors, including those induced by chemoradiation treatment. The discoveries in this study shed light on the complex web of mechanisms that control the survival and resistance of GSCs, offering promising new avenues for targeting and potentially overcoming therapy resistance. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20183-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f508110c54ab3c78e02eb486850e272c3a091d" target='_blank'>
              CRISPR screen reveals SOX2 as a critical regulator of CD133 and cellular stress response in glioblastoma
              </a>
            </td>
          <td>
            N. Savage, Etienne Danis, C. Chokshi, Stefan Custers, Muhammad Vaseem Shaikh, P. Miletic, C. Venugopal, Kevin R. Brown, Rajeev Vibhakar, Jason Moffat, Sheila K Singh
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cellular organisms store heritable information in two forms, genetic and epigenetic, the latter being largely dependent on cytosine methylation (5mC). Chargaff’s Second Parity Rule (CSPR) describes the nucleotide composition of cellular genomes in terms of intra-strand DNA symmetry. However, it remains unknown whether DNA methylation patterns display intra-strand DNA symmetry. Computational analysis was conducted of the DNA methylation patterns observed in human cell lines and in tissue samples from healthy donors. Analysis of 5mC marks in mutually reverse-complementary pairs of short oligomers, containing CpG dinucleotide in the middle, revealed deviations from CSPR and methylation asymmetry that can be observed for two non-overlapping mirror groups defined by CpG methylation values. Deviations from CSPR, together with combinatorial probabilities of pattern distributions and computer simulations, highlight the non-random nature of methylation processes and enabled us to identify specific cell types as outliers. Further analysis revealed a compensatory methylation asymmetry that reduces deviations from intra-strand symmetry and implies the existence of strand-specific methylation during cell differentiation. Among six pairs of reverse-complementary tetranucleotides, four pairs with specific sequence motifs display pronounced methylation asymmetry. This mirror asymmetry may be associated with chromosome folding and the formation of a complex three-dimensional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387c52e696253beba52f792a823e42cb5be6a863" target='_blank'>
              Study of the Patterns of DNA Methylation in Human Cells Through the Prism of Intra-Strand DNA Symmetry
              </a>
            </td>
          <td>
            Z. Ramazanova, Aizhan Alikul, D.A. Begimbetova, S. Taipakova, Bakhyt Matkarimov, Murat K. Saparbaev
          </td>
          <td>2025-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nuclear biomolecular condensates are essential sub-compartments within the cell nucleus and play key roles in transcription and RNA processing. Bottom-up construction of nuclear architectures in synthetic settings is non-trivial but vital for understanding the mechanisms of condensates in real cellular systems. Here, we present a facile and versatile synthetic DNA protonucleus (PN) platform that facilitates localized transcription of branched RNA motifs with kissing loops (KLs) for subsequent condensation into complex condensate architectures. We identify salinity, monomer feeding, and KL-PN interactions as key parameters to control co-transcriptional condensation of these KLs into diverse artificial nuclear patterns, including single and multiple condensates, interface condensates, and biphasic condensates. Over time, KL transcripts co-condense with the PN matrix, with the final architecture determined by their interactions, which can be precisely modulated using a short DNA invader strand that outcompetes these interactions. Our findings deepen the understanding of RNA condensation in nuclear environments and provide strategies for designing functional nucleus-mimetic systems with precise architectural control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13da8ed75a9cef66af63d63bcdff443dc8cb37d3" target='_blank'>
              Constructing synthetic nuclear architectures via transcriptional condensates in a DNA protonucleus
              </a>
            </td>
          <td>
            Miao Xie, Weixiang Chen, Maria Vonk-de Roy, Andreas Walther
          </td>
          <td>2025-09-10</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b865648502532b64b511597f51b1acb1d7cb69bb" target='_blank'>
              Disrupting the epigenetic alliance: structural insights and therapeutic strategies targeting DNMT1-UHRF1.
              </a>
            </td>
          <td>
            Emadeldin M. Kamel, Mohamed A M Ali, A. A. Allam, Noha A. Ahmed, Adil Abalkhail, Faris F. Aba Alkhayl, Al Mokhtar Lamsabhi
          </td>
          <td>2025-09-17</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Estrogen receptor alpha (ERα)-positive (ER+) breast cancers are driven by 17β-estradiol (E2) binding to ERα, which transcriptionally regulates downstream target genes. Although microarrays and conventional RNA sequencing have identified E2 target genes, pre-designed probes and short read lengths are limited in accurately capturing complex transcript structures. Long-read RNA sequencing offers a solution by spanning entire transcripts, providing a more complete view of the transcriptome. Here, we employed nanopore long-read direct RNA sequencing (DRS) complemented with 3’-end sequencing, in vitro experiments, and deep learning-based protein modeling to explore the intricate landscape of E2-responsive transcriptome and protein level implications. Our analysis revealed a range of E2-responsive non-coding and coding isoforms, including intronically polyadenylated (IPA) mRNAs. One of these IPA isoforms was detected for TLE1 (Transducin-like enhancer protein 1), which positively assists ERα-chromatin interactions for a subset of E2 target genes. The IPA isoform produces a C-terminus truncated protein, lacking the WDR interaction domain, but retains dimerization/tetramerization capacity through its intact N-terminus Q-domain. Structural modeling and protein-based assays confirmed the truncated protein’s dimerization potential and nuclear localization. Functional assays showed that overexpression of truncated TLE1 reduces the E2-induced upregulation of GREB1, an E2-responsive gene, thereby disrupting transcriptional regulation. Importantly, a lower IPA isoform ratio is associated with worse survival in ER+ patients, highlighting clinical relevance. Our study uncovers new layers of complexity in the E2-regulated transcriptome, providing insights into truncated proteins. These findings contribute to a deeper understanding of gene regulation and may help the development of new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0bac48baf3ce0314e9eaa5b424b9e568bd79fa" target='_blank'>
              E2-regulated transcriptome complexity revealed by long-read direct RNA sequencing: from isoform discovery to truncated proteins
              </a>
            </td>
          <td>
            Didem Naz Dioken, Ibrahim Ozgul, Irem Yilmazbilek, Elanur Almeric, I. C. Eroglu, Mustafa Cicek, Utku Cem Yilmaz, Deniz Karagozoglu, Esra Çiçek, E. Karaca, Tolga Can, A. E. Erson-Bensan
          </td>
          <td>2025-08-02</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The synergy between SWI/SNF and Polycomb Repressive Complexes (PRCs) has been a topic of extensive research. The demonstrated heightened preclinical sensitivity of SWI/SNF gene mutations to EZH2 inhibition together with the accelerated approval of Tazemetostat for selected EZH2 mutant tumors by the FDA, has significantly expanded our understanding of chromatin remodeling complexes and their vulnerabilities. Here we report a durable response in a patient with PBRM1 mutated cholangiocarcinoma treated with Tazemetostat. We provide hypothesis-generating mechanistic findings into the relationship between PBRM1 mutations and EZH2 inhibition and the interplay between the SWI/SNF and PRC2 complexes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab2893d50862157c104f2f52b2eb5fb27cb5f5db" target='_blank'>
              Prolonged survival in a patient with pbrm1 mutated metastatic cholangiocarcinoma receiving tazemetostat
              </a>
            </td>
          <td>
            Manuel Pedregal, E. C. Morafraile, Alberto Garrido, B. Doger, T. Hernández, Daniel Morillo, Balázs Győrffy, Alberto Ocaña, Pablo Minguez, Víctor Moreno
          </td>
          <td>2025-10-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. Results In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Conclusion Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements. Supplementary Information The online version contains supplementary material available at 10.1186/s40348-025-00208-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear.We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells.High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells.Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5ee3b9a577273caee9d178991970953321a673" target='_blank'>
              Integrated Genomic Profiling Reveals Mechanisms of Broad Drug Resistance and Opportunities for Phenotypic Reprogramming in Cancer Cells
              </a>
            </td>
          <td>
            Ian Mersich, Brian S. J. Blagg, Aktar Ali
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff9dd6e8b4e7b70e0de9451cb1c5dd37501bb1c" target='_blank'>
              A comprehensive genomic framework for identifying genes predisposing to homologous recombination repair deficient breast cancer
              </a>
            </td>
          <td>
            José Camacho-Valenzuela, Thibaut Matis, Carla Roca, Jorge Luis, Cuamatzi Flores, N. Hamel, Barbara Rivera, Simon Gravel, P. Polak, Carla Daniela, Robles-Espinoza, William D. Foulkes
          </td>
          <td>2025-10-23</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract UVB radiation (280–320 nm) from sunlight induces skin cancer through DNA damage-induced mutations. Melanomas carry mutational signatures associated with UVB-induced cyclobutane pyrimidine dimers (CPDs). However, there are several other melanoma signatures of unknown origin. To test if these signatures are linked to UVA, we exposed human melanocytes to UVA (340–400 nm) and to UVB for comparison. We mapped DNA damage in the form of CPDs or 8-oxoguanines (8-oxoG) genome-wide at base resolution. We then determined mutational patterns in single melanocyte cell clones by whole genome sequencing. Different from UVB, UVA induces CPDs more selectively at TT sequences resembling melanoma signature SBS7d. We did not observe rising CPD levels after cessation of radiation (dark CPDs). The UVA-induced CPDs were not mutagenic in the mutation analysis. 8-oxoG was present in melanocytes but did not substantially increase after UVA. G/C to T/A mutations were prominent in melanocyte single-cell clones with no major shift after UVA radiation. These mutations matched SBS18, a signature present in melanomas. Although UVA damages DNA, it has a surprisingly limited mutagenic effect on human melanocytes. However, the oxidative base lesions in melanocytes and their associated mutations may be linked to a subset of melanoma mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b21e19c0b878b06400096dc62c3652b28e09c65" target='_blank'>
              UVA-induced DNA damage and mutations in human melanocytes: relevance for melanoma mutations
              </a>
            </td>
          <td>
            , Jennifer Johnson, Paul Y Kim, Gerd P Pfeifer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The genomic regions surrounding tumor suppressor genes are often lost as collateral damage during tumorigenesis, providing novel therapeutic vulnerabilities. To systematically identify synthetic lethal pairs of genes within DNA damage repair (DDR) pathways, we conducted a combinatorial genome-wide CRISPR screen. We found that RNASEH2B, a gene frequently co-deleted with RB1 in prostate cancers, emerged as a top vulnerability with knockout of APEX2, which is a nuclease involved in DNA repair. This interaction was validated in RNASEH2B isogenic cell line pairs using whole-genome CRISPR screens and further confirmed across multiple cancer lineages using a focused mini-pool sgRNA library. Single gene knockouts demonstrated that loss of APEX2, specifically its catalytic activity, is sufficient to eliminate RNASEH2B mutant cells both in vitro and in vivo. Importantly, loss of RB1 did not impact the synthetic lethal interaction between RNASEH2B and APEX2. Consistent with prior reports, APEX2 knockout also impairs growth of BRCA1 mutant cells and enhances sensitivity to PARP inhibitors. Overall, these findings position APEX2 as a compelling synthetic lethal target in tumors with RNASEH2B deletion or BRCA1 mutations and support the development of APEX2 inhibitors as a potential monotherapy to improve patient outcome.



 Antoine Simoneau, Steve Lombardo, Ruthie Swain, Serge Gueroussov, Binzhang Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H. Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, Douglas A. Whittington, Yi Yu, Jannik N. Andersen, Teng Teng. APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f840d30e01fa153da53003edd382b7d06deea42" target='_blank'>
              Abstract B039: APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors
              </a>
            </td>
          <td>
            Antoine Simoneau, Steven A. Lombardo, Ruthie Swain, Serge Gueroussov, B. Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, D. Whittington, Yi Yu, Jannik N Andersen, T. Teng
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 African Americans (AAs) have higher incidence rates of advanced lung cancer when compared to European Americans (EAs). DNA double-strand breaks (DNA DSBs) are the major cause of genomic instability that initiates cancer. Our previous work found AA lung cancer patients have more germline DNA DSB repair deficiencies than EAs, suggesting unrelated AAs may share genomic risk factors for developing lung cancer. We hypothesize African ancestry drives disease-causing alterations in DNA DSB repair pathway genes.



 ClinVar criteria were used to profile germline pathogenic variants (PVs) across the four major DNA DSB repair pathways: homologous recombination repair (HRR), classical non-homologous end joining (c-NHEJ), alternative end-joining (alt-EJ), and single-strand annealing (SSA). The discovery cohort consisted of populations of African (n=37,545) and European (n=622,057) descent in the Genome Aggregation Database. The validation cohort included study participants in the NIH All of Us Research Program (African, n=50,080) and (European, n=125,860). In silico RNA (SpliceA1, Pangolin, and PhyloP) and protein (VEP and CADD) annotation tools were used to predict functional consequences.



 In the African ancestry populations, 20 PVs across eight HRR genes, nine PVs across six c-NHEJ genes, six PVs across three alt-EJ genes, and 14 PVs across five SSA genes we discovered. In comparison, 115 PVs across 11 HRR genes, 52 PVs across eight c-NHEJ genes, 43 PVs across six alt-EJ genes, and 113 PVs across seven SSA genes were observed in the European ancestry populations. Interestingly, a RAD51 PV was exclusively enriched in African ancestry and not European ancestry populations for the HRR pathway. By contrast, NHEJ1, EXO1 and DCLRE1C PVs were uniquely present in those with European ancestry and not in individuals with African ancestry for the alt-EJ and SSA pathways, respectively. Frameshift mutations were the most common functional protein change in both populations.



 A total of 32 germline PVs across 13 DNA DSB repair genes were observed in African ancestry populations only, suggesting genetic ancestry may drive DNA repair deficiencies that increase susceptibility to developing lung cancer.



 Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell. Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C077.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11a81bc23bab29ba6a3aa864a49c700d5fec40e" target='_blank'>
              Abstract C077: Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry
              </a>
            </td>
          <td>
            Musinu Zakari, E. Aigboeghian, Khadijah A. Mitchell
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT: Gene editing exploits endogenous DNA repair pathways to introduce precise modifications into the genome. The CRISPR system was first identified in Escherichia coli (1987) and, based on Cas protein architecture, is divided into Class I (types I, III, IV) and Class II (types II, V, VI). The widely adopted CRISPR/Cas9 system comprises a guide RNA (gRNA) and the Cas9 endonuclease, which orchestrate genome editing through a tripartite mechanism: target sequence recognition via gRNA binding to a complementary DNA site adjacent to a protospacer adjacent motif (PAM), double-stranded DNA cleavage by two nuclease domains (RuvC cleaving the non‑target strand and HNH the target strand), and repair of the resulting break by cellular pathways. This approach has been applied to in vitro fertilization-derived embryos and meiotically developing oocytes to disrupt or correct genes, offering potential for eliminating heritable diseases. However, concerns remain regarding off-target effects that introduces unwanted genetic mutations, necessitating improved specificity and ethical scrutiny.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e32b11816cdfe9dbe37c9b7f4fa9f2f1d5d3ed5" target='_blank'>
              CRISPR/CAS9-Mediated Gene Editing in Human Gametes: A Review
              </a>
            </td>
          <td>
            Esha Kumari, Neha Banu, Kathrina Marbaniang, Faridha Jane R.M. Momin, B. C. Hynniewta
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03559-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. U. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="For a positive-strand RNA virus, the encoded viral RNA-dependent RNA polymerase (oRdRP) synthesizes complementary antigenome strand and uses it as a template for amplifying the viral genome, generating various replication intermediates. Structural proteins and viral genome are packaged into virions, but the fate of replication intermediates is underexplored. Here, we investigate Orsay Virus (OV) replication intermediates, including antigenome, oRdRP and double stranded RNA (dsRNA), using PCR and fluorescence-based imaging in C. elegans intestines. As for other positive-strand RNA viruses, we find that genome is in vast excess of antigenome. Antigenome is only visualized in cells when using denaturation protocols, indicating basepaired intermediates. OV antigenome is observed with distinct cytoplasmic and perinuclear localization patterns that depend on factors required for generation of primary, but not secondary, siRNAs. In both wildtype and RNA interference (RNAi) mutants, viral dsRNA is observed in the cytoplasm associated with oRdRP, suggesting cytoplasmic virus replication hubs. Additionally, using antibodies to oRdRP, we observed spherical structures of ∼1μm in diameter defined by oRdRP at their surface; over 75% of infected wildtype animals show these structures, which associate with mitochondria and autophagosomes in an antiviral RNAi- and autophagy-dependent manner, respectively. Our study defines new features of OV replication intermediates in wildtype animals, setting the stage for understanding their connection to the viral life cycle and host antiviral pathways. Author Summary Viruses infect and cause disease in all organisms. Once they enter an organism, or host, viruses make copies of themselves in a process called replication, and this process generates byproducts that are recognized as foreign by the host. This signals the host to begin protecting itself using pathways such as the innate immune response, underscoring the importance of understanding where and when replication byproducts are generated. Here we utilize Caenorhabditis elegans, a microscopic roundworm, to detect the viral replication intermediates of its naturally infecting pathogen, Orsay Virus (OV). For the first time, we detect OV antigenome, a byproduct that is used as a template for viral genome replication. We report a previously uncharacterized antigenome-containing perinuclear granule that is partially dependent on the antiviral RNA interference (RNAi) pathway, an invertebrate immune response. We also identify regions of cytoplasmic viral replication and unusually large (∼1 micron) spherical structures lined with the viral RNA-dependent RNA polymerase, an enzyme made by the virus for its replication. By characterizing important viral replication byproducts, our studies set the stage for understanding mechanistic connections between viral infection and the host response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191f2c795c34a173bf3ec9d8e533cdddf262fc53" target='_blank'>
              Detection of Orsay viral replication intermediates reveals spatial and regulatory links to Caenorhabditis elegans innate immune responses
              </a>
            </td>
          <td>
            Supraja Ranganathan, P. Aruscavage, Brenda L. Bass
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell epigenetic regulation relies on spatially organized enzymatic activities within nuclear compartments, which establish DNA methylation patterns that dictate gene expression and cellular identity. Given this spatial organization, we present a modular DNA-assembled nanopipette (mDNP) platform for ultrasensitive detection of DNA methyltransferase 1 (DNMT1) within the nucleus of individual cells. The device incorporates tripartite DNA molecular sentinel probes that undergo methylation-dependent sequence discrimination and initiate rolling circle amplification, producing quantifiable ionic current signals within a confined nanofluidic interface. This system offers high analytical precision and a broad linear range, enabling quantitative evaluation of DNMT1 inhibition and differentiation between malignant and nonmalignant cell populations. The regenerable mDNP device provides an integrated platform for single-cell epigenetic profiling and inhibitor screening at subcellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67457ecde67c0610e59fff5e8a9ab63e43837fb" target='_blank'>
              A Modular DNA-Assembled Nanopipette (mDNP) Platform for Spatially Confined Single-Cell Nuclear Detection of Methyltransferase Activity.
              </a>
            </td>
          <td>
            Juan Song, Chao-Wei Xia, Xian Zhang, Jing Xu
          </td>
          <td>2025-09-17</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma stem cells (GSCs), a stem-like tumorigenic subpopulation within glioblastoma (GBM), exhibit remarkable self-renewal capacity and therapeutic resistance. Zinc finger BED domain-containing protein 1 (ZBED1), a dual-function transcription factor and SUMO E3 ligase, has been implicated in oncogenic processes across malignancies, its functional role and regulatory mechanisms in GSCs remain enigmatic.


METHODS
Multimodal approaches including immunohistochemistry, immunoblotting, and immunofluorescence were employed to evaluate ZBED1 expression patterns in GSCs and clinical GBM specimens. Functional characterization utilized in vitro models (proliferation assays, tumor-sphere formation assays, limiting dilution analysis) complemented by in vivo orthotopic xenograft models. Mechanistic investigations integrated RNA sequencing, label-free proteomics, chromatin immunoprecipitation (ChIP), immunohistochemistry, and western blotting to delineate the EGFR/ZBED1 regulatory axis.


RESULTS
We demonstrated that ZBED1 was significantly upregulated in GSCs and linked to unfavorable prognosis. Genetic ablation of ZBED1 significantly impaired GSC proliferation and self-renewal capacity, while extending survival in xenograft models. Mechanistically, EGFR-mediated ZBED1 phosphorylation at tyrosine residues Y160/Y513 enhanced ZBED1-UBC9 interaction, promoting SUMOylation-dependent protein stabilization. Remarkably, ZBED1 reciprocally sustained EGFR expression through transcriptional repression of the E3 ubiquitin ligase PARK2, establishing a self-reinforcing EGFR/ZBED1/PARK2 signaling circuit critical for GSC maintenance.


CONCLUSIONS
Our findings elucidate a novel EGFR/ZBED1 positive feedback loop that drives GSC propagation and tumorigenesis, highlighting ZBED1 as an attractive candidate for therapeutic targeting in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5de1c18b91342ef1a1f3fd2247cd9768fb0fd3" target='_blank'>
              EGFR/ZBED1 reciprocal regulation promotes stemness and tumorigenesis in glioblastoma.
              </a>
            </td>
          <td>
            Nana Hou, Yutao Wang, Qiuxiang You, Lei Zhang, Wenjia Zhang, Xinyi Wang, Chunyan Deng, Jiachuan Yan, Saiyu Cheng, Jianwen Ji, Jianbing Hou, Hongjuan Cui, Yundong Zhang
          </td>
          <td>2025-09-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomavirus (HPV) causes 4.5% of all human cancers, although a complete cellular basis of infection remains unclear. The mechanisms of virus disassembly and transport are particularly enigmatic. Here, we use in situ cryo-electron tomography (cryo-ET) to provide direct visualization of high-risk HPV16 during its infection cycle. We demonstrate that, when HPV reaches the lysosome, it remains intact and infection competent, contrary to the prevailing view of viral inactivation within lysosomes. We challenge the current model of HPV trafficking, which predicts progressive disassembly of the L1 capsid during retrograde transport, by showing that HPV remains intact when it reaches the Golgi en route to the nucleus for infection. Finally, we provide snapshots of the HPV containing transport vesicles budding from the Golgi lumen in a COPI coat-dependent fashion and show that HPV is connected to the overlying membrane by proteinaceous structural bridges, possibly formed by the L2 capsid, to guide virus trafficking. Our results redefine the dominant models of HPV entry, revealing the virus capsid’s resilience to disassembly during retrograde trafficking, a coat-dependent budding mechanism for transport, and demonstrate that the lysosome serves as a reservoir for infectious HPV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4b59059795c579b75b489033dd5f90be08bd69" target='_blank'>
              In situ cryo-ET redefines HPV disassembly and transport paradigms
              </a>
            </td>
          <td>
            Renaldo Sutanto, Jaimin K. Rana, Billy Tsai, Shyamal Mosalaganti
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, , S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres.
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ad0cbeca9935096e33fe6872d833adddc5508b" target='_blank'>
              Arabidopsis YEATS domain proteins facilitate DNA double strand break repair via homologous recombination pathways
              </a>
            </td>
          <td>
            Neeraja Vegesna, Clara Bourbousse, Y. Jami-Alahmadi, Ana Marie S. Palanca, James A. Wohlschlegel, Julie A. Law
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This review examines the potential of disrupting telomere maintenance in Plasmodium as a novel antimalarial strategy. Telomeres are repetitive DNA–protein structures located at chromosome termini, where they preserve genome stability and protect against degradation. Telomere maintenance is crucial for rapid growth, genome integrity, and immune evasion in Plasmodium parasites. Unlike humans, Plasmodium maintains continuous telomerase activity and uses unique telomere-binding proteins across its lifecycle. These features drive parasite virulence and antigenic variation. Emerging evidence suggests that Plasmodium telomeres harbor G-quadruplex (G4) DNA structures, which help stabilize telomeres during replication and may be good targets for small molecules to disrupt their function. Additionally, the parasite depends heavily on its telomerase catalytic subunit, PfTERT, for survival. Inhibiting PfTERT has shown promising results in blocking telomere elongation and impairing replication. Targeting this parasite-specific telomere–telomerase axis may offer a strategic means to destabilize chromosomes, weaken immune evasion, and limit parasite survival, making it a promising antimalarial approach. However, researchers must consider the risks of off-target effects in future drug designs. Though current studies are limited and remain inconclusive, we suggest that future research should investigate combining telomere-directed therapies with existing antimalarials to help overcome resistance and improve treatment outcomes. Herein, we review advances in understanding Plasmodium telomere biology, highlighting its distinct structures, critical telomere-associated proteins, and roles in pathogenesis. We further explore how selective targeting could exploit an Achilles’ heel in parasite survival, offering fresh possibilities for next-generation, parasite-specific malaria therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bb521b6b69871da352e8c028549958d3812bd32" target='_blank'>
              Plasmodium telomere maintenance: uncovering the Achilles’ heel for novel antimalarials
              </a>
            </td>
          <td>
            Theophilus N. Wakai, D. Anzaku, Israel S. Afolabi
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Influenza A virus (IAV) relies on the host cellular machinery to support its replication. Understanding these host dependencies can inform the development of novel antiviral strategies. In this study, we identified conserved oligomeric Golgi complex subunit 6 (COG6) as a novel host factor critical for IAV replication through a genome-wide clustered regularly interspaced short palindromic repeats/CRISPR-associated protein 9 (CRISPR/Cas9) knockout screen. Disruption of COG6 significantly impaired viral replication. Mechanistically, COG6 supports IAV replication via two distinct means. First, consistent with the role of the COG complex in Golgi homeostasis, COG6 is required for the proper presentation of surface sialic acids, the primary receptor for IAV entry. Second, COG6 deficiency unexpectedly led to lysosome-dependent degradation of viral proteins. Notably, lysosomal activity was also upregulated in IAV-infected wild-type cells, albeit to a lesser extent than in COG6-deficient cells. Treatment with lysosomal inhibitors rescued viral protein stability in COG6 knockout cells. Protein interaction analysis further demonstrated that COG6-mediated stabilization of viral proteins did not rely on viral protein-COG6 interaction, refuting the hypothesis that COG6 acts as a shield factor to protect viral protein from lysosomal degradation. Moreover, knockout of other COG subunits produced similar antiviral effects, suggesting that an intact COG complex is required for IAV replication. Together, these findings uncover a critical role of the COG complex in regulating IAV replication and highlight a previously unappreciated functional link between the Golgi and lysosomes that could be exploited for treating IAV infections. IMPORTANCE Despite advances in virology, numerous host determinants facilitating influenza A virus (IAV) pathogenesis remain uncharacterized. Our study establishes conserved oligomeric Golgi complex subunit 6 (COG6) as a critical host factor promoting IAV infection through complementary mechanisms: receptor modulation and viral protein stabilization. This represents the first demonstration that the COG complex regulates viral pathogenesis through proteostasis mechanisms, fundamentally expanding our understanding of host-virus interactions at the organelle interface. These findings not only provide new perspectives on viral exploitation of Golgi trafficking networks but also identify potential therapeutic targets against evolving influenza strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dc6445af89a95f3ea7403011a50233d6985ecd" target='_blank'>
              COG6 is an essential host factor for influenza A virus infection
              </a>
            </td>
          <td>
            Daobin Feng, Jiaohan Guo, Jingjing Yan, Jian Chen, Longfei Ding, Xinyu Zhu, Zhilu Chen, Yangyang Hu, Miaomiao Zhang, Jian Liu, Cuisong Zhu, Mingbin Liu, Chen Zhao, Xiaoyan Zhang, Jianqing Xu
          </td>
          <td>2025-09-05</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The fact that breast cancer exhibits notable individual and racial variations in incidence, prognosis, and response to treatment. This highlights the need for a more thorough examination of the molecular mechanisms underlying these differences. KAISO (ZBTB33), long recognized as a methyl-CpG binding protein and transcriptional repressor, is increasingly understood to have more complex regulatory roles. KAISO has two DNA-binding sites: a methylated CpG motif and a consensus sequence unique to KAISO. Previous ATAC-seq study showed that KAISO can localize to the areas of open chromatin in different cancer cells, indicating possible roles beyond repression. Based on this data, we hypothesized that KAISO might interact with important regulatory proteins and form unique chromatin complexes to contribute to both transcriptional repression and activation. To investigate this, we analyzed MCF7 (luminal, Non-Metastatic, European American origin) and HCC1806 (Basal, Metastatic, African American origin) breast cancer cell lines, which differ in molecular subtype and racial background. We used Cut&Run to construct genome-wide KAISO binding profiles and Rapid Immunoprecipitation Mass Spectrometry of Endogenous Protein (RIME) to map KAISO-interacting protein networks. Our RIME data showed clear distinctions between the two lines: in HCC1806, KAISO strongly associated with molecular chaperones (HSPA8, HSP90AB1), elongation factor (EEF1A1), and immune-related proteins (e.g., C4B, APOC3), while MCF7 interactions favored structural and co-regulatory proteins (e.g., MYH10, CLTC, RBMXL1). Different chromatin engagement patterns were revealed by Cut&Run profiling. With promoter-associated peaks at about 90 locations, MCF7 showed an abundance of intronic (∼160 peaks) and intergenic binding (∼110 peaks), encompassing genes such as ATP9B, LINC01667, and TSPAN9. HCC1806, on the other hand, showed greater intronic enrichment (∼140 peaks) and fewer promoter peaks (∼50). All of these results point to KAISO's potential as a molecular marker for precision oncology by indicating that it is involved in transcriptional regulatory complexes that are unique by subtypes (Basal/Luminal, Metastatic/Non-Metastatic) in breast cancer. In order to further understand the regulatory function of KAISO, ongoing research attempts to expand these investigations to more related cell lines.



 Sazaul Morshed Sazib, Huixian Lin, Aliyu Muhammad, Md. Abu Siddique, Isra Elhussin, Moray Campbell, Shahjalal Chowdhury, Samuel Mwamburi, Malik McRae, Balasubramanyam Karanam, Clayton Yates. Kaiso defines distinct chromatin and protein interaction networks in breast cancer: Implications for precision therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C083.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4449400024ebdc9467232adda56233e690bba18f" target='_blank'>
              Abstract C083: Kaiso defines distinct chromatin and protein interaction networks in breast cancer: Implications for precision therapy
              </a>
            </td>
          <td>
            Sazaul M. Sazib, Huixian Lin, Aliyu Muhammad, Md. Abu Siddique, Isra Elhussin, Moray J. Campbell, Shahjalal Chowdhury, Samuel Mwamburi, Malik McRae, Balasubramanyam Karanam, Clayton C. Yates
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="The MRE11-RAD50-NBS1 (MRN) complex is a central, multifunctional factor in the detection, signaling and nucleolytic processing of DNA double-strand breaks (DSBs). To clarify how human MRN binds generic and telomeric DNA ends and can separate DNA end sensing from nuclease activities, we determined cryo-electron microscopy (cryo-EM) structures of human MRN bound to DNA and to DNA and the telomere protection factor TRF2. MRN senses DSBs through a tight clamp-like sensing state with closed coiled-coil domains, but auto-inhibited MRE11 nuclease. NBS1 wraps around the MRE11 dimer, with NBS1’s ATM recruitment motif sequestered by binding to the regulatory RAD50 S site, necessitating a switch in the NBS1 C helix for ATM activation. At telomeric DNA, TRF2 blocks the second S site via the iDDR motif to prevent nuclease and ATM activation. Our results provide a structural framework for DNA sensing via a gating mechanism and separation of sensing, signaling and processing activities of mammalian MRN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7645e8c109e5a6f384b9d506f7906e33acb9e2b8" target='_blank'>
              Structural basis for DNA break sensing by human MRE11-RAD50-NBS1 and its regulation by telomeric factor TRF2
              </a>
            </td>
          <td>
            Yilan Fan, Filiz Kuybu, Hengjun Cui, K. Lammens, Jia-Xuan Chen, Michael Kugler, Christophe Jung, K. Hopfner
          </td>
          <td>2025-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d18456bf0573eaf86f58e770fae2b90f97839de" target='_blank'>
              USP44, ZNF454, and GPRC5B ctDNA Methylation Markers in Breast Cancer: Limited Clinical Relevance for Disease Monitoring and Tumor Characteristics.
              </a>
            </td>
          <td>
            Y. Jeong, Chang-Ho Jeon, Hoon Kyu Oh, Jeong Kyu Kim, Chunseok Yang, Na-Rang Lee, Doyeon Kim
          </td>
          <td>2025-09-08</td>
          <td>Asia-Pacific journal of clinical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Detection of cancer at early stage can significantly improve the five-year survival rate of patients. Bisulfite-based methylation detection can cause DNA damage, especially in high GC content regions which was associated with the development of cancers. Loss of aberrant methylated CpG sites in cfDNA will lead to the undetectability of certain circulating tumor DNA (ctDNA), consequently may affect the cancer detection. Our study uses enzymatic method to detect whole genome abnormal methylation regions in esophageal squamous cell carcinoma (ESCC). We also provide a pretrained neural network, hybrid of BERT and CNN (BCNN), to identify ctDNA robustly. Maximum posterior probability is utilized to estimate the fraction of ESCC-derived ctDNA in plasma for predicting the risk of ESCC cancer. Our results analysis indicated that enzyme-based whole-genome methylation sequencing retained more longer cfDNA and detected more CpG sites than bisulfite-based method in both gDNA and cfDNA. Enrichment analysis of differentially methylated regions (DMR) showed that top five pathways were associated with ESCC. Compared to traditional models, our BCNN demonstrates the best performance in identifying ctDNA (AUC = 0.970). By estimating the fraction of plasma ctDNA, our BCNN exhibits high accuracy in ESCC detection even at ultralow sequencing depths (AUC = 0.946). Specifically, in the validation cohort, when the specificity is 93.75%, 7 out of 8 early-stage ESCC (TNM Stage I) were identified as positive in our preliminary results. In conclusion, our preliminary results reinforce the idea of employing BCNN as a novel strategy for ESCC early detection in clinical practice. However, to be applied in clinical, further validation with a larger sample size is necessary. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18278-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5af110454f8624475f32847f585947aa544c81e" target='_blank'>
              Improved circulating tumor DNA identification for detection of esophageal squamous cell carcinoma by enzymatic methyl sequencing and hybrid neural network
              </a>
            </td>
          <td>
            Jiangfeng Shen, Yuqi Ren, Ziyong Mao, Qiuxiang Hu, Yilong Wan, Jiangcun Du, Yanting Lai, Shengxiang Shao, Liuqing Zhang, Hao Wu, Jiaxi Li, Sheng Ju, Xin Tong, Jun Zhao, Lei Cao, Deyi Xiong, ChengCheng Xu, Jun-Chi Xu, Dong Jiang
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="EZH2, a core component of PRC2 complex, silences global gene expression by tri-methylating histone H3K27. It remains an elusive question that EZH2 hyperexpression discords with its H3K27me3 activity of gene suppression in advanced prostate cancer. Here, we report a nascent RNA-dependent PHF19-YTHDC1 condensate capable of switching EZH2-mediated gene suppression to activation during prostate cancer progression. We found that the long isoform of PRC2 accessory subunit PHF19, PHF19L, was highly expressed in advanced prostate cancer that promoted the tumor progression and hormonal therapy resistance. Mechanistically, PHF19L was recruited to the m6A modified nascent RNA through YTHDC1 and formed a liquid-like YTHDC1-PHF19L condensate that pulled the EZH2 away from chromatin, resulting in reduced H3K27me3 deposition and the activated expression of EZH2-repressed genes. Therefore, our study reveals a biomolecular condensate that modulates the switch from EZH2-mediated epigenetic gene silence to activation during the progression of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b355598211255f729cad3da89ca026d172d11aa5" target='_blank'>
              A PHF19-YTHDC1 condensate switches EZH2-mediated gene suppression to activation for prostate cancer progression.
              </a>
            </td>
          <td>
            Shuai Yuan, Dao-Jing Ming, Jiapeng He, Meng-Yang Liu, Shaoe He, Hong Weng, Shuqiang Xi, Jin-Hui Zhang, , , , Haozhe Zhang, Dan-qi Wang, Fei Li, , Yi Cai, Shi-Di Tang, Shuang Wang, , Xian-Tao Zeng, Hailiang Hu
          </td>
          <td>2025-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Human adenoviruses (HAdVs) induce significant reorganization of the nuclear environment, leading to the formation of virus-induced subnuclear structures known as replication compartments (RCs). Within these RCs, viral genome replication, gene expression, and modulation of cellular antiviral responses are tightly coordinated, making them valuable models for studying virus-host interactions. In a recent study, we analyzed the protein composition of HAdV type 5 (HAdV-C5) RCs isolated from infected primary cells at different time points during infection using quantitative proteomics. We identified several chromatin modifiers, including the high-mobility group box 1 protein (HMGB1) as components associated with RCs and demonstrated that HMGB1 can be relocalized to RCs from different HAdV species, thereby modulating viral replication in a species-specific manner. In the present work, using click-chemistry and proximity ligation assays, we discovered that HMGB1 localizes to sites of DNA replication within RCs and that its interaction with DBP in RCs is dependent on both DNA replication and RC assembly. HMGB1-knockdown experiments demonstrated that HMGB1 is required for efficient viral gene expression. However, despite its proviral role in viral replication, we found that HMGB1 levels decreased in late stages of infection due to transcriptional downregulation. Furthermore, by overexpressing HMGB1, we showed that this regulation of HMGB1 levels during infection is critical for optimal HAdV-C5 replication. These results highlight the complex regulatory relationship between HMGB1 and HAdV-C5 infection. IMPORTANCE In an extensive proteomics analysis, we found that HMGB1, an important cellular chromatin protein, was enriched in adenovirus replication compartments. In this study, we aimed to better understand the role of HMGB1 in the infection process of a human DNA virus, HAdV-C5. We tested different virus types, including some with specific gene deletions and mutations. Our results showed that during infection, HMGB1 levels decreased because the virus suppressed its production. Despite this, even at lower levels, HMGB1 still helped the virus replicate by interacting with key viral proteins and DNA at sites where the virus is actively replicating. Overall, our findings highlight how HMGB1 plays a crucial role in facilitating efficient virus replication, making it an important factor in the infection process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49fba7303cb5baed6552718dc0a39540a2f69987" target='_blank'>
              Proviral functions of HMGB1 in HAdV-C5 replication compartments
              </a>
            </td>
          <td>
            Paloma Hidalgo, Britta Gornott, L. D. Bertzbach, Thomas Dobner
          </td>
          <td>2025-09-08</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c629e7baaceae40832311e536a58f80a6d8ec6a8" target='_blank'>
              Transcriptomic Mutational Profiling of Gastric Adenocarcinoma in Northern Brazil
              </a>
            </td>
          <td>
            Rubem Ferreira da Silva, Diego Pereira, Mateus Silva Tavares, Daniel de Souza Avelar, Jéssica Manoelli Costa da Silva, R. Mourão, Valéria Cristiane Santos da Silva, Juliana Barreto Albuquerque Pinto, Samir Mansour Moraes Casseb, S. Demachki, Tatiane Neotti, Ana Karyssa Mendes Anaissi, W. Barra, A. Vidal, G. Ishak, P. P. de Assumpção, R. Burbano, Fabiano Cordeiro Moreira
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aging constitutes a major risk factor for pan-cancer development, with epidemiological studies indicating that 60% of new malignancies occur in adults age 65 and older. This review synthesizes cutting-edge insights from single-cell sequencing databases (e.g., TCGA and GEO) that decipher how aging reprograms the tumor microenvironment (TME) to fuel carcinogenesis. Single-cell RNA sequencing (scRNA-seq) has revealed that senescent cell subpopulations (e.g., CDKN2A+/LMNB1- cells) accumulate in aged tissues at frequencies up to 15%, driving genomic instability and secrete pro-tumorigenic senescence-associated secretory phenotype (SASP) factors (IL-6 and TGF-β). These factors remodel the TME by inducing fibroblast activation and extracellular matrix degradation, accelerating metastasis by 40-70% in murine models. Crucially, immunosenescence diminishes anti-tumor immunity, with scRNA-seq profiling showing 40-60% increases in exhausted PD-1+ T cells and immunosuppressive myeloid cells in aged TMEs. Pan-cancer analyses have identified conserved aging gene signatures (e.g., p16INK4a upregulation in 12+ cancer types) that correlate with 30-50% poorer survival. While technical challenges persist - including batch effects in scRNA-seq data and low senescent cell abundance (< 5%) - emerging solutions like deep learning can enhance detection sensitivity. Therapeutically, senolytic strategies deplete senescent cells, improving drug response by 3.5-fold in preclinical trials. Future research must integrate multi-omics and AI to examine aging-related targets, advancing personalized interventions for aging-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e67442df6e2549af855ce3f0cc864e0099eaa2" target='_blank'>
              Deciphering the nexus of aging and pan-cancer: Single-cell sequencing reveals microenvironmental remodeling and cellular drivers.
              </a>
            </td>
          <td>
            Yue Han, Nuo Chen, Ping Wang, Chu Zhou, Kenji Karako, Peipei Song, Wei Tang
          </td>
          <td>2025-10-26</td>
          <td>Bioscience trends</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Malignant brain tumors remain highly challenging to treat due to intrinsic and acquired therapy resistance and limited therapeutic options, consequently contributing to poor prognosis. Advancing the understanding of resistance mechanisms alongside novel treatment strategies is essential to improve clinical outcomes. Altered gene expression is common in tumors, and a specific class of non-coding RNAs, particularly long non-coding RNAs (lncRNAs), is frequently deregulated. LncRNAs play critical roles in processes such as cell proliferation, cell cycle arrest, and metastasis in brain cancer, functioning either as tumor promoters or suppressors. They exert their effects through transcriptional and post-transcriptional regulatory mechanisms. Understanding the functional roles of lncRNAs in malignant brain tumors has become a priority, as they are differentially expressed in tumors compared to healthy tissue. These molecules are studied for their potential as therapeutic targets and biomarkers in oncology. This review provides an overview of current research on brain cancer and lncRNAs, emphasizing the need for further investigation into their specific roles in therapy resistance and their involvement in various pathways. A better understanding of lncRNAs and their role in brain cancer could offer valuable insights into their prognostic and therapeutic potential, with the promise of improving early diagnosis and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f629bb465bf4a59c23101191cc50b7fe057523" target='_blank'>
              The Role of LncRNAs in Radio- and Chemoresistance of Glioblastoma: Prognostic or Therapeutic?
              </a>
            </td>
          <td>
            Elisa Tremante, Ana Belén Díaz Méndez, Maria Giulia Rizzo
          </td>
          <td>2025-09-27</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The initiation and progression of tumors are closely linked to aberrant regulation of multiple genes. Among these genes, sister chromatid cohesion acetyltransferase 2 (ESCO2) has emerged as an important target in cancer research. ESCO2 exhibits complex and heterogeneous expression patterns across various types of malignancies and tumor progression stages. Extensive studies have shown that ESCO2 primarily functions in chromosomal cohesion, significantly contributing to genomic stability maintenance. Dysregulated expression of ESCO2 in tumor cells is often correlated with malignant behaviors. Given its essential role in tumorigenesis, ESCO2 has increasingly been explored as a promising therapeutic target in recent years. This review article summarizes recent research progress regarding ESCO2 in malignant tumors, aiming to identify a novel target for their treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03164-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b1b540beced4302a863f075307c9fa3cb6930a" target='_blank'>
              The expression of establishment of cohesion 1 homolog 2 (ESCO2) in tumor cells and its research progress as a therapeutic target
              </a>
            </td>
          <td>
            Chunmei Lu, Hanchi Wu, Huiyu Wang
          </td>
          <td>2025-10-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metazoans suppress transposons via small RNA–mediated silencing, but ciliates physically eliminate transposons from somatic genomes. Here, we report the high-quality germline genome assembly of a marine ciliate Euplotes vannus, where ~80% are transposons and other germline-specific DNA that has to be precisely eliminated during soma development. We demonstrate that a class of soma-derived 30-nucleotide small RNAs precisely targets nontransposon DNA against elimination during this process. Small RNA–mediated targeting remains functional across varying small RNA lengths and is compatible with heterozygous sites. These small RNAs are cleaved by Dicer-like ribonuclease from long noncoding RNAs, which are bidirectionally transcribed of somatic chromosomes, initiated at subtelomeric chromosome breakage sequences (5′-TTGAA-3′). On the basis of these findings and time-course transcriptomic profiling, we propose a model elucidating the molecular mechanism of DNA elimination in Euplotes. These findings provide insights into the role of small RNAs in transmitting genetic information across generations and maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42400a7ce7bd33d02826f168bc703d3c0d0af547" target='_blank'>
              Soma-derived 30-nt small RNAs are coupled with chromosome breakage and precisely target nontransposon DNA against elimination in Euplotes vannus
              </a>
            </td>
          <td>
            Li-Yang Lyu, Bijia Ding, Jinyu Fu, I. Rzeszutek, E. Swart, M. Nowacki, Feng Gao
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889c401af57de3889b95cf28e228d76bc1fc7c6d" target='_blank'>
              Repeated losses of self-fertility shaped heterozygosity and polyploidy in yeast evolution
              </a>
            </td>
          <td>
            Nina Vittorelli, C. Gómez-Muñoz, Irina Andriushchenko, Louis Ollivier, Nicolas Agier, Stéphane Delmas, Yann Corbeau, Guillaume Achaz, M. Cosentino Lagomarsino, Gianni Liti, Bertrand Llorente, G. Fischer
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Introduction Colibactin is a small genotoxic molecule of polyketide produced by a subset of enteric bacteria including certain Escherichia coli (E. coli) harbored in the human gut microbiota. Its biosynthesis is governed by a multistep enzymatic process encoded by the polyketide synthase (pks) gene cluster. Colibactin is thought to exert its carcinogenic potential primarily through the induction of DNA interstrand crosslinks (ICLs); however, the precise mechanisms underlying its genotoxicity remain largely unresolved. In this study, we focused on ICL formation and its associated repair pathways to investigate whether colibactin-induced ICLs play a central role in the induction of chromosomal aberrations and inhibition of cell proliferation. Findings HAP1 cells deficient in FANCD2, a gene essential for ICL repair, and their wild-type counterparts were infected with colibactin producing (clb⁺) E. coli strains isolated from a Japanese colorectal cancer (CRC) patient. Following recovery culture, the frequency of micronucleated (MN) cells was assessed. The results showed that FANCD2-deficient cells exhibited a significantly higher frequency of MN cells compared to wild-type cells. Additionally, the cytotoxicity of the clb⁺ strains was evaluated using the XTT assay. FANCD2-deficient cells demonstrated higher sensitivity to the clb⁺ E. coli strains than wild-type cells. Conclusion These findings suggest that colibactin, produced by clb⁺ E. coli, can play a role in the formation of ICLs, thereby contributing significantly to the induction of chromosomal aberrations and the inhibition of human cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5434e698c009fe5d0b86871ed28e1115dcedc77" target='_blank'>
              DNA cross-link repair deficiency enhances human cell sensitivity to colibactin-induced genotoxicity
              </a>
            </td>
          <td>
            M. Kawanishi, Osamu Tsubohira, Ai Ueshima, Yuuta Hisatomi, Yoshimitsu Oda, Michio Sato, Noriyuki Miyoshi, M. Mutoh, Hideki Ishikawa, Keiji Wakabayashi, Takashi Yagi, Kenji Watanabe
          </td>
          <td>2025-09-22</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The DREAM complex has emerged as a central repressor of DNA repair, raising questions as to whether such repression exerts long-term effects on human health. Here we establish that DREAM activity significantly impacts lifetime somatic mutation burden, and that such effects are linked to altered lifespan and age-related disease pathology. First, joint profiling of DREAM activity and somatic mutations across a single-cell atlas of 21 mouse tissues shows that cellular niches with lower DREAM activity have decreased mutation rates. Second, DREAM activity predicts the varied lifespans observed across 92 mammals, with low activity marking longer-lived species. Third, reduced DREAM activity in Alzheimer’s patients predicts late disease onset and decreased risk for severe neuropathology. Finally, we show DREAM knockout protects against mutation accumulation in vivo, reducing single-base substitutions by 4.2% and insertion/deletions by 19.6% in brains of mice. These findings position DREAM as a key regulator of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ead31f56ffdb72707b88fc850bc30a7d800daa3" target='_blank'>
              The DREAM complex links somatic mutation, lifespan, and disease
              </a>
            </td>
          <td>
            Zane Koch, Shuvro P. Nandi, Kate Licon, Arturo Bujarrabal-Dueso, David H. Meyer, Safa Saeed, Pirunthan Perampalam, Frederick A. Dick, B. Schumacher, Ludmil B. Alexandrov, T. Ideker
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae85fb3f25140318cac9ff830fec7b82bf6f487" target='_blank'>
              Characterization of Human Endogenous Retroviruses in Gastric Cancer with Helicobacter pylori: A Study from Northern Brazil
              </a>
            </td>
          <td>
            Marcos da Conceição, Juliana Barreto Albuquerque Pinto, Diego Pereira, Rubem Ferreira da Silva, Sérgio Augusto Antunes Ramos, Daniel de Souza Avelar, Jéssica Manoelli Costa da Silva, Ronald Matheus da Silva Mourão, Valéria Cristiane Santos da Silva, Samir Mansour Moraes Casseb, S. Demachki, Tatiane Neotti, A. Anaissi, W. Barra, A. Vidal, G. Ishak, P. P. de Assumpção, R. Burbano, Fabiano Cordeiro Moreira
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, with an estimated 27.5 million new cases projected by 2040. Disruptions in cell cycle control cause DNA replication errors to accumulate during cell growth, leading to genomic instability and tumor development. Proteins that regulate cell cycle progression and checkpoint mechanisms are crucial targets for cancer therapy. NIMA-related kinases (NEKs) are a family of serine/threonine kinases involved in regulating various aspects of the cell cycle and mitotic checkpoints in humans. Among these, NEK10 is the most divergent member and has been associated with both cancer and ciliopathies, a group of disorders caused by defects in cilia structure or function. Despite its biological significance and distinctive domain architecture, the structural details of NEK10 remain largely unknown. To address this gap, we employed computational modeling techniques to predict the complete structure of the NEK10 protein. Our analysis revealed a catalytic domain flanked by two coiled-coil domains, armadillo repeats (ARM repeats), an ATP binding site, two putative ubiquitin-associated (UBA) domains, and a PEST sequence known to regulate protein degradation. Furthermore, we mapped a comprehensive interactome of NEK10, uncovering previously unreported interactions with the cancer-related proteins MAP3K1 and HSPB1. MAP3K1, a serine/threonine kinase and E3 ubiquitin ligase frequently mutated in cancers, interacts with the catalytic region of NEK10. The interaction with HSPB1, a molecular chaperone associated with poor cancer prognosis, is mediated by NEK10's ARM repeats. Our findings highlight a potential connection between NEK10, ciliogenesis, and cancer, suggesting an important role in cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84bfc7864763ec8c742e96b5b16848a5dc021a94" target='_blank'>
              In Silico Analysis of Human NEK10 Reveals Novel Domain Architecture and Protein-Protein Interactions.
              </a>
            </td>
          <td>
            A. S. Eichner, Nathaniel Zimmerman, Avdar San, Shaneen Singh
          </td>
          <td>2025-10-03</td>
          <td>Proteins</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979fccda6253a03a36cbc921b71b8ecae3f2890f" target='_blank'>
              Chromatin state dynamics of autosomes and the B chromosome during spermatogenesis in Pseudococcus viburni.
              </a>
            </td>
          <td>
            M. Herbette, Laura Ross
          </td>
          <td>2025-10-13</td>
          <td>Heredity</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa1e73dcc984320ef956ec066c98a00971bc8f16" target='_blank'>
              Progressively altered genes in colorectal carcinogenesis link oncogenesis immune cycle and tumor microenvironment.
              </a>
            </td>
          <td>
            Bingwen Zhou, Qingrui Liu, Chuyue Huang, Hao Chen, Pei Wang, Yueyang Lu, Shujun Jiang, Desong Kong, Lu Wang, Zhimin Fan
          </td>
          <td>2025-10-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07b66fdfdb5795523b8bbc3556aa7471128f3b" target='_blank'>
              Epithelial cell fusion is required for tissue repair following UV-A irradiation
              </a>
            </td>
          <td>
            Minqi Shen, Lillie G Mitchell, Lydia W Boer, Vicki P. Losick
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191c8d732cdb45c3ca75ffd01a7bb1837632f681" target='_blank'>
              Reverse transcribed ssDNA derepresses translation of a retron antiviral protein
              </a>
            </td>
          <td>
            Karen Zhang, Matías Rojas-Montero, Darshini Poola, Josepha M Klas, Arturo Carabias, Dennis J. Zhang, M. R. Mestre, Ruiliang Zhao, Ana Dávila-Hidalgo, Guillermo Montoya, R. Pinilla-Redondo, A. González-Delgado, Seth L. Shipman
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pathogenic mutations in the DNA repair genes, BRCA1/2, are responsible for approximately 11-15% of ovarian cancers (OC). Notably, BRCA1/2 mutations significantly increase the risk of developing an aggressive OC subtype known as high-grade serous ovarian cancer (HGSOC). The role of progesterone (P4) in the initiation and progression of OC subtypes remains poorly understood. Research suggests a potential interplay between Progesterone Receptor (PR) signaling and BRCA1/2 actions. BRCA1 regulates PR expression and transcriptional activity, while P4 can drive estrogen receptor/PR complexes to BRCA1 DNA binding motifs. Normal fallopian tube tissues from healthy BRCA1/2 carriers (the cells of origin for HGSOC) exhibit gene signatures similar to HGSOC only during the luteal phase of the menstrual cycle, when circulating P4 levels are elevated. This suggests that PR/BRCA complexes may modulate the initiation and progression of OC. To explore this PR/BRCA crosstalk, CRISPR-mediated BRCA1 knockdown (KD) was performed in ES-2 OC cell models that express the PR-B isoform. We hypothesized that depletion of BRCA1 would modulate PR-driven signaling and reveal novel regulatory mechanisms. Parental and KD cell pools were treated with vehicle or 10 nM R5020 (a synthetic progestin) for 6 hr, followed by RNA collection for analysis of R5020-induced gene expression using RNA sequencing. A high-confidence list of BRCA1-dependent, R5020-induced genes was generated for pathway analysis, which revealed that migration and cytoskeletal organization were the predominant PR-regulated processes. Significant genes of interest included ROCK2 and PIP5K1C, both of which are kinases and regulators of the cytoskeletal organization. Transwell migration assays showed that R5020 treatment promotes migration, which is further enhanced in the context of BRCA1 depletion. Co-treatment with a pan-ROCK1/2 inhibitor Y-27632 resulted in an increase in R5020-induced migration, which was further enhanced with BRCA1 KD. In contrast, inhibition of PIP5K1C using UNC3230 blocked R5020-induced migration in one of the BRCA1-depleted cell pools. Treatment with pan-Rho inhibitor CT04 led to a slight reduction in R5020-induced migration in the context of BRCA1 KD. Finally, the Rac1 inhibitor EHT-1864 suppressed R5020-induced migration. Analysis of known BRCA1 and PR DNA-binding sites revealed several binding motifs within genes related to cytoskeleton function, like ARHGEF37 and COL16A1. Additionally, some BRCA1 and/or PR binding sites also contain motifs for p130, a component of the DREAM complex, which regulates cell cycle genes. This suggests that a novel complex of BRCA1, PR, and DREAM proteins may exist which drives cytoskeleton rearrangement towards a more pro-metastatic state. Ongoing co-immunoprecipitation (IP) and chromatin-IP assays should identify these novel complexes and their target DNA binding sites. Future goals include validation in additional models of BRCA1/2 loss and biological assays, such as collagen invasion.



 Jocelyn Baquier, Noelle E. Gillis, Caroline H. Diep, Laura J. Mauro, Carol A. Lange. BRCA1 depletion enhances progesterone receptor-driven cytoskeletal remodeling in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A072.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9f53abea3f6459e27056f6425dca020d3d2e4c" target='_blank'>
              Abstract A072: BRCA1 depletion enhances progesterone receptor-driven cytoskeletal remodeling in ovarian cancer
              </a>
            </td>
          <td>
            Jocelyn Baquier, Noelle E. Gillis, Caroline H Diep, Laura J. Mauro, C. Lange
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Breast cancer can sometimes return after initial treatment, but predicting who is most at risk remains difficult. Our matched case–control study examined DNA methylation, defined as chemical variations in the DNA that influence gene activity. We analysed blood samples from women with breast cancer who did not carry BRCA mutations, comparing those who later had recurrence with those who remained cancer-free. Three DNA regions showed different methylation patterns between the two groups. These regions contain three genes known to contribute to important processes such as cell growth, invasion, and how the body metabolises sugar. These findings suggest that DNA methylation patterns in blood could one day help identify women at higher risk of recurrence and guide more personalised follow-up and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f83da598253a64b021ec56c05f3a8aa976ea56e" target='_blank'>
              Impact of Epigenome-Wide Methylation and Breast Cancer Recurrence in Women Tested Negative for BRCA Genes: The Breast Methylation Risk (BREMERI) Study
              </a>
            </td>
          <td>
            Silvia Polidoro, H. Johansson, Giovanni Cugliari, A. Guerrieri-Gonzaga, Valentina Aristarco, D. Macis, M. Calvello, Monica Marabelli, I. Feroce, D. Serrano, S. Cagnacci, C. Zanzottera, Francesca Fava, F. Bellerba, B. Bonanni, S. Gandini
          </td>
          <td>2025-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Topoisomerase I (TopoI) is the sole DNA relaxase in Mycobacterium tuberculosis. Despite being a validated drug target and indispensable to the pathogen, only a limited repertoire of inhibitors targeting the enzyme have been identified. We employed monoclonal antibodies (mAbs) to address this shortfall. From the pool of a large number of mAbs, we describe an inhibitory mAb specific to mycobacterial TopoI with a distinct mechanism of action. Among the various steps of the TopoI reaction cycle, the mAb does not interfere with DNA binding but impedes DNA cleavage. It does not alter the religation activity of TopoI; however, it inhibits its strand passage activity. Probing with the mAb, we show the precise step at which the topology of the DNA is changed during DNA relaxation reaction. Surprisingly, instead of the initial strand scission action of the enzyme, the subsequent strand passage followed by the second transesterification entails the alteration in DNA topology. With their selective and specific inhibitory properties, the mAb and its derived single-chain variable fragment (ScFv) would serve to probe the structure of mycobacterial TopoI and as a starting point in designing peptide inhibitors with therapeutic potential to combat the rampant drug-resistant M. tuberculosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c8f325ed8b9afae307dd8fcfa34c017dd69964" target='_blank'>
              Inhibition of DNA cleavage and strand passage activities of Mycobacterium tuberculosis topoisomerase I
              </a>
            </td>
          <td>
            Iqball Faheem, M. N. Leelaram, Immadi Siva Ratnakar, Rinkee Verma, V. Nagaraja
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Megaphages are bacteriophages (i.e., phages) with exceptionally large genomes that have been discovered computationally across the globe. To date, all have evaded cultivation except phage G, which we examined with multiomics and artificial intelligence (AI) to resolve its 50-year cultivated history. Phage G is one of the largest phages with a size of >0.6 µm, about half the width of the host cell, and a 499 kbp, non-permuted, linear genome that has, uniquely among known phages, two pairs of ends. Phage G has >650 protein-coding open reading frames (ORFs), with >65% being hypothetical proteins, with the expansive repertoire of auxiliary metabolic genes (AMGs) acquired from its bacterial host, manipulate host sporulation (sspD, RsfA, spoK), and antiviral escape genes (e.g., anti-CBass nuclease and Anti-Pycsar protein). Our study represents a doorway into the complexity of the genomic repertoire of the only cultivated megaphage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14446808f1116f5a42848927b1c983a852439d05" target='_blank'>
              Unlocking the genomic repertoire of a cultivated megaphage
              </a>
            </td>
          <td>
            Andra Buchan, Stephanie Wiedman, Kevin C Lambirth, Madeline Bellanger-Perry, Jose L. Figueroa, E. Wright, Patil Shivprasad Suresh, Qibin Zhang, Julie A. Thomas, Philip Serwer, Richard Allen White
          </td>
          <td>2025-09-30</td>
          <td>npj Viruses</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes
 TP53
 and
 RB1
 . However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yue Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="DNA Viruses manipulate numerous host factors and cellular pathways to usurp and control cellular activities through the orchestrated recruitment of macromolecules to induce abnormal growth of the host cell. In this review, polyomaviruses and their structures, replication and their roles in abnormal growth formation are summarized. Polyomaviruses are non-enveloped icosahedral double stranded DNA viruses with genome size of 5kb. The genome codes for early protein (large tumor antigen and small tumor antigen) which play major role in replication of the viruses and abnormal cell proliferation and late proteins (VP1, VP2, VP3 and Agno proteins) for capsid formation, viral exit and regulation of the viral life cycle. The replicative strategies of the viruses and their roles in cancer formation are reviewed. Polyomaviruses (JC virus, BK virus and sinia virus 40) establish subclinical and resistance infection and share the capacity for reactivation from latency in their host under immunosuppression. Polyomaviruses are mostly common in children, with little or no symptoms, and probably lifelong persistent in adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5da69bd87cd13122d935c1a6a504b506a897061" target='_blank'>
              Polyomaviruses: Structure, Replication Strategies, and Oncogenic Roles in Human Diseases
              </a>
            </td>
          <td>
            I. Iheukwumere, C. Iheukwumere, B. U. Unaeze, V. E. Ike, H. C. Nnadozie, S. O. Onyema
          </td>
          <td>2025-10-13</td>
          <td>IPS Journal of Public Health</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) play a crucial role in normal and dysregulated hematopoiesis. However, the functional repertoire of lncRNAs across various hematopoietic cell types remains elusive. In this study, we constructed a comprehensive single-cell lncRNA atlas containing 207,113 cells, spanning hematopoietic stem and progenitor cells (HSPCs) to differentiated blood cells, by integrating nine single-cell RNA sequencing (scRNA-seq) datasets derived from 30 healthy donors. The hematopoietic hierarchy based on lncRNA expression was highly consistent with that based on protein-coding genes. We identified 3,463 lineage-specific lncRNAs in HSPCs, neutrophils, monocytes, B cells, and T/natural killer cells; 23 of 30 selected lncRNAs were experimentally validated. Importantly, upregulated lncRNAs in pediatric patients with B cell acute lymphoblastic leukemia, T cell acute lymphoblastic leukemia, and acute myeloid leukemia were primarily associated with oxygen response and immune regulation, indicating the potential contribution of lncRNAs to leukemogenesis. In conclusion, our results portray the landscape of lncRNAs in the hematopoietic system, revealing the functional significance of lncRNAs in both normal and abnormal hematopoiesis and providing potential therapeutic targets for the clinical treatment of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b7702ca18330467cd9744909a67f48d812dd85" target='_blank'>
              Single-cell profiling of long non-coding RNAs in the hematopoietic system
              </a>
            </td>
          <td>
            Yu Zhang, Qiuju Huang, Yawen Zhang, Shihong Lu, Lianshuo Li, Shangda Yang, Tao Cheng, Hui Cheng, Xiaowei Xie
          </td>
          <td>2025-09-17</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2µm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epigenomic dysregulation is widespread in cancer. However, the specific epigenomic regulators and the processes they control to drive cancer phenotypes are poorly understood. We employed a novel high-throughput in vivo method to perform iterative functional screens of >250 epigenomic regulators within autochthonous oncogenic KRAS-driven lung tumors. We identified many previously unappreciated epigenomic tumor-suppressor and tumor-dependency genes. We show that a specific HBO1 complex and MLL1 complex are robust tumor suppressors in lung adenocarcinoma. Histone modifications generated by HBO1 complex are frequently reduced in human lung adenocarcinomas and are associated with worse clinical features. HBO1 and MLL1 complexes co-occupy shared genomic regions, impact chromatin accessibility, and control the expression of canonical tumor suppressor genes and lineage fidelity. The HBO1 complex is epistatic with the MLL1 complex and other tumor suppressor genes in lung adenocarcinoma development. Collectively, these results provide a phenotypic roadmap of epigenomic regulators in lung tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37f95c52f78e3109ff8dd022582b5c7f1653fce" target='_blank'>
              Functional mapping of epigenomic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes.
              </a>
            </td>
          <td>
            Yuning J Tang, Haiqing Xu, Nicholas W. Hughes, Paloma A. Ruiz, Samuel H. Kim, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Christian A Kunder, Colin Detrick, Sarah E. Pierce, Emily L Ashkin, William J. Greenleaf, Anne K. Voss, Tim Thomas, Matt van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2025-09-26</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The clonal evolution model provides a framework for understanding the evolution of cancer cells. According to this model, cancer cells accumulate genetic mutations over time, and these mutations are passed down to their descendants, leading to genetic diversity within the tumor. Some of these mutations confer selective advantages, causing certain lineages of cancer cells (clones) to dominate and expand. However, this model is rooted in certain conceptual assumptions, which we propose to revisit by considering the potential involvement of retrotransposons in cancer initiation and progression. In recent years, it has become evident that transposable elements, particularly retrotransposons, play a significant role in driving cancer transformation and progression. We first review how current knowledge about retrotransposon activity aligns with the clonal evolution model by highlighting its ability to modulate cancer cell fitness. We then take a forward-looking perspective to explore additional ways retrotransposons may also influence clonal dynamics beyond the current model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb2629152d1c164e115163464d0e7599e1333a7" target='_blank'>
              Revisiting Clonal Evolution Through the Light of Retrotransposons.
              </a>
            </td>
          <td>
            Anaïs Lamoureux, Emilie Elvira-Matelot, Françoise Porteu, L. Laplane
          </td>
          <td>2025-10-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Gastric cancer (GC) is the fifth most common malignancy and one of the leading causes of cancer-related death worldwide. Its histological and molecular heterogeneity make it particularly challenging to manage. The Cancer Genome Atlas classifies GC into four molecular subtypes: Epstein-Barr virus-associated GC (EBVaGC), microsatellite instability-high (MSI-H), chromosomal instability (CIN), and genomic stability (GS), each of which has distinct genetic and epigenetic characteristics. Among this biomarker diversity, non-coding RNAs (ncRNAs) such as microRNAs (miRNAs) and long non-coding RNAs play a key role in diagnosis, prognosis, and targeted therapy. For instance, the EBVaGC subtype features PIK3CA mutations and hypermethylation of tumor suppressor genes such as CDKN2A, alongside ncRNAs such as EBV-encoded RNAs and H19 that enhance immunogenicity and response to programmed death-1/programmed death-ligand 1 inhibitors. MSI-H-GC is characterized by high mutational load and DNA mismatch repair defects, and ncRNAs such as MIR99AHG serve as prognostic and immunomodulatory markers. CIN-GC, the most common subtype, is associated with amplification of genes such as ERBB2/HER2 (human epidermal growth factor receptor 2) and ncRNAs such as miR-22 and TERRA, which exacerbate CIN and are linked to a poor prognosis may be amenable to HER2-targeted therapies. GS-GC is characterized by RHOA and CDH1 mutations and epithelial-to-mesenchymal transition (EMT) features, where ncRNAs such as HOX antisense intergenic RNA, ZFAS1, and Linc00152 affect invasion and metastasis by regulating EMT. Through interactions with miRNAs and signaling pathways, these ncRNAs not only influence prognosis but also represent novel therapeutic targets. Integrating multiomics approaches and developing ncRNA-based biomarker panels are essential for advancing precision medicine in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c22941612a5bd78d41128902ec19da9cf9443676" target='_blank'>
              Subtype-Specific Noncoding Ribonucleic Acid Signatures in Gastric Cancer: Insights from The Cancer Genome Atlas-Based Classification.
              </a>
            </td>
          <td>
            Mohsen Sharif-Zak, Omid Dehqani Dashtabi, Fatemeh Binayi, Behnam Saeidi, Siavash Shirzadeh Barough, Amir Sadeghi, Mehdi Totonchi
          </td>
          <td>2025-10-20</td>
          <td>DNA and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In this study, we delineated the distinct transcriptomic effects of VHL mutation and chromosome 3p (chr3p) loss, revealing that chr3p loss is specifically associated with immune suppression in clear cell renal cell carcinoma (ccRCC). Furthermore, we developed driver genomic aberration (DGA) gene signatures that demonstrate superior performance in predicting both patient prognosis and treatment response compared to traditional mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8edc71bb1913e06b7ed3c5e92c20979a6f59e6" target='_blank'>
              Unravelling the distinct effects of VHL mutations and chromosome 3p loss in clear cell renal cell carcinoma: Implications for prognosis and treatment
              </a>
            </td>
          <td>
            Xiang Wang, Jian-Rong Li, N. Chowdhury, Lang Wu, Cheng Chao
          </td>
          <td>2025-09-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Advances in genetic studies have not only improved the diagnosis and treatment of neurodevelopmental disorders but also uncovered human-specific aspects of nervous system development. The generation of neuronal diversity in the human brain relies on tightly regulated epigenetic mechanisms, with Polycomb Repressive Complex 2 (PRC2) emerging as a key player. In this review, we first summarize foundational studies that established the role of PRC2 in the epigenetic maintenance of transcriptional silencing. We then highlight recent insights into the increasing evolutionary complexity of PRC2 subcomplexes, their roles in neurodevelopment, and their contribution to human developmental disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529c3449f4db7c46e51c854c049686e64626ce24" target='_blank'>
              PRC2 Diversity in Neuronal Differentiation and Developmental Disorders
              </a>
            </td>
          <td>
            Jasmine Akoto, Thomas Roulé, Naiara Akizu
          </td>
          <td>2025-10-13</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) and homopolymers are among the most mutable loci in the human genome. Despite their presumed mutability owing to replication slippage, homopolymer loci exhibit lower mutation rates and minimal paternal age effects compared to other STRs. This paradox questions if technical limitations, rather than biological mechanisms, explain these observations. Results We used the Element Biosciences AVITI platform to sequence the genomes of a 48-member, four-generation CEPH/Utah pedigree. As the AVITI platform reduces error rates at repetitive sequences compared to Illumina, this design enabled accurate mutation discovery at 90% of assayed homopolymers and a 1.7-fold increase in discoverable mutations compared to Illumina. We identified a median of 35 de novo homopolymer mutations per trio and a mutation rate of 5.28 × 10−5 DNMs per locus per generation, confirming a lower rate than dinucleotides (1.94 × 10−4). Most DNMs were single base-pair expansions or contractions. Despite comprising <1% of homopolymer loci, G/C homopolymers showed 18-fold higher mutation rates than A/T homopolymers; in contrast, the high dinucleotide mutation rate is not driven by a particular motif class. Parent-of-origin analysis revealed 78% of homopolymer mutations are paternal in origin, but no significant paternal age effect was observed. Conclusions This study confirms that homopolymers exhibit lower mutation rates and lack strong paternal age effects compared to other STRs, likely owing to the combination of a lower propensity to form slippage-causing secondary structures and more efficient mismatch repair. Our set of high-quality mutations suggest these phenomena are biological rather than technical in nature. Finally, we demonstrate that AVITI sequencing unlocks previously intractable regions of the genome and will be a powerful tool for continued investigation of repeat mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8a40830239596b30389d22f950b3a9f1ae5008" target='_blank'>
              AVITI sequencing of a four-generation CEPH/Utah pedigree confirms low mutation rates at homopolymer loci despite their low sequence complexity
              </a>
            </td>
          <td>
            Hannah C. Happ, T. Sasani, Derek Warner, Deborah W Neklason, Aaron R. Quinlan
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is a fundamental epigenetic process that balances X-linked gene expression between females and males by silencing one X chromosome in female cells. Variability or skewing of XCI can influence the clinical presentation of X-linked disorders. Bain type X-linked intellectual disability syndrome (MRXSB), caused by mutations in the X-linked HNRNPH2 gene, is characterized by intellectual disability, developmental delay, and neurological abnormalities. In female patients, XCI heterogeneity complicates disease modeling and therapeutic development. Induced pluripotent stem cells (iPSCs) offer a unique platform to study patient-specific disease mechanisms, but the dynamics of XCI during iPSC reprogramming, maintenance, and differentiation are not fully understood. In this study, we generated 12 iPSC clones from fibroblasts of a female MRXSB patient heterozygous for the HNRNPH2 c.340C > T mutation. Four clones expressed the mutant HNRNPH2 allele and eight expressed the wild-type allele, indicating X chromosome reactivation (XCR) followed by random XCI during reprogramming. Importantly, these XCI patterns remained stable during long-term iPSC propagation and subsequent differentiation into the three germ layers and neural stem cells. Our findings provide new insights into XCI and XCR dynamics in the context of X-linked neurodevelopmental disorders and emphasize the importance of careful clone selection for accurate disease modeling using iPSC-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6b0bf32c5498df56453ec6772d7356b34b8b65" target='_blank'>
              Reactivation of Human X-Linked Gene and Stable X-Chromosome Inactivation Observed in Generation and Differentiation of iPSCs from a Female Patient with HNRNPH2 Mutation
              </a>
            </td>
          <td>
            Guibin Chen, Alexander Rodriguez-Lopez, Darawalee Wangsa, Richa Madan Lomash, Xiuli Huang, Catherine Z. Chen, Rodney A. Bowling, Neda Ghousifam, Courtney J. Banks, Kerstin A. Hurd, Jizhong Zou, Wei Zheng
          </td>
          <td>2025-09-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT Background Liquid biopsy with circulating tumor DNA (ctDNA) has rapidly emerged as a new paradigm for assessing tumor burden, genetic heterogeneity, and therapeutic response in a real‐time, noninvasive manner. However, ctDNA is often visually low (sometimes < 0.1% of the total circulating cell‐free DNA), creating a significant challenge for reliable detection (especially for early‐stage disease and minimal residual disease). Discussion New technologies for structural variant (SV)‐based ctDNA assays, nanomaterial‐based electrochemical sensors, magnetic nano‐electrode platforms, and fragment‐enriched library preparation have improved sensitivity to attomolar concentrations and less in some populations. In some cancers, ctDNA may provide early evidence of recurrence (i.e., > 1 year) before being clinically evident using traditional metrics. These technologies allow for unprecedented opportunities and sensitivity for early detection, monitoring of treatment response, and early detection of molecular recurrence. Nevertheless, a barrier remains for widespread clinical application owing to pre‐analytical technique variability, analytical platform variability, cost, and the necessity of large‐scale, prospective trials. Conclusion This study will analyze new innovative technology‐based ultrasensitive ctDNA assay detection and future research and clinical applications for breast, colorectal, lung, lymphoid, and gastroesophageal cancers, and studies assessing ctDNA for monitoring treatment. Prospects for ctDNA detection utilizing multiplexed CRISPR‐Cas ctDNA assays, microfluidic point‐of‐care (POC) devices, and AI‐based error suppression methods may be the next horizon for ctDNA liquid biopsy technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d92e4bd8fe5b760142696465f97e66c2d58c3bdc" target='_blank'>
              Monitoring and Assessment of Circulating Tumor DNA in Cancers Using Ultrarapid Sensitivity as an Innovative Practice
              </a>
            </td>
          <td>
            Md Mohiuddin
          </td>
          <td>2025-10-01</td>
          <td>Health Science Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>