<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-26 09:52:22 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>34</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>71</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>112</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>335</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>293</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a common source of oncogene amplification across many types of cancer. The non-Mendelian inheritance of ecDNA contributes to heterogeneous tumour genomes that rapidly evolve to resist treatment. Here, using single-cell and live-cell imaging, single-micronucleus sequencing, and computational modelling, we demonstrate that elevated levels of ecDNA predisposes cells to micronucleation. Damage on ecDNA, commonly arising from replication stress, detaches ecDNA from the chromosomes upon which they hitchhike during cell division, thereby causing micronucleus formation in daughter cells. Clusters of oncogene-containing, CIP2A-TOPBP1-associated ecDNA molecules form, and asymmetrically segregate into daughter cell micronuclei during cell division. ecDNA chromatin remains highly active during mitosis, but upon micronucleation, it undergoes suppressive chromatin remodeling, largely ceasing oncogene transcription. These studies provide insight into the fate of damaged ecDNA during cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7372198dd7ad93e0ac09354eea55bbe4096844" target='_blank'>
              Human Papillomavirus Genomes Associate with Active Host Chromatin during Persistent Viral Infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, P. Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a97fcf511bf3ac6d39959b0d32a3431fd3b9263" target='_blank'>
              Abstract 7504: Integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH-wildtype integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH wild-type glioblastoma
              </a>
            </td>
          <td>
            Dacheng Zhao, K. Anderson, Amit Gujar, Megan L Costa, Djamel Nehar-Belaid, F. Varn, Elise T. Courtois, Paul Gabrie, Paul Robson, H. Moon, A. Gołȩbiewska, S. Paek, S. Niclou
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, Katie Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, M. Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M. Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b54c36ea8dc8ae6d831b9cec1f08634da1f520c" target='_blank'>
              Abstract 1152: The landscape of copy number variations following WGD in 5,980 colorectal cancers
              </a>
            </td>
          <td>
            Jierui Xu, H. Walch, Paolo Manca, Maria A Perry, Christopher Fong, Justin Jee, K. Pichotta, E. Vakiani, Neil H. Segal, Adam Price, Allison L Richards, Chai Bandlamudi, M. Donoghue, A. Cercek, R. Yaeger, N. Schultz, W. Chatila
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9dfba50807f4d056e7d436de2c34ed7ddd425f" target='_blank'>
              Abstract 1502: Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx
              </a>
            </td>
          <td>
            Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, C. Swanton, M. Jamal-Hanjani, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e63be79d13b386a8293331209bd30415a9a147" target='_blank'>
              Enrichment of Z-DNA-Forming Sequences Within Super-Enhancers: A Computational and Population-Based Study
              </a>
            </td>
          <td>
            Yulia V. Makus, G. A. Ashniev, Alexey V. Orlov, Petr I. Nikitin, Zoia G. Zaitseva, N. Orlova
          </td>
          <td>2025-05-04</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de502f1073df10f45cbf431033ea87108c552c" target='_blank'>
              ARID1A stabilises non-homologous end joining factors at DNA breaks induced by the G4 ligand pyridostatin
              </a>
            </td>
          <td>
            Susana Simões-Sousa, Noa Amin, Karen A. Lane, Alison Harrod, Malin Pedersen, Mercedes Pardo, Christine Rasetti-Escargueil, Z. Kozik, Katheryn A. G. Begg, Alison Ribeiro, Susana Banerjee, Manuel Stucki, Kevin J. Harrington, Jyoti S. Choudhary, Jessica A. Downs
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8be723b80118653ca09b51446fe4768c5505b8" target='_blank'>
              A Guide to Extrachromosomal DNA: Cancer's Dynamic Circular Genome.
              </a>
            </td>
          <td>
            Natasha E Weiser, Thomas B K Watkins, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Chromosomal rearrangements involving additional material in individuals with phenotypic alterations usually result in partial trisomy, often accompanied by partial monosomy. To characterize chromosomal rearrangements and analyze genomic characteristics in the breakpoint regions in 31 patients with additional material on an autosomal chromosome. Different tests were performed to characterize these patients, including karyotyping, chromosomal microarray analysis (CMA), and fluorescent in situ hybridization (FISH). In silico analyses evaluated A/B chromosomal compartments, segmental duplications, and repetitive elements at breakpoints. The 31 rearrangements resulted in 47 copy number variations (CNVs) and a range of structural aberrations were identified, including six tandem duplications, 19 derivative chromosomes, two intrachromosomal rearrangements, one recombinant, two dicentric chromosomes, and one triplication. A deleted segment was associated with the duplication in 16 of the 19 patients with derivative chromosomes from translocation. Among the trios whose chromosome rearrangement origin could be investigated, 54,5% were de novo, 31,9% were maternally inherited, and 13,6% were paternally inherited from balanced translocations or inversion. Breakpoint analysis revealed that 22 were in the A compartment (euchromatin), 25 were in the B compartment (heterochromatin), and five were in an undefined compartment. Additionally, 14 patients had breakpoints in regions of segmental duplications and repeat elements. Our study found that a deletion accompanied by additional genetic material was present in 51.6% of the patients, uncovering the underlying genetic imbalances. Statistical analyses revealed a positive correlation between chromosome size and the occurrence of CNVs in the rearrangements. Furthermore, no preference was observed for breakpoints occurring in compartments A and B, repetitive elements, or segmental duplications. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97077-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4680fc424cbc00a4561d2d03671779c2ded30f6" target='_blank'>
              Cytogenomic characterization of karyotypes with additional autosomal material
              </a>
            </td>
          <td>
            Priscila Soares Rodrigues, Bruna Burssed, F. Bellucco, Débora Cristina Batista Rosolen, Chong Ae Kim, M. Melaragno
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) are circular, double-stranded DNA molecules ubiquitously present across various organisms, playing a critical role in tumorigenesis and tumor progression. However, their precise contribution to prostate cancer (PCa) remains incompletely understood. To elucidate the function of eccDNAs in PCa, eccDNA sequencing and annotation were performed in PCa tissues and cell lines using Circle-seq. Amplified genes on eccDNAs were identified by cross-referencing annotated eccDNA-associated genes with those overexpressed in PCa based on TCGA data. Furthermore, eccDNA profiles were compared between Enzalutamide-sensitive and -resistant cell lines to investigate their role in resistance mechanisms. Notably, VAV2 was detected on both linear and circular DNA, as confirmed by PCR and Sanger sequencing. Functional analyses demonstrated that VAV2 overexpression promotes PCa proliferation and metastasis by activating the PAK1/AKT signaling pathway through PAK1 phosphorylation. Additionally, VAV2 contributes to Enzalutamide resistance by enhancing AR/ARv7 protein stability via reduced ubiquitination, mediated through the recruitment of the deubiquitinating enzyme USP48. These findings establish VAV2, identified through eccDNA sequencing, as a potential oncogene and a promising biomarker for PCa diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin’an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1dc55203f4f61b62677fd9af379d34ddca08f1" target='_blank'>
              Abstract 1479: Defining the interplay between DNAPKcs and shieldin during chromosomal break repair
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogene activation is a key driver of cancer development, inducing aberrant cellular proliferation and DNA replication stress. This in turn, leads to DNA damage-which accumulates in specific genomic regions-contributing to genomic instability in cancer. However, the interplay between oncogene-induced DNA damage and chromatin organization is still poorly understood. In this study, we introduce a QUantitative ANalysis of DNA cOunterstains (QUANDO) to investigate the subnuclear localization of DNA damage in single-cell nuclei of U937-PR9 cells, an in vitro model of acute promyelocytic leukemia (APL). Using advanced imaging techniques, including DNA intensity analysis and colocalization by image cross-correlation spectroscopy (ICCS), we map DNA damage foci and correlate them with chromatin regions of different density. QUANDO is applied to dual-color confocal images of the DNA damage marker γ-H2AX and the DNA counterstain DAPI, allowing single-cell measurements of foci distribution within areas of low or high DNA density. We find that spontaneous DNA damage and DNA damage induced by the activation of PML-RARα oncogene predominantly localize in euchromatic regions. Conversely, when DNA damage is induced by the radiomimetic agent neocarzinostatin (NCS), the foci appear more evenly distributed in euchromatic and heterochromatic regions. These findings underscore the complex interplay between oncogene activation and chromatin organization, revealing how disruptions in DNA damage distribution can contribute to genomic instability and offering new insights for targeting DNA repair mechanisms in cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524273685b3a7b2a132bed81549714ac0e395f5e" target='_blank'>
              Location of oncogene-induced DNA damage sites revealed by quantitative analysis of a DNA counterstain.
              </a>
            </td>
          <td>
            Greta Paternò, Silvia Scalisi, G. Dellino, M. Faretta, P. Pelicci, Alberto Diaspro, Luca Lanzanò
          </td>
          <td>2025-05-07</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e357c0e1274581f338acdf17e4b08fe74b5c828" target='_blank'>
              Safeguarding spermatogenesis from retrotransposon insertions by forming ecDNA
              </a>
            </td>
          <td>
            Lauren Tracy, Z. Zhang
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy J. Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNA topoisomerase II beta (TOP2B) is required for correct execution of certain developmental transcriptional programs and for signal-induced transcriptional activation, including transcriptional activation by nuclear hormone ligands such as retinoic acid. In addition, TOP2B is enriched at genomic locations occupied by CCCTC-Binding factor (CTCF) and cohesin (RAD21). suggesting a role in chromosome looping and/or establishing or maintaining aspects of chromosome 3D structure. This led us to investigate the effect of TOP2B inactivation on patterns of intra- and inter- chromosomal interaction that reflect the 3D architecture of the genome. Using the retinoic acid responsive SH-SY5Y neuroblastoma cell line model, we had previously demonstrated many gene expression changes upon retinoic acid treatment and upon deletion of TOP2B. We report here that these expression changes in TOP2B null versus WT cells are accompanied by surprisingly subtle changes in local chromosome organization. However, we do observe quantitative changes in chromosome organization on a megabase scale. First, lack of TOP2B did affect compartment strength changes that occur upon ATRA treatment. Second, we observe an excess of very long-range interactions, reminiscent of interactions seen in mitotic cells, suggesting the possibility that in the absence of TOP2B some mitotic interactions are retained. Third, we see quantitative changes in centromere-telomere interactions, again indicating global changes at the megabase and chromosome level. These data support the surprising conclusion that TOP2B has only a minor role in chromosome dynamics and organization. Supplementary Information The online version contains supplementary material available at 10.1007/s10577-025-09764-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a98f3b2f42315551f858c79318b6bc8b811fe0" target='_blank'>
              TOP2B is required for compartment strength changes upon retinoic acid treatment in SH-SY5Y cells
              </a>
            </td>
          <td>
            Erica M Hildebrand, I. Cowell, M. M. Khazeem, Snehal Sambare, Ozgun Uyan, J. Dekker, Caroline A Austin
          </td>
          <td>2025-04-04</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a9cd59ca60eda9644fa34626fa28cc8acd1e4d" target='_blank'>
              The LINC complex component Kms1 and CENP-B protein Cbp1 cooperate to enforce faithful homology-directed DNA repair at the nuclear periphery in S. pombe
              </a>
            </td>
          <td>
            Alyssa Laffitte, Dongxu Lin, Yingzhen Jenny Tian, Na Liu, C. Lusk, S. Mochrie, Megan C. King
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are the effective lesion of cancer radiotherapy and induce gene editing. 53BP1 accumulates at DSBs and is implicated in end joining (EJ) repair, but its influence on DSB repair is distinct from canonical non-homologous end joining (C-NHEJ). We sought to define the genetic interplay of 53BP1 with C-NHEJ, focusing on the DNA-PKcs kinase. We examined Cas9 DSBs, which largely generates blunt DSBs, since blunt DSB EJ is dependent on C-NHEJ. Loss of 53BP1 does not affect blunt DSB EJ, but causes a reduction in such repair in DNA-PKcs deficient cells. In contrast, disrupting 53BP1 and DNA-PKcs, both alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We found similar effects on EJ with RIF1 loss, which is a downstream effector of 53BP1. Thus, 53BP1/RIF1 appear to play a backup role for DNA-PKcs during blunt DSB EJ, but function in the same pathway to suppress microhomology deletions. In contrast, 53BP1 and DNA-PKcs function independently to suppress homology-directed repair. Finally, DNA-PKcs kinase inhibition causes marked radiosensitivity, which is not additive with loss of 53BP1 and RIF1. Altogether, 53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb1f8550e47aed05c0a51b4b294e0030758aa9f" target='_blank'>
              53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62dd3d2788ffe055d52a584d9b884866f23c266" target='_blank'>
              Delayed indel formation after Cas9 cleavage promotes mosaicism and aneuploidy in human embryos
              </a>
            </td>
          <td>
            Alex Robles, Julie Sung, Stepan Jerabek, Jishnu Talukdar, Diego Marin, Jia Xu, Nathan Treff, Dieter Egli
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d18b66a6ee1c58ac1f33fec9b297a5aefdcb0434" target='_blank'>
              c-MYC can Form Phase Condensates via its DNA Binding Domain
              </a>
            </td>
          <td>
            Udhaya Vijayakumar, Ruchi Choudhary, W. Chng, Melissa Fullwood
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bd3d524ef4cdc3c87b3f69c07be4376d8e44c3" target='_blank'>
              Novel Avenues for the Detection of Cancer-Associated Viral Genome Integrations Using Long-Read Sequencing Technologies
              </a>
            </td>
          <td>
            Larissa-Anna Bergmann, Alicja Pacholewska, Michal R. Schweiger
          </td>
          <td>2025-05-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract APOBEC mutagenesis is one of the most common endogenous sources of mutations in human cancer and is a major source of genetic intratumor heterogeneity. High levels of APOBEC mutagenesis are associated with poor prognosis and aggressive disease across diverse cancers, but the mechanistic and functional impacts of APOBEC mutagenesis on tumor evolution and therapy resistance remain relatively unexplored. To address this, we investigated the contribution of APOBEC mutagenesis to acquired therapy resistance in a model of EGFR-mutant non–small cell lung cancer. We find that inhibition of EGFR in lung cancer cells leads to a rapid and pronounced induction of APOBEC3 expression and activity. Functionally, APOBEC expression promotes the survival of drug-tolerant persister cells (DTP) following EGFR inhibition. Constitutive expression of APOBEC3B alters the evolutionary trajectory of acquired resistance to the EGFR inhibitor gefitinib, making it more likely that resistance arises through de novo acquisition of the T790M gatekeeper mutation and squamous transdifferentiation during the DTP state. APOBEC3B expression is associated with increased expression of the squamous cell transcription factor ΔNp63 and squamous cell transdifferentiation in gefitinib-resistant cells. Knockout of p63 in gefitinib-resistant cells reduces the expression of the ΔNp63 target genes IL-1α/β and sensitizes these cells to the third-generation EGFR inhibitor osimertinib. These results suggest that APOBEC activity promotes acquired resistance by facilitating evolution and transdifferentiation in DTPs and that approaches to target ΔNp63 in gefitinib-resistant lung cancers may have therapeutic benefit. Significance: APOBEC mutagenesis is a common source of genetic heterogeneity in cancer, and APOBEC mutational signatures are enriched in metastatic and drug-resistant tumors. However, the mechanisms through which APOBEC3 enzymes promote tumor evolution remain unknown. In this study, we show that APOBEC3 activity contributes to the development of therapy-resistant cancer cells by promoting evolution of DTP cells. These findings offer insights into the role of APOBEC mutagenesis in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 Activity Promotes the Survival and Evolution of Drug-Tolerant Persister Cells during EGFR Inhibitor Resistance in Lung Cancer
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Stem cells have the unique ability to self-renew and differentiate into specialized cell types. Epigenetic mechanisms, including histones and their post-translational modifications, play a crucial role in regulating programs integral to a cell’s identity, like gene expression and DNA replication. However, the transcriptional, chromatin, and replication timing profiles of adult stem cells in vivo remain poorly understood. Containing germline stem cells (GSCs) and somatic cyst stem cells (CySCs), the Drosophila testis provides an excellent in vivo model for studying adult stem cells. However, the small number of stem cells and cellular heterogeneity of this tissue have limited comprehensive genomic studies. In this study, we developed cell type-specific genomic techniques to analyze the transcriptome, histone modification patterns, and replication timing of GSC-like and CySC-like cells. Single cell RNA-sequencing validated previous findings on GSC-CySC intercellular communication and revealed high expression of chromatin regulators in GSCs. To characterize chromatin landscapes, we developed a cell-type-specific chromatin profiling assay to map H3K4me3-, H3K27me3-, and H3K9me3-enriched regions, corresponding to euchromatic, facultative heterochromatic, and constitutive heterochromatic domains, respectively. Finally, we determined cell type-specific replication timing profiles, integrating our in vivo datasets with published data using cultured cell lines. Our results reveal that GSCs display a distinct replication program compared to somatic lineages, that aligns with chromatin state differences. Collectively, our integrated transcriptomic, chromatin, and replication datasets provide a comprehensive framework for understanding genome regulation differences between these in vivo stem cell populations, demonstrating the power of multi-omics in uncovering cell type-specific regulatory features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db91bf6625baf7a6f5b436dd4b6c6b57e16b4fc" target='_blank'>
              Cell type- and chromosome-specific chromatin landscapes and DNA replication programs of Drosophila testicular stem cells
              </a>
            </td>
          <td>
            Jennifer A. Urban, Daniel Ringwalt, John M. Urban, Wingel Xue, Ryan J Gleason, Keji Zhao, Xin Chen
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02e806519d051257de949147c9dc431b4e0f1b1" target='_blank'>
              ER-tethering directs TREX1 penetration of a BAF-dependent barrier at micronuclei
              </a>
            </td>
          <td>
            Yanyang Chen, Eléonore Toufektchan, Xiaohan Luan, James H Hickling, R. Norman, P. Cifani, A. Kentsis, Wen Zhou, J. Maciejowski
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e2db1b98cbc1b9edbee352bacad54d120b81a4" target='_blank'>
              Severus detects somatic structural variation and complex rearrangements in cancer genomes using long-read sequencing.
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, B. Yoo, Sergey Aganezov, Anton Goretsky, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-04</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44c370bd7521d24afccebc5dd604dc8d4b6c7b" target='_blank'>
              Abstract 2617: Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing
              </a>
            </td>
          <td>
            Yanling Liu, B. Ju, Li Dong, Melanie Loyd, Yuan Feng, H. Mulder, P. Kolekar, Li Fan, Hanxia Li, S. Foy, Geoffrey Neale, Patrick A. Brown, William L Carroll, M. Loh, John Easton, Xiaotu Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841194928a5faee8a089ace06821025bbd9fd89a" target='_blank'>
              Abstract LB347: Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, B. Lawson, R. Hillman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96f5ae7db10ebf2238c71827c45c782f6d159f1b" target='_blank'>
              New insights into chromosomal regions 15p11.2 to 15q11.2 by studying submicroscopic variations using molecular cytogenetics.
              </a>
            </td>
          <td>
            Maria N Correia, Stefanie Kankel, I. Carreira, Joana B. Melo, Thomas Liehr
          </td>
          <td>2025-04-17</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d02fce4e58a33328241a1cb2df16862b6b79d6e" target='_blank'>
              Abstract 3901: Charting copy number gain histories from mutational divergence among multi-samples
              </a>
            </td>
          <td>
            Yunong Xia, Ruping Sun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b3cddb316bdce4c6828dd308d722612ae6e4f52" target='_blank'>
              Abstract 1070: siderGrimoire: a database of retroCNVs in human tumors and populations
              </a>
            </td>
          <td>
            T. L. Miller, Helena Beatriz da Conceição, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C-and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract TAQing technologies are based on the restriction enzyme-induced DNA double-strand break (DSB) formation in living cells, which results in large-scale genomic rearrangements and phenotypic alterations. Originally, the TAQing system requires heat treatments to activate restriction enzymes, which sometimes leads to cell toxicity or stress responses. Here, we developed a blue-light-controlled MagTAQing system, which induces DSBs exclusively upon blue-light exposure by assembling the split restriction enzymes via Magnet modules. Application of MagTAQing to mitotic budding yeast cells successfully triggered various genomic rearrangements upon blue-light exposure. Since this technology enables the conditional induction of genomic rearrangements in specific cells or tissues, we employed MagTAQing on meiotic yeast cells lacking the recombinase Spo11 to induce artificial DSBs. Consequently, Spo11-independent meiotic DSBs resulted in aneuploidies and nonallelic homologous recombinations between repetitive sequences such as ribosomal DNA and retrotransposons. These results suggest a pivotal role of Spo11-induced recombination in preventing chromosomal abnormality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c560ade8948ca0b121c00661a4ad61330184b0" target='_blank'>
              Light-controlled Spo11-less meiotic DNA breaks by MagTAQing lead to chromosomal aberrations
              </a>
            </td>
          <td>
            Hideyuki Yone, Yuri Kawashima, Hayato Hirai, Arisa H. Oda, Moritoshi Sato, Hiromitsu Kono, Kunihiro Ohta
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39136831a82d4694ee219ef38ce5763db7fa3a19" target='_blank'>
              ECCFP: a consecutive full pass based bioinformatic analysis for eccDNA identification using Nanopore sequencing data
              </a>
            </td>
          <td>
            Tangxuan Zhang, Wang Li, Qingsong Zen, Jun Zhang, Biyuan Miao, Mengting Li, Juan Luo, Tianliang Liu, Shifu Chen, Shaogui Wan
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520fb2116ae4feadf96591801b8f58200ddc5cec" target='_blank'>
              Oxidative Base Damage to Telomeres Sensitizes Cancer Cells to ATR Inhibition
              </a>
            </td>
          <td>
            Alex Garbouchian, Natalia Cestari Moreno, A. Dey, Patricia Opresko, Ryan Barnes
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce720a5b79eb9fb996bed883c2b0fde9438b7fa" target='_blank'>
              Genome evolution in parthenogenetic nematodes shaped by chromosome rearrangements and rare sex
              </a>
            </td>
          <td>
            George Chung, Lewis Stevens, Xiaoguang Dai, Scott Hickey, K. Kiontke, Priyesh Rughani, M. Kieninger, Joanna C. Collins, S. Juul, Fabio Piano, David H.A. Fitch, Mark Blaxter, Kristin C. Gunsalus
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The study examines Opisthorchis viverrini (OV)-related cholangiocarcinoma (CCA), a serious malignancy common in Southeast Asia. Through multi-regional whole-exome sequencing of 52 tumor samples and 13 adjacent tissues from 13 patients, significant intratumoral heterogeneity (ITH) and inter-patient heterogeneity are shown. Chronic liver fluke infection induces a distinct mutational landscape, with 48–90% of mutations concentrated in each region of the tumor. The average mutation burden is 95 non-synonymous mutations per area, exceeding previous CCA investigations. Critical driver mutations in TP53, SMAD4, and other genes underscore their significance in pathogenesis. Mutational markers elucidate mechanisms including spontaneous deamination and impaired DNA repair. Unique mutation patterns distinguish OV-associated CCA from other variants. Chromosomal instability in patient K110 signifies aggressive tumor behavior and unfavorable prognosis. Targetable mutations such as ERBB2 underscore the possibility for personalized therapeutics. These findings underscore the necessity for personalized strategies for treatment that target both trunk and branch mutations in endemic areas. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95142-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48c940bd80f998f283a697250c46b3404991cb4" target='_blank'>
              Whole exome sequencing of multi-regions reveals tumor heterogeneity in Opisthorchis viverrini-associated cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, A. Jusakul, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Bin Tean Teh, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In vivo genetic diversifiers have previously enabled efficient searches of genetic variant fitness landscapes for continuous directed evolution. However, existing genomic diversification modalities for mammalian genomic loci exclusively rely on deaminases to generate transition mutations within target loci, forfeiting access to most missense mutations. Here, we engineer CRISPR-guided error-prone DNA polymerases (EvolvR) to diversify all four nucleotides within genomic loci in mammalian cells. We demonstrate that EvolvR generates both transition and transversion mutations throughout a mutation window of at least 40 bp and implement EvolvR to evolve previously unreported drug-resistant MAP2K1 variants via substitutions not achievable with deaminases. Moreover, we discover that the nickase’s mismatch tolerance limits EvolvR’s mutation window and substitution biases in a gRNA-specific fashion. To compensate for gRNA-to-gRNA variability in mutagenesis, we maximize the number of gRNA target sequences by incorporating a PAM-flexible nickase into EvolvR. Finally, we find a strong correlation between predicted free energy changes underlying R-loop formation and EvolvR’s performance using a given gRNA. The EvolvR system diversifies all four nucleotides to enable the evolution of mammalian cells, while nuclease and gRNA-specific properties underlying nickase fidelity can be engineered to further enhance EvolvR’s mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4485500132f714892fcf79f15fa49a1b187bbe0" target='_blank'>
              Nickase fidelity drives EvolvR-mediated diversification in mammalian cells
              </a>
            </td>
          <td>
            Juan E Hurtado, Adam J. Schieferecke, Shakked O Halperin, John Guan, Dylan Aidlen, David V. Schaffer, J. Dueber
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Micronuclei originate from DNA damage generated by clastogenic and/or by aneugenic effects. Depending on the pattern of damage, they may have distinct genomic origin and composition. Sequences of the centromere, telomere and rDNA have been identified in plant micronuclei. However, other DNA sequences may also be present in the micronuclei, as well as their DNA contents may be different. Here, we investigate the DNA content, genomic composition and origin of micronuclei induced in Zea mays by methyl methanesulfonate (MMS). DNA contents showed a wide range of distribution, suggesting their diverse genomic origins and illustrating how much of the nuclear genome can be lost due to mutagen effects. Micronuclei diversity was also evidenced by in situ probing with different DNA sequences (5S and 18S rDNAs, 180-bp knob and Grande LTR-retrotransposon) and by 6-diamidino-2 phenylindole (DAPI) fluorochrome. Perhaps these sequences are hotspots for MMS damage, especially the Grande LTR-retrotransposon, 5S and 18S rDNAs, which are rich in guanine. In addition, probe pools were constructed from individual genomic DNA of two microdissected micronuclei. These probe pools hybridized on all Z. mays chromosomes. However, the centromere, knob and secondary constriction were hybridized by only one probe pool, evidencing the distinct genomic composition of the micronuclei. We illustrate the micronuclei genomic diversity as they originated from several different chromosomes following the MMS treatment, and demonstrate the extent of the genotoxic damage to the genome. We provide some insights into micronuclei structure and diversity, and show that they can be further explored in mutagenesis research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31c4f2a6572ceed6e9b3fda5f1cc0379119cd48" target='_blank'>
              DNA content, repeatome composition and origin of the Zea mays micronuclei
              </a>
            </td>
          <td>
            Nathália Vállery Tostes, Marcos Vitor Rosa Ferreira, Fernanda Aparecida Ferrari Soares, J. C. Silva, L. Bhering, W. Clarindo
          </td>
          <td>2025-04-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb48bc35dc30e8efcbd8268b117fd6382f30da8" target='_blank'>
              Abstract 2395: The role of exonized mobile elements in chimeric transcript emergence in glioblastoma
              </a>
            </td>
          <td>
            Rafael L. V. Mercuri, T. L. Miller, Filipe F. dos Santos, M. D. de Lima, Aline Rangel-Pozzo, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Tumor-associated macrophages (TAMs) play a crucial role in the tumor microenvironment, regulating immune escape and promoting cancer progression. Understanding the role of phagocytosis regulators in female-specific cancers is essential for developing effective therapeutic strategies. Methods We performed comprehensive analyses of public databases to evaluate the expression, somatic mutations, and copy number variations of phagocytosis regulators. DNA methylation patterns, biological pathways, survival outcomes, and drug sensitivity were assessed. Additionally, immune modulators, immune cell infiltration, and single-cell sequencing were used to explore alterations in phagocytosis and their cellular origins. The functional role of HMGB2 in tumor cell behavior was validated through in vitro assays. Results Phagocytosis regulators exhibited differential expression across various female-specific cancers, with key genes such as CD47 and FOXO1 playing significant roles in modulating tumor progression. High-frequency mutations were found in PTEN, ARID1A, and UBR4. Genes like COX5B and MS4A1 emerged as potential predictors of clinical outcomes and therapeutic response. HMGB2 knockdown significantly inhibited cancer cell proliferation, migration, and invasion in female-specific cancers. HMGB2 knockdown in macrophages led to a significant impairment in phagocytosis of breast, cervical, ovarian, and endometrial cancer cells. Furthermore, when HMGB2 knockdown was combined with Palbociclib treatment, a significant decrease in tumor cell proliferation was observed across multiple cancer models. Conclusion This study highlights the pivotal role of phagocytosis regulators, particularly HMGB2, in the progression of female-specific cancers. Targeting HMGB2 offers promising therapeutic opportunities, potentially enhancing precision oncology and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04106b265eee3d853f545357f94ba192188932f5" target='_blank'>
              Pan-cancer analysis of phagocytosis regulators in female-specific cancers: a focus on HMGB2
              </a>
            </td>
          <td>
            Xiaoqin Lu, Dan Ren, Panpan Zhao, Yanfang Li, Zhenhui Wang, Jingyan Zhang
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a9bc7f4f8a4cf97f32bbe5a9d88c7eccc61afe" target='_blank'>
              Abstract 2923: Targeting DNA repair mechanisms to overcome CIC::DUX4 undifferentiated sarcomas
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumor suppressor p53 plays a crucial role in maintaining genome integrity in response to exogenous or endogenous stresses. The dynamics of p53 activation are stimulus- and cell type- dependent and regulate cell fate. Acting as a transcription factor, p53 induces the expression of target genes involved in apoptosis, cell cycle arrest and DNA repair. However, transcription is not a deterministic process, but rather occurs in bursts of activity and promoters switch stochastically between ON and OFF states, resulting in substantial cell-to-cell variability. Here, we characterized how stimulus-dependent p53 dynamics are converted into specific gene regulation patterns by inducing diverse forms of DNA damage ranging from ionizing and UV radiation to clinically relevant chemotherapeutics. We employed single molecule fluorescence in-situ hybridization (smFISH) to quantify the activity of target gene promoters at the single-cell and single-molecule level. To analyse this comprehensive data set, we developed a new framework for determining parameters of stochastic gene expression by Bayesian inference. Using this combined theoretical and experimental approach, we revealed that features of promoter activity are differentially regulated depending on the target gene and the nature and extent of the DNA damage induced. Indeed, stimulus-specific stochastic gene expression is predominantly regulated by promoter activation and deactivation rates. Interestingly, we found that in many situations, transcriptional activity was uncoupled from the total amount of p53 and the fraction bound to DNA, highlighting that transcriptional regulation by p53 is a multi- dimensional process. Taken together, our study provides insights into p53-mediated transcriptional regulation as an example of a dynamic transcription factor that shapes the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8f8f817463be32115a6d7b633baecddd3a2d98" target='_blank'>
              Decoding stimulus-specific regulation of promoter activity of p53 target genes
              </a>
            </td>
          <td>
            Flavia Vigliotti, Nicolai Engelmann, Mark Sinzger-D’Angelo, Heinz Koeppl, Alexander Loewer
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA double strand break repair (DSBR) represents a fundamental process required to maintain genome stability and prevent the onset of disease. Whilst cell cycle phase and the chromatin context largely dictate which repair pathway is utilised to restore damaged DNA, it has been recently shown that nuclear actin filaments play a major role in clustering DNA breaks to facilitate DSBR by homologous recombination (HR). However, the mechanism with which nuclear actin and the different actin nucleating factors regulate HR is unclear. Interestingly, patients with biallelic mutations in the actin nucleating factor DIAPH1 exhibit a striking overlap of clinical features with the HR deficiency disorders, Nijmegen Breakage Syndrome (NBS) and Warsaw Breakage Syndrome (WABS). This suggests that DIAPH1 may play a role in regulating HR and that some of the clinical deficits associated with DIAPH1 mutations may be caused by an underlying DSBR defect. In keeping with this clinical similarity, we demonstrate that cells from DIAL (DIAPH1 Loss-of-function) Syndrome patients display an HR repair defect comparable to loss of NBS1. Moreover, we show that this DSBR defect is also observed in a subset of patients with Baraitser-Winter Cerebrofrontofacial (BWCFF) syndrome associated with mutations in ACTG1 (γ-actin) but not ACTB (β-actin). Lastly, we demonstrate that DIAPH1 and γ-actin promote HR-dependent repair by facilitating the relocalisation of the MRE11/RAD50/NBS1 complex to sites of DNA breaks to initiate end-resection. Taken together, these data provide a mechanistic explanation for the overlapping clinical symptoms exhibited by patients with DIAL syndrome, BWCFF syndrome and NBS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fdfe6f0c4067d89c1d4aa10073bf8c75525183b" target='_blank'>
              Inherited deficiency of DIAPH1 identifies a DNA double strand break repair pathway regulated by γ-actin
              </a>
            </td>
          <td>
            Beth L. Woodward, Sudipta Lahiri, Anoop S Chauhan, Marcos Rios Garcia, Lucy E Goodley, Thomas L Clarke, Mohinder Pal, A. Agathanggelou, Satpal S. Jhujh, Anil N Ganesh, Fay M Hollins, Valentina Galassi Deforie, R. Maroofian, S. Efthymiou, Andrea Meinhardt, C. Mathew, Michael A Simpson, Heather C Mefford, E. Faqeih, Sergio D. Rosenzweig, Stefano Volpi, G. Di Matteo, C. Cancrini, A. Scardamaglia, Fiona Shackley, E. G. Davies, Shahnaz Ibrahim, Peter D. Arkwright, Maha S. Zaki, Tatjana Stankovic, A. M. Taylor, A. Mazur, Nataliya Di Donato, H. Houlden, Eli Rothenberg, Grant S Stewart
          </td>
          <td>2025-05-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Chemo-/radioresistance of malignant tumors hampers cancer control and increases patient mortality. Efficient repair of damaged DNA is critical for the maintenance of genomic integrity and fidelity of genetic information. In reverse, increased DNA repair capability in cancer cells contributes to chemo-/radioresistance of malignant tumors. DNA double-strand break (DSB) is the most serious DNA damage and is also the principal molecular basis of radiotherapy. Upon DNA damage, the Ku80 is recruited and forms a critical DNA-PK complex at the DSB sites with Ku70 and the catalytic subunit (DNA-PKcs) to initiate DNA repair. How DNA-PK is assembled and activated is not fully understood. Based on the identification of radiation-reduced Ku80 K568 crotonylation through quantitative global lysine crotonylome analysis, we reveal that Ku80 K568 is crotonylated by p300-CBP-associated factor (PCAF). Upon DNA damage, the K568cr is decrotonylated by HDAC8 (Histone deacetylase 8). Decrotonylation of K568cr empties this site for the subsequent SUMOylation of Ku80 by CBX4. The conversion of Ku80 from K568 crotonylation to SUMOylation facilitates the assembly of DNA-PK complex and autophosphorylation of DNA-PKcs S2056, consequently activating the DSB repair. Moreover, mutation disrupting the post-translational modification (PTM) of Ku80 K568 site sensitizes cancer cells to radiotherapy in tumor-bearing nude mice models. This study elucidates the conversion model between two different forms of PTMs in the regulation of DNA-PK complex assembly and DSB repair, highlighting this model’s potential in controlling chemo-/radioresistance of malignant tumors, as well as expands the atlas of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e28a599caae10368a0a56703d29a0e46b8a2e" target='_blank'>
              Conversion of Ku80 K568 crotonylation to SUMOylation facilitates DNA non-homologous end joining and cancer radioresistance
              </a>
            </td>
          <td>
            Hongling Zhao, Shan-shan Gao, Yang Han, Dafei Xie, Lihui Xuan, Xin Huang, Jinhua Luo, Q. Ran, Gang Li, Hejiang Guo, Weixiang Hu, Jin Jia, Xiaochang Liu, Yuhao Liu, Jinpeng Tan, Chenjun Bai, Yongqing Gu, Teng Ma, Zhongjun Li, Hua Guan, Ruixue Huang, Ping-Kun Zhou
          </td>
          <td>2025-04-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4868141c39dc560c925882ee816da6d52c4deb" target='_blank'>
              Uev1A counteracts oncogenic Ras stimuli in both polyploid and diploid cells
              </a>
            </td>
          <td>
            Qi Zhang, Yunfeng Wang, Xueli Fu, Ziguang Wang, Yang Zhang, Lizhong Yan, Yuejia Wang, Muhan Yang, Dongze Song, Ruixing Zhang, Hongru Zhang, Shian Wu, Shaowei Zhao
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Micronuclei exhibit defective proteomes rendering their chromatin vulnerable to fragmentation. This fragmentation process, known as chromothripsis, promotes tumorigenesis by catalysing the activation of oncogenes and the silencing of tumor suppressors. With this role in mind, micronuclei serve as promising targets for therapeutic intervention. This review will explore recent discoveries regarding how micronuclei form, their function in catalysing chromothripsis and how chromothripsis provides a selective advantage for cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ac4a5ea94375865c4d842a9f93ca345cb31b4e0" target='_blank'>
              Cancer cells' chamber of secrets: the link between micronuclei, chromothripsis and malignancy.
              </a>
            </td>
          <td>
            Truman McNeil, Sweta Sikder, Yamini Dalal
          </td>
          <td>2025-05-01</td>
          <td>Open biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct “fragmentomic” characteristics that are related with their cellular origins. Despite the wealth of low-pass whole genome sequencing data from recent cancer liquid biopsy studies, somatic variants within this data have often been overlooked without detailed characterization. Here, we reported that in low-pass sequencing data, somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, our investigation into cfDNA fragmentomics showed that even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like traits, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilized artificial intelligence to integrate genomic, fragmentomic, and epigenomic features, developing diagnostic models named FreeSV (Fragmentomic and epigenomic examinations of Somatic Variants in cfDNA) and FreeSV-m. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features were also included, the AUCs of FreeSV-m model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64ed6f9ef46f8ac541b4cf9a70df9227b7d0f90" target='_blank'>
              Cancer-like fragmentomic characteristics of somatic variants in cell-free DNA
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Wanqiu Wang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract SRSF2 (serine/arginine-rich splicing factor 2) is a critical regulator of pre-messenger RNA splicing, which also plays noncanonical functions in transcription initiation and elongation. Although elevated levels of SRSF2 are associated with advanced stages of lung adenocarcinoma (LUAD), the mechanisms connecting SRSF2 to lung tumor progression remain unknown. We show that SRSF2 overexpression increases global transcription and replicative stress in LUAD cells, which correlates with the production of DNA damage, notably double-strand breaks (DSBs), likely resulting from conflicts between transcription and replication. Moreover, SRSF2 regulates DNA repair pathways by promoting homologous recombination and inhibiting nonhomologous end joining. Mechanistically, SRSF2 interacts with and enhances MRE11 (meiotic recombination 11) recruitment to chromatin, while downregulating 53BP1 messenger RNA and protein levels. Both events are likely contributing to SRSF2-mediated DNA repair process rerouting. Lastly, we show that SRSF2 and MRE11 expression is commonly elevated in LUAD and predicts poor outcome of patients. Altogether, our results identify a mechanism by which SRSF2 overexpression promotes lung cancer progression through a fine control of both DSB production and repair. Finally, we show that SRSF2 knockdown impairs late repair of ionizing radiation-induced DSBs, suggesting a more global function of SRSF2 in DSB repair by homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae11d3e666c0a06576a20bb8706265500fce3a5" target='_blank'>
              SRSF2 overexpression induces transcription-/replication-dependent DNA double-strand breaks and interferes with DNA repair pathways to promote lung tumor progression
              </a>
            </td>
          <td>
            Manal Khalife, Tao Jia, Pierre Caron, A. Shreim, Aurelie Genoux, Agnese Cristini, Amélie Pucciarelli, Marie Leverve, Nina Lepeltier, Néstor García-Rodríguez, Fabien Dalonneau, Shaliny Ramachandran, Lara Fernandez Martinez, Guillaume Marcion, Nicolas Lemaitre, Elisabeth Brambilla, Carmen Garrido, Ester M. Hammond, Pablo Huertas, S. Gazzeri, O. Sordet, B. Eymin
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncoproteins are strong drivers of pediatric acute myeloid leukemia (AML) with poor prognosis. Here we show that NUP98 fusion-expressing AML harbors an epigenetic signature that is characterized by increased accessibility of hematopoietic stem cell genes and enrichment of activating histone marks. We employ an AML model for ligand-induced degradation of the NUP98::KDM5A fusion oncoprotein to identify epigenetic programs and transcriptional targets that are directly regulated by NUP98::KDM5A through CUT&Tag and nascent RNA-seq. Orthogonal genome-wide CRISPR/Cas9 screening identifies 12 direct NUP98::KDM5A target genes, which are essential for AML cell growth. Among these, we validate cyclin-dependent kinase 12 (CDK12) as a druggable vulnerability in NUP98::KDM5A-expressing AML. In line with its role in the transcription of DNA damage repair genes, small-molecule-mediated CDK12 inactivation causes increased DNA damage, leading to AML cell death. Altogether, we show that NUP98::KDM5A directly regulates a core set of essential target genes and reveal CDK12 as an actionable vulnerability in AML with oncogenic NUP98 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac2ff38eb74ff54c2c90e5931c96b450175dda5" target='_blank'>
              Transcriptional and epigenetic rewiring by the NUP98::KDM5A fusion oncoprotein directly activates CDK12
              </a>
            </td>
          <td>
            Selina Troester, Thomas Eder, Nadja Wukowits, M. Piontek, P. Fernández-Pernas, Johannes Schmoellerl, B. Haladik, G. Manhart, M. Allram, Margarita Maurer-Granofszky, Nastassja Scheidegger, K. Nebral, Giulio Superti-Furga, Roland Meisel, B. Bornhauser, P. Valent, Michael N Dworzak, Johannes Zuber, Kaan Boztug, Florian Grebien
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a leading cause of cancer-related mortality among men worldwide. Despite progress in the understanding of tumor biology, the prognosis for advanced prostate cancer remains poor, necessitating the identification of novel diagnostic, prognostic, and therapeutic biomarkers. Methuosis, a recently identified form of programmed cell death (PCD), is characterized by cytoplasmic vacuole accumulation and subsequent cell rupture, distinct from classical apoptosis and necrosis. The key regulatory gene in Methuosis, ARF6 (ADP-ribosylation factor 6), has emerged as a potential marker for cancer diagnosis and treatment. However, its role in prostate cancer and other malignancies remains insufficiently understood. Methods In this study, we performed a comprehensive pan-cancer multi-omics analysis to investigate the role of ARF6 in Methuosis across multiple cancer types, with a specific focus on PCa as the primary context. Using data from public databases, including RNA sequencing, gene expression profiling, and clinical outcomes, we assessed the association between ARF6 expression and patient prognosis in PCa within this broader pan-cancer framework. Additionally, we employed functional enrichment analyses and survival analysis to explore the potential of ARF6 as a diagnostic and prognostic marker for prostate cancer. Immunotherapy-related gene expression signatures were also evaluated to determine the therapeutic relevance of ARF6. Results ARF6 was significantly overexpressed in PCa tissues compared to normal tissues and was associated with poor prognosis (p < 0.05), particularly in advanced and metastatic stages. Receiver operating characteristic (ROC) analysis revealed a diagnostic AUC of 0.792 for ARF6. Functional analyses indicated that ARF6 regulates pathways critical to cell migration, invasion, and drug resistance. Moreover, ARF6 expression showed a strong negative correlation with immune checkpoint markers, such as PD-L1 (r = − 0.74), suggesting its potential as an immunotherapy target. These findings underscore ARF6’s pivotal role in Methuosis and its promise as a biomarker in PCa. Conclusion ARF6 is a key regulator of Methuosis in prostate cancer, contributing to tumor progression, metastasis, and resistance to treatment. Our findings support the potential of ARF6 as a diagnostic, prognostic, and immunotherapeutic target in prostate cancer. Further experimental validation is needed to confirm these observations and to explore the therapeutic implications of targeting ARF6 in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c475c0dde283a17762878be80bdc39f46da628" target='_blank'>
              Methuosis key gene ARF6 as a diagnostic, prognostic and immunotherapeutic marker for prostate cancer: based on a comprehensive pan-cancer multi-omics analysis
              </a>
            </td>
          <td>
            Yongjin Yang, Liangliang Qing, Chengyu You, Qingchao Li, Wenbo Xu, Zhilong Dong
          </td>
          <td>2025-05-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5717acd2d0393973aea7e3766b9be2a2a3a6876f" target='_blank'>
              One for All and All for One: New Insights into Enhancers Driving MYC Dysregulation.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2025-04-16</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c41f6c6066f9f1caf2b6797a1f6cfc0b6b61d7" target='_blank'>
              Abstract 1958: Chromatin accessibility landscapes of colorectal cancer development and genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, J. Medina, Stephen B. Baylin, Kasper D. Hansen, V. Adleff, R. Fijneman, Michael A. Beer, Robert B Scharpf, G. Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d56727982104a6a78307d0f1220221468bf7fd" target='_blank'>
              Abstract 2740: Mechanisms of phase-separation driven chromatin looping in acute myeloid leukemia
              </a>
            </td>
          <td>
            Tooba Rashid, Z. Drum, Doris Cruz Alonso, Gelila Petros, Yoseli Quiroga, Douglas Phanstiel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa24230eaf15e64077b2640e42b580f6dcef4855" target='_blank'>
              Abstract 2838: Modeling putative oncogenic variants of human topoisomerase II by genome editing of a near haploid cell line
              </a>
            </td>
          <td>
            K. C. Nitiss, Brigid Conroy, M. N. McCoy, J. Nitiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage.
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Herpesviruses are a group of double-stranded DNA viruses known to develop versatile viral strategies to escape host immune surveillance for promoting their replication and propagation. This is illustrated by Kaposi’s sarcoma-associated herpesvirus (KSHV), an oncogenic gamma-herpesvirus that overcomes host immune suppression by multiple mechanisms. In this study, we reported that KSHV dysregulates 5-methylcytosine (m5C) modification and mRNA stability of host antiviral factors to benefit its lytic replication. KSHV lytic reactivation or de novo challenge led to downregulation of m5C RNA methyltransferases, NSUN2 and NSUN1 (NSUN2/1), while NSUN2/1 depletion promoted KSHV lytic replication. Such KSHV-mediated downregulation of NSUN2/1 is via suppression of the transcriptional factor c-Myc. We further performed the RNA bisulfite sequencing (RNA-BS-seq) to identify KSHV-dependent m5C modification of host mRNAs. KSHV lytic reactivation led to the significant reduction of m5C methylation and mRNA stability of TRIM25, a key activator of the RIG-I pathway, while TRIM25 depletion indeed promoted KSHV lytic replication. These host-virus interaction events were also observed in the infection of another oncogenic gamma-herpesvirus Epstein-Barr virus (EBV). Overall, our results highlighted a new strategy for human gamma-herpesviruses to counteract host antiviral factors and promote their lytic replication by manipulating host m5C RNA methylation. Significance Statement Our study has identified a novel viral mechanism of human gamma-herpesviruses to manipulate host RNA methylation machineries to subvert immune defenses and enhance viral lytic replication. In particular, our new data showed that KSHV/EBV downregulate the key 5-methylcytosine (m5C) RNA writers NSUN2/1 via c-Myc, and thus decrease m5C modification and stability of TRIM25 mRNA. As TRIM25 is a key E3 ubiquitin ligase in RIG-I signal transduction, its inhibition disrupts RIG-I mediated antiviral sensing and thus favors viral lytic replication. As human gamma-herpesviruses are critical pathogens that highly associate with multiple human diseases especially certain tumors, such studies are significant to shed light in improving the fundamental understanding of virus-host interactions and identifying new host targets for future translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb3410ec4df12657e97cb05060a2d93de53558c7" target='_blank'>
              m5C RNA Methylation Is Dysregulated by Oncogenic Herpesviruses via c-Myc Signaling to Counteract Host Antiviral Factors
              </a>
            </td>
          <td>
            Zhenyu Wu, Dawei Zhou, Zhixian He, Guillaume N. Fiches, Youngmin Park, Jinshan He, Jianwen Chen, K.A.S.N Shanaka, Thurbu Tshering Lepcha, Prashant Desai, Jian Zhu, Netty G Santoso
          </td>
          <td>2025-05-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Homology, or relationship among characters by common descent, has been notoriously difficult to assess for many morphological features, and cell types in particular. The ontogenetic origin of morphological traits means that the only physically inherited information is encoded in the genomes. However, the complexity of the underlying gene regulatory network and often miniscule changes that can impact gene expression, make it practically impossible to postulate a clear demarcation line for what molecular signature should "define" a homologous cell type between two deeply branching animals. In this Hypothesis article, we propose the use of the recently characterized irreversible genomic states, that occur after chromosomal and sub-chromosomal mixing of genes and regulatory elements, to dissect regulatory signatures of each cell type into irreversible and reversible configurations. While many of such states will be non-functional, some may permanently impact gene expression in a given cell type. Our proposal is that such evolutionarily irreversible, and thus synapomorphic, functional genomic states can constitute a criterion for the timing of the origin of deep evolutionary cell type homologies. Our proposal thus aims to close the gap between the clearly defined homology of the individual genomic characters and their genomic states to the homology at the phenotypic level through the identification of the underlying evolutionarily irreversible and regulatory linked states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1c7265ffd0750ec9e349f309b4f00f20641d114" target='_blank'>
              The application of irreversible genomic states to define and trace ancient cell type homologies
              </a>
            </td>
          <td>
            Oleg Simakov, Günter P. Wagner
          </td>
          <td>2025-05-03</td>
          <td>EvoDevo</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The rnhAB mutants of Escherichia coli lacking both RNase H enzymes are unexpectedly UV-sensitive, being unable to restore normal levels of post-UV replication. Examining patterns of chromosomal replication in the rnhAB mutants after UV could identify the problem sites. We show that normal rnhA (B) mutant replication initiates at three distinct oriK areas in the origin macrodomain, none of them coinciding with oriC proper, the dominant origin being some 400 kb away. Interestingly, initiation after UV switches to the DnaA-dependent oriK closest to oriC and continues from there until the growth replication pattern is restored, like in the rnhA single mutants. However, in the rnhAB double mutant, post-UV forks initiated at the new origin have difficulty reaching the terminus, with the major stalling sites at the rrn operons. In the rnhAB recBC mutants, additionally deficient in linear DNA degradation/repair, post-UV replication forks cannot traverse the origin-distal ribosomal RNA operons, rrnG and rrnH, showing that restoration of disintegrated replication forks is essential for replication in the rnhAB mutant. In contrast, the rnhAB rpoB* mutant, in which transcription complexes are unstable, is UV-resistant and resumes normal replication even faster than WT cells, indicating that the rnhAB mutants suffer from UV-induced replication–transcription conflicts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc49e043c7c31c72b8caa41cfc59c42930701fb5" target='_blank'>
              UV induces codirectional replication–transcription conflicts and an alternative DnaA-dependent replication origin in the rnhAB mutants of Escherichiacoli
              </a>
            </td>
          <td>
            E. Kouzminova, Glen E. Cronan, A. Kuzminov
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is crucial for maintaining genomic stability by repairing DNA damage. Despite its importance, HRR's role in cancer progression is not fully elucidated. Here, this work shows that nuclear-localized branched-chain α-ketoacid dehydrogenase kinase (BCKDK) acts as a modulator of HRR, promoting cell resistance against DNA damage-inducing therapy in breast cancer. Mechanistically, this work demonstrates that BCKDK is localized in the nucleus and phosphorylates RNF8 at Ser157, preventing the ubiquitin-mediated degradation of RAD51, thereby facilitating HRR-mediated DNA repair under replication stress. Notably, aberrant expression of the BCKDK/p-RNF8/RAD51 axis correlates with breast cancer progression and poor patient survival. Furthermore, this work identifies a small molecule inhibitor of BCKDK, GSK180736A, that disrupts its HRR function and exhibits strong tumor suppression when combined with DNA damage-inducing drugs. Collectively, this study reveals a new role of BCKDK in regulating HRR, independent of its metabolic function, presenting it as a potential therapeutic target and predictive biomarker in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac4a9c9be6494a377793b137fd2fa59c3a2efc2e" target='_blank'>
              Nuclear-Localized BCKDK Facilitates Homologous Recombination Repair to Support Breast Cancer Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Haiying Liu, Jiaqian Feng, Tingting Pan, Pinggen Zhang, Ling Ye, Zetan Jiang, Zilong Zhou, Qiankun Mao, Jian Li, Xinyi Yang, P. Gao, De Huang, Huafeng Zhang
          </td>
          <td>2025-04-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background
 Spatial and temporal control of DNA damage response pathways after DNA damage is crucial for maintenance of genomic stability. Ataxia telangiectasia mutated (ATM) protein plays a central role in DNA damage response pathways. The chain of events following induction of DNA damage that results in full activation of ATM is still evolving. Here we set out to explore the role of CREB-binding protein (CBP), a histone acetyltransferase (HAT), in DNA damage response, particularly in the ATM activation pathway. Results In response to DNA damage, CBP is stabilized and is recruited at sites of DNA double-strand breaks where it acetylates ATM and promotes its kinase activity. Cells deficient in CBP display an impairment in DNA double-strand break repair and high sensitivity to chemo- and radiotherapy. Importantly, re-expressing CBP’s HAT domain in CBP-deficient cells restores the DNA repair capability, demonstrating the essential role of CBP’s HAT domain in repairing DNA double-strand breaks. Conclusions Together, our findings shed the light on CBP as a key participant in the ATM activation pathway and in the subsequent repair of DNA double-strand breaks, which may serve as a potential target to modulate the cellular response to DNA damaging agents in cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03528-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a883f9f1e4c2e6460111ee2a491dce39c437792" target='_blank'>
              The histone acetyltransferase CBP participates in regulating the DNA damage response through ATM after double-strand breaks
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, Samrein B M Ahmed, Iman M Talaat, L. Lozon, S. Mouffak, Timo Gemoll, Wael Y Mansour, Raafat El-Awady
          </td>
          <td>2025-04-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54ccd0aa3ae124063240ed82d6af46f8d425d72" target='_blank'>
              Aneuploidy generates enhanced nucleotide dependency and sensitivity to metabolic perturbation.
              </a>
            </td>
          <td>
            R. Magesh, Arshia N Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, Jessica B Spinelli, S. Elledge, E. Watson
          </td>
          <td>2025-05-05</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Pluripotent stem cells (PSC) are known to recurrently acquire genetic aberrations including chromosomal aneuploidy during long-term culture of which the consequences remain largely unknown. A recent study reported that xenografting of aneuploid mouse embryonic stem cells (mESC) gives rise to teratomas containing undifferentiated elements with metastatic capacity. This contrasts with benign teratomas composed of fully differentiated tissues from the three embryonic germ layers obtained by xenografting diploid mESCs. The highlighted study thus proposes that general aneuploidy may drive metastasis of PSC-derived teratomas, thus directly correlating it with malignant potential of PSCs. Main The aneuploid mESC lines used in the aforementioned study showed trisomies of chromosomes 6, 8, 11, 15 or combinations thereof. Interestingly, several of these trisomies, specifically in chromosomes 6, 8, and 11, constitute well-known chromosomal aberrations recurrently acquired in mESCs upon (long-term) in vitro culture. This is suggested to be driven by numerous (proto)oncogenes in these regions. Notably, recurrent chromosomal aberrations are also observed in human PSCs (hPSCs), mostly in chromosome 1, 12, 17, 20 and X, and have similarly been associated with the acquisition of a malignant phenotype in vitro, posing a risk to their potential clinical application. There is synteny between the chromosomes recurrently affected by such aberrations in mouse and human PSCs, namely mouse chromosome 6 and human chromosome 12 as well as mouse and human chromosome 1, suggesting a chromosome-specific phenomenon that may transcend the species barrier. Additionally, in the field of cancer, malignant (but not benign) human germ cell tumors (hGCTs), pathologically similar to the tumors derived from the aforementioned aneuploid mouse PSCs, are characterized by supernumerical copies of chromosome 12, highlighting the additional clinical relevance of these findings and the aneuploid mESC models for GCT research and treatment. Conclusions We suggest that chromosome-specific trisomies, rather than general aneuploidy, might drive teratoma metastasis upon mESC xenografting in vivo. We also observed indications of synteny between the recurrent chromosomal aberrations among human and mouse PSCs, suggesting potentially common intraspecies selection mechanisms. We finally reiterate the similarities observed between the PSC and GCT field related to chromosomal aberrations and malignancy, highlighting the relevance of these experimental models in both research fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9961bd4e5533b7648a3a993149a92ce0795a5ee" target='_blank'>
              Chromosome-specific aberrations, rather than general aneuploidy, may drive mouse embryonic stem cell-derived teratoma metastasis
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, Sanne Hillenius, T. F. Eleveld, Daniela C F Salvatori, Leendert H J Looijenga
          </td>
          <td>2025-04-15</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d11e895cea0cf1e967c3e4adda2d90a1183e147" target='_blank'>
              Abstract 215: A novel framework for deciphering 3D genome structure in colorectal cancer
              </a>
            </td>
          <td>
            Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Perturbed DNA replication can lead to incompletely replicated DNA when cells enter mitosis and can interfere with chromosome segregation. Cells therefore require mechanisms to resolve these lesions during mitosis. The CIP2A-TOPBP1 complex is described to function as a molecular tether that connects fragmented DNA molecules. However, whether CIP2A also functions in processing of incompletely replicated DNA remained unclear. We show that CIP2A-TOPBP1 forms large filamentous structures at sites of incomplete DNA replication during mitosis, which recruit the SMX tri-nuclease complex members SLX4, MUS81 and ERCC1/XPF. These structures form in proximity to sites of mitotic DNA synthesis, although CIP2A is not required for mitotic DNA synthesis. In addition to its globular and coiled-coil domain, the unstructured C-terminal domain of CIP2A is essential for CIP2A-TOPBP1 filamentous structure formation and recruitment of the SMX complex. BRCA1-/- and BRCA2-/- cells have increased mitotic DNA lesions that recruit CIP2A and SLX4. We show that the C-terminal part of CIP2A is required for survival of BRCA2-/- cells. Moreover, SLX4 is crucial for genome stability in BRCA2-/- cells. Combined, we demonstrate that CIP2A-TOPBP1 recruits the SMX complex during mitosis, which is required to resolve mitotic DNA lesions, allows faithful chromosome segregation and maintain viability of BRCA2-/- cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b232ddba76a37f75a5a20189f87cbdb729879f8" target='_blank'>
              CIP2A is required for mitotic recruitment of the SLX1/XPF/MUS81 tri-nuclease complex to replication stress-induced DNA lesions to maintain genome integrity
              </a>
            </td>
          <td>
            Lauren de Haan, Sietse J Dijt, Alejandro García López, Dan Ruan, Femke J. Bakker, M. Everts, Harry M Warner, Frank N. Mol, H. R. de Boer, Pim J. Huis in ’t Veld, Bert van de Kooij, R. Vlijm, M. V. van Vugt
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Addiction to oncogene-rewired transcriptional networks is a therapeutic vulnerability in cancer cells, underscoring a need to better understand mechanisms that relay oncogene signals to the transcriptional machinery. Here, using human and mouse T cell acute lymphoblastic leukemia (T-ALL) models, we identify an essential requirement for the endosomal sorting complex required for transport protein CHMP5 in T-ALL epigenetic and transcriptional programming. CHMP5 is highly expressed in T-ALL cells where it mediates recruitment of the coactivator BRD4 and the histone acetyl transferase p300 to enhancers and super-enhancers that enable transcription of T-ALL genes. Consequently, CHMP5 depletion causes severe downregulation of critical T-ALL genes, mitigates chemoresistance and impairs T-ALL initiation by oncogenic NOTCH1 in vivo. Altogether, our findings uncover a non-oncogene dependency on CHMP5 that enables T-ALL initiation and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768bb462ad3f5f7bfdc0d91a1f7eb42525d3db0" target='_blank'>
              The ESCRT protein CHMP5 promotes T cell leukemia by enabling BRD4-p300-dependent transcription
              </a>
            </td>
          <td>
            Katharine Umphred-Wilson, Shashikala Ratnayake, Qianzi Tang, Rui Wang, S. Chaudhary, Devaiah N Ballachanda, Josephine Trichka, Jan Wisniewski, Lan Zhou, Qing-rong Chen, D. Meerzaman, Dinah S. Singer, S. Adoro
          </td>
          <td>2025-05-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genotoxic therapies such as ionizing radiation eliminate cancer cells by inducing extensive DNA damage but often cause normal tissue toxicity, including cutaneous injury. Extrachromosomal circular DNA (eccDNA) refers to circular DNA fragments outside the chromosomal context, with their formation and persistence linked to DNA damage repair and genomic instability. Despite growing recognition of eccDNA in oncogenesis, its role under genotoxic stress in normal tissues remains poorly understood. Here, eccDNA is profiled in irradiated rat skin using Circle-seq, identifying alterations in eccDNA number and composition. Specifically, radiation induced circle17:44148731-48208624, in which vacuolar protein sorting 41 homolog (VPS41) is the sole radiation-induced amplification gene by semiquantitative PCR and gel electrophoresis. The findings show that eccDNA or VPS41 overexpression reduces radiation-induced skin injury (RISI) in vitro and in vivo. Proteomic and interaction analyses identified metastasis suppressor kangai-1 (KAI1) as a VPS41-interacting partner. Notably, VPS41 overexpression promotes KAI1 lysosomal degradation, protecting against radiation-induced apoptotic cell death. Peptide array analysis pinpoints the VPS41-KAI1 interaction through the K263 residue, consistent with AlphaFold prediction. The findings uncover a novel mechanism in which radiation-induced eccDNA, specifically VPS41, mitigates skin injury by modulating KAI1 degradation. This study highlights the role of eccDNA in cellular defense, providing strategies to enhance tissue resilience to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821c77d7a2a02d4160cf107b29b6628f9d6cb1ed" target='_blank'>
              EccDNA-Driven VPS41 Amplification Alleviates Genotoxic Stress via Lysosomal KAI1 Degradation.
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Jinchen He, Bin Song, Hao Bai, Fengdi Jiang, Yining Zhang, Qian Li, Tao Yan, Wenling Tu, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability.
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are widely existing stable DNA secondary structures in mammalian cells. A long-standing hypothesis is that timely resolution of G4s is needed for efficient and faithful DNA replication. In vitro, G4s may be unwound by helicases or alternatively resolved via DNA2 nuclease mediated G4 cleavage. However, little is known about the biological significance and regulatory mechanism of the DNA2-mediated G4 removal pathway. Here, we report that DNA2 deficiency or its chemical inhibition leads to a significant accumulation of G4s and stalled replication forks at telomeres, which is demonstrated by a high-resolution technology: Single molecular analysis of replicating DNA (SMARD). We further identify that the DNA repair complex MutSα (MSH2-MSH6) binds G4s and stimulates G4 resolution via DNA2-mediated G4 excision. MSH2 deficiency, like DNA2 deficiency or inhibition, causes G4 accumulation and defective telomere replication. Meanwhile, G4-stabilizing environmental compounds block G4 unwinding by helicases but not G4 cleavage by DNA2. Consequently, G4 stabilizers impair telomere replication and cause telomere instabilities, especially in cells deficient in DNA2 or MSH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica D Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d82d634c4ca446f74e247158a8446b3ecb227ab" target='_blank'>
              MicroFinder: Conserved gene-set mapping and assembly ordering for manual curation of bird microchromosomes
              </a>
            </td>
          <td>
            TC Mathers, M. Paulini, Cibele G. Sotero-Caio, Jonathan M. D. Wood
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29460fc2cc42604173072239d7b085ed803a493" target='_blank'>
              Abstract 4305: Targeting the exonuclease domain of POLE to induce synthetic lethality in highly aneuploidic tumor cells
              </a>
            </td>
          <td>
            Meng He, Eun Jung Kim, Yangmeng Wang, A. Chibly, Kristel M. Dorighi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87641a322ac1dbfc7184a0c61b91e897e328415f" target='_blank'>
              Human papillomavirus integration induces oncogenic host gene fusions in oropharyngeal cancers.
              </a>
            </td>
          <td>
            N. Khan, Keiko Akagi, Shiming Jiang, J. D. Dunn, Bo Jiang, Weihong Xiao, Madison P O'Hara, Li Shen, Qi Wang, Vakul Mohanty, Jing Wang, Sara Goodwin, Jamie L Hutchins, Kevin R. Coombes, Jagannadha K Sastry, D. Symer, M. Gillison
          </td>
          <td>2025-05-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair.
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Understanding lineage differentiation is an ongoing challenge in the field of stem cell research. We acquired eight gain-of-function (GOF) oncogenic mutations associated with DNA methylation alteration from the TCGA datasets. After introducing each oncogene into wild-type stem cells separately, we found that the mutant stem cells are difficult to reprogram to iPSCs and terminally differentiate to induced neuronal cells, and retain proliferation capacity in the process of differentiation. This findings suggests that oncogenic mutations impair the lineage differentiation. After analyzing the resulting DNA methylation profiles, we identified many novel lineage differentiation genes in mutant cells showing DNA methylation alteration compared with wild-type cells, which is consistent with their methylation states in low-grade gliomas (LGGs) compared with control brain. Collectively, our results indicate that oncogenic mutations in stem cells are associated with DNA methylation changes for genes related to multipotency maintenance, differentiation resistance, and tumorigenesis, therefore, displaying impairment in lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2672c1b1d54b288de2f8ea63a8be97f128f351b6" target='_blank'>
              Oncogenic mutations disrupt lineage differentiation via epigenetic perturbations
              </a>
            </td>
          <td>
            Lingjun Meng, Alice Yu, Huanwei Huang, Jodie Meng, Yongjin Yoo, Katie Schaukowitch, Justyna A. Janas, David A. Knowles
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            V. Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea8da16c37659abb745c723fb795dcd02f7d43b2" target='_blank'>
              Quantitative AI-based DNA fiber workflow to study replication stress
              </a>
            </td>
          <td>
            Paolo Fagherazzi, Timo Diekmann, Alessandra Ardizzoia, Zuzana Machacova, Simran Negi, A. K. Yadav, Pavel Moudry, Vincenzo Costanzo, Hana Polasek-Sedlackova
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05f154cb832f6f7744c5bac28235208f5d152b7" target='_blank'>
              Mitochondrial DNA copy numbers in gastric cancer tissues: a possible biomarker for estimating cancer progression.
              </a>
            </td>
          <td>
            Hisanori Fukunaga, Mayuko Fukunaga
          </td>
          <td>2025-04-23</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282bf0c515738689c3395ab25c9b857f16da5ffb" target='_blank'>
              Interplay between BET Proteins and the Pre-Replication Complex Unveils Therapeutic Vulnerabilities in Cancer
              </a>
            </td>
          <td>
            Kai Doberstein, Vivica Messner, Sebastian Berlit, Benjamin Tuschy, Marc Sütterlin, F. Marmé
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb63902e2d7d2fc71c9d70de05dacb96d781fb72" target='_blank'>
              Antagonistic Regulation of LINE-1/Alu Elements and Their Repressor APOBEC3B in Cellular Senescence
              </a>
            </td>
          <td>
            Daksha Munot, Gregoire Najjar, Manvendra Singh, Cagatay Günes, D. Sauter, Ankit Arora
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aefc77b0bd9501cf8a67b9339aecd7d49dfb34bb" target='_blank'>
              The novel RNA polymerase I transcription inhibitor PMR-116 exploits a critical therapeutic vulnerability in a broad-spectrum of high MYC malignancies
              </a>
            </td>
          <td>
            Rita Ferreira, K. Hannan, Konstantin Panov, Thejaani Udumanne, Amee J. George, Adria Closa, Zaka Yuen, E. Eyras, Perlita Poh, Kylee H. Maclachlan, Mitchell Dwyer, Vanessa Barn, Jinshu He, Eric P. Kusnadi, R. Rebello, Alisee Huglo, S. Hedwards, Mitchell Lawrence, Gail Risbridger, Renea Taylor, A. Clark, Geoffrey Farrell, S. Pok, Narci Teoh, Madushani Amarasini, Philip L. Norcott, M. Banwell, Yulia Manenkova, M. Haddach, D. Drygin, Luc Furic, Ross D. Hannan, Nadine Hein
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8baa6a9cc6acf1a8fc2f2802c0c1373dc7878226" target='_blank'>
              Steroid-Modulated Transcription Synergistically Forms DNA Double-Strand Breaks With Topoisomerase II Inhibitor.
              </a>
            </td>
          <td>
            Ying Zhao, Tetsuro Hisayoshi, Doudou Zhang, Saaya Suzuki, Takashi Watanabe, Atsuo Kobayashi, Qianqian Guo, Y. Momozawa, Takashi Shimokawa, Shunsuke Kato, Yoshio Miki, Shigeaki Sunada
          </td>
          <td>2025-04-15</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95cf76b81497b2613567d4d58faa4859bccba0a9" target='_blank'>
              Attenuation of mosaic aneuploidy and erroneous first mitotic division of human embryos
              </a>
            </td>
          <td>
            Cerys E. Currie, A. Byrska, D. Taylor, M. Erent, Daniela N. Bakalova, Xuhui Sun, Constandina Koki, Nigel J. Burroughs, Richard. A. Anderson, A. Marston, Geraldine M. Hartshorne, A. McAinsh
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19d291488e785e3492d2b652ca0281a0fb7489d" target='_blank'>
              eLaRodON: identification of large genomic rearrangements in Oxford Nanopore sequencing data
              </a>
            </td>
          <td>
            Regina Mikheeva, M. Koryukov, Pavel Leonov, Victoriya Mukhametdinova, Victoriya Borobova, Petr Laktionov, Alexander Tarkhov, M. Filipenko, Andrey Kechin
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncogenes are known to promote aggressive pediatric leukemia by disrupting chromatin structure and modulating the expression of homeobox (HOX) genes, yet the precise molecular events are unclear. In this issue of the JCI, K. Hamamoto et al. explore the mechanistic underpinnings of NUP98 fusion–driven pediatric leukemia, with a focus on aberrant activation of the Hoxb-associated long, noncoding RNA (lncRNA) HoxBlinc. The authors provide compelling evidence that HoxBlinc plays a central role in the oncogenic transformation associated with NUP98 fusion protein. The study underscores a CTCF-independent role of HoxBlinc in the regulation of topologically associated domains (TADs) and chromatin accessibility, which has not been fully appreciated in previous research on the NUP98 fusion oncogenes. The discovery of HoxBlinc lncRNA as a downstream regulator of NUP98 fusion oncoproteins offers a potential target for therapeutic intervention in pediatric leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5beef2985107dd5f326a5747e1e6f58922f40b5d" target='_blank'>
              HoxBlinc: a key driver of chromatin dynamics in NUP98 fusion–driven leukemia
              </a>
            </td>
          <td>
            Jian Xu, Wei Du
          </td>
          <td>2025-04-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5545243aab210759476cd520384aa54505acdf5" target='_blank'>
              SAUNA - Simulated Annealing for Unique Nucleosome Arrangements from Cell-Free DNA-Sequencing Data
              </a>
            </td>
          <td>
            Maria Rafaela Lambrinos, Tara Rahimi, Tarun Mehra, Michael P Schmid, Nicola Miglino, A. Romanens, Kirsten D. Mertz, Michael Krauthammer, Ronny Nienhold, Andreas Wicki, Zsolt Balázs
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Triggering cancer cell death by inducing DNA damage is the primary aim of radiation therapy; however, normal cells are also damaged. Here, we showed that delivery of only four synthetic guide RNAs (sgRNAs) with Cas9 endonuclease efficiently induced simultaneous DNA double-strand breaks, resulting in efficient cell death in a cell type-specific manner. Off-target effects of Cas9 endonuclease were prevented by using Cas9-nickase to induce DNA single-strand breaks and blocking their repair with poly ADP-ribose polymerase (PARP) inhibitors. When recombinant Cas9-nickase protein and multiple sgRNAs were delivered with PARP inhibitors into cultured cells, in vivo xenografts, and patient-derived cancer organoids via lipid nanoparticles, cancer cells were unable to tolerate the induced DNA damage, even in the presence of a functional BRCA2 gene. This approach has the potential to expand the use of PARP inhibitors with verified safety and thus is a potentially powerful tool for personalized genome-based anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fde0090503581f24fa582fe8e81b3d566923d8b" target='_blank'>
              Combining Multiplexed CRISPR/Cas9-Nickase and PARP Inhibitors Efficiently and Precisely Targets Cancer Cells.
              </a>
            </td>
          <td>
            Soyoung Lee, Kyunghwan Kim, Hye-Jin Jeong, Subin Choi, Himchan Cheng, Dayoung Kim, Soomin Heo, Jinhee Mun, Minjong Kim, Eunjin Lee, Yoon Ji Choi, Seon-Gyeong Lee, Eunjin Lee, Yewon Jang, Kayeong Lim, Heon Seok Kim, Euihwan Jeong, Seung-Jae Myung, Deok-Beom Jung, Chang Sik Yu, In Ho Song, M. Corces, Joo H. Kang, Kyungjae Myung, Taejoon Kwon, Tae-Eun Park, Jinmyoung Joo, Seung Woo Cho
          </td>
          <td>2025-05-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, J. Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d1559fbf64f46ba5665af2509f044f1131070a" target='_blank'>
              SF3B1K700E Drives Oncogenic Alternative Splicing of Cell Cycle Checkpoint Genes in Cancer
              </a>
            </td>
          <td>
            Mai Baker, Eden Engal, Aveksha Sharma, M. Petasny, Shiri Jaffe-Herman, Ophir Geminder, Min-Hua Su, Mercedes Bentata, Gillian Kay, M. Salton
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24df485a542088c9fc326f43378ca5c1c6afe808" target='_blank'>
              Abstract 2483: DirectHRD enables sensitive scar-based classification of homologous recombination deficiency (HRD)
              </a>
            </td>
          <td>
            Ruolin Liu, Eugenia Roberts, H. Parsons, Elizabeth H. Stover, A. Choudhury, J. Rhoades, T. Blewett, David D. Yang, Joyce F. Liu, Erica L. Mayer, V. Adalsteinsson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32199ced2607113320cf436c50b12773694e23a" target='_blank'>
              Abstract 6530: A comprehensive genomic analysis elucidates the microbial landscape of cancer
              </a>
            </td>
          <td>
            Anders B. Dohlman, Robin Mjelle, Henry Wood, I. Lee, Kevin X. Jiang, Ethan Shurberg, Alaina Shumate, Blake E. Sanders, Guanxi Qiao, Jodi Hirshman, Andrew D Cherniack, Phil Quirke, Matthew L. Meyerson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Bleomycin (Bleo) is a glycopeptide with potent antitumor activity that induces DNA double-strand breaks (DSBs) through free radical generation, similar to ionizing radiation (IR). Therefore, Bleo is considered a radiomimetic drug. However, differences in DNA repair mechanisms between IR- and Bleo-induced DNA damage have not been fully elucidated. Therefore, in the present study, we examined a panel of repair-deficient human TK6 cell lines to elucidate the relative contributions of individual repair factors. Results Our comprehensive profiling indicated that both non-homologous end joining (NHEJ) and homologous recombination (HR) contributed to DSB repair induced by X-rays and Bleo. Furthermore, tyrosyl-DNA phosphodiesterase (TDP)-related repair was a significant factor for cellular sensitivity to Bleo treatment. TDP1−/−/TDP2−/− cells exhibited greater sensitivity to Bleo than TDP1−/− or TDP2−/− cells, but not to X-rays. In addition, we determined whether TDP2 is involved in the repair of Bleo-induced DSBs using a neutral comet assay. In TDP1-deficient cells, knockout of TDP2 resulted in a significant delay in the repair kinetics of DSBs induced by Bleo, but not by X-rays. Conclusions The contribution of the TDP-related pathway to DSB repair significantly differed between IR and radiomimetic drugs. The discovery of this novel TDP2-dependent repair of DSBs resulting from radiomimetic drug exposure indicates that TDP1 and TDP2 inhibition in combination with radiomimetic drugs represents a strategy for cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00329-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b896deb7b39c22219dcb649c924ec1a29423e547" target='_blank'>
              Role of TDP2 in the repair of DNA damage induced by the radiomimetic drug Bleomycin
              </a>
            </td>
          <td>
            N. Shimizu, Kazuki Izawa, Mubasshir Washif, Ryosuke Morozumi, Kouji Hirota, Masataka Tsuda
          </td>
          <td>2025-03-28</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA cytosine deaminase APOBEC3B (A3B) is one of the endogenous sources of somatic mutations in many types of human cancers and is associated with tumor progression rather than tumorigenesis. However, it remains uncertain whether APOBEC3B‐induced mutations accelerate tumor progression or not. In this paper, we established a mouse model with A3B overexpression and investigated whether the introduction of A3B overexpression accelerates tumor development in Tp53 hemizygous mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b2e1c29be3a5ab7793714efd1b22c6b4cc4521" target='_blank'>
              APOBEC3B Does Not Promote Tumor Progression in Tp53 Hemizygous Mice
              </a>
            </td>
          <td>
            Yoshihito Horisawa, Tadahiko Matsumoto, June Takeda, Yusuke Tashiro, Ryosuke Nomura, Suguru Takeuchi, Yugo Kawai, Yasuhiro Kazuma, Y. Konishi, H. Yamazaki, Hiroyuki Matsui, Kotaro Shirakawa, A. Takaori-Kondo
          </td>
          <td>2025-04-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/899cd118b3360232a14d8360e2de669be295c1de" target='_blank'>
              The Establishment of Prostate-specific, SKP2 Humanized Mice by CRISPR Knock-in Method Reveals Neoplastic Initiation and Microenvironmental Reprogramming
              </a>
            </td>
          <td>
            Liankun Song, Vyvyan Nguyen, Shan Xu, Kelly Vy Tuong Ho, Bang H Hoang, Jianhua Yu, Edward Uchio, Xiaolin Zi
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dedc758c45f400b459a2e2b62f3f89ff408d534" target='_blank'>
              Abstract 6624: Comprehensive, multi-omic detection of somatic variants from the GIAB HG008 matched tumor-normal pair using highly accurate long- and short-read whole-genome sequencing
              </a>
            </td>
          <td>
            Alexandra Sockell, Khi Pin Chua, Christopher Kingsley, Dan Nasko, Matthew Boitano, Young Kim, Melanie Wescott, Ian J. McLaughlin, P. Baybayan, Jennifer H. McDaniel, Andrew Shaver, J. Zook, Aaron Wenger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-17</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Chromosomal translocations are key events in cancer, driving oncogenesis by disrupting and deregulating critical genes. While specific tumor‐associated translocations are well studied, the frequencies and distributions of most remain unknown. Additionally, the role of chromosomal reshuffling in translocations has received little attention. This study presents data on the chromosomal involvement in 59 251 translocations reported in 58 tumor entities, including both benign and malignant tumors. Unlike studies focusing on tumor‐specific abnormalities identified at the chromosome band level, this study examines translocations at the chromosomal level, offering a novel perspective on their distribution. This broader approach aims to uncover patterns that do not emerge or are disregarded in studies limited to tumor‐specific aberrations. The resulting dataset provides a novel resource for deepening our understanding of the chromosomal origins of translocations in neoplasia. Comparisons of translocation frequency distributions among tumor types, when excluding the characteristic tumor‐associated translocations, revealed that the patterns of chromosomal involvement in translocations are largely unique to each tumor entity. Statistical analyses of 241 pairwise comparisons of translocation spectra within hematologic disorders, solid tumors, and between groups of hematologic malignancies and both benign and malignant solid tumors showed insignificant/very weak associations (R 2 ≤ 0.3) in 98% of the comparisons. The findings hence demonstrate that different tumor types are characterized by distinct chromosomal translocation signatures, strongly suggesting that most translocations encountered in tumor cells are not merely random events. Consequently, our study highlights the potential of rare translocations to serve as indicators of disease‐specific processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c3b03e0bf3342743907df4f51e09cd0d87a1db" target='_blank'>
              Distinct Signatures of Chromosomal Involvement in 59 251 Translocations Across 58 Tumor Types. A Novel Perspective
              </a>
            </td>
          <td>
            Felix Mitelman, Nils Mandahl
          </td>
          <td>2025-05-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background The incidence of colorectal cancer (CRC) in young adults (age of diagnosis < 50 years old) has been rapidly increasing. Although ∼20% of early-onset (EO) CRC cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Non-genetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as differences in the EOCRC epigenome, microbiome and immunome. We sought to address this by comparing differences in DNA methylation from the cohort of colorectal cancer patients in The Cancer Genome Atlas (TCGA). Further, we carefully identified intra-tumoral microbes from TCGA and two other datasets and then related the microbes to EOCRC status and deconvolved immune cell abundances. We found that DNA methylation (DNAm) age acceleration by12 years when compared with average-onset CRC (AOCRC) patients. Differentially methylated sites associated with genes are related to CREB signaling in neurons, G protein coupled receptor signaling, phagosome formation and S100 family signaling. These differences were validated in the gene expression from TCGA and a second, larger real-world dataset from the Oncology Research Information Exchange Network (ORIEN). However, no consistent differences were observed in the intra-tumor microbes between EOCRC and AOCRC. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more, larger, positive correlations in EOCRC. These data suggest epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. SIGNIFICANCE We investigated whether environmentally driven factors contribute to early-onset colorectal cancer (EOCRC). We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than average-onset CRC. These data suggest a dysregulation of immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8859cb309a43420dfdd0f2da144b37f5d23d1c73" target='_blank'>
              Epigenetic Modulation, Intra-tumoral Microbiome and Immunity in Early Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Ning Jin, R. Hoyd, Ayse Selen Yilmaz, Jian Zhu, YunZhou Liu, M. J. Singh, Dennis Grencewicz, Xiaokui Mo, Matthew Kalady, Daniel Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos HF Chan, Michelle L Churchman, N. Denko, Frances Di Clemente, Rebecca D Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, A. Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, A. Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A. Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, Aik Choon Tan, Daniel J. Spakowicz
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Kaposi sarcoma (KS) is the most common cancer in people living with HIV (PLWH), particularly in sub-Saharan Africa (SSA), where Kaposi sarcoma herpesvirus (KSHV or human herpesvirus 8 [HHV-8]) is endemic. In KSHV endemic areas, the overall survival of KS patients has changed little over the past 20 years. A phylogenetic analysis of available full-length viral genomes (n = 164) identified two different virus lineages that co-circulate in KSHV endemic regions today. Their sequences differ from the GenBank reference sequence and those of common laboratory strains, which originated in the 1990s in the US and Europe. Targeted short-read sequencing accuracy was validated by PacBio-based long-read sequencing to resolve repeats. This analysis identified over 1,000 single nucleotide variants (SNV) in a new 14-member sequence collection from tumor biopsies and blood in Malawi with 127 ± 32 (median ± SD) SNV per genome. Most were private, i.e., specific to one individual’s virus. Within each of the two lineages, KSHV continues to evolve over time and across national borders by genetic drift and recombination. Analyses of shared SNVs by AlphaFold2 predicted some changes in the conformation of key viral proteins. These findings may help our understanding of herpesvirus evolution. IMPORTANCE To understand viruses, the field needs to know their genetic makeup. To develop mechanistic models, targeted therapies, and vaccines, we need comprehensive and up-to-date sequence information on the viral strains that circulate where the diseases appear today. Our knowledge of Kaposi sarcoma herpesvirus (KSHV) sequence distribution and evolution is behind that of other human herpesviruses and RNA viruses. Here, we add to community knowledge using new technologies and artificial intelligence. To understand viruses, the field needs to know their genetic makeup. To develop mechanistic models, targeted therapies, and vaccines, we need comprehensive and up-to-date sequence information on the viral strains that circulate where the diseases appear today. Our knowledge of Kaposi sarcoma herpesvirus (KSHV) sequence distribution and evolution is behind that of other human herpesviruses and RNA viruses. Here, we add to community knowledge using new technologies and artificial intelligence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae7db59edece38ee1f1c9bf22d41ac3663a4efd" target='_blank'>
              Genome evolution of Kaposi sarcoma-associated herpesvirus (KSHV)
              </a>
            </td>
          <td>
            Razia Moorad, Alice Peng, Justin T Landis, Linda J Pluta, Patricio Cano, Angelica Juarez, Dirk P. Dittmer
          </td>
          <td>2025-04-16</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c91a033b11a5d910b45df1fe7d4556a0c6c8b6e" target='_blank'>
              Abstract 5585: DNA methylation and transcription factors regulate overexpression of PVT1 in prostate cancer
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Colin Finnegan, Murtaza Barkarar, Oluwabusola Ajagbe, Rachel E. Bonacci, O. Ogunwobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d51acfd255e9f2c3851395fb88bf941b1c544b" target='_blank'>
              Abstract 1735: The discovery of PKMYT1 inhibitors to exploit synthetic lethality caused by replication stress
              </a>
            </td>
          <td>
            R. Marais, J. Grabarek, M. Ioannou, L. Leung, A. Raoof, Mike Brown, D. Smithen, C. Fitzpatrick, C. Chester, J. Leigh, M. Aljarah, M. Lisauskaite, O. Grytsai, R. Burgin, D. Niculescu-Duvaz, G. Thomson, C. Springer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d08007dfb89c4d2387e3dfb8d798320475546d" target='_blank'>
              RF-SIRF defining reversed DNA replication forks with single-cell and spatio-temporal resolution reveals a replication stress specific epigenetic code
              </a>
            </td>
          <td>
            Sunetra Roy, Morgan Fimreite, Yue Chen, Francesca Citron, Giulio Draetta, K. Schlacher
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Archaea, often thriving in extreme habitats, are believed to have evolved efficient DNA repair pathways to cope with constant insults to their genomes. However, how these organisms repair DNA double-strand breaks (DSBs), the most lethal DNA lesions, remains unclear. Here, we show that replicative primase consisting of the catalytic subunit PriS and the noncatalytic subunits PriL and PriX from the hyperthermophilic archaeon Saccharolobus islandicus is involved in DSB repair. We show that the overproduction or knockdown of PriL increases or decreases, respectively, the rate of survival and mutation frequency of S. islandicus cells following treatment with a DNA damaging agent. The increase in mutation is attributed primarily to an increase in small insertions or deletions. Further, overproduction of PriL enhances the repair of CRISPR-generated DSBs in vivo. These results are consistent with the extraordinary ability of PriSL to promote annealing between DNA strands sharing microhomology in addition to the activity of the heterodimer in terminal transfer and primer extension. The primase-mediated DSB repair is cell-cycle dependent since PriL is barely detectable during the S/G2 transition. Our data demonstrate that replicative primase is involved in DSB repair through microhomology-mediated end joining in Archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41c802b392e2aa09bc0c77d38347ce453e14aa3" target='_blank'>
              Archaeal replicative primase mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Daijiang Xiong, Zhimeng Li, Wen Qi, Shaoying Wang, Junkai Huang, Ningning Zhang, Zhenfeng Zhang, Li Huang
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The development of hematopoietic cell lineages is a highly complex process governed by a delicate interplay of various transcription factors. The expression of these factors is influenced, in part, by epigenetic signatures that define each stage of cell differentiation. In particular, the formation of B lymphocytes depends on the sequential silencing of stemness genes and the balanced expression of interdependent transcription factors, along with DNA rearrangement. We have investigated the impact of Dido3 deficiency, a protein involved in chromatin status readout, on B cell differentiation within the hematopoietic compartment of mice. Our findings revealed significant impairments in the successive stages of B cell development. The absence of Dido3 resulted in remarkable alterations in the expression of essential transcription factors and differentiation markers, which are crucial for orchestrating the differentiation process. Additionally, the somatic recombination process, responsible for generation of antigen receptor diversity, was also adversely affected. These observations highlight the vital role of epigenetic regulation, particularly the involvement of Dido3, in ensuring proper B cell differentiation. This study reveals new mechanisms underlying disruptive alterations, deepening our understanding of hematopoiesis and may potentially lead to insights that aid in the development of therapeutic interventions for disorders involving aberrant B cell development. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01394-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba226143005da64aa14cb154db38e86341f77b0e" target='_blank'>
              The chromatin reader Dido3 is a regulator of the gene network that controls B cell differentiation
              </a>
            </td>
          <td>
            Fernando Gutiérrez del Burgo, María Ángeles García-López, Tirso Pons, Enrique Vázquez de Luis, Carlos Martínez-A, R. Villares
          </td>
          <td>2025-04-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Replication stress is a key driver of DNA damage and genome instability. Replication stress-induced fork remodelling generates a new DNA end that is vulnerable to the action of nucleases, and which is protected by a range of factors including the canonical tumour suppressors BRCA1 and BRCA2. Here we report that replication stress drives elevated production of cytokines and chemokines in the absence of DNA damage. The DNA sensor IFI16 binds nascent DNA at stalled replication forks and signals via the DNA sensing adaptor STING, to induce the activation of NF-κB and the production of pro-inflammatory cytokines in response to replication stress. IFI16 also acts directly at stalled replication forks to protect nascent DNA from degradation by the nucleases MRE11 and DNA2. Furthermore, IFI16 is required for the interferon-mediated rescue of fork protection in BRCA-deficient cells, highlighting the critical role of IFI16 in the cross-talk between innate immunity and fork protection during replication stress. Highlights Replication stress induces an early innate immune response, which is dependent on the DNA sensing factors IFI16 and STING, but not cGAS IFI16 binds directly to nascent DNA at stalled replication forks IFI16 prevents nucleolytic degradation of reversed forks IFI16 is required for interferon-mediated fork protection in BRCA-deficient cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3ae95a322db83408cb26a59a9ec45c07506cd8" target='_blank'>
              IFI16 senses and protects stalled replication forks
              </a>
            </td>
          <td>
            Amelia Gamble, Thomas A. Ward, O. P. Wheeler, Caryl M. Jones, L. G. Bennett, Ellen G. Vernon, Vithursha Thanendran, Jessica P Morris, Ilaria Ceppi, Swagata Halder, Damiano Borello, Thomas D J Walker, Jahnavi Rajan, Gillian Dunphy, Petr Cejka, Leonie Unterholzner, C. J. Staples
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6032320cee71742e5a132f4662a3a337579e09" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe.
              </a>
            </td>
          <td>
            Joshua M Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-05-03</td>
          <td>Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Inaccurate chromosome segregation can lead to the formation of aneuploid cells that harbor an imbalanced complement of chromosomes. Several fungal species are not only able to tolerate the detrimental effects of aneuploidy but can use it to adapt to environmental pressures. The fungal pathobiont Candida albicans frequently acquires supernumerary chromosomes that enable growth in the presence of antifungal drugs or in specific host niches, yet the transcriptional changes associated with aneuploidy are not globally defined. Here, a karyotypically diverse set of C. albicans strains revealed that expression generally correlated with gene copy number regardless of the strain karyotype. Unexpectedly, aneuploid strains shared a characteristic transcriptional profile that was distinct from a generalized environmental stress response previously defined in aneuploid yeast cells. This aneuploid transcriptional response led to altered growth and oxidative balances relative to euploid control strains. The increased expression of reactive oxygen species (ROS) mitigating enzymes in aneuploid cells reduced the levels of ROS but caused an acute sensitivity to both internal and external sources of oxidative stress. Taken together, our work demonstrates common transcriptional and phenotypic features of aneuploid C. albicans cells with consequences for infection of different host niches and susceptibility to environmental stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e3ff6416e6e4c37bdd4902bf219f1aee84a69b" target='_blank'>
              Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans
              </a>
            </td>
          <td>
            Anna I Mackey, Robert J. Fillinger, P. S. Hendricks, Gregory J. Thomson, Christina A. Cuomo, Richard J. Bennett, Matthew Z. Anderson
          </td>
          <td>2025-04-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de660154998a356e9b85c8c4e7d59ed34b5f5dd0" target='_blank'>
              Abstract 4668: Prognostic significance and therapeutic potential of TOP2A expression in rare cancers
              </a>
            </td>
          <td>
            Jifa Li, Hua Ye, Xiaoqiong Bao, Wenwen Yu, Xiangxiang Chen, Yangyang Ni, Jincong Wang, Yubo Shi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6270b4747c6c065b93137fa387d570507ac041" target='_blank'>
              Compact Origins and Where to Find Them: ORC's Guide to Genome-Wide Licensing.
              </a>
            </td>
          <td>
            Christian Speck, L. M. Reuter
          </td>
          <td>2025-05-19</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d431bff55b178eb615284d91d43a85ede147802" target='_blank'>
              Abstract 5488: Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay
              </a>
            </td>
          <td>
            Darian Williams, Andrew McFadden, Austin J. Hepperla, Min Shen, Tino W Sanchez, Gordon M. Stott, M. Henderson, Ian Davis, Samantha Pattenden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) present a critical threat to genomic integrity, often precipitating genomic instability and oncogenesis. Repair of DSBs predominantly occurs through homologous recombination (HR) and non-homologous end joining (NHEJ). In HR-deficient cells, DNA polymerase theta (Polθ) becomes critical for DSB repair via microhomology-mediated end joining (MMEJ), also termed theta-mediated end joining (TMEJ). Thus, Polθ is synthetically lethal with BRCA1/2 and other HR factors, underscoring its potential as a therapeutic target in HR-deficient cancers. However, the molecular mechanisms governing Polθ-mediated MMEJ remain poorly understood. Here we present a series of cryo-electron microscopy structures of the Polθ helicase domain (Polθ-hel) in complex with DNA containing different 3′-ssDNA overhangs. The structures reveal the sequential conformations adopted by Polθ-hel during the critical phases of DNA binding, microhomology searching, and microhomology annealing. The stepwise conformational changes within the Polθ-hel subdomains and its functional dimeric state are pivotal for aligning the 3′-ssDNA overhangs, facilitating the microhomology search and subsequent annealing necessary for DSB repair via MMEJ. Our findings illustrate the essential molecular switches within Polθ-hel that orchestrate the MMEJ process in DSB repair, laying the groundwork for the development of targeted therapies against the Polθ-hel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdae7f64677f27faa8b83647b04ea59147fae30b" target='_blank'>
              Structural basis for Polθ-helicase DNA binding and microhomology-mediated end-joining
              </a>
            </td>
          <td>
            Fumiaki Ito, Ziyuan Li, Leonid Minakhin, Htet A. Khant, R. Pomerantz, Xiaojiang S. Chen
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA) is present in food, water, and drugs and is considered a probable human carcinogen by the International Agency for Research on Cancer. The mechanism of action of NDMA involves the generation of carcinogenic methyl lesions such as 3-methyladenine (3MeA) on DNA bases. Alkyladenine DNA Glycosylase (AAG) removes 3MeA to initiate Base Excision Repair, leaving an intermediary lesion that is subsequently resolved by backbone cleavage, nucleotide insertion, and backbone ligation. The intermediate steps following lesion removal produce potentially toxic and mutagenic single-strand DNA breaks. Here, we explored differences between males and females regarding downstream DNA damage, toxicity, mutations and cancer arising from 3MeA in the livers of WT, Aag-/-, and Aag-overexpressing (AagTg) mice. We found that males were more susceptible to NDMA-induced mutations (WT and Aag-/-) and cancer (all genotypes). In contrast, AagTg females were more prone than males to micronucleus induction. As we showed in our prior analyses where data were pooled for males and females, Aag-/- mice were significantly more susceptible to NDMA-induced mutations and cancer, and AagTg mice displayed significantly greater toxicity. Building on these findings, our analyses of sex-related differences show that Aag deficiency and maleness are both susceptibility factors for NDMA-induced liver cancer, while Aag overexpression drives toxicity, potentially with a greater effect on females. By assessing differences between males and females, this study reveals a deeper mechanistic understanding of the underpinnings for a well-known increased risk of liver cancer in men versus women by demonstrating a higher susceptibility of male mice to both mutations and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2ad8f9213bd9ddfa98933a729d0b81e465af3f" target='_blank'>
              Sex and Alkyladenine DNA Glycosylase Expression are Key Susceptibility Factors for NDMA-induced Mutations, Toxicity, and Cancer
              </a>
            </td>
          <td>
            Jennifer E. Kay, J. Corrigan, Lindsay B Volk, Amanda L Armijo, Ilana S Nazari, D. Torous, Svetlana L. Avlasevich, R. G. Croy, D. Wadduwage, S. Dertinger, J. Essigmann, Leona D. Samson, B. Engelward
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, C. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Most solid tumors harbor somatic mutations attributed to off-target activities of APOBEC3A (A3A) and/or APOBEC3B (A3B). However, how APOBEC3A/B enzymes affect tumor evolution in the presence of exogenous mutagenic processes is largely unknown. Here, multi-omics profiling of 309 lung cancers from smokers identifies two subtypes defined by low (LAS) and high (HAS) APOBEC mutagenesis. LAS are enriched for A3B-like mutagenesis and KRAS mutations; HAS for A3A-like mutagenesis and TP53 mutations. Compared to LAS, HAS have older age at onset and high proportions of newly generated progenitor-like cells likely due to the combined tobacco smoking- and APOBEC3A-associated DNA damage and apoptosis. Consistently, HAS exhibit high expression of pulmonary healing signaling pathway, stemness markers, distal cell-of-origin, more neoantigens, slower clonal expansion, but no smoking-associated genomic/epigenomic changes. With validation in 184 lung tumor samples, these findings show how heterogeneity in mutational burden across co-occurring mutational processes and cell types contributes to tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c82f658add82b575c331a2df70337786a4843a5f" target='_blank'>
              APOBEC affects tumor evolution and age at onset of lung cancer in smokers
              </a>
            </td>
          <td>
            Tongwu Zhang, Jian Sang, P. Hoang, Wei Zhao, J. Rosenbaum, K. E. Johnson, L. Klimczak, John P. McElderry, Alyssa Klein, Christopher Wirth, E. Bergstrom, M. Díaz-Gay, Raviteja Vangara, Frank J. Colón-Matos, A. Hutchinson, Scott M Lawrence, N. Cole, Bin Zhu, T. Przytycka, Jianxin Shi, N. E. Caporaso, Robert J Homer, A. Pesatori, D. Consonni, Marcin Imieliński, Stephen J Chanock, David C Wedge, D. Gordenin, Ludmil B. Alexandrov, R. Harris, Marion Landi
          </td>
          <td>2025-05-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3da42eb5cba4f866717a7ca7075985f83d3c3fb" target='_blank'>
              Review Article
‘An Overview Genetics of Inherited Neurological and Non-Neurological Disorders in India’
              </a>
            </td>
          <td>
            Male D. N., Amle V. S., Ade R. P., Vyvahare S. S., Pawar V. K.
          </td>
          <td>2025-04-05</td>
          <td>International Journal of Advanced Research in Science, Communication and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cells are the fundamental structural and functional units of living organisms and the study of these entities has remained a central focus throughout the history of biological sciences. Traditional cell research techniques, including fluorescent protein tagging and microscopy, have provided preliminary insights into the lineage history and clonal relationships between progenitor and descendant cells. However, these techniques exhibit inherent limitations in tracking the full developmental trajectory of cells and elucidating their heterogeneity, including sensitivity, stability and barcode drift. In developmental biology, nucleic acid barcode technology has introduced an innovative approach to cell lineage tracing. By assigning unique barcodes to individual cells, researchers can accurately identify and trace the origin and differentiation pathways of cells at various developmental stages, thereby illuminating the dynamic processes underlying tissue development and organogenesis. In cancer research, nucleic acid barcoding has played a pivotal role in analyzing the clonal architecture of tumor cells, exploring their heterogeneity and resistance mechanisms and enhancing our understanding of cancer evolution and inter-clonal interactions. Furthermore, nucleic acid barcodes play a crucial role in stem cell research, enabling the tracking of stem cells from diverse origins and their derived progeny. This has offered novel perspectives on the mechanisms of stem cell self-renewal and differentiation. The present review presented a comprehensive examination of the principles, applications and challenges associated with nucleic acid barcode technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5cf40687295751922bb2f7725d94b4cd317977" target='_blank'>
              Barcodes based on nucleic acid sequences: Applications and challenges (Review)
              </a>
            </td>
          <td>
            Ying Hong Wei, Faquan Lin
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The integrity of the replication forks is essential for an accurate and timely completion of genome duplication. However, little is known about how cells deal with broken replication forks. We have generated in yeast a system based on a chimera of the largest subunit of the ssDNA binding complex RPA fused to the micrococcal nuclease (Rfa1-MN) to induce double-strand breaks (DSBs) at replication forks and searched for mutants affected in their repair. Our results show that the core homologous recombination (HR) proteins involved in the formation of the ssDNA/Rad51 filament are essential for the repair of DSBs at forks, whereas non-homologous end joining plays no role. Apart from the endonucleases Mus81 and Yen1, the repair process employs fork-associated HR factors, break-induced replication (BIR)-associated factors and replisome components involved in sister chromatid cohesion and fork stability, pointing to replication fork restart by BIR followed by fork restoration. Notably, we also found factors controlling the length of G1, suggesting that a minimal number of active origins facilitates the repair by converging forks. Our study has also revealed a requirement for checkpoint functions, including the synthesis of Dun1-mediated dNTPs. Finally, our screening revealed minimal impact from the loss of chromatin factors, suggesting that the partially disassembled nucleosome structure at the replication fork facilitates the accessibility of the repair machinery. In conclusion, this study provides an overview of the factors and mechanisms that cooperate to repair broken forks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9dcc0d5d251baac60feb7bd1bf610ba60b8d75" target='_blank'>
              A Rfa1-MN–based system reveals new factors involved in the rescue of broken replication forks
              </a>
            </td>
          <td>
            Ana Amiama-Roig, Marta Barrientos-Moreno, Esther Cruz-Zambrano, Luz M López-Ruiz, R. González-Prieto, Gabriel Ríos-Orelogio, F. Prado
          </td>
          <td>2025-04-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4f3702444f966febda408aa09a277d286c200" target='_blank'>
              Abstract 1409: Reprogramming the alternative polyadenylation of an oncogenic transcription factor as a therapeutic strategy for prostate cancer
              </a>
            </td>
          <td>
            Giulia Pagani, Chiara Casirati, Stefania Fornezza, Roberta Cacioppo, P. Gandellini
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, S. Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53622534a6ad4bf417cb989b871bcc51c0303f4" target='_blank'>
              Abstract 1420: Long-read sequencing of single-stranded DNA of colorectal cancer patients
              </a>
            </td>
          <td>
            Béla Molnár, K. Rada, Nikolett Szakállas, B. Barták, Alexandra Kalmár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997f4fdd1e9ff15b92b139ad11fa72e835f8dc6a" target='_blank'>
              c-Abl Phosphorylates Plk1 in Facilitating DNA Damage-Induced G2/M Checkpoint Release with a Trade-off of Micronuclei Formation
              </a>
            </td>
          <td>
            Victoria Meltser, M. Ben-Yehoyada, Julia Adler, N. Reuven, Y. Shaul
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e879216882ba777a52dac02a65cb65b0e331d4" target='_blank'>
              Abstract 5361: Clinical implications of The Cancer Genome Atlas molecular classification system in esophagogastric cancer
              </a>
            </td>
          <td>
            H. Walch, Raktim Borpatragohain, Justin Jee, W. Chatila, Christopher Fong, S. Maron, Geoffrey Y Ku, D. Ilson, Yelena Y Janjigian, Abraham J Wu, Pari Shah, Dan Coit, M. Bains, Valerie W Rusch, B. Park, M. Bott, Katherine Gray, David R. Jones, Michael F. Berger, N. Schultz, V. Strong, D. Molena, S. Sihag
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bde3f65de807bb9cec86aaa0dfc6c09be8467d" target='_blank'>
              Abstract 1942: Copy number aberrations in the ctDNA uncover clinical and biological heterogeneity in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Maare Arffman, L. Meriranta, J. Jørgensen, M. Karjalainen-Lindsberg, K. Beiske, Mette Pedersen, Kristina DrottØystein Fluge, S. Jyrkkiö, Peter Brown, Harald Holte, S. Leppä
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="KRAS is a proto-oncogene that contains activating mutations in up to 30% of tumors. Many conventional therapies inhibit both cancerous and normal cells, which may cause toxicity. Thus, programmable mutant-selective targeted inhibitors are needed. Peptide nucleic acids (PNAs) incorporate base sequences analogous to DNA, with modified peptide backbones instead of ribose-phosphate backbones, allowing PNAs to hybridize with DNA with high avidity to suppress transcription. Here, we developed KRAS G12D-selective PNA oligomers with novel cell-penetrating flanking regions. Fluorescein-labeled PNA oligomers displayed high uptake rates in cells and nuclei. Exposure to PNA-delivery peptide conjugates resulted in repression of KRAS G12D mRNA and protein expression within 2 hours and lasting up to 48 hours. Varying cell-penetrating peptide (CPP) compositions and lengths of complementary KRAS sequences were tested using dose-response cell viability assays. These experiments identified configurations that were effective at selectively preventing growth of on-target KRAS G12D cells, while relatively sparing off-target KRAS G12C cells. Electrophoretic mobility shift assays demonstrated in vitro binding and selectivity for KRAS G12D DNA sequences. CPP-PNA-G12D-1 was effective against a panel of pancreatic ductal adenocarcinoma cell lines and patient-derived xenografts in vivo. These results show promise for an enhanced PNA-delivery peptide conjugate strategy as both a tool for studying tumors driven by oncogenic point mutations and as a potential therapeutic strategy to selectively target mutant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/809bc7a2732828bb8a1dec23dc2dcff988fd6804" target='_blank'>
              [Aberrant RNA Splicing Regulation as a Novel Source of Neoantigens].
              </a>
            </td>
          <td>
            Masahiko Ajiro
          </td>
          <td>2025-04-01</td>
          <td>Gan to kagaku ryoho. Cancer & chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973aff9217cd2f6bd7d72f2139353fd552c0fa4e" target='_blank'>
              SARE: an optimized method for detection functional regulatory elements in genome
              </a>
            </td>
          <td>
            Mohammadjavad Hosseinpoor
          </td>
          <td>2025-04-16</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2065faaa8c8b01822bceb610976fd9fb9507b1" target='_blank'>
              Mechanism of DNA replication fork breakage and PARP1 hyperactivation during replication catastrophe.
              </a>
            </td>
          <td>
            Pedro Ortega, Elodie Bournique, Junyi Li, Ambrocio Sanchez, Gisselle Santiago, Brooke R. Harris, Josefine Striepen, J. Maciejowski, Abby M. Green, Rémi Buisson
          </td>
          <td>2025-04-18</td>
          <td>Science advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034830e2f3dfdd6e357c8df85c64b85ab42aa12c" target='_blank'>
              Abstract LB317: Liquid-liquid phase separation drives tumorigenesis by modulating the chromatin accessibility of paused and non-paused genes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Alavaro Lucci, Anna E. Barry, Victoria Johnson, Brittany Ruiz, Laura Reynolds, Adam Wojnar, Kasonde Chewe, Y. Pomyen, Christoph Eckert, Kai Zhang, Jason Hill, Ashesh Shah, Adam Bodzin, Elda Grabocka, Matthias M Gaida, Nicolas Fawzi, Hien Dang
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Eukaryotic genomes are physically organized into chromatin, which regulates virtually every dimension of genome biology, including replication, recombination, transcription, and DNA repair. Precocious Dissociation of Sisters 5 (PDS5) protein is a key regulator of the cohesin complex, responsible for generating higher-order chromatin structures within and between chromosomes. In yeast and vertebrates, PDS5 can lead to cohesin establishment or disassembly, depending on its interaction with other proteins. In this study, we explored the diversity of PDS5 proteins across eukaryotes; HEAT-repeats forming a hook-like structure (PDS5 domain), followed by a predicted disordered region of variable lengths, are observed across the Tree of Life. Interestingly, two homolog types (A and B) with signs of varying chromatin localization emerged independently in later branches of animal and plant kingdoms, in vertebrates and vascular plants, respectively (∼400-500 MYA). We also investigated the emergence of two plant-specific features in the PDS5 proteins: 1.) An ancestral acquisition of a Tudor domain with conserved H3K4me1-binding amino acids in land plants 2.) A truncated version of PDS5 proteins that lacks essential features of PDS5 domain and is involved in the RNA-directed DNA methylation (RdDM) plant-specific pathway which emerged in seed plants. This work provides the most comprehensive study of PDS5 diversity, and suggests the emergence of specialized cohesin regulatory mechanisms that could shape distinct strategies for genome organization across eukaryotes. Short summary Here, we analyzed the diversification of PDS5 proteins; sequence, domain content, and structure, providing new observations that evidence different evolutionary trajectories of this protein across the Tree of Life. These results suggest neofunctional evolution in cohesin regulatory proteins that could shape distinct strategies for genome organization across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c1664fc04ea9ae186e4173c40cb2e49fd9d273" target='_blank'>
              Evolutionary trajectories of cohesin gatekeepers across the Tree of Life
              </a>
            </td>
          <td>
            Daniela Quiroz, Alice Pierce, Grey Monroe
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b447161a1815b3267cbfa70059d3d28a80f8d107" target='_blank'>
              Targeting redox-sensitive MBD2-NuRD condensate in cancer cells.
              </a>
            </td>
          <td>
            Heyang Wei, Hongdan Zheng, Siqing Wang, Yun Yang, Ruiqian Zhao, Aihong Gu, Ronggui Hu, Fei Lan, Wenyu Wen
          </td>
          <td>2025-04-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9b8b5ce673ffe516ad9412ab9c6b0cc6972287" target='_blank'>
              Damage-induced phosphorylation of BRC-1/BRD-1 in meiosis preserves germline integrity
              </a>
            </td>
          <td>
            N. Fernández-Fernández, M. Chacón, LP Camino, T. García-Muse
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Toxoplasma gondii is an obligate intracellular parasite with a high replication rate that can lead to DNA replicative stress, in turn associated with the generation of DNA double-strand breaks (DSBs). Cells have two main pathways to repair DSBs: non-homologous end joining and homologous recombination repair (NHEJ and HRR respectively). RAD51 is the key recombinase in the HRR pathway. In this work, we achieved endogenous tagging of the RAD51 gene using the Auxin Inducible Degron (AID) system, to generate the clonal line RH RAD51HA-AID. Here we demonstrate that RAD51 is expressed in replicative tachyzoites and establishes damage foci. Auxin-induced knock-down (KD) affects the correct replication of tachyzoites which show loss of synchronization. The use of the RAD51 inhibitor B02 also affects parasite growth, with an IC50 of 4.8 µM. B02 produced alterations in tachyzoite replication and arrest in the S phase of the cell cycle. Additionally, B02 induced tachyzoite to bradyzoite differentiation showing small cyst-like structures. In conclusion, the HRR pathway is necessary for maintaining proper tachyzoite replication under normal growth conditions, supporting that replicative stress occurs during the cell cycle. Our findings also suggest that DNA replication stress can induce bradyzoite differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a99ccd548e29dfb7eccfd9c686305c90fbab532" target='_blank'>
              Toxoplasma gondii RAD51 recombinase is required to overcome DNA replication stress and its inactivation leads to bradyzoite differentiation
              </a>
            </td>
          <td>
            Ana M. Saldarriaga Cartagena, Ayelén Aparicio Arias, C. Cristaldi, Agustina Ganuza, M. M. Gonzalez, María M. Corvi, William J. Sullivan, Laura Vanagas, Sergio O. Angel
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46739b61c6c2d94cf2bbcd86bcaf1b74602dd7eb" target='_blank'>
              cellSTAAR: Incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of non-coding regions
              </a>
            </td>
          <td>
            E. V. Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, Joanne E. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear Kenny, C. Kooperberg, B. Kral, Leslie Lange, Dan Levy, Changwei Li, Simin Liu, Donald Lloyd-Jones, R. F. Loos, A. Manichaikul, Lisa W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari North, Jeffery R O'connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Vasan S Ramachandran, S. Redline, Alexander P. Reiner, Steve Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D Taylor, Russell Tracy, Satupa'itea Viali, Lisa Yanek, Wei Zhao, Jerome I. Rotter, G. Peloso, Pradeep Natarajan, Xihong Lin
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af01fc45afb6596f1391be85d304128cf0bbe91" target='_blank'>
              Abstract 1481: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8572ed5aac9e75313380c3e2cd2f1a7e899e36b7" target='_blank'>
              Abstract 1903: Actionable mutational landscape in solid tumors: a case study of ovarian and colon cancer
              </a>
            </td>
          <td>
            Amit Kumar, Samir Agarwal, Pankaj Khanna, Shobit Gupta, Rama Chandran, Ananth S. Bommakanti
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84c48d97f5402828d77d20a338b0d89c4f940ea2" target='_blank'>
              Abstract 7138: Whole genome sequencing-based detection of minimal residual disease using individual-specific features
              </a>
            </td>
          <td>
            Jun-Hyeok Lim, Jeong-Seon Ryu, Ji Won Byun, Jongsuk Chung, Hyun-Tae Shin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/905721fd736b7552781498ea677657c472cb1c43" target='_blank'>
              Aflatoxin B1‐Induced Hepatic Mutagenesis in Mice Expressing Gene‐Edited Neil1
              </a>
            </td>
          <td>
            I. Minko, Vladimir L Vartanian, Michael M Luzadder, Yingming Wang, Lev M Fedorov, A. McCullough, R. S. Lloyd
          </td>
          <td>2025-04-01</td>
          <td>Environmental and Molecular Mutagenesis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Bone metastases represent frequent and severe complications in various cancers, notably impacting prognosis and quality of life. This review article delves into the genetic and epigenetic mechanisms underpinning drug resistance in bone metastases, a key challenge in effective cancer treatment. The development of drug resistance in cancer can manifest as either intrinsic or acquired, with genetic heterogeneity playing a pivotal role. Intrinsic resistance is often due to pre-existing mutations, while acquired resistance evolves through genetic and epigenetic alterations during treatment. These alterations include mutations in driver genes like TP53 and RB1, epigenetic modifications such as DNA methylation and histone changes, and pathway alterations, notably involving RANK-RANKL signaling and the PI3K/AKT/mTOR cascade. Recent studies underline the significance of the tumor microenvironment in fostering drug resistance, with components such as cancer-associated fibroblasts and hypoxia playing crucial roles. The interactions between metastatic cancer cells and the bone microenvironment facilitate survival and the proliferation of drug-resistant clones. This review highlights the necessity of understanding these complex interactions to develop targeted therapies that can overcome resistance and improve treatment outcomes. Current therapeutic strategies and future directions are discussed, emphasizing the integration of genomic profiling and targeted interventions in managing bone metastases. The evolving landscape of genetic research, including the application of next-generation sequencing and CRISPR technology, offers promising avenues for novel and more effective therapeutic strategies. This comprehensive exploration aims to provide insights into the molecular intricacies of drug resistance in bone metastases, paving the way for improved clinical management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3b250a10d84b32555ed5529bedbf646475e86e" target='_blank'>
              The genetic architecture of bone metastases: unveiling the role of epigenetic and genetic modifications in drug resistance
              </a>
            </td>
          <td>
            Ahmad Dawalibi, Mohamad Bakir, Khalid S. Mohammad
          </td>
          <td>2025-04-22</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d31436619804913b5e6fe39fc1dbf3eb2207ea0" target='_blank'>
              Abstract 2920: ZBP1 antagonizes MRE11-mediated DNA end resection and confers synthetic lethality to PARP inhibition in Ovarian cancer
              </a>
            </td>
          <td>
            Shen-nan Shi, Qiuyang Xu, Xuejiao Zhao, Gordon B. Mills, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63aefe6d3e777332427bd46910b51b4d2635b3ae" target='_blank'>
              Abstract LB476: Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            R. Harris
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34833006929f8c53f4dd25f1eecd3e7760e96187" target='_blank'>
              Biochemical Mechanisms of Genetic Recombination and DNA Repair.
              </a>
            </td>
          <td>
            Vivek B Raina, Aidan M. Jessop, E. Greene
          </td>
          <td>2025-03-28</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The intricacies of tumour microenvironment, particularly the extracellular matrix (ECM), underscore its pivotal function in modulating tumour progression and drug resistance. Among the key regulators of ECM remodelling and homeostasis, the lysyl oxidases (LOXs) emerge as promising therapeutic targets of tumour treatment. Despite their significance, a holistic evaluation of the LOX family's genomics and clinical implications across diverse cancer types remains elusive. Herein, this study aimed to investigate the correlation between LOX family expression and patient outcomes, drug responsiveness and tumour microenvironment (TME) characteristics in a cohort of 33 tumours based on The Cancer Genome Atlas (TCGA) database. Notably, patients exhibiting elevated LOX family expression suffer from worse prognosis and resistance to a spectrum of antitumor therapies, encompassing chemotherapy, endocrine therapy, targeted therapy and immunotherapy, in contrast to counterparts with subdued LOX family expression levels. Furthermore, enrichment analysis indicated that the LOX family fosters tumour progression and drug resistance. These findings were further validated by multiplex immunofluorescence staining in breast, gastric and rectal cancer, as well as breast cancer organoids. Altogether, this study unravels the intricate association between the LOX family and tumour progression, alongside multidrug resistance. We have gained further insights into the roles of LOX family genes in various tumour types, offering a novel avenue for future research into the relationship between LOX family genes and tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de26c0c5ab2c00eb5a4adc28b824cfdafa9df26" target='_blank'>
              Systematic Characterisation and Analysis of Lysyl Oxidase Family Members as Drivers of Tumour Progression and Multiple Drug Resistance
              </a>
            </td>
          <td>
            Hongjin Liu, Xiaojiao Sun, Bingqi Dong, Jixin Zhang, Junling Zhang, Yanlun Gu, Lin Chen, Xiaocong Pang, Jingming Ye, Xin Wang, Zhuona Rong
          </td>
          <td>2025-04-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b042bad9ffec7e55a0de94ab786a57c1a323f2d" target='_blank'>
              Abstract 4568: Biochemical mitigation of technical artifacts improves performance of a ctDNA methyl sequencing assay
              </a>
            </td>
          <td>
            James Yu, Peter George, J. Close, Andy Eang, N. Peterman, Yu Sun, Nafisa Bulsara, Eileen Dimalanta, Alex Robertson, Yexun Wang, Nicole Lambert
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18c7e36e3b876db6b83aaa4bdaf8e2434f722c8a" target='_blank'>
              Oncogenic fusions converge on shared mechanisms in initiating astroblastoma.
              </a>
            </td>
          <td>
            Yixing Shi, Qianqian Sun, Fuchuan Jia, Xiangyu Xie, Xiangyu Zhou, Rong Guo, Yangfan Zeng, Shanshan Chen, Zhenzhen Guo, Wenli Sun, Tong Guo, Yu Xia, Wenlong Li, Li Zhang, Wei Shi, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Nature</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The three-dimensional (3D) organization of the genome is crucial for gene regulation, with disruptions linked to various diseases. High-throughput Chromosome Conformation Capture (Hi-C) and related technologies have advanced our understanding of 3D genome organization by mapping interactions between distal genomic regions. However, capturing enhancer–promoter interactions at high resolution remains challenging due to the high sequencing depth required. We introduce pC-SAC (probabilistically Constrained Self-Avoiding Chromatin), a novel computational method for producing accurate high-resolution Hi-C matrices from low-resolution data. pC-SAC uses adaptive importance sampling with sequential Monte Carlo to generate ensembles of 3D chromatin chains that satisfy physical constraints derived from low-resolution Hi-C data. Our method achieves over 95% accuracy in reconstructing high-resolution chromatin maps and identifies novel interactions enriched with candidate cis-regulatory elements (cCREs) and expression quantitative trait loci (eQTLs). Benchmarking against state-of-the-art deep learning models demonstrates pC-SAC’s performance in both short- and long-range interaction reconstruction. pC-SAC offers a cost-effective solution for enhancing the resolution of Hi-C data, thus enabling deeper insights into 3D genome organization and its role in gene regulation and disease. Our tool can be found at https://github.com/G2Lab/pCSAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bc34f801c6a85a73e83d49ad30fa8158531f9d" target='_blank'>
              pC-SAC: A method for high-resolution 3D genome reconstruction from low-resolution Hi-C data
              </a>
            </td>
          <td>
            J. C. Angel, Narjis El Amraoui, Gamze Gürsoy
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="DNA topoisomerase 1 (TOP1) is essential for transcription, replication, and repair. Its function relies on two catalytic steps, DNA breakage and rejoining. Inhibitors of the second step prevent DNA rejoining and lead to persistent DNA breaks, acting as topoisomerase poisons, used as anticancer drugs. However, reliable inhibitors of the first step are not available. Here, we provide genetic and molecular evidence supporting that Patulin and, to a lesser extent, Xestoquinol inhibit the first catalytic step of TOP1 in vitro, in yeast and in human cells. Particularly, Patulin prevents the accumulation of TOP1 cleavage complexes caused by the TOP1 poison camptothecin (CPT) in human cells. Moreover, Patulin pretreatment of human or yeast cells reduces DNA damage and the accumulation of DNA breaks upon CPT exposure. Consistent with the protective role of TOP1 against harmful R-loops, Patulin treatment increases R-loops and R-loop-associated cytotoxicity, mimicking the effect of TOP1 silencing. Altogether our findings indicate that Patulin and Xestoquinol are nonpoisoning inhibitors of TOP1, which should potentiate new research approaches in molecular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c91b4af793ee46a1c397d0e749fad65c6dce6ba" target='_blank'>
              Patulin and Xestoquinol are inhibitors of DNA topoisomerase 1.
              </a>
            </td>
          <td>
            E. Tumini, R. Wellinger, Emilia Herrera-Moyano, Patricia Navarro-Cansino, María L. García-Rubio, Daniel Salas-Lloret, Alejandro Losada, M. Muñoz-Alonso, H. Gaillard, Rosa Luna, A. Aguilera
          </td>
          <td>2025-04-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b4705110ed5255e541024e217bd1b38f97bdc5" target='_blank'>
              Abstract 2802: Circular RNA variants of theNUMB gene modulate the oncogenicity of oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Chung-Hsien Chou, Pin-Yu Chen, Shu-Chun Lin, Kuo-Wei Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ABSTRACT Dysregulated translation is a hallmark of cancer, and recent genome-wide studies in tumour cells have uncovered widespread translation of non-canonical reading frames that often initiate at non-AUG codons. If an upstream non-canonical start site is located within a frame with an annotated coding sequence (CDS), such translation events can lead to the production of proteoforms with altered N-termini (PANTs). Certain examples of PANTs from oncogenes (e.g. c-MYC) and tumour suppressors (e.g. PTEN) have been previously linked to cancer. We have performed a systematic computational analysis on recently identified non-AUG initiation-derived N-terminal extensions of cancer-associated proteins, and we discuss how these extended proteoforms may acquire new oncogenic properties. We identified a loss of stability for the N-terminally extended proteoforms of oncogenes TCF-4 and SOX2. Furthermore, we discovered likely functional short linear motifs within the N-terminal extensions of oncogenes and tumour suppressors (SOX2, SUFU, SFPQ, TOP1 and SPEN/SHARP) that could provide an explanation for previously described functionalities or interactions of the proteins. In all, we identify novel cases where PANTs likely show different localization, functions, partner binding or turnover rates compared to the annotated proteoforms. Therefore, we propose that alterations in the stringency of translation initiation, often seen under conditions of cellular stress, may result in reprogramming of translation to generate novel PANTs that influence cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741ec3f780294355c033fd7132d60244ed1b970e" target='_blank'>
              The implication of non-AUG-initiated N-terminally extended proteoforms in cancer
              </a>
            </td>
          <td>
            Rita Pancsa, D. E. Andreev, Kellie Dean
          </td>
          <td>2025-04-25</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="SMC5/6 is unique amongst the Structural Maintenance of Chromosomes (SMC) complexes in its ability to repress transcription from extrachromosomal circular DNA (ecDNA), including viral genomes and plasmids. Previously, we showed that human SMC5/6 is regulated by two mutually exclusive subcomplexes—SIMC1-SLF2 and SLF1/2—the counterparts of yeast Nse5/6 (Oravcová, eLife, 2022). Notably, only SIMC1-SLF2 recruits SMC5/6 to SV40 Large T antigen (LT) foci in PML nuclear bodies (PML NBs), suggesting that these regulatory subcomplexes direct distinct roles of SMC5/6 on chromosomal versus ecDNA. However, their roles in plasmid repression remain unclear. Here, we demonstrate that SMC5/6-mediated repression of plasmid transcription depends exclusively on SIMC1-SLF2, whereas SLF1/2 is dispensable. Reinforcing its specialized role in ecDNA suppression, SIMC1-SLF2 does not participate in SMC5/6 recruitment to chromosomal DNA lesions. We further show that plasmid silencing requires a conserved interaction between SIMC1-SLF2 and SMC6, mirroring the functional relationship observed between yeast Nse5/6 and Smc6. As for viral silencing, plasmid repression depends on the SUMO pathway; however, unlike viral silencing, it does not require PML NBs. Additionally, we find that LT interacts with SMC5/6 and increases plasmid transcription to levels observed in SIMC1-SLF2-deficient cells–echoing the antagonistic roles of HBx (HBV) and Vpr (HIV-1) in viral genome repression. These findings expand the paradigm of viral antagonism against SMC5/6-mediated silencing, positioning LT as a novel player in this evolutionary tug-of-war.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d38f987b8e2b4050f00463fe09e763117f64f73" target='_blank'>
              SMC5/6-Mediated Plasmid Silencing is Directed by SIMC1-SLF2 and Antagonized by LT
              </a>
            </td>
          <td>
            Martina Oravcová, Minghua Nie, Takanori Otomo, M. N. Boddy
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/879969a6652e4d8acc441456b7bcb05f2b8a551c" target='_blank'>
              Abstract 3345: Application of cGAS-immunopositive micronuclei as an indicator of tumor aggressiveness in liposarcoma
              </a>
            </td>
          <td>
            Kiyota Kato, Toru Motoi, Yu Naito, Masumi Ogawa, Chihiro Onda, Hideo Arai, Shin-ichiro Horiguchi, Toshihide Hirai, T. Hishima
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f151aed2abf27a7b67be8871fbd4b51979dcabff" target='_blank'>
              Abstract 1485: BRCA2 haploinsufficiency leads to mitotic DNA synthesis at common fragile sites in human mammary epithelial cells
              </a>
            </td>
          <td>
            Kavya Vipparthi, Ariana Bellare, Mihriban Karaayvaz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T. Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT We investigated the presence of viral DNA and RNA in cutaneous squamous cell carcinoma (cSCC) tumor and normal tissues from nine individuals with a history of hematopoietic stem cell transplantation (HCT). Microbiome quantification through DNA and RNA sequencing (RNA-seq) revealed the presence of 18 viruses in both tumor and normal tissues. DNA sequencing (DNA-seq) identified Torque teno virus, Saimiriine herpesvirus 1, Merkel cell polyomavirus, Human parvovirus B19, Human gammaherpesvirus-4, Human herpesvirus-6, and others. RNA-seq revealed additional viruses such as Tobamovirus, Pinus nigra virus, Orthohepadnavirus, Human papillomavirus-5, Human herpesvirus-7, Human gammaherpesvirus-4, Gammaretrovirus, and others. Notably, DNA-seq indicated that tumor samples exhibited low levels of Escherichia virus in three out of nine subjects and elevated levels of Human gammaherpesvirus-4 in one subject, while normal samples frequently contained Gammaretrovirus and occasionally Escherichia virus. A comparative analysis using both DNA- and RNA-seq captured three common viruses: Abelson murine leukemia virus, Murine type C retrovirus, and Human gammaherpesvirus-4. These findings were corroborated by an independent data set, supporting the reliability of the viral detection methods utilized. The study provides insights into the viral landscape in post-HCT patients, emphasizing the need for comprehensive viral monitoring in this vulnerable population. IMPORTANCE This study is important because it explores the potential role of viruses in the development of cSCC in individuals who have undergone allogeneic HCT. cSCC is common in this population, particularly in those with chronic graft-versus-host disease on long-term immunosuppression. By using advanced metagenomic and metatranscriptomic next-generation sequencing, we aimed to identify viral pathogens present in tumor and adjacent normal tissue. The results could lead to targeted preventive or therapeutic interventions for these high-risk people, potentially improving their outcomes and management of cSCC. This study is important because it explores the potential role of viruses in the development of cSCC in individuals who have undergone allogeneic HCT. cSCC is common in this population, particularly in those with chronic graft-versus-host disease on long-term immunosuppression. By using advanced metagenomic and metatranscriptomic next-generation sequencing, we aimed to identify viral pathogens present in tumor and adjacent normal tissue. The results could lead to targeted preventive or therapeutic interventions for these high-risk people, potentially improving their outcomes and management of cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56b53109651f083205e0fc320c14b9f0267e185" target='_blank'>
              Mining metagenomes and metatranscriptomes unveils viruses associated with cutaneous squamous cell carcinoma in hematopoietic stem cell transplant recipients
              </a>
            </td>
          <td>
            Jana K Dickter, Yuqi Zhao, Vishwas Parekh, Huiyan Ma, Badri G Modi, Wai-Yee Li, S. Armenian, Xiwei Wu, Farah R Abdulla
          </td>
          <td>2025-03-31</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c16d5ebf6f1d5ea4dbccbf5a0178380f13bbf886" target='_blank'>
              Abstract 7429: Illuminating the dark genome in cancer using artificial intelligence
              </a>
            </td>
          <td>
            M. D. M. Álvarez-Torres, Xi Fu, Raul Rabadan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43e1e965f05b28c3994c4e8f0fb6a1f9448be497" target='_blank'>
              Abstract 2856: Pks+ E. coli trigger intestinal stem cell plasticity and early onset colorectal cancer
              </a>
            </td>
          <td>
            Stefanie Gerstberger, Melissa Lumish, S. Hartner, Farheen Shah, Seongmin Choi, Krystal Lum, Christopher Cowley, A. Luthra, Qingwen Jiang, Hyung Jun Woo, Ahmed Mahmoud, H. Walch, Asha Saxena, Tavis J. Reed, A. Cercek, R. Yaeger, Ileana Cristea, Andrew McPherson, F. Sánchez-Vega, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377a4743e9ceeda080173ff3bd910fc589bc9196" target='_blank'>
              Abstract 5890: Personalized detection of circulating cell-free tumor DNA in gynecologic cancer
              </a>
            </td>
          <td>
            Andrea B. Moffitt, J. Kendall, Joan Alexander, A. Stepansky, A. Kapedani, Zi-Hua Wang, D. Levy, Kenny Ye, A. Krieger, Marina Frimer, Gary L. Goldberg, Michael Wigler
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/533adcd1257075cd22692e86abacf9c16278afc3" target='_blank'>
              Abstract 4284: Determining selectivity of MYC inhibitors via transcriptional monitoring of sentinel genes
              </a>
            </td>
          <td>
            Susanne Stockinger, Sabine Hartl, Manfred Koegl, Johannes Zuber, Matthias Samwer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b13792be21e8e149fad24f47b79b2c67bef2640" target='_blank'>
              The protein deacetylase HDAC10 controls DNA replication in malignant lymphoid cells.
              </a>
            </td>
          <td>
            Andreas O Mieland, Giuseppe Petrosino, Mario Dejung, Jia-Xuan Chen, Amitkumar Fulzele, F. Mahmoudi, J.-W. Tu, Al-Hassan M. Mustafa, Yanira Zeyn, Christoph Hieber, Matthias Bros, T. Schnöder, F. Heidel, Sara Najafi, Ina Oehme, Ilse Hofmann, Mike Schutkowski, Sebastian Hilscher, Christian Kosan, F. Butter, S. Bhatia, W. Sippl, Oliver H. Krämer
          </td>
          <td>2025-04-29</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b17c8f4c66aab3898b741804dfbd546dbdd92394" target='_blank'>
              Abstract 6825: Novel therapeutic strategies targeting MECOM rearranged AMLs discovered by unbiased drug screen
              </a>
            </td>
          <td>
            Christine E. Birdwell, W. Fiskus, Christopher P. Mill, T. Kadia, Naval G Daver, C. Dinardo, K. Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, R. K. Shamanna, Michele Ceribelli, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="BACKGROUND
Proline-rich transmembrane protein 4 (PRRT4) has been infrequently studied, with limited literature suggesting its potential as a prognostic marker for gastric cancer. This study aims to investigate the prognostic value of the PRRT4 gene in pan-cancer.


METHODS
We acquired and analyzed data from several platforms, including The Cancer Genome Atlas (TCGA), Genotype Tissue Expression Project (GTEx), Cancer Cell Line Encyclopedia (CCLE), cBioPortal, HPA, and TIMER 2.0. In addition, we have further analyzed the data using multivariate analyzes and RT-qPCR. In vitro experiments were performed to detect the proliferation and apoptosis of AML cells before and after PRRT4 knockdown.


RESULTS
PRRT4 exhibited low expression in 10 types of cancers and high expression in 3 types, and this expression was significantly correlated with tumor stage, age, and gender across various cancer types. PRRT4, identified as a potential independent prognostic factor for overall survival (OS) in several cancers including LAML, PAAD, SKCM, STAD, THYM, and UVM, and exhibited a high frequency of mutation in UCEC. Moreover, PRRT4 was found to be correlated with DNA methylation and immune infiltration in various cancers. Ultimately, in the multivariate analysis model, PRRT4 was discerned as an independent prognostic biomarker for AML, predicated on the statistics based from our institution. After PRRT4 knockdown, the proliferation ability of THP1 cells was significantly enhanced, and the apoptosis ratio was significantly decreased.


CONCLUSION
PRRT4 may serve as a potential therapeutic target and prognostic marker for various malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66047aa9da0269c010edbf7b80afbec9051b4fd1" target='_blank'>
              Pan-cancer analysis reveals PRRT4 is a potential prognostic factor of AML.
              </a>
            </td>
          <td>
            Wenqiong Xiang, Fangjun Chen, Haoyu Zhou, Guilin Ren, Guangliang Qiang, Li Wang
          </td>
          <td>2025-04-25</td>
          <td>Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA repair is a hierarchically organized, spatially and temporally regulated process involving numerous repair factors that respond to various types of damage. Despite decades of research, the mechanisms by which these factors are recruited to and depart from repair sites have been a subject of intrigue. Recent advancements in the field have increasingly highlighted the role of phase separation as a critical facilitator of the efficiency of DNA repair. This review emphasizes how phase separation enhances the concentration and coordination of repair factors at damage sites, optimizing repair efficiency. Understanding how dysregulation of phase separation can impair DNA repair and alter nuclear organization, potentially leading to diseases such as cancer and neurodegenerative disorders, is crucial. This manuscript provides a comprehensive understanding of the pivotal role of phase separation in DNA repair, sheds light on the current research, and suggests potential future directions for research and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c72f1d4a9e08095f98c1ef277b37623b6195f2e8" target='_blank'>
              Phase separation in DNA repair: orchestrating the cellular response to genomic stability
              </a>
            </td>
          <td>
            Juxin Deng, Zhaoyang Du, Lei Li, Min Zhu, Hongchang Zhao
          </td>
          <td>2025-05-02</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98cacc886578df29800eafabd86208b4844205b6" target='_blank'>
              The intratumoral microbiota heterogenicity is related to the prognosis and tumorigenesis of cervical cancer
              </a>
            </td>
          <td>
            Yi Guo, Yuhang Xiao, Changyi Zhang, Ying Wang, Guang-xu Cao, Ka Yu Tse, Zhiqiang Han, Fang Li, Yong Zhi
          </td>
          <td>2025-05-13</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The formation of purinosomes, dynamic complexes involved in de novo purine biosynthesis, has been recognized as a critical process for cell growth. Although their upregulation in cancer cells suggests their potential as a therapeutic target, the specific role of purinosomes in hepatocellular carcinoma (HCC) remains uncertain. The purinosome score was found to have prognostic value. Enrichment analyses indicated a connection between purinosome-related genes and cell cycle regulation. Moreover, our research has demonstrated a correlation between the upregulation of genes associated with purinosomes and the enhanced formation of purinosomes in Huh-7 cells. Pyrimethamine has been identified as a promising therapeutic option for targeting purinosome to exert anti-cancer effects. Furthermore, the purinosome score exhibited an positive relationship with the response to immunotherapy. It may guide the stratification of liver cancer patients and screen for populations that may benefit from immunotherapy. This study examines the prognostic and predictive value of purinosome in liver cancer, suggesting that targeting purinosome formation with pyrimethamine or immunotherapy could benefit patients with high purinosome scores. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02366-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0192a6b9a738b5258e86b28eb806c7d763631cb" target='_blank'>
              Purinosomes as a therapeutic target in hepatocellular carcinoma: insights and opportunities
              </a>
            </td>
          <td>
            Zhen Lin, Jia-Wei Long, Ming-chun Zhao, Pin Guo, Jin Wen, Guangdi Chen
          </td>
          <td>2025-04-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Ovarian Cancer (OC) is a gynecological malignant tumor with an extremely high mortality rate, seriously endangering women’s health. Due to its insidious clinical manifestations, most patients are diagnosed in the advanced stage of the disease. The currently clinically relied CA125 has limited specificity for the early diagnosis of ovarian cancer. Hence, identifying new promising biomarkers is crucial for the early screening, diagnosis, and treatment of ovarian cancer. Based on differential expression analysis, WGCNA and survival analysis, we identified a centromere-associated gene, WDR62, which is highly expressed in ovarian cancer and highly correlated with ovarian cancer, as well as the poor prognosis of ovarian cancer patients with high expression, suggesting that WDR62 may be a potential biomarker for ovarian cancer. Previous studies have shown that WDR62 is closely associated with the occurrence, development and prognosis of a variety of tumors. However, its role in ovarian cancer has not been studied in depth. Methods Using combined TCGA and GTEx datasets from the UCSC database, along with WGCNA, and survival analysis, WDR62 was identified as a potential biomarker. GEPIA2 database, GEO database, qRT-PCR, and Western blot proved the expression of WDR62. Enrichment analysis, cell transfection, Western blots and CCK8 demonstrated the regulatory mechanism of WDR62, and the detailed mechanism of WDR62 involvement in the occurrence and development of ovarian cancer was predicted by interaction analysis and correlation analysis. Results WDR62 was highly expressed in ovarian cancer cells compared to normal ovarian epithelial cells, both at the RNA and protein levels. Patients with high WDR62 expression had a poor survival prognosis. Upon WDR62 knockdown, the expression of cell cycle-related proteins CDK1 and C-Myc decreased in ovarian cancer cells, and the cell proliferative capacity was decreased. Based on bioinformatic analysis, it was hypothesized that WDR62 might mediate the JNK signaling pathway by interacting with MAPK8, thus affecting ovarian cancer progression through cell cycle regulation. Conclusions WDR62 is overexpressed in ovarian cancer and is closely related to the prognosis of ovarian cancer patients. WDR62 promotes ovarian cancer progression by regulating the cell cycle and may influence its development through interaction with MAPK8 to mediate the JNK signaling pathway. These findings suggest that WDR62 could be a potential target for the early screening, diagnosis, and treatment of ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00444-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376041204294ddbace31c696e08e81456f3e0e9d" target='_blank'>
              WDR62 affects the progression of ovarian cancer by regulating the cell cycle
              </a>
            </td>
          <td>
            Yuqi Yang, Wanting Jing, Lingqi Zhang, Yuhang Zhang, Ying Shang, Ye Kuang
          </td>
          <td>2025-05-14</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e60b89ff761a466f37436f2b2c0b429bc7460b9" target='_blank'>
              VN1K: a genome graph-based and function-driven multi-omics and phenomics resource for the Vietnamese population
              </a>
            </td>
          <td>
            Trang T. H. Tran, T. Hoang, Mai H. Tran, Nguyen T. Nguyen, D. Nguyen, Tien M. Pham, Nam N. Nguyen, G. Vu, Vinh C. Duong, Quang T. Vu, T. K. Nguyen, Sang V. Nguyen, Hien Q. Vu, Trang M. Nguyen, Toan Dang, H. Nguyen, Tuan Do, Cuong Le, Hung T. T. Nguyen, Nam Q. Le, Quan Nguyen, Linh T. Le, Thang Pham, D. Vu, Huong T. T. Le, Tho D. Ngo, Liem T. Nguyen, Yen Hoang, D. X. Dao, G. H. Phan, Chi Trung Ha, Hung N. Luu, Minh Dao, Vinh Le, Ly Le, Duong Thuy Nguyen, T. Tran, Quan Nguyen, Duc-Hau Le, Van H. Vu, Nam S. Vo
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Many plant species can develop embryos from somatic cells without fertilization. During this process, known as somatic embryogenesis, changes in the DNA methylation patterns are characteristic of reprogramming somatic cells into an embryogenic state. However, the underlying mechanisms connecting DNA methylation and activating totipotency-regulating genes have remained largely unknown. Here, we show that during somatic embryogenesis induced by overexpressing the totipotency-regulating transcription factor LEAFY COTYLEDON2 (LEC2) in Arabidopsis, CHH hypermethylation is deposited by the LEC2-activated RNA-directed DNA methylation (RdDM) pathway. A reader complex composed of SU(VAR)3-9 HOMOLOGS (SUVH) and its chaperone SUVH-INTERACTING DNAJ DOMAIN-CONTAINING PROTEIN (SDJ) binds to the CHH hypermethylated regions and recruits AT-HOOK MOTIF CONTAINING NUCLEAR LOCALIZED (AHL) chromatin modification proteins to increase chromatin accessibility, resulting in the transcriptional activation of totipotency-regulating genes. Our work reveals a molecular framework of how epigenetic modifications mediate somatic cell reprogramming, offering a pathway toward enhancing somatic embryogenesis in agricultural regeneration biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70eee015496dfcf48bad094490ca17c7c91314ba" target='_blank'>
              LEC2 induces somatic cell reprogramming through epigenetic activation of plant cell totipotency regulators
              </a>
            </td>
          <td>
            Jing Peng, Qi Zhang, Li Ping Tang, Biao Jie Xu, Thomas Laux, Xian Sheng Zhang, Yingchun Su
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22322ec3b603c7144cae90723f090b99de790ee7" target='_blank'>
              Reference-guided genome assembly at scale using ultra-low-coverage high-fidelity long-reads with HiFiCCL
              </a>
            </td>
          <td>
            Zhongjun Jiang, Weihua Pan, Runtian Gao, Heng Hu, Wentao Gao, Murong Zhou, Yu-Hang Yin, Zhipeng Qian, Shuilin Jin, Guohua Wang
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2213c097bc0bf3842b91050d879626cc38ef3b6" target='_blank'>
              Abstract 1488: DNA polymerase kappa slows replication fork speed by promoting fork reversal in glioblastoma
              </a>
            </td>
          <td>
            Reham S. Sewilam, Megan R. Reed, R. Eoff
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029a236a2721717e13973bdc5e5b49afdd737065" target='_blank'>
              Abstract 2624: Aneuploidy induction mechanism in Ewing sarcoma
              </a>
            </td>
          <td>
            Priyani H. Hapugaswatta, M. Azuma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/304d6e4c38ec9dd6d17c47e93a7fbe83d1abb83b" target='_blank'>
              OGT prevents DNA demethylation and suppresses the expression of transposable elements in heterochromatin by restraining TET activity genome-wide.
              </a>
            </td>
          <td>
            Hugo Sepulveda, Xiang Li, Leo J. Arteaga-Vazquez, Isaac F. López-Moyado, Melina Brunelli, Lot M Hernández-Espinosa, Xiaojing Yue, J. C. Angel, Caitlin Brown, Zhen Dong, Natasha Jansz, Fabio Puddu, Aurélie Modat, Jamie Scotcher, Páidí Creed, Patrick H Kennedy, Cindy Manriquez-Rodriguez, Samuel A Myers, Robert Crawford, Geoffrey J Faulkner, Anjana Rao
          </td>
          <td>2025-03-28</td>
          <td>Nature structural & molecular biology</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2daedc5d36f880d0e2fc581751bb206d1ca1d20" target='_blank'>
              Recurrent spontaneous miscarriages from sperm after ABVD chemotherapy in a patient with Hodgkin's lymphoma: sperm DNA and methylation profiling.
              </a>
            </td>
          <td>
            G. Lecuyer, A. Rolland, A. Neyroud, Bertrand Evrard, Nathan Alary, Clemence Genthon, Nathalie Dejucq-Rainsford, Célia Ravel, Jessika Moreau, N. Moinard, Mohamed Hadi Mohamed Abdelhamid, Christophe Klopp, Louis Bujan, Frédéric Chalmel
          </td>
          <td>2025-04-15</td>
          <td>Asian journal of andrology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ffb5861f3091086b8e6d39be2fc47951f44015b" target='_blank'>
              Cohesins: Crossroad Between Cornelia de Lange Spectrum and Cancer Predisposition.
              </a>
            </td>
          <td>
            Laura Rigotti, Stefano Rebellato, Antonella Lettieri, Silvia Castiglioni, M. Mariani, Simona Totaro, C. Saitta, C. Gervasini, G. Fazio, Valentina Massa, G. Cazzaniga, Angelo Selicorni
          </td>
          <td>2025-04-05</td>
          <td>American journal of medical genetics. Part A</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f55512a8c15d7d2448765df1044f7457324d44a" target='_blank'>
              Abstract 2488: Employing RNA base editors to engineer personalized tumor-specific neoantigens
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Netanel Landesman, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>