<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-26 09:43:44 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>47</td>
          <td>107</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>93</td>
          <td>141</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>126</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>406</td>
          <td>141</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>344</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract In this Perspective article, we follow the journey of a gene that breaks free from its stringent chromosomal transmission dynamics to become an independently multiplying agent on so-called extrachromosomal circular DNA (ecDNA/eccDNA). We discuss how the release of a gene from its chromosomal anchor can affect its dosage, regulatory context, and potential evolutionary trajectory before examining the phenotypic implications for unicellular and multicellular eukaryotes that transmit genes on circular DNA. We also briefly explore the fundamental difference between circularized genes in flowering plants (angiosperms) and mammals (primarily cancer) concerning somatic and germline inheritance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2 The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent mini-chromatin. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template is mediated by decreased efficacy of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS with LANA as similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA TR binding establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memories from prior activation, determines the degree of KSHV lytic replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43595752cf1e93615641d0b346a78e9c25448ab3" target='_blank'>
              KSHV TR deletion episomes uncover enhancer–promoter dynamics in gene regulation
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, J. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-12-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Aim: Prostate cancer features profound transcriptional dysregulation within the androgen receptor (AR) signaling axis. The pioneer factor FOXA1, which facilitates AR binding to chromatin, is recurrently altered in 10-40% of tumors. Recent studies classify FOXA1 mutations as Class 1 Wing 2 mutations, which enhance AR-dependent tumorigenesis, and Class 2 C-terminal truncations, which promote lineage plasticity and therapy resistance. The interplay of FOXA1 alterations with TMPRSS2-ERG fusions and PROX1 remains incompletely understood. Materials and Methods: Data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) cohort (n=492) were analyzed via UCSC Xena and cBioPortal. FOXA1 mutations were categorized following Eyunni et al. Copy number was assessed by log2(tumor/normal) ratios. Mutual exclusivity and co-occurrence were evaluated using Fisher’s exact test with false-discovery-rate correction. Associations between FOXA1 status and genomic instability were assessed using the fraction genome altered (FGA) metric. Results: FOXA1 was broadly expressed, with subsets showing elevation. Class 1 mutations localized to the Wing 2 region, while Class 2 truncations clustered in the C-terminal domain. Copy number changes were infrequent, indicating mutation-driven reprogramming as the main oncogenic mechanism. TMPRSS2 and ERG strongly co-occurred (log2 OR >3, q<0.001), whereas FOXA1 was mutually exclusive with both TMPRSS2 and ERG (q<0.001). Although FOXA1 alterations showed no significant Pearson correlation with FGA (r=−0.01, p=0.76), a moderate Spearman correlation (ρ=0.52, p<0.001) suggested enrichment in genomically unstable tumors. Conclusion: FOXA1 defines a major oncogenic axis in prostate cancer, distinct from TMPRSS2-ERG fusion and PROX1 induction. Class 1 and 2 FOXA1 mutations drive alternative transcriptional programs leading to therapy resistance, highlighting FOXA1 as a critical biomarker and target for chromatin-directed interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962282da85200138085414df96f69e7ea38e7831" target='_blank'>
              FOXA1 Alterations in Prostate Cancer: Expression, Mutation Classes, and Copy Number Changes
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450603a806f69ebb4194f3b7b3ab19ed0a5b5513" target='_blank'>
              A holocentric pangenome links karyotype evolution to meiotic recombination
              </a>
            </td>
          <td>
            Meng Zhang, Stefan Steckenborn, Marco Castellani, Laia Marín-Gual, A. S. Câmara, Laura A. Robledillo, Letícia Maria-Parteka, N. Sargheini, Magdalena Marek, Eduardo Chacón-Madrigal, Andrés Gatica Arias, L. P. Felix, William W. Thomas, B. Huettel, A. Pedrosa‐Harand, John T. Lovell, André L. L. Vanzela, Aurora Ruiz-Herrera, 
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A. Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common form of primary liver cancer and has limited therapeutic options. Epigenetic dysregulation plays a crucial role in hepatocarcinogenesis; however, its impact on cis-regulatory elements (CREs) and retrotransposons remains underexplored. Here, we investigated the epigenetic changes underlying the aberrant CRE and retrotransposon activity in HCC. We show that focal DNA hypomethylation of these elements is associated with transcriptional reprogramming. Notably, we uncover a dual regulatory mechanism for GPC3, a key diagnostic biomarker and immunotherapeutic target in HCC. This requires the concomitant reactivation of a fetal liver super-enhancer (SE) and DNA hypomethylation of CpG islands. Furthermore, DNA methylation loss drives cryptic activation of retrotransposons, some of which exhibit prognostic potential. Intriguingly, we identify a HERVE-int-derived long non-coding RNA that shows higher expression in patients with more aggressive tumors, poorer disease outcomes, and is correlated with molecular signatures associated with improved response to immunotherapy. Collectively, our findings reveal widespread epigenomic dysregulation of CREs and retrotransposons in HCC, highlighting new potential therapeutic strategies and biomarkers. Integrative epigenomic profiling of hepatocellular carcinoma reveals dual regulation of the immunotherapeutic target GPC3 and identifies retrotransposons as potential biomarkers for prognosis and immunotherapy response prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed2d6556ed99cbdaa0f49af5ea87b73a7cb6583" target='_blank'>
              Multi-omic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potential
              </a>
            </td>
          <td>
            Clooney C Y Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y J Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B S Lai, K. Yip, Alfred S Cheng, Danny Leung
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0f98bd86315e8e5aa3b48daf84b94980e7dd61" target='_blank'>
              Dynamic Supercoiling Sponsors Transcription Amplification by MYC
              </a>
            </td>
          <td>
            R. Jha, Fedor Kouzine, Bo Wang, James D. Phelan, Subhendu K Das, Brian A Lewis, D. Levens
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/308ec9e744b73dd3ecd51f7b2385a3a46667c2cb" target='_blank'>
              Pangenome analysis reveals the evolutionary dynamics of repeat-based holocentromeres
              </a>
            </td>
          <td>
            Piotr Włodzimierz, Estela Perez-Roman, A. S. Câmara, Laura A. Robledillo, Gokilavani Thangavel, Meng Zhang, Jacob González Isa, L. M. Parteka, Marco Castellani, B. Huettel, André L. L. Vanzela, Ian R. Henderson, Alexandros Bousios, 
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078474078b9a084c2455a85f367e87d76cce9bc4" target='_blank'>
              Lineage-determining transcription factors EBF1 and TCF1 shape chromatin fibre folding
              </a>
            </td>
          <td>
            R. Faryabi, Yeqiao Zhou
          </td>
          <td>2025-12-10</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. Despite improved therapies, refractory and relapsed ALL remain the leading cause of cancer-related mortality in children. There is a need for accessible biomarkers for frequent, minimally invasive disease monitoring and prompt intervention. MicroDNA is a novel extrachromosomal DNA that preferentially originates from gene segments with high transcriptional activity and/or increased chromatin accessibility. We investigated whether microDNA-producing genes repertoire changes in a disease-dependent manner. We characterized microDNAs in 52 paired bone marrow (BM) and plasma samples from pediatric patients with ALL at diagnosis, relapse, and remission. No difference in the length or number of microDNA was noted across stages, but comparative analysis of microDNA profiles led to the identification of microDNA gene panels associated with active disease. The relative distribution of these genes was significantly different from that expected by chance (P < 0.0001). Analyses of BM samples identified a signature comprising 289 distinct microDNA-producing genes present in multiple patients at diagnosis and relapse but absent in remission. The best biomarker candidates were 11 microDNA-producing genes identified also in plasma samples at diagnosis and overrepresented in patients who relapsed (P = 0.006). MicroDNA from the same genes was confirmed in relapse plasma samples. All signature genes are known to be involved in cancer proliferation or drug response. MicroDNA seems to be a candidate for a novel class of biomarkers for ALL, with the potential to improve precision diagnostics, particularly through their identification in plasma samples. Further validation in an independent cohort of patients is warranted. Significance: Despite high cure rates, 10% to 15% of pediatric patients with ALL experience relapse. We identified a plasma-detectable microDNA signature from 11 genes that persists from diagnosis through relapse but disappears in remission. These findings demonstrate the potential of microDNA profiles as prognostic biomarkers in pediatric ALL, enabling noninvasive monitoring of disease status and risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9503e4bfe97b36cabeed806790d4fca481f09664" target='_blank'>
              Extrachromosomal microDNA Signature as a Candidate Biomarker in Pediatric Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Ivan Brukner, V. Gagné, Alex Richard-St-Hilaire, P. Tremblay-Dauphinais, Claire Fuchs, Henrique Bittencourt, Teodor Veres, D. Sinnett, Maja Krajinovic
          </td>
          <td>2025-12-11</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Structural variations in cancer predisposition genes significantly contribute to the pool of pathogenic variants underlying heritable cancer susceptibility. In this report, we describe and functionally characterize a novel transposon-mediated germline pathogenic insertion identified within the coding region of the BRCA1 gene. The inserted sequence comprises the entire processed transcript of RPL18A, a ribosomal protein-coding gene, and may have been generated through a target-primed reverse transcription event. Robust molecular testing confirmed the heritability of the variant and provided evidence of its correlation with the observed phenotype. This is the first documented case of a germline cancer susceptibility variant arising through this unique mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab6faf5664bd66e2e317ebc80d8ac230de46d4a" target='_blank'>
              Processed Transcript Insertion as a Novel Germline Mutational Mechanism in BRCA1-Associated Hereditary Breast Cancer
              </a>
            </td>
          <td>
            A. Bozsik, H. Butz, V. Grolmusz, Petra Nagy, T. Pócza, Erika Tóth, E. Csernák, Attila Patócs, János Papp
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pedigree analysis remains the gold standard for rare disease diagnostics, yet whole genome sequencing studies typically omit critical regions like centromeres, telomeres, and acrocentric chromosome p-arms. Here, we present telomere-to-telomere (T2T) reference genomes for four self-identified African American individuals of admixed ancestry spanning three generations. Our parent-of-origin assigned, chromosome-level assemblies revealed precise meiotic recombination breakpoints in previously inaccessible regions, including recombination events across acrocentric and subtelomeric sequences. Centromeric regions were highly stable, with multi-megabase arrays inherited intact across three generations, while the position of kinetochore assembly sites remained consistent and predominantly associated with the p-arm proximal region. The relative lengths of telomeres on individual chromosomes were maintained across generations. Using a targeted rDNA assembly approach, we reconstructed a complete megabase-scale ribosomal DNA (rDNA) array corresponding to the paternal chromosome 14. This openly available pedigree provides a benchmark dataset for studying recombination and genetic and epigenetic variation across the complete genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56491af219e3e15ccd2642a4c21320991315613" target='_blank'>
              Complete genomes of a multi-generational pedigree to expand studies of genetic and epigenetic inheritance
              </a>
            </td>
          <td>
            Monika Cechova, T. Potapova, A. Rechtsteiner, Glenn Hickey, Rebecca Serra Mari, Mira Mastoras, Julian Menendez, Nikol Poláková, Prajna Hebbar, Fedor Ryabov, Hailey Loucks, Aljona Groot, Tomáš Pavlík, Mobin Asri, Shihua Dong, Stephanie M. Yan, Julian K. Lucas, Steven J. Solar, M. Borchers, Mark Mattingly, S. McKinney, M. Krátká, Catherine Mikhailova, Ondřej Hanák, Sohinee Tiffany Saha, Emily Xu, D. Antipov, S. Koren, Jordan M. Eizenga, B. McNulty, Joshua M.V. Gardner, T. Hillaker, I. Violich, Christopher Markovic, S. Kruglyak, Shawn Levy, Trevor Wolf, Matthew W. Mitchell, Laura Scheinfeldt, Haoyu Cheng, I. Alexandrov, R. McCoy, B. Paten, A. Phillippy, J. Zook, J. Gerton, Robert S. Fulton, N. Stitziel, Ting Wang, T. Marschall, C. Greider, K. Miga
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Centromeres are essential for accurate chromosome segregation during cell division, yet their highly repetitive sequence has historically hindered their complete assembly and characterization. Consequently, the full spectrum of centromere diversity across individuals, populations, and evolutionary contexts remains largely unexplored. Here, we address this gap in knowledge by assembling and characterizing 2,110 complete human centromeres from a diverse cohort of individuals representing 5 continental and 28 population groups. By developing a novel suite of bioinformatic tools tailored for centromeric regions, we uncover previously unknown variation within centromeres, including 226 novel centromere haplotypes and 1,870 new α-satellite higher-order repeat (HOR) variants. We find that mobile element insertions are present in 30% of centromeres, with chromosome 16 harboring Alu elements within the kinetochore site at an 11-fold higher frequency than expected. While most centromeres have a single kinetochore site, 6% of them have di-kinetochores, and <<1% have tri-kinetochores, which we confirm with long-read CENP-A CUT&RUN, DiMeLo-seq, and multi-generational inheritance. We further show that the position of the kinetochore is not random and is, instead, closely associated with the underlying sequence and structure of the centromere. To understand the nature of evolutionary change, we compared 2,110 complete human centromeres to 5,747 complete centromeres recently assembled from the Human Pangenome Reference Consortium. We show that centromeres have a >50-fold variation in mutation rate, with the most rapidly mutating centromeres on chromosome 1 and the slowest mutating centromeres on chromosome Y. Additionally, a subset of centromeres show evidence of introgression from archaic hominins, shaping their sequence, structure, and evolutionary history. We validate these centromere mutation rates in a four-generation family, spanning 28 family members and 483 accurately assembled centromeres, and show that the kinetochore site is the most rapidly mutating region in the centromere, with twofold more single-nucleotide variants than the rest of the centromeric α-satellite HOR array on average. We propose a model that reveals an ‘arms race’ between centromeric sequence and proteins, with frequent mutations within the site of the kinetochore that lead to changes in genetic and epigenetic landscapes and, ultimately, rapid evolution of these critically important regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="


 Genomic instability is a hallmark of cancer and contributes to disease progression through various mechanisms. DNA copy number variation (CNV) is one of the most significant consequences causing a phenomenon called aneuploidy. Aneuploidy is characterized by an imbalanced genome and is associated with lethal progression in prostate cancer. Currently, sequencing-based CNV analysis is widely used to identify cancer-specific aneuploidy that contribute to tumorigenesis, immune evasion, and disease progression. Performing low-pass whole genome sequencing (LP-WGS) on DNA extracted from bulk tissues or cell cultures, is a convenient approach to use for detecting aneuploidy and focal CNVs. However, conventional bulk LP-WGS methods average genomic signals across thousands of cells, often overlooking tumor heterogeneity and small populations with distinct CNV patterns. To address this limitation, we optimized a microfluid nanowell-based system to analyze aneuploidy patterns at a single-cell level using disassociated tissues and isolated nuclei from prostate samples. This pipeline is compatible with both preclinical models and clinical specimens and can be integrated with other multi-omics platforms to generate multimodal datasets for basic and translational research. By leveraging high-throughput single-cell sequencing technologies, our approach enables direct DNA CNV measurement with high sensitivity at a single-cell resolution. Ultimately, this approach aims to improve the resolution of CNV detection, enhance molecular stratification and enable multimodal data integration for better understanding of clinical prostate cancer samples.



 Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, William D. Figg, Gregory Chesnut, Matthew G. Vander Heiden, Xiaofeng A. Su. Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A060.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6642b028922ed7c8904563bae6731f81ce2d7a38" target='_blank'>
              Abstract A060: Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing
              </a>
            </td>
          <td>
            Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, , Duanduan Ma, Elise G. DeArment, Amina Ali, W. Figg, Gregory T. Chesnut, M. V. Vander Heiden, 
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte‐rich malignant tumours with few somatic mutations in protein‐coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte‐rich tissues, and then performed genome‐wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis‐regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour‐specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D‐loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour‐specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome‐wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Large-scale genomic rearrangements in cancer cells can lead to the creation of fusion genes which alter the expression or function of the partner genes. Some fusion genes act as tumour drivers and the use of kinase inhibitors in patients with specific fusions have led to breakthroughs in cancer therapy. The large clinical and scientific interest in fusions has led to the development of software that use RNA sequencing data to identify fusion transcripts. Unfortunately, fusion transcript callers output many predictions which lack underlying genomic rearrangements and large datasets with validated fusions are scarce. This paper and the accompanying Fusions4U web application present a resource of validated and annotated gene fusions for 328 cell lines from the Cancer Cell Line Encyclopedia. Predicted fusion transcripts from Arriba and STAR-Fusion were analysed with our published validation pipeline that uses matched whole-genome sequencing data to identify discordantly mapped read pairs and candidate genomic breakpoints that support genuine fusion events. This resulted in 8,753 and 2,244 validated fusion transcripts for Arriba and STAR-Fusion predictions, respectively, with 1,596 fusions common to both. Additional layers of annotation include alternative splicing of fusion transcripts, kinases, microRNA host genes, genes in the COSMIC Cancer Gene Census, as well as known fusion gene pairs from the Mitelman Database of Chromosome Aberrations and Gene Fusions in Cancer and the TumorFusions dataset. We furthermore analysed information about fusion genes together with cell line data from the PRISM drug repurposing screening as an example of how this dataset can be used. This resource can be used to design experiments for functional studies and drug development, alone or in combination with publicly available information for the Cancer Cell Line Encyclopedia cell lines. The large collection of validated fusion transcripts with candidate genomic breakpoints can also be used in development and evaluation of bioinformatic tools for fusion transcript detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1751293449b5956386c8f2f0dbda4aebaafaddec" target='_blank'>
              Fusions4U: a resource of validated and annotated gene fusions in 328 cancer cell lines
              </a>
            </td>
          <td>
            Arianna Alamshahi, Helena Persson
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is among the most prevalent malignancies. However, the regulatory networks involved in tumor occurrence and development are still poorly understood. Human endogenous retroviruses (HERVs), a class of transposable elements, have been implicated in the development and progression of various human cancers. This study presents the first comprehensive locus-specific profiling of the expression of HERV gene transcripts in rectal cancer, revealing significantly dysregulated HERVs. Analysis of data from three Gene Expression Omnibus data sets revealed 25 upregulated HERVs and 7 downregulated HERVs. Dysregulation of HERV6196, a type of HERVH, was validated through reverse transcription quantitative PCR and droplet digital PCR in cells and tissues. Additionally, HERV6196 promoted the proliferation, inhibited the apoptosis, enhanced the colony formation ability, and enhanced the migration capability of rectal cancer cells. Moreover, HERV6196 functioned as an enhancer, promoting the expression of neighboring genes and the development of CRC. In summary, the present results revealed that HERV6196 is involved in the pathogenesis of rectal cancer, indicating the potential contribution of dysregulated HERVs to the development and progression of CRC through gene expression modulation. IMPORTANCE The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis. The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8dcd97edf79418a2a506bad015f01e1c7fbd0b0" target='_blank'>
              HERV6196 as an enhancer with oncogenic potential in rectal cancer
              </a>
            </td>
          <td>
            Yi-Xiu Gan, Xin Jiang, Zhi-Yu Wang, Yi-Lin Yu, Ling-Dong Shao, Jianmin Wang, Jun-Xin Wu
          </td>
          <td>2025-01-06</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfde1f929fb1d5e109604b5adc00a4ea625430b8" target='_blank'>
              Localised activity of reverse gyrase at gene regulatory elements
              </a>
            </td>
          <td>
            Paul Villain, Vladislav Kuzin, Augustin Darennes-Degaugue, Florence Lorieux, Antoine Hocher, Romain Le Bars, Tobias Warnecke, Laura Baranello, Tamara Basta
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="DNA topology is critical for regulating transcription and maintaining cellular homeostasis. Z-DNA is a left-handed DNA helix in regions with high transcriptional activity. Its physiological function remains poorly understood. Here, we demonstrate that oncoprotein MYC induces the formation of Z-DNA by recruiting the chromatin remodeler FACT, independent of RNA Polymerase II activity. FACT facilitates Z-DNA formation by remodeling H2A/H2B dimers within intact nucleosomes. Additionally, the phosphorylation of FACT regulates its liquid-liquid phase separation, promoting its efficient recruitment to chromatin by MYC. Through a genome-wide analysis and characterization of engineered Z-DNA promoters, we found that Z-DNA directly facilitates the loading of RNA Polymerase II, thereby promoting transcriptional activity. This study elucidates the molecular mechanisms of Z-DNA dynamics and emphasizes its functional importance in transcriptional regulation, providing insights into the role of left-handed DNA structures in chromatin biology and MYC-driven cancer. Z-DNA forms at highly active genes, but its function is unclear. Here, the authors show that MYC recruits FACT to promote Z-DNA formation within nucleosomes, enabling RNA Polymerase II loading and enhancing transcription, revealing a mechanistic role for Z-DNA in MYC-driven regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa2c8a18db8f4cae4958cd346c31f66f592e3d3" target='_blank'>
              MYC drives left-handed Z-DNA formation to shape gene expression
              </a>
            </td>
          <td>
            Xinyi Wan, Haonan Fan, Zhuoning Li, Yixuan Du, Yu Liu, Weiwu Zeng, Zhongxing Sun, Juejia Shao, Junqi Jia, Yanjun Zhang, Dong Fang
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The dynamic three-dimensional (3D) organization of the human genome (the 4D nucleome) is linked to genome function. Here we describe efforts by the 4D Nucleome Project1 to map and analyse the 4D nucleome in widely used H1 human embryonic stem cells and immortalized fibroblasts (HFFc6). We produced and integrated diverse genomic datasets of the 4D nucleome, each contributing unique observations, which enabled us to assemble extensive catalogues of more than 140,000 looping interactions per cell type, to generate detailed classifications and annotations of chromosomal domain types and their subnuclear positions, and to obtain single-cell 3D models of the nuclear environment of all genes including their long-range interactions with distal elements. Through extensive benchmarking, we describe the unique strengths of different genomic assays for studying the 4D nucleome, providing guidelines for future studies. Three-dimensional models of population-based and individual cell-to-cell variation in genome structure showed connections between chromosome folding, nuclear organization, chromatin looping, gene transcription and DNA replication. Finally, we demonstrate the use of computational methods to predict genome folding from DNA sequence, which will facilitate the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function. The 4D Nucleome Project demonstrates the use of genomic assays and computational methods to measure genome folding and then predict genomic structure from DNA sequence, facilitating the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25233c096475ded164c99fc34db37f934f70f997" target='_blank'>
              The Genomic Architecture of Human DNA Replication Origins
              </a>
            </td>
          <td>
            Zhenhua Li, Wenjian Yang, Gang Wu, S. Yoshimura, C. Mullighan, Tracie C. Rosser, E. J. Leslie-Clarkson, Karen R. Rabin, P. J. Lupo, D. Teachey, Jun J. Yang
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c71eacb567a19cb84e29c03543a7fbd15b5975" target='_blank'>
              Negative Selection Maintains Grossly Altered but Broadly Stable Karyotypes in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            William Cross, S. Nowinski, George D. Cresswell, M. Mossner, Abhirup Banerjee, B. Lu, Marc J Williams, G. Vlachogiannis, Laura J Gay, Ann-Marie Baker, C. Kimberley, F. J. Whiting, Hayley L. Belnoue-Davis, Pierre Martinez, Maria Traki, Viola Walther, Kane Smith, J. Fernández-Mateos, Erika Yara-Romero, Erica A Oliveira, Salvatore Milite, G. Caravagna, Chela T James, G. Elia, A. Berner, Chang-Ho Ryan Choi, Pradeep Ramagiri, Ritika Chauhan, Nik Matthews, Jamie Murphy, Anthony Antoniou, Susan K Clark, M. Mitchison, J. Chin Aleong, Enric Domingo, Inma Spiteri, S. McDonald, Darryl Shibata, M. Laclé, L. Wang, Morgan Moorghen, Ian P.M. Tomlinson, Marco Novelli, Marnix Jansen, A. Watson, Nicola Valeri, Nicholas A. Wright, John Bridgewater, M. Rodriguez-Justo, C. Barnes, Hemant M Kocher, Simon J. Leedham, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2026-01-21</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d39ef025a1bb316bde2c6dad77b6828031b8946" target='_blank'>
              Functional characterization of the 9q34.13 locus identifies RAPGEF1 as modulating risk for melanoma and nevi via RAS activation
              </a>
            </td>
          <td>
            Rohit Thakur, Mai Xu, Alexandra M. Thornock, Joshuah Yon, Samuel Anyaso-Samuel, M. Lauss, Thomas Rehling, Linh Bui-Raborn, Hayley A. Sowards, Gerard Duncan, Lea Jessop, Timothy Myers, Raj Chari, Erping Long, K. Funderburk, Jinhu Yin, Rebecca Hennessey, Emory Hseih, Hannah Levin, Mitchell John Machiela, Tongwu Zhang, Goran Jonsson, D. T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Law, T. Andresson, Jiyeon Choi, Leonard I. Zon, Jianxin Shi, Kevin M. Brown
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The ciliate Oxytricha undergoes massive genome rearrangements during development to produce a functional nucleus from an encrypted zygotic genome. PIWI-interacting small RNAs protect DNA regions against deletion, but how that protective mark is established has been a mystery. Recently our lab discovered MTA1, a methyltransferase that catalyzes DNA N6-adenine (6mA) methylation. Both MTA1 and the Oxytricha Piwi protein, Otiwi1, are required for development, and Otiwi1 mutation eliminates 6mA signal. To examine the role of 6mA, we analyzed its genome-wide distribution across development in wild-type and MTA1 mutant backcrossed cells. We find specific and abundant enrichment on retained sequences, suggesting a protective role for this epigenetic mark. Furthermore, programmed retention of a DNA region that is normally deleted leads to accumulation of new 6mA marks on the ectopically retained DNA sequence. Together, these results suggest that piRNA-guided 6mA DNA methylation leads to protection of DNA sequences against deletion during nuclear differentiation. Highlights DNA N6-methyladenine accumulates on retained DNA regions during Oxytricha development. mta1 mutant backcrosses have disrupted DNA methylation and a developmental delay. DNA N6-adenine methylation during genome rearrangement requires the presence of Otiwi1. Programmed retention of a germline-limited region leads to developmental methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecfb36caac2df07b165f21a1abfc41a4ec2bd5d" target='_blank'>
              A PIWI protein-dependent DNA N6-adenine methylation pathway in Oxytricha protects genomic sequences from deletion
              </a>
            </td>
          <td>
            Margarita T. Angelova, Yi Feng, D. Villano, Erhan Aslan, , P. Esteve, Sagnik Sen, William E. Jack, S. Pradhan, L. Landweber
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, C. Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Non-homologous end joining (NHEJ) is a critical DNA double-strand break (DSB) repair pathway that operates throughout the cell cycle to maintain the genomic stability of the cell. Unlike homologous recombination (HR), NHEJ is capable of repairing DSBs without the need for a homologous template, making it a rapid response mechanism, but potentially prone to errors. Central to NHEJ function and essential for the ligation through the recruitment and activation of additional repair factors, such as Artemis, XRCC4, and DNA ligase IV, is the DNA-dependent protein kinase (DNA-PK) complex. Dysregulation in the NHEJ pathway contributes to genomic instability, oncogenesis, and resistance to genotoxic therapies. Consequently, inhibitors of DNA-PK have emerged as promising therapeutic agents to sensitize tumor cells to radiation and DNA-damaging chemotherapeutics. Inhibiting the DNA-PK ability to recruit the protein complex needed for successful DSB repair promotes cell death through apoptosis or mitotic catastrophe. While inhibitors of DNA-PK can be used to enhance the effects of genotoxic therapies, the field still struggles to address critical problems: how to best exploit the differential DNA repair capacities among tumor subtypes, how to maximize radiosensitization of cancerous cells while sparing normal tissues, and how to translate preclinical studies into clinical benefits. Given that NHEJ constitutes the primary line of defense against radiation-induced damage, rapidly repairing the majority of double-strand breaks throughout the cell cycle, this review concentrates on targeting the DNA-PK complex, as the master regulator of this rapid-response mechanism, highlighting why its inhibition represents a strategic action to overcome intrinsic radioresistance. The implementation of DNA-PK inhibitors into medical practice can enable the stratification of oncologic patients into two categories, based on the tumors’ vulnerability to NHEJ disruptions. Thus, the therapeutic pathways of patients with NHEJ tumors could branch, combining traditional genotoxic therapies (radiation and DNA-damaging chemotherapeutics) with DNA-PK inhibitors to achieve an enhanced effect and improved survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212ac44d554e98203d51f1eb9fd409f026d7de87" target='_blank'>
              Impeding the NHEJ Pathway for Overcoming Radioresistance in the Context of Precision Radiotherapy of Cancer
              </a>
            </td>
          <td>
            D. Niculae, R. Șerban, D. Niculae, D. Drăgănescu
          </td>
          <td>2026-01-20</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4185cccd9e0a3e27a9efe577810b6e58e6b70" target='_blank'>
              Polymorphic 3D genome architecture mediated by transposable elements
              </a>
            </td>
          <td>
            Harsh G. Shukla, Yuheng Huang, Zita Y. Gao, 
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a1d0df44d9ce5038b1336c99cf98a4e708b134" target='_blank'>
              The alternative lengthening of telomeres pathway through a DNA repair lens: mechanism and therapeutic opportunities.
              </a>
            </td>
          <td>
            D. Muoio, Elise Fouquerel
          </td>
          <td>2025-12-01</td>
          <td>NAR cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Rationale: Neuroblastoma (NB) is a predominant extra-cranial malignancy in childhood, while molecular drivers of its progression and effective treatment strategies have yet to be clarified. Methods: RNA sequencing was performed to identify transcriptional regulators and corresponding target genes. To explore the biological effects and underlying mechanisms of these regulators, a comprehensive methodology was utilized, encompassing chromatin immunoprecipitation, dual-luciferase reporter assay, qRT-PCR, western blot, alongside gene over-expression and silencing techniques, co-immunoprecipitation, and mass spectrometry. The MTT assay, soft agar colony formation, Matrigel invasion, and nude mouse xenograft models were applied to assess oncogenic properties. Patient survival was analyzed using the log-rank test. Results: Armadillo repeat containing 12 (ARMC12) was identified as a MYC-interacting modulator within liquid condensates to up-regulate critical nucleoporin-encoding targets (NUP62/NUP93/NUP98), which promoted nuclear pore complex (NPC) biogenesis to facilitate nuclear trafficking of oncogenic effectors, thereby enhancing invasion and metastasis of NB. As a protein within extracellular vesicles of Malassezia globosa colonizing NB tissues, MGL_0381 also facilitated MYC transactivation via physical interaction to accelerate NPC biogenesis and NB progression. Tioconazole (TCZ) and UU-T02 were identified as efficient inhibitors blocking ARMC12-MYC and MGL_0381-MYC interaction, and synergistically reduced NPC number and aggressive features of NB. High ARMC12, MYC, NUP62, NUP93, or NUP98 levels served as markers of unfavorable patient outcomes in clinical cohorts. Conclusions: These findings collectively demonstrate that dual targeting of AMRC12 and Malassezia globosa disrupts MYC liquid condensates-driven NPC biogenesis during NB progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d399122edcdb055994dc03e897307f03c8f832ff" target='_blank'>
              Dual targeting of AMRC12 and Malassezia globosa disrupts MYC liquid condensates-driven nuclear pore complex biogenesis in neuroblastoma
              </a>
            </td>
          <td>
            A. Hu, Chunhui Yang, Zhijie Wang, Xiaolin Wang, Xinyue Li, Shunchen Zhou, Bosen Zhao, Jiaying Qu, Xiaojing Wang, Liduan Zheng, Q. Tong
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionarily conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as Scm3-depleted cells are sensitive to DNA damage. We further observe that Scm3 depletion genetically interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. The revelation that these chaperones promote genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae82b4df78dabaa8a230fa6f4f8b11503bb8e823" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway in budding yeast
              </a>
            </td>
          <td>
            Prakhar Agarwal, Anushka Alekar, Shubhomita Mallick, Kannan Harini, S. Ghosh
          </td>
          <td>2025-12-10</td>
          <td>eLife</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40038069867c0720d29003167b0971829e037042" target='_blank'>
              Nanopore whole-genome sequencing reveals conserved chromosome-specific telomere architecture across tissues and populations
              </a>
            </td>
          <td>
            Niklas L. Engel, B. Brors, L. Feuerbach, Peter J. Park
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a25c976684f63347d1fe095c70af1dbd2cc6459" target='_blank'>
              A molecular framework of chromatin extrusion in plants
              </a>
            </td>
          <td>
            Sofia Tzourtzou, Zhidan Wang, Sarah Leuchtenberg, D. Latrasse, M. Benhamed, Chang Liu
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Transposable elements (TEs) are indispensable components of eukaryotic genomes, mechanistically linked to carcinogenesis, aging and other degenerative diseases. The ability of TEs to self-propagate and cause deletions, inversions or insertions within the genome poses a real threat to the fidelity of genomic integrity. This review discusses the fundamental properties of TEs, with a focus on cellular interactions associated with mechanisms involved in recombination, replication, and DNA repair. Since mobilization of TEs induces double-strand breaks (DSBs), faulty repair mechanisms could lead to cellular dysfunction, pathology and death. The TE-induced DNA DSB repair cascade follows either homologous recombination (HR) or non-homologous end-joining (NHEJ) pathways. Importantly, epigenetic regulatory mechanisms including DNA methylation and histone acetylation provide additional control in ensuring accurate DNA repair and could prove to be key targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65023eda36836bf677b82b04598ca5b47b85bce0" target='_blank'>
              Impact of Transposable Elements on DNA Double-Strand Break Repair and Genomic Stability.
              </a>
            </td>
          <td>
            Mohd Iqbal Bhat, R. Pandita, Arjamand Mushtaq, U. S. Mir, Najumu Saqib, Parthas S. Sarkar, Audesh Bhat, Kenneth S. Ramos, Tej K. Pandita, Mohammad Altaf
          </td>
          <td>2026-01-11</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Low-pass whole-genome sequencing (LP-WGS) of circulating tumor DNA (ctDNA) is increasingly recognized for its utility in identifying somatic copy-number aberration (CNA). In this study, we analyzed LP-WGS ctDNA data from 73 pediatric patients with neuroblastic tumor and 11 healthy controls to explore diagnostic value of ctDNA CNA burden (including the genotypings) with a customized bioinformatics workflow. We found that a high baseline ctDNA CNA burden [tumor DNA fraction (TFx) ≥0.2%] was present in 36 of 41 patients (87.80%) with neuroblastoma, six of 22 patients (27.27%) with ganglioneuroblastoma, and three of 10 patients (30%) with ganglioneuroma. High baseline ctDNA CNA burden could predict high-risk neuroblastic tumors with an area under curve (AUC) of 0.95, sensitivity of 94.12%, and specificity of 100%. Frequent chromosomal copy-number changes, including chr17q gain, chr7 gain, chr3p loss, and chr11q loss, were found in ctDNA. Gain of chr17q demonstrated the highest diagnostic value with an AUC of 0.92, indicating strong sensitivity and specificity for detecting high-risk neuroblastic tumors. The homologous recombination deficiency score in the high- and intermediate-risk groups was significantly elevated compared with those in the low-/very low–risk group. The TFx levels and segmental alterations significantly decreased in patients with neuroblastic tumor who underwent chemotherapy, from median TFx = 13.82% before treatment to 0.24% after treatment (P < 0.0001). Our findings highlight the effectiveness of LP-WGS ctDNA CNA analysis as a promising approach for diagnosis and risk stratification of pediatric neuroblastic tumors and for monitoring chemotherapy response. Particularly, ctDNA analysis is minimally invasive, rapid, and cost-effective, which could bring additional benefits in pediatric practices. Significance: Our results support the development of ctDNA CNA analysis as a robust and minimally invasive approach for early detection, molecular diagnosis, and risk stratification of peripheral neuroblastic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01256a37f8920b62f2d81f1a7bbc4542d1a51191" target='_blank'>
              Copy-Number Aberrations in Circulating Tumor DNA Enable Diagnosis and Risk Stratification of Pediatric Neuroblastic Tumors
              </a>
            </td>
          <td>
            Ting Tao, Jiabin Cai, Yinbing Tang, Dong-Fang Lu, Lifeng Zhang, Jinkai Peng, Yilong Wang, Weizhong Gu, Shouhua Zhang, Jinhu Wang
          </td>
          <td>2025-12-03</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA:RNA hybrids are unusual structures found throughout the genomes of many species, including yeast and mammals. While DNA:RNA hybrids may promote various cellular functions, persistent hybrids lead to double strand breaks, resulting in genomic instability. DNA:RNA hybrid formation and removal are therefore highly regulated, including by enzymes that either degrade or unwind RNA from the hybrid. Meiosis is the specialized cell division that creates haploid gametes for sexual reproduction. Previous work in yeast and mammals showed that elimination of DNA:RNA hybrids by RNase H facilitates meiotic recombination. This work demonstrates that the conserved Sen1 DNA/RNA helicase functions during three temporally distinct processes during yeast meiosis. First, SEN1 allows meiosis-specific genes to be expressed at the proper time to allow entry into meiosis. Second, SEN1 prevents the accumulation of hybrids during premeiotic DNA replication. Third, SEN1 promotes the repair of programmed meiotic double strand breaks that are necessary to form crossovers between homologous chromosomes to allow their proper segregation at the first meiotic division. Given the evolutionary conservation of Sen1 with its mammalian counterpart, Senataxin, studies of Sen1 function in yeast are likely to be informative about the regulation of DNA:RNA hybrids during human meiosis as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8231ab0fa8d6701759423756235a545680a4b00a" target='_blank'>
              The conserved SEN1 DNA/RNA helicase has multiple functions during yeast meiosis
              </a>
            </td>
          <td>
            Robert Gaglione, Leonidas Pierrakeas, Lihong Wan, Jonathan Caradonna, A. MacQueen, Ed Luk, N. M. Hollingsworth
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2ceccde009f3d35721416ae6a0441554cd9fa0" target='_blank'>
              The mitotic stopwatch synergizes with mild p53 activation to halt cell proliferation
              </a>
            </td>
          <td>
            Beata E. Mierzwa, Franz Meitinger, A. Desai, K. Oegema
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c06b0dfa9d58dc239114dc3e3b926c74b839c6" target='_blank'>
              Genomic analysis of PLNTY-like tumor progression into epithelioid glioblastoma: a case report.
              </a>
            </td>
          <td>
            S. Mäntylä, Anssi Nurminen, Sanna Huovinen, S. Jaatinen, T. Haapaniemi, R. Nurminen, Ismaïl Hermelo, Stefanie Volz, K. Maass, K. Pajtler, K. Nordfors, H. Haapasalo, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2026-01-08</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) was characterized by a highly complex genome, with structural variations (SVs) playing a significant role in its development. In this study, we employed Oxford Nanopore Technology long‐read sequencing in paired tumor and adjacent normal liver tissues from 74 Chinese HCC patients to thoroughly characterize the landscape of somatic SVs. Our analysis revealed that somatic SVs were more prevalent in hepatitis B virus (HBV)‐related HCC, with chromosome 1 emerging as a major hotspot, and several members of the chromosome 1 open reading frame (C1orf) family genes expression level exhibited significant age‐related difference. Notably, HBV‐related HCC cases exhibited a higher frequency of deletions, particularly among younger ones (≤ 35 years old). In addition, we observed an increased burden of HBV integration events in younger ones. Remarkably, the divergent‐paired related homeobox (DPRX) loci was identified as a novel gene for HBV integration in younger patients. Together, these findings delineated the somatic SV landscape in HCC and underscored age‐associated HBV‐related genomic alterations as key pathological features of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cdc214e4351424b89dd91edf26352c6acad965" target='_blank'>
              Landscape of Somatic and Age‐Related Pathogenic Structural Variations in Hepatocellular Carcinoma Revealed by Long‐Read Sequencing
              </a>
            </td>
          <td>
            Zhewen Wei, Yinghao Cao, Hongchao Liu, Mei Liu, Bolun Zhang, Jianming Ying, Jianqiang Cai, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Zhen Huang, Jianmei Liu, Xueyan Lv, Zhiwen Luo, Zhicheng Wei, Xiaoshi Zhang, Yi Yang, Yiqiao Deng, Yanjiang Yin, Jinghua Chen, Junbo Liang, Xiaoyue Wang, Yefan Zhang, Hong Zhao
          </td>
          <td>2026-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0bf2ebc24868e52a4bf9e9e5a2567d97f5b5a0" target='_blank'>
              Integrated proteogenomics uncovers ancestry-specific and shared molecular drivers in localized prostate cancer
              </a>
            </td>
          <td>
            Cara C. Schafer, T. Abulez, Xijun Zhang, Kun-Lin Ho, Jiji Jiang, D. Young, Jesse M. Fox, K. Conrads, B. Hood, G. Sukumar, Darryl Nousome, Praveen-Kumar Raj-Kumar, Mariano Russo, Ayesha A. Shafi, Xiaofeng A. Su, A. Dobi, Amina Ali, S. Elsamanoudi, J. Cullen, W. Figg, G. Petrovics, C. Dalgard, M. D. Wilkerson, N. Bateman, T. Conrads, I. Sesterhenn, Leigh Ellis, Craig D. Shriver, Gregory T. Chesnut, Shyh-Han Tan
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e5545d6ea0cdff6467688544916e041cb664d5" target='_blank'>
              Human CCR4-NOT suppresses pervasive transcription and retrotransposable elements
              </a>
            </td>
          <td>
            Shardul Kulkarni, Alexis Morrissey, A. Sebastian, O. T. Akinniyi, C. Keller, I. Albert, Shaun Mahony, J. Reese.
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Potassium bromate-induced DNA damage, including 8-oxo-7,8-dihydroguanine (OG) and 2,6-diamino-4-hydroxy-5-formamidopyrimidine (FapyG), are common oxidatively generated DNA lesions with mutagenic potential if not efficiently repaired. While sequencing-based studies have shown that damage formation is influenced by DNA sequence context, secondary structures, and chromatin features, how OGG1-mediated repair is regulated within chromatin remains unclear. Here, we apply CLAPS-seq to generate genome-wide, single-nucleotide resolution maps of OGG1-mediated repair over time in human cells, and systematically analyze how chromatin context affects repair efficiency across hierarchical scales. We find that chromatin accessibility governs rapid initial repair, whereas higher-order chromatin structures increasingly influence later-phase repair. In addition, nucleosome occupancy and transcription factor binding, exemplified by CCCTC-binding factor (CTCF), modulate OGG1-mediated repair at both local and base scales. Overall, mutational outcomes correlate more strongly with repair dynamics than with damage levels. Together, these findings establish a comprehensive framework linking chromatin organization, DNA repair kinetics, and oxidatively induced mutagenesis, and offer new insights into the origins of mutation patterns in cancer and diseases associated with oxidative stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6701e71af4dcc5a5f77f208b4a6942e24ef9ab11" target='_blank'>
              Dynamic genome-wide mapping reveals how chromatin context shapes OGG1-mediated repair and related mutagenesis in human cells
              </a>
            </td>
          <td>
            Jie Li, Lin Li, Yuanqing Tan, Mengdie Yin, Yuxuan Li, Dongrui Yin, Jiao An, Guanglei Zhuang, Maoxiang Qian, Jinchuan Hu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC.
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Replication-dependent (RD) histones are crucial for packaging newly replicated DNA into chromatin, ensuring genome stability. In metazoans, the mRNA of RD histones is uniquely regulated through a conserved 3′ stem–loop bound by stem–loop binding protein (SLBP). This allows cell cycle-coupled regulation of these important transcripts. However, oocytes must stabilise histone mRNAs independently of the cell cycle to ensure maternal loading to support the first embryonic divisions. Using Caenorhabditis elegans as a model system, we discovered an SLBP-independent mechanism that ensures RD histone transcript stability during oogenesis. This is mediated by the protein complex PETISCO, bound to the effector protein TOST-1, which directly binds the histone stem–loop region and maintains maternal histone mRNA levels during oogenesis and early embryogenesis. Loss of this mechanism disrupts histone homeostasis, leading to premature genome activation, mitotic defects, and embryonic lethality. Interestingly, the same complex, PETISCO, acts in piRNA biogenesis when bound to the effector PID-1, revealing an intriguing co-option of this histone mRNA homeostasis mechanism by the piRNA pathway. Our findings reveal a unique SLBP-independent mechanism of histone mRNA regulation, that served as a basis for the evolution of a novel piRNA biogenesis mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c7ef4add89341d88d3d2d7bf38e0138671d9e2" target='_blank'>
              SLBP-independent control of maternal histone mRNA
              </a>
            </td>
          <td>
            Joana Pereirinha, Martin Brehm, Shamitha Govind, Anke Busch, Nadezda Podvalnaya, Ann-Sophie Seistrup, Kamila Delaney, Florian A. Steiner, Julian Konig, Sebastian Falk, R. Ketting
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0453b8fd3ee789a5730a5cb95cb52509da58c9" target='_blank'>
              Unraveling miRNA-Driven DNA Damage Response Networks in Pancreatic Adenocarcinoma: A Multi-Omics and Machine Learning Approach
              </a>
            </td>
          <td>
            Shayori Bose, Paramita Basak Upama, S. Ahamed
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Termination of DNA replication is a surprisingly complex process that contributes critically to genome stability and cell viability. And even though progress was made to establish the consequences that arise if termination is going awry, the precise molecular mechanisms of fork fusion events and the coordination with key factors that ensure that DNA replication is brought to a successful conclusion remain poorly understood. We therefore investigated replication termination in Escherichia coli, focusing specifically on the interplay between replication fork fusions and genomic stability, the Tus–ter replication fork trap, and key DNA-processing enzymes. By utilizing whole genome sequencing, immunoblotting, and recombination reporter assays, we demonstrate that local hyper-recombination is induced wherever forks meet and that the combined loss of factors such as RecG helicase and 3′ exonucleases causes extreme over-replication in the terminus region of the chromosome. Unexpectedly, cells lacking Tus exhibit elevated R-loop levels, revealing an unanticipated connection between the fork trap and R-loop metabolism. These findings underscore the complexity of replication termination and its central role in maintaining bacterial genome stability, while providing mechanistic insights with implications for understanding replication termination in more complex organisms and developing new antimicrobial strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7313e7e0c5ea82dfcde902877c06fc46c8baafab" target='_blank'>
              Termination of DNA replication drives genomic instability via multiple mechanisms
              </a>
            </td>
          <td>
            Daniel J. Goodall, J. U. Dimude, M. A. Hashemloo, Emma L Dunbar, Iren Grigoryan, Amy L. Upton, E. Bolt, Christian J. Rudolph
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy a significant fraction of a wide variety of eukaryotic genomes and can be domesticated into functional sequences harbouring a coding or regulatory potential. While studies in mammals have revealed that retrotransposons can frequently give rise to tissue-specific transcriptional enhancers our understanding of this phenomenon in other vertebrate groups is scarcer. Here, we examined TE occupancy at tissue-specific nucleosome free regions (NFRs) which are not annotated as promoters in the amphibian model organism Xenopus tropicalis. We report three distinct miniature inverted-repeat TEs (MITEs) enriched at distal liver-specific NFRs and belonging to the hAT, Harbinger and Kolobok superfamilies of DNA transposons. These MITEs show a marked depletion at NFRs specific to the bone tissue, probably reflecting a process of negative selection. In addition, we show that they are enriched for transcription factor binding sites known to be bound by key regulators of liver biology, hematopoiesis, and the immune system, and that they are more likely to be located in the vicinity of genes specifically expressed in the liver than other MITE copies that are not associated to a NFR. We also find that these MITEs are not present at orthologous positions in the genome of the related allotetraploid frog Xenopus laevis, while they globally are abundant in this species. We discuss how independent bursts of MITE amplification followed by subsequent domestication episodes might independently have given rise to liver-specific transcriptional enhancers in the Xenopus tropicalis lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24529c91db0790a41856023783a87c69aa0832af" target='_blank'>
              Liver-specific enhancers evolved from independent episodes of MITE domestication in Xenopus tropicalis
              </a>
            </td>
          <td>
            Japhet Rojas, Héctor Castillo, Marco Mundaca, Jorge Fraga, Clément Gilbert, Braulio Valdebenito-Maturana, Sylvain Marcellini
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, S. R. Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transcription factors from the Zinc Finger Protein (ZFP) family are extensively implicated in tumorigenesis, yet the roles of many members, such as ZNF668, remain uncharacterized. This study presents a comprehensive pan-cancer analysis of ZNF668, investigating its expression profiles, genetic alterations, functional pathways, association with immune infiltration, and clinical correlations across cancer types from TCGA. Our pan-cancer analysis identifies ZNF668 as a frequently overexpressed gene with significant diagnostic and prognostic value. Its overexpression, often driven by gene amplification, is linked to fundamental cellular processes such as RNA splicing and transcriptional regulation. Critically, ZNF668 is implicated in promoting a state of adaptive immune resistance. While its expression positively correlates with the immunogenic MSI phenotype and suggests T-cell infiltration, this is likely offset by a dual immunosuppressive mechanism comprising a strong association with a cancer-associated fibroblast (CAF)-driven, T-cell-exhausted TME and a concurrent suppression of neutrophil recruitment. Furthermore, molecular docking identified Dasatinib as a potential ZNF668 inhibitor. These findings establish ZNF668 as a key regulator of CAF-mediated immune suppression, presenting it as a novel therapeutic target for restoring effective anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3fd7e5133bfa569359741047afb60d0b7cc992d" target='_blank'>
              Comprehensive Pan-Cancer Analysis of ZNF668 Reveals the Prognostic and Immunological Significance of ZNF668
              </a>
            </td>
          <td>
            Xiaoyan Hu, Jiali Guo, Hua Zhong, Wenxin Huang, Size Chen, Canfeng He
          </td>
          <td>2025-11-28</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Brain cancers are the most common solid tumors in children and adolescents and remain the leading cause of disease-related death in these populations. Despite their clinical importance, therapeutic progress has been limited by extensive intra- and inter-tumor heterogeneity. In particular, the clonal architecture and evolutionary dynamics of pediatric brain tumors remain poorly understood. Addressing this knowledge gap is critical, as genetically distinct subclones can shape disease progression, treatment resistance, and clinical outcome. To investigate clonal evolution at the population level, we analyzed longitudinal whole-genome sequencing data from 1,742 tumors across 27 brain cancer types in 1,479 pediatric patients from the Children’s Brain Tumor Network. We inferred clonal composition and phylogenetic relationships for each patient and reconstructed the space of tumor phylogenies for the entire cohort using a regularized Gromov-Hausdorff distance. Tumor phylogenetic structure was significantly associated with mutational burden, mutational signatures, cancer classification, and patient's survival status. In longitudinal samples, survival status remained significantly associated with tumor clonal architecture after adjusting for mutational burden, tumor subtype, and anatomical location, underscoring the clinical value of this readout. In contrast, in primary tumors, mutational signatures remained significant after adjusting for mutation count, tumor subtype, anatomical location, and purity, while survival status associations were no longer significant. Mutational signature timing analysis revealed that pediatric brain tumor genetic architecture is shaped by distinct mutational processes acting at different stages of tumor evolution. Truncal somatic mutations were enriched for age-related, defective proofreading, and homologous recombination deficiency signatures, whereas subclonal mutations showed significant contributions from DNA mismatch repair, chemotherapy-related, temozolomide-related, and somatic hypermutation activity signatures. Comparisons of matched longitudinal and primary phylogenetic trees further revealed that deceased patients and those with recurrent or metastatic disease exhibited significantly greater evolutionary divergence, suggesting that primary tumors are less predictive of complete evolutionary trajectories in poor-outcome cases. Together, these findings suggest that pediatric brain tumor evolution is shaped by cancer type and various mutational processes, and that patients with adverse outcomes follow distinct evolutionary trajectories independently of subtype. Our study provides a population-level view of tumor evolution in pediatric brain cancers and highlights the value of genetic clonal dynamics for understanding tumor etiology and progression.



 Minh A. Nguyen, Pablo G. Cámara. Mapping clonal architecture and evolution in pediatric brain cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d01cdf935c0ff4316725768de123e5059b9b4ae1" target='_blank'>
              Abstract B008: Mapping clonal architecture and evolution in pediatric brain cancers
              </a>
            </td>
          <td>
            Minh A. Nguyen, Pablo G. Cámara
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Receptor tyrosine kinase (RTK) fusions are a large class of oncoproteins found in ∼5% of cancers. Key questions remain, however, about how RTK fusions transmit oncogenic signals, including how these largely cytoplasmic proteins activate downstream pathways that originate at the plasma membrane. Fusions are multimeric and can form mesoscale condensates in cancer cells, and condensation has been implicated as an essential mechanism to enable signal transmission from the cytoplasm. However, whether condensates play a causal role, or whether smaller ‘diffuse’ assemblies are sufficient to transduce signals, has been challenging to establish. Here we apply advanced microscopy, single-cell analysis, and synthetic fusions to determine the principles by which multimerization and condensation drive signaling from cytoplasmic RTK fusions. For EML4-ALK, a prominent fusion that forms condensates, we found poor correlation between condensation and signaling. By contrast, EML4-ALK activity was abundant in the diffuse phase, and the kinetics of diffuse-phase activity aligned more closely with downstream Erk signaling than did kinetics of signaling within condensates. Synthetic RTK fusions showed that cytoplasmic ALK or RET fusion dimers—and even constitutively active monomers—were sufficient to induce strong Ras-Erk signaling despite the absence of condensates, and diffuse fusions were sufficient to transform cells in vitro and in subcutaneous tumor models. A panel of various other cancer-driving RTK fusions showed that low-order multimerization was universal across fusions, whereas mesoscale condensation was rare and did not correlate with signaling. Our results suggest that low-order fusion multimerization is sufficient to drive its phosphorylation, which is necessary and sufficient to trigger downstream oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6563f441f99dac16cec7dceb7a7c5f130e146e" target='_blank'>
              Low-order assemblies drive oncogenic RTK fusion signaling without condensation
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Thomas R. Mumford, Delaney Wilde, Sofia Wissert, Y. Gao, Emily Brackhahn, R. Kriwacki, Elizabeth Rhoades, Lukasz J. Bugaj
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Comprehensive genomic analysis is essential for advancing our understanding of human genetics and disease. However, short-read sequencing technologies are inherently limited in their ability to resolve highly repetitive, structurally complex, and low-mappability genomic regions, previously coined as “dark” regions. Long-read sequencing technologies, such as PacBio and Oxford Nanopore Technologies (ONT), offer improved resolution of these regions, yet they are not perfect. With the advent of the new Telomere-to-Telomere (T2T) CHM13 reference genome, exploring its effect on dark regions is prudent. In this study, we systematically analyze dark regions across four human genome references—HG19, HG38 (with and without alternate contigs), and CHM13—using both short- and long-read sequencing data. We found that dark regions increase as the reference becomes more complete, especially dark-by-MAPQ regions, but that long-read sequencing significantly reduces the number of dark regions in the genome, particularly within gene bodies. However, we identify potential alignment challenges in long-read data, such as centromeric regions. These findings highlight the importance of both reference genome selection and sequencing technology choice in achieving a truly comprehensive genomic analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0af39dc3f40d9c2bc549d4b7b8ccbbcdca891e97" target='_blank'>
              Dark and camouflaged genomic regions remain challenging in CHM13
              </a>
            </td>
          <td>
            Mark E. Wadsworth, Madeline L. Page, B. A. Heberle, Justin B. Miller, Cody J. Steely, M. Ebbert
          </td>
          <td>2026-01-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, Gang Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract DNA methylation (DNAm) is widely used to leverage biological information in the context of cancer. Active regulatory elements marked by increased chromatin accessibility and specific transcription factor (TF) binding, such as enhancers, show tumor-specific DNAm patterns reflecting the biological forces shaping tumorigenesis. However, DNAm changes also occur at regions that are inactive at histone-mark level in fully developed tumors. Beyond hypermethylation of promoters, these changes are often overlooked, leaving their functional relevance poorly understood. By analyzing DNAm and chromatin state annotations from conventional mantle cell lymphoma (cMCL), we identified a subset of ~300 CpGs with homogeneous cMCL-specific demethylation patterns located in cMCL-inactive regions. We show that these regions contain cMCL-related TF motifs and are flanked by genes with cMCL-specific expression patterns, suggesting a potential regulatory role in lymphomagenesis. We hypothesize that these regions represent DNA demethylation imprints of genome activation prior to full-blown tumor formation, either present in its cell type of origin (COO) or during early stages of tumor formation. Altogether, we put forward a new, DNAm-centered approach to gain insights into potential early tumorigenic events, allowing us to generate novel, further explorable hypotheses to better understand the features playing a role in lymphomagenesis and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a0fc34737aa002d9118359a92f24cc2e094aa9" target='_blank'>
              DNA methylation profiles aid to identify putative genome activation histories along lymphomagenesis
              </a>
            </td>
          <td>
            Leone Albinati, Irene D’Onofrio, Rabia Gül Aydin, Houyem Toukabri, Renée Beekman
          </td>
          <td>2026-01-10</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc8f584519863394cabad2d307177d0a23fce8d" target='_blank'>
              An EGR1-dependent cascade modulates genome architecture at the CSF1R locus.
              </a>
            </td>
          <td>
            Sandra Deliard, Elisa Barbieri, Marco Trizzino, Kelsey A. Leach, Avery J. Zucco, Francis Picone, Filippo Veglia, Alessandro Gardini
          </td>
          <td>2026-01-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="SRY-related high-mobility group protein B5 (SOX5) functions as an oncogene in diverse human malignancies, contributing to tumor progression, metastasis and therapy resistance across various cancer types. In light of this, a pan-cancer analysis was performed in the present study to investigate the oncogenic function of SOX5 in various human malignancies. SOX5 mRNA expression levels were compared between normal and malignant tissues using The Cancer Genome Atlas database and diagnostic effectiveness was evaluated using receiver operating characteristic curve analysis. Kaplan-Meier survival analysis was performed to investigate the association between SOX5 expression and overall survival, disease-specific survival, disease-free interval and progression-free interval. The influence of SOX5 on immune infiltration and immunological scores in the tumor microenvironment was assessed utilizing Estimation of Stromal and Immune Cells in Malignant Tumor Tissues Using Expression Data, Cell-Type Identification by Estimating Relative Subsets of RNA Transcripts and single-sample Gene Set Enrichment Analysis methodologies, in conjunction with its relationship to drug sensitivity and genetic modifications. SOX5 exhibited tissue-specific dysregulation, being markedly downregulated in carcinomas such as lung adenocarcinoma (LUAD), lung squamous cell carcinoma and breast cancer, while increased in testicular germ cell tumor and non-small cell lung cancer (NSCLC) cell lines. It demonstrated notable diagnostic potential, with good performance in LUAD (area under the curve=0.916). The prognostic relevance of SOX5 was contingent upon context as lower expression conferred protection in specific malignancies; however, it was associated with worse outcomes in others, such as low-grade glioma and pancreatic adenocarcinoma. Increased SOX5 expression was associated with an immunosuppressive milieu marked by a rise in regulatory T cells, a decline in cytotoxic T cells and the activation of immunological checkpoints including programmed death-ligand 1 and cytotoxic T lymphocyte associated protein 4. Moreover, SOX5 was associated with genomic instability, susceptibility to medicines such as azacitidine and distinct mutation patterns. SOX5 suppression in NSCLC cells in vitro impeded proliferation, migration and invasion. These findings collectively emphasize the key function of SOX5 in tumor biology and highlight its potential as a biomarker for cancer diagnosis, prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9787a21f3dc3c6488d2a12dfbb8cfdefc53c847" target='_blank'>
              Pan-cancer analysis of the oncogenic role of SRY-related high-mobility group box protein B5 in human tumors
              </a>
            </td>
          <td>
            Yajun Tong, Huini Da, Kewei Tang, Qiang Zhou, Songlian Liu, Leilan Yin, Ling Long, Site Bai, Lu-di Ou, Qinghua Yin
          </td>
          <td>2025-11-28</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Genomic copy number alterations cause aneuploidy, a hallmark of cancer characterized by imbalanced chromosome numbers. Aneuploidy has been shown to negatively associate with cancer immune signaling, but these mechanisms remain understudied. Of note, high levels of aneuploidy are strongly correlated with lethal progression and poor patient outcomes in prostate cancer. Our recent work identified chromosome 8q (chr8q) gain, one of the most frequent aneuploidy events in prostate tumors, as a key driver of disease progression, in part through the cohesin RAD21 gene located on the chr8q24 region. In both prostate cell line and organoid models, we demonstrated that increased RAD21 expression accelerates oncogenesis and aggressive tumor proliferation by mitigating cellular stress during early-stage tumorigenesis. Therefore, we hypothesize that elevated RAD21 levels enable cancer cells to evade immune surveillance by alleviating oncogenic stress and DNA damage. In this study, we analyzed publicly available genomic datasets and discovered that higher RAD21 expression negatively correlated with natural killer (NK) cell infiltration in prostate tumors. To further test our hypothesis, we modulated RAD21 levels in isogenic prostate cell culture models. Using an NK cell co-culture assay, we observed that cells with increased RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited greater survivorship when co-cultured with NK cells. In addition, we found that cellular levels of various pro-inflammatory cytokines, including those that stimulate NK cell activation, were significantly downregulated when RAD21 was overexpressed. These findings are consistent with our observation that tumors with high RAD21 levels are linked to reduced NK cell infiltration in prostate cancer cases. Our results suggest that amplification of RAD21, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Targeting RAD21-driven immune evasion may reveal novel therapeutic strategies to restore immune surveillance mechanisms in prostate cancer.



 Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R. Lozada, Christine Luo, Andrew Elliott4, Nicholas A. Zorko3, Justin H. Hwang3, David Takeda5, Xiaofeng A. Su. Elevated RAD21 levels promote immune evasion in prostatic malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A032.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba3556a59b07de1fb3550ddaa2e2a0eb7e783af" target='_blank'>
              Abstract A032: Elevated
 RAD21
 levels promote immune evasion in prostatic malignancies
              </a>
            </td>
          <td>
            Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R Lozada, Christine Luo, A. Elliott, Nicholas A. Zorko, Justin H. Hwang, David Takeda, 
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is characterized by uncontrolled proliferation accompanied by oncogene hypertranscription, leading to transcription stress, a key source of DNA double-strand breaks (DSBs) that jeopardize genomic stability. Despite its importance, the landscape and consequences of transcription stress remain underexplored. Here, we used maps of DSBs identified through sBLISS (in-suspension break labeling in situ and sequencing) with transcription stress markers to delineate the transcription stress landscape in cancer. We found that transcription stress sites are shaped by the superenhancer regulatory landscape. Notably, γH2AX is enriched at transcription stress sites; however, not all DSB-enriched genes show similar γH2AX marking. Instead, genes with DSBs tied to transcription stress are distinctly marked. Genes with high DSBs marked by γH2AX exhibited substantially higher DSB turnover and repair than those with low γH2AX, and are associated with vulnerability to mutagenesis. These findings underscore superenhancer activity as a determinant of the transcription stress landscape in cancer, posing a threat to the genomic stability of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fad140fabdd35a840bb4652e1678621ab279f4b" target='_blank'>
              Superenhancers shape the landscape and repair dynamics of transcription-associated DNA breaks in cancer
              </a>
            </td>
          <td>
            Osama Hidmi, Diala Shatleh, Sara Oster Flayshman, Jonathan Monin, Rami I. Aqeilan
          </td>
          <td>2026-01-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Optical genome mapping (OGM) detects genome-wide structural variants (SVs), including balanced rearrangements and complex copy-number alterations beyond standard-of-care cytogenomic assays. In chronic lymphocytic leukemia (CLL), cytogenetic and genomic risk stratification is traditionally based on fluorescence in situ hybridization (FISH), karyotyping, targeted next-generation sequencing (NGS), and immunogenetic assessment of immunoglobulin heavy chain variable region (IGHV) somatic hypermutation status, each of which interrogates only a limited aspect of disease biology. Methods: We retrospectively evaluated fifty patients with CLL using OGM and integrated these findings with cytogenomics, targeted NGS, IGHV mutational status, and clinical time-to-first-treatment (TTFT) data. Structural variants were detected using OGM and pathogenic NGS variants were derived from a clinical heme malignancy panel. Clinical outcomes were extracted from the electronic medical record. Results: OGM identified reportable structural variants in 82% (41/50) of cases. The most frequent abnormality was del(13q), observed in 29/50 (58%) and comprising 73% (29/40) of all OGM-detected deletions with pathologic significance. Among these, 12/29 (42%) represented large RB1-spanning deletions, while 17/29 (58%) were focal deletions restricted to the miR15a/miR16-1 minimal region, mapping to the non-coding host gene DLEU2. Co-occurrence of adverse lesions, including deletion 11q/ATM, BIRC3 loss, trisomy 12, and deletion 17p/TP53, were recurrent and strongly associated with shorter TTFT. OGM also uncovered multiple cryptic rearrangements involving chromosomal loci that are not represented in the canonical CLL FISH probe panel, including IGL::CCND1, IGH::BCL2, IGH::BCL11A, IGH::BCL3, and multi-chromosomal copy-number complexity. IGHV data were available in 37/50 (74%) of patients; IGHV-unmutated status frequently co-segregated with OGM-defined high-risk profiles (del(11q), del(17p), trisomy 12 with secondary hits, and complex genomes whereas mutated IGHV predominated in OGM-negative or structurally simple del(13q) cases and aligned with indolent TTFT. Integration of OGM with NGS further improved genomic risk classification, particularly in cases with discordant or inconclusive routine testing. Conclusions: OGM provides a comprehensive, genome-wide view of structural variation in CLL, resolving deletion architecture, identifying cryptic translocations, and defining complex multi-hit genomic profiles that tracked closely with clinical behavior. Combining OGM and NGS analysis refined risk stratification beyond standard FISH panels and supports more precise, individualized management strategies in CLL. Prospective studies are warranted to evaluate the clinical utility of OGM-guided genomic profiling in contemporary treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28f49f64c743393cb2116dc5e9531fab4be082e7" target='_blank'>
              Optical Genome Mapping Enhances Structural Variant Detection and Refines Risk Stratification in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            S. Chakraborty, Michelle A. Bickford, Narcisa A. Smuliac, Kyle A. Tonseth, Jing Bao, Farzana Murad, Irma G. Domínguez Vigil, Heather B. Steinmetz, Lauren M. Wainman, Parth Shah, Elizabeth M. Bengtson, Swaroopa PonnamReddy, Gabriella A. Harmon, Liam L. Donnelly, L. Tafe, Jeremiah X. Karrs, Prabhjot Kaur, Wahab A. Khan
          </td>
          <td>2026-01-19</td>
          <td>Genes</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 The minor spliceosome (MiS) is a specialized RNA-processing machinery upregulated in cancer to promote expression of oncogenic genes. Here, we identify its catalytic component, U6atac snRNA, as a druggable vulnerability in prostate and breast cancers. U6atac knockdown triggers R-loop–mediated DNA damage while impairing DNA repair by downregulating key factors such as BRCA1, PARP1, TP53BP1, and CHK1/2, disabling both homologous recombination and non-homologous end joining. This dual effect sensitizes tumors to PARP inhibitors, cisplatin, and radiation, independent of BRCA status. Moreover, we uncover an adaptive resistance mechanism driven by extracellular vesicles enriched in U6atac, which amplify MiS activity and facilitate therapy escape; a process reversed by U6atac depletion. Across multiple in vitro and in vivo models, MiS targeting demonstrates tumor-selective activity with minimal toxicity. These findings position U6atac as a central regulator of genome stability and establish MiS targeting as a promising approach to potentiate genotoxic therapy and overcome resistance.



 Anke Katharina. Augspach, Mark Rubin, Paola Francica, Sven Rottenberg, Rahul Kanadia. Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c71eaaa6cc7e28ae4d142a474ab9e0350163c6b" target='_blank'>
              Abstract B003: Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, Mark A. Rubin, P. Francica, Sven Rottenberg, Rahul Kanadia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da2aeba8554dfbf99a705198a393f82144d0d18d" target='_blank'>
              Menin maintains enhancer-promoter interactions in a leukemia-specific manner
              </a>
            </td>
          <td>
            Vassilena Sharlandjieva, Catherine Chahrour, F. Lassen, Joseph C. Hamley, Andreas Damianou, N. Denny, Alastair L. Smith, Svenja S Hester, I. Vendrell, Ronald W. Stam, Marina Konopleva, Anindita Roy, J. O. Davies, Nicholas T. Crump, Benedikt M. Kessler, Thomas A. Milne
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ef4954d051c6c3069f377db379389ba701990a" target='_blank'>
              Interplay between cohesin and RNA polymerase II in regulating chromatin interactions and gene transcription.
              </a>
            </td>
          <td>
            Minji Kim, Ping Wang, P. A. Clow, Eli Chien, Xiaotao Wang, Jianhao Peng, Haoxi Chai, Xiyuan Liu, Byoungkoo Lee, C. Ngan, O. Milenkovic, Jeffrey H. Chuang, Chia-Lin Wei, Rafael Casellas, Albert Cheng, Yijun Ruan
          </td>
          <td>2026-01-13</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The field of epitranscriptomics discovered N6-methyladenosine (m6A) modifications, which function as fundamental elements that control RNA metabolism properties that powerfully affect cancer biology. This review examines the way m6A modifications shape RNA stability while regulating translation, together with their eraser and reader proteins. We demonstrate that m6A modifications guide oncogene and tumor suppressor transcript outcomes, which promote tumor growth, metastasis, and therapeutic resistance. The regulatory function of m6A depends significantly on its relationship with ncRNAs that mainly include miRNAs, lncRNAs, and circRNAs. The review examines the effects of m6A on ncRNA production, stability, export, and degradation, as well as the regulation of m6A protein expression by ncRNAs, highlighting intricate reciprocal feedback loops that drive cancer progression. The interplay between m6A RNA modifications and ncRNAs provides emerging evidence on how they collectively influence the tumor microenvironment, modulate immune system responses, and contribute to resistance. Harnessing ncRNA-m6A interactions for managing drug resistance offers promising therapeutic avenues. However, advancing our understanding of the context-specific roles of m6A modifications and translating these insights into clinical applications remains a significant challenge. This review synthesizes recent findings on ncRNA-m6A crosstalk to lay the groundwork for developing epitranscriptomic strategies in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e0eb5d331194e0274cc28e14543827ba3e7f5f" target='_blank'>
              The epitranscriptome meets non-coding RNA: m6A-mediated regulation in oncogenesis and therapy
              </a>
            </td>
          <td>
            Prasanna Srinivasan Ramalingam, M. Rejili, Faouzi Haouala, Md Sadique Hussain, Yumna Khan, Mudasir Maqbool, Janaki Ramaiah Mekala, S. Arumugam
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, T. F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism.
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Break-induced replication (BIR) is a primary homologous recombination pathway for repairing one-ended double-strand DNA breaks, including those arising from collapsed replication forks and eroded telomeres. BIR frequently leads to loss of heterozygosity, genetic mutations, and gross chromosomal rearrangements, all hallmarks of cancer. Here, we conducted a genome-wide screen that allowed us to identify and validate the involvement of 33 novel yeast genes in BIR. We report that, while DNA damage and spindle checkpoint machineries are both required to delay nuclear division and provide adequate time for BIR to complete, the spindle position checkpoint is required to coordinate between nuclear division and cytokinesis. Furthermore, we show that two nucleopore proteins play a sequential role during BIR: Nup84 acts before DNA synthesis, while Nup188 functions later to support repair completion. Given the conservation of BIR between yeast and humans and the role of BIR in cancer development, human homologs of the identified BIR proteins may represent promising targets for anti-cancer therapeutics. Break-induced replication (BIR) repairs broken DNA but can also destabilize genomes. The authors identify 33 new genes controlling BIR completion, showing that spindle assembly and spindle positioning checkpoints coordinate repair, and that nuclear pore proteins regulate BIR at multiple steps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, D. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6136e0a41212ee02eab7d57de4a28f7d297bef9a" target='_blank'>
              Sensing of DNA double-strand breaks by the NHEJ system stabilizes RORγt transcriptional activity and shapes Th17 pathogenicity in autoimmunity.
              </a>
            </td>
          <td>
            Guan-Yu Chen, Wen-Jie Zhu, Zhuang Li, Yun-Wei Hu, Xiao-Shuang Luo, Zhi-Qing Mai, Yuan Pan, Yu-Xun Shi, Zuo-Yi Li, Jun Huang, Pei-Dong Yuan, Zhi-Qiang Xiao, Qian Chen, Yan-Yan Xie, Hai-Xiang Huang, Yu-Xi Chen, Yao Lu, Min-Zhen Wang, Yi-Wen Xia, Xiao-Qing Chen, Dong-Ming Kuang, Dan Liang
          </td>
          <td>2026-01-07</td>
          <td>Cell research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The genome folds inside the cell nucleus into hierarchical architectural features, such as chromatin loops and domains. If and how this genome organization influences the regulation of gene expression remains only partially understood. The structure-function relationship of genomes has traditionally been probed by population-wide measurements after mutation of critical DNA elements or by perturbation of chromatin-associated proteins. To circumvent possible pleiotropic effects of such approaches, we have developed OptoLoop, an optogenetic system that allows direct manipulation of chromatin contacts by light in a controlled fashion. OptoLoop is based on the fusion between a nuclease-dead SpCas9 protein and the light-inducible oligomerizing protein CRY2. We demonstrate that OptoLoop can bring together genomically distant, repetitive DNA loci. As a proof-of-principle application of OptoLoop, we probed the functional role of DNA looping in the regulation of the human telomerase gene TERT by long-range contacts with the telomere. By analyzing the extent of chromatin looping and nascent RNA production at individual alleles, we find evidence for looping-mediated repression of TERT. In sum, OptoLoop represents a novel means for the interrogation of structure-function relationships in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5676ff6c24cf2aa8e41cc35c8a6bbe06bad743e9" target='_blank'>
              OptoLoop: An optogenetic tool to probe the functional role of genome organization.
              </a>
            </td>
          <td>
            Martin Stortz, Adib Keikhosravi, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2026-01-19</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The recruitment of telomerase to telomeres is a tightly regulated process which is stimulated by replication stress and the DNA damage response regulatory kinase ATR, via an unknown mechanism. Here, we demonstrate that nuclear filamentous actin is important for the stable interaction of telomerase with telomeres in immortal human cells, resulting in productive telomere elongation by telomerase in an actin-dependent manner. This process is regulated by both ATR and mTOR kinases, and employs other regulators of actin structure and function, such as WASP, ARP2/3 and myosin. Nuclear filamentous actin serves as a site for telomerase recruitment, which is mediated by telomere tethering on actin fibers in response to replication stress, allowing telomerase to localize to telomeres containing stalled replication forks. Overall, these data demonstrate that, in human cells which express telomerase, telomeric replication stress triggers the recruitment of telomerase to telomeres via a nuclear actin network, enabling telomere length maintenance. Telomerase recruitment to telomeres is a tightly regulated process which is stimulated by replication stress. Here, the authors identify that nuclear filamentous actin is important for interaction between telomerase and telomeres, ultimately facilitating productive telomere extension by telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b5841386203be359af687a74dbd6f41356ead9" target='_blank'>
              Nuclear actin and DNA replication stress regulate telomere maintenance by telomerase
              </a>
            </td>
          <td>
            Ashley Harman, Melissa Kartawinata, Nohad M. Maroun, Darren R. Nguyen, Shabita Rahman, William E. Hughes, Kevin Winardi, Scott B. Cohen, Anthony J. Cesare, Noa Lamm, T. M. Bryan
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a5c7aee5589d3f24718b4f0d1ab22f036f7184d" target='_blank'>
              Retrotransposon Activation in the Aged and Alzheimer’s Disease Brain Examined by Nanopore Long-read DNA Sequencing
              </a>
            </td>
          <td>
            M. Kelsey, Anjalika Chongtham, J. LaCava, Martin S. Taylor, J. Boeke, F. H. Gage, A. Seluanov, Vera Gorbunova, Ana C. Pereira, J. Sedivy
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410d0943e5dad068f895d42bbe237bbb7cbc02b7" target='_blank'>
              DNA topoisomerase I acts as supercoiling sensor for bacterial transcription elongation
              </a>
            </td>
          <td>
            Vita Vidmar, Céline Borde, Lisa Bruno, Nataliya Miropolskaya, Maria Takacs, Claire Batisse, C. Saint-André, Chengjin Zhu, O. Espéli, Valérie Lamour, A. Weixlbaumer
          </td>
          <td>2025-12-01</td>
          <td>Nature Structural & Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Spindle cell lipomas (SCL) and pleomorphic lipomas (PL) are today considered a single tumor entity (SCLPL). Atypical SCLPL, in contrast, represents a recently recognized related but distinct entity. Here we explored the correlation between genomic features and morphological aspects of SCLPL and atypical SCLPL and the role of the RB1 gene in tumor development. Seventy‐one samples from 68 patients with SCLPL or atypical SCLPL, as well as a lipomatous tumor from a retinoblastoma patient with a germline pathogenic variant in the RB1 gene, and two pleomorphic liposarcomas were analyzed using chromosome banding, high‐resolution genomic arrays (SNP array), whole exome sequencing (WES), and/or RNA sequencing (RNA‐seq). Common for all tumors was involvement of 13q; other recurring variants were deletion of 16q, 6q, and 17p. A minimally deleted region that only contained RB1 was found on 13q. A distinction was seen between SCL on the one hand and PL and atypical SCLPL on the other; SCL had fewer copy number aberrations in general, and loss of 17p/TP53 gene or a SNV affecting TP53 was only rarely detected in SCL but seen in the vast majority of PL and atypical SCLPL tumors. Thus, at the molecular level, SCL is different from PL/atypical SCLPL. Furthermore, the finding of the same copy number changes (loss of 13q, 6q, 16q, and 17p) in some pleomorphic liposarcomas raises the possibility that a subset of SCLPL/atypical SCLPL have the potential for malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b1988ef8c66c3a7438406d89e232e1e91d6e09" target='_blank'>
              The Role of RB1 and Secondary Genomic Changes in the Development of Spindle Cell and Pleomorphic Lipomas
              </a>
            </td>
          <td>
            Maria Hellberg, V. Difilippo, Emilia Gottberg, A. Hesla, Felix Haglund de Flon, J. Nilsson, L. Magnusson, Saskia Sydow, Paul Piccinelli, Jakob Hofvander, Fredrik Mertens
          </td>
          <td>2025-12-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumor (ATRT) is a highly aggressive but genetically simple pediatric central nervous system tumor, defined by biallelic inactivation of the chromatin regulator SMARCB1 with remarkably few other cooperating mutations. Despite its genetic homogeneity, ATRT exhibits profound clinical and epigenetic heterogeneity, with three major subgroups (ATRT-TYR, ATRT-MYC, and ATRT-SHH) defined by DNA methylation and transcriptional signatures. Beyond these subgroup-defining features, we aimed to investigate epigenetic variability within tumors by applying whole-genome bisulfite sequencing and probabilistic modeling to quantify stochastic DNA methylation in primary ATRT samples encompassing all three subgroups. We show that ATRT exhibits a destabilized and increasingly stochastic methylome. While ATRT global methylation patterns diverge according to subgroup, some methylation perturbations, such as hypermethylation and increased methylation entropy over bivalent promoters, are consistent across subgroups. We find that methylation stochasticity alterations map onto potential drivers of ATRT, such as LIN28a, the HOXD cluster for ATRT-MYC, and OTX2 for ATRT-TYR, and identify actionable targets, such as hypermethylation of the tumor suppressor CDKN2a across all subgroups. We investigate the sensitivity of the aberrant DNA methylation landscape of ATRT to pharmacologic DNA methyltransferase inhibition (DNMTi) and histone deacetylase inhibition (HDACi). We show that decitabine leads to profound demethylation of patient-derived ATRT cell lines, including reversal of hypermethylation at bivalent promoters and the CDKN2a locus. The addition of HDACi leads to dramatic gene expression changes, including upregulation of innate immune signaling pathways, such as STING/interferon signaling, genes under the regulation of bivalent promoters, and reactivation of the tumor suppressor CDKN2A. The combination of DNMTi and HDACi synergistically reduces cell viability. Taken together, we show that ATRT has a highly stochastic methylome sensitive to epigenetic manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e14cbda965cb78ec2d086d8bbd22a534ed3d3a3" target='_blank'>
              Disordered DNA methylation leads to targetable transcriptional plasticity in ATRT
              </a>
            </td>
          <td>
            Ashley R Tetens, Tyler R Findlay, Jordyn Craig-Schwartz, Athanasia Liapodimitri, Oscar Camacho, Kegan O Skalitzky, Adrian Idrizi, Rakel Tryggvadottir, Kayleigh Lunsford, Eric H. Raabe, Michael A Koldobskiy
          </td>
          <td>2025-12-17</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tandem amplification of genomic fragments is quite common in bacteria growing under stress conditions, while spontaneous genome amplification events are rare, unstable and generally poorly described. Plague pathogen Yersinia pestis is a unique microorganism that contains an enormous number of short repeat sequences in its genome and as a result is very prone to spontaneous genome rearrangements including large tandem genome amplification events. Eleven Y. pestis strains sequenced during this study and more than thousand read archives from SRA were analyzed in this study. It was shown that genomes of more than half of Y. pestis laboratory isolates contain tandem repeats. They are mainly caused by the presence of multicopy IS-elements but a few of them are associated with multicopy rRNA clusters, so the rearrangement mechanism is most likely RecA-dependent recombination. Four regions with unstable copy number reproduced between different bioprojects were found. One of them was identified as an integrative mobilizable element carrying a probably incomplete Type 4 secretion system. More interesting, two other reproducible regions were not identified as mobile elements but had the length and GC-content almost identical to the length and GC-content of pMT1 and pCD1 plasmids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bf8e6e37f6935779620869a8b92594a383a1b6" target='_blank'>
              Systematic analysis of spontaneous tandem genome amplification events in Yersinia pestis
              </a>
            </td>
          <td>
            D. Konanov, Olga N. Liubimova, Alexander V. Kovrizhnikov, Ignat V. Sonets, A. N. Balykova, Alexandra V. Lukina-Gronskaya, Anna S. Speranskaya, D. Krivonos, G. A. Eroshenko, Elena N. Ilina, Vadim M. Govorun, V. V. Kutyrev
          </td>
          <td>2025-12-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5e464495ba1f67ebcfcfd2e8d42c98c8759239d" target='_blank'>
              A data-driven chromatin model reveals spatial and dynamic features of genome organization.
              </a>
            </td>
          <td>
            A. B. Oliveira Junior, Matheus F. Mello, Ronaldo J Oliveira, Esteban Dodero-Rojas, Sumitabha Brahmachari, V. Contessoto, J. Onuchic
          </td>
          <td>2026-01-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Retinoblastoma is the most common intraocular malignancy of childhood, yet its genomic landscape remains incompletely defined, particularly in understudied populations. Beyond RB1 loss, the contribution of additional somatic and germline alterations to disease heterogeneity and clinical behavior is unclear.



 We performed whole-exome sequencing of 166 retinoblastoma samples from 166 patients with matched germline DNA, representing the largest cohort analyzed to date. Clinical data were available for 160 patients. Variant calling, copy number alteration (CNA) profiling, and integrative analyses were performed to characterize genetic drivers and their associations with clinical features.



 Pathogenic RB1 variants were identified in 120 patients, and MYCN amplification in six patients. Additional recurrent alterations involved BCOR, CCND3, ERBB2, and PDGFRB. Copy number gains of 6p (41.3%) and 17q (8.1%) were significantly associated with high-risk features including rubeosis, subretinal seeding, and tumor extension beyond the lamina cribrosa. Germline ERBB2 variants correlated with orbital invasion, while germline PDGFRB variants were associated with second primary cancers. Together, these findings underscore the genetic heterogeneity of retinoblastoma and reveal novel genotype–phenotype correlations.



 This study provides the most comprehensive genomic characterization of retinoblastoma to date, expands the known mutational spectrum, and identifies biomarkers with direct clinical relevance. These insights have potential to refine risk stratification, inform precision therapeutic strategies, and improve long-term outcomes for children with retinoblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca1a82a307761a6281f02514b112932e6b6ae23" target='_blank'>
              Genomic landscape of retinoblastoma: Insights into risk stratification and precision pediatric Neuro-Oncology
              </a>
            </td>
          <td>
            A. Maktabi, Yang Liu, Saleh A. Almesfer, M. Abdelhakim, Hind Aldakhil, Maxat Kulmanov, Deepak P. Edward, R. Hoehndorf, M. Abedalthagafi
          </td>
          <td>2025-12-12</td>
          <td>Neuro-Oncology Pediatrics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kaposi sarcoma‐associated herpesvirus (KSHV), a gamma‐herpesvirus, is the main etiological agent of several tumors, including Kaposi's sarcoma (KS) and primary effusion lymphoma (PEL). The large, double‐stranded viral genome of KSHV is maintained as a latent episome in the nucleus of host cells, where a small subset of viral genes is expressed that facilitate evasion of immune responses and promotion of cell survival and proliferation supporting tumorigenesis. The major latency‐associated nuclear antigen (LANA) is an essential viral factor that is required for genome replication and segregation to maintain viral genomes within dividing cells. Given the essential role of LANA to maintain viral latency, efforts have focused on targeting LANA's role in replication and segregation as a mechanism to overcome latency and tumorigenesis. However, given the complexity in small molecule targeting to DNA‐binding domains, current efforts focused on drugging LANA continue to reach potency milestones for clinical trials. Here, we developed a HaloTag‐based PROTAC model as a proof‐of‐concept for targeted LANA protein degradation. Our findings highlight the power of a PROTAC‐based strategy on eliminating LANA and viral persistence in tissue culture demonstrating promise in the on‐going challenge for targeting this major viral factor in KSHV latency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfa79b63dd2067eec7b314131de284f9e1fe9922" target='_blank'>
              Targeted Protein Degradation of the Latency‐Associated Nuclear Antigen Evicts Kaposi Sarcoma‐Associated Herpesvirus Episomes From Infected Cells In Vitro
              </a>
            </td>
          <td>
            Sarah McMahon, Maha Amer, Brooke Hall, Ritu Shekhar, Rolf Renne
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e82aa8297f80f569a06497cad1d25c2344da9284" target='_blank'>
              The 3D Genome of Gigaspora margarita Unveils Stable Chromatin and Nucleolar Organization and Symbiont-Dependent Genome Dynamics
              </a>
            </td>
          <td>
            Ken Mugambi, Jordana Oliveira, F. Magurno, Alessandra Salvioli di Fossalunga, M. Novero, L. Lanfranco, Stefano Ghignone, Gokalp Yildirir, Yan Wang, Paola Bonfante, Nicolas Corradi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5 A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21b610301b7b55b77a33ac6317a40f28dcdb985" target='_blank'>
              A gastric microbial chromatin remodeler drives gastric cancer progression and immune evasion by reprogramming the host epigenome
              </a>
            </td>
          <td>
            Xiaoshan Xie, Yue Wei, Zhikai Zheng, Jiaying Zheng, Xijie Chen, Jiarui Wang, Ning Ma, Xiaoling Huang, Peng Zhang, Boyu Zhang, Hanyong Cai, Li Ma, Lishi Xiao, Qingxin Liu, Wenyu Wang, Sachiyo Nomura, Shi Chen, Xiangqi Meng, Mong-Hong Lee
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background and objectives Somatic mutation patterns in cancer remain largely unexplored outside humans, despite their significance for aging and oncogenesis. Chimpanzees (Pan troglodytes), sharing >98% genomic similarity with humans, display markedly different cancer spectra. To gain comparative insights into cancer susceptibility and resistance, we sequenced chimpanzee hepatocellular carcinoma (HCC) genomes and analyzed their mutational profiles alongside human counterparts. Methodology HCC and matched non-cancerous tissues from five chimpanzees were examined using histopathology, immunohistochemistry (β-catenin, ARID1A, TSC2, FAP, vimentin, TGF-β), whole-genome sequencing (one pair), and whole-exome sequencing (four pairs). Somatic variants were identified with GATK MuTect2, annotated with Ensembl VEP, and analyzed for functional enrichment. Comparative analyses were performed with subsets of human HCC datasets (TCGA, ICGC) including TSC2-positive and TSC2-negative cases. Results Chimpanzee HCCs exhibited histological and immunohistochemical features similar to human tumors but displayed sharply divergent genomic landscapes. Chimpanzee tumors carried significantly higher coding mutation loads (mean 5632 per sample vs. 96–275 in humans). Non-synonymous TSC2 mutations occurred in 80% of chimpanzees, versus ~7% in human HCC, suggesting a species-specific oncogenic pathway linked to the scirrhous subtype. Additional recurrently mutated genes included ARID1A, FAT1–4, TP53, and FGA. Despite greater heterogeneity in chimpanzee tumors, humans showed stronger enrichment of non-synonymous single nucleotide variants, implying more intense positive selection. Shared alterations across species involved canonical drivers such as TP53, CTNNB1, FAT4, and TTN. Conclusions and implications Chimpanzee HCCs are defined by high mutational burden and frequent TSC2 alterations, contrasting with the more selectively constrained mutation spectrum of human HCC. Divergent evolutionary patterns highlight species-specific oncogenic routes while underscoring conserved pathways. Comparative primate cancer genomics offers novel insights into cancer evolution, biomarkers, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7fdead0d4830205015111b5a2fbfd3bc9dc0ec" target='_blank'>
              Divergent evolution of hepatocellular carcinoma genomes in chimpanzees and humans
              </a>
            </td>
          <td>
            Lin Kang, K. Michalak, Robin T. Varghese, R. Anandakrishnan, Edward J Dick, Z. A. Abd Elmageed, Pawel Michalak
          </td>
          <td>2025-12-15</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Ewing sarcoma is driven by the EWSR1–FLI1 fusion oncoprotein, a transcription factor created by chromosomal translocation in otherwise low-mutation tumors. EWSR1–FLI1 recruits chromatin-remodeling complexes, opens chromatin, and establishes broad H3K27ac/H3K4me1 domains. Uniquely, it binds normally silent GGAA microsatellites dispersed across the genome and converts them into de novo enhancers, elevating transcription of downstream targets. We hypothesize that EWSR1–FLI1 engagement of GGAA repeats globally destabilizes chromatin and enforces a differentiation block, and that direct inhibition of its DNA binding will restore transcriptional control. To test this, we established a propidium iodide (PI) uptake assay after exposure to the chromatin-damaging agent CBL0137 as a surrogate for chromatin accessibility, benchmarking ES against non-ES tumor and non-tumor cells, and in parallel engineered a DNA-binding–dependent reporter to screen small molecules that disrupt fusion–DNA interactions. ES cells showed greater PI uptake following CBL0137 than controls, consistent with globally relaxed chromatin. The reporter screen identified candidate compounds that decreased fusion-dependent activity in a dose-responsive manner; orthogonal luciferase assays supported potential on-target inhibition, and viability assays revealed selective toxicity in ES relative to fusion-negative sarcomas. These findings support a model in which fusion oncoproteins promote chromatin relaxation and establish a complementary screening platform that nominates direct fusion inhibitors, providing mechanistic insight and therapeutic starting points for pediatric sarcomas.



 Ayah Salameh. GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c2a09d1dd63318d72cdbf406bfb1ca4ece465c" target='_blank'>
              Abstract A010: GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform
              </a>
            </td>
          <td>
            Ayah Salameh
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is a paediatric liver malignancy arising from hepatic precursor cells, with >90% of cases harbouring a mutation in exon 3 of CTNNB1. We present a fully genetically characterised HB tumour organoid (tumoroid) biobank, which allows for in vitro studies of disease progression and clonal dynamics in vitro. We established a biobank of 14 tumoroid lines from 9 different patients. Tumours and tumoroids were characterised by whole genome sequencing (WGS) and histology, revealing strong concordance in cell morphology and β-catenin staining. In tumour—tumoroid pairs, identical pathogenic CTNNB1 variants were found, alongside shared copy number alterations (CNAs) and mutations. Variant allele frequency (VAF) was consistently higher in tumoroids, indicating increased tumour purity in vitro. In addition to CTNNB1, we frequently observed ARID1A alterations (single-nucleotide variants [SNVs] or CNAs in 56% of patients), and MYC gains as described previously. In paired pre- and post-treatment samples, we observed a clear increase in mutational load, attributed to a chemotherapy signature. Notably, from one patient, we analysed 4 tumour samples (3 post-treatment) with 4 matching tumoroid lines, all carrying a novel BCL6 mutation and loss of ARID1A. Mutational profiles varied across samples from different locations, suggesting intratumoral heterogeneity and clonal selection during tumoroid derivation. Taken together, this biobank allows detailed analysis of HB tumour biology, including treatment-induced progression and clonal dynamics across temporally and spatially distinct samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af5f1ebd1dcbe4bda8108f789b036cfe1c91f713" target='_blank'>
              A Fully Annotated Hepatoblastoma Tumoroid Biobank Details Treatment-Induced Evolution and Clonal Dynamics in Paediatric Cancer Patients
              </a>
            </td>
          <td>
            Gijs J. F. van Son, F. Ringnalda, Markus J. van Roosmalen, Thomas A. Kluiver, Quinty Hansen, E. Duiker, Marius C. van den Heuvel, V. E. de Meijer, R. D. de Kleine, R. R. de Krijger, J. Zsiros, Weng Chuan Peng, R. van Boxtel, M. van de Wetering, Karin Sanders, Hans Clevers
          </td>
          <td>2026-01-18</td>
          <td>Organoids</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) can be induced by cellular byproducts or genotoxic agents. Improper processing of these lesions leads to increased genome instability, which constitutes a hallmark of pathological conditions and fuels carcinogenesis. DSBs are primarily repaired by homologous recombination (HR) and non-homologous end joining (NHEJ) and the proper balance between these two pathways is finely modulated by specific molecular events. Here, we report that the histone chaperone DAXX plays a fundamental role in the response to DSBs. Indeed, in human cells, DSBs induce ATM/ATR-dependent phosphorylation of DAXX on serine 424 and 712 and promote its binding to chromatin and the deposition of the histone variant H3.3 in proximity to DNA breaks. Enrichment of H3.3 at DSBs promotes 53BP1 recruitment to these lesions and the repair of DNA breaks by HR pathways. Moreover, H3.3-specific post translational modifications, particularly K36 tri-methylation, play a key role in these processes. Altogether, these findings indicate that DAXX and H3.3 mutations may contribute to tumorigenesis-enhancing genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/009a4d5a2fca64469e1daafccf7d54c8bc81a3b2" target='_blank'>
              Daxx-Dependent H3.3 Deposition Promotes Double-Strand Breaks Repair by Homologous Recombination
              </a>
            </td>
          <td>
            L. Zannini, Simona Aliprandi, Domenico Delia, G. Buscemi
          </td>
          <td>2026-01-16</td>
          <td>Cells</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background & objectives Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer. Methods This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6low (negative/weak) or SMC6high (moderate/strong). Results SMC6low tumours tend to show higher CNV. SMC6high group presented poorer disease-free survival than the SMC6low group (P=0.050), mainly for the luminal subtype (P=0.005). SMC6low/ERpos were protective biomarkers for recurrence. Interpretation & Conclusions There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-11-01</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences capable of self-replication (especially retrotransposons) within the genome, which may lead to various forms of DNA damage. The introduction of this review encompasses the diverse classes and subclasses of TEs, particularly emphasizing the most active TEs present in the human genome. An analysis of the retrotransposition process of TEs is presented, illustrating how this mechanism can result in DNA damage and gene rearrangements. Furthermore, the review meticulously examines the implications of TE insertions on gene expression and genomic organization, which may contribute to the development of various diseases, including cancer. The relationship between TE activation and the aging process is also explored, with an emphasis on that epigenetic modifications associated with aging can lead to the derepression of TEs, thereby promoting genomic instability and inflammation. These factors may play a significant role in the pathogenesis of age-related diseases, such as cancer, cardiovascular disorders, and neurodegenerative conditions. Finally, the review considers potential therapeutic approaches aimed at targeting TE activity to alleviate the impacts of aging and associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1552ca333c184d411cd1fdc06105d2ca4b0a5435" target='_blank'>
              The role of transposable elements activity in genomic instability and their relationship to aging process.
              </a>
            </td>
          <td>
            Jingran Hu, Tianhao Mao, Kainan Huang, Shangzhi Yang, Wenrui Yu, Jiacheng Huang, Shiqi Jin, Chuanyu Sun, Zeyidan Jiapaer, Xianli Wang
          </td>
          <td>2026-01-18</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is one of the most common cancers and a leading cause of cancer-related mortality worldwide, highlighting the need for novel therapeutic strategies. Proteasome 26S Subunit, Non-ATPase 12 (PSMD12), a component of the proteasomal 19S regulatory particle, is associated with tumorigenesis; however, its role in LUAD remains poorly understood. Integrative bioinformatic analysis of The Cancer Genome Atlas (TCGA) and other publicly available LUAD datasets identified PSMD12 as a candidate driver gene on chromosome 17q, a region frequently amplified in LUAD. Clinicopathological and prognostic analyses revealed that PSMD12 was significantly upregulated in tumor tissues because of DNA copy number gain. High PSMD12 expression was associated with poor prognosis and advanced pathological stages. Gene set enrichment analysis of TCGA LUAD dataset demonstrated that samples with high PSMD12 expression were enriched for cell cycle-related pathways. Using CRISPR-Cas9-mediated PSMD12 knockout and lentivirus-mediated overexpression models, we demonstrated that PSMD12 promoted tumor cell proliferation by accelerating the G2/M cell cycle transition in vitro, and xenograft experiments confirmed its tumor-promoting effect in vivo. Mechanistically, PSMD12 overexpression reduced the ubiquitination of CDK1, a key regulator of mitotic entry. Cycloheximide chase and MG132 assays confirmed that PSMD12 stabilized CDK1 by inhibiting proteasome-mediated degradation. In conclusion, we identified PSMD12 as a novel driver gene and prognostic biomarker of LUAD. PSMD12 promoted LUAD progression by modulating CDK1 ubiquitination and enhancing cell cycle progression. These findings suggest that PSMD12 is a promising molecular target for future LUAD therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb3bc3b62a546392f132deef63cdce1ec580a2a" target='_blank'>
              PSMD12 Overexpression Promotes Lung Adenocarcinoma Progression via Ubiquitin-Proteasome Pathway Dysregulation.
              </a>
            </td>
          <td>
            Yuya Ono, Hajime Otsu, Takaaki Masuda, K. Kosai, Shohei Shibuta, Kosuke Hirose, Takashi Ofuchi, Yuki Ando, Koto Kawata, Yasuo Tsuda, Yusuke Yonemura, Taro Tobo, T. Takenaka, T. Yoshizumi, Koshi Mimori
          </td>
          <td>2026-01-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Identifying viruses in tumor transcriptome helps to unravel the potential role of viruses in oncogenesis and tumor progression. Most of the current tools for virus identification in RNA-Seq data rely on sequence alignment, whose performance is constrained by fast mutations, large divergence, and the incompleteness of viral genomes. In this study, we develop ViTrace to detect viral sequences in human transcriptomic data by a hybrid language representation learning model, which integrates DNA contexts, position relationships and amino acid coding information. Although ViTrace is only trained on 13 species from 7 genera, it achieves 86.39% recall in 1179 absent-in-train virus strains of 935 species belonging to 167 genera across 10 phyla. Applied to single-cell RNA-seq data from esophageal and oropharyngeal squamous cell carcinomas, the model reveals tumor-, cell-, and patient-specific viral colonization patterns, uncovering both known and previously unreported viruses. Overall, ViTrace provides a scalable framework for guiding precision oncology and facilitating future discoveries of previously uncharacterized tumor-associated viruses. Hybrid language model ViTrace identifies viral sequences from human transcriptomes, revealing tumor-, cell-, and patient-specific viral colonization patterns and uncovering novel tumor-associated viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235fa47ea6e35bee83f6f73e3fb62ee13668fc7d" target='_blank'>
              ViTrace detects viral signatures in tumor transcriptomes using a hybrid language model
              </a>
            </td>
          <td>
            Feng Zhou, Yushuang He, Fan Yang, Jin Gu, Fengzhu Sun, Shunzhi Zhu, Xiaobing Huang, Ying Wang
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Following cell entry, HIV-1 capsids enter the nucleus by passage through nuclear pores and reach nuclear speckles with subsequent uncoating of the reverse-transcribed genome and its integration into speckle-associated chromatin domains. Here, we characterized the ultrastructure of HIV-1 subviral complexes in nuclei of primary monocyte-derived macrophages and cell lines using live-cell imaging, super-resolution microscopy, and correlative light and electron tomography in the absence and presence of capsid-targeting inhibitors Lenacapavir and PF74. Capsid-like structures containing viral DNA, as well as broken capsids, clustered in nuclear speckles and were displaced from speckles by drug treatment. This was accompanied by alteration of the nuclear capsid structure, with electron-dense protrusions emanating from the narrow end of capsid cones and exposure of integration-competent genomic HIV-1 DNA. Our data indicate that synthesis of genomic dsDNA can be completed inside the closed HIV-1 capsid, and speckle-associated factors could regulate genome uncoating. This may ensure that genome uncoating occurs at optimal sites for integration into transcriptionally active chromatin. The results also shed further light on the mechanism of action of Lenacapavir. The mechanism of HIV-1 genome uncoating is currently not well understood. This work shows that capsid-targeting drugs lay bare integration-competent genomes enclosed in nuclear HIV-1 capsids, indicating that reverse transcription can be functionally completed inside the closed capsid. Conical and broken capsids cluster in nuclear speckles of primary monocyte-derived macrophages. Lenacapavir or PF74 displace CPSF6 from nuclear capsids, resulting in rapid exit of subviral structures from nuclear speckles. Lenacapavir or PF74 also expose previously hidden genomes, indicating that reverse transcription by itself is not sufficient to induce immediate uncoating. Brief Lenacapavir pulses increase the amount of integrated provirus in macrophages, indicating integration competence of the exposed genomes. CLEM-ET reveals Lenacapavir-induced bifurcated protrusions at the narrow end and flattened lattice remnants. Conical and broken capsids cluster in nuclear speckles of primary monocyte-derived macrophages. Lenacapavir or PF74 displace CPSF6 from nuclear capsids, resulting in rapid exit of subviral structures from nuclear speckles. Lenacapavir or PF74 also expose previously hidden genomes, indicating that reverse transcription by itself is not sufficient to induce immediate uncoating. Brief Lenacapavir pulses increase the amount of integrated provirus in macrophages, indicating integration competence of the exposed genomes. CLEM-ET reveals Lenacapavir-induced bifurcated protrusions at the narrow end and flattened lattice remnants. Exposure of integration-competent genomes upon treatment with capsid-targeting drugs reveals that HIV-1 reverse transcription can be completed in closed viral capsids without automatically inducing uncoating.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f8abb290a62c5b3aa53581143c941a3ce167f6" target='_blank'>
              Lenacapavir-induced capsid damage uncovers HIV-1 genomes emanating from nuclear speckles
              </a>
            </td>
          <td>
            Thorsten G. Müller, Severina Klaus, V. Zila, Bojana Lucic, C. Penzo, Svenja L. Nopper, Gonen Golani, Maria Anders-Össwein, Vera Sonntag-Buck, A. Heuser, U. Schwarz, V. Laketa, M. Lusic, Barbara Müller, H. Kräusslich
          </td>
          <td>2025-12-01</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbea23483bb3d1849086cf711f40322d375ebc8f" target='_blank'>
              NAT10 triggers colorectal cancer progression via promoting PPAN-regulated DNA damage repair
              </a>
            </td>
          <td>
            Haoran Wang, Lichen Ge, Jianing Li, Ke Zhong, Shanzhi Li, Ningjing Ma, Lijun Tao, Jiawang Zhou, Zhaotong Wang, Xing Chang, Yunqing Lu, Yalan Rui, G. Xie, Weifeng Yang, Zuanzong Xu, A. Saad, Xiansong Wang, Zhuojia Chen, Wanglin Li, Cheng Yi, Hongsheng Wang, Kun Zhang
          </td>
          <td>2025-12-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Trichomonas vaginalis is a common sexually transmitted parasite that colonizes the human urogenital tract, causing infections that range from asymptomatic to highly inflammatory. As an extracellular pathogen, adherence to host epithelial cells is an important step to colonize the human host. Hence, understanding how the process of attachment to host cells is regulated remains an important goal in parasitology research and human health. T. vaginalis-host interaction is regulated by changes in gene expression, but it is still largely unknown how these changes in transcriptional profiles are controlled, as very few transcriptional regulatory elements have been described. Our recent work highlighted the importance of epigenetics in the regulation of transcription, and a specific role for N6-methyladenine (6mA) in modulating three-dimensional chromatin structure has been suggested. Building on these findings, we analyzed here the role of 6mA and chromatin accessibility during the process of host-parasite interaction by integrating MeDIP-seq and assay for transposase-accessible chromatin sequencing data with RNA-seq in free vs host cell-attached parasites. Consistent with our previous results, we identified transcriptionally active and repressive regions flanked by 6mA modifications, observed both in the presence and absence of host cells. Importantly, we detected differentially accessible chromatin regions that influence the gene expression of key pathogenesis-related genes during T. vaginalis host cell interaction. These findings highlight the importance of chromatin architecture in regulating gene expression during parasitic infection. IMPORTANCE Trichomonas vaginalis, the most common non-viral sexually transmitted parasite, relies on adherence to host epithelial cells to establish infection. Our previous work highlighted the importance of N6-methyladenine (6mA) DNA methylation in the regulation of transcription and three-dimensional chromatin structure. Now, our study integrates RNA-seq, MeDIP-seq, and assay for transposase-accessible chromatin sequencing data to reveal how 6mA and chromatin accessibility modulate gene expression during T. vaginalis interaction with human host cells. We identified over 3,600 differentially expressed genes upon parasite contact with prostate cells, including pathogenesis-related genes. Moreover, we identified transcriptionally active and repressive regions flanked by 6mA that remain largely stable during the process of host interaction. We mapped genome-wide chromatin accessibility and uncovered differentially accessible regions upon host cell contact associated with a subset of genes involved in adhesion. These results suggest that local chromatin accessibility has a major role in modulating gene expression of key virulence genes during host interaction. Trichomonas vaginalis, the most common non-viral sexually transmitted parasite, relies on adherence to host epithelial cells to establish infection. Our previous work highlighted the importance of N6-methyladenine (6mA) DNA methylation in the regulation of transcription and three-dimensional chromatin structure. Now, our study integrates RNA-seq, MeDIP-seq, and assay for transposase-accessible chromatin sequencing data to reveal how 6mA and chromatin accessibility modulate gene expression during T. vaginalis interaction with human host cells. We identified over 3,600 differentially expressed genes upon parasite contact with prostate cells, including pathogenesis-related genes. Moreover, we identified transcriptionally active and repressive regions flanked by 6mA that remain largely stable during the process of host interaction. We mapped genome-wide chromatin accessibility and uncovered differentially accessible regions upon host cell contact associated with a subset of genes involved in adhesion. These results suggest that local chromatin accessibility has a major role in modulating gene expression of key virulence genes during host interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a18326734a8af0f8385e13e0eb222d101b7528" target='_blank'>
              Influence of DNA methylation and chromatin accessibility on regulation of gene expression during Trichomonas vaginalis-host cell interaction
              </a>
            </td>
          <td>
            Daniela Muñoz, Ayelén Lizarraga, Patricia J. Johnson, P. Strobl-Mazzulla, N. de Miguel
          </td>
          <td>2025-12-03</td>
          <td>mBio</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf9f556241d0007b43f3963aee0e9a310306573" target='_blank'>
              lncRNA EGFR-AS1 promotes DNA damage repair by enhancing PARP1-mediated PARylation.
              </a>
            </td>
          <td>
            Kun Gao, Ruiya Shi, Chenying Xu, Zhaoyang Mao, Changying Guo, Liang Jin
          </td>
          <td>2026-01-05</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: Non-homologous end joining (NHEJ) is a crucial pathway for repairing DNA double-strand breaks and a key contributor to chemoresistance in cancer. The assembly of the DNA Ligase IV (LIG4)–XRCC4 complex is essential for NHEJ fidelity, however, the regulatory mechanisms governing this complex in cancer remain poorly understood. This study aims to investigate whether and how lactate, a key metabolic byproduct of the Warburg effect, regulates the XRCC4–LIG4 complex and influences chemoresistance. Methods: The functional role of lactate in NHEJ was assessed using DNA repair reporter assays in ovarian cancer cells. Protein–protein interactions were examined through co-immunoprecipitation and pull-down assays. The molecular mechanism of lactate’s action was delineated using a combination of site-directed mutagenesis, in vitro binding assays, and molecular docking. Finally, the physiological relevance of lactate-mediated NHEJ was validated in a preclinical ovarian cancer mouse model treated with cisplatin. Results: We demonstrated that lactate enhances NHEJ repair efficiency and confers resistance to DNA-damaging chemotherapeutics. Mechanistically, lactate directly binds to XRCC4 at key residues, including Y66, E55, and S110, thereby strengthening the XRCC4–LIG4 association. This interaction is independent of protein lactylation. In vivo studies confirmed that lactate-driven NHEJ promotes chemoresistance in ovarian cancer. Conclusions: Our findings reveal lactate as a novel metabolic regulator of the NHEJ pathway by directly allosterically modulating the XRCC4–LIG4 complex. This work establishes a direct molecular link between the Warburg effect and DNA repair-driven chemoresistance, offering new insights into potential therapeutic strategies for ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85e9b949622bf51d02f40bafe00f73cb3235d18e" target='_blank'>
              Lactate Enhances Non-Homologous End Joining Repair and Chemoresistance Through Facilitating XRCC4–LIG4 Complex Assembly in Ovarian Cancer
              </a>
            </td>
          <td>
            Jingyi Lu, Jiayu Zhu, Huanxiao Zhang, Zhou Zhou, Haoyuan Li, Cuimiao Zheng, Xi Huang, Siqi Chen, Chaoyun Pan, Jie Li, Hao Tan
          </td>
          <td>2025-11-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Micronuclei are formed during cell division when acentric fragments or lagging chromosomes cannot be incorporated into the primary nucleus. Macroautophagy/autophagy may reduce chromosomal instability (CIN) by clearing isolated, atypical micronuclei. Other studies implicate that the loss of autophagy disrupts DNA repair pathways. However, whether aberrant mitosis contributing to CIN occurs when autophagy is inhibited has yet to be evaluated. We found impaired autophagy initiation contributes to CIN and facilitates the formation of micronuclei and other abnormal nuclear phenotypes either by genetic or pharmacological manipulation in multiple cell lines. We also found that loss of the integral autophagy protein ATG9A resulted in various types of mitotic errors that can contribute to the formation of micronuclei. ATG9A also localizes to centrosomes and midbody during cell division. Autophagy inhibition causes the overactivation and mislocalization of TBK1 (TANK binding kinase 1) into cytoplasmic, punctate structures that colocalize with SQSTM1/p62. This overactivation interferes with its function in cell division as a mitotic kinase and its role at the centrosome. These results indicate that loss of autophagy contributes to genomic instability from multiple angles, one of which being aberrant cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7652d445dff3bc9d16db0af8750b841d562ae2" target='_blank'>
              Disrupted autophagy overactivates TBK1 and results in mitotic defects promoting chromosomal instability.
              </a>
            </td>
          <td>
            Swagatika Paul, Porter L Tomsick, Julia P. Milner, S. Biswas, Samantha Brindley, Nicole DeFoor, Leila Zavar, Grace Wright, Yairis Soto, A. Pickrell
          </td>
          <td>2026-01-16</td>
          <td>Autophagy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Homology search is a means through which DNA double-strand breaks (DSBs) explore the genome for sequences that enable error-free repair, known as homologous recombination. A better understanding of this search process is fundamental to the relationship between higher-order chromosome organization and DNA damage. Here, we use an entropic bead-spring polymer chain model to simulate the spatiotemporal dynamics of the yeast genome during interphase. The chromosome is organized by transient and dynamic cross-links representing structural maintenance of chromosome (SMC) complexes. DNA damage is modeled as a break in the bead-spring chain, coupled with a removal of crosslinks from beads proximal to the break site. We show that the removal of cross-links drives the exploration of genomic space by the damaged ends, while rates and densities of intact dynamic crosslinking have only a minor role. Local depletion of SMC cross-links proximal to the break site enables the damaged segment to escape the chromosome territory and enhances its ability to explore the genome. Our study reveals a foundational principle by which DSBs can encounter distant regions of sequence homology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/388d042975052534a72a2fe02098ce9e9ab9f1ee" target='_blank'>
              The Role of Transient Crosslinks in the Chromatin Search Response to DNA Damage
              </a>
            </td>
          <td>
            Andrew T. Atanasiu, Caitlin Hult, Daniel Kolbin, Ben Walker, M. G. Forest, E. Yeh, Kerry S Bloom
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Apurinic/apyrimidinic endonucleases - APE1 and APE2 are central to genome maintenance and the cellular DNA damage response, with expanding relevance in cancer biology. APE1 is the primary endonuclease in base excision repair and functions as a redox coactivator of transcription factors. In contrast, APE2 exhibits PCNA dependent 3′–5′ exonuclease and 3′-phosphodiesterase activities, contributing to microhomology-mediated end joining, ATR-Chk1 activation, and immunoglobulin diversification. Both enzymes are often deregulated in cancer: APE1 is frequently overexpressed, drives tumor progression and chemoresistance, while APE2 is similarly upregulated in multiple malignancies. APE1 can be targeted by redox-specific or endonuclease inhibitors, with early clinical evidence of biological activity and tolerability. Although APE2-specific inhibitors remain in early development, emerging synthetic lethality data and preclinical studies highlight APE2 as a novel clinical target in breast cancer type 1/2 susceptibility (BRCA)-mutated cancers. This review discusses the structural and functional roles of APE1 and APE2, their contributions to cancer biology and therapeutics, recent advances in inhibitor development, and future strategies for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42492162323d747cbf3ed4c8e187621438e9d88" target='_blank'>
              Contrasting roles of APE1 and APE2 in genome maintenance, cancer development, and therapeutic targeting
              </a>
            </td>
          <td>
            Aman Sharma, Helen E. Grimsley, Katharine Courtemanche, Simon N. Powell
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cellular differentiation is driven by epigenetic modifiers and readers, including the methyl CpG binding protein 2 (MeCP2), whose level and mutations cause the neurological disorder Rett syndrome. During differentiation, most of the genome gets densely packed into heterochromatin, whose function has been simplistically viewed as gene silencing. However, gene expression changes reported in mutations leading to Rett syndrome have failed to be a predictor of disease severity. Here we show that MeCP2 increases nuclear stiffness in a concentration-dependent manner and dependent on its ability to cluster heterochromatin during differentiation. MeCP2-dependent stiffness increase could not be explained by changes in the expression of mechanobiology-related genes, but we found that it is disrupted by Rett syndrome mutations and correlated with disease severity. Our results highlight the impact of chromatin organization on the mechanical properties of the cell as an alternative or complementary mechanism to changes in cytoskeleton components. A correlative AFM–fluorescence approach elucidates how epigenetic control of chromatin organization contributes to nuclear stiffness and its dysregulation in Rett syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d330823c212a15c08be21ea0eb068997201f78c8" target='_blank'>
              MeCP2-driven chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Andreas Zhadan, Marah Mahmoud, Nina Trautwein, Bodo Laube, Christian Dietz, R. Stark, Cristina Cardoso
          </td>
          <td>2025-12-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="N4BP1 specifically degrades a subset of mRNA targets through their coding sequences and functions as a negative regulator of inflammation; however, its role in cancer development remains undefined. N4BP1 exhibits the highest expression in head and neck squamous cell carcinoma among all analyzed cancer types. Unlike wild-type mice, N4bp1−/− mice did not develop visible tongue tumor masses in a 4-NQO-induced oral carcinogenesis model. Furthermore, N4bp1−/− mice (86% vs 0%) exhibited significantly prolonged survival compared to wild-type mice within 26 weeks in 4-NQO-induced oral carcinogenesis model. Single-cell profiling demonstrated that N4BP1-deficient epithelial cells arrest at an early stage of cancerous transformation, while wild-type epithelial cells efficiently progress to an advanced stage of cancer. In established human cancer cell lines, N4BP1 also plays a crucial role in proliferation, migration, colony formation, and in vivo growth. Transcriptome profiling identified CCL2 and GM-CSF as downstream targets of N4BP1 in oral cancer. Apart from its intrinsic role in cancer cells, N4BP1-deficient cancer cells induce the differentiation of macrophages into the M1 phenotype. In N4BP1-deficient tissues, CCL2 and GM-CSF were significantly increased, accompanied by the accumulation of M1 macrophages and neutrophils. Our results demonstrate that N4BP1 is an essential gene in tongue cancer development. N4BP1 not only drives cancer cell evolution but also establishes an immune-suppressive microenvironment. N4BP1 is an endoribonuclease that specifically regulates a subset of mRNA targets (including CCL2 and GM-CSF) and plays an essential role in oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/453fe4fbe3e2a6087004d4f6f19727a6b189bb9b" target='_blank'>
              N4BP1 is essential for the development of oral cancer via controlling both cancer cells and immune microenvironment
              </a>
            </td>
          <td>
            Yihua Song, Rong Sun, Jie Ji, Wen Zheng, Yanli Li, Xiaohong Guo, Liuting Chen, Yuanyuan Wu, Miaomiao Chen, Xingmei Feng, Mingbing Xiao, Renfang Mao, Yihui Fan
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The formation of RNA-DNA hybrid (RDH) primers by primase is an essential step in the recruitment of DNA polymerase during replication initiation and for the synthesis of each Okazaki fragment on the lagging strand. In addition to primers, RDHs form through misincorporation of ribonucleotides by DNA polymerase during elongation and by formation of R-loops during transcription. R-loops are three-stranded structures that form when the nascent mRNA anneals to the template DNA strand, displacing the complementary DNA strand. The persistence of RDHs is deleterious to genome stability in all cells because they increase susceptibility to mutations, impaired replication fork progression, DNA double-stranded breaks, and genomic rearrangements. In many bacteria, it is well established that components of the replicative DNA polymerase form a macromolecular complex that can be imaged using single-molecule or ensemble fluorescence approaches. The spatiotemporal regulation of proteins involved in RDH removal during lagging-strand maturation is less clear. Here, we study three proteins that are involved in the removal of RDHs from the lagging strand during DNA replication in the Gram-positive bacterium Bacillus subtilis: DNA polymerase I (Pol I), FenA, and RNase HIII. We characterized the behavior of each PAmCherry-tagged lagging-strand enzyme in living cells using single-particle tracking photactivated localization microscopy. In this work, we find that all three proteins are highly mobile, suggesting residence times at their target substrates are below our temporal resolution. We also find evidence that Pol I activity is modulated through interaction with the replisome, whereas FenA and RNase HIII are regulated through access to the nucleoid. Our results provide new insight into how enzymes are recruited to resolve RDHs during lagging-strand replication in vivo. Significance RNA-DNA hybrids (RDHs) are essential, transient intermediates in DNA replication, yet their presence significantly increases the susceptibility of the genome to damage. We characterized the single-molecule behavior of three proteins important for processing RDHs in Okazaki fragments in living bacteria. We find that enzyme activity is modulated by access to the replisome and nucleoid. Specifically, we find that DNA polymerase I is preferentially localized to the replisome, while FenA and RNase HIII dwell times at the replisome are very short and below our detection limit. Our work shows that Pol I, FenA, and RNase HIII turn over rapidly in cells, providing new insight into how lagging-strand replication is coordinated in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db5e47289c016454ae2f3837c6615674a8ecba75" target='_blank'>
              Single-molecule tracking of RNA-DNA hybrid removal enzymes important for lagging-strand replication
              </a>
            </td>
          <td>
            Daniel J. Foust, Frances C. Lowder, Jessica Chung, Julianna R. Cresti, Lieke A. van Gijtenbeek, Lyle A. Simmons, J. Biteen
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Foxp3+ regulatory T (Treg) cells rely on DNA demethylation to establish and maintain the gene expression program that defines their identity and suppressive function. Although sustained transcription of Treg-specific genes is known to drive this demethylation, the precise mechanism has remained unclear. Here, we show that Dot1L-catalyzed methylation of histone H3 at lysine 79 (H3K79me) is essential for Treg-specific DNA demethylation in both thymic and induced Treg lineages. H3K79me promotes chromatin activation and recruits TET family DNA demethylases to key regulatory loci. Treg-restricted deletion of Dot1L disrupts locus-specific DNA demethylation, diminishes expression of core Treg genes, and precipitates a fatal, early-onset autoimmune syndrome. Conversely, pharmacologic enhancement of TET activity during differentiation rescues DNA demethylation, restores Treg-gene expression, and reinstates suppressive function even in the presence of Dot1L inhibition. Together, these findings identify a critical epigenetic axis—Dot1L-mediated H3K79 methylation driving TET-dependent DNA demethylation—that safeguards Treg cellular identity, function, and immune homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542929f5fc79d680747f50c5a7526c3c0b1a5dde" target='_blank'>
              Dot1L licenses DNA demethylation to establish regulatory T cell identity
              </a>
            </td>
          <td>
            James Cameron, Tyler R. Colson, Xudong Li
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Due to its influence in transcriptional potential of genes, genetic regulation by means of epigenetic  mechanisms  is  essential for normal growth and development. In mammary cancer, epigenetic modifications play a key role for its development and progression. In early carcinogenesis stages, due to genetic alterations or environmental factors, chromatin structure alterations due to DNA methylation and post-translational modifications of DNA-bound proteins may appear. As with other types of tumor, genome-wide hypomethylation and hypermethylation of specific genes, particularly in CpG islands that normally are not methylated, are observed. In order to compare global DNA methylation levels between tumor tissue and normal mammary tissue, we studied 11 intact female dogs with mammary tumors. Both types of tissue were collected during surgery, with subsequent clinical staging and histopathological classification of tumors. For each animal, DNA was extracted from paired samples of tumor tissue and normal mammary tissue. Global genomic methylation levels were calculated by relative quantitation of 5-methyl-2’-deoxycytidine (5mdC) with HPLC. Results showed that tumoral tissue had a global DNA hypomethylation when compared with normal mammary tissue (P < 0.05). This difference was greater in high histopathological grade tumors, characterized by their aggressive clinical behavior and high metastatic rate. These findings underscore the importance of additional studies in this line of research, with greater sample sizes. In the future, global DNA methylation may be used as a prognosis biomarker for mammary cancer in dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927c93e449341b53ac59cfa7b6470adfd1ef1f9b" target='_blank'>
              Global DNA hypomethylation in canine mammary tumors
              </a>
            </td>
          <td>
            Alicia Decuadro, María del Carmen Montenegro, Silvia Llambí, N. Balemian, M. Cappetta
          </td>
          <td>2025-12-04</td>
          <td>Revista Científica de la Facultad de Ciencias Veterinarias</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA oxidation is one of the main types of damage to the genetic material of living organisms. Of the many dozens of oxidative lesions, the most abundant is 8-oxoguanine (8-oxoG), a premutagenic base that leads to G→T transversions during replication. Double-stranded DNA can conduct holes through the π system of stacked nucleobases. Such electron vacancies are ultimately localized at the 5’-terminal nucleotides of polyguanine runs (G-runs), making these positions characteristic sites of 8-oxoG formation. While such properties of G-runs have been studied in vitro at the level of chemical reactivity, the extent to which they can influence mutagenesis spectra in vivo remains unclear. Here, we have analyzed the nucleotide context of G-runs in a representative set of 62 high-quality prokaryotic genomes and in the human telomere-to-telomere genome. G-runs were, on average, shorter than polyadenine runs (A- runs), and the probability of a G-run being elongated by one nucleotide is lower than in the case of A-runs. The representation of T in the position 5’-flanking G-runs is increased, especially in organisms with aerobic metabolism, which is consistent with the model of preferential G→T substitutions at the 5’-position with 8-oxoG as a precursor. Conversely, the frequency of G and C is increased and the frequency of T is decreased in the position 5’-flanking A- runs. A biphasic pattern of G-run expansion is observed in the human genome: the probability of sequences longer than 8–9 nucleotides being elongated by one nucleotide increases significantly. An increased representation of C in the 5’-flanking position to long G-runs was found, together with an elevated frequency of 5’-G→A substitutions in telomere repeats. This may indicate the existence of mutagenic processes whose mechanism has not yet been characterized but may be associated with DNA polymerase errors during replication of the products of further oxidation of 8-oxoG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83075f585c67b259c09dcc5cab31e9fb64185e60" target='_blank'>
              DNA damage reflected in the evolution of G-runs in genomes
              </a>
            </td>
          <td>
            I. Grin, D. Zharkov
          </td>
          <td>2025-12-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Clinical genomic sequencing has revealed that prostate cancer (PCa) progression is associated with changes to chromosome number and structure, termed chromosomal instability (or CIN). While mutations in common driver genes are rare in primary PCa, CIN is detected in >90% of cases, suggesting that CIN may be a driver of the disease. However, the underlying mechanisms that cause CIN in PCa are poorly understood. Cells are susceptible to CIN during mitosis, when the duplicated genome must be equally segregated into two daughter cells. Normally, two centrosomes (cytoplasmic, non-membranous organelles) nucleate and organize microtubules to guide assembly of the mitotic spindle, thus ensuring the fidelity of cell division. However, we recently discovered that cells within primary PCa lack centrosomes, the first cancer type to show this phenomenon. Furthermore, we showed that centrosome elimination in non-tumorigenic cells generates CIN which can transform these cells, causing them to form xenograft tumors in mice. Therefore, we sought to understand the underlying mechanisms that drive the centrosome loss observed in PCa. The prostate is a naturally hypoxic organ, averaging around 4% oxygen tension in the healthy organ which declines as men age. Additionally, tumor hypoxia is strongly correlated with high CIN, particularly in prostate cancers. However, there are few mechanistic links between hypoxia and CIN. Therefore, we hypothesized that hypoxia could cause centrosome loss and, thus, predispose cells to CIN. Indeed, we found that immortalized prostate RWPE-1 cells progressively lost centrosomes when exposed to 1% O2 for as short as 6 hours in vitro. Using ultrastructure expansion microscopy (U-ExM), we find centrosomes are replaced by non-functional microtubule husks devoid of centrosome proteins which we call ‘remnants’. These phenomena also occur in PC-3 and LNCaP prostate cancer lines. Furthermore, we observe centrosome loss specifically in the hypoxic regions of xenograft tumors. Using a combination of RNAseq and chemical inhibitors, we determined that centrosome elimination sits at the crossroads of 3 transcriptional programs: (1) centrosome elimination occurs in basal cells and basal-like cancers that express p63; (2) cells must be confluent with active Hippo pathway; and (3) hypoxia induces a HIF-independent transcriptional response. When these conditions are met, cells transcriptionally downregulate 4 key centrosome proteins (Cep63, Cep152, Cep192, and Cep215), resulting in the depletion of a protective shell of pericentriolar material (PCM) and subsequent centrosome elimination. By overexpressing the mitotic kinase Polo-like kinase 1 (PLK1), we can strengthen the PCM and prevent elimination. Finally, we identify PCa cells containing remnants within prostatectomy tissue samples. This work is significant as it identifies the first examples and mechanisms of centrosome elimination in human somatic and cancer cells, provides a mechanistic link between hypoxia and CIN, and identifies remnants as a cellular indicator for CIN predisposition.



 John M. Ryniawec, Natalya K. Seppanen, Gregory C. Rogers, Anne E. Cress. Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0fec5efccdcc580bc977b0a1bb739d683cd475" target='_blank'>
              Abstract B065: Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer
              </a>
            </td>
          <td>
            John M. Ryniawec, Natalya K. Seppanen, Gregory C Rogers, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that dictate complex transcriptional repertoire. Herpes viruses exhibit programmed latent and lytic gene expression depending on the infected tissue and physiological state. Previously, using a systematic functional assay, we identified six enhancers within the genome of Kaposi’s sarcoma-associated herpesvirus (KSHV). In this study, we present a natural language processing model (NLP)-based tool, ENHAvir, that is trained with these six enhancers and non-enhancer control sequences from the KSHV genome. ENHAvir identifies known enhancers and predicted novel enhancer elements in human herpesviruses. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV is confirmed in an enhancer reporter assay. The terminal repeats of all the herpes viruses also serve as a strong enhancer. Comparing herpesvirus enhancers with human enhancers reveals conserved enhancer signatures and the involvement of Alu elements. Here, we present an AI tool that successfully predicts enhancers in both viral and human genomes. Artificial intelligence (AI) can detect and predict patterns that are hidden from the human eye and from conventional homology-detection tools. Enhancers are distal DNA cis-regulatory elements that regulate complex transcriptional repertoire. A natural language processing (NLP) model was trained using only six enhancer sequences from the Kaposi’s sarcoma-associated herpesvirus (KSHV) genome. This tool, termed ENHAvir, can identify known enhancers and predict novel enhancer elements in other herpesviruses, different viruses, and the human genome. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV was confirmed experimentally, enabling the creation of a comprehensive enhancer map of human herpesviruses. All human herpesviruses contain terminal repeats (TRs), which play important roles in cleaving the viral genome into genome-size units, genome encapsidation, and genome circularization following entry into the nucleus of a newly infected cell. This study adds another role for the TR of all human herpesviruses, a strong enhancer with the features of a “viral super enhancer”. Comparing herpesvirus enhancers with human enhancers revealed conserved enhancer signatures and the involvement of Alu elements. Here, an AI tool is presented that successfully predicts enhancers in both viral and human genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980482a475765af78491bb69bee337aba4e9cbe1" target='_blank'>
              Discovering the complete enhancer map of human herpesviruses using a natural language processing model
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Deepanway Ghosal, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-12-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310d13cc1aa7835ffdb46ba871516e1ca05c1d42" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cells are constantly exposed to various sources of DNA damage, including radiation, chemicals, replicative stress and oxidative stress, that threaten genome stability. To ensure faithful DNA repair, transcription regulation needs to be tightly controlled. This regulation involves transcriptional suppression, selective activation of DNA repair-related genes and transcriptional recovery post-repair. Failure to properly modulate transcription during DNA damage can result in collisions between transcriptional and repair machineries, misregulation of repair genes and delayed recovery, ultimately compromising genomic integrity. Chromatin modifications play a central role in this process. These modifications include phosphorylation, methylation, acetylation and ubiquitination, which orchestrate DNA accessibility for repair machinery and fine-tune transcriptional responses. Absence of these modifications leads to inefficient DNA repair and transcriptional errors that are implicated in diseases such as cancer, premature ageing and neurodegenerative disorders. In this review, we delve into the role of various types of histone modifications, such as phosphorylation, methylation, acetylation and ubiquitination and how they regulate transcription in response to DNA damage. Impact Statement This review elucidates how histone modifications orchestrate transcription regulation during DNA damage response, safeguarding genome stability. We also discuss transcription dysregulation in diseases such as cancer and premature aging. Our review provide insights on chromatin-based repair pathways and guide researchers in developing therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7f9af5f72a6e092a88139e5a65f7153c565964" target='_blank'>
              The role of histone modifications in transcription regulation upon DNA damage.
              </a>
            </td>
          <td>
            Angelina Job Kolady, Siyao Wang
          </td>
          <td>2025-11-28</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Papillary renal cell carcinoma (pRCC) is characterized by marked intratumoral heterogeneity, which contributes to therapeutic resistance and disease progression. In this study, we identify STK38 as a key regulator of tumor heterogeneity in pRCC, functioning through non-canonical activation of the Hedgehog (Hh) signaling pathway. STK38 interacts with both KIF7 and GSK3β to promote Hh signaling by facilitating KIF7 ciliary localization and reprogramming GSK3β substrate selectivity, leading to GLI1 stabilization and β-catenin suppression. Moreover, GLI1 directly enhances STK38 transcription, establishing a positive feedback loop that reinforces pathway activation. Notably, depletion of STK38 sensitizes tumor cells to a NETosis-like chromatin release process (tNET release), a form of stress-induced nuclear expulsion associated with immune evasion and metastatic potential. Given the potential pro-metastatic consequences of STK38 inhibition, we instead targeted its downstream effector GLI1 using Glabrescione B, which potently suppressed tumor growth and induced apoptosis in both xenograft and patient-derived organoid models, particularly in STK38-high tumors. These findings position STK38 as a critical modulator of pRCC heterogeneity and support GLI1 inhibition as a promising strategy to disrupt oncogenic signaling while minimizing adverse effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a249254f9dd46bc3c20e8d81598af9b9ed7c0662" target='_blank'>
              STK38-mediated feedback loop regulation of the hedgehog pathway governing tumor heterogeneity in renal papillary carcinoma
              </a>
            </td>
          <td>
            Yifan Du, Xiuyuan Sui, Zeyuan Zheng, Zhengying Zhang, Bin Liu, Yang Bai, Yue Zhao, Qingqing Wu, Haodong Wu, Min Zhong, Liyan Li, Huimin Sun, Chen Shao
          </td>
          <td>2026-01-15</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs), constituting approximately 8% of the human genome, represent genomic remnants of ancestral retroviral infections that colonized the germline through evolutionary processes. While most HERVs remain epigenetically silenced, their reactivation through environmental stimuli or epigenetic dysregulation enables participation in oncogenesis via viral mimicry, immunomodulation, and insertional mutagenesis. Substantial evidence now implicates aberrant HERVs activity across urologic malignancies—including prostate cancer, renal cell carcinoma (RCC), bladder cancer, and testicular germ cell tumors—where cancer-type-specific mechanisms drive tumor development and progression. These encompass androgen-responsive HERV-K activation in prostate malignancies, hypoxia-inducible factor-mediated ERV immunogenicity in RCC, HERV-derived microRNA silencing of tumor suppressors in bladder cancer, and DNA hypomethylation-associated HERV expression in testicular germ cell tumors. This review synthesizes fundamental HERV biology with recent advances in their diagnostic and therapeutic applications for urologic neoplasms. Key clinical translations include ERV-based stratification models predicting immune checkpoint inhibitor response in metastatic RCC, HERV-E-targeted adoptive T cell therapies, and noncoding RNA biomarkers for early bladder cancer detection. We further discuss unresolved mechanistic paradoxes such as contradictory prognostic associations between HERV superfamily expression and PBRM1 inactivation in RCC, concluding with priorities for future research: validating HERV-derived neoantigens in immunotherapy platforms, optimizing epigenetic priming strategies to enhance viral mimicry effects, and establishing standardized HERV signatures as clinical biomarkers through multi-institutional cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38383e2f180ac1b254c2d12832458ab1cd318585" target='_blank'>
              The expression, regulation, and function of human endogenous retroviruses in genitourinary cancers
              </a>
            </td>
          <td>
            Wenjie Ma, Chencheng Ji, Abudukelimu Abudushataer, Ning Liu, Tao Xu, Kunlun Zhao, Yiguan Qian, Paerhati Tuerxun, Xiaotian Jiang, Zhongli Xiong, Min Wang, Ruipeng Jia, Zheng Xu, Yang Li, Yu-Zheng Ge
          </td>
          <td>2025-11-28</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W Snyder, Katherine L Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="
 Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology.
 CDKN2AIPNL
 exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze
 CDKN2AIPNL’s
 expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcaa198969b30b8a12d959cf21f0fe139aa2797" target='_blank'>
              The ERCC6L2-MRI-KU complex coordinates NHEJ at staggered DNA double-strand breaks
              </a>
            </td>
          <td>
            Pia I. Reichl, Ceylan Sonmez, Yueru Sun, Ashleigh King, Jean S. Metson, Benjamin Davies, Adam C. Wilkinson, Francisca Lottersberger, J. R. Chapman
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c6e2a39a6240d55a8cd9334672f5c4459bbc0c" target='_blank'>
              A compendium of chromatin interaction maps in the Giant Panda genome.
              </a>
            </td>
          <td>
            Pengliang Liu, Jiaman Zhang, Kailai Cai, Juan Wang, Hong Liu, Liang Zhang, Wei Xu, Yuliang Liu, Fujun Shen, Rong Hou, Yan Li
          </td>
          <td>2026-01-13</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee41f49d06aaeb019cdfc5399ce87f114ac10230" target='_blank'>
              Clinical Implications of SRPK1 Expression in Human Tumours: A Comprehensive Pan-Cancer Analysis Based on Multi-Omics Databases
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 In Non-Small Cell Lung Cancer (NSCLC), low-fidelity transcription mechanisms such as ecDNA (extra-chromosomal DNA) are an increasingly relevant aspect of progression and prognosis, warranting comprehensive investigation into stability and prevalence of chimeric RNA transcripts. Here, we describe global distribution of fusion transcripts in encounter-matched primary tumor and lymph node metastases and their prognostic implications.



 RNAseq was performed on resected FFPE (encounter-matched primary and lymph nodes) from 121 stage IIB-IIIA (T1-4N1-2M0) NSCLC patients, in addition to 693 Stage IA1-IB cases (T1a-2aN0M0). STAR and STAR-Fusion were used to align data to the GRCh37 genome. Differential gene expression was based on specimen source (primary or lymph node). Aggregate analyses were also performed, including determination of a fusion burden metric, assessing the relationship between primary tumors, lymph node metastases, and recurrence. Deconvolution of Tumor Microenvironment was performed via Quantiseq.



 Comparing primary tumors to paired lymph node metastases, 7,930 ensembl IDs were differentially-expressed at FDR < 0.001 (n=101 pairs). A total of 613 fusions (685 unique transcripts) were detected from paired primaries and lymph node metastases (n = 242). Frequency of specific gene fusions in primaries ranged from 0-0.963 (median 0.019), compared to 0-0.870 (median 0.0093) in LN metastases (p=1.87×10-4). We observed fusion of TULP4-RP11-732M18.3 as pervasive in NSCLC, occurring in 84-87% of primary tumors and lymph nodes, with a 91% retention rate. Conversely, fusions involving CCDC7 or SEPT14 were preferentially observed in metastases, where 82% of CCDC7 fusions and 75% of SEPT14 fusions were not observed in the primary tumor despite occurring at high frequency (66% and 18.5%, respectively) among LN metastases. Interestingly, median fusion burden (unique gene-gene fusions) for primaries was 14 compared to 11 (p=2.06×10-7) in LN metastases (median change -4). Among N0 cases that did not receive adjuvant therapy, median fusion burden in non-recurrent tumors was 15 (n=570), compared to 13 in those that recurred over 3-60 months (n=123; p = 0.04). Interestingly, there was no significant difference in fusion burden between recurrent cases and primary tumors with LN metastases (n = 108). For both recurrent cases and LN-paired primaries, the difference in fusion burden compared to LN metastases (n = 110) was significant (p=2.10×10-4 and 2.06×10-7). A subset of LN metastases (n=27) increased in fusion burden relative to the primary tumor; these primaries uniquely exhibited 33% decrease in regulatory T cell content (p=0.008), 20% decrease in M2 macrophage content (p = 0.01 with M1 macrophages static), and 40% decrease in B-cell content (p=0.05).



 Fusion transcripts are pervasive in RNAseq data, suggesting a broader and more variable landscape than accounted for by primary tumors alone. Concurrent ongoing studies explore the role of ecDNA in fusion burden changes related to NSCLC progression and metastasis.



 Kelly M. Cagin, Keri L. Denson, Sierra R. Broad, Wara M. Naeem, Arsalan M. Khan, Michael M. Liptay, Christopher W. Seder, Jeffrey A. Borgia. A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abeb8dbe1ed35e15732a9cc5bda99a8030ed67" target='_blank'>
              Abstract B027: A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs
              </a>
            </td>
          <td>
            K. Cagin, Keri L. Denson, Sierra R. Broad, Wara Naeem, Arsalan M. Khan, Michael M. Liptay, C.W. Seder, Jeffrey Borgia
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898be89e91659bf810fb7787929dbf8540600811" target='_blank'>
              Advances in functional mechanisms of genomic G-quadruplex structures in transcriptional regulation.
              </a>
            </td>
          <td>
            Zhen-Zhen He, Xiao-Feng Chen, Yue Hou, Tie-Lin Yang, Bo Yang, Yan Guo
          </td>
          <td>2025-12-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA methylation is a most heritable epigenetic modification. Being a major vegetable crop, pepper (Capsicum spp.) possesses an over 3 Gb genome populated with TEs. This indicates a rich reservoir of epigenetic regulatory mechanisms. However, this large and complex genome renders the study of DNA methylation unaffordable and technically challenging. In this study, we analyzed DNA methylome in Capsicum spp., with a focus on C. annuum ST-8. We found that the genomes of Capsicum spp. are heavily methylated, particularly in the non-CG contexts. This is true when comparing to wheat, whose genome is over 16 Gb, containing over 80% TEs and repeats. Interestingly, we observed genic non-CG methylation and found that it is likely maintained by the CMTs, instead of RdDM. Overall, there is a negative relationship between gene expression and H3K9me2, and a positive relationship between genic non-CG methylation and H3K9me2, despite that genes without genic CHH methylation also possess some H3K9me2. Finally, we performed salt stress treatment with and without priming, and profiled active chromatin features as well as transcriptomes. We found that regardless of the environmental stimuli and developmental stages, the overall negative relationship between transcription and H3K9me2 is stably maintained. Altogether, our study revealed features of DNA methylation in ST-8 and we suggest that these features are likely common in Capsicum spp.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee5cf6576edac68d63b332c1994530f8d8f03ba" target='_blank'>
              Features of DNA methylome in Capsicum spp.
              </a>
            </td>
          <td>
            Ye Liu, Yingjie Mi, Manru Song, Hongji Yang, Lulu Liu, Qian Yin, Xueyan Yao, Guorong Yu, Ying Fu, Guochen Qin, Feng Liu, Xuan Huang, Yan Xue
          </td>
          <td>2025-12-10</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct fragmentomic characteristics that are related with their cellular origins. Here, we report that somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like fragmentomic characteristics, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Further investigations suggest that such cancer-like traits are associated with somatic variants derived from clonal hematopoiesis. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilize AI to integrate genomic, fragmentomic, and epigenomic features to develop diagnostic models named FreeSV and FreeSV+. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features are also included, the AUCs of FreeSV+ model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d931b70db2d8495e7d9b12e6b1f59f47b890a1" target='_blank'>
              Cancer-Like Fragmentomic Characteristics of Somatic Variants in Cell-Free DNA.
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2026-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Replication stress involves the slowing or stalling of the replication fork as DNA is copied during S phase. This stress can drive genomic instability, a cancer hallmark. RNA:DNA hybrids, such as R-loops and single genome-embedded ribonucleotides, are significant sources of replication stress. RNA:DNA hybrid homeostasis must therefore be tightly regulated through prevention and removal. Ribonuclease H2 (RNase H2) functions both in R-loop removal and excision of single ribonucleotides from genomic DNA. Recent research has generated new mechanistic insights into the functions of RNase H2 in the replication stress response, and implicated both loss and overexpression of RNase H2 in cancer development and therapy response. These findings help generate new models but also raise new questions. This Review explores the contribution of RNA:DNA hybrids to replication stress, the involvement of RNase H2 in regulating these structures, and the emerging roles of RNase H2 in replication stress response and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3660ad79bb728b1c188f68e1b705105f627767e4" target='_blank'>
              Emerging roles of RNA:DNA hybrid regulation by mammalian ribonuclease H2 in replication stress and cancer
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, E. Petermann
          </td>
          <td>2025-12-01</td>
          <td>Journal of Cell Science</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Three-dimensional genome architecture is essential for gene regulation and cellular homeostasis. Its perturbation underlies pathologies. Advances in chromosome conformation capture technologies, such as Hi-C [1], have enabled researchers to study genome architecture at high resolution, yet the resulting matrices are difficult to explore visually, especially when one wishes to overlay genomic annotations or examine inter-chromosomal interactions. We introduce gghic, an R package that extends the ggplot2 grammar for the publication-quality visualization of chromatin-interaction data. Gghic supplies layers that draw (i) triangular Hi-C heatmaps, (ii) chromatin loops, topologically associating domains, (iii) gene or transcript models, (iv) one-dimensional signal tracks, (v) and multi-way contacts. The package supports Bioconductor classes, including HiCExperiment [2] and GInteractions [3]. It handles data from multiple chromosomes, enabling inspection of inter-chromosomal contacts. We illustrate the utility of gghic with acute lymphoblastic leukemia Hi-C datasets, producing publication-ready figures that reveal translocation breakpoints and higher-order chromatin hubs. By coupling advanced genome-structure analyses with the ggplot2 syntax, gghic makes the exploration and communication of 3D genomics data both intuitive and reproducible. References 1. Belton JM, McCord RP, Gibcus JH et al. ‘Hi-C: a comprehensive technique to capture the conformation of genomes’ Methods 2012;58:268–276. 2. Serizay J, Matthey-Doret C, Bignaud A et al. ‘Orchestrating chromosome conformation capture analysis with Bioconductor’ Nat Commun 2024;15:1072. 3. Lun AT, Perry M, Ing-Simmons E. ‘Infrastructure for genomic interactions: Bioconductor classes for Hi-C, ChIA-PET and related experiments’ F1000Res 2016;5:950.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbff5aed0fd8c28e8521d832d596353743e33b7" target='_blank'>
              Gghic: a versatile R package for exploring and visualizing 3D genome organization
              </a>
            </td>
          <td>
            Minghao Jiang, Duohui Jing, Jason W.H. Wong
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2857a0fc338d1571175ba6c4c210b5ca1dafdfd9" target='_blank'>
              Identification of Extrachromosomal Circular DNA Isolated from Cell Culture Supernatant and Its Potential Applications for Hepatocellular Carcinoma Diagnosis.
              </a>
            </td>
          <td>
            Suchitraporn Sukthaworn, Suchanuch Ondee, Opas Choksupmanee, Suparat Taengchaiyaphum, Prasert Yodsawat, Ponsit Sathapondecha
          </td>
          <td>2026-01-21</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Like many cancers, prostate cancer (PCa) relies on tissue- and lineage-specific transcription factors essential for normal tissue function and tumor progression. A key master factor is the androgen receptor (AR), which regulates prostate tissue identity and differentiation, maintaining normal, growth-suppressive, prostate-specific luminal programs via androgen response elements (AREs). However, during tumorigenesis, AR is co-opted to drive oncogenic transcription programs, including reprogramming factors like FOXA1 and HOXB13. Despite AR’s critical role, the mechanisms and functional implications of both its oncogenic and growth-suppressive programs in PCa remain incompletely understood. We hypothesized that the methyl-CpG reader and DNA repair protein MBD4 safeguards lineage trajectories by restraining pioneer factor engagement.



 We performed an epigenetic-focused CRISPR screen (2,508 genes) in an LNCaP ARE-activated model to nominate the regulators of AR programs. We engineered MBD4 knockout and inducible overexpression models and profiled chromatin and transcription by ATAC-seq; CUT&RUN (AR, FOXA1, H3K27ac, H3K4me3, H3K27me3); and RNA-seq across AR+ and AR- lines. Proliferation and lineage-identity programs were also evaluated in normal mouse prostate organoids. Clinical relevance was assessed using multi-cohort patient transcriptomes and cancer-dependency datasets.



 MBD4 emerged as a gatekeeper of the AR/ARE-mediated growth-suppressive program. MBD4 loss increased chromatin accessibility and enhancer acetylation at FOXA1-enriched loci, with FOXA1 binding expanding upon loss and decreasing with MBD4 overexpression. MBD4 localized to AR/FOXA1 enhancers and limited FOXA1 engagement. Functionally, MBD4 knockout accelerated proliferation in AR+ lines and slowed growth in AR- lines (e.g., PC3, DU145). In normal mouse prostate organoids, MBD4 disruption biased luminal epithelial identity programs and reprogrammed enhancer architecture. Across patient cohorts, MBD4 expression and dependency patterns tightly tracked with AR status/lineage, linking lineage context to chromatin plasticity.



 MBD4 acts as a gatekeeper of the AR/ARE-mediated growth-suppressive program by limiting lineage-specific enhancers and restraining FOXA1 engagement. Loss of MBD4 promotes FOXA1-driven reprogramming and lineage plasticity, nominating the MBD4–FOXA1 axis as a potential therapeutic target. AI DISCLOSURE: Generative AI was used to help draft the wording of this abstract; all content was supplied, reviewed, and approved by the authors.



 Xuanrong Chen, Janny Villa-Pulgarin, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, Andrea Sboner, Christopher Barbieri. MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/144dd6ef15d0789330238173b369f5abfb772a9e" target='_blank'>
              Abstract A010: MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression
              </a>
            </td>
          <td>
            Xuanrong Chen, J.A. Villa-Pulgarín, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, A. Sboner, C. Barbieri
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="DExD-box (DDX) proteins are essential for RNA metabolism and are targets for treatment of cancers and neurodevelopmental disorders. The biochemical mechanisms of many DDX proteins remain unclear, including human DDX52. DDX52 is essential for cell survival and is an emerging biomarker for the onset of metastatic melanoma. In this work, we identified that human DDX52 is an ATP-dependent translocase with 3'-5' polarity, which can unwind DNA duplexes and DNA/RNA hybrids. Further, DDX52 is a nucleic acid annealase, an activity that requires an N-terminal intrinsically disordered protein region. DDX52 becomes hyperactive at DNA annealing if DDX52 helicase activity is inactivated by mutagenesis. Using CRISPR-Cas9 genetic editing, we generated U2OS cell lines heterozygous for DDX52 (DDX52+/-), which exhibit growth defects and impaired cell migration, providing direct support for previous suggestions that DDX52 may promote cancer cell metastasis and C-myc regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f186703aec8952740844b8d3f7ea360a94084d2a" target='_blank'>
              The human DDX52 protein is a nucleic acid helicase and strand annealase that promotes cell migration.
              </a>
            </td>
          <td>
            Ashley J Parkes, Philipp J Springer, E. Bolt
          </td>
          <td>2026-01-01</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The eukaryotic DNA damage and replication stress checkpoint is an essential component of the DNA damage response and crucial for genome maintenance. In budding yeast, the apical kinase Mec1 (ATR ortholog), along with binding partner Ddc2 (ATRIP ortholog), senses persistent RPA-bound ssDNA in the cell. Mec1 is activated by interaction with a Mec1-activating protein. One such activator, Dpb11 (TopBP1 ortholog), is recruited to a 5’ ss-dsDNA junction via the 9-1-1 checkpoint clamp. Due to their differential DNA binding preferences, it remains to be determined how Mec1 encounters its activators on damaged DNA. Using real-time single-molecule imaging of checkpoint proteins binding to dsDNA containing a long ssDNA gap, we show that, even in the absence of 9-1-1, Dpb11 binds to ssDNA and localizes to ss-dsDNA junctions in an RPA-dependent manner. Importantly, we directly visualize that Dpb11 recruits Mec1-Ddc2 to ss-dsDNA junctions. Additionally, single-molecule force spectroscopy was used to demonstrate that Dpb11 can interact with multiple DNA sites simultaneously to form bridges both alone and in the presence of RPA, stabilizing ssDNA loops and reducing the end-to-end distance of gapped DNA. Taken together, these data support a model in which Dpb11 facilitates Mec1 colocalization with its activators both directly by recruiting Mec1 to gap junctions and indirectly by decreasing the effective gap length.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bedc09d70f0d2e014d3d04b22125a1341e1dbf0" target='_blank'>
              Molecular interplay between the DNA damage checkpoint kinase Mec1-Ddc2 and its activator Dpb11 on gapped DNA
              </a>
            </td>
          <td>
            Emily C Beckwitt, Gabriella N. L. Chua, Shixin Liu, Michael E. O’Donnell
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Meiotic recombination is initiated by DNA double-strand breaks (DSBs); factors that control DSB frequencies are important to produce viable progeny. In many organisms, the ATM (Tel1) protein kinase prevents excessive meiotic DSBs, especially nearby DSBs on the same chromatid. Normally, two close DSBs are less frequent than expected from independence, a feature called DSB interference, which is lost in tel1Δ mutants. In the fission yeast Schizosaccharomyces pombe, high-level DSB formation depends on linear elements, Hop1, and meiotic cohesin complexes; we show here that these complexes impart competition between nearby DSB sites. When these complexes are impaired, Tel1 substantially represses DSB formation, and in its absence, two close DSBs on the same chromatid occur frequently and manifest high negative interference. After mitotic DNA damage, the conserved Mre11–Rad50–Nbs1 (MRN) complex is required for DNA resection, and the Tel1 kinase activity is needed to complete DSB repair. We found that during meiosis mre11Δ and rad50Δ mutants, like tel1Δ mutants, lack DSB interference and display highly negative DSB interference in meiotic complex mutants. Thus, MRN at a DSB site appears critical for Tel1 function in meiosis and reveals a complex interplay of positive and negative factors controlling meiotic DSB formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee1a2d4929cc57ee0b0191527c1dc8fe91c21069" target='_blank'>
              Mutual, spatially limited control of meiotic DNA break formation by Mre11–Rad50–Nbs1 DNA repair complex and Tel1 (ATM) protein kinase
              </a>
            </td>
          <td>
            Randy W. Hyppa, Gerald R. Smith
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9106ce16a20b6a59754668f52e05e0f1c8f3c2c3" target='_blank'>
              SET1/MLL complexes control transcription independently of H3K4me3
              </a>
            </td>
          <td>
            Hya Au, AT Szczurek, I. de Krijger, N. Kjelstrup-Osorio, AL Hughes, A. Lastuvkova, M. Vermeulen, RJ Klose
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bromodomain and Extra-Terminal domain (BET) proteins are key epigenetic readers that recognize and bind acetylated lysine residues on histones, orchestrating transcriptional programs that drive oncogenic processes. BET proteins regulate the expression of oncogenes involved in proliferation, survival, and differentiation, thereby promoting tumor initiation, progression, and therapy resistance across a wide range of solid tumors. Recent findings implicate BET proteins in maintaining cancer stem cells (CSCs), a subpopulation of tumor cells characterized with self-renewal capacity, plasticity, and the ability to evade conventional therapies. In CSCs, BET proteins coordinate stemness-associated transcriptional networks, and drive tumor persistence, metastasis, and relapse following treatment. BET proteins also shape the tumor immune microenvironment by modulating the expression of key immune checkpoint molecules such as PD-L1, regulating cytokine production, and controlling antigen presentation, which collectively influence adaptive and innate immune responses. BET inhibition enhances T cell infiltration and activation while suppressing the immunosuppressive functions of tumor-associated macrophages. The dual role of BET proteins in controlling both stemness and immune regulation positions them as central regulators of tumor-intrinsic and immune-mediated mechanisms in cancer. This makes BET proteins attractive therapeutic targets, as their inhibition offers the potential to simultaneously suppress tumor growth and reprogram the immune microenvironment. Preclinical and early clinical studies demonstrate that combining BET inhibitors with chemotherapy, targeted therapies, or immune checkpoint blockade synergizes anti-tumor responses. Future research focused on understanding the context-specific functions of BET proteins, and optimizing combination strategies will be critical to fully harness their therapeutic potential in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d3925c41fe48c57ca19baf2a5a75d366b0fdfae" target='_blank'>
              Bromodomain and extra-terminal proteins in solid tumors: regulators of immune microenvironment and emerging therapeutic targets
              </a>
            </td>
          <td>
            Deeksha Sharma, Grace G. Bushnell, Alexander P. Kalman, Chloe M Hutchens, Monika L. Burness
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The maintenance of genome stability requires efficient leading strand synthesis by DNA Polymerase Epsilon (Polε). By performing CRISPR genetic screens in cells lacking the POLE4 subunit of Polε we define a genetic map of the factors required to support Polε function in the absence of its accessory subunits. A set of genes involved in iron metabolism emerge as required to sustain Iron Sulphur Cluster (ISC)-dependent Polε activity. We then dissect a synthetic lethal interaction between POLE3-POLE4 and the CHTF18-RFC2/5 complex. By combining cell biology, structural modelling and biochemistry, we define the existence of two tiers of regulation of Polε processivity: leading strand-specific loading of PCNA by CHTF18-RFC2/5 and “gripping” of newly synthesised dsDNA by POLE3-POLE4. The combined loss of these functions is incompatible with leading strand synthesis and viability. In summary, we describe the biochemical basis of human leading strand synthesis and the consequence of its dysfunction in genome stability. How DNA Polymerase Epsilon accomplishes continuous leading strand synthesis during DNA replication is not understood. Here, the authors describe a two tiers mechanism required to sustain Pol Epsilon processivity: CHTF18-dependent loading of PCNA at leading strand and dsDNA binding by its POLE3-POLE4 subunits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9c13b963d3208ae6cad837c1eaa9236a2286b16" target='_blank'>
              The genetic and biochemical basis of human leading strand synthesis
              </a>
            </td>
          <td>
            Alessandro Agnarelli, Lauryn Buckley-Benbow, Meryem Ozgencil, Melanie Lad, K. K. Ampah, Alex Kalinka, Ondrej Belan, Sarah Maslen, Mark Skehel, David Walter, Matthew Day, Roberto Bellelli
          </td>
          <td>2025-12-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background and Objectives: Cervical adenocarcinoma (CAC) is a histologically distinct subtype of cervical cancer with a rising incidence in many regions. While the roles of key driver mutations are known, a comprehensive understanding of its genomic landscape, particularly variations across different populations and tumor stages, remains incomplete. This study aims to characterize the somatic genomic landscape of CAC by identifying recurrent mutations, copy number alterations (CNAs), and patterns of co-occurrence, with a focus on variations across racial groups and between primary and metastatic tumors. Materials and Methods: We conducted a comprehensive genomic analysis of 102 tumor samples from 99 patients diagnosed with cervical adenocarcinoma using data from the American Association for Cancer Research (AACR) Project Genomics Evidence Neoplasia Information Exchange (GENIE) database. Results: The most frequently mutated genes were PIK3CA (25.5%), TP53 (21.6%), ARID1A (20.6%), and KRAS (16.7%). Significant amplification of ERBB2 was also observed (n = 3; 4.83%). Our analysis revealed notable genomic disparities across racial groups, with TP53 mutations being significantly more frequent in White patients compared to Asian and Black patients (p = 0.0236). Furthermore, we identified significant co-occurrence between mutations in KRAS and MSH2 (p = 0.011) as well as ATM and STK11 (p = 0.037). In comparing tumor types, mutations in BCL6 were found to be significantly enriched in metastatic samples. Conclusions: This study validates the primary drivers of cervical adenocarcinoma and reveals novel findings, including notable racial disparities in TP53 mutation frequency and unique patterns of co-occurring mutations. These findings highlight the genomic heterogeneity of the disease and suggest that ancestry and tumor evolution may influence its molecular pathogenesis, offering potential avenues for the development of targeted therapies and personalized biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3066f0da5ff1e91963618a1014e74836869aa1ba" target='_blank'>
              Comprehensive Genomic Characterization of 102 Cervical Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Gejla Toromani, Grace S. Saglimbeni, Bhanu Surabi Upadhyayula, Eugene Manu, Tyson J. Morris, Beau Hsia, A. Tauseef
          </td>
          <td>2026-01-07</td>
          <td>Medicina</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Long Interspersed Nuclear Elements-1 (LINE-1, L1) are transposable elements that make up roughly 17% of the human genome. These elements can copy and insert themselves into new genomic locations (Kazazian and Moran, N Engl J Med 377:361–370, 2017). Typically, LINE-1 is repressed in healthy tissues but may become activated in various human diseases. LINE-1 expression has been associated with aging (Simon, et al., Cell Metab 29:871–885.e5, 2019; De Cecco et al. Nature 566:73–78, 2019; Della Valle et al. Nat Rev Genet 26:1–12, 2025), neurodegenerative disorders (Roy et al., Acta Neuropathol 148:75, 2024;Frost and Dubnau, Annu Rev Neurosci 47:123–143, 2024; Ravel-Godreuil et al. FEBS Lett 595:2733–2755, 2021), cancer (Rodriguez-Martin et al., Nat Genet 52:306-319, 2020; Taylor et al. Cancer Discov 13:2532–2547, 2023; Solovyov et al. Nat Commun 16:2049, 2025), and autoimmune diseases (Rice et al., N Engl J Med 379:2275–2277, 2018), (Carter et al., Arthritis Rheumatol 72:89–99, 2020). Despite the strong association between LINE-1 expression and disease, the regulatory mechanisms controlling the expression of LINE-1-encoded ORF1p and ORF2p and the link between LINE-1 activity and cancer cell survival remain poorly understood. Gaining insights into these regulatory pathways may help elucidate how LINE-1 contributes to disease pathogenesis. To identify upstream regulators of LINE-1 and genes associated with LINE-1 activity-dependent lethality, we developed a dual-reporter system that simultaneously monitors the protein levels of LINE-1-encoded ORF1p and ORF2p (wild-type or catalytically inactive EN/RT mutant). Using genome-wide CRISPR/Cas9-based screens with this system, we identified candidate genes that may influence LINE-1 regulation at multiple levels, including RNA and protein expression. Alongside known factors such as the HUSH complex, the screens revealed additional genes not previously linked to LINE-1 regulation, suggesting possible new regulatory mechanisms for ORF1p and ORF2p expression. We also identified genes whose loss correlated with reduced viability in a manner dependent on LINE-1 activity. These findings collectively provide a broad resource for exploring cellular factors that may modulate LINE-1 expression and activity. This study provides a resource for investigating the cellular regulation of LINE-1, highlighting distinct candidate factors that may modulate ORF1p and ORF2p expression and influence LINE-1 activity-associated cytotoxicity. While functional validation of these candidate regulators remains necessary, the findings offer a foundation for future studies aimed at experimentally confirming their roles and elucidating the molecular mechanisms underlying LINE-1 regulation and its potential contributions to disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc75de6176e9aaa531a302da776789c267f8ce" target='_blank'>
              Identification of novel regulators of LINE-1 expression via CRISPR/Cas9 screening
              </a>
            </td>
          <td>
            Ozgur Oksuz, Chong Chu, Cedric Arisdakessian, Liyang Diao, Dennis M. Zaller, Kimberly K Long, Heike Keilhack, Sarah Knutson
          </td>
          <td>2025-12-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Retroviruses are exclusive group of positive-sense RNA viruses defined by their ability to reverse transcribe their RNA genome and integrate it into the host’s chromosomal DNA. This distinctive replication strategy enables persistent infection and has profoundly shaped our understanding of molecular biology, gene regulation, and evolution. Retroviruses have contributed to landmark discoveries, including the identification of oncogenes, mechanisms of transcriptional control, and the development of gene therapy vectors. This review provides an updated overview of retroviral molecular biology, emphasizing the coordinated steps of the viral life cycle and emerging insights that are reshaping classical models. It explores virion structure, genome organization, and the interplay of cis-acting sequences and trans-acting factors that govern replication. Special focus is given to recent advances in understanding nuclear trafficking of capsids, spatial dynamics of reverse transcription and integration leading to provirus formation, RNA nuclear export, and selective genome packaging. The structural and functional roles of viral proteins, particularly Gag, are discussed in the context of assembly and maturation. By integrating foundational concepts with new discoveries, this review highlights the molecular sophistication of retroviral replication and identifies outstanding questions that guide future research, with implications extending to antiviral strategies, gene therapy, cancer biology, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2158d0cf0754098c7826e6edf394a24a8aa054" target='_blank'>
              Beyond reverse transcription: molecular mechanisms and emerging paradigms in retroviral replication
              </a>
            </td>
          <td>
            Mohammad Abdullah Jehad, L. Ali, Vineeta N Pillai, Suresha G. Prabhu, Farah Mustafa, Tahir A. Rizvi
          </td>
          <td>2025-12-26</td>
          <td>FEMS Microbiology Reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell metabolism has a profound impact on maintaining genomic stability. AMP-activated protein kinase (AMPK) is a crucial regulator of cell metabolism and the maintenance of genomic stability. There is increasing evidence that AMPK plays a crucial role in the efficient response to DNA damage (DDR). However, the underlying mechanism is still unclear. Here, we show that glucose deprivation rapidly reduces γH2AX levels, a hallmark of DNA damage. We then found that WIP1, rather than PP2A or PP4C, is the primary phosphatase responsible for dephosphorylating γH2AX under both normal and damaged conditions. Molecular studies have revealed that AMPK directly binds and phosphorylates WIP1 at Thr25 (T25). This action enhances protein stability and the binding ability of WIP1 with γH2AX, likely promoting the enzyme activity of WIP1 and subsequently reducing the level of γH2AX. These processes facilitate DNA damage repair and contribute to the radioresistance of tumor cells. The findings provide experimental evidence of a novel link between metabolic stress and DDR, suggesting that AMPK may promote the resistance of tumor cells to radiation therapy by phosphorylating WIP1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ded97f76015b39710193c6311700750806454773" target='_blank'>
              AMPK phosphorylates WIP1 to promote DNA repair and radioresistance in cancer cells
              </a>
            </td>
          <td>
            Manman Lu, Xiaochuan Dong, Chunrui Wu, Guisong Wang, Haiyang Wang, Yingli Pan, Yali Qin, Yushuai Song, Hongming Pan, Shenzhi Liu, Kun Zhang, Xuewu Zhang, Jing Qu, Zhenhua Yang
          </td>
          <td>2025-11-28</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, the roles of which are under debate. We found that double-strand breaks (DSBs) induce de novo formation of chromatin loops in human cells, with the loop base positioned at the DSB site. These loops form in the S and G2 phases of the cell cycle during homologous recombination repair, concomitantly with DNA end resection and radiation-sensitive protein 51 (RAD51) recruitment. RAD51 shows a broad (megabase-sized) chromatin domain reflective of the homology search. This domain is regulated by cohesin unloader and overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of the loop-extruding cohesin subunit NIPBL lowers homologous recombination in mouse embryonic stem cells, and this effect is more pronounced when the homologous recombination donor is hundreds of kilobases from the DSB. Our data indicate that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e61307e4b5e3ffa40f9315b36ee4db40b3f94b6" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam T. Rybczynski, Namrata Nilavar, Daniel Nguyen, Andrew G. Li, Violetta Karwacki-Neisius, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>2</td>
          <td>39</td>
        </tr>

        <tr id="Colorectal liver metastasis (CRLM) occurs frequently in patients with colorectal cancer (CRC). Methionine adenosyltransferase (MAT) catalyzes the formation of S-adenosylmethionine, the principal methyl donor. MAT1A (encodes MATα1) is expressed mainly in normal adult liver, whereas MAT2A (encodes MATα2) is expressed in all extrahepatic tissues. MAT1A is a major defense against CRLM as loss of Mat1a sensitizes the liver to CRLM. In contrast, MAT2A is overexpressed in CRC and promotes oncogenicity. Here, we sought to determine if CRCs secrete MATα2 and if this influences CRLM. Our study included human hepatocytes, human CRC cells, extracellular vesicle (EV) isolation, chromatin immunoprecipitation (ChIP), ChIP-seq, promoter activity assays, proliferation, migration, and invasion assays, western blotting, immunohistochemistry and immunofluorescence. We confirmed some of the findings using human hepatocyte spheroids, CRLM and normal liver tissue array, and plasma samples. CRCs secrete MATα2 in free but truncated form (MATα2-t) and intact within EVs (EV-MATα2). EV-MATα2 can be internalized by human hepatocytes and CRCs, found within the nucleus, which then binds to MAT1A and MAT2A promoters on ChIP to lower and increase MAT1A and MAT2A promoter activities, respectively. In human CRLM samples, hepatocytes in nontumor regions express lower MATα1 but higher MATα2 as compared to normal liver. Treating RKO cells with EVs released from RKO cells overexpressing MAT2A promoted cell proliferation, migration, and invasion. MATα2-t was detected at a higher level in media from colon, pancreatic, and prostate cancer cell lines than corresponding normal epithelial cells as well as in the plasma of CRC patients as compared to healthy controls. RKO cells treated with MATα2-t activated focal adhesion kinase (FAK), an important kinase for cancer cell evasion of apoptosis. Conversely, treatment with MATα2 neutralizing antibody inhibited FAK and induced apoptosis. CRC cells secrete both MATα2 within EVs and free MATα2-t. EV-MATα2 can be internalized and act as a transcription factor to lower hepatocytes’ MAT1A, the major defense against CRLM, while promoting CRC oncogenicity. Freely released MATα2-t acts as a ligand in an autocrine fashion to activate FAK, which is essential for CRC survival. Taken together, secreted MATα2 plays an essential role in promoting CRLM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d717a5a8641241a8445c43509a6ec64d1a59947" target='_blank'>
              A novel role of secreted methionine adenosyltransferase α2 in colorectal liver metastases
              </a>
            </td>
          <td>
            Monica Justo, Youngyi Lim, Heping Yang, Andrea Floris, Swati Chandla, Manisha Dagar, Alexandra Gangi, E. Posadas, M. Edderkaoui, Stephen J. Pandol, Neil A. Bhowmick, M. Tomasi, Shelly C. Lu
          </td>
          <td>2025-12-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bfffe74b13f5a75dbf73aa9e042097d2141c09" target='_blank'>
              POLQ variants with aberrant DNA polymerase activity protect against UV-induced cell death
              </a>
            </td>
          <td>
            Steven E Weicksel, Corey Thomas, Ethan Hall, Kylie Davis, Chase Michalczik, Sreerupa Ray, Jamie B Towle-Weicksel
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Androgen receptor (AR) signaling is critical for the survival and proliferation of prostate cancer (PCa) cells, thus making androgen deprivation therapy (ADT; e.g., castration) the mainstay for treatment. Notably, the oncogenic transcriptional functions of AR rely on a host of chromatin-binding regulatory proteins, which includes a pioneer transcription factor called FOXA1. FOXA1 de-compacts chromatin to enable DNA binding of AR and activation of target gene expression. Our lab found FOXA1 alterations to recur within three distinct structural classes in over 35% of metastatic castration-resistant PCa (mCRPC) cases in Caucasian White men. Subsequent studies reported that FOXA1 mutations are prevalent in over 40% of primary PCa in Chinese patients, thus positioning FOXA1 as a principal oncogene in this disease. Yet, the tumorigenic potential and pathobiology of FOXA1 alterations remain unexplored in vivo. Here, we have developed and characterized the first in-field transgenic mouse models that conditionally overexpress FOXA1 mutants in the prostate luminal epithelia. Our findings reveal that truncal FOXA1 class1 mutations (i.e., wing 2 alterations) in a Trp53-null background triggered high-grade adenocarcinoma mimicking human disease, characterized by abnormal induction of NSD2 expression, extensive AR reprogramming to chimeric AR-half neo-enhancers, and sensitivity to androgen deprivation therapies. In contrast, FOXA1 class 2 mutations (i.e., C-terminal truncations) that are acquired in human mCRPC do not drive prostate luminal transformation in mice. Instead, these mutations induce intra-luminal plasticity by reprogramming differentiated cells into a progenitor stem-like state within androgen-replete normal tissues—a phenomenon otherwise induced upon castration. Class 2 mutants lead to a 20-fold expansion of Ar+/Ck8+ luminal epithelia that gain expression of stemness markers such as Trop2, Ck4, and Psca. These stem cells are similar to the Club cells detected in the human prostate gland that have been implicated in driving resistance to ADT. Mechanistically, we found the cistromically-dominant Class 2 mutants to pioneer over 40,000 neo-enhancer elements that were bound by stemness-associated transcription factors, like KLF5 and AP-1, which together instruct an androgen-insensitive luminal progenitor cell fate. Consistently, we found that Class 2-mutant mouse prostates showed minimal atrophy upon castration, with immunohistological assessment revealing a higher density of Ki67+ luminal epithelial cells relative to both wild-type and Class 1-mutant tissues. Class 2-mutant organoids also showed greater subcutaneous grafting ability in limiting-dilution assays in mice. Collectively, our data establish FOXA1 as a multifaceted oncogene in AR-dependent prostate cancers, in which divergent evolution of FOXA1 mutational classes distinctly drives either cancer formation or therapy-resistant, intra-luminal plasticity during disease progression. These findings also represent the first report of FOXA1-driven prostate adenocarcinoma in mice.



 Sanjana Eyunni, Rahul Mannan, Yuping Zhang, Eleanor Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James George, Mustapha Jaber, Hamzah Hakkani, Sandra E. Carson, Abigail J. Todd, Noshad Hosseini, Mahnoor Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, Arul M. Chinnaiyan, Abhijit Parolia. Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e838d76f934b647aed2d9fe44ea43b83b285eb1c" target='_blank'>
              Abstract PR012: Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>170</td>
        </tr>

        <tr id="ABSTRACT During cell division and gene expression, the DNA double‐helical structure unwinds, thereby generating torsional stress. DNA topoisomerases are enzymes that relieve this stress. During this process, topoisomerases form temporary covalent bonds with the phosphate backbone of DNA, generating DNA strand breaks and relieving torsional stress. Topoisomerases then dissociate from DNA after rejoining the DNA breaks. Torsional stress associated with replication or transcription is primarily relieved by topoisomerase I (TOP1) and II (TOP2). Some anticancer drugs targeting topoisomerases, known as topoisomerase poisons, trap the topoisomerase reaction intermediates and cause DNA strand breaks bearing topoisomerase–DNA–protein crosslinks (TOP–DPCs). TOP1 poisons, such as camptothecin, cause DNA single‐strand breaks bearing TOP1–DPCs, which are converted to DNA double‐strand breaks (DSBs) when they collide with DNA replication forks. In contrast, TOP2 poisons, such as etoposide, directly induce DSBs in TOP2–DPCs. However, to elicit a DSB response, TOP2–DPC must first be removed from the DSB ends. Cells possess various pathways to remove TOP2–DPC, and these pathways are thought to function in coordination depending on the situation. This review summarizes these sophisticated TOP2–DPC removal pathways and discusses the clinical applications of TOP2 poison as an anticancer drug, as well as the related challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1a3ae05a710b9f77cc942446aa4e72d459275cd" target='_blank'>
              Processing of DNA Topoisomerase II–DNA–Protein Crosslinks Associated With Anticancer Drugs
              </a>
            </td>
          <td>
            Ryo Sakasai, Kuniyoshi Iwabuchi
          </td>
          <td>2025-12-17</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4efb8218ff34b8674a51cd71fbecd16898befa89" target='_blank'>
              Developing Topics.
              </a>
            </td>
          <td>
            Morgan Elizabeth Lambert, Paulino Ramirez, Wenyan Sun, Bess Frost
          </td>
          <td>2025-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872145069c4ef81fa1ace0d713f50e945a6721cc" target='_blank'>
              The multiple roles of the disordered protein CCDC6 in cancer development.
              </a>
            </td>
          <td>
            Guifeng Wei, Yiji Chen, Yichao Kong, Donglai Li, Yang Wang, Ting Qiu, Xiabin Chen
          </td>
          <td>2026-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Lysine specific demethylase 1 (LSD1), encoded by the gene KDM1A, is overexpressed and correlates with poor patient prognosis in Ewing sarcoma. LSD1 and the pathognomonic fusion oncoprotein, EWSR1::FLI1, colocalize throughout the genome, suggesting LSD1 is a critical co-regulator driving the progression of Ewing sarcoma. However, therapeutic targeting of LSD1 by competitive and noncompetitive inhibitors has yielded mixed results. Irreversible, enzymatic inhibition seems ineffective, but reversible noncompetitive inhibition has predominant off target mechanisms, leaving open the question of LSD1 function in Ewing sarcoma. Here we take a robust approach through multiple methods of depletion in multiple EwS cell lines to define enzymatic and nonenzymatic contributions of LSD1 to transcriptional regulation. We define a core set of 22 genes that are commonly repressed by LSD1 in all cell lines, and that repression of these genes downregulates synapse functioning and e-cadherin target genes. Derepression of these genes with LSD1 loss is an early and sustained genotype in all cell lines tested. We further define distinct gene sets in each cell line that are regulated by enzymatic and nonenzymatic LSD1 activity and find repression of e-cadherin target genes to be nonenzymatically regulated. This finding supports the growing body of evidence that in addition to their canonical catalytic activity, chromatin regulatory enzymes serve essential noncanonical roles as well. Furthermore, we uncovered evidence through use of the irreversible inhibitor OG-L002 that 2D cytotoxicity and proliferation assays may be insufficient to determine Ewing sarcoma response to LSD1 inhibition. SIGNIFICANCE Here we address a long-standing question in the field surrounding LSD1 and define the distinct enzymatic and nonenzymatic functions of LSD1 in Ewing sarcoma. In doing so, we have created a robust data set using genetic and pharmacological techniques in multiple models to thoroughly characterize LSD1 function in Ewing sarcoma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa31c55d8000608b4f5b4507892563a7a74a977c" target='_blank'>
              LSD1 Performs Demethylase-Independent and Context-Specific Roles in Ewing Sarcoma
              </a>
            </td>
          <td>
            Rachel D. Dreher, C. Taslim, Ira Miller, John W. Sherman, Ariunaa Bayanjargal, Emily R. Theisen
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In cells, highly coordinated multivalent interactions give rise to discrete functional assemblies-commonly referred to as biomolecular condensates-that compartmentalize the molecular components required for specific biological reactions. These condensates are increasingly recognized as organizational entities with central roles in normal cellular regulation and in the pathogenesis of human cancers. In a recent issue of Cell, Datar and colleagues investigated the condensation of NPM1c, a common gene mutation in acute myeloid leukemias (AMLs). They demonstrated the necessity and sufficiency of NPM1c in forming nuclear condensates termed coordinating bodies (C-bodies), which show co-partitioning of a suite of transcriptional coactivators such as NUP98, KMT2A/MLL1, Menin and XPO1/CRM1. While C-bodies are necessary for driving NPM1c-mutant AMLs, blockade of the co-partitioned component within C-bodies, such as the XPO1/CRM1 or Menin interaction by inhibitors, significantly alters the condensate composition and functionality. Likewise, a systematic deletion study of various regions within NPM1c pointed to a role for the coordinated multivalent interaction in establishing the functional condensates, as previously reported in studies of the Wilms tumor-causing ENL mutants and AML-causing NUP98 onco-fusions. Co-mixing of C-bodies and condensates formed by the onco-fusion of NUP98 or KMT2A/MLL1 in cells suggested them to be biophysically indistinguishable, indicative of a shared pathogenic mechanism. Altogether, recent studies of multiple genetic drivers in human cancers have revealed a type of chromatin-bound multi-component onco-condensates, which shall motivate the development of onco-condensate disruptors that could potentially be used as the broad treatments for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cc999451ae4cdf83273f4d0ad6380f1649aa9ab" target='_blank'>
              A unified model: chromatin-bound multi-component onco-condensates.
              </a>
            </td>
          <td>
            Benjamin K Lau, Elena Haarer, Jeong Hyun Ahn, G. Wang
          </td>
          <td>2026-01-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="


 Despite advancements in treatment, 5-year survival for medulloblastoma (MB) remains 60-70%. Human cytomegalovirus(HCMV) has been implicated as an oncomodulator in MB, but its impact on survival has not been explored. This study evaluates HCMV expression in pediatric MB tissue and its association with molecular risk groups and survival.



 A retrospective study of pediatric MB cases (≤19 years) from 2007-2023 was conducted using the WHO 2021 classification, the Northcott brain-tumor classifier, and the SIOP-Europe/ERN PaedCan risk stratification. HCMV immediate-early (HCMV-IE) and late-antigen (HCMV-LA) expression were assessed by optimized immunohistochemistry, and whole-genome sequencing (WGS) data from 20 tumors were analyzed for viral gene expression.



 Forty-five patients (mean age 8.2 years; 22% ≤3 years) were included: WNT (18%), SHH TP53-wildtype (18%), and non-WNT/non-SHH (64%). 20% of MB belonged to low-risk, 33% to standard-risk, and 47% to high-risk (HR-MB) groups. Four tumors exhibited MYC/N amplification. HCMV-LA positivity was found in 84% of cases, with 53% showing high expression (≥25% of cells). Cox-regression identified HR-MB (HR = 4.197, p = 0.021) and high HCMV-LA (HR = 4.334, p = 0.027) as independent predictors of poor outcomes. WGS revealed UL88 as the most abundantly expressed HCMV gene(log2[TPM+1] ≈14–15), with maximal expression in Group 3 MB.



 This study provides the first evidence linking high HCMV-LA expression to adverse outcomes and high-risk molecular features in pediatric MB. UL88 emerged as the most strongly expressed HCMV gene across MB samples. These findings suggest a potential prognostic and therapeutic role for HCMV in MB, warranting validation in larger, prospective cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60feabdb3f3fd823a43f478c57c8aa48c2982d72" target='_blank'>
              High Human Cytomegalovirus antigen Expression in pediatric medulloblastoma tissue is associated with poor event-free survival
              </a>
            </td>
          <td>
            Maria F De la Cerda-Vargas, J. Schittenhelm, Martin Ebinger, J. Zipfel, R. Beschorner, Ghazaleh Tabatabai, Martin U Schuhmann
          </td>
          <td>2025-12-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer is among the most prevalent cancer types and advanced castration-resistant prostate cancer (CRPC) remains a lethal disease. It is driven by oncogenic transcription factors, most notably the androgen receptor (AR) which represents to this day the main therapeutic target. That said, more transcription factors have emerged supporting tumor growth in late stages when prostate cancer cells lose AR expression through lineage plasticity. However, targeting this plethora of transcriptional drivers throughout disease progression is a daunting task. Yet, disrupting common enablers upstream of oncogenic transcription factors may provide an opportunity to shut down oncogenic output of prostate cancer cells regardless of their CRPC subtype. Indeed, the pioneer transcription factor FOXA1 is a strong and highly selective Achilles' heel in prostate cancer (www.depmap.org), critical for opening chromatin and enabling AR to exert its oncogenic function. Moreover, its homologue FOXA2 has been shown to be upregulated in AR-negative lineage plasticity prostate cancers as well. That said, effective FOXA1/2 inhibitors are currently not available. Here, we show that the FOXA1 homologue FOXA2 cooperates with the AR-pioneer factor FOXA1 in mediating AR-independent cell proliferation in various prostate cancer subtypes. In AR-negative cells, this colocalization is reinforced and supports the gene transcription of lineage-specific oncogenic transcription factors. Joint loss-of-function of both FOXA1 and FOXA2 leads to the collapse of lineage-specific oncogenic transcription factors. Importantly, using virtual drug screening we identify a non-covalent small molecule capable of directly binding and disrupting the interaction of the conserved forkhead domains with DNA. The inhibitors target both FOXA1 wild-type tumors and tumor characterized by recurrent driver mutations. Mechanistically, the inhibitors bind to a hydrophobic cavity within the N-terminus of the conserved forkhead domains of FOXA1 and FOXA2. Treatment with T and derivatives inhibits specifically the oncogenic transcriptional output programs related to different AR-positive and AR-negative prostate cancer subtypes and effectively blocks tumor growth in culture and after intraperitoneal injection in xenograft models in vivo. In summary, our findings uncover an unexpected collaboration of FOXA1 and FOXA2 in disease progression and a druggable key dependency for AR-positive and -negative CRPC regardless of their specific lineage-specific oncogenic program.



 Nicolo Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat. Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR020.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af0e2296f18efc47feaa25f0dfc62945878e4ac" target='_blank'>
              Abstract PR020: Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer
              </a>
            </td>
          <td>
            Nicolò Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3ef8e203c378a8d1a7856db6aaa8c3b98aa293" target='_blank'>
              Dissecting the contribution of transposable elements to interphase chromosome structure.
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, Xuemeng Zhou, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhuoqi Huang, Li Sun, Jiangping He, Alexander V. Strunnikov, A. P. Hutchins
          </td>
          <td>2026-01-20</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA single-strand break (SSB) repair defects lead to hereditary neurological syndromes. Spinocerebellar ataxia with axonal neuropathy type 1 (SCAN1), is caused by the homozygous H493R mutation in tyrosyl-DNA phosphodiesterase 1 (TDP1), an enzyme that initiates the repair of DNA topoisomerase 1 (TOP1)-induced SSBs by unlinking the TOP1 peptide from the break. Although TDP1 also initiates the repair of TOP1-induced DNA double-strand breaks (DSBs) associated with transcription, the role of TOP1-induced DSBs in SCAN1 pathology remains unclear. Here, we have addressed the impact of the SCAN1/H493R mutation on the repair of TOP1-induced DSBs. We demonstrate that while TDP1 loss delays the repair of these breaks, SCAN1/H493R completely blocks it in RPE-1 quiescent cells. This blockage is specific to DSBs and is accompanied by a prolonged trapping of mutated TDP1 on DNA, but not of TOP1 cleavage complexes (TOP1cc). Intriguingly, the H263A inactivating mutation of TDP1, which accumulates TOP1cc, also blocks TOP1-induced DSB repair. Importantly, both SCAN1/H493R and H263A mutations exhibit genome instability and cell death. Moreover, we demonstrate that tyrosyl-DNA phosphodiesterase 2 (TDP2) can compensate for TDP1 loss in RPE-1 quiescent cells. Collectively, our data support the potential role of TOP1-induced DSBs as a main contributor to certain hereditary neurological syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebb4417da0d9de2b620ef12c2f66c797e441d074" target='_blank'>
              H263A and SCAN1/H493R mutant TDP1 block TOP1-induced double-strand break repair during gene transcription in quiescent cells and promote cell death
              </a>
            </td>
          <td>
            Diana Rubio-Contreras, Daniel Hidalgo-García, Carmen Angulo-Jiménez, Esperanza Granado-Calle, Margarita Sabio-Bonilla, Jose F. Ruiz, Fernando Gómez-Herreros
          </td>
          <td>2025-11-28</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease and often comprise molecularly distinct subtypes that differ in prognosis and therapeutic response. Super-Enhancers (SEs) are large enhancer clusters that robustly drive expression of genes controlling developmental processes and cell identity. In this study, we developed an epigenetic-based framework to determine whether mCRPC harbors subtype-specific SE programs that may reactivate distinct developmental transcriptional programs and define molecularly subtypes of disease progression.



 We developed a computational workflow, Super-Enhancer Analysis for Lineages (SEAL), to map SE landscapes, classify tumor subtypes and identify subtype-specific SE-driven genes. Using ROSE analysis on H3K27ac ChIP-seq data from the LuCaP and MURAL PDX mCRPC series, we constructed an integrated SE atlas. Molecular subtypes were defined through consensus clustering of SE regions and further characterized by genomic, transcriptional, and clinical features. We integrated matched RNA sequencing data to identify top-ranked SE-driven transcription factors, which were further evaluated via loss- and gain-of-function studies as well as ChIP-seq and RNA-seq analyses to define their cistromes, transcriptomes, and interacting networks.



 We identified five distinct SE programs in mCRPC: three AR-positive subtypes (AR-1, AR-2, AR-3) and two AR-independent subtypes (NEPC-like and DNPC-like). Among the three AR-driven subtypes, AR-1 and AR-2 displayed aggressive tumor features, enriched for cell cycle, EMT, and hypoxia pathways. Each subtype exhibited a distinct SE-driven transcriptional program. Notably, we identified TWIST1, HNF1A, and TBX10 as key SE-driven transcription factors in AR-1, AR-2, and AR-3 subtypes, respectively. Among these, HNF1A, a critical transcription factor involved in hepatic development and metabolic regulation, was exclusively expressed in the AR-2 subtype. HNF1A silencing significantly reduced proliferation in vitro and in vivo and markedly suppressed glycolytic activity. AR-2 tumors produced high levels of secreted albumin, a well-established HNF1A hepatic target, which was decreased upon HNF1A silencing. ChIP-seq and RNA-seq analyses revealed that HOXB13 co-occupied HNF1A-mediated enhancers and cooperatively regulated the hepatic transcriptional programs. Conversely, HNF1A overexpression in AR-3 cells induced the expression of hepatic markers, enhanced proliferation and migration, and recapitulated features of the AR-2 subtype, indicating that HNF1A acts as a driver of hepatic lineage reprogramming.



 Our study reveals a novel HOXB13-HNF1A transcriptional axis that governs a previously undefined hepatic reprogramming in a subset of AR-driven mCRPC tumors. These findings establish a SE-based molecular classification of mCRPC and highlight HNF1A as a key regulator of metabolic and lineage programs, with potential implications for precision therapy and biomarker development in advanced prostate cancer.



 Mingyu Liu, Songqi Zhang, Nolan D. Patten, Jared G. Lourie, Xiaolin Zi, Kai Zou, Shuai Gao, Kourosh Zarringhalam, Changmeng Cai. Super-enhancer landscape analysis reveals a HOXB13-HNF1A transcriptional axis driving hepatic reprogramming in castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c8c15e0835123218bb04bb3827f4c746859379" target='_blank'>
              Abstract PR015: Super-enhancer landscape analysis reveals a HOXB13-HNF1A transcriptional axis driving hepatic reprogramming in castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Mingyu Liu, Songqi Zhang, Nolan D. Patten, Jared Lourie, Xiaolin Zi, Kai Zou, Shuai Gao, Kourosh Zarringhalam, Changmeng Cai
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Prostate cancer lineage plasticity, characterized by histologic transformation of prostate adenocarcinoma (PRAD) to neuroendocrine prostate cancer (NEPC), is an emerging mechanism of treatment resistance. NEPC accounts for up to 15% of treatment-resistant prostate cancers and is associated with poor prognosis, highlighting an unmet need for new therapies. NEPC lineage reprogramming is primarily driven by epigenetic dysregulation including differential activity of histone and DNA methyltransferases. EZH2, the catalytic component of the Polycomb repressive complex 2 (Polycomb), is overexpressed in most treatment-resistant prostate cancers and is implicated as a driver of disease progression. In this study, we define the differential, lineage-specific action of Polycomb in both PRAD and NEPC subtypes to better understand its role in modulating differentiation and lineage plasticity, and to identify novel targetable drivers of NEPC. Epigenetic H3K27me3 CUT&Tag profiling of treatment-resistant PRAD (n=9) and NEPC (n=9) rapid autopsy clinical samples revealed that Polycomb targets, including NE-lineage transcription factors, are de-repressed in NEPC. Mechanistically, Polycomb modulates H3K4/K27me3 bivalent promoters, leading to the upregulation of NEPC-associated transcriptional drivers (e.g., ASCL1) and neuronal gene programs which facilitate forward differentiation following EZH2 targeting in NEPC patient-derived organoid/xenograft models. Notably, we identified prospero-homeobox 1 (PROX1) as an understudied cell-fate determining transcription factor that is epigenetically de-repression by differential Polycomb activity. Integrative H3K27ac CUT&RUN and Capture Hi-C analyses revealed potential enhancers of PROX1 in NEPC that may be regulated by ASCL1. We sought to functionally characterize the role of PROX1 in NEPC. An unbiased CRISPR screen in two NEPC patient-derived organoid models demonstrated high cellular dependency for PROX1; knockout of PROX1 impeded tumor growth in NEPC models, while overexpression of PROX1 in PRAD promoted tumor growth and spontaneous metastases. Transcriptomic and cistromic analyses across models of CRPC and NEPC pointed to PROX1 regulation of neuroendocrine-lineage transcriptional programs. Immunoprecipitation followed by mass spectrometry identified three novel phosphorylated sites in the DNA-binding domain of PROX1 that are critical for its stability and function. In silico analyses of these phosphorylation sites predicted CHEK1 as a potential upstream kinase which could be exploited for therapeutic targeting of PROX1. Our findings provide insights into the potential role for bivalent promoters in Polycomb-mediated lineage reprogramming, which may facilitate forward differentiation in NEPC upon EZH2 inhibition. Further investigation of de-repressed candidates defines the role of PROX1 as a driver of NEPC and a potential therapeutic target.



 Varadha Balaji Venkadakrishnan, Nathaniel C. E. Voss, Nicole Traphagen, Richa Singh, James Neiswender, Keira Prenza. Sosa, Kenny Weng, Francisca Vazquez, David S. Rickman, Myles Brown, Himisha Beltran. Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16b90ea02984495a11ed149a37089c1519a17fc" target='_blank'>
              Abstract PR022: Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity
              </a>
            </td>
          <td>
            V. B. Venkadakrishnan, Nathaniel C E Voss, Nicole A. Traphagen, Richa Singh, James Neiswender, Keira Prenza Sosa, Kenny Weng, Francisca Vazquez, D. Rickman, Myles Brown, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, Hyeonjin Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, Daniel Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Intestinal metaplasia (IM) is a premalignant condition associated with increased risk of gastric cancer-a deadly malignancy with varying geographic incidence. High-depth targeted sequencing of more than 1,500 IM samples from six countries identified 47 significantly mutated genes, including driver genes associated with high-risk populations and worse prognosis (ARID1A), KRAS/MAPK signaling (KRAS, BRAF, MAP2K1, MAP3K1, and MAP2K4), and altered mucosal immunity (PIGR). IM whole-genome sequencing and DNA methylation analysis revealed SBS17 as a specific mutational signature separating IMs from normal gastric tissues, associated with late DNA replication, genomic hypomethylation, and tobacco exposure. Beyond epithelial-derived somatic mutations, we observed elevated clonal hematopoiesis (CH) in patients with IM associated with age, smoking, and enhanced risk of progressing to gastric cancer. Patients with CH expansions exhibited co-occurring IM PIGR truncating mutations and greater colonization of the IM microenvironment by orally derived bacteria, suggesting that CH may promote IM progression by modulating host-microbe mucosal immunity.


SIGNIFICANCE
This international study identifies recurrent IM driver genes, IM-specific mutational signatures, and alterations in IM-associated immune landscapes and microbiomes. Our results highlight a role for nonepithelial somatic alterations (CH) in IM progression to gastric cancer, offering new translational opportunities for early cancer detection and interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f6a437b6939f544182c7d61adabe509b3b8f5bd" target='_blank'>
              Mutational Signatures and Clonal Hematopoiesis in Intestinal Metaplasia across Countries with Varying Stomach Cancer Incidence.
              </a>
            </td>
          <td>
            Kiekyong Huang, T. Hagihara, Benedict Shi Xiang Lian, Zhi Xuan Ong, Shen Kiat Lim, R. Chong, Supriya Srivastava, Jason Xing Kang, May Yin Lee, A. Tan, M. Lee, Shamaine Wei Ting Ho, Siti Aishah Binte Abdul Ghani, Clara Shi Ya Ng, Ruanyi Liang, Lin Liu, S. Tay, Xuewen Ong, Feng Zhu, Hui Chen, Zhen Li, T. Ang, Takuji Gotoda, Robert J. Huang, Christopher J L Khor, Hyun-Soo Kim, L. H. Lau, Yi-Chia Lee, Ayaka Takasu, Ming Teh, M. Thian, Wai Leong Tam, Xin Lu, S. Wong, J. B. So, H. Chung, J. Lee, K. Yeoh, Patrick Tan
          </td>
          <td>2026-01-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="DNA replication is often challenged by endogenous and exogenous sources of DNA damage, which can stall replication forks and result in single‐stranded DNA (ssDNA) gaps, double‐strand breaks (DSBs), and genomic instability. Detecting DNA damage specifically in newly synthesized DNA strands is essential for understanding how eukaryotic cells respond to replication stress and continue the cell cycle progression through DNA repair or DNA damage tolerance (DDT) mechanisms. Here, we present an optimized and accessible protocol for the alkaline BrdU comet assay—a single‐cell technique that combines bromodeoxyuridine (BrdU; a thymidine analog) pulse‐labeling of newly synthesized DNA with the alkaline comet assay (single‐cell gel electrophoresis) followed by fluorescence immunodetection. This method enables the specific detection and measurement of DNA strand breaks occurring in newly replicated DNA during and immediately after the S phase, even without cell synchronization, allowing researchers to differentiate replication‐associated DNA damage from overall genomic damage. We provide detailed instructions for performing the assay using human cells (RPE‐1 h‐TERT TP53 KO) in vitro after exposure to DNA replication‐stress‐inducing agents, such as hydroxyurea (HU) and ultraviolet‐C (UV‐C) radiation. We also demonstrate its application in translesion‐synthesis‐deficient (Pol eta‐deficient) human fibroblasts in vitro. Importantly, this protocol supports time‐course chase experiments (e.g., 0, 1, 2, and 4 hr post‐treatment) to monitor the kinetics of DNA damage in nascent DNA strands. This BrdU‐based protocol offers high specificity, single‐cell resolution, and cost‐effectiveness, making it particularly valuable for laboratories studying replication stress, post‐replication DNA repair proficiency, DDT, and/or genotoxic responses in unsynchronized human cells in vitro. This protocol also adheres to the Minimum Information for Reporting Comet Assay (MIRCA) guidelines and is aligned with the objectives of the International Comet Assay Working Group (ICAW), ensuring high reproducibility and standardization. © 2025 The Author(s). Current Protocols published by Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505cb9b8ddb6fb8797b9c24b040c2169c26f48a1" target='_blank'>
              Alkaline Bromodeoxyuridine (BrdU) Comet Assay to Detect Replication‐Associated DNA Damage
              </a>
            </td>
          <td>
            Diego Luis Ribeiro, J. Londero, D. J. Martins, C. Menck
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a456f6d05532472cc323e2f12ad32a49047c8ee2" target='_blank'>
              Biallelic NF1 Inactivation and Widespread Loss of Heterozygosity in a Subset of Unclassified Fibromyxoid Mesenchymal Neoplasms.
              </a>
            </td>
          <td>
            Carla Saoud, Narasimhan P. Agaram, S. Singer, L. Villafania, Jamal Benhamida, C. Antonescu
          </td>
          <td>2026-01-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>138</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability.
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Prostate cancer (PC) is one of the most prevalent malignancies in men, with rising incidence and mortality rates globally. Despite advances in therapeutic options such as androgen deprivation therapy and chemotherapy, effective cures, especially for advanced stages of the disease, remain limited. Recent research has highlighted the significant roles of alternative splicing (AS) and noncoding RNAs in tumor progression and drug resistance. This study aims to investigate the role of circIMP3, derived from the IMP3 gene, in prostate cancer development. Methods In this study, we employed quantitative PCR, RNA sequencing, and immunoblotting to identify and characterize circIMP3 in prostate cancer tissues and patient blood samples. Functional assays, including cell proliferation and in vivo tumorigenicity assays, were conducted to assess the biological role of circIMP3 in PC cells. RNA immunoprecipitation sequencing (RIP-seq) was used to identify alternative splicing events regulated by circIMP3. Additionally, exosome isolation and uptake assays were performed to explore the paracrine signaling function of circIMP3 within the tumor microenvironment (TME). Results We identified circIMP3, which is significantly upregulated in both prostate cancer tissues and peripheral blood of patients. CircIMP3 contains an internal ribosome entry site (IRES) and encodes a previously uncharacterized 288-amino-acid protein, circIMP3_288aa. Functional assays revealed that circIMP3_288aa promotes cell proliferation in vitro and accelerates tumor growth in vivo. Mechanistically, circIMP3_ 288aa regulates the alternative splicing of FBXW7, leading to impaired c-Myc ubiquitination and stabilization, which enhances oncogenic signaling. RIP-seq analysis identified over 2,000 alternative splicing events regulated by IMP3, with a notable enrichment in pathways related to ubiquitin-mediated proteolysis. Furthermore, circIMP3 is secreted into the TME via exosomes, where it is taken up by recipient cells, contributing to their proliferation. Discussion Our findings demonstrate that circIMP3 acts as a key regulator of both intracellular alternative splicing and extracellular paracrine signaling within the TME. The ability of circIMP3 to influence FBXW7 splicing and stabilize c-Myc provides a mechanistic basis for its role in promoting oncogenesis in prostate cancer. Clinically, high expression levels of circIMP3 correlate with poorer event-free survival in prostate cancer patients, suggesting its potential as a prognostic biomarker. Additionally, the detection of circIMP3 in peripheral blood positions it as a promising target for liquid biopsy applications in PC diagnosis and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44898723fc23dea0100abadafda89694a83449" target='_blank'>
              A novel protein encoded by circIMP3 promotes prostate cancer progression by regulating alternative splicing and tumor microenvironment
              </a>
            </td>
          <td>
            Wenren Zuo, Weizhou Huang, Haojie Chen, Yan Xu, Yang Zhang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42a1f257a6f0a0fa70f42113e6cb33278380e36e" target='_blank'>
              A novel super-enhancer-driven lncRNA LINC00973 governs head and neck squamous cell carcinoma progression through EN2.
              </a>
            </td>
          <td>
            An Wang, Pengfei Diao, Na Xiao, Yuxiang Wei, Yaping Wu, Yanling Wang, Xuejing Wang, Enshi Yan, Hongbing Jiang, Jin Li, Jie Cheng
          </td>
          <td>2025-12-19</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Sorex araneus, the Eurasian common shrew, has seasonal brain size plasticity (Dehnel's phenomenon) and abundant intraspecific chromosomal rearrangements. Genomic contributions to these traits, however, remain unknown. We couple a chromosome-scale genome assembly with seasonal brain transcriptomes to discover relationships between molecular changes and both traits. While positively selected genes enriched the Fanconi anemia DNA repair pathway (FANCI, FAAP100), which is likely involved in chromosomal rearrangements by preventing the accumulation of chromosomal aberrations, genes under positive selection or showing seasonal differential expression in the brain implicate neurogenesis (PCDHA6, SOX9, Notch signaling) and metabolic regulation (VEGFA, SPHK2) as key mechanisms underlying Dehnel's phenomenon. We also find that both positively selected and differentially expressed genes in the hippocampus are overrepresented near Sor. araneus evolutionary breakpoints. This relates both positive selection and differential expression to accessible chromatin configuration, suggesting that chromosomal rearrangements are integral to adaptive evolution and the regulation of brain size plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e5d59bc3d794d733f5961db0a47844813d84f1" target='_blank'>
              Genomic comparisons shed light on the adaptive basis of brain size plasticity and chromosomal instability in the Eurasian common shrew.
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Cecilia Baldoni, Laia Marín-Gual, Diana D Moreno Santillán, Marta Farré, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Ying Sims, Alan Tracey, Jonathan M. D. Wood, David A. Ray, Dominik von Elverfeldt, J. Nieland, Angelique P. Corthals, A. Ruiz-Herrera, D. Dechmann, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2026-01-09</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Regulation of gene expression is central to the development of immune cells and their ability to respond to infection. As part of a clinical evaluation, we identified two sisters with recurrent infections, hypogammaglobulinemia, and memory B cell deficiency, diagnosed as common variable immunodeficiency. Whole exome sequencing identified a heterozygous variant (Leu50Ser, L50S) in a conserved region of EZH2, the catalytic subunit of the epigenetic gene repressor Polycomb Repressive Complex 2 (PRC2). EZH2-catalyzed histone H3 lysine 27 methylation (H3K27me) in bulk was not overall significantly disrupted by this variant, in patient samples or cell lines expressing EZH2-L50S. EZH2-L50S protein is expressed similar to wild-type and can form PRC2. However, we find that specific genomic regions that normally have high wild-type levels of H3K27me3 are deficient in the L50S context, particularly around gene promoters. EZH2-L50S is still recruited to these sites, but is not as active. Using recombinant purified PRC2, we determine that L50S affects methylation of nucleosomes and disrupts allosteric stimulation that normally amplifies H3K27me3, consistent with the location of L50 in the allosteric regulatory region of PRC2. Thus, variation of EZH2 L50, occurring at low frequency in the population may interfere with normal B cell gene expression patterns, contributing to immunodeficiency. This study has implications for genetic variation in PRC2 in the general population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a59bd0112470a94559bebc44def6044177da01" target='_blank'>
              Characterization of an immunodeficiency-associated EZH2 variant
              </a>
            </td>
          <td>
            Francisco Perez de los Santos, Lily Beck, Emily DeCurtis, April M. Griffin, Maggie M. Balas, Charlotte Marchioni, Attila Kumánovics, Matthew R. G. Taylor, Kimberly R. Jordan, Charles H. Kirkpatrick, Aaron M. Johnson
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer encompasses a wide array of pathology stemming from complex genetic and epigenetic changes, with N6-methyladenosine (m6A) RNA methylation being the most identified type of modification involved in the initiation or progression causing cancer. This dissertation will expand the m6A epitranscriptome research of 12 cancers and their matched noncancer samples into depth with the integration of a combined total of 167 profiling datasets. In assessing any mechanisms involving m6A, the method we suggest conducting our analysis combines protozoological and generalist methods. Here, we will also develop 12 different deep-learning models to assist with identifying the cancerous m6A signatures. Additionally, in any of our models, and attempts to statistically differentiate tumor mass from healthy samples, we also include RNA sequencing type features and genomic labelling data to help optimize accuracy. Following our example, we also used the newly discovered features to investigate type-specific molecular subtype patterns across different cancers by developing a pan-cancer m6A model, which led to findings of an evolutionary conserved epitranscriptomic profile associated with damaging disruption of RNA hybridization and splice regulatory mechanisms. Together the cancer type specific and cross cancer research of the present study provides a new complementary perspective on the role m6A modifications have in cancer pathogenesis. The results present a new understanding of the molecular mechanisms of cancer progression which aids both the development of new treatments and potentially other cancer biomarkers based on the conserved features identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb628bff5ea50e491d6da61ea9720c855c0429e" target='_blank'>
              DCN Analysis Identifies Co-occurring Epitranscriptomic Alterations Across Dozens of Cancer Subtypes
              </a>
            </td>
          <td>
            N. N. Jose, C. Anuradha, Namburi Devika, Sankaran D, K. Joshi, N. Prakash
          </td>
          <td>2025-11-28</td>
          <td>2025 IEEE 7th International Conference on Computing, Communication and Automation (ICCCA)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bbd34479581895296f8a2acf6b6977e90cd1df9" target='_blank'>
              Mechanisms of Intracellular Selection of Mitochondrial DNA
              </a>
            </td>
          <td>
            Georgii Muravyov, Dmitry A. Knorre
          </td>
          <td>2025-12-01</td>
          <td>Biochemistry (Moscow)</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human cells possess a complex network of regulatory systems——the DNA damage repair pathway (DDR), to deal with various DNA damage to prevent the inheritance of deleterious mutations. DDR perform its functions via cell cycle checkpoint activation. Mutations in the DDR gene are also the basis for tumor progression, metastasis, and therapeutic effect to treatments that cause DNA damage. Consequently, DDR has received high attention for its role in the therapy against cancer, and the related experiments have been carried out to further explore. Up to now, PARP inhibitors such as Olaparib, Rucaparib, Niraparib, Talazoparib, etc. have been approved for marketing and clinical use, other DDR inhibitors (DDRi) such as ATMi/ATRi/DNA-PKi, Chk1/2i, and WEE1i have partially entered clinical trials. Synthetic lethality has led to the exploration of combination of DDRi with other anti-tumor therapies, in particular, the combination of DDRi with immunotherapies such as chemoradiotherapy and immune checkpoint inhibitors (ICIs) has achieved potential efficacy. This article reviewed the DDR inhibitors that are currently in development or in clinical therapy, analyzed the limitations and future development prospects, and stated the mechanisms and clinical trials in antitumor and immunotherapy, to provide ideas for the subsequent development of DDRi and enhance the efficacy of antitumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9a6e3b57be514835a193a70ee6587199732e497" target='_blank'>
              Research progress of DNA damage repair (DDR) and DDR inhibitors in tumor immunotherapy
              </a>
            </td>
          <td>
            Qi Liu, Chunmei Zhang, Yixuan Gao, Dongmei Feng, Duo Deng, Yun Pan
          </td>
          <td>2025-12-01</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Systemic sclerosis (SSc) is a rare, complex autoimmune disease characterized by fibrosis of the skin and internal organs. While its pathogenesis is not fully understood, chromosomal instability and telomere attrition have emerged as significant areas of investigation. Methods: This review provides a historical narrative perspective and synthesizes current findings on the role of these genomic anomalies in SSc pathogenesis. We synthesized findings from foundational and recent research articles investigating genotoxic factors, chromosomal aberrations, and telomere biology in SSc. Results: There is a strong historical basis for chromosomal instability in SSc, manifesting as micronuclei, translocations, and breaks. This instability is driven by clastogenic factors and oxidative stress. SSc-specific autoantibodies are implicated; anti-centromere antibodies correlate with aneuploidy and micronuclei, while anti-topoisomerase I may inhibit DNA repair. SSc is also characterized by significant telomere attrition, first reported in 1996 and now confirmed by additional genetic studies. This telomere loss is associated with reduced telomerase activity and the presence of autoantibodies against telomere-associated proteins, including shelterin components. Conclusions: We conclude that inflammation, telomere attrition, and chromosomal instability are linked in a self-perpetuating cycle that drives SSc pathogenesis. We propose that an initial inflammatory stimulus leads to reactive oxygen species production, causing telomere damage and attrition. Critically short telomeres trigger faulty DNA repair mechanisms, such as breakage–fusion–bridge cycles, resulting in chromosomal instability. This genomic damage, in turn, acts as a danger signal, further activating inflammatory pathways and creating a feedback loop that perpetuates fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a5b157843adfebc89c062c9bea135fbbb56036" target='_blank'>
              Chromosomal Instability and Telomere Attrition in Systemic Sclerosis: A Historical Perspective
              </a>
            </td>
          <td>
            C. M. Artlett
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer has the worst prognosis among major gynecological cancers. Current therapies include platinum, Taxol, angiogenesis inhibitors, and poly[ADP-ribose]polymerase (PARP) inhibitors. However, resistance develops in most ovarian cancer patients. Identification of more pro-tumor factors in ovarian cancer may provide insights into ovarian cancer biology and therapy. In this study, we find ZKSCAN3, a zinc-finger transcription factor, is overexpressed in ovarian cancer. We show that ZKSCAN3 promotes ovarian cancer cell proliferation. Through RNA-Seq and chromatin immunoprecipitation (ChIP)-seq, HSPB1 is identified as a target gene of ZKSCAN3. HSPB1 expression is significantly decreased upon suppressing ZKSCAN3 expression. Suppressing HSPB1 expression also inhibits ovarian cancer cell proliferation. In contrast, expressing exogenous HSPB1 partially rescues the cell proliferation in ZKSCAN3 knockdown cells, which supports HSPB1 as a functional target gene of ZKSCAN3. Collectively, our study uncovers a functional ZKSCAN3-HSPB1 axis that promotes ovarian cancer cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dcfe8bb644055044600a01a7e4f21c3861ebc99" target='_blank'>
              ZKSCAN3 promotes ovarian cancer cell proliferation by increasing HSPB1 expression
              </a>
            </td>
          <td>
            Qian Ke, Zhenyong Li, Li Fan, Neng Li, Lidong Sun, Hongbo Zhao, T. Song
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6f0f556d95e6eb279f30e807f354e50710c0e3" target='_blank'>
              Restoring Shugoshin 1 reduces chromosome errors in human eggs
              </a>
            </td>
          <td>
            Debojit Saha, Saba Manshaei, T. Cavazza, Z. Holubcová, B. Maierová, Agata P. Zielinska, Lena Wartosch, Martyn Blaney, Kay Elder, Melina Schuh
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is an aggressive pediatric soft tissue cancer, with approximately 350 cases per year in the US. The highly aggressive tumor subtype, Fusion-Positive Rhabdomyosarcoma (FP-RMS) is characterized by chromosomal translocations yielding oncogenic fusion transcription factors (TFs). The most common RMS fusion (PAX3::FOXO1) forms an in-frame chimera of the DNA-binding domain of PAX3 with the activator domain of FOXO1. We hypothesize that understanding the precise mechanisms of PAX3::FOXO1’s chromatin engagement and nucleosome invasion would enable precision therapy to target these mechanisms to ultimately improve patient outcomes. Our studies are providing evidence that PAX3::FOXO1 has pioneer activity, including nucleosome engagement, and an ability to recognize multiple chromatin states including both open and repressive chromatin regions. We are investigating the biochemical regulatory functions of PAX3::FOXO1 binding to specific DNA motifs embedded within chromatin or nucleosomes, developing synthetic epigenetics to investigate motifs from our genomics-scale studies. We are interested in the preservation of naturally occurring post-translational modifications from the mammalian cellular context, and have sought to build a highly versatile system for investigation of TF/nucleosome targeting from cell extracts. In our studies, we are finding new evidence for PAX3::FOXO1’s targeting of homeodomain and paired domain DNA motifs in free DNA and also in a nucleosome context. We are validating these findings through parallel approaches including coupling electrophoretic mobility shift assays (EMSAs) with westerns and antibody-targeting of complexes within EMSA experiments. We are finding that PAX3::FOXO1’s binding to high-complexity motifs, conjoining paired domain and homeodomain sequences, occurs during nucleosome invasion. Through iterative comparisons, custom unmodified mono-nucleosomes containing three major classes of DNA binding motifs reveal PAX3::FOXO1’s binding to these nucleosomes without a strong motif preference within the categories examined in our series. These findings support a mechanistic hypothesis where PAX3::FOXO1 may invade nucleosomes, altering their local positioning prior to generation of chromatin accessibility. Additional investigations are underway to unravel these mechanisms further and probe the contexts for motif specificity or generality in nucleosome engagement, and to understand conformational changes in the target nucleosomes. The mechanisms we are uncovering will contribute to the search for new therapeutic vulnerabilities as our community progresses toward targeted therapies for FP-RMS.



 Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic Barr, Michael G. Poirier, Benjamin Stanton. Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbbbd914885e1df8de9eefd492cd42d843d809e" target='_blank'>
              Abstract B016: Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic G. Barr, Michael G. Poirier, Benjamin Z. Stanton
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell analysis has refined our understanding of cellular heterogeneity in glioma, yet RNA alternative splicing, a critical layer of transcriptome regulation, remains underexplored at single-cell resolution. Here, we present a pan-glioma single-cell alternative splicing analysis in both tumor and immune cells through integrating seven SMART-seq2 datasets of human gliomas to uncover overlooked isoform-level regulations shaping glioma progression and immune responses. Our analysis reveals lineage-specific alternative splicing regulation among glioma cellular states, with the most divergent alternative splicing landscapes observed between mesenchymal and neuronal-like glioma cells. Notably, this includes events in TCF12 exon 15 and PTBP2 exon 10, two key regulators of mesenchymal and neuronal transcriptomes. Inducing TCF12 exon 15 inclusion in glioma cells with a dCasRx-RBM25 system promotes neuronal gene expression, suppresses genes related to extracellular matrix organization, and enhances sensitivity to radiotherapy. Comparison of core and peripheral glioma cells highlights alternative splicing redox co-regulation of TPM1 and ACTN4, key genes involved in cytoskeletal organization. Further analysis of glioma-infiltrating immune cells reveals altered first exon usage of UGP2 in regulatory T cells and an association between MS4A7 alternative splicing in macrophages and clinical response to anti-PD-1 therapy. This study emphasizes the role of alternative splicing in glioma cellular heterogeneity, highlighting the importance of an isoform-centric approach to better understand the complex biological processes driving tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0607c4921b5c1be62ed81a8c96b4eebea1605a52" target='_blank'>
              A single-cell atlas of RNA alternative splicing in the glioma-immune ecosystem
              </a>
            </td>
          <td>
            Xiao Song, D. Tiek, Minghui Lu, Xiaozhou Yu, Runxin Wu, Maya Walker, Qiu He, Derek Sisbarro, Bo Hu, Shi-Yuan Cheng
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77606e4919065b0b8c161216da3a8b4fee1b4a2c" target='_blank'>
              Deficiency of SMARCB1 drives an immunosuppressive microenvironment in meningioma.
              </a>
            </td>
          <td>
            Ben Jin, Hao Hu, Yanhua Lu, Xia Tian, Guanghui Hu, Jingjing Xu, Xingqi Wu, Long Zhang, Juxiang Chen, Miaoxia He
          </td>
          <td>2026-01-24</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76ffd3264105dafb3fc57d7ef47ddb286552ab9" target='_blank'>
              Mechanistic basis for relaxation of DNA supercoils by human topoisomerase IIIα-RMI1-RMI2.
              </a>
            </td>
          <td>
            Dian Spakman, A. Biebricher, Anna H. Bizard, I. Hickson, Erwin J. G. Peterman, G. Wuite, Graeme A King
          </td>
          <td>2026-01-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Cell-free DNA (cfDNA) refers to DNA fragments that circulate freely in bodily fluids, primarily blood plasma. These fragments are released from cells via apoptosis and necrosis. Discovered in 1948, cfDNA gained clinical importance when linked to vascular dysfunction, positioning it as a promising non-invasive biomarker.



 To highlight cfDNA's diagnostic and prognostic applications in oncology and other fields, and to examine key challenges and future directions.



 In healthy individuals, cfDNA originates from normal cell turnover. In contrast, in diseases like cancer, circulating tumor DNA (ctDNA)—a subset of cfDNA—originates from tumor cells and contains mutations such as point mutations, rearrangements, and copy number variations. These genetic alterations reflect the primary tumor, allowing cfDNA analysis to guide clinical decision-making.



 In cancer, cfDNA enables early detection, therapy monitoring, tumor evolution tracking, and residual disease assessment. It also plays a role in prenatal screening, transplant rejection monitoring, and detecting infectious and cardiovascular conditions. Its short half-life (15 minutes to 2.5 hours) offers dynamic, real-time monitoring. However, low cfDNA concentrations, technical limitations (e.g., background noise, sequencing errors), and high costs in low-resource settings pose significant challenges. Detection methods like digital PCR and NGS are essential but expensive. Standardization issues in protocols and ethical concerns regarding genetic privacy further complicate adoption.



 cfDNA represents a transformative tool in precision medicine. Technological advances, integration with AI, and combining cfDNA with other biomarkers could enhance sensitivity and specificity. As clinical validation progresses and protocols standardize, cfDNA testing is expected to become routine, improving early disease detection and patient outcomes.



 Mohamed Tharwat. Kamouna. Cell-Free DNA: A Paradigm Shift in Cancer Diagnostics and Precision Medicine [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: The Rise in Early-Onset Cancers—Knowledge Gaps and Research Opportunities; 2025 Dec 10-13; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(23_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aec98a64fa684695d33212f886da7270836c0f" target='_blank'>
              Abstract A023: Cell-Free DNA: A Paradigm Shift in Cancer Diagnostics and Precision Medicine
              </a>
            </td>
          <td>
            Mohamed Tharwat. Kamouna
          </td>
          <td>2025-12-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The preservation and faithful propagation of genetic information is essential for all life forms and depends on cellular pathways that enable replication, recombination, and repair of DNA. The multifunctional XPF-ERCC1 DNA endonuclease complex acts in several DNA repair pathways and interacts with numerous partner proteins and large DNA repair assemblies, including the nucleotide excision repair machinery and the SMX tri-endonuclease complex. Here, we report structures of XPF-ERCC1 in complex with the DNA repair factors SLX4 and SLX4IP, thereby identifying key residues responsible for direct interactions with XPF-ERCC1. When introduced into human cells, point mutations in these interfaces impair the interactions between XPF-ERCC1 and SLX4 or SLX4IP, and disruption of the XPF-SLX4IP interface leads to cis-platin sensitivity. Furthermore, our data reveal the structure of the human XPF-ERCC1-SLX4IP-SLX4330-555 complex with DNA bound at its active site, and they complete the structural characterisation of molecular interactions required to assemble the SMX complex. Structure-specific endonucleases play an important role in several DNA repair pathways. Here the authors present structures of the endonuclease XPF-ERCC1 in complex with SLX4, SLX4IP, and DNA. Combined with functional analysis, these results provide insight into the mechanisms of XPF-ERCC1 recruitment and activation during DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff06e1b310b6d6c04354cb0699bf090be746399" target='_blank'>
              Molecular basis of XPF-ERCC1 targeting to SLX4-dependent DNA repair pathways
              </a>
            </td>
          <td>
            Junjie Feng, Peter R. Martin, Szymon Kowalski, Maxime Lecot, Nora B Cronin, T. Matthews-Palmer, Wojciech Niedzwiedz, Basil J. Greber
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Malaria remains one of the devastating illnesses, and drug-resistant malaria has incurred enormous societal costs. A few host kinases are vital for the liver stage malaria and might be promising drug targets against drug-resistant malaria. STK35L1 is one of the host kinases that is highly upregulated during the liver stage of malaria, and the knockdown of STK35L1 significantly suppresses Plasmodium sporozoite infection. In this study, we retrieved the promoter region of STK35L1 based on 5′ complete transcripts, transcription start sites, and cap analysis of gene expression tags. Furthermore, we identify transcriptionally active regions by analyzing CpG islands, histone acetylation (H3K27ac), and histone methylation (H3K4me3). It suggests that the identified promoter region is active and has cis-regulatory elements and enhancer regions. We identified various putative transcription factors (TFs) from the various high-throughput ChIP data that might bind to the promoter region of STK35L1. These TFs were differentially regulated during the infection of Plasmodium sporozoites in HepG2 cells. Our molecular modeling study suggests that, except for SMAD3, the identified TFs may be directly bound to the promoter. Together, the data suggest that these TFs may play a role in sporozoite infection and in regulating STK35L1 expression during the liver stage of malaria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5083d80d5c64f4a36358495eb61056154d577e3" target='_blank'>
              Differential Expression of STK35L1-Associated Transcription Factors in Plasmodium Infection During the Liver Stage of Malaria
              </a>
            </td>
          <td>
            Arpana Yadav, P. Sharma, Mayuree Hazarika, Pragya Gehlot, Saloni Bage, Mahesh Saini, Kritika Gaur, Acham Parambath Aswathi, Malti Thakur, Devesh M. Sawant, A. Singh, D. Brünnert, Pankaj Goyal
          </td>
          <td>2025-12-12</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) infects mainly CD4+ T lymphocytes and causes both malignant and inflammatory diseases: the aggressive malignancy known as Adult T-cell leukemia/lymphoma (ATL) and several chronic inflammatory syndromes. HTLV-1 infection is established by integration of proviral DNA (~9000 bp) into the host genome. In HTLV-1, two viral genes, tax and HBZ, play critical roles in viral transcription and promotion of T-cell proliferation, respectively. The present study was undertaken to test the hypothesis that the higher-order structure of proviral chromatin regulates its transcription on both the plus strand and the minus strand. ATAC-seq analysis identified an open chromatin region in the pol gene of proviral DNA, which we name IPOR, in many ATL cases. Using reporter assays, it was found that the sequence of IPOR suppresses the transcription of the plus strand and activates that of the minus strand. Binding motifs of Eomes and TEAD proteins were predicted in this region, and we confirmed recruitment of the transcription factors to their respective motifs by ChIP-qPCR. A mutant of IPOR which cannot bind the transcription factors weakened the transcriptional activating effects compared with the wild type, suggesting that the IPOR and those transcription factors suppress the 5' long terminal repeat (LTR) but activate the 3'LTR. In addition, a mutant HTLV-1 molecular clone, which possesses the IPOR mutant, produced a higher titer of virus than the wild type. RNA-seq analysis of HTLV-1-infected cell lines, in which Tax expression can be traced after induction, revealed that EOMES expression decreases during the tax transcriptional burst and resumes following termination of the burst. These findings suggested that the expression dynamics of Eomes affect the transient expression of Tax. The IPOR sequence appears to regulate the transcription from both LTRs, suppressing the 5'LTR but activating the 3'LTR. Recruitment of Eomes to the IPOR is likely to influence the expression of Tax and HBZ. Since both viral genes are involved in diverse mechanisms for viral replication, cellular proliferation, and immune regulation, the IPOR may play a role in fine-tuning the modes of viral persistence in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2f1e6ae01e0a8f9b5438f95ad7675208ffd69" target='_blank'>
              Intra-pol proviral open region of HTLV-1 controls the transcription from both long terminal repeats
              </a>
            </td>
          <td>
            Miyu Sonoda, Azusa Tanaka, Xueda Chen, Wenyi Zhang, Helen Kiik, S. Ramanayake, K. Nosaka, N. Takenouchi, Masanori Nakagawa, Akihiro Fujimoto, C. Bangham, Masao Matsuoka, Jun-ichirou Yasunaga
          </td>
          <td>2025-12-08</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d96eca4dc76fb7d7dcfd02bd48a5d396e340f16" target='_blank'>
              Role of circAGFG1 as an oncogene in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wei Zhang, Lizhe Zhu, Xiaoqian Li, Jingyi Yang, Yu Ren, Jianjun He, Jing Xu
          </td>
          <td>2026-01-01</td>
          <td>Apoptosis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor that occurs in the mucosal epithelial cells of the colon or rectum, with high mortality and morbidity. Therefore, the study of its molecular principle is more important. In recent years, more attention has been paid to the regulation of CRC by epigenetic factors, especially the regulation of non-coding RNAs (ncRNAs) and RNA modifications. ncRNAs include miRNA, circRNA, and lncRNA. Among them, miRNAs regulate MAPKs, Wnt, and TGF-β signaling pathways and are involved in the tumor process. CircRNAs, as regulators of gene expression, can participate in various biological processes. lncRNA interacts with DNA, RNA, and proteins to regulate signaling pathways in various ways, potentially acting as biomarkers in non-invasive examinations. RNA modifications, especially N6-methyladenosine (m6A), affect CRC development. m6A consists of three proteins, writers, erasers, and readers, which maintain the stability of RNA modifications. Different levels of expression of these three proteins in CRC can lead to changes in the tumor microenvironment, gene expression, and other factors, and thus play an important role in CRC. ncRNAs and RNA modifications comprise the CRC epigenetic regulatory network, providing a new direction for the diagnosis and treatment of CRC in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d5caa5bf293c7752dde20eabcef8521208dcce" target='_blank'>
              Center Molecular Mechanisms of Non-coding RNAs and RNA Modifications in Colorectal Cancer
              </a>
            </td>
          <td>
            Tongyue Yu
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An R-loop is a three-stranded nucleic acid structure that serves as a transcriptional intermediate, consisting of an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA). Small RNAs are RNA molecules shorter than 300 nucleotides that perform a wide range of essential functions within cells. Both R-loops and small RNAs are widely present in the genomes of prokaryotes and eukaryotes, where they play crucial roles in regulating gene expression, maintaining genomic stability, and facilitating DNA damage repair. Aberrant formation and accumulation of R-loops, coupled with dysregulation of small RNA pathways, can induce DNA damage and genomic instability, ultimately contributing to cellular senescence or cell death. Here, we discuss recent advances in understanding the crosstalk between R-loops and small RNAs, with a focus on their synergistic roles in maintaining genome stability and their therapeutic potential in oncology and neurodegeneration. We propose a novel model integrating R-loop dynamics with small RNA-mediated epigenetic regulation, supported by emerging clinical trial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853eaea8f06e1b31e5bea0e5a97d17746ae77587" target='_blank'>
              R-loops and small RNA regulatory interactions: mechanisms and clinical perspectives
              </a>
            </td>
          <td>
            Si-Yao Wang, Ya-Ni Liu, Si-Qiao Zhao, Wen Liu, Jin Hu, Fan Yang, Sheng Wang
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="X chromosome inactivation is a process that compensates X-linked gene dosage in mammalian female cells. The silencing of a randomly selected chromosome is accompanied by dramatic three-dimensional reorganization across the entire chromosome. To investigate the four-dimensional chromatin dynamics during early inactivation stages, we applied the multi-omics sequencing technique HiRES (Hi-C and RNA-seq employed simultaneously), which simultaneously detects the three-dimensional genome and transcriptome in single cells, in a mouse embryonic stem cell line with induced random inactivation. This three-dimensional genome and transcriptome dual-omics data allowed us to identify random inactivation trajectories at single-cell resolution. We characterized multiple layers of X-chromosome reorganization and discovered a transient structural state shared by both X chromosomes, associated with biallelic X-inactive specific transcript (Xist) expression. By constructing single-cell inactivation trajectories, we found that most chromatin remodeling either accompanied or followed gene silencing. Further analysis of interaction decay kinetics revealed that topologically associating domain (TAD) attenuation began from loss of interactions on TAD anchors. This study thus provides a detailed depiction of fine-scale chromatin reorganization during the initiation of random X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf9c97bb415329f626b342326dffac87d2e88f04" target='_blank'>
              4D Chromatin Dynamics Resolved During Early Random X Chromosome Inactivation.
              </a>
            </td>
          <td>
            Xiaowen Liu, Hao Xie, Zhiyuan Liu, Yujie Chen, Qimin Xia, Heming Xu, Yi-Chun Chi, Shuai Gao, Dong Xing
          </td>
          <td>2026-01-13</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [0, 13, 5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>