<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-05 09:41:42 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>44</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>90</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>123</td>
          <td>147</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>394</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>341</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres, stochastically segregate during cell division3–6 and persist over many generations. It has been more than 40 years since ecDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei, but the mechanism underlying this process remains unclear3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. Using Retain-seq, a genome-scale assay that we developed, we reveal thousands of human retention elements that confer generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromosome conformation capture show that retention elements physically interact with mitotic chromosomes at regions that are mitotically bookmarked by transcription factors and chromatin proteins. This activity intermolecularly recapitulates promoter–enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated. Targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, which suggests that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, and reveal the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D.
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, Jessica Alsing, J. Luebeck, Maria E. Stefanova, Owen S Chapman, Katerina Kraft, Shu Zhang, Jun Yi Stanley Lim, Yipeng Xie, Yoon Jung Kim, Sihan Wu, Lukas Chavez, Guy Nir, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract In this Perspective article, we follow the journey of a gene that breaks free from its stringent chromosomal transmission dynamics to become an independently multiplying agent on so-called extrachromosomal circular DNA (ecDNA/eccDNA). We discuss how the release of a gene from its chromosomal anchor can affect its dosage, regulatory context, and potential evolutionary trajectory before examining the phenotypic implications for unicellular and multicellular eukaryotes that transmit genes on circular DNA. We also briefly explore the fundamental difference between circularized genes in flowering plants (angiosperms) and mammals (primarily cancer) concerning somatic and germline inheritance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2 The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage-replication/fusion process explains complex rearrangements and segmental DNA amplification.
              </a>
            </td>
          <td>
            , Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db26ccc6dc7e76ac6a63d600396d97f0ca6b89f2" target='_blank'>
              Passenger mutations link cellular origin and transcriptional identity in human lung adenocarcinomas
              </a>
            </td>
          <td>
            Sukanya Panja, Padmaja Mantri, K. E. Johnson, J. Andrade-Martínez, S. Yang, Aditya Deshpande, Huasong Tian, Shaham Beg, Kentaro Ohara, Alessandro Leal, Joel Rosiene, Marlon Stoeckius, Peter Smibert, William D. Travis, J. Mosquera, P. Polak, Marcin Imieliński
          </td>
          <td>2025-11-27</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary This study investigates the genomic alterations of cleavage and polyadenylation factors and their clinical relevance in cancer. We show that mutations in CPA genes may not be clinically relevant. However, copy number variations, particularly amplification of CPSF1, demonstrate significant impact on patient prognosis. We show that CPSF1 amplification, although co-occurring with MYC amplification, can independently change alternative polyadenylation (APA) patterns and dysregulate cancer-associated gene signatures. These results highlight the crucial impact of CPA gene copy number variation on biological processes and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The genome folds inside the cell nucleus into hierarchical architectural features, such as chromatin loops and domains. If and how this genome organization influences the regulation of gene expression remains only partially understood. The structure-function relationship of genomes has traditionally been probed by population-wide measurements after mutation of critical DNA elements or by perturbation of chromatin-associated proteins. To circumvent possible pleiotropic effects of such approaches, we have developed OptoLoop, an optogenetic system that allows direct manipulation of chromatin contacts by light in a controlled fashion. OptoLoop is based on the fusion between a nuclease-dead SpCas9 protein and the light-inducible oligomerizing protein CRY2. We demonstrate that OptoLoop can drive the ind uction of contacts between genomically distant, repetitive DNA loci. As a proof-of-principle application of OptoLoop, we probed the functional role of DNA looping in the regulation of the human telomerase gene TERT by long-range contacts with the telomere. By analyzing the extent of chromatin looping and nascent RNA production at individual alleles, we find evidence for looping-mediated repression of TERT. In sum, OptoLoop represents a novel means for the interrogation of structure-function relationships in the genome at single-allele resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6351708081597f9b8207513e5bf5a9d141739521" target='_blank'>
              OptoLoop: An optogenetic tool to probe the functional role of genome organization
              </a>
            </td>
          <td>
            Martin Stortz, Adib Keikhosravi, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has emerged as a critical determinant of poor clinical outcomes and immune escape in tumors, but the high cost and technical complexity of current detecting techniques limit its broader investigation in cancer immunotherapy. Leveraging the combined machine learning algorithms including the least absolute shrinkage and selection operator (LASSO) regression, RandomForest (RF) and Recursive Feature Elimination (RFE), we developed a 12-gene transcriptomic score (EC_score) to predict the existence of ecDNA through RNA-seq. EC_score demonstrated reliable predictive performance in two independent cohorts (AUC > 0.70), validated by fluorescence in situ hybridization (FISH) in both cell lines and clinical samples. Next, we found that EC_score emerged as an independent adverse prognostic factor across multiple immunotherapy cohorts. Notably, high EC_score correlated with cell cycle activation and immunosuppression, characterized by reduced lymphocytes infiltration and upregulated immunosuppressive markers, including MHC molecules, co-inhibitory immune checkpoints and TGF-β signals. In general, we established and validated a 12-gene signature (EC_score) derived from RNA-seq, offering a novel computational tool for predicting the presence of extrachromosomal circular DNA and stratifying cancer immunotherapy response. Graphical abstract Overview of the analytical framework integrating multi-omics data, machine learning, and experimental validation to establish EC_score and characterize its clinical implications. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04056-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97428a7d98b9f7fbbd3050ae63fe8f2b1c83fe29" target='_blank'>
              Machine learning identified extrachromosomal DNA-related 12 gene signatures to predict cancer immunotherapy response
              </a>
            </td>
          <td>
            Yan Ju, Jingwei Zhang, Jiaming Deng, Xingyuan Wu, Haotian Yang, Changyu Tao, Xiao Li
          </td>
          <td>2025-12-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Motivation Tumor viruses account for ∼10% of cancer diagnoses. Virally induced tumorigenesis is understood as direct signaling through oncogenes such as E6 and E7 genes in the case of human papillomavirus. Furthermore, pathogen characteristics such as viral oncogene dose may impact the disease course. To our knowledge, no tool has been proposed to assess the intra-viral copy number alterations that define the gene dose of viral oncogenes and associated suppressive pathways native to the pathogen’s normal life cycle. Results We propose an R package, “ELViS,” that analyzes viral copy number changes from DNA sequencing of whole viral genomes. The method adjusts for viral load with 2D transformation and segmentation to offer the relative viral gene doses. Availability and implementation The ELViS R package is available from https://bioconductor.org/packages/ELViS. This article used controlled access data from dbGaP (phs001713.v1.p1).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee403f958df88d61608425c7ffde897d6bfc505e" target='_blank'>
              ELViS: an R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, J. Holt, Xiaobei Zhao, K. Hoadley, D. N. Hayes, Hyo Young Choi, Neil Hayes
          </td>
          <td>2025-11-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) assays are being widely adopted in oncology and maternal-fetal medicine. Patterns of cfDNA fragmentation can provide useful information about gene regulation and expression in human disease from a blood draw. Here, we demonstrate that enhancer RNA expression – a marker of enhancer activity – can be inferred from local patterns of cfDNA fragmentation. We define a transcriptional activation score (TAS) that predicts expression of enhancers and genes based on cfDNA fragment sizes and positions near transcriptional start sites (TSSs). The TAS identifies activity of cancer-associated enhancers in patients with cancer, distinguishes clinically relevant cancer subtypes, and identifies activation of enhancers associated with treatment resistance and therapy response. We propose a simple model to account for our findings based on chromatin fiber structure and the depletion of H1 histone proteins near active TSSs. Our model provides a unified framework that reconciles seemingly conflicting observations from prior fragmentomics studies. Broadly, this work enables blood-based assessments of gene regulation in cancer and non-oncologic diseases to inform pathobiology, diagnosis, and treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs), widely distributed in the human genome, are usually epigenetically silenced but can be reactivated in cancer. Activated ERVs may act as regulatory elements or produce noncoding RNAs that modulate gene expression and affect tumor biology. Their role in pancreatic ductal adenocarcinoma (PDAC), particularly in ferroptosis regulation, remains unclear. Here, we identify human endogenous retrovirus-H (HERVH) derived enhancer RNAs (eRNAs) aberrantly overexpressed in PDAC by integrating gene expression and histone modification data. Silencing HERVH markedly suppresses PDAC proliferation and tumor growth. Mechanistically, HERVH transcription is driven by the super-enhancer–associated transcription factor Krüppel-like factor 5 (KLF5), and HERVH eRNAs cooperate with KLF5 to transactivate a distal super-enhancer upstream of aldehyde dehydrogenase family 1 member 3 (ALDH1A3), increasing its expression. ALDH1A3 inhibits the reactive oxygen species/specificity protein 1/spermidine/spermine N1-acetyltransferase 1 (ROS/Sp1/SAT1) cascade, reducing lipid peroxidation and ferroptosis. Disrupting this axis induces ferroptotic death and impairs PDAC development. Our study reveals a previously unrecognized mechanism in which HERVH-derived eRNAs regulate ferroptosis via super-enhancer–mediated transcriptional reprogramming and highlights the KLF5/HERVH/ALDH1A3 pathway as a potential therapeutic target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd3b7b06a171128bfce591ad0b7d6fc1bf653f9" target='_blank'>
              HERVH-derived eRNA activates a super-enhancer–driven ALDH1A3/SAT1 axis to promote ferroptosis escape and pancreatic cancer development
              </a>
            </td>
          <td>
            Lili Yuan, Jiahui Yu, Xiaoli Xu, Yiwen Huang, Dian-Fang Wei, Zhenbo Hou, Huizhen Nie, Tiantian Jing, Yizhen Liu, Boshi Wang
          </td>
          <td>2025-12-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Abstract Chromosomal structural changes happen when genomic stability is compromised, such as in disease or in species hybrids. In these contexts, diminished control of repetitive elements has been reported, but the reasons for this are not yet well understood. There are causal associations between repetitive elements and phenotypes such as disease progression, leading us to the hypothesis that chromosomal structure may be affected by transposable elements (TEs). In an intraspecific hybrid Drosophila melanogaster cell line (PnM), the degree of pairing among trans homologous chromosomes was affected by the presence of nearby TEs, in particular, LINE and LTR elements, such as Baggins1 or Gypsy. Chromosomal pairing was significantly lower in windows containing TEs than in windows without any TEs. Pairing was also affected by TEs in mouse, which suggests a possible general association between TEs and pairing that is highly conserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b05bf5cfcd82a8fc82e293a52e5e86b57f277d" target='_blank'>
              Haplotype-Resolved Chromatin Conformation Data Reveals Relationship Between Transposable Elements and Chromosomal Pairing
              </a>
            </td>
          <td>
            Luke K Genutis, Patrick Villanueva, Rita M. Graze, Jumana Alhaj Abed, Lauren M McIntyre, S. Nuzhdin
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Most animal genomes are partitioned into topologically associating domains (TADs), created by cohesin-mediated loop extrusion and defined by convergently oriented CCCTC-binding factor (CTCF) sites. The dynamics of loop extrusion and its regulation remain poorly characterized in vivo. Here we tracked the motion of TAD anchors in living human cells to visualize and quantify cohesin-dependent loop extrusion across multiple endogenous genomic regions. We show that TADs are dynamic structures whose anchors are brought in proximity about once per hour and for 6–19 min (~16% of the time). Moreover, TADs are continuously extruded by multiple cohesin complexes. Remarkably, despite strong differences in Hi-C patterns across chromatin regions, their dynamics is consistent with the same density, residence time and speed of cohesin. Our results suggest that TAD dynamics is primarily governed by the location and affinity of CTCF sites, enabling genome-wide predictive models of cohesin-dependent chromatin interactions. Live-cell fluorescence microscopy and polymer simulations in human HCT116 cells show similar cohesin density, residence times and extrusion speed across multiple genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715a0a11e407c2847c391eae8db4fee949a2d73b" target='_blank'>
              Uniform dynamics of cohesin-mediated loop extrusion in living human cells
              </a>
            </td>
          <td>
            Thomas Sabaté, Benoît Lelandais, Marie-Cécile Robert, Michael Szalay, J. Tinevez, Edouard Bertrand, Christophe Zimmer
          </td>
          <td>2025-11-14</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-29898-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tandemly repeated non-coding sequences, widely known as satellite DNAs (satDNAs), are extremely diverse and highly variable components of eukaryotic genomes. In recent years, advances in high-throughput sequencing and new bioinformatics platforms have enabled in-depth studies of all (or nearly all) tandem repeats in any genome (the satellitome), while a growing number of telomere-to-telomere assemblies facilitates their detailed mapping. Research performed on a large number of non-model plant and animal species changed significantly the “classical” view on these sequences, both in an organizational and functional sense, from ballast compacted in the form of heterochromatin to elements that are important for structuring the entire genome, as well as for its functions and evolution. The diversity of repeat families, and the complexity of their intraspecies and interspecies distribution patterns, posed new questions, urging for species-by-species comparative analyses. Here we integrate some basic features of different forms of sequences repeated in tandem and rapidly growing data evidencing extensive dispersal of satDNA sequences in euchromatin, their putative roles and evolutionary significance. Importantly, we also present and discuss various issues brought on by the use of new methodological approaches and point out potential threats to the analysis of satDNAs and satellitomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df02ce17404430620feb58c2c132e48c76b2721" target='_blank'>
              Satellite DNA Genomics: The Ongoing Story
              </a>
            </td>
          <td>
            M. Garrido-Ramos, Miroslav Plohl, Eva Šatović-Vukšić
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract The clonal evolution of breast cancer involves a complex dialogue between tumor cells and their environment. In this process, epigenetic mechanisms play a crucial role in regulating the cellular transcriptome without altering the underlying DNA sequence. Here, we provide an updated summary of three main epigenetic mechanisms: histone modifications, long non-coding RNAs (lncRNAs), and higher-order chromatin structures. Post-translational modifications of DNA or histones influence gene expression by altering chromatin accessibility and/or recruiting regulatory protein complexes. This process is dynamically regulated by enzymes that add or remove these marks, as well as by reader proteins that recognize them. Dysregulated expression or malfunction of these regulators creates an aberrant epigenetic landscape and gene expression profile, contributing to breast cancer initiation, metastasis, and drug resistance. Notably, the donor molecules for chromatin modifications are largely derived from intermediate metabolites shaped by environmental cues, highlighting the intricate crosstalk between epigenetic regulation and both cellular and systemic metabolic states. DNA and histone modifications are further interrelated with lncRNAs and higher-order chromatin architectures, which have been actively investigated in breast carcinogenesis. We also briefly introduce the role of epigenetics in other chromatin-associated events such as replication initiation. Aberrant replication initiation can drive gene duplication and genomic alterations resembling those observed in clinical breast cancer, endowing tumor cells with growth advantages and therapeutic resistance. Finally, we summarize emerging therapeutic strategies that target epigenetic vulnerabilities in breast cancer and discuss their current limitations and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502ad7aff3505dc7bf1c94f78c3fe440208426d" target='_blank'>
              Advances in the epigenetic regulation of breast cancer
              </a>
            </td>
          <td>
            Xiao Liu, Xin Wu, Jianan Ding, Xiaoying Qiao, Jing Liang
          </td>
          <td>2025-11-21</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplexes act as transcriptional triggers in oncogenes, and small molecules targeting them show anticancer potential but lack sequence specificity, risking off-target effects. Here, we introduce original fold-back triplex-forming oligonucleotides (FTFOs) as precise alternatives. Physicochemical and biological analyses revealed that FTFOs unfold the c-Myc G-quadruplex and repress oncogene regulatory element by forming noncanonical triplexes that block G-quadruplex DNA from interacting with transcription factors. Derived design rules enabled the identification of FTFOs that reduce c-Myc expression in the Del4 plasmid by up to 80%, significantly impairing cancer cell growth, confluence, mobility, and endogenous c-Myc expression. This work presents FTFOs as an innovative sequence-specific strategy for arresting G-quadruplex structures via triplex formation and suppressing oncogene activity, offering a promising avenue for targeted anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96215ef814f7004690d52622b09c1efcdd6d6e9c" target='_blank'>
              Highly Specific Inhibition of c-Myc Oncogene Expression through Shifting the G-Quadruplex toward a Triplex Structure.
              </a>
            </td>
          <td>
            Jolanta Lisowiec‐Wachnicka, Karolina Zielińska, A. Pasternak
          </td>
          <td>2025-11-12</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental hematology & oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5adbde5a06ec4e198106c731d03948bb0cc143" target='_blank'>
              CAD-C reveals centromere pairing and near-perfect alignment of sister chromatids
              </a>
            </td>
          <td>
            Axel Delamarre, Maurane Reveil, David Pintor Pichon, Michael A. Boemo, I. Whitehouse
          </td>
          <td>2025-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract The human malaria parasite, Plasmodium falciparum, faces unique DNA repair challenges; it is haploid, undergoes asynchronous mitosis termed schizogony, and lacks canonical non-homologous end joining (C-NHEJ). Yet, it has adapted DNA repair pathways that enable survival in distinct environments, including human erythrocytes and hepatocytes, as well as the mosquito vector. Plasmodium falciparum chromosomes are partitioned into a conserved core genome and highly diverse subtelomeric regions containing hypervariable, multicopy gene families, including var, which encodes a critical parasite virulence factor. The molecular mechanisms maintaining this chromosomal structure remain unclear. Here, we describe specific DNA repair pathways that distinguish hypervariable subtelomeric regions from the conserved core genome. By disrupting the DNA repair enzyme PfRad51 and TLS polymerases PfPolζ and PfRev1, we identified differential irradiation hypersensitivity across the cell cycle for TLSΔ parasites and uniform hypersensitivity for PfRad51Δ parasites, highlighting variable roles for these repair pathways. Repair of targeted double-strand breaks demonstrated that PfRad51 is essential for HR-mediated repair in the core genome, whereas a Rad51-independent, homology-directed repair pathway was observed in subtelomeric regions. This previously unidentified alternative repair pathway was independent of TLS polymerases. We propose that these differential DNA repair responses maintain the unique structure that defines P. falciparum chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865a5a32816c7b5bb2a2b106850064ee7a2becd" target='_blank'>
              Plasmodium falciparum DNA repair dynamics reveal unique roles for TLS polymerases and PfRad51 in genome diversification
              </a>
            </td>
          <td>
            Akshay Vishwanatha, Xu Zhang, Yi Jing Liu, Annie Leung, Mikayla Herring, Joseph E. Visone, Amanda Chan, Susanah Calhoun, Kirk Deitsch, Laura Kirkman
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Objectives To address the unelucidated mechanisms of breakpoint formation in TFE3-rearranged renal cell carcinoma (TFE3-rRCC), this study characterizes breakpoint distribution within the TFE3 gene. We further explore how non-canonical DNA structures and their thermodynamic stability fluctuation may act as predisposing factors for the genomic instability driving these characteristic translocations. Methods TFE3 breakpoints were identified in a cohort of 31 TFE3-rRCC tumor samples. The chi-square test was used to assess the statistical significance of breakpoint clustering. To investigate potential structural determinants, we predicted the distribution of G-quadruplex-forming sequences and palindromic motifs. Moving beyond simple motif density, we calculated the local Gibbs free energy changes (ΔG) associated with DNA secondary structures using Mfold and RNAfold to model thermodynamic stability across the TFE3 gene. This thermodynamic stability fluctuation was quantified as the maximum absolute local change in folding free energy (|dΔG|). Finally, this correlation between thermodynamic stability fluctuation and breakpoint location was validated by analyzing the 13 most frequently rearranged genes reported in the COSMIC database. Results A significant breakpoint cluster was identified within intron 5 of TFE3, containing 23 of 31 breakpoints (74.19%; chi-square test, P < 0.05). While the simple density of G-quadruplex or palindromic motifs did not directly correlate with breakpoint locations, a strong association with local thermodynamic stability fluctuation was observed. The region within intron 5 exhibited the highest thermodynamic stability fluctuation. This result suggests that regions of high thermodynamic stability fluctuation are correlated with increased susceptibility to DNA breakage. This finding was corroborated in the COSMIC dataset, where breakpoints in 12 of the 13 most frequently rearranged genes were similarly located near peaks of high |dΔG|. Conclusion Our findings indicate that breakpoint events in TFE3-rRCC are non-randomly clustered within intron 5. This clustering correlates strongly with regions characterized by high thermodynamic stability fluctuation (|dΔG|) of potential non-canonical DNA secondary structures. The principle that elevated local thermodynamic stability fluctuation is a feature of breakpoint locations was supported by analysis of a broader set of oncogenes, suggesting that high local thermodynamic stability fluctuation is a common feature of translocation-prone regions in cancer, representing a plausible, though not proven, contributor to genomic fragility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba18c1a28435183414957442b6594dd9961a8358" target='_blank'>
              Mechanistic study of TFE3 breakage in TFE3-rearranged renal cell carcinoma: the perspective of non-canonical DNA structures and their stability
              </a>
            </td>
          <td>
            Xiaopo Zhang, Qinshuang Dang, Xinghe Pan, Zhenggen Deng, Yanhao Xu, W. Gan, Hongqian Guo
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DIS3 is the main catalytic subunit of the nuclear RNA exosome, a complex playing a crucial role in RNA processing and the degradation of various noncoding RNA substrates. In mice, DIS3 is essential for genomic rearrangements during B cell development, but its role in terminal plasma cell (PC) differentiation has not been explored. Although DIS3 gene alterations are frequent in multiple myeloma (MM), a PC malignancy, their molecular impact remains poorly understood. In this study, we developed an antisense oligonucleotide strategy to knock down DIS3 expression in a well-characterized model of human PC differentiation. Reducing DIS3 expression systematically led to decreased B cell proliferation and impaired PC differentiation with lower levels of switched immunoglobulin secretion. Transcriptome analyses confirmed alterations in the proliferation and differentiation programs, alongside an accumulation of noncoding RNAs. Notably, centromere-associated noncoding RNAs were highly sensitive to DIS3 activity, and their accumulation in DIS3-deficient cells, either as transcripts or DNA-associated RNAs, correlated with the mislocalization of the centromere-specific histone variant CENP-A. We finally observed reduced physiological DNA recombination and somatic hypermutation but increased genomic instability in DIS3-deficient cells, in agreement with the higher levels of IGH translocations observed in our large cohort of DIS3-mutant MM patients. Together, these results underscore the essential role of DIS3 in regulating B cell proliferation, DNA recombination, and physiological or malignant PC differentiation in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eccad993afcb380e43e9bb3557c59e3b0bb73b7" target='_blank'>
              DIS3 licenses B cells for plasma cell differentiation in humans
              </a>
            </td>
          <td>
            Emma Miglierina, Julien Bouder, Delfina Ordanoska, Maiwenn Pineau, S. Léonard, Anaïs Schavgoulidze, Gwenaëlle Quéré, Maeva Le Goff, Maé Bouchet, Steve Genebrier, Samuel Bastos Serra Trinca, Laurent Deleurme, C. Monvoisin, Laure Derrier, Charles Marie Dumontet, L. Delpy, J. Moreaux, Jill Corre, Michel Cogné, Brice Laffleur
          </td>
          <td>2025-11-25</td>
          <td>Cellular and Molecular Immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a1d0df44d9ce5038b1336c99cf98a4e708b134" target='_blank'>
              The alternative lengthening of telomeres pathway through a DNA repair lens: mechanism and therapeutic opportunities.
              </a>
            </td>
          <td>
            D. Muoio, Elise Fouquerel
          </td>
          <td>2025-12-01</td>
          <td>NAR cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA methylation is a key regulator of eukaryotic genomes, most commonly through 5-methylcytosine (5mC). In contrast, the existence and function of N6-methyladenine (6mA) in eukaryotes have been controversial, with conflicting reports resulting from methodological artifacts. Nevertheless, some unicellular lineages, including ciliates, early-branching fungi and the alga Chlamydomonas, show robust 6mA signals, raising questions about their origin and evolutionary role. Here we apply Oxford Nanopore sequencing to profile 6mA at base-pair resolution across 18 unicellular eukaryotes representing all major supergroups. We find that robust 6mA patterns occur only in species that encode the adenine methyltransferase AMT1. Notably, 6mA consistently accumulates downstream of transcriptional start sites, positioned between H3K4me3-marked nucleosomes, indicating a conserved association with transcriptional activation. Our results support the idea that the last eukaryotic common ancestor had a dual methylation system, with transcription-linked 6mA and repressive 5mC, which has been repeatedly simplified in both multicellular and unicellular lineages through the loss of the AMT1 pathway. Long-read sequencing in 18 unicellular eukaryotes reveals that 6mA is widespread across eukaryotes and is enriched at transcriptionally permissive regions, which are also marked by H3K4me3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84210062581c67c09a67215ecb3a57b62da836b" target='_blank'>
              Adenine DNA methylation associated with transcriptionally permissive chromatin is widespread across eukaryotes
              </a>
            </td>
          <td>
            Pedro Romero Charria, Cristina Navarrete, Vladimir Ovchinnikov, Lan Xu, Luke A. Sarre, Victoria Shabardina, Ewa Księżopolska, Elena Casacuberta, David Lara-Astiaso, Arnau Sebé-Pedrós, Alex de Mendoza
          </td>
          <td>2025-11-18</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40038069867c0720d29003167b0971829e037042" target='_blank'>
              Nanopore whole-genome sequencing reveals conserved chromosome-specific telomere architecture across tissues and populations
              </a>
            </td>
          <td>
            Niklas L. Engel, B. Brors, L. Feuerbach, Peter J. Park
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Flap endonuclease 1 (FEN1) is a structure-specific nuclease with critical functions in DNA replication and repair. FEN1 has been proposed as an anti-cancer drug target because of its synthetic lethal interaction with homologous recombination deficiency (HRD). However, the exploration of FEN1 in this context has been hampered by the lack of suitable small molecule tools. Here, we describe MSC778, a highly potent, specific and selective small molecule inhibitor of FEN1 nuclease activity. MSC778 directly engages FEN1 in cells, enhances its retention on chromatin, and selectively kills BRCA-deficient cells. Mechanistically, MSC778 suppresses DNA replication causing S-phase accumulation, DNA damage, and ultimately, cell death. Cancer cell panel screening identified Ewing sarcoma (EWS) cells as sensitive to MSC778, which is driven by the expression of SLFN11. In addition to HRD, CRISPR screening revealed a spectrum of synthetic lethal interactions between MSC778 and DNA damage response (DDR) factors, such as PARP1, USP1, PARG, and ATR. Furthermore, we demonstrate that combined inhibition of these factors with MSC778 induces synergistic killing of cancer cells. Together these data highlight FEN1 inhibition as an attractive precision oncology strategy either as monotherapy or as a combination therapy with a broad range of current and next generation DDR-targeting agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d720ebfea477ebcfaccacc0ad325d640a78165dc" target='_blank'>
              Mechanistic insights into the monotherapy and combination potential of FEN1 inhibition in cancer therapy
              </a>
            </td>
          <td>
            E. Rajendra, Claudio A Lademann, Bethany Mason, Balca R. Mardin, Lucy Armstrong, Silvia Peripolli, Julian Kreis, Timothea Konstantinou, U. Pehl, J. Bomke, Stefanie Scharrelmann, David Perera, B. Leuthner, Maria Filipa Pinto, Owen A Davis, A. Galbiati, Ana Toste Rêgo, Claire L McWhirter, E. Elinati, Ada Sala-Hojman, Julien Lefranc, Sam E Mann, Robert A Heald, F. Zenke, Graeme C M Smith, Helen M R Robinson, L. Burgdorf
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Homologous recombination (HR) and non-homologous end joining (NHEJ) are two main repair pathways of DNA double-strand breaks (DSBs). The regulation of repair pathway choice is crucial for maintaining genomic stability and preventing carcinogenesis. Consequently, there is increasing interest in elucidating the molecular mechanisms that govern DSB repair pathway selection. Here, we identify chromosome 8 open reading frame 33 (C8orf33) as a novel regulator of DSB repair choice. We show that C8orf33 is a nuclear protein localized predominantly to the nucleolus and is recruited to DSB sites within both nuclear and nucleolar regions. We demonstrate that C8orf33 promotes the recruitment of 53BP1, thus channeling DSB repair toward NHEJ. Consequently, C8orf33 inhibits DNA end resection and counteracts the recruitment of HR factors, BRCA1 and RAD51, to DSB sites. Mechanistically, chromatin profiling analysis reveals that C8orf33 antagonizes the chromatin association of KAT8 acetyltransferase at DSB sites leading to reduced histone 4 lysine 16 acetylation (H4K16ac) levels. Accordingly, the loss of C8orf33 enhances KAT8 chromatin binding that increases H4K16ac levels. This promotes the recruitment of HR factors while suppressing the accumulation of NHEJ factors at DSB sites, thereby favoring HR over NHEJ. Additionally, we demonstrate that the elevated HR activity in C8orf33-deficient cells causes genomic instability, as evidenced by accelerated loss of ribosomal DNA repeats and increases cell death. Collectively, our findings establish C8orf33 as a critical regulator of DSB repair pathway choice, safeguarding genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1d5cfef9dd20c22a9cd88671f578821b21baa" target='_blank'>
              C8orf33 dictates DNA double-strand break repair choice by modulating KAT8-mediated H4K16 acetylation
              </a>
            </td>
          <td>
            Laila A. Bishara, Enas R. Abu-Zhayia, Marian Nicola, N. Ayoub
          </td>
          <td>2025-11-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA:RNA hybrids are unusual structures found throughout the genomes of many species, including yeast and mammals. While DNA:RNA hybrids may promote various cellular functions, persistent hybrids lead to double strand breaks, resulting in genomic instability. DNA:RNA hybrid formation and removal are therefore highly regulated, including by enzymes that either degrade or unwind RNA from the hybrid. Meiosis is the specialized cell division that creates haploid gametes for sexual reproduction. Previous work in yeast and mammals showed that elimination of DNA:RNA hybrids by RNase H facilitates meiotic recombination. This work demonstrates that the conserved Sen1 DNA/RNA helicase functions during three temporally distinct processes during yeast meiosis. First, SEN1 allows meiosis-specific genes to be expressed at the proper time to allow entry into meiosis. Second, SEN1 prevents the accumulation of hybrids during premeiotic DNA replication. Third, SEN1 promotes the repair of programmed meiotic double strand breaks that are necessary to form crossovers between homologous chromosomes to allow their proper segregation at the first meiotic division. Given the evolutionary conservation of Sen1 with its mammalian counterpart, Senataxin, studies of Sen1 function in yeast are likely to be informative about the regulation of DNA:RNA hybrids during human meiosis as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8231ab0fa8d6701759423756235a545680a4b00a" target='_blank'>
              The conserved SEN1 DNA/RNA helicase has multiple functions during yeast meiosis
              </a>
            </td>
          <td>
            Robert Gaglione, Leonidas Pierrakeas, Lihong Wan, Jonathan Caradonna, A. MacQueen, Ed Luk, N. M. Hollingsworth
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6212cc21f6c9c548c0deb4b30bef6b44b0cc40c" target='_blank'>
              Analyses of Genome-Wide Recombination in Sulfolobus islandicus
              </a>
            </td>
          <td>
            David J. Krause, Changyi Zhang, Rachel J. Whitaker
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2ceccde009f3d35721416ae6a0441554cd9fa0" target='_blank'>
              The mitotic stopwatch synergizes with mild p53 activation to halt cell proliferation
              </a>
            </td>
          <td>
            Beata E. Mierzwa, Franz Meitinger, A. Desai, K. Oegema
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Low-pass whole-genome sequencing (LP-WGS) of circulating tumor DNA (ctDNA) is increasingly recognized for its utility in identifying somatic copy number aberration (CNA). Here we analyzed LP-WGS ctDNA data from 73 pediatric neuroblastic tumor patients and 11 healthy controls to explore diagnostic value of ctDNA CNA burden (including the genotypings) with a customized bioinformatics workflow. We found that a high baseline ctDNA CNA burden (Tumor DNA fraction, TFx ≥ 0.2%) was present in 36/41 (87.80%) neuroblastoma, 6/22 (27.27%) ganglioneuroblastoma, and 3/10 (30.00%) ganglioneuroma patients. High baseline ctDNA CNA burden could predict high-risk neuroblastic tumors with an area under curve (AUC) of 0.95, sensitivity of 94.12% and specificity of 100%. Frequent chromosomal copy number changes, including chr17q gain, chr7 gain, chr3p loss and chr11q loss, were found in ctDNA. Gain of chr17q demonstrated the highest diagnostic value with an AUC of 0.92, indicating strong sensitivity and specificity for detecting high-risk neuroblastic tumors. The homologous recombination deficiency (HRD) score in the high and intermediate-risk groups was significantly elevated compared to those in the low/very-low-risk group. The TFx levels and segmental alterations significantly decreased in patients of neuroblastic tumor underwent chemotherapy, from median TFx = 13.82% before treatment to 0.24% after treatment (P < 0.0001). Our findings highlight the effectiveness of LP-WGS ctDNA CNA analysis as a promising approach for diagnosis and risk stratification of pediatric neuroblastic tumors, and for monitoring chemotherapy response. Particularly, ctDNA analysis is minimally invasive, rapid and cost-effective, which could bring additional benefits in pediatric practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01256a37f8920b62f2d81f1a7bbc4542d1a51191" target='_blank'>
              Copy number aberrations in circulating tumor DNA enable diagnosis and risk stratification of pediatric neuroblastic tumors.
              </a>
            </td>
          <td>
            Ting Tao, Jiabin Cai, Yinbing Tang, Dong-Fang Lu, Lifeng Zhang, Jinkai Peng, Yilong Wang, Weizhong Gu, Shouhua Zhang, Jinhu Wang
          </td>
          <td>2025-12-03</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0bf2ebc24868e52a4bf9e9e5a2567d97f5b5a0" target='_blank'>
              Integrated proteogenomics uncovers ancestry-specific and shared molecular drivers in localized prostate cancer
              </a>
            </td>
          <td>
            Cara C. Schafer, T. Abulez, Xijun Zhang, Kun-Lin Ho, Jiji Jiang, D. Young, Jesse M. Fox, K. Conrads, B. Hood, G. Sukumar, Darryl Nousome, Praveen-Kumar Raj-Kumar, Mariano Russo, Ayesha A. Shafi, Xiaofeng A. Su, A. Dobi, Amina Ali, S. Elsamanoudi, J. Cullen, W. Figg, G. Petrovics, C. Dalgard, M. D. Wilkerson, N. Bateman, T. Conrads, I. Sesterhenn, Leigh Ellis, Craig D. Shriver, Gregory T. Chesnut, Shyh-Han Tan
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Retroviruses are exclusive group of positive-sense RNA viruses defined by their ability to reverse transcribe their RNA genome and integrate it into the host's chromosomal DNA. This distinctive replication strategy enables persistent infection and has profoundly shaped our understanding of molecular biology, gene regulation, and evolution. Retroviruses have contributed to landmark discoveries, including the identification of oncogenes, mechanisms of transcriptional control, and the development of gene therapy vectors. This review provides an updated overview of retroviral molecular biology, emphasizing the coordinated steps of the viral life cycle and emerging insights that are reshaping classical models. It explores virion structure, genome organization, and the interplay of cis-acting sequences and trans-acting factors that govern replication. Special focus is given to recent advances in understanding nuclear trafficking of capsids, spatial dynamics of reverse transcription and integration leading to provirus formation, RNA nuclear export, and selective genome packaging. The structural and functional roles of viral proteins, particularly Gag, are discussed in the context of assembly and maturation. By integrating foundational concepts with new discoveries, this review highlights the molecular sophistication of retroviral replication and identifies outstanding questions that guide future research, with implications extending to antiviral strategies, gene therapy, cancer biology, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2158d0cf0754098c7826e6edf394a24a8aa054" target='_blank'>
              Beyond Reverse Transcription: Molecular Mechanisms and Emerging Paradigms in Retroviral Replication.
              </a>
            </td>
          <td>
            Mohammad Abdullah Jehad, L. Ali, Vineeta N Pillai, Suresha G. Prabhu, Farah Mustafa, Tahir A. Rizvi
          </td>
          <td>2025-12-26</td>
          <td>FEMS microbiology reviews</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The T‐cell Leukemia Homeobox 1 (TLX1) oncogene is frequently deregulated in T‐cell acute lymphoblastic leukemia (T‐ALL) and T‐cell lymphoblastic lymphoma (T‐LBL). In most instances, TLX1 overexpression results from its juxtaposition with a T‐cell receptor (TCR) locus caused by interchromosomal translocations. However, in the subset of non‐TCR‐translocated TLX1‐positive (non‐TCR TLX1+) T‐ALL cases, the underlying mechanisms driving TLX1 overexpression and their clinico‐biological implications remain unclear. By integrating high‐resolution data from next‐generation sequencing, fluorescence in situ hybridization, karyotyping, optical genome mapping, and long‐read whole‐genome sequencing across a cohort of 1122 adult and pediatric T‐ALL/T‐LBL cases, we identified TLX1 overexpression in 11% of T‐ALL/T‐LBL cases, with an unexpected 14% incidence of non‐TCR TLX1+ cases among TLX1+ cases. Non‐TCR TLX1 + T‐ALL cases show distinct clinico‐biological features, including a lower frequency of cortical phenotype compared to the TCR‐translocated TLX1+ (TCR TLX1+) subgroup, regardless of TLX1 expression levels. In non‐TCR TLX1⁺ cases, TLX1 deregulation results mostly (83%) from the hijacking of the ANKRD1 and PCGF5 enhancer at 10q23.31 through various chromosomal aberrations. Genomic analyses revealed that these aberrations juxtapose the ANKRD1 and PCGF5 enhancer with TLX1, driving its overexpression. Importantly, survival analysis revealed that non‐TCR TLX1+ patients had significantly poorer outcomes compared to their TCR TLX1⁺ counterparts (3y‐OS: 55% vs. 90%, P < 0.001 and 3y‐DFS: 45% vs. 75%, P = 0.02). These findings, consistent with our previous observations in TAL1‐driven T‐ALL, confirm that the clinico‐biological impact of an oncogene is influenced by the specific mechanism underlying its deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd726bac8a093d55c346f606656874776ef2007c" target='_blank'>
              TLX1 translocation variants and enhancer hijacking influence clinical outcome in T‐cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            E. Balducci, T. Steimlé, A. Cieslak, G. Charbonnier, A. Pinton, M. Simonin, Charlotte Smith, Valentine Louis, M. Féroul, Manon Delafoy, Sophie Kaltenbach, Audrey Bidet, Béatrice Grange, Lauren Rigollet, P. Villarese, A. Touzart, Hervé Dombret, André Baruchel, N. Boissel, Vahid Asnafi
          </td>
          <td>2025-12-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffefccab81cd9b2f6152654f3b31bd309e658b64" target='_blank'>
              HDAC1 has a role in double strand break repair by regulating γH2AX signalling
              </a>
            </td>
          <td>
            José Javier Marqueta-Gracia, Marina Bejarano-Franco, Salome Spaag, Sónia Silva, A. Aguilera, Belén Gómez‐González
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel C. de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="High-grade gliomas are one of the most lethal cancers and the leading cause of cancer-related mortality in children. Standard treatment typically involves resection, chemotherapy and radiation, yet offer limited improvement in survival rates. Targeted therapies are emerging to show effect in solid tumors harboring tyrosine-kinase activating fusion genes. Here, we present a thorough characterization of an epithelioid glioblastoma in a six-year-old patient, with four relapses and a variety of different treatment strategies. We identified a rare TRIM24::NTRK2 fusion in the primary tumor, which enabled a targeted TRK-inhibitor treatment. Our findings show that the TRIM24::NTRK2 fusion initially had oncogenic abilities, but this became less imperative as the tumor evolved. A general drug resistance to TRK-inhibition was documented. This study addresses the complex and adaptive nature of pediatric epithelioid glioblastomas and highlights the need for continued molecular profiling from relapses and various tumor regions to enable multitarget treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e83f96185edbab3b833499e7bdcbd141d1efe42" target='_blank'>
              Comprehensive characterization and targeted treatment of a pediatric epithelioid glioblastoma with a rare TRIM24-NTRK2 fusion
              </a>
            </td>
          <td>
            S. Keane, Ida Sjöberg, L. Deland, A. Lindlöf, T. O. Bontell, T. Kling, M. Blomstrand, H. Fagman, J. Nilsson, D. Nilsson, M. Sabel, H. Carén, F. Abel
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous single-nucleotide polymorphisms (SNPs) associated with various diseases, including cancer. However, the mechanisms by which these SNPs contribute to disease susceptibility remain largely unclear. While recent studies have explored the transcriptional impact of disease-associated SNPs, their role in post-transcriptional regulation has been less extensively investigated. In this study, we investigated whether cancer-associated SNPs influence gene expression by altering N6-methyladenosine (m6A) RNA methylation. We collected GWAS-identified SNPs across nine cancer types and integrated these with matched tumor and normal m6A RNA immunoprecipitation sequencing (m6A-seq) and RNA sequencing (RNA-seq) datasets. We first identified differentially methylated m6A sites and assessed whether cancer-associated SNPs were enriched within these regions. These analyses revealed that cancer-associated SNPs were significantly enriched within hypermethylated m6A regions in colon cancer. Integrative analysis revealed that SNPs enriched in m6A-modified regions are associated with altered gene expression and RNA splicing, suggesting that m6A methylation mediates the post-transcriptional impact of genetic variants. Experimental validation further confirmed altered gene expression following ALKBH5 knockdown, consistent with patient-derived data. Collectively, our findings support a novel mechanistic connection between genetic variants and RNA methylation-driven transcriptomic regulation in colorectal cancer, underscoring the epitranscriptome as a potential axis of oncogenic control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8030aadad77b99c149edba7083659d634eaa70" target='_blank'>
              Genetic variants reshape the m6A epitranscriptome and drive transcriptomic reprogramming in colorectal cancer
              </a>
            </td>
          <td>
            Seung Hun Han, Seongmin Jang, Yeongwon Kim, Kun Tan, M. F. Wilkinson, Hyobin Jeong, Junho Choe
          </td>
          <td>2025-11-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA) copy number (CN) varies widely both across and within species. In humans, it has been hypothesized that rDNA transcriptional activity is partially regulated by rDNA methylation status. Here we describe a rapid and scalable approach for the simultaneous detection of methylated and unmethylated rDNA copy number using a well characterized digital PCR technology. The linear range for detecting rDNA CN and methylation is from 0.3 ng to 20 ng DNA with a practical range from 1.25 to 15 ng DNA and can be normalized with a single copy gene, ACTB, to reflect copies per genome. Our approach detects 28S rDNA methylation using a modified methyl-specific/bisulfite PCR targeting a region with at least four CpGs. In a population of healthy donors, we observed wide variation in both methylated and unmethylated rDNA CN, as well as in the percentage of methylated rDNA copies. We found significant correlations between the total rDNA CN and both methylated rDNA CN and percent methylated rDNA CN. Individuals with a greater number of rDNA copies had more methylated copies and a higher percentage of methylated copies. The methylated rDNA CN percentage varied widely across individuals and ranged from 9% to 69% of the total rDNA copies being methylated. Finally, four human cell lines (BEAS-2B, HeLa, MCF-7, MDA-MB-231) also displayed wide differences in methylated and unmethylated rDNA CN. Overall, this method provides a new tool to study the relationship between rDNA CN and methylation while offering insight into rDNA methylation variability and the regulation of active rDNA copies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fca978e97cfbb455b43e385f1afc7beb553557e7" target='_blank'>
              Simultaneous quantification of rDNA methylation and copy number: Constraints to natural variation in humans and cell lines
              </a>
            </td>
          <td>
            Stephen Flint Smith, Bernardo Lemos
          </td>
          <td>2025-11-13</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ECT2 gene, encoding a guanine nucleotide exchange factor, plays crucial roles in cell cycle progression and cytoskeletal dynamics, implicating its involvement in various cancers. However, a comprehensive pan-cancer analysis integrating genomic data is still lacking. This study employed an integrated approach using data from multiple cancer genomics databases to assess ECT2 across various malignancies. Expression profiles of ECT2 were analyzed for differential expression across tumor stages and its association with clinical outcomes. Correlation analyses examined the relationship between ECT2 expression and immune cell infiltration levels. Pathway enrichment analysis identified biological processes influenced by ECT2 dysregulation in cancer progression. These methods facilitated a comprehensive exploration of ECT2’s role in cancer biology, revealing potential implications for diagnosis, prognosis, and therapy. Analysis across 33 tumor types consistently shows elevated ECT2 expression. ECT2 correlates with tumor staging in 8 cancers and molecular subtypes in 13 cancers, and shows associations with immune subtypes in 22 cancers, suggesting its role in cancer progression and immune modulation. ECT2 demonstrates strong diagnostic potential (the area under the ROC curve > 0.9) in 16 cancers and correlates with poorer overall survival in 11 cancers. ECT2 expression correlates positively with microsatellite instability in STAD, MESO, UCEC, and READ, and negatively in DLBC; it correlates positively with tumor mutational burden in STAD, PAAD, ACC, LGG, and LUAD, and negatively in THYM. ECT2 also exhibits diverse correlations with immune checkpoint genes and specific immune cell types identified through CIBERSORT analysis. ECT2 interacts with proteins like RACGAP1, KIF23, enriched in pathways involving cell polarity, Ras signaling, and tight junctions, impacting cancer progression and stemness in various cancer types. This study offers comprehensive insights into ECT2’s role in cancer biology through integrative bioinformatics analyses. The results advocate for ECT2 as a potential biomarker and therapeutic target in diverse malignancies, suggesting avenues for personalized oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc5e11d8db9ee77a83f99f490a898c98d9b4538" target='_blank'>
              Computational identification of ECT2 as a potential pan-cancer biomarker and therapeutic target through integrated genomic data analysis
              </a>
            </td>
          <td>
            Ming-Hui Chen, Li Wang, Guodong Song
          </td>
          <td>2025-11-07</td>
          <td>Medicine</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90cf835255eacd329046c889367761e186d358b" target='_blank'>
              Cohesin forms fountains at active enhancers in C. elegans.
              </a>
            </td>
          <td>
            Bolaji N. Lüthi, Jennifer I Semple, Anja Haemmerli, Saurabh Thapliyal, Kalyan Ghadage, Klement Stojanovski, Dario D’Asaro, Moushumi Das, Nick Gilbert, Dominique A. Glauser, Benjamin D. Towbin, Daniel Jost, Peter Meister
          </td>
          <td>2025-12-11</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0453b8fd3ee789a5730a5cb95cb52509da58c9" target='_blank'>
              Unraveling miRNA-Driven DNA Damage Response Networks in Pancreatic Adenocarcinoma: A Multi-Omics and Machine Learning Approach
              </a>
            </td>
          <td>
            Shayori Bose, Paramita Basak Upama, S. Ahamed
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Liquid-Liquid Phase Separation (LLPS) allows biomolecules to form membrane-less organelles or condensates through weak multivalent interactions. Recent studies have revealed the critical role of LLPS in super-enhancer (SE)-associated tumorigenesis, progression, and immune evasion. This mini-review summarizes recent advances in the role of LLPS in SE-driven oncogenic and immune evasion processes, while discussing its potential therapeutic implications in cancer. Current evidence indicates that LLPS regulates the expression of tumor-associated genes through multiple mechanisms, such as facilitating transcription factor recruitment, promoting chromatin looping, modulating chromatin topology, and maintaining chromatin accessibility. Notably, LLPS-associated SEs functionally regulate not only tumor cells but also immune cells within the tumor microenvironment, contributing to tumor immune evasion. Thus, understanding the relationship between LLPS and SEs is crucial for elucidating the mechanisms underlying tumor initiation and progression. These insights may provide foundational evidence for developing novel anti-tumor therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea7f753405e25c1a9b559df0088bd9407d452584" target='_blank'>
              Liquid-liquid phase separation in super enhancer-driven oncogenesis: mechanisms, immune evasion, and therapeutic implications
              </a>
            </td>
          <td>
            Ruoxi Yu, Deyu Sun, Chuang Zhang
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogene-targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance. Therapeutic approaches to treat tumours that have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms. Rather than treating resistance after it emerges, it may be possible to prevent it by inhibiting the adaptive processes that initiate resistance, but these are poorly understood. Here we report that residual cancer persister cells that survive oncogene-targeted therapy are growth arrested by drug stress-induced intrinsic type I interferon signalling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA fragmentation factor B (also known as caspase-activated DNase), which induces DNA damage, mutagenesis and stress response factor activating transcription factor 3 (ATF3). ATF3 limits activator protein 1-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DNA fragmentation factor B or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes the regrowth of residual cancer cells that survive drug treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0eb23b0262f48e292a21b5682ac6f6faf0a4d24" target='_blank'>
              DNA fragmentation factor B suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F. Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X Wang, B.E. Mauch, Rhea Plawat, Ariel H. Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J. Hangauer
          </td>
          <td>2025-11-17</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma.
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Brain cancers are the most common solid tumors in children and adolescents and remain the leading cause of disease-related death in these populations. Despite their clinical importance, therapeutic progress has been limited by extensive intra- and inter-tumor heterogeneity. In particular, the clonal architecture and evolutionary dynamics of pediatric brain tumors remain poorly understood. Addressing this knowledge gap is critical, as genetically distinct subclones can shape disease progression, treatment resistance, and clinical outcome. To investigate clonal evolution at the population level, we analyzed longitudinal whole-genome sequencing data from 1,742 tumors across 27 brain cancer types in 1,479 pediatric patients from the Children’s Brain Tumor Network. We inferred clonal composition and phylogenetic relationships for each patient and reconstructed the space of tumor phylogenies for the entire cohort using a regularized Gromov-Hausdorff distance. Tumor phylogenetic structure was significantly associated with mutational burden, mutational signatures, cancer classification, and patient's survival status. In longitudinal samples, survival status remained significantly associated with tumor clonal architecture after adjusting for mutational burden, tumor subtype, and anatomical location, underscoring the clinical value of this readout. In contrast, in primary tumors, mutational signatures remained significant after adjusting for mutation count, tumor subtype, anatomical location, and purity, while survival status associations were no longer significant. Mutational signature timing analysis revealed that pediatric brain tumor genetic architecture is shaped by distinct mutational processes acting at different stages of tumor evolution. Truncal somatic mutations were enriched for age-related, defective proofreading, and homologous recombination deficiency signatures, whereas subclonal mutations showed significant contributions from DNA mismatch repair, chemotherapy-related, temozolomide-related, and somatic hypermutation activity signatures. Comparisons of matched longitudinal and primary phylogenetic trees further revealed that deceased patients and those with recurrent or metastatic disease exhibited significantly greater evolutionary divergence, suggesting that primary tumors are less predictive of complete evolutionary trajectories in poor-outcome cases. Together, these findings suggest that pediatric brain tumor evolution is shaped by cancer type and various mutational processes, and that patients with adverse outcomes follow distinct evolutionary trajectories independently of subtype. Our study provides a population-level view of tumor evolution in pediatric brain cancers and highlights the value of genetic clonal dynamics for understanding tumor etiology and progression.



 Minh A. Nguyen, Pablo G. Cámara. Mapping clonal architecture and evolution in pediatric brain cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d01cdf935c0ff4316725768de123e5059b9b4ae1" target='_blank'>
              Abstract B008: Mapping clonal architecture and evolution in pediatric brain cancers
              </a>
            </td>
          <td>
            Minh A. Nguyen, Pablo G. Cámara
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is critical for genetic stability. DSB repair acts according to two steps: (1) the choice between nonhomologous end-joining (NHEJ) and the resection of DNA ends, generating single-stranded DNA (ssDNA) and (2) on generated ssDNA, homologous recombination (HR) and sub-processes. Here, we show that 53BP1, which plays a prime role in the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of cAMP-dependent protein kinase A (PKAcs). PKA favours the recruitment of 53BP1 at DNA damage sites and consistently prevents resection, favouring NHEJ. Inhibition of PKA stimulated resection and concurrently reduced NHEJ. Conversely, the activation of PKA with 8-Bromo-cAMP stimulated NHEJ and reduced HR. These data provide new avenues for potential anticancer strategies. More generally, these findings underscore the high complexity of the regulation of DSB repair, identifying PKA as a novel participant in the DNA damage response that acts in the DSB repair process, which may be essential for maintaining genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff78c19cf3ac765f9d790f02f928cce37461ed5" target='_blank'>
              PKA prevents the resection of DNA double-strand breaks and favours nonhomologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, E. Dardillac, Elise Rabut, I. Magdalou, Vasily Ogryzko, J. Guirouilh-Barbat, Bernard S. Lopez
          </td>
          <td>2025-11-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Receptor tyrosine kinase (RTK) fusions are a large class of oncoproteins found in ∼5% of cancers. Key questions remain, however, about how RTK fusions transmit oncogenic signals, including how these largely cytoplasmic proteins activate downstream pathways that originate at the plasma membrane. Fusions are multimeric and can form mesoscale condensates in cancer cells, and condensation has been implicated as an essential mechanism to enable signal transmission from the cytoplasm. However, whether condensates play a causal role, or whether smaller ‘diffuse’ assemblies are sufficient to transduce signals, has been challenging to establish. Here we apply advanced microscopy, single-cell analysis, and synthetic fusions to determine the principles by which multimerization and condensation drive signaling from cytoplasmic RTK fusions. For EML4-ALK, a prominent fusion that forms condensates, we found poor correlation between condensation and signaling. By contrast, EML4-ALK activity was abundant in the diffuse phase, and the kinetics of diffuse-phase activity aligned more closely with downstream Erk signaling than did kinetics of signaling within condensates. Synthetic RTK fusions showed that cytoplasmic ALK or RET fusion dimers—and even constitutively active monomers—were sufficient to induce strong Ras-Erk signaling despite the absence of condensates, and diffuse fusions were sufficient to transform cells in vitro and in subcutaneous tumor models. A panel of various other cancer-driving RTK fusions showed that low-order multimerization was universal across fusions, whereas mesoscale condensation was rare and did not correlate with signaling. Our results suggest that low-order fusion multimerization is sufficient to drive its phosphorylation, which is necessary and sufficient to trigger downstream oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6563f441f99dac16cec7dceb7a7c5f130e146e" target='_blank'>
              Low-order assemblies drive oncogenic RTK fusion signaling without condensation
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Thomas R. Mumford, Delaney Wilde, Sofia Wissert, Yuzhi (Carol) Gao, Emily Brackhahn, R. Kriwacki, Elizabeth Rhoades, Lukasz J. Bugaj
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fde5c67c0f8ab3015db102bca2f37f2904ea7f9" target='_blank'>
              Role of Myc family proteins in transcriptional regulation of growth and oncogenic transformation in fusion-positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bishwanath Chatterjee, P. K. Raut, Hana Kim, Rachel A. Hoffman, P. Pandey, Shabir Zargar, Wenyue Sun, Benjamin Z. Stanton, Frederic G Barr
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Gene expression and regulation with or without alternative splicing are key factors for cells to properly function. Distant splicing quantitative trait loci (distant sQTLs) are genomic mutations that impact the alternative splicing patterns of far-away genes. Nevertheless, the procedures causing a distant sQTL to regulate the alternative splicing of genes are not well defined. Higher resolution chromosome conformation capture experiments like Micro-C or Hi-C together with an expanding number of sQTL datasets on humans help us in understanding the spatial processes governing distant sQTL relationships at a genome-wide scale. In this study, we focus on analyzing whether spatial closeness helps in regulating sQTL-gene interactions over high-order chromatin topological domain structure, which is inferred from chromosome conformation experiments. We discover larger-scale chromatin shape to be in line with sQTL associations. In detail, sQTLs are generally spatially near their splicing genes in 3D, they frequently appear near topologically associating domain (TAD) and frequently interacting region (FIRE) boundaries, and are favorably related to genes over TADs and FIREs. Additionally, we discover that inside-domain sQTLs accompanied by functional regulatory elements, including enhancers and promoters, are spatially closer than all inside-domain sQTLs. This result suggests that spatial closeness between sQTLs and their distant splicing genes obtained from chromatin’s TAD structure has major importance in regulating alternative splicing and thus in gene regulation. Our results are robust across different experiments such as Hi-C and Micro-C, different TAD inference methods, different Hi-C binning resolutions, different alternative splicing events, and once we control for eQTLs, which are shown to be spatially close to their genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3f85319df6c80cf2868c0242d3d5eb8c32dec3b" target='_blank'>
              Analysis of chromatin structure reveals the connection between sQTLs and the splicing of distant genes
              </a>
            </td>
          <td>
            Batuhan Eralp, Emre Sefer
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11–RAD50–NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Bioscience-Scholar</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin plays a critical role in regulating gene expression and genomic processes like DNA replication, repair, and genome stability. Although these processes occur at the individual-cell level, most chromatin structure data are derived from population-averaged assays, such as Hi-C, obscuring the heterogeneity of single-cell conformations. To address this limitation, we developed a polymer physics-based modeling framework, the sequential Bayesian inference framework (sBIF), that deconvolutes bulk Hi-C data to reconstruct single-cell 3D chromatin conformations. To support a broader use of sBIF, we created ChromPolymerDB, a publicly accessible, high-resolution database of single-cell chromatin structures inferred by sBIF. The database contains ∼108 reconstructed 5 kb-resolution single cell structures, spanning over 60,000 genomic loci across 50 human cell types and experimental conditions. ChromPolymerDB features an interactive web interface with tools for 3D structural analysis and multi-omics integration. Users can explore associations between chromatin conformation and gene expression, epigenetic modifications, and regulatory elements. The platform also supports comparative analyses to identify structural changes across cell types, developmental stages, or disease contexts. ChromPolymerDB offers a unique resource for researchers studying the relationship between genome architecture and gene regulation, and for advancing comparative 3D genomics. ChromPolymerDB is available online at https://chrompolymerdb.bme.uic.edu/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff3acfa75c8e6fa42bde5e8caa704c202f1fa88" target='_blank'>
              ChromPolymerDB: a high-resolution database of single-cell 3D chromatin structures for functional genomics.
              </a>
            </td>
          <td>
            Min Chen, Lin Du, Siyuan Zhao, Bowei Ye, Pourya Delafrouz, Hammad Farooq, Debaleena Chattopadhyay, G. Marai, Zhifeng Shao, Jie Liang, D. Czajkowsky, Constantinos Chronis
          </td>
          <td>2025-11-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00991a0395a2490dd6217e4ff4ad6e01917b30a6" target='_blank'>
              Menin inhibition impairs metastatic colonization of Ewing sarcoma
              </a>
            </td>
          <td>
            Katherine A. Braun, Nicolas M Garcia, Mohamed Ahmed, Darleen S Tu, Stephanie I. Walter, Emma D. Wrenn, Megan E.B. Dean, Elizabeth R. Lawlor
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416974525ed871e08b5c483505e950ab4b46e07f" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation.
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah Burget, Tobias Friedrich, So-Yoon Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-12-02</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC), an epigenetic modification derived from the oxidation of 5-methylcytosine (5mC) by the ten-eleven translocation (TET) family of dioxygenases, plays a pivotal role in the regulation of gene expression, cellular differentiation, and developmental plasticity. Once considered an intermediate in DNA demethylation, 5hmC is now recognized as a stable and functionally significant epigenetic mark with distinct genomic distributions and significant regulatory implications. This review provides a comprehensive analysis of the biological functions of 5hmC in normal cellular processes, including its role in maintaining tissue-specific gene expression, lineage commitment, and genomic integrity. We also describe its role in cancer, the mechanistic underpinnings of its loss or redistribution in tumor cells, and how these changes contribute to oncogenic signaling pathways, epithelial-mesenchymal transition, and tumor heterogeneity. Furthermore, we explore the utility of 5hmC as a biomarker in cancer diagnostics and prognostics, supported by recent advances in sequencing technologies and cell-free DNA profiling. We also examine the intersection of 5hmC and chemotherapy, highlighting how aberrant 5hmC levels can influence drug resistance and sensitivity, and assess the therapeutic potential of targeting TET enzymes and associated pathways. By integrating insights from basic epigenetics, cancer biology, and therapeutic research, this review underscores the multifaceted role of 5hmC in human malignancies and outlines the translational opportunities for exploiting 5hmC-related mechanisms in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3164793ee65481d7edac6287a5b81d63211bc" target='_blank'>
              5-Hydroxymethylcytosine: a key epigenetic mark in cancer and chemotherapy response
              </a>
            </td>
          <td>
            S. S. Kharat, Shyam K. Sharan
          </td>
          <td>2025-11-17</td>
          <td>Epigenetics & Chromatin</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, Thomas F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03533d5a535f8136a5f4c27e74c671f6ec467639" target='_blank'>
              cGAS restricts PARP1-mediated microhomology-mediated end joining by suppressing poly-ADP-ribosylation.
              </a>
            </td>
          <td>
            Haibo Yang, Boya Gao, Bo Wu, Wenjing Li, Xueping Zhu, Laiyee Phoon, Jing Luo, Fu Gui, Weixing Zhao, Li Jia, Li Lan
          </td>
          <td>2025-12-17</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceca0fc9a67ea509cda7d2d2cbdefbe5c21b1e0b" target='_blank'>
              Evidence for Transcription and Horizontal Gene Transfer in Dipteran Germline-Restricted Chromosomes
              </a>
            </td>
          <td>
            Riccardo G. Kyriacou, M. Herbette, Robert B. Baird, Katy M. Monteith, Laura Ross, Christina N. Hodson
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, Allen L Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Eukaryotic DNA has been covalently modified by DNA methylation and folded into a three-dimensional conformation in the nucleus. While the functions of DNA methylation and chromosome organization have been widely discussed, respectively, the interplay between DNA methylation and chromosome organization remains less clear and needs to be further explained. In this review, we first discuss the cross-talk between DNA methylation and chromosome conformation, highlighting the complexity and importance of DNA methylation on chromosome organization. We also summarize the current methodologies that capture DNA methylation and chromosome organization individually or simultaneously in bulk and single cells. These mechanistic and methodological advancements facilitate broad interest in unveiling the interplay between DNA methylation and chromosome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1272377f23956ec52130bb31428bff7433f3f0" target='_blank'>
              Unraveling the interplay of DNA methylation and chromosome organization
              </a>
            </td>
          <td>
            Yuhe Pei, Guoqiang Li
          </td>
          <td>2025-11-10</td>
          <td>Biochemical Society Transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) are often epigenetically repressed in eukaryotic cells, but still affect the molecular state of the cell in certain contexts. A flurry of recent studies have elucidated new effects of TE sequences in eukaryotic cells. We review these emerging molecular effects of TEs, including a variety of new mechanisms by which TE sequences affect the cell, including pre- and post-transcriptional regulation of gene expression; cell-to-cell transmission of genes within a multicellular organism through virus-like activity; and RNA-guided DNA insertion. Recent demonstration of TE-guided genome editing underscores the importance of these investigations for both basic and translational research. Future work is needed to continue to unravel the molecular effects of TE sequences across developmental stages, across cell types, and in various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3445e991f8e7a3fbd17d3feddadbe0f7c8e2d56" target='_blank'>
              Molecular effects of transposable element sequences in mammalian cells
              </a>
            </td>
          <td>
            Ming-Ching C. Wen, Joshua D. Welch
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh R. Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, G. Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Spindle cell lipomas (SCL) and pleomorphic lipomas (PL) are today considered a single tumor entity (SCLPL). Atypical SCLPL, in contrast, represents a recently recognized related but distinct entity. Here we explored the correlation between genomic features and morphological aspects of SCLPL and atypical SCLPL and the role of the RB1 gene in tumor development. Seventy‐one samples from 68 patients with SCLPL or atypical SCLPL, as well as a lipomatous tumor from a retinoblastoma patient with a germline pathogenic variant in the RB1 gene, and two pleomorphic liposarcomas were analyzed using chromosome banding, high‐resolution genomic arrays (SNP array), whole exome sequencing (WES), and/or RNA sequencing (RNA‐seq). Common for all tumors was involvement of 13q; other recurring variants were deletion of 16q, 6q, and 17p. A minimally deleted region that only contained RB1 was found on 13q. A distinction was seen between SCL on the one hand and PL and atypical SCLPL on the other; SCL had fewer copy number aberrations in general, and loss of 17p/TP53 gene or a SNV affecting TP53 was only rarely detected in SCL but seen in the vast majority of PL and atypical SCLPL tumors. Thus, at the molecular level, SCL is different from PL/atypical SCLPL. Furthermore, the finding of the same copy number changes (loss of 13q, 6q, 16q, and 17p) in some pleomorphic liposarcomas raises the possibility that a subset of SCLPL/atypical SCLPL have the potential for malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b1988ef8c66c3a7438406d89e232e1e91d6e09" target='_blank'>
              The Role of RB1 and Secondary Genomic Changes in the Development of Spindle Cell and Pleomorphic Lipomas
              </a>
            </td>
          <td>
            Maria Hellberg, V. Difilippo, Emilia Gottberg, A. Hesla, Felix Haglund de Flon, J. Nilsson, L. Magnusson, Saskia Sydow, Paul Piccinelli, Jakob Hofvander, Fredrik Mertens
          </td>
          <td>2025-12-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="AIMS
Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease.


MATERIALS AND METHODS
Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines.


RESULTS
We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression.


CONCLUSIONS
We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome.
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="The human papillomavirus (HPV) is a small, non-enveloped virus with a circular double-stranded DNA genome. The HPV genome encodes the E2 activator protein, which is required for viral transcription. R-loops are triple-stranded nucleic acid structures that occur when newly synthesized single-stranded RNA anneals to duplex DNA. These structures form during papillomavirus transcription. We and others have demonstrated that resolution of viral R loops is crucial for HPV episomal maintenance. ZPR1 is a zinc finger protein that can recruit SETX to mammalian R-loops to mediate resolution. E2 binds to and recruits SETX, an R-loop helicase, to the viral promoter. We observed E2 in complex with SETX and ZPR1. However, we found that ZPR1 depletion decreased viral R-loops while enhancing cellular R-loops. ZPR1 depletion also increased SETX binding to the viral promoter. These data suggest that ZPR1 is not required for HPV R-loop resolution, in contrast to what has been observed in mammalian cells. We detected the E2 protein associated with R-loops and found that E2 overexpression increases cell-derived R-loop formation. Analysis of TCGA datasets revealed that ZPR1, but not SETX, mRNA expression is significantly reduced in HPV-positive cervical and head and neck cancers. Together, these findings indicate that while E2 mediates HPV R-loop resolution, it also promotes R-loop accumulation in the host genome, likely through SETX sequestration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f66b5cc6cbe8cc615469465c98084907e962c34" target='_blank'>
              ZPR1 Is Dispensable for HPV R-Loop Resolution but Regulates Host R-Loop Dynamics
              </a>
            </td>
          <td>
            Rylann Moffitt, Steven Brooks, EJ Androphy, M. DeSmet
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Prostate cancer (PCa) is one of the most frequently diagnosed cancers in men. PCa growth is dependent on the androgen receptor (AR) which regulates transcription directly as well as through interactions with other transcription factors (TFs). While androgen deprivation therapy is often successful upon progression, a fraction of PCa cases possess constitutively active AR that functions independently of hormonal regulation. Understanding the TF regulatory landscape, especially androgen-dependent versus androgen-independent PCa, is critical to developing better therapeutic strategies. Here we investigated the epigenomic landscape of two prostate cancer cell lines, VCaP (androgen-dependent) and 22Rv1 (androgen-independent), using ATAC-seq to map chromatin accessibility and a high-resolution version of ChIP-seq (ChIP-exo) that characterizes binding of proteins to the genome at base-pair resolution. We identified distinct AR binding between cell lines in response to androgen-treatment, with a subset of AR sites showing strong enhancer enrichment and preferential colocalization with FoxA1 at characteristic spacing, consistent with cooperative AR–FoxA1 regulation. Interestingly, many FoxA1 binding events occurred independently of AR and were enriched in quiescent chromatin, suggesting a context-dependent role for FoxA1. Together, these findings highlight how differential AR–FoxA1 crosstalk and chromatin states contribute to hormone sensitivity versus resistance in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17226d24f8c1296f5c00f2bdc188a6c8cf5c1161" target='_blank'>
              Genome-wide analysis of transcription factors in prostate cancer
              </a>
            </td>
          <td>
            Zimo Zhu, Anne M Gardner, David Wilkes, Alicia Alonso, Cora N Sternberg, J. Mosquera, William KM Lai, B. F. Pugh
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B. Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Argonaute proteins provide innate immunity in all domains of life through guide-dependent recognition and cleavage of invader nucleic acids. Many short prokaryotic Argonautes (pAgos) lack nuclease activity and are instead co-encoded with tentative nuclease effectors, but their activation mechanisms remain unknown. Here, we characterize SPARHA systems (short prokaryotic argonautes, HNH-associated), containing HNH nuclease effectors. RNA-guided target DNA recognition by pAgo induces formation of SPARHA filaments with a chain of double nuclease sites formed by HNH tetramers, leading to indiscriminate collateral degradation of DNA and protecting bacterial population from invaders. We show that the assembly of filaments proceeds via a universal activation pathway involving a cascade of target-induced conformational changes in SPARHA, conserved in other short pAgo systems containing various types of effectors. pAgos and associated effectors act as modular defense systems that translate recognition of specific DNA into immune response through assembly of supramolecular complexes, deleterious for invaders and potentially useful for biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121cd79715591f5d74c4c48c7a84d96dfd056de" target='_blank'>
              Argonaute-HNH filaments triggered by invader DNA confer bacterial immunity
              </a>
            </td>
          <td>
            A. Kanevskaya, M. Narwal, L. Lisitskaya, A. Moiseenko, O. S. Sokolova, N. N. Sluchanko, Katsuhiko S. Murakami, A. Kulbachinskiy
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999aa4e6a47ef459adcf0a9e608c45b2e8af7576" target='_blank'>
              A human supergene: BRCA1, CCDC200, and U2 snRNAs
              </a>
            </td>
          <td>
            Pelle Scholten, Michael Segel, David Haig
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Retinoblastoma is the most common intraocular malignancy of childhood, yet its genomic landscape remains incompletely defined, particularly in understudied populations. Beyond RB1 loss, the contribution of additional somatic and germline alterations to disease heterogeneity and clinical behavior is unclear.



 We performed whole-exome sequencing of 166 retinoblastoma samples from 166 patients with matched germline DNA, representing the largest cohort analyzed to date. Clinical data were available for 160 patients. Variant calling, copy number alteration (CNA) profiling, and integrative analyses were performed to characterize genetic drivers and their associations with clinical features.



 Pathogenic RB1 variants were identified in 120 patients, and MYCN amplification in six patients. Additional recurrent alterations involved BCOR, CCND3, ERBB2, and PDGFRB. Copy number gains of 6p (41.3%) and 17q (8.1%) were significantly associated with high-risk features including rubeosis, subretinal seeding, and tumor extension beyond the lamina cribrosa. Germline ERBB2 variants correlated with orbital invasion, while germline PDGFRB variants were associated with second primary cancers. Together, these findings underscore the genetic heterogeneity of retinoblastoma and reveal novel genotype–phenotype correlations.



 This study provides the most comprehensive genomic characterization of retinoblastoma to date, expands the known mutational spectrum, and identifies biomarkers with direct clinical relevance. These insights have potential to refine risk stratification, inform precision therapeutic strategies, and improve long-term outcomes for children with retinoblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca1a82a307761a6281f02514b112932e6b6ae23" target='_blank'>
              Genomic landscape of retinoblastoma: Insights into risk stratification and precision pediatric Neuro-Oncology
              </a>
            </td>
          <td>
            A. Maktabi, Yang Liu, Saleh A. Almesfer, M. Abdelhakim, Hind Aldakhil, Maxat Kulmanov, Deepak P Edward, R. Hoehndorf, M. Abedalthagafi
          </td>
          <td>2025-12-12</td>
          <td>Neuro-Oncology Pediatrics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kaposi sarcoma‐associated herpesvirus (KSHV), a gamma‐herpesvirus, is the main etiological agent of several tumors, including Kaposi's sarcoma (KS) and primary effusion lymphoma (PEL). The large, double‐stranded viral genome of KSHV is maintained as a latent episome in the nucleus of host cells, where a small subset of viral genes is expressed that facilitate evasion of immune responses and promotion of cell survival and proliferation supporting tumorigenesis. The major latency‐associated nuclear antigen (LANA) is an essential viral factor that is required for genome replication and segregation to maintain viral genomes within dividing cells. Given the essential role of LANA to maintain viral latency, efforts have focused on targeting LANA's role in replication and segregation as a mechanism to overcome latency and tumorigenesis. However, given the complexity in small molecule targeting to DNA‐binding domains, current efforts focused on drugging LANA continue to reach potency milestones for clinical trials. Here, we developed a HaloTag‐based PROTAC model as a proof‐of‐concept for targeted LANA protein degradation. Our findings highlight the power of a PROTAC‐based strategy on eliminating LANA and viral persistence in tissue culture demonstrating promise in the on‐going challenge for targeting this major viral factor in KSHV latency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfa79b63dd2067eec7b314131de284f9e1fe9922" target='_blank'>
              Targeted Protein Degradation of the Latency‐Associated Nuclear Antigen Evicts Kaposi Sarcoma‐Associated Herpesvirus Episomes From Infected Cells In Vitro
              </a>
            </td>
          <td>
            Sarah McMahon, Maha Amer, Brooke Hall, Ritu Shekhar, Rolf Renne
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints.
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="

 Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer.




 This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6
 low
 (negative/weak) or SMC6
 high
 (moderate/strong).





 SMC6
 low
 tumours tend to show higher CNV. SMC6
 high
 group presented poorer disease-free survival than the SMC6
 low
 group (
 P
 =0.050), mainly for the luminal subtype (
 P
 =0.005). SMC6
 low
 /ER
 pos
 were protective biomarkers for recurrence.




 There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-12-31</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d7d2e338e9c2ff093882c73a51e612889ed2af" target='_blank'>
              Histone H1 Promotes Silencing of Unintegrated HIV-1 DNA
              </a>
            </td>
          <td>
            Jiaqi Zhu, Gary Z. Wang, Hugo D. Pinto, Sean E. Healton, Arthur I Skoultchi, Stephen P. Goff
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Turner syndrome (TS) arises from partial or complete loss of a sex chromosome, yet the mechanistic basis for Y chromosome loss (LoY), which may contribute to a subset of TS cases, remains unclear. This article addresses the existing gap in knowledge by proposing a hypothesis linking a transient physiological window of elevated HSATII RNA levels during preimplantation embryogenesis to recent bioinformatic predictions indicating that Y-linked HSATII arrays possess uniquely high triplex-forming propensity. In this context, HSATII-derived RNAs could form RNA-DNA triplexes in early embryogenesis preferentially at Y-linked HSATII tracts. If unresolved, these structures may stall replication forks and promote Y chromosome instability which may ultimately lead to complete or partial LoY. The proposed model reframes part of the TS etiology from a stochastic segregation error toward a definable process, and motivates experimental validation of its predictions. If supported by experimental evidence, this framework could further guide the search for modifying factors - such as interindividual variation in the Y-linked HSATII sequence or triplex-resolution efficiency - and, in the longer term, enable risk stratification for Y chromosome instability in potential embryos based on parental molecular profiles. In a broader context, the hypothesis underscores pericentromeric satellite biology as a potentially underexplored contributor to genome stability in early human development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c46522e14c6e3a97b0b36fd21c4c86fd8062a686" target='_blank'>
              HSATII RNA-dependent triplex formation in early human embryogenesis as a potential mechanism for Y chromosome loss in Turner syndrome
              </a>
            </td>
          <td>
            Krystian Fularski
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5 A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Significance Cytotoxic and mutagenic DNA-damaging agents are ubiquitous. Bulky DNA damage is removed genome-wide in bacteria by excision repair, and from transcribed strands (TS) by transcription-coupled repair (TCR), in which the Mfd protein targets transcription-blocking damage for accelerated excision repair. In analyzing UV-induced mutations in Escherichia coli, genome-wide, we find fewer TS mutations in active genes consistent with removal of premutagenic damage by TCR. mfd− cells exhibit more mutations in the TS, reflecting inhibition of repair by RNA polymerase that remains stalled at template strand lesions. These effects are dependent upon transcription level and are meaningful in characterizing the mutagenic phenotype associated with Mfd, which has additional, incompletely characterized roles in recombination, gene regulation, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6740a28098ae2b4ef147f96b5ce5e313471a7a1" target='_blank'>
              Genome-wide strand-specific UV mutagenesis in Escherichia coli is directed by the Mfd translocase
              </a>
            </td>
          <td>
            Ogun Adebali, P. Mieczkowski, A. Sancar, C. Selby
          </td>
          <td>2025-11-13</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21b610301b7b55b77a33ac6317a40f28dcdb985" target='_blank'>
              A gastric microbial chromatin remodeler drives gastric cancer progression and immune evasion by reprogramming the host epigenome
              </a>
            </td>
          <td>
            Xiaoshan Xie, Yue Wei, Zhikai Zheng, Jiaying Zheng, Xijie Chen, Jiarui Wang, Ning Ma, Xiaoling Huang, Peng Zhang, Boyu Zhang, Hanyong Cai, Li Ma, Lishi Xiao, Qingxin Liu, Wenyu Wang, Sachiyo Nomura, Shi Chen, Xiangqi Meng, Mong-Hong Lee
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Chromosomal translocation (CT) is characterized by incorrect ligation between chromosome fragments when multiple DNA double-strand breaks (DSBs) exist simultaneously. DNA repair, the three-dimensional structure of the nucleoskeleton, and the spatial and temporal movement of DSB ends contribute to CTs. Our earlier research showed that Intermediate Filament Family Orphan 1 (IFFO1) links the nucleoskeleton and non-homologous end joining (NHEJ) to prevent CTs. In this study, we identified the paralog IFFO2, which, together with IFFO1, forms a complex interaction network. We demonstrate that the C-terminus of these IFFOs binds to XRCC4 to facilitate its participation in the NHEJ process. In contrast, their N-termini oversee the building of the nucleoskeleton by connecting with each other and Lamin A/C. Interestingly, IFFO1 and IFFO2 show epistatic effects in suppressing CT by anchoring broken DNA ends and have non-epistatic roles in NHEJ-mediated DSB repair. Our results define an integrated nucleoskeleton composed of IFFO1–IFFO2–Lamin A/C, and reveal its dual functions in genome stability maintenance, the promotion of end-joining, and the suppression of CT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddeaea17cd3e6c4aa994f084e3e194472b64771e" target='_blank'>
              A nucleoskeleton network preserves genomic integrity by promoting NHEJ and restraining chromosome translocations
              </a>
            </td>
          <td>
            Jingyan Liu, Xiuzhen Bai, Xinpeng Chen, Bohan Li, Huayu Zhao, Jiahui Wu, Yuanling Ye, Jiayi Yu, Zhenxin Yan, Rong Guo, Dongyi Xu, Wen Li
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retrovirus K (HERV-K) proviruses of the HML-2 subgroup are the most recently integrated retroviral elements within the human genome. The HERV-K Rec protein, a functional homolog of HIV Rev, is essential for the expression of viral mRNAs with retained introns. However, their diversity and functional capacities have remained largely unexplored. We identified Rec coding sequences in the human genome and selected intact Rec proteins from 28 proviral loci for functional characterization. Using a dual-color fluorescent reporter containing the HERV-K Rec-response element and a complementary enzyme-linked immunosorbent assay, we found that Rec proteins from nine proviral genomic loci promoted function at the post-transcriptional level. All but one of these proviruses are insertionally polymorphic in the human genome. In addition, several of the non-functional Rec proteins were trans-dominant negative. Detailed mutational analysis of the most potent inhibitory variant, encoded by the HERV-K provirus 12q14.1, displayed only two amino acid changes (N2H and E34del) relative to the prototypical functional Rec protein encoded by the reconstructed consensus HERV-K. Insertion of a glutamic acid at position 34 in this mutant fully restored the functional activity. Our results unveil an unexpected complexity in HERV-K (HML-2) post-transcriptional regulation, with a few Rec variants showing function, while others are trans-dominant negative. These findings significantly expand our understanding of the various Rec proteins that can be expressed in human cells and how they function at the post-transcriptional level. IMPORTANCE In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects. In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42022bba88ac63086a6e5b60150a1271cd5595a" target='_blank'>
              Characterization of HERV-K (HML-2) Rec proteins encoded in the human genome and their post-transcriptional function
              </a>
            </td>
          <td>
            Katarzyna Zurowska, Godfrey Dzhivhuho, David Grabski, D. Rekosh, M. Hammarskjold
          </td>
          <td>2025-11-18</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-25</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT High-quality reference genomes are essential for comparative genomics and accurate genotype-phenotype mapping. Here, we corrected the Mycobacterium tuberculosis Erdman strain reference genome (ErdmanTI) using ultra-deep HiFi sequencing. Among the small variants (n = 275) between ErdmanTI and the current Erdman reference NC_020559.1 (ErdmanSTJ), numerous are likely errors in ErdmanSTJ. We identified a novel bias toward in-frame structural variations (SVs) in pe/ppe genes and 28 SVs between ErdmanTI and ErdmanSTJ, half representing likely errors in ErdmanSTJ. Other SVs were consistent with in vitro evolution, including copy number variation (CNV) of promoter tandem repeats (PTRs). PTR CNVs were polyphyletic and within isogenic populations (10−2–10−3 CNVs/chromosome), demonstrating the impact of phase-variable CNV across evolutionary timescales. These hypervariable PTRs pinpoint a genomic basis for rapidly switching nitric oxide resistance (Dop), biofilm formation (LpdA), drug tolerance (EfpA), and glycerol utilization (GlpD2) phenotypes. This work uncovers a common phase variation mechanism obscured by short-read sequencing limitations and provides an improved reference for comparative studies. IMPORTANCE Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response. Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b608d990d2aa84315a169548ac17ba8eb0b7b1e2" target='_blank'>
              Updated Erdman reveals tandem repeat copy number is phase-variable and impacts M. tuberculosis adaptation across evolutionary timescales
              </a>
            </td>
          <td>
            Samuel J. Modlin, Nachiket Thosar, Paulina M. Mejía-Ponce, Raegan L. Lunceford, Gaëlle Guiewi Makafe, Brian Weinrick, F. Valafar
          </td>
          <td>2025-12-17</td>
          <td>mSystems</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colibactin is a secondary metabolite produced by bacteria present in the human gut and is implicated in the development of colorectal cancer. This genotoxin alkylates deoxyadenosines on opposite strands of host cell DNA to produce DNA interstrand cross-links. While cells have evolved multiple mechanisms to resolve (“unhook”) interstrand cross-links, little is known about which of these pathways promote resistance to colibactin. Here, we use Xenopus egg extracts to investigate replication-coupled repair of colibactin-induced interstrand cross-links. We show that replication fork stalling at a colibactin-induced interstrand cross-link activates the Fanconi anemia interstrand cross-link repair pathway, which unhooks the interstrand cross-link through nucleolytic incisions. These incisions generate a DNA double-strand break intermediate in one sister chromatid, which can be repaired by homologous recombination, and a monoadduct (“interstrand cross-link remnant”) in the other. Translesion synthesis past the colibactin-induced interstrand cross-link remnant depends on Pol η and the Pol κ-REV1-Pol ζ polymerase complex and introduces predominantly T>A point mutations at the sites of colibactin alkylation. Taken together, our work provides a molecular framework for understanding how cells tolerate a naturally occurring and clinically relevant interstrand cross-link. The bacterial genotoxin colibactin induces DNA interstrand cross-links which pose a barrier to DNA replication. Here, the authors use Xenopus egg extracts to show that the Fanconi anemia pathway is responsible for repairing these cross-links.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af9af8aec073ccec4ec8bcc56779d2d58c22a2a" target='_blank'>
              The Fanconi anemia pathway repairs colibactin-induced DNA interstrand cross-links
              </a>
            </td>
          <td>
            Maria Altshuller, Xu He, Elliot J. MacKrell, Kevin M. Wernke, Yougant Airan, Joel W. H. Wong, Selene Sellés-Baiget, T. Wang, Tsui-Fen Chou, Julien P. Duxin, E. Balskus, Seth B. Herzon, Daniel R. Semlow
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Purpose To delineate the genomic and cellular features of CCNE1 amplification across gynecologic cancer types using large-scale multi-omics data and assess its relevance for patient stratification and therapeutic targeting. Methods We analyzed genomic and transcriptomic data from 12,845 patients with gynecological cancers curated from the cBioPortal database, including ovarian, endometrial, and cervical carcinomas. CCNE1 amplification rates, cooccurring genomic alterations, and pathway enrichment were evaluated. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic data from cervical, endometrial, and ovarian tumors were used to investigate CCNE1 expression patterns, cell type specificity, and microenvironmental interactions. Results CCNE1 amplification was detected in 8.40% of all patients, with the highest rates in patients with ovarian cancer (8.01%) and endometrial cancer (5.06%). Among the ovarian subtypes, carcinosarcoma showed the highest amplification frequency (12.46%), followed by high-grade serous carcinoma (9.84%). Amplified tumors are frequently co-altered by TP53, PIK3CA, and PI3K-AKT pathway genes. Single-cell analyses revealed that CCNE1 expression was largely confined to epithelial cells and was associated with proliferative and structural remodeling pathways, with limited immune engagement across tumor types. Spatial transcriptomics confirmed the localization of CCNE1 in epithelial-rich regions and its correlation with adhesion- and motility-related genes in clear cell and high-grade serous ovarian carcinomas. Conclusion CCNE1 amplification is enriched in aggressive histologic subtypes of gynecological cancers and is linked to epithelial-specific expression and immune exclusion. These findings indicate that CCNE1 represents a potential molecular marker for tumor aggressiveness, warranting further investigation in the context of patient stratification and therapeutic targeting. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-04032-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07c358e69444ecc527d46e34a80b43d34d202949" target='_blank'>
              Multiomic analysis of CCNE1 amplification associated molecular and immune features in gynecological cancers
              </a>
            </td>
          <td>
            Linlin Liu, Wenjin Zheng, Liang Wen, Zhe Zhang, Yuanguang Meng
          </td>
          <td>2025-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Identifying viruses in tumor transcriptome helps to unravel the potential role of viruses in oncogenesis and tumor progression. Most of the current tools for virus identification in RNA-Seq data rely on sequence alignment, whose performance is constrained by fast mutations, large divergence, and the incompleteness of viral genomes. In this study, we develop ViTrace to detect viral sequences in human transcriptomic data by a hybrid language representation learning model, which integrates DNA contexts, position relationships and amino acid coding information. Although ViTrace is only trained on 13 species from 7 genera, it achieves 86.39% recall in 1179 absent-in-train virus strains of 935 species belonging to 167 genera across 10 phyla. Applied to single-cell RNA-seq data from esophageal and oropharyngeal squamous cell carcinomas, the model reveals tumor-, cell-, and patient-specific viral colonization patterns, uncovering both known and previously unreported viruses. Overall, ViTrace provides a scalable framework for guiding precision oncology and facilitating future discoveries of previously uncharacterized tumor-associated viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235fa47ea6e35bee83f6f73e3fb62ee13668fc7d" target='_blank'>
              ViTrace detects viral signatures in tumor transcriptomes using a hybrid language model
              </a>
            </td>
          <td>
            Feng Zhou, Yushuang He, Fan Yang, Jin Gu, Fengzhu Sun, Shunzhi Zhu, Xiaobing Huang, Ying Wang
          </td>
          <td>2025-12-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Structural variants (SVs) are widespread in vertebrate genomes, yet their evolutionary dynamics remain poorly understood. Using 45 long-read de novo genome assemblies and pangenome tools, we analyze SVs among three closely related species of North American jays (Aphelocoma, scrub-jays) displaying a 55-fold range in effective population size. We find rapid evolution of genome architecture, including ~100-megabase decreases in genome size driven by shifts in complex satellite landscapes. SVs exhibit slightly deleterious dynamics modulated by variant length and population size, with consistent evidence of adaptive fixation only in the largest population. Gene copy number variants exhibit an inverse relationship with population size, indicating strongly deleterious dynamics, with consequences for gene expression. Our long-read dataset and pangenome analysis demonstrate how population size shapes genome complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863e46ddcf563b99d9357e8ebcba5089d2175b17" target='_blank'>
              Multispecies pangenomes reveal a pervasive influence of population size on structural variation.
              </a>
            </td>
          <td>
            Scott V. Edwards, Bohao Fang, Danielle Khost, George E Kolyfetis, R. G. Cheek, Devon A. DeRaad, Nancy Chen, John W. Fitzpatrick, John E. McCormack, W. C. Funk, C. K. Ghalambor, Erik Garrison, A. Guarracino, Heng Li, Tim Sackton
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Age is a dominant risk factor for all major breast cancer subtypes. However, the mechanisms by which aging influences tumor development remain unclear. Using a novel mouse model whereby breast cancer is induced in situ in young and old wild-type mice via intraductal delivery of a lentivirus encoding the HER2/neu oncogene, we found that old mice exhibited a higher oncogene-induced tumor burden than young mice. Old tumor cells showed reduced expression of interferon-related genes, particularly the T cell-recruiting chemokines Cxcl9 and Cxcl10, linked to their altered chromatin accessibility. CXCL9/10 expression also declined with age in human HER2+ tumors. Correspondingly, old tumors exhibited fewer T cells within tumor lesions. Targeted interventions showed that decreased expression of Cxcl9/10 is responsible for reduced T cell infiltration and weakened anti-tumor immunity. These results show how aged tumor cells are impaired in their recruitment of immune cells, leading to a defective anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38011df7fcbd774b6090832583a2578934863ea" target='_blank'>
              Aging alters tumor cell - T cell crosstalk to promote breast cancer progression
              </a>
            </td>
          <td>
            Shanshan Yin, Kay T Yeung, S. Mamde, Kate Lin, Armin Gandhi, Xue Lei, Andrew Davis, Rouven Arnold, Jing Yang, P. Adams
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Platinum-based chemotherapeutic agents, such as cisplatin (Cis-Pt(II)), are widely used in cancer treatment but are limited by toxicity, DNA repair by cancer cells, and drug resistance. To address these limitations, we designed and synthesized in our laboratories a novel platinum-based compound, Aurkine16. In our previous studies, Aurkine16 demonstrated outstanding therapeutic efficacy, selectively inducing double-strand DNA breaks in both naïve and cisplatin-resistant cancer cells, without detectable toxic side effects at clinically relevant doses. In the present work, we report a computational analysis of Aurkine16, which reveals its unique activation pathway and its capacity to form stable [Aurki-GGG]3+ complexes. Unlike Cis-Pt(II), which induces single-strand DNA breaks, Aurkine16 simultaneously targets multiple nucleic bases, causing double-strand breaks and significant DNA disruption. Additionally, molecular dynamics simulations suggest that Aurkine16 is likely to exhibit specificity for cancer cells, avoiding off-target interactions within the nucleosome core. This selectivity, attributed to the steric hindrance from histone tails, underscores its potential for effectively targeting chromatin-accessible cancer cells. These computational findings position Aurkine16 as a promising alternative to platinum-based drugs, particularly for CisPt-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84eb8874122360d21b6369990c283ed3ad196e5" target='_blank'>
              Computational Analysis of a Next-Generation Platinum-Based Chemotherapies that Induce DNA Double-Strand Breaks.
              </a>
            </td>
          <td>
            Amanda R. Guimaraes, Óscar R Ballesteros, I. Rivilla, I. Olaizola, M. Odriozola-Gimeno, A. de Cózar, D. de Sancho, Xabier Lopez, J. Banales, F. Cossío
          </td>
          <td>2025-11-22</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf9f556241d0007b43f3963aee0e9a310306573" target='_blank'>
              lncRNA EGFR-AS1 promotes DNA damage repair by enhancing PARP1-mediated PARylation.
              </a>
            </td>
          <td>
            Kun Gao, Ruiya Shi, Chenying Xu, Zhaoyang Mao, Changying Guo, Liang Jin
          </td>
          <td>2026-01-05</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Apurinic/apyrimidinic endonucleases - APE1 and APE2 are central to genome maintenance and the cellular DNA damage response, with expanding relevance in cancer biology. APE1 is the primary endonuclease in base excision repair and functions as a redox coactivator of transcription factors. In contrast, APE2 exhibits PCNA dependent 3′–5′ exonuclease and 3′-phosphodiesterase activities, contributing to microhomology-mediated end joining, ATR-Chk1 activation, and immunoglobulin diversification. Both enzymes are often deregulated in cancer: APE1 is frequently overexpressed, drives tumor progression and chemoresistance, while APE2 is similarly upregulated in multiple malignancies. APE1 can be targeted by redox-specific or endonuclease inhibitors, with early clinical evidence of biological activity and tolerability. Although APE2-specific inhibitors remain in early development, emerging synthetic lethality data and preclinical studies highlight APE2 as a novel clinical target in breast cancer type 1/2 susceptibility (BRCA)-mutated cancers. This review discusses the structural and functional roles of APE1 and APE2, their contributions to cancer biology and therapeutics, recent advances in inhibitor development, and future strategies for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42492162323d747cbf3ed4c8e187621438e9d88" target='_blank'>
              Contrasting roles of APE1 and APE2 in genome maintenance, cancer development, and therapeutic targeting
              </a>
            </td>
          <td>
            Aman Sharma, Helen E. Grimsley, Katharine Courtemanche, Simon N. Powell
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma.
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, Jiil Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, Laura Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome.
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="To test the ability of a fission yeast CRISPR-Cas9 system ( SpEDIT ) to carry out genome editing over distance, we constructed a 1,935 bp-long, dsDNA repair template that contained 45 base pair substitutions (SNPs), relative to the wild-type target locus ade6 . Template-directed repair was efficient in the vicinity of the recombination-initiating dsDNA break, but the efficiency fell rapidly with distance (median editing tract length of 163 bp). The regularly distributed markers also revealed evidence for heteroduplex DNA at the ends of repair tracks and, unexpectedly, that DNA ends of the repair template participate in many (~18%) of the genome editing events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6863837635b53e863cff918447b5d90730e6d61" target='_blank'>
              Long multiply marked DNA repair template reveals lengths and fidelity of genome editing tracts in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Reine U. Protacio, Nyera A Ali, Akshara Chevireddy, W. P. Wahls
          </td>
          <td>2025-11-13</td>
          <td>microPublication Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Dysregulated transcription is a defining hallmark of cancer. Recently, novel chemically induced proximity approaches have enabled the rewiring of transcriptional machinery to drive expression of pro-apoptotic genes using bivalent small molecules. In this work, we demonstrate that this strategy is amenable to relocalizing DNA bound transcriptional machinery, such as fusion transcription factors that commonly drive pediatric malignancies. Targeting fusion transcription factors, such as EWSR1::FLI1 in Ewing sarcoma, with these bivalent compounds may open new therapeutic avenues. Here, we develop a small molecule, EB-TCIP, that recruits FKBP12F36V-tagged EWSR1::FLI1 to DNA sites bound by the transcriptional regulator BCL6, leading to rapid chromatin remodeling and expression of BCL6 target genes. This proof-of-concept study demonstrates that DNA binding proteins with pioneering transcription factor activity, such as EWSR1::FLI1, can be relocalized on chromatin to induce expression of repressed genes. Insights herein will guide the development of future bivalent molecules that rewire DNA binding transcriptional machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f7b6214e9ef05ce0f8a706c26c1ffab5a52705" target='_blank'>
              Rewiring the Fusion Oncoprotein EWSR1::FLI1 in Ewing Sarcoma with Bivalent Small Molecules.
              </a>
            </td>
          <td>
            Michael J Bond, Ryan P. Golden, G. Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, Hannah M Jones, Brendan G. Dwyer, N. Gray, K. Stegmaier
          </td>
          <td>2025-11-27</td>
          <td>Journal of the American Chemical Society</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Transcriptional regulation arises from the dynamic and combinatorial actions of multiple regulatory factors on genomic DNA. Although many epigenomic regulators have been identified, the precise order in which these factors accumulate at individual gene loci to activate transcription remains unclear. Here we show a single-cell data integration framework that infers the binding order of multiple chromatin factors at single-cell resolution. Central to this framework is sci-mtChIL-seq, a scalable single-cell method that simultaneously profiles genome-wide binding of RNA polymerase II (RNAPII) and diverse epigenomic regulators. By defining transcriptional states through RNAPII occupancy and integrating multiple sci-mtChIL-seq datasets, we systematically link the combinatorial patterns of transcription factor binding, histone modifications and chromatin remodeling. This framework reveals the temporal coordination among chromatin factors during transcriptional activation, providing a powerful approach to uncover context-dependent epigenomic dynamics and the principles of gene regulation in complex cellular systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1664f18d321dfb8507036c6decf249d794f38a" target='_blank'>
              Reconstructing epigenomic dynamics through a single-cell multi-epigenome data integration framework
              </a>
            </td>
          <td>
            Takeru Fujii, Kosuke Tomimatsu, Michiko Kato, Miho Ito, Shoko Sato, Hitoshi Kurumizaka, Y. Sato, Kazumitsu Maehara, Hiroshi Kimura, Akihito Harada, Yasuyuki Ohkawa
          </td>
          <td>2025-12-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980482a475765af78491bb69bee337aba4e9cbe1" target='_blank'>
              Discovering the complete enhancer map of human herpesviruses using a natural language processing model.
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Deepanway Ghosal, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-12-02</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c276e2f22ad6897e1f3b914ccbc31d7d9f5eea" target='_blank'>
              Uncovering High-Resolution Organization of Genomic Loci using Experimentally Informed Polymer Model
              </a>
            </td>
          <td>
            Rahul Mittal, D. Heermann, A. Bhattacherjee
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Sequestosome 1 (p62/SQSTM1) is a multifunctional adaptor protein whose dysregulation promotes tumorigenesis through autophagy, metabolic reprogramming, and immune modulation. However, its role across cancer types and impact on the tumor immune microenvironment remain poorly defined. Here, we performed a comprehensive pan-cancer analysis to delineate the molecular and immunological landscape of p62 across human malignancies. TCGA analysis revealed rare mutations and limited prognostic impact of genomic alterations but marked transcriptomic upregulation in LIHC, LUAD, BRCA, KIRP, KICH, KIRC, and READ, correlating with poor survival, advanced stage, and higher T classification, particularly in BRCA and LUAD. Pathway analysis showed strong positive associations between p62 and metabolic adaptation and stress tolerance pathways, including oxidative phosphorylation, reactive oxygen species, and DNA repair, in most cancers. Notably, high p62 expression inversely correlated with immune cell infiltration in most epithelial cancers, such as BRCA, COAD, ESCA, HNSC, KIRC, LIHC, LUAD, LUSC, PAAD, PRAD, READ, THCA, while coinciding with elevated expression of immunosuppressive checkpoints such as PD-L1, B7-H3, EBAG9, PVR, and TGFB1, supporting a link between p62-driven metabolic remodeling and an immune-excluded tumor microenvironment. In contrast, GBM, LGG, OV, SARC, and TGCT showed positive correlations between p62 and immunoscore, with enrichment of interferon and pro-inflammatory pathways, reflecting a distinct immune-activated phenotype. Collectively, these findings identify p62 as a central regulator of tumor metabolism and immunity, suggesting that its context-dependent activity may dictate the balance between immune suppression and activation across cancers, and highlight two natural-product-derived PB1 inhibitors (ZINC70669789 and ZINC08877690) as promising candidates for therapeutic development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-04135-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716e845c24cbd3bb41e4b234ab94e5c67dc1b5b3" target='_blank'>
              Comprehensive pan-cancer analysis of p62 reveals its contribution to shaping tumor microenvironment and anti-tumor immunity
              </a>
            </td>
          <td>
            Zahra Nayeri, Elahe Tavakol, M. Rahmati, Marco Cordani, M. Djavaheri-Mergny, Vahid Shariati, M. Moosavi
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cells continuously receive mechanical inputs from their environment. For example, during migration in narrow spaces, or in solid tumors, cells and their nuclei experience confinement which they sense and respond to. Emerging evidence identifies the genome as a central mediator of these responses, dynamically reorganizing its three-dimensional (3D) structure to regulate both short– and long-term gene expression programs, enabling cellular adaptation to mechanical constraints. However, the mechanisms underlying such responses, especially those linking the 3D genome and transcriptome, are unknown. Here, we utilize controlled cell confinement followed by Hi-C and transcriptome analyses to map and model temporal responses to mechano-confinement in the 3D genome. We identify clusters of genes (termed “TECs”) exhibiting coordinated temporal transcriptional responses tied to their differential radial positioning within the nucleus. Additionally, we uncover a genome-wide, partially reversible response, wherein chromosomes reposition under confinement to enhance select genome compartments, aligning with temporal gene regulation patterns. Specifically, one such cluster, TEC7, is linked to 3D genome restructuring and nuclear translocation of NF-κB, driving cytokine expression and secretion. Our findings reveal how 3D genome reorganization and transcriptional programs can drive mechanoresponses, opening up for new views into the mechanisms of mechanogenomic regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d44f4c892c64801826a96729296664e6be36ddd8" target='_blank'>
              The 3D nuclear position and compartmentalization of genes prime their response to mechano-confinement
              </a>
            </td>
          <td>
            Saleh Oshaghi, Oda Hovet, Andrea Halaburkova, C. Progida, Jonas Paulsen
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8461ad6cad0781da64419f6c03309084a55c23fc" target='_blank'>
              CHAMP1 complex promotes heterochromatin assembly and reduces replication stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y. Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2026-01-06</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive and recurrent malignancy with a poor prognosis. Although temozolomide (TMZ) is a cornerstone of GBM treatment, its efficacy is often compromised by inherent or acquired resistance, underscoring the urgent need to uncover molecular mechanisms, discover new therapeutic targets, and develop innovative treatment strategies. In this study, we found an increased formation of filamentous actin (F-actin) within the nuclei of TMZ-resistant GBM cells. We also showed that overexpression of FSCN1 in TMZ-resistant GBM cells promotes F-actin formation and facilitates the repair of DNA double-strand breaks (DSBs). Further investigation revealed a marked decrease in the expression of YTHDC1 in TMZ-resistant GBM cells, which regulates FSCN1 through m6A modification. Additionally, FSCN1 activates the CDC42/N-WASP/Arp2/3 signaling pathway by recruiting FGD1 to activate CDC42GTP, which drives nuclear F-actin formation. Importantly, combining the FSCN1 inhibitor NP-G2-044, with TMZ therapy resulted in stronger anti-tumor effects both in vitro and in vivo. In conclusion, the study demonstrates that nuclear F-actin formation in GBM promotes DSB repair and reveals that targeting FSCN1 with NP-G2-044 could be a promising strategy for enhancing treatment outcomes and improving the prognosis for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ed165e5a36d8842faf93528f5c59ba7eabd365" target='_blank'>
              Low YTHDC1 Expression Upregulates FSCN1 to Promote Nuclear F-Actin Formation and Facilitate Double-strand DNA Breaks Repair in TMZ-Resistant Glioblastoma.
              </a>
            </td>
          <td>
            Minglong Yang, Wan-xiang Niu, Yuanfei Wang, Peng Chen, Maolin Mu, Xiaoming Zhang, Ben Xu, Shanshan Hu, Chaoshi Niu, Pengfei Wu
          </td>
          <td>2025-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4070428f7f9a105c81fda51b099b892dbebdd1" target='_blank'>
              Novel HSPA8-NR4A2 rearrangement in extraskeletal myxoid chondrosarcoma: a sarcoma mimicker of neuroendocrine neoplasia.
              </a>
            </td>
          <td>
            S. Barresi, R. Passa, Silvia Vallese, E. Miele, S. Patrizi, Carla Rabitti, B. Vincenzi, Sergio Valeri, F. Camassei, A. Minucci, M. Pieraccioli, Claudio Sette, R. Alaggio, G. Rindi
          </td>
          <td>2025-11-28</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs), constituting approximately 8% of the human genome, represent genomic remnants of ancestral retroviral infections that colonized the germline through evolutionary processes. While most HERVs remain epigenetically silenced, their reactivation through environmental stimuli or epigenetic dysregulation enables participation in oncogenesis via viral mimicry, immunomodulation, and insertional mutagenesis. Substantial evidence now implicates aberrant HERVs activity across urologic malignancies—including prostate cancer, renal cell carcinoma (RCC), bladder cancer, and testicular germ cell tumors—where cancer-type-specific mechanisms drive tumor development and progression. These encompass androgen-responsive HERV-K activation in prostate malignancies, hypoxia-inducible factor-mediated ERV immunogenicity in RCC, HERV-derived microRNA silencing of tumor suppressors in bladder cancer, and DNA hypomethylation-associated HERV expression in testicular germ cell tumors. This review synthesizes fundamental HERV biology with recent advances in their diagnostic and therapeutic applications for urologic neoplasms. Key clinical translations include ERV-based stratification models predicting immune checkpoint inhibitor response in metastatic RCC, HERV-E-targeted adoptive T cell therapies, and noncoding RNA biomarkers for early bladder cancer detection. We further discuss unresolved mechanistic paradoxes such as contradictory prognostic associations between HERV superfamily expression and PBRM1 inactivation in RCC, concluding with priorities for future research: validating HERV-derived neoantigens in immunotherapy platforms, optimizing epigenetic priming strategies to enhance viral mimicry effects, and establishing standardized HERV signatures as clinical biomarkers through multi-institutional cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38383e2f180ac1b254c2d12832458ab1cd318585" target='_blank'>
              The expression, regulation, and function of human endogenous retroviruses in genitourinary cancers
              </a>
            </td>
          <td>
            Wenjie Ma, Chencheng Ji, Abudukelimu Abudushataer, Ning Liu, Tao Xu, Kunlun Zhao, Yiguan Qian, Paerhati Tuerxun, Xiaotian Jiang, Zhongli Xiong, Min Wang, Ruipeng Jia, Zheng Xu, Yang Li, Yu-Zheng Ge
          </td>
          <td>2025-11-28</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Meningiomas are the most common primary tumors of the central nervous system and are typically treated with surgery or radiation, as targeted therapies remain limited. Despite extensive study, seventeen percent of meningiomas lack known genetic drivers. Our analysis of meningiomas without driver mutations or major chromosomal alterations identifies a subset with recurrent genomic rearrangements involving the FOS and FOSB genes. These tumors exhibit elevated FOS/FOSB protein levels and retain meningothelial lineage. Here we show that FOS/FOSB fusion-positive meningiomas represent a distinct molecular subgroup, defined by unique gene expression patterns, including activation of AP-1 target genes and signatures resembling preadipocyte-like and mast cell–associated profiles. Clinically, these tumors display low-grade behavior and DNA methylation profiles consistent with benign subtypes. Our findings identify a meningioma subgroup with distinct genetic, transcriptomic, and clinical features, expanding the molecular classification of meningiomas and opening new avenues for targeted treatment strategies. Approximately 17% of meningiomas remain genomically uncharacterized. Here, the authors analyze 105 meningiomas without known driver mutations or significant copy number alterations and identify a subgroup of meningiomas, defined by FOS/FOSB gene fusions with distinctive transcriptomic and histopathological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/848d67ce829255f08e0caafaf14bc914e2cca146" target='_blank'>
              Subgroup of meningiomas involving FOS and FOSB gene fusions
              </a>
            </td>
          <td>
            K. Yalcin, Hasan Alanya, B. Gultekin, Diego Samper Figuera, Tanyeri Barak, D. Miyagishima, Mark W. Youngblood, Mateo Gomez Hjerthen, Amos Brooks, Natalia Samuel Lopez, Joseph O’Brien, M. M. Chavez, Alper Dincer, Hao Wu, S. Omay, Katsuhito Yasuno, K. Bilguvar, A. Ercan-Sencicek, Ketu Mishra-Gorur, D. McGuone, Jennifer Moliterno, Murat Gunel, E. Erson-Omay
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020f876f8e4a48a7cc99413edfa413effb89316e" target='_blank'>
              Interphase chromosome conformation is specified by distinct folding programmes inherited through mitotic chromosomes or the cytoplasm.
              </a>
            </td>
          <td>
            A. Schooley, Sergey V. Venev, V. Aksenova, Jesse W. Lehman, E. Navarrete, A. Pai, M. Dasso, Job Dekker
          </td>
          <td>2025-12-22</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The pathogenesis of koala retrovirus (KoRV) has been explored in various contexts, yet its role in tumorigenesis remains incompletely understood. Unlike acute transforming retroviruses, KoRV lacks a viral oncogene but may contribute to oncogenesis via indirect mechanisms. However, the relationship between KoRV and telomere length, as a potential indicator of telomerase activity, has not been examined. This study investigates the effect of KoRV infection on telomere length in 47 samples from Southern Australian koalas in a novel telomere length quantification method. Telomere lengths of 30 KoRV-negative samples were compared to those of 17 KoRV-positive samples using the Absolute Human Telomere Length Quantification qPCR kit (ScienCell Research Laboratories, California, USA). The telomere length in KoRV-infected WBCs was significantly longer than the uninfected ones (t = −2.059, p-value = 0.045). In line with this, telomere length correlated positively with proviral load (r = 0.421, p-value = 0.003), further linking viral burden to telomere elongation. Furthermore, the effect of age on telomere length differed by infection status (β = −5329.7, p-value = 0.0038); KoRV-positive individuals exhibited longer telomeres at a younger age but experienced more rapid telomere attrition over time compared to KoRV-negative individuals. These results suggest KoRV promotes telomerase elongation ability and modulates age-related telomere dynamics, potentially contributing to subsequent cellular immortality and oncogenesis. These pathways may overlap with other retroviruses, where telomerase dysregulation contributes to their oncogenic potential. This study provides new insights into KoRV pathogenesis and DNA quantification methodology, which could be valuable for future research by identifying predictive markers for tumour progression and potential therapeutic targets in affected koalas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f125ee840ff1c5667ff63226a6db37a09c90835b" target='_blank'>
              Infection-Induced Telomere Length Variation: Insights into Pathogenesis of Koala Retrovirus
              </a>
            </td>
          <td>
            Hiu Ming Cheung, Sze Wing Jamie Lin, Hanh Thi Hong Nguyen, T. Stephenson, N. Speight, F. Hemmatzadeh
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA methylation (DNAm) is an epigenetic modification involved in gene regulation. DNAm quantitative trait loci (mQTLs) have been identified in many tissues, but bulk-tissue studies obscure cell type-specific effects. Here, we present the first multi-tissue landscape of cell type-dependent regulation of DNAm in humans by mapping cell type-interaction mQTLs (imQTLs) across seven tissue types (breast, colon, lung, ovary, prostate, kidney, and whole blood), identifying 3,150 imQTLs. Inter-individual variability in cell type proportion, rather than mean proportion, was most associated with imQTL discovery. The cell type with the most imQTLs tended to have interaction effects directionally consistent with mQTL marginal effects from bulk-tissue. imQTLs exhibited biologically relevant effect sharing across related cell types. Compared to cell-agnostic mQTLs, imQTLs exhibited stronger enrichment in regulatory elements and higher colocalization with eQTLs and GWAS loci. Our cell type deconvolution-based approach provides a scalable alternative to single-cell DNAm profiling for uncovering the cellular contexts of genetic regulation of DNAm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9156b5f2cddfd87e50484abcf953648e65b081c5" target='_blank'>
              Mapping the landscape of cell type-dependent genetic regulation of DNA methylation across human tissues
              </a>
            </td>
          <td>
            James L. Li, Niyati Jain, Jason Kiat, Lin Tong, Meritxell Oliva, Kathryn Demanelis, Jasmine Farzana, M. Kibriya, Habibul Ahsan, Lin Chen, Andrew E. Teschendorff, Brandon Pierce
          </td>
          <td>2025-11-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f8abb290a62c5b3aa53581143c941a3ce167f6" target='_blank'>
              Lenacapavir-induced capsid damage uncovers HIV-1 genomes emanating from nuclear speckles.
              </a>
            </td>
          <td>
            Thorsten G. Müller, Severina Klaus, V. Zila, Bojana Lucic, C. Penzo, Svenja L. Nopper, Gonen Golani, Maria Anders-Össwein, Vera Sonntag-Buck, A. Heuser, U. Schwarz, V. Laketa, M. Lusic, Barbara Müller, H. Kräusslich
          </td>
          <td>2025-12-01</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background While nuclear DNA (nDNA) damage and alterations in nDNA repair are known to play a role in colon cancer (CC), there is insufficient research investigating these processes in mitochondrial DNA (mtDNA). Methods This study investigates mtDNA changes in CC, focusing on mitochondrial DNA copy number (mtDNA-CN) variations, mtDNA damage, and the expression and mutation status of DNA repair genes. Three cohorts were analyzed: healthy controls, colon adenoma patients, and CC patients, divided into a pilot and a validation set. Results Our findings revealed that mtDNA-CN was elevated in colon adenomas compared to adenoma-adjacent mucosa (FDR = 0.04), healthy mucosa (FDR = 0.005), and tumor-adjacent mucosa (FDR = 0.005). Moreover, mtDNA-CN was elevated in adenoma-adjacent mucosa compared to healthy mucosa (FDR = 0.04). MtDNA damage was greater in tumor-adjacent mucosa compared to tumor tissue in both the pilot and validation sets (FDR = 0.031 and FDR = 2.06e-05, respectively). Additionally, we identified novel DNA repair genes associated with mtDNA damage, predominantly upregulated in adenoma and tumor tissues compared to healthy colon tissues. Conclusions To conclude, this study highlights the importance of mtDNA alterations in CC development and identifies potential mtDNA biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01400-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/267276dd26cb529d07f65beba245c3f404b9cf18" target='_blank'>
              Dysregulated mitochondrial homeostasis and DNA repair in the progression from colon adenoma to cancer
              </a>
            </td>
          <td>
            Natalie Danesova, Josef Horak, Anna Valickova, Adrian Gil-Korilis, Jorge Ergui-Arbizu, Richard Palek, Jan Bruha, Miroslav Levy, Pavel Škrobánek, Jan Kral, J. Jungwirth, Jiří Neužil, Veronika Vymetálková, P. Vodicka, Sona Vodenkova, Kristyna Tomasova
          </td>
          <td>2025-11-22</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The single-stranded DNA deaminase APOBEC3B (A3B) is capable of potently restricting the replication of a range of viruses including retroviruses (cDNA) and herpesviruses (genomic DNA). However, these and likely other DNA virus families have evolved host species-specific counter-defenses that are equally potent and serve to protect viral DNA from restriction. Although high-risk human papillomavirus (HPV) infection triggers A3B upregulation, potentially as part of an antiviral response, the impact of this restriction factor on papillomavirus replication and pathogenesis has yet to be assessed. To study human A3B antiviral function in the absence of a species-specific counter-defense, here we ask whether human A3B is capable of restricting Mus musculus papillomavirus (MmuPV1) in cellulo and in vivo. First, we created human A3B and catalytic mutant A3B-E255A expressing FVB/N mice. Second, MmuPV1 gene expression and replication was quantified in primary keratinocytes from these animals and, surprisingly, enzymatically active human A3B caused no measurable impairment in viral transcript or DNA accumulation. Third, A3B, catalytic mutant A3B-E255A, and nontransgenic FVB/N animals were infected with MmuPV1 and similar pathologies were found regardless of A3B functionality. Thus, despite likely never being exposed to human A3B during evolution, MmuPV1 appears to be unaffected by this potent, primate-specific antiviral factor. These results suggest that MmuPV1 and perhaps papillomaviruses more broadly possess a conserved mechanism to efficiently escape restriction by human A3B and related DNA deaminases. IMPORTANCE Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes such as A3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43450515e8b62fe85e717aebfac6a246b0159e06" target='_blank'>
              Mus Musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, Sang Hyun Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Epstein–Barr virus (EBV)-associated gastric carcinoma (EBVaGC) represents a distinct molecular subtype characterized by extensive DNA methylation and altered cell death signaling. This study investigated the regulation and function of gasdermin E (GSDME), a key mediator of pyroptosis, in EBVaGC. Transcriptomic analysis of The Cancer Genome Atlas (TCGA) data revealed that GSDME expression was selectively suppressed in EBV-positive gastric cancer, while other gasdermin family members were upregulated. Validation in multiple cell lines confirmed that EBV infection markedly reduced GSDME expression through promoter hypermethylation, which was reversed by treatment with the DNA methyltransferase inhibitor 5-azacytidine. EBV-positive cells exhibited enhanced caspase-3 activation and increased GSDME cleavage upon paclitaxel (PTX) exposure, leading to elevated lactate dehydrogenase (LDH) release and pyroptotic morphology. Overexpression of GSDME amplified, whereas siRNA-mediated knockdown or caspase-3 inhibition suppressed, PTX-induced pyroptosis without significantly altering overall cell viability. These findings demonstrate that EBV-induced epigenetic silencing of GSDME contributes to the modulation of chemotherapy-induced cell death, and that GSDME acts as a critical effector converting apoptosis to pyroptosis through caspase-3 activation. Collectively, our results reveal a novel link between EBV-driven DNA methylation and pyroptotic cell death, suggesting that restoration of GSDME expression may enhance therapeutic responses in EBV-associated gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a0b5107e1c2a7dee621e1b78f47e3daf342a13" target='_blank'>
              Epstein–Barr Virus Silences GSDME and Pyroptosis in Gastric Cancer
              </a>
            </td>
          <td>
            Min-Hyeok Lee, Ju Yeon Lee, Jun Yeob Kim, Yea Rim An, Suk Kyeong Lee
          </td>
          <td>2025-11-27</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Three-dimensional genome architecture is essential for gene regulation and cellular homeostasis. Its perturbation underlies pathologies. Advances in chromosome conformation capture technologies, such as Hi-C [1], have enabled researchers to study genome architecture at high resolution, yet the resulting matrices are difficult to explore visually, especially when one wishes to overlay genomic annotations or examine inter-chromosomal interactions. We introduce gghic, an R package that extends the ggplot2 grammar for the publication-quality visualization of chromatin-interaction data. Gghic supplies layers that draw (i) triangular Hi-C heatmaps, (ii) chromatin loops, topologically associating domains, (iii) gene or transcript models, (iv) one-dimensional signal tracks, (v) and multi-way contacts. The package supports Bioconductor classes, including HiCExperiment [2] and GInteractions [3]. It handles data from multiple chromosomes, enabling inspection of inter-chromosomal contacts. We illustrate the utility of gghic with acute lymphoblastic leukemia Hi-C datasets, producing publication-ready figures that reveal translocation breakpoints and higher-order chromatin hubs. By coupling advanced genome-structure analyses with the ggplot2 syntax, gghic makes the exploration and communication of 3D genomics data both intuitive and reproducible. References 1. Belton JM, McCord RP, Gibcus JH et al. ‘Hi-C: a comprehensive technique to capture the conformation of genomes’ Methods 2012;58:268–276. 2. Serizay J, Matthey-Doret C, Bignaud A et al. ‘Orchestrating chromosome conformation capture analysis with Bioconductor’ Nat Commun 2024;15:1072. 3. Lun AT, Perry M, Ing-Simmons E. ‘Infrastructure for genomic interactions: Bioconductor classes for Hi-C, ChIA-PET and related experiments’ F1000Res 2016;5:950.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbff5aed0fd8c28e8521d832d596353743e33b7" target='_blank'>
              Gghic: a versatile R package for exploring and visualizing 3D genome organization
              </a>
            </td>
          <td>
            Minghao Jiang, Duohui Jing, Jason W.H. Wong
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Medulloblastoma (MB) is the most common malignant pediatric brain tumor and exhibits high heterogeneity in genetic alterations and molecular subgroups. However, current treatment remains largely uniform, and more precise risk stratification is necessary to improve prognosis. Large-scale genomic studies have primarily focused on Western populations, while comprehensive data on Japanese patients remain limited due to low incidence. This study aimed to elucidate molecular genetic characteristics and prognostic factors in Japanese MB through integrative genomic and transcriptomic analyses. Methods Targeted DNA and RNA sequencing was performed on 200 cases from the Japanese Children’s Cancer Group (JCCG), and publicly available whole-genome and RNA sequencing data from 472 Western MB patients were analyzed for comparison. Results Known subgroup-specific driver mutations were identified in the Japanese cohort, and age-adjusted comparison with the Western cohort showed no significant differences in mutation frequencies, suggesting limited racial differences in MB pathogenesis. Deep sequencing detected subclonal mutations, including SMARCA4 in the WNT subgroup. Missense mutations in the MYC functional domain were observed in cases lacking MYC amplification; these are known drivers in hematological malignancies and may also play an oncogenic role in MB. MYC mutations were found in WNT and SHH subgroups, suggesting broader involvement beyond Group 3. Intron retention analysis revealed a non-canonical intronic ZMYM3 mutation in a Group 4 case, highlighting improved driver detection by integrating DNA and RNA data. Whole chromosomal aberrations correlated with better outcomes, supporting their role in risk classification. Conclusions This study identified novel MYC alterations and emphasized the utility of RNA-integrated analysis. Cohort expansion may uncover additional diversity and aid in precision medicine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394acb3917250c67e7f0d75d4a797dbd4bea5da9" target='_blank'>
              10102-GEN-2 Integrated Genomic and Transcriptomic Analysis Reveals Molecular Diversity and Novel Drivers in Japanese Medulloblastoma
              </a>
            </td>
          <td>
            Tsubasa Miyauchi, Takuma Nakashima, Miho Kato, Junko Hirato, Takako Yoshioka, Hajime Okita, Y. Kanemura, Yuko Watanabe, Masahiro Toda, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dd4fb7e5f25ab6e316e3038078f86548a96c3c" target='_blank'>
              ALC1 Finds a New Foothold on the Nucleosome’s Super-Groove
              </a>
            </td>
          <td>
            Hannah R. Bridges, L. Bacic, S. Deindl, G. Gaullier
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde6df8438fec9eca3014692df8a1930274045c8" target='_blank'>
              A pangenome framework uncovers the role of deletions in repeated evolution of cave-derived traits
              </a>
            </td>
          <td>
            Emma Y. Roback, X. Maggs, Edward S Ricemeyer, Adam Warlen, Rachel A. Carroll, Christine G Elsik, Alex C. Keene, Nicolas Rohner, S. McGaugh, Wes C. Warren
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11e6d7e02da2372f0b7447b006156f71a150d88" target='_blank'>
              Evo2HiC: a multimodal foundation model for integrative analysis of genome sequence and architecture
              </a>
            </td>
          <td>
            Tangqi Fang, Xiao Wang, Zhiping Xiao, Shengqi Hang, Ghulam Murtaza, Junwei Yang, Hanwen Xu, Anupama Jha, W. Noble, Sheng Wang
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) is a DNA virus that establishes a persistent asymptomatic infection during childhood and can cause Merkel cell carcinoma (MCC) later in life. Its Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, drive viral replication and tumorigenesis by binding to and perturbing the function of host proteins. LT binds and inhibits RB1, deregulating E2F activity and host cell cycle control to permit viral replication during S phase. While the functions of LT and ST are relatively well characterized, how their expression is controlled is poorly defined. Here, we discovered that E2F1-3/DP1 dimers bind the MCPyV Non-Coding Control Region (NCCR) via a consensus E2 site close to the LT/ST transcriptional start site. Inhibiting E2F-NCCR binding, either by deleting the E2 site or treatment with a E2F small molecule inhibitor, downregulated LT/ST mRNA and protein expression in MCC cells and in 293A cells transfected with MCPyV. Our findings reveal an E2F/LT/RB1 positive feedback loop that appears to have evolved to support viral replication and is hijacked in MCC cells to promote cellular proliferation. Furthermore, we identified E2 sites in the NCCRs of PyVs closely related to MCPyV, including murine PyV, which mediate E2F/DP binding and potentiate viral early transcription. E2F/DP also bound weakly to the SV40 and BKPyV NCCRs, despite lacking an E2 site. Our findings challenge the prevailing model that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV early transcription and replication. Significance statement Polyomaviruses (PyVs) express Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, that drive viral replication and tumorigenesis by binding and perturbing the function of host cells. LTs bind and inhibit RB1 via conserved LxCxE motifs, deregulating E2F activity and host cell cycle control to permit viral replication. Here, we discovered that E2F1-3 bind to the Non-Coding Control Regions (NCCRs) of many, but not all, PyVs to activate LT/ST transcription, revealing an E2F/LT/RB1 positive feedback loop that appears to have evolved to promote viral replication. Our findings challenge the existing paradigm that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV LT/ST transcription and replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92fb2e450770315d4079985e1a3aa56f6892c80a" target='_blank'>
              E2F1–3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Nick Salisbury, Supriya Patil-Amonkar, Ann Roman, Denise A. Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, the roles of which are under debate. We found that double-strand breaks (DSBs) induce de novo formation of chromatin loops in human cells, with the loop base positioned at the DSB site. These loops form in the S and G2 phases of the cell cycle during homologous recombination repair, concomitantly with DNA end resection and radiation-sensitive protein 51 (RAD51) recruitment. RAD51 shows a broad (megabase-sized) chromatin domain reflective of the homology search. This domain is regulated by cohesin unloader and overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of the loop-extruding cohesin subunit NIPBL lowers homologous recombination in mouse embryonic stem cells, and this effect is more pronounced when the homologous recombination donor is hundreds of kilobases from the DSB. Our data indicate that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e61307e4b5e3ffa40f9315b36ee4db40b3f94b6" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam T. Rybczynski, Namrata Nilavar, Daniel Nguyen, Andrew G. Li, Violetta Karwacki-Neisius, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g., “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, S. Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Analysis of coding areas has long been used to study monogenic illnesses, but despite the extensive use of whole-exome sequencing (WES), up to half of suspected cases remain genetically unexplained. Variants outside coding areas can alter splicing, transcript stability, or gene regulation, compromising normal gene activity. These include mutations in noncoding RNAs, promoters, enhancers, deep intronic sequences, and untranslated regions (UTRs). Several well-known disorders have been linked to these mechanisms, including β-thalassemia caused by deep intronic mutations leading to aberrant splicing, familial hypercholesterolemia caused by promoter defects affecting LDLR expression, and inherited retinal diseases driven by noncoding variants influencing retinal gene regulation. These instances show that pathogenic variation is not limited to the exome and can have significant clinical implications. This review summarizes current understanding of noncoding and regulatory variants in monogenic diseases, discusses how they influence diagnosis and therapy, and highlights integrative approaches combining genomic, transcriptomic, and epigenomic data. Multi-layered research has increased diagnostic accuracy and unveiled new therapeutic potentials, although noncoding variations make the connection between genotype and phenotype more complex. Noncoding regions will need to be incorporated into standard diagnostic procedures to convert molecular insights into concrete therapeutic applications in the future. Predictive algorithms, patient-derived model systems, and functional validation testing will all help to simplify this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffd879f4fd9d85153dc94e05643890e26eb43d7" target='_blank'>
              Beyond the Exome: The Role of Noncoding and Regulatory Variants in Monogenic Diseases
              </a>
            </td>
          <td>
            Efthalia Moustakli, Nektaria Zagorianakou, Stylianos Makrydimas, Andreas Miltiadous, Alexandros T. Tzallas, G. Makrydimas
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="A correlation between karyotype diversity and species richness was first observed in mammals in 1980, and subsequently confirmed after controlling for phylogenetic signal. The correlation was attributed to submicroscopic factors, presumably operating at the level of the genome. At the same time, an unexpected association between mutation rates and substitution rates has been observed in all eukaryotes so far examined. One hypothesis to explain the latter observation proposed that neutral mutation (dS) and non-neutral (dN) substitution rates in gene codons co-vary according to genomic position, or location in the genome. Later, it was found that mutation and substitution rates in eukaryotes increase with DNA replication timing during the Synthetic phase, or S phase, of the cell cycle. In 1991, Motoo Kimura proposed a molecular theory of non-adaptive radiation (NAR). Accordingly, genetic drift plays a significant role in speciation, albeit in parallel to and in conjunction with natural selection. The following will examine the contribution of DNA replication timing and DNA repair factors to the relationship between species richness and karyotype diversity, and by extension, to the large variation in species richness across the mammalian Tree of Life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8f06330c7331a025a26ac3aac8d62db4f0ac6f" target='_blank'>
              DNA Replication Timing, Genome Stability and Non-adaptive Radiation
              </a>
            </td>
          <td>
            John Herrick
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Maintaining genome integrity in germ cells is crucial for fertility and species survival. However, the DNA repair mechanisms that sustain genome integrity in primordial germ cells (PGCs), which cope with high levels of replication stress, remain largely unknown. While the TatD family of proteins, evolutionarily conserved nucleases, has been found to play roles in various DNA-related processes, their in vivo functions in vertebrates have yet to be fully elucidated. TATDN2 has recently been implicated in resolving R-loops and participating in the replication stress response in BRCA1-deficient cancer cells. Here we found that tatdn2 exhibits conserved expression in mitotic and early meiotic germ cells across teleosts and mammals. Using medaka fish as a model, we then showed that loss of tatdn2 leads to all-phenotypically male adults and infertility due to PGC depletion during mitotic proliferation. We further demonstrated that knockout of tatdn2 increases R-loop accumulation and DNA damage, subsequently triggering apoptosis in PGCs. These findings indicate that tatdn2 plays a critical role in DNA damage repair associated with R-loop resolution in mitotic PGCs. Our study provides novel insights into the physiological function of TATDN2 and the mechanisms of genome maintenance in PGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aba7e41b7a154253da5adb67bf89677856b6ff" target='_blank'>
              Tatdn2 is required for DNA repair to safeguard genome stability in primordial germ cells
              </a>
            </td>
          <td>
            Hanqiao Shang, Puxuan Jiang, Zehui Wang, Zhaojun Shan, Yuze Chen, Ting Zhang, Yingshu Li, Qiang Tu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT During cell division and gene expression, the DNA double‐helical structure unwinds, thereby generating torsional stress. DNA topoisomerases are enzymes that relieve this stress. During this process, topoisomerases form temporary covalent bonds with the phosphate backbone of DNA, generating DNA strand breaks and relieving torsional stress. Topoisomerases then dissociate from DNA after rejoining the DNA breaks. Torsional stress associated with replication or transcription is primarily relieved by topoisomerase I (TOP1) and II (TOP2). Some anticancer drugs targeting topoisomerases, known as topoisomerase poisons, trap the topoisomerase reaction intermediates and cause DNA strand breaks bearing topoisomerase–DNA–protein crosslinks (TOP–DPCs). TOP1 poisons, such as camptothecin, cause DNA single‐strand breaks bearing TOP1–DPCs, which are converted to DNA double‐strand breaks (DSBs) when they collide with DNA replication forks. In contrast, TOP2 poisons, such as etoposide, directly induce DSBs in TOP2–DPCs. However, to elicit a DSB response, TOP2–DPC must first be removed from the DSB ends. Cells possess various pathways to remove TOP2–DPC, and these pathways are thought to function in coordination depending on the situation. This review summarizes these sophisticated TOP2–DPC removal pathways and discusses the clinical applications of TOP2 poison as an anticancer drug, as well as the related challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1a3ae05a710b9f77cc942446aa4e72d459275cd" target='_blank'>
              Processing of DNA Topoisomerase II–DNA–Protein Crosslinks Associated With Anticancer Drugs
              </a>
            </td>
          <td>
            Ryo Sakasai, Kuniyoshi Iwabuchi
          </td>
          <td>2025-12-17</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Using the UCSC genome browser, this study primarily examined the density of CpG Islands in each gene to investigate epigenetic changes, particularly methylation patterns or profile throughout the genome of TNBC genes. Here the analysis of 10 hub TNBC genes were study for the methylation profiling and additionally focused on its relevance of reprograming to these genes.
 It can also analyze how certain genomic areas are conserved across species, which can reveal information about their functional significance. It also examines the existence of CpG islands, which could be connected to other regulatory elements or gene promoters. Examining the data on histone modifications to find possible promoters or enhancers. Genes such as AURKA, CDK1, EZH2, CCNA2, CTNNB1, MKI67, etc., are known cell cycle regulators, chromatin modifiers, and proliferation markers, many of which are overexpressed in cancer and stem cells, including induced pluripotent stem cells (iPSCs). These 10 genes are reprogramming-supportive genes and indirectly modulated or supported by Yamanaka factors. Their hypomethylation and expression patterns in these results indicate that the cellular environment in TNBC resembles a partially reprogrammed state, which may explain aggressive growth and plasticity — a useful insight for drug targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7b6a886f40c24727327b2385ecbc05a3309cb8" target='_blank'>
              Methylation Profiling of TNBC Hub Genes Using UCSC Genome Browser: A Comparative Study Relevance to Reprogramming
              </a>
            </td>
          <td>
            Shivi Sharma, Prashant Ankur Jain, Eliza Chakraborty
          </td>
          <td>2025-11-07</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4efb8218ff34b8674a51cd71fbecd16898befa89" target='_blank'>
              Developing Topics.
              </a>
            </td>
          <td>
            Morgan Elizabeth Lambert, Paulino Ramirez, Wenyan Sun, Bess Frost
          </td>
          <td>2025-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd886b9447e566661dd9631aefea2ff0dec8c519" target='_blank'>
              Mechanisms of transcription-coupled repair and DNA damage surveillance in health and disease.
              </a>
            </td>
          <td>
            Marjolein van Sluis, Camila Gonzalo-Hansen, Qingrong Li, H. Lans, Dong Wang, J. Marteijn
          </td>
          <td>2025-11-18</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Despite advancements in treatment, 5-year survival for medulloblastoma (MB) remains 60-70%. Human cytomegalovirus(HCMV) has been implicated as an oncomodulator in MB, but its impact on survival has not been explored. This study evaluates HCMV expression in pediatric MB tissue and its association with molecular risk groups and survival.



 A retrospective study of pediatric MB cases (≤19 years) from 2007-2023 was conducted using the WHO 2021 classification, the Northcott brain-tumor classifier, and the SIOP-Europe/ERN PaedCan risk stratification. HCMV immediate-early (HCMV-IE) and late-antigen (HCMV-LA) expression were assessed by optimized immunohistochemistry, and whole-genome sequencing (WGS) data from 20 tumors were analyzed for viral gene expression.



 Forty-five patients (mean age 8.2 years; 22% ≤3 years) were included: WNT (18%), SHH TP53-wildtype (18%), and non-WNT/non-SHH (64%). 20% of MB belonged to low-risk, 33% to standard-risk, and 47% to high-risk (HR-MB) groups. Four tumors exhibited MYC/N amplification. HCMV-LA positivity was found in 84% of cases, with 53% showing high expression (≥25% of cells). Cox-regression identified HR-MB (HR = 4.197, p = 0.021) and high HCMV-LA (HR = 4.334, p = 0.027) as independent predictors of poor outcomes. WGS revealed UL88 as the most abundantly expressed HCMV gene(log2[TPM+1] ≈14–15), with maximal expression in Group 3 MB.



 This study provides the first evidence linking high HCMV-LA expression to adverse outcomes and high-risk molecular features in pediatric MB. UL88 emerged as the most strongly expressed HCMV gene across MB samples. These findings suggest a potential prognostic and therapeutic role for HCMV in MB, warranting validation in larger, prospective cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60feabdb3f3fd823a43f478c57c8aa48c2982d72" target='_blank'>
              High Human Cytomegalovirus antigen Expression in pediatric medulloblastoma tissue is associated with poor event-free survival
              </a>
            </td>
          <td>
            Maria F De la Cerda-Vargas, J. Schittenhelm, Martin Ebinger, J. Zipfel, R. Beschorner, Ghazaleh Tabatabai, Martin U Schuhmann
          </td>
          <td>2025-12-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f141cbdb131c2a8476f2ecad7ca548ca036f80" target='_blank'>
              The role of homologous recombination in human retrovirus-associated diseases.
              </a>
            </td>
          <td>
            Mohammad Mehdi Akbarin, Zahra Farjami, Gabriel Eduardo Acevedo-Jiménez, Cecilia Rodríguez Murillo, V. D. González-Fernández, Lucero de María Ávila-De la Vega, Hugo Ramírez Álvarez
          </td>
          <td>2025-12-20</td>
          <td>Virus genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Kaposi’s sarcoma–associated herpesvirus (KSHV) causes cancers for which no vaccine exists, and current treatments are often imperfect. Recent work shows that the virus and infected cells produce circular RNAs, which are stable RNA loops that can influence gene activity and build up during active infection. Because these molecules resist degradation and may travel inside virus particles, they can help track disease or shape how the virus behaves. This review explains how viral circular RNAs are produced, how researchers detect and assay them, and what is known about their roles during quiet and active stages of infection in the context of KSHV. It also outlines experimental tools to test what these RNAs do and how to target them. By organizing current knowledge and practical methods, this review aims to guide future studies and speed progress toward better diagnostics and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c49ebc6839a316e30284da0f7440e889a77090bc" target='_blank'>
              Recent Studies on Kaposi’s Sarcoma-Associated Herpesvirus Circular RNAs
              </a>
            </td>
          <td>
            Cristian J. Pagtalunan, Isadora Zhang, Ariella Turley, Fenyong Liu
          </td>
          <td>2025-11-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="   DNA oxidation is one of the main types of damage to the genetic material of living organisms. Of the many dozens of oxidative lesions, the most abundant is 8-oxoguanine (8-oxoG), a premutagenic base that leads to G→T transversions during replication. Double-stranded DNA can conduct holes through the π system of stacked nucleobases. Such electron vacancies are ultimately localized at the 5’-terminal nucleotides of polyguanine runs (G-runs), making these positions characteristic sites of 8-oxoG formation. While such properties of G-runs have been studied in vitro at the level of chemical reactivity, the extent to which they can influence mutagenesis spectra in vivo remains unclear. Here, we have analyzed the nucleotide context of G-runs in a representative set of 62 high-quality prokaryotic genomes and in the human telomere-to-telomere genome. G-runs were, on average, shorter than polyadenine runs (A- runs), and the probability of a G-run being elongated by one nucleotide is lower than in the case of A-runs. The re presentation of T in the position 5’-flanking G-runs is increased, especially in organisms with aerobic metabolism, which is consistent with the model of preferential G→T substitutions at the 5’-position with 8-oxoG as a precursor. Conversely, the frequency of G and C is increased and the frequency of T is decreased in the position 5’-flanking A- runs. A biphasic pattern of G-run expansion is observed in the human genome: the probability of sequences longer than 8–9 nucleotides being elongated by one nucleotide increases significantly. An increased representation of C in the 5’-flanking position to long G-runs was found, together with an elevated frequency of 5’-G→A substitutions in telomere repeats. This may indicate the existence of mutagenic processes whose mechanism has not yet been charac terized but may be associated with DNA polymerase errors during replication of the products of further oxidation of 8-oxoG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83075f585c67b259c09dcc5cab31e9fb64185e60" target='_blank'>
              DNA damage reflected in the evolution of G-runs in genomes
              </a>
            </td>
          <td>
            I. Grin, D. Zharkov
          </td>
          <td>2025-12-12</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, limiting the efficiency and precision of genome editing in many clinically relevant tissues. Here, we address this barrier by using induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. CRISPR editing outcomes differ dramatically in neurons compared to genetically identical dividing cells: neurons take longer to fully resolve this damage, and upregulate non-canonical DNA repair factors in the process. Manipulating this response with chemical or genetic perturbations allows us to direct DNA repair toward desired editing outcomes in nondividing human neurons, cardiomyocytes, and primary T cells. By studying DNA repair in clinically relevant cells, we reveal unforeseen challenges and opportunities for precise therapeutic editing. Genome editing tools can precisely introduce a specified lesion into the DNA, but ultimately rely on the cell’s DNA repair machinery to determine the editing outcome. Here, authors demonstrate how neurons’ unique DNA repair pathways impact the safety, efficiency, and precision of CRISPR edits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3be0abd5edf19f4fcddf4c37efa7ec4d61c80" target='_blank'>
              Characterizing and controlling CRISPR repair outcomes in nondividing human cells
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J. Namaganda, Manasi M. Kumar, J. Hamilton, Rohit Sharma, Karena G. Chow, Luke A. Workley, Bria L. Macklin, Mengyuan Sun, Alvin Ha, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Philip H. Dierks, Rudra Bose, Julianne Jin, Barbara S. Perez, Cindy R. Sandoval Espinoza, Madeline P. Matia, Serena H. Lu, Luke M. Judge, Brian R. Shy, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2025-11-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf04ff0b527b927234e0bc04a121079916f0a15" target='_blank'>
              HERV-R (ERV3-1) Env gene knockout reduces invasion, migration, and tumor growth in DLD1 colon cancer cells.
              </a>
            </td>
          <td>
            Min-Hye Kim, Eun-Ji Ko, Hyojin An, Seongsoo Choi, Heui-Soo Kim, S. Leem, H. Cha
          </td>
          <td>2025-11-20</td>
          <td>BMB reports</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f15103704f8e9b30f80ef5678e5b582040bdbc" target='_blank'>
              XChrom: a cross-cell chromatin accessibility prediction model integrating genomic sequence and cellular context
              </a>
            </td>
          <td>
            Yuanyuan Miao, Xuan Liang, Wenwen Zhang, Dongmei Han, Yurun Li, Zhen Wang
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common malignancy in women globally. The NOTCH signaling pathway is aberrantly activated in multiple tumors, and NOTCH1, its core transmembrane receptor, is highly expressed in cervical cancer. However, NOTCH1’s mechanisms in cervical cancer progression and radiotherapy resistance, as well as its interaction with key molecules, remain unclear. Methods This study explored the role of NOTCH1 in cervical cancer from multi-omics perspectives, including single-cell sequencing, cDNA microarrays, high-throughput sequencing, and immunohistochemistry, combined with a series of in vitro and in vivo experiments. Results NOTCH signaling activity was negatively correlated with the overall survival and recurrence-free survival of cervical cancer patients. As the core molecule of this signaling pathway, NOTCH1 was significantly highly expressed in cervical cancer tissues and promoted cervical cancer cell proliferation in vitro. Single-cell analysis revealed that NOTCH1 was relatively highly expressed in CPA6+ and CEL+ malignant cells and involved in cell cycle regulation; further cell cycle detection assays confirmed that NOTCH1 could promote the G1-S phase transition. In addition, patients with high NOTCH1 expression showed decreased plasma cell infiltration in the microenvironment. Cell communication analysis indicated that malignant cells with high NOTCH1 expression might lead to impaired plasma cell differentiation due to the impairment of the MIF ligand-receptor pathway. Finally, NOTCH1 could reduce the radiosensitivity of cervical cancer cells to radiotherapy both in vitro and in vivo; whereas has-miR-449a, as an upstream regulatory miRNA of NOTCH1, could inhibit cervical cancer cell proliferation and enhance radiosensitivity by inhibiting NOTCH1 expression. Conclusions This study clarifies NOTCH1’s role in promoting cervical cancer progression and reducing radiosensitivity, with has-miR-449a as a negative regulator, providing targets for optimizing cervical cancer radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e889481c892941401d223ac892318dcf009227f7" target='_blank'>
              Multi-omics reveals that NOTCH1 promotes cervical cancer progression and reduces radiosensitivity
              </a>
            </td>
          <td>
            Aihua Guo, Zhixiong Su, Enhuan Zhang, Huaqin Lin, Peilin Zhong, Zhiyuan Xie, Qingzhen Zhan, Ting Ye, Yang Sun
          </td>
          <td>2025-11-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Upon an organism’s death, enzymatic DNA repair ceases, exposing the genome to destructive factors such as free cellular nucleases and proliferating microorganisms, which can cause DNA loss. DNA preservation is highly dependent on environmental conditions, and less favorable environments accelerate degradation. Despite this, advanced extraction and analytical methods now enable the study of poorly preserved and degraded DNA. DNA typing is a foundation of forensic genomics, enabling the identification of individuals and the individualization of biological evidence through the generation of unique genetic profiles. Although DNA is relatively stable, environmental exposure initiates its degradation into progressively shorter fragments, complicating analysis. The extent of DNA preservation in biological evidence depends on numerous factors, and this review focuses on the environmental factors—including temperature, humidity, ultraviolet radiation, pH, chemical agents, and microbial activity—as the most influential variables. In samples with degraded DNA, the maximum amplicon length achievable through polymerase chain reaction (PCR) is inherently limited. This review discusses genetic markers and analytical strategies improvements that enable the examination of highly degraded samples, particularly when conventional short tandem repeat (STR) typing fails. In these situations, successful identification requires targeting short DNA fragments, which are more likely to persist. Single-nucleotide polymorphisms (SNPs) are a valuable alternative, as their high allelic variability and short amplicon requirements make them more amenable to amplification from fragmented templates than STRs. Advances in next-generation sequencing (NGS) technologies have further enhanced this capacity by enabling high-resolution SNP profiling, thereby improving outcomes in challenging forensic cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9e532e3edd771cebd0b6a4e276660662ebf439" target='_blank'>
              Analysis of Human Degraded DNA in Forensic Genetics
              </a>
            </td>
          <td>
            Irena Zupanič Pajnič
          </td>
          <td>2025-11-01</td>
          <td>Genes</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Organisms rely on coordinated networks of DNA repair pathways to protect genomes against toxic double-strand breaks (DSBs), particularly in germ cells. All repair mechanisms must successfully negotiate the local chromatin environment in order to access DNA. For example, nucleosomes can be repositioned by the highly conserved Nucleosome Remodeling and Deacetylase (NuRD) complex. In Caenorhabditis elegans, NuRD functions in the germline to repair DSBs - the loss of NuRD's ATPase subunit, LET-418/CHD4, prevents DSB resolution and therefore reduces fertility. In this study, we challenge germlines with exogenous DNA damage to better understand NuRD's role in repairing DSBs. We find that let-418 mutants are sensitive to cisplatin and hydroxyurea: exposure to either mutagen impedes DSB repair, generates aneuploid oocytes, and reduces fertility and embryonic survival. These defects resemble those seen when the Fanconi anemia (FA) DNA repair pathway is compromised, and we find that LET-418's activity is epistatic to that of the FA component FCD-2/FANCD2. We propose a model in which NuRD is recruited to the site of DNA lesions to remodel chromatin and allow access for FA pathway components. Together, these results implicate NuRD in the repair of both endogenous DSBs and exogenous DNA lesions to preserve genome integrity in developing germ cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77bfc2c8ad230781b0b0137c6deec8663495ed6" target='_blank'>
              NuRD chromatin remodeling contributes to repairing exogenous DSBs in the Caenorhabditis elegans germline.
              </a>
            </td>
          <td>
            Deepshikha Ananthaswamy, Kelin Funes, Thiago Borges, Scott Roques, Nina Fassnacht, Sereen El Jamal, P. Checchi, T. W. Lee
          </td>
          <td>2025-11-17</td>
          <td>Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract In bacteria, DNA gyrase plays a key role in cellular processes by affecting supercoiling. However, how gyrase localizes on the chromosome to affect supercoiling is unclear. Here, we interrogate the genome-wide distribution of gyrase in Streptococcus pneumoniae by mapping its cleavage sites in the presence of fluoroquinolones. Fluoroquinolones induce the formation of DNA—fluoroquinolone—topoisomerase complexes and generate detrimental double-stranded DNA breaks. Using a chromatin immunoprecipitation-sequencing-based method, we localize gyrase cleavage sites with single-nucleotide resolution. A total of 1517 sites were detected; the majority (92.7%) were located within protein-coding genes and were associated with high-level transcription. This trend was maintained at the level of transcriptional topological domains, where the most transcribed genes exhibited the highest frequency of cleavage sites, while the least transcribed genes exhibited the lowest frequency. We identified a four-base motif, GxxC, in 79% of the sites. The most frequent sequence was GATC (21.2% of all cleavage sites). GATC is the target of three pneumococcal restriction systems: DpnI, DpnII, and DpnIII. We demonstrated that methylation at GATC by the DpnII (GAmeTC) or DpnIII (GATCme) systems decreased gyrase cleavage and supercoiling activity. We hypothesized that strains with methylated GATC would be more resistant to subinhibitory concentrations of fluoroquinolones. We indeed demonstrated that gyrase mutants appeared more frequently in strains with GATC methylation (DpnII and DpnIII) than in strains without methylation (DpnI).This is significant for the evolution of fluoroquinolone resistance in S. pneumoniae, as the three Dpn phenotypes are present in the population of clinical isolates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4491c882122469de74b54cb35f779495e8828c2" target='_blank'>
              Distribution of DNA gyrase cleavage sites across the Streptococcus pneumoniae genome: relation to transcription and methylation at GATC sites
              </a>
            </td>
          <td>
            M. Ferrándiz, Pablo Hernández, A. G. de la Campa
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Multiplexed DNA FISH (also known as FISH Omics) is a powerful imaging technology to map three-dimensional (3D) genome organization in thousands of single cells at kilobase resolution. However, existing computational approaches to identify 3D genome features, such as A/B compartments, topologically associating domains (TADs) and chromatin loops, have been mainly adapted from Hi-C and do not account for the distinct measurement errors across the three XYZ physical axes. Here we present ArcFISH (accurate and robust chromatin analysis from FISH data), which explicitly models axis-specific measurement errors in a unified statistical framework to detect multi-level 3D genome features. Comprehensive evaluations using multiple simulated as well as experimental datasets show that ArcFISH reliably identifies chromatin loops under heterogeneous measurement errors, detects TAD boundaries highly enriched for active histone marks H3K4me3 and H3K36me3, and requires as few as 30 chromatin traces to faithfully assign A/B compartments. Applying ArcFISH to multiplexed DNA FISH data from mouse embryonic stem cells and mouse excitatory neurons reveals cell-type-specific chromatin loops and TADs, overlapping with differentially expressed genes. ArcFISH is freely available at https://hyuyu104.github.io/ArcFISH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9edbaba3bc8b0ff6aa7fa6a3e2e3e50014db57c6" target='_blank'>
              Accurate and robust 3D genome feature discovery from multiplexed DNA FISH
              </a>
            </td>
          <td>
            Hongyu Yu, Lingbo Zhou, Liangqi Xie, C. Strambio‐De‐Castillia, Ming Hu, Yun Li
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="The DNA Damage Repair (DDR) signaling pathway serves as a crucial molecular hub that regulates chemotherapy efficacy, offering significant translational value in the field of precision oncology. This review systematically analyzes the molecular mechanisms of five core DDR pathways (Homologous Recombination Repair, Non‐Homologous End Joining, Base Excision Repair, Nucleotide Excision Repair, and Mismatch Repair) in mediating chemotherapy resistance in tumors, and thoroughly elucidates the correlation between key molecular events—such as BRCA1/2 deficiency, MMR functional abnormalities, and Ataxia Telangiectasia Mutated/Ataxia Telangiectasia and Rad3‐related (ATM/ATR) signaling pathway dysregulation—and chemotherapy sensitivity. The DDR deficiency biomarker system established through the integration of multi‐omics data provides molecular classification tools for predicting the efficacy of platinum‐based drugs. This study focuses on the mechanism by which Poly ADP‐Ribose Polymerase inhibitors reverse Homologous Recombination‐Deficient tumor resistance through “synthetic lethality” effects while also revealing the synergistic anti‐tumor effects of ATM/ATR inhibitors in combination with chemotherapeutic agents. The research presents an innovative molecular synergy model between DDR regulation and Immune Checkpoint Blockade, confirming that tumor neoantigen release induced by DDR deficiency can enhance immunotherapy responses. This article also provides perspectives on multidimensional intervention strategies based on the DDR network, including the development of inhibitors targeting novel DDR targets, the establishment of DDR pathway functional assessment systems based on multidimensional biomarkers, and the investigation of synergistic paradigms between DDR and novel therapeutic modalities. Additionally, we explore the dynamic evolution mechanisms of DDR‐mediated chemotherapy resistance by analyzing the interactions between DDR and metabolic reprogramming, as well as other related processes. These breakthrough advances provide theoretical foundations and innovative directions for overcoming chemotherapy resistance and advancing personalized treatment, marking a new era in cancer therapy characterized by precision targeting of DDR pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/944e141ebbf24071dde5281d6930372251bae23d" target='_blank'>
              Translational Aspects of DNA Damage Repair in Optimizing Cancer Chemotherapy
              </a>
            </td>
          <td>
            Anqi Lin, Jinyue He, Aimin Jiang, Jian Zhang, Quan Cheng, Hengguo Zhang, Wenjun Mao, Peng Luo
          </td>
          <td>2025-12-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background CUB domain-containing protein 1 (CDCP1) is implicated in pancreatic ductal adenocarcinoma (PDAC) prognosis, but its relationship to the tumor microenvironment (TME) and oncogenic signaling remains incompletely defined. We hypothesized that CDCP1 expression is associated with hallmark cancer signaling pathways and transcriptionally inferred TME remodeling in PDAC. Methods We analyzed transcriptomic and clinical data from 214 PDAC cases (The Cancer Genome Atlas (TCGA), n = 145; GSE62452, n = 69). Patients were stratified into high CDCP1 and low CDCP1 groups based on the top tertile of expression. Immune and stromal components of the TME were quantified using the xCell algorithm. Gene Set Enrichment Analysis (GSEA) with Hallmark gene sets was used for pathway enrichment. Results High CDCP1 expression was significantly associated with reduced infiltration of CD8+ T cells, adipocytes, and fibroblasts, suggesting an immune-excluded and stromally depleted TME. It also correlated with increased homologous recombination deficiency scores, mutation burden, and single-nucleotide variants. CDCP1 expression correlated with CDKN2A mutation but was only weakly associated with KRAS, TP53, and SMAD4 alterations. GSEA showed consistent enrichment of proliferative (E2F, MYC, G2M, p53) and protumorigenic (transforming growth factor-β, hypoxia, glycolysis) pathways in high CDCP1 tumors across both datasets. Conclusion CDCP1 defines a transcriptionally distinct PDAC subtype characterized by immune evasion, stromal depletion, and genomic instability. These findings highlight CDCP1 as a potential therapeutic target and biomarker reflecting interplay between oncogenic signaling and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc74e64f603534d40445c3a46e26eba6ab33298" target='_blank'>
              High CDCP1 Expression Reflects Immune and Stromal Remodeling and Oncogenic Signaling in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Kentaro Miyake, M. Oshi, Akira Takenouchi, Kota Sahara, Jun Yamamoto, Yutaro Kikuchi, Y. Sawada, Y. Homma, R. Matsuyama, I. Endo
          </td>
          <td>2025-12-17</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10944325f830532ecfb5fb037e3e00b3a58c4a6" target='_blank'>
              Sema7a drives an immunosuppressive microenvironment of breast cancer via Kdm4a-mediated DNA replication regulation.
              </a>
            </td>
          <td>
            Xianan Bai, Shanshan Cai, Jie Jiang, Maojin Tian, Jianxin Du, Peiqing Zhao
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromatin stripes are architectural chromatin features in which a singular loop anchor interacts with a contiguous region of DNA so, at the bulk sequencing level, it appears as a long stripe on chromatin contact matrices. Stripes are thought to play an important role in gene regulation and have been implicated in regulating a cell's lineage determination. Therefore, integrated analysis of stripes with genomic and epigenomic features at a genome-wide scale shows vast potential in understanding the cooperation between regulatory elements in 3D nucleome. To this end, we present Quagga, a computational tool for detection and statistical verification of genomic architectural stripes from Hi-C or Micro-C chromatin contact maps, that relies on robust image processing techniques and rigorous statistical tests for enrichment. Quagga outperforms other stripe detection methods in accuracy and is highly versatile, working with Hi-C, Micro-C, and other chromatin conformation capture data. By reporting on all tools’ performance in classifying CTCF-cohesin anchored stripes, enhancer–promoter interacting stripes, and indeterminate stripes, we also demonstrate a thorough, integrated analysis to determine the output stripes’ quality. Our work provides a flexible and convenient tool to help scientists explore the relationships between chromatin architectural stripes and important biological questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806f320e619f5d811b2460f2b773780a1e56f123" target='_blank'>
              Statistically rigorous and computationally efficient chromatin stripe detection with Quagga
              </a>
            </td>
          <td>
            Fan Feng, Sean Moran, Anders Hansen, Xiaotian Zhang, Jie Liu
          </td>
          <td>2025-11-12</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f661ff00b14a8a4b6cfed87b54b789edd9cd47" target='_blank'>
              GenomoBase: an integrated resource for Genomoviridae genomics and metagenomics
              </a>
            </td>
          <td>
            Y. Cárdenas-Conejo
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Mitoxantrone (MX) is regularly used to treat several cancers. Despite its long history in the clinic, recent studies continue to unveil novel protein targets. These targets may contribute to the cytotoxic effects of the drug, as well as potential non-canonical antitumor activity. A better understanding of MX’s cellular targets is required to fully comprehend the molecular consequences of treatment and to interpret MX sensitivity in homologous recombination (HR)-deficient cancer. Methods: Here, we evaluated MX activity in HR-deficient UWB1.289 (BRCA1−) ovarian cancer cells and surveyed the binding profile of MX using TMT-labeled quantitative proteomics and chemoproteomics. Results: Mass spectrometry (MS) analysis of cellular extracts from MX-treated BRCA1−UWB1.289 cells revealed unique downregulation of pathways instrumental in maintaining genomic stability, including single-strand annealing. Moreover, the BRCA1− cells exhibited a significant upregulation of proteins involved in ribosome biogenesis and RNA processing. Additional MS analyses following affinity-purification using a biotinylated-mitoxantrone probe corroborated these findings, which showed considerable targeting of proteins involved in genome maintenance and RNA processing. Conclusions: Our results suggest that an interplay of both canonical and non-canonical MX-antitumor activity overwhelms the BRCA1− UWB1.289 cells. Furthermore, this study characterizes the target landscape of MX, providing insights into off-target effects and MX action in HR-deficient cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed8f08582f0dd8776423c5be2d239307bc42082" target='_blank'>
              Surveying the Proteome-Wide Landscape of Mitoxantrone and Examining Drug Sensitivity in BRCA1-Deficient Ovarian Cancer Using Quantitative Proteomics
              </a>
            </td>
          <td>
            Savanna Wallin, Sneha Pandithar, Sarbjit Singh, Siddhartha Kumar, Amar Natarajan, G. Borgstahl, Nicholas T. Woods
          </td>
          <td>2025-11-14</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aim. The current study aims to investigate changes in the expression of proprotein convertases (PCs) genes during the development of lung tumors. PCs is a family of highly specific serine endopeptidases of mammals that process precursors of various proteins and peptides. Nine genes encoding PCs have been identified in the human genome that are essential for normal body functioning. Additionally, PCs can activate proteins involved in carcinogenesis, as well as the expression levels of their genes were shown to correlate with tumor aggressiveness and patient survival. We previously evaluated the expression of all nine PC genes using quantitative PCR on paired samples of lung tumor and adjacent normal tissue. For the first time, we identified four distinct patterns of PCs gene expression change in tumor tissue, which we have called «scenarios». For three of them, covering more than two thirds of the samples, a dominant change in the expression of one PC gene in the tumor tissue was shown. These results may indicate the existence of a limited number of possible options for changes in PCs gene expression during the malignant transformation of lung cells. However, these results need to be confirmed using expanded cohort of tumor samples. Material and Methods. To confirm our previous findings, we analyzed the expression of PCs genes using modern methods of mathematical statistics and data from three previously published studies, which evaluated gene expression through high-throughput RNA sequencing in paired tumor and normal tissue samples from 194 patients with non-small cell lung cancer. Results. Our meta-analysis confirmed that the changes in PCs gene expression in lung tumor tissue compared to surrounding normal tissue follow a limited set of possible scenarios, each having a unique profile of PCs gene expression. Conclusion. The reasons for implementing each scenario may be linked to the origin of tumors, their mutation status, characteristics of the tumor microenvironment, and other factors. Correspondingly, these scenarios may correlate, for example, with tumor aggressiveness and resistance to therapy, and therefore may potentially be used to choose the treatment approach and/or to predict the course of the disease. However, this issue requires further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3af10872808404c4c77f8841898be6b45125c83" target='_blank'>
              Alterations in proprotein convertase genes’ expression during malignization of human lung cells have a limited set of scenarios
              </a>
            </td>
          <td>
            A. A. Komissarov, I. N. Vlasov, P. A. Slominsky, I. Demidyuk
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the 40 years since the discovery of the CENP proteins, many studies have examined the role of these proteins and their interactions with other chromosomal proteins of the centromere and beyond. Together, these studies have yielded vast amounts of sequencing and proteomics data. Typically, each study has focused on a single question and the majority of each dataset remains largely unexplored. Often the interesting details of publicly deposited data are left behind, buried in archives online, while more and more new data are generated. Reanalysing these databases can represent a new paradigm for investigating diverse biological pathways in unprecedented detail. Here, we explore two publicly available pan-cancer proteomic datasets to compare proteins whose abundance correlates with CENP proteins, with a particular focus on CENP-C. Our analysis confirms an expected link between CENP-C and cohesin levels but reveals a surprising and unexpected correlation between CENP-C and proteins of the inner nuclear membrane and the NuMA protein. This guilt-by-association analysis has the potential to identify proteins that act in common pathways but never associate or colocalize and may not even be expressed at the same time in cells. As an example, we show here that it can reveal unexpected links that expand our conception of centromeric chromatin beyond chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b131b447bc4a00bf7e5247f66ecd03a10d2ff6" target='_blank'>
              Sustainable integrative cell biology: CENP-C is guilty by association
              </a>
            </td>
          <td>
            N. Y. Kochanova, I. Samejima, William C. Earnshaw
          </td>
          <td>2025-11-25</td>
          <td>Chromosome Research</td>
          <td>2</td>
          <td>21</td>
        </tr>

        <tr id=" 
Chemical modifications of DNA and RNA play a pivotal role in the regulation of gene expression, enabling precise control of cellular functions and adaptation to changing environmental conditions. The most common modification in human DNA is cytosine methylation at the 5th position (5mC), while in RNA it is adenosine methylation at the N6 position (m⁶A), recognized as the predominant epitranscriptomic modification. Regulation of these modifications relies on the coordinated action of three groups of proteins: methyltransferases, demethylases, and reader proteins, which recognize modified nucleobases and recruit effector protein complexes, thereby translating the pattern of nucleic acid modifications into a specific biological response. Dysfunction of these protein groups leads to aberrant 5mC and m⁶A patterns, which play a crucial role in the pathogenesis and progression of many human diseases, including cancer. Deregulation of DNA and RNA methylation affects, among others, the control of genes involved in proliferation, apoptosis, and genome stability, which may promote tumor progression; moreover, the dynamic and reversible nature of these modifications makes them attractive diagnostic and therapeutic targets in cancer treatment.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47af133e2a4dd42a4d276d86f2bf75ce46e7e7b6" target='_blank'>
              Rola metylacji DNA i RNA w nowotworach: Mechanizmy i potencjał terapeutyczny leków epigenetycznych i epitranskryptomicznych
              </a>
            </td>
          <td>
            Julia Pisarek, Marta Koblowska
          </td>
          <td>2025-12-19</td>
          <td>Postępy Biochemii</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, , M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Purpose of Review Clonal hematopoiesis (CH) arises from the expansion of a single hematopoietic stem cell harboring somatic mutations that confer growth advantage. Recent studies highlight a substantial heritable component to CH, implicating germline mutations in DNA damage repair (DDR) genes. These genes are essential for maintaining genomic integrity and pathogenic variants in key DDR genes are well-established genetic underpinnings of several hereditary cancer syndromes. This review synthesizes current data linking germline DDR mutations – including ATM, CHEK2, TP53, PPM1D, BRCA1/2, and PARP1 – to CH and the development of myeloid malignancies. Recent Findings Emerging evidence suggests that germline perturbations in DDR pathway contribute to CH, though mechanisms remain incompletely defined. Large scale genome-wide association studies (GWAS) have identified strong associations between ATM and CHEK2 variants and CH. Assessing prevalence and CH risk in individuals with germline TP53 variants presents unique challenges, as distinguishing between somatic and constitutional lesions is often complex and requires careful tissue evaluation. The link between germline BRCA1/2 and CH remains inconclusive, confounded by concurrent diagnosis of solid malignancy and prior exposure to chemoradiation therapy in studied patient populations. Although germline mutations in PPM1D and PARP1 are rare, a potential germline predisposition to CH cannot be excluded. Summary The totality of current evidence suggests that germline DDR pathway mutations not only predispose to well-established solid malignancy syndromes but also to CH, which independently increases the risk of hematologic malignancies. Recognizing germline contributions to CH has broad implications for risk assessment, surveillance strategies, and development of preventive strategies in myeloid neoplasia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7301780c127edf4d26a740d2302af1182d86f99" target='_blank'>
              Impact of Germline DNA Repair Mutations on Clonal Hematopoiesis and Myeloid Neoplasm Development
              </a>
            </td>
          <td>
            Kateryna Fedorov, Leo Y. Luo, Alexander G. Bick, M. Savona
          </td>
          <td>2025-12-01</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181f022b28db4f3ed7a11dbea0e2ef041b80de4b" target='_blank'>
              Discovery of the TGCB microprotein translated from the lncRNA CHASERR
              </a>
            </td>
          <td>
            Bernardo Bonilauri, Xiaochun Yang
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="RecQ family of DNA helicases play pivotal roles in DNA replication, repair and responses to DNA damage or replication stress. Several human RecQ helicases are defective in diseases associated with chromosomal instability, premature aging and cancer. We recently discovered novel mutations in the RECQL4 gene in glioblastoma (GBM), the most malignant brain tumor in adults. Transcriptomic profiles of GBMs with REQCL4 mutations resembled those in REQCL4 KO glioma cells. We employ structural modelling and biochemical approaches to elucidate impacts of novel mutations on RECQL4 helicase activities. Using recombinant RECQL4P532S and RECQL4R766Q proteins we demonstrate that P532S substitution reduces the RECQL4 ability to unwind DNA and disrupts DNA-coupled ATP-hydrolysis activity. WT and mutated RECQL4 were overexpressed in RECQL4 KO glioma cells to study interactions with BLM helicases, cell viability and specific responses to UVC- and chemotherapy-induced DNA damage/repair. Overexpression of RECQL4P532S or RECQL4R766Q variants affected DNA repair and responses to chemotherapeutics in glioma cells, and RECQL4R766Q disturbed interactions with the BLM helicase. Our results reveal deleterious consequences of novel RECQL4 mutations in GBMs. The newly identified RECQL4 mutations affect RECQL4 helicases and their interactions with BLM contributing to glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650345dd7db490bb44b3ecdcc3b0ad865cd59a8" target='_blank'>
              Novel mutations in the RECQL4 gene affect its helicase functions, interactions with the BLM helicase and chemotherapeutics-induced cell death
              </a>
            </td>
          <td>
            Agnieszka Kaczmarczyk, Mikołaj Sokołowski, K. Wojnicki, Marta Pabiś, B. Wojtaś, I. Ciechomska, K. Poleszak, B. Gielniewski, S. K. Król, Matthew Guille, Sebastian Glatt, B. Kaminska
          </td>
          <td>2025-12-19</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4e250eb9e54e9ec9e1ccc32cfb46c720c85b6f" target='_blank'>
              Computationally-designed aptamers targeting RAD51-BRCA2 interaction impair homologous recombination and induce synthetic lethality.
              </a>
            </td>
          <td>
            Giulia Milordini, E. Zacco, Mirco Masi, Alexandros Armaos, Francesco Di Palma, Michele Oneto, Martina Gilodi, Jakob Rupert, Laura Broglia, Giulia Varignani, Marco Scotto, Roberto Marotta, S. Girotto, A. Cavalli, G. Tartaglia
          </td>
          <td>2025-12-01</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Type II Topoisomerases (topo II) are critical to simplify genome topology during transcription and replication. They identify topological problems and resolve them by passing a double-stranded DNA segment through a transient break in another segment. The precise mechanisms underpinning topo IIs ability to maintain a topologically simple genome are not fully understood. Here, we investigate how binding kinetics affects the resolution of two distinct forms of topological entanglement: decatenation and torsional relaxation. First, by single-molecule measurements, we quantify how monovalent cation concentration affects the dissociation rate of topo II from DNA. Second, we discover that increasing dissociation rates accelerate decatenation while slowing down relaxation catalytic activities. Finally, by using molecular dynamics simulations, we uncover that this opposite behaviour is due to a trade-off between search of target through facilitated diffusion and processivity of the enzyme in catenated versus super-coiled DNA. Thus, our findings reveal that a modulation of topo II binding kinetics can oppositely regulate its topological simplification activity, and in turn can have a significant impact in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02756b92213fd9ac89f480ffc7b784d0e58cb003" target='_blank'>
              Binding Kinetics Oppositely Regulates type II Topoisomerase Relaxation and Decatenation Activities
              </a>
            </td>
          <td>
            Cleis Battaglia, Filippo Conforto, Yair Augusto Gutiérrez Fosado, Matt Newton, E. Cutts, Davide Michieletto, Antonio Valdés
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction DNA double-strand breaks (DSBs) are repaired via homologous recombination (HR) or the more error-prone non-homologous end joining (NHEJ). breast cancer gene 1 (BRCA1) and breast cancer gene 2 (BRCA2) are key genes in HR, and their mutations are associated with aggressive prostate cancer (PCa). While PARP inhibitors (PARPi) improve survival in BRCA-mutated PCa, their efficacy in late-stage disease is limited and often accompanied by serious side effects. This study aims to develop an in vitro model of BRCA-mutated PCa and evaluate the therapeutic potential of DNA-dependent protein kinase (DNA-PK) inhibitors that target the NHEJ pathway. Methods The genes BRCA1 and BRCA2 were targeted for knockout (KO) in lymphnode cancer of the prostate (cell line) [LNCaP] using clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 technology. The KO were assessed via Western blot analysis. Scramble LNCaP, BRCA1 KO, and BRCA2 KO cells were treated with the PARPi talazoparib, the DNA-PK inhibitor nedisertib and their combination. The impact on cell proliferation was evaluated using the CellTiter-Glo assay and synergy of the treatments was analyzed with SynergyFinder. Cytotoxic effects were measured by flow cytometry using an Annexin V-fluorescein isothiocyanate (FITC) apoptosis detection kit. The presence of DSB was quantified using immunofluorescence. Results BRCA2 and BRCA1 protein expression were successfully downregulated in the knockout (KO) cell lines. After two days of treatment with talazoparib and/or nedisertib, a significant decrease in cell proliferation was observed. Additive effects of the combination treatment were detected exclusively in the BRCA KO cells. These cells also exhibited significantly higher rates of necrosis after treatment compared to scramble cells and, DNA DSB were significantly more prevalent in the BRCA KO cells. Additionally, BRCA1/2 loss is inversely correlated with DNA-PK, with knockout leading to increased DNA-PK expression to support NHEJ. Conclusion BRCA knockout LNCaP models were established, exhibiting increased DNA-PK activity and indicating greater susceptibility to DNA-PK inhibition. Therapeutically targeting NHEJ presents a promising approach in treating BRCA-mutated PCa. Further in vivo investigations are required to assess the tolerability of this drug combination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a995f399e397271dd4a847fb2e46b35d96ad921b" target='_blank'>
              Combined DNA-PK and PARP Inhibition as a Therapeutic Strategy in BRCA-Mutated Prostate Cancer: An in Vitro Pilot Study
              </a>
            </td>
          <td>
            T. Scherer, S. Salemi, Valentin Baumgartner, Dominik Enderlin, Alekhya Mazumdar, Daniel Eberli
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="An R-loop is a three-stranded nucleic acid structure that serves as a transcriptional intermediate, consisting of an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA). Small RNAs are RNA molecules shorter than 300 nucleotides that perform a wide range of essential functions within cells. Both R-loops and small RNAs are widely present in the genomes of prokaryotes and eukaryotes, where they play crucial roles in regulating gene expression, maintaining genomic stability, and facilitating DNA damage repair. Aberrant formation and accumulation of R-loops, coupled with dysregulation of small RNA pathways, can induce DNA damage and genomic instability, ultimately contributing to cellular senescence or cell death. Here, we discuss recent advances in understanding the crosstalk between R-loops and small RNAs, with a focus on their synergistic roles in maintaining genome stability and their therapeutic potential in oncology and neurodegeneration. We propose a novel model integrating R-loop dynamics with small RNA-mediated epigenetic regulation, supported by emerging clinical trial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853eaea8f06e1b31e5bea0e5a97d17746ae77587" target='_blank'>
              R-loops and small RNA regulatory interactions: mechanisms and clinical perspectives
              </a>
            </td>
          <td>
            , Ya-Ni Liu, Si-Qiao Zhao, Wen Liu, Jin Hu, Fan Yang, Sheng Wang
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Transposable elements comprise over 50% of the human genome, yet their role in chromatin organization is insufficiently studied. This study was motivated by the hypothesis that transposable elements drive chromatin contacts through homotypic coupling—that is, pairs of identical TEs physically pull genomic regions together. Methods: Analyzing public Micro-C and Hi-C datasets, I compared focal contact areas that stand out from low backgrounds against contact-depleted regions at kilobase resolution. Results: I discovered that transposable elements show enrichment at these focal contact points and even stronger depletion in contact-poor regions. Ancient mammalian transposable element families (MIR, L2) preferentially form homotypic pairs at chromatin contacts, while young primate-specific families (Alu, SVA) actively avoid homotypic pairing. The depletion of homotypic pairs in contact-poor regions exceeded their enrichment at focal contacts, suggesting that homotypically coupled transposable elements may be sequestered in protein-protected compartments. Unexpectedly, sequence-unrelated families like MIR and L2 showed similarly strong pairing behavior, indicating a mechanism independent of DNA sequence similarity. While my data demonstrates clear homotypic specificity patterns reproducible across cell types and individuals, it cannot distinguish whether transposable elements actively drive chromatin contacts or are passive markers of chromatin states. Conclusions: Study findings reveal previously uncharacterized patterns of transposable element organization at chromatin contact and suggest that homotypic pairing may provide anchoring points for coherent chromatin folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8affccf1386987c38a82b88deb3bb60212d9d82d" target='_blank'>
              Homotypic Transposable Element Pairing May Drive Coherent Chromatin Folding
              </a>
            </td>
          <td>
            Max Myakishev-Rempel
          </td>
          <td>2025-12-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a widely occurring malignancy with significant mortality on a global scale, making up close to 10% of all diagnosed cancers in 2020. While traditional CRC diagnostics and research have focused on nuclear genomic alterations, emerging evidence has highlighted the multifaceted roles of mitochondrial DNA (mtDNA) in the pathogenesis and clinical management of CRC. In this review, we examine three interlaced aspects of mtDNA in CRC: (1) Liquid biopsy biomarkers: Cell-free mtDNA circulating in the blood serving as a minimally invasive diagnostic and monitoring tool; (2) Nuclear mtDNA (NUMT)-driven genomic instability: The somatic nuclear incorporation of mtDNA (NUMT segments, or NUMT), contributing to mutational burden and chromosomal disruption in tumours; and (3) Mitochondria-encoded micropeptide signaling - small peptides encoded by the mtDNA that modulate cellular pathways and tumour behaviour. Recent high-impact studies have demonstrated that tumour-derived mtDNA in biofluids can augment cancer detection sensitivity, although technical challenges remain due to mtDNA fragmentation and background noise. Meanwhile, genomic analyses have uncovered a significant increase in NUMT insertion events in CRC cells, linking mitochondrial genome escape to nuclear genome instability and identifying potential numtogenesis suppressor genes. A novel dimension of mito-nuclear interactions in cancer was discovered in mitochondrial microproteins, such as humanin and mitochondrial open reading frame of the 12S rRNA type-c. Humanin exhibits both tumour-promoting and cytoprotective properties under specific conditions, while mitochondrial open reading frame of the 12S rRNA type-c possesses tumour-suppressive activities under other conditions. The outcomes of clinical, mechanistic, and translational research, revealing how mtDNA-based biomarkers and involvement contribute towards early detection, prognostication, and treatment of CRC, are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba21880b5e33a449e69dca805fc3ec140c49eb10" target='_blank'>
              Interlaced roles of mitochondrial DNA in colorectal cancer: Liquid-biopsy biomarkers, nuclear mtDNA-driven genomic instability, and mito-encoded micro peptide signaling
              </a>
            </td>
          <td>
            T. Koo, X. Leong, Y. Lee, Firdaus Hayati, A. Zakaria
          </td>
          <td>2025-12-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Nuclear Factor I (NFI) proteins are involved in adenovirus DNA replication and regulate gene transcription, stem cell proliferation, and differentiation. They play key roles in development, cancer, and congenital disorders. Within the NFI family, NFI-X is critical for neural stem cell biology, hematopoiesis, muscle development, muscular dystrophies, and oncogenesis. Here, we present the structural characterization of the NFI transcription factor NFI-X, both alone and bound to its consensus palindromic DNA site. Our analyses reveal a MH1-like fold within NFI-X DNA-binding domain (DBD) and identify crucial structural determinants for activity, such as a Zn²⁺ binding site, dimeric assembly, and DNA-binding specificity. Given the ~85% sequence identity within the NFI DBDs, our structural data are prototypic for the entire family, a NFI Rosetta Stone that allows decoding a wealth of biochemical and functional data and provides a precise target for drug design in a wider disease context. The Nuclear Factor I (NFI) family includes transcription factors key to development, cancer, and genetic disorders. Here, the authors present the structure and DNA-binding properties of NFI-X, a prototype for the entire NFI family.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09b9f75cb385c6026fbb070d6c58ca1ca1038c7" target='_blank'>
              Structural basis of Nuclear Factor 1-X DNA recognition provides prototypic insight into the NFI family
              </a>
            </td>
          <td>
            Michele Tiberi, M. Lapi, L. Gourlay, A. Chaves-Sanjuan, Maurizio Polentarutti, N. Demitri, Miriam Cavinato, Diane Marie Valérie Jeanne Bonnet, Valentina Taglietti, Anna Righetti, Rachele Sala, S. Cauteruccio, Amit Kumawat, R. Russo, A. Barbiroli, N. Gnesutta, C. Camilloni, M. Bolognesi, Graziella Messina, Marco Nardini
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="OBJECTIVE
to establish the level of chromosome aberrations in human peripheral blood lymphocytes during thedevelopment of a tumor induced bystander effect over several mitotic divisions.


MATERIALS AND METHODS



MATERIAL
human peripheral blood lymphocytes; human nonsmall cell lung cancer A549cell culture.


METHOD
cytogenetic analysis of uniformly stained human peripheral blood lymphocyte chromosomesto determine the frequency of all types of aberrations.


RESULTS
In human peripheral blood lymphocytes after longterm (120hour) cultivation with the addition of conditioned medium from human nonsmall cell lung cancer A549 cells, an increase in the level of chromosome aberrations was registered compared to the control: from 1.90 to 3.40 per 100 cells in the first mitosis; from 2.00 to 3.60per 100 cells in the second; from 1.70 to 3.10 per 100 cells in the third; and from 1.50 to 2.80 per 100 cells in thefourth cell division (p < 0.05). The increase in chromosomal instability occurred due to an increase in the frequency of single fragments: from 0.70 to 2.20 per 100 cells in the first mitosis; from 1.00 to 2.10 per 100 cells in thesecond; from 0.90 to 2.00 per 100 cells in the third; and from 1.10 to 2.40 per 100 cells in the fourth cell division(p < 0.05). The frequencies of chromosomal type aberrations (acentric paired fragments, abnormal monocentrics)showed no significant difference from the corresponding control levels (p > 0.05) during the observation period.Dicentric chromosomes were not registered.


CONCLUSIONS
Persistence of genomic instability due to the development of a tumor induced bystander effect was established in peripheral blood lymphocytes of healthy individuals after four cell divisions of cultivation with a conditioned medium from human nonsmall cell lung cancer A549 cells. Its cytogenetic manifestation was an increase in the frequency of chromosome aberrations due to chromatidtype damage. The frequencies of chromatidtypeaberrations (single fragments) exceeded the corresponding controls and did not differ significantly between mitotic divisions. The development of the tumor induced bystander effect did not influence the induction of chromosomal type aberrations (acentric fragments, abnormal monocentrics, dicentric chromosomes), whose levels did not differ significantly from the controls.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343107654c073187c208193c0627e4bb6f4e512a" target='_blank'>
              PERSISTENCE OF GENOMICAL INSTABILITY DUE TO THE DEVELOPMENT OF THE TUMOR-INDUCED BYSTANDER EFFECT IN HUMAN SOMATIC CELLS.
              </a>
            </td>
          <td>
            O. Shemetun, O. A. Talan, O. B. Dibska
          </td>
          <td>2025-12-01</td>
          <td>Problemy radiatsiinoi medytsyny ta radiobiolohii</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ada3f386540f45c703b805477be0ad3ac9ca8a" target='_blank'>
              CircTarget: a comprehensive database of circRNA-target RNA interactions across multiple cell types.
              </a>
            </td>
          <td>
            Hongmei Zhang, Yuyang Zhang, Peng Li, Zhen Yuan, Wanglong Liu, Jinyue Zhang, Ruitian Li, Chen Lu, Ping Zhu, Yuanchao Xue
          </td>
          <td>2025-12-31</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Merkel cell carcinomas (MCCs) are rare, highly aggressive skin cancers with poor outcome due to early lymphatic tumor spread and frequent recurrences. MCCs mostly occur in the head and neck and are mainly caused by an infection with the Merkel cell polyomavirus (MCPyV). Increasing evidence suggests that Piwil-2 and small non-coding PIWI-interacting RNAs (piRNAs) play an important role in solid malignancies and we thought that this might also be the case for MCCs. Therefore, Piwil-2 expression was first evaluated in 27 MCC specimens and correlated with oncological outcome. We found an association with high Piwil-2 expression and advanced tumor stage, MCPyV positivity and poor outcome. Next, we utilized siRNAs for Piwil-2 knock-down in MCC cells. Downregulation of Piwil-2 caused a significant change of 202 different piRNAs and 419 proteins. Interestingly, proteins related to viral driven MCC pathways (TRRAP, BRD8, PRIM2, ORC4) were significantly downregulated. Moreover, there was a moderate cell cycle arrest of cells in the G0/G1-phase, as well as a significant upregulation of SOX-2, a key regulator of Merkel cells. Altogether, Piwil-2 poses a poor prognosticator in MCCs, which is linked to MCC oncogenesis and SOX-2. Further research is needed to better understand underlying mechanisms and to prove their clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558526f13e11a497bc4a8d819813ba94d26aaab9" target='_blank'>
              Piwil-2 represents a poor prognosticator in Merkel cell carcinomas that regulates oncoproteins, cell cycle arrest and SOX-2 expression
              </a>
            </td>
          <td>
            S. Janik, Johannes Pammer, Elisabeth Simader, Ulana Kotowski, S. Grasl, R. Houben, D. Copic, M. Mildner, Goran Mitulovic, Martin Bilban, S. Derdak, Markus Unterwurzacher, Klaus Kratochwill, B. Erovic
          </td>
          <td>2025-11-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251a4fb00acc97a635e84c6467a865b534993fc8" target='_blank'>
              PRC2 Restricts Malignant Peripheral Nerve Sheath Tumorigenesis in a Genetically Engineered Mouse Model of MPNST
              </a>
            </td>
          <td>
            Woo Hyun Cho, Amish J. Patel, Makzhuna Khudoynazarova, Sara Warda, Juan Yan, Dan Li, Dana M. Schoeps, Eve Fishinevich, Mohini R Pachai, Cindy J. Lee, Zoe Jacobs, Kae Kristoff, Jessica Sher, C. Antonescu, Yu Chen, Ping Chi
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>138</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [24, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>