<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-18 09:33:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>77</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>348</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>307</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a common form of oncogene amplification in aggressive cancers. The frequency and diversity of ecDNA has been catalogued in adult and some childhood cancers; however, its role in most pediatric cancers is not well-understood. To address this gap, we accessed large pediatric cancer genomics data repositories and identified ecDNA algorithmically using cloud computing. This retrospective cohort comprises 2,968 patients covering all major childhood solid tumor types. Aggressive tumor types had particularly high incidences of ecDNA. Pediatric patients whose tumors harbored extrachromosomal DNA had poorer five-year overall survival than children whose tumors contain only chromosomal amplifications. We catalogue known and potentially novel oncogenes recurrently amplified on ecDNA and show that ecDNA often evolves during disease progression. These results highlight patient populations that could potentially benefit from future ecDNA-directed therapies. To facilitate discovery, an interactive catalogue of ecDNA in childhood cancer is available at https://ccdi-ecdna.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b9576b9fdef51f927edfe8dbc69e8e7f1f4c7f" target='_blank'>
              Extrachromosomal DNA associates with poor survival across a broad spectrum of childhood solid tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Eugene Yui-Ching Chow, Rishaan Kenkre, Jonathan Kirkland, Aditi Dutta, Shanqing Wang, Wenshu Zhang, Miguel Brown, J. Luebeck, Yan Yuen Lo, Elias Rodriguez-Fos, A. Henssen, K. Okonechnikov, David R. Ghasemi, K. Pajtler, Daisuke Kawauchi, V. Bafna, Megan Paul, Kevin Yip, Jill P. Mesirov, Lukas Chavez
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) of chromosomal origin is commonly present in all eukaryotic organisms and tissue tested so far. EccDNA populations exhibit immense diversity and a characteristically low degree of overlap between samples, suggesting low inherence of eccDNA between cells or a deficiency the methods by which eccDNA is detected. This study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap. We optimized the protocol by reducing the time. Linear DNA is digested by increasing exonuclease V activity. We employed CRISPR-Cas9 for mitochondrial linearization, which proved superior to restriction enzymes. A key finding is the critical role of random hexamer primer concentration and genomic DNA input in Rolling Circle Amplification (RCA) for generating high-quality long amplicons from eccDNA (concatemeric tandem copy, CTC), essential for confident de novo eccDNA construction from long-read sequencing data. Lower primer concentrations substantially increased the percentage of CTC-derived eccDNA and improved the overlap of identified eccDNAs in technical replicates. Applying this revisited approach to human myeloma and breast cancer cell lines, as well as xenograft models, demonstrated that the optimized conditions enhanced the overlap of detected eccDNA up to over 50% overlap which substantially improved over previous studies (less than 1%). Moreover, Additionally, the oncogenic signature of eccDNAs can be identified across all replicates. These findings provide guidelines for developing standardized procedures for eccDNA profiling, advancing our understanding of eccDNA biology and its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extremely Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Increases in the copy number of large genomic regions, termed amplifications, are an important adaptive strategy for many organisms. Numerous amplifications across the AT-rich Plasmodium falciparum genome contribute directly to drug resistance or impact the fitness of this protozoan parasite. During the characterization of malaria parasites selected with a dihydroorotate dehydrogenase (DHODH) inhibitor that targets pyrimidine biosynthesis, we detected increased copies of a genomic region that encompassed 3 genes (~ 5 kb) including GTP cyclohydrolase I (GCH1 amplicon). While amplification of this gene is reported in antifolate-resistant parasites, GCH1 amplicons had not previously been implicated in DHODH inhibitor resistance. Results Here, we explored the expansion of the GCH1 locus in this family of parasite lines using long-read sequencing and single-read visualization. We directly quantified higher numbers of tandem GCH1 amplicons in selected parasite lines (up to 9 GCH1 amplicons) compared to parental P. falciparum parasites (strictly 3 GCH1 amplicons). Because each read represents DNA from an individual genome, we were able to appreciate hidden variation within a single parasite line (3, to 5, to 7 amplicons) that was not reflected in other DNA-based analysis methods. While all GCH1 amplicons shared a consistent structure, expansions arose in precise 2-unit steps within selected lines. We found conserved AT-rich sequences at amplicon boundaries, which is consistent with the Plasmodium model of CNV formation. Parasite lines with expanded GCH1 also had DHODH amplicons on a separate chromosome. When we evaluated prior DHODH inhibitor selections, we observed that GCH1 amplification was not required for resistance; however, selection outcomes suggest that pre-existing GCH1 amplicons may support amplification at the DHODH locus. Conclusions We identified previously undetected heterogeneity in gene copy number by viewing long pieces of DNA from individual genomes. This approach was possible due to the amplicon’s tandem orientation and relatively small size that can be spanned by a single long ONT read. The positive association between DHODH and GCH1 copy number, combined with the metabolic connection between P. falciparum pyrimidine and folate biosynthesis, justifies further investigation into the adaptive evolution of these two genomic loci. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11859-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df91538345e9b090469612002190dc6883763df6" target='_blank'>
              Direct long-read visualization reveals hidden variation in GCH1 gene copy number and precise expansion steps
              </a>
            </td>
          <td>
            Shiwei Liu, Julia Zulawinska, Emily R. Ebel, A. Luniewski, Charles Danis, Mary Lewis Simpson, Jane Kim, Nnenna Ene, Thomas Braukmann, Molly Congdon, Webster Santos, Ellen Yeh, J. Guler
          </td>
          <td>2025-07-17</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction while we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection.
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-11</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eeecebde718f44128a5af1add184f4e02b1d49" target='_blank'>
              NuMA promotes constitutive heterochromatin compaction by stabilizing linker histone H1 on chromatin
              </a>
            </td>
          <td>
            Yao Wang, Wenxue Zhao, Jiahao Niu, Cuifang Liu, Xiaotian Wang, Weihong Yuan, Shanshan Ai, Wolfgang Baumeister, Guohong Li, Aibin He, Peng Xu, Cheng Li, Yujie Sun
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-directed drugs are often clinically deployed without definitive understanding of their molecular mechanisms of action (MOA). Hypomethylating agents (HMAs), which result in the degradation of the DNA methyltransferase 1 (DNMT1), have been deployed for decades in the treatment of haematological malignancies1,2. The precise mechanism of action of these drugs, however, has been debated, rendering the design of rational combination therapies challenging. Here, we identified the deubiquitinating enzyme USP48 as a crucial regulator of posttranslational histone modification in the context of DNA demethylation. USP48 loss selectively enhances response to DNMT1 inhibition, leading to a rapid induction of cell death. We demonstrate that USP48 is localized at sites of DNA damage and deubiquitinates H2A variants and proteins important for DNA damage repair. Functionally, loss of USP48 triggers an increase in chromatin accessibility upon HMA treatment, rendering AML cells more susceptible to DNA damage. Our results support USP48 as a posttranslational histone modifier for chromatin stability and DNA damage in response to HMA-related DNA demethylation. These findings propose USP48 as a new target for combination therapy with HMAs for acute myeloid leukaemia (AML).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42e9d360cbac606431a70f9cd70adaccce3729a4" target='_blank'>
              Chromatin Perturbation Promotes Susceptibility to Hypomethylating Agents
              </a>
            </td>
          <td>
            C. Schneider, G. Alexe, L. Merickel, Ashleigh Meyer, Michelle L. Swift, Rodolfo B. Serafim, A. Basanthakumar, A. Taillon, S. Salhotra, Fabio Boniolo, Sander Lambo, Yara Rodriguez, Björn Häupl, Rani E. George, D. E. Root, T. Oellerich, Volker Hovestadt, Dipanjan Chowdhury, K. Stegmaier
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0407eee77f76a83d31f10dc4bc4ede91f52ba5a" target='_blank'>
              Modelling Acral Melanoma in Admixed Brazilians Uncovers Genomic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Annie Cristhine, Moraes Sousa-Squiavinato, Sara Santos Bernardes, Flavia C. Aguiar, Antonio C. Facciolo, Martín del Castillo, Velasco Herrera, J. Rene, C. Wong-Ramirez, P. Basurto-Lozada, Aretha Brito Nobre, Geethanjali Annamalai, Rebecca Martins Cadimo do Nascimento, Jacqueline M Boccacino, Rafaela Fagundes, Pedro Sodré, R. Barros, Mariana de Moraes Pitombo, R. Olvera-León, Larissa Satiko Alcantara, Sekimoto Matsuyama, J. Billington, I. Vermes, Irving Simonin-9 Wilmer, Danielle G. Carvalho, João Pedro, Cavalcante Simoes, Priscila Valverde Fernandes, Luiz Fernando, Nunes, Andreia Cristina, de Melo, Jadivan Leite de Oliveira, M. Herlyn, Andrew E. Aplin, Daniela Robles-Espinoza, David J. Adams, P. Possik
          </td>
          <td>2025-08-12</td>
          <td>None</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Objectives: To investigate the clinical diagnostic and prognostic value of extrachromosomal circular DNA (eccDNA) in breast cancer, eccDNA profiles were constructed for 81 breast cancer tumor tissues and 33 adjacent non-tumor tissues. Methods: The distribution characteristics of eccDNA across functional genomic elements and repetitive sequences were systematically analyzed. Furthermore, a diagnostic model for differentiating malignant and normal breast tissues, as well as a prognostic prediction model, was developed using a random forest algorithm. Results: EccDNA in breast cancer tissues harbor a higher proportion of functional elements and repetitive sequences, with their annotated genes significantly enriched in tumor- and immune-related pathways. However, no significant differences in eccDNA features were observed across breast cancer subtypes or pathological stages. In the validation cohort, the eccDNA-based diagnostic model achieved an AUC of 0.83, with repetitive elements and enhancer-associated features contributing the most to diagnostic performance. The prognostic model achieved an AUC of 0.78, with repetitive element annotations also showing strong prognostic relevance. Conclusions: These findings highlight the promising potential of eccDNA in the development of precision diagnostics and prognostic systems for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f033fc855abaceeac39bacf8c5fcd1d56a705c9" target='_blank'>
              A Study on the Diagnostic and Prognostic Value of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Fuyu Li, Wenxiang Lu, Lingsong Yao, Yunfei Bai
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA damage from routine cellular processes or exogenous insults can have a lasting impact on gene regulation beyond genetic mutations. The prevailing paradigm for the consequences of DNA damage repair revolves around restoration of the original genetic sequence, but long-term changes in chromatin configuration, gene expression and DNA modifications have not been analyzed. We introduced numerous, simultaneous Cas9-mediated DNA double strand breaks (DSBs) at defined locations in human glioblastoma cells and tracked both non-genetic and genetic alterations over time. Megabase-scale genomic alterations that endured two weeks after the initial damage were detected, involving a shift from transiently increased intra-TAD interactions to persistent long range cis and trans contacts, alterations in gene-expression and associated large structural variations. These findings reveal that widespread DNA damage, such as chemotherapy or radiotherapy, can trigger long-term genetic and non-genetic modifications which alter cellular function and may impact tumor outcome and the emergence of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Mammalian genomes are subdivided into euchromatic A compartments that contain mostly active chromatin, and inactive, heterochromatic B compartments. However, it is not well understood how A and B genome compartments are established and maintained. Here we study SMCHD1, an SMC-like protein best known for its role in X chromosome inactivation, in human male myoblasts. SMCHD1 colocalizes with Lamin B1 and the heterochromatin mark H3K9me3. Loss of SMCHD1 leads to extensive heterochromatin and Lamin B1 depletion at the nuclear lamina, acquisition of active chromatin states and increased DNA methylation along chromosomes. In absence of SMCHD1, long range intra-chromosomal contacts between B compartments are lost while many new TADs and loops are formed. Inactivation of SMCHD1 promotes numerous B to A compartment transitions accompanied by activation of silenced genes. The data suggests that SMCHD1 functions as an anchor for heterochromatin domains at the nuclear lamina ensuring that these domains are poorly accessible to DNA methyltransferases and to epigenome modification enzymes that typically operate in active chromatin. Thus, the properties of SMCHD1 in heterochromatin maintenance extend well beyond its role in X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d576a82dc3af4516675e7e1ac94325c2cc7c05" target='_blank'>
              SMCHD1 maintains heterochromatin, genome compartments and epigenome landscape in human myoblasts
              </a>
            </td>
          <td>
            Zhijun Huang, Wei Cui, Ishara Ratnayake, Kristin Gallik, Lorna Cohen, Rabi Tawil, Gerd P Pfeifer
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Copy number variants (CNVs) are duplications and deletions of DNA sequences that contribute to genetic variation between individuals and underlie rapid adaptive evolution. Increases in gene copy number can confer a strong selective advantage in some circumstances, but CNVs can also incur fitness costs. Prior research has shown that experimental evolution of Saccharomyces cerevisiae in nutrient-limited chemostats recurrently selects for amplifications of nutrient transporter genes, including GAP1, MEP2, and PUT4 in glutamine, ammonium and proline limited conditions, respectively. However, the fate of these CNVs upon return of the organism to a non-selective environment, is unknown. To investigate the fitness costs and stability of CNVs upon reversal of the original selection pressure, we studied 15 unique CNV lineages that had been selected in different nitrogen-limited chemostats, including both segmental amplifications and whole-chromosome aneuploidies. We studied the stability of CNVs using a fluorescent CNV reporter system during propagation in nutrient-rich media using serial dilution of batch cultures for 110-220 generations. We found that only one-third of CNV lineages repeatedly underwent loss of CNVs and reversion to a single-copy genotype. All lineages containing aneuploidies showed rapid reversion dynamics, whereas lineages containing segmental amplifications were remarkably stable – only one of 11 strains reverted to a single copy. Pairwise competitive fitness assays revealed strong fitness defects associated with reverting CNVs in the nutrient-rich environment but minimal fitness defects in lineages that did not revert. Reversion of CNVs led to increased fitness. Using simulation-based inference to estimate reversion rates and fitness effects we find that negative selection is the primary driver of CNV loss although aneuploid reversion rates are high with estimates of 10 −5 to 10 −3 . Whole-genome sequencing of clones isolated from evolved populations revealed that reversion of both aneuploids and a segmental amplification left no evidence of prior existence of the CNV, making the genomes of revertants indistinguishable from the single-copy ancestor. Our findings provide novel evidence of the low fitness cost and high stability of most segmental CNVs upon removal of the selection pressure, and that gene amplifications with large fitness costs are readily reversible, highlighting the significance of CNVs for genome evolution as well as rapid and reversible adaptation to transient selection pressures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478ede766edac8c6526ad5e7502f78feea49ddaf" target='_blank'>
              Segmental copy number amplifications are stable in the absence of selection
              </a>
            </td>
          <td>
            Titir De, Nadav Ben Nun, P. Spealman, Ina Suresh, Grace Avecilla, Farah Abdul-Rahman, Yoav Ram, David Gresham
          </td>
          <td>2025-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d14e9bed5ce451868baa1e549e50257d339b27d" target='_blank'>
              Transposable elements hitchhike on Starships across fungal genomes
              </a>
            </td>
          <td>
            Hanne Griem-Krey, Julia de Fraga Sant’Ana, Ursula Oggenfuss, Y. P. Calegari-Alves, Ana Luiza Marques, Markus Berger, L. Santi, W. O. Beys-da-Silva, M. Habig
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Summary Complex chromosomal rearrangements in tumors involve intricate genomic alterations that significantly affect gene function and contribute to cancer development. Identifying these events is crucial for cancer research but is often challenging due to the complexity and limitations of existing tools. We developed the Complex Chromosomal Rearrangements Resolver (CCRR), a comprehensive, reproducible, and user-friendly platform for analyzing complex rearrangements in tumors. CCRR integrates multiple SV and CNV detection tools within a Docker container environment, simplifying installation and configuration. It can be easily deployed, automating the execution and merging of results, providing high-confidence consensus SV and CNV calls, allowing researchers to efficiently analyze complex chromosomal rearrangements in tumors without extensive bioinformatics expertise. CCRR also includes a web server for one-click analysis and customized visualization. Availability and implementation The CCRR platform is freely available at https://www.ccrr.life. Source code and executables can be accessed at https://github.com/laslk/CCRR. An archived version is available at Zenodo: https://doi.org/10.5281/zenodo.15386513.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad11c6be118606cfac854968e8601d07763e154" target='_blank'>
              CCRR: a user-friendly platform for analyzing complex chromosomal rearrangements in tumors
              </a>
            </td>
          <td>
            Jinjiang Liu, Kun Wang, Yawen Yuan, Guangchao Bao, Hang Ci, Mingqin Liu, Yunpan Lyu, Jingxin Tang, Jian Yang, Haoyang Cai
          </td>
          <td>2025-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The strength of the repressive histone H3 lysine 27 trimethylation modification signal varies drastically at individual silencers. Focusing on cases of an unusually strong repressive signal in regions that we refer to as super-silencers, we demonstrate that the regions that become B-cell super-silencers are originally associated with gene upregulation during development, and their target genes are highly expressed in stem cells, especially during early developmental stages. About 13% of B-cell super-silencers transmute to super-enhancers in B-cell lymphoma and 22% of these conversions recur across more than half of patients. Notably, genes associated with these conversions, like BCL6 and BACH2, are downregulated more swiftly than others when subjected to JQ1, a super-enhancer-disrupting bromodomain and extra-terminal domain inhibitor utilized in cancer chemotherapy. Furthermore, super-silencers are characterized by an over-representation of B-cell-cancer-associated mutations, both somatic and germline, and B-cell-cancer translocation breakpoints. This surpasses the prevalence found in other regulatory elements, such as CTCF binding sites, underlining the crucial role of super-silencers in forming and stabilizing regulatory topologies in standard B cells. For example, over 80% of cases involving the B-cell-lymphoma translocation t(3;14)(q27;q32) fuse super-silencers in the BCL6 locus with enhancer-rich domains. Finally, we demonstrate that the repressive mechanisms of super-silencers are partially governed by the CpG content in their sequences. While CpG-rich super-silencers often prevent promoters from interacting with enhancers, CpG-depleted super-silencers typically suppress the chromatin looping of nearby enhancers. In summary, our findings accentuate the critical role super-silencers play in the normal function of B-cells, suggesting that sequence mutations and activity modifications in these elements could be primary factors in B-cell carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12eb40d352b5dd3f41e425053f82e55565a55cc" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith N Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation – DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b0e24503d3afcd35a7b4c4c2e06e9d2060436" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT KRAS is mutationally activated in 45-50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRAS G12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, Jinru Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/929755ae2e9de92ac1b7d8df8a259bc5e4e63f92" target='_blank'>
              Hicberg: Reconstruction of contact signals from repeated elements
              </a>
            </td>
          <td>
            Sébastien Gradit, Samuel Ortion, Pauline Larrous, Maëlys Delouis, Romain Koszul, A. Cournac
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Summary DNA damage can result from external sources or occur during programmed genome rearrangements in processes like immunity or meiosis. To maintain genome integrity, cells activate DNA repair pathways that prevent harmful outcomes such as cancer or immune dysfunction. In this study, we uncover a novel role for DNA damage during the terminal differentiation of multiciliated cells (MCCs). MCCs, which line the airways, reproductive tracts, and brain ventricles, produce hundreds of motile cilia, each anchored by a centriole. Therefore, MCCs must generate hundreds of centrioles during differentiation. Normally, centriole duplication is tightly linked to the S and G2 phases of the cell cycle, raising questions about how MCCs override numerical and temporal restrictions on centriole duplication. We find that differentiating MCCs accumulate extensive double-stranded DNA breaks during centriole amplification, with damage levels correlating with centriole number. DNA damage response (DDR) kinases are essential for supporting centriole biogenesis and ciliogenesis. We also observe that transcriptional activity, required for the expression of centriole and cilia genes, produces RNA-DNA hybrids (R-loops) that co-localize with DNA damage. This suggests that transcription-coupled DNA damage helps initiate a pseudo–cell cycle program, permitting centriole amplification without triggering full S/G2 phase processes. Our findings indicate that MCCs harness DDR signaling as part of their developmental program, revealing a broader principle in which the canonical cell cycle is adaptively rewired during differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/025db960c1d15e49e5330c45782572ca46dc80ab" target='_blank'>
              The DNA damage response pathway is required for multiciliated cell differentiation
              </a>
            </td>
          <td>
            Cayla E. Jewett, Andrew J Holland, Chad G Pearson
          </td>
          <td>2025-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The identification of CENPA, CENPB, and CENPC by Earnshaw and Rothfield 40 years ago has revealed the remarkable diversity and complexity of centromeres and confirmed most seed plants and animals have centromeres comprised of complex satellite arrays. The rapid evolution of centromeres and positive selection on CENPA and CENPC led to the centromere drive model, in which competition between tandem satellite arrays of differing size and centromere strength for inclusion in the egg of animals or megaspore of seed plants during female meiosis drives rapid evolution of centromeres and kinetochore proteins. Here we review recent work showing that non-B-form DNA structures in satellite centromeres make them sites of frequent replication fork stalling, and that repair of collapsed forks by break-induced replication rather than unequal sister chromatid exchange is likely the primary mode of satellite expansion and contraction, providing the variation in satellite copy number that is the raw material of centromere drive. Centromere breaks at replication, rather than errors at mitosis, can account for most centromere misdivisions that underlie aneuploidies in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fb737fe39c4474778577289e08e6983374bf35" target='_blank'>
              Centromeres drive and take a break
              </a>
            </td>
          <td>
            P. Talbert, S. Henikoff
          </td>
          <td>2025-08-04</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Significance Modified cytosine bases are essential components of mammalian genomes, but how they are distributed in double-stranded DNA remains unclear. Here, we report a method to simultaneously sequence the two most abundant modified bases—5-methylcytosine and 5-hydroxymethylcytosine (hmC)—across both strands of genomic DNA. Applying this method to DNA from mouse embryonic stem cells, we find that at CpG sites, hmC forms different combinations with cytosine variants across the DNA double helix, exhibiting unique distributions at enhancers and gene bodies, and distinct relationships with transcription. Our findings demonstrate that the double-stranded context of hmC is a key feature of the epigenome, revealing a further layer of regulatory information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8822428faae43abf3590008b224a007bfe559580" target='_blank'>
              SCoTCH-seq reveals that 5-hydroxymethylcytosine encodes regulatory information across DNA strands
              </a>
            </td>
          <td>
            Jack S Hardwick, Somdutta Dhir, Angie Kirchner, A. Simeone, Sean M. Flynn, J. M. Edgerton, Rafael de Cesaris Araujo Tavares, Isabel Esain-Garcia, D. Tannahill, Pauline Y. Golder, Jack M Monahan, Walraj S. Gosal, Shankar Balasubramanian
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Tumors of infectious origin globally represent 13%. Oncogenic DNA viruses such as human papillomavirus (HPV), hepatitis B virus (HBV), and Epstein-Barr virus (EBV) are responsible for approximately 60% of these tumors. These oncoviruses are extensively studied to understand their role in cancer development, particularly through viral genome integration into the host DNA. Retroviruses require integration mediated by viral integrase for persistence, whereas DNA oncoviruses do not need integration for replication; instead, integration occurs incidentally. This process often targets fragile sites in the human genome, causing structural rearrangements that disrupt genes, activate proto-oncogenes, and increase genomic instability, all contributing to tumorigenesis. Integration near promoter regions and active genes is closely linked to carcinogenesis, highlighting its importance in developing diagnostic and therapeutic strategies. This review summarizes viral integration's role in oncogenesis, mechanisms of integration, and methods to study this process, focusing on DNA tumor viruses such as HBV, EBV, HPV, and Merkel cell polyomavirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef1eb5cd84b08745894e948edf6d9875f648ea7" target='_blank'>
              Genome integration of human DNA oncoviruses.
              </a>
            </td>
          <td>
            Zuzana Vojtěchová, R. Tachezy
          </td>
          <td>2025-07-23</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            Zhi-cheng Huang, , Qian Liu, Ruo-Dan Chen, Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved – specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4849479727b73a833dd0ff467a2ebd5a79ba5e8" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of the both variants of each isolate were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a phage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variantsh.
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-14</td>
          <td>PLoS pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="CIC::DUX4 translocation-positive sarcomas (CDS) are rare, highly proliferative tumors associated with chemotherapy resistance. Without a clear understanding of the molecular pathways driving CDS, no effective treatment regimens have been developed. Here, we identify protection against DNA damage through upregulation of POLE and DNA repair pathways specific to CDS. These data may explain the high proliferation rate and chemotherapy resistance of these tumors and suggest targeted treatment strategies for CDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c1daa0e52dfd54fcd1b1c744a6ff9aaea6aa92" target='_blank'>
              Upregulation of POLE and proficient DNA repair are features of CIC::DUX4 sarcomas
              </a>
            </td>
          <td>
            Nathan D. Seligson, Anjali Paragji, Feha Shahalam, Binyam Yilma, James L Chen
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, Natalia Y Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c0eb75e7f5a0febc3a26c4b5b85dcca6192f2" target='_blank'>
              Validation of the MAGIC genomic signature on RNAseq in soft-tissue sarcomas
              </a>
            </td>
          <td>
            A. Benhaddou, G. Pérot, P. Rochaix, T. Valentin, G. Ferron, F. Chibon
          </td>
          <td>2025-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The capacity of a plasmid to express genes is constrained by its length and copy number. However, the interplay between these parameters and their constraints on plasmid evolution have remained elusive due to the absence of comprehensive quantitative analyses. Here, we present ‘Pseudoalignment and Probabilistic Iterative Read Assignment’ (pseuPIRA), a computational method that overcomes previous computational bottlenecks, enabling rapid and accurate determination of plasmid copy numbers at large scale. We apply pseuPIRA to all microbial genomes in the NCBI RefSeq database with linked short-read sequencing data (4644 bacterial and archaeal genomes including 12,006 plasmids). The analysis reveals three scaling laws of plasmids: first, an inverse power-law correlation between plasmid copy number and plasmid length; second, a positive linear correlation between protein-coding genes and plasmid length; and third, a positive correlation between metabolic genes per plasmid and plasmid length, particularly for large plasmids. These scaling laws imply fundamental constraints on plasmid evolution and functional organization, indicating that as plasmids increase in length, they converge toward chromosomal characteristics in copy number and functional content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d0a945dacbeb8801a3cb57a4c7f7eb8870d06a" target='_blank'>
              Scaling laws of bacterial and archaeal plasmids
              </a>
            </td>
          <td>
            Rohan Maddamsetti, Irida Shyti, Maggie L. Wilson, Hye-in Son, Yasa Baig, Zhengqing Zhou, Jia Lu, Lingchong You
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary This study examined unclassified chromosomal abnormalities (UnCA) in patients with Hodgkin’s lymphoma (HL) before, during, and after anticancer treatment. HL is a rare cancer affecting 2–4 people per 100,000 individuals annually. Although treatment is highly effective, survivors face an increased risk of developing other cancers. In this study, we analyzed blood samples from HL patients and survivors at different stages of treatment, focusing on changes in their chromosomes that do not fit standard categories. These changes include alterations in chromosomes and nuclei. We found that UnCA levels were higher in HL patients than in healthy individuals at all stages; the general UnCA frequency decreased during treatment but increased again one year after treatment, at the expense of two specific UnCAs, free chromatin and micronuclei clusters. Given their persistence and association with genomic instability, these UnCAs might serve as early indicators of secondary cancers in HL survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb5a19d761a4166c2c377dcfbbe9dbb9f00192f1" target='_blank'>
              Unclassified Chromosomal Abnormalities as an Indicator of Genomic Damage in Survivors of Hodgkin’s Lymphoma
              </a>
            </td>
          <td>
            S. Ramos, B. Molina, María Del Pilar Navarrete-Meneses, D. Cervantes-Barragan, Valentín Lozano, S. Frias
          </td>
          <td>2025-07-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Transposable elements (TEs) are dynamic repetitive regions constituting 50% of the genome. In adult cancers, TEs contribute to carcinogenesis through structural rearrangements that disrupt tumor suppressors and by onco-exaptation, which drives aberrant oncogene expression. TEs are characterized in adult malignancies, with epithelial cancers harbouring significantly more new insertions. However, the role of TEs in pediatric cancer is unknown. Many pediatric cancers arise from predisposition, such as Li-Fraumeni Syndrome, which is caused by germline TP53 (gTP53) variants. TP53 suppresses TEs by binding L1 elements to inhibit transcription and by inducing apoptosis in cells with high retrotransposition. Moreover, adult cancers with somatic TP53 variants harbour significantly more TEs than wildtype tumors. To investigate the pediatric context, we conducted a pan-cancer analysis of germline and tumor TE landscapes in pediatric patients with (n = 66 germline; n = 40 tumor) and without (n = 465 germline; n = 276 tumor) gTP53 variants. Germline TEs were identified using MELT, xTEA, and INSurVeyor, while tumor TEs were called with xTEA and TotalReCall. TEs were consolidated across individuals using JASMINE and annotated with AnnotSV. We excluded 21% of germline and 2% of tumor TEs classified as common, defined as present in over 3% of gnomAD or pediatric non-cancer controls (n=189). Overall, 11% of pediatric tumors harbored at least one L1 insertion, with a median of 0 insertions per tumor, substantially lower than the 21% reported in adult cancers using the same TE callers (Solovyov et al). Frequency in germline or tumor TEs did not vary significantly by tumor type, age of onset, sex, prior treatment, or disease state. Germline and tumor TEs were found in 35 and 147 cancer genes respectively. Tumor TEs significantly affected many cancer pathways, including those involved in mitosis. While no structural variants (SVs) involved TEs at both breakpoints, 171 SVs in six samples were associated with a TE at one breakpoint. Importantly, both germline and tumor TE burden did not differ between gTP53 carriers and wildtype individuals, nor did the number of evolutionarily young or full-length elements. Pathway enrichment analyses revealed no significant differences in TE-affected pathways between gTP53 and wildtype groups. However, a gradient-boosted tree model trained on TE distribution across genomic windows predicted gTP53 status with an AUPRC of 0.77, suggesting that gTP53 variants shape the germline TE landscape at a positional level rather than through global frequency changes. In summary, pediatric cancers exhibit a quiescent TE landscape, with low insertion rates across tumor types. gTP53 variants do not alter the germline or tumor TE frequency,but influence the location of germline insertions. TE analysis in pediatric cancer and gTP53 carriers will enhance our understanding of tumorigenesis, informing future treatment approaches.



 Brianne Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, Mehdi Layeghifard, Adam Shlien, Vallijah Subasri, David Malkin. Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba8aeceb59adfe162dadfd953074aed2a705a85" target='_blank'>
              Abstract A039: Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, M. Layeghifard, A. Shlien, Vallijah Subasri, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Repetitive elements account for a large proportion of the human genome and undergo alterations during early tumorigenesis. However, the exclusive fragmentation pattern of DNA-derived cell-free repetitive elements (cfREs) remains unclear. Methods This study enrolled 32 healthy volunteers and 112 patients with five types of cancer. A novel repetitive fragmentomics approach was proposed to profile cfREs using low-pass whole genome sequencing (WGS). Five innovative repetitive fragmentomic features were designed: fragment ratio, fragment length, fragment distribution, fragment complexity, and fragment expansion. A machine learning-based multimodal model was developed using these features. Results The multimodal model achieved high prediction performance for early tumor detection, even at ultra-low sequencing depths (0.1×, AUC = 0.9824). Alu and short tandem repeat (STR) were identified as the primary cfREs after filtering out low-efficiency subfamilies. Characterization of cfREs within tumor-specific regulatory regions enabled accurate tissue-of-origin (TOO) prediction (0.1×, accuracy = 0.8286) and identified aberrantly transcribed tumor driver genes. Conclusion This study highlights the abundance of repetitive DNA in plasma. The innovative fragmentomics approach provides a sensitive, robust, and cost-effective method for early tumor detection and localization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f71e0eea2828ba986a7986cc3d935c9f50cf8f6" target='_blank'>
              Comprehensive fragmentation of cell-free repetitive DNA for enhanced cancer detection in plasma
              </a>
            </td>
          <td>
            Mingguang Zhang, Shuohui Dong, Wei Rao, Shiwen Mei, G. Hu, Ling Liu, Zhen Wang, Jianqiang Tang
          </td>
          <td>2025-07-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer arises from a convergence of genetic mutations and epigenetic alterations. Among the consequences of epigenetic deregulation is the aberrant activation of transposable elements, particularly retrotransposons, the most expansive and functionally impactful class. This review examines the paradoxical role of retrotransposons in cancer, emphasizing their dual capacity to support oncogenesis while simultaneously triggering innate and adaptive immune responses that may limit tumor growth. A central question is discussed: Is retrotransposon activation merely a stochastic byproduct of epigenetic instability, or could it represent an intrinsic layer of early tumor surveillance that inadvertently turns against the host as malignancy progresses? Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00368-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a2bf5f051e3bd6c0fa28e6e14cd088f5983472" target='_blank'>
              A plan or pandemonium? The conundrum of retrotransposon activation in cancer
              </a>
            </td>
          <td>
            Elina Zueva
          </td>
          <td>2025-08-09</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Human development and health depend on the precise expression of relevant genes in specific cells, at precise times and in response to appropriate stimuli. This is known as context-dependent gene regulation and relies on the activities of a diverse 'zoo' of DNA elements within the genome that are collectively called context-dependent cis-regulatory elements (cdCREs). cdCREs may comprise as much as 10% of the genome and include better-known sequences such as enhancers, silencers and promoters that form the basis of complex multicellularity. Diverse vertebrate body plans not only share considerable phenotypic similarities but also cell types, the genes they express and, in a growing number of cases, in the function and nucleotide sequence of cdCREs. The current review will critically evaluate current methodologies to identify cdCREs and re-evaluate a place for comparative genomics amongst them. We will also explore the function of cdCREs and discuss methods of analysing their function in disease-associated physiologies and behaviours using in vivo models such as CRISPR-generated GA mouse lines. Finally, we will study the effects of epigenetic mechanisms such as DNA methylation on cdCRE activity and examine how genetics and epigenetics can interact to alter disease susceptibility. Given that genome-wide association studies (GWAS) suggest that 95% of disease-associated genomic variation reside in the 98% of the less understood noncoding genome, the need to understand the role of conserved vertebrate cdCREs in development and health in vivo has never been more pressing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382d555c7aff11257c9330d9a7647ebcfb9ae6bb" target='_blank'>
              Exploring the roles of conserved context-dependent cis-regulatory elements (cdCREs) in multicellularity, human health and disease.
              </a>
            </td>
          <td>
            Andrew McEwan, Alexander Rattray, Greg Hutchings, Elizabeth A. Hay, Chris Murgatroyd, A. MacKenzie
          </td>
          <td>2025-07-04</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background The selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well defined. Potential DNA replication origins are licensed through the recruitment of a pair of minichromosome maintenance complexes (MCMs). In yeast, a subset of MCMs is selected for initiation by the SLD3–SLD7 firing factors during G1. Like in yeast, excessive numbers of MCM complexes are loaded onto chromatin during G1 in human cells, but it is unclear how MCMs are selected for firing. Results We examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast replication initiation factors Sld3 and Sld7. TRESLIN and MTBP binding patterns are detectable in a G1 synchronized population and have a higher binding signal compared to asynchronously cycling cells. MTBP binds to early-mid replicating regions in an early S population. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. We show that TRESLIN and MTBP binding during G1 does not require origins licensed with loaded MCMs. Conclusions We provide evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins. MTBP binding location and footprint during S phase differs from that seen in G1, implicating two separate modes of binding. These data highlight binding mechanisms for DNA replication initiation factors in human cells that have diverged from those shown in yeast, suggesting differences in origin selection. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03665-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b74245a8a7e7194eb9255132a18f436ef65a462" target='_blank'>
              Cell cycle-dependent TICRR/TRESLIN and MTBP chromatin binding mechanisms and patterns
              </a>
            </td>
          <td>
            Tyler D. Noble, Courtney G. Sansam, Kimberlie A. Wittig, Blanka Majchrzycka, Christopher L. Sansam
          </td>
          <td>2025-07-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="With age, clonal expansions occur pervasively across normal tissues yet only in rare instances lead to cancer, despite being driven by well-established cancer drivers. Characterization of the factors that influence clonal progression is needed to inform interventional approaches. Germline genetic variation influences cancer risk and shapes tumor mutational profile, but its influence on the mutational landscape of normal tissues is not well known. Here we studied the impact of germline genetic variation on clonal hematopoiesis (CH) in 731,835 individuals. We identified 22 new CH-predisposition genes, most of which predispose to CH driven by specific mutational events. CH-predisposition genes contribute to unique somatic landscapes, reflecting the influence of germline genetic backdrop on gene-specific CH fitness. Correspondingly, somatic–germline interactions influence the risk of CH progression to hematologic malignancies. These results demonstrate that germline genetic variation influences somatic evolution in the blood, findings that likely extend to other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcdcfe442326883ec1ff1d1b7880290f079cb82" target='_blank'>
              Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy
              </a>
            </td>
          <td>
            Jie Liu, Duc Tran, Liying Xue, B. Wiley, C. Vlasschaert, Caroline J. Watson, Hamish A J MacGregor, X. Zong, I. Chan, Indraniel Das, M. Uddin, A. Niroula, Gabriel Griffin, Benjamin L Ebert, T. Mack, Y. Pershad, B. Sharber, Michael F. Berger, A. Zehir, R. Ptashkin, Ross L. Levine, E. Papaemmanuil, V. Joseph, Teng Gao, Y. Kemel, D. Mandelker, Konrad H Stopsack, Paul D. P. Pharoah, Semanti Mukherjee, Li Ding, Yinyin Cao, MJ Walter, Jamie R Blundell, Nilanjan Chatterjee, K. Offit, Lucy A. Godley, Daniel C. Link, Z. Stadler, Alexander G. Bick, P. Natarajan, Kelly L Bolton
          </td>
          <td>2025-07-15</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays a significant role in cancer development. However, our understanding of its role in normal cells is limited. This study aimed to explore the roles of eccDNA in the differentiation of human bone marrow mesenchymal stem cells (BMSCs). Methods Through circular DNA sequencing (Circle-seq) and computational analysis, we documented eccDNAs in human BMSCs and their differentiation into osteoblasts, adipocytes, and chondrocytes. Next, the assay for transposase-accessible chromatin with high throughput sequencing (ATAC-seq) and RNA sequencing (RNA-seq) data were integrated with Circle-seq data. The roles of phosphatidylinositol 4-kinase alpha (PI4KA) and eccDNA as functional enhancers in human BMSC differentiation were assessed in vitro. Results Our results demonstrate that eccDNA is common in human BMSCs. In the differentiated groups including osteoblasts, adipocytes and chondrocytes, eccDNA-encoded genes presented higher expression compared to the background. We found eccDNA displayed enhanced chromatin accessibility in human BMSCs, which contribute to increased transcription of genes, such as runt-related transcription factor 2 (RUNX2), a key transcription factor for BMSC osteogenesis. We further found PI4KA, an eccDNA-encoded gene, promoted osteogenic differentiation of human BMSCs via the PI3K/AKT/mTOR pathway. Finally, we demonstrated eccDNA as functional enhancers to regulate BMSC differentiation in a dose-dependent manner. Conclusions Our study provides the evidence of eccDNA regulating BMSC differentiation and highlights the roles of eccDNA as transcription template and enhancer in normal cells, which will facilitate future research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04516-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9298ef8d097b3fec00680e4438db26b85d21b6" target='_blank'>
              Extrachromosomal circular DNAs in the differentiation of human bone marrow mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuxi Gu, Yidan Song, Shuhua Wang, Jun Liu
          </td>
          <td>2025-07-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic repetitive elements are essential structural and functional components of most eukaryotes and have attracted the attention of researchers in recent years. Despite the well-documented features of the karyotype of the tapeworm species Caryophyllaeus laticeps, knowledge about the content and distribution of repetitive elements on its chromosomes is limited. To gain a more comprehensive understanding of the composition of the repetitive DNA, we used a combination of bioinformatic, cytogenetic and molecular techniques to characterise the repetitive DNA content of this species. RepeatExplorer2 analysis revealed that transposable elements are predominantly present in the C. laticeps genome. The most suitable repeat families were selected for fluorescence mapping on the chromosomes, and most of them showed a scattered hybridisation pattern. In this study, we identified three new chromosomal markers: small nuclear U1 DNA (snDNA), the 5S ribosomal DNA (5S rDNA) and a satellite DNA named ClatSat14-167, each of which hybridises to a single locus at different chromosomal locations, allowing accurate identification of three pairs of chromosomes. These markers could be very valuable tools for future evolutionary studies and greatly enrich the limited cytogenetic data for tapeworms. In addition, an increased frequency of chromosomal aberrations was detected, reaffirming previous findings on the possible adverse effects of environmental pollution on the chromosomes of the species studied. Supplementary Information The online version contains supplementary material available at 10.1007/s00436-025-08530-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4e24b509116b93b7418ab18f10e22249cbe4d2" target='_blank'>
              Discovery of new chromosomal markers through repeatome analysis of Caryophyllaeus laticeps (Caryophyllidea)
              </a>
            </td>
          <td>
            Anna Marková, M. Orosová
          </td>
          <td>2025-07-18</td>
          <td>Parasitology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Sporadic colorectal cancer (CRC), the third leading cause of cancer-related death globally, arises through a continuum from normal tissue to adenomas, progressing from low-grade (LGD) to high-grade dysplasia (HGD); yet, the early epigenetic drivers of this transition remain unclear. To investigate these events, we profiled LGD and HGD adenomas using EM-seq, and identified a consensus differential methylation signature (DMS) of 626 regions through two independent bioinformatics pipelines. This signature effectively distinguished LGD from HGD in both tissue and plasma-derived cell-free DNA (cfDNA), highlighting specific methylation patterns. Functional annotation indicated enrichment for regulatory elements associated with transcription factor activity and cell signaling. Applying the DMS to the TCGA CRC dataset revealed three tumor subtypes with increasing hypermethylation and one normal cluster. The most hypermethylated subtype exhibited poor survival, high mutation burden, and disrupted transcriptional networks. While overlapping with classical CpG Island Methylator Phenotype (CIMP) categories, the DMS captured a broader spectrum of methylation alterations. These findings suggest that the DMS captures functionally relevant, antecedent epigenetic alterations in CRC progression, enabling the robust stratification of dysplasia severity and tumor subtypes. This signature holds promise for enhancing preclinical detection and molecular classification, and warrants further evaluation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b0ef917c9876876a5f66fb35ed8c368fc721c" target='_blank'>
              Multiomics Signature Reveals Network Regulatory Mechanisms in a CRC Continuum
              </a>
            </td>
          <td>
            Juan Carlos Higareda-Almaraz, Francesco Mattia Mancuso, P. Canal-Noguer, Kristi Kruusmaa, Arianna Bertossi
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Somatic mutations are desirable targets for selective cancer elimination, yet most are found within noncoding regions. We have adapted CRISPR-Cas9 as a novel, cancer-specific killing strategy by targeting the subset of somatic mutations that create protospacer adjacent motifs (PAMs), which have evolutionally allowed bacterial cells to distinguish between self and non-self DNA for Cas9-induced double strand breaks (DSBs). We hypothesized that somatic 5’-NGG-3’ PAMs arising from base substitutions (e.g. NCG→NGG) could serve as CRISPR-Cas9 targets for selective cancer elimination through the induction of multiple DSBs. Whole genome sequencing (WGS) of paired tumor-normal (T-N) samples from three PC patients were put through PAMfinder (software for somatic PAM discovery). On average, 417 somatic PAMs per PC case were found (>99% of them in non-exons) and 97% of the selected subset were Sanger confirmed. We then analysed T-N samples from ICGC which revealed medians of 455 somatic PAMs per tumor in two PC cohorts (N=174 combined), 2790 in lung (N=29), and 3236 in esophageal (N=388) cancer cohorts. We performed γH2AX staining on PC cell lines transduced with 4-9 sgRNAs designed from our PAM discovery approach, and observed more foci with increased number of sgRNAs in the target cell line but not in the non-target cell line. Deep sequencing and WGS on clones transduced with cancer-specific sgRNAs also demonstrated on-target activity in the target cell line but not in either the patient’s normal cells or an irrelevant cancer cell line. No off-target events were detected. To test selective toxicity in vitro and in vivo, we transduced 4-9 sgRNAs into PC co-cultures and found 69-99% selective cell death of three target cell lines through flow cytometry. Finally, using mouse xenograft models, 92% weight reduction (N=10, P=0.0004) of tumors transduced with a pool of 9 cancer-specific sgRNAs was achieved compared to tumors transduced with non-targeting sgRNAs (N=8).



 Selina Shiqing K. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, Michael Goldstein, Nicholas J. Roberts, and James R. Eshleman. CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b50e15f1763d831356c9974d95d4708f8806a3e" target='_blank'>
              Abstract LT02: CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers
              </a>
            </td>
          <td>
            S. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, M. Goldstein, Nicholas J. Roberts, James R. Eshleman
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485aaa6c6ae7c10944eff4bdd9fd536b93adfcd9" target='_blank'>
              Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
              </a>
            </td>
          <td>
            R. Edahiro, Go Sato, Tatsuhiko Naito, Y. Shirai, R. Saiki, K. Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, S. Namba, Noa Sasa, Genta Nagao, Qingbo S. Wang, Yugo Takahashi, Takanori Hasegawa, T. Kishikawa, Ken Suzuki, Yu-Chen Liu, D. Motooka, Ayako Takuwa, Hiromu Tanaka, S. Azekawa, H. Namkoong, R. Koike, Akinori Kimura, S. Imoto, Satoru Miyano, Takanori Kanai, K. Fukunaga, Mamoru Uemura, T. Morita, Yasuhiro Kato, H. Hirata, Y. Takeda, Y. Doki, H. Eguchi, D. Okuzaki, Shuhei Sakakibara, Seishi Ogawa, Atsushi Kumanogoh, Yukinori Okada
          </td>
          <td>2025-07-28</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Neuroblastoma is a heterogeneous disease where patient stratification is critical for prognosis and treatment decisions and where it recently has been suggested that the presence of telomere maintenance mechanisms (TMM) should be considered in risk stratification. We investigated the utility of DNA methylation-based classification for neuroblastoma diagnostics by analysing 303 tumours samples from two cohorts. We show that of the total number of cases, an average of 90% of the samples classified as neuroblastoma, while 66% also achieved confident classification into the three NB subclasses: “MYCN-type”, “ALT/TERT TMM positive” and “TMM negative”. The tumours classified as MYCN-type showed genomic amplification of MYCN (MNA); however, some MYCN-type cases lacked evident MNA, suggesting that epigenetic states might be influenced by other factors such as activating ALK mutations. Survival analysis indicated similar poor survival probabilities for patients classified as TMM positive or MYCN type, distinct from the inferior survival of TMM-negative cases. All cases, with complementary genomic data available, associated with TMM positivity also presented features associated with telomere lengthening mechanisms, including TERT or ATRX alteration. However, some tumours positive for these features, especially TERT rearrangement, classified as MYCN type rather than TMM positive, indicating that MNA and other mechanisms introduce a methylation pattern that supersede or overlap with pattern imposed by TERT. Chromosomal copy number alterations (CNAs) characteristic of methylation subclasses were identified, including 1p deletion and 17q gain in MYCN type and combinations of 11q loss, 3p loss, 7q gain, and 17q gain in TMM-positive cases, highlighting the potential of the methylation arrays to replace SNP arrays for prognostic genomic assessments. Our study demonstrates that DNA methylation-based classification stratifies neuroblastoma into clinically relevant subgroups, aiding diagnostic and prognostic decisions, although discrepancy between genomic features and methylation classification does occur. The interplay between genomic alterations and methylation patterns could give clues into the discrepancy and underscores the complexity of neuroblastoma biology and the need for further research and validation of clinical outcomes of the patients in the respective subclasses. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01936-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f713d2afc90db793e049e383b6f7ea1cd34f79" target='_blank'>
              DNA methylation profiling for molecular classification of neuroblastoma
              </a>
            </td>
          <td>
            Maja Löfgren, A. Djos, Shiva Rezaei, Medha Suman, P. Kogner, Tommy Martinsson, S. Fransson, Helena Carén
          </td>
          <td>2025-07-27</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract The multitude of DNA lesion types, and the nuclear dynamic context in which they occur, presents a challenge for genome integrity maintenance as this requires the engagement of different DNA repair pathways. Specific “repair controllers” that facilitate DNA repair pathway crosstalk between double-strand break (DSB) repair and base excision repair (BER) and that regulate BER protein engagement at lesion sites have yet to be identified. Here, we find that DNA polymerase β (Polβ), crucial for BER, is ubiquitylated in a BER complex-dependent manner by TRIP12, an E3 ligase that partners with UBR5 to restrain DSB repair signaling. Furthermore, we find that TRIP12, but not UBR5, controls cellular levels and chromatin loading of Polβ. Required for Polβ foci formation, TRIP12 influences Polβ involvement after radiation-induced DNA damage, a process regulated by TRIP12-mediated ubiquitylation of Polβ. Notably, excessive TRIP12-mediated engagement of Polβ affects DSB formation and radiation sensitivity, underscoring its role in promoting precedence for BER over DSB repair. The herein discovered function of TRIP12, in the governance of Polβ-directed BER, supports a role for TRIP12 in assuring BER lesion removal at complex DSB sites to optimize DSB repair at the nexus of DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976cd6688072a28607e4b25adebc5f7fdda3bbf" target='_blank'>
              TRIP12’s role in the governance of DNA polymerase β involvement in DNA damage response and repair
              </a>
            </td>
          <td>
            B. Inanc, Qingming Fang, Wynand P. Roos, Joel F. Andrews, Xuemei Zeng, Jennifer Clark, Jianfeng Li, Nupur B. Dey, Md Ibrahim, Peter Sykora, Zhongxun Yu, Charlotte R Pearson, Andrea Braganza, Marcel Verheij, Jos Jonkers, Nathan A. Yates, C. Vens, Robert W. Sobol
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia.
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Background. Medulloblastoma (MB) prognosis and response to therapy depend largely on genetic changes in tumor cells. Many genes and chromosomal abnormalities have been identified as prognostic factors, including amplification of MYC oncogenes, gains in 1q and 17q, deletions in 10q and 21p, or isochromosomes 17 (i(17)(q10)). The frequency of these abnormalities varies greatly between ethnic populations, but the frequency of specific abnormalities, such as MYCC and MYCN amplification, 17q gain, and deletions, in the Russian population is unknown. Objective: The aim is to study the frequency of MYCC and MYCN amplifications, 17q gain, and 17p deletion and determine their prognostic value in Russian patients with MB. Methods. This study was performed on MB cells obtained from 18 patients (12 boys and 6 girls, aged between 3 months and 17 years, with a median age of 6.5 years). Determination of cytogenetic aberrations was carried out using FISH assays with MYCC-SO, MYCN-SO, and MYCN-SG/cen2 probes, as well as cen7/p53 dual color probes and PML/RARα dual color probes (Abbott Molecular, USA). One-way ANOVA and Fisher’s F-test were used to compare the two groups. The differences were considered significant when p < 0.05. Results. In 77.7% of patients (14/18), the classical type of MB was present; in 16.7% (3/18), desmoplastic type; and in 5.6% (1/18), nodular desmoplasic types of neoplasms. Amplification of MYC genes was detected in 22.2% of Russian patients (n = 4 out of 18). Patients with MYC amplification had the worst overall survival (OS: 0% vs. 68%, p = 0.0004). Changes on the 17th chromosome were found in 58.3% of patients. Deletion of 17p occurred in 23.1%, and gain of 17q occurred in 46.2%. There were no significant differences in OS, clinical signs, or the presence of additional 17q material or 17p deletion among patients with MB. Conclusions: Amplification of the MYC gene is a predictor of poor overall survival to therapy and a high risk of metastatic relapse. This allows us to more accurately stratify patients into risk groups in order to determine the intensity and duration of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f5e5b1eac08421c304a6da77fd1d4ce4707e64" target='_blank'>
              The Prognostic Value of Amplification of the MYCC and MYCN Oncogenes in Russian Patients with Medulloblastoma
              </a>
            </td>
          <td>
            A. Chernov, Ekaterina Batotsyrenova, Sergey N Zheregelya, S. Pyurveev, V. Kashuro, Dmitry Ivanov, Elvira Galimova
          </td>
          <td>2025-07-27</td>
          <td>Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 DNA repair and autophagy are distinct biological processes vital for cell survival. Although autophagy helps maintain genome stability, there is no evidence of its direct role in the repair of DNA lesions. We discovered that lysosomes process Topoisomerase 1-cleavage complexes (TOP1cc) DNA lesions in human cancer cell. TOP1 resolves DNA topological stress ahead of DNA replication and transcription. If not promptly repaired, TOP1cc hinder the progression of DNA replication and transcription fostering genomic instability. Thus, stabilising TOP1cc with TOP1 poisons, such as Camptothecin or its analogue, is highly effective and widely used in cancer therapy. Proteolysis of TOP1cc by the proteasome is described at excessively high doses of CPT that are not relevant in a clinical setting, calling into question the relevance of the proteasome role in TOP1cc repair for cancer therapy and patient response. Here we described that repair of TOP1cc induced by clinically relevant dose of Camptothecin depends on selective autophagy. Proteomics, sequencing, biochemistry techniques and live imaging all demonstrate the lysosomal uptake of TOP1cc upon induction of replication stress. Selective degradation of TOP1cc by autophagy directs DNA damage repair and cell survival. TOP1cc are exported from the nucleus to lysosomes through transient alteration of the nuclear envelope and independent of the proteasome. Mechanistically, the autophagy receptor TEX264 acts as a TOP1cc sensor at DNA replication forks, triggering TOP1cc processing by the p97 ATPase and mediating the delivery of TOP1cc to lysosomes in an MRE11 nuclease and ATR kinase-dependent manner. We found an evolutionarily conserved role for selective autophagy in DNA damage repair that enables cell survival, protects genome stability, and is clinically relevant for colorectal cancer patients.



 Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Antičević, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, and Kristijan Ramadan. TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ada9e8b0a0e36cefc3f3e65806469a480324003" target='_blank'>
              Abstract P11: TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL
              </a>
            </td>
          <td>
            Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Anticevic, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, L. Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, Kristijan Ramadan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are prevalent at promoters and superenhancers (SEs), exclude nucleosomes, and recruit transcription factors. This study sought to determine whether the nucleosome exclusion affects the recruitment of the SE marker BRD4, which typically binds to acetylated histones and facilitates SE–promoter contacts via the phase separation-dependent mechanism. Analyses of the available whole-genome data revealed that SEs with the highest G4 density were depleted of nucleosomes but not of BRD4. This led us to test the possibility of histone-independent BRD4 maintenance at G4-rich SEs. A typical SE G4 destabilized a nearby nucleosome in vitro and, unlike B-DNA, bound weakly to BRD4 bromodomains. Similar to an acetylated nucleosome, the G4 promoted phase separation in BRD4 solutions. This effect was not altered by the histone competitor JQ1. However, it was attenuated by two known G4 ligands, suggesting that they could disrupt SE–promoter communication in cells. Consistently, these ligands downregulated several genes regulated by G4-rich SE-contacting promoters more efficiently than they did SE-independent genes. Our findings underscore the significance of G4-rich SEs as transcriptional regulators and provide new insights into their organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca6b9e0a4bd9850a35faa9e07c7255af9d5b35c" target='_blank'>
              G-quadruplexes as potential traps for superenhancer marker BRD4: ligand-sensitive binding and co-separation in vitro
              </a>
            </td>
          <td>
            Iuliia Pavlova, Olga M. Ivanova, Mikhail S Iudin, Anastasiya V Surdina, Nikolay A Barinov, M. E. Bogomiakova, Sergey D Oreshkov, Z. Shenkarev, V. Severov, Dmitriy V Klinov, Victoria O Shender, A. Bogomazova, M. Lagarkova, A. Varizhuk, Vladimir B Tsvetkov
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Endogenized giant viruses are emerging as major contributors to the genome evolution of microbial eukaryotes, with both degraded and fully functional latent viruses being found integrated in diverse lineages. The mechanisms that determine the fate of viral integrants are poorly understood, however. Acanthamoeba is a unicellular eukaryote known for undergoing lateral gene transfer (LGT) with viruses. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba, Neff and C3, to investigate the genomic mechanisms that mediate the fate of viral integrations in eukaryotic genomes. Results Viral integrations in the C3 and Neff genomes are largely non-overlapping and disproportionately found in sub-telomeric regions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed, do not obviously encode functions associated with their own mobility, and are colonized by host mobile elements. Viral regions are hypermethylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. Conclusions We propose a model for the trajectory of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, allowing them to persist in the genome, and (iii) deterioration of viral genomes by point mutation, mobile element colonization, and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the amoeba. Our work highlights the importance of host genome dynamics and epigenetic silencing for understanding the evolution of endogenized viral elements. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02280-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f54d3754da37f780fd27a13a0f10078e856d51" target='_blank'>
              Epigenetic silencing and genome dynamics determine the fate of giant virus endogenizations in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2025-07-01</td>
          <td>BMC Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosomes (SMC) complexes organize genomes by extruding DNA loops, while replisomes duplicate entire chromosomes. These essential molecular machines must collide frequently in every cell cycle, yet how such collisions are resolved in vivo remains poorly understood. Taking advantage of the ability to load SMC complexes at defined sites in the Bacillus subtilis genome, we engineered head-on and head-to-tail collisions between SMC complexes and the replisome. Replisome progression was monitored by genome-wide marker frequency analysis, and SMC translocation was monitored by time-resolved ChIP-seq and Hi-C. We found that SMC complexes do not impede replisome progression. By contrast, replisomes restrict SMC translocation regardless of collision orientations. Combining experimental data with simulations, we determined that SMC complexes are blocked by the replisome and then released from the chromosome. Occasionally, SMC complexes can bypass the replisome and continue translocating. Our findings establish that the replisome is a barrier to SMC-mediated DNA-loop extrusion in vivo, with implications for processes such as chromosome segregation, DNA repair, and gene regulation that require dynamic chromosome organization in all organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686025c7ec346de8dd0c0dabfe2ed3d2aaa2a6a1" target='_blank'>
              Replisomes restrict SMC translocation in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cells require highly polymorphic receptors to perform accurate self/non-self recognition. In the amoeba Dicytostelium discoideum, polymorphic TgrB1 & TgrC1 proteins are used to bind sister cells and exclude cheaters, but it remains unknown how cells continually generate this extreme genetic diversity. Here, we created a collection of chromosome-length, whole genome sequences from 10 Dictyostelium discoideum isolates and sister species to understand the evolution of the large tgr gene family. Our dataset includes AX2–214, a widely used D. discoideum lab strain, as well as complete genomes for two Chlamydia-like endosymbionts harbored within amoebae. We find that tgrB1 and C1 lie in a hypermutational hotspot, with haplotypes that undergo repeated intralocus recombination, duplications, transpositions, and inversions. These structural dynamics are highly localized adjacent to tgrB and C, resulting in the gain and loss of dozens of genes. The tgrBC genes themselves frequently duplicate and recombine, leading to the rapid generation of unique tgrBC repertoires. In the broader tgr gene family, some genes (e.g. tgrN) are single copy and syntenic across all the genomes, whereas others (e.g. tgrA) prolifically duplicate at similar rates to Dictyostelium transposons. Thus, the tgr genes are among the most rapidly evolving families genome-wide. We propose that the intense diversification within the tgrBC locus can help explain how these genes acquire such extreme levels of polymorphism, with parallels to the MHC immune genes in mammals and other allorecognition systems. This collection of amoeba genomes is also an ideal dataset for comparative genomics and molecular evolution in Amoebozoa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c40a8c46cc0bf1ac242e15ce2267877baf6fe3" target='_blank'>
              Hypermutable hotspot enables the rapid evolution of self/non-self recognition genes in Dictyostelium
              </a>
            </td>
          <td>
            Mische Holland, Mayar Ahmed, Janet M. Young, Sean McFadyen, James R Drurey, E.A. Ostrowski, T. Levin
          </td>
          <td>2025-08-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Kaposi's sarcoma‐associated herpesvirus (KSHV), the etiological agent of Kaposi's sarcoma and other lymphoproliferative disorders, presents a delicate balance between host cell manipulation, and orchestrating its transcriptional repertoire. Like all herpes viruses, the virus can reside in latency, expressing only a handful of proteins or launch a lytic program that expresses over 80 proteins responsible for the generation and release of new virions. Enhancers, as their name suggests, can enhance the transcription rate from a promoter depending on the cellular environment and tissue. While the promoters in KSHV have been extensively studied, the enhancers were only recently identified. Here we will describe our current view on KSHV enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0b1a93efde5ee2df29eb9e253f26d14a84bff84" target='_blank'>
              Viral Enhancers Orchestrate Kaposi's Sarcoma‐Associated Herpesvirus Transcription
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Meir Shamay
          </td>
          <td>2025-07-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Despite the comprehensive advancement in the field of cancer therapeutics, there remains an urgent need to identify new pathophysiological mechanisms that can be targeted in isolation or in combination with existing therapeutic regimens. The epithelial-to-mesenchymal transitions (EMT) induced by hypoxia, cytokines, and growth factors involves acquisition of invasive and migratory properties by cancer cells. Epigenetic alterations of DNA methylations and/or histone modifications cause substantial transcriptomic reprogramming in cancer cells during EMT and metastasis, which can be therapeutically targeted by a thorough understanding of the mutual interactions among the epigenetic processes. Previously, the mammalian DNA methyltransferases (DNMTs) have been shown to possess redox- and Ca++- dependent active DNA 5mC demethylation activities in addition to the cytosine methylation activity. Methods In this study, we have carried out experiments using a range of molecular, cellular, and genome editing approaches including cell culturing, CRISPR/Cas9-editing, si- or sh-RNA-mediated knockdown, quantitative RT-PCR, western blotting, ChIP-qPCR, Na-bisulfite sequencing, EMT and lung colonization assays in conjunction with DNA methylome and DNMT3A ChIP-Seq analyses, Results We found that active DNA demethylation activity of DNMT3A is essential for hypoxia-induced EMT of the SW480 colon cancer cells, its global genomic DNA demethylation, and promoter DNA demethylation/transcriptional activation of EMT-associated genes including TWIST1 and SNAIL1. DNMT3A also regulates hypoxia-induced HIF-1α binding to and transcriptional activation of the TWIST1 promoter as well as genome-wide DNA demethylation and EMT of breast cancer and liver cancer cells. Mechanistic analysis supports a regulatory model where hypoxia-induced H3K36me3 mark recruits DNMT3A to demethylate CpG in the hypoxia-responsive element (HRE), thereby facilitating HIF-1α binding and activation of the promoters of EMT genes. Conclusions Altogether, this study has provided the first demonstration of a physiological function of the active DNA demethylation activity of the DNMTs. Equally important, our findings have revealed a missing link between the HIF-1α pathway and the O2-sensing KDM pathway both of which are known to be essential for a wide set of normal and disease-associated cellular processes. Finally, the active DNA demethylation activity of DNMT3A has now emerged as a new potential target for therapeutic development to prevent EMT and metastasis of cancer cells. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00775-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754e95e75763fae58f7d2af9f820e9cabdaae0fe" target='_blank'>
              Hypoxia-induced genome-wide DNA demethylation by DNMT3A and EMT of cancer cells
              </a>
            </td>
          <td>
            Biswanath Chatterjee, P. Majumder, Chun-Chang Chen, Jing-Ping Wang, P. Su, Hung-Cheng Lai, Ching-Chen Liu, Hsin-Nan Lin, Chen-Hsin A Yu, Hanna S Yuan, Che-Kun James Shen
          </td>
          <td>2025-08-05</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Therapeutic resistance remains a primary obstacle to curing cancer. Healthy cells exposed to genotoxic insult rapidly activate both p53-dependent and -independent non-genetic programs that pause the cell cycle and direct either DNA repair or apoptosis. Cancer cells exploit these same pathways as they respond to stresses induced by cancer therapies. In this study, we investigated a potential role for upstream stimulatory factor 1 (USF1) and USF2 in the p53-independent response of lymphoma cells to genotoxic therapy. We previously found that lymphocytes utilize the responsiveness of USF1 to double-stranded DNA breaks to coordinate T cell receptor beta (Tcrb) gene expression during V(D)J recombination. Here, microarray gene expression analysis of derivatives of the p53-deficient mouse B lymphoma cell line, M12, revealed that simultaneously depleting cells of both USF1 and USF2 altered the expression of 940 gene transcripts (>1.50-fold change, < 0.05 FDR), relative to cells expressing a scrambled control shRNA. Seven days after exposure to a single sublethal (5 Gy) dose of ionizing radiation, USF-depleted (USFKD) cells exhibited widespread and distinct transcriptional responses from those of irradiated controls (5035 and 5054 differentially expressed gene transcripts, respectively, with roughly half shared between both cell types). Gene ontology analyses revealed that USF knockdown induced numerous changes in the expression of genes critical for immune development and function while diminishing the responsiveness of genes linked to DNA damage pathways. Microarray findings were confirmed by RT-qPCR for a panel of genes responsive to USF knockdown and/or irradiation. These findings shed further light on transcriptional responses to ionizing radiation that manifest over time in transformed cells, identifying a novel p53-independent role in lymphocytic DNA damage stress responses for USF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6bb6187205d63162ff7fe5e73d3f94fd90940ba" target='_blank'>
              Knockdown of USF1 and USF2 drives prolonged changes in the gene expression response of M12-5B3 cells to DNA damage
              </a>
            </td>
          <td>
            Kimberly S Bellingham-Johnstun, Lisa A Metzger, Jennifer L Stone, Kevin Gerrish, Pierre R Bushel, Michael L Sikes
          </td>
          <td>2025-07-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Despite being a consolidated practice in modern cancer genomics, mutational signature analysis poses various challenges. First, determining the number of signatures is a complex task and depends on heuristics. Second, several signatures lack a clear etiology, raising concerns about whether they result from computational artifacts or actual mutagenic processes. Last, approaches for signature assignment are highly influenced by the catalogue of signatures used for the analysis. To overcome these limitations, we introduce RESOLVE (Robust EStimation Of mutationaL signatures Via rEgularization), a framework designed for the efficient extraction, assignment, and confidence estimation of mutational signatures. RESOLVE enables the stratification of cancer genomes according to the active mutational processes, providing insights into those that are consistently shared among various cancer types. We applied RESOLVE to 20 000 samples from adult and pediatric cancers, providing a comprehensive characterization of subtypes associated with specific mutational processes and alterations in driver genes. Our analysis demonstrates that RESOLVE accurately fits observed mutations with a smaller set of signatures compared to existing catalogues, suggesting the existence of a limited number of dominant mutational processes in cancer genomes. Clustering analysis revealed distinct patient groups characterized by specific signatures, with significant associations between certain signatures and patient prognosis. Additionally, we identified strong associations between signatures and driver gene mutations, offering insights into cancer subtype mechanisms and evolution. Our findings highlight the efficiency of RESOLVE and its potential impact on personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33c3f703237609301199e20410a7b8d03b3cefb" target='_blank'>
              Comprehensive analysis of mutational processes across 20 000 adult and pediatric tumors
              </a>
            </td>
          <td>
            M. Villa, Federica Malighetti, Luca De Sano, Alberto Maria Villa, N. Cordani, A. Aroldi, M. Antoniotti, G. Caravagna, Alex Graudenzi, R. Piazza, L. Mologni, Daniele Ramazzotti
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Oncogene expression can cause replication stress (RS), leading to DNA double-strand breaks (DSB) that require repair through pathways such as homologous recombination, non-homologous end-joining, and microhomology-mediated end-joining (MMEJ). Cyclin D1 (encoded by CCND1) is a well-known oncoprotein overexpressed in cancer; however, its role in RS is unknown. Using mantle cell lymphoma (MCL) as a naturally occurring model of cyclin D1 overexpression, we examined its impact on RS and DSB-repair mechanisms. Cyclin D1 overexpression elevated RS, increased DNA damage, especially during mitosis, and caused specific upregulation of MMEJ. Furthermore, cyclin D1 activates the polymerase theta (POLQ) transcription by binding its promoter loci, driving POLΘ-mediated MMEJ that is essential to withstand cyclin D1-induced RS. Moreover, concurrent ATM deficiency further intensifies RS, enhances POLQ expression and heightens reliance on MMEJ mediated DNA damage repair. Consequently, inhibition of POLΘ in cyclin D1-overexpressed settings further exacerbates RS, causing single-strand DNA gap accumulations and chromosomal instability, ultimately leading to apoptosis, an effect amplified in ATM-deficient cells. Targeting MMEJ via POLΘ inhibition is, therefore, an effective strategy in the context of cyclin D1 overexpression and ATM deficiency and may provide a unique therapeutic approach for treating MCL and other malignancies characterized by similar alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e92152fc5a144bdda47d831e771bd37fa31380d" target='_blank'>
              Cyclin D1 overexpression induces replication stress and microhomology-mediated end-joining dependence in mantle cell lymphoma.
              </a>
            </td>
          <td>
            J. Abeykoon, S. Asada, G. Zhu, Y. Hirohashi, L. Moreau, Divya R Iyer, S. Mukkavalli, K. Parmar, G. Zambrano, Lige Jiang, Dongni Yi, M. Manske, K. Gwin, Rebecca L. King, J. Cerhan, Xiaosheng Wu, Zhenkun Lou, G. Shapiro, T. Witzig, A. D'Andrea
          </td>
          <td>2025-07-03</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Background Telomere homeostasis is pivotal in various biological processes including ontogeny, reproduction, physiological aging, and the onset of numerous diseases such as tumors. In human stem cells and approximately 85% of tumor cells, telomerase formed by TERT and TERC RNA complex is responsible for elongating telomeres. However, the intricate and precise regulatory mechanisms governing telomerase remain largely elusive. Methods and results We developed a genome-wide trimolecular fluorescence complementation (TriFC) screen to identify TERC RNA-interacting proteins and found ZC3H15 (Zinc finger CCCH domain-containing protein 15) to interact with telomerase. ZC3H15 interacts with TERT via its N-terminal domain in an RNA-dependent manner. The proximity labeling technique PhastID revealed that ZC3H15 associates with proteins involved in regulation of ribonucleoprotein (RNP) complex biogenesis, snRNP assembly and RNA localization. Deletion of ZC3H15 upregulated telomerase activity but interestingly resulted in shortened telomeres and induced senescence in HTC75 cells, suggesting an unknown mechanism in regulating telomere length. Notably, we found ZC3H15 to associate with GEMs nuclear bodies, and its deletion led to the spatiotemporal fusion of GEMs and Cajal bodies, resulting in the sequestration of telomerase within Cajal bodies and a reduction in telomerase recruitment to telomeres during the S phase. Consistent with these findings, ZC3H15 ablation accumulated TERC precursor RNA. Conclusions These observations provide valuable insights into the molecular mechanisms by which ZC3H15 regulates telomerase dynamics and cellular senescence. ZC3H15 may represent a new target for cancer treatment and anti-aging therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01449-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df824506b64a6082d42aacda64e880366cc87c4" target='_blank'>
              Disruption of ZC3H15 compromises telomere length maintenance by entrapping telomerase within cajal bodies
              </a>
            </td>
          <td>
            Chuanlei Wang, Wei Chen, Ruofei Li, Yue Yang, Jiali Wu, Yuyang Tian, Zibin He, Song Lin, Xining Wang, Jianxi Zhu, Wenbin Ma, Songyang Zhou, Yan Huang
          </td>
          <td>2025-07-22</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Acremonium chrysogenum is a notable filamentous fungus recognized for its essential contribution to the pharmaceutical sector through the biosynthesis of cephalosporin C (CPC). CPC functions as a key intermediate in the biosynthesis of β-lactam antibiotics, which are employed to combat bacterial infections. This study successfully generated a telomere-to-telomere (T2T) chromosome-scale genome sequence for A. chrysogenum, combining BGI short reads, PacBio HiFi long reads, and Hi-C technology. This genome sequence contained eight complete chromosomes (29.00 Mb) and a circular mitochondrial genome (27.27 kb), featuring an N50 length of 3.87 Mb. Repetitive elements accounted for 9.65% of genomic content, and a total of 7,745 genes involved in protein coding were annotated. This well-assembled reference genome of A. chrysogenum serves as an important foundation for elucidating the biosynthetic pathway of cephalosporin C and for molecular breeding. Furthermore, it offers valuable insights into chromosome organization, genome evolution, and regulatory mechanisms, facilitating future advancements in antibiotic research and fungal biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12587d2631e390bb270a6373edb55bfcb1c7374e" target='_blank'>
              The telomere-to-telomere chromosome-scale genome assembly of Acremonium chrysogenum
              </a>
            </td>
          <td>
            Chao Han, Yiping Zhang, Hengyu Liang, Minliang Chen, Jiahuang Li, Zichun Hua
          </td>
          <td>2025-08-07</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Plasmids constitute a key tool in synthetic biology, providing a versatile framework for various research and industrial ventures. An essential determinant of plasmid functionality is its copy number, impacting both protein production rates and host cell metabolic burden. However, currently there is no model that can computationally predict the plasmid copy number from the sequence of its origin of replication (ORI). We present a novel software solution tailored to simplify plasmid copy number design, poised to redefine plasmid engineering workflows. At the heart of our tool lies a comprehensive machine learning model, informed by numerous features extracted from the ORI. This computational model emphasizes the importance of promoter strength and the RNA folding dynamics of regulatory elements within the ORI. Additionally, we detail a robust protocol for the efficient manipulation of plasmid ORIs used to validate our model's predictive capabilities. This innovation represents a paradigm shift in plasmid-centric methodologies, offering unprecedented avenues for advancement in synthetic biology research and industrial applications. The software is available at: https://pcn-gradient.vercel.app/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef47a6451c0062151b76e5642c026f3faf686c38" target='_blank'>
              Deciphering plasmid replication dynamics: a computational approach to predict ColE1-like plasmid copy number.
              </a>
            </td>
          <td>
            Rinat Saban, Ofri Tfilin, David Haggiag, Yam Tawachi, Matan Arbel, T. Tuller
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract R-loops or DNA–RNA hybrids are prominent nucleic acid structures that commonly arise during transcription. These structures play important biological functions, such as regulating gene expression and DNA repair. However, when unresolved by nucleic acid processing factors, pathological R-loops can be harmful and lead to genome instability. N6-Methyladenosine (m6A), the most prevalent modification in messenger RNA, has been recently identified to be crucial for regulating R-loop balance and maintaining genome stability. Strikingly, m6A-modified R-loop formation can have opposing consequences, either stabilization or resolution, depending on the biological context. In this review, we discuss the current knowledge of the regulatory roles of m6A on R-loops across various processes, including gene transcription, DNA repair, and centromere and telomere stability. Additionally, we explore other m6A-mediated processes, such as nascent transcription and chromatin landscape, that potentially affect R-loop dynamics. Finally, we discuss the current limitations and future directions of studying the m6A–R-loop axis, as well as the opportunities to target this pathway as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d8ee286e956882ce3a5149e41ece752c38e0d" target='_blank'>
              m6A modification in R-loop homeostasis: a potential target for cancer therapeutics
              </a>
            </td>
          <td>
            Minh-Anh Vu, Manuela Spagnuolo, Chun-Long Chen
          </td>
          <td>2025-08-11</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="R-loops, chromatin structures containing DNA-RNA hybrids with displaced single-stranded DNA, play crucial roles in various cellular processes. Their formation is influenced by factors such as DNA topology, RNA stability, and the presence of GC-rich regions. However, excessive or uncontrolled R-loop accumulation can threaten genomic stability, leading to DNA damage, particularly double-strand breaks. To preserve genome integrity, cells have developed mechanisms to regulate R-loop formation and resolution. Dysregulation of these processes is linked to several diseases, including cancer and neurodegenerative disorders. In this review, we will explore the dynamics of R-loop formation and resolution and how they are detected, their roles in DNA damage and repair, and how their dysregulation may lead to immune responses and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3728d9fd65e0fa0dfc3340b92946a313355d5d1" target='_blank'>
              The paradox of R-loops: guardians of the genome or drivers of disease?
              </a>
            </td>
          <td>
            Shachinthaka D Dissanayaka Mudiyanselage, Phillip Wulfridge, K. Sarma
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a novel biomarker for cancer detection due to its tumor-specific amplification and stable structure in circulation. However, its clinical application is hindered by extremely low abundance in biofluids and the lack of robust detection techniques. To address this, we screened for tumor-associated eccDNA biomarkers and developed NPCC (Nested PCR-CRISPR/Cas12a), a novel method combining nested PCR for ultrasensitive amplification with CRISPR/Cas12a for sequence-specific detection. The assay employs two rounds of junction-specific PCR to enrich eccDNA, followed by CRISPR/Cas12a-mediated cleavage guided by target-specific crRNA. Validation using synthetic circular DNA standards demonstrated a limit of detection (LoD) of 10-6 fM, representing a >100-fold improvement over conventional PCR, with no cross-reactivity to linear or genomic DNA fragments. In plasma samples from 88 cancer patients, NPCC successfully detected multiple tumor-specific eccDNAs, including the hepatocellular carcinoma marker eccDNA-HCC-1 (AUC = 0.8977). NPCC overcomes key technical barriers in liquid biopsy, offering a cost-effective, highly sensitive, and specific platform for noninvasive cancer diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeca77128a740c764bbb2a48a28ef00f3cfa9edd" target='_blank'>
              Ultrasensitive eccDNA Detection for Tumor Diagnostics by Using CRISPR/Cas12a-Coupled Nested PCR.
              </a>
            </td>
          <td>
            Gege Liu, Junlu Wu, Bin Yang, J. Kong, Yaran Li, Chen Chen, Jiayi Hu, Zujun Sun, Yanqiang Hou, Ruixin Sun, Dong Li
          </td>
          <td>2025-07-31</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The Mre11 nuclease, part of the conserved MRX complex involved in the repair of DNA double-strand breaks (DSBs), is also essential to initiate meiotic recombination in budding yeast by promoting Spo11-induced DSBs. Recruitment of Mre11 to meiotic DSB sites depends on Rec114-Mei4 and Mer2 (RMM) that organize the meiotic DSB machinery by a mechanism involving biomolecular condensation. Here, we explored the role of Mre11 during meiosis and its relationship to RMM condensation. We show that both Mre11 and MRX complexes form DNA-dependent, hexanediol sensitive condensates in vitro. In vivo, Mre11 assembles into DNA damage-dependent foci in vegetative cells and DSB-independent foci in meiotic cells. In vitro condensates and in vivo foci both depend on the C-terminal intrinsically-disordered region (IDR) of Mre11. Importantly, while the Mre11 IDR is dispensable for vegetative DNA repair it is essential during meiosis. The C-terminus of Mre11 forms a short α-helix that binds a conserved region of Mer2, and mutating residues within this interface reduces Mre11 foci and DSB formation. Finally, we identified a SUMO-interacting motif within the Mre11 IDR that enhances recruitment of Mre11 during meiosis and facilitates DSB formation. This work identifies multiple mechanisms that collaborate to recruit Mre11 during meiosis to initiate recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d416cf934fcb6f095caa1612b3bfc936b858edeb" target='_blank'>
              Recruitment of Mre11 to recombination sites during meiosis
              </a>
            </td>
          <td>
            Priyanka Priyadarshini, Mahesh Survi, Wael El Yazidi Mouloud, Regina Bohn, Steven Ballet, Neil Hunter, Alexander N Volkov, Corentin Claeys Bouuaert
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Circular RNAs, or circRNAs, play a key role in breast cancer biology, directly impacting the diagnosis, prognosis, and treatment of the disease. This review explores the mechanisms, regulatory roles, and functional significance of circRNAs in breast cancer. Overexpressed circRNAs regulate gene expression, cell cycle progression, and drug response in breast cancer. This process is facilitated by the interaction between small RNA molecules (miRNAs) and proteins that can bind to RNA (RBPs), which target the main messenger RNA (mRNA). Consequently, they influence gene expression, cellular proliferation, and drug resistance. Dysregulated circRNA expression contributes to breast cancer progression by promoting tumor aggressiveness and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc3a0ca218e95d883f029cc06f1e2cad3dd65cf" target='_blank'>
              An In-Depth Study of Circular RNAs Related to Breast Cancer Treatment and their Implications.
              </a>
            </td>
          <td>
            F. Akram, Zoha Naeem Rana, Ifrah Shabbir, Taseer Fatima, I. Haq
          </td>
          <td>2025-08-07</td>
          <td>Current protein & peptide science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Multicellular organisms arise from a single genome template in the zygote, necessitating the cells of the developing embryo to up- and downregulate specific genes to establish and maintain their identity. This template is maintained, propagated, and interpreted as chromatin, a polymer of nucleic acids and associated structural and regulatory proteins. Recent genome-wide surveys documented a wealth of disease-associated mutations in chromatin factors, indicating their fundamental significance and potential for therapeutic targeting. However, chromatin factors exist in a complex balance, with a single deficiency often leading to pleiotropic downstream effects. Here, we review the mechanisms of chromatin regulation and partitioning, highlighting examples of how these processes are altered in human diseases. We argue that loss of chromatin fidelity, both locally at specific genes and regulatory elements, and globally at the megabase-scale, contributes to many pathological states and may thus represent an intriguing target for corrective interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b3c6ccb1965f22efefabec61c021483fcae985" target='_blank'>
              Aere perennius: how chromatin fidelity is maintained and lost in disease
              </a>
            </td>
          <td>
            Dustin Fetch, Amina Jumamyradova, Cameron Chapa, Yong Ge, Mansour Mohamadzadeh, Alexey A Soshnev
          </td>
          <td>2025-07-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spatial resolution of omics dynamics is fundamental to understanding tissue biology. Spatial profiling of DNA methylation, which is a canonical epigenetic mark extensively implicated in transcriptional regulation, remains an unmet demand. Here, we introduce a method for whole genome spatial co-profiling of DNA methylation and transcriptome of the same tissue section at near single-cell resolution. Applying this technology to mouse embryogenesis and postnatal brain resulted in rich DNA-RNA bimodal tissue maps. These maps revealed the spatial context of known methylation biology and its interplay with gene expression. The two modalities’ concordance and distinction in spatial patterns highlighted a synergistic molecular definition of cell identity in spatial programming of mammalian development and brain function. By integrating spatial maps of mouse embryos at two different developmental stages, we reconstructed the dynamics of both epigenome and transcriptome underlying mammalian embryogenesis, revealing details in sequence, cell type, and region-specific methylation-mediated transcriptional regulation. This method extends the scope of spatial omics to DNA cytosine methylation for a more comprehensive understanding of tissue biology over development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e580ba04766eb95572371b96391de4351133faff" target='_blank'>
              Spatial joint profiling of DNA methylome and transcriptome in mammalian tissues
              </a>
            </td>
          <td>
            Chin Nien Lee, Hongxiang Fu, Angelysia Cardilla, Wanding Zhou, Yanxiang Deng
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Methylglyoxal (MG) is a highly reactive aldehyde that is produced endogenously during metabolism, and from exogenous sources like sugary food and cigarette smoke. Unless detoxified by glyoxalases, MG can readily react with DNA and proteins, generating characteristic glycation-derived lesions. As a result, MG exposure has been linked to a variety of human diseases, including cancers. Prior studies show that MG preferentially makes adducts on guanine residues, causing DNA damage. However, in vivo, how such events impact genome-wide MG mutagenicity is poorly understood. Such information is essential to comprehend the true contribution of MG to genome instability and global mutational burden. In the present study, we show that MG can robustly mutagenize single-stranded DNA in the yeast genome, within a guanine-centered mutable motif. We demonstrate that genome-wide MG mutagenesis is greatly elevated in the absence of the glyoxalase Glo1, and abrogated in the presence of the aldehyde quencher aminoguanidine. Importantly, we uncovered strand slippage and mispairing as the predominant mechanism for generation of all MG-associated mutations, and demonstrate that the translesion polymerase Rev1 is a key player in this pathway. Finally, we find that the primary MG-associated mutation is enriched in a variety of sequenced tumor datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ac197fe47da2319b761beabd3304e807a2cc0f" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b074d01f6fb78ae72372c0d26a26df7f99fb224" target='_blank'>
              Fungi as models of centromere innovation: from DNA sequence to 3-dimensional arrangement.
              </a>
            </td>
          <td>
            Srijana Dutta, Krishna Bhat, Rashi Aggarwal, K. Sanyal
          </td>
          <td>2025-08-11</td>
          <td>Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="PURPOSE
NUT carcinoma (NC) is an underdiagnosed, poorly differentiated squamous cell cancer with a median survival of 6.7 months. Defined by NUTM1 fusions, NC enhances oncogene transcription, including MYC. We investigated the ability of standard next-generation sequencing (NGS) to identify NUTM1 fusions and describe additional molecular features of NC.


EXPERIMENTAL DESIGN
This study included 116 NC patients whose tumors underwent broad-panel NGS (>80 genes) of DNA, ctDNA, and/or RNA fusion sequencing between 2013-2024. NGS reports and medical records were manually reviewed.


RESULTS
Of 116 patients (median age 38, 40.5% female), 84.5% had DNA, 12.1% had ctDNA, and 51.7% had RNA fusion testing. In a subset of 100 patients with DNA/ctDNA testing, 92.9% (n=79/85) had <10 pack-years/never-smoking history, and 58.8% (n=47/80) had a BRD4::NUTM1 fusion. Median TMB was 1.0 mut/Mb (range 0.0-16.0; n=71 known), and 19.7% (n=13/66) had PD-L1 expression ≥1%. DNA, ctDNA, RNA fusion, NUT IHC, and NUTM1 FISH detected NC fusions in 21.6%, 21.4%, 83.9%, 100.0%, and 91.9% of tests, respectively. Co-occurring pathogenic mutations included oncogenes PIK3CA, RET, FGFR3, and tumor suppressors ATM and BRCA1 (n=1 each). Secondary genes altered in >5% of NCs included LRP1B (10.4%), MLL2/KMT2D (8.0%), and FAT1 (5.5%); common pathways with mutated genes were epigenetic (57.0%), cell cycle (26.0%), and DNA repair (24.0%).


CONCLUSIONS
Standard DNA NGS detects less than a quarter of NUT carcinomas; RNA-based fusion testing, or NUT IHC/NUTM1 FISH, should be routine for suspected NC. NCs are enriched in co-occurring epigenetic, cell cycle, and DNA repair alterations, warranting further evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e381b2a39bcb76ca463ed69b86f3eaddf24f8bb" target='_blank'>
              Molecular characterization of NUT carcinoma: a report from the NUT carcinoma registry.
              </a>
            </td>
          <td>
            Justin J. Kim, Sara A Walton, N. Mahadevan, J. Haradon, Francesco Paoloni, P. Paik, J. Chaft, R. Hsu, S. Piha-Paul, P. Jänne, D. Barbie, L. Sholl, Steven G Dubois, Glenn J Hanna, Geoffrey I. Shapiro, Christopher A. French, Jia Luo
          </td>
          <td>2025-07-24</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>147</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is a common subtype of breast cancer that is defined in part by genetic loss of CDH1 caused by mutation or deletion, leading to loss of cell adhesion protein E-cadherin in >90% of ILC. Genetic loss of CDH1 is an early event in ILC oncogenesis, yet the mechanisms by which CDH1/E-cadherin acts as a tumor suppressor are not well understood. To study how early CDH1 loss drives ILC oncogenesis, we used a series of non-transformed human mammary epithelial cell (HMEC) models to target CDH1/E-cadherin, inhibiting extracellular E-cadherin signaling using antibodies versus modeling genetic CDH1 loss using siRNA or knockout via CRISPR/Cas9. Through transcriptome analyses across four HMEC models, we found that the mode of E-cadherin loss or suppression is critical for the subsequent phenotype. Antibody-mediated inhibition of cell-cell contacts induced gene signatures of epithelial-mesenchymal transition (EMT), consistent with the role of E-cadherin suppression during the EMT process. Conversely, genetic CDH1 loss – as in ILC oncogenesis – repressed EMT signatures, and instead remodeled gene expression toward a luminal epithelial phenotype. Using single cell transcriptomics and flow cytometry analyses of cell lineage markers, we found that genetic loss of CDH1 reprogrammed cells to a luminal progenitor-like phenotype. By isolating luminal versus basal cells prior to CDH1 knockout, we found that CDH1 loss led to remodeling of lineage identity in both populations, converging on a new lineage homeostasis with a luminal progenitor-like phenotype. Consistent with increased progenitor features, CDH1 loss enhanced proliferative capacity over the finite lifespan of the HMECs, highlighting a feature of early CDH1 loss that may contribute to clonal advantage during tumor initiation. Our findings support that inhibition of E-cadherin results in different transcriptional response compared to CDH1 loss, with the latter driving a transcriptional and phenotypic state characteristic of a luminal progenitor-like population, which offers new insight into early events in ILC oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55a904a9de6859d0ee1985b86b2f0ef97b6b1ec" target='_blank'>
              CDH1 loss remodels gene expression and lineage identity in human mammary epithelial cells
              </a>
            </td>
          <td>
            Maggie Musick, Chinasa A Ufondu, Carmen E. Rowland, Joseph L. Sottnik, Madeleine T. Shackleford, Camryn S. Nesiba, J. Ostrander, Matthew J. Sikora
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Nonhomologous end joining (NHEJ) is required for repairing DNA double strand breaks (DSBs) generated by the RAG endonuclease during lymphocyte antigen receptor gene assembly by V(D)J recombination. The ataxia telangiectasia–mutated (ATM) and DNA-dependent protein kinase catalytic subunit (DNA-PKcs) kinases regulate functionally redundant pathways required for NHEJ. Here, we report that loss of the senataxin helicase leads to a strong defect in RAG DSB repair upon inactivation of DNA-PKcs. The NHEJ function of senataxin is redundant with the RECQL5 helicase and the HLTF translocase and is epistatic with ATM. Co-inactivation of ATM, RECQL5, and HLTF results in an NHEJ defect similar to that from the combined deficiency of DNA-PKcs and senataxin or losing senataxin, RECQL5, and HLTF. These data suggest that ATM and DNA-PKcs regulate the functions of senataxin and RECQL5/HLTF, respectively, to provide redundant support for NHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19bdfb0688308baf4bbac48b67309551e1f66348" target='_blank'>
              Senataxin and DNA-PKcs redundantly promote non-homologous end joining repair of DNA double strand breaks during V(D)J recombination
              </a>
            </td>
          <td>
            Bo-Ruei Chen, Thu Pham, Lance D. Reynolds, Nghi Dang, Yanfeng Zhang, Kimberly Manalang, Gabriel Matos-Rodrigues, J. R. Neidigk, A. Nussenzweig, Jessica K Tyler, Barry P Sleckman
          </td>
          <td>2025-06-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Replication stress is a major driver of genomic instability and a hallmark of cancer cells. Although dynamic heterochromatin remodeling has been implicated in replication stress response, the precise mechanisms remain unclear. Here, we identify the CHAMP1 complex, composed of CHAMP1, POGZ, HP1α, and the H3K9 methyltransferase SETDB1, as a critical regulator of heterochromatin assembly at stalled replication forks. Upon replication stress, the CHAMP1 complex is recruited to stalled forks where it facilitates H3K9me3 deposition, creating a repressive chromatin environment that shields replication forks from MRE11-mediated degradation. The complex promotes the recruitment of the origin recognition complex (ORC) to sites of replication stress, such as the telomeric heterochromatin in alternative lengthening of telomeres (ALT)-positive tumor cells, thereby supporting efficient telomeric DNA replication. Loss of CHAMP1 disrupts ORC2 recruitment and impairs fork restart, leading to increased micronuclei formation and heightened sensitivity to replication stress. Notably, CHAMP1 deficiency induces synthetic lethality with FANCM inhibition in ALT-positive tumor cells, and the CHAMP1 complex is essential for the survival of CCNE1-amplified ovarian cancers. These findings uncover a chromatin-based mechanism of replication fork stabilization and suggest that CHAMP1 may represent a candidate therapeutic vulnerability in cancers with elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da656da84cf3e9dc1144dda4829e9c83aef2891" target='_blank'>
              CHAMP1 Complex Promotes Heterochromatin Assembly and Reduces Replication Stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0586d8fafa9a29f6a58bf48055edaf80d268f3" target='_blank'>
              RIPOR2 promotes multinucleation of melanoma cells downstream of the RAS/ERK oncogenic pathway
              </a>
            </td>
          <td>
            Axelle Wilmerding, Aurélie Richard, Nicolas Macagno, E. Hirsinger, Tarek Gharsalli, Léa Bellenger, N. Naouar, Caroline Gaudy, S. Mallet, N. Degardin, Lauranne Bouteille, Nathalie Caruso, D. Duprez, Y. Graba, S. Medjkane, H. Etchevers, Marie-Claire Delfini
          </td>
          <td>2025-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Elephants exhibit remarkable resistance to cancer, and understanding these mechanisms has focused on their potential applications in cancer prevention and treatment in humans. A genome-wide comparative analysis identified that the accelerated regions in elephants are enriched in Fanconi anemia (FA) complementation group L (FANCL), a ubiquitin E3 ligase that mediates the monoubiquitylation of FANCD2 as an essential step in the FA pathway. The FA pathway plays a crucial role in DNA interstrand crosslink (ICL) repair, contributing substantially to genome stability and cancer resistance. In this study, we investigated the differences in ICL repair via the FA pathway, including the function of FANCL, as well as the DNA damage response to ICLs between elephants and humans. We found that elephant fibroblasts exhibited higher sensitivity to ICL-inducing treatments, such as mitomycin C and trimethylpsoralen plus UVA (PUVA), than human fibroblasts, while showing comparable or reduced sensitivity to other DNA-damaging agents, such as doxorubicin and bleomycin. Functional analyses revealed that elephant and human FANCL performed similarly in mediating FANCD2 monoubiquitylation and cell viability following mitomycin C treatment. Interestingly, elephant fibroblasts exhibited a more potent and prolonged activation of p21 and sustained DNA repair responses, such as FANCD2 monoubiquitylation and increased RAD51expression, following ICL-induced treatments. Moreover, elephant fibroblasts showed significantly greater RAD51 foci formation than human fibroblasts after PUVA treatment, even under comparable levels of DNA damage. These findings suggest that elephants efficiently repair ICLs in growth-arrested cells likely through robust p21 activation. This study provides new insights into the cancer resistance mechanisms of elephants and offers novel approaches for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5eb799fee658096c981c29abe3d04c60e57627a" target='_blank'>
              Enhanced sensitivity, robust p21 activation, and sustained DNA repair responses to interstrand crosslinks in elephant cells compared to humans
              </a>
            </td>
          <td>
            Taisuke Kitano, Zida Zhu, Naoya Minami, K. Orino, Yasunaga Yoshikawa
          </td>
          <td>2025-07-10</td>
          <td>Frontiers in Veterinary Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long-read sequencing (LRS) has improved sensitivity to discover variation in complex repetitive regions, assign parent–of–origin, and distinguish de novo germline from postzygotic mutations (PZMs). Most studies have been limited to population genetic surveys or a few families. We applied three orthogonal sequencing technologies—Illumina, Oxford Nanopore Technologies, and Pacific Biosciences—to discover and validate de novo mutations (DNMs) in 73 children from 42 autism families (157 individuals). Assaying 2.77 Gbp of the human genome using read-based approaches, we discover on average 95 DNMs per transmission (87.5 de novo single–nucleotide variants and 7.8 indels), including sex chromosomes. We estimate that LRS increases DNM discovery by 20–40% over previous Illumina-based studies of the same families, and more than doubles the discoverable number of PZMs that emerged early in embryonic development. The strict germline mutation rate is 1.30×10 −8 substitutions per base pair per generation, strongly driven by the father's germline (3.95:1), while PZMs increase the rate by 0.23×10 −8 with a modest but significant bias toward paternal haplotypes (1.15:1). We show that the mutation rate is significantly increased for classes of repetitive DNA, where segmental duplication (SD) mutation shows a dependence on the length and percent identity of the SD. We find that the mutation rate enrichment in repeats occurs predominantly postzygotically as opposed to in the germline, a likely result of faulty DNA repair and interlocus gene conversion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b545aeba0586ba78cffa00d695ca70d8352ddc9" target='_blank'>
              Long-read sequencing of trios reveals increased germline and postzygotic mutation rates in repetitive DNA
              </a>
            </td>
          <td>
            Michelle D. Noyes, Yang Sui, Youngjun Kwon, Nidhi Koundinya, Isaac Wong, Katherine M. Munson, Kendra Hoekzema, Jennifer R. Kordosky, Gage H. Garcia, Jordan Knuth, Alexandra P. Lewis, E. Eichler
          </td>
          <td>2025-07-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Many essential functions of organisms are encoded in highly repetitive genomic regions, including histones involved in DNA packaging, centromeres that are core components of chromosome segregation, ribosomal RNA comprising the protein translation machinery, telomeres that ensure chromosome integrity, piRNA clusters encoding host defenses against selfish elements, and virtually the entire Y Chromosome. These regions, formed by highly similar tandem arrays, pose significant challenges for experimental and computational studies, impeding sequence-level descriptions essential for understanding genetic variation. Here, we report the assembly and variation analysis of such repetitive regions in Drosophila melanogaster, offering significant improvements to the existing community reference assembly. Our work successfully recovers previously elusive segments, including complete reconstructions of the histone locus and the pericentric heterochromatin of the X Chromosome, spanning the Stellate locus to the distal flank of the rDNA cluster. To infer structural changes in these regions where alignments are often not practicable, we introduce landmark anchors based on unique variants that are putatively orthologous. These regions display considerable structural variation between different D. melanogaster strains, exhibiting differences in copy number and organization of homologous repeat units between haplotypes. In the histone cluster, although we observe minimal genetic exchange indicative of meiotic crossing over, the variation patterns suggest mechanisms such as unequal sister chromatid exchange. We also examine the prevalence and scale of concerted evolution in the histone and Stellate clusters and discuss the mechanisms underlying these observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624ae86af4f411108a369c23810a20e4ee6dde43" target='_blank'>
              Genetic variation in recalcitrant repetitive regions of the Drosophila melanogaster genome.
              </a>
            </td>
          <td>
            Harsh G. Shukla, Mahul Chakraborty, J. Emerson
          </td>
          <td>2025-07-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Transposable elements (TEs) have hijacked cellular machineries to replicate and spread throughout host genomes. TEs now make up a significant portion of eukaryotic genomes and play notable roles in genomic evolution, driving both speciation and providing raw material for genetic innovation. Barbara McClintock's pioneering work on these “jumping genes” laid the foundation for modern TE research; however, her paradigm‐shifting theories in which TEs act as “controlling elements” were initially rejected due to the long‐held belief that TEs were “junk” or parasitic DNA elements. Historically, the highly repetitive nature of TEs made it challenging to both identify and investigate functions. However, recent advances in genomics have greatly accelerated our understanding of TEs. Despite their potential to cause insertional mutagenesis and disease, many transposable elements have been co‐opted by host genomes to contribute to gene regulation and development. In contrast to protein‐coding genes that typically begin their journey as DNA, are transcribed into RNA, and reach their ultimate functional form as proteins, TEs can function as cis‐regulatory DNA, functional RNA, and in rare cases, domesticated proteins and fusion events between TE and host genes. Driven by rapidly advancing technologies, the roles of TEs in both development and disease are being uncovered faster than ever, making current and future work an exciting continuation of Barbara McClintock's groundbreaking legacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831a0252c52d2b92a8d6f28251facae6b2cf9eff" target='_blank'>
              From Junk DNA to Genomic Treasure: Impacts of Transposable Element DNA, RNA, and Protein in Mammalian Development and Disease
              </a>
            </td>
          <td>
            Ten D. Li, Katelyn Toohill, A. Modzelewski
          </td>
          <td>2025-08-13</td>
          <td>Wiley Interdisciplinary Reviews. RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Although long interspersed element-1 (LINE-1) retrotransposons are poor prognostic indicators of non-small cell lung cancer, the genetic consequences of aberrant LINE-1 expression remain poorly understood. In this study, human bronchial epithelial cells were exposed chronically to 25 µM NiCl₂ to induce malignant transformation and compared to PBS-treated control cells or lung cancer cell lines. A 4 kb intronic LINE-1 insertion into the NACC2 (nucleus accumbens-associated protein 2) locus in nickel-transformed cells was associated with significant reductions in NACC2 mRNA and protein, along with constitutive increases in HDM2 and TP53α mRNAs, and aberrant expression of TP53β and TP53γ mRNAs. Steady-state levels of p53 and RB decreased, while EGFR and LINE-1 ORF1p increased. Silencing of NACC2 in TP53 wildtype NCI-H460 or TP53-null NCI-1299 cell lines replicated many of these changes, with profiles varying as a function of p53 status. Stable overexpression of HDM2 increased LINE-1 ORF1p in cancer cell lines. Human lung adenocarcinomas with wild-type TP53 showed sexually dimorphic profiles, with higher HDM2 levels in females than males, and shifts in LINE-1 ORF1p, p53, and RB that mirrored cancer cell lines. This study identifies the NACC2 locus as a target of LINE-1 retrotransposition and highlights critical interactions with the HDM2/TP53/RB regulatory axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adfffe6caad02c3b19f108090ea32e2ac145796" target='_blank'>
              LINE1 insertion into the NACC2 locus disrupts the HDM2 axis and activates lung oncogenic signaling
              </a>
            </td>
          <td>
            P. Bojang, Yingshan Wang, Kenneth S Ramos
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomic instability underlies various diseases, including cancer. This instability arises from defects in critical cellular processes, particularly those involved in DNA damage repair. Therefore, a detailed understanding of these repair mechanisms is essential for developing strategies to prevent or diagnose such diseases. The MRN complex, composed of MRE11, NBS1, and RAD50, is among the earliest elements involved in detecting DNA damage. Upon detecting DNA breaks, this complex triggers a cascade of signaling events that regulate both cell cycle arrest and DNA repair. These signaling pathways are tightly controlled by various post-translational modifications, notably ubiquitination. Although several ubiquitin ligases have been implicated in different stages of the DNA damage response, our knowledge remains limited. In this study, we reveal that βTrCP, a substrate-recognizing subunit of the SCF (SKP1/CUL1/F-box protein) ubiquitin ligase, interacts in vivo with the proteins of the MRN complex. These interactions occur in normally proliferating cells and are dependent on the GSK3 kinase. Moreover, we show that βTrCP enhances the recruitment of the MRN complex to chromatin through MRE11, thereby promoting the efficient DNA damage repair. Hence, alterations in βTrCP function affecting MRN dynamics could have severe consequences for the cell homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47da9e36f36ee547f6e7e42f2ff12a54a26f4687" target='_blank'>
              βTrCP facilitates MRN complex localization on chromatin to enhance DNA repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-07-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, K. Cheng, Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xin-Yuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, Ann Wing-Mui Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1369dba20e9061226bb6a83875ed3d5ee27fc660" target='_blank'>
              Saturation genome editing of BRCA1 across cell types accurately resolves cancer risk
              </a>
            </td>
          <td>
            Phoebe Dace, Nicole M. Forrester, M. Zanti, L. Cubitt, Chloé Terwagne, Megan Buckley, Elke M. van Veen, Thomas Stuart Wilson, Paola Scaffidi, K. Michailidou, Gregory M. Findlay
          </td>
          <td>2025-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Human cytomegalovirus extensively alters nuclear organization and the cellular transcriptome, yet understanding of these genome-wide events remains relatively limited. Here, chromatin conformation capture (Hi-C) revealed how cytomegalovirus alters chromosome organization at both large- and small-scales. Nascent transcriptomics further revealed how transcriptional changes correlate with genomic reorganization, while also uncovering infection-induced transcriptional dysregulation that contributes to the induction of neuronal gene signatures in infected fibroblasts. Combining Hi-C and Cleavage Under Targets & Release Using Nuclease (CUT&RUN) we find that viral genomes preferentially localize to highly euchromatic compartments, further dysregulating transcription of host genes. Finally, RNAi-mediated depletion of two key effectors of microtubule-based forces that are exerted on the nucleus provides insights into their diverging roles in regulating compartment-scale contacts and viral genomic interactions with host chromatin. Combined, we reveal the extent to which HCMV interacts with and alters host chromatin and transcription, and the influence of microtubule mechanotransduction on these processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0c2779fbea7b388894e9412fc49c8051594da3" target='_blank'>
              Microtubule mechanotransduction refines cytomegalovirus interactions with and remodeling of host chromatin
              </a>
            </td>
          <td>
            Celeste D Rosencrance, Derek Walsh
          </td>
          <td>2025-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbc0047fe628bc7d0d1f2c63478bc9274ca9ad" target='_blank'>
              Punctuated mutagenesis promotes multi-step evolutionary adaptation in human cancers
              </a>
            </td>
          <td>
            Christopher J Graser, Wenbo Wu, Cole Christini, Mia Petljak, Franziska Michor
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eee44887a88c522cc74478c5f67d3f1813be9b0" target='_blank'>
              DPYSL3B is a regulator of chemoresistance via DNA repair and metabolic reprogramming in prostate cancer
              </a>
            </td>
          <td>
            R. Kaarijärvi, H. Kaljunen, Nella Itkonen, Kseniia Bureiko, K. Ketola
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colon cancer is a highly aggressive solid tumor. Because most studies have focused on the intrinsic oncogenic pathways of tumors, we focused on the relationship between DNA methylation genes in the tumor immune microenvironment and colon cancer prognosis. We download RNA-seq data from the TCGA dataset. DNA methylation genes were scored using the GSVA method, and this score was subjected to GSEA, GO and KEGG enrichment analysis. Then, in vitro experiments examined the effect of silencing LARS2 on the proliferation and apoptosis of HCT116 cells. We obtained 2635 genes, 144 were obtained after single factor screening, and 8 genes were obtained through random forest dimensionality reduction. They are LARS2, TEX2, BRIP1, QSOX2, HOOK1, COX19, NEK4 and STXBP4. Survival analysis indicated that patients with high Riskscore had poor prognosis. There were significant differences in the expression of DNA methylation genes between the two groups with high and low Riskscore. The immune checkpoints of different Riskscore groups include Antigen present, Ligand, Receptor, Co-inhibitor, Co-stimulator, Other, and Cell adhesion. There were significant differences in the expression of genes and DNA integrity checkpoint, histone deacetylation, mitotic DNA damage checkpoint, TGF-beta signaling pathway, Platinum drug resistance and Protein processing in endoplasmic reticlum pathway at the level of Antigen present. Silencing of LARS2 decreased the proliferative capacity and increased the apoptosis rate of HCT116. Our findings suggest that LARS2 methylation could affects colon cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10669-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/146fe9925f0f237535e6dcfdafa1d3f52c10c07d" target='_blank'>
              LARS2 DNA methylation predicts the prognosis in colon cancer
              </a>
            </td>
          <td>
            Yingzhou Fu, Wei Xie, Qifan Ren, Jiao Wu, Ying Long, Ying Liu
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Approximately 2.6% of live births in the United States are conceived using assisted reproductive technologies (ARTs). While some ART procedures, including in vitro fertilization (IVF) and intracytoplasmic sperm injection, are known to alter the epigenetic landscape of early embryonic development, their impact on DNA sequence stability is unclear. Here, we leverage the strengths of the laboratory mouse model system to investigate whether a standard ART regimen—ovarian hyperstimulation, gamete isolation, IVF, embryo culture, and embryo transfer—affects genome stability. Age-matched cohorts of ART-derived and naturally conceived C57BL/6J inbred mice were reared in a controlled setting and whole genome sequenced to ∼50x coverage. Using a rigorous pipeline for de novo single nucleotide variant (dnSNV) discovery, we observe a ∼30% increase in the dnSNV rate in ART-compared to naturally-conceived mice. Analysis of the dnSNV mutation spectrum identified signature contributions related to germline DNA repair activity, affirming expectations and evidencing the quality of our dnSNV calls. We observed no enrichment of dnSNVs in specific genomic contexts, suggesting that the observed rate increase in ART-derived mice is a general genome-wide phenomenon. Similarly, we show that the developmental timing of dnSNVs is similar in ART- and natural-born cohorts. Together, our findings show that ART is moderately mutagenic in house mice and motivate future work to define the precise procedure(s) associated with this increased mutational vulnerability. While we caution that our findings cannot be immediately translated to humans, they nonetheless emphasize a pressing need for investigations on the potential mutagenicity of ART in our species. SIGNIFICANCE STATEMENT This study investigates whether assisted reproductive technologies (ARTs) increase the risk of inherited genetic mutations in offspring. Using a well-controlled mouse model system, we compared the de novo mutation burden in genomes of mice conceived through ART to a naturally conceived cohort. We find a ∼30% increase in new DNA mutations in ART-conceived mice, suggesting that ART procedures have a genome destabilizing effect. This increase in mutation rate appears to be uniform across the genome, rather than attributable to specific genomic contexts. While we caution against the direct translation of our findings to humans, our work nonetheless highlights the need for further research into the genetic safety of ART in people.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdfbaa7908e3b1759956bbd1d3334471005f5d2" target='_blank'>
              Increased rate of de novo single nucleotide mutation in house mice born through assisted reproduction
              </a>
            </td>
          <td>
            Laura Blanco-Berdugo, Alexis C Garretson, Beth L. Dumont
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Melanomas are genetically heterogeneous, displaying mitogen-activated protein kinase mutations and homozygous loss of tumor suppressor genes. Mouse models combining such mutations produce fast-growing tumors. In contrast, rare, slow-growing tumors arise in mice combining Braf activation with heterozygous loss of Pten. Here we show that similar tumors can arise in albino mice bearing only a Braf mutation. Incidence kinetics suggest a stochastic event underlies tumorigenesis in tumors that arise with only a Braf mutation, yet de novo mutations or structural variants that could explain the incidence of most tumors could not be found. Single-cell transcriptomics of tumors identify a cell type resembling “neural crest-like” cells in human and mouse melanomas. These exist in normal mouse skin, expand upon Braf activation, and persist through serial transplantation; analyses of gene expression suggest they serve as precursors of malignant cells. This state may serve as an intermediate on a slow path to malignancy that may provide a diagnostically and therapeutically important source of cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a8b6d33ff26b499d62c39adf32521b06fa0db2" target='_blank'>
              Uncovering minimal pathways in melanoma initiation
              </a>
            </td>
          <td>
            Hui Xiao, Jessica Shiu, Chi-Fen Chen, Jie Wu, Peijie Zhou, Sahil S. Telang, Rolando Ruiz-Vega, Robert A Edwards, Qing Nie, A. D. Lander, Anand K. Ganesan
          </td>
          <td>2025-06-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="2’-deoxyuridine (dU) is a common DNA lesion resulting from cytosine deamination and from dUMP incorporation by DNA polymerases, both of which are prevalent in cancer. The primary mechanism that repairs dU lesions in genomic DNA is base excision repair initiated by Uracil-DNA Glycosylase 1 (UNG1). We generated Ung knockout mouse B16 melanoma cells to investigate the consequences of UNG deficiency in a well-characterized, immunoproficient, syngeneic mouse cancer model. We show that UNG-deficient (ΔUNG) B16 tumors have altered growth kinetics in vivo and that their delayed growth is T-cell dependent. Immune profiling revealed reduced CD8+ T cell infiltration but augmented CD4+ Th1 responses in ΔUNG tumors. In vitro, ΔUNG tumor cells exhibit strongly suppressed cell-intrinsic type-I interferon, type-II interferon, and inflammatory signaling gene expression signatures as well as altered cytokine and chemokine secretion. In vivo, ΔUNG tumors exhibit a modified inflammatory cytokine and chemokine milieu. Furthermore, ΔUNG tumor cells have altered sensitivity to exogenous interferons in vitro, with increased sensitivity to IFN-γ but decreased sensitivity to IFN-α/β. Collectively, our data show that tumor cell-specific UNG deficiency results in an altered tumor microenvironment in vivo and provide proof-of-concept data for the use of UNG inhibitors to modulate inflammatory pathways in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc26c13b6ac7757d4f5f6a8fbbc2a735a2ecafa" target='_blank'>
              Uracil-DNA glycosylase deficiency is associated with repressed tumor cell-intrinsic inflammatory signaling and altered sensitivity to exogenous interferons
              </a>
            </td>
          <td>
            F. Vendetti, Pinakin Pandya, Carina R. Sclafani, Reyna E. Jones, Daniel Ivanov, Robert W. Sobol, Christopher J. Bakkenist
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, , , Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, 
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gene regulation often entails a cooperative dynamic interplay among several protein molecules and several distinct DNA segments. Intersegment transfer of the Fis protein stimulates DNA inversion during DNA recombination. Individual DNA segments have been found to facilitate the dissociation of Fis proteins already bound to DNA and also allow for the transfer of the Fis between segments. Here, we use the hybrid coarse-grained AWSEM/3SPN.2C model to simulate the Fis protein intersegment transfer and explore its mechanism. We show that entropic effects within the Fis protein-DNA complex dictate the transfer pathway through specific structural configurations, involving specific grooves and consequent orientation constraints on the DNA segments. Multiple copies of the Fis protein facilitate intersegment transfer, explaining how changes in protein-DNA stoichiometry and concentration influence how the Fis-DNA complex architecture is established. This orientational dependence indicates that the assembly of the Fis-DNA complex mimics the interlocking of screws, functioning as a molecular machine that may couple to DNA supercoiling and torsional stress in the DNA generated by motor proteins, thus offering a potential regulatory mechanism for chromosomal organization and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a31cc6e4951420f062570d943b327e0539b5ca" target='_blank'>
              Intersegment Transfer and the Dynamical Architecture of Fis Protein-DNA Multimer Complexes.
              </a>
            </td>
          <td>
            Xun Chen, Shikai Jin, Cheng-Han Liu, Yaakov Levy, Min-Yeh Tsai, Peter G. Wolynes
          </td>
          <td>2025-08-05</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Endometrial carcinoma (EC), the most common gynecologic cancer type in developed countries, encompasses four molecular subtypes (POLEmut, MMRd, p53abn, and NSMP) that have prognostic values and guide treatment decisions. Additionally, dual loss of ARID1A and ARID1B (referred to as ARID1A/B) characterizes a significant portion of dedifferentiated/undifferentiated EC (DD/UDEC), a rare but highly aggressive subtype of EC. To advance the translational research for ECs, we analyzed the genomic features of a panel of 39 EC cell lines, leading to the identification of cell lines representing each of these EC molecular subtype. Histologic and immunohistochemical analyses of xenografted tumors from these cell lines confirmed their resemblance of cognate primary EC molecular subtypes. Further investigation of the publicly available genome-wide CRISPR screen data for EC cell lines identified multiple specific genetic dependencies in MMRd, p53abn, and ARID1A/B-dual deficient EC cell lines. Particularly, ARID1A/B-dual deficient DD/UDEC cells selectively rely on mitochondrial oxidative phosphorylation in vitro and in vivo. Therefore, through molecular subtyping of EC cell lines and subsequent characterization of molecular subtype-specific genetic dependencies, our study provides a framework that guides the utility of the EC cell line models for accelerating translational research in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1f3b25336fe4feba408a071575d0aa7bd9be8e" target='_blank'>
              Molecular subtyping of endometrial carcinoma cell lines uncovers subtype-specific targetable vulnerabilities
              </a>
            </td>
          <td>
            Eunice Li, Rebecca Ho, Ran Tao, Yannes Wai Yan Choi, Chae Young Shin, Shary Chen, Bengul Gokbayrak, J. Senz, Betty Yao, Liam Johnston, Spencer D Martin, Eric Yang, Mark S Carey, Bryan T. Hennessy, D. Huntsman, Ramon I Klein Geltink, Lynn Hoang, Yemin Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Diversification and selective propagation are the main driving forces of evolution, resulting in the emergence of organisms possessing various strategies. Here, we conducted in vitro evolution of the replication origin (oriC) under the pressure of an AT-rich mini-chromosome amplification using a reconstituted Escherichia coli replication-cycle reaction (RCR). Using next-generation sequencing, we identified that these evolved oriCs contain select mutations within the duplex unwinding element (DUE) and DnaA-binding site (DnaA box) regions, which are crucial for replication initiation. Real-time detection of RCR amplification (real-time RCR) revealed that the DUE mutations, which decreased the GC content, along with the introduction of a specific A/T sequence in DUE-M and a consecutive KAK (K = T or G) motif in DUE-R, enhanced the RCR amplification efficiency compared to the wild-type oriC (oriCwt). A competitive amplification assay also elucidated that the DnaA box mutations confer a competitive advantage over coexistent oriCwt. Although these DUE and DnaA box mutations were selected through the same amplification reaction, they exhibited distinct competitive amplification strategies. The DUE mutant represents a faster propagation strategy (r-strategy), while the DnaA box mutant represents an adaptive strategy with a competitive advantage (K-strategy), representing the r/K-selection theory in molecular evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d8adc8e642b9b4fca6ba9da0b2f823fd1341a1" target='_blank'>
              Adaptive evolution of oriC through in vitro propagation of a mini-chromosome in RCR
              </a>
            </td>
          <td>
            Shota Suzuki, M. Su’etsugu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT The Ribosomal DNA (rDNA) in mammals is organised into large clusters of tandem repeats each of which encodes a single 47S precursor for the 18S, 5.8S, and 28S ribosomal RNAs (rRNAs) that is flanked upstream and downstream by an Intergenic Spacer (IGS) originally referred to as the Non‐Transcribed Spacer (NTS). However, in certain cells and under certain environmental conditions the IGS has been found to be transcribed at low level to generate a range of “Noncoding” RNAs (ncRNAs). These ncRNAs have been implicated in the regulation of rRNA synthesis, rDNA silencing and protein sequestration in response to environmental and oncogenic stresses and tumour suppression. Here we review data on the generation, regulation and potential functions of these ncRNAs. We suggest that the majority of the ncRNAs originate from a failure of RNA polymerase I transcription termination by the Reb1‐ and Myb‐related transcriptional “road‐block” factor TTF1 and link their expression with tumour suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4490c7ffdc58697d4328badb68a0f8a348678695" target='_blank'>
              Interpreting the Origins and Functions of Noncoding RNAs From the Ribosomal Genes
              </a>
            </td>
          <td>
            Tom Moss, Dany S. Sibai, Frédéric Lessard
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular Physiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR/Cas technology has revolutionized genome engineering, unlocking unprecedented therapeutic potential. However, beyond well-documented concerns of off-target (OT) mutagenesis, recent studies reveal a more pressing challenge: large structural variations (SVs), including chromosomal translocations and megabase-scale deletions, particularly in cells treated with DNA-PKcs inhibitors. These undervalued genomic alterations raise substantial safety concerns for clinical translation. As more CRISPR-based therapies progress toward the clinic, understanding and mitigating these risks is paramount. Here, we review emerging evidence on on-target aberrations and chromosomal translocations, identify key gaps in our understanding of the DNA repair pathways underlying these adverse effects, and discuss strategies to improve the safety of genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34e819820702e599ff9b58db37b63e7aab2e777" target='_blank'>
              The hidden risks of CRISPR/Cas: structural variations and genome integrity
              </a>
            </td>
          <td>
            Clotilde Aussel, Toni Cathomen, Carla Fuster-García
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Auboiron et al. developed a CRISPR-Cas9-based system in yeast Saccharomyces cerevisiae that allows precise, dose-dependent induction of DNA double-strand breaks by targeting Ty retrotransposons. This system combines sequence specificity with control over break numbers (×1, ×15, or ×59), enabling a more detailed and precise study of DNA damage response. It revealed that the key DNA damage response kinase, Tel1, localizes to the nuclear periphery and forms multiple foci after induction of DNA double-strand breaks. The system also exposed limitations in Cas9 availability at higher break levels and offers a scalable platform for studying genome stability across organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914334ccbc709e992e8c671a50009bcad26862ee" target='_blank'>
              A CRISPR-Cas9-based tool for dose-dependent DNA damage detection.
              </a>
            </td>
          <td>
            Valentyn Oksenych, P. Petakh, Denis E. Kainov, O. Kamyshnyi
          </td>
          <td>2025-07-13</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: The long-term goal of chronic hepatitis B research is a functional cure (HBsAg seroclearance). Although currently used nucleos(t)ide analogs can efficiently inhibit viral replication, they do not reduce viral RNAs or proteins produced from covalently closed circular DNA (cccDNA), and rarely achieve a functional cure. To overcome this situation, revealing the mode of the existence of cccDNA is required, including identifying the interreacting proteins with cccDNA. Here, we aimed to identify novel proteins that interact with cccDNA. Methods: Using an in vitro HBV infection model and a sequence-specific proximity labelling method consisting of dead Cas9 and biotin identification (BioID2), we comprehensively determined proteins that possibly interact with cccDNA. After identifying the candidate proteins, the HBV RNA transcription levels were examined by knocking out the associated genes. Results:﻿ We identified ABHD14B as a protein that interacts with cccDNA and inhibits HBV RNA transcription from cccDNA. ABHD14B decreases the acetylation levels of histone proteins that control the transcription levels of HBV RNA in cccDNA. Moreover, ABHD14B interacts with TFII-I, which binds directly to cccDNA in a sequence-dependent manner. These results suggest that the host protein, ABHD14B, is recruited to cccDNA via the TFII-I protein, inhibiting HBV RNA transcription from cccDNA by deacetylating cccDNA histones. Conclusions: ABHD14B was newly identified as a suppressor of HBV RNA transcription from cccDNA, which may improve our understanding of the mode of existence of cccDNA, providing a basis for development of a functional cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bd95fe07e8a95469056b65d704b3979e1b6c0b9" target='_blank'>
              CRISPR-mediated proximity labeling unveils ABHD14B as a host factor to regulate HBV cccDNA transcriptional activity
              </a>
            </td>
          <td>
            Kazuma Sekiba, Nozomi Miyake, Yu Miyakawa, Chikako Shibata, Takahiro Seimiya, T. Kishikawa, Mitsuhiro Fujishiro, Motoyuki Otsuka
          </td>
          <td>2025-07-14</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA exists biologically as a highly dynamic macromolecular complex subject to myriad chemical modifications that alter its physiological interpretation, yet most sequencing technologies only measure Watson-Crick base pairing interactions. Third-generation sequencing technologies can directly detect novel and modified bases, yet the difficulty and cost of training these techniques for each novel base has so far limited this potential. Here, we present a method based on barcoded split-pool synthesis to generate reference standard oligonucleotides allowing novel base sequencing. Using novel base detection, we perform multidimensional sequencing to retrieve information, both physiologically stored and experimentally encoded, from DNA, allowing us to characterize the preferential replication of deleterious mitochondrial genome mutations, the infection dynamics of a host-pathogen model, and the effect of chemotherapy on cancer cell DNA at the single molecule level. The low cost and experimental simplicity of this method make this approach widely accessible to the research community, enabling complex experimental interrogation across the biological sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72de8476da20b409721bf544eb2f184f300e0091" target='_blank'>
              Multidimensional third-generation sequencing of modified DNA bases allows interrogation of complex biological systems
              </a>
            </td>
          <td>
            Serena S David, Brendan A Pacheco, Kensei Kishimoto, Sam Vantine, Kai Hu, Haibo Liu, Diana L Davis, Hoang Tran, Benjamin F Sallis, Levi Ali, Cole M. Haynes, Beth A McCormick, L. Zhu, William A Flavahan
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Long interspersed nucleotide element 1 (LINE1), the most abundant repetitive element in the human genome, plays a crucial role in genomic instability. Aberrant LINE1 activation, primarily regulated by DNA methylation, is a hallmark of cancer. Non-small cell lung cancer (NSCLC), the most prevalent form of lung cancer worldwide, continues to pose significant challenges due to the invasiveness, high cost, and susceptibility to false positives of current diagnostic methods, as well as the emergence of treatment resistance. This review highlights the potential of LINE1 methylation as a biomarker for NSCLC, offering novel insights into its role in diagnosis, prognosis, and therapeutic strategies. Recent studies uncovered that LINE1 hypomethylation was strongly associated with poor overall survival, suggesting its utility as both a prognostic marker and a therapeutic target. However, further research is required to elucidate its precise regulatory mechanisms in LINE1 retrotransposition and to evaluate its potential as a non-invasive biomarker for improving NSCLC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4219f213fb15eaa64e362c434180801a85ab734e" target='_blank'>
              Long interspersed nuclear element 1 methylation in non-small cell lung cancer: implications for diagnosis, prognosis, and therapeutic targeting
              </a>
            </td>
          <td>
            Dileesha Prabani Wanasundara Arachchillage, Wanvisa Udomsinprasert
          </td>
          <td>2025-07-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Structural variations (SVs) represent genomic variations that involve breakage and rejoining of DNA segments. SVs can alter normal gene dosage, lead to rearrangements of genes and regulatory elements within a topologically associated domain, and potentially contribute to physical traits, genomic disorders, or complex traits. Recent advances in sequencing technologies and bioinformatics have greatly improved SV detection and interpretation at unprecedented resolution and scale. Despite these advances, the functional impact of SVs, the underlying SV mechanism(s) contributing to complex traits, and the technical challenges associated with SV detection and annotation remain active areas of research. This review aims to provide an overview of structural variations, their mutagenesis mechanisms, and their detection in the genomics era, focusing on the biological significance, methodologies, and future directions in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005d90812c5790ee3488e2503f389fb8c3ab57b" target='_blank'>
              Structural Variants: Mechanisms, Mapping, and Interpretation in Human Genetics
              </a>
            </td>
          <td>
            Shruti Pande, Moez Dawood, Christopher M. Grochowski
          </td>
          <td>2025-07-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="PRMT5 is a member of a class of enzymes called protein arginine methyltransferases (PRMTs) that play a role in maintaining genomic stability through post-translational modification of components of the TIP60 chromatin remodeling super complex. TIP60 is required primarily for chromatin remodeling at DNA double-strand breaks. Mutations in either TIP60 or PRMT5 affect repair by homologous recombination. A recent study has shown that in leukemia and lymphoma, PRMT5 also controls mRNA expression of TIP60 and other DNA repair genes by regulating alternative splicing. This analysis utilizes publicly available data from the Catalogue of Somatic Mutations in Cancer (COSMIC) to interrogate how PRMT5 expression correlates with expression levels of other key DNA repair genes: KAT5 (TIP60), H2A (H2AFX), TP53, TP53BP1, and RAD51 in all cancers. We find that indeed an increase in PRMT5 expression levels correlates with an increase in KAT5 levels. A weak correlation was also observed between PRMT5 and TP53, TP53BP1, and RAD51. These findings extend previous PRMT5 roles in controlling gene expression to all cancer types and further highlight the role of this enzyme not only in post-translational modification but also regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240e36d96cf32cd6101ee0c2c3177761bcf3950" target='_blank'>
              Regulation of DNA repair gene expression by PRMT5
              </a>
            </td>
          <td>
            Hunter J. Bliss, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-07-14</td>
          <td>microPublication Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pancreatic cancer is known for its lethal condition, with most cases being diagnosed at advanced stage. Recently, liquid biopsy has emerged as a promising tool in cancer detection. Here we develop both an early detection model and a prognostic model for pancreatic cancer using cell-free DNA (cfDNA) end motif, fragmentation, nucleosome footprint (NF), and copy number alteration (CNA) features from plasma cfDNA. A total of 975 individuals were enrolled in our study. We developed an integrated model that demonstrated superior performance in distinguishing patients with early-stage pancreatic cancer from non-cancer controls. Moreover, we find that cfDNA features are associated with prognostic outcomes among pancreatic cancer patients. In this study, a cfDNA-based liquid biopsy signature is established for the early detection and prognostic prediction of pancreatic cancer. CfDNA may become a valuable tool for enhancing early diagnosis and prognosis assessment in this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2b38ea3144ac464585d901d0e6435daee2918d3" target='_blank'>
              Cell-free DNA testing for the detection and prognosis prediction of pancreatic cancer
              </a>
            </td>
          <td>
            Jianmin Wu, Xiongfei Xu, Qingzheng Zhang, Peilong Li, Tong Wu, Shiwei Guo, Lutao Du, Dongdong Xue, S. Shen, Fuming Sun, Ji Hu, Lu Zheng, Xuan Wu, Jian Bai, Yin Wang, Lin Wu, Wei-wen Liu, Hongyang Wang, Gang Jin, Lei Chen
          </td>
          <td>2025-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e7f45bb9128c765dc6171cc504564b0c6834197" target='_blank'>
              SMCHD1 compacts DNA directly in an ATP-regulated manner
              </a>
            </td>
          <td>
            Joel Ng, Xiaodan Zhao, Xuyao Liu, Aghil Soman, Lars Nordenskiöld, Jie Yan, Shifeng Xue
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common renal malignancy. However, the combined clinical and biological scores commonly used to predict the clinical outcome are imperfect and need improvement. The main goal of our study was to assess the effect of mtDNA genetics on the prognosis of ccRCC patients and to explore morphologic correlation. Mitochondrial DNA copy number (mtDNAcn) variation between tumor and paired matched healthy kidney tissue was assessed by real-time quantitative PCR and expressed as a ratio in 105 patients. According to this median ratio, the cohort was divided into two groups: “LOW” (n = 53) and “HIGH” (n = 52). Cancer-Specific Survival (CSS) and Disease-Free Survival were assessed in each group. The tumor samples were classified into two subtypes (Clear or Eosinophilic cells) according to the cytoplasmic morphology. CSS was significantly reduced in the “HIGH” than in the “LOW” group with respective 5-year survival rates: 78.7% (CI 95: 64.8–95.5) and 95.5% (CI 95 87.1–100.0) (Hazard Ratio: 7.4 (CI 95: 1.9–29.9, p = 0.027*) in multivariate analysis, including pathological classification, tumor size, International Society of Urological Pathology grade, lymphovascular invasion, dedifferentiated pattern, necrosis and adjuvant therapy. Next-generation sequencing of mtDNA was performed on 14 tumors and matched healthy kidney tissue. No hotspot mutation or redundant large deletion was found. None of the variants or large deletions identified had an impact on prognosis. MtDNAcn variation in tumor relative to normal kidney appears as an independent prognostic factor in ccRCC, which was also associated with eosinophilic morphology. MtDNA content could be considered an additional prognostic factor, in combination with other predictive parameters. Furthermore, these results underline the importance of the role of mitochondria in ccRCC and the need for further functional studies to understand the pathophysiological mechanisms better and consider therapies targeting mitochondrial metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288c14d8ba3e745b3421617281d655151662ced6" target='_blank'>
              MtDNA copy number enrichment is associated with poor prognosis and eosinophilic morphology in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Sarah Bellal, C. Rolley, Jérémy Richard, Nolwenn Bounaix, Vincent Le Corre, Marie-Christine Copin, Odile Blanchet, Pierre Bigot, Vincent Procaccio, C. Bris
          </td>
          <td>2025-07-23</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b94cf63206b221ede960c719c7b5d998ebadf2b" target='_blank'>
              Specific TP53 mutations impair the recruitment of 53BP1 to DNA double-strand breaks underlying the mechanism of radioresistance.
              </a>
            </td>
          <td>
            Paolo Fagherazzi, L. Stixová, E. Bártová
          </td>
          <td>2025-07-14</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved by specialized helicases. Previous in vitro studies showed that DEAH-box helicase 36 (DHX36/G4R1/RHAU) has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in the accumulation of cytoplasmic DNA fragments, an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression, and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 messenger RNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd74a217dd008b29b89824a440aee3aba255159" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, Shangming Tang, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Yuh-Hwa Wang
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for removing bulky DNA damage, such as pyrimidine dimers induced by ultraviolet (UV) light. The NER pathway excises the damaged strand through incisions at the 5’ and 3’ ends of the damage, with the 5’ incision catalyzed by the XPF-ERCC1 endonuclease complex. Here, we identify an XPF ortholog in Trypanosoma brucei, the causative agent of human African trypanosomiasis (sleeping sickness). XPF-deficient parasites exhibit hypersensitivity to UV irradiation and a slower rate of DNA damage repair. Consistent with its role in DNA repair, XPF localizes to the nucleus, associating with nucleoplasmic and nucleolar regions. Additionally, we demonstrate that TbXPF protects against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C, respectively. The presence of a functional NER pathway in trypanosomes suggests that these organisms are susceptible to replication- and transcription-blocking DNA damage in vivo. Under genotoxic stress, genome stability and parasite survival may heavily rely on DNA repair systems such as NER which, for this reason, could be an effective target for chemotherapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdd4ff3d6a3681f1e698b78ba5f4c131040ae39" target='_blank'>
              Role of the nucleotide excision repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, Laura Montosa-Hidalgo, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="STUDY QUESTION
Are segmental aneuploidies identified in human embryos more likely to occur within known fragile sites of the genome?


SUMMARY ANSWER
Segmental breaks in the autosomes of human preimplantation embryos occur more frequently in known fragile areas of the genome.


WHAT IS KNOWN ALREADY
Fragile sites represent specific loci in the genome characterized by inhibition of DNA synthesis when exposed to known inhibitors and are particularly sensitive to replication stress and instability.


STUDY DESIGN, SIZE, DURATION
This was a retrospective analysis of single nucleotide polymorphism (SNP) array-based preimplantation genetic testing data from biopsies performed on 2066 human blastocysts in 98 assisted reproduction laboratories around the world from September 2019 to January 2023.


PARTICIPANTS/MATERIALS, SETTING, METHODS
This multicenter study included eligible patients undergoing IVF with preimplantation genetic testing (PGT), in which at least one embryo was diagnosed with a segmental aneuploidy. The mean maternal age was 36.4 years (SD 4.1), ranging from 25 to 44 years. These samples were processed on high-density SNP arrays. Chromosome level copy number and B allele frequency (BAF) plots from these embryos were used to determine segmental aneuploidy breakpoints. Known fragile sites catalogued by the HumCFS database were used for correlation analyses.


MAIN RESULTS AND THE ROLE OF CHANCE
Overall, a side-by-side pairing of observed breakpoints and known fragile sites demonstrated a strong concordance (r = 0.81, 95% CI [0.6, 0.92]). A chi-square test for independence for stratified groups showed a highly significant correlation between all observed breakpoints and known fragile sites (597 expected vs. 848 observed; P < 0.001) and for telomeric breaks alone (521 expected vs. 784 observed; P < 0.001). Observed interstitial breaks alone were not correlated to expected breakpoints (75 expected vs. 64 observed; P > 0.05).


LIMITATIONS, REASONS FOR CAUTION
These findings should be interpreted with caution, as limitations in genomic resolution may bias detection and classification of smaller segmental aneuploidies. Additionally, this study touched upon the distribution of meiotic to mitotic breakpoints in human blastocysts as they relate to known fragile sites. Since meiotic aneuploidies increase with advanced maternal age and many IVF patients undergoing PGT-A testing fall in this category, a sampling bias should be considered for this specific metric.


WIDER IMPLICATIONS OF THE FINDINGS
Demonstrating that segmental aneuploidies significantly correlate with known fragile sites highly susceptible to replication stress offers insight into the origin of subchromosomal imbalances and hints at the influence of stressors on reproductive success.


STUDY FUNDING/COMPETING INTEREST(S)
This study received no external funding and was fully supported by the participating authors and their affiliated institutions. The authors declare no conflicts of interest related to this study.


TRIAL REGISTRATION NUMBER
N/A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f488ce17d9309a0718d17d57c1fee173909b44" target='_blank'>
              Blastocyst segmental aneuploidy breakpoints are highly correlated with human genome fragile sites.
              </a>
            </td>
          <td>
            Elaine de Quadros, Jia Xu, Nathan Treff, Diego Marin, Arielle Freedman, C. Milevski, Kathleen Miller, Minglei Bian
          </td>
          <td>2025-07-30</td>
          <td>Human reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract The MYC and RAS oncogenes were found early on to cooperate in the transformation of nonimmortalized primary cells, and in tumor development in transgenic mouse models. MYC prevents RAS-induced senescence. Moreover, tumor regression resulting from the suppression of MYC expression is associated with cellular senescence. How MYC prevents RAS-induced senescence and why suppression of MYC in tumors causes senescence remains to be elucidated. Here, we show that MYC interacts with the Pol β DNA polymerase and stimulates its enzymatic activities to accelerate the repair of oxidative DNA damage by the base excision repair (BER) pathway, thereby enabling RAS-driven cancer cells to avoid senescence and continue to proliferate despite producing excess levels of reactive oxygen species. The carboxy-terminal domain of MYC, which is needed for heterodimerization with its MAX partner and DNA binding, is not required for the DNA repair activity of MYC. Ectopic expression of MYCΔCTD accelerates DNA repair, protects against RAS-induced senescence, and cooperates with RAS in the transformation of primary cells, whereas MYC mutants inactive in DNA repair exhibit weaker or no activity in these assays. These results demonstrate that the function of MYC in BER plays an important role in the MYC–RAS cooperation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c13c7980326ac55cbce4b685dfa42090cd38be7" target='_blank'>
              The function of MYC in base excision repair protects against RAS-induced senescence
              </a>
            </td>
          <td>
            C. C. F. Faraco, Wanting Zhu, Anne-Marie Fortier, Z. Ramdzan, Elise Vickridge, Virakbott Ong, Félix Goudreau, Billel Djerir, Jérôme Bellemare, Shujing Gu, Huanyi Zhang, Brittany L Carroll, Shreya S Krishnan, L. Leduy, Alba Guarné, Morag Park, Alexandre Maréchal, Alain Nepveu
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The B Cell Lymphoma-2 (BCL-2) family proteins are central regulators of apoptosis, and their dysregulation is frequently associated with cancer progression and resistance to therapy. While small molecules like venetoclax have shown promise, nucleic acid-based therapeutics targeting BCL-2 remain underexplored. Here, we report a novel in vitro evolution strategy to generate trans-acting RNA-cleaving DNAzymes targeting natural BCL-2 mRNA without requiring covalent substrate-linking. Using a 50-base region of BCL-2 mRNA as a selection target, we evolved several DNAzymes that demonstrate significant RNA cleavage activity. These DNAzymes downregulated BCL-2 expression, induced apoptosis, and reduced cell viability in HepG2 and MCF-7 cancer cells. In vivo, our novel DNAzymes significantly suppressed tumor growth in a syngeneic mouse breast cancer model, with efficacy comparable to 5-Fluorouracil. This study presents a proof of concept for a novel strategy to evolve functional DNAzymes against native mRNA sequences and highlights their potential as gene-silencing tools in cancer therapy. Future studies will explore the therapeutic potential of these findings in cancer patients. Additionally, investigating the underlying molecular mechanisms in more complex cancer models will further validate the observed effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f088a27b7dc74b3baf7aba2745350a95e2e2501f" target='_blank'>
              Novel In Vitro Selection of Trans-Acting BCL-2 mRNA-Cleaving Deoxyribozymes for Cancer Therapy
              </a>
            </td>
          <td>
            Veera Vijaya Basamshetty, Vijay Kumar Gangipangi, Uppulapu Shravan Kumar, Santhosh Shanthi Bhupathi, S. R. Kaulagari, Prashant Giri, Swapnil Sinha, Utpal Mohan, Konstantinos Sdrimas
          </td>
          <td>2025-06-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are distinct nucleic acid secondary structures formed by guanine-rich sequences in both DNA and RNA. These structures readily form and fulfill diverse biological functions. The structural diversity of G4s is influenced by several factors, including their strand orientation, glycosidic bond angles, and loop configurations. G4s are widely distributed in functionally significant genomic regions, including telomeres, promoter regions, exons, 5' untranslated region (5' UTR), intron region, and 3' untranslated region (3' UTR). G4s are implicated in critical biological processes, including telomere elongation, DNA replication, DNA damage repair, transcription, translation, and epigenetic regulation. This overview offers a comprehensive analysis of the determinants of G4 structure and their impact on associated biological processes. Briefly, it describes the effects of G4s on cancers, viruses, and other pathogenic substances. This overview aims to contribute new ideas for the regulation of related mechanisms and their potential impact on the treatment strategies of related diseases. This article is categorized under: RNA Structure and Dynamics > RNA Structure, Dynamics and Chemistry RNA Structure and Dynamics > Influence of RNA Structure in Biological Systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0eb5827dbe52a8e0e2bc9b47ded6461cd148b5b" target='_blank'>
              G-Quadruplexes in Gene Regulation and Cellular Function.
              </a>
            </td>
          <td>
            Qi Wen, Lijin Guo, Farhad Bordbar, Qinghua Nie
          </td>
          <td>2025-07-01</td>
          <td>Wiley interdisciplinary reviews. RNA</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The mammalian Tumor Susceptibility Gene 101 (TSG101) encodes a protein with diverse functions that control the proliferation and survival of cells, but its role in malignant transformation and cancer development has remained enigmatic. Methods To study the pro-tumorigenic functions of TSG101, we developed a bi-transgenic mouse model that expresses exogenous TSG101 along with a luciferase reporter in a ligand-controlled manner in the mammary gland epithelium. We performed a comprehensive histopathologic, biochemical, and molecular characterization of ductal hyperplasia and mammary tumors. Unsupervised hierarchical clustering based on 1,723 intrinsic genes of ten TSG101-overexpressing cancers alongside 251 tissue samples representing 31 reference mammary tumor models and normal mammary glands was conducted. Results Females overexpressing TSG101 develop ductal hyperplasia, adenomyoepitheliomas, and palpable adenosquamous carcinomas at an average latency of approximately ten months. These metaplastic mammary tumors are comprised of transforming basal and luminal epithelial cells. Using a GFP reporter strain to monitor the transgene activation at the single-cell level, we determined that the epithelial heterogeneity within transforming ducts and ensuing carcinomas originated from the luminal epithelium. At the molecular level, TSG101-induced mammary tumors are triple-negative and exhibit gene expression signatures of Wnt and inflammatory cytokine signaling, which are key regulators of epithelial cell fate. The ligand-controlled downregulation of exogenous TSG101 in established carcinomas led to tumor regression. We demonstrated that the TSG101-mediated activation of PI3K/AKT signaling, as well as upregulation of Cyclin D1 and MDM2, are dependent on the perpetual expression of the TSG101 oncoprotein. Conclusions The collective findings of this study provide in vivo evidence that TSG101 possesses pro-tumorigenic properties that extend to cancer progression and maintenance, suggesting that this protein could be a rational molecular target to prevent and treat a subset of mammary tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02007-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca345fd462ac7f96e7ae408ea2e3598885c80b06" target='_blank'>
              Overexpression of TSG101 causes the development of adenosquamous mammary carcinoma
              </a>
            </td>
          <td>
            Rayane Dennaoui, Patrick D. Rädler, Madison N. Wicker, Kerry Vistisen, R. Ferraiuolo, A. Triplett, Hridaya Shrestha, Tessa A. Liner, Karoline C. Manthey, Hallgeir Rui, R. Cardiff, Teresa M. Gunn, C. Perou, Kay-Uwe Wagner
          </td>
          <td>2025-07-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="Abstract DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the mechanisms by which DDAs induce cell death are not fully understood, and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and decreasing tRNALeu(TAA) levels. Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), triggering ER stress and protein aggregate formation, leading to cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11-knockout or transfection of tRNALeu(TAA). Proteomic analysis suggests tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bd7e8511d20689e4f33be87517895928d1815" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, Atsuo T Sasaki, Akio Kanai
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer with poor prognosis and limited therapeutic options. DNA damage-based radiotherapy plays a significant role in the treatment of TNBC. However, radioresistance continues to pose a significant challenge, often resulting in the unsuccessful outcome of radiation therapy for patients with TNBC. Type I gamma phosphatidylinositol phosphate kinase (PIPKIγ), a key enzyme in phosphoinositide metabolism, is associated with poor prognostic outcomes in TNBC. Here, we discovered that PIPKIγ overexpression significantly boosts nonhomologous end joining (NHEJ), a principal mechanism for repairing DNA double-strand breaks (DSBs). This enhancement of NHEJ confers increased radioresistance in TNBC. At the molecular level, PIPKIγ directly interacts with LIG4, a crucial NHEJ component, and strengthens its interaction with XRCC4, a key regulator of LIG4 nuclear translocation. This facilitates LIG4’s nuclear translocation, improving DNA repair efficiency and genomic stability in TNBC cells. Furthermore, elevated PIPKIγ levels enhance radioresistance in TNBC cells and tumors in xenograft models, whereas depleting PIPKIγ has the opposite effects. These insights reveal a new mechanism by which PIPKIγ promotes radioresistance through the facilitation of DSBs repair in TNBC. Collectively, our findings suggest that the PIPKIγ-LIG4 interaction represents a potential therapeutic target for improving radiotherapy efficacy in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3e4673049a1a5be98ac7c58399992250f7d541" target='_blank'>
              PIPKIγ promotes non-homologous end joining through LIG4 to enhance radiotherapy resistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wenge Dong, Haiping Zhang, Zhigang Zhuang, Ying Jiang
          </td>
          <td>2025-07-31</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background In-solution hybridization enrichment of genetic markers is a method of choice in paleogenomic studies, where the DNA of interest is generally heavily fragmented and contaminated with environmental DNA, and where the retrieval of genetic data comparable between individuals is challenging. Here, we benchmark the commercial “Twist Ancient DNA” reagent from Twist Biosciences using sequencing libraries from ancient human samples of diverse demographic origin with low to high endogenous DNA content (0.1–44%). For each library, we tested one and two rounds of enrichment and assessed performance compared to deep shotgun sequencing. Results We find that the “Twist Ancient DNA” assay provides robust enrichment of approximately 1.2M target SNPs without introducing allelic bias that may interfere with downstream population genetics analyses. Additionally, we show that pooling up to 4 sequencing libraries and performing two rounds of enrichment is both reliable and cost-effective for libraries with less than 27% endogenous DNA content. Above 38% endogenous content, a maximum of one round of enrichment is recommended for cost-effectiveness and to preserve library complexity. Conclusions In conclusion, we provide researchers in the field of human paleogenomics with a comprehensive understanding of the strengths and limitations of different sequencing and enrichment strategies, and our results offer practical guidance for optimizing experimental protocols. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03622-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c477c495125e3cd66f843708e054b0141348cbd8" target='_blank'>
              Optimized in-solution enrichment of over a million ancient human SNPs
              </a>
            </td>
          <td>
            Roberta Davidson, Xavier Roca-Rada, Shyamsundar Ravishankar, Leonard Taufik, Christian Haarkötter, Evelyn Collen, Matthew P. Williams, Peter Webb, M. I. Mahmud, Erlin Novita, Idje Djami, Gludhug A. Purnomo, Cristina Santos, A. Malgosa, Linda R. Manzanilla, Ana Maria Silva, Sofia Tereso, Vítor Matos, Pedro C. Carvalho, Teresa Fernandes, A. Maurer, João C. Teixeira, Ray Tobler, Lars Fehren-Schmitz, B. Llamas
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Super-enhancers are a super-cluster of enhancers formed by serially arranged regulatory elements that can strongly drive the expression of cell-related genes. Hundreds of SEs in cells affect cell identity and fate-determining processes. Previous studies have verified that the expression of pathogenic genes is highly correlated with the abnormal activation of SEs in malignant tumorigenesis, dementia, diabetes, and many autoimmune diseases. Also, enhancer RNAs (eRNAs) can be regarded as crucial markers for SEs. Here, we summarize the discovery process and basic concepts of SEs, describe the structural characteristics and functional regulation of SEs in different tumor diseases, Alzheimer’s disease, and immune-related diseases, with a focus on typical diseases such as rheumatoid arthritis, systemic lupus erythematosus, and multiple sclerosis. In this review, we also discuss the potential clinical applications of SE, as well as the research prospects in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f255bb941ea062fdb933b412745b2b17de7146b1" target='_blank'>
              Current progress and future perspective of super-enhancers: a viable and effective bridge between the transcriptional apparatus and disease
              </a>
            </td>
          <td>
            Ran Wang, Aiying Li, Zongran Pang
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objectives Senescent cells release a senescence-associated secretory phenotype, promoting polyploid giant cancer cells (PGCCs) to emerge, fostering tumor heterogeneity and resistance. Pentagamavunone-1 (PGV-1) emerges as a promising agent inducing senescence and prometaphase arrest, resulting in permanent cytotoxicity. This study was aimed to investigate the effect of PGV-1 in dysregulating mitosis through the modulation of PGCCs and senescence in low MYCN-expressing hepatocellular carcinoma (HCC) cells JHH4. Materials and Methods To assess the physiological effects of PGV-1, several in vitro tests were done including the MTT assay, cell cycle assay, May-Grünwald-Giemsa staining, senescence associated-β-galactosidase (SA-β-Gal) assay, and apoptosis assay. The protein levels of the apoptosis regulatory protein were evaluated using western blot analysis. The interaction of PGV-1 toward the protein that plays a role in PGCCs formation was simulated by molecular docking and molecular dynamics (MD). Results The cytotoxic assay revealed that PGV-1 inhibited the proliferation of JHH4 liver cancer cells permanently. Inhibition of cell proliferation by PGV-1 was associated with the modulation of G2/M phase, particularly mitotic arrest and formation of PGCCs. The SA-β-Gal verified that PGV-1 induced senescence in cells (p<0.01), inducing PGCCs formation. Apoptotic mechanisms were validated via Annexin V staining, which showed the level of cleaved poly (ADP-ribose) polymerase (p<0.001). Molecular docking and MD simulations suggested that PGV-1 could interfere with the conformation of the chromosomal passenger complex (CPC) protein, particularly disrupting essential interactions within the inner centromere protein, Survivin, and Borealin. Conclusion PGV-1 induced strong cytotoxicity in HCC cells by disrupting mitosis leading to PGCC formation, senescence, and subsequent apoptotic cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502c9c775859944526ea05e10cf871f8f1529c85" target='_blank'>
              Effect of PGV-1 on Apoptosis via Mitotic Arrest and Senescence in Polyploid Giant Cancer Cells of Hepatocellular Carcinoma JHH4
              </a>
            </td>
          <td>
            Nadzifa Nugraheni, Beni Lestari, E. Meiyanto, R. Y. Utomo
          </td>
          <td>2025-06-01</td>
          <td>Turkish Journal of Pharmaceutical Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Maintaining genome integrity is essential for the proper functioning and development of organisms. An intriguing aspect is that neocentromeres can form at non-centromeric sites. CENP-A, a key epigenetic marker of centromeres, is often mislocalized to ectopic sites in cancers when overexpressed. Its deposition on centromeres relies on transcription of centromeric non-coding RNAs. Subsequently, ectopic CENP-A is frequently found at transcriptionally active and chromosome breakpoint regions. We previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing PCAT2, a non-centromeric oncogenic lncRNA that recruits CENP-A to its transcribing locus. We tracked cells with this transgene to analyze the longevity of ectopic CENP-A. We discovered that this induced epigenetic memory was lost due to suppression by epigenetic silencing mechanisms, restoring CENP-A to previous levels. These findings suggest that cells have mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription unless selective pressure favors it">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7b7b4ef76af3d3bde1c36b2a65ab677cc6c45" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-07-31</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Amplification of human chromosome 20q has been reported as the most frequently recurring genetic abnormality associated with large scale changes in mRNA and protein levels in sporadic colorectal carcinomas. While some studies have found 20q amplification to be consistent between primary and metastatic samples from the same patient with a role in the development of metastasis and worse patient prognosis, others have reported association with improved overall survival for a subset of these patients with colorectal cancer (CRC). To fine map the Minimal Common Regions (MCRs) of amplification on chromosome 20q and identify the candidate genes playing roles in progression of the disease, microarray comparative genomic hybridization analyses of two in vitro cultured CRC liver metastasis cell line model systems was utilized. Microarray expression analysis led to the identification of a candidate gene signature comprising of four genes, BMP7, DNMT3B, UBE2C and YWHAB, residing in the MCRs that were over expressed in CRC cells. By validating our results in a training set of 23 adenocarcinomas (tumors) and five adenomas (polyps) using reverse transcription-quantitative PCR, as well as analyses of two larger colorectal cancer test data sets derived from 195 The Cancer Genome Atlas and 182 MD Anderson Cancer Center patients with colorectal adenocarcinoma patients, this gene signature was ascertained to be associated with lymph node spread and/or distant metastasis (P<0.05). Previously reported functional studies of the gene signature indicated their involvement in inflammatory and immune response pathways driving CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea2107086249d394733ab4fbf6c82be7552a16" target='_blank'>
              Chromosome 20q gene signature associated with colorectal cancer progression
              </a>
            </td>
          <td>
            Jennifer Carter Jones, A. Hegde, Yu-Jing Huang, G. Manyam, Vibhuti Srivastava, J. Song, Yulan Cheng, Ralf Krahe, Warapen Treekitkarnmongkol, Stephen J. Meltzer, S. Kopetz, Stanley R Hamilton, Hiroshi Katayama, Subrata Sen
          </td>
          <td>2025-07-18</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The mitotic inhibitor docetaxel (DTX) is often used to treat endocrine-refractory metastatic breast cancer, but initial responses are mitigated as patients eventually have disease progression. Using a cohort of ex vivo cultures of circulating tumor cells (CTCs) from patients with heavily pretreated breast cancer (n=18), we find two distinct patterns of DTX susceptibility, independent of clinical treatment history. In CTCs cultured from some patients, treatment with a single dose of DTX results in complete cell killing, associated with accumulation of non-viable polyploid (≥8N) cells arising from endomitosis. In others, a transient viable drug-tolerant persister (DTP) population emerges, ultimately enabling renewed proliferation of CTCs with preserved parental cell ploidy and DTX sensitivity. In these CTC cultures, efficient cell cycle exit generates a ≤4N drug-tolerant state dependent on CDKN1B (p27Kip1). Exposure to DTX triggers stabilization of CDKN1B through AKT-mediated phosphorylation at serine 10. Suppression of CDKN1B reduces the number of persister CTCs, increases ≥8N mitotic cells and abrogates regrowth after DTX exposure. Thus, CDKN1B-mediated suppression of endomitosis contributes to a reversible persister state following mitotic inhibitors in patient-derived treatment refractory breast cancer cells. Summary in bullets Transient DTX tolerant persister cells emerge in some patient-derived cultured CTCs. DTX-tolerant persisters restrict endoreduplication and polyploidy through CDKN1 (p27kip1). DTX exposure induces CDKN1B stabilization through AKT mediated phosphorylation at serine 10. Suppression of polyploidy underlies a drug tolerant persister state specific to mitotic inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c80ac0aa94dfd6f02a8336f8c8d7fde16ba668bb" target='_blank'>
              CDKN1B (p27/kip1) enhances drug-tolerant persister CTCs by restricting polyploidy following mitotic inhibitors
              </a>
            </td>
          <td>
            Elad Horwitz, Taronish D. Dubash, Annamaria Szabolcs, Ben S. Wittner, J. Kreuzer, Robert Morris, Aditya Bardia, Brian Chirn, Devon F. Wiley, Dante Che, Hunter C. Russel, Xcanda Ixchel Herrera Lopez, Douglas B. Fox, Ezgi Antmen, David T. Ting, Wilhelm Haas, Moshe Sade-Feldman, S. Maheswaran, Daniel A Haber
          </td>
          <td>2024-02-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Maintaining genomic stability is essential for reducing the risk of carcinogenesis. Homologous recombination (HR) is a high-fidelity DNA repair mechanism that addresses double-strand breaks and interstrand crosslinks. The present review examined two key components of HR: RAD51, the eukaryotic recombinase and PALB2, a scaffolding protein. Their structural and functional roles are explored in the context of breast and ovarian cancer. RAD51 facilitates homology search and strand invasion, while PALB2 links BRCA1 and BRCA2, stabilizing RAD51 filaments. Mutations in these genes compromise HR, increasing susceptibility to various cancers and impacting treatment efficacy by impairing DNA repair. The present review discussed the clinical implications of RAD51 and PALB2 mutations, focusing on risk stratification, PARP inhibitor efficacy and emerging therapies. Additionally, it highlighted the potential of RAD51 and PALB2 as biomarkers and therapeutic targets, contributing to advances in personalized cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cbc5968373fbc15750cedb90b63387fd5376bb" target='_blank'>
              RAD51 and PALB2 in precision oncology: Clinical implications for HRD associated breast and ovarian cancers (Review)
              </a>
            </td>
          <td>
            M. A. Kausar, K. Alshammari, Fahaad S Alenazi, Sadaf Anwar, Amany Mohammed Khalifa, Tarig A. N. Ginawi, Abdulaziz Asiri, M. Najm, S. Rabbani, Mohamed El-Tanani, Saumyatika Gantayat
          </td>
          <td>2025-07-03</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nuclear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syndrome in the family, the maternal grandfather had a similar tumor requiring amputation. Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-04-01</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
Long noncoding RNAs are essential regulators in numerous biological processes and have been linked to various diseases, including cancer. Despite their initial classification as transcriptional byproducts, lncRNAs have been shown to modulate chromatin structure, transcription, RNA processing, protein translation, and intranuclear transport. LINC-PINT, a lncRNA induced by P53, is particularly noteworthy for its role in tumor suppression across multiple cancers.


METHODS
By utilizing the PubMed database and applying inclusion criteria based on relevance, literature quality, and data availability, we conducted a comprehensive analysis of 128 studies to provide an overview of the functions of LINC-PINT and its mechanisms of action in cancers.


RESULTS
LINC-PINT was confirmed to function as a tumor suppressor factor in many cancers, such as triple-negative breast cancer, non-small cell lung cancer, gastric cancer, glioma, melanoma, osteosarcoma, laryngeal squamous cell carcinoma, esophageal cancer, colorectal cancer, nasopharyngeal carcinoma, retinoblastoma, ovarian cancer, thyroid cancer, hepatocellular carcinoma, and pancreatic cancer by promoting apoptosis and senescence, inhibiting proliferation, migration, invasion, drug resistance, cell stemness, EMT, radioresistance, and DNA damage repair.


DISCUSSION
LINC-PINT serves as a tumor suppressor with its ability to sponge miRNAs, regulate epigenetic modulation, DNA damage repair, etc. Despite the promising findings, the complex and tissue-specific functions of LINC-PINT, along with the need for further clinical validation, underscore the importance of continued research to fully understand its mechanisms and potential as a therapeutic target.


CONCLUSION
LINC-PINT is a potential target in cancer progression and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c614cdfeae321548caa4f0f7d2f981f35ae5a41" target='_blank'>
              LINC-PINT: A Distinctive Long Non-Coding RNA Functioning as a Potential Suppressor in Tumorigenesis.
              </a>
            </td>
          <td>
            Jiayi Li, Yining Pan, Songqiang Li, Cheng Chen, Chengfu Yuan
          </td>
          <td>2025-07-22</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) represent one of the most severe forms of genomic damage. Although substantial progress has been made in elucidating general patterns associated with DSBs, the influence of 3D chromatin structure on DSB formation remains underexplored, particularly concerning its spatial configuration. Here, the ChromInSight framework is introduced. Using standardized datasets,Hi-DSB is developed and deployed in ChromInSight, a genome-wide DSB prediction model based on graph contrastive learning (GCL), and applied advanced interpretability techniques to identify DSB-associated genomic patterns. The findings reveal that the spatial cluster-scene between hub nodes and DSB sites is predominantly shaped by the 3D conformation of chromatin, rather than by linear genomic distance. This phenomenon is validated at both the Loop and topologically associating domain (TAD) levels and proposed a "spatial isolation - damage containment" hypothesis, which illustrates the genome strategy for managing damage. These findings support the role of 3D genome architecture in genomic instability. Consequently, the framework provides a powerful tool for investigating the intricate relationship between chromatin structure and genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31e5b0b135f6e23dfe9751ad20057b548c477ad6" target='_blank'>
              ChromInSight: Revealing DNA Double-Strand Breaks Through Chromatin Structural Insights With an Interpretable Graph Neural Network Framework.
              </a>
            </td>
          <td>
            Kang Xu, Zongyuan Yu, Canzhuang Sun, Conglin Gou, Jiangyue Zhu, Jun Wang, Xiaochen Bo, Guoxian Yu, Hao Li, Hebing Chen
          </td>
          <td>2025-06-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="This review examines the increasingly prominent role of mechanics within cancer formation and progression. The extremely varied and contradictory genetic landscape of cancer is in stark contrast to the seemingly universal mechanical characteristics of cancer cells and their tumour microenvironment, and mechanics may be a principal unifying trait of this disease. The tight regulation of innate cell mechanical properties raises the possibility that destabilisation of the cell drives tumour formation in an attempt to restore cell mechanical homeostasis. With losses in cell stiffness more pronounced at the cell nucleus, we hypothesise that destabilisation occurs within the nucleus, likely within the nucleosome. Beyond the mechanical properties of the cell, this compromise to the chromatin structure holds significant repercussions for both genetic and epigenetic regulation, providing scope for significant genetic dysregulation and mutation. However, the nature of such genetic events will be dependent upon the region of mechanical destabilisation; thus, introducing greater variability and heterogeneity to genetic changes. We conclude with the hypothesis that cancer has a mechanical genesis, in which cell nuclear destabilisation functions as the enabling hallmark of cancer. It is theorised that both genetic and structural dysfunction stem from this nuclear destabilisation, driving disease pathology and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/440a5c35e068d9fe1560a5f26a4055f7a6f2a62d" target='_blank'>
              Nuclear destabilisation - a possible genesis of cancer?
              </a>
            </td>
          <td>
            Daniel D Scott, Francesco Bettariga, Marco Ventin, Chris Bishop, Britta Stordal
          </td>
          <td>2025-07-14</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f558af3eae687f94c53483af6c5bcb5fea880573" target='_blank'>
              Human fibroblasts from aged individuals exhibit chromosomal instability through replication stress caused by oxidative stress
              </a>
            </td>
          <td>
            Kailin Zhu, Guan Chen, Yueyi Ren, K. Iemura, Kozo Tanaka
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/813fcceb7d9e7dd4a1d1ee8db0535b3e857d7b20" target='_blank'>
              Unraveling the Potential of ctDNA in Precision Medicine for Breast Cancer.
              </a>
            </td>
          <td>
            Juscelino Carvalho de Azevedo Junior, Fernanda Jardim da Silva, Anna Carolina Lima Rodrigues, Stefanie Braga Maia de Sousa, Jéssica Manoelli Costa da Silva, Iago Barroso Ramos, Bárbara do Nascimento Borges, Vanessa Morais Freitas, D. Calcagno
          </td>
          <td>2025-08-07</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The XPO1 (exportin 1) gene encodes exportin 1 protein responsible for transporting proteins and RNA from the nucleus to the cytoplasm. It has been used as a biomarker for lymphoma detection. XPO1E571K mutation has been frequently observed and identified as a good prognostic indicator for lymphoma patients. The detection of a target molecule released by lymphoma cells into blood circulation (cell-free circulating tumor DNA, cfDNA) is a better method than tissue biopsy. However, cfDNA concentration in blood circulation is very low in cancer patients. Therefore, a precise and sensitive method is needed. In this study, cfDNA was extracted, and then the XPO1 gene was detected and amplified using conventional PCR. Sanger sequencing was employed to verify the DNA sequences. FAST-COLD-PCR was developed to detect XPO1E571K gene mutation using a CFX96 Touch Real-Time PCR System. The optimal critical temperature (Tc) was 73.3 °C, allowing selective amplification of XPO1E571K mutant DNA while wild-type XPO1 could not be amplified. XPO1E571K gene mutation can be detected by this method with high specificity and sensitivity in lymphoma patients. This approach facilitates rapid and straightforward detection in a timely manner after the diagnosis. Accordingly, the optimized FAST-COLD-PCR conditions can be used as a prototype for XPO1E571K mutant detection in lymphoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2337549ca06913503c667aac38d66d4d170ff79" target='_blank'>
              Detection of XPO1E571K Gene Mutation from Cell-Free DNA in Blood Circulation of Lymphoma Patients by FAST-COLD PCR
              </a>
            </td>
          <td>
            S. Duangmano, Natsima Viriyaadhammaa, Pinyaphat Khamphikham, Nutjeera Intasai, A. Tantiworawit, Teerada Daroontum, S. Chiampanichayakul, S. Anuchapreeda
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Targeting the DNA damage response (DDR) exhibits potent efficacy in inducing immune activation and enhancing patient prognosis. However, the benefits of DDR regulation are not universally observed across all patients, owing to the intricate compensatory mechanisms operative in certain cancers. There still exists a gap in the function of activated DDR protein in esophageal squamous cell carcinoma (ESCC). Here, we demonstrate that increased expression of DDR genes contributes to the progression of esophageal squamous cell carcinoma and suppresses the tumor immune microenvironment. Notably, the abundant presence of the DDR protein KIN in ESCC tissues facilitates efficient DNA damage clearance and promotes escape from apoptosis. Depletion of KIN significantly inhibited proliferation and induced DNA damage accumulation in ESCC cells. Mechanistically, KIN functions to support the recruitment of the R-loop regulator DHX9 to R-loop sites, thereby addressing DNA damage associated R-loops. Intriguingly, the depletion of KIN activates the STING pathway via NFκB signaling, which is induced by the accumulation of R-loops, ultimately initiating an innate immune response. Depletion of KIN improved the immune microenvironment and the effect of immune therapy in mouse model. Collectively, our findings identify KIN as a novel R-loop binding protein that facilitates the recruitment of the R-loop resolution complex and suppresses tumor-intrinsic innate immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e3c03a2576fc82b57363fed2fb18edfc9c71a5" target='_blank'>
              DNA/RNA-binding protein KIN17 supports esophageal cancer progression via resolving noncanonical STING activation induced by R-loop
              </a>
            </td>
          <td>
            Zichao Wei, N. Zhao, Lu Kuang, J. Cong, Sujuan Zheng, Yi Li, Zhihua Liu
          </td>
          <td>2025-08-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05361b43fcc5f75fe89674ef423d0f325e486737" target='_blank'>
              Characterizing neurofibromin 1-altered breast cancer through genomic, functional, and clinical analyses.
              </a>
            </td>
          <td>
            Xueyan Mao, Kang Ma, Yunjie Wang, Cheng Long, Xuejing Tan, Xi Chen, Bo Chen
          </td>
          <td>2025-08-14</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is the most prevalent type of leukemia in adults, characterized by a complex pathogenesis. The nuclear pore complex is pivotal in the development of various of malignant tumors, including hematological malignancies. While the NUP210 gene is upregulated in several solid tumors, its role in AML remains unreported. RNA-Sequence data and clinical profiles of AML patients were obtained from TCGA databases. Bone marrow samples were obtained from the First Affiliated Hospital of Ningbo University. GEPIA2, UALCAN and TCGA Portal databases were employed for survival prognosis analysis based on NUP210 expression. The GO/KEGG enrichment analysis of NUP210 co-expression and GSEA were performed using R software package. ENCORI and STRING databases were utilized to validate the association between NUP210 and MEIS1, PBX3 or HOXAs. CCK8, RT-qPCR, and Western blot assays were used to detect the viability and related gene expression in AML cell lines.Our findings revealed that NUP210 was significantly upregulated in AML patients compared to healthy controls. Multivariate analyses identified NUP210 as an independent poor prognostic factor in AML. Increased expression of NUP210 inhibited the viability of AML cells. The HOXA9, MEIS1, and PBX3 genes were notably upregulated in the NUP210 high-expression group. The RT-qPCR and Western blot assays demonstrated that NUP210 downregulation in leukemia cells leads to the inhibition of HOXA9 expression. Positive correlations were observed between NUP210 and MEIS1/PBX3, with MEIS1 and PBX3 proteins interacting with the HOXA9 protein. Our results demonstrated that NUP210 was remarkably upregulated in AML and may promote the malignant behavior of AML cells by upregulate HOXA9 gene expression, offering new insights for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60a036833093bfe76d9adec4575ddd5cae1b05c" target='_blank'>
              High NUP210 mRNA expression and its role in prognosis of acute myeloid leukemia
              </a>
            </td>
          <td>
            An Wu, Liyun Yang, Xiao Yan, Yanli Lai, Lieguang Chen, Shanhao Tang, Miao Zhou, Cong Shi, Ying Chen, Lixia Sheng, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-07-22</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Radiotherapy is a mainstay of treatment for head and neck squamous cell carcinoma (HNSCC), either definitive or adjuvant to surgery. Biological factors known to affect radiation response are hypoxia and DNA repair capacity, but several lines of evidence indicate that other genes and pathways in the tumor cells might be involved that have not been elucidated. Here, we report the results of a genome-wide CRISPR-Cas9 functional genomics screen in HNSCC cells to identify radiosensitizing genes. Remarkably, microtubule organizing genes were identified with CLASP1 as most unexpected radiosensitizing hit. Clonogenic assay confirmed the radiosensitizing effect of CLASP1 knockout. Functional analysis showed that CLASP1 knockout has major impact during S-phase, and resulted in mitotic cells with broken chromosomes and cell death. CLASP1 and possibly the microtubule machinery in broader sense seem involved in protection of HNSCC cells against radiation–induced DNA damage. This newly identified mechanism provides an outlook for novel treatment approaches in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284fd55d39146101abef95bbef149eda9c32d030" target='_blank'>
              An unexpected role of CLASP1 in radiation response and S-phase regulation of head and neck cancer cells
              </a>
            </td>
          <td>
            R. D. de Roest, Marijke Buijze, Myrthe Veth, Klaas de Lint, G. Pai, M. Rooimans, Rob M. F. Wolthuis, A. Brink, J. Poell, R. Brakenhoff
          </td>
          <td>2025-08-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Eukaryotic genomes are packaged into chromatin, a regulatory nucleoprotein assembly. Establishment, maintenance, and interconversion of chromatin states is required for correct patterns of gene expression, genome integrity, and organismal survival. Transcriptionally repressive heterochromatin minimizes mobilization of transposable elements and limits expansion of other repetitive DNA, but mechanisms for recognition of the latter sequences are not well established. We previously demonstrated in Drosophila melanogaster that transcripts derived from 1360 and Invader4 transposon insertions can trigger local conversion of transcriptionally permissive euchromatin to heterochromatin through the piRNA system, but only in a subset of genomic locations near existing heterochromatin masses. Here we show that a ~9 kb tandem array of the 36-nucleotide lac operator (lacO) sequence of Escherichia coli can form ectopic heterochromatin at a similar subset of sites, resulting in variegating gene expression of an adjacent reporter gene. HP1a and histone deacetylation are required for lacO repeat-induced silencing, but in contrast with previously described Position Effect Variegation (PEV) we do not observe increased histone H3 lysine 9 methylation. Silencing is suppressed at lower temperatures (another striking contrast to canonical PEV, which is enhanced at lower temperatures), implying the involvement of a temperature-sensitive component. Temperature switching experiments show that lacO repeat-induced heterochromatin formation is reversible throughout larval development following an HP1a-dependent licensing step in the early embryo. We conclude that the Drosophila nucleus can recognize a completely foreign tandem repeat as a target for heterochromatin formation, but find that the heterochromatin structure established is distinct from that of endogenous tandem arrays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46571b3d364267bc5025e336c0e4f5602612e789" target='_blank'>
              Heterochromatin-based silencing of a foreign tandem repeat in Drosophila melanogaster shows unusual biochemistry and temperature sensitivity
              </a>
            </td>
          <td>
            Tingting Gu, Elena Gracheva, Michael Lee, Wilson M Leung, Sophia Bieser, Alixandria Nielsen, Adam T. Smiley, Nhi N.T. Vuong, Matthias Walther, Gunter Reuter, Sarah C. R. Elgin, Andrew M Arsham
          </td>
          <td>2025-08-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3cf9b1f7da5af7fa436901f6e280557bb0b8010" target='_blank'>
              An M13 phagemid toolbox for engineering tuneable DNA communication in bacterial consortia
              </a>
            </td>
          <td>
            Abhinav Pujar, Anchita Sharma, Hadi Jbara, Manish Kushwaha
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b570c21348c29c543cb996a976f642797b0edb0" target='_blank'>
              A SETD2-CDK1-lamin axis maintains nuclear morphology and genome stability.
              </a>
            </td>
          <td>
            Abid Khan, Cheng Zhang, Phu G Nguyen, James M. Metts, Lucas C. Collins, Kanishk Jain, C. A. Mills, Logan Vlach, Kelin Li, Amanda L. Brademeyer, Brittany M. Bowman, M. Major, Jeffrey Aubé, Laura E. Herring, W. Rathmell, Frank M. Mason, Ian J. Davis, Qing Zhang, B. Strahl
          </td>
          <td>2025-08-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feceede34593c05cdca35b89dd9284a12fd1290a" target='_blank'>
              Exercise serum promotes DNA damage repair and upregulates DNA repair gene PNKP in colon cancer cells
              </a>
            </td>
          <td>
            Samuel T. Orange, Emily Dodd, Sharanya Nath, Hannah Bowden, Alastair Jordan, Hannah Smith, Ann Hedley, Ifeoma Chukwuma, Ian Hickson, Sweta Sharma Saha
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic cancer is a lethal malignancy prone to gemcitabine resistance. The single-strand selective monofunctional uracil DNA glycosylase (SMUG1), which is responsible for initiating base excision repair, has been reported to predict the outcomes of different cancer types. However, the function of SMUG1 in pancreatic cancer is still unclear.


METHODS
Gene and protein expression of SMUG1 as well as survival outcomes were assessed by bioinformatic analysis and verified in a cohort from Fudan University Shanghai Cancer Center (FUSCC). Subsequently, the effect of SMUG1 on proliferation, cell cycle, and migration abilities of SMUG1 cells were detected in vitro. DNA damage repair, apoptosis, and gemcitabine resistance were also tested. RNA sequencing was performed to determine the differentially expressed genes and signaling pathways, followed by quantitative real-time polymerase chain reaction and Western blotting verification. The cancer-promoting effect of forkhead box Q1 (FOXQ1) and SMUG1 on the ubiquitylation of c-Myc was also evaluated. Finally, a xenograft model was established to verify the results.


RESULTS
SMUG1 was highly expressed in pancreatic tumor tissues and cells, which also predicted a poor prognosis. Downregulation of SMUG1 inhibited the proliferation, G1 to S transition, migration, and DNA damage repair ability against gemcitabine in pancreatic cancer cells. SMUG1 exerted its function by binding with FOXQ1 to activate the Protein Kinase B (AKT)/p21 and p27 pathway. Moreover, SMUG1 also stabilized the c-Myc protein via AKT signaling in pancreatic cancer cells.


CONCLUSIONS
SMUG1 promotes proliferation, migration, gemcitabine resistance, and c-Myc protein stability in pancreatic cancer via AKT signaling through binding with FOXQ1. Furthermore, SMUG1 may be a new potential prognostic and gemcitabine resistance predictor in pancreatic ductal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2710beab663874401717dbc841996bd488ffd37a" target='_blank'>
              SMUG1 promotes the progression of pancreatic cancer via AKT signaling pathway through binding with FOXQ1.
              </a>
            </td>
          <td>
            Zijian Wu, Wei Wang, J. Hua, Jing-yao Zhang, Jiang Liu, S. Shi, Bo Zhang, Xiaohui Wang, Xianjun Yu, Jin Xu
          </td>
          <td>2025-08-11</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Repair of DNA damage is essential for genome integrity. DNA damage elicits a DNA damage response (DDR) that includes error-free and error-prone, i.e., mutagenic, repair. The SOS response is a widely conserved system in bacteria that regulates the DDR and depends on the recombinase RecA and the transcriptional repressor LexA. However, RecA/LexA-independent DDRs have been identified in several bacterial species. Here, using a whole-cell, label-free quantitative proteomics approach, we map the proteomic response in Myxococcus xanthus to mitomycin C treatment and the lack of LexA. In doing so, we demonstrate a LexA-independent proteomic DDR in M. xanthus. Using a candidate approach, we identify DNA damage-induced protein A (DdiA), a transcriptional regulator of the Xenobiotic Response Element (XRE) family, and demonstrate that it is involved in regulating the abundance of a subset of the LexA-independent DDR proteins. ddiA is expressed heterogeneously in a subpopulation of cells in the absence of exogenous genotoxic stress and reversibly induced population wide in response to such stress. DdiA, indirectly or directly, activates the expression of dnaE2, which encodes the DnaE2 error-prone DNA polymerase, and inhibits the expression of recX, which encodes RecX, a negative regulator of RecA. Accordingly, the ΔddiA mutant not only has a lower mutation frequency than the wild type but also a fitness defect, suggesting that DdiA mediates a trade-off between fitness and mutagenesis. We speculate that the DdiA-dependent response is tailored to counter replication stress, thereby preventing the induction of the complete RecA/LexA-dependent DDR in the absence of exogenous genotoxic stress. IMPORTANCE DNA damage repair is essential for genome integrity and depends on the DNA damage response (DDR). While the RecA/LexA-dependent SOS response is widely conserved in bacteria, there are also RecA/LexA-independent DDRs. Here, we identify the DNA damage-induced transcriptional regulator DdiA in Myxococcus xanthus and demonstrate that it regulates part of a LexA-independent DDR. DdiA activates the expression of dnaE2, which encodes the DnaE2 error-prone DNA polymerase, and inhibits the expression of recX, which encodes RecX, a negative regulator of RecA. Because the ΔddiA mutant has a lower mutation frequency than the wild type but also a fitness defect, we suggest that DdiA mediates a trade-off between fitness and mutagenesis, and the DdiA-dependent DDR is specifically tailored to counter replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cedd081d1cd78d3f392ec00cf935619d3da8229" target='_blank'>
              DdiA, an XRE family transcriptional regulator, is a co-regulator of the DNA damage response in Myxococcus xanthus
              </a>
            </td>
          <td>
            Jana Jung, Timo Glatter, Marco Herfurth, L. Søgaard‐Andersen
          </td>
          <td>2025-07-01</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Specialized DNA polymerases facilitate various cellular processes. Despite extensive research, the mutagenic effects of these error-prone enzymes on genomes are not fully understood. Here we show that Pol IV promotes genomic instability in Pseudomonas aeruginosa by misincorporating oxidized guanine nucleotides. This activity led to a distinctive mutational signature, characterized by A-to-C transversions occurring preferentially at AT sites flanked by a 5’G and/or 3’C. Furthermore, Pol IV preferentially targeted pathogenicity genes located at specific chromosomal locations near the replication termination region and rRNA-encoding operons. Half of the mutation events catalyzed by Pol IV impaired gene function. This can be attributed to the bias of Pol IV for mutating codons with its preferred sequence contexts, leading to substitutions to unreactive alanine and glycine residues. Remarkably, mutation signatures identified for Pol IV were found in clinical isolate genomes of P. aeruginosa, providing compelling evidence for its role in genetic diversification during pathogen adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f7c02761e12b8b73eebc31b5beda14fbe3ac0c" target='_blank'>
              The low-fidelity DNA Pol IV accelerates evolution of pathogenicity genes in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Sofía D. Castell, Consuelo M. Fernandez, Ignacio N. Tumas, L. M. Margara, María C. Miserendino, Danilo Ceschin, R. Pezza, Mariela R. Monti
          </td>
          <td>2025-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Bladder cancer pathogenesis is closely linked to epigenetic alterations, particularly DNA methylation and demethylation processes. Environmental carcinogens and persistent inflammatory stimuli—such as recurrent urinary tract infections—can induce aberrant DNA methylation, altering gene expression profiles and contributing to malignant transformation. This review synthesizes current evidence on the role of DNA methyltransferases (DNMT1, DNMT3a, DNMT3b) and the hypermethylation of key tumour suppressor genes, including A2BP1, NPTX2, SOX11, PENK, NKX6-2, DBC1, MYO3A, and CA10, in bladder cancer. It also evaluates the therapeutic application of DNA-demethylating agents such as 5-azacytidine and highlights the impact of chronic inflammation on epigenetic regulation. Promoter hypermethylation of tumour suppressor genes leads to transcriptional silencing and unchecked cell proliferation. Urine-based DNA methylation assays provide a sensitive and specific method for non-invasive early detection, with single-target approaches offering high diagnostic precision. Animal models are increasingly employed to validate these findings, allowing the study of methylation dynamics and gene–environment interactions in vivo. DNA methylation represents a key epigenetic mechanism in bladder cancer, with significant diagnostic, prognostic, and therapeutic implications. Integration of human and experimental data supports the use of methylation-based biomarkers for early detection and targeted treatment, paving the way for personalized approaches in bladder cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f706e4ed85cef4708cafccd425d482bfabf2666" target='_blank'>
              DNA Methylation in Bladder Cancer: Diagnostic and Therapeutic Perspectives—A Narrative Review
              </a>
            </td>
          <td>
            D. Puia, M. Ivănuță, C. Pricop
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mitotic bookmarking, the retention of regulatory proteins and lncRNAs on chromatin during mitosis, epigenetically sustains competency for phenotype-specific gene expression in progeny cells. Gene expression is predominantly suppressed during mitosis. Bookmarking provides the guidance for the resumption of gene expression in progeny cells that is obligatory for physiological control of lineage commitment, specialized cell structure and phenotypic function. While regulatory continuity is supported by the persistence of genome-associated regulatory complexes, altered bookmarking mediates plasticity for responsiveness to physiological cues. Bookmarking fidelity ensures genome integrity and controls expression of tumor suppressors and proto-oncogenes. Cancer-compromised aberrations in bookmarking results in transcriptional dysregulation and the initiation of tumor-associated processes. Keywords: Mitotic Bookmarking, Mitosis, Epigenetic Control, Gene Expression, Cell Division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d78052a96f4610f52d1801e1448869b1e34100" target='_blank'>
              Mitotic Bookmarking Provides Epigenetic Persistence or Plasticity for Biological Control and Cancer.
              </a>
            </td>
          <td>
            Andrew J. Fritz, Emory Pacht, R. Toor, Janine Warren, Haley W Greenyer, Jackson R Del Porto, Abigail G Person, Sadie J Korzec, Georgiy Zotkin, Jessica L Heath, Prachi N Ghule, J. Gordon, Andre J. van Wijnen, Seth E Frietze, Karen C. Glass, Jane Lian, Jane L. Stein, Gary S. Stein
          </td>
          <td>2025-07-16</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="m6A RNA methylation represents the most prevalent epitranscriptomic modification and modulates diverse dimensions of RNA metabolism. As a dynamic and reversible post-transcriptional mark, m6A is broadly distributed across multiple RNA types, including mRNA, ribosomal RNA (rRNA), microRNA (miRNA), long non-coding RNA (lncRNA), circular RNA (circRNA), and small nuclear RNA (snRNA). ALKB homolog 5 (ALKBH5), an m6A demethylase, has been extensively characterized for its involvement in tumorigenesis and progression through the removal of methylated modification from RNA, subsequently influencing RNA metabolism and gene expression regulation. A growing body of evidence indicates that ALKBH5 contributes to the therapeutic response in various cancers. Its aberrant expression is frequently observed across malignancies, where it modulates oncoprotein levels, promotes tumor initiation, and accelerates cancer cell proliferation, progression, and metastasis. Nevertheless, its implications for therapy responsiveness and the emergence of drug resistance remain poorly understood. This review centers on ALKBH5-mediated regulatory pathways and mechanisms in the contexts of chemotherapy, radiotherapy, and immunotherapy in malignancies, while also evaluating the therapeutic relevance of ALKBH5 as a molecular target and the potential of its pharmacological inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b5ca13f0779674eaa9f618a13016f82a968135" target='_blank'>
              Mechanistic insights into the role of RNA demethylase ALKBH5 in malignant tumor therapy
              </a>
            </td>
          <td>
            Ruijie An, Yingjie Shao, Wendong Gu
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1c9b213829c1acaa830ee9ae3ad3ef8e6169153" target='_blank'>
              MUC1-C auto-regulatory complex with EBNA1 is responsible for latent Epstein-Barr virus-associated gastric cancer progression.
              </a>
            </td>
          <td>
            Hiroki Ozawa, Yin Wang, Henry G. Withers, Naoki Haratake, Ayako Nakashoji, Atrayee Bhattacharya, A. Fushimi, Chie Kikutake, Kazuhiro Yamanoi, Shaowen White, Keyi Wang, Tatsuaki Daimon, Keisuke Shigeta, Kazumasa Fukuda, H. Kawakubo, Yuko Kitagawa, Mark D Long, B. Gewurz, Donald W Kufe
          </td>
          <td>2025-08-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent and deadly neoplasms globally; this fact puts emphasis on the need for accurate molecular biomarkers for early detection and accurate prognosis. Circular RNAs (circRNAs) have recently emerged as very promising cancer biomarkers. In this study, we thoroughly examined whether the expression levels of circular transcripts of the protein arginine methyltransferase 1 (PRMT1) gene can predict the prognosis of patients diagnosed with colorectal adenocarcinoma, the most frequent type of CRC. Hence, a highly sensitive quantitative PCR (qPCR) assay was developed and applied to quantify circ-PRMT1 expression in cDNAs from 210 primary colorectal adenocarcinoma tissue specimens and 86 paired normal colorectal mucosae. Extensive biostatistical analysis was then performed to assess the potential prognostic power of circ-PRMT1. Significant overexpression of this molecule was observed in colorectal adenocarcinoma tissue samples in contrast to their non-cancerous counterparts. Moreover, higher circ-PRMT1 expression was correlated with poorer disease-free survival (DFS) and worse overall survival (OS) in colorectal adenocarcinoma patients. Interestingly, multivariate Cox regression analysis revealed that the prognostic value of the expression of this circRNA does not depend on other established prognostic factors included in the prognostic model. Furthermore, the stratification of patients based on TNM staging revealed that higher circ-PRMT1 levels were significantly related to shorter DFS and OS intervals, particularly in patients with colorectal adenocarcinoma of TNM stage II or III. In summary, this original research study provides evidence that circ-PRMT1 overexpression represents a promising molecular biomarker of poor prognosis in colorectal adenocarcinoma, not depending on other established prognostic factors such as TNM staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b06ea5f530ca9a15a72069b8a7542753ca04c1f" target='_blank'>
              Overexpression of Circular PRMT1 Transcripts in Colorectal Adenocarcinoma Predicts Recurrence and Poor Overall Survival
              </a>
            </td>
          <td>
            P. Kokoropoulos, S. Christodoulou, P. Tsiakanikas, P. Vassiliu, C. Kontos, Nikolaos Arkadopoulos
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has been described in several eukaryotic species and has been shown to impact phenomena as diverse as cancer and herbicide tolerance. EccDNA is thought to arise mainly through transposable element (TE) mobilization. Because studies based on short-read sequencing cannot efficiently identify full-length eccDNA forms generated from TEs, we employed the CIDER-Seq pipeline based on long-read sequencing, to obtain full-length eccDNAs from Arabidopsis. The generated eccDNA datasets identified centromeric/pericentromeric regions as hotspots of eccDNAs with several eccDNA molecules originating from Helitron and LTR TEs. To investigate the role of epigenetic marks on TE-derived eccDNA biogenesis, we studied Arabidopsis methylation mutants dcl3, rdr6, ros1, and ddm1. Contrasting the TE-suppression previously reported in the hypermethylated ros1 mutants, we identified activation of TEs in ros1, specifically of LTR/Gypsy TEs. An enrichment of LTR/Copia elements was identified in actively dividing calli and the shoot apical meristem (SAM). Uncharacterized “variable TEs” with high eccDNA and expression were identified in the SAM, including ATCOPIA58. Together, our study reveals the genomic origins of eccDNAs and delineates the link between epigenetic regulation, transposon mobilization, and eccDNA biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8fe16883b4dfe42fe6b49221fc2e6a347cfdc2" target='_blank'>
              A comprehensive atlas of full-length Arabidopsis eccDNA populations identifies their genomic origins and epigenetic regulation
              </a>
            </td>
          <td>
            S. S. Zaidi, Sara Shakir, Hanne De Kort, Devang Mehta, V. Nguyen, Ruben Gutzat, Hervé Vanderschuren
          </td>
          <td>2025-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Microsatellite instability (MSI) is a key feature of cancers with defective DNA mismatch repair, including colorectal, gastric, endometrial, and ovarian cancers. Tumors with MSI depend on the Werner syndrome protein (WRN) for genomic stability, making WRN an attractive therapeutic target. WRN inhibitors exploit the concept of synthetic lethality, inducing selective DNA damage and cell death in MSI tumors while sparing microsatellite stability (MSS) tumor cells or normal cells. Preclinical studies have shown that the efficacy of WRN inhibitors is enhanced when combined with DNA damage response inhibitors or immunotherapy. This review delineates the molecular mechanisms underlying WRN dependency in MSI cancers and explores the therapeutic potential of WRN inhibition. WRN inhibitors represent a promising strategy in precision oncology, especially for MSI tumors, and have the potential to enhance patient outcomes, either as monotherapy or in combination with other treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a02ce022ffd342f3bce0913263e5473d488faa6" target='_blank'>
              Targeting the Werner syndrome protein in microsatellite instability cancers: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Shuling Chen, Zhiming Wang, Zhifei Cao, Mengmeng Xu, Yongsheng Zhang
          </td>
          <td>2025-08-06</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Survival strategy of bacteria is expanded by extrachromosomal elements (ECEs). However, their genetic diversity and functional roles for adaptability are largely unknown. Here, we discover a novel family of intracellular ECEs using 56 saliva samples by developing an efficient microbial DNA extraction method coupled with long-read metagenomics assembly. Even though this ECE family was not hitherto identified, our global prevalence analysis using 476 salivary metagenomic datasets elucidates that these ECEs reside in 74% of the population. These ECEs, which we named, “Inocles”, are giant plasmid-like circular genomic elements of 395 kb in length, including Streptococcus as a host bacterium. Inocles encode a series of genes that contribute to intracellular stress tolerance, such as oxidative stress and DNA damage, and cell wall biosynthesis and modification involved in the interactions with oral epithelial cells. Moreover, Inocles exhibit significant positive correlations with immune cells and proteins responding to microbial infection in peripheral blood. Intriguingly, we examine and find their marked reductions among 68 patients of head and neck cancers and colorectal cancers, suggesting its potential usage for a novel biomarker of gastrointestinal cancers. Our results suggest that Inocles potentially boost the adaptive capacity of host bacteria against various stressors in the oral environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ef6b2491714273e3a5acd966b2b147c6dd3e3d" target='_blank'>
              Giant extrachromosomal element “Inocle” potentially expands the adaptive capacity of the human oral microbiome
              </a>
            </td>
          <td>
            Yuya Kiguchi, Nagisa Hamamoto, Y. Kashima, L. Runtuwene, A. Ishizaka, Yuta Kuze, T. Enokida, N. Tanaka, Makato Tahara, S. Kageyama, T. Fujisawa, Riu Yamashita, A. Kanai, Josef S. B. Tuda, Taketoshi Mizutani, Yutaka Suzuki
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract The bacterial cytoplasm is characterized by a distinctive membrane-less organelle, the nucleoid, which harbors chromosomal DNA. We investigate the effects of dynamic processes associated with transcription and translation on the structure of this organelle, using coarsegrained molecular dynamics (MD) simulations implemented with out-of-equilibrium reactions. Our model captures the scale of the entire cell and incorporates a reaction-diffusion system for ribosomes and polyribosomes, combining their out-of-equilibrium dynamics with excluded volume interactions with DNA. Our findings demonstrate that out-of-equilibrium reactions increase the size of the nucleoid. In addition, we show that the nucleoid size increase is proportional to transcriptional activity. Our model reproduces the time-dependent change in nucleoid size observed in rifampicin treatment experiments, where the pool of polyribosomes is depleted. Furthermore, we find these active processes are essential for complete sister chromosome separation and correct nucleoid positioning within the cell. Overall, our study reveals the effects of the central dogma processes on the internal organization and localization of bacterial nucleoids. Significance Understanding how bacteria organize their chromosomes is fundamental to cell biology. Through our coarse-grained molecular dynamics simulations incorporating out-of-equilibrium processes of transcription and translation, we are able to capture the effects of these central dogma processes on DNA organization and demonstrate that these active biological processes expand the nucleoid and facilitate the separation of daughter chromosomes. Our simulations are compared to experimental measurements and highlight the impact of the out-of-equilibrium conditions of the living cell. These findings point out the crucial interplay between physical forces and biological activity in cellular organization, suggesting that cellular structure depends on non-equilibrium processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5869c17c0b2f03dcafbf482f5c2ecb59859979e3" target='_blank'>
              Effects of central dogma processes on the compaction and segregation of bacterial nucleoids
              </a>
            </td>
          <td>
            Mu-Hung Chang, Maxim O. Lavrentovich, Jan-Michael Y. Carrillo, Jaan Männik
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) originates from neuroepithelial tissue and is one of the most common intracranial malignant tumors in adults, with high recurrence rate and poor prognosis. In recent years, SOX9 has been reported to play an important role in many diseases and cancers, and is a promising target, but it has been rarely reported in GBM. Methods RNA sequencing data of GBM were obtained from the Cancer Genome Atlas (TCGA) database and the Genotype-Tissue Expression (GTEx) database for analysis of SOX9 expression and differentially expressed genes (DEGs). Moreover, functional enrichment analysis of GBM-related DEGs was performed by GO/KEGG, GSEA, and protein-protein interaction (PPI) network. Additionally, the clinical significance of SOX9 in GBM was assessed by Kaplan-Meier Cox regression and prognostic model. What’s more, we analyzed SOX9-related immune cell infiltration and expression of immune checkpoints in GBM. The incorporated studies were analyzed using the R package. Results SOX9 was highly expressed in a range of malignant tumor tissues, including GBM. Surprisingly, high SOX9 expression was remarkably associated with better prognosis in the lymphoid invasion subgroups in a sample of 478 cases (P < 0.05). Totally, 126 differentially significant genes (DSGs) were identified between high- and low- expression group, of which 29 genes were upregulated and 97 genes were downregulated. Furthermore, high expression of SOX9 was an independent prognostic factor for IDH (isocitrate dehydrogenase)-mutant in Cox regression analysis. Screening was performed by LASSO coefficients to select non-zero variables that satisfied the coefficients of lambda. min, and four genes were screened out. OR4K2 and IDH status were prognostic factors associated with THCA in multifactorial COX regression analysis. SOX9, OR4K2 and IDH status were included in the nomogram prognostic model. Correlation analysis indicated SOX9 expression was correlated with immune cell infiltration and expression of immune checkpoints in GBM. Conclusion SOX9 was identified as a diagnostic and prognostic biomarker in glioblastoma, particularly in IDH-mutant cases. Its expression was closely correlated with immune infiltration and checkpoint expression, indicating its involvement in the immunosuppressive tumor microenvironment. SOX9-based gene signatures further supported a robust nomogram model, underscoring its potential as a therapeutic and prognostic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2a8c97c4df9bf9651c0f0af4a4efc671535b22" target='_blank'>
              High expression of SOX9 is a diagnostic and prognostic indicator of glioma
              </a>
            </td>
          <td>
            Libo Xu, Zhenhao Wang, Mao Li, Qingsong Li
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT
Transposable elements (TEs), once considered genomic "junk", are now recognized as critical regulators of genome function and human disease. These mobile genetic elements-including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons-are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages-such as zygotic genome activation and cell fate determination-where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs-and balancing their contributions to normal development and disease-is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background DNA changes induced by stress may be stable through cell division and passed to subsequent generations. Plant improvement programs require that individuals develop a new heritable trait. The action of mutagens applied in classical mutagenesis is known from their transgenerational inheritance, observed as mutations. DNA methylation is an epigenetic mechanism that generates heritable phenotypic variation by influencing gene expression and modifying DNA accessibility to mutagens. Fagopyrum esculentum (common buckwheat) and F. tataricum (Tartary buckwheat) are essential for food production and valuable for medical purposes. Results In this study, we aimed to investigate the transgenerational changes in the genome and epigenome of two Fagopyrum species, by analysing 1st and 3rd generations, followed by treatment with gamma rays. The genomic instability, observed as DNA fragmentation, micronuclei, and changes in cell cycle profile, was observed in 1st generation, whereas 3rd generation shows recovery in DNA stability. Gamma-ray stress caused alterations in the pattern and level of DNA methylation in the roots of both analysed generations. The response to gamma rays depends on the species – in 1st generation, a higher increase in DNA methylation level in F. esculentum and only slight changes in F. tataricum were observed. The DNA methylation level in the 3rd generation changed only in F. esculentum. Conclusions Gamma ray is a stress factor that affects the pattern and level of DNA methylation in F. esculentum and F. tataricum. Significant differences were observed between control and 3rd generation only in F. esculentum. Differences in DNA methylation following gamma irradiation between the analysed Fagopyrum species may result from various forms of pollination in the analysed species. By using gamma ray-induced mutation, we got a highly stable mutant of F. tataricum, which may result in a self-pollinating trait. Understanding the changes in DNA methylation following mutagenic treatment can be essential in a breeding programme. Supplementary Information The online version contains supplementary material available at 10.1186/s12870-025-07033-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c669739a40e63380b2299a72c170061851584f" target='_blank'>
              Transgenerational genetic and epigenetic changes induced by gamma-ray in Fagopyrum species
              </a>
            </td>
          <td>
            Katarzyna Sala-Cholewa, Magdalena Rojek-Jelonek, Jolanta Kwasniewska
          </td>
          <td>2025-07-31</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7803de2661e62cd9d27e86eb2d0ae2e148e6e4" target='_blank'>
              Chromatin Control of EBV Infection and Latency.
              </a>
            </td>
          <td>
            P. Lieberman, I. Tempera
          </td>
          <td>2025-07-13</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Metabolic alterations and genomic instability are the hallmark features of many cancers. However, the precise mechanisms underlying the intricate links among these processes remain largely unknown. Here, a molecular mechanism is presented that regulates the interplay between cellular energy metabolism and DNA repair. These findings demonstrate that during energy deficiency, ATG4B translocates from the cytoplasm to the nucleus and disrupts DNA repair by directly interacting with PRMT1. This interaction inhibits the PRMT1-dependent methylation of MRE11, a key regulator of DNA repair, leading to genomic instability. Importantly, it is shown that ATG4B-mediated DNA repair defects are significantly enhanced in patient-derived acute myeloid leukemia (AML) cells and in mouse AML cells induced by MLLT3-KMT2A overexpression. Inhibition of ATG4B enhanced PRMT1-mediated DNA damage responses, suppressed cell proliferation, reduced the mutation burden, and prolonged survival in mice with MLLT3-KMT2A-induced AML and in those bearing AML patient-derived xenografts. These findings revealed that energy deficiency compromises DNA repair through ATG4B nuclear translocation, and ATG4B inhibition enhances DNA repair in AML cells, alleviating the malignant evolution of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fb40ddbc348d6bdf921a2ddd2696429a42111c" target='_blank'>
              Energy Deficiency-Induced ATG4B Nuclear Translocation Inhibits PRMT1-Mediated DNA Repair and Promotes Leukemia Progression.
              </a>
            </td>
          <td>
            Zhenkun Wang, Xianli Zhang, Yuanyuan Zhou, Shuyi Lin, Yuting Fu, Haiping Liu, Qiongdan Gao, YanHong Xiao, Zhao Yin, Shuang Liu, Kexiu Huang, Suqi Deng, Juan Du, Hui Zeng, Jia Wang, Jinping Zheng, Qinghua Zhou, Tianfeng Chen, Xiaoling Gao, Zhenyu Ju, Bo Liu
          </td>
          <td>2025-08-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The induced “recombinogenic situation” in hematopoietic stem cells and the activation of the cell’s reparative systems create the basis for recombination events between fragments of extracellular double-stranded DNA delivered into the cell and chromosomal DNA or other forms of the reparative-recombination process. In mouse and rat model organisms as well as in human bone marrow cells, changes in the amount of telomeric DNA in hematopoietic stem cells were assessed as an indicator of repair and recombination events that have occurred. In all experiments performed, recombinant human angiogenin was used as a comparison factor. Dot blot hybridization showed that in the colony cells obtained from the bone marrow cells of the model organisms as well as from human bone marrow cells treated with a double-stranded DNA preparation, there was a significant increase in the amount of telomeric DNA. Amplification of telomeric DNA in colony cells is not associated with contamination of the original DNA preparation with which the bone marrow cells were treated. Treatment of bone marrow cells with DNA that does not carry telomeric sequences (AluI PCR fragment) does not lead to an increase in the amount of telomeric DNA in the cells of grown colonies. This suggests the participation in the amplification of telomeric DNA of an extrachromosomal DNA template carrying telomeric DNA. It has been established that treatment of bone marrow cells with angiogenin also leads to an increase in telomeric DNA in colony cells. A comparison of the type of colonies with the intensity of hybridization (i. e. the amount of telomeric DNA in the sample) suggests that the increase in the amount of detectable telomeric DNA following treatment with angiogenin and hDNAgr has a fundamentally different origin. Western blot analysis and real-time PCR revealed that the increase in the amount of telomeric DNA following treatment of bone marrow cells with a double-stranded DNA preparation does not correlate with the activity of endogenous/exogenous telomerase. For angiogenin, it has been shown that an increase in the amount of telomeric DNA may be the result of activation of endogenous telomerase activity. A principle has been developed for the amplification of a new genetic trait that came into hematopoietic stem cells with extracellular double-stranded DNA material and was fixed in the recipient genome or was transitively present in the cell as new genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f0c77ec133a09bb58f64518224081f769056e8" target='_blank'>
              Concept of natural genome reconstruction. Part 3. Analysis of changes in the amount of telomeric DNA in colony cells as a new amplified feature that arose during the processing of hematopoietic bone marrow stem cells
              </a>
            </td>
          <td>
            V. Ruzanova, S. Oshikhmina, G. S. Ritter, E. Dolgova, S. Kirikovich, E. Levites, Y. R. Efremov, T. V. Karamysheva, A. Bogomolov, M.I. Meschaninova, A. L. Mamaev, O. Taranov, S. V. Sidorov, S. D. Nikonov, O. Leplina, A. Ostanin, E. Chernykh, N. A. Kolchanov, A. Proskurina, S. S. Bogachev
          </td>
          <td>2025-07-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Mobile genetic elements (e.g. transposons and introns) are genomic parasites, DNA sequences that can proliferate within host genomes. In prokaryotes, these elements are minimized by purifying selection and host-cell defenses. In eukaryotes, however, these parasitic sequences may comprise most of the genome. This extreme imbalance in eukaryotes is maintained by a metabolically quiescent germline through which the genomic parasites are transmitted to the next generation via gametes and spread throughout the population via sexual reproduction. Some eukaryotes avoid the threat posed by genomic parasites by eliminating them from somatic cells, whereas others retain the parasites in somatic cells and attempt to suppress them by epigenetic silencing mechanisms. Here, we review the evidence for the ongoing competition between host and parasite for genome occupancy. We conclude that defenses against mobile genetic elements vary greatly among organisms, and this variation accounts for the enormous range in genome size among organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5828d52742180cad7b0c29c98a298b55b8bda" target='_blank'>
              The biological and evolutionary consequences of competition between DNA sequences that benefit the cell and DNA sequences that benefit themselves
              </a>
            </td>
          <td>
            A. J. Bendich, Scott O. Rogers
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Chromatin relaxation is a permissiven progress for DNA repair through enabling repair factors to access the damaged DNA. Linker histone H1 is important in maintaining chromatin compaction under physiological state. The recent evidence highlights the importance of H1 modifications in response to cellular stress. Following DNA double‐strand breaks, the metabolic enzyme phosphorylated CTP synthase 1 (CTPS1) functions as a deamidase, catalyzing the rapid conversion of H1 residues Asn76 and Asn77 into aspartate. This modification enables subsequent acetylation at Lys75 by the histone acetyltransferase p300, thereby reducing H1‐DNA affinity and promoting chromatin decompaction. This sequential modification‐H1 deamidation followed by acetylation‐facilitates the recruitment of repair factors involving both homologous recombination and non‐homologous end joining repair pathways, and consequently promoting DNA repair. Importantly, high expression of CTPS1 is associated with resistance to radiotherapy in mouse models and clinical cancer patients, suggesting that the CTPS1 may serve as a potential therapeutic target. While targeting CTPS1 may offer opportunities to enhance radiosensitivity of cancer patients, challenges related to specificity and off‐target effects require further studies. This article highlights an emerging role of H1 modification in the DNA damage repair and discusses the therapeutic potential of manipulating H1 deamidation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ef2febfbdf0354039134d0a6f7b0aac3792561" target='_blank'>
              H1N76/77 deamidation facilitates chromatin remodelling and genome stability during DNA damage repair
              </a>
            </td>
          <td>
            Tingting Feng, Mengyan Li, Chenmiao Hu, Yuan Tian, 
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41c845deccf83a402934c97c50b9796b93645c5b" target='_blank'>
              Establishment of chromatin architecture interplays with embryo hypertranscription.
              </a>
            </td>
          <td>
            Guang Yu, Kai Xu, Weikun Xia, Ke Zhang, Qianhua Xu, Lijia Li, Zili Lin, Ling Liu, Bofeng Liu, Zhenhai Du, Xia Chen, Qiang Fan, Fangnong Lai, Wenying Wang, Lijuan Wang, Feng Kong, Chao Wang, Haiqiang Dai, Huili Wang, Wei Xie
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Lynch syndrome (LS) and familial adenomatous polyposis (FAP) are hereditary cancer predisposing syndromes characterized by increased risk of especially early-onset colorectal cancer. Predisposition to LS is caused by germline mutations in DNA mismatch repair genes leading to elevated cancer progression and microsatellite instability. FAP is associated with germline mutations in APC promoting cancer initiation and chromosomal instability. DNA methylation is an important epigenetic mechanism in early tumorigenesis via, e.g., field defects in non-neoplastic colon. Our aim was to study genome-wide methylation changes in colorectal specimens (adenomas and carcinomas supplemented with paired normal colon) obtained during colonoscopy surveillance, and explore the role of such alterations in tumorigenesis, with a special focus on early changes. To our best knowledge, this study is the first one to compare altered DNA methylation genome-wide in LS and FAP-associated colorectal neoplasia. Results DNA methylation alterations were subtle in FAP adenomas, whereas in LS adenomas, changes were abundant when compared to their normal counterparts. When FAP normal and LS normal colon were compared, DNA methylation changes of FAP normal colon mirrored those occurring in LS tumors, suggesting that colorectal tumorigenesis-associated DNA methylation alterations take place already in FAP normal colon mucosa. DNA methylation age was more variable in LS than FAP normal colon, and in proximal than distal colon, when compared to individuals’ age at the time of sampling. In LS tumors, DNA methylation changes (hyper- and hypomethylation) were abundant even in adenomas with low-grade dysplasia and stable microsatellites and peaked in adenomas with high-grade dysplasia. LINE-1 hypomethylation was more prominent in LS adenomas than FAP adenomas, but normal colon of LS and FAP displayed similar levels of LINE-1 methylation. Conclusions Genome-wide DNA methylation changes are an integral part of FAP and LS-associated colorectal tumorigenesis. Occurrence at early stages, even in non-neoplastic colonic mucosa, and increased prevalence with progressive dysplasia suggest a role in tumor development. Overlap of many of the topmost DNA methylation alterations between LS and FAP, and previous reports of their occurrence in sporadic colorectal and other tumors as well, imply their broad biological relevance and possible biomarker potential for clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01940-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f732072d5015ea229362c0044955d5acfa518d6c" target='_blank'>
              Genome-wide DNA methylation profiles of colorectal tumors in Lynch syndrome and familial adenomatous polyposis
              </a>
            </td>
          <td>
            Satu Mäki-Nevala, Anni K Kauppinen, A. Olkinuora, Aleksi Laiho, Petri Törönen, L. Renkonen-Sinisalo, A. Lepistö, T. Seppälä, J. Mecklin, P. Peltomäki
          </td>
          <td>2025-08-02</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Somatic mutations arise in normal tissues and precursor lesions, often targeting cancer-driver genes involved in cell cycle regulation. Most checkpoint-mutant clones, however, remain dormant throughout an individual's lifetime and seldom progress to malignancy, implying the presence of protective mechanisms that limit their expansion and malignant transformation. One such safeguard is telomere crisis-a potent tumor-suppressive barrier that eliminates cells lacking functional checkpoints and evading p53- and pRb-mediated surveillance. While the genomic instability unleashed during telomere crisis can drive clonal evolution, cell death is typically the dominant outcome, with only a rare subset of cells escaping elimination to initiate malignancy. Recognizing the dual role of telomere crisis-suppressing tumor initiation while enabling clonal evolution-is essential for understanding early cancer development and designing strategies to eliminate tumor-initiating cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The AP-1 transcription factor FOSL1, also known as Fra-1, is a crucial oncoprotein that plays an important role in human tumor progression and metastasis and has thus emerged as a promising therapeutic target. FOSL1 regulates the expression of a large protein-coding gene network, and this molecular mechanism can promote the progression of tumors. Interestingly, recent studies have shown that FOSL1 can also achieve the same protumor effect by regulating certain noncoding RNAs (ncRNAs). However, more studies have shown that ncRNAs can regulate the expression and activity of FOSL1, thereby affecting the occurrence and development of tumors, which indicates that ncRNAs can be regulators of FOSL1 in cancer. In this review, we first provide a comprehensive overview of the expression and function of FOSL1 and ncRNAs in tumors and then focus on the mutual regulatory relationship between ncRNAs and FOSL1, as well as their regulatory effects on and mechanisms of tumor progression. In addition, we further explored the potential clinical applications of the FOSL1-ncRNA system in cancer treatment, providing a theoretical basis for the study of FOSL1 and/or ncRNA-related molecular markers or targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f413c89530da64009a6941f40cf33c862cd2843" target='_blank'>
              Noncoding RNAs as regulators of FOSL1 in cancer
              </a>
            </td>
          <td>
            Xiaochang Wang, Li Wang, Shoushi Wang, Jinjie Zhang, Xueyang Wang, Ting Zhang, Linwei Li, Jin Wei, Yi Zhao, Zhixia Zhou
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-02</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Krüppel-like factor 4 (KLF4) emerged as an epigenetically regulated gene in a variety of settings, including cell reprogramming and malignant cell proliferation. The aim of the present manuscript is to explore the relationship described in recent years between circular RNAs, miRNAs, and KLF4. These have been shown to be involved in cancers having diverse histological origins, including some of the most prevalent and deadly tumors for the human population. Expression and protein levels of this transcription factor correlate with invasiveness and prognosis in a context- and tissue-specific fashion. Methods: The literature was obtained through two main PubMed queries. The first is “miRNA and KLF4 and cancer” and is limited to the last 5 years. The second is “circRNA and KLF4”, which yielded publications between 2013 and 2024. The oncological publications were selected. Results: A number of circRNA/miRNA axes that regulate the downstream transcription factor KLF4 emerged in the last few years. circRNAs act as sponges for miRNAs and synergize with KLF4, which can function as either a tumor promoter or suppressor in different tumors. Conclusions: The axes represented by circRNA/miRNA/KLF4 emerged as a new layer of epigenetic regulation. These RNA-based modulators explain the complex regulation of this transcription factor and open the way to new therapeutic targeting possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a3e7b131d3c048a24a47a22d4b39bf8b794eef" target='_blank'>
              circRNA/miRNA Networks Regulate KLF4 in Tumor Development
              </a>
            </td>
          <td>
            R. Frazzi, E. Farnetti, D. Nicoli
          </td>
          <td>2025-07-29</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="While the majority of viruses falls within a handful of well-defined realms, many novel viruses and virus-like genetic elements remain to be discovered. Given the abundance and diversity of small, circular DNA genomes in animal gut viromes, we hypothesized that viruses of similar size might have been overlooked in other environments. Re-analyzing existing ocean virome datasets for small viruses, we discover a novel group of circular mobile genetic elements of ~4.6kb in size, likely with double-stranded DNA, which we term Charybdis elements. Charybdis elements form a singular group based on cohesive gene content, encode around 10 hypothetical genes but apparently contain no virus, transposon or plasmid associated hallmark proteins. They are detected exclusively in marine datasets ranging from surface water to hydrothermal vents and sediments, from the Arctic to the Antarctic. Virome datasets separating virus-like particles by density gradients show that Charybdis elements accumulate in the fraction representing extracellular vesicles and small, tail-less viruses that are often associated with lipid membranes. One specific clade of Charybdis elements is shown to integrate into the genomes of the common marine group II archaeon candidate order Poseidoniales, the putative host of these elements. Analysis of thousands genomes shows a common genome architecture consisting of two modules: An accessory module containing a diverse range methyltransferases and other genes likely involved in host interactions, and a more conserved (putative) structural module. The structural module encodes proteins that are confidently predicted to form striking tri- or pentameric structures. These structures show stark resemblance and in some cases homology to protrusion or tail-elements on the capsids of viruses from the realms Varidnaviria and Duplodnaviria. Given these traits, Charybdis elements likely represent a hitherto undiscovered group of virus or virus-satellites, with at least one protein of a novel double-Greek-key fold serving as a potential capsid element. However, since their hosts remain unculturable, the true nature of these ubiquitous marine elements remains as of yet unresolvable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d6ae6d60b7d2c07e786f432ef49127c628ff31" target='_blank'>
              A novel mobile genetic element with virus-like characteristics is widespread in the world's oceans
              </a>
            </td>
          <td>
            P. Kirchberger, Antoni Luque
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="With the advancement of assisted reproductive technology (ART), sperm cryopreservation has become an essential technique in both clinical and research applications. However, maintaining DNA integrity during the freezing and thawing process remains a significant challenge. This study introduces an innovative TdT/Cas12a-based biosensor to enable precise detection and quantification of sperm DNA breakages. Compared to traditional methods such as the sperm chromatin structure assay (SCSA), this biosensor exhibits exceptional sensitivity (capable of detecting DNA breakages at as low as 0.001 nM) and molecular-level resolution, allowing for the precise localization and quantification of DNA breakpoints. By integrating TdT for nucleotide labeling and Cas12a for signal amplification, the system achieves high specificity and reliability. Additionally, the introduction of a standard strand as a quality control significantly enhances reproducibility. Furthermore, the study explores the protective effects of natural antioxidants on sperm DNA integrity during the freeze–thaw process. The results reveal a marked increase in DNA breakpoints caused by cryopreservation, with the TdT/Cas12a system outperforming traditional methods in providing detailed and accurate assessments. Notably, the addition of antioxidants effectively mitigates DNA damage during the freeze–thaw process, safeguarding sperm DNA integrity. This platform not only represents a significant advancement in DNA damage detection and monitoring but also supports the optimization of cryopreservation protocols and contributes to improving the success and safety of ART. It offers valuable insights and innovative tools for the field of reproductive medicine. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09768-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9586c02107a796889cf2ea0a3decb4fa6079e18f" target='_blank'>
              TdT/Cas12a-based biosensor for sensitive detection of DNA breakpoints in sperm cryopreservation
              </a>
            </td>
          <td>
            Bianbian Gao, Ziyang Liu, Bei Yan, Kunhao Du, Liguo Pei, Juan Wang
          </td>
          <td>2025-07-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Mutational landscape is prognostic in colorectal cancer (CRC). Rat sarcoma (RAS) oncogenes, such as KRAS and NRAS, with driver mutations, portend poor survival outcomes, whereas pathologic mutations in HRAS are extremely rare, and their prognostic value remains uncertain. Methods This retrospective study analyzed the Oncology Research Information Exchange Network (ORIEN) alliance tumor RNA‐Seq data in Stages II and III CRC to investigate the association between RAS gene expression and survival outcomes. Results High transcript levels of HRAS were associated with superior overall survival (OS). The high HRAS‐associated OS benefit was most pronounced in patients with right‐sided primary expressing low KRAS transcript levels in the absence of pathologic KRAS mutations. Conclusions Contrary to the notion that RAS family genes are proto‐oncogenic, this study demonstrates that high HRAS transcript levels are associated with superior OS in Stages II and III CRC. The potential of HRAS as a prognostic biomarker should be explored further.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6201baf2d0af4b663c5c54e1af500df1e4f27566" target='_blank'>
              Proto‐Oncogene HRAS Transcript Level and Overall Survival in Stages II and III Colorectal Cancer
              </a>
            </td>
          <td>
            Donghyun Kim, Saima Sharif, Juan Antonio Raygoza Garay, Avanish S. Bhakta, P. Boland, M. Cavnar, M. Churchman, H. Hatoum, Lyen C. Huang, Joseph Kim, Richard Kim, Robert W Lentz, Sarbajit Mukherjee, Mary T. O'Donnell, Benjamin Quartey, M. Reilley, Robert J. Rounbehler, B. Salhia, Bryan P. Schneider, C. Chan
          </td>
          <td>2025-07-31</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs), defined as noncoding transcripts exceeding 200 nucleotides without protein-coding potential, have been increasingly recognized for encoding polipeptides or proteinsimplicated in the regulation of malignant tumor initiation, progression, and prognosis. Among them, small nucleolar RNA host gene 29 (SNHG29) has attracted increasing attention because of its aberrant expression across diverse malignancies. This report reviews the latest research regarding its expression patterns, regulatory functions, and involved signaling pathways. This review synthesizes current insights into the contributions of SNHG29 to oncogenic processes, including cell proliferation, migration, epithelial‒mesenchymal transition, chemoresistance, and immunosuppression. Furthermore, evidence indicates that SNHG29 functions as a competing endogenous RNA (ceRNA), modulating tumor progression via the miRNA‒mRNA axis and signaling pathways such as the ATR‒Chk1, PI3K‒AKT, and Wnt‒β-catenin pathways. Cumulative data position SNHG29 as a promising diagnostic and prognostic biomarker, as well as a potential molecular target for therapeutic intervention in various malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef7951689b147bbb46276371501a41cc5d89fba" target='_blank'>
              The role of long noncoding RNA SNHG29 in malignant tumors
              </a>
            </td>
          <td>
            Tingcheng Bu, Wendong Gu, Yingjie Shao
          </td>
          <td>2025-07-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, S. Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Nuclear architecture and chromosome folding are often speculated to influence genome replication. In yeasts, centromeres cluster close to the spindle pole body and telomeres position at the nuclear periphery. This ‘Rabl configuration’ spatially segregates the most early and late replicating parts of the genome, centromeres and telomeres, respectively, suggesting that origin position along the centromere–telomere axis may influence origin activity. Here, we investigated DNA replication in a wild-type, 16-chromosome Saccharomyces cerevisiae strain and in its single-chromosome counterpart engineered by chromosome fusion and elimination of all but two telomeres and one centromere, which strongly affects genome folding and abrogates the Rabl conformation. Using nanopore sequencing-based methods, we found that the DNA replication program of both strains was virtually indistinguishable, with the exception of origin inactivation next to deleted centromeres and changes in origin efficiency and fork direction at chromosome fusions, as anticipated from the known origin-regulation properties of centromeres and telomeres. Only a handful of replication changes, mostly due to local origin repression, were observed elsewhere. Fork speed was also unaffected except at deleted centromeres. In conclusion, the DNA replication program of budding yeast is remarkably resilient to perturbations of chromosome folding and loss of the Rabl conformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e28f5bc4a9bd5e110c57ffd165964c5f1a57708" target='_blank'>
              Replication program of a single-chromosome budding yeast strain
              </a>
            </td>
          <td>
            Jade Pellet, Laurent Lacroix, Bertrand Theulot, Emma Simonin Chavignier, Alan Tourancheau, F. Proux, Sylvie Hermann-Le Denmat, G. Millot, Benoît Le Tallec, O. Hyrien
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Bladder cancer has been considered as one of the most common urinary malignancies. Growing evidence has indicated that Centromere Protein F (CENPF) is a promising molecular biomarker for many human malignant tumors. However, the role of CENPF in bladder cancer (BC) proliferation, migration, invasion, cell cycle and apoptosis thereof remain unclear. In the present study, high-throughput sequencing technology and bioinformatics analysis were conducted to identify mRNAs profiles in 10 pairs of bladder cancer tissues and adjacent noncancerous tissues. CENPF was overexpression in bladder cancer tissues, and higher in MIBC than NMIBC. We investigated the expression of CENPF in 30 other pairs bladder cancers tissues, and the results were in accordance with the sequencing results. Furthermore, Immunohistochemical staining, showed that strong intensity of CENPF in BC tissues than normal tissues. Increased staining of CENPF was detected of tumor cells in MIBC compared with NMIBC. This suggests that CENPF might be highly expressed in aggressive and invasive tumor cells. Subsequently, in vitro functional experiments also demonstrated that the siRNA interference of CENPF expression significantly weakened the proliferation, migration, invasion and apoptosis of BC cells, and the cells were arrested in the G2/S phase in Cell cycle. Moreover, functional enrichment analyses, lncRNA-miRNA-mRNA, and protein-protein interaction networks revealed that CENPF was potentially involved in carcinogenesis and evolution of bladder cancer. Taken together, these results demonstrated that CENPF may serve as a potential biomarker of tumor occurrence, progression, and even prognosis for bladder cancer. However, further research is needed to be further clarified the pathway mechanisms of CENPF in bladder cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10677-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcea81a86745e041752c09be80ee21f9ca4c7d66" target='_blank'>
              CENPF overexpression in bladder cancer cells enhances proliferation, migration, invasion, and apoptosis
              </a>
            </td>
          <td>
            Ruixiang Song, Xin Chen, Zhensheng Zhang, Huiqing Wang, Jinshan Xu, Shuxiong Zeng, Wentao Zhang, Zhaoyang Sheng, Xudong Yao
          </td>
          <td>2025-07-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Small Ubiquitin-like Modifier 1 Pseudogene 3 (SUMO1P3) is a novel long non-coding RNA (lncRNA) located at the 1q23.2 locus of the human chromosome. Recent evidence indicates that SUMO1P3 is aberrantly upregulated in nine types of human cancer and functions as an oncogene. Elevated SUMO1P3 expression is strongly associated with unfavorable clinicopathological features and poor prognosis in eight cancer types. Mechanistically, SUMO1P3 functions as a miRNA sponge, an epigenetic regulator, and directly interacting with proteins. It activates key signaling pathways, such as the Wnt/β-catenin and AKT pathways, and regulates Epithelial- Mesenchymal Transition (EMT), which facilitates cancer progression and therapy resistance. Due to its diverse functional roles, SUMO1P3 emerges as a promising diagnostic and prognostic biomarker, as well as a potential therapeutic target in precision oncology. This review provides a comprehensive summary of current research on SUMO1P3, highlighting its regulatory mechanisms, biological functions, and clinical significance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4f3e3fbd6a4fd472988ea5e4c6612a35b0592f" target='_blank'>
              The Vital Role of Long Non-Coding RNA SUMO1P3 in the Regulation of Human Cancer: Current Perspectives and Future Challenges.
              </a>
            </td>
          <td>
            Jingjie Yang, Yuzhang Wei, Chengran Gao, Zihang Wang, Yulong Liu, Haodong He, Hao Zhou, Guihua Liao, Gang Zhou, Chengfu Yuan
          </td>
          <td>2025-07-29</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Circular RNA (circRNA), a class of evolutionarily conserved, structurally stable, and tissue-specifically expressed noncoding RNA, is increasingly recognized as a key regulator of diverse biological processes and disease pathogenesis, including cancer. While the roles of circRNAs in tumorigenesis are well-documented, their involvement in the tumorigenesis induced by environmental chemical carcinogens (ECCs) remains relatively unexplored. Recent studies have identified aberrant expressions of specific circRNAs during ECC exposure-related carcinogenesis, suggesting their critical regulatory functions. Given their unique structure and broad regulatory roles, circRNAs exhibit great potential as diagnostic, therapeutic, and prognostic biomarkers for ECC exposure-associated cancers. This review summarizes the characteristics and functions of circRNAs, as well as the potential regulatory mechanisms in ECC exposure-induced cancer and the dysregulations of circRNAs caused by ECCs. We highlight the complexity and heterogeneity of circRNA regulatory networks, emphasizing the need for integrated and dynamic investigations to fully elucidate the underlying mechanisms. Future research efforts should prioritize biomarker studies to facilitate the prevention, early detection, and effective treatment of ECC exposure-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760c4eaf87c1e364f138dd2080aea7e51f03d039" target='_blank'>
              Circular RNAs: A Potential Regulator in Environmental Chemical Carcinogenesis.
              </a>
            </td>
          <td>
            Huijun Huang, Yiqi Zhou, Yiyan Huang, Jiaxin Wang, Shiyi Ouyang, Meiqi Lan, Lieyang Fan, Yun Zhou
          </td>
          <td>2025-07-17</td>
          <td>Chemical research in toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Christelle M. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Mobile genetic elements drive bacterial evolution by promoting genetic plasticity and adaptation; among them, mycoplasma Integrative and Conjugative Elements (ICEs) challenge the notion that mycoplasmas evolved solely through genome reduction. However, they remain poorly characterized, particularly in Mycoplasma hominis, which is a human genital pathogen. In this study, we investigated the circularization, transcription, and protein expression of M. hominis 4788 ICE (ICEHo4788) under the following four environmental conditions: axenic growth, coculture with HeLa cells, mitomycin C exposure, and thermal stress. During axenic growth, ICEHo4788 circularization peaked at 12 h with a 9.4-fold increase in the number of circular forms. Mitomycin C and cold shock induced only a moderate increase in circularization (approximately 3-fold), whereas heat shock significantly reduced the number of circular forms. In coculture with HeLa cells, a strong increase in the number of circular forms was observed at 72 h and 7 days postinfection with 10- and 23-fold increases, respectively. Coculture also led to a 5- to 23-fold increase in transcription at 7 days, whereas axenic growth, mitomycin C exposure, and thermal stress did not differ. Proteomic analysis revealed that MhoH(a), CDS18, CDS17, and CDS11 were significantly overexpressed at 12 h. Neither mitomycin C nor cold shock induced changes in ICE-related proteins, but heat shock upregulated two M. hominis molecular chaperones and ICEHo4788 MhoH(a). In conclusion, environmental conditions modulate M. hominis ICE activity with strong stimulation in the presence of eukaryotic cells. These findings offer new insights into the regulation and environmental responses of M. hominis ICEs. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00364-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77153e15d1086f019614f6f40398cb2e7ebb62eb" target='_blank'>
              Interaction with eukaryotic cells enhances the circularization and transcription of integrative and conjugative elements in Mycoplasma hominis
              </a>
            </td>
          <td>
            Bachir Boureima Abdou, Alicia Silvant, C. Le Roy, Jennifer Guiraud, Léa Bientz, Laure Béven, Véronique Dubois, Eric Baranowski, Cécile Bébéar, S. Pereyre
          </td>
          <td>2025-07-04</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a rare genetic disorder caused by defects in the repair of DNA interstrand crosslinks (ICLs)—highly toxic lesions that impede essential processes like DNA replication and transcription, leading to severe genome instability. Clinically, FA presents with a broad spectrum of symptoms, including progressive bone marrow failure, congenital abnormalities, and an elevated predisposition to various malignancies, particularly acute myeloid leukemia and squamous cell carcinomas. This review provides a comprehensive overview of both the endogenous and exogenous sources of ICLs and the DNA repair pathways responsible for their resolution, with a primary focus on the FA pathway. We also discuss the tumorigenic consequences of FA pathway deficiencies, highlighting the molecular mechanisms that contribute to the heightened cancer risk observed in FA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6689c7c75058ca5686e353d84e63237862e17744" target='_blank'>
              Comprehensive review on Fanconi anemia: insights into DNA interstrand cross-links, repair pathways, and associated tumors
              </a>
            </td>
          <td>
            Chenyan Fang, Zhoujun Zhu, Jun Cao, Jun Huang, Yipeng Xu
          </td>
          <td>2025-07-30</td>
          <td>Orphanet Journal of Rare Diseases</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain tumors are one of the most severe types of malignant tumors and glioma accounts for ~80% of malignant brain tumors. The current treatment methods for glioma are limited and patients with glioma often experience relapse following treatment, which leads to a poor prognosis for these patients. Therefore, novel therapeutic targets and methods urgently need to be explored. The present review screened studies that mainly focused on the epigenetic regulation of small guanosine triphosphate (GTP)ase in glioma. These small GTPases participate in most cellular biological processes, including differentiation, proliferation, cell migration, apoptosis, vesicle and organelle dynamics and transport, nuclear dynamics and cytoskeleton regulation. Due to the diversity and importance of the biological functions of small GTPases, an increasing number of studies have focused on them; however, the incidence of changes in the gene structure of small GTPases is considered to be low in glioma. Several studies have shown that the abnormal expression of genes encoding small GTPases is often influenced by epigenetic regulation in glioma. Epigenetic regulation is a dynamic and reversible process, which implies that the reversal of abnormal epigenetic modifications is a potential treatment strategy for glioma. These previous studies, which are summarized in the present review, not only provide new therapeutic targets and prognostic markers, but also provide information regarding the treatment of glioma. The current review may provide valuable insights for future research and promote the clinical translation of relevant research results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b0933c82de7174662d27dd036b6327ca1cf162" target='_blank'>
              The role of abnormal epigenetic regulation of small GTPases in glioma (Review)
              </a>
            </td>
          <td>
            Mingyang Zhang, Yimin Huang, Qiang Zhang, Xiaoyan Zhang, Lumei Kang, Jianguo Wang
          </td>
          <td>2025-07-02</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Adenoid cystic carcinoma (ACC) is a rare, malignant neoplasm of the salivary glands that, despite its slow growth, exhibits aggressive behavior, including perineural invasion, high recurrence rates, and distant metastasis. The lack of effective systemic therapies, combined with the resistance of cancer stem cells (CSCs) to standard treatments, highlights the urgent need for novel therapeutic strategies. Methods In this study, we employed a high-throughput screening approach to evaluate a library of 157 histone modification drugs (HMDs) using the UM-HACC-2 A cell line. Two complementary in vitro models were used: traditional two-dimensional culture representing the bulk non‑CSC tumor population and three‑dimensional tumorspheres that enrich for CSCs. Automated liquid handling, high‑content imaging (ImageXpress), and image analysis (MetaXpress) enabled precise quantification of cell viability and tumorsphere characteristics following treatment with HMDs. Results Screening identified 14 candidate compounds that induced statistically significant cell death in non‑CSCs and 11 compounds that were effective against tumorspheres. Key epigenetic targets included histone deacetylases, histone methyltransferases, EZH2, c‑RET, and EGFR. Notably, ITF2357 (Givinostat) induced 84% cell death in non-CSC populations (p < 0.0001), while the histone methyltransferase inhibitors, UNC0631 and LLY-507, elicited cell death rates of 82.5% (p < 0.0001) and 82.3% (p < 0.0001), respectively. In the CSC model, GSK343, an EZH2 inhibitor, induced 35.2% cell death (p < 0.0001). Furthermore, combination assays, pairing HMD hits from CSC and non‑CSC screenings, revealed synergistic effects that significantly enhanced tumor cell death compared to monotherapy. Conclusion These findings demonstrate that CSCs and non‑CSCs respond differently to epigenetic modulation, suggesting that personalized combination therapies targeting both subpopulations may be necessary to improve treatment outcomes for ACC. Although these targeted therapies are still in the early stages of investigation, our study provides a promising pre-clinical foundation for the development of effective, personalized therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33e3493b3bf6ce0048e0a466a02b73273090d02" target='_blank'>
              Assessing the selective impact of histone modifying drugs on adenoid cystic carcinoma cells and their stem cell counterparts
              </a>
            </td>
          <td>
            Carolina Emerick, Luan César da Silva, Yeejin Jang, Pablo A. Vargas, C. Squarize, Rogerio M. Castilho
          </td>
          <td>2025-08-07</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Previously, we identified that mutant p53 expression in cancer cells promotes engulfment of neighbouring cancer cells to form cell-in-cell (CIC) structures. This process gave mutant p53 cells an advantage in tumour formation in mouse xenograft experiments. TP53 can be found mutated at nearly every amino acid in cancers and mutant p53 expression is associated with various GOF (Gain-of-function) processes, including cancer cell invasion, metastasis, stemness and drug resistance. In the current manuscript, we identified SH3BGRL (Src homology 3 binding glutamate rich protein like) as a mutant p53-regulated gene and investigated to what extent SH3BGRL expression and cell engulfment are responsible for mutant p53-dependent anchorage-independent growth and chemoresistance. We demonstrate that mutant p53 expression drives cell engulfment in which the mutant p53 host cell moves in the direction of the target internal cell to form CIC structures. This is therefore more reminiscent of cell engulfment rather than cell entosis, in which cells invade into host cells. Using NGS (Next Generation Sequencing), we identified novel target genes of mutant p53 and demonstrate that cell engulfment requires SH3BGRL expression. We generated mutant p53 and p53 KO cell lines that stably overexpressed SH3BGRL and determined that SH3BGRL promotes etoposide resistance in mutant p53 cells and anchorage-independent growth independent of mutant p53 expression. Through FACS sorting of pure cell engulfing (CIC) populations, we could also show that engulfing cells have an enhanced etoposide resistance. These data suggest that SH3BGRL and cell engulfment are required for certain GOFs of mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba6ff758bccb84d82f6118096ce3f5daf639f6e" target='_blank'>
              Mutant p53 induces SH3BGRL expression to promote cell engulfment
              </a>
            </td>
          <td>
            Lobsang Dolma, Mary I Patterson, Antonia Banyard, C. Hall, Steven J Bell, Wolfgang Breitwieser, Sudhakar Sahoo, John Weightman, Maria Pazos Gil, Garry Ashton, C. Behan, Nicola Fullard, Lewis D Williams, P. Muller
          </td>
          <td>2025-07-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Eukaryotic Argonautes (eAgos) have traditionally been characterized by their ability to utilize RNA guides to identify RNA targets, thereby engaging in post-transcriptional gene silencing pathways. While some eAgos have been demonstrated to use DNA guides for RNA cleavage, the ability of eAgos to cleave DNA targets remains unclear. In this study, we characterized CsAgo, an eAgo protein derived from thermophilic eukaryote Chaetomiumsp. MPI-CAGE-AT-0009, demonstrating a novel ability to cleave both DNA and RNA targets in vitro. Guided by short single-stranded DNA (ssDNA) or RNA, CsAgo exhibits robust RNA cleavage activity at 20–90°C in vitro. CsAgo can effectively cleave ssDNA guided by RNA guides at 20–50°C in vitro. Notably, CsAgo can utilize DNA guides to effectively cleave ssDNA, plasmid double-stranded DNA (dsDNA), and linear dsDNA at ≥80°C in vitro. Based on its ability to cleave dsDNA at high temperatures, CsAgo demonstrates versatility and efficacy in simplifying routine cloning workflows. Additionally, we have developed a CsAgo-based nucleic acid detection method based on a Pyrococcus furiosus Ago-mediated nucleic acid detection method, which exhibits a high sensitivity of six copies/reaction. These results suggest that eAgos include that, in theory, can be utilized for potential DNA-targeting applications. This not only enhances our understanding of eAgos but also expands the toolkit for DNA manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83907ec94e09a230d08b65521fe7a733061ab1d" target='_blank'>
              In vitro programmable DNA cleavage by a eukaryotic Argonaute
              </a>
            </td>
          <td>
            Guangbo Yan, Xia Li, Longyu Wang, Fei Wang, Siyuan Lv, Xiaolan Yu, Zhongchen Li, Jiakai Cui, Baotong Sun, Jia Hou, Wenqiang Li, Xiaochen Xie, Shuai Zhao, Zhiwei Zhang, Wanping Chen, Yanhong Peng, Yang Liu, Lixin Ma
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background SUGT1 (Suppressor of the G2 allele of SKP1) and FH (fumarate hydratase) have recently garnered significant attention from the research community. SUGT1 functions as a molecular chaperone, regulating the stability and activity of various proteins, while FH is a key enzyme in the tricarboxylic acid cycle, catalyzing the reversible conversion of fumarate to malate. Existing literature has established their essential roles in signaling, tumorigenesis, and cancer progression. However, their functions and mechanisms in ovarian cancer (OC) remain poorly understood. Results We found that high SUGT1 expression is associated with a more advanced FIGO stage in OC. SUGT1 knockdown significantly inhibits OC cell proliferation and metastasis, while its overexpression has the opposite oncogenic effect. Mechanistically, we revealed that SUGT1 promotes FH protein degradation via the ubiquitin-proteasome pathway. Moreover, FH knockdown partly reversed the inhibitory effects of SUGT1 knockdown on tumor cell proliferation, migration, and proteins of phosphorylated PI3K/AKT and Vimentin. In summary, We demonstrated that SUGT1 exerts oncogenic functions in OC by regulating FH stability. Conclusions Our study is the first to provide experimental evidence elucidating the SUGT1-FH relation and its role in OC progression, offering potential significance for clinical diagnosis and therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01744-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467d1c72a28cbccbc2152c3f132424b0a5db294b" target='_blank'>
              Mechanism by which SUGT1 downregulates FH to promote proliferation and migration in serous ovarian cancer
              </a>
            </td>
          <td>
            Tianli Mu, Bo Ren, Ziteng Kuang, Runze He, Bingjie Rui, Ye Yang, Yuxi Liu, Danbo Geng, Yuci Zhang, Min Wang
          </td>
          <td>2025-07-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Q. Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 200,000 deaths each year. Despite recent advances in treating HGSC with neoadjuvant chemotherapy, the majority of patients ultimately develop chemotherapy resistance. HGSC is characterized by TP53 mutations and widespread copy number alterations and occurs frequently in the setting of deleterious germline BRCA1/2 variations, but many cases lack putative driver mutations. Here, we performed whole-exome, whole-genome, and whole transcriptome sequencing along with mass spectrometry to characterize the molecular landscape of HGSC in 22 paired samples obtained before and after neoadjuvant chemotherapy. Responsiveness to chemotherapy was determined for each patient. Evidence at the DNA, RNA, and protein level revealed numerous defects in cell-cell and cell-matrix interactions as well as disruption of cell polarity and cytoskeletal regulation in HGSC, indicating that defects in epithelial integrity were present in the majority of HGSC patients. Non-responsive HGSC harbored subclones with putative survival mutations. Additionally, ineffective nonsense mediated decay resulted in the persistence of transcripts with frameshift mutations that were translated into aberrant proteins detectable in HGSC samples. Together, these findings suggest that HGSC may arise through defects in maintenance of epithelial integrity that lead to shedding of malignant cells throughout the peritoneum, and the presence of resistant subclones prior to chemotherapy may decrease the chemosensitivity of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/896a702c0b394e4d17c91e5aa33566545b16c4dc" target='_blank'>
              Multi-omic Characterization of Pre- and Post-neoadjuvant Chemotherapy Treated Ovarian Cancer Reveals Mediators of Tumorigenesis and Chemotherapy Response.
              </a>
            </td>
          <td>
            Mark C Valentine, Mary M Mullen, E. Kotnik, Matthew A Powell, Premal H Thaker, C. McCourt, L. Kuroki, Andrea R Hagemann, Rebecca C Arend, A. Holehouse, R. Mitra, D. Mutch, Katherine C Fuh
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Development is characterized by dynamic changes in gene expression as cells traverse genetic pathways and make lineage-specific commitments. Transcription factors, which drive gene expression, and microRNAs, the largest class of post-transcriptional regulators, often function together within the same genetic networks. These interactions frequently include direct regulation of one another and shared target genes, forming feedback and feedforward loops that fine-tune gene expression to establish and maintain cell identity. The interplay between transcriptional and post-transcriptional regulation is particularly extensive during development, where disruptions in gene expression programs can cause cells to become trapped in immature proliferative states that result in paediatric cancers. This review focuses on the intricate cross-regulation between transcription factors and microRNAs, highlighting their contributions to developmental cancers with a particular emphasis on neuroblastoma, the most prevalent extracranial solid tumour in children, which arises from the failure of neural crest-derived cells to properly differentiate during sympathoadrenal development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008a3cb7bc612567fe4ce248462398bebd221cff" target='_blank'>
              Regulation and dysregulation of microRNA - transcription factor axes in differentiation and neuroblastoma
              </a>
            </td>
          <td>
            Fakhira H Nazki, C. Bracken
          </td>
          <td>2025-08-08</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The mechanisms of cancer are discussed by analyzing the characteristics of the functional state and biological behavior of the abnormal nuclear cells. The abnormal nuclear cells with abnormal nuclear structure and function are a kind of sick cell or functional defect cells having existed in human body for a long time. The abnormal nuclear cells are resulted from the nuclear damage caused by the radiation, viruses and various carcinogenic compounds. Some of genes in human body are expressed, some are not expressed for life. The expressional genes are functional genes, the genes never expressed for life in human body are dormant genes or sealed genes. The nuclear damages destroy cell state of differentiation, affect gene expressional regulation and change gene expressional profiling, resulting in loss of expression of the functional genes and reactivation of the sealed genes; which finally leads to cancer, aging and other chronic refractory diseases. The cancer is not resulted from the genetic mutations or chromosomal aberrations, but rather the reactivation of genes involved in proliferation due to the nuclear damage. The biological characteristics of the cancer cells, such as the shedding and metastasis, immune tolerance, uncontrolled, loss of contact inhibition function and so on, all originate from the nuclear aberrant cells. The nuclear damage can trigger the genes that drive mitosis, leading to cancer. Thus, re-sealing the several genes that trigger the proliferation may completely prevent or cure cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd09554e196fbeabd1e2fd9b39e2eabe92efdf78" target='_blank'>
              Cancers Are Derived from the Disruption of Cell Differentiation
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-06-23</td>
          <td>Cancer Research Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary bone tumor, which is associated with a high mortality rate. The c-Fos transgenic mouse model has been described to spontaneously develop osteosarcoma, and the ribosomal S6 kinase 2 (Rsk2) was found to be essential for c-Fos-induced osteosarcoma formation in mice. By isolating and characterizing osteosarcoma cell lines from FosTg and FosTg;Rsk2−/y mice, we observed that Rsk2 deficiency impairs the growth advantage of FosTg cells. This can be explained by the aberrant number of nuclei due to impaired cytokinesis, inducing mitotic catastrophe. We therefore tested a pharmacological Rsk inhibitor (BI-D1870) for its ability to inhibit the proliferation of osteosarcoma cells and found that the effects observed by genetic Rsk2 inactivation were mimicked. BI-D1870 administration to FosTg cell lines led to reduced expression of Aurora kinase B. Therefore, the influence of a pharmacological Aurora kinase B inhibitor (Hesperadin) was tested. Similar to BI-D1870, Hesperadin caused impaired cytokinesis, resulting in the accumulation of polynuclear cells. This effect was also observed for two human osteosarcoma cell lines, U2OS and SaOS-2. Based on our findings, Rsk2 and/or Aurora kinase B can serve as potential targets for the design of new osteosarcoma therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0b0bb70e617786bf522d9e1ae67f5c3186803" target='_blank'>
              Rsk2 inhibition induces an aneuploid post-mitotic arrest of cell cycle progression in osteosarcoma cells
              </a>
            </td>
          <td>
            Armelle Carreau, C. Baldauf, Lena Warlich, Magdalena Weingartner, Laura J Brylka, M. Amling, T. Schinke, Julia Luther
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) represents the most common solid renal malignancy, accounting for 90% of kidney cancers and 3% of all malignancies. Clear cell RCC (ccRCC) is the predominant histologic subtype, comprising roughly 80% of cases. Despite a 5-year survival rate of 75%, up to 30% of patients with localized ccRCC eventually develop recurrence or distant metastases. Mitotic spindle organizing protein 2A (MZT2A), also known as FAM128A, GCP8, or MOZART2A, is located on chromosome 2q21.1 and encodes a 158-amino acid protein first described by Hutchins et al. in 2010. To date, limited studies have linked MZT2A to tumorigenesis, primarily in lung and gastric cancers. The tumor microenvironment (TME), particularly tumor-infiltrating immune cells, plays a critical role in cancer progression and prognosis. Using paired tumor and normal samples from TCGA, we assessed MZT2A expression across multiple cancer types. Collectively, these findings suggest that MZT2A is involved in immune cell infiltration and the tumor microenvironment, may promote KIRC progression, and is overexpressed in cases with poor outcomes, supporting its potential as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d43bd634f019908094dd923cc5fef3d5a13d2f5" target='_blank'>
              Integrated transcriptomic analysis identifies MZT2A as a prognostic and immune-related biomarker in KIRC.
              </a>
            </td>
          <td>
            Yunjing Zhang, Zekun Xin, Xinnuo Li, Yanmei Chen
          </td>
          <td>2025-07-15</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many pathogenic fungi display 'two-speed genome', with the fast-evolving genomic compartments enriched with repetitive sequences, particularly the transposons, which have been shown to drive the variation of pathogenicity-associated genes. Supernumerary chromosomes (SCs) are known to facilitate genomic variation in fungal pathogens, but their specific role in such processes remains understudied. In this study, we assessed the transferability of SCs between asexual Magnaporthe oryzae strains during co-culture and co-infection, and investigated their role in genome reconstruction through experimental evolution assays. We found that SCs could be horizontally transferred between M. oryzae strains and revealed frequent structural variations facilitated by SCs, including deletions, duplications, translocations, and SC-core chromosome recombinations during and after horizontal transfer. Remarkably, all observed intra- and inter-chromosome rearrangements were confined to core chromosome ends and SCs, indicating a robust role of SCs in facilitating genetic exchange within fast-evolving genomic compartments. Additionally, SC carrying the avirulence gene AvrPikE modulates M. oryzae virulence against Pikh rice through horizontal transfer, loss of whole SC, and segmental deletions. Our findings establish SCs as critical players in shaping the diversity and dynamics of the pathogenic fungal genomes, highlighting them as a cradle for the variation of pathogenicity-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fc7ba98841d16bfd287154023ca3410a86f26c" target='_blank'>
              Experimental insights into genome reconstruction driven by horizontal transfer of supernumerary chromosomes in Magnaporthe oryzae.
              </a>
            </td>
          <td>
            Zhenyu Fang, Yuyong Li, Jianqiang Huang, Meilian Chen, Xiali Chen, Xueting Mo, Z. Zhong, Xiuxiu Li, G. Lu, Guifang Lin, Zonghua Wang, Huakun Zheng
          </td>
          <td>2025-08-03</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Non-coding RNAs (ncRNAs) in human cells constitute a substantial portion of the transcriptome but do not lead to protein synthesis. Among them, long non-coding RNAs (lncRNAs, > 200 nucleotides long) are fascinating in their ability to orchestrate critical cellular functions that govern cell development, differentiation, and metabolism. Therefore, the dysregulation of lncRNAs has been linked with several diseases, chiefly cancers. Results We focused here on colorectal cancer (CRC), the second-highest cause of mortalities related to cancer worldwide, and more particularly on three lncRNAs, i.e., LINC01589, MELTF-AS1, and UXT-AS1, known to be dysregulated in CRC. We identified a vulnerability in these lncRNAs that could be exploited from a therapeutic point of view: a part of their sequence folds into a secondary structure referred to as G-quadruplex (G4), which is suspected to play active roles in the lncRNA functions. We demonstrate here that these sequences do fold into G4s both in vitro and in CRC cells, and that these G4s can be modulated using PhpC, a prototype molecule for destabilizing G4s. Conclusion We describe an innovative anticancer strategy that fully abides by the rules of chemical biology. We indeed modulate the formation of G4s in cells using ad hoc molecular tools in the aim of disturbing the homeostasis and inner functioning of lncRNAs. By exploiting cellular outcomes, we infer how this pharmacomodulation affects CRC biology and, beyond this, the fate of CRC cells owing to the flawed repertoire of correction and/or compensatory mechanisms in cancer cells. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02322-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754c36f086e107d8742747ebe134ba178418b705" target='_blank'>
              Pharmacomodulation of G-quadruplexes in long non-coding RNAs dysregulated in colorectal cancer
              </a>
            </td>
          <td>
            Shubham Sharma, Jérémie Mitteaux, A. Pipier, Marc Pirrotta, Marie-José Penouilh, D. Monchaud, Bhaskar Datta
          </td>
          <td>2025-08-08</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="NUT carcinoma (NC) is a highly aggressive malignancy driven by BRD4::NUTM1 and other NUTM1 fusion oncogenes. BRD4::NUTM1 aberrantly activates transcription factors (TFs) linked to basal progenitor identity, producing poorly differentiated squamous phenotypes. Among these TFs, SOX2 has been proposed as a critical oncogenic driver, but its functional requirement in NC has not been tested in vivo. Using a genetically engineered mouse model that faithfully recapitulates human NC, we performed lineage-specific conditional deletion of Sox2 in both squamous and non-squamous tissues. We found that Sox2 is dispensable for NC initiation and progression, with tumors retaining characteristic histology and expression of key drivers including BRD4::NUTM1 , MYC , and TP63 . Transcriptomic profiling revealed only modest changes in Sox2 -deficient tumors, mainly affecting metabolic and biosynthetic pathways, without disrupting core oncogenic programs. These findings challenge the assumption that SOX2 is universally required in NC and suggest that SOX2-targeted therapies may have limited utility, refining the framework for therapeutic prioritization.


Summary blurb
This study shows that SOX2 is not required for NUT carcinoma initiation or maintenance in vivo, challenging its assumed oncogenic role and refining therapeutic target prioritization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4cff3ee3c7d90bb73c3a8d68c2134c4b228a48" target='_blank'>
              SOX2 is a dispensable modulator of NUT carcinoma oncogenesis in mouse.
              </a>
            </td>
          <td>
            Chenxiang Luo, Dejin Zheng, Ahmed Elnegiry, Sheetal Bhatara, G. Mias, M. Tsoi, Jiyang Yu, Bin Gu
          </td>
          <td>2025-07-14</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are covalently closed RNA molecules generated through a non-canonical splicing event known as back-splicing. This particular class of non-coding RNAs has attracted growing interest due to its evolutionary conservation across eukaryotes, high expression in the central nervous system, and frequent dysregulation in various pathological conditions, including cancer. Traditionally, circRNAs have been characterised by their ability to function as microRNA (miRNA) and protein sponges. However, recent discoveries from multiple research groups have uncovered a novel and potentially transformative mechanism of action: the direct interaction of circRNAs with messenger RNAs (mRNAs) to regulate their fate. These interactions can influence mRNA stability and translation, revealing a new layer of post-transcriptional gene regulation. In this review, we present and analyse the latest evidence supporting the emerging role of circRNAs in diverse biological contexts. We highlight the growing body of research demonstrating circRNA-mRNA interactions as a functional regulatory mechanism and explore their involvement in key physiological and pathophysiological processes. Understanding this novel mechanism expands our knowledge of RNA-based regulation and opens new opportunities for therapeutic strategies targeting circRNA-mRNA networks in human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d4d3f0032738d3daf69daef3e7b0d31248f1c4" target='_blank'>
              Direct circRNA-mRNA Binding Controls mRNA Fate: A New Mechanism for circRNAs
              </a>
            </td>
          <td>
            Raffaele Garraffo, Manuel Beltran Nebot
          </td>
          <td>2025-07-18</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Clinical outcomes can be linked to the fundamental role played by the regulatory 3D architecture of the genome in a living cell and the unique properties offered as a molecular biomarker modality. Here, we describe how this molecular phenomenon acts as an imprint of the cellular genetic, epigenetic and metabolic status which form discriminatory stable conditional structures that inform on the state of the cell and its phenotype. Consequently, 3D architecture, as a biomarker, offers a more comprehensive translation from molecular readouts to clinical outcomes, with examples and evidence provided by the highly accurate diagnostic and prognostic clinical tests already reduced to practice in the US and UK. Moreover, with support from Large Language Models, we can elucidate the complexities and inter-relationships of molecular pathways and regulation using a 3D genomic architecture knowledge base, to gain insights into various molecular modalities and associated clinical conditions. Abstract Background/Objectives: The 3-dimensional (3D) architecture of the genome in the nucleus of a living cell plays an unexpected yet fundamental regulatory role in cell biology. As an imprint of the cellular genetic, epigenetic and metabolic status, it discriminates pathological conditions through conditional changes to long-range 3D interactions (up to 300 kb) and thus could act as a powerful molecular biomarker linked closely to clinical outcomes. Methods: Here an assessment is made of the latest paradigm shift in molecular biology from a supply chain where information flows from DNA to RNA to protein, to the concept of heritable 3D folding of the genome reflecting the epigenetic and metabolic state of the cell, and which serves as a molecular biomarker for complex clinical outcome. Results: While biomarkers based on individual components of the supply chain fail to accurately reflect clinical outcomes, 3D genomics offers highly informative insights, exemplified for immuno-oncology and prostate cancer diagnosis by clinical tests of superior performance, already in practice in the US and UK. Conclusions: A more complete understanding of human biology will require models that account for the flow of information to and from the 3D genomic architecture in living cells, together with its regulation and logic. Integrating these principles into biomarker discovery and therapeutic design, along with other frontline approaches in precision medicine, including multi-omics and other system-level tools, will be essential for advancing precision medicine beyond its current limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe42115687f08e85dd377e033fb44be5d4f1cc1" target='_blank'>
              Paradigm Lost
              </a>
            </td>
          <td>
            Jane Mellor, E. Hunter, A. Akoulitchev
          </td>
          <td>2025-06-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer is an enigma, and different types and subtypes add to the complexity of this disease. Innumerable molecules play important roles in glial pathogenesis. One such molecular component discovered recently and originating from the non-coding region of the human genome is the long non-coding RNA, or lncRNA. Multiple lncRNAs have been shown to play a role in tumor development in different cancer types. Although underappreciated as a therapeutic component, emerging studies have highlighted their crucial contributions to intercellular signaling. Studies suggest that lncRNAs play multiple, diverse roles in cancer cells. Their contribution to disease pathogenesis seems contrasting because the same lncRNA facilitating tumorigenesis in a particular cancer type may play a completely antagonistic role in another. In this review, we discuss multiple lncRNAs and the intracellular targets that are pertinent to glioma pathology, the sensitive expression levels, their impact on the disease progression, the pathways they affect, and their consideration as potential biomarkers for glioma. We further discuss multiple lncRNAs in the most aggressive glioma subtype, glioblastoma. Overall, we provide comprehensive information about key lncRNAs in glioma pathogenesis which could be of future fundamental and therapeutic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f473a5d748d1ada307293cc0b8a9bbbf79222c54" target='_blank'>
              Long non-coding RNA as modulators of brain pathology: insights into cellular signaling diversity
              </a>
            </td>
          <td>
            Megha Chaudhary, Shreya Maity, Naveen Soni, Shweta Dongre, Ashwani Tiwari, Bhawana Bissa
          </td>
          <td>2025-07-24</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Mismatch repair (MMR) deficiency is a hallmark of microsatellite instability (MSI) in hereditary non-polyposis colorectal cancer, Lynch syndrome, contributing to resistance against conventional chemotherapy and posing a significant therapeutic challenge. In this study, we introduce UNI110, a novel small molecule derived from Baicalein, engineered for enhanced selectivity against MMR-deficient cancer cells. UNI110 exhibits a remarkable sevenfold increase in potency over Baicalein, demonstrating significantly lower IC50 values and heightened cytotoxic effects in MMR-deficient cell lines. Mechanistically, UNI110 selectively induces DNA damage in MMR-deficient cancer cells, ultimately resulting in cell death. Furthermore, UNI110 disrupts homologous recombination (HR) repair by inhibiting the MSH2-MSH3 complex, specifically blocking the interaction between MSH2 and EXO1, thereby impairing long-range end resection during double-strand break (DSB) repair. These findings establish UNI110 as a promising lead compound for the targeted treatment of MMR-deficient colorectal cancers, offering a potential breakthrough in overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5c30430d4a1e738e7be9c0b1bd6508b0c419a8" target='_blank'>
              Targeting MMR-deficient colorectal cancer with a potent small molecule UNI110
              </a>
            </td>
          <td>
            Enkhzul Amarsanaa, Jung-Min Oh, Seon Young Lee, Saikat Maiti, Sung You Hong, Kyungjae Myung
          </td>
          <td>2025-08-04</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite significant successes, immunotherapy continues to face numerous challenges in practical applications. CD82 plays an important role in the tumor microenvironment (TME) across various cancers. However, the prognostic value of CD82 in immunotherapy and its role in the immune response remain unclear. The expression levels of CD82 in healthy human tissues were assessed using the human protein atlas (HPA) database tool. The association among CD82 expression levels, copy number variations (CNV), and methylation was explored using the gene set cancer analysis (GSCA) database. Timer tumor immune estimation resource (TIMER) database is used for immune checkpoint and immune infiltration. The LinkedOmincs database is used for enrichment analysis. The statistical inquiry was performed through the R program (version 4.4.1). The result has shown that CD82 mRNA levels were significantly downregulated in most tumor types. Abnormal CD82 mRNA expression was associated with CNV and methylation changes in tumor tissues. Furthermore, several immune checkpoint genes related to CD82 influenced prognosis and were linked to increased immune cell infiltration across various cancer forms, including liver hepatocellular carcinoma (LIHC). The prognosis of the patients under multiple cancer forms, particularly LIHC, was remarkably influenced by CD82, which plays a vital role in modulating immune infiltration. CD82 has the potential to serve as both a prognostic indicator and an immunological biomarker in human cancer. CD82 expression is frequently downregulated across various cancer associated with poor prognosis, influenced by CNV and DNA methylation alterations. Its abnormal expression correlates with changes in immune checkpoint gene activity and enhanced immune cell infiltration, notably in liver. These associations suggest that CD82 plays a significant role in shaping the tumor immune microenvironment and affects patient prognosis. Consequently, CD82 holds promise as both a prognostic biomarker and an immunomodulatory target in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e6b1867f8cfd33352d9c4d7c23852fa5423b5ed" target='_blank'>
              Prognostic and immunological significance of CD82 in cancers with focus on hepatocellular carcinoma: A bioinformatics approach
              </a>
            </td>
          <td>
            Sikandar Ali, Shiyu Liu, Jiali Chen, Jawad Ullah, Guoyu Mou, Xiaohua Huang
          </td>
          <td>2025-07-25</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Neuroblastoma, a sympathetic nervous system tumor, is known for its remarkable biological heterogeneity. Unlike other neurogenic malignancies driven by mutations, neuroblastoma carries a significant 'transcriptional burden.' Deregulation of transcription unveils in the course of the tumor's natural history, making the advanced stages of the disease intractable.


AREAS COVERED
Recent research investigated the molecular profiles of key neuroblastoma states: adrenergic (ADRN) and mesenchymal (MES). These categories are defined by core regulatory circuit (CRC) genes whose products (transcription factors) modulate a variety of malignant properties. This review analyzes fundamental mechanisms of regulation of CRC in neuroblastoma, focusing on transcriptional alterations as tentative therapeutic targets. We dissect differential CRC patterns in ADRN and MES states and discuss opportunities for therapeutic combinations targeting CRC along with other survival mechanisms.


EXPERT OPINION
We emphasize the critical importance of combined CRC inactivation as a promising therapeutic avenue. This approach is substantiated by basic biological mechanisms largely demonstrated in experimental models. Although the detailed roles of CRC in individual clinical situations remain to be elucidated, interference with the subtype-specific patterns of CRC-dependent gene expression, together with inactivation of an additional survival factor(s), offers new opportunities for mechanism-based, personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa3ab0bece749f3dd05030992350c29b3fbf920" target='_blank'>
              Targeting transcription in neuroblastoma: focus on the Core regulatory circuit.
              </a>
            </td>
          <td>
            O. Kuchur, S. S. Pogodaeva, Victoria I Zhdankina, Anna V Scherbakova, Alexander A. Shtil, N. V. Antipova
          </td>
          <td>2025-08-11</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Müller glia (MG) are retinal resident cells with diverse functions, including reprograming and regeneration in certain species. While the mammalian retina possesses molecular mechanisms for MG dedifferentiation and neuronal differentiation, it fails to generate neural progenitors in vivo. We previously proposed that an epigenetic barrier, driven by DNA methylation, may prevent complete MG reprograming in response to damage. DNA demethylases, such as Ten-Eleven Translocation (TET) and Growth Arrest and DNA Damage-Inducible Protein 45 (GADD45) families, are induced by damage and play roles in cellular reprograming and regeneration. Tet enzymes are regulated by metabolic factors (iron, vitamins A and C) and respond to environmental stimuli. Here, we investigated DNA demethylation as a regulatory signal for MG dedifferentiation and neuronal differentiation in response to N-methyl-D-aspartate (NMDA), a neurotransmitter used to model retinal neurodegeneration. Using mouse primary cultures and antibodies against methylated (5mC, MeCP2) and unmethylated (5 hmC, H3K4Me3) DNA, we analyzed MG epigenetic changes under control, vitamin-supplemented, and NMDA-stimulated conditions. We also assessed DNA demethylase expression and key reprograming genes (Ascl1, Lin28, Nestin). Vitamin A and C increased 5hmC levels but did not upregulate Tet enzymes or reprograming genes. In contrast, NMDA increased Tet3 and reprograming gene expression. Tet3 knockdown led to a rapid 5mC increase and impaired NMDA-induced upregulation of Ascl1, Lin28, and Nestin, suggesting its critical role in MG dedifferentiation. Conversely, Tet3 overexpression induced morphological changes and early neuronal marker expression. These findings identify Tet3 as a key epigenetic regulator of MG reprograming and a potential target for retinal regeneration strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ab362578948bd169fde4a8815b59e370d39ecc" target='_blank'>
              Tet3-mediated DNA demethylation is essential for maintaining the dedifferentiation capacity of mammalian Müller glia
              </a>
            </td>
          <td>
            Erick J. Martinez-Colin, Ivonne Lezama, Lenin D. Ochoa-de la Paz, Mónica Lamas
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Increasing evidence suggests that lncRNAs are core regulators in the field of tumor progression, with context-specific functions in oncogenic tumorigenesis. LINC01133, a lncRNA that has been identified as both an oncogene and a tumor suppressor, remains largely unexplored in terms of its molecular mechanisms. The purpose of this study was to conduct an in silico analysis, incorporating literature research on various cancer types, to investigate the structural and functional duality of LINC01133. This analysis aimed to identify pathways influenced by LINC01133 and evaluate its mechanism of action as a potential therapeutic target and diagnostic biomarker. Methods: In silico analyses and a narrative review of the literature were performed to predict conserved structural elements, functional internal loops, and overall conservation of the LINC01133 sequence among different vertebrate organisms, summarizing the empirical evidence regarding its roles as a tumor suppressor and tumor-promoting roles in various types of tumors. Results: LINC01133 harbors the evolutionarily conserved structural regions that might allow for binding to relevant driver signaling pathways, substantiating its specific functionality. Its action extends beyond classical tumor mechanisms, affecting proliferation, migration, invasion, and epigenetic pathways in various types of tumors, as indicated by the in silico results and narrative review of the literature we present here. Clinical outcome associations pointed to its potential as a biomarker. Conclusions: The dual character of LINC01133 in tumor biology further demonstrates its prospective therapeutic value, but complete elucidation of its mechanisms of action requires further investigation. This study establishes LINC01133 as a multifaceted lncRNA, supporting context-specific strategies in targeting its pathways, and calls for expanded research to harness its full potential in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f66180b93b3358b4aae4204b9c68eb1c73a7d7f" target='_blank'>
              The Good, the Bad, or Both? Unveiling the Molecular Functions of LINC01133 in Tumors
              </a>
            </td>
          <td>
            Leandro Teodoro Júnior, M.C. Sogayar
          </td>
          <td>2025-07-30</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mitomycin C (MC) is a powerful chemotherapy agent currently used in clinics for the treatment of various types of cancer. MC functions by inhibiting cellular growth through the formation of cytotoxic interstrand crosslinks (ICLs). These ICLs induced by MC have minimal impact on the DNA backbone, preserving its B-DNA structure. Recent research suggests that the cellular machinery recognizes and repairs ICLs differently based on their specific structure. To better understand how DNA distortion caused by MC ICLs influences cytotoxic effects, Herein, a novel mitomycin ICL is synthiesized that, unlike MC, significantly distorts DNA and widens the minor groove. This work outlines the synthesis of oligonucleotides bearing a single monoadduct or a single ICL of this new MC derivative at a defined position. Such substrates are widely used for investigations into biological processes such as DNA damage/repair studies. The monoadducted and crosslinked oligonucleotides are thoroughly characterized using various techniques, including enzymatic digestion to nucleosides, mass and circular dichroism spectroscopy, as well as thermal denaturation studies. Furthermore, the structure of this novel crosslinked duplex is compared with that of less-distorting mitomycin ICLs using Molecular Dynamics simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c38bcffc5c385d9f9befa33ae4e8c24ff19f54" target='_blank'>
              Synthesis of Oligonucleotides Containing a Dna Distorting Interstrand Crosslink Produced by Mitomycins.
              </a>
            </td>
          <td>
            Sarah Marks, Gabriel Martinez, Michael Adamov, Christopher Alley, Bianca Davila, Ana G Petrovic, R. Abzalimov, Padmanava Pradhan, Marta Concheiro-Guisan, Elise Champeil
          </td>
          <td>2025-08-14</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Integration of exogenous gene fragments into the host genomes is a widely used and powerful method for studying gene functions, advancing molecular breeding, and conducting gene therapy. Accurately identifying the integration sites is essential for ensuring both the safety and efficacy of genome engineering efforts. However, current mapping techniques are constrained by high costs and a low signal-to-noise ratio. In this study, we developed an innovative tool for mapping integration sites, leveraging T7 polymerase-mediated in vitro transcription (T7-IVT) to capture the junction fragments surrounding integration loci. This approach converts genomic flanking sequences into RNA, enabling the simultaneous enrichment of junction fragments and the elimination of background genomic DNA, thereby significantly enhancing the signal-to-noise ratio. We have validated the efficiency of this method, named T7-PCR, across yeast, plant, and human cells under diverse integration scenarios. T7-PCR outperforms current next-generation sequencing (NGS)-based mapping strategies in terms of efficiency and accuracy, with minimal positional effects. This method is highly applicable for high-throughput transgene screening and also supports the development of next-generation tools for targeted integration of large fragments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ce4746633f2d4d42f9a7aafe673ea0e10428f4" target='_blank'>
              Genome-wide profiling the integration patterns with T7-PCR.
              </a>
            </td>
          <td>
            Ang Li, Chichen Xiao, Jinwen Wang, Xiao Huang, Xionglei He, Kehui Liu, Fan-Nv Xia
          </td>
          <td>2025-08-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Gliomas are classified as one of the most lethal cancers owing to their invasive and aggressive nature. Numerous genetic and epigenetic changes have been studied in the development and progression of GBM. Dysregulation of epigenetic function may bring about profound changes in gene expression to facilitate glioma formation and development. SETD2, a key epigenetic regulator is frequently mutated in various cancers; SETD2 regulates m6A mRNA methylation (epitranscriptome) via H3K36 trimethylation. The m6A RNA methylation modification regulates various biological functions like gene regulation, cell self–renewal and maintenance, cell differentiation, invasion, and tumorigenesis. The RNA modifications are executed by a set of complex proteins collectively known as writers, readers and erasers (RNA modifiers). These modifiers work along with the molecular cues laid down by SETD2 via H3K36 trimethylation. The positive correlation between SETD2 expression and m6A RNA modifiers highlights its direct involvement in epitranscriptomics. Glioma, a highly malignant grade IV brain tumor, is characterized by limited therapeutic options and poor clinical outcomes. However, the specific role of SETD2 in glioma and its impact on m6A RNA methylation remain inadequately defined. In this study, we investigated the expression of m6A RNA methylation regulators, the molecular pathways contributing to tumor progression, and the outcomes associated with SETD2-mediated m6A modifications in glioma. Our in vitro analyses demonstrated that knocking down SETD2 in glioma cell lines resulted in reduced oncogenic properties, such as decreased cell proliferation and invasiveness. This effect was accompanied by a global reduction in m6A methylation levels within the transcriptome, which was closely linked to the downregulation of key m6A writers, METTL3 and METTL14. These findings suggest that SETD2 plays a crucial role in maintaining m6A methylation, and its disruption can drive glioma progression through altered epitranscriptomic landscapes, offering new insights into molecular oncogenesis and potential therapeutic targets.



 Subhadra Kumari, Srinivasan M. THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P75.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a10b3b7631ab056fcfe5ee35c047e63418abb6" target='_blank'>
              Abstract P75: THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION
              </a>
            </td>
          <td>
            Subhadra Kumari, M. Srinivasan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mucosal melanoma (MM) is a deadly cancer derived from mucosal melanocytes. To test the consequences of MM genetics, we develop a zebrafish model in which all melanocytes experience CCND1 expression and loss of PTEN and TP53. Surprisingly, melanoma only develops from melanocytes lining internal organs, analogous to the location of patient MM. We find that zebrafish MMs have a unique chromatin landscape from cutaneous melanomas. Internal melanocytes are labeled using a MM-specific transcriptional enhancer. Normal zebrafish internal melanocytes share a gene expression signature with MMs. Patient and zebrafish MMs show increased migratory neural crest and decreased antigen presentation gene expression, consistent with the increased metastatic behavior and decreased immunotherapy sensitivity of MM. Our work suggests that the cell state of the originating melanocyte influences the behavior of derived melanomas. Our animal model phenotypically and transcriptionally mimics patient tumors, allowing this model to be used for MM therapeutic discovery. As this is a non-MAPK driven genetically engineered model of melanoma, our work also has implications for the 15% of cutaneous melanoma patients who lack MAPK-driving mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99321e90c4b2d4aa26110c1e8b75f84421ebcd50" target='_blank'>
              Specific oncogene activation of the cell of origin in mucosal melanoma
              </a>
            </td>
          <td>
            Swathy Babu, Jiajia Chen, Chloé S. Baron, Kaiwen Sun, Emily J. Robitschek, Alicia M. McConnell, Constance Wu, Aikaterini Dedeilia, Moshe Sade-Feldman, Rodsy Modhurima, Michael P. Manos, Kevin Y. Chen, Anna M. Cox, Calvin G. Ludwig, M. Kellis, Elizabeth I Buchbinder, N. Hacohen, Jie-Ying Yang, Genevieve M Boland, Brian J. Abraham, David Liu, Leonard I. Zon, Megan L. Insco
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="The cyclin-dependent kinase CDK11 is an understudied kinase that has been the subject of conflicting reports regarding its function in cancer. Here, we combine genetic and pharmacological approaches to demonstrate that CDK11 is a critical regulator of cancer cell survival that is required for RNA splicing and the expression of homologous recombination genes. Inhibition of CDK11 disrupts genome stability, promotes the retention of intronic sequences in mature mRNAs, and induces synthetic lethality with PARP inhibitors. Through integrative analysis of functional genomics datasets, we identify heterozygous deletions of chromosome 1p36 - which encompasses CDK11 and its activating cyclin CCNL2 - as a recurrent and predictive biomarker of sensitivity to CDK11 inhibition. To assess the therapeutic potential of CDK11, we develop MEL-495R, a selective and orally bioavailable CDK11 inhibitor. Additionally, we establish a genetically-engineered mouse model that allows us to differentiate between the on-target and off-target effects of CDK11 inhibitors in vivo. Using this platform, we demonstrate that MEL-495R induces widespread on-target toxicity, revealing a narrow therapeutic index. Together, these findings define CDK11 as a core cancer dependency, uncover a chromosomal deletion that sensitizes tumors to CDK11 inhibition, and provide a generalizable strategy for deconvolving drug efficacy and toxicity in vivo for novel oncology targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af348abea2aee38e920e3e84752e406507b1375" target='_blank'>
              On-target toxicity limits the efficacy of CDK11 inhibition against cancers with 1p36 deletions
              </a>
            </td>
          <td>
            Linda A. Julian, Lisa Crozier, Devon A. Lukow, Sanat Mishra, Aditi Swamy, Ryan A. Hagenson, Peter Sennhenn, Erin L. Sausville, Brianna E. Mendelson, Claudio Chuaqui, Lu Qiao, Anand Vasudevan, , Sonam Bhatia, Thierry Bertomeu, Andrew Chatr-Aryamontri, Li Zhang, M. Rees, M. Ronan, Jennifer A. Roth, Tim Nottoli, Suxia Bai, Jayalakshmi Lakshmipathi, Viswanathan Muthusamy, Jonathan G. Van Vranken, Steve P Gygi, Sarah L. Thompson, Joan C. Smith, K. Anderson, Sanjana Shah, R. Bindra, Martin Akerman, David L. Spector, A. Krainer, J. Sheltzer
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Triple negative breast cancers (TNBCs) are a heterogenous subcategory of breast cancers which have significantly worse survival outcomes compared to other breast cancer subtypes. However, limited advances have been made towards a targeted TNBC therapeutic and traditional chemotherapies remain the frontline therapy. Therefore, this study aimed to examine the therapeutic potential of a novel TNBC target, Barrier-to-Autointegration Factor 1 (Banf1), including determining Banf1 expression and cellular localisation in non-malignant breast cells and a panel of TNBC cell lines. Bioinformatic analysis of patient samples demonstrated that Banf1 is overexpressed in all breast cancer stages and subtypes. Banf1 depletion inhibited proliferation and induced mitotic arrest in TNBC cells via loss of nuclear envelope integrity and aberrant nuclear morphology, inducing TNBC tumour cell-specific cell death. These findings highlight the significant overexpression and functional involvement of Banf1 in TNBC progression and suggests that it may have potential as a novel anti-cancer target, supporting further investigation. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10346-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd1f5909e830afa31cad48c33f6b5d6c0d1eeae" target='_blank'>
              The inner nuclear membrane protein, Banf1, has an essential role in triple negative breast cancer cell proliferation and survival
              </a>
            </td>
          <td>
            M. Rose, J. Burgess, Chee Man Cheong, Iain Richard, Amila Suraweera, M. Adams, Pascal H. G. Duijf, Kenneth J. O’Byrne, Derek J. Richard, E. Bolderson
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687af5811b08e28febe794844c74138cc2e493e9" target='_blank'>
              The PARP1-EXD2 axis orchestrates R-loop resolution to safeguard genome stability.
              </a>
            </td>
          <td>
            Zhaoshuang Li, Yu Liu, Yuanhui Liu, Yiting Zhang, M. Huen, Huasong Lu, Zhigang Zhang, Jianwei Zhou, Dong Fang, Ting Liu, Jun Huang
          </td>
          <td>2025-06-27</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Natural transformation drives the spread of antibiotic resistance among bacteria. The DNA receptor ComEA is essential for transporting external transforming DNA into the periplasm by an unknown mechanism. Here, single-molecule optical tweezers and electron microscopy approaches show that Geobacillus stearothermophilus ComEA forms dynamic oligomers on DNA that can switch between two conformations depending on local concentration. When ComEA sparsely decorates DNA, it forms bridging oligomers that condense the DNA to generate sub-pN pulling forces. When ComEA more fully decorates DNA, it forms non-bridging oligomers that decondense DNA and cannot generate force. Mutating ComEA to favor either bridging or non-bridging conformations causes transformation deficiency in Bacillus subtilis , meaning condensation and decondensation are separately essential. Our results show that ComEA reversibly condenses DNA during natural transformation, first producing force to pull DNA into the periplasm and then abating force production to promote transport into the cytoplasm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71c17ea6f14ca131f9682284c6a60466ccc4f05" target='_blank'>
              Reversible DNA condensation drives natural transformation.
              </a>
            </td>
          <td>
            Joshua I. Santiago, Ishtiyaq Ahmed, J. Hahn, Abigail Rubino, David Dubnau, M. B. Neiditch, Keith J. Mickolajczyk
          </td>
          <td>2025-07-29</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BACKGROUND
Efferocytosis (ER) plays a crucial role in the programmed clearance of dead cells, a process that is mediated by phagocytic immune cells. However, further exploration is needed to determine the full extent of its impact on the progression of pancreatic ductal adenocarcinoma (PDAC), particularly through interactions among tumor cells, stromal cells, and immune cells within the tumor microenvironment (TME).


METHODOLOGY AND RESULTS
In this study, we comprehensively analyzed the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) database, as well as additional databases from multiple bioinformatics websites, utilizing 167 ER features derived from the integration of single-cell RNA sequencing (scRNA-seq) and bulk transcriptomic data. A set of 14 ER-associated prognostic signatures, referred to as the "14-gene panel" genes, was identified based on overall survival (OS)/disease-free survival (DFS) data, Pearson correlation coefficients, and multivariate Cox regression analyses. The model pathways enriched by the four-gene combination represented by "LEAF" and the 14-gene combination represented by the "14-gene panel" presented a high degree of similarity, including among the adhesion, mitotic, G2/M checkpoint, and epithelial‒mesenchymal transition (EMT) signaling pathways. Least absolute shrinkage and selection operator (LASSO) regression was subsequently employed to construct an ER risk scoring system using deep learning, based on the following formula: LGALS3, EMP1, ASPH, and FNDC3B, collectively termed the "LEAF" panel. Additionally, random survival forest (RSF) algorithms facilitated the identification of a key panel of genes, designated "LEAP" genes, including LGALS3, EREG, ASPH, and PLS3; three of which genes (ASPH, LGALS3, and EREG) were identified as key factors influencing the behaviors of PDAC tumors, tumor-associated stroma, and macrophages. Finally, we utilized experimental methods, including Boyden chamber analyses, immunohistochemical staining, and cell cycle analyses, to demonstrate that interference with ASPH suppresses the malignant properties of tumors, including proliferation and migration. Multiplex immunofluorescence staining was employed to identify EREG as highly relevant to the M2 macrophage subpopulation.


CONCLUSION
Our findings underscore the importance of considering a novel prognostic signature comprising 14 ER genes in the context of the TME when investigating the biology of PDAC. Future studies may explore how modulating these interactions could lead to novel therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c81897cb1c769e789b52358a323bc255ac9a02" target='_blank'>
              A Novel 14-Gene Panel Associated With Efferocytosis for Predicting Pancreatic Cancer Prognosis Through Bulk and Single-Cell Databases.
              </a>
            </td>
          <td>
            Yaheng Wu, Lin Zhao, Dingyan Yi, Zhihua Tian, Bin Dong, Chunxiang Ye, Jingtao Liu, Huachong Ma, Wei Zhao
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has recently become a focus of cardiovascular disease (CVD) research as it is possibly involved in disease mechanisms and may function as a biomarker. The literature suggests that eccDNAs may modify gene expression patterns and contribute to the pathogenesis of diseases such as pulmonary arterial hypertension (PAH). Many eccDNAs have been reported to be upregulated in PAH patients with promisingly high diagnostic sensitivity and specificity values. The advent of eccDNA detection from plasma samples using high‐throughput sequencing technologies has opened new avenues for non‐invasive diagnosis. In this review, we focus on the role of eccDNA in CVDs, concentrating on its prospects for use as a biomarker of disease development and stage. However, further studies should be conducted to explicate the functional aspects of eccDNAs in cardiovascular health and disease. This work may eventually lead to the development of novel treatment options and improved clinical outcomes for CVD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903ec3642b6c2ce94d9db06e20223312c1b439d8" target='_blank'>
              Extrachromosomal Circular DNA: Emerging Insights Into Cardiovascular Disease Mechanisms and Biomarker Potential
              </a>
            </td>
          <td>
            Saim Mahmood Khan, Jawairya Muhammad Hussain, Surraiya Riaz Mahmood Khan, Bushra Nadeem, Taniyat Naseem, Hammad Jawed, Aina Lashari
          </td>
          <td>2025-06-29</td>
          <td>iLABMED</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Female cancers such as breast and gynaecological cancers contribute to a significant global health burden and are a leading cause of fatality among women. With current treatment options often limited by resistance to cytotoxic drugs, side effects and lack of specificity to the cancer, there is a pressing need for alternative treatments. Recent research has highlighted the promising role of non-coding RNAs (ncRNA) in regulating these issues and providing more targeted approaches to suppressing key cancer pathways. This review explores the involvement of the various types of non-coding RNAs in regulating key oncogenic pathways, namely, the MAPK, PI3K/Akt/mTOR, Wnt/β-catenin and p53 pathways, in a range of female cancers such as breast, cervical, ovarian and endometrial cancers. Evidence from a multitude of studies suggests that non-coding RNAs function as double-edged swords, serving as both oncogenes and tumour suppressors, depending on their expression and cellular interactions. By mapping and investigating these regulatory interactions, this review demonstrates the complexity and dual functionality of ncRNAs in cancer. Understanding these complex mechanisms is essential for the development of new and effective ncRNA-based diagnostic methods and targeted therapies in female cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edcace8dce2041819a34f6be3b0322cc08e2907" target='_blank'>
              The Role of Non-Coding RNAs in the Regulation of Oncogenic Pathways in Breast and Gynaecological Cancers
              </a>
            </td>
          <td>
            Ammar Ansari, Aleksandra Szczesnowska, Natalia Haddad, A. Elbediwy, Nadine Wehida
          </td>
          <td>2025-08-06</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Human topoisomerase II alpha (TOP2α) resolves DNA intertwines between sister chromatids during mitosis. How cohesin, an SMC complex that holds sister chromatids, affects TOP2α decatenation is unclear. To addres this, we developed a quadruple-trap optical tweezers assay to create DNA braids and study TOP2α decatenation at the single-molecule level in real-time. We show that TOP2α resolves both single and multiple braids but becomes inefficient at forces exceeding 28 pN. TOP2α is sensitive to DNA geometry, exhibiting a chiral preference for right-handed braid crossings, and requires loading directly at DNA crossovers to act. Pre-loading TOP2α onto individual DNA strands before braid formation, in the presence of ATP, prevents decatenation. Finally, we show that human cohesin, but not condensin I, binds stably to DNA braids and blocks TOP2α activity. Our study provides novel insights into the role of substrate accessibility in regulating TOP2α‘s activity and highlights cohesin as a barrier to decatenation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7fc5b97ef55e14d8e05557d238a795889fb56aa" target='_blank'>
              Substrate accessibility regulation of human TopIIa decatenation by cohesin
              </a>
            </td>
          <td>
            E. Cutts, Sanjana Saravanan, Paul Girvan, Benjamin Ambrose, Gemma L M Fisher, David S. Rueda, Luis Aragon
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Prostate cancer(PCa) ranks among the most frequently diagnosed malignancies in men. The progression and heterogeneity of tumors pose significant challenges to clinical prognosis and treatment strategies. Recently, extrachromosomal DNA(ecDNA) has emerged as a critical player in cancer biology, influencing tumor progression, metastasis, and resistance to therapy. Oncogenes and regulatory sequences carried on ecDNA(ecDNA genes) can significantly alter the biological characteristics of tumors and their clinical outcomes. Methods In this study, we obtained ecDNA genes specifically expressed in PCa from the ECGA database. To construct a prognostic risk model for Biochemical Recurrence-Free Survival (BRFS), the two most common types of ecDNA genes which are protein-coding genes and long non-coding RNAs, were analyzed using Cox regression and LASSO regression techniques. Through KEGG/GO pathway enrichment analysis, we identified relevant pathways and analyzed the immune cell infiltration status. Functional assays, such as colony formation, CCK-8, migration, and invasion assays, were employed to assess the cellular functions of a key lncRNA AC016394.2. Results Our analysis identified six key ecDNA lncRNAs(ec-lncRNAs), including the ec-lncRNA AC016394.2, with significant prognostic value in PCa. By employing our risk scoring model, patients were classified into high-risk and low-risk groups, revealing significant differences in their BRFS outcomes. The model demonstrated strong predictive accuracy and clinical relevance. The 1/3/5-year AUC of the model is close to 0.8, which is higher than most common clinical indicators such as Gleason score and TM staging. KEGG and GO pathway enrichment analyses revealed that the high-risk group was predominantly enriched in immune-related pathways. Additionally, immune cell infiltration analysis demonstrated notable differences in the distribution of specific immune cell populations between the high-risk and low-risk groups. Knockdown of AC016394.2 inhibited PCa cell proliferation, migration, and invasion. Conclusions This study presents a novel ecDNA gene-based prognostic risk model for PCa, highlighting the functional importance of ec-lncRNA AC016394.2. These findings offer valuable insights into the biological role of ec-lncRNAs, highlighting their potential as targets for precision oncology and therapeutic intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03886-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ee0338b68e0a302760ee35e3fa258d092e1aae" target='_blank'>
              An EcDNA gene-based risk model and functional verification of a key ec-lncRNA AC016394.2 for prostate cancer
              </a>
            </td>
          <td>
            Jiangping Qiu, Jiang Wu, Nan Zhou, Cong Lai, Xin Huang, Cheng Liu, Xiaoqing Yuan, Kewei Xu
          </td>
          <td>2025-07-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are critical regulators in breast cancer (BC) progression, particularly through their interactions with the Wnt/β-catenin signaling pathway. This comprehensive review elucidates the regulatory roles of key circRNAs, including circABCC4, circFAT1, circARL8B, and circDONSON, in modulating BC behavior. These circRNAs primarily function as microRNA sponges, influencing essential processes such as proliferation, metastasis, drug resistance, and cell survival. For instance, circDONSON promotes tumor growth and radioresistance through SOX4-mediated Wnt signaling, whereas circRNF10 exhibits tumor-suppressive properties. The dual role of circRNAs as oncogenes or tumor suppressors highlights their complexity. Their high stability and tissue-specific expression patterns position them as promising diagnostic biomarkers and therapeutic targets. Dysregulation of circRNAs modulates Wnt/β-catenin signaling, a key driver of BC progression, promoting oncogenesis and therapeutic resistance. This review synthesizes evidence from peer-reviewed literature, emphasizing the molecular mechanisms and clinical implications of circRNA–Wnt interactions. By exploring these intricate networks, we identify novel opportunities for targeted BC therapies, underscoring the potential of circRNAs to transform diagnosis and treatment. Future research should prioritize standardizing circRNA quantification and validating findings across diverse patient cohorts to enhance clinical applicability. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3275f38a09ca245096add341cf2a4e561cb8b14" target='_blank'>
              Regulatory roles of circular RNAs in Wnt and other oncogenic signaling pathways in breast cancer progression: a comprehensive review
              </a>
            </td>
          <td>
            Amirreza Khalaji, Yousef Nazari, Mojtaba Pandeh, Aram Farhoudian, Leila Ghorbi, Pedram Naderi, Elham Mohebi Janagard, Samira Amin Afshari, Reza Morovatshoar
          </td>
          <td>2025-08-14</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e559b4aa95ac215d2b0dba18f35a9867916e3c3" target='_blank'>
              ADH5/ALDH2 dehydrogenases and DNA polymerase theta protect normal and malignant hematopoietic cells from formaldehyde challenge: therapeutic implications.
              </a>
            </td>
          <td>
            Jessica Atkins, Anna-Mariya Kukuyan, Monika M. Toma, M. Drzewiecka, Umeshkumar Vekariya, Adam Karami, M. Nieborowska-Skorska, R. Nejati, E. Hadzijusufovic, P. Valent, Tomasz Stokłosa, Tomasz Sliwinski, Mariusz A. Wasik, Tomasz Skorski
          </td>
          <td>2025-07-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from archived, formalin-fixed, paraffin-embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed several oncogenic ligand-encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Here we present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the National Institute of General Medical Sciences Human Genetic Cell Repository (NIGMS Repository). This resource includes five iPSCs and matched parental cell lines with accompanying publicly available, HiFi whole-genome sequencing data. Structural variant (SV) and single nucleotide variant (SNV) concordance between iPSC and parental lines was generally high; however, we found a notable reduction in concordance between the iPSC reprogrammed with retroviral reprogramming and its parental line consistent with previous work showing newer Sendai approaches to be more robust in preserving genomic integrity. This iPSC resource additionally includes pharmacogenomic and human leukocyte antigen (HLA) gene annotations as well as a set of user-friendly, web-based search tools to visualize and explore SVs and SNVs. This new resource is designed to offer a highly characterized set of in vitro models for research into cell-type specific functional characterization of genetic, genomic and pharmacogenomic variation. More generally, these renewable biospecimens and genomic data search tools are available to the scientific community to support high-quality and reproducible biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/664bbdaaf704bf6f26d8a71b37d0bd99ff2c3655" target='_blank'>
              New iPSC resource with long-read whole genome sequencing characterizations for enhanced in vitro modeling
              </a>
            </td>
          <td>
            Laura Scheinfeldt, Anthony Pompetti, G. Calendo, Tatyana Pozner, Christine Grandizio, Gretchen Smith, Kelly Hodges, Neda Gharani, D. Kusic, Matthew W Mitchell, N. Turan
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Systems that perform continuous hypermutation of designated genes without compromising the integrity of the host genome can substantially accelerate the evolution of new or enhanced protein functions. We describe an orthogonal DNA replication system in Escherichia coli based on the controlled expression of the replisome of bacteriophage T7 (T7-ORACLE). The system replicates circular plasmids that enable high transformation efficiencies and seamless integration into standard molecular biology workflows. Engineering of T7 DNA polymerase yielded variant proteins with mutation rates of 1.7 × 10-5 substitutions per base in vivo-100,000-fold above the genomic mutation rate. We demonstrated continuous evolution using the T7 replisome by expanding the substrate scope of TEM-1 β-lactamase and increasing activity 5000-fold against clinically relevant monobactam and cephalosporin antibiotics in less than 1 week.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed28e398210a42a2a56f735d7fe9f761f9cf0a6" target='_blank'>
              An orthogonal T7 replisome for continuous hypermutation and accelerated evolution in E. coli.
              </a>
            </td>
          <td>
            Christian S Diercks, Philipp Sondermann, Cynthia Rong, Thomas G Gillis, Yahui Ban, Celine Wang, David A. Dik, Peter G. Schultz
          </td>
          <td>2025-08-07</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer remains one of the leading causes of cancer-related morbidity and mortality among women worldwide. While mutations in BRCA1 and BRCA2 have long been recognized as key drivers of hereditary breast cancer, recent advances in genomic research have uncovered a growing set of additional genes such as PALB2, CHEK2, ATM, FANCM, and RAD51C—that contribute to breast cancer susceptibility. These genes are involved in critical molecular pathways, including homologous recombination DNA repair, cell cycle checkpoint control, apoptosis, and tumor suppression. Dysfunction in these pathways, often triggered by germline mutations, can lead to genomic instability, uncontrolled proliferation, and tumorigenesis. This review provides a comprehensive analysis of these emerging genetic factors, detailing their mechanisms of action, pathway interactions (e.g., BRCA–PALB2–RAD51 axis, ATM–CHEK2–p53 checkpoint signaling), and their impact on both familial and sporadic forms of breast cancer. We also discuss the clinical relevance of integrating multi-gene panels and polygenic risk scores (PRS) into personalized risk assessment and prevention strategies. By highlighting the molecular pathways disrupted by these gene mutations, we emphasize the importance of expanding genetic testing beyond BRCA1/2. A deeper understanding of these pathways not only refines risk stratification but also opens new avenues for targeted therapy, early detection, and tailored clinical management of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d949c73ab2411bd172c49fda9d693c1d37e0094" target='_blank'>
              Unraveling emerging genes and their functional pathways in breast cancer susceptibility and progression
              </a>
            </td>
          <td>

          </td>
          <td>2025-07-07</td>
          <td>International Journal of Biosciences (IJB)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT R2 elements serve as a class of non-long terminal repeat (non-LTR) retrotransposons found in animal genomes that specifically insert into the ribosomal DNA (rDNA) sequences of host genomes. Each R2 element contains a single open reading frame (ORF) encoding a multifunctional protein with nucleic acid-binding, reverse transcriptase, and endonuclease activities, enabling specific genomic integration via a mechanism called target-primed reverse transcription (TPRT). As a classical model for studying retrotransposition mechanisms, R2 elements possess unique biological properties and precise integration capabilities, which have inspired novel genome engineering strategies. In this review, we summarize the components and integration mechanisms of R2 retrotransposons and highlight the recent advances in employing these mobile elements for targeted gene integration. Finally, we present future directions for the utilization of R2 retrotransposons as novel biotechnological tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99dcaf5091c4dee3c3649a8044b57944d6b999" target='_blank'>
              Biology and utilization of R2 retrotransposons
              </a>
            </td>
          <td>
            Shengqiu Luo, Qicheng Chen, Yangcan Chen, Wei Li
          </td>
          <td>2025-06-25</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>