<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-29 09:38:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>43</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>90</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>123</td>
          <td>147</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>395</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>341</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres, stochastically segregate during cell division3–6 and persist over many generations. It has been more than 40 years since ecDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei, but the mechanism underlying this process remains unclear3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. Using Retain-seq, a genome-scale assay that we developed, we reveal thousands of human retention elements that confer generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromosome conformation capture show that retention elements physically interact with mitotic chromosomes at regions that are mitotically bookmarked by transcription factors and chromatin proteins. This activity intermolecularly recapitulates promoter–enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated. Targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, which suggests that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, and reveal the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D.
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, Jessica Alsing, J. Luebeck, Maria E. Stefanova, Owen S Chapman, Katerina Kraft, Shu Zhang, Jun Yi Stanley Lim, Yipeng Xie, Yoon Jung Kim, Sihan Wu, Lukas Chavez, Guy Nir, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract In this Perspective article, we follow the journey of a gene that breaks free from its stringent chromosomal transmission dynamics to become an independently multiplying agent on so-called extrachromosomal circular DNA (ecDNA/eccDNA). We discuss how the release of a gene from its chromosomal anchor can affect its dosage, regulatory context, and potential evolutionary trajectory before examining the phenotypic implications for unicellular and multicellular eukaryotes that transmit genes on circular DNA. We also briefly explore the fundamental difference between circularized genes in flowering plants (angiosperms) and mammals (primarily cancer) concerning somatic and germline inheritance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="


 Understanding of the heterogeneity in the formation of the Ph has evolved as technology has advanced. In the pre-TKI era, FISH revealed derivative 9 deletions that were associated with shorter survival. The poor outcomes were overcome with imatinib. Whole exome and transcriptome sequencing revealed greater complexity: inversions; novel 9;22 fusions; sequence fragmentation; and random reassembly. These were termed Ph-associated rearrangements. They occurred on the Ph and/or the derivative 9, were thought to occur at the same time as the Ph and correlated with poorer response to imatinib.



 To use whole genome sequencing (WGS) to examine derivative chromosomes and further resolve the complexity of Ph-associated rearrangements in patients at diagnosis and pan-TKI resistant cell lines.



 WGS was performed for 7 chronic phase CML patients at diagnosis and 7 matched sensitive/resistant Ph-positive cell lines. Resistance was induced by exposure to increasing concentrations of imatinib. Data was analyzed using Oncoanalyzer. Derivative chromosome structure was resolved using the LINX algorithm, which clusters structural variants that are highly unlikely to occur independently. Rearrangements were validated using orthogonal methods, including Optical Genome Mapping and SNP Array. Six of 7 patients had pre-characterized Ph-associated fusions identified using RNAseq.



 For most patients, a balanced reciprocal Ph translocation involves 2 rearrangements: 1 that generates the Ph and BCR::ABL1; and 1 reciprocal derivative 9 with ABL1::BCR. These structures were resolved for 1 patient without pre-characterized Ph-associated rearrangements and for 5 of 7 parental cell lines.
 A distinct class of complex structural rearrangement called chromoplexy was identified involving the Ph translocation in 11 samples. These Ph-associated rearrangements had the highest number of linked structural variants within the individual genomes of 10 of the 11 samples. Chromoplexy involves the formation of extensive chained rearrangements affecting multiple chromosomes that create an intricate series of inter/intra-chromosomal fusions, deletions, duplications and inversions. Many DNA strands were broken and ligated in new configurations in the affected samples. Multiple genes on the derivative chromosomes were disrupted, including recurrent cancer drivers.
 For 3 of 7 matched sensitive/resistant cell lines, Ph-associated rearrangements were acquired in the resistant lines. The most complex events occurred in the LAMA84 resistant line. The parallel sensitive line had the standard 2 rearrangements. However, the resistant line gained 17 rearrangements with highly amplified Ph sequences, inversions and novel genomic fusions. Remarkably, small fragments from 6 chromosomes (2, 4, 10, 13, 17, X) were inserted upstream and downstream of BCR::ABL1, and disrupted the NUP214 driver gene.
 The K562 sensitive line displayed 24 rearrangements with non-contiguous amplified Ph regions of different copy number. Events included NOTCH1 deletion, 9;22 NUP214 fusion and an adjacent 9;13 fusion. Six fragments from chromosome 13 were incorporated into the Ph. The K562 resistant line displayed further evolution with 31 Ph rearrangements.
 The AR230 sensitive line displayed Ph chromoplexy with 9 rearrangements. The resistant line evolved with 19 additional Ph rearrangements. A fragment from chromosome 21 was inserted upstream of BCR::ABL1 and disrupted the MAPK1 driver gene. The LZTR1 tumor suppressor was deleted. All other cell lines maintained the standard Ph configuration.
 Ph-associated chromoplexy was evident in 6 patient samples: average 10 rearrangements. The patient with the highest number (15) progressed to blast phase at 4 months of imatinib. Four Ph-associated subclones were detected at diagnosis with non-contiguous amplified regions, consistent with the evolved cell lines. Furthermore, the LZTR1 tumor suppressor gene was deleted, as detected in AR230. The blast phase sample was unavailable. Deep molecular response was achieved in 1 patient on imatinib and 4 on frontline or switch to nilotinib/dasatinib.



 We have demonstrated that Ph chromoplectic rearrangement can be present at diagnosis but can also be initiated during leukemic progression and disrupt multiple genes, including cancer drivers. The Ph may be prone to extensive DNA breakage. Ph chromoplexy could constitute a new mechanism that contributes to acquired TKI resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed250c9c3a664955ee35598f9870a47e6c84f2b" target='_blank'>
              Whole genome sequencing reveals that the Philadelphia chromosome (Ph) can evolve and gain complex chromoplectic sequence rearrangements associated with tyrosine kinase inhibitor (TKI) resistance
              </a>
            </td>
          <td>
            S. Branford, C. Wadham, N. Shahrin, V. Saunders, Rosalie R. Kenyon, Ming Lin, Dominic Kaczorowski, Muneeza Maqsood, Hamish Scott, John Toubia, Naranie Shanmuganathan, C. Kok, Timothy Hughes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Genomic analysis of multiple myeloma has advanced greatly in recent years, but traditional short-read sequencing has limited the power to examine complex or linked events across the genome. Linking markers together into large phase-blocks, and examining the interaction of different sequencing modalities, allows complex genomic and epigenomic states to be integrated to generate true multiomic haplotypes at the chromosomal level.



 Fourteen multiple myeloma patient-derived xenografts were used to generate multiomic data including short- (Illumina) and long-read (PacBio) whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities, structural variation (SV), and DNA methylation, as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) and HiChIP (Dovetail, H3K27ac mark) to identify 3D chromatin architecture/folding and enhancer-target gene interactions. Micro-C/LinkPrep interaction reads were also used for chromosome-scale haplotype phasing, onto which SNVs, SVs, CNVs, DNA methylation, expression, and chromatin marks were super-imposed based on heterozygous SNPs. Data integration resulted in haplotype-phased somatic events for genomic architecture, SNVs, SVs, epigenomic states, and expression markers.



 For the first time, complete haplotype-resolved assemblies at the chromosomal level have been generated in multiple myeloma. A 100x coverage genome Micro-C/LinkPrep libraries we were able to phase the long and short arms of each chromosome, resulting in an average 96.3% haplotype phasing per autosome – thereby generating chromosome length haplotypes up to 241.9 Mb in length. In comparison, HiFi long-read (15 kb reads) WGS (PacBio) was able to generate phase-blocks of up to 4.6 Mb (median 44.1 kb).
 We examined the prevalence of somatic mutations on each haplotype across all chromosomes to determine if one parental haplotype was more likely to be mutated than the other. In one sample, of 19,306 mutations 56% were phased to their respective haplotypes and were generally equally distributed across both haplotypes.
 Haplotype-specific interaction heatmaps were generated allowing us to examine the interaction of complex SVs across chromosomes. In one sample, we identified a primary t(11;14) and additional SV events linked to the t(11;14) including a t(3;14), t(11;17), and t(3;17) which created a cyclical pattern. Using the chromosome-scale haplotype maps we were able to determine the proportion of interacting reads for the four possible haplotype combinations between any two chromosomes. We found that 70-83.5% of reads support specific combinations of haplotype interactions, and that all four SV events were linked and involved the same haplotypes on each chromosome. The pattern of interactions combined with breakpoint analysis in this case indicated a four-way complex reciprocal translocation between chromosomes 3,11,14 and 17.
 Integration of epigenetic data in this complex SV, including DNA methylation (HiFi reads) and super-enhancer histone marks (CUT&Tag-IT), showed DNA hyper-methylation 17 kb upstream of CCND1 next to the t(11;14) breakpoint as well as increased H3K27ac marks on the same haplotype, indicating spreading of the activating broad domain from the IGH super-enhancer on chromosome 14. Equally, the hypomethylated DNA marks at the IGH promoter are spread to chromosome 3, via the t(3;14), resulting in over-expression of the proto-oncogene SKIL. The same is true for the t(3;17), which shows hypomethylation on both sides of the breakpoint, compared to the non-translocated allele. Overall, this exemplifies the underlying intricacy and impact of complex SVs across multiple linked chromosomes and their epigenetic states.



 We have generated the first chromosome scale haplotype-resolved genomes in multiple myeloma and integrated them with epigenetic states. We have shown that it is possible to identify interactions across chromosomes to resolve complex SVs as well as their epigenomic consequences to understand the intricate nature of how the genome is organized.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e6488520a62edae9da4095a747986d863374bdb" target='_blank'>
              Multiomic evidence of coordinated complex rearrangements, enhancer hijacking, and epigenomic signatures in the first whole-chromosome-phased myeloma genomes
              </a>
            </td>
          <td>
            Nathan Becker, Enze Liu, Zackary Sanborn, A. Suvannasankha, Kelvin Lee, D. Kazandjian, James Hoffman, B. Diamond, Abhishek Pandey, R. Abonour, O. Landgren, Lisa Munding, Aneta Mikulasova, Brian A. Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) fragments in the plasma capture cellular nucleosomal profiles since nucleosome-protected regions escape enzymatic degradation while nucleosome-depleted regions can not. We developed cfOncoXpress, a machine learning framework that uses fragmentation patterns to predict oncogene expression from cfDNA WGS. cfOncoXpress incorporates gene copy number aberrations inferred from cfDNA, including those associated with extrachromosomal DNA. Its application in prostate and breast cancers shows it can predict tumor subtype based on expression of signature genes and activated pathways. cfOncoXpress shows superior performance relative to other state-of-the-art methods and can be used to predict tumor gene expression when tissue biopsies are infeasible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd03a827003a113eb12bee1a6a13f9987699c5c0" target='_blank'>
              cfOncoXpress: Tumor gene expression prediction from cell-free DNA whole-genome sequences
              </a>
            </td>
          <td>
            Weiling Li, M. Roskes, A. Martínez-Fundichely, Ekta Khurana
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Simple Summary This study investigates the genomic alterations of cleavage and polyadenylation factors and their clinical relevance in cancer. We show that mutations in CPA genes may not be clinically relevant. However, copy number variations, particularly amplification of CPSF1, demonstrate significant impact on patient prognosis. We show that CPSF1 amplification, although co-occurring with MYC amplification, can independently change alternative polyadenylation (APA) patterns and dysregulate cancer-associated gene signatures. These results highlight the crucial impact of CPA gene copy number variation on biological processes and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Background: Chromothripsis, a catastrophic event of chromosomal shattering and reassembly, is a key driver of oncogenesis and is associated with poor prognosis in multiple myeloma (MM). However, the specific functional consequences and the molecular mechanisms by which it rewires the MM regulatory landscape remained largely unknown.
 Methods: Using previous definitions, we identified chromothripsis (≥10 breakpoints, ≥7 CN oscillations, ≥200 kb segment sizes), chromothripsis-like (5-9 breakpoints and 3-6 CN oscillation) and non-chromothripsis (WT) samples. We utilized multiomic data from the CoMMpass dataset (n=1,066) and an independent Indiana University (IU) dataset (n=134), as well as 14 patient-derived xenograft (PDX) models with short-/long-read whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities and structural variation (SV), as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) to identify 3D chromatin architecture/folding inside the cell and enhancer-target interactions.
 Results: We identified 417 chromothripsis events in 268 patients (25%) from the CoMMpass dataset. Events were identified more frequently in relapsed and refractory (RRMM) samples compared to newly diagnosed (NDMM) samples (30% vs. 22%), suggesting an association between chromothripsis and disease progression. The number of events found in chromosomes highly correlated with chromosome sizes (r=0.63, Pearson correlation). However, events were particularly enriched on chromosomes 1, and 17 (0.26 and 0.30 events per million bps). Breakpoints of these events were enriched on 17q and 1p (odds ratio=2.2 and 1.7, p=0.0003 and 0.001). Chromothriptic samples were associated with inferior progression-free (PFS; 970 vs. 1227 days, p=0.003) and overall survival (2176 vs. not reached, p=0.006) compared to WT. In chromothriptic samples, 34% exhibited breakpoints on >1 chromosome and an higher number of affected chromosomes correlated with worse PFS (≥3 vs. 2, p=0.03; ≥3 vs. 1, p=0.02). Meanwhile, CN oscillations plus breakpoints was an independent factor associated with inferior PFS(p=0.05,: 700 vs. 1358 days). Del(TENT5C), del(TP53)and t(4;14) were enriched in the chromothripsis group (32%, 18% and 20%, odds ratio=2.6, 3.4 and 2.4, p<0.001) and are associated with genomic instability. We applied the same approach and further identified 19 chromothriptic samples from the IU dataset, where t(4;14) and del(TP53) were also enriched (odds ratio=2.6 and inf.) We identified differentially expressed genes (DEGs) between the chromothripsis group and a subset of WT group with balanced frequencies of high-risk events (e.g. t(4;14) & del(TP53)) to generate a chromothripsis signature. Pathway analysis on 37 consistent DEGs between the two datasets identified ‘G2/M checkpoint’ and ‘interferon α/γ responses‘(p<0.05), suggesting selection for a growth advantage and activation of the cGAS/STING pathway. A neuron network model for predicting chromothripsis was trained with DEGs and reached performance of AUC=0.90 and AUPR=0.87.
 We utilized multi-omics data from established PDX models to further examine the functional consequences of chromothripsis events. Among 9 events detected in 7 samples, 5 events exhibited significantly enriched APOBEC mutational signature and kataegis within events. Copy number change alone did not account for changes in gene expression within the chromothripsis regions, with 71% of genes dysregulated through neo-TAD formation and rewiring of the epigenome, specifically through juxtaposition of super-enhancers and allele-specific expression associated with DNA methylation changes. Key dysregulated genes included MYCN (Log2FC=6.3), the tumor suppressor PMAIP1 (Log2FC=-4.9), and cell adhesion proteins (EPCAM/CD326, Log2FC=3.5) that were rarely expressed in other MM samples.
 Conclusion: We revealed novel molecular characteristics of chromothripsis and the mechanisms caused it. We determined that the number of events and CN abnormalities are associated with poor outcome and that epigenetic states are reprogrammed within the events. These findings advance the clinical risk stratification and reveal unique therapeutic vulnerabilities tied to chromothripsis-associated epigenetic dysregulation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eeb625eee780834dfe72e26b192d78e2662c5ec" target='_blank'>
              Chromothripsis drives high-risk multiple myeloma by copy number abnormalities and rewiring the epigenome to activate oncogenic programs.
              </a>
            </td>
          <td>
            Enze Liu, Nathan Becker, Sreejeta Mondal, A. Suvannasankha, D. Coffey, B. Diamond, Abhishek Pandey, Kelvin Lee, D. Kazandjian, R. Abonour, Aneta Mikulasova, G. Morgan, O. Landgren, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has emerged as a critical determinant of poor clinical outcomes and immune escape in tumors, but the high cost and technical complexity of current detecting techniques limit its broader investigation in cancer immunotherapy. Leveraging the combined machine learning algorithms including the least absolute shrinkage and selection operator (LASSO) regression, RandomForest (RF) and Recursive Feature Elimination (RFE), we developed a 12-gene transcriptomic score (EC_score) to predict the existence of ecDNA through RNA-seq. EC_score demonstrated reliable predictive performance in two independent cohorts (AUC > 0.70), validated by fluorescence in situ hybridization (FISH) in both cell lines and clinical samples. Next, we found that EC_score emerged as an independent adverse prognostic factor across multiple immunotherapy cohorts. Notably, high EC_score correlated with cell cycle activation and immunosuppression, characterized by reduced lymphocytes infiltration and upregulated immunosuppressive markers, including MHC molecules, co-inhibitory immune checkpoints and TGF-β signals. In general, we established and validated a 12-gene signature (EC_score) derived from RNA-seq, offering a novel computational tool for predicting the presence of extrachromosomal circular DNA and stratifying cancer immunotherapy response. Graphical abstract Overview of the analytical framework integrating multi-omics data, machine learning, and experimental validation to establish EC_score and characterize its clinical implications. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04056-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97428a7d98b9f7fbbd3050ae63fe8f2b1c83fe29" target='_blank'>
              Machine learning identified extrachromosomal DNA-related 12 gene signatures to predict cancer immunotherapy response
              </a>
            </td>
          <td>
            Yan Ju, Jingwei Zhang, Jiaming Deng, Xingyuan Wu, Haotian Yang, Changyu Tao, Xiao Li
          </td>
          <td>2025-12-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83420d91db40081aef405ab1feed30e61b3d080" target='_blank'>
              Monoallelic expression characterizes a distinct molecular and clinical group of breast tumors
              </a>
            </td>
          <td>
            Mona Arabzadeh, Amartya Singh, Kyle Payne, Shridar Ganesan, Hossein Khiabanian
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Motivation Tumor viruses account for ∼10% of cancer diagnoses. Virally induced tumorigenesis is understood as direct signaling through oncogenes such as E6 and E7 genes in the case of human papillomavirus. Furthermore, pathogen characteristics such as viral oncogene dose may impact the disease course. To our knowledge, no tool has been proposed to assess the intra-viral copy number alterations that define the gene dose of viral oncogenes and associated suppressive pathways native to the pathogen’s normal life cycle. Results We propose an R package, “ELViS,” that analyzes viral copy number changes from DNA sequencing of whole viral genomes. The method adjusts for viral load with 2D transformation and segmentation to offer the relative viral gene doses. Availability and implementation The ELViS R package is available from https://bioconductor.org/packages/ELViS. This article used controlled access data from dbGaP (phs001713.v1.p1).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee403f958df88d61608425c7ffde897d6bfc505e" target='_blank'>
              ELViS: an R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, J. Holt, Xiaobei Zhao, K. Hoadley, D. N. Hayes, Hyo Young Choi, Neil Hayes
          </td>
          <td>2025-11-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Immune system control is a principal hurdle in cancer evolution. The temporal dynamics of immune evasion remain incompletely characterized, and how immune-mediated selection interrelates with epigenome alteration is unclear. Here we infer the genome- and epigenome-driven evolutionary dynamics of tumor-immune coevolution within primary colorectal cancers (CRCs). We utilize a multiregion multiomic dataset of matched genome, transcriptome and chromatin accessibility profiling from 495 single glands (from 29 CRCs) supplemented with high-resolution spatially resolved neoantigen sequencing data and multiplexed imaging of the tumor microenvironment from 82 microbiopsies within 11 CRCs. Somatic chromatin accessibility alterations contribute to accessibility loss of antigen-presenting genes and silencing of neoantigens. Immune escape and exclusion occur at the outset of CRC formation, and later intratumoral differences in immuno-editing are negligible or exclusive to sites of invasion. Collectively, immune evasion in CRC follows a ‘Big Bang’ evolutionary pattern, whereby it is acquired close to transformation and defines subsequent cancer-immune evolution. This study nominates immune escape as an early event in colorectal cancer and shows how this can be driven through both genetic and epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Chromosomal structural changes happen when genomic stability is compromised, such as in disease or in species hybrids. In these contexts, diminished control of repetitive elements has been reported, but the reasons for this are not yet well understood. There are causal associations between repetitive elements and phenotypes such as disease progression, leading us to the hypothesis that chromosomal structure may be affected by transposable elements (TEs). In an intraspecific hybrid Drosophila melanogaster cell line (PnM), the degree of pairing among trans homologous chromosomes was affected by the presence of nearby TEs, in particular, LINE and LTR elements, such as Baggins1 or Gypsy. Chromosomal pairing was significantly lower in windows containing TEs than in windows without any TEs. Pairing was also affected by TEs in mouse, which suggests a possible general association between TEs and pairing that is highly conserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b05bf5cfcd82a8fc82e293a52e5e86b57f277d" target='_blank'>
              Haplotype-Resolved Chromatin Conformation Data Reveals Relationship Between Transposable Elements and Chromosomal Pairing
              </a>
            </td>
          <td>
            Luke K Genutis, Patrick Villanueva, Rita M. Graze, Jumana Alhaj Abed, Lauren M McIntyre, S. Nuzhdin
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277dcab08835ae1612852095fa2fa4e5142ac6cf" target='_blank'>
              Proximity of pre-mRNA 3′ end processing and transcription termination predicts enhanced gene expression
              </a>
            </td>
          <td>
            Agata Stępień, Deepshika Pulimamidi, Martyna Plens-Gałąska, Maxime Lalonde, Magda Kopczyńska, Hiroshi Kimura, S. Irshad, M. Gdula, Stephan Hamperl, Kinga Kamieniarz-Gdula
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Most animal genomes are partitioned into topologically associating domains (TADs), created by cohesin-mediated loop extrusion and defined by convergently oriented CCCTC-binding factor (CTCF) sites. The dynamics of loop extrusion and its regulation remain poorly characterized in vivo. Here we tracked the motion of TAD anchors in living human cells to visualize and quantify cohesin-dependent loop extrusion across multiple endogenous genomic regions. We show that TADs are dynamic structures whose anchors are brought in proximity about once per hour and for 6–19 min (~16% of the time). Moreover, TADs are continuously extruded by multiple cohesin complexes. Remarkably, despite strong differences in Hi-C patterns across chromatin regions, their dynamics is consistent with the same density, residence time and speed of cohesin. Our results suggest that TAD dynamics is primarily governed by the location and affinity of CTCF sites, enabling genome-wide predictive models of cohesin-dependent chromatin interactions. Live-cell fluorescence microscopy and polymer simulations in human HCT116 cells show similar cohesin density, residence times and extrusion speed across multiple genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715a0a11e407c2847c391eae8db4fee949a2d73b" target='_blank'>
              Uniform dynamics of cohesin-mediated loop extrusion in living human cells
              </a>
            </td>
          <td>
            Thomas Sabaté, Benoît Lelandais, Marie-Cécile Robert, Michael Szalay, J. Tinevez, Edouard Bertrand, Christophe Zimmer
          </td>
          <td>2025-11-14</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e47a668958565050b674aeca2b9f77b67dc49b4" target='_blank'>
              Investigation of a global mouse methylome atlas reveals subtype-specific copy number alterations in pediatric cancer models.
              </a>
            </td>
          <td>
            Melanie Schoof, T. Zheng, M. Sill, R. Imle, Alessia Cais, L. Altendorf, Alicia Fürst, Nina Hofmann, K. Ernst, Dominik Vonficht, Kenneth Chun-Ho Chan, T. Holland-Letz, Andreas Postlmayr, Ryo Shiraishi, Wanchen Wang, A. Morcavallo, Michael Spohn, C. Göbel, J. Niesen, Levke-Sophie Peter, Franck Bourdeaut, , Yanxin Pei, Najiba Murad, F. Swartling, Jessica Taylor, Monika Yadav, Garrett R Gibson, Richard J Gilbertson, M. Dottermusch, Rajanya Roy, K. Kerl, R. Glass, Jiying Cheng, M. Horstmann, G. Wolters-Eisfeld, Haotian Zhao, D. Sturm, Viveka Nand Yadav, L. Chesler, Simon Haas, William A. Weiss, P. Northcott, L. Kutscher, A. G. Guerreiro Stucklin, O. Ayrault, J. Neumann, Daisuke Kawauchi, David T. W. Jones, Kristian W Pajtler, A. Banito, S. Pfister, U. Schüller, Marc Zuckermann
          </td>
          <td>2025-12-11</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Tandemly repeated non-coding sequences, widely known as satellite DNAs (satDNAs), are extremely diverse and highly variable components of eukaryotic genomes. In recent years, advances in high-throughput sequencing and new bioinformatics platforms have enabled in-depth studies of all (or nearly all) tandem repeats in any genome (the satellitome), while a growing number of telomere-to-telomere assemblies facilitates their detailed mapping. Research performed on a large number of non-model plant and animal species changed significantly the “classical” view on these sequences, both in an organizational and functional sense, from ballast compacted in the form of heterochromatin to elements that are important for structuring the entire genome, as well as for its functions and evolution. The diversity of repeat families, and the complexity of their intraspecies and interspecies distribution patterns, posed new questions, urging for species-by-species comparative analyses. Here we integrate some basic features of different forms of sequences repeated in tandem and rapidly growing data evidencing extensive dispersal of satDNA sequences in euchromatin, their putative roles and evolutionary significance. Importantly, we also present and discuss various issues brought on by the use of new methodological approaches and point out potential threats to the analysis of satDNAs and satellitomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df02ce17404430620feb58c2c132e48c76b2721" target='_blank'>
              Satellite DNA Genomics: The Ongoing Story
              </a>
            </td>
          <td>
            M. Garrido-Ramos, Miroslav Plohl, Eva Šatović-Vukšić
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Mutations in non-coding regulatory elements are increasingly recognized as critical drivers of cancer development and progression. Among these, structural alterations such as duplications of super-enhancers (SEs), large clusters of enhancers that regulate key oncogenes, have emerged as significant contributors to oncogenesis. In T-cell acute lymphoblastic leukemia (T-ALL), a particularly aggressive hematologic malignancy, SE duplications have been identified in approximately 5% of human cases. These duplications frequently involve a non-coding region located approximately 1.3 megabases downstream of the Myc oncogene, a region known as N-Me. Despite the growing recognition of the role of SE duplications in T-ALL, functional studies to determine their molecular origins and contributions to leukemogenesis have been limited, largely due to the absence of suitable and genetically tractable preclinical models. This gap in knowledge hinders both our understanding of disease pathogenesis and our ability to design targeted therapeutic interventions.
 Here, we report the generation and characterization of a novel genetically engineered mouse model of T-ALL that spontaneously develops oncogenic SE duplications at the Myc locus. This model offers the first spontaneous in vivo system to investigate the origin, regulation, and oncogenic function of SE duplications. This model was created by conditionally stabilizing β-catenin in CD4⁺CD8⁺ double-positive (DP) thymocytes in combination with targeted deletion of the transcription factor HEB. This combination is sufficient to drive the development of aggressive T-cell leukemias that consistently harbor duplications of the N-Me super-enhancer. This finding recapitulates the genomic alteration observed in a subset of human T-ALL and provides a unique platform for mechanistic studies.
 We further demonstrate that the molecular events leading to SE duplication and oncogenic Myc activation unfold in a stepwise manner. At the preleukemic stage, stabilization of β-catenin alone is sufficient to enhance recruitment of HEB to the N-Me region. The simultaneous loss of HEB greatly increased chromatin accessibility at the N-Me enhancer. This altered chromatin landscape permits aberrant binding of transcription factors including TCF-1, which plays a critical role in T-cell development and leads to N-Me SE activation associated with significant three-dimensional chromatin reorganization. Chromosome conformation analyses in the leukemic cells reveal the formation of a novel chromatin loop that brings the activated N-Me SE into close spatial proximity with the Myc promoter, enabling oncogenic Myc transcription.
 These findings uncover a novel mechanistic pathway by which β-catenin and HEB interact to reshape the enhancer landscape and three-dimensional chromatin architecture, ultimately promoting SE duplication and aberrant Myc expression in T-ALL. β-catenin appears to play a dual role, first facilitating HEB recruitment under physiological conditions and then, upon HEB loss, permitting the reprogramming of enhancer activity through chromatin opening and ectopic transcription factor binding. Our data suggest that the cooperative interplay between signaling pathways and transcriptional regulators at enhancer elements can initiate structural genomic changes that drive malignancy.
 This work provides a long-sought preclinical model that faithfully mirrors key features of human T-ALL. Our findings open new avenues for understanding how non-coding regulatory mutations arise and exert oncogenic effects and have the potential to provide insights that may extend beyond T-ALL to other cancers driven by enhancer dysregulation. Moreover, by revealing molecular intermediates such as β-catenin, HEB, and TCF-1 in the regulation of enhancer activity, our findings identify potential therapeutic targets to disrupt the oncogenic enhancer-promoter interactions that underlie Myc-driven leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3354f2ef9255003e9c97bc93c32af47ddbd359" target='_blank'>
              Myc super-enhancer dynamics in T-ALL
              </a>
            </td>
          <td>
            Georgios Tousinas, Vasili Toliopoulos, Melissa Tracy, Akinola Emmanuel, K. Khazaie, Thomais Papamarcaki, P. L. Bergsagel, F. Gounari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="G-quadruplexes act as transcriptional triggers in oncogenes, and small molecules targeting them show anticancer potential but lack sequence specificity, risking off-target effects. Here, we introduce original fold-back triplex-forming oligonucleotides (FTFOs) as precise alternatives. Physicochemical and biological analyses revealed that FTFOs unfold the c-Myc G-quadruplex and repress oncogene regulatory element by forming noncanonical triplexes that block G-quadruplex DNA from interacting with transcription factors. Derived design rules enabled the identification of FTFOs that reduce c-Myc expression in the Del4 plasmid by up to 80%, significantly impairing cancer cell growth, confluence, mobility, and endogenous c-Myc expression. This work presents FTFOs as an innovative sequence-specific strategy for arresting G-quadruplex structures via triplex formation and suppressing oncogene activity, offering a promising avenue for targeted anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96215ef814f7004690d52622b09c1efcdd6d6e9c" target='_blank'>
              Highly Specific Inhibition of c-Myc Oncogene Expression through Shifting the G-Quadruplex toward a Triplex Structure.
              </a>
            </td>
          <td>
            Jolanta Lisowiec‐Wachnicka, Karolina Zielińska, A. Pasternak
          </td>
          <td>2025-11-12</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfde1f929fb1d5e109604b5adc00a4ea625430b8" target='_blank'>
              Localised activity of reverse gyrase at gene regulatory elements
              </a>
            </td>
          <td>
            Paul Villain, Vladislav Kuzin, Augustin Darennes-Degaugue, Florence Lorieux, Antoine Hocher, Romain Le Bars, Tobias Warnecke, Laura Baranello, Tamara Basta
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5adbde5a06ec4e198106c731d03948bb0cc143" target='_blank'>
              CAD-C reveals centromere pairing and near-perfect alignment of sister chromatids
              </a>
            </td>
          <td>
            Axel Delamarre, Maurane Reveil, David Pintor Pichon, Michael A. Boemo, I. Whitehouse
          </td>
          <td>2025-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="


 Introduction: Diffuse large B-cell lymphoma (DLBCL) is a common and aggressive B-cell malignancy with heterogeneous genomic findings, including a variety of fusions and copy number alterations. However, standardization among laboratories for using cytogenetic techniques is lacking and is often targeted, thus precluding a genome-wide analysis. Herein, we present a case of DLBCL with unresolved fluorescence in situ hybridization (FISH) findings while also providing additional genomic findings that may facilitate therapy selection.
 Case details and methods: Case details are that of a 70-year-old male with DLBCL with a non-GC phenotype. FISH revealed an IGH rearrangement to an unknown partner while MYC revealed copy number gain, likely indicating polysomy of chromosome 8. The patient was treated with polatuzumab vedotin-rituximab-cyclophosphamide, doxorubicin, and prednisone (pola-R-CHP) but subsequently progressed through therapy. To assess the IGH rearrangement and identify additional genomic events, whole-genome Hi-C sequencing was performed from 5 micron formalin-fixed, paraffin embedded (FFPE) sections and cross-linked chromatin was digested using a restriction enzyme. The 5'-overhangs were filled and digested ends were labeled with biotinylated nucleotide. Spatially proximal digested ends of DNA were ligated, which was then purified, fragmented, and enriched. DNA libraries were then prepared and sequenced by paired-end sequencing. Structural variant (SV) analysis was performed using HiCUP and hic_breakfinder. SV were then manually curated and processed to create a single VCF file.
 Results: Whole-genome Hi-C sequencing revealed a fusion of IGH to an intergenic partner on chromosome 10, resulting in a t(14;10). Copy number variant analysis revealed gains of chromosomes 4, 7, 8, and 15q with loss at 5p. Interestingly, an 18 Mb tandem amplification on chromosome 17 resulted in amplification of PRKCA, CD79B, CLTC, MSI2, and HLF.
 Conclusion: Hi-C sequencing was able to define the partner of the IGH gene rearrangements to be an intergenic region of chromosome 10, revealing that this was likely not related to the pathogenesis in this case. Furthermore, Hi-C sequencing found copy number aberrations and a tandem amplification of CD79B, a key regulator in B-cell signaling. Accordingly, this finding is associated with a more aggressive phenotype in DLBCL, as was observed in this patient. Coincidentally, the patient was already appropriately treated with pola-R-CHP as polatuzumab vedotin is an antibody-drug conjugate targeting CD79b. Herein, we demonstrated that whole-genome Hi-C sequencing is a robust technology that facilitates SV analysis on FFPE in DLBCL. Accordingly, this technique is sensitive and specific as it can refine unclear FISH findings while potentially providing additional genomic biomarkers, ultimately guiding therapy selection and/or clinical trial enrolment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a9f091bb1e0da1accf8110ac74b9a1d49c33bd6" target='_blank'>
              Clinical use of genome-wide hi-c sequencing for assessment of structural variants in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Phillip Michaels, D. Qualls, Anthony D Schmitt, Alex Hastie
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Background: Multiple Myeloma (MM) is a plasma cell malignancy characterized by genomic instability, often manifesting as chromothripsis. Although CNVs in DNA repair genes are recurrent in MM, the combined effect of germline defects and somatic alterations in driving aggressive, therapy-resistant disease remains incompletely understood.
 Methods: We performed whole-genome sequencing (WGS) on tumor DNA from a patient with newly diagnosed (NDMM), progressive, treatment-resistant multiple myeloma and CNV gains across multiple chromosomes. This patient was from the original cohort of 17 patients we reported where we detected a possible correlation of copy number gains in DDR genes and aggressive myeloma.1 Analysis included detection of SNVs, indels, CNVs, structural variants (SVs), HRD scoring, mutational signatures, and pathway enrichment.
 Results: Germline mutations were identified in key DDR genes (RAD50, BRCA1, CHD1, SEMA4D), resulting in an HR-deficient phenotype (high HRD score/ high Genomic Instability). Variants in CYP2B6, CYP2D6, CYP3A5 indicated impaired drug metabolism. Somatic mutations in TRAF3andARID5B affected immune signaling and transcription. The genome showed near-triploidy (ploidy score: 2.98), high SV burden, and dominant CNV signatures, associated with whole-genome duplication and chromothripsis (or, chromoplexy).
 Mutational signatures included SBS1, SBS5, and DBS17, indicating age-related processes, DNA repair defects, and oxidative stress. A high MATH score along with local hypermutation events reflected tumor heterogeneity. SV analysis revealed extensive deletions, duplications, and complex rearrangements, especially on chromosomes 1 and 11. Widespread Alu element duplications and Alu-mediated rearrangements suggest non-allelic homologous recombination and L1 retrotransposition events.
 Pathway analysis revealed:DNA repair: HRR and NHEJ dysfunction; BRCAness without BRCA1/2 biallelic inactivation.Cell cycle: Impaired G2/M checkpoints.Immune signaling: TRAF3 mutation disrupted cytokine and interferon pathways.Gene transcription: TP53 pathway dysregulation.Metabolism: CYP variants affected drug/xenobiotic metabolism.Signal transduction: CHD1 mutation impacted nuclear receptor signaling.Developmental biology: SEMA4D and BRCA1 affected migration and differentiation.Hemostasis: F5 mutation altered fibrin clot formation, suggesting thrombotic risk, which is a known feature of myeloma.
 Discussion: Thibaud et al.2 found pathogenic germline variants in BRCA1/2 contributing to hereditary risk in 10 percent myeloma cases. Nazaryan-Petersen et al.3 showed germline chromothripsis driven by L1-Mediated Retrotransposition and Alu/Alu Homologous Recombination. Aoki et al.4 found hypomethylation of LINE elements correlated with progression and poorer prognosis of myeloma. They detected an association between common breakpoint loci at commonly lost regions with increased LINE-1 densities. Our findings of widespread Alu duplications in this case suggests the possibility that chromoplexy in myeloma may be induced by retrotransposon activity.
 This case illustrates a “two-hit” MM pathogenesis model: germline mediated HRD creates a permissive genomic background for catastrophic somatic events, including chromothripsis (or, chromoplexy) and immune gene disruption. Similar “two-hit” model involving tumor suppressor genes and oncogenes has been reported by Rodrigues et al.5 These findings emphasize the value of integrated germline-somatic multimodal analysis to identify high-risk disease biology and guide therapeutic strategies. Pawlyn et al.6 have suggested using PARP inhibitors in HRD-LOH MM, which correlated with high-risk disease in the relapsed MM (RRMM) setting.
 Our case suggests that such high-risk presentation can occur in newly diagnosed myeloma (NDMM) too and begs the question: should PARP inhibitors be evaluated for upfront induction treatment of such cases?
 References: Raychaudhuri J et al. Abstract 507/Transplantation and Cellular Therapy 30 2S (2024) S370-S398 Thibaud et al. Blood Cancer Discovery (2024);5: 428–41 Nazaryan-Petersen L et al. Hum Mutat. (2016) Apr;37(4):385-95 Aoki et al. Genome Medicine (2012), 4:101 Rodrigues et al. iScience, (2025) 28, 111620, January 17, Pawlyn C et al. Leukemia (2018) 32:1561–1566
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faab97a25ec8eddcce44ce6eb8cdc61a206c2acd" target='_blank'>
              Germline mediated HR deficiency, alu element duplication, and immune pathway alterations define aggressive, drug-resistant multiple myeloma
              </a>
            </td>
          <td>
            Anindyajit Banerjee, Sombuddharoy Bhowmick, S. Vasudevan, Anurag Mehta, N. Agrawal, V. Sheth, D. Bhurani, Neeraj Arora, Jyotishankar Raychaudhuri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives To address the unelucidated mechanisms of breakpoint formation in TFE3-rearranged renal cell carcinoma (TFE3-rRCC), this study characterizes breakpoint distribution within the TFE3 gene. We further explore how non-canonical DNA structures and their thermodynamic stability fluctuation may act as predisposing factors for the genomic instability driving these characteristic translocations. Methods TFE3 breakpoints were identified in a cohort of 31 TFE3-rRCC tumor samples. The chi-square test was used to assess the statistical significance of breakpoint clustering. To investigate potential structural determinants, we predicted the distribution of G-quadruplex-forming sequences and palindromic motifs. Moving beyond simple motif density, we calculated the local Gibbs free energy changes (ΔG) associated with DNA secondary structures using Mfold and RNAfold to model thermodynamic stability across the TFE3 gene. This thermodynamic stability fluctuation was quantified as the maximum absolute local change in folding free energy (|dΔG|). Finally, this correlation between thermodynamic stability fluctuation and breakpoint location was validated by analyzing the 13 most frequently rearranged genes reported in the COSMIC database. Results A significant breakpoint cluster was identified within intron 5 of TFE3, containing 23 of 31 breakpoints (74.19%; chi-square test, P < 0.05). While the simple density of G-quadruplex or palindromic motifs did not directly correlate with breakpoint locations, a strong association with local thermodynamic stability fluctuation was observed. The region within intron 5 exhibited the highest thermodynamic stability fluctuation. This result suggests that regions of high thermodynamic stability fluctuation are correlated with increased susceptibility to DNA breakage. This finding was corroborated in the COSMIC dataset, where breakpoints in 12 of the 13 most frequently rearranged genes were similarly located near peaks of high |dΔG|. Conclusion Our findings indicate that breakpoint events in TFE3-rRCC are non-randomly clustered within intron 5. This clustering correlates strongly with regions characterized by high thermodynamic stability fluctuation (|dΔG|) of potential non-canonical DNA secondary structures. The principle that elevated local thermodynamic stability fluctuation is a feature of breakpoint locations was supported by analysis of a broader set of oncogenes, suggesting that high local thermodynamic stability fluctuation is a common feature of translocation-prone regions in cancer, representing a plausible, though not proven, contributor to genomic fragility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba18c1a28435183414957442b6594dd9961a8358" target='_blank'>
              Mechanistic study of TFE3 breakage in TFE3-rearranged renal cell carcinoma: the perspective of non-canonical DNA structures and their stability
              </a>
            </td>
          <td>
            Xiaopo Zhang, Qinshuang Dang, Xinghe Pan, Zhenggen Deng, Yanhao Xu, W. Gan, Hongqian Guo
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e1c75f6af2c61e333ce8d8a40a65dbb9822c6f" target='_blank'>
              Lysine-specific histone demethylase complex restricts Epstein–Barr virus lytic reactivation
              </a>
            </td>
          <td>
            Yifei Liao, Jinjie Yan, Isabella Y. Kong, Zhixuan Li, Weiyue Ding, Sarah Clark, Davide Maestri, Tetsuya Yoshida, L. Giulino-Roth, B. Gewurz
          </td>
          <td>2025-10-31</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Introduction:
 Multiple myeloma (MM) is characterized by its genomic complexity that includes recurrent driver single nucleotide variation (SNVs) alongside a high burden of structural variation (SV). Complete characterization of somatic variation in a MM genome is dependent on a technology that can accurately and sensitively detect both small and large variants. While traditional short read sequencing is the gold standard for characterization of simple SNVs, it has limited accuracy for characterization of SVs especially those involving complex rearrangements and challenging to map loci (i.e. IGH). Here, we introduce LinkPrep™ technology, a novel linked-read approach compatible with short read sequencers that delivers high accuracy for both small and large somatic variant detection. We demonstrate LinkPrep performance across four separate relapse MM genomes.
 Methods:
 Four MM patient-derived xenografts were profiled with the Dovetail® LinkPrep™ kit and sequenced to ~30X on an Illumina platform. Data was analyzed through the Dovetail Analysis Portal to detect SNVs/Indels (DeepSomatic), CNVs (Purple), and SVs (HiC-breakfinder and proprietary tools). Call sets were compared against other genomics technologies including WGS (80X tumor/ 30X normal), PacBio (15X), and OGM (400X). We focus analysis on one sample (M24) containing complex SV events and several driver SNVs. Using this sample, we highlight the capabilities of LinkPrep data to detect SNV/Indels, CNVs, and SVs and to link this information together to provide a candidate solution for the complex templated insertions. We further describe SV-driven 3D chromatin alterations (i.e. neo-loops, potential enhancer hijacking) from the same 30X LinkPrep dataset.
 Results:
 LinkPrep + Dovetail Analysis Portal (~8 hours analysis time) returned high quality variant results for the four MM samples. LinkPrep data recalled all pathogenic SNV/Indels detected by 80X WGS (notable pathogenic variants were detected in NRAS, KRAS, TP53). LinkPrep data further detected SVs involving the IGH locus (inversions and translocations) in all four samples. Deeper analysis of one sample (M24) highlights that LinkPrep data detects genetic alterations common in MM including SNV mutations in NRAS and TP53 (both occurring at 100% VAF), IGH rearrangements, chr 1q gain, chr 1p loss of heterozygosity, trisomy of chr 5,7, and 15, and complex rearrangements involving chromosomes 2, 3, 4, 16, and 19. With the exception of a confident subclonal (1% SV-VAF) unbalanced translocation detected by LinkPrep, the remaining LinkPrep large SV calls were validated by other genomics methods (tumor/normal 80X WGS/15X PacBio/400X OGM). Underscoring the capability of LinkPrep for SV detection, LinkPrep calls at 30X contain >10 times more read support for every SV call over other technologies. Using the linked-read feature of LinkPrep data along with CNV segment calls and SV calls provided through the Dovetail Analysis Portal a candidate solution is derived for the complex rearrangement involving chromosomes 2, 3, 4, 16, and 19. This LinkPrep solution, refined down to base pair resolution for every breakpoint, is verified by 400X OGM data. Uniquely, LinkPrep data further enables visualization of the SV-reconstructed 3D genome and reveals neo-loops connecting enhancers with promoters to suggest enhancer hijacking mechanisms associated with the complex rearrangement.
 Conclusion:
 We applied and validated LinkPrep technology, a novel, multi-dimensional, linked-read method, to a MM sample cohort. The technology enables high accuracy comprehensive variant detection, offering improved sensitivity and accuracy for SV detection over short and long reads, including those in challenging genomic regions such as IGH. Linked-reads captured in LinkPrep data enable phased reconstructions of complex rearrangements often found in MM genomes. Furthermore, the same LinkPrep data enables detection of epigenetic state and neo-3D genome interactions driven by SVs to elucidate enhancer hijacking events. Together, these data demonstrate that LinkPrep linked-reads enable improved characterization of somatic variation in highly rearranged genomes, while simultaneously detecting epigenetic states – all enabled with the accuracy and cost-effectiveness of short read sequencing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339458d030fcdb32def9e53403cc8c5fb11c7937" target='_blank'>
              Characterization of multiple myeloma genomes with LinkPrep™ assay enables sensitive detection of somatic variation and SV-driven interactions of the 3D genome.
              </a>
            </td>
          <td>
            Nathan Becker, Enze Liu, Alexander Fortuna, Jonathon Torchia, Aneta Mikulasova, Zackary Sanborn, Lisa Munding, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) and its aggressive subtype plasma cell leukemia (PCL) are transcriptionally-driven hematological malignancies, heavily dependent on enhancer activity. Enhancers are distal regulatory elements at which transcription factors (TFs) bind, recruiting coactivators that promote chromatin remodeling to stimulate target gene expression via physical proximity of enhancer and promoter. MM cells show high expression of essential TFs, including IRF4, MYC, PRDM1 and IKZF1/3, which function by binding to and activating oncogenic enhancers.
 A detailed understanding of chromatin regulation in MM patients requires the comparison of histone modifications, chromatin accessibility and TF/cofactor binding distribution within the same cells, which has so far been unachievable. Patient analysis has mainly relied on ATAC-seq and RNA-seq, with limited inference of TF binding and chromatin state, owing to the difficulty in obtaining the large numbers of MM cells required for more detailed characterization. Many observations of enhancer function in MM therefore come from cell lines, which only offer an approximation of MM physiology.
 To overcome the limitations of in vitro models and primary cell availability, we employed TOPmentation (TF-OPtimized ChIPmentation) to generate ChIP-seq-quality data for CD138+ cells from MM and PCL patients. This technique uses as few as 100,000 cells for histone modifications, and 250,000 cells for TFs, allowing us to profile up to 15 features per patient. In addition, we used the base-pair resolution chromosome conformation capture technique micro-capture-C (MCC) to visualize enhancer-promoter interactions in a PCL patient sample. This allowed us to dissect large oncogenic enhancers, identifying the key TF binding sites within each enhancer that directly contact the gene promoter. Using this dual approach, for the first time we have generated integrated maps of TF occupancy, active and repressive histone modifications, and chromatin interactions in MM patient samples.
 We characterized the epigenetic landscapes of two t(11;14) and two t(4;14) patients, finding broadly similar chromatin profiles in each subtype. Key MM enhancers were retained at oncogenes including IRF4, MYC,PRDM1 and IKZF1/3, indicating a convergence of gene regulation in MM originating from distinct initiating genetic events. At many enhancers, we observed multiple TFs co-binding at the same sites, indicating cooperative activation of target genes. By combining TOPmentation with MCC, we found that many of these sites interact with target gene promoters, directly implicating them in gene regulation.
 In MM, patients typically display mutually exclusive upregulation of CCND1 or CCND2. As previously established, we found strong activation at the CCND1 locus by IGH translocation in t(11;14) patient samples, whereas in t(4;14) patients CCND2 was upregulated via an upstream super-enhancer. Surprisingly, in each subtype the silent CCND gene was not actively repressed, but rather existed in a poised, bivalent state marked by active H3K4me3 and repressive H3K27me3 modifications. This argues that intricate regulation of CCND1/2 is required to maintain optimal levels of expression for tumor growth.
 Comparison of patient and cell line data showed broad conservation of epigenetic features. However, we also observed examples of distinct gene regulation, for example at the PRDM1 locus. In several MM cell lines (including KMS12, H929 and MM1S), transcription of PRDM1 initiates from a distal promoter. However, this locus was inactive in the patient samples analyzed, and instead an alternate, proximal promoter was favored.
 This study for the first time reports the genomic binding distribution of TFs and histone modifications in MM patient cells, demonstrating the importance of comprehensive epigenomic analysis to capture the complexity in MM gene regulation. This emphasizes the need for validation of cell line models to provide a more physiologically relevant understanding of enhancer-driven oncogene regulation. As we expand this patient-derived dataset, it will provide a valuable resource to better understand MM biology and inform future strategies for precision epigenetic therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab64ef7d2d4fd86af25053430e4f99417b63dfb" target='_blank'>
              Revealing oncogenic enhancer regulation by epigenomic profiling of multiple myeloma and plasma cell leukemia patient samples
              </a>
            </td>
          <td>
            Katrina Fordwor, Jason Taslim, Sophie Ball, P. Sabbattini, A. Katsarou, T. Milne, A. Karadimitris, A. Chaidos, Nick Crump
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16016e64ba44704fcf2f544c64c327f4597e0734" target='_blank'>
              Lamin B1 Safeguards the B Cell Genome and Shapes Lymphoma Outcome
              </a>
            </td>
          <td>
            Filip Filipsky, Katarina B Chapman, Johannes Bloehdorn, Jun Wang, John Gribben, Michael Hausmann, Christoph Cremer, Andrejs Braun, Marta C Sallán, Tanya Klymenko
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2ceccde009f3d35721416ae6a0441554cd9fa0" target='_blank'>
              The mitotic stopwatch synergizes with mild p53 activation to halt cell proliferation
              </a>
            </td>
          <td>
            Beata E. Mierzwa, Franz Meitinger, A. Desai, K. Oegema
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8df6dc2d72a4159b55a77cb1f1691bd8b0d91d08" target='_blank'>
              Click-code-seq reveals strand biases of DNA oxidation and depurination in human genome.
              </a>
            </td>
          <td>
            Vakil Takhaveev, N. Püllen, Navnit K. Singh, Lucie Lefevre, Emilie A Aghajani, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2025-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2158d0cf0754098c7826e6edf394a24a8aa054" target='_blank'>
              Beyond Reverse Transcription: Molecular Mechanisms and Emerging Paradigms in Retroviral Replication.
              </a>
            </td>
          <td>
            Mohammad Abdullah Jehad, L. Ali, Vineeta N Pillai, Suresha G. Prabhu, Farah Mustafa, Tahir A. Rizvi
          </td>
          <td>2025-12-26</td>
          <td>FEMS microbiology reviews</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterizing tissue-specific (TSp) gene expression is crucial for understanding development and disease; however, traditional expression-based methods often overlook the latent “regulatory grammar” embedded in the non-coding DNA, particularly in distal promoter regions. Here, we introduce TSProm, a framework that specializes a DNA foundation model (DNABERT2) to decipher the long-range regulatory logic of TSp promoters at the gene isoform level. The contributions of our work are two-fold. First, we propose a novel comparative design that trains two distinct models, A: for general promoter biology and B: for TSp regulation. These models enable the precise isolation of sequence motifs around the transcription start site that uniquely define tissue identity. Second, we introduce a comprehensive explainable AI (xAI) module that integrates attention-based discovery with model-agnostic SHAP analysis to provide robust, cross-validated interpretations of learned features. Applying this framework to human brain, liver, and testis promoters, we identified and validated clinically relevant transcription factors (TFs) in the brain, including SP1, MYC, and HES6, and confirmed their known roles in diseases such as gliomas and neuroblastomas. Our analysis further revealed that C2H2 Zinc Finger proteins are a dominant feature of the global landscape of TSp gene regulation. TSProm provides a novel and interpretable framework for identifying TSp gene regulatory elements, offering powerful computational tools for the study of tissue-specific gene regulation in normal and disease conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a83e73abe66e3648e75b598545eed1991706b17" target='_blank'>
              TSProm: Deciphering the Genomic Context of Tissue Specificity
              </a>
            </td>
          <td>
            Pallavi Surana, Pratik Dutta, Nimisha Papineni, Rekha Sathian, Zhihan Zhou, Han Liu, Ramana V. Davuluri
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel C. de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Intracranial germ cell tumors (IGCTs), primarily affecting infants and children, exhibit malignancy and prognosis that largely depend on tumor histological subtypes. While somatic coding alterations have revealed activation of the MAPK and PI3K pathways in the germinoma subtype, the genetic basis of the more malignant yolk sac tumor (YST) subtype remains poorly understood. In this study, we analyzed whole-genome sequencing data from blood-tumor matched samples of 69 IGCT patients, identifying a significant enrichment of somatic chromosome 1p36 copy number loss in YST (7 out of 13 YST cases compared to 3 out of 56 non-YST cases). Further analysis of whole-genome level somatic single nucleotide variations revealed a YST-specific single base substitution signature (5 out of 13 YST cases), characterized by frequent ACG to AGG mutations. Alongside this novel signature, we observed a high burden of point mutations, structural variations, and LINE1 retrotransposition, indicating increased genomic instability in this subgroup of IGCTs. Additionally, association analyses revealed that these genomic instability events were significantly linked to the copy number loss of 1p36. These findings elucidate the distinct mutational landscape of YSTs and suggest potential treatment vulnerabilities for this subtype by targeting genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d3aac5eea035f8521957bf70a25b2e906a88c96" target='_blank'>
              EPCO-29. WHOLE-GENOME SEQUENCING REVEALS GENOMIC SIGNATURE OF INTRACRANIAL YOLK SAC TUMORS
              </a>
            </td>
          <td>
            Dong Song, Yang Zhao, Ran Liu, Jie Ma, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified numerous single-nucleotide polymorphisms (SNPs) associated with various diseases, including cancer. However, the mechanisms by which these SNPs contribute to disease susceptibility remain largely unclear. While recent studies have explored the transcriptional impact of disease-associated SNPs, their role in post-transcriptional regulation has been less extensively investigated. In this study, we investigated whether cancer-associated SNPs influence gene expression by altering N6-methyladenosine (m6A) RNA methylation. We collected GWAS-identified SNPs across nine cancer types and integrated these with matched tumor and normal m6A RNA immunoprecipitation sequencing (m6A-seq) and RNA sequencing (RNA-seq) datasets. We first identified differentially methylated m6A sites and assessed whether cancer-associated SNPs were enriched within these regions. These analyses revealed that cancer-associated SNPs were significantly enriched within hypermethylated m6A regions in colon cancer. Integrative analysis revealed that SNPs enriched in m6A-modified regions are associated with altered gene expression and RNA splicing, suggesting that m6A methylation mediates the post-transcriptional impact of genetic variants. Experimental validation further confirmed altered gene expression following ALKBH5 knockdown, consistent with patient-derived data. Collectively, our findings support a novel mechanistic connection between genetic variants and RNA methylation-driven transcriptomic regulation in colorectal cancer, underscoring the epitranscriptome as a potential axis of oncogenic control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8030aadad77b99c149edba7083659d634eaa70" target='_blank'>
              Genetic variants reshape the m6A epitranscriptome and drive transcriptomic reprogramming in colorectal cancer
              </a>
            </td>
          <td>
            Seung Hun Han, Seongmin Jang, Yeongwon Kim, Kun Tan, M. F. Wilkinson, Hyobin Jeong, Junho Choe
          </td>
          <td>2025-11-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec5c4b6c3108853fca772396903b6fe2c2600a23" target='_blank'>
              Chromosome centromere copy number amplification associated with exceptional response in HER2-positive metastatic breast cancer patients.
              </a>
            </td>
          <td>
            Charlotte Andrieu, Jojanneke Stoof, Dalal AlSultan, L. Ivers, José Javier Berenguer Pina, D. Skrobo, J. Ballot, Debbie O'Reilly, D. Collins, Alexander J. Eustace, Cecily Quinn, J. Walshe, G. Gullo, N. Walsh, J. Crown
          </td>
          <td>2025-12-20</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Extrachromosomal arrays are unique chromosome-like structures created from DNA injected into the C. elegans germline. Arrays are easy to create and allow for high expression of multiple transgenes. They are, however, unstable unless integrated into a chromosome. Current methods for integration, such as X-rays and CRISPR, damage DNA and are low-efficiency. Here, we demonstrate that the viral integrase PhiC31, which mediates a non-mutagenic recombination between short attB and attP sequences, can be used for extremely efficient and targeted integration of arrays. In this method, a transgene, a selectable marker, and attP sites are injected into the gonad of a strain that (1) has an attB site in its genome, and (2) expresses PhiC31 in its germline. F1 extrachromosomal arrays are cloned, grown for multiple generations with selection, and then screened for homozygous array integrations. The procedure is simple, requires less time than screening for extrachromosomal arrays, and arrays can be screened for transgene function after stable integration. Arrays that transmit are integrated by PhiC31 with 50-95% efficiency, allowing for the isolation of many unique integrants from a single injection. Arrays can also be integrated at fluorescent landing pads and arbitrary sites in the genome. Using nanopore sequencing, we show that three new integrated arrays are between 1.6 and 18 megabases in length, assemble with large repeats, and can contain hundreds of copies of injected transgenes. We have built a collection of strains and plasmids to enable array integration at multiple sites in the genome using various selections. PhiC1-mediated Integration of Arrays of Transgenes (PhiAT) will allow C. elegans researchers to shift from using unstable extrachromosomal arrays to directly integrating arrays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bdbf12680646328af6ba712e12d3d958eba198e" target='_blank'>
              High-efficiency targeted integration of extrachromosomal arrays in C. elegans using PhiC31 integrase
              </a>
            </td>
          <td>
            Matthew S. Rich, Ryan Pellow, Adam Hefel, O. Rog, Erik M. Jorgensen
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90cf835255eacd329046c889367761e186d358b" target='_blank'>
              Cohesin forms fountains at active enhancers in C. elegans.
              </a>
            </td>
          <td>
            Bolaji N. Lüthi, Jennifer I Semple, Anja Haemmerli, Saurabh Thapliyal, Kalyan Ghadage, Klement Stojanovski, Dario D’Asaro, Moushumi Das, Nick Gilbert, Dominique A. Glauser, Benjamin D. Towbin, Daniel Jost, Peter Meister
          </td>
          <td>2025-12-11</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Oncogene-targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance. Therapeutic approaches to treat tumours that have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms. Rather than treating resistance after it emerges, it may be possible to prevent it by inhibiting the adaptive processes that initiate resistance, but these are poorly understood. Here we report that residual cancer persister cells that survive oncogene-targeted therapy are growth arrested by drug stress-induced intrinsic type I interferon signalling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA fragmentation factor B (also known as caspase-activated DNase), which induces DNA damage, mutagenesis and stress response factor activating transcription factor 3 (ATF3). ATF3 limits activator protein 1-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DNA fragmentation factor B or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes the regrowth of residual cancer cells that survive drug treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0eb23b0262f48e292a21b5682ac6f6faf0a4d24" target='_blank'>
              DNA fragmentation factor B suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F. Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X Wang, B.E. Mauch, Rhea Plawat, Ariel H. Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J. Hangauer
          </td>
          <td>2025-11-17</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Polycomb targeted loci form long-range chromatin interactions independent of CTCF-cohesion and are demarcated by low DNA methylation(1–3). Polycomb targets form extremely long-range loops (long Polycomb loops) that can separate anchors up to 60 Mb. These loops predominantly occur in cells of self-renewal status, such as human hematopoietic stem cells (HSC) and mouse embryonic stem cells (ESC), but rarely in cell lines. To identify long Polycomb loops in cancer, we initiated a pan-cancer survey of long Polycomb loops in a collection of 264 tumor samples (33 acute myeloid leukemias, 17 T cell lymphoblastic leukemia, 29 Breast cancer samples, 63 pediatric brain tumors, 70 prostate cancers, and 42 colon cancers). We found most cancers, including all prostate cancers and colon cancers, lack long Polycomb loops except pediatric brain tumors and certain AMLs. In pediatric brain tumors, we found 30% of the ependymoma PFA subtype (which shows globally depleted H3K27me3) notably displayed strong long Polycomb loop interactions. AML displayed more diverse levels of long Polycomb loop interactions. Most AMLs lost both long Polycomb loops with Polycomb binding loss at the loop anchors. Whereas 10% of AMLs retain long Polycomb loops as strong as in HSCs. These AMLs recurrently carry mutations in CEBPA and STAG2, which are not associated with the Polycomb complex or DNA methylation machinery. We found that long Polycomb loop strong AML is sensitive to EZH2 inhibition, which induces cell differentiation. Conversely, PRC1 component dependency can be used to predict long Polycomb loop formation in B-cell lymphoblastic leukemia cell lines. Our analysis suggests that the oncogenesis process antagonizes the long Polycomb loop maintenance in most cancers, yet certain cancers may still preserve strong long Polycomb loops from cell-of-origin. The maintenance of long Polycomb loops sensitizes cells to Polycomb inhibition, indicating that such loops could be an epigenomic biomarker for pharmacological or genetic Polycomb inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd40648aae14140db85bd195a593affdaa280e98" target='_blank'>
              Pan-cancer 3D genomic analysis revealed extremely long Polycomb loops as the biomarker for sensitivity to Polycomb inhibition
              </a>
            </td>
          <td>
            Sean Moran, Zhong Fan, M. Zanovello, Feng Fan, David Xue Qing Wang, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Gene expression and regulation with or without alternative splicing are key factors for cells to properly function. Distant splicing quantitative trait loci (distant sQTLs) are genomic mutations that impact the alternative splicing patterns of far-away genes. Nevertheless, the procedures causing a distant sQTL to regulate the alternative splicing of genes are not well defined. Higher resolution chromosome conformation capture experiments like Micro-C or Hi-C together with an expanding number of sQTL datasets on humans help us in understanding the spatial processes governing distant sQTL relationships at a genome-wide scale. In this study, we focus on analyzing whether spatial closeness helps in regulating sQTL-gene interactions over high-order chromatin topological domain structure, which is inferred from chromosome conformation experiments. We discover larger-scale chromatin shape to be in line with sQTL associations. In detail, sQTLs are generally spatially near their splicing genes in 3D, they frequently appear near topologically associating domain (TAD) and frequently interacting region (FIRE) boundaries, and are favorably related to genes over TADs and FIREs. Additionally, we discover that inside-domain sQTLs accompanied by functional regulatory elements, including enhancers and promoters, are spatially closer than all inside-domain sQTLs. This result suggests that spatial closeness between sQTLs and their distant splicing genes obtained from chromatin’s TAD structure has major importance in regulating alternative splicing and thus in gene regulation. Our results are robust across different experiments such as Hi-C and Micro-C, different TAD inference methods, different Hi-C binning resolutions, different alternative splicing events, and once we control for eQTLs, which are shown to be spatially close to their genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3f85319df6c80cf2868c0242d3d5eb8c32dec3b" target='_blank'>
              Analysis of chromatin structure reveals the connection between sQTLs and the splicing of distant genes
              </a>
            </td>
          <td>
            Batuhan Eralp, Emre Sefer
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00991a0395a2490dd6217e4ff4ad6e01917b30a6" target='_blank'>
              Menin inhibition impairs metastatic colonization of Ewing sarcoma
              </a>
            </td>
          <td>
            Katherine A. Braun, Nicolas M Garcia, Mohamed Ahmed, Darleen S Tu, Stephanie I. Walter, Emma D. Wrenn, Megan E.B. Dean, Elizabeth R. Lawlor
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D. Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Alternative splicing significantly contributes to gene expression heterogeneity and disease progression, yet analyzing its dynamics in short genetic regions such as microexons remains challenging. Here we identified notable variations in ribosomal protein S24 (RPS24) splicing patterns across breast cancer subtypes and investigated this novel regulatory mechanism. To overcome the complexity of analyzing three consecutive microexons (3 bp, 18 bp and 22 bp), we developed a specialized approach combining splice junction read analysis with fragment analysis for accurate isoform quantification. We observed distinct isoform compositions across breast cancer cell lines. The 3-bp exon-containing isoform (ex4:3 bp) of RPS24 showed significantly higher expression in estrogen receptor-positive (ER+) cells, demonstrating the strongest association with estrogen receptor signaling among all analyzed genes. This isoform functions as a molecular sensor for therapeutic interventions, being consistently upregulated following mTOR or CDK4/6 inhibition but consistently reduced across diverse drug-resistant cell lines, regardless of resistance mechanism. Through systematic RNA-binding protein screening and crosslinking immunoprecipitation followed by high-throughput sequencing analysis, we identified PTBP1 as a critical upstream regulator mediating microexon skipping. Analysis of multiple patient cohorts demonstrated that decreased ex4:3 bp expression strongly correlated with poor epithelial differentiation and metastatic progression specifically in ER+ breast cancer. Our findings suggest that RPS24 alternative splicing is associated with a multilayered regulatory network integrating ER signaling, cell cycle and PTBP1-mediated splicing. The ex4:3 bp isoform serves as a potential biomarker for drug resistance and treatment response in ER+ breast cancer. This study explores how a gene called RPS24behaves differently in breast cancer. The authors wanted to understand how changes in RPS24 might affect cancer progression and treatment response. The team analyzed data from breast cancer cell lines and patient samples to study the different forms of RPS24. They used RNA sequencing and experimental techniques to identify these isoforms. The researchers found that one specific isoform, called ex4:3 bp, is linked to estrogen receptor status in breast cancer. This means it might play a role in how breast cancer cells respond to hormone therapies. They also discovered that a protein called PTBP1 helps to regulate these isoforms by controlling which parts of the gene are included or skipped. This study suggests that the ex4:3 bp isoform could be a useful marker for predicting how breast cancer will respond to treatment. This summary was initially drafted using artificial intelligence, then revised and fact-checked by the author.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c63398fd709145e269cf1f4053116b33fc925f52" target='_blank'>
              RPS24 microexon isoform as a novel biomarker for estrogen receptor-positive breast cancer progression and therapeutic resistance
              </a>
            </td>
          <td>
            Jiyeon Park, D. Nam, Seung-Hyun Jung, Bin Tian, Yeun-Jun Chung
          </td>
          <td>2025-11-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese M. Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, Jangkeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416974525ed871e08b5c483505e950ab4b46e07f" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation.
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah Burget, Tobias Friedrich, So-Yoon Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-12-02</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Genomic alterations in germ cell tumors (GCTs) have been identified in the RTK, MAPK, and PI3K pathways through whole exome sequencing, with a prevalence of 71% in germinomas but only 33% in non-germinomas GCTs (NGGCTs). Despite evolution in treatment, GCTs present therapeutic challenges and lack molecularly targeted strategies.



 Whole genome sequencing was performed on 71 cases (37 germinomas and 34 NGGCTs) from the iGCT Genome Analysis Consortium and the Japan Children’s Cancer Group.



 Putative driver mutations were identified in 83% of tumors, primarily involving RAS/MAPK followed by mTOR/Akt pathways. Germinomas exhibited a higher mutation frequency than NGGCTs (84% vs 62%), with KIT enriched in germinomas and USP28 in NGGCTs. All USP28 alterations were bi-allelic and highly specific to CNS GCTs, with only one diffuse midline glioma out of 1448 cases of pediatric cancers affected. Variant allele frequency and clonal analyses revealed that these mutations occurred early, preceding copy number alterations. Germline analysis identified Kleinfelter or Down syndrome in 5.6% of cases, particularly in NGGCTs lacking other alterations.



 Many cases exhibited driver mutations present in major clonal populations and could serve as therapeutic targets. There were notable differences based on histological type, with a diverse range of single nucleotide variants/structural variants observed in NGGCTs. A timeline of tumorigenesis from SNVs/SVs to copy number abnormalities was observed. The utility of understanding germline variants was also implicated in the pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6633a5d8f98dc245c8a327ae1335afe0892cad2" target='_blank'>
              PTHP-06. Molecular Mechanisms and Developmental Pathways of CNS Germ Cell Tumors Elucidated through Whole Genome Analysis
              </a>
            </td>
          <td>
            Hirokazu Takami, S. Kato, Yuki Yamagishi, Eiji Sugihara, Y. Matsushita, Takako Yoshioka, Junko Hirato, Miho Kato, Hajime Okita, Tomonari Suzuki, Masayuki Kanamori, Akio Asai, Akitake Mukasa, Toshihiro Kumabe, Akihide Kondo, Yuichi Hirose, M. Nagane, Kazuhiko Sugiyama, T. Shibata, Kenichi Yoshida, Y. Shiraishi, Takaaki Sato, Ryo Nishikawa, Motohiro Kato, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, Thomas F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Determining the correct structure of large, interspersed duplications and related complex genomic rearrangements in genetic disease is critical when establishing causal roles and requires a technology able to span the entire duplicated segment(s) on single molecules. We assessed the use of Bionano optical genome mapping (OGM) for this purpose. We combined OGM, Illumina short-read sequencing and fluorescence in situ hybridisation (FISH) to characterise three large interspersed duplications/triplications, and used the deepC algorithm to predict impact on local topologically-associating domains (TADs), assisting functional interpretation. Case 1 harboured paired interspersed duplications (244/323 kb) on chromosome 13. By analysing multiple molecules > 300 kb completely spanning the smaller duplication, we unambiguously determined the correct structure, which potentially alters the TAD containing FGF9, a candidate gene. In Case 2, involving a child with hypertrichosis and gingival hyperplasia (HTC), duplications on chromosomes 16 (2.01 Mb) and 17 (564 kb) were linked on short-read sequencing. By obtaining three OGM molecules spanning the 564 kb segment, we deduced that a t(16;17) translocation was present, which we confirmed by FISH. This interpretation has important implications for clinical risk and highlights KCNJ2 as a potential driver of the HTC3 locus at 17q24.3. Case 3 involved a complex chromoanasynthesis event on chromosome 20. OGM readily resolved all but two of 12 alternative structures; however full resolution required reads to span a 627 kb duplication, which we could not achieve consistently. OGM represents a powerful tool for disambiguating complex structural variants, but requires multiple individual reads to completely span the duplicated segment. In our hands the upper size of duplications that could be resolved was ~ 550 kb. Deducing the correct configuration is critical both for mechanistic understanding of pathogenesis and accurate recurrence risk counselling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e568829e237f4a57a0d68d6063ed9794d08d203" target='_blank'>
              Exploring the size limits of Bionano optical genome mapping to resolve alternative structures of linked interspersed chromosomal duplications
              </a>
            </td>
          <td>
            Yang Pei, E. Calpena, Jill M. Brown, Ron Schwessinger, Lucy Platts, Simon J. McGowan, Tazeen Ashraf, F. Forzano, Jane A. Hurst, Wendy D Jones, Ajoy Sarkar, Richard J. Gibbons, S. Twigg, Andrew O. M. Wilkie
          </td>
          <td>2025-11-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceca0fc9a67ea509cda7d2d2cbdefbe5c21b1e0b" target='_blank'>
              Evidence for Transcription and Horizontal Gene Transfer in Dipteran Germline-Restricted Chromosomes
              </a>
            </td>
          <td>
            Riccardo G. Kyriacou, M. Herbette, Robert B. Baird, Katy M. Monteith, Laura Ross, Christina N. Hodson
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transposable elements (TEs) are often epigenetically repressed in eukaryotic cells, but still affect the molecular state of the cell in certain contexts. A flurry of recent studies have elucidated new effects of TE sequences in eukaryotic cells. We review these emerging molecular effects of TEs, including a variety of new mechanisms by which TE sequences affect the cell, including pre- and post-transcriptional regulation of gene expression; cell-to-cell transmission of genes within a multicellular organism through virus-like activity; and RNA-guided DNA insertion. Recent demonstration of TE-guided genome editing underscores the importance of these investigations for both basic and translational research. Future work is needed to continue to unravel the molecular effects of TE sequences across developmental stages, across cell types, and in various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3445e991f8e7a3fbd17d3feddadbe0f7c8e2d56" target='_blank'>
              Molecular effects of transposable element sequences in mammalian cells
              </a>
            </td>
          <td>
            Ming-Ching C. Wen, Joshua D. Welch
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most common (~80%) and lethal ovarian cancer subtype in the United States, characterized by TP53 mutations and DNA repair defects causing chromosomal instability (CIN). KIF18A is an essential cytoskeletal motor protein for cell division in CIN+ cancer cells, but it is not necessary for cell division in normal cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CIN+ cancers. We investigated the use of a novel KIF18A inhibitor ATX020, for selectively targeting CIN+ HGSOC cells using growth inhibition assays, invasion assays, immunoassays, cell cycle analysis, and immunofluorescence techniques. Using DepMap and flow cytometry, we classified a panel of HGSOC cell lines based on aneuploidy scores (AS) and ploidy levels and identified a correlation between these classifications and sensitivity against ATX020. ATX020 induced cytotoxicity through mitotic arrest and DNA damage, and reduced tumor growth in HGSOC with high aneuploidy scores (AS). Mechanistically, ATX020 blocks KIF18A’s plus-end movement on spindle fibers, increasing spindle length, resulting in chromosomal mis-segregation, aneuploidy, and DNA damage. Our findings suggest that ATX020 inhibits CIN+ HGSOC cells mainly by inducing mitotic arrest and DNA damage, disrupting KIF18A’s function crucial for mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f42a6d74edbdfa8c878ba6b61be09681cd9d76" target='_blank'>
              The KIF18A Inhibitor ATX020 Induces Mitotic Arrest and DNA Damage in Chromosomally Instable High-Grade Serous Ovarian Cancer Cells
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Tzu-Ting Huang, Maureen Lynes, Sanjoy Khan, Serena J Silver, Jung-min Lee
          </td>
          <td>2025-11-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh R. Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, Garrett Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ee418dd0e60cea320d764ace68e662b4dbd193" target='_blank'>
              Structural Analysis of Recombinant AAV Vector Genomes at Single-Molecule Resolution
              </a>
            </td>
          <td>
            David Rouleau, Dimpal Lata, Serena Dollive, Robert E. Bruccoleri, Laura Van Lieshout, Diane Golebiowski, Ify Iwuchukwu
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common form of primary liver cancer and has limited therapeutic options. Epigenetic dysregulation plays a crucial role in hepatocarcinogenesis; however, its impact on cis-regulatory elements (CREs) and retrotransposons remains underexplored. Here, we investigated the epigenetic changes underlying the aberrant CRE and retrotransposon activity in HCC. We show that focal DNA hypomethylation of these elements is associated with transcriptional reprogramming. Notably, we uncover a dual regulatory mechanism for GPC3, a key diagnostic biomarker and immunotherapeutic target in HCC. This requires the concomitant reactivation of a fetal liver super-enhancer (SE) and DNA hypomethylation of CpG islands. Furthermore, DNA methylation loss drives cryptic activation of retrotransposons, some of which exhibit prognostic potential. Intriguingly, we identify a HERVE-int-derived long non-coding RNA that shows higher expression in patients with more aggressive tumors, poorer disease outcomes, and is correlated with molecular signatures associated with improved response to immunotherapy. Collectively, our findings reveal widespread epigenomic dysregulation of CREs and retrotransposons in HCC, highlighting new potential therapeutic strategies and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed2d6556ed99cbdaa0f49af5ea87b73a7cb6583" target='_blank'>
              Multi-omic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potential
              </a>
            </td>
          <td>
            Clooney C Y Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y J Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B S Lai, K. Yip, Alfred S Cheng, Danny Leung
          </td>
          <td>2025-12-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Extrachromosomal circular DNA (eccDNA) has emerged as a critical player in cancer biology. Accumulating evidence indicates that eccDNA promotes tumor progression by inducing genomic instability, amplifying oncogenes, and regulating gene expression. However, its biological function and mechanistic role in acute myeloid leukemia (AML) remain unexplored. In this study, we investigated the functional significance and regulatory mechanisms of eccDNA in AML, aiming to identify potential therapeutic targets.



 Bone marrow samples were collected from newly-diagnosed and chemo-resistant AML patients with informed consent. High-throughput eccDNA sequencing was performed to determine eccDNA length distribution. The presence of gene-containing eccDNAs was validated by PCR and fluorescence in situ hybridization (FISH), and synthetic eccDNA constructs were generated to investigate their functional roles in AML. In vivo, we evaluated the impact of eccDNA-encoded LIPIN1 (ecc-LIPIN1) on AML progression and chemoresistance, and developed a PEI/PC7A nanoparticle system for therapeutic intervention. Proteomic and transcriptomic analyses were performed to explore the downstream targets.



 High-throughput eccDNA sequencing revealed abundant eccDNAs (0.3-1 kb) across newly-diagnosed and chemo-resistant AML samples, mapped to diverse genomic regions. Analysis of the GEO dataset (GSE165430) identified genes upregulated in chemo-resistant AML patients. Integrating these data with eccDNA-derived genes highlighted ecc-LIPIN1 as a potential resistance driver.
 PCR and FISH confirmed the presence of eccDNA harboring LIPIN1 nucleotide fragments. Ecc-LIPIN1 levels were significantly elevated in AML samples, particularly in chemo-resistant cases, and associated with poor prognosis. A circular DNA containing LIPIN1 was synthesized based on sequencing data and transfected into AML cells, resulting in elevated LIPIN1 expression. Functional assays demonstrated that ecc-LIPIN1 promoted AML cell proliferation and chemoresistance.
 Given its metabolic role, metabolomic profiling assay revealed that ecc-LIPIN1 primarily regulated lipid metabolism. Fluorescence microscopy and flow cytometry confirmed that ecc-LIPIN1 enhanced lipid droplet formation, contributing to AML progression and chemo-resistance. Proteomic and transcriptomic analyses of ecc-LIPIN1-overexpressed AML cells identified DGAT1, a key enzyme in lipid metabolism, as a downstream effector. Silencing DGAT1 in ecc-LIPIN1-overexpressed AML cells markedly reduced lipid droplet accumulation, indicating that ecc-LIPIN1 drives lipid storage through DGAT1.
 Further analyses revealed that LIPIN1 also modulated endoplasmic reticulum (ER) stress pathways. In LIPIN1-overexpressed AML cells, ER-stress markers CHOP and ATF4 were markedly elevated. Silencing DGAT1 reversed ER-stress activation, suggesting that LIPIN1 promoted AML progression by regulating ER stress via DGAT1.
 To validate the oncogenic function of ecc-LIPIN1, we generated eccDNA constructs containing either wild-type or mutant LIPIN1 nucleotide sequences and transfected them into AML cells. The wild-type ecc-LIPIN1 markedly enhanced AML progression and chemo-resistance both in vitro and in vivo. To counteract this effect, we developed a PEI/PC7A/siLIPIN1 nanoparticle system. Treatment with these nanoparticles efficiently suppressed ecc-LIPIN1 expression, inhibited AML progression, and mitigated chemo-resistance.



 This study provides preclinical evidence that eccDNA plays a critical role in AML progression and chemoresistance by encoding LIPIN1. Mechanistically, ecc-LIPIN1 promotes lipid droplet formation and ER stress via DGAT1, ultimately driving chemoresistance. These findings identify ecc-LIPIN1 as a novel oncogenic factor in AML and highlight its potential as a therapeutic target. Future research into eccDNA biogenesis in AML may uncover new strategies for combating drug resistance and disease progression.



 Extrachromosomal circular DNA; Acute Myeloid Leukemia; drug resistance; lipid metabolism; LIPIN1.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cecd3734536748347f09719465be9ffb74475eb" target='_blank'>
              Eccdna-encoded LIPIN1 drives DGAT1-dependent lipid droplet biogenesis to promote Acute Myeloid Leukemia progression and chemoresistance
              </a>
            </td>
          <td>
            Wancheng Liu, Amin Zhang, Xiangling Xing, Daoxin Ma
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Central nervous system germ cell tumors (CNSGCTs) are the most enigmatic brain tumors. They consist of several histologically distinct types of tumors, which are generally sub-grouped into germinomas or non-germinomatous germ cell tumors (NGGCTs). While germinomas are characterized by alterations in the MAPK/PI3K-pathway genes and global hypomethylation, pathogenesis of NGGCTs is largely unknown. To investigate genetic and epigenetic mechanisms of CNSGCT development, 33 CNSGCTs including 7 germinomas and 26 NGGCTs of various histology collected through the Intracranial Germ Cell Tumor Genome Analysis Consortium and Japan Children’s Cancer Group, together with their matched normal DNA, were subjected to a long-read whole genome sequencing using the PromethION system from Oxford Nanopore Technologies. The mean reading depth was 39X for tumor specimen and 31x for blood DNA. The mean N50 value was 9,565 bp. The mean/median number of structural abnormalities was 16.6/16 for germinomas and 27.9/10.5 for NGGCTs, which was not statistically significantly different. A 2 kb deletion involving an exon of USP28 was observed in a single mixed-germ cell tumor case. Deletions involving PRKN were observed in 4 cases. The mean number of ectopic somatic insertions of the LINE-1 element was 1.8 per case. A single mixed GCT had 21 LINE-1 insertions, one of which was located within an intron of NTRK2. The mean of the whole genome DNA methylation level was 38.2% in germinomas and 75.9% in NGGCTs (p=0.0005). Germinomas mostly had biallelic DNA hypomethylation in the imprinted genes, consistent with their presumed primordial germ cell (PGC) origin. On the other hand, NGGCTs showed various levels of methylation in the imprinted genes, suggesting that they may also have originated from PGCs, however at different stages from that of germinomas. Taken together, our whole genome DNA methylation haplotyping provided a new clue to elucidate the pathogenesis of NGGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a0bfd993cd8d6ea3ee9b1101edf3e65daa2bfa" target='_blank'>
              EPCO-07. A WHOLE GENOME DNA METHYLATION ANALYSIS REVEALED DISTINCT MECHANISMS OF DEVELOPMENT IN DIVERSE TYPES OF CENTRAL NERVOUS SYSTEM GERM CELL TUMORS
              </a>
            </td>
          <td>
            Shoko Yoshimoto, Hirokazu Takami, Erina Ishikawa, Yuta Kuze, Takako Yoshioka, Junko Hirato, Nobuhito Saito, Toshihiro Kumabe, Ryo Nishikawa, Yutaka Suzuki, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Telomerase reactivation via mutations in the telomerase reverse transcriptase promoter (TERTp) is a recurrent feature of glioblastoma, enabling replicative immortality in ~80% of patient cases and presents a critical oncogenic event attractive for therapeutic targeting. TERTp hotspot mutations in glioblastoma generate de novo transcription factor binding sites that recruit the transcription factor GABP, driving aberrant TERT expression. CRISPRoff is an engineered CRISPR/Cas9 fusion protein that can potently and heritably silence cis-regulatory elements such as promoters and enhancers through combined action of DNA and histone methylation. Here, we use CRISPRoff to perform targeted silencing of the TERTp in three patient-derived primary glioblastoma models harboring TERTp mutations. We delivered mRNA encoding CRISPRoff and TERTp-targeting sgRNAs using optimized lipid nanoparticles (LNPs) into glioblastoma primary cultures and organoids. Epigenetic and transcriptional phenotypes were profiled using RT-qPCR, RNA-seq, targeted bisulfite DNA sequencing, long read native DNA sequencing using Oxford Nanopore, ChIP-qPCR, and CUT&TAG. A single transient CRISPRoff delivery was sufficient to stably silence TERT expression by >95% in primary glioblastoma cultures, leading to induction of apoptosis over time and complete replicative senescence in clonal populations. The CpG island encompassing TERTp exhibited ~5-fold increases in cytosine DNA methylation and long read Nanopore sequencing revealed that increased 5mC DNA methylation was specific to the mutant allele of TERTp. Analysis of 5hmC revealed enrichment of this modification upstream of the -124C>T promoter mutation, which was lost upon CRISPRoff activity. ChIP-qPCR revealed a 68% decrease in GABPA occupancy at the TERT promoter. Analysis of H3K9me3 enrichment revealed deposition of heterochromatin within the CpG island of TERTp, and this enrichment was among the most statistically significant increased H3K9me3 peaks genome-wide. In sum, the molecular characterization of CRISPRoff activity against TERTp revealed the ability to achieve mutant allele-specific reprogramming of epigenetic state that could be harnessed for epigenetic therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fd2ebf3520d8717e155c35619009f3a21ba993" target='_blank'>
              EPCO-23. MOLECULAR CHARACTERIZATION OF CRISPROFF EPIGENETIC EDITOR ACTIVITY REVEALS MUTANT ALLELE-SPECIFIC
 TERT
 REPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Akane Hubbard, Zerong Ma, Nicholas O. Stevers, C. Zou, K. Lin, Krish Raval, Rohit Sharma, L. Goudy, Luke A. Gilbert, Niren Murthy, Mitchell S. Berger, Joseph F. Costello, S. J. Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pulmonary pleomorphic carcinoma (PPC) is a rare subtype of lung cancer that comprises both epithelial and sarcomatoid components. The molecular basis of PPC, including the cellular dynamics of its components, remains largely unknown. To elucidate potential therapeutic targets for PPC, we perform a multi-omics analysis incorporating digital spatial profiling and single-cell RNA sequencing (scRNA-seq). PPC exhibits diverse driver gene alterations, including MET exon 14 skipping mutation (METex14) and ALK fusion. In spatial transcriptomics, MET gene and protein are overexpressed exclusively within the epithelial component and not in the sarcomatoid component, even in patients harboring METex14. Epithelial-mesenchymal transition (EMT)-related transcriptional changes, along with extracellular matrix (ECM) remodeling between the epithelial and sarcomatoid components, are observed. scRNA-seq identifies cell populations within the epithelial component that contribute to the malignant transformation and differentiation of the sarcomatoid component. They are characterized by an intermediate EMT state with ECM remodeling signature, suggesting their potential as novel therapeutic targets for PPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f421d642675df6e6bdc9b51c7d9b23970c7bec20" target='_blank'>
              Single-cell and spatial transcriptomic characterization of pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Masayoshi Ohki, Kazuya Hisamatsu, Ryota Fujiwara, Kosei Ishimura, Ryunosuke Fujii, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Ryo Yoshichika, Fumiaki Mukohara, Mao Yoshikawa, Yuma Fukumoto, K. Suzawa, Y. Tomioka, Shin Tanaka, Kentaroh Miyoshi, Mikio Okazaki, S. Sugimoto, Yusuke Otani, Atsushi Tanaka, Hirofumi Inoue, Yosuke Togashi, Hidetaka Yamamoto, D. Ennishi, Shinichi Toyooka
          </td>
          <td>2025-12-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Argonaute proteins provide innate immunity in all domains of life through guide-dependent recognition and cleavage of invader nucleic acids. Many short prokaryotic Argonautes (pAgos) lack nuclease activity and are instead co-encoded with tentative nuclease effectors, but their activation mechanisms remain unknown. Here, we characterize SPARHA systems (short prokaryotic argonautes, HNH-associated), containing HNH nuclease effectors. RNA-guided target DNA recognition by pAgo induces formation of SPARHA filaments with a chain of double nuclease sites formed by HNH tetramers, leading to indiscriminate collateral degradation of DNA and protecting bacterial population from invaders. We show that the assembly of filaments proceeds via a universal activation pathway involving a cascade of target-induced conformational changes in SPARHA, conserved in other short pAgo systems containing various types of effectors. pAgos and associated effectors act as modular defense systems that translate recognition of specific DNA into immune response through assembly of supramolecular complexes, deleterious for invaders and potentially useful for biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121cd79715591f5d74c4c48c7a84d96dfd056de" target='_blank'>
              Argonaute-HNH filaments triggered by invader DNA confer bacterial immunity
              </a>
            </td>
          <td>
            A. Kanevskaya, M. Narwal, L. Lisitskaya, A. Moiseenko, O. S. Sokolova, N. N. Sluchanko, Katsuhiko S. Murakami, A. Kulbachinskiy
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B. Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999aa4e6a47ef459adcf0a9e608c45b2e8af7576" target='_blank'>
              A human supergene: BRCA1, CCDC200, and U2 snRNAs
              </a>
            </td>
          <td>
            Pelle Scholten, Michael Segel, David Haig
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d7d2e338e9c2ff093882c73a51e612889ed2af" target='_blank'>
              Histone H1 Promotes Silencing of Unintegrated HIV-1 DNA
              </a>
            </td>
          <td>
            Jiaqi Zhu, Gary Z. Wang, Hugo D. Pinto, Sean E. Healton, Arthur I Skoultchi, Stephen P. Goff
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints.
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="INTRODUCTION
The transposase protein, InsAB', is the key cytoplastic determinant that catalyzes IS1 movement within a single DNA molecule or between two DNA molecules. Regulation of InsAB' synthesis, its structural characteristics, and its potent cis effect on IS1 transposition have been subjects of thorough investigation. However, limited efforts have been devoted to measuring the levels of IS1 transcripts and transposases made by native IS1 elements and examining their trans-acting function, particularly concerning their role in transposing otherwise immobile IS1 elements and other target DNA segments within the bacterial genome.


METHODS
All constructs were made on the E. coli chromosome using the lambda-Red system. IS1 transposition frequencies were determined using Bgl+ mutation assays, colony PCR and DNA sequencing. IS1 transcripts and transposases were quantitated using lacZ transcriptional and translational reporters.


RESULTS
We first confirm that a native IS1 element, IS1E, can transpose at a dramatically elevated rate in the absence of both transcriptional and post-transcriptional regulations. Using lacZ reporters targeting the transposase gene insAB', we reveal that the InsA repressor moderately reduces insAB' transcription by as much as 16.8 fold. However, the combined action of InsA and ribosomal frameshifting leads to a much stronger reduction in InsAB' production, by up to 735 fold. We show that only a small percentage (roughly 1.6%) of IS1 transcripts are successfully translated into InsAB', highlighting the primary role of post-transcriptional regulation over transcriptional repression in governing IS1 transposition. We also demonstrate, for the first time, that high levels of InsAB' exhibit a strong trans effect, being capable of efficiently transposing not only normally non-mobile IS1 elements but also a larger miniIS1 cassette within the genome, with greater amounts of InsAB' leading to higher frequencies of transposition.


CONCLUSIONS
By dissecting and quantitating the roles of transcriptional and translational controls in transposase production, our study reveals that post-transcriptional regulation via ribosomal frameshifting is the central mechanism governing IS1 intragenomic transposition. Our findings demonstrate that IS1 transposases function in trans, mediating the movement of DNA segments specifically bordered by its IRL and IRR sequences. These findings advance our comprehension of how IS elements mobilize within bacterial genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc63253e7e508e0ba8aecec525150bf22bacf59e" target='_blank'>
              Investigating the Trans Effects of IS1 Transposases on Intragenomic DNA Movements in Escherichia coli.
              </a>
            </td>
          <td>
            J. Ord, Zhongge Zhang, Janie Wu, Sofia Smith, Zhenhao Feng, Keisha Ancheta, Luis Fernando Garrido, Ge Yao, Sahil Gupta, Ethan T Lee, Milton H. Saier Jr.
          </td>
          <td>2025-11-25</td>
          <td>Microbial physiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Introduction: During the past two decades, our knowledge about the molecular pathogenesis of myeloid neoplasms (MNs), including acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) has dramatically been improved through the identification of major driver alterations using next generation sequencing. However, with the lack of large-scale analysis using whole genome sequencing (WGS) with sufficient sequencing depths, the analyses have mainly focused on the alterations affecting protein-coding sequences, while the role of non-coding alterations and structural variations (SVs) in myeloid leukemogenesis has not fully been investigated. The etiology of mutagenic processes has also been poorly understood. In the current study, we therefore conducted large-scale deep WGS to address these issues.
 Methods: We performed deep WGS with matched normal samples (mean sequence depth 127.0/30.3 for tumor/normal, respectively) for a total of 903 patients with MN (494 AML and 419 MDS), with an additional 263 cases (94 AML and 169 MDS) currently under analysis. Most cases were also analyzed using deep targeted capture sequencing (n=886) of 446 known/putative driver genes. In addition, RNA sequencing was newly performed for nearly half of cases (n=406). Mutation calling was performed using Mutect2 and GRIDSS2, with in-house modifications to control false positivity in the face of frequent contaminations of tumor cells into germline samples.
 Results: We identified 1,159,638 SNVs and 76,842 sInDels with median mutation burden of 0.46 SNVs and 0.030 sInDels per Mb in AML and 0.39 and 0.022 per Mb in MDS. Validation by deep targeted capture sequencing estimated a true-positive rate of 95% and 78.1% sensitivity for known drivers. In the analysis of mutational signatures, we found that clock-like signatures SBS1 and SBS5 were predominant. SBS19 and SBS32 were detected in 62%/14% and 68%/14% of AML and MDS cases, respectively. As expected, chemotherapy related signatures (SBS25, 31 and 99) were significantly enriched in cases with therapy-related MN cases. The SBS18 signature, which is implicated in oxidative stress–induced mutagenesis, was highly enriched in AML cases carrying t(8;21) and inv(16), but rarely found in those with MDS, highlighting the role of oxidative stress in core binding factor leukemia. Other novel signatures such as SBS24, 39 and 40b were detected in a small number of cases (~1%).
 Our study of a large cohort of MNs also revealed novel candidate driver genes. By evaluating mutation enrichment, we identified a total of 82 candidates of driver genes. Among these, 15 genes were not previously reported as driver genes in MNs, including 7 genes listed in COSMIC cancer gene census database as associated with other cancer types. RNA-seq enabled the identification of intronic mutations predicted to cause alternative splicing in known driver genes, including TET2 and DNMT3A, as well as in novel candidates, observed in 2.1% of cases. In addition to coding genes, mutations in non-coding RNAs with potential driver role (e.g. seed region of mir-142) were detected in 3.3% of the cases.
 We identified a total of 11,050 SV events, of which 48 disrupted common tumor suppressor genes, such as RUNX1, ETV6, CBL, and TP53. Chromothripsis events were detected in 5.3% of cases, which were highly enriched in TP53-mutated cases and frequently affected KMT2A, ETS1, ETV6, TP53, EPOR and ERG. Median of 4 chromosomes were affected in a single chromothripsis event.
 We detected a total of 340 fusion events in 22.6% of all cases, of which 105 fusions involved previously known targets, whereas the remaining 235 represented non-recurrent in-frame gene fusions detected in 11.7% of cases. Although such fusion events were significantly associated with TP53 mutations, 5.6% of TP53-wild-type cases also had in-frame gene fusions, some of which were suggestive of oncogenic potential (e.g. KAT6A::NUTM1). Although low in frequency (1.1%), analysis of expression change revealed SVs affecting cis-regulatory elements including potential enhancer hijacking of a RAS pathway gene.
 Conclusion: Through deep WGS of a large cohort of MN cases, we have delineated a comprehensive landscape of driver mutations and detected new mutational signatures and candidates for driver mutations, including those affecting non-coding regions, underscoring the importance of WGS for better understanding of the pathogenesis of MNs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4576f85031bf75aa25ad786753882bdfc5ce2543" target='_blank'>
              Repertoire of driver events in coding and non-coding regions identified by whole-genome sequencing of myeloid neoplasms
              </a>
            </td>
          <td>
            Koji Okazaki, R. Saiki, Y. Ochi, Kenichi Yoshida, K. Katayama, Hidehito Fukushima, Kaito Mimura, M. Nakagawa, Masanori Motomura, A. Yoda, R. Okuda, Lanying Zhao, Motohiro Kato, Tatsuki Ogasawara, Shigeo Fuji, Lee-yung Shih, N. Uoshima, Mitsumasa Watanabe, Yasushi Miyazaki, M. Ichikawa, Takeshi Maeda, Toshiyuki Kitano, Satoshi Yoshihara, Kazunori Imada, Kinuko Mitani, J. Jansen, M. Sakata-Yanagimoto, Luca Malcovati, Akihiko Gotoh, Hitoshi Kiyoi, N. Kanemura, N. Sezaki, N. Hiramoto, S. Kasahara, S. Imoto, A. Takaori-Kondo, Y. Nannya, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Chromosomal loops are CTCF–cohesin-mediated 3D genomic structures in the mammalian nucleus. In addition to these CTCF–cohesin loops, other epigenomic features can also form long-range interactions. Polycomb-targeted loci, regulated by the Polycomb repressive complex and trithorax group during development, form long-range chromatin interactions independent of CTCF–cohesin and are demarcated by regions of low DNA methylation—referred to as DNA methylation canyons.
 We previously identified extremely long Polycomb loops occurring between DNA methylation Canyon demarked Polycomb-targeted loci, spanning distances of up to 60 Mb (Zhang et al, 2020). These loops are found exclusively in self-renewing cells, such as human hematopoietic stem cells and mouse embryonic stem cells.
 In hematopoietic malignancies, both DNA methylation and Polycomb binding are significantly altered. To explore this further, we conducted a pan-cancer survey of long Polycomb loops across 223 tumor samples, with a focus on hematopoietic malignancies—given that their likely cell-of-origin, the hematopoietic stem cell, exhibits strong long Polycomb loops. Our cohort included 32 acute myeloid leukemias (AMLs), 24 T-cell lymphoblastic leukemias, 61 pediatric brain tumors, 80 prostate cancers, and 26 colon cancers. We found that most cancers—including all prostate and colon cancers—lack long Polycomb loops. However, long Polycomb loops are retained in pediatric brain tumors and a subset of AMLs. Interestingly, strong long Polycomb loop interactions are observed in normal developing brain tissue and hematopoietic stem cell. Our data suggest that the presence of long Polycomb loops is likely inherited from the epigenomic state of the cell of origin, as they are also observed in normal pediatric brain tissue and hematopoietic stem cells.
 Loss of long Polycomb loops in primary cancer samples is accompanied by DNA hypermethylation and reduced Polycomb binding at loop anchor loci, and these disruptions are further exacerbated in cultured cell lines. Interestingly, in AML, many of these previously silenced loci—often mesodermal transcription factors—become activated and form de novo 3D interaction anchors. For example, we observed the formation of a new domain around the leukemogenic ZEB2 gene, driven by leukemia-specific HOXA9 binding at an upstream enhancer within the TEX41 locus.
 Notably, AML samples displayed a wide range of long Polycomb loop strength. While most AMLs lose these loops, approximately 12% (4 out of 33) retain strong long Polycomb loops comparable to hematopoietic stem cells. These cases recurrently harbor somatic mutations in CEBPA (2 out of 4) and STAG2 (2 out of 4). These genes are not directly linked to Polycomb or DNA methylation machinery, suggesting alternative mechanisms of loop maintenance. The CEBPA mutant AMLs also exhibited H3K27me3 spreading to non-Polycomb target loci across the genome.
 We therefore tested whether EZH2 inhibition could disrupt long Polycomb loops in these AMLs and impact disease maintenance. Indeed, EZH2 inhibition attenuated long Polycomb loops, reduced colony-forming capacity, and promoted differentiation in long Polycomb loop–retaining AMLs. Treated AML cells activated a macrophage differentiation program and showed downregulation of cell cycle and DNA replication genes. These findings suggest that AMLs retaining long Polycomb loops are dependent on this 3D chromatin architecture. This rare but strong sensitivity to EZH2 inhibition in AML indicates that long Polycomb loops could serve as an epigenomic biomarker for EZH2 or other Polycomb-targeted therapies in cancer rather than the current genomic marker such as EZH2 gain-of-function mutations and SWI/SNF loss-of-function mutations.
 Overall, long Polycomb loops are commonly lost during leukemogenesis due to epigenomic disruption. However, a subset of AMLs maintains these loops from the cell of origin and appears to rely on the Polycomb network to sustain self-renewal.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2c56d775b22a7b9218367e039106ed0000301e" target='_blank'>
              Pan-leukemia 3D genomic profile reveals extreme long polycomb loop in subset of CEBPA mutated AMLs sensitive to EZH2 inhibition
              </a>
            </td>
          <td>
            Sean Moran, M. Zanovello, Zhong Fan, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retrovirus K (HERV-K) proviruses of the HML-2 subgroup are the most recently integrated retroviral elements within the human genome. The HERV-K Rec protein, a functional homolog of HIV Rev, is essential for the expression of viral mRNAs with retained introns. However, their diversity and functional capacities have remained largely unexplored. We identified Rec coding sequences in the human genome and selected intact Rec proteins from 28 proviral loci for functional characterization. Using a dual-color fluorescent reporter containing the HERV-K Rec-response element and a complementary enzyme-linked immunosorbent assay, we found that Rec proteins from nine proviral genomic loci promoted function at the post-transcriptional level. All but one of these proviruses are insertionally polymorphic in the human genome. In addition, several of the non-functional Rec proteins were trans-dominant negative. Detailed mutational analysis of the most potent inhibitory variant, encoded by the HERV-K provirus 12q14.1, displayed only two amino acid changes (N2H and E34del) relative to the prototypical functional Rec protein encoded by the reconstructed consensus HERV-K. Insertion of a glutamic acid at position 34 in this mutant fully restored the functional activity. Our results unveil an unexpected complexity in HERV-K (HML-2) post-transcriptional regulation, with a few Rec variants showing function, while others are trans-dominant negative. These findings significantly expand our understanding of the various Rec proteins that can be expressed in human cells and how they function at the post-transcriptional level. IMPORTANCE In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects. In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42022bba88ac63086a6e5b60150a1271cd5595a" target='_blank'>
              Characterization of HERV-K (HML-2) Rec proteins encoded in the human genome and their post-transcriptional function
              </a>
            </td>
          <td>
            Katarzyna Zurowska, Godfrey Dzhivhuho, David Grabski, D. Rekosh, M. Hammarskjold
          </td>
          <td>2025-11-18</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-25</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Reversine is a small-molecule Aurora kinase inhibitor known for its pro-apoptotic effects and potential to remodel chromatin architecture. Although its impact on mitotic regulation is established, its effects on telomere dynamics and nuclear organization in chronic myeloid leukemia (CML) remain unclear. This study aimed to investigate the effects of reversine on telomere architecture, genomic instability, and apoptosis in CML cell lines (K-562 and MEG-01). Reversine was applied at increasing concentrations, and cytotoxicity was assessed using caspase-3/7 activation assays. Quantitative PCR was used to measure AURKA and AURKB mRNA expressions. Three-dimensional telomere architecture was analyzed with TeloView® v1.03 software after Q-FISH labeling to quantify telomere number, signal intensity, aggregation, nuclear volume, and a/c ratio. Reversine induced a dose- and time-dependent apoptotic response in both cell lines and significantly downregulated AURKA and AURKB expressions. Three-dimensional telomere analysis revealed a marked reduction in telomere number and aggregates, signal intensity, and nuclear volume. While reduced signal intensity may indicate telomere shortening, the concurrent decrease in aggregation and altered spatial parameters suggests telomeric reorganization rather than progressive instability. These features reflect structural nuclear remodeling and early apoptotic commitment. Differences between K-562 and MEG-01 responses underscore potential heterogeneity in telomere maintenance mechanisms. Reversine modulates genomic stability in CML cells through dual mechanisms involving Aurora kinase inhibition and telomere architecture remodeling. The integration of 3D telomere profiling highlights reversine’s potential as a therapeutic agent targeting nuclear disorganization and mitotic dysregulation in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e06bedd101bf31c7285fbbc7d8f35b57bf8952e" target='_blank'>
              Reversine-Induced Telomere Architecture Remodeling in Chronic Myeloid Leukemia Cell Lines: Insights from TeloView® Analysis of 3D Nuclear Architecture
              </a>
            </td>
          <td>
            Fábio Morato de Oliveira, Isabela Dias Cruvinel, Bruno Machado Rezende Ferreira, Sabine Mai
          </td>
          <td>2025-10-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ab2b82b250c7fe0872534fbb6e35f7aaf3d36d" target='_blank'>
              Cell State Chaos Underpins the Evolution of SMARCA4-Deficient Dedifferentiated Endometrial Cancer
              </a>
            </td>
          <td>
            M. Brandon-Coatham, Julia Vassalakis, Hannah Plummer, Jasleen Kaur, Farzaneh Afzali, Zhihua Xu, Guihua Zhang, Jiahui Liu, Wei Wang, Einav Wajsbrot Renert, Tyler T. Cooper, Alan Dimitriev, Cierra Perron, Andrew Garven, Simatsidk Haregu Abebe, David M Berman, Amber L Simpson, G. Wong, Gilles A. Lajoie, Felix K.F. Kommos, Andreas von Diemlin, Sheela A. Abraham, David J.H.F. Knapp, Alan Underhill, M. Koebel, Cheng Han-Lee, Lynne-Marie Postovit
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objective The calcium-dependent activator protein for secretion (CAPS) has emerged as a protein of interest in tumor biology; however, its functional significance in endometrial carcinoma (EC) remains largely unexplored. Methods We performed an integrative analysis leveraging bulk RNA sequencing data from The Cancer Genome Atlas, GTEx, and publicly available EC datasets, coupled with gene set enrichment analysis (GSEA) and single-cell transcriptomic profiling. The association between CAPS expression and clinicopathological parameters as well as patient prognosis was systematically assessed. In addition, functional enrichment, intercellular communication, and immune infiltration analyses were conducted to elucidate the molecular and microenvironmental roles of CAPS. In vitro assays were employed to validate the biological effects of CAPS knockdown on EC cell proliferation, migration, and invasion. Results CAPS was significantly upregulated in early-stage, low-grade, and endometrioid endometrial carcinoma, and its elevated expression was associated with improved patient survival. GSEA indicated that high CAPS expression was correlated with enhanced ribosome biogenesis and oxidative phosphorylation, along with suppression of proliferative signaling, reflecting a metabolically differentiated phenotype. Single-cell transcriptomic analysis revealed that CAPS was specifically enriched in terminally differentiated secretory epithelial cells, promoting epithelial maturation and oxidative metabolism. Furthermore, CAPS-positive cells exhibited extensive crosstalk with immune cells, suggesting a potential role in facilitating anti-tumor immune responses. Functional assays confirmed that CAPS knockdown markedly enhanced EC cell proliferation, migration, and invasion. Conclusion CAPS may act as a tumor suppressor in EC by stabilizing microtubule architecture, reprogramming cellular metabolism, and modulating immune interactions within the tumor microenvironment. These findings highlight CAPS as a promising prognostic biomarker and potential therapeutic target in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/986f64ba61a7288a4dba4bccd0822c838c05f3b8" target='_blank'>
              Pan-cancer multi-omics characterization of calcyphosine and its revealed links to the immune microenvironment and regulatory networks in endometrial carcinoma
              </a>
            </td>
          <td>
            Xi Chen, Ruiyue Wu, Zijing Wang, Fengfeng Wang, Dan Hu, Xue-han Bi, Xiaolei Liang, Yongxiu Yang
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Identifying homologous cell types across species is essential for understanding cell type evolution. The retina is ideal for comparative analysis because its six major cell classes have persisted since the origin of vertebrates more than half a billion years ago. Here, we show that the retina’s conserved cellular architecture is mirrored by deep conservation of the cis-regulatory codes that govern gene expression. Through single-cell chromatin accessibility analysis of lamprey, fish, bird, and mammalian retinas, we demonstrate cross-species conservation of cis-regulatory codes in all retinal cell classes despite extensive turnover of cis-regulatory regions. Conservation manifests as clustering of high-affinity transcription factor binding sites in cell class–specific open chromatin regions. Thus, the retina’s cellular Bauplan is controlled by cis-regulatory codes, which predate the divergence of extant vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7409622936e88e77d72521a01e9a055cd5634334" target='_blank'>
              Conservation of cis-regulatory codes over half a billion years of evolution
              </a>
            </td>
          <td>
            Yohey Ogawa, Yu Liu, Connie Myers, Ala Morshedian, Gordon L. Fain, A. Sampath, Joseph C. Corbo
          </td>
          <td>2025-12-12</td>
          <td>Science Advances</td>
          <td>5</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Super-enhancers (SEs) are key DNA cis-regulatory elements that play a central role in regulating tissue/cell-specific gene expression, thereby maintaining cellular identity and function. SEdb 3.0 (http://www.licpathway.net/sedb) provides an extensively updated resource of SEs and their regulatory annotations across multiple species. The current version of SEdb now curates 3 478 186 SEs from 5387 H3K27ac ChIP-seq samples across four species. Compared to SEdb 2.0, it has achieved a two-fold expansion in human and mouse SE entries while newly incorporating Arabidopsis thaliana and maize data, significantly enhancing both the database's scale and its utility in plant research. Furthermore, abundant (epi)genomic features have been added, such as enhancer RNAs (eRNAs), binding sites of transcription co-factors (TcoFs), and chromatin regulators (CRs). The inclusion of eRNAs provides insights into SE transcriptional activity. Mapping TcoF binding sites highlights their roles in mediating enhancer-promoter looping and stabilizing transcriptional complexes at SEs. The integration of CRs uncovers how SEs are associated with histone modifications and chromatin remodeling, which are critical for maintaining an open chromatin state. Collectively, these annotations not only reveal the diverse mechanisms by which SEs exert regulatory functions but also enable more detailed investigations into their biological significance and functional roles. Meanwhile, existing annotations have been substantially expanded, such as an approximately five-fold increase in transcription factor (TF) ChIP-seq data, a 2.3-fold rise in TF motifs, and a roughly 1.8-fold growth in SE-associated eQTL-gene regulatory pairs. SEdb 3.0 introduces two advanced inference strategies for associating genes with SEs. Moreover, two newly developed analysis tools are provided in SEdb 3.0, including SE blast alignment analysis and SE-driven core TF enrichment analysis. In summary, SEdb 3.0 represents a significant upgrade over SEdb 2.0, with a substantial expansion in SE coverage across multiple species, alongside enhanced functional annotations encompassing SE upstream/downstream regulatory information, thereby offering a more comprehensive and user-friendly platform for exploring the biological roles of SEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c473d9211ee5f186ffb574ffbc44abe478a16ce8" target='_blank'>
              SEdb 3.0: a comprehensive super-enhancer database across multiple species.
              </a>
            </td>
          <td>
            Shuang Song, Liyuan Liu, Chenchen Feng, Liyuan Xie, Guorui Zhang, Yuexin Zhang, Yichen Gao, Mingxue Yin, Xiuyun Tang, Wenya Pei, Chao Song, Runping Liu, Chunquan Li
          </td>
          <td>2025-12-08</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) is a powerful isothermal strategy for nucleic acid detection, but its linear kinetics and dependence on externally supplied primers limit its sensitivity and programmability. Here, we report an exponential RCA (E-RCA) platform that integrates primer regeneration and signal amplification into a single DNA-encoded system. The design uses a circular DNA template encoding the I-R3 self-cleaving DNAzyme sequence; during RCA, tandem I-R3 units are generated within the DNA amplicons, which then catalyze site-specific cleavage to release new primers. This self-sustained (RCA ↔ DNAzyme) amplification circuit enables robust exponential signal growth using only a circular DNA probe and a DNA polymerase without requiring external primers or protein enzymes. We elucidate the mechanism through biochemical experiments and kinetic modeling and validate the system in multiplexed intracellular microRNA imaging and quantitative dual-marker profiling of clinical breast cancer tissues. The E-RCA strategy achieved high diagnostic accuracy (AUC = 0.914; specificity = 100%; sensitivity = 81.3%), demonstrating its potential for sensitive, programmable, and autonomous molecular analysis in both biological and clinical contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779a0bfc06573a96632b1849e24d829296851cc1" target='_blank'>
              Autonomous Exponential Amplification via Rolling Circle-DNAzyme Feedback Programming.
              </a>
            </td>
          <td>
            Jinhua Shang, Mengdi Yu, Yifei Wang, Shanshan Yu, Xianzheng Zhang, Fuan Wang, Yingfu Li
          </td>
          <td>2025-11-26</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Efficient proteostasis and immune evasion are both critical for tumor progression. The chaperonin TRiC/CCT complex, which mediates the folding of cytoskeletal and signaling proteins, has been associated with oncogenesis; however, the specific role of its subunit CCT4 in tumor–immune interactions remain unclear. Methods To address this gap, we integrated transcriptomics, proteomics, genomics, epigenetics and immunogenomics data. A comprehensive pan-cancer analysis was conducted (including the expression patterns, clinical relevance, prognosis value, immune infiltration of pan-cancer). Then an in-depth analysis of lung adenocarcinoma (LUAD) was carried out through enrichment analysis and single-cell RNA sequencing, and verified through in vitro cell experiments. Results CCT4 was found to be aberrantly upregulated across a majority of tumor types, particularly in LUAD, where elevated expression was associated with advanced stage and inferior survival outcomes. High CCT4 levels were linked to reduced immune cell infiltration and diminished anti-tumor immune signaling, specifically manifested as increased Th2 cell infiltration and decreased Th1 and CD8+ T-cell signatures. Single-cell analyses revealed coordinated overexpression of all CCT subunits in tumor epithelial cells, supporting a global TRiC activation. However, CCT4 was preferentially enriched within highly proliferative subclusters, suggesting partial subunit-specific regulation. CCT4 knockdown suppressed LUAD cell proliferation, migration, and invasion in vitro. Conclusions CCT4 links enhanced proteostasis with immune evasion in LUAD, acting partly through TRiC complex activity and possibly through independent nuclear functions. These findings refine the understanding of how proteostatic machinery contributes to immune modulation in cancer and highlight CCT4 as a potential molecular node bridging tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4892bb2a6429b687729b7113a17a2bf3b326e37" target='_blank'>
              Pan-cancer multi-omics analysis of CCT4 in tumor progression and cancer immunity, with focus on lung adenocarcinoma
              </a>
            </td>
          <td>
            Yonghui Wu, Zhenpeng Wu, Junjie Wen, Desheng Zhou
          </td>
          <td>2025-12-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 In T-cell acute lymphoblastic leukemia (T-ALL), LMO2 (Lim domain only 2) is frequently overexpressed due to chromosomal translocations or epigenetic changes. LMO2 is a molecular scaffold connecting LDB1 (Lim domain binding 1) to DNA-binding transcription factors. LDB1 is a major chromatin architectural factor involved in long-range enhancer-promoter interactions. Given the prevalence of aberrant LMO2 expression in T-ALL, we hypothesized that LDB1 promotes T-ALL proliferation/survival by connecting oncogenic enhancers to their target genes.
 We performed LDB1 ChIP-seq in 2 T-ALL cell lines with high LMO2 expression: LOUCY (early thymic precursor [ETP]; SET::NUP214) and KOPT K1 (non-ETP; TCRG::LMO2). LDB1 is predominantly recruited to cis-regulatory elements, with 40-60% of peaks localizing to enhancers. However, LDB1 enhancer occupancy differed significantly between cell lines, suggesting subtype-specific regulation of transcriptional programs. Indeed, motif enrichment analysis at LDB1-occupied enhancers demonstrated distinct recruiting factors in LOUCY versus KOPT K1. To enable rapid LDB1 degradation, we tagged endogenous LDB1 with dTAG in both cell lines. Treatment with dTAG-V1 ligand for 4hrs depleted LDB1 by >90%. To assess LDB1-dependent enhancer-promoter connectivity, we treated cells with dTAG-V1 for 4hrs and performed Micro-C sequenced to a depth of ~1 billion valid contacts. Globally, compartmentalization and topologically associated domains remained largely intact in the absence of LDB1. However, we observed widespread changes to loops involving LDB1-occupied enhancers and/or promoters. Importantly, histone modifications and chromatin accessibility at enhancers remained intact upon acute LDB1 depletion, suggesting that architectural changes were directly due to loss of LDB1's looping function rather than indirect consequences of enhancer loss.
 To explore how architectural changes impact gene expression, we measured nascent transcription using TT-seq upon acute LDB1 depletion and observed 328 and 134 differentially expressed genes in LOUCY and KOPT K1, respectively. LDB1-dependent transcripts in LOUCY included critical hematopoietic oncogenes (HHEX, MYB, MYCN, ERG, MEF2C) while those in KOPT K1 were enriched in immune activation pathways (DUSP6, BACH2, CD69). Reduced transcriptional output at these loci was associated with weakened enhancer-promoter connectivity. We analyzed 26 H3K27ac HiChIP datasets from patients with T-ALL and found that LDB1-dependent loops at these oncogenic loci are recapitulated and often correlate with T-ALL differentiation state. These findings suggest that LDB1 connects enhancers with oncogenic driver genes in both cultured and primary T-ALL cells.
 To determine if LDB1 is required for T-ALL proliferation/survival, we passaged LOUCY and KOPT K1 cells in the presence of dTAG-V1 for 15 days and observed loss of cell expansion. Loss of LDB1 in LOUCY induced apoptosis, while in KOPT K1 it elicited a senescent phenotype characterized by beta-galactosidase activation, cell cycle exit, and senescence-associated gene expression. To determine which gene expression changes accounted for these phenotypes, we restored expression of several LDB1-dependent genes in LDB1-depleted cells using CRISPRa. Reactivation of oncogenes like MYB partially rescued cell expansion, suggesting that LDB1 dependency in T-ALL may in part be conveyed through MYB.
 Finally, we compared LDB1-dependent transcriptional signatures from LOUCY and KOPT K1 to transcriptomic data from >1300 patients with T-ALL. The LOUCY signature was enriched in several high-risk T-ALL subtypes, including the recently defined ETP-like and LMO2 γδ-like entities. The KOPT K1 signature was conversely enriched in more differentiated subtypes such as TAL1/DP-like, together suggesting that LDB1 may be required to maintain transcriptional identity of T-ALL subtypes in vivo. Notably, LOUCY signature enrichment conferred worse overall survival (p=0.026) and event-free survival (p=0.00032), driven in part due to LDB1 regulation of high-risk transcriptional features in the ETP-like subtype.
 Together, our study highlights LDB1 as a critical architectural factor that connects T-ALL subtype-specific enhancers and promoters to regulate transcriptional programs required for T-ALL cell expansion. Analyses of patient-derived data corroborate our findings and implicate the prognostic and therapeutic value of LDB1 regulatory networks.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb526de9e9a759ff10b0913b39b5e0ac7836593" target='_blank'>
              LDB1-dependent enhancer networks drive high-risk transcriptional programs in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Rahul Bhansali, Juan Long, Petri Polonen, Shuo Zhang, Mariah C. Antopia, Siqing Wang, Nicholas G. Aboreden, Ahnaf Tausif, Zhuangzhuang Geng, Cheryl Keller, B. Giardine, Ross Hardison, C. Mullighan, Gerd Blobel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Y. Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 TET2 mutations (TET2MT) are among the most frequent somatic lesions in myeloid malignancies, found at ~30% in MDS/AML and over 66% in CMML. TET2MT is also common in clonal hematopoiesis of indeterminate potential (CHIP), a pre-malignant condition that increases the risk of myeloid neoplasms (MN) and other diseases including treatment response in certain solid tumors. However, TET2MT alone is insufficient for malignant transformation, as they are frequently observed in CHIP without apparent disease. Thus, strongly suggesting the presence of additional mechanisms of lineage perturbation and evolution to MN. Chromodomain Helicase DNA-binding Protein 2 (CHD2) is an ATP-dependent DNA helicase that modulates gene expression via nucleosome repositioning. Despite its essential role in transcriptional fidelity and genomic integrity, CHD2 function remains poorly characterized in hematopoiesis and leukemogenesis. Here we report that CHD2 directly interacts with TET2 and co-occupies chromatin regulatory sites to maintain epigenetic homeostatic hematopoiesis. In the absence of TET2, CHD2 is co-opted to sustain oncogenic transcriptional programs that skew differentiation to myeloid precursors and prevent differentiation seen in TET2MT AML.
 Methods. CRISPR-Cas9 was used to generate clonal isogenic TET2 knockout (TET2KO) and TET2WT cells derived from K562 and THP1. Endogenous immunoprecipitation (IP) of CHD2 and TET2 was performed to purify associated protein complexes, followed by analysis and identification of the CHD2-TET2 interactome with high performance liquid chromatography coupled with tandem mass spectrometry (LCMS/MS). To investigate functional consequences of TET2 loss on CHD2 activity, we conducted integrative multi-omics analysis combining proteomics, publicly available chromatin immunoprecipitation sequencing (ChIP-seq), and RNA sequencing (RNA-seq). Genome-wide co-occupancy of CHD2 and TET2 was evaluated through comparative analysis of published ChIP-seq datasets.
 Results. Reciprocal IP of TET2/CHD2 in K562 and CMK cells revealed robust interaction between CHD2-TET2. Global proteomic comparisons using LCMS/MS revealed significant overlap between CHD2 and TET2 interactomes. 268 proteins were unique to CHD2 IPs from TET2WT that reduced to 106 in isogeneic TET2KO cells with CHD2 retaining a conserved overlapping interaction network in both cases. Notably, CHD2 IPs were enriched for components of the CoREST complex (RCOR1, RCOR3, GSE1, KDM1A), suggesting a potential role in transcriptional repression. In contrast, SWI/SNF components were particularly enriched in TET2 IP, highlighting TET2's role as a critical hub for chromatin remodeling. TET2 interactome was markedly broader, comprising over 3,000 proteins, including key regulators of polycomb repressive complexes (PRC1/2) and lineage-defining transcription factors like GATA1, GATA2, RUNX1 and STAT5A/B.
 Our analysis of ChIP-seq data showed 95% overlap between CHD2- and TET2-bound genes. RNAseq analysis in matched isogeneic cells revealed that these co-targeted genes are differentially expressed in the TET2KO cells. The oncogenic pathways were enriched among genes upregulated in TET2KO, while RNA processing and chromatin homeostasis genes were down regulated compared to TET2WT. ChIP-seq track analysis showed CHD2 and TET2 binding at promoter-proximal nucleosome-depleted regions, supporting a cooperative regulatory mechanism. These findings suggest that in TET2-proficient cells, CHD2 is directed to key regulatory elements to maintain transcriptional fidelity that is essential for homoeostasis hematopoiesis. In TET2-deficient cells, CHD2 is hijacked to sustain aberrant transcription that may promote myeloid skewing and leukemogenesis.
 Conclusions. For the first time, here we define a previously unrecognized CHD2-TET2 interactome and its potential functional role in the hematopoiesis. This, in part, may provide a probable mechanism of myeloid skewing in TET2MT HSPCs. These findings may provide rational for the dependency of TET2MT clones on CHD2 for malignant evolution. By integrating proteomic, genomic and functional analyses, we elucidate the role of CHD2 in transcriptional control that define lineage fate in the absence of TET2. In addition, we address a significant knowledge gap in CHD2 biology that may form the basis for novel therapeutic approaches targeting epigenetic vulnerabilities in TET2MT associated disorders.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded1853073e3baae3c88026feb94596db68dae6" target='_blank'>
              Chromodomain helicase DNA binding protein CHD2 interacts with TET2 and regulates lineage defining transcription in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Songa Bae, Xiaorong Gu, Mamta Sumi, Dongxu Jiang, Daniel J Vail, Yahan Zhang, Kacey O'Donnell, Yogenthiran Saunthararajah, Jaroslaw P. Maciejewski, B. Jha
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome.
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="DNA repair mechanisms in human primary cells, including error-free repair, and, recurrent nuclease cleavage events, remain largely uncharacterised. We elucidate gene-editing related repair processes using Cleavage and Lesion Evaluation via Absolute Real-time dPCR (CLEAR-time dPCR), an ensemble of multiplexed dPCR assays that quantifies genome integrity at targeted sites. Utilising CLEAR-time dPCR we track active DSBs, small indels, large deletions, and other aberrations in absolute terms in clinically relevant edited cells, including HSPCs, iPSCs, and T-cells. By quantifying up to 90% of loci with unresolved DSBs, CLEAR-time dPCR reveals biases inherent to conventional mutation screening assays. Furthermore, we accurately quantify DNA repair precision, revealing prevalent scarless repair after blunt and staggered end DSBs and recurrent nucleases cleavage. This work provides one of the most precise analyses of DNA repair and mutation dynamics, paving the way for mechanistic studies to advance gene therapy, designer editors, and small molecule discovery. Quantifying genomic aberrations resulting from designer nucleases activity is essential for gene therapy clinical translation. Here, the authors present a modular digital PCR technique that profiles DNA repair precision and cut-repair cycles at the edited loci, exposing current evaluation biases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347cbf692d67c07b13519f5e18f058f81e33be4d" target='_blank'>
              Unveiling the cut-and-repair cycle of designer nucleases in human stem and T cells via CLEAR-time dPCR
              </a>
            </td>
          <td>
            Nathan White, John Alexander Chalk, Yi-Ting Hu, Samuel Mark Pins, Chinnu Rose Joseph, Panagiotis Antoniou, Sandra Wimberger, Stina Svensson, S. Caetano-Silva, Anne Charlotte Adriane Mudde, R. Rai, Sridhar Selvaraj, William N. Feist, Marianna Romito, Grzegorz Sienski, Roberto Nitsch, Claire Booth, G. Santilli, Alessia Cavazza, M. Porteus, Marcello Maresca, A. Thrasher, G. Turchiano
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Dysregulated transcription is a defining hallmark of cancer. Recently, novel chemically induced proximity approaches have enabled the rewiring of transcriptional machinery to drive expression of pro-apoptotic genes using bivalent small molecules. In this work, we demonstrate that this strategy is amenable to relocalizing DNA bound transcriptional machinery, such as fusion transcription factors that commonly drive pediatric malignancies. Targeting fusion transcription factors, such as EWSR1::FLI1 in Ewing sarcoma, with these bivalent compounds may open new therapeutic avenues. Here, we develop a small molecule, EB-TCIP, that recruits FKBP12F36V-tagged EWSR1::FLI1 to DNA sites bound by the transcriptional regulator BCL6, leading to rapid chromatin remodeling and expression of BCL6 target genes. This proof-of-concept study demonstrates that DNA binding proteins with pioneering transcription factor activity, such as EWSR1::FLI1, can be relocalized on chromatin to induce expression of repressed genes. Insights herein will guide the development of future bivalent molecules that rewire DNA binding transcriptional machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f7b6214e9ef05ce0f8a706c26c1ffab5a52705" target='_blank'>
              Rewiring the Fusion Oncoprotein EWSR1::FLI1 in Ewing Sarcoma with Bivalent Small Molecules.
              </a>
            </td>
          <td>
            Michael J Bond, Ryan P. Golden, G. Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, Hannah M Jones, Brendan G. Dwyer, N. Gray, K. Stegmaier
          </td>
          <td>2025-11-27</td>
          <td>Journal of the American Chemical Society</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Transposable elements (TEs) make up half of the human and mouse genome but remain understudied. Evidence is growing that TEs play important roles in immunity as a subset of TE enhancers regulate innate immunity genes. However, the cis-regulatory contribution of TEs across immune cell development is not well understood. Mining more than 300 chromatin accessibility datasets (ATAC-seq) across mouse hematopoietic cell lineages, we identified two subfamilies of rodent-specific retroviral LTRs, ORR1E and ORR1D2 (ODE), which are enriched in accessible chromatin across all immune cell types. Unsupervised clustering show that ∼2,900 ODEs stratify by cell type-specific accessibility. Each ODE cluster is enriched for a unique combination of sequence motifs for cell lineage-specifying TFs, such as PU.1, IRF8, and RORγ, which we validated with ChIP-seq. By correlating ODE accessibility with gene expression (RNA-seq), we predicted hundreds of ODE-gene interactions. We generated Micro-C (DNA-DNA contact) in CD8+ T cells and validated ∼120 ODE-gene interactions. Predicted target genes are more expressed in mouse than in human CD8+ T cells. Lastly, accessible ODEs have higher sequence conservation across rodents than other ODEs. Together, these data suggest that despite their common origin, ODE LTRs have subfunctionalized into cell type-specific enhancers across mouse hematopoiesis. We propose that ODE enhancers were co-opted during rodent evolution to reinforce cell type identity.



 F31AI183775



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c25d5af8adc02b84a26acb002fe252a06d958df" target='_blank'>
              Functionalization of transposable element-derived enhancers across mouse hematopoiesis 3835
              </a>
            </td>
          <td>
            Jason D. Chobirko, Elizabeth A. Fogarty, Cedric Feschotte, Andrew Grimson
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28230d8645abe62a48dfc936c6a5bcaf5f9746da" target='_blank'>
              Optogenetic tripwires resolve models of how cells count DNA breaks
              </a>
            </td>
          <td>
            Marco Labagnara, Vojislav Gligorovski, S. Rahi
          </td>
          <td>2025-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Foxp3+ regulatory T (Treg) cells rely on DNA demethylation to establish and maintain the gene expression program that defines their identity and suppressive function. Although sustained transcription of Treg-specific genes is known to drive this demethylation, the precise mechanism has remained unclear. Here, we show that Dot1L-catalyzed methylation of histone H3 at lysine 79 (H3K79me) is essential for Treg-specific DNA demethylation in both thymic and induced Treg lineages. H3K79me promotes chromatin activation and recruits TET family DNA demethylases to key regulatory loci. Treg-restricted deletion of Dot1L disrupts locus-specific DNA demethylation, diminishes expression of core Treg genes, and precipitates a fatal, early-onset autoimmune syndrome. Conversely, pharmacologic enhancement of TET activity during differentiation rescues DNA demethylation, restores Treg-gene expression, and reinstates suppressive function even in the presence of Dot1L inhibition. Together, these findings identify a critical epigenetic axis—Dot1L-mediated H3K79 methylation driving TET-dependent DNA demethylation—that safeguards Treg cellular identity, function, and immune homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6954d6e8b9f31304e6906e32eee0a0c357cc099c" target='_blank'>
              Dot1L licenses DNA demethylation to establish regulatory T cell identity
              </a>
            </td>
          <td>
            James Cameron, Tyler R. Colson, Xudong Li
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Eukaryotic DNA has been covalently modified by DNA methylation and folded into a three-dimensional conformation in the nucleus. While the functions of DNA methylation and chromosome organization have been widely discussed, respectively, the interplay between DNA methylation and chromosome organization remains less clear and needs to be further explained. In this review, we first discuss the cross-talk between DNA methylation and chromosome conformation, highlighting the complexity and importance of DNA methylation on chromosome organization. We also summarize the current methodologies that capture DNA methylation and chromosome organization individually or simultaneously in bulk and single cells. These mechanistic and methodological advancements facilitate broad interest in unveiling the interplay between DNA methylation and chromosome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1272377f23956ec52130bb31428bff7433f3f0" target='_blank'>
              Unraveling the interplay of DNA methylation and chromosome organization.
              </a>
            </td>
          <td>
            Yuhe Pei, Guoqiang Li
          </td>
          <td>2025-11-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171126bf14024769a7ffeadfaccab1f7a77e1513" target='_blank'>
              Mitochondrial genome microhomology-mediated editing by donor DNA delivery into mitochondria in human cells
              </a>
            </td>
          <td>
            Vadim V. Maximov, N. Shebanov, Natalia Nikitchina, Rachel Rapoport, Y. Maor, I. Tarassov, O. Pines, N. Entelis
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>34</td>
        </tr>

        <tr id="


 Introduction: Ribosomal DNA (rDNA) encodes ribosomal RNA (rRNA), which undergoes processing to become ribosomes. Although rDNA dysregulation has been observed in various pathologies, comprehensive epigenetic profiling in cancer has been limited due to its repetitive sequence. To overcome this challenge, a customized genome has been developed to map human and mouse rDNA using high-throughput sequencing data. Using this genome and more than 2,200 ChIP-seq datasets, Antony et al performed a systematic analysis to identify the transcription factors, e.g., C/EBP-A, that support rRNA production in hematopoietic cells. Our previous publications suggested that DNA cytosine modifications influence genomic binding of C/EBP. Accumulated studies, including our own, have reported aberrant DNA methylation and hydroxymethylation in myeloid malignancies. Yet, it remains unclear whether the leukemia-associated DNA methylation dysregulation extends to genomic regions encoding rRNA (rDNA), and how these epigenetic changes in rDNA affect downstream targets.
 Methods: To address these critical knowledge gaps, we employed innovative rDNA assembly methods, along with a DNA methylation analysis pipeline, to evaluate the whole-genome bisulfite sequencing (WGBS) data from 136 cancer patients covering 9 cancer types, aiming to uncover DNA methylation changes in rDNA in cancer. To further investigate unique patterns observed in hematological malignancies, we expanded our analysis to a dataset containing DNA methylation profiles from 292 AML patients. Additional AML datasets were analyzed to validate our initial findings with integrative methylation analysis methods. Furthermore, we analyzed published nanopore long-read data to independently confirm the methylation status of rDNA.
 Results: Our pan-cancer analysis revealed several patterns of rDNA methylation across cancer types, often at odds with non-rDNA methylation patterns. Intriguingly, we discovered that hematological malignancies exhibited distinct DNA methylation signatures in rDNA, with higher DNA methylation in the rDNA-coding region compared with other cancer types. This finding was validated in multiple independent datasets totaling 484 AML samples. Additionally, long-read nanopore sequencing verified these methylation patterns across the entire rDNA tandem array. Further characterization of this hypermethylation in AML patient samples revealed CpG loci that show highly conserved changes. A significant correlation was observed between hypermethylation at these loci and decreased time-to-relapse, suggesting that increased rDNA coding region methylation may serve as a biomarker. Interestingly, along with prognostic significance related to time-to-relapse, we revealed distinct patterns of rDNA methylation in patients treated with hypomethylating agents (HMA). To study potential mechanisms, we first validated published findings that many transcription factors exhibit consistent binding patterns on rDNA. To elucidate the connection between DNA methylation and transcription factor binding within rDNA, we first performed a motif analysis of the rDNA sequence, revealing 4000+ canonical binding motifs. We then used our large AML datasets to identify motifs with differential methylation between healthy bone marrow and AML patients, both at diagnosis and relapse. Across both datasets, we identified several potential transcription factors, e.g., CEBP families, for downstream analysis. We will use multiple leukemia cell lines to validate the correlation between DNA methylation and TF binding at rDNA. We will further use catalytically dead Cas9 (dCas9) mediated epigenome editing to probe the causal relationship to support our hypothesis that rDNA methylation can influence TF binding.
 Conclusions: Our data revealed distinct DNA methylation signatures in rDNA specific to myeloid leukemia. Using multiple published DNA methylation datasets from AML patients, we identified unique DNA methylation changes in rDNA, highlighting its potential diagnostic and prognostic value. At the molecular level, we observed differential DNA methylation levels within several TF binding motifs, which might alter TF binding and impact their transcriptional activity in rDNA, ultimately contributing to aberrant ribosomal biogenesis. Overall, our study provides a comprehensive analysis of previously neglected DNA methylation changes in rDNA in AML, uncovering novel epigenetic mechanisms with potential clinical relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba5a2d1360b7125af2d5a2084348209fa94e181" target='_blank'>
              Hypermethylation of the ribosomal DNA coding sequence and disordered transcription factor binding as hallmarks of acute myeloid leukemia
              </a>
            </td>
          <td>
            James Wengler, Y. Huang, Leng Han, Lei Guo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afde2671402c0ec98e756c36e5ab2f520fb96100" target='_blank'>
              Long read nanopore DNA sequencing with adaptive sampling to identify tyrosine kinase fusion genes.
              </a>
            </td>
          <td>
            Matthew Salmon, Nicole Naumann, J. Rinke, M. Meggendorfer, D. Radia, Mark Pomfret, Thomas Ernst, A. Hochhaus, A. Reiter, William J. Tapper, Helen E. White, Nicholas C. P. Cross
          </td>
          <td>2025-11-18</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant childhood cerebellar tumor comprising molecularly and clinically distinct subgroups. Groups 3 and 4-MB originate from overlapping progenitor pools of the developing rhombic lip when neuronal differentiation hierarchies are subverted through somatic alterations. Mutations suspected to deregulate the activity of chromatin-modifying genes are pervasive in Group 3/4-MB. However, molecular consequences of these alterations remain largely undefined. We characterized the chromatin landscape of fifty-two primary MBs using chromatin immunoprecipitation followed by sequencing (ChIP-seq) for six histone modifications and performed multi-modal integration with sample-matched somatic lesions, genome-wide DNA methylation, and transcriptomic profiles. Analysis of differential chromatin states across MB subgroups revealed significant enrichment of the bivalent enhancer state (EnhBiv; marked by coincident H3K4me1 and H3K27me3) in Group 3/4-MB overlapping neurodevelopmental genes. Integrative bioinformatics coupled with CUT&RUN analysis revealed significant enrichment of KDM2B, a histone lysine demethylase overexpressed in Group 4-MB, in gene promoters marked by the EnhBiv state. CRISPR gene targeting or targeted protein degradation of KDM2B selectively suppressed growth of Group 3 and Group 4-MB models in vitro and in vivo, which was dependent on KDM2B DNA-binding domain but not demethylase activity. Acute and chronic KDM2B degradation impaired EZH2 recruitment through direct physical interactions and indirect KDM2B-mediated PRC1 activity, reducing H3K27me3 deposition and chromatin bivalency, while profoundly derepressing PRC2 targets involved in neuronal differentiation. Our comprehensive characterization of the MB chromatin landscape in a large cohort of primary tumors provides unprecedented insights into the epigenetic basis of MB subgroups, implicating novel mechanisms of tumorigenesis, and disclosing a unique subgroup-specific dependency conferred by KDM2B-mediated Polycomb activity that warrants consideration as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff433ac7a60b68bf5a2b5c1d8d57a287494ba1b0" target='_blank'>
              EPCO-22. DIVERGENT MEDULLOBLASTOMA CHROMATIN STATES DISCLOSE KDM2B AS A SELECTIVE DEPENDENCY
              </a>
            </td>
          <td>
            Ran Tao, Serap Erkek-Ozhan, Beisi Xu, Yurika Matsui, Priya Mittal, Kyle S Smith, Yiran Li, Qingsong Gao, Emily Darrow, Ruijie Xu, Nadhir Djekidel, Richa Bajpai, Rahul Kumar, Colleen Reilly, T. Soliman, Jennifer L. Hadley, Melissa Batts, Natarajan Bhanu, B. Gudenas, Leena Paul, Hong Lin, K. Lowe, L. Kutscher, Benjamin Garcia, S. Pruett-Miller, Xin Zhou, Brent A. Orr, G. Robinson, Gang Wu, Jan O. Korbel, Jamy C Peng, Stefan M. Pfister, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 While radiation therapy offers strong anti-cancer benefits, it can also have serious side effects for children undergoing brain tumor treatment. These include disruptions in white matter development, delays in functional and cognitive growth, and an increased risk of secondary tumors within the area that was treated. Age-dependent susceptibility to these adverse effects is tightly linked to critical windows of neurodevelopment, particularly during the periods of active neural cell differentiation, maturation, and the rapid onset of myelination in infancy and early childhood. We previously demonstrated that oligodendrocyte precursor cells (OPCs) exhibit heightened sensitivity to IR-induced DNA damage compared to other neural stem/progenitor cells (NSPC) populations – a difference that is associated with their decreased ability to form RAD51 filaments (a critical step in RAD51-mediated homologous recombination repair of DNA double-strand breaks). Recently, we performed genome-wide chromatin immunoprecipitation followed by sequencing (ChIP-seq) to map endogenous γH2AX binding sites—a sensitive marker of DNA damage response—in both S-phase OPCs and NSPCs. Our data identified 71,794 γH2AX-enriched regions in OPCs versus 29,154 in NSPCs, highlighting the elevated genotoxic stress burden in OPCs. Interestingly, we observed γH2AX binding is depleted at transcription start sites and transcription termination sites, however OPCs show more γH2AX peaks compare to NSPCs across all protein-coding genes. Furthermore, our data show cell type specific γH2AX binding sites. Together, these findings establish a comprehensive and unbiased genomic framework for elucidating the mechanisms underlying OPC vulnerability to DNA damage. Future studies will examine different DNA damage pattern in response to radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d211959973f2a6f48caa461ee162af3afdec9c4b" target='_blank'>
              DNAR-11. Differential DNA Damage and Repair burden in proliferating neural progenitor cells
              </a>
            </td>
          <td>
            Askar Yimit, Peter Brownlee, Sean Nesdoly, Sorana A. Morrissy, Jennifer A Chan, Aaron Goodarzi
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs), constituting approximately 8% of the human genome, represent genomic remnants of ancestral retroviral infections that colonized the germline through evolutionary processes. While most HERVs remain epigenetically silenced, their reactivation through environmental stimuli or epigenetic dysregulation enables participation in oncogenesis via viral mimicry, immunomodulation, and insertional mutagenesis. Substantial evidence now implicates aberrant HERVs activity across urologic malignancies—including prostate cancer, renal cell carcinoma (RCC), bladder cancer, and testicular germ cell tumors—where cancer-type-specific mechanisms drive tumor development and progression. These encompass androgen-responsive HERV-K activation in prostate malignancies, hypoxia-inducible factor-mediated ERV immunogenicity in RCC, HERV-derived microRNA silencing of tumor suppressors in bladder cancer, and DNA hypomethylation-associated HERV expression in testicular germ cell tumors. This review synthesizes fundamental HERV biology with recent advances in their diagnostic and therapeutic applications for urologic neoplasms. Key clinical translations include ERV-based stratification models predicting immune checkpoint inhibitor response in metastatic RCC, HERV-E-targeted adoptive T cell therapies, and noncoding RNA biomarkers for early bladder cancer detection. We further discuss unresolved mechanistic paradoxes such as contradictory prognostic associations between HERV superfamily expression and PBRM1 inactivation in RCC, concluding with priorities for future research: validating HERV-derived neoantigens in immunotherapy platforms, optimizing epigenetic priming strategies to enhance viral mimicry effects, and establishing standardized HERV signatures as clinical biomarkers through multi-institutional cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38383e2f180ac1b254c2d12832458ab1cd318585" target='_blank'>
              The expression, regulation, and function of human endogenous retroviruses in genitourinary cancers
              </a>
            </td>
          <td>
            Wenjie Ma, Chencheng Ji, Abudukelimu Abudushataer, Ning Liu, Tao Xu, Kunlun Zhao, Yiguan Qian, Paerhati Tuerxun, Xiaotian Jiang, Zhongli Xiong, Min Wang, Ruipeng Jia, Zheng Xu, Yang Li, Yu-Zheng Ge
          </td>
          <td>2025-11-28</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The pathogenesis of koala retrovirus (KoRV) has been explored in various contexts, yet its role in tumorigenesis remains incompletely understood. Unlike acute transforming retroviruses, KoRV lacks a viral oncogene but may contribute to oncogenesis via indirect mechanisms. However, the relationship between KoRV and telomere length, as a potential indicator of telomerase activity, has not been examined. This study investigates the effect of KoRV infection on telomere length in 47 samples from Southern Australian koalas in a novel telomere length quantification method. Telomere lengths of 30 KoRV-negative samples were compared to those of 17 KoRV-positive samples using the Absolute Human Telomere Length Quantification qPCR kit (ScienCell Research Laboratories, California, USA). The telomere length in KoRV-infected WBCs was significantly longer than the uninfected ones (t = −2.059, p-value = 0.045). In line with this, telomere length correlated positively with proviral load (r = 0.421, p-value = 0.003), further linking viral burden to telomere elongation. Furthermore, the effect of age on telomere length differed by infection status (β = −5329.7, p-value = 0.0038); KoRV-positive individuals exhibited longer telomeres at a younger age but experienced more rapid telomere attrition over time compared to KoRV-negative individuals. These results suggest KoRV promotes telomerase elongation ability and modulates age-related telomere dynamics, potentially contributing to subsequent cellular immortality and oncogenesis. These pathways may overlap with other retroviruses, where telomerase dysregulation contributes to their oncogenic potential. This study provides new insights into KoRV pathogenesis and DNA quantification methodology, which could be valuable for future research by identifying predictive markers for tumour progression and potential therapeutic targets in affected koalas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f125ee840ff1c5667ff63226a6db37a09c90835b" target='_blank'>
              Infection-Induced Telomere Length Variation: Insights into Pathogenesis of Koala Retrovirus
              </a>
            </td>
          <td>
            Hiu Ming Cheung, Sze Wing Jamie Lin, Hanh Thi Hong Nguyen, T. Stephenson, N. Speight, F. Hemmatzadeh
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020f876f8e4a48a7cc99413edfa413effb89316e" target='_blank'>
              Interphase chromosome conformation is specified by distinct folding programmes inherited through mitotic chromosomes or the cytoplasm.
              </a>
            </td>
          <td>
            A. Schooley, Sergey V. Venev, V. Aksenova, Jesse W. Lehman, E. Navarrete, A. Pai, M. Dasso, Job Dekker
          </td>
          <td>2025-12-22</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="RhoGDI2 is a RhoGTPase regulator that has roles in cytoskeleton organization and cell survival, amongst others. It is differentially expressed in many cell types and tissues, including several human cancers, where its expression has been correlated with either good or bad prognosis. To identify the underlying mechanisms, we knocked down its expression in human cancer cell lines. We observed that repression of RhoGDI2 expression, but not that of the closely related RhoGDI1, significantly reduces their proliferation rate. In parallel, RhoGDI2 suppression induces supernumerary centrosomes and inhibits ciliogenesis. As RhoGDIs are regulators of GTPases, we checked whether key RhoGTPases are involved in these effects. We found that silencing RhoA partially rescued the induction of supernumerary centrosomes and ciliary defects observed upon RhoGDI2 silencing. It was previously shown that RhoGDI2 is strongly expressed in immune cells and that there are striking similarities between primary cilia and immune synapses. Based on this knowledge, we silenced RhoGDI2 in NK cells and could demonstrate that this strongly affects their immune synapse-related cancer cell-killing activity. Altogether, these data suggest novel roles for RhoGDI2 in centrosome functions in human cancer and immune synapses in immune cells, which provides an explanation for its reported dual role in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c0ac8c4a86b7bca62ea906513e7218a435f267" target='_blank'>
              Reduced RhoGDI2 Expression Disrupts Centrosome Functions and Promotes Mitotic Errors
              </a>
            </td>
          <td>
            M. Tripathi, N. Garbacki, J. Willems, G. Cobraiville, Marianne Fillet, Alain Colige, Christophe F. Deroanne
          </td>
          <td>2025-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Complex chromosomal rearrangement (CCR) refers to a structural rearrangement involving at least two chromosomes or a minimum of three breakpoints. CCR may lead to intellectual disability, structural anomalies, infertility, and recurrent miscarriages. Chromosome karyotyping and chromosomal microarray analysis (CMA) are unable to detect complex chromosomal rearrangements. As multiple diagnostic approaches are available in clinical practice for detecting chromosomal structural abnormalities and copy number variations—each with its own advantages and limitations—selecting the appropriate testing method is crucial for effective clinical management. Optical genome mapping (OGM) is an advanced genomic technology that utilizes ultra-long single-molecule analysis to comprehensively detect chromosomal aberrations and structural variants at high resolution. Amniocentesis was performed for a 36-year-old multipara (advanced maternal age), with subsequent comprehensive fetal genetic analysis including chromosome karyotyping, CMA, and OGM. Family members underwent peripheral blood karyotyping and OGM. The fetal karyotype derived from amniotic fluid was 46,XN,?ins(18)(q21.2;p11.31p11.2). CMA demonstrated duplications of four segments and a deletion of one segment on chromosome 18. Therefore, OGM was performed on the fetal and family members to further elucidate the chromosomal structure. The fetus has derived CCRs on chromosome 18 of maternal origin. In contrast, both the mother and the second daughter, who carried the identical CCRs, were phenotypically normal. OGM is of significant importance in the diagnosis and characterization of CCRs. OGM plays a critical role in diagnosing complex chromosomal rearrangements and has proven to be invaluable in clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a05870664d5ee26da7943564ddcc7debed72e97b" target='_blank'>
              Optical genome mapping of a complex structural rearrangement family line on chromosome 18
              </a>
            </td>
          <td>
            Liyi Cai, Yuying Jiang, Na Zhang, Xinying Chen
          </td>
          <td>2025-12-01</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Introduction Regulation of gene expression by histone modification is critical in drug resistance and progression. KMT2A-rearranged (KMT2A-r) AML can activate aberrant leukemogenic HOXA and MEIS1 gene signatures through its interaction with menin. KMT2A is part of COMPASS complex along with ASH2L, WDR5, RBBP5, and DPY30, sometimes called 'WRAD’ complex, and functions as writers and readers to catalyze H3K4 methylation and drives transcription of leukemogenic genes such as HOXA9 and MEIS1. ASH2L is part of the core complex and inherently enhances the enzymatic activity amongst the other scaffolding proteins. We hypothesized that ASH2L (a histone methyltransferase writer) and WDR5 (a chromatin reader) constitute an escape, non-canonical resistance mechanism that rewires the leukemogenic transcription program even in the presence of menin inhibitor KMT2A-r AML cells.
 Methods To this note, we aimed to establish an in-vitro resistant model in KMT2Ar leukemia cell lines (MV4-11 and MOLM-13) by chronically exposing them to sequentially escalating, sub-therapeutic doses of a menin inhibitor (ziftomenib, Selleckchem, cat.no:E1290). COMPASS complex gene and downstream transcripts and protein expressions were detected by quantitative RT-qPCR and western blotting. SiRNA-mediated knockdown (kd) of ASH2L and WDR5wasperformed to investigate their functional role in resistance, and western blotting was used to confirm protein depletion and assess downstream effects on MLL1/COMPASS components. An ATP-based cell proliferation assay was performed to assess cell viability and growth inhibition following SiRNA KD and by using WDR5 inhibitor (OICR-9429, Selleckchem, cat.no:S7833) to pharmacologically target the complex. ChIP-qPCR was performed to determine whether resistance is mediated by altered chromatin occupancy by alternate COMPASS components to the HOXA9 promoter in the resistant models.
 Results Incremental exposure of MV4-11 cells to a menin inhibitor (ziftomenib) over six months resulted in a progressive increase in IC₅₀ from baseline, with the cells maintaining proliferative capacity even at continued exposure of 60 nM of the inhibitor. Cell line authentication using short-tandem-repeats verified that the resistant subline was genetically identical to the parental MV4-11 cell line, and samples were processed for sequencing of the MEN1-binding pocket to screen for menin inhibitor resistance-associated mutations. RT-qPCR and immunobloting exposed clear transcriptional and protein level differences in the resistant and sensitive AML cells. Sensitive cells retained low HOXA10 and MEIS1/2 expression whereas resistant cells exhibited reduced menin levels but increased ASH2L, WDR5, and HOXA9 expression, suggesting that HOXA9 upregulation may serve as a potential escape mechanism driving resistance. SiRNA knockdown of ASH2L and WDR5 in resistant AML cells led to a marked decrease in leukemogenic transcripts, including HOXA9 and MEIS1, as confirmed by immunoblotting. In parallel, we generated dose-response curves for WDR5 dependency in sensitive (IC50 for ziftomenib, 14nm) and menin-inhibitor-resistant MV4-11 cells (IC50 for ziftomenib, 65nm), and notably, treatment with a WDR5 inhibitor restored the IC₅₀ of resistant cells to levels comparable to the sensitive parental line (IC50 for WDR5 inhibitor 12nm). ChIP-qPCR with ASH2L immunoprecipitation in menin-inhibitor-resistant cell line showed markedly increased ASH2L occupancy at HOXA9 promoter, suggesting that higher occupancy may contribute to resistance to menin inhibition. Ongoing work examines selective ASH2L inhibition—alone or combined with menin and WDR5 blockade—to overcome resistance in KMT2A-r AML. Genome-wide ChIP-seq to map ASH2L and WDR5 redistribution at promoters/enhancers, extending our HOXA9 data. TurboID proximity labeling to compare protein-interaction networks in sensitive versus resistant cells to uncover COMPASS partners. CRISPR knockdowns and patient cells will be tested in xenografts for therapeutic response.
 Conclusion ASH2L and WDR5 are critical mediators of COMPASS-dependent therapy resistance in KMT2A-r AML. Targeting this complex, particularly ASH2L and WDR5, represents a promising strategy to possibly reverse resistance and improve therapeutic outcomes in resistant KMT2A-r AML.Our preliminary data indicate other components of the COMPASS complex to possibly cause resistance in KMT2A-r AML, and further validations are in progress.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d0d6261eed83d4bc7612c32cd10a8f9626c2b5" target='_blank'>
              Targeting the compass complex to overcome resistance in KMT2A-rearranged Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sandhya Dhiman, Vikramjit Dhillon, J. Aguilar, Julie Boerner, Yang Shi, Jay Yang, S. Balasubramanian
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor state characterized by expansion of blood clones carrying somatic mutations that originate in the hematopoietic stem cells (HSCs). Individuals with CHIP have a 10-fold increased risk of developing myeloid malignancies such as myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). Progression of CHIP to MDS/AML takes several decades and hence targeting the CHIP clones during this period provides a viable strategy to prevent MDS/AML. Despite its importance, we still don't understand the mechanisms of CHIP predisposition and its progression. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP, but the causal variants remain unknown because most of the variants are found in non-coding genomic regions. We have previously shown thatmodulation of gene regulatory elements in HSCsunderlies genetic risk of myeloid malignancies; whether this is also true for premalignant state such as CHIP is currently unknown.
 We hypothesize that germline CHIP risk variants in non-coding loci affect enhancer elements in HSCs. In order to identify these variants we utilized a high throughput approach termed Massively Parallel Reporter Assay (MPRA) to simultaneously screen for regulatory effects of 1,374 non-coding variants within 24 GWAS loci from a comprehensive study of CHIP risk on ~600,000 individuals. In the MPRA approach, we synthesized pools of barcoded lentiviral constructs harboring non-coding variants (risk and non-risk alleles) and surrounding genomic sequence (250bp) upstream of a minimal promoter driving a reporter gene. Sequencing of the plasmid pool confirmed an uniform distribution of barcodes with a median of around 50 barcodes per variant. Since enhancers show cell-type specific activity, we next searched for the appropriate cell-type to perform the MPRA. As the disease relevant population in CHIP is the HSCs, which is rare, we searched for alternative cell types. We identified MUTZ-3, a AML cell line that maintains a self-renewing CD34+ population and is dependent on HSC transcription factors. We confirmed that MUTZ-3 cells have trans-acting factors similar to primary HSPCs, by testing several validated HSPC enhancers including PU.1 upstream regulatory element (URE) and GATA2 inv(3) enhancer using individual reporter assays.
 We performed our lentiviral MPRA screen using a library of ~73,000 constructs in CD34+ fraction of MUTZ-3 cells. 48 hours post-infection we lysed the cells for DNA and RNA and sequenced the barcodes by NextSeq. The experiments were highly reproducible and all 5 replicates correlated well with each other at both the DNA and RNA barcode level. When we measure the enhancer activity by ratio of RNA to DNA barcode read counts, positive controls in the pool (PU.1 URE, GATA2 enhancer) showed higher activity compared to negative ORF controls. The activity of test constructs that overlap with hematopoietic ATAC peaks was higher that non-overlapping constructs indicating MPRA can identify endogenous enhancers. We identified several hits that showed allele specific reporter activity based on CHIP variant (Ref and Alt allele).
 We prioritized rs17834140, a variant identified by the MPRA for further validation studies. The variant is located within the 4th intron of the MSI2 gene in an hematopoietic ATAC peak and MPRA and individual reporter assays indicate enhancer activity and the risk allele increases MSI2 expression. We further confirmed endogenous enhancer activity by both CRISPR/Cas9 deletions and CRISPR inactivation experiments in human hematopoietic cells. To test whether MSI2 overexpression can also contribute to clonal expansion of CHIP mutant HSCs, we infected LSK cells from TET2 knockout mice and wildtype controls with the MSI2 overexpressing and control lentiviral vectors and observed an increased in vitro serial colony replating ability of TET2 cells upon MSI2 upregulation.
 Our study paves the way to systematically assess non-coding risk variants associated with CHIP risk. We expect the MPRA approach will help to draw out key unknown myeloid neoplasm predisposition genes and their regulatory elements that are influenced by the germline variants.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691b91de07b0de85cfcd28321c80391149dbd376" target='_blank'>
              Assessing germline noncoding risk variants impacting clonal hematopoiesis using a massively parallel reporter assay
              </a>
            </td>
          <td>
            Jessica Jeejan, Trieu Nguyen, S. Kales, Chie Yanase, Dominika Kwasniak, Aruna Shekhar Nannivala Chandra Shekhar, Gowri Poigaialwar, Richard Voit, Keisuke Ito, Ryan Tewhey, Satish Nandakumar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a524bd974d6e1f504ec3318cf5435988050a086" target='_blank'>
              XIST Drives X-Chromosome Inactivation and Safeguards Female Extraembryonic Cells in Humans
              </a>
            </td>
          <td>
            Amitesh Panda, Léo Carrillo, B. Balaton, Jeanne Brouillet, Solomon Nshemereirwe, Jarne Bonroy, Charbel Alfeghaly, Romina Facchinello, Sherif Khodeer, N. Peredo, R. Boers, G. Castel, Charlie London, Emmanuel Cazottes, Madeleine Moscatelli, Raissa Songwa Tchinda, Thi Xuan Ai Pham, San Kit To, Ryan N. Allsop, Yang Wang, Desislava P. Staneva, P. Rugg-Gunn, Kathy K. Niakan, J. Gribnau, Jean-François Ouimette, C. Rougeulle, Vincent Pasque
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The single-stranded DNA deaminase APOBEC3B (A3B) is capable of potently restricting the replication of a range of viruses including retroviruses (cDNA) and herpesviruses (genomic DNA). However, these and likely other DNA virus families have evolved host species-specific counter-defenses that are equally potent and serve to protect viral DNA from restriction. Although high-risk human papillomavirus (HPV) infection triggers A3B upregulation, potentially as part of an antiviral response, the impact of this restriction factor on papillomavirus replication and pathogenesis has yet to be assessed. To study human A3B antiviral function in the absence of a species-specific counter-defense, here we ask whether human A3B is capable of restricting Mus musculus papillomavirus (MmuPV1) in cellulo and in vivo. First, we created human A3B and catalytic mutant A3B-E255A expressing FVB/N mice. Second, MmuPV1 gene expression and replication was quantified in primary keratinocytes from these animals and, surprisingly, enzymatically active human A3B caused no measurable impairment in viral transcript or DNA accumulation. Third, A3B, catalytic mutant A3B-E255A, and nontransgenic FVB/N animals were infected with MmuPV1 and similar pathologies were found regardless of A3B functionality. Thus, despite likely never being exposed to human A3B during evolution, MmuPV1 appears to be unaffected by this potent, primate-specific antiviral factor. These results suggest that MmuPV1 and perhaps papillomaviruses more broadly possess a conserved mechanism to efficiently escape restriction by human A3B and related DNA deaminases. IMPORTANCE Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes such as A3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43450515e8b62fe85e717aebfac6a246b0159e06" target='_blank'>
              Mus Musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, Sang Hyun Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Background Since its inception, RNA sequencing has been pivotal in studying differential gene expression. Despite its extensive results in large-scale oncological studies, differential expression predominantly reflects a response to cancer. Therefore, we introduce differential allelic dispersion (AD) as a more effective measure. AD highlights consistent differences in expression between the 2 alleles of a gene that is, unlike cis-expression quantitative trait loci, independent of normal genetic variation. Such differences can, for example, arise from prevalent copy number alterations or epimutations occurring in the original cancer cell, which are mitotically expanded during cancer growth, making increased AD a marker for allele-specific dysregulation in early carcinogenesis. Findings We present the maelstRom R/C++ software package that enables (differential) AD analysis solely requiring large-scale RNA sequencing data. Using the The Cancer Genome Atlas renal clear cell carcinoma cohort as a case study, we successfully benchmark maelstRom’s AD modeling using known copy number alterations. We also detect increased AD for loci featuring normal random monoallelic expression, including the X chromosome, but demonstrate minimal interference with cancer-specific AD detection. Finally, we identify early dysregulated genes (e.g., FBP1, CCDC8, ECHS1, CLDN7) and pathways in renal cancer, often related to metabolism (e.g., pentose phosphate pathway). Strikingly, many of these genes are known causal contributors to renal carcinogenesis. Conclusions Differential AD clearly indicates early dysregulation in renal cancer, complementing basic differential expression analysis in cancer transcriptomics. AD is also relevant to study random monoallelic expression and may equally detect allele-specific (dys)regulation during early development or in noncancer diseases. maelstRom is available as an open-source software package at github.com/Biobix/maelstRom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e775646c46e5a03c9c6239d832825d07a63f0" target='_blank'>
              Population-level allelic dispersion modeling by maelstRom yields genome-wide maps of allele-specific dysregulation during early carcinogenesis
              </a>
            </td>
          <td>
            Cedric Stroobandt, Louis Coussement, T. Goovaerts, Femke De Graeve, Jeroen Galle, Wim Van Criekinge, Tim De Meyer
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Two rounds of whole-genome duplication (WGD) occurred about 500 million years ago and played a major role in the evolution of the vertebrate genomes. Human genes derived from WGD are called “ohnologs”. Ohnologs are involved in fundamental biological processes and significantly contributed to the complexity of the human gene regulatory network. Given the central role of miRNAs in gene regulation, we investigated the contribution of ohnolog miRNAs (ohno-miRNAs) to the human gene regulatory network. We identified intragenic ohno-miRNAs as having higher retention rates compared to intragenic Small Scale Duplicated (SSD) miRNAs. Ohno-miRNAs also show high sequence similarity, a stronger tendency to regulate common target genes and are typically more expressed compared to miRNAs unrelated to WGD events. Analyzing the role of ohno-miRNAs in the human gene regulatory network, we showed that ohno-miRNAs are statistically overrepresented in specific network motifs commonly associated with redundancy and complexity, highlighting their central role in gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c5d0c75cd069a45340fb97f72963794c6c56d0e" target='_blank'>
              Ohno-miRNAs: miRNA pairs derived from whole-genome duplication
              </a>
            </td>
          <td>
            Leonardo Agasso, Ivan Molineris, Michele Caselle
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Medulloblastoma (MB) is the most common malignant pediatric brain tumor and exhibits high heterogeneity in genetic alterations and molecular subgroups. However, current treatment remains largely uniform, and more precise risk stratification is necessary to improve prognosis. Large-scale genomic studies have primarily focused on Western populations, while comprehensive data on Japanese patients remain limited due to low incidence. This study aimed to elucidate molecular genetic characteristics and prognostic factors in Japanese MB through integrative genomic and transcriptomic analyses. Methods Targeted DNA and RNA sequencing was performed on 200 cases from the Japanese Children’s Cancer Group (JCCG), and publicly available whole-genome and RNA sequencing data from 472 Western MB patients were analyzed for comparison. Results Known subgroup-specific driver mutations were identified in the Japanese cohort, and age-adjusted comparison with the Western cohort showed no significant differences in mutation frequencies, suggesting limited racial differences in MB pathogenesis. Deep sequencing detected subclonal mutations, including SMARCA4 in the WNT subgroup. Missense mutations in the MYC functional domain were observed in cases lacking MYC amplification; these are known drivers in hematological malignancies and may also play an oncogenic role in MB. MYC mutations were found in WNT and SHH subgroups, suggesting broader involvement beyond Group 3. Intron retention analysis revealed a non-canonical intronic ZMYM3 mutation in a Group 4 case, highlighting improved driver detection by integrating DNA and RNA data. Whole chromosomal aberrations correlated with better outcomes, supporting their role in risk classification. Conclusions This study identified novel MYC alterations and emphasized the utility of RNA-integrated analysis. Cohort expansion may uncover additional diversity and aid in precision medicine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394acb3917250c67e7f0d75d4a797dbd4bea5da9" target='_blank'>
              10102-GEN-2Integrated Genomic and Transcriptomic Analysis Reveals Molecular Diversity and Novel Drivers in Japanese Medulloblastoma
              </a>
            </td>
          <td>
            Tsubasa Miyauchi, Takuma Nakashima, Miho Kato, Junko Hirato, Takako Yoshioka, Hajime Okita, Y. Kanemura, Yuko Watanabe, Masahiro Toda, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chromatin remodelers regulate DNA-nucleosome interactions, influencing chromatin structure, accessibility, and gene expression (Gourisankar, S., Nat Rev Genet, 2024). Their dysregulation is frequently observed in cancer and often represents a therapeutic vulnerability (Johnstone, S. E., Science, 2022). HELLS, a key chromatin remodeler, maintains genome stability and regulates gene expression in aggressive T-cell lymphomas, such as ALK-negative anaplastic large cell lymphoma (ALK- ALCL) (Fragliasso V, Leukemia 2020; Tameni A., Cell Death Dis. 2021). Previous studies have demonstrated that HELLS knockdown sensitizes ALCL cells to chemotherapy, supporting its potential as a therapeutic target (Tameni A., Nucleic Acids Res. 2024).
 This study aimed to define the transcriptional role of HELLS and evaluate its therapeutic relevance in ALCL. To achieve this, we analyzed formalin-fixed, paraffin-embedded biopsies from a cohort of 44 ALCL patients and performed an integrated in vitro multi-omics analysis (RNA-seq, ChIP-seq, ATAC-seq, and Connectivity Map), alongside functional assays.
 To understand how HELLS coordinates and supports transcription, we integrated HELLS Chromatin immunoprecipitation-sequencing (ChIP-seq) data from the TLBR-2 cell line with RNA-sequencing data from TLBR-2 HELLS-knockdown (KD) and control cells. Of the 729 genes significantly affected by HELLS KD, 467 (64%) were identified as direct HELLS targets based on ChIP-seq analysis. Gene-set enrichment analysis of these HELLS-direct genes (HDGs) revealed their association with diverse biological processes, including the JAK/STAT signaling pathway and chemokine- and cytokine-mediated inflammatory signaling. The clinical relevance of HDGs was confirmed by nCounter profiling in a cohort of 44 ALCL patients (15 ALK+, 29 ALK- ALCL).
 To assess the contribution of HELLS to HDG transcription, we investigated the distribution of RNA Polymerase II (RNAPII) via ChIP-seq in TLBR-2 HELLS-KD and control cells. RNAPII ChIP-seq data revealed RNAPII elongation defects in approximately 60% of HDGs and altered RNAPII occupancy in ~40% of HDGs following HELLS depletion. Gene ontology analysis of this latter group of HDGs highlighted an enrichment in immune-related pathways, including T-cell-mediated immunity, cytokine signaling, and JAK/STAT signaling. Furthermore, ATAC-seq and H3K4me3 profiling indicated that HELLS promotes chromatin accessibility and transcriptional activation at immune-related loci.
 To evaluate the therapeutic relevance of HELLS, we employed a drug repurposing approach based on the HDG signature. This led to the identification of 10 classes of synergistic pathways, with PI3K, JAK/STAT, and DNA-PK emerging as top-scoring synergistic targets. In several ALCL cell lines, HELLS depletion synergized with an IC20 dose of Ruxolitinib (a JAK inhibitor) or AZD7648 (a DNA-PK inhibitor), resulting in significant synthetic lethality compared to single-agent treatment or control cells.
 In conclusion, this study demonstrates that HELLS drives the expression of immune-related genes through chromatin remodeling in aggressive ALK- ALCL. Its inhibition uncovers combinatorial vulnerabilities, providing a rationale for dual-targeted therapies involving JAK/STAT or DNA-PK pathway inhibition.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088deae46f89f2511cdb635d4408d6da56d67fdf" target='_blank'>
              Targeting hells‑mediated chromatin accessibility to sensitize ALK‑negative anaplastic large cell lymphoma to JAK/STAT and DNA‑PK inhibition
              </a>
            </td>
          <td>
            Giulia Gambarelli, Selene Mallia, B. Donati, N. Puccio, Magda Zanelli, Emanuele Vitale, Federica Torricelli, V. Manicardi, E. Salviato, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="An R-loop is a three-stranded nucleic acid structure that serves as a transcriptional intermediate, consisting of an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA). Small RNAs are RNA molecules shorter than 300 nucleotides that perform a wide range of essential functions within cells. Both R-loops and small RNAs are widely present in the genomes of prokaryotes and eukaryotes, where they play crucial roles in regulating gene expression, maintaining genomic stability, and facilitating DNA damage repair. Aberrant formation and accumulation of R-loops, coupled with dysregulation of small RNA pathways, can induce DNA damage and genomic instability, ultimately contributing to cellular senescence or cell death. Here, we discuss recent advances in understanding the crosstalk between R-loops and small RNAs, with a focus on their synergistic roles in maintaining genome stability and their therapeutic potential in oncology and neurodegeneration. We propose a novel model integrating R-loop dynamics with small RNA-mediated epigenetic regulation, supported by emerging clinical trial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853eaea8f06e1b31e5bea0e5a97d17746ae77587" target='_blank'>
              R-loops and small RNA regulatory interactions: mechanisms and clinical perspectives
              </a>
            </td>
          <td>
            Si-Yao Wang, Ya-Ni Liu, Si-Qiao Zhao, Wen Liu, Jin Hu, Fan Yang, Sheng Wang
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) is a DNA virus that establishes a persistent asymptomatic infection during childhood and can cause Merkel cell carcinoma (MCC) later in life. Its Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, drive viral replication and tumorigenesis by binding to and perturbing the function of host proteins. LT binds and inhibits RB1, deregulating E2F activity and host cell cycle control to permit viral replication during S phase. While the functions of LT and ST are relatively well characterized, how their expression is controlled is poorly defined. Here, we discovered that E2F1-3/DP1 dimers bind the MCPyV Non-Coding Control Region (NCCR) via a consensus E2 site close to the LT/ST transcriptional start site. Inhibiting E2F-NCCR binding, either by deleting the E2 site or treatment with a E2F small molecule inhibitor, downregulated LT/ST mRNA and protein expression in MCC cells and in 293A cells transfected with MCPyV. Our findings reveal an E2F/LT/RB1 positive feedback loop that appears to have evolved to support viral replication and is hijacked in MCC cells to promote cellular proliferation. Furthermore, we identified E2 sites in the NCCRs of PyVs closely related to MCPyV, including murine PyV, which mediate E2F/DP binding and potentiate viral early transcription. E2F/DP also bound weakly to the SV40 and BKPyV NCCRs, despite lacking an E2 site. Our findings challenge the prevailing model that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV early transcription and replication. Significance statement Polyomaviruses (PyVs) express Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, that drive viral replication and tumorigenesis by binding and perturbing the function of host cells. LTs bind and inhibit RB1 via conserved LxCxE motifs, deregulating E2F activity and host cell cycle control to permit viral replication. Here, we discovered that E2F1-3 bind to the Non-Coding Control Regions (NCCRs) of many, but not all, PyVs to activate LT/ST transcription, revealing an E2F/LT/RB1 positive feedback loop that appears to have evolved to promote viral replication. Our findings challenge the existing paradigm that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV LT/ST transcription and replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92fb2e450770315d4079985e1a3aa56f6892c80a" target='_blank'>
              E2F1–3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Nick Salisbury, Supriya Patil-Amonkar, Ann Roman, Denise A. Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Potassium is the most abundant intracellular cation. Potassium flux across cell membranes is controlled by potassium channels. While cytoplasmic potassium has been extensively studied, very little is known about the function of potassium in the cell nucleus. Here, we discover that a nuclear potassium channel complex composed of KCNG1 and KCNB1 subunits is a regulator of nuclear potassium in medulloblastoma, the most common malignant pediatric brain tumour. Using genetically encoded potassium sensors, we show that KCNG1 regulates nuclear potassium level independently of cytoplasmic potassium level. KCNG1 knockdown alters the genomic architecture of medulloblastoma cells by stabilizing G-quadruplex DNA and inducing DNA damage. Ultimately, KCNG1 deficiency results in nuclear swelling, thereby compromising nuclear envelope integrity, promoting the formation of micronuclei, and inducing DNA spillage into the cytoplasm to activate the immune-stimulatory cGAS/STING pathway. Kcng1 knockout in a genetically engineered mouse model of medulloblastoma remodels the immune microenvironment, reduces tumour growth, and prolongs mouse survival. Altogether, we identified a potassium channel complex that regulates nuclear potassium to govern nuclear integrity, tumour immune microenvironment, and medulloblastoma growth. Our results demonstrate that targeting nuclear potassium can be leveraged to treat medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3156f37831fc433a976d1d47ab5180bff944abd" target='_blank'>
              DNAR-02. Nuclear potassium governs nuclear integrity and tumour growth in medulloblastoma
              </a>
            </td>
          <td>
            Jade Chan, Xi Huang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, the roles of which are under debate. We found that double-strand breaks (DSBs) induce de novo formation of chromatin loops in human cells, with the loop base positioned at the DSB site. These loops form in the S and G2 phases of the cell cycle during homologous recombination repair, concomitantly with DNA end resection and radiation-sensitive protein 51 (RAD51) recruitment. RAD51 shows a broad (megabase-sized) chromatin domain reflective of the homology search. This domain is regulated by cohesin unloader and overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of the loop-extruding cohesin subunit NIPBL lowers homologous recombination in mouse embryonic stem cells, and this effect is more pronounced when the homologous recombination donor is hundreds of kilobases from the DSB. Our data indicate that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e61307e4b5e3ffa40f9315b36ee4db40b3f94b6" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam T. Rybczynski, Namrata Nilavar, Daniel Nguyen, Andrew G. Li, Violetta Karwacki-Neisius, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g., “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, S. Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Analysis of coding areas has long been used to study monogenic illnesses, but despite the extensive use of whole-exome sequencing (WES), up to half of suspected cases remain genetically unexplained. Variants outside coding areas can alter splicing, transcript stability, or gene regulation, compromising normal gene activity. These include mutations in noncoding RNAs, promoters, enhancers, deep intronic sequences, and untranslated regions (UTRs). Several well-known disorders have been linked to these mechanisms, including β-thalassemia caused by deep intronic mutations leading to aberrant splicing, familial hypercholesterolemia caused by promoter defects affecting LDLR expression, and inherited retinal diseases driven by noncoding variants influencing retinal gene regulation. These instances show that pathogenic variation is not limited to the exome and can have significant clinical implications. This review summarizes current understanding of noncoding and regulatory variants in monogenic diseases, discusses how they influence diagnosis and therapy, and highlights integrative approaches combining genomic, transcriptomic, and epigenomic data. Multi-layered research has increased diagnostic accuracy and unveiled new therapeutic potentials, although noncoding variations make the connection between genotype and phenotype more complex. Noncoding regions will need to be incorporated into standard diagnostic procedures to convert molecular insights into concrete therapeutic applications in the future. Predictive algorithms, patient-derived model systems, and functional validation testing will all help to simplify this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffd879f4fd9d85153dc94e05643890e26eb43d7" target='_blank'>
              Beyond the Exome: The Role of Noncoding and Regulatory Variants in Monogenic Diseases
              </a>
            </td>
          <td>
            Efthalia Moustakli, Nektaria Zagorianakou, Stylianos Makrydimas, Andreas Miltiadous, Alexandros T. Tzallas, G. Makrydimas
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Maintaining genome integrity in germ cells is crucial for fertility and species survival. However, the DNA repair mechanisms that sustain genome integrity in primordial germ cells (PGCs), which cope with high levels of replication stress, remain largely unknown. While the TatD family of proteins, evolutionarily conserved nucleases, has been found to play roles in various DNA-related processes, their in vivo functions in vertebrates have yet to be fully elucidated. TATDN2 has recently been implicated in resolving R-loops and participating in the replication stress response in BRCA1-deficient cancer cells. Here we found that tatdn2 exhibits conserved expression in mitotic and early meiotic germ cells across teleosts and mammals. Using medaka fish as a model, we then showed that loss of tatdn2 leads to all-phenotypically male adults and infertility due to PGC depletion during mitotic proliferation. We further demonstrated that knockout of tatdn2 increases R-loop accumulation and DNA damage, subsequently triggering apoptosis in PGCs. These findings indicate that tatdn2 plays a critical role in DNA damage repair associated with R-loop resolution in mitotic PGCs. Our study provides novel insights into the physiological function of TATDN2 and the mechanisms of genome maintenance in PGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aba7e41b7a154253da5adb67bf89677856b6ff" target='_blank'>
              Tatdn2 is required for DNA repair to safeguard genome stability in primordial germ cells
              </a>
            </td>
          <td>
            Hanqiao Shang, Puxuan Jiang, Zehui Wang, Zhaojun Shan, Yuze Chen, Ting Zhang, Yingshu Li, Qiang Tu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Using the UCSC genome browser, this study primarily examined the density of CpG Islands in each gene to investigate epigenetic changes, particularly methylation patterns or profile throughout the genome of TNBC genes. Here the analysis of 10 hub TNBC genes were study for the methylation profiling and additionally focused on its relevance of reprograming to these genes.
 It can also analyze how certain genomic areas are conserved across species, which can reveal information about their functional significance. It also examines the existence of CpG islands, which could be connected to other regulatory elements or gene promoters. Examining the data on histone modifications to find possible promoters or enhancers. Genes such as AURKA, CDK1, EZH2, CCNA2, CTNNB1, MKI67, etc., are known cell cycle regulators, chromatin modifiers, and proliferation markers, many of which are overexpressed in cancer and stem cells, including induced pluripotent stem cells (iPSCs). These 10 genes are reprogramming-supportive genes and indirectly modulated or supported by Yamanaka factors. Their hypomethylation and expression patterns in these results indicate that the cellular environment in TNBC resembles a partially reprogrammed state, which may explain aggressive growth and plasticity — a useful insight for drug targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7b6a886f40c24727327b2385ecbc05a3309cb8" target='_blank'>
              Methylation Profiling of TNBC Hub Genes Using UCSC Genome Browser: A Comparative Study Relevance to Reprogramming
              </a>
            </td>
          <td>
            Shivi Sharma, Prashant Ankur Jain, Eliza Chakraborty
          </td>
          <td>2025-11-07</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Dysregulation of gene expression is an essential feature of most human cancers, including acute myeloid leukemia (AML). Despite distinct biology of different subtypes of AML, there is also increasing evidence for shared pathologic mechanisms. Varied genetic subsets of AML exhibit shared dependencies on specific transcription factors and gene expression regulatory mechanisms. In particular, MYB is aberrantly activated in most studied forms of AML, where it is required for blockade of differentiation and self-renewal of leukemia cells. MYB is rarely mutated and generally not over-expressed in most human myeloid leukemias, presenting a key question concerning mechanisms of its activation that should be targeted therapeutically.
 To elucidate potential mechanisms of MYB activation, we investigated post-translational modifications (PTMs) of endogenous MYB proteins purified from human AML cells. Using comprehensive bottom-up high-resolution mass spectrometry, we achieved 95% MYB peptide sequence coverage and identified 10 unique MYB amino acid residues containing phosphorylated, acetylated, methylated, or ubiquitin-like PTMs. This included phosphorylation of S11 and S12 in the N-terminal domain (NTD) proximal to the DNA-binding domain of MYB, as validated using phospho-specific MYB antibody, mutagenesis, and dephosphorylation experiments. Notably, S11/S12 phosphorylation was preferentially detected across multiple AML cell lines and patient specimens from diverse molecular subtypes as compared to healthy human blood progenitor and stem cells.
 We found that MYB is required for the growth and survival of leukemia cells, which can be rescued by the expression of wild-type MYB but not MYB mutants that cannot be phosphorylated at S11/S12 in its NTD. Using label-free quantitative mass spectrometry combined with affinity purifications, we defined specific protein co-factors that are selectively assembled with phosphorylated as compared to non-phosphorylated MYB in the nuclei of AML cells, as well as cells expressing MYB-PLEKHO1 and MYB-ZFAT fusion oncogenes from blastic plasmacytoid dendritic cell neoplasm (BPDCN) cells. We found that S11-phosphorylated MYB preferentially assembles with other hematopoietic transcription factors including LMO2, LYL1, JUN, and TCF12. A subset of transcription factors including ZEB2, SATB1, IKZF1, SPI1, and RUNX1, 2, and 3 were also found to be co-assembled with BPDCN MYB oncogene fusion proteins. Co-assembled MYB transcription factors were associated with cooperative chromatin complexes regulating genes controlling leukemia cell differentiation, growth and survival, as defined using gene expression, chromatin profiling, and functional genomic assays.
 In all, these studies identify DNA-proximal phosphorylation of MYB and MYB DNA-binding domain fusions as a critical regulatory mechanism required for MYB oncogenesis through cooperative organization of transcription factors. Modifications of DNA-proximal NTD leading to MYB enhanceosome assembly suggest a new target for therapeutic intervention of MYB-dependent transcriptional circuits in AML and other transcription factor-dependent malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97715039497852e85a982a7d2a4af32af301c7f5" target='_blank'>
              DNA-proximal MYB phosphorylation is required for oncogenic transcription in acute myeloid leukemia
              </a>
            </td>
          <td>
            Shuyuan Cheng, Katarzyna Kulej, Christopher A G Booth, Masahiro Uni, Nicole Cruz, Andrew Lane, R. Koche, A. Kentsis
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Radiation-induced damage to nucleic acid and other important cellular constituents could likely cause cancer. The present study has analyzed mutated genes with dysregulated expression signatures due to radiation. We have predicted biomarkers specific to hormone-sensitive cancers (HSCs) namely breast cancer (BC), prostate cancer (PC), ovarian cancer (OC), and endometrial cancer (EC). The study has also attempted to prospect the pathways associated with HSCs and radiation exposure through network analysis and functional enrichment analysis. An overlap of important cell cycle pathways, DNA binding, and transcription activity pathways were found in comparison with HSCs and the radiation hallmark pathways. Hub genes TNF, STAT3, CTNNB1, and MYC in BC; IL1B, CTNNB1, ESR1, and SRC in PC; CTNNB1, BRCA1, JUN, and KRAS in OC; PIK3CA, PTEN, and ESR1 in EC were found as hypoxic signatures due to radiation. Further, the expression survival analysis found MYC and STAT3 in BC, CTNNB1 in PC, JUN, and BRCA1 in OC, and ESR1 in EC were significantly dysregulated and further mutational profiling validated MYC with 18%, STAT3 with 2.6%, and all other genes with 4% mutation. Thus, these genes are proposed as radiation-sensitive diagnostic biomarkers in HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e92757f335699ebbedff521e546648301a15ad2e" target='_blank'>
              Systems biology-driven identification of biomarkers and significant pathways in radiation-induced hormone-sensitive cancers
              </a>
            </td>
          <td>
            Suvitha Anbarasu, Sathyanarayan Balaji, Sudha Ramaiah, A. Anbarasu
          </td>
          <td>2025-11-04</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="

 Primary malignant melanoma in the cervix (PMMC) currently has no standardized therapy. Its pathogenesis remains unclear, and the prognosis is poor. Given the high mortality of PMMC, its causes and pathogenic mechanisms need to be unraveled, and novel biomarkers must be identified.



 To further understand the genomic characteristics of PMMC, whole-exome sequencing was performed on the cancerous and adjacent tissues from three patients with PMMC, and PMMC-related tumor-susceptibility genes, mutation spectrum, mutation characteristics, driver genes, and high-frequency tumor mutations were analyzed. Concurrently, SNP and copy number variation (CNV) data from melanoma patients in the COSMIC and TCGA datasets were analyzed.Key protein interactions were further validated in vitro using co-immunoprecipitation assays.



 The results showed that C>T/G>A is predominant in PMMC, with mutation characteristics more closely resembling those of SBS5 and SBS40, though not completely matching those of melanoma. Comparison with the COSMIC and TCGA databases revealed overlapping genes between PMMC and melanoma, such as TP53, AHNAK2, and PMMC, as well as previously unreported mutated genes such as AKT3, SMYD4, ATAD3A, SIRPB1, XKR6, etc. In vitro experiments and co-immunoprecipitation validation revealed that P53 interacts with AKT3, SMYD4, and ATAD3A, forming a network potentially involved in the pathogenesis of PMMC. Finally, integration of significantly mutated genes and CNV analysis revealed that copy number gains may regulate the expression of SIRPB1, AHNAK2, and XKR6, thereby activating the nuclear factor kappa B (NFkB) and protein kinase (AKT) signaling pathways, which could further promote the occurrence and development of PMMC.



 This study shows that while PMMC has similarities with other melanomas, it also harbors unique mutant genes. These findings provide a basis for exploring PMMC-related molecular markers and developing personalized diagnosis and treatment plans for patients with PMMC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8169bcb0a16009a6f92e0cc377919f341d410d5" target='_blank'>
              The genetic landscape of primary malignant melanoma of the cervix using integrated bioinformatics analysis and whole-exome sequencing
              </a>
            </td>
          <td>
            Jialiu Zhou, Tian Xu, Chujun Chen, Yi Ping
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, L. Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Polyploidy considerably influences eukaryotic evolution, often leading to structural and functional imbalances during the merging of subgenomes. Studies have demonstrated asymmetric subgenome dominance in polyploids, but the effects of accessible chromatin regions (ACRs) and DNA methylation-especially in resynthesized allotriploids-remain underexplored. In this study, we generated two Brassica allotriploid hybrids by crossing Brassica napus (AnAnCnCn) with Brassica rapa (ArAr). Among the three subgenomes (Ar, An, and Cn) in F1 hybrids, gene expression was highest in the An subgenome; however, the Cn subgenome exhibited greater dominance among homoeologous triplet genes (hGenes), which correlated with ACRs in proximal and genic regions of these dominant triplets. Variations in DNA methylation alone did not fully explain the subgenomic expression biases; however, RNA-directed DNA methylation pathway genes contributed to the differences in methylation levels. Mutants of BnaDCL3 and BnaRDR2 revealed the roles of their encoded proteins in regulating non-CG methylation through 24-nt siRNA interactions in the An and Cn subgenomes. We detected higher methylation levels in the gene bodies of Cn homologs in B. napus, regardless of their expression levels. Overall, this work reveals complex interactions among ACRs, DNA methylation, and subgenome dominance, advancing our understanding of polyploid genome regulation in resynthesized allotriploids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e90b99c6926e83c457449be99aa23c8f00a1263" target='_blank'>
              Chromatin accessibility and DNA methylation impact asymmetric subgenome dominance in allotriploid Brassica species.
              </a>
            </td>
          <td>
            Chengtao Quan, Tian Xia, Lulin Wang, Ruifan Liu, Chaozhi Ma, Shengwei Dou, Cheng Dai
          </td>
          <td>2025-12-05</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Despite advancements in treatment, 5-year survival for medulloblastoma (MB) remains 60-70%. Human cytomegalovirus(HCMV) has been implicated as an oncomodulator in MB, but its impact on survival has not been explored. This study evaluates HCMV expression in pediatric MB tissue and its association with molecular risk groups and survival.



 A retrospective study of pediatric MB cases (≤19 years) from 2007-2023 was conducted using the WHO 2021 classification, the Northcott brain-tumor classifier, and the SIOP-Europe/ERN PaedCan risk stratification. HCMV immediate-early (HCMV-IE) and late-antigen (HCMV-LA) expression were assessed by optimized immunohistochemistry, and whole-genome sequencing (WGS) data from 20 tumors were analyzed for viral gene expression.



 Forty-five patients (mean age 8.2 years; 22% ≤3 years) were included: WNT (18%), SHH TP53-wildtype (18%), and non-WNT/non-SHH (64%). 20% of MB belonged to low-risk, 33% to standard-risk, and 47% to high-risk (HR-MB) groups. Four tumors exhibited MYC/N amplification. HCMV-LA positivity was found in 84% of cases, with 53% showing high expression (≥25% of cells). Cox-regression identified HR-MB (HR = 4.197, p = 0.021) and high HCMV-LA (HR = 4.334, p = 0.027) as independent predictors of poor outcomes. WGS revealed UL88 as the most abundantly expressed HCMV gene(log2[TPM+1] ≈14–15), with maximal expression in Group 3 MB.



 This study provides the first evidence linking high HCMV-LA expression to adverse outcomes and high-risk molecular features in pediatric MB. UL88 emerged as the most strongly expressed HCMV gene across MB samples. These findings suggest a potential prognostic and therapeutic role for HCMV in MB, warranting validation in larger, prospective cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60feabdb3f3fd823a43f478c57c8aa48c2982d72" target='_blank'>
              High Human Cytomegalovirus antigen Expression in pediatric medulloblastoma tissue is associated with poor event-free survival
              </a>
            </td>
          <td>
            Maria F De la Cerda-Vargas, J. Schittenhelm, Martin Ebinger, J. Zipfel, R. Beschorner, Ghazaleh Tabatabai, Martin U Schuhmann
          </td>
          <td>2025-12-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte-rich malignant tumours with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte-rich tissues, and then performed genome-wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis-regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour-specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D-loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour-specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome-wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas.
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Introduction Ph-like B-ALL is a poor-risk B-ALL subtype defined by gene expression signatures that resemble the BCR::ABL1-associated transcriptional program in the absence of the BCR-ABL1 oncoprotein. Recent studies have demonstrated that up to 80% of Ph-like B-ALLs have recurrent gene fusions involving cytokine receptors or tyrosine kinases that can be identified via cytogenetics, fluorescence in situ hybridization (FISH) or RNA sequencing. However, the inability to rapidly and accurately identify these alterations in the clinical setting is a major impediment to developing therapeutic strategies that may have efficacy in this B-ALL subtype. Here we used 88 patient (pt) samples to validate a rapid, streamlined clinical WGS assay (ChromoSeq) for B-ALL and assessed the diagnostic yield for Ph-like-associated SVs in 136 pts enrolled in the Alliance A041501 clinical trial.
 Methods ChromoSeq is a clinical WGS assay for hematologic malignancies that uses rapid laboratory workflows and focused analysis methods to identify clinically relevant SVs, copy number alterations and gene-level mutations. We updated the ChromoSeq pipeline for use in B-ALL pts and validated it using 88 retrospective B-ALL samples (80 adult/young adult and 8 pediatric from Washington University School of Medicine and St. Louis Children's Hospital), 54 of which were analyzed via RNAseq for gene fusions and a 6-gene Ph-like B-ALL signature (CA6, SPATS2L, MUC4, JCHAIN, BMPR1B, ADGRF1, NRXN3 and CRLF2). The “ChromoSeqV2” assay was then used to analyze 136 pretreatment blood or bone marrow samples from the Alliance A041501 (trial NCT03150693). These results were compared to data from RNAseq obtained for research purposes (N=35), clinical cytogenetics (N=69) and Ph-like classification via low-density microarray (LDA) card analysis (all pts).
 Results ChromoSeqV2 was validated in a CLIA-licensed setting using cohort of 88 retrospective B-ALL samples, 45 of which had ICC and WHO class-defining SVs based on G-banded karyotyping, FISH or RNAseq. ChromoSeqV2 detected 100% of the non-Ph-like SVs (37/37), including BCR::ABL1 (N=20), KMT2Ar (N=9), ETV6::RUNX1 (N=3), TCF3::PBX1 (N=1), and rearrangements involving MYC (N=2) or MEF2D (N=2). Fifteen patients met criteria for Ph-like B-ALL via cytogenetics or RNAseq either due to rearrangements of CRLF2 (N=5), JAK2 (N=2) or ABL2 (N=1) or high expression of a 6-gene Ph-like signature (N=7). ChromoSeqV2 detected a Ph-like SV in 93% of these cases (14/15) with 100% specificity in samples negative by RNAseq (N=23).
 ChromoSeqV2 was then used to profile 136 samples from the Alliance A041501 trial, where it identified ICC and WHO class-defining SVs in 87 pts (64%; 95% CI: 55-72%), including Ph-like SVs (56/136, 41%), KMT2A rearrangements (N=5), TCF3::PBX1 (N=3) and SVs involving MYC (N=3), MEF2D (N=2), DUX4 (N=2), and ZNF384 (N=1). Ph-like SVs were dominated by CRLF2 rearrangements (N=45) and included SVs involving EPOR (N=4), JAK2 (N=3), ABL1/ABL2 (N=2) and PDGFRB (N=1). IGH was the most common CRLF2 partner (N=32), followed by P2RY8::CRLF2 (N=7) and other genes (N=5). Multiple CRLF2 SVs were detected in 4 pts, and 33% (15/45) of pts with a CRLF2 rearrangement had a co-occurring CRLF2 gene mutation. The yield of ChromoSeqV2 for SVs in pts with the Ph-like expression signature via LDA card analysis was 66% (54/82; 95% CI 55%-76%) compared to 24% (11/45; 95% CI 13%-40%) by cytogenetics when these studies were successful and 70% (12/17; 95% CI 44%-90%) vs RNAseq in pts with available data. The sensitivity of ChromoSeqV2 for Ph-like SVs identified by either cytogenetics, FISH or RNAseq was 100% (21/21; 95% CI 84%-100%) and the positive predictive value of Ph-like SVs for the Ph-like expression signature by the LDA card assay was 96% (54/56; 95% CI 88%-99%).
 Conclusions Genomic assessment of 136 B-ALL pts from the Alliance A041501 trial using the ChromoSeqV2 rapid clinical WGS assay was 100% sensitive for Ph-like SVs identified by other molecular methods and had a 96% PPV for the Ph-like expression signature via LDA card analysis. The diagnostic yield of SV pts with the Ph-like signature was 66% using ChromoSeqV2, which was similar to RNAseq and superior to cytogenetics. These results demonstrate that ChromoSeqV2 is an effective approach for identifying potentially targetable rearrangements in B-ALL pts suspected of having the Ph-like phenotype. Support: U10CA180821, U10CA180882; . Servier; Pfizer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/496ee390ef7f76b531cb6af7732693c780f64abe" target='_blank'>
              Increased yield of recurrent structural variants (SVs) in Ph-like B-cell acute lymphoblastic leukemia (B-ALL) using a streamlined clinical whole-genome sequencing (WGS) assay
              </a>
            </td>
          <td>
            Andrew Hughes, Feiyu Du, Michele O'Laughlin, Gregory Sprenger, Mohamed Mahgoub, Nidhi Davarapalli, Heidi Struthers, Haley Abel, Christopher Manring, R. Harvey, C. Saygin, Jun Yin, Jun J. Yang, D. DeAngelo, S. Dinner, Richard M. Stone, Marina Konopleva, Ann-Kathrin Eisfeld, Molly Schroeder, Eric Duncavage, A. Advani, Wendy Stock, Geoffrey Uy, David Spencer
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, limiting the efficiency and precision of genome editing in many clinically relevant tissues. Here, we address this barrier by using induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. CRISPR editing outcomes differ dramatically in neurons compared to genetically identical dividing cells: neurons take longer to fully resolve this damage, and upregulate non-canonical DNA repair factors in the process. Manipulating this response with chemical or genetic perturbations allows us to direct DNA repair toward desired editing outcomes in nondividing human neurons, cardiomyocytes, and primary T cells. By studying DNA repair in clinically relevant cells, we reveal unforeseen challenges and opportunities for precise therapeutic editing. Genome editing tools can precisely introduce a specified lesion into the DNA, but ultimately rely on the cell’s DNA repair machinery to determine the editing outcome. Here, authors demonstrate how neurons’ unique DNA repair pathways impact the safety, efficiency, and precision of CRISPR edits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3be0abd5edf19f4fcddf4c37efa7ec4d61c80" target='_blank'>
              Characterizing and controlling CRISPR repair outcomes in nondividing human cells
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J. Namaganda, Manasi M. Kumar, J. Hamilton, Rohit Sharma, Karena G. Chow, Luke A. Workley, Bria L. Macklin, Mengyuan Sun, Alvin Ha, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Philip H. Dierks, Rudra Bose, Julianne Jin, Barbara S. Perez, Cindy R. Sandoval Espinoza, Madeline P. Matia, Serena H. Lu, Luke M. Judge, Brian R. Shy, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2025-11-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b270922941e97bc91f75cdb1ab9aa350f3916bd" target='_blank'>
              Mitotic bypass and endocycling promote cancer cell survival after genotoxic chemotherapy
              </a>
            </td>
          <td>
            Kevin Truskowski, Louis T A Rolle, George Butler, Margaret E Yang, Kenneth J. Pienta, S. Amend
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="


 Transcription and replication both utilize genomic DNA as a template, creating the potential for frequent interference between the two machineries. Such transcription–replication conflicts (TRCs) can compromise genome stability due to the abnormal accumulation and persistence of R-loops—three-stranded nucleic acid structures comprising a DNA:RNA hybrid and a displaced single-stranded DNA strand (Bhowmick R., et al., Mol Cell. 2023). To safeguard genome integrity, cells deploy multiple R-loop resolution factors that prevent R-loop-associated replication stress (Brickner J. R., et al., Mol Cell. 2022). Persistent R-loops are also associated with the stalling of RNA polymerase II (RNAPII) on chromatin, which further impedes replication fork progression (Zardoni L., et al., NAR 2021).
 Our observation of cells depleted of the chromatin remodelerHELLS, which displayed an accumulation of R-loops and stalled RNAPII on highly transcribed genes (Tameni A., Mallia S., et al., NAR 2024) in aggressive anaplastic large cell lymphoma (ALCL), led us to hypothesize that HELLS-dependent R-loop accumulation can induce replication stress and TRCs. To test replication forks progression, we performed DNA fiber analysis by sequentially labeling the cells with two thymidine analogs, 5-chloro-2′-deoxyuridine (CldU, red) and 5-iodo-2′-deoxyuridine (IdU, green). While control cells showed a CIdU/ldU ratio close to 1, indicating the absence of stalled forks, TLBR-2 and MAC2A HELLS-KD cells displayed an CIdU/ldU ratio close to 2, strongly indicating the presence of stalled forks in absence of HELLS. Then, we performed proximity-ligation assays (PLA) for measuring the presence of TRCs. Specifically, we used antibodies against phospho-forms of RNAPII, pSer5-RNAPII and pSer2-RNAPII, corresponding to initiation and elongation transcriptional complexes respectively and PCNA, as key component of DNA replicative machinery. We found that PLA signal between PCNA and pSer5-RNAPII, but not pSer2 RNAPII, increased in HELLS-KD cells compared to control, suggesting that the depletion of HELLS increases the collision between RNAPII and replicative complex at promoter-proximal sites before pause release. Importantly, ectopic expression of RNaseH1 in HELLS-KD cells reduced R-loop-dependent TRCs, confirming the functional involvement of R-loops in this process.
 Despite the large number of genes demonstrated or proposed to regulate R-loop homeostasis and, therefore, to impact TRCs and in turn on genome integrity, our knowledge of specific HELLS-regulated factors is still limited. To explore the mechanism contributing to TRCs and thus to genome instability, we performed gene set enrichment analysis focusing on genes belonging to functional categories related to chromatin regulation previously identified as HELLS transcriptional direct genes. Among them, mini chromosome maintenance 5 (MCM5) emerged as a potential candidate. Inducible knockdown of MCM5 significantly reduced cell proliferation, increased TRCs levels in lymphoma cells, as measured by pSer5-RNAPII–PCNA PLA foci, led to forks stall and to enhanced nuclear R-loop accumulation, phenocopying HELLS depletion. Notably, the ectopic overexpression of MCM5 in TLBR-2 HELLS-KDcells fully rescues cell proliferation reducing R-loop formation and the presence of TRCs. Additionally, perturbations observed upon MCM5-KD were dramatically reduced when R-loops were suppressed by overexpression of RNaseH1 indicating that, in absence of MCM5, the persistence of R-loop disrupts the replication fork process through TRCs, mimicking HELLS. These data highligh MCM5 as the downstream target of HELLS in replicative processes.
 Integrative analysis of MCM5, and RNAPII chromatin occupancy (ChIP-seq), alongside chromatin accessibility profiles (ATAC-seq) in TLBR-2 cells, further revealed that MCM5 was excluded from accessible, actively transcribed loci, confirming the not-transcriptional role of MCM5.
 Collectively, these findings position the axis HELLS/MCM5 as a critical regulator of replicative processes in aggressive ALCL. Disruption of HELLS/MCM5 activity induces aberrant TRCs and R-loop accumulation, ultimately compromising genome integrity and highlighting a potential therapeutic vulnerability in lymphoma cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caa8a5ddfc9df7b3d2ed0ec9481cd381c40701c0" target='_blank'>
              The chromatin remodeler hells regulates transcription-replication conflicts and genome integrity in ALK-negative anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Selene Mallia, Giulia Gambarelli, V. Manicardi, B. Donati, Emanuele Vitale, Federica Torricelli, E. Salviato, Magda Zanelli, N. Puccio, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Centromeres are the chromosomal sites at which the kinetochore forms and attaches to spindle microtubules, directing chromosome segregation. Plasmids based on centromeres can maintain stability and distribute accurately during cell division, supporting them as effective genetic tools for research and biotechnological applications. Here, the centromeric regions on seven chromosomes of Ogataea polymorpha, a methylotrophic non‐conventional yeast with great potential in biotechnology, were located by ChIP‐seq with the native Cse4. The actual centromeric sequences of chromosomes 1, 2 and 5 were characterised, which are very different from those of other eukaryotes and each unique. Although long terminal repeats (LTR) were found in these centromeres, they are ‘solo LTR elements’ and not important for the function of centromeres. Hence, the O. polymorpha centromeres were categorised into small regional centromeres. O. polymorpha centromeric plasmids were constructed for the first time, which exhibit high genetic stability and compatibility. Application potential for multigene or multicopy expression was validated by the production of uricase and triterpene squalene. This research elucidates the structural features of O. polymorpha centromeres and constructs new, stable centromeric plasmids, expanding the phenomenal diversity of centromeres and providing a powerful genetic toolbox for multi‐node and multiple‐layered engineering of cell metabolism and physiological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8595896eccb7f21123d64f19b9f60bcb2fa63adb" target='_blank'>
              Centromeric Sequences in Ogataea polymorpha Genome Enable Development of Stable Multigene Expression Plasmid Tools
              </a>
            </td>
          <td>
            Yanfei Cheng, Yuanyuan Shi, Ning Sun, Xin Zhang, Mengwei Sun, Xiuping He
          </td>
          <td>2025-11-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d6b8d33286dfd47e06aa024a1a7a5fb5466454" target='_blank'>
              Deciphering APOBEC1 in Avians: Unravelling loss events and functional insights
              </a>
            </td>
          <td>
            Aswin S Soman, N. Vijay
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea9805a426118d493f4c3e4084ab4ad0614ebdb" target='_blank'>
              DNA Repair in Ensuring Genome Stability and Human Health
              </a>
            </td>
          <td>
            N. Rechkunova, O. I. Lavrik
          </td>
          <td>2025-11-01</td>
          <td>Russian Journal of Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Nucleophosmin mutations are among the most common aberrations in acute myeloid leukemia (AML). We previously uncovered a novel pivotal role for mutant NPM1 (NPM1c) in the nucleus, where it binds chromatin at key self-renewal gene loci, such as HOXA/B and MEIS1, directly regulating oncogenic transcription. How NPM1c regulates the transcriptional activity of its chromatin targets remains unclear. To identify potential nuclear cooperating partners of NPM1c, we employed TurboID proximity labeling of NPM1c and wild-type NPM1 (NPM1wt). Mass spectrometric analysis revealed an enrichment of nucleoporins in the NPM1c interactome, with Nucleoporin 98 (NUP98) emerging as a top candidate.
 NUP98, a key component of the nuclear pore complex (NPC), has been established as a critical oncogenic player in AML as part of oncogenic fusion proteins. In these oncofusions the N-terminal FG (phenylalanine-glycine) repeat domain of NUP98 is preserved and fused to a diverse set of partner genes. NUP98-fusion partners are believed to mediate the chromatin binding specificity of the oncoprotein while the NUP98 FG repeats have been shown to form biomolecular condensates that enhance transcriptional output. Interestingly, wild-type NUP98 (NUP98wt) itself has been linked to gene regulatory functions during embryonic development and normal hematopoiesis. However, the role of NUP98wt in transcriptional regulation in the context of leukemia remains unresolved.
 To explore a potential cooperation between NUP98wt and NPM1c we first validated their interaction by co-immunoprecipitation assays. We next wanted to determine whether NUP98 and NPM1c are recruited to the same chromatin sites. To this end, we performed ChIP-seq in an NPM1c endogenous degrader cell line. These experiments revealed co-occupancy of NUP98 at NPM1c target genes and a significant reduction of NUP98 binding upon degradation of NPM1c. These findings suggest that NPM1c is required for the recruitment of NUP98 to key oncogenic target loci.
 To dissect the functional role of NUP98wt in NPM1c AML, we performed loss of function studies. CRISPR mediated knock-out of NUP98 in NPM1 mutant (OCI-AML3) cells resulted in a significant repression of NPM1c target genes, including HOXA/B and MEIS1. Given the essential nature of NUP98, we also engineered an endogenous NUP98 degrader system in OCI-AML3 and NPM1wt (MV4-11) cells to allow for the study of the effects of rapid protein depletion. Upon degradation of NUP98, OCI-AML3 cells exhibited robust myeloid differentiation (~70% increase in CD11b) and increased apoptosis (~50% increase in Annexin V+). MV411 cells on the other hand did not show the same differentiation phenotype. To characterize the immediate transcriptional consequences of NUP98 depletion, we performed nascent RNAseq after 1 and 3 hours of degradation. In the first hours of degradation we observed a rapid and significant repression of NPM1c target genes such as HOXA/B cluster genes, MEIS1, SMC4, TNRC18, SATB1/2 and UNCX, thus demonstrating the strong dependency of the NPM1c-driven transcriptional program on the presence of NUP98. In contrast, in MV4-11 cells, NUP98 loss had no effect on HOXA/MEIS1 expression, suggesting a leukemia subtype-specific role of NUP98wt.
 To delineate the domains of NUP98 required for its chromatin function, we performed a CRISPR gene tiling screen of full-length NUP98 (635 sgRNAs) and based on this we generated several domain-deletion mutants for rescue experiments in the NUP98-degrader cell lines. Retroviral overexpression of the N-terminal FG-repeat containing domains of NUP98 was sufficient to rescue cell viability and prevent the differentiation phenotype typically observed upon NUP98 degradation. Strikingly, gene expression analysis showed that the NUP98 FG-repeats were sufficient to restore the expression of HOXA10, MEIS1, and HOXB9 expression.
 Overall, our study identifies NUP98wt as a critical cofactor for NPM1c-driven oncogenic transcription. NPM1c recruits NUP98 to its chromatin targets, functionally substituting for an oncogenic fusion partner to concentrate transcriptional condensates at oncogenic loci. This establishes a previously unrecognized, subtype-specific dependency on NUP98wt in leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff1e4b1ddc4587a7bab297c9d60234d4ef12e51" target='_blank'>
              Wildtype NUP98 cooperates with NPM1c to drive oncogenic transcription
              </a>
            </td>
          <td>
            Sara Gama, Marcel Müller, H. González Dammers, Elena Haarer, Sara Peruzzi, José Gonçalves-Dias, J. Klusmann, Lu Yang, Renee Chen, Chun-Wei Chen, Stamatis Papathanasiou, B. Falini, Lorenzo Brunetti, Hannah J Uckelmann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e18dad88e53153ebbc18b5cd85b8d8ac08fba59" target='_blank'>
              Compound-specific DNA adduct profiling with nanopore sequencing and IonStats
              </a>
            </td>
          <td>
            Yrjö Koski, Divyesh Patel, Natalia Kakko von Koch, P. Jouhten, Lauri A. Aaltonen, Kimmo Palin, Biswajyoti Sahu, Esa Pitkänen
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastomas represent the most prevalent primary brain tumors in adults. Due to their highly malignant biological behavior, they are classified as grade 4 according to the World Health Organization (WHO) classification of brain tumors. Despite the progress in understanding the molecular pathogenesis of these tumors, no curative therapy has been developed for patients with glioblastoma. In this study, an integrated comparative analysis of cyclin-dependent kinase inhibitor (CDKN) 2A/B chromosomal deletion was performed on 45 glioblastomas, representing the most frequent molecular subtypes of glioblastomas, receptor tyrosine kinase (RTK) I (n=13), RTK II (n=15), and the mesenchymal subtype (MES) (n=17). The analysis of copy number variation (CNV) profiles was conducted on CDKN2A/B losses. Subsequent statistical analysis was then applied to correlate the collected data with molecular glioblastoma epigenotypes. Loss of CDKN2A/B was found 44% (20/45) of all glioblastomas, thereby, in 46% (6/13) of RTK I, 67% (10/15) RTK II, and 24% (4/17) of MES. Statistical analysis showed that loss of CDKN2A/B is significant (p < 0.01) in RTK II compared with MES. Even though CDKN2A/B does not per se function as a molecular target, there is great potential for enhancing treatment outcomes through the restoration of the tumor-suppressing capabilities of CDKN2A/B. This strategy can be employed in therapeutic interventions and is a promising avenue for research. This efficacy of this approach demonstrates high potential, as evidenced by its efficacy in other tumors, including melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63cd4d627e40e75d39c878d929a5f0f76baef680" target='_blank'>
              Loss of CDKN2A/B is a Hallmark of RTK II Glioblastomas.
              </a>
            </td>
          <td>
            Celina K Langwieder, Dorothee Hölzl, Georg Hutarew, H. Schlicker, Beate Alinger-Scharinger, Christoph Schwartz, Karl Sotlar, Theo F. J. Kraus
          </td>
          <td>2026-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2fcd1fe48f0aed8f8d06a8b12971cf0f801b0f5" target='_blank'>
              Deconvolution of haematological cancer methylation patterns reveals a predominantly non-disease related proliferation signal and uncovers true disease associated methylation changes.
              </a>
            </td>
          <td>
            H. Lalchungnunga, H. Atasoy, Edward C Schwalbe, C. M. Bacon, G. Strathdee
          </td>
          <td>2025-10-31</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cyclin D1, encoded by CCND1, is a cell cycle regulator, overexpressed in ~15% of multiple myeloma (MM) cases via CCND1::IGH super-enhancer juxtaposition through t(11;14). Cyclin D1 also plays a non-canonical role in transcriptional regulation, but in t(11;14) MM its function in gene regulation remains undefined. We investigated the global transcriptional and epigenomic consequences of CCND1overexpression in t(11;14) MM.
 To define a CCND1 signature, RNA-seq from two independent primary MM patient cohorts, CoMMpass and Mayo, was utilized to compare high CCND1 t(11;14) cases (≥300 transcripts-per-million [TPM], n=271) vs. low CCND1 hyperdiploid cases (<30 TPM, n=233). Two CCND1 knockout (KO) clones in t(11;14) MM U266B1 were also generated by CRISPR/Cas9, introducing a frameshift insertion (A) after chr11:69641478 (hg38) in exon 1. ATAC-seq and ChIP-seq (H3K4me3, H3K4me1, and H3K27ac) were performed in duplicates, and RNA-seq in triplicates in U266B1 wild-type (WT) and KO. Libraries were sequenced on Illumina NextSeq 2000 (2x51 bp for ChIP/ATAC, 2x150 bp for RNA-seq). Reads were mapped to hg38 by BWA (ChIP/ATAC) or STAR (RNA-seq), peaks called by MACS2 (FDR< 0.01), peak and gene raw counts estimated by featureCounts. Differential peaks (fold-change ≥1.5) and differentially expressed genes (fold-change ≥2) were identified by edgeR (FDR<0.05), including peaks detected in ≥2 samples and genes with TPM ≥0.1 in ≥1 sample. Interactions between promoters and distal regulatory regions were identified using publicly available chromatin interaction data in blood cell types. Pathway analysis was performed using Enrichr. Co-immunoprecipitation was performed in t(11;14) KMS12BM cells.
 Differential analysis comparing CCND1-high t(11;14) and CCND1-low MM cases in the two patient cohorts identified 21 upregulated and 21 downregulated genes associated with CCND1 expression which were also captured in the wild-typevs. CCND1 KO U266 cells. Enriched pathways (p<0.05) included Notch signaling, early/late estrogen response, epithelial-mesenchymal transition and reduced mitotic spindle and myogenesis. Dysregulated genes included those related to apoptosis, ubiquitin-proteosome, extracellular matrix, adhesion, and immune response, demonstrating a broader role for cyclin D1 in MM beyond cell cycle control. To explore the epigenomic basis of this transcriptional regulation, for each of the 42 differentially expressed genes we examined the changes of histone marks and chromatin accessibility in promoter and the interacting distal regulatory regions in the U266 KO model. Ten of the 21 genes upregulated in CCND1-high (downloaded in KO; e.g., argininosuccinate synthase 1 [ASS1], PMAIP1, MYEOV, CCR7, and DTX1) and 12 of the 21 genes downregulated in CCND1-high (upregulated in KO; e.g., UACA, RASAL2, PTPRZ1, PPT2, and IL23R) showed concordant changes in histone marks and/or chromatin accessibility, predominantly in distal regulatory regions, suggesting a chromatin-mediated mechanism of gene regulation by cyclin D1. The most prominent changes involved H3K27ac, a mark defining active enhancers. As H3K27ac is deposited by histone acetyltransferases CBP, we tested for the possible cyclin D1-CBP interaction. Co-IP confirmed this interaction, suggesting a role for this complex in chromatin remodeling and transcriptional control in t(11;14) MM.
 One gene notably upregulated in association with CCND1 is ASS1, which encodes a urea cycle enzyme responsible for endogenous arginine synthesis. Metabolomic studies demonstrated KO cells had elevated aspartate and citrulline and lowered arginine and ornithine compared to WT cells (p<0.01) demonstrating impaired urea cycle function. Further, KO cells displayed significantly reduced ASS1 protein expression, cell growth and S-phase entry compared to WT cells in arginine-replete conditions, with both WT and KO cell types further inhibited by arginine depletion (p<0.05). Together, a cyclin D1-associated gene signature was identified supporting its role in t(11;14) MM beyond the CDK4/6 mediated cell cycle control, including a role in metabolic plasticity and survival under arginine depleted conditions. These findings highlight cyclin D1 as a therapeutic vulnerability and open new avenues for personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2041ffc4d8646057f972b418e4b1a71a5a0c2" target='_blank'>
              Cyclin D1 beyond the cell cycle: A new role in t(11;14) multiple myeloma
              </a>
            </td>
          <td>
            Huihuang Yan, Aneta Mikulasova, Suganti Shivaram, Hongwei Tang, Stacey Lehman, Samuel Notch, Anderson Hans, Shulan Tian, Michael D Howe, Abiola Bolarinwa, Cinthya Mendoza Zepeda, Marcella Kaddoura, Rafael Fonseca, P. L. Bergsagel, E. Braggio, W. Gonsalves, S. V. Rajkumar, Shaji K Kumar, Francesco Maura, Daniel Rico, Brian Walker, L. Baughn
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) embryos exhibit widespread epigenetic defects, particularly aberrant DNA methylation. DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development, yet its role in SCNT embryos remains largely unknown. Here, the genome-wide 5hmC landscapes in mouse SCNT embryos are systematically profiled with parental allele specificity. It is revealed that both maternal and paternal genomes of donor somatic cells acquire a transient, sperm-like but attenuated and allele symmetric distribution of 5hmC at the 2-cell stage, distinct from the parental asymmetric pattern observed in naturally fertilized eggs. This is characterized by insufficient DNA hydroxymethylation of the X chromosome in female SCNT embryos, as well as resistance to 5hmC-associated DNA demethylation at germline imprinting control regions (gICRs). While de novo 5hmC generation is closely associated with initial DNA demethylation during somatic-to-zygotic transition, it later becomes uncoupled from ongoing methylation changes. Importantly, global elevation of 5hmC via Tet3 overexpression leads to premature activation of developmental genes at the 2-cell stage and severely impairs SCNT embryo development. These findings reveal unique dynamics and functional consequences of abnormal 5hmC remodeling in SCNT embryos, highlighting the precise regulation of 5hmC generation as a key epigenetic event for successful mammalian cloning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e7a45e185e638b752c93b5629a29c7ec2760ec" target='_blank'>
              5-Hydroxymethylcytosine Dynamics Reveals Coordinated Reprogramming of Parental Genomes and X Chromosome Dosage Balance in Mouse SCNT Embryos.
              </a>
            </td>
          <td>
            Zeming Xiang, Rui Yan, Jing Guo, Mengyao Wang, Xin Cheng, Fan Zhang, Tianzi Guo, Xin Long, Fan Guo, Dan Liang
          </td>
          <td>2025-11-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Motivation Non-coding RNA (ncRNA) plays a crucial role in gene regulation, including by forming sequence-specific RNA–DNA interactions at gene regulatory elements. One form of interaction takes place via the formation of RNA:DNA:DNA triple helices (triplexes). Accurate computational prediction of triplex formation from nucleotide sequences is an important tool in ncRNA research but remains somewhat inaccessible and complex. To address this, we created TripLexicon, a web-based interface for accessing and analyzing predicted gene regulatory RNA–DNA interactions in human and mouse. Results Predicted interactions can be accessed from RNA-, DNA-, and region-centric perspectives. For each RNA transcript, visualizations at genome and nucleotide resolution are available, providing insight into target genes and regions, as well as putative functional domains of the transcript. Predicted target genes can immediately be subjected to ontology and pathway enrichment analysis, providing rapid insight into potential functions mediated by the RNA–DNA interactions of the queried transcript. DNA and region queries are designed to identify potentially important ncRNA interactors at sites of interest. Availability and implementation TripLexicon is accessible at https://triplexicon.uni-frankfurt.de. This website is free and open to all users and there is no login requirement. All data and code is uploaded to Zenodo: https://zenodo.org/records/17143608 and the code for the webserver is available on Github: https://github.com/SchulzLab/TripLexicon.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff43ee467527f28383e03b026cf9032e05af845" target='_blank'>
              TripLexicon: prediction and analysis of gene regulatory RNA–DNA interactions
              </a>
            </td>
          <td>
            T. Warwick, Christina Kalk, Ralf Peter Brandes, Marcel H Schulz
          </td>
          <td>2025-12-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Inactivation of tumor suppressor genes (TSGs) impart a cellular fitness in cancers, including acute myeloid leukemia (AML). The silencing of TSGs without direct mutations presents challenges in cancer therapy but also presents a therapeutic opportunity to restore their function. In this study, we identified the transcriptional repressor ZBTB7A as a TSG that is downregulated in AML and associated with poor survival outcomes. Loss of ZBTB7A amplifies TNF signaling, driving a dysfunctional inflammatory state that accelerates leukemia progression. Mechanistically, the mRNA decay factor ZFP36L2 binds to the 3' untranslated region (3'UTR) of ZBTB7A, promoting its transcript degradation. To uncover therapeutic strategies, we developed a CRISPR-based screening approach coupled with in situ FISH-Flow, pinpointing KDM4 as a vulnerability to restore ZBTB7A function. Pharmacologic inhibition of KDM4 enhanced ZBTB7A expression, promoted terminal differentiation of leukemic cells, and demonstrated broad anti-leukemic activity across AML subtypes while preserving normal hematopoiesis. These findings reveal critical regulatory mechanisms of ZBTB7A and support epigenetic therapy as a promising strategy to reactivate its tumor suppressor function in hematologic cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaf7a74c25b86553fb4e494e6a32f6ab68f9d8f" target='_blank'>
              Epigenetic reactivation of a tumor suppressor program in AML.
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Arnuk, I. Aifantis, Hussein A. Abbas, A. Tsirigos, Sadik Karma, S. Rajendran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3160e385fe96505bcb09a45df54fdae63937a125" target='_blank'>
              NONO as a Sensor of Intracellular Oxidation: Relevance to Neuroblastoma Cell Death
              </a>
            </td>
          <td>
            S. S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10944325f830532ecfb5fb037e3e00b3a58c4a6" target='_blank'>
              Sema7a drives an immunosuppressive microenvironment of breast cancer via Kdm4a-mediated DNA replication regulation.
              </a>
            </td>
          <td>
            Xianan Bai, Shanshan Cai, Jie Jiang, Maojin Tian, Jianxin Du, Peiqing Zhao
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f661ff00b14a8a4b6cfed87b54b789edd9cd47" target='_blank'>
              GenomoBase: an integrated resource for Genomoviridae genomics and metagenomics
              </a>
            </td>
          <td>
            Y. Cárdenas-Conejo
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Mitoxantrone (MX) is regularly used to treat several cancers. Despite its long history in the clinic, recent studies continue to unveil novel protein targets. These targets may contribute to the cytotoxic effects of the drug, as well as potential non-canonical antitumor activity. A better understanding of MX’s cellular targets is required to fully comprehend the molecular consequences of treatment and to interpret MX sensitivity in homologous recombination (HR)-deficient cancer. Methods: Here, we evaluated MX activity in HR-deficient UWB1.289 (BRCA1−) ovarian cancer cells and surveyed the binding profile of MX using TMT-labeled quantitative proteomics and chemoproteomics. Results: Mass spectrometry (MS) analysis of cellular extracts from MX-treated BRCA1−UWB1.289 cells revealed unique downregulation of pathways instrumental in maintaining genomic stability, including single-strand annealing. Moreover, the BRCA1− cells exhibited a significant upregulation of proteins involved in ribosome biogenesis and RNA processing. Additional MS analyses following affinity-purification using a biotinylated-mitoxantrone probe corroborated these findings, which showed considerable targeting of proteins involved in genome maintenance and RNA processing. Conclusions: Our results suggest that an interplay of both canonical and non-canonical MX-antitumor activity overwhelms the BRCA1− UWB1.289 cells. Furthermore, this study characterizes the target landscape of MX, providing insights into off-target effects and MX action in HR-deficient cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed8f08582f0dd8776423c5be2d239307bc42082" target='_blank'>
              Surveying the Proteome-Wide Landscape of Mitoxantrone and Examining Drug Sensitivity in BRCA1-Deficient Ovarian Cancer Using Quantitative Proteomics
              </a>
            </td>
          <td>
            Savanna Wallin, Sneha Pandithar, Sarbjit Singh, Siddhartha Kumar, Amar Natarajan, G. Borgstahl, Nicholas T. Woods
          </td>
          <td>2025-11-14</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="B-lineage acute lymphoblastic leukemia (ALL) with CRLF2 rearrangements is characterized by high genetic heterogeneity and can be accompanied with mutations in the JAK-STAT signaling pathway, intrachromosomal amplification of chromosome 21 (iAMP21), dic (9;20), as well as IKZF1 deletions and has an unfavorable outome. Purpose. To characterize clinical, immunological and molecular genetic features of children with B-lineage ALL with CRLF2 rearrangements. Materials and methods. The study included 958 children aged 1-18 years with primarily diagnosed B-lineage ALL. Among them, a search for CRLF2 rearrangements was carried out using the fluorescence in situ hybridization, and the spectrum of genetic aberrations was analyzed using multiplex ligase-dependent probe amplification (MLPA) and targeted next-generation sequencing of RNA. Results. CRLF2 rearrangements were detected in 58 (6.1%) patients, including 42 with P2RY8::CRLF2 (72.4%) and 16 with IGH::CRLF2 (27.6%). Patients with IGH::CRLF2 fusion gene were significantly older, more often had an initial WBC ≥50×109/L and were significantly more often classified as high-risk according to the criteria of the US National Cancer Institute, however no significant differences in survival and cumulative incidence of relapse between these groups were found. JAK1/2 mutations were detected in 14.0% of CRLF2-rearranged patients, RAS family gene mutations in 23.3% and PAX5 alterations (both point mutations and fusion genes) in 18.6%. Seventeen patients (37.7%) with IKZF1 deletions were identified, while 10 of them (22.2%) had the IKZF1plus profile. With a median follow-up time of 5.3 years, the event-free survival of patients with ALL and CRLF2 rearrangements was 0.49 (standard error (SE) 0.22) and the cumulative incidence of relapse was 0.46 (SE 0.12). The most unfavorable outcome was observed in patients with IKZF1plus profile. Of note, the persistence of minimal residual disease at the end of remission induction did not have a statistically significant impact on the prognosis of patients with ALL and CRLF2 rearrangements. Conclusions. In our cohort of pediatric patients with B-lineage ALL the incidence of CRLF2 rearrangements was 6.1%. These patients showed significant genetic heterogeneity and unfavorable outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09670363fe35f81dfa3677dcfc619844dc195727" target='_blank'>
              Pediatric B-lineage acute lymphoblastic leukemia with CRLF2 rearrangements. Own experience in molecular genetic diagnostics and patient monitoring
              </a>
            </td>
          <td>
            I. Kotov, G.A. Tsaur, E.A. Chetverikova, E. Nokhrina, T.A. Mukhacheva, A. Ponomarev, Zh. V. Permikin, T. Verzhbitskaya, O. Arakaev, A. Vlasova, A.D. Tarasova, L. I. Savelyev, S. Tsvirenko, A. Popov, Y. Rumyantseva, A. Karachunsky, O. P. Kovtun, L. Fechina
          </td>
          <td>2025-11-25</td>
          <td>Laboratory Service</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In the 40 years since the discovery of the CENP proteins, many studies have examined the role of these proteins and their interactions with other chromosomal proteins of the centromere and beyond. Together, these studies have yielded vast amounts of sequencing and proteomics data. Typically, each study has focused on a single question and the majority of each dataset remains largely unexplored. Often the interesting details of publicly deposited data are left behind, buried in archives online, while more and more new data are generated. Reanalysing these databases can represent a new paradigm for investigating diverse biological pathways in unprecedented detail. Here, we explore two publicly available pan-cancer proteomic datasets to compare proteins whose abundance correlates with CENP proteins, with a particular focus on CENP-C. Our analysis confirms an expected link between CENP-C and cohesin levels but reveals a surprising and unexpected correlation between CENP-C and proteins of the inner nuclear membrane and the NuMA protein. This guilt-by-association analysis has the potential to identify proteins that act in common pathways but never associate or colocalize and may not even be expressed at the same time in cells. As an example, we show here that it can reveal unexpected links that expand our conception of centromeric chromatin beyond chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b131b447bc4a00bf7e5247f66ecd03a10d2ff6" target='_blank'>
              Sustainable integrative cell biology: CENP-C is guilty by association
              </a>
            </td>
          <td>
            N. Y. Kochanova, I. Samejima, William C. Earnshaw
          </td>
          <td>2025-11-25</td>
          <td>Chromosome Research</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Aim. The current study aims to investigate changes in the expression of proprotein convertases (PCs) genes during the development of lung tumors. PCs is a family of highly specific serine endopeptidases of mammals that process precursors of various proteins and peptides. Nine genes encoding PCs have been identified in the human genome that are essential for normal body functioning. Additionally, PCs can activate proteins involved in carcinogenesis, as well as the expression levels of their genes were shown to correlate with tumor aggressiveness and patient survival. We previously evaluated the expression of all nine PC genes using quantitative PCR on paired samples of lung tumor and adjacent normal tissue. For the first time, we identified four distinct patterns of PCs gene expression change in tumor tissue, which we have called «scenarios». For three of them, covering more than two thirds of the samples, a dominant change in the expression of one PC gene in the tumor tissue was shown. These results may indicate the existence of a limited number of possible options for changes in PCs gene expression during the malignant transformation of lung cells. However, these results need to be confirmed using expanded cohort of tumor samples. Material and Methods. To confirm our previous findings, we analyzed the expression of PCs genes using modern methods of mathematical statistics and data from three previously published studies, which evaluated gene expression through high-throughput RNA sequencing in paired tumor and normal tissue samples from 194 patients with non-small cell lung cancer. Results. Our meta-analysis confirmed that the changes in PCs gene expression in lung tumor tissue compared to surrounding normal tissue follow a limited set of possible scenarios, each having a unique profile of PCs gene expression. Conclusion. The reasons for implementing each scenario may be linked to the origin of tumors, their mutation status, characteristics of the tumor microenvironment, and other factors. Correspondingly, these scenarios may correlate, for example, with tumor aggressiveness and resistance to therapy, and therefore may potentially be used to choose the treatment approach and/or to predict the course of the disease. However, this issue requires further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3af10872808404c4c77f8841898be6b45125c83" target='_blank'>
              Alterations in proprotein convertase genes’ expression during malignization of human lung cells have a limited set of scenarios
              </a>
            </td>
          <td>
            A. A. Komissarov, I. N. Vlasov, P. A. Slominsky, I. Demidyuk
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cellular senescence is a hallmark of cancer and induces senescence-associated inflammatory responses in the tumor microenvironment (TME). CDKN1A, an important cellular senescence marker, plays significant roles in cell proliferation, invasion, migration, and apoptosis. Previous studies have implied its drug resistance role in certain cancer types. However, its impact on immunotherapy efficacy in advanced LUAD remains unclear. Using TCGA and SU2C-MARK cohorts, we investigated CDKN1A’s biological features in advanced LUAD, analyzing pathway regulation, immune infiltration, and immunotherapy associations. Single-cell RNA sequencing (GSE148071) validated TME and cellular communication differences between CDKN1A high/low expressing tumors in advanced LUAD. CDKN1A expression was positively associated with immunosuppressive environment including extracellular matrix, cancer-associated fibroblasts (CAFs) and myeloid-derived suppressor cells (MDSCs) in advanced LUAD for both TCGA and SU2C-MARK cohorts (P < 0.05). CDKN1A showed significantly positive correlations with many senescence genes in advanced LUAD (P < 0.05), which were also positively associated with endothelial cells, epithelial cells, fibroblast cells and macrophages, but negatively associated with immune cells (P < 0.05). In multivariable cox regression, patients with high CDKN1A expression had worse OS (HR = 2.74, 95% CI = 1.31–5.73, P = 0.007) and PFS (HR = 1.78, 95% CI = 1.01–3.11, P = 0.045) than those with low CDKN1A expression when treated with immunotherapy. In contrast, the high CDKN1A expression was not associated with PFS and OS in the TCGA cohort, in which the LUAD patients received standard chemotherapy, suggesting the immunotherapy predictive role instead prognostic role of CDKN1A. Moreover, single-cell analysis revealed that CDKN1A highly expressed tumors were accompanied by an enrichment of stromal and endothelial cells within the tumor microenvironment, along with enhanced activity of SMAD3/4, the downstream transcription factors of TGFB signaling. These tumors exhibited increased cell–cell communication with stromal cells (COL1A1/COL1A2-ITGA1/ITGB1/SDC1) and endothelial cells (NAMPT-ITGAS/ITGB1/INSR). CDKN1A expression was associated with cellular senescence, immunosuppressive environment and exhibited resistance to immunotherapy in advanced LUAD, suggesting a potential combination strategy with senolytic or senomorphic therapies to overcome immunotherapeutic resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f34f805e0422258abbde7dbe983d9c6f010ed5" target='_blank'>
              CDKN1A and cellular senescence are associated with immune resistance in advanced lung adenocarcinoma
              </a>
            </td>
          <td>
            Wang Shen, Feidie Duan, Shuiting Fu, Peng Cui, Yezhen Shi, Lei Zhang, Xiao Mi, Yu Xu, Guoqiang Wang, S. Cai, Xuebing Li, Xiaojun Tang
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="PURPOSE
Longitudinal measuring of circulating tumor DNA (ctDNA) during systemic treatment of metastatic colorectal cancer (mCRC) is promising for disease monitoring, but it is hampered by high costs and lacks formal demonstration of clinical usefulness.


PATIENTS AND METHODS
We leveraged METER, a novel, highly reproducible, computational workflow that infers ctDNA presence and fraction from low-pass whole genome bisulfite sequencing to investigate baseline and 8-week ctDNA dynamics in patients with RAS wild-type mCRC undergoing first-line treatment with panitumumab/FOLFOX in the VALENTINO randomized phase 2 trial. IchorCNA was used to provide a benchmark for METER.


RESULTS
A total of 154 patients were evaluable. Baseline ctDNA was detected in 72.7% of patients and was associated with significantly higher risk of progression (1.65, 95%CI: 1.13-2.42; p=0.010) and death (HR: 2.24, 95%CI: 1.37-3.66; p<0.001). CtDNA clearance at 8 weeks was observed in 80.2% of patients with baseline detectable ctDNA; persistence of ctDNA was associated with significantly higher risk of progression (2.70, 95%CI: 1.63-4.49; p<0.001) and death (HR: 3.37, 95%CI: 2.00-5.69; p<0.001). CtDNA clearance was associated with a more profound depth of response (DoR) (median -48.4% vs -41.2%; p=0.023) but not with a higher frequency of early tumor shrinkage (72.0% vs 73.7%, p=1.00). METER expanded the number of ctDNA-positive patients relative to a copy number alteration- and a variant allele frequency-based methods.


CONCLUSION
CtDNA detection and quantification with METER is a promising tool for cost-effective treatment monitoring in mCRC and can complement radiological assessment of response dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06be2814eb4fcee08947f2d86f4f47f3ad995788" target='_blank'>
              CtDNA detection with low-pass whole genome bisulfite sequencing in RAS wild-type metastatic colorectal cancer: an exploratory objective of the VALENTINO trial.
              </a>
            </td>
          <td>
            P. Manca, Marta Paoli, F. Galardi, F. Morano, Samantha Di Donato, L. Biganzoli, L. Malorni, A. Nardone, M. Ambrosini, C. Sciortino, S. Oldani, F. Ghelardi, Vincenzo Nasca, Camilla Damonte, Cecilia Villa, Roberta Fazio, Salsabil Mohamed, F. Demichelis, I. Montroni, S. Pusceddu, C. Cremolini, S. Lonardi, M. Benelli, F. Pietrantonio
          </td>
          <td>2025-12-22</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="


 Cohesin is a multimeric protein complex that regulates three-dimensional chromatin architecture and gene regulation. In myeloid malignancies, mutations in the cohesin subunit STAG2 are common and associated with poor outcomes. STAG2 mutations frequently emerge as secondary drivers in myelodysplastic syndrome (MDS), following mutations in epigenetic modifiers such as TET2 and ASXL1 in clonal hematopoiesis of indeterminate potential (CHIP). While a chronic inflammatory milieu can select for mutant hematopoietic stem cells (HSCs), the specific cause of STAG2-mutant clonal expansion in MDS remains unknown. Using models of cohesin-mutant MDS, here we describe endogenous retroviral element (ERV) reactivation as a driver of inflammation and stem cell transcriptional programs during MDS development.
 Interferon (IFN) signaling is crucial for antiviral responses and has been linked to the transcriptional reactivation of genomic repeats, which can in turn create double-stranded RNA (dsRNA). We found that patients with STAG2-mutant MDS (n=39) had increased expression of Type 1 IFN signaling compared to wild type (WT) patients (n=126). To understand the mechanism behind STAG2-mutant IFN signaling and its role in cohesin-mutant myeloid malignancies, we first examined isogenic cell line models with and without STAG2 mutations. STAG2-mutant cells showed increased Type 1 IFN signatures, an accumulation of dsRNA, and increased transcription of open chromatin in intergenic regions, as identified by ATAC-Seq. Given the abundance of retroviral sequences in these regions, we hypothesized that altered chromatin structure and subsequent expression of ERV loci could be key drivers of inflammation and subsequent progressionof STAG2-mutant MDS.
 To address this hypothesis, we characterized differentially expressed transposable elements and their chromatin state in multiple isogenic STAG2-KO leukemia cell lines. We observed transcription of a specific ERV subfamily, HERV3-int and its flanking LTR4 in the absence of STAG2. While these loci were accessible in WT cells, the STAG2 mutation resulted in a loss of the repressive H3K9me3 histone mark and gain of the activating H3K27ac mark, driving transcription, and in specific cases also translation of ERV-derived peptides, which were presented on MHC Class I. Further analysis revealed the presence of the AP-1 transcription factor motifs within these overexpressed ERV loci and binding of canonical AP-1 complex members Fos and Fosl1. We also observed a significant increase in ERV expression in STAG2-mutant MDS patients, further highlighting the association between STAG2 mutations, Type 1 IFN signaling, and ERV reactivation in MDS. Experimentally, repression of the HERV3-int/LTR4 subfamily loci using CRISPRi reversed Type 1 IFN signaling in STAG2-mutant cell lines.
 Finally, we examined the role or ERV expression in disease progression from CHIP to STAG2-mutant MDS. We generated transgenic mouse models of Stag2-mutant MDS in the context of most frequently co-occurring CHIP lesions. These models showed features of MDS, including macrocytic anemia, thrombocytopenia, and monocytosis with an increase in immature bone marrow populations and significant B-cell aplasia. Analogously to human cells, we identified increased expression of the murine IAPEY4 ERV subfamily and its associated LTR in hematopoietic stem and progenitor cells across all of our Stag2-mutant MDS mouse models. Using scRNA-Seq from sorted lineage-Sca1+kit+ cells, we mapped the highest abundance of IAPEY4 expression to short- and long-term HSCs. To probe the role of ERV reactivation in Stag2-mutant MDS disease progression, we simultaneously repressed multiple members of the IAPEY4 subfamily using CRISPRi in Tet2/Stag2-mutant HSCs in vivo. ERV repression led to decreased inflammatory gene expression signatures and reversed multiple features of the MDS phenotype, including the expansion of short-term HSCs and B-cell aplasia.
 In summary, our data suggest that STAG2 loss of function mutations lead to chromatin remodeling that promotes ERV expression. These transcripts can then induce an IFN response, which drives HSC dysfunction in MDS, highlighting a cell-autonomous mechanism for inflammation, clonal selection, and MDS progression. We also find that STAG2-mutant MDS cells present ERV-encoded peptides on MHC Class I molecules. Thus, our data not only inform the pathogenesis of MDS development but also have significant therapeutic implications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a4be84b55b57275de2a79256cdf07a06db357ed" target='_blank'>
              Activation of endogenous retroviral elements drives inflammation and transformation in STAG2-mutant myeloid malignancies
              </a>
            </td>
          <td>
            Johann-Christoph Jann, Ariel Swett, Yihua Wang, Rishika Vadlamudi, Benjamin Braun, Jennifer Karlow, Amy Wang, Lin Han, Griffin Lawrence, Caroline N. Pitton, Caroline A. Conway, M. Meggendorfer, K. Clauser, Steven A. Carr, Jenn Abelin, T. Haferlach, Kathleen Burns, Zuzana Tothova
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, K. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran C. Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Owing to their transcription factor binding sites, endogenous retroviruses (ERVs) can act as cis-regulatory-elements (CREs). By invading genomes in waves, ERVs offer a substrate for lineage-specific adaptations but also, when dysregulated, for lineage-specific disorders. Their role as CREs in rapid placental evolution, and in the human-specific placental disorder preeclampsia, may thus provide a paradigmatic exemplar. Here then we systematically identify ERV-derived CREs controlling human placental gene expression with dysregulation in preeclampsia. We identify 87 ERV-derived CREs located upstream of genes expressed in the placenta. A subset of nine, all belonging to the ERV3-MLT1/2 families and dating to the mouse–human common ancestor, are consistently dysregulated in trophoblasts from preeclampsia samples. Of the nine ERV3-MLT1-linked genes dysregulated in preeclampsia, five are novel candidates, while four were previously associated with preeclampsia, though their ERV-based regulation was not recognized. Focusing on EPS8L1, we predict enhancer activity of upstream MLT1(G1) and validate using reporter assay and genome editing. This vertebrate-specific gene is expressed in progenitor cytotrophoblasts and syncytiotrophoblasts and is overexpressed in preeclampsia, correlating with preeclampsia biomarkers and is not elevated in related pregnancy disorders. A soluble form of EPS8L1 is detectable in maternal plasma as early as between 24 weeks of gestation. EPS8L1 knockout in trophoblast in vitro is lethal, and its overexpression alters trophoblast behaviors characteristic of preeclampsia. We conclude that ERV3-MLT1functions as a trophoblast-specific CRE for several human genes and may be dysregulated in preeclampsia. As EPS8L1 has a form in maternal circulation, it may have utility in diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b721addfcf450b74bec4415619bad752451d80" target='_blank'>
              ERV3-MLT1 provides cis-regulatory elements for human placental functioning and are commonly dysregulated in human-specific preeclampsia
              </a>
            </td>
          <td>
            Rabia Anwar, Amit Pande, Manvendra Singh, Zhi Huang, Eve Hallett, Yiran Xie, A. Gellhaus, F. Herse, O. Nonn, M. Gauster, Tamás Raskó, Matthias Selbach, Stefan Verlohren, A. C. Staff, Ralf Dechend, Ulrich Pecks, S. M. Blois, Laurence D. Hurst, Z. Izsvák
          </td>
          <td>2025-11-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Type II Topoisomerases (topo II) are critical to simplify genome topology during transcription and replication. They identify topological problems and resolve them by passing a double-stranded DNA segment through a transient break in another segment. The precise mechanisms underpinning topo IIs ability to maintain a topologically simple genome are not fully understood. Here, we investigate how binding kinetics affects the resolution of two distinct forms of topological entanglement: decatenation and torsional relaxation. First, by single-molecule measurements, we quantify how monovalent cation concentration affects the dissociation rate of topo II from DNA. Second, we discover that increasing dissociation rates accelerate decatenation while slowing down relaxation catalytic activities. Finally, by using molecular dynamics simulations, we uncover that this opposite behaviour is due to a trade-off between search of target through facilitated diffusion and processivity of the enzyme in catenated versus super-coiled DNA. Thus, our findings reveal that a modulation of topo II binding kinetics can oppositely regulate its topological simplification activity, and in turn can have a significant impact in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02756b92213fd9ac89f480ffc7b784d0e58cb003" target='_blank'>
              Binding Kinetics Oppositely Regulates type II Topoisomerase Relaxation and Decatenation Activities
              </a>
            </td>
          <td>
            Cleis Battaglia, Filippo Conforto, Yair Augusto Gutiérrez Fosado, Matt Newton, E. Cutts, Davide Michieletto, Antonio Valdés
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Dysregulation of the MAF transcription factor, frequently overexpressed in multiple myeloma (MM) due to adverse translocations like t(14;16), is a well-established driver of poor prognosis. While the mechanisms are complex and involve super-enhancers (SEs), the specific downstream effectors contributing to MAF's oncogenicity remain incompletely understood. To identify critical MAF targets, we integrated genes associated with poor MM prognosis, MAF-regulated genes, and SE-associated genes, pinpointing TRIM47 as a prime candidate. Although TRIM47 is a crucial E3 ubiquitin ligase implicated in various cancers and inflammatory disorders, its functional role in MM remains undefined.
 Knockdown of MAF in MM cell lines (ARP-1, KMS-11) significantly reduced both TRIM47 transcription and protein levels. Mechanistically, ChIP-seq and ATAC-seq analyses revealed that MAF binding extends beyond the TRIM47 promoter. Multiple MAF ChIP-seq peaks co-localized strongly with H3K27ac peaks within a ±10 kb window flanking the TRIM47 coding sequence (CDS), suggesting MAF-mediated SE activation drives TRIM47 expression. Functional validation using the SE inhibitor JQ1 and sgRNA-mediated targeted deletion confirmed four critical SE sites; disruption of these sites markedly diminished both MAF/H3K27ac enrichment and TRIM47 expression. To elucidate TRIM47's function, ubiquitinomics and proteomics screens identified potential interacting partners. Subsequent validation revealed a specific and robust endogenous and exogenous interaction solely between TRIM47 and RARα (Retinoic Acid Receptor Alpha), with their protein levels often showing an inverse correlation across MM cell lines. Notably, RARα activation by all-trans retinoic acid (ATRA) has been linked to enhanced efficacy of CD38-targeted immunotherapy in MM.
 Using truncation mutagenesis, we mapped the interaction: RARα binds the SPRY domain of TRIM47, while TRIM47 binds RARα's ligand-binding domain (LBD). Critically, we demonstrated that TRIM47, via its RING domain, mediates K48-linked polyubiquitination of RARα specifically at lysine 134 (K134), targeting it for proteasomal degradation. Functionally, ATRA treatment induced CD38 expression more readily in MAF-knockdown or TRIM47-knockdown cells. This enhanced response was rescued by re-expressing wild-type TRIM47, but not by a RING domain-deletion mutant (TRIM47-ΔRING).
 We next established that TRIM47 ablation specifically licenses ATRA-driven enhancement of CD38-targeted immunotherapy through integrated in vitro and in vivo models. Critically, co-culture experiments demonstrated that while TRIM47 knockdown alone did not alter baseline CD38-CAR-T cytotoxicity or T-cell activation profiles, ATRA pretreatment of shTRIM47 myeloma cells triggered dramatically enhanced CAR-T-mediated killing accompanied by significantly amplified IFN-γ/TNF-α secretion and robust CAR-T proliferative expansion – revealing TRIM47 targeting as a molecular prerequisite for ATRA-driven CAR-T potentiation. Paralleling these findings, in vivo studies using NSG mice engrafted with luciferase-coupled shTRIM47 myeloma cells showed comparable tumor progression and CD38-CAR-T monotherapy response relative to shCtrl controls; yet when combined with ATRA, the shTRIM47 cohort exhibited profound tumor regression and survival extension unseen in control groups. Mechanistic interrogation across both systems converged on a unified pathway: TRIM47 ablation stabilizes RARα, enabling ATRA to elevate CD38 surface expression which subsequently licenses enhanced CAR-T recognition, triggers amplified effector cytokine production, and drives clonal CAR-T-cell expansion – establishing a self-reinforcing therapeutic circuit.
 In summary, our findings establish a novel MAF-TRIM47-RARα axis in MM pathogenesis: MAF orchestrates super-enhancer remodeling to drive TRIM47 expression, which subsequently targets RARα for degradation. This axis critically modulates the response to ATRA. Importantly, these results suggest that patients with t(14;16) or other high-MAF-expressing MM subtypes may exhibit diminished responsiveness to ATRA-mediated enhancement of CD38-targeted immunotherapy efficacy due to TRIM47-dependent RARα degradation. Targeting this axis represents a potential therapeutic strategy for this adverse-risk patient group.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2636bbc8566cb42ae6aaf4d94c40b29e10850104" target='_blank'>
              Super-enhancer-mediated regulation of CD38 inducibility in multiple myeloma via the MAF/TRIM47/RARα axis
              </a>
            </td>
          <td>
            Zhenfeng Dai, Yunlu Jia, Qiqi Shen, Haoguang Chen, Xinyuan Dai, Shengchun Jin, E. Zhang, Jianbiao Zhou, Gang Xiao, Wee J. Chng, Zhen Cai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 RAD51 is a highly conserved gene whose protein product, RAD51, is instrumental in the maintenance of genome integrity in eukaryotes. RAD51 is well known to function in homologous recombination repair ('HRR') in conjunction with the BRCA1 and BRCA2 gene products. It also has other less well-known functions including protection from replication stress ('RS'). RS occurs downstream of incessant oncogene signaling and is therefore typical of cancers in general, and leukemias in particular, but not normal hematopoietic cells. RS is defined by stalled replication forks, which, if unprotected, are susceptible to endonuclease-mediated digestion leading to potentially lethal double strand DNA breaks. RAD51 gathers around stalled forks and protects them by a variety of processes including fork reversal to allow survival and proliferation of cancer cells. RAD51 is, therefore, a potentially important target in leukemias: its inhibition has potential to selectively kill malignant cells.
 We have shown that lead compounds of the IBR family of RAD51 inhibitors (IBR2, IBR120 and JKYN-1) have low micromolar single agent activity in a variety of cancer cell lines including representatives of both acute ('AML') and chronic ('CML') myeloid leukemia. Furthermore, these compounds often exhibit strong synergy and at least additivity in combination with both on-label and off-label use of FDA-approved compounds, as well as investigational molecularly-targeted agents, in relevant cell lines; these include FLT3 inhibitors quizartinib, gilteritinib and tuspetinib ((in MV-4-11 and HL60 cell lines) and BCR/ABL inhibitors (imatinib and regorafenib) in the K562 cell line (AACR 2025). Interestingly, strong synergy was demonstrated with JKYN1 and quizartinib, and separately, tuspetinib, in the FLT3 WT line HL60, but only additivity between JKYN-1 and gilteritinib in that cell line; all three of these compounds were extremely potent in the FLT3 mutated MV-4-11 cell line; additivity with JKYN-1 was nonetheless demonstrable, but not synergy. IBR2 exhibited low micromolar single agent activity in the K562 CML line and synergized with both imatinib and (separately) the off-label agent regorafenib. Regorafenib was also strongly synergistic with IBR2 in multiple other solid tumor cell lines, both within and beyond its label in colorectal cancer.
 We have confirmed the IBRs inhibit multiple RAD51 functions including hydroxyurea-induced RAD51 focus formation, HRR by DR-GFP assay, and RS protection by RPA focus formation and a DNA fibre assay. They markedly reduce intracellular RAD51 levels. All the IBRs tested to date bind RAD51 strongly by surface plasmon resonance (‘SPR‘), especially JKYN-1, and JKYN-1 is more potent in this and other assays than the standard classical RAD51 inhibitor B02. Our hypothesis is that both the single agent activity and synergy is due primarily to deprotection of RS.
 JKYN-1 is a partially optimized (more soluble and more potent) version of IBR120 but is not resistant to degradation by liver microsomes or sufficiently capable of entry into target cells. We have therefore selected 15-20 IBR analogues amongst a pool of ~35,000 created by generative chemistry (InSilico Medicine, Boston) for synthesis and in vitro testing. Anti-leukemic properties of these and our current IBRs will be tested on primary patient derived cells from genetically diverse AML patients. The assay will target stemness properties of AML patients in a high throughput screen designed to provide clinically actionable data for relapsed AML. Treatment dose and combinations will be pre-screened using a highly predictive cell line as a surrogate of human cancer stem cells for subsequently AML patient panels of leukemic stem cells to be examined. These studies will focus on single agent anti-proliferative activity and synergy with a suite of standard and investigational antileukemic drugs including cytosine arabinoside, daunorubicin, quizartinib, gilteritinib, tuspetinib, venetoclax, and azacitadine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79dc795d42c064ddfb139b88cd12f7483902f754" target='_blank'>
              Inhibitors of RAD51 as potential novel therapies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Mick Bhatia, P. Ferguson, A. Xenocostas, James Koropatnick, Morgan Black, Mark D. Vincent
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5ca99f1d92208d16804d01a5ade96f5db584369" target='_blank'>
              NAT10-mediated N4-acetylcytidine (ac4C) modification of PIK3R2 mRNA promotes malignant progression of glioblastoma.
              </a>
            </td>
          <td>
            Xiannan Meng, Tao Dong, Jiayu Xu, Kaiwen Tian, Wendong Yang, Zixuan Liu, Yongjing Qian, Dingding Liu, Changxiu Chen, Jin Bai, Hong-Mei Yong, Xiaojin Wu, Zhigang Shen
          </td>
          <td>2025-12-17</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Genotoxic therapy resistance in glioblastoma (GBM) is mediated by rapid DNA repair. We hypothesized that non-malignant cells in the heterogenous GBM microenvironment assist rapid DNA repair in neoplastic cells. Supporting this hypothesis, intracranial GBM tumors (orthotopic GBM PDXs and syngeneic murine GBMs) in mice had rapid DNA repair and poorer survival after radiation compared to flank GBM tumors of identical genotype. Gamma-H2AX immunofluorescence analysis of irradiated intracranial tumors revealed spatial heterogeneity in resolution of DNA damage. Coupling these analyses with spatial transcriptomics in irradiated intracranial syngeneic murine GBM tissues suggested that regions with fast repair were rich in microglia. Coculture of patient derived GBM cells with human primary microglia ex vivo enhanced DNA repair post irradiation in GBM cells. This phenotype was independent of cell contact, suggesting a microglia-derived soluble factor. Indeed, mass spectrometry analysis revealed that microglia secrete acetylated amino acids that are then consumed by neoplastic cells. Supplementation of acetylated amino acids hastened the repair of radiation-induced DNA damage and promoted an invasive phenotype. Metabolomic analysis of GBM cells supplemented with acetylated amino acids showed that glioma cells re-route glucose utilization towards nucleotide synthesis. Spatial transcriptomic analysis of human GBM tissue explant treated with radiation and lung cancer metastasis to the brain resected after irradiation, showed higher expression of genes essential for synthesis of the acetylated amino acids in myeloid cells. Furthermore, supplementation of acetylated amino acids increased acetylation of H2AX post radiation in GBM cells. Hence, we show that microglia in the GBM microenvironment secrete acetylated amino acids which are consumed by GBM cells and used to repair radiation induced DNA damage by repurposing glucose for nucleotide synthesis and increase H2AX acetylation, thereby conferring treatment resistance to the GBM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef28425f01748e3f178727e96499580896a377fc" target='_blank'>
              TMIC-85. Microenvironment-derived acetylated amino acids promote glioblastoma DNA repair by rerouting glucose fates and enhancing histone acetylation
              </a>
            </td>
          <td>
            Sravya Palavalasa, Jack Freeman, Aditri Gokul, Ava Singer, A. Scott, Kari Wilder-Romans, Emily Baker, Navyateja Korimerla, Alexandra M O'Brien, Angelica Lin, Erik R Peterson, Ningning Liang, Jie Xu, V. Ravikumar, Peter Sajjakulnukit, Harrison Wong, A. Andren, Li Zhang, Sunita Shankar, Sagnik Bhadury, J. Heth, Weihua Zhou, Arvind Rao, Deepak Nagrath, W. Al-Holou, C. Lyssiotis, D. Wahl
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [24, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>