<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-16 10:03:18 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>48</td>
          <td>107</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>97</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>127</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>416</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>349</td>
          <td>160</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a circular, double-stranded DNA molecule distinct from chromosomal DNA, primarily found in cancer cells as a subtype of extrachromosomal circular DNA (eccDNA). Unlike eccDNAs found in normal cells, cancer-associated ecDNAs are large, clonal, and carry complete oncogenes. These ecDNAs are increasingly recognized as crucial drivers of cancer pathogenesis, contributing significantly to the evolution of tumor heterogeneity and the acquisition of therapeutic resistance through mechanisms such as high-level gene amplification and altered gene regulation. Historically, the understanding of these mobile genetic elements in cancer has been limited. This review synthesizes current knowledge on ecDNA’s structural and functional features, formation mechanisms and roles in cancer initiation, progression and therapeutic resistance. Moreover, we summarize five emerging therapeutic approaches that target ecDNA to inform future cancer research and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb27a6296222e1300898cc02edbec7c0adb86fa2" target='_blank'>
              Extrachromosomal DNA and cancer: function, formation, and clinical implications
              </a>
            </td>
          <td>
            Yucai Wu, Rui Rui, Tai Tian, Jiaying Zheng, Shi-ming He, Liqun Zhou, Xuesong Li, Yanqing Gong
          </td>
          <td>2026-01-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated genomic segments, identify multi-way interactions, and distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>76</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Oncogene amplifications fuel some of the most lethal, therapy‑refractory cancers, yet remain clinically untargeted. We report a single‑guide CRISPR/Cas9 strategy that converts the sheer copy‑number excess of oncogene amplicons into an Achilles’ heel. A solitary intronic double‑strand break is innocuous in diploid genomes but collapses oncogene amplification‑positive cells across neuroblastoma, small‑cell lung and colorectal carcinoma models, driving > 90% loss of viability, G₂/M blockade and catastrophic DNA‑damage signalling. Amplified‑locus cleavage rewires transcription toward cell death activation, necroptosis and cGAS–STING–mediated immunogenic cell death, enabling dendritic‑cell cross‑priming and T‑cell activation and proliferation. In xenografts, delivery of the intronic sgRNA shrinks tumours by 90%, prolongs survival and remodels the innate tumour microenvironment. Deep sequencing confirms negligible off‑target editing, and combination with doxorubicin achieves supra‑additive killing. These findings establish amplification density, not sequence content, as a tractable, tumour‑exclusive target and unveil a dual‑action platform that is simultaneously cytotoxic and immunostimulatory. Editing of tumor amplifications therefore offers a blueprint for translating copy‑number aberrations into precision genome‑editing therapies for treatment‑resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/794649b9036f93693e77ad613d6e85889d3eff72" target='_blank'>
              Targeting extrachromosomal DNA in human cancers.
              </a>
            </td>
          <td>
            I. Wong, Hyerim Yi, Bruno Melillo, Benjamin F. Cravatt, Howard Y. Chang, P. Mischel
          </td>
          <td>2026-02-03</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="The human genome is not confined to chromosomes. Extrachromosomal circular DNA—eccDNA and its megabase-scale counterpart ecDNA—is a persistent, transcriptionally active, and mutationally accelerated compartment present in normal and neoplastic cells. Yet every stage of genomic analysis, from DNA extraction to variant annotation, is engineered for linear, diploid templates. This review argues that circular DNA mutations are not rare events; they are systematically erased by methodologies that mistake the map for the territory. We synthesize evidence that circular DNA exhibits a mutational spectrum fundamentally distinct from chromosomal DNA: microhomology-driven indels, complex structural rearrangements within single molecules, and interchromosomal chimeric fusions that generate novel oncogenic reading frames. Its mutation rate is accelerated by rolling-circle replication, suppressed homologous recombination, and torsional stress—quantified as μ_eff = μ_bp × CN × f_rep, exceeding chromosomal rates by two orders of magnitude. Experimental isolation and detection methods each carry specific biases that distort mutational discovery. Computational reconstruction requires graph-based assembly and copy-number-normalized burden metrics. Validation by inverse PCR, mutation-specific FISH, and single-cell sequencing reveals that ecDNA mutations exhibit non-Mendelian inheritance yet drive transient transcriptional dominance. Major barriers include the absence of a circular reference genome, indistinguishable replication errors, and zero clinical standardization. Nevertheless, ecDNA amplifications occur in 14% of cancers, confer prognosis independent of chromosomal status, and serve as privileged sites for resistance mutations. eccDNA burden tracks cellular age and genotoxic stress. Emerging therapies targeting ecDNA segregation or transcription are plausible but lack specificity. The central conclusion is not that circular DNA matters—it is that the genome is not what we thought it was.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/829f7f2d91e8a0a5e3b416bb5b2bd0074f23ae03" target='_blank'>
              The Hidden Mutational Landscape of Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Lali Latsabidze
          </td>
          <td>2026-02-12</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is an aggressive malignancy associated with single-nucleotide variants in VHL, PBRM1, and SETD2. However, the role of structural variation (SV)—which have broader genomic impacts—and 3D genome architecture in ccRCC remains inadequately understood. Here, we reported a comprehensive molecular characterization. Through multi-omics analysis, we identify novel SV-associated oncogenic targets and reveal multidimensional 3D genome reorganization during ccRCC progression. We elucidate the dynamic interplay between SV and 3D chromatin architecture, demonstrating how structural rearrangements drive oncogenic dysregulation through 3D genome reorganization. Notably, an unrecognized pathogenic enhancer hijacking event is discovered and experimentally validated, leading to constitutive activation of the proto-oncogene SEMA5B. Furthermore, we developed a machine learning-based prognostic framework using enhancer hijacking signatures. Collectively, this work establishes a valuable resource for ccRCC research by elucidating how SVs and 3D genome reorganization collectively drive oncogenesis, and translates these findings into a clinically applicable prognostic tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome 3p (chr3p) is frequently deleted in multiple cancers, indicating the presence of shared tumor suppressors. Analysis of genomic alterations in 33 different cancer types implicates the deletion or deleterious mutations of SET-domain-containing 2 (SETD2) at chr3p21 in significantly facilitating the formation of isochromosomes, consisting of two identical mirror-imaged arms, thereby promoting genomic instability conducive to large-scale chromosomal rearrangements and rapid cancer genome evolution. Fracturing of dicentric isochromosomes during cell division is pervasive and follows the dynamic fragmentation pattern of solids under impulse. Across cancers, isochromosomes form most frequently on chr8 to amplify the MYC-containing q-arm and chr17 to delete the TP53-containing p-arm. In the most aggressive uveal melanoma (UVM) subtype, chr3 deletion also includes MITF, a critical melanocyte differentiation and survival factor, and co-occurs with chr8q amplification. We demonstrate that MITF is a master transcriptional regulator of GNAQ/GNA11, mutated in 90% of UVM patients, and the associated synthetic-lethal genes identified by recent CRISPR screening studies. MITF maintains MAPK and calcium homeostasis in UVM, and its deletion is thus accidental, creating an early crisis during oncogenesis. We further show that MITF, MYC, and GNAQ/GNA11 form coupled regulatory feedback loops in the melanocyte lineage, and MITF deletion in UVM creates acute dependency on MYC-mediated rescue via chr8q amplification, often as a consequence of isochromosome formation. The discovered feedback loops predict both overall and relapse-free patient survival within the most aggressive UVM subtype, explain sensitivity to therapeutic gene perturbations, and inform effective combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, Jun S. Song
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="TFE3 translocation renal cell carcinoma (tRCC), an aggressive kidney cancer driven by TFE3 gene fusions, is frequently misdiagnosed owing to morphologic overlap with other kidney cancer subtypes. Conventional liquid biopsy assays that detect tumor DNA via somatic mutations or copy number alterations are unsuitable for tRCC since it often lacks recurrent genetic alterations and because fusion breakpoints are highly variable between patients. We reasoned that epigenomic profiling could more effectively detect tRCC because the driver fusion constitutes an oncogenic transcription factor that alters gene regulation. By defining a TFE3-driven epigenomic signature in tRCC cell lines and detecting it in patient plasma using ChIP-seq, we distinguished tRCC from clear-cell RCC (AUC = 0.86) and samples of individuals without evidence of cancer (AUC = 0.92) at low tumor fractions (<1%). This work establishes a framework for noninvasive epigenomic detection, diagnosis, and monitoring of tRCC, with implications for other mutationally quiet, fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181d702745222ebeaadeb7b01c3a274dc1851478" target='_blank'>
              Cell-free DNA epigenomic profiling enables noninvasive detection and monitoring of translocation renal cell carcinoma
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ze Zhang, Prathyusha Konda, A. Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Pandey, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, B. Fortunato, Rashad Nawfal, Razane El Hajj Chehade, J. O’Toole, J. Horst, D. Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D. Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, S. Viswanathan
          </td>
          <td>2026-02-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c21032e04b711532f2c1af652faef7052c5a06e" target='_blank'>
              PanCNV-Explorer: Deciphering copy number alterations across human cancers
              </a>
            </td>
          <td>
            Nadine S. Kurz, Kevin Kornrumpf, Anna-Rosa Krüger, Jürgen Dönitz
          </td>
          <td>2026-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63674f265b1ac623907383e45f60b5724b528b86" target='_blank'>
              Recurrent Escape from Osimertinib-Induced Senescence Promotes Genomic Instability Associated with Therapeutic Resistance
              </a>
            </td>
          <td>
            H. McDaid, Nazia S Jamil, Qualia Hooker, Nadjet Cornejal, Dean Hosgood
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The human genome is dominated by noncoding sequences, most of which are poorly conserved across species. How genetic information is distributed between coding and noncoding regions remains a fundamental unresolved question. Using CRISPR saturation mutagenesis at base-pair resolution, we mapped the functional fitness landscape of the 10-kb human MYC locus with a near-PAMless, high-fidelity SpRY-Cas9. This unbiased interrogation revealed that the majority (67%) of functionally essential base-pairs in this locus are noncoding. Paradoxically, the phenotypic impact of noncoding sequences correlates inversely with evolutionary conservation, driven in part by rapidly diverging cis-regulatory DNA elements that remain functionally constrained in humans. Within this landscape, we identified an ultraconserved RNA element in the 3’ untranslated region (UTR) that is indispensable for MYC-dependent cancer cells. Remarkably, steric-blocking antisense oligos targeting this RNA element selectively eliminate MYC-addicted cancer cells by suppressing MYC function without reducing MYC abundance. Mechanistically, this 3′ UTR element promotes perinuclear localization of MYC mRNA and efficient nuclear import of the short-lived MYC protein, enabling its function as a nuclear transcription factor. Together, these findings highlight noncoding sequences as major carriers of functional genetic information, provide a comprehensive fitness map of the MYC locus, and uncover a therapeutically actionable RNA element that disables MYC-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9f715a277680bacb435d2c9602b972a237e4aca" target='_blank'>
              Decoding the MYC locus reveals a druggable ultraconserved RNA element
              </a>
            </td>
          <td>
            Peiguo Shi, Feiyue Yang, Fnu Tala, Wesley Huang, Alexis O. Aparicio, Colin H. Kalicki, Aditi Trehan, Michael R. Murphy, Esther Rotlevi, Linqing Xing, Muredach P. Reilly, Jianwen Que, Xuebing Wu
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce9b91a05b2c305ecf72c3bf59327984c56dba6" target='_blank'>
              Functional inactivation of the telomerase chaperone TCAB1 primes cells for the activation of ALT in osteosarcoma
              </a>
            </td>
          <td>
            Joshua Keegan, Sydney Sorbello, Joakin O. Mori, Shugo Muratani, I. Leshchiner, Christopher M. Heaphy, R. L. Flynn
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Eukaryotic genomes are abundant in satellite DNA (satDNA): large blocks of tandemly-repeated sequences that accumulate in heterochromatic genome regions. SatDNAs are dynamic in their genomic location and abundance across species. Some satDNAs overlap essential genome regions such as centromeres and telomeres, but even pericentromeric satDNA can have effects on phenotypes, raising questions about their functional significance. However, it remains unclear whether these effects depend on satDNA sequence, copy number, higher-order structural organization, or genomic context. The highly repetitive nature of satDNA arrays has long hindered detailed genomic and genetic analyses. Recent advances in long-read sequencing now facilitate both the detailed characterization of satDNA structure and the development of more targeted approaches to genetic analysis. Here we present a sequential CRISPR/Cas9-based strategy to make mutations in satDNA arrays and demonstrate its utility using an autosomal pericentromeric satDNA in Drosophila melanogaster called Responder (Rsp). Rsp is the target of a sperm-killing male meiotic driver, Segregation Distorter (SD), where sensitivity to sperm killing positively correlates with Rsp copy number. Using our CRISPR/Cas9 approach, we generated an allelic series of Rsp deletion and expansion variants in two genetic backgrounds and examined their effects on spermatogenesis. Our approach produced precise satDNA variants efficiently, with minimal detectable off-target effects. The resulting mutations affect sensitivity to SD that scale with Rsp copy number. This work establishes a new framework for experimentally dissecting satDNA function and provides insights into the evolutionary and functional roles of satDNA in genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Long interspersed element-1 sequences are normally silenced in healthy cells but frequently reactivated in cancer, where they contribute to genomic instability and transcriptional dysregulation. However, the determinants of their transcriptional heterogeneity in non-small cell lung cancer remain poorly defined. In this study, we systematically characterized locus-specific LINE-1 transcript expression across a large panel of non-small cell lung cancer cell lines to determine how genomic context shapes retrotransposon activity. We show that LINE-1 transcription is dominated by evolutionarily young elements and follows conserved chromosomal patterns across tumor subtypes, with the highest activity observed in lung squamous cell carcinoma. Importantly, LINE-1 expression is anchored by a small number of conserved genomic hotspot loci that remain active across tumor backgrounds and are located near cancer-relevant genes. These findings provide a genome-scale framework for understanding retrotransposon regulation in cancer and highlight conserved LINE-1 loci as potential markers of genome instability and regulatory vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Yicong Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract Despite the many advances in single cell genomics, detecting structural rearrangements in single cells, particularly error-free sister-chromatid exchanges, remains challenging. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells, as a platform for mapping mitotic crossover (CO) and resulting genome instability events. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic CO outcomes. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational COs in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair and structural variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a3596ca3c9b197876145ec4548ef0ebb22dbce" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2026-02-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450603a806f69ebb4194f3b7b3ab19ed0a5b5513" target='_blank'>
              A holocentric pangenome links karyotype evolution to meiotic recombination
              </a>
            </td>
          <td>
            Meng Zhang, Stefan Steckenborn, Marco Castellani, Laia Marín-Gual, A. S. Câmara, Laura A. Robledillo, Letícia Maria-Parteka, N. Sargheini, Magdalena Marek, Eduardo Chacón-Madrigal, Andrés Gatica Arias, L. Felix, William W. Thomas, B. Huettel, A. Pedrosa‐Harand, John T. Lovell, André L. L. Vanzela, Aurora Ruiz-Herrera, A. Marques
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A. Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Human artificial chromosomes (HACs) are inherited through cell divisions alongside natural chromosomes, serving as tools for interrogating chromosomal elements and as vectors for large genetic cargoes (1–5). Despite recent progress (6–8), large (i.e., multiple Mb) HACs have not been reported. Further, HAC formation via epigenetic seeding of centromeric chromatin currently requires prior engineering of recipient human cells (6–8), hampering their potential deployment in many useful cell types. Here, we designed, built, and delivered to human cells a 2 Mb HAC construct that is ∼3 times larger than the prior generation. We also report a robust epigenetic centromere seeding approach that initiates immediately upon delivery to the human cell cytoplasm and bypasses genetic engineering of target cells. The HACs are then faithfully inherited in the absence of selection. Thus, formation of functional centromeric chromatin in the same cell cycle of HAC delivery drives high efficiency HAC formation. Teaser A system for efficient protein-only centromere seeding permits visualizing the first step of HAC formation">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/755b604be820a14cd765634bf7256cd13a7ee21b" target='_blank'>
              Protein-only centromeric chromatin assembly streamlines human artificial chromosome formation
              </a>
            </td>
          <td>
            Gabriel J. Birchak, P. K. Allu, Prakriti Kashyap, G. K. Anbalagan, Shu-Cheng Chuang, Glennis A. Logsdon, Daniel G. Gibson, John I. Glass, Ben E. Black
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="MYC dysregulation occurs in most cancers provoking, profound changes in gene expression. Although sometimes considered to be an E-box-dependent transcription factor, an alternate model posits MYC to be a universal amplifier of active genes. Although MYC is associated with accelerated pause release, the full extent of its participation in the transcription cycle remains poorly illuminated. MYC also stimulates topoisomerases to resolve topological issues that complicate DNA transactions; whether and how this stimulation is coordinated with or independent of its transcriptional role has not been examined. We have developed a genetic tool that discriminates between MYC’s ability to activate versus amplify reporter gene expression in any cell. This system enables the interrogation of genes, cofactors, and compounds that execute or modulate MYC activity. Using a combination of biochemical and cellular assays, we reveal the dynamic interplay between MYC-driven transcription amplification and DNA supercoiling. The early stages of transcription are highly sensitive to the level of DNA supercoiling. Reducing the activity of TOP1 either through genetic knockdown or low-dose inhibition potentiates transcription amplification by MYC. This enhancement is associated with pre-initiation complex (PIC) stabilization by DNA supercoiling. MYC helps to mobilize these stabilized PICs. Significance Statement MYC is dysregulated in most cancers, yet its precise functions in transcription are poorly understood. Using a genetic system that distinguishes transcriptional activation from amplification, we reveal a functional interplay between MYC-driven amplification and DNA supercoiling. Reduced topoisomerase I activity increases DNA-supercoiling that in turn stabilizes PICs to provide more substrate for MYC-dependent transcription amplification. Thus, DNA topology as a critical regulator of MYC-dependent transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0f98bd86315e8e5aa3b48daf84b94980e7dd61" target='_blank'>
              Dynamic Supercoiling Sponsors Transcription Amplification by MYC
              </a>
            </td>
          <td>
            R. Jha, Fedor Kouzine, Bo Wang, James D. Phelan, S. K. Das, Brian A Lewis, D. Levens
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01dc9e9620e1364bb485bdf2ea0135eedca2399" target='_blank'>
              Chromosomal instability and chromosome 17p loss drive convergent NDE1 synthetic lethality in metastatic cancer cells
              </a>
            </td>
          <td>
            Caroline M. Teddy, Jacob P. Hoj, Julia D Caci, Min Lu, Shane T. Killarney, Tal Ben-Yishay, U. Ben-David, Kris C. Wood
          </td>
          <td>2026-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6dd597d404f6624bed3a43a3ecd045b834d7d8b" target='_blank'>
              Integration of an anellovirus genome in the SKNO-1 acute myeloid leukemia cell line
              </a>
            </td>
          <td>
            Na Cui, S. Goya, E. Piliper, Alexander L. Greninger
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41a324c4452deb74bae1f6fd5783a031c4bb944" target='_blank'>
              Aurora kinase A is a synthetic lethal target in FANCA-deficient cancers
              </a>
            </td>
          <td>
            Dilara Akhoundova, Martín González-Fernández, Aavash Baral, Andrej Benjak, Carmen Perry, Dinda Shezaria Hardy Lubis, Ladina Hörtensteiner, Saskia Hussung, Lea Lingg, S. Maletti, Aino Paasinen-Sohns, Marika Lehner, B. Dislich, E. Vassella, Simone de Brot, Therese Waldburger, Petros Tsiridis, Carmen Cardozo, Ralph Fritsch, C. Pistoni, Seynabou Diop, Phillip Thienger, Taina S. Kaiponen, Muriel Jaquet, Mariana Ricca, S. Sivakumar, Christopher McDermott-Roe, David E. Root, P. Francica, Sven Rottenberg, M. Rubin
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 With the development of whole genome sequencing and CRISPR technology, it is now possible to identify sequence features that are unique to an individual’s cancer genome and to precisely target them. Such personalized precision therapy holds the promise of ushering in a new era of safe and effective cancer treatments with minimal side-effects. To specifically target the cancer genome, we have developed a CRISPR-based therapeutic approach, “KLIPP,” which is designed to target structural variants junctions (SVJs) specific to cancer genomes, with few, or no off-target effects expected in normal cell. KLIPP uses a “split enzyme” approach consisting of a dead Cas9 endonuclease (dCas9) fused to the endonuclease Fok1 where two Fok1 endonucleases need to homodimerize to become active. To “nucleate” and activate these complexes, sgRNAs are designed to bind sequences flanking cancer-specific SVJs, bringing two Fok1-dCas9 complexes together to induce double-strand breaks (DSBs). While any SVJ in the cancer genome may be targeted with KLIPP, we have found that junctions in oncogenic fusion genes represents a particularly valuable target. We show effective targeting of the EWS::FLI1 fusion oncogene in Ewing sarcoma cells using lipid nanoparticle delivery of the Fok1-dCas9 mRNA and SVJ-targeting sgRNA, leading to the induction of DSBs, diminished expression of the fusion oncogene and loss of cell survival. Other fusion oncogenes that we are currently targeting with KLIPP involve TMPSS2::ERG in prostate cancer, DNAJB1::PRKACA in fibrolamellar liver cancer and BCR::ABL1 in leukemia. We believe that KLIPP is a safe and cancer-specific approach for precision targeting of fusion oncogenes in cancer cells. This paradigm-shifting personalized therapy could revolutionize how we treat fusion-driven cancers without inflicting long-term side effects.



 Huibin Yang, Radhika Suhas. Hulbatte, Natalie Gratsch, Ann Urzynicok, Ashley Sutter, Meyer Cusnir, Mario Ashaka, Ishwarya Venkata. Narayanan, Michelle Paulsen, Anna Schwendeman, Tom E. Wilson, Erika Newman, Mats Ljungman. KLIPP: Targeting fusion oncogenes with CRISPR [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d57ac69015505f843b828afc6c4dc168f0ae8afa" target='_blank'>
              Abstract PR003: KLIPP: Targeting fusion oncogenes with CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Anne Urzynicok, Ashley Sutter, Meyer Cusnir, Mario Ashaka, Ishwarya Narayanan, Michelle Paulsen, A. Schwendeman, Tom E. Wilson, E. Newman, Mats Ljungman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Hepatocellular carcinoma (HCC) is the most common primary liver malignancy worldwide and is associated with a high mortality rate. The unique genetic features and structural properties of extrachromosomal circular DNA (eccDNA) offer a promising approach for early cancer diagnosis.


OBJECTIVE
This study aims to investigate the potential of eccDNA as a diagnostic marker by identifying its distinguishing characteristics using a two-dimensional cell culture model.


METHODS
EccDNA was isolated from the supernatant of HepG2 and THLE-2 cell cultures. Long-read sequencing and bioinformatic analysis were employed to detect and characterize eccDNA. The size distribution, chromosomal origin, and genomic annotation of eccDNA from both cell types were compared. PCR was performed to validate DNA fragments at the junctions of the closed-circular forms.


RESULTS
The total number of cell-free eccDNA derived from HepG2 cells averaged 2,669,673 bases, encompassing 2,542 sequences, while THLE-2 cells averaged 857,718 bases with 975 sequences. Size distribution analysis revealed that most cell-free eccDNA from both cell types ranged from 351 to 600 bp. Bioinformatic analysis revealed that approximately 80% of eccDNAs corresponded to gene coding regions. Ten eccDNAs were the most frequently detected in HepG2 cells, with four eccDNAs harboring CDC27P11, RAC1P3, LOC112268123, and LOC124902279 genes successfully validated and uniquely detected in HepG2 cells, suggesting their potential as biomarkers for HCC diagnosis.


CONCLUSION
Distinct eccDNA types were identified in HepG2 cells, which may serve as promising biomarkers for HCC diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2857a0fc338d1571175ba6c4c210b5ca1dafdfd9" target='_blank'>
              Identification of Extrachromosomal Circular DNA Isolated from Cell Culture Supernatant and Its Potential Applications for Hepatocellular Carcinoma Diagnosis.
              </a>
            </td>
          <td>
            Suchitraporn Sukthaworn, Suchanuch Ondee, Opas Choksupmanee, Suparat Taengchaiyaphum, Prasert Yodsawat, Ponsit Sathapondecha
          </td>
          <td>2026-01-21</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent advances in genomic technologies have greatly enhanced our understanding of genotype–phenotype relationships and improved diagnostic of genetic diseases. However, the dissection of complex structural variants remains challenging due to the limitations of current methods in resolving their breakpoint and interpreting phenotypes involving multiple disrupted genes. In this study, we demonstrate how an integrative approach—combining cytological, genomic, and transcriptomic methods—enables the detection, structural and functional characterization of a complex structural variants affecting the MBD5, USP34, and XPO1 genes. Our findings underscore the utility of the Exo-C, a modified chromosome conformation capture technique, in resolving complex rearrangements. We also report, for the first time, a composite neurodevelopmental phenotype resulting from the combined effects of MBD5-associated intellectual disability and 2p15p16.1 microdeletion syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/308ec9e744b73dd3ecd51f7b2385a3a46667c2cb" target='_blank'>
              Pangenome analysis reveals the evolutionary dynamics of repeat-based holocentromeres
              </a>
            </td>
          <td>
            Piotr Włodzimierz, Estela Perez-Roman, A. S. Câmara, Laura A. Robledillo, Gokilavani Thangavel, Meng Zhang, Jacob González Isa, L. M. Parteka, Marco Castellani, B. Huettel, André L. L. Vanzela, Ian R. Henderson, Alexandros Bousios, A. Marques
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Neuroblastoma RAS viral oncogene homolog (NRAS) mutations occur in 20-30% of cutaneous melanomas, defining a distinct molecular subtype. While NRAS mutations drive unique gene expression programs, locus-specific transcriptional regulation has not been systematically explored. This study characterized coordinated transcriptional upregulation in NRAS-mutant melanoma, focusing on chromosome 1p13.2. RNA-seq and mutation data for 472 melanoma samples from The Cancer Genome Atlas Skin Cutaneous Melanoma were analyzed. Differential expression, coexpression assessment, and copy number variation (CNV) correlation were performed for 45 protein-coding genes within 1p13.2. Findings were validated in MSK-IMPACT 2021 (n = 696). In NRAS-mutant melanomas, 56% of 1p13.2 genes were upregulated (false discovery rate < 0.05) versus 12% genome-wide (P < 0.001), including TRIM33, AMPD1, GNAI3, and SLC25A24. Upregulation was NRAS-specific: 84% showed higher expression versus BRAF-mutant tumors. Six of the 10 most NRAS-correlated genes localized to 1p13.2. CNV gains showed dose-dependent effects (gains: 1.8-3.2 fold; amplifications: 3.5-5.4 fold), with strong correlations (AMPD1 ρ = 0.68, GNAI3 ρ = 0.62, TRIM33 ρ = 0.58; P < 0.001). MSK-IMPACT validation confirmed 1p13.2 upregulation. As these findings are based on transcriptomic and copy number analyses, additional protein-level and functional studies across independent cohorts are required to confirm biological significance. NRAS-mutant melanomas harbor a coordinated transcriptional program within 1p13.2, driven by CNV gains. This locus contains genes with potential druggability, offering new avenues for combinatorial targeting alongside mitogen-activated protein kinase pathway inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb61b6e16a4770e24198eb7ffe671bddbc211884" target='_blank'>
              Chromosome 1p13.2 gene cluster upregulation in neuroblastoma RAS viral oncogene homolog pathogenic variant melanoma: a potential biomarker axis.
              </a>
            </td>
          <td>
            Sophie M. Bilik, Olivia M Burke, S. Elman
          </td>
          <td>2026-01-29</td>
          <td>Melanoma research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="CTNNB1, the gene encoding β-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signaling pathway, typically through missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signaling phenotypes for all 342 possible missense mutations in the mutation hotspot. Our data define the genetic requirements for β-catenin degron function, refine the consensus motif for substrate recognition by β-TRCP and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of predicted activity. In hepatocellular carcinoma, mutation effect scores distinguish two tumor subclasses with different levels of β-catenin signaling, and weaker mutations predict greater immune cell infiltration in the tumor microenvironment. Our work provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy. Saturation genome editing of a cancer mutation hotspot in CTNNB1, the gene encoding β-catenin, reveals a gradient of effects of missense mutations on Wnt signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0951b8c87bfe861a396eafd5a171689184ab2592" target='_blank'>
              Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signaling
              </a>
            </td>
          <td>
            Anagha Krishna, A. Meynert, K. S. Dolt, Martijn J. E. Kelder, A. Mesropian, Ailith Ewing, C.M. Brouwers, J. Claassens, M. Linssen, Shahida Sheraz, G. Taylor, Philippe Gautier, Anna Ferrer-Vaquer, Graeme R Grimes, Hannes Becher, Ryan Silk, A. Gris-Oliver, R. Pinyol, C. Semple, T. Kendall, Tom Bird, A. Hadjantonakis, Joseph A. Marsh, J. Llovet, P. Hohenstein, Andrew J Wood, Derya D. Ozdemir
          </td>
          <td>2026-02-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Significance Replication stress (RS) is a major driver of genome instability in cancer. We discovered that the CHAMP1 complex, a set of proteins that assembles heterochromatin, plays a critical role in relieving RS. By depositing repressive histone marks, the complex stabilizes replication forks and prevents their degradation. Disruption of CHAMP1 complex creates unique vulnerabilities in cancer cells, including those cells with the (ALT) mechanism of telomere maintenance or with CCNE1 amplification. Our findings reveal a mechanism by which heterochromatin structure can safeguard DNA replication and highlight CHAMP1 as a potential therapeutic target for tumors with high levels of RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8461ad6cad0781da64419f6c03309084a55c23fc" target='_blank'>
              CHAMP1 complex promotes heterochromatin assembly and reduces replication stress
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y. Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Significance The classic Luria–Delbrück experiment showed how mutations arise in single-copy bacterial chromosomes. Yet many genetic systems, including plasmids and the genomes of mitochondria, plastids, and cancer cells, exist in multiple copies. In such systems, polyploidy generates opposing forces: Extra copies increase the supply of new mutations, while random segregation slows their fixation. Therefore, it remains unclear whether polyploidy promotes or constrains evolution. Using plasmids as a case study, we combine theory, modeling, experiments, and a bioinformatic extension of the Luria–Delbrück framework to show that mutation rates increase logarithmically with copy number. Our results settle a long-standing debate, demonstrating that plasmids are hotspots of bacterial evolution and revealing the rules by which polyploidy sets the tempo of evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecbf44e01949a84dc5dd9b45780af8eafce6a78d" target='_blank'>
              Plasmid mutation rates scale with copy number
              </a>
            </td>
          <td>
            Paula Ramiro-Martínez, Ignacio de Quinto, Laura Jaraba-Soto, V. Lanza, Cristina Herencias-Rodríguez, Adrián González Casanova, R. Peña-Miller, J. Rodríguez-Beltrán
          </td>
          <td>2026-01-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Genomic instability is a hallmark of cancer and contributes to disease progression through various mechanisms. DNA copy number variation (CNV) is one of the most significant consequences causing a phenomenon called aneuploidy. Aneuploidy is characterized by an imbalanced genome and is associated with lethal progression in prostate cancer. Currently, sequencing-based CNV analysis is widely used to identify cancer-specific aneuploidy that contribute to tumorigenesis, immune evasion, and disease progression. Performing low-pass whole genome sequencing (LP-WGS) on DNA extracted from bulk tissues or cell cultures, is a convenient approach to use for detecting aneuploidy and focal CNVs. However, conventional bulk LP-WGS methods average genomic signals across thousands of cells, often overlooking tumor heterogeneity and small populations with distinct CNV patterns. To address this limitation, we optimized a microfluid nanowell-based system to analyze aneuploidy patterns at a single-cell level using disassociated tissues and isolated nuclei from prostate samples. This pipeline is compatible with both preclinical models and clinical specimens and can be integrated with other multi-omics platforms to generate multimodal datasets for basic and translational research. By leveraging high-throughput single-cell sequencing technologies, our approach enables direct DNA CNV measurement with high sensitivity at a single-cell resolution. Ultimately, this approach aims to improve the resolution of CNV detection, enhance molecular stratification and enable multimodal data integration for better understanding of clinical prostate cancer samples.



 Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, William D. Figg, Gregory Chesnut, Matthew G. Vander Heiden, Xiaofeng A. Su. Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A060.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6642b028922ed7c8904563bae6731f81ce2d7a38" target='_blank'>
              Abstract A060: Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing
              </a>
            </td>
          <td>
            Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline M. Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, W. Figg, Gregory T. Chesnut, M. V. Vander Heiden, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Background/Aim Lipomas are benign tumors of adipocytic origin. The most frequent chromosome rearrangement in lipomas involves the high-mobility group AT-hook 2 gene (HMGA2), which maps to 12q14. We investigated two lipomas showing rearrangements of chromosomal bands 8q22 and 12q14 in their karyotypes, with the aim of identifying the gene products affected by these aberrations. Materials and Methods The two lipomas were selected because their karyotype showed an 8;12-rearrangement with the very same breakpoint position. The cases were investigated using RNA sequencing, reverse transcription polymerase chain reaction (RT-PCR), Sanger sequencing techniques, Fluorescence in situ hybridization (FISH), and array comparative genomic hybridization (aCGH). Results RNA sequencing and RT-PCR of the two lipomas showed the presence of the HMGA2::CIBAR1-DT chimera. Conclusions The HMGA2::CIBAR1-DT fusion, identified here for the first time, is a recurrent transcript in lipomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35b7cf2f12e853b8a43f5d8374c50a16fbddd11" target='_blank'>
              Identification of the HMGA2::CIBAR1-DT fusion transcript in two lipomas with chromosomal rearrangements involving chromosomes 8 and 12
              </a>
            </td>
          <td>
            M. Brunetti, K. Andersen, I. Lobmaier, F. Micci
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0ee0a18c3d0ebfc5acdb328d589c81fb8981b3" target='_blank'>
              Comprehensive repertoire of the chromosomal alteration and mutational signatures across 16 cancer types.
              </a>
            </td>
          <td>
            Andrew Everall, Avraam Tapinos, A. Hawari, A. Cornish, Amit Sud, D. Chubb, B. Kinnersley, A. Frangou, Miguel Barquin, Josephine Jung, David N. Church, Ludmil B. Alexandrov, R. Houlston, Andreas J. Gruber, D. Wedge
          </td>
          <td>2026-02-13</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Abstract DNA is most often found in its canonical B-form double-helical structure, but can also adopt alternative conformations, known as non-B DNA structures. Numerous non-B structures have been characterized, including G-quadruplexes, i-motifs, Z-DNA, hairpins, cruciforms, slipped structures, R-loops, and H-DNA. Non-B DNA motifs are enriched in functional regions, including near transcription start and end sites, topologically associated domains, and replication origins, suggesting their importance in gene regulation, genome organization, and replication. However, these structures are intrinsically prone to error-generating processing, leading to genomic instability and hence have been implicated in the development of human diseases. Here, we discuss recent advances in understanding the biological roles of non-B DNA structures and their contribution to genomic instability in somatic and germline contexts. We highlight how they promote replication stress, transcription stalling, and DNA breaks, resulting in the formation of mutational hotspots. Emerging technologies have enabled the detailed mapping of previously challenging repetitive regions that harbor potential non-B DNA-forming sequences, and are poised to unravel additional contributions in human disease and evolution. Furthermore, we explore the dual role of non-B DNA as a driver of genetic variation that facilitates evolutionary adaptation and as a source of mutations that contribute to tissue dysfunction and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81eab064c5f9cbe45e223c9b4a2bde11b74f6f7" target='_blank'>
              Non-B DNA structures and their contributions to genetic diversity, aging, and disease
              </a>
            </td>
          <td>
            Eleftherios Bochalis, Irene Dereki, Guliang Wang, Argyro Sgourou, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2026-02-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Extrachromosomal circular DNA has emerged as a pivotal factor in tumor biology, contributing to intratumor heterogeneity, oncogene amplification, and tumor evolution. Despite its relevance, the presence and molecular characteristics of plasma-derived cell-free extrachromosomal circular DNA (eccDNA) in cancer patients remain insufficiently explored. In this study, we profiled plasma-derived cell-free eccDNA from a multi-cancer cohort consisting of 413 cancer patients and 239 healthy individuals. We analyzed eccDNA fragment size distributions using a patient-derived xenograft (PDX) mouse model and identified fragment size features to distinguish tumor-derived eccDNA from non-tumor-derived eccDNA. We further performed in silico fragment size selection and developed a gene-based annotation method to characterize the gene content carried by cell-free eccDNA across different cancer types. By utilizing these cell-free eccDNA signatures, we developed ScanTecc (screening cancer types with cell-free eccDNA), a machine learning-based approach for cancer detection and tissue-of-origin classification. The classification performance of ScanTecc was further evaluated at the individual sample level using multiple machine learning classifiers, including adaptive boosting and logistic regression. We observed a significantly higher abundance and longer fragment lengths of eccDNA in cancer patients’ plasma. Analysis of the PDX mouse model revealed a distinct fragment size threshold of approximately 1,000 base pairs that effectively differentiates tumor-derived eccDNA from non-tumor-derived counterparts. Following fragment size selection, gene-based annotation of large-sized eccDNA revealed cancer type–specific enrichment of tumor-associated genes. Leveraging these features, ScanTecc achieved an overall AUC of 0.92 for distinguishing cancer patients from healthy individuals, with consistently high performance across disease stages, including stage I (AUC = 0.92) and stage IV (AUC = 0.93). ScanTecc also enabled accurate tissue-of-origin classification and achieved an overall AUC of 0.79 in identifying specific cancer types, with AUC values ranging from 0.70 for gastric cancer to 0.81 for ovarian cancer. Our study establishes a multi-cancer plasma cell-free eccDNA landscape and introduces a non-invasive cancer screening framework based on cell-free eccDNA features, highlighting the potential of plasma cell-free eccDNA for early cancer detection and tumor classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd2c300c02b9f286a9e46bf7c8db5ce03368c9f" target='_blank'>
              Detection of primary cancer types via fragment size selection in circulating cell-free extrachromosomal circular DNA
              </a>
            </td>
          <td>
            Jingwen Fang, Songwen Luo, Shouzhen Li, Yehong Xu, Jing Wang, Benjie Shan, Mingjun Hu, Qiaoni Yu, Wen Zhang, Ke Liu, Yunying Shao, Jiaxuan Yang, Youyang Zhou, Guangtao Xu, Xinfeng Yao, Ruoming Sun, Mengyuan Zhang, Kun Li, Xihai Xu, Yongliang Zhang, Zhihong Zhang, Xinghua Han, Yueyin Pan, Chuang Guo, Kun Qu
          </td>
          <td>2026-01-06</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The use of programmable nucleases has transformed genome editing and functional genomics. Clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated 9 (Cas9) was developed such that targeted genomic lesions [usually DNA double-stranded breaks (DSBs)] could be introduced in vivo with ease and precision. In the presence of homology donors, these lesions facilitate high-efficiency precise genome editing (PGE) via homology-directed repair (HDR) pathways. Because DSBs can lead to genomic instability, however, a large amount of effort has been invested in methodologies (e.g., base editors) that only require nicking the chromosomal DNA on one strand. Indeed, we have demonstrated in human cells that oligodeoxynucleotide (ODN)-mediated PGE using nickase variants of Cas9 can proceed by at least two HDR subpathways termed synthesis-dependent strand annealing (SDSA) and single-stranded DNA incorporation (ssDI). Which pathway is utilized is determined by which chromosomal strand (sense or anti-sense/Watson or Crick) is nicked and by the strandedness (sense or anti-sense/Watson or Crick) of the donor ODN. While the mechanism of mammalian SDSA is moderately well understood, that of ssDI is not. To gain genetic insight into ssDI, we carried out a genome-wide CRISPR knockout screen to identify those genes which, when absent, enable increased ssDI. This screen identified the protein lysine methyl transferase (PKMT) Su(var)3–9, enhancer-of-zeste and trithorax (SET) domain bifurcated histone lysine methyltransferase 1 (SETDB1):activating transcription factor 7-interacting protein (ATF7IP) heterodimer and the downstream human silencing hub (HUSH) complex as strong negative regulators of ssDI. Consistent with their well-known biological effects, the negative regulation of ssDI by SETDB1/ATF7IP and HUSH was specific for transgenic reporters and for a HUSH-regulated single-copy gene, but was not observed at other (non-HUSH regulated) single-copy endogenous loci. In toto, these experiments underscore the profound impact that chromatin modifiers - and by extension, chromatin structure – have on PGE outcomes. Specifically, we have identified SETDB1/ATF7IP and the HUSH complex as major negative regulators of the HDR subpathway, ssDI, when the target is a transgene. These experiments are a proof-of-principle that chromatin can act as a potent barrier to genetic recombination and they strongly support the feasibility of extending similar chromatin modulating strategies to enhance PGE efficiency at endogenous single-copy loci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827654ec916ba3c373ade5ee57ce9ecca2279a3e" target='_blank'>
              SETDB1/ATF7IP regulate the precise genome engineering of HUSH-regulated genes
              </a>
            </td>
          <td>
            B. Ruis, Henry Ward, Chad L. Myers, A. Bielinsky, Eric A. Hendrickson
          </td>
          <td>2026-02-09</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Ewing sarcoma is an aggressive bone tumor of adolescence characterized by a hallmark EWSR1::ETS fusion oncogene. The resulting chimeric oncoprotein drives tumorigenesis by reshaping transcriptional and epigenetic landscapes. However, how it is transcriptionally regulated and whether additional master transcription factors (MTFs) form a core regulatory circuit (CRC) in Ewing sarcoma remain unclear. Using an extensive panel of Ewing sarcoma cell lines and primary tumors, we mapped super-enhancers and identified enrichment of GGAA microsatellites, confirming their specificity to Ewing sarcoma as compared to other pediatric cancers and normal tissues. Integrating transcriptomic, epigenetic, 3D chromatin conformation, and dependency data, we predicted a set of MTFs potentially forming a CRC. However, functional validation demonstrated that these MTFs neither establish auto-regulatory loops nor confer robust proliferative dependencies typical of CRCs in other pediatric tumors. Instead, EWSR1::FLI1 emerged as an “hegemonic” oncoprotein, regulating expression of these MTFs without reciprocal regulation. Knockdown of EWSR1::FLI1 strongly shifted H3K27ac profiles toward mesenchymal states, whereas silencing individual or combined MTFs did not alter cell growth or EWSR1::FLI1 expression. These findings highlight the absence of a classical CRC in Ewing sarcoma and emphasize EWSR1::FLI1 as the dominant oncoprotein and a major vulnerability in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb53da812dbac416c29e7f7225ad2ab6decfdb9" target='_blank'>
              Exploring the Space of Tumor Phylogenies Consistent with Single-Cell Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Samin Rahman Khan, P. Sashittal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            Madison L. Hall, Aliya Quintal, Samantha Worme, Thai T. Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, Randall Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy nearly half of the human genome and play diverse biological roles. Despite their abundance, the extent to which TEs contribute to three-dimensional (3D) genome structure remains unclear. To investigate this, we developed te_hic, a computational pipeline that integrates genomic data with chromatin interactions. Our analysis reveals that TE sequences are responsible for 3D genome structure in interphase nuclei. This phenomenon is mediated by the recruitment of specific transcription factors to TEs, which enables both the formation and disruption of chromatin contacts. We computationally identified known contact-forming proteins (CTCF, RAD21, SMC3) and breaking proteins (RNF2), as well as novel candidate contact-formers (SMARCA4, MAFK). Knockdown of the predicted contact-formers SMARCA4 and MAFK decreased contacts and loops at and between TEs. Notably, SMARCA4 knockdown selectively reduced short-range contacts, highlighting its role in maintaining 3D genome structure mediated by TE binding. Overall, our findings demonstrate that TEs are crucial determinants of 3D genome organization in mammalian cells. Key findings TEs determine 48% of the 3D genome structure alone, and 70% of heterotypic contacts. A/B compartments, TADs, and loop signatures can be retrieved using TE-mapped reads only. TFs can be divided into contact formers and breakers at TEs. SMARCA4 and MAFK build chromatin contacts between TE sequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3ef8e203c378a8d1a7856db6aaa8c3b98aa293" target='_blank'>
              Dissecting the contribution of transposable elements to interphase chromosome structure
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhouqi Huang, Li Sun, Xuemeng Zhou, Jiangping He, Alexander V. Strunnikov, A. P. Hutchins
          </td>
          <td>2024-10-27</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The three-dimensional (3D) folding of the genome plays a crucial role in genome regulation. However, how 3D genome structure varies between individuals and consequently influences genome function and evolution remains poorly understood. One potential source of this variation is transposable elements (TEs), genomic parasites whose location and composition vary between species and individuals. Hosts typically silence TEs through enriching them with repressive epigenetic marks, turning euchromatic TEs into heterochromatin islands, which were shown to spatially interact with pericentromeric heterochromatin (PCH). Because most TE insertions are present in only a few individuals within Drosophila populations, we asked whether polymorphism in the presence/absence of TEs drives varying 3D structures through TE-PCH spatial interactions. We performed deep-coverage Hi-C of two wild-type Drosophila strains and developed a Hi-C analysis framework enabling allelic comparisons of spatial interactions with PCH. Supporting our hypothesis, nearly 40% of strain-specific euchromatic TEs cause their adjacent euchromatic regions to be spatially closer to PCH than TE-free homologous alleles. These interactions are not limited to specific TE families, and, surprisingly, telomere-proximal TEs show a similar propensity as centromere-proximal TEs to enhance PCH interactions. The most defining feature of TEs involved in PCH interactions is H3K9me3 enrichment, revealing a chromatin-based mechanism for TE-mediated 3D genome organization broadly applicable across TE families and genome locations. Importantly, TEs involved in PCH interactions reduce the expression of adjacent genes and are evolutionarily young, indicating stronger selection against them. Our study reveals a previously uncharacterized mechanism by which TEs influence the function and evolution of host genomes by generating polymorphic 3D genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4185cccd9e0a3e27a9efe577810b6e58e6b70" target='_blank'>
              Polymorphic 3D genome architecture mediated by transposable elements
              </a>
            </td>
          <td>
            Harsh G. Shukla, Yuheng Huang, Zita Y. Gao, Y. C. G. Lee
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, Ralph J. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1504ceb179d5285f33902944e8fdc793d756f87c" target='_blank'>
              Extrachromosomal DNA drives molecular and clinical heterogeneity in hepatocellular carcinoma: a multi-omics analysis and prognostic model development.
              </a>
            </td>
          <td>
            Kai Huang, Guangquan Zhang, Shuai Hu, Yongfei He, Hang Zhai, Junming Xu, Jicai Wang, Shengjie Hong, Fenfang Wu, Xianjie Shi
          </td>
          <td>2026-02-03</td>
          <td>Human genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Oncogenic viruses can induce epigenetic changes in host cells, which may contribute to cell transformation and tumorigenesis by altering gene expression patterns. Human adenoviruses, while primarily known for their lytic infections, have also been implicated in oncogenic transformation in experimental settings, but the mechanisms underlying adenovirus oncoprotein-induced epigenetic dysregulation remain poorly defined. Building on our recent findings that the adenoviral oncoprotein E1B-55K coordinates cellular transformation through interactions with DNA-bound host transcription factors, we next sought to determine whether these interactions also influence the chromatin landscape. For this purpose, we investigated the epigenetic consequences of E1B-55K expression in primary human mesenchymal stromal cells using histone-targeting MNase chromatin immunoprecipitation-seq, complemented by RNA-seq analysis. We demonstrate that stable expression of HAdV-C5 E1B-55K leads to widespread changes in histone post-translational modifications across the genome. Most notably, E1B-55K expression results in a marked loss of the activating histone marks H3K4me3 and H3K27ac at specific promoters and enhancers. These epigenetic alterations correspond with significant changes in gene expression, suggesting that E1B-55K interferes with normal transcriptional programming by altering the establishment of key histone modifications. Our findings help explain how E1B-55K drives viral transformation by changing epigenetic patterns and improving our understanding of virus-host interactions at the chromatin level. By describing how human adenoviruses disrupt epigenetic homeostasis, we provide insights into the mechanisms by which oncogenic viruses promote early events in cellular transformation. IMPORTANCE Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level. Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77947d567465053bbac94ee829a515fc47178965" target='_blank'>
              The adenovirus oncoprotein E1B-55K reshapes epigenetic histone modifications in primary human cells
              </a>
            </td>
          <td>
            Konstantin von Stromberg, Laura Seddar, Britta Gornott, Thomas Dobner, L. D. Bertzbach, W. Ip
          </td>
          <td>2026-01-26</td>
          <td>mBio</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Fusion genes are pivotal drivers of tumorigenesis, often generating oncogenic chimeric RNAs and fusion circular RNAs. However, the mechanisms by which these transcripts synergistically contribute to cancer progression remain poorly understood. Here, we identified a lung cancer-specific chimeric RNA KANSL1-ARL17A (chKANSARL) and its circular variant fusion circular RNA KANSL1-ARL17 A (F-circKA), both derived from the fusion gene KANSARL. Functional assays revealed that overexpression of either chKANSARL or F-circKA significantly enhanced lung cancer cell proliferation, migration, and invasion, while their knockdown suppressed these malignant phenotypes. In vivo experiments demonstrated that chKANSARL overexpression accelerated tumor growth in immunodeficient mice. Notably, coexpression experiments uncovered a synergistic regulatory interaction between F-circKA and chKANSARL, amplifying oncogenic effects. Mechanistically, miRNA sequencing and dual-luciferase assays revealed that F-circKA acts as a molecular sponge for miR-6860, thereby derepressing chKANSARL expression. Rescue experiments further validated this regulatory axis, wherein miR-6860 inhibition reversed the tumor-suppressive effects of F-circKA knockdown. Collectively, our study identifies and characterizes a novel F-circKA/miR-6860/chKANSARL regulatory axis, revealing how dual transcriptional outputs from the KANSARL fusion gene can synergistically drive lung cancer progression. These findings highlight a previously unrecognized layer of cooperative regulation between linear and circular fusion RNAs in oncogenesis and provide a new framework for understanding fusion gene-mediated tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c7a04a77f3f0847188162738372118fe5132c" target='_blank'>
              The KANSL1-ARL17A fusion gene generates oncogenic chKANSARL and F-circKA RNAs that synergistically drive lung cancer progression via a novel F-circKA/miR-6860/chKANSARL axis
              </a>
            </td>
          <td>
            Xinchao Guan, Tao Liu, Sili Chen, Junwei Zhang, Huanliang Huang, Dexiong Chen, Qiaoyuan Yang
          </td>
          <td>2026-01-01</td>
          <td>The Journal of Biological Chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) was characterized by a highly complex genome, with structural variations (SVs) playing a significant role in its development. In this study, we employed Oxford Nanopore Technology long‐read sequencing in paired tumor and adjacent normal liver tissues from 74 Chinese HCC patients to thoroughly characterize the landscape of somatic SVs. Our analysis revealed that somatic SVs were more prevalent in hepatitis B virus (HBV)‐related HCC, with chromosome 1 emerging as a major hotspot, and several members of the chromosome 1 open reading frame (C1orf) family genes expression level exhibited significant age‐related difference. Notably, HBV‐related HCC cases exhibited a higher frequency of deletions, particularly among younger ones (≤ 35 years old). In addition, we observed an increased burden of HBV integration events in younger ones. Remarkably, the divergent‐paired related homeobox (DPRX) loci was identified as a novel gene for HBV integration in younger patients. Together, these findings delineated the somatic SV landscape in HCC and underscored age‐associated HBV‐related genomic alterations as key pathological features of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cdc214e4351424b89dd91edf26352c6acad965" target='_blank'>
              Landscape of Somatic and Age‐Related Pathogenic Structural Variations in Hepatocellular Carcinoma Revealed by Long‐Read Sequencing
              </a>
            </td>
          <td>
            Zhewen Wei, Yinghao Cao, Hongchao Liu, Mei Liu, Bolun Zhang, Jianming Ying, Jianqiang Cai, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Zhen Huang, Jianmei Liu, Xueyan Lv, Zhiwen Luo, Zhicheng Wei, Xiaoshi Zhang, Yi Yang, Yiqiao Deng, Yanjiang Yin, Jinghua Chen, Junbo Liang, Xiaoyue Wang, Ye‐fan Zhang, Hong Zhao
          </td>
          <td>2026-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369c5e33b0cad07e2b10ea8763123bd8af3e00d3" target='_blank'>
              TERT drives liver tumorigenesis beyond telomere elongation
              </a>
            </td>
          <td>
            Laura Braud, Julien Vernerey, Arnaud Guille, Pierre Cordier, Clémence Ginet, Tom Egger, Manuel Bernabe, D. Churikov, Quentin Da Costa, Aïda Meghraoui, C. Desdouets, Li Gu, F. Bertucci, Christophe Lachaud, Vincent Géli
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f08e435f05db1aa85f8917f417c0101833b4e71" target='_blank'>
              Meiotic pairing through barcode-like satellite DNA repeats
              </a>
            </td>
          <td>
            Lena Skrutl, Ankita Chavan, A. Sintsova, Ilaria Ceppi, Corin J. Ropp, S. Sunagawa, Petr Cejka, Madhav Jagannathan
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9dbefd676963f04b85639797c520690ca798555" target='_blank'>
              Human and viral whole genome sequencing identify HPV and APOBEC as oncogenic drivers in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            H. B. Chong, M. E. Bryan, M. Lin, W. C. Faquin, L. J. Mirabello, S. K. Mishra, J. Lewis, M. S. Lawrence, D. Faden
          </td>
          <td>2026-02-09</td>
          <td>None</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna C. M. Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596e7d27f8f208d19adb1081b3f37426d428f0a7" target='_blank'>
              PTEN deficiency linked to chromosome 10q loss leads to aggressive NF2 mutant meningioma biology
              </a>
            </td>
          <td>
            Abigail G. Parrish, H. N. Thirimanne, Sonali Arora, C. Herold-Mende, P. Sievers, F. Sahm, Frank Szulzewsky, Eric C. Holland
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cefe8c9af442e8e2260763df51e56fb57fff1f0d" target='_blank'>
              Molecular architecture of the tumor microenvironment caused by BRCA1 and BRCA2 somatic mutations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gaoming Liao, Xinbin Yang, Qi Liu, Shufeng Nan, Yan Liu, Jinwei Li, Si Huang, Wang Ning, Xionghai Qin, Gang Xu
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2b7d842c50e19f140b3dba7c07392f64035b86" target='_blank'>
              Single-nucleus multi-omics delineates distinct epigenetic programs associated with tumor progression in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yeji Kim, Huiram Kang, Y. Hwang, Kyung Soo Kim, Seok Whan Moon, Jeonghan Kim, Seung Joon Kim, Hae-Ock Lee
          </td>
          <td>2026-02-11</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Circular DNA molecules—mitochondrial DNA (mtDNA) and extrachromosomal circular DNA (ecDNA)—are fundamental genetic elements whose structural integrity is essential for cellular homeostasis. Pathological alterations in their length, caused by deletions or duplications, are directly implicated in a broad spectrum of human diseases, from inherited mitochondrial disorders to aggressive cancers. This review provides a comprehensive methodological guide for detecting these specific structural changes. We systematically evaluate classical techniques (long-range PCR, Southern blotting, electron microscopy) that offer direct visualization and validation, and modern high-throughput approaches (short- and long-read sequencing) that enable genome-wide discovery. Special emphasis is placed on quantitative clinical tools like digital droplet PCR (ddPCR) for ultrasensitive monitoring. By presenting a comparative framework, we guide the selection of optimal methods based on resolution, throughput, and clinical applicability. We conclude that an integrated, multi-method strategy is indispensable for robust analysis, positioning the precise detection of circular DNA length alterations as a cornerstone of emerging precision diagnostics and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9367ea411656b4af8c5e9c55df093cb7809d8cc8" target='_blank'>
              The Circular DNA Size Code
              </a>
            </td>
          <td>
            Zurab Abashidze
          </td>
          <td>2026-01-31</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal translocations are prevalent genetic events across multiple pediatric cancers, notably in CNS tumors, solid tumors, and leukemias. For decades, Fusion oncoproteins resulting from chromosomal translocations have been proposed as a hallmark of cancers, some of which can drive the process of cancers as the initial event of the disease. In addition, studies have shown that some tumor cells become addicted to the activity of fusion proteins, and cell death occurs when the fusion proteins are depleted. These researches suggest that fusion oncoproteins are one of the most promising targets for cancer treatment. Although fusion proteins are already recognized as critical oncogenic drivers, increasing evidence suggests that they can also form positive feedback loops with other proteins. In cancer patients, positive feedback loops have been shown to activate various oncogenic signals to drive tumor development, and influencing tumor cells’ sensitivity to different therapies. Therefore, these loops not only amplify the functions of the fusion proteins but also render single-agent targeting of the fusion protein insufficient to suppress tumor growth, highlighting the therapeutic potential of combination strategies in treating fusion-positive tumors. This review highlights the oncogenic roles of fusion protein-driven positive feedback loops in tumor initiation and progression, outline the molecular mechanisms underlying their formation and function, and summarize emerging therapeutic strategies targeting these circuits, offering new insights into the treatment of fusion-positive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b4b4433336cdba04cf2ea66983445def093bd8" target='_blank'>
              Genomic insights into chromosomal fusion and its evolutionary implications for zokors.
              </a>
            </td>
          <td>
            Zhuoran Kuang, Xiaojie Yang, Na Wan, Jiaqi Chen, Qijiao Duan, Bowen Li, Xi Liu, Xiaolong Liang, Xinfeng Liu, Wenyu Liu, E. Nevo, Kexin Li
          </td>
          <td>2026-02-03</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysplastic nodules (DN) are precursors to cirrhosis‐associated malignancy, and the HBV DNA integration in the human genome plays a critical role in tumorigenesis. However, the precise relationship between DN and HBV integration remains unclear. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. 1936 and 1450 HBV integration sites were identified in the DN and RN samples, respectively. The number of HBV integration sites in DN correlated with nodule size. Breakpoints in HBV genome were concentrated within 100 bps towards the 5' or 3' end of involved HBV genes. Furthermore, integration numbers also positively correlated with number of point mutations in DN samples. We identified 53 and 29 recurrent genes containing HBV integrations in >=2 samples in DN and RN samples, respectively. Higher clonality was observed for HBV integrations in recurrent genes than other HBV‐integrated genes. The HBV integrations in recurrent genes were predominantly located in intron regions. Notably, among those recurrently HBV‐integrated genes in DN samples, SCHIP1, ZDHHC14, YPEL2, RABGAP1L, and SOX5 displayed significant expression alterations. Moreover, clinical indicators revealed significant prolongation of prothrombin time in two DN patients with HBV integrations in SCHIP1 and ZDHHC14. As a new insight regarding HBV integrations in DN stage, our findings suggest a possible role of HBV integration in the transformation of DN to early‐stage liver cancer by affecting the expression of key genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef5e9a714b1a7dd80b4097dd4b95245da6b9365c" target='_blank'>
              Analysis of HBV Integration Reveals New Insights of Oncogenic Mechanism in Dysplastic Liver Nodule
              </a>
            </td>
          <td>
            Xi Zeng, Hui Liu, Zheqi Xu, Xinjie Rao, Yuyouye Wang, Fang Peng, Wei Dong, Ziying Wang, Zhenguang Wang, Xing Gu, Fuchen Liu, Guoliang Li, Weiping Zhou, Linghao Zhao
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="SUMMARY Plant nuclear genomes contain a variable, though typically minor, fraction of DNA sequences of plastid origin known as NUPTs. Unlike the massive transfer of DNA and genes from the proto‐organelle genome to the nucleus that occurred during the endosymbiotic event that gave rise to plastids, the formation of NUPTs is an ongoing process that does not imply concomitant DNA loss. Although NUPTs are generally considered to be potentially deleterious insertions that are continuously generated and rapidly eliminated at near‐constant turnover rates, accumulating evidence reveals alternative evolutionary trajectories. In this review, we discuss recent findings that highlight the episodic formation of NUPTs, their subsequent proliferation, and their eventual long‐term fixation within the nuclear genome. We also explore their non‐random spatial association with specific genomic elements. NUPTs show preferential overlap with specific superfamilies of transposable elements, which may facilitate their proliferation and dispersal throughout the nuclear genome. Regarding protein‐coding genes, the contribution of NUPTs varies among species. In contrast, NUPTs are found to be consistently enriched among certain classes of non‐coding RNA genes, notably rRNA, tRNA, and specific regulatory RNA families, suggesting that they are involved in the evolution of gene regulation and translational machinery. Overall, these findings underscore the unexpected complexity of the mechanisms underlying NUPT formation and support the idea that they are a significant source of genome variation and evolutionary innovation. Further research is necessary to fully elucidate the mechanisms underlying NUPT formation, as well as to determine their potential adaptive significance in plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72824b12a89eeb048c0f674beb8dd4ae9c4603a5" target='_blank'>
              Nuclear DNA of plastid origin (NUPTs), neglected driver of genome variation and evolutionary innovation
              </a>
            </td>
          <td>
            Lorenzo Carretero-Paulet, Juan Pablo Marczuk-Rojas, Aaron Gálvez‐Salido
          </td>
          <td>2026-02-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, G. Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Short tandem repeats (STRs), or microsatellites, are highly mutable genomic elements that modulate gene regulations and are implicated in a range of human diseases. However, detecting mosaic STR mutations at single-cell resolution remains challenging due to both technical and biological complexities. To address this, we developed BayesMonSTR, a robust algorithm that enables accurate detection of mosaic STR mutations. Using this tool in single-cell analysis of human tissues, we reveal an accumulation of longer mosaic STR insertions and deletions (indels) in aging mitotic and post-mitotic cells. Strikingly, prefrontal cortex (PFC) neurons accumulate a higher burden of STR mutations than B cells or lung epithelium, with aged neurons exhibiting a particularly pronounced increase in longer STR deletions. These mutations are enriched at transcription start sites (TSSs) and active enhancers of highly expressed genes. Our work establishes a foundation for genome-wide, hypothesis-free discovery of disease-associated mosaic STR mutations and reveals a previously unexplored landscape of mosaic STR variation in development and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1d43260f2ab61bb3b1461c097bc6e22ec5bab5" target='_blank'>
              Unravelling genome-wide mosaic microsatellite mutations at single-cell resolution
              </a>
            </td>
          <td>
            Chunyi Wang, Wenxuan Fan, Weixiang Wang, Yonghe Xia, Jinhong Lu, Xiaoyu Ma, Jichuan Yu, Yunchao Zheng, Yan Luo, Wenlong Li, Qing Yang, Meizhen Lin, Huan Liu, Yangning Lan, Chengyu Li, Xiaodong Liu, Danyang He, Shang Cai, Xiuyan Yu, Dan Zhou, M. Kellis, Xushen Xiong, Qi Xie, Yanmei Dou
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="To investigates Flap endonuclease 1 (FEN1) as a prognostic and immune-related biomarker for different cancer types by combining multi-omics analyses and experimental validation. We determined the expression, mutational profiles and clinical implications of FEN1 in 33 cancer types based on multi-omics data from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx) and Clinical Proteomic Tumor Analysis Consortium (CPTAC). Functional assays such as FEN1 knockout in liver hepatocellular carcinoma (LIHC) cell lines (Huh-7 and HCCLM3) were conducted to investigate the effect of FEN1 on proliferation, migration, cell cycle progression, and DNA damage repair. Our study has found that FEN1 was overexpressed in 17 cancers, including breast cancer (BRCA), LIHC, lung adenocarcinoma (LUAD). FEN1 overexpression was significantly associated with tumor stage, poor prognosis and diagnosis. Mechanistically, FEN1 promotes the malignant cell phenotypes by promoting cell cycle progression and the DNA damage repair process. External dataset analyses verified FEN1 upregulation in liver cancer tissues, with a positive correlation with poor survival. In vitro and in vivo experiments revealed that FEN1 knockdown can inhibit the proliferation, migration, cell cycle progression, and DNA repair capacity of HCC cells and tumors. In conclusion, pan-cancer studies revealed that FEN1 is overexpressed and positively associated with adverse clinicopathological characteristics in various cancer types. This indicates its potential role as a wide-range biomarker. Furthermore, FEN1 is associated with immune microenvironment regulation, and in some cases, may be associated with immunotherapy response. In LIHC models, experimental validation revealed that FEN1 stimulates tumorigenesis by stimulating proliferation, migration, and genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7474f848decbc0e7dd032bf72f7e503b26f0af8d" target='_blank'>
              Flap endonuclease 1 drives tumorigenesis across cancers and is a master regulator of proliferation and genomic instability in liver cancer.
              </a>
            </td>
          <td>
            Fang Li, Xu Ji, Yuzhe Zhang, Aoran Liu, Ruipeng Li, Xiaoli Peng, Yan Wang, Jipeng Mei, Yuan Yang, Yanliang Wang, Lina Wu, Bengang Wang, Ye Zhang
          </td>
          <td>2026-02-11</td>
          <td>International journal of biological macromolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Optical genome mapping (OGM) detects genome-wide structural variants (SVs), including balanced rearrangements and complex copy-number alterations beyond standard-of-care cytogenomic assays. In chronic lymphocytic leukemia (CLL), cytogenetic and genomic risk stratification is traditionally based on fluorescence in situ hybridization (FISH), karyotyping, targeted next-generation sequencing (NGS), and immunogenetic assessment of immunoglobulin heavy chain variable region (IGHV) somatic hypermutation status, each of which interrogates only a limited aspect of disease biology. Methods: We retrospectively evaluated fifty patients with CLL using OGM and integrated these findings with cytogenomics, targeted NGS, IGHV mutational status, and clinical time-to-first-treatment (TTFT) data. Structural variants were detected using OGM and pathogenic NGS variants were derived from a clinical heme malignancy panel. Clinical outcomes were extracted from the electronic medical record. Results: OGM identified reportable structural variants in 82% (41/50) of cases. The most frequent abnormality was del(13q), observed in 29/50 (58%) and comprising 73% (29/40) of all OGM-detected deletions with pathologic significance. Among these, 12/29 (42%) represented large RB1-spanning deletions, while 17/29 (58%) were focal deletions restricted to the miR15a/miR16-1 minimal region, mapping to the non-coding host gene DLEU2. Co-occurrence of adverse lesions, including deletion 11q/ATM, BIRC3 loss, trisomy 12, and deletion 17p/TP53, were recurrent and strongly associated with shorter TTFT. OGM also uncovered multiple cryptic rearrangements involving chromosomal loci that are not represented in the canonical CLL FISH probe panel, including IGL::CCND1, IGH::BCL2, IGH::BCL11A, IGH::BCL3, and multi-chromosomal copy-number complexity. IGHV data were available in 37/50 (74%) of patients; IGHV-unmutated status frequently co-segregated with OGM-defined high-risk profiles (del(11q), del(17p), trisomy 12 with secondary hits, and complex genomes whereas mutated IGHV predominated in OGM-negative or structurally simple del(13q) cases and aligned with indolent TTFT. Integration of OGM with NGS further improved genomic risk classification, particularly in cases with discordant or inconclusive routine testing. Conclusions: OGM provides a comprehensive, genome-wide view of structural variation in CLL, resolving deletion architecture, identifying cryptic translocations, and defining complex multi-hit genomic profiles that tracked closely with clinical behavior. Combining OGM and NGS analysis refined risk stratification beyond standard FISH panels and supports more precise, individualized management strategies in CLL. Prospective studies are warranted to evaluate the clinical utility of OGM-guided genomic profiling in contemporary treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28f49f64c743393cb2116dc5e9531fab4be082e7" target='_blank'>
              Optical Genome Mapping Enhances Structural Variant Detection and Refines Risk Stratification in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            S. Chakraborty, Michelle A. Bickford, Narcisa A. Smuliac, Kyle A. Tonseth, Jing Bao, Farzana Murad, Irma G. Domínguez Vigil, Heather B. Steinmetz, Lauren M. Wainman, Parth Shah, Elizabeth M. Bengtson, Swaroopa PonnamReddy, Gabriella A. Harmon, Liam L. Donnelly, L. Tafe, Jeremiah X. Karrs, Prabhjot Kaur, Wahab A. Khan
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="A large number of extrachromosomal circular DNAs (eccDNAs) are found in eukaryotic cells, but the mechanism behind the formation of eccDNAs remains unknown. EccDNAs smaller than 1000 bp are commonly referred to as microDNAs. In a prior study, we identified a 542 bp microDNA, designated eccDNAfib-L, mapping to Chromosome 14: 9,692,083-9,692,624 nt in the silk gland of Bombyx mori. There is a direct short repeat “GAGT” at both 5′ and 3′ break points of eccDNAfib-L, but only one copy of “GAGT” is retained in eccDNAfib-L. Here, we find that the specific junction observed can also be detected by transfecting with a DNA fragment containing upstream sequences of the 5′ break point and downstream sequences of the 3′ break point of eccDNAfib-L. Additionally, the length of the flanking sequences and the direct short repeats “GAGT” affect the formation efficiency of eccDNAfib-L. We confirm that eccDNAfib-L formation is associated with DNA repair pathways, with the expression of eccDNAfib-L undergoing significant alterations following the silencing of genes related to DNA repair pathways. In vitro, a cell-free reaction system confirms that Polθ and the direct short repeat “GAGT” are essential for the circularization of eccDNAfib-L, and the DExH-box helicase domain of Polθ plays critical role in mediating this circularization process. Collectively, our data support a mechanism whereby Polθ can mediate the joining of linear DNA fragments with direct short repeat at the 5′ and 3′ ends to form a circular DNA through MMEJ. The results not only clarify the essential factors driving eccDNAfib-L formation but also complement previous reports on eccDNA formation. DNA repair pathways are involved in eccDNAfib-L formation. Specifically, Polθ’s DExH-box helicase domain and direct short repeat “GAGT” are essential for its MMEJ-mediated circularization, delineating key factors driving this microDNA’s biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Dysplastic nodules (DN) are precursors to cirrhosis‐associated malignancy, and the HBV DNA integration in the human genome plays a critical role in tumorigenesis. However, the precise relationship between DN and HBV integration remains unclear. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. In total, 1936 and 1450 HBV integration sites were identified in the DN and RN samples, respectively. The number of HBV integration sites in DN correlated with nodule size. Breakpoints in HBV genome were concentrated within 100 bps toward the 5′ or 3′ end of involved HBV genes. Furthermore, integration numbers also positively correlated with number of point mutations in DN samples. We identified 53 and 29 recurrent genes containing HBV integrations in ≥ 2 samples in DN and RN samples, respectively. Higher clonality was observed for HBV integrations in recurrent genes than other HBV‐integrated genes. The HBV integrations in recurrent genes were predominantly located in intron regions. Notably, among those recurrently HBV‐integrated genes in DN samples, SCHIP1, ZDHHC14, YPEL2, RABGAP1L, and SOX5 displayed significant expression alterations. Moreover, clinical indicators revealed significant prolongation of prothrombin time in two DN patients with HBV integrations in SCHIP1 and ZDHHC14. As a new insight regarding HBV integrations in DN stage, our findings suggest a possible role of HBV integration in the transformation of DN to early‐stage liver cancer by affecting the expression of key genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8808ccbc83696379caab37a3ec557df0b07f831b" target='_blank'>
              Analysis of Hepatitis B Virus Integration Reveals New Insights of Oncogenic Mechanism in Dysplastic Liver Nodule
              </a>
            </td>
          <td>
            Xi Zeng, Hui Liu, Zheqi Xu, Xinjie Rao, Yuyouye Wang, Fang Peng, Wei Dong, Ziying Wang, Zhenguang Wang, Xing Gu, Fuchen Liu, Guoliang Li, Weiping Zhou, Linghao Zhao
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="


 Genomic copy number alterations cause aneuploidy, a hallmark of cancer characterized by imbalanced chromosome numbers. Aneuploidy has been shown to negatively associate with cancer immune signaling, but these mechanisms remain understudied. Of note, high levels of aneuploidy are strongly correlated with lethal progression and poor patient outcomes in prostate cancer. Our recent work identified chromosome 8q (chr8q) gain, one of the most frequent aneuploidy events in prostate tumors, as a key driver of disease progression, in part through the cohesin RAD21 gene located on the chr8q24 region. In both prostate cell line and organoid models, we demonstrated that increased RAD21 expression accelerates oncogenesis and aggressive tumor proliferation by mitigating cellular stress during early-stage tumorigenesis. Therefore, we hypothesize that elevated RAD21 levels enable cancer cells to evade immune surveillance by alleviating oncogenic stress and DNA damage. In this study, we analyzed publicly available genomic datasets and discovered that higher RAD21 expression negatively correlated with natural killer (NK) cell infiltration in prostate tumors. To further test our hypothesis, we modulated RAD21 levels in isogenic prostate cell culture models. Using an NK cell co-culture assay, we observed that cells with increased RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited greater survivorship when co-cultured with NK cells. In addition, we found that cellular levels of various pro-inflammatory cytokines, including those that stimulate NK cell activation, were significantly downregulated when RAD21 was overexpressed. These findings are consistent with our observation that tumors with high RAD21 levels are linked to reduced NK cell infiltration in prostate cancer cases. Our results suggest that amplification of RAD21, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Targeting RAD21-driven immune evasion may reveal novel therapeutic strategies to restore immune surveillance mechanisms in prostate cancer.



 Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R. Lozada, Christine Luo, Andrew Elliott4, Nicholas A. Zorko3, Justin H. Hwang3, David Takeda5, Xiaofeng A. Su. Elevated RAD21 levels promote immune evasion in prostatic malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A032.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba3556a59b07de1fb3550ddaa2e2a0eb7e783af" target='_blank'>
              Abstract A032: Elevated
 RAD21
 levels promote immune evasion in prostatic malignancies
              </a>
            </td>
          <td>
            Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R Lozada, Christine Luo, A. Elliott, Nicholas A. Zorko, Justin H. Hwang, David Takeda, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76c242e56ab4f45f6d1e940622fe0e76775cae6" target='_blank'>
              Formation of homolog pairing-induced domains in early Drosophila embryo genome
              </a>
            </td>
          <td>
            Lei Liu, Yanxia Jin, Yingyu Tao, Changbong Hyeon
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The epigenetic model for the development of cancer is based on the concept that cancer develops due to overall hypomethylation of the genome with concomitant hypermethylation of promoters of oncogenes. Recent research unravels the epigenetic mechanisms of the malignant transformation of the cell and it contributes to the development of more efficient therapies.

Methods. Detailed investigation of recent literature on PubMed using the keywords DNA repair, epigenetics

Conclusions. It is known that a prominent mechanism of the surveillance of the integrity of the genome is the DNA repair system that protects the cell from deleterious insults by the recruitment of the DNA Damage Response (DDR). Mutations of the genes of this complex system are associated with immunodeficiencies and a predisposition to cancer. Recently, defects of the epigenetic DNA repair sequential regulation are described in cancer tissues. The epigenetic dysfunction involves changes in the methylation patterns of promoters and genes, aberrant histone modifications and the expression of variants, deregulation of histone readers and miRNA processing which will be analyzed in the review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a012ab5b95d13c2f716ead45fcd5505e77df30ec" target='_blank'>
              Epigenetics in DNA Repair Mechanism and their Relation to Cancer
              </a>
            </td>
          <td>
            Salavoura Aikaterini
          </td>
          <td>2026-01-25</td>
          <td>Journal of Human and Clinical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The minor spliceosome (MiS) is a specialized RNA-processing machinery upregulated in cancer to promote expression of oncogenic genes. Here, we identify its catalytic component, U6atac snRNA, as a druggable vulnerability in prostate and breast cancers. U6atac knockdown triggers R-loop–mediated DNA damage while impairing DNA repair by downregulating key factors such as BRCA1, PARP1, TP53BP1, and CHK1/2, disabling both homologous recombination and non-homologous end joining. This dual effect sensitizes tumors to PARP inhibitors, cisplatin, and radiation, independent of BRCA status. Moreover, we uncover an adaptive resistance mechanism driven by extracellular vesicles enriched in U6atac, which amplify MiS activity and facilitate therapy escape; a process reversed by U6atac depletion. Across multiple in vitro and in vivo models, MiS targeting demonstrates tumor-selective activity with minimal toxicity. These findings position U6atac as a central regulator of genome stability and establish MiS targeting as a promising approach to potentiate genotoxic therapy and overcome resistance.



 Anke Katharina. Augspach, Mark Rubin, Paola Francica, Sven Rottenberg, Rahul Kanadia. Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c71eaaa6cc7e28ae4d142a474ab9e0350163c6b" target='_blank'>
              Abstract B003: Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, M. Rubin, P. Francica, Sven Rottenberg, Rahul Kanadia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The CRISPR-Cas9 gene editing system represents a powerful approach for precision targeting of genetic determinants involved in cancer cell survival. This study evaluated the ability of CRISPR-Cas9 to induce apoptosis in cancer cells through targeted disruption of anti-apoptotic genes. Acute myeloid leukemia (HL-60 and KG-1) and anaplastic thyroid cancer (8505C and C643) cell lines were used to knockout BIRC5 (survivin) and MADD using specific single-guide RNAs. Efficient genome editing was achieved, with indel formation exceeding 50% across all models. Targeting BIRC5 and MADD significantly reduced cell viability to 39.8–54.6% of control levels and induced substantial apoptosis. Total apoptotic populations reached 52.8–60.1% following BIRC5 knockout and 45.3–50.6% following MADD knockout. Apoptosis induction was confirmed by caspase-3/7 activation, increased Sub-G1 accumulation, and cleavage of apoptosis-related proteins, indicating activation of the intrinsic apoptotic pathway.Notably, BIRC5 disruption consistently produced stronger pro-apoptotic effects than MADD across all cancer models. Overall, these findings highlight the therapeutic potential of CRISPR-Cas9–mediated targeting of anti-apoptotic genes as a promising strategy for precision oncology and for overcoming resistance to conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c6b9d8f35f7365dc8f3a9d540781b27c8b647d" target='_blank'>
              Exploration of CRISPR-Cas9 Gene Editing System Utilization for Targeted Induction of Apoptosis in Cancer Cells
              </a>
            </td>
          <td>
            Resti Ariani, Eka Cahya Muliawati, Zola Efa Harnis
          </td>
          <td>2026-01-31</td>
          <td>Science Get Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="MYC genomic amplification and pathway activation is associated with aggressive behavior and poor prognosis in osteosarcoma (OS). However, a discordance exists between copy-number gains and transcriptional output from MYC in OS, and defining these mechanisms is critical to understand and intercept persistent MYC signaling. Here, we showed that cytoplasmic mRNA (poly(A)) sustains MYC activation in OS. Multi-omics profiling and single-cell transcriptomics identified TENT5A, a non-canonical RNA-binding poly(A) polymerase, as selectively upregulated in MYC-activated tumors and enriched in proliferative, stem-like populations. Biochemical and genetic evidence demonstrated that TENT5A directly bound MYC mRNA via its PAP/OAS1 domain, extended its poly(A) tail, and stabilized the transcript, thereby reinforcing MYC-driven stemness and chemoresistance. Gain- and loss-of-function assays, orthotopic xenografts, and patient-derived organoids confirmed that elevated TENT5A enhanced tumor-initiating capacity and reduced chemotherapy sensitivity. Pharmacologic inhibition of TENT5A disrupted MYC mRNA stabilization, shortened poly(A) tails, and reversed chemoresistance in preclinical models. These findings delineate a post-transcriptional RNA-stabilization pathway that reconciles the disconnect between MYC genetic alterations and transcriptional activity and nominate the RNA-binding protein TENT5A as a therapeutically tractable target in OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d515c1b7f0b534d63c1002ea77732498dd44f1" target='_blank'>
              TENT5A Maintains MYC mRNA Stability to Enhance Osteosarcoma Stemness.
              </a>
            </td>
          <td>
            Yining Tao, Qi Zhang, Haoyu Wang, Binghui Yang, Haoran Mu, Kaiyuan Liu, Weisong Zhao, Xiyu Yang, Bowen Zhao, D. Zuo, Liu Yang, Zhengdong Cai, Zongyi Wang, Hongsheng Wang, Yingqi Hua, Wei Sun
          </td>
          <td>2026-01-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Analytical challenges in detecting sequencing-unknown extrachromosomal circular DNA (eccDNA)─a molecule with critical roles in tumorigenesis and therapeutic potential─primarily arise from interference by long linear genomic and mitochondrial DNA. Here, we for the first time introduce a selective real-time rolling circle amplification (sRT-RCA) method that overcomes this limitation by combining RecBCD and PacI dual-enzyme digestion to remove interfering nucleic acids, thus permitting direct Phi29 polymerase-mediated RT-RCA without the need for conventional physical purification step. This approach achieves unprecedented sensitivity (2 pg, ∼4 × 10-19 mol circular DNA molecules) within a 104-fold higher background of genomic DNA. Furthermore, the established method, utilizing a simplified direct-lysis workflow, is capable of detecting eccDNA from as few as 125 cells. By this approach, we estimated that eccDNA averages 540 copies (8kb plasmid equivalents) in certain cancer cells (K562). Clinically, eccDNA was detected in 3/10 early stage and 6/10 advanced-stage patients. These findings indicate that eccDNA is widespread in cancer patients and demonstrate the potential utility of our method in clinical applications and translational research, thereby bridging a key technical gap and accelerating the translation of eccDNA-based diagnostics and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad0f1ea2bf7309e407f53aa8233632e2a1ae6a7f" target='_blank'>
              Ultrasensitive and Rapid Screening of Tumor-Associated Extrachromosomal Circular DNA by Selective Real-Time Rolling Circle Amplification.
              </a>
            </td>
          <td>
            Yiran Liu, Gang Li, Jing Zheng, Qing Li, Wei-Min Tong, Weiyi Lai, Hailin Wang
          </td>
          <td>2026-02-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Our studies show the levels of type I topoisomerases are increased in human papillomaviruses (HPV)-positive cells, leading to enhanced levels of DNA breaks and activation of DNA repair pathways. Furthermore, knockdown of either TOP1α or TOP3β impaired viral replication and transcription. Importantly, these two topoisomerases act by distinct mechanisms to regulate viral functions. Elucidating the mechanisms by which topoisomerases regulate viral gene expression and replication offers a promising avenue for targeted therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/843f6cbaaa32b6f9a8d2bc8fb897a0a89ef42241" target='_blank'>
              Differential roles of type I topoisomerases in regulating HPV pathogenesis
              </a>
            </td>
          <td>
            A. Vats, Conor W Templeton, Lou Laimins
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Meiotic drivers are selfish genetic elements that gain transmission advantages by distorting equal, Mendelian segregation. For decades, biologists have considered meiotic drivers as interesting, albeit esoteric, case studies. It is now clear, however, that meiotic drive is more common and phylogenetically widespread than previously supposed. Indeed, intensive study of a few well-known cases has begun to reveal the evolutionary genomic consequences of meiotic drive. We argue here that many features of genome evolution, content, and organization that are seemingly inexplicable by organismal adaptation or nearly neutral processes are instead best accounted for by recurrent histories of meiotic drive. We review how meiotic drive can affect the evolution of sequences, gene copy numbers, genes with functions in meiosis and gametogenesis, signatures of "selection", chromosome rearrangements, and karyotype evolution. We also explore the interactions of meiotic drive elements with other classes of selfish genetic elements, including satellite DNAs, transposable elements, and with the endogenous host genes involved in drive suppression. Finally, we argue that some aspects of drive-mediated genome evolution are now sufficiently well established that we might reverse the direction of discovery- rather than ask how drive affects genome evolution, we can use genome data to discover new putative drive elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d881fb701a16213929f614073f26c44a691e3d6" target='_blank'>
              The Evolutionary Genomics of Meiotic Drive.
              </a>
            </td>
          <td>
            Daven C. Presgraves, R. Dawe, K. Dyer, L. Fishman, Soumitra A Bhide, Sasha L Bradshaw, Meghan J. Brady, A. Burga, C. Courret, Brandon L Fagen, A. B. S. Ferretti, Reka K. Kelemen, Jun Kitano, Yiran Liu, Emiliano Martí, Theresa R Erlenbach, Josephine A. Reinhardt, Laura Ross, Jan-Niklas Runge, Callie M. Swanepoel, Beatriz Viçoso, Aaron A. Vogan, A. Lindholm, A. Larracuente, Robert L. Unckless
          </td>
          <td>2026-01-23</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The total DNA of an organism is called the genome. DNA contains all the instructions for how to build an organism and how that organism will function. Some DNA sequences are repeated thousands and thousands of times throughout the genome. Repeats lined up one after another are called satellite DNA. The number of copies of a given satellite DNA sequence can rapidly change, and differ among individuals. Although satellite DNAs were once considered useless, researchers are continuously discovering their important roles in various organisms. Satellite DNAs are essential for keeping organisms functioning properly. They help cells divide and keep the genome integrity. They can affect behavior, health, and help the organism to overcome stressful conditions. In these ways, satellite DNAs increase Earth’s biodiversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e93be9ccc179d27f89aeee0b54094657efcfefe" target='_blank'>
              Mysteries of Repetitive DNA Segments
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Miroslav Plohl
          </td>
          <td>2025-12-29</td>
          <td>Frontiers for Young Minds</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Late detection and tumor recurrence are major factors driving the lethality of high-grade serous ovarian carcinoma (HGSOC). PARP inhibitors (PARPi) have achieved significant clinical efficacy by selectively targeting DNA repair deficiencies in HGSOC patients with BRCA mutations and homologous recombination deficiency (HRD). However, a subset of patients ultimately develops resistance to PARPi, necessitating alternative effective treatment options. The mutational signatures of APOBEC3 family of DNA deaminases are widespread across a broad array of cancer types. Here, we report that cancer stem cell (CSC)-like tumorspheres exhibit reduced A3B expression compared to non-CSC adherent counterparts. Importantly, inhibition of A3B leads to PARPi resistance, elevated frequency of CSCs, and enhanced expression of stemness factors. In addition, we found that high A3B-expressing cells are under strong replication stress and thus synergize efficiently with PARPi. These studies reveal the important role A3B plays in regulating PARPi response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bba6bed6771f6f297d4c8454e956586305fa06f" target='_blank'>
              APOBEC3B enhances the efficacy of PARP inhibitors in elimination of ovarian cancer stem cell
              </a>
            </td>
          <td>
            Maria Rivera, Lucy Liu, Sabina Enlund, Chae-eun Lim, Haoran Zhang, Kaifu Yang, Roman Sasik, Leslie A. Crews, Kathleen M Fisch, R.N. Eskander, Frida Holm, Qingfei Jiang
          </td>
          <td>2026-01-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Patients with von Hippel-Lindau (VHL) disease develop multiple distinct kidney cysts and clear cell renal cell carcinomas (ccRCCs) throughout their lifetime. Although cysts are typically considered precursors of ccRCC, no molecular evidence of cysts to tumor progression was provided to date, due to the lack of an accurate molecular characterization of these lesions. We performed a comprehensive molecular characterization of four kidney cysts and six ccRCCs obtained from the same VHL-disease patient, thus all sharing the same genetic germline and host environment. We combined whole genome sequencing (WGS) and RNA sequencing (RNA-seq) profiling with pathological examinations. Cysts and tumors exhibited distinct transcriptomic profiles. Tumors showed increased glycolysis and response to hypoxia, while cysts were associated with enrichment in extracellular matrix organization and inflammation and displayed elevated expression of nephron distal portion markers. Deconvolution analysis further revealed different stromal and immune microenvironments with cysts enriched in myofibroblasts and plasma cells signatures whereas tumors were associated with tumor-associated macrophages (TAMs) and tumor vasculature signatures. Genomically, most cysts and tumors were clonally independent, harboring distinct somatic single-nucleotide variants. Despite sharing some somatic mutational signatures, tumors and cysts exhibited divergent somatic copy number alterations. Only tumors displayed chromosome 3p loss, resulting in VHL loss of heterozygosity, without other driver mutations. VHL patients develop multiple kidney cysts and solid tumors throughout their lifetime and their clinical management is challenging. This study presents the first in-depth molecular analysis of kidney cysts and ccRCC in a single VHL patient. We observed distinct molecular profiles between cysts and tumors, suggesting independent origins. While preliminary, these findings challenge the assumption that cysts always serve as precursors of ccRCC in VHL disease and underscore the need for larger studies to improve surveillance and management of renal lesions in patients with VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a5cddad02651b1247d054c02f906a3863c26a3" target='_blank'>
              Distinct genomic, microenvironmental, and nephron signatures in VHL kidney cysts and tumors
              </a>
            </td>
          <td>
            I. Rowe, F. Corea, G. Pipitone, G. Scotti, Roberta Lucianò, M. Sant'angelo, N. Tenace, Irene Franco, Annamaria Ferrara, Claudio Doglioni, Maurilio Ponzoni, F. de Cobelli, F. Montorsi, U. Capitanio, Paola Carrera, A. Larcher, Andrea Salonia, Federico Daniela Francesco Giorgia Maria Grazia Annalisa Ra Belladelli Canibus Cei Guazzarotti Patricelli Russ, F. Belladelli, D. Canibus, F. Cei, G. Guazzarotti, M. Patricelli, Annalisa Raucci Russo, Chiara Re, L. Salerno
          </td>
          <td>2026-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3fdd78ac1c63cef7b04dfd2c26d05276098c61" target='_blank'>
              Visualizing Interchromosomal Interactions at Sub-Megabase Resolution Using Network Clustering Coefficients
              </a>
            </td>
          <td>
            Yingjie Xu, Ian J. Anderson, R. P. McCord, Tongye Shen
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89078a30fc642f15d7607009ddb82791df527ad7" target='_blank'>
              Analysis of genetic mutation characteristics in 40 Chinese patients with hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Shaoshao Xu, Lei Li, Dongchang Yang, Tao Liu, Binyu Liu, Qingqing Xun, Yanrong Liu
          </td>
          <td>2026-02-09</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Autoimmune disorders (AID) are complex, multifactorial, and pervasive human health conditions that affect people globally. Nearly all AID are characterized by heightened inflammation and a breakdown of immune regulation, resulting in a loss of tissue homeostasis. Additionally, the hyper-inflammatory environment observed in AID is ideal for triggering neoplasia, resulting in various forms of cancer and further complicating disease treatment. Within the dysregulated immune microenvironment underlying AID, there is abundant DNA damage and genome instability. We posit that recurrent cycles of inflammation and DNA damage can result in stochastically elevated levels of somatic mutations in autoimmune disorders, and contribute to downstream malignancies. Understanding AID-associated DNA damage and mutagenicity would illuminate mechanisms that would subsequently inform disease prognosis, pathology, and progression. We explore this theme in our review by highlighting studies linking AID with genome instability and mutations, and carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fc6f49f8355a19b9e8cefc2dd834d5b14d2315b" target='_blank'>
              Genome Instability and Somatic Mutagenesis in Autoimmune Diseases
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Natalie Saini
          </td>
          <td>2026-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd77a79644efdb57765d420fb7be18f39a993472" target='_blank'>
              Cancer-associated synonymous mutations reveal stress signal-dependent mRNA folding that selectively modulates protein function
              </a>
            </td>
          <td>
            Sivakumar Vadivel Gnanasundram, Lixiao Wang, Sa Chen, Ondrej Bonczek, B. Vojtěšek, Robin Fåhraeus
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, may also play an important role in tetraploid (4N) cancer cell physiology. Here, we showed that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. Small size was associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size may contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Essential meiotic structure‐specific endonuclease 1 (EME1) is integral to the maintenance of genomic stability in various cancers. However, its biological role and expression profile of this molecule in nasopharyngeal carcinoma (NPC) remain to be explored. In this study, we found that EME1 was overexpressed in NPC specimens compared to adjacent noncancerous tissues and was correlated with poorer overall survival outcomes. Furthermore, EME1 knockdown significantly inhibited the proliferation and migration of NPC cells in vitro and in vivo, with a corresponding sensitization to either camptothecin (CPT) or olaparib, evidenced by a further suppression of proliferation upon drug treatment. Notably, silencing EME1 significantly increased the sensitivity of NPC cell lines to CPT by enhancing ATM‐CHEK2 phosphorylation and inducing nuclear abnormalities. Collectively, our findings suggest that combining EME1 modulation with agents such as CPT or olaparib could be an effective treatment strategy for NPC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4b0d132ea96aa28bec17ad81c4b1af8056bcb" target='_blank'>
              Targeting EME1 Increases the Sensitivity of Camptothecin in Nasopharyngeal Carcinoma Cells
              </a>
            </td>
          <td>
            Xizhen Jiang, Falian Liang, Zhirui Lin, Fang Yang, Dong-ping Chen, Gengde Hong, Jinquan Liu, Wenjing Yin, Mengyao Wang, Bin Qi
          </td>
          <td>2026-01-14</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="RecQ family of DNA helicases play pivotal roles in DNA replication, repair and responses to DNA damage or replication stress. Several human RecQ helicases are defective in diseases associated with chromosomal instability, premature aging and cancer. We recently discovered novel mutations in the RECQL4 gene in glioblastoma (GBM), the most malignant brain tumor in adults. Transcriptomic profiles of GBMs with REQCL4 mutations resembled those in REQCL4 KO glioma cells. We employ structural modelling and biochemical approaches to elucidate impacts of novel mutations on RECQL4 helicase activities. Using recombinant RECQL4P532S and RECQL4R766Q proteins we demonstrate that P532S substitution reduces the RECQL4 ability to unwind DNA and disrupts DNA-coupled ATP-hydrolysis activity. WT and mutated RECQL4 were overexpressed in RECQL4 KO glioma cells to study interactions with BLM helicases, cell viability and specific responses to UVC- and chemotherapy-induced DNA damage/repair. Overexpression of RECQL4P532S or RECQL4R766Q variants affected DNA repair and responses to chemotherapeutics in glioma cells, and RECQL4R766Q disturbed interactions with the BLM helicase. Our results reveal deleterious consequences of novel RECQL4 mutations in GBMs. The newly identified RECQL4 mutations affect RECQL4 helicases and their interactions with BLM contributing to glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650345dd7db490bb44b3ecdcc3b0ad865cd59a8" target='_blank'>
              Novel mutations in the RECQL4 gene affect its helicase functions, interactions with the BLM helicase and chemotherapeutics-induced cell death
              </a>
            </td>
          <td>
            Agnieszka Kaczmarczyk, Mikołaj Sokołowski, K. Wojnicki, Marta Pabiś, B. Wojtaś, I. Ciechomska, K. Poleszak, B. Gielniewski, S. K. Król, Matthew Guille, Sebastian Glatt, B. Kaminska
          </td>
          <td>2025-12-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da30b80b8917619722f2ccc295095fa65d05c44e" target='_blank'>
              Gene functions determine stochastic or adaptive futile transcription in cancers through deregulated start site deployments
              </a>
            </td>
          <td>
            Aditi Lakshmi Satish, Praveen Kumar, Ishani Morbia, Umashankar Singh
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Characterizing the physical organization of the genome is essential for understanding long-range gene regulation, chromatin compartmentalization, and epigenetic accessibility. Hi-C experiments generate two-dimensional (2D) genome-wide contact maps of chromatin interactions by capturing the spatial proximity between genomic loci, which reveal interaction frequencies but lack the spatial resolution needed to interpret the three-dimensional (3D) genome structure(s). Emerging evidence suggests that epigenetic regulation is closely linked to 3D genome architecture, and that structural changes over time (4D) drive key biological processes in development, disease, and environmental response. Thus, integrating 3D structure with functional data is critical for a more complete understanding of genome regulation. Previous work, most notably the 4DHiC chromosome modeling framework, has shown that physical multi-dimensional modeling approaches rooted in polymer physics and molecular dynamics can resolve these structures at biologically meaningful resolutions by integrating temporal Hi-C data with physical constraints to uncover dynamic chromosome reorganization. Thus, molecular dynamics simulations, constrained by Hi-C contact matrices, can resolve fine-scale structural changes and reveal functionally significant transitions in chromatin conformation. Herein, we present the 4D Genome Browser Workflow (4DGBWorkflow) and the 4D Genome Browser (4DGB). The algorithm is based on the 4DHiC method, and the containerized tool is an end-to-end workflow that can transform, filter, and view 4D epigenomics and chromatin datasets, allowing non-specialists to apply three-dimensional modeling principles to diverse datasets and experimental conditions. The software executes on a laptop running macOS, Linux or Windows. From input Hi-C files (.hic), the 4DGBWorkflow produces 3D reconstructions of chromosomes, integrates the reconstruction with track data (e.g., epigenetic marks, transcriptome profiles), and provides comparative visualization of the results in a single workflow. The 4DGBWorkflow and 4D Genome Browser are open-source tools for comparative analysis and visualization of 4D chromosome datasets, including chromatin architecture and epigenomic signals. Automatic integration of Hi-C data with molecular dynamics democratizes the construction of time resolved 3D genome structures, simplifying complex simulations and data integration schemes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e99ba06ed5c338fe5fbe39dbd1637f6375018c6c" target='_blank'>
              From 2D to 4D: a containerized workflow and browser to explore dynamic chromatin architecture
              </a>
            </td>
          <td>
            David H. Rogers, Cullen Roth, Cameron Tauxe, Jeannie T. Lee, Christina R. Steadman, K. Sanbonmatsu, Anna Lappala, Shawns Starkenburg
          </td>
          <td>2026-01-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DEAD-box (DDX) RNA helicases are essential regulators of RNA metabolism and gene expression. Among them, DDX10 remains poorly characterized despite growing evidence supporting its involvement in human diseases. This review provides a comprehensive analysis of DDX10, from its structural and functional features to its emerging roles in solid tumors and hematologic malignancies. We discuss how DDX10, through its conserved domains, contributes to pre-rRNA processing, ribosome biogenesis, and cell proliferation, and explore potential links between DDX10 and processes such as liquid–liquid phase separation (LLPS) and epigenetic regulation, which may underlie its roles in cancer cell plasticity and stress response. We argue that the dysregulation of these fundamental cellular processes positions DDX10 as a focal point where aberrant RNA metabolism and altered molecular condensates converge to disrupt transcriptional homeostasis and drive oncogenic transformation. Aberrant DDX10 expression is a recurrent feature across multiple cancers, where it promotes tumor progression, therapy resistance, and poor prognosis. Moreover, DDX10 participates in oncogenic fusion events, most notably the NUP98::DDX10 fusion identified in a subset of acute myeloid leukemias, which drives leukemogenesis by disrupting transcriptional regulation and cellular differentiation. Given its tumor-associated expression and diverse biological functions, DDX10 is increasingly recognized as a potential diagnostic biomarker and a promising target for therapeutic strategies. By consolidating current knowledge under this unifying framework, this review highlights the multifaceted roles of DDX10 in cancer biology, advocating further research into its molecular functions and translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6180292fbacfd0407c4b7281fb3b5202eef78597" target='_blank'>
              DDX10 RNA Helicase: Structure, Function, and Oncogenic Roles Across Solid and Hematologic Tumors
              </a>
            </td>
          <td>
            Giorgia Isinelli, Genny Scacci, Arianna Capocchia, C. Emiliani, Cristina Mecucci, R. la Starza, Danika Di Giacomo
          </td>
          <td>2026-01-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pheochromocytomas and paragangliomas (PPGLs) are rare neuroendocrine tumors primarily involving the adrenal medulla and its associated paraganglia, with heterogeneous clinical behavior and complex molecular drivers. This study aimed to characterize DNA methylation and gene expression patterns in PPGLs to understand the molecular differences between tumor subtypes and malignancy. We performed an integrative analysis of DNA methylation (Illumina EPIC 850K) and gene expression profiles (Affymetrix microarrays) in 24 PPGLs, comparing these with The Cancer Genome Atlas (TCGA) data, to delineate cluster- and malignancy-specific epigenetic patterns. Comparison between pseudohypoxic Cluster I and kinase-signaling Cluster II tumors revealed 13 differentially methylated CpG sites, with a specific CpG within DSCAML1 showing hypermethylation in Cluster II accompanied by increased expression, suggesting context-dependent gene body methylation effects. Benign versus malignant comparisons identified 101 differentially methylated CpGs, including hypermethylated CpG in BAIAP2L1 and hypomethylated CpG in SHANK1 in malignant tumors. Pathway enrichment of differentially methylated genes revealed alterations in Notch signaling, adherens junctions, cytoskeletal regulation, and intracellular transport. Gene expression analysis demonstrated partial overlap between clusters, with malignant tumors exhibiting distinct transcriptional profiles involving RNA processing, metabolism, and adhesion pathways. Correlation between methylation and expression was generally limited, emphasizing that methylation-dependent gene regulation is a locus-specific and context-dependent regulation. These findings illustrate a complex interplay between epigenetic modifications and transcriptional programs in PPGLs, enhancing our understanding of molecular heterogeneity and tumor classification, and identifying candidate biomarkers and therapeutic targets for malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abb446f2ee7aba49b9c00aa8f44910b6b716df36" target='_blank'>
              Integrative Epigenomic and Transcriptomic Profiling Define Malignancy- and Cluster-Specific Signatures in Pheochromocytomas and Paragangliomas
              </a>
            </td>
          <td>
            Mouna Tabebi, M. Łysiak, Oliver Gimm, P. Söderkvist
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Compacting chromatin within the cellular nucleus presents a significant challenge for biology. Chromosomes must be both condensed and spatially organized to enable essential processes such as transcription and replication. Chromosome conformation capture experiments (e.g., Hi-C) provide valuable information about the spatial organization and, therefore, the connectivity between different genomic regions. These experiments inspired polymer models that describe the physical mechanism of the chromosomal energy landscape. The Full-Inversion Chromatin model (FI-Chrom), a data-driven approach for modeling genome organization, uses Hi-C contact maps to infer pairwise interaction potentials between all chromosomal loci. It combines Graphics Processing Unit (GPU)-accelerated simulations with efficient training of tens of millions of parameters derived from the maximum-entropy principle to determine 3D structures of chromosomes that accurately reproduce Hi-C-like data. FI-Chrom does not make any a priori assumptions regarding chromosome architecture, making it applicable to any chromosome conformation capture experiment. Its derived structural ensembles capture all essential features from the short- and long-range interactions of typical chromosome organization, such as segregated compartments, chromosome territories, and fully or partially formed loops. Although Hi-C contains only structural information, FI-Chrom extends these data by revealing an emergent dynamical mechanism encoded in the inferred energy landscape. For example, simulations show that chromatin loops are not static architectural features but rather transient structural elements. Statistical analyses further indicate that loops confined within a single compartment occur more frequently than those spanning multiple compartments, highlighting the dynamic and compartment-dependent nature of chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5e464495ba1f67ebcfcfd2e8d42c98c8759239d" target='_blank'>
              A data-driven chromatin model reveals spatial and dynamic features of genome organization.
              </a>
            </td>
          <td>
            A. B. Oliveira Junior, Matheus F. Mello, Ronaldo J Oliveira, Esteban Dodero-Rojas, Sumitabha Brahmachari, V. Contessoto, J. Onuchic
          </td>
          <td>2026-01-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The human genomic triad refers to the presence in every human somatic cell of a small, maternally inherited mitochondrial genome, as well as two nuclear genomes, inherited from the father and mother. Men carry distinct sex chromosomes, Y and X, in their two genomes. Women also have two distinct nuclear genomes with X sex chromosomes, one inherited from the mother and one from the father. Currently, the state-of-the-art in genome research, including forensic applications, is the sequencing and assembly of the diploid genome by haplotypes (two genomes). This is crucial, primarily for DNA phenotyping, which is a valuable complement to other approaches used for DNA identification in criminal investigations. Moreover, the role of DNA phenotyping after the start of universal DNA population registration (which will happen sooner or later) will diminish over time, but will not disappear completely, as the remains of people who have not yet undergone such DNA registration may still be present. The ethical issues raised in connection with both DNA registration and DNA phenotyping are largely imaginary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36148c17214e356d391370566f85696093ca0bf1" target='_blank'>
              THE TRIAD OF HUMAN GENOMES AND GENOMIC REGISTRATION OF THE ENTIRE POPULATION
              </a>
            </td>
          <td>
            A. Chemeris, R. Garafutdinov, A. Khaliullina, Rushan Galyautdinov, F. Aminev
          </td>
          <td>2025-12-30</td>
          <td>Bulletin of the Institute of Law of the Bashkir State University</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, , Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, , Namiko Abe, Gonçalo R. Abecasis, F. Aguet, Christine Albert, L. Almasy, Alvaro Alonso, Seth Ament, Peter Anderson, Pramod Anugu, Deborah Applebaum-Bowden, K. Ardlie, D. Arking, A. Ashley-Koch, Stella Aslibekyan, T. Assimes, Paul L. Auer, D. Avramopoulos, N. Ayas, Adithya Balasubramanian, John Barnard, Kathleen Barnes, R. G. Barr, E. Barron-Casella, L. Barwick, Terri L Beaty, Gerald Beck, Diane M. Becker, Lewis C. Becker, Rebecca L. Beer, A. Beitelshees, Emelia Benjamin, Takis Benos, Marcos Bezerra, Larry Bielak, Joshua C. Bis, Thomas Blackwell, Nathan Blue, Don Bowden, Russell P Bowler, Jennifer Brody, U. Broeckel, J. Broome, Deborah L. Brown, Karen L. Bunting, Esteban G. Burchard, Carlos Bustamante, Erin J. Buth, J. Cardwell, V. Carey, J. Carrier, C. Carty, Richard Casaburi, , James M. Casella, Peter Castaldi, M. Chaffin, Christy Chang, Yi-Cheng Chang, Daniel I. Chasman, S. Chavan, Bo-Juen Chen, Wei-Min Chen, Michael Cho, Seung Hoan Choi, Lee-Ming Chuang, M. Chung, Ren-Hua Chung, Clary Clish, S. Comhair, M. Conomos, Elaine Cornell, Adolfo Correa, Carolyn Crandall, J. Crapo, L. Cupples, J. Curtis, B. Custer, Coleen Damcott, D. Darbar, S. David, Colleen P. Davis, M. Daya, Michael R. DeBaun, Dawn Demeo, R. Deka, Scott Devine, H. Dinh, H. Doddapaneni, Qing Duan, Shannon Dugan-Perez, J. P. Durda, S. Dutcher, Charles B. Eaton, L. Ekunwe, Adel Boueiz, Patrick T Ellinor, L. Emery, S. Erzurum, Charles Farber, J. Farek, T. Fingerlin, M. Flickinger, C. Frazar, Mao Fu, Stephanie M. Fullerton, L. Fulton, Stacey Gabriel, Weiniu Gan, Shanshan Gao, Yan Gao, Margery L S Gass, Heather Geiger, B. Gelb, M. Geraci, S. Germer, R. Gerszten, Auyon Ghosh, Richard Gibbs, Chris Gignoux, M. Gladwin, David E. Glahn, S. Gogarten, Da-Wei Gong, H. Goring, S. Graw, Kathryn J. Gray, Daniel Grine, Colin Gross, Yue Guan, Namrata Gupta, Jeffery Haessler, Michael Hall, Yi Han, Patrick J. Hanly, Daniel N Harris, Nicola L. Hawley, Benjamin D. Heavner, S. Heckbert, Ryan Hernandez, David M. Herrington, Craig Hersh, Bertha Hidalgo, James E Hixson, Brian Hobbs, John Hokanson, Elliott Hong, Karin F. Hoth, C. Hsiung, Jianhong Hu, Yi-Jen Hung, H. Huston, C. Hwu, M. Irvin, Rebecca D. Jackson, D. Jain, C. Jaquish, J. Johnsen, Andrew Johnson, Craig W Johnson, R. Johnston, Kimberly Jones, H. M. Kang, R. Kaplan, Shannon Kelly, Michael Kessler, Alyna T Khan, Ziad M. Khan, Wonji Kim, J. Kimoff, G. Kinney, Barbara A. Konkle, Holly Kramer, Christoph Lange, Ethan Lange, Cathy C. Laurie, C. Laurie, M. LeBoff, J. Lefaive, Jiwon Lee, Sandra Lee, Wen-Jane Lee, David Levine, Joshua Lewis, Xiaohui Li, Yun Li, Henry J. Lin, Honghuang Lin, Yongmei Liu, Yu Liu, S. Lubitz, K. Lunetta, James Luo, U. Magalang, Michael P. Mahaney, B. Make, Alisa K Manning, J. Manson, Melissa A Marton, S. Mathai, Susanne May, P. McArdle, Merry-Lynn N. McDonald, Sean K. McFarland, S. McGarvey, Daniel J. McGoldrick, Caitlin P. McHugh, B. McNeil, Hao Mei, J. Meigs, Vipin Menon, L. Mestroni, G. Metcalf, Deborah Meyers, Emmanuel Mignot, Julie Mikulla, N. Min, Mollie A Minear, M. Moll, Z. Momin, M. Montasser, Courtney G. Montgomery, Donna M Muzny, Girish N. Nadkarni, Rakhi Naik, Sergei Nekhai, Sarah C. Nelson, Bonnie Neltner, C. Nessner, Deborah Nickerson, Osuji Nkechinyere, T. O’Connor, H. Ochs-Balcom, Geoffrey O. Okwuonu, Allan Pack, David T. Paik, Nicholette D. Palmer, J. Pankow, G. Papanicolaou, Cora Parker, J. Peralta, Marco V. Perez, U. Peters, Lawrence S. Phillips, Jacob Pleiness, Toni Pollin, Wendy S. Post, Julia Powers Becker, M. Boorgula, Michael H. Preuss, P. Qasba, D. Qiao, Zhaohui Qin, N. Rafaels, M. Rajendran, , Laura Rasmussen-Torvik, A. Ratan, Robert M Reed, C. Reeves, E. Regan, M. Reupena, Kenneth M. Rice, Rébecca Robillard, N. Robine, Dan Roden, C. Roselli, I. Ruczinski, Alexi Runnels, Pamela H. Russell, S. Ruuska, Kathleen A. Ryan, E. Sabino, D. Saleheen, S. Salimi, Sejal Salvi, Steven Salzberg, Kevin Sandow, Vijay G Sankaran, J. Santibanez, K. Schwander, D. Schwartz, F. Sciurba, C. Seidman, J. Seidman, Vivien Sheehan, S. Sherman, Amol C. Shetty, A. Shetty, Wayne Hui-Heng Sheu, M. B. Shoemaker, Brian H. Silver, Edwin Silverman, Robert Skomro, A. V. Smith, Josh Smith, Nicholas L. Smith, Tanja Smith, S. Smoller, Beverly Snively, M. Snyder, Tamar Sofer, N. Sotoodehnia, A. Stilp, G. Storm, E. Streeten, Jessica Lasky Su, Y. Sung, Jody M Sylvia, Adam A. Szpiro, F. Sériès, D. Taliun, M. Taub, Matthew Taylor, Simeon Taylor, M. Telen, Timothy A. Thornton, M. Threlkeld, Lesley E. Tinker, D. Tirschwell, Sarah Tishkoff, Hemant K. Tiwari, Catherine Tong, Michael Tsai, Dhananjay Vaidya, David Van Den Berg, Peter Vandehaar, S. Vrieze, Tarik Walker, R. Wallace, A. Walts, Fei Fei Wang, Heming Wang, Jiongming Wang, Karol Watson, Jenn Watt, D. Weeks, J. Weinstock, B. Weir, Scott T. Weiss, L. Weng, Jennifer Wessel, Cristen J. Willer, Kayleen Williams, , Scott Williams, Carla Wilson, James G. Wilson, L. Winterkorn, Q. Wong, Baojun Wu, Joseph Wu, Huichun Xu, Lisa R. Yanek, Ivana V. Yang, Ketian Yu, S. Zekavat, Yingze Zhang, Snow Xueyan Zhao, Xiaofeng Zhu, Elad Ziv, Michael C. Zody, S. Zoellner, Mariza de Andrade, Paul de Vries, L. Fuentes, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="ABSTRACT The single-stranded DNA deaminase APOBEC3B (A3B) is capable of potently restricting the replication of a range of viruses, including retroviruses (cDNA) and herpesviruses (genomic DNA). However, these and likely other DNA virus families have evolved host species-specific counter-defenses that are equally potent and serve to protect viral DNA from restriction. Although high-risk human papillomavirus (HPV) infection triggers A3B upregulation, potentially as part of an antiviral response, the impact of this restriction factor on papillomavirus replication and pathogenesis has yet to be assessed. To study human A3B antiviral function in the absence of a species-specific counter-defense, here, we ask whether human A3B is capable of restricting Mus musculus papillomavirus (MmuPV1) in cellulo and in vivo. First, we created human A3B and catalytic mutant A3B-E255A expressing FVB/N mice. Second, MmuPV1 gene expression and replication were quantified in primary keratinocytes from these animals and, surprisingly, enzymatically active human A3B caused no measurable impairment in viral transcript or DNA accumulation. Third, A3B, catalytic mutant A3B-E255A, and nontransgenic FVB/N animals were infected with MmuPV1, and similar pathologies were found, regardless of the A3B functionality. Thus, despite likely never being exposed to human A3B during evolution, MmuPV1 appears to be unaffected by this potent, primate-specific antiviral factor. These results suggest that MmuPV1 and perhaps papillomaviruses more broadly possess a conserved mechanism to efficiently escape restriction by human A3B and related DNA deaminases. IMPORTANCE Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes, such as A3B. Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes, such as A3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352aeb9cedcc877ce026a88960367a20f3937484" target='_blank'>
              Mus musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, S. H. Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2026-02-11</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Loss of ATRX function, a diagnostic criterion of IDH-mutant astrocytoma, is closely associated with alternative lengthening of telomeres (ALT), a telomere maintenance mechanism (TMM). As immunohistochemical (IHC) assessment of ATRX is error-prone, sequencing has been integrated into clinical workflows. While frameshift and nonsense variants can be classified as loss-of-function (LOF) mutations, missense variants remain difficult to interpret. To address this, we analyzed ATRX-altered gliomas from TCGA (N = 539 tumors, 587 alterations) and a local cohort of 100 diffuse gliomas. Aside from IDH-mutant astrocytoma and H3.3-mutant glioma, glioblastoma (13–19%), oligodendroglioma (2%), and MAPK-altered tumors (2–7%) were consistently represented in both cohorts. Missense mutations accounted for 18% (TCGA) and 28% (local) of variants. Functional annotation of TCGA missense mutations using IHC, NGS-based TMM status, transcript levels, and RNA-seq–derived telomerase signatures (EXTEND) allowed us to functionally annotate only 3 of 106 variants. Therefore, we directly assessed the TMM status in the local cohort, using the TeloDIAG assay and retrieved 80% of ALT-positive samples, corresponding to 96% (N = 53) of IDH-mutant astrocytomas and 74% (N = 19) of histone-mutant gliomas, as expected. Importantly, ALT activity was also observed in 42% (N = 19) of glioblastomas and 86% (N = 7) of MAPK-altered gliomas, indicating potential implications for clinical behavior and therapeutic targeting. While sensitivity for LOF variant detection was similar—88% for TMM and 80% for IHC—TMM performed better for missense alterations (85 vs. 65%) and is more discriminant in overall survival prediction. Overall, 83% (N = 24) of missense mutations associated with an ALT phenotype were located within the major functional domains of the ATRX protein, compared to only 59% when considering all missense mutations, highlighting the functional relevance of the assay. Collectively, these findings emphasize the clinical value of functional TMM assessment in refining glioma diagnosis. Last, targeting of ALT-associated DNA-damage deficiency and immunogenic context is under clinical investigation, highlighting the promising theranostic potential of TMM assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f0f2e7f115b422093b4581e65109cca6ca78ab8" target='_blank'>
              Functional analysis of telomere maintenance mechanisms is more informative than immunohistochemistry for ATRX mutation interpretation in Gliomas
              </a>
            </td>
          <td>
            Clémence Guerriau, Camille Léonce, C. Carpentier, K. Mokhtari, F. Bielle, A. Dridi-Aloulou, P. Lomonte, D. Meyronet, Marc Sanson, L. Castro-Vega, D. Poncet
          </td>
          <td>2025-12-20</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is a paediatric liver malignancy arising from hepatic precursor cells, with >90% of cases harbouring a mutation in exon 3 of CTNNB1. We present a fully genetically characterised HB tumour organoid (tumoroid) biobank, which allows for in vitro studies of disease progression and clonal dynamics in vitro. We established a biobank of 14 tumoroid lines from 9 different patients. Tumours and tumoroids were characterised by whole genome sequencing (WGS) and histology, revealing strong concordance in cell morphology and β-catenin staining. In tumour—tumoroid pairs, identical pathogenic CTNNB1 variants were found, alongside shared copy number alterations (CNAs) and mutations. Variant allele frequency (VAF) was consistently higher in tumoroids, indicating increased tumour purity in vitro. In addition to CTNNB1, we frequently observed ARID1A alterations (single-nucleotide variants [SNVs] or CNAs in 56% of patients), and MYC gains as described previously. In paired pre- and post-treatment samples, we observed a clear increase in mutational load, attributed to a chemotherapy signature. Notably, from one patient, we analysed 4 tumour samples (3 post-treatment) with 4 matching tumoroid lines, all carrying a novel BCL6 mutation and loss of ARID1A. Mutational profiles varied across samples from different locations, suggesting intratumoral heterogeneity and clonal selection during tumoroid derivation. Taken together, this biobank allows detailed analysis of HB tumour biology, including treatment-induced progression and clonal dynamics across temporally and spatially distinct samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af5f1ebd1dcbe4bda8108f789b036cfe1c91f713" target='_blank'>
              A Fully Annotated Hepatoblastoma Tumoroid Biobank Details Treatment-Induced Evolution and Clonal Dynamics in Paediatric Cancer Patients
              </a>
            </td>
          <td>
            Gijs J. F. van Son, F. Ringnalda, M. V. van Roosmalen, Thomas A. Kluiver, Quinty Hansen, E. Duiker, Marius C. van den Heuvel, V. E. de Meijer, R. D. de Kleine, R. R. de Krijger, J. Zsiros, Weng Chuan Peng, R. van Boxtel, M. van de Wetering, Karin Sanders, Hans Clevers
          </td>
          <td>2026-01-18</td>
          <td>Organoids</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="High mobility group A1 (HMGA1), a non-histone chromatin structural protein encoded by the HMGA1 gene, plays a critical role in cancer. Recent studies have increasingly focused on its functions in genomic stability and cell death, revealing its involvement in tumorigenesis, cancer progression, and chemotherapy resistance. Consequently, inhibiting HMGA1 represents a promising strategy for developing novel cancer therapies. This review summarizes the cellular and molecular functions of HMGA1 in regulating genomic integrity and cell death in cancer. Furthermore, we discuss current HMGA1-targeting strategies, with emphasis on approaches leveraging its structural and functional characteristics, aiming to provide new insights for future research on HMGA1-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dfc0cc9a26137eed4ac1b69fa1e9407df182b1" target='_blank'>
              The role of HMGA1 in genome stability: Implications in human cancer
              </a>
            </td>
          <td>
            Xin-Yuan Lei, Kai-yue He, Yong-Ping Jian, Z. Xu
          </td>
          <td>2026-01-26</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The genome is compacted in the nucleus through a hierarchical chromatin organization, ranging from chromosome territories to compartments, topologically associating domains (TADs), and individual nucleosomes. Nucleosome remodeling complexes hydrolyze ATP to translocate DNA and thereby mobilize histone proteins. While nucleosome remodeling complexes have been extensively studied for their roles in regulating nucleosome positioning and accessibility, their contributions to higher-order chromatin architecture remain less well understood. Here, we investigate the roles of two key nucleosome remodelers, esBAF and INO8°C, in shaping 3D genome organization in mouse embryonic stem cells. Using Hi-C, we find that loss of either remodeler has minimal effects on global compartment or TAD structures. In contrast, subcompartment organization is notably altered, suggesting that esBAF and INO8°C contribute to finer-scale chromatin topology. To overcome the limited resolution of Hi-C for detecting regulatory loops, we employed promoter capture Micro-C (PCMC), which revealed that the loss of esBAF or INO8°C alters a subset of promoter anchored looping interactions. Although these changes occur at distinct genomic loci for each remodeler, the affected sites are commonly enriched for bivalent chromatin regions bound by OCT4, SOX2, and NANOG (OSN), as well as BRG1 and INO80 themselves. Together, our findings reveal that esBAF and INO8°C selectively influence subcompartment identity and enhancer-promoter communication at key regulatory loci, highlighting a previously underappreciated role for nucleosome remodelers in higher-order chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa668452cd533a08523fe27084c36abb8bbb84f0" target='_blank'>
              esBAF and INO8°C fine-tune subcompartments and differentially regulate enhancer-promoter interactions.
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2026-02-05</td>
          <td>Genetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="PHF20 encodes plant homeodomain finger protein 20 (PHF20), a component of the KAT8‐containing nonspecific lethal (NSL) complex that deposits acetylation on histone H4 to activate gene expression. We report two unrelated individuals with developmental delay, microcephaly, and distinctive facial features, in whom exome sequencing and chromosomal microarray analysis revealed a homozygous deletion of PHF20 that segregated with the disease phenotype in their families. Breakpoint junction sequencing revealed an Alu–Alu‐mediated deletion event. Western blot in cells from an affected individual showed undetectable PHF20, while levels of other NSL complex subunits were unaltered. Transcriptomic and epigenomic analysis revealed significant downregulation of gene pathways related to cell projection and neuronal development, associated with reduced histone H4K16 acetylation at these genes. In conclusion, our data suggest that homozygous deletion of PHF20 leads to a neurodevelopmental syndrome, potentially through targeted epigenetic dysregulation and altered gene expression essential for neuronal development. Identifying additional families with biallelic PHF20 variants will further delineate the phenotypic spectrum, and molecular studies in neuronal cell lines will be essential for understanding the disease mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ebe2b3854a5aa35254b0ecad5b6631cc63aebb0" target='_blank'>
              Homozygous Deletion of the Epigenetic Regulator PHF20 in Individuals With Neurodevelopmental Disorder
              </a>
            </td>
          <td>
            Shira Yanovsky Dagan, Hongwen Xuan, Jonathan Rips, Emuna Paz-Ebstein, Talia Baer, Shira Gross, A. Frumkin, Xiaobing Shi, T. Harel
          </td>
          <td>2025-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background & objectives Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer. Methods This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6low (negative/weak) or SMC6high (moderate/strong). Results SMC6low tumours tend to show higher CNV. SMC6high group presented poorer disease-free survival than the SMC6low group (P=0.050), mainly for the luminal subtype (P=0.005). SMC6low/ERpos were protective biomarkers for recurrence. Interpretation & Conclusions There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-11-01</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences capable of self-replication (especially retrotransposons) within the genome, which may lead to various forms of DNA damage. The introduction of this review encompasses the diverse classes and subclasses of TEs, particularly emphasizing the most active TEs present in the human genome. An analysis of the retrotransposition process of TEs is presented, illustrating how this mechanism can result in DNA damage and gene rearrangements. Furthermore, the review meticulously examines the implications of TE insertions on gene expression and genomic organization, which may contribute to the development of various diseases, including cancer. The relationship between TE activation and the aging process is also explored, with an emphasis on that epigenetic modifications associated with aging can lead to the derepression of TEs, thereby promoting genomic instability and inflammation. These factors may play a significant role in the pathogenesis of age-related diseases, such as cancer, cardiovascular disorders, and neurodegenerative conditions. Finally, the review considers potential therapeutic approaches aimed at targeting TE activity to alleviate the impacts of aging and associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1552ca333c184d411cd1fdc06105d2ca4b0a5435" target='_blank'>
              The role of transposable elements activity in genomic instability and their relationship to aging process.
              </a>
            </td>
          <td>
            Jingran Hu, Tianhao Mao, Kainan Huang, Shangzhi Yang, Wenrui Yu, Jiacheng Huang, Shiqi Jin, Chuanyu Sun, Zeyidan Jiapaer, Xianli Wang
          </td>
          <td>2026-01-18</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Highlights What are the main findings? Only 2.3% (52 total) of all human antisense genes have been examined in head and neck cancer (HNC) thus far. HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1 have clinical promise for future biomarker development. What are the implications of the main findings? Robust cross-validations on antisense gene aberrations will promote biomarker development and implementation in the HNC field, as genetic biomarkers are lacking. Antisense gene aberrations and treatment responses are underexplored and warrant future studies. Abstract Antisense genes (usually suffixed by -AS) represent a class of long non-coding RNAs (lncRNAs) transcribed from the opposite strand of annotated human genes or exon(s). A total of ~2236 human antisense genes exist in the human genome. Their genomic locations with respect to the corresponding sense genes, their dysregulated expression patterns in cancer specimens, and clinical associations with patient outcomes reveal their potential importance in clinical settings. As of today, there lacks a comprehensive review of HNC-associated antisense genes/transcripts to help move forward the antisense field for genetic biomarker development or future drug research. In total, 2.3% (52/2236 antisense genes) of all known human antisense genes have been investigated in head and neck cancer (HNC). Thus, we perform a comprehensive review of the genomic aberrations (mutations, copy number changes, RNA-expression dysregulation, and single nucleotide polymorphisms) associated with HNC patient prognosis, disease progression, cancer cell signaling, drug sensitivity, and radio-resistance. Four antisense genes, namely HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1, have been clinically cross-validated and have consistently demonstrated to be associated with patient outcomes in multiple independent cohorts by different research teams, with clear evidence for the prioritization of clinical biomarker development in HNC. Single nucleotide polymorphisms (SNPs) of antisense genes with evidence for HNC risk or outcomes should be further validated in different ethnic groups, for potential global HNC applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb801fbeee9cde00558b26b3b11efef6455c2950" target='_blank'>
              Genomic Aberrations of Antisense Gene Transcripts in Head and Neck Cancer
              </a>
            </td>
          <td>
            Jishi Ye, S. M. Abbang, Yuen-Keng Ng, V. Lui
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b560eb7504f0651c24b57c1e03e1ecc5ce721e" target='_blank'>
              DNAJB6 as an immuno-oncogenic hub in liver hepatocellular carcinoma: multi-omic profiling reveals prognostic significance and therapeutic vulnerability
              </a>
            </td>
          <td>
            Yiting Deng, Jin Li, Peng Wang, Xuelian Luo, Hongxia Xu, Yu Xiao, Zhenchuan Wang, Yifei Lu, Ying Chen, Feng Ge
          </td>
          <td>2026-01-21</td>
          <td>Journal of Molecular Histology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hereditary paragangliomas (PGLs) caused by germline SDHD pathogenic variants (PVs) exhibit a parent‐of‐origin effect, with tumors arising almost exclusively when the PV is inherited from the paternal allele. The Hensen model proposes that a cluster of maternally expressed tumor suppressor genes (TSGs) on chromosome 11p15.5 may play a crucial role in SDHD‐related PGL tumorigenesis, wherein somatic loss of maternal 11p and wild‐type SDHD allele, in conjunction with a paternally inherited SDHD PV, triggers tumor development. To systematically localize and identify the most crucial maternal‐expressed TSGs within 11p15.5, we developed a novel single nucleotide variant (SNV)‐oriented, capture‐based targeted enrichment approach followed by next‐generation sequencing (NGS) to enable high‐resolution loss‐of‐heterozygosity (LOH) analysis. Among 13 SDHD‐related PGLs and 23 non‐SDHD‐related PGLs, a somatic loss of 11p15.5‐15.4 was detected in 92% and 47%, respectively, a significant difference (p = 0.0035). Parental genotype analysis confirmed that the lost chromosome was of maternal origin. In our studies, 12/13 SDHD‐related tumors demonstrated complete loss of the maternal 11p15.5‐15.4 region, preventing localization of a specific driver TSG. Only one exceptional SDHD‐related tumor retained this region, warranting further investigation into the mechanism underlying parent‐of‐origin tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27b365a662db25feb9efe53f1684d83f7862183" target='_blank'>
              A Novel Targeted Sequence for Chromosome 11p15.5 Maternal Loss in SDHD‐Related Paragangliomas
              </a>
            </td>
          <td>
            Huei-Pin Lai, Chen-Hui Lee, Hsuan Hu, An-Ko Chung, Pei-Lung Chen, Wei-Shiung Yang, Wan-Chen Wu
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a leading cause of cervical cancer and a significant contributor to anogenital and oropharyngeal malignancies worldwide. While the oncogenic functions of HPV oncoproteins E6 and E7 in disrupting nuclear tumor suppressor pathways are well established, their influence on mitochondrial biology has only recently emerged as a critical facet of HPV-driven carcinogenesis. This review synthesizes current evidence on the qualitative and quantitative alterations of mitochondrial DNA (mtDNA) and their functional consequences in HPV-associated cancers. We discuss how E6 and E7 modulate mitochondrial dynamics, bioenergetics, and redox balance, contributing to metabolic reprogramming, resistance to apoptosis, and adaptation to tumor microenvironmental stress. We also examine the clinical significance of mtDNA mutations, deletions, and copy number variations as potential biomarkers for diagnosis, prognosis, and therapy response. Advances in multi-omics approaches, high-throughput sequencing, and patient-derived organoid models have accelerated the exploration of mitochondria as therapeutic targets. Integrating mitochondrial profiling into HPV-related cancer research holds promise for identifying novel metabolic vulnerabilities and guiding the development of mitochondria-directed treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25dca4a25a42e375c179e79296851a7aeadff46b" target='_blank'>
              Mitochondrial DNA Alterations in HPV-Related Cancers: Emerging Insights and Future Directions
              </a>
            </td>
          <td>
            M. O. Cakir, Melis Selek, Guldide Kayhan, Betül Yılmaz, Mustafa Ozdogan, G. H. Ashrafi
          </td>
          <td>2026-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How retrotransposon mobilization that can be retained for genome evolution persists despite negative outcomes of retrotransposon activity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR retrotransposon nomad completes the full mobilization cascade (including mRNA export, protein translation, and reverse transcription) to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings show that retrotransposon-derived DNA largely remains as ecDNA. This could prevent widespread genomic integration during spermatogenesis, potentially preserving genome stability with the presence of limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9420ae83c1dc149dca66d3f18d2a4cfd87487588" target='_blank'>
              Retrotransposon activation during spermatogenesis achieves massive ecDNA biogenesis but rare integration.
              </a>
            </td>
          <td>
            Lauren Tracy, Yao Chen, Z. Zhang
          </td>
          <td>2026-01-22</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="5-Methylcytosine (mC) is a key regulatory element of mammalian genomes, and plays important roles in development and disease. mC is predominantly written onto CpG dyads by DNA methyltransferases, and can be further oxidized by ten-eleven translocation dioxygenases (TETs) to 5-hydroxymethyl-, 5-formyl-, and 5-carboxylcytosine. This process results in different symmetric and asymmetric combinations of cytosine forms across the two strands of CpGs, each of which represents a unique physicochemical signature in the major groove of DNA. A comprehensive understanding of the individual functions of oxidized mC modifications can therefore only be achieved by considering both strands of CpG dyads. Here, we provide a brief overview of the current state of knowledge on the sequencing and mapping of individual CpG dyad states, their influence on the intrinsic properties of DNA, and their interactions with chromatin proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce94d7f0a338919de147b40cf1b27947501ec1c" target='_blank'>
              The role of cytosine modification symmetry in mammalian epigenome regulation
              </a>
            </td>
          <td>
            Zeyneb Vildan Cakil, Lena Engelhard, Daniel Summerer
          </td>
          <td>2025-12-29</td>
          <td>Chemical Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease at the cellular level and analyzing the genetic and molecular profile is essential for targeted therapy. Cancer cells continue to mutate, often resulting in drug resistance. In addition, cancers such as triple-negative breast cancer (TNBC) lack the target proteins used in some of the most effective therapies. This necessitates the identification of novel target proteins and biomarkers for effective treatment strategies. Ubiquitin E3 ligases are often differentially expressed in cancer cells, and numerous anticancer agents have been developed to inhibit them. SMURF2 is an E3 ligase that is differentially expressed in multiple cancer types. Although inhibiting upregulated SMURF2 may be strategically straightforward, enhancing the downregulated gene is often difficult. In addition, because E3 ligases ubiquitinate a variety of substrate proteins, targeting SMURF2 requires detailed analysis to achieve anticancer effect. This review discusses the dual role of SMURF2 in carcinogenesis and addresses the complex context-dependent function of SMURF2 in the various cellular pathways. In addition, resistance to existing cancer therapy related to SMURF2 and sensitivity mechanisms is discussed. Lastly, theranostic strategies for anticancer agents and biomarker development are suggested.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f6a4efa5a333f3bf6aa2ee9217bfa27f7bea38" target='_blank'>
              SMURF2 in Anticancer Therapy: Dual Role in Carcinogenesis and Theranostics
              </a>
            </td>
          <td>
            Joy Eom, Yejin Chun, Hae Ryung Chang
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d56a6de92ca30529df87362f7d733257748de198" target='_blank'>
              Fibrillarin and fibrillarin-like their role in cancer progression: new approaches and perspectives
              </a>
            </td>
          <td>
            Jaime Abelardo Ceja-López, Jany Valdes, Ana Miladinović, Ludovica Antiga, L. Rodríguez-Zapata, Pavel Hozak, Enrique Castaño
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology Reports</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The NUDT family comprises evolutionarily conserved enzymes that hydrolyze diverse substrates, including nucleoside phosphates, inositol polyphosphates, and RNA caps. Contrary to earlier perspectives focusing primarily on genome protection, compelling evidence now indicates that the majority of NUDT function as pro-tumorigenic factors. Cancer’s complex landscape, characterized by uncontrolled proliferation, evasion of apoptosis, metabolic reprogramming (like the Warburg effect), and genomic instability, creates an environment where NUDT exert significant influence. Key NUDT members, such as MTH1 (NUDT1), NUDT5, NUDT15, and NUDT22, are frequently overexpressed in cancers and actively promote tumor survival and progression. They achieve this not only by “sanitizing” the nucleotide pool to maintain genomic stability in cancer cells (hydrolyzing damaged nucleotides/caps), but also by dysregulating critical signaling pathways. The relationship between NUDT and cancer is multifaceted, involving intricate roles in nucleotide metabolism, redox homeostasis, and DNA repair. This functional diversity underscores their potential as therapeutic targets. Pharmacological inhibition of specific NUDT, particularly MTH1 and NUDT5, is an active area of research. Such inhibition aims to exploit cancer cell vulnerabilities by increasing the accumulation of damaged nucleotides and enhancing susceptibility to DNA-damaging agents (e.g., chemotherapy, radiotherapy) or PARP inhibitors, offering promising avenues for novel combination therapies. This review comprehensively overviews the mechanisms, diverse functions, and pathophysiological roles of NUDT in cancer biology, critically evaluating their therapeutic potential and the challenges in targeting them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32bb55be44a45d931e8f7256b168a3f66129eaa7" target='_blank'>
              The multifaceted regulatory roles of Nudix hydrolases in cancer and their therapeutic potential
              </a>
            </td>
          <td>
            Jianguo Pan, Jiaxi Yang, Kewei Wang
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT DNA surveillance mechanisms are crucial for maintaining genome stability and minimizing mutation rates. Mismatch repair (MMR) corrects replication errors that escape DNA-polymerase proofreading. In most organisms, MMR is orchestrated by MutS and MutL proteins. However, certain Archaea and Actinobacteria, including the major human pathogen Mycobacterium tuberculosis, lack these components. Instead, they appear to rely on the nuclease EndoMS/NucS, a structurally distinct enzyme governing a non-canonical MMR pathway. Since M. tuberculosis acquires drug resistance exclusively through chromosomal mutations, understanding its mutation rate regulation is critical. Nevertheless, the role of NucS in drug resistance evolution remains largely unexplored. We investigated NucS function in M. tuberculosis and uncovered a unique resistance dynamic distinct from other Actinobacteria. Deleting nucS altered the mutational spectrum but had minimal impact on the emergence of antibiotic-resistant mutants, contrasting sharply with other Actinobacteria where nucS loss dramatically increases mutation rates. This atypical behavior cannot be attributed solely to the R144S NucS polymorphism present in the H37Rv reference strain. Introducing the consensus NucS sequence in H37Rv produced only subtle changes in mutational spectrum without major effects on mutation rates. Analysis of 44,921 M. tuberculosis genomes revealed that most R144S-containing strains belong to the Euro-American sub-lineage 4.9, with no significant association with antibiotic resistance. However, nucS is under strong purifying selection, and R144S changes arose independently during M. tuberculosis evolution (homoplasy). Overall, our findings challenge the view that nucS serves as a genome stability guardian in M. tuberculosis and suggest additional mismatch repair mechanism(s) beyond NucS in this pathogen. IMPORTANCE DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat. DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d734f5ee057101186e729eca6288974decb87e6d" target='_blank'>
              The unique role of nucS-mediated noncanonical mismatch repair in Mycobacterium tuberculosis resistance evolution
              </a>
            </td>
          <td>
            I. Martín-Blecua, Jorge Sastre-Domínguez, J. R. Valverde, Pablo García-Bravo, Ángel Ruiz-Enamorado, Rafael Prados-Rosales, Lahari Das, William R. Jacobs, Á. San Millán, Jesús Blázquez, Sonia Gullón
          </td>
          <td>2025-12-22</td>
          <td>mBio</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ea2c9c7605b557b8e4ec203f1280961cf9800c" target='_blank'>
              Exploring the oncogenic mechanism of plasmacytoma variant translocation 1 (PVT1) gene in solid cancers; emphasis on microRNA regulation pathways.
              </a>
            </td>
          <td>
            Mojtaba Zehtabi, Shirin Azizidoost, Abdolah Mousavi Salehi, Mahrokh Abouali Gale Dari, Mohammad Amin Zargar, Maryam Khombi Shooshtari, Krzysztof Data, Dominika Domagała, Julia Niebora, Małgorzata Józkowiak, P. Chmielewski, Aleksandra Partyńska, Aleksandra Górska, A. Bryja, M. Kulus, Adam Kamiński, Teresa Wysocka, Marek Spaczyński, Piotr Dzięgiel, P. Mozdziak, Maryam Farzaneh, B. Kempisty
          </td>
          <td>2026-01-29</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PURPOSE
Gene fusions are molecular rearrangements with oncogenic potential. However, their role in disease progression and therapy resistance remains largely unexplored, particularly in breast cancer.


EXPERIMENTAL DESIGN
In this study, we utilized multi-omics data from the AURORA program to characterize gene fusions in metastatic breast cancer. We analyzed RNA sequencing data from 325 primary tumors and 350 metastatic lesions across 476 patients to develop a high-confidence catalog of gene fusions.


RESULTS
In a subset of 398 matched samples from 199 patients, we observed a higher burden of gene fusions in metastatic tumors compared to their corresponding primary tumors. Metastatic-specific (acquired) gene fusions were characterized by few recurrent rearrangements and were generally absent in patients with de novo metastatic disease. We observed a colocalization of gene fusions with subtype-specific copy-number gains. Various scores indicative of genomic instability were found to be associated with the gene fusion burden. Fusions involving genes located within the same topologically associating domain (TAD) were common in HER2-positive tumors and frequently acquired in metastatic triple-negative breast cancer. The presence of gene fusions, particularly those that were acquired or involved genes within the same TAD, was associated with poor prognosis in ER-positive/HER2-negative tumors. Patients with acquired ESR1 fusions exhibited a more aggressive disease course, characterized by shorter treatment response and poorer clinical outcomes.


CONCLUSIONS
The AURORA gene fusion catalog may serve as a valuable resource for identifying targetable genetic alterations, thereby supporting the development of more effective therapeutic strategies for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67d22cb70a54186d4c79975c42b05e6e536bd92" target='_blank'>
              Molecular and clinical analyses of gene fusions identify therapeutic targets in paired primary and metastatic breast cancer from the AURORA program (BIG 14-01).
              </a>
            </td>
          <td>
            C. Biagioni, D. Fimereli, Alexandre Irrthum, A. Guerrero-Zotano, M. Piccart, D. Cameron, C. Caballero, Alice Raimbault, D. Martins-Branco, Dario Romagnoli, E. Agostinetto, D. Venet, Arnau Llinas-Bertran, Jose A Seoane, G. Zoppoli, Ana Vivancos, Andrea Vingiani, Mafalda Oliveira, G. Curigliano, Thayane Crestani, C. Sotiriou, E. Nili Gal-Yam, F. Hilbers, Philippe Aftimos, M. Benelli
          </td>
          <td>2025-12-22</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Elucidating transcriptional regulatory mechanisms within tissues is essential for both physiological and pathological studies of giant panda. Here, we generate transcriptomic profiles and construct 3D genome maps for nine giant panda tissues. For the first time, we describe alterations in 3D genome organization among giant panda tissues, providing evidence for changes of A/B compartmentalization, variations in topologically associating domains, and rewiring of promoter-enhancer interactions. We demonstrate that many 3D genome reorganization events are tissue-specific and these structural changes are associated with tissue-specific expression of genes and tissue-relevant biological functions. Furthermore, we use these 3D genome maps to systematically interpret adaptive evolution in giant panda genome, emphasizing importance of considering the genome as a 3D configuration when dissecting potential functions of DNA sequence variation. Collectively, our study increases understanding of functional regulatory mechanisms of giant panda tissues and provides a foundational dataset for analysis of tissue-specific regulatory networks in giant panda. The authors generate transcriptomic profiles and construct 3D genome maps for nine giant panda tissues, revealing tissue-specific 3D genome reorganization linked to gene expression and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c6e2a39a6240d55a8cd9334672f5c4459bbc0c" target='_blank'>
              A compendium of chromatin interaction maps in the Giant Panda genome
              </a>
            </td>
          <td>
            Pengliang Liu, Jiaman Zhang, Kailai Cai, Juan Wang, Hong Liu, Liang Zhang, Wei Xu, Yuliang Liu, Fujun Shen, Rong Hou, Yan Li
          </td>
          <td>2026-01-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3dd774faf6e3b7da8495305c2cee4721accab18" target='_blank'>
              Substantial unannotated noncoding transcripts in tumors may transcriptionally regulate cancer-related genes.
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2026-01-28</td>
          <td>BMC biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f88af5bb8874c2112f831be5ec9601df23ed25e" target='_blank'>
              MER57E3 transposable elements regulate gene expression in a human cell model of neural development.
              </a>
            </td>
          <td>
            Michelle Almeida da Paz, Umut Yildiz, Minyoung Kim, Víctor Campos-Fornés, Marina Pinkasz, Thomas Dahlet, K. Noh, Leila Taher
          </td>
          <td>2026-01-31</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Clinical genomic sequencing has revealed that prostate cancer (PCa) progression is associated with changes to chromosome number and structure, termed chromosomal instability (or CIN). While mutations in common driver genes are rare in primary PCa, CIN is detected in >90% of cases, suggesting that CIN may be a driver of the disease. However, the underlying mechanisms that cause CIN in PCa are poorly understood. Cells are susceptible to CIN during mitosis, when the duplicated genome must be equally segregated into two daughter cells. Normally, two centrosomes (cytoplasmic, non-membranous organelles) nucleate and organize microtubules to guide assembly of the mitotic spindle, thus ensuring the fidelity of cell division. However, we recently discovered that cells within primary PCa lack centrosomes, the first cancer type to show this phenomenon. Furthermore, we showed that centrosome elimination in non-tumorigenic cells generates CIN which can transform these cells, causing them to form xenograft tumors in mice. Therefore, we sought to understand the underlying mechanisms that drive the centrosome loss observed in PCa. The prostate is a naturally hypoxic organ, averaging around 4% oxygen tension in the healthy organ which declines as men age. Additionally, tumor hypoxia is strongly correlated with high CIN, particularly in prostate cancers. However, there are few mechanistic links between hypoxia and CIN. Therefore, we hypothesized that hypoxia could cause centrosome loss and, thus, predispose cells to CIN. Indeed, we found that immortalized prostate RWPE-1 cells progressively lost centrosomes when exposed to 1% O2 for as short as 6 hours in vitro. Using ultrastructure expansion microscopy (U-ExM), we find centrosomes are replaced by non-functional microtubule husks devoid of centrosome proteins which we call ‘remnants’. These phenomena also occur in PC-3 and LNCaP prostate cancer lines. Furthermore, we observe centrosome loss specifically in the hypoxic regions of xenograft tumors. Using a combination of RNAseq and chemical inhibitors, we determined that centrosome elimination sits at the crossroads of 3 transcriptional programs: (1) centrosome elimination occurs in basal cells and basal-like cancers that express p63; (2) cells must be confluent with active Hippo pathway; and (3) hypoxia induces a HIF-independent transcriptional response. When these conditions are met, cells transcriptionally downregulate 4 key centrosome proteins (Cep63, Cep152, Cep192, and Cep215), resulting in the depletion of a protective shell of pericentriolar material (PCM) and subsequent centrosome elimination. By overexpressing the mitotic kinase Polo-like kinase 1 (PLK1), we can strengthen the PCM and prevent elimination. Finally, we identify PCa cells containing remnants within prostatectomy tissue samples. This work is significant as it identifies the first examples and mechanisms of centrosome elimination in human somatic and cancer cells, provides a mechanistic link between hypoxia and CIN, and identifies remnants as a cellular indicator for CIN predisposition.



 John M. Ryniawec, Natalya K. Seppanen, Gregory C. Rogers, Anne E. Cress. Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0fec5efccdcc580bc977b0a1bb739d683cd475" target='_blank'>
              Abstract B065: Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer
              </a>
            </td>
          <td>
            John M. Ryniawec, Natalya K. Seppanen, Gregory C Rogers, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Rashedunnabi Akanda, Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is driven by a combination of genetic alterations and non-mutational mechanisms that disrupt normal hematopoiesis and support leukemic cell survival. While the mutational landscape of AML is well characterized, the non-genetic processes that sustain leukemic maintenance remain comparatively less understood. Using human AML cell lines and murine models of AML, we identify BCL2-associated transcription factor 1 (BCLAF1) as a key regulator of leukemic progression through control of mRNA processing. BCLAF1 physically associates with core spliceosome components and regulates alternative splicing, with a predominant effect on intron retention. We demonstrate that BCLAF1 is required for the productive splicing of activating transcription factor 4 (ATF4) mRNA, thereby sustaining ATF4 protein expression. Loss of BCLAF1 reduces ATF4 protein levels, leading to downregulation of metabolic target genes and disruption of de novo amino acid biosynthesis. Furthermore, depletion of BCLAF1 sensitizes AML cells to venetoclax, a clinically relevant BCL-2 inhibitor. Together, these findings uncover a previously unrecognized role for BCLAF1 in coordinating mRNA splicing and metabolic adaptation in AML, highlighting its potential as a therapeutic target. Statement of significance Aberrant RNA splicing and metabolic reprogramming are hallmarks of cancer, yet how these processes are mechanistically linked remains unclear. This study identifies BCLAF1 as a key regulator connecting splicing control to amino acid metabolism in acute myeloid leukemia, revealing a previously unrecognized functional vulnerability at the intersection of these pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2686f36b152260cde0d49166c250ce1a1551afc7" target='_blank'>
              BCLAF1 links RNA splicing to ATF4-dependent metabolic adaptation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Laura López-Hernández, Stephanie J Crowley, Sara Cea-Sánchez, Ricardo de Arellano, Neetij Krishnan, Patrick Toolan-Kerr, Lynn S. White, Domenico Ignoti, Emily Soto-Hidalgo, Stefano Gustincich, Román González-Prieto, Luca Pandolfini, Isaia Barbieri, Patricia Altea-Manzano, Jeffrey A. Magee, J. Bednarski, Gonzalo Millán-Zambrano
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b6108bbfc8477081a30cd15de6db71d5ddd4a1" target='_blank'>
              Exploiting synthetic lethality in epithelial ovarian cancer: multi-dimensional approaches beyond DNA damage repair.
              </a>
            </td>
          <td>
            Mandana Bigdeli, Elizabeth Tremblay, Diane Provencher, A. Mes-Masson, Francis Rodier
          </td>
          <td>2026-02-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d108778b128ae240ef789dd566dc8ad0a8e1f3" target='_blank'>
              Characterization and clinical implications of CpG island methylator phenotypes of resistant tumors.
              </a>
            </td>
          <td>
            Fei Hou, Xu Zhou, Yu-E Huang, Haizhou Liu, Mengqin Yuan, Jiahao Chen, Quan Wang, Wei Jiang
          </td>
          <td>2026-01-30</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfec9467e3b3849fb1a9045da602c08a96d6544" target='_blank'>
              Prospects for Studying the Interrelations of the Neurofibromin 1 Gene with Retroelements
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2026-01-01</td>
          <td>Russian Journal of Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Oncogenic KRAS and NRAS mutations are common in hematologic malignancies, but how they signal is less well characterized than in carcinomas. To uncover novel RAS biology and potential therapeutic vulnerabilities, we employed a multi-omics screening approach in multiple myeloma to identify regulators of RAS activity. We report that the phosphatase PP1C dephosphorylates the conserved T148 residue on RAS, which in turn permits LZTR1-dependent proteasomal degradation. Notably, LZTR1 is ineffective against KRAS A146 gain-of-function mutations, which are adjacent to T148 and prevalent in hematologic cancers. Remarkably, we find that KRAS protein is four-fold less stable in hematologic versus carcinoma cells, offering a unique therapeutic opportunity targeting RAS protein stability mechanisms. The kinases PAK1 and PAK2 shield RAS from LZTR1-dependent degradation by phosphorylating T148, and targeting PAK1/2 activity improves RAS-directed therapy. Collectively, our findings reveal a novel regulatory circuit governing RAS stability that is preferentially active in blood cancers and potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b303e43e88283dfa0d5dd96708dd15950394c6bf" target='_blank'>
              Phosphorylation Protects Oncogenic RAS from LZTR1-Mediated Degradation
              </a>
            </td>
          <td>
            Lin Zhang, Arnold Bolomsky, Omar Al-Odat, Callie K VanWinkle, Aaliyah Battle, Papiya Chakraborty, Ronald J. Holewinski, T. Andresson, Qingcai Meng, James D. Phelan, J. Muppidi, Ryan M. Young
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The field of epitranscriptomics discovered N6-methyladenosine (m6A) modifications, which function as fundamental elements that control RNA metabolism properties that powerfully affect cancer biology. This review examines the way m6A modifications shape RNA stability while regulating translation, together with their eraser and reader proteins. We demonstrate that m6A modifications guide oncogene and tumor suppressor transcript outcomes, which promote tumor growth, metastasis, and therapeutic resistance. The regulatory function of m6A depends significantly on its relationship with ncRNAs that mainly include miRNAs, lncRNAs, and circRNAs. The review examines the effects of m6A on ncRNA production, stability, export, and degradation, as well as the regulation of m6A protein expression by ncRNAs, highlighting intricate reciprocal feedback loops that drive cancer progression. The interplay between m6A RNA modifications and ncRNAs provides emerging evidence on how they collectively influence the tumor microenvironment, modulate immune system responses, and contribute to resistance. Harnessing ncRNA-m6A interactions for managing drug resistance offers promising therapeutic avenues. However, advancing our understanding of the context-specific roles of m6A modifications and translating these insights into clinical applications remains a significant challenge. This review synthesizes recent findings on ncRNA-m6A crosstalk to lay the groundwork for developing epitranscriptomic strategies in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e0eb5d331194e0274cc28e14543827ba3e7f5f" target='_blank'>
              The epitranscriptome meets non-coding RNA: m6A-mediated regulation in oncogenesis and therapy
              </a>
            </td>
          <td>
            Prasanna Srinivasan Ramalingam, M. Rejili, Faouzi Haouala, Md Sadique Hussain, Yumna Khan, Mudasir Maqbool, Janaki Ramaiah Mekala, S. Arumugam
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19852969bd81baff3364e7db6ef026b54ff18d9" target='_blank'>
              Circulating extrachromosomal circular DNA as a prognostic biomarker for colorectal cancer.
              </a>
            </td>
          <td>
            Xuanmei Luo, Jian Cui, Jinxin Shi, Gaoyuan Sun, Lili Zhang, Yayu Li, Yingyu Guo, Lu Kuai, Tianhan Sun, Qi Luo, Jiahui Cai, Q. An, Wei Zhang, Fei Xiao, Gang Zhao
          </td>
          <td>2026-02-07</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia-myelodysplasia related (AML-MR) is a biologically and clinically distinct subtype of AML that arises in the context of prior dysplasia. It is characterized by adverse cytogenetics and poor prognosis compared to other AML subtypes. Several genetic mechanisms underpin the pathogenesis of AML-MR; however, additional findings are likely to come to light over time with advanced genomic technologies, enhancing our understanding of their evolution. This report details a case of AML-MR involving unreported gene fusion. A 59-year-old female with multiple comorbidities presented with slurred speech. Pathological evaluation and DNA-based next-generation sequencing results were consistent with AML-MR. AML fluorescence in situ hybridization (FISH) panel revealed an extra signal for RUNX1. G-banding karyotype revealed a solitary rare t(X;21)(q26.1;q22.12) in 18 out of 20 cells analyzed. Optical genome mapping (OGM) was performed to precisely localize the breakpoints and identify the specific genes or gene fusions created by the translocation. OGM identified a novel fusion involving ENOX2 (Xq26.1) and RUNX1 (21q22.12), which was subsequently confirmed by a retrospective custom FISH probe targeting ENOX2. The identification of an ENOX2::RUNX1 fusion in AML-MR expands the spectrum of rare RUNX1 fusions. High-resolution approaches such as OGM enable precise delineation of fusion partners and breakpoints beyond the resolution of conventional cytogenetics. While the biological and clinical significance of this fusion remains to be determined, this finding highlights the value of OGM in the identification of novel and rare genomic rearrangements in leukemia and other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ef03479e0b598417cb6526e465be7236182fc" target='_blank'>
              Optical genome mapping identifies novel ENOX2::RUNX1 t(X;21)(q26.1;q22.12) rearrangement in acute myeloid leukemia-myelodysplasia related: first case report
              </a>
            </td>
          <td>
            G. V. George, Sarmad Ali, Chauncey R. Syposs, M. Iqbal
          </td>
          <td>2025-12-19</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colon cancer (CC) is a malignancy with high global incidence and mortality, and elucidating its underlying molecular mechanisms is critical for improving prognostic assessment and therapeutic strategies. In this study, transcriptomic data from a large cohort of CC samples and a limited number of normal controls from the TCGA database were used to construct a multigene prognostic risk model using univariate Cox and least absolute shrinkage and selection operator (LASSO) regression analyses. The expression of key prognostic genes, including immunoglobulin superfamily member 9 (IGSF9), was further validated in CC tissues by PCR and Western blotting. Functional assays were performed in HCT116 cells to investigate the biological effects of IGSF9 overexpression and its regulatory relationship with p53. The prognostic model identified IGSF9 as a gene significantly associated with patient survival. Although IGSF9 expression was reduced at both the mRNA and protein levels in CC tissues, its overexpression in vitro markedly promoted apoptosis, alleviated DNA damage, and suppressed cell migration and invasion. Notably, silencing of p53 partially reversed the tumor‐suppressive effects induced by IGSF9 overexpression, indicating that IGSF9 exerts its biological functions in a p53‐dependent manner. Collectively, these findings demonstrate that IGSF9 acts as a tumor suppressor in colorectal cancer and regulates DNA damage responses and apoptosis through a mechanism partially dependent on p53, highlighting its potential value as a prognostic biomarker and therapeutic target in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ff548ce8387a244b0397236c92065d62ddf1f1" target='_blank'>
              P53 Inhibition Diminishes IGSF9 Gene Activity to Promote DNA Repair and Exacerbate the Progression of Colon Cancer
              </a>
            </td>
          <td>
            Huan-Yu Zhang, Dan Tian, Wan-Fu Zhang, Ying-Hui Zhang, Jia-Li Feng, Juan Sheng, Xue-Qin Shang
          </td>
          <td>2026-01-01</td>
          <td>Journal of Biochemical and Molecular Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) is characterised by profound genomic instability and limited durable responses to standard therapy, leading to poor prognosis. The use of next-generation sequencing technologies has improved understanding of its molecular landscape, revealing consistent Tumour Protein p53 (TP53) mutations, homologous recombination defects, pathway alterations, and epigenetic dysregulation. Such genomic profiling now underpins the classification criteria between the ovarian cancer subtypes described by the Cancer Genome Atlas. Widespread chromosomal instability and pathogenic variants in multiple genes distinguish HGSOC from other subtypes of ovarian cancer and, further, from low-grade serous ovarian cancer. Importantly, the new-found understanding of the genomic landscape of HGSOC guides the use of platinum-based chemotherapies and Poly(ADP-ribose) Polymerase (PARP) inhibitors, with homologous recombination deficiency emerging as a cancer vulnerability that enhances treatment response. A combined multi-omics approach integrates transcriptomics, proteomics, metabolomics, and epigenomics to further the understanding of the characteristics, therapeutic targets and treatment resistance within HGSOC. Despite these advances, major challenges persist, including intratumoural heterogeneity and the poor diversity of genomic datasets. Artificial Intelligence (AI) technology, Clustered regularly interspaced short palindromic repeats (CRISPR)-based gene editing, neoantigen-guided immunotherapy and ovarian cancer vaccination indicate a promising future for genomics-guided interventions and support the integration of genomics within multi-omic approaches to improve HGSOC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63375259d1ee4ee2c082f84da6b555791abc91c8" target='_blank'>
              High-Grade Serous Ovarian Carcinoma in the Genomics Era: Current Applications, Challenges and Future Directions
              </a>
            </td>
          <td>
            Molly Elizabeth Lewis, Chiara Caricato, Hannah Leigh Roberts, Subhasheenee Ganesan, Nadia Amel Seksaf, E. Maniati, Michail Sideris
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DNA-protein crosslinks (DPCs) are highly toxic DNA lesions that arise both from normal cellular metabolism and as an intended consequence of cancer chemotherapy. Key anticancer agents, including topoisomerase poisons and PARP inhibitors, exert their therapeutic effects by trapping enzymes on DNA, converting them into toxic barriers that block replication. To counteract this threat, cells have evolved specialized mechanisms to detect and remove DPCs. This review explores the molecular mechanisms by which these therapies trap proteins on DNA and the multi-layered defense systems cells use to resolve them-ranging from enzymatic degradation to mechanical extraction. We further examine how these processes are modulated by the cell cycle and chromatin landscape. Importantly, we highlight emerging evidence that alterations in DPC repair pathways are frequent in cancer and serve as critical determinants of treatment response. Ultimately, this review integrates mechanistic insights with clinical data to highlight how exploiting DPC repair defects can overcome drug resistance and guide the development of rational, synthetic lethal combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7911590556e8eee40c77772227dc1932d925b21a" target='_blank'>
              Cellular strategies for repairing trapped protein-DNA complexes
              </a>
            </td>
          <td>
            Maria Sideridou, Doukissa Ioanna Machli, Dora Lontra, Theodoros Rampias
          </td>
          <td>2026-02-10</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc055c9a06947ee16301772ce73247173385166" target='_blank'>
              GPSeq maps the radial organization of eukaryotic genomes along the nuclear periphery-center axis.
              </a>
            </td>
          <td>
            Wing Hin Yip, Kaja Harton, Ilaria Castiglioni, Britta A. M. Bouwman, Carlos Jiménez, Emily Georgiades, L. Harbers, Wenjing Kang, Erik Wernersson, Nicola Crosetto, M. Bienko
          </td>
          <td>2026-01-19</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Fusion oncogenes drive malignant transformation by altering transcriptional programs, disrupting cellular homeostasis, and frequently rewiring protein condensate networks. Emerging evidence suggests that aberrant condensate formation is a key mechanism through which these fusion proteins exert oncogenic control. Specifically, by amplifying transcriptional output, sequestering cofactors, and enforcing pathological gene expression states. Despite their genetic validation as oncogenic drivers, many fusion proteins remain pharmacologically intractable due to the absence of catalytic domains or well-defined binding pockets. To overcome this, we developed an integrated discovery platform that directly assesses the biophysical and functional properties of fusion protein-driven condensates and enables identification of small molecules that disrupt these oncogenic assemblies. Using protein language models and in silico predictors of phase separation, we systematically mapped condensate-forming potential across recurrent oncogenic fusions catalogued in TCGA. This analysis revealed that most fusion proteins (including EML4-ALK and EWS-FLI1) harbor extensive intrinsically disordered regions (IDRs) that promote abnormal condensation and transcriptional dysregulation. We further identified disease-specific variants that alter condensate formation relative to wild-type proteins, providing mechanistic insight into oncogenic gain- and loss-of-function events. To experimentally validate and therapeutically exploit these findings, we established PhaseScan™, a high-throughput droplet microfluidics screening platform capable of quantitatively profiling condensate dynamics and screening thousands of compounds against reconstituted fusion condensates to identify therapeutically relevant small molecule inhibitors. Across multiple fusion proteins, PhaseScan™ identified small molecules that selectively disrupt aberrant condensates while sparing physiological assemblies such as nuclear speckles and stress granules. Using EML4-ALK as a representative model, we demonstrated that compound-mediated condensate dissolution disrupts oncogenic ALK signaling and reverses oncogenic phenotypes in cell-based systems. Our results highlight condensate biology as a generalizable vulnerability across fusion-driven cancers and demonstrate the feasibility of discovering small molecules against fusion proteins traditionally considered undruggable. By integrating AI-driven predictive modeling, and on-target molecular high-throughput screening, our platform provides a roadmap for rationally identifying and drugging fusion protein condensates. This approach establishes condensate-targeted therapeutics as a novel class of precision medicines for patients with fusion-positive malignancies.



 Andrew Seeber, William E. Arter, Seema Qamar, Cinzia Sgambato, Amal Alex, Kadi Saar, Julia Doh, Martin Kulander, Tuomas Knowles, Shilpi Arora. Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0c93f691d35862b44375fdee303e25769c39e0" target='_blank'>
              Abstract PR004: Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform
              </a>
            </td>
          <td>
            A. Seeber, W. Arter, S. Qamar, Cinzia Sgambato, Amal Alex, K. Saar, Julia Doh, M. Kulander, Tuomas Knowles, Shilpi Arora
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct fragmentomic characteristics that are related with their cellular origins. Here, we report that somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like fragmentomic characteristics, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Further investigations suggest that such cancer-like traits are associated with somatic variants derived from clonal hematopoiesis. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilize AI to integrate genomic, fragmentomic, and epigenomic features to develop diagnostic models named FreeSV and FreeSV+. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features are also included, the AUCs of FreeSV+ model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d931b70db2d8495e7d9b12e6b1f59f47b890a1" target='_blank'>
              Cancer-Like Fragmentomic Characteristics of Somatic Variants in Cell-Free DNA.
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2026-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af6246b41466868460c788c77446ef122ab1c54" target='_blank'>
              Epigenetic evolution of IDHwt glioblastomas.
              </a>
            </td>
          <td>
            B. Deng, R. Head, K. Draaisma, A. Hoeben, S. Lukacova, G. Lombardi, S. Leenstra, M. Hanse, R. Fleischeuer, C. Watts, N. Angelopoulos, T. Gorlia, Vincent Bours, M. J. van den Bent, Pierre A. Robe, Y. Hoogstrate, P. French
          </td>
          <td>2026-01-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Breast cancer is one of the most severe global health challenges, with its incidence continuing to rise. The development of this malignancy is associated with numerous risk factors; however, its primary etiology often remains unclear. Research on the potential association between breast cancer and carcinogenic human papillomavirus (HPV) has been conducted for three decades, yet a definitive consensus has not been reached. The development of prophylactic cervical cancer vaccines has greatly heightened the urgency of this issue: if a causal relationship can be established, it would open a feasible avenue for preventing this common cancer. This review aims to systematically integrate relevant evidence and provide a critical commentary on the association between HPV and breast cancer. We analyzed evidence from 34 studies on HPV DNA detection in breast cancer cells. We discuss the prevalence and genotypes of HPV detected in breast cancer across different geographical regions, the state of the viral genome (integrated vs. episomal), viral load, and potential mechanisms for HPV-associated breast carcinogenesis—particularly its role in inducing genomic instability via activation of APOBEC3B. Additionally, we explore the histopathological and clinical characteristics of HPV-positive breast cancer. The synthesized evidence suggests that high-risk HPV types are present in a subset of breast cancers worldwide, but with lower viral loads compared to cervical cancer, indicating a potentially different mode of action. The association between HPV and breast cancer warrants further rigorous investigation to clarify its clinical and preventive implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c5a2843a4cdacf97e3a7acb855e5ff458bbd2b" target='_blank'>
              HPV, APOBEC3B, and the origins of breast cancer: a narrative review and perspectives on novel mechanisms
              </a>
            </td>
          <td>
            Zhiyong Liu, Ran Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract PDCD4 is a multifunctional RNA-binding protein that has tumour suppressor function. To more fully understand how dysregulation of this protein contributes to carcinogenesis, we have carried out a comprehensive analysis of the role of PDCD4 in RNA metabolism in untransformed epithelial cells. We show that PDCD4 predominantly localises in the nucleus, where it interacts with proteins involved in a range of different RNA metabolic processes. We find that PDCD4 knockdown is associated with significant changes in either the expression or splicing of a number of transcripts, although it appears to have an indirect role in splicing. We identified the RNA targets of PDCD4 using iCLIP and observed an enrichment in binding to transcripts encoding cell adhesion and structural proteins. Consistent with these data, we show that PDCD4 acts as a general regulator of cell adhesion, which in a tumour setting would increase the metastatic potential of cells, and demonstrate that the nuclear localisation of PDCD4 is crucial in this process. Overall, the information obtained in untransformed cells provides a new perspective for the role of PDCD4 as a tumour suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8430c006a64f398091860452b4a00729178986be" target='_blank'>
              Functional characterisation of tumour suppressor PDCD4 reveals previously undisclosed role in the control of cell adhesion
              </a>
            </td>
          <td>
            Veronica Dezi, Robert F. Harvey, Tom Smith, Cameron H Cole, Mariavittoria Pizzinga, M. Stoneley, Gavin D Garland, Emilie Horvilleur, Lajos Kalmár, Ritwick Sawarkar, Kathryn S Lilley, Anne E Willis
          </td>
          <td>2026-01-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction MECOM rearrangements are frequently observed in myeloid neoplasms and associated with poor prognosis. Among the genomic alterations leading to MECOM rearrangements, t(2;3)(p13~p25;q26.2) accounts for approximately 13% of reported cases. However, the precise DNA breakpoints of this translocation have not been previously reported, nor has the mechanism by which it alters MECOM expression been fully elucidated. In this report, we describe two cases with myelodysplastic syndromes (MDS) and t(2;3)(p23;q26.2). Our genomic characterization of these two t(2;3) translocations provided insights into the molecular mechanism of MECOM activation. Case Presentation Case 1 is a 44-year-old female presented with new anemia and thrombocytopenia. She was treated with azacitidine. After two allogeneic stem cell transplants, her disease relapsed with rapid progression to acute myeloid leukemia (AML). Patient passed away one year after progression to AML and eight years after initial diagnosis. Case 2 is a 75-year-old female who was incidentally found to have macrocytic anemia with rare circulating blasts. She remained asymptomatic from anemia and did not require transfusions or treatment. Her disease progressed to MDS with excess blasts three years later. Patient was treated with azacitidine. Fifteen months later, her disease further progressed to AML. She passed away five months later and four and a half years after initial diagnosis. DNA sequencing analysis of these two cases revealed that the t(2;3)(p23;q26.2) breakpoints were within the regulatory regions of ZFP36L2 and THADA on chromosome 2 and the proximity of MECOM on chromosome 3, creating a novel regulatory configuration for MECOM. Notably, the translocation breakpoints differed by 270 kb on chromosome 2 and 93 kb on chromosome 3, resulting in distinct translocated regulatory elements with varying sizes and proximities to MECOM. These structural differences may influence the level of MECOM upregulation and contribute to variation in disease severity and progression. Conclusion Our findings highlighted that, despite cytogenetic similarity, different t(2;3) translocations may exert distinct regulatory effects depending on the precise breakpoint locations. Thus, molecular characterization of MECOM rearrangements is critical for understanding disease pathogenesis and prognosis in myeloid neoplasms and may lead to the development of novel treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad833dbfa2155f8b4eb617717741f0ce48e8ac9" target='_blank'>
              Molecular Characterization of MECOM Rearrangements in Two Cases with Myelodysplastic Syndrome and t(2;3)(p23;q26.2).
              </a>
            </td>
          <td>
            Zhongxia Qi, Sonam Prakash, Jingwei Yu
          </td>
          <td>2025-12-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The development of biomarkers for population screening, early cancer detection, monitoring, and recurrence surveillance offers substantial potential to improve patient outcomes and save lives. Nullomers are short k-mers that are absent from a human genome, and neomers are the subset of nullomers that emerge recurrently due to somatic mutations during cancer development. Here, we have developed neomerDB, a database that encompasses a catalogue of neomers across cancer types and organs. We examined 10 000 whole exome sequencing and 2658 whole genome sequencing tumour-matched samples and identified the set of neomers associated with each cancer type and organ. We also analysed 76 215 whole genomes and 730 947 whole exomes of individuals from diverse ancestries, from which we removed nullomers and neomers that can arise due to germline variants in the population. Finally, we conducted a case study demonstrating that neomers can be utilized to detect glioblastoma from liquid biopsy samples (n = 38), utilizing cell-free DNA and cell-free RNA, achieving a Receiver Operating Characteristic - Area Under the Curve score of 0.98 and a precision-recall score of 0.99. neomerDB is a user-friendly database that enables advanced searches, provides interactive visualizations, and download options for neomer biomarkers. neomerDB is publicly available at https://neomerDB.com/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d61e31009fa3725049e1e6c9f385fb155746c12" target='_blank'>
              neomerDB: a comprehensive database of neomer biomarkers in cancer
              </a>
            </td>
          <td>
            Kimonas Provatas, Candace S. Y. Chan, Ioannis Kerasiotis, Eleftherios Bochalis, Akshatha Nayak, Brad E Zacharia, Georgios A. Pavlopoulos, Wei Li, IIias Georgakopoulos-Soares
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2f190e8d71a0ebf23cad42633c41105de560d7" target='_blank'>
              Chromosome length is constrained by spindle scaling to ensure faithful mitosis in mammals.
              </a>
            </td>
          <td>
            Yu-Long Zhao, Yi-Ming Zhao, Yi-Fang Yan, Ning Yang, Si-Nan Ma, Rui-Jia Wang, Guihai Feng, Zhi-Kun Li, Wei Li, Li-Bin Wang
          </td>
          <td>2026-01-15</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A significant cancerous growth with a high global incidence and death rate, cancer of the lungs—particularly non-small cell lung cancer (NSCLC), which makes up 80% to 85% of cases—poses a serious threat to public health. Drug resistance, damage to cells that are healthy, and limited effectiveness in advanced stages represent a few of the drawbacks of conventional treatments such radiation and targeted therapy. CRISPR-Cas9 gene editing technology, consisting of key elements like Cas9 protein and guide RNA (sgRNA), enables precise DNA cleavage and editing, offering new prospects for NSCLC treatment with advantages such as strong targeting and high efficiency. The procedures for employing CRISPR-Cas9 for curing NSCLC are explained in detail in this publication, including the use of delivery vectors optimized to overcome lung physiological barriers, treatment strategies like oncogene knockout using Cas9 nickases and adjuvant CAR-T therapy through PD-1 gene knockout, and its application in detection and diagnosis for early screening. Additionally, it analyzes the technical advantages of CRISPR-Cas9 compared to traditional therapies and discusses the challenges and solutions in clinical transformation, with the intention of offering a theoretical guide for the accurate management of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3f6e41f0ec4c718c50a0a00f99bdd6b6402f217" target='_blank'>
              Applying CRISPR-Cas9 technologies for lung cancer treatment
              </a>
            </td>
          <td>
            Shimiao Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bd31d7ad6a7db4d77a69357d29f94ed64bdd6e" target='_blank'>
              DNA polymerase kappa stabilized by Ptbp2 interacts with MRE11 and promotes genomic instability in leukemia.
              </a>
            </td>
          <td>
            Shristi Lama, Bibhudev Barik, Sajitha Is, Tannistha Sarkar, Sayantan Chanda, Monalisa Behera, Payel Guha, Subhankar Priyadarshi Behera, S. Biswas, Sonali Mohapatra, G. Biswas, Soumen Chakraborty
          </td>
          <td>2026-02-10</td>
          <td>Cell death discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Failures in chromosome segregation during meiosis result in aneuploid gametes and are the leading cause of miscarriage. The position and number of crossovers, genetic exchanges between homologous chromosomes, are essential to their accurate disjunction in meiosis. Previous research analyzing nondisjunction of acro- and telocentric chromosomes in human and Drosophila has identified altered positioning and number of crossovers that differs between meiosis I and meiosis II nondisjunction. Limited data from positioning in chromosomes that segregated normally in metacentric trisomies has suggested discrepancies between the behavior of these chromosomes and the acro/telocentrics in nondisjunction, which warrants further investigation. Here, we employ whole-genome sequencing to study spontaneous meiotic nondisjunction of the metacentric chromosome 2 in Drosophila melanogaster. Several patterns of recombination may differentially impact telo- versus metacentric chromosomes: lack of a crossover, distal crossovers, and proximal crossovers. We find that unlike meiotic nondisjunction of the Drosophila X chromosome, nondisjunction of chromosome 2 is not associated with dramatic changes in crossover landscape, but is associated with reduced recombination. Differences in the proportions of NDJ events with altered recombination patterns between chromosomes X and 2 suggest that abnormal crossover positions disparately affect chromosomes with different shapes. Taken together, these findings highlight that the underlying triggers of meiotic nondisjunction are chromosome-specific.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8208b29f245fc09b23a7d8f70794f7fb4b094e6" target='_blank'>
              Chromosome-specific differences in the recombination landscape of spontaneous meiotic nondisjunction
              </a>
            </td>
          <td>
            Carolyn A. Turcotte, J. Sekelsky
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Gene fusions are critical oncogenic drivers and therapeutic targets in diverse cancers. Long-read ribonucleic acid sequencing (RNA-seq) offers an unprecedented opportunity to resolve the full-length structure of fusion isoforms, but its high intrinsic error rates pose significant challenges to the precise identification of true fusion events. Here, we developed GFSeeker, an innovative splicing-graph-based computational framework for accurate gene fusion detection from long-read RNA-seq. GFSeeker employs a unique pipeline based on a splicing graph reference and a dual re-alignment validation to effectively overcome data noise from high error rates. Benchmarking across simulated, non-tumor, and cancer cell line datasets demonstrated GFSeeker’s state-of-the-art performance, achieving 6%–15% higher F1 score compared to existing methods. Notably, GFSeeker successfully identified the known fusion event, MATN2–POP1, in the MCF-7 cancer cell line, missed by other tools, highlighting its superior sensitivity in resolving complex fusion events. These results validate GFSeeker as a powerful and reliable tool for gene fusion discovery, heralding its significant potential to advance cancer research and precision diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86fa854d23dc7ec4d6c96653cc3d0b39ac6f3a32" target='_blank'>
              GFSeeker: a splicing-graph-based approach for accurate gene fusion detection from long-read RNA sequencing data
              </a>
            </td>
          <td>
            Bingyan Wang, Heng Hu, Runtian Gao, Guohua Wang, Tao Jiang
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Head and neck squamous cell carcinoma remains a highly aggressive malignancy with limited predictive biomarkers for prognosis and radiotherapy response. Increasing evidence indicates that pseudogenes are functionally active regulators of cancer biology, yet their clinical relevance in HNSCC is poorly defined. Methods: Using transcriptomic and clinical data from The Cancer Genome Atlas, we analyzed the expression and clinical significance of two pseudogenes, ANXA2P2 and PA2G4P4, in HNSCC. Associations with clinicopathological features, HPV status, tumor subtypes, survival, genomic instability, radiotherapy response, and immune landscape were assessed using bioinformatic tools. Results: Both pseudogenes were significantly upregulated in HNSCC compared to normal tissues. Higher expression levels correlated with adverse clinicopathological features, increased tumor proliferation and wound-healing capacity, and unfavorable TCGA molecular subtypes. High ANXA2P2 and PA2G4P4 expression was associated with reduced overall survival, while their combined low-expression signature identified patients with significantly improved overall and disease-free survival. Notably, lower expression of both pseudogenes was observed in patients responding to radiotherapy, whereas higher expression was linked to genomic instability parameters and enrichment of oncogenic pathways, including MYC, PI3K/AKT/mTOR, cell cycle regulation, and DNA repair. ANXA2P2 expression differed significantly by HPV status, showing reduced levels in HPV-positive tumors. Furthermore, pseudogene expression stratified distinct immune profiles, including immune subtypes, stromal and immune scores, and specific immune cell populations. Conclusions: ANXA2P2 and PA2G4P4 are clinically relevant pseudogenes associated with tumor aggressiveness, immune modulation, and radiotherapy response in HNSCC. These findings support their potential utility as prognostic and predictive biomarkers and provide a rationale for further functional validation in experimental models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33872a456524f0aadce25f6d9d2640840e11a5a0" target='_blank'>
              ANXA2P2 and PA2G4P4 Pseudogenes Are Associated with the Response to Ionizing Radiation and Could Be Used as Potential Biomarkers: In Silico Study
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Białas, Kacper Kamiński, Maria Dziuba, Małgorzata Czernecka, Aleksandra Leszczyńska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Potter, Klaudia Dudek, Nina Grzejda, Karina Tylkowska, Anna Zapłata, Marlena Janiczek-Polewska, Paulina Gieremek, Katarzyna Regulska, Patrycja Mantaj, Anna Florczak-Substyk, Anna Przybyła, U. Kazimierczak, E. Leporowska, Zefiryn Cybulski, Beata J Stanisz, Anna Teresiak
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Schematic view of the development of CK-AML driven by the TP53 absence. The occurrence of the first, often dominant negative TP53 mutation is quickly followed by the loss of the second TP53 allele and numerous further chromosomal aberrations. The occurrence of the first, often dominant negative TP53 mutation is quickly followed by the loss of the second TP53 allele and numerous further chromosomal aberrations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f17929b09ba88a3438823b48a185f995e9dcd19" target='_blank'>
              Etiology of TP53 mutated complex karyotype acute myeloid leukemia
              </a>
            </td>
          <td>
            Anna Fedenko, H. Czapinska, A. Krämer, F. Stölzel, T. Bochtler, Matthias Bochtler
          </td>
          <td>2025-12-19</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Fibrolamellar Carcinoma (FLC) is a liver cancer of adolescents and young adults driven by the fusion of DNAJB1, a heat shock protein and PRKACA, the catalytic subunit of protein kinase A (PKA). It is characterized by increased PKA activity leading to widespread changes. A profile of FLC at genomics, transcriptomics, methylomics and proteomics in a multi-omic study that studied 1,412 tumor-normal paired liver samples from patients with FLC, Hepatoblastoma, Intrahepatic Cholangiocarcinoma, and Hepatocellular Carcinoma, yielded the FLC Data Base (FLC-DB). To facilitate the visualization of the multiomics, we developed a user-friendly web interface for the FLC-DB. It uses functions from the R-library OmicsKit and enables interactive exploration for non-bioinformaticians to retrieve information and generate publication-ready plots. It comprises 3 main tabs. First, Documentation, which describes the samples and datasets, the transcriptomic FLC Signature, and a reduced set of 35 genes with strong and recurrent upregulation across all samples, selected for clinical purposes. Second, the Transcriptomics tab, where users can interactively explore the distribution of gene counts across liver cancers, comparing normal samples against tumors and tumor subtypes. This has the option to directly perform Differential Expression Analysis using subsets that can be dynamically defined by the user in the web interface. Third, the Genomics tab, which allows exploration of Single Nucleotide Variants (SNV) from Whole-Genome and Whole-Exome Sequencing (WGS/WES) and download of the SNVs annotations and perform downstream analyses. This allows exploring questions such as: Is a transcript consistently upregulated across FLC studies? Are there recurrent SNVs in FLC? What gene signatures are shared between FLC and HBL? The FLC-DB web interface facilitates exploration of complex multi-omics data, broadening its access to non-bioinformatic researchers, patients and the community. It is currently available at https://fibrolamellardatabase.org/.



 Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon. The fibrolamellar carcinoma database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6039ebc7493e6eddced2c3191b76583f0aea8b24" target='_blank'>
              Abstract B031: The fibrolamellar carcinoma database
              </a>
            </td>
          <td>
            Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cell characteristics are determined by gene expression, influenced by genomic, epigenetic, and transcriptional modifications. Genomic rearrangements and transcriptional splicing can result in the formation of fusion genes. BCR-ABL1 is an established fusion gene employed as a biomarker in leukemia. A single gene can amalgamate with several other genes and may impact cellular fate. Ethnicity-specific variants of fusion genes have been identified, such as the TMPRSS2-ERG variation observed in prostate malignancies among African-American, Caucasian, and Japanese populations in research studies. Next-generation sequencing has provided a new method for predicting genomic and transcriptomic changes. We aim to identify fusion genes in the Indian population using cancer samples to enhance diagnostic outcomes. This study performed a meta-analysis of tumor-specific RNA sequencing data for liver, tongue, and ovarian cancers, which are available online. It identified known fusion genes, including TRO-MAGED2, KRT14-S100A9, RNASE10-CD38, ACTN4-ACTN1, RGPD1-RANBP2, CTSC-RAB38, C15orf57-CBX3, AMBRA1-CKAP5, ATP2B3-ATP2B4, CNKSR3-IPCEF1, E2F4-RPL14, and MZT2A-MZT2B, along with 101 novel fusion genes. Novel fusion genes GABRP_SCGB3A2 and WWOX_FUT1 were identified in all three tumor tissues. GABRP acts as a tumor inducer, whereas SCGB3A2 functions as a tumor suppressor. WWOX2 serves as a tumor suppressor, whereas FUT1 functions as a promoter of malignancy. The interplay between tumor inducers and suppressors may serve as a survival mechanism for cancer cells, a subject that has received limited research attention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4406b3ded6f3886093045dfec09d59265a20dd7e" target='_blank'>
              RNAseq-based meta-analyses revealed tumor suppressor-inducer fusion events in liver, oral, and ovarian cancer in the Indian population: a cancer cell surviving mechanism.
              </a>
            </td>
          <td>
            Rahul Yadav, Hafsa Khan, Poonam Singh, Pramod Kumar, Dinesh Kumar Singhal
          </td>
          <td>2026-02-09</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63700b1580a28e6e919137fb8581425a7fd29e67" target='_blank'>
              Digging deeper into cancer resistance in the extraordinarily long-lived naked mole-rat
              </a>
            </td>
          <td>
            Tanvi T Patel, Vince G Amoroso, Kaitlyn N. Lewis Hardell, Vikram Narayan, Martha A. Delaney, Rochelle Buffenstein
          </td>
          <td>2026-01-27</td>
          <td>Mammalian Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Recurrence and metastasis are the main causes of death in ovarian cancer (OC). Long non-coding RNAs (lncRNAs) are considered as good prognostic models and potential therapeutic targets for cancer patients because of their easy detection and strong correlation. Our study identifies an OC-associated lncRNA with tumor progression and therapeutic implications. It’s found that lncRNA AC093895.1 is highly expressed in OC tissues and correlated with poor prognosis. AC093895.1 has a potentiating effect during the progression and metastasis of ovarian cancer. The effects of AC093895.1 on ovarian cancer cells are miR-1253 dependent. Results showed that by interacting with tumor-suppressive gene miR-1253 as competing endogenous RNA (ceRNAs), AC093895.1 significantly upregulated the downstream gene SOX4 of AC093895.1/ miR-1253 axis, leading to tumor metastasis. In addition, chromatin immunoprecipitation (ChIP) results further confirmed that SOX4 could bind to the AC093895.1 promoter, forming a positive feedback loop SOX4/AC093895.1/miR-1253/SOX4. Therapeutic strategy to break the loop through AC093895.1 knockdown exhibited attenuated OC growth and metastasis in vivo both in SK-OV-3 subcutaneous model and pulmonary metastatic model. Our study unveiled the potentiating effects of SOX4/AC093895.1/miR-1253/SOX4 on ovarian cancer cell survival, migration, and invasion. AC093895.1 may be a promising patient prognostic biomarker and therapeutic candidate. Created with BioRender.com. Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7712cd5a20808194d5452cca2e781e0718918fa" target='_blank'>
              The long noncoding RNA AC093895.1 promotes ovarian cancer formation and metastasis through a positive feedback network dependent on the transcription factor SOX4
              </a>
            </td>
          <td>
            Bin Huang, Honglin An, Yiman Qiu, Zhuona Ni, Liming Chen, Jiahui Lin, Shihan Lin, Han Wu, Hanqi Zhu, Yueting Fan, Shu Jiang, Yixin Chen, Wenqi Yu, Jiumao Lin
          </td>
          <td>2026-02-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost-effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry-based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN-associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS-STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of "Micronuclear Epigenetics" is highlighted, with its potential application in high-throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure-induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability.
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75cc8466b22f0adff57986e1c7ec41afd7e67c91" target='_blank'>
              Targeting NHEJ activates STING signaling through MYC degradation to boost antitumor immunity in SCLC.
              </a>
            </td>
          <td>
            Subhamoy Chakraborty, Andrew Elliott, Utsav Sen, Charles Coleman, V. Jethalia, Kedwin Ventura, Chih-Wei Fan, R. Sritharan, Avisek Banerjee, Yazhini Mahendravarman, A. Vanderwalde, B. Halmos, Elisa de Stanchina, H. Taniguchi, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2026-02-10</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer ranks third in global incidence and second in cancer mortality. Patient-derived models are irreplaceable for studying tumor biology. We established a human epithelial cell line from a rectal adenocarcinoma overexpressing cancer stem cell marker ALDH1A1, and we investigated the effect of ALDH1A1 knockout on tumor cell traits. The cell line and its CRISPR-Cas9 ALDH1A1 knockouts were characterized by genomic and cytogenetic methods (CNV, WES, RNAseq, karyotype), in vitro (proliferation, response to chemotherapy, migration, invasion, apoptosis), and in vivo methods. We identified the landscape of somatic mutations and copy number alterations in the original tumor and the derived cell line. Genetic attenuation of ALDH1A1 was characterized by an increase in migratory potential and extensive metastatic ability, accompanied by reduced growth of subcutaneous xenografts and alterations in gene expression associated with inhibited proliferation and promoted invasion and metastasis, ultimately resulting in dysregulation of the Wnt signaling pathway. Increased metastatic potential was also confirmed in HT-29 cells after ALDH1A1 genetic attenuation. CRISPR-Cas9-mediated editing led to functional, cellular, and molecular changes confirming the role of ALDH1A1 in colorectal cancer carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b72d554a35d6005b3610cd9e7621ca3ab6650f1a" target='_blank'>
              Genetic attenuation of ALDH1A1 increases metastatic potential and aggressiveness in colorectal cancer.
              </a>
            </td>
          <td>
            M. Poturnajova, Z. Kozovska, Ondrej Pos, Kristína Pavlov, Sachin Gulati, Peter Makovický, Kristina Jakic, M. Burikova, Eva Sedlackova, Barbora Svitková, S. Tyčiaková, V. Bystrý, Nicolas Blavet, Boris Tichý, Matej Hrnciar, J. Budiš, Miroslav Tomas, Peter Dubovan, Georgina Kolnikova, Veronika Repaska, Nikoleta Mojzesova, E. Zomborská, D. Pindak, Michal Mego, T. Szemeš, M. Matúšková
          </td>
          <td>2026-02-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="As a tumor suppressor gene, CCDC6 encodes a coiled-coil domain-containing protein that is ubiquitously expressed and involved in crucial cellular processes such as DNA damage response and apoptosis, although its precise mechanisms remain elusive. Initially identified as part of a fusion gene, CCDC6 can form fusion genes with a variety of proto-oncogenes, including both kinase- and non-kinase-coding genes, thereby facilitating oncogenesis. Alterations in CCDC6 expression across various cancers underscore its intricate role and potential influence on the efficacy of anticancer therapies. Recent findings have demonstrated that CCDC6 can undergo liquid-liquid phase separation (LLPS) and facilitate the LLPS of its associated fusion proteins, providing new perspectives on its functional characterization and potential therapeutic implications in related diseases. We present a comprehensive overview of CCDC6, encompassing its protein characteristics and physiological and genomic aspects. Furthermore, we explored the association between CCDC6 alterations and carcinogenesis, as well as their implications for therapeutic interventions. The objective of this review is to furnish the medicinal community with current information and valuable insights pertaining to diseases associated with CCDC6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872145069c4ef81fa1ace0d713f50e945a6721cc" target='_blank'>
              The multiple roles of the disordered protein CCDC6 in cancer development.
              </a>
            </td>
          <td>
            Guifeng Wei, Yiji Chen, Yichao Kong, Donglai Li, Yang Wang, Ting Qiu, Xiabin Chen
          </td>
          <td>2026-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Understanding DNA double-strand break (DSB) repair is crucial for the development of targeted anticancer therapies, yet the roles of many genes remain unclear. Recent studies show that disruption of known DSB repair genes can alter the sequence-specific distribution of mutations arising after DSB repair, suggesting that genome-wide perturbation screens could be leveraged to identify new DSB genes leading to distinct deviations from the expected wild-type distribution. Given the challenges of designing controls for a genome-wide screen, we explore the high gene throughput to forgo the use of traditional controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair outcomes. We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation impact on mutational spectra without controls by measuring deviations from the central tendency considering the distribution of all spectra. We show that MUSICiAn effectively estimates pseudo-controls for the Repair-seq screen, yielding 476 genes and 60 nontargeting controls. We further apply MUSICiAn to the first genome-wide screen of 18 406 genes with mutational spectra readout, MUSIC, reporting that MUSICiAn successfully recovers known DSB repair genes, highlights the spliceosome as a lesser-appreciated player, and reveals candidates for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a1d3962960ca885e8c3efafcd749c2130bf3c7" target='_blank'>
              MUSICiAn: genome-wide identification of genes involved in DNA repair via control-free mutational spectra analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2026-01-06</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer keeps posing a major threat to human health all over the word. CRISPR technology is an emerging gene editing technique, which offers new ideas for cancer treatment. Compared with traditional therapies including surgery, radiotherapy, and chemotherapy, it has many advantages such as complete cure, high precision, efficiency, and low side effects. The CRISPR system mainly consists of Cas9 and gRNA, of which gRNA helps recognize specific DNA sequences and guides the Cas9 protein to cleavage doublestrand. Subsequently, genetic mutations are produced in the process of repair mechanisms. CRISPR therapy for cancer has two main approaches, that are modifying immune cells and editing the genes of cancer cells. This article focused on CRISPR-based CAR-T technology to enhance their ability to recognize and destroy tumor cells. Another strategy directly targets cancer genomes, disabling oncogenes or restoring the function of tumorsuppressor genes. Despite these promising advances, several limitations of CRISPR-based technology remain including off-target effects, reducing editing efficiency, in vivo immune reactions to Cas proteins. Therefore, CRISPR-Cas9 is a versatile and powerful tool for advancing precision oncology, contributing to future clinical applications of more effective, gene-based cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48846d6dcc32f670edbb00e43e2c38195ad65e24" target='_blank'>
              Cancer Treatment Strategy Depending on CRISPR Technology
              </a>
            </td>
          <td>
            Tiancong Du
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objectives: Although Aurora kinase A (AURKA) expression has been investigated in many cancer types, studies focusing on its role in hepatitis B virus-associated hepatocellular carcinoma (HBV-HCC) are limited. In this study, we examined the activity of AURKA and its substrates (PLK1, P53, and BRCA1) in HBV-HCC and cryptogenic hepatocellular carcinoma (Cr-HCC) cases. Methods: The study groups consisted of HBV-HCC, Cr-HCC, and healthy liver tissue cases. AURKA copy number variation (CNV) was analyzed using molecular methods. AURKA expression was evaluated by molecular and immunohistochemical (IHC) methods. AURKA substrates P53Ser315, PLK1Thr210, and BRCA1 were also analyzed by IHC. Results: There was no increase in AURKA gene copy number among the groups (2−∆∆Ct < 2). AURKA level was significantly increased in both test groups (p < 0.001). At the protein level, AURKA was significantly higher in both cancer groups compared to the control group (p < 0.001). Phospho-P53Ser315 levels were significantly higher in both HBV-HCC and Cr-HCC groups compared to the control group (p = 0.002 and p < 0.001, respectively). Cr-HCC cases also showed significantly higher levels compared to HBV-HCC (p = 0.025). For phospho-PLK1Thr210, Cr-HCC cases showed statistically higher expression compared to both the control group and HBV-HCC cases (p = 0.001).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e170fbdf0949ac51576d078567eb9acab99746e" target='_blank'>
              The Role of Aurora Kinase A in HBV-Associated Hepatocellular Carcinomas: A Molecular and Immunohistochemical Study
              </a>
            </td>
          <td>
            M. Huz, Nese Karadag Soylu, Ahmet Koç, Z. Kucukakcali, Nefsun Danis, O. Ozhan
          </td>
          <td>2026-01-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The post-genomic era has transformed medical genetics, raising renewed debate over the role of medical cytogenetics in clinical practice. High-throughput sequencing and chromosomal microarray technologies now dominate cancer diagnostics, prenatal testing, and rare disease evaluation by enabling rapid detection of gene-level variation, often leading to the perception that cytogenetics is obsolete. However, this view overlooks the unique and complementary strengths of cytogenetic analysis. Although the relationship between cytogenetics and current NGS technologies can be compared to that between forests and trees versus leaves—both of which are necessary for clinical diagnosis—cytogenetic methods uniquely enable direct in situ visualization of chromosomes, allowing detection of large-scale structural and numerical genome alterations at the level of individual cells and cell populations. These system-level features that are frequently invisible or difficult to interpret using sequencing-based approaches alone yet are critical in disease contexts where genome architecture itself carries biological and clinical significance beyond individual genes. This article, therefore, advances a new perspective based on Genome Architecture Theory: that karyotype-level information organizes gene-level function and that many previous gene-centric genetic concepts require reexamination within a unified framework of clinical genomics. Rather than being replaced, cytogenetics is increasingly integrated with sequencing within a unified framework of clinical genomics that combines high-resolution molecular detail with system-level insight into genome organization. Reassessing the role of cytogenetics, therefore, has important implications for medical education, diagnostic strategy, and healthcare policy, as cytogenetics provides the appropriate platform for understanding system-level inheritance through karyotype coding and for advancing molecular medicine from a genome systems perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55a3cccc1e3b9d9b3c2e2530a657ae5830cf723" target='_blank'>
              Conversation with Future Clinical Cytogeneticists: The New Frontiers
              </a>
            </td>
          <td>
            Jing Christine Ye, Rishi Chowdhury, Henry H. Heng
          </td>
          <td>2026-02-12</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Approximately 5–10% of all cancer types are hereditary cancer syndromes, which are caused by pathogenic mutations in cancer susceptibility genes. In this investigation, a hereditary cancer pedigree was collected from a province in southern China, and the proband was a 31-year-old woman with breast cancer. Utilizing blood whole exome sequencing technology and bioinformatics analysis, the sole heterozygous missense mutation in PIBF1 that exhibits trait segregation was identified: PIBF1 (p.R405Q). The pedigree also included two other mutations that may be linked to carcinogenesis: RAD51D (p.K91Ifs*13) and BRCA2 (p.G3134Afs*29). This research concentrated on PIBF1 (p.R405Q) and employed breast cancer as a tumor model. In vitro and in vivo experiments showed that PIBF1-WT suppressed breast cancer cell proliferation, colony formation, invasive ability, and tumorigenesis. However, PIBF1 (p.R405Q) attenuated or inhibited the function of PIBF1-WT. Mechanistically, PIBF1-WT resisted cisplatin-induced DNA damage, significantly down-regulated the expression of γ-H2AX, and affected DNA damage repair, thus exerting a cancer inhibitory function. Interestingly, PIBF1 (p.R405Q) affects protein stability, thereby mitigating or eliminating the inhibitory effect of PIBF1. Furthermore, this family’s polygenic risk factors for cancer are analyzed, and there is speculation about potential synergistic effects between PIBF1 and DNA damage repair genes like BRCA2 and RAD51D. In conclusion, PIBF1 regulates the cell cycle and DNA damage repair, PIBF1(p.R405Q) increases susceptibility to cancer, multiple DNA damage repair gene mutations may synergistically promote cancer progression in this cancer family lineage, and PIBF1(p.R405Q) may be one of the polygenic risk factors for familial hereditary cancer syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cccf48b367531a228006163c115c6d6e4b0e36b" target='_blank'>
              PIBF1 (p.R405Q) germline variant identified in cancer susceptibility family impairs protein stability and function
              </a>
            </td>
          <td>
            Lili Shao, Liang Wen, Qiao Xu, Jianing Zhao, Rongji Yu, Yang Wang, Xinchun Liu, Jinzheng Wu, Shubing Zhang
          </td>
          <td>2025-12-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) has been identified as the causative agent of Merkel cell carcinoma, and its non-coding control region (NCCR) has been demonstrated to play a critical role in regulating viral transcription. While NCCR variants exist, their comparative impact on bidirectional promoter activity remains poorly characterized. The present study conducted an in vitro evaluation of bidirectional transcription levels of five major MCPyV NCCR types (I, IIa-1, IIa-2, IIb, IIc). The NCCRs were subsequently cloned into a bidirectional reporter vector, which expresses green (EGFP, early) and red (RFP, late) fluorescent proteins. Subsequent to transfection into HEK293 cells, promoter activity was quantitatively analyzed via fluorescence imaging and flow cytometry. Bioinformatic analysis revealed high sequence similarity (> 94%) among the five NCCRs and predicted conserved transcription factor binding sites. The results indicated that all the variants exhibited stronger late promoter activity compared with the early promoter activity (p < 0.01). These observations are in alignment with the established biology of MCPyV. However, no statistically significant differences in the early/late transcription ratio or overall fluorescence intensity were observed between the different NCCR types under these conditions. These findings suggest that the core promoter function is conserved among these major NCCR variants in this model system. This study provides a foundational comparison of MCPyV NCCR activity, highlighting the need for further investigation in more physiologically relevant models to understand how NCCR diversity may influence viral pathogenesis in vivo. Moreover, incorporating models of viral genome integration is essential to understand mechanism of MCPyV carcinogenesis and viral-host interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287fc6d54ee53d0d2c193c6a1dedcea18ce71955" target='_blank'>
              In vitro evaluation of bidirectional transcription levels of five types of non-coding control regions of Merkel cell polyomavirus
              </a>
            </td>
          <td>
            Han Mo, Xiaotong Qi, Xuan Wu, Liang Lu, Yong Ai, Xiaohua Tao, Xianfeng Zhou
          </td>
          <td>2025-12-31</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6da8c58eace175b556f1165210d11f554c8a306d" target='_blank'>
              Regulatory Mechanisms of Maternal Imprinting at the Dlk1-Dio3 Domain
              </a>
            </td>
          <td>
            Ariella Weinberg-Shukron, Frances L Dearden, Ana Moreno-Barriga, Lok Ting Nick New, Marco Müller, Rebecca C. Rancourt, C. Edwards, A. Ferguson-Smith
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The conceptual understanding of genetic information has evolved from early philosophical speculation to the molecular precision of contemporary biology. Initial debates over the nature of heredity, including Mendel’s hereditary factors and the longstanding protein versus nucleic acid controversy, underscored the difficulty of identifying the true substrate of inheritance. Subsequent discoveries, including reverse transcription, protein-based infectivity (prions), transposable elements, and the regulatory functions of non-coding RNAs, revealed molecular processes that operate at the boundaries of, or alongside, Crick’s original formulation of the Central Dogma of Molecular Biology. Importantly, these findings do not violate the directional rules of information transfer defined by the Central Dogma (DNA → RNA → protein), but instead reshape how, when, and under what constraints these canonical flows are executed in living systems. Epigenetic and epigenetic-like mechanisms, including DNA methylation, histone modifications, chromatin topology, non-canonical DNA conformations, and cytoplasmic inheritance, introduce regulatory layers that modulate information flow without constituting independent information matrices. In parallel, genome innovation, through de novo gene birth, and genome erosion, through pseudogenization, demonstrate that the repertoire of DNA → RNA → protein pathways is itself evolutionarily dynamic. This narrative integrative review reconstructs the historical milestones that culminated in the Central Dogma and synthesizes subsequent discoveries that expand its functional realization. By revisiting the Central Dogma through an extended, holistic lens, this article argues that DNA, RNA, and proteins function not only as carriers of genetic information, but also as active participants in its regulation, contextualization, and evolutionary diversification, without departing from the core directional principles originally articulated by Crick. For reader convenience, a dedicated section entitled “Abbreviations and Key Molecular Terms” is provided at the end of the manuscript to facilitate navigation and interdisciplinary accessibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41f27beaafe5b4b7080189a3f9a01074f63b64bb" target='_blank'>
              Discovering, Integrating, and Reinterpreting the Molecular Logic of Life: From Classical Theories of Heredity to an Extended Functional Perspective on the Central Dogma
              </a>
            </td>
          <td>
            A. C. Grădinaru
          </td>
          <td>2026-01-01</td>
          <td>Life</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e3dfbd14bf43361d23b6ad2843724145ac656" target='_blank'>
              Synthetic lethality and DNA damage response targeting in cancer stem cells: a comprehensive review.
              </a>
            </td>
          <td>
            Usamah Sayed, Waleed K. Abdulsahib, Ihsan Khudhair Jasim, H. Malathi, Priya Priyadarshini Nayak, D. Alex Anand, Gunjan Mukherjee, Aashna Sinha, Sadridin Eshkaraev
          </td>
          <td>2026-02-13</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Transposable elements (TEs) are mobile genetic elements that are present in great numbers in the majority of eukaryotic genomes. They are major drivers of genome evolution as they can facilitate chromosome rearrangements, provide mechanisms for genomic shuffling, and contribute to genome expansion, thereby altering genome architecture. Additionally, TEs can also modify gene expression by disrupting regulatory sequences, impairing genes, and promoting the emergence of new sequences. Despite their abundance, identifying TEs is challenging and time-consuming due to their extreme diversity in DNA sequence. Many TE families are ancient, and most of their sequences have become inactive due to accumulated mutations and fragmentation. Consequently, different copies of the same TE can differ greatly from each other, which makes identifying decayed copies particularly difficult. In this work, we employ transformer architecture for TE classification. TEclass2 is an integrated classifier that rapidly predicts TE orders and superfamilies using models built on this advanced machine learning approach. The software is available through a web interface, allowing users to classify sequences into sixteen superfamilies according to the Wicker classification system, or alternatively, users can download the source code to train and build custom classification models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf277b82c7226a65ad3cdcbe7a255c99eb31947" target='_blank'>
              Transformer-Based Classification of Transposable Element Consensus Sequences with TEclass2
              </a>
            </td>
          <td>
            Lucas Bickmann, Matias Rodriguez, Xiaoyi Jiang, Wojciech Makałowski
          </td>
          <td>2025-12-29</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Our genome is duplicated during every round of cell division through the process of DNA replication, but this fundamental process is subjected to various stresses arising from endogenous or exogenous sources. Thus, studying replication dynamics is crucial for understanding the mechanisms underlying genome duplication in physiological and replication stress conditions. Earlier, radioisotope-based autoradiography and density-labeling methods were used to study replication dynamics, which were limited in spatial resolution, representing only average estimates from many DNA samples. Here, we describe a DNA fiber assay that utilizes different thymidine analog incorporation, like 5-chloro-2’-deoxyuridine (CldU) and 5-iodo-2’-deoxyuridine (IdU), into replicating DNA. Such labeled DNA can be stretched and fixed on silanized glass slides, which are denatured with mild acidic treatment to expose the labeled nascent DNA. This DNA can then be visualized by using primary antibodies against CldU and IdU, followed by fluorophore-conjugated secondary antibodies, and observing them using a fluorescence microscope. The DNA fiber assay allows the visualization of individually replicating DNA at a single-molecular resolution and is highly quantitative, high-throughput, and easily reproducible. This technique offers insights into different replication parameters, like rate of DNA synthesis, extent of reversed fork protection, restart of stalled forks, and fork asymmetry under untreated or replication stress conditions at a single-molecule level. Key features • Single-molecule resolution of DNA replication dynamics. • Diverse replication parameters can be quantified with variations of the labeling protocol. • Easily reproducible across different cell lines in any lab with a basic molecular biology setup. • Effective in studying the effects of different genotoxic treatments on replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8515bc5d0becb61ee3d4dcdb4139998b45e857f" target='_blank'>
              A Quantitative DNA Fiber Assay to Monitor Replication Fork Progression, Protection, and Restart
              </a>
            </td>
          <td>
            Debanjali Bhattacharya, G. Nagaraju
          </td>
          <td>2026-02-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91fe51488198f3ff4051c25194da67b2c9762e1" target='_blank'>
              Human Accelerated Regions: Functional Studies and Methodological Approaches (A Review)
              </a>
            </td>
          <td>
            A. Knyazeva, T. A. Shnaider
          </td>
          <td>2026-01-30</td>
          <td>Cell and Tissue Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, Hyeonjin Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, Daniel Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Li-Fraumeni Syndrome (LFS) is linked to mutations in the p53 gene and is characterized by autosomal dominant early-onset familial cancer susceptibility. The p53R181C mutation is one of the earliest described mutations associated with early-onset familial hereditary breast cancer. Highly stable mutant p53 protein is often a prerequisite for tumor initiation and progression, but the pathways leading to p53R181C accumulation and carcinogenesis are not understood. Here, we found that p53 R181C mutation decreases the interaction of p53 with the Sirt1 deacetylase, resulting in increased p53 K382 acetylation and inhibition of MDM2-mediated ubiquitination and degradation of p53. Moreover, the R181C mutation leads to “loss-of-function” of transcriptional regulating tumor suppressor genes like p21, bax, and PUMA as well as “gain-of-function” of transcriptional regulating tumor promoting genes of PIK3CA, SHC1, SRC and PAK4. This dysregulation promotes genomic instability, enabling cancer cells to evade cell cycle control, senescence and apoptosis, thus facilitating tumor development. Our findings unraveled the mechanism by which the p53R181C mutant protein accumulates and facilitates tumor development. The p53 R181C mutation accumulates through impaired deacetylation by Sirt1 and altered ubiquitination, which reveals the role of p53 R181C mutation in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee27937c98ec8fa67a8f44824ed0213a66d8c39" target='_blank'>
              The p53 R181C mutation accumulates through impaired deacetylation by Sirt1 and facilitates tumor development
              </a>
            </td>
          <td>
            Erli Wang, Liang Cao, Wendong Guo, Xiang Dong, Wanying Zhang, Xiaowen Zhang, Zhuo Wang, Xia Chen, Aicun Li, Yu Zou, Shikun Li, Kedian Chen, Han Su, Sunrun Cao, Ning Du, Pingyuan Wang, Paul M Hwang, Xiangsheng Liu, Xiaoyu Song, Hongde Xu, Jingwei Liu, Liu Cao
          </td>
          <td>2026-01-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683398225f12a992fd68b945ba84e877fc29b6de" target='_blank'>
              Interpreting cancer genetics through a two-step "evolutionary cascade hypothesis": bridging neutral and selective perspectives.
              </a>
            </td>
          <td>
            A. Ottaiano, M. Santorsola, Francesco Sabbatino, Roberto Sirica, Francesco Caraglia, Anna Ceccarelli, Vincenza Granata, Ines Simeone, S. Zappavigna, M. Berretta, Giovanni Savarese, Michele Caraglia
          </td>
          <td>2026-02-13</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is prevalent across human cancers, yet its specific contributions to tumor evolution remain poorly understood. In this issue, Cross, Nowinski, Cresswell, Mossner, and colleagues present a longitudinal analysis of 755 samples from 167 patients with colorectal cancer, uncovering the temporal dynamics of karyotype evolution in this tumor type. See related article by Cross et al., p. 218.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c1772974604d4bdebde97d913a898acbd365b8" target='_blank'>
              Holding the Line: Negative Selection Maintains the Karyotype in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Hajime Okada, Rachel Slutsky, U. Ben-David
          </td>
          <td>2026-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oncogenic mutations of SF3B1 are common in myeloid cancers, chronic lymphocytic leukemia (CLL) and select solid tumors. Their mechanistic basis for promoting oncogenesis has been investigated in detail, with the stereotyped missplicing of mRNA protein coding sequences most intensively studied. These changes, in genes such as MAP3K7, BRD9, and ABCB7, typically lead to loss-of-function, thus contributing to cancer pathogenesis.Here we systematically analyzed the impact of mutant SF3B1 on non-coding regions of mRNA transcripts across disease types, in both cell lines and primary patient specimens. This identified numerous novel and highly reproducible splicing alterations in such regions. Studies of one target gene, DCAF16, revealed multiple complex mutation-induced alterations in its 5' and 3' untranslated regions (5', 3' UTRs). Remarkably, these were mechanistically associated with increased DCAF16 protein levels in SF3B1 mutant cells, representing the first time that oncogenic SF3B1 has been shown to increase levels of a target protein in a gain-of-function manner. DCAF16 is a substrate recognition adapter for the DDB1/CUL4 E3 ubiquitin ligase complex. Novel protein degrader small molecules which co-opt DCAF16 to degrade BRD4 as a neosubstrate demonstrated preferential selectivity for SF3B1 mutant cancers and CLL primary patient specimens due to increased DCAF16 protein levels. In turn, this reveals the therapeutic relevance of mutant SF3B1 dysregulation of transcript untranslated regions and uncovers a novel strategy for the treatment of these important neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc898e0fdc805e68eb2bae19245eee2675983440" target='_blank'>
              Oncogenic SF3B1 mutations alter the splicing of mRNA noncoding regions to induce a novel therapeutic vulnerability.
              </a>
            </td>
          <td>
            Michal Sekrecki, Agata Sekrecka, Rohan Lattupally, Kenny Le, Xiaokang Jin, Chen Mozes, Brendan G. Dwyer, Zhe Zhuang, Bryan A. Romero, J. Pineda, Xunhong Cao, Linh Nguyen, Vicky W Chen, Crystal Zhou, Jared A Wallace, Kailee L Tanaka, C. Tiwari, Austin Gabel, Won Jun Kim, Robert F Stanley, Salima Benbarche, Jaspreet Thind, Abhinaya Muruganandham, T. Y. Zhang, Peter L Greenberg, J. Gotlib, G. Mannis, William Shomali, G. Salmasi, Calvin J Kuo, T. Shanafelt, Irtisha Singh, Daichi Inoue, Finn K. Hansen, N. Gray, Capucine Van Rechem, B. Fakhri, Xiaoyu Zhang, Sydney X. Lu
          </td>
          <td>2026-01-26</td>
          <td>Blood</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="BRCA1 and BRCA2 are highly conserved tumor suppressor genes, which have dispensable roles in maintaining genomic integrity and whose coordinated functions are mainly owed to DNA repair conducted by homologous recombination (HR) mechanisms. BRCA1/2 loss-of-function mutations cause defective repair of DNA double-strands breaks, destabilization of stalled replication forks, and instability of replication stress responses, leading to increased chromosomal instability and extensive genomic restructuring. BRCA1 deficiency is associated, at the molecular level, with alterations in the DNA end resection program, the checkpoint signaling program, and chromatin remodeling; BRCA2 dysfunction is associated with RAD51 loading and filament stabilization, and changes in the bias of cells toward error-prone repair programs. Besides these canonical repair errors, BRCA1/2 mutations have an effect on transcriptional regulation, chromatin structure and cell-cycle control which further promotes mutational gain and altered cell fate choices. A combination of faulty DNA repair, replication stress, and failure of genome maintenance form a permissive environment for oncogenic transformation. Such interactive molecular pathways can be used to explain the significant predisposition to breast, ovarian, and other solid tumors that are seen in subjects that carry pathological BRCA1/2 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5fac4b549ba24ec939a11edbcaa77ad4453e42" target='_blank'>
              The Mechanisms of Tumor Susceptibility Associated with BRCA1/BRCA2 Genetic Mutations
              </a>
            </td>
          <td>
            Zhenan Shi
          </td>
          <td>2026-02-02</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Chordoma is a rare bone tumor that is considered to have originated from the remnants of notochord by the general consensus in the field. Chordoma patients suffer from local invasion and recurrence of the disease, along with high resistance to conventional therapies. Despite the scarcity of the disease, new biomarkers must be discovered to develop targeted therapeutics. Methods This study aimed at analyzing the whole exome sequencing data of a chordoma cell line, CH22, which held more than 3,000 missense variants in the genome. These missense-carrying genes were investigated in terms of their association with chordoma pathogenesis. Results Out of them, neurotrophic tyrosine kinase receptor type 3 (NTRK3) and nuclear factor erythroid 2-related factor 2 were found to be associated with the altered pathways involved in chordoma. Conclusion Further examination through structural modeling, molecular docking, and phylogenetic analyses related to the missense variants of NTRK3 was conducted in the scope of this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2fe9308814058a063733e1cf9abe4023d9644d" target='_blank'>
              Deciphering the Structural and Molecular Effects of a Novel Genetic Mutation by In Silico Analysis in Chordoma Cell Line
              </a>
            </td>
          <td>
            Muhammed Ümeyr Fettah, Nur Ekimci-Gürcan, Emre Yorulmaz, Batuhan Oğuz, A. Köseoğlu, Furkan Ayaz, Esra Aydemir
          </td>
          <td>2025-09-30</td>
          <td>Journal of Neurological Surgery Part B: Skull Base</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9ee68261862574cb9b7d1551d82bf740d99cfd" target='_blank'>
              Elevated mutation in haploid yeast driven by translesion synthesis
              </a>
            </td>
          <td>
            Jacob Fredette-Roman, Denise R. Smith, Sanad B. Omari, Nathaniel P. Sharp
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ANLN is a highly conserved actin-binding protein that plays a critical role in cytokinesis by interacting with key cytoskeletal components such as actin, myosin, and RhoA. Increasing evidence shows that ANLN is aberrantly overexpressed in various cancers, including lung, breast, and liver cancers, and that its elevated expression is associated with enhanced tumor cell proliferation, migration, and invasion. Because of its central involvement in tumorigenesis and disease progression, ANLN has emerged as a promising prognostic biomarker and potential therapeutic target. Recent studies have demonstrated that ANLN contributes to resistance to chemotherapy, targeted therapy, and immunotherapy through multiple molecular mechanisms. This review provides a systematic overview of the physiological functions of ANLN, its roles in cancer initiation and progression, and its regulatory mechanisms in treatment resistance, offering biological insights into precision oncology and potential strategies for overcoming therapeutic resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a40af0d42ab2b0397c3e4bc1fc5ecc54b07af4" target='_blank'>
              The role of ANLN in malignant tumors: pathogenesis, treatment resistance and targeted strategies
              </a>
            </td>
          <td>
            Yixuan Wang, Yang Xiao, Jielin Yu, YuanHua Zou, Zhe Wang, Xianghong Yang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The helix–loop–helix transcription factor ID3 is a critical regulator of tissue development and homeostasis. Aberrations in ID3 are strongly associated with numerous human disease processes including Burkitt’s lymphoma. We previously identified that a single-nucleotide polymorphism (SNP) in ID3 at rs11574 is associated with increased vascular disease burden in three independent cohorts; however, the mechanisms by which this SNP alters ID3 function and impacts vascular cells are unknown. Here, we show that the minor allele of rs11574 specifically disrupts ID3’s ability to bind the E-protein E12. Computational analysis and confirmatory biochemical experiments revealed that rs11574’s effects on ID3:E12 dimerization are dependent upon a key residue within E12’s unique loop domain. Functionally, the disruption of ID3:E12-binding promotes E12 binding to and activation of the p21 promoter. Isogenic human cell lines harboring the rs11574 minor allele exhibited decreased cell proliferation and altered expression of genes involved in proliferation. RNAseq revealed distinct roles for E12 and E47 in transcriptional regulation of vascular smooth muscle cells (VSMCs). Primary human VSMCs from subjects with the rs11574 minor allele exhibited reduced mitogen-stimulated proliferation. Taken together, these results provide the first characterization of altered molecular and cellular functions of this disease-associated SNP in ID3 at rs11574.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7404931dcf35f2a31866627d88ca7b550ec70fa9" target='_blank'>
              A prevalent disease-associated SNP in the human ID3 gene regulates E-protein activity and cellular proliferation
              </a>
            </td>
          <td>
            Christopher A Henderson, Brett P. Ransegnola, James Garmey, Ali Khan, R. Aherrahrou, Michael Purdy, Shijie Wu, Andrea Zhou, Jennifer L Kirby, Michael J Lipinski, Vicki Gordon, Mark Yeager, M. Civelek, Coleen A. McNamara
          </td>
          <td>2026-01-01</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver carcinoma with high lethality. Both of hepatitis B virus (HBV) and Clonorchis sinensis (C. sinensis) are critical infectious contributors to HCC development. However, the inter-tumor heterogeneity and tumor microenvironment (TME) of HCC patients with different infectious background remain largely unknown. We compiled a cohort of 269 primary HCC patients to assess the clinical impact of C. sinensis and HBV infections on patient prognosis. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST-seq) analyses were performed on tumor and adjacent normal samples from C. sinensis-associated HCC (CP), and double-infection HCC (DP) patients. Additionally, we integrated publicly available scRNA-seq and ST-seq datasets from HBV-associated (HP) patients. Immunofluorescence, immunohistochemistry and in vitro experiments were conducted to validate inter-tumor heterogeneity among the three HCC subtypes. C. sinensis infection is significantly associated with poorer prognosis in HCC patients. Multi-omics analyses revealed distinct inter-tumor heterogeneity in epithelial, immune, and stromal compartments across different HCC subtypes. Tumor cells in the DP group exhibited more malignant marker expression, higher copy number variation scores, increased activation of p53 pathway, and worse survival outcomes. Compared with other HCC subtypes, the TME in DP samples was enriched with SPP1+ macrophages, exhausted CD8+ T cells and COL1A1+ fibroblasts. In contrast, the CP and HP groups showed higher proportions of M2-like macrophages and ENPP2+ liver vascular endothelial cells, respectively. These findings decipher the cellular signatures and their interactions within the TME, shedding light on the inter-tumoral heterogeneity driven by different infections, and the development of targeted therapies for infectious HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3fb4e883deefe93c32ac0665db0506da077e80" target='_blank'>
              Integrating single cell- and spatial- resolved transcriptomics unravels the inter-tumor heterogeneity and immunosuppressive landscape in HBV- and Clonorchis sinensis-associated hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiayun Chen, Wenmin Lu, Yanni Lou, Jing Liu, Xiwen Liao, Yunmeng Bai, Guangqing Cheng, Guangzhi Zhu, Ji Feng, Junqi Liu, Zhaoji Liu, Liqun Jia, Jing Zhou, Tao Peng, Guo-Dong Lu, Jigang Wang
          </td>
          <td>2026-01-05</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Primary hepatic angiosarcoma is a very rare and extremely aggressive malignancy that has hadfew therapeutic options and has been closely linked to aristolochic acid which is a strongenvironmental carcinogen commonly found in some herbal medicines. The hallmark mutationalsignature of aristolochic acid is the observed A: T↔ T: A transversion at context-specifictrinucleotide motifs, and this pattern can be used to discriminatively identify AA-induced tumorswith genome-wide analyses. This signature can act as a molecular signature that has the potentialto explain the etiology of cancers, can aid retrospective exposure assessment, as well ascontribute to early diagnosis. Coincidently, the molecular profiling has highlighted newbiomarkers, serine/threonine kinase p53 pathways, aberrations of ERCC1 DNA repair enzyme,and up-regulation of endothelial cell markers. which have collectively enhanced diagnosis andstratification outcomes. These markers have been tested as potential predictive targets andprovide the basis of precision approaches that encompass PARP inhibitors that target DNA-repairdefects, targeted kinase inhibitors, and monoclonal antibodies against angiogenic pathways.Combination therapies that include the targeted with conventional chemotherapy have beenshown to be more effective. Even more, the development of innovative technologies, includingliquid biopsy, single-cell sequencing, and artificial intelligence-mediated searches in mutationalsignatures, further expand personalized medicine of Primary hepatic angiosarcoma and allow anearlier manifestation and precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b9f27b6bcd4ee7287251279079994596ebf2bc" target='_blank'>
              Uncovering Aristolochic Acid–Driven Mutational Signatures and Novel Biomarkers inPrimary Hepatic Angiosarcoma for Targeted Therapeutic Development
              </a>
            </td>
          <td>
            Ramzan Mohammad, Jasvir Kaur, Ruhit Ashraf, Shafkat Hussain Malik, Pursuing B. Pharmacy
          </td>
          <td>2026-01-10</td>
          <td>Pan-African Journal of Health and Psychological Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ab7d87288d1c4f12531bc9f091ec799a59efb" target='_blank'>
              Insights into DNA repeat expansions among 900,000 biobank participants.
              </a>
            </td>
          <td>
            M. Hujoel, R. Handsaker, David Tang, N. Kamitaki, R. E. Mukamel, Simone Rubinacci, P. Palamara, S. Mccarroll, Po-Ru Loh
          </td>
          <td>2026-01-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1158dbe48515eb2e89f44b8a0aabe075c2b9fc" target='_blank'>
              The αβTCR repertoire at scale in the immgenT dataset
              </a>
            </td>
          <td>
            Myriam Croze, Liang Yang, Serge Candéias, I. Magill, O. Casey, Val Piekarsa, Brinda Vijaykumar, V. Giudicelli, Sofia Kossida, D. Zemmour, C. Benoist
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Acquired resistance to targeted and immune therapies severely limits the success of non-small cell lung cancer (NSCLC) treatment. Single-cell sequencing technologies now empower researchers to dissect this resistance at unprecedented resolution, moving beyond the averaging limitations of bulk genomics. This review highlights how single-cell and spatial multiomics approaches reveal key mechanisms of NSCLC resistance, from rare drug-tolerant subpopulations and cellular plasticity to immunosuppressive niches and metabolic adaptation within the TME. We also discuss emerging strategies-such as liquid biopsy and AI-driven data integration-that hold promise for translating these insights into more effective therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bbafdd9d2b462eb3acbb295ea722516ab04fec" target='_blank'>
              Deciphering Treatment Resistance in NSCLC with Single-Cell Sequencing Technology
              </a>
            </td>
          <td>
            Na Li
          </td>
          <td>2026-02-04</td>
          <td>Exploring Science Academic Conference Series</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Centromeres are essential for accurate chromosome segregation and genome stability; with the advent of telomere-to-telomere genome assemblies, they have become central targets of genome-wide studies. Here, we present CentriVision, a modular bioinformatics platform that integrates candidate centromere identification, structural similarity assessment, DNA repeat unit decomposition, and a framework for exploring potential relationships between single-nucleotide conservation and functional features. CentriVision provides a comprehensive suite of analytical tools, including edit-distance dot plots, intra-segment heatmaps, kilobase-scale mini-dot plots, repeat monomer scanning with conserved-site visualization, and satellite DNA expansion–divergence estimation, all of which can be seamlessly integrated with CENH3 chromatin immunoprecipitation sequencing (ChIP-seq) data. When applied to representative plant species, CentriVision achieved high predictive accuracy and revealed diverse organizational patterns. Arabidopsis thaliana centromeres are primarily composed of 178–188-bp repeats interspersed with rarer ∼502-bp variants that exhibit pronounced sequence conservation but only background CENH3-ChIP signal, suggesting that these elements represent pre-centromeric sequences overlooked in earlier studies. Oryza sativa contains two dominant classes of centromeric repeats rather than the single class previously reported. In contrast, Zea mays exhibits strongly biased expansion toward the evolution of a single dominant repeat unit, reflecting a distinct evolutionary strategy of centromere reconstruction, whereas Papaver setigerum displays a notable three-layered nested repeat structure. Integration of repeat sequence divergence with CENH3 binding further revealed lineage-specific evolutionary trajectories of centromere specification. Collectively, these findings advance our understanding of centromere structure and function. CentriVision offers a reproducible, scalable, and user-friendly framework that quantitatively links repeat evolution, structural variation, and functional epigenomics, providing new insights into the architecture and diversification of plant centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c704d00cdc16a661ffc1c6549dc789fd8e569dc" target='_blank'>
              CentriVision: An integrated platform for multiscale centromere analysis in plants
              </a>
            </td>
          <td>
            Mei-Fang Lan, Xi-Yin Wang, Xian-chun Zhang
          </td>
          <td>2025-12-01</td>
          <td>Plant Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a41cb33c7197e06376843089b1b62d0d1ef0e9" target='_blank'>
              Intrachromosomal insertion as a diagnostic challenge: a hidden structural rearrangement causing recurrent duplication and deletion.
              </a>
            </td>
          <td>
            Rie Kawamura, Yui Shichiri, Hideki Suzuki, Yuri Murase, Yuki Naru, T. Hara, Ayana Tsuboi, Hanae Satano, Eiji Sugihara, Hiroki Kurahashi
          </td>
          <td>2026-01-25</td>
          <td>Molecular cytogenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc7d69196c7a92034fbf097685494ed388c2124" target='_blank'>
              A polymer physics model of the interphase cell nucleus for radiobiological simulations.
              </a>
            </td>
          <td>
            Man Zhao, Guoming Huang, Zi Xu, Xiaoyan Huang, Ying Liang
          </td>
          <td>2026-02-09</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be59cd38ce72b5622d35251bdca260d68eabf2d9" target='_blank'>
              pCancer and FOXK2 (Forkhead Box K2): Oncogenic and Tumor-Suppressive Roles of FOXK2 in Cancer.
              </a>
            </td>
          <td>
            Şeyda Akin, İbrahim Ozturk, C. Hepokur
          </td>
          <td>2026-02-03</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This study aims to characterize the mutational profiles of the TP53 and CYP17A1 genes in Senegalese patients with colorectal cancer (CRC) in order to better understand the local molecular mechanisms of tumorigenesis. The analysis included 24 patients with CRC and 24 healthy controls. The sequences of exon 4 of TP53 and the promoter region at exon 1 of CYP17A1 were concatenated, then subjected to genetic structuring, historical demography, and phylogenetic relationship analyses using DnaSP (v5.10), MEGA (v7.014), and Arlequin (v3.1). The results reveal significant genetic differentiation between cancerous and healthy tissues (FST = 0.113, p = 0.009), as well as greater genetic diversity within tumors. Neutrality tests (Tajima's D and Fu's FS) indicate recent demographic expansion in the tumor population, with an excess of rare variants. The multimodal distribution of mismatches and the haplotype network confirm this evolutionary dynamic, marked by the emergence of haplotypes specific to cancerous tissues. These data suggest that the accumulation of mutations in TP53 and CYP17A1 contributes to genetic heterogeneity and CRC progression in the Senegalese population, opening up prospects for targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0b969d6ec5ed2bb17722c362958a0e34367f77" target='_blank'>
              Evolutionary Signature and Genetic Structure of Concatenated TP53-CYP17A1 Genes in Colorectal Cancer in Senegal
              </a>
            </td>
          <td>
            Anna Ndong, Bineta Kénémé, M. Sembène
          </td>
          <td>2026-01-10</td>
          <td>European Journal of Applied Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer caused by a chromosomal translocation. The most common translocations occur between PAX3 or PAX7 and FOXO1, which generates a chimeric transcription factor with gain-of-function activities. These fusion proteins contain the PAX3/7 DNA-binding domains (paired and homeobox) and the FOXO1 transactivation domain. For over thirty years, these fusions have been established as the defining genetic drivers of this disease; however, they remain challenging to target pharmacologically. As such, improvements to the standard of care, which include surgery, radiation, and general chemotherapy, have focused on optimizing chemotherapy dosage and are rarely curative. Therapeutic development has, in part, been hindered by unclear in vivo regulatory mechanisms of PAX3/7::FOXO1 across cellular contexts and tumorigenic stages. Previously, our groups showed that PAX3::FOXO1, the most common fusion, is a pioneer transcription factor that reshapes chromatin architecture and can transform zebrafish cells to recapitulate human tumors. Recently, we aimed to understand how PAX3::FOXO1 initially interfaces with and modulates chromatin in vivo. We developed a zebrafish mRNA injection model and expressed human PAX3::FOXO1 during early vertebrate development. PAX3::FOXO1 utilized its homeobox domain and partial homeobox motif recognition for initial genomic binding, a new binding mode that varies from in vitro models. Moreover, PAX3::FOXO1 bound and activated inaccessible chromatin in vivo, two hallmarks of pioneer transcription factor activity. Chromatin and transcriptional profiling revealed that PAX3::FOXO1 binding resulted in activation of rhabdomyosarcoma and neural developmental gene signatures. Despite the common association of rhabdomyosarcoma with incomplete myogenesis, neural pathways and populations are present in primary patient tumors. The neural transcription factor HES3 is one example that we previously characterized as a PAX3::FOXO1 cooperating gene. The neural population is also enriched during chemotherapy treatment in patient-derived xenografts. We are continuing to utilize our zebrafish modeling and parallel approaches to understand the functional role and regulation of these neural pathways. Currently, we are investigating how HES3 modulates this activation, given that it developmentally promotes neural stem renewal, is an established target of PAX3::FOXO1, and correlates with worse patient outcomes. Neural signatures are seen in both PAX3::FOXO1 and PAX7::FOXO1 tumors, therefore we are leveraging in vivo comparative analyses between these fusions to assess if differences in neural activation can explain their divergent clinical outcomes. Altogether, this work highlights neural pathway activation as a core activity of PAX3::FOXO1 and suggests that these programs could play a critical role across multiple stages of tumorigenesis. Our long-term goal is to delineate how PAX3/7::FOXO1 cooperate with neural factors across cellular contexts to identify novel therapeutic vulnerabilities.



 Jack Kucinski, Alexi Tallan, Matthew Kent, Andrew Vontell, Katherine Silvius, Cenny Taslim, Benjamin Stanton, Genevieve Kendall. In vivo activation of neural gene signatures in fusion-positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B021.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51dec928b1aed79a0677d3cbac12867c633157e2" target='_blank'>
              Abstract B021: In vivo activation of neural gene signatures in fusion-positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, Matthew R. Kent, Andrew M. Vontell, Katherine M. Silvius, C. Taslim, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id=": Cholangiocarcinoma (CCA) is a rare but aggressive malignancy with a rising global incidence and few therapeutic options for advanced disease. In recent decades, precision oncology for CCA has advanced rapidly, particularly through the development of targeted therapies for patients with actionable genetic alterations. These therapies have markedly prolonged survival and improved other clinical outcomes among patients with unresectable, advanced CCA. The implementation of precision oncology largely depends on detecting genetic mutations to guide patient selection and treatment, using tumor tissue biopsies or liquid biopsies, including circulating tumor DNA (ctDNA) from blood or bile. As a minimally invasive biomarker, ctDNA shows great promise for transforming the clinical management of CCA. This review provides a comprehensive overview of the roles of ctDNA in CCA, including early detection, prognostic stratification, minimal residual disease assessment, recurrence monitoring, therapeutic target identification, and treatment response evaluation. A synthesis of existing studies indicates that bile-derived ctDNA shows superior sensitivity compared with blood-based ctDNA in capturing the genetic profiles and heterogeneity of CCA. We also propose an integrative framework that illustrates how ctDNA profiling can inform diagnosis, treatment, and surveillance across the disease continuum. Because research on ctDNA in CCA remains in its infancy, we discuss current challenges and outline future directions for translating these findings into clinical practice. Collectively, the evidence positions ctDNA—particularly bile-derived ctDNA—as a dynamic tool for real-time genomic profiling, sensitive residual disease detection, and therapy monitoring. This integrative framework provides a roadmap for translating these capabilities into clinical practice, with the potential to enable earlier, more personalized interventions and improve outcomes for patients with CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b18fcde379c42d695605db7b0fc02141c31e64b" target='_blank'>
              Circulating Tumor DNA in Cholangiocarcinoma: A Precision Oncology Roadmap
              </a>
            </td>
          <td>
            Haixing Wei, Jie Wang, Qing Wu, Mengbin Qin
          </td>
          <td>2026-02-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The exchange of genetic material during meiosis requires the formation and repair of DNA double-strand breaks (DSBs), which may not be repaired with perfect fidelity. If meiotic exchange is mutagenic, this would add to the costs of sexual reproduction and affect patterns of genome evolution, but much of the evidence for this is indirect. In the fruit fly Drosophila melanogaster, it is possible to completely suppress endogenous DSBs while retaining normal fertility. We took advantage of this system to generate fly strains with and without a mutant allele of mei-P22, a gene that is essential for meiotic DSB formation, on a common genetic background. This allowed us to investigate the relationship between DSBs and genome-wide mutation patterns, using a mutation accumulation design to allow un-selected spontaneous mutations to be observed. Following 30 generations of mutation accumulation, we identified over 1800 mutations by whole-genome sequencing. The presence of meiotic DSBs had little effect on the rate and spectrum of point mutations. We found that mutations were more likely to occur in areas of the genome with higher rates of crossover recombination, regardless of whether meiotic DSBs were occurring. We also found that the rate of transposable element insertions across multiple TE families was substantially elevated in the group lacking meiotic DSBs, suggesting that host suppression of mobile genetic elements is closely associated with meiotic recombination mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51dfbc7778053febffec2da8d339a92ce276d8e7" target='_blank'>
              Meiotic double strand DNA breaks and spontaneous mutation in Drosophila melanogaster.
              </a>
            </td>
          <td>
            Rob Melde, Austin Daigle, JoHanna Abraham, Nathaniel P. Sharp
          </td>
          <td>2026-01-23</td>
          <td>G3</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Comparative transcriptome analyses of gamma herpesviruses have been revolutionized by long‐read sequencing and integrative multi‐omics approaches, enabling unprecedented resolution of viral gene expression. Cross‐species studies of Kaposi's sarcoma‐associated herpesvirus (KSHV), Epstein–Barr virus (EBV), and murine gamma herpesvirus 68 (MHV68) have uncovered a vast diversity of transcript isoforms, including alternative 5′/3′ UTR variants, alternative splice isoforms, and transcripts encoding truncated or extended coding sequences. These efforts revealed novel transcript isoforms harboring partial open reading frames, entirely new ORFs, and an expanding repertoire of viral non‐coding RNAs. Additional regulatory mechanisms, such as promoter dependence on viral replication factors, selective polyadenylation site usage, frameshifting via alternative splicing, and extensive readthrough transcription, further illustrate the complex strategies governing gamma herpesvirus transcriptomes. This review synthesizes these discoveries, highlighting transcriptional strategies employed by gamma herpesviruses for persistence through different phases of replication, while outlining how integrative transcriptomics has reshaped our understanding of herpesvirus gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8150269b600bc18ee3d2193456ba4418ddf8a3de" target='_blank'>
              Cross‐Species Insights Into Gamma Herpesvirus Transcriptomes: Long‐Read and Multi‐Omics Perspectives
              </a>
            </td>
          <td>
            Ritu Shekhar, Sarah McMahon, S. Tibbetts, Erik K. Flemington, Rolf Renne
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468c034edcdec5179082358a5f9944277909facb" target='_blank'>
              Identification of DNA methylation based prognostic subtype and signature in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Jingjing Zhou, Yuxin Chen, Wei Liu, Chao Wang, Lei Liu, Wei Geng, Linqing Chai, Dong Yang, H. Dai, Haixin Li, Yanrui Zhao, Kexin Chen, Hong Zheng, Lian Li
          </td>
          <td>2025-12-27</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The study aimed to identify the clinically relevant gene variants in unresectable cutaneous melanoma samples from Ukrainian patients using NGS technology and to investigate some pharmacogenomic markers useful for the development of cancer treat- ment strategies.


MATERIALS AND METHODS
30 samples of unresectable cutaneous melanomas of various localizations and differen- tiation grades were analyzed. The Ion Torrent NGS technology of targeted gene sequencing (Custom AmpliSeq™ Cancer hotspot and Pharmacogenomic panels) was applied to identify the genetic alterations, which were classifi d using franklin by Gennox database and custom pharmacogenomic Ion Reporter Software pipeline.


RESULTS
A total of 148 different gene alterations were identifi d in 40 genes (SNVs, MNVs, INdELs) by the Cancer hotspot Panel, revealing the mutation patterns consistent with the international data. however, notable discrepancies exist, such as a high KRAS mutation rate (29.3%), predominantly in stage III tumors, suggesting their role in tumor aggressiveness and progression. We identifi d the frequent TP53 and BRAF mutations, with BRAF V600E being the most common, and observed a higher prevalence of BRAF mutations in females. TP53 mutations were prevalent (59.3%) and varied with age and sex, though their prognostic signifi ance requires further validation. A second key novel fi ding was the detection of FLT3 mutations in 22.2% of the samples, with a signifi antly higher prevalence in stage IV disease, suggesting a potential role of FLT3 in melanoma progression and warranting further investigation as a prognostic biomarker and a potential target for the existing FLT3 inhibitors. Ultimately, our pilot analysis of pharmacogenomic markers un- derscored their potential clinical utility in choosing personalized treatment decisions. for instance, the identifi ation of a patient with the rs35599367 (G/A) risk allele for adverse drug reactions underscores the value of using pharmacogenetic data to make a correct selection between approved therapies.


CONCLUSIONS
Our study presents the fi st comprehensive analysis of somatic mutations and pharmacogenomic markers in unresectable cutaneous melanoma tissue samples from Ukrainian patients. We found that while common somatic variants generally align with global trends, the mutational landscape in this cohort presents several unique features: a high KRAS mutation rate and its apparent stage-specifi prevalence, and a high FLT3 mutation rate, predominantly in stage IV tumors. further validation on a larger number of samples, as well as more exhaustive analysis employ- ing alternative methods, is necessary to substantiate these fi dings and to facilitate more effective melanoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ade3027bdd0b58ffd4bfe0744885c0eb045394f1" target='_blank'>
              SOMATIC GENE VARIANTS IN UNRESECTABLE CUTANEOUS MELANOMA CELLS AND DETECTION OF PHARMACOGENOMIC MARKERS: INFLUENCE ON STRATEGY OF EFFECTIVE CANCER TREATMENT.
              </a>
            </td>
          <td>
            R. Gulkovskyi, G. Gerashchenko, A. Bezverkhiy, N. Melnichuk, O. Mankovska, T. Marchyshak, H. Karaman, M. Krotevych, S. Trokhymych, O. Ruban, Z. Tkachuk, M. Tukalo, V. Kashuba
          </td>
          <td>2025-12-30</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction: Endometrial cancer is the most common gynecological neoplasm, with its origins closely linked to genetic alterations. This study highlights mutations in genes involved in DNA repair, especially the mismatch repair system and the BRCA1/2 genes, as central elements in the development of the disease, its molecular profile, and response to treatments. Understanding these genetic impacts is crucial to improving diagnostic methods and driving the development of personalized therapeutic approaches.
Methodology: A literature review was conducted, searching for articles in the PubMed and Scopus databases. Inclusion criteria included molecular, clinical, and epidemiological studies that established a correlation between mutations in DNA repair genes and endometrial cancer, published in the last ten years. The selected studies were analyzed to gather information on the prevalence of mutations in the mismatch repair system and BRCA1/2 genes, the frequency of microsatellite instability, the clinical and molecular profiles of the tumors, and the impact of these alterations on the response to therapies, especially immunotherapy.
Results: The results of the review indicate that mismatch repair deficiency, caused by germline mutations or epigenetic methylation, represents a relevant molecular event in approximately 20 to 30% of endometrial cancer cases. This deficiency is associated with microsatellite instability and a hypermutated phenotype, characterized by a high mutational burden. This condition makes the tumor more immunogenic, favoring a positive response to immunotherapy with immune checkpoint inhibitors. Furthermore, germline mutations in the MMR genes are linked to Lynch Syndrome, which significantly increases the risk of developing endometrial cancer in young women. Mutations in the BRCA1/2 genes have also been identified, especially in the more aggressive serous subtype, suggesting a role in homologous recombination deficiency and tumor progression.
Conclusion: Genetic alterations in genes responsible for DNA repair play a key role in the molecular biology of endometrial cancer. They not only determine specific molecular subtypes of the disease but also serve as valuable indicators for diagnosis, prognostic assessment, and patient selection for targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c325770f41a4ab3814fbe13355802187ffd82c" target='_blank'>
              Factors in Genetic Alterations in DNA Repair Pathways and Their Effects on Endometrial Cancer
              </a>
            </td>
          <td>
            Mari Uyeda, Yussef Zenun Messias Gonçalves, Gabriel Maluf
          </td>
          <td>2026-02-10</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc2fec4e9f04332da41613e03bc3640d81e2d45" target='_blank'>
              Sclerosing melanocytic tumors with MAP2K1 in frame deletions and 15q gains: A distinctive pathway of nevogenesis with reproducible morphology.
              </a>
            </td>
          <td>
            B. Friedman, Olena Kis, B. Shaw, Jeffrey M. Cloutier, Shaofeng Yan, J. Lefferts, A. Alomari, Carina A. Dehner, Aofei Li, A. Moy, K. Linos
          </td>
          <td>2025-12-19</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020-present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy's that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights.
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-26</td>
          <td>Drug metabolism reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: RASAL1 is a prominent tumor suppressor gene through inactivating RAS, whose functional loss has been widely found to play an important role in thyroid cancer and occurs usually through its genetic and epigenetic inactivation. In this study, we intended to explore aberrant alternative splicing (AS) as an important novel mechanism for the oncogenic inactivation of RASAL1 in thyroid cancer. Methods: We used comprehensive bioinformatic and molecular experimental approaches to identify and characterize aberrant alternative splice variants of RASAL1. This included structural and functional investigation of the potentially oncogenic splice variants of RASAL1, with a focus on exploring the molecular mechanisms and clinical impacts on thyroid cancer. Results: We identified an aberrant alternative splice variant of RASAL1, known as RASAL1-004, that commonly compromised the function of RASAL1 in thyroid cancer. Specifically, we found common skipping of exon 18.1 in RASAL1, leading to the abundant formation of transcript RASAL1-004 in cancer, which was significantly associated with poor survival of patients with thyroid cancer. Mechanistically, Argonaute2 regulates exon 18.1 splicing by binding to the response element in exon 17 containing CCAGCC motif, promoting RASAL1-004 formation. The exon 18.1 skipping caused a conformational change in the RNA structure of RASAL1-004 at the junction of exons 17 and 18, resulting in ribosome stalling, halting RASAL1 translation. This reduced RAS GTPase-activating-like protein 1 (RASAL1 protein) synthesis, consequently leading to the functional loss of RASAL1. Compared with RASAL1-001, the canonical wild-type RASAL1 transcript, the absence of exon 18.1 in RASAL1-004 also conformationally altered the pleckstrin homology domain of RASAL1 protein, which, as we demonstrated, led to the loss of the ability of RASAL1 to localize with cell membrane, thereby impairing its RAS-inactivating function. We further demonstrated that compared with RASAL1-001, RASAL1-004 displayed impaired RAS-signaling pathway-suppressing and cancer cell-suppressing functions. Conclusions: This study identified a novel RASAL1-impairing mechanism, alternative to the classically known genetic and epigenetic mechanisms, for the inactivation of the tumor suppressor gene RASAL1 through aberrant AS to form RASAL1-004 with impaired protein translation. This represents a new oncogenic mechanism in thyroid cancer, with novel cancer biological, prognostic, and therapeutic-targeting implications in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb95b9e95603f8cd6fd23f4253e107361cda23d" target='_blank'>
              Functional Loss of the Tumor Suppressor Gene RASAL1 Through Formation of Aberrant Splice Variant RASAL1-004 as a Novel Oncogenic Mechanism in Thyroid Cancer
              </a>
            </td>
          <td>
            Jing Liao, Junwei Song, Zhuo Wang, Mingzhao Xing
          </td>
          <td>2025-12-22</td>
          <td>Thyroid</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a0df4c6515ac6a39ef14098c63501a9e1cf89d" target='_blank'>
              A rapid and ultrasensitive CRISPR-Cas12a assay for clinical detection of pathogens and mutations.
              </a>
            </td>
          <td>
            Tao Fu, Dongdong Zhao, Qin Wang, Nanyue Peng, Le Zhang, Siqi Chen, Jiangbin Ren, Fan Tang, Ke Pu, Qiaoli Wu, C. Bi, Xueli Zhang, Qingguo Li
          </td>
          <td>2026-02-07</td>
          <td>Analytical and bioanalytical chemistry</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Whilst most mutations cells accumulate are neutral, some can alter cellular functions leading to positive selection and clonal expansions. In inflammatory bowel disease (IBD), epithelial injury-repair cycles raise mutation rates and favour clones with growth advantages. Previous studies [1-3] showed that clones with IL-17 signalling pathway mutations, including PIGR, NFKBIZ and ZC3H12A, recurrently expand in IBD despite being uncommon in colorectal cancer. Organoid work suggested that NFKBIZ mutations provide resistance to IL-17A-induced apoptosis and loss of PIGR was hypothesised to worsen inflammation by promoting dysbiosis. Thus, a positive feedback loop could be established, in which mutant clones drive tissue damage while expanding further. We aim to improve our understanding of how clonal expansions and driver genes influence disease biology in IBD. To achieve this, we performed paired DNA-RNA sequencing of individual human colonic crypts to characterise the transcriptional consequences of driver mutations.



 Multiomics laser-microdissection of single colonic crypts was possible with an improved low-input protocol. Fresh biopsies were collected during colonoscopies from inflamed and uninflamed colon regions. We obtained over 200 new biopsies from 25 patients which, combined with prior data, enabled selection analysis across 1500+ crypts from 50+ patients. We generated 700+ single crypts with paired DNA-RNA data, enabling differential expression analysis (DGE) between mutant and wild-type crypts.



 Our work confirms elevated mutation rates and characteristic mutational signatures associated with azathioprine treatment. Beyond identifying PIGR, NFKBIZ and ZC3H12A as strongly selected genes, consistent with prior work, we show that NFKBIZ is also highly prevalent in individuals of European ancestry. We also expand the catalogue of putative driver genes, particularly highlighting IL13RA1, ELF3 and MED12. DGE analysis suggests mutations in some genes e.g., NFKBIZ, may protect crypts from inflammatory effects, while others remain influenced by inflammation, potentially expanding primarily through increased crypt fission. Crypts with cancer-related mutations display elevated inflammatory signalling and epithelial-to-mesenchymal transition features compared to crypts with IBD-characteristic mutations e.g., ZC3H12A. These findings reveal how distinct driver mutations shape epithelial behaviour in IBD.



 Overall, paired DNA and RNA sequencing of single colonic crypts enabled us to broaden the catalogue of driver genes implicated in IBD and characterise their transcriptomic consequences in human tissue for the first time. Thus, improving our understanding of how mutations shape the epithelial landscape in IBD.
 References:
 1.Olafsson S, McIntyre RE, Coorens T, Butler T, Jung H, Robinson PS, Lee-Six H, Sanders MA, Arestang K, Dawson C, Tripathi M, Strongili K, Hooks Y, Stratton MR, Parkes M, Martincorena I, Raine T, Campbell PJ, Anderson CA. Somatic Evolution in Non-neoplastic IBD-Affected Colon. Cell. 2020 Aug 6;182(3):672-684.e11. doi: 10.1016/j.cell.2020.06.036. Epub 2020 Jul 21. PMID: 32697969; PMCID: PMC7427325.
 2.Kakiuchi N, Yoshida K, Uchino M, Kihara T, Akaki K, Inoue Y, Kawada K, Nagayama S, Yokoyama A, Yamamoto S, Matsuura M, Horimatsu T, Hirano T, Goto N, Takeuchi Y, Ochi Y, Shiozawa Y, Kogure Y, Watatani Y, Fujii Y, Kim SK, Kon A, Kataoka K, Yoshizato T, Nakagawa MM, Yoda A, Nanya Y, Makishima H, Shiraishi Y, Chiba K, Tanaka H, Sanada M, Sugihara E, Sato TA, Maruyama T, Miyoshi H, Taketo MM, Oishi J, Inagaki R, Ueda Y, Okamoto S, Okajima H, Sakai Y, Sakurai T, Haga H, Hirota S, Ikeuchi H, Nakase H, Marusawa H, Chiba T, Takeuchi O, Miyano S, Seno H, Ogawa S. Frequent mutations that converge on the NFKBIZ pathway in ulcerative colitis. Nature. 2020 Jan;577(7789):260-265. doi: 10.1038/s41586-019-1856-1. Epub 2019 Dec 18. PMID: 31853061.
 3.Nanki K, Fujii M, Shimokawa M, Matano M, Nishikori S, Date S, Takano A, Toshimitsu K, Ohta Y, Takahashi S, Sugimoto S, Ishimaru K, Kawasaki K, Nagai Y, Ishii R, Yoshida K, Sasaki N, Hibi T, Ishihara S, Kanai T, Sato T. Somatic inflammatory gene mutations in human ulcerative colitis epithelium. Nature. 2020 Jan;577(7789):254-259. doi: 10.1038/s41586-019-1844-5. Epub 2019 Dec 18. PMID: 31853059.
 4.EllisP,MooreL,SandersMA,ButlerTM,BrunnerSF,Lee-SixH,OsborneR,FarrB,CoorensTHH,LawsonARJ,CaganA,Stratton MR, Martincorena I, Campbell PJ. Reliable detection of somatic mutations in solid tissues by laser-capture microdissection and low-input DNA sequencing. Nat Protoc. 2021 Feb;16(2):841-871. doi: 10.1038/s41596-020-00437-6. Epub 2020 Dec 14. PMID: 33318691.
 Conflict of interest:
 Torra I Benach, Maria: No conflict of interest
 Marshall, Henry: No conflict of interest
 Jung, Hyunchul: No conflict of interest
 Abascal, Federico: No conflict of interest
 Lawson, Andrew: No conflict of interest
 Hu, May: No conflict of interest
 Goodwin, Scott: No conflict of interest
 Garri, Wendy: No conflict of interest
 Harris, Bradley: Personal Fees: I have received honoraria for a presentation at a BridgeBio board meeting.
 Garcia Salinas, Isaac: No conflict of interest
 Jarman, Georgee: No conflict of interest
 El Garwany, Omar: No conflict of interest
 Hooks, Yvette: No conflict of interest
 Sanghvi, Rashesh: No conflict of interest
 O’Neill, Laura: No conflict of interest
 Yogakanthi, Saiumaeswar: No conflict of interest
 Brezina, Biljana: No conflict of interest
 Chappel, Lia: No conflict of interest
 Cotobal Martin, Cristina: No conflict of interest
 Rahbari, Raheleh: No conflict of interest
 Raine, Timothy: Grant: Abbvie, Takeda Personal Fees: TR has received research/educational grants and/or speaker/consultation fees from Abbvie, Alfasigma, Arena, Aslan, AstraZeneca, Boehringer-Ingelheim, BMS, Celgene, Domain Therapeutics, Eli Lilly, Ferring, Galapagos, Gilead, GSK, Heptares, LabGenius, Janssen, MonteRosa, Mylan, MSD, Novartis, Numab, Pfizer, Roche, Sandoz, Scientia, Takeda, UCB and XAP therapeutics
 Anderson, Carl: Personal Fees: Carl Anderson (CAA) has received consultancy or speaker fees from Genomics plc, BridgeBio Ltd and Glaxo Smith Kline.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c73cadd26922fc11658f33d34a414918b714ee81" target='_blank'>
              DOP109 Investigating the role of driver mutations in Ulcerative Colitis through paired DNA and RNA sequencing of single colonic crypts
              </a>
            </td>
          <td>
            M. Torra, I. Benach, H. Marshall, H. Jung, F. Abascal, A. Lawson, M. Hu, S. Goodwin, W. Garri, B. Harris, I. G. Salinas, G. Jarman, O. El Garwany, Y. Hooks, R. Sanghvi, L. O'Neill, S. Yogakanthi, B. Brezina, L. Chappel, C. Martin, R. Rahbari, T. Raine, C. Anderson
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Mutant p53(Mtp53) not only loses its canonical tumor-suppressive functions but also acquires oncogenic gain-of-function properties, positioning it as a central orchestrator in reshaping the tumor immune microenvironment. This review systematically delineates how Mtp53 actively establishes and sustains an immunosuppressive niche through multiple interconnected mechanisms, including chronic inflammation, immune cell dysfunction, reprogramming of cancer-associated fibroblasts, metabolic dysregulation, epigenetic hijacking, and potentially aberrant liquid–liquid phase separation, thereby promoting immune evasion and therapeutic resistance. We integrate current evidence to propose a conceptual “metabolism–epigenetics–immunity” axis: Mtp53-driven metabolic reprogramming—such as accumulation of lactate or α-ketoglutarate—can modulate chromatin modifications and immune gene expression. Notably, the full
 in vivo
 causal chain of this axis remains unestablished; existing support derives primarily from stepwise experimental data and strong correlations. The immunological impact of Mtp53 is highly context-dependent, shaped by co-mutations and tissue origin. In TP53/KRAS co-mutant non-small cell lung cancer (NSCLC), Mtp53 enhances tumor immunogenicity and improves response to immune checkpoint inhibitors (ICIs); conversely, in immunologically “cold” tumors—such as triple-negative breast cancer, pancreatic ductal adenocarcinoma, and colorectal cancer—it promotes T-cell exhaustion or myeloid suppression, reflecting marked cancer-type heterogeneity. Therapeutic approaches include Mtp53 reactivators (e.g., APR-246, PC14586), degraders, synthetic lethal strategies, and neoantigen vaccines. Although APR-246 showed efficacy in a phase II trial (NCT03072043), it failed to improve survival in phase III (NCT03745716) due to lack of TP53 mutation stratification. Its combination with pembrolizumab (NCT04383938) demonstrated acceptable safety (immune-related adverse events in ∼12%) but limited efficacy, underscoring the need for biomarker-guided, precision-based combinations. Thus, a multidimensional biomarker platform is urgently needed—one integrating TP53 mutation subtypes (e.g., R175H
 vs
 . nonsense mutations), dynamic ctDNA monitoring (VAF ≥ 0.01%), tumor immune microenvironment (TIME) features (e.g., TILs, MDSCs), and spatial multi-omics—to enable precise molecular stratification and personalized intervention in Mtp53-driven cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3f2c235e49b5522a1b8c8e57fe450edd58eb9f" target='_blank'>
              The impact of p53 mutation on tumor immune evasion: mechanistic insights and clinical implications
              </a>
            </td>
          <td>
            Luo Liang, Weidong Wang
          </td>
          <td>2026-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing catalytic polypeptide-like (APOBEC) family was first defined as an innate antiviral defense system, but the APOBEC3 subfamily (APOBEC3s) is now recognized as a major endogenous source of somatic mutagenesis in cancer. APOBEC3s enzymes, particularly APOBEC3A and APOBEC3B, generate characteristic mutation patterns that promote genomic instability, clonal evolution, and adaptation to therapy. Beyond driving tumor evolution, APOBEC3 activity reshapes antitumor immunity in solid cancers. APOBEC3-induced mutations increase tumor mutational burden and create neoantigens that can enhance CD8
 +
 T-cell infiltration and interferon signaling. However, sustained APOBEC activation may also reinforce immunosuppressive circuits: through chronic inflammation and PD-1/PD-L1–interferon signaling, tumors can induce T-cell dysfunction, immune escape, and resistance to immune checkpoint blockade. This functional ambivalence has sparked debate over whether APOBEC3s should be inhibited to limit genomic instability, leveraged to enhance tumor immunogenicity, or modulated dynamically in a context-dependent manner. This review outlines the immune landscape and biochemical characteristics of the APOBEC3 family and situates these features within broader cancer-related disease contexts. APOBEC3-mediated mutagenesis is discussed as a mechanistic link between genomic instability and tumor–immune crosstalk in solid tumors, with emphasis on its relationships to immunoediting, immune checkpoint pathways, and therapeutic responses. Context-dependent associations of APOBEC3 activity with immune activation or immune evasion are also considered, together with their implications for strategies that modulate this pathway.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3a672ede5d48a83adb79c8044488b1ee3d37535" target='_blank'>
              Reprogramming tumor immunity through APOBEC3s-mediated mutagenesis: from genome instability to immune checkpoint interactions
              </a>
            </td>
          <td>
            Qiaoxi Li, Wenyu Wan, Zihan Zhu, Xuanduo Lin, Fei Wang, Mengmeng Li, Hao Guo, Yang Yang
          </td>
          <td>2026-02-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are a class of non-coding RNAs characterized by their covalently closed loop structures, which confer remarkable stability. Recent studies have highlighted the role of exosomal circRNAs, which are encapsulated within extracellular vesicles known as exosomes, in intercellular communication and tumor progression. In colorectal cancer (CRC), exosomal circRNAs have been implicated in various oncogenic processes, including cell proliferation, migration, invasion, epithelial-mesenchymal transition (EMT), and drug resistance. These molecules exhibit dysregulated expression patterns in CRC tissues and bodily fluids, making them promising candidates for non-invasive biomarkers. Moreover, their functional roles in modulating signaling pathways and the tumor microenvironment suggest their potential therapeutic applications. This review summarizes our current understanding of exosomal circRNA biology, highlights their roles in the CRC development and progression, and discusses their clinical significance as diagnostic, prognostic and therapeutic tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/490f7019972e1dff0c8c3442eb3b1f9903b34509" target='_blank'>
              Exosome-Derived Circular RNAs in Colorectal Cancer: Emerging Roles in Tumorigenesis, Diagnosis, and Therapy.
              </a>
            </td>
          <td>
            Amir Mohammad Aghaie, Fereydoun Rahmani, N. F. Maroufi, Vahid Zarezade, Soleyman Bafadam, Mostafa Mostafazadeh
          </td>
          <td>2026-02-01</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Accurate detection of gene fusions is critical for the diagnosis and management of sarcomas. However, occasional false-negative results may occur due to factors such as low tumor cellularity, limited assay coverage of relevant genomic regions, or mis-annotation by bioinformatic pipelines. We recently encountered a case in which our clinical SARCP fusion detection assay failed to identify an HMGA2::NCOR2 (exon 3–exon 16) fusion.



 The SARCP assay is a targeted, amplicon-based RNA-seq next-generation sequencing (NGS) assay designed to detect gene fusions across 138 genes, including HMGA2. Although NCOR2 is not among the targeted genes, the assay is capable of detecting fusions when only one gene partner is represented.



 Suspicion of a false-negative result arose when the pathologist noted that the tumor’s histologic features—keratin-positive, giant cell-rich soft tissue—were consistent with a subtype previously associated with the HMGA2::NCOR2 fusion. This prompted a review of the bioinformatic output, which had erroneously reported an HMGA2::MAML2 fusion. The misannotation was traced to sequence homology involving a CAA/CAG repeat at the 5′ end of NCOR2 exon 16, likely resulting from DNA polymerase slippage and misalignment in low-complexity, partially complementary repetitive regions. This erroneously lead to the target genomic sequence getting mapped to MAML2 exon 2 as opposed to the NCOR2 exon 16.



 This case highlights how bioinformatic mis-annotation can lead to occasional false-negative fusion calls. It underscores the importance of manual review in cases with strong clinical or pathologic suspicion and calls attention to the need for fusion-calling algorithms that better handle repetitive and homopolymeric sequences.



 Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, Kevin C. Halling. Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b640522fda2138d045848d46734cd774a873c147" target='_blank'>
              Abstract A012: Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection
              </a>
            </td>
          <td>
            Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, K.C. Halling
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a881ff1a144f552223142321c11b88b5a0aae16" target='_blank'>
              Targeting epigenetic methylation: emerging diagnosis and therapeutic strategies in cancer.
              </a>
            </td>
          <td>
            Youwen Zhu, Kun Liu, Hong Zhu
          </td>
          <td>2026-02-11</td>
          <td>Experimental hematology & oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f592e511a29f5a3530a8b2e5941ac79708dfe187" target='_blank'>
              DMN-seq enriches DNA hypomethylated regions for biomarker discovery using 5-methylcytosine glycosylase.
              </a>
            </td>
          <td>
            Yiding Wang, Yang Li, Chang Ye, Iryna Irkliyenko, Lu Gao, Marc Bissonnette, Qing Dai, Weixin Tang, Chuan He
          </td>
          <td>2026-02-11</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a3d75c8b930697a81a7aeceffc4df22265c789" target='_blank'>
              3D confinement reshapes RNA folding and enhances circularisation in the Zika virus
              </a>
            </td>
          <td>
            J. K. Novev, J. Y. Lau, D. Marenduzzo, G. Kudla
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f687787e5f5da10c05fbeb66dd1e619139451c70" target='_blank'>
              Multiomics reveals the mechanism of KNG1 in promoting tumor progression in adamantinomatous craniopharyngioma
              </a>
            </td>
          <td>
            Lin Zhou, Zhicheng Xu, Chao Fang, Hui Ren, C. Yang, Zhigang Wang, Tao Hong, Jie Wu
          </td>
          <td>2026-01-01</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffc1b13f619526ac6806146d0d7bfc28faa51e9" target='_blank'>
              HiCInterpolate: 4D Spatiotemporal Interpolation of Hi-C Data for Genome Architecture Analysis
              </a>
            </td>
          <td>
            H. M. Mohit Chowdhury, Oluwatosin Oluwadare
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In all cells, hexameric helicases drive the unwinding of parental chromosomal DNA at replication forks to provide the single-stranded DNA templates required by replicative DNA polymerases. DNA unwinding proceeds via a steric exclusion mechanism in which the helicase encircles and translocates along one DNA strand while sterically excluding the opposite strand from its central channel. The details of how hexameric helicases translocate on single-stranded DNA remain incompletely understood and likely vary among species, as structural and mechanistic features-such as motor domain architecture and translocation polarity-shape helicase function. Recent high-resolution cryo-EM structures of the eukaryotic CMG (Cdc45-MCM-GINS) helicase, including complexes stalled at leading-strand G-quadruplexes, reveal two predominant DNA-bound conformations: planar and spiral. These structures show that different subsets of MCM subunits alternately engage the leading-strand template, defining intermediates of a nonrotary, hand-over-hand translocation mechanism. This mode of translocation differs from the sequential rotary hand-over-hand mechanism proposed for bacterial hexameric helicases, instead resembling that of other ring-shaped ATPase motors and can be described as a variant of the helical inchworm model. The evolution of this mechanism may reflect CMG's specialized role as a replisome organizer, enabling it to coordinate accessory factors and optimize replication fork progression. Together, these findings highlight the mechanistic diversity and evolutionary adaptability of hexameric helicases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39a99042bbff405717d35a7afa256356bd265f4" target='_blank'>
              DNA translocation by the CMG helicase: the helical inchworm model.
              </a>
            </td>
          <td>
            S. Batra, Benjamin Allwein, Y. Lucia Wang, R. Hite, Dirk Remus
          </td>
          <td>2026-02-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 The paradigm of the
 Hallmarks of Cancer
 , updated by Douglas Hanahan in 2022, represents one of the most influential syntheses for understanding the functional capabilities that sustain neoplastic transformation. However, its traditional interpretation, often reductionist and fragmentary, does not capture the non-linear, emergent, and adaptive dynamics of tumor behavior. This review proposes a reinterpretation of the hallmarks through the lens of complexity theory, conceptualizing colorectal cancer (CRC) as a self-organizing, open system operating far from equilibrium. Using an integrative conceptual approach, we map the ten classical hallmarks and the new dimensions proposed in 2022 (phenotypic plasticity, non-mutational epigenetic reprogramming, polymorphic microbiomes, and senescence) onto the fundamental properties of complex systems: nonlinearity, emergence, feedback, openness, and historical dependence. We argue that CRC should not be understood as a simple sum of molecular alterations but as a dynamic network of interactions among cells, tissues, and microenvironments where global organization emerges from local rules. This systems-based perspective provides a conceptual foundation for translational models and integrative methodologies in oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0199a76d1009c249f4505d66d97a40ec8df69d8" target='_blank'>
              Colorectal cancer as a complex adaptive system: integrating the hallmarks of cancer with complexity theory
              </a>
            </td>
          <td>
            Luis R. Basbus
          </td>
          <td>2026-02-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="NUTM1-rearrangement (NUTM1-r) defines a significant subset of B-cell acute lymphoblastic leukemia (B-ALL), particularly in infants lacking KMT2A-rearrangements (KMT2A-r), yet its underlying molecular characteristics remain poorly understood. Here, we establish that NUTM1-r leukemia is a discrete entity characterized by a unique transcriptional and epigenetic landscape, notably featuring global DNA hypomethylation, irrespective of the 5' fusion partner. Functional interrogation of NUTM1 fusions reveals a dual oncogenic role: they drive commitment towards the B-lymphoid lineage while concurrently conferring potent leukemic stem cell properties. Strikingly, expression of a representative fusion, BRD9-NUTM1, is sufficient to induce serially-transplantable prepro-B-like leukemia in vivo, faithfully recapitulating the key molecular and immunophenotypic features of human NUTM1-r B-ALL. Mechanistically, NUTM1 fusions establish an aberrant chromatin state, marked by global enhancement of H3K27 acetylation and the creation of distinctive open chromatin regions that co-opt both B-lineage and stemness-related transcriptional programs, including those involving NF-κB and posterior HoxA genes. In stark contrast to resistant KMT2A-r leukemias, NUTM1-r leukemic cells exhibit a profound sensitivity to chemotherapy. This vulnerability is mechanistically linked to the leukemia's dependence on active transcription. Our findings delineate the unique molecular profile of NUTM1-r leukemias, revealing specific vulnerabilities that rationalize their favorable clinical outcomes and suggest opportunities for modified therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec942964a1f8b552f79cec056ff910b4f7724fe5" target='_blank'>
              Comprehensive Molecular and Functional Analysis of NUTM1-Rearranged Leukemia.
              </a>
            </td>
          <td>
            Koutarou Nishimura, Tomoya Isobe, Tsukasa Shigehiro, Masaki Nomura, Weijia Zang, Muran Xiao, Wenjuan Liao, Yui Koike, Akira Nishimura, A. Sato-Otsubo, Hiromi Yamazaki, Hiromi Ito, Shinri Okada, Naomi Matsumoto, Wataru Saika, Yifan Zhang, Yumi Aoyama, Chihiro Hasegawa, Takaya Yamasaki, Yasuo Kubota, Kentaro Oki, N. Kiyokawa, G. Nagae, Kenichi Yoshida, Y. Nannya, Hiroo Ueno, Shiro Fukuda, K. Tatsuno, S. Tsutsumi, Yusuke Shiozawa, Y. Shiraishi, K. Chiba, Hiroko Tanaka, Mariko Eguchi, Yuki Arakawa, K. Koh, Takao Deguchi, Daisuke Tomizawa, T. Miyamura, Eiichi Ishii, Shuki Mizutani, Satoru Miyano, Hiroyuki Aburatani, Seishi Ogawa, A. Takaori-Kondo, Akihiko Yokoyama, Omar Abdel-Wahab, Patricia Ernst, Junko Takita, Tomokatsu Ikawa, Masatoshi Takagi, Daichi Inoue
          </td>
          <td>2025-12-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="INTRODUCTION
Sarcomas are rare, molecularly diverse malignancies that challenge accurate diagnosis and effective therapy. Japan has implemented nationwide comprehensive cancer genomic profiling (CGP) since 2019, creating an integrated framework that enables precision oncology in routine care.


AREAS COVERED
This review synthesizes the status of cancer genomic medicine in Japan, delineates the genomic landscape of sarcomas, and summarizes clinically actionable alterations. We describe the diagnostic value of CGP - particularly the contribution of RNA sequencing for fusion detection - and outline national infrastructures (expert panels, registries) and platform trials that operationalize genome-matched therapy access. We also examine emerging directions, including liquid biopsy, multi-omics integration, whole-genome approaches, and artificial intelligence, and discuss practical barriers (tissue quality, turnaround time, drug access) with potential solutions.


EXPERT OPINION
Japan's coordinated model - CGP embedded in care, centralized data, and adaptive trial platforms - offers a blueprint for rare cancer precision medicine. Over the next five years, earlier CGP use, broader RNA-enabled diagnostics, and trial-mediated access to targeted agents should expand; coupling liquid biopsy with multi-omics and AI will refine classification and treatment selection. Addressing access and implementation gaps will be critical to translate genomic insights into better outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6d29025ec515e35f9718c74eca1a1a7206ef836" target='_blank'>
              Cancer genome profiling in sarcomas: current status and future perspectives in Japan.
              </a>
            </td>
          <td>
            Tatsusnori Shimoi, Shosuke Kita, K. Yonemori
          </td>
          <td>2026-01-16</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chemotherapy selection traditionally relies on tumor tissue of origin. However, since genetic alterations drive tumor behavior, it remains unclear if mutations can better predict response. We hypothesized that genetic aberrations might influence chemotherapy outcomes more than tissue origin. We retrospectively analyzed 15,474 Japanese patients with solid tumors who underwent comprehensive genomic profiling (CGP) and received cytotoxic chemotherapy. Genetic alterations and tumor origin were evaluated for objective response rate (ORR) and time to next treatment (TNT). Gene mutations were assessed across five chemotherapy classes: platinum-based, alkylating agents, antimetabolites, microtubule inhibitors, and topoisomerase inhibitors. Genomic alteration data alone did not surpass organ-based models in predicting response. For platinum-based agents, the gene-only model had an AUC of 0.575 versus 0.604 for the organ-only model. A combined gene-organ model yielded an AUC of 0.618 (P < 0.01). Certain gene-organ interactions were associated with improved outcomes. For example, APC-mutated colorectal cancer showed higher ORR and prolonged TNT (hazard ratio, 0.82; 95% CI, 0.73–0.92; P < 0.001) for platinum-based drugs. While genetic alterations alone did not outperform tumor origin as a predictor, incorporating both may improve exploratory predictions of chemotherapy response. These exploratory findings require prospective validation before any clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1934ddadd51140930510ad12ba5b67db9adbcd8d" target='_blank'>
              Exploratory analysis of gene aberrations and chemotherapy response: findings from a real-world database in Japan
              </a>
            </td>
          <td>
            N. Ishibashi, T. Kamatani, Satoru Aoyama, Masanori Tokunaga, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-12-22</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Reactive species within cells and external agents constantly attack cellular DNA. Different repair pathways reduce toxic and mutagenic effects, and this article describes 130 known human DNA repair genes. There is currently only one system to remove the primary DNA lesions caused by ultraviolet light, despite notable features such as four enzymes that can remove uracil from DNA, seven recombination genes linked to RAD51, and numerous recently identified DNA polymerases that avoid damage. By comparing with model organisms and identifying common folds in three-dimensional protein structures, more human DNA repair genes will be discovered. Clinical applications such as better radiotherapy, anticancer medication treatment, and a deeper comprehension of the forces that cause cellular ageing should result from modification to DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/900d68c84a9ba3abf2d4fb2edaf262168b10c163" target='_blank'>
              Genetic Mutations and DNA Repair
              </a>
            </td>
          <td>
            R. H. Ali, Ibrahim ramzi Hadi, Saba R. Jaafar, D. Nasar, Doaa Hazem Mohammed, Nada Hassan Mohammed, Basma Hassan Mohammed
          </td>
          <td>2026-02-03</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Phage research has played a pioneering role in the transition from genetics to molecular biology, primarily by advancing our understanding of the "3Rs": Replication, Recombination, and Repair. Historically, discoveries arising from the evolutionary arms race between phages and their hosts have provided essential molecular tools, such as restriction enzymes and polymerases, and more recently, have led to genome engineering technologies like CRISPR/Cas9. This commentary outlines the multifaced function of T7 endonuclease I, another important enzyme derived from 3R research, based on new findings by Matsubara et al. (1)).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0db8199361692c384a147dec7cbfece4e2611f6" target='_blank'>
              A Multifunctional Enzyme: Commentary on "Structure-specific DNA Endonuclease T7 Endonuclease I Cleaves DNA Containing UV-Induced DNA Lesions".
              </a>
            </td>
          <td>
            Takuma Shiraki, Tsuyoshi Ikura
          </td>
          <td>2025-12-24</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Telomerase is known as a very specific marker of embryonic cells. It is responsible for telomere elongation (bypassing the end-replication problem) and thus supports normal cell division during tissue and organ development. But it is generally absent or very low in most normal adult somatic cells. However, its overexpression in adulthood (due to secondary expression and activity restoration) is commonly known to be associated with cancer. Apart from its canonical function (associated with telomere length restoration), it also carries out various other roles. Its non-canonical activity covers mitochondrial and epigenetic processes. Consequently, it contributes to the cell response to stress and chemotherapeutic drug treatment. A more detailed understanding of these phenomena offers the opportunity to identify new pathways and targets that may serve as critical factors in breast cancer diagnostics and therapy. In this article, we summarize the latest reports on the discovery of telomerase’s nature, including its canonical and non-canonical roles. The manuscript highlights how these mechanisms contribute to tumorigenesis, therapy resistance, and the survival of cancer cells. Understanding these multifaceted mechanisms behind hTERT’s role in (breast) cancer progression and therapy resistance is crucial for developing more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5301bfbbaf0a4c2684d1a5d04cf7184a1b01801a" target='_blank'>
              Telomeric Ends and Telomerase—Canonical and Non-Canonical Roles in Breast Cancer Tumorigenesis and Therapy Resistance
              </a>
            </td>
          <td>
            Magdalena Kostrzewa, Julia Niedzielska, Wiktoria Mieczkowska, Maja Hoffmann, Julia Rypińska, Adrianna Kowalczyk, Magdalena Stachowiak, B. Rubiś
          </td>
          <td>2026-01-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b7b9d009213c7ffc5a98ac8b16a699f1f4455" target='_blank'>
              Anchored random reverse primer sequencing for quantitative detection of novel gene fusions.
              </a>
            </td>
          <td>
            Xuehao Xiu, Yi Wu, Jiangxue Li, Dongfa Lin, Xiao Sun, Xinglei Su, Zhi Weng, Xiaolei Zuo, Xiurong Yang, Chunhai Fan, Yudong Wang, David Yu Zhang, Ping Song
          </td>
          <td>2026-01-02</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The Sprouty (SPRY) proteins are evolutionarily conserved modulators of growth factor-induced signaling pathways. The four different SPRY isoforms (SPRY1-4) are implicated in different types of cancer, acting as oncogenes or tumor suppressors depending on the SPRY isoform and the malignancy. Despite being tumor suppressors in many types of cancer, SPRY1 is an oncogene in rhabdomyosarcoma, SPRY2 in colorectal cancer, and SPRY4 in gastric cancer. In this review, we summarize the current literature about the functions of SPRY1-4 in glioblastoma (GB) and neuroblastoma (NB). To further delineate the effects of SPRY1-4 in the tumorigenesis of the nervous system, we analyzed the association of SPRY1-4 with the overall and event/progression-free survival of patients with pediatric and adult glioma, GB, and NB using public datasets. Together, there is evidence that SPRY1 and -2 are oncogenes in GB, whereas the role of SPRY3 and -4 in GB is not well defined. In NB, SPRY2 acts as a tumor suppressor, whereas the effects of SPRY1, -3, and -4 in NB have not been investigated so far, although the survival analysis revealed increased survival of NB patients with low SPRY3 levels in different datasets. Thus, this review demonstrates the requirement for further studies about the functions of the SPRY proteins in tumors of the nervous system to define their clinical relevance as potential therapeutic targets in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ca61dd0a09ec9bb1d2e78b16408cd7d446d741" target='_blank'>
              Sprouty in Tumors of the Nervous System
              </a>
            </td>
          <td>
            Petra Obexer, Barbara Hausott
          </td>
          <td>2025-12-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Small DNA tumor viruses such as polyomaviruses have evolved persistent, in some cases lifelong infections despite their compact genomes and host immune pressure. This review synthesizes historical and recent insights into the mechanisms underlying polyomavirus persistence and shedding, including dynamic host cell cycle regulation, viral non-coding control region modulation, and viral microRNA-mediated repression. We highlight modes of shedding consistent with concurrent latent/lytic and smoldering infections, discuss emerging evidence of reversible latency, and identify unresolved questions in viral-host interplay. Understanding these strategies is critical for managing viral reactivation and disease in immunocompromised patients and exemplifies the remarkable evolutionary success of polyomaviruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4cbf37e5d9b69dc6b4a023237fb48855d94d952" target='_blank'>
              mGem: Deciphering how polyomaviruses coexist with their hosts for a lifetime
              </a>
            </td>
          <td>
            James M. Pipas, Christopher S. Sullivan
          </td>
          <td>2026-01-05</td>
          <td>mBio</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Deinococcus radiodurans is an extremely radioresistant bacterium that has captured the interest of researchers in pursuit of innovations in radioresistant technology and potential applications for bioremediation. The following paper provides an overview of the bacterium’s unique biological mechanisms that enable it to maintain such resilience to ionizing radiation. Genetic redundancies and genomic copies play a considerable role in withstanding radiation-induced mutations and DNA damage. Similarly, antioxidant systems, including both enzymatic and non-enzymatic ones, counteract the effects of radiation by removing radiation-induced reactive oxygen species that harm the bacterium’s proteome and genome. Lastly, highly efficient DNA repair mechanisms ensure a speedy recovery from DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8de85ccb80d750a749a5ea83961684c96bf549bd" target='_blank'>
              The Biological Survival Mechanisms Behind the World’s Most Radioresistant Bacterium
              </a>
            </td>
          <td>
            Brandon Cho
          </td>
          <td>2026-01-15</td>
          <td>Scholarly Review Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Therapy of chronic myeloid leukemia (CML) with tyrosine kinase inhibitors (TKIs) targeting the BCR::ABL1 kinase has become a paradigm for precision oncology. Despite the tremendous success of this strategy, with an overall long-term survival rate of 83%, approximately 25% of CML patients experience therapy failure within 5 years of treatment. TKI resistance is multifaceted, involving mutations in BCR::ABL1, but also BCR::ABL1-independent mechanisms. Among them, deregulation of cell adhesion and motility of CML cells has been observed in TKI-resistance. The extracellular matrix protein fibronectin 1 (FN1) has been shown to be deregulated in solid tumors promoting proliferation and metastasis. However, the role of FN1 in hematopoietic neoplasms remains to be fully elucidated. The aim of our study was to gain deeper insights into the role of FN1. Methods FN1 mRNA and protein levels were analyzed using qPCR and immunoblotting. Transfection was performed using nucleofection or stable transfection, followed by analyses of cell number, proliferation and viability. Cell adhesion was assessed using Matrigel-coated surfaces, and FN1 localization was analyzed using immunofluorescence. Results FN1 levels were significantly downregulated in CML cell lines resistant against BCR::ABL1 inhibitors in vitro. SiRNA-mediated FN1 knockdown reduced the cell’s susceptibility to all generations of TKIs employed in treatment of CML, including asciminib. In contrast, the restoration of FN1 expression in TKI-resistant cells re-sensitized the cells to TKI treatment. This effect was also observed in K-562 cells that intrinsically harbor the BCR::ABL1 mutation p. E255K (−35.2%, p < 0.001), as well as in K-562 and Ba/F3 cells after stable transfection of the BCR::ABL1 wild-type or the p. T315I gatekeeper mutation. Clinically, deregulation of FN1 was also observed in peripheral blood cells derived from CML patients. Conclusion Our data indicate that FN1 may serve as a potential therapeutic target to address TKI resistance or as a suitable biomarker for the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72e8af2960b729ef0eaa5d999253e5a9e4a987de" target='_blank'>
              Association of fibronectin 1 deregulation with tyrosine kinase inhibitor resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Lina Tiedemann, S. P. Gorantla, Philine Ahlf, Lucie Sophie Schmidt, Christiane Pott, Merit Litterst, Vicki Waetzig, Inga Nagel, Johanna Ruemenapp, N. von Bubnoff, I. Cascorbi, Meike Kaehler
          </td>
          <td>2025-12-19</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Background/Objectives: Comprehensive genomic profiling (CGP) is a cornerstone of personalized oncology. However, large-scale, systematic data on the somatic mutation spectrum in Russian cancer patients are scarce. This study aimed to characterize the genomic landscape and assess the potential for matched therapy in a Russian cohort of patients with solid tumors. Methods: This retrospective study included 204 patients with various solid tumors. CGP was performed using the FoundationOne®CDx (FFPE tissue) and FoundationOne®Liquid CDx (cfDNA) platforms. The analysis assessed single-nucleotide variants, indels, copy number alterations, gene fusions, tumor mutational burden (TMB), microsatellite instability (MSI), and PD-L1 expression. Results: The most frequently mutated genes were TP53 (61.5%) and KRAS. The median TMB was 4.0 mut/Mb and was significantly lower in stage IV tumors. Significant co-occurrence was observed between KRAS and TP53 mutations, as well as between APC and KRAS mutations, which were particularly characteristic of colorectal cancer. KRAS mutations were associated with higher combined positive score (CPS) values in cases with lung cancer. Based on the CGP results, 44% of patients had findings that supported the use of an approved matched targeted therapy or immunotherapy for their tumor type. An additional 36% of patients had alterations indicating potential benefit from off-label targeted therapy. Conclusions: This study reveals the distinct genomic characteristics of solid tumors in a Russian cohort and confirms the high clinical utility of CGP for identifying actionable targets. Implementing CGP early in the diagnostic process is a necessary step towards realizing personalized treatment strategies for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66aca90a2bac69b97335224b751826ae6cb88014" target='_blank'>
              Genetic Landscape of Solid Malignant Tumors in a Russian Cohort of Patients
              </a>
            </td>
          <td>
            Iurii K. Slepov, E.D. Kopylov, Anton A. Turchin, D. Khmelkova, V. Kaimonov, A. Isaev, R. Deev
          </td>
          <td>2025-12-19</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="AREAS COVERED
we discuss WRN as a caretaker of the genome that can function as a tumor suppressor or tumor enabler, and review progress on targeting functional WRN in microsatellite-unstable (MSI) cancers. We discuss mechanisms and distinct properties of synthetic lethality between the loss of WRN and the MSI-high tumor status, and derive lessons that can be used to identify or improve upon other cases of WRN synthetic lethality. We highlight the properties of a new generation of WRN helicase inhibitors that may confer increased lethality to MSI cancer cells while sparing normal tissues. We also discuss targeting mutant WRN in Werner syndrome and in cancer.


EXPERT OPINION
Significant gaps remain in understanding how WRN loss promotes pathogenesis. Filling these conceptual and informational gaps will allow better prediction of tissue-specific adverse effects of WRN helicase inhibition or WRN loss. We highlight important, unanswered questions in WRN biology and newly emerging research and translational opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7eb2ac748841065c2a53ce43595a163ef1c66b9" target='_blank'>
              The Werner syndrome RECQ helicase as a therapeutic target: new insights.
              </a>
            </td>
          <td>
            Julia M Sidorova, Raymond J. Monnat
          </td>
          <td>2026-01-20</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long-read whole-transcriptome sequencing (WTS) has the potential to precisely characterize fusion oncogenes that drive leukemia and other cancers. While there are a variety of general-purpose fusion detection algorithms that use modern long-read sequencing data, they show poor sensitivity for precision diagnostics in B-ALL and do not robustly assess technical and analytical parameters (ex. sequencing depth) to reliably detect fusion transcripts. FUSILLI (FUSions In Leukemia Long-read sequencing Investigator) is a novel long-read fusion detection algorithm, with a focus on targeted genomic subtyping in B-ALL. FUSILLI was evaluated against extant methods using nanopore WTS from 51 pediatric B-ALL samples sequenced at high depth and 68 at low depth (mean of 11.2M and 1.4M reads, respectively). In the high-depth cohort, FUSILLI demonstrated increased sensitivity (0.81) compared to FusionSeeker, JAFFAL, and LongGF (0.63, 0.76, and 0.70, respectively), while maintaining high specificity (0.92). At lower sequencing depth, FUSILLI showed correspondingly lower sensitivity (0.27) but still outperformed the other fusion callers (sensitivities ranging from 0.09 to 0.16). Computational down sampling suggests that ten million reads is sufficient to sensitively detect B-ALL-relevant fusions using this approach. FUSILLI detects B-ALL fusions with high sensitivity at modest sequencing depth, supporting implementation of nanopore WTS as a low-cost and globally accessible sequencing-based molecular diagnostic platform for pediatric B-ALL and other fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884c4daa4f67a077c5abf5eb62ebb345ad0e753d" target='_blank'>
              Long-read whole-transcriptome sequencing and selective gene panel profiling enable sensitive detection of fusion oncogenes in pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            John Lin, Kofi B. Opoku, M. Litzow, E. Paietta, Ching-Hon Pui, S. Jeha, K. Roberts, C. Mullighan, Thomas B Alexander, Jeremy R. Wang
          </td>
          <td>2026-02-01</td>
          <td>The Journal of molecular diagnostics : JMD</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="We read with great interest the recent advances highlighting epitranscriptomic regulation, particularly N6-methyladenosine (m6A) RNA modification, as a pivotal determinant of cancer progression and therapeutic response. The role of m6A regulators in modulating tumor immune evasion and checkpoint inhibitor efficacy is now emerging as a transformative paradigm in precision oncology [ 1 ] . The m6A modification dynamically influences RNA splicing, translation, and decay through the coordinated actions of “writers” (METTL3/14), “erasers” (FTO, ALKBH5), and “read-ers” (YTHDF proteins) [ 2 ] . In cancer, aberrant m6A signaling affects tumor immunogenicity, antigen presentation, and T-cell infiltration. A recent Nature Cancer study by Liu et al demonstrated that loss of METTL3 in tumor cells enhances PD-L1 expression via interferon-γ signaling, thereby promoting immune evasion [ 3 ] . Conversely, high METTL3 activity correlates with reduced checkpoint inhibitor efficacy across multiple solid tumors. Mechanistically, m6A methylation regulates the stability of mRNAs encoding immune checkpoint and interferon pathway mediators. Zhang et al showed that ALKBH5 overexpression stabilizes PD-L1 and VEGFA transcripts, impairing antitumor immunity [ 4 ] . These findings suggest that targeting m6A modifiers could enhance tumor immunogenicity and improve immune check-point blockade (ICB) outcomes. Indeed, selective FTO inhibitors have been shown to synergize with anti–PD-1 therapy in preclinical melanoma and NSCLC models [ 5 ] . A 2025 Cell Reports Medicine study extended these insights to clinical practice by identifying an m6A immune response score (m6A-IRS) that stratifies ICB responders">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222542ef3a9b269cb34a3fc7069ba1701785faba" target='_blank'>
              Epitranscriptomic regulation via m6A RNA methylation: a determinant of cancer immunotherapy response
              </a>
            </td>
          <td>
            Syeda Tayyaba Batool, Umair Ali, R. K. Mahato
          </td>
          <td>2026-01-20</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The nongenetic mechanisms by which cancer cells escape cell cycle inhibition remain inadequately understood. Here, we uncover an epigenetic pathway driving adaptive resistance to cyclin-dependent kinase 4/6 (CDK4/6) inhibitors in hepatobiliary cancers using integrative approach combining genome-wide CRISPR screenings with transcriptional, epigenetic, and proteomic profiling. Sustained CDK4/6 inhibition triggers BAP1-dependent chromatin remodeling that induces a stem cell–like epigenetic state. Specifically, BAP1 removes ubiquitin modification (H2AK119ub) at the TCF4 promoter, activating WNT and EMT signaling to enhance cellular plasticity and survival under therapy. Notably, genetic and pharmacologic inhibition of BAP1 markedly improves abemaciclib efficacy in multiple mouse models and patient-derived organoids (PDOs). These findings establish BAP1 as a key regulator of tumor plasticity and adaptive resistance through epigenetic reprogramming and suggest a promising strategy for overcoming adaptive therapeutic CDK4/6i resistance by targeting quiescent, drug-resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5133a1e9adcefa74e28b40f72f42e408180a42" target='_blank'>
              Genome-wide screenings identify BAP1 as a synthetic-lethality target with CDK4/6 inhibitors
              </a>
            </td>
          <td>
            Mei Feng, Hong Liu, Lu Zheng, Yang Liu, Zhuang Hao, Hao Xu, Tingting Zhang, Zhen Wu, Xiaolong Qian, Huikai Li, Tengfei Xiao, Yisheng Pan, Shaokun Shu, Ning Zhang
          </td>
          <td>2026-01-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
Centrosome Amplification (CA) is a state where malignant cells contain excessive centrosomes due to cell cycle dysregulation. Altered CA has been observed in Glioblastoma (GBM). This study developed a CA-related gene model to assess the Tumor Immune Microenvironment (TIME) and prognostic outcomes for patients with GBM.


METHODS
TCGA-GBM and mRNAseq_325 cohorts were obtained from the Chinese Glioma Genome Atlas (CGGA) database. CA-relevant gene modules and feature genes were identified via WGCNA analysis. Key genes were selected to develop a risk model, followed by validation of the model's performance. We further compared the gene mutation landscape, TIME characteristics, drug sensitivity, and enriched pathways between high- and low-risk patient groups.


RESULTS
The brown module, which showed the highest correlation with CA, was selected to identify CA-related key genes to develop a Riskscore model. The model can accurately categorize patients into high- and low-risk groups and predict their clinical outcomes with precision. Notably, high-risk GBM patients exhibited higher StromalScore and dendritic score, and the Riskscore was positively correlated with fibroblast infiltration. Moreover, patients with different risk levels displayed distinct enriched pathways and gene mutation landscapes. Further, the high-risk group showed an evidently higher CAF score, and the differential relation between drug sensitivity and the Riskscore was detected.


DISCUSSION
Though CA was altered in GBM, its prognostic utility remained to be explored. The current study addressed this gap by developing a 6-gene risk model capable of predicting the prognosis and TIME of GBM patients.


CONCLUSION
A CA-related model was constructed to assess the prognosis and TIME of GBM patients, contributing to the management of GBM in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a138456b7292f37100a222cac761a7879df828c" target='_blank'>
              Establishment of Centrosome Amplification-Correlated Model to Evaluate the Tumor Immune Microenvironment and Prognosis of Patients with Glioblastoma.
              </a>
            </td>
          <td>
            Jiale Li, Zigui Chen, Chunyuan Zhang, Qisheng Luo, Jun Peng, Jiachong Wang
          </td>
          <td>2026-01-08</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [0, 2, 12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>