<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-21 06:32:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>57</td>
          <td>132</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>102</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>290</td>
          <td>132</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>248</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, J. Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, Sílvia Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4921d9b380eadcd80f19284f6396e453cfba83e" target='_blank'>
              Exploring the potential of extrachromosomal DNA as a novel oncogenic driver.
              </a>
            </td>
          <td>
            Huanbo Zhu, Longtao Huangfu, Junbing Chen, Jiafu Ji, Xiaofang Xing
          </td>
          <td>2024-09-27</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution.
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Great apes have maintained a stable karyotype with few large-scale rearrangements; in contrast, gibbons have undergone a high rate of chromosomal rearrangements coincident with rapid centromere turnover. Here we characterize assembled centromeres in the Eastern hoolock gibbon, Hoolock leuconedys (HLE), finding a diverse group of transposable elements (TEs) that differ from the canonical alpha satellites found across centromeres of other apes. We find that HLE centromeres contain a CpG methylation centromere dip region, providing evidence this epigenetic feature is conserved in the absence of satellite arrays; nevertheless, we report a variety of atypical centromeric features, including protein-coding genes and mismatched replication timing. Further, large structural variations define HLE centromeres and distinguish them from other gibbons. Combined with differentially methylated TEs, topologically associated domain boundaries, and segmental duplications at chromosomal breakpoints, we propose that a “perfect storm” of multiple genomic attributes with propensities for chromosome instability shaped gibbon centromere evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ad367a909b0a60178d576c36304f1f927c20ef" target='_blank'>
              Centromeric transposable elements and epigenetic status drive karyotypic variation in the eastern hoolock gibbon
              </a>
            </td>
          <td>
            Gabrielle Hartley, M. Okhovat, Savannah J. Hoyt, Emily Fuller, Nicole Pauloski, Nicolas Alexandre, I. Alexandrov, R. Drennan, Danilo Dubocanin, David M. Gilbert, Yizi Mao, Christine McCann, Shane Neph, Fedor Ryabov, Takayo Sasaki, Jessica M. Storer, Derek Svendsen, William Troy, Jackson Wells, LM Core, A. Stergachis, Lucia Carbone, Rachel J. O’Neill
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal G. Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K. Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Micronuclei (MN) are a commonly used marker of chromosome instability that form when missegregated chromatin recruits its own nuclear envelope (NE) after mitosis. MN frequently rupture, which results in genome instability, upregulation of metastatic genes, and increased immune signaling. MN rupture is linked to NE defects, but the cause of these defects is poorly understood. Previous work from our lab found that chromosome identity correlates with rupture timing for small MN, i.e. MN containing a short chromosome, with more euchromatic chromosomes forming more stable MN with fewer nuclear lamina gaps. Here we demonstrate that histone methylation promotes rupture and nuclear lamina defects in small MN. This correlates with increased MN size, and we go on to find that all MN have a constitutive nuclear export defect that drives MN growth and nuclear lamina gap expansion, making the MN susceptible to rupture. We demonstrate that these export defects arise from decreased RCC1 levels in MN and that additional loss of RCC1 caused by low histone methylation in small euchromatic MN results in additional import defects that suppress nuclear lamina gaps and MN rupture. Through analysis of mutational signatures associated with early and late rupturing chromosomes in the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset, we identify an enrichment of APOBEC and DNA polymerase E hypermutation signatures in chromothripsis events on early and mid rupturing chromosomes, respectively, suggesting that MN rupture timing could determine the landscape of structural variation in chromothripsis. Our study defines a new model of MN rupture where increased MN growth, caused by defects in protein export, drives gaps in nuclear lamina organization that make the MN susceptible to membrane rupture with long-lasting effects on genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6815113609c0299acce4dadc060265ccb48bb9ff" target='_blank'>
              RCC1 depletion drives protein transport defects and rupture in micronuclei
              </a>
            </td>
          <td>
            Molly G. Zych, Maya Contreras, M. Vashisth, Anna E Mammel, Gavin Ha, Emily M. Hatch
          </td>
          <td>2024-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="As a major type of structural variants, tandem duplication plays a critical role in tumorigenesis by increasing oncogene dosage. Recent work has revealed that noncoding enhancers are also affected by duplications leading to the activation of oncogenes that are inside or outside of the duplicated regions. However, the prevalence of enhancer duplication and the identity of their target genes remains largely unknown in the cancer genome. Here, by analyzing whole-genome sequencing data in a non-gene-centric manner, we identify 881 duplication hotspots in 13 major cancer types, most of which do not contain protein-coding genes. We show that the hotspots are enriched with distal enhancer elements and are highly lineage-specific. We develop a HiChIP-based methodology that navigates enhancer-promoter contact maps to prioritize the target genes for the duplication hotspots harboring enhancer elements. The methodology identifies many novel enhancer duplication events activating oncogenes such as ESR1, FOXA1, GATA3, GATA6, TP63, and VEGFA, as well as potentially novel oncogenes such as GRHL2, IRF2BP2, and CREB3L1 In particular, we identify a duplication hotspot on Chromosome 10p15 harboring a cluster of enhancers, which skips over two genes, through a long-range chromatin interaction, to activate an oncogenic isoform of the NET1 gene to promote migration of gastric cancer cells. Focusing on tandem duplications, our study substantially extends the catalog of noncoding driver alterations in multiple cancer types, revealing attractive targets for functional characterization and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269520ff92f1d6c1a566c78a1f6e2344309fcdcb" target='_blank'>
              Chromatin interaction maps identify oncogenic targets of enhancer duplications in cancer.
              </a>
            </td>
          <td>
            Yueqiang Song, Fuyuan Li, Shangzi Wang, Yuntong Wang, Cong Lai, Lian Chen, Ning Jiang, Jin Li, Xingdong Chen, Swneke D. Bailey, Xiaoyang Zhang
          </td>
          <td>2024-10-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-17</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a180cfc530d1d7fe2d98876dc6894d589798a6e8" target='_blank'>
              Mapping extrachromosomal DNA amplifications during cancer progression.
              </a>
            </td>
          <td>
            Hoon Kim, Soyeon Kim, Taylor E. Wade, Eunchae Yeo, A. Lipsa, A. Gołȩbiewska, Kevin C Johnson, Sepil An, Junyong Ko, Yoonjoo Nam, Hwa Yeon Lee, Seunghyun Kang, Heesuk Chung, S. Niclou, H. Moon, S. Paek, V. Bafna, J. Luebeck, R.G.W. Verhaak
          </td>
          <td>2024-10-14</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Harnessing DNA double-strand breaks (DSBs) is a powerful approach for gene editing, but it may provoke loss of heterozygosity (LOH), which predisposes to tumorigenesis. To interrogate this risk, we developed a two- color flow cytometry-based system (Flo-LOH), detecting LOH in ∼5% of cells following a DSB. After this initial increase, cells with LOH decrease due to a competitive disadvantage with parental cells, but if isolated, they stably propagate. Segmental loss from terminal deletions with de novo telomere addition and nonreciprocal translocations is observed as well as whole chromosome loss, especially following a centromeric DSB. LOH spans megabases distal from the DSB, but also frequently tens of megabases centromere-proximal. Inhibition of microhomology-mediated end joining massively increases LOH, which is synergistically increased with concomitant inhibition of canonical nonhomologous end joining. The capacity for large-scale LOH must therefore be considered when using DSB-based gene editing, especially in conjunction with end joining inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d1b33368b2fb7ae9c66926efe7d4fcbd491246" target='_blank'>
              Megabase-scale loss of heterozygosity provoked by CRISPR-Cas9 DNA double-strand breaks
              </a>
            </td>
          <td>
            Samantha B. Regan, Darpan Medhi, Travis B. White, Yi-Zhen Jiang, Su Jia, Qichen Deng, M. Jasin
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Expanding tandem gene arrays facilitates adaptation through dosage effects and gene family formation via sequence diversification. However, experimental induction of such expansions remains challenging. Here we introduce a method termed break-induced replication (BIR)-mediated tandem repeat expansion (BITREx) to address this challenge. BITREx strategically places Cas9 nickase adjacent to a tandem gene array to break the replication fork that has replicated the array, forming a single-end double-strand break. This break is subsequently end-resected to become single-stranded. Since there is no repeat unit downstream of the break, the single-stranded DNA often invades an upstream unit to initiate ectopic BIR, resulting in array expansion. BITREx has successfully expanded gene arrays in budding yeast, with the CUP1 array reaching ∼1 Mb. Furthermore, appropriate splint DNA allows BITREx to generate tandem arrays de novo from single-copy genes. We have also demonstrated BITREx in mammalian cells. Therefore, BITREx will find various unique applications in genome engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33093f987bef3bbf4086ae4d4df3919c786e836c" target='_blank'>
              Strategic targeting of Cas9 nickase expands tandem gene arrays
              </a>
            </td>
          <td>
            Hiroaki Takesue, Satoshi Okada, Goro Doi, Yuki Sugiyama, Emiko Kusumoto, Takashi Ito
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0970058b2af0e5ea082fb819477f1708b4c4db9a" target='_blank'>
              DNA demethylation triggers cell free DNA release in colorectal cancer cells
              </a>
            </td>
          <td>
            V. Pessei, M. Macagno, Elisa Mariella, Noemi Congiusta, V. Battaglieri, Paolo Battuello, Marco Viviani, Giulia Gionfriddo, S. Lamba, Annalisa Lorenzato, D. Oddo, Fariha Idrees, Alessandro Cavaliere, A. Bartolini, S. Guarrera, M. Linnebacher, Laura Monteonofrio, Luca Cardone, Michele Milella, A. Bertotti, Silvia Soddu, Elena Grassi, G. Crisafulli, A. Bardelli, L. Barault, F. Di Nicolantonio
          </td>
          <td>2024-10-09</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2+ tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One Sentence Summary Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6498a7e5198294cbede8c8c23e085245d62203" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Significance Chromosome arm 8q is the most frequently gained chromosome arm in aneuploid prostate cancer and is associated with worse prognosis. Using population science and laboratory approaches, we identified increased RAD21 expression in prostate cancer as a key adverse prognostic factor related to 8q gains that enables tumors to sustain proliferation despite oncogenic stress and DNA damage from oncogenic mutations. These data pinpoint one mechanism through which aneuploidy promotes tumor progression and suggests a potential therapeutic target for aneuploid prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f061db09f8cb87400e631c0dc03d0078eb8a3c" target='_blank'>
              RAD21 promotes oncogenesis and lethal progression of prostate cancer
              </a>
            </td>
          <td>
            Xiaofeng A Su, Konrad H. Stopsack, Daniel R. Schmidt, Duanduan Ma, Zhe Li, P. Scheet, K. Penney, T. Lotan, W. Abida, Elise G DeArment, Kate Lu, Thomas Janas, Sofia Hu, M. V. Vander Heiden, Massimo Loda, Monica G. Boselli, Angelika Amon, Lorelei Mucci
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="MYC deregulation occurs in the majority of multiple myeloma (MM) cases and is associated with progression and worse prognosis. Enhanced MYC expression occurs in about 70% of MM patients, but it is known to be driven by translocation or amplification events in only ~40% of myelomas. Here, we used CRISPR interference (CRISPRi) to uncover an epigenetic mechanism of MYC regulation whereby increased accessibility of a plasma cell-type specific enhancer leads to increased MYC expression. This native enhancer activity was not associated with enhancer hijacking events but led to specific binding of c-MAF, IRF4, and SPIB transcription factors that activated MYC expression in the absence of known genetic aberrations. In addition, focal amplification was another mechanism of activation of this enhancer in approximately 3.4% of MM patients. Together, these findings define an epigenetic mechanism of MYC deregulation in MM beyond known translocations or amplifications and point to the importance of non-coding regulatory elements and their associated transcription factor networks as drivers of MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89586e6b56717e928b88a8526a759effc353dc88" target='_blank'>
              Selective Enhancer Gain of Function Deregulates MYC Expression in Multiple Myeloma.
              </a>
            </td>
          <td>
            M. Rahmat, Kendell Clement, Jean-Baptiste Alberge, Romanos Sklavenitis-Pistofidis, R. Kodgule, C. Fulco, Daniel Heilpern-Mallory, Katarina Nilsson, David Dorfman, Jesse M Engreitz, G. Getz, Luca Pinello, R. Ryan, I. M. Ghobrial
          </td>
          <td>2024-09-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Satellite DNA makes up ∼11% of the mouse genome, predominantly located in centromeric and pericentric regions, which are crucial for chromosome segregation. While comprehensive assemblies of these regions have been established in the human genome, they are still lacking in the mouse genome. In this study, we used PacBio long-read sequencing, CUT&RUN sequencing, DNA methylation analysis, and RNA sequencing to generate genomic and epigenomic maps of these regions. We find that centromeric regions are primarily occupied by 120-mer Minor satellites, with other Minor Satellite length variants, 112-mers and 112-64-dimers, localized at centromere-pericentric junctions. Pericentromeric regions are mainly composed of homogeneous Major satellites, while pericentric-chromosomal junctions contain a higher density of divergent satellites. Additionally, the density of non-satellite repeats increases progressively from centromeres to pericentromeres, and further toward chromosomal arm junctions. We found that 120-mer Minor satellites in the core centromere are highly enriched with CENP-A, while the 112-mers and 112-64-dimers show lower CENP-A levels. Homogeneous Major satellites are more enriched with H3K9me3 heterochromatin, whereas divergent Major satellites are preferentially associated with H3K27me3. Furthermore, DNA methylation levels are lower in centromeres compared to pericentric regions. We also observed that only a small subset of satellites is transcribed into RNA, particularly regions exhibiting lower DNA methylation density. Our comprehensive assembly and characterization of the genomic and epigenomic landscape of mouse centromeric and pericentric regions have major implications for satellite biology and ongoing mouse telomere-to-telomere (T2T) assembly efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951d9f9f9b8e262cc6e6811c22c27c69e6bb984f" target='_blank'>
              Genomic and epigenomic maps of mouse centromeres and pericentromeres
              </a>
            </td>
          <td>
            Gitika Chaudhry, Jinguye Chen, Lucy Snipes, Smriti Bahl, Jenika Packiaraj, Xuan Lin, Jitendra Thakur
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract N6-Methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3 and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cb34b460873bf086e54bf45d997fe617e03818" target='_blank'>
              The three YTHDF paralogs and VIRMA are strong cross-histotype tumor driver candidates among m6A core genes
              </a>
            </td>
          <td>
            E. Destefanis, D. Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, Alessandro Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-10-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract During mammalian embryogenesis, both the 5-cytosine DNA methylation (5meC) landscape and three dimensional (3D) chromatin architecture are profoundly remodeled during a process known as ‘epigenetic reprogramming.’ An understudied aspect of epigenetic reprogramming is how the 5meC flux, per se, affects the 3D genome. This is pertinent given the 5meC-sensitivity of DNA binding for a key regulator of chromosome folding: CTCF. We profiled the CTCF binding landscape using a mouse embryonic stem cell (ESC) differentiation protocol that models embryonic 5meC dynamics. Mouse ESCs lacking DNA methylation machinery are able to exit naive pluripotency, thus allowing for dissection of subtle effects of CTCF on gene expression. We performed CTCF HiChIP in both wild-type and mutant conditions to assess gained CTCF–CTCF contacts in the absence of 5meC. We performed H3K27ac HiChIP to determine the impact that ectopic CTCF binding has on cis-regulatory contacts. Using 5meC epigenome editing, we demonstrated that the methyl-mark is able to impair CTCF binding at select loci. Finally, a detailed dissection of the imprinted Zdbf2 locus showed how 5meC-antagonism of CTCF allows for proper gene regulation during differentiation. This work provides a comprehensive overview of how 5meC impacts the 3D genome in a relevant model for early embryonic events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4286154f07173854ef3d5bb731fafdeabe579e02" target='_blank'>
              The impact of the embryonic DNA methylation program on CTCF-mediated genome regulation
              </a>
            </td>
          <td>
            Ana Monteagudo-Sánchez, Julien Richard Albert, Margherita Scarpa, D. Noordermeer, Maxim V. C. Greenberg
          </td>
          <td>2024-08-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). Using a functional genome-wide CRISPRi screen in PDAC cells, we identified activators of selective MYC translation through its 5’ untranslated region (5’UTR) and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. Among these RBPs, our top hit was RBM42, which is highly expressed in PDAC and predicts poor survival. Combining polysome sequencing and CLIP-seq analyses, we find that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing IP-mass spectrometry analysis, we find that RMB42 is a novel ribosome-associated protein (RAP). Using DMS-Seq and mutagenesis analysis, we show that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex. Importantly, RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/168afbf485caea55a52d0fa0de95551cbcbbc5ea" target='_blank'>
              Functional screen for mediators of onco-mRNA translation specificity
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Break-induced replication (BIR) is mutagenic, and thus its use requires tight regulation, yet the underlying mechanisms remain elusive. Here we uncover an important role of 53BP1 in suppressing BIR after end resection at double strand breaks (DSBs), distinct from its end protection activity, providing insight into the mechanisms governing BIR regulation and DSB repair pathway selection. We demonstrate that loss of 53BP1 induces BIR-like hyperrecombination, in a manner dependent on Polα-primase-mediated end fill-in DNA synthesis on single-stranded DNA (ssDNA) overhangs at DSBs, leading to PCNA ubiquitination and PIF1 recruitment to activate BIR. On broken replication forks, where BIR is required for repairing single-ended DSBs (seDSBs), SMARCAD1 displaces 53BP1 to facilitate the localization of ubiquitinated PCNA and PIF1 to DSBs for BIR activation. Hyper BIR associated with 53BP1 deficiency manifests template switching and large deletions, underscoring another aspect of 53BP1 in suppressing genome instability. The synthetic lethal interaction between the 53BP1 and BIR pathways provides opportunities for targeted cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b30096c06a55bc35f817e3d1b46b909b538103" target='_blank'>
              53BP1 deficiency leads to hyperrecombination using break-induced replication (BIR)
              </a>
            </td>
          <td>
            Sameer Bikram Shah, Youhang Li, Shibo Li, Qing Hu, Tong Wu, Yanmeng Shi, Tran Nguyen, Isaac Ive, Linda Shi, Hailong Wang, Xiaohua Wu
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1155864d234b3aeabe252449e0eb74460699d3b9" target='_blank'>
              Prediction of the 3D cancer genome from whole-genome sequencing using InfoHiC.
              </a>
            </td>
          <td>
            Yeonghun Lee, Sung-Hye Park, Hyunju Lee
          </td>
          <td>2024-09-25</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication stresses are the major source of break-induced replication (BIR). Here, we show that in alternative lengthening of telomeres (ALT) cells, replication stress-induced polyubiquitinated proliferating cell nuclear antigen (PCNA) (polyUb-PCNA) triggers BIR at telomeres and the common fragile site (CFS). Consistently, depleting RAD18, a PCNA ubiquitinating enzyme, reduces the occurrence of ALT-associated promyelocytic leukemia (PML) bodies (APBs) and mitotic DNA synthesis at telomeres and CFS, both of which are mediated by BIR. In contrast, inhibiting ubiquitin-specific protease 1 (USP1), an Ub-PCNA deubiquitinating enzyme, results in an increase in the above phenotypes in a RAD18- and UBE2N (the PCNA polyubiquitinating enzyme)-dependent manner. Furthermore, deficiency of ATAD5, which facilitates USP1 activity and unloads PCNAs, augments recombination-associated phenotypes. Mechanistically, telomeric polyUb-PCNA accumulates SLX4, a nuclease scaffold, at telomeres through its ubiquitin-binding domain and increases telomere damage. Consistently, APB increase induced by Ub-PCNA depends on SLX4 and structure-specific endonucleases. Taken together, our results identified the polyUb-PCNA-SLX4 axis as a trigger for directing BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5983b64d73ca9ddeaaf22475670db85ecf93eab0" target='_blank'>
              Polyubiquitinated PCNA triggers SLX4-mediated break-induced replication in alternative lengthening of telomeres (ALT) cancer cells.
              </a>
            </td>
          <td>
            Sangin Kim, Su Hyung Park, Nalae Kang, J. Ra, Kyungjae Myung, Kyoo-young Lee
          </td>
          <td>2024-09-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Histones are essential for maintaining chromatin structure and function. Histone mutations lead to changes in chromatin compaction, gene expression, and the recruitment of DNA repair proteins to the DNA lesion. These disruptions can impair critical DNA repair pathways, such as homologous recombination and non-homologous end joining, resulting in increased genomic instability, which promotes an environment favorable to tumor development and progression. Understanding these mechanisms underscores the potential of targeting DNA repair pathways in cancers harboring mutated histones, offering novel therapeutic strategies to exploit their inherent genomic instability for better treatment outcomes. Here, we examine how mutations in histone H3 disrupt normal chromatin function and DNA damage repair processes and how these mechanisms can be exploited for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae6d859b6a16ea85b27994b62cbff421ebf7a45" target='_blank'>
              Histone H3 mutations and their impact on genome stability maintenance.
              </a>
            </td>
          <td>
            Lucas D Caeiro, Ramiro E Verdun, Lluis Morey
          </td>
          <td>2024-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment
              </a>
            </td>
          <td>
            Marta Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbbfa74d4b379f3ac75ff31b1584f4b3b436bc" target='_blank'>
              Heterochromatin formation and remodeling by IRTKS condensates counteract cellular senescence
              </a>
            </td>
          <td>
            Jia Xie, Zhao-Ning Lu, Shi-Hao Bai, Xiao-Fang Cui, He-Yuan Lian, Chen-Yi Xie, Na Wang, Lan Wang, Ze-Guang Han
          </td>
          <td>2024-08-27</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV) uses latency programs to colonize the memory B-cell reservoir, and each program is associated with human malignancies. However, knowledge remains incomplete of epigenetic mechanisms that maintain the highly restricted latency I program, present in memory and Burkitt lymphoma cells, in which EBNA1 is the only EBV-encoded protein expressed. Given increasing appreciation that higher order chromatin architecture is an important determinant of viral and host gene expression, we investigated roles of Wings Apart-Like Protein Homolog (WAPL), a host factor that unloads cohesin to control DNA loop size and that was discovered as an EBNA2-associated protein. WAPL knockout (KO) in Burkitt cells de-repressed LMP1 and LMP2A expression, but not other EBV oncogenes, to yield a viral program reminiscent of EBV latency II, which is rarely observed in B-cells. WAPL KO also increased LMP1/2A levels in latency III lymphoblastoid cells. WAPL KO altered EBV genome architecture, triggering formation of DNA loops between the LMP promoter region and the EBV origins of lytic replication (oriLyt). Hi-C analysis further demonstrated that WAPL KO reprogrammed EBV genomic DNA looping. LMP1 and LMP2A de-repression correlated with decreased histone repressive marks at their promoters. We propose that EBV coopts WAPL to negatively regulate latent membrane protein expression to maintain Burkitt latency I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb753cd4a98829975ef9e5d570c14e44b7ff8" target='_blank'>
              The DNA loop release factor WAPL suppresses Epstein-Barr virus latent membrane protein expression to maintain the highly restricted latency I program
              </a>
            </td>
          <td>
            Laura A. Murray-Nerger, Davide Maestri, Xiang Liu, Zhixuan Li, Nina R Auld, I. Tempera, Mingxiang Teng, B. Gewurz
          </td>
          <td>2024-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Aneuploidy typically poses challenges for cell survival and growth. However, recent studies have identified exceptions where aneuploidy is beneficial for cells with mutations in certain regulatory genes. Our research reveals that cells lacking the spindle checkpoint gene BUB3 exhibit aneuploidy of select chromosomes. While the spindle checkpoint is not essential in budding yeast, the loss of BUB3 and BUB1 increases the probability of chromosome missegregation compared to wildtype cells. Contrary to the prevailing assumption that the aneuploid cells would be outcompeted due to growth defects, our findings demonstrate that bub3Δ cells consistently maintained aneuploidy of specific chromosomes over many generations. We investigated whether the persistence of these additional chromosomes in bub3Δ cells resulted from the beneficial elevated expression of certain genes, or mere tolerance. We identified several genes involved in chromosome segregation and cell cycle regulation that confer an advantage to Bub3-depleted cells. Overall, our results suggest that the upregulation of specific genes through aneuploidy may provide a survival and growth advantage to strains with poor chromosome segregation fidelity. AUTHOR SUMMARY Accurate chromosome segregation is crucial for the proper development of all living organisms. Errors in chromosome segregation can lead to aneuploidy, characterized by an abnormal number of chromosomes, which generally impairs cell survival and growth. However, under certain stress conditions, such as in various cancers, cells with specific mutations and extra copies of advantageous chromosomes exhibit improved survival and proliferation. In our study, we discovered that cells lacking the spindle checkpoint protein Bub3 became aneuploid, retaining specific chromosomes. This finding was unexpected because although bub3Δ cells have a higher rate of chromosome mis-segregation, they were not thought to maintain an aneuploid karyotype. We investigated whether the increased copy number of specific genes on these acquired chromosomes offered a benefit to Bub3-deficient cells. Our results revealed that several genes involved in chromosome segregation and cell cycle regulation prevented the gain of chromosomes upon Bub3-depletion, suggesting that these genes confer a survival advantage. Overall, our study demonstrates that cells lacking Bub3 selectively retain specific chromosomes to increase the copy number of genes that promote proper chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe65305b6ce867bfbed31e843276ed59a3f7ed3" target='_blank'>
              Aneuploidy of Specific Chromosomes is Beneficial to Cells Lacking Spindle Checkpoint Protein Bub3
              </a>
            </td>
          <td>
            Pallavi Gadgil, Olivia Ballew, Timothy J. Sullivan, Soni Lacefield
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f6b0780202531821c7a75fa420779272072de3" target='_blank'>
              Retrotransposon addiction promotes centromere function via epigenetically activated small RNAs
              </a>
            </td>
          <td>
            Atsushi Shimada, Jonathan Cahn, E. Ernst, Jason Lynn, D. Grimanelli, Ian R. Henderson, T. Kakutani, R. Martienssen
          </td>
          <td>2024-09-02</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>93</td>
        </tr>

        <tr id="Epigenetic mechanisms govern the transcriptional activity of lineage-specifying enhancers; but recent work challenges the dogma that joint chromatin accessibility and DNA demethylation are prerequisites for transcription. To understand this paradox, we established a highly-resolved timeline of DNA demethylation, chromatin accessibility, and transcription factor occupancy during neural progenitor cell differentiation. We show thousands of enhancers undergo rapid, transient accessibility changes associated with distinct periods of transcription factor expression. However, most DNA methylation changes are unidirectional and delayed relative to chromatin dynamics, creating transiently discordant epigenetic states. Genome-wide detection of 5-hydroxymethylcytosine further revealed active demethylation begins ahead of chromatin and transcription factor activity, while enhancer hypomethylation persists long after these activities have dissipated. We demonstrate that these timepoint specific methylation states predict past, present and future chromatin accessibility using machine learning models. Thus, chromatin and DNA methylation collaborate on different timescales to mediate short and long-term enhancer regulation during cell fate specification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd62d46779b15aae2e25fda71c67bc8ebbdad51f" target='_blank'>
              Temporally discordant chromatin accessibility and DNA demethylation define short and long-term enhancer regulation during cell fate specification
              </a>
            </td>
          <td>
            Lindsey N. Guerin, Timothy J. Scott, Jacqueline A. Yap, Annelie Johansson, Fabio Puddu, Tom Charlesworth, Yilin Yang, Alan J Simmons, Ken S. Lau, Rebecca A. Ihrie, Emily Hodges
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Genome instability relies on preserving the chromatin structure, with any histone imbalances threating DNA integrity. Histone synthesis occurs in the cytoplasm, followed by a maturation process before their nuclear translocation. This maturation involves protein folding and the establishment of post-translational modifications. Disruptions in this pathway hinder chromatin assembly and contribute to genome instability. JMJD1B, a histone demethylase, not only regulates gene expression but also ensures a proper supply of histones H3 and H4 for the chromatin assembly. Reduced JMJD1B levels lead to the cytoplasmic accumulation of histones, causing defects in the chromatin assembly and resulting in DNA damage. To investigate the role of JMJD1B in regulating genome stability and the malignancy of melanoma tumors, we used a JMJD1B/KDM3B knockout in B16F10 mouse melanoma cells to perform tumorigenic and genome instability assays. Additionally, we analyzed the transcriptomic data of human cutaneous melanoma tumors. Our results show the enhanced tumorigenic properties of JMJD1B knockout melanoma cells both in vitro and in vivo. The γH2AX staining, Micrococcal Nuclease sensitivity, and comet assays demonstrated increased DNA damage and genome instability. The JMJD1B expression in human melanoma tumors correlates with a lower mutational burden and fewer oncogenic driver mutations. Our findings highlight JMJD1B’s role in maintaining genome integrity by ensuring a proper histone supply to the nucleus, expanding its function beyond gene expression regulation. JMJD1B emerges as a crucial player in preserving genome stability and the development of melanoma, with a potential role as a safeguard against oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490db96e0fd741bd69913b1e9fe76e21ec91750" target='_blank'>
              Unraveling the Role of JMJD1B in Genome Stability and the Malignancy of Melanomas
              </a>
            </td>
          <td>
            Perla Cruz, Diego Peña-Lopez, Diego Figueroa, Isidora Riobó, Vincenzo Benedetti, Francisco Saavedra, Claudia Espinoza-Arratia, Thelma M. Escobar, A. Lladser, Alejandra Loyola
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast Cancer Type 1 Susceptibility Protein (BRCA1) is a tumor-suppressor protein that regulates various cellular pathways, including those that are essential for preserving genome stability. One essential mechanism involves a BRCA1-A complex that is recruited to double-strand breaks (DSBs) by RAP80 before initiating DNA damage repair (DDR). How RAP80 itself is recruited to DNA damage sites, however, is unclear. Here, we demonstrate an intrinsic correlation between a methyltransferase DOT1L-mediated RAP80 methylation and BRCA1-A complex chromatin recruitment that occurs during cancer cell radiotherapy resistance. Mechanistically, DOT1L is quickly recruited onto chromatin and methylates RAP80 at multiple lysines in response to DNA damage. Methylated RAP80 is then indispensable for binding to ubiquitinated H2A and subsequently triggering BRCA1-A complex recruitment onto DSBs. Importantly, DOT1L-catalyzed RAP80 methylation and recruitment of BRCA1 have clinical relevance, as inhibition of DOT1L or RAP80 methylation seems to enhance the radiosensitivity of cancer cells both in vivo and in vitro. These data reveal a crucial role for DOT1L in DDR through initiating recruitment of RAP80 and BRCA1 onto chromatin and underscore a therapeutic strategy based on targeting DOT1L to overcome tumor radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a53d6b1d8429c7df99fb0b7dc3e6119436184d" target='_blank'>
              DOT1L-mediated RAP80 methylation promotes BRCA1 recruitment to elicit DNA repair.
              </a>
            </td>
          <td>
            Huangqi Tang, Ya-Fei Lu, Rongsheng Zeng, Chaohua Liu, Yuxin Shu, Yupei Wu, Jiajie Su, Longjiang Di, Jinqin Qian, Jun Zhang, Yuan Tian, Xiaopeng Lu, Xin-Hai Pei, Qian Zhu, Wei-Guo Zhu
          </td>
          <td>2024-08-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>2</td>
          <td>15</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and show that STAG2 is uniquely tumor suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility and 3D genome conformation that impact cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relates STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin mediated mechanisms of lung tumor suppression. SUMMARY STAG2 is a frequently mutated cohesin subunit across several cancers and one of the most important functional suppressors of lung adenocarcinoma. Our findings underscore important roles of STAG2 in suppressing lung tumorigenesis and highlight a STAG2-PAXIP1/PAGR1 tumor-suppressive program that may transcend cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0708e85569c4f6a48ce19a8abe96d4d4b9af8050" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis
              </a>
            </td>
          <td>
            Emily L. Ashkin, Yuning J. Tang, Haiqing Xu, King L. Hung, Julia Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="In cancer evolution, genome alterations often occur in a specific order, implying selection depends on the prior clonal genotype1-3. It is unknown if similar constraints operate in normal epithelia. Here, we mapped mutations in normal mid-esophagus of aged UK subjects. Mutant NOTCH1 clones colonized most of the epithelium by age 60 and the tissue became saturated with mutants under strong competitive selection by age 70. Compared to samples that were mostly NOTCH1 wildtype, samples predominantly mutant for NOTCH1 showed weaker selection of mutant TP53 and increased selection of NOTCH2 mutants. In mouse esophagus lacking Notch1 we observed strong selection of mutant Notch2, not seen in wild type epithelium. In aging esophagus, the first driver mutation may change the trajectory of subsequent somatic evolution by altering mutational selection and by restricting space available for the expansion of other mutant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/592dbfe38e6c8c93e0cf80a384a68fa37ab62985" target='_blank'>
              Somatic mutant selection is altered by prior NOTCH1 mutation in aging esophageal epithelium
              </a>
            </td>
          <td>
            C. King, E. Abby, J. Fowler, Irina Abnizova, R. Sood, S. H. Ong, M. Hall, Faye Lynch-Williams, Benjamin A. Hall, P. Jones
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5462de017e3ff2bccdaec319bc740a6c77b29d76" target='_blank'>
              PROTAC-mediated conditional degradation of the WRN helicase as a potential strategy for selective killing of cancer cells with microsatellite instability
              </a>
            </td>
          <td>
            Vikram Tejwani, Thomas Carroll, Thomas Macartney, Susanne Bandau, C. Alabert, Giulia Saredi, R. Toth, John Rouse
          </td>
          <td>2024-09-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Structural Maintenance of Chromosomes (SMC) complexes are an evolutionary conserved protein family. In most eukaryotes, three SMC complexes have been characterized, as follows: cohesin, condensin, and SMC5/6 complexes. These complexes are involved in a plethora of functions, and defects in SMC genes can lead to an increased risk of chromosomal abnormalities, infertility, and cancer. To investigate the evolution of SMC complex genes in mammals, we analyzed their selective patterns in an extended phylogeny. Signals of positive selection were identified for condensin NCAPG, for two SMC5/6 complex genes (SMC5 and NSMCE4A), and for all cohesin genes with almost exclusive meiotic expression (RAD21L1, REC8, SMC1B, and STAG3). For the latter, evolutionary rates correlate with expression during female meiosis, and most positively selected sites fall in intrinsically disordered regions (IDRs). Our results support growing evidence that IDRs are fast evolving, and that they most likely contribute to adaptation through modulation of phase separation. We suggest that the natural selection signals identified in SMC complexes may be the result of different selective pressures: a host-pathogen arms race in the condensin and SMC5/6 complexes, and an intragenomic conflict for meiotic cohesin genes that is similar to that described for centromeres and telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc74a6b808c47f9b7e227a7f90d663f5e5a7ca2" target='_blank'>
              Positive Selection Drives the Evolution of the Structural Maintenance of Chromosomes (SMC) Complexes
              </a>
            </td>
          <td>
            D. Forni, A. Mozzi, M. Sironi, R. Cagliani
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA replication is constantly challenged by a wide variety of endogenous and exogenous stressors that can damage DNA. Such lesions encountered during genome duplication can stall replisomes and convert replication forks into double-strand breaks. If left unrepaired, these toxic DNA breaks can trigger chromosomal rearrangements, leading to heightened genome instability and an increased likelihood of cellular transformation. Additionally, cancer cells exhibit persistent replication stress, making the targeting of replication fork vulnerabilities in tumor cells an attractive strategy for chemotherapy. A highly versatile and powerful technique to study DNA breaks during replication is the comet assay. This gel electrophoresis technique reliably detects the induction and repair of DNA breaks at the single-cell level. Herein, a protocol is outlined that allows investigators to measure the extent of DNA damage in mitotically dividing human cells using fork-stalling agents across multiple cell types. Coupling this with automated comet scoring facilitates rapid analysis and enhances the reliability in studying induction of DNA breaks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb1e23cd556c7368ee2eef1c5cc145deb5fd207" target='_blank'>
              Detection of DNA Breaks in Dividing Human Cells by Neutral Comet Assay.
              </a>
            </td>
          <td>
            Anthony Nelligan, Huzefa Dungrawala
          </td>
          <td>2024-08-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Inhibitors of DNA methylation such as 5-aza-deoxycytidine are widely used in experimental and clinical settings. However, their mechanism of action is such that DNA damage inevitably co-occurs with loss of DNA methylation, making it challenging to discern their respective effects. Here we deconvolute the effects of decreased DNA methylation and DNA damage on cancer cells, by using degron alleles of key DNA methylation regulators. We report that cancer cells with decreased DNA methylation —but no DNA damage— enter cellular senescence, with G1 arrest, SASP expression, and SA-β-gal positivity. This senescence is independent of p53 and Rb, but involves p21, which is cytoplasmic and inhibits apoptosis, and cGAS, playing a STING-independent role in the nucleus. Xenograft experiments show that tumor cells can be made senescent in vivo by decreasing DNA methylation. These findings reveal the intrinsic effects of loss of DNA methylation in cancer cells and have practical implications for future therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1085e473421f1f7e1c330f1b10963b8e65fce073" target='_blank'>
              DNA methylation protects cancer cells against senescence
              </a>
            </td>
          <td>
            Xiaoying Chen, Kosuke Yamaguchi, Brianna Rodgers, D. Goehrig, D. Vindrieux, Xavier Lahaye, Matthieu Nolot, Laure Ferry, Nadine Martin, Pierre Dubus, F. Miura, Takashi Ito, Nicolas Manel, Masato T. Kanemaki, David Bernard, P. Defossez
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogen-induced mutations are thought near-random, with rare cancer-driver mutations underlying clonal expansion. Using high-fidelity Duplex Sequencing to reach a mutation frequency sensitivity of 4×10-9 per nt, we report that sun exposure creates pervasive mutations at sites with ∼100-fold UV-sensitivity in RNA-processing gene promoters – cyclobutane pyrimidine dimer (CPD) hyperhotspots – and these mutations have a mini-driver clonal expansion phenotype. Numerically, human skin harbored 10-fold more genuine mutations than previously reported, with neonatal skin containing 90,000 per cell; UV signature mutations increased 8,000-fold in sun-exposed skin, averaging 3×10-5 per nt. Clonal expansion by neutral drift or passenger formation was nil. Tumor suppressor gene hotspots reached variant allele frequency 0.1-10% via 30-3,000 fold clonal expansion, in occasional biopsies. CPD hyperhotspots reached those frequencies in every biopsy, with modest clonal expansion. In vitro, tumor hotspot mutations arose occasionally over weeks of chronic low-dose exposure, whereas CPD hyperhotspot mutations arose in days at 1000-fold higher frequencies, growing exponentially. UV targeted mini-drivers in every skin cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccbc5cb5ec1678ecc35a610c5268f3895b55100" target='_blank'>
              Pervasive Induction of Regulatory Mutation Microclones in Sun-exposed Skin
              </a>
            </td>
          <td>
            Vijay Menon, Alejandro Garcia-Ruiz, Susan Neveu, Brenda Cartmel, Leah M. Ferrucci, Meg Palmatier, Christine Ko, Kenneth Y. Tsai, Mio Nakamura, Sa Rang Kim, Michael Girardi, Karl Kornacker, Douglas E. Brash
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ee5fba3b3bfc377211f04436065c1bd9deb16" target='_blank'>
              Elucidating the genotoxicity of Fusobacterium nucleatum-secreted mutagens in colorectal cancer carcinogenesis
              </a>
            </td>
          <td>
            Wenye Xu, Yuchen Zhang, Dongjiao Chen, Dan Huang, Yang Zhao, Wei Hu, Ling Lin, Yingzhi Liu, Shilan Wang, Judeng Zeng, Chuan Xie, H. Chan, Qing Li, Huarong Chen, Xiaodong Liu, Sunny H Wong, Jun Yu, Francis K. L. Chan, Matthew T V Chan, Siew C Ng, William K. K. Wu, Lin Zhang
          </td>
          <td>2024-09-27</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Developmental genes are controlled by an ensemble of cis-acting regulatory elements (REs), which in turn respond to multiple trans-acting transcription factors (TFs). Understanding how a cis-regulatory landscape integrates information from many dynamically expressed TFs has remained a challenge. We develop a combined CRISPR-screening approach using endogenous RNA and RE-reporters as readouts. Applied to the Xist locus, crucial for X-chromosome inactivation in females, this method allows us to comprehensively identify Xist-controlling TFs and map their TF-RE wiring. We find a group of transiently expressed TFs that regulate proximal REs, driving the binary activation of Xist expression. These basal activators are more highly expressed in cells with two X chromosomes, potentially driving female-specific Xist upregulation. A second set of developmental TFs is upregulated later during differentiation and targets distal REs. This regulatory axis is crucial to achieve high levels of Xist RNA, which is necessary for X-chromosome inactivation. Our findings support a model for developmental gene regulation in which factors targeting proximal REs drive binary ON-OFF decisions, while factors interacting with distal REs control the transcription output.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c7a0fccbc6e2685b156155cb3fdb69def2ff6a" target='_blank'>
              Reporter CRISPR screens decipher cis- and trans-regulatory principles at the Xist locus
              </a>
            </td>
          <td>
            Till Schwämmle, Gemma Noviello, Eleni Kanata, Jonathan J. Froehlich, Melissa Bothe, Aybuge Altay, Jade Scouarnec, Vivi-Yun Feng, Martin Vingron, Edda G. Schulz
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: The genes cadherin 4 (CDH4), Kirsten rat sarcoma viral oncogene homolog (KRAS), V-Raf murine sarcoma viral oncogene homolog B (BRAF), and microsatellite instability (MSI) each play a role in the development of colorectal cancer (CRC). CDH4 gene is located in chromosome 20q13.33 and its amplification or gain is the earliest mutational event found in the majority of CRC and colon polyps. This study aimed to identify copy number alteration in the CDH4 gene and its association with the clinical profile of CRC patients, including gender, age, tumor location and differentiation, frequency of BRAF and KRAS mutations, and MSI status. Methods: The DNA was extracted from 50 tumors and adjacent normal tissues based on the manufacturer’s instructions. Detection of MSI status was carried out by pentaplex polymerase chain reaction (PCR) and PCR products were size separated by capillary electrophoresis (CE) using an ABI 310 Genetic Analyzer. Both KRAS and BRAF mutations were identified by PCR and sequencing while CDH4 copy measurement was measured using TaqManTM Copy Number Assay. Results: Our findings showed 22 (44%) samples with no changes in the copy number of CDH4 gene. Interestingly, 21 (42%) cases had an amplification of the CDH4 gene or CDH4 gene gain, and seven (14%)  cases decreased in CDH4 gene expression or CDH4 gene loss. We found an association between changes in the CDH4 gene and gender (p=0.001). However, there was no association between changes in the CDH4 gene and age (p=0.979), tumor location (p=0.145) and differentiation (p=0.648), the frequency of BRAF (p=0.171) and KRAS mutations (p=0.184) and MSI status (p=0.923). Conclusions: Copy number alteration in CDH4 gene in CRC patients and this alteration is significantly associated with gender. Further studies with a larger number of samples are essential to confirm this result and to identify the cause of CDH4 copy number alteration and its biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/069088e6da87904889fd1ecfbf85b7ca30e5a1c7" target='_blank'>
              Identification of CDH4 Gene Copy Number Alteration and Its Association with Clinical Profile of Colorectal Cancer Patient
              </a>
            </td>
          <td>
            Ni Nyoman Ayu Dewi, A. Y. Rompis, I. M. Mulyawan, Ni Putu Ekawati, H. S. Sun
          </td>
          <td>2024-09-30</td>
          <td>Indonesian Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD. Methods Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration. Results In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment. Conclusion ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4ff74f564f7b2a57419b04c8a2f80af41275c3" target='_blank'>
              Multiregional transcriptomic profiling provides improved prognostic insight in localized non-small cell lung cancer
              </a>
            </td>
          <td>
            Chenyang Li, Thinh T. Nguyen, Jian-Rong Li, Xingzhi Song, J. Fujimoto, L. Little, Curtis Gumb, C. Chow, Ignacio I. Wistuba, Andrew P Futreal, Jianhua Zhang, Shawna M. Hubert, J. Heymach, Jia Wu, Christopher I Amos, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-10-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Impaired control of the G1/S checkpoint allows initiation of DNA replication under non-permissive conditions. Unscheduled S-phase entry is associated with DNA replication stress, demanding for other checkpoints or cellular pathways to maintain proliferation. Here, we uncovered a requirement for ADARp150 to sustain proliferation of G1/S-checkpoint-defective cells under growth-restricting conditions. Besides its well-established mRNA editing function in inversely oriented short interspersed nuclear elements (SINEs), we found ADARp150 to exert a critical function in mitosis. ADARp150 depletion resulted in tetraploidization, impeding cell proliferation in mitogen-deprived conditions. Mechanistically we show that ADAR1 depletion induced aberrant expression of Cyclin B3, which was causative for mitotic failure and whole-genome duplication. Finally, we find that also in vivo ADAR1-depletion-provoked tetraploidization hampers tumor outgrowth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d2bb8be1edc7659c2bc78816fc95a0c749bf23" target='_blank'>
              ADARp150 counteracts whole genome duplication
              </a>
            </td>
          <td>
            Frank van Gemert, Alexandra Drakaki, Isabel Morales Lozano, Daniel de Groot, Maud Schoot Uiterkamp, N. Proost, C. Lieftink, M. van de Ven, R. Beijersbergen, Heinz Jacobs, H. te Riele
          </td>
          <td>2024-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c42616846e496900baa7e96476fa73f42799ad" target='_blank'>
              APC mutations dysregulate alternative polyadenylation in cancer
              </a>
            </td>
          <td>
            Austin M. Gabel, Andrea E. Belleville, James D Thomas, Jose Mario Bello Pineda, Robert K Bradley
          </td>
          <td>2024-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endogenous retroelements (EREs), which comprise half of the human genome, play a pivotal role in genome dynamics. Some EREs retained the ability to encode proteins, although most degenerated or served as a source for novel genes and regulatory elements during evolution. Despite ERE repression mechanisms developed to maintain genome stability, widespread pervasive ERE activation is observed in cancer including hematological malignancies. Challenging the perception of noncoding DNA as "junk," EREs are underestimated contributors to cancer driver mechanisms as well as antitumoral immunity by providing innate immune ligands and tumor antigens. This review highlights recent progress in understanding ERE co-option events in cancer and focuses on the controversial debate surrounding their causal role in shaping malignant phenotype. We provide insights into the rapidly evolving landscape of ERE research in hematological malignancies and their clinical implications in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70dce85bfb8495c63680f246f15584cd08410198" target='_blank'>
              Endogenous retroelements in hematological malignancies: From epigenetic dysregulation to therapeutic targeting.
              </a>
            </td>
          <td>
            Mohamed Chour, Françoise Porteu, S. Depil, V. Alcazer
          </td>
          <td>2024-10-10</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eba36ecae0bb22585b48631895c433e55a25862" target='_blank'>
              Nucleolus and centromere Tyramide Signal Amplification-Seq reveals variable localization of heterochromatin in different cell types
              </a>
            </td>
          <td>
            Pradeep Kumar, Omid Gholamalamdari, Yang Zhang, Liguo Zhang, Anastassiia Vertii, Tom van Schaik, Daan Peric-Hupkes, Takayo Sasaki, David M. Gilbert, B. van Steensel, Jian Ma, Paul D. Kaufman, A. Belmont
          </td>
          <td>2024-09-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3335db864e73ace960f7e153ca0a3871e879b0c6" target='_blank'>
              Hyper-recombination in ribosomal DNA is driven by long-range resection-independent RAD51 accumulation
              </a>
            </td>
          <td>
            Zita Gál, Stavroula Boukoura, Kezia Catharina Oxe, Sara Badawi, Blanca Nieto, L. Korsholm, Sille Blangstrup Geisler, E. Dulina, Anna Vestergaard Rasmussen, Christina Dahl, Wei Lv, Huixin Xu, Xiaoguang Pan, Stefanos Arampatzis, Danai-Eleni Stratou, P. Galanos, Lin Lin, Per Guldberg, Jiri Bartek, Yonglun Luo, D. H. Larsen
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromatin, with its complex spatial and temporal organization, plays a crucial role in regulating gene expression. Recent advancements in super-resolution microscopy have revealed that nanoscale domains of heterochromatin (repressed segments) embedded within a euchromatin (active segments) background are fundamental units of 3D chromatin organization. In tissue-resident cells, the size of these heterochromatin domains varies with the microenvironment, particularly its stiffness, and chromatin organization is also influenced by pharmacological and epigenetic drugs. However, the mechanisms governing heterochromatin domain size under various conditions and their impact on gene expression remain unclear. To address this knowledge gap, we have developed a dynamic, next-generation sequencing informed chromatin copolymer model. Our model simulates the spatiotemporal evolution of chromatin, driven by passive diffusion and active epigenetic reactions, which interconvert euchromatin and heterochromatin. By integrating chromatin-chromatin interaction energetics and diffusion-reaction dynamics, we predict the formation of nanoscale heterochromatin-rich domains and establish a scaling relationship between their size and the modulation of epigenetic reaction rates. Additionally, our model predicts that epigenetic and chromatin compaction changes in response to changes in global reaction rates occur predominantly at domain boundaries. We validated these predictions via Hi-C contact map analysis and super-resolution imaging of hyperacetylated melanoma cells. Subsequent RNA-seq analysis suggested a pivotal role of these epigenetic shifts in influencing the metastatic potential of these cells. We further validated our mesoscale findings against chromatin rearrangement in hMSCs, which exhibit sensitivity of epigenetic reaction rates to changes in microenvironmental stiffness. Finally, we evaluated the effects of cycling of epigenetic reaction rates in silico, mimicking the cellular transition to different extracellular conditions, and back again. This finding reveals a cell-type invariant mechanism driven by domain boundaries, whereby chromatin organization guides epigenetic memory formation. Our findings show that chromatin reorganization in response to changes in epigenetic reaction rates resulting from alterations in the microenvironment, drug exposure and disease progression impacts both immediate cellular responses and long-term epigenetic memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e55d9db8c8d7ff0628f2ac7f62c8baea4b57b" target='_blank'>
              Polymer Model Integrates Super-Resolution Imaging and Epigenomic Sequencing to Elucidate the Role of Epigenetic Reactions in Shaping 4D Chromatin Organization
              </a>
            </td>
          <td>
            Vinayak Vinayak, Ramin Basir, Rosela Golloshi, Joshua Toth, Lucas E. Sant’Anna, Melike Lakadamyali, R. McCord, Vivek B. Shenoy
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4728df16ec9a48de516d9a37941944d6293186" target='_blank'>
              Intragenic DNA inversions expand bacterial coding capacity.
              </a>
            </td>
          <td>
            Rachael B Chanin, Patrick T. West, Jakob Wirbel, Matthew O Gill, Gabriella Z M Green, Ryan M. Park, Nora J. Enright, Arjun M. Miklos, A. S. Hickey, Erin F Brooks, K. Lum, I. Cristea, A. Bhatt
          </td>
          <td>2024-09-25</td>
          <td>Nature</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454e4622c260f66b26c8cb18854b8ca092214e10" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be5f58320c6cf366bc684bebd0ed4433dc1746c" target='_blank'>
              Pramel15 facilitates zygotic nuclear DNMT1 degradation and DNA demethylation
              </a>
            </td>
          <td>
            Jiajun Tan, Yingfeng Li, Xiang Li, Xiaoxiao Zhu, Liping Liu, Hua Huang, Jiahua Wei, Hailin Wang, Yong Tian, Zhigao Wang, Zhuqiang Zhang, Bing Zhu
          </td>
          <td>2024-08-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary DNA-dependent protein kinase (DNA-PK) is a key player in repairing DNA damage. Ku proteins detect DNA damage and activate DNA-PK. Blocking DNA-PK can make cancer treatments such as radiation more effective. We have developed Ku–DNA binding inhibitors (Ku-DBis) that stop DNA-PK activation, making cancer cells more sensitive to radiation. We recently discovered new Ku-DBis that enter cells better than predecessor compounds, allowing cellular and in vivo analyses. Some non-small cell lung cancer (NSCLC) cells, especially those lacking ATM, were very sensitive to these inhibitors and radiation. However, cells lacking BRCA1 were resistant, which reduced the effectiveness of radiation. In animal studies of NSCLC, Ku-DBi treatment inhibited DNA-PK and enhanced a radiation-dependent decrease in tumor cell proliferation. This is the first time a Ku-targeted inhibitor has been shown to work in living organisms, suggesting their potential application in cancer treatment. Abstract Background: DNA-dependent protein kinase (DNA-PK) is a validated cancer therapeutic target involved in DNA damage response (DDR) and non-homologous end-joining (NHEJ) repair of DNA double-strand breaks (DSBs). Ku serves as a sensor of DSBs by binding to DNA ends and activating DNA-PK. Inhibition of DNA-PK is a common strategy to block DSB repair and improve efficacy of ionizing radiation (IR) therapy and radiomimetic drug therapies. We have previously developed Ku–DNA binding inhibitors (Ku-DBis) that block in vitro and cellular NHEJ activity, abrogate DNA-PK autophosphorylation, and potentiate cellular sensitivity to IR. Results and Conclusions: Here we report the discovery of oxindole Ku-DBis with improved cellular uptake and retained potent Ku-inhibitory activity. Variable monotherapy activity was observed in a panel of non-small cell lung cancer (NSCLC) cell lines, with ATM-null cells being the most sensitive and showing synergy with IR. BRCA1-deficient cells were resistant to single-agent treatment and antagonistic when combined with DSB-generating therapies. In vivo studies in an NSCLC xenograft model demonstrated that the Ku-DBi treatment blocked IR-dependent DNA-PKcs autophosphorylation, modulated DDR, and reduced tumor cell proliferation. This represents the first in vivo demonstration of a Ku-targeted DNA-binding inhibitor impacting IR response and highlights the potential therapeutic utility of Ku-DBis for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d24d7ccb0dfea0d53a93383e79bae41d2a8cf" target='_blank'>
              Impact of Optimized Ku–DNA Binding Inhibitors on the Cellular and In Vivo DNA Damage Response
              </a>
            </td>
          <td>
            Pamela L. Mendoza-Munoz, N. Kushwaha, Dineshsinha Chauhan, Karim Ben Ali Gacem, Joy E. Garrett, J. Dynlacht, J. Charbonnier, Navnath S. Gavande, J. Turchi
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are critical for the efficacy of radiotherapy as they lead to cell death if not repaired. DSBs caused by ionizing radiation (IR) initiate histone modifications and accumulate DNA repair proteins, including 53BP1, which forms distinct foci at damage sites and serves as a marker for DSBs. DSB repair primarily occurs through Non-Homologous End Joining (NHEJ) and Homologous Recombination (HR). NHEJ directly ligates DNA ends, employing proteins such as DNA-PKcs, while HR, involving proteins such as Rad54, uses a sister chromatid template for accurate repair and functions in the S and G2 phases of the cell cycle. Both pathways are crucial, as illustrated by the IR sensitivity in cells lacking DNA-PKcs or Rad54. We generated mouse embryonic stem (mES) cells which are knockout (KO) for DNA-PKcs and Rad54 to explore the combined role of HR and NHEJ in DSB repair. We found that cells lacking both DNA-PKcs and Rad54 are hypersensitive to X-ray radiation, coinciding with impaired 53BP1 focus resolution and a more persistent G2 phase cell cycle block. Additionally, mES cells deficient in DNA-PKcs or both DNA-PKcs and Rad54 exhibit an increased nuclear size approximately 18–24 h post-irradiation. To further explore the role of Rad54 in the absence of DNA-PKcs, we generated DNA-PKcs KO mES cells expressing GFP-tagged wild-type (WT) or ATPase-defective Rad54 to track the Rad54 foci over time post-irradiation. Cells lacking DNA-PKcs and expressing ATPase-defective Rad54 exhibited a similar phenotypic response to IR as those lacking both DNA-PKcs and Rad54. Despite a strong G2 phase arrest, live-cell imaging showed these cells eventually progress through mitosis, forming micronuclei. Additionally, mES cells lacking DNA-PKcs showed increased Rad54 foci over time post-irradiation, indicating an enhanced reliance on HR for DSB repair without DNA-PKcs. Our findings underscore the essential roles of HR and NHEJ in maintaining genomic stability post-IR in mES cells. The interplay between these pathways is crucial for effective DSB repair and cell cycle progression, highlighting potential targets for enhancing radiotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4800ec280608dc95677441accad3c221afaf2bd" target='_blank'>
              Synergistic Roles of Non-Homologous End Joining and Homologous Recombination in Repair of Ionizing Radiation-Induced DNA Double Strand Breaks in Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Gerarda van de Kamp, Tim Heemskerk, Roland Kanaar, Jeroen Essers
          </td>
          <td>2024-08-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human stem cells rely on enhanced DNA repair mechanisms to safeguard their ability to replenish somatic tissues. Telomerase counteracts telomere shortening and is a component of the stem cell DNA repair system that is regulated by ATM and ATR kinases. Here, we report that the DNA repair enzyme APEX2, but not its close paralog APEX1, is required for efficient telomerase reverse transcriptase (TERT) gene expression in human embryonic stem cells (hESC) and a melanoma cell line. We also observed that APEX2 knockdown significantly diminished telomerase enzyme activity. While APEX1 is known to regulate certain transcription factors, APEX2 has not been reported to influence gene expression. To gain insight into how APEX2 influences gene expression, we conducted RNA-seq following APEX2 knockdown in hESC. These results indicated that a number of genes, in addition to TERT, relied on APEX2 for efficient expression. Genes affected by APEX2 knockdown were significantly enriched for specific repetitive DNA families. These include mammalian-wide interspersed repeats (MIRs) and Alu elements. Chromatin immunoprecipitation experiments demonstrated the highest APEX2 binding near MIR sequences in TERT intron 2. Surprisingly, binding was low in the TERT proximal promoter, a region known to control TERT transcription. MIR and other repetitive DNA regions are common sites of DNA damage, suggesting that APEX2 recruitment and repair of TERT MIR sequences may play a role in influencing TERT expression. This new role for APEX2 in promoting efficient gene expression deepens our understanding of an emerging cancer therapeutic target. Further, as the TERT gene plays critical roles in stem cell maintenance, organismal development and aging, as well as in short telomere disorders and cancer, our observations provide insight into new strategies to modulate the expression of this important enzyme.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c18f456e9f9ac926ada6d6d91f9b7e89f672dbdc" target='_blank'>
              Apurinic/apyrimidinic endodeoxyribonuclease 2 (APEX2/APE2) is required for efficient expression of TERT in human embryonic stem cells
              </a>
            </td>
          <td>
            Josh L. Stern, Lindsay F. Rizzardi, Natalie R. Gassman
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The DNA damage response (DDR) constitutes a vital cellular process that safeguards genome integrity. This biological process involves substantial alterations in chromatin structure, commonly orchestrated by epigenetic enzymes. Here, we show that the epigenetic modifier N-terminal acetyltransferase 4 (Nat4), known to acetylate the alpha-amino group of serine 1 on histones H4 and H2A, is implicated in the response to DNA damage in S. cerevisiae. Initially, we demonstrate that yeast cells lacking Nat4 have an increased sensitivity to DNA damage and accumulate more DNA breaks than wild-type cells. Accordingly, upon DNA damage, NAT4 gene expression is elevated, and the enzyme is specifically recruited at double-strand breaks. Delving deeper into its effects on the DNA damage signaling cascade, nat4-deleted cells exhibit lower levels of the damage-induced modification H2AS129ph (γH2A), accompanied by diminished binding of the checkpoint control protein Rad9 surrounding the double-strand break. Consistently, Mec1 kinase recruitment at double-strand breaks, critical for H2AS129ph deposition and Rad9 retention, is significantly impaired in nat4Δ cells. Consequently, Mec1-dependent phosphorylation of downstream effector kinase Rad53, indicative of DNA damage checkpoint activation, is reduced. Importantly, we found that the effects of Nat4 in regulating the checkpoint signaling cascade are mediated by its N-terminal acetyltransferase activity targeted specifically towards histone H4. Overall, this study points towards a novel functional link between histone N-terminal acetyltransferase Nat4 and the DDR, associating a new histone-modifying activity in the maintenance of genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b98f7886cb4a1ffd1be67a17caa54c6264f441" target='_blank'>
              Yeast Nat4 regulates DNA damage checkpoint signaling through its N-terminal acetyltransferase activity on histone H4
              </a>
            </td>
          <td>
            Mamantia Constantinou, Evelina Charidemou, Izge Shanlitourk, K. Strati, A. Kirmizis
          </td>
          <td>2024-10-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Topoisomerases alter DNA topology by making transient DNA strand breaks (DSBs) in DNA. The DNA cleavage reaction mechanism includes the formation of a reversible protein/DNA complex that allows rapid resealing of the transient break. This mechanism allows changes in DNA topology with minimal risks of persistent DNA damage. Nonetheless, small molecules, alternate DNA structures, or mutations in topoisomerase proteins can impede the resealing of the transient breaks, leading to genome instability and potentially cell death. The consequences of high levels of enzyme/DNA adducts differ for type I and type II topoisomerases. Top1 action on DNA containing ribonucleotides leads to 2–5 nucleotide deletions in repeated sequences, while mutant Top1 enzymes can generate large deletions. By contrast, small molecules that target Top2, or mutant Top2 enzymes with elevated levels of cleavage lead to small de novo duplications. Both Top1 and Top2 have the potential to generate large rearrangements and translocations. Thus, genome instability due to topoisomerase mis-function is a potential pathogenic mechanism especially leading to oncogenic progression. Recent studies support the potential roles of topoisomerases in genetic changes in cancer cells, highlighting the need to understand how cells limit genome instability induced by topoisomerases. This review highlights recent studies that bear on these questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc620e1f3abd707db6dd8102f3cc18d304a5ef7" target='_blank'>
              Genome Instability Induced by Topoisomerase Misfunction
              </a>
            </td>
          <td>
            K. C. Nitiss, Afif F. Bandak, James M. Berger, J. Nitiss
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac0738a96662e313aea5fd9200de2f16be79bfc" target='_blank'>
              R-loops’ m6A modification and its roles in cancers
              </a>
            </td>
          <td>
            Yue Qiu, Changfeng Man, Luyu Zhu, Shiqi Zhang, Xiaoyan Wang, Dandan Gong, Yu Fan
          </td>
          <td>2024-10-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy.
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, T. Chen, Jie Lei, Changwen Chen, Shun-qin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="C10orf90, a tumor suppressor, can inhibit the occurrence and development of tumors. Therefore, we investigated the gene function of C10orf90 in various tumors using multiple pan-cancer datasets. Pan-cancer analysis results reveal that the expression levels of C10orf90 vary across different tumors and hold significant value in the clinical diagnosis and prognosis of patients with various tumors. In some cancers, the expression level of C10orf90 is correlated with CNV, DNA methylation, immune subtypes, immune cell infiltration, and drug sensitivity in the tumors. In particular, in COAD, the C10orf90 gene is implicated in multiple processes associated with COAD. Cell experiments demonstrate that C10orf90 suppresses the proliferation and migration of colon cancer cells while promoting apoptosis. In summary, C10orf90 plays a role in the onset and progression of various cancers and could potentially serve as an effective diagnostic and prognostic marker for cancer patients. Notably, in COAD, C10orf90 inhibits the proliferation and migration of colon cancer cells, induces apoptosis, and is linked to the advancement of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1559c858c67c63e394ff32e5068255c695dededf" target='_blank'>
              Novel Oncogenic Value of C10orf90 in Colon Cancer Identified as a Clinical Diagnostic and Prognostic Marker
              </a>
            </td>
          <td>
            Chuangdong Ruan, Yuqin Zhang, Daoyang Chen, Mengyi Zhu, Penghui Yang, Rongxin Zhang, Yan Li
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA topoisomerase II (TOP2) is an enzyme that resolves DNA topological problems arising in various nuclear processes, such as transcription. Aclarubicin, a member of the anthracyclines, is known to prevent the association of TOP2 with DNA, inhibiting the early step of TOP2 catalytic reactions. During our research on the subnuclear distribution of human TOP2B, we found that aclarubicin affects the mobility of TOP2B in the nucleus. FRAP analysis demonstrated that aclarubicin decreased the nuclear mobility of EGFP-tagged TOP2B in a concentration-dependent manner. Aclarubicin exerted its inhibitory effects independently of TOP2B enzymatic activities: TOP2B mutants defective for either ATPase or topoisomerase activity also exhibited reduced nuclear mobility in the presence of aclarubicin. Immunofluorescence analysis showed that aclarubicin antagonized the induction of DNA damage by etoposide. Although the prevention of the TOP2-DNA association is generally considered a primary action of aclarubicin in TOP2 inhibition, our findings highlight a previously unanticipated effect of aclarubicin on TOP2B in the cellular environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0224dd55993d9d812699857cf7d079f645a3e5" target='_blank'>
              Aclarubicin Reduces the Nuclear Mobility of Human DNA Topoisomerase IIβ
              </a>
            </td>
          <td>
            Keiko Morotomi‐Yano, Ken-ichi Yano
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Long Interspersed Nuclear Element-1 (LINE-1) is an autonomous retrotransposon that makes up a significant portion of the human genome, contributing to genetic diversity and genome evolution. LINE-1 encodes two proteins, ORF1p and ORF2p, both essential for successful retrotransposition. ORF2p has endonuclease and reverse transcription activity, while ORF1p binds RNA. Many copies of ORF1p assemble onto the LINE-1 RNA to form a ribonucleoprotein complex (RNP). However, the functional role of RNPs in the LINE-1 life cycle is unclear. Using reconstitution assays on DNA curtains, we show that L1 RNPs gain DNA binding activity only when RNA is super-saturated with ORF1p. In cells, L1 RNPs bind to chromosomes during mitosis. Mutational analysis reveals that DNA binding is crucial for nuclear entry and LINE-1 retrotransposition activity. Thus, a key function of ORF1p is to form an RNP that gains access to the genome through DNA binding upon nuclear envelope breakdown.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e74dde49305338c91091f85bb6d6995fffb8559" target='_blank'>
              LINE-1 ribonucleoprotein complexes bind DNA to enable nuclear entry during mitosis
              </a>
            </td>
          <td>
            Sarah Zernia, Farida Ettefa, Srinjoy Sil, Cas Koeman, Joëlle Deplazes-Lauber, Marvin Freitag, Liam J. Holt, Johannes Stigler
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The p53 tumor suppressor is an indispensable regulator of DNA damage responses that accelerates carcinogenesis when mutated. In this report, we uncover a new mechanism by which p53 maintains genomic integrity in the absence of canonical DNA damage response activation. Specifically, loss of p53 dramatically alters chromatin structure at the nuclear periphery, allowing increased transmission of an environmental carcinogen, ultraviolet (UV) radiation, into the nucleus. Genome-wide mapping of UV-induced DNA lesions in p53-deficient primary cells reveals elevated lesion abundance in regions corresponding to locations of high mutation burden in malignant melanomas. These findings uncover a novel role of p53 in the suppression of mutations that contribute to cancer and highlight the critical influence of nuclear architecture in regulating sensitivity to carcinogens. One-Sentence Summary The p53 tumor suppressor reduces carcinogen sensitivity and mutagenic potential by maintaining nuclear architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce95350c5fb375408fd587e0eea1b652005b9ec" target='_blank'>
              p53 Regulates Nuclear Architecture to Reduce Carcinogen Sensitivity and Mutagenic Potential
              </a>
            </td>
          <td>
            Devin A. King, Dakota E. McCoy, Adrian Perdyan, Jakub Mieczkowski, Thierry Douki, Jennifer A. Dionne, Rafael E. Herrera, Ashby J. Morrison
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Ataxia telangiectasia and Rad3-related protein (ATR) is a key DNA damage response protein that facilitates DNA damage repair and regulates cell cycle progression. As such, ATR is an important component of the cellular response to radiation, particularly in cancer cells, which show altered DNA damage response and aberrant cell cycle checkpoints. Therefore, ATR’s pharmacological inhibition could be an effective radiosensitization strategy to improve radiotherapy. We assessed the ability of an ATR inhibitor, AZD6738, to sensitize cancer cell lines of various histologic types to photon and proton radiotherapy. We found that radiosensitization took place through persistent DNA damage and abrogated G2 cell cycle arrest. We also found that AZD6738 increased the number of micronuclei after exposure to radiotherapy. We found that combining radiation with AZD6738 led to tumor growth delay and prolonged survival relative to radiation alone in a breast cancer model. Combining AZD6738 with photons or protons also led to increased macrophage infiltration at the tumor microenvironment. These results provide a rationale for further investigation of ATR inhibition in combination with radiotherapy and with other agents such as immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0ba0de680ef188bcc8f9e29b419411fda7ecce4" target='_blank'>
              ATR inhibition radiosensitizes cells through augmented DNA damage and G2 cell cycle arrest abrogation
              </a>
            </td>
          <td>
            Scott J. Bright, Mandira Manandhar, D. Flint, Rishab Kolachina, Mariam Ben Kacem, D. Martinus, B. Turner, Ilsa Qureshi, C. McFadden, Poliana Camila Marinello, Simona F Shaitelman, G. Sawakuchi
          </td>
          <td>2024-09-03</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e0df7d22a45444a437c740c4db84a4c490aa49" target='_blank'>
              Membrane RRM2-positive cells represent a malignant population with cancer stem cell features in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yongzhi Zhao, Shuting Xue, Danduo Wei, Jianjuan Zhang, Nachuan Zhang, Liping Mao, Niya Liu, Lei Zhao, Jianing Yan, Yifan Wang, Xiujun Cai, Saiyong Zhu, Stephanie Roessler, Junfang Ji
          </td>
          <td>2024-09-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved tool for the removal of DNA double-strand breaks (DSBs) and the preservation of stalled and damaged DNA replication forks. Successful completion of HR requires the tumor suppressor BRCA2. Germline mutations in BRCA2 lead to familial breast, ovarian, and other cancers, underscoring the importance of this protein for maintaining genome stability. BRCA2 harbors two distinct DNA binding domains, one that possesses three oligonucleotide/oligosaccharide binding (OB) folds (known as the OB-DBD), and with the other residing in the C-terminal recombinase binding domain (termed the CTRB-DBD) encoded by the last gene exon. Here, we employ a combination of genetic, biochemical, and cellular approaches to delineate contributions of these two DNA binding domains toward HR and the maintenance of stressed DNA replication forks. We show that OB-DBD and CTRB-DBD confer ssDNA and dsDNA binding capabilities to BRCA2, respectively, and that BRCA2 variants mutated in either DNA binding domain are impaired in the ability to load the recombinase RAD51 onto ssDNA pre-occupied by RPA. While the CTRB-DBD mutant is modestly affected for HR, it exhibits a strong defect in the protection of stressed replication forks. In contrast, the OB-DBD is indispensable for both BRCA2 functions. Our study thus defines the unique contributions of the two BRCA2 DNA binding domains in genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b867a23dce384adad0b091d20596685ac211bb92" target='_blank'>
              Distinct roles of the two BRCA2 DNA binding domains in DNA damage repair and replication fork preservation
              </a>
            </td>
          <td>
            Francisco Neal, Wenjing Li, Mollie E. Uhrig, Neelam Sharma, Shahrez Syed, Sandeep Burma, R. Hromas, Alexander V. Mazin, Eloise Dray, David S. Libich, Shaun K Olsen, Elizabeth V. Wasmuth, Weixing Zhao, Claus S. Sørensen, C. Wiese, Youngho Kwon, Patrick Sung
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome-containing micronuclei are a hallmark of aggressive cancers. Micronuclei frequently undergo irreversible collapse, exposing their enclosed chromatin to the cytosol. Micronuclear rupture catalyzes chromosomal rearrangements, epigenetic abnormalities, and inflammation, yet mechanisms safeguarding micronuclear integrity are poorly understood. In this study, we found that mitochondria-derived reactive oxygen species (ROS) disrupt micronuclei by promoting a noncanonical function of charged multivesicular body protein 7 (CHMP7), a scaffolding protein for the membrane repair complex known as endosomal sorting complex required for transport III (ESCRT-III). ROS retained CHMP7 in micronuclei while disrupting its interaction with other ESCRT-III components. ROS-induced cysteine oxidation stimulated CHMP7 oligomerization and binding to the nuclear membrane protein LEMD2, disrupting micronuclear envelopes. Furthermore, this ROS-CHMP7 pathological axis engendered chromosome shattering known to result from micronuclear rupture. It also mediated micronuclear disintegrity under hypoxic conditions, linking tumor hypoxia with downstream processes driving cancer progression. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION Chromosomal instability, a hallmark of aggressive cancers, is characterized by the presence of micronuclei, which are cytosolic rupture-prone structures that contain entire chromosomes or chromosome arms. The irreversible collapse of micronuclear envelopes is a central event in tumor progression. Micronuclear collapse exposes the encapsulated DNA to the cytosol, catalyzing chromosomal rearrangements and epigenetic alterations that drive tumor heterogeneity as well as therapy resistance. Micronuclear rupture also activates inflammatory signaling pathways that reshape the tumor immune microenvironment, promoting metastasis. Despite its importance, the underlying mechanisms of micronuclear collapse are unclear. RATIONALE Maintaining the integrity of the nuclear membrane is essential for cellular functions and organismal viability. It is unclear what cellular safeguards are compromised in micronuclei that lead to the breakdown of nuclear membrane integrity. We posited that inherent dissimilarities between micronuclei and primary nuclei might explain their respective differences in membrane stability. For instance, micronuclei are 5- to 20-fold smaller than the primary nucleus. Secondly, they possess abnormal nuclear envelopes, as evidenced by aberrant nuclear pore function and reduced lamin B1 levels. In addition, once ruptured, micronuclear envelopes are seldom repaired. RESULTS In this study, we integrated prior observations with our findings that micronuclear envelope rupture occurs because of the aberrant interaction between micronuclei and mitochondria, a process that is mediated by reactive oxygen species (ROS). Indeed, ruptured micronuclei were more likely to be intermixed with the mitochondrial network compared with intact ones, and manipulation of ROS levels correspondingly altered the rates of micronuclear rupture. Subsequent proteomic analysis and genetic manipulations led us to the observation that ROS interfere with micronuclear integrity by disrupting the normal activity of the nuclear membrane repair complex known as the endosomal sorting complex required for transport III (ESCRT-III). Increases in ROS levels led to the micronuclear accumulation of the ESCRT-III scaffolding protein, charged multivesicular body protein 7 (CHMP7), promoting a noncanonical function. ROS inhibited the export of CHMP7, leading to its persistence in the micronucleus and aberrant binding to its inner nuclear membrane partner LEM domain nuclear envelope protein 2 (LEMD2). ROS-induced cysteine oxidation catalyzed CHMP7 aggregation and reduced its interaction with its canonical binding partners in the ESCRT-III complex. The binding of CHMP7 aggregates to LEMD2 promoted micronuclear membrane deformation and collapse. This effect was further exacerbated by ROS-dependent recruitment of the autophagy-related protein p62, which mediated the degradation of canonical ESCRT-III members, limiting any chances of subsequent membrane repair. Lastly, we found that this mechanism is relevant to human tumors. Elevated levels of ROS and aberrant CHMP7 function led to complex chromosomal rearrangements, or chromothripsis, known to arise from micronuclear rupture. Furthermore, ROS generated under hypoxia induced micronuclear rupture in a CHMP7-dependent manner. In line with these findings, we observed a significant increase in micronuclear rupture along with CHMP7 accumulation in hypoxic regions of human head and neck cancer as well as in ovarian tumors. CONCLUSION Our results have uncovered a pathological interaction between micronuclei and mitochondria that underlies micronuclear rupture. By implicating ROS as regulators of micronuclear integrity, this work provides a mechanistic link between ROS-inducing conditions (such as free-radical generation and hypoxia) and downstream processes known to arise from the presence of micronuclei, including chromosomal rearrangements, epigenetic dysregulation, and tumor-promoting inflammation. Mechanisms of micronuclear rupture. The proximity of mitochondria to micronuclei drives micronuclear membrane rupture through mitochondria-derived reactive oxygen species (ROS). ROS inhibit micronuclear export, leading to the excessive accumulation of CHMP7, a scaffolding protein associated with the nuclear membrane repair complex ESCRT-III. ROS-dependent cysteine oxidation promotes CHMP7 self-aggregation and its aberrant binding to the membrane protein LEMD2, causing micronuclear collapse. [Figure created with BioRender.com]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1748efac80bee30de3b1de5b76ab1af0bb5d42d" target='_blank'>
              Micronuclear collapse from oxidative damage
              </a>
            </td>
          <td>
            Melody Di Bona, Yanyang Chen, Albert S. Agustinus, Alice Mazzagatti, Mercedes A. Duran, Matthew Deyell, Daniel Bronder, James Hickling, Christy Hong, Lorenzo Scipioni, Giulia Tedeschi, Sara Martin, Jun Li, Aušrinė Ruzgaitė, N. Riaz, Parin Shah, Edridge K. D’Souza, D. Z. Brodtman, Simone Sidoli, B.H. Diplas, Manisha Jalan, Nancy Y Lee, Alban Ordureau, Benjamin Izar, A. Laughney, Simon N. Powell, Enrico Gratton, S. Santaguida, J. Maciejowski, Peter Ly, Thomas M. Jeitner, S. Bakhoum
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>4</td>
          <td>62</td>
        </tr>

        <tr id="Simple Summary Mitochondria are important organelles in the human body, and they act as a source of chemical energy for different cellular processes. Multiple copies of mitochondria are normally detected in cells, and their copy number has also been reported to be aberrated in cancers, including liver cancer. Target-panel sequencing is frequently used in oncology studies, but the optimal pooling ratio of nuclear and mitochondrial probes has remained elusive. Therefore, it is necessary to determine the physiological mitochondrial DNA copy number in non-liver cancer subjects, which can serve as a useful reference for deciding the desirable ratio of nuclear and mitochondrial probes for a less biased target enrichment of nuclear and mitochondrial genomes and hence their even representation. In addition, it can also be used as a physiological basis that can distinguish normal physiological variations from the pathological conditions in liver cancer. Abstract Mitochondria generate energy to support cells. They are important organelles that engage in key biological pathways. The dysfunction of mitochondria can be linked to hepatocarcinogenesis, which has been actively explored in recent years. To investigate the mitochondrial dysfunction caused by genetic variations, target-panel sequencing is a flexible and promising strategy. However, the copy number of mitochondria generally exceeds nuclear DNA, which raises a concern that uneven target enrichment of mitochondrial DNA (mtDNA) and nuclear DNA (ncDNA) in target-panel sequencing would lead to an undesirably biased representation of them. To resolve this issue, we evaluated the optimal pooling of mtDNA probes and ncDNA probes by a series of dilutions of mtDNA probes in both genomic DNA (gDNA) and cell-free DNA (cfDNA) samples. The evaluation was based on read count, average sequencing depth and coverage of targeted regions. We determined that an mtDNA:ncDNA probe ratio of around 1:10 would offer a good balance of sequencing performance and cost effectiveness. Moreover, we estimated the median physiological mtDNA:ncDNA copy ratio as 38.1 and 2.9 in cfDNA and gDNA samples of non-liver cancer subjects, respectively, whereas they were 20.0 and 2.1 in the liver cancer patients. Taken together, this study revealed the appropriate pooling strategy of mtDNA probes and ncDNA probes in target-panel sequencing and suggested the normal range of physiological variation of the mtDNA:ncDNA copy ratio in non-liver cancer individuals. This can serve as a useful reference for future target-panel sequencing investigations of the mitochondrial genome in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57399731b5753fe08b7b9f7c5d0be77ea77fed4d" target='_blank'>
              Resolution of Optimal Mitochondrial and Nuclear DNA Enrichment in Target-Panel Sequencing and Physiological Mitochondrial DNA Copy Number Estimation in Liver Cancer and Non-Liver Cancer Subjects
              </a>
            </td>
          <td>
            Xueying Lyu, Y. Tsui, Ivan Ka-Kit Tam, Po-Man Li, Gary Cheuk-Hang Cheung, J. Lee, I. Ng, D. Ho
          </td>
          <td>2024-08-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is decisive for genetic stability. DSB repair act according to two steps: 1-the choice between nonhomologous end-joining (NHEJ) versus resection of DNA ends, generating single-stranded DNA (ssDNA); 2-on ssDNA, homologous recombination (HR) and sub-processes. Here we show that 53BP1, which plays a prime role at the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of the cAMP dependant protein kinase A (PKAcs). PKA favors the recruitment of 53BP1 at DNA damages sites, and consistently, prevents resection, favoring NHEJ. Inhibition of PKA stimulated resection and coherently reduces NHEJ. Reversely, activation of PKA with 8-Bromo-cAMP stimulate NHEJ and reduces HR. These data open new avenues for potential anticancer strategies. More generally, they underline the high complexity of the regulation of DSB repair, identifying PKA as a novel actor of the DNA damage response, acting at the choice of the DSB repair process, which might be essential for genome integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb615918d35bbed71f2c3e0b64876e126043052" target='_blank'>
              PKA prevents resection of DNA double strand breaks and favor non-homologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, I. Magdalou, Vasily Ogryzko, Bernard S. Lopez
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA lies at the heart of the central dogma of life. Altering DNA can modify the flow of information in fundamental cellular processes such as transcription and translation. The ability to precisely manipulate DNA has led to remarkable advances in treating incurable human genetic ailments and has changed the landscape of biological research. Genome editors such as CRISPR-Cas nucleases and TALENs have become ubiquitous tools in basic and applied biological research and have been translated to the clinic to treat patients. The specificity and modularity of these genome editors have made it possible to efficiently engineer genomic DNA; however, underlying principles governing editing outcomes in eukaryotes are still being uncovered. Editing efficiency can vary from cell type to cell type for the same DNA target sequence, necessitating de novo design and validation efforts. Chromatin structure and epigenetic modifications have been shown to affect the activity of genome editors because of the role they play in hierarchical organization of the underlying DNA. Understanding the nuclear search mechanism of genome editors and their molecular interactions with higher order chromatin will lead to improved models for predicting precise genome editing outcomes. Insights from such studies will unlock the entire genome to be engineered for the creation of novel therapies to treat critical illnesses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de15652cba672686fe241fc8c1aaf364c6e3c7f6" target='_blank'>
              Impact of Chromatin Organization and Epigenetics on CRISPR-Cas and TALEN Genome Editing.
              </a>
            </td>
          <td>
            Surbhi Jain, Guanhua Xun, Huimin Zhao
          </td>
          <td>2024-09-24</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>155</td>
        </tr>

        <tr id="Anillin (ANLN) is a cytoskeletal binding protein involved in mitosis. ANLN is located in the nucleus during interphase and in the cytoplasmic contractile rings during mitosis. Our previous studies found that ANLN is abnormally overexpressed in esophageal squamous cell carcinoma (ESCC), promoting cell division by regulating contractile ring localization. However, the exact function of ANLN in the nucleus remains unclear. Here, we found that the expression of ANLN in the nucleus is associated with poor prognosis in ESCC patients, rather than in the cytoplasm. Protein mass spectrometry and bioinformatics analysis revealed that ANLN is related to DNA synthesis, and proliferating cell nuclear antigen (PCNA) is found to be a potential interacting protein of ANLN. PCNA directly interacts with the PIP box domain of ANLN and co-localizes in the nucleus. ANLN promotes DNA replication and S phase progression in a PCNA dependent manner and independent with the cytoskeletal function of ANLN. Importantly, ANLN is involved in transletion synthesis (TLS), a type of DNA synthesis under stress, by promoting PCNA monoubiquitination at K164 residue. Mechanistically, ANLN binds and recruits the E3 ligase RAD18 to promote PCNA monoubiquitination and DNA polymerase eta loading under UV radiation conditions. Consistently, depletion of ANLN leads to increased genomic instability and increased sensitivity to UV radiation. The findings of the study showed that ANLN in the nucleus as a protein scaffold is involved in UV induced DNA synthesis pathway, providing new insights into the function and mechanism of ANLN in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd15e0ed7760b725834190cbadc37e8a4fb6ead5" target='_blank'>
              ANLN directly interacts with PCNA to regulate UV induced translesion synthesis
              </a>
            </td>
          <td>
            Bei-Bei Tong, Yu-Fei Cao, Bing Wen, Teng Fu, Danxia Deng, Qian-Hui Yang, Yu-Qiu Wu, Hua-Yan Zou, Lian-di Liao, Liyan Xu, E. Li
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Abstract Activating FLT3 and RAS mutations commonly occur in leukemia with KMT2A‐gene rearrangements (KMT2A‐r). However, how these mutations cooperate with the KMT2A‐r to remodel the epigenetic landscape is unknown. Using a retroviral acute myeloid leukemia (AML) mouse model driven by KMT2A::MLLT3, we show that FLT3 ITD , FLT3 N676K , and NRAS G12D remodeled the chromatin accessibility landscape and associated transcriptional networks. Although the activating mutations shared a common core of chromatin changes, each mutation exhibits unique profiles with most opened peaks associating with enhancers in intronic or intergenic regions. Specifically, FLT3 N676K and NRAS G12D rewired similar chromatin and transcriptional networks, distinct from those mediated by FLT3 ITD . Motif analysis uncovered a role for the AP‐1 family of transcription factors in KMT2A::MLLT3 leukemia with FLT3 N676K and NRAS G12D , whereas Runx1 and Stat5a/Stat5b were active in the presence of FLT3 ITD . Furthermore, transcriptional programs linked to immune cell regulation were activated in KMT2A‐r AML expressing NRAS G12D or FLT3 N676K , and the expression of NKG2D‐ligands on KMT2A‐r cells rendered them sensitive to CAR T cell‐mediated killing. Human KMT2A‐r AML cells could be pharmacologically sensitized to NKG2D‐CAR T cells by treatment with the histone deacetylase inhibitor LBH589 (panobinostat) which caused upregulation of NKG2D‐ligand levels. Co‐treatment with LBH589 and NKG2D‐CAR T cells enabled robust AML cell killing, and the strongest effect was observed for cells expressing NRAS G12D . Finally, the results were validated and extended to acute leukemia in infancy. Combined, activating mutations induced mutation‐specific changes in the epigenetic landscape, leading to changes in transcriptional programs orchestrated by specific transcription factor networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad94e713ba8dadd871506c2d3b1dbcd4274d1950" target='_blank'>
              Activating mutations remodel the chromatin accessibility landscape to drive distinct regulatory networks in KMT2A‐rearranged acute leukemia
              </a>
            </td>
          <td>
            Qirui Zhang, Ton Falqués-Costa, Mattias Pilheden, Helena Sturesson, Tina Ovlund, Vendela Rissler, Anders Castor, Hanne V H Marquart, B. Lausen, T. Fioretos, Axel Hyrenius-Wittsten, Anna K. Hagström-Andersson
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K C Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K J Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Breast cancer is a major problem in the health sector worldwide. The prevalence of breast cancer incidence is on the rise in many countries despite efforts to eradicate it. The substantial efforts for the treatment of breast cancer are quite successful but the main hindrance is early detection which in turn is dependent on various factors i.e. genetic history, environmental exposures, hormonal causes, and sedentary lifestyle. In previous studies, lncRNAs were known to modulate the progression of many cancers like gastric, lung, ovarian, and colorectal cancers. This study was designed to find these lncRNAs role in breast cancer development. Bioinformatics tools were used for in silco analysis of these lncRNAs LINC01356, LINC01357, ARF4-AS1, and FAM83B. These lncRNAs were evaluated in GEPIA2 for subtype analysis and survival probability, Phylo CSF UCSC genome browser for conserved sequences and protein-coding potential, BcGenExMiner for correlation analysis and UALCAN for protein-coding gene expression. These above-mentioned lncRNAs showed high expression in the basal subtype of TNBC as compared with other subtypes and they showed poor survival rates in the basal subtype. LINC01356 and LINC01357 showed positive correlation with TAFA3 and SLC16A1. ARF4-AS1 positively correlated with ASB14 and FAM83B positively correlated with HCRTR2. These lncRNAs showing positive correlation with different genes have a role in breast cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10a7530434e87606ba2330c319f2c88880cdad0" target='_blank'>
              Long Non-coding RNA and Progression of Breast Cancer
              </a>
            </td>
          <td>
            Usama Ahmed, Muhammad Abubakar, Salma Saeed Khan, Baqaur Rehman
          </td>
          <td>2024-10-02</td>
          <td>Oncology Treatment Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks (DSBs) through homologous recombination (HR) is vital for maintaining the stability and integrity of the genome. RNA binding proteins (RBPs) intricately regulate the DNA damage repair process, yet the precise molecular mechanisms underlying their function remain incompletely understood. In this study, we highlight the pivotal role of RPS15, a representative RBP, in homologous recombination repair. Specifically, we demonstrate that RPS15 promotes DNA end resection, a crucial step in homologous recombination. Notably, we identify an interaction between RPS15 and CtIP, a key factor in homologous recombination repair. This interaction is essential for CtIP recruitment to DSB sites, subsequent RPA coating, and RAD51 replacement, all critical steps in efficient homologous recombination repair and conferring resistance to genotoxic treatments. Functionally, suppressing RPS15 expression sensitizes cancer cells to X-ray radiation and enhances the therapeutic synergistic effect of PARP1 inhibitors in breast cancer cells. In summary, our findings reveal that RPS15 promotes DNA end resection to ensure effective homologous recombination repair, suggesting its potential as a therapeutic target in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2aedc66ec12ddcad5319b720ce461d7c9233af5" target='_blank'>
              RPS15 Coordinates with CtIP to Facilitate Homologous Recombination and Confer Therapeutic Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Baohang Lin, Guan Huang, Zishan Yuan, Xun Peng, Chunliang Yu, Jialu Zheng, Zequn Li, Juanyun Li, Jinan Liang, Bo Xu
          </td>
          <td>2024-10-03</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e828a68b6f097a43f327c00f62a47eac243544" target='_blank'>
              SEPT9: From pan-cancer to lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenwen Wang, Xiaochen Zhang, Ping Gui, Qizhen Zou, Yuzhou Nie, Shenglin Ma, Shirong Zhang
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accurate monitoring of base excision repair (BER) activity in cancer cells is critical for advancing the comprehension of DNA repair processes, gaining insights into cancer development, and guiding treatment strategies. However, current assay techniques for assessing BER activity in cancer cells face challenges due to the heterogeneous origins and diversity of BER enzymes. In this work, we present a highly reliable triple loop-interlocked DNA codec (GATED) that enables precise assessment of BER activity in cancer cells through signal amplification mediated by multienzyme orthogonal activation. The GATED device features a dumbbell-shaped DNA probe to encode two BER enzymes for BER-related signal conversion as well as two bound circular DNA to decode the apurinic/apyrimidinic sites for apurinic/apyrimidinic endonuclease 1 (APE1)-mediated signal amplification. Importantly, GATED is orthogonally activated by multiple target BER enzymes (i.e., uracil DNA glycosylase, thymine DNA glycosylase, and APE1), resulting in a unified fluorescent signal that significantly improves the detection specificity and sensitivity to BER enzymes. Additionally, we demonstrate that the GATED has exceptional biostability within complex biological systems, where it was successfully employed to monitor BER activity in cancer cells with high specificity and enabled cell-based high-throughput screening for BER inhibitors. The GATED provides a much-needed tool for the real-time monitoring of BER activity and the screening of BER inhibitors in cancer cells, potentially advancing both the investigation and clinical application of BER biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4455a54c11e6664a89c2b008ca24acc5e5bb02" target='_blank'>
              Multienzymatic Orthogonal Activation of DNA Codec Enables Tumor-Specific Imaging of Base Excision Repair Activity.
              </a>
            </td>
          <td>
            Wei-Jia Zeng, Zhaopeng Chen, Yan-Mei Lei, W. Liang, Yaqin Chai, Ruo Yuan, Ying Zhuo
          </td>
          <td>2024-09-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A R Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In mammalian cells, repair centers for DNA double-strand breaks (DSBs) have been identified. However, previous researches predominantly rely on methods that induce specific DSBs by cutting particular DNA sequences. The clustering and its spatiotemporal properties of non-specifically DSBs, especially those induced by environmental stresses such as irradiation, remains unclear. In this study, we used Dragonfly microscopy to induce high-precision damage in cells and discovered that DSB clustering during the early stages of DNA damage response (DDR) and repair, but not during the repair plateau phase. Early in DDR, DSB clustered into existing 53BP1 foci. The DSB clustering at different stages has different implications for DNA repair. By controlling the distance between adjacent damage points, we found that the probability of DSB clustering remains constant at distances of 0.8 - 1.4 um, while clustering does not occur beyond 1.4 um. Within the 0.8 um range, the probability of clustering significantly increases due to the phase separation effect of 53BP1. Using a Monte Carlo approach, we developed a dynamic model of 53BP1 foci formation, fission, and fusion. This model accurately predicts experimental outcomes and further demonstrates the temporal and spatial influences on DSB clustering. These results showed that, similarly to specifically induced DSBs, non-specifically induced DSBs can also cluster. The extent of DSB clustering is influenced by both temporal and spatial factors, which provide new insights into the dynamics of DSB clustering and the role of 53BP1 in DNA repair processes. Such findings could enhance our understanding of DNA damage responses and help us improve DNA repair therapies in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a1ca8ef0d6b8c925946d87a4968b5bc72de6f8" target='_blank'>
              Double-Strand Break Clustering: An Economical and Effective Strategy for DNA Repair
              </a>
            </td>
          <td>
            Junyi Chen, Wenzong Ma, Yuqi Ma, Gen Yang
          </td>
          <td>2024-10-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, E. Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, L. Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA damage tolerance pathways that allow for the completion of replication following fork arrest are critical in maintaining genome stability during cell division. The main DNA damage tolerance pathways include strand switching, replication fork reversal and translesion synthesis (TLS). The TLS pathway is mediated by specialised DNA polymerases that can accommodate altered DNA structures during DNA synthesis, and are important in allowing replication to proceed after fork arrest, preventing fork collapse that can generate more deleterious double-strand breaks in the genome. TLS may occur directly at the fork, or at gaps remaining behind the fork, in the process of post-replication repair. Inactivating mutations in the human POLH gene encoding the Y-family DNA polymerase Pol η causes the skin cancer-prone genetic disease xeroderma pigmentosum variant (XPV). Pol η also contributes to chemoresistance during cancer treatment by bypassing DNA lesions induced by anti-cancer drugs including cisplatin. We review the current understanding of the canonical role of Pol η in translesion synthesis following replication arrest, as well as a number of emerging non-canonical roles of the protein in other aspects of DNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247363b13f865e5ca1882a279d7c40f1f96c114f" target='_blank'>
              Canonical and Non-Canonical Roles of Human DNA Polymerase η
              </a>
            </td>
          <td>
            Salma Bedaiwi, Anam Usmani, Michael P. Carty
          </td>
          <td>2024-09-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Extrachromosomal circular DNA (eccDNA) has emerged as a central focus in molecular biology, with various types being found across species through advanced techniques, including high-throughput sequencing. This dynamic molecule exerts significant influence on aging and immune function and plays pivotal roles in autoimmune diseases, type 2 diabetes mellitus, cancer, and genetic disorders.


SUMMARY
This comprehensive review investigates the classification, characteristics, formation processes, and multifaceted functions of eccDNA, providing an in-depth exploration of its mechanisms in diverse diseases.


KEY MESSAGES
The goal of this review is to establish a robust theoretical foundation for a more comprehensive understanding of eccDNA, offering valuable insights for the development of clinical diagnostics and innovative therapeutic strategies in the context of related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f915db54a9ed6fe76aa5229ced4103ebd9df09e" target='_blank'>
              Exploring the Role of Extrachromosomal Circular DNA in Human Diseases.
              </a>
            </td>
          <td>
            Yali Peng, Huihui Tao, Guoying Wang, Mengyao Wu, Tinatin Xu, Chunmei Wen, Xuejia Zheng, Yong Dai
          </td>
          <td>2024-09-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary By hypothesizing that molecular hyper-recombination and cellular proliferation are among the major features of multi-factorial cancer proneness, by using skin fibroblasts derived from eight major cancer syndromes, a significant correlation was found between the hyper-recombination rate quantified by plasmid assay, proliferation capacity assessed by flow cytometry, and excess of relative cancer risk (ERR). The product of the hyper-recombination rate and capacity of proliferation described a linear function of ERR. Abstract Context: Although carcinogenesis is a multi-factorial process, the mutability and the capacity of cells to proliferate are among the major features of the cells that contribute together to the initiation and promotion steps of cancer formation. Particularly, mutability can be quantified by hyper-recombination rate assessed with specific plasmid assay, hypoxanthine-guanine phosphoribosyltransferase (HPRT) mutations frequency rate, or MRE11 nuclease activities. Cell proliferation can be assessed by flow cytometry by quantifying G2/M, G1 arrests, or global cellular evasion. Methods: All these assays were applied to skin untransformed fibroblasts derived from eight major cancer syndromes characterized by their excess of relative cancer risk (ERR). Results: Significant correlations with ERR were found between hyper-recombination assessed by the plasmid assay and G2/M arrest and described a third-degree polynomial ERR function and a sigmoidal ERR function, respectively. The product of the hyper-recombination rate and capacity of proliferation described a linear ERR function that permits one to better discriminate each cancer syndrome. Conclusions: Hyper-recombination and cell proliferation were found to obey differential equations that better highlight the intrinsic bases of cancer formation. Further investigations to verify their relevance for cancer proneness induced by exogenous agents are in progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6745d80a485fc8627c999d4c147aa5ba6f6eca06" target='_blank'>
              Prediction of Cancer Proneness under Influence of X-rays with Four DNA Mutability and/or Three Cellular Proliferation Assays
              </a>
            </td>
          <td>
            Laura El Nachef, L. Bodgi, Maxime Estavoyer, Simon Buré, Anne-Catherine Jallas, A. Granzotto, Juliette Restier-Verlet, Laurène Sonzogni, J. Al-Choboq, Michel Bourguignon, Laurent Pujo-Menjouet, N. Foray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Disclosure: N. Gillis: None. C.A. Lange: None. Current treatments for estrogen receptor (ER) positive breast cancer largely target rapidly dividing tumor cells. However, aggressive breast cancers often contain long-lived and weakly proliferative cancer stem cells (CSCs) that readily escape treatments aimed at cycling cells. We previously reported that progesterone receptors (PR) and their target genes aid the growth of these CSCs and help them evade anti-estrogen therapy. As proliferation and stemness are directed by separate signaling pathways that lead to the expression of distinct gene programs, it's crucial to understand how PR facilitates this cellular transformation at the genomic level. Establishing a gene expression program that may limit proliferation but encourages stemness requires coordination between transcription factors and their coregulators to alter the nuclear microenvironment and chromatin landscape.To reveal these mechanisms, we profiled PR and ER genomic binding using CUT&RUN in the T47D luminal breast cancer cell line grown in ultra-low attachment (3D) mammosphere conditions. Our analysis of genomic binding profiles displayed a pattern of PR/ER co-occupancy at distant sites more than 10 kilobases away from the nearest transcriptional start site. We used Rank Ordering of Super Enhancer (ROSE) analysis on these PR/ER binding sites to predict which are likely super enhancer elements capable of coordinating the regulation of multi-gene subsets. We employed a genetic approach to directly test the function of novel enhancer sites linked to PR-target genes by disrupting the PR/ER binding with a dCas9-KRAB construct directed to the predicted enhancer elements. We propose that abnormal growth factor signaling in breast cancer cells permits PR/ER complex formation which promotes expression of genes that induce endocrine resistance and CSC expansion through regulation of super enhancer elements. Further studies to elucidate these mechanisms are ongoing. Understanding the intricacies of PR/ER crosstalk is the crucial next step in developing new therapies that target PR-driven processes in ER+ breast cancers. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc697beea9d371162709b515b1a10bf7a5a06aca" target='_blank'>
              7324 Regulation of Super Enhancers by Phosphorylated Progesterone Receptors Drives Breast Cancer Stem Cell Expansion
              </a>
            </td>
          <td>
            N. Gillis, C. Lange
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bba6936c2ac2d76bcee0a653b32c7b9ef4038fa" target='_blank'>
              Yeast EndoG prevents genome instability by degrading extranuclear DNA species
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Ruofan Yu, P. Thakre, Brenna S. McCauley, Nic Nikoloutsos, Yang Yu, Qian Li, P. Hastings, Weiwei Dang, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-09-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) are widely recognized as the result of exogenous retroviruses infecting the ancestral germline, stabilizing integration and vertical transmission during human genetic evolution. To date, endogenous retroviruses (ERVs) appear to have been selected for human physiological functions with the loss of retrotransposable capabilities. ERV elements were previously regarded as junk DNA for a long time. Since then, the aberrant activation and expression of ERVs have been observed in the development of many kinds of human diseases, and their role has been explored in a variety of human disorders such as cancer. The results show that specific ERV elements play respective crucial roles. Among them, long non-coding RNAs (lncRNAs) transcribed from specific long-terminal repeat regions of ERVs are often key factors. lncRNAs are over 200 nucleotides in size and typically bind to DNA, RNA, and proteins to perform biological functions. Dysregulated lncRNAs have been implicated in a variety of diseases. In particular, studies have shown that the aberrant expression of some ERV-derived lncRNAs has a tumor-suppressive or oncogenic effect, displaying significant functional bidirectionality. Therefore, theses lncRNAs have a promising future as novel biomarkers and therapeutic targets to explore the concise relationship between ERVs and cancers. In this review, we first summarize the role of ERV-derived lncRNAs in physiological regulation, mainly including immunomodulation, the maintenance of pluripotency, and erythropoiesis. In addition, pathological regulation examples of their aberrant activation and expression leading to carcinogenesis are highlighted, and specific mechanisms of occurrence are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362567f05869d7bba363004ce51b667dfd018994" target='_blank'>
              Functional Bidirectionality of ERV-Derived Long Non-Coding RNAs in Humans
              </a>
            </td>
          <td>
            Yanmei Song, Hongling Wen, Xiu-wei Zhai, Lei Jia, Lin Li
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Small-cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. Assay for transposase-accessible chromatin using sequencing and chromatin immunoprecipitation sequencing revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLCs. Analyses of human SCLC tumors and patient-derived xenografts (PDX) also showed an increase in TREX1 expression in postchemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase stimulator of interferon gene pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug-resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs. Significance: In this study, we show that targeting TREX1 induces an innate immune response and resensitizes SCLC cells to chemotherapy, representing a promising novel target for “immunologically” cold tumors, such as SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 Induces Innate Immune Response in Drug-Resistant Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56053b18f2967a281eeeeebbf2213c5fdde7cd2a" target='_blank'>
              Embryonic genome instability upon DNA replication timing program emergence
              </a>
            </td>
          <td>
            Saori Takahashi, Hirohisa Kyogoku, Takuya Hayakawa, Hisashi Miura, Asami Oji, Yoshiko Kondo, S. Takebayashi, Tomoya S. Kitajima, I. Hiratani
          </td>
          <td>2024-08-28</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Background R3HDM1, an RNA binding protein with one R3H domain, remains uncharacterized in terms of its association with tumor progression, malignant cell regulation, and the tumor immune microenvironment. This paper aims to fill this gap by analyzing the potential of R3HDM1 in diagnosis, prognosis, chemotherapy, and immune function across various cancers. Methods Data was collected from the Firehost database (http://gdac.broadinstitute.org) to obtain the TCGA pan-cancer queue containing tumor and normal samples. Additional data on miRNA, TCPA, mutations, and clinical information were gathered from the UCSC Xena database (https://xenabrowser.net/datapages/). The mutation frequency and locus of R3HDM1 in the TCGA database were examined using the cBioPortal. External validation through GEO data was conducted to assess the differential expression of R3HDM1 in different cancers. Protein expression levels were evaluated using the Clinical Proteomics Tumor Analysis Alliance (CPTAC). The differential expression of R3HDM1 was verified in lung adenocarcinoma cell lines and normal lung glandular epithelial cells via RT-qPCR. Cell migration and proliferation experiments were conducted by knocking down the expression of R3HDM1 in two lung adenocarcinoma cell lines using small interfering RNA. The biological role of R3HDM1 in pan-cancer was explored using the GSEA method. Multiple immune infiltration algorithms from the TIMER2.0 database was employed to investigate the correlation between R3HDM1 expression and the tumor immune microenvironment. Validation of transcriptome immune infiltration was based on 140 single-cell datasets from the TISCH database. The study also characterized a pan-cancer survival profile and analyzed the differential expression of R3HDM1 in different molecular subtypes. The relationship between R3HDM1 and drug resistance was investigated using four chemotherapy data sources: CellMiner, GDSC, CTRP and PRISM. The impact of chemicals on the expression of R3HDM1 was explored through the CTD database. Result The study revealed differential expression of R3HDM1 in various tumors, indicating its potential as an early diagnostic marker. Changes in somatic copy number (SCNA) and DNA methylation were identified as factors contributing to abnormal expression levels. Additionally, the study found that R3HDM1 expression is associated with clinical features, metabolic pathways, and important pathways related to metastasis and the immune system. High expression of R3HDM1 was linked to poor prognosis across different tumors and altered drug sensitivity. Furthermore, the expression of R3HDM1 showed significant correlations with immune modulatory molecules and biomarkers of lymphocyte subpopulation infiltration. Finally, the study highlighted four chemicals that could influence the expression of R3HDM1. Conclusion Overall, this study proposes that R3HDM1 expression is a promising biomarker for predicting the prognosis of cancer, especially lung adenocarcinoma, and the efficacy of immunotherapy, demonstrating the rationale for further exploration in the development of anti-tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379d6a67141459985f554b1a56ebc613e9c22b34" target='_blank'>
              Comprehensive pan-cancer analysis and experiments revealed R3HDM1 as a novel predictive biomarker for prognosis and immune therapy response
              </a>
            </td>
          <td>
            Jiawei Liu, Zhitong Bing, Junling Wang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dfb4f972dc9db981473d705bf7ba9bf069116ae" target='_blank'>
              Overexpression of TPD52L2 in HNSCC: prognostic significance and correlation with immune infiltrates
              </a>
            </td>
          <td>
            Min Lu, Haoyu Xia, Jing Xu, Zijun Liao, Yuwen Li, Hanwei Peng
          </td>
          <td>2024-10-07</td>
          <td>BMC Oral Health</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objective: This study was aimed at exploring a specific open region of chromatin in the peripheral blood mononuclear cells (PBMCs) of patients with breast cancer and evaluating its feasibility as a biomarker for diagnosing and predicting breast cancer prognosis. Methods: We obtained PBMCs from breast cancer patients and healthy people for the assay for transposase-accessible chromatin (ATAC) sequencing (n = 3) and obtained the GSE27562 chip sequencing data for secondary analyses. Through bioinformatics analysis, we mined the pattern changes for chromatin accessibility in the PBMCs of breast cancer patients. Results: A total of 1,906 differentially accessible regions (DARs) and 1,632 differentially expressed genes (DEGs) were identified via ATAC sequencing. The upregulated DEGs in the disease group were mainly distributed in the cells, organelles, and cell-intima-related structures and were mainly responsible for biological functions such as cell nitrogen complex metabolism, macromolecular metabolism, and cell communication, in addition to functions such as nucleic acid binding, enzyme binding, hydrolase reaction, and transferase activity. Combined with microarray data analysis, the following set of nine DEGs showed intersection between the ATAC and microarray data: JUN, MSL2, CDC42, TRIB1, SERTAD3, RAB14, RHOB, RAB40B, and PRKDC. HOMER predicted and identified five transcription factors that could potentially bind to these peak sites, namely NFY, Sp 2, GFY, NRF, and ELK 1. Conclusion: Chromatin accessibility analysis of the PBMCs from patients with early-stage breast cancer underscores its potential as a significant avenue for biomarker discovery in breast cancer diagnostics and treatment. By screening the transcription factors and DEGs related to breast cancer, this study provides a comprehensive theoretical foundation that is expected to guide future clinical applications and therapeutic developments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3171ecf6420e675e47a99c9a77102cb706140c79" target='_blank'>
              Analysis of chromatin accessibility in peripheral blood mononuclear cells from patients with early-stage breast cancer
              </a>
            </td>
          <td>
            Longjie Xia, Jiamin Lu, Yixuan Qin, Runchun Huang, Fanbiao Kong, Yu Deng
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The most common cancer in Korea is thyroid cancer, followed by lung cancer, and the most common cancer in men is lung cancer. The cancer with the highest mortality rate is lung cancer. Recently, the precision medical approach is becoming a major treatment strategy for lung cancer. For precision medical approach, the usefulness of next-generation sequencing (NGS) analysis targeting circulating tumor DNA (ctDNA) is being emphasized. Therefore, in this study, NGS analysis was performed on blood and tissues of advanced lung cancer patients to compare and analyze genetic mutation patterns to confirm the usefulness of ctDNA NGS assay.



 A total of 101 patients with advanced lung cancer (stage 3 or 4) were recruited, and ctDNA NGS assays were performed on their plasma using a Pan100 panel (Dxome). Tissues from primary lung cancer, metastatic lymph nodes, or metastatic organs of 22 patients were analyzed using ONCOaccuPanel (NGeneBio). All variants were classified into four tiers based on the level of clinical significance, according to the standards and guidelines established by the Association for Molecular Pathology, American Society of Clinical Oncology, and College of American Pathologists.



 In the ctDNA NGS assay of 101 patients, 70 (69.3%) harbored 154 tier 1 or 2 variants. Mutations in TP53 (45, 29.2%) and EGFR (23, 14.9%) were the most frequently detected, followed by mutations in RB1, CHEK2, ATM, JAK2, ATR, DNMT3A, and KRAS. Among the 22 patients whose tissue was examined, tier 1 or 2 variants were detected in both the plasma and tissue of 14 patients and only in the tissue of six patients. All genes for these variants were targeted using both methods. Identical variants were detected in the plasma and tissue of six of the 14 patients, and discordant results were observed in the remaining eight patients. Twenty copy number variations were detected in 13 of the 101 patients, with copy number gain of EGFR being the most common (6, 30%). Fusion genes were detected in five of the 101 patients, two of which were not detected in the tissue. Overall, 74 (73.3%) of the 101 patients had positive ctDNA NGS assay results.



 Although the NGS assay with ctDNA is less sensitive than that with tissue DNA, some genetic alterations are detected only in ctDNA NGS assays; therefore, it is useful to apply the ctDNA NGS assay as a complement.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8e0e6eed278b41bf977b6ec4d0bb9e829c01bb" target='_blank'>
              B-191 Genomic Profile of Circulating Tumor DNA in Advanced Lung Cancer: Comparison with Tissue DNA
              </a>
            </td>
          <td>
            J. Park, Y. Choi, S. Jeon, S. Kee, E. Na, M. Shin
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal disease with limited therapies. In our screening using patient-derived organoids, we identified cyclin-dependent kinase 12 (CDK12) as a promising target in a subset of PDAC. Both pharmacological and genetic inhibition of CDK12 exerted potent antitumor activity in vitro and in vivo, inducing double-strand DNA breaks and apoptosis. CDK12 is a member of cyclin-dependent kinases that is involved in gene expression regulations at multiple levels. Recently, CDK12 has gained attention as a therapeutic target in several types of cancers including triple-negative breast cancer and Ewing sarcoma. However, little is known about the impact and possibility of CDK12 inhibition in pancreatic cancer treatment. As the lack of specific inhibitors that selectively target CDK12, studying CDK12 has long been challenging. To overcome this, we used CRISPR-Cas9 mediated genome editing to generate FKBP12F36V (dTAG) knock-in PDAC cell lines, enabling rapid elimination of endogenous CDK12 using a degrader against dTAG (dTAG-13). Quantitative-image-based cytometry and time lapse imaging showed that CDK12 ablation induces mitotic arrest, DNA damage and apoptosis (mitotic catastrophe). Specifically, loss of CDK12 kinase activity in the S phase was sufficient to slow DNA replication speed, leading to spindle assembly checkpoint activation and mitotic catastrophe. Importantly, cells that escaped mitotic death accumulate micronuclei, triggering immune response signaling that is mediated by the cGAS-STING-type I interferon pathway. Our results suggest that CDK12 is a promising target in a subset of PDAC patients, where CDK12 inhibition exerts antitumor activity both in cell autonomous and non-cell autonomous ways through inducing mitotic catastrophe as well as triggering immune response.
 Citation Format: Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro. CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A084.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632d48b78e0c9131037bdfa10d93bd71182ca006" target='_blank'>
              Abstract A084: CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer
              </a>
            </td>
          <td>
            Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Long interspersed elements type 1 (LINE-1s or L1s) are highly abundant non-LTR retrotransposons in mammalian genomes. Apart from an evolutionary role in altering the genomic landscape and gene expression, recent discoveries have revealed the scale and impact of active movement by L1, both in the germline and in somatic niches (e.g., the brain and many cancers). As transcriptional activation is a prerequisite for L1 retrotransposition, it is important to understand how L1 transcription is regulated. L1 transcription is controlled by an internal promoter contained in the 5’ untranslated region (5’UTR) of a full-length element. It has been shown that transcription start sites (TSSs) of human L1 are clustered at the beginning of its 5’UTR and the precision of transcription initiation is controlled by transcription factor YY1. The laboratory mouse is an essential model for investigating L1 retrotransposition and the associated consequences in vivo. However, mouse L1 5’UTRs are distinctly different from human L1’s: the former are organized into tandem repeats called monomers and separated from the open reading frames by a nonmonomeric sequence called tether. Unlike the human L1 promoter, transcription initiation and its regulation in mouse L1 subfamilies remain unknown. In this study, we streamlined the cloning of different types of mouse L1 promoters. TSSs were determined using RNA ligation-mediated 5’ rapid amplification of cDNA ends (RLM-5’RACE) incorporating a highly efficient reverse transcriptase enzyme followed by Nanopore sequencing. In the context of two monomer promoter reporter construct, transcription could initiate from either monomer or the tether sequence. While mouse L1 TSSs are distributive, a higher number of transcripts were initiated from the inner monomer. Surprisingly, a significant fraction of transcripts was also initiated from the tether sequence. In addition, we showed that YY1 transcriptionally activates mouse L1 5’UTR and is responsible for the synergy between the monomers. As transcription initiation is a major determinant for the proportion of active L1 loci in the next round of retrotransposition, these findings have important implications on L1 biology in the mouse model.
 Citation Format: Karabi Saha, Wenfeng An. Transcription initiation and its regulation in mouse L1s [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a796751004ae7aa371fe2f625b26733c662788" target='_blank'>
              Abstract A013: Transcription initiation and its regulation in mouse L1s
              </a>
            </td>
          <td>
            Karabi Saha, Wenfeng An
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA rearrangements, including inversions, translocations, and large insertions/deletions (indels), are crucial for crop evolution, domestication, and improvement. The rearrangements are frequently induced by ion beams via the mis-repair of DNA double-strand breaks (DSBs). Unfortunately, how ion beam-induced DSBs are repaired has not been comprehensively analyzed and the mechanisms underlying DNA rearrangements remain unclear. In this study, clonal sectors originating from single mutated cells in carbon ion-irradiated plants were used for whole-genome sequencing analyses after Arabidopsis seeds and seedlings were irradiated. Comparative analyses of the induced mutations (e.g., size and frequency of indels and microhomology at the junctions of the rearrangements) in the irradiated materials suggested that the broken/rejoined DSB ends were more extensively processed in seedlings than in seeds. A mutation to canonical non-homologous end-joining (c-NHEJ), which is a DSB repair pathway with minimal processing of DSB ends, increased the sensitivity to ion beams more in the seeds than in the seedlings, which was consistent with the junction analysis results, indicative of the minor contribution of c-NHEJ to the carbon ion-induced DSB repair in seedlings. Considering the characteristics of the large templated insertions in irradiated seedlings, ion-beam-induced DSBs in seedlings are likely repaired primarily by a polymerase theta-mediated pathway. Polymerase theta-deficient seedlings were more sensitive to ion beams than the c-NHEJ-deficient seedlings, consistent with this hypothesis. This study revealed the key characteristics of ion beam-induced DSBs and the associated repair mechanisms related to the physiological status of the irradiated materials, with implications for elucidating the occurrence and induction of rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a80bdd3149f4bee83a298c8c32839582a3cb4dc" target='_blank'>
              Differential contributions of double-strand break repair pathways to DNA rearrangements following the irradiation of Arabidopsis seeds and seedlings with ion beams.
              </a>
            </td>
          <td>
            S. Kitamura, K. Satoh, Yoshihiro Hase, Ryouhei Yoshihara, Yutaka Oono, N. Shikazono
          </td>
          <td>2024-09-23</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, R. Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This review starts off with the first germline homozygous variants of the Nucleoporin 98 gene (NUP98) in siblings whose clinical presentation recalls Rothmund–Thomson (RTS) and Werner (WS) syndromes. The progeroid phenotype caused by a gene associated with haematological malignancies and neurodegenerative disorders primed the search for interplay between caretakers involved in genome instability syndromes and Nuclear Pore Complex (NPC) components. In the context of basic information on NPC architecture and functions, we discuss the studies on the interdependence of caretakers and gatekeepers in WS and Hereditary Fibrosing Poikiloderma (POIKTMP), both entering in differential diagnosis with RTS. In WS, the WRN/WRNIP complex interacts with nucleoporins of the Y-complex and NDC1 altering NPC architecture. In POIKTMP, the mutated FAM111B, recruited by the Y-complex’s SEC13 and NUP96, interacts with several Nups safeguarding NPC structure. The linkage of both defective caretakers to the NPC highlights the attempt to activate a repair hub at the nuclear periphery to restore the DNA damage. The two separate WS and POIKTMP syndromes are drawn close by the interaction of their damage sensors with the NPC and by the shared hallmark of short fragile telomeres disclosing a major role of both caretakers in telomere maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f8c0edcf3a3bd5207393373e2885f587217e1c" target='_blank'>
              Interdependence between Nuclear Pore Gatekeepers and Genome Caretakers: Cues from Genome Instability Syndromes
              </a>
            </td>
          <td>
            Lidia Larizza, E. Colombo
          </td>
          <td>2024-08-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Atypical Teratoid Rhabdoid Tumor (ATRT) is a highly aggressive pediatric brain tumor and characterized by SMARCB1 deletion. SMARCB1 is a member of the SWI/SNF chromatin remodeling complex that is responsible for determining cellular pluripotency and lineage commitment. We performed an RNAi-based functional genomic screen to identify key epigenetic regulators that co-operate with SMARCB1 loss and contribute to tumorigenesis in ATRT. 406 genes associated with epigenetic regulation in patient derived ATRT cell lines were targeted. This screen identified BMI1, a component of the Polycomb Repressive Complex 1 (PRC1), as one of the top targets essential for ATRT cell growth. Using RNA-seq and ChIP-Seq analysis we demonstrated that BMI1 co-operates with SMARCB1 deletion to suppress transcription of pro-differentiation pathways and promote self-renewal of tumor stem cells. In SMARCB1 deficient cells BMI1 forms a partial PRC1 complex devoid of DNA binding components. We found that genetic inactivation of BMI1 suppresses ATRT cell clonogenicity in vitro and decreases S-phase length. By Doxycycline-inducible SMARCB1 re-expression system we revealed activation of proapoptotic proteins p16 and p21. With immunoprecipitation for BMI1, followed by mass spectrometry analysis we demonstrated that re-expression of SMARCB1 activates two PRC1 chromatin localizing components CBX4 and CBX8. Moreover, inhibition of BMI1 significantly prolongs mice survival and decreases tumor size as evaluated by IVIS imaging system and T2-MRI. Thus, we demonstrated that SMARCB1 deletion results in reprograming of BMI1 chromatin occupancy away from lineage specification by altering the components of the PRC1 complex and that PRC1/BMI1 inhibition is a novel therapeutic approach in ATRT.
 Citation Format: Irina Irina Alimova, Gillian Murdock, Angela M. Pierce, Dong Wang, Nathan Dahl, Sujatha Venkataraman, Rajeev Vibhakar. PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d643026696b6226900916830deb821d92b3f76f" target='_blank'>
              Abstract A080 PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Irina Irina Alimova, Gillian Murdock, Angela M Pierce, Dong Wang, N. Dahl, S. Venkataraman, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The Mediator complex regulates various aspects of hematopoietic development, but whether composition of the Mediator complex undergoes dynamic changes for diversifying transcription and functional outputs is unknown. Here, we found that MED26, a subunit in the core Mediator complex, played a distinctive role in facilitating transcription pausing essential for erythroid development. While most Mediator subunits drastically decreased during this process, MED26 remained relatively abundant. Intriguingly, in the early stages, more than half of MED26 occupancy sites did not co-localize with MED1, a representative Mediator subunit, suggesting these subunits exert context-dependent gene regulation. We revealed that MED26-enriched loci were associated with RNA polymerase Ⅱ pausing. MED26 manifested a markedly preferential recruitment of pausing-related factors, leading to an increase in Pol Ⅱ pausing critical for genome-wide transcription repression during erythropoiesis. Moreover, MED26 exhibited pronounced condensate-forming capability, which was necessary for its function in promoting erythropoiesis and recruiting pausing-related factors. Collectively, this study provides mechanistic insights into the functional coordination of distinct Mediator subunits during development and highlights the switch of transcription condensates towards a MED26 enriched form, which modulates transcription pausing to facilitate transcription repression and erythroid development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a384294c55ca07ded3b8f72b1e5a9d773103d322" target='_blank'>
              MED26-enriched condensates drive erythropoiesis through modulating transcription pausing
              </a>
            </td>
          <td>
            Shicong Zhu, Xiaoting Zhang, Na Li, Xinying Zhao, Man Li, Si Xie, Qiuyu Yue, Yunfeng Li, Dong Li, Fan Wu, Zile Zhang, Ziqi Feng, Yiyang Zhang, Wonhyung Choi, Xinyi Jia, Yuelin Deng, Qi Hu, Xingyun Yao, Xiaofei Gao, Hsiang-Ying Lee
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082a7d22b12accfd1876717033920e6fb8f44c27" target='_blank'>
              Understanding m6A changes in chromophobe renal cell carcinoma and predicting patient outcomes survival
              </a>
            </td>
          <td>
            Zhigang Chen, Junbo Yang, Wei Zhang, Yang Qian, Nan Zhang, Zixin Chen, Min Lu, L. Ge, Cheng Liu, X. Tian, Guifang Jia, Lulin Ma, Baoguo Li
          </td>
          <td>2024-09-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA damage compromises not only genome stability but also the integrity of the chromatin template, which plays a central role in controlling cell identity. Our understanding of chromatin repair mechanisms is very incomplete. To bridge this knowledge gap, here we devise a novel proteomic strategy to characterize dynamic changes in the chromatin landscape during the repair of UV-induced DNA lesions in human cells, in a quantitative, unbiased and time-resolved manner. Thus, we identify the histone chaperones DNAJC9 and MCM2 as central players in chromatin repair. We demonstrate that DNAJC9 and MCM2 are independently recruited to sites of UV damage repair. DNAJC9 provides new H3-H4 histones to CAF-1 and HIRA chaperones for deposition into chromatin and also stimulates old H3-H4 histone recovery. DNAJC9 cooperates with MCM2 to coordinate old and new histone dynamics during UV damage repair. Together, our proteomic dataset provides a molecular framework for further dissecting epigenome maintenance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62f11b9db8e84916012d25bd485040e15634b9b" target='_blank'>
              Proteomic profiling of UV damage repair patches uncovers histone chaperones with central functions in chromatin repair
              </a>
            </td>
          <td>
            Alexandre Plessier, Audrey Chansard, Eliane Petit, Julia Novion Ducassou, Yohann Couté, Sophie E. Polo
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Eukaryotic chromosomes segregate faithfully prior to nuclear division to ensure genome stability. If segregation becomes defective, the chromosome copy number of the cell may alter leading to aneuploidy and/or polyploidy, both common hallmarks of cancers. In eukaryotes, aurora kinases regulate chromosome segregation during mitosis and meiosis, but their functions in the divergent, single-celled eukaryotic pathogen Trypanosoma brucei are less understood. Here, we focused on one of three aurora kinases in these parasites, TbAUK3, a homologue of the human aurora kinase AURKC, whose functions are primarily restricted to meiosis. We show that RNAi targeted depletion of TbAUK3 correlates with nuclear segregation defects, reduced proliferation, and decreased DNA synthesis, suggestive of a role for TbAUK3 during mitotic, not meiotic, chromosome segregation. Moreover, we uncover a putative role for TbAUK3 during the parasite’s response to DNA damage since we show that depletion of TbAUK3 enhances DNA instability and sensitivity to genotoxic agents. HIGHLIGHTS The C-terminus of TbAUK3 is disordered TbAUK3 depletion coincides with nuclear segregation defects Depletion of TbAUK3 enhances DNA instability">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125d96f157b6efd3c7ef3b8d6665537e416170cc" target='_blank'>
              AUK3 is required for faithful nuclear segregation in the bloodstream form of Trypanosoma brucei
              </a>
            </td>
          <td>
            J. A. Black, B. Poulton, B.S. Gonzaga, A. Iskantar, L. Tosi, R. McCulloch
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Despite recent advances in the treatment of pancreatic adenocarcinoma (PDAC), the median survival remains <12 months. Patients typically present with late-stage disease and are often unable to tolerate drug combination regimens. Large-scale genomic approaches categorise PDAC into two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal) subtypes. Although it is well-described that PDAC harbours gene mutations (i.e. KRAS G12D, p53 R172H) that initiate the disease, little is known about the influence of lineage-defining Transcription Factors (TF) on PDAC subtypes and their contribution to tumor progression. We took an unbiased approach to discover driving TFs in PDAC, we performed ChIP-seq of the active enhancer mark H3K27Ac from surplus tissue following resection of primary PDAC tumors (n=6) and normal adjacent tissue (n=4). Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements were HNF motifs (HNF4G & HNF4A) and Forkhead (i.e., FOXA1), confirming that these two classes of TFs constitute the lineage-defining factors in these clinical samples. Investigation of publicly available clinical datasets revealed HNF4G to be amplified in 9% of PDAC patients. Using Rapid Immunoprecipitation and Mass Spectrometry of Endogenous protein complexes (RIME), our method for unbiased discovery of endogenous protein complexes, we investigated the fundamental differences in the FOXA1 interactome between models of classical and squamous subtypes. HNF4A and HNF4G were only seen as FOXA1-interactors in models of the classical subtype. HNF4G Knockout (KO) clones injected orthotopically into the pancreas of mice, showed a significant increase (p-value<0.0006) in median survival and slower tumor growth compared to Control (Cas-9) (p-value <0.0001). HNF4G-KO dependent genes were specifically expressed in primary tumors (with decreased EMT signalling) versus metastatic samples (PanCRux dataset), implying that the HNF4G target genes were mostly limited to primary tumor contexts. We found Protein Arginine Methylate Transferase (PRMT1) to be a significant and reproducible interactor of FOXA1 and HNF4G. We observed a significant global reduction in PRMT1 chromatin binding in both the HNF4G-KO and FOXA1 depleted cells compared to the control. This data suggests a dependency on both HNF4G and FOXA1 for recruitment of PRMT1 to chromatin. We find HNF4G to be the key driver of classical subtype primary disease, in part by recruiting the methyltransferase PRMT1 to regulatory regions revealing a therapeutically exploitable opportunity. A molecular switch occurs in advanced disease, whereby HNF4G expression decreases, unmasking FOXA1’s transcriptional activity. De-repressed FOXA1 drives late-stage disease, by orchestrating metastasis-specific enhancer-promoter loops to direct metastatic gene regulation. We suggest the existence of stage-dependent TF activity, triggered by molecular compartmentalisation that mediates multistage progression of PDAC.
 Citation Format: Shalini Vasudev Rao, Lisa V Young, Danya Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll. Transcription factor switching drives progression of the classical subtype of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e663bb2d35a48fcc777219947b4d5e9554b7ef19" target='_blank'>
              Abstract C071: Transcription factor switching drives progression of the classical subtype of pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Three-dimensional chromatin control of eukaryotic transcription is pivotal for regulating gene expression. This additional layer of epigenetic regulation is also utilized by DNA viruses, including herpesviruses. Dynamic, spatial genomic organization often involves looping of chromatin anchored by host-encoded CCCTC-binding factor (CTCF) and other factors, which control crosstalk between promoters and enhancers. Herein, we review the contribution of CTCF-mediated looping in regulating transcription during herpesvirus infection, with a specific focus on the betaherpesvirus, human cytomegalovirus (HCMV).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ca71344a757b4c351d6ffdff351925d0ab026" target='_blank'>
              Loopy virus or controlled contortionist? 3D regulation of HCMV gene expression by CTCF-driven chromatin interactions.
              </a>
            </td>
          <td>
            Ian J. Groves, Christine M. O’Connor
          </td>
          <td>2024-08-30</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) describes the process where distinct epigenetic states may be transmitted between generations, resulting in stable gene expression and phenotypic differences between individuals that persist independently of DNA sequence variation. Chromatin modifications have been demonstrated as important in TEI, however, the extent to which they require other signals to establish and maintain epigenetic states is still unclear. Here we investigate whether small non-coding RNAs contribute to different epigenetic states of the Fab2L transgene in Drosophila triggered by transient chromatin contacts, which requires Polycomb complex activity to deposit the H3K27me3 modification for long-term TEI. Using mutants deficient in known small non-coding RNAs, high throughput sequencing, investigation of chromatin conformation and gene expression analysis we demonstrate that small non-coding RNAs do not contribute directly to initiation or maintenance of silencing. However, we uncover an indirect role for microRNA expression in transgene silencing through effects on Polycomb complex expression. Additionally, we show that a commonly used marker gene, Stubble (Sb), affects Polycomb complex expression, which may be important in interpreting experiments assaying Polycomb function in Drosophila development. By ruling out a plausible candidate for TEI at the Fab2L transgene our work highlights the variability in different modes of TEI across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be548511a3f566003319963d618b6d4697f22992" target='_blank'>
              Polycomb-mediated transgenerational epigenetic inheritance of Drosophila eye colour is independent of small RNAs
              </a>
            </td>
          <td>
            Maximilian H. Fitz-James, Poppy Sparrow, Christopher Paton, P. Sarkies
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e3c1ad704a586aa01f4f3735cd6a0421899426" target='_blank'>
              Rise and SINE: roles of transcription factors and retrotransposons in zygotic genome activation.
              </a>
            </td>
          <td>
            Pavel Kravchenko, Kikuë Tachibana
          </td>
          <td>2024-10-02</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for assessing chromatin stability in healthy and sick dogs (with squamous cell carcinoma). The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable a more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. Abstract Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Disclosure: A. Sanghi: None. S. Wu: None. T. Chou: None. L.A. Orloff: None. M. Kasowski: None. M. Snyder: None. Although genetic mutations are causal in human cancers, the contribution of chromatin changes to cancer onset and progression remains less well understood. Multi-omics profiling of human tumors can provide insight into how alterations in chromatin structure are propagated through the pathway of gene expression to result in cancer signaling. We applied multi-omics profiling of 36 human thyroid cancer primary tumors, metastases, and patient-matched normal tissue. Over 80% of the tumors harbored BRAFV600E mutations, and the remaining tumors mostly had activating mutations in the MAPK pathway. Through quantification of chromatin accessibility associated with active transcription units and global protein expression, we identify local enhancers that are highly correlated with coordinated RNA and protein expression. The enhancers are predicted to be actively bound by MAPK transcription factor, indicating they are under MAPK control. These cancer-specific enhancers associated with upregulation of lysosomal peptidase expression and dipeptide abundance in tumors. We show that upregulated dipeptides stimulate MAPK activity, and BRAF inhibition results in loss of lysosomal function in mutants. Perturbation of lysosomal peptidases and dipeptide abundance decreases MAPK activity only in mutant cells compared to wildtype. These analyses suggest that BRAF thyroid tumors integrate MAPK-driven epigenetic regulation with lysosomal metabolite signaling in a positive feedback circuit. Using local cancer-specific epigenetic features, we isolated key cancer signaling, making way for potential targets for cancer therapeutics. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032e66998348d599db74b255c1962f11618e9fbe" target='_blank'>
              7948 Integration of Epigenomics and Metabolomics Reveals New Signaling in Aggressive Thyroid Cancer
              </a>
            </td>
          <td>
            A. Sanghi, S. Wu, T. Chou, L. A. Orloff, M. Kasowski, M. Snyder
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The integrity of the actively transcribed genome against helix-distorting DNA lesions relies on a multilayered cellular response that enhances Transcription-Coupled Nucleotide Excision Repair (TC-NER). When defective, TC-NER is causatively associated with Cockayne-Syndrome (CS), a rare severe human progeroid disorder. Although the presence of unresolved transcription-blocking lesions is considered a driver of the aging process, the molecular features of the transcription-driven response to genotoxic stress in CS-B cells remain largely unknown. Here, an in-depth view of the transcriptional and associated chromatin dynamics that occur in CS-B cells illuminates the role of CSB therein. By employing high-throughput genome-wide approaches, we observed that absence of a functional CSB protein results in a delay in transcription progression, more positioned +1 nucleosomes, and less dynamic chromatin structure, compared to normal cells. We found that early after exposure to UV, CS-B cells released RNA polymerase II (RNAPII) from promoter-proximal pause sites into elongation. However, the magnitude of this response and the progression of RNAPII were reduced compared to normal counterparts. Notably, we detected increased post-UV retainment of unprocessed nascent RNA transcripts and chromatin-associated elongating RNAPII molecules. Contrary to the prevailing models, we found that transcription initiation is operational in CS-B fibroblasts early after UV and that chromatin accessibility showed a marginal increase. Our study provides robust evidence for the role of CSB in shaping the transcription and chromatin landscape both in homeostasis and in response to genotoxic insults, which is independent of its known role in TC-NER, and which may underlie major aspects of the CS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966a15785f4d1763d145786faf0eda45c3c2df7f" target='_blank'>
              Cockayne syndrome B protein is implicated in transcription and associated chromatin dynamics in homeostatic and genotoxic conditions.
              </a>
            </td>
          <td>
            Anastasios Liakos, Katerina Z Ntakou-Zamplara, Nelina Angelova, Dimitris Konstantopoulos, Anna-Chloe Synacheri, Zoi Spyropoulou, Iason A Tsarmaklis, Despoina Korrou-Karava, Georgios Nikolopoulos, Matthieu D. Lavigne, Maria Fousteri
          </td>
          <td>2024-10-06</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7afaab9db2ee6172d70bbe5180a07660e6f1ac0" target='_blank'>
              Mitochondrial DNA content and methylation in sperm of patients with asthenozoospermia.
              </a>
            </td>
          <td>
            Qiang Geng, Ruifang Gao, Yuan Sun, Shaofeng Chen, Lili Sun, Wei Li, Zhong Li, Yu Zhao, Feng Zhao, Ying Zhang, Anwen Li, Hongbin Liu
          </td>
          <td>2024-08-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Genome size is influenced by natural selection and genetic drift acting on variations from polyploidy and repetitive DNA sequences. We hypothesized that centromere drive, where centromeres compete for inclusion in the functional gamete during meiosis, may also affect genome and chromosome size. This competition occurs in asymmetric meiosis, where only one of the four meiotic products becomes a gamete. If centromere drive influences chromosome size evolution, it may also impact post-polyploid diploidization, where a polyploid genome is restructured to function more like a diploid through chromosomal rearrangements, including fusions. We tested if plant lineages with asymmetric meiosis exhibit faster chromosome size evolution compared to those with only symmetric meiosis, which lack centromere drive as all four meiotic products become gametes. We also examined if positive selection on centromeric histone H3 (CENH3), a protein that can suppress centromere drive, is more frequent in these asymmetric lineages.


METHODS
We analyzed plant groups with different meiotic modes: asymmetric in gymnosperms and angiosperms, and symmetric in bryophytes, lycophytes, and ferns. We selected species based on available CENH3 gene sequences and chromosome size data. Using Ornstein-Uhlenbeck evolutionary models and phylogenetic regressions, we assessed the rates of chromosome size evolution and the frequency of positive selection on CENH3 in these clades.


RESULTS
Our analyses showed that clades with asymmetric meiosis have a higher frequency of positive selection on CENH3 and increased rates of chromosome size evolution compared to symmetric clades.


CONCLUSIONS
Our findings support the hypothesis that centromere drive accelerates chromosome and genome size evolution, potenatially also influencing the process of post-polyploid diploidization. We propose a model which in a single famework helps explain the stability of chromosome size in symmetric lineages (bryophytes, lycophytes, and ferns) and its variability in asymmetric lineages (gymnosperms and angiosperms), providing a foundation for future research in plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fabc90ee2c5c9cff34a1bbfeb39ccf65f0a8155b" target='_blank'>
              Centromere drive may propel the evolution of chromosome and genome size in plants.
              </a>
            </td>
          <td>
            Klára Plačková, P. Bureš, M. Lysak, F. Zedek
          </td>
          <td>2024-08-28</td>
          <td>Annals of botany</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7300fe63a85fac8fe37bf10af42231e58f921f5" target='_blank'>
              Analysis of the role of CHPF in colorectal cancer tumorigenesis and immunotherapy based on bioinformatics and experiments
              </a>
            </td>
          <td>
            Qingyu Song, Pengchao Wang, Jingyu Wu, Ming Lu, Qingcheng Xia, Yexin Shi, Zijun Wang, Xiang Ma, Qinghong Zhao
          </td>
          <td>2024-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oxidative damage triggers micronuclear membrane rupture and defective repair Errors in chromosome segregation during cell division (mitosis) can lead to chromosomal instability, a hallmark of human cancer. These mitotic errors give rise to the formation of micronuclei—small cytoplasmic structures that are spatially separated from the primary nucleus and contain lagging chromosomes or chromosome fragments. Unlike the primary nucleus of a cell, micronuclei frequently undergo irreparable rupture and collapse. This breakdown releases micronuclear DNA into the cytosol, causing DNA damage, chromosomal rearrangements, and genomic instability, thereby contributing to cancer progression. The rupture of micronuclei also activates a component of the innate immune system called the cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING) pathway, leading to tumorrelated inflammation and metastasis (1). On pages 951 and 952 of this issue, Di Bona et al. (2) and Martin et al. (3), respectively, report the molecular underpinnings of micronuclear collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3bfa9de0e1bf705d54ddc01f0ac3869ca5a791" target='_blank'>
              Micronuclear collapse mechanisms in cancer
              </a>
            </td>
          <td>
            Marianna Maddaluno, C. Settembre
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Mutations of the SNF2 family ATPase HELLS and its activator CDCA7 cause immunodeficiency, centromeric instability, and facial anomalies syndrome, characterized by DNA hypomethylation at heterochromatin. It remains unclear why CDCA7-HELLS is the sole nucleosome remodeling complex whose deficiency abrogates the maintenance of DNA methylation. We here identify the unique zinc-finger domain of CDCA7 as an evolutionarily conserved hemimethylation-sensing zinc finger (HMZF) domain. Cryo–electron microscopy structural analysis of the CDCA7-nucleosome complex reveals that the HMZF domain can recognize hemimethylated CpG in the outward-facing DNA major groove within the nucleosome core particle, whereas UHRF1, the critical activator of the maintenance methyltransferase DNMT1, cannot. CDCA7 recruits HELLS to hemimethylated chromatin and facilitates UHRF1-mediated H3 ubiquitylation associated with replication-uncoupled maintenance DNA methylation. We propose that the CDCA7-HELLS nucleosome remodeling complex assists the maintenance of DNA methylation on chromatin by sensing hemimethylated CpG that is otherwise inaccessible to UHRF1 and DNMT1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a71a94532c3792f0d86967a15a54fee4902486" target='_blank'>
              CDCA7 is an evolutionarily conserved hemimethylated DNA sensor in eukaryotes
              </a>
            </td>
          <td>
            Isabel E. Wassing, Atsuya Nishiyama, Reia Shikimachi, Qingyuan Jia, Amika Kikuchi, Moeri Hiruta, Keita Sugimura, Xin Hong, Yoshie Chiba, Junhui Peng, Christopher Jenness, M. Nakanishi, Li Zhao, K. Arita, H. Funabiki
          </td>
          <td>2024-08-23</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Centrosome amplification is a feature of cancer cells associated with chromosome instability and invasiveness. Enhancing chromosome instability and subsequent cancer cell death via centrosome unclustering and multipolar divisions is an aimed-for therapeutic approach. Here, we show that centrosome amplification potentiates responses to conventional chemotherapy in addition to its effect on multipolar divisions and chromosome instability. We perform single-cell live imaging of chemotherapy responses in epithelial ovarian cancer cell lines and observe increased cell death when centrosome amplification is induced. By correlating cell fate with mitotic behaviors, we show that enhanced cell death can occur independently of chromosome instability. We identify that cells with centrosome amplification are primed for apoptosis. We show they are dependent on the apoptotic inhibitor BCL-XL and that this is not a consequence of mitotic stresses associated with centrosome amplification. Given the multiple mechanisms that promote chemotherapy responses in cells with centrosome amplification, we assess such a relationship in an epithelial ovarian cancer patient cohort. We show that high centrosome numbers associate with improved treatment responses and longer overall survival. Our work identifies apoptotic priming as a clinically relevant consequence of centrosome amplification, expanding our understanding of this pleiotropic cancer cell feature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9ce2e6325eaf9b5466f08fa222bbaf8fdd0b70" target='_blank'>
              Centrosome amplification primes ovarian cancer cells for apoptosis and potentiates the response to chemotherapy
              </a>
            </td>
          <td>
            Frances Edwards, Giulia Fantozzi, Anthony Y Simon, Jean-Philippe Morretton, A. Herbette, Andrea E. Tijhuis, René Wardenaar, Stacy Foulane, Simon Gemble, D. Spierings, F. Foijer, Odette Mariani, A. Vincent-Salomon, S. Roman-Roman, X. Sastre-Garau, O. Goundiam, R. Basto
          </td>
          <td>2024-09-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bb750f22d3a6ebea43d03722b56d7da477c099" target='_blank'>
              Causes and consequences of DNA double-stranded breaks in cardiovascular disease.
              </a>
            </td>
          <td>
            A. Marian
          </td>
          <td>2024-10-15</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The fragmentomics-based cell-free DNA (cfDNA) assays have recently demonstrated remarkable capabilities in identifying various cancers from non-conditional healthy controls. However, their accuracy in differentiating early-stage cancers from benign lesions, especially with inconclusive imaging results, remains unclear. In breast cancer, imaging-based screening methods often yield high false-positive rates, particularly in women with breast nodules, leading to unnecessary biopsies and increasing patient discomfort and healthcare burden. In this multi-center study, we enrolled 560 female participants and showed that cfDNA fragmentomics, using whole-genome sequencing, is a strong non-invasive biomarker for breast cancer. Among various cfDNA fragmentomics profiles, the fragment size ratio (FSR), fragment size distribution (FSD), and copy number variation (CNV) exhibited superior distinguishing ability compared to Griffin, motif breakpoint (MBP), and neomer features. Our cfDNA fragmentomics (cfFrag) model, which incorporates the three optimal fragmentomics features, successfully discriminated early-stage breast cancers from benign nodules, even at a low sequencing depth (3x). Notably, it demonstrated a specificity of 94.1% in asymptomatic healthy women at a 90% sensitivity for breast cancer. Furthermore, we highlight the clinical utility of the cfFrag model in predicting patient responses to neoadjuvant chemotherapy (NAC), and its integration with multimodal features, including radiological results and cfDNA methylation features, achieved high AUC values of 0.93-0.94 and 0.96, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec1ffa1e5f7be8782a9fb046c2b6f1ed5cb766" target='_blank'>
              Cell-free DNA Fragmentomics Assay to Discriminate the Malignancy of Breast Nodules and Evaluate Treatment Response
              </a>
            </td>
          <td>
            Jiaqi Liu, Yalun Li, Wanxiangfu Tang, Lijun Dai, Ziqi Jia, Heng Cao, Chenghao Li, Yuchen Liu, Yansong Huang, Jiang Wu, Dongxu Ma, Guangdong Qiao, H. Bao, Shuang Chang, Dongqin Zhu, Shanshan Yang, Xuxiaochen Wu, Xue Wu, Hengyi Xu, Hongyan Chen, Yang Shao, Xiang Wang, Zhihua Liu, Jianzhong Su
          </td>
          <td>2024-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3260115731c81c0bb245a4f0f48eb9f7d32bce58" target='_blank'>
              Single-strand DNA-binding protein suppresses illegitimate recombination in Escherichia coli, acting in synergy with RecQ helicase
              </a>
            </td>
          <td>
            Isidoro Feliciello, Sven Ljubić, E. Đermić, S. Ivanković, D. Zahradka, Damir Đermić
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA in eukaryotic genomes is under constant assault from both exogenous and endogenous sources, leading to DNA damage, which is considered a major molecular driver of ageing. Fortunately, the genome and the central exome are safeguarded against these attacks by abundant peripheral non-coding DNA. Non-coding DNA codes for small non-coding RNAs that inactivate foreign nucleic acids in the cytoplasm and physically blocks these attacks in the nucleus. Damage to non-coding DNA produced during such blockage is removed in the form of extrachromosomal circular DNA (eccDNA) through nucleic pore complexes. Consequently, non-coding DNA serves as a line of defence for the exome against DNA damage. The total amount of non-coding DNA/heterochromatin declines with age, resulting in a decrease in both physical blockage and eccDNA exclusion, and thus an increase in the accumulation of DNA damage in the nucleus during ageing and in age-related diseases. Here, we summarize recent evidence supporting a protective role of non-coding DNA in healthy and pathological states and argue that DNA damage is the proximate cause of ageing and age-related genetic diseases. Strategies aimed at strengthening the protective role of non-coding DNA/heterochromatin could potentially offer better systematic protection for the dynamic genome and the exome against diverse assaults, reduce the burden of DNA damage to the exome, and thus slow ageing, counteract age-related genetic diseases and promote a healthier life for individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7879deb4075aa031b3e715005da950caf3fffb2" target='_blank'>
              Protection of the genome and the central exome by peripheral non-coding DNA against DNA damage in health, ageing and age-related diseases.
              </a>
            </td>
          <td>
            Guo-Hua Qiu, Mingjun Fu, Xintian Zheng, Cuiqin Huang
          </td>
          <td>2024-09-26</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by fluctuations in chromosome number or structure within cells, stands out as a hallmark of cancer, enabling tumors to thrive in hostile conditions. CIN serves as a driver of genetic diversity, giving rise to clonal heterogeneity (CH). Emerging evidence points to a potential correlation between CIN, CH, and the prognosis of breast cancer (BC) patients, especially in tumors exhibiting overexpression of the human epidermal growth factor receptor 2 (HER2+). However, our understanding of the role of CIN in other subtypes of BC is limited. Furthermore, it remains unclear whether CIN levels above a certain threshold in BC tumors could adversely affect tumor growth, or if lower to moderate levels of CIN might be associated with a more favorable prognosis for BC patients compared to elevated levels. Elucidating these relationships could significantly influence risk assessment and the formulation of future therapeutic approaches targeting CIN in BC. This study aimed to assess CIN and CH in tumor tissue samples obtained from Colombian patients diagnosed with luminal A, luminal B, HER2+, or triple-negative BC, and compare them with established clinicopathological parameters. The findings of this study indicate that BC patients, exhibit intermediate CIN, high CH, and stable aneuploidy. All these characteristics were found to be related with clinicopathological features. Our results suggest that the identification of CIN, CH and aneuploidy could improve cancer risk stratification, which could help to clarify the prediction of clinical outcomes and guiding personalized therapeutic strategies for patients with different BC subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f4c53715c6bdbb1cc8b85a27c36e138c73e29" target='_blank'>
              Exploring chromosomal instability and clonal heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            Giovanny Castellanos, Laura Valentina Camargo-Herrera, N. Rangel, G. A. Jiménez-Tobón, María Martínez-Agüero, Milena Rondón-Lagos
          </td>
          <td>2024-09-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="We developed a linear amplification-mediated high-throughput genome-wide translocation sequencing method to profile Ig class-switch recombination (CSR) in human B cells in an unbiased and quantitative manner. This enables us to characterize CSR junctions resulting from either deletional recombination or inversion for each Ig class/subclass. Our data showed that more than 90% of CSR junctions detected in peripheral blood in healthy control subjects were due to deletional recombination. We further identified two major CSR junction signatures/patterns in human B cells. Signature 1 consists of recombination junctions resulting from both IgG and IgA switching, with a dominance of Sµ-Sγ junctions (72%) and deletional recombination (87%). Signature 2 is contributed mainly by Sµ-Sα junctions (96%), and these junctions were almost all due to deletional recombination (99%) and were characterized by longer microhomologies. CSR junctions identified in healthy individuals can be assigned to both signatures but with a dominance of signature 1, whereas almost all CSR junctions found in patients with defects in DNA-PKcs or Artemis, two classical nonhomologous end joining (c-NHEJ) factors, align with signature 2. Thus, signature 1 may represent c-NHEJ activity during CSR, whereas signature 2 is associated with microhomology-mediated alternative end joining in the absence of the studied c-NHEJ factors. Our findings suggest that in human B cells, the efficiency of the c-NHEJ machinery and the features of switch regions are crucial for the regulation of CSR orientation. Finally, our high-throughput method can also be applied to study the mechanism of rare types of recombination, such as switching to IgD and locus suicide switching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76f6ad58cd1fa3e33c899adc7067ec2a694fd440" target='_blank'>
              Orientation Regulation of Class-switch Recombination in Human B Cells.
              </a>
            </td>
          <td>
            Likun Du, Valentyn Oksenych, Hui Wan, Xiaofei Ye, Junchao Dong, Adam Yongxin Ye, Hassan Abolhassani, S. Vlachiotis, Xuefei Zhang, K. de la Rosa, L. Hammarström, M. van der Burg, Frederick W. Alt, Q. Pan-Hammarström
          </td>
          <td>2024-09-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Super-enhancers (SEs) have been recognized as key epigenetic regulators underlying cancer stemness and malignant traits by aberrant transcriptional control and promising therapeutic targets against human cancers. However, the SE landscape and their roles during head and neck squamous cell carcinoma (HNSCC) development especially in cancer stem cells (CSCs) maintenance remain underexplored yet. Here, we identify leukemia inhibitory factor (LIF)-SE as a representative oncogenic SE to activate LIF transcription in HNSCC. LIF secreted from cancer cells and cancer-associated fibroblasts promotes cancer stemness by driving SOX2 transcription in an autocrine/paracrine manner, respectively. Mechanistically, enhancer elements E1, 2, 4 within LIF-SE recruit SOX2/SMAD3/BRD4/EP300 to facilitate LIF transcription; LIF activates downstream LIFR-STAT3 signaling to drive SOX2 transcription, thus forming a previously unknown regulatory feedback loop (LIF-SE-LIF/LIFR-STAT3-SOX2) to maintain LIF overexpression and CSCs stemness. Clinically, increased LIF abundance in clinical samples correlate with malignant clinicopathological features and patient prognosis; higher LIF concentrations in presurgical plasma dramatically diminish following cancer eradication. Therapeutically, pharmacological targeting LIF-SE-LIF/LIFR-STAT3 significantly impairs tumor growth and reduces CSC subpopulations in xenograft and PDX models. Our findings reveal a hitherto uncharacterized LIF-SE-mediated auto-regulatory loop in regulating HNSCC stemness and highlight LIF as a novel noninvasive biomarker and potential therapeutic target for HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1045a702382129e5a5792c38e5ce8df803663" target='_blank'>
              Super-Enhancer Driven LIF/LIFR-STAT3-SOX2 Regulatory Feedback Loop Promotes Cancer Stemness in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jin Li, Yuhan Wang, Ziyu Wang, Yuxiang Wei, Pengfei Diao, Yaping Wu, Dongmiao Wang, Hongbing Jiang, Yanling Wang, Jie Cheng
          </td>
          <td>2024-08-29</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Humans are exposed to DNA alkylating agents through endogenous metabolism, environmental exposure and cancer chemotherapy. The resulting alkylated DNA adducts may elicit genome instability by perturbing DNA replication and transcription. R-loops regulate various cellular processes, including transcription, DNA repair, and telomere maintenance. However, unscheduled R-loops are also recognized as potential sources of DNA damage and genome instability. In this study, by employing fluorescence microscopy and R-loop sequencing approaches, we uncovered, for the first time, that minor-groove N2-alkyl-dG lesions elicit elevated R-loop accumulation in chromatin and in plasmid DNA in cells. We also demonstrated that the N2-alkyl-dG-induced R-loops impede transcription elongation and compromise genome integrity. Moreover, genetic depletion of DDX23, a R-loop helicase, renders cells more sensitive toward benzo[a]pyrene diolepoxide, a carcinogen that induces mainly the minor-groove N2-dG adduct. Together, our work unveiled that unrepaired minor-groove N2-alkyl-dG lesions may perturb genome integrity through augmenting R-loop levels in chromatin. Our findings suggest a potential therapeutic strategy involving the combination of R-loop helicase inhibitors with DNA alkylating drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bca69f7ad1277a7093d1fb762d63dd7f27acf5" target='_blank'>
              N 2-Alkyl-dG lesions elicit R-loop accumulation in the genome.
              </a>
            </td>
          <td>
            Yinan Wang, Feng Tang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast Cancer stands on the second position in the world in being common and women happen to have it with high rate of about five-folds around the world. The causes of occurrence can matter with different humans be it external factors or the internal genetic ones. Breast cancer is primarily driven by mutations in the BRCA1 and BRCA2 susceptibility genes. These BC susceptibility genes encode proteins critical for DNA homologous recombination repair (HRR). Poly (ADP ribose) polymerases (PARP) are the essential enzymes involved in the repairing of the damaged DNA. So the inhibition of these inhibitors can be considered as the promising strategy for targeting cancers with defective damage in the deoxyribonucleic acid. Olaparib and talazoparib are PARP inhibitors (PARPi) are being employed for the monotherapies in case of the deleterious germline HER2-negative and BRCA-mutated breast cancer. The potency of PARP for trapping on DNA and causes cytotoxicity may have difference in the safety and efficacy with the PARPi. The PARPi have been found its place in the all different types of Breast Cancers and have shown potential benefits. The purpose of this review is to provide an update on the oral poly(ADP-ribose) polymerase (PARP)inhibitors for the improvement in the treatment and management of Breast Cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45e72ba4b14c0918fa21b6e7ff5f0987ade38944" target='_blank'>
              PARP Pioneers: Using BRCA1/2 Mutation-targeted Inhibition to Revolutionize Breast Cancer Treatment.
              </a>
            </td>
          <td>
            Navneet Sharma, Akash Bhati, Shagun Aggarwal, Kamal Shah, H. K. Dewangan
          </td>
          <td>2024-10-17</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Radiotherapy (RT) is the primary treatment for diffuse midline glioma (DMG), a lethal pediatric malignancy defined by histone H3 lysine 27-to-methionine (H3K27M) mutation. Based on the loss of H3K27 trimethylation producing broad epigenomic alterations, we hypothesized that H3K27M causes a functional double-strand break (DSB) repair defect that could be leveraged therapeutically with PARP inhibitor and RT for selective radiosensitization and antitumor immune responses. Methods H3K27M isogenic DMG cells and orthotopic brainstem DMG tumors in immune deficient and syngeneic, immune competent mice were used to evaluate the efficacy and mechanisms of PARP1/2 inhibition by olaparib or PARP1 inhibition by AZD9574 with concurrent RT. Results H3K27M mutation caused an HRR defect characterized by impaired RT-induced K63-linked polyubiquitination of histone H1 and inhibition of HRR protein recruitment. H3K27M DMG cells were selectively radiosensitized by olaparib in comparison to isogenic controls, and this effect translated to efficacy in H3K27M orthotopic brainstem tumors. Olaparib and RT induced an innate immune response and induction of NK cell (NKG2D) activating ligands leading to increased NK cell-mediated lysis of DMG tumor cells. In immunocompetent syngeneic orthotopic DMG tumors, either olaparib or AZD9574 in combination with RT enhanced intratumoral NK cell infiltration and activity in association with NK cell-mediated therapeutic responses and favorable activity of AZD9574. Conclusions The HRR deficiency in H3K27M DMG can be therapeutically leveraged with PARP inhibitors to radiosensitize and induce an NK cell-mediated antitumor immune response selectively in H3K27M DMG, supporting the clinical investigation of best-in-class PARP inhibitors with RT in DMG patients. Key points H3K27M DMG are HRR defective and selectively radiosensitized by PARP inhibitor. PARP inhibitor with RT enhances NKG2D ligand expression and NK cell-mediated lysis. NK cells are required for the therapeutic efficacy of PARP inhibitor and RT. Importance of the Study Radiotherapy is the cornerstone of H3K27M-mutant diffuse midline glioma treatment, but almost all patients succumb to tumor recurrence with poor overall survival, underscoring the need for RT-based precision combination therapy. Here, we reveal HRR deficiency as an H3K27M-mediated vulnerability and identify a novel mechanism linking impaired RT-induced histone H1 polyubiquitination and the subsequent RNF168/BRCA1/RAD51 recruitment in H3K27M DMG. This model is supported by selective radiosensitization of H3K27M DMG by PARP inhibitor. Notably, the combination treatment results in NKG2D ligand expression that confers susceptibility to NK cell killing in H3K27M DMG. We also show that the novel brain penetrant, PARP1-selective inhibitor AZD9574 compares favorably to olaparib when combined with RT, prolonging survival in a syngeneic orthotopic model of H3K27M DMG. This study highlights the ability of PARP1 inhibition to radiosensitize and induce an NK cell-mediated antitumor immunity in H3K27M DMG and supports future clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa409894637aca4cc204fe5ba28199f9da5c66db" target='_blank'>
              H3K27M diffuse midline glioma is homologous recombination defective and sensitized to radiotherapy and NK cell-mediated antitumor immunity by PARP inhibition
              </a>
            </td>
          <td>
            Yupei Guo, Zian Li, L. A. Parsels, Zhuwen Wang, J. Parsels, Anushka Dalvi, Stephanie The, Nan Hu, Victoria M. Valvo, Robert Doherty, Erik Peterson, Xinjun Wang, S. Venkataraman, S. Agnihotri, S. Venneti, Daniel R. Wahl, Michael D. Green, Theodore S. Lawrence, C. Koschmann, Meredith A. Morgan, Qiang Zhang
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b57b76c24e922e56ff6f3fa5ae5de6f644999f2" target='_blank'>
              Human papillomavirus-encoded circular RNA circE7 promotes immune evasion in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Junshang Ge, Yi Meng, Jiayue Guo, Pan Chen, Jie Wang, Lei Shi, Dan Wang, Hongke Qu, Pan Wu, Chunmei Fan, Shanshan Zhang, Q. Liao, Ming Zhou, Bo Xiang, Fuyan Wang, Ming-Liang Tan, Zhaojian Gong, Wei Xiong, Zhaoyang Zeng
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Endometrial carcinoma (EC) is a prevalent gynecological malignancy that poses a significant threat to women’s health worldwide. However, its pathogenesis and underlying mechanisms remains unclear. In this study, expression quantitative trait loci data, Mendelian randomization analysis, and differential expression analysis were performed to identify potential targets. A prognostic risk signature was subsequently constructed for EC patients based on the expression of these genes. Four bioinformatics algorithms, including generalized linear model, extreme gradient boosting, support vector machine, and random forest, were used to identify hub genes in EC. The expression of ring finger protein 144A (RNF144A) was validated using quantitative real-time polymerase chain reaction. Cellular proliferation and migration ability were evaluated using CCK-8 and Transwell assays, respectively. The genes RNF144A, ketohexokinase, and Rab interacting lysosomal protein like 2 were identified as potential targets for EC. Their differential expression was observed in EC patients, and Mendelian randomization analysis revealed a negative correlation between these genes and the development of EC. Mechanistic analyses suggested a strong association between these genes and the tumor immune microenvironment. The constructed risk signature was significantly associated with the prognosis, age, cancer stage, and grade of EC patients. Furthermore, based on interacted model algorithms, RNF144A was identified as a hub gene in EC. It was found to be significantly downregulated in EC samples, and its expression was positively correlated with the stage and grade of EC patients. In vitro experiments showed that overexpression of RNF144A significantly promoted cell growth and migration in EC cells. In conclusion, this study provides insights into the molecular mechanisms underlying EC progression and identifies preliminary candidate biomarkers for the development of EC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ccab2586145d3cccf21eccbbc93cc5b295b360" target='_blank'>
              RNF144A as a potential risk gene for endometrial carcinoma: Insights from Mendelian randomization, bulk RNA sequencing, single-cell RNA, and experimental analysis
              </a>
            </td>
          <td>
            Xiqin Ruan, Ni Wang, Qingwen Xie, Yi Du
          </td>
          <td>2024-10-04</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Unexplained stillbirth is hypothesised to occur due to premature placental ageing, with unexpected deterioration of placental function for gestational age. Circular RNAs (circRNAs) are enzyme resistant RNA molecules that accumulate in ageing tissues. Furthermore, circRNAs bind gDNA directly, forming circRNA:DNA complexes which can induce DNA breaks. This study investigated circRNA accumulation with gestational age in healthy and stillbirth placentae and determined whether circRNAs directly interact with placental DNA causing DNA damage. Placenta samples (n=60 term uncomplicated; n=4 unexplained stillbirth, 23, 26, 31, 34 weeks’ gestation) were assessed for DNA damage using an alkaline Comet Assay. Expression levels of 6 candidate circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277), and their linear transcripts, were quantified using qPCR. Physical interaction of candidate circRNAs with DNA was confirmed by DNA:RNA ImmunoPrecipitation (DRIP). The effect of circ_0009000 knockdown in HEK293T cells was assessed following transfection with either a siRNA (designed to knockdown circ_0009000) or a scrambled siRNA control, at 5, 10 and 20 nM final concentrations using Lipofectamine RNAiMax. DNA damage was assessed by Comet Assay. Compared with earlier gestations (37, 38, 39 and 40 weeks’), placental DNA damage and expression of all 6 candidate circRNAs, but not their linear transcripts, were increased in 40 and 41+ weeks’ gestation samples, and in stillbirth. DRIP-qPCR signal size was significantly larger in term placentae than in enzyme-treated controls, confirming that all candidate circRNA loci bind to placental DNA. Depletion of circ_0009000 by specific siRNA in HEK293T cells, significantly reduced DNA damage compared to control. Stillbirth placentae show accelerated ageing with premature accumulation of candidate circRNAs (first evidence in humans) at levels consistent with older gestation tissue. Importantly, these circRNAs bind to DNA and circ_0009000 causes DNA breaks in placenta. Therefore, circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277) play a role in placental ageing and associate with stillbirth, likely via DNA damage. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9adf6d93429ec373710f176ef7b50a5c1efd5e40" target='_blank'>
              Circular RNAs correlate with DNA damage in ageing human placental tissue and in stillbirth
              </a>
            </td>
          <td>
            A. Arthurs, Matilda R. Jackson, Dylan McCullough, Hamish S. Scott, Christopher P. Barnett, Stuart T. Webb, Melanie D. Smith, Tanja Jankovic-Karasoulos, Gustaaf A. Dekker, Claire T. Roberts
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eaf6453d4c9d143250dede3ed41dacec77b88be" target='_blank'>
              Epigenetic landscape reorganisation and reactivation of embryonic development genes are associated with malignancy in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            S. Ghisai, L. van Hijfte, W. Vallentgoed, C. M. S. Tesileanu, I. D. de Heer, J. Kros, Marc Sanson, T. Gorlia, Wolfgang Wick, M. Vogelbaum, A. A. Brandes, Enrico Franceschi, Paul M. Clement, Anna K. Nowak, V. Golfinopoulos, M. J. van den Bent, P. French, Y. Hoogstrate
          </td>
          <td>2024-10-09</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09cf4def72c527eab48684dc58deaf767dec6890" target='_blank'>
              Exploring a new signature for lung adenocarcinoma: analyzing cuproptosis-related genes through Integrated single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Jiangtao Liu, Wei Xia, Feng Xue, Chen Xu
          </td>
          <td>2024-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer (PCa) is among the three main types of cancer. Although prostate-specific antigen (PSA) is routinely tested, it has disadvantages, such as poor prognostic ability. Therefore, finding more PCa markers and therapeutic targets remains a subject of study. CircRNAs have been found to have regulatory roles in various diseases, such as diabetes, Central Nervous System (CNS) neuropathy, etc. where their application in cancer is even more valuable. Therefore, this paper aims to search for differentially expressed circRNAs in PCa and find downstream targeting pathways related to autophagy. Method By detecting the expression of circRNA in the samples, hsa_circ_0119816 was finally identified as the research target. The properties of circRNA were verified by RNase R, actinomycin D, and fluorescence in situ hybridization (FISH). The downstream target miRNAs and target proteins were predicted by an online database, and the targeting relationship was verified using dual luciferase and RNA Immunoprecipitation. The effects of circRNAs and their downstream signalling pathways on prostate cancer cell proliferation, migration, EMT and autophagy were examined by CCK-8, Transwell, immunofluorescence and Western blotting. Results CircBIRC6 is highly expressed in prostate cancer samples. Knockdown of its expression inhibits cell proliferation, invasion, EMT and autophagy and promotes apoptosis. CircBIRC6/miRNA-574-5p/DNAJB1 is a molecular axis that regulates prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d1bad6dbf3ace4247e9abe87ef37ab3c3b685" target='_blank'>
              CircBIRC6 affects prostate cancer progression by regulating miR-574-5p and DNAJB1
              </a>
            </td>
          <td>
            Bin Zhao, Jinye Yang, Fengming Ran, Yuanlong Shi, Libo Yang, Yuanpeng Duan, Zhiyu Shi, Xin Li, Jianpeng Zhang, Zhiyao Li, Jiansong Wang
          </td>
          <td>2024-09-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="The incidence of colorectal cancer (CRC) among individuals under age 50, or early-onset CRC (EOCRC), has been rising over the past few decades for unclear reasons, and the etiology of the disease remains largely unknown. Known genetic risk factors do not explain this increase, pointing to possible environmental and as-yet unidentified genetic contributors and their interactions. Previous research linked genetic variation on chromosome 6 to increased CRC risk. This region harbors multiple immune genes, including the gene encoding Major Histocompatibility Complex (MHC) class I polypeptide-related sequence A (MICA). MICA is a polygenic ligand for the Natural Killer Group 2D receptor (NKG2D), a receptor expressed on Natural Killer (NK) cells and other lymphocytes. Given that intra-tumoral NK cell infiltration correlates with favorable CRC outcomes, we hypothesized that germline genetic variation in MICA could influence CRC risk. In a discovery set of 40,125 cases and controls, we show that the minor G allele at Chr6:31373718C>G (hg19) is associated with increased risk for CRC (odds ratio (OR) = 1.09, 95% confidence interval (CI) 1.04 - 1.15, p = 0.0009). The effect is stronger in EOCRC (OR = 1.26, 95% CI 1.08 - 1.44, p = 0.0023) than in those 50 and over (OR = 1.07, 95% CI 1.02 - 1.13; p = 0.012) (Ratio of ORs = 1.32, 95% CI 1.14 - 1.52, p = 0.0002). In an independent validation set of 77,983 cases and controls, the adjusted interaction by age -of-onset was significant at OR = 1.15 (95% CI 1.03 - 1.34, p = 0.0150) with a higher risk in EOCRC. Expression quantitative trait locus analysis in normal colonic epithelia showed that MICA RNA expression decreases linearly with each additional copy of the minor G allele (p = 3.345 x 10-18). Bulk RNA analysis of the tumor immune microenvironment revealed that tumors from patients with CG or GG genotypes have lower resting and activated NK cell infiltration as compared to tumors from patients with CC genotype. Multiplex immunofluorescence analysis demonstrated that patients with a G allele (i.e. CG or GG genotype, but not CC genotype) have a statistically significant decrease in the number of NK cells in tumor compared to adjacent normal colonic mucosa. Taken together, population-based epidemiologic, molecular, genetic, cellular and immunologic evidence demonstrate that MICA genotype is associated with increased risk of EOCRC and reduced number of NK cells in colorectal tumors, suggesting that patients with a G allele have altered NK cell-mediated immunosurveillance. These novel findings suggest that EOCRC may have a previously unrecognized innate immune-mediated etiology which merits further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5c740e57eda493d70527270b12cf7926d01624" target='_blank'>
              Genetic Variation and Regulation of MICA Alters Natural Killer Cell-Mediated Immunosurveillance in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Heather M. McGee, J. Bonner, Yubo Fu, Oscar Colunga Flores, S. Lindsey, Lawrence Shaktah, Ferran Moratalla-Navarro, Yasmin Kamal, Kevin K. Tsang, C. P. Walker, G. Idos, K. McDonnell, H. Rennert, Elizabeth L Barry, Hermann Brenner, Daniel D. Buchanan, P. Campbell, A. T. Chan, J. Chang-Claude, Jane C Figueiredo, M. Gago-Domínguez, M. Hoffmeister, Li Hsu, J. Huyghe, M. Jenkins, L. Marchand, Heinz-Josef Lenz, Li Li, A. Lindblom, Yun Ru Liu, B. Lynch, Christina C. Newton, K. Offit, Shuji Ogino, Rebeca Sanz Pamplona, Andrew J. Pellatt, Paul D. P. Pharoah, Amanda I. Phipps, Lorena Reynaga, Allyson S. Templeton, C. Um, Alicja Wolk, M. Woods, A. H. Wu, Yen Yun, Weishan Zheng, Terence M. Williams, David V Conti, Ulrike Peters, F. Lejbkowicz, J. Greenson, S. Schmit, W. Gauderman, Stanley R. Hamilton, Victor Moreno, Gadi Rennert, Stephen B. Gruber, †Gad Rennert, B. Gruber, MD M. McGee, MD Stephen B. Gruber
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="High Grade Serous Ovarian Cancer (HG-SOC), the most prevalent and aggressive gynecological malignancy, is marked by ubiquitous loss of functional p53, largely due to point mutations that arise very early in carcinogenesis. These mutations often lead to p53 protein misfolding and subsequent aggregation, yet the alterations in intracellular p53 dynamics throughout ovarian cancer progression remain poorly understood. HG-SOC originates from the fallopian tube epithelium, with a well-documented stepwise progression beginning with early pre-malignant p53 signatures. These signatures represent largely normal cells that express and accumulate mutant p53, which then transform into benign serous tubal intraepithelial lesions (STIL), progress into late pre-malignant serous tubal intraepithelial carcinoma (STIC), and ultimately lead to HGSOC. Here, we show that the transition from folded, soluble to aggregated mutant p53 occurs during the malignant transformation of benign precursor lesions into HGSOC. We analyzed fallopian tube tissue collected from ten salpingo-oophorectomy cases and determined the proportion of cells carrying soluble versus mis-folded/mutant p53 through conformation-sensitive staining and quantification. Misfolded p53 protein, prone to aggregation, is present in STICs and HG-SOCs, but notably absent from preneoplastic lesions and surrounding healthy tissue. Overall, our results indicate that aggregation of mutant p53 is a structural defect that distinguishes preneoplastic early lesions from late premalignant and malignant ones, offering a potential treatment window for targeting p53 aggregation and halting ovarian cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e004df7ee189f194b9dc43a60b80d81862858bf" target='_blank'>
              Mutant p53 Misfolding and Aggregation Precedes Transformation into High-Grade Serous Ovarian Carcinoma
              </a>
            </td>
          <td>
            S. Sartini, Lexi Omholt, Neda A Moatamed, A. Soragni
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Forkhead box A1 (FOXA1) acts as a pioneer factor for Estrogen Receptor alpha (ERα), playing a crucial role in determining where ERα binds to DNA and regulates gene expression in breast cancer cells. A pioneer factor is a class of transcription factors (TF) that is responsible for opening the chromatin to allow other TFs to attach and activate transcription of that gene. FOXA1 acts as a facilitator for ERα-DNA interactions. It binds to specific DNA regions, opening the chromatin for ERα to bind. It “pre-marks” these sites in the genome even in the absence of estrogen. In fact, FOXA1 is frequently overexpressed in ER+ breast cancers, suggesting it has a crucial role in driving tumor growth. As such, FOXA1 is an attractive therapeutic target that already serves as a great biomarker for many types of breast cancers. Inhibiting FOXA1 can potentially block ER activity and inhibit growth of ER+ breast cancer, especially in those resistant to anti-estrogen therapies, such as tamoxifen. In this project, we identified sites that FOXA1 opens and correlated them with SNP mutations commonly found in the Hispanic population. The presence of these SNPs portends a worse outcome in Hispanic patients. To assess which open chromatin regions are crucial for breast cancer pathogenesis, a ChIP-seq analysis was done on vehicle-treated and estrogen-treated samples to determine FOXA1 binding sites. After identifying gained, lost, and differentially expressed peaks set, these were compared to genome-wide association studies (GWAS) of Hispanic patients to develop an understanding of how much overlap is common between sites of mutations and open sections of chromatin due to FOXA1. Through our analysis, we uncovered a subset of FOXA1 binding sites that overlap with sites of inherited mutation in Hispanic patients. This informs on the subset of genes that are linked to breast cancer and potential therapeutic targets.
 Citation Format: Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil. FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e221203c9b5991c2cb9afa79cdfe37b8baf68cb3" target='_blank'>
              Abstract B011: FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations
              </a>
            </td>
          <td>
            Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Ovarian cancer (OC) is a global malignancy characterized by metastatic invasiveness and recurrence. Long non-coding RNAs (lncRNAs) and Telomeres are closely connected with several cancers, but their potential as practical prognostic markers in OC is less well-defined. Methods Relevant mRNA and clinical data for OC were sourced from The Cancer Genome Atlas (TCGA) database. The telomere-related lncRNAs (TRLs) prognostic model was established by univariate/LASSO/multivariate regression analyses. The effectiveness of the TRLs model was evaluated and measured via the nomogram. Additionally, immune infiltration, tumor mutational load (TMB), and drug sensitivity were evaluated. We validated the expression levels of prognostic genes. Subsequently, PTPRD-AS1 knockdown was utilized to perform the CCK8 assay, colony formation assay, transwell assay, and wound healing assay of CAOV3 cells. Results A six-TRLs prognostic model (PTPRD-AS1, SPAG5-AS1, CHRM3-AS2, AC074286.1, FAM27E3, and AC018647.3) was established, which can effectively predict patient survival rates and was successfully validated using external datasets. According to the nomogram, the model could effectively predict prognosis. Furthermore, we detected the levels of regulatory T cells and M2 macrophages were comparatively higher in the high-risk TRLs group, but the levels of activated CD8 T cells and monocytes were the opposite. Finally, the low-risk group was more sensitive to anti-cancer drugs. The mRNA levels of PTPRD-AS1, SPAG5-AS1, FAM27E3, and AC018647.3 were significantly over-expressed in OC cell lines (SKOV3, A2780, CAOV3) in comparison to normal IOSE-80 cells. AC074286.1 were over-expressed in A2780 and CAOV3 cells and CHRM3-AS2 only in A2780 cells. PTPRD-AS1 knockdown decreased the proliferation, cloning, and migration of CAOV3 cells. Conclusion Our study identified potential biomarkers for the six-TRLs model related to the prognosis of OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3abf27265900e867b786b472cf6d79fdb5355c6" target='_blank'>
              Identification of telomere-related lncRNAs and immunological analysis in ovarian cancer
              </a>
            </td>
          <td>
            Weina Xu, Shuliu Sang, Jun Wang, Shanshan Guo, Xiao Zhang, Hailun Zhou, Yijia Chen
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA double‐strand breaks are known to be the most lethal kind of DNA damage as they cause genomic instability. Visualization and quantification of these breaks are possible via staining of the phosphorylated histone H2A variant H2AX at serine 139 with the anti‐phospho‐histone H2AX (γ‐H2AX) antibody.The literature was searched with the following keywords: DNA damage, double‐strand break, PBMC, DNA repair, H2AX, γ‐H2AX.This review discusses various methods for the quantification of γ‐H2AX and the use of γ‐H2AX in cell culture, tissue biopsies, and peripheral blood mononuclear cells. The current research into basal DNA damage as quantified by γ‐H2AX is discussed in relation to cancer.This review suggests that γ‐H2AX has the potential for use in the prediction of cancer risk when applied to healthy tissue and peripheral blood mononuclear cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33e877d68f634e794ddbbc9b19ea6bbbfae2515" target='_blank'>
              Considerations for the Use of the DNA Damage Marker γ‐H2AX in Disease Modeling, Detection, Diagnosis, and Prognosis
              </a>
            </td>
          <td>
            Holly Hosking, Wayne Pederick, P. Neilsen, Andrew Fenning
          </td>
          <td>2024-08-29</td>
          <td>Aging and Cancer</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9411c9f3fb4c71b345263ce180b256a79c6227" target='_blank'>
              Base excision repair and double strand break repair cooperate to modulate the formation of unrepaired double strand breaks in mouse brain
              </a>
            </td>
          <td>
            Aris A. Polyzos, Ana Cheong, Jung Hyun Yoo, Lana Blagec, Sneh M. Toprani, Zachary D. Nagel, Cynthia T McMurray
          </td>
          <td>2024-09-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Childhood cancer survivors treated with radiotherapy face the likelihood of long-term complications, including mutations. In order to assess the long-term effect of radiotherapy on the capacity of cells to repair their DNA double-strand breaks (DSBs), five lambs received medulloblastoma radiotherapy to the thecal sac, with three lambs serving as controls. Four years later, lung biopsies were taken and fibroblast cells were amplified and re-irradiated. The cells from the previously treated sheep showed a significant impairment of their DNA DSB repair mechanism, highlighting a potential increase in radiosensitivity. Our results show that previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts. Abstract Children with cancer previously treated with radiotherapy face the likelihood of side effects that can be debilitating or fatal. This study aimed to assess the long-term effect of medulloblastoma radiotherapy on the DNA double-strand break (DSB) repair capability of primary fibroblasts derived from lung biopsies of previously irradiated young sheep. This study included biopsies from three control and five irradiated sheep. The treated sheep had previously received spinal radiotherapy at a total dose of 28 Gy, which is equivalent to pediatric medulloblastoma treatment. Lung biopsies were taken 4 years post-irradiation from high-dose (HD, >18 Gy) and low-dose (LD, <2 Gy) regions. Fifteen cell lines were extracted (six control, four LD and five HD). The cells were irradiated, and DNA DSB repair was analyzed by immunofluorescence. Clonogenic, trypan blue and micronuclei assays were performed. Both the HD and LD cell lines had a significantly higher number of residual γH2AX foci 24 h and a significant decrease in pATM activity post-irradiation compared to the control. There was no statistically significant difference in the clonogenic assay, trypan blue and micronuclei results. Our study showed that a previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5ea88bd1bedea4941e4772289fe7a7d48fca439" target='_blank'>
              Impaired DNA Double-Strand Break Repair in Irradiated Sheep Lung Fibroblasts: Late Effects of Previous Irradiation of the Spinal Thecal Sac
              </a>
            </td>
          <td>
            B. Youssef, C. Feghaly, Joelle Al Choboq, J. Bou-Gharios, R. Challita, J. Azzi, H. Bou Hadir, F. Abi Antoun, T. Araji, Phillip J. Taddei, F. Geara, Pierre Sfeir, Abdo Jurjus, W. Abou-Kheir, Larry Bodgi
          </td>
          <td>2024-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Telomeres, the protective structures at the ends of chromosomes, are crucial for maintaining cellular longevity and genome stability. Their proper function depends on tightly regulated processes of replication, elongation, and damage response. The shelterin complex, especially Telomere Repeat-binding Factor 1 (TRF1) and TRF2, plays a pivotal role in telomere protection and has emerged as a potential anti-cancer target for drug discovery. These proteins bind to the repetitive telomeric DNA motif TTAGGG, facilitating the formation of protective structures and recruitment of other telomeric proteins. Structural methods and advanced imaging techniques have provided insights into telomeric protein-DNA interactions, but probing the dynamic processes requires single-molecule approaches. Tools like magnetic tweezers, optical tweezers, and atomic force microscopy (AFM) have been employed to study telomeric protein-DNA interactions, revealing important details such as TRF2-dependent DNA distortion and telomerase catalysis. However, the preparation of single-molecule constructs with telomeric repetitive motifs continues to be a challenging task, potentially limiting the breadth of studies utilizing single-molecule mechanical methods. To address this, we developed a method to study interactions using full-length human telomeric DNA with magnetic tweezers. This protocol describes how to express and purify TRF2, prepare telomeric DNA, set up single-molecule mechanical assays, and analyze data. This detailed guide will benefit researchers in telomere biology and telomere-targeted drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f2fc75b3e952e11dfa401640112a1b316a2cff" target='_blank'>
              Analyzing Telomeric Protein-DNA Interactions Using Single-Molecule Magnetic Tweezers.
              </a>
            </td>
          <td>
            Han Gao, Yanling Liu, Zhongbo Yu
          </td>
          <td>2024-08-30</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited. Methods In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC. Results Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis. Conclusion These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Although the concept of cancer stem cells is still controversial, previous studies have shown that blood cancers, as well as specific types of solid cancers such as colorectal cancer, rely on stem cells during the onset of tumor growth and further tumor development. Moreover, resistance to therapeutic treatment in leukemias such as acute myeloid leukemia and in colorectal cancer can be attributed to a small population of cells with stemness properties known as minimal residual disease. In this review, we look back on the discovery of cancer stem cells and the contribution of the findings in blood cancer to a parallel discovery in solid cancers. We focus on CD44 as a stem cell marker, both in blood cancers and in several types of solid cancers, particularly of the gastro-intestinal tract. This review highlights newly discovered molecular mechanisms of action of CD44 which indicate that CD44 has indeed a function in stemness, stem cell maintenance and drug resistance. We attempt here to make the link between the functions of CD44 isoforms in stemness and their involvement in specific steps of tumor growth and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8580e7867b570560c73223a09cea13edca2587fc" target='_blank'>
              CD44: A Stemness Driver, Regulator and Marker - all in one?
              </a>
            </td>
          <td>
            S. J. Sonnentag, Nagwa S M Ibrahim, V. Orian-Rousseau
          </td>
          <td>2024-10-04</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50a7e64600927dc10c8fe8c8c8d1cc7dd03965f" target='_blank'>
              PHF6 cooperates with SWI/SNF complexes to facilitate transcriptional progression
              </a>
            </td>
          <td>
            Priya Mittal, Jacquelyn A. Myers, Raymond D. Carter, Sandi Radko-Juettner, Hayden A Malone, Wojciech Rosikiewicz, Alexis N Robertson, Zhexin Zhu, Ishwarya Narayanan, Baranda S Hansen, Meadow Parrish, N. Bhanu, Robert J. Mobley, J. Rehg, Beisi Xu, Yiannis Drosos, Shondra M. Pruett-Miller, Mats Ljungman, Benjamin A. Garcia, Gang Wu, Janet F Partridge, Charles W. M. Roberts
          </td>
          <td>2024-08-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Genome-wide association studies (GWAS) have identified independent signals at 5p15.33 across numerous cancers, with protective alleles for one cancer often conferring risk for another. Many of these associations are thought to act via allele-specific alterations in the cis- regulation of target genes. Transcriptomic analyses in multiple tissue types implicate two plausible target genes: TERT and CLPTM1L. However, the mechanisms linking cancer susceptibility alleles to their presumed target genes, along with the observed antagonistic pleiotropy between different cancers remain elusive. With this in mind, we employed a comprehensive approach, integrating statistical fine-mapping of GWAS summary statistics with massively parallel reporter assays (MPRA) and a densely tiled CRISPRi screen performed across eight cell lines, representative of four distinct cancers associated with 5p15.33 alleles – pancreatic, melanoma, lung, and bladder cancer. Integrative screening identified up to 30 variants across multiple signals in the locus with putative allele-specific transcriptional activity. Furthermore, a variable number tandem repeat (VNTR) in intron 9 of CLPTM1L emerged as a potent transcriptional enhancer in CRISPRi screening. Leveraging PacBio sequencing, VNTR alleles of European ancestry samples from the 1000 Genomes Project were genotyped and found to be highly polymorphic, with longer VNTRs linked to the haplotype tagged by rs31490-A (risk increasing for pancreatic cancer). Subsequent imputation into a pancreatic cancer GWAS of 9,040 cases and 12,496 controls revealed an association between longer VNTRs and risk of pancreatic cancer (OR for VNTR in the 75th percentile = 1.17, 95% CI = 1.13-1.22, P = 2.21x10-12). The significance of the association for the lead SNP rs31490 after conditioning on the VNTR genotype dropped to P= 2.20x10-5, suggesting that VNTR alleles merit consideration as part of the credible variant set for pancreatic cancer. Targeted CRISPRi assays suggested VNTR-mediated cis-regulation of both TERT and CLPTM1L in PANC-1 and MIA PaCa-2 pancreatic cancer cell lines, while dual luciferase reporter assays in the same cell lines revealed that VNTR enhancer activity is mediated by Hippo pathway transcription factors. In ongoing experiments, we aim to expand on luciferase reporter assays and DNA-protein binding assays in relevant cell lines with the aim of not only deciphering how longer VNTRs differentially exert their transcriptional effects compared to shorter alleles but also characterizing the divergent directions of effect between different cancer types. These findings underscore the proposition that pancreatic cancer susceptibility at the 5p15.33 locus may be mediated by both SNPs and an uncharted class of variants, yet to be characterized in the post-GWAS landscape.
 Citation Format: Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E Connelly, Mai Xu, Irene Collins, Jun Zhong, Jason Hoskins, Michelle Ho, Brenen Papenburg, Mark M Iles, Matthew H Law, Maria-Teresa Landi, Rachael Stolzenber- Solomon, Brian M Wolpin, Alison P Klein, Nick Orr, Stephen J Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, Laufey T Amundadottir. Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0777eadfb43a66af250607da2ed11cfe90c7930b" target='_blank'>
              Abstract C055: Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element
              </a>
            </td>
          <td>
            Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E. Connelly, Mai Xu, I. Collins, Jun Zhong, Jason W. Hoskins, Michelle Ho, Brenen Papenburg, M. Iles, M. Law, Marion Landi, Rachael Stolzenber- Solomon, B. Wolpin, Alison P Klein, N. Orr, S. Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, L. Amundadottir
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Alterations in deoxyribonucleoside triphosphate (dNTP) pools have been linked to increased mutation rates and genome instability in unicellular organisms and cell cultures. However, the role of dNTP pool changes in tumor development in mammals remains unclear. In this study, we present a mouse model with a point mutation at the allosteric specificity site of ribonucleotide reductase, RRM1-Y285A. This mutation reduced ribonucleotide reductase activity, impairing the synthesis of deoxyadenosine triphosphate (dATP) and deoxyguanosine triphosphate (dGTP). Heterozygous Rrm1+/Y285A mice exhibited distinct alterations in dNTP pools across various organs, shorter lifespans and earlier tumor onset compared with wild-type controls. Mutational spectrum analysis of tumors revealed two distinct signatures, one resembling a signature extracted from a human cancer harboring a mutation of the same amino acid residue in ribonucleotide reductase, RRM1Y285C. Our findings suggest that mutations in enzymes involved in dNTP metabolism can serve as drivers of cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe3b3a3839a433e41d5ae4062ca8fa74fde87a0" target='_blank'>
              Altered dNTP pools accelerate tumor formation in mice.
              </a>
            </td>
          <td>
            Phong Tran, Pradeep Mishra, Leonard G. Williams, Roman Moskalenko, Sushma Sharma, A. K. Nilsson, Danielle L. Watt, Pernilla Andersson, Anders Bergh, Z. Pursell, A. Chabes
          </td>
          <td>2024-10-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Breast cancer is one of the most prevalent types of cancer and a leading cause of cancer-related death among females worldwide. Anoikis, a specific type of apoptosis that is triggered by the loss of anchoring between cells and the native extracellular matrix, plays a vital role in cancer invasion and metastasis. However, studies that focus on the prognostic values of anoikis-related genes (ARGs) in breast cancer are scarce. Methods Gene expression data were obtained from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), and Molecular Taxonomy of Breast Cancer International Consortium (METABRIC) databases. Five anoikis-related signatures (ARS) were selected from ARGs through univariate Cox regression analysis, LASSO regression analysis, and multivariate Cox regression analysis. Subsequently, an ARGs risk score model was established, and breast cancer patients were divided into high and low risk groups. The correlation between risk groups and overall survival (OS), tumor mutation burden (TMB), tumor microenvironment (TME), stemness, and drug sensitivity were analyzed. Moreover, RT-qPCR was performed to verify the gene expression levels of the five ARS in breast cancer tissues. Furthermore, a nomogram model was constructed based on ARGs risk score and clinicopathological factors. Results A novel ARGs risk score model was constructed based on five ARS (CEMIP, LAMB3, CD24, PTK6, and PLK1), and breast cancer patients were divided into high and low risk groups. Correlation analysis showed that the high and low risk groups had different OS, TMB, TME, stemness, and drug sensitivity. Both the ARGs risk score model and the nomogram showed promising prognosis predictive value in breast cancer. Conclusion ARS could be used as promising biomarkers for breast cancer prognosis predication and treatment options selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2304d0d98fc1550ec1a30e39d41f3e284722407" target='_blank'>
              Identification of Novel Anoikis-Related Gene Signatures to Predict the Prognosis, Immune Microenvironment, and Drug Sensitivity of Breast Cancer Patients
              </a>
            </td>
          <td>
            Jiena Liu, Hao Wu, Qin Wang, Shengye Jin, Siyu Hou, Zibo Shen, Liuying Zhao, Shouping Xu, Da Pang
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a–RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metapolycentromeres consist of multiple sequential domains of centromeric chromatin associated with a centromere-specific variant of histone H3 (CENP-A), functioning collectively as a single centromere. To date, they have been revealed in nine flowering plant, five insect and six vertebrate species. In this paper, we focus on their structure and possible mechanisms of emergence and evolution. The metapolycentromeres may vary in the number of centromeric domains and in their genetic content and epigenetic modifications. However, these variations do not seem to affect their function. The emergence of metapolycentromeres has been attributed to multiple Robertsonian translocations and segmental duplications. Conditions of genomic instability, such as interspecific hybridization and malignant neoplasms, are suggested as triggers for the de novo emergence of metapolycentromeres. Addressing the “centromere paradox” – the rapid evolution of centromeric DNA and proteins despite their conserved cellular function – we explore the centromere drive hypothesis as a plausible explanation for the dynamic evolution of centromeres in general, and in particular the emergence of metapolycentromeres and holocentromeres. Apparently, metapolycentromeres are more common across different species than it was believed until recently. Indeed, a systematic review of the available cytogenetic publications allowed us to identify 27 candidate species with metapolycentromeres. Тhe list of the already established and newly revealed candidate species thus spans 27 species of flowering plants and eight species of gymnosperm plants, five species of insects, and seven species of vertebrates. This indicates an erratic phylogenetic distribution of the species with metapolycentromeres and may suggest an independent emergence of the metapolycentromeres in the course of evolution. However, the current catalog of species with identified and likely metapolycentromeres remains too short to draw reliable conclusions about their evolution, particularly in the absence of knowledge about related species without metapolycentromeres for comparative analysis. More studies are necessary to shed light on the mechanisms of metapolycentromere formation and evolution. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a7898b3ea2f2d63e1ff7e6046657eb47951d50" target='_blank'>
              Structure and evolution of metapolycentromeres
              </a>
            </td>
          <td>
            E. Grishko, P. Borodin
          </td>
          <td>2024-10-08</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28f4fc739f098123caf0e6c10a4c334ed4246af1" target='_blank'>
              Discovery of the DNA-PKcs inhibitor DA-143 which exhibits enhanced solubility relative to NU7441
              </a>
            </td>
          <td>
            Zachary J Waldrip, Baku Acharya, Daniel Armstrong, Maha Hanafi, Randall R. Rainwater, Sharon Amole, Madeline Fulmer, Ana Clara Azevedo-Pouly, Alaina Burns, L. Burdine, Brendan Frett, M. Burdine
          </td>
          <td>2024-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec25d0d3af6b468f3736b6388bc33a3af02a628" target='_blank'>
              DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor
              </a>
            </td>
          <td>
            Ninoslav Pandiloski, V. Horváth, Ofelia Karlsson, Symela Koutounidou, F. Dorazehi, Georgia Christoforidou, Jon Matas-Fuentes, Patricia Gerdes, R. Garza, M. Jönsson, Anita Adami, Diahann A. M. Atacho, Jenny G. Johansson, Elisabet Englund, Z. Kokaia, Johan Jakobsson, C. Douse
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Double-strand breaks (DSBs) are genotoxic DNA lesions that pose significant threats to genomic stability, necessitating precise and efficient repair mechanisms to prevent cell death or mutations. DSBs are repaired through nonhomologous end-joining (NHEJ) or homology-directed repair (HDR), which includes homologous recombination (HR) and single-strand annealing (SSA). CtIP and Rif1 are conserved proteins implicated in DSB repair pathway choice, possibly through redundant roles in promoting DNA end-resection required for HDR. Although the roles of these proteins have been well-established in other organisms, the role of Rif1 and its potential redundancies with CtIP in Drosophila melanogaster remain elusive. To examine the roles of DmCtIP and DmRif1 in DSB repair, this study employed the direct repeat of white (DR-white) assay, tracking across indels by decomposition (TIDE) analysis, and P{wIw_2 kb 3'} assay to track repair outcomes in HR, NHEJ, and SSA, respectively. These experiments were performed in DmCtIPΔ/Δ single mutants, DmRif1Δ/Δ single mutants, and DmRif1Δ/Δ; DmCtIPΔ/Δ double mutants. This work demonstrates significant defects in both HR and SSA repair in DmCtIPΔ/Δ and DmRif1Δ/Δ single mutants. However, experiments in DmRif1Δ/Δ; DmCtIPΔ/Δ double mutants reveal that DmCtIP is epistatic to DmRif1 in promoting HDR. Overall, this study concludes that DmRif1 and DmCtIP do not perform their activities in a redundant pathway, but rather DmCtIP is the main driver in promoting HR and SSA, most likely through its role in end resection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08e472bf52ff9285514c3e4d9737736fd04e7317" target='_blank'>
              The epistatic relationship of Drosophila melanogaster CtIP and Rif1 in homology-directed repair of DNA double-strand breaks.
              </a>
            </td>
          <td>
            Makenzie S Thomas, Gautham S Pillai, Margaret A Butler, Joel Fernandez, Jeannine R. LaRocque
          </td>
          <td>2024-10-14</td>
          <td>G3</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Gene expression patterns are very sensitive to external influences and are reflected in phenotypic changes. It was previously described that transferring melanoma cells from a plastic surface to Matrigel led to formation of de novo vascular networks—vasculogenic mimicry—that are characteristic to a stemness phenotype in aggressive tumors. Up to now there was no detailed data about the gene signature accompanying this process. Here, we show that this transfer shortly led to extremely strong epigenetic changes in gene expression in the melanoma cells. We observed that on Matrigel numerous genes controlling ribosome biogenesis were upregulated. However, most of the activated genes were inhibitors of the differentiation genes (ID1, ID2, and ID3). At the same time, the genes that control differentiation were downregulated. Both the upregulated and the downregulated genes are simultaneously targeted by different transcription factors shaping sets of co-expressed genes. The specific group of downregulated genes shaping contacts with rDNA genes are also associated with the H3K27me3 mark and with numerous lincRNAs and miRNAs. We conclude that the stemness phenotype of melanoma cells is due to the downregulation of developmental genes and formation of dedifferentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e3983f49ac992e0e94ffb5e7e55f9651ff1ef6" target='_blank'>
              Strong Activation of ID1, ID2, and ID3 Genes Is Coupled with the Formation of Vasculogenic Mimicry Phenotype in Melanoma Cells
              </a>
            </td>
          <td>
            N. Tchurikov, Amalia A. Vartanian, E. S. Klushevskaya, I. R. Alembekov, A. N. Kretova, Viktoriya N. Lukicheva, V. Chechetkin, G. I. Kravatskaya, V. S. Kosorukov, Y. V. Kravatsky
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are non-canonical nucleic acid structures that form in guanine (G)-rich genomic regions. X-linked dystonia parkinsonism (XDP) is an inherited neurodegenerative disease in which a SINE-VNTR-Alu (SVA) retrotransposon, characterised by amplification of a G-rich repeat, is inserted into the coding sequence of TAF1, a key partner of RNA polymerase II. XDP SVA alters TAF1 expression, but the cause of this outcome in XDP remains unknown. To assess whether G4s form in XDP SVA and affect TAF1 expression, we first characterised bioinformatically predicted XDP SVA G4s in vitro. We next showed that highly stable G4s can form and stop polymerase amplification at the SVA region from patient-derived fibroblasts and neural progenitor cells. Using chromatin immunoprecipitazion (ChIP) with an anti-G4 antibody coupled to sequencing or quantitative PCR, we showed that XDP SVA G4s are folded even when embedded in a chromatin context in patient-derived cells. Using the G4 ligands BRACO-19 and quarfloxin and total RNA-sequencing analysis, we showed that stabilisation of the XDP SVA G4s reduces TAF1 transcripts downstream and around the SVA, and increases upstream transcripts, while destabilisation using the G4 unfolder PhpC increases TAF1 transcripts. Our data indicate that G4 formation in the XDP SVA is a major cause of aberrant TAF1 expression, opening the way for the development of strategies to unfold G4s and potentially target the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5192db61c04dc7aaf8c6132c02058635a498b855" target='_blank'>
              G-quadruplexes in an SVA retrotransposon cause aberrant TAF1 gene expression in X-linked dystonia parkinsonism.
              </a>
            </td>
          <td>
            Giulia Nicoletto, Marianna Terreri, Ilaria Maurizio, E. Ruggiero, F. Cernilogar, Christine A. Vaine, Maria Vittoria Cottini, Irina Shcherbakova, Ellen B. Penney, Irene Gallina, David Monchaud, D. C. Bragg, Gunnar Schotta, Sara N. Richter
          </td>
          <td>2024-09-17</td>
          <td>Nucleic acids research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Cellular plasticity plays critical roles in tissue regeneration, tumour progression and therapeutic resistance. However, the mechanism underlying this cell state transition remains elusive. Here, we show that the transcription factor SOX2 induces fetal reprogramming and reversible dormancy in colorectal cancer (CRC). SOX2 expression correlates with fetal reprogramming and poor prognosis in human primary and metastatic colorectal adenocarcinomas. In mouse CRC models, rare slow-cycling SOX2+SCA1+ cells are detected in early Apc-deleted mouse tumours that undergo slow clonal expansion over time. In contrast, the SOX2+ clones were found proliferative in advanced Apc-/-;KrasG12D/+;p53-/-;Tgfbr2-/-(AKPT) tumours, accompanied by dynamic cell state reprogramming from LGR5+ to LGR5-SCA1+ cells. Using transgenic mouse models, we demonstrate that ectopic expression of SOX2 inhibits intestinal lineage differentiation and induces fetal reprogramming. SOX2+ cells adopt dynamic cell cycle states depending on its expression level. High SOX2 expression results in hyperproliferation, whereas low SOX2 levels induce senescence-mediated dormancy. We find that loss of p53 can reverse SOX2-induced senescence, in line with the dormant cell state exit of the SOX2+ cells observed in advanced tumours. Finally, SOX2 expression is induced by 5-FU treatment in CRC. SOX2-expressing organoids exhibit increased tolerance to chemotherapy treatment, whilst deletion of SOX2 in AKPT tumour organoids sensitises drug responses. We propose that SOX2-induced plasticity and reversible dormancy promotes tumour progression and drug tolerance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2c2dcfb70e2e15d1ce94c401c626a55033b7a7" target='_blank'>
              SOX2 drives fetal reprogramming and reversible dormancy in colorectal cancer
              </a>
            </td>
          <td>
            Anna Baulies, V. Moncho-Amor, Diana Drago-Garcia, Anna Kucharska, Probir Chakravarty, M. Moreno-Valladares, Sara Cruces-Salguero, Florian Hubl, Colin Hutton, Hanue Kim, A. Matheu, Robin Lovell-Badge, V. Li
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Aldehyde exposure has been shown to lead to the formation of DNA damage comprising of DNA-protein crosslinks (DPCs), base adducts and interstrand or intrastrand crosslinks. DPCs have recently drawn more attention because of recent advances in detection and quantification of these adducts. DPCs are highly deleterious to genome stability and have been shown to block replication forks, leading to wide-spread mutagenesis. Cellular mechanisms to prevent DPC-induced damage include excision repair pathways, homologous recombination, and specialized proteases involved in cleaving the covalently bound proteins from DNA. These pathways were first discovered in formaldehyde-treated cells, however, since then, various other aldehydes have been shown to induce formation of DPCs in cells. Defects in DPC repair or aldehyde clearance mechanisms lead to various diseases including Ruijs-Aalfs syndrome and AMeD syndrome in humans. Here, we discuss recent developments in understanding how aldehydes form DPCs, how they are repaired, and the consequences of defects in these repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c65abd47bc550aa3a183e59e4afc72e60fec4cd" target='_blank'>
              Aldehyde-induced DNA-protein crosslinks- DNA damage, repair and mutagenesis
              </a>
            </td>
          <td>
            Thomas Blouin, Natalie Saini
          </td>
          <td>2024-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9f95f3c1037cba6e0a1fef491cda140ab1950b2" target='_blank'>
              53BP1 deficiency leads to hyperrecombination using break-induced replication (BIR)
              </a>
            </td>
          <td>
            Sameer Bikram Shah, Youhang Li, Shibo Li, Qing Hu, Tong Wu, Yanmeng Shi, Tran Nguyen, Isaac Ive, Linda Shi, Hailong Wang, Xiaohua Wu
          </td>
          <td>2024-10-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ADP-ribosyltransferases of the PARP family encompass a group of enzymes with variegated regulatory functions in cells, ranging from DNA damage repair to the control of cell-cycle progression and immune response. Over the years, this knowledge has led to the use of PARP1/2 inhibitors as mainstay pharmaceutical strategies for the treatment of ovarian, pancreatic, prostate and breast cancers, holding mutations in genes encoding for proteins involved in the DNA repair mechanisms (synthetic lethality). Meanwhile, the last decade has witnessed significant progress in comprehending cellular pathways regulated by mono-ADP-ribosylation, with a huge effort in the development of novel selective compounds to inhibit those PARPs endowed with mono-ADP-ribosylation activity. This review focuses on the progress achieved in the cancer field, delving into most recent findings regarding the role of a subset of enzymes – the interferon-stimulated PARPs – in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f412723c8383ef54497f485a2de57f5ede1cc4b" target='_blank'>
              PARP enzymes and mono-ADP-ribosylation: advancing the connection from interferon-signalling to cancer biology
              </a>
            </td>
          <td>
            Barbara Morone, Giovanna Grimaldi
          </td>
          <td>2024-08-27</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Determining a novel etiology and mechanism of human cancer requires extraction of characteristic mutational signatures derived from chemical substances. This study explored the mutational signatures of N-nitroso bile acid conjugates using Salmonella strains. Exposing S. typhimurium TA1535 to N-nitroso-glycine/taurine bile acid conjugates induced a predominance of C:G to T:A transitions. Two mutational signatures, B1 and B2, were extracted. Signature B1 is associated with N-nitroso-glycine bile acid conjugates, while Signature B2 is linked to N-nitroso-taurine bile acid conjugates. Signature B1 revealed a strong transcribed strand bias with GCC and GCT contexts, and the mutation pattern of N-nitroso-glycine bile acid conjugates in YG7108, which lacks O6-methylguanine DNA methyltransferases, matched that of the wild-type strain TA1535, suggesting that O6-methyl-deoxyguanosine contributes to mutations in the relevant regions. COSMIC database-based similarity analysis revealed that Signature B1 closely resembled SBS42, which is associated with occupational cholangiocarcinoma caused by overexposure to 1,2-dichlolopropane (1,2-DCP) and/or dichloromethane (DCM). Moreover, the inflammatory response pathway was induced by 1,2-DCP exposure in a human cholangiocyte cell line, and iNOS expression was positive in occupational cholangiocarcinomas. These results suggest that 1,2-DCP triggers an inflammatory response in biliary epithelial cells by upregulating iNOS and N-nitroso-glycine bile acid conjugate production, resulting in cholangiocarcinoma via DNA adduct formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16ffe28d485b3038d354ba43187bea62b1635ee" target='_blank'>
              Whole Genome Sequencing Analysis of Model Organisms Elucidates the Association Between Environmental Factors and Human Cancer Development
              </a>
            </td>
          <td>
            Shinya Hasegawa, Yutaka Shoji, Mamoru Kato, Asmaa Elzawahry, M. Nagai, Min Gi, Shugo Suzuki, Hideki Wanibuchi, S. Mimaki, K. Tsuchihara, Y. Totsuka
          </td>
          <td>2024-10-17</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Simple Summary The rare ovarian cancer subtypes ovarian clear cell carcinoma (OCCC) and Small cell carcinoma of the ovary, hypercalcaemic type (SCCOHT), can have mutations in members of a complex known as SWI/SNF that regulates the accessibility of chromatin to factors involved in DNA repair and gene expression. Some mutations in a particular complex member also occur in endometrioid ovarian cancer (EnOC) and endometriosis. Patients with endometriosis have a greater risk of developing OCCC and EnOC, with endometriosis frequently present at the time of diagnosis of these malignancies. The OCCC and SCCOHT ovarian cancer subtypes are notoriously difficult to treat with chemotherapies based on platinum-drugs that are standard-of-care for most cases of ovarian cancer. Mutations in members of this chromatin-remodelling complex offer new opportunities for molecular therapeutics using drugs that inhibit different aspects of cellular processes, including DNA repair, epigenetic regulation, kinase activity, and immune checkpoints. Abstract SWI/SNF (SWItch/Sucrose Non-Fermentable) is the most frequently mutated chromatin-remodelling complex in human malignancy, with over 20% of tumours having a mutation in a SWI/SNF complex member. Mutations in specific SWI/SNF complex members are characteristic of rare chemoresistant ovarian cancer histopathological subtypes. Somatic mutations in ARID1A, encoding one of the mutually exclusive DNA-binding subunits of SWI/SNF, occur in 42–67% of ovarian clear cell carcinomas (OCCC). The concomitant somatic or germline mutation and epigenetic silencing of the mutually exclusive ATPase subunits SMARCA4 and SMARCA2, respectively, occurs in Small cell carcinoma of the ovary, hypercalcaemic type (SCCOHT), with SMARCA4 mutation reported in 69–100% of SCCOHT cases and SMARCA2 silencing seen 86–100% of the time. Somatic ARID1A mutations also occur in endometrioid ovarian cancer (EnOC), as well as in the chronic benign condition endometriosis, possibly as precursors to the development of the endometriosis-associated cancers OCCC and EnOC. Mutation of the ARID1A paralogue ARID1B can also occur in both OCCC and SCCOHT. Mutations in other SWI/SNF complex members, including SMARCA2, SMARCB1 and SMARCC1, occur rarely in either OCCC or SCCOHT. Abrogated SWI/SNF raises opportunities for pharmacological inhibition, including the use of DNA damage repair inhibitors, kinase and epigenetic inhibitors, as well as immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2127c723b58b759e0f4d567bf3305c6e1009f7" target='_blank'>
              Aberrant SWI/SNF Complex Members Are Predominant in Rare Ovarian Malignancies—Therapeutic Vulnerabilities in Treatment-Resistant Subtypes
              </a>
            </td>
          <td>
            Yue Ma, Natisha R Field, Tao Xie, Sarina Briscas, Emily G. Kokinogoulis, Tali S. Skipper, Amani Alghalayini, Farhana A. Sarker, Nham Tran, N. Bowden, Kristie-Ann Dickson, Deborah J. Marsh
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93b37a3e55555e578d0b3e673e940425e7441335" target='_blank'>
              Anoikis-related genes in breast cancer patients: reliable biomarker of prognosis
              </a>
            </td>
          <td>
            Mingzheng Tang, Yao Rong, Xiaofeng Li, Haibang Pan, Pengxian Tao, Zhihang Wu, Songhua Liu, Renmei Tang, Zhilong Liu, Hui Cai
          </td>
          <td>2024-09-19</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mitotic Arrest Deficient 1 (gene name MAD1L1), an essential component of the mitotic spindle assembly checkpoint, is frequently overexpressed in colon cancer, which correlates with poor disease-free survival. MAD1 upregulation induces two phenotypes associated with tumor promotion in tissue culture cells–low rates of chromosomal instability (CIN) and destabilization of the tumor suppressor p53. Using CRISPR/Cas9 gene editing, we generated a novel mouse model by inserting a doxycycline (dox)-inducible promoter and HA tag into the endogenous mouse Mad1l1 gene, enabling inducible expression of HA-MAD1 following exposure to dox in the presence of the reverse tet transactivator (rtTA). A modest 2-fold overexpression of MAD1 in murine colon resulted in decreased p53 expression and increased mitotic defects consistent with CIN. After exposure to the colon-specific inflammatory agent dextran sulfate sodium (DSS), 31% of mice developed colon lesions, including a mucinous adenocarcinoma, while none formed in control animals. Lesion incidence was particularly high in male mice, 57% of which developed at least one hyperplastic polyp, adenoma or adenocarcinoma in the colon. Notably, mice expressing HA-MAD1 also developed lesions in tissues in which DSS is not expected to induce inflammation. These findings demonstrate that MAD1 upregulation is sufficient to promote colon tumorigenesis in the context of inflammation in immune-competent mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b8f033c5cf9d7fd01adbc62c82ee49aacc06c1c" target='_blank'>
              MAD1 upregulation sensitizes to inflammation-mediated tumor formation
              </a>
            </td>
          <td>
            Sarah E Copeland, Santina M. Snow, Jun Wan, Kristina A Matkowskyj, R. Halberg, B. Weaver
          </td>
          <td>2024-10-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Recent studies have linked certain colibactin producing subphylas of E. coli to colorectal cancer. The bacterium synthesizes colibactin via the pks genomic island containing a series of clbA to clbS genes. The genotoxin causes inter-strand DNA cross-links, which lead to double strand breaks and promotion of colorectal adenocarcinoma. The published studies have prioritized the colorectal regions of the body where E. coli is commonly found, but none have identified the same genotoxic relationship with other types of adenocarcinoma. In this study, we evaluate DNA damage and cellular changes of mammary breast tissue based on high and low levels of colibactin producing E. coli exposure. The goal is to evaluate if colibactin can affect cells in the mammary breast tissue promoting breast adenocarcinoma growth.



 Two cell lines were acquired from ATCC: primary mammary epithelial cells (PCS-600-010) and MCF-7 breast adenocarcinoma cells (HTB-22). Both cell lines were split into six culture flasks and maintained until ∼80% confluence was reached using the ATCC cell culture protocols. Once all the flasks had reached the target confluence, they were divided into exposure groups of high and low concentrations of colibactin producing E. coli, a group of non-colibactin producing E. coli, a group exposed only to phosphate buffered saline, and a control group with no exposure. Cellular morphology was observed using light microscopy before, after exposure, and every 5 minutes up to 15 minutes. E. coli and colibactin DNA concentrations were quantified using quantitative polymerase chain reaction. Genomic DNA was extracted from each group of cells. The DNA from each group was whole genome sequenced and analyzed for genomic variation.



 The normal cells exposed to E. coli showed immediate lysing and continued to lyse after a 5 minute incubation; however, the adenocarcinoma positive cells showed no lysing up to 30 minutes incubation. Sequencing results show genomic variation between the groups and specific break points associated with the E. coli exposed groups.



 Our results show that the presence of E. coli with mammary breast tissue can destroy normal tissue, promote lysis resistance, and allow adenocarcinoma cells to continue to grow. The colibactin producing E. coli does cause double stranded breaks and can lead to cell death. This new information points out a critical need to monitor the microbiome and infection levels of people at risk or in early stages of cancer to prevent adenocarcinoma growth and destruction of normal tissue. This study indicates that colibactin producing E. coli may play a larger role in tissue damage and promotion of other types of cancer. More research is needed to elucidate the molecular pathway and mechanisms of colibactin, which will lead to new diagnostics or therapeutics that monitor E. coli levels or inhibit colibactin’s toxic effects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b3fd078061817b8e9827d3c0f82f869fc3cb02" target='_blank'>
              B-351 Genotoxic Producing Escherichia coli Protects and Promotes Breast Adenocarcinoma
              </a>
            </td>
          <td>
            L. Heaton, S. D. Figard, R. A. Garcia
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncoviruses can integrate into the host genome and cause tumorigenesis. In particular, hepatitis B virus (HBV) infection accounts for more than 50% of hepatocellular carcinoma (HCC) worldwide. We revealed the global geographical disparity of HBV integration that the landscape of HBV integration between HCC tumor and non-tumorous liver varied in regional cohorts, suggesting the different degrees of clonal enrichment. Most HBV integrations were positionally enriched at telomeres and centromeres (T&C) and they highlighted the novel co-involvement of HBV integration, which likely introduces genomic instability in HCC development. This was confirmed by phospho-H2AX staining. We constructed a large meta-cohort of multiple ethnicities to refine the landscape of HBV integration. This enables the gene set/family level exploration. As TERT is the most frequently integrated gene, we further investigated the underlying mechanistic modulation of TERT transcription activation and revealed the concurrent influence by the orientation and relative distance of HBV integration. Additionally, clonal disparity of HBV integration was observed among patients and the higher level of clonal disparity score can indicate poor patients' prognostication. Taken together, our study uncovered the different levels of clonal enrichment of HBV integration, mechanistic insights, and prognostic biomarker signature, to strengthen our understanding in HBV-associated hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ec7bd105c83b5d222005af0db05b395d835b64" target='_blank'>
              Disparity landscapes of viral-induced structural variations in hepatocellular carcinoma: Mechanistic characterization and functional implications.
              </a>
            </td>
          <td>
            Xueying Lyu, K. Sze, J. Lee, Abdullah Husain, Lu Tian, S. Imbeaud, J. Zucman-Rossi, I. Ng, DanielWai-Hung Ho
          </td>
          <td>2024-09-13</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d871afd05449a9bea959b164b547c76fe5c42e1b" target='_blank'>
              Single-cell sequencing analysis revealed that NEAT1 was a potential biomarker and therapeutic target of prostate cancer
              </a>
            </td>
          <td>
            Xing Li, Yanjun Li, Lei Zhang, Huimin Long
          </td>
          <td>2024-10-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d9e40df709b51d001bd72cce2707582e44efe8" target='_blank'>
              Loss of miR-200c-3p promotes resistance to radiation therapy via the DNA repair pathway in prostate cancer
              </a>
            </td>
          <td>
            Maureen Labbé, Manon Chang, Benjamin Saintpierre, Franck Letourneur, Laurence de Beaurepaire, J. Veziers, S. Deshayes, Marine Cotinat, J. Fonteneau, C. Blanquart, Vincent Potiron, Stéphane Supiot, D. Fradin
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ad72ce0344d5d02adc297674f10ddcc6fc4d478" target='_blank'>
              The role of TERT C228T and KDM6A alterations and TME in NMIBC treated with BCG
              </a>
            </td>
          <td>
            Q. Xia, Jian-Xuan Sun, Zhipeng Yao, Junlin Lu, Chen-Qian Liu, Jinzhou Xu, Ye An, Mengmeng Xu, Si-Han Zhang, Xing-Yu Zhong, Na Zeng, Siyang Ma, Haodong He, Heng-Long Hu, Jia Hu, Yi Lu, Bing Li, Yao Chen, Zhenghao Liu, Shaogang Wang
          </td>
          <td>2024-10-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Aberrant expression of repeat RNAs in pancreatic ductal adenocarcinoma (PDAC) mimics viral-like responses with implications for tumor cell state and the surrounding microenvironment. To better understand the relationship of repeat RNAs in human PDAC, we employed the NanoString CosMx™ spatial molecular imaging (SMI) platform, which utilizes a 1,000-plex RNA panel with custom repeat RNA probes targeting long interspersed nuclear element 1 (LINE-1) retrotransposon ORF1 and ORF2, HSATII satellite (SAT) repeat, and two human endogenous retroviruses (HERV-K and HERV-H) in 46 primary tumors. This analysis revealed correlations of high repeat RNA expression with alterations in the epithelial state of PDAC cells and the myofibroblast phenotype in cancer-associated fibroblasts (CAFs). The loss of cellular identity observed with dosing of extracellular vesicles (EVs) and individual repeat RNAs in PDAC and CAF cell culture models points to cell-cell intercommunication involving these viral-like elements. Differences in the PDAC and CAF response are driven by distinct innate immune signaling pathways through interferon regulatory transcription factor 3 (IRF3). Altogether, our data indicate that cell context-specific viral-like responses driven by tumor cells have broad impacts on single-cell heterogeneity within cancer cells and the pancreatic cancer microenvironment.
 Citation Format: Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca Porter, Bidish Patel, Linda Nieman, Nathan Schurman, Briana Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew Liss, Tae Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin Aryee, David Ting. Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffeaf733e03a440a3ea9633bad9509d48160c804" target='_blank'>
              Abstract A039: Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko E. Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael J. Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca L. Porter, Bidish Kumar Patel, Linda T. Nieman, Nathan Schurman, Briana M. Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew S. Liss, T. Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin J. Aryee, David T. Ting
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48331813faa24a4afbda2fd597fcf7d1591ecec4" target='_blank'>
              Comprehensive analysis reveals that P4HA3 is a prognostic and diagnostic gastric cancer biomarker that can predict immunotherapy efficacy
              </a>
            </td>
          <td>
            Yuanhang Yu, Kexin Luo, Meihan Liu, Long Chen, Xi Gao, Lijuan Zhang, Xianfu Li, Hongpan Zhang
          </td>
          <td>2024-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e94f7ba0df6a060dbe3e396fdf91b0198ef78ddc" target='_blank'>
              PRMT1 inhibition perturbs RNA metabolism and induces DNA damage in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Joseph Walton, A. S. N. Ng, Karen Arevalo, A. Apostoli, Jalna Meens, Christina Karamboulas, Jonathan R St-Germain, Panagiotis Prinos, J. Dmytryshyn, Eric X Chen, Cheryl H. Arrowsmith, Brian Raught, L. Ailles
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small population of less differentiated cells with robust self-renewal ability. CSCs have been recognized as the root cause of tumor initiation, progression, relapse, and drug resistance. Recent studies from us and others have highlighted that N6-methyladenosine (m6A), the most prevalent modification in mRNA, plays a crucial role in carcinogenesis and CSC homeostasis. Dysregulation of the m6A modification machinery has been implicated in CSC survival and self-renewal, thereby regulating cancer progression and therapeutic resistance. In this review, we provide an overview of the roles and molecular mechanisms of the RNA m6A modification machinery in CSC survival and self-renewal. Additionally, we summarize the currently known small-molecule inhibitors targeting the dysregulated m6A modification machinery and discuss proof-of-concept studies focusing on the efficacy of these compounds in eliminating CSCs and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a2c463331686ce335a111580d49921b4543bf" target='_blank'>
              Combating cancer stem cells: RNA m6A methylation and small-molecule drug discovery
              </a>
            </td>
          <td>
            Honghai Zhang, Xueer Wang, Jianjun Chen, R. Su
          </td>
          <td>2024-09-30</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4d24999230a049c47c4ad63da565877c2b583c" target='_blank'>
              PPM1D activity promotes cellular transformation by preventing senescence and cell death
              </a>
            </td>
          <td>
            Miroslav Stoyanov, Andra S. Martinikova, Kateřina Matějková, K. Horackova, P. Zemánková, Kamila Burdová, Zuzana Zemanova, P. Kleiblova, Z. Kleibl, L. Macurek
          </td>
          <td>2024-09-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction: Over 90% of PDAC patients harbor an activating KRAS mutation, the most common form being KRASG12D. However, additional genetic mutations and environmental events, including chronic or hereditary pancreatitis, are needed for KRAS mutations to lead to PDAC. Our laboratory showed Activating Transcription Factor 3 (ATF3) was required for repressing genes that stabilize the mature acinar cell phenotype and for KRASG12D-driven progression to advanced PanIN lesions. However, ATF3 contributes to gene activation and repression and has been linked to numerous transcription factors and epigenetic mediators, and how ATF3 affects PDAC progression remains unknown. We hypothesize that KRASG12D promotes ATF3-driven histone acetylation in precursor PanINs lesions. Methods: C57Bl/6l mice allowing for acinar-inducible KRASG12D combined with (Ptf1acreERTKRASG12D) or without (Ptf1acreERTKRASG12DAtf3-/-; APK) Atf3 deletion were treated with tamoxifen to induce KRASG12D. After 10 days, mice were treated with cerulein to induce injury and PanIN progression. Mice were sacrificed after two weeks and 3D organoid lines developed. RNA-seq and chromatin immunoprecipitation for H3K27 acetylation (H3K27ac) followed by sequencing (ChIP-seq) was performed. Bioinformatic analysis was used to identify differentially enriched pathways. Levels of acetyl-CoA, the substrate for acetylation, was compared between lines along with levels of H3K27ac, H3K4me1, H3K4me3, and H3K9me3 through western blots. Results and Discussion: KRASG12D expression promoted epigenetic reprogramming in PanINs, including dysregulation of H3K27ac enrichment. ChIP-seq results showed the absence of ATF3 reduces H3K27 acetylation preferentially at gene promoters and pathway analysis showed differential enrichment and activation of KRAS signaling. Mechanistically, ATF3-deleted mice have lower acetyl-CoA levels, linking ATF3 to metabolic pathways as a possible mechanism for gene regulation. Comparing genes with an ATF3-dependent acetylation pattern to a curated list of KRAS-targeted genes identified SMARCC2 (BAF170), a SWI/SNF complex subunit, and KDM1B, a histone demethylase affecting H3K4, suggesting a potential role for ATF3 in widespread epigenetic regulation. Public sequencing data from PDAC patients, supports these findings as ATF3 expression is positively correlated with expression of several SWI/SNF complex subunits and demethylases. Conclusion: This study highlights a role for ATF3- mediated epigenetic dysregulatio. In oncogenic KRAS-dependent PDAC. Targeting this ATF3- mediated epigenetic landscape could be a promising new therapeutic avenue.
 Citation Format: Fatemeh Mousavi, Christopher L Pin, Parisa Shooshtari. Oncogenic KRAS-mediated epigenetic reprogramming is altered by loss of Activating Transcription Factor 3 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B082.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2493dfe1e95be582f58087b933860056a415fef3" target='_blank'>
              Abstract B082: Oncogenic KRAS-mediated epigenetic reprogramming is altered by loss of Activating Transcription Factor 3
              </a>
            </td>
          <td>
            Fatemeh Mousavi, Christopher Pin, Parisa Shooshtari
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58ba92b023525bc5ca52aaa30ac9bfb58d8d97b" target='_blank'>
              Visualizing epigenetic modifications and their spatial proximities in single cells using three DNA-encoded amplifying FISH imaging strategies: BEA-FISH, PPDA-FISH and Cell-TALKING.
              </a>
            </td>
          <td>
            Feng Chen, Xinyin Li, Min Bai, Yongxi Zhao
          </td>
          <td>2024-09-04</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="This review offers an in-depth analysis of mitochondrial DNA (mtDNA) mutations in colorectal cancer stem cells (CSCs), emphasizing their significant impact on tumor dynamics and potential therapeutic strategies. CSCs are a special subpopulation due to their unique capabilities for self-renewal, differentiation, and resistance to conventional therapies. Given that CSCs significantly differ from other tumor cell subpopulations, particularly in their metabolic properties, and considering that colorectal cancer is a malignancy characterized by mitochondrial dysfunction, this review aims to put together existing data on the differences in the mitochondrial genome of CSCs compared to other colorectal tumor cell subpopulations. Additionally, the review seeks to explore the potential roles of these differences and to identify new ideas for therapeutic strategies. Key topics include the identification and properties of CSCs in colorectal cancer, the distinctive features of the mitochondrial genome, and the functional consequences of mtDNA mutations. The review hypothesizes that CSCs rely on well-functioning mitochondria for crucial aspects like energy production; yet, mtDNA mutations can lead to mitochondrial dysfunction, altering CSC characteristics and influencing cancer progression. The article discusses emerging therapeutic approaches targeting mitochondrial function in colorectal CSCs and highlights the need for advanced research, including the development of preclinical models and exploration of targeted therapies, to improve the understanding and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebae925d5c7100b4c278e4586e85603f7ee40d86" target='_blank'>
              Mitochondrial DNA Mutations in Colorectal Cancer Stem Cells: Implications for Tumor Dynamics and Therapeutic Strategies.
              </a>
            </td>
          <td>
            N. Shakhpazyan, L. Mikhaleva, Arcady L Bedzhanyan, Z. Gioeva, A. Mikhalev, K. Midiber, A. Konyukova, D. Atiakshin, Igor Buchwalow, Markus Tiemann, Alexander N Orekhov
          </td>
          <td>2024-09-11</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Dietary exposure to aflatoxin B₁ (AFB₁) is a risk factor for the development of hepatocellular carcinomas (HCCs). Following metabolic activation, AFB₁ reacts with guanines to form covalent DNA adducts, which induce high-frequency G > T transversions. The molecular signature associated with these mutational events aligns with the single base substitution signature 24 (SBS24) in the Catalog of Somatic Mutations in Cancer (COSMIC) database. Deficiencies in either base excision repair (BER) due to the absence of Nei-like DNA glycosylase 1 (NEIL1) or nucleotide excision repair (NER) due to the absence of xeroderma complementation group A protein (XPA) contribute to HCCs in murine models. In the current study, ultra-low error duplex sequencing was used to characterize mutational profiles in liver DNAs of NEIL1-deficient, XPA-deficient, and DNA repair-proficient mice following neonatal injection of 1 mg/kg AFB₁. Analyses of AFB₁-induced mutations showed high cosine similarity to SBS24, regardless of repair proficiency status. The absence of NEIL1 resulted in an approximately 30% increase in the frequency of mutations, with distribution suggesting preferential NEIL1-dependent repair of AFB₁ lesions in open chromatin regions. A trend of increased mutagenesis was also observed in the absence of XPA. Consistent with the role of XPA in transcription-coupled repair, mutational profiles in XPA-deficient mice showed disruption of the transcriptional bias in mutations associated with SBS24. Implications: Our findings define the roles of DNA repair pathways in AFB₁-induced mutagenesis and carcinogenesis in murine models, with these findings having implications in human health for those with BER and NER deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15959721470d4aedb8df8805f673bf93355c9b41" target='_blank'>
              The distinct roles of NEIL1 and XPA in limiting aflatoxin B₁-induced mutagenesis in mice.
              </a>
            </td>
          <td>
            Michael M. Luzadder, I. Minko, Vladimir L. Vartanian, Marten Davenport, Lev M Fedorov, A. McCullough, R. S. Lloyd
          </td>
          <td>2024-10-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Lung cancer is the predominant form of cancer globally, arising from the dysfunction of genetic mutations. Although PTEN mutation is crucial in the aetiology of lung cancer, the mapping of these major drivers has to be determined. We leverage computational algorithms on 43,855 SNPs of PTEN to discover the mutational impact contributing to lung cancer. Fifteen variations were identified as detrimental, and no pertinent studies have previously addressed their structural and functional aspects. Notably, seven variations were identified as the most significant contributors to lethal effects in functional aberration, as demonstrated by the computational assessment. Subsequently, molecular simulation elucidated the structural instability associated with these alterations. Furthermore, drug binding experiments at the mutational site corroborated the destabilization experiments by demonstrating the conformational alteration of the structure, resulting in varied amino acid interactions. In summary, the present study elucidates the influence of mutations in PTEN structure on its functional architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda07e1a807dbd9e289ad89058637502087cc04f" target='_blank'>
              Mapping the PTEN Mutation Landscape: Structural and Functional Drivers of Lung Cancer
              </a>
            </td>
          <td>
            Mohammad Uzzal Hossain, Mohammad Nazmus Sakib, A.B.Z Naimur Rahman, SM Sajid Hasan, Nazia Hassan Nisha, Arittra Bhattacharjee, Z. Chowdhury, Ishtiaque Ahammad, K. C. Das, Mohammad Shahedur Rahman, Md Salimullah
          </td>
          <td>2024-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
The PANoptosis pathway is a recently identified mechanism of cellular death that involves the interaction and synchronization among cellular pyroptosis, apoptosis, and necrosis. More and more evidence suggests that PANoptosis is involved in the development and treatment of cancer. However, a comprehensive understanding of the influence of PANoptosis genes on prognostic value, tumor microenvironment characteristics, and therapeutic outcomes in patients with ovarian cancer (OC) remains incomplete.


OBJECTIVE
The present work was designed to devise a PANoptosis signature for OC prognosis and explore its potential molecular function.


METHODS
For this study, we obtained RNA sequencing and clinical data for ovarian cancer from the Cancer Genome Atlas (TCGA) and the GSE32062 cohort. Somatic variants of PANoptosis-related genes (PRGs) in OC were analyzed using GSCA. TCGA-OC and GSE32062 were used to construct training and validation cohorts for the model. Differential expression and correlation analyses were performed following the screening of genes with prognostic ability using univariate Cox analysis. Least Absolute Shrinkage and Selection Operator (LASSO) regression was performed to construct PRG signature based on genes that were differentially expressed and correlated with prognosis. CIBERSORT and ESTIMATE were used to analyze the relationship between the PRGs signature and immune infiltration. TIDE was used to analyze the relationship between the PRG signature and immune checkpoint genes. OncoPredict was used to analyze the relationship between the PRG signature and the drug sensitivity. Quantitative real-time PCR (qRT-PCR) was used to validate the expression of PRGs in OC.


RESULTS
The PRG signature was constructed using three prognostic genes (AIM2, APAF1, and ZBP1) in both TCGA-OC. The results showed that the PRGs signature had an AUC of 0.521, 0.546, and 0.598 in TCGA-OC and 0.620, 0.586, and 0.579 in GSE32062 to predict to predict OS at 1-, 3-, and 5-year intervals. Furthermore, a higher PRG signature risk score was significantly associated with shorter OS (HR = 1.693, 95% CI: 1.303 - 2.202, p = 8.34 × 10^-5 in TCGA-OC and HR = 1.63, 95% CI: 1.13 - 2.35, p = 0.009 in GSE32062). The risk score was identified as the independent prognostic factor for OC. Patients categorized according to their risk score exhibited notable variations in immune status, response to immunotherapy, and sensitivity to drugs. AIM2, APAF1, and ZBP1 were significantly aberrantly expressed in OC cell lines.


CONCLUSION
The PRG signature has the potential to serve as a prognostic predictor for OC and to provide new insights into OC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f42d7af21a82156db04a98c57ead8b95bbe459" target='_blank'>
              Construction of a PANoptosis-related Prognostic Signature for Predicting Prognosis, Tumor Microenvironment, and Immune Response in Ovarian Cancer.
              </a>
            </td>
          <td>
            Yonghong Li, G. Lyu
          </td>
          <td>2024-09-06</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a6600c23276f57e2410dc539cd5e025998c13a" target='_blank'>
              SMARCD1 is an essential expression-restricted metastasis modifier
              </a>
            </td>
          <td>
            Christina Ross, Li-Yun Gong, Lisa M. Jenkins, Ngoc-Han Ha, Megan Majocha, Kent W. Hunter
          </td>
          <td>2024-10-10</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a02e258d3d5db383f939c9889a3838282dd7be9" target='_blank'>
              Mechanisms of Berberine in anti-pancreatic ductal adenocarcinoma revealed by integrated multi-omics profiling
              </a>
            </td>
          <td>
            Jia Yang, Tingting Xu, Hongwei Wang, Lei Wang, Yanmei Cheng
          </td>
          <td>2024-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare and aggressive malignancy characterised by diagnostic challenges, high recurrence rates, and poor prognosis. This study explored the role microRNA (miRNA) processing genes in ACC, and their potential role as diagnostic and prognostic biomarkers. We analysed the mRNA expression levels of miRNA machinery components (DROSHA, DGCR8, XPO5, RAN, DICER, TARBP2 and AGO2) utilising mRNA-Seq data from The Cancer Genome Atlas (TCGA) and The Genotype-Tissue Expression (GTEx) projects. Additionally, protein levels were quantified in tissue samples from the Kolling Institute of Medical Research's tumour bank. Our results demonstrated that among all miRNA processing components, AGO2 exhibited significant overexpression in ACC compared to the normal adrenal cortex (NAC) and benign adrenal adenoma (AA) (p < 0.001). Kaplan-Meier survival analysis indicated that higher AGO2 expression correlated with significantly worse overall survival in ACC patients (HR 7.07, p < 0.001). Among 32 cancer types in TCGA, the prognostic significance of AGO2 was most prominent in ACC. This study is the first to report AGO2's potential as a diagnostic and prognostic biomarker in ACC, emphasising its significance in ACC pathogenesis and potential application as a non-invasive liquid biopsy biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f899d2ba353b49ab4e44563dbef80c246c8d995" target='_blank'>
              AGO2 protein: a key enzyme in the miRNA pathway as a novel biomarker in adrenocortical carcinoma.
              </a>
            </td>
          <td>
            A. Hashmi, Alexander Papachristos, S. Sidhu, G. Hutvagner
          </td>
          <td>2024-10-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human exploration of the solar system will expose crew members to galactic cosmic radiation (GCR), with a potential for adverse health effects. GCR particles (protons and ions) move at nearly the speed of light and easily penetrate space station walls, as well as the human body. Previously, we have shown reactivation of latent herpesviruses, including herpes simplex virus, Varicella zoster virus, Epstein–Barr virus, and cytomegalovirus (CMV), during stays at the International Space Station. Given the prevalence of latent CMV and the known propensity of space radiation to cause alterations in many cellular processes, we undertook this study to understand the role of GCR in reactivating latent CMV. Latently infected Kasumi cells with CMV were irradiated with 137Cs gamma rays, 150 MeV protons, 600 MeV/n carbon ions, 600 MeV/n iron ions, proton ions, and simulated GCR. The CMV copy number increased significantly in the cells exposed to radiation as compared with the non-irradiated controls. Viral genome sequencing did not reveal significant nucleotide differences among the compared groups. However, transcriptome analysis showed the upregulation of transcription of the UL49 ORF, implicating it in the switch from latent to lytic replication. These findings support our hypothesis that GCR may be a strong contributor to the reactivation of CMV infection seen in ISS crew members.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5befbefc112dbf5f93c2aad5df059a35e0acc4cd" target='_blank'>
              Effect of Simulated Cosmic Radiation on Cytomegalovirus Reactivation and Lytic Replication
              </a>
            </td>
          <td>
            Satish K. Mehta, D. Diak, Sara Bustos-Lopez, Mayra Nelman-Gonzalez, Xi Chen, I. Plante, Stephen J. Stray, Ritesh Tandon, Brian E. Crucian
          </td>
          <td>2024-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This study aims to investigate the expression variation, biological significance, and prognostic value of cyclin-dependent kinase inhibitor 2A (CDKN2A) as a common biomarker across 33 malignancies. Various bioinformatics tools, including UALCAN, GEPIA2, OncoDB, cBioPortal, TIMER2, STRING, DAVID, and the GSCA database, were employed for this pan-cancer analysis. The results revealed significant up-regulation of CDKN2A in 24 major human cancer subtypes (P < 0.05). This up-regulation was strongly associated with poor overall survival and tumor dissemination, particularly in uterine corpus endometrial carcinoma (UCEC), colon adenocarcinoma (COAD), and liver hepatocellular carcinoma (LIHC), highlighting its potential as a prognostic biomarker. Moreover, CDKN2A overexpression was linked to diverse clinicopathological characteristics of patients. Genetic alterations recorded via cBioPortal indicated minimal mutation rates in COAD, LIHC, and UCEC. Additionally, promoter methylation, drug sensitivity, and enrichment analyses were performed to explore associations with CDKN2A expression. Overall, the findings emphasize the potential of CDKN2A as a shared diagnostic and prognostic biomarker, as well as a therapeutic target in COAD, LIHC, and UCEC, particularly in patients with varied clinicopathological traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72e6f689c5d7504c2fc12228151a6ec81e45abaa" target='_blank'>
              Exploring the Role of CDKN2A in Human Cancers Using an Integrative Pan-Cancer Approach
              </a>
            </td>
          <td>
            Syed Hussain Raza, Akbar Ali
          </td>
          <td>2024-10-15</td>
          <td>Dermatological Health</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [25],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>