<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-06 09:26:09 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>307</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>263</td>
          <td>157</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="
 Extrachromosomal DNA (ecDNA), highly amplified circular DNA, is widespread in human cancers and serves as a primary location for oncogene amplification. Due to its dynamic structural rearrangement and asymmetric inheritance during cell division, ecDNA increases tumor genome complexity, contributing to drug resistance and shortened patient survival. Here, we show that ecDNA-mediated genomic amplification is highly associated with transcript fusion events, which are prevalent in cancer and often lead to tumor development. We discovered that transcripts with the highest fusion events originate from genes amplified on ecDNA across ecDNA(+) cell lines. Notably, PVT1 (Plasmacytoma Variant Translocation 1) long noncoding RNA, which is known to be a hotspot for chromosomal translocation, is the most frequently fused RNA species in MYC/PVT1-amplified ecDNA(+) cancer cells. Exon 1 of PVT1 is the predominant fusion partner and confers RNA stability, increasing the RNA abundance of the partner oncogene. Using a model cell line with PVT1-MYC fusion on ecDNA, we found that RNA expression of PVT1-MYC increases in vivo in an ecDNA-dependent manner, while canonical MYC does not show a prominent increase. Additionally, ectopic expression of PVT1- MYC in MYC-depleted cancer cells provides higher rescue efficiency than canonical MYC. In contrast, a PVT1-MYC mutant that is unable to enhance RNA stability fails to rescue, highlighting the functional significance of PVT1-mediated RNA stabilization in cancer. This study unveils the link between PVT1 fusion and ecDNA in cancer, providing insights for developing diagnostic and therapeutic approaches tailored to target ecDNA.
 Citation Format: Hyerim Yi, Shu Zhang, Matthew G Jones, Julia A Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, Paul S Mischel, Howard Y Chang. PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b099680e1d89fbf4dffca6c72e1bc67243614d39" target='_blank'>
              Abstract PR014: PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Matthew G Jones, J. Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through controlled double-strand breaks. While TOP2A is a chemotherapy target in proliferating cells, the ubiquitously expressed TOP2B is a potential off-target. Here we explore roles of TOP2B in mutagenesis by generating DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analysing these maps for driver mutations and mutational processes in 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants (SVs), especially at evolutionary conserved sites displaying high transcription and long-range chromatin interactions. TOP2B binding underlies SVs and hotspot mutations in cancer-driving genes such as TP53, MYC, FOXA1, and VHL, and many cis-regulatory elements. We show that the TOP2B-bound mutational hotspot at RMRP drives tumor initiation and growth in vivo. These data highlight TOP2B as a protector of the genome from topological challenges whose aberrant activity promotes driver and passenger mutations in cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0cbfffd4039fee7d6be03df4eefb1698ccef5df" target='_blank'>
              Topoisomerase IIb binding underlies frequently mutated elements in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, S. Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, D. Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, Daniel Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery.
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-03</td>
          <td>Cell research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) can drive oncogene amplification, gene expression and intratumor heterogeneity, representing a major force in cancer initiation and progression. The phenomenon becomes even more intricate as distinct types of ecDNA present within a single cancer cell. While exciting as a new and significant observation across various cancer types, there is a lack of a general framework capturing the dynamics of multiple ecDNA types theoretically. Here, we present novel mathematical models investigating the proliferation and expansion of multiple ecDNA types in a growing cell population. By switching on and off a single parameter, we model different scenarios including ecDNA species with different oncogenes, genotypes with same oncogenes but different point mutations and phenotypes with identical genetic compositions but different functions. We analyse the fraction of ecDNA-positive and free cells as well as how the mean and variance of the copy number of cells carrying one or more ecDNA types change over time. Our results showed that switching does not play a role in the fraction and copy number distribution of total ecDNA-positive cells, if selection is identical among different ecDNA types. In addition, while cells with multiple ecDNA cannot be maintained in the scenario of ecDNA species without extra fitness advantages, they can persist and even dominate the ecDNA-positive population if switching is possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f328238baa1a06ddbda4cdafb65854f0db985125" target='_blank'>
              Population dynamics of multiple ecDNA types
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-11-21</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Somatic genomic rearrangements are hallmarks of cancer. Complex genomic rearrangements (CGRs) involving multiple intertwined structural alterations are often present in tumor genomes. CGRs frequently harbor oncogenic drivers, but their genomic architectures and etiologies are poorly understood. We used deep-coverage optical mapping technology to profile the genomic landscapes of normal lung tissues, benign pulmonary lesions, carcinoma in situ, and advanced carcinomas to examine the patterns of genome disorganization and instability in different stages of carcinogenesis in lung. Large rearrangements and CGRs were prevalent in the carcinomas. We developed omcplR to resolve the architecture of CGRs and predict their genesis from optical mapping data using genome-graph concept. We found that CGRs often arose from hierarchical combinations of multiple, localized simple structural variations, and harbored allelic heterogeneity at the affected loci. Rearrangement patterns and associated genomic signatures suggested that chromoanasynthesis was a likely prevalent mechanism driving complex genomic rearrangements. The early rearrangement junctions in intra-chromosomal CGRs were usually localized within the same chromatin domains, but the late junctions in advanced tumors had more heterogeneous contexts suggesting progressive organizational heterogeneity. The CGRs, especially the late events therein were under positive selection. A composite signature of genomic alterations including the CGRs captured the trajectory of progressive genomic disorganization and instability with carcinogenesis in lung and underscored the extent of genomic structural heterogeneity among the in-situ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0b28163735146a5869b061b33efdd0283e9308" target='_blank'>
              Deconstructing evolutionary histories of complex genomic rearrangements in lung malignancies
              </a>
            </td>
          <td>
            Xiaoju Hu, Vignesh Venkat, Gregory R Riedlinger, Zhiyuan Shen, Jyoti Malhotra, Subhajyoti De
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc1dada885aaee19cff1e9a95787aee6cf49a1c" target='_blank'>
              Comprehensive profiling of extrachromosomal circular DNAs in colorectal cancer progression
              </a>
            </td>
          <td>
            Lu Lu, Mingjie Chen, Guicheng Zhang, Yujing Liu, Xiangyuan Xu, Zenghua Jiang, Yangxian Xu, Tao Liu, Fan Yang, Guang Ji, Hanchen Xu
          </td>
          <td>2024-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Independently of the pathways or circuits deregulated in cancer cells, these present altered transcription patterns, often also direct consequence of deregulation of transcription factors. In this sense, also the RNA Polymerase complexes responsible for transcription can be affected in cancers. We find that upregulations of RNA Polymerase II subunits, especially the largest ones, correlates with poor cancer patients’ outcome across a range of tumor types, presenting increased genome instability. Overexpressing the subunits RPB1, RPB3 and RPB4 in cells we find that these induce DNA damage. However, the mechanisms behind this increased genome instability are specific for each subunit, linked to the unique transcription alterations generated by the subunit overexpression. Importantly, we find significant overlap between the genes with more DNA damage in our cell line models and those more affected in cancers with subunit upregulation, indicating that upregulations could be responsible for some of the phenotypes present in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1c9c056673cff0cd8a0e04f5d4022f59e547e9" target='_blank'>
              RNA Polymerase II subunits overexpressions induce genome instability and deregulate transcription
              </a>
            </td>
          <td>
            Martina Mustè Sadurnì, Megan Louise Jones, Lucas Pavlou, Mohamed Radi, Smaragda Kompocholi, Paolo Passaretti, Agnieszka Gambus, Lea H. Gregersen, Marco Saponaro
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) can drive tumor initiation, progression and resistance in some of the most aggressive cancers and is emerging as a promising anti-cancer target. However, detection currently requires costly whole-genome sequencing (WGS) or labor-intensive cytogenetic or FISH imaging, limiting its application in routine clinical diagnosis. To overcome this, we developed ecPath (ecDNA from histopathology), a computational method for predicting ecDNA status from routinely available hematoxylin and eosin (H&E) images. ecPath implements a deep-learning method we call transcriptomics-guided learning, which utilizes both transcriptomics and H&E images during the training phase to enable successful ecDNA prediction from H&E images alone, a task not achievable with models trained on H&E images only. It is trained on more than 6,000 tumor whole-slide images from the TCGA cohort with the best performance in predicting ecDNA status in brain and stomach tumors (average AUC=0.78). ecPath revealed that ecDNA-positive tumors are enriched with pleomorphic, larger and high-density nuclei. Testing in an independent cohort, ecPath predicted ecDNA status of 985 pediatric brain tumor patients with an AUC of 0.72. Finally, we applied ecPath to identify ecDNA-positive tumors in the TCGA cohort for which no WGS data were available. Like WGS-based ecDNA-positive labels, the predicted ecDNA-positive status also identify poor prognoses for low grade glioma patients. These results demonstrate that ecPath enables the detection of ecDNA from routinely available H&E imaging alone and help nominate aggressive tumors with ecDNA to study and target it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ffb85a2375bcb28d5b3da1f1dc6239edbfdf5f" target='_blank'>
              ecPath detects ecDNA in tumors from histopathology images
              </a>
            </td>
          <td>
            Mudra Choudhury, Lihe Liu, Anamika Yadav, Owen S Chapman, Zahra Ahmadi, Raneen Younis, Chinmay Sharma, Navansh Goel, S. Sridhar, Rishaan Kenkre, Aditi Dutta, Shanqing Wang, E. Shulman, S. R. Dhruba, Danh-Tai Hoang, Kevin M. Tharp, Megan R Paul, Denise Malicki, K. Yip, E. Ruppin, Lukas Chavez, Sanju Sinha
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Purpose of the study: to determine the role of retroelements in chromoanagenesis mechanisms in cancer etiopathogenesis.Material and Methods. The search for relevant sources was carried out in the Scopus, Web of Science, PubMed, Elibrary systems, including publications from February 2002 to December 2023. Of the 864 scientifc articles found, 60 were used to write a systematic review.Results. According to original works and meta-analyses results, the cause of complex chromosomal rearrangements during cancer development may be retroelement pathological activation. Chromoanagenesis involves LINE1, SVA, Alu, HERV, which cause double-stranded DNA breaks, insertions in tumor suppressor genes region, the formation of chimeric oncogenes due to retroelement use as new promoters, and function as molecular “band-aids” in non-homologous end junctions and form bridges of distal DNA fragments. Global structural rearrangements of chromosomes observed during chromoanagenesis may be consequences of retroelements activation, which participate in non-allelic homologous recombination and in microhomology-mediated joining of ends characteristic. Certain types of neoplasms, such as colon cancer, are characterized by both high levels of chromothripsis and retroelement activity. In head and neck squamous cell carcinoma, chromoplexy is specifc, the sources of sequences at the breakpoints of which are retroelements. During chromoanagenesis, activation of proto-oncogenes and inactivation of tumor suppressor genes are observed, which is also a consequence of retroelement activation. This is due to the presence of retroelement sequences in proto-oncogenes promoter regions and introns (which become the basis for chimeric oncogene formation) and hot spots of insertional mutagenesis in tumor suppressor genes (transpositions into these regions inactivate these genes).Conclusion. The results obtained on the driver effect of retroelements in chromothripsis, chromoplexy and chromoanasynthesis mechanisms, which are the basis for the formation of clonal evolution of tumors, indicate promise of targeted therapy aimed at silencing the activity of retroelements in cancer patients treatment. For this purpose, it is possible to use microRNAs complementary to retroelements, which are also involved in tumor development, as tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3dd701cb78af4fbe30529399084bdaa803a811" target='_blank'>
              Participation of retroelements in chromoanagenesis in cancer development
              </a>
            </td>
          <td>
            R. N. Mustafin
          </td>
          <td>2024-11-15</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Completion of DNA replication before chromosome segregation is essential for the stable maintenance of the genome. Under replication stress, DNA synthesis may persist beyond S phase, especially in genomic regions that are difficult to proceed with the replication processes. Incomplete replication in mitosis emerges as non-disjoined segment in mitotic chromosomes leading to anaphase bridges. The resulting chromosome rearrangements are not well characterized, however. Here, we report that incomplete replication due to SMC5/6 deficiency impairs sister chromatid disjunction at difficult-to-replicate regions, including common fragile sites. These non-disjoined regions manifest as cytologically defined symmetric gaps, causing anaphase bridges. These bridges break at the gaps, leading to telomere loss, micronucleation, and fragmentation. Subsequently, fusions between telomere-deficient chromosomes generate complex chromosomal rearrangements, including dicentric chromosomes, suggesting the occurrence of breakage-fusion-bridge cycle. Additionally, chromosomes in micronuclei were pulverized, indicative of chromothripsis. Our findings suggest that incomplete replication facilitates complex chromosomal rearrangements, which may contribute to genomic instability in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb56da5499cf4165223ba91582989de2546ca468" target='_blank'>
              Chromosomal rearrangements associated with SMC5/6 deficiency in DNA replication.
              </a>
            </td>
          <td>
            Y. Kusano, Y. Kinugasa, Satoshi Tashiro, Toru Hirota
          </td>
          <td>2024-11-14</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Acquisition of a telomere maintenance mechanism is essential for cancer cells. In a minority of tumors, telomeres are lengthened via Alternative Maintenance of Telomeres (ALT), a telomerase-independent pathway based on homologous recombination. ALT tumors have heavily rearranged genomes with many structural variants containing telomere repeats. To better understand the genetic evolution of these tumors, we seek to determine if certain genomic loci tend to spatially associate with telomeres in ALT and are especially liable to experience telomere recombination events as a result. Assays that reveal close spatial associations between genomic loci, such as SPRITE and Hi-C, have enabled extensive exploration of genomic spatial organization. However, as analysis pipelines for these next-generation sequencing-based assays typically discard reads aligning to repetitive elements, little is known about the spatial arrangement of telomeres and other repetitive loci in the nucleus. Here, we present TelSPRITE, a novel approach to extracting telomere contact frequencies from SPRITE data. We identify reads containing telomere repeats and sort them into a single bin, quantifying spatial contacts between the telomere bin and the rest of the genome. Our analysis reveals a strong dependency of telomere contact frequency on chromosomal distance from the telomere, consistent with the known effect of linear distance on 3-dimensional spatial contacts. Telomere contacts are also strongly enriched near centromeres, a phenomenon that may be reflective of spatial clustering of heterochromatic regions. ALT cell lines are globally enriched for telomere content and display distinctive intrachromosomal spikes in telomere contact frequency. Our customized analysis of long read sequencing data suggests that loci with high telomere contact frequencies represent structural variants containing telomere repeats in ALT cells. Collectively, our results demonstrate general principles of telomeric spatial organization while also profiling the spectrum of genomic rearrangements in ALT cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca078135018e0ad318b04cc28cb3eb2ce26683a8" target='_blank'>
              Telomere interactions and structural variants in ALT cells revealed with TelSPRITE
              </a>
            </td>
          <td>
            David G. Wilson, Sarah F. Clatterbuck Soper, M. Pineda, Robert L. Walker, Paul S. Meltzer
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Oncogenic fusion transcription factors (TFs) frequently drive hematopoietic malignancies by altering gene expression in key developmental programs. TCF3::HLF is a fusion TF that characterizes a rare, treatment-resistant subtype of B-cell acute lymphoblastic leukemia (t(17;19) TCF3::HLF-positive B-ALL). Despite its clinical significance, the mechanisms by which TCF3::HLF induces leukemia are unclear. We used HiChIP mapping and genetic interference to analyze TCF3::HLF at the 3D-genome level, revealing enhancer-promoter interactions that control gene activation or repression. Notably, TCF3::HLF directly regulates MEF2C expression through its enhancer, as interference disrupted MEF2C transcription and inhibited leukemia propagation. This disruption also diminished embryonal hematopoietic stem cell (HSC) gene signatures and restored mature HSC and B-lymphoid markers. These findings highlight MEF2C as a critical component of the transcriptional network reprogrammed by TCF3::HLF. Our study provides insight into how TCF3::HLF rewires the 3D genome to drive leukemia and serves as a resource for further exploration of the TCF3::HLF regulome. Teaser Unraveling the 3D genomic interactions mediated by TCF3::HLF fusion protein in t(17;19) positive acute lymphoblastic leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6a83250843f16bfbcbf3d66e2f1917ed01c269" target='_blank'>
              TCF3::HLF Orchestrates an Enhancer-Promoter Network with Activation of MEF2C to Promote Immature HSC gene Expression in Leukemia
              </a>
            </td>
          <td>
            Valdemar Priebe, Aneta Drakul, Bartimée Galvan, J. Aguadé-Gorgorió, Hanna K. A. Mikkola, B. Bornhauser, Raffaella Santoro, Jean-Pierre Bourquin
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2ef81377921558c83081b12d35bab26833147b" target='_blank'>
              Extrachromosomal circular DNA containing DTX1 promotes cell growth in hydroquinone-induced malignantly transformed cells by regulating the transcription of DTX1
              </a>
            </td>
          <td>
            X. Ling, Qunfang Jiao, Daifan Lin, Jialong Chen, Yali Han, Jinxue Meng, Bohuan Zhong, He Zhang, Gongda Zhang, Fangling Zhu, Jiheng Qin, Yongdui Ruan, Linhua Liu
          </td>
          <td>2024-11-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a4f47822586a1fd8d669e38036044aa6da804e" target='_blank'>
              Chromothripsis in cancer.
              </a>
            </td>
          <td>
            Milena Simovic-Lorenz, Aurélie Ernst
          </td>
          <td>2024-11-15</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9f63e7b6f9db6db3b912732c476af942c10e05" target='_blank'>
              Decreased mitochondrial transcription factor A and mitochondrial DNA copy number promote cyclin-dependent kinase inhibitor 1A expression and reduce tumorigenic properties of colorectal cancer cells
              </a>
            </td>
          <td>
            Jessika Buchwaldt, Tania Fritsch, Monika Hartmann, H. Witzel, Michael Kloth, Wilfried Roth, K. Tagscherer, Nils Hartmann
          </td>
          <td>2024-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chromosomal rearrangements on the short arm of chromosome 8 cause 8p syndrome, a rare developmental disorder characterized by neurodevelopmental delays, epilepsy, and cardiac abnormalities. While significant progress has been made in managing the symptoms of 8p syndrome and other conditions caused by large-scale chromosomal aneuploidies, no therapeutic approach has yet been demonstrated to target the underlying disease-causing chromosome. Here, we establish a two-step approach to eliminate the abnormal copy of chromosome 8 and restore euploidy in cells derived from an individual with a complex rearrangement of chromosome 8p. Transcriptomic analysis revealed 361 differentially expressed genes between the proband and the euploid revertant, highlighting genes both within and outside the 8p region that may contribute to 8p syndrome pathology. Our work demonstrates the feasibility of using chromosome engineering to correct complex aneuploidies in vitro and suggests a potential therapeutic avenue for disorders caused by chromosomal rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9c5433c28b14d1239c7306435b8438d42ccf6" target='_blank'>
              Chromosome engineering to restore euploidy in cells harboring a complex rearrangement of chromosome 8
              </a>
            </td>
          <td>
            Sophia N. Lee, Lu Qiao, Sarah L. Thompson, R. Hagenson, Teresa Davoli, J. Sheltzer
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ebab4a081c455b627a48f2ce35c264b01bf2033" target='_blank'>
              EZH2 inhibition sensitizes MYC-high medulloblastoma cancers to PARP inhibition by regulating NUPR1-mediated DNA repair.
              </a>
            </td>
          <td>
            Jianzhong Yu, Jichang Han, Meng Yu, Huanwen Rui, An Sun, Hao Li
          </td>
          <td>2024-11-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alternative Lengthening of Telomeres (ALT) is a homologous recombination-dependent telomere elongation mechanism utilized by at least 10-15% of all cancers. Here we identified that the DNA topoisomerase, TOP3A is enriched at the telomeres of ALT cells but not at the telomeres of telomerase-positive (Tel) cancer cells. We demonstrate that TOP3A stabilizes the shelterin protein TERF2 in ALT cancer cell lines but not in Tel cells and that long non-coding telomere transcribed RNA (TERRA) enrichment at telomeres depends upon TOP3A. TOP3A also promotes the generation of single-stranded telomeric C-strand (ssTeloC) DNA, which is a recently discovered marker for ALT. Additionally, we found that inducing TOP3A-DNA-protein crosslinks in ALT cells suppresses TERRA enrichment as well as destabilizes TERF2. Taken together these observations uncover the unexplored functions of TOP3A at ALT telomeres and suggest the potential of developing an ALT-specific cancer therapeutic strategy targeting TOP3A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e2b4b91b59c932435955d5d27a82b4340d75f06" target='_blank'>
              Topoisomerase 3α (TOP3A) Dependent Alternative Lengthening of Telomeres (ALT)
              </a>
            </td>
          <td>
            Prashant Khandagale, Yilun Sun, Sourav Saha, L. Saha, Y. Pommier
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ (WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors, and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72eaa094c63444d04a8a1f5ac2cb2674cf4fc8fe" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Alexander Howard, Bertram Aschenbrenner, Emily Lay, Natalia Krajic, Emma Bailey, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis.
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcd0f4cc9961826189852b167ab5b2ef51c1fec" target='_blank'>
              Integrating Circle-Seq with transcriptomics reveals genome-wide characterization of extrachromosomal circular DNA for dilated cardiomyopathy
              </a>
            </td>
          <td>
            Zhenhao Lin, Fangjie Dai, Bo Li, Yongchao Zhao, Changqian Wang
          </td>
          <td>2024-11-29</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin D. Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G. Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Understanding genome duplication requires characterisation of the locations where DNA replication initiates, termed origins. Genome-wide mapping of DNA replication origins has mainly been derived from population-based techniques, with only a few studies examining origin location and usage at the single-cell or single-molecule level. Leishmania are protozoan parasites where the first attempt to map DNA replication suggested the unprecedented use, for a eukaryote, of just a single origin per chromosome, while a subsequent approach suggested around 200-fold more origins. To reconcile these data and understand DNA replication dynamics in Leishmania major, we have applied DNAscent, a deep learning assay that uses long-read Nanopore sequencing to detect patterns of BrdU incorporation in individual DNA molecules, allowing the description of DNA replication fork movement and prediction of initiation and termination sites across the parasite genome. Our findings confirm the pre-eminence of a single locus of DNA replication initiation in each chromosome and reveal that this locus alone is constitutively activated in S-phase, with bidirectional forks emerging from discrete sites at the ends of multigene transcription units. DNAscent also reveals a much larger number of DNA replication initiation events that have not been detected in any previous mapping and are used stochastically, but whose abundance is greater as chromosome size increases. We show that each of these stochastic initiation sites localise to regions with high AT content, increased G-quadruplex levels and lower chromatin occupancy. In addition, we find markedly increased stochastic DNA replication initiation at sites with lower levels of nascent RNA transcripts. Finally, we show that all DNA replication initiation events result in mutagenesis. This work reveals a novel, bimodal strategy for DNA replication programming in Leishmania that drives genome transmission, replication timing and variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac0123b2b2131e9fee9fa34ca58baebaafb46eac" target='_blank'>
              Nuclear DNA replication in Leishmania major relies on a single constitutive origin per chromosome supplemented by thousands of stochastic initiation events
              </a>
            </td>
          <td>
            J. Damasceno, G.L.A. Silva, C. Marques, M. Krasiļņikova, C. Lapsley, Dario Beraldi, R. McCulloch
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Large genomic rearrangements of the PALB2 gene, especially deletions and duplications, are associated with hereditary breast–ovarian cancer. This study investigates intronic breakpoints linked to rearrangements in PALB2 exon 11, which is crucial for understanding the mechanisms affecting patients with hereditary breast and ovarian syndrome. Through next-generation sequencing, one duplication and three deletions were identified, confirmed by Multiplex Ligation-Dependent Probe Amplification. Detailed characterization revealed a tandem duplication of 5134 base pairs mediated by AluY repeats and identical deletions in three unrelated patients promoted by AluSx elements, resulting in a truncated PALB2 protein. These findings highlight the instability of intronic regions flanking exon 11 and suggest directions for future research on the prevalence and functional implications of these genomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30610e3b0b4ee5300bddce9e6a181cf9f25f21c9" target='_blank'>
              Alu–Mediated Duplication and Deletion of Exon 11 Are Frequent Mechanisms of PALB2 Inactivation, Predisposing Individuals to Hereditary Breast–Ovarian Cancer Syndrome
              </a>
            </td>
          <td>
            Diletta Sidoti, Valeria Margotta, Diletta Calosci, Erika Fiorentini, C. Bacci, Francesca Gensini, Laura Papi, Marco Montini
          </td>
          <td>2024-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA methylation (DNAm) is a key epigenetic mark that modulates regulatory elements and gene expression, playing a crucial role in mammalian development and physiological function. Despite extensive characterization of DNAm profiles across species, little is known about its evolutionary conservation. Here, we conducted a comparative epigenome-wide analysis of great apes to identify and characterize sequence- and methylation-conserved CpGs (MCCs). Using 202 DNAm arrays, alongside 6 matched genotype and 13 matched transcriptomic datasets, we identified 11,500 MCCs for which methylation was evolutionarily related to sequences of CpGs and methylation quantitative trait loci. MCCs were the most stable across healthy human tissues and exhibited weaker genetic associations than other CpGs. Moreover, MCCs showed minimal associations with demographic, environmental factors, and noncancer diseases, yet demonstrated stronger associations with certain cancers than other CpGs, particularly gastrointestinal cancers. Functional enrichment analysis revealed that genes associated with MCC methylation in cancer were enriched for cancer driver genes and canonical cancer pathways, highlighting a significant regulatory role for MCCs in tumorigenesis. Collectively, our findings reveal the extent of DNAm conservation in great ape evolution, its association with genetic conservation, and its relevance to human diseases. These integrative analyses offer evolutionary insights into epigenetic variation and its functional implications in human populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ff2d13f4df89f2515822a3ea179e1bce4a04b6" target='_blank'>
              Genetic Basis, Quantitative Nature, and Functional Relevance of Evolutionarily Conserved DNA Methylation
              </a>
            </td>
          <td>
            Zheng (Joe) Dong, Samantha Schaffner, Maggie Fu, Joanne Whitehead, Julia L. MacIsaac, David H. Rehkopf, W. T. Boyce, Luis Rosero-Bixby, L. Quintana-Murci, E. Patin, Gregory E. Miller, Keegan Korthauer, Michael S. Kobor
          </td>
          <td>2024-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5-34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks.
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="For long, genome-wide coverage has been used as a measure of sequencing quality and quantity, but the biology hidden beneath has not been fully exploited. Here we performed comparative analyses on genome-wide coverage profiles between nucleus genome DNA (gDNA) samples from the 1000 Genomes Project (n=3,202) and cell-free DNA (cfDNA) samples from healthy controls (n=113) or cancer patients (n=362). Regardless of sample type, we observed an overall conserved landscape with coverage segmentation, where similar levels of coverage were shared among adjacent windows of genome positions. Besides GC-content, we identified protein-coding gene density and nucleosome density as major factors affecting the coverage of gDNA and cfDNA, respectively. Differential coverage of cfDNA vs gDNA was found in immune-receptor loci, intergenic regions and non-coding genes, reflecting distinct genome activities in different cell types. A further rise in coverage at non-coding genes/intergenic regions and a further drop of coverage at protein-coding genes/genic regions within cancer cfDNA samples suggested a relative loss of contribution by normal cells. Importantly, we observed the distinctive convergence of coverage in cancer-derived cfDNA, with the extent of convergence positively correlated to stages. Based on the findings we developed and validated an outlier-detection approach for cfDNA-based cancer screening without the need of cancer samples for training. The method achieved 97% sensitivity on pediatric sarcomas (n=241) and 44% sensitivity on early-stage lung cancers (n=36) with >90% specificity for condition-matched tasks, 100% sensitivity on late-stage cancers (n=85) for condition-unmatched tasks, outperforming current benchmarks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58763bc618f0e854c5267409a63aba70151b9e8e" target='_blank'>
              Coverage landscape of the human genome in nucleus DNA and cell-free DNA
              </a>
            </td>
          <td>
            Jiaqi Luo, Shuaicheng Li
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the most common cancer in women and the 2nd most common cancer worldwide, yearly impacting over 2 million females and causing 650 thousand deaths. It has been widely studied, but its epigenetic variation is not entirely unveiled. We aimed to identify epigenetic mechanisms impacting the expression of breast cancer related genes to detect new potential biomarkers and therapeutic targets. We considered The Cancer Genome Atlas database with over 800 samples and several omics datasets such as mRNA, miRNA, DNA methylation, which we used to select 2701 features that were statistically significant to differ between cancer and control samples using the Monte Carlo Feature Selection and Interdependency Discovery algorithm, from an initial total of 417,486. Their biological impact on cancerogenesis was confirmed using: statistical analysis, natural language processing, linear and machine learning models as well as: transcription factors identification, drugs and 3D chromatin structure analyses. Classification of cancer vs control samples on the selected features returned high classification weighted Accuracy from 0.91 to 0.98 depending on feature-type: mRNA, miRNA, DNA methylation, and classification algorithm. In general, cancer samples showed lower expression of differentially expressed genes and increased β-values of differentially methylated sites. We identified mRNAs whose expression is well explained by miRNA expression and differentially methylated sites β-values. We recognized differentially methylated sites possibly affecting NRF1 and MXI1 transcription factors binding, causing a disturbance in NKAPL and PITX1 expression, respectively. Our 3D models showed more loosely packed chromatin in cancer. This study successfully points out numerous possible regulatory dependencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471e2fdeae8d795259c36bd54b5931925e83e88a" target='_blank'>
              Unveiling epigenetic regulatory elements associated with breast cancer development
              </a>
            </td>
          <td>
            Marta Jardanowska-Kotuniak, Michał Dramiński, Michał Wlasnowolski, Marcin Łapiński, Kaustav Sengupta, Abhishek Agarwal, Adam Filip, Nimisha Ghosh, Vera Pancaldi, Marcin Grynberg, I. Saha, Dariusz Plewczyński, Michal J Dabrowski
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Telomeric repeat-containing RNA (TERRA), transcribed from subtelomeric regions towards telomeric ends, poses challenges in deciphering its complete sequences. Utilizing TERRA-capture RNA-seq and Oxford Nanopore direct RNA sequencing to acquire full-length TERRA, we annotate TERRA transcription regions in the human T2T-CHM13 reference genome. TERRA transcripts encompass hundreds to over a thousand nucleotides of telomeric repeats, predominantly originating from 61-29-37 bp repeat promoters enriched with H3K4me3, RNA pol II, CTCF, and R-loops. We develop a bioinformatics tool, TERRA-QUANT, for quantifying TERRA using RNA-seq datasets and find that TERRA increases with age in blood, brain, and fibroblasts. TERRA upregulation in aged leukocytes is confirmed by RT-qPCR. Single-cell RNA-seq analysis demonstrates TERRA expression across various cell types, with upregulation observed in neurons during human embryonic stem cell differentiation. Additionally, TERRA levels are elevated in brain cells in the early stage of Alzheimer’s disease. Our study provides evidence linking TERRA to human aging and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fedf4165a727cf3608fb4e1f9bd6a9359a39d7e" target='_blank'>
              Telomeric Repeat-Containing RNA Increases in Aged Human Cells
              </a>
            </td>
          <td>
            Yu-Hung Hsieh, Chin-Hua Tai, Meng-Ting Yeh, Yu-Chen Chen, Po-Cheng Yang, Chien-Ping Yen, Der-Sheng Han, Hsueh-Ping Catherine Chu
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 The stepwise progression of colorectal cancer (CRC) from healthy epithelium, to premalignant adenoma, to cancer is accompanied by genome-wide epigenetic reprogramming which may be reflected in circulating cell-free DNA (cfDNA). However, current analyses of these processes have been hampered by complex mixtures of cells in normal, adenoma and cancer tissue samples, as well as absence of suitable adenoma model systems. Cultured colorectal organoids, comprised of pure epithelial cells, could enable insights on the epigenetic changes leading to progression from healthy tissue to cancer. Such features could be instrumental in the early cancer detection using genome-wide analyses of cfDNA fragmentation. Using 43 patient-derived colorectal organoids derived from healthy, adenoma, and cancer tissues, as well as 10 freshly collected peripheral blood mononuclear cell (PBMC) control samples, we examined the chromatin landscape of tumor progression using transposase accessible chromatin analyses with next-generation sequencing. We defined a consensus set of nucleosome depleted regions for each disease state and identified loci with differential accessibility between PBMCs, colon healthy, colon adenoma, and colon cancer tissues. We analyzed these differential loci for fragmentation characteristics in the circulating cfDNA of 250 healthy individuals and 51 patients with metastatic CRC. Across all organoid samples, we identified >35 million nucleosome depleted regions that we coalesced into a consensus set of 62,708 regions. As 78% of these regions could be linked to genes, we assessed whether these signatures identify known pathways involved in colorectal cancer. Within these regions, we identified unique colon tumor-related signatures comprising genes of known and novel pathways, including those involved in WNT, hippo, and RAS signaling. While the majority of chromatin accessibility differences were shared between adenomas and cancers, we identified 895 chromatin changes in adenomas that were not present in either healthy or malignant tissues. In healthy individuals, we found that the cfDNA coverage was negatively correlated with accessibility of white blood cell accessibility loci (Spearman ρ= - 0.60) while in individuals with cancer, changes in chromatin accessibility were associated with independent circulating tumor DNA abundance by droplet digital PCR (R2=0.69, p<0.001). Chromatin accessibility profiles were correlated with genome-wide cell-free DNA fragmentation features and could be used to distinguish healthy individuals from those with colorectal cancer (AUC=0.97). These analyses are consistent with dynamic chromatin remodeling during colorectal tumor development and may provide an avenue for evaluating these through genome-wide analyses of cfDNA fragmentation.
 Citation Format: Nicholas A. Vulpescu, Zachariah H. Foda, Pieter H.A. Wisse, Christopher Cherry, Jaime E. Medina, Vilmos Adleff, Remond J.A. Fijneman, Robert B. Scharpf, Gerrit A. Meijer, Beatriz Carvalho, Victor E. Velculescu. Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37db1a1abcfb5adfff7334fbd173d8e5404d5cef" target='_blank'>
              Abstract PR016: Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, Jaime E. Medina, V. Adleff, R. Fijneman, Robert B Scharpf, Gerrit A Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="PURPOSE
Mutational data from multiple solid and liquid biospecimens of a single patient is often integrated to track cancer evolution. However, there is no accepted framework to resolve if individual samples from the same individual share variants due to common identity versus coincidence.


EXPERIMENTAL DESIGN
Utilizing 8,000 patient tumors from The Cancer Genome Atlas (TCGA) across 33 cancer types, we estimated background rates of co-occurrence rates of mutations between discrete pairs of samples across cancers and by cancer type. We developed a mutational profile similarity score (MPS) that uses a large background database to produce confidence estimates that two tumors share a unique, related molecular profile. The MPS algorithm was applied to randomly paired tumor profiles, including patients who underwent repeat solid tumor biopsies sequenced with MSK-IMPACT (n=53,113). We also evaluated the MPS in sample pairs from single patients with multiple cancers (n=2,012), as well as patients with plasma and solid-tumor variant profiles (n=884 patients).


RESULTS
In unrelated tumors, nucleotide-specific variants are shared in 1.3% (cancer-type agnostic) and in 10-13% (cancer-type specific) of cases. The mutational profile similarity (MPS) method contextualized shared variants to specify whether patients had a single cancer versus multiple distinct cancers. When multiple tumors were compared from the same patient, and an initial clinicopathologic diagnosis was discordant with molecular findings, the MPS anticipated future diagnosis changes in 28% of examined cases.


CONCLUSIONS
Use of a novel shared variant framework can provide information to clarify the molecular relationship between compared biospecimens with minimal required input.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d94e9e3b930eb4e5ed54aac8313a47e9f94748e0" target='_blank'>
              Analysis of shared variants between cancer biospecimens.
              </a>
            </td>
          <td>
            Michael B Foote, J. R. White, W. Chatila, Guillem Argilés, Steve Lu, B. Rousseau, O. Artz, P. Johannet, H. Walch, Miteshkumar Patel, M. Lamendola-Essel, D. Casadevall, Somer Abdelfattah, Shrey Patel, R. Yaeger, A. Cercek, Clara Montagut, Michael F. Berger, N. Schultz, Luis A. Diaz
          </td>
          <td>2024-11-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The mitochondrial genome, which encodes genes essential for respiration and cellular homeostasis, is the target of abundant and highly diverse somatic alterations in cancers. Somatic alterations to mitochondrial DNA (mtDNA) nearly always arise heteroplasmically, producing heterogeneous ensembles of mtDNA within single cells. Here, we review new insights derived from exponential increases in genomic sequencing data that have uncovered the nature of, selective pressure for, and functional consequences of cancer-associated mtDNA alterations. As many discoveries have been limited by their ability to determine cell-to-cell variation in mtDNA genotype, we describe a new generation of single-cell sequencing approaches that resolve otherwise indeterminate models of mtDNA heteroplasmy. In tandem with novel approaches for mtDNA editing and modeling of mutations, these advances foreshadow the quantitative dissection of dosage-dependent mtDNA phenotypes that underlie both tumor evolution and heterogeneous response to therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823dce23cbbddef91125605bfb3305bbb4cdca3d" target='_blank'>
              Single-Cell Technologies for Studying the Evolution and Function of Mitochondrial DNA Heteroplasmy in Cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Erin M. Cumming, Minsoo Kim, Caleb Lareau, Ed Reznik
          </td>
          <td>2024-11-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) refers to a class of circular, non-chromosomal DNA that has recently gained widespread attention due to its potential role in aging and neurodegenerative diseases. The generation of ecDNA is closely associated with processes such as double-strand breaks, micronuclei formation, and the breakage-fusion-bridge (BFB) cycle, all of which are integral to regulation of gene expression, genetic stability, and clonal evolution. In neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, and Huntington's disease, the aberrant formation of ecDNA is closely linked to defects in DNA repair, alterations in synaptic plasticity, and neuronal dysfunction. The distinct distribution and functional roles of ecDNA in these conditions make it a potential diagnostic biomarker and therapeutic target. This review provides an overview of the mechanisms underlying ecDNA formation and its functions in the nervous system. Additionally, it explores the clinical potential of ecDNA in disease diagnosis, targeted therapy, and personalized medicine, offering new insights for future research and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d58ac80378b185b8a630604a71f2ebc6449a88f9" target='_blank'>
              Extrachromosomal DNA: Molecular perspectives in aging and neurodegenerative diseases.
              </a>
            </td>
          <td>
            Yafeng Ma, Ying Xia, Kenji Karako, Peipei Song, Xiqi Hu
          </td>
          <td>2024-11-30</td>
          <td>Intractable & rare diseases research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The study explores the role of KIF18A, a gene linked to whole-genome doubling (WGD) and chromosomal instability (CIN), in the metastatic progression of hepatocellular carcinoma (HCC). We found that KIF18A is a critical link between CIN and anoikis resistance, key factors in HCC metastasis. Through the bulk-seq analysis, we identified the pressures of anoikis and CIN faced by tumor cells during HCC metastasis and identified hub genes central to this process. Our results reveal that E2F activation during HCC progression leads to the transcription of KIF18A, promoting the survival and metastasis of CIN tumors. Overexpression of KIF18A leads to longer tumor life, more micronuclei, and increased survival and metastasis through non-canonical NF-kB activation. Deletion of KIF18A stabilizes the nuclear membrane of the micronucleus, silences cGAS-STING and PI3K-AKT pathways, and inhibits classical NF-kB. The study’s limitations include the need for further animal studies to validate the findings and explore the transient activation of the cGAS-STING pathway. Additionally, future research could focus on the potential therapeutic implications of targeting KIF18A in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae22aedbaa5a2bdc1f487c6a0f986379d3bb08c" target='_blank'>
              KIF18A in HCC Metastasis: Dual Role in Anoikis Resistance and Chromosome Instability
              </a>
            </td>
          <td>
            Changhao Kan, Weiyu Bai, Xinru Yang, Hayam Hamdy, Xianghui Yang, Xin Liu, Yan Lv, Junling Shen, Jianwei Sun
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) liquid biopsies offer a powerful, non-invasive approach for cancer diagnostics, leveraging DNA fragments primarily released during apoptosis. Using droplet digital PCR on cfDNA samples from 88 breast and colorectal cancer patients, we uncovered highly consistent nucleosome breakpoints across individuals. Proximity to these cfDNA breakpoints diminishes the sensitivity of somatic mutation assays, including those targeting IDH1 and NRAS hotspot mutations, underscoring the importance of precise nucleosome positioning data. We introduce a probabilistic nucleosome protection scoring method that enables high-resolution mapping of nucleosome positions and cfDNA breakpoints. These maps reveal distinct nucleosome spacing patterns between heterochromatin and euchromatin, corresponding to different chromatin fiber topologies. Analysis of X-chromosome inactivation further supports the topoisomer model: heterochromatin’s 187 bp nucleosome repeat length differs from euchromatin’s 182 bp, generating wave interference patterns in nucleosome signals. Our findings suggest that loop-mediated transitions from T2 to T1 topoisomers shape chromatin accessibility and cfDNA fragmentation, advancing the diagnostic potential of liquid biopsy assays. Highlights Nucleosome breakpoint conservation impacts somatic mutation assay sensitivity Probabilistic protection scoring enhances cfDNA nucleosome peak resolution Chromatin states exhibit unique nucleosome spacing linked to distinct topologies, influencing cfDNA fragmentation Chromosome X-inactivation phasing analyses reveal waveform interference in nucleosome positioning signals Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72833584f0c36d4300251f882a1e6ae1ee535bda" target='_blank'>
              Unraveling cfDNA Fragmentation Patterns: Linking Chromatin Architecture to Cancer Diagnostic Sensitivity
              </a>
            </td>
          <td>
            Andrew D Johnston, Fiach Antaw, Jennifer Lu, C. Stirzaker, M. Trau, D. Korbie
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The 22q11.2 deletion syndrome (22q11.2DS) is the most common microdeletion disorder. Why the incidence of 22q11.2DS is much greater than that of other genomic disorders remains unknown. Short read sequencing cannot resolve the complex segmental duplicon structure to provide direct confirmation of the hypothesis that the rearrangements are caused by nonallelic homologous recombination between the low copy repeats on Chromosome 22 (LCR22s). To enable haplotype-specific assembly and rearrangement mapping in LCR22 regions, we combined fiber-FISH optical mapping with whole genome (ultra-)long read sequencing or rearrangement-specific long-range PCR on 24 duos (22q11.2DS patient and parent-of-origin) comprising several different LCR22-mediated rearrangements. Unexpectedly, we demonstrate that not only different paralogous segmental duplicon but also palindromic AT-rich repeats (PATRR) are driving 22q11.2 rearrangements. In addition, we show the existence of two different inversion polymorphisms preceding rearrangement, and somatic mosaicism. The existence of different recombination sites and mechanisms in paralogues and PATRRs which are copy number expanding in the human population are a likely explanation for the high 22q11.2DS incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769f0987422b68baa29d9145bb1d8e87d9879537" target='_blank'>
              Multiple paralogues and recombination mechanisms contribute to the high incidence of 22q11.2 Deletion Syndrome.
              </a>
            </td>
          <td>
            Lisanne Vervoort, Nicolas Dierckxsens, Marta Sousa Santos, Senne Meynants, E. Souche, Ruben Cools, T. Heung, Koen Devriendt, Hilde Peeters, Donna M. McDonald-McGinn, A. Swillen, J. Breckpot, Beverly S Emanuel, H. Van Esch, Anne S. Bassett, J. Vermeesch
          </td>
          <td>2024-11-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Glioblastoma is recognized as the most aggressive type of primary brain tumor. Despite recent advances in understanding the molecular mechanisms involved in the biology of glioblastoma, patient survival rates remain disappointing, primarily due to the lack of effective treatment options. Tumor necrosis factor receptor-associated protein 1 (TRAP1), a member of the heat shock protein 90 (Hsp90) family, refers to a protein predominantly localized in the mitochondria that regulates both cellular metabolic reprogramming and mitochondrial apoptosis. This protein is highly expressed in several types of tumors, including colorectal cancer, breast cancer, prostate cancer, and lung cancer, and is often associated with drug resistance. However, TRAP1 is also downregulated in certain cancers such as ovarian cancer, bladder cancer, and renal cancer, where its lower expression correlates with poorer prognoses and chemoresistance. The role of TRAP1 lies in enhancing or suppressing oxidative phosphorylation, with the impact of such regulation on tumor development and progression being a matter of ongoing debate. These observations prompt further investigation into the mechanisms responsible for the dual role of TRAP1 as both an oncogene and a tumor suppressor in specific types of tumors, particularly glioblastoma. The present study reviews the role of TRAP1 in the development and progression of glioblastoma and discusses the potential of targeting TRAP1 as a novel therapeutic approach against tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/236ff07b8d8eaf07109bcd843d2d90948b45e7ad" target='_blank'>
              Role of TRAP1 Protein in the Development and Progression of Glioblastoma
              </a>
            </td>
          <td>
            I. Gareev, A. Yasinskaya, S. Roumiantsev
          </td>
          <td>2024-12-28</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most common malignant brain tumour in children. The Sonic Hedgehog (SHH)-medulloblastoma subtype arises from the cerebellar granule neuron lineage. Terminally differentiated neurons are incapable of undergoing further cell division, so an effective treatment for this tumour could be to force neuronal differentiation. Differentiation therapy provides a potential alternative for patients with medulloblastoma who harbor mutations that impair cell death pathways (TP53), which is associated a with high mortality. To this end, our goal was to explore epigenetic regulation of cerebellar granule neuron differentiation in medulloblastoma cells. Key regulators were discovered using chromatin immunoprecipitation with high-throughput sequencing. DNA-bound protein and chromatin protein modifications were investigated across all genes. We discovered that Ezh2-mediated tri-methylation of the H3 histone (H3K27me3), occurred on more than half of the 787 genes whose transcription normally increases as granule neurons terminally differentiate. Conditional knockout of Ezh2 led to early initiation of differentiation in granule neuron precursors (GNPs), but only after cell cycle exit had occurred. Similarly, in MB cells, neuronal differentiation could be induced by preventing H3K27me3 modifications using an Ezh2 inhibitor (UNC1999), but only when UNC1999 was combined with forced cell cycle exit driven by a CDK4/6 inhibitor (Palbociclib). Ezh2 emerges as a powerful restraint upon post-mitotic differentiation during normal GNP development and combination of Ezh2 inhibition with cell cycle exit leads to MB cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43dfd31ed674b4e0eafa420d9f64b5c59f37dda8" target='_blank'>
              Ezh2 Delays Activation of Differentiation Genes During Normal Cerebellar Granule Neuron Development and in Medulloblastoma
              </a>
            </td>
          <td>
            James Purzner, Alexander S. Brown, Teresa Purzner, Lauren Ellis, Sara Broski, Ulrike Litzenburger, Kaytlin Andrews, Aryaman Sharma, Xin Wang, Michael D. Taylor, Yoon-Jae Cho, Margaret T. Fuller, Matthew P. Scott
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Alterations in DNA methylation profiles are typically found in cancer cells, combining genome-wide hypomethylation with hypermethylation of specific regions, such as CpG islands, which are normally unmethylated. Driving effects in cancer development have been associated with alteration of DNA methylation in certain regions, inducing, for example, the repression of tumor suppressor genes or the activation of oncogenes and retrotransposons. These alterations represent prime candidates for the development of specific markers for the detection, diagnosis and prognosis of cancer. In particular, these markers, distributed along the genome, provide a wealth of information that offers potential for innovation in the field of liquid biopsy, in particular thanks to the emergence of artificial intelligence for diagnostic purposes. This could overcome the limitations related to sensitivities and specificities, which remain too low for the most difficult applications in oncology: the detection of cancers at an early stage, the monitoring of residual disease and the analysis of brain tumors. In addition, targeting the enzymatic processes that control the epigenome offers new therapeutic strategies that could reverse the regulatory anomalies of these altered epigenomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6734e14b7462be1947bbbaa4edaec73c090d31d" target='_blank'>
              [Epigenetics and cancer: the role of DNA methylation].
              </a>
            </td>
          <td>
            Marine Gorse, Charline Bianchi, Charlotte Proudhon
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-stranded DNA secondary structures such as G-quadruplexes (G4s) can potentially disrupt transcription, replication, and repair. Using bio-informatic analysis, here we show that BMI1 is enriched at putative G4s flanked by heterochromatin domains, and that BMI1 knockdown in human dermal fibroblasts (HDFs) resulted in heterochromatin relaxation and G4 induction, followed by replication stress and genomic instability. In these cells, G4s co-localized with large 53BP1 and PCNA foci resembling replication catastrophes. Inhibiting transcription partly attenuated DNA damage, suggesting rescue of transcription-replication collisions at difficult-to-replicate sequences. In BMI1 knockdown or pyridostatin-exposed HDFs, the Werner helicase accumulated and co-localized with G4s. Acute WRN knockdown also resulted in G4 induction. In HDFs from Werner and Hutchinson-Gilford progeria syndromes, loss of heterochromatin and nuclear envelope anomalies were associated with G4 induction and DNA damage. Nuclear envelope anomalies were also prominent following BMI1 knockdown. These findings suggests that heterochromatin-mediated repression of G4s attenuates replication stress and genomic instability, and that this mechanism is shared across distinct progeroid models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731920cfa53eda1eb3b00a50ee2eec0629da79d1" target='_blank'>
              BMI1-mediated heterochromatinization represses G-quadruplex DNA formation to maintain genomic stability during replication
              </a>
            </td>
          <td>
            Roy Hanna, Gilbert Bernier
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge.


EXPERIMENTAL DESIGN
We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. The DIAMOND (Detection of Long Interspersed Nuclear Element Altered Methylation ON plasma DNA) method targets 30-40,000 young L1 scattered throughout the genome, covering about 100,000 CpG sites and is based on a reference-free analysis pipeline.


RESULTS
Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers (colorectal, breast, lung, ovarian, gastric cancers and uveal melanoma including localized stages) in two independent cohorts (AUC = 88% to 100%, N = 747). DIAMOND can also be used to perform copy number alterations (CNA) analysis which improves cancer detection.


CONCLUSIONS
This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321c21ecf5626d3880de9fbd35f995ec3b32a1f7" target='_blank'>
              Non-invasive multi-cancer detection using DNA hypomethylation of LINE-1 retrotransposons.
              </a>
            </td>
          <td>
            M. Michel, Maryam Heidary, Anissa Mechri, Kévin Da Silva, Marine Gorse, Victoria Dixon, Klaus von Grafenstein, Charline Bianchi, C. Hego, A. Rampanou, C. Lamy, M. Kamal, C. Le Tourneau, M. Séné, I. Bieche, Cécile Reyes, D. Gentien, Marc-Henri Stern, Olivier Lantz, L. Cabel, J-Y Pierga, F. Bidard, Chloé-Agathe Azencott, C. Proudhon
          </td>
          <td>2024-12-02</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="To maintain genome stability, proliferating cells must enact a program of telomere maintenance. While most tumors maintain telomeres through the action of telomerase, a subset of tumors utilize a DNA-templated process termed Alternative Lengthening of Telomeres or ALT. ALT is associated with mutations in the ATRX/DAXX/H3.3 histone chaperone complex, which is responsible for deposition of non-replicative histone variant H3.3 at heterochromatic regions of the genome including telomeres. We wished to better understand the role DAXX plays in ALT suppression, and to determine which disease-associated DAXX mutations are unable to suppress ALT. To answer this question, we have leveraged the G292 cell line, in which ATRX is wild type but DAXX has undergone a fusion event with the non-canonical kinesin KIFC3. Restoration of wild-type DAXX in G292 localizes ATRX and abrogates ALT. Using this model system, we tested the ability of a panel of disease-associated DAXX missense variants to suppress ALT. Missense mutations in the ATRX binding domain, the histone binding domain, and the C-terminal SUMO interaction motif reduce the ability of DAXX to suppress ALT. Unexpectedly, we find that mutations in the DAXX histone binding domain lead to failure of ATRX localization. We conclude that a key function of DAXX in ALT suppression is the localization of ATRX to nuclear foci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecce9b8b3920bbf0f0b14800927fbabcdf8182c8" target='_blank'>
              Cancer-associated DAXX mutations reveal a critical role for ATRX localization in ALT suppression
              </a>
            </td>
          <td>
            Sarah F. Clatterbuck Soper, Robert L. Walker, M. Pineda, Y. Zhu, James Dalgleish, Jasmine Wang, Paul S. Meltzer
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e490249b3d823e7663d67cf908762a5ed74aa1" target='_blank'>
              YY1 drives PARP1 expression essential for PARylation of NONO in mRNA maturation during neuroblastoma progression
              </a>
            </td>
          <td>
            Chunhui Yang, Jiaying Qu, Yang Cheng, Minxiu Tian, Zhijie Wang, Xiaolin Wang, Xinyue Li, Shunchen Zhou, Bosen Zhao, Yanhua Guo, Liduan Zheng, Q. Tong
          </td>
          <td>2024-12-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R. Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia frequently harbor fusion oncogenes associated with poor prognosis, including KMT2A, NUP98 and GLIS2 rearrangements. While murine models have demonstrated their leukemogenic activities, the steps from a normal human cell to leukemic blasts remain unclear. Here, we precisely reproduced the inversion of chromosome 16 resulting in ETO2::GLIS2 fusion in human induced pluripotent stem cells (iPSC). IPSC-derived ETO2::GLIS2-expressing hematopoietic cells showed differentiation alterations in vitro and efficiently induced in vivo development of leukemia that closely phenocopied human acute megakaryoblastic leukemia (AMKL) reflected by flow cytometry and single cell transcriptomes. Comparison of iPS-derived cells with patient-derived cells revealed altered chromatin accessibility at early and later bona fide leukemia stages with aberrantly higher accessibility and expression of the osteogenic homeobox factor DLX3 that preceded increased accessibility to ETS factors. DLX3 overexpression in normal CD34+ cells increased accessibility to ETS motifs and reduced accessibility to GATA motifs. A DLX3 transcriptional module was globally enriched in both ETO2::GLIS2 AMKL and some aggressive pediatric osteosarcoma. Importantly, DLX3 knock-out abrogated leukemia initiation in this ETO2::GLIS2 iPSC model. Collectively, characterization of a novel human iPSC-derived AMKL model revealed hijacking of the osteogenic homeobox transcription factor DLX3 as an essential early step in chromatin changes and leukemogenesis driven by the ETO2::GLIS2 fusion oncogene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d639e01bc6f7f80f46142030b31449774684dd" target='_blank'>
              Progressive chromatin rewiring by ETO2::GLIS2 revealed in a human iPSC model of pediatric leukemia initiation.
              </a>
            </td>
          <td>
            Fabien Boudia, Marie-Laurence Baille, Loélia Babin, Zakia Aid, E. Robert, Julie Riviere, Klaudia Galant, Verónica Alonso-Pérez, Laura Anselmi, Brahim Arkoun, N. Abermil, Christophe Marzac, S. Bertuccio, Alexia Regnault de Premesnil, Cécile K. Lopez, Alexandre Eeckhoutte, Audrey Naimo, B. Leite, C. Catelain, Christophe Metereau, Patrick Gonin, Nathalie Gaspar, J. Schwaller, Olivier A Bernard, H. Raslova, Murielle Gaudry, A. Marchais, Hélène Lapillonne, Arnaud Petit, Françoise Pflumio, M. Arcangeli, Erika Brunet, T. Mercher
          </td>
          <td>2024-12-10</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras-particularly in diseases such as oral squamous cell carcinoma and glioblastoma-now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy.
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, Tali Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Emerging evidence indicates that aberrations in the writing, reading, or erasure of chromatin modification codes are pivotal events in various human cancers. In this study, we conducted a systematic investigation of histone modification recognition proteins in a pair of colorectal cancer cell lines, SW480 and SW620. Using chromatin fractionation combined with data-independent acquisition mass spectrometry (DIA-MS), we developed a robust method to identify changes in histone modification recognition proteins during cancer progression. Our analysis revealed 22 proteins that were significantly upregulated and 22 proteins that were significantly downregulated in SW620 cells compared to SW480 cells. Notably, SMARCA1, a member of the ISWI family belonging to ATP-dependent chromatin remodeling complexes, was downregulated in SW620 cells compared to SW480 cells. Its high expression was strongly correlated with poor patient prognosis, aligning with the proposed role of SMARCA1 in promoting colorectal cancer (CRC) metastasis. Furthermore, reduced SMARCA1 expression altered the levels of metastasis-related matrix metalloproteinases (MMPs) in these cells. In conclusion, by systematically profiling histone modification recognition proteins in a matched pair of primary and metastatic CRC cell lines, we identified SMARCA1 as a potential driver of CRC metastasis and a promising therapeutic target for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43b84672f56bb75e7d699d7465b27140159f571" target='_blank'>
              Identification of SMARCA1 as a key regulator for Colorectal Cancer
              </a>
            </td>
          <td>
            Chunqing Fu, Shoufeng Duan, Chengwen Zhang, Xinyu Cui, Fengxian Wang, Lichen Wang, Jinglei Hu, Tengjiao Li, Lin Li
          </td>
          <td>2024-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The nucleolus is a major subnuclear compartment where ribosomal DNA (rDNA) is transcribed and ribosomes are assembled. In addition, recent studies have shown that the nucleolus is a dynamic organizer of chromatin architecture that modulates developmental gene expression. rDNA gene units are assembled into arrays located in the p-arms of five human acrocentric chromosomes. Distal junctions (DJs) are ∼400 kb sequences adjacent to rDNA arrays that are thought to anchor them at the nucleolus, although the underlying regulatory elements remain unclear. Here we show that DJs display a dynamic chromosome conformation profile in human embryonic stem cells (hESCs). We identified a primate-specific, full-length insertion of the retrotransposon long interspersed nuclear element 1 (LINE1) in a conserved position across all human DJs. This DJ-LINE1 locus interacts with specific regions of the DJ and is upregulated in naïve hESCs. CRISPR-based deletion and interference approaches revealed that DJ-LINE1 contributes to nucleolar positioning of the DJs. Moreover, we found that the expression of DJ-LINE1 is required for maintenance of the structure and transcriptional output of the nucleolus in hESCs. Silencing of DJ-LINE1 leads to loss of self-renewal, disruption of the landscape of chromatin accessibility, and derepression of earlier developmental programs in naïve hESCs. This work uncovers specific LINE1 elements with a fundamental role in nucleolar organization in hESCs and provides new insights into how the nucleolus functions as a key genome-organizing hub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6f03971b0054eedbed60fad051bf9b167c6012" target='_blank'>
              LINE1 elements at distal junctions of rDNA repeats regulate nucleolar organization in human embryonic stem cells
              </a>
            </td>
          <td>
            Lamisa Ataei, Juan Zhang, Simon Monis, Krystyna Giemza, Kirti Mittal, Joshua Yang, Mayu Shimomura, B. McStay, Michael D. Wilson, M. Ramalho-Santos
          </td>
          <td>2024-12-20</td>
          <td>Genes &amp; Development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5907495884a56c80db9ffbc9876a701f5ced793e" target='_blank'>
              Role of BRCA1 in glioblastoma etiology.
              </a>
            </td>
          <td>
            Emirhan Harbi, Michael Aschner
          </td>
          <td>2024-12-10</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12abdeefb14dd8084b12c06a67b90a6d3dae88f" target='_blank'>
              CPNE7 promotes colorectal tumorigenesis by interacting with NONO to initiate ZFP42 transcription
              </a>
            </td>
          <td>
            Liangbo Zhao, Xiao Sun, Chenying Hou, Yanmei Yang, Peiwen Wang, Zhaoyuan Xu, Zhenzhen Chen, Xiangrui Zhang, Guanghua Wu, Hong Chen, Hao Xing, Huimin Xie, Luyun He, Shuiling Jin, Benyu Liu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 We evaluated the ability of plasma chromatin immunoprecipitation and sequencing (ChIP-seq) and methylated DNA immunoprecipitation sequencing (MeDIP-seq) to detect translocation renal cell carcinoma (tRCC), to distinguish tRCC from clear cell renal cell carcinoma (ccRCC), and to monitor the disease evolution. tRCC is an aggressive subtype of kidney cancer usually driven by a fusion involving the TFE3 gene. Due to histologic overlap with other subtypes of RCC, tRCC is frequently misclassified. Methods for accurate diagnosis and detection of this molecularly distinct entity are therefore a pressing need. Most cell-free DNA (cfDNA) technologies are genomic-based and not optimized for tRCC as molecular fusions are not easily detected in circulating tumor DNA (ctDNA), and 50% of tRCC cases do not have other alteration. Epigenomic profiling of ctDNA via ChIP-seq has recently emerged as a powerful tool to detect and molecularly subtype cancers and may offer a more sensitive and specific detection of fusion- driven tumors. We first identified differentially methylated regions (DMRs) and regulatory elements (REs) specific to tRCC vs. ccRCC via MeDIP-seq and H3K4me3/H3K27Ac ChIP-Seq, of 4 tRCC and 5 ccRCC cell lines. We also identified TFE3 transcription factor binding sites (TFBS) via TFE3 ChIP-seq in 3 tRCC cell lines. We collected 30 plasma samples from metastatic patients with tRCC, 12 with ccRCC and 9 healthy individuals. Ultra low pass whole genome sequencing (ulpWGS) was performed to infer ctDNA fraction (TF), and cfMeDIP-seq, H3K4me3/H3K27ac cfChIP-seq for epigenomic profiling. Signal at tRCC-specific regions derived from cell lines profiling was aggregated for each mark and normalized to common active REs/methylated regions and used to discriminate tRCC vs. ccRCC vs. healthy plasma. Classification performance was evaluated using the area under the receiver operating characteristic (AUROC) curve. Overall 10/30 tRCC and 5/12 ccRCC samples had >3% TF by ulpWGS. H3K4me3 cfChIP-seq signal was significantly higher in all tRCC samples compared to healthy plasma (AUROC=1) at tRCC-specific REs and TFE3 TFBS. Furthermore, H3K27ac and H3K4me3 cfChIP-seq signal in plasma were also significantly higher at tRCC-specific REs and TFE3 TFBS in tRCC samples compared to ccRCC (AUROC=0.86-0.89). A cfDNA-based classifier combining the four scores distinguished tRCC from ccRCC with an AUROC=0.94. MeDIP-seq could not discriminate between tRCC and ccRCC. Moreover, three patients had multiple time points, and the integrated cfChIP H3K4me3/H3K27Ac score was able to accurately monitor the evolution of the disease. In conclusion, although a majority of tRCC plasma samples profiled had TF <3%, all could be distinguished from healthy samples on the basis of cfChIP. The combined score of H3K27ac and H3K4me3 cfChIP-Seq was also discriminatory for tRCC vs. ccRCC. Epigenomic profiling of cfDNA appears as a powerful tool for both detection of tRCC and discrimination from ccRCC/healthy plasma, with potential implications for diagnosis, disease monitoring and guiding therapy.
 Citation Format: Simon Garinet, Karl Semaan, John Caniff, Noa Phillips, Jacob Berchuck, Jiao Li, Kevin Lyons, Ananthan Sadagopan, Brad Fortunato, Mary Lee, Jack Horst, Rachel Trowbridge, Ji-Heui Seo, Matthew Freedman, Toni Choueiri, Sylvan Baca, Srinivas Viswanathan. Detection and monitoring of translocation renal cell carcinoma via epigenomic profiling of cell-free DNA [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a06aa07833837aaa2b73e0767951c466dd68ebe8" target='_blank'>
              Abstract B035: Detection and monitoring of translocation renal cell carcinoma via epigenomic profiling of cell-free DNA
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, John Caniff, Noa Phillips, Jacob E. Berchuck, Jiao Li, Kevin Lyons, Ananthan Sadagopan, Brad Fortunato, Mary Lee, Jack Horst, Rachel Trowbridge, J-H. Seo, M. Freedman, T. Choueiri, Sylvan Baca, Srinivas R Viswanathan
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia-telangiectasia-mutated) and ATR (Ataxia-telangiectasia-mutated and Rad3-related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure to UV-B, UV-C and protons induced the microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana.
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Insertion sequences (ISs) are mobile pieces of DNA that are widespread in bacterial genomes. IS movements typically involve (i) excision of the IS element, (ii) cutting of target site DNA, and (iii) IS element insertion. This process generates a new copy of the IS element and a short duplication at the target site. It has been noted that, for some extant IS copies, no target site duplications (TSDs) are readily identifiable. TSD absence has been attributed to degeneration of the TSD after the insertion event, recombination between identical ISs, or adjacent deletions. Indeed, the latter two-recombination between ISs and adjacent deletions-are frequent causes for the absence of TSDs, which we demonstrate here in an analysis of genome sequence data from the Lenski long-term evolution experiment. Furthermore, we propose that some IS movements-namely, those that occur in association with large-scale genomic rearrangements-do not generate TSDs, and occur without evidence for recombination between ISs or adjacent deletions. In support of this hypothesis, we provide two direct, empirical observations of such IS transposition events: an IS5 movement plus a large deletion in Escherichia coli C, and an IS481 movement occurring with a large duplication in Pseudomonas fluorescens SBW25. Although unlikely, it is possible that the observed deletion and associated IS movement occurred in two successive events in one overnight culture. However, an IS at the center of a large-scale duplication is not readily explained, suggesting that IS element activity may promote both large-scale deletions and duplications.


IMPORTANCE
Insertion sequences are the most common mobile genetic elements found in bacterial genomes, and hence they significantly impact bacterial evolution. We observe insertion sequence movement at the center of large-scale deletions and duplications that occurred during laboratory evolution experiments with Escherichia coli and Pseudomonas fluorescens, involving three distinct types of transposase. We raise the possibility that the transposase does not mediate DNA cleavage but instead inserts into existing DNA breaks. Our research highlights the importance of insertion sequences for the generation of large-scale genomic rearrangements and raises questions concerning the mechanistic basis of these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00001ae08c5c01e65cbd4f4a4b234bdd427cfa24" target='_blank'>
              A more significant role for insertion sequences in large-scale rearrangements in bacterial genomes.
              </a>
            </td>
          <td>
            Wing Y. Ngan, Lavisha Parab, Frederic Bertels, J. Gallie
          </td>
          <td>2024-12-05</td>
          <td>mBio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) is a retrovirus responsible for adult T-cell leukemia/lymphoma (ATLL) and HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive neurodegenerative disease. Regulation of viral gene expression plays a key role in viral persistence and pathogenesis. However, the molecular mechanisms underlying this fine-tuned regulation remain poorly understood. Little is known regarding RNA chemical modifications of HTLV-1 RNA and how these affect viral biology and disease development. Posttranscriptional chemical modification of RNA is common in eukaryotes, with N6-methyladenosine (m6A) being the most prevalent. In this study, we investigated the role of m6A RNA modifications on HTLV-1 gene expression. Using MeRIP-Seq, we mapped sites of m6A modification to the 3’ end of the viral genome. We found HTLV-1 RNA, as well as viral oncogene transcripts tax and hbz, contained m6A modifications. m6A-depletion in HTLV-1-transformed cells decreased sense-derived viral genes (Tax, Gag, Env) and increased antisense-derived Hbz expression. Tax and hbz transcripts were bound by reader proteins YTHDF1 and YTHDC1 in a panel of HTLV-1 T-cell lines. Using expression vectors and shRNA-mediated knockdown, we found YTHDF1 had opposing effects on viral gene expression – decreasing sense-derived viral genes and increasing antisense-derived Hbz. Upon further study, the YTHDF1 effects on tax abundance were dependent on tax m6A deposition. The nuclear m6A reader protein YTHDC1 affected the abundance of both sense- and antisense-derived viral transcripts and specifically enhanced the nuclear export of tax transcript. Collectively, our results demonstrate global m6A levels and m6A reader proteins YTHDF1 and YTHDC1 regulate HTLV-1 gene expression. Importance Human T-cell leukemia virus type 1 (HTLV-1) persistence and pathogenesis are controlled through tight regulation of viral gene expression. The fate of RNA can be controlled by epigenetic modifications which impact gene expression without altering the DNA sequence. Our study details the impact of N6-methyladenosine (m6A) RNA chemical modifications on HTLV-1 gene expression. We found reductions in global m6A levels affected viral gene expression, decreasing Tax and other sense-derived viral genes, while increasing the antisense-derived Hbz. Our results suggest the oncogenic viral transcripts, tax and hbz, are m6A-modified in cells. We found these viral RNA modifications are interpreted by reader proteins YTHDF1 and YTHDC1, which dictate the fate of the viral RNA. Understanding HTLV-1 RNA chemical modifications offers potential insights into novel therapeutic strategies for HTLV-1-associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6351be4fe50d4418a28feb2032fbd8b3eec3a2ff" target='_blank'>
              YTHDF1 and YTHDC1 m6A reader proteins regulate HTLV-1 tax and hbz activity
              </a>
            </td>
          <td>
            Emily M. King, A. Midkiff, Karsyn McClain, Sanggu Kim, A. Panfil
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), a highly aggressive brain tumor, frequently develops resistance to temozolomide (TMZ), the current standard chemotherapy. Our study investigates the potential of combining TMZ with the poly (ADP-ribose) polymerase inhibitor Olaparib (OLA) to overcome TMZ resistance. Using in vitro models, including U251 cell lines and patient-derived GBM primary cultures, we demonstrate that OLA enhances TMZ efficacy by disrupting base excision repair and potentiating DNA damage-induced cytotoxicity. A CRISPR knockout screen identified DNA mismatch repair (MMR) gene deficiencies as key drivers of TMZ resistance, which OLA effectively counteracts. Co-treatment with TMZ and OLA significantly reduced tumor cell viability, even in MMR-deficient and MGMT-expressing contexts, suggesting a synergistic mechanism. Gene expression analysis revealed that the combination therapy impacts cell cycle regulation, stress responses, and extracellular matrix integrity, leading to mitotic catastrophe and apoptosis. These findings propose the TMZ-OLA combination as a promising therapeutic strategy for overcoming chemoresistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edc2d11f92b839420cb397da78a6ec7ab2aa82c" target='_blank'>
              PARP inhibitor counteracts Temozolomide Resistance in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Edouard Samarut, Tabourel Gaston, Briand Joséphine, Landais Yuna, Gratas Catherine, Cartron Pierre-François, Vallette François, Oliver Lisa, Sérandour A. Aurélien
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c1bfcf6aa3fe5a68e31d2b5f792bae1437a5f6" target='_blank'>
              Allium chromosome evolution and DNA sequence localization.
              </a>
            </td>
          <td>
            Rafiq Ahmad, Paulina Tomaszewska, M. K. N. Shah, Nadeem Khan, D. Ibrar, Naeem Akhtar, J. S. Heslop-Harrison
          </td>
          <td>2024-12-26</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A central goal of cancer genomics is to identify, in each patient, all the cancer-driving mutations. Among them, point mutations are referred to as cancer-driving nucleotides (CDNs), which recur in cancers. The companion study shows that the probability of i recurrent hits in n patients would decrease exponentially with i; hence, any mutation with i ≥ 3 hits in The Cancer Genome Atlas (TCGA) database is a high-probability CDN. This study characterizes the 50–150 CDNs identifiable for each cancer type of TCGA (while anticipating 10 times more undiscovered ones) as follows: (i) CDNs tend to code for amino acids of divergent chemical properties. (ii) At the genic level, far more CDNs (more than fivefold) fall on noncanonical than canonical cancer-driving genes (CDGs). Most undiscovered CDNs are expected to be on unknown CDGs. (iii) CDNs tend to be more widely shared among cancer types than canonical CDGs, mainly because of the higher resolution at the nucleotide than the whole-gene level. (iv) Most important, among the 50–100 coding region mutations carried by a cancer patient, 5–8 CDNs are expected but only 0–2 CDNs have been identified at present. This low level of identification has hampered functional test and gene-targeted therapy. We show that, by expanding the sample size to 105, most CDNs can be identified. Full CDN identification will then facilitate the design of patient-specific targeting against multiple CDN-harboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb003947f43738ae589b3a6538f7d27fb518829" target='_blank'>
              Characterization of cancer-driving nucleotides (CDNs) across genes, cancer types, and patients
              </a>
            </td>
          <td>
            Lingjie Zhang, Tong Deng, Zhongqi Liufu, Xiangnyu Chen, Shijie Wu, Xueyu Liu, Changhao Shi, Bingjie Chen, Zheng Hu, Qichun Cai, Chenli Liu, Mengfeng Li, Miles E. Tracy, Xuemei Lu, Chung-I Wu, Hai-Jun Wen
          </td>
          <td>2024-12-17</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="TNK2 is a ubiquitously expressed nonreceptor-type tyrosine kinase. TNK2 participates in tumorigenesis, and TNK2 activation has been found in various cancers; therefore, TNK2 is a promising target for cancer chemotherapy. While the TNK2 inhibitor XMD16-5 is highly selective, it inhibits cytokinesis at higher concentrations by targeting Aurora B kinase, a key enzyme for cell division. Cytokinesis failure frequently generates polyploid cells, and the surviving polyploid cells risk leading to cancer development and malignant progression via chromosome instability. In this study, to investigate the possibility that (R)-9bMS, a TNK2 inhibitor structurally related to XMD16-5, drives malignant progression by inducing abnormal cell division, we examined its effects on cell division, Aurora B autophosphorylation, and colony formation. Cell count results showed a reduction in the number of A431, HeLa S3, HCT116, and MCF7 cells upon TNK2 inhibitor treatment. Microscopic observation indicated the formation of multinucleated and nucleus-enlarged cells. An increase in DNA content was confirmed with flow cytometry, which was underpinned by an increased number of centrosomes. Time-lapse imaging revealed mitotic failure, such as mitotic slippage and cytokinesis failure, as a cause of polyploidization. Of note, TNK2 knockdown significantly increased multinucleated cells, but the effect was quite weak, suggesting that TNK2 inhibition may only partially contribute to mitotic failure and polyploidization. Expectedly, Aurora B phosphorylation was reduced by (R)-9bMS like XMD16-5, but not by TNK2 knockdown. Collectively, TNK2 inhibitors (R)-9bMS and XMD16-5 induce polyploidization via mitotic failure caused by the inhibition of Aurora B kinase rather than TNK2. Notably, (R)-9bMS treatment promoted anchorage-independent colony formation, a hallmark of cancer. Our findings suggest that (R)-9bMS at a high concentration risks promoting cancer development or malignant progression. Therefore, caution should be used when using TNK2 inhibitors for cancers where TNK2 activation is not the transforming mutation and higher concentrations of TNK2 inhibitors are required to slow proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83021c8768a54cea8919a992b5bb8d7150bf2fc1" target='_blank'>
              TNK2 Inhibitor (R)-9bMS Causes Polyploidization Through Mitotic Failure by Targeting Aurora B.
              </a>
            </td>
          <td>
            Mayu Murata, Hiroki Kuwajima, Junna Tanaka, Nanami Hasegawa, Ryuzaburo Yuki, Youhei Saito, Yuji Nakayama
          </td>
          <td>2024-12-01</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aede480e99c41ff823fe07aee10fb1b86fa8b" target='_blank'>
              DNA methylation-based analysis reveals accelerated epigenetic aging in giant cell-enriched adult-type glioblastoma
              </a>
            </td>
          <td>
            P. Cakmak, Philipp Jurmeister, I. Divé, P. Zeiner, Joachim P. Steinbach, Tim R Fenton, K. H. Plate, M. Czabanka, Patrick Harter, Katharina J. Weber
          </td>
          <td>2024-12-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62a212b47508047875bfe6549eb4ee11cd53c6" target='_blank'>
              Survey for Activating Oncogenic Mutation Variants in Metazoan Germline Genes.
              </a>
            </td>
          <td>
            Karl E Krueger
          </td>
          <td>2024-11-26</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 DNA methylation is a fundamental epigenetic mark that governs gene expression and chromatin organization, thus providing a window into cellular identity and developmental processes. Current datasets typically include only a fraction of methylation sites and are often based either on cell lines that underwent massive changes in culture or on tissues containing unspecified mixtures of cells. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing, allowing fragment-level analysis across thousands of unique markers for 39 cell types sorted from 205 healthy tissue samples. Replicates of the same cell type are more than 99.5% identical, demonstrating the robustness of cell identity programmes to environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny and identifies methylation patterns retained since embryonic development. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely hypermethylated loci are rare and are enriched for CpG islands, Polycomb targets and CTCF binding sites, suggesting a new role in shaping cell-type-specific chromatin looping. The atlas provides an essential resource for study of gene regulation and disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
 Citation Format: Tommy Kaplan. A DNA methylation atlas of normal human cell types [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr IA025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9c66abd8a0c90b855e4304f66a149ab9fbef96" target='_blank'>
              Abstract IA025: A DNA methylation atlas of normal human cell types
              </a>
            </td>
          <td>
            Tommy Kaplan
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pseudouridine (Ψ) is a post-transcriptional modifier of RNA, often referred to as the ‘fifth nucleotide’ owing to its regulatory role in various biological functions as well as because of its significant involvement in the pathogenesis of human cancer. In recent years, research has revealed various Ψ modifications in different RNA types, including messenger RNA, transfer RNA, ribosomal RNA, small nuclear RNA, and long noncoding RNA. Pseudouridylation can significantly alter RNA structure and thermodynamic stability, as the Ψ-adenine (A) base pair is more stable than the typical uridine (U)-A base pair is due to its structural similarity to adenine. Studies have linked Ψ expression to the development and progression of several digestive system cancers, such as liver cancer and colorectal cancer, and nondigestive system cancers, such as breast cancer, non-small cell lung cancer, prostate cancer, glioblastoma, ovarian cancer, oral squamous cell carcinoma, and pituitary cancer. The present review briefly outlines the chemical structure, synthesis, and regulatory mechanisms of Ψ. This review summarizes the effects of pseudouridylation on various substrates of RNA and briefly discusses methods for detecting Ψ. Last, it focuses on how RNA pseudouridylation influences different cancers, emphasizing the search for novel approaches to cancer diagnosis, treatment, and prognosis through Ψ modification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9466c719e8cf6ff18a146061c808211210df53e3" target='_blank'>
              Advancements in pseudouridine modifying enzyme and cancer
              </a>
            </td>
          <td>
            Kaijie Liu, Shujun Zhang, Yafeng Liu, Xinjun Hu, Xinyu Gu
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2100497dbb37ac929dfe5b9230f9192ece871ee3" target='_blank'>
              Inter-chromosomal contacts demarcate genome topology along a spatial gradient
              </a>
            </td>
          <td>
            Milad Mokhtaridoost, Jordan J. Chalmers, Marzieh Soleimanpoor, Brandon J. McMurray, Daniella F. Lato, Son C Nguyen, Viktoria Musienko, Joshua O. Nash, S. Espeso-Gil, Sameen Ahmed, Kate Delfosse, Jared W. L. Browning, A. R. Barutcu, Michael D. Wilson, Thomas Liehr, Adam Shlien, Samin Aref, Eric F Joyce, A. Weise, Philipp G. Maass
          </td>
          <td>2024-11-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687f46374d8d1b1ff94fa27535e6920c3b5b0c73" target='_blank'>
              Myc 9aaTAD activation domain binds to mediator of transcription with superior high affinity
              </a>
            </td>
          <td>
            Andrea Knight, Josef Houser, Tomas Otasevic, V. Juráň, V. Vybíhal, M. Smrčka, Martin Piskacek
          </td>
          <td>2024-11-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8117cfd933320eeb9a62cf47bdb749f3f59bcf32" target='_blank'>
              Normal breast tissues harbour rare populations of aneuploid epithelial cells.
              </a>
            </td>
          <td>
            Yiyun Lin, Junke Wang, Kaile Wang, S. Bai, A. Thennavan, Runmin Wei, Yun Yan, Jianzhuo Li, Heba Elgamal, Emi Sei, Anna K. Casasent, Mitchell Rao, Chenling Tang, A. Multani, Jin Ma, Jessica Montalvan, C. Nagi, Sebastian Winocour, Bora Lim, Alastair Thompson, Nicholas Navin
          </td>
          <td>2024-11-20</td>
          <td>Nature</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be789fb687d3f78efa3b5343344eaabbca303d35" target='_blank'>
              Chromatin licensing and DNA replication factor 1 as a potential prognostic and diagnostic biomarker for gastric cancer
              </a>
            </td>
          <td>
            Sijia Wang, Min Cui, Na Zhou, Meichen Tong, Shuhua Wu, Hong Li, Zhang Cao
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA replication in humans requires precise regulation to ensure accurate genome duplication and maintain genome integrity. A key indicator of this regulation is replication timing, which reflects the interplay between origin firing and fork dynamics. We present a high-resolution (1-kilobase) mathematical model that maps firing rate distributions to replication timing profiles across various cell lines, validated using Repliseq data. The model effectively captures genome-wide replication patterns while identifying local discrepancies. Notably, regions where the model and data diverge often overlap with fragile sites and large genes, highlighting the influence of genomic architecture on replication dynamics. Conversely, regions of high concordance are associated with open chromatin and active promoters, where elevated firing rates facilitate timely fork progression and reduce replication stress. By establishing these correlations, our model provides a valuable framework for exploring the structural interplay between replication timing, transcription, and chromatin organisation, offering new insights into the mechanisms underlying replication stress and its implications for genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0e43f661ffe2f761f8d760908a49fb8c512704" target='_blank'>
              DNA replication timing reveals genome-wide features of transcription and fragility
              </a>
            </td>
          <td>
            Francis X Berkemeier, Peter R. Cook, M. Boemo
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Targeting DNA damage response (DDR) pathways represents one of the principal approaches in cancer therapy. However, defects in DDR mechanisms, exhibited by various tumors, can also promote tumor progression and resistance to therapy, negatively impacting patient survival. Therefore, identifying new molecules from natural extracts could provide a powerful source of novel compounds for cancer treatment strategies. In this context, we investigated the role of oleanolic acid (OA), identified in fermented Aglianico red grape pomace, in modulating the DDR in response to camptothecin (CPT), an inhibitor of topoisomerase I. Specifically, we found that OA can influence the choice of DNA repair pathway upon CPT treatment, shifting the repair process from homologous recombination gene conversion to single-strand annealing. Moreover, our data demonstrate that combining sub-lethal concentrations of OA with CPT enhances the efficacy of topoisomerase I inhibition compared to CPT alone. Overall, these findings highlight a new role for OA in the DDR, leading to a more mutagenic DNA repair pathway and increased sensitivity in the HeLa cancer cell line.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba28716fad27e1778281b97439ea3f957928277" target='_blank'>
              Oleanolic Acid Modulates DNA Damage Response to Camptothecin Increasing Cancer Cell Death
              </a>
            </td>
          <td>
            Giulio Mazzarotti, Maria Cuomo, Maria Carmen Ragosta, Andrea Russo, Margherita D’Angelo, Annamaria Medugno, Giuseppe Maria Napolitano, C. A. Iannuzzi, I. Forte, Rosa Camerlingo, Sharon Burk, F. Errichiello, Luigi Frusciante, M. Forino, M. R. Campitiello, M. de Laurentiis, A. Giordano, Luigi Alfano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e87243dcc8a034ecc3e29fa0ee06621055ecda9" target='_blank'>
              Engrailed 2 facilitates progression of triple-negative and HER2-enriched breast cancer by binding to enhancer region of Tenascin-C
              </a>
            </td>
          <td>
            Dandan Chen, Rongping Yin
          </td>
          <td>2024-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This review explores articles concerning the experimental research cycle on genome instability in cell populations of highly malignant recurrent organotropic rhabdomyosarcoma RA- 2 in rats. Clonal analysis and cloning were pivotal components of this research, which relies on the frequency of cells with micronuclei and internuclear bridges to gauge the intensity of chromothripsis and break-fusion-bridge cycles. The efficacy of cloning, determined by these indicators, stemmed from the deliberate isolation of tumor stem cells, yielding clones in which chromothripsis activity and breakage-fusion-bridge cycles were sustained. Notably, it is plausible that the stem cells themselves, progenitors of these clones, harbor dicentric chromosomes and chromosomal fragments, contributing to the formation of "fatal micronuclei" in their karyotype. Cloning based on bridges and micronuclei has proven effective up to a certain threshold (15%-18%), reaffirming the predicted reproductive extinction of malignant cell populations under mutational pressure and genome chaos, as posited by the genetic theory of cell populations. Furthermore, this review highlights the potential of ergodic cancer therapy as a novel therapeutic strategy. Ergodic therapy offers promising prospects for late-stage and terminal malignant tumors, where conventional treatments may fall short due to advanced progression. Furthermore, by "enhancing chromothripsis" through the induction of additional micronuclei and bridges, ergodic cancer therapy seeks to increase genome chaos to a critical threshold, potentially halting malignant progression. This innovative approach presents opportunities to explore new drugs and targets for chromothripsis-based oncotherapy, addressing the pressing need for effective treatments in advanced stages of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ee83cc51235034563b6307f2bcb528ce64b269" target='_blank'>
              Perspectives on Ergodic Cancer Therapy Derived from Cloning Genome Chaos via In Vivo Rhabdomyosarcoma RA-2 Models: a Narrative Review.
              </a>
            </td>
          <td>
            Sergey Shityakov, Natalia Lubinets, Viacheslav Kravtsov
          </td>
          <td>2024-11-08</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Aneuploidy, characterized by imbalanced chromosome numbers, is a hallmark of cancer and is strongly correlated with lethal progression in prostate cancer. Our recent work identified that gains of chromosome 8q, the most frequently gained chromosome in aneuploidy prostate cancers, drive cancer progression, partially through the cohesin RAD21 gene located on the chr8q24 region. We demonstrated that increased RAD21 expression accelerates lethal progression and oncogenesis in both prostate cancer and Ewing sarcoma by alleviating oncogenic stress during early-stage oncogenesis in both cell line and organoid models. Therefore, we hypothesize that elevated RAD21 levels could enable cancer cells to evade immune surveillance by mitigating oncogenic stress and DNA damage. In this study, we developed isogenic prostate and breast cancer cell culture models with varying levels of RAD21. Using a natural killer (NK) cell co-culture assay, we observed that cells with elevated RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited increased survivorship when co-cultured with NK cells. These findings align with publicly available data showing that tumors with high RAD21 levels have reduced NK cell infiltration in prostate cancer. Our results suggest that increased RAD21 expression, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Understanding the role of RAD21 in immune evasion could reveal novel therapeutic strategies targeting immune surveillance mechanisms in prostate cancer.
 Citation Format: Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, Andrew Elliott, Nicholas A. Zorko, Justin H. Hwang, Xiaofeng A. Su. Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e04861d70a2d0fe00d7781f4f4827f407b579d8" target='_blank'>
              Abstract B021: Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers
              </a>
            </td>
          <td>
            Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, A. Elliott, Nicholas A Zorko, Justin H. Hwang
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Gene conversion is a process in which genetic material from a donor sequence is unidirectionally copied to an acceptor sequence during the homologous recombination repair of a DNA double-strand break. Although gene conversion has been widely studied in the context of meiosis, hereditary diseases, and cancer development, gene conversion between parental homologs in the zygotes remains controversial. Here, we developed a method to detect interparental gene conversions by focusing on Mendelian errors and identified gene conversion events in one out of every 21.8 births. Some of these events were observed in genetic regions, potentially affecting offspring phenotypes. Interparental gene conversion leads to the offspring inheriting two identical alleles from one parent, resulting in a loss of heterozygosity. Our findings suggest that naturally occurring interparental gene conversions may provide a novel mechanism for the development of certain genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ddbca341c42c36a2e20339125b0cf1e5d09505" target='_blank'>
              Interparental Gene Conversion in General Population: A Novel Mechanism For Loss of Heterozygosity
              </a>
            </td>
          <td>
            Jumpei Toratani, Masahito Tachibana, Junichi Sugawara, Atsushi Sugawara, Takeki Sato, Yuri Takahashi, Hiroaki Hiraga, Emi Yokoyama, Zen Watanabe, Masatoshi Saito, Nobuo Yaegashi, Gen Tamiya, J. Takayama
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Rearrangements between genes can yield neomorphic fusions that drive oncogenesis. Fusion oncogenes are made up of fractional segments of the partner genes that comprise them, with each partner potentially contributing some of its own function to the nascent fusion oncoprotein. Clinically, fusion oncoproteins driving one diagnostic entity are typically clustered into a single molecular subset and are often treated a similar fashion. However, knowledge of where specific fusion breakpoints occur in partner genes, and the resulting retention of functional domains in the fusion, is an important determinant of fusion oncoprotein activity and may differ between patients. This study investigates these phenomena through the example of capicua (CIC)::double homeobox 4 (DUX4), a fusion between transcriptional repressor CIC and DUX4 genes, which drives an aggressive subset of undifferentiated round cell sarcoma. Using a harmonized dataset of more than 100 patient fusion breakpoints from the literature, we show that most bona fide CIC::DUX4 fusions retain the C1 domain, which is known to contribute to DNA binding by wild-type CIC. Mechanistically, deletion or mutation of the C1 domain reduces, but does not eliminate, the activation of CIC target genes by CIC::DUX4. We also find that expression of C1-deleted CIC::DUX4 is capable of exerting intermediate transformation-related phenotypes compared with those imparted by full-length CIC::DUX4 but was not sufficient for tumorigenesis in a subcutaneous mouse model. In summary, our results suggest a supercharging role for the C1 domain in the activity of CIC::DUX4. Significance: We show in mammalian settings that the capicua C1 functional domain is a supercharger for CIC::DUX4, a poorly studied fusion oncoprotein which drives a rare sarcoma with dismal outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33a18cf240663d7acde75f57d78717f3cd1075" target='_blank'>
              The Capicua C1 Domain Is Required for Full Activity of the CIC::DUX4 Fusion Oncoprotein
              </a>
            </td>
          <td>
            Cuyler Luck, Kyle A. Jacobs, Ross A. Okimoto
          </td>
          <td>2024-11-12</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary KMT2A partial tandem duplication (PTD) involves intragenic duplications within the KMT2A gene and has been associated with acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS). KMT2A PTD cannot be detected by conventional karyotyping or fluorescence in situ hybridization (FISH). In this study, we used optical genome mapping (OGM) to analyze the cytogenomic alterations in 1277 hematolymphoid neoplasms and identified KMT2A PTD exclusively in patients with myeloid neoplasms, including 35 (7%) with AML, 5 (2.2%) with MDS, and 5 (7.2%) with chronic myelomonocytic leukemia (CMML). Neoplasms with KMT2A PTD frequently exhibit a normal or non-complex karyotype and consistently harbor gene mutations involving epigenetic regulators, the FLT3/RAS signaling pathway, transcription factors, and spliceosome genes. Patients with KMT2A PTD are generally resistant to conventional chemotherapy, with the exception of those with de novo AML, which demonstrates a high remission rate. Patients with KMT2A PTD positive secondary AML and MDS had a poor outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce3f21aa3468dded682d3ad1d494e6eb3d9bfdb" target='_blank'>
              Detection of KMT2A Partial Tandem Duplication by Optical Genome Mapping in Myeloid Neoplasms: Associated Cytogenetics, Gene Mutations, Treatment Responses, and Patient Outcomes
              </a>
            </td>
          <td>
            Qing Wei, Shimin Hu, Jie Xu, Sanam Loghavi, Naval G Daver, G. Toruner, Wei Wang, L. J. Medeiros, Guilin Tang
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The treatment of many solid tumors presents significant challenges with chemotherapy, radiation therapy, and the limited effectiveness of immunotherapy. Targeted therapy offers a promising approach, yet the lack of validated targets limits its applicability across the spectrum of solid tumors. Our past studies have revealed that an R-loop and mRNA-dependent DNA repair (RDDR) pathway, induced by damage at the transcribed regions of the genome, contributes to cell survival and drug resistance in cancer cells. This is particularly due to the high levels of transcription and DNA damage in these cells. We identified the RNA methyltransferase TRDMT1 as the primary modifier of mRNA methyl-5-cytosine in the RDDR pathway. Overcoming technical difficulties in monitoring RNA modifications, we developed the first TRDMT1 inhibitor (TRDMT1i) using in-house built cell-based and in vitro assays. We have identified highly potent and specific lead compounds that significantly reduce TRDMT1 activity at nanomolar concentrations. Our current lead TRDMT1 inhibitors meets most of the criteria that are required for optimal lead. Initial assessments involving the screening of normal and cancer cell lines, xenograft models, and patient specimens indicate that TRDMT1i sensitizes tumors with genomic instability when used as monotherapy. We aim to develop TRDMT1i as an investigational new drug (IND) for future clinical trials, initially targeting ovarian cancer as a monotherapy or in combination therapy for first-line maintenance and systematic therapy. Our goal is to pioneer the discovery of the RDDR pathway, leading to the development of innovative platforms and next-generation medications to revolutionize the targeting of mRNA modifications in cancer treatment.
 Citation Format: Li Lan. Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5c058059942f92471fbbee99fbac2c5e418612" target='_blank'>
              Abstract PR001: Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy
              </a>
            </td>
          <td>
            Li Lan
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The status of engineered mini-chromosomes/artificial chromosomes/synthetic chromosomes in plants is summarized. Their promise is that they provide a means to accumulate foreign genes on an independent entity other than the normal chromosomes, which would facilitate stacking of novel traits in a way that would not be linked to endogenous genes and that would facilitate transfer between lines. Centromeres in plants are epigenetic, and therefore the isolation of DNA underlying centromeres and reintroduction into plant cells will not establish a functional kinetochore, which obviates this approach for in vitro assembly of plant artificial chromosomes. This issue was bypassed by using telomere-mediated chromosomal truncation to produce mini-chromosomes with little more than an endogenous centromere that could in turn be used as a foundation to build synthetic chromosomes. Site-specific recombinases and various iterations of CRISPR-Cas9 editing provide many tools for the development and re-engineering of synthetic chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e680445acc15f9ceab9c4fa429b615d08a559e2d" target='_blank'>
              Synthetic minichromosomes in plants: past, present, and promise.
              </a>
            </td>
          <td>
            J. Birchler, Jacob A. Kelly, Jasnoor Singh, Hua Liu, Zhengzhi Zhang, Si Nian Char, Malika Sharma, Hua Yang, Patrice S. Albert, Bing Yang
          </td>
          <td>2024-11-15</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract H-DNA is an intramolecular DNA triplex formed by homopurine/homopyrimidine mirror repeats. Since its discovery, the field has advanced from characterizing the structure in vitro to discovering its existence and role in vivo. H-DNA interacts with cellular machinery in unique ways, stalling DNA and RNA polymerases and causing genome instability. The foundational S1 nuclease and chemical probing technologies originally used to show H-DNA formation have been updated and combined with genome-wide sequencing methods for large-scale mapping of secondary structures. There is evidence for triplex H-DNA’s role in polycystic kidney disease (PKD), cancer, and numerous repeat expansion diseases (REDs). In PKD, an H-DNA forming repeat region within the PKD1 gene stalls DNA replication and induces fragility. H-DNA-forming repeats in various genes have a role in cancer; the most well-studied examples involve H-DNA-mediated fragility causing translocations in multiple lymphomas. Lastly, H-DNA-forming repeats have been implicated in four REDs: Friedreich’s ataxia, GAA-FGF14-related ataxia, X-linked Dystonia Parkinsonism, and cerebellar ataxia, neuropathy and vestibular areflexia syndrome. In this review, we summarize H-DNA’s discovery and characterization, evidence for its existence and function in vivo, and the field’s current knowledge on its role in physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d258222649c5f8f00b18b553e105c9ceb00bafd" target='_blank'>
              Triplex H-DNA structure: the long and winding road from the discovery to its role in human disease
              </a>
            </td>
          <td>
            Julia A. Hisey, Chiara Masnovo, S. Mirkin
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The phosphorylation of histone H2A.X into γH2A.X is a crucial early event in the DNA damage response, marking DNA damage sites and initiating repair processes. While ATM kinase is traditionally recognized as the primary mediator of H2A.X phosphorylation, our study identifies serine/threonine kinase 39 (STK39) as a novel enhancer of this critical signaling pathway. We demonstrate that after DNA damage, STK39 undergoes phosphorylation by the ATM kinase, facilitating its interaction with the Mre11-Rad50-Nbs1 complex and subsequent recruitment to chromatin. This recruitment enables STK39 to further phosphorylate H2A.X, thus amplifying γH2A.X production and promoting homologous recombination repair. Notably, we observe a significant upregulation of STK39 in pancreatic adenocarcinoma (PAAD) tissues, correlating with heightened resistance to PARPi therapy. Furthermore, we demonstrate the synergistic efficacy of combining STK39 inhibition with PARP inhibitors in suppressing and reversing PAAD growth. This study not only provides new insights into the molecular dynamics of H2A.X phosphorylation but also highlights the therapeutic potential of targeting STK39 to enhance PARPi sensitivity in PAAD (created with BioRender).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b5527588649efe249bff697a10bc6cd9dfa473" target='_blank'>
              STK39-mediated amplification of γ-H2A.X promotes homologous recombination and contributes to PARP inhibitor resistance
              </a>
            </td>
          <td>
            Yi Xu, Changying Li, Huan Yin, S. Nowsheen, Xin Xu, Wenjuan Kang, Xin Liu, Lifeng Chen, Zhenkun Lou, Junlin Yi, Min Deng
          </td>
          <td>2024-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37b95c1e16c6aa07dc838b1c35715a1b58b94d" target='_blank'>
              Transcription factors ASCL1 and OLIG2 drive glioblastoma initiation and co-regulate tumor cell types and migration
              </a>
            </td>
          <td>
            Bianca L Myers, Kathryn J. Brayer, Luis E Paez-Beltran, Estrella Villicana, Matthew S. Keith, Hideaki Suzuki, Jessie Newville, Rebekka H Anderson, Yunee Lo, Conner M Mertz, R. Kollipara, Mark D. Borromeo, Q. R. Lu, R. Bachoo, Jane E Johnson, T. Vue
          </td>
          <td>2024-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Cancer development involves a complex interplay between genetic and epigenetic factors, with emerging evidence highlighting the pivotal role of competitive endogenous RNA (ceRNA) networks in regulating gene expression. However, the influence of ceRNA networks by aberrant DNA methylation remains incompletely understood. In our study, we proposed DMceNet, a computational method to characterize the effects of DNA methylation on ceRNA regulatory mechanisms and apply it across eight prevalent cancers. By integrating methylation and transcriptomic data, we constructed methylation-driven ceRNA networks and identified a dominant role of lncRNAs within these networks in two key ways: (i) 17 cancer-shared differential methylation lncRNAs (DMlncs), including PVT1 and CASC2, form a Common Cancer Network (CCN) affecting key pathways such as the G2/M checkpoint, and (ii) 24 cancer-specific DMlncs construct unique ceRNA networks for each cancer type. For instance, in LUAD and STAD, hypomethylation drives DMlncs like PCAT6 and MINCR, disrupting the Wnt signaling pathway and apoptosis. We further investigated the characteristics of these methylation-driven ceRNA networks at the cellular level, revealing how methylation-driven dysregulation varies across distinct cell populations within the tumor microenvironment. Our findings also demonstrate the prognostic potential of cancer-specific ceRNA relationships, highlighting their relevance in predicting patient survival outcomes. This integrated transcriptomic and epigenomic analysis provides new insights into cancer biology and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7483d8802056f68722b0432bf0cd28325b090cf3" target='_blank'>
              Constructing methylation-driven ceRNA networks unveil tumor heterogeneity and predict patient prognosis.
              </a>
            </td>
          <td>
            Xinyu Li, Chuo Peng, Hongyu Liu, Mingjie Dong, Shujuan Li, Weixin Liang, Xia Li, Jing Bai
          </td>
          <td>2024-11-27</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The autonomous and active Long-Interspersed Element-1 (LINE-1, L1) and the non-autonomous Alu retrotransposon elements, contributing to 30% of the human genome, are the most abundant repeated sequences. With more than 90% of their sequences being methylated in normal cells, these elements undeniably contribute to the global DNA methylation level and constitute a major part of circulating-cell-free DNA (cfDNA). So far, the hypomethylation status of LINE-1 and Alu in cellular and extracellular DNA has long been considered a prevailing hallmark of ageing-related diseases and cancer. This study demonstrated that errors in LINE-1 and Alu methylation level measurements were caused by an excessive input quantity of genomic DNA used for bisulfite conversion. Using the minuscule DNA amount of 0.5 ng, much less than what has been used and recommended so far (500 ng-2 μg) or 1 μL of cfDNA extracted from 1 mL of blood, we revealed hypermethylation of LINE-1 and Alu in 407 tumour samples of primary breast, colon and lung cancers when compared with the corresponding pair-matched adjacent normal tissue samples (P < 0.05–0.001), and in cfDNA from 296 samples of lung cancers as compared with 477 samples from healthy controls (P < 0.0001). More importantly, LINE-1 hypermethylation in cfDNA is associated with healthy ageing. Our results have not only contributed to the standardized bisulfite-based protocols for DNA methylation assays, particularly in applications on repeated sequences but also provided another perspective for other repetitive sequences whose epigenetic properties may have crucial impacts on genome architecture and human health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af9de458a2650aad970d8197a4d94f0fb4283e9" target='_blank'>
              An appropriate DNA input for bisulfite conversion reveals LINE-1 and Alu hypermethylation in tissues and circulating cell-free DNA from cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Tung The Pham, Than Thi Nguyen, Trang Hien Do, Phuong Thi Thu Luu, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Son Van Ho, Hang Viet Dao, Tong Van Hoang, Lan Thi Thuong Vo
          </td>
          <td>2024-12-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067cac427326fdea9459d93348affd2bc0e1346c" target='_blank'>
              Nucleus-translocated GCLM promotes chemoresistance in colorectal cancer through a moonlighting function
              </a>
            </td>
          <td>
            Jin-Fei Lin, Ze-xian Liu, Dong-Liang Chen, , Fen Cao, Kai Yu, Ting Li, Hai-Yu Mo, Hui Sheng, , Kun Liao, Yi Han, , , Song Gao, Huai-qiang Ju, 
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The remarkable capability of Tardigrade to survive under extreme conditions has been partially attributed to Dsup, an intrinsically disordered, highly positively charged protein. Dsup has been shown to bind to DNA in vitro, a property that has been associated with the capability of Dsup to exhibit stress-protective effects when expressed in mammalian cells. However, DNA binding of Dsup has not been visualized in living cells and expression of Dsup in different cell types was associated with either protective or detrimental effects. In addition, the effect of Dsup expression has not been clearly demonstrated at the organism level. Here we combined molecular dynamics (MD) simulations and fluorescence lifetime imaging microscopy (FLIM)-Förster resonance energy transfer (FRET) to interrogate Dsup-DNA interactions and demonstrated Dsup binding to DNA in living mammalian cells. Furthermore, Dsup expression in both HEK293T cells and yeast enhanced cell survival in the presence of hydrogen peroxide, suggesting that the presence of Dsup allows both mammalian and yeast cells to better cope with oxidative stress conditions. This study provides a better understanding of the property and functional role of Dsup and lays a foundation to explore new approaches to enhance stress resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c6d0a3ae54b0eb2c8b74ea39dd22a5dd18674ac" target='_blank'>
              Tardigrade Dsup: Interactions with DNA and protection of cells from oxidative stress
              </a>
            </td>
          <td>
            Gavin Ni, Hetian Su, Yingqi Zhu, Anshika Dhiman, Huan-Xiang Zhou, Wei Lin, Nan Hao
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant pediatric brain tumor. MB is categorized into four main molecular subgroups: wingless pathway (WNT)-activated, sonic hedgehog pathway (SHH)-activated, group 3 (G3), and group 4 (G4). Although the driver pathways for G3 and G4 MB are unclear, most of these tumors are characterized by c-Myc and MYCN signatures, respectively. Recent studies suggest that long non-coding RNAs (lncRNAs) contribute to medulloblastoma formation and progression. We have identified lncRNA, lnc-HLX-2-7, as an oncogene and a potential therapeutic target in group 3 (G3) medulloblastomas. lnc-HLX-2-7 RNA specifically accumulates in the promoter region of its host coding gene HLX, a sense overlapping gene of lnc-HLX-2-7, which activates HLX expression by recruiting multiple factors, including enhancer elements. RNA sequencing and chromatin immunoprecipitation reveal that HLX binds to and activates the promoters of several oncogenes, including TBX2, LIN9, HOXM1, and MYC. Intravenous treatment with cerium oxide nanoparticle–coated antisense oligonucleotides targeting lnc-HLX-2-7 (CNP-ASO-lnc-HLX-2-7) and lipid nanoparticle-coated HLX (LNP-si-HLX) inhibits tumor growth by 40-50% in an intracranial medulloblastoma xenograft mouse model (p<0.01). Combining nanoparticle-coated lnc-HLX-2-7 and HLX with standard-of-care cisplatin further inhibits tumor growth and significantly prolongs mouse survival compared with monotherapy (p<0.01). Thus, the lnc-HLX-2-7–HLX–MYC axis is important for regulating G3 medulloblastoma progression, providing a strong rationale for using lnc-HLX-2-7 and HLX as therapeutic targets for G3 MBs.
 Citation Format: Ranjan R Perera. A therapeutically targetable positive feedback loop between long noncoding RNA-HLX-2-7, HLX, and MYC that promotes childhood cancer group 3 medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b36b54a80e66b87d4edb7a4cc9aa5f5951c66c" target='_blank'>
              Abstract B002: A therapeutically targetable positive feedback loop between long noncoding RNA-HLX-2-7, HLX, and MYC that promotes childhood cancer group 3 medulloblastoma
              </a>
            </td>
          <td>
            Ranjan R Perera
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Precursor B cell acute lymphoblastic leukemia (pre-B-ALL) arises as a result of precursor B cells acquiring driver mutations that lead to arrested differentiation and increased proliferation. Identification of driver mutations and understanding their biological function is critical to understanding pre-B-ALL development and advancing disease treatment. Using a mouse model of pre-B-ALL driven by deletion of genes encoding the related E26-transformation-specific (ETS) transcription factors PU.1 and Spi-B, we performed whole exome sequencing to identify secondary driver mutations. We identified recurrent variants in E26 transformation-specific transcription variant 5 (ETV5) resulting in R392P, V444I, and T505A amino acid changes. We found that the R392P and V444I variants altered the ability of ETV5 to bind to DNA using electrophoretic mobility shift assay. R392P and V444I variants did not activate a Dual-Specificity-Phosphatase 6 (DUSP6) reporter. In contrast, T505A ETV5 could interact with DNA and activate the DUSP6 promoter. To determine biological function, we forced expression of wild type, R392P, V444I, or T505A ETV5 in an interleukin-7-dependent pre-B cell line. Proliferation and apoptosis assays showed that T505A ETV5 conferred a proliferative advantage to pre-B cells. RNA sequencing showed that expression of ETV5 variants significantly altered gene expression in cultured cells. Through gene set enrichment analysis, T505A was suggested to downregulate the p53 pathway and the anti-proliferative protein, B cell translocation gene 2 (encoded by Btg2). In summary, these data suggest that ETV5 mutations play a role in pre-B-ALL by affecting proliferation and cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f29fab27dbb4d851eea7f525bf89bbde3e863a" target='_blank'>
              T505A variant of ETV5 promotes proliferation of precursor B cells in a mouse model of acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Joshua Yi, Michael Wu, Michaela L. Dowling, Allanna C. E. MacKenzie, James Iansavitchous, R. DeKoter
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Current development in stool- and cell-free DNA (cfDNA)-based technologies have demonstrated promising potentials in colorectal cancer (CRC) diagnosis. However, early detection of CRC and advanced adenoma (AA) remains challenging. Micronuclei (MN) are extranuclear bodies containing chromatin segments resulting from errors in DNA repair. Elevated levels of MN+ erythrocytes have been studied in genotoxicity testing and cancer diagnosis. We further performed whole-genome sequencing on purified and isolated micronuclei DNA (MN-DNA) from erythrocytes in peripheral blood using the approach (WO2021/228246 A1) we previously developed. By comparing MN-DNA from colorectal cancer (CRC) patients and healthy individuals, we identified a series of tumor-associated MN-DNA features (taMN- DNA) with significant differences in read counts and developed a classification model. Here, we established an independent cohort to clinically validate the CRC model and explore its application in early cancer diagnosis. Methods: We launched a prospective observational case- control clinical trial (NCT05875584), which was used as an independent cohort to validate taMN-DNA features in CRC and to perform a comparative analysis with quantitative fecal immunochemical test (qFIT). A total of 598 samples were enrolled in this study (585 was available, including 299 HDs, 206 AAs, and 80 CRCs). We gathered peripheral blood (1-2ml) from a training cohort of 1226 individuals, a test cohort of 309 individuals and this independent cohort of 598 individuals for the MN-DNA isolation from erythrocytes and whole-genome sequencing. Predictive models were developed using distinctive taMN-DNA features identified by genome-wide analysis from training cohort to differentiate between HD and AA or CRC. Results: The predictive CRC model built on taMN-DNA features achieved an AUC of 93% and the detection of AA achieved an AUC of 82% in this clinical validation cohort. Comparing MN-DNA with qFIT, the overall sensitivity was generally consistent (91.3% vs. 87.5%), with MN-DNA achieving 86.4% sensitivity in early-stage (stage I) CRC, significantly higher than 68.2% in qFIT. Additionally, MN-DNA showed superior sensitivity in detecting AA (71.4% vs. 18.4%), with comparable specificity (90.3% vs. 96.3%). Conclusions: Our results demonstrate that MN-DNA within erythrocytes from 1-2ml peripheral blood enable accurate detection of AA and early-stage CRC. Research sponsor: Timing Biotech.
 Citation Format: Yurong Jiao, Xingyun Yao, Haobo Sun, Chengcheng Liu, Xiangxing Kong, Honghao Liang, Fei Meng, Jun Li, Kefeng Ding, Xiaofei Gao. Evaluating the potential of micronuclei DNA from erythrocytes for early detection of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5ef616e28e97fac15d13e7a4fdf0802ddd0c685" target='_blank'>
              Abstract A045: Evaluating the potential of micronuclei DNA from erythrocytes for early detection of colorectal cancer
              </a>
            </td>
          <td>
            Yurong Jiao, Xingyun Yao, Haobo Sun, Chengcheng Liu, Xiangxing Kong, Honghao Liang, Fei Meng, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Histone methyltransferases (HMTs) and histone demethylases (HDMs) are enzymes that modify histones, the proteins around which DNA is wrapped. These modifications play a crucial role in regulating gene expression by adding or removing methyl groups, which can activate or silence genes. In this review, we explore the roles of HMTs and HDMs in the development of cancer and neurodegenerative diseases, specifically, Alzheimer’s, Parkinson’s, and Huntington’s diseases. Regarding cancer, HMTs and HDMs can drive tumor growth and progression by altering gene expression patterns that regulate cell proliferation and survival. For example, the dysregulation of enzymes like KMT2D (an HMT) and KDM2A (an HDM) is linked to various cancers including gastric cancer by influencing the chromatin structure and gene activity. In neurodegenerative diseases, these enzymes impact the health of neurons by modifying genes involved in brain function. In Alzheimer’s, Parkinson’s, and Huntington’s diseases, changes in HMT and HDM activity can lead to the dysregulation of genes critical for neuron survival, contributing to cognitive decline and motor dysfunction. Herein, we highlight the similarities and differences in how HMTs and HDMs function in cancer and neurodegeneration. Understanding these roles may reveal new therapeutic targets that address the epigenetic underpinnings of both cancer and neurodegenerative diseases, offering hope for innovative treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb434586ac7de7e0c22b593818a08f0a3dc6b5b6" target='_blank'>
              Epigenetic Modifiers: Exploring the Roles of Histone Methyltransferases and Demethylases in Cancer and Neurodegeneration
              </a>
            </td>
          <td>
            Lauren Reed, Janak Abraham, Shay Patel, Shilpa S. Dhar
          </td>
          <td>2024-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small non-coding RNAs (ncRNAs) that negatively regulate gene expression. MiRNAs regulate fundamental biological processes and have significant roles in several pathologies, including cancer. Cervical cancer is the best-known example of a widespread human malignancy with a demonstrated viral etiology. Infection with high-risk human papillomavirus (hrHPV) has been shown to be a causative factor for cervical carcinogenesis. Despite the occurrence of prophylactic vaccines, highly sensitive HPV diagnostics, and innovative new therapies, cervical cancer remains a main cause of death in developing countries. The relationship between hrHPV infection and cervical cancer depends on the integration of viral DNA to the host genome, disrupting the viral regulator E2 and the continuous production of the viral E6 and E7 proteins, which are necessary to acquire and maintain a transformed phenotype but insufficient for malignant cervical carcinogenesis. Lately, miRNAs, the tumor microenvironment, and immune evasion have been found to be major players in cervical carcinogenesis after hrHPV infection. Many miRNAs have been widely reported as deregulated in cervical cancer. Here, the relevance of miRNA in HPV-mediated transformation is critically reviewed in the context of the natural history of hrHPV infection and cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/803f13a1c54c6112e209a331f8578096768dd866" target='_blank'>
              The Natural History of Cervical Cancer and the Case for MicroRNAs: Is Human Papillomavirus Infection the Whole Story?
              </a>
            </td>
          <td>
            Giovanni Palomino-Vizcaino, Evelyn Gabriela Bañuelos-Villegas, L. M. Álvarez-Salas
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d66a87eaf162b506f3d34ebd1845180969ed36" target='_blank'>
              Mitochondrial transcription elongation factor TEFM promotes malignant progression of gliomas
              </a>
            </td>
          <td>
            Yin Wang, Wenxuan Hu, Boya Zhou, Yu Zhao, Yufei Tang, Zhiyong Deng, Minbin Chen
          </td>
          <td>2024-12-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Malignant gliomas are heterogeneous tumors, mostly incurable, arising in the central nervous system (CNS) driven by genetic, epigenetic, and metabolic aberrations. Mutations in isocitrate dehydrogenase (IDH1/2
 mut
 ) enzymes are predominantly found in low-grade gliomas and secondary high-grade gliomas, with IDH1 mutations being more prevalent. Mutant-IDH1/2 confers a gain-of-function activity that favors the conversion of a-ketoglutarate (α-KG) to the oncometabolite 2-hydroxyglutarate (2-HG), resulting in an aberrant hypermethylation phenotype. Yet, the complete depiction of the epigenetic alterations in IDH
 mut
 cells has not been thoroughly explored. Here, we applied an unbiased approach, leveraging epigenetic-focused cytometry by time-of-flight (CyTOF) analysis, to systematically profile the effect of mutant-IDH1 expression on a broad panel of histone modifications at single-cell resolution. This analysis revealed extensive remodeling of chromatin patterns by mutant-IDH1, with the most prominent being deregulation of histone acetylation marks. The loss of histone acetylation occurs rapidly following mutant-IDH1 induction and affects acetylation patterns over enhancers and intergenic regions. Notably, the changes in acetylation are not predominantly driven by 2-HG, can be rescued by pharmacological inhibition of mutant-IDH1, and reversed by acetate supplementations. Furthermore, cells expressing mutant-IDH1 show higher epigenetic and transcriptional heterogeneity and upregulation of oncogenes such as KRAS and MYC, highlighting its tumorigenic potential. Our study underscores the tight interaction between chromatin and metabolism dysregulation in glioma and highlights epigenetic and oncogenic pathways affected by mutant-IDH1-driven metabolic rewiring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1395bf17f297913f9bbe2a4e1db052179fa9f3f" target='_blank'>
              Oncogenic IDH1
 mut
 drives robust loss of histone acetylation and increases chromatin heterogeneity
              </a>
            </td>
          <td>
            N. Furth, Niv Cohen, Avishay Spitzer, T. Salame, B. Dassa, Tevie Mehlman, Alexander Brandis, Arieh Moussaieff, D. Friedmann-Morvinski, , Jerome Fortin, M. Suvà, I. Tirosh, A. Erez, Guy Ron, E. Shema
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Alternative polyadenylation (APA) affects most human genes and is recurrently dysregulated in cancers. However, the mechanistic origins of this dysregulation are incompletely understood. We completed an unbiased computational analysis of molecular regulators of poly(A) site selection across The Cancer Genome Atlas and identified that colorectal adenocarcinoma is distinct from all other cancer subtypes. We linked this distinction to the frequent presence of loss-of-function APC mutations in colorectal adenocarcinoma, which were strongly associated with expression of long 3′ UTRs relative to tumors lacking APC mutations. APC knockout similarly dysregulated APA in human colon organoids, and reduced APC expression was associated with APA dysregulation in tumor types lacking APC mutations. Building on previous work that identified APC as an RNA-binding protein that preferentially binds 3′ UTRs during mouse neurogenesis, we found that APC binding is most significantly enriched just upstream of proximal poly(A) sites within 3′ UTRs. Our results suggest that APC promotes proximal poly(A) site use and that APC loss contributes to pervasive APA dysregulation in human cancers.
 Citation Format: Austin M Gabel, Andrea E Belliville, James D Thomas, Jose Mario B Pineda, Robert K Bradley. APC mutations dysregulate alternative polyadenylation in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr A005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eeba394d3e2c3feb62667f6a25f162e6a65f993" target='_blank'>
              Abstract A005: APC mutations dysregulate alternative polyadenylation in cancer
              </a>
            </td>
          <td>
            Austin M. Gabel, Andrea E Belliville, James D Thomas, Jose Mario B Pineda, Robert K Bradley
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background B-cell receptor-associated protein 31 (BCAP31) is a widely expressed transmembrane protein primarily located in the endoplasmic reticulum (ER), including the ER-mitochondria associated membranes. Emerging evidence suggests that BCAP31 may play a role in cancer development and progression, although its specific effects across different cancer types remain incompletely understood. Methods The raw data on BCAP31 expression in tumor and adjacent non-tumor (paracancerous) samples were obtained from the Broad Institute Cancer Cell Line Encyclopedia (CCLE) and UCSC databases. We also examined the association between BCAP31 expression and clinicopathological factors. Using the Cox proportional hazards model, we found that high BCAP31 levels were linked to poor prognosis. To further explore BCAP31’s role, we analyzed the relationship between copy number variations (CNV) and BCAP31 mRNA expression using data from The Cancer Genome Atlas (TCGA). Additionally, the association between BCAP31 expression and signature pathway scores from the MsigDB database provided insights into the tumor biology and immunological characteristics of BCAP31.We assessed the relationship between tumor immune infiltration and BCAP31 expression using the TIMER2 and ImmuCellAI databases. The ESTIMATE computational method was employed to estimate the proportion of immune cells infiltrating the tumors, as well as the stromal and immune components, based on TCGA data. To investigate drug sensitivity in relation to BCAP31 expression, we utilized GDSC2 data, which included responses to 198 medications. We explored the relationship between BCAP31 gene expression and response to immunotherapy. Additionally, the study involved culturing KYSE-150 cells under standard conditions and using siRNA-mediated knockdown of BCAP31 to assess its function. Key experiments included Western blotting (WB) to confirm BCAP31 knockdown, MTT assays for cell proliferation, colony formation assays for growth potential, Transwell assays for migration and invasion, and wound healing assays for motility. Additionally, immunohistochemistry (IHC) was performed on tumor and adjacent normal tissue samples to evaluate BCAP31 expression levels. Results BCAP31 was found to be significantly overexpressed in several prevalent malignancies and was associated with poor prognosis. Cox regression analysis across all cancer types revealed that higher BCAP31 levels were predominantly linked to worse overall survival (OS), disease-free interval (DFI), disease-specific survival (DSS), and progression-free interval (PFI). In most malignancies, increased BCAP31 expression was positively correlated with higher CNV. Additionally, BCAP31 expression was strongly associated with the tumor microenvironment (TME), influencing the levels of infiltrating immune cells, immune-related genes, and immune-related pathways. Drug sensitivity analysis identified six medications that showed a significant positive correlation with BCAP31 expression. Furthermore, BCAP31 expression impacted the outcomes and prognosis of cancer patients undergoing immune therapy. The functional assays demonstrated that BCAP31 knockdown in KYSE-150 cells significantly inhibited cell migration, invasion, and proliferation while enhancing colony formation ability. WB and immunohistochemistry analyses confirmed elevated BCAP31 expression in tumor tissues compared to adjacent normal tissues in esophageal cancer, lung adenocarcinoma, and gastric adenocarcinoma. Conclusion BCAP31 has the potential to serve as a biomarker for cancer immunology, particularly in relation to immune cell infiltration, and as an indicator of poor prognosis. These findings provide a new perspective that could inform the development of more targeted cancer therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1980cf877b3cea99dfe3c2e67568921e58ff565b" target='_blank'>
              Pan-cancer analysis shows that BCAP31 is a potential prognostic and immunotherapeutic biomarker for multiple cancer types
              </a>
            </td>
          <td>
            Yangyong Sun, Zhi Li, Jianchao Liu, Ying Xiao, Yaqiang Pan, Benbo Lv, Xufeng Wang, Zhiqiang Lin
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, , Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Breast cancer is the most common type of cancer in females and recurrence increases over time, unlike many other cancers. Treatment for recurrent cancer is often the same as the primary with no additional biopsies taken. Current research suggests that subtype switching and tumor character changes frequently occur between primary and recurrent breast cancers. Therefore, it may be beneficial to patients to switch treatment based on these changes. Since physical biopsies are cumbersome and not always feasible, liquid biopsies open a way to monitor tumor changes less invasively and more comprehensively. It has been previously established that cell free DNA (cfDNA) shed into the bloodstream from dying cells can reflect cell type of origin via methylation pattern and rate of death via concentration of cfDNA. In this pilot study, we seek to look at the cfDNA of fifteen late stage pre-/ post-surgical breast cancer patients who also received radiation treatment. We will perform cfDNA extraction on the serum of these patients and both whole genome bisulfite sequencing (WGBS), which chemically converts unmethylated cytosine to uracil/thymine in the DNA and is the current gold-standard of methylation sequencing, and a newer method, enzymatic methylation sequencing (EM-seq), which enzymatically converts (TET2/APOBEC) unmethylated cytosine to uracil/thymine and potentially preserves more of the cfDNA. To validate any signatures found in the cfDNA of the breast cancer patients, we have begun the genomic DNA (gDNA) extraction and WGBS/ EM-seq protocols on a variety of breast cancer cell lines including: MCF10A, MCFDCIS, MCF7, T47D, BT474 MDA MB453, MDA MB436 and MDA MB231 (including in-lab brain, bone, and lung metastatic clones). Bioanalyzer traces are produced from the extracted cfDNA/gDNA and also for the final sequencing libraries. The success of the methylation conversion is evaluated after sequencing data is returned and conversion rates of the cytosine to uracil/thymine are compared to unmethylated DNA control (lambda) and methylated DNA control (pUC19). Once this sequencing data is obtained, we use an in-lab deconvolution algorithm to detect cell types of origin and intend to make the algorithm more robust for cancer cell types as well. We have currently produced breast cancer cell line methylation sequencing libraries and are in the process of producing the libraries for the patient samples. Our current data suggests that there are changes in the cfDNA general fragmentation patterns and cfDNA concentrations between pre-/post- surgery samples. Once our sequencing data is obtained for the patient samples, we will run our deconvolution algorithm. The potential of characterizing breast cancer subtype and progression signatures in cfDNA of late stage pre-/ post-surgical breast cancer patients can have significant impact on patient treatment options. Identifying these breast cancer signatures less invasively and, therefore, more frequently may allow for early and more targeted intervention to improve breast cancer patient outcomes.
 Citation Format: Amber Alley, Megan McNamara, Sidharth Jain, Anton Wellstein. Investigating distinct methylation signatures characteristic of breast cancer subtypes in residual disease via cell free DNA methylation [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ff4bbac5be2eece6c153ec40b5da54d78512fc" target='_blank'>
              Abstract B005: Investigating distinct methylation signatures characteristic of breast cancer subtypes in residual disease via cell free DNA methylation
              </a>
            </td>
          <td>
            Amber Alley, Megan E McNamara, Sidharth S Jain, Anton Wellstein
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite being considered a rare tumor, uveal melanoma (UVM) is the most common adult intraocular malignancy. With a poor prognosis and limited treatment options, up to 50% of patients develop metastases, primarily in the liver. A range of mutations and chromosomal aberrations with significant prognostic value has been associated with UVM pathogenesis. The most frequently mutated genes are GNAQ and GNA11, which encode the α subunits of Gq proteins and are described as driver mutations that activate multiple signaling cascades involved in cell growth and proliferation. Directly downstream of Gαq/11 activation, PLCβ engagement leads to sustained production of DAG and IP3. While the DAG/PKC/RasGRP3/MAPK signaling branch has been identified as an essential component of UVM unregulated proliferation, the role of IP3-mediated signals has been largely overlooked. Here, we demonstrate that, whilst maintaining Ca²⁺ homeostasis, UVM cells have developed a decoupling mechanism between IP3 and ER Ca²⁺ release by altering IP3 receptor (IP3R) expression. This correlation was observed in human UVM tumors, where IP3Rs were found to be downregulated. Critically, when IP3R3 expression was restored, UVM cells exhibited an increased tendency to undergo spontaneous cell death and became more sensitive to pro-apoptotic modulators of IP3R-mediated Ca²⁺ signaling, such as staurosporine and the Bcl2-IP3R disrupter peptide BIRD2. Finally, inhibition of the Gαq/11 signaling pathway revealed that IP3R expression is negatively regulated by GNAQ/11 oncogenic activation. Hence, we demonstrated that by remodeling IP3R expression, GNAQ/11 oncogenes protect UVM cells against IP3-triggered Ca²⁺ overload and cell death. Therefore, the GNAQ/11 pathway not only drives proliferation through DAG activity but also provides a protective mechanism to evade IP3/Ca²⁺-mediated cell death. These dual functions could potentially be exploited in novel combinatorial therapeutic strategies to effectively block UVM cell proliferation while simultaneously sensitizing them to cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b07937d318b87332a263586ad5eb9529b6d71388" target='_blank'>
              Oncogenic GNAQ/11-induced remodeling of the IP3/Calcium signaling pathway protects Uveal Melanoma against Calcium-driven cell death
              </a>
            </td>
          <td>
            Garcia Céline, Roussel Louis, Massaad Sarah, Brard Laura, La Rovere Rita, Tartare-Deckert Sophie, Bertolotto Corine, Bultynck Geert, Leverrier-Penna Sabrina, Penna Aubin
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="In cancer research, the term epigenetics was used in the 1970s in its modern sense encompassing non-genetic events modifying the chromatin state, mainly to oppose the emerging oncogene paradigm. However, starting from the establishment of this prominent concept, the importance of these epigenetic phenomena in cancer rarely led to questioning the causal role of genetic alterations. Only in the last 10 years, the accumulation of problematic data, better experimental technologies, and some ambitious models pushed the idea that epigenetics could be at least as important as genetics in early oncogenesis. Until this year, a direct demonstration of epigenetic oncogenesis was still lacking. Now, Parreno, Cavalli and colleagues, using a refined experimental model in the fruit fly Drosophila melanogaster, enforced the initiation of tumors solely by imposing a transient loss of Polycomb repression, leading to a purely epigenetic oncogenesis phenomenon. Despite a few caveats that we discuss, this pioneering work represents a major breakpoint in cancer research. We are led to consider the theoretical and conceptual implications on oncogenesis and to search for links between this artificial experimental model and naturally occurring processes, while revisiting cancer theories that were previously proposed as alternatives to the oncogene-centered paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f2f2c2f1a7eb766c5cfd45527bf643538142c9e" target='_blank'>
              Evidence of Epigenetic Oncogenesis: A Turning Point in Cancer Research.
              </a>
            </td>
          <td>
            Jean-Pascal Capp, Benoît Aliaga, Vera Pancaldi
          </td>
          <td>2024-11-21</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA is frequently damaged by genotoxic stresses such as ionizing radiation, reactive oxygen species, and nitrogen species. DNA damage is a key contributor to cancer initiation and progression, and thus the precise and timely repair of these harmful lesions is required. Recent studies revealed transcription as a source of genome instability, and transcription-coupled DNA damage has been a focus in cancer research. Impaired mRNA export is closely related to DNA damage through R-loop formation. The molecular machineries of transcription-coupled DNA damage have been extensively analyzed in Saccharomyces cerevisiae. However, the molecular basis of these phenomena in higher eukaryotes remains elusive. In this review, we focus on the relationship between deregulated mRNA export through the transcription-export-2 (TREX-2) complex and cancer development. Particularly, the expression of germinal center-associated nuclear protein (GANP), a molecular scaffold in the TREX-2 complex, is highly associated with tumorigenesis in mice and humans. Although the deregulated expression of other components in the TREX-2 complex might affect cancer development, we have directly demonstrated the significance of GANP in tumorigenesis using genetically modified mice. Additionally, we describe recent evidence for medical applications demonstrating that the downregulation of the other components may be a good candidate for a chemotherapeutic target in terms of reducing the side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e429b3bb884f7cc3d479875de747738fafbdce" target='_blank'>
              Tumorigenesis Caused by Aberrant Expression of GANP, a Central Component in the Mammalian TREX-2 Complex—Lessons from Transcription-Coupled DNA Damages
              </a>
            </td>
          <td>
            Andri Rezano, Naomi Gondo, Yasuhiro Sakai, Yuko Nakamura, S. Phimsen, Tokio Tani, Akihiko Ito, Seiji Okada, Kazuhiko Kuwahara
          </td>
          <td>2024-12-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Hepatocellular carcinoma (HCC) is the most common form of liver cancer and accounts for ∼90% of cases globally. Detection of early-stage HCC is vital as it allows for potentially curative treatment, yet remains a significant challenge. Micronuclei (MN) are a hallmark of genomic instability. An increased frequency of MN is commonly observed in tumor cells as well as other somatic cells, including erythrocytes, in cancer patients. Here, we have established a technique (WO2021/228246 A1), for the isolation and analysis of micronuclei DNA (MN-DNA) in erythrocytes from peripheral blood. Significant changes in read densities at specific genomic locations within MN-DNA were identified in patients, termed as tumor- associated MN-DNA (taMN-DNA) features. This study evaluates the potential of these taMN- DNA features for early-stage HCC detection across human and murine model. Methods: To explore the potential of MN-DNA in cancer detection, we first collected 1-2 mL of whole blood from HDs (N = 53) and HCC patients (N = 53). MN-DNA was isolated and purified from erythrocytes, followed by whole-genome sequencing. Participants were randomly divided into training, test cohorts in an 8:2 ratio. We applied machine learning algorithms to leverage these features for cancer detection in the human model. Additionally, to investigate the formation of taMN-DNA features, we employed a well-established mouse model that mimics HCC development. Mice were injected with the genotoxic agent diethylnitrosamine (DEN) 2 weeks after birth, and developed malignant macroscopic liver nodules at approximately 40 weeks of age. Results: In the human cohort, we enrolled 106 participants, with more than half of the HCC cases were diagnosed at early-stage (stage 0-II). The cancer detection model utilizing taMN- DNA features achieved an overall accuracy of 86.3%, with a sensitivity of 81.8% and specificity of 90.9%. For early-stage, the model demonstrated sensitivities of 54.5% at a specificity of 90.9%. In the DEN-induced HCC mouse model, unsupervised hierarchical clustering based on these selected taMN-DNA features clearly separated WT and tumor mice into two distinct groups. Notably, mice that were not tumorigenic at 36 weeks post-treatment in the DEN-treated group fell into the WT group when classified by the same taMN-DNA features, indicating that the formation of taMN-DNA signatures is associated with the presence of tumors specifically. Conclusions: This pilot study highlights the potential of MN-DNA as a promising tool for early HCC detection. Leveraging these taMN-DNA features can accurately distinguish early-stage HCC patients from HDs with high sensitivity. In both human and murine models, there is a significant relationship between these signatures and tumor presence, suggesting that taMN- DNA features might reflect the diseased state across mammalian species. Research sponsor: Timing Biotech.
 Citation Format: Hui Lin, Haobo Sun, Xingyun Yao, Xiaoxiao Fan, Fei Meng, Honghao Liang, Xiaofei Gao. Leveraging micronuclei DNA from erythrocytes for early detection of hepatocellular carcinoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15221ad2cc8dccf6b314a7408989c4da875e19bd" target='_blank'>
              Abstract B073: Leveraging micronuclei DNA from erythrocytes for early detection of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hui Lin, Haobo Sun, Xingyun Yao, Xiaoxiao Fan, Fei Meng, Honghao Liang, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer.
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-02</td>
          <td>Cell research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa78a29add9f8f7c58ef77a453eb2141f303497" target='_blank'>
              Differentiation signals induce APOBEC3A expression via GRHL3 in squamous epithelia and squamous cell carcinoma
              </a>
            </td>
          <td>
            Nicola J Smith, Ian Reddin, Paige Policelli, Sunwoo Oh, Nur Zainal, Emma Howes, Benjamin Jenkins, Ian Tracy, Mark Edmond, Benjamin Sharpe, Damian Amendra, Ke Zheng, Nagayasu Egawa, John Doorbar, Anjali Rao, Sangeetha Mahadevan, Michael A. Carpenter, R. Harris, Simak Ali, Christopher Hanley, Rémi Buisson, Emma King, Gareth J. Thomas, Tim R. Fenton
          </td>
          <td>2024-11-15</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Cancers arising from dysregulation of generally operative signaling pathways are often tissue specific, but the mechanisms underlying this paradox are poorly understood. Based on striking cell-type specificity, we postulated that these mechanisms must operate early in cancer development and set out to study them in a model of von Hippel Lindau (VHL) disease. Biallelic mutation of the VHL ubiquitin ligase leads to constitutive activation of hypoxia inducible factors HIF1A and HIF2A and is generally a truncal event in clear cell renal carcinoma. We used an oncogenic tagging strategy in which VHL-mutant cells are marked by tdTomato, enabling their observation, retrieval, and analysis early after VHL-inactivation. Our findings reveal markedly different consequences of HIF1A and HIF2A activation, but that both contribute to renal cell-type specific consequences of VHL-inactivation in the kidney. Early involvement of HIF2A in promoting proliferation within proximal tubular epithelium supports therapeutic targeting of HIF2A early in VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be97f9c0f0b00dada5534b4bd4b5d705fdb68b10" target='_blank'>
              HIFα isoform specific activities drive cell-type specificity of VHL-associated oncogenesis
              </a>
            </td>
          <td>
            Joana Darc Carola Correia Lima, Madeleine Hooker, Ran Li, Ayslan B. Barros, N. Masson, Christopher W. Pugh, D. Mole, Julie Adam, Peter J. Ratcliffe, Samvid Kurlekar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated oncogenes in cancer and is involved in various tumorigenic processes. Among its diverse roles, KRAS mutations have been shown to upregulate macropinocytosis, a form of endocytosis that allows cells to engulf extracellular fluid and its contained solutes, subsequently supporting the increased need for cancer cells' growth and proliferation. The nutrient uptake function of macropinocytosis was initially described within the framework of KRAS-driven pancreatic tumors. Although constitutive macropinocytosis can be induced by activating mutations of proteins that are commonly found in cancer, KRAS-induced macropinocytosis constitutes a fundamental area of research due to the high occurrence of mutated KRAS in cancer (~1/3 of all cancers).  Furthermore, KRAS-mutated macropinocytosis not only contributes to tumorigenesis but also plays a critical role in developing resistance to treatments, as it was found to be implicated in Multi-Drug Resistance (MDR) in cancer cells. This mini-review aims to synthesize current knowledge of mechanisms of KRAS-mutated macropinocytosis briefly and examines the relationship between KRAS and macropinocytosis in the light of its role in cancer progression and drug resistance, highlighting therapeutic implications, targeting potential vulnerabilities and outlining clinical advancements in pertinent therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4b47f4761d206b1edf858bd1fc0d092897567" target='_blank'>
              The Role of KRAS in Macropinocytosis and its Implications of Multi-Drug Resistance in Cancer
              </a>
            </td>
          <td>
            F. Fenyvesi, Zeinab Ibrahim
          </td>
          <td>2024-12-23</td>
          <td>De Remediis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes. The UW/Fred Hutch metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting cell-free DNA (cfDNA) and matched normal tissue, primary tumor, and metastatic tumor samples from advanced BLCA patients, with a focus on these variant subtype tumors. This dataset, consisting of 20 patients with up to five metastases per patient, is a unique opportunity to assess how cfDNA is able to capture heterogeneity across BLCA. We assessed both inter-patient heterogeneity between BLCA subtypes and inter-tumoral heterogeneity within each patient using epigenetic and genomic profiling. Using nucleosome profiling, we identified the activity of lineage markers and transcription factors that distinguish healthy individuals from BLCA patients, and between BLCA subtypes. Furthermore, we conducted a detailed analysis of whole genome (30X) and targeted (4,000X) cfDNA sequencing in their ability to capture the evolutionary history of tumor mutations. Detection of tumor mutations by cfDNA is highly dependent on clonality, with >90% of founder tumor mutations captured by either sequencing method, while capture is decreased for either subclonal mutations or those private to a single metastasis. Importantly, deleterious mutations in BLCA driver genes had close to 100% detection using targeted panel sequencing, even for subclonal mutations. These findings lay the groundwork for the use of cfDNA in clinical BLCA decision-making, including tracking the emergence of pathogenic, targetable mutations and variant subtypes.
 Citation Format: Samantha L Schuster, Pushpa Itagi, Sonali Arora, Patricia C Galipeau, Thomas W Persse, Michael Yang, Allie Kreitman, Alan Min, Funda Vakar-Lopez, John K Lee, Petros Grivas, Robert B Montgomery, Jonathan L Wright, Andrew C Hsieh, Hung- Ming Lam, Gavin Ha. Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb84ec36801f20a2ac043ba0422da74b9249b7" target='_blank'>
              Abstract B040: Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer
              </a>
            </td>
          <td>
            Samantha L Schuster, P. Itagi, Sonali Arora, Patricia Galipeau, Thomas Persse, Michael Yang, Allie Kreitman, Alan Min, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Squamous cell carcinomas (SCC) are often preceded by potentially malignant precursor lesions, most of which remain benign. The terminal exhaustion phenotypes of effector T-cells and the accumulation of myeloid-derived suppressor cells (MDSC) have been thoroughly characterized in established SCC. However, it is unclear what precancerous lesions harbor a bona fide high risk for malignant transformation and how precancerous epithelial dysplasia drives the immune system to the point of no return. Here we show that expression of SRY-box transcription factor 2 (SOX2) in precancerous lesions imparts an irreversible risk that recruits suppressive myeloid cells by promoting the release of CCL2. We developed a unique genetically engineered mouse model (GEMM) to recapitulate the malignant transformation of epithelial dysplasia to SCC in the oral mucosa with high histologic and phenotypic fidelity. Using a combination of longitudinal human specimens and the Sox2-GEMM, we found that the myeloid cells in precancerous epithelial dysplasia exhibit a distinctive dichotomous profile featuring high levels of IL-1α-SLC2A1 and low levels of type-I interferon (IFN-I) signatures, which occurs before SCC emerges histologically. Brief priming of myeloid cells with IL-1α desensitizes them to IFN-I agonists and makes myeloid-derived suppressor cells (MDSC) even more suppressive of T-cell activation. Mechanistically, IL-1 activation represses the expression of DHHC3/7 enzymes, which are responsible for the palmitoylation of stimulator of interferon genes (STING). Early blockade of IL1 signaling using pharmacologic and genetic approaches similarly reduces MDSC and SLC2A1high myeloid cells, suppresses epithelial dysplasia transformation, and extends survival. This work establishes a previously unrecognized SOX2-CCL2-IL1 pathway that leads to irreversible immune escape when precancerous epithelial lesions transform.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68906ee8a14c680b07808e1d20602c4baa417802" target='_blank'>
              SOX2-induced IL1α-mediated immune suppression drives epithelial dysplasia malignant transformation
              </a>
            </td>
          <td>
            Hülya F. Taner, Wang Gong, Z. Fitzsimonds, Zaiye Li, Yuesong Wu, Yumin He, Kohei Okuyama, Wanqing Cheng, Jung Kuczura, Sashider Rajesh, Andriana Manousidaki, Shuo Feng, Miki Lee, Felipe Nör, E. Lanzel, S. Demehri, Peter J. Polverini, Jacques E. Nör, Thomas D. Wang, Jianwen Que, Haitao Wen, Yuying Xie, James J. Moon, Yu Leo Lei
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background: Chronic Myeloid Leukemia (CML) is a malignant genetic disorder commonly caused by the translocation of two genes among chromosomes 9 and 22. Nucleophosmin1 (NPM1) is a nuclear protein-coded gene located on chromosome 5q35. NPM1 protein, in association with other tumor suppressor proteins, prevents cell proliferation in an uncontrolled way. Several mutations are located within exon 12 of the NPM1 gene. More than 95% of these mutations are frameshift due to the insertion of tetra‐nucleotides at positions 863 and 864 of exon 12. Three common mutation types (A, B, and D) represent about 90% of NPM1 mutations. Objectives: To analyze the NPM1 gene variants and their association with CML. Methodology: Genomic DNA was extracted from the whole blood of 50 cases and health control subjects. NPM1 (563 bp) was amplified using gene-specific primers by PCR and sequenced using BigDye® Terminator. The mutation was analyzed using Sanger sequencing in an ABI Gene Analyzer (3130XL). The results were analyzed and compared with the reference human NPM1 gene (accession # NG_016018.1). Results: The gene sequencing analysis revealed that NPM1 mutation (type A: Dup TCTG) was not present in DNA-sequenced samples of CML patients. Conclusion: It may be concluded that NPM1 mutations are unlikely to be linked to CML illness in a Pakistani community. Further research with a larger sample size may aid in determining the role of other gene mutations with CML for the effective treatment of this cancer.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc6733160cff5450ae996e96cd43d480700fa23" target='_blank'>
              MUTATION ANALYSIS FOR NUCLEOPHOSMIN-1 (NPM1) GENE VARIANT OF CHRONIC MYELOID LEUKEMIA PATIENTS FROM LAHORE-PAKISTAN
              </a>
            </td>
          <td>
            Sumaira Mehboob, Kokab Durri, Muhammad Farooq Sabar, Hira Muzzamal, Farkhanda Manzoor
          </td>
          <td>2024-11-25</td>
          <td>International Journal of Pharmacy &amp; Integrated Health Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: Advancements in non-invasive diagnostic technologies for gastric carcinoma (GC), including serum microRNA and cell-free DNA assays, have shown promise but still face challenges in achieving consistent early detection. Micronuclei (MN), extranuclear bodies harboring fragmented chromosomal segments, are indicative of genomic instability and have been employed in genotoxicity assessments and cancer diagnostics. We have developed a novel technique (WO2021/228246 A1) for the purification and characterization of micronuclei DNA (MN-DNA) extracted from erythrocytes in peripheral blood. Here, we evaluated the potential of micronuclei DNA (MN-DNA) from 2ml peripheral blood for early gastric carcinoma detection and explored its application for early cancer diagnosis. Methods: We assembled a clinical cohort consisting of 425 healthy donors (HDs) and 282 gastric cancer patients, from whom 1-2 mL peripheral blood samples were collected. Participants were randomly allocated into training, validation, and independent test cohorts in a 7:2:1 ratio, ensuring similar distributions of cancer stages and gender. MN-DNA was isolated from erythrocytes and subjected to comprehensive whole-genome sequencing. Predictive machine learning models were constructed utilizing distinct tumor-associated MN-DNA features identified through genome-wide analysis to distinguish between HDs and GC. Results: Comprehensive genome-wide analysis of MN-DNA demonstrated pronounced disparities in distribution patterns between HDs and GC patients. The predictive model built on MN-DNA features achieved an AUC of 97.69% (95%CI: 95.57- 99.81%), with an 96.47% specificity and 91.07% sensitivity in the independent test cohort. The model identified early GC (stage 0-I) and advanced GC (stage II-IV) with sensitivities of 95.24% or 78.57%, respectively. Conclusions: Our results demonstrate that MN-DNA from 1-2mL peripheral blood enable accurate detection of GC, especially in early GC. As a type of cytoplasmic DNA, MN-DNA can provide a valuable tool for early cancer detection, offering different underlying mechanisms compared to current methods. Research sponsor: Timing Biotech.
 Citation Format: Yuehua Han, Xingyun Yao, Haobo Sun, Cheng Fang, Peiwei Li, Honghao Liang, Jie Jin, Xiaofei Gao. Detection of early gastric carcinoma by micronuclei DNA from erythrocytes [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560ab2187f0cf9f33cc6777d6ea65c8f8ff3b8b4" target='_blank'>
              Abstract B075: Detection of early gastric carcinoma by micronuclei DNA from erythrocytes
              </a>
            </td>
          <td>
            Yuehua Han, Xingyun Yao, Haobo Sun, Cheng Fang, Peiwei Li, Honghao Liang, Jie Jin, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1113f4b65f7bb6543b590129c22b6e32ee76e8" target='_blank'>
              Transposable element 5mC methylation state of blood cells predicts age and disease.
              </a>
            </td>
          <td>
            Francesco Morandini, Jinlong Y Lu, Cheyenne Rechsteiner, A. Shadyab, Ramon Casanova, Beverly M Snively, A. Seluanov, Vera Gorbunova
          </td>
          <td>2024-11-27</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="The idea that our genetic material contains Junk DNA is fading. Many DNA segments that were previously believed to be worthless now have important functions, according to recent studies. These non-coding areas, once known as Junk DNA, are crucial for regulating when and how genes function even though they do not directly make proteins. They assist our cells respond to various circumstances and direct the growth of organisms. Certain non-coding DNA sequences function as switches that determine where and when genes should be activated or deactivated. Furthermore, some non-coding DNA segments help maintain the orderly structure of our chromosomes. They also contribute to the evolution of species and may contain remnants of ancient viruses that once infected our forebears. Although scientists still don’t fully understand every aspect of non-coding DNA, it is now recognized as a complex and active element of our biology. Researchers believe that what was once labelled as Junk DNA could actually be very important for our health, development, and ability to adapt to environmental changes. This new perspective is helping us understand how valuable Junk DNA really is.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599edea5da951ac4d9e655e4b32b9a268a345ece" target='_blank'>
              Junk DNA: Is it Really Junk - or a Code Waiting to Be Cracked?
              </a>
            </td>
          <td>
            Md. Imran Wahab, IPS
          </td>
          <td>2024-11-12</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An important reason for the poor prognosis of nasopharyngeal carcinoma (NPC) patients is radioresistance. Our previous studies demonstrated that BRD7 is expressed at low levels in NPC and functions as a tumor suppressor to inhibit NPC progression and metastasis. However, the role and mechanism of BRD7 in the development of radioresistance in NPC cells remain unclear. In this study, we first found that BRD7 was lowly expressed in radioresistant NPC tissues and cells compared to radiosensitive tissues and cells and that overexpression of BRD7 promoted the induction of DNA double-strand breaks and increased radiosensitivity in NPC cells. Mechanistically, BRD7 competitively inhibits the binding of the deubiquitinating enzyme USP5 to METTL3, thereby reducing the protein stability of METTL3 through the ubiquitin-proteasome pathway. Furthermore, METTL3 was confirmed to suppress the induction of DSBs and promote the development of NPC radioresistance by regulating BRCA1- and RAD51-mediated homologous recombination repair. Moreover, high BRD7 expression and low METTL3 expression are positively correlated with radiosensitivity and good prognosis in NPC patients. Taken together, our findings reveal that BRD7 promotes the radiosensitization of NPC cells by negatively regulating USP5/METTL3 axis activity and indicate that targeting the BRD7/METTL3 axis might be a novel therapeutic strategy for NPC radiosensitization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93379b9722efedab86d3cad8b6b1b05b3f02e3b" target='_blank'>
              BRD7 enhances the radiosensitivity of nasopharyngeal carcinoma cells by negatively regulating USP5/METTL3 axis-mediated homologous recombination repair
              </a>
            </td>
          <td>
            Mengna Li, Jianxia Wei, Changning Xue, Shipeng Chen, Xiangting Zhou, Lemei Zheng, Yumei Duan, Hongyu Deng, Songqing Fan, Wei Xiong, Faqing Tang, Ming Zhou
          </td>
          <td>2024-11-11</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Micronuclei are the extra-nuclear chromatin compartments separated from the primary nucleus and surrounded by their own nuclear envelope. For a long time it has been thought that micronuclei is the final stage of the pathological process in a cell. They have been used as biomarkers of the influence of genotoxic factors as well as of genome instability in various diseases. Nowadays, it is demonstrated that micronuclei could be involved in the cellular activities, affect the nuclear genome and lead to the changes in cell and tissue physiology. It is known that the formation of micronuclei is one of the steps in selective chromatin elimination in the ontogenesis of plant and animal species. The regions to be marked and eliminated from cell nucleus are recognized at the level of genome. This process is often accompanied by modifications with the heterochromatin formation, changes in the chromosome condensation and in the position of chromosomes in the nucleus. The processes observed in selective and non-selective chromatin elimination are similar to a great extent. The fact that the role of micronuclei in the cell functioning is not well-known yet, and the composition of the micronuclei and the ways of chromatin elimination could influence their role in the development of the pathogenesis, emphasizes the importance of additional studies for a more profound investigation of this phenomenon.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeddb8a010fe6c42592e816c3b4480c6ec74d17" target='_blank'>
              The role of micronuclei in chromatin elimination
              </a>
            </td>
          <td>
            Yu. R. Akhmadullina, P. O. Khomenko
          </td>
          <td>2024-11-18</td>
          <td>Žurnal obŝej biologii</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer has become the leading cause of death worldwide, and it is the second most common cancer in women and the third most common cancer in men. Accumulating evidence suggests that genetic and epigenetic factors play a key role in the development of colorectal cancer. Cancer Stem Cells (CSC) play an important role in the suppression or development of cancer in various conditions. In recent years, non-coding RNAs (ncRNA) have been the focus, and the association of CSC and non-coding RNA has played a crucial role in the development of human cancers. These non-coding RNAs are known to be expressed in many cancers. Studies have suggested that ncRNAs are dysregulated in colorectal cancer cells, and different factors, like Wnt and Notch, are involved in this dysregulation. ncRNAs play a significant role in cancer initiation, migration, and resistance to therapies. Moreover, long noncoding RNAs are known to regulate tumor suppressor genes or oncogenes. Targeting different ncRNAs like miRNA, circular RNA, long noncoding RNAs, and small interfering RNA may provide efficient, targeted therapeutic strategies for colon cancer treatment. This review aims to briefly discuss the latest findings on the role of noncoding RNAs in the prognosis and diagnosis of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf75cafb36b85765499ce0fa8a4cfb98a12e479" target='_blank'>
              The Role of Noncoding RNAs in the Prognosis and Diagnosis of Colorectal Cancer: An Emerging Biomarker.
              </a>
            </td>
          <td>
            O. S. Vani, Kavitha R Thangaraj, Varshaa Ravichandran, Solomon F D Paul
          </td>
          <td>2024-11-26</td>
          <td>MicroRNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089aac455387b5d1dba7549c993d3541b59537a0" target='_blank'>
              Circular RNA circPHLPP2 promotes tumor growth and anti-PD-1 resistance through binding ILF3 to regulate IL36γ transcription in colorectal cancer
              </a>
            </td>
          <td>
            Yan Hu, Ze-Rong Cai, Ren-Ze Huang, De-Shen Wang, Huai-qiang Ju, Dong-Liang Chen
          </td>
          <td>2024-12-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Cells release extracellular vesicles (EVs) loaded with functional biomolecules that facilitate intercellular communication in health and disease. These EVs contain DNA (EV-DNA), mirroring the mutational profile of parental cells, yet the mechanism of DNA packaging onto EVs and its role in diseases like cancer remain unclear. Using advanced imaging and biochemical techniques, we discovered that EV-DNA predominantly resides on the vesicle surface, bound to modified histones. Through genome-wide CRISPR knockout screens in cancer cell lines, we identified immune-related pathways and genes crucial for EV-DNA loading, such as APAF1 and NCF1. In several cancer models, loss of APAF1 reduced EV-DNA packaging, and enhancing metastasis, thus implicating EV-DNA in immune surveillance against cancer spread. We demonstrated that EV-DNA uptake by liver macrophages triggers DNA damage response, altering cytokine production and promoting immune activation in pre-metastatic sites. Moreover, quantifying EV-DNA in colorectal cancer biopsies revealed its potential as a predictive metastatic risk biomarker. Our findings shed light on EV-DNA packaging mechanisms and its significance, suggesting that elevated levels of tumor-derived EV-DNA bolster anti-tumor immunity and inhibit metastasis.
 Citation Format: Inbal Wortzel, Han-Sang Kim, David C. Lyden. DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edef13099e4837d7d8441f1fcd7734045284c7c9" target='_blank'>
              Abstract PR014: DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity
              </a>
            </td>
          <td>
            I. Wortzel, H. Kim, David C. Lyden
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aea14196d457e30b1e0c0858af5e94a5516dad2" target='_blank'>
              RBM47 promotes cell proliferation and immune evasion by upregulating PDIA6: a novel mechanism of pancreatic cancer progression
              </a>
            </td>
          <td>
            Yihui Ma, Enjie Liu, Huijie Fan, Chenfei Li, Pei Huang, Meiying Cui, Zhengyang Wang, Jing Zhou, Kuisheng Chen
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed9e928f87fb979a12e3b014048518d7d5d625c" target='_blank'>
              Engineered transcription-associated Cas9 targeting in eukaryotic cells
              </a>
            </td>
          <td>
            Gregory W. Goldberg, M. Kogenaru, Sarah Keegan, Max A B Haase, Larisa Kagermazova, Mauricio A. Arias, Kenenna Onyebeke, Samantha Adams, Daniel K. Beyer, D. Fenyö, M. Noyes, J. Boeke
          </td>
          <td>2024-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="
 The majority of the human genome (>70%) contains non-coding DNA consisting of retrotransposable elements (RTE), such as LINE-1, SINE/Alu elements, human endogenous retroviruses (HERV), and satellite repeat DNA (Hsat2 and Hsat3). The expression of these elements is suppressed in normal adult tissues but are abnormally expressed in many cancers. Another key mechanism in cancer initiation and progression is the release of tumor-derived extracellular vesicles (EV) that are long-distance communication vehicles propagating signals from tumors inducing metabolic changes, immune dysregulation and promoting metastases across the body. Our work has uncovered a link between RTE, HERV and Hsat expression in cancer and EV in driving tumor progression and immune dysregulation. W hole transcriptome sequencing of EV derived from several different cancer cell lines and patient samples, including Ewing sarcoma, osteosarcoma, prostate, pancreatic, and breast cancer found that RTE, HERV and Hsat2,3 transcripts are highly enriched in EV versus coding region transcripts and in much higher abundance than in the tumor cell themselves. Tumor-derived EV contained dsRNA, dsDNA and RNA:DNA hybrids from these non-coding regions. Plasma EV isolated from both metastatic and non-metastatic patients also showed an enrichment of these RTE, HERV and Hsat2,3 elements and clearly differentiated healthy, normal donor EV that had little or no detectable signal. Evidence of local dissemination of these non-coding elements in EV was also found in vivo in tumor xenografts that showed transfer of Hsat2,3 transcripts into the murine stroma that naturally lack these satellite repeats. Treatment of fibroblasts and myeloid cells with tumor-derived EV triggered innate immune responses via type I IFN and pro-inflammatory cytokines, including IL-1beta, IL-6, IL-8, and TNFalpha owing to the pathogen-like nucleic acid sequences (viral mimicry) of these elements. These EV-induced chronic inflammatory effects were mainly induced through the cGAS-STING nucleic acid sensing pathway. Evidence of DNA damage in stromal fibroblasts and induced senescence was also found as a result of EV treatment. A key element of RTE is the expression of LINE-1 and HERV derived reverse transcriptase (RT) that propagates the action of RTE via RNA:DNA hybrids, dsDNA and retrotransposition in the genome. In a spontaneous estradiol-driven breast cancer model in ACI rats, long-term treatment with an HIV RT inhibitor (lamivudine/3TC) inhibited breast tumor development associated with reduced RTE activation in the mammary gland and in circulating immune cells as well as reduced chronic systemic innate inflammation. Our results suggest that RTE, HERV and Hsat activation, together with their local and systemic dissemination in EV plays an important role in cancer initiation and progression associated with chronic inflammation. We also suggest that abnormal dissemination of these activated non-coding elements in EV may also play a role in "inflammaging" facilitating not only cancer but also other chronic diseases.
 Citation Format: Laszlo Radvanyi, Valentina Evdokimova, Peter Ruzanov, Zhenbo Zhang, Poul Sorenson, Hendrik Gassmann, Stefan Burdach, Lincoln D. Stein. Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4545e23fd117647dd7aa5ce20199e63c3fd7396" target='_blank'>
              Abstract C012: Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation
              </a>
            </td>
          <td>
            Laszlo Radvanyi, V. Evdokimova, P. Ruzanov, Zhenbo Zhang, Poul Sorenson, H. Gassmann, Stefan Burdach, Lincoln D. Stein
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Objective: Long non-coding RNAs (lncRNAs) participate in the formation, progression, and metastasis of cancer. This study aimed to explore the roles of the lncRNA ST8SIA6 antisense RNA 1 (ST8SIA6-AS1) in tumorigenesis and elucidate the underlying molecular mechanism of its upregulation in hepatocellular carcinoma (HCC). Material and Methods: A total of 56 in-house pairs of HCC tissues were examined, and ST8SIA6-AS1 levels were determined through real-time polymerase chain reaction (PCR). The biological behavior of ST8SIA6-AS1 by Crispr-Cas9-based gene repression and activation was determined in vitro and in vivo. The binding sites and biological behavior of Myc proto-oncogene and forkhead box A on chromatin were investigated through luciferase reporter assays, chromatin immunoprecipitation–quantitative PCR, and co-immunoprecipitation (co-IP) assays. The regulatory mechanisms of ST8SIA6-AS1 expression were analyzed with encyclopedia of DNA elements and gene expression profiling interactive analysis. Results: The expression of ST8SIA6-AS1 significantly increased in multiple HCC cell lines and the 56 in-house pairs of HCC tissues (P = 0.0018). Functionally, high-efficiency Crispr-Cas9-based knockdown of ST8SIA6-AS1 revealed that ST8SIA6-AS1 knockdown attenuated the proliferation, migration, and infiltration of HCC cells and considerably reduced the growth rate of subcutaneous and orthotopic HCC tumors. Conversely, ST8SIA6-AS1 overexpression considerably improved the oncogenic characteristics of the HCC cells. Furthermore, ST8SIA6-AS1 upregulation was regulated by the direct binding of transcription factor Myc to the −260 bp to+155 bp and +1003 bp to +1312 bp regions of the ST8SIA6-AS1 transcription start site, which is a segment with high level of H3K27 acetylation. Myc knockdown or treatment with the BET bromodomain inhibitor JQ-1 considerably reduced ST8SIA6-AS1 RNA expression in the HCC cells. Conclusion: Our study has established the oncogenic role of ST8SIA6-AS1 and elucidated the Myc-dependent upregulation mechanism of ST8SIA6-AS1 in HCC, providing a profound theoretical molecular basis for the carcinogenic function of ST8SIA6-AS1 in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33616059336d41e344c8cfbb1d3985982730aa02" target='_blank'>
              Crispr-Cas9-based long non-coding RNA interference and activation identified that the aberrant expression of Myc-regulated ST8SIA6 antisense RNA 1 promotes tumorigenesis and metastasis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xueqian Liu, Dong Jiang, Yang Liu, Kun Xie, Yijun Zhao, Fubao Liu
          </td>
          <td>2024-11-25</td>
          <td>CytoJournal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer is known as one of the most predominant cancers that affect both females and
males worldwide. The most crucial risk factor in breast cancer is the mutations in the RAD51C
gene that have been considered in most hereditary breast cancers. RAD51C, the RAD51
paralogs, is also a deoxyribonucleic acid (DNA) repair protein related to breast and ovarian
cancers. DNA double-strand breaks (DSBs) account for the significant detrimental form of
DNA damage. RAD51C mutants also have been recognized in breast/ovarian cancer patients.
However, the role of the RAD51C protein in hereditary breast cancer and its three-dimensional
(3D) structures remains unclear. Thus, this study was conducted to identify the 3D structure of
RAD51C protein from its amino acid sequences. The homology modeling for the 3D structure
of the RAD51C protein was carried out by using three automated webservers: I-TASSER,
SWISS-MODEL, and Phyre2. PyMOL was applied to visualize the 3D structure of RAD51C
protein. Next, the MolProbity, ProSA, and SAVES v6.0 programs have been employed to
check the stereo-chemical quality of RAD51C protein. The RAD51C-IT models were found to
be the best models for the RAD1C protein after being evaluated and validated, and the models
were constructed using full-length RAD51C protein sequences. Thus, these protein models can
be utilized as a virtual screening tool in discovering potential inhibitors of RAD51C protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be054c5f435a476d5df616bc8bacb7f34a1b8877" target='_blank'>
              Homology Modeling of Human DNA Repair Protein RAD51 Homolog 3 (RAD51C) in Breast Cancer
              </a>
            </td>
          <td>
            Ruzianisra Mohamed
          </td>
          <td>2024-12-30</td>
          <td>International Journal of Pharmaceuticals, Nutraceuticals and Cosmetic Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Retrotransposable elements are implicated in genome rearrangements and gene expression alterations that result in various human disorders. In the current study, we sought to investigate the potential effects of long interspersed elements-1 (LINE-1) overexpression on the integrity and methylation of DNA and on the expression of three major pluripotency factors (OCT4, SOX2, NANOG) during the preimplantation stages of human embryo development. Human MI oocytes were matured in vitro to MII and transfected through intracytoplasmic sperm injection (ICSI) either with an EGFP vector carrying a cloned active human LINE-1 retroelement or with the same EGFP vector without insert as control. The occurrence of retrotransposition events was screened by fluorescent microscopy. The in vitro preimplantation development as well as the methylation, pluripotency, and DNA double-strand breaks (DSBs) of the transfected embryos were examined. LINE-1 retrotransposons gave rise to new retrotransposition events in the transfected embryos. LINE-1 injected embryos were characterized by accelerated asymmetrical cell division, multiple cellular fragments, cleavage arrest, and degeneration. Early OCT4 expression remained unaltered, but cleavage arrest and a high fragmentation rate hindered the expression of SOX2/NANOG at the morula stage. Increased DNA DSBs were observed in cleavage-stage blastomeres, while no methylation changes were detected before the cleavage arrest. Our data provide evidence that LINE-1 retrotransposition in human preimplantation embryos may induce DNA DSBs, while at the same time, it appears to interfere with the expression patterns of pluripotency factors. The morphological, structural, and cleavage abnormalities of the transfected embryos show that aberrant retroelement expression may negatively affect human embryo development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c448bfb57d8b4a52f5e6135b0463d8036afab" target='_blank'>
              LINE-1-Induced Retrotransposition Affects Early Preimplantation Embryo DNA Integrity and Pluripotency
              </a>
            </td>
          <td>
            P. Sakaloglou, L. Lazaros, I. Bouba, S. Markoula, Athanasios Zikopoulos, E. Drakaki, Ismini Anagnostaki, Anastasios Potiris, S. Stavros, A. Gerede, Ekaterini Domali, Peter Drakakis, Theodoros Tzavaras, Ioannis Georgiou
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mismatch repair deficiency leads to high mutation rates and microsatellite instability (MSI-H), associated with immune infiltration and responsiveness to immunotherapies. In early stages, MSI-H tumors generally have a better prognosis and lower metastatic potential than microsatellite-stable (MSS) tumors, especially in colorectal cancer. However, in advanced stages, MSI-H tumors lose this survival advantage for reasons that remain unclear. We developed a syngeneic mouse model of MSI cancer by knocking out the MMR gene Msh2 in the metastatic 4T1 breast cancer cell line. This model mirrored genomic features of MSI-H cancers and showed reduction in metastatic incidence compared to their MSS counterparts. In MSI-H tumors, we observed an enrichment of immune gene-signatures that negatively correlated with metastasis incidence. Importantly, a hybrid epithelial-mesenchymal signature, related to aggressiveness was detected only in metastatic MSI-H tumors which may explain the worse outcomes after recurrence of MSI tumors compared to MSS. Interestingly, we identified immature myeloid cells at primary and metastatic sites in MSI-H tumor-bearing mice, suggesting that MMR deficiency elicits specific immune responses beyond T-cell activation. Significance A novel syngeneic mouse model of MSI cancer demonstrates that the immune system regulates MSI cancer cell dissemination, offering an important tool to model advanced stages of human MSI-driven disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4d06352a13d99fbaae03568dc6632e6da89c31" target='_blank'>
              Effect of MisMatch Repair Deficiency on metastasis occurrence modelized in a syngeneic mouse model
              </a>
            </td>
          <td>
            Pierre Laplante, Reginaldo Rosa, L. Nebot-Bral, Jordane Goulas, Sergey I. Nikolaev, Aymeric Silvin, P. Kannouche
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d0309ba86fc8bda71e7e3770fa64ea2730267d" target='_blank'>
              Nuclear IMPDH2 controls the DNA damage response by modulating PARP1 activity
              </a>
            </td>
          <td>
            L. Espinar, Marta Garcia-Cao, Alisa Schmidt, S. Kourtis, Antoni Gañez Zapater, Carla Aranda-Vallejo, R. Ghose, Laura García-López, Ilir Sheraj, N. Pardo-Lorente, Marina Bantula', Laura Pascual-Reguant, Evangelia Darai, M. Guirola, Joan Montero, S. Sdelci
          </td>
          <td>2024-11-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile genomic elements constituting a big fraction of eukaryotic genomes. They ignite an evolutionary arms race with host genomes, which in turn evolve strategies to restrict their activity. Despite being tightly repressed, TEs display precisely regulated expression patterns during specific stages of mammalian development, suggesting potential benefits for the host. Among TEs, the long interspersed nuclear element (LINE-1 or L1) has been found to be active in neurons. This activity prompted extensive research into its possible role in cognition. So far, no specific cause-effect relationship between L1 retrotransposition and brain functions has been conclusively identified. Nevertheless, accumulating evidence suggests that interactions between L1 RNAs and RNA/DNA binding proteins encode specific messages that cells utilize to activate or repress entire transcriptional programs. We summarize recent findings highlighting the activity of L1 RNAs at the non-coding level during early embryonic and brain development. We propose a hypothesis suggesting a mutualistic relationship between L1 mRNAs and the host cell. In this scenario, cells tolerate a certain rate of retrotransposition to leverage the regulatory effects of L1s as non-coding RNAs on potentiating their mitotic potential. In turn, L1s benefit from the cell's proliferative state to increase their chance to mobilize.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc6d372a85f4508f7413dd971b49c4cc08587c1" target='_blank'>
              From the genome's perspective: Bearing somatic retrotransposition to leverage the regulatory potential of L1 RNAs.
              </a>
            </td>
          <td>
            D. Mangoni, Aurora Mazzetti, Federico Ansaloni, Alessandro Simi, Gian Gaetano Tartaglia, Luca Pandolfini, Stefano Gustincich, R. Sanges
          </td>
          <td>2024-11-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c05acb941cf0a62aa3debf0ac989295949d0f0ac" target='_blank'>
              Brca1 haploinsufficiency promotes early tumor onset and epigenetic alterations in a mouse model of hereditary breast cancer.
              </a>
            </td>
          <td>
            Carman Man Chung Li, Alyssa Cordes, Michael U J Oliphant, S. A. Quinn, Mayura Thomas, L. Selfors, Francesca Silvestri, Nomeda Girnius, Gianmarco Rinaldi, Jason J. Zoeller, H. Shapiro, C. Tsiobikas, Kushali P Gupta, S. Pathania, Aviv Regev, S. C. Kadoch, S. Muthuswamy, Joan S. Brugge
          </td>
          <td>2024-11-11</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) demonstrated lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated to either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf89a175114fc9ad8dc2a85a68772e9b4013140e" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Background: Alongside traditional imaging, current development in non-invasive breast carcinoma (BC) diagnosis, such as cell-free DNA-based technologies, have demonstrated promising potential in BC diagnosis. However, achieving early detection remains a challenge. Micronuclei (MN) are extranuclear bodies containing damaged chromosome segments indicative of genomic instability. Elevated levels of MN-containing erythrocytes have been studied in genotoxicity testing and cancer diagnosis. Here, we sequenced the isolated and purified micronuclei DNA (MN-DNA) from erythrocytes in 2ml peripheral blood using a method we previously developed (WO2021/228246 A1) and identified a series of distinct features in read counts that enabled to detect early-stage BC patients. Methods: We collected a clinical study cohort of 172 BC patients and 216 non-BC controls from 1-2ml peripheral blood. MN-DNA was isolated from erythrocytes and performed whole-genome sequencing. We selected distinctive tumor-associated MN-DNA features identified by genome-wide analysis to differentiate between controls and BC for KEGG pathway enrichment and constructed a classification model based on these features. Results: Genome-wide analysis of MN-DNA revealed significant regions between controls and BC patients. Based on covered genes, we observed significant enrichment in the estrogen signaling pathway. Utilizing these distinct MN- DNA features, we constructed a classification model that achieved an AUC of 96%, with a 95% specificity and 88% sensitivity in the test cohort. The model identified early BC (stage 0-I) and advanced BC (stage II-IV) with sensitivities of 100% or 84%, respectively. Conclusions: Our findings highlight that MN-DNA, a form of cytoplasmic DNA, provides a valuable tool for the accurate detection of BC, particularly in its early stages. The MN-DNA pattern is linked to the estrogen signaling pathway, offering novel insights and mechanisms that differ from current diagnostic approaches. Research sponsor: Timing Biotech. Early Detection of Primary Cancer and Relapse
 Citation Format: Wenjv Mo, Xingyun Yao, Haobo Sun, Honghao Liang, Jie Jin, Xiaowen Ding, Xiaofei Gao. Utilizing breast cancer-specific micronuclei DNA features from erythrocytes for early detection of breast carcinoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c23c26b60e573fd02881dd86d0c7097541abb7d1" target='_blank'>
              Abstract A044: Utilizing breast cancer-specific micronuclei DNA features from erythrocytes for early detection of breast carcinoma
              </a>
            </td>
          <td>
            Wenjv Mo, Xingyun Yao, Haobo Sun, Honghao Liang, Jie Jin, Xiaowen Ding, Xiaofei Gao
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548909e3fc8f58fb6f1caf3e51e4cd2b9742adb3" target='_blank'>
              Unique structural configuration of EV-DNA primes Kupffer cell-mediated antitumor immunity to prevent metastatic progression.
              </a>
            </td>
          <td>
            I. Wortzel, Yura Seo, Ife Akano, Lee Shaashua, G. Tobias, Jakob Hebert, Kyung-A Kim, DooA Kim, Shani Dror, Yanshen Liu, Griffin Campbell Azrak, Michele Cioffi, Kofi Ennu Johnson, Tammy Hennika, Meshulam Zisha Twerski, Alexis Kushner, Robert Math, Yoon Dae Han, Dai Hoon Han, Minsun Jung, Juyeong Park, Soonmyung Paik, Jeon-Soo Shin, Min Goo Lee, Marco Vincenzo Russo, D. Zakheim, Jesse Barnes, Sunjoy Mehta, Katia Manova, R. Schwartz, B. K. Thakur, Nancy Boudreau, Irina R Matei, Haiying Zhang, Simone Sidoli, Jacqueline Bromberg, Yael David, H. Kim, D. Lyden
          </td>
          <td>2024-12-03</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most prevalent forms of cancer globally, and has recently become the leading cause of cancer-related mortality in women. BC is a heterogeneous disease comprising various histopathological and molecular subtypes with differing levels of malignancy, and each patient has an individual prognosis. Etiology and pathogenesis are complex and involve a considerable number of genetic alterations and dozens of alterations in non-coding RNA expression. Non-coding RNAs are part of an abundant family of single-stranded RNA molecules acting as key regulators in DNA replication, mRNA processing and translation, cell differentiation, growth, and overall genomic stability. In the context of breast cancer, non-coding RNAs are involved in cell cycle control and tumor cell migration and invasion, as well as treatment resistance. Alterations in non-coding RNA expression may contribute to the development and progression of breast cancer, making them promising biomarkers and targets for novel therapeutic approaches. Currently, the use of non-coding RNAs has not yet been applied to routine practice; however, their potential has been very well studied. The present review is a literature overview of current knowledge and its objective is to delineate the function of diverse classes of non-coding RNAs in breast cancer, with a particular emphasis on their potential utility as diagnostic and prognostic markers or as therapeutic targets and tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0045975d00155c439ba80ba1b588114b3af277" target='_blank'>
              Non-Coding RNAs in Breast Cancer: Diagnostic and Therapeutic Implications
              </a>
            </td>
          <td>
            Roman Beňačka, Daniela Szabóová, Z. Guľašová, Z. Hertelyová
          </td>
          <td>2024-12-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Human C1q is a multifaceted complement protein whose functions range from activating the complement classical pathway to immunomodulation and promoting placental development and tumorigenesis. It is encoded by the C1QA, C1QB, and C1QC genes located on chromosome 1. C1q, unlike most complement components, has extrahepatic expression by a range of cells including macrophages, monocytes and immature dendritic cells. Its local synthesis under the conditions of inflammation and for the purpose of removal of altered self requires its strict transcriptional regulation. To delve into C1Q transcriptional regulation and unravel potential epigenetic influences, we conducted an in silico analysis utilizing a range of online tools and datasets. Co-expression analysis revealed tight coordination between C1QA, C1QB, and C1QC genes. Strikingly, distinct epigenetic patterns emerged across various cell types expressing or lacking these genes, with unique histone marks and DNA methylation status characterizing their regulatory landscape. Notably, the investigation extended to tumor contexts, unveiled potential epigenetic roles in malignancies. The cell type and tumor-specific histone modifications and chromatin accessibility patterns underscore the dynamic nature of epigenetic regulation of C1Q, providing crucial insights into the intricate mechanisms governing the expression of these immunologically significant genes. The findings provide a foundation for future investigations into targeted epigenetic modulation, offering insights into potential therapeutic avenues for immune-related disorders and cancer mediated via C1q.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a922c9cb297a68f52182376903f0bf14881a5d4e" target='_blank'>
              Epigenetic regulation of complement C1Q gene expression
              </a>
            </td>
          <td>
            S. Pegoraro, A. Balduit, A. Mangogna, Uday Kishore, G. Ricci, C. Agostinis, R. Bulla
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent type of liver cancer. Dosage suppressor of NNF1 (DSN1), a component of the MIS12 kinetochore complex, encodes a kinetochore protein crucial for proper mitotic assembly. The role of DSN1 in HCC remains to be elucidated. In this study, we utilized The Cancer Genome Atlas, the Hepatocellular carcinoma Cell Database, and other databases to analyze DSN1 expression and prognosis in samples from patients with HCC. We investigated the signaling pathways regulated by DSN1 and their implications in HCC. Additionally, we engineered siRNAsiRNA/shRNA/shRNA and overexpression vectors for DSN1 and assessed the specific mechanisms of regulatory pathways of DSN1 in hepatoma cell lines and subcutaneous tumor xenograft model. Our findings revealed that DSN1 expression was significantly upregulated in patients with HCC, correlating with decreased survival rates. Elevated DSN1 expression led to the overproduction of cell cycle-related proteins through direct interaction with Centromere Protein T. This interaction contributes to chromosomal instability in patients with HCC, resulting in an aberrant cell cycle and fostering the development and progression of HCC. Increased DSN1 expression is pivotal in HCC initiation and progression. Investigating DSN1 offers valuable insights into the pathogenesis, treatment, and prevention of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162c8061c9e0cfc53558ac673f085aa56ba63f7e" target='_blank'>
              DSN1 Interaction With Centromere-Associated Proteins Promotes Chromosomal Instability in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Hongrui Zhou, Mengxue Zhang, Jiabing Lian, Ruichang Wang, Zhen-Jun Yang, Jin Wang, Xiuli Bi
          </td>
          <td>2024-11-19</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Medulloblastoma (MB) is a common primary brain cancer in children. The sonic hedgehog (SHH) pathway is indispensable for the normal development of the cerebellum, and MB is often caused by persistent SHH activation owing to mutations in pathway components. Patched1 (PTCH1) is the primary receptor for the SHH ligand and a negative regulator of the SHH signal transduction pathway. Mice heterozygous for the Ptch1 gene (Ptch1+/−) are predisposed to MB development. Irradiation of newborn Ptch1+/− mice dramatically increases MB occurrence. A genetic background carrying the Ptch1 mutation significantly influences the risk of developing MB. This study aims to investigate the genetic background-related mechanisms that regulate radiation-induced cellular response and oncogenesis in the cerebellum. We employed multiple approaches, including: (a) analysis of cellular radiosensitivity in granule cell precursors (GCPs), the MB cells of origin, derived from Ptch1 mice with a genetic background that is sensitive (CD1) or resistant (C57Bl/6) to the induction of radiogenic MB; (b) identification of genes differentially expressed in spontaneous and radiation-induced MBs from these two mouse strains; (c) bioinformatic analysis to correlate the expression of radiation-induced genes with survival in MB patients; and (d) examining the expression of these genes in ex vivo MBs induced by single or repeated radiation doses. We have identified a potential gene expression signature—Trp53bp1, Bax, Cyclin D1, p21, and Nanog—that influences tumor response. In ex vivo cultured spontaneous MBs, the expression levels of these genes increase after irradiation in CD1 mice, but not in mice with a C57Bl/6 genetic background, suggesting that this signature could predict tumor response to radiation therapy and help develop strategies for targeting DNA damage repair in tumors. A detailed understanding of the mechanisms behind genetic background-related susceptibility to radiation-induced oncogenic responses is crucial for translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19adfa1196ccdabbcc31c191cdbb05c4c3e4c33" target='_blank'>
              Dissecting the Impact of Genetic Background on Oncogenic Response to Radiation Exposure in the Ptch1+/− Mouse Model
              </a>
            </td>
          <td>
            B. Tanno, Emiliano Fratini, S. Leonardi, Flavia Novelli, Valentina Pisano, Mariateresa Mancuso, S. Pazzaglia
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hematopoietic malignancies emerge through the gradual acquisition of genetic mutations within hematopoietic stem and progenitor cells (HSPCs). Mutations that occur early in disease progression impart a selective growth advantage to HSPCs, which allows them to expand and contribute to a substantial percentage of mature blood cells. This increased expansion is termed clonal hematopoiesis (CH) and is a preleukemic phase associated with an increased risk of developing leukemia. Inhibitor of DNA binding 1 (ID1) protein is a transcriptional regulator of proliferation/differentiation of neuronal, muscle, hematopoietic and other cells, and is frequently overexpressed in cancer. Id1 is expressed at low levels in normal HSCs and is induced by growth factors and other mediators of inflammatory stress and promotes HSPC proliferation in vitro and in vivo. Since chronic inflammation is associated with the progression of hematopoietic malignancies, reducing Id1 expression during CH may be therapeutic. Mutations in TET2 are frequently observed in patients with CH, and Tet2−/−mice develop CH that progress to hematopoietic malignancies. Id1 is upregulated in murine Tet2−/− HSPCs and in AML, CMML and MDS patient samples with TET2 mutations. Genetic ablation of Id1 in Tet2−/− HSPCs reduces HSPC expansion/self-renewal/CH, extramedullary hematopoiesis, myeloid skewing, genetic instability and delays the onset of disease. Mechanistically, p16 expression, senescence and apoptosis were increased and proliferation decreased in Tet2−/−; Id1−/− HSPCs. Thus, ID1 may represent a potential therapeutic target to reduce CH, hematopoietic hyperplasia, and delay the onset of disease. One Sentence Summary Genetic ablation of Id1 in Tet2−/− mice rescues clonal hematopoiesis by increasing CDKI expression, apoptosis, senescence, and differentiation, and reducing cell growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3312a0ebaea58379a205a9959b262e76e84831" target='_blank'>
              Id1 Promotes Clonal Hematopoiesis in Mice with Tet2 Loss of Function
              </a>
            </td>
          <td>
            Shweta Singh, Tanmoy Sarkar, K. Gudmundsson, Brad L. Jakubison, Holly Morris, Sandra Burkett, Karim Baktiar, Gary T. Pauly, D. Sigano, Joel P. Schneider, Jonathan Keller
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef930bd56fde447e683b8e74f0950f751b9d7434" target='_blank'>
              Reassessing the roles of oxidative DNA base lesion 8-oxoGua and repair enzyme OGG1 in tumorigenesis
              </a>
            </td>
          <td>
            Jing Wang, Chunshuang Li, Jinling Han, Yaoyao Xue, Xu Zheng, Ruoxi Wang, Z. Radák, Y. Nakabeppu, Istvan Boldogh, Xueqing Ba
          </td>
          <td>2025-01-01</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer.
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, , Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="BACKGROUND
The TMEM106B protein is critical for proper functioning of the endolysomal system, which is utilised by all cells to traffic and degrade molecular cargo. Genome-wide association studies identified a haplotype in the TMEM106B gene that is associated with increased risk for Alzheimer's disease (AD), amyotrophic lateral sclerosis (ALS), and frontotemporal lobar degeneration with TAR DNA binding protein inclusions (FTLD-TDP). However, the causal variant that drives the association has thus far remained elusive.


METHODS
We generated long-read whole-genome sequencing data of 256 individuals, primarily from Dutch descent. We characterized SNPs and larger structural variants in the TMEM106B locus using de novo genome assembly.


RESULTS
We identified an insertion of an AluYb8 retrotransposon in the 3' UTR of TMEM106B gene, that was in complete linkage with the TMEM106B risk-SNP. AluYb8 retrotransposons have the propensity to propagate through our genomes by utilising a 'copy-paste' mechanism, and once integrated can disrupt transcription and translation of nearby genes. However, propagation of retrotransposons can be suppressed by methylation of the insert and its surrounding regions. Indeed, we observed that the risk haplotype with the AluYb8 insertion, but not the protective haplotype, accumulated CpG islands over evolutionary time. Notably, we observed similar retrotransposon insertions in the 3' UTR of TMEM106B orthologs in non-primate species. This suggests a survival advantage, which may be explained by recent findings that TMEM106B is an entry-receptor for specific viruses in lung-tissue, such that SINE-mediated downregulation of TMEM106B may limit viral infection-load across species.


CONCLUSIONS
We speculate that AluYb8-mediated downregulation of TMEM106B may be protective at younger ages in lung tissues, but that at advanced ages its downregulation in the brain may contribute to increased risk of neurodegenerative diseases. Furthermore, next to the supression of AluYb8 activation by DNA methylation, it may also be suppressed by TDP-43, in its role in post-translational RNA-processing. This leads us to further speculate that age-related demethylation and age-related dysregulation of TDP-43 may result in a negative feedback loop that ultimately reduces the endolysosomal activity in cells. We argue that such a mechanism would explain why increased age is among the strongest risk factors of neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0ddbe2d5eb34ffe1a771f4250ca1d7cd8781a2" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Alex N Salazar, N. Tesi, Y. Pijnenburg, S. van der Lee, Lydian Knoop, Sanduni Wijesekera, Jana Krizova, Mikko Hiltunen, Markus Damme, L. Petrucelli, M. Reinders, M. Hulsman, H. Holstege
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32d455915f14c9cee5327c8bd03503f5d7e8b33" target='_blank'>
              SNAI1 promotes epithelial-mesenchymal transition and maintains cancer stem cell-like properties in thymic epithelial tumors through the PIK3R2/p-EphA2 Axis
              </a>
            </td>
          <td>
            H. E, Lei Zhang, Zhenhua Yang, Long Xu, Tao Wang, Junhong Guo, Lang Xia, Juemin Yu, Heyong Wang, Y. She, Junqi Wu, Yue Zhao, Chang Chen, Deping Zhao
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Epigenetic alterations in gene expression have been implicated in cancer development and tumor immune escape, with posttranslational histone or non-histone modifications representing attractive targets for disease surveillance and therapy. SET domain bifurcated 1 (SETDB1) is a histone lysine methyltransferase that reversibly catalyzes the di- and tri-methylation of histone 3 lysine 9 (H3K9) on euchromatin, inhibiting gene transcription within these regions and facilitating the switch from euchromatic to heterochromatic states. Emerging evidence suggests that SETDB1 amplification and aberrant activation are significantly associated with poor prognosis in hepatocellular carcinoma (HCC), and contribute to HCC development, immune escape, and immune checkpoint blockade (ICB) resistance. Here, we provide an updated overview of the cellular and molecular effects of SETDB1 activity in hepatocarcinogenesis and progression and focus on studies linking its function to immunotherapy for HCC, and present current challenges and future perspectives for targeting SETDB1 in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775e9dd157e3b9dd836833865f11fe90ae06aaa7" target='_blank'>
              The function of histone methyltransferase SETDB1 and its roles in liver cancer
              </a>
            </td>
          <td>
            Enxiang Zhang, Pingping He
          </td>
          <td>2024-11-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA replication represents a series of precisely regulated events performed by a complex protein machinery that guarantees accurate duplication of the genetic information. Since DNA replication is permanently faced by a variety of exogenous and endogenous stressors, DNA damage response, repair and replication must be closely coordinated to maintain genomic integrity. HROB has been identified recently as a binding partner and activator of the Mcm8/9 helicase involved in DNA interstrand crosslink (ICL) repair. We identified HROB independently as a nuclear protein whose expression is co-regulated with various DNA replication factors. Accordingly, the HROB protein level showed a maximum in S phase and a downregulation in quiescence. Structural prediction and homology searches revealed that HROB is a largely intrinsically disordered protein bearing a helix-rich region and a canonical oligonucleotide/oligosaccharide-binding-fold motif that originated early in eukaryotic evolution. Employing a flow cytometry Förster resonance energy transfer (FRET) assay, we detected associations between HROB and proteins of the DNA replication machinery. Moreover, ectopic expression of HROB protein led to an almost complete shutdown of DNA replication. The available data imply a function for HROB during DNA replication across barriers such as ICLs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/063ed9d559a592d4694044ef441583042ee83050" target='_blank'>
              HROB Is Implicated in DNA Replication
              </a>
            </td>
          <td>
            Julia Kutz, Hannes Schmietendorf, Sheikh Anika Rahman, Franz Opel, Helmut Pospiech
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6c4cb49f91070291b1cb8d9f3de6129d9c0bd5" target='_blank'>
              Base-excision repair pathway shapes 5-methylcytosine deamination signatures in pan-cancer genomes
              </a>
            </td>
          <td>
            André Bortolini Silveira, Alexandre Houy, Olivier Ganier, Begüm Özemek, Sandra Vanhuele, A. Vincent-Salomon, N. Cassoux, Pascale Mariani, G. Pierron, Serge Leyvraz, Damian Rieke, Alberto Picca, Franck Bielle, M. Yaspo, M. Rodrigues, Marc-Henri Stern
          </td>
          <td>2024-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Prostate cancer is a malignant tumor caused by the malignant proliferation of epithelial cells, which is highly heterogeneous and drug-resistant, and neuroendocrine prostate cancer (NEPC) is an essential cause of drug resistance in its late stage. Elucidating the evolution of NEPC and the resistance process of enzalutamide, a novel antiandrogen, will be of great help in improving the prognosis of patients. As a research hotspot in the field of molecular biology in recent years, the wide range of biological functions of long noncoding RNAs (lncRNAs) has demonstrated their position in the therapeutic process of many diseases, and a large number of studies have revealed their critical roles in tumor progression and drug resistance. Therefore, elucidating the involvement of lncRNAs in the formation of NEPCs and their interrelationship with enzalutamide resistance may provide new ideas for a deeper understanding of the development of this disease and the occurrence of enzalutamide resistance and give a new direction for reversing the therapeutic dilemma of advanced prostate cancer. This article focuses on lncRNAs that regulate enzalutamide resistance and the neuroendocrine transition of prostate cancer through epigenetic, androgen receptor (AR) signaling, and non-AR pathways that act as “molecular sponges” interacting with miRNAs. Some insights into these mechanisms are used to provide some help for subsequent research in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50743b295e6011754fbae884a2dd3b94e2ad534d" target='_blank'>
              Long noncoding RNA mediates enzalutamide resistance and transformation in neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Zhe Zhu, Wenjing Xuan, Chaohui Wang, Chancan Li
          </td>
          <td>2024-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10-10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5'-TGT*-3' motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential.IMPORTANCEYarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica.
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and environmental microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Motivation Centromeres are chromosomal regions historically understudied with sequencing technologies due to their repetitive nature and short-read mapping limitations. However, recent improvements in long-read sequencing allow for the investigation of complex regions of the genome at the sequence and epigenetic levels. Results Here, we present Centromere Dip Region (CDR)-Finder: a tool to identify regions of hypomethylation within the centromeres of high-quality, contiguous genome assemblies. These regions are typically associated with a unique type of chromatin containing the histone H3 variant CENP-A, which marks the location of the kinetochore. CDR-Finder identifies the CDRs in large and short centromeres and generates a BED file indicating the location of the CDRs within the centromere. It also outputs a plot for visualization, validation, and downstream analysis. Availability and implementation CDR-Finder is available at https://github.com/EichlerLab/CDR-Finder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8a2c873ad9f738bfcea9319867ca10efdb76a5" target='_blank'>
              Identification and annotation of centromeric hypomethylated regions with CDR-Finder
              </a>
            </td>
          <td>
            Francesco Kumara Mastrorosa, Keisuke K. Oshima, Allison N. Rozanski, William T. Harvey, E. Eichler, Glennis A. Logsdon
          </td>
          <td>2024-12-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Deoxyribose nucleic acid (DNA) methylation is an important epigenetic modification that plays an important role in the occurrence and development of tumors. Identifying key methylation-driven genes that affect the prognosis of lung squamous cell carcinoma (LUSC) can provide direction for targeted therapy research.


METHODS AND RESULTS
Methylation and RNA-seq data were downloaded from The Cancer Genome Atlas (TCGA). The MethylMix package was used to integrate and analyze the methylation and gene expression data from TCGA, and the LUSC dataset (GSE37745) was downloaded from GEO for validation. Forty-five DNA-methylation-driven genes (MDGs) were obtained, and 3 genes (TRIM61, SMIM22, and ALDH7A1) were significantly associated with survival by using univariate and multivariate Cox regression. A risk model was constructed. KM analysis showed that patients with high-risk scores had poor survival. A nomination plot for prognosis prediction of LUSC patients was constructed, which showed a good predictive efficiency for tumor prognosis. The high expression of ALDH7A1 was an independent risk factor for poor prognosis in LUSC. The expression of ALDH7A1 in LUSC was negatively correlated with its methylation status (COR = -0.655). GSEA analysis showed that high expression of ALDH7A1 could activate multiple signaling pathways (JAK-STAT signaling pathway and mTOR signaling pathway). In vitro cell experiments confirmed that in LUSC, silencing ALDH7A1 could inhibit tumor progression, while overexpression of ALDH7A1 could promote tumor progression.


CONCLUSION
Our results indicated that ALDH7A1, a newly discovered MDG in LUSC, could act as an independent prognostic factor for OS in LUSC, with the potential to become a potential target for LUSC diagnosis and treatment. High expression of ALDH7A1 in LUSC could promote the occurrence and development of tumors. Signaling pathways, such as JAK-STAT and mTOR signaling pathways, might regulate the high expression of ALDH7A1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa4fa0f5654e845798a1c36db8d638da005aaf3" target='_blank'>
              Identification of ALDH7A1 as a DNA-methylation-driven gene in lung squamous cell carcinoma.
              </a>
            </td>
          <td>
            Gaofeng Liang, Jinxian He, Tian Chen, Liang Zhang, Kaizhong Yu, Weiyu Shen
          </td>
          <td>2024-12-23</td>
          <td>Annals of medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Metabolic reprogramming from oxidative respiration to glycolysis is generally considered to be advantageous for tumor initiation and progression. However, we found that breast cancer cells forced to perform glycolysis acquired a vulnerability to PARP inhibitors. Small-molecule inhibition of mitochondrial respiration-using glyceollin I, metformin, or phenformin-induced overproduction of the oncometabolite lactate, which acidified the extracellular milieu and repressed the expression of homologous recombination (HR)-associated DNA repair genes. These serial events created so-called "BRCAness," in which cells exhibit an HR deficiency phenotype despite lacking germline mutations in HR genes such as BRCA1 and BRCA2, and, thus, sensitized the cancer cells to clinically available poly(ADP-ribose) polymerase inhibitors. The increase in lactate repressed HR-associated gene expression by decreasing histone acetylation. These effects were selective to breast cancer cells; normal epithelial cells retained HR proficiency and cell viability. These mechanistic insights into the BRCAness-prone properties of breast cancer cells support the therapeutic utility and cancer cell-specific potential of mitochondria-targeting drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d552f07898d1f01f3a3d3064ba56a79d2a0073d0" target='_blank'>
              Metabolically inducing defects in DNA repair sensitizes BRCA-wild-type cancer cells to replication stress.
              </a>
            </td>
          <td>
            Kenji Watanabe, Tatsuro Yamamoto, Tomoko Fujita, Shinjiro Hino, Yuko Hino, K. Yamazaki, Y. Ohashi, Shun Sakuraba, H. Kono, M. Nakao, Koji Ochiai, Shingo Dan, Noriko Saitoh
          </td>
          <td>2024-11-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mutations in the histone methyltransferase EZH2, particularly the Y641 hotspot mutation, have been implicated in hematologic malignancies, yet the effect of timing and cellular context on their oncogenic potential has remained unknown. In this study, we utilized a conditional allele with tissue-specific Cre drivers to investigate the effects of Ezh2Y641F mutations at various stages of development, with a focus on the hematopoietic system. We found that ubiquitous heterozygous Ezh2Y641F expression at birth, or conditional expression in hematopoietic or mesenchymal stem cells, led to decreased survival due to hematopoietic defects and bone marrow failure, with no evidence of malignancy. In contrast, Ezh2Y641F expression in committed B cells drives lymphoma formation, highlighting the lineage-specific oncogenic activity of the mutation. Transcriptomic analysis of B cell progenitors revealed key pathway alterations between Cre models such as altered IL2-Stat5 signaling pathway, differential expression of E2F targets, and altered GTPase pathway expression driven by upregulation of Guanylate Binding Proteins (GBPs) in Mx1-Cre Ezh2Y641F pro-B cells. We further found that the GBP locus is regulated by Ezh2-mediated H3K27me3, it is associated with poorer survival in Acute Myeloid Leukemia patients and has variable effects on apoptosis in human lymphoma and leukemia cell lines. These findings suggest that the Ezh2Y641F mutation may alter immune regulatory pathways, cell differentiation and apoptosis, with potential implications for disease progression. Our results highlight the critical role of mutation timing and cellular context in EZH2-driven hematopoietic disease, resulting in distinct downstream changes that shape the oncogenic impact of EZH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58373fa943a83c479e1a45c9450b16a84a7d051a" target='_blank'>
              Developmental Stage and Cellular Context Determine Oncogenic and Molecular Outcomes of Ezh2Y641F Mutation in Hematopoiesis
              </a>
            </td>
          <td>
            Sarah M. Zimmerman, Samantha J. Procasky, Sofia R. Smith, Jie-Yu Liu, Chad Torrice, George P. Souroullas
          </td>
          <td>2024-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceda101cbd27e107d725418d6adeff717071b560" target='_blank'>
              RTEL1 is upregulated in gastric cancer and promotes tumor growth
              </a>
            </td>
          <td>
            Chunyu Yang, Suzeng Wang, Ge Gao, Peiwen Xu, Mengyuan Qian, Yuan Yin, Surui Yao, Zhaohui Huang, Zehua Bian
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Methyltransferase-like 3 (METTL3) is a primary RNA methyltransferase that catalyzes N6-methyladenosine (m6A) modification. The current study aims to further delineate the effect and mechanism of METTL3 in hepatocellular carcinoma (HCC). By using a murine model of hepatocellular cancer development induced via hydrodynamic tail vein injection, we showed that METTL3 enhanced HCC development. In cultured human HCC cell lines (Huh7 and PLC/PRF/5), we observed that stable knockdown of METTL3 by shRNA significantly decreased tumor cell proliferation, colony formation and invasion, in vitro. When Huh7 and PLC/PRF/5 cells with shRNA knockdown of METTL3 were inoculated into the livers of SCID mice, we found that METTL3 knockdown significantly inhibited the growth of HCC xenograft tumors. These findings establish METTL3 as an important oncogene in HCC. Through N6-methyladenosine-sequencing (m6A-Seq), RNA sequencing (RNA-Seq) and subsequent validation studies, we identified BMI1 and RNF2, two key components of the polycomb repressive complex 1 (PRC1), as direct downstream targets of METTL3-mediated m6A modification in HCC cells. Our data indicated that METTL3 catalyzed m6A modification of BMI1 and RNF2 mRNAs which led to increased mRNA stability via the m6A reader proteins IGF2BP1/2/3. Furthermore, we showed that the METTL3 inhibitor, STM2457, significantly inhibited HCC cell growth in vitro and in mice. Collectively, this study provides novel evidence that METTL3 promotes HCC development and progression through m6A modification of BMI1 and RNF2. Our findings suggest that the METTL3-m6A-BMI1/RNF2 signaling axis may represent a new therapeutic target for the treatment of HCC. Implications: The METTL3-m6A-BMI1/RNF2 signaling axis promotes HCC development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2828db6fb6b4ab3788a6525cfb5da2199636a58" target='_blank'>
              METTL3-mediated m6A modification regulates the polycomb repressive complex 1 (PRC1) components BMI1 and RNF2 in hepatocellular carcinoma cells.
              </a>
            </td>
          <td>
            Tong Wu, Weina Chen, Jinqiang Zhang, Wenbo Ma, Nianli Liu
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f2150cfbdcf9bf561de7bbb85cf14321344131" target='_blank'>
              Extrajunctional CLDN10 cooperates with LAT1 and accelerates clear cell renal cell carcinoma progression
              </a>
            </td>
          <td>
            Akifumi Onagi, K. Sugimoto, M. Kobayashi, Yumi Sato, Yasuyuki Kobayashi, Kei Yaginuma, Satoru Meguro, Seiji Hoshi, Jyunya Hata, Yuko Hashimoto, Yoshiyuki Kojima, Hideki Chiba
          </td>
          <td>2024-12-05</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aging constitutes complex and dynamic alterations in molecular and physiological processes and is associated with numerous disorders, in part due to increased genetic instability. The aging population is projected to double by 2050, underscoring the urgent need to better understand the relationships between aging and age-related disorders. Repetitive DNA elements are intrinsic sources of genetic instability and have been found to co-localize with mutation hotspots in human cancer genomes. In this study, we explored the relationship between aging and DNA repeat-mediated genetic instability in vivo using an H-DNA-forming mirror-repeat sequence from the cancer-associated human c-MYC gene. Utilizing a unique mutation-reporter mouse model, we observed tissue-specific effects of aging on H-DNA-induced genetic instability, with mutation frequencies increasing in spleen tissues and remaining unchanged in testis tissues. Analysis of the mutation spectra revealed large deletion mutations as the primary contributor to increasing H-DNA-induced mutations, supported by increased cleavage activity of H-DNA structures in aged spleen tissues. Our findings demonstrate that aging has distinct tissue-specific effects on repeat-mediated, cancer-associated mutations, providing insights into the complex relationship between aging and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8066191467ac61ad7ea1a2f5a55dbdea73e14d68" target='_blank'>
              Tissue-Specific Effects of Aging on Repeat-Mediated Mutation Hotspots In Vivo
              </a>
            </td>
          <td>
            Alexandra M. D’Amico, Tonia T. Li, Karen M. Vasquez
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 This abstract is being presented as a short talk in the scientific program. A full abstract is printed in the Proffered Abstracts section (PR016) of the Conference Program/Proceedings.
 Citation Format: Nicholas A. Vulpescu, Zachariah H. Foda, Pieter H.A. Wisse, Christopher Cherry, Jaime E. Medina, Vilmos Adleff, Remond J.A. Fijneman, Robert B. Scharpf, Gerrit A. Meijer, Beatriz Carvalho, Victor E. Velculescu. Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B067.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149abff6f3105222c01e116a50cadf7efa8009c7" target='_blank'>
              Abstract B067: Dynamic chromatin landscapes of colorectal cancer development and cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, Jaime E. Medina, V. Adleff, R. Fijneman, Robert B Scharpf, Gerrit A Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7746fcc03c2930c5c3eb9fafbd71a061c1f28654" target='_blank'>
              KDM1A epigenetically enhances RAD51 expression to suppress the STING-associated anti-tumor immunity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Qingyuan Yang, Shiyin Wei, Cen Qiu, Chenjie Han, Zunguo Du, Ning Wu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Aneuploidy, arising from gain or loss of chromosomes due to nondisjunction, is a special class of mutation. It can create significant phenotypic changes by altering abundance of hundreds of genes in a single event, providing material for adaptive evolution. But it can also incur large fitness costs relative to other types of mutations. Understanding mutational dynamics of aneuploidy is important for modeling its impact in nature, but aneuploidy rates are difficult to measure accurately. One challenge is that aneuploid karyotypes may revert back to euploidy, biasing forward mutation rate estimates - yet the rate of aneuploidy reversion is largely uncharacterized. Furthermore, current rate estimates are confounded because fitness differences between euploids and aneuploids are typically not accounted for in rate calculations. We developed a unique fluctuation assay in a wild-yeast model to measure the rate of extra-chromosome loss across three aneuploid chromosomes, while accounting for fitness differences between aneuploid and euploid cells. We show that incorporating fitness effects is essential to obtain accurate estimates of aneuploidy rates. Furthermore, the rate of extra-chromosome loss, separate from karyotype fitness differences, varies across chromosomes. We also measured rates in a strain lacking RNA-binding protein Ssd1, important for aneuploidy tolerance and implicated in chromosome segregation. We found no role for Ssd1 in the loss of native aneuploid chromosomes, although it did impact an engineered chromosome XV with a perturbed centromeric sequence. We discuss the impacts and challenges of modeling aneuploidy dynamics in real world situations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f543ecd0458b30a0801433f2e7a5623276dbbac" target='_blank'>
              On the rate of aneuploidy reversion in a wild yeast model.
              </a>
            </td>
          <td>
            James Hose, Qi Zhang, Nathaniel P. Sharp, A. Gasch
          </td>
          <td>2024-11-25</td>
          <td>Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Abstract Background Lung adenocarcinoma is the most common type of lung cancer, accounting for approximately 40% of all lung cancer cases, and has the highest incidence among lung cancer subtypes. Recent studies have suggested that long non-coding RNAs (lncRNAs) play a crucial role in the initiation and progression of lung adenocarcinoma. Methods Based on integrative analysis through databases, we screened Long intergenic non-protein coding RNA 00839 (LINC00839) as one of the most highly upregulated lncRNAs in lung adenocarcinoma. In vitro and in vivo experiments demonstrated that LINC00839 promotes lung adenocarcinoma proliferation, migration, and invasion and that it is present in exosomes secreted by lung adenocarcinoma cells. Results In the cytoplasm, LINC00839 regulates the Toll-like receptor 4 (TLR4)/NF-κB signaling pathway by acting as a molecular sponge of miR-17-5p, thereby influencing the biological behavior of lung adenocarcinoma cells. LINC00839 binds to Polypyrimidine tract binding protein 1 (PTBP1) in the nucleus to regulate the nuclear translocation of NF-κB p65 molecules and, consequently, the transcription of downstream molecules. Conclusions Our study confirmed that LINC00839 promotes the biological progression of lung adenocarcinoma by performing dual roles in the cytoplasm and nucleus to co-regulate the NF-κB signaling pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46547bc3e4e21d3caf821f6cee47fe9308f4d352" target='_blank'>
              Exosomal long non-coding RNA-LINC00839 promotes lung adenocarcinoma progression by activating NF-κB signaling pathway
              </a>
            </td>
          <td>
            Yue Sun, Fang Zhen, Hongyi Wang, Xiao Liang, Yaru Wang, Feiran Wang, Jing Hu
          </td>
          <td>2024-11-25</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="lncRNAs (long non-coding RNAs) are heterogeneous RNA molecules that modulate various cellular processes, such as proliferation, differentiation, migration, invasion, and apoptosis, via different mechanisms. An increasing amount of research indicates that abnormal expression of lncRNA influences the development of drug resistance as well as the genesis and advancement of cancer, including melanoma. Furthermore, they are attractive biomarkers for non-invasive cancer diagnostics due to their strongly modulated expression and improved tissue and disease specificity. This review offers a succinct overview of the present understanding concerning the potential diagnostic biomarker potential of lncRNAs in melanoma. Cell death occurs frequently during growth and throughout life and is an active, organized, and genetically determined process. It is essential for the regulation of homeostasis. Controlled cell death and non-programmed cell death are both forms of cell death. The most prevalent forms of regulatory cell death are pyroptosis, ferroptosis, autophagy, necroptosis, necrosis, and apoptosis. Ferroptosis, pyroptosis, and autophagy are less common forms of cell death compared to necrosis, apoptosis, and necroptosis. ncRNAs are regulatory RNA molecules that are not involved in encoding proteins. They primarily consist of circular RNAs (circ RNAs), lncRNAs, and microRNAs (miRNAs). Moreover, non-coding RNAs have the ability to modulate tumor cell autophagy, pyroptosis, and ferroptosis at the transcriptional or post-transcriptional stage, as well as function as oncogenes and tumor suppressor genes, which can have considerable effects on the incidence and growth of tumors. This review concentrated on the recent advancements in the research of the diagnostic and therapeutic functions of ncRNAs in the regulation of programmed cell death in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c2d5632ba8374554ff55670c724793b6787a625" target='_blank'>
              Diagnostic and therapeutic role of non-coding RNAs regulating programmed cell death in melanoma
              </a>
            </td>
          <td>
            Zixu Wang, Cong Xie, Xiao Chen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b3b6f34b71d4b5f3b8e63dfe568f53d1e1b13e" target='_blank'>
              Stabilization of expandable DNA repeats by the replication factor Mcm10 promotes cell viability
              </a>
            </td>
          <td>
            Chiara Masnovo, Zohar Paleiov, Daniel Dovrat, Laurel K Baxter, Sofia Movafaghi, A. Aharoni, S. Mirkin
          </td>
          <td>2024-12-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f105728d3e05cb655e57590a8ecd77f62627ac7b" target='_blank'>
              Mechanistic basis of atypical TERT promoter mutations
              </a>
            </td>
          <td>
            K. Elliott, Vinod Kumar Singh, Alan Bäckerholm, Linnea Ögren, M. Lindberg, K. Soczek, E. Hoberg, Tom Luijts, Jimmy Van den Eynden, Maria Falkenberg, Jennifer A. Doudna, Anders Ståhlberg, Erik Larsson
          </td>
          <td>2024-11-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="
 Cancer-associated fibroblasts (CAFs) are a heterogenous cell population that can promote esophageal cancer progression through multiple molecular mechanisms including increased cancer cell proliferation and migration, immunosuppression, and therapeutic resistance. Despite the divergence of CAFs from normal fibroblasts with regards to phenotype and functions, CAFs rarely exhibit widespread genetic alterations. Instead, it is proposed that CAFs are regulated at the epigenetic level. Our objective is to delineate this epigenetic regulation of CAFs in esophageal cancer. Ultimately, we aim to identify the epigenetic regulatory complexes that mediate CAF heterogeneity and pro-tumorigenic functions. To this end, we completed RNA-seq and ATAC-seq on fibroblasts isolated from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC) and non-cancerous esophageal tissue. Enrichment analysis identified the polycomb repressive complex 2 (PRC2) as a potential regulator of ESCC and EAC CAFs. PRC2 catalyzes the methylation (mono-, di- or tri-) of lysine 27 on histone 3 (H3K27me1/2/3) resulting in transcriptional repression. Inhibition of two key proteins of PRC2 (EZH2 and SUZ12) through either pharmacological or genetic modulation, altered expression of multiple CAF phenotype related proteins and inhibited CAF functions including CAF proliferation. Ongoing studies, using co-cultures of fibroblasts and tumor cells, seek to identify if inhibition of PRC2 can prevent the initial transition of normal esophageal fibroblasts into CAFs. Importantly, we are conducting in vivo studies to determine if selective PRC2 inhibition in CAFs can inhibit esophageal tumor progression with specific focus on primary tumor growth and metastatic spread. In summary, initial studies have identify that PRC2 complex mediates CAF phenotype and functions, and we aim to establish the larger impact of these CAF specific effects on esophageal cancer.
 Citation Format: Karen J. Dunbar, Raul Navaridas Fernandez de Bobadil, Katherine M. Cunningham, Emily Esquea, Gizem Efe, Anil K. Rustgi. Polycomb repressive complex 2 (PRC2) mediates the epigenetic regulation of cancer-associated fibroblasts in esophageal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d29e0ba845d63b544dd787fbf41b70e6dd633d82" target='_blank'>
              Abstract A030: Polycomb repressive complex 2 (PRC2) mediates the epigenetic regulation of cancer-associated fibroblasts in esophageal cancer
              </a>
            </td>
          <td>
            Karen J Dunbar, Raul Navaridas Fernandez de Bobadil, Katherine Cunningham, Emily Esquea, Gizem Efe, Anil K. Rustgi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 DICER1 cancer predisposition syndrome is a heritable condition where affected patients have increased risk of neoplasms including fusion-negative rhabdomyosarcoma (FN-RMS). The syndrome is defined by germline heterozygous loss of function mutations in DICER1, but it remains unclear exactly how these mutations predispose children to cancer. To gain insight into the role of Dicer1 loss in DICER1 cancer predisposition, we deleted the Dicer1 gene both in the germline or conditionally within the tumor cell in our previously established mouse model of FN-RMS. One would expect the germline loss of one allele to provide a sensitized background in the tumor cell to acquire a second hit mutation. However, we observed faster tumor onset and increased penetrance in mice with germline heterozygous Dicer1 loss (Dicer1 +/- ) but not in mice when Dicer1 loss was restricted to the tumor cells (Dicer1 Flox/+ ), demonstrating that haploinsufficient tumor suppressor activity of Dicer1 loss requires non-cell autonomous cell types. Single cell RNA sequencing (scRNA-Seq) revealed significant expansion of immature neutrophils in Dicer1 +/- tumors, which were enriched for proteases associated with neutrophil extracellular traps (NETs). NETs are webs of chromatin and proteases released when neutrophils in response to inflammatory stimuli in process termed NETosis. We show Dicer1 +/- murine and human tumors are enriched for neutrophils and tumor-bearing mice had abundant circulating NETs. NETs cause significant damage to the local extracellular matrix (ECM) and have been implicated in tumor promotion. In addition, neutrophil enrichment, we found that Dicer1 +/- tumors show significant ECM remodeling and are increased C5a, a potent neutrophil chemoattractant and NET-priming factor. These results suggest NETosis is upregulated in Dicer1 +/- FN-RMS tumors. Ligand-receptor pair analysis (iTALK) on the scRNA- Seq data revealed a putative interaction between a NET-derived ligand and EGFR and IGF1R on tumor cells, suggesting direct tumor promoting signaling from NET-to-tumor. We validated this novel FN-RMS promotion mechanism in vitro using human FN-RMS cell lines and observed that this NET-derived ligand promoted FN-RMS growth through both EGFR and IGF1R signaling. NET release requires citrullination of histone H3 mediated by peptidyl arginine deiminase 4 (PADI4) to facilitate decondensation of chromatin. We intercrossed the Padi4 -/- allele into our tumor model to genetically block NETosis and observed complete rescue of the accelerated tumor onset and increased penetrance observed in Dicer1 +/- mice. These results demonstrate that NETosis promotes the growth of Dicer1 +/- FN-RMS tumors and is required for Dicer1 +/- haploinsufficiency. These findings may be applicable to many DICER1 syndrome- associated cancers, as neutrophils which get converted to tumor promoters by heterozygous DICER1 loss are present in DICER1 syndrome patients regardless of tumor type and suggest targeting neutrophils/NET-release may reduce cancer risk in DICER1 +/- individuals.
 Citation Format: Randolph K Larsen, Jason A Hanna, Hongjian Jin, Kristen B Reed, Darden W Kimbrough, Kyna Voung, Myron K Evans, Casey G Langdon, Catherine J Drummond, Matthew R Garcia, David Finkelstein, Patrick A Schreiner, Jerold E Rehg, Kris Ann P Schultz, Mark E Hatley. DICER1 cancer predisposition tumor promotion mediated non-cell autonomously via neutrophils and NETosis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266771b89aebf534b5e742d8cc356c61afa9ade6" target='_blank'>
              Abstract PR010: DICER1 cancer predisposition tumor promotion mediated non-cell autonomously via neutrophils and NETosis
              </a>
            </td>
          <td>
            Randolph K. Larsen, Jason A. Hanna, Hongjian Jin, Kristin B Reed, Darden W. Kimbrough, Kyna Voung, Myron K Evans, Casey G Langdon, Catherine J Drummond, Matthew R. Garcia, David Finkelstein, Patrick A Schreiner, J. Rehg, Kris Ann P Schultz, M. Hatley
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d82943330ae4bd543dc2a3fa4eed0cbbe56c34e9" target='_blank'>
              TRIM59/RBPJ positive feedback circuit confers gemcitabine resistance in pancreatic cancer by activating the Notch signaling pathway
              </a>
            </td>
          <td>
            Shiyu Chen, Zhiwei He, Kun Cai, Yan Zhang, Hongyan Zhu, Chong Pang, Jiaqi Zhang, Dong Wang, Xundi Xu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract In addition to the four canonical nucleosides dA, dG, dC and T, genomic DNA contains the additional base 5‐methyldeoxycytidine (mdC). The presence of this methylated cytidine nucleoside in promoter regions or gene bodies significantly affects the transcriptional activity of the corresponding gene. Consequently, the methylation patterns of genes are crucial for either silencing or activating genes. Sequencing the positions of mdC in the genome is therefore of paramount importance for early cancer diagnostics as it helps determine incorrect gene expression. Currently, the bisulfite method is the gold standard for mdC‐sequencing. However, this method has the drawback that the majority of the input DNA is degraded during the bisulfite treatment. Additionally, bisulfite sequencing is prone to errors. Here, we report a benign, bisulfite‐free mdC sequencing method termed EMox‐seq, which is based on third‐generation single‐molecule SMRT sequencing. The foundation of this technology is a new Tet3 enzyme that efficiently oxidizes mdCs to 5‐carboxycytidine (cadC). In turn, cadC provides an excellent readout by SMRT sequencing using specially trained AI‐based algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f21a683a431b12c2f63f4ad68b3420a96d3be8" target='_blank'>
              Robust Bisulfite‐Free Single‐Molecule Real‐Time Sequencing of Methyldeoxycytidine Based on a Novel hpTet3 Enzyme
              </a>
            </td>
          <td>
            Hanife Sahin, Raheleh Salehi, Shariful Islam, Markus Müller, Pascal Giehr, Thomas Carell
          </td>
          <td>2024-11-13</td>
          <td>Angewandte Chemie (International Ed. in English)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
The transfer of mitochondrial DNA into the nuclear genomes of eukaryotes (Numts) has been linked to lifespan in non-human species and recently demonstrated to occur in rare instances from one human generation to the next.


METHOD
Here we investigated numtogenesis dynamics in humans in two ways. First, we quantified Numts in 1,187 post-mortem brain and blood samples from different individuals. Second, we tested the dynamic transfer of Numts using a repeated-measures whole genome sequencing design in a human fibroblast model that recapitulates several molecular hallmarks of aging.


RESULT
In the ROSMAP dataset, compared to circulating immune cells (n = 389), post-mitotic brain tissue (n = 798) contained more Numts, consistent with their potential somatic accumulation. Within brain samples we observed a 5.5-fold enrichment of somatic Numt insertions in the dorsolateral prefrontal cortex compared to cerebellum samples, suggesting that brain Numts arose spontaneously during development or across the lifespan. Moreover, more brain Numts was linked to earlier mortality. The brains of individuals with no cognitive impairment who died at younger ages carried approximately 2 more Numts per decade of life lost than those who lived longer. In the lifespan model, the longitudinal experiments revealed a gradual accumulation of one Numt every ∼13 days. Numtogenesis was independent of large-scale genomic instability and unlikely driven cell clonality. Targeted pharmacological perturbations including chronic glucocorticoid signaling or impairing mitochondrial oxidative phosphorylation (OxPhos) only modestly increased the rate of numtogenesis, whereas patient-derived SURF1-mutant cells exhibiting mtDNA instability accumulated Numts 4.7-fold faster than healthy donors.


CONCLUSION
Combined, our data document spontaneous numtogenesis in human cells and demonstrate an association between brain cortical somatic Numts and human lifespan. These findings open the possibility that mito-nuclear horizontal gene transfer among human post-mitotic tissues produces functionally-relevant human Numts over timescales shorter than previously assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576cc9de30e3b7f80b5c6a3ef93ab242e8481166" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Weichen Zhou, Kalpita R. Karan, Wenjin Gu, Hans Klein, Gabriel Sturm, P. L. De Jager, David A Bennett, M. Hirano, Martin Picard, Ryan E Mills
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung cancer (LC) is a leading cause of cancer‐related deathworldwide, and altered cholesterol metabolism is a hallmark of cancer cells. Acyl‐CoA:cholesterol acyltransferase 1(ACAT1), or Sterol O‐acyltransferase 1 (SOAT1), isa key cholesterol esterification enzyme. Its overexpression promotes tumorprogression by accumulating cholesterol esters. Inhibition of ACAT1 also potentiatesCD8+ T cells medicated anti‐tumor immunity by increasing plasma membranecholesterol level. This study, as the first of its kind, shows the ACAT1/SOAT1 overexpressioncorrelates with poor prognosis in early‐stage lung adenocarcinoma (LUAD) patients.Long‐term treatment with ACAT1 inhibitor avasimibe suppresses tumorigenesis inboth Kirsten rat sarcoma viral oncogene homolog (KRAS) and epidermal growthfactor receptor (EGFR) mutation‐induced LC mouse models without overttoxicity. ACAT1 inhibition reduces tumor cell proliferation, migration, andinvasion and causes G0/G1 cell cycle arrest, while boosting CD8+ T cells'effector function and memory phenotype. Single‐cell RNA sequencing reveals thatACAT1 inhibition downregulates cholesterol biosynthesis and central carbon andnitrogen metabolism pathways in tumor cells, while upregulating genes relatedto oxidative phosphorylation and fatty acid oxidation in CD8+ T cells. Finally, avasimibe improves the efficacy of a human EGFR vaccine in preventing LCprogression. These novel findings suggest potential strategies for cancer preventionand therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e334be2ad31ab64ceda2a5bfacd602b4168fd08" target='_blank'>
              Targeting ACAT1 for Precision Chemo‐Immunoprevention in Lung Cancer
              </a>
            </td>
          <td>
            Mofei Huang, Jing Pan, Shizuko Sei, Yian Wang, Ming You
          </td>
          <td>2024-12-23</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ASXL transcriptional regulator 1 (ASXL1) is one of the three most frequently mutated genes in age-related clonal hematopoiesis (CH), alongside DNA methyltransferase 3 alpha (DNMT3A) and Tet methylcytosine dioxygenase 2 (TET2). CH can progress to myeloid malignancies including chronic monomyelocytic leukemia (CMML) and is also strongly associated with inflammatory cardiovascular disease and all-cause mortality in humans. DNMT3A and TET2 regulate DNA methylation and demethylation pathways, respectively, and loss-of-function mutations in these genes reduce DNA methylation in heterochromatin, allowing derepression of silenced elements in heterochromatin. In contrast, the mechanisms that connect mutant ASXL1 and CH are not yet fully understood. CH/CMML-associated ASXL1 mutations encode C-terminally truncated proteins that enhance the deubiquitinase activity of the ASXL-BAP1 "PR-DUB" deubiquitinase complex, which removes monoubiquitin from H2AK119Ub. Here, we show that ASXL1 mutant proteins interact with the euchromatic histone lysine methyltransferases 1 and 2 (EHMT1-EHMT2) complex, which generates H3K9me1 and me2, the latter a repressive modification in constitutive heterochromatin. Compared to cells from age-matched wild-type mice, we found that expanded myeloid cells from old (≥18-mo-old) Asxl1tm/+ mice, a heterozygous knock-in mouse model of CH, display genome-wide decreases of H3K9me2, H3K9me3, and H2AK119Ub as well as an associated increase in expression of transposable elements (TEs) and satellite repeats. Increased TE expression was also observed in monocytes from ASXL1-mutant CMML patients compared to monocytes from healthy controls. Our data suggest that mutant ASXL1 proteins compromise the integrity of both constitutive and facultative heterochromatin in an age-dependent manner by reducing the levels of H3K9me2/3 and H2AK119Ub. This increase in TE expression correlated with increased expression of nearby genes, including many interferon-inducible (inflammation-associated) genes (ISGs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e7123fc0bf81467e6f3de419413862ec10652" target='_blank'>
              A mutant ASXL1-BAP1-EHMT complex contributes to heterochromatin dysfunction in clonal hematopoiesis and chronic monomyelocytic leukemia.
              </a>
            </td>
          <td>
            Zhen Dong, Hugo Sepulveda, Leo J. Arteaga-Vazquez, Chad Blouin, Jenna A. Fernandez, Moritz Binder, Wen-Chien Chou, H. Tien, M. Patnaik, Geoffrey J Faulkner, Samuel A Myers, Anjana Rao
          </td>
          <td>2025-01-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy.
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer refers to malignant tumors occurring in the walls of the colon or rectum. The roles of WD Repeat Domain 12 (WDR12) and mitochondrial ribosome-associated tumor suppressor 4 (MRTO4) genes in colorectal cancer remain unclear. The colorectal cancer dataset GSE113513 configuration file was downloaded from the gene expression omnibus database generated from GPL15207. Differentially expressed genes screening, functional enrichment analysis, gene set enrichment analysis, Weighted Gene Co-expression Network Analysis, construction and analysis of protein–protein interaction networks, survival analysis, and gene expression heatmap plotting were conducted. Comparative toxicogenomics database analysis was performed to find diseases most relevant to core genes. TargetScan was used to screen miRNAs regulating core genes. A total of 3106 differentially expressed genes were identified. According to gene ontology analysis, they mainly enriched in organic acid metabolic processes, condensed chromosome kinetochore, oxidoreductase activity, and cell cycle. In Kyoto encyclopedia of genes and genomes analysis, they primarily concentrated in the cell cycle, TGF-β signaling pathway, Jak-STAT signaling pathway, PI3K-Akt signaling pathway, Ras signaling pathway, TNF signaling pathway, p53 signaling pathway, NF-kB signaling pathway, and WNT signaling pathway. Weighted Gene Co-expression Network Analysis with a soft thresholding power set to 12 generated 29 modules. The protein–protein interaction network identified 6 core genes (DDX27, NAT10, WDR12, DKC1, MRTO4, and NOP56). Survival analysis showed core genes (POSTN, MYH11, LUM, COL6A3, and COL4A1) as risk factors. Gene expression heatmap revealed high expression of core genes (WDR12 and MRTO4) in colorectal samples. Comparative toxicogenomics database analysis linked core genes (WDR12 and MRTO4) with local tumor infiltration, bowel obstruction, abdominal pain, and colorectal neoplasms. WDR12 and MRTO4 genes are highly expressed in colorectal cancer, potentially influencing its progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3eaef691d4ef44bd15d0a47a76d9dd081032185" target='_blank'>
              Roles of WDR12 and MRTO4 genes in colorectal cancer
              </a>
            </td>
          <td>
            Huanping Luo
          </td>
          <td>2024-12-27</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We established two types of site-specific DNA double strand breaks (DSB) induction systems to elucidate factors which affect the efficiency or quality of DSB repair. For mutation assays, a site specific DSB was generated by the transient expression of a zinc finger nuclease which targets the human HPRT1 gene. A cell line in which time and site specific DSBs can be generated in a HR reporter construct was used for homology-directed repair (HR) analysis. By using these two systems, we investigated the effects of PI3-kinase inhibitors on the efficiency and quality of DSB repair. Ataxia telangiectasia mutated (ATM) kinase inhibition resulted a decrease in mutant frequency and a slight increase in deletion-type mutations accompanied by microhomology. Furthermore, the HR frequency increased significantly when ATM kinase activity was inhibited. Thus, ATM kinase activity might be involved in the suppression of DSB end resection, and this may promote DSB repair through canonical non-homologous end joining.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77902eba8c12283aa64fb19147cfc9bae88ccb88" target='_blank'>
              Effect of PI3-kinase inhibitors on DNA double strand break repair pathways: observations using a site specific DSB induction system.
              </a>
            </td>
          <td>
            Tomoki Myodo, Yuki Sakamoto, Keita Sato, Honami Kobayashi, K. Iijima, Kenshi Komatsu, Shinya Matsuura, Hiroshi Tauchi
          </td>
          <td>2024-11-01</td>
          <td>Radiation protection dosimetry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The memory of cellular identity is crucial for the correct development of an individual and is maintained throughout life by the epigenome. Chromatin marks, such as DNA methylation and histone modifications, ensure the stability of gene expression programmes over time and through cell division. Loss of these marks can lead to severe pathologies, including cancer and developmental syndromes. However, reprogramming of cellular identity is also a natural phenomenon that occurs early in mammalian development, particularly in the germ line, which enables the production of mature and functional gametes. The germ line transmits genetic and epigenetic information to the next generation, contributing to the survival of the species. Primordial germ cells (PGCs) undergo extensive chromatin remodelling, including global DNA demethylation and erasure of the parental imprints. This review introduces the concept of epigenetic reprogramming, its discovery and key steps, as well as the transcriptional and chromatin changes that accompany germ cell formation in mice. Finally, we discuss the epigenetic mechanisms of genomic imprinting, its discovery, regulation and relevance to human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2611fcc1b3dfdfc6647da0d47d718b5970da542" target='_blank'>
              [Epigenetic reprogramming, germline and genomic imprinting].
              </a>
            </td>
          <td>
            Clara Roidor, Karim Chebli, Maud Borensztein
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) broadly impacts host genes, affecting the infected cell population and inducing the development of a disease with a poor prognosis, adult T-cell leukemia-lymphoma (ATL). This study aimed to provide a comprehensive epigenomic characterization of the infected cell population and evaluated the transcriptome and chromatin structures of peripheral blood cells in HTLV-1-infected individuals using RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq). The infected cells showed significant changes in gene expression patterns from the polyclonal stage and before ATL onset while demonstrating similarities to tumor-forming ATL cells. These similarities were a result of large-scale open chromatin changes, supporting the independent early formation of epigenomic aberrations as an underlying mechanism for later clonal propagation. This study also demonstrated that HTLV-1 Tax directly affects the host chromatin structure, thereby developing fundamental epigenomic characteristics. Several Tax target genes, including the RASGRP3-ERK pathway, were recognized, indicating an impact on signaling pathways. This genome-wide variability in chromatin structural property is a novel feature of HTLV-1 infection and may contribute to pathogenic mechanisms. In addition, it has crucial implications for better understanding the impact of HTLV-1 on the host genome and identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f7c450fea8921cdae499b880a5d0560a4c49aa" target='_blank'>
              Rewired chromatin structure and epigenetic gene dysregulation during HTLV-1 infection to leukemogenesis.
              </a>
            </td>
          <td>
            Jun Mizuike, Kako Suzuki, Shu Tosaka, Yuta Kuze, Seiichiro Kobayashi, Makoto Nakashima, Koji Jimbo, Y. Nannya, Yutaka Suzuki, K. Uchimaru, Makoto Yamagishi
          </td>
          <td>2024-11-19</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6513315e5e6fa9d02f83c4fec4109bb8e64824c" target='_blank'>
              Gliomagenesis mimics an injury response orchestrated by neural crest-like cells.
              </a>
            </td>
          <td>
            Akram A. Hamed, Kui Hua, Quang M Trinh, B. D. Simons, J. Marioni, Lincoln D. Stein, Peter B. Dirks
          </td>
          <td>2025-01-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract In mammals, fertilization is followed by extensive reprogramming and reorganization of the chromatin accompanying the transcriptional activation of the embryo. This reprogramming results in blastomeres with the ability to give rise to all cell types and a complete organism, including extra‐embryonic tissues, and is known as totipotency. Transcriptional activation occurs in a process known as zygotic genome activation (ZGA) and is tightly linked to the expression of transposable elements, including endogenous retroviruses (ERVs) such as endogenous retrovirus with leucine tRNA primer (ERVL). Recent studies discovered the importance of ERVs in this process, yet the race to decipher the network surrounding these elements is still ongoing, and the molecular mechanism behind their involvement remains a mystery. Amid a recent surge of studies reporting the discovery of various factors and pathways involved in the regulation of ERVs, this review provides an overview of the knowns and unknowns in the field, with a particular emphasis on the chromatin landscape and how ERVs shape preimplantation development in mammals. In so doing, we highlight recent discoveries that have advanced our understanding of how these elements are involved in transforming the quiescent zygote into the most powerful cell type in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de59ad5f3d58d084f36e3b3323bf2a35634e22da" target='_blank'>
              The impact of retrotransposons on zygotic genome activation and the chromatin landscape of early embryos
              </a>
            </td>
          <td>
            Therese Solberg, Mie Kobayashi-Ishihara, Haruhiko Siomi
          </td>
          <td>2024-11-22</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In the era of transcriptomics, the role of epigenetics in the study of cancers in females has gained increasing recognition. This article explores the impact of epigenetic modifications, such as DNA methylation, histone modification, and non-coding RNA, on cancers in females, including breast, cervical, and ovarian cancers (1). Our findings suggest that these epigenetic markers not only influence tumor onset, progression, and metastasis but also present novel targets for therapeutic intervention. Detailed analyses of DNA methylation patterns have revealed aberrant events in cancer cells, particularly promoter region hypermethylation, which may lead to silencing of tumor suppressor genes. Furthermore, we examined the complex roles of histone modifications and long non-coding RNAs in regulating the expression of cancer-related genes, thereby providing a scientific basis for developing targeted epigenetic therapies. Our research emphasizes the importance of understanding the functions and mechanisms of epigenetics in cancers in females to develop effective treatment strategies. Future therapeutic approaches may include drugs targeting specific epigenetic markers, which could not only improve therapeutic outcomes but also enhance patient survival and quality of life. Through these efforts, we aim to offer new perspectives and hope for the prevention, diagnosis, and treatment of cancers in females.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cc4082f981f81654c23a63365924b300c8aa1c" target='_blank'>
              Transcriptomic era of cancers in females: new epigenetic perspectives and therapeutic prospects.
              </a>
            </td>
          <td>
            Runhe Zhu, Jiawei Ni, Jiayin Ren, Dongye Li, Jiawei Xu, Xinru Yu, Yingjie Ma, Luan Kou
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="B3GNT5, a critical member of the β-1,3-N-acetylglucosaminyl transferase gene family involved in lactose and glycosphingolipids biosynthesis, has been documented to promote tumor-infiltrating T-cell responses. Our research utilized the Pan-Cancer dataset from The Cancer Genome Atlas (TCGA) to explore the functional role of B3GNT5. Our study demonstrated that the antibody-driven inhibition of B3GNT5 diminished T cell-mediated anti-tumor responses in both in vitro and in vivo settings. By analyzing RNA-seq data from Genotype-Tissue Expression (GTEx) and TCGA databases, we observed differential expression levels of B3GNT5 across various tumor types accompanied by an unfavorable prognostic correlation. We further utilized integrated clinical survival data from TCGA and immune cell infiltration scoring patterns to identify significant associations between B3GNT5 expression and immune checkpoints, cancer stemness, chemokines, chemokine receptors, and immune-activating genes. B3GNT5’s expression was highly correlated with different immunoregulatory factors, including T cell infiltration, chemokine receptors, and activation genes. Subsequent experiments discovered that suppressing B3GNT5 expression in pancreatic adenocarcinoma cells significantly reduced their tumorigenicity by limiting sphere-forming ability and self-renewal capacity, thus underscoring B3GNT5’s vital role as a prognostic factor in immune regulation across pan-cancer. Our findings suggest that B3GNT5 presents a viable target for cancer immunotherapy by enabling effective communication between cancer stem cells and immune cells during tumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5b6436fc5a53eb0b4f39869b6fe79f483d37e9d" target='_blank'>
              Pan-cancer analysis of B3GNT5 with potential implications for cancer immunotherapy and cancer stem cell stemness
              </a>
            </td>
          <td>
            F. Peng, Yechen Feng, Shuo Yu, Ruizhi He, Hebin Wang, Yu Xie, Renyi Qin
          </td>
          <td>2024-12-13</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Orphan DNA adenine methyltransferases (MTases) of Mycobacterium tuberculosis (Mtb) exhibit diversity across clinical isolates, but the forces driving this variation are not entirely clear. Recently, we observed several isolates exhibiting anomalous hypomethylation by Type I MTase Mycobacterial Adenine Methyltransferase C (MamC) despite a wild-type mamC genotype (‘MamC-anomalous’ isolates). Investigating this hypomethylation through multiple analyses revealed three key findings. First, heterogeneity analysis revealed intercellular mosaic methylation (IMM) in MamC-anomalous isolates. While they often exhibit phase-variable heterogeneity, this is the first report of IMM by a prokaryotic Type I MTase. Second, MamC-anomalous isolates exhibited a large, stable difference in chromosome copy number along the replication axis (a proxy for bacterial growth rate), suggesting a distinct growth phase accompanied by MamC hypomethylation. Third, MamC methylation efficiency decreased progressively with distance from origin of replication on both strands, with a marked exaggeration in MamC-anomalous isolates. In contrast, other Mtb MTases (MamA and MamB) exhibited lower methylation levels away from the origin only on the lagging strand, and without exaggeration in MamC-anomalous isolates. We conclude that, among Mtb MTases, MamC is uniquely linked to growth dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55d608751f82425c62948173e78abe1baca136d0" target='_blank'>
              Intercellular mosaic methylation in fast-growing Mycobacterium tuberculosis clinical isolates
              </a>
            </td>
          <td>
            Maryam Ahmadi Jeshvaghane, Samuel J. Modlin, D. Conkle-Gutierrez, Monica E Espinoza, F. Valafar
          </td>
          <td>2024-11-15</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND
DNA repair plays a major role in maintaining genomic stability, thus limiting the transformation of normal cells into cancer cells. However, in cancer patients treated with DNA-targeting drugs, DNA repair can decrease efficacy by removing the damage generated by such molecules that is needed to induce pharmacological activity. Inhibiting DNA repair thus represents an interesting approach to potentiating the activity of chemotherapy in this setting.


OBJECTIVES
Here, we continue the characterization of an inhibitor of the interaction between Excision Repair Cross-Complementing Rrodent repair deficiency complementation group 1 (ERCC1) and Xeroderma Pigmentousum group F (XPF) (B9), two key proteins of nucleotide excision repair.


METHODS
We used various cell lines and co-incubation studies for the determination of cell survival and DNA repair capacities.


RESULTS
We show that it is synergistic with other platinum derivatives than previously described, and that synergy is lacking in cells not expressing ERCC1 or XPF. Finally, a series of experiments show that potentiation is observed only in cells expressing wild-type p53.


CONCLUSION
Our results confirm the mechanism of action of our ERCC1-XPF inhibitor and give important additional data on this approach to enhance the activity of already existing cancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613d6f1a60a48e1ee4460eb20db61d009e7dd40f" target='_blank'>
              The synergy between alkylating agents and ERCC1-XPF inhibitors is p53 dependent.
              </a>
            </td>
          <td>
            G. Ciniero, Tiago Marques Pedro, Charles Dumontet, Ahmed H. Elmenoufy, Frederick G. West, Michael Weinfeld, Francesco Gentile, J. A. Tuszyński, Emeline Cros-Perrial, L. Jordheim
          </td>
          <td>2024-11-08</td>
          <td>Fundamental & clinical pharmacology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Increased extracellular DNA (exDNA) levels in the blood are a hallmark of metastatic cancer, arising from tumor lysis, apoptosis, necrosis, and neutrophil extracellular traps (NETs) released by tumor-expanded neutrophils. NETs, web-like chromatin structures extruded by neutrophils to trap pathogens, are highly immunogenic due to their DNA and histone content and their persistent interaction with dendritic cells. Tumor cells can also release extracellular traps. In myeloid leukemia, blasts release nuclear content via traps to activate coagulation or sustain myeloproliferation. Extracellular traps have been found in NPM1 mutant AML patients, with mutant NPM co-localizing with histones along exDNA traps. Forcing trap extrusion in NPM mutant leukemia cells has been used to create dendritic cell-based vaccines that break tolerance against NPMc antigens. This suggests that NETs in the tumor microenvironment (TME) may promote tolerance, with breaking tolerance requiring consistent NET formation and dendritic cell interaction. NETs are also seen in solid tumors, such as triple-negative breast cancer (TNBC), where they aid in cancer cell migration, invasion, and awakening of circulating tumor cells. However, the capacity of tumor cells to directly extrude DNA traps is not fully understood. To investigate this, various human and murine breast cancer cell lines were seeded on poly-D-lysine coated slides, stained with DAPI and Sytoxgreen, and observed via confocal microscopy. Tumor cells extruded DNA traps within 4 hours without stimuli. Notably, trap extrusion correlated with cell line aggressiveness and was abolished by DNase, indicating dsDNA composition. The absence of traps in 24-hour cultures suggests DNA extrusion may help rare tumor cells survive and proliferate. In line, DNase treatment reduced tumor proliferation and increased apoptosis. To assess the significance of DNA traps in vivo, BALB/c mice were injected with the highly metastatic 4T1 clone 5 breast cancer model, with or without DNase treatment, and monitored starting 5 days post-injection and weekly until day 28. Histological analysis at the injection site 5 days post-injection showed that DNase treatment reduced both local trap formation and tumor cell proliferation, as indicated by BrdU incorporation. Moreover, immunohistochemistry revealed that in DNase-treated mice, initial tumor growth was followed by complete tumor regression, accompanied by local and systemic CD8 T cell activation. Correspondingly, although a few metastases were detectable at day 14 in DNase-treated mice, no metastases were observed by day 28. Although it remains unclear whether tumor-derived or immune cell-derived traps are most relevant in our model, the results demonstrate that their presence in the TME promotes tolerance despite the inherent immune adjuvant properties of the traps. This suggests that DNase treatment could be particularly effective in combating early tumor development or relapse when used as part of a combination therapy designed to sustain anti-tumor immune responses.
 Citation Format: Sabina Sangaletti, Paola Portararo, Laura Botti, Valeria Cancila, Claudio Tripodo, Mario P. Colombo, Claudia Chiodoni. Degradation of extracellular trap DNA sustains anti-tumor immune responses in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9102f696549c2a39f10ac4008739665e02303d60" target='_blank'>
              Abstract C014: Degradation of extracellular trap DNA sustains anti-tumor immune responses in breast cancer
              </a>
            </td>
          <td>
            S. Sangaletti, P. Portararo, Laura Botti, V. Cancila, Claudio Tripodo, M. P. Colombo, C. Chiodoni
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) represent one of the most severe forms of genetic damage in organisms, yet vertebrate models capable of monitoring DSBs in real-time remain scarce. BRCA1/BRCA2-containing complex subunit 3 (BRCC3), also known as BRCC36, functions within various multiprotein complexes to mediate diverse biological processes. However, the physiological role of BRCC3 in vertebrates, as well as the underlying mechanisms that govern its activity, are not well understood. To explore these questions, we generated brcc3-knockout zebrafish using CRISPR/Cas9 gene-editing technology. While brcc3 mutant zebrafish appear phenotypically normal and remain fertile, they exhibit significantly increased rates of mortality and deformity following exposure to DNA damage. Furthermore, embryos lacking Brcc3 display heightened p53 signaling, elevated γ-H2AX levels, and increased apoptosis in response to DNA-damaging agents such as ultraviolet (UV) light and Etoposide (ETO). Notably, genetic inactivation of p53 or pharmacological inhibition of Ataxia-telangiectasia mutated (ATM) activity rescues the hypersensitivity to UV and ETO observed in Brcc3-deficient embryos. These findings suggest that Brcc3 plays a critical role in DNA damage response (DDR), promoting cell survival during embryogenesis. Additionally, brcc3-null mutant zebrafish offer a promising vertebrate model for real-time monitoring of DSBs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d1549cbda55b49c02b764216ac718c73d8cdce" target='_blank'>
              Loss of Brcc3 in Zebrafish Embryos Increases Their Susceptibility to DNA Damage Stress
              </a>
            </td>
          <td>
            Zhengyang Wang, Caixia Wang, Yanpeng Zhai, Yanhong Bai, Hongying Wang, Xiaozhi Rong
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The de-ubiquitinase USP33 has been shown to possess either tumour-promoting or inhibitory effect on human cancer cells. However, all these findings are mainly based on in vitro cell culture models, and the in vivo evidence, which is more plausible to digest the functional role of USP33 in carcinogenic process, is still lacking. Here, we demonstrate that USP33 modulates DNA damage responses including cell cycle arrest and apoptosis induction through associating with p53. It directly interacts with p53 to mediate its de-ubiquitination and further  stabilisation under DNA damage condition. Depletion of USP33 induces an enhanced level of p53 ubiquitination, which de-stabilises p53 protein leading to impaired DNA damage responses. Furthermore, USP33 silencing shows either promoted or inhibited effect on cell proliferation in human cancer cells with p53 WT and mutant background, respectively. Consistently, mice with hepatocyte-specific USP33 knockout are more sensitive to nitrosodiethylamine (DEN)-induced hepatocarcinogenesis compared to wild type mice. Thus, our in vitro and in vivo evidences illustrate that USP33 possesses anti-tumour activity via regulating p53 stability and activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf79241c93d773eae434c4f3d8db032442a0c49" target='_blank'>
              USP33 Regulates DNA Damage Response and Carcinogenesis Through Deubiquitylating and Stabilising p53.
              </a>
            </td>
          <td>
            Yuqi Zhu, Zixiang Chen, Kaifeng Niu, Mengge Li, Yuchun Deng, Ji Zhang, D. Wei, Jiaqi Wang, YongLiang Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
Diffuse hemispheric glioma, H3G34R/V-mutant (DHG-H3G34) is characterized by poor prognosis and lack of effective treatment options. DHG-H3G34R further harbor deactivation of Alpha-Thalassemia/Mental Retardation Syndrome X-linked protein (ATRX; DHG-H3G34R_ATRX) suggesting a unique interaction of these two oncogenic alterations. In this study, we dissect their cell biological interplay, investigate the impact on telomere stabilization and, consequently, validate a targeted therapy approach.


METHODS
We characterized patient-derived primary pediatric high-grade glioma (pHGG) models for telomere-maintenance mechanisms, DNA damage stress (including protein expression, pH2AX/Rad51 foci, cell-cycle arrest) and their sensitivity towards poly-ADP polymerase inhibitor (PARPi) combinations. Human induced pluripotent stem cells (iPSCs) were used for modelling the disease. The anticancer activity of PARPi combinations in vivo was studied in Chorioallantoic Membrane (CAM) and orthotopic in vivo experiments. Finally, we treated a DHG-H3G34R_ATRX patient with a PARPi combination therapy.


RESULTS
We elaborate that alternative lengthening of telomeres (ALT) is a key characteristic of DHG-H3G34R_ATRX. A dominant cooperative effect between H3G34R and ATRX loss in ALT activation also became apparent in iPSCs, which endogenously exert telomerase activity. In both, patient-derived DHG-H3G34R_ATRX models and H3G34R+/ATRX- iPSCs, the ALT phenotype was associated with increased basal DNA damage stress, mediating synergistic susceptibility towards PARPi (talazoparib, niraparib) combinations with topoisomerase-I inhibitors (topotecan, irinotecan). In a first-of-its-kind case, treatment of a DHG-H3G34R_ATRX patient with the brain-penetrant PARP inhibitor niraparib and topotecan resulted in a significant tumor reduction.


CONCLUSION
Our preclinical and clinical data strongly support the further development of PARPis together with DNA damage stress-inducing treatment regimens for DHG-H3G34R_ATRX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77b9cc12ad1d9d295c346bb0f9c67781ab01cbd" target='_blank'>
              Alternative lengthening of telomere-based immortalization renders H3G34R -mutant diffuse hemispheric glioma hypersensitive to PARP inhibitor combination regimens.
              </a>
            </td>
          <td>
            Anna Laemmerer, Christian Lehmann, L. Mayr, K. Bruckner, L. Gabler, D. Senfter, Philipp Meyer, Theresa Balber, C. Pirker, C. Jaunecker, D. Kirchhofer, P. Vician, Michelle Griesser, S. Spiegl-Kreinecker, M. Schmook, Tatjana Traub‐Weidinger, Peter Kuess, Franziska Eckert, A. Federico, S. Madlener, Natalia Stepien, Bernhard Robl, A. Baumgartner, Johannes Hainfellner, Karin Dieckmann, C. Dorfer, Karl Roessler, Nina S. Corsini, K. Holzmann, Wolfgang M. Schmidt, A. Peyrl, Amedeo A Azizi, C. Haberler, A. Beck, Stefan M Pfister, Julia Schueler, D. Loetsch-Gojo, Jürgen A Knoblich, W. Berger, J. Gojo
          </td>
          <td>2024-11-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Circular RNA (circRNA) emerges as a significant sub-type of single-stranded non-coding RNA within colorectal cancer (CRC), boasting high abundance. Delving into research, numerous pivotal roles of circRNA in therapeutic contexts within CRC have come to light, encompassing areas such as metastasis, apoptosis, and proliferation. Moreover, circRNAs exhibit significant involvement in the advancement of therapeutic strategies, demonstrating unique correlations with tumor staging, size and overall survival rates in colorectal cancer. These associations position circRNAs as potential candidates for both anticancer interventions and prognostic biomarkers. Among all cancers, colorectal cancer is the second most prevalent cause of cancer-related death and the third most common disease to be diagnosed worldwide. To gain deeper insights into the impact of circRNA-based therapeutic developments on CRC and its progression, this comprehensive review aims to synthesize the roles of specific therapeutic applications targeting circRNAs in CRC. It also aims to evaluate circRNAs' potential as useful therapeutic targets and prognostic indicators in the context of colorectal cancer. The overarching goal of this review is to illuminate the landscape of therapeutic strategies and aid in clinical decision-making processes related to CRC. By elucidating the intricate interplay between circRNAs and therapeutic interventions, this review seeks to contribute to the advancement of therapeutic modalities and improve patient outcomes in the realm of colorectal cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e2a54ee5b48b0544173f0c3c5f63c22c950fe" target='_blank'>
              Circular RNAs: Therapeutic Uses in Colorectal Cancer
              </a>
            </td>
          <td>
            Muthusamy Thangavel, Chalini Vijayakumar, Deepalakshmi Balakrishnan
          </td>
          <td>2024-11-30</td>
          <td>International Journal of Experimental Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f14ce0de021c18e7f8fa32d88be80097576943" target='_blank'>
              BCAS2 and hnRNPH1 orchestrate alternative splicing for DNA double-strand break repair and synapsis in meiotic prophase I
              </a>
            </td>
          <td>
            Longjie Sun, Rong Ye, Changchang Cao, Zheng Lv, Chaofan Wang, Xiaomei Xie, Xuexue Chen, Xiaohong Yao, Shuang Tian, Lu Yan, Yujing Shao, Sheng Cui, Chen Chen, Yuanchao Xue, Lei Li, Juan Chen, Jiali Liu
          </td>
          <td>2024-11-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85d17441ac44a75b94aa59b4d0c571fba9dac0d" target='_blank'>
              RNA-binding motif protein RBM39 enhances the proliferation of gastric cancer cells by facilitating an oncogenic splicing switch in MRPL33.
              </a>
            </td>
          <td>
            Cheng-Piao Lu, Jiabao Li, Dong-Bao Li, , , , Guoqiang Xu
          </td>
          <td>2025-01-03</td>
          <td>Acta pharmacologica Sinica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: NTHL1 is a DNA glycosylase, and mutations in this gene are linked to colorectal polyps and colorectal cancer (CRC). SBS30, a mutational signature associated with NTHL1 inactivation, correlates with c-alkylation (c-alkyl), a CRC mutational signature of alkylating damage. We aimed to investigate the potential for synthetic lethality between c-alkyl and NTHL1 inactivation as a novel therapeutic strategy for CRC. Methods: Using DepMap data, we categorized CRC cell lines into low and high c-alkyl groups. We performed CRISPR-mediated NTHL1 knockouts (KOs) in high vs low c-alkyl groups of cell lines and patient-derived CRC organoids and tested for synthetic lethality and its mechanisms. Results: High c-alkyl cells with NTHL1 KO showed decreased proliferation compared to low c-alkyl cells, with cell cycle arrest predominantly in the G2/S phase. Treatment with N-Nitroso-N-methylurea (MNU, alkylating reagent) converted low c-alkyl cells to a high c-alkyl state and conferred increased susceptibility to NTHL1 KO. High c-alkyl patient-derived organoids similarly showed lower proliferation with NTHL1 KO compared to low c-alkyl ones. High c-alkyl cells exhibited increased DNA damage, indicated by γH2AX and CHK1 phosphorylation. Low c-alkyl cells exhibited significant HDAC1 and HDAC2 upregulation, which was not seen in high c-alkyl cells. This could explain their protection against NTHL1 KO, whereas the lack of change in HDAC protein level may explain the failure of high c-alkyl cells to repair NTHL1-induced DNA damage. High c-alkyl cells with NTHL1 KO additionally showed increased DNA damage and cell death when treated with checkpoint inhibitors AZD-7762, LY2606368, and Rabusertib. Conclusions: NTHL1 KO presents a promising therapeutic target, especially for high c-alkyl CRCs that are more susceptible to DNA damage. Our findings also suggest that combining NTHL1 inhibition with CHK1 inhibition may offer new therapeutic opportunities in CRC.
 Citation Format: Hyeoncheol Kim, Jules Cazaubiel,, Tung-Lin Chiang, Carino Gurjao, Samantha Fitzgerald, Marios Giannakis. Increased vulnerability of colorectal cancer with high alkylation damage to NTHL1 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad8e781bd0deb99dc7d838c25939d2a03c2eb3f7" target='_blank'>
              Abstract A019: Increased vulnerability of colorectal cancer with high alkylation damage to NTHL1 inactivation
              </a>
            </td>
          <td>
            Hyeoncheol Kim, Jules Cazaubiel, Tung-Lin Chiang, Carino Gurjao, Samantha Fitzgerald, M. Giannakis
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Infections have imposed strong selection pressures throughout human evolution, making the study of natural selection's effects on immunity genes highly complementary to disease-focused research. This review discusses how ancient DNA studies, which have revolutionized evolutionary genetics, increase our understanding of the evolution of human immunity. These studies have shown that interbreeding between modern humans and Neanderthals or Denisovans has influenced present-day immune responses, particularly to viruses. Additionally, ancient genomics enables the tracking of how human immunity has evolved across cultural transitions, highlighting strong selection since the Bronze Age in Europe (<4,500 years) and potential genetic adaptations to epidemics raging during the Middle Ages and the European colonization of the Americas. Furthermore, ancient genomic studies suggest that the genetic risk for noninfectious immune disorders has gradually increased over millennia because alleles associated with increased risk for autoimmunity and inflammation once conferred resistance to infections. The challenge now is to extend these findings to diverse, non-European populations and to provide a more global understanding of the evolution of human immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b729418da43553d3cfa26e68fd325ebc987180" target='_blank'>
              Tracing the Evolution of Human Immunity Through Ancient DNA.
              </a>
            </td>
          <td>
            E. Patin, L. Quintana-Murci
          </td>
          <td>2024-12-20</td>
          <td>Annual review of immunology</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42ec81f614bb1f21e82ec3dff15614039d3cf708" target='_blank'>
              SPAG5 is a potential therapeutic target affecting proliferation, apoptosis, and invasion of esophageal cancer cells
              </a>
            </td>
          <td>
            Xiaohui Zhang, Lingmin Zhang, M. Cui, Shiyu Ji, Yanan Zhang, Qian Li, Mingxin Zhang
          </td>
          <td>2024-12-19</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b91b716a484954fac596a012c478c6ddaee7a8e" target='_blank'>
              HMGA2 overexpression with specific chromosomal abnormalities predominate in CALR and ASXL1 mutated myelofibrosis.
              </a>
            </td>
          <td>
            Shivani Handa, Christoph Schaniel, J. Tripodi, Daiva Ahire, Md Babu Mia, Sophie Klingborg, D. Tremblay, Bridget K Marcellino, R. Hoffman, V. Najfeld
          </td>
          <td>2024-12-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7636cc7694bca6c468bc6e4c750272c2be1508" target='_blank'>
              SPRTN metalloprotease participates in repair of ROS-mediated DNA-protein crosslinks
              </a>
            </td>
          <td>
            Luke Erber, Arnold Scott Groehler IV, Cesar I. Cyuzuzo, Jahan Baker-Wainwright, Reeja S. Maskey, Lei Li, Yuichi J Machida, Natalia Tretyakova
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Taxanes are frequently used anticancer drugs known to kill tumor cells by inducing mitotic aberrations and segregation defects. A defining feature of specific cancers, notably triple-negative breast cancer (TNBC) and particularly those deficient in BRCA1, is chromosomal instability (CIN). Here, we focused on understanding the mechanisms of docetaxel-induced cytotoxicity, especially in the context of BRCA1-deficient TNBC. Using functional genetic screens in CIN+ cells, we identified genes that mediate docetaxel response and found an interaction between Huntingtin (HTT) and BRCA1-associated protein-1 (BAP1). We employed Brca1-/-;p53-/- mammary tumor cells, derived from genetically engineered mouse tumors that closely mimic the human disease, to investigate the role of these genes in CIN+ BRCA1-deficient cells. Specifically, we observed that loss of HTT sensitizes CIN+ BRCA1-deficient mammary tumor cells to docetaxel by shortening mitotic spindle poles and increasing spindle multipolarity. In contrast, BAP1 depletion protected cells against these spindle aberrations by restoring spindle length and enhancing mitotic clustering of the extra centrosomes. In conclusion, our findings shed light on the roles of HTT and BAP1 in controlling mitotic spindle multipolarity and centrosome clustering, specifically in the absence of BRCA1. This affects the response to microtubule-targeting agents and suggests that further studies of the interaction of these genes with the mitotic spindle may provide useful insights into how to target CIN+ cells, particularly in the challenging therapeutic landscape of BRCA1-deficient TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2088ba0887dd7b3e666579053f1f608773cc6a" target='_blank'>
              Docetaxel response in BRCA1,p53-deficient mammary tumor cells is affected by Huntingtin and BAP1.
              </a>
            </td>
          <td>
            Martín González-Fernández, Carmen Perry, Nora Merete Gerhards, P. Francica, Sven Rottenberg
          </td>
          <td>2024-12-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
 Esophageal squamous cell carcinoma (ESCC) is a malignant tumor with high morbidity and mortality, and easy to develop resistance to chemotherapeutic agents. Telomeres are DNA-protein complexes located at the termini of chromosomes in eukaryotic cells, which are unreplaceable in maintaining the stability and integrity of genome. Telomerase, an RNA-dependent DNA polymerase, play vital role in telomere length maintain, targeting telomerase is a promising therapeutic strategy for cancer.
 AIM
 To investigate the efficacy and underlying mechanisms of BIBR1532, a telomerase inhibitor, in ESCC.
 METHODS
 KYSE150 and KYSE410 cells were cultured and exposed to various concentrations of BIBR1532. Cell viability was assessed at 48 hours and 72 hours to determine the IC50 values. The effects of BIBR1532 on ESCC cell proliferation, migration, and cellular senescence were evaluated using the cell counting kit-8 assay, plate colony formation assay, scratch assay, transwell assay, and β-galactosidase staining, respectively. Western blotting was performed to detect the expression of proteins in BIBR1532-treated ESCC cells, such as human telomerase reverse transcriptase (hTERT), key molecules involved in DNA damage response (DDR) or cellular senescence, as well as telomere-binding proteins. Additionally, a tumor-bearing nude mouse model was established to evaluate the anti-cancer effect of BIBR1532 in vivo .
 RESULTS
 The IC50 values for KYSE150 and KYSE410 cells after 48 hours of BIBR1532 exposure were 48.53 μM and 39.59 μM, respectively. These values decreased to 37.22 μM and 22.71 μM, respectively, following a longer exposure of 72 hours. BIBR1532 exhibited dose-dependent effects on KYSE150 and KYSE410 cells, including decreased hTERT expression, inhibition of proliferation and metastasis, and induction of cellular senescence. Mechanistically, BIBR1532 upregulated the expression of the DDR protein, γ-H2AX, and activated the ataxia telangiectasia and Rad3-related protein (ATR)/ check point kinase 1 (CHK-1) and ataxia-telangiectasia mutated gene (ATM)/CHK2 pathways. BIBR1532 downregulated the expression of telomere-binding proteins, including telomeric-repeat binding factor 1 (TRF1), TRF2, protection of telomeres 1, and TIN2-interacting protein 1. In a nude mouse xenograft model, BIBR1532 significantly suppressed tumor growth, reduced hTERT expression, and increased γ-H2AX protein levels. Hematoxylin and eosin staining of various organs, including the heart, liver, spleen, lungs, and kidneys, revealed no apparent adverse effects.
 CONCLUSION
 BIBR1532 exerts anti-cancer effects on ESCC by inducing DDR through the ATR/CHK1 and ATM/CHK2 pathways and downregulating the expression of telomere-binding proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c9a7cb4627655be894378fe9cf7a95af0081d7" target='_blank'>
              BIBR1532 inhibits proliferation and metastasis of esophageal squamous cancer cells by inducing telomere dysregulation
              </a>
            </td>
          <td>
            Qin Wang, Qingrong Li, Lei Xu, Zichun Yuan, Xiao Liu, Mao-Ju Tang, Man Luo, Xiao-Wu Zhong, Qiang Ma, Xiao-Lan Guo
          </td>
          <td>2025-01-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Fidelity of DNA replication is crucial for the accurate transmission of genetic information across generations, yet errors still occur despite multiple control mechanisms. This study investigated the factors influencing spontaneous replication errors across the Escherichia coli genome. We detected errors using the MutS and MutL mismatch repair proteins in rapidly proliferating mutH-deficient cells, where errors can be detected but not corrected. Our findings reveal that replication error hotspots are non-randomly distributed along the chromosome and are enriched in sequences with distinct features: lower thermal stability facilitating DNA strand separation, mononucleotide repeats prone to DNA polymerase slippage and sequences prone to forming secondary structures like cruciforms and G4 structures, which increase likelihood of DNA polymerase stalling. These hotspots showed enrichment for binding sites of nucleoid-associated proteins, RpoB and GyrA, as well as highly expressed genes, and depletion of GATC sequence. Finally, the enrichment of single-stranded DNA stretches in the hotspot regions establishes a nexus between the formation of secondary structures, transcriptional activity and replication stress. In conclusion, this study provides a comprehensive genome-wide map of replication error hotspots, offering a holistic perspective on the intricate interplay between various mechanisms that can compromise the faithful transmission of genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c1b75e2f5b4dd53e47629f123a3c0b8af8ac5b" target='_blank'>
              Genome-wide mapping of spontaneous DNA replication error-hotspots using mismatch repair proteins in rapidly proliferating Escherichia coli.
              </a>
            </td>
          <td>
            Flavia C Hasenauer, Hugo C Barreto, Chantal Lotton, Ivan Matic
          </td>
          <td>2024-12-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Long non-coding RNA Rhabdomyosarcoma 2-associated Transcript (RMST) is a crucial regulator in various biological processes, particularly in neurogenesis and cancer progression. This review summarizes current knowledge on structure, expression patterns, and functional roles across different organs and diseases of RMST. RMST exhibits tissue-specific expression, notably in brain tissues and vascular endothelial cells, and plays a significant role in neuronal differentiation through interaction with SRY-box 2 (SOX2). In cancer, RMST predominantly functions as a tumor suppressor, with context-dependent roles observed across different cancer types. RMST is also implicated in neurological disorders, cardiovascular diseases, and Hirschsprung's disease. Mechanistically, RMST acts as a competing endogenous RNA and a transcriptional regulator, interacting with various microRNAs and proteins to modulate gene expression. The potential of RMST as a biomarker and therapeutic target is increasingly recognized, particularly in atherosclerosis and cancer. While current findings are promising, further research is needed to fully elucidate the functions and translate these insights into clinical applications of RMST. This review underscores the significance of RMST in cellular processes and disease pathogenesis, highlighting its potential as a novel target for diagnostic and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274785be6ba4141ed74a6cfcddf7cc469ab67f46" target='_blank'>
              RMST: A long noncoding RNA involved in cancer and disease.
              </a>
            </td>
          <td>
            Hidenori Tani
          </td>
          <td>2024-11-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetics which examines the regulation of genes without modification of the DNA sequence, plays a crucial role in various biological processes and disease mechanisms. Among the different forms of epigenetic modifications, histone post-translational modifications (PTMs) are important for modulating chromatin structure and gene expression. Aberrant levels of histone PTMs are implicated in a wide range of diseases, including cancer, making them promising targets for biomarker discovery and therapeutic intervention. In this context, blood, tissues, or cells serve as valuable resources for epigenetic research and analysis. Traditional methods such as mass spectrometry and western blotting are widely used to study histone PTMs, providing qualitative and (semi)quantitative information. However, these techniques often face limitations that could include throughput and scalability, particularly when applied to clinical samples. To overcome these challenges, we developed and validated 13 Nu.Q® immunoassays to detect and quantify specific histone PTM-nucleosomes from K2EDTA plasma samples. Then, we tested these assays on other types of samples, including chromatin extracts from frozen tissues, as well as cell lines and white blood cells Our findings demonstrate that the Nu.Q® assays offer high specificity, sensitivity, precision and linearity, making them effective tools for epigenetic profiling. A comparative analysis of HeLa cells using mass spectrometry, Western blot, and Nu.Q® immunoassays revealed a consistent histone PTMs signature, further validating the effectiveness of these assays. Additionally, we successfully applied Nu.Q® assays across various biological samples, including human tissues from different organs and specific white blood cell subtypes, highlighting their versatility and applicability in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea54269010715f8d7295e217ed70c9663ed44c1" target='_blank'>
              High-Throughput Epigenetic Profiling Immunoassays for Accelerated Disease Research and Clinical Development
              </a>
            </td>
          <td>
            P. Van den Ackerveken, Clotilde Hannart, Dorian Pamart, Robin Varsebroucq, Marion Wargnies, Olivia Thiry, Marie Lurkin, Séverine Vincent, Muriel Chapelier, Guillaume Rommelaere, M. Herzog
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92e2f5cabd9e2a4c286349a54228e7bf25b75d61" target='_blank'>
              E2F1-driven CENPM expression promotes glycolytic reprogramming and tumorigenicity in glioblastoma
              </a>
            </td>
          <td>
            Zhiqiang Yi, Yanfei Jia, Runchun Lu, Chunwei Li, Long Wen, Xiangdong Yin, Junfei Yi, Liang Li
          </td>
          <td>2024-12-21</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Dense, fibrotic stroma is a defining feature of pancreatic ductal adenocarcinoma (PDAC), comprising up to 90% of tumor volume. Multiple studies have shown that this stroma strongly influences cancer progression and response to treatment. Cancer Associated Fibroblasts (CAFs) make up the majority of the Tumor Microenvironment (TME) and directly alter the tumor landscape and response to therapeutics. CAFs are a heterogenous population comprising of a mixture of inflammatory, myofibroblastic, and antigen-presenting CAFs, with both pro- and anti- tumorigenic functions. While we have been able to define CAFs phenotypically, we lack a deeper understanding of the mechanisms underlying their transcriptional heterogeneity and ultimately their contribution to driving PDAC progression. Our lab has previously shown 3’UTR-Alternative Polyadenylation (APA) to be an important driver of genetic dysregulation in PDAC. APA is an important process regulating mRNA stability, protein localization, and expression, ultimately determining cellular processes such as proliferation, metastasis, and migration. We have also shown that increased 3’ UTR shortening is associated with the pro- tumorigenic inflammatory CAF subtype. However, how APA regulates CAF function is unknown. We are now trying to understand the possible mechanisms of these changes in vitro using human-derived CAF cell lines and treating them with a known pharmacological modulator of APA. We performed PAC-Seq analyses on these samples and show that important drivers of CAF metabolism are hindered by APA, ultimately affecting the ability of CAFs to support cancer cell growth. We find that inhibition of APA drives CAF migration and senescence phenotype and alters gene expression of critical genes required for CAF function, and metabolism. We identified a novel mechanism by which a key metabolic gene SLC7A11 is regulated by APA. To our knowledge, our work is one of the first studies to study how APA is an important regulator of CAF function.
 Citation Format: Aditi H Chaubey, Michael E Feigin, Eric J Wagner. Alternative Polyadenylation Regulates Pancreatic Cancer Fibroblast Metabolism [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc12f648898131bc451f0e3369b7f9a0afe340d8" target='_blank'>
              Abstract A018: Alternative Polyadenylation Regulates Pancreatic Cancer Fibroblast Metabolism
              </a>
            </td>
          <td>
            Aditi H. Chaubey, Michael E. Feigin, Eric J Wagner
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Activation of the ubiquitin ligase APC/C by the protein Cdc20 is an essential requirement for proper cell division in higher organisms, including humans. APC/C is the ultimate effector of the Spindle Assembly Checkpoint (SAC), the signalling system that monitors the proper attachment of chromosomes to microtubules during cell division. Defects in this process result in genome instability and cancer. Interfering with APC/C substrate ubiquitylation in cancer cells delays mitotic exit, which induces cell death. Therefore, impairing APC/C function represents an opportunity for the treatment of cancer and malignancies associated with SAC dysregulation. In this study, we report a new class of pyrimidinethylcarbamate apcin analogues that interfere with APC/C activity in 2D and 3D breast cancer cells. The new pyrimidinethylcarbamate apcin analogues exhibited higher cytotoxicity than apcin in all breast cancer cell subtypes investigated, with much lower cytotoxicity observed in fibroblasts and RPE-1 cells. Further molecular rationalisation of apcin and its derivatives was conducted using molecular docking studies. These structural modifications selected from the in silico studies provide a rational basis for the development of more potent chemotypes to treat highly aggressive breast cancer and possibly other aggressive tumour types of diverse tissue origins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce9677b341aba6693250527e3e3e6188a07d0c3" target='_blank'>
              Targeting APC/C Ubiquitin E3-Ligase Activation with Pyrimidinethylcarbamate Apcin Analogues for the Treatment of Breast Cancer
              </a>
            </td>
          <td>
            Maria Kapanidou, Natalie L. Curtis, Sandra S. Díaz-Mínguez, Sandra Agudo-Álvarez, Alfredo Rus Sanchez, Ammar Mayah, Rosette Agena, Paul Brennan, Paula Morales, Raúl Benito-Arenas, A. Bastida, V. Bolanos-Garcia
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Noncoding RNA 886 has emerged as a pivotal regulatory RNA with distinct functions across tissues, acting as a regulator of protein activity by directly binding to protein partners. While it is well recognized as a tumor suppressor in prostate cancer, the underlying molecular mechanisms remain elusive. To discover the principal pathways regulated by nc886 in prostate cancer, we used a transcriptomic and proteomic approach analyzing malignant DU145, LNCaP, PC3, and benign RWPE-1 prostate cell line models transiently transfected with in vitro transcribed nc886 or antisense oligonucleotides. Multiomics revelead a significant enrichment of immune system-related pathways across the cell lines, including cytokines and interferon signaling. The interferon response provoked by nc886 was validated by functional assays. The invariability of PKR phosphorylation and NF-κB activity in the gain/loss of nc886 function experiments and the positive regulation of innate immunity suggest a PKR-independent mechanism of nc886 action. Accordingly, the GSEA of the PRAD-TCGA data set revealed immune stimulation as the nc886 most associated node also in the clinical setting. Our study showed that the reduction of nc886 leads to a blunted immune response in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f4cce51821d8413b62bbd710ee2168d2423850" target='_blank'>
              Multi-Omics Study Reveals Nc886/vtRNA2-1 as a Positive Regulator of Prostate Cancer Cell Immunity.
              </a>
            </td>
          <td>
            Carolina Oliveira-Rizzo, Camilla L Colantuono, Ana J Fernández-Alvarez, Graciela L Boccaccio, Beatriz Garat, J. Sotelo-Silveira, Shahbaz Khan, V. Ignatchenko, Yong Sun Lee, T. Kislinger, Stanley K Liu, R. Fort, M. A. Duhagon
          </td>
          <td>2024-12-26</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Small nucleolar RNAs (snoRNAs) a class of ncRNAs that canonically guide post-transcriptional modifications, including 2'-O-methylation, on ribosomal RNA (rRNA) and small nuclear RNA (snRNA). While traditionally considered housekeeping genes, snoRNAs have increasingly been found to function in diverse physiologic and pathologic processes, including cancer. In an immune-competent murine model of triple negative breast cancer (TNBC) lymphatic metastasis, we identified the snoRNA Snord67 as one of the most upregulated noncoding RNAs in axillary LN (AxLN) tumors relative to mammary fat pad (MFP) tumors and lung metastases. Loss of Snord67 resulted in decreased colony formation and spheroid size in murine and human TNBC cell lines, and led to decreased lymph node tumor growth and decreased distant metastases in two immune-competent murine models of TNBC lymphatic dissemination. To determine the mechanism by which Snord67 promotes tumor growth and metastasis, we examined the impact of Snord67 on 2'-O-methylation, gene expression, and alternative splicing. Loss of Snord67 in TNBC cell lines led to decreased 2'-O-methylation at the C60 nucleotide (Cm60) in the core spliceosome component U6 snRNA. Re-introduction of wild-type Snord67 rescued Cm60 in U6 snRNA and rescued the colony formation and spheroid phenotypes of the Snord67 knockout cell lines. However, a mutant Snord67 incapable of guiding U6 Cm60 only partially rescued these in vitro phenotypes, suggesting that Snord67 promotes in vitro tumor cell growth at least in part by guiding Cm60 in U6 snRNA. We then performed RNA sequencing of Snord67 knockout and wild-type murine and human TNBC cell lines. We found that loss of Snord67 led to widespread changes in alternative splicing, consistent with its role in guiding 2'-O-methylation of the core spliceosome component U6. We further demonstrated that the inclusion of alternatively spliced cassette exons in MYO18A and NFYA was not only downregulated upon Snord67 knockout in a human TNBC cell line, but also positively correlated with Snord67 expression levels in primary breast tumors and lymph node metastases from breast cancer patients. Based on these results, we propose a model in which Snord67 guides U6 Cm60, which leads to a pro-metastatic alternative splicing program and thereby promotes lymph node tumor growth and distant metastasis.
 Citation Format: Katherine I Zhou, Yvonne Chao, Kwame K Forbes, Alessandro Porrello, Gabrielle M Gentile, Aaron C Chack, Dixcy J.S. John Mary, Haizhou Liu, Yinzhou Zhu, Eric Cockman, Lincy Edatt, Grant A Goda, Justin Zhao, Hala Abou Assi, Hannah J Wiedner, Yi-Hsuan Tsai, Lily Wilkinson, Amanda E Van Swearingen, Lisa A Carey, Jimena Giudice, Daniel Dominguez, Christopher L Holley, Chad V Pecot. Snord67 promotes breast cancer metastasis through U6-mediated alternative splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr A003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aadf8a97c1e9a5781e07659076ceb1abfdcb56d" target='_blank'>
              Abstract A003: Snord67 promotes breast cancer metastasis through U6-mediated alternative splicing
              </a>
            </td>
          <td>
            Katherine I Zhou, Yvonne Chao, Kwame Forbes, A. Porrello, Gabrielle M Gentile, Aaron C Chack, Dixcy Jaba Sheeba John Mary, Haizhou Liu, Yinzhou Zhu, Eric Cockman, Lincy Edatt, Grant A. Goda, Justin Zhao, Hala Abou Assi, H. Wiedner, Yi-Hsuan Tsai, Lily Wilkinson, Amanda Elyse Day Van Swearingen, Lisa A Carey, Jimena Giudice, Daniel Dominguez, Christopher L Holley, Chad V Pecot
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a malignancy associated with high morbidity and mortality, yet treatment options are limited. In addition to genetic alterations, aberrant gene expression contributes to the pathology of malignant diseases. In the present study, we identified 629 genes consistently dysregulated between OSCC and normal oral mucosa across nine public gene expression datasets. Among them, mitosis-related genes were significantly enriched, including spindle and kinetochore-associated complex subunit 1 (SKA1), whose roles in OSCC had been studied only to a very limited extent. We show that SKA1 promoted proliferation and colony formation in 2D and 3D, shortened the duration of metaphase, and increased the migration of OSCC cell lines. In addition, high SKA1 expression enhanced radioresistance, a previously unknown effect of this gene, which was accompanied by a reduction of radiation-induced senescence. SKA1 was also upregulated in a subset of advanced oral premalignancies and promoted tumor-relevant properties in a corresponding cell line. Gene expression patterns evoked by SKA1 overexpression confirmed that this gene is able to advance properties required for both early and advanced stages of tumorigenesis. In summary, our data show that SKA1 contributes to malignant progression in OSCC and may be a useful marker of radioresistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec70f60faa1954d800f562931fb2bf10f070ffa" target='_blank'>
              SKA1 promotes oncogenic properties in oral dysplasia and oral squamous cell carcinoma, and augments resistance to radiotherapy.
              </a>
            </td>
          <td>
            A. Grandits, Barbara Andrea Reinoehl, Renate Wagner, Peter Kuess, Franziska Eckert, A. Berghoff, Thorsten Fuereder, Rotraud Wieser
          </td>
          <td>2024-12-10</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="ABSTRACT Missing an entire chromosome or chromosome arm in normal diploid cells has a deleterious impact on cell viability, which may contribute to the development of specific birth defects. Nevertheless, the effects of chromosome loss in human cells have remained unexplored due to the lack of suitable model systems. Here, we developed an efficient, selection‐free approach to generate partial monosomy in human induced pluripotent stem cells (iPSCs). The introduction of Cas9 proteins and a pair of gRNAs induces over megabase‐sized interstitial chromosomal deletions. Using human chromosome 21 (HSA21) as a model, partial monosomy 21q (PM21q) iPSC lines with deletions ranging from 4.5 to 27.9 Mb were isolated. A 33.6 Mb deletion, encompassing all protein‐coding genes on 21q, was also achieved, establishing the first 21q monosomy human iPSC line. Transcriptome and proteome analyses revealed that the abundances of mRNA and protein encoded by the majority of genes in the monosomic regions are half of the diploid expression level, indicating an absence of dosage compensation. The ability to generate customized partial monosomy cell lines on an isogenic, karyotypically normal background should facilitate the gain of novel insights into the impact of chromosome loss on cellular fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f457fb7efd63146141d89c41994690b979076d69" target='_blank'>
              Generation of Monosomy 21q Human iPS Cells by CRISPR/Cas9‐Mediated Interstitial Megabase Deletion
              </a>
            </td>
          <td>
            Masaya Egawa, N. Uno, Rina Komazaki, Yusuke Ohkame, Kyotaro Yamazaki, Chihiro Yoshimatsu, Yuki Ishizu, Yusaku Okano, Hitomaru Miyamoto, Mitsuhiko Osaki, Teruhiko Suzuki, Kazuyoshi Hosomichi, Yasunori Aizawa, Yasuhiro Kazuki, Kazuma Tomizuka
          </td>
          <td>2024-11-24</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mycoplasma bovis (M. bovis) is characterized by a reduced genomic size and limited synthetic capacity, including the inability to synthesize nucleotides de novo, relies on nucleases for nutrient acquisition and survival. A number of nucleases have been implicated in M. bovis pathogenicity, facilitating substrate degradation and contributing to DNA repair mechanisms that enhance bacterial persistence. The present study confirmed that the T5.808 mutant, in which a novel nuclease gene (Mbov_0701) was disrupted by the mini-Tn4001 transposon, exhibits a growth defect when co-cultured with EBL cells. However, the restoration of Mbov_0701 resulted in the resumption of growth in the mutant. The characterization of MbovP701 revealed that it had high activity in hydrolyzing dsDNA with 5′- to 3′- polarity. Furthermore, the substrates of MbovP701 were extended to include linear dsDNA, ssDNA, RNA, and plasmid DNA. The exonuclease activity is dependent on the presence of Mn2+ and/or Mg2+ ions, with an optimal pH and temperature of 8.3 and 43 °C, respectively. The truncation experiments of rMbovP701 revealed that YqaJ (41–185 aa) is the key functional domain of MbovP701 exonuclease. In conclusion, the present study identified a novel nuclease in M. bovis that plays an essential role in the proliferation of this minimal organism. This finding elucidates the survival strategy and pathogenesis of M. bovis, suggesting a potential therapeutic strategy for the treatment of M. bovis through targeting the inhibition of MbovP701. Moreover, it provides a foundation for future investigations into the interactions between MbovP701 and other nucleases involved in M. bovis biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edf88c44ad6d18a2167a54f0906a3be9810ae56b" target='_blank'>
              Novel Nuclease MbovP701 with a Yqaj Domain Is Interrelated with the Growth of Mycoplasma bovis
              </a>
            </td>
          <td>
            Zhiyu Hao, Doukun Lu, Xixi Li, Abdul Raheem, Gang Zhao, A. Dawood, Yingyu Chen, Xi Chen, Changmin Hu, Jianguo Chen, Lei Zhang, Xifang Zhu, A. Guo
          </td>
          <td>2024-12-01</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Known as molecular parasites that invaded our genome through ancestors, transposons (mainly retrotransposons) are interspersed genomic repeats that constitute ∼40% of the mammalian genome. Primarily residing in the heterochromatin, retrotransposons exhibit dynamic expressions to orchestrate embryonic development and shape adult functions. Although rare, a cohort of evolutionarily young retrotransposons resisted extinction, whose selfish activities underlie polymorphic variations and genetic diseases including cancer. While the extraordinary molecular and functional heterogeneity of retrotransposons has been long recognized, mechanistic dissection remains challenging. To tackle this challenge, we use murine skin as our model, given its well-characterized, abundant, and highly accessible adult stem cells, not only mediating postnatal remodeling and tissue repair, but also serving as cell of origin for squamous cell carcinomas. By analyzing a genetic model lacking a known retrotransposon suppressor, histone methyltransferase Setdb1, which we found to be essential in the adult skin, we saw hair loss and hair follicle stem cell exhaustion phenotype, accompanied by a robust and selective surge of endogenous retroviruses (ERVs). When combined with squamous cell carcinoma drivers, Setdb1 loss significantly blocked tumor progression in vivo. We detected abundant retroviral peptides originated from its full-length copies through mass spectrometry and viral-like particles through transmission electron microscopy that are immunoreactive to previously reported ERV surface envelope (env) antibody. Similar viral-like particles were broadly evident across several epithelial tissues beyond skin, implicating its conserved function across squamous cancers. Mechanistically, epithelial originated ERVs elicit tissue wide inflammatory response, and can be ameliorated by pharmacological or genetic inhibition of retroviral activity. Moreover, ERV reactivation induced replication stress functionally accounts for stem cell exhaustion, therefore serving as a specific therapeutic vulnerability in squamous cancers. As an evolutionary conundrum, viral-coding ERVs pose a threat to host fitness and yet, they’ve resisted extinction persisting in the mammalian genome. Our findings suggest ERVs wrestle with the host surveillance programs to regulate adult tissue physiology and pathology, thus providing a key target in eliciting innate immunity in squamous cell carcinomas.
 Citation Format: Ying Lyu, Yejing Ge. Unlocking antiviral immunity against endogenous retroviruses in skin squamous cell carcinomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc7dbf1b898d5bb5d141e58e30c5943543ace80" target='_blank'>
              Abstract B007: Unlocking antiviral immunity against endogenous retroviruses in skin squamous cell carcinomas
              </a>
            </td>
          <td>
            Ying Lyu, Yejing Ge
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains challenging to treat, which in part relates to genetic heterogeneity of the disease, to the protective tumor microenvironment driving resistance to therapy, and also to immune evasion characteristics of leukemic cells. Targeting epigenetic programs in AML provides an attractive opportunity to impair long-term proliferation and induce differentiation. The novel inhibitor JNJ- 75276617 (bleximenib) targets the menin-KMT2A interaction and provides preclinical efficacy in AML (Kwon et al1). Here, we provide mechanistic insight in how JNJ- 75276617 impairs proliferation and drives differentiation of primary AML patient cells. A large-scale drug screen was set up in which genetic alterations and quantitative proteomics were compared with drug sensitivity in a preclinical setting, which revealed that granulocyte macrophage progenitor (GMP)-like AMLs display the greatest sensitivity. Furthermore, we identified that NPM1c/DNMT3Amut AMLs are sensitive, and some NPM1wt AML subtypes without KMT2A-MLLT3 rearrangements benefit from menin-KMT2A inhibition. Genome-wide ChIP-seq studies revealed patient-specific epigenetic alterations upon JNJ-75276617 treatment, uncovering a striking upregulation of MHC class I and class II expression as a consequence of epigenetic changes upon menin-KMT2A inhibition, independent of MEIS1 loss but involving CIITA activation. Functionally, this results in enhanced sensitivity of leukemic blasts to T cell-mediated cytotoxicity in allogeneic and autologous settings. Our data indicate that JNJ-75276617 provides a potential therapeutic approach whereby not only proliferation is impaired and differentiation is induced, but whereby therapeutic benefit might also be achieved by reactivating the antigen presentation machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eecdee97edcaeb1d7fdc720efba9913f9174c55" target='_blank'>
              Bleximenib, the novel menin-KMT2A inhibitor JNJ-75276617, impairs long-term proliferation and immune evasion in acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Hogeling, Duy Minh Lê, Nikita D. La Rose, Min Chul Kwon, Albertus T J Wierenga, Fiona A J Van den Heuvel, V. van den Boom, Anna Kuchnio, Ulrike Philippar, G. Huls, J. Schuringa
          </td>
          <td>2024-12-19</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c98de3dc5aa7f28bfe8e8b94e8de314706c46e1" target='_blank'>
              TBP activates DCBLD1 transcription to promote cell cycle progression in cervical cancer.
              </a>
            </td>
          <td>
            Zhigang Shen, Mei Li, He Zhu, Tao Song
          </td>
          <td>2024-11-26</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e501c572b2c39ffe6df84d823c30df850a5f3204" target='_blank'>
              PU.1 eviction at lymphocyte-specific chromatin domains mediates glucocorticoid response in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Dominik Beck, Honghui Cao, Feng Tian, Yizhou Huang, Miao Jiang, Han Zhao, Xiaolu Tai, Wenqian Xu, Hansen J. Kosasih, David Kealy, Weiye Zhao, Samuel J. Taylor, Timothy A. Couttas, Gaoxian Song, Diego Chacon-Fajardo, Yashna Walia, Meng Wang, A. Dowle, Andrew N. Holding, Katherine S. Bridge, Chao Zhang, Jin Wang, Jian-Qing Mi, Richard B. Lock, Charles E de Bock, Duohui Jing
          </td>
          <td>2024-11-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT DNA damage repair is a crucial cellular mechanism for rectifying DNA lesions arising during growth and development. Among the various repair pathways, postreplication repair (PRR) plays a pivotal role in resolving single-stranded gaps induced by DNA damage. However, the contribution of PRR to virulence remains elusive in the fungal pathogen Candida albicans (C. albicans). In this study, we investigated the role of Rad18, a critical component of PRR, in DNA damage response and virulence in C. albicans. We observed that deletion of RAD18 in C. albicans resulted in heightened sensitivity to DNA damage stress. Through deletion of specific internal domains coupled with spot assay analysis, we show that the internal RING and SAP domains play essential roles in DNA damage response, whereas the ZNF domain was less important. Surprisingly, the lack of Rad18 in C. albicans resulted in heightened intracellular survival within macrophages and elevated virulence in the Galleria mellonella model. RNAseq analysis revealed that loss of Rad18 upregulated the transcription of genes encoding transporters and oxidoreductases, as well as virulence genes, including HWP1 and ECE1. Suppression of the transcription of these virulence genes in the RAD18 deletion strain by a dCas9-mediated CRISPRi system reversed this increased virulence. Taken together, these data demonstrate that Rad18 plays a significant role in virulence partially through transcriptional suppression of virulence genes HWP1 and ECE1 in C. albicans. Our findings provide valuable insights into the intricate relationship between DNA damage response and virulence in C. albicans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85478461426a57f06cbb96c87887e10f784922c8" target='_blank'>
              DNA damage repair factor Rad18 controls virulence partially via transcriptional suppression of genes HWP1 and ECE1 in Candida albicans
              </a>
            </td>
          <td>
            Runlu Chen, Yuting Feng, Huaxin Cai, Shaling Yang, Xiaoyin She, Jinrong Feng
          </td>
          <td>2024-11-21</td>
          <td>Virulence</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent evidence suggests a crucial biological role for Circular RNAs (circRNAs) in keloid diseases, yet the underlying mechanisms remain unclear. This study explored the biological effects and molecular mechanisms of hsa_circ_0002198 in keloid formation. Methods Real-time quantitative PCR (qRT-PCR) was employed to assess the expression of circ_0002198 in keloid tissues, normal skin tissues, keloid fibroblasts (KFs), and normal skin fibroblasts (NFs) from nine patients. To investigate the role of circ_0002198 in keloid pathogenesis, cell transfection technology was utilized to knock down circ_0002198. Various experiments including Cell Counting Kit-8 (CCK-8), 5-Ethynyl-2′-deoxyuridine (EdU), Transwell, wound healing assay, flow cytometry, and others were conducted to explore the potential mechanisms associated with circ_0002198 expression. The RNA-binding protein Eukaryotic translation initiation factor 4A, isoform 3 (EIF4A3) binding to circ_0002198 was identified and confirmed through bioinformatics databases prediction and RNA immunoprecipitation (RIP) assay. Finally, the expression of EIF4A3 was assessed, and both silencing and overexpression were employed to verify its role in circ_0002198 regulation. Results The expression levels of circ_0002198 and EIF4A3 were notably elevated in keloid tissues and KFs compared to normal skin tissues and NFs. The reduction of circ_0002198 expression in KFs significantly impeded their proliferation, migration, and invasion. It also hindered the cell cycle process and the expression of associated proteins while concurrently promoting apoptosis in KFs. EIF4A3 was identified to bind to the flanks of circ_0002198, enhancing the occurrence of circ_0002198 and its role in regulating the progression of KFs. Conclusion Our study offers insights into how Circular RNA may contribute to the pathogenesis of keloid formation, highlighting Circ_0002198 as a potential novel biomarker for keloids in association with EIF4A3. Further research, involving larger study cohorts, is necessary to broaden our understanding of keloid mechanisms and potential treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bfb73631e54cdc07daef4fa2c27e715644f2cfb" target='_blank'>
              EIF4A3 Enhances the Proliferation and Cell Cycle Progression of Keloid Fibroblasts by Inducing the hsa_circ_0002198 Expression
              </a>
            </td>
          <td>
            Zidi Xu, Chang Li, Xueyi Liu, Yongting Zhou, Yingbo Zhang, Jie-Xin Wang, Hao Wu, Abdullah Al-danakh, Yixuan Peng, Zhibo Xiao
          </td>
          <td>2024-12-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest solid cancers; thus, identifying more effective therapies is a major unmet need. In this study, we characterized the super enhancer (SE) landscape of human PDAC to identify drivers of the disease that might be targetable. This analysis revealed MICAL2 as a super enhancer-associated gene in human PDAC, which encodes the flavin monooxygenase MICAL2 that induces actin depolymerization and indirectly promotes SRF transcription by modulating the availability of serum response factor coactivators myocardin-related transcription factors (MRTF-A and MRTF-B). MICAL2 was overexpressed in PDAC, and high MICAL2 expression correlated with poor patient prognosis. Transcriptional analysis revealed that MICAL2 upregulates KRAS and EMT signaling pathways, contributing to tumor growth and metastasis. In loss and gain of function experiments in human and mouse PDAC cells, MICAL2 promoted both ERK1/2 and AKT activation. Consistent with its role in actin depolymerization and KRAS signaling, loss of MICAL2 also inhibited macropinocytosis. MICAL2, MRTF-A, and MRTF-B influenced PDAC cell proliferation and migration and promoted cell cycle progression in vitro. Importantly, MICAL2 supported in vivo tumor growth and metastasis. Interestingly, MRTF-B, but not MRTF-A, phenocopied MICAL2-driven phenotypes in vivo. This study highlights the multiple ways in which MICAL2 impacts PDAC biology and provides a foundation for future investigations into the potential of targeting MICAL2 for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dad317044b8f46c0a8b2261932606b72978f45" target='_blank'>
              MICAL2 Promotes Pancreatic Cancer Growth and Metastasis.
              </a>
            </td>
          <td>
            Bharti Garg, Sohini Khan, Asimina Courelli, Ponmathi Panneerpandian, Deepa Sheik Pran Babu, Evangeline S Mose, K. Gulay, Shweta Sharma, Divya Sood, Alexander T Wenzel, Alexei Martsinkovskiy, Nirakar Rajbhandari, Jay Patel, Dawn Jaquish, Edgar Esparza, Katelin Jaque, Neetu Aggarwal, Guillem Lambies, Anthony D'Ippolito, Kathryn Austgen, Brian Johnston, David A Orlando, Gung Ho Jang, Steven Gallinger, Elliot Goodfellow, P. Brodt, C. Commisso, Pablo Tamayo, Jill P. Mesirov, Hervé Tiriac, Andrew M Lowy
          </td>
          <td>2025-01-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The increased spontaneous fragility in the centromere regions of the chromosomes predisposes to abnormal cellular division due to abnormal interaction with the mitotic spindle, which leads to number chromosomal aberrations. The abnormal structure of the microtubule-associated proteins (MAPs) could also be among the internal factors, predisposing to these pathologies. On the other hand, the increased frequency of the spontaneous chromosomal fragility suggests abnormal structure of the respective chromosomal/chromatin components and thus an increased risk about gene mutations and structural chromosomal aberrations. The spontaneous chromosomal fragility was tested by light microscopy observation of metaphases from peripheral blood lymphocytes of 8 patients with polygene pathologies (neoplasms, neuro-degenerative diseases, and neuro-psychiatric disorders) and of 18 healthy controls. In the tested patients was established significantly higher frequency of the spontaneous chromosomal fragility, including in the centromere chromosomal regions, compared to the controls. Different inter-molecular interactions, which could prevent the pathology appearance, were investigated by appropriate experimental in vivo models. As a whole, in the most cases, the presented data showed significantly lower average titers of anti-ganglioside antibodies and gangliosides in the samples from myocardium and liver than in the samples from brain and pancreas. By taking into consideration that the myocardium and liver are known as the most enriched of GSH anatomic organs, the presented data could be explained with the literature messages of higher amounts of de novo-synthesized free GSH tri-peptide in the same two organs. Moreover, a possibility about the production of antibodies/immunoglobulins by non-lymphoid types of cells, tissues, and organs in appropriate conditions was shown. Because the so-produced antibodies are out of the germinative centers in the specialized lymphoid tissues and organs, the control of their functions by small ions and molecules as gangliosides is very important. In the current study, the attention was particularly directed to the iteractions with the participation of gangliosides, the reduced form of the tri-peptide glutathione (GSH), and the protein HACE1. Besides the tumor-suppressor activity of protein HACE1, also its neuroprotective influence was proven, namely by interactions with specific molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9eb48ff05dc7fb85029675d1a16bf94321085ae" target='_blank'>
              Experimental in vivo models for understanding inter-molecular interactions and preventing multifactor disease development
              </a>
            </td>
          <td>
            Iskra Sainova, Vera Kolyovska, Veselin Nanev, D. Drenska, D. Maslarov, Dimitrina Dimitrova-Dikanarova, Tzvetanka Markova
          </td>
          <td>2024-11-11</td>
          <td>Pharmacia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Angiosarcoma (AS) is an aggressive cancer of endothelial cells arising in the blood and lymphatic vasculature. Among all the rare soft tissue sarcomas, AS patients have some of the worst outcomes, with an estimated 5-year survival rate of only 30%. In mouse models, we discovered that endothelial loss of miRNAs through the conditional deletion of Dicer1 leads to AS development. Since most cells, and even most cancer cells require some level of miRNA expression for survival, this demonstrates that endothelial cells are uniquely sensitive to miRNA loss mediated transformation. Consistent with this finding we have found that Dgcr8 deletion with aP2-Cre phenocopies Dicer1 loss resulting in AS with 100% penetrance. This implicates miRNAs as critical tumor suppressors in AS. Thus, we tested the repurposing of the antibiotic, enoxacin as an anticancer therapeutic based on its known potential to enhance miRNA processing. Indeed, enoxacin significantly reduces AS cell proliferation and migration. Small RNA-sequencing revealed that several known tumor suppressing miRNAs were increased in abundance following enoxacin treatment. To discover other miRNAs critical in AS, we conducted an unbiased CRISPR-Cas9 screen with a miRNA specific gRNA library and identified several potential candidates. Among these, our preliminary data indicates miR-410 as a potent tumor suppressor. To determine the miRNA and mRNA target interactome of these miRNAs in AS, we are utilizing AgoTRIBE. This method fuses the Ago2 miRNA effector protein to the RNA editing domain of ADAR2 to generate RNA edits on transcripts that are associated with Ago2-miRNAs as determined by RNA-seq. This work will further utilize in vitro and in vivo tools to dissect the underlying biology and role of these tumor suppressing miRNAs in AS. We aim to leverage these findings to develop novel strategies for therapeutic interventions, including miRNA-based approaches that may improve patient outcomes.
 Citation Format: Bozhi Liu, Ant Murphy, Annaleighh Benton, Lauren E. Gartenhaus, Jason A Hanna. Identification, therapeutic potential, and regulatory networks of tumor suppressing miRNAs in angiosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce50a32efa6eb421263456484353dc2d754ccc64" target='_blank'>
              Abstract PR013: Identification, therapeutic potential, and regulatory networks of tumor suppressing miRNAs in angiosarcoma
              </a>
            </td>
          <td>
            Bozhi Liu, Ant Murphy, Annaleigh Benton, Lauren E Gartenhaus, Jason A. Hanna
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfa96c68ddc7260bde1a2ca148e053288fbd50cb" target='_blank'>
              Fundamentals of DNA methylation in development.
              </a>
            </td>
          <td>
            Caitlyn A Gillespie, Amrin Chowdhury, Katie A Quinn, Michael W. Jenkins, Andrew M Rollins, M. Watanabe, Stephanie M Ford
          </td>
          <td>2024-12-10</td>
          <td>Pediatric research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA polymerases frequently misincorporate ribonucleoside 5′-triphosphates into nascent DNA strands. This study examined the effects of an incorporated ribonucleoside on untargeted mutations in human cells. Riboguanosine (rG) was introduced into the downstream region of the supF gene to preferentially detect the untargeted mutations. The plasmid containing rG was transfected into U2OS cells and the replicated DNA was recovered after 48 h. The mutation analysis using the indicator Escherichia coli RF01 strain showed the frequent induction of untargeted base substitutions at the G bases of 5′-GpA-3′ dinucleotides, similar to action-at-a-distance mutations induced by an oxidatively damaged base, 8-oxo-7,8-dihydroguanine, and an apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3 (APOBEC3) cytosine deaminase. APOBEC3B was then knocked down by RNA interference and the plasmid bearing rG was introduced into the knockdown cells. The untargeted mutations at 5′-GpA-3′ sites were reduced by ~80%. These results suggested that ribonucleosides embedded in DNA induce base substitution mutations at G bases in the same strand by an APOBEC3B-dependent mechanism, implying that ribonucleosides contribute to APOBEC3-dependent cancer initiation events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb0de9186686963f356e0e609ad78d3f431dea9b" target='_blank'>
              Untargeted Mutation Triggered by Ribonucleoside Embedded in DNA
              </a>
            </td>
          <td>
            Tetsuya Suzuki, Kiyoharu Yasui, Yasuo Komatsu, Hiroyuki Kamiya
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e371697d9c914794d0b48ee32bc9a4939073aa5" target='_blank'>
              Oncogene activated human breast luminal progenitors contribute basally located myoepithelial cells
              </a>
            </td>
          <td>
            K. T. Kohler, Jiyoung Kim, R. Villadsen, Lone Rønnov-Jessen, O. Petersen
          </td>
          <td>2024-12-18</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Most cancer cells show increased chromosome missegregation, known as chromosomal instability (CIN), which promotes cancer progression and drug resistance. The underlying causes of CIN in cancer cells are not fully understood. Here we found that breast cancer cell lines show a reduced kinetochore localization of ROD, ZW10, and Zwilch, components of the fibrous corona, compared with non-transformed breast epithelial cell lines. The fibrous corona is a structure formed on kinetochores before their end-on attachment to microtubules and plays a role in efficient kinetochore capture and the spindle assembly checkpoint. The reduction in the fibrous corona was not due to reduced expression levels of the fibrous corona components or to a reduction in outer kinetochore components. Kinetochore localization of Bub1 and CENP-E, which play a role in the recruitment of the fibrous corona to kinetochores, was reduced in cancer cell lines, presumably due to reduced activity of Mps1, which is required for their recruitment to kinetochores through phosphorylating KNL1. Increasing kinetochore localization of Bub1 and CENP-E in cancer cells restored the level of the fibrous corona. Cancer cell lines showed a reduced capacity to nucleate microtubules from kinetochores, which was recently shown to be dependent on the fibrous corona, and increasing kinetochore localization of Bub1 and CENP-E restored the microtubule nucleation capacity on kinetochores. Our study revealed a distinct feature of cancer cell lines that may be related to CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54368cadf3707166a1cb068412715ab9475ab2b6" target='_blank'>
              Fibrous corona is reduced in cancer cell lines that attenuate microtubule nucleation from kinetochores.
              </a>
            </td>
          <td>
            Yudai Ishikawa, Hirotaka Fukue, Runa Iwakami, Masanori Ikeda, K. Iemura, Kozo Tanaka
          </td>
          <td>2024-11-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fee039445989781ab8a7ddb4e737986ea83ee7ff" target='_blank'>
              Targeting mitochondrial RNAs enhances the efficacy of the DNA-demethylating agents
              </a>
            </td>
          <td>
            Stephanie Tan, Sujin Kim, Yoo-Ree Kim
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract We previously reported endogenous activation of the DNA damage response (DDR) in the epidermis surrounding basal cell carcinoma resected from Nagasaki atomic bomb survivors, suggesting the presence of genomic instability (GIN) in the survivors as a late effect of radiation. Dual-color immunofluorescence (IF) analysis of TP53-binding protein-1 (53BP1) and a proliferative indicator, Ki-67, to elucidate GIN in tumor tissues revealed that abnormal 53BP1 expression is closely associated with carcinogenesis in several organs. The present study aimed to confirm the presence of radiation-induced GIN in the non-neoplastic epidermis of patients with radiation-induced skin cancer. Formalin-fixed paraffin-embedded tissues were obtained from all participants between 2008 and 2019 at the Nagasaki University Hospital. 53BP1 nuclear expression was examined using dual-color IF analysis with Ki-67 expression to assess the extent and integrity of the DDR. Expressions of gamma-H2AX, p53 and p21 were also analyzed using the dual-color IF analysis for their association with 53BP1. The results of this study provide evidence for sporadic activation of the DDR in medically irradiated and ultraviolet-exposed epidermis as a long-lasting radiation effect, which is a predisposition to skin cancer. Furthermore, the incidence of abnormal 53BP1 expression in cancer cells was higher than in non-neoplastic epidermal cells surrounding cancer, suggesting a correlation between the type of 53BP1 and the malignant potential of skin tumors. This study highlights the usefulness of dual-color IF for 53BP1 (and Ki-67) as an indicator to estimate the level of GIN as a long-lasting health effect of radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab225f148a06d2534be09b0fe51d429a9fdf5fb" target='_blank'>
              Detection of genome instability by 53BP1 expression as a long-lasting health effect in human epidermis surrounding radiation-induced skin cancers
              </a>
            </td>
          <td>
            Katsuya Matsuda, Hirokazu Kurohama, Yutaka Kuwatsuka, Akira Iwanaga, Hiroyuki Murota, Masahiro Nakashima
          </td>
          <td>2024-12-01</td>
          <td>Journal of Radiation Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Type 1 diabetes (T1D) is characterized by a prolonged autoimmune attack resulting in the massive loss of insulin-producing beta cells. The initiation and progression of T1D depends on a complex interaction between genetic, immunological and environmental factors. Epidemiological, experimental and clinical evidence suggest a link between viral infections, particularly Coxsackievirus type B (CVB), and T1D development. Specifically, infections by the CVB serotype 1 (CVB1) contribute to the triggering of autoimmunity against beta cells in genetically predisposed individuals, and prolonged and probably non-lytic infections by CVB are associated with the development of T1D. However, the molecular mechanisms underlying CVB1 replication and establishing persistent infections in human pancreatic beta cells remain poorly understood. Here we show that the N6-methyladenosine (m6A) RNA epigenetic modification machinery regulates CVB1 amplification in the human beta cells. Using small interfering RNA (siRNA) targeting m6A writers and erasers, we observed that downregulation of m6A writers increases CVB1 amplification, while the downregulation of m6A erasers decreases it. Notably, the inhibition of Fat Mass and Obesity-associated protein (FTO), a key m6A eraser, reduced by 95% the production of infectious CVB1 in both human insulin-producing EndoC-βH1 cells and in induced pluripotent stem cell (iPSC)-derived islets. The FTO inhibitor reduced CVB1 expression within 6 h post-infection, suggesting a direct regulation of the CVB1 genome by m6A modification. Furthermore, in the absence of viral replication, FTO inhibition also decreased the translation of the incoming CVB1 genome, indicating that m6A plays a critical role in the initial stages of viral RNA translation. In addition, modulation of the m6A machinery affected the type I interferon response after poly-IC transfection, a mimic of RNA virus replication, but did not affect the cellular antiviral response in CVB1-infected cells. Altogether, these observations suggest that m6A directly affects CVB1 production. Our study provides the first evidence that the m6A epigenetic modification machinery controls CVB amplification in human pancreatic beta cells. This suggests that the m6A machinery is a potential target to control CVB infection in T1D and raises the possibility of an epigenetic control in the establishment of persistent CVB infections observed in the pancreas in individuals with type 1 diabetes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f84a79510112b5ffb5b1f824dbf1aa61a0de7c" target='_blank'>
              The N6-methyladenosine RNA epigenetic modification modulates the amplification of coxsackievirus B1 in human pancreatic beta cells
              </a>
            </td>
          <td>
            Maressa Fernandes Bonfim, Camille Aitchedji, Flore Van Goethem, Lionel Sauvage, Thibault Poinsot, E. Calonne, Rachel Deplus, François Fuks, D. Eizirik, A. Op de Beeck
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background/objectives: Nucleolin is a major component of the nucleolus and is involved in various aspects of ribosome biogenesis. However, it is also implicated in non-nucleolar functions such as cell cycle regulation and proliferation, linking it to various pathologic processes. The aim of this study was to use differential gene expression analysis and Weighted Gene Co-expression Network analysis (WGCNA) to identify nucleolin-related regulatory pathways and possible key genes as novel therapeutic targets for cancer, viral infections and other diseases. Methods: We used two different siRNAs to downregulate the expression of nucleolin in a human hepatoblastoma (HepG2) cell line. We carried out RNA-sequencing (RNA-Seq), performed enrichment analysis of the pathways of the differentially expressed genes (DEGs) and identified protein–protein interaction (PPI) networks. Results: Both siRNAs showed high knockdown efficiency in HepG2 cells, resulting in the disruption of the nucleolar architecture and the downregulation of rRNA gene expression, both downstream hallmarks of a loss of nucleolin function. RNA-Seq identified 44 robust DEGs in both siRNA cell models. The enrichment analysis of the pathways of the downregulated genes confirmed the essential role of nucleolin in DNA replication and cell cycle processes. In addition, we identified seven hub genes linked to NCL: MCM6, MCM3, FEN1, MYBL2, MSH6, CDC6 and RBM14; all are known to be implicated in DNA replication, cell cycle progression and oncogenesis. Conclusions: Our findings demonstrate the functional consequences of nucleolin depletion in HepG2 and confirm the importance of nucleolin in DNA replication and cell cycle processes. These data will further enhance our understanding of the molecular and pathologic mechanisms of nucleolin and provide new therapeutic perspectives in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dd5e76e6da6ae07561d89499ef908c15696609" target='_blank'>
              RNA Sequencing and Weighted Gene Co-Expression Network Analysis Highlight DNA Replication and Key Genes in Nucleolin-Depleted Hepatoblastoma Cells
              </a>
            </td>
          <td>
            Hannes Steinkellner, Silvia Madritsch, Mara Kluge, Teresa Seipel, Victoria Sarne, Anna Huber, Markus Schosserer, Raimund Oberle, Winfried Neuhaus, Alexander V. Beribisky, Franco Laccone
          </td>
          <td>2024-11-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="It is important to systematically identify tumor suppressor genes (TSGs) to improve our understanding of tumorigenesis and develop strategies for early diagnosis and mitigating disease progression. In the present study, we used an in vivo genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein 9 (Cas9) screen and identified FPS/FES-related (FER) as a TSG. Single-cell RNA sequencing (scRNA-seq) revealed that normal cells with low FER expression exhibited elevated malignant transformation potential and stemness properties. FER knockout promoted the tumorigenic transformation, characterized by high colony-forming efficiency and suspension growth ability, acquired tumorigenicity in vivo, increased metabolic activity, dedifferentiation properties, and immune evasion. Moreover, analysis revealed that low FER expression tumors share molecular phenotypes with FER knockout cells, suggesting the consistent role of FER in tumor initiation and progression. Taken together, our findings not only provide insights into the essential role of FER as a tumor suppressor in tumor initiation and progression but also highlight its potential as a target for future clinical diagnosis. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0ef168e6b6a500c7fdbd949d7e67ca980d8052" target='_blank'>
              CRISPR-Cas9 screening identifies the role of FER as a tumor suppressor.
              </a>
            </td>
          <td>
            Jiaqi Wang, Ran Yang, Feng-Po Wang, Junlei Zhang, Yutong Dong, Jiangjun Wang, Meng Yu, Yixiao Xu, Lianlian Liu, Yuda Cheng, Chen Zhang, Yi Yang, Wubin Yang, Jiali Wang, Guangxing Chen, Yi Huang, Yanping Tian, Rui Jian, Bing Ni, Wei Wu, Y. Ruan
          </td>
          <td>2024-12-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="According to the principles of Evolutionary Cancer Cell Biology (ECCB), cancer stem cells (CSCs) do not derive from normal stem cells; rather, they originate from a distinct functional phenotype of germline cells characterized by asymmetric cell division (ACD). This phenotype proliferates through ACD, producing self-renewing cells alongside non-proliferating daughter cells with CSC qualities. ECCB posits that CSCs do not proliferate. Similar to protists, there exists a close reciprocal relationship between sister cells that collectively form a germline and stem cells. These sister cells perform complementary roles: proliferating cancer germline cells generate stem cells, whereas the non-proliferating CSCs give rise to progenitor cells for the formation of new germline clones. ECCB distinguishes between primary CSCs, which are associated with carcinogenesis and primary tumors, and secondary CSCs, which are linked to metastases. This unicellular stem cell system is homologous to that of parasitic protists, such as amebae. Both CSCs and amebae stem cells are produced by an oxygen-sensitive germline and are vulnerable to damage when oxygen levels exceed 6.0% (germline hyperoxia), as elevated oxygen concentrations can harm the germline genome. Germline cells that lose their stemness quality continue to cycle through defective symmetric cell divisions (DSCD). However, to restore functionality, the DSCD genome must be repaired through hyperpolyploidization. This process occurs in native polyploid giant cancer cells, which are homologous to the multinucleated genome repair structures found in protists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22fd9b25c7727535490357470b16f77d5c7c81c2" target='_blank'>
              Reevaluating cancer stem cells and polyploid giant cancer cells from the evolutionary cancer cell biology perspective
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2024-11-21</td>
          <td>Cancer Plus</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c2410d9b633d4c17e5953af5897b6315595eef9" target='_blank'>
              KDM1A-mediated ZFP64 demethylation activates CENPL to promote epithelial ovarian cancer progression.
              </a>
            </td>
          <td>
            Jie Wang, Xinjian Fang, Yajun Xing, M. Ding, Liangxue Zhu, Mingyun Wang
          </td>
          <td>2024-12-01</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is the most lethal tumor arising from thyroid follicular epithelium. Lenvatinib is an off-label use option for ATC patients in many countries but an approved prescription in Japan. However, lenvatinib resistance is a substantial clinical challenge. Clinical ATC samples including lenvatinib-resistant tumors are used to build patient-derived cells and patient-derived xenografts. High-throughput drug screening and synergy analyses are performed to identify an effective combination partner for lenvatinib. Cellular functions are detected by cell senescence, apoptosis, cell cycle, cell viability and colony formation assays. CDK2 inhibition showed the significant synthetic lethality with lenvatinib via inhibiting G1/S transition and inducing cell senescence in ATC. High expression of CDK2 is associated with lenvatinib resistance and poor clinical outcomes of ATC patients. Lenvatinib increased protein expression of CDK2 in lenvatinib-resistant ATC cells. Mechanistically, lenvatinib inhibited protein degradation of CDK2 via reducing CDK2's interaction with the RACK1-FBW7 complex, which is involved in ubiquitination and subsequent proteasomal degradation of CDK2. Combination of CDK2 inhibitors in clinical trials (Dinaciclib or PF-07104091) and lenvatinib markedly suppressed growth of xenograft tumors from the lenvatinib-resistant patient. The findings support the combination therapy strategy of lenvatinib and CDK2 inhibitor for lenvatinib-resistant ATC patients with high CDK2 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2b7e2d82a92cc7bf250d9214904a5c6b09e971" target='_blank'>
              Targeting CDK2 Confers Vulnerability to Lenvatinib Via Driving Senescence in Anaplastic Thyroid Cancer.
              </a>
            </td>
          <td>
            Ben Ma, Youzhou Sang, Xiaoxue Du, Yanzhi Zhang, Min Yin, Weibo Xu, Wanlin Liu, Jiayi Lu, Qing Guan, Yunjun Wang, Tian Liao, Yuting Wang, J. Xiang, Rongliang Shi, N. Qu, Qinghai Ji, Jiwei Zhang, Dongmei Ji, Yu Wang
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Telomeres protect the terminal ends of eukaryotic chromosomes, making them essential for genome stability, cellular senescence, and species evolution. These structures consist of tandem repeats of nucleotides, referred to as telomeric repeat sequences. The enzyme telomerase, along with various telomere-binding proteins, manages the synthesis of these repeat sequences. Telomerase functions by adding telomeric repeats to the ends of chromosomes, counteracting the shortening that occurs during DNA replication. In most somatic cells, telomerase activity is generally low or absent, leading to gradual telomere loss and, eventually, cellular senescence. Although the ancestral telomeric repeat sequence is often proposed to be TTAGGG, studies have shown that convergent evolution has produced similar telomeric motifs in various evolutionary lineages. This is evidenced by variations in telomeric repeat sequences among organisms, which may include single or double nucleotide substitutions, deletions, and even exceptionally large telomeric repeats in certain species. Investigating these telomeric repeat sequences across a wide range of organisms and identifying the evolutionary changes within each phylum offer valuable insights into telomeric sequences and the telomerases responsible for their synthesis. Such research not only deepens our understanding of telomere biology across different organisms but also holds potential for addressing age-related challenges and cancer, paving the way for innovative solutions to some of the most critical issues in contemporary medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c54b2377d4f8de003848b40b972b9328835" target='_blank'>
              A Comprehensive Review on the Telomeric Repeat Sequence in Different Organisms
              </a>
            </td>
          <td>
            Monish Prasanna, Kousik Varadan, Achsha Babu, Arun Arumugaperumal
          </td>
          <td>2024-11-18</td>
          <td>Medinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c804fe505d873573d79f164c86523ae09b15f4c" target='_blank'>
              PCNA and Rnh1 independently participate in the protection of mitochondrial genome against UV-induced mutagenesis in yeast cells
              </a>
            </td>
          <td>
            Martyna Latoszek, Katarzyna Bagińska-Drabiuk, E. Sledziewska-Gojska, Aneta Kaniak-Golik
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Chemical modification of RNAs is important for posttranscriptional gene regulation. The METTL3-METTL14 complex generates most N6-methyladenosine (m6A) modifications in messenger RNAs (mRNAs), and dysregulated methyltransferase expression has been linked to cancers. Here we show that a changed sequence context for m6A can promote oncogenesis. A gain-of-function missense mutation from patients with cancer, METTL14R298P, increases malignant cell growth in culture and transgenic mice without increasing global m6A levels in mRNAs. The mutant methyltransferase preferentially modifies noncanonical sites containing a GGAU motif, in vitro and in vivo. The m6A in GGAU context is detected by the YTH family of readers similarly to the canonical sites but is demethylated less efficiently by an eraser, ALKBH5. Combining the biochemical and structural data, we provide a model for how the cognate RNA sequences are selected for methylation by METTL3-METTL14. Our work highlights that sequence-specific m6A deposition is important and that increased GGAU methylation can promote oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb873e5c13473ef69e6a927a44ac7bf8be076b4" target='_blank'>
              Cancer mutations rewire the RNA methylation specificity of METTL3-METTL14
              </a>
            </td>
          <td>
            Chi Zhang, Robyn L Scott, Luiza Tunes, Meng-Hsiung Hsieh, Ping Wang, Ashwani Kumar, Brijesh B Khadgi, Yen-Yu Yang, Katelyn A Doxtader Lacy, E. Herrell, Xunzhi Zhang, Bret M. Evers, Yinsheng Wang, Chao Xing, Hao Zhu, Yunsun Nam
          </td>
          <td>2024-12-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [10, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>