<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-07 09:28:35 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>30</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>67</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>106</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>320</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>279</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis. Both breakage fusion bridge (BFB) cycles and extrachromosomal DNA (ecDNA) can lead to high oncogene copy numbers, but the impact of BFB amplifications on intratumoral heterogeneity, treatment response, and patient survival remains poorly understood due to detection challenges with DNA sequencing. We introduce an algorithm, OM2BFB, designed to detect and reconstruct BFB amplifications using optical genome mapping (OGM). OM2BFB demonstrates high precision (>93%) and recall (92%) in identifying BFB amplifications across cancer cell lines, patient-derived xenograft models, and primary tumors. Comparisons using OGM reveal that BFB detection with our AmpliconSuite toolkit for short-read sequencing also achieves high precision, though with reduced sensitivity. We identify 371 BFB events through whole genome sequencing of 2557 primary tumors and cancer cell lines. BFB amplifications are prevalent in cervical, head and neck, lung, and esophageal cancers, but rare in brain cancers. Genes amplified through BFB exhibit lower expression variance, with limited potential for regulatory adaptation compared to ecDNA-amplified genes. Tumors with BFB amplifications (BFB(+)) show reduced structural heterogeneity in amplicons and delayed resistance onset relative to ecDNA(+) tumors. These findings highlight ecDNA and BFB amplifications as distinct oncogene amplification mechanisms with differing biological characteristics, suggesting distinct avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="As a well-known cancer risk region, the 8q24 locus is frequently amplified in a variety of solid tumors. Here we identify a pseudogene-derived oncogenic lncRNA, ZNF252P, which is upregulated in a variety of cancer types by copy number gain as well as c-Myc-mediated transcriptional activation. Mechanistically, ZNF252P binds and drives “phase separation” of HNRNPK and ILF3 protein in the nucleus and cytoplasm, respectively, to transcriptionally and posttranscriptionally activate c-Myc, thus forming a c-Myc/ZNF252P/c-Myc positive feedback loop. These findings expand the understanding of the relationship between genomic instability in the 8q24 region and tumorigenesis and clarify a regulatory mechanism involved in transcription and posttranscription from the perspective of RNA-mediated nuclear and cytoplasmic protein phase separation, which sheds light on the dialogue with the driver oncogene c-Myc. The pivotal regulatory axis of ZNF252P/c-Myc has potential as a promising biomarker and therapeutic target in cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution.
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-03</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Introduction The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood. Methods In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU). Results Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome. Discussion These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, S. Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="N6-methyladenosine (m6A), the most abundant internal RNA modification in humans, regulates most aspects of RNA processing. Prostate cancer is characterized by widespread transcriptomic dysregulation; therefore, we characterized the m6A landscape of 162 localized prostate tumors with matched DNA, RNA and protein profiling. m6A abundance varied dramatically across tumors, with global patterns emerging via complex germline-somatic cooperative regulation. Individual germline polymorphisms regulated m6A abundance, cooperating with somatic mutation of cancer driver genes and m6A regulators. The resulting complex patterns were associated with prognostic clinical features and established the biomarker potential of global and locus-specific m6A patterns. Tumor hypoxia dysregulates m6A profiles, bridging prior genomic and proteomic observations. Specific m6A sites, such as those in VCAN, drive disease aggression, associating with poor outcomes, tumor growth and metastasis. m6A dysregulation is thus associated with key events in the natural history of prostate cancer: germline risk, microenvironmental dysregulation, somatic mutation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19695af6dbb1a44b8e5db5cde53d9277c7377300" target='_blank'>
              The landscape of N6-methyladenosine in localized primary prostate cancer.
              </a>
            </td>
          <td>
            Xin Xu, Helen Zhu, Rupert Hugh-White, J. Livingstone, Stefan E. Eng, Nicole Zeltser, Yujuan Wang, Kinga Pajdzik, Sujun Chen, Kathleen E. Houlahan, Wenqin Luo, Shun Liu, Xi Xu, Minzhi Sheng, Wangyuan Guo, J. Arbet, Yuxi Song, Miranda Wang, Yong Zeng, Shiyan Wang, Guanghui Zhu, Tingxiao Gao, Wei Chen, Xinpei Ci, Wenjie Xu, Kexin Xu, M. Orain, V. Picard, H. Hovington, A. Bergeron, L. Lacombe, Bernard Têtu, Yves Fradet, Mathieu Lupien, Gong-Hong Wei, Marianne Koritzinsky, Robert G Bristow, Neil Fleshner, Xue Wu, Yang Shao, Chuan He, Alejandro Berlin, T. H. van der Kwast, Hon Leong, P. Boutros, H. H. He
          </td>
          <td>2025-03-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, driving oncogenic mutations that enhance tumor aggressiveness and drug resistance. MYC, a master transcription factor that is deregulated in nearly all human tumors, paradoxically induces replication stress and associated DNA damage while also increasing expression of DNA repair factors and mediating resistance to DNA-damaging therapies. Emerging evidence supports a non-transcriptional role for MYC in preserving genomic integrity at sites of active transcription and protecting stalled replication forks under stress. Understanding how MYC’s genotoxic and genoprotective functions diverge may reveal new therapeutic strategies for MYC-driven cancers. Here, we identify a non-canonical role of MYC in DNA damage response (DDR) through its direct association with DNA breaks. We show that phosphorylation at serine 62 (pS62-MYC) is crucial for the efficient recruitment of MYC to damage sites, its interaction with repair factors BRCA1 and RAD51, and effective DNA repair to support cell survival under stress. Mass spectrometry analysis with MYC-BioID2 during replication stress reveals a shift in MYC’s interactome, maintaining DDR associations while losing transcriptional regulators. These findings establish pS62-MYC as a key regulator of genomic stability and a potential therapeutic target in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73224953c4cbe478fbdd30504471b72d2f0b5d71" target='_blank'>
              MYC Serine 62 phosphorylation promotes its binding to DNA double strand breaks to facilitate repair and cell survival under genotoxic stress
              </a>
            </td>
          <td>
            Gabriel M. Cohn, Colin J. Daniel, Jenny Eng, Xiao-Xin Sun, C. Pelz, Koei Chin, Alexander Smith, Charles D. Lopez, Jonathan R. Brody, M. Dai, Rosalie C. Sears
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Prior studies showed that structural loops collapse upon acute cohesin depletion, while regulatory enhancer-promoter (E-P) loops largely persist, consistent with minimal transcriptional changes. However, these studies, conducted in asynchronous cells, could not resolve whether cohesin is required for the establishment of regulatory interactions and transcriptional activation during cell division or cell state transitions. To address this gap, we degraded RAD21, a core cohesin subunit, in naïve mouse embryonic stem cells (ESCs) transitioning from mitosis to G1 either in self-renewal condition or during differentiation toward formative pluripotency. Although most structural loops failed to be re-established without cohesin, about 35% of regulatory loops reformed at normal or higher frequencies. Cohesin-independent loops showed characteristics of strong active enhancers and promoters and a significant association with H3K27ac mitotic bookmarks. However, inhibition of CBP/p300 during mitotic exit did not impact these cohesin-independent interactions, suggesting the presence of complex compensatory mechanisms. At the transcriptional level, cohesin depletion induced only minor changes, supporting that post-mitotic transcriptional reactivation is largely independent of cohesin. The few genes with impaired reactivation were directly bound by RAD21 at their promoters, engaged in many structural loops, and located within strongly insulated TADs with low gene density. Importantly, degrading cohesin during the M-to-G1 transition in the presence of EpiLC differentiation signals revealed a larger group of susceptible genes, including key signature genes and transcription factors. Impaired activation of these genes was partly due to the failure to establish de novo EpiLC-specific interactions in the absence of cohesin. These experiments revealed locus-specific and context-specific dependencies between cohesin, E-P interactions, and transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3acd22c348292a09acc57cb36b9467e0afd8f399" target='_blank'>
              Post-mitotic transcriptional activation and 3D regulatory interactions show locus- and differentiation-specific sensitivity to cohesin depletion
              </a>
            </td>
          <td>
            UkJin Lee, Alejandra Laguillo-Diego, Wilfred Wong, Zhangli Ni, Lingling Cheng, Jieru Li, B. Pelham-Webb, A. Pertsinidis, Christina S. Leslie, E. Apostolou
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="One fundamental yet open question is how eukaryotic chromosomes fold into segregated territories, a process essential for gene transcription and cell fate. Through analyzing Hi-C and chromatin-tracing DNA-FISH data, we identify long-range chromo skeleton loop structures that span over 100 Mb, extending beyond the reach of loop extrusion models. Spatial density analyses point to assembly formation independent of major nuclear structures. A subset of genomic loci serves as nucleation centers, driving loop clustering. These complexes are highly stable, as shown by live-cell imaging with sequence-specific fluorescent labeling, and biophysical model analyses reveal a multivalent binding mechanism. Our findings suggest a redundant, distributed cluster mechanism that ensures robustness across cell types and against mutations, guiding both chromosome compaction and the formation of smaller-scale chromosomal structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04b5ac6a8c671d63e4c9bd85fcca4ea46155b48" target='_blank'>
              Long-range genomic loci stochastically assemble into combinatorial forms of chromosome skeleton
              </a>
            </td>
          <td>
            Jingyu Zhang, Siyuan Wang, Simon C Watkins, Jianhua Xing
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In many organisms, meiotic crossover recombination is suppressed near the extreme ends of chromosomes. Here, we identified two chromatin modifiers, the histone methyltransferase Dot1 and the Sir silencing complex, as regulators of this process in Saccharomyces cerevisiae. We show that the recombination-promoting axis proteins Red1 and Hop1, but not the axis-associated cohesin Rec8, are significantly reduced within 20 kb of telomeres compared to the chromosome interior. Dot1, which preferentially methylates histones in the chromosome interior, is required for this pattern by directing Red1 binding toward the chromosome interior. In parallel, the Sir complex suppresses the induction of meiotic DNA double-strand breaks (DSBs) at chromosome ends. Sir-dependent DSB suppression is independent of axis deposition and occurs in a chromosome end-specific manner that mirrors the spreading and transcriptional silencing activity of the complex, suggesting that the Sir complex suppresses DSB formation by limiting the openness of promoters, the preferred sites of meiotic DSB formation. We conclude that multiple chromatin-based mechanisms collaborate to achieve a robust reduction of meiotic recombination near chromosome ends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091c445669f4b199e48685b922281d0349f41b22" target='_blank'>
              Distinct chromatin regulators downmodulate meiotic axis formation and DNA break induction at chromosome ends
              </a>
            </td>
          <td>
            A. Raghavan, Kieron May, Vijayalakshmi V. Subramanian, Hannah G. Blitzblau, Neem J. Patel, Jon Houseley, Andreas Hochwagen
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22caf0363586789d3ef13130aa7bdb69bf876e9" target='_blank'>
              Nuclear and genome dynamics underlying DNA double-strand break repair.
              </a>
            </td>
          <td>
            I. Chiolo, Matthias Altmeyer, G. Legube, Karim Mekhail
          </td>
          <td>2025-03-17</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47bd8518f8db94e121fb18dd96d7456a23942c6d" target='_blank'>
              Polycomb group proteins protect latent Kaposi sarcoma-associated herpesvirus from episome clearance and HUSH-dependent chromatin silencing
              </a>
            </td>
          <td>
            Simon Weissmann, Alexis Robitaille, Thomas Günther, Marion Ziegler, A. Grundhoff
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract Regulator of telomere length 1 (RTEL1) helicase facilitates telomere replication by disassembling DNA secondary structures, such as G-quadruplexes and telomeric loops (t-loops), at the ends of the chromosomes. The recruitment of RTEL1 to telomeres occurs during the S-phase of the cell cycle, but the dynamics of the process has not been studied. Here, we utilized CRISPR genome editing and single-molecule imaging to monitor RTEL1 movement within human cell nuclei. RTEL1 utilizes rapid three-dimensional diffusion to search for telomeres and other nuclear targets. Only 5% of the chromatin-bound RTEL1 is associated with telomeres at any time in the S-phase, but the telomere-bound RTEL1 has much more extended associations. This binding is enhanced by the interaction between RTEL1 and the telomeric protein TRF2 but is largely independent of RTEL1 ATPase activity. The absence of RTEL1 catalytic activity leads to severe defects in cell proliferation, slow progression out of S-phase, and chromosome end-to-end fusion events. We propose that the rapid diffusion of RTEL1 allows this low-abundance protein to explore the nucleus, bind TRF2, and be recruited to telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d521287870dd985b41001fde2641677325213069" target='_blank'>
              Rapid dynamics allow the low-abundance RTEL1 helicase to promote telomere replication
              </a>
            </td>
          <td>
            Guanhui Wu, Erin Taylor, Daniel T. Youmans, Nausica Arnoult, Thomas R. Cech
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Abstract The alternative lengthening of telomeres (ALT) pathway is a telomerase-independent mechanism for immortalization in cancer cells and is commonly activated in low-grade and high-grade glioma, as well as osteosarcoma. The ALT pathway can be activated under various conditions and has often been shown to include mutational loss of ATRX. However, this is insufficient in isolation and so other cellular event must also be implicated. It has been shown that excessive accumulation of DNA:RNA hybrid structures (R-loops) and/or formation of DNA–protein crosslinks (DPCs) can be other important driving factors. The underlying cellular events leading to R-loop and DPC formation in ALT cancer cells to date remain unclear. Here, we demonstrate that excessive cellular reactive oxygen species (ROS) is an important causative factor in the evolution of ALT-telomere maintenance in ATRX-deficient glioma. We identified three sources of elevated ROS in ALT-positive gliomas: co-mutation of SETD2, downregulation of DRG2, and hypoxic tumour microenvironment. We demonstrate that elevated ROS leads to accumulation of R-loops and, crucially, resolution of R-loops by the enzyme RNase H1 prevents ALT pathway activity in cells exposed to elevated ROS. Further, we found a possible causal link between the formation of R-loops and the accumulation of DPCs, in particular, formation of TOP1 complexes covalently linked to DNA (Top1cc). We also demonstrate that elevation of ROS can trigger over-activity of the ALT pathway in osteosarcoma and glioma cell lines, resulting in excessive DNA damage and cell death. This work presents important mechanistic insights into the endogenous origin of excessive R-loops and DPCs in ALT-positive cancers, as well as highlighting potential novel therapeutic approaches in these difficult-to-treat cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33ca69e995ecf5806fb82d5a6c050835cc9faa" target='_blank'>
              Elevated reactive oxygen species can drive the alternative lengthening of telomeres pathway in ATRX-null cancers
              </a>
            </td>
          <td>
            Tomas Goncalves, S. Cunniffe, Tiffany S Ma, Natalie Mattis, Andrew W Rose, Thomas B. Kent, David R Mole, Helene E B Geiller, Linda van Bijsterveldt, Timothy C Humphrey, Ester M. Hammond, Richard J Gibbons, David Clynes, Anna M Rose
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genomic heterogeneity has largely been overlooked in single-cell replication timing (scRT) studies. Here, we develop MnM, an efficient machine learning-based tool that allows disentangling scRT profiles from heterogenous samples. We use single-cell copy number data to accurately perform missing value imputation, identify cell replication states, and detect genomic heterogeneity. This allows us to separate somatic copy number alterations from copy number changes resulting from DNA replication. Our methodology brings critical insights into chromosomal aberrations and highlights the ubiquitous aneuploidy process during tumorigenesis. The copy number and scRT profiles obtained by analysing >119,000 high-quality human single cells from different cell lines, patient tumours and patient-derived xenograft samples leads to a multi-sample heterogeneity-resolved scRT atlas. This atlas is an important resource for cancer research and demonstrates that scRT profiles can be used to study replication timing heterogeneity in cancer. Our findings also highlight the importance of studying cancer tissue samples to comprehensively grasp the complexities of DNA replication because cell lines, although convenient, lack dynamic environmental factors. These results facilitate future research at the interface of genomic instability and replication stress during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Oncogenes drive cancer progression, but few are active exclusively in tumor cells. Connexins (Cxs), traditionally recognized as ion channel proteins, can localize to the nucleus and regulate gene expression, playing key roles in both physiological and pathological processes. Cx46, once thought to be restricted to the eye lens, has been implicated in tumor growth, though its underlying mechanisms remain unclear. This study investigates the nuclear presence of Cx46 in cancer cells and its potential role as a transcriptional modulator. Methods We employed ChIP-Seq, confocal immunofluorescence, and nuclear protein purification to assess Cx46 localization and DNA interactions. Functional assays were conducted to evaluate its effects on invasion, division, spheroid formation, and mesenchymal marker expression. Single-point mutations and molecular dynamics simulations were used to explore potential Cx46-DNA interactions. Results Cx46 mRNA upregulation was found in a variety of tumors compared to adjacent healthy tissue. In HeLa cells, which do not express Cx46, its transfection promoted proliferation, invasion and self-renewal capacity, cancer stem cell traits and mesenchymal features. Consistently, in Sk-Mel-2, which naturally express Cx46, reduced Cx46 expression led to a decrease in the similar parameters. In HeLa cells, nuclear Cx46 was detected in two forms, full length 46 kDa and a 30 kDa fragment (GJA3-30 k), ChIP-Seq experiments revealed that Cx46 binds to the DNA at intergenic and promoter regions, leading to the activation of oncogenic pathways. Molecular dynamics simulations suggest that GJA3-30 k dimerizes in a RAD50-like structure, forming stable DNA complexes. Cx46 and in some cases GJA3-30 k were detected in the nuclei of multiple cancer cell lines, including prostate, breast and skin cancers. Conclusions Our findings reveal a novel nuclear role for Cx46 in cancer, demonstrating its function as a transcriptional regulator and its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02151-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60952dbc2d22fbba5173287bd600c08d8141690" target='_blank'>
              Connexin46 in the nucleus of cancer cells: a possible role as transcription modulator
              </a>
            </td>
          <td>
            Ainoa Fernández-Olivares, Viviana P Orellana, Jesús Llanquinao, Gonzalo Nuñez, Pablo Pérez-Moreno, Sebastián Contreras-Riquelme, Alberto JM Martin, Fabio Mammano, Ivan E Alfaro, Juan F Calderón, Jimmy Stehberg, Mauricio A Sáez, Mauricio A Retamal
          </td>
          <td>2025-03-27</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc713b72314da7faa35c83a1d8de687d85095e55" target='_blank'>
              HRDetect in Ovarian Carcinoma: Stratification and Therapeutic Implications.
              </a>
            </td>
          <td>
            K. Banda, H. R. Davies, Y. Kumar, Y. Memari, A. Degasperi, M. Radke, I. Rodriguez, T. A. Gooley, F. Menghi, T. Harding, K. Lin, E. T. Liu, S. Nik-Zainal, E. M. Swisher
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Replicative stress (RS) is emerging as a promising therapeutic target in oncology, yet full exploitation of its potential requires a detailed understanding of the mechanisms and genes involved. Here, we investigated the RNA helicase Senataxin (SETX), an enzyme that resolves RNA-DNA hybrids and R-loops, to address its role in preventing RS by oncogenic Myc. Upon Myc activation, silencing of SETX led to selective engagement of the DNA damage response (DDR) and robust cytotoxicity. Pharmacological dissection of the upstream kinases regulating the DDR uncovered a protective role of the ATR pathway, that once inhibited, boosted SETX driven-DDR. While SETX loss did not lead to a genome-wide increase of R-loops, mechanistic analyses revealed enhanced R-loops localized at DDR-foci and newly replicated genomic loci, compatible with a selective role of SETX in resolving RNA-DNA hybrids to alleviate Myc-induced RS. Genome-wide mapping of DNA double-strand breaks confirmed that SETX silencing exacerbated DNA damage at transcription-replication conflict (TRC) regions at early replicated sites. We propose that SETX prevents Myc-induced TRCs by resolving transcription-associated R-loops that encounter the replisome. The identification of SETX as a genetic liability of oncogenic Myc opens up new therapeutic options against aggressive Myc-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer exhibits altered metabolism, influencing its response to radiation. To investigate the metabolic regulation of radiation response, we conducted a comprehensive, metabolic-wide CRISPR-Cas9 loss-of-function screen using radiation as selection pressure in human non–small cell lung cancer. Lipoylation emerged as a key metabolic target for radiosensitization, with lipoyltransferase 1 (LIPT1) identified as a top hit. LIPT1 covalently conjugates mitochondrial 2-ketoacid dehydrogenases with lipoic acid, facilitating enzymatic functions involved in the tricarboxylic acid cycle. Inhibiting lipoylation, either through genetic LIPT1 knockout or a lipoylation inhibitor (CPI-613), enhanced tumor control by radiation. Mechanistically, lipoylation inhibition increased 2-hydroxyglutarate, leading to H3K9 trimethylation, disrupting TIP60 recruitment and ataxia telangiectasia mutated (ATM)–mediated DNA damage repair signaling, impairing homologous recombination repair. In summary, our findings reveal a critical role of LIPT1 in regulating DNA damage and chromosome stability and may suggest a means to enhance therapeutic outcomes with DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c8836655ab7f9cea8b4528bffe4e53726a560" target='_blank'>
              Lipoylation inhibition enhances radiation control of lung cancer by suppressing homologous recombination DNA damage repair
              </a>
            </td>
          <td>
            Jui-Chung Chiang, Zeng Shang, Tracy Rosales, Ling Cai, Wei-Min Chen, Feng Cai, Hieu S Vu, John D Minna, Min Ni, Anthony J. Davis, Robert D Timmerman, Ralph J DeBerardinis, Yuanyuan Zhang
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) extracted from peripheral blood has emerged as a crucial biomarker source in oncology research. To enhance the detection of somatic copy number alterations (SCNAs) and circulating tumor DNA (ctDNA), we developed eSENSES, a 2 Mb breast cancer-targeted NGS panel. It includes 15,000 genome-wide SNPs, 500 focal SNPs in breast cancer driver regions, and exons from 81 commonly altered genes, alongside a custom computational approach. We assessed the performance of eSENSES using both synthetic and clinical samples showing that eSENSES can detect ctDNA levels below 1%, exhibiting high sensitivity and specificity at 2-3% ctDNA levels. In patients with metastatic breast cancer, ctDNA estimations correlated with disease progression. When compared with other technologies and state-of-the-art approaches, eSENSES demonstrated enhanced performance. eSENSES provides a reliable, powerful and cost-effective tool for monitoring disease progression and guiding therapeutic decisions in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d0020163f84c6d4b9e21837bb90e991331c10b" target='_blank'>
              Enabling sensitive and precise detection of ctDNA through somatic copy number aberrations in breast cancer
              </a>
            </td>
          <td>
            Riccardo Scandino, A. Nardone, Nicola Casiraghi, F. Galardi, Mattia Genovese, Dario Romagnoli, Marta Paoli, C. Biagioni, Andrea Tonina, I. Migliaccio, M. Pestrin, E. Moretti, L. Malorni, L. Biganzoli, M. Benelli, Alessandro Romanel
          </td>
          <td>2025-03-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1afe205564d682ee54296311b108e6d8a61e8b1" target='_blank'>
              Spatial mapping of DNA synthesis reveals dynamics and geometry of human replication nanostructures
              </a>
            </td>
          <td>
            Michael Hawgood, Bruno Urién González, Ana Agostinho, Praghadhesh Thiagarajan, Yiqiu Yang, Xue Zhang, Giovanni Giglio, Gemma Quijada, Matilde Fonseca, J. Bartek, Hans Blom, B. Lemmens
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="YTHDC1, a key protein in the m6A-related regulatory network within cells, is involved in multiple cellular processes such as chromatin-related regulation, RNA splicing, and nuclear export. Understanding its role in colorectal cancer (CRC) development and DNA damage repair is critical for the advancement of treatment strategies. Our study found that YTHDC1 was highly expressed in high-malignancy CRC tissues compared with low-malignancy ones. Upon silencing YTHDC1, we observed a pronounced suppression of the proliferation of CRC cell lines, accompanied by a substantial increase in cell apoptosis. Furthermore, we identified RAD51 as a crucial downstream target of YTHDC1. Knocking down YTHDC1 led to a notable decrease in RAD51 protein levels, and silencing RAD51 also inhibited cancer cell proliferation. Interestingly, RNA-sequencing data indicated that the YTHDC1 deletion did not affect RAD51 transcription. However, Western blot revealed that this deletion increased the ubiquitination of RAD51, likely due to the upregulated E3 ligase UBE3A. Ubiquitination experiments subsequently confirmed that RAD51 is indeed one of the substrates of UBE3A. In summary, our study provides novel insights into how YTHDC1 modulates the expression of RAD51 through post-translational modifications. These findings offer valuable information that may potentially contribute to the development of more effective therapeutic strategies for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92925-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cf4d3cb02cc31b95af37c1defc2a6fa9b119aa" target='_blank'>
              YTHDC1 negatively regulates UBE3A to influence RAD51 ubiquitination and inhibit apoptosis in colorectal cancer cells
              </a>
            </td>
          <td>
            Mingwei Gao, Yueguang Wu, Li Zhang, Yan Zhou, Huijuan Liu, Weimin Zhang, Shubin Wang, Yongping Cui
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6227b696ba91084606d106cac80b2ab83ed05f1" target='_blank'>
              Extracellular alpha satellite DNA in human plasma as a novel molecular biomarker for early diagnosis of bladder cancer
              </a>
            </td>
          <td>
            I. Feliciello, D. Dermic
          </td>
          <td>2025-02-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The apicoplast organelle of Plasmodium falciparum is essential for parasite’s replication, however the details of its biogenesis, inheritance and regulation throughout the cell cycle are unknown. Here, we report the development of a dynamic imaging platform coupled with an analytical pipeline that enables us to follow and measure subcellular structures throughout the 48-hour cell cycle of live parasites. We reveal a predetermined sequence of four discrete morphological steps in organelle development, which are tightly correlated with nuclear replication. We show that one of these steps, which we term the Crown morphology, is required for nucleus-apicoplast attachment. During Crown, apicoplast is stretched over multiple nuclei, fastened by centriolar tubulin. A complementary molecular approach was used to discover the basic ploidy of apicoplast and mitochondrial genomes, their replication rates and association with nuclear DNA replication. We inhibited nuclear DNA replication and found that it completely blocks apicoplast biogenesis in its most initial stages, demonstrating dependency on S-phase initiation. Conversely, specific inhibition of apicoplast genome replication resulted in an almost-undisturbed organelle development and division. However, it affected the Crown step, preventing association to tubulin-nuclear structures, leading to failure in accurate organelle sorting into daughter cells. Collectively, these experiments reveal a central cellular pathway linking apicoplast development to the parasite’s cell cycle, and a second independent organellar mechanism responsible for segregation into daughter cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bfca923aa81dca1cabb89b41f451971dd56d89" target='_blank'>
              Organelle Development and Inheritance are Driven by Independent Nuclear and Organellar Mechanisms in Malaria Parasites
              </a>
            </td>
          <td>
            Shahar Michal, Qasem Alia, Freedman Eshkar, Maron Yariv, Tissawak Amanda, Florentin Anat
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, Pascale Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Recurrent/metastatic head and neck squamous cell carcinoma (R/M-HNSCC) is a severe, frequently lethal condition. Oncogene addiction to epidermal growth factor receptor (EGFR) is a hallmark of HNSCC, but the clinical efficacy of EGFR-targeted therapies remains low. Understanding molecular networks governing EGFR-driven progression is paramount to the exploration of (co)-treatment targets and predictive markers. Methods We performed function-based mapping of differentially expressed genes in EGFR-mediated local invasion (fDEGs) using photoconvertible tracers and RNA-sequencing (RNA-seq) in a cellular 3D-model. Results Upon alignment with public single-cell RNA-seq (scRNA-seq) datasets and HNSCC-specific regulons, a gene regulatory network of local invasion (invGRN) was inferred from gene expression data, which was overrepresented in budding tumors. InvGRN comprises the central hubs inhibin subunit beta alpha (INHBA) and snail family transcriptional repressor 2 (SNAI2), and druggable fDEGs integrin subunit beta 4 (ITGB4), laminin 5 (LAMB3/LAMC2), and sphingosine kinase 1 (SPHK1). Blockade of INHBA repressed local invasion and was reverted by activin A, laminin 5, and sphingosine-1-phosphate, demonstrating a functional interconnectivity of the invGRN. Epithelial-to-mesenchymal transition (EMT) of malignant cells and the invGRN are induced by newly defined EGFR-activity subtypes with prognostic value that are promoted by amphiregulin (AREG) and epiregulin (EREG). Importantly, co-inhibition of SPHK1 showed synthetic effects on Cetuximab-mediated invasion blockade and high expression of selected fDEGs was associated with response to Cetuximab in patient-derived xenotransplantation (PDX) and R/M-HNSCC patients. Conclusions We describe an actionable network of EGFR-mediated local invasion and define druggable effectors with predictive potential regarding the response of R/M-HNSCC to Cetuximab. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02290-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3a765a8bcbfd4bc6d82c05747a49dc01756024" target='_blank'>
              EGFR-mediated local invasiveness and response to Cetuximab in head and neck cancer
              </a>
            </td>
          <td>
            Jiefu Zhou, Min He, Qiong Zhao, Enxian Shi, Hairong Wang, Vaidehi Ponkshe, Jiahang Song, Zhengquan Wu, Dongmei Ji, Gisela Kranz, Anna Tscherne, S. Schwenk‑Zieger, Nilofer Abdul Razak, J. Hess, C. Belka, H. Zitzelsberger, I. Ourailidis, F. Stögbauer, M. Boxberg, J. Budczies, Christoph A. Reichel, M. Canis, Philipp Baumeister, Hongxia Wang, Kristian Unger, Andreas Mock, O. Gires
          </td>
          <td>2025-03-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2be5d83a44f2113bcc7ca92950bf19d6878d25c9" target='_blank'>
              Functional landscape of circular RNAs in human cancer cells
              </a>
            </td>
          <td>
            Peter Hyunwuk Her, Tiantian Li, Ziwei Huang, Xin Xu, Weining Yang, Moliang Chen, Mona Teng, Sujun Chen, Yong Zeng, Stanley Liu, Benjamin Haibe-Kains, Fraser Soares, Jie Ming, H. H. He
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, whose roles are under debate. We report that double strand breaks (DSB) induce de novo formation of chromatin loops, with the break positioned at the loop base. These loops form only in S/G2 phases and occur during repair via homologous recombination (HR), concomitant with DNA end resection and RAD51 assembly. RAD51 showed two-tiered accumulation around DSBs, with a broad (~Mb) domain arising from the homology search. This domain is regulated by cohesin unloader, is constrained by TAD boundaries, and it overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of NIPBL results in reduced HR, and this effect is more pronounced when the HR donor is far (~100 kb) from the break. Our data indicates that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions within the damaged TAD. One-Sentence Summary High spatiotemporal resolution analysis of double strand beak repair in 3D genome revealed the role of cohesin-driven loop extrusion in the homology search.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e040dd0709d7779dc334b56e7eb34dab3396724" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam Rybczynski, Namrata Nilavar, Daniel Nguyen, Violetta Karwacki-Neisius, Andrew G. Li, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R Chu, William Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, Iman Hajirasouliha, Marcin Imielinski, D. Nanus, Scott Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cff48bb1e4b4c69ed649cac63da394d5d1ac2d2" target='_blank'>
              Developmental regulation of Drosophila dosage compensation in a 3D genome context
              </a>
            </td>
          <td>
            Lubna Younas, Mujahid Ali, Xinpei Zhang, C. Regnard, Qi Zhou
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. The ribosomal DNA (rDNA) region is particularly vulnerable to such changes due to its repetitive nature. Here, we demonstrate that Rad27/FEN-1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining rDNA stability. The production of extrachromosomal rDNA circles and severe chromosomal rDNA instability are observed in the rad27Δ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates unprocessed Okazaki fragments in the rDNA region, without inducing DSB formation. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, we show that Exonuclease 1 and PCNA can compensate for the loss of Rad27 function in the rDNA stabilization. These findings highlight the importance of proper Okazaki fragment processing in preventing non-DSB-induced rDNA copy number changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35141ab23e752c4a99955ef6952f7857bccfbbe" target='_blank'>
              Rad27/FEN1 prevents accumulation of unprocessed Okazaki fragments and ribosomal DNA copy number changes
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a clinically challenging and molecularly heterogenous breast cancer subgroup. Here, we investigate the DNA methylation landscape of TNBC. By analyzing tumor methylome profiles and accounting for the genomic context of CpG methylation, we divide TNBC into two epigenetic subtypes corresponding to a Basal and a non-Basal group, in which characteristic transcriptional patterns are correlated with DNA methylation of distal regulatory elements and epigenetic regulation of key steroid response genes and developmental transcription factors. Further subdivision of the Basal and non-Basal subtypes identifies subgroups transcending genetic and proposed TNBC mRNA subtypes, demonstrating widely differing immunological microenvironments, putative epigenetically-mediated immune evasion strategies, and a specific metabolic gene network in older patients that may be epigenetically regulated. Our study attempts to target the epigenetic backbone of TNBC, an approach that may inform future studies regarding tumor origins and the role of the microenvironment in shaping the cancer epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704abc40683ff045fcf6f261937f8b6b3f4d7c59" target='_blank'>
              The DNA methylation landscape of primary triple-negative breast cancer
              </a>
            </td>
          <td>
            Mattias Aine, Deborah F. Nacer, E. Arbajian, Srinivas Veerla, Anna Karlsson, J. Häkkinen, Henrik J Johansson, Frida Rosengren, J. Vallon-Christersson, Åke Borg, J. Staaf
          </td>
          <td>2025-03-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Genomic instability and inflammation are distinct hallmarks of aging, but the connection between them is poorly understood. Here we report a mechanism directly linking genomic instability and inflammation in senescent cells through a mitochondria-regulated molecular circuit involving p53 and cytoplasmic chromatin fragments (CCF) that are enriched for DNA damage signaling marker γH2A.X. We show that p53 suppresses CCF accumulation and its downstream inflammatory phenotype. p53 activation suppresses CCF formation linked to enhanced DNA repair and genome integrity. Activation of p53 in aged mice by pharmacological inhibition of MDM2 reverses transcriptomic signatures of aging and age-associated accumulation of monocytes and macrophages in liver. Mitochondrial ablation in senescent cells suppresses CCF formation and activates p53 in an ATM-dependent manner, suggesting that mitochondria-dependent formation of γH2A.X + CCF dampens nuclear DNA damage signaling and p53 activity. These data provide evidence for a mitochondria-regulated p53 signaling circuit in senescent cells that controls DNA repair, genome integrity, and senescence- and age-associated inflammation, with relevance to therapeutic targeting of age-associated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4941baca9720335671879d8e4ca5f2fe5568326" target='_blank'>
              p53 enhances DNA repair and suppresses cytoplasmic chromatin fragments and inflammation in senescent cells
              </a>
            </td>
          <td>
            Karl N. Miller, Brightany Li, Hannah R. Pierce-Hoffman, Shreeya Patel, Xue Lei, Adarsh Rajesh, M. G. Teneche, Aaron Havas, Armin Gandhi, Carolina Cano Macip, Jun Lyu, Stella G. Victorelli, Seung-Hwa Woo, A. Lagnado, Michael A LaPorta, Tianhui Liu, Nirmalya Dasgupta, Sha Li, Andrew Davis, Anatoly Korotkov, Erik Hultenius, Zichen Gao, Yoav Altman, Rebecca A Porritt, Guillermina Garcia, C. Mogler, A. Seluanov, Vera Gorbunova, S. Kaech, Xiao Tian, Zhixun Dou, Chongyi Chen, João F. Passos, P. Adams
          </td>
          <td>2025-03-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background CTCF is considered as the most essential transcription factor regulating chromatin architecture and gene expression. However, genome-wide impact of CTCF on erythropoiesis has not been extensively investigated. Results Using a state-of-the-art human erythroid progenitor cell model (HUDEP-2 and HEL cell lines), we systematically investigate the effects of acute CTCF loss by an auxin-inducible degron system on transcriptional programs, chromatin accessibility, CTCF genome occupancy, and genome architecture. By integrating multi-omics datasets, we reveal that acute CTCF loss notably disrupts genome-wide chromatin accessibility and the transcription network. We detect over thousands of decreased chromatin accessibility regions but only a few hundred increased regions after CTCF depletion in HUDEP-2 and HEL lines, suggesting the role of CTCF in maintaining proper chromatin openness in the erythroid lineage. CTCF depletion in the erythroid context notably disrupts the boundary integrity of topologically associating domains and chromatin loops but does not affect nuclear compartmentalization. We find erythroid lineage-specific genes, including some metabolism-related genes, are suppressed at immature and mature stages. Notably, we find a subset of genes whose transcriptional levels increase upon CTCF depletion, accompanied by decreased chromatin accessibility regions enriched with the GATA motif. We further decipher the molecular mechanism underlying the CTCF/GATA2 repression axis through distal non-coding chromatin regions. These results suggest a suppressive role of CTCF in gene expression during erythroid lineage specification. Conclusions Our study reveals a novel role of CTCF in regulating erythroid differentiation by maintaining its proper chromatin openness and gene expression network, which extends our understanding of CTCF biology. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03510-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861d404b3fd5272bc30487378b974c5ae16b843d" target='_blank'>
              CTCF is selectively required for maintaining chromatin accessibility and gene expression in human erythropoiesis
              </a>
            </td>
          <td>
            Xue Yang, Li Cheng, Ye Xin, Jianxiang Zhang, Xinfeng Chen, Jinchao Xu, Mengli Zhang, Ruopeng Feng, Judith Hyle, Wenjie Qi, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li, Peng Xu
          </td>
          <td>2025-02-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The ability to replicate and pass information to descendants is a fundamental requirement for life. In addition to the DNA-based genetic information, modifications of the DNA or DNA-associated proteins can create patterns of heritable gene regulation. Such epigenetic inheritance allows for adaptation without mutation, but its limits and regulation are incompletely understood. Here we developed a C. elegans system to study the transgenerational epigenetic inheritance of H3K27me3, a conserved histone posttranslational modification associated with gene repression. We find that induced alterations of the genome-wide H3K27me3 landscape and the associated fertility defects persist for many generations in genetically wildtype descendants under selective pressure. We uncover that the inheritance of the altered H3K27me3 landscape is regulated by two chromodomain proteins with antagonizing functions, and provide mechanistic insight into how this molecular memory is initiated and maintained. Our results demonstrate that epigenetic inheritance can act as a mutation-independent, heritable mechanism of adaptation. In Brief Özdemir et al. demonstrate that an altered genomic distribution of the histone modification H3K27me3 can be epigenetically inherited across many generations through the activity of HERI-1/SET-32/MES-4, which is antagonized by CEC-6/PRC2 in C. elegans. Highlights Altered H3K27me3 landscapes can be inherited for at least 15 generations in C. elegans. The chromodomain proteins CEC-6 and HERI-1 have opposite roles in antagonizing or promoting the maintenance of the altered H3K27me3 landscape. H3K23me3 and H3K36me3 replace H3K27me3 to promote the intergenerational and transgenerational inheritance of the altered epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9477951c95e1934dfca6ddb054dc1db83139c5" target='_blank'>
              Transgenerational maintenance of H3K27me3 heterochromatin is balanced by chromodomain proteins in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Isa Özdemir, Anna Höfler, Kamila Delaney, Joanna M. Wenda, Chengyin Li, Arneet L. Saltzman, Andreas Boland, Florian A. Steiner
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1bc086a943a7e73ef25816e4af3f165ae6d1" target='_blank'>
              PARP inhibitors affect the replisome and replication fork progression during a single S phase
              </a>
            </td>
          <td>
            Lina-Marie Briu, C. Maric, Nicolas Valentin, Nicolas Panara, Guillaume Chevreux, Giuseppe Baldacci, Jean-Charles Cadoret
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome segregation is a fundamental process in all life forms and requires coordination with genome organization, replication and cell division. The mechanism that mediates chromosome segregation in archaea remains enigmatic, despite the centre-stage role assumed by these organisms in the discourse about the origin of eukaryotes. Previously, we identified two proteins, SegA and SegB, which form a minimalist chromosome partition machine in Sulfolobales. Here we uncover patterns and mechanisms that the SegAB system employs to link chromosome organization to genome segregation. Deletion of the genes causes growth and chromosome partition defects. SegB binds to multiple sites scattered across the chromosome, but mainly localised close to the segAB locus in most archaeal genera. The sites are predominantly intragenic and enriched in one of the two compartments into which the chromosome folds. We show that SegB coalesces into multiple foci through the nucleoid, exhibiting a biased localisation towards the cell periphery, which hints at potential tethers to the cell membrane. The short-range DNA compaction activity of SegB and long-range looping of distant sites point to a significant role for SegB in chromosome condensation that in turn enables genome segregation. Collectively, our data put forward SegAB as important players in bridging chromosome organization to genome segregation in archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea5d93484886454e63c62bbf7266af68724c725" target='_blank'>
              Coupling chromosome organization to genome segregation in Archaea
              </a>
            </td>
          <td>
            Azhar F. Kabli, Irene W. Ng, Nicholas Read, Parul Pal, Julia Reimann, N. Tran, Sonja-Verena Albers, Tung B. K. Le, D. Barillà
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Most cancers upregulate the telomere lengthening enzyme telomerase to achieve unlimited cell division. How chemotherapeutic nucleoside 6-thio-2’-deoxyguanosine (6-thio-dG) targets telomerase to inhibit telomere maintenance in cancer cells and tumors was unclear. Here, we demonstrate that telomerase insertion of 6-thio-dGTP prevents synthesis of additional telomeric repeats but does not disrupt telomerase binding to telomeres. Specifically, 6-thio-dG inhibits telomere extension after telomerase translocates along its product DNA to reposition the template, inducing a non-productive complex rather than enzyme dissociation. Furthermore, we provide direct evidence that 6-thio-dG treatment inhibits telomere synthesis by telomerase in cancer cells. In agreement, telomerase-expressing cancer cells harboring critically short telomeres are more sensitive to 6-thio-dG and show a greater induction of telomere losses compared to cancer cells with long telomere reserves. Our studies reveal that telomere length and telomerase status determine 6-thio-dG sensitivity and uncover the molecular mechanism by which 6-thio-dG selectively inhibits telomerase synthesis of telomeric DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67802c158f17e32f3595b80b0a3086893c47def" target='_blank'>
              Chemotherapeutic 6-thio-2’-deoxyguanosine selectively targets and inhibits telomerase by inducing a non-productive telomere-bound telomerase complex
              </a>
            </td>
          <td>
            Samantha L. Sanford, Mareike Badstübner, Michael Gerber, William Mannherz, Noah Lampl, Rachel L. Dannenberg, Angela M. Hinchie, M. Schaich, Sua Myong, Mark Hedglin, Suneet Agarwal, Jonathan Alder, Michael D. Stone, P. Opresko
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D. Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Background The cap-binding complex (CBC) plays a crucial role in facilitating gene expression by safeguarding mRNA from nonsense-mediated decay, promoting mRNA splicing, 3′-end processing, and facilitating nuclear export. Nevertheless, the precise biological functions and clinical implications of CBC in cancer remain ambiguous, necessitating further investigation for clarification. Methods The present study utilized the cBioPortal database to investigate the genetic alterations of nuclear cap binding protein subunit 2 (NCBP2) in pan-cancer. The Cancer Genome Atlas (TCGA) and online web tools were employed to analyze the correlation between NCBP2 and prognosis, genome instability, immune infiltration, immune response, cancer stemness, and chemotherapeutic efficacy in pan-cancer. Furthermore, the expression of NCBP2 was confirmed by immunohistochemistry (IHC) and functional analysis at the single-cell level was conducted using the CancerSEA database. Results NCBP2 exhibited distinct genetic alterations in pan-cancer with an increased expression in 24/32, while decreased expression in 3/32, types of cancers. IHC confirmed the aberrant expression of NCBP2 in lung squamous cell carcinoma (LUSC), pancreatic adenocarcinoma (PAAD), kidney renal papillary cell carcinoma (KIRP) and kidney renal clear cell carcinoma (KIRC). NCBP2 was correlated with overall survival (OS), disease-specific survival (DSS), and progression-free survival (PFS) in various cancers. Importantly, it was identified as a risk factor for OS, DSS and PFS in PAAD and uterine corpus endometrial carcinoma (UCEC). Gene Set Enrichment Analysis (GSEA) demonstrated that elevated NCBP2 was linked to immune and proliferation related pathways across multiple cancer types. Furthermore, a negative association between NCBP2 and stromal score, immune score, and ESTIMATE score was detected, and a positive correlation was observed between NCBP2 and diverse immune cells as well as stemness-indexes in the majority of cancer types. Drug sensitivity analysis revealed that drugs associated with NCBP2 primarily targeted DNA replication, chromatin histone methylation, ABL signaling, cell cycle, and PI3K signaling. Additionally, an examination at the single-cell level indicated that NCBP2 was positively correlated with cell cycle progression, DNA damage, DNA repair, invasion, and stemness in most cancer types, while negatively correlated with apoptosis, inflammation, and hypoxia in certain cancers. Conclusion In this study, we revealed the correlation of NCBP2 with prognosis, microenvironment and stemness, indicating that NCBP2 might be a potential therapeutic target for more effective and personalized therapy strategies in pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8319ec7e3e2dd161ac113462d4f85e344ef0e334" target='_blank'>
              NCBP2 predicts the prognosis and the immunotherapy response of cancers: a pan-cancer analysis
              </a>
            </td>
          <td>
            Shichao Li, Yulan Wang, Xi Yang, Miao Li, Guoxiang Li, Qiangqiang Song, Junyu Liu
          </td>
          <td>2025-03-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is complex because of the critical imbalance in genetic regulation as characterized by both the overexpression of oncogenes (OGs), mainly through mutations, amplifications, and translocations, and the inactivation of tumor-suppressor genes (TSGs), which entail the preservation of genomic integrity by inducing apoptosis to counter the malignant growth. Reviewing the intricate molecular interplay between OGs and TSGs draws attention to their cell cycle, apoptosis, and cancer metabolism regulation. In the present review, we discuss seminal discoveries, such as Knudson’s two-hit hypothesis, which framed the field’s understanding of cancer genetics, leading to the next breakthroughs with next-generation sequencing and epigenetic profiling, revealing novel insights into OG and TSG dysregulation with opportunities for targeted therapy. The key pathways, such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, are presented in the context of tumor progression. Importantly, we further highlighted the advances in therapeutic strategies, including inhibitors of KRAS and MYC and restoration of TSG function, despite which mechanisms of resistance and tumor heterogeneity pose daunting challenges. A high-level understanding of interactions between OG-TSGs forms the basis for effective, personalized cancer treatment—something to strive for in better clinical outcomes. This synthesis should integrate foundational biology with translation and, in this case, contribute to the ongoing effort against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Sarcomas are rare malignant tumors of mesenchymal origin with a high misdiagnosis rate due to their heterogeneity and low incidence. Conventional diagnostic techniques, such as Fluorescence In Situ Hybridization (FISH) and Next-Generation Sequencing (NGS), have limitations in detecting structural variations (SVs), copy number variations (CNVs), and predicting clinical behavior. Optical genome mapping (OGM) provides high-resolution genome-wide analysis, improving sarcoma diagnosis and prognosis assessment. This study analyzed 53 sarcoma samples using OGM. Ultra-high molecular weight (UHMW) DNA was extracted from core and resection biopsies, and data acquisition was performed with the Bionano Saphyr platform. Bioinformatic pipelines identified structural variations, comparing them with known alterations for each sarcoma subtype. OGM successfully analyzed 62.3% of samples. Diagnostic-defining alterations were found in 95.2% of cases, refining diagnoses and revealing novel oncogenic and tumor suppressor gene alterations. The challenges included DNA extraction and quality issues from some tissue samples. Despite these limitations, OGM proved to be a powerful diagnostic and predictive tool for bone and soft tissue sarcomas, surpassing conventional methods in resolution and scope, enhancing the understanding of sarcoma genetics, and enabling better patient stratification and personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7451322e1f10fdefd4135068d35cc6376d1125" target='_blank'>
              Exploring the Potential of Optical Genome Mapping in the Diagnosis and Prognosis of Soft Tissue and Bone Tumors
              </a>
            </td>
          <td>
            Alejandro Berenguer-Rubio, Esperanza Such, Neus Torres Hernández, P. González-Rojo, Álvaro Díaz-González, G. Avetisyan, Carolina Gil-Aparicio, Judith González-López, Nicolay Pantoja-Borja, L. Rubio-Martínez, Soraya Hernández-Girón, María Soledad Valera-Cuesta, Cristina Ramírez-Fuentes, María Simonet-Redondo, Roberto Díaz-Beveridge, C. de la Calva, J. Amaya-Valero, Cristina Ballester-Ibáñez, A. Liquori, Francisco Giner, E. Mayordomo-Aranda
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer that drives metastasis, immune evasion and treatment resistance. CIN may result from chromosome mis-segregation errors and excessive chromatin is frequently packaged in micronuclei (MN), which can be enumerated to quantify CIN. The assessment of CIN remains a predominantly manual and time-consuming task. Here, we present micronuclAI, a pipeline for automated and reliable quantification of MN of varying size and morphology in cells stained only for DNA. micronuclAI can achieve close to human-level performance on various human and murine cancer cell line datasets. The pipeline achieved a Pearson’s correlation of 0.9278 on images obtained at 10X magnification. We tested the approach in otherwise isogenic cell lines in which we genetically dialed up or down CIN rates, and on several publicly available image datasets where we achieved a Pearson’s correlation of 0.9620. Given the increasing interest in developing therapies for CIN-driven cancers, this method provides an important, scalable, and rapid approach to quantifying CIN on images that are routinely obtained for research purposes. We release a GUI-implementation for easy access and utilization of the pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad501d60562e0f7baf32d920b40de21a7a01759" target='_blank'>
              micronuclAI enables automated quantification of micronuclei for assessment of chromosomal instability
              </a>
            </td>
          <td>
            Miguel A. Ibarra-Arellano, Lindsay A. Caprio, Aroj Hada, Niklas Stotzem, Luke L Cai, Shivem B Shah, Zachary H. Walsh, Johannes C. Melms, Florian Wünneman, K. Bestak, Ibrahim Mansaray, Benjamin Izar, Denis Schapiro
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Research on chromosome organization and cell cycle progression in spherical bacteria, particularly Staphylococcus aureus, remains limited and fragmented. In this study, we established a working model to investigate chromosome dynamics in S. aureus using a Fluorescent Repressor-Operator System (FROS), which enabled precise localization of specific chromosomal loci. This approach revealed that the S. aureus cell cycle and chromosome replication cycle are not coupled, with cells exhibiting two segregated origins of replication at the start of the cell cycle. The chromosome has a specific origin-terminus-origin conformation, with origins localizing near the membrane, towards the tip of each hemisphere, or the “cell poles”. We further used this system to assess the role of various proteins with a role in S. aureus chromosome biology, focusing on the ParB-parS and SMC-ScpAB systems. Our results demonstrate that ParB binds five parS chromosomal sequences and the resulting complexes influence chromosome conformation, but play a minor role in chromosome compaction and segregation. In contrast, the SMC-ScpAB complex plays a key role in S. aureus chromosome biology, contributing to chromosome compaction, segregation and spatial organization. Additionally, we systematically assessed and compared the impact of proteins linking chromosome segregation to cell division—Noc, FtsK, SpoIIIE and XerC—on origin and terminus number and positioning. This work provides a comprehensive study of the factors governing chromosome dynamics and organization in S. aureus, contributing to our knowledge on chromosome biology of spherical bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbeb0ffed138b49203f6a896929817f3b337f1fa" target='_blank'>
              Chromosome segregation dynamics during the cell cycle of Staphylococcus aureus
              </a>
            </td>
          <td>
            Adrian Izquierdo-Martinez, Simon Schäper, António D. Brito, Qinchan Liao, Coralie Tesseur, Moritz Sorg, Daniela S. Botinas, Xindan Wang, M. G. Pinho
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The Mitotic Surveillance Pathway (MSP) monitors the duration of M-phase. Prolonged mitosis, caused by spindle attachment defects or microtubule-targeting drugs such as the taxane paclitaxel, induces the formation of the ternary “mitotic stopwatch” complex consisting of 53BP1, USP28, and p53. This event protects p53 from degradation, resulting in cell cycle arrest or apoptosis in daughter cells. In paclitaxel-resistant cancers, cells bypass the MSP, enabling unchecked proliferation and survival, although the underlying mechanisms remain unknown. Here, we demonstrate that 53BP1 physically interacts with GMCL1 but not its paralog, GMCL2, and we mapped the interaction regions on both proteins. CRL3GMCL1 functions as a ubiquitin ligase that targets 53BP1 for degradation during M phase, impacting p53 levels in daughter cells. High GMCL1 expression significantly correlates with resistance to paclitaxel in cancer cell lines with wild-type p53, including endometrial, breast, and upper aerodigestive tract cancer cells. Loss of GMCL1 restores paclitaxel sensitivity in p53 expressing cells but not in p53 deficient cells. We propose that in cancers with high GMCL1 levels, the CRL3GMCL1-mediated degradation of 53BP1 prevents the formation of the mitotic stopwatch complex, leading to p53 degradation and sustained proliferation. Finally, our results indicate that GMCL1 inhibition represents a novel strategy to restore taxane sensitivity in resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/709ee165cbc090bfd04b08f1a64c12e9f13af2e9" target='_blank'>
              GMCL1 Controls 53BP1 Stability and Modulates Paclitaxel Sensitivity in Cancer
              </a>
            </td>
          <td>
            Yuki Kito, Tania J. González-Robles, Sharon Kaisari, Juhee Pae, Sheena Faye Garcia, Juliana Ortiz-Pacheco, Beatrix Ueberheide, Ruth Lehmann, Antonio Marzio, G. Róna, Michele Pagano
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, Lodewyk F. A. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Despite the striking conservation of animal chromosomes, their repetitive element complements are vastly diverse. Only recently, high quality chromosome-level genome assemblies enabled identification of repeat compositions along a broad range of animal chromosomes. Here, utilizing the chromosome-level genome assembly of Hydractinia symbiolongicarpus, a colonial hydrozoan cnidarian, we describe an accumulation of a single 372 bp repeat unit in the subtelomeric regions. Based on the sequence divergence, its partial affinity with the Helitron group can be detected. This sequence is associated with a repeated minisatellite unit of about 150 bp. Together, they account for 26.1% of the genome (126 Mb of the 483 Mb). This could explain the genome size increase observed in H. symbiolongicarpus compared with other cnidarians, yet distinguishes this expansion from other large cnidarian genomes, such as Hydra vulgaris, where such localized propagation is absent. Additionally, we identify a derivative of an IS3EU-like DNA element accumulated at the putative centromeric regions. Our analysis further reveals that Helitrons generally comprise a large proportion of H. symbiolongicarpus (11.8%). We investigated Helitron presence and distributions across several cnidarian genomes. We find that in Nematostella vectensis, an anthozoan cnidarian, Helitron-like sequences were similarly accumulated at the subtelomeric regions. All these findings suggest that Helitron derivatives are prone to forming chromosomal extensions in cnidarians through local amplification in subtelomeric regions, driving variable genome expansions within the clade. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00355-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fb42c253af0c01d931cd52e6e5278687b51b25" target='_blank'>
              Subtelomeric repeat expansion in Hydractinia symbiolongicarpus chromosomes
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Oleg Simakov
          </td>
          <td>2025-03-25</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The burden of triple-negative breast cancer (TNBC) may be shaped by genetic factors, particularly inherited and somatic mutation profiles. However, data on this topic remain limited, especially for the African continent, where a higher TNBC incidence is observed. In the age of precision medicine, cataloguing TNBC diversity in African patients becomes imperative. We performed whole exome sequencing, including untranslated regions, on 30 samples from Angola and Cape Verde, which allowed to ascertain on potential regulatory mutations in TNBC for the first time. A high somatic burden was observed for the African cohort, with 86% of variants being so far unreported. Recurring to predictive functional algorithms, 17% of the somatic single nucleotide variants were predicted to be deleterious at the protein level, and 20% overlapped with candidate cis-regulatory elements controlling gene expression. Several of these somatic functionally-impactful mutations and copy number variation (mainly in 1q, 8q, 6 and 10p) occur in known BC- and all cancer-driver genes, enriched for several cancer mechanisms, including response to radiation and related DNA repair mechanisms. TP53 is the top of these known BC-driver genes, but our results identified possible novel TNBC driver genes that may play a main role in the African context, as TTN, CEACAM7, DEFB132, COPZ2 and GAS1. These findings emphasize the need to expand cancer omics screenings across the African continent, the region of the globe with highest genomic diversity, accelerating the discovery of new somatic mutations and cancer-related pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67fac74a3ac724fb478678a7a632cf9d7052cf76" target='_blank'>
              Coding and regulatory somatic profiling of triple-negative breast cancer in Sub-Saharan African patients
              </a>
            </td>
          <td>
            Ricardo J Pinto, Dylan Ferreira, Paulo Salamanca, Fernando Miguel, Pamela Borges, Carla Barbosa, Vitor Costa, Carlos Lopes, Lúcio Lara Santos, Luisa Pereira
          </td>
          <td>2025-03-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In maize, there are two meiotic drive systems that operate on large tandem repeat arrays called knobs that are found on chromosome arms. One meiotic drive haplotype, Abnormal chromosome 10 (Ab10), encodes two kinesin proteins that interact with two distinct tandem repeat arrays in a sequence-specific manner to confer meiotic drive. The kinesin KINDR associates with knob180 repeats while the kinesin TRKIN associates with TR-1 repeats. Prior data show that meiotic drive is conferred primarily by the KINDR/knob180 system, with the TRKIN/TR-1 system having little or no role. The second meiotic drive haplotype, K10L2, shows low levels of meiotic drive and only encodes the TRKIN/TR-1 system. Here we used long-read sequencing to assemble the K10L2 haplotype and showed that it has strong homology to an internal portion of the Ab10 haplotype. We also carried out CRISPR mutagenesis of Trkin to test the role of Trkin on Ab10 and K10L2. The data indicate that the Trkin gene on Ab10 does not improve drive or fitness but instead has a weak deleterious effect when paired with a normal chromosome 10. The deleterious effect is more severe when Ab10 is paired with K10L2: in this context functional Trkin on either chromosome nearly abolishes Ab10 drive. We modeled the effect of Trkin on Ab10 and found it should not persist in the population. We conclude that Trkin either confers an advantage to Ab10 in untested circumstances or that it is in the process of being purged from the Ab10 population. ARTICLE SUMMARY Mendel’s first law states that paired chromosomes are transmitted through meiosis at equal frequencies. Some chromosome variants, however, are transmitted at higher frequencies in a process called meiotic drive. We wanted to know the function of a motor protein called TRKIN that is encoded on a maize meiotic drive chromosome. Surprisingly, we found that TRKIN provides no advantage to the meiotic driver and instead seems to be deleterious, suggesting it had a function in a wild ancestor but is now being purged from the population. The results illustrate how genomes are shaped by often-conflicting forces of selection and selfish genetic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70166f6dd8807cf7e9553c926b2c5721c9de8336" target='_blank'>
              Antagonistic kinesin-14s within a single chromosomal drive haplotype
              </a>
            </td>
          <td>
            Meghan J. Brady, Anjali Gupta, J. Gent, Kyle W. Swentowsky, Robert L. Unckless, R. Dawe
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Genes that escape X-chromosome inactivation (XCI) in female somatic cells vary in number and levels of escape among mammalian species and tissues, potentially contributing to species- and tissue-specific sex differences. CTCF, a master chromatin conformation regulator, is enriched at escape regions and may play an important role in regulating escape, but the molecular mechanisms remain elusive. Results CTCF binding profiles and epigenetic features were systematically examined at escape genes (escapees) using mouse allelic systems with skewed XCI to distinguish the inactive X (Xi) and active X (Xa) chromosomes. We found that six constitutive and two facultative escapees are located inside 30-800 kb domains marked by convergent arrays of CTCF binding sites, consistent with the formation of chromatin loops. Facultative escapees show clear differences in CTCF binding depending on their XCI status in specific cell types/tissues. In addition, sets of strong and in some cases divergent CTCF binding sites located at the boundary between an escapee and its adjacent neighbors subject to XCI would also help insulate domains. Indeed, deletion but not inversion of a CTCF binding site at the boundary between the facultative escapee Car5b and its silent neighbor Siah1b results in a dramatic reduction of Car5b escape. This is associated with reduced CTCF and cohesin binding, which indicates loss of looping and insulation and is supported by 3C combined with Hi-C analysis. In addition, enrichment in the repressive mark H3K27me3 invades the Car5b domain in deleted cells, consistent with loss of expression from the Xi. In contrast, cells with an inversion of the CTCF binding site retain CTCF and cohesin binding, as well as looping, in line with persistence of escape. Interestingly, the levels of escape increase in cells with deletion of either Dxz4, which disrupts the Xi-specific compact 3D structure, or Firre, which results in lower H3K27me3 enrichment on the Xi, indicating that the structural and epigenetic features of the Xi constrain escape from XCI in wild type conditions. Conclusions Taken together, our findings support the idea that escape from XCI in female somatic cells is modulated by both the topological insulation of domains via CTCF binding and the surrounding heterochromatin environment. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02137-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fde8cc47c2c193748a3f06ce03baac9ac12fe7" target='_blank'>
              CTCF-mediated insulation and chromatin environment modulate Car5b escape from X inactivation
              </a>
            </td>
          <td>
            H. Fang, A. Tronco, G. Bonora, Truong Nguyen, Jitendra Thakur, J. B. Berletch, Galina N. Filippova, S. Henikoff, Jay Shendure, W. S. Noble, Zhijun Duan, Christine M. Disteche, Xinxian Deng
          </td>
          <td>2025-03-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Monoallelic deletion of the short arm of chromosome 17 (del17p) is a recurrent abnormality in cancers with poor outcomes. Best studied in relation to haematological malignancies, associated functional outcomes are attributed mainly to loss and/or dysfunction of TP53, which is located at 17p13.1, but the wider impact of deletion of other genes located on 17p is poorly understood. 17p is one of the most gene-dense regions of the genome and includes tumour suppressor genes additional to TP53, genes essential for cell survival and proliferation, as well as small and long non-coding RNAs. In this review we utilise a data-driven approach to demarcate the extent of 17p deletion in multiple cancers and identify a common loss-of-function gene signature. We discuss how the resultant loss of heterozygosity (LOH) and haploinsufficiency may influence cell behaviour but also identify vulnerabilities that can potentially be exploited therapeutically. Finally, we highlight how emerging animal and isogenic cell line models of del17p can provide critical biological insights for cancer cell behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Background Chromosomal instability (CIN) is involved in about 70% of colorectal cancers (CRCs) and is associated with poor prognosis and drug resistance. From a clinical perspective, a better knowledge of these tumour’s biology will help to guide therapeutic strategies more effectively. Methods We used high-density chromosomal microarray analysis to evaluate CIN level of patient-derived organoids (PDOs) and their original mCRC tissues. We integrated the RNA-seq and mass spectrometry-based proteomics data from PDOs in a functional interaction network to identify the significantly dysregulated processes in CIN. This was followed by a proteome-wGII Pearson correlation analysis and an in silico validation of main findings using functional genomic databases and patient-tissues datasets to prioritize the high-confidence CIN features. Results By applying the weighted Genome Instability Index (wGII) to identify CIN, we classified PDOs and demonstrated a good correlation with tissues. Multi-omics analysis showed that our organoids recapitulated genomic, transcriptomic and proteomic CIN features of independent tissues cohorts. Thanks to proteotranscriptomics, we uncovered significant associations between mitochondrial metabolism and epithelial-mesenchymal transition in CIN CRC PDOs. Correlating PDOs wGII with protein abundance, we identified a subset of proteins significantly correlated with CIN. Co-localisation analysis in PDOs strengthened the putative role of IPO7 and YAP, and, through in silico analysis, we found that some of the targets give significant dependencies in cell lines with CIN compatible status. Conclusions We first demonstrated that PDO models are a faithful reflection of CIN tissues at the genetic and phenotypic level. Our new findings prioritize a subset of genes and molecular processes putatively required to cope with the burden on cellular fitness imposed by CIN and associated with disease aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03308-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Accurate sister chromatid segregation requires remodeling chromosome architecture, decatenation, and attachment to the mitotic spindle. Some of these events are initiated during S-phase, but they accelerate and conclude during mitosis. Here we describe SRBD1 as a histone and nucleic acid binding protein that prevents DNA damage in interphase cells, localizes to nascent DNA during replication and the chromosome scaffold in mitosis, and is required for chromosome segregation. SRBD1 inactivation causes micronuclei, chromatin bridges, and cell death. Inactivating SRBD1 immediately prior to mitotic entry causes anaphase failure, with a reduction in topoisomerase IIα localization to mitotic chromosomes and defects in properly condensing and decatenating chromosomes. In contrast, SRBD1 is not required to complete cell division after chromosomes are condensed. Strikingly, depleting condensin II reduces the severity of the anaphase defects in SRBD1-deficient cells by restoring topoisomerase IIα localization. Thus, SRBD1 is an essential genome maintenance protein required for mitotic chromosome organization and segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966c52bd12100322ff3d9cac1b57f3d49dccb625" target='_blank'>
              SRBD1 facilitates chromosome segregation by promoting topoisomerase IIα localization to mitotic chromosomes
              </a>
            </td>
          <td>
            Courtney A. Lovejoy, Sarah R. Wessel, Rahul Bhowmick, Yuki Hatoyama, Masato T. Kanemaki, Runxiang Zhao, David Cortez
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The structural integrity of the chromosomes is essential to every functional process within eukaryotic nuclei. Chromosomes are DNA-histone complexes that are essential for the inheritance of genetic information to the offspring, and any defect in them is linked to mitotic errors, cancer growth, and cellular aging. Changes in the mechanical properties of a chromosome could lead to its compromised function and stability, leading to chromosome breaks. Here, we studied the changes in chromosome physical properties using metaphase chromosomes isolated from moderately malignant (MCF7) and highly malignant (MDA-MB-231) human breast cancer cells exposed to valproic acid (VPA), a known epigenetic modifier drug involved in histone hyperacetylation and DNA demethylation. Due to chromosomal structural intricacy and preparative and technical limitations of analytical tools, we employed a label-free atomic force microscopy approach for simultaneously visualizing and mapping single chromosome elasticity and stretching modulus. Additionally, we performed electron transport characteristics through metaphase chromosomes to elucidate the effect of VPA. The chromosomal elasticity and electron transport alterations are manifestations of VPA-mediated chromatin’s epigenetic changes. Our multiparametric strategy, as shown by receiver operating characteristics analyses with the physical properties of chromosomes, offers a new scope in terms of analytical tools for studying chromosomal structural changes/aberrations linked to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4081e3a40a851c1d9b5593b224de0890423bfe7" target='_blank'>
              Epigenetic Modifier Drug Valproic Acid Enhances Cancer Metaphase Chromosome Elasticity and Electron Transport: An Atomic Force Microscopy Approach
              </a>
            </td>
          <td>
            Tanya Agrawal, Debashish Paul, Amita Mishra, Ganesan Arunkumar, T. Rakshit
          </td>
          <td>2025-02-12</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This study explores the multifaceted role of p21 in mediating cellular responses to DNA-damaging agents, with a focus on doxorubicin treatment in HCT116 colon carcinoma cells. We investigated how different doses of doxorubicin affect cells with varied p21 and p53 statuses, revealing distinct roles for p21 depending on the drug dosage. At high doses (HD), p21 is more critical than p53 in mediating apoptosis, whereas at low doses (LD), p21 is essential for preventing mitotic defects and multinucleation. Notably, reintroducing p21 or pharmacologically inhibiting CDK1 reduced multinucleation. The absence of p21 upon LD doxorubicin exposure led to aberrant chromosome segregation, persistent DNA damage response (DDR) activation, and increased non-homologous end-joining (NHEJ) activity, resulting in unrepaired DNA accumulation and multinucleation. Additionally, mitotic defects in p21-deficient cells were associated with mislocalization of key mitotic regulators, Aurora B and mitotic kinesin-like protein 1 (MKLP1), exacerbating defective mitosis. In summary, p21 functions as a dual regulator in response to DNA damage, promoting apoptosis at HD and preventing mitotic failure at LD. These insights have significant implications for cancer therapy, highlighting the potential of targeting the p21 to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62798e3e091c52a2538fbed6da27e2c926098463" target='_blank'>
              Dual role of p21 in regulating apoptosis and mitotic integrity in response to doxorubicin in colon cancer cells
              </a>
            </td>
          <td>
            , Haein Kim, Eunjung Jang, Young-Woo Eom, Gyesoon Yoon, Kyeong Sook Choi, Eunhee Kim
          </td>
          <td>2025-04-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is a major cause of cancer-related deaths globally. Targeted therapies, specifically attacking cancer cells based on genetic mutations, offer promising alternatives. ALK (anaplastic lymphoma kinase) fusions result in aberrant proteins that drive cancer growth. Drugs like crizotinib and ceritinib have shown efficacy in treating ALK-positive NSCLC. Accurate detection of ALK fusions is crucial for guiding these therapies. We conducted a retrospective analysis of a Chinese cohort of 131 ALK rearrangement-positive patients detected by DNA NGS between January 2017 and December 2021. Among those 131 ALK fusions, RNA-NGS confirmed positive transcripts in 88% of canonical ALK fusions and 75% of ALK fusions with rare partners in samples sequenced by both DNA NGS and RNA NGS. The secondary classification approach increased transcript prediction accuracy to 95.4% when combining common breakpoints and inframe fusion analysis in canonical ALK fusions. Combining rare breakpoints and inframe fusion could increase transcript prediction accuracy to 100%. For ALK fusions with rare partners, combining common breakpoints and frameshift improved transcript prediction accuracy to 100%. Additionally, combining rare breakpoints with inframe or frameshift could enhance the prediction accuracy to 100%. Combining DNA NGS and RNA NGS with a secondary classification approach significantly enhances the transcript prediction accuracy at the RNA level. This method optimizes clinical diagnostic and therapeutic strategies for ALK-positive NSCLC, highlighting the importance of advanced sequencing techniques in precision oncology. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92590-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34804a6772ac12af08ec23d6c1744145feb9ff1c" target='_blank'>
              Combined utility of genomic breakpoints and frame is a reliable predictor of ALK transcript function
              </a>
            </td>
          <td>
            Qiong Yang, Yumin Guo, Weihui Guo, Dongdong Liu, Haitao Wang, Xufan Cai
          </td>
          <td>2025-03-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers that arise from germline mutations of breast cancer associated gene 1 (BRCA1), which is a crucial player in homologous recombination (HR) DNA repair, are vulnerable to DNA-damaging agents such as platinum and PARP inhibitors (PARPis). Increasing evidence suggests that BRCA1 is an essential driver of all phases of the cell cycle, thereby maintaining orderly steps during cell cycle progression. Specifically, loss of BRCA1 activity causes the S-phase, G2/M, spindle checkpoints, and centrosome duplication to be dysregulated, thereby blocking cell proliferation and inducing apoptosis. In vertebrates, loss of HR genes such as BRCA1 and/or BRCA2 is lethal, since HR is a prerequisite for genome integrity. Thus, cancer cells utilize alternative DNA repair pathways such as non-homologous end joining (NHEJ) to cope with the loss of BRCA1 function. In this review, we attempt to update and discuss how these novel components are crucial for regulating DNA damage repair (DDR) in BRCA1-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca96851a56316ea96ea1fc73475ff478d721c03" target='_blank'>
              Overview of Roles of Novel Components in the Regulation of DNA Damage Repair in BRCA1-Deficient Cancers: An Update
              </a>
            </td>
          <td>
            Nhat Nguyen, Dominic Arris, M. T. Tran
          </td>
          <td>2025-04-01</td>
          <td>DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction The HIV regulatory protein Tat enhances viral transcription and also modifies host gene expression, affecting cell functions like cell cycle and apoptosis. Residual expression of Tat protein is detected in blood and other tissues even under antiretroviral treatment. Cohort studies have indicated that, despite virologic suppression, people with HIV (PWH) are at increased risk of comorbidities linked to chronic inflammation, accelerated immune ageing, and cellular senescence, sometimes associated with abnormal genomic methylation patterns. We analysed whether Tat influences DNA methylation and subsequently impacts the transcriptional signature, contributing to inflammation and accelerated ageing. Methods We transfected Jurkat cells with full-length Tat (Tat101), Tat’s first exon (Tat72), or an empty vector (TetOFF). We assessed DNA methylation modifications via the Infinium MethylationEPIC array, and we evaluated transcriptomic alterations through RNA-Seq. Methylation levels in gene promoters or body regions were correlated to their expression data, and subsequently, we performed an overrepresentation analysis to identify the biological terms containing differentially methylated and expressed genes. Results Tat101 expression caused significant hyper- and hypomethylation changes at individual CpG sites, resulting in slightly global DNA hypermethylation. Methylation changes at gene promoters and bodies resulted in altered gene expression, specifically regulating gene transcription in 5.1% of differentially expressed genes (DEGs) in Tat101- expressing cells. In contrast, Tat72 had a minimal impact on this epigenetic process. The observed differentially methylated and expressed genes were involved in inflammatory responses, lipid antigen presentation, and apoptosis. Discussion Tat expression in HIV infection may constitute a key epigenetic modelling actor that contributes to HIV pathogenesis and chronic inflammation. Clinical interventions targeting Tat blockade may reduce chronic inflammation and cellular senescence related to HIV infection comorbidities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5e958923998010006c6ab274e56aa113315573" target='_blank'>
              Intracellular HIV-1 Tat regulator induces epigenetic changes in the DNA methylation landscape
              </a>
            </td>
          <td>
            Andrea Rodríguez-Agustín, Rubén Ayala-Suárez, F. Díez-Fuertes, M. Maleno, Izar de Villasante, Angelika Merkel, M. Coiras, Víctor Casanova, José Alcamí, Núria Climent
          </td>
          <td>2025-03-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5522614d16253d3331d81500710e925011629fd8" target='_blank'>
              Polo-like kinase 1 maintains transcription and chromosomal accessibility during mitosis
              </a>
            </td>
          <td>
            Zhouyuan Shen, Kristin M. Adams, Ryan Moreno, Robert F. Lera, Emily Kaufman, Jessica D. Lang, Mark Burkard
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Genomes in eukaryotes normally undergo DNA replication in a choreographed temporal order, resulting in early and late replicating chromosome compartments. Leishmania, a human protozoan parasite, displays an unconventional DNA replication program in which the timing of DNA replication completion is chromosome size-dependent: larger chromosomes complete replication later then smaller ones. Here we show that both R-loops and RNase H1, a ribonuclease that resolves RNA-DNA hybrids, accumulate in Leishmania major chromosomes in a pattern that reflects their replication timing. Furthermore, we demonstrate that such differential organisation of R-loops, RNase H1 and DNA replication timing across the parasite’s chromosomes correlates with size-dependent differences in chromatin accessibility, G quadruplex distribution and sequence content. Using conditional gene excision, we show that loss of RNase H1 leads to transient growth perturbation and permanently abrogates the differences in DNA replication timing across chromosomes, as well as altering levels of aneuploidy and increasing chromosome instability in a size-dependent manner. This work provides a link between R-loop homeostasis and DNA replication timing in a eukaryotic parasite and demonstrates that orchestration of DNA replication dictates levels of genome plasticity in Leishmania.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f444e75e46ba8c4938d4f849b2096f3e296708" target='_blank'>
              Alectinib enhances response to RBM39 degradation via SRPK1 inhibition
              </a>
            </td>
          <td>
            Yurui Ma, E. Poon, Chenchen Jin, Barbara Martins Da Costa, Yuewei Xu, Sadiya Quazi, Nikolaos Zourdoumis, Chiharu Wickremesinghe, L. Chesler, Hector C. Keun, Anke Nijhuis
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1324a05392de89320b697d282b9ec5e482ee4e56" target='_blank'>
              γ-tubulin mediates DNA double-strand break repair.
              </a>
            </td>
          <td>
            Abhishikt David Solomon, Odjo G. Gouttia, Ling Wang, Songli Zhu, Feifei Wang, Yanqui Li, Mohammadjavad Paydar, T. Bessho, Benjamin H Kwok, A. Peng
          </td>
          <td>2025-03-15</td>
          <td>Journal of cell science</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c305dc46bcd16adb030553a33a9a8b7556cef2ef" target='_blank'>
              Trypanosoma cruzi Has 32 Chromosomes: A Telomere-to-Telomere Assembly Defines Its Karyotype
              </a>
            </td>
          <td>
            G. Greif, F. Díaz-Viraqué, M. L. Chiribao, Carlos E. Sanz-Rodriguez, C. Robello
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background The tumor suppressor p53 (Trp53), also known as p53, is the most commonly mutated gene in cancer. Canonical p53 DNA damage response pathways are well characterized and classically thought to underlie the tumor suppressive effect of p53. Challenging this dogma, mouse models have revealed that p53-driven apoptosis and cell cycle arrest are dispensable for tumor suppression. Here, we investigated the inverse context of a p53 mutation predicted to drive the expression of canonical targets but is detected in human cancer. Methods We established a novel mouse model with a single base pair mutation (GAG>GAT, p53E221D) in the DNA-Binding domain that has wild-type function in screening assays, but is paradoxically found in human cancer in Li-Fraumeni syndrome. Using mouse p53E221D and the analogous human p53E224D mutants, we evaluated expression, transcriptional activation, and tumor suppression in vitro and in vivo. Results Expression of human p53E224D from cDNA translated to a fully functional p53 protein. However, p53E221D/E221D RNA transcribed from the endogenous locus is mis-spliced resulting in nonsense-mediated decay. Moreover, fibroblasts derived from p53E221D/E221D mice do not express a detectable protein product. Mice homozygous for p53E221D exhibited increased tumor penetrance and decreased life expectancy compared to p53WT/WT animals. Conclusions Mouse p53E221D and human p53E224D mutations lead to splice variation and a biologically relevant p53 loss of function in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a5ab888592bac1b40fc9b15ac14955958bd526" target='_blank'>
              Mis-splicing drives loss of function of p53E224D point mutation
              </a>
            </td>
          <td>
            Ian C. Lock, Nathan H Leisenring, Warren Floyd, Eric S. Xu, Lixia Luo, Yan Ma, Erin C. Mansell, Diana M. Cardona, Chang-Lung Lee, D. Kirsch
          </td>
          <td>2025-03-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fee2b13a33b992ead84ceb784d179cc70d6fa2a" target='_blank'>
              Nuclear Galectin-1 promotes KRAS-dependent activation of pancreatic cancer stellate cells.
              </a>
            </td>
          <td>
            J. Vinaixa, Neus Martínez-Bosch, Joan Gibert, Noemí Manero-Rupérez, P. Santofimia-Castaño, Federico G Baudou, Renzo E Vera, David R Pease, Mar Iglesias, Sandhya Sen, Xiyin Wang, Luciana L Almada, David L Marks, Mireia Moreno, J. Iovanna, Gabriel A. Rabinovich, Martin E Fernandez-Zapico, Pilar Navarro
          </td>
          <td>2025-04-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/230adcbb775deb02eb0452f0dd751e2a887a0270" target='_blank'>
              KRAB zinc-finger proteins regulate endogenous retroviruses to sculpt germline transcriptomes and genome evolution.
              </a>
            </td>
          <td>
            Kai Otsuka, A. Sakashita, So Maezawa, Richard M Schultz, S. Namekawa
          </td>
          <td>2025-03-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbddfb38eb3b46fe2a2daba20fe0e51c20e3f4a" target='_blank'>
              βTrCP is involved in the localization of the MRN complex on chromatin to enhance DNA damage repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea8da16c37659abb745c723fb795dcd02f7d43b2" target='_blank'>
              Quantitative AI-based DNA fiber workflow to study replication stress
              </a>
            </td>
          <td>
            Paolo Fagherazzi, Timo Diekmann, Alessandra Ardizzoia, Zuzana Machacova, Simran Negi, Anoop Kumar Yadav, Pavel Moudry, Vincenzo Costanzo, Hana Polasek-Sedlackova
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a highly heterogeneous malignancy, often characterized by complex genomic alterations that drive tumor progression and therapy resistance. In this paper, we report a novel de novo BRCA2 germline variant NM_000059.3:c.(8693_8695delinsGT) associated with early-onset OC that featured two regions with differential MMR (Mismatch Repair) gene expression. To date, only six cases of de novo BRCA2 variants have been reported, none of which were associated with early-onset high-grade serous OC. The immunohistochemical analysis of MMR genes revealed two distinct tumor areas, separated by a clear topographic boundary, with the heterogeneous expression of MLH1 and PMS2 proteins. Seventy-five percent of the tumor tissue showed positivity, while the remaining 25% exhibited a complete absence of expression, underscoring the spatial variability in MMR gene expression within the tumor. Integrated comparative spatial genomic profiling identified several tumor features associated with the genetic variant as regions of loss of heterozygosity (LOH) that involved BRCA2 and MLH1 genes, along with a significantly higher mutational tumor burden in the tumor area that lacked MLH1 and PMS2 expression, indicating its further molecular evolution. The following variants were acquired: c.6572C>T in NOTCH2, c.1852C>T in BCL6, c.191A>T in INHBA, c.749C>T in CUX1, c.898C>A in FANCG, and c.1712G>C in KDM6A. Integrated comparative spatial proteomic profiles revealed defects in the DNA repair pathways, as well as significant alterations in the extracellular matrix (ECM). The differential expression of proteins involved in DNA repair, particularly those associated with MMR and Base Excision Repair (BER), highlights the critical role of defective repair mechanisms in driving genomic instability. Furthermore, ECM components, such as collagen isoforms, Fibrillin-1, EMILIN-1, Prolargin, and Lumican, were found to be highly expressed in the MLH1/PMS2-deficient tumor area, suggesting a connection between DNA repair deficiencies, ECM remodeling, and tumor progression. Thus, the identification of the BRCA2 variant sheds light on the poorly understood interplay between DNA repair deficiencies and ECM remodeling in OC, providing new insights into their dual role in shaping tumor evolution and suggesting potential targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3d22f654834c47296af7f38cdf9042ad40753" target='_blank'>
              Novel De Novo BRCA2 Variant in an Early-Onset Ovarian Cancer Reveals a Unique Tumor Evolution Pathway
              </a>
            </td>
          <td>
            G. Miolo, Giovanni Canil, Maurizio Polano, M. Dal Bo, Alessia Mondello, Antonio Palumbo, F. Puglisi, Giuseppe Corona
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed8e001dc646667003aedc35613a33f0517e9a4" target='_blank'>
              Common variation in meiosis genes shapes human recombination phenotypes and aneuploidy risk
              </a>
            </td>
          <td>
            S. A. Carioscia, A. Biddanda, M. R. Starostik, X. Tang, E. R. Hoffmann, Z. P. Demko, R. C. McCoy
          </td>
          <td>2025-04-04</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8904feaf495af004b999f1283404430b8205d7d8" target='_blank'>
              p53-deficient cancer cells hyperactivate DNA double-strand break repair pathways to overcome chemotherapeutic damage and augment survival.
              </a>
            </td>
          <td>
            Rebecca Dsouza, Meghna Jain, Ekta Khattar
          </td>
          <td>2025-03-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background AZIN1 is a cell cycle regulator that is upregulated in a variety of cancers. AZIN1 overexpression can induce a more aggressive tumor phenotype via increased binding and resultant inhibition of antizyme. Antizyme is a protein that normally functions as an anti-tumor regulator that facilitates the deactivation of several growth-promoting proteins including c-Myc. MYC plays a critical role in medulloblastoma pathogenesis. Its amplification serves as a defining characteristic of group 3 medulloblastomas, associated with the most aggressive clinical course, greater frequency of metastases, and shorter survival times. Methods Medulloblastoma tissues (68 TMA, and 45 fresh tissues, and 31 controls) were stained (fluorescence and immunohistochemical) for AZIN1. Western blotting and ELISA were used to detect the AZIN1 level. Phenotypically aggressive cellular features were measured by increased invasion, colony formation and proliferation. CRISPR-Cas9-mediated AZIN1 knocked-out cells were orthotopically implanted in the cerebellum of nude mice (n = 8/group) with a stereotactic frame. Tumor growth was monitored using the In Vivo Imaging System (IVIS). Results Here, we investigated the role of AZIN1 expression in medulloblastoma. We found that overexpression of AZIN1 in medulloblastoma cells induces phenotypically aggressive features. Conducting in vivo studies we found that knocking-out AZIN1 in tumors corresponds with reduced tumor progression and prolonged survival. Clinical specimens are revealing that AZIN1 is highly expressed and directly correlates with MYC amplification status in patients. Conclusion These data implicate AZIN1 as a putative regulator of medulloblastoma pathogenesis and suggest that it may have clinical application as both a biomarker and novel therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03274-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff1e1e85866d4cc9a2e1386acb32f48ee6d86bb6" target='_blank'>
              AZIN1 level is increased in medulloblastoma and correlates with c-Myc activity and tumor phenotype
              </a>
            </td>
          <td>
            Julie Sesen, Tyra Martinez, Sara Busatto, L. Poluben, Hassan Nassour, Caroline Stone, Karthik Ashok, Marsha A. Moses, Edward R. Smith, Aram Ghalali
          </td>
          <td>2025-02-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DNA-dependent protein kinase catalytic subunit (DNA-PKcs) is a key kinase in the DNA repair process that responds to DNA damage caused by various factors and maintains genomic stability. However, DNA-PKcs is overexpressed in some solid tumors and is frequently associated with poor prognosis. DNA-PKcs was initially identified as a part of the transcription complex. In recent years, many studies have focused on its nonclassical functions, including transcriptional regulation, metabolism, innate immunity, and inflammatory response. Given the pleiotropic roles of DNA-PKcs in tumors, pharmacological inhibition of DNA-PK can exert antitumor effects and may serve as a potential target for tumor therapy in the future. This review summarizes several aspects of DNA-PKcs regulation of RNA metabolism, including its impact on transcriptional machinery, alternative splicing, and interaction with noncoding RNAs, and provides insights into DNA-PKcs beyond its DNA damage repair function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f296dc7b10466d316234339cf9617d0c3278a601" target='_blank'>
              DNA-PKcs, a player winding and dancing with RNA metabolism and diseases
              </a>
            </td>
          <td>
            Jiabao Hou, Mingjun Lu, Jingwei Guo, Jinghong Wu, Chenyang Wang, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2025-03-04</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract During meiosis, programmed DNA double-strand breaks (DSBs) are formed at hotspots to initiate homologous recombination, which is vital for reassorting genetic material. In fission yeast, the linear element (LinE) proteins Mug20, Rec25, and Rec27 interdependently bind chromosomal hotspots with high specificity and are necessary for high-level DSB formation. However, their mechanistic role in regulating the meiotic DSB machinery remains unknown. Here, using purified Mug20–Rec25–Rec27 (MRR) complex and functional intracellular analyses, we reveal that the MRR–DNA nucleoprotein complex assembles phase-separated condensates that compact the DNA. Notably, MRR complex formation is a prerequisite for DNA binding and condensate assembly, with Rec27 playing a pivotal role in directly binding DNA. Consistent with this finding, failure to form MRR–DNA condensates results in defective intracellular meiotic DSB formation and recombination. Our results provide mechanistic insights into how LinEs enhance meiotic DSB formation and provide a paradigm for studies in other species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2043e56114bc07be8ef392e7f7e4ffa5372e0c1" target='_blank'>
              Mug20–Rec25–Rec27 binds DNA and enhances meiotic DNA break formation via phase-separated condensates
              </a>
            </td>
          <td>
            Max F Wang, Meng-Yun Li, Ya-Ching Yang, Yu-Chien Chuang, Chieh-Yu Tsai, Mai-Chi Nguyen Binder, Lijuan Ma, Sheng-Wei Lin, Hung-Wen Li, Gerald R Smith, Peter Chi
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Garra rufa, commonly known as doctor fish, is a small freshwater cyprinid fish recognized for its high-temperature tolerance. G. rufa is primarily known for its role in ichthyotherapy, in which it helps remove keratinized skin from human hands and feet. However, in recent years, its high-temperature tolerance has attracted attention owing to its potential use as a model for human disease research, including infectious diseases and cancer xenograft models. Despite this interest, the genomic basis underlying high-temperature tolerance remains largely unexplored, primarily because of the limited availability of genomic resources, which hinders its development as an experimental model. In the present study, a high-quality chromosome-level genome assembly of G. rufa was generated using a combination of PacBio HiFi long-read sequencing and Hi-C technology. We generated a chromosome-level genome assembly with 25 chromosomes, 1.38 Gb in total length, and scaffold N50 of 49.3 Mb. Approximately 59% of the genome comprises repetitive elements, with DNA and LTR elements being particularly abundant. In total, 27,352 protein-coding genes were annotated, of which 26,900 genes (98.3%) were functionally annotated. Benchmarking Universal Single-Copy Orthologs (BUSCO) benchmark for genome assembly and gene annotation demonstrated 94.5% and 94.7% of complete BUSCOs, respectively. We identified two heat shock transcription factor (HSF) and 239 heat shock protein (HSP)-related genes. Heat shock elements, which are HSF-binding motifs, were present within 3 kb upstream of 944 genes, with statistically significant enrichment of HSP-related genes in this set. Furthermore, molecular phylogenetic analysis and whole-genome comparisons revealed that G. rufa is evolutionarily closely related to species in the Labeoninae subfamily of the Cyprinidae family. This chromosome-level reference genome provides a valuable resource for future research aimed at elucidating the molecular mechanisms underlying high-temperature tolerance in G. rufa and establishing it as a model organism for human biomedical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba084e3b18995a64de9db1867a75cd406732e5f" target='_blank'>
              Chromosome-level genome assembly of the doctor fish (Garra rufa)
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Kiki Syaputri Handayani, Liqing Zang, Fahrurrozi, Oleg Simakov, Victor David, Nico Gultom, Yasuhito Shimada
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Somatic mutations that increase hematopoietic stem cell (HSC) fitness drive their expansion in clonal hematopoiesis (CH) and predispose to blood cancers. Although CH frequently occurs with aging, it rarely progresses to overt malignancy. Population variation in the growth rate and potential of mutant clones suggests the presence of genetic factors protecting against CH, but these remain largely undefined. Here, we identify a non-coding regulatory variant, rs17834140-T, that significantly protects against CH and myeloid malignancies by downregulating HSC-selective expression and function of the RNA-binding protein MSI2. By modeling variant effects and mapping MSI2 binding targets, we uncover an RNA network that maintains human HSCs and influences CH risk. Importantly, rs17834140-T is associated with slower CH expansion rates in humans, and stem cell MSI2 levels modify ASXL1-mutant HSC clonal dominance in experimental models. These findings leverage natural resilience to highlight a key role for post-transcriptional regulation in human HSCs, and offer genetic evidence supporting inhibition of MSI2 or its downstream targets as rational strategies for blood cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, Chun-Jie Guo, Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, James M Allan, Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome replication in eukaryotic cells is surveyed by the S-phase checkpoint, which orchestrates sequential replication origin activation to avoid exhaustion of hitherto poorly defined rate-limiting replisome components. Here, we find that excessive activation of replication origins depletes chromatin-bound PCNA and lagging strand components, thereby limiting additional PCNA loading at new origins when checkpoint control is disrupted. PAF15 (PCNA-associated factor 15) emerges as a dosage-sensitive regulator of PCNA, delineating the dynamic range of global genome duplication and defining distinct roles for PCNA on the leading and lagging strands. Through its high-affinity PIP motif and interaction within the DNA encircling channel of PCNA, PAF15 stabilizes PCNA exclusively on the lagging strand, optimizing and rate-limiting lagging strand processing. On the other hand, misregulation of PAF15—whether by overexpression or mislocalization to the leading strand—impairs replication fork progression and leads to cell death. These defects are mitigated by TIMELESS and CLASPIN, which restrain PAF15-PCNA interactions beyond the lagging strand. E2F4-mediated repression orchestrates PAF15 expression in normal and cancer cells, maintaining its optimal dosage for lagging strand-specific interactions with PCNA. Thus, the S-phase checkpoint functions in concert to restrict origin activation when lagging strand PAF15-PCNA assembly is exhausted, linking a previously concealed strand-specific rate limitation to overall replication dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe32e8db1ccfcee1501ea6b48ca4b9c67236d177" target='_blank'>
              PAF15-PCNA assembly exhaustion governs lagging strand replication and replisome integrity
              </a>
            </td>
          <td>
            Gita Chhetri, S. Badugu, Narcis-Adrian Petriman, Mikkel Bo Petersen, Ganesha P. Pitchai, Aylin Seren Güller, Jan Novotný, Barath Balarasa, Morten Frendø Ebbesen, Frederik Tibert Larsen, Tina Ravnsborg, A. K. Yadav, Anita Lunding, Hana Polasek-Sedlackova, Ole Nørregaard Jensen, Kim Ravnskjaer, Jonathan R. Brewer, Jesper G.S. Madsen, Jens S. Andersen, Kumar Somyajit
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Ribosome biogenesis is one of the most fundamental and energetically demanding cellular processes. In humans, the ribosomal DNA (rDNA) repeats span a large region of DNA and comprise 200 to 600 copies of a ~43 kb unit spread over five different chromosomes. Control over ribosome biogenesis is closely tied to the regulation of the chromatin environment of this large genomic region. The proportion of rDNA loci which are active or silent is altered depending on the proliferative or metabolic state of the cell. Repeat silencing is driven by epigenetic changes culminating in a repressive heterochromatin environment. One group of proteins facilitating these epigenetic changes in response to growth or metabolic demands are ATP-dependent chromatin remodeling protein complexes that use ATP hydrolysis to reposition nucleosomes. Indeed, some chromatin remodelers are known to have indispensable roles in regulating the chromatin environment of rDNA. In this review, we highlight these proteins and their complexes and describe their mechanistic roles at rDNA. We also introduce the developmental disorders arising from the dysfunction of these proteins and discuss how the consequent dysregulation of rDNA loci may be reflected in the phenotypes observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b44d8b12e96514a790a07032e2485fc056c2a29" target='_blank'>
              It Takes a Village of Chromatin Remodelers to Regulate rDNA Expression
              </a>
            </td>
          <td>
            Mathieu G. Levesque, D. Picketts
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2e9255d35debfd9a15445b40a466e6a5d29ad6" target='_blank'>
              Chromosomal Instability and Clonal Heterogeneity in Breast Cancer: From Mechanisms to Clinical Applications
              </a>
            </td>
          <td>
            María Paula Meléndez-Flórez, O. Ortega-Recalde, N. Rangel, Milena Rondón-Lagos
          </td>
          <td>2025-04-04</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8774da0a06c116c814c2652572a737b8d86a0c02" target='_blank'>
              Genome-wide ribonucleotide detection in Archaea
              </a>
            </td>
          <td>
            Y. Moalic, Maurane Reveil, Deepali L. Kundnani, S. Balachander, Taehwan Yang, Alli L. Gombolay, Farahnaz Ranjbarian, Raphaël Brizard, Patrick Durand, H. Myllykallio, Mohamed Jebbar, Anders Hofer, F. Storici, Ghislaine Henneke
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2140165b325769cf3794c12431eadebdd635d22" target='_blank'>
              Chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Nina Trautwein, Bodo Laube, Christian Dietz, Robert W. Stark, M. C. Cardoso
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a03b89d9e1c30e4ab6bb1a8b31d9bee97c5524d" target='_blank'>
              Replisomes restrict SMC-mediated DNA-loop extrusion in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a494e730aa784cad617baf97d8efcc5bc5c5345" target='_blank'>
              Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer-associated mutations in IDH are associated with multiple types of human malignancies, which exhibit distinctive metabolic reprogramming, production of oncometabolite D-2-HG, and shifted epigenetic landscape. IDH mutated malignancies are signatured with “BRCAness”, highlighted with the sensitivity to DNA repair inhibitors and genotoxic agents, although the underlying molecular mechanism remains elusive. In the present study, we demonstrate that D-2-HG impacts the chromatin conformation adjustments, which are associated with DNA repair process. Mechanistically, D-2-HG diminishes the chromatin interactions in the DNA damage regions via revoking CTCF binding. The hypermethylation of cytosine, resulting from the suppression of TET1 and TET2 activities by D-2-HG, contributes to the dissociation of CTCF from DNA damage regions. CTCF depletion leads to the disruption of chromatin organization around the DNA damage sites, which abolishes the recruitment of essential DNA damage repair proteins BRCA2 and RAD51, as well as impairs homologous repair in the IDH mutant cancer cells. These findings provide evidence that CTCF-mediated chromatin interactions play a key role in DNA damage repair proceedings. Oncometabolites jeopardize genome stability and DNA repair by affecting high-order chromatin structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Natural plasmids are common in prokaryotes, but few have been documented in eukaryotes. The natural 2µ plasmid present in the yeast Saccharomyces cerevisiae is one of these best-characterized exceptions. This highly stable genetic element has coexisted with its host for millions of years, faithfully segregating at each cell division through a mechanism that remains unclear. Using proximity ligation methods (such as Hi-C, Micro-C) to map the contacts between 2µ plasmid and yeast chromosomes under dozens of different biological conditions, we found that the plasmid is tethered preferentially to regions with low transcriptional activity, often corresponding to long, inactive genes. These contacts do not depend on common chromosome-structuring factors, such as members of the structural maintenance of chromosome complexes (SMC) but depend on a nucleosome-encoded signal associated with RNA Pol II depletion. They appear stable throughout the cell cycle and can be established within minutes. This chromosome hitchhiking strategy may extend beyond the 2µ plasmid/S. cerevisiae pair, as suggested by the binding pattern of the natural eukaryotic plasmid Ddp5 along silent chromosome regions of the amoeba Dictyostelium discoideum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60d701689f8f3be285cb82707065b4218a7a99" target='_blank'>
              Parasitic plasmids are anchored to inactive regions of eukaryotic chromosomes through a nucleosome signal
              </a>
            </td>
          <td>
            Fabien Girard, Antoine Even, Agnès Thierry, Myriam Ruault, Léa Meneu, Pauline Larrous, Mickael Garnier, Sandrine Adiba, A. Taddei, Romain Koszul, A. Cournac
          </td>
          <td>2025-02-27</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e847c35fd0feaa6983cfa58c60c7da74934800f2" target='_blank'>
              Moss BRCA2 lacking canonical DNA binding domain promotes efficient homologous recombination and binds to DNA
              </a>
            </td>
          <td>
            Alice Chanteau, Suliane Quilleré, Arthur Crouset, Sreejith Allipra, Ulysse Tuquoi, P. Perroud, Simona Miron, P. Dupaigne, Sophie Zinn-Justin, Fabien Nogué, Rajeev Kumar
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ERBB2 (HER2) is a well-studied oncogene with several driver mutations apart from the well-known amplification defect in some breast cancers. We used the GigaAssay to test the functional effect of HER2 missense mutations on its receptor tyrosine kinase function. The GigaAssay is a modular high-throughput one-pot assay system for simultaneously measuring molecular function of thousands of genetic variants at very high accuracy. The activities of 5,886 mutations were classified, significantly more than mutants previously reported. These variants include 112 new in vitro, 10 known, and 9 new in vivo gain-of-function (GOF) mutations. Many of the GOFs spatially cluster in sequence and structure, supporting the activation mechanisms of heterodimerization with EGFR and release of kinase inhibition by the juxtamembrane domain. Retrospective analysis of patient outcomes from the Genomic Data Commons predicts increased survival with the newly identified HER2 GOF variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b136d3cab282e49a197992ab3575cc9958742822" target='_blank'>
              New Gain-of-Function Mutations Prioritize Mechanisms of HER2 Activation
              </a>
            </td>
          <td>
            C. Giacoletto, Liz J. Valente, Lancer Brown, Sara Patterson, Rewatee H Gokhale, S. Mockus, Wayne W. Grody, Hong-Wen Deng, Jerome I. Rotter, Martin R. Schiller
          </td>
          <td>2025-03-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d82ec08c9ad540baf17cea17836bedb8e4f45c6" target='_blank'>
              Chemical inhibition of Exon Junction Complex assembly impairs mRNA localization and neural stem cells ciliogenesis
              </a>
            </td>
          <td>
            Tommaso Villa, Oriane Pourcelot, D. Dierks, Marion Faucourt, Cindy Burel, Floric Slimani, Léa Guyonnet, Nathalie Spassky, Schraga Schwartz, Edouard Bertrand, Olivier Bensaude, H. Le Hir
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Transposable elements are abundant in host genomes but are generally considered to be confined to the cell in which they are expressed, with the notable exception of endogenous retroviruses. Here, we identify a group of LTR retrotransposons that infect the germline from somatic cells within the Drosophila ovary, despite lacking the fusogenic Envelope protein typically required for retroviral entry. Instead, these elements encode a short transmembrane protein, sORF2, with structural features reminiscent of viral cell-cell fusogens. Through genetics, imaging, and electron microscopy, we show that sORF2 localizes to invasive somatic protrusions, enabling the direct transfer of retrotransposon capsids into the oocyte. Remarkably, sORF2-like proteins are widespread among insect retrotransposons and also occur in piscine nackednaviruses and avian picornaviruses. These findings reveal a noncanonical, Envelope-independent transmission mechanism shared by retrotransposons and non-enveloped viruses, offering important insights into host-pathogen evolution and soma-germline interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81230deb71ae111c0bc4f1081eb1e2d5be4c7d01" target='_blank'>
              Direct cell-to-cell transmission of retrotransposons
              </a>
            </td>
          <td>
            Maya Voichek, Andreas Bernhard, Maria Novatchkova, Dominik Handler, Paul Möseneder, Baptiste Rafanel, P. Duchek, Kirsten A. Senti, Julius Brennecke
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, , Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Simple Summary Our research aims to test the suitability of ddPCR for the detection of tumor-related genomic imbalances (CNVs) in ovarian cancer (OC) in the peripheral blood system. In principle, in an artificial CNV simulation, we can show that CNVs can be detected even in larger dilutions in diploid DNA using ddPCR. Furthermore, we identified ovarian cancer CNVs by using aCGH and used this information to produce suitable ddPCR tests. These ddPCR tests are based on the principle of two differently colored comparative DNA loci that mark both very common genome imbalances in OC and regions that hardly indicate genome imbalances in OC. We were able to generate six ddPCR tests that indicate OC by identifying cell-free tumor DNA with a sensitivity of 0.8 and a specificity of 0.8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a29b17cdaeb68fcde9e2c47b4a309aeadaf32f5" target='_blank'>
              Detection of Genomic Copy Number Variations in Ovarian Cancer in the Peripheral Blood System
              </a>
            </td>
          <td>
            Lena Wahl, Ulyana Hliabtsova, Xueqian Qian, Anastasia Klopf, N. Hedemann, Inken Flörkemeier, C. Rogmans, Michael Kalab, Astrid Dempfle, Nicolai Maass, D. Bauerschlag, M. V. van Mackelenbergh, J. P. Weimer, Huy Duc Le
          </td>
          <td>2025-02-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f33e9da345df8224f97295c4e7efc05d28c012" target='_blank'>
              Crosstalk Between Plk1 and PTEN in Mitosis Affects Chromosomal Stability.
              </a>
            </td>
          <td>
            Wei Li, Xianning Wang, Jiannan Liu, Bing Liu, Yongjian Hao
          </td>
          <td>2025-03-21</td>
          <td>DNA and cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fca76d81501556f6f871b8db3defb5ed37bf0b72" target='_blank'>
              Loss of BCL7A Permits IRF4 Transcriptional Activity and Cellular Growth in Multiple Myeloma.
              </a>
            </td>
          <td>
            C. Chakraborty, S. Talluri, Moritz Binder, E. Morelli, Jessica Encinas Mayoral, Sanika Derebail, Anil Aktas Samur, Charles B Epstein, Kenneth C Anderson, M. Shammas, M. Samur, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4b38200a1231a3f3c1164315cc7d10135fa36f" target='_blank'>
              Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system
              </a>
            </td>
          <td>
            Stefanie Egetemaier, H. Chauvistré, R. Váraljai, Yichao Hua, Smiths S. Lueong, Samira Makhzami, Nalini Srinivas, Jan Forster, V. Ullrich, Simone Stupia, Valéria Schroeder, Sarah Scharfenberg, A. Hoewner, B. Shannan, J. Siveke, M. Baietti, Eleonora Leucci, J. Marine, A. Paschen, Bjoern Scheffler, D. R. Engel, Lisa M. Becker, Felix Nensa, Johannes Koester, Barbara Grünwald, S. Poepsel, Sabrina Ninck, F. Kaschani, Dirk Schadendorf, Juergen C. Becker, Alpaslan Tasdogan, Florian Rambow, Alexander Roesch
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, Min-Jung Lee, Nahoko Sato, Shraddha Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) is crucial for carcinogenesis and bile has direct contact with malignant biliary strictures, yet eccDNA features in bile and its function in malignant biliary strictures remain underexplored. Results We observed the widespread presence of eccDNA in bile and systematically profiled the landscape of bile cell-free eccDNA (bcf-eccDNA). For functional exploration, a simple and efficient workflow was designed to synthesize large eccDNA particularly containing multiple regions. Compared with the noncancer group, bcf-eccDNAs in the cancer group had different origins and larger sizes with six characteristic peaks. These peaks were also identified in the validation cohort (100%). There were more bcf-eccDNA carrying LINC00598 or CELF2 in malignant biliary strictures, showing potential diagnostic performance in training and validation cohorts (all AUCs > 0.9). Bcf-eccDNAs carried cancer-related mutations, which could guide treatment. EccDNA carrying miR-106a/363 cluster or miR-374b/421 cluster were proven to regulate cancer gene expression, accelerate tumor proliferation, and inhibit tumor apoptosis. Conclusions This study profiles a comprehensive bcf-eccDNA landscape in patients with biliary strictures and offers valuable insights into eccDNA's role in bile liquid biopsy and carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, , Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e5a35f02b297af4af1d7c2312aeb3f7e720c6f2" target='_blank'>
              p53-dependent chromatin relaxation is required for DNA double-strand break repair.
              </a>
            </td>
          <td>
            Hongyu Chen, Jin Shan, Wenjing Qi, Lili Chen, Xianlu Zeng
          </td>
          <td>2025-02-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214e281d859cf2be94a12cc54dca351d068768d3" target='_blank'>
              Genome-wide CRISPR activation screen identifies ARL11 as a sensitivity determinant of PARP inhibitor therapy.
              </a>
            </td>
          <td>
            Teng Zhang, Yuan Zhang, Xuxiang Wang, Haitian Hu, Christopher G. Lin, Yaru Xu, Hanqiu Zheng
          </td>
          <td>2025-03-23</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC), a highly heterogeneous disease, shows genomic instability and a high mutation rate, making it difficult to treat. Recent studies revealed that cancer stem cells (CSCs) play a critical role in MIBC frequent recurrence and high morbidity. Previous research has shown that Cyclooxygenases-2 (COX-2) is particularly highly expressed in bladder cancer cells. In recent years, the development of oncolytic adenoviruses and their use in clinical trials have gained increased attention. In this study, we composed a conditionally replicative adenovirus vector (CRAd-synNotch) that carries the COX-2 promotor driving adenoviral E1 gene, the synNotch receptor therapeutic gene, and the Ad5/35 fiber gene. Activation of the COX-2 promoter gene causes replication only within COX-2 expressing cancer cells, thereby leading to tumor oncolysis. Also, CD44 and HIF signals contribute to cancer stemness and maintaining CSCs in bladder cancer, and the transduced synNotch receptor inhibits both CD44 and HIF signals simultaneously. We performed an in vivo study using a mouse xenograft model of T24 human MIBC cells and confirmed the significant antitumor activity of CRAd-synNotch. Our findings in this study warrant the further development of CRAd-synNotch for treating patients with MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eee590b5a2cfc96e3c5fb66879d16597ce9245" target='_blank'>
              A conditionally replicative adenovirus vector containing the synNotch receptor gene for the treatment of muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Ruhan A, H. Ueki, Shunya Nishioka, Rion Yamazaki, Marina Maekawa, Koichi Kitagawa, Hideaki Miyake, Toshiro Shirakawa
          </td>
          <td>2025-02-26</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long interspersed element 1 (LINE-1) retrotransposons are repetitive sequences that can move within the genome by an autonomous mechanism. To limit their mutagenic potential, benign cells restrict LINE-1 expression through molecular mechanisms such as DNA methylation and histone modification, but these mechanisms are usually impaired in cancer. Clear cell ovarian carcinoma (CCOC) represents 5–10% of ovarian cancers and is thought to arise from endometriosis. Women with advanced CCOC face poor prognoses, highlighting the importance of understanding early disease pathogenesis. In our study, 33 of 40 cases (over 82%) of CCOC tumors express ORF1p, a LINE-1-encoded protein. We found that LINE-1 de-repression is an early event in CCOC, as ORF1p is enhanced during the transition from typical to atypical endometriosis and persists in invasive cancer. Finally, using single-molecule array (Simoa) assays, we detected ORF1p in patient blood, suggesting it as a potential minimally invasive biomarker for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5dfde3e4f7d39011a34eb5a718ac54f39aa6ee4" target='_blank'>
              Single cell spatial transcriptomics identifies coordinated cellular programs that are associated with good prognosis in microsatellite stable colorectal cancer
              </a>
            </td>
          <td>
            Victoria T. Karlsen, Umair Majid, Håvard T. Lindholm, S. U. Umu, Karoline Rapp Vander-Elst, A-C Røberg Beitnes, Marianne A. Merok, Kjersti Thorvaldsen Hagen, H. R. Nilsen, Sheraz Yaqub, E. Bækkevold, Diana Domanska, F. Jahnsen
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The translesion DNA synthesis (TLS) pathway mediated by proliferating cell nuclear antigen (PCNA) monoubiquitination is an essential mechanism by which cancer cells bypass DNA damage caused by DNA damage to maintain genomic stability and cell survival. Chromatin assembly factor 1 subunit A (CHAF1A) traditionally promotes histone assembly during DNA replication. Here, we revealed that CHAF1A is a novel regulator of the TLS pathway in cancer cells. CHAF1A promotes restart and elongation of the replication fork under DNA replication stress. Mechanistically, the C-terminal domain of CHAF1A directly interacts with E3 ubiquitin ligase RAD18, enhancing RAD18 binding on the stalled replication fork. CHAF1A facilitates PCNA K164 monoubiquitination mediated by RAD18, thereby promoting the recruitment of Y-family DNA polymerases and enhancing cancer cell resistance to DNA damage. In addition, CHAF1A-mediated RAD18 recruitment and PCNA monoubiquitination are independent of the CHAF1A-PCNA interaction and its histone assembly function. Taken together, these findings improve our understanding of the mechanisms that regulate the TLS pathway and provide insights into the relationship between CHAF1A and DNA replication stress in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d19733c6d85f7d8431cd411691114ef80a3ce053" target='_blank'>
              Chromatin assembly factor 1 subunit A promotes TLS pathway by recruiting E3 ubiquitin ligase RAD18 in cancer cells
              </a>
            </td>
          <td>
            Bing Wen, Hai-Xiang Zheng, Jing-Hua Heng, Qian Tang, Danxia Deng, Zhi-Da Zhang, Lian-di Liao, Liyan Xu, E. Li
          </td>
          <td>2025-03-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a92d215150e22b00d817410c07ac0e4856717a42" target='_blank'>
              Multidimensional OMICs reveal ARID1A orchestrated control of DNA damage, splicing, and cell cycle in normal-like and malignant urothelial cells.
              </a>
            </td>
          <td>
            Rebecca M Schlösser, Florian Krumbach, Eyleen Corrales, G. Andrieux, Christian Preisinger, Franziska Liss, A. Golzmann, M. Boerries, Kerstin Becker, Ruth Knüchel, Stefan Garczyk, Bernhard Lüscher
          </td>
          <td>2025-04-01</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background. Mutations in homologous recombination repair (HRR) genes (BRCA2, ATM, CHEK2, NBN, etc.) are found in 20–25 % of patients with metastatic prostate cancer (PC) and are an indication for prescription of PARP inhibitors. Sensitivity to these drugs results from homologous recombination deficiency (HRD) in the tumor. It is currently not fully elucidated which HRR genes besides BRCA1/2 cause HRD. The aim of the study was to evaluate the presence of homologous recombination deficiency in pc associated with mutations in different HRR genes. Material and methods. Paired tumor and normal DNA samples from 272 pc patients were examined using the HiSNP Ultra Panel v1.0 NGS panel (Nanodigmbio). Tumor copy number variation profiles were used to obtain the HRD score, which was determined as the unweighted sum of three characteristics: the number of chromosomal regions with loss of heterozygosity (LOH), large scale state transitions (LST), and telomeric allelic imbalances (TAI). HRD scores in different pc categories were compared using the Mann–Whitney test. Results. The studied case series included 58 pcs with pathogenic/ likely pathogenic mutations in at least one of 34 HRR genes, and 214 pcs without HRR mutations. The median HRD scores in the groups with BRCA2, ATM, CHEK2, other HRR mutations and without HRR mutations were 41, 22.5, 7.5, 7.5, 14 and 9, respectively. The HRD score was significantly higher in BRCA2-associated tumors than in other PC categories (p<0.01 for all comparisons), except in cases with atm mutations, where the difference did not reach formal significance (p=0.051). Homologous recombination deficiency, defined as HRD score ≥25, was observed in 19/58 (32.8 %) PCs with HRR mutations and 40/214 (18.7 %) tumors without HRR mutations (p=0.03). In tumors without HRR mutations, the HRD score was significantly higher in the presence of somatic TP53 mutations (p<0.0001). Conclusion. In contrast to BRCA2-associated PCs, most tumors with mutations in the CHEK2, NBN, BLM, FANCM, BRCA1 genes are not characterized by homologous recombination deficiency. in the case of ATM gene lesions, approximately half of pcs have a high HRD score. Testing for HRD score allows identification of a significant proportion (5–19 %, depending on the threshold chosen) of tumors with HRD among pcs that do not contain mutations in HRR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/475a2bac32f3b3fba84f14504ff2c301042d61c8" target='_blank'>
              Analysis of homologous recombination deficiency in prostate cancer
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, N. S. Morozova, A. D. Shestakova, E. Kuligina, E. Imyanitov
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA–RNA hybrids triggered by double-strand breaks (DSBs) are crucial intermediates during DSB repair, and their timely resolution requires numbers of RNA helicases, including DEAD box 1 (DDX1). However, how these helicases are recruited to DSB-induced hybrids in time remains largely unclear. Here, we revealed that squamous cell carcinoma antigen recognized by T cells 3 (SART3) promotes DDX1 binding to DNA–RNA hybrids at DSBs for optimal homologous recombination (HR) repair. SART3 itself associates with DNA–RNA hybrids and PAR chains and accumulates at DSBs in both PARylation- and DNA–RNA hybrids-dependent fashion. SART3 also associates with DDX1 and is necessary for DDX1 enrichment at DSBs. The defective SART3-DDX1 association observed in cells expressing the cancer-associated variant SART3-R836W impairs not only the accumulation of DDX1, but also hybrid removal and HR efficiency. Moreover, SART3 promotes DNA end resection through enhancing USP15-BARD1 association and BRCA1-BARD1 retention. Together, our study reveals an role of SART3 in DSB repair, rendering SART3 a promising target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0676fb55661a5af6ba07cbca79e578a0d145a7" target='_blank'>
              SART3 promotes homologous recombination repair by stimulating DNA-RNA hybrids removal and DNA end resection
              </a>
            </td>
          <td>
            Hui Fu, Min Huang, Honglin Wu, Hui Zheng, Yifei Gong, Lingyu Xing, Juanjuan Gong, Ruiyuan An, Qian Li, Xinyu Jie, Xiaolu Ma, Tie-Shan Tang, Caixia Guo
          </td>
          <td>2025-03-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cohesin and condensin, two related protein complexes, play essential roles in ensuring the accurate segregation of the genome into daughter cells during cell division. However, the interaction between cohesin and condensin in embryonic stem cells remains unclear, as does the specific function of the meiosis-specific cohesin complex. Cohesin maintains the cohesion of replicated sister chromatids until their separation at anaphase, whereas condensin facilitates the reorganization of chromosomes into a highly compact structure characteristic of mitosis. First, we found via ChIP-seq analysis that cohesins (SMC3, RAD21, and REC8) and condensin (SMC4) share DNA binding sites in close proximity and directly interact with the insulator protein CTCF. Second, siRNA-regulated SMC3 depletion led to nuclear accumulation of SMC4. Third, embryonic stem (ES) cells uniquely harbor cohesin complexes containing the meiotic kleisin subunit REC8. RAD21 knockdown increased the proportion of SMC3–REC8 complexes. Our findings indicate that cohesin and condensin make important contributions to the functions of the chromosomal organization, and that meiotic cohesin may be specifically required for the mitotic program in ES cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94533-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52632683a10ffe4a0eee9d00044e285b23727556" target='_blank'>
              Cohesin and condensin regulate chromosome topology and play an essential role in maintaining pluripotency in embryonic stem cells
              </a>
            </td>
          <td>
            Eui-Hwan Choi, Keun P. Kim
          </td>
          <td>2025-03-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Decitabine (DAC), a well-recognized DNA hypomethylating agent, has been applied to treat acute myeloid leukemia. However, clinic investigations revealed that DNA methylation reduction does not correlate with a clinical response, and relapse is prevalent. To gain a better understanding of its anti-tumor mechanism, we perform a temporally resolved CRISPR-Cas9 screen to identify factors governing the DAC response. We show that DNA damage generated by DNMT-DNA adducts and 5-aza-dUTP misincorporation through the dCMP deaminase DCTD act as drivers of DAC-induced acute cytotoxicity. The DNA damage that arises during the next S phase is dependent on DNA replication, unveiling a trans-cell cycle effect of DAC on genome stability. By exploring candidates for synthetic lethality, we unexpectedly uncover that KDM1A promotes survival after DAC treatment through interactions with ZMYM3 and CoREST, independent of its demethylase activity or regulation of viral mimicry. These findings emphasize the importance of DNA repair pathways in DAC response and provide potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, J. Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Background
 A critical area of recent cancer research is the emergence of transition states between normal and cancer that exhibit increased cell plasticity which underlies tumor cell heterogeneity. Pancreatic ductal adenocarcinoma (PDAC) can arise from the combination of a transition state termed acinar-to-ductal metaplasia (ADM) and a gain-of-function mutation in the proto-oncogene KRAS. During ADM, digestive enzyme-producing acinar cells acquire a transient ductal epithelium-like phenotype while maintaining their geographical acinar organization. One route of ADM initiation is the overexpression of the Krüppel-like factor 4 gene (KLF4) in the absence of oncogenic driver mutations. Here, we asked to what extent cells acquire and retain an epigenetic memory of the ADM transition state in the absence of oncogene mutation. Methods We profiled the DNA methylome and transcriptome of KLF4-induced ADM in transgenic mice at various timepoints during and after recovery from ADM. We validated the identified DNA methylation and transcriptomic signatures in the widely used caerulein model of inducible pancreatitis. Results We identified differential DNA methylation at Kras-downstream PI3K and Rho/Rac/Cdc42 GTPase pathway genes during ADM, as well as a corresponding gene expression increase in these pathways. Importantly, differential methylation persisted after gene expression returned to normal. Caerulein exposure, which induces widespread digestive system changes in addition to ADM, showed similar changes in DNA methylation in ADM cells. Regions of differential methylation were enriched for motifs of KLF and AP-1 family transcription factors, as were those of human pancreatic intraepithelial neoplasia (PanIN) samples, demonstrating the relevance of this epigenetic transition state memory in human carcinogenesis. Finally, single-cell spatial transcriptomics revealed that these ADM transition cells were enriched for PI3K pathway and AP1 family members. Conclusions Our comprehensive study of DNA methylation in the acinar-ductal metaplasia transition state links epigenetic memory to cancer-related cell plasticity even in the absence of oncogenic mutation. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01452-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4322b5bf32e8bd656498dedccd6681209c229fa5" target='_blank'>
              DNA methylation memory of pancreatic acinar-ductal metaplasia transition state altering Kras-downstream PI3K and Rho GTPase signaling in the absence of Kras mutation
              </a>
            </td>
          <td>
            E. K. Lo, Adrian Idrizi, Rakel Tryggvadottir, Weiqiang Zhou, Wenpin Hou, Hongkai Ji, Patrick Cahan, Andrew P. Feinberg
          </td>
          <td>2025-03-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0818d31f654dc2a6f66d1620f983e96d2fa6b573" target='_blank'>
              Genomic and fragmentomic landscapes of cell-free DNA for early cancer detection.
              </a>
            </td>
          <td>
            D. C. Bruhm, Nicholas A. Vulpescu, Zachariah H. Foda, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2025-03-04</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The current treatment of triple-negative breast cancer (TNBC) is still primarily based on platinum-based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin-tolerant cells after platinum administration. Here, it is reported that upregulated N 6-methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase-like 3 (METTL3), facilitating METTL3 interaction with Wilms'-tumor-1-associated protein and subsequently increasing m6A of transcript-associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2-Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe1e275d0931ab0098bd9862f4488d0d9192898d" target='_blank'>
              Unveiling FAM111B: A Pan-Cancer Biomarker for DNA Repair and Immune Infiltration
              </a>
            </td>
          <td>
            Fang Wei, Wanying Li, Ting Zhou, Xianglin Yuan, Lihong Zhang
          </td>
          <td>2025-03-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer driver mutations are defined by their high prevalence in cancers and presumed rarity in normal tissues. However, recent studies show that positive selection in normal epithelia can increase the prevalence of some cancer drivers. To determine their true cancer-driving potential, it is essential to evaluate how frequent these mutations are in normal tissues and what are their phenotypes. Here, we explore the bioavailability of somatic variants by quantifying age-related mutational burdens in normal human colonic epithelium using immunodetection in FFPE samples (N = 181 patients). Positive selection of variants of tumour suppressor genes PTEN and ARID1A associates with monoallelic gene loss as confirmed by CRISPR/Cas9 mutagenesis and changes in their downstream effectors. Comparison of the mutational burden in normal tissue and colorectal cancers allows quantification of cancer driver potency based on relative representation. Additionally, immune exclusion, a cancer hallmark feature, is observed within ARID1A-deficient clones in histologically normal tissue. The behaviour resulting from haploinsufficiency of PTEN and ARID1A demonstrates how somatic mosaicism of tumour suppressors arises and can predispose to cancer initiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea14c2cc6a1c9c0bd6cb85f297b48295dc464ac" target='_blank'>
              Immunogenomic precision medicine: a personalized approach based on immunogenomic cancer evolution.
              </a>
            </td>
          <td>
            Yusaku Momoi, S. Kumagai, H. Nishikawa
          </td>
          <td>2025-03-29</td>
          <td>International immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9245a141fa9c9180cb54c8e88a50ab331a25714d" target='_blank'>
              Organisation of axial regions of isolated mitotic chromosomes visualised by cryo correlative light and electron tomography
              </a>
            </td>
          <td>
            Gurudatt Patra, M. Harastani, K. Samejima, Lucy Remnant, Nathalie Troffer-Charlier, Corrine Crucifix, Alexandre Durand, Nils Marechal, Yves Lutz, A. Steyer, Zhengyi Yang, Wim J. H. Hagen, William C. Earnshaw, Mikhail Eltsov
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0fddfdbd53dd2702eeb20d67caa73e1dfbdec2c" target='_blank'>
              Impact of loss of PRLHR on aneuploidy and initiation of bladder cancer.
              </a>
            </td>
          <td>
            Xi Zheng, Bingjie Yang, Wenya Wang, A. Azoitei, Felix Wezel, Christian Bolenz, Hongqian Guo, Cagatay Günes
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S-adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient-deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti-Lat4 small interfering RNA complex nanoregulator (AS-F-NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor-infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation.
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background/Objectives: Breast cancer is the most prevalent cancer in adult women. Currently, new therapies and protein determinations with prognostic value are under development. Fascin (encoded by the FSCN1 gene) is an actin-binding protein that is critical for the development of cytoplasmic projections that are essential for tumor invasion. DNA topoisomerase 2-alpha (TOP2A) is a nuclear protein crucial for ATP-dependent breakage, passage, and rejoining of double-stranded DNA and cell division. Both proteins are associated with higher proliferation rates and worse prognosis in breast cancer and together can provide comprehensive information on prognosis and treatment response. Methods: We simultaneously assessed fascin expression and TOP2A/CEP17 DNA copy number ratios in various histological and molecular subtypes. Additionally, these markers were analyzed along with previously established diagnostic markers and other relevant clinical data. Results: Our series included 265 patients, four of whom were male, and all of which were diagnosed with breast carcinoma. Of the 265 patients initially included, sufficient material for analysis was available for 175 cases, as some samples were excluded because of insufficient tissue quantity, poor preservation, or lack of hybridization in certain assays. Immunohistochemical (IHC) expression of fascin, both in its aggregated form and by category, showed no association with the TOP2A gene alteration ratio. Fascin expression was significantly associated with histological subtype (p < 0.001), molecular subtype (p < 0.001), hormone receptor (HR) (p < 0.001), BCL2 (p = 0.003), Ki67 (p = 0.002), and histological grade (p < 0.001). TOP2A was significantly associated with molecular subtype (p = 0.041), Ki67 (p = 0.048), and histological grade (p = 0.033). In our study, molecular subtype (p = 0.037) emerged as an independent variable for the complete histological response to neoadjuvant treatment. Multivariate analysis linked pathological stage (p = 0.002) and estrogen receptor (ER) expression (p = 0.004) to overall survival (OS) and disease-free survival (DFS). Conclusions: No statistical relationship was evident between fascin expression (IHC) and the TOP2A copy ratio. The results of this study suggested that the mechanisms of increased cell proliferation associated with alterations in fascin and TOP2A are independent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbe80a72237e1c2774d5c95c120ee2f1e9208a6" target='_blank'>
              Expression of Fascin and DNA Topoisomerase 2-Alpha in Breast Carcinoma: Correlation with Histological Subtypes and Other Prognostic Markers
              </a>
            </td>
          <td>
            Alberto Sánchez-Espinosa, José García-Rodríguez, Virginia Alonso-Aguirre, Jesús Acosta‐Ortega, Pablo Conesa-Zamora, J. García‐Solano, G. Luengo-Gil
          </td>
          <td>2025-03-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5083118ad8ca69bf9d2b345a596f9d2646c08163" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M. Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, , Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R. Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the single nucleotide polymorphism (SNP) rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC (OR = 0.78, P = 0.015). Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling.
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is a rising issue worldwide, and numerous studies have focused on understanding the underlying reasons for its occurrence and finding proper ways to defeat it. By applying technological advances, researchers are continuously uncovering and updating treatments in cancer therapy. Their vast functions in the regulation of cell growth and proliferation and their significant role in the progression of diseases, including cancer. This review provides a comprehensive analysis of ncRNAs in breast cancer, focusing on long non-coding RNAs such as HOTAIR, MALAT1, and NEAT1, as well as microRNAs such as miR-21, miR-221/222, and miR-155. These ncRNAs are pivotal in regulating cell proliferation, metastasis, drug resistance, and apoptosis. Additionally, we discuss experimental approaches that are useful for studying them and highlight the advantages and challenges of each method. We then explain the results of these clinical trials and offer insights for future studies by discussing major existing gaps. On the basis of an extensive number of studies, this review provides valuable insights into the potential of ncRNAs in cancer therapy. Key findings show that even though the functions of ncRNAs are vast and undeniable in cancer, there are still complications associated with their therapeutic use. Moreover, there is an absence of sufficient experiments regarding their application in mouse models, which is an area to work on. By emphasizing the crucial role of ncRNAs, this review underscores the need for innovative approaches and further studies to explore their potential in cancer therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d581bb8c9e5870e5b44e989456c39919368fe9" target='_blank'>
              Non-coding RNAs, a double-edged sword in breast cancer prognosis
              </a>
            </td>
          <td>
            Maryam Solaimani, Sahar Hosseinzadeh, M. Abasi
          </td>
          <td>2025-04-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Gastrointestinal stromal tumors (GISTs) span a broad clinical spectrum, from indolent neoplasms to life‐threatening metastatic tumors. A persistent limitation of current risk stratification systems is that a subset of GISTs is graded as low‐risk but nevertheless metastasizes. Therefore, new predictive factors that improve risk stratification are needed. In this exploratory study, we investigated the potential of genome‐wide DNA methylation profiling and copy number variation (CNV) analysis as additional prognostic tools for GISTs. We collected a cohort of 28 patients with GIST diagnosed between 2001 and 2022, with available follow‐up and molecular data. This included 15 patients without progressive disease (seven low‐risk and eight moderate‐ to high‐risk GISTs) and 13 with progressive disease. Among those with progression, eight experienced recurrence or metastasis post‐surgery (one low‐risk, seven high‐risk GISTs), while five had metastatic disease at initial diagnosis. Risk stratification was determined according to Miettinen's criteria. Genome‐wide DNA methylation data and CNV plots were generated from imatinib‐naïve primary GISTs using the Illumina Infinium MethylationEPIC BeadChip array. Unsupervised cluster analysis revealed distinct DNA methylation patterns predominantly associated with anatomical location and genotype. Differential DNA methylation analysis comparing primary gastric GISTs associated with and without progressive disease showed 8 differentially methylated regions spanning the coding and promoter areas of 6 genes. CNV analysis demonstrated that GISTs associated with progressive disease had the most CNVs, whereas low‐risk, non‐progressive GISTs had the fewest. Despite the limited sample size, this exploratory study indicates that genome‐wide DNA methylation profiling and CNV analysis could enhance GIST risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cde7ce155d81225e7ec733ade903dd81397bee4" target='_blank'>
              Genome‐Wide DNA Methylation and Copy Number Alterations in Gastrointestinal Stromal Tumors
              </a>
            </td>
          <td>
            T. Kleijn, B. Ameline, R. Bleckman, W. Kooistra, E. van den Broek, G. Diercks, B. V. van Hemel, Bert Timmer, W. Timens, G. Kats-Ugurlu, L. V. van Kempen, B. van Etten, E. Schuuring, Albert J H Suurmeijer, J.J. de Haan, D. Baumhoer, A. K. Reyners, A. Cleven
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce2dfa193c78b50465920b8e77584f1d48b10014" target='_blank'>
              SCFRMF-dependent degradation of the nuclear lamina releases the somatic chromatin mobility restriction for meiotic recombination.
              </a>
            </td>
          <td>
            Xinjie Yuan, Bowei Cai, Yuki Hamamura, A. Schnittger, Chao Yang
          </td>
          <td>2025-02-21</td>
          <td>Science advances</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cellular differentiation limits cellular plasticity allowing cells to attain their specialized functional characteristics and phenotypes, whereas loss of differentiation is a hallmark of cancer. Thus, characterizing mechanisms underlying differentiation is key to discover new cancer therapeutics. We report a novel functional antagonistic relationship between the prolactin (PRL)/prolactin receptor (PRLR) differentiation pathway and YAP-CCN2 oncogenic pathway in normal mammary epithelial cells and breast cancer cells that is essential for establishing/maintaining acinar morphogenesis, cell-cell junctions and the intracellular localization of apical-basal polarity protein complexes (Par, Crumb and Scrib). Importantly, using CRISPR knockout of the PRLR in MCF7, HR+ breast cancer cells, further revealed that the negative relationship between PRL/PRLR pathway and YAP-CCN2 pathway is critical in suppressing luminal-to-basal stem-like lineage plasticity. Furthermore, the clinical relevance of this interplay was evaluated using bioinformatics approaches on several human datasets, including samples from normal breast epithelium, breast cancer, and 33 other cancer types. This analysis revealed a positive correlation between PRLR and the YAP suppressor Hippo pathway and a co-expression gene network driving favourable patients’ survival outcomes in breast cancer. The therapeutic potential of this interplay was also evaluated in vitro using MDA-MB-231 cells, a preclinical model of human triple-negative breast cancer, where treatment with PRL and Verteporfin, an FDA-approved pharmacological YAP-inhibitor, alone or their combination suppressed the expression of the mesenchymal marker vimentin and the stem cell marker CD44 as well as reduced their Ki67 proliferative marker expression. Collectively, our results emphasize the pro-differentiation role of PRL/PRLR pathway in mammary and breast cancer cells and highlight that promoting PRL/PRLR signaling while inhibiting the YAP-CCN2 oncogenic pathway can be exploited as a differentiation-based combination therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ad66c62499b6e33f65c5f1c8b999c11d058d23" target='_blank'>
              A novel clinically relevant antagonistic interplay between prolactin and oncogenic YAP-CCN2 pathways as a differentiation therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Xueqing Liu, Alaa Moamer, Roger Gomes da Silva, Aidan Shoham-Amizlev, Dana Hamam, Anwar M. Shams, Jean-Jacques Lebrun, Suhad Ali
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. Methods We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (n = 119) and longitudinal cohorts (n = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. Results In the longitudinal cohorts, the only statistically significant associations with time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR 12.13, p = 0.003, Wald test with FDR correction), ER status (HR 0.16 for ER+ compared to ER−, p = 0.0045), and divergence in SNVs between the two samples (HR 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant associations with time to progression to invasive disease were the combination of the width of the surgical margin (HR 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR 1.30 per 10 SNVs, p = 0.02). No predictors were significantly associated with both DCIS recurrence and progression to invasive disease, suggesting that the evolutionary scenarios that lead to these clinical outcomes are markedly different. Conclusions These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-01966-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce21ed4e0ea0f10efa07b89c2a3fd98167bc680" target='_blank'>
              Evolutionary measures show that recurrence of DCIS is distinct from progression to breast cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, Marc D. Ryser, Joseph Y Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C Maley
          </td>
          <td>2025-03-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Birt-Hogg-Dubé syndrome (BHD) patients are uniquely susceptible to all renal tumor subtypes. However, the underlying mechanism of carcinogenesis is unclear. To study cancer development in BHD, we used human proximal kidney (HK2) cells and found that long-term folliculin (FLCN) knockdown was required to increase the tumorigenic potential of these cells, as evidenced by the formation of larger spheroids under nonadherent conditions. Transcriptomic and proteomic analyses revealed links between the FLCN, cell cycle control and DNA damage response (DDR) machinery. In addition, HK2 cells lacking FLCN had an altered transcriptome profile and enriched cell cycle control genes. G1/S cell cycle checkpoint signaling was compromised by increased protein levels of cyclin D1 (CCND1) and hyperphosphorylation of retinoblastoma 1 (RB1). A FLCN interactome screen revealed that FLCN binds to DNA-dependent protein kinase (DNA-PK). This novel interaction was reversed in an irradiation-responsive manner. Knockdown of FLCN in HK2 cells caused a marked increase in γH2AX and RB1 phosphorylation. The levels of both CCND1 and phosphorylated RB1 remained high during DNA damage, which was associated with defective cell cycle control caused by FLCN knockdown. Furthermore, Flcn-knockdown C. elegans were defective in cell cycle arrest caused by DNA damage. This work revealed that long-term FLCN loss and associated cell cycle defects in BHD patients could contribute to their increased risk of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce72e1df530d7f8cfbe56a7a8e356852f9f7441" target='_blank'>
              Characterizing the tumor suppressor activity of FLCN in Birt-Hogg-Dubé syndrome cell models through transcriptomic and proteomic analysis.
              </a>
            </td>
          <td>
            Rachel-Ann Jones, E. Dunlop, Jesse D Champion, Peter F Doubleday, T. Claessens, Zahra Jalali, Sara Seifan, Iain Perry, Peter J. Giles, Oliver Harrison, Barry J. Coull, A. Houweling, A. Pause, Bryan A Ballif, Andrew R. Tee
          </td>
          <td>2025-03-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, Marc van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc6f21df46de0763db659ebbc92b87b44e41294" target='_blank'>
              Centrophilic Retrotransposons of Plant Genomes.
              </a>
            </td>
          <td>
            Alexandros Bousios, T. Kakutani, Ian R. Henderson
          </td>
          <td>2025-02-14</td>
          <td>Annual review of plant biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Over the past decade, circular RNAs (circRNAs) have gained recognition as a novel class of genetic molecules, many of which are implicated in cancer pathogenesis via different mechanisms, including drug resistance, immune escape, and radio-resistance. ExosomalcircRNAs, in particular, facilitatecommunication between tumour cells and micro-environmental cells, including immune cells, fibroblasts, and other components. Notably, micro-environmental cells can reportedly influence tumour progression and treatment resistance by releasing exosomalcircRNAs. circRNAs often exhibit tissue- and cancer-specific expression patterns, and growing evidence highlights their potential clinical relevance and utility. These molecules show strong promise as potential biomarkers and therapeutic targets for cancer diagnosis and treatment. Therefore, this review aimed to briefly discuss the latest findings on the roles and resistance mechanisms of key circRNAs in the treatment of various malignancies, including lung, breast, liver, colorectal, and gastric cancers, as well as haematological malignancies and neuroblastoma.This review will contribute to the identification of new circRNA biomarkers for the early diagnosis as well as therapeutic targets for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e068de948072497de4cda02de43b3f132918d12" target='_blank'>
              Role of DNA methylation and non‑coding RNAs expression in pathogenesis, detection, prognosis, and therapy‑resistant ovarian carcinoma (Review).
              </a>
            </td>
          <td>
            Victor M Del Castillo Falconi, Jenny Godinez Rodriguez, Verónica Fragoso‑Ontiveros, Laura Contreras-Espinosa, Abraham Pedroza‑Torres, José Díaz‑chávez, Luis A Herrera
          </td>
          <td>2025-04-01</td>
          <td>Molecular medicine reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background TP53 mutations are linked to aggressive progression and chemoresistance in gastric cancer (GC). Frameshift mutation is the second most common mutation type of TP53. However, the consequences of this mutation type in GC were not well understood, and targeted therapies for cancer patients harboring frameshift mutations were also not established. Histone methylation significantly influences tumorigenesis in TP53-mutated cancers, and related inhibitors are emerging as specific therapeutic strategies. Methods and results By treating GC cell lines harboring various TP53 mutation types with a library of histone demethylase inhibitors, we identified that GSK690, a reversible inhibitor of lysine-specific demethylase 1 (LSD1), selectively inhibits GC cells harboring TP53 frameshift mutations without nuclear localization sequence (NLS) (termed TP53 Frameshift NLS), which accounts for 89% TP53 frameshift mutations in GC patients. GSK690 showed significant specific inhibition in vitro and in vivo against this subtype by inducing G1/S cell cycle arrest via the LSD1-CCNA2 axis. Importantly, dual-luciferase assays and ChIP-qPCR confirmed that the loss of transcriptional repression activities of p53 in drives LSD1 upregulation in TP53 Frameshift NLS cancer cells. Conclusions In summary, our results indicate that the nuclear localization deficiency of p53 accounts for increased expression of LSD1 in TP53 Frameshift NLS GCs. GSK690 inhibits cell cycle progression and tumor growth by suppressing aberrantly activated LSD1-CCNA2 signaling in this GC subtype, counteracting malignant proliferation and thereby providing a precise therapeutic strategy for GC patients with TP53 Frameshift NLS. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01829-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Rationale: Meiotic homologous recombination is a critical event in gametogenesis, which is tightly regulated to ensure the generation of crossovers on homologous chromosomes. This process is crucial for ensuring the accurate segregation of genetic material and maintaining genetic diversity within species, ultimately contributing to reproductive success. Nevertheless, comprehensive mechanisms of post-translational modification (PTM) regulating homologous recombination during meiosis require further investigation. The aim of this study is to investigate the regulatory mechanisms and physiological functions of NAE1-mediated neddylation during meiosis of mammalian spermatogenesis and its consequential role in infertility. Methods: The dynamic localization of NAE1 at various sub-stages during spermatogenesis was determined using immunofluorescence staining and seminiferous tubule staging. We explore the role of NAE1-mediated neddylation by utilizing germ cell-specific Nae1-knockout mice. The impact on homologous synapsis and recombination during the meiosis prophase I were verified through chromosome spread fluorescence staining. We used 10 × Genomics single cell transcriptomics and ubiquitinomics to analysis the causes of spermatogenesis arrest and spermatogenic apoptosis. Results: NAE1 exhibited high nuclear expression within spermatocytes from the pachytene stage onwards. Nae1-SKO male mice showed a late-pachytene arrest in spermatocytes, resulting in infertility. In NAE1-deficient spermatocytes, there is an increase in apoptosis. Nae1 deletion led to double-strand break (DSB) repair failure with normal autosomes synapsis. From a mechanistic perspective, we verified excessive recombination intermediate stabilization and failed crossover formation, which ultimately resulted in impaired meiotic recombination. Further analysis showed that ubiquitination regulation coordinated with NAE1-mediated neddylation was implicated in meiotic recombination. Conclusion: NAE1-mediated neddylation regulates ubiquitination during meiosis and is involved in the stabilization of recombination proteins related to crossover differentiation. We provide cytological evidence for the neddylation-ubiquitination system (NUS) in mammalian meiotic recombination during spermatogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d73e475fa9d9d51166b51741f2be3758e72a36" target='_blank'>
              NAE1-mediated neddylation coordinates ubiquitination regulation of meiotic recombination during spermatogenesis
              </a>
            </td>
          <td>
            Yu Xi, Chenjia Gong, Zhe Zhang, Feiyin Zhu, Ying Zhang, Yanlin Tang, Li-Hua Yan, Hui Jiang, Jie Qiao, Qiang Liu
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality in homologous recombination (HR)-deficient cancers caused by Poly (ADP-ribose) polymerase inhibitors (PARPi) has been classically attributed to its role in DNA repair. The mode of action of PARPi and resistance thereof are now believed to be predominantly replication associated. Therefore, effective combinatorial approaches of targeting replication fork processing along with HR-downregulation to target HR-proficient and possibly PARPi-resistant tumors are warranted. Stilbenes are a privileged class of molecules, which include resveratrol, pterostilbene, piceatannol, etc, that modulate both replication processes and RAD51-expression. In this investigation, by screening a small library of stilbenes, including in-house synthesized molecules, trans-4,4'-dihydroxystilbene (DHS) was discovered as a potent natural agent, which downregulates RAD51 expression and HR repair (GFP-reporter assay). DHS induces extensive synergistic cell death in ovarian cancers when combined with talazoparib (PARPi). Mechanistically, DHS elicits replication-stress through severely impeding replication fork progress, speed, and inducing fork-asymmetry. This leads to robust induction of single stranded DNA (ssDNA) gaps and poly-ADP-ribosylation (PARylation) in S-phase cells, signifying issues related to lagging (Okazaki) strand synthesis. PARPi, which abrogates PARylation, potentiates DHS induced ssDNA gaps, and their conversion into lethal double strand breaks through MRE11 action. Furthermore, the combination is highly effective in mitigating ovarian tumor xenograft growth in SCID mice and exhibited a good therapeutic-index with no/minimal tissue-toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e984c0229d51f87822834c928220d9d0bc66005" target='_blank'>
              Targeting Replication Fork Processing Synergizes with PARP Inhibition to Potentiate Lethality in Homologous Recombination Proficient Ovarian Cancers.
              </a>
            </td>
          <td>
            Ganesh Pai Bellare, Kshama Kundu, Papiya Dey, Krupa Thankam Philip, Nitish Chauhan, Muskan Sharma, Sankarsingh Kesharsingh Rajput, B. Patro
          </td>
          <td>2025-03-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd93a55f84ce2095fede9891e98cc26d5f84458d" target='_blank'>
              Doxorubicin, a DNA intercalator, inhibits transcription elongation.
              </a>
            </td>
          <td>
            Mathew Tempel, Kari Green, Dhanvi Prajapati, Angela Duaqui, Mahboobeh Norouzi, Hedieh Sattarifard, Ahmed Ashraf, Elly Wu, Athanasios Zovoilis, Ted M Lakowski, James R Davie
          </td>
          <td>2025-02-14</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492c2179e3d3bf51d18c5cb81a6a73a7606c5faa" target='_blank'>
              A Nucleus-Targeting Ruthenium(II) Complex Induces DNA Condensation in Cisplatin-Resistant Tumor Cells.
              </a>
            </td>
          <td>
            Ying Zhou, Kai Xiong, Tao Feng, Xianbo Wu, Jinzhe Liang, Yu Chen, Hui Chao
          </td>
          <td>2025-04-01</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26334a481a2a25fb7be72ec2c09273c472c1ce03" target='_blank'>
              Intrinsic immunity against HAdV is achieved by a novel epigenetic silencing complex
              </a>
            </td>
          <td>
            Julia Mai, Maryam Karimi, A. Kuderna, Anna Heider, Thomas Günther, C. Friedel, A. Grundhoff, Thomas Stamminger, Sabrina Schreiner
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f8c8e278af4ce203bace432e47ab666e9ddfe2" target='_blank'>
              Small Genomes, Big Disruptions: Parvoviruses and the DNA Damage Response
              </a>
            </td>
          <td>
            Rhiannon R. Abrahams, Kinjal Majumder
          </td>
          <td>2025-03-29</td>
          <td>Viruses</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: The cell cycle, a critical and intricate biological process, comprises various phases, and its dysregulation plays a pivotal role in tumorigenesis and metastasis. The exploration of cell cycle-based molecular subtypes across pan-cancers, along with the application of synthetic lethality concepts, holds promise for advancing cancer therapies. Methods: A pan-cancer analysis was conducted to assess the cell cycle serves as a reliable signature for classifying molecular subtypes and to understand the potential clinical application of genes as potential drug targets based on synthetic lethality. Results: Molecular subtypes derived from cell cycle features in certain cancers, particularly kidney-related malignancies, exhibited distinct immune characteristics. Synthetic lethal interactions within the cell cycle pathway were common, with significant genetic interactions further identifying potential drug targets through the exploitation of genetic relationships with key driver genes. Additionally, miRNAs and lncRNAs may influence the cell cycle through miRNA:mRNA interactions and ceRNA networks, thereby enriching the genetic interaction landscape. Conclusions: These findings suggest that the cell cycle pathway could serve as a promising molecular subtype signature to enhance cancer prognostication and offer potential targets for anticancer drug development through synthetic lethality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f0509708b74787ef228103fd17aa6cf4b5bae5" target='_blank'>
              Cell Cycle-Based Molecular Features via Synthetic Lethality and Non-Coding RNA Interactions in Cancer
              </a>
            </td>
          <td>
            Shizheng Xiong, Jiaming Jin, Xinmiao Zhao, Yang Zhao, Zhiheng He, Haochuan Guo, Chengjun Gong, Jiafeng Yu, Li Guo, Tingming Liang
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The retinoblastoma (RB) gene is an important tumor suppressor gene with a higher mutation frequency than other tumor suppressor genes. The mutation or inactivation of RB has been found in various cancers. The discovery of small molecules to promote RB expression is an effective anti-cancer strategy. Special DNA secondary structures with G-quadruplex and i-motif on the RB promoter could act as “molecular switches” for gene transcriptional regulation and are potentially important targets for the development of new anti-cancer drugs. After extensive screening, we found that the bisacridine derivative A06 had selective binding and destabilization for both the G-quadruplex and i-motif on the RB promoter, which significantly up-regulated RB gene transcription and translation, resulting in the inhibition of tumor cell proliferation and metastasis. A06 exhibited potent anti-tumor activity on Hela cells and strongly suppressed tumor growth on the Hela xenograft mice model without significant toxicity. In comparison, A02 exhibited strong binding and destabilization to the RB promoter G-quadruplex only, which showed a much weaker effect than A06 on regulating RB expression and producing anti-tumor activity. As we know, this is the first study for up-regulating a tumor suppressor gene through destabilization of both the G-quadruplex and i-motif on the gene promoter, which provides a new strategy for innovative anti-cancer drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2432014fea9848b263f5ea65101b4b9216843d82" target='_blank'>
              Selective Up-Regulation of Tumor Suppressor Gene Retinoblastoma by Bisacridine Derivative Through Gene Promoter Quadruplex Structures for Cancer Treatment
              </a>
            </td>
          <td>
            Xiaomin Lin, Jiahui Zhang, Jihai Liang, Dongsheng Ji, Zhi-Shu Huang, Ding Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e899cf4a3d1394fc8fe742ffd50bbb8e81d067" target='_blank'>
              Extensive mutual influences of SMC complexes shape 3D genome folding.
              </a>
            </td>
          <td>
            Han Zhao, Lirong Shu, Shiyi Qin, Fangxuan Lyu, Fuhai Liu, En Lin, Sijian Xia, Baiyue Wang, Manzhu Wang, Fengnian Shan, Yinzhi Lin, Lin Zhang, Yufei Gu, Gerd A. Blobel, Kai Huang, Haoyue Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f25e9aede0dd0d8017cba72b4efd6a1c07f803" target='_blank'>
              Molecular evolution of HRR gene alterations in metastatic prostate cancer: A ctDNA-based study.
              </a>
            </td>
          <td>
            Chinmay T Jani, Eli Tran, Nicole Zhang, J. Tsai, Jianjie Dong, Elizabeth Pan, Justine Panian, Rana R McKay
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="eccDNA is a driver of many cancers and a potential intermediate in other age-related disorders. However, little is known about the mechanisms underlying eccDNA formation in healthy tissue and how aging affects these processes. Here, we present an atlas of eccDNA across seven tissues of male mice spanning four ages. EccDNA correlates with open chromatin characterized by signatures of H3K27ac and H3K4me1. Additionally, the mutational load of eccDNA on genes correlates with tissue-specific transcription and increases logarithmically as a function of transcript level. Still, a population of intron-dense genes with many splice forms remains sheltered from eccDNA formation. We also find that the total number of eccDNA molecules does not increase as mice age, unlike other types of mutations. Our data reveal a link between eccDNA formation and transcript level that may drive gene architecture in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3340bae1e6fb34018ca87e367952455af8907d" target='_blank'>
              The CRL7FBXW8 Complex Controls the Mammary Stem Cell Compartment Through Regulation of NUMB Levels
              </a>
            </td>
          <td>
            Simone Sabbioni, M. G. Filippone, Letizia Amadori, S. Confalonieri, R. Bonfanti, Stefano Capoano, I. Colaluca, S. Freddi, G. Bertalot, Giovanni Fagá, Elisa Zagarrí, Mario Varasi, R. H. Gunby, C. Mercurio, S. Pece, P. P. Di Fiore, D. Tosoni
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa0c56c50991697be5ea164c0309587220e3483" target='_blank'>
              Beyond Telomeres: Unveiling the Extratelomeric Functions of TERT in B-Cell Malignancies
              </a>
            </td>
          <td>
            S. Giunco, M. R. Petrara, Stefano Indraccolo, Vincenzo Ciminale, Anita De Rossi
          </td>
          <td>2025-03-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9782e1bf6046bb6e08da1b33f9dcb44ab84e474" target='_blank'>
              A pivotal contribution of proteostasis failure and mitochondrial dysfunction to chromosomal instability-induced microcephaly
              </a>
            </td>
          <td>
            Amanda González-Blanco, Adrián Acuña-Higaki, David Boettger, Marco Milán
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acfe5ecebe48c685d0ade29fb13cedf2e576caf" target='_blank'>
              Frequency and impact of a genome-wide homologous recombination deficiency signature (HRDsig+) on the genomic landscape of clinically advanced urothelial bladder carcinoma (CAUBC).
              </a>
            </td>
          <td>
            Michael F. Basin, Andrea Necchi, Roger Li, Philippe E. Spiess, A. Kamat, P. Grivas, Shilpa Gupta, Neeraj Agarwal, Liang Cheng, D. Pavlick, E. Sokol, Ryon P Graf, Douglas I. Lin, J. Ross, A. Schrock, J. Quintanilha, Gerald Li, J. Jacob, A. Basnet, G. Bratslavsky
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Vault particles (VPs) are highly conserved large ribonucleoprotein complexes found exclusively in eukaryotes. They play critical roles in various cellular processes, but their involvement in cancer progression and multidrug resistance (MDR) is the most extensively studied. VPs are composed of the major vault protein (MVP), vault RNAs (vtRNAs), vault poly (ADP-ribose) polymerase, and telomerase-associated protein-1. These components are involved in the regulation of signaling pathways that affect tumor survival, proliferation, and metastasis. MVP has been associated with aggressive tumor phenotypes, while vtRNAs modulate cell proliferation, apoptosis, and autophagy. VPs also contribute to MDR by sequestering chemotherapeutic agents, altering their accumulation in the nucleus, and regulating lysosomal dynamics. Furthermore, small vault RNA-derived fragments participate in gene silencing and intercellular communication, reinforcing the role of precursors of vtRNAs in cancer development. Beyond their biological roles, VPs present a promising platform for drug delivery, due to their unique ability to encapsulate a wide range of biomolecules and therapeutic agents, followed by controlled release. This review compiles data from PubMed and Scopus, with a literature search conducted up until December 2024, highlighting current knowledge regarding VPs and their crucial involvement in cancer-related mechanisms and their applications in overcoming cancer drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738816da7a54860cbc1fb1817cc3a1dc28c1ea21" target='_blank'>
              Vault Particles in Cancer Progression, Multidrug Resistance, and Drug Delivery: Current Insights and Future Applications
              </a>
            </td>
          <td>
            Alexandros Maniatis, Dimitra Rizopoulou, Athanasios-Nasir Shaukat, K. Grafanaki, Vassiliki Stamatopoulou, Constantinos Stathopoulos
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The RAD51-BRCA2 interaction is central to DNA repair through homologous recombination. Emerging evidence indicates RAD51 overexpression and its correlation with chemoresistance in various cancers, suggesting RAD51-BRCA2 inhibition as a compelling avenue for intervention. We previously showed that combining olaparib (a PARP inhibitor (PARPi)) with RS-35d (a BRCA2-RAD51 inhibitor) was efficient in killing pancreatic ductal adenocarcinoma (PDAC) cells. However, RS-35d impaired cell viability even when administered alone, suggesting potential off-target effects. Here, through multiple, integrated orthogonal biological approaches in different 2D and 3D PDAC cultures, we characterised RS-35d enantiomers, in terms of mode of action and single contributions. By differentially inhibiting both RAD51-BRCA2 interaction and sensor kinases ATM, ATR and DNA-PK, RS-35d enantiomers exhibit a ‘within-pathway synthetic lethality’ profile. To the best of our knowledge, this is the first reported proof-of-concept single small molecule capable of demonstrating this built-in synergism. In addition, RS-35d effect on BRCA2-mutated, olaparib-resistant PDAC cells suggests that this compound may be effective as an anticancer agent possibly capable of overcoming PARPi resistance. Our results demonstrate the potential of synthetic lethality, with its diversified applications, to propose new and concrete opportunities to effectively kill cancer cells while limiting side effects and potentially overcoming emerging drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d347103c381bd6fd50aa622f9e2cee5595e830" target='_blank'>
              Investigating synthetic lethality and PARP inhibitor resistance in pancreatic cancer through enantiomer differential activity
              </a>
            </td>
          <td>
            Mirco Masi, Laura Poppi, Viola Previtali, Shannon R. Nelson, Kieran Wynne, Giulia Varignani, Federico Falchi, Marina Veronesi, E. Albanesi, Daniele Tedesco, Francesca De Franco, Andrea Ciamarone, Samuel H. Myers, José Antonio Ortega, Greta Bagnolini, Giovanni Ferrandi, Fulvia Farabegoli, Nicola Tirelli, Giuseppina Di Stefano, G. Oliviero, Naomi Walsh, M. Roberti, S. Girotto, Andrea Cavalli
          </td>
          <td>2025-03-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chromosomes are spatially organized and functionally folded into a specific macro-structure in the nucleus. Recently, we and others created haploid cells with chromosome fusions. However, there is still lack of an effective strategy for precisely investigating how the genome copes with fusions. Here, we developed a down-sampling method to convert the populational Hi-C dataset into single cell-like Khimaira Matrix (K-matrix). K-matrix preserves not only the most prominent functional genomic features but also cell-to-cell variations. K-matrix-originated genome 3D models display spatial approach of fused chromosomes and minor global structure alterations. Combined with a layered positional decomposition analysis, our models indicate slight re-adjustment of chromosome distributions accordingly with an increasing tendency following more fusions involved. Nevertheless, the radial distribution of the A/B compartment is not affected dramatically. By contrast, natural populations harboring Rb fusions display significant alterations of chromosome radial location. Overall, K-matrix-originated models enable visualization of chromosomal reorganization with high resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Developing accurate, clinically convenient, and non-invasive methods for early cancer detection, monitoring, and prognosis assessment is essential for improving patient survival rates, enhancing quality of life, and reducing the socioeconomic burden associated with cancer. This goal is critical in precision oncology. Genetic and epigenetic alterations in circulating cell-free DNA (cfDNA) have emerged as transformative tools for advancing early cancer detection, monitoring, and improving patient outcomes. Among these, 5-hydroxymethylcytosine (5hmC) modifications in circulating cfDNA stand out as promising epigenetic markers, offering insights into cancer initiation, progression, metastasis, and prognosis across various cancer types, such as lung cancer, colorectal cancer, and hepatocellular carcinoma. This review comprehensively explores the biology and sequencing methodologies of 5hmC, emphasizing their potential in cancer screening, diagnosis, treatment monitoring, and prognostic assessment. It highlights recent advancements in cfDNA-derived 5hmC signatures’ applications, addressing their strengths and limitations in the context of clinical translation. Furthermore, this review outlines key challenges and future directions for integrating 5hmC modifications in cfDNA into routine clinical practice, facilitating personalized and non-invasive cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9511d11d156015850cbe89f91beec11c4c2a8ba2" target='_blank'>
              5-Hydroxymethylcytosine modifications in circulating cell-free DNA: frontiers of cancer detection, monitoring, and prognostic evaluation
              </a>
            </td>
          <td>
            Danjun Song, Zhou Zhang, Jiaping Zheng, Wei Zhang, Jiabin Cai
          </td>
          <td>2025-03-07</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Inflammation is a multifaceted defense mechanism of the immune system against infection. Chronic inflammation is intricately linked to all stages of tumorigenesis and is therefore associated with an elevated risk of developing serious cancers. Epigenetic mechanisms have the capacity to trigger inflammation as well as facilitate tumor development and transformation within an inflammatory context. They achieve this by dynamically modulating the expression of both pro—inflammatory and anti—inflammatory cytokines, which in turn sustains chronic inflammation. The aberrant epigenetic landscape reconfigures the transcriptional programs of inflammatory and oncogenic genes. This reconfiguration is pivotal in dictating the biological functions of both tumor cells and immune cells. Aberrant histone H3 lysine 27 site (H3K27) methylation has been shown to be involved in biological behaviors such as inflammation development, tumor progression, and immune response. The establishment and maintenance of this repressive epigenetic mark is dependent on the involvement of the responsible histone modifying enzymes enhancer of zeste homologue 2 (EZH2), jumonji domain containing 3 (JMJD3) and ubiquitously transcribed tetratricopeptide repeat gene X (UTX) as well as multiple cofactors. In addition, specific pharmacological agents have been shown to modulate H3K27 methylation levels, thereby modulating inflammation and carcinogenesis. This review comprehensively summarises the current characteristics and clinical significance of epigenetic regulation of H3K27 methylation in the context of inflammatory response and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a333da9e2131a9a75deaa130f4cc548713bf46a8" target='_blank'>
              Regulation of histone H3K27 methylation in inflammation and cancer
              </a>
            </td>
          <td>
            Jing Ma, Yalin Zhang, Jingyuan Li, Y. Dang, Dan Hu
          </td>
          <td>2025-03-05</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background A blood-based approach to monitor metastasis and recurrence of esophageal squamous cell carcinoma (ESCC) remains undeveloped. This study aimed to establish a dependable model utilizing cfDNA 5-hydroxymethylcytosines (5hmC) signatures to detect these conditions in ESCC. Methods The 5hmC-Seal technique was employed to generate comprehensive 5hmC profiles from the plasma cell-free DNA (cfDNA) of 122 ESCC patients, classified into 72 with metastasis, 50 without metastasis, 30 with recurrence, and 92 without recurrence. Initial steps involved identifying distinct hydroxymethylation signatures linked to metastasis and recurrence. Machine learning algorithms were then utilized to construct predictive models. Results The study confirmed that 5hmC-based markers are predictive of metastasis and recurrence among ESCC patients. The analysis of 14 5hmC biomarkers revealed a sensitivity of 88.90% and a specificity of 84.00% (AUC = 0.922) in differentiating patients with ESCC metastasis from those without in the validation cohort. Similarly, 11 5hmC biomarkers showed a sensitivity of 93.30% and a specificity of 89.10% (AUC = 0.936) in identifying recurrent versus non-recurrent ESCC cases. Additionally, a wp-score for metastasis and recurrence, derived from the 5hmC marker, prognosticated patient outcomes. Conclusions The findings indicate that 5hmC markers from cfDNA serve as effective epigenetic indicators for the non-invasive detection of ESCC metastasis and recurrence. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03747-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5308006ac6e8e81a3d1d782decf9b4d6665fd13f" target='_blank'>
              cfDNA hydroxymethylcytosine profiling for detection metastasis and recurrence of Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Subinuer Kuerban, Hangyu Chen, Long Chen, Lei Zhang, Xuehui Li, Baixin Zhen, Hong Xiao, Yingzhu Chen, Haitao Zhou, Zhen Liang, Guobing Xu, Yicun Tao, Jian Lin, X. Kang
          </td>
          <td>2025-03-15</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b51d4673aaea8a53878437c3a918ca1f1fbfbf" target='_blank'>
              Oncogenic KRASG12D Transfer from Platelet-like Particles Enhances Proliferation and Survival in Non-Small Cell Lung Cancer Cells
              </a>
            </td>
          <td>
            J. Ceron-Hernandez, Gonzalo Martínez-Navajas, Jose Manuel Sanchez-Manas, María Pilar Molina, Jiajun Xie, Inés Aznar-Peralta, Abel Garcia-Diaz, Sonia Perales, Carolina Torres, M. J. Serrano, Pedro J. Real
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7749502012ae50efd549b8d9691628c7f5bce04c" target='_blank'>
              Epigenetics of Human Telomeres.
              </a>
            </td>
          <td>
            Nicole Bettin, Mélina Vaurs, Anabelle Decottignies
          </td>
          <td>2025-03-17</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The specialised structure of the centromere is critical for effective chromosome segregation, but its repetitive nature makes it vulnerable to rearrangements. Centromere fragility can drive tumorigenesis, but protective mechanisms preventing fragility are still not fully understood. The PBAF chromatin remodelling complex is frequently misregulated in cancer, but its role in cancer is incompletely characterized. Here, we identify PBAF as a protector of centromere and pericentromere structure with profound consequences for genome stability. A conserved feature of isogenic cell lines lacking PBRM1, a subunit of PBAF, is compromised centromere and pericentromere integrity. PBAF is present at these regions, and binding patterns of PBAF and H3K9 methylation change when PBRM1 is absent. PBRM1 loss creates a dependence on the spindle assembly checkpoint, which represents a therapeutic vulnerability. Importantly, we find that even in the absence of any perturbations, PBRM1 loss leads to centromere fragility, thus identifying a key player in centromere protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4067eaba6eda6efd31e27843f706f972ee2e6e" target='_blank'>
              PBRM1 directs PBAF to pericentromeres and protects centromere integrity
              </a>
            </td>
          <td>
            Karen A. Lane, Alison Harrod, Lillian Wu, T. I. Roumeliotis, Hugang Feng, Shane Foo, Katheryn A. G. Begg, Federica Schiavoni, Noa Amin, Frank T Zenke, Alan A. Melcher, J. Choudhary, Jessica A Downs
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Dysbiosis of intratumoral fungal and bacterial communities is associated with poor prognosis in various cancers. However, the mycobiome characteristics in laryngeal squamous cell carcinoma (LSCC) and its correlation with recurrence have not yet been investigated. The mycobiome in 80 LSCC samples was characterized using internal transcribed spacer sequencing, encompassing both tumor tissues and their matched para-cancerous tissues. The intratumoral bacteriome was further identified using 16S rRNA sequencing. These two microbial communities were analyzed using bioinformatics and statistical methods to determine its potential correlation with LSCC recurrence. The fungal alpha diversity in tumors was higher compared with that in para-cancerous tissues (P < 0.001). A significant difference in the overall fungal community patterns between tumor tissues and para-cancerous tissues was observed based on Bray-Curtis dissimilarity (P < 0.001). The presence of Alloprevotella, Porphyromonas, Candida, and Fusarium in tumors exhibited a correlation with alcohol consumption. The relative abundance of Penicillium, Exophiala, and Aspergillus in the mycobiome, as well as that of Alloprevotella, Porphyromonas, and Peptostreptococcus in the bacteriome significantly increased the risk of LSCC recurrence (P < 0.05). These six microorganisms can combine to form a microbial complex, which may independently contribute to recurrence risk in patients with LSCC when enriched within the tumor (hazard ratio = 6.844, P < 0.01). Intratumoral fungi and bacteria can be valuable indicators for assessing recurrence in patients with LSCC, indicating their potential as valuable targets for therapeutic intervention. IMPORTANCE Our results revealed that dysbiosis of intratumoral microbiota, including increased fungal community diversity and overgrowth of several fungal or bacterial organisms, is substantially linked to the recurrence of LSCC. Drinking habits might alter the laryngeal microbiota to influence the recurrence of LSCC. We also explored a method to potentially predict the recurrence of LSCC from a novel perspective. These findings could offer insights into the etiology of LSCC and pave way to prevent and treat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc011a9eb62b9ac278bb5cc6ef5e19d7054ea20" target='_blank'>
              Complex of intratumoral mycobiome and bacteriome predicts the recurrence of laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xinhui Mao, Hui-ying Huang, Limin Zhao, Feiran Li, Zhenwei Wang, Xiaohui Yuan, Hui-Ching Lau, Chi-Yao Hsueh, Ming Zhang
          </td>
          <td>2025-02-21</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tetraploidisation plays a crucial role in evolution, development, stress adaptation, and disease, but its beneficial or pathological effects in different tissues remain unclear. This study aims to compare physiological and unphysiological tetraploidy in eight steps: 1) mechanisms of diploidy-to-tetraploidy transition, 2) induction and elimination of unphysiological tetraploidy, 3) tetraploid cell characteristics, 4) stress-induced unphysiological tetraploidy, 5) comparison of physiological vs. unphysiological tetraploidy, 6) consequences of unphysiological stress-induced tetraploidy, 7) nutritional or pharmacological prevention strategies of tetraploidisation, and 8) knowledge gaps and future perspectives. Unphysiological tetraploidy is an adaptive stress response at a given threshold, often involving mitotic slippage. If tetraploid cells evade elimination through apoptosis or immune surveillance, they may re-enter the cell cycle, causing genetic instability, micronuclei formation, aneuploidy, modification of the epigenome and the development of diseases. The potential contributions of unphysiological tetraploidy to neurodegenerative, cardiovascular and diabetes related diseases are summarized in schematic figures and contrasted with its role in cancer development. The mechanisms responsible for the transition from physiological to unphysiological tetraploidy and the tolerance to tetraploidisation in unphysiological tetraploidy are not fully understood. Understanding these mechanisms is of critical importance to allow the development of targeted nutritional and pharmacological prevention strategies and therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb4b60e7683b10ceeabf0dcb4384ffb5dd6b80ee" target='_blank'>
              Tetraploidy in normal tissues and diseases: mechanisms and consequences
              </a>
            </td>
          <td>
            Micheline Kirsch-Volders, Miroslav Mišík, Michael Fenech
          </td>
          <td>2025-03-21</td>
          <td>Chromosoma</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic defects caused by truncating mutations or deletions in the Retinoblastoma tumor suppressor gene (RB1) are frequently observed in many cancer types leading to dysregulation of the RB pathway. Here, we propose an integrative proteogenomic approach that predicts cancers with dysregulation in the RB pathway. A subset of these cancers, which we term as “RBness,” lack RB1 genomic defects and yet phenocopy the transcriptional profile of RB1-defective cancers. We report RBness as a pan-cancer phenomenon, associated with patient outcome and chemotherapy response in multiple cancer types, and predictive of CDK4/6 inhibitor response in estrogen-positive breast cancer. Using RNA interference and a CRISPR-Cas9 screen in isogenic models, we find that RBness cancers also phenocopy synthetic lethal vulnerabilities of cells with RB1 genomic defects. In summary, our findings suggest that dysregulation of the RB pathway in cancers lacking RB1 genomic defects provides a molecular rationale for how these cancers could be treated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b769a127908e214629b60c835bc205b258e11d" target='_blank'>
              Proteogenomic discovery of RB1-defective phenocopy in cancer predicts disease outcome, response to treatment, and therapeutic targets
              </a>
            </td>
          <td>
            Jacopo Iacovacci, R. Brough, Fatemeh Ahmadi Moughari, J. Alexander, H. Kemp, Andrew N J Tutt, R. Natrajan, C. J. Lord, Syed Haider
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Acute myelocytic leukemia (AML) is a common hematological malignancy in adults. Although several risk stratifications based on cytogenetic and molecular abnormalities are available to guide the indications for allogeneic hematopoietic cell transplantation (allo-HCT), determining optimal treatment strategies for AML remains challenging. In this study, using transcriptome datasets, we investigated the association between event-free survival (EFS) in intensively treated AML patients and the aberrant expression of endogenous viral element (EVE)-derived open reading frames (ORFs), which have been reported to be associated with the pathophysiology of various malignancies and have the potential to serve as neoantigens in specific cancers. Results The expression levels of human endogenous retrovirus family K9 (HERVK9) ORFs were associated with EFS, independent of conventional risk stratification. Furthermore, AML cells with higher levels of HERVK9 expression exhibited enhanced antigen processing and presentation, along with increased expression of genes associated with adaptive immune responses and apoptosis, indicating that aberrant HERVK9 expression may initiate an anti-neoplastic immune response via increased antigen presentation. Conclusions HERVK9 expression may have serve as a crucial prognostic indicator that could aid in determining the indications for upfront allo-HCT in AML patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12977-025-00661-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5f29628a7105d4d628b72c6ef996a0f86eb8dca" target='_blank'>
              Aberrant expression of human endogenous retrovirus K9-derived elements is associated with better clinical outcome of acute myelocytic leukemia
              </a>
            </td>
          <td>
            Ryo Yanagiya, So Nakagawa, M. Onizuka, Ai Kotani
          </td>
          <td>2025-04-01</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="R loops are three-stranded nucleic acid structures that form naturally in cells under various conditions, mainly as intermediates during replication or as by-products during transcription. R loops are involved in the regulation of many important cellular processes, including replication, transcription, centromere stabilization, protection of chromosome ends, or control of telomere length. Unscheduled R loops are linked to many diseases, including cancer, neurodegenerative, or inflammatory disorders. The list of cancer diseases linked to excessive R loop accumulation is growing rapidly. There is currently much debate about the understanding of abnormal R loop formation and its impact on genome instability and cancer development. In this review, we briefly describe the nature of R loops, their formation under physiological and pathological conditions, and the proteins involved in the regulation of R loops. In addition, we emphasize the possible role of the human ribonuclease Dicer, a multi-tasking protein mostly known for its important role in microRNA biogenesis, in the regulation of R loops. We also discuss the involvement of R loops in cancer development and their potential use as diagnostic biomarkers. Knowledge of the molecular mechanisms underlying R loop dysregulation may significantly improve our understanding of cancer biology and provide new directions for research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b3552d408d3cbf3b7a331a1abe18a81efb7900" target='_blank'>
              Possible role of human ribonuclease dicer in the regulation of R loops.
              </a>
            </td>
          <td>
            Klaudia Wójcik, Paulina Krzeminska, A. Kurzyńska-Kokorniak
          </td>
          <td>2025-03-24</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Extracellular vesicles (EVs) play a key role in cell‐to‐cell communication and have drawn significant attention due to their potential clinical applications. However, much remains to be understood about the biology of EV‐associated DNA (EV‐DNA). EV‐DNA is actively released by both normal and malignant cells and consists of diverse fragments with varying structures. Because EV‐DNA spans the entire genome of cells from which it originates, it continues to be attractive as a biomarker for cancer diagnosis and monitoring. Further, EV‐DNA delivery can alter the function of recipient cells by interfering with cytoplasmic DNA sensor pathways. This review explores the biology and significance of EV‐DNA, including its topology and fragmentomics features, modality of association with EVs, packaging mechanisms, and potential functions. It also emphasizes the specificity of vesicular DNA in identifying genetic and epigenetic changes in cancer. Additionally, it delves into the impact of EV‐DNA on cellular behaviour and its potential use as a therapeutic target in cancer. The review discusses new insights into EV‐DNA biology and provides perspectives and alternatives to address the challenges and concerns for future EV‐DNA studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80fa234e6a41ee735b1a29ab26a8f1f47cfb63ad" target='_blank'>
              Unravelling the Significance of Extracellular Vesicle‐Associated DNA in Cancer Biology and Its Potential Clinical Applications
              </a>
            </td>
          <td>
            J. Ghanam, Kristína Lichá, V. K. Chetty, Ommolbanin Asad Pour, Dirk Reinhardt, Barbora Tamášová, Peter Hoyer, Jan Lötvall, B. K. Thakur
          </td>
          <td>2025-03-01</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromobox (CBX) proteins play a crucial role in regulating epigenetic processes. They are extensively involved in various biological processes, including embryonic development, stem cell maintenance, cell proliferation and apoptosis control. The disruption and malfunction of CBXs in cancer typically results in the interference or abnormal activation of developmental pathways, which facilitate the onset, growth, and advancement of cancer. This review initially introduces the physiological properties and functions of the CBXs. Subsequently, it examines the involvement of CBXs in different cancer types. Cancer hallmarks driven by CBXs are mediated through multiple mechanisms, including changes in gene expression patterns, epigenetic dysregulation of chromatin control, disruption of intracellular signaling and alterations in cell metabolism. The study also highlights novel potential anticancer therapeutics targeting CBXs in cancer. In this review we provide novel perspectives and a solid foundation for future investigations on CBXs as promising therapeutic targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/002145f5e64f86b17aaad5541d3b6d5e23778a2a" target='_blank'>
              Unravelling the impact of the chromobox proteins in human cancers
              </a>
            </td>
          <td>
            Shabana Noreen, Nicla Simonelli, R. Benedetti, V. Carafa, Michele Grieco, C. Ambrosino, C. Dell'Aversana, A. Nebbioso, Mariarosaria Conte, Nunzio Del Gaudio, Lucia Altucci
          </td>
          <td>2025-04-02</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA methylation at cytosine bases (5-methylcytosine, 5mC) is a heritable epigenetic mark regulating gene expression. While enzymes that metabolize 5mC are well-characterized, endogenous signaling molecules that regulate DNA methylation machinery have not been described. We report that physiological nitric oxide (NO) concentrations reversibly inhibit the DNA demethylases TET and ALKBH2 by binding to the mononuclear non-heme iron atom forming a dinitrosyliron complex (DNIC) and preventing cosubstrates from binding. In cancer cells treated with exogenous NO, or endogenously synthesizing NO, 5mC and 5-hydroxymethylcytosine (5hmC) increase, with no changes in DNA methyltransferase activity. 5mC is also significantly increased in NO-producing patient-derived xenograft tumors from mice. Genome-wide methylome analysis of cells chronically treated with NO (10 days) shows enrichment of 5mC and 5hmC at gene-regulatory loci, correlating with altered expression of NO-regulated tumor-associated genes. Regulation of DNA methylation is distinctly different from canonical NO signaling and represents a unique epigenetic role for NO.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028d5d550a57838f21d155635ad9782c7d043afb" target='_blank'>
              Nitric oxide inhibits ten-eleven translocation DNA demethylases to regulate 5mC and 5hmC across the genome
              </a>
            </td>
          <td>
            Marianne B. Palczewski, H. Kuschman, Brian Hoffman, Venkatesan Kathiresan, Hao Yang, Sharon Glynn, David L. Wilson, Eric T. Kool, William R. Montfort, Jenny Chang, Aydolun Petenkaya, Constantinos Chronis, Thomas R Cundari, Sushma Sappa, Kabirul Islam, Daniel W. McVicar, Yu Fan, Qing-rong Chen, D. Meerzaman, Michael Sierk, Douglas D. Thomas
          </td>
          <td>2025-02-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The canonical DNA damage response (DDR) maintains genome stability, involving DNA synthesis/cell cycle arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously discovered and characterized a noncanonical cellular response that is specific to nonblocking replication stress, i.e., low-level stress (LoL-DDR), in primary cells. Although this response generates replication stress-induced reactive oxygen species (RIR), it triggers a program that prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a precise threshold triggers the canonical DDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: 1-RIR are not detoxified under high-level stress conditions, resulting in a continuous increase in the dose-response curve of RIR production; 2-RIR are not produced by NADPH oxidases; and 3-replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” involving replication-stress and ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfa2fab8a78a43f5b506788fc0cb6d9a375f31e" target='_blank'>
              Dysregulation of replication stress-induced ROS in transformed cell lines: a vicious circle at cancer initiation
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8b9305ad5bdf4a40999bdb1d8deb688207b86c2" target='_blank'>
              Utilization of DNA Replication factor MCM2 by Cancer.
              </a>
            </td>
          <td>
            Yuwei Yuan, Tian Zeng, Anbo Gao, Yang Guan, Qun-feng Zhang, Yukun Li, Hui Tan, Juan Zou
          </td>
          <td>2025-02-13</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Oncogenic codon V600E mutations of the BRAF gene affect 10-15% of colorectal cancers, resulting in activation of the MAPK/ERK signaling pathway and increased cell proliferation and survival. BRAF-mutated colorectal tumors are often microsatellite unstable and characterized by high DNA methylation levels. However, the mechanistic link between BRAF mutations and hypermethylation remains controversial. Understanding this link, particularly in microsatellite stable tumors is of great interest as these often show poor survival. We characterized the metabolomic, epigenetic and transcriptomic patterns of altogether 39 microsatellite stable BRAF-mutated colorectal cancers. Metabolomic analysis of tumor tissue showed low levels of vitamin C and its metabolites in BRAF-mutated tumors. Gene expression analysis indicated dysregulation of vitamin C antioxidant activity in these lesions. As vitamin C is an important cofactor for the activity of TET DNA demethylase enzymes, low vitamin C levels could directly contribute to the high methylation levels in these tumors by decreasing enzymatic TET activity. Vitamin C transporter gene SLC23A1 expression, as well as vitamin C metabolite levels, were inversely correlated with DNA methylation levels. This work proposes a new mechanistic link between BRAF mutations and hypermethylation, inspiring further work on the role of vitamin C in the genesis of BRAF-mutated colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62245272a3e5378cb87bd7150e412b406fb1921b" target='_blank'>
              Comprehensive metabolomic and epigenomic characterization of microsatellite stable BRAF-mutated colorectal cancer.
              </a>
            </td>
          <td>
            Aurora Taira, M. Aavikko, Riku Katainen, Eevi Kaasinen, Niko Välimäki, Janne Ravantti, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, J. Böhm, J. Bramsen, C. Andersen, Kimmo Palin, K. Rajamäki, L. Aaltonen
          </td>
          <td>2025-03-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Introduction Pancreatic adenocarcinoma (PAAD) is characterized by a profoundly immunosuppressive tumor microenvironment (TME) that limits the efficacy of immunotherapy. Emerging evidence suggests that tumor-specific metabolic reprogramming may drive disease progression and shape the immune landscape in PAAD. Methods We integrated multi-omics data from TCGA, GEO, and ICGC to identify key metabolism-related genes (MRGs) that influence immune cell infiltration, tumor progression, and patient survival. Based on nine pivotal MRGs (including ANLN, PKMYT1, and HMGA1), we developed and validated a novel metabolic-prognostic index (MPI). Functional enrichment analyses were conducted to elucidate the metabolic pathways associated with different MPI risk groups. In vitro experiments and drug sensitivity analyses were performed to confirm the oncogenic role of selected MRGs and to explore their therapeutic implications. Results The MPI effectively stratified patients into high- and low-risk groups. High-MPI scores correlated with poor overall survival, elevated tumor mutation burden (TMB), and an immunosuppressive TME, evidenced by reduced CD8⁺ T-cell infiltration and increased expression of immune checkpoints (PD-L1, TGF-β). Functional enrichment revealed glycolysis and folate biosynthesis as dominant pathways in high-MPI groups, whereas fatty acid metabolism prevailed in low-MPI groups. Experimental validation underscored the role of ANLN in promoting epithelial-mesenchymal transition (EMT) and immune evasion via NF-κB signaling. ANLN knockdown significantly reduced glycolytic activity, tumor cell migration, and immune evasion. Drug sensitivity analyses indicated resistance to gemcitabine but sensitivity to afatinib in high-MPI patients. Although TIDE analysis predicted immune checkpoint inhibitor (ICI) resistance in high-MPI tumors, a subset of patients showed favorable responses to anti-PD-L1 therapy. Discussion These findings provide a comprehensive framework for understanding how metabolic reprogramming shapes PAAD’s immunosuppressive TME and affects treatment outcomes. By accurately stratifying patients, the MPI serves as a promising tool to guide therapeutic decisions, including targeted therapy selection and immunotherapy prediction, ultimately offering potential for more personalized management of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42608b06fe89e88fa2b364a076f0a8a5c2fa634d" target='_blank'>
              Metabolic reprogramming shapes the immune microenvironment in pancreatic adenocarcinoma: prognostic implications and therapeutic targets
              </a>
            </td>
          <td>
            Weihua Song, Yabin Yu, Siqi Wang, Zhengyi Cui, Qiusi Zhu, Wangrui Liu, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a25e85628074c1ce8e497277968236f1e14cdb" target='_blank'>
              Spo11: from topoisomerase VI to meiotic recombination initiator.
              </a>
            </td>
          <td>
            J. A. Harper, George G B Brown, Matthew J. Neale
          </td>
          <td>2025-04-02</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive malignancy characterized by rapid proliferation and high metastatic potential. It is characterized by universal inactivation of and RB1, overexpression of the MYC family and dysregulation of multiple oncogenic signaling pathways. Among different patients, SCLCs are similar at the genetic level but exhibit significant heterogeneity at the molecular level. The classification of SCLC has evolved from a simple neuroendocrine (NE)/non-neuroendocrine (non-NE) classification system to a transcription factor-based molecular subtype system; lineage plasticity adds further complexity and poses challenges for therapeutic development. While SCLC is initially sensitive to platinum-based chemotherapy, resistance develops rapidly, leading to a dismal prognosis. Various antibodies, including PD-1/PD-L1 inhibitors and antibody‒drug conjugates, have been introduced into clinical practice or are being evaluated in clinical trials. However, their therapeutic benefits for SCLC patients remain limited. This review summarizes SCLC carcinogenic mechanisms, tumor heterogeneity, and the immune microenvironment of SCLC, with a focus on recent advances in metastasis and resistance mechanisms. Additionally, the corresponding clinical progress in tackling these challenges is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98544ce43f27a0684b668c8dba29b6f68626bac0" target='_blank'>
              Current and future therapies for small cell lung carcinoma
              </a>
            </td>
          <td>
            X. Zhai, Zhengkun Zhang, Yuxin Chen, Yanmou Wu, Cheng Zhen, Yu Liu, Yiyun Lin, Chong Chen
          </td>
          <td>2025-04-01</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d32b1ca7eec503a083b45ef28636d90a045aa" target='_blank'>
              Fork-barrier-independent roles of topoisomerase I in the ribosomal DNA.
              </a>
            </td>
          <td>
            Temistocles Molinar, D. Sultanov, Hannah Klein, Andreas Hochwagen
          </td>
          <td>2025-03-22</td>
          <td>Genetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d77495b125c5aaab4dcf465d06c71c35fe30e9b" target='_blank'>
              SNHG10: A Novel Long Non-coding RNA with Multifaceted Roles in Human Cancers.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Lihan Chen, Chuyuan Liao, Ting Gong, Tongtong Li, Haoran Liu, Yunfei Pan, Pengbo Zhang, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2025-03-05</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6803d57b0e8461fd7e3f611a51d0f96a34cd2cb" target='_blank'>
              Abstract B020: Early detection of colorectal cancer using micronuclei DNA
              </a>
            </td>
          <td>
            Xingyun Yao, Haobo Sun, Yurong Jiao, Chengcheng Liu, Xiangxing Kong, Honghao Liang, Fei Meng, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a6d24b89c2f40e1c162089c5eebb1560efb4a3e" target='_blank'>
              Radioprotection of healthy tissue via nanoparticle-delivered mRNA encoding for a damage-suppressor protein found in tardigrades.
              </a>
            </td>
          <td>
            Ameya R. Kirtane, Jianling Bi, N. U. Rajesh, Chaoyang Tang, Miguel Jimenez, Emily Witt, Megan K. McGovern, Arielle B. Cafi, Sam Hatfield, Lauren L Rosenstock, Sarah L Becker, Nicole Machado, Veena Venkatachalam, Dylan Freitas, Xisha Huang, Alvin Chan, Aaron Lopes, Hyunjoon Kim, Nayoon Kim, Joy E. Collins, Michelle E. Howard, Srija Manchkanti, Theodore S Hong, James D. Byrne, G. Traverso
          </td>
          <td>2025-02-26</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>45</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>