<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-28 06:32:23 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>57</td>
          <td>132</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>102</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>290</td>
          <td>132</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>249</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, J. Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, S. Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4921d9b380eadcd80f19284f6396e453cfba83e" target='_blank'>
              Exploring the potential of extrachromosomal DNA as a novel oncogenic driver.
              </a>
            </td>
          <td>
            Huanbo Zhu, Longtao Huangfu, Junbing Chen, Jiafu Ji, Xiaofang Xing
          </td>
          <td>2024-09-27</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution.
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Micronuclei (MN) are a commonly used marker of chromosome instability that form when missegregated chromatin recruits its own nuclear envelope (NE) after mitosis. MN frequently rupture, which results in genome instability, upregulation of metastatic genes, and increased immune signaling. MN rupture is linked to NE defects, but the cause of these defects is poorly understood. Previous work from our lab found that chromosome identity correlates with rupture timing for small MN, i.e. MN containing a short chromosome, with more euchromatic chromosomes forming more stable MN with fewer nuclear lamina gaps. Here we demonstrate that histone methylation promotes rupture and nuclear lamina defects in small MN. This correlates with increased MN size, and we go on to find that all MN have a constitutive nuclear export defect that drives MN growth and nuclear lamina gap expansion, making the MN susceptible to rupture. We demonstrate that these export defects arise from decreased RCC1 levels in MN and that additional loss of RCC1 caused by low histone methylation in small euchromatic MN results in additional import defects that suppress nuclear lamina gaps and MN rupture. Through analysis of mutational signatures associated with early and late rupturing chromosomes in the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset, we identify an enrichment of APOBEC and DNA polymerase E hypermutation signatures in chromothripsis events on early and mid rupturing chromosomes, respectively, suggesting that MN rupture timing could determine the landscape of structural variation in chromothripsis. Our study defines a new model of MN rupture where increased MN growth, caused by defects in protein export, drives gaps in nuclear lamina organization that make the MN susceptible to membrane rupture with long-lasting effects on genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6815113609c0299acce4dadc060265ccb48bb9ff" target='_blank'>
              RCC1 depletion drives protein transport defects and rupture in micronuclei
              </a>
            </td>
          <td>
            Molly G. Zych, Maya Contreras, M. Vashisth, Anna E Mammel, Gavin Ha, Emily M. Hatch
          </td>
          <td>2024-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal G. Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K. Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Copy number variations (CNVs) are structural genomic alterations that involve changes in the number of copies of specific DNA regions. These variations can include deletions, duplications, and more complex rearrangements, and play a critical role in cancer progression by amplifying oncogenes, deleting tumor suppressor genes, or altering other key genomic regions. Despite the importance of CNVs in cancer biology, there is a lack of comprehensive resources that aggregate CNV data across multiple cancer types, impeding the exploration of their role in different malignancies. The PanCNV-Explorer was developed to fill this gap by providing a pan-cancer resource that integrates CNV data for over 21 cancer types. Methods: The PanCNV-Explorer was developed, utilizing data from repositories such as COSMIC, DepMap, DGV, and dbVAR. Amplifications and deletions can be visualized, with detailed chromosome-specific plots highlighting CNV patterns. In addition, a genome browser has been implemented to display regions of interest for searching for genes of potential relevance. Results: The database contains over 5 million CNVs from 15,809 samples, spanning 21 primary cancer types. Pathogenic CNVs from ClinVar and DepMap were compared to benign data from DGV and dbVAR, allowing a comprehensive analysis across tissues. CNV patterns were visualized for individual cancers. We demonstrate with use cases, focusing on common oncogenes such as MYC and tumor suppressor genes such as TP53, the ability of the resource to search for cancer driver genes. For example, chromosome 12 in pancreatic cancer showed frequent amplifications in the region containing the KRAS oncogene. In another example, we show how driver gene candidates already described in certain cancer entities can also be found with high frequency in other entities. Conclusion: The PanCNV-Explorer provides a valuable resource for pan-cancer CNV analysis, facilitating the exploration of CNV-driven tumor heterogeneity. The integrated visualization tools and genome browser enable detailed exploration of genomic regions, enhancing cancer research. As a valuable and easy-to-use platform that allows researchers to quantify and compare CNVs across cancer types, the PanCNV-Explorer is an essential tool for advancing cancer genomics research. The main page is available at https://mtb.bioinf.med.uni-goettingen.de/pancnv-explorer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0740ab5d12642c500ab7be4dd721a73d9e32d198" target='_blank'>
              PanCNV Explorer: A pan-cancer database for copy number variations.
              </a>
            </td>
          <td>
            Kevin Kornrumpf, Jürgen Dönitz
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="As a major type of structural variants, tandem duplication plays a critical role in tumorigenesis by increasing oncogene dosage. Recent work has revealed that noncoding enhancers are also affected by duplications leading to the activation of oncogenes that are inside or outside of the duplicated regions. However, the prevalence of enhancer duplication and the identity of their target genes remains largely unknown in the cancer genome. Here, by analyzing whole-genome sequencing data in a non-gene-centric manner, we identify 881 duplication hotspots in 13 major cancer types, most of which do not contain protein-coding genes. We show that the hotspots are enriched with distal enhancer elements and are highly lineage-specific. We develop a HiChIP-based methodology that navigates enhancer-promoter contact maps to prioritize the target genes for the duplication hotspots harboring enhancer elements. The methodology identifies many novel enhancer duplication events activating oncogenes such as ESR1, FOXA1, GATA3, GATA6, TP63, and VEGFA, as well as potentially novel oncogenes such as GRHL2, IRF2BP2, and CREB3L1 In particular, we identify a duplication hotspot on Chromosome 10p15 harboring a cluster of enhancers, which skips over two genes, through a long-range chromatin interaction, to activate an oncogenic isoform of the NET1 gene to promote migration of gastric cancer cells. Focusing on tandem duplications, our study substantially extends the catalog of noncoding driver alterations in multiple cancer types, revealing attractive targets for functional characterization and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269520ff92f1d6c1a566c78a1f6e2344309fcdcb" target='_blank'>
              Chromatin interaction maps identify oncogenic targets of enhancer duplications in cancer.
              </a>
            </td>
          <td>
            Yueqiang Song, Fuyuan Li, Shangzi Wang, Yuntong Wang, Cong Lai, Lian Chen, Ning Jiang, Jin Li, Xingdong Chen, Swneke D. Bailey, Xiaoyang Zhang
          </td>
          <td>2024-10-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafb0eaf13b484b31a40158ac36bc0076c421a97" target='_blank'>
              Bivalent chromatin accommodates survivin and BRG1/SWI complex to activate DNA damage response in CD4+ cells
              </a>
            </td>
          <td>
            V. Chandrasekaran, K. Andersson, M. Erlandsson, Shuxiang Li, Torbjörn Nur Olsson, M. García-Bonete, E. Malmhäll-Bah, Pegah Johansson, G. Katona, M. Bokarewa
          </td>
          <td>2024-09-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a180cfc530d1d7fe2d98876dc6894d589798a6e8" target='_blank'>
              Mapping extrachromosomal DNA amplifications during cancer progression.
              </a>
            </td>
          <td>
            Hoon Kim, Soyeon Kim, Taylor E. Wade, Eunchae Yeo, A. Lipsa, A. Gołȩbiewska, Kevin C Johnson, Sepil An, Junyong Ko, Yoonjoo Nam, Hwa Yeon Lee, Seunghyun Kang, Heesuk Chung, S. Niclou, H. Moon, S. Paek, V. Bafna, J. Luebeck, R.G.W. Verhaak
          </td>
          <td>2024-10-14</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Harnessing DNA double-strand breaks (DSBs) is a powerful approach for gene editing, but it may provoke loss of heterozygosity (LOH), which predisposes to tumorigenesis. To interrogate this risk, we developed a two- color flow cytometry-based system (Flo-LOH), detecting LOH in ∼5% of cells following a DSB. After this initial increase, cells with LOH decrease due to a competitive disadvantage with parental cells, but if isolated, they stably propagate. Segmental loss from terminal deletions with de novo telomere addition and nonreciprocal translocations is observed as well as whole chromosome loss, especially following a centromeric DSB. LOH spans megabases distal from the DSB, but also frequently tens of megabases centromere-proximal. Inhibition of microhomology-mediated end joining massively increases LOH, which is synergistically increased with concomitant inhibition of canonical nonhomologous end joining. The capacity for large-scale LOH must therefore be considered when using DSB-based gene editing, especially in conjunction with end joining inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d1b33368b2fb7ae9c66926efe7d4fcbd491246" target='_blank'>
              Megabase-scale loss of heterozygosity provoked by CRISPR-Cas9 DNA double-strand breaks
              </a>
            </td>
          <td>
            Samantha B. Regan, Darpan Medhi, Travis B. White, Yi-Zhen Jiang, Su Jia, Qichen Deng, M. Jasin
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0970058b2af0e5ea082fb819477f1708b4c4db9a" target='_blank'>
              DNA demethylation triggers cell free DNA release in colorectal cancer cells
              </a>
            </td>
          <td>
            V. Pessei, M. Macagno, Elisa Mariella, Noemi Congiusta, V. Battaglieri, Paolo Battuello, Marco Viviani, Giulia Gionfriddo, S. Lamba, Annalisa Lorenzato, D. Oddo, Fariha Idrees, Alessandro Cavaliere, A. Bartolini, S. Guarrera, M. Linnebacher, Laura Monteonofrio, Luca Cardone, Michele Milella, A. Bertotti, Silvia Soddu, Elena Grassi, G. Crisafulli, A. Bardelli, L. Barault, F. Di Nicolantonio
          </td>
          <td>2024-10-09</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2+ tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One Sentence Summary Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6498a7e5198294cbede8c8c23e085245d62203" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="MYC deregulation occurs in the majority of multiple myeloma (MM) cases and is associated with progression and worse prognosis. Enhanced MYC expression occurs in about 70% of MM patients, but it is known to be driven by translocation or amplification events in only ~40% of myelomas. Here, we used CRISPR interference (CRISPRi) to uncover an epigenetic mechanism of MYC regulation whereby increased accessibility of a plasma cell-type specific enhancer leads to increased MYC expression. This native enhancer activity was not associated with enhancer hijacking events but led to specific binding of c-MAF, IRF4, and SPIB transcription factors that activated MYC expression in the absence of known genetic aberrations. In addition, focal amplification was another mechanism of activation of this enhancer in approximately 3.4% of MM patients. Together, these findings define an epigenetic mechanism of MYC deregulation in MM beyond known translocations or amplifications and point to the importance of non-coding regulatory elements and their associated transcription factor networks as drivers of MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89586e6b56717e928b88a8526a759effc353dc88" target='_blank'>
              Selective Enhancer Gain of Function Deregulates MYC Expression in Multiple Myeloma.
              </a>
            </td>
          <td>
            M. Rahmat, Kendell Clement, Jean-Baptiste Alberge, Romanos Sklavenitis-Pistofidis, R. Kodgule, C. Fulco, Daniel Heilpern-Mallory, Katarina Nilsson, David Dorfman, Jesse M Engreitz, G. Getz, Luca Pinello, R. Ryan, I. M. Ghobrial
          </td>
          <td>2024-09-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Satellite DNA makes up ∼11% of the mouse genome, predominantly located in centromeric and pericentric regions, which are crucial for chromosome segregation. While comprehensive assemblies of these regions have been established in the human genome, they are still lacking in the mouse genome. In this study, we used PacBio long-read sequencing, CUT&RUN sequencing, DNA methylation analysis, and RNA sequencing to generate genomic and epigenomic maps of these regions. We find that centromeric regions are primarily occupied by 120-mer Minor satellites, with other Minor Satellite length variants, 112-mers and 112-64-dimers, localized at centromere-pericentric junctions. Pericentromeric regions are mainly composed of homogeneous Major satellites, while pericentric-chromosomal junctions contain a higher density of divergent satellites. Additionally, the density of non-satellite repeats increases progressively from centromeres to pericentromeres, and further toward chromosomal arm junctions. We found that 120-mer Minor satellites in the core centromere are highly enriched with CENP-A, while the 112-mers and 112-64-dimers show lower CENP-A levels. Homogeneous Major satellites are more enriched with H3K9me3 heterochromatin, whereas divergent Major satellites are preferentially associated with H3K27me3. Furthermore, DNA methylation levels are lower in centromeres compared to pericentric regions. We also observed that only a small subset of satellites is transcribed into RNA, particularly regions exhibiting lower DNA methylation density. Our comprehensive assembly and characterization of the genomic and epigenomic landscape of mouse centromeric and pericentric regions have major implications for satellite biology and ongoing mouse telomere-to-telomere (T2T) assembly efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951d9f9f9b8e262cc6e6811c22c27c69e6bb984f" target='_blank'>
              Genomic and epigenomic maps of mouse centromeres and pericentromeres
              </a>
            </td>
          <td>
            Gitika Chaudhry, Jinguye Chen, Lucy Snipes, Smriti Bahl, Jenika Packiaraj, Xuan Lin, Jitendra Thakur
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract N6-Methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3 and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cb34b460873bf086e54bf45d997fe617e03818" target='_blank'>
              The three YTHDF paralogs and VIRMA are strong cross-histotype tumor driver candidates among m6A core genes
              </a>
            </td>
          <td>
            E. Destefanis, D. Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, Alessandro Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-10-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1079c92bf1b2279090cf5d7068dd405cec2a37a" target='_blank'>
              Single-cell multi-omics map of human fetal blood in Down syndrome
              </a>
            </td>
          <td>
            Andrew R. Marderstein, Marco De Zuani, Rebecca Moeller, Jon Bezney, Evin M Padhi, Shuo Wong, Tim H. H. Coorens, Yilin Xie, Haoliang Xue, Stephen B. Montgomery, A. Cvejic
          </td>
          <td>2024-09-25</td>
          <td>Nature</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Optimizing prevention and early detection of cancer requires understanding the number, types and timing of driver mutations. To quantify this, we exploited the elevated cancer incidence and mutation rates in germline BRCA1 and BRCA2 (gBRCA1/2) carriers. Using novel statistical models, we identify genomic deletions as the likely rate-limiting mutational processes, with 1-3 deletions required to initiate breast and ovarian tumors. gBRCA1/2-driven hereditary and sporadic tumors undergo convergent evolution to develop a similar set of driver deletions, and deletions explain the elevated cancer risk of gBRCA1/2-carriers. Orthogonal mutation timing analysis identifies deletions of chromosome 17 and 13q as early, recurrent events. Single-cell analyses confirmed deletion rate differences in gBRCA1/2 vs. non-carrier tumors as well as cells engineered to harbor gBRCA1/2. The centrality of deletion-associated chromosomal instability to tumorigenesis shapes interpretation of the somatic evolution of non-malignant tissue and guides strategies for precision prevention and early detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/142f79a57f8e5624ca9c9770400c8298f772b241" target='_blank'>
              Deletions Rate-Limit Breast and Ovarian Cancer Initiation
              </a>
            </td>
          <td>
            Kathleen E. Houlahan, Mahad Bihie, Julián Grandvallet Contreras, Daniel J. Fulop, Gonzalo Lopez, Hsin-Hsiung Huang, P. Van Loo, Christina Curtis, P. Boutros, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9a6a91b5f3ac052c57405c773236aad0d8d4b4" target='_blank'>
              Recent insights into the causes and consequences of chromosome mis-segregation.
              </a>
            </td>
          <td>
            Romain Devillers, Alexsandro dos Santos, Quentin Destombes, Mathieu Laplante, S. Elowe
          </td>
          <td>2024-09-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs), and circulating tumor related cells, are extremely rare cells that intravasate from the tumor into the circulatory system and can be captured via a liquid biopsy. Although CTCs contribute to the metastatic cascade, and diverse phenotypes of CTCs have been observed – including the cytokeratin expressing CTCs, CTC clusters, large polyploid CTCs, and CTCs undergoing epithelial to mesenchymal transition – little is known about their functionality. By virtue of CTCs being rare, a detection method that maximizes the information obtained per cell would be ideal to understand their biology and for use in diagnostic approaches. The challenge is that rare cell detection necessitates extensive processing steps, during which molecular content, such as RNA and DNA, needs to be preserved for downstream single cell analysis. We developed GEnomic, Transcriptomic, and MOrphological profiling of Rare cells (GeTMoR), a method that extends the High Definition Single Cell Assay for detecting rare cancer related cells to simultaneously image and profile the genome and transcriptome from single rare cells. We validated GeTMoR by spiking in cancer cell lines into whole blood to evaluate the quality of recovered gene expression and copy number profile from the same cell. The GeTMoR approach provides the ability to link the phenotype of rare cells, including CTCs, to their genome and transcriptome, thereby enabling insight into rare cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6df3d1f532e0cca0520b19b2419d2d9cd51ad9" target='_blank'>
              GeTMoR: Simultaneous genomic, transcriptomic, and morphological profiling of rare single cells
              </a>
            </td>
          <td>
            Rishvanth K. Prabakar, Michael J. Schmidt, Peter Kuhn, James B. Hicks
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="B-Myb, also known as MYB proto-oncogene like 2 (MYBL2), is an important transcription factor implicated in transcription regulation, cell cycle and tumorigenesis. However, the molecular mechanism underlying B-Myb-controlled transactivation in different cell contexts as well as its functional implication in cancers remains elusive. In this study, we have conducted a comprehensive genome-wide analysis of B-Myb binding sites in multiple immortalized or cancer cell lines and identified its critical target genes. The results revealed that B-Myb regulates a common set of core cell cycle genes and cell type-specific genes through collaboration with other important transcription factors (e.g. NFY and MuvB complex) and binding to cell type-invariant promoters and cell type-specific enhancers and super-enhancers. KIF2C, UBE2C and MYC were further validated as B-Myb target genes. Loss-of-function analysis demonstrated that KIF2C knockdown inhibited tumor cell growth both in vitro and in vivo, suppressed cell motility and cell cycle progression, accompanied with defects in microtubule organization and mitosis, strongly suggesting that KIF2C is a critical regulator of cancer cell growth and mitosis, and maintains high cancer cell motility ability and microtubule dynamics. Pan-cancer transcriptomic analysis revealed that the overexpression of both B-Myb and KIF2C presents as independent prognostic markers in various types of cancer. Notably, B-Myb associates with NFYB, binds to target gene promoters, enhancers and super-enhancers, and provokes a cascade of oncogenic gene expression profiles in cancers. Overall, our results highly suggest the critical implication of B-Myb-mediated gene regulation in cancers, and the promising therapeutic and prognostic potentials of B-Myb and KIF2C for cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0f31a53d23c8c9c328816abe67e6d92d102de4" target='_blank'>
              Genome-wide binding analysis unveils critical implication of B-Myb-mediated transactivation in cancers
              </a>
            </td>
          <td>
            Chuntao Tao, Tao Liu, Zongrong Zhao, Xuanqi Dou, X. Xia, Kailong Du, Xiaofeng Zuo, Yitao Wang, Tingting Wang, Youquan Bu
          </td>
          <td>2024-09-03</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="During cancer development, the interplay between the nucleus and the cell cycle leads to a state of genomic instability, often accompanied by observable morphological aberrations. These aberrations can be controlled by tumor cells to evade cell death, either by preventing or eliminating genomic instability. In epithelial ovarian cancer (EOC), overexpression of the multifunctional protein claudin-4 is a key contributor to therapy resistance through mechanisms associated with genomic instability. However, the molecular mechanisms underlying claudin-4 overexpression in EOC remain poorly understood. Here, we altered claudin-4 expression and employed a unique claudin-4 targeting peptide (CMP) to manipulate the function of claudin-4. We found that claudin-4 facilitates genome maintenance by linking the nuclear envelope and cytoskeleton dynamics with cell cycle progression. Claudin-4 caused nuclei constriction by excluding lamin B1 and promoting perinuclear F-actin accumulation, associated with remodeling nuclear architecture, thus altering nuclear envelope dynamics. Consequently, cell cycle modifications due to claudin-4 overexpression resulted in fewer cells entering the S-phase and reduced genomic instability. Importantly, disrupting biological interactions of claudin-4 using CMP and forskolin altered oxidative stress cellular response and increased the efficacy of PARP inhibitor treatment. Our data indicate that claudin-4 protects tumor genome integrity by remodeling the crosstalk between the nuclei and the cell cycle, leading to resistance to genomic instability formation and the effects of genomic instability-inducing agents. Graphical abstract Claudin-4 plays a crucial role in remodeling the cytoskeleton, particularly influencing nuclear architecture. This remodeling appears to act as a regulatory mechanism, limiting the progression of ovarian cancer cells into the S-phase and correlating with reduced genomic instability in ovarian tumors. Moreover, since olaparib treatment triggered a cellular oxidative response, it is likely that this claudin-4-mediated remodeling contributes to resistance against the effects of the PARP inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1779163fc52ccc64570aa04b3ca709410f5487db" target='_blank'>
              Claudin-4 remodeling of nucleus-cell cycle crosstalk maintains ovarian tumor genome stability and drives resistance to genomic instability-inducing agents
              </a>
            </td>
          <td>
            Fabian R. Villagomez, Julie Lang, Daniel Nunez-Avellaneda, K. Behbakht, Hannah L. Dimmick, Patricia Webb, K. Nephew, Margaret Neville, Elizabeth R. Woodruff, Benjamin G. Bitler
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Histones are essential for maintaining chromatin structure and function. Histone mutations lead to changes in chromatin compaction, gene expression, and the recruitment of DNA repair proteins to the DNA lesion. These disruptions can impair critical DNA repair pathways, such as homologous recombination and non-homologous end joining, resulting in increased genomic instability, which promotes an environment favorable to tumor development and progression. Understanding these mechanisms underscores the potential of targeting DNA repair pathways in cancers harboring mutated histones, offering novel therapeutic strategies to exploit their inherent genomic instability for better treatment outcomes. Here, we examine how mutations in histone H3 disrupt normal chromatin function and DNA damage repair processes and how these mechanisms can be exploited for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae6d859b6a16ea85b27994b62cbff421ebf7a45" target='_blank'>
              Histone H3 mutations and their impact on genome stability maintenance.
              </a>
            </td>
          <td>
            Lucas D Caeiro, Ramiro E Verdun, Lluis Morey
          </td>
          <td>2024-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Dinoflagellate birefringent chromosomes (BfCs) contain some of the largest known genomes, yet they lack typical nucleosomal micrococcal-nuclease protection patterns despite containing variant core histones. One BfC end interacts with extranuclear mitotic microtubules at the nuclear envelope (NE), which remains intact throughout the cell cycle. Ultrastructural studies, polarized light and fluorescence microscopy, and micrococcal nuclease-resistant profiles (MNRPs) revealed that NE-associated chromosome ends persisted post-mitosis. Histone H3K9me3 inhibition caused S-G2 delay in synchronous cells, without any effects at G1. Differential labeling and nuclear envelope swelling upon decompaction indicate an extension of the inner compartment into telosomal anchorages (TAs). Additionally, limited effects of low-concentration sirtinol on bulk BfCs, coupled with distinct mobility patterns in MNase-digested and psoralen-crosslinked nuclei observed on 2D gels, suggest that telomeric nucleosomes (TNs) are the primary histone structures. The absence of a nucleosomal ladder with cDNA probes, the presence of histone H2A and telomere-enriched H3.3 variants, along with the immuno-localization of H3 variants mainly at the NE further reinforce telomeric regions as the main nucleosomal domains. Cumulative biochemical and molecular analyses suggest that telomeric repeats constitute the major octameric MNRPs that provision chromosomal anchorage at the NE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a9b44dbc33f3ede2c75be82f4a1fe4ece31f7c3" target='_blank'>
              Dinochromosome Heterotermini with Telosomal Anchorages
              </a>
            </td>
          <td>
            A. Kwok, Kosmo T. H. Yan, Shaoping Wen, Shiyong Sun, Chongping Li, Joseph Tin Yum Wong
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6d816344be553ab4e2f5399f5fa15a18a391b" target='_blank'>
              High prevalence of chromosome 17 in breast cancer micronuclei: a means to get rid of tumor suppressors?
              </a>
            </td>
          <td>
            Laxmi Kumari, S. Sreedharanunni, Divya Dahiya, Pranab Dey, Alka Bhatia
          </td>
          <td>2024-10-22</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment
              </a>
            </td>
          <td>
            Marta Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV) uses latency programs to colonize the memory B-cell reservoir, and each program is associated with human malignancies. However, knowledge remains incomplete of epigenetic mechanisms that maintain the highly restricted latency I program, present in memory and Burkitt lymphoma cells, in which EBNA1 is the only EBV-encoded protein expressed. Given increasing appreciation that higher order chromatin architecture is an important determinant of viral and host gene expression, we investigated roles of Wings Apart-Like Protein Homolog (WAPL), a host factor that unloads cohesin to control DNA loop size and that was discovered as an EBNA2-associated protein. WAPL knockout (KO) in Burkitt cells de-repressed LMP1 and LMP2A expression, but not other EBV oncogenes, to yield a viral program reminiscent of EBV latency II, which is rarely observed in B-cells. WAPL KO also increased LMP1/2A levels in latency III lymphoblastoid cells. WAPL KO altered EBV genome architecture, triggering formation of DNA loops between the LMP promoter region and the EBV origins of lytic replication (oriLyt). Hi-C analysis further demonstrated that WAPL KO reprogrammed EBV genomic DNA looping. LMP1 and LMP2A de-repression correlated with decreased histone repressive marks at their promoters. We propose that EBV coopts WAPL to negatively regulate latent membrane protein expression to maintain Burkitt latency I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb753cd4a98829975ef9e5d570c14e44b7ff8" target='_blank'>
              The DNA loop release factor WAPL suppresses Epstein-Barr virus latent membrane protein expression to maintain the highly restricted latency I program
              </a>
            </td>
          <td>
            Laura A. Murray-Nerger, Davide Maestri, Xiang Liu, Zhixuan Li, Nina R Auld, I. Tempera, Mingxiang Teng, B. Gewurz
          </td>
          <td>2024-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Aneuploidy typically poses challenges for cell survival and growth. However, recent studies have identified exceptions where aneuploidy is beneficial for cells with mutations in certain regulatory genes. Our research reveals that cells lacking the spindle checkpoint gene BUB3 exhibit aneuploidy of select chromosomes. While the spindle checkpoint is not essential in budding yeast, the loss of BUB3 and BUB1 increases the probability of chromosome missegregation compared to wildtype cells. Contrary to the prevailing assumption that the aneuploid cells would be outcompeted due to growth defects, our findings demonstrate that bub3Δ cells consistently maintained aneuploidy of specific chromosomes over many generations. We investigated whether the persistence of these additional chromosomes in bub3Δ cells resulted from the beneficial elevated expression of certain genes, or mere tolerance. We identified several genes involved in chromosome segregation and cell cycle regulation that confer an advantage to Bub3-depleted cells. Overall, our results suggest that the upregulation of specific genes through aneuploidy may provide a survival and growth advantage to strains with poor chromosome segregation fidelity. AUTHOR SUMMARY Accurate chromosome segregation is crucial for the proper development of all living organisms. Errors in chromosome segregation can lead to aneuploidy, characterized by an abnormal number of chromosomes, which generally impairs cell survival and growth. However, under certain stress conditions, such as in various cancers, cells with specific mutations and extra copies of advantageous chromosomes exhibit improved survival and proliferation. In our study, we discovered that cells lacking the spindle checkpoint protein Bub3 became aneuploid, retaining specific chromosomes. This finding was unexpected because although bub3Δ cells have a higher rate of chromosome mis-segregation, they were not thought to maintain an aneuploid karyotype. We investigated whether the increased copy number of specific genes on these acquired chromosomes offered a benefit to Bub3-deficient cells. Our results revealed that several genes involved in chromosome segregation and cell cycle regulation prevented the gain of chromosomes upon Bub3-depletion, suggesting that these genes confer a survival advantage. Overall, our study demonstrates that cells lacking Bub3 selectively retain specific chromosomes to increase the copy number of genes that promote proper chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe65305b6ce867bfbed31e843276ed59a3f7ed3" target='_blank'>
              Aneuploidy of Specific Chromosomes is Beneficial to Cells Lacking Spindle Checkpoint Protein Bub3
              </a>
            </td>
          <td>
            Pallavi Gadgil, Olivia Ballew, Timothy J. Sullivan, Soni Lacefield
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f6b0780202531821c7a75fa420779272072de3" target='_blank'>
              Retrotransposon addiction promotes centromere function via epigenetically activated small RNAs
              </a>
            </td>
          <td>
            Atsushi Shimada, Jonathan Cahn, E. Ernst, Jason Lynn, D. Grimanelli, Ian R. Henderson, T. Kakutani, R. Martienssen
          </td>
          <td>2024-09-02</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0db6723b29afbac67e23443af4e69118004b98b8" target='_blank'>
              Chromatin conformation capture in the clinic: 4C-seq/HiC distinguishes pathogenic from neutral duplications at the GPR101 locus
              </a>
            </td>
          <td>
            Adrian F. Daly, Leslie A. Dunnington, D. F. Rodriguez-Buritica, Erica S. Spiegel, Francesco Brancati, G. Mantovani, Vandana M. Rawal, F. Faucz, H. Hijazi, J. Caberg, A. Nardone, Mario Bengala, P. Fortugno, G. Del Sindaco, M. Ragonese, Helen Gould, S. Cannavò, P. Pétrossians, A. Lania, J. R. Lupski, Albert M Beckers, Constantine A. Stratakis, Brynn Levy, G. Trivellin, Martin Franke
          </td>
          <td>2024-09-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Genome instability relies on preserving the chromatin structure, with any histone imbalances threating DNA integrity. Histone synthesis occurs in the cytoplasm, followed by a maturation process before their nuclear translocation. This maturation involves protein folding and the establishment of post-translational modifications. Disruptions in this pathway hinder chromatin assembly and contribute to genome instability. JMJD1B, a histone demethylase, not only regulates gene expression but also ensures a proper supply of histones H3 and H4 for the chromatin assembly. Reduced JMJD1B levels lead to the cytoplasmic accumulation of histones, causing defects in the chromatin assembly and resulting in DNA damage. To investigate the role of JMJD1B in regulating genome stability and the malignancy of melanoma tumors, we used a JMJD1B/KDM3B knockout in B16F10 mouse melanoma cells to perform tumorigenic and genome instability assays. Additionally, we analyzed the transcriptomic data of human cutaneous melanoma tumors. Our results show the enhanced tumorigenic properties of JMJD1B knockout melanoma cells both in vitro and in vivo. The γH2AX staining, Micrococcal Nuclease sensitivity, and comet assays demonstrated increased DNA damage and genome instability. The JMJD1B expression in human melanoma tumors correlates with a lower mutational burden and fewer oncogenic driver mutations. Our findings highlight JMJD1B’s role in maintaining genome integrity by ensuring a proper histone supply to the nucleus, expanding its function beyond gene expression regulation. JMJD1B emerges as a crucial player in preserving genome stability and the development of melanoma, with a potential role as a safeguard against oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490db96e0fd741bd69913b1e9fe76e21ec91750" target='_blank'>
              Unraveling the Role of JMJD1B in Genome Stability and the Malignancy of Melanomas
              </a>
            </td>
          <td>
            Perla Cruz, Diego Peña-Lopez, Diego Figueroa, Isidora Riobó, Vincenzo Benedetti, Francisco Saavedra, Claudia Espinoza-Arratia, Thelma M. Escobar, A. Lladser, Alejandra Loyola
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and show that STAG2 is uniquely tumor suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility and 3D genome conformation that impact cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relates STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin mediated mechanisms of lung tumor suppression. SUMMARY STAG2 is a frequently mutated cohesin subunit across several cancers and one of the most important functional suppressors of lung adenocarcinoma. Our findings underscore important roles of STAG2 in suppressing lung tumorigenesis and highlight a STAG2-PAXIP1/PAGR1 tumor-suppressive program that may transcend cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0708e85569c4f6a48ce19a8abe96d4d4b9af8050" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis
              </a>
            </td>
          <td>
            Emily L. Ashkin, Yuning J. Tang, Haiqing Xu, King L. Hung, Julia Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5462de017e3ff2bccdaec319bc740a6c77b29d76" target='_blank'>
              PROTAC-mediated conditional degradation of the WRN helicase as a potential strategy for selective killing of cancer cells with microsatellite instability
              </a>
            </td>
          <td>
            Vikram Tejwani, Thomas Carroll, Thomas Macartney, Susanne Bandau, C. Alabert, Giulia Saredi, R. Toth, John Rouse
          </td>
          <td>2024-09-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogen-induced mutations are thought near-random, with rare cancer-driver mutations underlying clonal expansion. Using high-fidelity Duplex Sequencing to reach a mutation frequency sensitivity of 4×10-9 per nt, we report that sun exposure creates pervasive mutations at sites with ∼100-fold UV-sensitivity in RNA-processing gene promoters – cyclobutane pyrimidine dimer (CPD) hyperhotspots – and these mutations have a mini-driver clonal expansion phenotype. Numerically, human skin harbored 10-fold more genuine mutations than previously reported, with neonatal skin containing 90,000 per cell; UV signature mutations increased 8,000-fold in sun-exposed skin, averaging 3×10-5 per nt. Clonal expansion by neutral drift or passenger formation was nil. Tumor suppressor gene hotspots reached variant allele frequency 0.1-10% via 30-3,000 fold clonal expansion, in occasional biopsies. CPD hyperhotspots reached those frequencies in every biopsy, with modest clonal expansion. In vitro, tumor hotspot mutations arose occasionally over weeks of chronic low-dose exposure, whereas CPD hyperhotspot mutations arose in days at 1000-fold higher frequencies, growing exponentially. UV targeted mini-drivers in every skin cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccbc5cb5ec1678ecc35a610c5268f3895b55100" target='_blank'>
              Pervasive Induction of Regulatory Mutation Microclones in Sun-exposed Skin
              </a>
            </td>
          <td>
            Vijay Menon, Alejandro Garcia-Ruiz, Susan Neveu, Brenda Cartmel, Leah M. Ferrucci, Meg Palmatier, Christine Ko, Kenneth Y. Tsai, Mio Nakamura, Sa Rang Kim, Michael Girardi, Karl Kornacker, Douglas E. Brash
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6cd4623814e1323d692f8305bc119537df808a" target='_blank'>
              Role of TRIP13 in human cancer development.
              </a>
            </td>
          <td>
            Chaohu Chen, Pan Li, Guangrui Fan, Enguang Yang, Suoshi Jing, Yibo Shi, Yuwen Gong, Luyang Zhang, Zhiping Wang
          </td>
          <td>2024-10-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metapolycentromeres consist of multiple sequential domains of centromeric chromatin associated with a centromere-specific variant of histone H3 (CENP-A), functioning collectively as a single centromere. To date, they have been revealed in nine flowering plant, five insect and six vertebrate species. In this paper, we focus on their structure and possible mechanisms of emergence and evolution. The metapolycentromeres may vary in the number of centromeric domains and in their genetic content and epigenetic modifications. However, these variations do not seem to affect their function. The emergence of metapolycentromeres has been attributed to multiple Robertsonian translocations and segmental duplications. Conditions of genomic instability, such as interspecific hybridization and malignant neoplasms, are suggested as triggers for the de novo emergence of metapolycentromeres. Addressing the “centromere paradox” – the rapid evolution of centromeric DNA and proteins despite their conserved cellular function – we explore the centromere drive hypothesis as a plausible explanation for the dynamic evolution of centromeres in general, and in particular the emergence of metapolycentromeres and holocentromeres. Apparently, metapolycentromeres are more common across different species than it was believed until recently. Indeed, a systematic review of the available cytogenetic publications allowed us to identify 27 candidate species with metapolycentromeres. Тhe list of the already established and newly revealed candidate species thus spans 27 species of flowering plants and eight species of gymnosperm plants, five species of insects, and seven species of vertebrates. This indicates an erratic phylogenetic distribution of the species with metapolycentromeres and may suggest an independent emergence of the metapolycentromeres in the course of evolution. However, the current catalog of species with identified and likely metapolycentromeres remains too short to draw reliable conclusions about their evolution, particularly in the absence of knowledge about related species without metapolycentromeres for comparative analysis. More studies are necessary to shed light on the mechanisms of metapolycentromere formation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a7898b3ea2f2d63e1ff7e6046657eb47951d50" target='_blank'>
              Structure and evolution of metapolycentromeres
              </a>
            </td>
          <td>
            E. Grishko, P. Borodin
          </td>
          <td>2024-10-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b108c069a149d86683e9a9aa9e66ae175bdebcb9" target='_blank'>
              MECOM Locus classical transcript isoforms affect tumor immune microenvironment and different targets in ovarian cancer
              </a>
            </td>
          <td>
            Ning Lan, Shuheng Bai, Min Chen, Xuan Wang, Zhaode Feng, Ying Gao, B. Hui, Wen Ma, Xiangxiang Zhang, Fengyuan Hu, Wanyi Liu, Wenyang Li, Fang Wu, Juan Ren
          </td>
          <td>2024-10-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Single-cell methods to assess DNA methylation have not yet achieved the same level of cell throughput compared to other modalities. Here, we describe sciMETv3, a combinatorial indexing-based technique that builds on our prior technology, sciMETv2. SciMETv3 achieves nearly a 100-fold improvement in cell throughput by increasing the index space while simultaneously reducing hands-on time and total costs per experiment. To reduce the sequencing burden of the assay, we demonstrate compatibility of sciMETv3 with capture techniques that enrich for regulatory regions, as well as the ability to leverage enzymatic conversion which can yield higher library diversity. We showcase the throughput of sciMETv3 by producing a >140k cell library from human middle frontal gyrus split across four multiplexed individuals using both Illumina and Ultima sequencing instrumentation. This library was prepared over two days by one individual and required no expensive equipment (e.g. a flow sorter, as required by sciMETv2). The same experiment produced an estimated 650k additional cells that were not sequenced, representing the power of sciMETv3 to meet the throughput needs of the most demanding atlas-scale projects. Finally, we demonstrate the compatibility of sciMETv3 with multimodal assays by introducing sciMET+ATAC, which will enable high- throughput exploration of the interplay between two layers of epigenetic regulation within the same cell, as well as the ability to directly integrate single-cell methylation datasets with existing single-cell ATAC-seq. Highlights Atlas-scale production of single-cell DNA methylation libraries in a single experiment Protocols and evaluation using both Illumina and Ultima Genomics sequencing platforms Compatibility of sciMETv3 with capture techniques to reduce sequencing burden Compatibility of sciMETv3 with enzymatic conversion methods Generation of an integrated >140,000 cell dataset from human middle frontal gyrus across four individuals Ability to profile both ATAC and genome-wide DNA methylation from the same cells and integration with datasets from each modality A novel implementation of the s3-ATAC technology that leverages a nanowell chip for increased throughput Motivation DNA methylation forms a basal layer of epigenomic regulatory control, shaping the genomic permissiveness of mammalian cells during lineage specification and development. Aberrant DNA methylation has been associated with myriad health conditions ranging from developmental disorders to cancer. The high cell type specificity necessitates analysis at the single-cell level, much like transcription or other epigenomic properties. However, robust and cost-effective techniques to produce atlas-scale datasets have not been realized for DNA methylation. Here, we directly meet this need by introducing sciMETv3, a high-throughput protocol capable of producing hundreds of thousands of single-cell DNA methylation profiles in a single experiment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac78c7a01f74f879240ac4385b6f1d516ec40258" target='_blank'>
              Atlas-scale Single-cell DNA Methylation Profiling with sciMETv3
              </a>
            </td>
          <td>
            Ruth V. Nichols, Lauren E. Rylaarsdam, Brendan L O'Connell, Z. Shipony, Nika Iremadze, Sonia N. Acharya, Andrew C. Adey
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ee5fba3b3bfc377211f04436065c1bd9deb16" target='_blank'>
              Elucidating the genotoxicity of Fusobacterium nucleatum-secreted mutagens in colorectal cancer carcinogenesis
              </a>
            </td>
          <td>
            Wenye Xu, Yuchen Zhang, Dongjiao Chen, Dan Huang, Yang Zhao, Wei Hu, Ling Lin, Yingzhi Liu, Shilan Wang, Judeng Zeng, Chuan Xie, H. Chan, Qing Li, Huarong Chen, Xiaodong Liu, Sunny H Wong, Jun Yu, Francis K. L. Chan, Matthew T V Chan, Siew C Ng, William K. K. Wu, Lin Zhang
          </td>
          <td>2024-09-27</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: The genes cadherin 4 (CDH4), Kirsten rat sarcoma viral oncogene homolog (KRAS), V-Raf murine sarcoma viral oncogene homolog B (BRAF), and microsatellite instability (MSI) each play a role in the development of colorectal cancer (CRC). CDH4 gene is located in chromosome 20q13.33 and its amplification or gain is the earliest mutational event found in the majority of CRC and colon polyps. This study aimed to identify copy number alteration in the CDH4 gene and its association with the clinical profile of CRC patients, including gender, age, tumor location and differentiation, frequency of BRAF and KRAS mutations, and MSI status. Methods: The DNA was extracted from 50 tumors and adjacent normal tissues based on the manufacturer’s instructions. Detection of MSI status was carried out by pentaplex polymerase chain reaction (PCR) and PCR products were size separated by capillary electrophoresis (CE) using an ABI 310 Genetic Analyzer. Both KRAS and BRAF mutations were identified by PCR and sequencing while CDH4 copy measurement was measured using TaqManTM Copy Number Assay. Results: Our findings showed 22 (44%) samples with no changes in the copy number of CDH4 gene. Interestingly, 21 (42%) cases had an amplification of the CDH4 gene or CDH4 gene gain, and seven (14%)  cases decreased in CDH4 gene expression or CDH4 gene loss. We found an association between changes in the CDH4 gene and gender (p=0.001). However, there was no association between changes in the CDH4 gene and age (p=0.979), tumor location (p=0.145) and differentiation (p=0.648), the frequency of BRAF (p=0.171) and KRAS mutations (p=0.184) and MSI status (p=0.923). Conclusions: Copy number alteration in CDH4 gene in CRC patients and this alteration is significantly associated with gender. Further studies with a larger number of samples are essential to confirm this result and to identify the cause of CDH4 copy number alteration and its biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/069088e6da87904889fd1ecfbf85b7ca30e5a1c7" target='_blank'>
              Identification of CDH4 Gene Copy Number Alteration and Its Association with Clinical Profile of Colorectal Cancer Patient
              </a>
            </td>
          <td>
            Ni Nyoman Ayu Dewi, A. Y. Rompis, I. M. Mulyawan, Ni Putu Ekawati, H. S. Sun
          </td>
          <td>2024-09-30</td>
          <td>Indonesian Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in genes involved in DNA damage repair (DDR) often lead to premature aging syndromes. While recent evidence suggests that inflammation, alongside mutation accumulation and cell death, may drive disease phenotypes, its precise contribution to in vivo pathophysiology remains unclear. Here, by modeling Ataxia Telangiectasia (A-T) and Bloom Syndrome in the African turquoise killifish (N. furzeri), we replicate key phenotypes of DDR syndromes, including infertility, cytoplasmic DNA fragments, and reduced lifespan. The link between DDR defects and inflammation is attributed to the activation of the cGAS-STING pathway and interferon signaling by cytoplasmic DNA. Accordingly, mutating cGAS partially rescues germline defects and senescence in A-T fish. Double mutants also display reversal of telomere abnormalities and suppression of transposable elements, underscoring cGAS’s non-canonical role as a DDR inhibitor. Our findings emphasize the role of interferon signaling in A-T pathology and identify the cGAS-STING pathway as a potential therapeutic target for genomic instability syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0762a0a7a477e0941f31815c1aa8f1b5b79a238" target='_blank'>
              The cGAS-STING pathway is an in vivo modifier of genomic instability syndromes
              </a>
            </td>
          <td>
            Marva Bergman, Uri Goshtchevsky, Tehila Atlan, Gwendoline Astre, Ryan Halabi, Hosniyah El, Eitan Moses, A. J. Lemus, BA Benayoun, Y. Tzfati, Ido Ben-Ami, I. Harel
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Background: Aberrant gene promoter methylation is one of the hallmarks of Acute Myeloid Leukemia (AML). RAD21 is an important gene, implicated in sister chromatids cohesion, DNA repair, the regulation of gene transcription, apoptosis and hematopoiesis. Methods: In this study, we investigate the possible implication of RAD21 promoter methylation in AML pathogenesis using a cohort of AML patients and a cohort of healthy individuals. Results: RAD21 promoter methylation was found in 24% of patients and in none of the controls (p = 0.023), indicating a possible contribution to AML development. Interestingly, a statistically higher frequency of RAD21 methylation was observed in patients with trisomy 8 (9/21, 42.9%, p = 0.021), while none of the patients with aberrations of chromosome 11 had RAD21 gene promoter methylation (0%, 0/11, p = 0.048). Patients with monosomal and complex karyotypes showed low frequencies of RAD21 methylation (7.7% and 15.4%, respectively) without reaching statistical significance. Moreover, ASXL1 mutations were not found to be associated with RAD21 methylation. Conclusions: This is the first study which provides evidence for a possible pathogenetic role of RAD21 promoter methylation in AML development and especially in AML with trisomy 8. Further studies of RAD21 promoter methylation in large series of different AML genetic subgroups may contribute to the elucidation of AML pathogenesis and to the identification of new epigenetic biomarkers with diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c902aea9aee61d29bf4e327d8db03e3f748b0fb1" target='_blank'>
              Cohesin RAD21 Gene Promoter Methylation in Patients with Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Manola, Sophia Zachaki, Katerina Kakosaiou, Agapi Ioannidou, Marina Kalomoiraki, T. Rampias
          </td>
          <td>2024-10-01</td>
          <td>Life</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD. Methods Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration. Results In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment. Conclusion ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4ff74f564f7b2a57419b04c8a2f80af41275c3" target='_blank'>
              Multiregional transcriptomic profiling provides improved prognostic insight in localized non-small cell lung cancer
              </a>
            </td>
          <td>
            Chenyang Li, Thinh T. Nguyen, Jian-Rong Li, Xingzhi Song, J. Fujimoto, L. Little, Curtis Gumb, C. Chow, Ignacio I. Wistuba, Andrew P Futreal, Jianhua Zhang, Shawna M. Hubert, J. Heymach, Jia Wu, Christopher I Amos, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-10-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c42616846e496900baa7e96476fa73f42799ad" target='_blank'>
              APC mutations dysregulate alternative polyadenylation in cancer
              </a>
            </td>
          <td>
            Austin M. Gabel, Andrea E. Belleville, James D Thomas, Jose Mario Bello Pineda, Robert K Bradley
          </td>
          <td>2024-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endogenous retroelements (EREs), which comprise half of the human genome, play a pivotal role in genome dynamics. Some EREs retained the ability to encode proteins, although most degenerated or served as a source for novel genes and regulatory elements during evolution. Despite ERE repression mechanisms developed to maintain genome stability, widespread pervasive ERE activation is observed in cancer including hematological malignancies. Challenging the perception of noncoding DNA as "junk," EREs are underestimated contributors to cancer driver mechanisms as well as antitumoral immunity by providing innate immune ligands and tumor antigens. This review highlights recent progress in understanding ERE co-option events in cancer and focuses on the controversial debate surrounding their causal role in shaping malignant phenotype. We provide insights into the rapidly evolving landscape of ERE research in hematological malignancies and their clinical implications in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70dce85bfb8495c63680f246f15584cd08410198" target='_blank'>
              Endogenous retroelements in hematological malignancies: From epigenetic dysregulation to therapeutic targeting.
              </a>
            </td>
          <td>
            Mohamed Chour, Françoise Porteu, S. Depil, V. Alcazer
          </td>
          <td>2024-10-10</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Differential lncRNA expression has been correlated to clinical characteristics of colorectal cancers (CRC), which are the second leading cause of cancer-related deaths. LncRNA UCA1 plays a role in epigenetic gene regulation in diverse cancers. We studied CRC cell properties in CRISPR/Cas9 HT29-derived models and, interestingly, UCA1-depleted HT29 cells presented an increased stem-cell phenotype. We show that loss of UCA1 affected SWI/SNF chromatin remodeling complexes, which are previously shown to be involved in maintaining stem-cell properties. Not only was UCA1 permissive for induced SWI/SNF subunit SMARCA2 (BRM) expression upon chemo-drug treatment, but it also affected subunit compositions of SWI/SNF complexes by direct interaction of UCA1 with both ATP helicase BRM and BRG1. UCA1 is known to stimulate proliferation and decrease apoptosis, and we here show that it can restrain cells from a stem cell phenotype. The dual action of UCA1 revealed in this study highlights the complex actions of lncRNAs in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628c7e858eaf7626c9eff2638deefeca33d7947c" target='_blank'>
              Long non-coding RNA UCA1 affects chromatin remodeling via SMARCA2-containing SWI/SNF complex in human colorectal cancer
              </a>
            </td>
          <td>
            Bernadette Neve, Elsa Hadj Bachir, B. Duchêne, Mouloud Souidi, Martin Figeac, Jean-Pascal Meneboo, Emannuelle Leteurtre, N. Jonckheere, Audrey Vincent, Isabelle Van Seuningen
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 High-grade gliomas are among the deadliest of all childhood cancers and the leading cause of cancer-related mortality in children. The standard treatment is resection followed by chemotherapy, most often temozolomide and radiation therapy. Unfortunately, there is no evidence that temozolomide influences survival in pediatric patients and only a limited number of new drugs have been identified for specific subgroups of tumors. The identification of fusion genes as possible drivers of the disease offers an opportunity for targeted therapies. We present a case of a six-year-old patient with a novel TRIM24::NTRK2 fusion positive epithelioid glioblastoma with subsequent relapses.



 We investigated the ramification of this fusion and evolution of the tumor through whole genome sequencing, RNA-sequencing and DNA methylation profiling, and studied the effect of the patient’s treatment using patient-derived cell cultures.



 Our studies show that the TRIM24::NTRK2 fusion has oncogenic abilities but loses its importance as the tumor evolves. In addition, we demonstrate an increased drug resistance over time, further indicating the diminished driver function of the fusion.



 The use of resequencing revealed parallel development of tumor subclones with different genomic alterations, highlighting the importance of genomic profiling from various parts of the tumor.



 This work was supported by the Swedish Research Council, the Swedish Childhood Cancer Foundation, and the Swedish Cancer Society
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6825c06ca5f91e7d7121415ac1f08b97ce06cc4e" target='_blank'>
              P16.10.A UNRAVELING THE CLINICAL IMPLICATIONS OF TRIM24::NTRK2 FUSION GENE IN PEDIATRIC HIGH-GRADE GLIOMA: A CASE STUDY
              </a>
            </td>
          <td>
            I. Karlsson, S. Keane, L. Deland, A. Lindlöf, T. Kling, T. Olsson Bontell, H. Fagman, J. Nilsson, D. Nilsson, M. Sabel, H. Carén, F. Abel
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The molecular biology of pituitary neuroendocrine tumors (PitNETs) revealed few recurrent mutations and extensive chromosomal alterations, with the latter being the driving force in a subset of these lesions. Addressing the need for an easily applicable diagnostic tool, we conducted a retrospective study of 61 PitNETs operated at a tertiary care center. All cases were subtyped according to the 2022 WHO classification of endocrine tumors. A genome wide NGS panel targeting 1500 single-nucleotide polymorphisms (SNP) was used to classify chromosomal imbalances, loss of heterozygosity and copy number variations in DNA from formalin fixed paraffin embedded tissues. We identified four distinct chromosomal patterns, with varying distribution among different tumor lineages. Forty-two of 61 (69%) PitNETs showed chromosomal alterations. Gonadotroph PitNETs showed mostly quiet genomes. The majority of lactotroph PitNETs (19/20, 95%) were altered, exhibiting a gained genome and a remarkably low recurrence rate. Nine of ten (90%) corticotroph PitNETs harbored chromosomal alterations, of which two aggressive corticotroph tumors and one metastatic corticotroph PitNET showed massive chromosomal losses leading to near haploid/near homozygous genomes. The comparison of the molecular profile of primary and recurrent PitNETs of five patients showed no significant accumulation of alterations over time. A simple genome wide 1500 SNP test can be used in the identification of outspoken aggressive subsets of PitNET by the occurrence of a near haploid/near homozygous genome. Furthermore, the presence of neoplastic tissue in resected material can be potentially confirmed for non-gonadotroph PitNETs in suboptimal histological assessment conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb2fdf64cf9cd448595700bdb95a06c56a9e233" target='_blank'>
              Chromosomal Alteration Patterns in PitNETs: Massive Losses in Aggressive Tumors.
              </a>
            </td>
          <td>
            Maaia-Margo Jentus, L. Bakker, M. Verstegen, Iris Pelsma, T. van Wezel, D. Ruano, E. Kapiteijn, S. Crobach, N. Biermasz, H. Morreau
          </td>
          <td>2024-10-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Chromatin, with its complex spatial and temporal organization, plays a crucial role in regulating gene expression. Recent advancements in super-resolution microscopy have revealed that nanoscale domains of heterochromatin (repressed segments) embedded within a euchromatin (active segments) background are fundamental units of 3D chromatin organization. In tissue-resident cells, the size of these heterochromatin domains varies with the microenvironment, particularly its stiffness, and chromatin organization is also influenced by pharmacological and epigenetic drugs. However, the mechanisms governing heterochromatin domain size under various conditions and their impact on gene expression remain unclear. To address this knowledge gap, we have developed a dynamic, next-generation sequencing informed chromatin copolymer model. Our model simulates the spatiotemporal evolution of chromatin, driven by passive diffusion and active epigenetic reactions, which interconvert euchromatin and heterochromatin. By integrating chromatin-chromatin interaction energetics and diffusion-reaction dynamics, we predict the formation of nanoscale heterochromatin-rich domains and establish a scaling relationship between their size and the modulation of epigenetic reaction rates. Additionally, our model predicts that epigenetic and chromatin compaction changes in response to changes in global reaction rates occur predominantly at domain boundaries. We validated these predictions via Hi-C contact map analysis and super-resolution imaging of hyperacetylated melanoma cells. Subsequent RNA-seq analysis suggested a pivotal role of these epigenetic shifts in influencing the metastatic potential of these cells. We further validated our mesoscale findings against chromatin rearrangement in hMSCs, which exhibit sensitivity of epigenetic reaction rates to changes in microenvironmental stiffness. Finally, we evaluated the effects of cycling of epigenetic reaction rates in silico, mimicking the cellular transition to different extracellular conditions, and back again. This finding reveals a cell-type invariant mechanism driven by domain boundaries, whereby chromatin organization guides epigenetic memory formation. Our findings show that chromatin reorganization in response to changes in epigenetic reaction rates resulting from alterations in the microenvironment, drug exposure and disease progression impacts both immediate cellular responses and long-term epigenetic memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e55d9db8c8d7ff0628f2ac7f62c8baea4b57b" target='_blank'>
              Polymer Model Integrates Super-Resolution Imaging and Epigenomic Sequencing to Elucidate the Role of Epigenetic Reactions in Shaping 4D Chromatin Organization
              </a>
            </td>
          <td>
            Vinayak Vinayak, Ramin Basir, Rosela Golloshi, Joshua Toth, Lucas E. Sant’Anna, Melike Lakadamyali, R. McCord, Vivek B. Shenoy
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The Genome in a Bottle Consortium (GIAB), hosted by the National Institute of Standards and Technology (NIST), is developing new matched tumor-normal samples, the first to be explicitly consented for public dissemination of genomic data and cell lines. Here, we describe a comprehensive genomic dataset from the first individual, HG008, including DNA from an adherent, epithelial-like pancreatic ductal adenocarcinoma (PDAC) tumor cell line and matched normal cells from duodenal and pancreatic tissues. Data for the tumor-normal matched samples comes from thirteen distinct state-of-the-art whole genome measurement technologies, including high depth short and long-read bulk whole genome sequencing (WGS), single cell WGS, and Hi-C, and karyotyping. These data will be used by the GIAB Consortium to develop matched tumor-normal benchmarks for somatic variant detection. We expect these data to facilitate innovation for whole genome measurement technologies, de novo assembly of tumor and normal genomes, and bioinformatic tools to identify small and structural somatic mutations. This first-of-its-kind broadly consented open-access resource will facilitate further understanding of sequencing methods used for cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa326219cd9eb4a9f8ee45c35471c32cd4348fc" target='_blank'>
              Development and extensive sequencing of a broadly-consented Genome in a Bottle matched tumor-normal pair
              </a>
            </td>
          <td>
            Jennifer H. McDaniel, Vaidehi Patel, Nathan D. Olson, Hua-Jun He, Zhiyong He, Kenneth D. Cole, Anthony Schmitt, Kristin Sikkink, F. Sedlazeck, Harsha Doddapaneni, S. Jhangiani, D. Muzny, M. Gingras, Heer Mehta, L. F. Paulin, A. Hastie, Hung-Chun Yu, Victor Weigman, Alison Rojas, Katie Kennedy, Jamie Remington, Isai Gonzalez, Mitch Sudkamp, Kelly N. Wiseman, Bryan R. Lajoie, Shawn Levy, Miten Jain, S. Akeson, G. Narzisi, Zoe Steinsnyder, C. Reeves, Jennifer Shelton, S. Kingan, Christine C. Lambert, Primo Bayabyan, Aaron M. Wenger, Ian J. McLaughlin, Aaron Adamson, Christopher Kingsley, Melanie Wescott, Young Kim, B. Paten, Jimin Park, I. Violich, K. Miga, J. Gardner, B. McNulty, G. Rosen, R. McCoy, Francesco Brundu, E. Sayyari, Konrad Scheffler, Sean Truong, Severine Catreux, Lesley Chapman Hannah, Doron Lipson, Hila Benjamin, Nika Iremadze, Ilya Soifer, Stephen M Eacker, Mary Wood, Erin Cross, Greg Husar, Stephen Gross, Michael Vernich, M. Kolmogorov, Tanveer Ahmad, Ayse G. Keskus, A. Bryant, F. Thibaud-Nissen, Jonathan Trow, Jacqueline Proszynski, J. Hirschberg, K. Ryon, Christopher E. Mason, Justin Wagner, Chunlin Xiao, Andrew S. Liss, J. Zook
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Abstract Recent advances in neoantigen research have accelerated the development of immunotherapies for cancers, such as glioblastoma (GBM). Neoantigens resulting from genomic mutations and dysregulated alternative splicing have been studied in GBM. However, these studies have primarily focused on annotated alternatively-spliced transcripts, leaving non-annotated transcripts largely unexplored. Circular ribonucleic acids (circRNAs), abnormally regulated in tumors, are correlated with the presence of non-annotated linear transcripts with exon skipping events. But the extent to which these linear transcripts truly exist and their functions in cancer immunotherapies remain unknown. Here, we found the ubiquitous co-occurrence of circRNA biogenesis and alternative splicing across various tumor types, resulting in large amounts of long-range alternatively-spliced transcripts (LRs). By comparing tumor and healthy tissues, we identified tumor-specific LRs more abundant in GBM than in normal tissues and other tumor types. This may be attributable to the upregulation of the protein quaking in GBM, which is reported to promote circRNA biogenesis. In total, we identified 1057 specific and recurrent LRs in GBM. Through in silico translation prediction and MS-based immunopeptidome analysis, 16 major histocompatibility complex class I-associated peptides were identified as potential immunotherapy targets in GBM. This study revealed long-range alternatively-spliced transcripts specifically upregulated in GBM may serve as recurrent, immunogenic tumor-specific antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38467e4c2c354585d56b10426872ad5289b2d75d" target='_blank'>
              Long-range alternative splicing contributes to neoantigen specificity in glioblastoma
              </a>
            </td>
          <td>
            Mingjun Ji, Qing Yu, Xin-Zhuang Yang, Xianhong Yu, Jiaxin Wang, Chun-hong Xiao, Ni A An, Chuanhui Han, Chuan-Yun Li, Wanqiu Ding
          </td>
          <td>2024-09-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feda6005d00312b29429cd211524a37b760d02e2" target='_blank'>
              Pan-cancer analysis of the immunological and oncogenic roles of ATAD2 with verification in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Zhecheng Li, Yaxin Zhao, Peng Huang, Zhaoyi Wu, D. Ouyang, AdolphusOsei Nyarko, Lei Ai, Zhejia Zhang, Shi Chang
          </td>
          <td>2024-09-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454e4622c260f66b26c8cb18854b8ca092214e10" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="AIMS
Identifying molecular alterations in the adenoma and carcinoma components within the same tumor would greatly contribute to understanding the neoplastic progression of early colorectal cancer.


METHODS AND RESULTS
We examined somatic copy number alterations (SCNAs) and mutations involved in the adenoma and carcinoma components obtained from the same tumor in 46 cases of microsatellite-stable carcinoma in adenoma, using a genome-wide SNP array and gene mutation panel. In addition, we also performed hierarchical clustering to determine the SCNA frequencies in the tumors, resulting in stratification of the samples into two subgroups according to SCNA frequency. Subgroup 1 was characterized by multiple SCNAs and carcinoma components exclusively, while Subgroup 2 was characterized by a low frequency of SCNAs and both the adenoma and carcinoma components. The numbers of total genes and genes with gains were higher in the carcinoma than adenoma components. The three most frequent gains in both components were located at 1p36.33-1q44, 2p25.3-2q37.3, and 3p26.3-3q29. However, no candidate genes mapped to these regions. APC and KRAS mutations were common in both components, whereas the frequency of TP53 mutations was statistically higher in the carcinoma than adenoma component. However, TP53 mutations were not correlated with SCNA frequency.


CONCLUSIONS
We suggest that considerable SCNAs and TP53 mutations are required for progression from adenoma to carcinoma within the same intramucosal neoplastic lesion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707a65c08f0d284d7dbfb0819241b827c75e5e85" target='_blank'>
              Comprehensive Analyses of Somatic Copy Number Alterations and Mutations Based on the Adenoma-Carcinoma Sequence.
              </a>
            </td>
          <td>
            Tamotsu Sugai, M. Osakabe, N. Uesugi, W. Habano, Naoki Yanagawa, Hiromu Suzuki
          </td>
          <td>2024-09-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Medulloblastoma and ependymoma are prevalent pediatric central nervous system tumors with significant molecular and clinical heterogeneity. We collected bulk RNA sequencing data from 888 medulloblastoma and 370 ependymoma tumors to establish a comprehensive reference landscape. Following rigorous batch effect correction, normalization, and dimensionality reduction, we constructed a unified landscape to explore gene expression, signaling pathways, gene fusions, and copy number variations. Our analysis revealed distinct clustering patterns, including two primary ependymoma compartments, EPN-E1 and EPN-E2, each with specific gene fusions and molecular signatures. In medulloblastoma, we achieved precise stratification of Group 3/4 tumors by subtype and in SHH tumors by patient age. Our landscape serves as a vital resource for identifying biomarkers, refining diagnoses, and enables the mapping of new patients’ bulk RNA-seq data onto the reference framework to facilitate accurate disease subtype identification. The landscape is accessible via Oncoscape, an interactive platform, empowering global exploration and application. One Sentence Summary A landscape built using only Transcriptomic analysis for medulloblastoma and ependymoma reveals novel insights about subtype specific biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7340d3bdec67105d60c89f8f44d73e7f797360" target='_blank'>
              Transcriptomic landscape identifies two unrecognized ependymoma subtypes and novel pathways in medulloblastoma
              </a>
            </td>
          <td>
            Sonali Arora, Nicholas Nuechterlein, Matt Jensen, Gregory Glatzer, P. Sievers, S. Varadharajan, A. Korshunov, Felix Sahm, Stephen C. Mack, Michael D Taylor, Eric C. Holland
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="The phenotypic plasticity of cancer cells has recently emerged as an important factor of treatment failure. The mechanisms of phenotypic plasticity are not fully understood. One of the hypotheses is that the degree of chromatin accessibility defines the easiness of cell transitions between different phenotypes. To test this, a method to compare overall chromatin accessibility between cells in a population or between cell populations is needed. We propose to measure chromatin accessibility by fluorescence signal from nuclei of cells stained with DNA binding fluorescent molecules. This method is based on the observations that small molecules bind nucleosome-free DNA more easily than nucleosomal DNA. Thus, nuclear fluorescence is proportional to the amount of nucleosome-free DNA, serving as a measure of chromatin accessibility. We optimized the method using several DNA intercalators and minor groove binders and known chromatin-modulating agents and demonstrated that chromatin accessibility is increased upon oncogene-induced transformation and further in tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf3429cc71eb3bb1823cf988315e0385b13e7d52" target='_blank'>
              Image-Based Quantitative Single-Cell Method Showed Increase of Global Chromatin Accessibility in Tumor Compared to Normal Cells
              </a>
            </td>
          <td>
            Mairead Commane, Vidula Jadhav, Katerina Leonova, Brian Buckley, Henry Withers, Katerina Gurova
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary DNA-dependent protein kinase (DNA-PK) is a key player in repairing DNA damage. Ku proteins detect DNA damage and activate DNA-PK. Blocking DNA-PK can make cancer treatments such as radiation more effective. We have developed Ku–DNA binding inhibitors (Ku-DBis) that stop DNA-PK activation, making cancer cells more sensitive to radiation. We recently discovered new Ku-DBis that enter cells better than predecessor compounds, allowing cellular and in vivo analyses. Some non-small cell lung cancer (NSCLC) cells, especially those lacking ATM, were very sensitive to these inhibitors and radiation. However, cells lacking BRCA1 were resistant, which reduced the effectiveness of radiation. In animal studies of NSCLC, Ku-DBi treatment inhibited DNA-PK and enhanced a radiation-dependent decrease in tumor cell proliferation. This is the first time a Ku-targeted inhibitor has been shown to work in living organisms, suggesting their potential application in cancer treatment. Abstract Background: DNA-dependent protein kinase (DNA-PK) is a validated cancer therapeutic target involved in DNA damage response (DDR) and non-homologous end-joining (NHEJ) repair of DNA double-strand breaks (DSBs). Ku serves as a sensor of DSBs by binding to DNA ends and activating DNA-PK. Inhibition of DNA-PK is a common strategy to block DSB repair and improve efficacy of ionizing radiation (IR) therapy and radiomimetic drug therapies. We have previously developed Ku–DNA binding inhibitors (Ku-DBis) that block in vitro and cellular NHEJ activity, abrogate DNA-PK autophosphorylation, and potentiate cellular sensitivity to IR. Results and Conclusions: Here we report the discovery of oxindole Ku-DBis with improved cellular uptake and retained potent Ku-inhibitory activity. Variable monotherapy activity was observed in a panel of non-small cell lung cancer (NSCLC) cell lines, with ATM-null cells being the most sensitive and showing synergy with IR. BRCA1-deficient cells were resistant to single-agent treatment and antagonistic when combined with DSB-generating therapies. In vivo studies in an NSCLC xenograft model demonstrated that the Ku-DBi treatment blocked IR-dependent DNA-PKcs autophosphorylation, modulated DDR, and reduced tumor cell proliferation. This represents the first in vivo demonstration of a Ku-targeted DNA-binding inhibitor impacting IR response and highlights the potential therapeutic utility of Ku-DBis for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d24d7ccb0dfea0d53a93383e79bae41d2a8cf" target='_blank'>
              Impact of Optimized Ku–DNA Binding Inhibitors on the Cellular and In Vivo DNA Damage Response
              </a>
            </td>
          <td>
            P. Mendoza-Munoz, N. Kushwaha, Dineshsinha Chauhan, Karim Ben Ali Gacem, Joy E. Garrett, J. Dynlacht, J. Charbonnier, Navnath S. Gavande, J. Turchi
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Human stem cells rely on enhanced DNA repair mechanisms to safeguard their ability to replenish somatic tissues. Telomerase counteracts telomere shortening and is a component of the stem cell DNA repair system that is regulated by ATM and ATR kinases. Here, we report that the DNA repair enzyme APEX2, but not its close paralog APEX1, is required for efficient telomerase reverse transcriptase (TERT) gene expression in human embryonic stem cells (hESC) and a melanoma cell line. We also observed that APEX2 knockdown significantly diminished telomerase enzyme activity. While APEX1 is known to regulate certain transcription factors, APEX2 has not been reported to influence gene expression. To gain insight into how APEX2 influences gene expression, we conducted RNA-seq following APEX2 knockdown in hESC. These results indicated that a number of genes, in addition to TERT, relied on APEX2 for efficient expression. Genes affected by APEX2 knockdown were significantly enriched for specific repetitive DNA families. These include mammalian-wide interspersed repeats (MIRs) and Alu elements. Chromatin immunoprecipitation experiments demonstrated the highest APEX2 binding near MIR sequences in TERT intron 2. Surprisingly, binding was low in the TERT proximal promoter, a region known to control TERT transcription. MIR and other repetitive DNA regions are common sites of DNA damage, suggesting that APEX2 recruitment and repair of TERT MIR sequences may play a role in influencing TERT expression. This new role for APEX2 in promoting efficient gene expression deepens our understanding of an emerging cancer therapeutic target. Further, as the TERT gene plays critical roles in stem cell maintenance, organismal development and aging, as well as in short telomere disorders and cancer, our observations provide insight into new strategies to modulate the expression of this important enzyme.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c18f456e9f9ac926ada6d6d91f9b7e89f672dbdc" target='_blank'>
              Apurinic/apyrimidinic endodeoxyribonuclease 2 (APEX2/APE2) is required for efficient expression of TERT in human embryonic stem cells
              </a>
            </td>
          <td>
            Josh L. Stern, Lindsay F. Rizzardi, Natalie R. Gassman
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Topoisomerases alter DNA topology by making transient DNA strand breaks (DSBs) in DNA. The DNA cleavage reaction mechanism includes the formation of a reversible protein/DNA complex that allows rapid resealing of the transient break. This mechanism allows changes in DNA topology with minimal risks of persistent DNA damage. Nonetheless, small molecules, alternate DNA structures, or mutations in topoisomerase proteins can impede the resealing of the transient breaks, leading to genome instability and potentially cell death. The consequences of high levels of enzyme/DNA adducts differ for type I and type II topoisomerases. Top1 action on DNA containing ribonucleotides leads to 2–5 nucleotide deletions in repeated sequences, while mutant Top1 enzymes can generate large deletions. By contrast, small molecules that target Top2, or mutant Top2 enzymes with elevated levels of cleavage lead to small de novo duplications. Both Top1 and Top2 have the potential to generate large rearrangements and translocations. Thus, genome instability due to topoisomerase mis-function is a potential pathogenic mechanism especially leading to oncogenic progression. Recent studies support the potential roles of topoisomerases in genetic changes in cancer cells, highlighting the need to understand how cells limit genome instability induced by topoisomerases. This review highlights recent studies that bear on these questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc620e1f3abd707db6dd8102f3cc18d304a5ef7" target='_blank'>
              Genome Instability Induced by Topoisomerase Misfunction
              </a>
            </td>
          <td>
            K. C. Nitiss, Afif F. Bandak, James M. Berger, J. Nitiss
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac0738a96662e313aea5fd9200de2f16be79bfc" target='_blank'>
              R-loops’ m6A modification and its roles in cancers
              </a>
            </td>
          <td>
            Yue Qiu, Changfeng Man, Luyu Zhu, Shiqi Zhang, Xiaoyan Wang, Dandan Gong, Yu Fan
          </td>
          <td>2024-10-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract STUDY QUESTION To what extent can genotype analysis aid in the classification of (mosaic) aneuploid embryos diagnosed through copy number analysis of a trophectoderm (TE) biopsy? SUMMARY ANSWER In a small portion of embryos, genotype analysis revealed signatures of meiotic or uniform aneuploidy in those diagnosed with intermediate copy number changes, and signatures of presumed mitotic or putative mosaic aneuploidy in those diagnosed with full copy number changes. WHAT IS KNOWN ALREADY Comprehensive chromosome screening (CCS) for preimplantation genetic testing has provided valuable insights into the prevalence of (mosaic) chromosomal aneuploidy at the blastocyst stage. However, diagnosis of (mosaic) aneuploidy often relies solely on (intermediate) copy number analysis of a single TE biopsy. Integrating genotype information allows for independent assessment of the origin and degree of aneuploidy. Yet, studies aligning both datasets to predict (putative mosaic) aneuploidy in embryos remain scarce. STUDY DESIGN, SIZE, DURATION A single TE biopsy was collected from 1560 embryos derived from 221 couples tested for a monogenic disorder (n = 218) or microdeletion-/microduplication syndrome (n = 3). TE samples were subjected to both copy number and genotyping analysis. PARTICIPANTS/MATERIALS, SETTING, METHODS Copy number and SNP genotyping analysis were conducted using GENType. Unbalanced chromosomal anomalies ≥10 Mb (or ≥20 Mb for copy number calls <50%) were classified by degree, based on low-range intermediate (LR, 30–50%), high-range intermediate (HR, 50–70%) or full (>70%) copy number changes. These categories were further subjected to genotyping analysis to ascertain the origin (and/or degree) of aneuploidy. For chromosomal gains, the meiotic division of origin (meiotic I/II versus non-meiotic or presumed mitotic) was established by studying the haplotypes. The level of monosomy (uniform versus putative mosaic) in the biopsy could be ascertained from the B-allele frequencies. For segmental aneuploidies, genotyping was restricted to deletions. MAIN RESULTS AND THE ROLE OF CHANCE Of 1479 analysed embryos, 24% (n = 356) exhibited a whole-chromosome aneuploidy, with 19% (n = 280) showing full copy number changes suggestive of uniform aneuploidy. Among 258 embryos further investigated by genotyping, 95% of trisomies with full copy number changes were identified to be of meiotic origin. For monosomies, a complete loss of heterozygosity (LOH) in the biopsy was observed in 97% of cases, yielding a 96% concordance rate at the embryo level (n = 248/258). Interestingly, 4% of embryos (n = 10/258) showed SNP signatures of non-meiotic gain or putative mosaic loss instead. Meanwhile, 5% of embryos (n = 76/1479) solely displayed HR (2.5%; n = 37) or LR (2.6%; n = 39) intermediate copy number changes, with an additional 2% showing both intermediate and full copy number changes. Among embryos with HR intermediate copy number changes where genotyping was feasible (n = 25/37), 92% (n = 23/25) showed SNP signatures consistent with putative mosaic aneuploidy. However, 8% (n = 2/25) exhibited evidence of meiotic trisomy (9%) or complete LOH in the biopsy (7%). In the LR intermediate group, 1 of 33 (3%) genotyped embryos displayed complete LOH. Furthermore, segmental aneuploidy was detected in 7% of embryos (n = 108/1479) (or 9% (n = 139) with added whole-chromosome aneuploidy). These errors were often (52%) characterized by intermediate copy number values, which closely aligned with genotyping data when examined (94–100%). LARGE SCALE DATA N/A. LIMITATIONS, REASONS FOR CAUTION The findings were based on single TE biopsies and the true extent of mosaicism was not validated through embryo dissection. Moreover, evidence of absence of a meiotic origin for a trisomy should not be construed as definitive proof of a mitotic error. Additionally, a genotyping diagnosis was not always attainable due to the absence of a recombination event necessary to discern between meiotic II and non-meiotic trisomy, or the unavailability of DNA from both parents. WIDER IMPLICATIONS OF THE FINDINGS Interpreting (intermediate) copy number changes of a single TE biopsy alone as evidence for (mosaic) aneuploidy in the embryo remains suboptimal. Integrating genotype information alongside the copy number status could provide a more comprehensive assessment of the embryo’s genetic makeup, within and beyond the single TE biopsy. By identifying meiotic aberrations, especially in presumed mosaic embryos, we underscore the potential value of genotyping analysis as a deselection tool, ultimately striving to reduce adverse clinical outcomes. STUDY FUNDING/COMPETING INTEREST(S) L.D.W. was supported by the Research Foundation Flanders (FWO; 1S74621N). M.B., K.T., F.V.M., S.J., A.V.T., V.S., D.S., A.D., and S.S. are supported by Ghent University Hospital. B.M. was funded by Ghent University. The authors have no conflicts of interest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ade8d5bd0bebf7b9ac605c17d25f1e4349645aa" target='_blank'>
              Aligning genotyping and copy number data in single trophectoderm biopsies for aneuploidy prediction: uncovering incomplete concordance
              </a>
            </td>
          <td>
            Lisa De Witte, M. Baetens, K. Tilleman, F. Vanden Meerschaut, Sandra Janssens, Ariane Van Tongerloo, Virginie Szymczak, D. Stoop, A. Dheedene, S. Symoens, Björn Menten
          </td>
          <td>2024-09-18</td>
          <td>Human Reproduction Open</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background: Colorectal cancer, a prevalent gastrointestinal carcinoma, has a high risk for recurrence when locally advanced and remains lethal when in an advanced stage. Prognostic biomarkers may help in better delineating the aggressiveness of this disease in individual patients and help to tailor appropriate therapies. CDX2, a transcription factor of gastrointestinal differentiation, has been proposed as a biomarker for good outcomes and could also be a marker of specific sub-types amenable to targeted therapies. Methods: Colorectal cancers from The Cancer Genome Atlas (TCGA) colorectal cohort and colon cancers from the Sidra-LUMC AC-ICAM cohort were categorized according to their expressions of CDX2 mRNA. Groups with CDX2 suppression were compared with cancers showing no suppression regarding their clinical and genomic characteristics. Results: CDX2-suppressed colorectal cancers showed a high prevalence of Microsatellite Instability (MSI) and a lower prevalence of chromosomal Instability (CIN) compared to non-CDX2-suppressed cancers. In addition, CDX2-suppressed cancers had a higher prevalence of mutations in several receptor tyrosine kinase genes, including EGFR, ERBB3, ERBB4, RET, and ROS1. In contrast, CDX2-suppressed cancers displayed lower mutation frequencies than non-CDX2-suppressed cancers in the genes encoding for the two most frequently mutated tumor suppressors, APC and TP53, and the most frequently mutated colorectal cancer oncogene, KRAS. However, CDX2-suppressed colorectal cancers had a higher prevalence of mutations in alternative genes of the WNT/APC/β-catenin and KRAS/BRAF/MEK pathways. In addition, they showed frequent mutations in DNA damage response (DDR) genes, such as BRCA2 and ATM. Conclusion: CDX2-suppressed colorectal cancers constitute a genomically distinct subset of colon and rectal cancers that have a lower prevalence of KRAS, APC, and TP53 mutations, but a high prevalence of mutations in less commonly mutated colorectal cancer genes. These alterations could serve as targets for personalized therapeutics in this subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f07fd50c1ab7d2b069279a733f924c136fa063e" target='_blank'>
              CDX2-Suppressed Colorectal Cancers Possess Potentially Targetable Alterations in Receptor Tyrosine Kinases and Other Colorectal-Cancer-Associated Pathways
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-10-01</td>
          <td>Diseases</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The p53 tumor suppressor is an indispensable regulator of DNA damage responses that accelerates carcinogenesis when mutated. In this report, we uncover a new mechanism by which p53 maintains genomic integrity in the absence of canonical DNA damage response activation. Specifically, loss of p53 dramatically alters chromatin structure at the nuclear periphery, allowing increased transmission of an environmental carcinogen, ultraviolet (UV) radiation, into the nucleus. Genome-wide mapping of UV-induced DNA lesions in p53-deficient primary cells reveals elevated lesion abundance in regions corresponding to locations of high mutation burden in malignant melanomas. These findings uncover a novel role of p53 in the suppression of mutations that contribute to cancer and highlight the critical influence of nuclear architecture in regulating sensitivity to carcinogens. One-Sentence Summary The p53 tumor suppressor reduces carcinogen sensitivity and mutagenic potential by maintaining nuclear architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce95350c5fb375408fd587e0eea1b652005b9ec" target='_blank'>
              p53 Regulates Nuclear Architecture to Reduce Carcinogen Sensitivity and Mutagenic Potential
              </a>
            </td>
          <td>
            Devin A. King, Dakota E. McCoy, Adrian Perdyan, Jakub Mieczkowski, Thierry Douki, Jennifer A. Dionne, Rafael E. Herrera, Ashby J. Morrison
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract RNA sequencing technology combining short read and long read analysis can be used to detect chimeric RNAs in malignant cells. Here, we propose an integrated approach that uses k-mers to analyze indexed datasets. This approach is used to identify chimeric RNA in chronic myelomonocytic leukemia (CMML) cells, a myeloid malignancy that associates features of myelodysplastic and myeloproliferative neoplasms. In virtually every CMML patient, new generation sequencing identifies one or several somatic driver mutations, typically affecting epigenetic, splicing and signaling genes. In contrast, cytogenetic aberrations are currently detected in only one third of the cases. Nevertheless, chromosomal abnormalities contribute to patient stratification, some of them being associated with higher risk of poor outcome, e.g. through transformation into acute myeloid leukemia (AML). Our approach selects four chimeric RNAs that have been detected and validated in CMML cells. We further focus on NRIP1-MIR99AHG, as this fusion has also recently been detected in AML cells. We show that this fusion encodes three isoforms, including a novel one. Further studies will decipher the biological significance of such a fusion and its potential to improve disease stratification. Taken together, this report demonstrates the ability of a large-scale approach to detect chimeric RNAs in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20cd2e1e29866db7633afbaeb3e311e423e92cc0" target='_blank'>
              Effective requesting method to detect fusion transcripts in chronic myelomonocytic leukemia RNA-seq
              </a>
            </td>
          <td>
            Florence Rufflé, Jérôme Reboul, A. Boureux, Benoit Guibert, Chloé Bessière, Raíssa Silva, Eric Jourdan, Jean-Baptiste Gaillard, A. Boland, J. Deleuze, Catherine Senamaud-Beaufort, D. Sélimoglu-Buet, Eric Solary, Nicolas Gilbert, Thérèse Commes
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e0df7d22a45444a437c740c4db84a4c490aa49" target='_blank'>
              Membrane RRM2-positive cells represent a malignant population with cancer stem cell features in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yongzhi Zhao, Shuting Xue, Danduo Wei, Jianjuan Zhang, Nachuan Zhang, Liping Mao, Niya Liu, Lei Zhao, Jianing Yan, Yifan Wang, Xiujun Cai, Saiyong Zhu, Stephanie Roessler, Junfang Ji
          </td>
          <td>2024-09-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Building on our prior work that RNA alternative splicing modulates the druggability of kinase fusions, this study probes the clinical significance of sole reciprocal fusions. These rare genomic arrangements, despite lacking kinase domains at the DNA level, demonstrated potential RNA‐level druggability in sporadic cases from our prior research. Methods Utilizing the large‐scale multicenter approach, we performed RNA sequencing and clinical follow‐up to evaluate a broad spectrum of kinase fusions, including ALK, ROS1, RET, BRAF, NTRK, MET, NRG1, and EGFR, in 1943 patients. Results Our findings revealed 51 instances (2.57%) of sole reciprocal fusions, predominantly in lung (57%), colorectal (14%), and glioma (10%) cancers. Comparative analysis with an MSKCC cohort confirmed the prevalence in diverse cancer types and identified unique fusion partners and chromosomal locales. Cross‐validation through RNA‐NGS and FISH authenticated the existence of functional kinase domains in subsets including ALK, ROS1, RET, and BRAF, which correlated with positive clinical responses to targeted kinase inhibitors (KIs). Conversely, fusions involving EGFR, NRG1, and NTRK1/2/3 generated nonfunctional transcripts, suggesting the need for alternative therapeutic interventions. Conclusion This inaugural multicenter study introduces a novel algorithm for detecting and treating sole reciprocal fusions in advanced cancers, expanding the patient population potentially amenable to KIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc03ed5b4cf3438881405d76d4a3c612f2c0230" target='_blank'>
              Clinical relevance and druggability of sole reciprocal kinase fusions: A large‐scale study
              </a>
            </td>
          <td>
            Jiao Feng, Tonghui Ma, Chunyang Wang, Baoming Wang, Qian Liu, Zhengchuang Liu, H. Tao, Zaiyuan Ye
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved tool for the removal of DNA double-strand breaks (DSBs) and the preservation of stalled and damaged DNA replication forks. Successful completion of HR requires the tumor suppressor BRCA2. Germline mutations in BRCA2 lead to familial breast, ovarian, and other cancers, underscoring the importance of this protein for maintaining genome stability. BRCA2 harbors two distinct DNA binding domains, one that possesses three oligonucleotide/oligosaccharide binding (OB) folds (known as the OB-DBD), and with the other residing in the C-terminal recombinase binding domain (termed the CTRB-DBD) encoded by the last gene exon. Here, we employ a combination of genetic, biochemical, and cellular approaches to delineate contributions of these two DNA binding domains toward HR and the maintenance of stressed DNA replication forks. We show that OB-DBD and CTRB-DBD confer ssDNA and dsDNA binding capabilities to BRCA2, respectively, and that BRCA2 variants mutated in either DNA binding domain are impaired in the ability to load the recombinase RAD51 onto ssDNA pre-occupied by RPA. While the CTRB-DBD mutant is modestly affected for HR, it exhibits a strong defect in the protection of stressed replication forks. In contrast, the OB-DBD is indispensable for both BRCA2 functions. Our study thus defines the unique contributions of the two BRCA2 DNA binding domains in genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b867a23dce384adad0b091d20596685ac211bb92" target='_blank'>
              Distinct roles of the two BRCA2 DNA binding domains in DNA damage repair and replication fork preservation
              </a>
            </td>
          <td>
            Francisco Neal, Wenjing Li, Mollie E. Uhrig, Neelam Sharma, Shahrez Syed, Sandeep Burma, R. Hromas, Alexander V. Mazin, Eloise Dray, David S. Libich, Shaun K. Olsen, Elizabeth V. Wasmuth, Weixing Zhao, Claus S. Sørensen, C. Wiese, Youngho Kwon, Patrick Sung
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The determination of the cancer prognosis is paramount for patients and medical personnel so that they can devise treatment strategies. Transcriptional-based signatures and subtypes derived from cancer biopsy material have been used in clinical practice for several cancer types to aid in setting the patient prognosis and forming treatment strategies. Other genomic features in cancer biopsies, such as copy number alterations (CNAs), have been underused in clinical practice, and yet they represent a complementary source of molecular information that can add detail to the prognosis, which is supported by recent work in breast, ovarian, and lung cancers. Here, through a systematic strategy, we explored the prognostic power of CNAs in 37 cancer types. In this analysis, we defined two modes of informative features, deep and soft, depending on the number of alleles gained or lost. These informative modes were grouped by amplifications or deletions to form four single-data prognostic models. Finally, the single-data models were summed or combined to generate four additional multidata prognostic models. First, we show that the modes of features are cancer-type dependent, where deep alterations generate better models. Nevertheless, some cancers require soft alterations to generate a feasible model due to the lack of significant deep alterations. Then, we show that the models generated by summing coefficients from amplifications and deletions appear to be more practical for many but not all cancer types. We show that the CNA-derived risk group is independent of other clinical factors. Furthermore, overall, we show that CNA-derived models can define clinically relevant risk groups in 33 of the 37 (90%) cancer types analyzed. Our study highlights the use of CNAs as biomarkers that are potentially clinically relevant to survival in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46f0ca721f6c654c87b0f219cadce02b53e07fbf" target='_blank'>
              Systematic Modeling of Risk-Associated Copy Number Alterations in Cancer
              </a>
            </td>
          <td>
            Alejandra Guardado, Raúl Aguirre-Gamboa, Víctor Treviño
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10bd05b394b6759a7d6959abc8009eb518a19281" target='_blank'>
              Multifaceted role of the DNA replication protein MCM10 in maintaining genome stability and its implication in human diseases.
              </a>
            </td>
          <td>
            Sumayyah M. Q. Ahmed, Jayaprakash Sasikumar, Suparna Laha, Shankar Prasad Das
          </td>
          <td>2024-09-06</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome-containing micronuclei are a hallmark of aggressive cancers. Micronuclei frequently undergo irreversible collapse, exposing their enclosed chromatin to the cytosol. Micronuclear rupture catalyzes chromosomal rearrangements, epigenetic abnormalities, and inflammation, yet mechanisms safeguarding micronuclear integrity are poorly understood. In this study, we found that mitochondria-derived reactive oxygen species (ROS) disrupt micronuclei by promoting a noncanonical function of charged multivesicular body protein 7 (CHMP7), a scaffolding protein for the membrane repair complex known as endosomal sorting complex required for transport III (ESCRT-III). ROS retained CHMP7 in micronuclei while disrupting its interaction with other ESCRT-III components. ROS-induced cysteine oxidation stimulated CHMP7 oligomerization and binding to the nuclear membrane protein LEMD2, disrupting micronuclear envelopes. Furthermore, this ROS-CHMP7 pathological axis engendered chromosome shattering known to result from micronuclear rupture. It also mediated micronuclear disintegrity under hypoxic conditions, linking tumor hypoxia with downstream processes driving cancer progression. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION Chromosomal instability, a hallmark of aggressive cancers, is characterized by the presence of micronuclei, which are cytosolic rupture-prone structures that contain entire chromosomes or chromosome arms. The irreversible collapse of micronuclear envelopes is a central event in tumor progression. Micronuclear collapse exposes the encapsulated DNA to the cytosol, catalyzing chromosomal rearrangements and epigenetic alterations that drive tumor heterogeneity as well as therapy resistance. Micronuclear rupture also activates inflammatory signaling pathways that reshape the tumor immune microenvironment, promoting metastasis. Despite its importance, the underlying mechanisms of micronuclear collapse are unclear. RATIONALE Maintaining the integrity of the nuclear membrane is essential for cellular functions and organismal viability. It is unclear what cellular safeguards are compromised in micronuclei that lead to the breakdown of nuclear membrane integrity. We posited that inherent dissimilarities between micronuclei and primary nuclei might explain their respective differences in membrane stability. For instance, micronuclei are 5- to 20-fold smaller than the primary nucleus. Secondly, they possess abnormal nuclear envelopes, as evidenced by aberrant nuclear pore function and reduced lamin B1 levels. In addition, once ruptured, micronuclear envelopes are seldom repaired. RESULTS In this study, we integrated prior observations with our findings that micronuclear envelope rupture occurs because of the aberrant interaction between micronuclei and mitochondria, a process that is mediated by reactive oxygen species (ROS). Indeed, ruptured micronuclei were more likely to be intermixed with the mitochondrial network compared with intact ones, and manipulation of ROS levels correspondingly altered the rates of micronuclear rupture. Subsequent proteomic analysis and genetic manipulations led us to the observation that ROS interfere with micronuclear integrity by disrupting the normal activity of the nuclear membrane repair complex known as the endosomal sorting complex required for transport III (ESCRT-III). Increases in ROS levels led to the micronuclear accumulation of the ESCRT-III scaffolding protein, charged multivesicular body protein 7 (CHMP7), promoting a noncanonical function. ROS inhibited the export of CHMP7, leading to its persistence in the micronucleus and aberrant binding to its inner nuclear membrane partner LEM domain nuclear envelope protein 2 (LEMD2). ROS-induced cysteine oxidation catalyzed CHMP7 aggregation and reduced its interaction with its canonical binding partners in the ESCRT-III complex. The binding of CHMP7 aggregates to LEMD2 promoted micronuclear membrane deformation and collapse. This effect was further exacerbated by ROS-dependent recruitment of the autophagy-related protein p62, which mediated the degradation of canonical ESCRT-III members, limiting any chances of subsequent membrane repair. Lastly, we found that this mechanism is relevant to human tumors. Elevated levels of ROS and aberrant CHMP7 function led to complex chromosomal rearrangements, or chromothripsis, known to arise from micronuclear rupture. Furthermore, ROS generated under hypoxia induced micronuclear rupture in a CHMP7-dependent manner. In line with these findings, we observed a significant increase in micronuclear rupture along with CHMP7 accumulation in hypoxic regions of human head and neck cancer as well as in ovarian tumors. CONCLUSION Our results have uncovered a pathological interaction between micronuclei and mitochondria that underlies micronuclear rupture. By implicating ROS as regulators of micronuclear integrity, this work provides a mechanistic link between ROS-inducing conditions (such as free-radical generation and hypoxia) and downstream processes known to arise from the presence of micronuclei, including chromosomal rearrangements, epigenetic dysregulation, and tumor-promoting inflammation. Mechanisms of micronuclear rupture. The proximity of mitochondria to micronuclei drives micronuclear membrane rupture through mitochondria-derived reactive oxygen species (ROS). ROS inhibit micronuclear export, leading to the excessive accumulation of CHMP7, a scaffolding protein associated with the nuclear membrane repair complex ESCRT-III. ROS-dependent cysteine oxidation promotes CHMP7 self-aggregation and its aberrant binding to the membrane protein LEMD2, causing micronuclear collapse. [Figure created with BioRender.com]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1748efac80bee30de3b1de5b76ab1af0bb5d42d" target='_blank'>
              Micronuclear collapse from oxidative damage
              </a>
            </td>
          <td>
            Melody Di Bona, Yanyang Chen, Albert S. Agustinus, Alice Mazzagatti, Mercedes A. Duran, Matthew Deyell, Daniel Bronder, James Hickling, Christy Hong, Lorenzo Scipioni, Giulia Tedeschi, Sara Martin, Jun Li, Aušrinė Ruzgaitė, N. Riaz, Parin Shah, Edridge K. D’Souza, D. Z. Brodtman, Simone Sidoli, B.H. Diplas, Manisha Jalan, Nancy Y Lee, Alban Ordureau, Benjamin Izar, A. Laughney, Simon N. Powell, Enrico Gratton, S. Santaguida, J. Maciejowski, Peter Ly, Thomas M. Jeitner, S. Bakhoum
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>4</td>
          <td>62</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aging-associated inflammation, or ‘inflammaging” is a driver of multiple age-associated diseases. Cyclic GMP-AMP Synthase (cGAS) is a cytosolic DNA sensor that functions to activate interferon response upon detecting viral DNA in the cytoplasm. cGAS contributes to inflammaging by responding to endogenous signals such as damaged DNA or LINE1 (L1) cDNA which forms in aged cells. While cGAS knockout mice are viable their aging has not been examined. Unexpectedly, we found that cGAS knockout mice exhibit accelerated aging phenotype associated with induction of inflammation. Transcription of L1 elements was increased in both cGAS knockout mice and in cGAS siRNA knockdown cells associated with high levels of cytoplasmic L1 DNA and expression of ORF1 protein. Cells from cGAS knockout mice showed increased chromatin accessibility and decreased DNA methylation on L1 transposons. Stimulated emission depletion microscopy (STED) showed that cGAS forms nuclear condensates that co-localize with H3K9me3 heterochromatin marks, and H3K9me3 pattern is disrupted in cGAS knockout cells. Taken together these results suggest a previously undescribed role for cGAS in maintaining heterochromatin on transposable elements. We propose that loss of cGAS leads to loss of chromatin organization, de-repression of transposable elements and induction of inflammation resulting in accelerated aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/252e22af7c9a39812c9fa757a0ab2586da5b1b30" target='_blank'>
              cGAS deficient mice display premature aging associated with de-repression of LINE1 elements and inflammation
              </a>
            </td>
          <td>
            John C. Martinez, Francesco Morandini, Lucinda Fitzgibbons, Natasha Sieczkiewicz, Sung Jae Bae, Michael E. Meadow, Eric Hillpot, Joseph Cutting, Victoria Paige, Seyed Ali Biashad, M. Simon, J. Sedivy, A. Seluanov, V. Gorbunova
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="DNA lies at the heart of the central dogma of life. Altering DNA can modify the flow of information in fundamental cellular processes such as transcription and translation. The ability to precisely manipulate DNA has led to remarkable advances in treating incurable human genetic ailments and has changed the landscape of biological research. Genome editors such as CRISPR-Cas nucleases and TALENs have become ubiquitous tools in basic and applied biological research and have been translated to the clinic to treat patients. The specificity and modularity of these genome editors have made it possible to efficiently engineer genomic DNA; however, underlying principles governing editing outcomes in eukaryotes are still being uncovered. Editing efficiency can vary from cell type to cell type for the same DNA target sequence, necessitating de novo design and validation efforts. Chromatin structure and epigenetic modifications have been shown to affect the activity of genome editors because of the role they play in hierarchical organization of the underlying DNA. Understanding the nuclear search mechanism of genome editors and their molecular interactions with higher order chromatin will lead to improved models for predicting precise genome editing outcomes. Insights from such studies will unlock the entire genome to be engineered for the creation of novel therapies to treat critical illnesses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de15652cba672686fe241fc8c1aaf364c6e3c7f6" target='_blank'>
              Impact of Chromatin Organization and Epigenetics on CRISPR-Cas and TALEN Genome Editing.
              </a>
            </td>
          <td>
            Surbhi Jain, Guanhua Xun, Huimin Zhao
          </td>
          <td>2024-09-24</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE
Nuclear protein in testis carcinoma (NC) is an underdiagnosed and aggressive squamous/poorly differentiated cancer characterized by rearrangement of the gene NUTM1 on chromosome 15q14. Co-occurring alternations have not been fully characterized.


METHODS
We analyzed the genomic and immune landscape of 54 cases of NC that underwent DNA- and RNA-based NGS sequencing (Caris).


RESULTS
While NC is driven by NUTM1 fusion oncoproteins, co-occurring DNA mutations in epigenetic or cell cycle pathways were observed in 26% of cases. There was no significant difference between the fusion partner of NUTM1 and co-occurring gene mutations. RNA sequencing analysis showed increased MYC pathway activity in NC compared with head and neck squamous cell carcinoma (HNSCC) and lung squamous cell carcinoma (LUSC), which is consistent with the known pathophysiology of NC. Characterization of the NC tumor microenvironment using RNA sequencing revealed significantly lower immune cell infiltration compared with HNSCC and LUSC. NC was 10× higher in patients with HNSCC and LUSC younger than 50 years than in those older than 70 years.


CONCLUSION
To our knowledge, this is the first series of NC profiled broadly at the DNA and RNA level. We observed fewer intratumoral immune cells by RNA sequencing, which may be associated with anecdotal data of lack of immunotherapy benefit in NC. High MYC pathway activity in NC supports ongoing trials targeting MYC suppression. The incidence of NC among patients younger than 50 years with LUSC/HNSCC supports testing for NC in these patients. The prognosis of NCs remains dismal, and future studies should focus on improving the response to immunotherapy and targeting MYC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c7a9669ff7523a745b866e303487bfb7d01d6d" target='_blank'>
              Multiomic Characterization and Molecular Profiling of Nuclear Protein in Testis Carcinoma.
              </a>
            </td>
          <td>
            G. Kroening, Jia Luo, Mark G Evans, T. Adeyelu, S. Ou, Z. Arter, Trisha M Wise-Draper, A. Sukari, Asfar S Azmi, David R Braxton, A. Elliott, David A Bryant, M. Oberley, Chul Kim, Geoffrey I. Shapiro, Christopher A French, M. Nagasaka
          </td>
          <td>2024-10-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf19432cc15369ce28dc15aa8918b2d7b3bd99a" target='_blank'>
              TP53: the unluckiest of genes?
              </a>
            </td>
          <td>
            A. Joerger, Thorsten Stiewe, Thierry Soussi
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1dca550adace3299de2700d468a5c183e07ebc1" target='_blank'>
              Widespread mutagenesis and chromosomal instability shape somatic genomes in systemic sclerosis
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Thomas Blouin, James McCollum, Latarsha Porcher, François Virard, Jiri Zavadil, Carol Feghali-Bostwick, Natalie Saini
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Activating FLT3 and RAS mutations commonly occur in leukemia with KMT2A‐gene rearrangements (KMT2A‐r). However, how these mutations cooperate with the KMT2A‐r to remodel the epigenetic landscape is unknown. Using a retroviral acute myeloid leukemia (AML) mouse model driven by KMT2A::MLLT3, we show that FLT3 ITD , FLT3 N676K , and NRAS G12D remodeled the chromatin accessibility landscape and associated transcriptional networks. Although the activating mutations shared a common core of chromatin changes, each mutation exhibits unique profiles with most opened peaks associating with enhancers in intronic or intergenic regions. Specifically, FLT3 N676K and NRAS G12D rewired similar chromatin and transcriptional networks, distinct from those mediated by FLT3 ITD . Motif analysis uncovered a role for the AP‐1 family of transcription factors in KMT2A::MLLT3 leukemia with FLT3 N676K and NRAS G12D , whereas Runx1 and Stat5a/Stat5b were active in the presence of FLT3 ITD . Furthermore, transcriptional programs linked to immune cell regulation were activated in KMT2A‐r AML expressing NRAS G12D or FLT3 N676K , and the expression of NKG2D‐ligands on KMT2A‐r cells rendered them sensitive to CAR T cell‐mediated killing. Human KMT2A‐r AML cells could be pharmacologically sensitized to NKG2D‐CAR T cells by treatment with the histone deacetylase inhibitor LBH589 (panobinostat) which caused upregulation of NKG2D‐ligand levels. Co‐treatment with LBH589 and NKG2D‐CAR T cells enabled robust AML cell killing, and the strongest effect was observed for cells expressing NRAS G12D . Finally, the results were validated and extended to acute leukemia in infancy. Combined, activating mutations induced mutation‐specific changes in the epigenetic landscape, leading to changes in transcriptional programs orchestrated by specific transcription factor networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad94e713ba8dadd871506c2d3b1dbcd4274d1950" target='_blank'>
              Activating mutations remodel the chromatin accessibility landscape to drive distinct regulatory networks in KMT2A‐rearranged acute leukemia
              </a>
            </td>
          <td>
            Qirui Zhang, Ton Falqués-Costa, Mattias Pilheden, Helena Sturesson, Tina Ovlund, Vendela Rissler, Anders Castor, Hanne V H Marquart, B. Lausen, T. Fioretos, Axel Hyrenius-Wittsten, Anna K. Hagström-Andersson
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1844470abca3c1fa6252bba317101f9730f2c0f4" target='_blank'>
              5-Formylcytosine: a new epigenetic player
              </a>
            </td>
          <td>
            Dharmendra Kumar, Iqbal Hyder, Wilfried A Kues
          </td>
          <td>2024-10-23</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b209080acf772da88d3de252f9c4248e66e84" target='_blank'>
              Hallmarks of a genomically distinct subclass of head and neck cancer
              </a>
            </td>
          <td>
            Tara Muijlwijk, I. Nauta, Anabel van der Lee, Kari J T Grünewald, A. Brink, Sonja H Ganzevles, Robert J. Baatenburg de Jong, L. Atanesyan, S. Savola, M. A. van de Wiel, Laura A N Peferoen, Elisabeth Bloemena, R. van de Ven, C. R. Leemans, J. Poell, R. Brakenhoff
          </td>
          <td>2024-10-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Breast cancer is a major problem in the health sector worldwide. The prevalence of breast cancer incidence is on the rise in many countries despite efforts to eradicate it. The substantial efforts for the treatment of breast cancer are quite successful but the main hindrance is early detection which in turn is dependent on various factors i.e. genetic history, environmental exposures, hormonal causes, and sedentary lifestyle. In previous studies, lncRNAs were known to modulate the progression of many cancers like gastric, lung, ovarian, and colorectal cancers. This study was designed to find these lncRNAs role in breast cancer development. Bioinformatics tools were used for in silco analysis of these lncRNAs LINC01356, LINC01357, ARF4-AS1, and FAM83B. These lncRNAs were evaluated in GEPIA2 for subtype analysis and survival probability, Phylo CSF UCSC genome browser for conserved sequences and protein-coding potential, BcGenExMiner for correlation analysis and UALCAN for protein-coding gene expression. These above-mentioned lncRNAs showed high expression in the basal subtype of TNBC as compared with other subtypes and they showed poor survival rates in the basal subtype. LINC01356 and LINC01357 showed positive correlation with TAFA3 and SLC16A1. ARF4-AS1 positively correlated with ASB14 and FAM83B positively correlated with HCRTR2. These lncRNAs showing positive correlation with different genes have a role in breast cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10a7530434e87606ba2330c319f2c88880cdad0" target='_blank'>
              Long Non-coding RNA and Progression of Breast Cancer
              </a>
            </td>
          <td>
            Usama Ahmed, Muhammad Abubakar, Salma Saeed Khan, Baqaur Rehman
          </td>
          <td>2024-10-02</td>
          <td>Oncology Treatment Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks (DSBs) through homologous recombination (HR) is vital for maintaining the stability and integrity of the genome. RNA binding proteins (RBPs) intricately regulate the DNA damage repair process, yet the precise molecular mechanisms underlying their function remain incompletely understood. In this study, we highlight the pivotal role of RPS15, a representative RBP, in homologous recombination repair. Specifically, we demonstrate that RPS15 promotes DNA end resection, a crucial step in homologous recombination. Notably, we identify an interaction between RPS15 and CtIP, a key factor in homologous recombination repair. This interaction is essential for CtIP recruitment to DSB sites, subsequent RPA coating, and RAD51 replacement, all critical steps in efficient homologous recombination repair and conferring resistance to genotoxic treatments. Functionally, suppressing RPS15 expression sensitizes cancer cells to X-ray radiation and enhances the therapeutic synergistic effect of PARP1 inhibitors in breast cancer cells. In summary, our findings reveal that RPS15 promotes DNA end resection to ensure effective homologous recombination repair, suggesting its potential as a therapeutic target in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2aedc66ec12ddcad5319b720ce461d7c9233af5" target='_blank'>
              RPS15 Coordinates with CtIP to Facilitate Homologous Recombination and Confer Therapeutic Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Baohang Lin, Guan Huang, Zishan Yuan, Xun Peng, Chunliang Yu, Jialu Zheng, Zequn Li, Juanyun Li, Jinan Liang, Bo Xu
          </td>
          <td>2024-10-03</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most SNPs associated with blood pressure (BP) are located in noncoding regions of the human genome, where they may regulate gene expression in BP-relevant cell types. However, the mechanisms by which these SNPs influence BP regulation are unclear. To address this, we developed comprehensive epigenomic landscapes in human induced pluripotent stem cell (iPSC)-derived endothelial cells (iECs) and vascular smooth muscle cells (iVSMCs), incorporating DNA methylation analysis (RRBS), transcriptome analysis (RNA-seq), chromatin state profiling (ATAC-seq), and chromatin interaction mapping (Micro-C).
 We identified 5301, 8916, and 6046 chromatin loops in iECs and 5329, 7598, and 5228 loops in iVSMCs at 4, 8, and 16 kb resolutions (FDR < 0.05) respectively based on the Micro-C analysis. Remarkably, 82.4% and 83.8% of these chromatin contact regions in iECs and iVSMCs, respectively, harbored at least one known regulatory element such as an enhancer, promoter, or transcription factor binding site. Notably, 109 and 119 BP-associated SNPs were found within the chromatin contact regions of iECs and iVSMCs, respectively.
 We further investigated chromatin contact regions that involved interactions of regulatory elements. In both iECs and iVSMCs, lower methylation levels were observed in chromatin contact regions involving promoter-promoter (PP) and enhancer-promoter (EP) interactions, compared to enhancer-enhancer (EE), enhancer-transcription factor binding site (ET), and promoter-transcription factor binding site (PT) interactions. Additionally, greater chromatin accessibility was observed in chromatin contact regions involving PP and EP interactions. BP-associated SNPs were significantly enriched in chromatin contact regions involving PP and EP interactions. Moreover, genes proximal to chromatin contact regions involving PP or EP interactions exhibited higher expression than those near EE or other regulatory element interactions.
 These findings offer new insights into how chromatin interactions between regulatory elements define transcriptional states in human endothelial and vascular smooth muscle cells, providing a novel basis for understanding the genetic regulation of BP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb4b46525efa16e36b116fa43729edbbc372f3c" target='_blank'>
              Abstract P364: Chromatin Interactions Between Regulatory Elements Influence Gene Expression in Human iPSC-Derived Endothelial and Vascular Smooth Muscle Cells
              </a>
            </td>
          <td>
            Pengyuan Liu, Yong Liu, Rajan Pandey, Atrayee Ray, Jing Huang, Qiongzi Qiu, Andrew S. Greene, Allen W. Cowley, Aron M. Geurts, Sridhar Rao, Mingyu Liang
          </td>
          <td>2024-09-01</td>
          <td>Hypertension</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e828a68b6f097a43f327c00f62a47eac243544" target='_blank'>
              SEPT9: From pan-cancer to lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenwen Wang, Xiaochen Zhang, Ping Gui, Qizhen Zou, Yuzhou Nie, Shenglin Ma, Shirong Zhang
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accurate monitoring of base excision repair (BER) activity in cancer cells is critical for advancing the comprehension of DNA repair processes, gaining insights into cancer development, and guiding treatment strategies. However, current assay techniques for assessing BER activity in cancer cells face challenges due to the heterogeneous origins and diversity of BER enzymes. In this work, we present a highly reliable triple loop-interlocked DNA codec (GATED) that enables precise assessment of BER activity in cancer cells through signal amplification mediated by multienzyme orthogonal activation. The GATED device features a dumbbell-shaped DNA probe to encode two BER enzymes for BER-related signal conversion as well as two bound circular DNA to decode the apurinic/apyrimidinic sites for apurinic/apyrimidinic endonuclease 1 (APE1)-mediated signal amplification. Importantly, GATED is orthogonally activated by multiple target BER enzymes (i.e., uracil DNA glycosylase, thymine DNA glycosylase, and APE1), resulting in a unified fluorescent signal that significantly improves the detection specificity and sensitivity to BER enzymes. Additionally, we demonstrate that the GATED has exceptional biostability within complex biological systems, where it was successfully employed to monitor BER activity in cancer cells with high specificity and enabled cell-based high-throughput screening for BER inhibitors. The GATED provides a much-needed tool for the real-time monitoring of BER activity and the screening of BER inhibitors in cancer cells, potentially advancing both the investigation and clinical application of BER biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4455a54c11e6664a89c2b008ca24acc5e5bb02" target='_blank'>
              Multienzymatic Orthogonal Activation of DNA Codec Enables Tumor-Specific Imaging of Base Excision Repair Activity.
              </a>
            </td>
          <td>
            Wei-Jia Zeng, Zhaopeng Chen, Yan-Mei Lei, W. Liang, Yaqin Chai, Ruo Yuan, Ying Zhuo
          </td>
          <td>2024-09-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The interplay between cancer and the immune system has captivated researchers for a long time. Recent developments in cancer immunotherapy have substantiated this interest with a significant benefit to cancer patients. Tumor and immune cells are regulated via a wide range of molecular mechanisms involving intricate transcriptional and epigenetic networks. Epigenetic processes influence chromatin structure and accessibility, thus governing gene expression, replication, and DNA damage repair. However, aberrations within epigenetic signatures are frequently observed in cancer. One of the key epigenetic marks is the trimethylation of histone 3 at lysine 9 (H3K9me3), confined mainly within constitutive heterochromatin to suppress DNA accessibility. It is deposited at repetitive elements, centromeric and telomeric loci, as well as at the promoters of various genes. Dysregulated H3K9me3 deposition disrupts multiple pathways, including immune signaling. Consequently, altered H3K9me3 dynamics may modify the efficacy of immunotherapy. Indeed, growing evidence highlights the pivotal roles of various proteins mediating H3K9me3 deposition (SETDB1/2, SUV39H1/2), erasure (KDM3, KDM4 families, KDM7B, LSD1) and interpretation (HP1 proteins, KAP1, CHD4, CDYL, UHRF1) in modulating immunotherapy effectiveness. Here, we review the existing literature to synthesize the available information on the influence of these H3K9me3 writers, erasers, and readers on the response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145e5ffc384f02a28629c78f8d74cf791460fc6e" target='_blank'>
              The Roles of H3K9me3 Writers, Readers, and Erasers in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Urszula Oleksiewicz, Monika Kuciak, Anna Jaworska, Dominika Adamczak, Anna Bisok, Julia Mierzejewska, Justyna Sadowska, Patrycja Czerwińska, Andrzej Mackiewicz
          </td>
          <td>2024-10-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be65892cddf3c3f0a2ac733c30a323fa81a3c5f5" target='_blank'>
              Accurate identification of locally aneuploid cells by incorporating cytogenetic information in single cell data analysis
              </a>
            </td>
          <td>
            Ziyi Li, Ruoxing Li, I. Ganan-Gomez, Hussein A. Abbas, Guillermo Garcia-Manero, Wei Sun
          </td>
          <td>2024-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recent advancements in spatial biology have revolutionized our understanding of the organization and functional dynamics of cells and tissues. In this study, we applied Molecular Pixelation (MPX), a single-cell spatial proteomics assay, to investigate the modulation of the cell surface proteome in an in vitro drug screening model using the ETV6::RUNX1 acute lymphoblastic leukemia (ALL) cell line, Reh. Specifically, we focused on the in vitro response to fludarabine, a chemotherapeutic agent used prior to allogenic stem cell transplantation and chimeric antigen receptor (CAR)-T cell therapy in high-risk, refractory, or relapsed ALL patients. Using MPX, we quantified changes in protein abundance, spatial distribution, and colocalization of 76 targeted cell surface proteins in Reh cells before and after fludarabine treatment. Our analysis revealed 25 proteins with altered abundance, 24 proteins with increased polarity, and 138 protein pairs with modified colocalization following treatment. Notably, the tetraspanins CD82 and CD53, which are known for their roles in chemotherapy resistance, exhibited increased abundance, polarization, and colocalization post-treatment, suggesting their potential as a therapeutic scaffold. These findings underscore the unique ability of spatially resolved single-cell proteomics to uncover nuanced cellular responses that would otherwise remain undetected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a352c544200f32c43afeccf0c1c6f723f3d81" target='_blank'>
              Mapping the Spatial Proteome of Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            , Henrik Gezelius, Anna Pia Enblad, Anders Lundmark, Mariya Lysenkova Wiklander, O. Krali, Martin Åberg, Arja Harila, A. Raine, Claes Andersson, J. Nordlund
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10c01fddb459444a48acb4656dc4b4986c01eac" target='_blank'>
              SASS6 promotes tumor proliferation and is associated with TP53 and immune infiltration in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zihao Li, Lingyun He, Jia-Yi Li, Jing Qian, Zuotao Wu, Yongjie Zhu, Ting-ting Zhuo, Ju-Sen Nong, Honghua Liang, Hua Zheng, Weijia Huang, Julu Huang, Jianbin Cao
          </td>
          <td>2024-10-24</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Cervical cancer (CC) remains a significant global health challenge, characterized by genetic heterogeneity and a complex molecular landscape, both of which contribute to its pathogenesis. This study aimed to investigate germline variants in proto-oncogenes and tumor suppressor genes in cervical cancer patients, with the objective of clarifying their potential role in disease development. Methods: We utilized a custom next-generation sequencing (NGS) panel targeting 48 genes implicated in oncogenesis. Germline DNA samples from cervical cancer patients were analyzed in order to identify nucleotide sequence alterations. Variants were classified according to pathogenicity and clinical relevance, based on established guidelines. Results: A total of 148 nucleotide variants were detected within the cohort. Of these, 35 variants (23.6%) were classified as benign. In contrast, 105 variants (70.9%) were identified as variants of uncertain significance (VUSs). Moreover, seven pathogenic or likely pathogenic mutations were discovered, along with the polymorphic variant rs1042522 in the TP53 gene, which has been associated with an increased risk of cervical cancer. Conclusions: Our findings contribute to expanding our understanding of the molecular genetic landscape of cervical cancer. They emphasize the potential contribution of rare germline mutations to its development and progression. These results highlight the importance of comprehensive genetic screening in order to improve diagnostic and therapeutic approaches for cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9744546f38ec2bc7fa36858f93c880cbede4f99" target='_blank'>
              Germline Variants in Proto-Oncogenes and Tumor Suppressor Genes in Women with Cervical Cancer
              </a>
            </td>
          <td>
            K. Lenkova, R. Khusainova, R. Minyazeva, A. Zaripova, I. Gilyazova, Natalia Mokrysheva, I. Minniakhmetov
          </td>
          <td>2024-10-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Extrachromosomal circular DNA (eccDNA) has emerged as a central focus in molecular biology, with various types being found across species through advanced techniques, including high-throughput sequencing. This dynamic molecule exerts significant influence on aging and immune function and plays pivotal roles in autoimmune diseases, type 2 diabetes mellitus, cancer, and genetic disorders.


SUMMARY
This comprehensive review investigates the classification, characteristics, formation processes, and multifaceted functions of eccDNA, providing an in-depth exploration of its mechanisms in diverse diseases.


KEY MESSAGES
The goal of this review is to establish a robust theoretical foundation for a more comprehensive understanding of eccDNA, offering valuable insights for the development of clinical diagnostics and innovative therapeutic strategies in the context of related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f915db54a9ed6fe76aa5229ced4103ebd9df09e" target='_blank'>
              Exploring the Role of Extrachromosomal Circular DNA in Human Diseases.
              </a>
            </td>
          <td>
            Yali Peng, Huihui Tao, Guoying Wang, Mengyao Wu, Tinatin Xu, Chunmei Wen, Xuejia Zheng, Yong Dai
          </td>
          <td>2024-09-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Benign prostatic hyperplasia (BPH) is a growing issue due to an ageing population. Our study investigated the possible associations between BPH and ageing hallmarks, including the telomere length (TL) and mitochondrial genome copy number (mtDNA CN), along with genetic variations in the TP53 gene and mtDNA. Methods: Prostate tissue samples were obtained from 32 patients with BPH, together with 30 blood samples. As a healthy control group, age-matching blood DNA samples were used. For the comparison of mtDNA sequence data, 50 DNA samples of the general Latvian population were used. The full mtDNA genome was analyzed by using Next-Generation Sequencing (NGS), the TP53 gene by Sanger sequencing, and the mtDNA copy number (mtDNA CN) and telomere length (TL) byqPCR assay. Results: The results showed that in BPH patients, telomeres in the prostate tissue were significantly longer than in blood cells, while the TL in blood cells of the healthy controls was the shortest. Also, the mtDNA amount in the prostate tissue of BPH patients was significantly greater in comparison with blood cells, and controls had the smallest mtDNA CN. We did not find any mutations in the TP53 gene that could be linked to BPH; however, in mtDNA, we found several unique mutations and heteroplasmic changes, as well as genetic changes that have been previously associated with prostate cancer. Conclusions: In conclusion, prolonged telomeres and changes in the mtDNA amount might be involved in the molecular mechanisms of BPH. Some of the heteroplasmic or homoplasmic mtDNA variants might also contribute to the development of BPH. Additional studies are needed to substantiate these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e647c0f85f9b55a3e894f703c6a09a5dbf930766" target='_blank'>
              Changes in TP53 Gene, Telomere Length, and Mitochondrial DNA in Benign Prostatic Hyperplasia Patients
              </a>
            </td>
          <td>
            E. Zole, Edgars Baumanis, Lauma Freimane, Rolands Dāle, A. Leiše, Vilnis Lietuvietis, Renāte Ranka
          </td>
          <td>2024-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary By hypothesizing that molecular hyper-recombination and cellular proliferation are among the major features of multi-factorial cancer proneness, by using skin fibroblasts derived from eight major cancer syndromes, a significant correlation was found between the hyper-recombination rate quantified by plasmid assay, proliferation capacity assessed by flow cytometry, and excess of relative cancer risk (ERR). The product of the hyper-recombination rate and capacity of proliferation described a linear function of ERR. Abstract Context: Although carcinogenesis is a multi-factorial process, the mutability and the capacity of cells to proliferate are among the major features of the cells that contribute together to the initiation and promotion steps of cancer formation. Particularly, mutability can be quantified by hyper-recombination rate assessed with specific plasmid assay, hypoxanthine-guanine phosphoribosyltransferase (HPRT) mutations frequency rate, or MRE11 nuclease activities. Cell proliferation can be assessed by flow cytometry by quantifying G2/M, G1 arrests, or global cellular evasion. Methods: All these assays were applied to skin untransformed fibroblasts derived from eight major cancer syndromes characterized by their excess of relative cancer risk (ERR). Results: Significant correlations with ERR were found between hyper-recombination assessed by the plasmid assay and G2/M arrest and described a third-degree polynomial ERR function and a sigmoidal ERR function, respectively. The product of the hyper-recombination rate and capacity of proliferation described a linear ERR function that permits one to better discriminate each cancer syndrome. Conclusions: Hyper-recombination and cell proliferation were found to obey differential equations that better highlight the intrinsic bases of cancer formation. Further investigations to verify their relevance for cancer proneness induced by exogenous agents are in progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6745d80a485fc8627c999d4c147aa5ba6f6eca06" target='_blank'>
              Prediction of Cancer Proneness under Influence of X-rays with Four DNA Mutability and/or Three Cellular Proliferation Assays
              </a>
            </td>
          <td>
            Laura El Nachef, L. Bodgi, Maxime Estavoyer, Simon Buré, Anne-Catherine Jallas, A. Granzotto, Juliette Restier-Verlet, Laurène Sonzogni, J. Al-Choboq, Michel Bourguignon, Laurent Pujo-Menjouet, N. Foray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Disclosure: N. Gillis: None. C.A. Lange: None. Current treatments for estrogen receptor (ER) positive breast cancer largely target rapidly dividing tumor cells. However, aggressive breast cancers often contain long-lived and weakly proliferative cancer stem cells (CSCs) that readily escape treatments aimed at cycling cells. We previously reported that progesterone receptors (PR) and their target genes aid the growth of these CSCs and help them evade anti-estrogen therapy. As proliferation and stemness are directed by separate signaling pathways that lead to the expression of distinct gene programs, it's crucial to understand how PR facilitates this cellular transformation at the genomic level. Establishing a gene expression program that may limit proliferation but encourages stemness requires coordination between transcription factors and their coregulators to alter the nuclear microenvironment and chromatin landscape.To reveal these mechanisms, we profiled PR and ER genomic binding using CUT&RUN in the T47D luminal breast cancer cell line grown in ultra-low attachment (3D) mammosphere conditions. Our analysis of genomic binding profiles displayed a pattern of PR/ER co-occupancy at distant sites more than 10 kilobases away from the nearest transcriptional start site. We used Rank Ordering of Super Enhancer (ROSE) analysis on these PR/ER binding sites to predict which are likely super enhancer elements capable of coordinating the regulation of multi-gene subsets. We employed a genetic approach to directly test the function of novel enhancer sites linked to PR-target genes by disrupting the PR/ER binding with a dCas9-KRAB construct directed to the predicted enhancer elements. We propose that abnormal growth factor signaling in breast cancer cells permits PR/ER complex formation which promotes expression of genes that induce endocrine resistance and CSC expansion through regulation of super enhancer elements. Further studies to elucidate these mechanisms are ongoing. Understanding the intricacies of PR/ER crosstalk is the crucial next step in developing new therapies that target PR-driven processes in ER+ breast cancers. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc697beea9d371162709b515b1a10bf7a5a06aca" target='_blank'>
              7324 Regulation of Super Enhancers by Phosphorylated Progesterone Receptors Drives Breast Cancer Stem Cell Expansion
              </a>
            </td>
          <td>
            N. Gillis, C. Lange
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Previous study indicated that CHK1 was important for repairing DNA damage and cell cycle regulation. Aims: To investigate the association of Checkpoint kinase 1 (CHK1) expression with clinicopathological features, prognosis, and immune infiltration in cancer. Methods: Several databases were searched for relevant publications to conduct a meta-analysis to reveal the association between CHK1 and clinicopathological features of cancer. TIMER and GEPIA datasets were utilized to explore the differential expression of CHK1 of tumors. GEPIA and Kaplan-Meier Plotter databases were adopted to detect the prognostic significance of CHK1 in tumor. TIMER and cBioPortal databases were used for the analysis regarding immune infiltration and mutation respectively. Results: We found that CHK1 expression was significantly associated with low differentiation (OR=3.94, 95% CI: 2.73-5.67, P<0.05), advanced stage (OR=3.20, 95% CI: 2.30-4.44, P<0.05), vascular infiltration (OR=3.24, 95% CI: 2.18-4.82, P<0.05) and lymph node metastasis (OR=3.55, 95% CI: 2.62-4.82, P<0.05) of various cancers. CHK1 was highly expressed in multiple cancers and was related to clinical stage, survival, immune infiltration in pan-cancers. Conclusions: Our study found that CHK1 was significantly related to prognosis and immunological status in various cancers, suggesting that CHK1 may serve as a useful biomarker for prognosis and immune infiltration in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48bd08dda4568b1d8c35e755803a8c000bea8fc" target='_blank'>
              Exploration of the carcinogenetic and immune role of CHK1 in human cancer
              </a>
            </td>
          <td>
            Jian Zhou, Ziyi Wu, Dilihumaer Aili, Lu Wang, Tanghai Liu
          </td>
          <td>2024-09-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) are widely recognized as the result of exogenous retroviruses infecting the ancestral germline, stabilizing integration and vertical transmission during human genetic evolution. To date, endogenous retroviruses (ERVs) appear to have been selected for human physiological functions with the loss of retrotransposable capabilities. ERV elements were previously regarded as junk DNA for a long time. Since then, the aberrant activation and expression of ERVs have been observed in the development of many kinds of human diseases, and their role has been explored in a variety of human disorders such as cancer. The results show that specific ERV elements play respective crucial roles. Among them, long non-coding RNAs (lncRNAs) transcribed from specific long-terminal repeat regions of ERVs are often key factors. lncRNAs are over 200 nucleotides in size and typically bind to DNA, RNA, and proteins to perform biological functions. Dysregulated lncRNAs have been implicated in a variety of diseases. In particular, studies have shown that the aberrant expression of some ERV-derived lncRNAs has a tumor-suppressive or oncogenic effect, displaying significant functional bidirectionality. Therefore, theses lncRNAs have a promising future as novel biomarkers and therapeutic targets to explore the concise relationship between ERVs and cancers. In this review, we first summarize the role of ERV-derived lncRNAs in physiological regulation, mainly including immunomodulation, the maintenance of pluripotency, and erythropoiesis. In addition, pathological regulation examples of their aberrant activation and expression leading to carcinogenesis are highlighted, and specific mechanisms of occurrence are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362567f05869d7bba363004ce51b667dfd018994" target='_blank'>
              Functional Bidirectionality of ERV-Derived Long Non-Coding RNAs in Humans
              </a>
            </td>
          <td>
            Yanmei Song, Hongling Wen, Xiu-wei Zhai, Lei Jia, Lin Li
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48d863d20b27ee2206ff6d80ccbbfa69c29c0c2" target='_blank'>
              3D genome contributes to MHC-II neoantigen prediction
              </a>
            </td>
          <td>
            Mofan Feng, Liangjie Liu, K. Su, Xianbin Su, Luming Meng, Zehua Guo, Dan Cao, Jiayi Wang, Guang He, Yi Shi
          </td>
          <td>2024-09-26</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Deficiency in the mismatch repair system (MMRd) causes microsatellite instability (MSI) in cancers and determines eligibility for immunotherapy. Here, we show that MMRd tumours harbour long-deletion signatures (≥2-5+ base pairs deleted in repetitive regions), which provide new insights into MSI evolution and enable sensitive MSI detection particularly in challenging clinical samples. Long deletions, accumulated through stepwise DNA slippage errors, are significantly more prevalent in metastatic MMRd tumours compared to primary tumours. Importantly, we show that long-deletion signatures harbour features that are distinct from background noise, making them robustly detectable even in shallow whole genome sequencing (sWGS, ∼0.1X coverage) of formalin-fixed samples. We constructed a machine learning classifier that uses these distinct features to detect Microsatellite Instability in LOw-quality (MILO) samples. MILO achieved 100% accuracy in detecting MSI in sWGS data with only 2%-15% tumour purity and demonstrated promise in identifying MMRd clones in precancerous intestinal lesions. We propose that MILO could be clinically used for the sensitive monitoring of MMRd cancer evolution from early to late stages, using minimal sequencing data from both archival and fresh-frozen samples with low tumour content. Significance Mutational signatures characterised by long deletions in repetitive genomic regions provide a sensitive route to detect and track MMRd clone evolution, even with low purity shallow whole genome sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce195ec34beddaa5854af1577dfad289507002f9" target='_blank'>
              Long deletion signatures in repetitive genomic regions track somatic evolution and enable sensitive detection of microsatellite instability
              </a>
            </td>
          <td>
            Qingli Guo, J. Househam, E. Lakatos, S. Nowinski, Ibrahim Al Bakir, Heather E Grant, V. Balarajah, Christine S. Hughes, Luis Zapata, Hemant M. Kocher, A. Sottoriva, A. Baker, Ville Mustonen, Trevor A. Graham
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Intratumoral heterogeneity is intrinsically comprised of molecular alterations of tumor cells and extrinsically from interconnections with microenvironments. This study explores the spatial heterogeneity of ovarian clear cell carcinoma (OCCC), a rare cancer with significance to East Asian women. We profiled 21 tumors from matched ovarian and metastatic sites with spatial transcriptomic (ST) platforms including GeoMx and Visium CytAssist. Three subclusters of OCCC tumor cells enriched in oxidative phosphorylation, epithelial-mesenchymal transition, and inflammation pathways were identified with preferential geospatial localizations in tumor centers and invasive margins/tumor-stromal interfaces. These ST subclusters correlated with epithelial-mesenchymal gradients and prognostic features in larger cohorts. Single-cell ST in CosMx identified 9 cancer cell populations with organ-specific tropism. Trajectory analysis indicated a NEAT1-high subclone having founder characteristics, progressively dominating the metastatic niche, displaying mesenchymal traits. This founder feature correlated with clinical outcomes in PanCancer datasets. Our findings highlight the adaptive and context-specific interactions between cancer cells and microenvironments within spatial complexities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8349570f5b5a5cc83b0effd91ea241eef3a2ce6c" target='_blank'>
              Spatial Transcriptomic Profiling Of Advanced Ovarian Clear Cell Carcinoma Reveals Intra-Tumor Heterogeneity In Epithelial-Mesenchymal Gradient
              </a>
            </td>
          <td>
            Thang Truong Le, Duncan Yi-Te Wang, Ko-Chen Chen, Sebastian Rui-Gu Yan, Pei-Yu Chu, Tuan Zea Tan, Yi-Chia Chiu, Jia-Yuh Sheu, , Jieru Ye, Ya-Ting Tai, Hsueh-Fen Juan, Feng-Chiao Tsai, , Lin-Hung Wei, 
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background R3HDM1, an RNA binding protein with one R3H domain, remains uncharacterized in terms of its association with tumor progression, malignant cell regulation, and the tumor immune microenvironment. This paper aims to fill this gap by analyzing the potential of R3HDM1 in diagnosis, prognosis, chemotherapy, and immune function across various cancers. Methods Data was collected from the Firehost database (http://gdac.broadinstitute.org) to obtain the TCGA pan-cancer queue containing tumor and normal samples. Additional data on miRNA, TCPA, mutations, and clinical information were gathered from the UCSC Xena database (https://xenabrowser.net/datapages/). The mutation frequency and locus of R3HDM1 in the TCGA database were examined using the cBioPortal. External validation through GEO data was conducted to assess the differential expression of R3HDM1 in different cancers. Protein expression levels were evaluated using the Clinical Proteomics Tumor Analysis Alliance (CPTAC). The differential expression of R3HDM1 was verified in lung adenocarcinoma cell lines and normal lung glandular epithelial cells via RT-qPCR. Cell migration and proliferation experiments were conducted by knocking down the expression of R3HDM1 in two lung adenocarcinoma cell lines using small interfering RNA. The biological role of R3HDM1 in pan-cancer was explored using the GSEA method. Multiple immune infiltration algorithms from the TIMER2.0 database was employed to investigate the correlation between R3HDM1 expression and the tumor immune microenvironment. Validation of transcriptome immune infiltration was based on 140 single-cell datasets from the TISCH database. The study also characterized a pan-cancer survival profile and analyzed the differential expression of R3HDM1 in different molecular subtypes. The relationship between R3HDM1 and drug resistance was investigated using four chemotherapy data sources: CellMiner, GDSC, CTRP and PRISM. The impact of chemicals on the expression of R3HDM1 was explored through the CTD database. Result The study revealed differential expression of R3HDM1 in various tumors, indicating its potential as an early diagnostic marker. Changes in somatic copy number (SCNA) and DNA methylation were identified as factors contributing to abnormal expression levels. Additionally, the study found that R3HDM1 expression is associated with clinical features, metabolic pathways, and important pathways related to metastasis and the immune system. High expression of R3HDM1 was linked to poor prognosis across different tumors and altered drug sensitivity. Furthermore, the expression of R3HDM1 showed significant correlations with immune modulatory molecules and biomarkers of lymphocyte subpopulation infiltration. Finally, the study highlighted four chemicals that could influence the expression of R3HDM1. Conclusion Overall, this study proposes that R3HDM1 expression is a promising biomarker for predicting the prognosis of cancer, especially lung adenocarcinoma, and the efficacy of immunotherapy, demonstrating the rationale for further exploration in the development of anti-tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379d6a67141459985f554b1a56ebc613e9c22b34" target='_blank'>
              Comprehensive pan-cancer analysis and experiments revealed R3HDM1 as a novel predictive biomarker for prognosis and immune therapy response
              </a>
            </td>
          <td>
            Jiawei Liu, Zhitong Bing, Junling Wang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dfb4f972dc9db981473d705bf7ba9bf069116ae" target='_blank'>
              Overexpression of TPD52L2 in HNSCC: prognostic significance and correlation with immune infiltrates
              </a>
            </td>
          <td>
            Min Lu, Haoyu Xia, Jing Xu, Zijun Liao, Yuwen Li, Hanwei Peng
          </td>
          <td>2024-10-07</td>
          <td>BMC Oral Health</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objective: This study was aimed at exploring a specific open region of chromatin in the peripheral blood mononuclear cells (PBMCs) of patients with breast cancer and evaluating its feasibility as a biomarker for diagnosing and predicting breast cancer prognosis. Methods: We obtained PBMCs from breast cancer patients and healthy people for the assay for transposase-accessible chromatin (ATAC) sequencing (n = 3) and obtained the GSE27562 chip sequencing data for secondary analyses. Through bioinformatics analysis, we mined the pattern changes for chromatin accessibility in the PBMCs of breast cancer patients. Results: A total of 1,906 differentially accessible regions (DARs) and 1,632 differentially expressed genes (DEGs) were identified via ATAC sequencing. The upregulated DEGs in the disease group were mainly distributed in the cells, organelles, and cell-intima-related structures and were mainly responsible for biological functions such as cell nitrogen complex metabolism, macromolecular metabolism, and cell communication, in addition to functions such as nucleic acid binding, enzyme binding, hydrolase reaction, and transferase activity. Combined with microarray data analysis, the following set of nine DEGs showed intersection between the ATAC and microarray data: JUN, MSL2, CDC42, TRIB1, SERTAD3, RAB14, RHOB, RAB40B, and PRKDC. HOMER predicted and identified five transcription factors that could potentially bind to these peak sites, namely NFY, Sp 2, GFY, NRF, and ELK 1. Conclusion: Chromatin accessibility analysis of the PBMCs from patients with early-stage breast cancer underscores its potential as a significant avenue for biomarker discovery in breast cancer diagnostics and treatment. By screening the transcription factors and DEGs related to breast cancer, this study provides a comprehensive theoretical foundation that is expected to guide future clinical applications and therapeutic developments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3171ecf6420e675e47a99c9a77102cb706140c79" target='_blank'>
              Analysis of chromatin accessibility in peripheral blood mononuclear cells from patients with early-stage breast cancer
              </a>
            </td>
          <td>
            Longjie Xia, Jiamin Lu, Yixuan Qin, Runchun Huang, Fanbiao Kong, Yu Deng
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The most common cancer in Korea is thyroid cancer, followed by lung cancer, and the most common cancer in men is lung cancer. The cancer with the highest mortality rate is lung cancer. Recently, the precision medical approach is becoming a major treatment strategy for lung cancer. For precision medical approach, the usefulness of next-generation sequencing (NGS) analysis targeting circulating tumor DNA (ctDNA) is being emphasized. Therefore, in this study, NGS analysis was performed on blood and tissues of advanced lung cancer patients to compare and analyze genetic mutation patterns to confirm the usefulness of ctDNA NGS assay.



 A total of 101 patients with advanced lung cancer (stage 3 or 4) were recruited, and ctDNA NGS assays were performed on their plasma using a Pan100 panel (Dxome). Tissues from primary lung cancer, metastatic lymph nodes, or metastatic organs of 22 patients were analyzed using ONCOaccuPanel (NGeneBio). All variants were classified into four tiers based on the level of clinical significance, according to the standards and guidelines established by the Association for Molecular Pathology, American Society of Clinical Oncology, and College of American Pathologists.



 In the ctDNA NGS assay of 101 patients, 70 (69.3%) harbored 154 tier 1 or 2 variants. Mutations in TP53 (45, 29.2%) and EGFR (23, 14.9%) were the most frequently detected, followed by mutations in RB1, CHEK2, ATM, JAK2, ATR, DNMT3A, and KRAS. Among the 22 patients whose tissue was examined, tier 1 or 2 variants were detected in both the plasma and tissue of 14 patients and only in the tissue of six patients. All genes for these variants were targeted using both methods. Identical variants were detected in the plasma and tissue of six of the 14 patients, and discordant results were observed in the remaining eight patients. Twenty copy number variations were detected in 13 of the 101 patients, with copy number gain of EGFR being the most common (6, 30%). Fusion genes were detected in five of the 101 patients, two of which were not detected in the tissue. Overall, 74 (73.3%) of the 101 patients had positive ctDNA NGS assay results.



 Although the NGS assay with ctDNA is less sensitive than that with tissue DNA, some genetic alterations are detected only in ctDNA NGS assays; therefore, it is useful to apply the ctDNA NGS assay as a complement.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8e0e6eed278b41bf977b6ec4d0bb9e829c01bb" target='_blank'>
              B-191 Genomic Profile of Circulating Tumor DNA in Advanced Lung Cancer: Comparison with Tissue DNA
              </a>
            </td>
          <td>
            J. Park, Y. Choi, S. Jeon, S. Kee, E. Na, M. Shin
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, M. Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal disease with limited therapies. In our screening using patient-derived organoids, we identified cyclin-dependent kinase 12 (CDK12) as a promising target in a subset of PDAC. Both pharmacological and genetic inhibition of CDK12 exerted potent antitumor activity in vitro and in vivo, inducing double-strand DNA breaks and apoptosis. CDK12 is a member of cyclin-dependent kinases that is involved in gene expression regulations at multiple levels. Recently, CDK12 has gained attention as a therapeutic target in several types of cancers including triple-negative breast cancer and Ewing sarcoma. However, little is known about the impact and possibility of CDK12 inhibition in pancreatic cancer treatment. As the lack of specific inhibitors that selectively target CDK12, studying CDK12 has long been challenging. To overcome this, we used CRISPR-Cas9 mediated genome editing to generate FKBP12F36V (dTAG) knock-in PDAC cell lines, enabling rapid elimination of endogenous CDK12 using a degrader against dTAG (dTAG-13). Quantitative-image-based cytometry and time lapse imaging showed that CDK12 ablation induces mitotic arrest, DNA damage and apoptosis (mitotic catastrophe). Specifically, loss of CDK12 kinase activity in the S phase was sufficient to slow DNA replication speed, leading to spindle assembly checkpoint activation and mitotic catastrophe. Importantly, cells that escaped mitotic death accumulate micronuclei, triggering immune response signaling that is mediated by the cGAS-STING-type I interferon pathway. Our results suggest that CDK12 is a promising target in a subset of PDAC patients, where CDK12 inhibition exerts antitumor activity both in cell autonomous and non-cell autonomous ways through inducing mitotic catastrophe as well as triggering immune response.
 Citation Format: Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro. CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A084.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632d48b78e0c9131037bdfa10d93bd71182ca006" target='_blank'>
              Abstract A084: CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer
              </a>
            </td>
          <td>
            Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Discovering the cell-of-origin harboring the initial driver mutation provides a fundamental basis for understanding tumor evolution and development of new treatments. For isocitrate dehydrogenase (IDH)-mutant gliomas – the most common malignant primary brain tumors in adults under 50 – the cell-of-origin remains poorly understood. Here, using patient brain tissues and genome-edited mice, we identified glial progenitor cells (GPCs), including oligodendrocyte progenitor cells (OPCs), as the glioma-originating cell type harboring the IDH mutation as the initial driver mutation. We conducted comprehensive deep sequencing, including droplet digital PCR and deep panel and amplicon sequencing to 128 tissues from 62 patients (29 IDH-mutant gliomas and 33 IDH-negative controls) comprising tumors, normal cortex or normal subventricular zone (SVZ), and blood. Surprisingly, low-level IDH mutation was found in the normal cortex away from the tumor in 38.5% (10 of 26) of IDH-mutant glioma patients, whereas no IDH mutation was detected in the normal SVZ. Furthermore, by analyzing cell-type–specific mutations, the direction of clonal evolution, the single-cell transcriptome from patient brains and novel mouse model of IDH-mutant glioma arising from mutation-carrying OPCs, we determined that GPCs, including OPCs, harboring the initial driver mutation are responsible for the development and evolution of IDH-mutant gliomas. In summary, our results demonstrate that GPCs containing the IDH mutation are the cells-of-origin harboring the initial driver mutation in IDH-mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c3e71571b3576b0df5e766d465ee9d5ba82b819" target='_blank'>
              IDH-mutant gliomas arise from glial progenitor cells harboring the initial driver mutation
              </a>
            </td>
          <td>
            Jung Won Park, Jiehoon Kwak, K. Kim, Saehoon Jung, Chang Hyun Nam, Hyun Jung Kim, Sang Mee Lee, Ji-Hyung Park, Jihwan Yoo, J. Shim, Chungyeul Kim, Sangjeong Ahn, Stefan Pusch, Andreas von Deimling, Jong Hee Chang, Se Hoon Kim, Young Seok Ju, S. Kang, Jeong Ho Lee
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Long interspersed elements type 1 (LINE-1s or L1s) are highly abundant non-LTR retrotransposons in mammalian genomes. Apart from an evolutionary role in altering the genomic landscape and gene expression, recent discoveries have revealed the scale and impact of active movement by L1, both in the germline and in somatic niches (e.g., the brain and many cancers). As transcriptional activation is a prerequisite for L1 retrotransposition, it is important to understand how L1 transcription is regulated. L1 transcription is controlled by an internal promoter contained in the 5’ untranslated region (5’UTR) of a full-length element. It has been shown that transcription start sites (TSSs) of human L1 are clustered at the beginning of its 5’UTR and the precision of transcription initiation is controlled by transcription factor YY1. The laboratory mouse is an essential model for investigating L1 retrotransposition and the associated consequences in vivo. However, mouse L1 5’UTRs are distinctly different from human L1’s: the former are organized into tandem repeats called monomers and separated from the open reading frames by a nonmonomeric sequence called tether. Unlike the human L1 promoter, transcription initiation and its regulation in mouse L1 subfamilies remain unknown. In this study, we streamlined the cloning of different types of mouse L1 promoters. TSSs were determined using RNA ligation-mediated 5’ rapid amplification of cDNA ends (RLM-5’RACE) incorporating a highly efficient reverse transcriptase enzyme followed by Nanopore sequencing. In the context of two monomer promoter reporter construct, transcription could initiate from either monomer or the tether sequence. While mouse L1 TSSs are distributive, a higher number of transcripts were initiated from the inner monomer. Surprisingly, a significant fraction of transcripts was also initiated from the tether sequence. In addition, we showed that YY1 transcriptionally activates mouse L1 5’UTR and is responsible for the synergy between the monomers. As transcription initiation is a major determinant for the proportion of active L1 loci in the next round of retrotransposition, these findings have important implications on L1 biology in the mouse model.
 Citation Format: Karabi Saha, Wenfeng An. Transcription initiation and its regulation in mouse L1s [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a796751004ae7aa371fe2f625b26733c662788" target='_blank'>
              Abstract A013: Transcription initiation and its regulation in mouse L1s
              </a>
            </td>
          <td>
            Karabi Saha, Wenfeng An
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb02af394c6a6d8f8817aaa11d8e343dfd55fae" target='_blank'>
              Chromosome 8q24 amplification associated with human hepatocellular carcinoma predicts MYC/ZEB1/MIZ1 transcriptional regulation
              </a>
            </td>
          <td>
            Joeffrey J. Chahine, Saniya S. Davis, Sumeyye Culfaci, Bhaskar V Kallakury, P. Tuma
          </td>
          <td>2024-10-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, R. Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Macrophages maintain hematopoietic stem cell (HSC) quality by assessing cell surface Calreticulin (Calr), an “eat-me” signal induced by reactive oxygen species (ROS). Using zebrafish genetics, we identified Beta-2-microglobulin (B2m) as a crucial “don’t eat-me” signal on blood stem cells. A chemical screen revealed inducers of surface Calr that promoted HSC proliferation without triggering ROS or macrophage clearance. Whole-genome CRISPR-Cas9 screening showed that Toll-like receptor 3 (Tlr3) signaling regulated b2m expression. Targeting b2m or tlr3 reduced the HSC clonality. Elevated B2m levels correlated with high expression of repetitive element (RE) transcripts. Overall, our data suggest that RE-associated double-stranded RNA could interact with TLR3 to stimulate surface expression of B2m on hematopoietic stem and progenitor cells. These findings suggest that the balance of Calr and B2m regulates macrophage-HSC interactions and defines hematopoietic clonality. Editor’s summary During embryonic development, macrophages interact with blood stem cells, removing those exhibiting high levels of cellular stress. The surviving hematopoietic stem and progenitor cells (HSPCs) can proliferate and give rise to blood cell linages in adults. Pessoa Rodrigues et al. used zebrafish models and mammalian systems to investigate signals within HSPCs that could prevent macrophages from engulfing them. Molecules associated with innate immune nucleic acid sensing promoted the expression of Beta-2-microglobulin on the surface of HSPCs to increase their survival when they interacted with macrophages. Impairing this pathway decreased the number of distinct stem cell clones in adult zebrafish. —Sarah H. Ross  INTRODUCTION Hematopoietic stem and progenitor cells (HSPCs) produce mature blood cells throughout life. During this process, HSPCs engage in interactions with the variety of cells present in the marrow niche, including macrophages. These macrophages mediate a myriad of processes by releasing cytokines and chemokines and patrolling to remove stressed, dead, or aging cells. In this manner, macrophages contribute to the maintenance of tissue homeostasis. During hematopoiesis, macrophages ensure the quality of normal blood stem cells and determine the number of hematopoietic clones that participate in adult hematopoiesis. They then either engulf a stem cell completely, referred to as “dooming,” or capture portions of the stem cell’s cellular material, referred to as “grooming.” After grooming, the stem cell continues to divide, whereas the doomed HSPC clone is eliminated. RATIONALE Past work has shown that the HSPC-macrophage interaction is mediated by an “eat-me” signal, calreticulin (Calr), on the surface of HSPCs. Surface Calr levels are increased in stem cells with higher levels of reactive oxygen species (ROS), and this triggers macrophages to engulf and kill stem cells that have high amounts of stress-activated proteins. However, the specific molecular cues that regulate the dooming versus grooming behavior are still unknown. RESULTS To investigate the cues mediating grooming versus dooming, we screened a panel of 1200 bioactive small molecules in human cells and identified 93 compounds that robustly increased surface CALR in a dose-dependent manner. Of these compounds, 22 also facilitated interactions between macrophages and stem cells in zebrafish. Compounds that depended on ROS to increase CALR showed higher dooming ratios and, conversely, animals treated with ROS-independent compounds exhibited a higher probability of grooming events despite increased CALR and increased macrophage–stem cell interactions. To investigate the signals involved in interactions under ROS-independent conditions, we undertook a genome-wide screen in human cells. We identified Toll-like receptor 3 (TLR3) as a CALR inducer in the “don’t eat me” context. Next, we found that TLR3 expression was required for the expression of the “don’t eat me” signal Beta-2-microglobulin (B2M). Antibody staining and studies of a b2m knockout zebrafish line showed that b2m on stem cells was required for preventing the macrophage dooming. To determine whether the increased dooming observed in the b2m mutant affected clonal dominance, we generated mosaic deletions using the zebrabow color barcoding system (TWISTR) as a lineage tracer. Mosaic deletion of b2m reduced the number of myeloid clones but increased clonal dominance. The dominant clones in these fish were wild type for b2m that were resistant to dooming and overtook the adult marrow. TLR3 signaling is activated by double-stranded RNA, which can sometimes be found in cells due to the expression of transcripts associated with repetitive elements (REs), including long terminal repeats (LTRs) and endogenous retroviruses. RE expression in HSPCs positively correlated with elevated levels of B2m. Moreover, overexpressing ltr4 in zebrafish led to higher HSPC proliferation, indicative of decreased dooming. Impairing the TLR3-B2M pathway by disrupting the expression of TLR3, IRF3, or B2M expression decreased the number of blood cell clones in zebrafish. Using the DNA methyltransferase inhibitor CM272 to promote up-regulation of REs amplified the proliferation of already established HSPC clones in zebrafish embryos. CONCLUSION Our findings support a model in which endogenous TLR3 ligands lead to expression of b2m, thereby providing a blockade against macrophage-induced dooming and determining hematopoietic stem cell clonality by monitoring stem cell quality with macrophages. The balance of Calr and B2m governs the number of long-lived HSC clones that contribute to the adult blood system through macrophage-mediated quality assurance. This sheds light on the intricate interplay between HSPC clonal expansion and immune-regulated cell clearance. We propose that this protective mechanism may also operate in adulthood in response to environmental stress, such as during infections or in clonal stem cell disorders. Manipulating the levels of “don’t eat-me” and “eat-me” signals may have important therapeutic implications for immune therapy by harnessing the macrophage-selective removal of a mutated stem cell clone. ROS and TLR3 signaling mediates the “eat-me” and “don’t eat me” signals that determine stem cell clonal selection by macrophages. Shown is a schematic overview of the proposed model showing the balance between the Calr-mediated “eat-me” and B2m-mediated “don’t eat-me” signals that influence macrophage behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da253e73cd7f41de67f6f4bba52c4af27467af93" target='_blank'>
              Transcripts of repetitive DNA elements signal to block phagocytosis of hematopoietic stem cells
              </a>
            </td>
          <td>
            Cecilia Pessoa Rodrigues, Joseph M. Collins, Song Yang, Catherine Martinez, Ji Wook Kim, Chhiring Lama, Anna S. Nam, Clemens Alt, Charles Lin, Leonard I. Zon
          </td>
          <td>2024-09-13</td>
          <td>Science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Atypical Teratoid Rhabdoid Tumor (ATRT) is a highly aggressive pediatric brain tumor and characterized by SMARCB1 deletion. SMARCB1 is a member of the SWI/SNF chromatin remodeling complex that is responsible for determining cellular pluripotency and lineage commitment. We performed an RNAi-based functional genomic screen to identify key epigenetic regulators that co-operate with SMARCB1 loss and contribute to tumorigenesis in ATRT. 406 genes associated with epigenetic regulation in patient derived ATRT cell lines were targeted. This screen identified BMI1, a component of the Polycomb Repressive Complex 1 (PRC1), as one of the top targets essential for ATRT cell growth. Using RNA-seq and ChIP-Seq analysis we demonstrated that BMI1 co-operates with SMARCB1 deletion to suppress transcription of pro-differentiation pathways and promote self-renewal of tumor stem cells. In SMARCB1 deficient cells BMI1 forms a partial PRC1 complex devoid of DNA binding components. We found that genetic inactivation of BMI1 suppresses ATRT cell clonogenicity in vitro and decreases S-phase length. By Doxycycline-inducible SMARCB1 re-expression system we revealed activation of proapoptotic proteins p16 and p21. With immunoprecipitation for BMI1, followed by mass spectrometry analysis we demonstrated that re-expression of SMARCB1 activates two PRC1 chromatin localizing components CBX4 and CBX8. Moreover, inhibition of BMI1 significantly prolongs mice survival and decreases tumor size as evaluated by IVIS imaging system and T2-MRI. Thus, we demonstrated that SMARCB1 deletion results in reprograming of BMI1 chromatin occupancy away from lineage specification by altering the components of the PRC1 complex and that PRC1/BMI1 inhibition is a novel therapeutic approach in ATRT.
 Citation Format: Irina Irina Alimova, Gillian Murdock, Angela M. Pierce, Dong Wang, Nathan Dahl, Sujatha Venkataraman, Rajeev Vibhakar. PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d643026696b6226900916830deb821d92b3f76f" target='_blank'>
              Abstract A080 PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Irina Irina Alimova, Gillian Murdock, Angela M Pierce, Dong Wang, N. Dahl, S. Venkataraman, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082a7d22b12accfd1876717033920e6fb8f44c27" target='_blank'>
              Understanding m6A changes in chromophobe renal cell carcinoma and predicting patient outcomes survival
              </a>
            </td>
          <td>
            Zhigang Chen, Junbo Yang, Wei Zhang, Yang Qian, Nan Zhang, Zixin Chen, Min Lu, L. Ge, Cheng Liu, X. Tian, Guifang Jia, Lulin Ma, Baoguo Li
          </td>
          <td>2024-09-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Eukaryotic chromosomes segregate faithfully prior to nuclear division to ensure genome stability. If segregation becomes defective, the chromosome copy number of the cell may alter leading to aneuploidy and/or polyploidy, both common hallmarks of cancers. In eukaryotes, aurora kinases regulate chromosome segregation during mitosis and meiosis, but their functions in the divergent, single-celled eukaryotic pathogen Trypanosoma brucei are less understood. Here, we focused on one of three aurora kinases in these parasites, TbAUK3, a homologue of the human aurora kinase AURKC, whose functions are primarily restricted to meiosis. We show that RNAi targeted depletion of TbAUK3 correlates with nuclear segregation defects, reduced proliferation, and decreased DNA synthesis, suggestive of a role for TbAUK3 during mitotic, not meiotic, chromosome segregation. Moreover, we uncover a putative role for TbAUK3 during the parasite’s response to DNA damage since we show that depletion of TbAUK3 enhances DNA instability and sensitivity to genotoxic agents. HIGHLIGHTS The C-terminus of TbAUK3 is disordered TbAUK3 depletion coincides with nuclear segregation defects Depletion of TbAUK3 enhances DNA instability">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125d96f157b6efd3c7ef3b8d6665537e416170cc" target='_blank'>
              AUK3 is required for faithful nuclear segregation in the bloodstream form of Trypanosoma brucei
              </a>
            </td>
          <td>
            J. A. Black, B. Poulton, B.S. Gonzaga, A. Iskantar, L. Tosi, R. McCulloch
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Despite recent advances in the treatment of pancreatic adenocarcinoma (PDAC), the median survival remains <12 months. Patients typically present with late-stage disease and are often unable to tolerate drug combination regimens. Large-scale genomic approaches categorise PDAC into two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal) subtypes. Although it is well-described that PDAC harbours gene mutations (i.e. KRAS G12D, p53 R172H) that initiate the disease, little is known about the influence of lineage-defining Transcription Factors (TF) on PDAC subtypes and their contribution to tumor progression. We took an unbiased approach to discover driving TFs in PDAC, we performed ChIP-seq of the active enhancer mark H3K27Ac from surplus tissue following resection of primary PDAC tumors (n=6) and normal adjacent tissue (n=4). Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements were HNF motifs (HNF4G & HNF4A) and Forkhead (i.e., FOXA1), confirming that these two classes of TFs constitute the lineage-defining factors in these clinical samples. Investigation of publicly available clinical datasets revealed HNF4G to be amplified in 9% of PDAC patients. Using Rapid Immunoprecipitation and Mass Spectrometry of Endogenous protein complexes (RIME), our method for unbiased discovery of endogenous protein complexes, we investigated the fundamental differences in the FOXA1 interactome between models of classical and squamous subtypes. HNF4A and HNF4G were only seen as FOXA1-interactors in models of the classical subtype. HNF4G Knockout (KO) clones injected orthotopically into the pancreas of mice, showed a significant increase (p-value<0.0006) in median survival and slower tumor growth compared to Control (Cas-9) (p-value <0.0001). HNF4G-KO dependent genes were specifically expressed in primary tumors (with decreased EMT signalling) versus metastatic samples (PanCRux dataset), implying that the HNF4G target genes were mostly limited to primary tumor contexts. We found Protein Arginine Methylate Transferase (PRMT1) to be a significant and reproducible interactor of FOXA1 and HNF4G. We observed a significant global reduction in PRMT1 chromatin binding in both the HNF4G-KO and FOXA1 depleted cells compared to the control. This data suggests a dependency on both HNF4G and FOXA1 for recruitment of PRMT1 to chromatin. We find HNF4G to be the key driver of classical subtype primary disease, in part by recruiting the methyltransferase PRMT1 to regulatory regions revealing a therapeutically exploitable opportunity. A molecular switch occurs in advanced disease, whereby HNF4G expression decreases, unmasking FOXA1’s transcriptional activity. De-repressed FOXA1 drives late-stage disease, by orchestrating metastasis-specific enhancer-promoter loops to direct metastatic gene regulation. We suggest the existence of stage-dependent TF activity, triggered by molecular compartmentalisation that mediates multistage progression of PDAC.
 Citation Format: Shalini Vasudev Rao, Lisa V Young, Danya Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll. Transcription factor switching drives progression of the classical subtype of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e663bb2d35a48fcc777219947b4d5e9554b7ef19" target='_blank'>
              Abstract C071: Transcription factor switching drives progression of the classical subtype of pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Three-dimensional chromatin control of eukaryotic transcription is pivotal for regulating gene expression. This additional layer of epigenetic regulation is also utilized by DNA viruses, including herpesviruses. Dynamic, spatial genomic organization often involves looping of chromatin anchored by host-encoded CCCTC-binding factor (CTCF) and other factors, which control crosstalk between promoters and enhancers. Herein, we review the contribution of CTCF-mediated looping in regulating transcription during herpesvirus infection, with a specific focus on the betaherpesvirus, human cytomegalovirus (HCMV).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ca71344a757b4c351d6ffdff351925d0ab026" target='_blank'>
              Loopy virus or controlled contortionist? 3D regulation of HCMV gene expression by CTCF-driven chromatin interactions.
              </a>
            </td>
          <td>
            Ian J. Groves, Christine M. O’Connor
          </td>
          <td>2024-08-30</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) describes the process where distinct epigenetic states may be transmitted between generations, resulting in stable gene expression and phenotypic differences between individuals that persist independently of DNA sequence variation. Chromatin modifications have been demonstrated as important in TEI, however, the extent to which they require other signals to establish and maintain epigenetic states is still unclear. Here we investigate whether small non-coding RNAs contribute to different epigenetic states of the Fab2L transgene in Drosophila triggered by transient chromatin contacts, which requires Polycomb complex activity to deposit the H3K27me3 modification for long-term TEI. Using mutants deficient in known small non-coding RNAs, high throughput sequencing, investigation of chromatin conformation and gene expression analysis we demonstrate that small non-coding RNAs do not contribute directly to initiation or maintenance of silencing. However, we uncover an indirect role for microRNA expression in transgene silencing through effects on Polycomb complex expression. Additionally, we show that a commonly used marker gene, Stubble (Sb), affects Polycomb complex expression, which may be important in interpreting experiments assaying Polycomb function in Drosophila development. By ruling out a plausible candidate for TEI at the Fab2L transgene our work highlights the variability in different modes of TEI across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be548511a3f566003319963d618b6d4697f22992" target='_blank'>
              Polycomb-mediated transgenerational epigenetic inheritance of Drosophila eye colour is independent of small RNAs
              </a>
            </td>
          <td>
            Maximilian H. Fitz-James, Poppy Sparrow, Christopher Paton, P. Sarkies
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e3c1ad704a586aa01f4f3735cd6a0421899426" target='_blank'>
              Rise and SINE: roles of transcription factors and retrotransposons in zygotic genome activation.
              </a>
            </td>
          <td>
            Pavel Kravchenko, Kikuë Tachibana
          </td>
          <td>2024-10-02</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary In tumorigenesis, aberrant DNA methylation may be an earlier and stronger modifier of gene expression than mutations. Herein, 128 serous ovarian tumors were analyzed, including borderline ovarian tumors (BOTS) with (BOT.V600E) and without (BOT) the BRAF V600E mutation, low-grade (lg), and high-grade (hg) ovarian cancers (OvCa). The methylome of the samples was profiled with Infinium MethylationEPIC microarrays. Global, genome-wide hypomethylation positively correlated with the increasing aggressiveness of tumors, being the strongest in hgOvCa. Remarkably, the ten most significant differentially methylated regions (DMRs) in the genome, discriminating BOT from lgOvCa, encompassed the MHC region on chromosome 6. We also identified hundreds of DMRs potentially useful as predictive biomarkers in BOTS and hgOvCa. DMRs with the best discriminative capabilities overlapped the following genes: BAIAP3, IL34, WNT10A, NEU1, SLC44A4, and HMOX1, TCN2, PES1, RP1-56J10.8, ABR, NCAM1, RP11-629G13.1, AC006372.4, NPTXR in BOTS and hgOvCa, respectively. By identifying potential biomarkers, this study might improve ovarian tumor outcome. Abstract Background: Changes in DNA methylation patterns are a pivotal mechanism of carcinogenesis. In some tumors, aberrant methylation precedes genetic changes, while gene expression may be more frequently modified due to methylation alterations than by mutations. Methods: Herein, 128 serous ovarian tumors were analyzed, including borderline ovarian tumors (BOTS) with (BOT.V600E) and without (BOT) the BRAF V600E mutation, low-grade (lg), and high-grade (hg) ovarian cancers (OvCa). The methylome of the samples was profiled with Infinium MethylationEPIC microarrays. Results: The biggest number of differentially methylated (DM) CpGs and regions (DMRs) was found between lgOvCa and hgOvCa. By contrast, the BOT.V600E tumors had the lowest number of DM CpGs and DMRs compared to all other groups and, in relation to BOT, their genome was strongly downmethylated. Remarkably, the ten most significant DMRs, discriminating BOT from lgOvCa, encompassed the MHC region on chromosome 6. We also identified hundreds of DMRs, being of potential use as predictive biomarkers in BOTS and hgOvCa. DMRs with the best discriminative capabilities overlapped the following genes: BAIAP3, IL34, WNT10A, NEU1, SLC44A4, and HMOX1, TCN2, PES1, RP1-56J10.8, ABR, NCAM1, RP11-629G13.1, AC006372.4, NPTXR in BOTS and hgOvCa, respectively. Conclusions: The global genome-wide hypomethylation positively correlates with the increasing aggressiveness of ovarian tumors. We also assume that the immune system may play a pivotal role in the transition from BOTS to lgOvCa. Given that the BOT.V600E tumors had the lowest number of DM CpGs and DMRs compared to all other groups, when methylome is considered, such tumors might be placed in-between BOT and OvCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c468b8eb0505c51d6a927efad191c6a8ba7d2c09" target='_blank'>
              The Diversity of Methylation Patterns in Serous Borderline Ovarian Tumors and Serous Ovarian Carcinomas
              </a>
            </td>
          <td>
            L. A. Szafron, R. Iwanicka-Nowicka, Piotr Sobiczewski, Marta Koblowska, A. Dansonka-Mieszkowska, J. Kupryjańczyk, L. Szafron
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for assessing chromatin stability in healthy and sick dogs (with squamous cell carcinoma). The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable a more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. Abstract Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Deleterious germline DDX41 variants constitute the most common inherited predisposition disorder linked to myeloid neoplasms (MNs). The role of DDX41 in hematopoiesis and how its germline and somatic mutations contribute to MNs remain unclear. Here we show that DDX41 is essential for erythropoiesis but dispensable for the development of other hematopoietic lineages. Using stage-specific Cre models for erythropoiesis, we reveal that Ddx41 knockout in early erythropoiesis is embryonically lethal, while knockout in late-stage terminal erythropoiesis allows mice to survive with normal blood counts. DDX41 deficiency induces a significant upregulation of G-quadruplexes (G4), noncanonical DNA structures that tend to accumulate in the early stages of erythroid precursors. We show that DDX41 co-localizes with G4 on the erythroid genome. DDX41 directly binds to and dissolves G4, which is significantly compromised in MN-associated DDX41 mutants. Accumulation of G4 by DDX41 deficiency induces erythroid genome instability, defects in ribosomal biogenesis, and upregulation of p53. However, p53 deficiency does not rescue the embryonic death of Ddx41 hematopoietic-specific knockout mice. In parallel, genome instability also activates the cGas-Sting pathway, which is detrimental to survival since cGas-deficient and hematopoietic-specific Ddx41 knockout mice are viable without detectable hematologic phenotypes, although these mice continue to show erythroid ribosomal defects and upregulation of p53. These findings are further supported by data from a DDX41 mutated MN patient and human iPSC-derived bone marrow organoids. Our study establishes DDX41 as a G4 dissolver, essential for erythroid genome stability and suppressing the cGAS-STING pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5f497f73b0f90438a8c4db6703bd3c7df40581" target='_blank'>
              DDX41 dissolves G-quadruplexes to maintain erythroid genome integrity and prevent cGAS-mediated cell death
              </a>
            </td>
          <td>
            Honghao Bi, Kehan Ren, Pan Wang, Ermin Li, Xu Han, Wen Wang, Jing Yang, Inci Aydemir, Kara Tao, Lucy Godley, Yan Liu, Vipul Shukla, Elizabeth Bartom, Yuefeng Tang, Lionel Blanc, M. Sukhanova, Peng Ji
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Disclosure: A. Sanghi: None. S. Wu: None. T. Chou: None. L.A. Orloff: None. M. Kasowski: None. M. Snyder: None. Although genetic mutations are causal in human cancers, the contribution of chromatin changes to cancer onset and progression remains less well understood. Multi-omics profiling of human tumors can provide insight into how alterations in chromatin structure are propagated through the pathway of gene expression to result in cancer signaling. We applied multi-omics profiling of 36 human thyroid cancer primary tumors, metastases, and patient-matched normal tissue. Over 80% of the tumors harbored BRAFV600E mutations, and the remaining tumors mostly had activating mutations in the MAPK pathway. Through quantification of chromatin accessibility associated with active transcription units and global protein expression, we identify local enhancers that are highly correlated with coordinated RNA and protein expression. The enhancers are predicted to be actively bound by MAPK transcription factor, indicating they are under MAPK control. These cancer-specific enhancers associated with upregulation of lysosomal peptidase expression and dipeptide abundance in tumors. We show that upregulated dipeptides stimulate MAPK activity, and BRAF inhibition results in loss of lysosomal function in mutants. Perturbation of lysosomal peptidases and dipeptide abundance decreases MAPK activity only in mutant cells compared to wildtype. These analyses suggest that BRAF thyroid tumors integrate MAPK-driven epigenetic regulation with lysosomal metabolite signaling in a positive feedback circuit. Using local cancer-specific epigenetic features, we isolated key cancer signaling, making way for potential targets for cancer therapeutics. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032e66998348d599db74b255c1962f11618e9fbe" target='_blank'>
              7948 Integration of Epigenomics and Metabolomics Reveals New Signaling in Aggressive Thyroid Cancer
              </a>
            </td>
          <td>
            A. Sanghi, S. Wu, T. Chou, L. A. Orloff, M. Kasowski, M. Snyder
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The integrity of the actively transcribed genome against helix-distorting DNA lesions relies on a multilayered cellular response that enhances Transcription-Coupled Nucleotide Excision Repair (TC-NER). When defective, TC-NER is causatively associated with Cockayne-Syndrome (CS), a rare severe human progeroid disorder. Although the presence of unresolved transcription-blocking lesions is considered a driver of the aging process, the molecular features of the transcription-driven response to genotoxic stress in CS-B cells remain largely unknown. Here, an in-depth view of the transcriptional and associated chromatin dynamics that occur in CS-B cells illuminates the role of CSB therein. By employing high-throughput genome-wide approaches, we observed that absence of a functional CSB protein results in a delay in transcription progression, more positioned +1 nucleosomes, and less dynamic chromatin structure, compared to normal cells. We found that early after exposure to UV, CS-B cells released RNA polymerase II (RNAPII) from promoter-proximal pause sites into elongation. However, the magnitude of this response and the progression of RNAPII were reduced compared to normal counterparts. Notably, we detected increased post-UV retainment of unprocessed nascent RNA transcripts and chromatin-associated elongating RNAPII molecules. Contrary to the prevailing models, we found that transcription initiation is operational in CS-B fibroblasts early after UV and that chromatin accessibility showed a marginal increase. Our study provides robust evidence for the role of CSB in shaping the transcription and chromatin landscape both in homeostasis and in response to genotoxic insults, which is independent of its known role in TC-NER, and which may underlie major aspects of the CS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966a15785f4d1763d145786faf0eda45c3c2df7f" target='_blank'>
              Cockayne syndrome B protein is implicated in transcription and associated chromatin dynamics in homeostatic and genotoxic conditions.
              </a>
            </td>
          <td>
            Anastasios Liakos, Katerina Z Ntakou-Zamplara, Nelina Angelova, Dimitris Konstantopoulos, Anna-Chloe Synacheri, Zoi Spyropoulou, Iason A Tsarmaklis, Despoina Korrou-Karava, Georgios Nikolopoulos, Matthieu D. Lavigne, Maria Fousteri
          </td>
          <td>2024-10-06</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The DEK chromatin remodeling protein was previously shown to confer oncogenic phenotypes to human and mouse mammary epithelial cells using in vitro and knockout mouse models. However, its functional role in normal mammary gland epithelium remained unexplored. We developed two novel mouse models to study the role of Dek in normal mammary gland biology in vivo. Mammary gland-specific Dek over-expression in mice resulted in hyperproliferation of cells that visually resembled alveolar cells, and a transcriptional profile that indicated increased expression of cell cycle, mammary stem/progenitor, and lactation-associated genes. Conversely, Dek knockout mice exhibited an alveologenesis or lactation defect, resulting in dramatically reduced pup survival. Analysis of previously published single-cell RNA-sequencing of mouse mammary glands revealed that Dek is most highly expressed in mammary stem cells and alveolar progenitor cells, and to a lesser extent in basal epithelial cells, supporting the observed phenotypes. Mechanistically, we discovered that Dek is a modifier of Ezh2 methyltransferase activity, upregulating the levels of histone H3 trimethylation on lysine 27 (H3K27me3) to control gene transcription. Combined, this work indicates that Dek promotes proliferation of mammary epithelial cells via cell cycle deregulation. Furthermore, we report a novel function for Dek in alveologenesis and histone H3 K27 trimethylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96378ddd56b750c6cb56cd86a3b8644a37da590" target='_blank'>
              The chromatin remodeler DEK promotes proliferation of mammary epithelium and is associated with H3K27me3 epigenetic modifications
              </a>
            </td>
          <td>
            Megan Johnstone, Ashley Leck, Taylor Lange, Katherine Wilcher, Miranda S. Shephard, Aditi Paranjpe, Sophia Schutte, Susanne Wells, Ferdinand Kappes, Nathan Salomonis, L. P. Privette Vinnedge
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42bb491bf59a8058f50cf3417e3f4f9a441fc86" target='_blank'>
              Cancer treatment monitoring using cell-free DNA fragmentomes
              </a>
            </td>
          <td>
            Iris van 't Erve, B. Alipanahi, K. Lumbard, Z. Skidmore, Lorenzo Rinaldi, Laurel K Millberg, J. Carey, B. Chesnick, S. Cristiano, Carter Portwood, Tony Wu, E. Peters, K. Bolhuis, Cornelis J A Punt, Jennifer Tom, P.B. Bach, N. Dracopoli, Gerrit A Meijer, Robert B Scharpf, V. Velculescu, R. Fijneman, A. Leal
          </td>
          <td>2024-10-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Long-read whole-genome sequencing (lrWGS) enhances haplotyping by providing more phasing information per read compared to short-read sequencing. However, its use for single-cell haplotype phasing remains underexplored. This proof-of-concept study examines lrWGS data from single cells for small variant (SNV and indel) calling and haplotyping using the Genome in a Bottle (GIAB) Ashkenazi trio. lrWGS was performed on single-cell (1 cell) and multi-cell (10 cells) samples from the offspring. Chromosome-length haplotypes were obtained by leveraging both long reads and pedigree information. These haplotypes were further refined by replacing them with matched parental haplotypes. In single-cell and multi-cell samples, 92% and 98% of heterozygous SNVs, and 74% and 78% of heterozygous indels were accurately haplotyped. Applied to human embryos for preimplantation genetic testing (PGT), lrWGS demonstrated 100% consistency with array-based methods for detecting monogenic disorders, without requiring phasing references. Aneuploidies were accurately detected, with insights into the mechanistic origins of chromosomal abnormalities inferred from the parental unique allele fractions. We show that lrWGS-based concurrent haplotyping and aneuploidy profiling of single cells provides an alternative to current PGT methods, with applications potential in areas such as cell-based prenatal diagnosis and animal and plant breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d746d349cc87d8a40445a5a0f94e881f7a16b3" target='_blank'>
              Long-read whole-genome sequencing-based concurrent haplotyping and aneuploidy profiling of single cells
              </a>
            </td>
          <td>
            Yan Zhao, O. Tšuiko, Tatjana Jatsenko, Greet Peeters, E. Souche, Mathilde Geysens, E. Dimitriadou, Arne Vanhie, K. Peeraer, Sophie Debrock, Hilde Van Esch, J. R. Vermeesch
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdd7f6016d3ca163a2a60df3216dfee270ad74c8" target='_blank'>
              The plant early recombinosome: a high security complex to break DNA during meiosis
              </a>
            </td>
          <td>
            N. Rafiei, Arnaud Ronceret
          </td>
          <td>2024-09-27</td>
          <td>Plant Reproduction</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Latinos have a higher risk of developing gastric cancer (GC) than non-Latino Whites (NLW). Multiple studies have shown increased treatment resistance in diffuse-type cancers, which are more prevalent in Latino patients and dominated by the genomically stable (GS) genomic subtype. However, there is little genomic data from Latino gastric tumors due to the few studies targeting this minority population. Here, we attempt to rectify the disparity of genomic data by performing whole exome and/or low-coverage whole genome sequencing on tumors from 195 Latino gastric adenocarcinoma patients, from Colombia and Mexico. We also analyze 42 Latino patients from the US from a previous study. In total, we analyze tumors from 237 patients, the largest cohort of Latino gastric cancer tumors comprehensively sequenced thus far. Molecular subtyping of our cohort shows a higher proportion of tumors with a genomically stable (GS) subtype, 59%, and less with chromosomal instability (CIN), 24%, than reported in non-Latino patients from the Cancer Genome Atlas (20% and 50% respectively). Our earlier study of germline variants from Latinos, indicated a potential enrichment of variant related to homologous recombination deficiency (HRD), an important DNA repair mechanism. Focusing on this pathway, we performed HRD scoring and show that CIN tumors had the highest HRD score compared to all other subtypes. An alternative method analyzing somatic mutational patterns identified an indel mutational signature (ID6), also related to HRD, was enriched in GS tumors. Overall, these data provide a broader picture of the genomic landscape of Hispanic tumors, highlighting HRD targeting drugs as a potential treatment for both GS diffuse-types cancers particularly seen in Latinos and CIN tumors found more generally in gastric cancer.
 Citation Format: Dennis J Montoya, Ana Estrada-Florez, Paul Lott, Katherine Chiu, John Suarez-Olaya, Fabian Castro, Magdalena Echeverry de Polanco, Javier Torres, Mabel Bohorquez, Luis Carvajal-Carmona. Molecular characterization of gastric adenocarcinoma from Latinos shows homologous recombination deficiency [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C095.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b22944f466a6785df0771f5067c90022abd54c7" target='_blank'>
              Abstract C095: Molecular characterization of gastric adenocarcinoma from Latinos shows homologous recombination deficiency
              </a>
            </td>
          <td>
            Dennis J Montoya, Ana P. Estrada-Florez, Paul C. Lott, Katherine Chiu, John J. Suarez-Olaya, Fabian Castro, Magdalena Echeverry de Polanco, Javier Torres, M. Bohórquez, Luis G Carvajal-Carmona
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c35f97c112d09b7e983874a9efe5c2fc2f69d6c" target='_blank'>
              Enhancers in T Cell development and malignant lesions
              </a>
            </td>
          <td>
            Tong Zhang, Lin Zou
          </td>
          <td>2024-09-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The secretory capacity of Chinese hamster ovary (CHO) cells remains a fundamental bottleneck in the manufacturing of protein-based therapeutics. Unconventional biological drugs with complex structures and processing requirements are particularly problematic. Although engineered vector DNA elements can achieve rapid and high-level therapeutic protein production, a high metabolic and protein folding burden is imposed on the host cell. Cellular adaptations to these conditions include differential gene expression profiles that can in turn influence the productivity and quality control of recombinant proteins. In this study, we used quantitative transcriptomics and proteomics analyses to investigate how biological pathways change with antibody titre. Gene and protein expression profiles of CHO pools and clones producing a panel of different monoclonal and bispecific antibodies were analysed during fed-batch production. Antibody-expressing CHO pools were heterogeneous, resulting in few discernible genetic signatures. Clonal lines derived from these pools, selected for high and low production, yielded a small number of differentially expressed proteins that correlated with productivity and were shared across biotherapeutics. However, the dominant feature associated with higher protein production was transgene copy number and resulting mRNA expression level. Moreover, variability between clones suggested that the process of cellular adaptation is variable with diverse cellular changes associated with individual adaptation events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1d3a5e4be12b78430da74e93177a4166bdd030d" target='_blank'>
              Identification of Cellular Signatures Associated with Chinese Hamster Ovary (CHO) Cell Adaptation for Secretion of Antibodies
              </a>
            </td>
          <td>
            Ying Bai, Ivan Domenech Mercadé, Ramy Elgendy, Giulia Lambiase, S. Peak-Chew, Catarina Franco, Steven W Wingett, Tim J Stevens, Luigi Grassi, Noah Hitchcock, Cristina Sayago Ferreira, D. Hatton, Elizabeth A. Miller, Rajesh K. Mistry
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oxidative damage triggers micronuclear membrane rupture and defective repair Errors in chromosome segregation during cell division (mitosis) can lead to chromosomal instability, a hallmark of human cancer. These mitotic errors give rise to the formation of micronuclei—small cytoplasmic structures that are spatially separated from the primary nucleus and contain lagging chromosomes or chromosome fragments. Unlike the primary nucleus of a cell, micronuclei frequently undergo irreparable rupture and collapse. This breakdown releases micronuclear DNA into the cytosol, causing DNA damage, chromosomal rearrangements, and genomic instability, thereby contributing to cancer progression. The rupture of micronuclei also activates a component of the innate immune system called the cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING) pathway, leading to tumorrelated inflammation and metastasis (1). On pages 951 and 952 of this issue, Di Bona et al. (2) and Martin et al. (3), respectively, report the molecular underpinnings of micronuclear collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3bfa9de0e1bf705d54ddc01f0ac3869ca5a791" target='_blank'>
              Micronuclear collapse mechanisms in cancer
              </a>
            </td>
          <td>
            Marianna Maddaluno, C. Settembre
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb6a4f203d0ffda30fcc3f0dfa56bd3b405a484" target='_blank'>
              Amplified centrosomes—more than just a threat
              </a>
            </td>
          <td>
            Eva Kiermaier, Isabel Stötzel, Marina A Schapfl, A. Villunger
          </td>
          <td>2024-09-16</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Formyl peptide receptor 3 (FPR3) is known to have implications in the progression of various cancer types. Despite this, its biological significance within pan-cancer datasets has yet to be investigated. In this investigation, we scrutinized FPR3’s expression profiles, genetic alterations, prognostic significance, immune-related characteristics, methylation status, tumor mutation burden (TMB), and microsatellite instability (MSI) across different types of cancer. We utilized TISCH’s single-cell data to identify immune cells closely associated with FPR3. The predictive significance of FPR3 was evaluated independently in gliomas using data from TCGA and CGGA datasets, leading to the development of a prognostic nomogram. Immunohistochemistry and Western blot analysis confirmed FPR3 expression in gliomas. Lastly, the CCK-8 and wound-healing assays were employed to assess the impact of FPR3 on the proliferation and metastasis of GBM cell lines. In numerous cancer types, heightened FPR3 expression correlated with adverse outcomes, immune cell infiltration, immune checkpoints, TMB, and MSI. In glioma, FPR3 emerged as a notable risk factor, with the prognostic model effectively forecasting patient results. The potential biological relevance of FPR3 was confirmed in glioma, and it was shown to have significant involvement in the processes of glioma growth, immune infiltration, and metastasis. Our results imply a potential association of FPR3 with tumor immunity, indicating its viability as a prognostic indicator in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5328a0c9ff56af2b1f2405bf7c63cf8babe18d19" target='_blank'>
              Pan-cancer analysis and experimental validation of FPR3 as a prognostic and immune infiltration-related biomarker for glioma
              </a>
            </td>
          <td>
            Chenglin Ye, Peng Li, Boxu Chen, Yong Mo, Qianrong Huang, Qiuyun Li, Qinhan Hou, Ligen Mo, Jun Yan
          </td>
          <td>2024-10-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary In the mammalian immune system, αβ T lymphocytes are a fundamental set of T cells capable of recognizing an extraordinary range of antigens, presented in the context of the major histocompatibility complex molecules. The function of αβ T cells is linked to the T-cell receptor, a heterodimeric cell surface protein composed of an α and β chain uniquely expressed on T cells. Unlike conventional proteins encoded by a single gene, α and β chains are encoded by a large set of noncontiguous TRA and TRB genes that are randomly assembled during a stochastic process known as V(D)J recombination, generating a large and diverse repertoire. The extent of diversification depends on the number of genes and their genomic organization at the TRA and TRB loci, which varies from species to species. In this paper, we describe the genomic organization and gene content of the horse TRB locus, based on the recently released horse genome. Compared to other mammalian species, the horse TRB locus was found to be the largest locus studied to date. A clonotype analysis of a transcriptomic dataset was also performed to assess the characteristics of the V(D)J somatic rearrangement. Overall, our data provide useful information for further functional studies in healthy and sick horses of different breeds. Abstract In this paper, we report a comprehensive and consistent annotation of the locus encoding the β-chain of the equine T-cell receptor (TRB), as inferred from recent genome assembly using bioinformatics tools. The horse TRB locus spans approximately 1 Mb, making it the largest locus among the mammalian species studied to date, with a significantly higher number of genes related to extensive duplicative events. In the region, 136 TRBV (belonging to 29 subgroups), 2 TRBD, 13 TRBJ, and 2 TRBC genes, were identified. The general genomic organization resembles that of other mammals, with a V cluster of 135 TRBV genes located upstream of two in-tandem aligned TRBD-J-C clusters and an inverted TRBV gene at the 3′ end of the last TRBC gene. However, the horse b-chain repertoire would be affected by a high number of non-functional TRBV genes. Thus, we queried a transcriptomic dataset derived from splenic tissue of a healthy adult horse, using each TRBJ gene as a probe to analyze clonotypes encompassing the V(D)J junction. This analysis provided insights into the usage of the TRBV, TRBD, and TRBJ genes and the variability of the non-germline-encoded CDR3. Our results clearly demonstrated that the horse β-chain constitutes a complex level of variability, broadly like that described in other mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0305a779634116e26592cd1373ce78d96d052cd0" target='_blank'>
              A Comprehensive Analysis of the Genomic and Expressed Repertoire of the T-Cell Receptor Beta Chain in Equus caballus
              </a>
            </td>
          <td>
            R. Antonacci, F. Giannico, Roberta Moschetti, Angela Pala, A. C. Jambrenghi, S. Massari
          </td>
          <td>2024-09-29</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39abde50129bd51799285e63a0661a0178d6716" target='_blank'>
              DNA damage response in breast cancer and its significant role in guiding novel precise therapies
              </a>
            </td>
          <td>
            Jiayi Li, Ziqi Jia, Lin Dong, Heng Cao, Yansong Huang, Hengyi Xu, Zhixuan Xie, Yiwen Jiang, Xiang Wang, Jiaqi Liu
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The fragmentomics-based cell-free DNA (cfDNA) assays have recently demonstrated remarkable capabilities in identifying various cancers from non-conditional healthy controls. However, their accuracy in differentiating early-stage cancers from benign lesions, especially with inconclusive imaging results, remains unclear. In breast cancer, imaging-based screening methods often yield high false-positive rates, particularly in women with breast nodules, leading to unnecessary biopsies and increasing patient discomfort and healthcare burden. In this multi-center study, we enrolled 560 female participants and showed that cfDNA fragmentomics, using whole-genome sequencing, is a strong non-invasive biomarker for breast cancer. Among various cfDNA fragmentomics profiles, the fragment size ratio (FSR), fragment size distribution (FSD), and copy number variation (CNV) exhibited superior distinguishing ability compared to Griffin, motif breakpoint (MBP), and neomer features. Our cfDNA fragmentomics (cfFrag) model, which incorporates the three optimal fragmentomics features, successfully discriminated early-stage breast cancers from benign nodules, even at a low sequencing depth (3x). Notably, it demonstrated a specificity of 94.1% in asymptomatic healthy women at a 90% sensitivity for breast cancer. Furthermore, we highlight the clinical utility of the cfFrag model in predicting patient responses to neoadjuvant chemotherapy (NAC), and its integration with multimodal features, including radiological results and cfDNA methylation features, achieved high AUC values of 0.93-0.94 and 0.96, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec1ffa1e5f7be8782a9fb046c2b6f1ed5cb766" target='_blank'>
              Cell-free DNA Fragmentomics Assay to Discriminate the Malignancy of Breast Nodules and Evaluate Treatment Response
              </a>
            </td>
          <td>
            Jiaqi Liu, Yalun Li, W. Tang, Lijun Dai, Ziqi Jia, Heng Cao, Chenghao Li, Yuchen Liu, Yansong Huang, Jiang Wu, Dongxu Ma, Guangdong Qiao, H. Bao, Shuang Chang, Dongqin Zhu, Shanshan Yang, Xuxiaochen Wu, Xue Wu, Hengyi Xu, Hongyan Chen, Yang Shao, Xiang Wang, Zhihua Liu, Jianzhong Su
          </td>
          <td>2024-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA in eukaryotic genomes is under constant assault from both exogenous and endogenous sources, leading to DNA damage, which is considered a major molecular driver of ageing. Fortunately, the genome and the central exome are safeguarded against these attacks by abundant peripheral non-coding DNA. Non-coding DNA codes for small non-coding RNAs that inactivate foreign nucleic acids in the cytoplasm and physically blocks these attacks in the nucleus. Damage to non-coding DNA produced during such blockage is removed in the form of extrachromosomal circular DNA (eccDNA) through nucleic pore complexes. Consequently, non-coding DNA serves as a line of defence for the exome against DNA damage. The total amount of non-coding DNA/heterochromatin declines with age, resulting in a decrease in both physical blockage and eccDNA exclusion, and thus an increase in the accumulation of DNA damage in the nucleus during ageing and in age-related diseases. Here, we summarize recent evidence supporting a protective role of non-coding DNA in healthy and pathological states and argue that DNA damage is the proximate cause of ageing and age-related genetic diseases. Strategies aimed at strengthening the protective role of non-coding DNA/heterochromatin could potentially offer better systematic protection for the dynamic genome and the exome against diverse assaults, reduce the burden of DNA damage to the exome, and thus slow ageing, counteract age-related genetic diseases and promote a healthier life for individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7879deb4075aa031b3e715005da950caf3fffb2" target='_blank'>
              Protection of the genome and the central exome by peripheral non-coding DNA against DNA damage in health, ageing and age-related diseases.
              </a>
            </td>
          <td>
            Guo-Hua Qiu, Mingjun Fu, Xintian Zheng, Cuiqin Huang
          </td>
          <td>2024-09-26</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by fluctuations in chromosome number or structure within cells, stands out as a hallmark of cancer, enabling tumors to thrive in hostile conditions. CIN serves as a driver of genetic diversity, giving rise to clonal heterogeneity (CH). Emerging evidence points to a potential correlation between CIN, CH, and the prognosis of breast cancer (BC) patients, especially in tumors exhibiting overexpression of the human epidermal growth factor receptor 2 (HER2+). However, our understanding of the role of CIN in other subtypes of BC is limited. Furthermore, it remains unclear whether CIN levels above a certain threshold in BC tumors could adversely affect tumor growth, or if lower to moderate levels of CIN might be associated with a more favorable prognosis for BC patients compared to elevated levels. Elucidating these relationships could significantly influence risk assessment and the formulation of future therapeutic approaches targeting CIN in BC. This study aimed to assess CIN and CH in tumor tissue samples obtained from Colombian patients diagnosed with luminal A, luminal B, HER2+, or triple-negative BC, and compare them with established clinicopathological parameters. The findings of this study indicate that BC patients, exhibit intermediate CIN, high CH, and stable aneuploidy. All these characteristics were found to be related with clinicopathological features. Our results suggest that the identification of CIN, CH and aneuploidy could improve cancer risk stratification, which could help to clarify the prediction of clinical outcomes and guiding personalized therapeutic strategies for patients with different BC subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f4c53715c6bdbb1cc8b85a27c36e138c73e29" target='_blank'>
              Exploring chromosomal instability and clonal heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            Giovanny Castellanos, Laura Valentina Camargo-Herrera, N. Rangel, G. A. Jiménez-Tobón, María Martínez-Agüero, Milena Rondón-Lagos
          </td>
          <td>2024-09-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="We developed a linear amplification-mediated high-throughput genome-wide translocation sequencing method to profile Ig class-switch recombination (CSR) in human B cells in an unbiased and quantitative manner. This enables us to characterize CSR junctions resulting from either deletional recombination or inversion for each Ig class/subclass. Our data showed that more than 90% of CSR junctions detected in peripheral blood in healthy control subjects were due to deletional recombination. We further identified two major CSR junction signatures/patterns in human B cells. Signature 1 consists of recombination junctions resulting from both IgG and IgA switching, with a dominance of Sµ-Sγ junctions (72%) and deletional recombination (87%). Signature 2 is contributed mainly by Sµ-Sα junctions (96%), and these junctions were almost all due to deletional recombination (99%) and were characterized by longer microhomologies. CSR junctions identified in healthy individuals can be assigned to both signatures but with a dominance of signature 1, whereas almost all CSR junctions found in patients with defects in DNA-PKcs or Artemis, two classical nonhomologous end joining (c-NHEJ) factors, align with signature 2. Thus, signature 1 may represent c-NHEJ activity during CSR, whereas signature 2 is associated with microhomology-mediated alternative end joining in the absence of the studied c-NHEJ factors. Our findings suggest that in human B cells, the efficiency of the c-NHEJ machinery and the features of switch regions are crucial for the regulation of CSR orientation. Finally, our high-throughput method can also be applied to study the mechanism of rare types of recombination, such as switching to IgD and locus suicide switching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76f6ad58cd1fa3e33c899adc7067ec2a694fd440" target='_blank'>
              Orientation Regulation of Class-switch Recombination in Human B Cells.
              </a>
            </td>
          <td>
            Likun Du, Valentyn Oksenych, Hui Wan, Xiaofei Ye, Junchao Dong, Adam Yongxin Ye, Hassan Abolhassani, S. Vlachiotis, Xuefei Zhang, K. de la Rosa, L. Hammarström, M. van der Burg, Frederick W. Alt, Q. Pan-Hammarström
          </td>
          <td>2024-09-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Simple Summary Head and neck squamous cell carcinoma (HNSCC) is a complex and varied cancer that is difficult to treat due to its biological diversity. Single-cell RNA sequencing (scRNAseq) offers a way to study HNSCC by examining individual cells within tumors. This technology helps identify different cell types and their roles in cancer development, diagnosis, and treatment. By understanding how various cells respond to treatments like chemotherapy and immunotherapy, scRNAseq can enhance the study of cancer biology and improve therapeutic outcomes. This review highlights the significant impact of scRNAseq on HNSCC research and its potential to advance the study of other cancers. Abstract Background/Objectives: Head and neck squamous cell carcinoma (HNSCC) is a highly heterogeneous and the most common form of head and neck cancer, posing significant challenges for disease management. The objective of this review is to assess the utility of single-cell RNA sequencing (scRNAseq) in addressing these challenges by enabling a detailed characterization of the tumor microenvironment (TME) at the cellular level. Methods: This review compiles and analyzes current strategies that utilize scRNAseq and other single-cell technologies in HNSCC research. Results: For HNSCC etiology, scRNAseq allows for the construction of cellular atlases, characterization of different cell types, and investigation of genes and processes involved in cancer initiation, development, and progression within the TME. In terms of HNSCC diagnosis and prognosis, the resolution offered by scRNAseq enables the identification of cell type-specific signatures, enhancing prognostic models and disease stratifiers for patient outcome assessments. Regarding HNSCC treatment, scRNAseq provides insights into cellular responses to various treatments, including radiotherapy, chemotherapy, and immunotherapy, contributing to a better understanding of treatment efficacy and patient outcomes. Conclusions: This review highlights the contributions of scRNAseq to HNSCC research, addressing its cellular and biological complexity, and emphasizes its potential for advancing research and clinical practice in other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e831682e82f2a5f8a0b8ba438418409c9819bff6" target='_blank'>
              Unravelling the Complexity of HNSCC Using Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Cristina Conde-Lopez, Divyasree Marripati, M. Elkabets, Jochen Hess, Ina Kurth
          </td>
          <td>2024-09-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bef3235f61a4b9794507ce8f7698a6c5db0de62" target='_blank'>
              Concurrent analysis of genome and transcriptome in one single cell
              </a>
            </td>
          <td>
            Johanna Heid, Ronald Cutler, Moonsook Lee, J. Vijg, A. Maslov
          </td>
          <td>2024-09-16</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281649fc3a01628d835005fce4ae40aa70c5e3f2" target='_blank'>
              Oncometabolites at the crossroads of genetic, epigenetic and ecological alterations in cancer.
              </a>
            </td>
          <td>
            Letizia Lanzetti
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alterations in deoxyribonucleoside triphosphate (dNTP) pools have been linked to increased mutation rates and genome instability in unicellular organisms and cell cultures. However, the role of dNTP pool changes in tumor development in mammals remains unclear. In this study, we present a mouse model with a point mutation at the allosteric specificity site of ribonucleotide reductase, RRM1-Y285A. This mutation reduced ribonucleotide reductase activity, impairing the synthesis of deoxyadenosine triphosphate (dATP) and deoxyguanosine triphosphate (dGTP). Heterozygous Rrm1+/Y285A mice exhibited distinct alterations in dNTP pools across various organs, shorter lifespans and earlier tumor onset compared with wild-type controls. Mutational spectrum analysis of tumors revealed two distinct signatures, one resembling a signature extracted from a human cancer harboring a mutation of the same amino acid residue in ribonucleotide reductase, RRM1Y285C. Our findings suggest that mutations in enzymes involved in dNTP metabolism can serve as drivers of cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe3b3a3839a433e41d5ae4062ca8fa74fde87a0" target='_blank'>
              Altered dNTP pools accelerate tumor formation in mice.
              </a>
            </td>
          <td>
            Phong Tran, Pradeep Mishra, Leonard G. Williams, Roman Moskalenko, Sushma Sharma, A. K. Nilsson, Danielle L. Watt, Pernilla Andersson, Anders Bergh, Z. Pursell, A. Chabes
          </td>
          <td>2024-10-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b57b76c24e922e56ff6f3fa5ae5de6f644999f2" target='_blank'>
              Human papillomavirus-encoded circular RNA circE7 promotes immune evasion in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Junshang Ge, Yi Meng, Jiayue Guo, Pan Chen, Jie Wang, Lei Shi, Dan Wang, Hongke Qu, Pan Wu, Chunmei Fan, Shanshan Zhang, Q. Liao, Ming Zhou, Bo Xiang, Fuyan Wang, Ming-Liang Tan, Zhaojian Gong, Wei Xiong, Zhaoyang Zeng
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Endometrial carcinoma (EC) is a prevalent gynecological malignancy that poses a significant threat to women’s health worldwide. However, its pathogenesis and underlying mechanisms remains unclear. In this study, expression quantitative trait loci data, Mendelian randomization analysis, and differential expression analysis were performed to identify potential targets. A prognostic risk signature was subsequently constructed for EC patients based on the expression of these genes. Four bioinformatics algorithms, including generalized linear model, extreme gradient boosting, support vector machine, and random forest, were used to identify hub genes in EC. The expression of ring finger protein 144A (RNF144A) was validated using quantitative real-time polymerase chain reaction. Cellular proliferation and migration ability were evaluated using CCK-8 and Transwell assays, respectively. The genes RNF144A, ketohexokinase, and Rab interacting lysosomal protein like 2 were identified as potential targets for EC. Their differential expression was observed in EC patients, and Mendelian randomization analysis revealed a negative correlation between these genes and the development of EC. Mechanistic analyses suggested a strong association between these genes and the tumor immune microenvironment. The constructed risk signature was significantly associated with the prognosis, age, cancer stage, and grade of EC patients. Furthermore, based on interacted model algorithms, RNF144A was identified as a hub gene in EC. It was found to be significantly downregulated in EC samples, and its expression was positively correlated with the stage and grade of EC patients. In vitro experiments showed that overexpression of RNF144A significantly promoted cell growth and migration in EC cells. In conclusion, this study provides insights into the molecular mechanisms underlying EC progression and identifies preliminary candidate biomarkers for the development of EC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ccab2586145d3cccf21eccbbc93cc5b295b360" target='_blank'>
              RNF144A as a potential risk gene for endometrial carcinoma: Insights from Mendelian randomization, bulk RNA sequencing, single-cell RNA, and experimental analysis
              </a>
            </td>
          <td>
            Xiqin Ruan, Ni Wang, Qingwen Xie, Yi Du
          </td>
          <td>2024-10-04</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Unexplained stillbirth is hypothesised to occur due to premature placental ageing, with unexpected deterioration of placental function for gestational age. Circular RNAs (circRNAs) are enzyme resistant RNA molecules that accumulate in ageing tissues. Furthermore, circRNAs bind gDNA directly, forming circRNA:DNA complexes which can induce DNA breaks. This study investigated circRNA accumulation with gestational age in healthy and stillbirth placentae and determined whether circRNAs directly interact with placental DNA causing DNA damage. Placenta samples (n=60 term uncomplicated; n=4 unexplained stillbirth, 23, 26, 31, 34 weeks’ gestation) were assessed for DNA damage using an alkaline Comet Assay. Expression levels of 6 candidate circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277), and their linear transcripts, were quantified using qPCR. Physical interaction of candidate circRNAs with DNA was confirmed by DNA:RNA ImmunoPrecipitation (DRIP). The effect of circ_0009000 knockdown in HEK293T cells was assessed following transfection with either a siRNA (designed to knockdown circ_0009000) or a scrambled siRNA control, at 5, 10 and 20 nM final concentrations using Lipofectamine RNAiMax. DNA damage was assessed by Comet Assay. Compared with earlier gestations (37, 38, 39 and 40 weeks’), placental DNA damage and expression of all 6 candidate circRNAs, but not their linear transcripts, were increased in 40 and 41+ weeks’ gestation samples, and in stillbirth. DRIP-qPCR signal size was significantly larger in term placentae than in enzyme-treated controls, confirming that all candidate circRNA loci bind to placental DNA. Depletion of circ_0009000 by specific siRNA in HEK293T cells, significantly reduced DNA damage compared to control. Stillbirth placentae show accelerated ageing with premature accumulation of candidate circRNAs (first evidence in humans) at levels consistent with older gestation tissue. Importantly, these circRNAs bind to DNA and circ_0009000 causes DNA breaks in placenta. Therefore, circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277) play a role in placental ageing and associate with stillbirth, likely via DNA damage. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9adf6d93429ec373710f176ef7b50a5c1efd5e40" target='_blank'>
              Circular RNAs correlate with DNA damage in ageing human placental tissue and in stillbirth
              </a>
            </td>
          <td>
            A. Arthurs, Matilda R. Jackson, Dylan McCullough, Hamish S. Scott, Christopher P. Barnett, Stuart T. Webb, Melanie D. Smith, Tanja Jankovic-Karasoulos, Gustaaf A. Dekker, Claire T. Roberts
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eaf6453d4c9d143250dede3ed41dacec77b88be" target='_blank'>
              Epigenetic landscape reorganisation and reactivation of embryonic development genes are associated with malignancy in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            S. Ghisai, L. van Hijfte, W. Vallentgoed, C. M. S. Tesileanu, I. D. de Heer, J. Kros, Marc Sanson, T. Gorlia, Wolfgang Wick, M. Vogelbaum, A. A. Brandes, Enrico Franceschi, Paul M. Clement, Anna K. Nowak, V. Golfinopoulos, M. J. van den Bent, P. French, Y. Hoogstrate
          </td>
          <td>2024-10-09</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09cf4def72c527eab48684dc58deaf767dec6890" target='_blank'>
              Exploring a new signature for lung adenocarcinoma: analyzing cuproptosis-related genes through Integrated single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Jiangtao Liu, Wei Xia, Feng Xue, Chen Xu
          </td>
          <td>2024-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer (PCa) is among the three main types of cancer. Although prostate-specific antigen (PSA) is routinely tested, it has disadvantages, such as poor prognostic ability. Therefore, finding more PCa markers and therapeutic targets remains a subject of study. CircRNAs have been found to have regulatory roles in various diseases, such as diabetes, Central Nervous System (CNS) neuropathy, etc. where their application in cancer is even more valuable. Therefore, this paper aims to search for differentially expressed circRNAs in PCa and find downstream targeting pathways related to autophagy. Method By detecting the expression of circRNA in the samples, hsa_circ_0119816 was finally identified as the research target. The properties of circRNA were verified by RNase R, actinomycin D, and fluorescence in situ hybridization (FISH). The downstream target miRNAs and target proteins were predicted by an online database, and the targeting relationship was verified using dual luciferase and RNA Immunoprecipitation. The effects of circRNAs and their downstream signalling pathways on prostate cancer cell proliferation, migration, EMT and autophagy were examined by CCK-8, Transwell, immunofluorescence and Western blotting. Results CircBIRC6 is highly expressed in prostate cancer samples. Knockdown of its expression inhibits cell proliferation, invasion, EMT and autophagy and promotes apoptosis. CircBIRC6/miRNA-574-5p/DNAJB1 is a molecular axis that regulates prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d1bad6dbf3ace4247e9abe87ef37ab3c3b685" target='_blank'>
              CircBIRC6 affects prostate cancer progression by regulating miR-574-5p and DNAJB1
              </a>
            </td>
          <td>
            Bin Zhao, Jinye Yang, Fengming Ran, Yuanlong Shi, Libo Yang, Yuanpeng Duan, Zhiyu Shi, Xin Li, Jianpeng Zhang, Zhiyao Li, Jiansong Wang
          </td>
          <td>2024-09-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="The incidence of colorectal cancer (CRC) among individuals under age 50, or early-onset CRC (EOCRC), has been rising over the past few decades for unclear reasons, and the etiology of the disease remains largely unknown. Known genetic risk factors do not explain this increase, pointing to possible environmental and as-yet unidentified genetic contributors and their interactions. Previous research linked genetic variation on chromosome 6 to increased CRC risk. This region harbors multiple immune genes, including the gene encoding Major Histocompatibility Complex (MHC) class I polypeptide-related sequence A (MICA). MICA is a polygenic ligand for the Natural Killer Group 2D receptor (NKG2D), a receptor expressed on Natural Killer (NK) cells and other lymphocytes. Given that intra-tumoral NK cell infiltration correlates with favorable CRC outcomes, we hypothesized that germline genetic variation in MICA could influence CRC risk. In a discovery set of 40,125 cases and controls, we show that the minor G allele at Chr6:31373718C>G (hg19) is associated with increased risk for CRC (odds ratio (OR) = 1.09, 95% confidence interval (CI) 1.04 - 1.15, p = 0.0009). The effect is stronger in EOCRC (OR = 1.26, 95% CI 1.08 - 1.44, p = 0.0023) than in those 50 and over (OR = 1.07, 95% CI 1.02 - 1.13; p = 0.012) (Ratio of ORs = 1.32, 95% CI 1.14 - 1.52, p = 0.0002). In an independent validation set of 77,983 cases and controls, the adjusted interaction by age -of-onset was significant at OR = 1.15 (95% CI 1.03 - 1.34, p = 0.0150) with a higher risk in EOCRC. Expression quantitative trait locus analysis in normal colonic epithelia showed that MICA RNA expression decreases linearly with each additional copy of the minor G allele (p = 3.345 x 10-18). Bulk RNA analysis of the tumor immune microenvironment revealed that tumors from patients with CG or GG genotypes have lower resting and activated NK cell infiltration as compared to tumors from patients with CC genotype. Multiplex immunofluorescence analysis demonstrated that patients with a G allele (i.e. CG or GG genotype, but not CC genotype) have a statistically significant decrease in the number of NK cells in tumor compared to adjacent normal colonic mucosa. Taken together, population-based epidemiologic, molecular, genetic, cellular and immunologic evidence demonstrate that MICA genotype is associated with increased risk of EOCRC and reduced number of NK cells in colorectal tumors, suggesting that patients with a G allele have altered NK cell-mediated immunosurveillance. These novel findings suggest that EOCRC may have a previously unrecognized innate immune-mediated etiology which merits further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5c740e57eda493d70527270b12cf7926d01624" target='_blank'>
              Genetic Variation and Regulation of MICA Alters Natural Killer Cell-Mediated Immunosurveillance in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Heather M. McGee, J. Bonner, Yubo Fu, Oscar Colunga Flores, S. Lindsey, Lawrence Shaktah, Ferran Moratalla-Navarro, Yasmin Kamal, Kevin K. Tsang, C. P. Walker, G. Idos, K. McDonnell, H. Rennert, Elizabeth L Barry, Hermann Brenner, Daniel D. Buchanan, P. Campbell, A. T. Chan, J. Chang-Claude, Jane C Figueiredo, M. Gago-Domínguez, M. Hoffmeister, Li Hsu, J. Huyghe, M. Jenkins, L. Marchand, Heinz-Josef Lenz, Li Li, A. Lindblom, Yun Ru Liu, B. Lynch, Christina C. Newton, K. Offit, Shuji Ogino, Rebeca Sanz Pamplona, Andrew J. Pellatt, Paul D. P. Pharoah, Amanda I. Phipps, Lorena Reynaga, Allyson S. Templeton, C. Um, Alicja Wolk, M. Woods, A. H. Wu, Yen Yun, Weishan Zheng, Terence M. Williams, David V Conti, Ulrike Peters, F. Lejbkowicz, J. Greenson, S. Schmit, W. Gauderman, Stanley R. Hamilton, Victor Moreno, Gadi Rennert, Stephen B. Gruber, †Gad Rennert, B. Gruber, MD M. McGee, MD Stephen B. Gruber
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="High Grade Serous Ovarian Cancer (HG-SOC), the most prevalent and aggressive gynecological malignancy, is marked by ubiquitous loss of functional p53, largely due to point mutations that arise very early in carcinogenesis. These mutations often lead to p53 protein misfolding and subsequent aggregation, yet the alterations in intracellular p53 dynamics throughout ovarian cancer progression remain poorly understood. HG-SOC originates from the fallopian tube epithelium, with a well-documented stepwise progression beginning with early pre-malignant p53 signatures. These signatures represent largely normal cells that express and accumulate mutant p53, which then transform into benign serous tubal intraepithelial lesions (STIL), progress into late pre-malignant serous tubal intraepithelial carcinoma (STIC), and ultimately lead to HGSOC. Here, we show that the transition from folded, soluble to aggregated mutant p53 occurs during the malignant transformation of benign precursor lesions into HGSOC. We analyzed fallopian tube tissue collected from ten salpingo-oophorectomy cases and determined the proportion of cells carrying soluble versus mis-folded/mutant p53 through conformation-sensitive staining and quantification. Misfolded p53 protein, prone to aggregation, is present in STICs and HG-SOCs, but notably absent from preneoplastic lesions and surrounding healthy tissue. Overall, our results indicate that aggregation of mutant p53 is a structural defect that distinguishes preneoplastic early lesions from late premalignant and malignant ones, offering a potential treatment window for targeting p53 aggregation and halting ovarian cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e004df7ee189f194b9dc43a60b80d81862858bf" target='_blank'>
              Mutant p53 Misfolding and Aggregation Precedes Transformation into High-Grade Serous Ovarian Carcinoma
              </a>
            </td>
          <td>
            S. Sartini, Lexi Omholt, Neda A Moatamed, A. Soragni
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Forkhead box A1 (FOXA1) acts as a pioneer factor for Estrogen Receptor alpha (ERα), playing a crucial role in determining where ERα binds to DNA and regulates gene expression in breast cancer cells. A pioneer factor is a class of transcription factors (TF) that is responsible for opening the chromatin to allow other TFs to attach and activate transcription of that gene. FOXA1 acts as a facilitator for ERα-DNA interactions. It binds to specific DNA regions, opening the chromatin for ERα to bind. It “pre-marks” these sites in the genome even in the absence of estrogen. In fact, FOXA1 is frequently overexpressed in ER+ breast cancers, suggesting it has a crucial role in driving tumor growth. As such, FOXA1 is an attractive therapeutic target that already serves as a great biomarker for many types of breast cancers. Inhibiting FOXA1 can potentially block ER activity and inhibit growth of ER+ breast cancer, especially in those resistant to anti-estrogen therapies, such as tamoxifen. In this project, we identified sites that FOXA1 opens and correlated them with SNP mutations commonly found in the Hispanic population. The presence of these SNPs portends a worse outcome in Hispanic patients. To assess which open chromatin regions are crucial for breast cancer pathogenesis, a ChIP-seq analysis was done on vehicle-treated and estrogen-treated samples to determine FOXA1 binding sites. After identifying gained, lost, and differentially expressed peaks set, these were compared to genome-wide association studies (GWAS) of Hispanic patients to develop an understanding of how much overlap is common between sites of mutations and open sections of chromatin due to FOXA1. Through our analysis, we uncovered a subset of FOXA1 binding sites that overlap with sites of inherited mutation in Hispanic patients. This informs on the subset of genes that are linked to breast cancer and potential therapeutic targets.
 Citation Format: Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil. FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e221203c9b5991c2cb9afa79cdfe37b8baf68cb3" target='_blank'>
              Abstract B011: FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations
              </a>
            </td>
          <td>
            Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf2bf343234e0f81f86aa377bfcd615616e7e98" target='_blank'>
              Methylation patterns at the adjacent CpG sites within enhancers are a part of cell identity
              </a>
            </td>
          <td>
            O. Taryma-Leśniak, Jan Bińkowski, P. K. Przybylowicz, K. E. Sokolowska, Konrad Borowski, T. Wojdacz
          </td>
          <td>2024-10-10</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628089f06d9782813abf7a0fd0e702990c7e28b0" target='_blank'>
              Chemoproteomic profiling unveils binding and functional diversity of endogenous proteins that interact with endogenous triplex DNA.
              </a>
            </td>
          <td>
            Hongzhan Xu, Jing Ye, Kui-Xing Zhang, Qingxi Hu, Tongxiao Cui, Chong Tong, Mengqi Wang, Huichao Geng, Kun-Ming Shui, Yan Sun, Jian Wang, Xiaomeng Hou, Kai Zhang, Ran Xie, Yafei Yin, Nan Chen, Jia-Yu Chen
          </td>
          <td>2024-09-02</td>
          <td>Nature chemistry</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Background Ovarian cancer (OC) is a global malignancy characterized by metastatic invasiveness and recurrence. Long non-coding RNAs (lncRNAs) and Telomeres are closely connected with several cancers, but their potential as practical prognostic markers in OC is less well-defined. Methods Relevant mRNA and clinical data for OC were sourced from The Cancer Genome Atlas (TCGA) database. The telomere-related lncRNAs (TRLs) prognostic model was established by univariate/LASSO/multivariate regression analyses. The effectiveness of the TRLs model was evaluated and measured via the nomogram. Additionally, immune infiltration, tumor mutational load (TMB), and drug sensitivity were evaluated. We validated the expression levels of prognostic genes. Subsequently, PTPRD-AS1 knockdown was utilized to perform the CCK8 assay, colony formation assay, transwell assay, and wound healing assay of CAOV3 cells. Results A six-TRLs prognostic model (PTPRD-AS1, SPAG5-AS1, CHRM3-AS2, AC074286.1, FAM27E3, and AC018647.3) was established, which can effectively predict patient survival rates and was successfully validated using external datasets. According to the nomogram, the model could effectively predict prognosis. Furthermore, we detected the levels of regulatory T cells and M2 macrophages were comparatively higher in the high-risk TRLs group, but the levels of activated CD8 T cells and monocytes were the opposite. Finally, the low-risk group was more sensitive to anti-cancer drugs. The mRNA levels of PTPRD-AS1, SPAG5-AS1, FAM27E3, and AC018647.3 were significantly over-expressed in OC cell lines (SKOV3, A2780, CAOV3) in comparison to normal IOSE-80 cells. AC074286.1 were over-expressed in A2780 and CAOV3 cells and CHRM3-AS2 only in A2780 cells. PTPRD-AS1 knockdown decreased the proliferation, cloning, and migration of CAOV3 cells. Conclusion Our study identified potential biomarkers for the six-TRLs model related to the prognosis of OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3abf27265900e867b786b472cf6d79fdb5355c6" target='_blank'>
              Identification of telomere-related lncRNAs and immunological analysis in ovarian cancer
              </a>
            </td>
          <td>
            Weina Xu, Shuliu Sang, Jun Wang, Shanshan Guo, Xiao Zhang, Hailun Zhou, Yijia Chen
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97dc334faa1e4fbb032cc8091b61ef3d8c4f930a" target='_blank'>
              Single-molecule imaging of SWI/SNF chromatin remodelers reveals bromodomain-mediated and cancer-mutants-specific landscape of multi-modal DNA-binding dynamics
              </a>
            </td>
          <td>
            Wilfried Engl, Aliz Kunstar-Thomas, Siyi Chen, Woei Shyuan Ng, Hendrik Sielaff, Ziqing Winston Zhao
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c75249f54c500f783739b3eb3c7b22b00e436d" target='_blank'>
              Azacytidine treatment affects the methylation pattern of genomic and cell-free DNA in uveal melanoma cell lines
              </a>
            </td>
          <td>
            Sarah Tadhg Ferrier, Mingyang Li, J. Burnier
          </td>
          <td>2024-10-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Immunotherapies such as immune checkpoint inhibitors (ICIs) have greatly improved patient outcomes in multiple tumor types, but ICIs have failed to achieve responses in many patients. Meta-analyses of patient cohorts treated with ICIs have observed that worse clinical outcomes are strongly correlated with high levels of intratumoral genetic heterogeneity (ITH). ITH occurs when tumor cells mutate as they divide, typically due to oxidative stress or DNA damage from rapid proliferation and continued carcinogenic insults, generating tumor subclones. However, the mechanisms by which ITH impairs the anti-tumor immune response are not fully understood. We propose a novel mechanism for how ITH in highly T cell-infiltrated inflamed tumors can adversely affect T cell behavior. We hypothesize that strongly pro-inflammatory tumor subclones secreting robust pro-inflammatory chemokine profiles can dominate T cell trafficking within the tumor, resulting in a weakened immune response against other tumor subclones. Using a murine squamous cell skin carcinoma (SCC) model, we have demonstrated that in immune-inflamed tumors, the spatial distribution and behavior of tumor-infiltrating T cells is influenced by ITH. A strongly pro-inflammatory tumor line expressing an endogenous RFP fluorophore was mixed with a weakly pro-inflammatory tumor line expressing an endogenous YFP fluorophore at a 1:1 ratio. The resulting tumors contained distinct RFP+ and YFP+ regions. These were analyzed by immunofluorescence (IF) and by manually dissecting the tumors and analyzing each region by flow cytometry. Flow cytometry analysis determined that the strongly pro-inflammatory RFP+ regions contained significantly more total T cells and CD4+ T cells than YFP+ regions. Furthermore, FoxP3- “conventional” CD4+ T cells were significantly more likely to be “terminal effector-like” CD44+CD62L- in RFP+ regions, and significantly more likely to be “naïve-like” CD62L+CD44- in YFP+ regions. IF analysis confirmed the preferential distribution of CD4+ T cells in RFP+ regions, and observed that this spatial pattern was evident as early as 11 days after subcutaneous injection. Notably, this localization trend did not occur with CD8+ T cells, which were evenly dispersed throughout the tumors. ELISA analysis of lysates of both tumor cell lines determined that the pro-inflammatory RFP+ cell line secretes significantly higher levels of the chemokine CCL5, which has been observed by other studies to selectively mediate CD4+ T cell chemotaxis. These results demonstrate that intratumoral CD4+ trafficking is shaped by the spatial distribution of specific tumor subclones, and suggest tumor cell chemokine signaling may directly mediate T cell spatial localization and behavior within tumors.
 Citation Format: Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Reeves. Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeeff4ff2175551d0a146978acdaeb3ec633ea2b" target='_blank'>
              Abstract A033: Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors
              </a>
            </td>
          <td>
            Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Q Reeves
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Telomeres, the protective structures at the ends of chromosomes, are crucial for maintaining cellular longevity and genome stability. Their proper function depends on tightly regulated processes of replication, elongation, and damage response. The shelterin complex, especially Telomere Repeat-binding Factor 1 (TRF1) and TRF2, plays a pivotal role in telomere protection and has emerged as a potential anti-cancer target for drug discovery. These proteins bind to the repetitive telomeric DNA motif TTAGGG, facilitating the formation of protective structures and recruitment of other telomeric proteins. Structural methods and advanced imaging techniques have provided insights into telomeric protein-DNA interactions, but probing the dynamic processes requires single-molecule approaches. Tools like magnetic tweezers, optical tweezers, and atomic force microscopy (AFM) have been employed to study telomeric protein-DNA interactions, revealing important details such as TRF2-dependent DNA distortion and telomerase catalysis. However, the preparation of single-molecule constructs with telomeric repetitive motifs continues to be a challenging task, potentially limiting the breadth of studies utilizing single-molecule mechanical methods. To address this, we developed a method to study interactions using full-length human telomeric DNA with magnetic tweezers. This protocol describes how to express and purify TRF2, prepare telomeric DNA, set up single-molecule mechanical assays, and analyze data. This detailed guide will benefit researchers in telomere biology and telomere-targeted drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f2fc75b3e952e11dfa401640112a1b316a2cff" target='_blank'>
              Analyzing Telomeric Protein-DNA Interactions Using Single-Molecule Magnetic Tweezers.
              </a>
            </td>
          <td>
            Han Gao, Yanling Liu, Zhongbo Yu
          </td>
          <td>2024-08-30</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immunoglobulin (IG) gene analysis provides fundamental insight into B-cell receptor structure and function. In B-cell tumors, it can inform the cell of origin and clinical outcomes. Its clinical value has been established in the two types of chronic lymphocytic leukemia with unmutated or mutated IGHV genes and is emerging in other B-cell tumors. The traditional PCR-based techniques, which are labor-intensive, rely on the attainment of either a dominant sequence or a small number of subclonal sequences and do not allow automated matching with the clonal phenotypic features. Extraction of the expressed tumor IG transcripts using high-throughput RNA sequencing (RNA-seq) can be faster and allow the collection of multiple sequences matched with the transcriptome profile. Analytical tools are regularly sought to increase the accuracy, depth, and speed of acquisition of the full IGV-(IGD)-IGJ-IGC sequences and combine the IG characteristics with other RNA-seq data. We provide here a user-friendly protocol for the rapid extraction, identification, and accurate determination of the full (leader to constant region) tumor IG templated and non-templated transcript sequence from RNA-seq. The derived amino acid sequences can be interrogated for their physico-chemical characteristics and, in certain lymphomas, predict tumor glycan types occupying acquired N-glycosylation sites. These features will then be available for association studies with the tumor transcriptome. The resulting information can also help refine diagnosis, prognosis, and potential therapeutic targeting in the most common lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea7468231c76081901be5a9f5bad586bdb3d961" target='_blank'>
              IgSeqR: a protocol for the identification, assembly, and characterization of full-length tumor Immunoglobulin transcripts from unselected RNA sequencing data
              </a>
            </td>
          <td>
            Dean Bryant, B. Sale, G. Chiodin, D. Tatterton, Ben Stevens, Alyssa Adlaon, Erin Snook, James Batchelor, Alberto Orfao, Francesco Forconi
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187812e12825019ab421661fdd39125bff16e32a" target='_blank'>
              Potential role of heteroplasmic mitochondrial DNA mutations in modulating the subtype-specific adaptation of oral squamous cell carcinoma to cisplatin therapy
              </a>
            </td>
          <td>
            Amnani Aminuddin, Pei Yuen Ng, Chee Onn Leong, S. Makpol, E. Chua
          </td>
          <td>2024-10-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited. Methods In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC. Results Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis. Conclusion These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Although the concept of cancer stem cells is still controversial, previous studies have shown that blood cancers, as well as specific types of solid cancers such as colorectal cancer, rely on stem cells during the onset of tumor growth and further tumor development. Moreover, resistance to therapeutic treatment in leukemias such as acute myeloid leukemia and in colorectal cancer can be attributed to a small population of cells with stemness properties known as minimal residual disease. In this review, we look back on the discovery of cancer stem cells and the contribution of the findings in blood cancer to a parallel discovery in solid cancers. We focus on CD44 as a stem cell marker, both in blood cancers and in several types of solid cancers, particularly of the gastro-intestinal tract. This review highlights newly discovered molecular mechanisms of action of CD44 which indicate that CD44 has indeed a function in stemness, stem cell maintenance and drug resistance. We attempt here to make the link between the functions of CD44 isoforms in stemness and their involvement in specific steps of tumor growth and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8580e7867b570560c73223a09cea13edca2587fc" target='_blank'>
              CD44: A Stemness Driver, Regulator and Marker - all in one?
              </a>
            </td>
          <td>
            S. J. Sonnentag, Nagwa S M Ibrahim, V. Orian-Rousseau
          </td>
          <td>2024-10-04</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Radiotherapy remains a cornerstone in cancer treatment, leveraging ionizing radiation to eradicate malignant cells. Its efficacy, however, is frequently challenged by the heterogeneous sensitivity of tumors and surrounding tissues to radiation. Therefore, understanding the molecular mechanisms underlying radiosensitivity is crucial for improving treatment outcomes. Among the myriad of molecular players involved, the tumor suppressor protein p53 stands out as a central regulator with significant implications for radiosensitivity. Known as the “guardian of the genome”, p53 plays a pivotal role in maintaining genomic stability and orchestrating cellular responses such as cell cycle arrest, DNA repair, apoptosis, and senescence in response to various stress signals, including radiation-induced DNA damage. Activation of p53 triggers the transcription of target genes involved in DNA repair pathways, such as p21, MDM2, and GADD45, facilitating the repair of radiation-induced DNA damage or the elimination of irreparably damaged cells. This, in turn, influences the overall radiosensitivity of tissues. Mutations in the TP53 gene, which encodes p53, are among the most frequent genetic alterations in human cancers. Loss or dysfunction of p53 can compromise the cellular response to radiation, leading to increased resistance to therapy and poorer clinical outcomes. Conversely, intact p53 function is associated with enhanced radiosensitivity due to its ability to promote cell cycle arrest and apoptosis in response to radiation-induced DNA damage. In conclusion, elucidating the molecular mechanisms by which p53 influences radiosensitivity is essential for advancing our understanding of the radiation response in cancer cells and developing more effective therapeutic approaches to cancer treatment. This review provides a comprehensive overview of the multifaceted role of p53 in modulating cellular responses to radiation, emphasizing its influence on radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edeb6b0914ac30942be5b714c5e52dd1ba564b4d" target='_blank'>
              Exploring the Role of p53 in Radiosensitivity: A Key Player in Cancer Therapy
              </a>
            </td>
          <td>
            Tusher- Al-Arafat, Aihong Mao, T. Katsube, Bing Wang
          </td>
          <td>2024-10-24</td>
          <td>Radiation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9108cf4e9dae8634b2222365d27b43224c1ce0ff" target='_blank'>
              Deciphering the ghost proteome in ovarian cancer cells by deep proteogenomic characterization
              </a>
            </td>
          <td>
            Diego Fernando Garcia-Del Rio, M. Derhourhi, Amélie Bonnefond, Sébastien Leblanc, Noé Guilloy, X. Roucou, S. Eyckerman, Kris Gevaert, M. Salzet, T. Cardon
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Genome-wide association studies (GWAS) have identified independent signals at 5p15.33 across numerous cancers, with protective alleles for one cancer often conferring risk for another. Many of these associations are thought to act via allele-specific alterations in the cis- regulation of target genes. Transcriptomic analyses in multiple tissue types implicate two plausible target genes: TERT and CLPTM1L. However, the mechanisms linking cancer susceptibility alleles to their presumed target genes, along with the observed antagonistic pleiotropy between different cancers remain elusive. With this in mind, we employed a comprehensive approach, integrating statistical fine-mapping of GWAS summary statistics with massively parallel reporter assays (MPRA) and a densely tiled CRISPRi screen performed across eight cell lines, representative of four distinct cancers associated with 5p15.33 alleles – pancreatic, melanoma, lung, and bladder cancer. Integrative screening identified up to 30 variants across multiple signals in the locus with putative allele-specific transcriptional activity. Furthermore, a variable number tandem repeat (VNTR) in intron 9 of CLPTM1L emerged as a potent transcriptional enhancer in CRISPRi screening. Leveraging PacBio sequencing, VNTR alleles of European ancestry samples from the 1000 Genomes Project were genotyped and found to be highly polymorphic, with longer VNTRs linked to the haplotype tagged by rs31490-A (risk increasing for pancreatic cancer). Subsequent imputation into a pancreatic cancer GWAS of 9,040 cases and 12,496 controls revealed an association between longer VNTRs and risk of pancreatic cancer (OR for VNTR in the 75th percentile = 1.17, 95% CI = 1.13-1.22, P = 2.21x10-12). The significance of the association for the lead SNP rs31490 after conditioning on the VNTR genotype dropped to P= 2.20x10-5, suggesting that VNTR alleles merit consideration as part of the credible variant set for pancreatic cancer. Targeted CRISPRi assays suggested VNTR-mediated cis-regulation of both TERT and CLPTM1L in PANC-1 and MIA PaCa-2 pancreatic cancer cell lines, while dual luciferase reporter assays in the same cell lines revealed that VNTR enhancer activity is mediated by Hippo pathway transcription factors. In ongoing experiments, we aim to expand on luciferase reporter assays and DNA-protein binding assays in relevant cell lines with the aim of not only deciphering how longer VNTRs differentially exert their transcriptional effects compared to shorter alleles but also characterizing the divergent directions of effect between different cancer types. These findings underscore the proposition that pancreatic cancer susceptibility at the 5p15.33 locus may be mediated by both SNPs and an uncharted class of variants, yet to be characterized in the post-GWAS landscape.
 Citation Format: Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E Connelly, Mai Xu, Irene Collins, Jun Zhong, Jason Hoskins, Michelle Ho, Brenen Papenburg, Mark M Iles, Matthew H Law, Maria-Teresa Landi, Rachael Stolzenber- Solomon, Brian M Wolpin, Alison P Klein, Nick Orr, Stephen J Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, Laufey T Amundadottir. Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0777eadfb43a66af250607da2ed11cfe90c7930b" target='_blank'>
              Abstract C055: Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element
              </a>
            </td>
          <td>
            Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E. Connelly, Mai Xu, I. Collins, Jun Zhong, Jason W. Hoskins, Michelle Ho, Brenen Papenburg, M. Iles, M. Law, Marion Landi, Rachael Stolzenber- Solomon, B. Wolpin, Alison P Klein, N. Orr, S. Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, L. Amundadottir
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Introduction Breast cancer is one of the most prevalent types of cancer and a leading cause of cancer-related death among females worldwide. Anoikis, a specific type of apoptosis that is triggered by the loss of anchoring between cells and the native extracellular matrix, plays a vital role in cancer invasion and metastasis. However, studies that focus on the prognostic values of anoikis-related genes (ARGs) in breast cancer are scarce. Methods Gene expression data were obtained from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), and Molecular Taxonomy of Breast Cancer International Consortium (METABRIC) databases. Five anoikis-related signatures (ARS) were selected from ARGs through univariate Cox regression analysis, LASSO regression analysis, and multivariate Cox regression analysis. Subsequently, an ARGs risk score model was established, and breast cancer patients were divided into high and low risk groups. The correlation between risk groups and overall survival (OS), tumor mutation burden (TMB), tumor microenvironment (TME), stemness, and drug sensitivity were analyzed. Moreover, RT-qPCR was performed to verify the gene expression levels of the five ARS in breast cancer tissues. Furthermore, a nomogram model was constructed based on ARGs risk score and clinicopathological factors. Results A novel ARGs risk score model was constructed based on five ARS (CEMIP, LAMB3, CD24, PTK6, and PLK1), and breast cancer patients were divided into high and low risk groups. Correlation analysis showed that the high and low risk groups had different OS, TMB, TME, stemness, and drug sensitivity. Both the ARGs risk score model and the nomogram showed promising prognosis predictive value in breast cancer. Conclusion ARS could be used as promising biomarkers for breast cancer prognosis predication and treatment options selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2304d0d98fc1550ec1a30e39d41f3e284722407" target='_blank'>
              Identification of Novel Anoikis-Related Gene Signatures to Predict the Prognosis, Immune Microenvironment, and Drug Sensitivity of Breast Cancer Patients
              </a>
            </td>
          <td>
            Jiena Liu, Hao Wu, Qin Wang, Shengye Jin, Siyu Hou, Zibo Shen, Liuying Zhao, Shouping Xu, Da Pang
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Homologous recombination (HR) is an important mechanism for repairing DNA double-strand breaks (DSBs) and preserving genome integrity. Pathogenic mutations in the HR proteins BRCA2 and the RAD51 paralogs predispose individuals to breast, ovarian, pancreatic, and prostate cancer. The RAD51 paralogs: RAD51B, RAD51C, RAD51D, XRCC2, and XRCC3 form two complexes RAD51B-RAD51C-RAD51D-XRCC2 (BCDX2) and RAD51C-XRCC3 (CX3). Similar to BRCA2, loss of RAD51 paralog functions in mammalian cells lead to chromosomal abnormalities, growth defects, disrupted RAD51 foci formation, and PARP inhibitor sensitivity. Despite significant e]ort over the past three decades, the specific molecular functions of the human RAD51 paralogs have remained elusive due to technical challenges such as low protein expression in human cell lines and instability of the purified proteins. Recent studies have determined the molecular structures of the BCDX2 and CX3 complexes dramatically enhancing our understanding of these challenging proteins. Using multiple approaches, we demonstrate that the RAD51 paralogs interact with BRCA2 at two distinct interaction hubs located in the BRC repeats and the DNA binding domain. We confirm, using a yeast 3-hybrid approach, that human RAD51 paralogs interact directly with BRC repeats one and two (BRC1-2) of BRCA2. Because of the dynamic nature of the RAD51B C-terminal domain (CTD), identified in the recently solved cryo-EM structures, we focused on elucidating the interaction with RAD51B. We determined that BRCA2 interacts with the CTD of RAD51B and not the N-terminal domain (NTD) that is involved in stacking interactions with RAD51C and RAD51D. Furthermore, the interaction with RAD51B is dependent upon an FxxA motif located on a surface exposed region of the CTD. Our study has identified novel interactions between the RAD51 paralogs and BRCA2 and further demonstrated that a previously unrecognized FxxA motif located within a mobile element of RAD51B is critical for the interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08aaf8658412f259814459f47746d486e139a2ec" target='_blank'>
              RAD51 Paralogs and RAD51 Paralog Complexes BCDX2 and CX3 Interact with BRCA2
              </a>
            </td>
          <td>
            Jacob G. Thrasher, Adeola A. Faunloye, Fabricio S. Justiniano, Kara A. Bernstein, Ryan B. Jensen
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a–RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Asymmetric partition of fate determinants during cell division is a hallmark of cell differentiation. Recent works suggested that such a mechanism is hijacked by cancer cells to increase both their phenotypic heterogeneity and plasticity and in turn their fitness. To quantify fluctuations in the partitioning of cellular elements, imaging-based approaches are used, whose accuracy is limited by the difficulty of detecting cell divisions. Our work addresses this gap proposing a general method based on high-throughput flow cytometry measurements coupled with a theoretical frameworks. We applied our method to a panel of both normal and cancerous human colon cells, showing that different kinds of colon adenocarcinoma cells display very distinct extents of fluctuations in their cytoplasm partition, explained by an asymmetric division of their size. To test the accuracy of our population-level protocol, we directly measure the inherited fractions of cellular elements from extensive time-lapses of live-cell laser scanning microscopy, finding excellent agreement across the cell types. Ultimately, our flow cytometry-based method promise to be accurate and easily applicable to a wide range of biological systems where the quantification of partition fluctuations would help accounting for the observed phenotypic heterogeneity and plasticity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89534ca95bbe7d2c00c135f356aa0991c7b534e4" target='_blank'>
              Robust assessment of asymmetric division in colon cancer cells
              </a>
            </td>
          <td>
            Domenico Caudo, Chiara Giannattasio, Simone Scalise, V. Turris, Fabio Giavazzi, G. Ruocco, Giorgio Gosti, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-09-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec25d0d3af6b468f3736b6388bc33a3af02a628" target='_blank'>
              DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor
              </a>
            </td>
          <td>
            Ninoslav Pandiloski, V. Horváth, Ofelia Karlsson, Symela Koutounidou, F. Dorazehi, Georgia Christoforidou, Jon Matas-Fuentes, Patricia Gerdes, R. Garza, M. Jönsson, Anita Adami, Diahann A. M. Atacho, Jenny G. Johansson, Elisabet Englund, Z. Kokaia, Johan Jakobsson, C. Douse
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The kinetochore complex, an important protein assembly situated on the centromere, plays a pivotal role in chromosome segregation during cell division. Like in animals and fungi, the plant kinetochore complex is important for maintaining chromosome stability, regulating microtubule attachment, executing error correction mechanisms, and participating in signaling pathways to ensure accurate chromosome segregation. This review summarizes the composition, function, and regulation of the plant kinetochore complex, emphasizing the interactions of kinetochore proteins with centromeric DNAs (cenDNAs) and RNAs (cenRNAs). Additionally, the applications of the centromeric histone H3 variant (the core kinetochore protein CENH3, first identified as CENP-A in mammals) in the generation of ploidy-variable plants and synthesis of plant artificial chromosomes (PACs) are discussed. The review serves as a comprehensive roadmap for researchers delving into plant kinetochore exploration, highlighting the potential of kinetochore proteins in driving technological innovations in synthetic genomics and plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f840a6e5c8eb7572f7b8a202d1475f565b0be6f0" target='_blank'>
              Plant kinetochore complex: composition, function, and regulation
              </a>
            </td>
          <td>
            Yuqian Xie, Mingliang Wang, Beixin Mo, Chao Liang
          </td>
          <td>2024-10-10</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7fe64901efe5b8716a11b155e04e670416432b8" target='_blank'>
              Endogenous retroviral ERVH48-1 promotes human urine cell reprogramming
              </a>
            </td>
          <td>
            Yuling Peng, Jieying Zhu, Qi Zhang, Ran Zhang, Zhenhua Wang, Zesen Ye, Ning Ma, Dajiang Qin, Duanqing Pei, Dongwei Li
          </td>
          <td>2024-09-13</td>
          <td>Cell Regeneration</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Diaphanous-related formin 3 (Diaph3) belongs to the formin family of dimeric multidomain proteins that are master regulators of actin and microtubule dynamics. It plays a role in the cytoskeleton remodeling, more specifically in cytokinesis and karyokinesis. In the mouse brain, loss of Diaph3 induced aneuploidy followed by massive apoptosis in neural stem cell, leading to microcephaly. Given the potential role of aneuploidy in cancer development, we are interested to unravel whether Diaph3 depletion during gliomagenesis could aggravate glioblastoma (GBM) aggressiveness



 To analyze whether Diaph3 is a potential therapeutic target in human GBM, we will use siRNA-mediated depletion of Diaph3 in a panel of GBM cell lines (U251 and U87). We conducted karyotyping followed by analysis of the DNA Damage response (DDR) induction, DNA repair and cell cycle progression.



 Our results showed that siRNA-mediated depletion of Diaph3 in p53 mutated GBM cells (U251 cells) disrupted microtubules and caused mitotic defects such as multipolar mitosis. Karyotyping showed an increase in aneuploidy in Diaph3-depleted U251 cells. Since it has been shown that numerical abnormalities in chromosomes segregation could trigger DDR, we hypothesize that the loss of Diaph3 induces increased activity of the DDR pathway and we aim to decipher their underlying molecular mechanism.



 We believe that this study may provide insights into the mechanisms by which Diaph3 prevents aneuploidy in glioblastoma cells but also, more generally, into the role of aneuploidy in gliomagenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7194c4ebe63addf64f3762f0c016671715c067f" target='_blank'>
              P05.02.B MOLECULAR MECHANISMS OF ANEUPLOIDY AND GLIOBLASTOMA AGGRESSIVENESS INDUCED BY DIAPH3 LOSS
              </a>
            </td>
          <td>
            C. Jabbour, N. Tajeddine
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The role of Epiplakin1 (EPPK1) in colon adenocarcinoma (COAD) was analyzed through a comprehensive evaluation of its expression, methylation, genetic mutations, and prognostic implications. A significant up-regulation of EPPK1 expression in COAD malignant cells compared to normal control samples was observed using data from the UALCAN database. EPPK1 expression was found to be elevated across different cancer development stages, racial groups, genders, and age groups, emphasizing its crucial role in cancer proliferation. Validation of EPPK1 expression through the GEPIA2.0 dataset further confirmed its overexpression in COAD when compared to normal samples. The analysis revealed dysregulation across all four stages of cancer development, with the highest expression in stage IV and the lowest in stage II. Additionally, promoter methylation analysis demonstrated a fundamental relationship between COAD samples and normal controls, revealing significant methylation patterns across different clinical parameters, including cancer stages, race, gender, and age. Overall survival (OS) and disease-free survival (DFS) analysis using the KM plotter showed a strong association between high EPPK1 expression and worse survival outcomes in COAD patients. Conversely, low EPPK1 expression was linked to better OS and DFS. Genetic mutation analysis using cBioPortal identified minimal EPPK1 mutations in COAD, predominantly truncating and missense mutations, highlighting their relevance to EPPK1 dysregulation in COAD. These findings underscore the important role of EPPK1 in COAD development and proliferation, suggesting its potential as a therapeutic target and prognostic marker. Further exploration of EPPK1’s molecular mechanisms and its involvement in the COAD microenvironment may reveal new pathways for targeted treatments and precision medicine strategies against this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1746eb2037e9f374ca2ed51f59d8d79d309f8805" target='_blank'>
              Deciphering the Role of EPPK1 in the Development of Colon Adenocarcinoma by Integrated Bioinformatics Approach
              </a>
            </td>
          <td>
            Muhammad Akram, Muhammad Umair Abid
          </td>
          <td>2024-10-24</td>
          <td>Journal of Clinical and Nursing Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Childhood acute lymphoblastic leukemia (ALL) is a common pediatric cancer. The heterogeneous characterization of B cells in ALL progression poses new challenges to researchers. We used single-cell sequencing to explore the critical role of B cells in regulating the ALL immune microenvironment. Method We collected the single cell (sc) RNA-seq data of ALL and health sample from the gene expression omnibus (GEO) database, the “Seurat” and “harmony” R package was used for quality control and scRNA-seq analysis, in which the CellMarker2.0 database was used for cell type annotation. Subsequently, the FindAllMarkers function was used to identify the differentially expressed genes (DEGs) among various cell types and the DAVID database was applied for the biological process of DEGs. Then, the “inferCNV” package was used for copy number variation, regulons and cell communication were performed by SCENIC tool and CellChat package. The role of the target gene in regulating ALL progression was assessed using RT-qPCR, Transwell and scratch healing assays. Results We identified nine mainly cell clusters after scRNA-seq analysis, in which the B cells had higher infiltration proportion in the ALL samples and were sub-clustered into five cell sub-groups. The B cells 1 is closely associated with cell proliferation and stemness (TNFAIP3 and KDM5B), and the significant CNV of amplification occurred on chr6 and chr21 that supported stemness of B cells1. RXRB is a key transcription factor mediated the proliferation of B cells 1, which in turn suppressed hematopoietic stem cells (HSCs) proliferation and promoted cytotoxic NK/T cells activation through diverse cell communication ways. One of the key regulators of B cells is MYC, which promotes the migration and invasive ability of cell line leukemia cell lines. Conclusion This study reveals the stemness characteristics of B cells and their critical role in ALL progression, a finding that provides new potential directions for the development of targeted therapies against ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dad923c86f54a54917cb6e376b8b3d5aae7831e" target='_blank'>
              Single-cell RNA-seq analysis revealed the stemness of a specific cluster of B cells in acute lymphoblastic leukemia progression
              </a>
            </td>
          <td>
            Guifang Wang, Ensheng Zhang, An Chen, Dachuan Meng
          </td>
          <td>2024-10-21</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain, where they are named AT/RT. RTs are characterized by the inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex, which is the only recurrent genetic event, rendering this malignancy a prototype of epigenetically-driven cancers. Hence, targeting epigenetic modifyers by so-called « epidrugs » is an attractive hypothesis for treatment of RTs. Interestingly, several studies have pointed out the overexpression of de novo DNA methyl transferases, i.e. DNMT3A and DNMT3B, in RTs, but their role in RT oncogenesis remains to be defined. Of note, we noticed that DNMT3A and DNMT3B expression is induced by SMARCB1 loss in a composite tumor that harbors both SMARCB1 proficient and SMARCB1 deficient components, which methylation profiles strongly differ from each other. To investigate their actual role in RTs, we knocked-out DNMT3A or DNMT3B in the I2A inducible RT cell line allowing the re-expression of SMARCB1. We demonstrated that both DNMT3A and DNMT3B KO affected I2A cell viability, showing the RT vulnerability to those genes, but with a more important dependency to DNMT3B than to DNMT3A. DNMT3B loss largely impacted gene expression and DNA methylation, much more than DNMT3A loss. We further investigated the overlap between DNMT3A/DNMT3B and SMARCB1 functions at the transcriptional level. We found that 43% of the genes deregulated after SMARCB1 re-expression were also deregulated by DNMT3B loss and mostly in the same direction. Those commonly differentially regulated genes were involved in development and cell adhesion. In contrast, only 19% of the genes deregulated after SMARCB1 re-expression were deregulated after DNMT3A loss. In conclusion, DNMT3B KO recapitulates SMARCB1 re-expression effect on gene expression showing that DNMT3B and SMARCB1 play antagonistic roles in cell adhesion and developmental programs. We finally demonstrated that the use of nanaomycin A, a specific DNMT3B inhibitor, as strong cytotoxic effects on RT cell lines, confirming that targeting DNMT3B offers a true vulnerability for RT that opens promising perspectives for innovative treatment.
 Citation Format: Céline Chauvin, Mamy Andrianteranagna, Sandrina Turczynski, Joanna Cyrta, Arnault Tauziede-Espariat, Rachida Bouarich, Zhi-Yan Han, Julien Masliah-Planchon, Gaelle Pierron, Mylène Bohec, Sylvain Baulande, Sergio Roman-Roman, Olivier Delattre, Franck Bourdeaut. DNMT3B DNA methyltransferase acts as a major player and therapeutic target in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A050.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070d8b7b763cbaadeb29946c3ff7884b9e6d490f" target='_blank'>
              Abstract A050 DNMT3B DNA methyltransferase acts as a major player and therapeutic target in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Mamy Andrianteranagna, Sandrina Turczynski, Joanna Cyrta, A. Tauziède-Espariat, Rachida Bouarich, Zhi-Yan Han, J. Masliah-Planchon, G. Pierron, Mylène Bohec, Sylvain Baulande, S. Roman-Roman, Olivier Delattre, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93b37a3e55555e578d0b3e673e940425e7441335" target='_blank'>
              Anoikis-related genes in breast cancer patients: reliable biomarker of prognosis
              </a>
            </td>
          <td>
            Mingzheng Tang, Yao Rong, Xiaofeng Li, Haibang Pan, Pengxian Tao, Zhihang Wu, Songhua Liu, Renmei Tang, Zhilong Liu, Hui Cai
          </td>
          <td>2024-09-19</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Li-Fraumeni Syndrome (LFS) is a genetic predisposition disorder associated with pathogenic germline Tp53 mutations. A subset of LFS-associated neoplasms occurs at an increased risk within the first 5 years of life, with the most common being soft tissue sarcomas (STS). Histological characterization of STS is one of the primary approaches to guiding treatment. However, the heterogeneity of STS has necessitated a further understanding of the molecular etiology of these tumors. Examining sarcomas using a single-cell transcriptomic approach has shown unique subsets of cells with dysregulations in cell fate and differentiation. Within LFS-associated tumors, including LFS-STS a major event in tumorigenesis is the copy-neutral loss of heterozygosity (LOH). LOH was determined to occur much earlier than cancer onset in LFS, suggesting a dormancy period from the LOH until tumorigenesis; cancer-defining events in LFS are likely occurring prenatally. During embryogenesis, Tp53 is involved in differentiation and pluripotency regulation, as well as cell cycle control and apoptosis. Here we hypothesize that the emergence of STS in LFS mice is associated with distinct disturbances in the cell fate and differentiation pathways. Specifically, there will be deficits in cell fate determination and the organization of cells across multiple cell types in LFS-Sarcomas. Single-cell RNAseq was performed on spontaneous tumors and healthy matched muscle tissue from male and female C57BL/6J Trp53R172H/WT mice. Cell types were determined based on differential gene expression between clusters defined by principal component analysis. Despite tumor heterogeneity between mice, there were consistent disruptions of developmental pathways among tumor cells. Moreover, when comparing matched cell types across tumor and healthy muscle tissue we observed dysregulation of pathways promoting tumorigenesis, and cell fate determination in cells originating from tumor tissue. For the first time to our knowledge, this study examines the single-cell transcriptome of LFS-STS. Current literature has speculated on the presence of an early precancer state of LFS-associated tumors. This study will provide evidence to determine whether this precancer state can be determined in tumor samples, and identify elements that constitute this state. Moreover, we will determine changes in supporting non-tumor cells and whether they also exhibit dysregulations in cell state and differentiation. Together these findings will hopefully contribute to improvements in molecular subtyping across STS, and particularly within LFS individuals.
 Citation Format: Ashby Kissoondoyal, David Malkin. Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc62fb2bdb474c15b956e6aac5478623a2fc2613" target='_blank'>
              Abstract A009 Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The viral intra-host genetic diversities and interactions with the human genome during decades of persistence remain poorly characterized. In this study, we analyzed the variability and integration sites of persisting viruses in nine organs from thirteen individuals who died suddenly from non-viral causes. The viruses studied included parvovirus B19, six herpesviruses, Merkel cell (MCPyV) and JC polyomaviruses, totaling 127 genomes. The viral sequences across organs were remarkably conserved within each individual, suggesting that persistence stems from single dominant strains. This indicates that intra-host viral evolution, thus far inferred primarily from immunocompromised patients, is likely overestimated in healthy subjects. Indeed, we detected increased viral subpopulations in two individuals with putative reactivations, suggesting that replication status influences diversity. Furthermore, we identified asymmetrical mutation patterns reflecting selective pressures exerted by the host. Strikingly, our analysis revealed non-clonal viral integrations even in individuals without cancer. These included MCPyV integrations and truncations resembling clonally expanded variants in Merkel cell carcinomas, as well as novel junctions between herpesvirus 6B and mitochondrial sequences, the significance of which remains to be evaluated. Our work systematically characterizes the genomic landscape of the tissue-resident virome, highlighting potential deviations occurring during disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36ceef157a914376048c24c979dc2d5bf598175" target='_blank'>
              Intra-host genomic diversity and integration landscape of human tissue-resident DNA virome.
              </a>
            </td>
          <td>
            Lari Pyöriä, Diogo Pratas, M. Toppinen, Peter Simmonds, Klaus Hedman, Antti Sajantila, M. Perdomo
          </td>
          <td>2024-10-21</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The adoptive cell transfer of ex vivo expanded tumor infiltrating lymphocytes (i.e., TIL therapy) is a promising clinical strategy and recently FDA approved for melanoma but has major limitations including that not all tumors are inflamed. Moreover, tumor-specific clones can be rare and in an exhausted state due to the suppressive tumor microenvironment. These obstacles can be overcome by engineering autologous peripheral blood T cells with pre-selected T cell receptors (TCRs) by viral vector-mediated gene insertion. While viral transduction is highly efficient, the insertional site is not specific and persistence of the T cells is oftentimes limited. In contrast, site-specific integration of the TCR into the TCR α chain (TRAC) locus by CRISPR/Cas9 has been shown to enable more consistent and physiological levels of exogenous TCR expression coupled with superior persistence and tumor control in preclinical studies. Here, we sought to improve the efficiency of CRISPR/Cas9 mediated TCR knockin (KI) into the TRAC locus of primary human T cells. In addition to the previously reported DNA-dependent protein kinase inhibitor M3814, we demonstrate that PI-103 and samotolisib markedly increase KI efficiency in a process that is GMP-compatible, while CC-115 had a variable effect. Importantly, PI-103 and samotolisib do not negatively impact cell viability, fold-expansion nor T cell phenotype and we conclude that they are suitable for the generation of gene-modified T cells for clinical use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb5c2cf46440f225983760473934c3ee7b03b3c2" target='_blank'>
              DNA-Dependent Protein Kinase Inhibitors PI-103 and Samotolisib Augment CRISPR/Cas9 Knockin Efficiency in Human T Cells
              </a>
            </td>
          <td>
            Emina Džafo, Morteza Hafezi, G. M. P. Giordano Attianese, Patrick Reichenbach, Stephane Grillet, Kirsten Scholten, George Coukos, M. Irving, Bernhard Gentner
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Recent studies have linked certain colibactin producing subphylas of E. coli to colorectal cancer. The bacterium synthesizes colibactin via the pks genomic island containing a series of clbA to clbS genes. The genotoxin causes inter-strand DNA cross-links, which lead to double strand breaks and promotion of colorectal adenocarcinoma. The published studies have prioritized the colorectal regions of the body where E. coli is commonly found, but none have identified the same genotoxic relationship with other types of adenocarcinoma. In this study, we evaluate DNA damage and cellular changes of mammary breast tissue based on high and low levels of colibactin producing E. coli exposure. The goal is to evaluate if colibactin can affect cells in the mammary breast tissue promoting breast adenocarcinoma growth.



 Two cell lines were acquired from ATCC: primary mammary epithelial cells (PCS-600-010) and MCF-7 breast adenocarcinoma cells (HTB-22). Both cell lines were split into six culture flasks and maintained until ∼80% confluence was reached using the ATCC cell culture protocols. Once all the flasks had reached the target confluence, they were divided into exposure groups of high and low concentrations of colibactin producing E. coli, a group of non-colibactin producing E. coli, a group exposed only to phosphate buffered saline, and a control group with no exposure. Cellular morphology was observed using light microscopy before, after exposure, and every 5 minutes up to 15 minutes. E. coli and colibactin DNA concentrations were quantified using quantitative polymerase chain reaction. Genomic DNA was extracted from each group of cells. The DNA from each group was whole genome sequenced and analyzed for genomic variation.



 The normal cells exposed to E. coli showed immediate lysing and continued to lyse after a 5 minute incubation; however, the adenocarcinoma positive cells showed no lysing up to 30 minutes incubation. Sequencing results show genomic variation between the groups and specific break points associated with the E. coli exposed groups.



 Our results show that the presence of E. coli with mammary breast tissue can destroy normal tissue, promote lysis resistance, and allow adenocarcinoma cells to continue to grow. The colibactin producing E. coli does cause double stranded breaks and can lead to cell death. This new information points out a critical need to monitor the microbiome and infection levels of people at risk or in early stages of cancer to prevent adenocarcinoma growth and destruction of normal tissue. This study indicates that colibactin producing E. coli may play a larger role in tissue damage and promotion of other types of cancer. More research is needed to elucidate the molecular pathway and mechanisms of colibactin, which will lead to new diagnostics or therapeutics that monitor E. coli levels or inhibit colibactin’s toxic effects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b3fd078061817b8e9827d3c0f82f869fc3cb02" target='_blank'>
              B-351 Genotoxic Producing Escherichia coli Protects and Promotes Breast Adenocarcinoma
              </a>
            </td>
          <td>
            L. Heaton, S. D. Figard, R. A. Garcia
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Nuclear envelope repair is a fundamental cellular response to stress, especially for cells experiencing frequent nuclear ruptures, such as cancer cells. Moreover, for chromosomally unstable cancer cells, characterized by the presence of micronuclei, the irreversible rupture of these structures constitutes a fundamental step toward cancer progression and therapy resistance. For these reasons, the study of nuclear envelope rupture and repair is of paramount importance. Nonetheless, due to the constraint imposed by the stochastic nature of rupture events, a precise characterization of the initial stage of nuclear repair remains elusive. In this study, we overcame this limitation by developing a new imaging pipeline that deterministically induces rupture while simultaneously imaging fluorescently tagged repair proteins. We provide a detailed step-by-step protocol to implement this method on any confocal microscope and applied it to study the major nuclear repair protein, barrier-to-autointegration factor (BAF). As a proof of principle, we demonstrated two different downstream analysis methods and showed how BAF is differentially recruited at sites of primary and micronuclear rupture. Additionally, we applied this method to study the recruitment at primary nuclei of the inner nuclear membrane protein LEM-domain 2 (LEMD2) and Charged Multivesicular Protein 7 (CHMP7), the scaffolding protein of the endosomal sorting complex required for transport III (ESCRT-III) membrane remodeling complex. The CHMP7-LEMD2 binding is the fundamental step allowing the recruitment of ESCRT-III, which represents the other major nuclear repair mechanism. This demonstrates the method’s applicability for investigating protein dynamics at sites of nuclear and micronuclear envelope rupture and paves the way to more time-resolved studies of nuclear envelope repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a3eed75e1523ea3787b3b825a08535004cedcb1" target='_blank'>
              A new microscopy pipeline for studying the initial stages of nuclear and micronuclear rupture and repair
              </a>
            </td>
          <td>
            Melody Di Bona, S. Bakhoum
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Kinetochores link chromosomes to dynamic microtubules of the mitotic spindle. To ensure equal chromosome segregation, sister chromatids must attach to microtubules from opposing spindle poles, a configuration referred to as biorientation. To avoid chromosome missegregation the conserved kinase Aurora B phosphorylates outer kinetochore proteins on attachments lacking tension, allowing re-establishment of new connections until biorientation is achieved. While Aurora B localizes to the centromere and inner kinetochore as part of the chromosomal passenger complex (CPC), the underlying recruitment pathways can be eliminated without fully disrupting biorientation. It therefore remains unclear how the kinase operates during error correction and whether additional mechanisms may facilitate tension-dependent phosphorylation of substrate proteins. In this study, we identify the conserved KMN subcomplex Spc105Knl1/Kre28Zwint as an outer kinetochore receptor of the Aurora kinase Ipl1 in Saccharomyces cerevisiae. We show that mutations in the helical bundle domain of Spc105/Kre28 cause defective mitotic error correction, closely resembling the effects of ipl1 or sli15 mutants. In biochemical experiments we show that Ipl1/Sli15 directly associates with the KMN network via the Spc105/Kre28 subcomplex and that binding is inhibited by Ipl1 autophosphorylation. The phenotype of the kre28 mutant cells can be efficiently suppressed by artificial recruitment of Ipl1, demonstrating the importance of Ipl1 docking to the outer kinetochore. These results have important implications for the mechanism of tension-dependent error correction during chromosome biorientation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f1b600818ea86aa42d2a4d09a969fbd96bff7c" target='_blank'>
              The Spc105Knl1/Kre28Zwint complex promotes mitotic error correction by outer kinetochore recruitment of the Ipl1Aurora B kinase
              </a>
            </td>
          <td>
            Alexander Dudziak, Jasmin Schmidt, Frederik Hamm, Sharvari Tendulkar, Karolin Jänen, Ingrid R. Vetter, Sylvia Singh, Josef Fischböck, Franz Herzog, Stefan Westermann
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematologic malignancies represent a diverse group of blood tumors accounting for approximately 10% of all cancer cases in the US. Based on the site of manifestation, presumed cell of origin, genetic abnormalities, and clinical features, we recognize more than 100 clinically important subtypes of hematologic malignancies broadly categorized into three main groups: leukemia, lymphoma, and multiple myeloma. While the prognosis for patients with specific subtypes of hematologic malignancies, such as Hodgkin lymphomas or chronic lymphocytic leukemia has significantly improved over the last several years, other subtypes such as acute myeloid leukemia or non-Hodgkin lymphomas still present with significant treatment challenges. A better understanding of molecular drivers of tumor maintenance is needed to improve existing therapies. Thyroid hormone receptor interactor 13 (TRIP13) is an AAA + ATPase that plays an important yet poorly understood role in the regulation of the mammalian cell cycle. The TRIP13 gene is highly expressed in various human tumors including colorectal carcinoma and glioblastoma, and it seems to promote tumorigenesis. Recently, we identified TRIP13 as a gene overexpressed in Peripheral T-cell lymphomas where it plays a critical role in tumor maintenance by regulating the G2-M phase of the cell cycle. However, the biological activities of TRIP13 in non-T cell hematologic malignancies have not been investigated to date. Here we used shRNA-mediated knockdown and a small molecule inhibitor to downregulate TRIP13 in B-cell lymphoid and myeloid lineages. We found genetic downregulation inhibited the proliferation of both B-cell lymphomas and acute myeloid leukemia cell lines in vitro. The pharmacologic inhibition of TRIP13 effectively induced the G0/G1 cell cycle arrest and apoptosis of B-cell lymphomas. Altogether these data demonstrate that TRIP13 is critical for the maintenance of hematologic malignancies of B-cell and myeloid malignancies in vitro. Thus, TRIP13 might be a good target in the treatment of various types of hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/259c736f6ff3f543df5c163334eab4381b3158b7" target='_blank'>
              TRIP13 is Critical for the Survival of B-cell Lymphomas and Myeloid Leukemias In Vitro
              </a>
            </td>
          <td>
            Gene Pozas, Julian Tobon, Reem Abdelghany, Jacob Fingeret, Emma Noel, Vedant Ramachandran, Arshee Badar, J. Opavska, Rene Opavsky
          </td>
          <td>2024-10-16</td>
          <td>UF Journal of Undergraduate Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncoviruses can integrate into the host genome and cause tumorigenesis. In particular, hepatitis B virus (HBV) infection accounts for more than 50% of hepatocellular carcinoma (HCC) worldwide. We revealed the global geographical disparity of HBV integration that the landscape of HBV integration between HCC tumor and non-tumorous liver varied in regional cohorts, suggesting the different degrees of clonal enrichment. Most HBV integrations were positionally enriched at telomeres and centromeres (T&C) and they highlighted the novel co-involvement of HBV integration, which likely introduces genomic instability in HCC development. This was confirmed by phospho-H2AX staining. We constructed a large meta-cohort of multiple ethnicities to refine the landscape of HBV integration. This enables the gene set/family level exploration. As TERT is the most frequently integrated gene, we further investigated the underlying mechanistic modulation of TERT transcription activation and revealed the concurrent influence by the orientation and relative distance of HBV integration. Additionally, clonal disparity of HBV integration was observed among patients and the higher level of clonal disparity score can indicate poor patients' prognostication. Taken together, our study uncovered the different levels of clonal enrichment of HBV integration, mechanistic insights, and prognostic biomarker signature, to strengthen our understanding in HBV-associated hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ec7bd105c83b5d222005af0db05b395d835b64" target='_blank'>
              Disparity landscapes of viral-induced structural variations in hepatocellular carcinoma: Mechanistic characterization and functional implications.
              </a>
            </td>
          <td>
            Xueying Lyu, K. Sze, J. Lee, Abdullah Husain, Lu Tian, S. Imbeaud, J. Zucman-Rossi, I. Ng, DanielWai-Hung Ho
          </td>
          <td>2024-09-13</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d871afd05449a9bea959b164b547c76fe5c42e1b" target='_blank'>
              Single-cell sequencing analysis revealed that NEAT1 was a potential biomarker and therapeutic target of prostate cancer
              </a>
            </td>
          <td>
            Xing Li, Yanjun Li, Lei Zhang, Huimin Long
          </td>
          <td>2024-10-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Philadelphia-positive leukemia is characterized by the translocation t(9;22)(q34;q11), which results in the oncogene BCR-ABL1 and a chromosome called the Philadelphia (Ph) chromosome (chr). The Philadelphia chromosome is best characterized in Chronic Myeloid Leukemia (CML); however, the translocation is also found in Acute Lymphoblastic Leukemia (ALL). Philadelphia-positive ALL has been historically understudied, especially in minority populations. Genome organization is a highly complex process by which chromatin is segregated within the nucleus into specific domains, making stable translocations highly unfavorable. How translocations are stabilized has been historically understudied. Here, we aim to examine how the Ph chr is stabilized and be the first to visualize an entire abnormal chr with high spatial and genomic resolution at the single-cell level. Using OligoSTORM, a super-resolution microscopy technique, we aim to image the entirety of the Ph chr at a nanoscale resolution in both CML and ALL cell lines. In addition to the DNA super-resolution trace, we have designed an RNA library to image a series of RNAs from the Ph chr on the same sample as the DNA trace, allowing us to connect the transcriptional data to the chromatin structure. We have successfully traced the entirety of the Ph chr. We have preliminary data suggesting that chr9 and chr22 segregate close to each other in non-leukemia healthy cells. We have also proven that our library design and methodology work to achieve the primary 500kb resolution. Preliminary data suggests that chr9 and chr22 in cells, that don’t typically have a Ph chr, segregate within proximity to translocate successfully. This study aims to compare the 3D structure of CML, ALL and non-leukemia cell lines to potentially identify genome organization-based biomarkers for identify and targeting the Philadelphia chromosome.
 Citation Format: Gabrielle S. Dewson, Guy Nir. Unraveling the structure of the Philadelphia chromosome using super-resolution microscopy [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df4f07534688b618ef1b83a3e8ca35d3688e2a3" target='_blank'>
              Abstract C015: Unraveling the structure of the Philadelphia chromosome using super-resolution microscopy
              </a>
            </td>
          <td>
            Gabrielle S. Dewson, Guy Nir
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aiming to identify novel driver mutations that cause pancreatic ductal adenocarcinoma (PDAC), besides p53 mutations, we performed a genetic interaction analysis based on the concept of mutually exclusive mutations, which suggests that mutations exclusive to one another function within the same pathway. Using our unbiased quantitative methods to evaluate the effects of somatic mutations on cancer, we analyzed 3900 human PDAC cases to prioritize mutations among hundreds that appear at low frequencies (5-10%) and are mutually exclusive with p53 mutations. Mutations in RNA splicing factors SF3B1 and RBM10 (∼15% of cases) were among the most significant and mutually exclusive to mutant p53. Thus, we hypothesized that aberrant RNA splicing promoted by mutant SF3B1K700E and truncated RBM10 (RBM10 loss) lead to tumorigenesis and therapy resistance. Through the generation of engineered mouse models to co-express KrasG12D with either Sf3b1K700E or Rbm10 loss in pancreatic cells, we observed that mice harboring KrasG12D and Sf3b1 K700E or Rbm10 loss—like mutant p53—caused PDAC in mice. To identify the RNA splicing mechanism that leads to PDAC formation, we performed deep RNA sequencing and unbiased transcriptome-wide splicing analyses in isolated PDAC cells bearing wild-type SF3B1K700E and RBM10 loss derived from murine and patient PDACs. We established that SF3B1 and RBM10 mutations induce wide splicing changes, primarily affecting exon selection. Since two-thirds of these changes may lead to nonsense-mediated decay (NMD), a mechanism that eliminates aberrant mRNA, we tested whether these changes could trigger this response. Using cycloheximide-dependent NMD inhibition, we confirmed that these mutations cause NMD, resulting in post-transcriptional gene expression loss in mouse chromosomes 2, 7, 11, and 17, also conserved in human models. Intriguingly, recent studies in Nature (2022) have shown that in murine PDACs with loss-of-function p53 mutations, genetic loss of chromosomes 11 and 7 is necessary for PDAC development. These results suggest that mutations in SF3B1 and RBM10, and p53 loss, affect the same chromosomal loci, albeit through different mechanisms, to promote PDAC, supporting the mutual exclusivity findings from patient samples. To determine a personalized therapy for patients with mutant splicing-factor tumors, we found that PDACs with these mutations are more sensitive to Gemcitabine, a component of a PDAC standard-of-care therapy. Moreover, these mutations sensitize cells to small-molecule splicing inhibitor H3B-8800— currently in phase 2 trial for hematologic malignancies. Lastly, we found that combining Gemcitabine and H3B-8800 synergizes to kill mutant splicing cells selectively in vitro and in vivo. We are starting a phase 2 trial for Gemcitabine/Abraxane with escalating doses of H3B-8800 for tumors that have either SF3B1K700E, RBM10 loss, or neomorphic p53, all of which have aberrant RNA splicing. This work has determined novel drivers and mechanisms of PDAC development and a precision therapeutic strategy for treating PDAC patients.
 Citation Format: Natasha Pinto Medici, Diana Martinez-Saucedo, Danny Lee, Tianyi Chu, Robert Tseng, Vincent Cannataro, Jeffrey Townsend, Christine Iacobuzio-Donahue, Marie Robert, Omar Abdel-Wahab, Steven D. Leach, Luisa Escobar-Hoyos. Altered mRNA splicing mimics chromosome loss and drives pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd337e10f64987a6e620b0612482d00a55126cb8" target='_blank'>
              Abstract PR-11: Altered mRNA splicing mimics chromosome loss and drives pancreatic cancer
              </a>
            </td>
          <td>
            Natasha Pinto Medici, Diana Martinez-Saucedo, Danny Lee, Tianyi Chu, Robert Tseng, Vincent L. Cannataro, Jeffrey P. Townsend, Christine A. Iacobuzio-Donahue, Marie Robert, Omar Abdel-Wahab, Steven D. Leach, Luisa F. Escobar‐Hoyos
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>