<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-31 09:28:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>30</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>107</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>320</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>279</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis. Both breakage fusion bridge (BFB) cycles and extrachromosomal DNA (ecDNA) can lead to high oncogene copy numbers, but the impact of BFB amplifications on intratumoral heterogeneity, treatment response, and patient survival remains poorly understood due to detection challenges with DNA sequencing. We introduce an algorithm, OM2BFB, designed to detect and reconstruct BFB amplifications using optical genome mapping (OGM). OM2BFB demonstrates high precision (>93%) and recall (92%) in identifying BFB amplifications across cancer cell lines, patient-derived xenograft models, and primary tumors. Comparisons using OGM reveal that BFB detection with our AmpliconSuite toolkit for short-read sequencing also achieves high precision, though with reduced sensitivity. We identify 371 BFB events through whole genome sequencing of 2557 primary tumors and cancer cell lines. BFB amplifications are prevalent in cervical, head and neck, lung, and esophageal cancers, but rare in brain cancers. Genes amplified through BFB exhibit lower expression variance, with limited potential for regulatory adaptation compared to ecDNA-amplified genes. Tumors with BFB amplifications (BFB(+)) show reduced structural heterogeneity in amplicons and delayed resistance onset relative to ecDNA(+) tumors. These findings highlight ecDNA and BFB amplifications as distinct oncogene amplification mechanisms with differing biological characteristics, suggesting distinct avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="As a well-known cancer risk region, the 8q24 locus is frequently amplified in a variety of solid tumors. Here we identify a pseudogene-derived oncogenic lncRNA, ZNF252P, which is upregulated in a variety of cancer types by copy number gain as well as c-Myc-mediated transcriptional activation. Mechanistically, ZNF252P binds and drives “phase separation” of HNRNPK and ILF3 protein in the nucleus and cytoplasm, respectively, to transcriptionally and posttranscriptionally activate c-Myc, thus forming a c-Myc/ZNF252P/c-Myc positive feedback loop. These findings expand the understanding of the relationship between genomic instability in the 8q24 region and tumorigenesis and clarify a regulatory mechanism involved in transcription and posttranscription from the perspective of RNA-mediated nuclear and cytoplasmic protein phase separation, which sheds light on the dialogue with the driver oncogene c-Myc. The pivotal regulatory axis of ZNF252P/c-Myc has potential as a promising biomarker and therapeutic target in cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8033a50256e64b94ba359a06046d4ac24e218fe" target='_blank'>
              Targeting SUV4-20H epigenetic enzymes as therapeutic strategy for enhancing topoisomerase II poisoning in prostate cancer
              </a>
            </td>
          <td>
            Fatima Alhourani, Marine Tauziet, Julie Patouillard, Benoit Miotto, Véronique Baldin, Aurélie Gennetier, Simon George, Xavier Miahle, Cyril Ribeyre, Philippe Pourquier, Florence M. Cammas, A. Torro, Diego Tosi, Céline Gongora, Mona Dergham, Raghida Abou-Merhi, Eric Julien
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background SETD2 is the sole epigenetic factor responsible for catalyzing histone 3, lysine 36, tri-methylation (H3K36me3) in mammals. Its role in regulating cellular processes such as RNA splicing, DNA repair, and spurious transcription initiation underlies its broader tumor suppressor function. SETD2 mutation promotes the epithelial-mesenchymal transition and is clinically associated with adverse outcomes highlighting a therapeutic need to develop targeted therapies against this dangerous mutation. Results We employ an unbiased genome-wide synthetic lethal screen, which identifies another H3K36me writer, NSD1, as a synthetic lethal modifier in SETD2-mutant cells. Confirmation of this synthetic lethal interaction is performed in isogenic clear cell renal cell carcinoma and immortalized renal epithelial cell lines, in mouse and human backgrounds. Depletion of NSD1 using a CRISPRi targeting approach promotes the loss of SETD2-mutant cells coincident with elevated levels of DNA damage and apoptosis. Surprisingly, only suppression of NSD1, but not related H3K36-methyltransferases, promotes synthetic lethality in these models. Mapping of genomic H3K36me2 targeting by NSD1 and NSD2 individually highlights the independent functions of these epigenetic writers. Furthermore, as a proof-of-principle, we demonstrate the therapeutic feasibility of targeting this synthetic lethal interaction by recapitulating the phenotype using BT5, a first-in-class pharmacologic inhibitor against NSD1. Conclusions These findings unify genome-wide screening approaches with the latest genetic and pharmacologic modeling methodologies to reveal an entirely novel epigenetic approach to individualize therapies against a challenging loss-of-function SETD2 mutation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03483-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73224953c4cbe478fbdd30504471b72d2f0b5d71" target='_blank'>
              MYC Serine 62 phosphorylation promotes its binding to DNA double strand breaks to facilitate repair and cell survival under genotoxic stress
              </a>
            </td>
          <td>
            Gabriel M. Cohn, Colin J. Daniel, Jenny Eng, Xiao-Xin Sun, C. Pelz, Koei Chin, Alexander Smith, Charles D. Lopez, Jonathan R. Brody, M. Dai, Rosalie C. Sears
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by the arrangement of one of these loci, found the consequent change in genome-wide DNA methylation patterns was associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of the both variants of each isolate were characterised using Illumina Hi-C, and Oxford Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was generally simple, with most pairwise interactions between loci <15 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with locally-elevated contact densities only evident around the origin of replication. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable in during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a phage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation is by the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477d2e85d15017f8eab23e5d5a172274fc4468ac" target='_blank'>
              Pneumococcal chromosome conformation variation between epigenetic variants is driven by episomal mobile genetic elements
              </a>
            </td>
          <td>
            Lim Tze Yee, Samuel T. Horsfield, Catherine M. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6be79817ee79c2803379821d2392c413312af98" target='_blank'>
              Personalized tumor-specific amplified DNA junctions in peripheral blood of patients with glioblastoma.
              </a>
            </td>
          <td>
            Mohamed F Ali, C. Riviere-Cazaux, Sarah H Johnson, Rebecca Salvatori, Alan R Penheiter, James B Smadbeck, Stephen J. Murphy, F. Harris, Lex F McCune, Lucas P Carlstrom, Michael T Barrett, F. Kosari, Leila A Jones, Cristiane M Ida, M. Borad, B. Bendok, Alfredo Quiñones-Hinojosa, Alyx B Porter, Maciej M Mrugala, K. Jaeckle, P. Anastasiadis, S. Kizilbash, John C Cheville, David M Routman, Terry C Burns, G. Vasmatzis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="One fundamental yet open question is how eukaryotic chromosomes fold into segregated territories, a process essential for gene transcription and cell fate. Through analyzing Hi-C and chromatin-tracing DNA-FISH data, we identify long-range chromo skeleton loop structures that span over 100 Mb, extending beyond the reach of loop extrusion models. Spatial density analyses point to assembly formation independent of major nuclear structures. A subset of genomic loci serves as nucleation centers, driving loop clustering. These complexes are highly stable, as shown by live-cell imaging with sequence-specific fluorescent labeling, and biophysical model analyses reveal a multivalent binding mechanism. Our findings suggest a redundant, distributed cluster mechanism that ensures robustness across cell types and against mutations, guiding both chromosome compaction and the formation of smaller-scale chromosomal structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04b5ac6a8c671d63e4c9bd85fcca4ea46155b48" target='_blank'>
              Long-range genomic loci stochastically assemble into combinatorial forms of chromosome skeleton
              </a>
            </td>
          <td>
            Jingyu Zhang, Siyuan Wang, Simon C Watkins, Jianhua Xing
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a deadly cancer with an aggressive nature, and how CRC tumor cells manage to translocate and proliferate in a new tissue environment remains not fully understood. Recently, higher-order chromatin structures and spatial genome organization are increasingly implicated in diseases including cancer, but in-depth studies of three-dimensional genome (3D genome) of metastatic cancer are currently lacking, preventing the understanding of the roles of genome organization during metastasis. Here we perform multi-omics profiling of matched normal colon, primary tumor, lymph node metastasis, liver metastasis and normal liver tissue from CRC patients using Hi-C, ATAC-seq and RNA-seq technologies. We find that widespread alteration of 3D chromatin structure is accompanied by dysregulation of genes including SPP1 during the tumorigenesis or metastasis of CRC. Remarkably, the hierarchy of topological associating domain (TAD) changes dynamically, which challenges the traditional view that the TAD structure between tumor and normal tissue is conservative. In addition, we define compartment stability score to measure large-scale alteration in metastatic tumors. To integrate multi-omics data and recognize candidate genes driving cancer metastasis, a pipeline is developed based on Hi-C, RNA-seq and ATAC-seq data. And three candidate genes ARL4C, FLNA, and RGCC are validated to be associated with CRC cell migration and invasion using in vitro knockout experiments. Overall, these data resources and results offer new insights into the involvement of 3D genome in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47bd8518f8db94e121fb18dd96d7456a23942c6d" target='_blank'>
              Polycomb group proteins protect latent Kaposi sarcoma-associated herpesvirus from episome clearance and HUSH-dependent chromatin silencing
              </a>
            </td>
          <td>
            Simon Weissmann, Alexis Robitaille, Thomas Günther, Marion Ziegler, A. Grundhoff
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract Regulator of telomere length 1 (RTEL1) helicase facilitates telomere replication by disassembling DNA secondary structures, such as G-quadruplexes and telomeric loops (t-loops), at the ends of the chromosomes. The recruitment of RTEL1 to telomeres occurs during the S-phase of the cell cycle, but the dynamics of the process has not been studied. Here, we utilized CRISPR genome editing and single-molecule imaging to monitor RTEL1 movement within human cell nuclei. RTEL1 utilizes rapid three-dimensional diffusion to search for telomeres and other nuclear targets. Only 5% of the chromatin-bound RTEL1 is associated with telomeres at any time in the S-phase, but the telomere-bound RTEL1 has much more extended associations. This binding is enhanced by the interaction between RTEL1 and the telomeric protein TRF2 but is largely independent of RTEL1 ATPase activity. The absence of RTEL1 catalytic activity leads to severe defects in cell proliferation, slow progression out of S-phase, and chromosome end-to-end fusion events. We propose that the rapid diffusion of RTEL1 allows this low-abundance protein to explore the nucleus, bind TRF2, and be recruited to telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d521287870dd985b41001fde2641677325213069" target='_blank'>
              Rapid dynamics allow the low-abundance RTEL1 helicase to promote telomere replication
              </a>
            </td>
          <td>
            Guanhui Wu, Erin Taylor, Daniel T. Youmans, Nausica Arnoult, Thomas R. Cech
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genomic heterogeneity has largely been overlooked in single-cell replication timing (scRT) studies. Here, we develop MnM, an efficient machine learning-based tool that allows disentangling scRT profiles from heterogenous samples. We use single-cell copy number data to accurately perform missing value imputation, identify cell replication states, and detect genomic heterogeneity. This allows us to separate somatic copy number alterations from copy number changes resulting from DNA replication. Our methodology brings critical insights into chromosomal aberrations and highlights the ubiquitous aneuploidy process during tumorigenesis. The copy number and scRT profiles obtained by analysing >119,000 high-quality human single cells from different cell lines, patient tumours and patient-derived xenograft samples leads to a multi-sample heterogeneity-resolved scRT atlas. This atlas is an important resource for cancer research and demonstrates that scRT profiles can be used to study replication timing heterogeneity in cancer. Our findings also highlight the importance of studying cancer tissue samples to comprehensively grasp the complexities of DNA replication because cell lines, although convenient, lack dynamic environmental factors. These results facilitate future research at the interface of genomic instability and replication stress during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Oncogenes drive cancer progression, but few are active exclusively in tumor cells. Connexins (Cxs), traditionally recognized as ion channel proteins, can localize to the nucleus and regulate gene expression, playing key roles in both physiological and pathological processes. Cx46, once thought to be restricted to the eye lens, has been implicated in tumor growth, though its underlying mechanisms remain unclear. This study investigates the nuclear presence of Cx46 in cancer cells and its potential role as a transcriptional modulator. Methods We employed ChIP-Seq, confocal immunofluorescence, and nuclear protein purification to assess Cx46 localization and DNA interactions. Functional assays were conducted to evaluate its effects on invasion, division, spheroid formation, and mesenchymal marker expression. Single-point mutations and molecular dynamics simulations were used to explore potential Cx46-DNA interactions. Results Cx46 mRNA upregulation was found in a variety of tumors compared to adjacent healthy tissue. In HeLa cells, which do not express Cx46, its transfection promoted proliferation, invasion and self-renewal capacity, cancer stem cell traits and mesenchymal features. Consistently, in Sk-Mel-2, which naturally express Cx46, reduced Cx46 expression led to a decrease in the similar parameters. In HeLa cells, nuclear Cx46 was detected in two forms, full length 46 kDa and a 30 kDa fragment (GJA3-30 k), ChIP-Seq experiments revealed that Cx46 binds to the DNA at intergenic and promoter regions, leading to the activation of oncogenic pathways. Molecular dynamics simulations suggest that GJA3-30 k dimerizes in a RAD50-like structure, forming stable DNA complexes. Cx46 and in some cases GJA3-30 k were detected in the nuclei of multiple cancer cell lines, including prostate, breast and skin cancers. Conclusions Our findings reveal a novel nuclear role for Cx46 in cancer, demonstrating its function as a transcriptional regulator and its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02151-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60952dbc2d22fbba5173287bd600c08d8141690" target='_blank'>
              Connexin46 in the nucleus of cancer cells: a possible role as transcription modulator
              </a>
            </td>
          <td>
            Ainoa Fernández-Olivares, Viviana P Orellana, Jesús Llanquinao, Gonzalo Nuñez, Pablo Pérez-Moreno, Sebastián Contreras-Riquelme, Alberto JM Martin, Fabio Mammano, Ivan E Alfaro, Juan F Calderón, Jimmy Stehberg, Mauricio A Sáez, Mauricio A Retamal
          </td>
          <td>2025-03-27</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc713b72314da7faa35c83a1d8de687d85095e55" target='_blank'>
              HRDetect in Ovarian Carcinoma: Stratification and Therapeutic Implications.
              </a>
            </td>
          <td>
            K. Banda, H. R. Davies, Y. Kumar, Y. Memari, A. Degasperi, M. Radke, I. Rodriguez, T. A. Gooley, F. Menghi, T. Harding, K. Lin, E. T. Liu, S. Nik-Zainal, E. M. Swisher
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Replicative stress (RS) is emerging as a promising therapeutic target in oncology, yet full exploitation of its potential requires a detailed understanding of the mechanisms and genes involved. Here, we investigated the RNA helicase Senataxin (SETX), an enzyme that resolves RNA-DNA hybrids and R-loops, to address its role in preventing RS by oncogenic Myc. Upon Myc activation, silencing of SETX led to selective engagement of the DNA damage response (DDR) and robust cytotoxicity. Pharmacological dissection of the upstream kinases regulating the DDR uncovered a protective role of the ATR pathway, that once inhibited, boosted SETX driven-DDR. While SETX loss did not lead to a genome-wide increase of R-loops, mechanistic analyses revealed enhanced R-loops localized at DDR-foci and newly replicated genomic loci, compatible with a selective role of SETX in resolving RNA-DNA hybrids to alleviate Myc-induced RS. Genome-wide mapping of DNA double-strand breaks confirmed that SETX silencing exacerbated DNA damage at transcription-replication conflict (TRC) regions at early replicated sites. We propose that SETX prevents Myc-induced TRCs by resolving transcription-associated R-loops that encounter the replisome. The identification of SETX as a genetic liability of oncogenic Myc opens up new therapeutic options against aggressive Myc-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lung cancer exhibits altered metabolism, influencing its response to radiation. To investigate the metabolic regulation of radiation response, we conducted a comprehensive, metabolic-wide CRISPR-Cas9 loss-of-function screen using radiation as selection pressure in human non–small cell lung cancer. Lipoylation emerged as a key metabolic target for radiosensitization, with lipoyltransferase 1 (LIPT1) identified as a top hit. LIPT1 covalently conjugates mitochondrial 2-ketoacid dehydrogenases with lipoic acid, facilitating enzymatic functions involved in the tricarboxylic acid cycle. Inhibiting lipoylation, either through genetic LIPT1 knockout or a lipoylation inhibitor (CPI-613), enhanced tumor control by radiation. Mechanistically, lipoylation inhibition increased 2-hydroxyglutarate, leading to H3K9 trimethylation, disrupting TIP60 recruitment and ataxia telangiectasia mutated (ATM)–mediated DNA damage repair signaling, impairing homologous recombination repair. In summary, our findings reveal a critical role of LIPT1 in regulating DNA damage and chromosome stability and may suggest a means to enhance therapeutic outcomes with DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c8836655ab7f9cea8b4528bffe4e53726a560" target='_blank'>
              Lipoylation inhibition enhances radiation control of lung cancer by suppressing homologous recombination DNA damage repair
              </a>
            </td>
          <td>
            Jui-Chung Chiang, Zeng Shang, Tracy Rosales, Ling Cai, Wei-Min Chen, Feng Cai, Hieu S Vu, John D Minna, Min Ni, Anthony J. Davis, Robert D Timmerman, Ralph J DeBerardinis, Yuanyuan Zhang
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) extracted from peripheral blood has emerged as a crucial biomarker source in oncology research. To enhance the detection of somatic copy number alterations (SCNAs) and circulating tumor DNA (ctDNA), we developed eSENSES, a 2 Mb breast cancer-targeted NGS panel. It includes 15,000 genome-wide SNPs, 500 focal SNPs in breast cancer driver regions, and exons from 81 commonly altered genes, alongside a custom computational approach. We assessed the performance of eSENSES using both synthetic and clinical samples showing that eSENSES can detect ctDNA levels below 1%, exhibiting high sensitivity and specificity at 2-3% ctDNA levels. In patients with metastatic breast cancer, ctDNA estimations correlated with disease progression. When compared with other technologies and state-of-the-art approaches, eSENSES demonstrated enhanced performance. eSENSES provides a reliable, powerful and cost-effective tool for monitoring disease progression and guiding therapeutic decisions in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d0020163f84c6d4b9e21837bb90e991331c10b" target='_blank'>
              Enabling sensitive and precise detection of ctDNA through somatic copy number aberrations in breast cancer
              </a>
            </td>
          <td>
            Riccardo Scandino, A. Nardone, Nicola Casiraghi, F. Galardi, Mattia Genovese, Dario Romagnoli, Marta Paoli, C. Biagioni, Andrea Tonina, I. Migliaccio, M. Pestrin, E. Moretti, L. Malorni, L. Biganzoli, M. Benelli, Alessandro Romanel
          </td>
          <td>2025-03-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1afe205564d682ee54296311b108e6d8a61e8b1" target='_blank'>
              Spatial mapping of DNA synthesis reveals dynamics and geometry of human replication nanostructures
              </a>
            </td>
          <td>
            Michael Hawgood, Bruno Urién González, Ana Agostinho, Praghadhesh Thiagarajan, Yiqiu Yang, Xue Zhang, Giovanni Giglio, Gemma Quijada, Matilde Fonseca, J. Bartek, Hans Blom, B. Lemmens
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="YTHDC1, a key protein in the m6A-related regulatory network within cells, is involved in multiple cellular processes such as chromatin-related regulation, RNA splicing, and nuclear export. Understanding its role in colorectal cancer (CRC) development and DNA damage repair is critical for the advancement of treatment strategies. Our study found that YTHDC1 was highly expressed in high-malignancy CRC tissues compared with low-malignancy ones. Upon silencing YTHDC1, we observed a pronounced suppression of the proliferation of CRC cell lines, accompanied by a substantial increase in cell apoptosis. Furthermore, we identified RAD51 as a crucial downstream target of YTHDC1. Knocking down YTHDC1 led to a notable decrease in RAD51 protein levels, and silencing RAD51 also inhibited cancer cell proliferation. Interestingly, RNA-sequencing data indicated that the YTHDC1 deletion did not affect RAD51 transcription. However, Western blot revealed that this deletion increased the ubiquitination of RAD51, likely due to the upregulated E3 ligase UBE3A. Ubiquitination experiments subsequently confirmed that RAD51 is indeed one of the substrates of UBE3A. In summary, our study provides novel insights into how YTHDC1 modulates the expression of RAD51 through post-translational modifications. These findings offer valuable information that may potentially contribute to the development of more effective therapeutic strategies for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92925-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cf4d3cb02cc31b95af37c1defc2a6fa9b119aa" target='_blank'>
              YTHDC1 negatively regulates UBE3A to influence RAD51 ubiquitination and inhibit apoptosis in colorectal cancer cells
              </a>
            </td>
          <td>
            Mingwei Gao, Yueguang Wu, Li Zhang, Yan Zhou, Huijuan Liu, Weimin Zhang, Shubin Wang, Yongping Cui
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c637d49373bdbc8ffeb399dfd912513e88f9743f" target='_blank'>
              A discrete region of the D4Z4 is sufficient to initiate epigenetic silencing
              </a>
            </td>
          <td>
            Ellen M. Paatela, Faith G. St. Amant, Danielle C Hamm, Sean R. Bennett, Taranjit S. Gujral, Silvère M. van der Maarel, Stephen J. Tapscott
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A string of nucleosomes, where genomic DNA is wrapped around histones, is organized in the cell as chromatin, ranging from euchromatin to heterochromatin, with distinct genome functions. Understanding physical differences between euchromatin and heterochromatin is crucial, yet specific labeling methods in living cells remain limited. Here, we have developed replication-dependent histone (Repli-Histo) labeling to mark nucleosomes in euchromatin and heterochromatin based on DNA replication timing. Using this approach, we investigated local nucleosome motion in the four known chromatin classes, from euchromatin to heterochromatin, of living human and mouse cells. The more euchromatic (earlier-replicated) and more heterochromatic (later-replicated) regions exhibit greater and lesser nucleosome motions, respectively. Notably, the motion profile in each chromatin class persists throughout interphase. Genome chromatin is essentially replicated from regions with greater nucleosome motions, although the replication timing is perturbed. Our findings, combined with computational modeling, suggest that earlier-replicated regions have more accessibility, and local chromatin motion can be a major determinant of genome-wide replication timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2b9973e20e3b9a1e91f7d1257dd38b93d96cad" target='_blank'>
              Replication-dependent histone labeling dissects the physical properties of euchromatin/heterochromatin in living human cells
              </a>
            </td>
          <td>
            Katsuhiko Minami, Kako Nakazato, Satoru Ide, Kazunari Kaizu, Koichi Higashi, Sachiko Tamura, Atsushi Toyoda, Koichi Takahashi, Ken Kurokawa, K. Maeshima
          </td>
          <td>2025-03-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The apicoplast organelle of Plasmodium falciparum is essential for parasite’s replication, however the details of its biogenesis, inheritance and regulation throughout the cell cycle are unknown. Here, we report the development of a dynamic imaging platform coupled with an analytical pipeline that enables us to follow and measure subcellular structures throughout the 48-hour cell cycle of live parasites. We reveal a predetermined sequence of four discrete morphological steps in organelle development, which are tightly correlated with nuclear replication. We show that one of these steps, which we term the Crown morphology, is required for nucleus-apicoplast attachment. During Crown, apicoplast is stretched over multiple nuclei, fastened by centriolar tubulin. A complementary molecular approach was used to discover the basic ploidy of apicoplast and mitochondrial genomes, their replication rates and association with nuclear DNA replication. We inhibited nuclear DNA replication and found that it completely blocks apicoplast biogenesis in its most initial stages, demonstrating dependency on S-phase initiation. Conversely, specific inhibition of apicoplast genome replication resulted in an almost-undisturbed organelle development and division. However, it affected the Crown step, preventing association to tubulin-nuclear structures, leading to failure in accurate organelle sorting into daughter cells. Collectively, these experiments reveal a central cellular pathway linking apicoplast development to the parasite’s cell cycle, and a second independent organellar mechanism responsible for segregation into daughter cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bfca923aa81dca1cabb89b41f451971dd56d89" target='_blank'>
              Organelle Development and Inheritance are Driven by Independent Nuclear and Organellar Mechanisms in Malaria Parasites
              </a>
            </td>
          <td>
            Shahar Michal, Qasem Alia, Freedman Eshkar, Maron Yariv, Tissawak Amanda, Florentin Anat
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, Pascale Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2 + tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One sentence summary: Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00341-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Recurrent/metastatic head and neck squamous cell carcinoma (R/M-HNSCC) is a severe, frequently lethal condition. Oncogene addiction to epidermal growth factor receptor (EGFR) is a hallmark of HNSCC, but the clinical efficacy of EGFR-targeted therapies remains low. Understanding molecular networks governing EGFR-driven progression is paramount to the exploration of (co)-treatment targets and predictive markers. Methods We performed function-based mapping of differentially expressed genes in EGFR-mediated local invasion (fDEGs) using photoconvertible tracers and RNA-sequencing (RNA-seq) in a cellular 3D-model. Results Upon alignment with public single-cell RNA-seq (scRNA-seq) datasets and HNSCC-specific regulons, a gene regulatory network of local invasion (invGRN) was inferred from gene expression data, which was overrepresented in budding tumors. InvGRN comprises the central hubs inhibin subunit beta alpha (INHBA) and snail family transcriptional repressor 2 (SNAI2), and druggable fDEGs integrin subunit beta 4 (ITGB4), laminin 5 (LAMB3/LAMC2), and sphingosine kinase 1 (SPHK1). Blockade of INHBA repressed local invasion and was reverted by activin A, laminin 5, and sphingosine-1-phosphate, demonstrating a functional interconnectivity of the invGRN. Epithelial-to-mesenchymal transition (EMT) of malignant cells and the invGRN are induced by newly defined EGFR-activity subtypes with prognostic value that are promoted by amphiregulin (AREG) and epiregulin (EREG). Importantly, co-inhibition of SPHK1 showed synthetic effects on Cetuximab-mediated invasion blockade and high expression of selected fDEGs was associated with response to Cetuximab in patient-derived xenotransplantation (PDX) and R/M-HNSCC patients. Conclusions We describe an actionable network of EGFR-mediated local invasion and define druggable effectors with predictive potential regarding the response of R/M-HNSCC to Cetuximab. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02290-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3a765a8bcbfd4bc6d82c05747a49dc01756024" target='_blank'>
              EGFR-mediated local invasiveness and response to Cetuximab in head and neck cancer
              </a>
            </td>
          <td>
            Jiefu Zhou, Min He, Qiong Zhao, Enxian Shi, Hairong Wang, Vaidehi Ponkshe, Jiahang Song, Zhengquan Wu, Dongmei Ji, Gisela Kranz, Anna Tscherne, S. Schwenk‑Zieger, Nilofer Abdul Razak, J. Hess, C. Belka, H. Zitzelsberger, I. Ourailidis, F. Stögbauer, M. Boxberg, J. Budczies, Christoph A. Reichel, M. Canis, Philipp Baumeister, Hongxia Wang, Kristian Unger, Andreas Mock, O. Gires
          </td>
          <td>2025-03-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic instability and inflammation are distinct hallmarks of aging, but the connection between them is poorly understood. Here we report a mechanism directly linking genomic instability and inflammation in senescent cells through a mitochondria-regulated molecular circuit involving p53 and cytoplasmic chromatin fragments (CCF) that are enriched for DNA damage signaling marker γH2A.X. We show that p53 suppresses CCF accumulation and its downstream inflammatory phenotype. p53 activation suppresses CCF formation linked to enhanced DNA repair and genome integrity. Activation of p53 in aged mice by pharmacological inhibition of MDM2 reverses transcriptomic signatures of aging and age-associated accumulation of monocytes and macrophages in liver. Mitochondrial ablation in senescent cells suppresses CCF formation and activates p53 in an ATM-dependent manner, suggesting that mitochondria-dependent formation of γH2A.X + CCF dampens nuclear DNA damage signaling and p53 activity. These data provide evidence for a mitochondria-regulated p53 signaling circuit in senescent cells that controls DNA repair, genome integrity, and senescence- and age-associated inflammation, with relevance to therapeutic targeting of age-associated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4941baca9720335671879d8e4ca5f2fe5568326" target='_blank'>
              p53 enhances DNA repair and suppresses cytoplasmic chromatin fragments and inflammation in senescent cells
              </a>
            </td>
          <td>
            Karl N. Miller, Brightany Li, Hannah R. Pierce-Hoffman, Shreeya Patel, Xue Lei, Adarsh Rajesh, M. G. Teneche, Aaron Havas, Armin Gandhi, Carolina Cano Macip, Jun Lyu, Stella G. Victorelli, Seung-Hwa Woo, A. Lagnado, Michael A LaPorta, Tianhui Liu, Nirmalya Dasgupta, Sha Li, Andrew Davis, Anatoly Korotkov, Erik Hultenius, Zichen Gao, Yoav Altman, Rebecca A Porritt, Guillermina Garcia, C. Mogler, A. Seluanov, Vera Gorbunova, S. Kaech, Xiao Tian, Zhixun Dou, Chongyi Chen, João F. Passos, P. Adams
          </td>
          <td>2025-03-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The ability to replicate and pass information to descendants is a fundamental requirement for life. In addition to the DNA-based genetic information, modifications of the DNA or DNA-associated proteins can create patterns of heritable gene regulation. Such epigenetic inheritance allows for adaptation without mutation, but its limits and regulation are incompletely understood. Here we developed a C. elegans system to study the transgenerational epigenetic inheritance of H3K27me3, a conserved histone posttranslational modification associated with gene repression. We find that induced alterations of the genome-wide H3K27me3 landscape and the associated fertility defects persist for many generations in genetically wildtype descendants under selective pressure. We uncover that the inheritance of the altered H3K27me3 landscape is regulated by two chromodomain proteins with antagonizing functions, and provide mechanistic insight into how this molecular memory is initiated and maintained. Our results demonstrate that epigenetic inheritance can act as a mutation-independent, heritable mechanism of adaptation. In Brief Özdemir et al. demonstrate that an altered genomic distribution of the histone modification H3K27me3 can be epigenetically inherited across many generations through the activity of HERI-1/SET-32/MES-4, which is antagonized by CEC-6/PRC2 in C. elegans. Highlights Altered H3K27me3 landscapes can be inherited for at least 15 generations in C. elegans. The chromodomain proteins CEC-6 and HERI-1 have opposite roles in antagonizing or promoting the maintenance of the altered H3K27me3 landscape. H3K23me3 and H3K36me3 replace H3K27me3 to promote the intergenerational and transgenerational inheritance of the altered epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9477951c95e1934dfca6ddb054dc1db83139c5" target='_blank'>
              Transgenerational maintenance of H3K27me3 heterochromatin is balanced by chromodomain proteins in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Isa Özdemir, Anna Höfler, Kamila Delaney, Joanna M. Wenda, Chengyin Li, Arneet L. Saltzman, Andreas Boland, Florian A. Steiner
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1ae2ca598b5cdc041c4c163b222a0b0f6d55027" target='_blank'>
              Tumor suppressor collateral damage screens reveal mRNA homeostasis protein HBS1L as a novel vulnerability in ch9p21 driven FOCAD deleted cancer
              </a>
            </td>
          <td>
            Hongxiang Zhang, M. Tonini, Lauren Martires, Charlotte B Pratt, Helena N. Jenkins, Eden N. Gordon, Shanchuan Zhao, Ashley H Choi, Samuel R. Meier, Tenzing Khendu, Shangtao Liu, B. Shen, Hannah Stowe, Katerina Pashiardis, Xuewen Pan, Madhavi Bandi, Minjie Zhang, Yi Yu, Chengyin Min, A. Huang, Jannik N Andersen, Hilary E. Nicholson, T. Teng
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, Titus K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1bc086a943a7e73ef25816e4af3f165ae6d1" target='_blank'>
              PARP inhibitors affect the replisome and replication fork progression during a single S phase
              </a>
            </td>
          <td>
            Lina-Marie Briu, C. Maric, Nicolas Valentin, Nicolas Panara, Guillaume Chevreux, Giuseppe Baldacci, Jean-Charles Cadoret
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Developing cancer therapies that induce robust death of the malignant cell is critical to prevent relapse. Highly effective strategies, such as immunotherapy, exemplify this observation. Here we provide the structural and molecular underpinnings for an approach that leverages chemical induced proximity to produce specific cell killing of diffuse large B cell lymphoma, the most common non-Hodgkin’s lymphoma. We develop KAT-TCIPs (lysine acetyltransferase transcriptional/epigenetic chemical inducers of proximity) that redirect p300 and CBP to activate programmed cell death genes normally repressed by the oncogenic driver, BCL6. Acute treatment rapidly reprograms the epigenome to initiate apoptosis and repress c-MYC. The crystal structure of the chemically induced p300-BCL6 complex reveals how chance interactions between the two proteins can be systematically exploited to produce the exquisite potency and selectivity of KAT-TCIPs. Thus, the malignant function of an oncogenic driver can be co-opted to activate robust cell death, with implications for precision epigenetic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d96f4bdeddc3af87404d0b766ff6c147fee90d7" target='_blank'>
              A Bivalent Molecular Glue Linking Lysine Acetyltransferases to Oncogene-induced Cell Death
              </a>
            </td>
          <td>
            Meredith N. Nix, S. Gourisankar, Roman C. Sarott, Brendan G. Dwyer, Sabin Nettles, Michael M. Martinez, Hind Abuzaid, Haopeng Yang, Yanlan Wang, Juste M. Simanauskaite, Bryan A. Romero, Hannah M. Jones, Andrey Krokhotin, Tara N. Lowensohn, Lei Chen, Cara Low, Mark M. Davis, Daniel Fernandez, Tinghu Zhang, M. Green, Stephen M. Hinshaw, N. Gray, G. R. Crabtree
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Because transposable elements (TEs) can cause heritable genetic changes, past work on TE mobility in Arabidopsis thaliana has mostly focused on new TE insertions in the germline of hypomethylated plants. It is, however, well-known that TEs can also be active in the soma, although the high-confidence detection of somatic events has been challenging. Here, we leveraged the high accuracy of PacBio HiFi long reads to evaluate the somatic mobility of TEs in individuals of an A. thaliana non-reference strain lacking activity of METHYLTRANSFERASE1 (MET1), a major component of the DNA methylation maintenance machinery. Most somatically mobile families coincided with those found in germline studies of hypomethylated genotypes, although the exact TE copies differed. We also discovered mobile elements that had been missed by standard TE annotation methods. Somatic TE activity was variable among individual plants, but also within TE families. Finally, our approach pointed to the possible involvement of alternative transposition as a cause for somatic hypermutability in a region that contains two closely spaced VANDAL21 elements. We conclude that long-read sequencing can reveal widespread TE transposition in the soma of A. thaliana hypomethylated mutants. Assessing somatic instead of germline mobilization is a fast and reliable method to investigate different aspects of TE mobility at the single plant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62607e49f83191e3ad9371253c4b7877fe10686a" target='_blank'>
              Long-read detection of transposable element mobilization in the soma of hypomethylated Arabidopsis thaliana individuals
              </a>
            </td>
          <td>
            Andrea Movilli, Svitlana Sushko, Fernando A. Rabanal, Detlef Weigel
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D. Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Cancer is complex because of the critical imbalance in genetic regulation as characterized by both the overexpression of oncogenes (OGs), mainly through mutations, amplifications, and translocations, and the inactivation of tumor-suppressor genes (TSGs), which entail the preservation of genomic integrity by inducing apoptosis to counter the malignant growth. Reviewing the intricate molecular interplay between OGs and TSGs draws attention to their cell cycle, apoptosis, and cancer metabolism regulation. In the present review, we discuss seminal discoveries, such as Knudson’s two-hit hypothesis, which framed the field’s understanding of cancer genetics, leading to the next breakthroughs with next-generation sequencing and epigenetic profiling, revealing novel insights into OG and TSG dysregulation with opportunities for targeted therapy. The key pathways, such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, are presented in the context of tumor progression. Importantly, we further highlighted the advances in therapeutic strategies, including inhibitors of KRAS and MYC and restoration of TSG function, despite which mechanisms of resistance and tumor heterogeneity pose daunting challenges. A high-level understanding of interactions between OG-TSGs forms the basis for effective, personalized cancer treatment—something to strive for in better clinical outcomes. This synthesis should integrate foundational biology with translation and, in this case, contribute to the ongoing effort against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Sarcomas are rare malignant tumors of mesenchymal origin with a high misdiagnosis rate due to their heterogeneity and low incidence. Conventional diagnostic techniques, such as Fluorescence In Situ Hybridization (FISH) and Next-Generation Sequencing (NGS), have limitations in detecting structural variations (SVs), copy number variations (CNVs), and predicting clinical behavior. Optical genome mapping (OGM) provides high-resolution genome-wide analysis, improving sarcoma diagnosis and prognosis assessment. This study analyzed 53 sarcoma samples using OGM. Ultra-high molecular weight (UHMW) DNA was extracted from core and resection biopsies, and data acquisition was performed with the Bionano Saphyr platform. Bioinformatic pipelines identified structural variations, comparing them with known alterations for each sarcoma subtype. OGM successfully analyzed 62.3% of samples. Diagnostic-defining alterations were found in 95.2% of cases, refining diagnoses and revealing novel oncogenic and tumor suppressor gene alterations. The challenges included DNA extraction and quality issues from some tissue samples. Despite these limitations, OGM proved to be a powerful diagnostic and predictive tool for bone and soft tissue sarcomas, surpassing conventional methods in resolution and scope, enhancing the understanding of sarcoma genetics, and enabling better patient stratification and personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7451322e1f10fdefd4135068d35cc6376d1125" target='_blank'>
              Exploring the Potential of Optical Genome Mapping in the Diagnosis and Prognosis of Soft Tissue and Bone Tumors
              </a>
            </td>
          <td>
            Alejandro Berenguer-Rubio, Esperanza Such, Neus Torres Hernández, P. González-Rojo, Álvaro Díaz-González, G. Avetisyan, Carolina Gil-Aparicio, Judith González-López, Nicolay Pantoja-Borja, L. Rubio-Martínez, Soraya Hernández-Girón, María Soledad Valera-Cuesta, Cristina Ramírez-Fuentes, María Simonet-Redondo, Roberto Díaz-Beveridge, C. de la Calva, J. Amaya-Valero, Cristina Ballester-Ibáñez, A. Liquori, Francisco Giner, E. Mayordomo-Aranda
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94444bc68fc048c7923ba3b90250650c889cfa29" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer that drives metastasis, immune evasion and treatment resistance. CIN may result from chromosome mis-segregation errors and excessive chromatin is frequently packaged in micronuclei (MN), which can be enumerated to quantify CIN. The assessment of CIN remains a predominantly manual and time-consuming task. Here, we present micronuclAI, a pipeline for automated and reliable quantification of MN of varying size and morphology in cells stained only for DNA. micronuclAI can achieve close to human-level performance on various human and murine cancer cell line datasets. The pipeline achieved a Pearson’s correlation of 0.9278 on images obtained at 10X magnification. We tested the approach in otherwise isogenic cell lines in which we genetically dialed up or down CIN rates, and on several publicly available image datasets where we achieved a Pearson’s correlation of 0.9620. Given the increasing interest in developing therapies for CIN-driven cancers, this method provides an important, scalable, and rapid approach to quantifying CIN on images that are routinely obtained for research purposes. We release a GUI-implementation for easy access and utilization of the pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad501d60562e0f7baf32d920b40de21a7a01759" target='_blank'>
              micronuclAI enables automated quantification of micronuclei for assessment of chromosomal instability
              </a>
            </td>
          <td>
            Miguel A. Ibarra-Arellano, Lindsay A. Caprio, Aroj Hada, Niklas Stotzem, Luke L Cai, Shivem B Shah, Zachary H. Walsh, Johannes C. Melms, Florian Wünneman, K. Bestak, Ibrahim Mansaray, Benjamin Izar, Denis Schapiro
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The Mitotic Surveillance Pathway (MSP) monitors the duration of M-phase. Prolonged mitosis, caused by spindle attachment defects or microtubule-targeting drugs such as the taxane paclitaxel, induces the formation of the ternary “mitotic stopwatch” complex consisting of 53BP1, USP28, and p53. This event protects p53 from degradation, resulting in cell cycle arrest or apoptosis in daughter cells. In paclitaxel-resistant cancers, cells bypass the MSP, enabling unchecked proliferation and survival, although the underlying mechanisms remain unknown. Here, we demonstrate that 53BP1 physically interacts with GMCL1 but not its paralog, GMCL2, and we mapped the interaction regions on both proteins. CRL3GMCL1 functions as a ubiquitin ligase that targets 53BP1 for degradation during M phase, impacting p53 levels in daughter cells. High GMCL1 expression significantly correlates with resistance to paclitaxel in cancer cell lines with wild-type p53, including endometrial, breast, and upper aerodigestive tract cancer cells. Loss of GMCL1 restores paclitaxel sensitivity in p53 expressing cells but not in p53 deficient cells. We propose that in cancers with high GMCL1 levels, the CRL3GMCL1-mediated degradation of 53BP1 prevents the formation of the mitotic stopwatch complex, leading to p53 degradation and sustained proliferation. Finally, our results indicate that GMCL1 inhibition represents a novel strategy to restore taxane sensitivity in resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/709ee165cbc090bfd04b08f1a64c12e9f13af2e9" target='_blank'>
              GMCL1 Controls 53BP1 Stability and Modulates Paclitaxel Sensitivity in Cancer
              </a>
            </td>
          <td>
            Yuki Kito, Tania J. González-Robles, Sharon Kaisari, Juhee Pae, Sheena Faye Garcia, Juliana Ortiz-Pacheco, Beatrix Ueberheide, Ruth Lehmann, Antonio Marzio, G. Róna, Michele Pagano
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, Lodewyk F. A. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Infections by the major opportunistic pathogen of human Candida albicans are commonly treated with echinocandin (ECN) drugs. However, C. albicans can adapt to grow in the presence of certain amounts of ECNs. Prior studies by several laboratories have defined multiple genes, as well as mechanisms involving induced aneuploidy, that can govern this. Still, the mechanisms of ECN adaptation are not fully understood. Here, we use genome-wide profiling of chromatin accessibility by ATAC-seq to determine if ECN adaptation is reflected in changes in the chromatin landscape in the absence of aneuploidy. We find that drug adaptation is coupled with multiple changes in chromatin accessibility genome-wide, which occur predominantly in gene promoter regions. Areas of increased accessibilities in promoters are enriched with the binding motifs for at least two types of transcription factors: zinc finger and basic leucine zipper. We also find that chromatin changes are often associated with differentially expressed genes including genes with functions relevant to the ECN-adapted phenotype, such as cell wall biosynthesis. Consistent with this, we find that the cell wall is remodeled in ECN-adapted mutants, with chitin up and glucan down and increased cell surface exposure. A full understanding of ECN adaptation processes is of critical importance for the prevention of clinical resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3773ea0cdd0e2f8c5db7467a8fc51861d1557f3" target='_blank'>
              Echinocandin Adaptation in Candida albicans Is Accompanied by Altered Chromatin Accessibility at Gene Promoters and by Cell Wall Remodeling
              </a>
            </td>
          <td>
            S. Sah, Anshuman Yadav, Tyler Stahl, Jeffrey J. Hayes, Michael Bulger, E. Rustchenko
          </td>
          <td>2025-02-01</td>
          <td>Journal of Fungi</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The burden of triple-negative breast cancer (TNBC) may be shaped by genetic factors, particularly inherited and somatic mutation profiles. However, data on this topic remain limited, especially for the African continent, where a higher TNBC incidence is observed. In the age of precision medicine, cataloguing TNBC diversity in African patients becomes imperative. We performed whole exome sequencing, including untranslated regions, on 30 samples from Angola and Cape Verde, which allowed to ascertain on potential regulatory mutations in TNBC for the first time. A high somatic burden was observed for the African cohort, with 86% of variants being so far unreported. Recurring to predictive functional algorithms, 17% of the somatic single nucleotide variants were predicted to be deleterious at the protein level, and 20% overlapped with candidate cis-regulatory elements controlling gene expression. Several of these somatic functionally-impactful mutations and copy number variation (mainly in 1q, 8q, 6 and 10p) occur in known BC- and all cancer-driver genes, enriched for several cancer mechanisms, including response to radiation and related DNA repair mechanisms. TP53 is the top of these known BC-driver genes, but our results identified possible novel TNBC driver genes that may play a main role in the African context, as TTN, CEACAM7, DEFB132, COPZ2 and GAS1. These findings emphasize the need to expand cancer omics screenings across the African continent, the region of the globe with highest genomic diversity, accelerating the discovery of new somatic mutations and cancer-related pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67fac74a3ac724fb478678a7a632cf9d7052cf76" target='_blank'>
              Coding and regulatory somatic profiling of triple-negative breast cancer in Sub-Saharan African patients
              </a>
            </td>
          <td>
            Ricardo J Pinto, Dylan Ferreira, Paulo Salamanca, Fernando Miguel, Pamela Borges, Carla Barbosa, Vitor Costa, Carlos Lopes, Lúcio Lara Santos, Luisa Pereira
          </td>
          <td>2025-03-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Genes that escape X-chromosome inactivation (XCI) in female somatic cells vary in number and levels of escape among mammalian species and tissues, potentially contributing to species- and tissue-specific sex differences. CTCF, a master chromatin conformation regulator, is enriched at escape regions and may play an important role in regulating escape, but the molecular mechanisms remain elusive. Results CTCF binding profiles and epigenetic features were systematically examined at escape genes (escapees) using mouse allelic systems with skewed XCI to distinguish the inactive X (Xi) and active X (Xa) chromosomes. We found that six constitutive and two facultative escapees are located inside 30-800 kb domains marked by convergent arrays of CTCF binding sites, consistent with the formation of chromatin loops. Facultative escapees show clear differences in CTCF binding depending on their XCI status in specific cell types/tissues. In addition, sets of strong and in some cases divergent CTCF binding sites located at the boundary between an escapee and its adjacent neighbors subject to XCI would also help insulate domains. Indeed, deletion but not inversion of a CTCF binding site at the boundary between the facultative escapee Car5b and its silent neighbor Siah1b results in a dramatic reduction of Car5b escape. This is associated with reduced CTCF and cohesin binding, which indicates loss of looping and insulation and is supported by 3C combined with Hi-C analysis. In addition, enrichment in the repressive mark H3K27me3 invades the Car5b domain in deleted cells, consistent with loss of expression from the Xi. In contrast, cells with an inversion of the CTCF binding site retain CTCF and cohesin binding, as well as looping, in line with persistence of escape. Interestingly, the levels of escape increase in cells with deletion of either Dxz4, which disrupts the Xi-specific compact 3D structure, or Firre, which results in lower H3K27me3 enrichment on the Xi, indicating that the structural and epigenetic features of the Xi constrain escape from XCI in wild type conditions. Conclusions Taken together, our findings support the idea that escape from XCI in female somatic cells is modulated by both the topological insulation of domains via CTCF binding and the surrounding heterochromatin environment. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02137-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fde8cc47c2c193748a3f06ce03baac9ac12fe7" target='_blank'>
              CTCF-mediated insulation and chromatin environment modulate Car5b escape from X inactivation
              </a>
            </td>
          <td>
            H. Fang, A. Tronco, G. Bonora, Truong Nguyen, Jitendra Thakur, J. B. Berletch, Galina N. Filippova, S. Henikoff, Jay Shendure, W. S. Noble, Zhijun Duan, Christine M. Disteche, Xinxian Deng
          </td>
          <td>2025-03-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Copy number variants (CNVs) are an important source of genetic variation underlying rapid adaptation and genome evolution. Whereas point mutation rates vary with genomic location and local DNA features, the role of genome architecture in the formation and evolutionary dynamics of CNVs is poorly understood. Previously, we found the GAP1 gene in Saccharomyces cerevisiae undergoes frequent amplification and selection in glutamine-limitation. The gene is flanked by two long terminal repeats (LTRs) and proximate to an origin of DNA replication (autonomously replicating sequence, ARS), which likely promote rapid GAP1 CNV formation. To test the role of these genomic elements on CNV-mediated adaptive evolution, we evolved engineered strains lacking either the adjacent LTRs, ARS, or all elements in glutamine-limited chemostats. Using a CNV reporter system and neural network simulation-based inference (nnSBI) we quantified the formation rate and fitness effect of CNVs for each strain. Removal of local DNA elements significantly impacts the fitness effect of GAP1 CNVs and the rate of adaptation. In 177 CNV lineages, across all four strains, between 26% and 80% of all GAP1 CNVs are mediated by Origin Dependent Inverted Repeat Amplification (ODIRA) which results from template switching between the leading and lagging strand during DNA synthesis. In the absence of the local ARS, distal ones mediate CNV formation via ODIRA. In the absence of local LTRs, homologous recombination can mediate gene amplification following de novo retrotransposon events. Our study reveals that template switching during DNA replication is a prevalent source of adaptive CNVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488ecf1ac91854034846373133d03867e6680f28" target='_blank'>
              Template switching during DNA replication is a prevalent source of adaptive gene amplification
              </a>
            </td>
          <td>
            Julie N. Chuong, Nadav Ben Nun, Ina Suresh, J. Matthews, Titir De, Grace Avecilla, Farah Abdul-Rahman, Nathan Brandt, Yoav Ram, David Gresham
          </td>
          <td>2025-02-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="The human APOBEC superfamily consists of eleven cytidine deaminase enzymes. Among them, APOBEC3 enzymes play a dual role in antiviral immunity and cancer development. APOBEC3 enzymes, including APOBEC3A (A3A) and APOBEC3B (A3B), induce mutations in viral DNA, effectively inhibiting viral replication but also promoting somatic mutations in the host genome, contributing to cancer development. A3A and A3B are linked to mutational signatures in over 50% of human cancers, with A3A being a potent mutagen. A3B, one of the first APOBEC3 enzymes linked to carcinogenesis, plays a significant role in HPV-associated cancers by driving somatic mutagenesis and tumor progression. The A3A_B deletion polymorphism results in a hybrid A3A_B gene, leading to increased A3A expression and enhanced mutagenic potential. Such polymorphism has been linked to an elevated risk of certain cancers, particularly in populations where it is more prevalent. This review explores the molecular mechanisms of APOBEC3 proteins, highlighting their dual roles in antiviral defense and tumorigenesis. We also discuss the clinical implications of genetic variants, such as the A3A_B polymorphism, mainly in HPV infection and associated cancers, providing a comprehensive understanding of their contributions to both viral restriction and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2370c8c875291e3fc4adec0ad421696e86fb060" target='_blank'>
              APOBEC3 Proteins: From Antiviral Immunity to Oncogenic Drivers in HPV-Positive Cancers
              </a>
            </td>
          <td>
            Eliza Pizarro Castilha, Rosalba Biondo, K. P. Trugilo, Giulia Mariane Fortunato, Tim R. Fenton, Karen Brajão de Oliveira
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, Luis A. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Higher-order chromatin structure and DNA methylation are critical for gene regulation, but how these vary across the human body remains unclear. We performed multi-omic profiling of 3D genome structure and DNA methylation for 86,689 single nuclei across 16 human tissues, identifying 35 major and 206 cell subtypes. We revealed extensive changes in CG and non-CG methylation across almost all cell types and characterized 3D chromatin structure at an unprecedented cellular resolution. Intriguingly, extensive discrepancies exist between cell types delineated by DNA methylation and genome structure, indicating that the role of distinct epigenomic features in maintaining cell identity may vary by lineage. This study expands our understanding of the diversity of DNA methylation and chromatin structure and offers an extensive reference for exploring gene regulation in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc53907293087756d961bef23e16d94d527ce994" target='_blank'>
              Human Body Single-Cell Atlas of 3D Genome Organization and DNA Methylation
              </a>
            </td>
          <td>
            Jingtian Zhou, Yue Wu, Hanqing Liu, Wei Tian, R. Castanon, Anna Bartlett, Zuolong Zhang, Guocong Yao, Dengxiaoyu Shi, Ben Clock, Samantha Marcotte, Joseph R. Nery, Michelle Liem, Naomi Claffey, Lara Boggeman, Cesar Barragan, R. A. e Drigo, Annika K. Weimer, Minyi Shi, Johnathan Cooper-Knock, Sai Zhang, Michael P. Snyder, S. Preissl, Bing Ren, Carolyn O’Connor, Shengbo Chen, C. Luo, Jesse R. Dixon, J. Ecker
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background The post-anal tail is a common physical feature of vertebrates including mammals. Although it exhibits rich phenotypic diversity, its development has been evolutionarily conserved as early as the embryonic period. Genes participating in embryonic tail morphogenesis have hitherto been widely explored on the basis of experimental discovery, whereas the associated cis-regulatory elements (CREs) have not yet been systematically investigated for vertebrate/mammalian tail development. Results Here, utilizing high-throughput sequencing schemes pioneered in mice, we profiled the dynamic transcriptome and CREs marked by active histone modifications during embryonic tail morphogenesis. Temporal and spatial disparity analyses revealed the genes specific to tail development and their putative CREs, which facilitated the identification of novel molecular expression features and potential regulatory influence of non-coding loci including long non-coding RNA (lncRNA) genes and CREs. Moreover, these identified sets of multi-omics data supply genetic clues for understanding the regulatory effects of relevant signaling pathways (such as Fgf, Wnt) dominating embryonic tail morphogenesis. Conclusions Our work brings new insights and provides exploitable fundamental datasets for the elucidation of the complex genetic mechanisms responsible for the formation of the vertebrate/mammalian tail. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02192-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4202fb3b2785be5b7f03e48a4ee994c37891080c" target='_blank'>
              A combination of transcriptomics and epigenomics identifies genes and regulatory elements involved in embryonic tail development in the mouse
              </a>
            </td>
          <td>
            Yong-Xuan Chen, Xiu-Ping Zhang, David N Cooper, Dong-Dong Wu, Wan-Dong Bao
          </td>
          <td>2025-03-26</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583a65cabb3a9c9f197a9eab6fd088b3c233c72e" target='_blank'>
              Naturally occurring variation in gene-associated transposable elements impacts gene expression and phenotypic diversity in woodland strawberry
              </a>
            </td>
          <td>
            Ileana Tossolini, Rocio Tolley, Julia Llinares-Gómez, Camila Zlauvinen, Tuomas Toivainen, T. Hytönen, David Posé, Carmen Martín-Pizarro, P. Manavella
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Monoallelic deletion of the short arm of chromosome 17 (del17p) is a recurrent abnormality in cancers with poor outcomes. Best studied in relation to haematological malignancies, associated functional outcomes are attributed mainly to loss and/or dysfunction of TP53, which is located at 17p13.1, but the wider impact of deletion of other genes located on 17p is poorly understood. 17p is one of the most gene-dense regions of the genome and includes tumour suppressor genes additional to TP53, genes essential for cell survival and proliferation, as well as small and long non-coding RNAs. In this review we utilise a data-driven approach to demarcate the extent of 17p deletion in multiple cancers and identify a common loss-of-function gene signature. We discuss how the resultant loss of heterozygosity (LOH) and haploinsufficiency may influence cell behaviour but also identify vulnerabilities that can potentially be exploited therapeutically. Finally, we highlight how emerging animal and isogenic cell line models of del17p can provide critical biological insights for cancer cell behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Background Chromosomal instability (CIN) is involved in about 70% of colorectal cancers (CRCs) and is associated with poor prognosis and drug resistance. From a clinical perspective, a better knowledge of these tumour’s biology will help to guide therapeutic strategies more effectively. Methods We used high-density chromosomal microarray analysis to evaluate CIN level of patient-derived organoids (PDOs) and their original mCRC tissues. We integrated the RNA-seq and mass spectrometry-based proteomics data from PDOs in a functional interaction network to identify the significantly dysregulated processes in CIN. This was followed by a proteome-wGII Pearson correlation analysis and an in silico validation of main findings using functional genomic databases and patient-tissues datasets to prioritize the high-confidence CIN features. Results By applying the weighted Genome Instability Index (wGII) to identify CIN, we classified PDOs and demonstrated a good correlation with tissues. Multi-omics analysis showed that our organoids recapitulated genomic, transcriptomic and proteomic CIN features of independent tissues cohorts. Thanks to proteotranscriptomics, we uncovered significant associations between mitochondrial metabolism and epithelial-mesenchymal transition in CIN CRC PDOs. Correlating PDOs wGII with protein abundance, we identified a subset of proteins significantly correlated with CIN. Co-localisation analysis in PDOs strengthened the putative role of IPO7 and YAP, and, through in silico analysis, we found that some of the targets give significant dependencies in cell lines with CIN compatible status. Conclusions We first demonstrated that PDO models are a faithful reflection of CIN tissues at the genetic and phenotypic level. Our new findings prioritize a subset of genes and molecular processes putatively required to cope with the burden on cellular fitness imposed by CIN and associated with disease aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03308-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The structural integrity of the chromosomes is essential to every functional process within eukaryotic nuclei. Chromosomes are DNA-histone complexes that are essential for the inheritance of genetic information to the offspring, and any defect in them is linked to mitotic errors, cancer growth, and cellular aging. Changes in the mechanical properties of a chromosome could lead to its compromised function and stability, leading to chromosome breaks. Here, we studied the changes in chromosome physical properties using metaphase chromosomes isolated from moderately malignant (MCF7) and highly malignant (MDA-MB-231) human breast cancer cells exposed to valproic acid (VPA), a known epigenetic modifier drug involved in histone hyperacetylation and DNA demethylation. Due to chromosomal structural intricacy and preparative and technical limitations of analytical tools, we employed a label-free atomic force microscopy approach for simultaneously visualizing and mapping single chromosome elasticity and stretching modulus. Additionally, we performed electron transport characteristics through metaphase chromosomes to elucidate the effect of VPA. The chromosomal elasticity and electron transport alterations are manifestations of VPA-mediated chromatin’s epigenetic changes. Our multiparametric strategy, as shown by receiver operating characteristics analyses with the physical properties of chromosomes, offers a new scope in terms of analytical tools for studying chromosomal structural changes/aberrations linked to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4081e3a40a851c1d9b5593b224de0890423bfe7" target='_blank'>
              Epigenetic Modifier Drug Valproic Acid Enhances Cancer Metaphase Chromosome Elasticity and Electron Transport: An Atomic Force Microscopy Approach
              </a>
            </td>
          <td>
            Tanya Agrawal, Debashish Paul, Amita Mishra, Ganesan Arunkumar, T. Rakshit
          </td>
          <td>2025-02-12</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung cancer is a major cause of cancer-related deaths globally. Targeted therapies, specifically attacking cancer cells based on genetic mutations, offer promising alternatives. ALK (anaplastic lymphoma kinase) fusions result in aberrant proteins that drive cancer growth. Drugs like crizotinib and ceritinib have shown efficacy in treating ALK-positive NSCLC. Accurate detection of ALK fusions is crucial for guiding these therapies. We conducted a retrospective analysis of a Chinese cohort of 131 ALK rearrangement-positive patients detected by DNA NGS between January 2017 and December 2021. Among those 131 ALK fusions, RNA-NGS confirmed positive transcripts in 88% of canonical ALK fusions and 75% of ALK fusions with rare partners in samples sequenced by both DNA NGS and RNA NGS. The secondary classification approach increased transcript prediction accuracy to 95.4% when combining common breakpoints and inframe fusion analysis in canonical ALK fusions. Combining rare breakpoints and inframe fusion could increase transcript prediction accuracy to 100%. For ALK fusions with rare partners, combining common breakpoints and frameshift improved transcript prediction accuracy to 100%. Additionally, combining rare breakpoints with inframe or frameshift could enhance the prediction accuracy to 100%. Combining DNA NGS and RNA NGS with a secondary classification approach significantly enhances the transcript prediction accuracy at the RNA level. This method optimizes clinical diagnostic and therapeutic strategies for ALK-positive NSCLC, highlighting the importance of advanced sequencing techniques in precision oncology. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92590-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34804a6772ac12af08ec23d6c1744145feb9ff1c" target='_blank'>
              Combined utility of genomic breakpoints and frame is a reliable predictor of ALK transcript function
              </a>
            </td>
          <td>
            Qiong Yang, Yumin Guo, Weihui Guo, Dongdong Liu, Haitao Wang, Xufan Cai
          </td>
          <td>2025-03-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Transposable elements (TEs), which constitute nearly half of the human genome, have long been regarded as genomic "dark matter". However, their reactivation in tumor cells, resulting in the production of TE-chimeric transcripts (TCTs), has emerged as a potential driver of cancer progression. The complexity and full extent of these transcripts remain elusive, largely due to the limitations of short-read next-generation sequencing technologies. These methods have struggled to comprehensively capture the diversity and structure of TCTs, particularly those involving short interspersed nuclear elements (SINEs) or closely co-transcribed TEs. Methods Leveraging full-length cDNA sequencing technology based on nanopore sequencing platform, we developed a customized pipeline for identifying and quantifying TCTs in 19 lung adenocarcinoma (LUAD) cell lines. The short-read RNA-seq dataset from a LUAD corhort (~ 200 tumor samples) was employed to validate the identified TCTs and explore their association with tumor progression. To assess the functional roles of a specific TCTs, cell migration and cell proliferation assays were performed. Results We uncovered 208 unique TCT candidates in the LUAD cell lines. Our approach allowed for the identification of cryptic promoters and terminators within non-transposing TEs. Notably, we identified a chimeric transcript involving MIR_HKDC1, which appears to play a significant role in the progression of LUAD. Furthermore, the expression of these TCTs were associated with poor clinical outcomes in a cohort of LUAD patients, suggesting their potential as novel biomarkers for both LUAD progression and prognosis. Conclusions Our study underscores the application of long-read sequencing to unravel the complex landscape of TCTs in LUAD. We provide a comprehensive characterization of TCTs in LUAD, exploring their potential regulatory roles in cancer progression. These findings contribute to a deeper understanding of the genomic intricacies underlying cancer, and offer new directions for the development of targeted therapies and personalized treatment strategies for LUAD. This research highlights the potential of TCTs as both biomarkers and therapeutic targets in the oncogenesis, offering new insights into the interplay between transposable elements and gene regulation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13888-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT Caviid gammaherpesvirus 1 (CaGHV-1), formerly known as the guinea pig herpes-like virus, is an oncogenic gammaherpesvirus with a sequenced genome but an as-yet uncharacterized transcriptome. Using nanopore long-read RNA sequencing, we annotated the CaGHV-1 genome and constructed a detailed transcriptomic atlas. Our findings reveal diverse viral mRNAs and non-coding RNAs, along with mapped promoter elements for each viral gene. We demonstrated that the CaGHV-1 RTA lytic cycle transcription factor activates its own promoter, similar to Kaposi’s sarcoma-associated herpesvirus (KSHV), and that the CaGHV-1 ORF50 promoter responds to RTA proteins from other gammaherpesviruses, highlighting the evolutionary conservation of RTA-mediated transcriptional mechanisms. Additionally, our analysis uncovered extensive transcriptional overlap within the viral genome, suggesting a role in regulating global gene expression. Given its tumorigenic properties, broad host range, and non-human pathogenicity, this work establishes CaGHV-1 as a promising small animal model for investigating human gammaherpesvirus pathogenesis. IMPORTANCE The molecular underpinnings of gammaherpesvirus pathogenesis remain poorly understood, partly due to limited animal models. This study provides the first comprehensive transcriptomic atlas of CaGHV-1, highlighting both coding and non-coding RNAs and revealing regulatory elements that drive viral gene expression. Functional studies of the CaGHV-1 RTA transcription factor demonstrated its ability to self-activate and cross-activate promoters from homologous gammaherpesviruses, reflecting conserved mechanisms of transcriptional control. These findings solidify CaGHV-1 as a unique and versatile small animal model, offering new opportunities to investigate gammaherpesvirus replication, transcriptional regulation, and tumorigenesis in a controlled experimental system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751314908691fbdc649b545362d1d54bdf0de7b5" target='_blank'>
              Long-read transcriptomics of caviid gammaherpesvirus 1: compiling a comprehensive RNA atlas
              </a>
            </td>
          <td>
            Gábor Torma, Ákos Dörmő, Á. Fülöp, Dóra Tombácz, M. Mizik, Amanda M. Pretory, Seeyoun Lee, Z. Tóth, Z. Boldogkői
          </td>
          <td>2025-02-27</td>
          <td>mSystems</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Genomes in eukaryotes normally undergo DNA replication in a choreographed temporal order, resulting in early and late replicating chromosome compartments. Leishmania, a human protozoan parasite, displays an unconventional DNA replication program in which the timing of DNA replication completion is chromosome size-dependent: larger chromosomes complete replication later then smaller ones. Here we show that both R-loops and RNase H1, a ribonuclease that resolves RNA-DNA hybrids, accumulate in Leishmania major chromosomes in a pattern that reflects their replication timing. Furthermore, we demonstrate that such differential organisation of R-loops, RNase H1 and DNA replication timing across the parasite’s chromosomes correlates with size-dependent differences in chromatin accessibility, G quadruplex distribution and sequence content. Using conditional gene excision, we show that loss of RNase H1 leads to transient growth perturbation and permanently abrogates the differences in DNA replication timing across chromosomes, as well as altering levels of aneuploidy and increasing chromosome instability in a size-dependent manner. This work provides a link between R-loop homeostasis and DNA replication timing in a eukaryotic parasite and demonstrates that orchestration of DNA replication dictates levels of genome plasticity in Leishmania.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3389d87da3fd17d24b405d1383fde338ac5299e" target='_blank'>
              MYC in cancer: from undruggable target to clinical trials.
              </a>
            </td>
          <td>
            Jonathan R Whitfield, Laura Soucek
          </td>
          <td>2025-02-19</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4692fdd18b494d4b749d2c1b7ef3cd9972515b" target='_blank'>
              Cholangiocytes’ Primary Cilia Regulate DNA Damage Response and Repair
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="All organisms depend on specific proteins to compact and organize their genomes. In eukaryotes, histones fulfil this role, while bacterial chromosomes are shaped by nucleoid-associated proteins (NAPs). Among its pleiotropic functions, the NAP Hfq plays a pivotal role in bacterial genome organization. In this study, we characterized the structure of the C-terminal extension of Hfq, which mediates chromosomal compaction, in its DNA-bound state. Using an integrative approach that combined transmission electron microscopy, neutron scattering, site-directed mutagenesis, and molecular modeling, we identified an amyloid module formed by the C-terminal region of Hfq. This module uniquely bridges and compacts six DNA molecules, marking the first documented instance of an amyloid structure with DNA-bridging properties. Our findings redefine the functional landscape of amyloids, linking them to genome architecture and gene regulation. This result suggests that amyloid-DNA interactions may represent a conserved mechanism across biological systems, with profound implications for understanding genome organization and the regulation of gene expression in both prokaryotes and eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c1a71c24ec17a7b470740db10c59903a0c75c1" target='_blank'>
              Amyloid-like DNA bridging: a new mode of DNA shaping.
              </a>
            </td>
          <td>
            Frank Wien, Marcos Gragera, Tatsuhito Matsuo, Gautier Moroy, M. Bueno-Carrasco, Rocío Arranz, A. Cossa, Anne Martel, Heloisa N Bordallo, S. Rudić, Marisela Vélez, J. V. D. van der Maarel, Judith Peters, V. Arluison
          </td>
          <td>2025-02-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1324a05392de89320b697d282b9ec5e482ee4e56" target='_blank'>
              γ-tubulin mediates DNA double-strand break repair.
              </a>
            </td>
          <td>
            Abhishikt David Solomon, Odjo G. Gouttia, Ling Wang, Songli Zhu, Feifei Wang, Yanqui Li, Mohammadjavad Paydar, T. Bessho, Benjamin H Kwok, A. Peng
          </td>
          <td>2025-03-15</td>
          <td>Journal of cell science</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7556721a82c0422b8d26aaeaf4a37ba842556557" target='_blank'>
              Abstract B006: Methylation variability and LINE-1 activation in plasmablastic multiple myleoma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Song, Dustin Schones
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The tumor suppressor p53 (Trp53), also known as p53, is the most commonly mutated gene in cancer. Canonical p53 DNA damage response pathways are well characterized and classically thought to underlie the tumor suppressive effect of p53. Challenging this dogma, mouse models have revealed that p53-driven apoptosis and cell cycle arrest are dispensable for tumor suppression. Here, we investigated the inverse context of a p53 mutation predicted to drive the expression of canonical targets but is detected in human cancer. Methods We established a novel mouse model with a single base pair mutation (GAG>GAT, p53E221D) in the DNA-Binding domain that has wild-type function in screening assays, but is paradoxically found in human cancer in Li-Fraumeni syndrome. Using mouse p53E221D and the analogous human p53E224D mutants, we evaluated expression, transcriptional activation, and tumor suppression in vitro and in vivo. Results Expression of human p53E224D from cDNA translated to a fully functional p53 protein. However, p53E221D/E221D RNA transcribed from the endogenous locus is mis-spliced resulting in nonsense-mediated decay. Moreover, fibroblasts derived from p53E221D/E221D mice do not express a detectable protein product. Mice homozygous for p53E221D exhibited increased tumor penetrance and decreased life expectancy compared to p53WT/WT animals. Conclusions Mouse p53E221D and human p53E224D mutations lead to splice variation and a biologically relevant p53 loss of function in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a5ab888592bac1b40fc9b15ac14955958bd526" target='_blank'>
              Mis-splicing drives loss of function of p53E224D point mutation
              </a>
            </td>
          <td>
            Ian C. Lock, Nathan H Leisenring, Warren Floyd, Eric S. Xu, Lixia Luo, Yan Ma, Erin C. Mansell, Diana M. Cardona, Chang-Lung Lee, D. Kirsch
          </td>
          <td>2025-03-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2b99c29abd6764313dfc087425813f749b307a" target='_blank'>
              Real-World Data Analysis of Genomic Alterations Detected by a Dual DNA-RNA Comprehensive Genomic Profiling Test.
              </a>
            </td>
          <td>
            Kousuke Watanabe, Miho Ogawa, A. Shinozaki-Ushiku, Shuichi Tsutsumi, K. Tatsuno, Hiroyuki Aburatani, H. Kage, K. Oda
          </td>
          <td>2025-03-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbddfb38eb3b46fe2a2daba20fe0e51c20e3f4a" target='_blank'>
              βTrCP is involved in the localization of the MRN complex on chromatin to enhance DNA damage repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The retinoic acid-related orphan receptor gamma (RORγt) acts as the major transcriptional activator in Th17 cell development and function to mediate adaptive immune defenses against pathogenic infection. RORγt engages accessible DNA response elements in the genome and interplays with coactivator proteins and accessory transcription factors to drive gene expression. However, how the chromatin environment mediates RORγt structure, dynamics, and function remains unclear. Here, we profile how the nucleosome promotes or restricts access to the main RORγt DNA response elements found in native enhancers and promoters, revealing preferential binding in regions of free DNA and nucleosomal entry/exit sites, with single base-pair resolution. Solution phase measurements using hydrogen deuterium exchange coupled to mass spectrometry identify novel allosteric effects that influence RORγt binding and mediate chromatin dynamics. A high-resolution structure of RORγt bound to the nucleosome reveals how structured elements assemble to confer binding specificity and avidity to chromatin substrates. The observations suggest an activation model where RORγt binding to chromatinized DNA promotes coregulator recruitment and chromatin decompaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5624b8ad84bcb73e88ab4dda70b0e6af9169b6fa" target='_blank'>
              Nucleosome Engagement Regulates RORγt Structure and Dynamics
              </a>
            </td>
          <td>
            Timothy S. Strutzenberg, Matthew D. Mann, Xiandu Li, Hyejeong Shin, Jordan Kelsey, S. Aiyer, Jingting Yu, Gennavieve Gray, Zeyuan Zhang, Zelin Shan, Bo Zhou, Ye Zheng, Patrick R. Griffin, Dmitry Lyumkis
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a highly heterogeneous malignancy, often characterized by complex genomic alterations that drive tumor progression and therapy resistance. In this paper, we report a novel de novo BRCA2 germline variant NM_000059.3:c.(8693_8695delinsGT) associated with early-onset OC that featured two regions with differential MMR (Mismatch Repair) gene expression. To date, only six cases of de novo BRCA2 variants have been reported, none of which were associated with early-onset high-grade serous OC. The immunohistochemical analysis of MMR genes revealed two distinct tumor areas, separated by a clear topographic boundary, with the heterogeneous expression of MLH1 and PMS2 proteins. Seventy-five percent of the tumor tissue showed positivity, while the remaining 25% exhibited a complete absence of expression, underscoring the spatial variability in MMR gene expression within the tumor. Integrated comparative spatial genomic profiling identified several tumor features associated with the genetic variant as regions of loss of heterozygosity (LOH) that involved BRCA2 and MLH1 genes, along with a significantly higher mutational tumor burden in the tumor area that lacked MLH1 and PMS2 expression, indicating its further molecular evolution. The following variants were acquired: c.6572C>T in NOTCH2, c.1852C>T in BCL6, c.191A>T in INHBA, c.749C>T in CUX1, c.898C>A in FANCG, and c.1712G>C in KDM6A. Integrated comparative spatial proteomic profiles revealed defects in the DNA repair pathways, as well as significant alterations in the extracellular matrix (ECM). The differential expression of proteins involved in DNA repair, particularly those associated with MMR and Base Excision Repair (BER), highlights the critical role of defective repair mechanisms in driving genomic instability. Furthermore, ECM components, such as collagen isoforms, Fibrillin-1, EMILIN-1, Prolargin, and Lumican, were found to be highly expressed in the MLH1/PMS2-deficient tumor area, suggesting a connection between DNA repair deficiencies, ECM remodeling, and tumor progression. Thus, the identification of the BRCA2 variant sheds light on the poorly understood interplay between DNA repair deficiencies and ECM remodeling in OC, providing new insights into their dual role in shaping tumor evolution and suggesting potential targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3d22f654834c47296af7f38cdf9042ad40753" target='_blank'>
              Novel De Novo BRCA2 Variant in an Early-Onset Ovarian Cancer Reveals a Unique Tumor Evolution Pathway
              </a>
            </td>
          <td>
            G. Miolo, Giovanni Canil, Maurizio Polano, M. Dal Bo, Alessia Mondello, Antonio Palumbo, F. Puglisi, Giuseppe Corona
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells display high levels of oncogene-induced replication stress (RS) and rely on DNA damage checkpoint for viability. This feature is exploited by cancer therapies to either increase RS to unbearable levels or inhibit checkpoint kinases involved in the DNA damage response. Thus far, treatments that combine these two strategies have shown promise but also have severe adverse effects. To identify novel, better-tolerated anticancer combinations, we screened a collection of plant extracts and found two natural compounds from the plant, Psoralea corylifolia, that synergistically inhibit cancer cell proliferation. Bakuchiol inhibited DNA replication and activated the checkpoint kinase CHK1 by targeting DNA polymerases. Isobavachalcone interfered with DNA double-strand break repair by inhibiting the checkpoint kinase CHK2 and DNA end resection. The combination of bakuchiol and isobavachalcone synergistically inhibited cancer cell proliferation in vitro. Importantly, it also prevented tumor development in xenografted NOD/SCID mice. The synergistic effect of inhibiting DNA replication and CHK2 signaling identifies a vulnerability of cancer cells that might be exploited by using clinically approved inhibitors in novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/042cde9aa3dc3d91bb528dfff54349708df45e8a" target='_blank'>
              Synergistic effect of inhibiting CHK2 and DNA replication on cancer cell growth
              </a>
            </td>
          <td>
            Flavie Coquel, Sing-Zong Ho, K. Tsai, Chun-Yen Yang, Antoine Aze, Julie Devin, Ting-Hsiang Chang, Marie Kong-Hap, Audrey Bioteau, J. Moreaux, Domenico Maiorano, Philippe Pourquier, Wen-Chin Yang, Yea-Lih Lin, Philippe Pasero
          </td>
          <td>2025-01-31</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract G-quadruplex (G4) DNAs are alternative nucleic acid structures, proposed to play important roles in regulating DNA replication, gene transcription, and translation. Several specialized DNA helicases are involved in cellular G4 metabolism, in some cases with redundant functions. Among them, human FANCJ/BRIP1, which has orthologs in all metazoans, is one of the most powerful G4 resolvases, believed to act mainly at DNA replication forks. Here, we tested the effects of a set of hydrazone-derivative G4 ligands in a FANCJ-knocked-out HeLa cell line and in a Caenorhabditis elegans strain, where DOG-1, a FANCJ ortholog, was disrupted, as a whole organism model system. Our results revealed that loss of FANCJ specifically sensitized cancer cells to FIM-15, a mono-guanylhydrazone derivative bearing the diimidazopyrimidine core, among the tested hydrazone-based compounds and induced enhanced DNA damage in different chromosomal sites including telomeric ends. Moreover, dietary administration of FIM-15 to dog-1−/− nematodes stabilized G4 structures in gonadal cell nuclei and resulted in compromised embryonic development in the first-generation post-treatment. Collectively, our findings unveil a specific vulnerability of FANCJ-knocked-out cancer cells (and DOG-1-lacking worms) to G4 stabilization by the FIM-15 compound. This study provides an important proof-of-principle for use of G4 ligands in synthetic lethality-based therapeutic approaches targeting FANCJ-defective cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a485077681fc1cddbc3b2edd2dc8610a684c82b" target='_blank'>
              Effects of hydrazone-based G-quadruplex ligands on FANCJ/BRIP1-depleted cancer cells and a Caenorhabditis elegans dog-1−/− strain
              </a>
            </td>
          <td>
            Marcello Germoglio, Federica D'Aria, G. Cortone, Antonello Prodomo, M. Mahtab, Rita Morigi, Jussara Amato, Francesca M. Pisani, C. Giancola
          </td>
          <td>2025-02-08</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8904feaf495af004b999f1283404430b8205d7d8" target='_blank'>
              p53-deficient cancer cells hyperactivate DNA double-strand break repair pathways to overcome chemotherapeutic damage and augment survival.
              </a>
            </td>
          <td>
            Rebecca Dsouza, Meghna Jain, Ekta Khattar
          </td>
          <td>2025-03-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background AZIN1 is a cell cycle regulator that is upregulated in a variety of cancers. AZIN1 overexpression can induce a more aggressive tumor phenotype via increased binding and resultant inhibition of antizyme. Antizyme is a protein that normally functions as an anti-tumor regulator that facilitates the deactivation of several growth-promoting proteins including c-Myc. MYC plays a critical role in medulloblastoma pathogenesis. Its amplification serves as a defining characteristic of group 3 medulloblastomas, associated with the most aggressive clinical course, greater frequency of metastases, and shorter survival times. Methods Medulloblastoma tissues (68 TMA, and 45 fresh tissues, and 31 controls) were stained (fluorescence and immunohistochemical) for AZIN1. Western blotting and ELISA were used to detect the AZIN1 level. Phenotypically aggressive cellular features were measured by increased invasion, colony formation and proliferation. CRISPR-Cas9-mediated AZIN1 knocked-out cells were orthotopically implanted in the cerebellum of nude mice (n = 8/group) with a stereotactic frame. Tumor growth was monitored using the In Vivo Imaging System (IVIS). Results Here, we investigated the role of AZIN1 expression in medulloblastoma. We found that overexpression of AZIN1 in medulloblastoma cells induces phenotypically aggressive features. Conducting in vivo studies we found that knocking-out AZIN1 in tumors corresponds with reduced tumor progression and prolonged survival. Clinical specimens are revealing that AZIN1 is highly expressed and directly correlates with MYC amplification status in patients. Conclusion These data implicate AZIN1 as a putative regulator of medulloblastoma pathogenesis and suggest that it may have clinical application as both a biomarker and novel therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03274-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff1e1e85866d4cc9a2e1386acb32f48ee6d86bb6" target='_blank'>
              AZIN1 level is increased in medulloblastoma and correlates with c-Myc activity and tumor phenotype
              </a>
            </td>
          <td>
            Julie Sesen, Tyra Martinez, Sara Busatto, L. Poluben, Hassan Nassour, Caroline Stone, Karthik Ashok, Marsha A. Moses, Edward R. Smith, Aram Ghalali
          </td>
          <td>2025-02-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DNA-dependent protein kinase catalytic subunit (DNA-PKcs) is a key kinase in the DNA repair process that responds to DNA damage caused by various factors and maintains genomic stability. However, DNA-PKcs is overexpressed in some solid tumors and is frequently associated with poor prognosis. DNA-PKcs was initially identified as a part of the transcription complex. In recent years, many studies have focused on its nonclassical functions, including transcriptional regulation, metabolism, innate immunity, and inflammatory response. Given the pleiotropic roles of DNA-PKcs in tumors, pharmacological inhibition of DNA-PK can exert antitumor effects and may serve as a potential target for tumor therapy in the future. This review summarizes several aspects of DNA-PKcs regulation of RNA metabolism, including its impact on transcriptional machinery, alternative splicing, and interaction with noncoding RNAs, and provides insights into DNA-PKcs beyond its DNA damage repair function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f296dc7b10466d316234339cf9617d0c3278a601" target='_blank'>
              DNA-PKcs, a player winding and dancing with RNA metabolism and diseases
              </a>
            </td>
          <td>
            Jiabao Hou, Mingjun Lu, Jingwei Guo, Jinghong Wu, Chenyang Wang, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2025-03-04</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0df68e8815856c88d69d3739a1de93ae04e1b66" target='_blank'>
              Abstract A009: Direct multi-omics for the masses: Linking DNA methylation to chromatin targets via CUT&RUN-EM
              </a>
            </td>
          <td>
            V. S. Kumary, Allison R. Hickman, Bryan J. Venters, Anup Vaidya, James Anderson, J. Burg, Louise Williams, Chaithanya Ponnaluri, P. Estève, Issac Meek, Zu-Wen Sun, M. Cowles, S. Pradhan, Michael-Christopher Keogh
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The sequence-driven organising principles of the 3D genome are crucial for interpreting the core effects of genomic variation and for understanding the evolution of genome organisation and function. We investigated these by isolating and analysing cell-type-persistent contacts, heavily dependent on the similarly cell-type-persistent genomic sequence. We stratified long-range contacts from a diverse group of human tissues and cell lines based on contact persistence, cp, reflecting their presence across cell or tissue types, presenting them as an atlas of contacts and the cell-type invariant (CETI) hubs they form across human chromosomes. Our survey of more than 300 chromatin and genome features revealed their association with cp, contrasting variable from persistent contacts in terms of co-localisation with genes, 3D architectural domains, epigenetic and sequence elements. We found persistent contacts to be predominantly comprised of AT-rich sequences and related to heterochromatin. A key outcome is finding a link between the experimental genomic contacts and the complementarity between pairs of contacting DNA loci. This work provides evidence for a sequence determinant of genomic contacts contributing to the decoding of the relationship between sequence and structure that is crucial for functional and evolutionary studies concerning the 3D genome organisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9e662847ce0652a3c57525411bd551072e0b44" target='_blank'>
              Analysis of long-range contacts across cell types outlines a core sequence determinant of 3D genome organisation
              </a>
            </td>
          <td>
            Liezel Tamon, Zahra Fahmi, James Ashford, Rosana Collepardo-Guevara, Aleksandr B. Sahakyan
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d000a35acf7de71fd2986badfaa9c30abc1a2a" target='_blank'>
              Integrative Computational Analysis of Common EXO5 Haplotypes: Impact on Protein Dynamics, Genome Stability, and Cancer Progression.
              </a>
            </td>
          <td>
            Fabio Mazza, Davide Dalfovo, Alessio Bartocci, Gianluca Lattanzi, Alessandro Romanel
          </td>
          <td>2025-03-21</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Garra rufa, commonly known as doctor fish, is a small freshwater cyprinid fish recognized for its high-temperature tolerance. G. rufa is primarily known for its role in ichthyotherapy, in which it helps remove keratinized skin from human hands and feet. However, in recent years, its high-temperature tolerance has attracted attention owing to its potential use as a model for human disease research, including infectious diseases and cancer xenograft models. Despite this interest, the genomic basis underlying high-temperature tolerance remains largely unexplored, primarily because of the limited availability of genomic resources, which hinders its development as an experimental model. In the present study, a high-quality chromosome-level genome assembly of G. rufa was generated using a combination of PacBio HiFi long-read sequencing and Hi-C technology. We generated a chromosome-level genome assembly with 25 chromosomes, 1.38 Gb in total length, and scaffold N50 of 49.3 Mb. Approximately 59% of the genome comprises repetitive elements, with DNA and LTR elements being particularly abundant. In total, 27,352 protein-coding genes were annotated, of which 26,900 genes (98.3%) were functionally annotated. Benchmarking Universal Single-Copy Orthologs (BUSCO) benchmark for genome assembly and gene annotation demonstrated 94.5% and 94.7% of complete BUSCOs, respectively. We identified two heat shock transcription factor (HSF) and 239 heat shock protein (HSP)-related genes. Heat shock elements, which are HSF-binding motifs, were present within 3 kb upstream of 944 genes, with statistically significant enrichment of HSP-related genes in this set. Furthermore, molecular phylogenetic analysis and whole-genome comparisons revealed that G. rufa is evolutionarily closely related to species in the Labeoninae subfamily of the Cyprinidae family. This chromosome-level reference genome provides a valuable resource for future research aimed at elucidating the molecular mechanisms underlying high-temperature tolerance in G. rufa and establishing it as a model organism for human biomedical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba084e3b18995a64de9db1867a75cd406732e5f" target='_blank'>
              Chromosome-level genome assembly of the doctor fish (Garra rufa)
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Kiki Syaputri Handayani, Liqing Zang, Fahrurrozi, Oleg Simakov, Victor David, Nico Gultom, Yasuhito Shimada
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24649fa8cee2fc610da3934361ee2a3da1b227a9" target='_blank'>
              Ku suppresses RNA-mediated innate immune responses in human cells to accommodate primate-specific Alu expansion
              </a>
            </td>
          <td>
            Yimeng Zhu, Angelina Li, Suvrajit Maji, Brian J. Lee, Sophie M. Korn, Jake A. Gertie, Tyler J. Dorrity, Jianhua Wang, Kyle J. Wang, Amandine Pelletier, Daniel F. Moakley, Rachel D. Kelly, Antony B. Holmes, Raul Rabadan, D. Edgell, Caroline Schild Poulter, Mauro Modesti, Anna-Lena Steckelberg, Eric A. Hendrickson, Hachung Chung, Chaolin Zhang, Shan Zha
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2140165b325769cf3794c12431eadebdd635d22" target='_blank'>
              Chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Nina Trautwein, Bodo Laube, Christian Dietz, Robert W. Stark, M. C. Cardoso
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell division drives somatic evolution but is challenging to quantify. We developed a framework to count cell divisions with DNA replication-related mutations in polyguanine homopolymers. Analyzing 505 samples from 37 patients, we studied the milestones of colorectal cancer evolution. Primary tumors diversify at ~250 divisions from the founder cell, while distant metastasis divergence occurs significantly later, at ~500 divisions. Notably, distant but not lymph node metastases originate from primary tumor regions that have undergone surplus divisions, tying subclonal expansion to metastatic capacity. Then, we analyzed a cohort of 73 multifocal lung cancers and showed that the cell division burden of the tumors' common ancestor distinguishes independent primary tumors from intrapulmonary metastases and correlates with patient survival. In lung cancer too, metastatic capacity is tied to more extensive proliferation. The cell division history of human cancers is easily accessible using our simple framework and contains valuable biological and clinical information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23fd0fe606d16164cbf75d226bfad5eec1b690f" target='_blank'>
              Complex determinants of R-loop formation at transposable elements and major DNA satellites.
              </a>
            </td>
          <td>
            Timothy J. Stanek, Adam Kneebone, Matthew A Lawlor, Weihuan Cao, Christopher E Ellison
          </td>
          <td>2025-02-28</td>
          <td>Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer-associated mutations in IDH are associated with multiple types of human malignancies, which exhibit distinctive metabolic reprogramming, production of oncometabolite D-2-HG, and shifted epigenetic landscape. IDH mutated malignancies are signatured with “BRCAness”, highlighted with the sensitivity to DNA repair inhibitors and genotoxic agents, although the underlying molecular mechanism remains elusive. In the present study, we demonstrate that D-2-HG impacts the chromatin conformation adjustments, which are associated with DNA repair process. Mechanistically, D-2-HG diminishes the chromatin interactions in the DNA damage regions via revoking CTCF binding. The hypermethylation of cytosine, resulting from the suppression of TET1 and TET2 activities by D-2-HG, contributes to the dissociation of CTCF from DNA damage regions. CTCF depletion leads to the disruption of chromatin organization around the DNA damage sites, which abolishes the recruitment of essential DNA damage repair proteins BRCA2 and RAD51, as well as impairs homologous repair in the IDH mutant cancer cells. These findings provide evidence that CTCF-mediated chromatin interactions play a key role in DNA damage repair proceedings. Oncometabolites jeopardize genome stability and DNA repair by affecting high-order chromatin structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Epigenetic regulation of gene expression and host defense is well established in microbial communities, with dozens of DNA modifications comprising the epigenomes of prokaryotes and bacteriophage. Phosphorothioation (PT) of DNA, in which a chemically reactive sulfur atom replaces a non-bridging oxygen in the sugar-phosphate backbone, is catalyzed by dnd and ssp gene families widespread in bacteria and archaea. However, little is known about the role of PTs or other microbial epigenetic modifications in the human microbiome. Here we optimized and applied fecal DNA extraction, mass spectrometric, and metagenomics technologies to characterize the landscape and temporal dynamics of gut microbes possessing PT modifications. Results Exploiting the nuclease-resistance of PTs, mass spectrometric analysis of limit digests of PT-containing DNA reveals PT dinucleotides as part of genomic consensus sequences, with 16 possible dinucleotide combinations. Analysis of mouse fecal DNA revealed a highly uniform spectrum of 11 PT dinucleotides in all littermates, with PTs estimated to occur in 5–10% of gut microbes. Though at similar levels, PT dinucleotides in fecal DNA from 11 healthy humans possessed signature combinations and levels of individual PTs. Comparison with a widely distributed microbial epigenetic mark, m6dA, suggested temporal dynamics consistent with expectations for gut microbial communities based on Taylor’s Power Law. Application of PT-seq for site-specific metagenomic analysis of PT-containing bacteria in one fecal donor revealed the larger consensus sequences for the PT dinucleotides in Bacteroidota, Bacillota (formerly Firmicutes), Actinomycetota (formerly Actinobacteria), and Pseudomonadota (formerly Proteobacteria), which differed from unbiased metagenomics and suggested that the abundance of PT-containing bacteria did not simply mirror the spectrum of gut bacteria. PT-seq further revealed low abundance PT sites not detected as dinucleotides by mass spectrometry, attesting to the complementarity of the technologies. Video Abstract Conclusions The results of our studies provide a benchmark for understanding the behavior of an abundant and chemically reactive epigenetic mark in the human gut microbiome, with implications for inflammatory conditions of the gut. Supplementary Information The online version contains supplementary material available at 10.1186/s40168-025-02071-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062eb16ec95e00ee15dbf662d41e9f99a1cc1275" target='_blank'>
              Temporal dynamics and metagenomics of phosphorothioate epigenomes in the human gut microbiome
              </a>
            </td>
          <td>
            Shane R Byrne, M. DeMott, Yifeng Yuan, Farzan Ghanegolmohammadi, Stefanie M. Kaiser, James G. Fox, Eric J. Alm, Peter C. Dedon
          </td>
          <td>2025-03-24</td>
          <td>Microbiome</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Scriptaid is a chemical compound with anti-tumoural effects due to its role as a histone deacetylase inhibitor. Despite sharing part of the chemical structure with other ligands of G-quadruplexes (G4s), the interaction of Scriptaid with G4s has not been explored before. We synthesized Scriptaid and screened its cytotoxic activity in cellular models of colorectal cancer (CRC). We extensively evaluated its biological activity by cell cycle, immunofluorescence, qRT-PCR and Western blot experiments. To identify the G4 targets of Scriptaid, we conducted a panel of binding assays. Here, we show that Scriptaid induced cytotoxicity, cell cycle arrest and nucleolar stress in CRC cells. Moreover, Scriptaid impaired RNA polymerase I (Pol I) transcription, stabilized G4s and caused DNA damage. Finally, we disclose that these effects were attributable to the binding of Scriptaid to G4s in ribosomal DNA. In conclusion, our work reveals that a primary impact of Scriptaid on human cells is the interaction with G4s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650a5d0a9e8fbb2585c46607a446ad7a0c9f6bc" target='_blank'>
              The histone deacetylase inhibitor Scriptaid targets G-quadruplexes
              </a>
            </td>
          <td>
            Victoria Sánchez-Martín, Dušan Ružić, María J Tello-López, Andrea Ortiz-Morales, Javier Murciano-Calles, M. Soriano, Katarina Nikolić, José Antonio García-Salcedo
          </td>
          <td>2025-02-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43e345844c8f33145e79c69db85ef993aa97f99" target='_blank'>
              Inhibition of cytosolic DNA sensing and transposon activity safeguards pluripotency
              </a>
            </td>
          <td>
            Ferran Garcia-Llagostera, Audrey L. K. Putman, Alexandros Choromidis, Bryony J. Leeke, Klaudia Stanik, Antonia Ramos-Guzmán, Benjamin Moyon, Jesus Gil, Alexis R. Barr, Michelle Percharde
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3893a0d8a3ff4afd4b28c233f6c8386b6ca8970" target='_blank'>
              The methyl-CpG-binding protein 2 inhibits cGAS-associated signaling
              </a>
            </td>
          <td>
            Hanane Chamma, Soumyabrata Guha, Moritz Schüssler, Yasmine Messaoud-Nacer, Pierre Le Hars, Mohammad Salma, Joe McKellar, Joanna Re, Morgane Chemarin, Arnaud Carrier, Michael A. Disyak, C. Taffoni, Robin Charpentier, Zoé Husson, E. Valjent, Charlotte Andrieu-Soler, Eric Soler, Karim Majzoub, I. Vila, Nadine Laguette
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d82ec08c9ad540baf17cea17836bedb8e4f45c6" target='_blank'>
              Chemical inhibition of Exon Junction Complex assembly impairs mRNA localization and neural stem cells ciliogenesis
              </a>
            </td>
          <td>
            Tommaso Villa, Oriane Pourcelot, D. Dierks, Marion Faucourt, Cindy Burel, Floric Slimani, Léa Guyonnet, Nathalie Spassky, Schraga Schwartz, Edouard Bertrand, Olivier Bensaude, H. Le Hir
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Eukaryotic somatic cells undergo continuous telomere shortening because of end-replication problems. Approximately 10%~15% of human cancers rely on alternative lengthening of telomeres (ALT) to overcome telomere shortening. ALT cells are characterized by persistent telomere DNA replication stress and rely on recombination-based DNA repair pathways for telomere elongation. The Bloom syndrome (BLM) helicase is a member of the RecQ family, which has been implicated as a key regulator of the ALT mechanism as it is required for either telomere length maintenance or telomere clustering in ALT-associated promyelocytic leukemia bodies (APBs). Here, we summarize recent evidence detailing the role of BLM in the activation and maintenance of ALT. We propose that the role of BLM-dependent recombination and its interacting proteins remains a crucial question for future research in dissecting the molecular mechanisms of ALT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecfa6b604141f203a1f4a0229bf2941af19cc8c" target='_blank'>
              Multiple functions of the ALT favorite helicase, BLM
              </a>
            </td>
          <td>
            Shun Chang, Jiang Tan, Ren Bao, Yanduo Zhang, Jinkai Tong, Tongxin Jia, Jing Liu, Juhua Dan, Shuting Jia
          </td>
          <td>2025-03-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response. Methods In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment. Results Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure. Discussion The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, S. Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, , Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f33e9da345df8224f97295c4e7efc05d28c012" target='_blank'>
              Crosstalk Between Plk1 and PTEN in Mitosis Affects Chromosomal Stability.
              </a>
            </td>
          <td>
            Wei Li, Xianning Wang, Jiannan Liu, Bing Liu, Yongjian Hao
          </td>
          <td>2025-03-21</td>
          <td>DNA and cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) play numerous roles in cellular biology and alterations in lncRNA expression profiles have been implicated in a variety of cancers. Here, we identify and characterize a lncRNA, TRIM28 Interacting DNA damage repair Enhancing Noncoding Transcript (TRIDENT), whose expression is induced upon epithelial growth factor receptor (EGFR) activation, and which exerts pro-oncogenic functions in EGFR-driven non-small cell lung cancer. Knocking down TRIDENT leads to decreased tumor-cell proliferation in both in vitro and in vivo model systems and induces sensitization to chemotherapeutic drugs. Using ChIRP-MS analysis we identified TRIM28 as a protein interactor of TRIDENT. TRIDENT promotes phosphorylation of TRIM28 and knocking down TRIDENT leads to accumulation of DNA damage in cancer cells via decreased TRIM28 phosphorylation. Altogether, our results reveal a molecular pathway in which TRIDENT regulates TRIM28 phosphorylation to promote tumor cell growth and drug resistance. Our findings suggest that TRIDENT can be developed as a biomarker or therapeutic target for EGFR mutant non-small cell lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80bdf03e598e81f8c991cd3c3dc15b156d2a5764" target='_blank'>
              EGFR-induced lncRNA TRIDENT promotes drug resistance in non-small cell lung cancer via phospho-TRIM28-mediated DNA damage repair.
              </a>
            </td>
          <td>
            Tanvi Saxena, Anan Quan, Erica Chan, Nina Kozlova, Latika Matai, Jonathan D. Lee, R. Rupaimoole, Francisco Beca, Taru Muranen, Frank J. Slack
          </td>
          <td>2025-03-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fca76d81501556f6f871b8db3defb5ed37bf0b72" target='_blank'>
              Loss of BCL7A Permits IRF4 Transcriptional Activity and Cellular Growth in Multiple Myeloma.
              </a>
            </td>
          <td>
            C. Chakraborty, S. Talluri, Moritz Binder, E. Morelli, Jessica Encinas Mayoral, Sanika Derebail, Anil Aktas Samur, Charles B Epstein, Kenneth C Anderson, M. Shammas, M. Samur, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4b38200a1231a3f3c1164315cc7d10135fa36f" target='_blank'>
              Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system
              </a>
            </td>
          <td>
            Stefanie Egetemaier, H. Chauvistré, R. Váraljai, Yichao Hua, Smiths S. Lueong, Samira Makhzami, Nalini Srinivas, Jan Forster, V. Ullrich, Simone Stupia, Valéria Schroeder, Sarah Scharfenberg, A. Hoewner, B. Shannan, J. Siveke, M. Baietti, Eleonora Leucci, J. Marine, A. Paschen, Bjoern Scheffler, D. R. Engel, Lisa M. Becker, Felix Nensa, Johannes Koester, Barbara Grünwald, S. Poepsel, Sabrina Ninck, F. Kaschani, Dirk Schadendorf, Juergen C. Becker, A. Tasdogan, Florian Rambow, Alexander Roesch
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, , Nahoko Sato, Shraddha Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) is crucial for carcinogenesis and bile has direct contact with malignant biliary strictures, yet eccDNA features in bile and its function in malignant biliary strictures remain underexplored. Results We observed the widespread presence of eccDNA in bile and systematically profiled the landscape of bile cell-free eccDNA (bcf-eccDNA). For functional exploration, a simple and efficient workflow was designed to synthesize large eccDNA particularly containing multiple regions. Compared with the noncancer group, bcf-eccDNAs in the cancer group had different origins and larger sizes with six characteristic peaks. These peaks were also identified in the validation cohort (100%). There were more bcf-eccDNA carrying LINC00598 or CELF2 in malignant biliary strictures, showing potential diagnostic performance in training and validation cohorts (all AUCs > 0.9). Bcf-eccDNAs carried cancer-related mutations, which could guide treatment. EccDNA carrying miR-106a/363 cluster or miR-374b/421 cluster were proven to regulate cancer gene expression, accelerate tumor proliferation, and inhibit tumor apoptosis. Conclusions This study profiles a comprehensive bcf-eccDNA landscape in patients with biliary strictures and offers valuable insights into eccDNA's role in bile liquid biopsy and carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e5a35f02b297af4af1d7c2312aeb3f7e720c6f2" target='_blank'>
              p53-dependent chromatin relaxation is required for DNA double-strand break repair.
              </a>
            </td>
          <td>
            Hongyu Chen, Jin Shan, Wenjing Qi, Lili Chen, Xianlu Zeng
          </td>
          <td>2025-02-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92ea67e64484dade77882133de154f4d5bb2f2" target='_blank'>
              CTCF directly binds G-quadruplex structures to regulate genome topology and gene expression
              </a>
            </td>
          <td>
            Daniela Samaniego-Castruita, Isabella Han, Roxroy C Morgan, Samantha Carpenter, Bryce Williams, Abhijit Chakraborty, Ishwar Radhakrishnan, F. Ay, Samuel A. Myers, Anjana Rao, Vipul Shukla
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC), a highly heterogeneous disease, shows genomic instability and a high mutation rate, making it difficult to treat. Recent studies revealed that cancer stem cells (CSCs) play a critical role in MIBC frequent recurrence and high morbidity. Previous research has shown that Cyclooxygenases-2 (COX-2) is particularly highly expressed in bladder cancer cells. In recent years, the development of oncolytic adenoviruses and their use in clinical trials have gained increased attention. In this study, we composed a conditionally replicative adenovirus vector (CRAd-synNotch) that carries the COX-2 promotor driving adenoviral E1 gene, the synNotch receptor therapeutic gene, and the Ad5/35 fiber gene. Activation of the COX-2 promoter gene causes replication only within COX-2 expressing cancer cells, thereby leading to tumor oncolysis. Also, CD44 and HIF signals contribute to cancer stemness and maintaining CSCs in bladder cancer, and the transduced synNotch receptor inhibits both CD44 and HIF signals simultaneously. We performed an in vivo study using a mouse xenograft model of T24 human MIBC cells and confirmed the significant antitumor activity of CRAd-synNotch. Our findings in this study warrant the further development of CRAd-synNotch for treating patients with MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eee590b5a2cfc96e3c5fb66879d16597ce9245" target='_blank'>
              A conditionally replicative adenovirus vector containing the synNotch receptor gene for the treatment of muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Ruhan A, H. Ueki, Shunya Nishioka, Rion Yamazaki, Marina Maekawa, Koichi Kitagawa, Hideaki Miyake, Toshiro Shirakawa
          </td>
          <td>2025-02-26</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long interspersed element 1 (LINE-1) retrotransposons are repetitive sequences that can move within the genome by an autonomous mechanism. To limit their mutagenic potential, benign cells restrict LINE-1 expression through molecular mechanisms such as DNA methylation and histone modification, but these mechanisms are usually impaired in cancer. Clear cell ovarian carcinoma (CCOC) represents 5–10% of ovarian cancers and is thought to arise from endometriosis. Women with advanced CCOC face poor prognoses, highlighting the importance of understanding early disease pathogenesis. In our study, 33 of 40 cases (over 82%) of CCOC tumors express ORF1p, a LINE-1-encoded protein. We found that LINE-1 de-repression is an early event in CCOC, as ORF1p is enhanced during the transition from typical to atypical endometriosis and persists in invasive cancer. Finally, using single-molecule array (Simoa) assays, we detected ORF1p in patient blood, suggesting it as a potential minimally invasive biomarker for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891a3a8df530bd356f3a5da9253356957c1253d" target='_blank'>
              Hyper-mutational processes provide a head-start for non-optimal cancer driver mutations explaining atypical KRAS variants
              </a>
            </td>
          <td>
            Michael D. Nicholson, Ian Tomlinson
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The bulk of RNA produced from the genome of complex organisms consists of a very large number of transcripts lacking protein translational potential and collectively known as noncoding RNAs (ncRNAs). Initially thought to be mere products of spurious transcriptional noise, ncRNAs are now universally recognized as pivotal players in cell regulatory networks across a broad spectrum of biological processes. Owing to their critical regulatory roles, ncRNA dysfunction is closely associated with the etiopathogenesis of various human malignancies, including cancer. As such, ncRNAs represent valuable diagnostic biomarkers as well as potential targets for innovative therapeutic intervention. In this review, we focus on microRNAs (miRNAs) and long noncoding RNAs (lncRNAs), the two most extensively studied classes in the field of ncRNA biology. After outlining key concepts of miRNA and lncRNA biogenesis pathways, we examine their multiple roles in mediating epigenetic regulation of gene expression and chromatin organization. Finally, by providing numerous examples of specific miRNAs and lncRNAs, we discuss how dysregulation of these mechanisms contributes to the onset and/or progression of various human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a1a4f6a39f0e1f7e7aaf4a6d2a6df1f0ec1714d" target='_blank'>
              Epigenetic Regulation of Chromatin Functions by MicroRNAs and Long Noncoding RNAs and Implications in Human Diseases
              </a>
            </td>
          <td>
            Salvatore Costa, Gaspare La Rocca, Vincenzo Cavalieri
          </td>
          <td>2025-03-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The translesion DNA synthesis (TLS) pathway mediated by proliferating cell nuclear antigen (PCNA) monoubiquitination is an essential mechanism by which cancer cells bypass DNA damage caused by DNA damage to maintain genomic stability and cell survival. Chromatin assembly factor 1 subunit A (CHAF1A) traditionally promotes histone assembly during DNA replication. Here, we revealed that CHAF1A is a novel regulator of the TLS pathway in cancer cells. CHAF1A promotes restart and elongation of the replication fork under DNA replication stress. Mechanistically, the C-terminal domain of CHAF1A directly interacts with E3 ubiquitin ligase RAD18, enhancing RAD18 binding on the stalled replication fork. CHAF1A facilitates PCNA K164 monoubiquitination mediated by RAD18, thereby promoting the recruitment of Y-family DNA polymerases and enhancing cancer cell resistance to DNA damage. In addition, CHAF1A-mediated RAD18 recruitment and PCNA monoubiquitination are independent of the CHAF1A-PCNA interaction and its histone assembly function. Taken together, these findings improve our understanding of the mechanisms that regulate the TLS pathway and provide insights into the relationship between CHAF1A and DNA replication stress in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d19733c6d85f7d8431cd411691114ef80a3ce053" target='_blank'>
              Chromatin assembly factor 1 subunit A promotes TLS pathway by recruiting E3 ubiquitin ligase RAD18 in cancer cells
              </a>
            </td>
          <td>
            Bing Wen, Hai-Xiang Zheng, Jing-Hua Heng, Qian Tang, Danxia Deng, Zhi-Da Zhang, Lian-di Liao, Liyan Xu, E. Li
          </td>
          <td>2025-03-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background. Mutations in homologous recombination repair (HRR) genes (BRCA2, ATM, CHEK2, NBN, etc.) are found in 20–25 % of patients with metastatic prostate cancer (PC) and are an indication for prescription of PARP inhibitors. Sensitivity to these drugs results from homologous recombination deficiency (HRD) in the tumor. It is currently not fully elucidated which HRR genes besides BRCA1/2 cause HRD. The aim of the study was to evaluate the presence of homologous recombination deficiency in pc associated with mutations in different HRR genes. Material and methods. Paired tumor and normal DNA samples from 272 pc patients were examined using the HiSNP Ultra Panel v1.0 NGS panel (Nanodigmbio). Tumor copy number variation profiles were used to obtain the HRD score, which was determined as the unweighted sum of three characteristics: the number of chromosomal regions with loss of heterozygosity (LOH), large scale state transitions (LST), and telomeric allelic imbalances (TAI). HRD scores in different pc categories were compared using the Mann–Whitney test. Results. The studied case series included 58 pcs with pathogenic/ likely pathogenic mutations in at least one of 34 HRR genes, and 214 pcs without HRR mutations. The median HRD scores in the groups with BRCA2, ATM, CHEK2, other HRR mutations and without HRR mutations were 41, 22.5, 7.5, 7.5, 14 and 9, respectively. The HRD score was significantly higher in BRCA2-associated tumors than in other PC categories (p<0.01 for all comparisons), except in cases with atm mutations, where the difference did not reach formal significance (p=0.051). Homologous recombination deficiency, defined as HRD score ≥25, was observed in 19/58 (32.8 %) PCs with HRR mutations and 40/214 (18.7 %) tumors without HRR mutations (p=0.03). In tumors without HRR mutations, the HRD score was significantly higher in the presence of somatic TP53 mutations (p<0.0001). Conclusion. In contrast to BRCA2-associated PCs, most tumors with mutations in the CHEK2, NBN, BLM, FANCM, BRCA1 genes are not characterized by homologous recombination deficiency. in the case of ATM gene lesions, approximately half of pcs have a high HRD score. Testing for HRD score allows identification of a significant proportion (5–19 %, depending on the threshold chosen) of tumors with HRD among pcs that do not contain mutations in HRR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/475a2bac32f3b3fba84f14504ff2c301042d61c8" target='_blank'>
              Analysis of homologous recombination deficiency in prostate cancer
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, N. S. Morozova, A. D. Shestakova, E. Kuligina, E. Imyanitov
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA–RNA hybrids triggered by double-strand breaks (DSBs) are crucial intermediates during DSB repair, and their timely resolution requires numbers of RNA helicases, including DEAD box 1 (DDX1). However, how these helicases are recruited to DSB-induced hybrids in time remains largely unclear. Here, we revealed that squamous cell carcinoma antigen recognized by T cells 3 (SART3) promotes DDX1 binding to DNA–RNA hybrids at DSBs for optimal homologous recombination (HR) repair. SART3 itself associates with DNA–RNA hybrids and PAR chains and accumulates at DSBs in both PARylation- and DNA–RNA hybrids-dependent fashion. SART3 also associates with DDX1 and is necessary for DDX1 enrichment at DSBs. The defective SART3-DDX1 association observed in cells expressing the cancer-associated variant SART3-R836W impairs not only the accumulation of DDX1, but also hybrid removal and HR efficiency. Moreover, SART3 promotes DNA end resection through enhancing USP15-BARD1 association and BRCA1-BARD1 retention. Together, our study reveals an role of SART3 in DSB repair, rendering SART3 a promising target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0676fb55661a5af6ba07cbca79e578a0d145a7" target='_blank'>
              SART3 promotes homologous recombination repair by stimulating DNA-RNA hybrids removal and DNA end resection
              </a>
            </td>
          <td>
            Hui Fu, Min Huang, Honglin Wu, Hui Zheng, Yifei Gong, Lingyu Xing, Juanjuan Gong, Ruiyuan An, Qian Li, Xinyu Jie, Xiaolu Ma, Tie-Shan Tang, Caixia Guo
          </td>
          <td>2025-03-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Progressing transcription and replication machineries profoundly impact their underlying chromatin template. Consequently, transcription-replication conflict (TRC) sites are vulnerable to chromatin and epigenome alterations, provoking genome instability. Here, we engineered an inducible TRC reporter system using a genome-integrated R-loop-prone sequence and characterized the dynamic changes of the local chromatin structure inflicted by TRCs, leading to reduced nucleosome occupancy and replication fork blockage. Strikingly, inducing a small number of TRCs on the genome results in a measurable global replication stress response. Furthermore, we find a TRC-dependent increase in H3K79 methylation specifically at the R-loop forming TRC site. Accordingly, inhibition of the H3K79 methyltransferase DOT1L leads to reduced transcriptional output and an exacerbated DNA damage response, suggesting that deposition of this mark is required for effective transcription recovery and resolution of TRCs. Our work shows the molecular dynamics and reveals a specific epigenetic modifier bookmarking TRC sites, relevant to cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2c4476d7a403d74c2111b85e2186854b6c2af7f" target='_blank'>
              Transcription-replication conflicts drive R-loop-dependent nucleosome eviction and require DOT1L activity for transcription recovery
              </a>
            </td>
          <td>
            Marcel Werner, Manuel Trauner, Tamas Schauer, Henning Ummethum, Elizabeth Márquez-Gómez, Maxime Lalonde, Clare S. K. Lee, Ioannis Tsirkas, Atiqa Sajid, Augusto C Murriello, Gernot Längst, Stephan Hamperl
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Germline mutations in the tumor suppressor phosphatase and tensin homolog (PTEN) cause PTEN hamartoma tumor syndrome (PHTS). PHTS is characterized by an elevated lifetime risk of differentiated thyroid cancer (DTC), 30 times higher than the general population. However, only 1 in 3 PHTS patients develop DTC, and it remains unknown whether specific PTEN variants are associated with an increased risk of DTC. PTEN antagonizes the phosphatidylinositol 3-kinase (PI3K)–AKT signaling pathway, a frequently affected pathway in sporadic DTC. PTEN also acts as a guardian of the genome by interacting with other tumor suppressors. Here, we report how ionizing radiation, an environmental tumorigenic contributor, modifies the DNA damage response based on the type of germline PTEN variants. We hypothesized that certain PTEN variants associated with DTC create a pro-oncogenic molecular signature upon radiation-induced DNA damage. DTC-associated (PTENM134R) or DTC-non-associated (PTENG132D) germline PTEN mutant alleles were introduced into a human induced pluripotent cell (hiPSC) line derived from a healthy donor utilizing CRISPR-Cas9 gene editing technology. We determined radiation-induced transcriptomic changes in functional thyroid organoids induced from wild-type and both heterozygous PTEN mutant hiPSCs. Both bulk and single-cell RNA sequencing data indicated that radiation upregulated the p53 network more potently in the thyroid organoids with PTENWT/G132D than those with PTENWT/M134R, which could be mediated by AKT-dependent MDM2 inactivation and PTEN–p53 physical interaction. Our data suggest that the lack of p53 pathway activation through PTEN–p53 network interactions explains why PTENM134R is a DTC-susceptible variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c92c95603d11b5160f145d679137c4a2e82b3ae" target='_blank'>
              Divergent PTEN–p53 interaction upon DNA damage in a human thyroid organoid model with germline PTEN mutations
              </a>
            </td>
          <td>
            Juan Venegas, Omer Enes Onur, Shin Chung Kang, Masahiro Hitomi, C. Eng
          </td>
          <td>2025-02-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Decitabine (DAC), a well-recognized DNA hypomethylating agent, has been applied to treat acute myeloid leukemia. However, clinic investigations revealed that DNA methylation reduction does not correlate with a clinical response, and relapse is prevalent. To gain a better understanding of its anti-tumor mechanism, we perform a temporally resolved CRISPR-Cas9 screen to identify factors governing the DAC response. We show that DNA damage generated by DNMT-DNA adducts and 5-aza-dUTP misincorporation through the dCMP deaminase DCTD act as drivers of DAC-induced acute cytotoxicity. The DNA damage that arises during the next S phase is dependent on DNA replication, unveiling a trans-cell cycle effect of DAC on genome stability. By exploring candidates for synthetic lethality, we unexpectedly uncover that KDM1A promotes survival after DAC treatment through interactions with ZMYM3 and CoREST, independent of its demethylase activity or regulation of viral mimicry. These findings emphasize the importance of DNA repair pathways in DAC response and provide potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, J. Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The current treatment of triple-negative breast cancer (TNBC) is still primarily based on platinum-based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin-tolerant cells after platinum administration. Here, it is reported that upregulated N 6-methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase-like 3 (METTL3), facilitating METTL3 interaction with Wilms'-tumor-1-associated protein and subsequently increasing m6A of transcript-associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2-Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer driver mutations are defined by their high prevalence in cancers and presumed rarity in normal tissues. However, recent studies show that positive selection in normal epithelia can increase the prevalence of some cancer drivers. To determine their true cancer-driving potential, it is essential to evaluate how frequent these mutations are in normal tissues and what are their phenotypes. Here, we explore the bioavailability of somatic variants by quantifying age-related mutational burdens in normal human colonic epithelium using immunodetection in FFPE samples (N = 181 patients). Positive selection of variants of tumour suppressor genes PTEN and ARID1A associates with monoallelic gene loss as confirmed by CRISPR/Cas9 mutagenesis and changes in their downstream effectors. Comparison of the mutational burden in normal tissue and colorectal cancers allows quantification of cancer driver potency based on relative representation. Additionally, immune exclusion, a cancer hallmark feature, is observed within ARID1A-deficient clones in histologically normal tissue. The behaviour resulting from haploinsufficiency of PTEN and ARID1A demonstrates how somatic mosaicism of tumour suppressors arises and can predispose to cancer initiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM) is one of the most lethal cancers translating into 75% of skin cancer-related deaths. Despite the advances in SKCM management and treatment strategies, the overall survival of patients remains unsatisfactory due to the metastatic properties of SKCM as well as the absence of effective prognostic biomarkers. Recent studies have shown that overload copper renders accumulation of mitochondrial proteins and fuels a form of cell death at odds with known death mechanisms and is hinged on mitochondrial respiration, the so-called cuproptosis. However, the exact role of cuproptosis in SKCM development and progression is unknown, and painting a clear picture of its functions in SKCM is fraught with challenges. A more systematic investigation is justified. In this study, we were posed to dissect the clout and latent regulatory mechanisms of cuproptosis-related genes (CRGs) in reining in SKCM progression. Also, we identified three CRGs that stood out were used to construct a prognostic model, which could be employed to predict the prognosis of patients with SKCM. Finally, through pan-cancer analysis, we found that the four cuproptosis key genes play a role in multiple tumors, suggesting that cuproptosis may impact tumor progression at the pan-cancer level. Taken together, these findings may not only contribute to the development of treatment strategies but also provide clues for treatment decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9961e6ed0a6d198d471203dc74fb5135efa471b" target='_blank'>
              CDKN2A, a key gene in copper-induced cell death model, influencing melanoma invasion and apoptosis
              </a>
            </td>
          <td>
            Jing Li, Xi Yang, Cunli Yin, Siru Li, Yan Xu, Bin Liu
          </td>
          <td>2025-02-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Epigenes are defined as proteins that perform post-translational modification of histones or DNA, reading of post-translational modifications, form complexes with epigenetic factors or changing the general structure of chromatin. This specialized group of proteins is responsible for controlling the organization of genomic DNA in a cell-type specific fashion, controlling normal development in a spatial and temporal fashion. Moreover, mutations in epigenes have been implicated as causal in germline pediatric disorders and as driver mutations in cancer. Despite their importance to human disease, to date, there has not been a systematic analysis of the sources of functional diversity for epigenes at large. Epigenes’ unique functions that require the assembly of pools within the nucleus suggest that their structure and amino acid composition would have been enriched for features that enable efficient assembly of chromatin and DNA for transcription, splicing, and post-translational modifications. Results In this study, we assess the functional diversity stemming from gene structure, isoforms, protein domains, and multiprotein complex formation that drive the functions of established epigenes. We found that there are specific structural features that enable epigenes to perform their variable roles depending on the cellular and environmental context. First, epigenes are significantly larger and have more exons compared with non-epigenes which contributes to increased isoform diversity. Second epigenes participate in more multimeric complexes than non-epigenes. Thirdly, given their proposed importance in membraneless organelles, we show epigenes are enriched for substantially larger intrinsically disordered regions (IDRs). Additionally, we assessed the specificity of their expression profiles and showed epigenes are more ubiquitously expressed consistent with their enrichment in pediatric syndromes with intellectual disability, multiorgan dysfunction, and developmental delay. Finally, in the L1000 dataset, we identify drugs that can potentially be used to modulate expression of these genes. Conclusions Here we identify significant differences in isoform usage, disordered domain content, and variable binding partners between human epigenes and non-epigenes using various functional genomics datasets from Ensembl, ENCODE, GTEx, HPO, LINCS L1000, and BrainSpan. Our results contribute new knowledge to the growing field focused on developing targeted therapies for diseases caused by epigene mutations, such as chromatinopathies and cancers. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00571-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedbdbd3810726b7db1ec867ee9197cd7581b73a" target='_blank'>
              Epigene functional diversity: isoform usage, disordered domain content, and variable binding partners
              </a>
            </td>
          <td>
            Leroy Bondhus, Aileen A. Nava, Isabelle S. Liu, Valerie A. Arboleda
          </td>
          <td>2025-02-01</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c78c5dd12eb473d301e95ef3085c40bdc1f7b393" target='_blank'>
              A History of Cancer Research: The P53 Pathway.
              </a>
            </td>
          <td>
            Joseph Lipsick
          </td>
          <td>2025-02-03</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a859188c2bcbbbd269e7e181ed7e2f2850df16d" target='_blank'>
              Amplified Production of a DNA Decoy Catalyzed by Intracellular MicroRNA.
              </a>
            </td>
          <td>
            Soshu Yasuda, K. Morihiro, Shuichiro Koga, Akimitsu Okamoto
          </td>
          <td>2025-02-03</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S-adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient-deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti-Lat4 small interfering RNA complex nanoregulator (AS-F-NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor-infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation.
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Cancer is a disease of multicellularity, observed across the tree of life. In principle, animals with larger body sizes and longer lifespans should be at increased risk of developing cancer. However, there is no strong relationship between these traits and cancer across mammals. Previous studies have proposed that increased copy number of cancer-related genes may enhance the robustness of cancer suppression pathways in long-lived mammals, but these studies have not extended beyond known cancer-related genes. In this study, we conducted a phylogenetic generalized least squares analysis to test for associations between copy number of all protein-coding genes and longevity, body size, and cancer prevalence across 94 species of mammals. In addition to investigating the copy number of individual genes, we tested sets of related genes for a relationship between the aggregated gene copy number of the set and these traits. We did not find strong evidence to support the hypothesis that adaptive changes in gene copy number contribute to the lack of correlation between cancer prevalence and body size or lifespan. However, we found several biological processes where aggregate copy number was associated with malignancy rate. The strongest association was for the gene set relating to transforming growth factor beta, a cytokine that plays a role in cancer progression. Overall, this study provides a comprehensive evaluation of the role of gene copy number in adaptation to body size and lifespan and sheds light on the contribution of gene copy number to variation in cancer prevalence across mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a5893a996e91a392fac375709075fef182cddc" target='_blank'>
              Variable Gene Copy Number in Cancer-Related Pathways Is Associated With Cancer Prevalence Across Mammals
              </a>
            </td>
          <td>
            Sophie Matthews, Vahid Nikoonejad Fard, Marc Tollis, Cathal Seoighe
          </td>
          <td>2025-03-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbe80a72237e1c2774d5c95c120ee2f1e9208a6" target='_blank'>
              Expression of Fascin and DNA Topoisomerase 2-Alpha in Breast Carcinoma: Correlation with Histological Subtypes and Other Prognostic Markers
              </a>
            </td>
          <td>
            Alberto Sánchez-Espinosa, José García-Rodríguez, Virginia Alonso-Aguirre, Jesús Acosta‐Ortega, Pablo Conesa-Zamora, J. García‐Solano, G. Luengo-Gil
          </td>
          <td>2025-03-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Gastrointestinal stromal tumors (GISTs) span a broad clinical spectrum, from indolent neoplasms to life‐threatening metastatic tumors. A persistent limitation of current risk stratification systems is that a subset of GISTs is graded as low‐risk but nevertheless metastasizes. Therefore, new predictive factors that improve risk stratification are needed. In this exploratory study, we investigated the potential of genome‐wide DNA methylation profiling and copy number variation (CNV) analysis as additional prognostic tools for GISTs. We collected a cohort of 28 patients with GIST diagnosed between 2001 and 2022, with available follow‐up and molecular data. This included 15 patients without progressive disease (seven low‐risk and eight moderate‐ to high‐risk GISTs) and 13 with progressive disease. Among those with progression, eight experienced recurrence or metastasis post‐surgery (one low‐risk, seven high‐risk GISTs), while five had metastatic disease at initial diagnosis. Risk stratification was determined according to Miettinen's criteria. Genome‐wide DNA methylation data and CNV plots were generated from imatinib‐naïve primary GISTs using the Illumina Infinium MethylationEPIC BeadChip array. Unsupervised cluster analysis revealed distinct DNA methylation patterns predominantly associated with anatomical location and genotype. Differential DNA methylation analysis comparing primary gastric GISTs associated with and without progressive disease showed 8 differentially methylated regions spanning the coding and promoter areas of 6 genes. CNV analysis demonstrated that GISTs associated with progressive disease had the most CNVs, whereas low‐risk, non‐progressive GISTs had the fewest. Despite the limited sample size, this exploratory study indicates that genome‐wide DNA methylation profiling and CNV analysis could enhance GIST risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cde7ce155d81225e7ec733ade903dd81397bee4" target='_blank'>
              Genome‐Wide DNA Methylation and Copy Number Alterations in Gastrointestinal Stromal Tumors
              </a>
            </td>
          <td>
            T. Kleijn, B. Ameline, R. Bleckman, W. Kooistra, E. van den Broek, G. Diercks, B. V. van Hemel, Bert Timmer, W. Timens, G. Kats-Ugurlu, L. V. van Kempen, B. van Etten, E. Schuuring, Albert J H Suurmeijer, J.J. de Haan, D. Baumhoer, A. K. Reyners, A. Cleven
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise approximately half of eukaryotic genomes and significantly contribute to genome plasticity. In this study, we focused on a specific TE, MERVL, which exhibits particular expression during the 2-cell stage and commonly serves as an indicator of embryonic totipotency. However, its precise role in embryo development remains mysterious. We utilized DRUG-seq to investigate the effects of oxidative damage on genes and TEs expression. Our findings revealed that exposure to hydrogen peroxide (H2O2) could induce DNA damage, apoptosis, and incomplete DNA demethylation in embryos, which were potentially associated with MERVL expression. To further explore its function, antisense nucleotides (ASO) targeting MERVL were constructed to knockdown the expression in early embryos. Notably, this knockdown led to the occurrence of DNA damage and apoptosis as early as the 2-cell stage, consequently reducing the number of embryos that could progress to the blastocyst stage. Moreover, we discovered that MERVL exerted an influence on the reprogramming of embryonic DNA methylation. In MERVL-deficient embryos, the activity of the DNA demethylase ten-eleven translocation 3 (TET3) was suppressed, resulting in impaired demethylation when compared to normal development. This impairment might underpin the mechanism that impacts embryonic development. Collectively, our study not only verified the crucial role of MERVL in embryonic development but also probed its regulatory function in DNA methylation reprogramming, thereby laying a solid foundation for further investigations into MERVL's role. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01143-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ed27bb6383161df0bd1281e88568674842389e" target='_blank'>
              Transposition element MERVL regulates DNA demethylation through TET3 in oxidative-damaged mouse preimplantation embryos
              </a>
            </td>
          <td>
            Lihong Liu, Siyao Ha, Dan Cao, MingQing Li, Zhiling Li
          </td>
          <td>2025-03-12</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Birt-Hogg-Dubé syndrome (BHD) patients are uniquely susceptible to all renal tumor subtypes. However, the underlying mechanism of carcinogenesis is unclear. To study cancer development in BHD, we used human proximal kidney (HK2) cells and found that long-term folliculin (FLCN) knockdown was required to increase the tumorigenic potential of these cells, as evidenced by the formation of larger spheroids under nonadherent conditions. Transcriptomic and proteomic analyses revealed links between the FLCN, cell cycle control and DNA damage response (DDR) machinery. In addition, HK2 cells lacking FLCN had an altered transcriptome profile and enriched cell cycle control genes. G1/S cell cycle checkpoint signaling was compromised by increased protein levels of cyclin D1 (CCND1) and hyperphosphorylation of retinoblastoma 1 (RB1). A FLCN interactome screen revealed that FLCN binds to DNA-dependent protein kinase (DNA-PK). This novel interaction was reversed in an irradiation-responsive manner. Knockdown of FLCN in HK2 cells caused a marked increase in γH2AX and RB1 phosphorylation. The levels of both CCND1 and phosphorylated RB1 remained high during DNA damage, which was associated with defective cell cycle control caused by FLCN knockdown. Furthermore, Flcn-knockdown C. elegans were defective in cell cycle arrest caused by DNA damage. This work revealed that long-term FLCN loss and associated cell cycle defects in BHD patients could contribute to their increased risk of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce72e1df530d7f8cfbe56a7a8e356852f9f7441" target='_blank'>
              Characterizing the tumor suppressor activity of FLCN in Birt-Hogg-Dubé syndrome cell models through transcriptomic and proteomic analysis.
              </a>
            </td>
          <td>
            Rachel-Ann Jones, E. Dunlop, Jesse D Champion, Peter F Doubleday, T. Claessens, Zahra Jalali, Sara Seifan, Iain Perry, Peter J. Giles, Oliver Harrison, Barry J. Coull, A. Houweling, A. Pause, Bryan A Ballif, Andrew R. Tee
          </td>
          <td>2025-03-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) gaps impact genome stability and PARP inhibitor (PARPi) sensitivity, especially in BRCA1/2-deficient tumors. Using single-molecule DNA fiber analysis, electron microscopy, and biochemical methods, we found that MRN, CtIP, EXO1, and DNA2-WRN/BLM resect ssDNA gaps through a mechanism different from their actions at DNA ends. MRN resects ssDNA gaps in the 3'-to-5' direction using its pCtIP-stimulated exonuclease activity. Unlike at DNA ends, MRN does not use its endonucleolytic activity to cleave the 5'-terminated strand flanking the gap or the ssDNA. EXO1 and DNA2-WRN/BLM specifically resect the 5' end of the gap independent of MRN-CtIP. This resection process alters ssDNA gap repair kinetics in BRCA1-proficient and -deficient cells. In BRCA1-deficient cells treated with PARPis, excessive resection results in larger ssDNA gaps, hindering their repair and leading to DNA breaks in subsequent cell cycle stages due to ssDNA gaps colliding with DNA replication forks. These findings broaden our understanding of the role of human nucleases in DNA metabolism and have significant implications for defining the mechanisms driving PARPi sensitivity in BRCA-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5bfadd87ec60e42f11dd8435521e451769fad" target='_blank'>
              MRN-CtIP, EXO1, and DNA2-WRN/BLM act bidirectionally to process DNA gaps in PARPi-treated cells without strand cleavage.
              </a>
            </td>
          <td>
            Isabelle M Seppa, Ilaria Ceppi, Mithila Tennakoon, Giordano Reginato, Jessica Jackson, Celia D Rouault, Sumedha Agashe, Vladislav O Sviderskiy, Mangsi Limbu, Erica Lantelme, A. Meroni, Stefan Braunshier, Damiano Borrello, Priyanka Verma, Petr Cejka, A. Vindigni
          </td>
          <td>2025-03-24</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d901b9b33dfa1544156c22b0678e85042f152af" target='_blank'>
              Genomic landscaping of receptor tyrosine kinase ALK with highly frequent rearrangements in cancers.
              </a>
            </td>
          <td>
            Wei Ye, Wen-Bin Ou
          </td>
          <td>2025-02-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Over the past decade, circular RNAs (circRNAs) have gained recognition as a novel class of genetic molecules, many of which are implicated in cancer pathogenesis via different mechanisms, including drug resistance, immune escape, and radio-resistance. ExosomalcircRNAs, in particular, facilitatecommunication between tumour cells and micro-environmental cells, including immune cells, fibroblasts, and other components. Notably, micro-environmental cells can reportedly influence tumour progression and treatment resistance by releasing exosomalcircRNAs. circRNAs often exhibit tissue- and cancer-specific expression patterns, and growing evidence highlights their potential clinical relevance and utility. These molecules show strong promise as potential biomarkers and therapeutic targets for cancer diagnosis and treatment. Therefore, this review aimed to briefly discuss the latest findings on the roles and resistance mechanisms of key circRNAs in the treatment of various malignancies, including lung, breast, liver, colorectal, and gastric cancers, as well as haematological malignancies and neuroblastoma.This review will contribute to the identification of new circRNA biomarkers for the early diagnosis as well as therapeutic targets for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background TP53 mutations are linked to aggressive progression and chemoresistance in gastric cancer (GC). Frameshift mutation is the second most common mutation type of TP53. However, the consequences of this mutation type in GC were not well understood, and targeted therapies for cancer patients harboring frameshift mutations were also not established. Histone methylation significantly influences tumorigenesis in TP53-mutated cancers, and related inhibitors are emerging as specific therapeutic strategies. Methods and results By treating GC cell lines harboring various TP53 mutation types with a library of histone demethylase inhibitors, we identified that GSK690, a reversible inhibitor of lysine-specific demethylase 1 (LSD1), selectively inhibits GC cells harboring TP53 frameshift mutations without nuclear localization sequence (NLS) (termed TP53 Frameshift NLS), which accounts for 89% TP53 frameshift mutations in GC patients. GSK690 showed significant specific inhibition in vitro and in vivo against this subtype by inducing G1/S cell cycle arrest via the LSD1-CCNA2 axis. Importantly, dual-luciferase assays and ChIP-qPCR confirmed that the loss of transcriptional repression activities of p53 in drives LSD1 upregulation in TP53 Frameshift NLS cancer cells. Conclusions In summary, our results indicate that the nuclear localization deficiency of p53 accounts for increased expression of LSD1 in TP53 Frameshift NLS GCs. GSK690 inhibits cell cycle progression and tumor growth by suppressing aberrantly activated LSD1-CCNA2 signaling in this GC subtype, counteracting malignant proliferation and thereby providing a precise therapeutic strategy for GC patients with TP53 Frameshift NLS. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01829-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality in homologous recombination (HR)-deficient cancers caused by Poly (ADP-ribose) polymerase inhibitors (PARPi) has been classically attributed to its role in DNA repair. The mode of action of PARPi and resistance thereof are now believed to be predominantly replication associated. Therefore, effective combinatorial approaches of targeting replication fork processing along with HR-downregulation to target HR-proficient and possibly PARPi-resistant tumors are warranted. Stilbenes are a privileged class of molecules, which include resveratrol, pterostilbene, piceatannol, etc, that modulate both replication processes and RAD51-expression. In this investigation, by screening a small library of stilbenes, including in-house synthesized molecules, trans-4,4'-dihydroxystilbene (DHS) was discovered as a potent natural agent, which downregulates RAD51 expression and HR repair (GFP-reporter assay). DHS induces extensive synergistic cell death in ovarian cancers when combined with talazoparib (PARPi). Mechanistically, DHS elicits replication-stress through severely impeding replication fork progress, speed, and inducing fork-asymmetry. This leads to robust induction of single stranded DNA (ssDNA) gaps and poly-ADP-ribosylation (PARylation) in S-phase cells, signifying issues related to lagging (Okazaki) strand synthesis. PARPi, which abrogates PARylation, potentiates DHS induced ssDNA gaps, and their conversion into lethal double strand breaks through MRE11 action. Furthermore, the combination is highly effective in mitigating ovarian tumor xenograft growth in SCID mice and exhibited a good therapeutic-index with no/minimal tissue-toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e984c0229d51f87822834c928220d9d0bc66005" target='_blank'>
              Targeting Replication Fork Processing Synergizes with PARP Inhibition to Potentiate Lethality in Homologous Recombination Proficient Ovarian Cancers.
              </a>
            </td>
          <td>
            Ganesh Pai Bellare, Kshama Kundu, Papiya Dey, Krupa Thankam Philip, Nitish Chauhan, Muskan Sharma, Sankarsingh Kesharsingh Rajput, B. Patro
          </td>
          <td>2025-03-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd93a55f84ce2095fede9891e98cc26d5f84458d" target='_blank'>
              Doxorubicin, a DNA intercalator, inhibits transcription elongation.
              </a>
            </td>
          <td>
            Mathew Tempel, Kari Green, Dhanvi Prajapati, Angela Duaqui, Mahboobeh Norouzi, Hedieh Sattarifard, Ahmed Ashraf, Elly Wu, Athanasios Zovoilis, Ted M Lakowski, James R Davie
          </td>
          <td>2025-02-14</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c19557776012bc520b8b5ecf4614f5b4aaa6f641" target='_blank'>
              The transcription factor SRF regulates MERVL retrotransposons and gene expression during zygotic genome activation.
              </a>
            </td>
          <td>
            C. Hermant, Carlos Michel Mourra-Díaz, Marlies E. Oomen, Luis Altamirano-Pacheco, Mrinmoy Pal, Tsunetoshi Nakatani, M. Torres-Padilla
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are unique nucleic acid structures composed of guanine-rich (G-rich) sequences that can form diverse topologies based on the arrangement of their four strands. G4s have attracted attention for their potential roles in various biological processes and human diseases. In this review, we focus on the G4 structures formed by human telomeric sequences, (GGGTTA)n, and the hexanucleotide repeat expansion, (GGGGCC)n, in the first intron region of the chromosome 9 open reading frame 72 (C9orf72) gene, highlighting their structural diversity and biological significance. Human telomeric G4s play crucial roles in telomere retention and gene regulation. In particular, we provide an in-depth summary of known telomeric G4s and focus on our recently discovered chair-type conformation, which exhibits distinct folding patterns. The chair-type G4s represent a novel folding pattern with unique characteristics, expanding our knowledge of telomeric G4 structural diversity and potential biological functions. Specifically, we emphasize the G4s formed by the (GGGGCC)n sequence of the C9orf72 gene, which represents the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). The thorough structural analysis in this review advances our comprehension of the disease mechanism and provides valuable insights into developing targeted therapeutic strategies in ALS/FTD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4afb7326b8980cb609e9c6033f3c9e1b8d005a43" target='_blank'>
              G-Quadruplex Structures Formed by Human Telomere and C9orf72 GGGGCC Repeats
              </a>
            </td>
          <td>
            Bing Yan, M. C. Suen, Naining Xu, Chao Lu, Changdong Liu, Guang Zhu
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background The HIV‐1 reservoir in CD4+ T cells (HRCD4) pose a major challenge to curing HIV, with many of its mechanisms still unclear. HIV‐1 DNA integration and immune responses may alter the host's epigenetic landscape, potentially silencing HIV‐1 replication. Methods This study used bisulphite capture DNA methylation sequencing in CD4+ T cells from the blood of 427 virally suppressed women with HIV to identify differentially methylated sites and regions associated with HRCD4. Results The average total HRCD4 size was 1409 copies per million cells, with most proviruses defective and only a small proportion intact. The study identified 245 differentially methylated CpG sites and 85 regions linked to HRCD4 size, with 52% of significant sites in intronic regions. Genes associated with HRCD4 were involved in viral replication, HIV‐1 latency and cell growth and apoptosis. HRCD4 size was inversely related to DNA methylation of interferon signalling genes and positively associated with methylation at known HIV‐1 integration sites. HRCD4‐associated genes were enriched on the pathways related to immune defence, transcription repression and host–virus interactions. Conclusions These findings suggest that HIV‐1 reservoir is linked to aberrant DNA methylation in CD4+ T cells, offering new insights into epigenetic mechanisms of HIV‐1 latency and potential molecular targets for eradication strategies. Key points Study involved 427 women with HIV. Identified 245 aberrant DNA methylation sites and 85 methylation regions in CD4+ T cells linked to the HIV‐1 reservoir. Highlighted genes are involved in viral replication, immune defence, and host genome integration. Findings suggest potential molecular targets for eradication strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae7ef3531f4c3263677a74b3760489606fbd9df" target='_blank'>
              Aberrant DNA methylation of genes regulating CD4+ T cell HIV‐1 reservoir in women with HIV
              </a>
            </td>
          <td>
            Ke Xu, Xinyu Zhang, Kesava Asam, B. C. Quach, Grier P Page, Deborah Konkle-Parker, Claudia Martinez, Cecile D Lahiri, Elizabeth F Topper, Mardge H. Cohen, S. Kassaye, Jack Dehovitz, M. Kuniholm, N. Archin, Amir Valizadeh, Phyllis C Tien, Vincent C Marconi, Dana B. Hancock, E. O. Johnson, Bradley E Aouizerat
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35698f05d68e24632b02180e50c1bfb7426f2b2" target='_blank'>
              The establishment of the 3D genome structure during zygotic genome activation.
              </a>
            </td>
          <td>
            Anastasiia L Sivkina, O. Iarovaia, S. V. Razin, S. Ulianov
          </td>
          <td>2025-03-03</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Human Immunodeficiency Virus (HIV) is one of the most prevalent viruses, causing significant immune depletion in affected individuals. Current treatments can control HIV and prolong patients’ lives, but new challenges have emerged. Increasing incidence of cancers occur in HIV patients. Esophageal squamous cell carcinoma (ESCC) is one of the most common cancers observed in HIV patients. However, the spatial cellular characteristics of HIV-related ESCC have not been explored, and the differences between HIV-ESCC and typical ESCC remain unclear. Methods We performed spatial transcriptome sequencing on HIV-ESCC samples to depict the microenvironment and employed cell communication analysis and multiplex immunofluorescence to investigate the molecular mechanism in HIV-ESCC. Results We found that HIV-ESCC exhibited a unique cellular composition, with fibroblasts and epithelial cells intermixed throughout the tumor tissue, lacking obvious spatial separation, while other cell types were sparse. Besides, HIV-ESCC exhibited an immune desert phenotype, characterized by a low degree of immune cell infiltration, with only a few SPP1+ macrophages showing immune resistance functions. Cell communication analysis and multiplex immunofluorescence staining revealed that tumor fibroblasts in HIV-ESCC interact with CD44+ epithelial cells via COL1A2, promoting the expression of PIK3R1 in epithelial cells. This interaction activates the PI3K-AKT signaling pathway, which contributes to the progression of HIV-ESCC. Conclusions Our findings depict the spatial microenvironment of HIV-ESCC and elucidate a molecular mechanism in the progression of HIV-ESCC. This will provide us insights into the molecular basis of HIV-ESCC and potential treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02248-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiaojun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26334a481a2a25fb7be72ec2c09273c472c1ce03" target='_blank'>
              Intrinsic immunity against HAdV is achieved by a novel epigenetic silencing complex
              </a>
            </td>
          <td>
            Julia Mai, Maryam Karimi, A. Kuderna, Anna Heider, Thomas Günther, C. Friedel, A. Grundhoff, Thomas Stamminger, Sabrina Schreiner
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: The cell cycle, a critical and intricate biological process, comprises various phases, and its dysregulation plays a pivotal role in tumorigenesis and metastasis. The exploration of cell cycle-based molecular subtypes across pan-cancers, along with the application of synthetic lethality concepts, holds promise for advancing cancer therapies. Methods: A pan-cancer analysis was conducted to assess the cell cycle serves as a reliable signature for classifying molecular subtypes and to understand the potential clinical application of genes as potential drug targets based on synthetic lethality. Results: Molecular subtypes derived from cell cycle features in certain cancers, particularly kidney-related malignancies, exhibited distinct immune characteristics. Synthetic lethal interactions within the cell cycle pathway were common, with significant genetic interactions further identifying potential drug targets through the exploitation of genetic relationships with key driver genes. Additionally, miRNAs and lncRNAs may influence the cell cycle through miRNA:mRNA interactions and ceRNA networks, thereby enriching the genetic interaction landscape. Conclusions: These findings suggest that the cell cycle pathway could serve as a promising molecular subtype signature to enhance cancer prognostication and offer potential targets for anticancer drug development through synthetic lethality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f0509708b74787ef228103fd17aa6cf4b5bae5" target='_blank'>
              Cell Cycle-Based Molecular Features via Synthetic Lethality and Non-Coding RNA Interactions in Cancer
              </a>
            </td>
          <td>
            Shizheng Xiong, Jiaming Jin, Xinmiao Zhao, Yang Zhao, Zhiheng He, Haochuan Guo, Chengjun Gong, Jiafeng Yu, Li Guo, Tingming Liang
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the fourth leading cause of cancer-related death worldwide, with approximately 800,000 deaths worldwide each year. Owing to the atypical early symptoms and characteristics of HCC, over 80% of HCC patients cannot receive curative treatment. The treatment of HCC is facing a bottleneck, and new treatment methods are urgently needed. Since the pathogenesis of HCC is not yet clear, identifying the molecular mechanisms and therapeutic targets related to it is crucial. Centromeres are considered special deoxyribonucleic acid (DNA) sequences with highly repetitive sequences that are physically connected to the spindle during cell division, ensuring equal division of genetic material between daughter cells. The numerous proteins that aggregate on this sequence during cell division are called centromere proteins (CENPs). Currently, numerous studies have shown that CENPs are abnormally expressed in tumor cells and are associated with patient prognosis. The abnormal expression of CENPs is a key cause of chromosomal instability. Furthermore, chromosomal instability is a common characteristic of the majority of tumors. Chromosomal instability can lead to uncontrolled and sustained division and proliferation of malignant tumors. Therapeutic plans targeting CENPs play important roles in the treatment of HCC. For example, small ribonucleic acid (RNA) can silence CENP expression and prevent the occurrence and development of liver cancer. In recent years, studies of HCC-targeting CENPs have gradually increased but are still relatively novel, requiring further systematic elaboration. In this review, we provide a detailed introduction to the characteristics of CENPs and discuss their roles in HCC. In addition, we discuss their application prospects in future clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f704aa07c09b6c48c3c1f8d354bbb48ab6359478" target='_blank'>
              Abstract PR003: DNA methylation reveals metastatic colonization routes in ovarian cancer
              </a>
            </td>
          <td>
            A. Lahtinen, G. Marchi, K. Lavikka, D. Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, T. Muranen, J. Oikkonen, Antti Häkkinen, A. Virtanen, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="eccDNA is a driver of many cancers and a potential intermediate in other age-related disorders. However, little is known about the mechanisms underlying eccDNA formation in healthy tissue and how aging affects these processes. Here, we present an atlas of eccDNA across seven tissues of male mice spanning four ages. EccDNA correlates with open chromatin characterized by signatures of H3K27ac and H3K4me1. Additionally, the mutational load of eccDNA on genes correlates with tissue-specific transcription and increases logarithmically as a function of transcript level. Still, a population of intron-dense genes with many splice forms remains sheltered from eccDNA formation. We also find that the total number of eccDNA molecules does not increase as mice age, unlike other types of mutations. Our data reveal a link between eccDNA formation and transcript level that may drive gene architecture in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3340bae1e6fb34018ca87e367952455af8907d" target='_blank'>
              The CRL7FBXW8 Complex Controls the Mammary Stem Cell Compartment Through Regulation of NUMB Levels
              </a>
            </td>
          <td>
            Simone Sabbioni, M. G. Filippone, Letizia Amadori, S. Confalonieri, R. Bonfanti, Stefano Capoano, I. Colaluca, S. Freddi, G. Bertalot, Giovanni Fagá, Elisa Zagarrí, Mario Varasi, R. H. Gunby, C. Mercurio, S. Pece, P. P. Di Fiore, D. Tosoni
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Recent research has shifted the focus from the genetic code of DNA to its structural variations, which significantly impact cancer, genetic diseases, and gene regulation. Structural changes, such as the transition from B-DNA to A-DNA, influence DNA stability and flexibility and are affected by factors like DNA methylation and sugar puckering. This study is the first to investigate the relationship between DNA conformational changes and lifespan in two rodent species. The analysis focused on long-lived Nannospalax xanthodon and shorter-lived Rattus rattus, utilizing infrared spectroscopy and principal component analysis (PCA) to examine liver DNA. Results indicated that transition from B-form to A- and Z-forms were more prevalent in N. xanthodon than in R. rattus. However, the dominant DNA conformations in both species are in B-form. Additionally, N-type sugar puckers (C3-endo conformation), associated with these DNA forms, were more prominent in N. xanthodon. In contrast, S-type sugar puckers (C2-endo conformation), characteristic of B-DNA, were found at lower levels in N. xanthodon. Furthermore, the variations in methylation-specific structural modifications of nucleobases were quantitatively assessed among these species. The study proposes a significant connection between the long lifespan of N. xanthodon, which live underground, and their unique DNA structure, offering insights into how different DNA forms, as well as the conformations of their backbone and sugar-base components, may affect longevity, highlighting potential research avenues regarding the biomolecular aspects of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e194ada8d97ff0f6ac7375a7dc3c14a580e955" target='_blank'>
              Structural features of DNA and their potential contribution to blind mole rat (Nannospalax xanthodon) longevity
              </a>
            </td>
          <td>
            Gamzenur Sönmez, Tuba Yağcı Gurbanov
          </td>
          <td>2025-03-25</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9782e1bf6046bb6e08da1b33f9dcb44ab84e474" target='_blank'>
              A pivotal contribution of proteostasis failure and mitochondrial dysfunction to chromosomal instability-induced microcephaly
              </a>
            </td>
          <td>
            Amanda González-Blanco, Adrián Acuña-Higaki, David Boettger, Marco Milán
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acfe5ecebe48c685d0ade29fb13cedf2e576caf" target='_blank'>
              Frequency and impact of a genome-wide homologous recombination deficiency signature (HRDsig+) on the genomic landscape of clinically advanced urothelial bladder carcinoma (CAUBC).
              </a>
            </td>
          <td>
            Michael F. Basin, Andrea Necchi, Roger Li, Philippe E. Spiess, A. Kamat, P. Grivas, Shilpa Gupta, Neeraj Agarwal, Liang Cheng, D. Pavlick, E. Sokol, Ryon P Graf, Douglas I. Lin, J. Ross, A. Schrock, J. Quintanilha, Gerald Li, J. Jacob, A. Basnet, G. Bratslavsky
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aca8c4eb6696cc9c11ef54c9581b2900ee7f04" target='_blank'>
              Active telomere elongation by a subclass of cancer-associated POT1 mutations.
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The RAD51-BRCA2 interaction is central to DNA repair through homologous recombination. Emerging evidence indicates RAD51 overexpression and its correlation with chemoresistance in various cancers, suggesting RAD51-BRCA2 inhibition as a compelling avenue for intervention. We previously showed that combining olaparib (a PARP inhibitor (PARPi)) with RS-35d (a BRCA2-RAD51 inhibitor) was efficient in killing pancreatic ductal adenocarcinoma (PDAC) cells. However, RS-35d impaired cell viability even when administered alone, suggesting potential off-target effects. Here, through multiple, integrated orthogonal biological approaches in different 2D and 3D PDAC cultures, we characterised RS-35d enantiomers, in terms of mode of action and single contributions. By differentially inhibiting both RAD51-BRCA2 interaction and sensor kinases ATM, ATR and DNA-PK, RS-35d enantiomers exhibit a ‘within-pathway synthetic lethality’ profile. To the best of our knowledge, this is the first reported proof-of-concept single small molecule capable of demonstrating this built-in synergism. In addition, RS-35d effect on BRCA2-mutated, olaparib-resistant PDAC cells suggests that this compound may be effective as an anticancer agent possibly capable of overcoming PARPi resistance. Our results demonstrate the potential of synthetic lethality, with its diversified applications, to propose new and concrete opportunities to effectively kill cancer cells while limiting side effects and potentially overcoming emerging drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d347103c381bd6fd50aa622f9e2cee5595e830" target='_blank'>
              Investigating synthetic lethality and PARP inhibitor resistance in pancreatic cancer through enantiomer differential activity
              </a>
            </td>
          <td>
            Mirco Masi, Laura Poppi, Viola Previtali, Shannon R. Nelson, Kieran Wynne, Giulia Varignani, Federico Falchi, Marina Veronesi, E. Albanesi, Daniele Tedesco, Francesca De Franco, Andrea Ciamarone, Samuel H. Myers, José Antonio Ortega, Greta Bagnolini, Giovanni Ferrandi, Fulvia Farabegoli, Nicola Tirelli, Giuseppina Di Stefano, G. Oliviero, Naomi Walsh, M. Roberti, S. Girotto, Andrea Cavalli
          </td>
          <td>2025-03-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="High mobility group A1 (HMGA1) is a class of non-histone chromosomal protein and is highly expressed in the embryonic period and many tumors. It is involved in multiple hallmarks of tumors and affects the occurrence and progression of tumors. Nowadays, many non-coding RNAs (ncRNAs), such as miRNA, lncRNA, and circRNA, have been found to play a crucial role in HMGA1 regulation. Moreover, some ncRNAs are reported to be the downstream effectors of HMGA1. They interact with each other to affect multiple hallmarks of cancer and targeting ncRNAs or HMGA1 may provide potential strategies for cancer therapy. In this review, we give an overview of recent studies describing the crosstalk between oncogenic HMGA1 and ncRNAs. We also point out the impact of this interaction on the biological behavior of tumors and their potential in tumor therapy in order to offer potential implications and directions for both basic science and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03713-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f65c0d127245d474792c410ee7afae8aec904" target='_blank'>
              Complicated crosstalk between HMGA and non-coding RNAs modulates hallmarks of cancer
              </a>
            </td>
          <td>
            Lijie Zhang, Xiaomin Zhao, Xianghong Gao, Hao Qin, Feng Chen, Zhijuan Lin
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosomes are spatially organized and functionally folded into a specific macro-structure in the nucleus. Recently, we and others created haploid cells with chromosome fusions. However, there is still lack of an effective strategy for precisely investigating how the genome copes with fusions. Here, we developed a down-sampling method to convert the populational Hi-C dataset into single cell-like Khimaira Matrix (K-matrix). K-matrix preserves not only the most prominent functional genomic features but also cell-to-cell variations. K-matrix-originated genome 3D models display spatial approach of fused chromosomes and minor global structure alterations. Combined with a layered positional decomposition analysis, our models indicate slight re-adjustment of chromosome distributions accordingly with an increasing tendency following more fusions involved. Nevertheless, the radial distribution of the A/B compartment is not affected dramatically. By contrast, natural populations harboring Rb fusions display significant alterations of chromosome radial location. Overall, K-matrix-originated models enable visualization of chromosomal reorganization with high resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Inflammation is a multifaceted defense mechanism of the immune system against infection. Chronic inflammation is intricately linked to all stages of tumorigenesis and is therefore associated with an elevated risk of developing serious cancers. Epigenetic mechanisms have the capacity to trigger inflammation as well as facilitate tumor development and transformation within an inflammatory context. They achieve this by dynamically modulating the expression of both pro—inflammatory and anti—inflammatory cytokines, which in turn sustains chronic inflammation. The aberrant epigenetic landscape reconfigures the transcriptional programs of inflammatory and oncogenic genes. This reconfiguration is pivotal in dictating the biological functions of both tumor cells and immune cells. Aberrant histone H3 lysine 27 site (H3K27) methylation has been shown to be involved in biological behaviors such as inflammation development, tumor progression, and immune response. The establishment and maintenance of this repressive epigenetic mark is dependent on the involvement of the responsible histone modifying enzymes enhancer of zeste homologue 2 (EZH2), jumonji domain containing 3 (JMJD3) and ubiquitously transcribed tetratricopeptide repeat gene X (UTX) as well as multiple cofactors. In addition, specific pharmacological agents have been shown to modulate H3K27 methylation levels, thereby modulating inflammation and carcinogenesis. This review comprehensively summarises the current characteristics and clinical significance of epigenetic regulation of H3K27 methylation in the context of inflammatory response and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a333da9e2131a9a75deaa130f4cc548713bf46a8" target='_blank'>
              Regulation of histone H3K27 methylation in inflammation and cancer
              </a>
            </td>
          <td>
            Jing Ma, Yalin Zhang, Jingyuan Li, Y. Dang, Dan Hu
          </td>
          <td>2025-03-05</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5836cab694b09de73f48a9a6f215a5a097f78290" target='_blank'>
              Abstract A010: Dnmt3a2 ensures proper enhancer DNA methylation during embryonic development
              </a>
            </td>
          <td>
            Minmin Liu, Guillermo A Urrutia, Rachel Shereda, Gangning Liang, Peter A. Jones
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tandem repeats play an important role in centromere structure, subtelomeric regions, DNA methylation, recombination and the regulation of gene activity. Analysis of their distribution in genomes offers a potential means for predicting putative centromere locations, which continues to be a challenge for genome annotation. Here we present RepeatOBserver (https://github.com/celphin/RepeatOBserverV1), a new tool for visualising repeat patterns and identifying putative centromere locations, using a Fourier transform of DNA walks. RepeatOBserver can identify and visualise a broad range of perfect and imperfect repeats (3-5000 bp long) in genome assemblies without any a priori knowledge of repeat sequences or the need for optimising parameters. RepeatOBserver heatmaps can distinguish between tandem and retrotransposon repeats. We analysed 159 chromosomes with experimentally-verified centromere positions from 12 plant and animal species. We find that 93% of experimentally-verified tandem repeat centromeres occur in regions of low sequence diversity and 97% of retrotransposon centromeres occur in regions with a high abundance of repeat lengths. Depending on the centromere type predicted by the heatmaps, putative centromere locations can be predicted using either a genomic Shannon diversity index or a repeat abundance sum. RepeatOBserver can also locate other regions of interest including potential neocentromeres and gene copy variation. Split and inverted tandem repeats at inversion boundaries suggest that chromosomal inversions or mis-assemblies can also be located. RepeatOBserver is a flexible tool for comprehensive characterisation of repeat patterns that can be used to visualise and identify a variety of regions of interest in genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b7a0c655a4ba01e7821226bff9c91a6b3f8818" target='_blank'>
              RepeatOBserver: Tandem Repeat Visualisation and Putative Centromere Detection.
              </a>
            </td>
          <td>
            Cassandra Elphinstone, Rob Elphinstone, Marco Todesco, Loren H. Rieseberg
          </td>
          <td>2025-03-04</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324fb87b02d8939636995cfccebb627de6d9cb0d" target='_blank'>
              Abstract B002: DNA methylation protects cancer cells against senescence
              </a>
            </td>
          <td>
            Xiaoying Chen, Kosuke Yamaguchi, Brianna Rodgers, D. Goehrig, D. Vindrieux, Xavier Lahaye, Matthieu Nolot, Laure Ferry, Nadine Martin, Pierre Dubus, F. Miura, Takashi Ito, Nicolas Manel, Masato Kanemaki, David Bernard, P. Defossez
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Noncoding microRNAs tend to evolve within introns of coding genes that provide them with transcriptional opportunity. As an outcome of natural selection, the intragenic position of microRNAs is crucial for their expression, evolution, and functional cooperation with the host gene. Therefore, understanding the evolution of intragenic microRNA structures may bring novel insights into genetic and phenotypic evolution. However, it remains largely unexplored. Here, by analyzing microRNA genomics in 34 metazoan species, we found that the majority (630/1,154) of microRNA families originated from introns of coding genes that provided them with initial transcriptional capacity. The most rapid expansion of intragenic microRNAs happened at the advent of vertebrates when 21 microRNA families emerged from introns of neural genes and reorganized the gene regulatory network, leading to the rise of vertebrate-specific neural crest cells, which transformed the invertebrate head and enabled the ecological shift from filter feeding to active predation. Intragenic microRNAs gradually gain independence from their host genes, which is accelerated by whole-genome duplications. After a whole-genome duplication, the purging of redundant host genes often set an orphaned microRNA “free” to diversify with the transcriptional elements inherited from the host. Whole-genome duplications facilitated a dramatic microRNA diversification during the initial divergence of vertebrates, as the intragenic status of 12 neural crest-regulating microRNAs was retained in jawed vertebrates but was lost in jawless cyclostomes, which diverged their neural crest development. We propose that coding genes not only facilitate the origination of new microRNAs, but also “sacrifice” themselves to help microRNA diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec14a5673e5811f8ba88226799b41e9414365cb1" target='_blank'>
              Coding Genes Helped the Origination and Diversification of Intragenic MicroRNAs
              </a>
            </td>
          <td>
            Chengzhang Liu, Xiaojun Zhang, Jianbo Yuan, J. Xiang, Fuhua Li
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53dcae9646767b55b828bf57d0d3c3835c11bb8d" target='_blank'>
              Abstract B035: Evaluating drug sensitivity heterogeneity in cancer organoids derived from circulating tumor cells
              </a>
            </td>
          <td>
            Y. Han, David Hsieh, Shian-Jiun Shih
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7749502012ae50efd549b8d9691628c7f5bce04c" target='_blank'>
              Epigenetics of Human Telomeres.
              </a>
            </td>
          <td>
            Nicole Bettin, Mélina Vaurs, Anabelle Decottignies
          </td>
          <td>2025-03-17</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd9f59a39a075b237d1c5fa6c39932e43dffde05" target='_blank'>
              Long term latency of highly mutated cells in normal mouse skin is reversed by exposure to tumor promoters and chronic tissue damage.
              </a>
            </td>
          <td>
            Y. R. Li, E. Kandyba, Kyle Halliwill, R. Delrosario, Quan Tran, Nora Bayani, Di Wu, Olga K. Mirzoeva, Melissa Q Reeves, S. M. A. Islam, Laura Riva, E. Bergstrom, Kavya Achanta, John DiGiovanni, Ludmil B. Alexandrov, Allan Balmain
          </td>
          <td>2025-03-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/590a8f98137453dc965f4fb64612753d1428ec38" target='_blank'>
              The functions and modifications of tRNA-derived small RNAs in cancer biology.
              </a>
            </td>
          <td>
            Abdulaziz Ahmed A Saad, Kun Zhang, Qianqian Deng, Jiawang Zhou, Lichen Ge, Hongsheng Wang
          </td>
          <td>2025-03-01</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb646ade7879700ef0b4267a1e8d7990cd0524fa" target='_blank'>
              Abstract IA018: The evolutionary origins of cancer lethality
              </a>
            </td>
          <td>
            P. Boutros
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression relies on the ability of cells to adapt to challenging environments overcoming stresses and growth constraints. Such adaptation is a multifactorial process that depends on the rapid reorganization of many basic cellular mechanisms. Protein synthesis is often dysregulated in cancer, and translational reprogramming is emerging as a driving force of cancer adaptive plasticity. Long non-coding RNAs (lncRNAs) represent the main product of genome transcription. They outnumber mRNAs by an order of magnitude and their expression is regulated in an extremely specific manner depending on context, space and time. This heterogeneity is functional and allows lncRNAs to act as context-specific, fine-tuning controllers of gene expression. Multiple recent evidence underlines how, besides their consolidated role in transcription, lncRNAs are major players in translation control. Their capacity to establish multiple and highly dynamic interactions with proteins and other transcripts makes these molecules able to play a central role across all phases of protein synthesis. Even if through a myriad of different mechanisms, the action of these transcripts is dual. On one hand, by modulating the overall translation speed, lncRNAs participate in the process of metabolic adaptation of cancer cells under stress conditions. On the other hand, by prioritizing the synthesis of specific transcripts they help cancer cells to maintain high levels of essential oncogenes. In this review, we aim to discuss the most relevant evidence regarding the involvement of lncRNAs in translation regulation and to discuss how this specific function may affect cancer plasticity and resistance to stress. We also expect to provide one of the first collective perspectives on the way these transcripts modulate gene expression beyond transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, M. K. Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc280a650e26ad4fbfbcabfa8901f734f1fa074d" target='_blank'>
              Chromosomal localization of PHOX2B during M-phase is disrupted in disease-associated mutants.
              </a>
            </td>
          <td>
            Yuki Sato, Shin-ichiro Hayashi, Souichi Oe, T. Koike, Y. Nakano, Ryohei Seki-Omura, Hikaru Iwashita, Y. Hirahara, Masaaki Kitada
          </td>
          <td>2025-02-11</td>
          <td>Development, growth & differentiation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tetraploidisation plays a crucial role in evolution, development, stress adaptation, and disease, but its beneficial or pathological effects in different tissues remain unclear. This study aims to compare physiological and unphysiological tetraploidy in eight steps: 1) mechanisms of diploidy-to-tetraploidy transition, 2) induction and elimination of unphysiological tetraploidy, 3) tetraploid cell characteristics, 4) stress-induced unphysiological tetraploidy, 5) comparison of physiological vs. unphysiological tetraploidy, 6) consequences of unphysiological stress-induced tetraploidy, 7) nutritional or pharmacological prevention strategies of tetraploidisation, and 8) knowledge gaps and future perspectives. Unphysiological tetraploidy is an adaptive stress response at a given threshold, often involving mitotic slippage. If tetraploid cells evade elimination through apoptosis or immune surveillance, they may re-enter the cell cycle, causing genetic instability, micronuclei formation, aneuploidy, modification of the epigenome and the development of diseases. The potential contributions of unphysiological tetraploidy to neurodegenerative, cardiovascular and diabetes related diseases are summarized in schematic figures and contrasted with its role in cancer development. The mechanisms responsible for the transition from physiological to unphysiological tetraploidy and the tolerance to tetraploidisation in unphysiological tetraploidy are not fully understood. Understanding these mechanisms is of critical importance to allow the development of targeted nutritional and pharmacological prevention strategies and therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb4b60e7683b10ceeabf0dcb4384ffb5dd6b80ee" target='_blank'>
              Tetraploidy in normal tissues and diseases: mechanisms and consequences
              </a>
            </td>
          <td>
            Micheline Kirsch-Volders, Miroslav Mišík, Michael Fenech
          </td>
          <td>2025-03-21</td>
          <td>Chromosoma</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60dc48b94558a55ad5ff2dbcdf85170057c59b44" target='_blank'>
              Natural resistance to cancers in long-lived mammals: genomic mechanisms and experimental evidence to explain Peto's paradox.
              </a>
            </td>
          <td>
            Linxia Sun, Zhikang Xu, Mengqi Shuai, Chengxu Li, Guang Yang, Shixia Xu
          </td>
          <td>2025-03-21</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic defects caused by truncating mutations or deletions in the Retinoblastoma tumor suppressor gene (RB1) are frequently observed in many cancer types leading to dysregulation of the RB pathway. Here, we propose an integrative proteogenomic approach that predicts cancers with dysregulation in the RB pathway. A subset of these cancers, which we term as “RBness,” lack RB1 genomic defects and yet phenocopy the transcriptional profile of RB1-defective cancers. We report RBness as a pan-cancer phenomenon, associated with patient outcome and chemotherapy response in multiple cancer types, and predictive of CDK4/6 inhibitor response in estrogen-positive breast cancer. Using RNA interference and a CRISPR-Cas9 screen in isogenic models, we find that RBness cancers also phenocopy synthetic lethal vulnerabilities of cells with RB1 genomic defects. In summary, our findings suggest that dysregulation of the RB pathway in cancers lacking RB1 genomic defects provides a molecular rationale for how these cancers could be treated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b769a127908e214629b60c835bc205b258e11d" target='_blank'>
              Proteogenomic discovery of RB1-defective phenocopy in cancer predicts disease outcome, response to treatment, and therapeutic targets
              </a>
            </td>
          <td>
            Jacopo Iacovacci, R. Brough, Fatemeh Ahmadi Moughari, J. Alexander, H. Kemp, Andrew N J Tutt, R. Natrajan, C. J. Lord, Syed Haider
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7430b5ac3f8cbb1ebc495236523dfec786758d7" target='_blank'>
              Active Hydrodynamic Theory of Euchromatin and Heterochromatin
              </a>
            </td>
          <td>
            S. Rautu, Alexandra Zidovska, D. Saintillan, Michael J. Shelley
          </td>
          <td>2025-03-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) are sequences that do not encode for proteins and play key roles in different cellular processes, including cell proliferation and differentiation. On the other hand, connexins (Cxs) are transmembrane proteins that principally allow intercellular communication. In pathological conditions such as cancer, there is a deregulation in the expression and/or function of ncRNAs and Cxs, which in turn leads to an enhancement in the aggressive phenotype, such as a greater proliferative and invasive capacity. This suggests a plausible interplay between ncRNAs and Cxs. Based on that, this review aims to summarize the current knowledge regarding this relationship and to analyze how it may influence the development of aggressive traits in cancer cells and the clinicopathological features of cancer patients. Finally, we discuss the potential of ncRNAs and Cxs as promising clinical biomarkers for cancer diagnosis, prognosis, and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738fef2061265ab78fa2e6e3568a43429fcdb019" target='_blank'>
              Molecular Interplay Between Non-Coding RNAs and Connexins and Its Possible Role in Cancer
              </a>
            </td>
          <td>
            Pablo Pérez-Moreno, Juan P Muñoz, Mauricio A Retamal
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R loops are three-stranded nucleic acid structures that form naturally in cells under various conditions, mainly as intermediates during replication or as by-products during transcription. R loops are involved in the regulation of many important cellular processes, including replication, transcription, centromere stabilization, protection of chromosome ends, or control of telomere length. Unscheduled R loops are linked to many diseases, including cancer, neurodegenerative, or inflammatory disorders. The list of cancer diseases linked to excessive R loop accumulation is growing rapidly. There is currently much debate about the understanding of abnormal R loop formation and its impact on genome instability and cancer development. In this review, we briefly describe the nature of R loops, their formation under physiological and pathological conditions, and the proteins involved in the regulation of R loops. In addition, we emphasize the possible role of the human ribonuclease Dicer, a multi-tasking protein mostly known for its important role in microRNA biogenesis, in the regulation of R loops. We also discuss the involvement of R loops in cancer development and their potential use as diagnostic biomarkers. Knowledge of the molecular mechanisms underlying R loop dysregulation may significantly improve our understanding of cancer biology and provide new directions for research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b3552d408d3cbf3b7a331a1abe18a81efb7900" target='_blank'>
              Possible role of human ribonuclease dicer in the regulation of R loops.
              </a>
            </td>
          <td>
            Klaudia Wójcik, Paulina Krzeminska, A. Kurzyńska-Kokorniak
          </td>
          <td>2025-03-24</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Extracellular vesicles (EVs) play a key role in cell‐to‐cell communication and have drawn significant attention due to their potential clinical applications. However, much remains to be understood about the biology of EV‐associated DNA (EV‐DNA). EV‐DNA is actively released by both normal and malignant cells and consists of diverse fragments with varying structures. Because EV‐DNA spans the entire genome of cells from which it originates, it continues to be attractive as a biomarker for cancer diagnosis and monitoring. Further, EV‐DNA delivery can alter the function of recipient cells by interfering with cytoplasmic DNA sensor pathways. This review explores the biology and significance of EV‐DNA, including its topology and fragmentomics features, modality of association with EVs, packaging mechanisms, and potential functions. It also emphasizes the specificity of vesicular DNA in identifying genetic and epigenetic changes in cancer. Additionally, it delves into the impact of EV‐DNA on cellular behaviour and its potential use as a therapeutic target in cancer. The review discusses new insights into EV‐DNA biology and provides perspectives and alternatives to address the challenges and concerns for future EV‐DNA studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80fa234e6a41ee735b1a29ab26a8f1f47cfb63ad" target='_blank'>
              Unravelling the Significance of Extracellular Vesicle‐Associated DNA in Cancer Biology and Its Potential Clinical Applications
              </a>
            </td>
          <td>
            J. Ghanam, Kristína Lichá, V. K. Chetty, Ommolbanin Asad Pour, Dirk Reinhardt, Barbora Tamášová, Peter Hoyer, Jan Lötvall, B. K. Thakur
          </td>
          <td>2025-03-01</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b941893a0145be37f05ce3f74abd52d43b06855" target='_blank'>
              Recent Progress of Triplex DNA Formation and Its Applications.
              </a>
            </td>
          <td>
            Lei Wang, Dongliang Ji, Xiao Liu, Wenzhuo Lei, Yosuke Taniguchi, Yong Ling
          </td>
          <td>2025-02-28</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Ewing sarcoma, a pediatric cancer of bone and soft tissue, is driven in most cases by an abnormal oncogenic fusion of the N-terminal region of EWS with the C-terminal region of FLI1 (EWS–FLI1). The FLI1 region contains a conserved DNA-binding domain (DBD) essential for the oncogenesis. Binding of EWS–FLI1 to microsatellites composed of contiguous GGAA sites, shown previously to be critical for the oncogenic program of this fusion, is not well understood. In this study, we demonstrate that the FLI1 DBD binds cooperatively to contiguous GGAA sites, thereby forming a nucleoprotein filament. A series of crystal structures of two, three, and four FLI1 DBD proteins in complexes with DNA oligomers containing two, three, and four contiguous GGAA sites, respectively, reveal the structure of this filament and the basis for its cooperative formation. We expect this mechanistic insight to be an important milestone in our understanding of the oncogenic function of EWS-FLI1 and exploiting it as a drug target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8965ffde0bcb8751385ea02c52b2a017ca74ff" target='_blank'>
              Structure and cooperative formation of a FLI1 filament on contiguous GGAA DNA sites
              </a>
            </td>
          <td>
            Caixia Hou, O. Tsodikov
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>