<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-04 09:40:25 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>77</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>115</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>345</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>303</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b9576b9fdef51f927edfe8dbc69e8e7f1f4c7f" target='_blank'>
              Extrachromosomal DNA associates with poor survival across a broad spectrum of childhood solid tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Yui-Ching Chow, Rishaan Kenkre, Jonathan Kirkland, Aditi Dutta, Shanqing Wang, Wenshu Zhang, Miguel Brown, J. Luebeck, Yan Yuen Lo, Elias Rodriguez-Fos, A. Henssen, K. Okonechnikov, David R. Ghasemi, K. Pajtler, Daisuke Kawauchi, V. Bafna, Megan Paul, Kevin Yip, Jill P. Mesirov, Lukas Chavez
          </td>
          <td>2025-07-23</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865d140b6544d7dfc0dca54fb01a54fdf212f1a" target='_blank'>
              Mitotic BLM functions are required to maintain genomic stability
              </a>
            </td>
          <td>
            Tamara Eleanore Hamann, A. Wieland, Andrea Tirincsi, Kruno Vukušić, Farbod Mohseni, René Wardenaar, Marialucrezia Losito, Ipek Ilgin Gönenc, B. Wollnik, F. Foijer, I. Tolic, Zuzana Storchová, M. Räschle
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) of chromosomal origin is commonly present in all eukaryotic organisms and tissue tested so far. EccDNA populations exhibit immense diversity and a characteristically low degree of overlap between samples, suggesting low inherence of eccDNA between cells or a deficiency the methods by which eccDNA is detected. This study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap. We optimized the protocol by reducing the time. Linear DNA is digested by increasing exonuclease V activity. We employed CRISPR-Cas9 for mitochondrial linearization, which proved superior to restriction enzymes. A key finding is the critical role of random hexamer primer concentration and genomic DNA input in Rolling Circle Amplification (RCA) for generating high-quality long amplicons from eccDNA (concatemeric tandem copy, CTC), essential for confident de novo eccDNA construction from long-read sequencing data. Lower primer concentrations substantially increased the percentage of CTC-derived eccDNA and improved the overlap of identified eccDNAs in technical replicates. Applying this revisited approach to human myeloma and breast cancer cell lines, as well as xenograft models, demonstrated that the optimized conditions enhanced the overlap of detected eccDNA up to over 50% overlap which substantially improved over previous studies (less than 1%). Moreover, Additionally, the oncogenic signature of eccDNAs can be identified across all replicates. These findings provide guidelines for developing standardized procedures for eccDNA profiling, advancing our understanding of eccDNA biology and its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extremely Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52c3844fc0c5bb373680e6bacbba142f0e508fa" target='_blank'>
              Forecasting oncogene amplification and tumour suppressor deletion
              </a>
            </td>
          <td>
            Bárbara Hernando, Ángel Fernández-Sanromán, Alice Cádiz, Patricia G. Santamaria, David Gómez-Sánchez, Maria Escobar-Rey, Blas Chaves-Urbano, Joseph Thompson, Marina Torres, G. R. de Garibay, Vera Adradas, Eva Álvarez, Maxime Tarabichi, T. Lesluyes, Juan Manuel Coya, Jon Zugazagoitia, Luis Paz-Ares, Geoff Macintyre
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Introduction Balanced chromosomal abnormalities (BCAs) are structural variations that can underlie a wide spectrum of neurodevelopmental disorders, often remaining undetected by conventional diagnostic approaches. Whole-genome sequencing (WGS) allows for base-pair resolution of structural variants across the entire genome, making it a powerful tool to detect cryptic chromosomal rearrangements and refine breakpoint mapping. RNA sequencing (RNA-Seq), by enabling the detection of gene expression changes and fusion transcripts, provides complementary functional insights into the consequences of genomic alterations. This study integrated WGS and RNA-Seq to precisely characterize the breakpoints and assess the functional impact of de novo BCAs in two unsolved cases of Neurodevelopmental Disorders. Materials and methods Short read WGS was used to identify the chromosomal breakpoints and gene disruptions caused by BCAs. RNA-Seq on blood RNA was employed to detect differential gene expression and potential fusion transcripts of disrupted genes. Results In the first case, the inversion inv(8) (p11.2q13) disrupted two genes at the breakpoints, namely, CHD7 and SLC20A2. These genes are in opposite orientations, and the inversion realigned them in the same direction, generating two novel fusion genes. Disruption of CHD7 confirmed the suspected diagnosis of CHARGE syndrome. The interruption of SLC20A2, commonly associated with neurological symptoms, prompted further clinical evaluation. RNA-Seq identified in-frame fusion transcripts from the chimeric genes in the blood, suggesting a potential deleterious phenotypic effect. In the second case, WGS revealed a balanced translocation t(17; 22) (q25; q13) that disrupted EP300 at 22q25, confirming Rubinstein-Taybi syndrome. The concurrent disruption of RBFOX3 at 17q13 suggested additional neurological implications, particularly related to epilepsy. Transcriptomic analysis demonstrated the monoallelic and significantly reduced expression of EP300. Conclusion These findings highlight the crucial role of WGS in identifying disease-associated BCAs and underscore the complementary value of RNA-Seq in assessing their functional consequences. This integrated approach enhanced diagnostic accuracy and clinical management, paving the way for more comprehensive and personalized care in these two patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8f54f85ebf927d3f8b7d5f4a6236e7f717fa51" target='_blank'>
              Integrating genome and transcriptome analysis to decipher balanced structural variants in unsolved cases of neurodevelopmental disorders
              </a>
            </td>
          <td>
            S. Mellone, Alice Spano, Denise Vurchio, Giulia Borgonovi, Alessandro Ugonotti, Giulia Paglino, Alba Bianco, Sara Ronzani, Maurizio Sciancalepore, Flavia Prodam, Amanda Papa, Maurizio Viri, Umberto Dianzani, Mara Giordano
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c19d5f1ca11d81a47f9f7924f6f192beb7cb85" target='_blank'>
              Cancer type-specific association of p53 deficiency with aneuploidy and chromosome losses
              </a>
            </td>
          <td>
            Joana F. Marques, Marco António Dias Louro, Teresa Davoli, G. Kops
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Objectives: To investigate the clinical diagnostic and prognostic value of extrachromosomal circular DNA (eccDNA) in breast cancer, eccDNA profiles were constructed for 81 breast cancer tumor tissues and 33 adjacent non-tumor tissues. Methods: The distribution characteristics of eccDNA across functional genomic elements and repetitive sequences were systematically analyzed. Furthermore, a diagnostic model for differentiating malignant and normal breast tissues, as well as a prognostic prediction model, was developed using a random forest algorithm. Results: EccDNA in breast cancer tissues harbor a higher proportion of functional elements and repetitive sequences, with their annotated genes significantly enriched in tumor- and immune-related pathways. However, no significant differences in eccDNA features were observed across breast cancer subtypes or pathological stages. In the validation cohort, the eccDNA-based diagnostic model achieved an AUC of 0.83, with repetitive elements and enhancer-associated features contributing the most to diagnostic performance. The prognostic model achieved an AUC of 0.78, with repetitive element annotations also showing strong prognostic relevance. Conclusions: These findings highlight the promising potential of eccDNA in the development of precision diagnostics and prognostic systems for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f033fc855abaceeac39bacf8c5fcd1d56a705c9" target='_blank'>
              A Study on the Diagnostic and Prognostic Value of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Fuyu Li, Wenxiang Lu, Lingsong Yao, Yunfei Bai
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA damage from routine cellular processes or exogenous insults can have a lasting impact on gene regulation beyond genetic mutations. The prevailing paradigm for the consequences of DNA damage repair revolves around restoration of the original genetic sequence, but long-term changes in chromatin configuration, gene expression and DNA modifications have not been analyzed. We introduced numerous, simultaneous Cas9-mediated DNA double strand breaks (DSBs) at defined locations in human glioblastoma cells and tracked both non-genetic and genetic alterations over time. Megabase-scale genomic alterations that endured two weeks after the initial damage were detected, involving a shift from transiently increased intra-TAD interactions to persistent long range cis and trans contacts, alterations in gene-expression and associated large structural variations. These findings reveal that widespread DNA damage, such as chemotherapy or radiotherapy, can trigger long-term genetic and non-genetic modifications which alter cellular function and may impact tumor outcome and the emergence of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Ribonucleoside monophosphates (rNMPs) are abundant in DNA, but their distribution and function in human nuclear genomes remain unknown. Here, we mapped nearly one million rNMPs per genome across diverse human cell types, defining a nuclear “ribome” with non-random distribution patterns. rNMPs are enriched in C/G-rich sequences, epigenetically marked regions, and telomeres. Conserved ribonucleotide-enriched zones (REZs) overlap with CpG islands and R-loops. rNMP concentration near transcription start sites (TSSs) correlates positively with gene expression. Wild-type cells display a broader gene-expression range than ribonuclease H2A (RNH2A) knockouts, in which loss of rNMP cleavage causes pronounced retention of embedded rG and strand-biased rC near TSSs, both increasing with gene expression. These findings establish DNA-embedded rNMPs as a novel epigenetic mark that modulates human gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a34d4cbfcfa7f6007c5ecd2489744cc0bcfeecb" target='_blank'>
              Human ribomes reveal DNA-embedded ribonucleotides as a new type of epigenetic mark
              </a>
            </td>
          <td>
            Deepali L. Kundnani, Taehwan Yang, Tejasvi Channagiri, Penghao Xu, Yeunsoo Lee, Mo Sun, Francisco Martinez-Figueroa, Supreet Randhawa, Ashlesha Gogate, Youngkyu Jeon, Stefania Marsili, Gary Newnam, Yilin Lu, Vivian S. Park, Sijia Tao, Justin A Ling, Raymond F. Schinazi, Z. Pursell, Abdulmelik Mohammed, P. Opresko, Bret D. Freudenthal, Baek Kim, Soojin V Yi, Nataša Jonoska, F. Storici
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc5ca4f590540c5787b14eba5758a0b8345da44" target='_blank'>
              TrAEL-seq captures DNA replication dynamics in mammalian cells
              </a>
            </td>
          <td>
            Neesha Kara, Laura Biggins, V. Grinkevich, Alex J. Whale, Paola Garran-Garcia, Jhanavi Srinivasan, P. Rugg-Gunn, Simon Andrews, Aled J. Parry, Helen M R Robinson, J. Houseley
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mammalian genomes are subdivided into euchromatic A compartments that contain mostly active chromatin, and inactive, heterochromatic B compartments. However, it is not well understood how A and B genome compartments are established and maintained. Here we study SMCHD1, an SMC-like protein best known for its role in X chromosome inactivation, in human male myoblasts. SMCHD1 colocalizes with Lamin B1 and the heterochromatin mark H3K9me3. Loss of SMCHD1 leads to extensive heterochromatin and Lamin B1 depletion at the nuclear lamina, acquisition of active chromatin states and increased DNA methylation along chromosomes. In absence of SMCHD1, long range intra-chromosomal contacts between B compartments are lost while many new TADs and loops are formed. Inactivation of SMCHD1 promotes numerous B to A compartment transitions accompanied by activation of silenced genes. The data suggests that SMCHD1 functions as an anchor for heterochromatin domains at the nuclear lamina ensuring that these domains are poorly accessible to DNA methyltransferases and to epigenome modification enzymes that typically operate in active chromatin. Thus, the properties of SMCHD1 in heterochromatin maintenance extend well beyond its role in X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d576a82dc3af4516675e7e1ac94325c2cc7c05" target='_blank'>
              SMCHD1 maintains heterochromatin, genome compartments and epigenome landscape in human myoblasts
              </a>
            </td>
          <td>
            Zhijun Huang, Wei Cui, Ishara Ratnayake, Kristin Gallik, Lorna Cohen, Rabi Tawil, Gerd P Pfeifer
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea18029e3c5c41c740db47ee4da2a815d1304135" target='_blank'>
              5-hydroxymethylcytosine deposition mediates Polycomb Repressive Complex 2 function in MYCN-amplified neuroblastoma
              </a>
            </td>
          <td>
            Mohansrinivas Chennakesavalu, Gabriel L. Lopez, Varsha Gupta, Kelley Moore, Yuqing Xue, Josie Majowka, Sahil Veeravalli, Ryan Borchert, Monica Pomaville, A. Chlenski, Chuan He, Andrea Piunti, M. Applebaum
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Background: CDK4/6i have established roles in treating patients (pts) with high-risk early-stage and metastatic HR+/HER2- breast cancer (HR+ MBC). Understanding the genomic and molecular factors that dictate CDK4/6i response is a critical area of research. Low-level copy number gains in the mitotic spindle regulator AURKA have been implicated in promoting CDK4/6i resistance via whole-exome sequencing of tumor samples, and constitutive overexpression provokes resistance in vitro [Wander et al. Cancer Discov 2020]. Developing a clinical biomarker of AURKA copy number gain poses unique challenges given the generally low-level AURKA gain observed in resistant samples. Validation of a reliable assay to detect AURKA-dependent CDK4/6i resistance in the clinic may enable personalized use of emerging AURKA inhibitors such as alisertib, which is currently being evaluated for this indication (ClinicalTrials.gov identifier NCT06369285). Methods: De-identified clinical and genomic records of pts with HR+ MBC in the Tempus database whose tumors were sequenced using xT (DNA) and xR (RNA) were investigated. Tempus xT is a targeted, tumor–normal matched DNA panel that detects single-nucleotide variants, insertions and/or deletions, and copy number variants in 648 genes. Tempus xR is a whole-exome capture, next-generation sequencing assay that identifies transcriptional evidence of chromosomal rearrangements resulting in expression of fusion RNA species. Copy number analyses were performed with a proprietary algorithm assessing tumor purity, ploidy, and B-allele frequencies to estimate copy numbers and minor allele counts. RNA-based analyses classified low and high expressors based on median gene expression values. Real-world progression-free survival (rwPFS) was the time from start of CDK46/i to first recorded disease progression (determined by the primary treating oncologist), death from any cause, or initiation of a subsequent treatment line. Survival analyses were right-censored at 5 years. Results: 848 pts with HR+ MBC with DNA sequencing prior to CDK4/6i were identified. Common pathogenic genomic alterations included PIK3CA (SNV: 43.7%; CNV: 2.4%) and ESR1 (SNV: 12.2%; CNV: 6%; fusion: 1.8%). 4.1% of biopsy samples harbored RB1 alterations. AURKA copy number gains were identified in 15.0% of pts, with most demonstrating low-level amplification (copy number 4–7: 13.8%; copy number >7: 1.2%). Of note, these low-level amplifications would not routinely be reported by available sequencing platforms in clinical settings. Linear regression modeling showed that AURKA copy number correlated with RNA expression (p-value <0.001). Overall, 713 pts with DNA sequencing prior to CDK4/6i exposure also had rwPFS information. Those whose tumors harbored high- or low-level AURKA amplifications (n=116) had inferior rwPFS on CDK4/6i (9.9 vs 17 months for no AURKA amplification (n=597; p=0.00032). Conclusions: In this large, real-world translational research cohort of pts with HR+ MBC, low-level AURKA copy number gain was common. These data report the first evidence suggesting that low-level amplifications in AURKA, which conventional sequencing platforms miss, can provoke meaningful changes in gene expression as assessed via RNA transcriptome analysis. Further, low-level AURKA amplifications predict inferior outcomes on CDK4/6i for pts with HR+ MBC. Real-world clinical data for pts whose tumors harbored RB1 mutations, which have been associated with alisertib sensitivity in both the preclinical and clinical settings, trended with shorter rwPFS on CDK4/6i in this dataset and will be presented at the meeting. Efforts to refine these and other potential biomarkers that may guide clinical deployment of alisertib are ongoing.
 Citation Format: Seth Wander, Ezgi Karaesmen Rizvi, Roosheel Patel, Erik Knudsen, Georg F. Bischof, Lisa D. Eli, Karthik Giridhar. Low-level Aurora kinase A (AURKA) amplification as a novel personalized biomarker of CDK4/6 inhibitor (CDK4/6i) resistance in patients with hormone-receptor positive (HR+) metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-09-21.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87772935069c579027cec2eb1b7a9bd7956c15d1" target='_blank'>
              Abstract P2-09-21: Low-level Aurora kinase A (AURKA) amplification as a novel personalized biomarker of CDK4/6 inhibitor (CDK4/6i) resistance in patients with hormone-receptor positive (HR+) metastatic breast cancer
              </a>
            </td>
          <td>
            Seth A. Wander, Ezgi Karaesmen Rizvi, Roosheel Patel, Erik S. Knudsen, G. F. Bischof, L. Eli, Karthik V Giridhar
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c01f9fcd4978a181214c5cc7b38fe087c3cef1" target='_blank'>
              ZCCHC4 Promotes Translation of Replication-dependent Histone mRNAs by Recruiting Cytoplasmic eIF3 complex
              </a>
            </td>
          <td>
            Ruiqi Wang, Xiaoyan Shi, Yangyi Zhang, Yuci Wang, Yanlan Cao, Rui Feng, Chen Chen, Yingchun Zhang, Hao Chen, Honghui Ma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription‐replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription‐induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription‐Replication Conflicts: Unlocking New Frontiers in Cancer
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis. Methods We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome. Results Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities. Discussion Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Extrachromosomal-circular DNA (eccDNA) are circular-DNA elements that are implicated in cellular processes such as genomic diversification and instability, along with other genomic mechanisms. However, their structure and chromatinization state is poorly understood. Here we identified a 358-bp circular micro-eccDNA molecule derived from the human Titin gene, which was selected for this study based on its size and sequence characteristics, and used atomistic molecular dynamics simulations to study its interaction with nucleosomes and corresponding topological state. We show the presence or absence of bound nucleosomes provides a topological switch between intact and denatured DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosome instability leading to aneuploidy and accumulation of copy number gains or losses is a hallmark of cancer. Copy number alteration (CNA) signatures are increasingly used for cancer risk stratification, but size thresholds for defining CNAs across cancers are variable and the biological and clinical implications of CNA size heterogeneity and co-occurrence are incompletely understood. Here we analyze CNA and clinical data from 691 meningiomas and 10,383 tumors from The Cancer Genome Atlas to develop cancer- and chromosome-specific size-dependent CNA and CNA co-occurrence models to predict tumor control and overall survival. Our results shed light on technical considerations for biomarker development and reveal prognostic CNAs with optimized size thresholds and co-occurrence patterns that refine risk stratification across a diversity of cancer types. These data suggest that consideration of CNA size, focality, number, and co-occurrence can be used to identify biomarkers of aggressive tumor behavior that may be useful for individualized risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa10dcbb2ebaf779a0622a82815ec7ac7b25cca" target='_blank'>
              Pan-cancer copy number analysis identifies optimized size thresholds and co-occurrence models for individualized risk stratification
              </a>
            </td>
          <td>
            M. P. Nguyen, William C Chen, K. Mirchia, A. Choudhury, Naomi Zakimi, Vijay Nitturi, Tiemo J. Klisch, Stephen T. Magill, Calixto-Hope G. Lucas, Akash J. Patel, David R Raleigh
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tumors of infectious origin globally represent 13%. Oncogenic DNA viruses such as human papillomavirus (HPV), hepatitis B virus (HBV), and Epstein-Barr virus (EBV) are responsible for approximately 60% of these tumors. These oncoviruses are extensively studied to understand their role in cancer development, particularly through viral genome integration into the host DNA. Retroviruses require integration mediated by viral integrase for persistence, whereas DNA oncoviruses do not need integration for replication; instead, integration occurs incidentally. This process often targets fragile sites in the human genome, causing structural rearrangements that disrupt genes, activate proto-oncogenes, and increase genomic instability, all contributing to tumorigenesis. Integration near promoter regions and active genes is closely linked to carcinogenesis, highlighting its importance in developing diagnostic and therapeutic strategies. This review summarizes viral integration's role in oncogenesis, mechanisms of integration, and methods to study this process, focusing on DNA tumor viruses such as HBV, EBV, HPV, and Merkel cell polyomavirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef1eb5cd84b08745894e948edf6d9875f648ea7" target='_blank'>
              Genome integration of human DNA oncoviruses.
              </a>
            </td>
          <td>
            Zuzana Vojtěchová, R. Tachezy
          </td>
          <td>2025-07-23</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 effect partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            , , Qian Liu, , Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Polarity genes form intracellular protein complexes that establish epithelial cell polarity and homeostasis for various normal cellular functions. Partitioning Defective 6B (PARD6B), a polarity gene, functions as a scaffold node for the Par protein complex. However, its contribution to colon cancer is not well understood. In this study, we showed that PARD6B regulates tumor progression in colorectal cancer (CRC). PARD6B is located on chromosome 20, which is frequently amplified in CRC, and its expression in CRC correlates with DNA copy number amplification, including enhancer regions. Immunohistochemistry and single-cell analyses also showed that PARD6B expression was significantly higher in cancer cells. Furthermore, unlike other polarity gene groups comprising the Par complex, PARD6B mRNA expression was the only independent poor prognostic factor. In vitro and in vivo experiments revealed that PARD6B positively regulates cell proliferation and cell cycle progression. In silico analysis also showed that PARD6B expression positively regulated MYC expression, a pathway believed to be associated. Additional in silico and in vitro analyses supported the hypothesis that PARD6B regulates miR-34c, which directly targets and represses MYC expression. Pan-cancer analysis indicated that PARD6B is highly expressed in gastrointestinal tumors, including CRC, and that high PARD6B mRNA expression is a poor prognostic factor in other cancer types. In summary, highly expressed PARD6B can promote CRC growth by upregulating MYC expression while suppressing miR-34c expression, making PARD6B a potential prognostic biomarker and therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e805e1cc21e6cdb6e4ec857af7492b495f17f7a1" target='_blank'>
              Polarity Gene PARD6B Promotes Tumor Growth of Colorectal Cancer via Increasing MYC Expression.
              </a>
            </td>
          <td>
            Kosuke Hirose, Takaaki Masuda, Hajime Otsu, Taro Tobo, Kiyotaka Hosoda, Tadashi Abe, Yusuke Nakano, M. Hashimoto, Takanari Tatsumi, Tomohiko Ikehara, Takashi Ofuchi, Shinsaku Itoyama, Yuki Ando, Yasuo Tsuda, Yusuke Yonemura, K. Sugimachi, E. Oki, T. Yoshizumi, K. Mimori
          </td>
          <td>2025-06-18</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/265afafadcb2732fd2db8e268a3a108da8ad155c" target='_blank'>
              Avian germline-restricted chromosomes are reservoirs for active long-terminal-repeat retroviruses
              </a>
            </td>
          <td>
            Bohao Fang, Scott V. Edwards
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="CIC::DUX4 translocation-positive sarcomas (CDS) are rare, highly proliferative tumors associated with chemotherapy resistance. Without a clear understanding of the molecular pathways driving CDS, no effective treatment regimens have been developed. Here, we identify protection against DNA damage through upregulation of POLE and DNA repair pathways specific to CDS. These data may explain the high proliferation rate and chemotherapy resistance of these tumors and suggest targeted treatment strategies for CDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c1daa0e52dfd54fcd1b1c744a6ff9aaea6aa92" target='_blank'>
              Upregulation of POLE and proficient DNA repair are features of CIC::DUX4 sarcomas
              </a>
            </td>
          <td>
            Nathan D. Seligson, Anjali Paragji, Feha Shahalam, Binyam Yilma, James L Chen
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Jumping translocations (JTs) are rare chromosomal abnormalities that play a crucial role in the pathogenesis of various cancer types. These rearrangements, especially those involving chromosome 1q, are frequently associated with tumor progression, therapeutic resistance, and poor prognosis. One gene of particular interest, human Jumping Translocation Breakpoint (JTB), has been identified at the site of translocation breakpoints and exhibits complex, context-dependent roles in cancer biology. JTB protein functions as a pivotal regulator in mitosis, chromosomal segregation, apoptosis, and cellular metabolism. It is functionally linked with the chromosomal passenger complex (CPC) and is implicated in processes such as epithelial–mesenchymal transition (EMT), immune evasion, and therapy resistance, especially in breast and prostate cancers. Advances in genomic, transcriptomic, and proteomic research have highlighted the significant potential of JTB as a diagnostic biomarker and a target for therapeutic interventions. This review underscores the dual role of JTB as both a tumor suppressor and oncogene, depending on the cellular context, and advocates for its continued investigation at the genomic, transcriptomic, and proteomic levels. Understanding JTB’s multifaceted contributions to tumor biology may pave the way for novel biomarkers and targeted treatments in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48960a3d042990da8ae30d21a39837a5c48dded4" target='_blank'>
              From Jumping Gene to Cancer: Revisiting the Role of JTB Protein
              </a>
            </td>
          <td>
            Taniya M Jayaweera, Madhuri Jayathirtha, Krishan Weraduwage, Petra Kraus, C. Darie, A. Neagu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d0d63114e49d00ab10dbb83b8f715f2bf6490e6" target='_blank'>
              Identification of Novel DNA Sequence Motifs that Modulate Transcription in T cells
              </a>
            </td>
          <td>
            Nicole Knoetze, Eric Yung, Anthony Bayega, S. Brown, R. A. Holt
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c0eb75e7f5a0febc3a26c4b5b85dcca6192f2" target='_blank'>
              Validation of the MAGIC genomic signature on RNAseq in soft-tissue sarcomas
              </a>
            </td>
          <td>
            A. Benhaddou, G. Pérot, P. Rochaix, T. Valentin, G. Ferron, F. Chibon
          </td>
          <td>2025-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="EGFR amplification frequently happens within extrachromosome DNAs (ecDNAs) and is a major mutation in glioblastoma (GBM). However, targeting EGFR for GBM treatments has been unsuccessful. Here we characterized a long non-coding RNA (lncRNA) that is co-amplified with EGFR within ecDNAs that we name hidden EGFR long non-coding downstream RNA (HELDR). HELDR is a GBM-specific lncRNA that promotes tumorigenicity independent of EGFR signaling. HELDR globally binds genomic DNA and recruits the transcription co-activator p300 to the KAT7 promoter. p300-induced H3K27ac at the KAT7 promoter enlists other co-transcription factors, activating KAT7 transcription. KAT7 induces H3K14ac and H4K12ac that activate KAT7-driven gene programs that are critical for GBM malignancy. Targeting KAT7 or HELDR markedly enhances therapeutic effects of anti-EGFR treatments for GBM. These results not only reveal the role of HELDR in EGFR-driven GBM but also provide a strong rationale to characterize the role of lncRNAs co-amplified with driver oncogenes in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d139cff5a8d8b08cb9f51de2aadc8bac2f117be" target='_blank'>
              An EGFR Co-Amplified and De Novo Long Noncoding RNA HELDR Promotes Glioblastoma Malignancy through KAT7-Driven Gene Programs
              </a>
            </td>
          <td>
            , Xiaozhou Yu, Xiao Song, Richard Schäfer, Qingshu Meng, D. Tiek, Runxin Wu, Qiu He, Maya Walker, Rendong Yang, Qi Cao, Bo Hu
          </td>
          <td>2025-06-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Transposable elements (TEs) are dynamic repetitive regions constituting 50% of the genome. In adult cancers, TEs contribute to carcinogenesis through structural rearrangements that disrupt tumor suppressors and by onco-exaptation, which drives aberrant oncogene expression. TEs are characterized in adult malignancies, with epithelial cancers harbouring significantly more new insertions. However, the role of TEs in pediatric cancer is unknown. Many pediatric cancers arise from predisposition, such as Li-Fraumeni Syndrome, which is caused by germline TP53 (gTP53) variants. TP53 suppresses TEs by binding L1 elements to inhibit transcription and by inducing apoptosis in cells with high retrotransposition. Moreover, adult cancers with somatic TP53 variants harbour significantly more TEs than wildtype tumors. To investigate the pediatric context, we conducted a pan-cancer analysis of germline and tumor TE landscapes in pediatric patients with (n = 66 germline; n = 40 tumor) and without (n = 465 germline; n = 276 tumor) gTP53 variants. Germline TEs were identified using MELT, xTEA, and INSurVeyor, while tumor TEs were called with xTEA and TotalReCall. TEs were consolidated across individuals using JASMINE and annotated with AnnotSV. We excluded 21% of germline and 2% of tumor TEs classified as common, defined as present in over 3% of gnomAD or pediatric non-cancer controls (n=189). Overall, 11% of pediatric tumors harbored at least one L1 insertion, with a median of 0 insertions per tumor, substantially lower than the 21% reported in adult cancers using the same TE callers (Solovyov et al). Frequency in germline or tumor TEs did not vary significantly by tumor type, age of onset, sex, prior treatment, or disease state. Germline and tumor TEs were found in 35 and 147 cancer genes respectively. Tumor TEs significantly affected many cancer pathways, including those involved in mitosis. While no structural variants (SVs) involved TEs at both breakpoints, 171 SVs in six samples were associated with a TE at one breakpoint. Importantly, both germline and tumor TE burden did not differ between gTP53 carriers and wildtype individuals, nor did the number of evolutionarily young or full-length elements. Pathway enrichment analyses revealed no significant differences in TE-affected pathways between gTP53 and wildtype groups. However, a gradient-boosted tree model trained on TE distribution across genomic windows predicted gTP53 status with an AUPRC of 0.77, suggesting that gTP53 variants shape the germline TE landscape at a positional level rather than through global frequency changes. In summary, pediatric cancers exhibit a quiescent TE landscape, with low insertion rates across tumor types. gTP53 variants do not alter the germline or tumor TE frequency,but influence the location of germline insertions. TE analysis in pediatric cancer and gTP53 carriers will enhance our understanding of tumorigenesis, informing future treatment approaches.



 Brianne Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, Mehdi Layeghifard, Adam Shlien, Vallijah Subasri, David Malkin. Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba8aeceb59adfe162dadfd953074aed2a705a85" target='_blank'>
              Abstract A039: Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, M. Layeghifard, A. Shlien, Vallijah Subasri, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Repetitive elements account for a large proportion of the human genome and undergo alterations during early tumorigenesis. However, the exclusive fragmentation pattern of DNA-derived cell-free repetitive elements (cfREs) remains unclear. Methods This study enrolled 32 healthy volunteers and 112 patients with five types of cancer. A novel repetitive fragmentomics approach was proposed to profile cfREs using low-pass whole genome sequencing (WGS). Five innovative repetitive fragmentomic features were designed: fragment ratio, fragment length, fragment distribution, fragment complexity, and fragment expansion. A machine learning-based multimodal model was developed using these features. Results The multimodal model achieved high prediction performance for early tumor detection, even at ultra-low sequencing depths (0.1×, AUC = 0.9824). Alu and short tandem repeat (STR) were identified as the primary cfREs after filtering out low-efficiency subfamilies. Characterization of cfREs within tumor-specific regulatory regions enabled accurate tissue-of-origin (TOO) prediction (0.1×, accuracy = 0.8286) and identified aberrantly transcribed tumor driver genes. Conclusion This study highlights the abundance of repetitive DNA in plasma. The innovative fragmentomics approach provides a sensitive, robust, and cost-effective method for early tumor detection and localization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f71e0eea2828ba986a7986cc3d935c9f50cf8f6" target='_blank'>
              Comprehensive fragmentation of cell-free repetitive DNA for enhanced cancer detection in plasma
              </a>
            </td>
          <td>
            Mingguang Zhang, Shuohui Dong, Wei Rao, Shiwen Mei, G. Hu, Ling Liu, Zhen Wang, Jianqiang Tang
          </td>
          <td>2025-07-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer affects over 2 million women annually and results in 650,000 deaths. This study aimed to identify epigenetic mechanisms impacting breast cancer-related gene expression, discover potential biomarkers, and present a novel approach integrating feature selection, Natural Language Processing, and 3D chromatin structure analysis. We used The Cancer Genome Atlas database with over 800 samples and multi-omics datasets (mRNA, miRNA, DNA methylation) to select 2701 features statistically significant in cancer versus control samples, from an initial 417,486, using the Monte Carlo Feature Selection and Interdependency Discovery algorithm. Classification of cancer vs. control samples on the selected features returned very high accuracy, depending on feature-type and classifier. The cancer samples generally showed lower expression of differentially expressed genes (DEGs) and increased β-values of differentially methylated sites (DMSs). We identified mRNAs whose expression is explained by miRNA expression and β-values of DMSs. We recognized DMSs affecting NRF1 and MXI1 transcription factors binding, causing a disturbance in NKAPL and PITX1 expression, respectively. Our 3D models showed more loosely packed chromatin in cancer. This study highlights numerous possible regulatory dependencies, and the presented bioinformatic approach provides a robust framework for data dimensionality reduction, enabling the identification of key features for further experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e55876d31494cc939dbaaad24449b7b1c509d3e" target='_blank'>
              Unveiling Epigenetic Regulatory Elements Associated with Breast Cancer Development
              </a>
            </td>
          <td>
            Marta Jardanowska-Kotuniak, Michał Dramiński, Michał Wlasnowolski, Marcin Łapiński, Kaustav Sengupta, Abhishek Agarwal, Adam Filip, Nimisha Ghosh, Vera Pancaldi, Marcin Grynberg, I. Saha, Dariusz Plewczyński, Michał J. Dąbrowski
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Motivation The wide range of cellular complexity created by multicellular organisms is due in large part to the intricate and synergistic interplay of regulatory complexes throughout the eukaryotic genome. These regulatory elements “enhance” specific gene programs and have been shown to operate in diverse networks that are distinct across cell states of the same organism. Attempts to characterize and predict enhancers have typically focused on leveraging information-dense DNA sequence in parallel with epigenomic assays. We examined the viability of enhancer prediction using only a minimal set of epigenomic datasets without direct DNA information. Results We demonstrate that chromatin datasets are sufficient to identify enhancers genome-wide with high accuracy. By training networks leveraging data from multiple cell types simultaneously, we generated a cell-type invariant enhancer prediction platform that utilized only the patterns of protein binding for inference. We also showed the utility of swarm-based adversarial attacks [adversarial particle swarm optimization (APSO)] to deconvolute trained genomic neural networks for the first time. Critically, unlike saliency mapping or other game-theory based approaches, APSO is completely network-architecture independent and can be applied to any prediction engine to derive the features that drive inference. Availability and implementation All software and code for data downloading, processing, enhancer inference, eXplainable AI (XAI), and complete figure generation are publicly available on GitHub at https://github.com/EpiGenomicsCode/ChromEnhancer and Zenodo at https://doi.org/10.5281/zenodo.15652797.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5515d8f4201165cbbdeefdd1affb6ab4930eae1" target='_blank'>
              Adversarial attack of sequence-free enhancer prediction identifies chromatin architecture
              </a>
            </td>
          <td>
            Jamil Gafur, Olivia W Lang, William Lai
          </td>
          <td>2025-06-24</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well defined. Potential DNA replication origins are licensed through the recruitment of a pair of minichromosome maintenance complexes (MCMs). In yeast, a subset of MCMs is selected for initiation by the SLD3–SLD7 firing factors during G1. Like in yeast, excessive numbers of MCM complexes are loaded onto chromatin during G1 in human cells, but it is unclear how MCMs are selected for firing. Results We examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast replication initiation factors Sld3 and Sld7. TRESLIN and MTBP binding patterns are detectable in a G1 synchronized population and have a higher binding signal compared to asynchronously cycling cells. MTBP binds to early-mid replicating regions in an early S population. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. We show that TRESLIN and MTBP binding during G1 does not require origins licensed with loaded MCMs. Conclusions We provide evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins. MTBP binding location and footprint during S phase differs from that seen in G1, implicating two separate modes of binding. These data highlight binding mechanisms for DNA replication initiation factors in human cells that have diverged from those shown in yeast, suggesting differences in origin selection. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03665-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b74245a8a7e7194eb9255132a18f436ef65a462" target='_blank'>
              Cell cycle-dependent TICRR/TRESLIN and MTBP chromatin binding mechanisms and patterns
              </a>
            </td>
          <td>
            Tyler D. Noble, Courtney G. Sansam, Kimberlie A. Wittig, Blanka Majchrzycka, Christopher L. Sansam
          </td>
          <td>2025-07-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive disease with poor survival outcomes and increasing incidence. There is a clear and present need for enhanced understanding of cellular mechanisms of tumorigenesis, validation of robust genetic signatures predictive of aggressive disease, and novel informatics tools to simplify analysis of Merkel cell polyomavirus (MCPyV)-host genome interactions. Genomic DNA was harvested from 54 MCC tumors for exome sequencing and in-depth genetic profiling of a 226-gene panel. We further developed a robust informatics package (MCPyViewer) optimized for MCPyV integration site analysis with graphical output to simplify usability for end-users. Finally, we assessed the prognostic impact of specific genetic signatures on MCC-specific survival in our cohort. Our study included 54 patients (n = 44 MCPyV-positive), 11 (20.4 %) of which had died of MCC at last follow-up. Human genes altered at high frequency included LRP1B (n = 10, 18.5 %), FAT1 (n = 9, 16.7 %), KMT2D (n = 9, 16.7 %), and RB1 (n = 7, 13.0 %). In 36 of 44 (81.8 %) MCPyV-positive tumors, we identified viral integration into the human genome with a median of two events per tumor. In six tumors, MCPyV integrated into COSMIC Tier 1 or Tier 2 cancer-related human genes. Implications: A combined genomics score incorporating tumor mutational burden and copy number variation was strongly prognostic of MCC-specific survival controlling for lymph node metastases and tumor MCPyV status, thus, our study adds critical understanding to prognostic markers and tumorigenic mechanisms in MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef7b14dad4673e2d7fa9c8cccb49e7531dcb5de" target='_blank'>
              Genomic Signatures of Poor Prognosis in Merkel Cell Carcinoma: A Single-Institution Prospective Study.
              </a>
            </td>
          <td>
            Joshua D Smith, A. Bhangale, Wenjin Gu, C. Brummel, E. Gensterblum-Miller, Scott McLean, Paul W. Harms, K. Harms, C. Bichakjian, M. Spector, R. E. Mills, J. C. Brenner
          </td>
          <td>2025-07-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays a significant role in cancer development. However, our understanding of its role in normal cells is limited. This study aimed to explore the roles of eccDNA in the differentiation of human bone marrow mesenchymal stem cells (BMSCs). Methods Through circular DNA sequencing (Circle-seq) and computational analysis, we documented eccDNAs in human BMSCs and their differentiation into osteoblasts, adipocytes, and chondrocytes. Next, the assay for transposase-accessible chromatin with high throughput sequencing (ATAC-seq) and RNA sequencing (RNA-seq) data were integrated with Circle-seq data. The roles of phosphatidylinositol 4-kinase alpha (PI4KA) and eccDNA as functional enhancers in human BMSC differentiation were assessed in vitro. Results Our results demonstrate that eccDNA is common in human BMSCs. In the differentiated groups including osteoblasts, adipocytes and chondrocytes, eccDNA-encoded genes presented higher expression compared to the background. We found eccDNA displayed enhanced chromatin accessibility in human BMSCs, which contribute to increased transcription of genes, such as runt-related transcription factor 2 (RUNX2), a key transcription factor for BMSC osteogenesis. We further found PI4KA, an eccDNA-encoded gene, promoted osteogenic differentiation of human BMSCs via the PI3K/AKT/mTOR pathway. Finally, we demonstrated eccDNA as functional enhancers to regulate BMSC differentiation in a dose-dependent manner. Conclusions Our study provides the evidence of eccDNA regulating BMSC differentiation and highlights the roles of eccDNA as transcription template and enhancer in normal cells, which will facilitate future research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04516-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9298ef8d097b3fec00680e4438db26b85d21b6" target='_blank'>
              Extrachromosomal circular DNAs in the differentiation of human bone marrow mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuxi Gu, Yidan Song, Shuhua Wang, Jun Liu
          </td>
          <td>2025-07-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CD4+ regulatory T (Treg) cells in tissues play crucial immunoregulatory and regenerative roles. Despite their importance, the epigenetics and differentiation of human tissue Treg cells are incompletely understood. Here, we performed genome-wide DNA methylation analysis of human Treg cells from skin and blood and integrated these data into a multiomic framework, including chromatin accessibility and gene expression. This analysis identified programs that governed the tissue adaptation of skin Treg cells. We found that subfamilies of transposable elements represented a major constituent of the hypomethylated landscape in tissue Treg cells. Based on T cell antigen receptor sequence and DNA hypomethylation homologies, our data indicate that blood CCR8+ Treg cells contain recirculating human skin Treg cells. Conversely, differences in chromatin accessibility and gene expression suggest a certain reversal of the tissue adaptation program during recirculation. Our findings provide insights into the biology of human tissue Treg cells, which may help harness these cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c24c6ab8c718bfe4323e796346a6209a2aa18e" target='_blank'>
              DNA hypomethylation traits define human regulatory T cells in cutaneous tissue and identify their blood recirculating counterparts
              </a>
            </td>
          <td>
            Niklas Beumer, Charles D. Imbusch, Tamara Kaufmann, Lisa Schmidleithner, Kathrin Gütter, Philipp Stüve, Harriet Marchel, D. Weichenhan, Marion Bähr, Brigitte Ruhland, Federico Marini, Lieke Sanderink, Uwe Ritter, Malte Simon, K. Braband, Morten Voss, Sara Salome Helbich, Delia Mihaela Mihoc, Agnes Hotz-Wagenblatt, Hadrian Nassabi, A. Eigenberger, Lukas Prantl, C. Gebhard, Michael Rehli, Nicholas Strieder, Kartikeya Singh, Christian Schmidl, C. Plass, J. Huehn, T. Hehlgans, J. Polansky, B. Brors, Michael Delacher, Markus Feuerer
          </td>
          <td>2025-07-16</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastoma is a heterogeneous disease where patient stratification is critical for prognosis and treatment decisions and where it recently has been suggested that the presence of telomere maintenance mechanisms (TMM) should be considered in risk stratification. We investigated the utility of DNA methylation-based classification for neuroblastoma diagnostics by analysing 303 tumours samples from two cohorts. We show that of the total number of cases, an average of 90% of the samples classified as neuroblastoma, while 66% also achieved confident classification into the three NB subclasses: “MYCN-type”, “ALT/TERT TMM positive” and “TMM negative”. The tumours classified as MYCN-type showed genomic amplification of MYCN (MNA); however, some MYCN-type cases lacked evident MNA, suggesting that epigenetic states might be influenced by other factors such as activating ALK mutations. Survival analysis indicated similar poor survival probabilities for patients classified as TMM positive or MYCN type, distinct from the inferior survival of TMM-negative cases. All cases, with complementary genomic data available, associated with TMM positivity also presented features associated with telomere lengthening mechanisms, including TERT or ATRX alteration. However, some tumours positive for these features, especially TERT rearrangement, classified as MYCN type rather than TMM positive, indicating that MNA and other mechanisms introduce a methylation pattern that supersede or overlap with pattern imposed by TERT. Chromosomal copy number alterations (CNAs) characteristic of methylation subclasses were identified, including 1p deletion and 17q gain in MYCN type and combinations of 11q loss, 3p loss, 7q gain, and 17q gain in TMM-positive cases, highlighting the potential of the methylation arrays to replace SNP arrays for prognostic genomic assessments. Our study demonstrates that DNA methylation-based classification stratifies neuroblastoma into clinically relevant subgroups, aiding diagnostic and prognostic decisions, although discrepancy between genomic features and methylation classification does occur. The interplay between genomic alterations and methylation patterns could give clues into the discrepancy and underscores the complexity of neuroblastoma biology and the need for further research and validation of clinical outcomes of the patients in the respective subclasses. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01936-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f713d2afc90db793e049e383b6f7ea1cd34f79" target='_blank'>
              DNA methylation profiling for molecular classification of neuroblastoma
              </a>
            </td>
          <td>
            Maja Löfgren, A. Djos, Shiva Rezaei, Medha Suman, P. Kogner, Tommy Martinsson, S. Fransson, Helena Carén
          </td>
          <td>2025-07-27</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) is considered one of the most aggressive B-cell lymphoid neoplasms. The transcription factor SOX11 is aberrantly expressed in conventional aggressive MCL, while it is not or weakly expressed in the leukemic non-nodal MCL subtype with a predominantly indolent clinical evolution. SOX11 is a key driver of MCL through the regulation of several oncogenic mechanisms, suggesting that it may be interacting with different protein complexes to exert its multiple actions. Using proteomic strategies, we characterized the SOX11-interactome and validated its physical interaction with SMARCA4, the catalytic subunit of the SWI/SNF chromatin-remodeling complex. SMARCA4 expression is directly regulated by SOX11, and its upregulation significantly associates with worse outcomes of patients. Integration of global DNA-binding and transcriptomic profiles revealed that SOX11 and SMARCA4 share binding sites enriched in open chromatin and active promoters and regulate common key oncogenic pathways crucial for MCL progression and aggressiveness. The SMARCA4-specific PROTAC-degrader AU-15330 significantly reduced SOX11 binding to specific regulatory regions and diminished the activation of BCR-, NIK-, and BCL2-signaling pathways. Moreover, SMARCA4 degradation significantly reduced proliferation and induced apoptosis of SOX11-positive MCL cells, highlighting AU-15330 as a promising therapeutic approach for patients who may relapse from current target therapies in MCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8cd948c125d001374e84641740fb017bb7f370" target='_blank'>
              The SOX11:SMARCA4 complex is a driver of oncogenic transcriptional programs in mantle cell lymphoma
              </a>
            </td>
          <td>
            A. De Bolòs, Maria Carreras-Caballé, M. Sureda-Gómez, M. Rodríguez, S. Charalampopoulou, R. Albero, A. Maiques-Diaz, Heribert Playa-Albinyana, G. Clot, , D. Colomer, J. Martín-Subero, E. Campo, David Reverter, Josep Villanueva, V. Amador
          </td>
          <td>2025-07-30</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Background. Medulloblastoma (MB) prognosis and response to therapy depend largely on genetic changes in tumor cells. Many genes and chromosomal abnormalities have been identified as prognostic factors, including amplification of MYC oncogenes, gains in 1q and 17q, deletions in 10q and 21p, or isochromosomes 17 (i(17)(q10)). The frequency of these abnormalities varies greatly between ethnic populations, but the frequency of specific abnormalities, such as MYCC and MYCN amplification, 17q gain, and deletions, in the Russian population is unknown. Objective: The aim is to study the frequency of MYCC and MYCN amplifications, 17q gain, and 17p deletion and determine their prognostic value in Russian patients with MB. Methods. This study was performed on MB cells obtained from 18 patients (12 boys and 6 girls, aged between 3 months and 17 years, with a median age of 6.5 years). Determination of cytogenetic aberrations was carried out using FISH assays with MYCC-SO, MYCN-SO, and MYCN-SG/cen2 probes, as well as cen7/p53 dual color probes and PML/RARα dual color probes (Abbott Molecular, USA). One-way ANOVA and Fisher’s F-test were used to compare the two groups. The differences were considered significant when p < 0.05. Results. In 77.7% of patients (14/18), the classical type of MB was present; in 16.7% (3/18), desmoplastic type; and in 5.6% (1/18), nodular desmoplasic types of neoplasms. Amplification of MYC genes was detected in 22.2% of Russian patients (n = 4 out of 18). Patients with MYC amplification had the worst overall survival (OS: 0% vs. 68%, p = 0.0004). Changes on the 17th chromosome were found in 58.3% of patients. Deletion of 17p occurred in 23.1%, and gain of 17q occurred in 46.2%. There were no significant differences in OS, clinical signs, or the presence of additional 17q material or 17p deletion among patients with MB. Conclusions: Amplification of the MYC gene is a predictor of poor overall survival to therapy and a high risk of metastatic relapse. This allows us to more accurately stratify patients into risk groups in order to determine the intensity and duration of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f5e5b1eac08421c304a6da77fd1d4ce4707e64" target='_blank'>
              The Prognostic Value of Amplification of the MYCC and MYCN Oncogenes in Russian Patients with Medulloblastoma
              </a>
            </td>
          <td>
            A. Chernov, Ekaterina Batotsyrenova, Sergey N Zheregelya, S. Pyurveev, V. Kashuro, Dmitry Ivanov, Elvira Galimova
          </td>
          <td>2025-07-27</td>
          <td>Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Despite many new drugs, multiple myeloma (MM) remains an incurable plasma cell malignancy, and drug resistance is a long-standing topic in this field. Characterized by efficient transcription without being limited by the double helix structure and promoter, extrachromosomal circular DNA (EccDNA) has been proven to be widely involved in cancer development and drug resistance. Methods We performed circle-seq and mRNA-seq on samples from three MM patients at the time of complete response and relapse to screen EccDNA candidate molecules. Outward PCR and Sanger sequencing were used to identify EccDNA molecules. RT‒qPCR and WB were performed to detect gene expression levels. Fluorescence in situ hybridization (FISH) was carried out to detect the deletion of chromosome 17p (del (17p)). Transmission electron microscopy (TEM) was conducted to observe autophagosomes. Luciferase reporter assays were performed to validate the binding of microRNAs to target genes. Cell viability assays and apoptosis assays were employed to assess drug resistance. Xenograft tumor mouse models were established for in vivo experiments. Immunohistochemistry (IHC) was used to detect protein expression levels. Results We successfully identified an EccDNA molecule (EccDNAchr17:38719676–38719812) in one relapsed MM patient with del(17p) and named it MIR4726EccDNA. We demonstrated that the overexpression of MIR4726EccDNA in MM cells can increase bortezomib resistance. We further confirmed that the precursor miRNA carried by MIR4726EccDNA can be efficiently transcribed in MM cells and that MIR4726EccDNA drives bortezomib resistance via the MIR4726-5p/NXF1/NKIRAS2 axis. We further revealed that downregulation of NFKB inhibitor interacting Ras like 2 (NKIRAS2) activated the NF-κB pathway and increased autophagy. Moreover, we established a xenograft model of human MM via subcutaneous inoculation. We administered intra-tumoral injection of AgoMIR4726-5p and intraperitoneal injection of bortezomib and found that AgoMIR4726-5p promoted tumor progression and partially drove bortezomib resistance. Conclusions In summary, our findings indicate that artificially synthesized MIR4726EccDNA is functional in cells and that MIR4726EccDNA enhances tumor progression and partially mediates drug resistance by enhancing MIR4726-5p/NXF1/NKIRAS2 axis dependent autophagy. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02340-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799fad2e2c9bef9366577ed2d5320bd4f2c64f37" target='_blank'>
              MIR4726EccDNA drives bortezomib resistance in multiple myeloma by enhancing MIR4726-5p/NXF1/NKIRAS2 axis dependent autophagy
              </a>
            </td>
          <td>
            Fangfang Li, Xinyi Long, Sishi Tang, Jinhua Yan, Jing Liu, Yunfeng Fu
          </td>
          <td>2025-07-18</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af90714fba177985e55619eba3e392595401a76" target='_blank'>
              Hierarchical Coordination of Polymerase Theta and RAD51 Resolves Clustered Replication Fork Collapse
              </a>
            </td>
          <td>
            Chelsea M Smith, Dennis A. Simpson, Paolo Guerra, W. Feng, Simon W Ellington, Dongbo Lu, Carel Fijen, Natalie M. Boylston, K. K. Chiruvella, Iman El-Shiekh, Rachel Lee, A. Wozny, A. Pregnall, B. Eckenroth, A. El-Sagheer, Tom Brown, S. Doublié, Dale A. Ramsden, Eli Rothenberg, S. Nik-Zainal, Gaorav P. Gupta
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="ABSTRACT Kaposi's sarcoma‐associated herpesvirus (KSHV), the etiological agent of Kaposi's sarcoma and other lymphoproliferative disorders, presents a delicate balance between host cell manipulation, and orchestrating its transcriptional repertoire. Like all herpes viruses, the virus can reside in latency, expressing only a handful of proteins or launch a lytic program that expresses over 80 proteins responsible for the generation and release of new virions. Enhancers, as their name suggests, can enhance the transcription rate from a promoter depending on the cellular environment and tissue. While the promoters in KSHV have been extensively studied, the enhancers were only recently identified. Here we will describe our current view on KSHV enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0b1a93efde5ee2df29eb9e253f26d14a84bff84" target='_blank'>
              Viral Enhancers Orchestrate Kaposi's Sarcoma‐Associated Herpesvirus Transcription
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Meir Shamay
          </td>
          <td>2025-07-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract MYC proteins are potent oncoproteins that drive tumorigenesis in a wide range of cancers, making it critical to understand their oncogenic functions and underlying mechanisms. Although MYC overexpression induces transcriptional and replication‐associated stress, recent studies have paradoxically identified MYC as a key resilience factor that protects cancer cells from these stressors. In this review, we explore the dual role of MYC in both driving and mitigating cellular stress to achieve its oncogenic function. We also examine how MYC‐induced transcriptional and replicative stress generates potentially immunogenic nucleic acid species while simultaneously helping cancer cells evade host immune recognition. We propose a model in which MYC plays a critical role in managing the stress it induces, thereby maintaining a balance that promotes tumor growth. Based on this model, we discuss potential therapeutic strategies targeting MYC‐dependent stress responses, offering new avenues for cancer treatment and highlighting the complexity of MYC‐driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ce286c658baf7f61e9c844f34e8e87797cdc79" target='_blank'>
              MYC: The Guardian of Its Own Chaos
              </a>
            </td>
          <td>
            A. Gaballa, Bastian Krenz, Leonie Uhl
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="The pediatric and adolescent liver cancers Hepatoblastoma (HBL) and Fibrolamellar hepatocellular carcinoma (FLC) are dangerous diseases requiring aggressive surgery, when feasible, and non-targeted toxic chemotherapy for a chance of cure, due to insufficient knowledge of underlying molecular mechanisms. We've previously reported the essential role of ph-S675-β-catenin in the reorganization of genomic structure in HBL and FLC by oncogenic activation via chromosomal regions called Cancer Enhancing Genomic Regions or Aggressive Liver Cancer Domains (CEGRs/ALCDs). In FLC, the fusion J-PKAc oncoprotein phosphorylates β-catenin at Ser675, triggering such CEGRs/ALCDs-mediated activation of oncogenes. In this paper, we found that all members of the cohesin ring - CTCF, Rad21, SMC1, SMC3 and STAG1 - and β-catenin-TCF4 are bound to CEGRs/ALCDs of oncogenes in HBL and FLC, as well as many other cancers, and that this binding increases transcription. Examination of a large cohort of HBL and FLC samples revealed that cohesin ring expression is dramatically elevated in the majority. The cohesin ring, as well as the ph-S675-β-catenin-TCF4-p300 complex, are detected on both the promoter and intron-located CEGRs/ALCDs of NRF2 and Thy1, correlating with increased transcription. This suggests that the cohesin ring creates the DNA-loop for oncogene activation. The inhibition of the cohesin ring by JQ1 reduces proliferation of HBL and FLC cells in culture, as well as cells expressing the FLC-specific DNAJB1-PKAc fusion oncogene. Implications: These studies provide evidence that J-PKAc-β-catenin and the cohesin ring cooperate in oncogenic activation for both HBL and FLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe133513089f190d4d540365b1cbabc148bb1d2" target='_blank'>
              β-catenin-cohesin ring-CEGRs/ALCDs axis activation contributes to the development of hepatoblastoma and fibrolamellar HCC.
              </a>
            </td>
          <td>
            Yasmeen Fleifil, Ruhi Gulati, Katherine Jennings, Ina Kycia, Alexander J Bondoc, Gregory M. Tiao, James I Geller, M. Weirauch, K. Vakili, Lubov Timchenko, Nikolai Timchenko
          </td>
          <td>2025-07-23</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Satellite DNA is essential for chromosome stability and gene regulation, yet its specific roles in early human embryogenesis remain poorly defined. Here, we integrated the complete human genome reference (T2T-CHM13) with RNA-seq data to investigate the expression and regulation of the satellite DNA element D20S16 across key stages of human embryonic development. We identified 20 distinct D20S16 tandem repeat clusters, but found that only two, both located on chromosome 20, were actively transcribed during early embryogenesis. Expression of D20S16 was high in early developmental stages, significantly declining thereafter. Comparative analysis revealed minimal expression of D20S16 in macaque embryos, correlating with fewer and shorter repeat units. Beyond embryogenesis, D20S16 also exhibited notably high expression levels in breast cancer and testicular tissues, suggesting additional biological roles. Furthermore, we investigated the evolutionary distribution of D20S16 across primates and other mammals. Our findings highlight the potential regulatory functions of satellite DNA in human development, pointing to the importance of specific chromosomal contexts for transcriptional activation. This study enhances our understanding of satellite DNA’s functional and evolutionary significance, laying the groundwork for future research into its roles in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3615f227ececbc4cca8de8d71737eff9957c4e38" target='_blank'>
              Unveiling unique expression patterns of D20S16 satellite DNA in human embryonic development
              </a>
            </td>
          <td>
            Yajie Hu, Kenji Mizuguchi, Kosuke Hashimoto
          </td>
          <td>2025-07-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e7385113aeec5ed3d01c2aa1c0786f66a695de" target='_blank'>
              Mammalian epigenomic conservation of promoters and enhancers in the heart associates with trait-associated variation and impacts cardiomyocyte phenotypes
              </a>
            </td>
          <td>
            S. Frost, Diego Fernandez-Aroca, Elise Parey, Adrian Rodriguez-Gonzalez, Daniel Pavon-Heredia, Diego Villar
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy that threaten women's health seriously. Many studies have shown that long noncoding RNAs can play significant role in the tumorigenesis of breast cancer. By analyzing the TCGA breast cancer genome data and transcriptome data, we found that copy number amplification drives the activation of long noncoding RNA MNX1-AS in breast cancer and indicates poor prognosis. Functionally, MNX1-AS1 could regulate pathogenesis of breast cancer in vitro and in vivo. Mechanistically, MNX1-AS1 could bind IGF2BP1, which increased the interaction of IGF2BP1 with MET mRNA to promote its stability, thus promoting tumorigenesis of breast cancer. Moreover, combination of MNX1-AS1 inhibition and MET small molecule inhibitor (PHA-665752, PHA) exhibited better antitumor efficacy in xenograft model, suggesting the therapeutic potential. Overall, our findings indicated that MNX1-AS1/MET regulatory axis may serve as a potential therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb4e1e2cea9e0a9b0b2c1fbdca7c268c8199624a" target='_blank'>
              Long Noncoding RNA MNX1-AS1 Promotes Tumorigenesis of Breast Cancer by Binding IGF2BP1 to Activate c-MET.
              </a>
            </td>
          <td>
            Jinrong Xie, Jianchun Gu, Wenjie Lv, Limin Cheng, Shuxian Chen, Ran Wu, Yan Liang, Meiling Zhu, Siyu Chen, Mawei Jiang, Jun Su
          </td>
          <td>2025-07-24</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a novel biomarker for cancer detection due to its tumor-specific amplification and stable structure in circulation. However, its clinical application is hindered by extremely low abundance in biofluids and the lack of robust detection techniques. To address this, we screened for tumor-associated eccDNA biomarkers and developed NPCC (Nested PCR-CRISPR/Cas12a), a novel method combining nested PCR for ultrasensitive amplification with CRISPR/Cas12a for sequence-specific detection. The assay employs two rounds of junction-specific PCR to enrich eccDNA, followed by CRISPR/Cas12a-mediated cleavage guided by target-specific crRNA. Validation using synthetic circular DNA standards demonstrated a limit of detection (LoD) of 10-6 fM, representing a >100-fold improvement over conventional PCR, with no cross-reactivity to linear or genomic DNA fragments. In plasma samples from 88 cancer patients, NPCC successfully detected multiple tumor-specific eccDNAs, including the hepatocellular carcinoma marker eccDNA-HCC-1 (AUC = 0.8977). NPCC overcomes key technical barriers in liquid biopsy, offering a cost-effective, highly sensitive, and specific platform for noninvasive cancer diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeca77128a740c764bbb2a48a28ef00f3cfa9edd" target='_blank'>
              Ultrasensitive eccDNA Detection for Tumor Diagnostics by Using CRISPR/Cas12a-Coupled Nested PCR.
              </a>
            </td>
          <td>
            Gege Liu, Junlu Wu, Bin Yang, J. Kong, Yaran Li, Chen Chen, Jiayi Hu, Zujun Sun, Yanqiang Hou, Ruixin Sun, Dong Li
          </td>
          <td>2025-07-31</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is a common subtype of breast cancer that is defined in part by genetic loss of CDH1 caused by mutation or deletion, leading to loss of cell adhesion protein E-cadherin in >90% of ILC. Genetic loss of CDH1 is an early event in ILC oncogenesis, yet the mechanisms by which CDH1/E-cadherin acts as a tumor suppressor are not well understood. To study how early CDH1 loss drives ILC oncogenesis, we used a series of non-transformed human mammary epithelial cell (HMEC) models to target CDH1/E-cadherin, inhibiting extracellular E-cadherin signaling using antibodies versus modeling genetic CDH1 loss using siRNA or knockout via CRISPR/Cas9. Through transcriptome analyses across four HMEC models, we found that the mode of E-cadherin loss or suppression is critical for the subsequent phenotype. Antibody-mediated inhibition of cell-cell contacts induced gene signatures of epithelial-mesenchymal transition (EMT), consistent with the role of E-cadherin suppression during the EMT process. Conversely, genetic CDH1 loss – as in ILC oncogenesis – repressed EMT signatures, and instead remodeled gene expression toward a luminal epithelial phenotype. Using single cell transcriptomics and flow cytometry analyses of cell lineage markers, we found that genetic loss of CDH1 reprogrammed cells to a luminal progenitor-like phenotype. By isolating luminal versus basal cells prior to CDH1 knockout, we found that CDH1 loss led to remodeling of lineage identity in both populations, converging on a new lineage homeostasis with a luminal progenitor-like phenotype. Consistent with increased progenitor features, CDH1 loss enhanced proliferative capacity over the finite lifespan of the HMECs, highlighting a feature of early CDH1 loss that may contribute to clonal advantage during tumor initiation. Our findings support that inhibition of E-cadherin results in different transcriptional response compared to CDH1 loss, with the latter driving a transcriptional and phenotypic state characteristic of a luminal progenitor-like population, which offers new insight into early events in ILC oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55a904a9de6859d0ee1985b86b2f0ef97b6b1ec" target='_blank'>
              CDH1 loss remodels gene expression and lineage identity in human mammary epithelial cells
              </a>
            </td>
          <td>
            Maggie Musick, Chinasa A Ufondu, Carmen E. Rowland, Joseph L. Sottnik, Madeleine T. Shackleford, Camryn S. Nesiba, J. Ostrander, Matthew J. Sikora
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Introduction: Aromatase inhibitors (AIs) are widely used in the treatment of estrogen receptor-positive (ER+) breast cancer to inhibit estrogen production and suppress the ER signaling pathway. Target genes regulated by estrogen stimulation of ER+ malignant cells lead to uncontrolled proliferation. The NEOLETEXE trial evaluated the sequential use of letrozole (2.5 mg daily) and exemestane (25 mg daily) in the neoadjuvant setting. We performed single-cell RNA sequencing (scRNA-seq) on 24 patients enrolled in the NEOLETEXE trial (i) before treatment, (ii) before change of AI (crossover at 3 months), and (iii) at the end of neoadjuvant therapy (6 months of treatment). Methods and aims: We aim at characterizing the distal and proximal regulatory elements in different cell types and states observed in longitudinally collected tumor samples. Transcription factor regulatory networks are important determinants of ER+ breast cancer pathogenesis. Understanding the dynamic changes in regulatory element usage under treatment pressure will help to map the molecular events associated with sensitivity or resistance to AI therapy. We used SCAFE (Single-Cell Analysis of Five-prime Ends), a computational method that maps transcribed cis-regulatory elements (CREs) in single cells using 5’ end scRNA-seq data. Results: Using unsupervised clustering of scRNA gene expression, we identified the major cell types in ER+ breast tumors. SCAFE was applied to identify 99,996 distal CREs (i.e., enhancers) in a cell type-agnostic manner. We quantified the reads mapping to each distal CRE in each cell creating a cell-by-distal CRE count matrix. We employed a Latent Dirichlet Allocation topic model implemented in pyCistopic to evaluate the significance of enhancers in determining cell phenotypes. Additionally, we characterized the distal CRE landscape specific to each cell type and state, providing an atlas of cell type-specific enhancer usage in breast tumors. By predicting direct transcription factor binding to the corresponding CRE’s DNA sequences using UniBind, we identified ESR1, FOXA1, and GATA3 as transcription factors associated with enhancers in malignant cells. In contrast, in immune cells such as T cells, we found enriched binding sites for RUNX1, ETS, and TBX21. Conclusions: Using 5' end scRNA-seq data, we successfully generated an atlas of distal regulatory elements for each cell in breast tumors. We identified the transcription factors likely to activate these cell type-specific distal regulatory elements. Mapping distal regulatory elements usage and transcriptomes in single cells allows us to study the interplay between enhancer activity, transcription factor binding, and gene regulation. It provides insights into the molecular mechanisms of AI therapy sensitivity and resistance and sheds light on the driver pathways associated with cell phenotypes and states in ER+ breast cancer.
 Citation Format: Villads Winton, Ilayda Altinönder, Marie Fongaard, Pål Marius Bjørnstad, Knut Selsås, Stephanie Beate Geisler, Manouchehr Seyedzadeh, Unn-Cathrin Buvarp, Torben Lüders, Marianne Lyngra, Arnoldo Frigessi, Vessela Kristensen, Anthony Mathelier, Jürgen Geisler, Xavier Tekpli. Characterization of cell type specific distal cis-regulatory elements from 5’ scRNA-seq in estrogen receptor positive breast cancer patients treated with aromatase inhibitors [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36055d544f5aaf03eb962008479042508d62220f" target='_blank'>
              Abstract P5-06-28: Characterization of cell type specific distal cis-regulatory elements from 5’ scRNA-seq in estrogen receptor positive breast cancer patients treated with aromatase inhibitors
              </a>
            </td>
          <td>
            Villads Winton, Ilayda Altinönder, Marie Fongaard, P. Bjørnstad, K. Selsås, Stephanie B Geisler, M. Seyedzadeh, Unn-Cathrin Buvarp, T. Lüders, M. Lyngra, A. Frigessi, Vessela N. Kristensen, Anthony Mathelier, Jürgen Geisler, Xavier Tekpli
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Among the various genotypes of the human papillomavirus (HPV), the HPV16 genotype is the most oncogenic by far, and some (sub)lineages may be associated with an increased risk of cancer. Thus, characterizing the full genome is important to understand the link between the HPV16 genome variability and its transforming role. We set up a multiplex PCR approach combined with Oxford Nanopore Technologies for the HPV16 full-genome characterization. The primers were designed with PrimalScheme, and the optimization of the monoplex/multiplex PCR was performed on SiHa cells. After library preparation, the sequencing was performed on a GridION sequencing device to determine the sensitivity, the specificity, and the performance of the method. Fourteen primer pairs were selected to span the full genome. With the monoplex PCR, 12 primer pairs showed good amplification, while a double concentration of two primers was required to improve the amplification. We then performed a multiplex PCR approach by generating two pools comprising non-overlapping primer pairs. The multiplex PCR showed good sensitivity, allowing the amplification of HPV16 with a Ct value below 27 and providing a coverage greater than 99.9% and a sequencing depth greater than 100×. In addition, the specificity of the method was validated by the absence of amplification of other high-risk HPV genotypes compared with HPV16. We set up an amplicon-based approach to characterize the full genome and diversity of the HPV16 (sub)lineages. This approach shows good sensibility and specificity with limited cost, opening new perspectives in the field of whole-genome HPV sequencing. IMPORTANCE HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field. HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836ca2c5fafa3f7dae222db6854bdcbbe566eaa9" target='_blank'>
              An amplicon-based approach for full-genome characterization of HPV16
              </a>
            </td>
          <td>
            I. Malet, Inès Draa, V. Leducq, Fanny Vuong, Pascale Bonnafous, A. Marcelin, V. Calvez, A. Jary
          </td>
          <td>2025-06-09</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38df6b4bc62b7ce1643805d5dd45feb9a54b999" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and Helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although long interspersed element-1 (LINE-1) retrotransposons are poor prognostic indicators of non-small cell lung cancer, the genetic consequences of aberrant LINE-1 expression remain poorly understood. In this study, human bronchial epithelial cells were exposed chronically to 25 µM NiCl₂ to induce malignant transformation and compared to PBS-treated control cells or lung cancer cell lines. A 4 kb intronic LINE-1 insertion into the NACC2 (nucleus accumbens-associated protein 2) locus in nickel-transformed cells was associated with significant reductions in NACC2 mRNA and protein, along with constitutive increases in HDM2 and TP53α mRNAs, and aberrant expression of TP53β and TP53γ mRNAs. Steady-state levels of p53 and RB decreased, while EGFR and LINE-1 ORF1p increased. Silencing of NACC2 in TP53 wildtype NCI-H460 or TP53-null NCI-1299 cell lines replicated many of these changes, with profiles varying as a function of p53 status. Stable overexpression of HDM2 increased LINE-1 ORF1p in cancer cell lines. Human lung adenocarcinomas with wild-type TP53 showed sexually dimorphic profiles, with higher HDM2 levels in females than males, and shifts in LINE-1 ORF1p, p53, and RB that mirrored cancer cell lines. This study identifies the NACC2 locus as a target of LINE-1 retrotransposition and highlights critical interactions with the HDM2/TP53/RB regulatory axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adfffe6caad02c3b19f108090ea32e2ac145796" target='_blank'>
              LINE1 insertion into the NACC2 locus disrupts the HDM2 axis and activates lung oncogenic signaling
              </a>
            </td>
          <td>
            P. Bojang, Yingshan Wang, Kenneth S Ramos
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Glioblastoma is the most aggressive type of brain tumor and is associated with a poor prognosis. First-line treatment is surgical resection followed by radiotherapy and temozolomide-based chemotherapy. However, the duration of treatment with temozolomide is limited due to both its toxicity and the development of drug resistance. The prognostic and predictive factor for response to temozolomide is the methylation status of the MGMT promoter. Indeed, loss of MGMT promoter methylation is a major cause of chemoresistance. However, the development of drug resistance is not only associated with changes in MGMT methylation. The entire epigenome changes and acquires specific properties necessary for tumor progression. Methods To study epigenetic alterations associated with temozolomide exposure, we generated a TMZ-resistant cell model. We investigated epigenetic alterations in the cell model and in a cohort of patients with recurrent glioblastoma using genome-wide methylome approaches (Epic Arrays 850 k). Results We investigated the epigenetic changes associated with temozolomide exposure. Therefore, we generated a TMZ-resistant cell model and studied the epigenetic features acquired after selective temozolomide pressure. Our next step was to investigate the epigenomic differences between primary and recurrent tumors in a small cohort of patients. Finally, we performed a cross-analysis between the epigenetic differences found in TMZ-resistant cells and recurrent glioblastomas to identify common signatures that could be used to guide future resistance-overcoming studies. Conclusions Temozolomide induces significant epigenetic changes in glioblastoma, which may contribute to treatment resistance and increased tumor aggressiveness. The results suggest that further research into DNA methylation changes associated with TMZ resistance is crucial. The use of primary tumor cells in resistance models may help identify strategies to overcome chemoresistance in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06767-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862b534a5409db78fa687bac68e4b9867edb5de0" target='_blank'>
              DNA methylation remodeling in temozolomide resistant recurrent glioblastoma: comparing epigenetic dynamics in vitro and in vivo
              </a>
            </td>
          <td>
            Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Federica Trio, Sara Ferraro, O. Affinito, Alberto De Bellis, Maria Laura Del Basso De Caro, Roberta Visconti, Lorenzo Chiariotti, G. Catapano, R. Della Monica
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Zwilch Kinetochore Protein(ZWILCH) has been reported to prevent cells from prematurely exiting mitosis. However, the underlying mechanisms or involvement of ZWILCH in the tumor immune microenvironment in various cancers remain largely unknown. Methods Generalized dysregulation of ZWILCH was observed through the whole transcriptome analysis in this study. The spatial transcriptome analysis was utilized to identify expressed regions of ZWILCH. Next, cells that mainly expressed ZWILCH in the tumor microenvironment were determined using the single-cell transcriptome analysis. Also, the “cellchat” R package was applied to estimate the effect of ZWILCH on malignant cell communication. Combining multiple analytic approaches including GSEA, GSVA, KEGG enrichment analysis, and Aucell, with TCPA functional protein data, Genome-wide CRISPR screening, potential functions of ZWILCH and the pathways in which ZWILCH participated were thoroughly exploited. Univariate Cox regression analysis calculated the association between ZWILCH and cancer patients’ adverse outcomes. Results ZWILCH is universally highly expressed in tumors. The spatial transcriptome analysis showed that ZWILCH overexpression comes from the tumoral region or mixed tumoral region. At the single-cell level, ZWILCH is chiefly expressed by malignant cells and proliferative T cells. The expression of ZWILCH mRNA is positively correlated with cell proliferation, repair of DNA damage, and cell cycle score. Plenty of metabolic pathways are inhibited in patients with high expression of ZWILCH. Moreover, after ZWILCH knockout, a large number of cancer cell lines are stagnated, inhibited, or died. Additionally, the malignant cells with positive expression of ZWILCH have a stronger ability for cell communication. In short, ZWILCH is meant to be a risk factor for clinical outcomes of multiple tumors. Conclusions ZWILCH is a promising therapeutic target that influences patient prognosis by participating in cell proliferation, cell communication, and reshaping the tumor microenvironment across different cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02981-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f14c1c066485285bab172ec02a8c4271d565b9" target='_blank'>
              Zwilch kinetochore protein affects the prognosis of cancer patients by participating in cell proliferation, enhancing cell communication, and reshaping the tumor microenvironment
              </a>
            </td>
          <td>
            Long Yao, Lianpo Liu, Jinsong Wu, Yunlong Huang, Renquan Zhang, Haoxue Zhang
          </td>
          <td>2025-07-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbc0047fe628bc7d0d1f2c63478bc9274ca9ad" target='_blank'>
              Punctuated mutagenesis promotes multi-step evolutionary adaptation in human cancers
              </a>
            </td>
          <td>
            Christopher J Graser, Wenbo Wu, Cole Christini, Mia Petljak, Franziska Michor
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8a7efbbd0fffc709e336945d36a6f61377b28f" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colon cancer is a highly aggressive solid tumor. Because most studies have focused on the intrinsic oncogenic pathways of tumors, we focused on the relationship between DNA methylation genes in the tumor immune microenvironment and colon cancer prognosis. We download RNA-seq data from the TCGA dataset. DNA methylation genes were scored using the GSVA method, and this score was subjected to GSEA, GO and KEGG enrichment analysis. Then, in vitro experiments examined the effect of silencing LARS2 on the proliferation and apoptosis of HCT116 cells. We obtained 2635 genes, 144 were obtained after single factor screening, and 8 genes were obtained through random forest dimensionality reduction. They are LARS2, TEX2, BRIP1, QSOX2, HOOK1, COX19, NEK4 and STXBP4. Survival analysis indicated that patients with high Riskscore had poor prognosis. There were significant differences in the expression of DNA methylation genes between the two groups with high and low Riskscore. The immune checkpoints of different Riskscore groups include Antigen present, Ligand, Receptor, Co-inhibitor, Co-stimulator, Other, and Cell adhesion. There were significant differences in the expression of genes and DNA integrity checkpoint, histone deacetylation, mitotic DNA damage checkpoint, TGF-beta signaling pathway, Platinum drug resistance and Protein processing in endoplasmic reticlum pathway at the level of Antigen present. Silencing of LARS2 decreased the proliferative capacity and increased the apoptosis rate of HCT116. Our findings suggest that LARS2 methylation could affects colon cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10669-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/146fe9925f0f237535e6dcfdafa1d3f52c10c07d" target='_blank'>
              LARS2 DNA methylation predicts the prognosis in colon cancer
              </a>
            </td>
          <td>
            Yingzhou Fu, Wei Xie, Qifan Ren, Jiao Wu, Ying Long, Ying Liu
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63cbc4e2ddb62951e5459ce050c5384d0e2fa224" target='_blank'>
              LIMD1 Loss Confers a GPX4-Dependent Cell State in Lung Cancer
              </a>
            </td>
          <td>
            Akash Saha, Paul Grevitt, Jair Marques Junior, A. von Kriegsheim, Kunal M. Shah, Zhen Gao, Brent R. Stockwell, Andrew Finch, Barrie Peck, T. Sharp
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aberrant DNA methylation is a hallmark of colorectal cancer (CRC), contributing to tumor progression through the silencing of tumor suppressor genes and activation of oncogenes. Indicaxanthin (IND), a dietary betalain pigment from Opuntia ficus indica, has shown antiproliferative effects in CRC models, yet its epigenetic impact remains unexplored. In this study, we investigated the effects of IND on the methylome of Caco-2 cells using Reduced Representation Bisulfite Sequencing (RRBS). IND induced a global hypermethylation profile, particularly at gene promoters and CpG islands. Among the differentially methylated genes, 60% were protein-coding, and 10% encoded transcription factors, including PAX5 and TFAP4, both hypermethylated at active enhancers. Functional enrichment analysis revealed pathways beyond canonical intestinal functions, suggesting altered cell identity and plasticity. Transcription factor targets (SOX10, NFKB1, AHR, ARNT) were significantly enriched among the affected genes, several of which are involved in transdifferentiation processes. Methylation changes also indicated potential reprogramming toward epithelial cell types from pulmonary or neuroectodermal origin. Moreover, IND induced selective hypomethylation of Alu elements on chromosome 21 and hypermethylation of rDNA loci, hinting at suppressed ribosomal biogenesis. Overall, these findings highlight the epigenetic remodeling potential of IND and its possible role in modulating cell fate and metabolism in CRC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aad87b105620f84af351d2656d30853c48a91d89" target='_blank'>
              Epigenetic Remodeling of Regulatory Regions by Indicaxanthin Suggests a Shift in Cell Identity Programs in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            M. Ragusa, C. Gentile, Aldo Nicosia, Salvatore Costa, Sara Volpes, Laura Greco, F. Naselli, Fabio Caradonna
          </td>
          <td>2025-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c33305648009e1896ce514a1bce98da6447d96" target='_blank'>
              Epigenetic dysregulation in cancer: mechanisms, diagnostic biomarkers and therapeutic strategies.
              </a>
            </td>
          <td>
            Komal Imran, Muhammad Javed Iqbal, Muhammad Mushtaq Ahmed, Ammara Khalid, H. Cortés, O. Reyes-Hernández, Gabriela Figueroa-González, G. Leyva-Gómez, L. Falzone, Massimo Libra, Federica Longo, Javad Sharifi‐Rad, D. Calina
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Estrogen receptor 1 (ESR1; also known as ERα, encoded by ESR1 gene) is the driving transcription factor in breast cancer development and progression. ESR1 genomic action is thought to operate under tight epigenetic control, with its chromatin binding and subsequent transcriptional output heavily reliant on the pioneer transcription factor FOXA1, which renders chromatin accessible for ESR1 binding. However, the exact contribution of the epigenome to selective enhancer activation by ESR1 remains to be fully elucidated. To address this, we employ a massively parallel reporter assay to profile 7576 individual ESR1 binding sites for hormone responsiveness. Only a minority of ESR1-occupied enhancers exhibit hormone-induced activity. These findings are confirmed by genomic data in situ, indicating that enhancer activation within a chromatinized context is robustly captured in a plasmid-based reporter assay. In silico integration of our findings with publicly available functional genomics data sets from breast cancer cell lines and tumor samples reveal distinct transcription complex compositions, 3D genome contexts, and regulatory dynamics associated with different classes of ESR1 binding sites. Overall, our results establish a comprehensive framework to highlight and elucidate the molecular basis underlying ESR1 genomic heterogeneity and its contribution to breast cancer biology and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d35cc452fcf0927d4ead83569deaccc08af058d" target='_blank'>
              Comprehensive functional annotation of ESR1-driven enhancers in breast cancer reveals hierarchical activity independent of genomic and epigenomic contexts.
              </a>
            </td>
          <td>
            Y. Zekri, Sebastian Gregoricchio, Elif Yapıcı, C. Huang, Tunc Morova, Umut Berkay Altıntaş, Gozde Korkmaz, Nathan A. Lack, W. Zwart
          </td>
          <td>2025-06-10</td>
          <td>Genome research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Medulloblastoma comprises four principal molecular disease groups and their component subgroups, each with distinct molecular and clinical features. Group assignment is currently achieved diagnostically using Illumina DNA methylation microarray. Whole-genome sequencing (WGS) capacity is rapidly expanding in the clinical setting and the development of platform-independent, sequence-based assays of molecular group offers significant potential. Specifically, whole-genome bisulfite sequencing (WGBS) enables assessment of genome-wide methylation status at single-base resolution, however its routine application has been limited by high DNA input requirements, cost, and a lack of pipelines tailored to more rapidly-acquired and cost-effective low-depth (< 10x) sequencing data. We utilised WGBS data for 69 medulloblastomas, comprising 35 in-house low-depth (~ 10x) and 34 publicly available high-depth (~ 30x) samples, alongside cerebellar controls (n = 8), all with matched DNA methylation microarray data. We assessed quality (QC) and imputation approaches using low-pass WGBS data, assessed inter-platform correlation and identified molecular groups and subgroups by directly integrating matched/associated loci from WGBS sample data with the MNP classifier probeset. We further assessed and optimised reference-free aneuploidy detection using low-pass WGBS and assessed concordance with microarray-derived calls. We developed and optimised pipelines for processing, QC, and analysis of low-pass WGBS data, suitable for routine molecular subgrouping and reference-free aneuploidy assessment. We demonstrate that low-pass WGBS data can (i) be integrated into existing array-trained models with high assignment probabilities for both principal molecular groups (97% concordance) and molecular subgroups (94.2% concordance), and (ii) detect clinically relevant focal copy number changes, including SNCAIP, with greater sensitivity than microarray approaches. Low-pass WGBS performs equivalently to array-based methods at comparable cost. Finally, its ascertainment of the full methylome enables elucidation of additional biological complexity and inter-tumoural heterogeneity that has hitherto been inaccessible. These findings provide proof-of-concept for clinical adoption of low-pass WGBS, applied using standard WGS technology. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02049-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe3d3a3f742b6178900eb1bb44957ac2eaf7cd1" target='_blank'>
              Robust molecular subgrouping and reference-free aneuploidy detection in medulloblastoma using low-depth whole genome bisulfite sequencing
              </a>
            </td>
          <td>
            Dean Thompson, Jemma Castle, M. Sill, S. Pfister, Simon Bailey, Debbie Hicks, Steven C. Clifford, E. Schwalbe
          </td>
          <td>2025-06-24</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bedd60872e779fcb4da0741c90e0eefb4d2f29f" target='_blank'>
              Unique molecular identifier-based single strand mitochondrial DNA sequencing for rare substitution mutations
              </a>
            </td>
          <td>
            Z. Li, Rachel Dunn, Loïc C. Caloren, Adam S. Ziada, Hailey Chapman, Izabelle Gadawska, H. Côté
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endometrial carcinoma (EC), the most common gynecologic cancer type in developed countries, encompasses four molecular subtypes (POLEmut, MMRd, p53abn, and NSMP) that have prognostic values and guide treatment decisions. Additionally, dual loss of ARID1A and ARID1B (referred to as ARID1A/B) characterizes a significant portion of dedifferentiated/undifferentiated EC (DD/UDEC), a rare but highly aggressive subtype of EC. To advance the translational research for ECs, we analyzed the genomic features of a panel of 39 EC cell lines, leading to the identification of cell lines representing each of these EC molecular subtype. Histologic and immunohistochemical analyses of xenografted tumors from these cell lines confirmed their resemblance of cognate primary EC molecular subtypes. Further investigation of the publicly available genome-wide CRISPR screen data for EC cell lines identified multiple specific genetic dependencies in MMRd, p53abn, and ARID1A/B-dual deficient EC cell lines. Particularly, ARID1A/B-dual deficient DD/UDEC cells selectively rely on mitochondrial oxidative phosphorylation in vitro and in vivo. Therefore, through molecular subtyping of EC cell lines and subsequent characterization of molecular subtype-specific genetic dependencies, our study provides a framework that guides the utility of the EC cell line models for accelerating translational research in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1f3b25336fe4feba408a071575d0aa7bd9be8e" target='_blank'>
              Molecular subtyping of endometrial carcinoma cell lines uncovers subtype-specific targetable vulnerabilities
              </a>
            </td>
          <td>
            Eunice Li, Rebecca Ho, Ran Tao, Yannes Wai Yan Choi, Chae Young Shin, Shary Chen, Bengul Gokbayrak, J. Senz, Betty Yao, Liam Johnston, Spencer D Martin, Eric Yang, Mark S Carey, Bryan T. Hennessy, D. Huntsman, Ramon I Klein Geltink, Lynn Hoang, Yemin Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Often, neoadjuvant therapy, which relies on the induction of double-strand breaks (DSBs), is used prior to surgery to shrink tumors by inducing cancer cell apoptosis. However, recent studies have suggested that this treatment may also induce a fluctuating state between senescence and stemness in PA-1 embryonal carcinoma cells, potentially affecting therapeutic outcomes. Thus, the respective epigenetic pathways are up or downregulated over a time period of days. These fluctuations go hand in hand with changes in spatial DNA organization. Methods: By means of Single-Molecule Localization Microscopy in combination with mathematical evaluation tools for pointillist data sets, we investigated the organization of euchromatin and heterochromatin at the nanoscale on the third and fifth day after etoposide treatment. Results: Using fluorescently labeled antibodies against H3K9me3 (heterochromatin tri-methylation sites) and H3K4me3 (euchromatin tri-methylation sites), we found that the induction of DSBs led to the de-condensation of heterochromatin and compaction of euchromatin, with a peak effect on day 3 after the treatment. On day 3, we also observed the co-localization of euchromatin and heterochromatin, which have marks that usually occur in exclusive low-overlapping network-like compartments. The evaluation of the SMLM data using topological tools (persistent homology and persistent imaging) and principal component analysis, as well as the confocal microscopy analysis of H3K9me3- and H3K4me3-stained PA-1 cells, supported the findings that distinct shifts in euchromatin and heterochromatin organization took place in a subpopulation of these cells during the days after the treatment. Furthermore, by means of flow cytometry, it was shown that the rearrangements in chromatin organization coincided with the simultaneous upregulation of the stemness promotors OCT4A and SOX2 and senescence promotors p21Cip1 and p27. Conclusions: Our findings suggest potential applications to improve cancer therapy by inhibiting chromatin remodeling and preventing therapy-induced senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3f66309c268ad41eb3fdc8156d8889c120362" target='_blank'>
              Chemotherapy (Etoposide)-Induced Intermingling of Heterochromatin and Euchromatin Compartments in Senescent PA-1 Embryonal Carcinoma Cells
              </a>
            </td>
          <td>
            Marc Bayer, Jaroslava Zajakina, Myriam Schäfer, K. Salmina, Fēlikss Rūmnieks, J. Jansons, F. Bestvater, R. Kurg, J. Erenpreisa, Michael Hausmann
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d072dee701b15034f00be3c11a965c7ef7b90a4" target='_blank'>
              Clonal memory of cell division in humans diverges between healthy haematopoiesis and acute myeloid leukaemia
              </a>
            </td>
          <td>
            A. Donada, G. Hermange, T. Tocci, A. Midoun, G. Prevedello, L. Hadj Abed, D. Dupré, W. Sun, I. Milo, S. Tenreira Bento, C. Pospori, A. Innes, C. Willekens, J. Vargaftig, D. Michonneau, C. Lo Celso, N. Servant, KR Duffy, H. Isambert, PH Cournède, L. Laplane, L. Perie
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Oncogene activation through DNA amplification or overexpression plays a critical role in cancer initiation and progression. Chromosome 17 harbors numerous cancer-associated genetic anomalies, some of which are strongly linked to poor prognosis in breast cancer patients. The FOXK2 gene, located on 17q25, encodes a transcription factor with a forkhead DNA binding domain. Integrative analyses of publicly available genomic datasets reveal that FOXK2 is frequently amplified and overexpressed in breast cancer, correlating with poor overall survival. FOXK2 knockdown via shRNA significantly inhibited the proliferation, migration, and anchorage-independent growth of breast cancer cells, causing G0/G1 cell cycle arrest. The C-terminal nuclear localization signal (NLS) is crucial for FOXK2's oncological function. Additionally, FOXK2 knockdown in MCF7 breast cancer cells delayed tumor growth in a xenograft mouse model. Inhibiting FOXK2 expression also sensitized breast cancer cells to frontline chemotherapy agents, including doxorubicin, 5-fluorouracil, and etoposide in vitro. Co-overexpression of FOXK2 and PI3KCA with oncogenic mutations (E545K or H1047R) induced the transformation of non-tumorigenic MCF-10A cells, suggesting that FOXK2 acts as an oncogene in breast cancer and contributes to PI3KCA-driven tumorigenesis. CCNE2, PDK1, and ESR1 were identified as direct transcriptional targets of FOXK2 in MCF-7 cells. Moreover, co-inhibition of FOXK2 function, either through knockdown or inhibition of its transcriptional targets (CCNE2/CDK2 by Dinaciclib or PDK1 by DCA) and PI3KCA by alpelisib, had synergistic anti-tumor effects in breast cancer cells with FOXK2 overexpression and PI3KCA oncogenic mutations. FOXK2 functional inhibition by the CDK2 inhibitor Dinaciclib enhanced the tumor-suppressive effect of alpelisib in a MCF7 breast cancer xenograft mouse model. These findings provide compelling evidence that FOXK2 plays an oncogenic role in breast tumorigenesis. The combination of PIK3CA and CCNE2/CDK2 inhibition by alpelisib and Dinaciclib offers a potential therapeutic strategy for breast cancer with FOXK2 overexpression and PIK3CA oncogenic mutation.
 Citation Format: Hong Zhang, Yang Yu, Weng-Ming Cao, Feng Cheng, Edaise da silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang. Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy? [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-24.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caaa74a368ec1f757b26670f47abf7954355f919" target='_blank'>
              Abstract P5-02-24: Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy?
              </a>
            </td>
          <td>
            Hong Zhang, Yang Yu, Wen-Ming Cao, Feng Cheng, E. D. da Silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Long interspersed nucleotide element 1 (LINE1), the most abundant repetitive element in the human genome, plays a crucial role in genomic instability. Aberrant LINE1 activation, primarily regulated by DNA methylation, is a hallmark of cancer. Non-small cell lung cancer (NSCLC), the most prevalent form of lung cancer worldwide, continues to pose significant challenges due to the invasiveness, high cost, and susceptibility to false positives of current diagnostic methods, as well as the emergence of treatment resistance. This review highlights the potential of LINE1 methylation as a biomarker for NSCLC, offering novel insights into its role in diagnosis, prognosis, and therapeutic strategies. Recent studies uncovered that LINE1 hypomethylation was strongly associated with poor overall survival, suggesting its utility as both a prognostic marker and a therapeutic target. However, further research is required to elucidate its precise regulatory mechanisms in LINE1 retrotransposition and to evaluate its potential as a non-invasive biomarker for improving NSCLC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4219f213fb15eaa64e362c434180801a85ab734e" target='_blank'>
              Long interspersed nuclear element 1 methylation in non-small cell lung cancer: implications for diagnosis, prognosis, and therapeutic targeting
              </a>
            </td>
          <td>
            Dileesha Prabani Wanasundara Arachchillage, Wanvisa Udomsinprasert
          </td>
          <td>2025-07-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found that oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS–expressing PDAC cells revealed differential levels of several nuclear envelope–associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS–driven PDAC model resulted in increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provide evidence of a potentially novel mechanism underlying nuclear size regulation and its effect in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Pancreatic cancer is known for its lethal condition, with most cases being diagnosed at advanced stage. Recently, liquid biopsy has emerged as a promising tool in cancer detection. Here we develop both an early detection model and a prognostic model for pancreatic cancer using cell-free DNA (cfDNA) end motif, fragmentation, nucleosome footprint (NF), and copy number alteration (CNA) features from plasma cfDNA. A total of 975 individuals were enrolled in our study. We developed an integrated model that demonstrated superior performance in distinguishing patients with early-stage pancreatic cancer from non-cancer controls. Moreover, we find that cfDNA features are associated with prognostic outcomes among pancreatic cancer patients. In this study, a cfDNA-based liquid biopsy signature is established for the early detection and prognostic prediction of pancreatic cancer. CfDNA may become a valuable tool for enhancing early diagnosis and prognosis assessment in this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2b38ea3144ac464585d901d0e6435daee2918d3" target='_blank'>
              Cell-free DNA testing for the detection and prognosis prediction of pancreatic cancer
              </a>
            </td>
          <td>
            Jianmin Wu, Xiongfei Xu, Qingzheng Zhang, Peilong Li, Tong Wu, Shiwei Guo, Lutao Du, Dongdong Xue, S. Shen, Fuming Sun, Ji Hu, Lu Zheng, Xuan Wu, Jian Bai, Yin Wang, Lin Wu, Wei-wen Liu, Hongyang Wang, Gang Jin, Lei Chen
          </td>
          <td>2025-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b94cf63206b221ede960c719c7b5d998ebadf2b" target='_blank'>
              Specific TP53 mutations impair the recruitment of 53BP1 to DNA double-strand breaks underlying the mechanism of radioresistance.
              </a>
            </td>
          <td>
            Paolo Fagherazzi, L. Stixová, E. Bártová
          </td>
          <td>2025-07-14</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for removing bulky DNA damage, such as pyrimidine dimers induced by ultraviolet (UV) light. The NER pathway excises the damaged strand through incisions at the 5’ and 3’ ends of the damage, with the 5’ incision catalyzed by the XPF-ERCC1 endonuclease complex. Here, we identify an XPF ortholog in Trypanosoma brucei, the causative agent of human African trypanosomiasis (sleeping sickness). XPF-deficient parasites exhibit hypersensitivity to UV irradiation and a slower rate of DNA damage repair. Consistent with its role in DNA repair, XPF localizes to the nucleus, associating with nucleoplasmic and nucleolar regions. Additionally, we demonstrate that TbXPF protects against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C, respectively. The presence of a functional NER pathway in trypanosomes suggests that these organisms are susceptible to replication- and transcription-blocking DNA damage in vivo. Under genotoxic stress, genome stability and parasite survival may heavily rely on DNA repair systems such as NER which, for this reason, could be an effective target for chemotherapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdd4ff3d6a3681f1e698b78ba5f4c131040ae39" target='_blank'>
              Role of the nucleotide excision repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, Laura Montosa-Hidalgo, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="STUDY QUESTION
Are segmental aneuploidies identified in human embryos more likely to occur within known fragile sites of the genome?


SUMMARY ANSWER
Segmental breaks in the autosomes of human preimplantation embryos occur more frequently in known fragile areas of the genome.


WHAT IS KNOWN ALREADY
Fragile sites represent specific loci in the genome characterized by inhibition of DNA synthesis when exposed to known inhibitors and are particularly sensitive to replication stress and instability.


STUDY DESIGN, SIZE, DURATION
This was a retrospective analysis of single nucleotide polymorphism (SNP) array-based preimplantation genetic testing data from biopsies performed on 2066 human blastocysts in 98 assisted reproduction laboratories around the world from September 2019 to January 2023.


PARTICIPANTS/MATERIALS, SETTING, METHODS
This multicenter study included eligible patients undergoing IVF with preimplantation genetic testing (PGT), in which at least one embryo was diagnosed with a segmental aneuploidy. The mean maternal age was 36.4 years (SD 4.1), ranging from 25 to 44 years. These samples were processed on high-density SNP arrays. Chromosome level copy number and B allele frequency (BAF) plots from these embryos were used to determine segmental aneuploidy breakpoints. Known fragile sites catalogued by the HumCFS database were used for correlation analyses.


MAIN RESULTS AND THE ROLE OF CHANCE
Overall, a side-by-side pairing of observed breakpoints and known fragile sites demonstrated a strong concordance (r = 0.81, 95% CI [0.6, 0.92]). A chi-square test for independence for stratified groups showed a highly significant correlation between all observed breakpoints and known fragile sites (597 expected vs. 848 observed; P < 0.001) and for telomeric breaks alone (521 expected vs. 784 observed; P < 0.001). Observed interstitial breaks alone were not correlated to expected breakpoints (75 expected vs. 64 observed; P > 0.05).


LIMITATIONS, REASONS FOR CAUTION
These findings should be interpreted with caution, as limitations in genomic resolution may bias detection and classification of smaller segmental aneuploidies. Additionally, this study touched upon the distribution of meiotic to mitotic breakpoints in human blastocysts as they relate to known fragile sites. Since meiotic aneuploidies increase with advanced maternal age and many IVF patients undergoing PGT-A testing fall in this category, a sampling bias should be considered for this specific metric.


WIDER IMPLICATIONS OF THE FINDINGS
Demonstrating that segmental aneuploidies significantly correlate with known fragile sites highly susceptible to replication stress offers insight into the origin of subchromosomal imbalances and hints at the influence of stressors on reproductive success.


STUDY FUNDING/COMPETING INTEREST(S)
This study received no external funding and was fully supported by the participating authors and their affiliated institutions. The authors declare no conflicts of interest related to this study.


TRIAL REGISTRATION NUMBER
N/A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f488ce17d9309a0718d17d57c1fee173909b44" target='_blank'>
              Blastocyst segmental aneuploidy breakpoints are highly correlated with human genome fragile sites.
              </a>
            </td>
          <td>
            Elaine de Quadros, Jia Xu, Nathan Treff, Diego Marin, Arielle Freedman, C. Milevski, Kathleen Miller, Minglei Bian
          </td>
          <td>2025-07-30</td>
          <td>Human reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Epigenetic changes are implicated in development, repair, and physiology of postnatal skeletal muscle (SkM). We generated methylomes for human myoblasts (SkM progenitor cells) and determined myoblast differentially methylated regions (DMRs) for comparison to the epigenomics and transcriptomics of diverse cell types. Analyses were from global genomic and single-gene perspectives and included reporter gene assays. One atypical finding was the association of promoter-adjacent hypermethylation in myoblasts with transcription turn-on, but at downmodulated levels, for certain genes (e.g., SIM2 and TWIST1). In contrast, brain-specific OLIG2 was in repressed chromatin and silent in most cell types but linked to hypermethylated DMRs specifically in myoblasts. The OLIG2-linked DMRs might be needed because of the overlapping or nearby binding of myogenic differentiation protein 1 (MYOD). We found genome-wide overlap of DMRs with MYOD or CCCTC-binding factor (CTCF) binding sites in myoblasts that is consistent with the importance of MYOD, as well as CTCF, in organizing myoblast transcription-enhancing chromatin interactions. We also observed some gene upregulation correlated with a special association of regional DNA hypomethylation with H3K36me3, H3K27ac, and H3K4me1 enrichment. Our study highlights unusual relationships between epigenetics and gene expression that illustrate the interplay between DNA methylation and chromatin epigenetics in the regulation of transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a2af4d92c56684b81b344205f48b6f58edf37d" target='_blank'>
              The myoblast methylome: multiple types of associations with chromatin and transcription
              </a>
            </td>
          <td>
            Sagnik Sen, M. Lacey, Carl Baribault, V. C. Ponnaluri, P. Esteve, Kenneth C Ehrlich, Mia Meletta, S. Pradhan, Melanie Ehrlich
          </td>
          <td>2025-06-11</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Multicellular organisms arise from a single genome template in the zygote, necessitating the cells of the developing embryo to up- and downregulate specific genes to establish and maintain their identity. This template is maintained, propagated, and interpreted as chromatin, a polymer of nucleic acids and associated structural and regulatory proteins. Recent genome-wide surveys documented a wealth of disease-associated mutations in chromatin factors, indicating their fundamental significance and potential for therapeutic targeting. However, chromatin factors exist in a complex balance, with a single deficiency often leading to pleiotropic downstream effects. Here, we review the mechanisms of chromatin regulation and partitioning, highlighting examples of how these processes are altered in human diseases. We argue that loss of chromatin fidelity, both locally at specific genes and regulatory elements, and globally at the megabase-scale, contributes to many pathological states and may thus represent an intriguing target for corrective interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b3c6ccb1965f22efefabec61c021483fcae985" target='_blank'>
              Aere Perennius: How Chromatin Fidelity is Maintained and Lost in Disease.
              </a>
            </td>
          <td>
            Dustin Fetch, Amina Jumamyradova, Cameron Chapa, Yong Ge, Mansour Mohamadzadeh, Alexey A Soshnev
          </td>
          <td>2025-07-22</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The mammalian Tumor Susceptibility Gene 101 (TSG101) encodes a protein with diverse functions that control the proliferation and survival of cells, but its role in malignant transformation and cancer development has remained enigmatic. Methods To study the pro-tumorigenic functions of TSG101, we developed a bi-transgenic mouse model that expresses exogenous TSG101 along with a luciferase reporter in a ligand-controlled manner in the mammary gland epithelium. We performed a comprehensive histopathologic, biochemical, and molecular characterization of ductal hyperplasia and mammary tumors. Unsupervised hierarchical clustering based on 1,723 intrinsic genes of ten TSG101-overexpressing cancers alongside 251 tissue samples representing 31 reference mammary tumor models and normal mammary glands was conducted. Results Females overexpressing TSG101 develop ductal hyperplasia, adenomyoepitheliomas, and palpable adenosquamous carcinomas at an average latency of approximately ten months. These metaplastic mammary tumors are comprised of transforming basal and luminal epithelial cells. Using a GFP reporter strain to monitor the transgene activation at the single-cell level, we determined that the epithelial heterogeneity within transforming ducts and ensuing carcinomas originated from the luminal epithelium. At the molecular level, TSG101-induced mammary tumors are triple-negative and exhibit gene expression signatures of Wnt and inflammatory cytokine signaling, which are key regulators of epithelial cell fate. The ligand-controlled downregulation of exogenous TSG101 in established carcinomas led to tumor regression. We demonstrated that the TSG101-mediated activation of PI3K/AKT signaling, as well as upregulation of Cyclin D1 and MDM2, are dependent on the perpetual expression of the TSG101 oncoprotein. Conclusions The collective findings of this study provide in vivo evidence that TSG101 possesses pro-tumorigenic properties that extend to cancer progression and maintenance, suggesting that this protein could be a rational molecular target to prevent and treat a subset of mammary tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02007-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca345fd462ac7f96e7ae408ea2e3598885c80b06" target='_blank'>
              Overexpression of TSG101 causes the development of adenosquamous mammary carcinoma
              </a>
            </td>
          <td>
            Rayane Dennaoui, Patrick D. Rädler, Madison N. Wicker, Kerry Vistisen, R. Ferraiuolo, A. Triplett, Hridaya Shrestha, Tessa A. Liner, Karoline C. Manthey, Hallgeir Rui, R. Cardiff, Teresa M. Gunn, C. Perou, Kay-Uwe Wagner
          </td>
          <td>2025-07-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="Background Cancer remains one of the leading causes of death worldwide. The lack of effective early diagnostic markers and comprehensive treatment strategies continues to limit progress in clinical outcomes. Although targeted therapies have brought some improvement, the need for more reliable and efficient treatment options persists. SEC13, a protein involved in cellular energy metabolism and nucleocytoplasmic transport, has been suggested to play a role in tumour progression. This study aims to explore the potential role of SEC13 across various types of cancer, with a particular focus on its value as a diagnostic and prognostic marker, using Mendelian randomization analysis. Methods In this study, we conducted a comprehensive analysis of SEC13 expression using data from TCGA and GTEx, and explored associated pathways through gene set enrichment analysis. Single-cell transcriptomic data were integrated to investigate the cell-type-specific expression of SEC13 within the tumour microenvironment. In parallel, we examined the regulatory potential of the traditional Chinese compound PC using publicly available gene expression profiles. Causal inference between SEC13 and cancer susceptibility was assessed through Mendelian randomization and data from the BEST database. Results SEC13 was found to be significantly upregulated in several cancer types. Higher expression levels appeared to be associated with more advanced tumour stages and poorer survival outcomes. However, Mendelian randomization did not provide clear evidence for a direct causal link between SEC13 and any of the six cancers studied (P-IVW > 0.05). Interestingly, a negative association was observed between SEC13 expression and chemotherapy sensitivity, which may point to a possible role in drug resistance. Additionally, single-cell analysis revealed marked expression of SEC13 in fibroblasts, tumour-associated macrophages, and endothelial cells, suggesting a possible role in immune modulation and microenvironmental remodelling. Conclusions These findings highlight the prognostic relevance of SEC13 across multiple cancers and suggest that it may be regulated by PC, potentially contributing to anti-tumour activity. While Mendelian randomization did not reveal a definitive causal link, the collective evidence warrants further investigation and external validation, particularly regarding SEC13 as a therapeutic target and its possible pharmacological modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd2edf41e62fcdfde093922f8d81e3e8022cd49" target='_blank'>
              Expression and prognostic value of SEC13 across multiple tumour types and its association with the regulation by Pulsatilla chinensis: a multi-omics and Mendelian Randomization Study
              </a>
            </td>
          <td>
            Huaqing Leng, Xin Luan, Ming Li
          </td>
          <td>2025-07-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background In-solution hybridization enrichment of genetic markers is a method of choice in paleogenomic studies, where the DNA of interest is generally heavily fragmented and contaminated with environmental DNA, and where the retrieval of genetic data comparable between individuals is challenging. Here, we benchmark the commercial “Twist Ancient DNA” reagent from Twist Biosciences using sequencing libraries from ancient human samples of diverse demographic origin with low to high endogenous DNA content (0.1–44%). For each library, we tested one and two rounds of enrichment and assessed performance compared to deep shotgun sequencing. Results We find that the “Twist Ancient DNA” assay provides robust enrichment of approximately 1.2M target SNPs without introducing allelic bias that may interfere with downstream population genetics analyses. Additionally, we show that pooling up to 4 sequencing libraries and performing two rounds of enrichment is both reliable and cost-effective for libraries with less than 27% endogenous DNA content. Above 38% endogenous content, a maximum of one round of enrichment is recommended for cost-effectiveness and to preserve library complexity. Conclusions In conclusion, we provide researchers in the field of human paleogenomics with a comprehensive understanding of the strengths and limitations of different sequencing and enrichment strategies, and our results offer practical guidance for optimizing experimental protocols. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03622-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c477c495125e3cd66f843708e054b0141348cbd8" target='_blank'>
              Optimized in-solution enrichment of over a million ancient human SNPs
              </a>
            </td>
          <td>
            Roberta Davidson, Xavier Roca-Rada, Shyamsundar Ravishankar, Leonard Taufik, Christian Haarkötter, Evelyn Collen, Matthew P. Williams, Peter Webb, M. I. Mahmud, Erlin Novita, Idje Djami, Gludhug A. Purnomo, Cristina Santos, A. Malgosa, Linda R. Manzanilla, Ana Maria Silva, Sofia Tereso, Vítor Matos, Pedro C. Carvalho, Teresa Fernandes, A. Maurer, João C. Teixeira, Ray Tobler, Lars Fehren-Schmitz, B. Llamas
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The phenotype of human placental extravillous trophoblast (EVT) at the end of pregnancy reflects both differentiation from villous cytotrophoblast (CTB) and later gestational changes, including loss of proliferative and invasive capacity. Invasion abnormalities are central to major obstetric pathologies, including placenta accreta spectrum, early onset preeclampsia, and fetal growth restriction. Characterization of the normal differentiation processes is, thus, essential for the analysis of these pathologies. Our gene expression analysis, employing purified human CTB and EVT cells, demonstrates a mechanism similar to the epithelial–mesenchymal transition (EMT), which underlies CTB–EVT differentiation. In parallel, DNA methylation profiling shows that CTB cells, already hypomethylated relative to non-trophoblast cell lineages, show further genome-wide hypomethylation in the transition to EVT. A small subgroup of genes undergoes gains of methylation (GOM), associated with differential gene expression (DE). Prominent in this GOM-DE group are genes involved in epithelial–mesenchymal plasticity (EMP). An exemplar is the transcription factor RUNX1, for which we demonstrate a functional role in regulating the migratory and invasive capacities of trophoblast cells. This analysis highlights epigenetically regulated genes acting to underpin the epithelial–mesenchymal plasticity characteristic of human trophoblast differentiation. Identification of these elements provides important information for the obstetric disorders in which these processes are dysregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39441ea778e73787eba121d4ae9fb690f494341" target='_blank'>
              Epigenetic Changes Regulating Epithelial–Mesenchymal Plasticity in Human Trophoblast Differentiation
              </a>
            </td>
          <td>
            William E. Ackerman IV, Maurício M. Rigo, S. DaSilva-Arnold, Catherine Do, Mariam Tariq, Martha Salas, Angelica Castano, Stacy Zamudio, Benjamin Tycko, N. Illsley
          </td>
          <td>2025-06-24</td>
          <td>Cells</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy. Methods The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments. Results This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion. Conclusion This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Amplification of human chromosome 20q has been reported as the most frequently recurring genetic abnormality associated with large scale changes in mRNA and protein levels in sporadic colorectal carcinomas. While some studies have found 20q amplification to be consistent between primary and metastatic samples from the same patient with a role in the development of metastasis and worse patient prognosis, others have reported association with improved overall survival for a subset of these patients with colorectal cancer (CRC). To fine map the Minimal Common Regions (MCRs) of amplification on chromosome 20q and identify the candidate genes playing roles in progression of the disease, microarray comparative genomic hybridization analyses of two in vitro cultured CRC liver metastasis cell line model systems was utilized. Microarray expression analysis led to the identification of a candidate gene signature comprising of four genes, BMP7, DNMT3B, UBE2C and YWHAB, residing in the MCRs that were over expressed in CRC cells. By validating our results in a training set of 23 adenocarcinomas (tumors) and five adenomas (polyps) using reverse transcription-quantitative PCR, as well as analyses of two larger colorectal cancer test data sets derived from 195 The Cancer Genome Atlas and 182 MD Anderson Cancer Center patients with colorectal adenocarcinoma patients, this gene signature was ascertained to be associated with lymph node spread and/or distant metastasis (P<0.05). Previously reported functional studies of the gene signature indicated their involvement in inflammatory and immune response pathways driving CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea2107086249d394733ab4fbf6c82be7552a16" target='_blank'>
              Chromosome 20q gene signature associated with colorectal cancer progression
              </a>
            </td>
          <td>
            Jennifer Carter Jones, A. Hegde, Yu-Jing Huang, G. Manyam, Vibhuti Srivastava, J. Song, Yulan Cheng, Ralf Krahe, Warapen Treekitkarnmongkol, Stephen J. Meltzer, S. Kopetz, Stanley R Hamilton, Hiroshi Katayama, Subrata Sen
          </td>
          <td>2025-07-18</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Post-transcriptional modifications on mRNA are crucial for mRNA fate and function. The current lack of a comprehensive method for high-coverage and sensitive quantitative analysis of mRNA modifications significantly limits the discovery of new mRNA modifications and understanding mRNA modifications’ occurrence, dynamics and function. Here, a highly sensitive, high-throughput and robust LC-MS/MS-based technique, mRQuant, was developed to simultaneously detect and quantify 84 modified ribonucleosides in cellular mRNA. Using mRQuant, we quantified 32–34 modified ribonucleosides across several human cancer and non-cancer cell lines and uncovered cancer- and cancer type-specific signatures. Analyses of cisplatin- and paclitaxel-treated HeLa cells and drug-resistant variants revealed several drug resistance-associated modifications. Among them, m1A exhibited significant differences across multiple cell types and between cancerous and non-cancerous cells. Knocking down mRNA m1A writer or eraser protein resulted in altered cell viability, cell cycle and apoptosis in HeLa cells, suggesting a role of mRNA m1A in cancer. Transcriptomic and proteomic analyses further revealed the molecular mechanisms underlying the phenotypic variation. Supplementary information The online version contains supplementary material available at 10.1007/s00018-025-05805-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b614524ca44941ff0254c44b4d860cba08475a2" target='_blank'>
              Quantifying the mRNA epitranscriptome reveals epitranscriptome signatures and roles in cancer
              </a>
            </td>
          <td>
            Ying Feng, Xiaoli He, Mingxin Guo, Ying Tang, Guantong Qi, Qian Huang, Wenran Ma, Hong Chen, Yifan Qin, Ruiqi Li, Jin Wang, Yu Liu
          </td>
          <td>2025-07-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Sporadic colorectal cancer (CRC), the third leading cause of cancer-related death globally, arises through a continuum from normal tissue to adenomas, progressing from low-grade (LGD) to high-grade dysplasia (HGD); yet, the early epigenetic drivers of this transition remain unclear. To investigate these events, we profiled LGD and HGD adenomas using EM-seq, and identified a consensus differential methylation signature (DMS) of 626 regions through two independent bioinformatics pipelines. This signature effectively distinguished LGD from HGD in both tissue and plasma-derived cell-free DNA (cfDNA), highlighting specific methylation patterns. Functional annotation indicated enrichment for regulatory elements associated with transcription factor activity and cell signaling. Applying the DMS to the TCGA CRC dataset revealed three tumor subtypes with increasing hypermethylation and one normal cluster. The most hypermethylated subtype exhibited poor survival, high mutation burden, and disrupted transcriptional networks. While overlapping with classical CpG Island Methylator Phenotype (CIMP) categories, the DMS captured a broader spectrum of methylation alterations. These findings suggest that the DMS captures functionally relevant, antecedent epigenetic alterations in CRC progression, enabling the robust stratification of dysplasia severity and tumor subtypes. This signature holds promise for enhancing preclinical detection and molecular classification, and warrants further evaluation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b0ef917c9876876a5f66fb35ed8c368fc721c" target='_blank'>
              Multiomics Signature Reveals Network Regulatory Mechanisms in a CRC Continuum
              </a>
            </td>
          <td>
            Juan Carlos Higareda-Almaraz, Francesco Mattia Mancuso, P. Canal-Noguer, Kristi Kruusmaa, Arianna Bertossi
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The impact of MYC point mutations on cancer development and progression is poorly explored, particularly compared with other MYC genetic alterations such as translocations and amplifications. MYC point mutations were first observed more than 40 years ago in lymphoid malignancies, and some of these were functionally characterized. In the following decades, only a few studies on MYC point mutations were reported until recently, when analyses of the myriad cancer-related high-throughput sequencing studies brought new life to this research topic and expanded the range of malignancies involved. However, to date, all this information can only be retrieved consulting the specific literature or navigating publicly available databases, and a proper collection and systematic description of these genetic changes is urgently needed. In this review, we run through the steps of the MYC point mutations line of research with a comprehensive illustration of the recurrent variants occurring at the MYC coding, non-coding and regulatory regions in cancer. With this work, we also aim to highlight the current gaps of knowledge to stimulate the research in this field which could ultimately result in the release of its translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d123406ed96af6e46fc4b8cbc92394a082d3e153" target='_blank'>
              MYC Point Mutations in Cancer: A Reboot and a Sequel.
              </a>
            </td>
          <td>
            Davide De Luca, Cristina Munafò, L. Lorenzi, Francesco Cucco
          </td>
          <td>2025-07-24</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nuclear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syndrome in the family, the maternal grandfather had a similar tumor requiring amputation. Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-04-01</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) remains a significant clinical challenge, characterized by its aggressive nature and poor prognosis. Current therapeutic strategies, including targeted therapies, have limitations due to the complex interplay between tumor heterogeneity and the tumor microenvironment (TME). However, the specific contributions of N6-methyladenosine (m6A) methylation to the TME in EC are yet to be fully elucidated. Methods Through comprehensive bioinformatics analyses, a detailed examination of m6A regulators were conducted in EC using datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO). Single-cell RNA sequencing (scRNA-seq) and a consensus clustering algorithm was employed to classify m6A modification patterns and analyze their relationships with immune cell infiltration and clinical outcomes. Additionally, an m6A scoring system was developed based on principal component analysis to assess the prognostic value of identified m6A modification patterns. Results The findings revealed two distinct m6A modification clusters associated with divergent TME characteristics and immune infiltration profiles. Patients exhibiting the immune-inflamed phenotype (m6A cluster B) demonstrated significantly improved survival compared to those with the immune-excluded phenotype (m6A cluster A). Notably, m6A scores correlated positively with immune cell presence and related with adverse prognostic outcomes, indicating their potential as predictive biomarkers for immunotherapy responses. A low m6A score indicated a better response to immunotherapy. Conclusion This study highlights the critical role of m6A methylation in shaping the TME and influencing immune dynamics in EC. The m6A score developed herein provides a novel quantitative tool for predicting tumor behavior and treatment efficacy, paving the way for more personalized immunotherapeutic strategies in clinical practice. This scoring system illustrates a strong correlation of EC with TME immune cell composition, suggesting potential as a biomarker for targeted therapeutic strategies for EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0acdf5e1c85f7477f30b39855d54ee48e97a040c" target='_blank'>
              Cross-talk of m6A methylation modification and the tumor microenvironment composition in esophageal cancer
              </a>
            </td>
          <td>
            Pan Song, Jinmao Ye, Haiyang Zhang, Yishu Li, Ruizhi Cao, Yang Feng, Lei Zhang, Min Sun
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This review examines the increasingly prominent role of mechanics within cancer formation and progression. The extremely varied and contradictory genetic landscape of cancer is in stark contrast to the seemingly universal mechanical characteristics of cancer cells and their tumour microenvironment, and mechanics may be a principal unifying trait of this disease. The tight regulation of innate cell mechanical properties raises the possibility that destabilisation of the cell drives tumour formation in an attempt to restore cell mechanical homeostasis. With losses in cell stiffness more pronounced at the cell nucleus, we hypothesise that destabilisation occurs within the nucleus, likely within the nucleosome. Beyond the mechanical properties of the cell, this compromise to the chromatin structure holds significant repercussions for both genetic and epigenetic regulation, providing scope for significant genetic dysregulation and mutation. However, the nature of such genetic events will be dependent upon the region of mechanical destabilisation; thus, introducing greater variability and heterogeneity to genetic changes. We conclude with the hypothesis that cancer has a mechanical genesis, in which cell nuclear destabilisation functions as the enabling hallmark of cancer. It is theorised that both genetic and structural dysfunction stem from this nuclear destabilisation, driving disease pathology and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/440a5c35e068d9fe1560a5f26a4055f7a6f2a62d" target='_blank'>
              Nuclear destabilisation - a possible genesis of cancer?
              </a>
            </td>
          <td>
            Daniel D Scott, Francesco Bettariga, Marco Ventin, Chris Bishop, Britta Stordal
          </td>
          <td>2025-07-14</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Colon cancer is a highly heterogeneous and most common malignant tumor of the gastrointestinal tract, which is prone to metastasis in advanced stages, leading to a poor prognosis for patients. Methods To gain a comprehensive understanding of genes associated with the prognosis of colon cancer, we identified a gene set (DPP7, TRPM5, ASB6, LINC00634 and GABRD) with potential prognostic value via genome-wide overall survival (OS) analysis. Further, LINC00634 was silenced by constructing small interfering RNAs (siRNAs) in Colo-320 cells. Results Multifactorial Cox regression analysis revealed that LINC00634, TRPM5, age, T and M stages were independent risk factors for the prognosis of colon cancer patients. Nomogram demonstrated that these risk factors had favorable predictive ability for 1-, 3- and 5-year OS in colon cancer patients. In addition, high expression of LINC00634 had a worse prognosis for colon cancer patients compared to TRPM5. In vitro experiments showed that silencing LINC00634 significantly inhibited cell proliferation and migration, and promoted cell apoptosis. Finally, immune infiltration analysis revealed that colon cancer patients with high expression of LINC00634 had lower infiltration levels of myeloid dendritic cells (mDCs) and CD8+ T cells in their tumor microenvironment (TME). Conclusion Our study found that LINC00634 can be used as a novel prognostic and immunological marker for colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91e56040af6b373172ce02c6431230dd7a0e5a3" target='_blank'>
              LINC00634 is a novel prognostic biomarker for colon cancer associated with immune infiltration
              </a>
            </td>
          <td>
            Haiyan Wang, Yanyan Huang, Lina Zhou, Xiaotong Zhu, Xuan Ye, Yi Cen
          </td>
          <td>2025-07-25</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f558af3eae687f94c53483af6c5bcb5fea880573" target='_blank'>
              Human fibroblasts from aged individuals exhibit chromosomal instability through replication stress caused by oxidative stress
              </a>
            </td>
          <td>
            Kailin Zhu, Guan Chen, Yueyi Ren, K. Iemura, Kozo Tanaka
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 The substantial improvement in childhood cancer survival rates has been primarily achieved through therapy intensification, often at the cost of life-long adverse effects. Therapies can be associated with characteristic patterns of mutations found in the tumor genome, known as mutational signatures. These signatures, therefore, can act as markers of resistance and late effects of therapy. We employed a large-scale analysis of 611 whole-genome sequenced pediatric tumors with harmonized therapy data, integrating de novo mutational signature extraction with ensemble machine learning (ML) models trained on comprehensive genomic features (single- and double-base substitutions, indels, structural variants, and copy number alterations) to identify therapy-associated patterns. De novo extraction across single-nucleotide variants, indels, structural variants, and copy number alterations revealed 95 distinct mutational signatures, including 31 novel ones. Platinum therapy was associated with the highest mutagenic burden, with canonical signatures (e.g., SBS31, SBS35, DBS5) detected in 41% of exposed tumors. By aligning with treatment timelines, we defined a median latency of 91 days and a burden threshold of ∼1500 mutations for platinum signatures to emerge—appearing in all osteosarcomas (most common primary bone cancer) and two-thirds of neuroblastomas (most common extracranial solid tumor in children). Despite these insights, the reasons why some tumors do not exhibit these signatures, and how this relates to their response to therapy, remain poorly understood. To uncover subtler therapy-associated signals beyond known COSMIC signatures, we trained ensemble ML models on various types of genomic alterations. Our ML models achieved high predictive performance (F1-score = 0.89) in classifying platinum-exposed tumors. Importantly, predictive accuracy persisted even after masking features corresponding to canonical platinum signatures, indicating the presence of non-canonical mutational patterns. Feature analysis identified biologically plausible non-canonical genomic features enriched in platinum-exposed tumors across pediatric cancers, suggesting previously unannotated mutational footprints of therapy. Notably, ML analysis revealed distinct non-canonical genomic features associated with anthracyclines, and select antimetabolites and alkylating agents, shedding light on therapies previously lacking characterized mutational signatures. These novel genomic signals were also validated across independent pediatric and adult cancer cohorts. This work highlights the power of ML to detect therapy-induced mutational patterns beyond known mutational signatures. As cancer remains the leading cause of disease-related death among children in many countries, this integrated ML-driven approach offers a powerful strategy for improved therapy monitoring, early resistance prediction, and ultimately, the development of more personalized and effective treatment strategies (e.g., stratification and de-escalation) for children with cancer.



 Mehdi Layeghifard, Marcos Díaz-Gay, Pedro L. Ballester, Elli Papaemmanuil, Mark Cowley, Anita Villani, Ludmil B. Alexandrov, Adam Shlien. Unmasking hidden mutational footprints of therapy in pediatric tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A038.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6789b44a7ab2f8906895c50474f02a3bfc36f14" target='_blank'>
              Abstract A038: Unmasking hidden mutational footprints of therapy in pediatric tumors
              </a>
            </td>
          <td>
            M. Layeghifard, M. Díaz-Gay, Pedro L. Ballester, E. Papaemmanuil, Mark J. Cowley, A. Villani, Ludmil B. Alexandrov, A. Shlien
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Cancer research aims to understand the cellular and molecular mechanisms involved, in order to identify new therapeutic targets and provide patients with more effective therapies that generate fewer side undesirable and toxic effects. Previous studies have demonstrated the role of small nucleolar RNAs (snoRNAs) in many physiological and pathological cellular processes, including cancers. SnoRNAs are a group of non-coding RNAs involved in different post-transcriptional modifications of ribosomal RNAs. Recently, we identified a new snoRNA (jouvence), first in Drosophila, and thereafter, by homology, in humans. Methods: Here, we characterize the effect of the knockdown of jouvence by a sh-lentivirus on human primary patient-derived glioblastoma cells. Results: The sh-lentivirus anti-jouvence induces a significant decrease in cell proliferation and leads to cell death. EdU staining confirmed this decrease, while TUNEL also showed the presence of apoptotic cells. An RNA-Seq analysis revealed a decrease, in particular, in the level of BAALC, a gene known to potentiate the oncogenic ERK pathway and deregulating p21, leading to cell cycle blockage. Conclusions: Altogether, these results allow the hypothesis that the knockdown of jouvence could potentially be used as a new anti-cancer treatment (sno-Therapy), especially against glioblastoma and also, potentially, against acute myeloid leukemia (AML) due to the BAALC deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1877da20439c8df3eeaccfbcddc1a2c347beb2b9" target='_blank'>
              Knockdown of the snoRNA-Jouvence Blocks the Proliferation and Leads to the Death of Human Primary Glioblastoma Cells
              </a>
            </td>
          <td>
            Lola Jaque-Cabrera, Julia Buggiani, Jérôme Bignon, Patricia Daira, Nathalie Bernoud-Hubac, Jean-René Martin
          </td>
          <td>2025-07-18</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is the most prevalent type of leukemia in adults, characterized by a complex pathogenesis. The nuclear pore complex is pivotal in the development of various of malignant tumors, including hematological malignancies. While the NUP210 gene is upregulated in several solid tumors, its role in AML remains unreported. RNA-Sequence data and clinical profiles of AML patients were obtained from TCGA databases. Bone marrow samples were obtained from the First Affiliated Hospital of Ningbo University. GEPIA2, UALCAN and TCGA Portal databases were employed for survival prognosis analysis based on NUP210 expression. The GO/KEGG enrichment analysis of NUP210 co-expression and GSEA were performed using R software package. ENCORI and STRING databases were utilized to validate the association between NUP210 and MEIS1, PBX3 or HOXAs. CCK8, RT-qPCR, and Western blot assays were used to detect the viability and related gene expression in AML cell lines.Our findings revealed that NUP210 was significantly upregulated in AML patients compared to healthy controls. Multivariate analyses identified NUP210 as an independent poor prognostic factor in AML. Increased expression of NUP210 inhibited the viability of AML cells. The HOXA9, MEIS1, and PBX3 genes were notably upregulated in the NUP210 high-expression group. The RT-qPCR and Western blot assays demonstrated that NUP210 downregulation in leukemia cells leads to the inhibition of HOXA9 expression. Positive correlations were observed between NUP210 and MEIS1/PBX3, with MEIS1 and PBX3 proteins interacting with the HOXA9 protein. Our results demonstrated that NUP210 was remarkably upregulated in AML and may promote the malignant behavior of AML cells by upregulate HOXA9 gene expression, offering new insights for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60a036833093bfe76d9adec4575ddd5cae1b05c" target='_blank'>
              High NUP210 mRNA expression and its role in prognosis of acute myeloid leukemia
              </a>
            </td>
          <td>
            An Wu, Liyun Yang, Xiao Yan, Yanli Lai, Lieguang Chen, Shanhao Tang, Miao Zhou, Cong Shi, Ying Chen, Lixia Sheng, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-07-22</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5bfa4068fc673194fc62b48dc4a50c589763e7" target='_blank'>
              Ataxin-2 preserves oocyte genome integrity by promoting timely premeiotic DNA replication
              </a>
            </td>
          <td>
            V. L. Monteiro, Crystal Yu, Camilla Roselli, B. Bakthavachalu, Mani Ramaswami, T. Hurd
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA-based evolutionary comparisons of regulatory genomic elements enable insight into functional changes driven in cis, partially overcoming tissue inaccessibility. Here, we harnessed adult and fetal cortex single-cell ATAC-seq datasets to uncover DNA substitutions specific to the human and human-ancestral lineages within apes. We found that fetal microglia identity is evolutionarily divergent in all lineages, whereas other cell types are conserved. Using multiomic datasets, we further identified genes linked to multiple lineage-divergent gene regulatory elements and implicated biological pathways associated with these divergent features. We also uncovered patterns of transcription factor binding site evolution across lineages and identified expansion of bHLH-PAS transcription factor targets in human-hominin lineages, and MEF2 transcription factor targets in the ape lineage. Finally, conserved features were more enriched in brain disease variants, whereas there was no distinct enrichment of brain disease variants on the human lineage compared to its ancestral lineages. Our study identifies ancestral evolutionary patterns of the human brain epigenome at cellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0e33bdd43574807fdd7c3073d7ed0b4672277a" target='_blank'>
              Decoding DNA sequence-driven evolution of the human brain epigenome at cellular resolution
              </a>
            </td>
          <td>
            Emre Caglayan, G. Konopka
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Natural biological agents that can transform normal somatic cells into cancer stem cells, have not been identified. We earlier reported that cell free chromatin particles (cfChPs) that circulate in blood of cancer patients can horizontally transfer themselves to healthy cells to induce dsDNA breaks and inflammation. Here we show that a single cell clone D5 developed from NIH3T3 mouse fibroblast cells treated with cfChPs isolated from sera of cancer patients exhibited upregulation of stemness related transcription factors OCT4, SOX2 and KLF4 and surface markers CD34, CD44 and CD133, and the ability to form spheroids in appropriate culture medium. Transcriptome analysis revealed upregulation of the stemness-associated gene Kruppel-like factor 4 (Klf4), a key Yamanaka factor for the induction of pluripotent stem cells. Additionally, several cancer-related pathways involved in invasion, metastasis, and stemness were found to be upregulated. These pathways included those related to ERBB signaling, P53, cellular senescence, ATM, CTCF and apoptosis. Subcutaneous inoculation into SCID mice resulted in development of malignant tumours which expressed the endoderm marker alpha fetoprotein confirming their stem cell property. Our results suggest the cfChPs that circulate in blood of cancer patients can transform susceptible somatic cells into cancer stem cells with the potential to promote metastasis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09686-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5905889dedfeab60767171035c4266975f0de7d2" target='_blank'>
              Induction of cancer stem cells from mouse fibroblasts
              </a>
            </td>
          <td>
            S. Shabrish, Leimarembi Devi Naorem, Shubham Mohanty, Rohit Kumar, Relestina Lopes, Nabila Akhter, Naveen Kumar Khare, Ruchi Joshi, Rajashree Sawant, G. V. Raghuram, Vishal Jadhav, Sushma Shinde, Pratik Chandrani, Indraneel Mittra
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The long non-coding RNA PVT1 reportedly forms a circular RNA variant (circPVT1). As circPVT1 is expressed in various cancers and has been implicated in promoting cancer cell proliferation and tumor progression, it is considered a potential biomarker and therapeutic target. We previously confirmed that circPVT1 expression varies according to the Gleason pattern, a morphological indicator of malignancy in prostate cancer. In this study, we assessed the expression of circPVT1 using BaseScopeTM assay with prostate cancer tissues and evaluated the correlation with the Grade Group (based on Gleason pattern), an indicator used to morphologically evaluate the degree of malignancy of prostate cancer. The relationship between circPVT1 expression and tumor proliferation was evaluated using cells in which circPVT1 expression was suppressed using the clustered regularly interspaced short palindromic repeats (CRISPR)/RfxCas13d system. BaseScopeTM assay confirmed that circPVT1 expression was significantly higher in Grade Group 2–5 (intermediate- and high-grade groups) than Grade Group 1 (low-grade group). In vitro experiments using the CRISPR/RfxCas13d system showed that specific suppression of circPVT1 expression resulted in a significant reduction in the number of prostate cancer cells. The results of this study suggest that circPVT1 is involved in tumor growth in prostate cancer and may serve as a therapeutic target for moderately and highly malignant prostate cancers that express circPVT1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ade680add5e5637baf3af381d242ed6326044739" target='_blank'>
              Impact of circPVT1 Expression on Prostate Cancer Malignancy and Tumor Growth
              </a>
            </td>
          <td>
            Miyaka Umemori, K. Tashiro, Ayana Horiguchi, Fumihiko Urabe, Takahiro Kimura, S. Sato, Takahashi Hiroyuki, Morito Kurata
          </td>
          <td>2025-06-18</td>
          <td>Acta Histochemica et Cytochemica</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Women who harbor germline heterozygous mutations of BRCA1 or BRCA2 have a high risk of breast cancer. Our previous study showed that patient-derived, ostensibly normal BRCA2mut/+ luminal progenitor (LP) cells are more prone to exhibit sub-chromosomal copy number variations and associated DNA damage relative to non-carriers, potentially reflecting early breast tumorigenesis. Clinically assessable biomarkers for early pathological changes of BRCA1/2 mutation in LP cells remain unknown. Single-cell RNA sequencing (scRNAseq) of LP cells of BRCA1/2mut/+ carriers revealed enrichment of KIT expression (KIT+) and subsequent transcriptional factor activations were observed in LP cells of BRCA1/2mut/+ carriers relative to non-carriers. Moreover, pathway analysis uncovered that KIT+ BRCA-mutated LP cells were enriched in pathways involving DNA binding transcription activator activity and oxidative phosphorylation. These gene signature profiles were recapitulated in bulk RNA-seq of BRCA2 mut/+ carrier LP cells. Collectively, our preliminary data suggest such analyses may identify potential biomarkers for near-term risk prediction that reflect early pathogenesis of BRCA1/2-associated carcinogenesis. Following further validation via tissue microassay analysis and clinical studies, our findings may eventually assist patients and clinicians in decision making regarding the timing and necessity of preventive surgeries for BRCA1/2 mutation carriers.
 Citation Format: Zuen Ren, Siang Boon Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, Veerle Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W. Ellisen. Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a865906cd027bf501016cc53a811825bba9923" target='_blank'>
              Abstract P3-04-06: Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer
              </a>
            </td>
          <td>
            Zuen Ren, S. Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, V. Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W Ellisen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The study explores the role of non-coding ribonucleic acids (ncRNAs) and higher enhancer of zeste homolog 2 (EZH2) gene in breast cancer progression, examining how microRNA (miRNA) and long noncoding RNA (lncRNA) control EZH2, potentially influencing oncogene growth and treatment failure. Materials and methods The databases used in the study included Cyclebase 3.0 and CellTracer to determine EZH2’s role in cell cycle, Oncomine, OncoMX and The University of Alabama At Birmingham Cancer Data Analysis Portal (UALCAN) for Pan-cancer analysis, The Cancer Genomic Atlas Portal (TCGA Portal), Gene Expression Profiling Interactive Analysis (GEPIA2), OncoDB, CR2Cancer, Encyclopaedia of RNA Interactomes (ENCORI), and The Cancer Genome Altas Analyzer (TCGAnalyzeR v1.0) for differential expression analysis, CR2Cancer, OncoDB, MethMarkerDB, and Wanderer databases for epigenetic alteration analysis, Kaplan-Meier Plotter for survival analysis, Breast Cancer Gene Expression Miner (bc-GenExMiner v5.0) for hormone receptor analysis, Tumor-Immune System Interaction Database (TISIDB), Cancer Single Cell State Atlas (CancerSEA), TNMplot, DriverDBv4, and ENCORI for biological processes, cell cycle checkpoints and metastasis analysis, Enrichr, Tumor Immune Estimation Resource (TIMER 2.0), Gepia2 for transcription factor analysis, miRNet, Transcriptome Alterations in Cancer Omnibus (TACCO), and CancerMIRome for miRNA analysis, Enrichr, UALCAN, and ENCORI for lncRNA analysis. Results The EZH2 gene is overexpressed in breast cancer (BRCA) tumors, metastatic tissues, and circulating tumor cells, potentially leading to cancer progression. Patients with high EZH2 expression have shorter overall survival (OS), distant metastasis-free survival (DMFS), and relapse-free survival (RFS) compared to those with low expression. Estrogen receptot (ER)-negative BRCA tumors and PR-negative tumors have high EZH2 gene and eukaryotic transcription factor (E2F2) levels. The EZH2/E2F2 axis may assist ER/PR-negative BRCA by sponging homosapiens microRNA family (hsa-let-7b-5p) through lncRNA-thymopoietin antisense transcript 1 (TMPO-AS1). Overexpression of the EZH2 protein is associated with BRCA metastasis. Conclusion EZH2 overexpression in basal-like BRCA is mediated by a competing endogenous RNA (ceRNA) network and regulating their expression levels may facilitate better survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7050692d6924a95d1983e91c0ebe8eef26bc9ef7" target='_blank'>
              TMPO-AS1-hsa-let-7b-5p-EZH2-RNA network predicts poor survival in basal-like breast cancer patients
              </a>
            </td>
          <td>
            Prerna Vats, Bhavika Baweja, Sakshi Nirmal, Aditi Singh, Rajeev Nema
          </td>
          <td>2025-06-07</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common renal malignancy. However, the combined clinical and biological scores commonly used to predict the clinical outcome are imperfect and need improvement. The main goal of our study was to assess the effect of mtDNA genetics on the prognosis of ccRCC patients and to explore morphologic correlation. Mitochondrial DNA copy number (mtDNAcn) variation between tumor and paired matched healthy kidney tissue was assessed by real-time quantitative PCR and expressed as a ratio in 105 patients. According to this median ratio, the cohort was divided into two groups: “LOW” (n = 53) and “HIGH” (n = 52). Cancer-Specific Survival (CSS) and Disease-Free Survival were assessed in each group. The tumor samples were classified into two subtypes (Clear or Eosinophilic cells) according to the cytoplasmic morphology. CSS was significantly reduced in the “HIGH” than in the “LOW” group with respective 5-year survival rates: 78.7% (CI 95: 64.8–95.5) and 95.5% (CI 95 87.1–100.0) (Hazard Ratio: 7.4 (CI 95: 1.9–29.9, p = 0.027*) in multivariate analysis, including pathological classification, tumor size, International Society of Urological Pathology grade, lymphovascular invasion, dedifferentiated pattern, necrosis and adjuvant therapy. Next-generation sequencing of mtDNA was performed on 14 tumors and matched healthy kidney tissue. No hotspot mutation or redundant large deletion was found. None of the variants or large deletions identified had an impact on prognosis. MtDNAcn variation in tumor relative to normal kidney appears as an independent prognostic factor in ccRCC, which was also associated with eosinophilic morphology. MtDNA content could be considered an additional prognostic factor, in combination with other predictive parameters. Furthermore, these results underline the importance of the role of mitochondria in ccRCC and the need for further functional studies to understand the pathophysiological mechanisms better and consider therapies targeting mitochondrial metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288c14d8ba3e745b3421617281d655151662ced6" target='_blank'>
              MtDNA copy number enrichment is associated with poor prognosis and eosinophilic morphology in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Sarah Bellal, C. Rolley, Jérémy Richard, Nolwenn Bounaix, Vincent Le Corre, Marie-Christine Copin, Odile Blanchet, Pierre Bigot, Vincent Procaccio, C. Bris
          </td>
          <td>2025-07-23</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mitotic bookmarking, the retention of regulatory proteins and lncRNAs on chromatin during mitosis, epigenetically sustains competency for phenotype-specific gene expression in progeny cells. Gene expression is predominantly suppressed during mitosis. Bookmarking provides the guidance for the resumption of gene expression in progeny cells that is obligatory for physiological control of lineage commitment, specialized cell structure and phenotypic function. While regulatory continuity is supported by the persistence of genome-associated regulatory complexes, altered bookmarking mediates plasticity for responsiveness to physiological cues. Bookmarking fidelity ensures genome integrity and controls expression of tumor suppressors and proto-oncogenes. Cancer-compromised aberrations in bookmarking results in transcriptional dysregulation and the initiation of tumor-associated processes. Keywords: Mitotic Bookmarking, Mitosis, Epigenetic Control, Gene Expression, Cell Division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d78052a96f4610f52d1801e1448869b1e34100" target='_blank'>
              Mitotic Bookmarking Provides Epigenetic Persistence or Plasticity for Biological Control and Cancer.
              </a>
            </td>
          <td>
            Andrew J. Fritz, Emory Pacht, R. Toor, Janine Warren, Haley W Greenyer, Jackson R Del Porto, Abigail G Person, Sadie J Korzec, Georgiy Zotkin, Jessica L Heath, Prachi N Ghule, J. Gordon, Andre J. van Wijnen, Seth E Frietze, Karen C. Glass, Jane Lian, Jane L. Stein, Gary S. Stein
          </td>
          <td>2025-07-16</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract The six-subunit origin recognition complex (ORC) loads excess MCM2–7 on chromosomes to promote initiation of DNA replication and is believed to be important for origin specification. Mapping of origins in cancer cell lines engineered to delete three of the subunits, ORC1, ORC2, or ORC5, shows that specific origins are still used and are mostly at the same sites in the genome as in wild-type cells. The few thousand origins that were upregulated in the absence of ORC suggest that GC/TA skewness and simple repeat sequences facilitate, but are not essential for, origin selection in the absence of the six-subunit ORC. Despite the lack of ORC, excess MCM2–7 is still loaded at comparable rates in G1 phase to license dormant origins and is also repeatedly loaded in the same S phase to permit re-replication. Thus, origin specification and excess MCM2–7 loading on origins do not require the six-subunit ORC in human cancer cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9d06157a04f12f90653a1aa8763b312684dcdf" target='_blank'>
              Specific origin selection and excess functional MCM2–7 loading in ORC-deficient cells
              </a>
            </td>
          <td>
            Y. Shibata, Mihaela Peycheva, Etsuko Shibata, Daniel Malzl, Rushad Pavri, Anindya Dutta
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Pituitary adenomas (PAs) are benign tumors in the pituitary gland. However, 30–40% of these tumors are invasive, complicating diagnosis and treatment. Invasive pituitary adenomas (IPAs) often respond poorly to conventional therapies, emphasizing the need for better diagnostic and therapeutic strategies. Understanding DNA methylation patterns in IPAs may reveal new biomarkers and therapeutic targets, leading to more effective management of this challenging disease. Methods Reduced representation bisulfite sequencing (RRBS) and RNA sequencing (RNA-seq) were performed on 129 samples from the Second Affiliated Hospital of Zhejiang University, including 69 tissue samples from invasive and non-invasive pituitary adenomas (NPA) and 60 blood samples from IPA, NPA and healthy individuals. Differentially methylated regions (DMRs) and differentially expressed genes (DEGs) were identified in tissues. Pearson correlation analysis was used to identify associations between DNA methylation status and gene expression, as well as the effect of methylation on gene expression at different sites. Blood samples were analyzed to detect DMRs and DEGs, correlating with tissue-derived findings. Finally, ROC analysis and a random forest model were used to identify biomarkers for discriminating invasive from non-invasive phenotypes. Results We identified 347 DMRs between IPA and NPA, of which 63% (219/347) were hypomethylated. Additionally, 543 mRNAs showed differential expression, with 350 upregulated and 193 downregulated. 17 genes demonstrated concurrent aberrant methylation and expression, primarily within introns, promoters, and CpG islands (CGIs). Notably, only protein tyrosine phosphatase receptor type T (PTPRT) exhibited a remarkably high correlation (r = 0.81) between its DNA methylation levels and mRNA expression levels. This correlation was observed within the intronic region/opensea of the gene's CGIs. Plasma sample analysis revealed 852 DMRs between IPA and NPA, with 52% (447/852) being hypomethylated. Three tumor tissue-derived blood biomarkers (MIR4535, SLC8A1-AS1, and TTC34) accurately discriminated between IPA and NPA patients with a combined AUC of 0.980. These markers also differentiated NPA from healthy controls, though with different methylation patterns. Conclusions The relationship between DNA methylation and gene expression is complex. Plasma-based DNA methylation markers can effectively discriminate between IPA and NPA, as well as between NPA and healthy individuals (N group). Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04164-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63168fb628263eeafdc395f9c006371c823b818" target='_blank'>
              Molecular signatures of invasive and non-invasive pituitary adenomas: a comprehensive analysis of DNA methylation and gene expression
              </a>
            </td>
          <td>
            Yike Chen, Ningning Zhao, Jiahao Zhang, Xinyi Wu, Jian Huang, Xiaohui Xu, Feng Cai, Sheng Chen, Liyin Xu, Wei Yan, Yuan Hong, Yunfei Wang, Hui Ling, Jianxiong Ji, Gao Chen, Hongcang Gu, Jianmin Zhang, Qun Wu
          </td>
          <td>2025-07-01</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutational signature analysis has greatly enhanced our understanding of the mutagenic processes found in cancer and normal tissues. As part of a recent study, we analyzed 802 treatment-naïve, microsatellite-stable colorectal cancers (CRC) and identified a de novo signature, SBS_D, which was conservatively decomposed into SBS18, a signature associated with reactive oxygen species. Here, we re-evaluate this decomposition and provide evidence that SBS_D represents a distinct mutational process from that of SBS18. Through an independent analysis of 2,616 whole-genome sequenced microsatellite-stable CRCs across three distinct cohorts, we demonstrate that SBS_D is consistently present at a similar prevalence, suggesting that this signature may have been previously overlooked. Using a naïve decomposition approach, we demonstrate that the pattern of SBS_D better aligns with signatures previously associated with deficiencies in DNA polymerase delta (POLD1) proofreading and mismatch repair. However, multiple lines of evidence, including the absence of pathogenic mutations in the exonuclease domain of POLD1 or in mismatch repair-associated genes, indicate that SBS_D is not driven by canonical defects in these DNA repair pathways. Overall, this study identifies a previously unrecognized mutational signature in microsatellite-stable CRC and proposes that its etiology may be linked to DNA repair infidelity emerging late in tumor development in samples without canonical defects in DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d7a0ecc45c91a460045023891d09c9a49fcad6" target='_blank'>
              Identification and Validation of a Previously Missed Mutational Signature in Colorectal Cancer
              </a>
            </td>
          <td>
            Mariya Kazachkova, Burçak Otlu, M. Díaz-Gay, Ammal Abbasi, Sarah Moody, Sandra Perdomo, David C Wedge, Paul Brennan, Michael R Stratton, Ludmil B. Alexandrov
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT The mechanisms underlying the assembly and regulation of enzymatic complexes responsible for plant organellar DNA replication remain poorly characterized. Unlike the monophyletic origin of the gene products involved in animal mitochondrial replication, derived from T‐odd bacteriophages, plant organellar DNA replication relies on genes either unique to plants or with origins traceable to bacteria and bacteriophages. Here, we demonstrate that the bacteriophage‐related primase‐helicase from Arabidopsis thaliana (AtTwinkle) is essential for double‐stranded DNA unwinding. AtTwinkle functionally interacts with bacterial‐related organellar DNA polymerases (AtPolIs), which lack the ability to unwind large regions of dsDNA, coupling DNA unwinding to processive DNA synthesis at the leading strand of the replisome. Analysis of two T‐DNA insertion mutants of AtTwinkle reveals distinct phenotypic outcomes; these mutant lines are hereafter referred to as ph. The ph1 (−/−) mutant, which carries a T‐DNA insertion in the 5´ UTR region, is viable and exhibits no noticeable developmental differences compared to wild‐type plants. In contrast, the ph2 mutant, with a T‐DNA insertion in the 19th exon, displays embryo lethality. Despite these differences, both ph1 (−/−) and heterozygous ph2 (+/−) mutants show a reduction in organellar DNA copy numbers under non‐stress conditions and exhibit heightened sensitivity to DNA‐damaging agents. In summary, our findings demonstrate that AtTwinkle is essential for organellar DNA replication. The heightened sensitivity of insertion mutants to organelle‐specific genotoxic agents indicates that loss of AtTwinkle function reduces the availability of template DNA necessary for double‐strand break (DSB) repair. Collectively, our findings reveal that two proteins of distinct evolutionary origins—AtTwinkle and plant organellar DNA polymerases—coevolved to coordinate DNA replication in plant organelles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa75f5ccf72ac5f810f8cdfaa6b416c03addc39" target='_blank'>
              A Bacteriophage‐Derived Primase‐Helicase Orchestrates Plant Organellar DNA Replication
              </a>
            </td>
          <td>
            Carlos M Morales-Vázquez, Mayra A. Dagio-Hernández, Laura D Camacho-Manriquez, Antolín Peralta-Castro, Claudia D Raygoza, Diana Solano-Argüello, Josue D Mora-Garduño, Rogelio González-González, H. Herrera-Ubaldo, C. Díaz‐Quezada, Alfredo Cruz-Ramírez, S. de Folter, J. Pedroza-García, Luis G. Brieba
          </td>
          <td>2025-07-01</td>
          <td>Physiologia Plantarum</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Cancer treatments that use drugs resembling DNA building blocks, known as nucleotide analogs, that insert themselves into DNA and prevent replication often stop working over time because cancer cells become resistant. This review explains how reversible epigenetic changes help cancer cells avoid being killed by these drugs. These changes do not alter the DNA sequence but can turn important genes on or off. We describe how cancer cells use these mechanisms to repair damaged DNA, pump drugs out of the cell, or break them down. We highlight the role of DNA packaging proteins (histones), small RNA molecules, and chemical tags on DNA that control how genes behave. Understanding these changes is important because it may allow clinicians to resensitize tumors and make them sensitive to treatment again and prevent relapse. We also explore new therapies, including drugs and gene editing tools, that target these epigenetic changes to improve the success of chemotherapy. These findings could lead to better outcomes for patients with difficult-to-treat cancers, especially those with pancreatic, colorectal, or breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeef0502ae8a946f9634749621e6cdd28f4f464" target='_blank'>
              Epigenetic Drivers of Chemoresistance in Nucleobase and Nucleoside Analog Therapies
              </a>
            </td>
          <td>
            John Kaszycki, Minji Kim
          </td>
          <td>2025-07-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Over the past decade, advancements in next-generation sequencing have significantly enhanced our understanding of the molecular pathogenesis of adamantinomatous craniopharyngiomas (ACP). Objective This study integrated methylome and transcriptome analyses in ACP samples to explore the potential interplay between DNA methylation and RNA expression signatures for diagnostic and prognostic applications in ACP patients. Methods This cross-sectional study evaluated clinicopathological features, DNA methylation, and gene expression profiles in 15 patients with ACP (33% women, age range: 3–55 years, 53% diagnosed before 18 years) treated at Ribeirao Preto Medical School, University of São Paulo. Results Multidimensional scaling and principal component analysis identified two distinct clusters (ACP-A: n=9, ACP-B: n=6) with consistent composition across DNA methylation and gene expression profiles. While most clinical and histopathological characteristics were similar between clusters, ACP-A exhibited a longer median progression-free survival. ACP-B showed a higher prevalence of hypomethylated probes in CGI sites, and 63% of differentially methylated positions (DMPs) located in gene body regions. Differential methylation patterns were categorized into Methyl-Set1 (hypomethylated in ACP-A and hypermethylated in ACP-B) and Methyl-Set2 (hypermethylated in ACP-A and hypomethylated in ACP-B). Clustering analyses based on the methylation levels of probes and expression levels of the stringently filtered 212- and 37-gene sets further confirmed these two distinct ACP subgroups. Functional enrichment analysis highlighted key roles in synaptic modulation, nervous system development, cell adhesion, as well as pathways linked to RAS signaling, GTPase activity, and membrane potential regulation. Conclusion Although clinical characteristics were largely comparable between the clusters, ACP-B patients exhibited shorter median progression-free survival, suggesting a more aggressive phenotype. The higher prevalence of hypomethylation in ACP-B indicates increased transcriptional activation, potentially driving tumor aggressiveness. The strong concordance between methylation and transcriptomic data in the 212- and 37-gene sets underscores their potential as a clinically relevant molecular biomarker panel. These gene sets demonstrate robustness in distinguishing ACP clusters, making it a promising tool for clinical sample classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1321c37193ba8ab134af26f85d3133bdcee99c23" target='_blank'>
              Methylation and gene expression patterns in adamantinomatous craniopharyngioma highlight a panel of genes associated with disease progression-free survival
              </a>
            </td>
          <td>
            J. Marrero-Gutiérrez, A. Bueno, C. Martins, Rui M Patrício Silva-Júnior, Antônio Carlos dos Santos, M. V. Santos, L. E. Wildemberg, Ximene Antunes, Mônica R Gadelha, A. C. Moreira, R. N. Vêncio, S. R. Antonini, Margaret de Castro
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent and deadly neoplasms globally; this fact puts emphasis on the need for accurate molecular biomarkers for early detection and accurate prognosis. Circular RNAs (circRNAs) have recently emerged as very promising cancer biomarkers. In this study, we thoroughly examined whether the expression levels of circular transcripts of the protein arginine methyltransferase 1 (PRMT1) gene can predict the prognosis of patients diagnosed with colorectal adenocarcinoma, the most frequent type of CRC. Hence, a highly sensitive quantitative PCR (qPCR) assay was developed and applied to quantify circ-PRMT1 expression in cDNAs from 210 primary colorectal adenocarcinoma tissue specimens and 86 paired normal colorectal mucosae. Extensive biostatistical analysis was then performed to assess the potential prognostic power of circ-PRMT1. Significant overexpression of this molecule was observed in colorectal adenocarcinoma tissue samples in contrast to their non-cancerous counterparts. Moreover, higher circ-PRMT1 expression was correlated with poorer disease-free survival (DFS) and worse overall survival (OS) in colorectal adenocarcinoma patients. Interestingly, multivariate Cox regression analysis revealed that the prognostic value of the expression of this circRNA does not depend on other established prognostic factors included in the prognostic model. Furthermore, the stratification of patients based on TNM staging revealed that higher circ-PRMT1 levels were significantly related to shorter DFS and OS intervals, particularly in patients with colorectal adenocarcinoma of TNM stage II or III. In summary, this original research study provides evidence that circ-PRMT1 overexpression represents a promising molecular biomarker of poor prognosis in colorectal adenocarcinoma, not depending on other established prognostic factors such as TNM staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b06ea5f530ca9a15a72069b8a7542753ca04c1f" target='_blank'>
              Overexpression of Circular PRMT1 Transcripts in Colorectal Adenocarcinoma Predicts Recurrence and Poor Overall Survival
              </a>
            </td>
          <td>
            P. Kokoropoulos, S. Christodoulou, P. Tsiakanikas, P. Vassiliu, C. Kontos, Nikolaos Arkadopoulos
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has been described in several eukaryotic species and has been shown to impact phenomena as diverse as cancer and herbicide tolerance. EccDNA is thought to arise mainly through transposable element (TE) mobilization. Because studies based on short-read sequencing cannot efficiently identify full-length eccDNA forms generated from TEs, we employed the CIDER-Seq pipeline based on long-read sequencing, to obtain full-length eccDNAs from Arabidopsis. The generated eccDNA datasets identified centromeric/pericentromeric regions as hotspots of eccDNAs with several eccDNA molecules originating from Helitron and LTR TEs. To investigate the role of epigenetic marks on TE-derived eccDNA biogenesis, we studied Arabidopsis methylation mutants dcl3, rdr6, ros1, and ddm1. Contrasting the TE-suppression previously reported in the hypermethylated ros1 mutants, we identified activation of TEs in ros1, specifically of LTR/Gypsy TEs. An enrichment of LTR/Copia elements was identified in actively dividing calli and the shoot apical meristem (SAM). Uncharacterized “variable TEs” with high eccDNA and expression were identified in the SAM, including ATCOPIA58. Together, our study reveals the genomic origins of eccDNAs and delineates the link between epigenetic regulation, transposon mobilization, and eccDNA biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8fe16883b4dfe42fe6b49221fc2e6a347cfdc2" target='_blank'>
              A comprehensive atlas of full-length Arabidopsis eccDNA populations identifies their genomic origins and epigenetic regulation
              </a>
            </td>
          <td>
            S. S. Zaidi, Sara Shakir, Hanne De Kort, Devang Mehta, V. Nguyen, Ruben Gutzat, Hervé Vanderschuren
          </td>
          <td>2025-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caf636d542e3849c7195363bf633c41f745b9f60" target='_blank'>
              Intranodal palisaded myofibroblastoma shows a unique epigenetic profile-first molecular study of their epigenetic and copy number variation profile.
              </a>
            </td>
          <td>
            Sandra Leisz, M. Scheer, Uwe Hildebrandt, Merle Wiegers, Christian Strauss, C. Scheller, T. Mentzel, Andreas von Deimling, Anja Harder
          </td>
          <td>2025-07-18</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) originates from neuroepithelial tissue and is one of the most common intracranial malignant tumors in adults, with high recurrence rate and poor prognosis. In recent years, SOX9 has been reported to play an important role in many diseases and cancers, and is a promising target, but it has been rarely reported in GBM. Methods RNA sequencing data of GBM were obtained from the Cancer Genome Atlas (TCGA) database and the Genotype-Tissue Expression (GTEx) database for analysis of SOX9 expression and differentially expressed genes (DEGs). Moreover, functional enrichment analysis of GBM-related DEGs was performed by GO/KEGG, GSEA, and protein-protein interaction (PPI) network. Additionally, the clinical significance of SOX9 in GBM was assessed by Kaplan-Meier Cox regression and prognostic model. What’s more, we analyzed SOX9-related immune cell infiltration and expression of immune checkpoints in GBM. The incorporated studies were analyzed using the R package. Results SOX9 was highly expressed in a range of malignant tumor tissues, including GBM. Surprisingly, high SOX9 expression was remarkably associated with better prognosis in the lymphoid invasion subgroups in a sample of 478 cases (P < 0.05). Totally, 126 differentially significant genes (DSGs) were identified between high- and low- expression group, of which 29 genes were upregulated and 97 genes were downregulated. Furthermore, high expression of SOX9 was an independent prognostic factor for IDH (isocitrate dehydrogenase)-mutant in Cox regression analysis. Screening was performed by LASSO coefficients to select non-zero variables that satisfied the coefficients of lambda. min, and four genes were screened out. OR4K2 and IDH status were prognostic factors associated with THCA in multifactorial COX regression analysis. SOX9, OR4K2 and IDH status were included in the nomogram prognostic model. Correlation analysis indicated SOX9 expression was correlated with immune cell infiltration and expression of immune checkpoints in GBM. Conclusion SOX9 was identified as a diagnostic and prognostic biomarker in glioblastoma, particularly in IDH-mutant cases. Its expression was closely correlated with immune infiltration and checkpoint expression, indicating its involvement in the immunosuppressive tumor microenvironment. SOX9-based gene signatures further supported a robust nomogram model, underscoring its potential as a therapeutic and prognostic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2a8c97c4df9bf9651c0f0af4a4efc671535b22" target='_blank'>
              High expression of SOX9 is a diagnostic and prognostic indicator of glioma
              </a>
            </td>
          <td>
            Libo Xu, Zhenhao Wang, Mao Li, Qingsong Li
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies. Methods We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration. Results Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration. Discussion Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a type of primary liver cancer that predominantly affects healthy adolescents and young adults in a background of normal liver. The DNAJB1-PRKACA fusion gene is an oncogenic driver in FLC tumors. To investigate the oncogenic mechanisms of this fusion gene, we developed a model using human liver progenitor organoids engineered to express DNAJB1-PRKACA. Single-nucleus RNA sequencing of these organoids revealed an upregulation of genes that significantly overlap with those expressed in FLC epithelial cells. Additionally, the expression of DNAJB1-PRKACA led to the downregulation of genes coding for markers of mature epithelial cells, indicating a shift toward a less differentiated state. When compared to wild-type liver progenitor organoids, which exhibit a strong ability to differentiate into hepatocytes, the DNAJB1-PRKACA-expressing liver progenitor organoids displayed a markedly reduced capacity for hepatocyte differentiation. These findings suggest that the DNAJB1-PRKACA fusion gene disrupts the normal differentiation process of liver progenitor cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11028-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c89f29b9713c31ac538d37aa5a7317a800ccc5c" target='_blank'>
              Expression of DNAJB1-PRKACA oncogene suppresses the differentiation potential of liver progenitor organoids towards a hepatocyte lineage
              </a>
            </td>
          <td>
            Emma DiPietro, Nihal Bharath, Michael Karski, Olivia Durfee, Marc S. Sherman, Qianyi Ma, Liang Sun, Alaa R. Farghli, Colton J. Smith, Ina Kycia, Praveen Sethupathy, Wolfram Goessling, Michael S. Rogers, K. Vakili
          </td>
          <td>2025-07-16</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Liver cancer is one of the most lethal human malignancies in the world. Although great efforts have been made to develop novel therapeutic targets, effective drug targeting remains limited. C20orf27, located on human chromosome 20, is a gene whose function has not been fully elucidated. In this study, we identified C20orf27 as a critical gene that facilitates liver cancer progression. We detected that C20orf27 was upregulated in liver cancer samples. Gain- and loss-of-function experiments demonstrated that C20orf27 enhanced liver cancer cell proliferation and migration by regulating cyclin-related proteins, including MDM2, PCNA, Cyclin E1, CDK2, and p-Rb. RNA sequencing and bioinformatics analyses revealed that NT5E, ACE, and TTR were potential downstream targets of C20orf27. Immunohistochemistry analysis of tissue microarrays suggested that NT5E expression was significantly associated with C20orf27 levels. Moreover, the combination of C20orf27 and NT5E predicted the prognosis of liver cancer patients, as higher levels of both were associated with poorer overall survival and disease-free survival after surgery. Our findings are the first to report the functional role of the C20orf27/NT5E axis in promoting hepatocellular carcinoma progression, highlighting its potential as a novel therapeutic target and biomarker for HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03233-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7f1bcfd855abf2f392100ce81d0167b9ca16b2" target='_blank'>
              C20orf27 promotes hepatocellular carcinoma progression via NT5E
              </a>
            </td>
          <td>
            Yu He, Wen Zhang, Ming Chang, Yuanjie Chen, Risheng Lin, Xinyu Ye, Wei Mao, Yubin Lu, Yilin Lu, Erbao Chen
          </td>
          <td>2025-07-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d082ef6098c2dc18ea4d90e8ab3334b1e46a5ee" target='_blank'>
              Patterns and drivers of 43,617 mosaic chromosomal alterations in blood
              </a>
            </td>
          <td>
            D. Tang, N. Kamitaki, R. E. Mukamel, S. Rubinacci, P.-R. Loh
          </td>
          <td>2025-07-30</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability is a major force driving tumorigenesis. The ssDNA-binding protein complex CTC1-STN1-TEN1 (CST) plays a pivotal role in maintaining genome stability by countering replication stress, modulating DNA damage repair, and maintaining telomere integrity. Despite its well-documented role in genome maintenance, the involvement of CST in skin cancer development has yet to be investigated. We recently found that CST localizes at stalled DNA replication sites after UV exposure and may suppress the unwanted repriming activity, suggesting a potential role of CST in suppressing genome instability caused by UV damage. In this study, we first analyzed CST expression and alterations in cutaneous melanoma database and found that the CST genes are frequently altered in cutaneous melanoma and their expression is significantly downregulated in melanoma samples compared to normal tissues. We then generated a conditional knockout (cKO) mouse model with STN1 deficiency specifically in melanocytes to investigate its role in skin cancer formation. Upon chronic exposure to UV irradiation, STN1-deficient mice exhibit no obvious difference in melanoma incidence compared to control littermates, suggesting that STN1 downregulation in mature melanocytes has no significant effect on UV-induced skin cancer development in lab mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b040e5d51366f43fd311948a74f723a17b17afb" target='_blank'>
              In vivo investigation of STN1 downregulation in melanoma formation in adult mice following UV irradiation
              </a>
            </td>
          <td>
            Sara Knowles, Fang Wang, Maarten C Bosland, S. Gaddameedhi, Weihang Chai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Esophageal cancer (EC) is a malignant tumor with high mortality rates, where early screening and diagnosis are critical for improving patient outcomes. DNA methylation, a key epigenetic modification, has emerged as a significant biomarker for early detection of EC. The advancement in DNA methylation sequencing technologies, including first-generation and next-generation sequencing (NGS), has revolutionized the way we identify and analyze these biomarkers. First-generation sequencing, has been instrumental in identifying specific methylation sites. However, its limited throughput renders it impractical for large-scale screening of multiple samples. In contrast, NGS offers high-throughput capabilities, allowing for the simultaneous analysis of thousands of DNA fragments. NGS significantly enhances the efficiency and accuracy of DNA methylation profiling, permitting genome-wide identification of multiple methylation markers. This approach offers a promising avenue for the enhanced early detection of EC by providing a comprehensive view of the methylation landscape. The integration of NGS into clinical practice is capable of transforming EC screening by offering heightened sensitive and specific approach to identifying patients at risk. As our comprehension of the role of DNA methylation in cancer progression deepens, the development of targeted therapies based on methylation profiles may also become a reality. In conclusion, the evolution of DNA methylation sequencing technologies has unlocked new avenues for the early EC detection. While first-generation sequencing has laid the groundwork for characterizing specific methylation events, NGS has expanded the scope of screening, offering a more robust and scalable solution for identifying early-stage EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51da9fdd5dce7ac7d6243e33871feecf4a54206" target='_blank'>
              DNA methylation in esophageal cancer: technological advances and early detection clinical applications
              </a>
            </td>
          <td>
            Jiyu Tang, Xiaoming Shi, Chao Song, Wenjing Zhang, Yan Yan, Linchao Dai, Di Wu, Jie Qiu, Jiahuan Liu, Tao Wang, Zuhong Lu
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The induced “recombinogenic situation” in hematopoietic stem cells and the activation of the cell’s reparative systems create the basis for recombination events between fragments of extracellular double-stranded DNA delivered into the cell and chromosomal DNA or other forms of the reparative-recombination process. In mouse and rat model organisms as well as in human bone marrow cells, changes in the amount of telomeric DNA in hematopoietic stem cells were assessed as an indicator of repair and recombination events that have occurred. In all experiments performed, recombinant human angiogenin was used as a comparison factor. Dot blot hybridization showed that in the colony cells obtained from the bone marrow cells of the model organisms as well as from human bone marrow cells treated with a double-stranded DNA preparation, there was a significant increase in the amount of telomeric DNA. Amplification of telomeric DNA in colony cells is not associated with contamination of the original DNA preparation with which the bone marrow cells were treated. Treatment of bone marrow cells with DNA that does not carry telomeric sequences (AluI PCR fragment) does not lead to an increase in the amount of telomeric DNA in the cells of grown colonies. This suggests the participation in the amplification of telomeric DNA of an extrachromosomal DNA template carrying telomeric DNA. It has been established that treatment of bone marrow cells with angiogenin also leads to an increase in telomeric DNA in colony cells. A comparison of the type of colonies with the intensity of hybridization (i. e. the amount of telomeric DNA in the sample) suggests that the increase in the amount of detectable telomeric DNA following treatment with angiogenin and hDNAgr has a fundamentally different origin. Western blot analysis and real-time PCR revealed that the increase in the amount of telomeric DNA following treatment of bone marrow cells with a double-stranded DNA preparation does not correlate with the activity of endogenous/exogenous telomerase. For angiogenin, it has been shown that an increase in the amount of telomeric DNA may be the result of activation of endogenous telomerase activity. A principle has been developed for the amplification of a new genetic trait that came into hematopoietic stem cells with extracellular double-stranded DNA material and was fixed in the recipient genome or was transitively present in the cell as new genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f0c77ec133a09bb58f64518224081f769056e8" target='_blank'>
              Concept of natural genome reconstruction. Part 3. Analysis of changes in the amount of telomeric DNA in colony cells as a new amplified feature that arose during the processing of hematopoietic bone marrow stem cells
              </a>
            </td>
          <td>
            V. Ruzanova, S. Oshikhmina, G. S. Ritter, E. Dolgova, S. Kirikovich, E. Levites, Y. R. Efremov, T. V. Karamysheva, A. Bogomolov, M.I. Meschaninova, A. L. Mamaev, O. Taranov, S. V. Sidorov, S. D. Nikonov, O. Leplina, A. Ostanin, E. Chernykh, N. A. Kolchanov, A. Proskurina, S. S. Bogachev
          </td>
          <td>2025-07-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The DNA helicase HELQ is involved in homologous recombination repair, interstrand cross‐link repair, and replication stress response. Its functional defects are associated with infertility and abnormal gametogenesis. However, the specific mechanisms of HELQ in the development of germ cells remain to be elucidated. Here, we uncovered that HELQ deficiency led to proliferation defects of primordial germ cells (PGCs) in mouse embryos, thus compromising the establishment of reproductive reserve. Mechanistically, we found that HELQ interacted with the H3K9me3 demethylase KDM4B, and the absence of HELQ led to a marked increase in both total and chromatin‐bound protein levels of KDM4B, resulting in reduced H3K9me3 levels in the region of the retrotransposon LINE‐1, which triggered its high expression and subsequently caused DNA damage accumulation. Moreover, the developmental defects of HELQ‐deficient PGCs were alleviated by inhibition of retrotransposition. These results indicate that HELQ maintains the genome stability of PGCs by repressing LINE‐1 expression. Our study reveals a critical role of HELQ in PGC development and provides new insights into reproductive disorders caused by defects in DNA damage response factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d6e1277ec183328a48e290753878371fe1fc44" target='_blank'>
              HELQ Maintains Genome Stability of Primordial Germ Cells by Inhibiting LINE‐1 Expression
              </a>
            </td>
          <td>
            Lili Cao, Jiayi Ren, Zhaojie Kong, Mengchun Hu, Yaxuan Zhang, Zi-Jiang Chen, Yingying Qin, Yajuan Yang, Shidou Zhao
          </td>
          <td>2025-06-21</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Transposons are mobile genetic elements capable of moving within the genome. Leveraging this property—particularly the cut-and-paste mechanism of DNA transposons—has enabled the development of technologies for inserting exogenous DNA fragments into host genomes. While targeted integration is a key goal for therapeutic applications, this review highlights the value of their intrinsic randomness. By combining the ability to freely design the DNA cargo with the stochastic nature of transposon integration, it becomes possible to generate highly sensitive reporter cells. These can be used to efficiently identify functional markers, uncover novel signaling pathways, and establish innovative platforms for drug screening. As more subfamilies of transposons become available for research use, their complementary biases may enhance the coverage and diversity of genome-wide screening approaches. Although inherently unpredictable, this strategy embraces randomness as a strength, and we propose that it holds great promise for driving new advances in biology, cellular engineering, and medical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d806cc8a8f4473c66fef98ec788d4bd5b59391" target='_blank'>
              Random Insertion Reporter Gimmicks Powered by Cut-and-Paste DNA Transposons
              </a>
            </td>
          <td>
            Yamato Kasahara, Kentaro Semba, Shinya Watanabe, K. Ishikawa
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In brief By tissue-selective next-generation sequencing, we showed that adenomyotic epithelium harbored genomic alterations thought to be relevant to the development and progression of adenomyosis, including somatic mutations in several cancer-associated genes with high mutant allele frequencies and the gain of chromosome 1q. Clonal relationships among multiple adenomyotic lesions and the normal uterine endometrium delineate the oligoclonal origin of adenomyosis and the spatial expansion of mutant clones. Abstract To identify the distinctive features of mutation profiles in adenomyosis compared to the coexisting normal endometrium, multi-regional sampling was performed to collect samples of adenomyotic epithelium (n = 41), adenomyotic stroma (n = 12), and uterine endometrial epithelium (n = 53) from 21 patients with adenomyosis. To enhance the purity in this genomic study, laser microdissection was used to isolate all the samples. Target-gene sequencing and whole-exome sequencing were performed to identify somatic mutations in cancer-associated genes and the pattern of cellular expansion in adenomyosis and clonality between adenomyosis and uterine endometrium. In adenomyotic epithelium, we identified somatic mutations in cancer-associated genes such as KRAS (34.1%), PIK3CA (12.2%), ARID1A (12.1%), and FBXW7 (9.8%) with high mutant allele frequency. In uterine endometrial epithelium, frequently mutated genes included KRAS (47.2%), PIK3CA (37.8%), and ARHGAP35 (28.3%). Whole-exome sequencing revealed clonal relationships among adenomyotic lesions, and between adenomyosis and uterine endometrium. The analysis of somatic copy number alterations (SCNAs) showed recurring gain of chromosome 1q in the adenomyotic epithelium but not in the uterine endometrial endometrium. Mutational signature analysis for SNVs revealed that similar mutational processes were shared in adenomyosis and uterine endometrium. In this study, we identified multiple cancer-associated gene mutations and SCNAs relevant to the development of adenomyosis, and also clonal relationships among multiple adenomyotic lesions and normal uterine endometrium.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/510dc7fd2a5ff02a8e619c015edd58c4ad51cc23" target='_blank'>
              Mutation profile and chromosomal abnormality in adenomyosis
              </a>
            </td>
          <td>
            K. Suda, Kotaro Takahashi, Ryo Tamura, Kyota Saito, Manako Yamaguchi, N. Yachida, S. Adachi, Hiroaki Kase, Shujiro Okuda, K. Yoshihara, Hirofumi Nakaoka
          </td>
          <td>2025-07-07</td>
          <td>Reproduction (Cambridge, England)</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Resistance to anoikis, a crucial factor in cancer cell survival, drives the development and progression of numerous malignancies. Hepatocellular carcinoma (HCC) is a malignant liver tumor characterized by high rates of recurrence and metastasis. However, the role of anoikis in HCC remains poorly understood. In this study, we identify 74 anoikis-related genes (ARGs) differentially expressed in HCC using the transcriptional data from The Cancer Genome Atlas (TCGA). Then, we develop a prognostic model incorporating 9 of these genes through LASSO-Cox regression analysis, and confirm the model's independent prognostic significance for overall survival in HCC patients by using multivariable Cox proportional hazards analysis. Furthermore, we observe significant enrichment of activated proliferation-related pathways, increased infiltration of immunosuppressive cells and resistance to anti-PD-L1 therapy in the high-risk group defined by this model. These findings suggest that the ARG model may serve as a novel prognostic indicator for HCC patients and underscore the critical role of anoikis in HCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf7459a67e0a8001e045b2bbac30fe5bf3c3250" target='_blank'>
              Potential value of anoikis transcriptional signatures in predicting prognosis and immune microenvironment in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Fu-Rong Cheng, , Pengbo Cao, 
          </td>
          <td>2025-07-20</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Wilms tumor (nephroblastoma), the most common pediatric renal malignancy, has a complex genetic and epigenetic landscape. While mutations in genes like WT1, CTNNB1, and WTX have been well characterized, accumulating evidence suggests that epigenetic dysregulation plays a pivotal role in WT pathogenesis. This review examines the various epigenetic mechanisms implicated in WT, including DNA methylation, histone modifications, chromatin remodeling, and non-coding RNA-mediated regulation. We discuss how epigenetic mechanisms contribute to tumor initiation, progression, and heterogeneity and their implications for improved diagnosis and targeted therapy. We also highlight recent advances in epigenomic profiling, discuss the interplay between epigenetics and developmental gene expression programs, and evaluate potential therapeutic strategies targeting epigenetic regulators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fa17235c52d629b5c3bb715ccfc06b98890be55" target='_blank'>
              Epigenetic Regulation in Wilms Tumor
              </a>
            </td>
          <td>
            Annabelle Bolitho, Hongbing Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The human body has a diverse range of microbiota that influences human physiological processes and alters disease risk, involving cancer. Metagenomic sequencing investigations have revealed that the microbiota is an element of the tumor microenvironment, affecting tumor proliferation and responsiveness to current anticancer treatments. The notion of intratumoral microbiota was subsequently introduced. Intratumoral microorganisms have been identified in kinds of cancer, including pancreatic, colorectal, liver, esophageal, breast, and lung malignancies. Microbiota may inhabit tumor tissues by mucosal breakdown, neighboring tissue migration, and hematogenous spread, influencing the biological behavior of tumors as a significant component of tumor’s microenvironment. The intratumoral microbiota may facilitate the onset and progression of malignancies through DNA mutations, activation of carcinogenic pathways, alteration of anticancer medication metabolism, and commencement of metastasis. This document is to present an overview of intratumoral microbiota, their prevalence and progression in cancer, their detection and therapy, and to evaluate the potential and limitations of research in this domain. We intend to generate ideas for investigating intratumoral microbiota as possible treatment targets and biomarkers for tumor assessment, prognosis, and detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/487e91c7e1620d8dcd87c884be958d2ecfa543d3" target='_blank'>
              Intratumoral microbiota: implications for cancer progression and treatment
              </a>
            </td>
          <td>
            Zehang Xie, Zhenguo Wu, Yan Liu, Yu Gu, Jiahao Niu, Kun Lv
          </td>
          <td>2025-07-28</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AIMS
Intrahepatic cholangiocarcinomas (ICCs) are aggressive liver tumors with high heterogeneity and limited therapeutic options. Although traditionally thought to arise from biliary cells, recent findings suggest that hepatocytes may also serve as a cellular origin for ICC. However, the mechanisms underlying hepatocyte malignant transformation and ICC initiation remain poorly understood.


APPROACH AND RESULTS
We employed oncogene-driven and chemically induced ICC murine models, along with cellular models, to recapitulate the transformation of hepatocytes into ICC. Our findings demonstrate that mature hepatocytes undergo a significant hyperpolyploid state during ICC initiation. Hyperpolyploidy promotes aberrant cell division and chromosomal instability, accelerating hepatocyte transformation and ICC onset. Furthermore, we identified the chromatin remodeling factor Arid1a as a critical suppressor of hyperpolyploidy. Arid1a deficiency disrupts mitotic machinery at the centrosome, driving hyperpolyploidization and ICC tumorigenesis.


CONCLUSIONS
Hepatocytes can transform into ICC through a process involving hyperpolyploidizaiton. This study offers new insights into the pathogenesis of ICC, particularly in patients harboring frequent ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb35330c9db09a4f3ded3d2167db23a4e30f36f2" target='_blank'>
              Arid1a deficiency promotes hepatocyte hyperpolyploidy and drives intrahepatic cholangiocarcinoma in mice.
              </a>
            </td>
          <td>
            Qi Bian, Shu Wang, Zimin Song, Fang Liu, Muqing Cao, Nan Yang, Jun Ying, Jia-Syuan Hu, Xinyuan Xiong, Huiqin Zhu, Jun Wu, Jie Yang, Xiaonan Wang, Shunli Shen, Xuxu Sun
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer is one of the most common malignant tumors among women worldwide. Its primary etiology is closely associated with human papillomavirus infection, which poses a serious threat to the health of women. N6-methyladenosine (m6A) modifications notably affect the biological characteristics of tumor cells, such as their proliferation, metastasis and chemoresistance, by regulating the stability, translation and degradation of RNA. It also serves an important regulatory role in the pathogenesis of cervical cancer. The present review details the mechanisms underlying m6A modification in cervical cancer and analyzes its impact on tumor progression. Moreover, it explores the potential clinical applications of m6A modification as a biomarker and therapeutic target to provide new insights and evidence regarding the early diagnosis and individualized treatment of patients with cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66d595728a32d4a2ecd97272d112fd5969eeebc" target='_blank'>
              N6-methyladenosine in cervical carcinogenesis: Mechanistic insights and therapeutic perspectives (Review)
              </a>
            </td>
          <td>
            Man Xu, Fanglei Yang, Huan Chen, Feiyun Jiang
          </td>
          <td>2025-07-14</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Spatial transcriptomic analysis has proposed valuable insights into the behavior of tongue cancer. However, the specific cell types involved in chemically-induced carcinogenesis and the process of tumor development remain elusive. Methods We leveraged artificial intelligence (AI) algorithms and spatial transcriptomic sequencing to meticulously characterize the spatial and temporal evolution of 4-nitroquinoline-1-oxide (4NQO)-induced tongue carcinogenesis and intratumor heterogeneity. Results An AI classifier effectively categorized dysplastic tongue tissue into 13 distinct groups. Spatial transcriptomics identified 13 corresponding cellular subgroups with unique features within the lesion. Both methods successfully distinguished subtle muscle phenotype and genetic lineage variations induced by 4NQO, despite limited morphological differences. Evolutionary tree analysis revealed the dynamic appearance and disappearance of functionally and genetically diverse cell subgroups during the progression from epithelial dysplasia to in situ carcinoma and invasive cancer. Key findings include the identification of specific switch genes associated with tumor invasion and the revelation of significant intratumor heterogeneity. Discussion This spatial transcriptomic analysis of 4NQO-induced tongue cancer provides a detailed characterization of tumor evolution and heterogeneity. It elucidates critical aspects of tongue cancer cell behavior and identifies potential therapeutic targets (switch genes). These findings offer novel insights for improving the diagnosis and treatment of tongue cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679c71fb0ae1356ab18cbbcb3f0859a89370b05e" target='_blank'>
              Spatial transcriptomic analysis of 4NQO-induced tongue cancer revealed cellular lineage diversity and evolutionary trajectory
              </a>
            </td>
          <td>
            Feng Liu, Xiaojun Wu, Shaoqing Yu, Deqiang Cheng, Jun Pan, Xiaodong Wang, Guanzhen Yu, Chaofu Li
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Although recent genetic studies have identified numerous genetic alterations in diffuse large B-cell lymphoma (DLBCL), their biological relevance remains elusive. Here we performed in vivo CRISPR loss-of-function screening targeting 86 genes recurrently altered in DLBCL to examine oncogenicity of sgRNA-targeted genes, association between genotype and lineage, occurrence of second-hit alterations, and cooperability among sgRNA-targeted genes and second-hit alterations. Transplantation of the CRISPR library-transduced hematopoietic stem/progenitor cells induces various hematologic malignancies, including B-lymphomas in mice. Enrichment analysis of sgRNA-targeted genes demonstrates significant overrepresentation of Kmt2d, Pax5, and Trp53 in B-lymphomas. Whole-exome sequencing identifies recurrent second-hit driver alterations, showing significant enrichment of Trp53 alterations in sgKmt2d-targeted B-lymphomas. Importantly, KMT2D and TP53 mutations are found to be the most prevalent co-occurring combination in human DLBCL, which is more prominent in relapsed/refractory DLBCL. Moreover, this combination confers significantly worse prognosis independent of clinical factors. Transcriptomic sequencing identifies overexpression of Yap1, the Hippo pathway component, in double sgKmt2d-targeted/Trp53-altered B-lymphomas. Furthermore, chromatin accessibility analysis demonstrates enrichment of TEAD1 binding motifs in regions that gained accessibility and increased expression of their nearest genes in these B-lymphomas. Most importantly, genetic and pharmacological inhibition of YAP1 suppresses in vitro cell proliferation and in vivo tumor growth of a human KMT2D/TP53-altered DLBCL cell line and prolongs survival of mice transplanted with double sgKmt2d-targeted/Trp53-altered B-lymphoma cells. Our findings demonstrate the utility of in vivo CRISPR screening to integrate human cancer genomics with mouse modeling, and highlight the functional interplay between KMT2D and TP53 aberrations, providing insights into therapeutic strategies in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87541f86c750b412ce2794d97282b0873c8733c8" target='_blank'>
              In vivo CRISPR screening reveals cooperation of KMT2D and TP53 deficiencies in B-cell lymphomagenesis.
              </a>
            </td>
          <td>
            Kentaro Yamaguchi, J. Koya, Kota Mizuno, Yosuke Mizukami, Kota Yoshifuji, Y. Saito, Mariko Tabata, S. Shingaki, Mitsuhiro Yuasa, Yuta Ito, Kazutaka Nakashima, K. Dreval, R. D. Morin, K. Chiba, Ai Okada, Yuichi Shiraishi, Koichi Murakami, Yasunori Kogure, Koichi Ohshima, Keisuke Kataoka
          </td>
          <td>2025-07-17</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Bladder cancer has been considered as one of the most common urinary malignancies. Growing evidence has indicated that Centromere Protein F (CENPF) is a promising molecular biomarker for many human malignant tumors. However, the role of CENPF in bladder cancer (BC) proliferation, migration, invasion, cell cycle and apoptosis thereof remain unclear. In the present study, high-throughput sequencing technology and bioinformatics analysis were conducted to identify mRNAs profiles in 10 pairs of bladder cancer tissues and adjacent noncancerous tissues. CENPF was overexpression in bladder cancer tissues, and higher in MIBC than NMIBC. We investigated the expression of CENPF in 30 other pairs bladder cancers tissues, and the results were in accordance with the sequencing results. Furthermore, Immunohistochemical staining, showed that strong intensity of CENPF in BC tissues than normal tissues. Increased staining of CENPF was detected of tumor cells in MIBC compared with NMIBC. This suggests that CENPF might be highly expressed in aggressive and invasive tumor cells. Subsequently, in vitro functional experiments also demonstrated that the siRNA interference of CENPF expression significantly weakened the proliferation, migration, invasion and apoptosis of BC cells, and the cells were arrested in the G2/S phase in Cell cycle. Moreover, functional enrichment analyses, lncRNA-miRNA-mRNA, and protein-protein interaction networks revealed that CENPF was potentially involved in carcinogenesis and evolution of bladder cancer. Taken together, these results demonstrated that CENPF may serve as a potential biomarker of tumor occurrence, progression, and even prognosis for bladder cancer. However, further research is needed to be further clarified the pathway mechanisms of CENPF in bladder cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10677-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcea81a86745e041752c09be80ee21f9ca4c7d66" target='_blank'>
              CENPF overexpression in bladder cancer cells enhances proliferation, migration, invasion, and apoptosis
              </a>
            </td>
          <td>
            Ruixiang Song, Xin Chen, Zhensheng Zhang, Huiqing Wang, Jinshan Xu, Shuxiong Zeng, Wentao Zhang, Zhaoyang Sheng, Xudong Yao
          </td>
          <td>2025-07-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The human mitochondrial helicase Twinkle is essential for mitochondrial DNA (mtDNA) replication and integrity. Using biochemical and single-molecule techniques, we investigated Twinkle’s real-time kinetics, including DNA loading, unwinding, and rewinding, and their regulation by its N-terminal Zinc-binding domain (ZBD), C-terminal tail, and mitochondrial SSB protein (mtSSB). Our results indicate that Twinkle rapidly scans dsDNA to locate the fork, where specific interactions halt diffusion. During unwinding, ZBD-DNA interactions and C-terminal tail control of ATPase activity downregulate kinetics, slowing down the helicase. Binding of mtSSB to DNA likely outcompetes ZBD-DNA interactions, alleviating the downregulatory effects of this domain. Furthermore, we show that ZBD-DNA interactions and ATP binding also regulate rewinding kinetics following helicase stalling. Our findings reveal that ZBD and C-terminal tail play a major role in regulation of Twinkle´s real-time kinetics. Their interplay constitutes an auto-regulatory mechanism that may be relevant for coordinating the mtDNA maintenance activities of the helicase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4e9b3c40e208324f97a7ecc98eca6a14e41660" target='_blank'>
              Autoregulation of the real-time kinetics of the human mitochondrial replicative helicase
              </a>
            </td>
          <td>
            Ismael Plaza-G A, María Ortiz-Rodríguez, Seth P Buchanan, Samuel Miguez-Amil, Kateryna M. Lemishko, F. Moreno-Herrero, R. Fernández-Leiro, Grzegorz L. Ciesielski, Borja Ibarra
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Circular RNA (circRNA), a class of evolutionarily conserved, structurally stable, and tissue-specifically expressed noncoding RNA, is increasingly recognized as a key regulator of diverse biological processes and disease pathogenesis, including cancer. While the roles of circRNAs in tumorigenesis are well-documented, their involvement in the tumorigenesis induced by environmental chemical carcinogens (ECCs) remains relatively unexplored. Recent studies have identified aberrant expressions of specific circRNAs during ECC exposure-related carcinogenesis, suggesting their critical regulatory functions. Given their unique structure and broad regulatory roles, circRNAs exhibit great potential as diagnostic, therapeutic, and prognostic biomarkers for ECC exposure-associated cancers. This review summarizes the characteristics and functions of circRNAs, as well as the potential regulatory mechanisms in ECC exposure-induced cancer and the dysregulations of circRNAs caused by ECCs. We highlight the complexity and heterogeneity of circRNA regulatory networks, emphasizing the need for integrated and dynamic investigations to fully elucidate the underlying mechanisms. Future research efforts should prioritize biomarker studies to facilitate the prevention, early detection, and effective treatment of ECC exposure-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760c4eaf87c1e364f138dd2080aea7e51f03d039" target='_blank'>
              Circular RNAs: A Potential Regulator in Environmental Chemical Carcinogenesis.
              </a>
            </td>
          <td>
            Huijun Huang, Yiqi Zhou, Yiyan Huang, Jiaxin Wang, Shiyi Ouyang, Meiqi Lan, Lieyang Fan, Yun Zhou
          </td>
          <td>2025-07-17</td>
          <td>Chemical research in toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fd329abfeae45d934694f31530eb9db169304e" target='_blank'>
              A mouse model of PTEN Hamartoma Tumour Syndrome reveals that loss of the nuclear function of PTEN drives macrocephaly, lymphoid overgrowth, and late-onset cancer
              </a>
            </td>
          <td>
            Priyanka Tibarewal, V. Rathbone, S. Conduit, G. Classen, Fiona M Black, Mohammad Amin Danesh, Georgia Constantinou, Zeinab Asgarian, Koujiro Tohyama, N. Kriplani, Virginia Alvarez Garcia, Elizabeth Foxall, Djenat Belarbi, M. Leverve, Wayne Pearce, Mahreen Adil, Zofia Varyova, Lucia Conde, Adriana Alves, G. Masson, Roger L. Williams, Adrienne M Flanagan, Javier Herrero, Katerina Stroud, M. Tischkowitz, K. Lachlan, Cheryl L Scudamore, Mark G H Scott, Nicholas R. Leslie, N. Kessaris, B. Vanhaesebroeck
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen T C Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Christelle M. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Brain tumors are one of the most severe types of malignant tumors and glioma accounts for ~80% of malignant brain tumors. The current treatment methods for glioma are limited and patients with glioma often experience relapse following treatment, which leads to a poor prognosis for these patients. Therefore, novel therapeutic targets and methods urgently need to be explored. The present review screened studies that mainly focused on the epigenetic regulation of small guanosine triphosphate (GTP)ase in glioma. These small GTPases participate in most cellular biological processes, including differentiation, proliferation, cell migration, apoptosis, vesicle and organelle dynamics and transport, nuclear dynamics and cytoskeleton regulation. Due to the diversity and importance of the biological functions of small GTPases, an increasing number of studies have focused on them; however, the incidence of changes in the gene structure of small GTPases is considered to be low in glioma. Several studies have shown that the abnormal expression of genes encoding small GTPases is often influenced by epigenetic regulation in glioma. Epigenetic regulation is a dynamic and reversible process, which implies that the reversal of abnormal epigenetic modifications is a potential treatment strategy for glioma. These previous studies, which are summarized in the present review, not only provide new therapeutic targets and prognostic markers, but also provide information regarding the treatment of glioma. The current review may provide valuable insights for future research and promote the clinical translation of relevant research results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b0933c82de7174662d27dd036b6327ca1cf162" target='_blank'>
              The role of abnormal epigenetic regulation of small GTPases in glioma (Review)
              </a>
            </td>
          <td>
            Mingyang Zhang, Yimin Huang, Qiang Zhang, Xiaoyan Zhang, Lumei Kang, Jianguo Wang
          </td>
          <td>2025-07-02</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As a diverse family of diseases, cancer is unified by a set of common dysfunctions, such as limitless growth potential and an insensitivity to antigrowth signals. These shared overarching biological processes have been termed the hallmarks of cancer. To better understand the root cause of cellular dysregulation, intense molecular characterization of tumors has utilized DNA, RNA, and protein measurement techniques to produce proteogenomic data. In large cancer cohort studies, genomic and proteogenomic data have frequently identified many cancer hallmarks including cell cycle and cell signaling. However, altered metabolism, a known cancer hallmark, is not as clearly identified in mutation screens or differential expression analyses. Here, we introduce a new computational method to identify changes in cellular regulation by focusing on the mRNA/protein relationship. We create a metric, Δ_corr, to capture when the mRNA/protein correlation changes significantly between tumor and normal tissues and show that it is distinct from differential expression and also not associated with DNA mutation profiles. Our method clearly highlights altered metabolic pathways across multiple tumor types. Δ_corr gives researchers a new perspective on the dysfunction of tumor cells and introduces a novel method for proteogenomic data integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0188673355274102e7547e046958401284e61db8" target='_blank'>
              Activating Cancer Hallmarks through Changes in mRNA/Protein Regulation.
              </a>
            </td>
          <td>
            Jose Humberto Giraldez Chavez, Nathaniel Barton, Caleb M Lindgren, Bryn Mendenhall, Benjamin Kimball, Samuel H. Payne
          </td>
          <td>2025-07-17</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) is closely associated with abnormal DNA repair and genome instability. The bone marrow microenvironment, particularly myeloma associated macrophages (MΦs) is critical to the progression of MM. However, there is limited understanding on the role of MΦs in DNA repair in MM. Here, we found that MΦs regulated DNA repair in MM cells by the CXCL5/8-CXCR2 axis. By promoting non-homologous end joining rather than homology-directed repair, MΦs increased the probability of chromosomal translocations in MM cells. Furthermore, clinical data confirmed that MΦs are closely associated to the increased genetic variations of MM patients' primary cells. The study elucidates a mechanism by which MΦs regulates DNA repair in MM in the microenvironment and provides a potentially new target to counter MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7463e4f2bfa77ad622fca89d072c6a09f3dba6b9" target='_blank'>
              Macrophages promote aberrant DNA repair in multiple myeloma via the CXCL5/8-CXCR2 axis.
              </a>
            </td>
          <td>
            Mengmeng Dong, D. He, Jinna Zhang, Haimeng Yan, Haoguang Chen, E. Zhang, Yili Feng, Jingsong He, Xi Huang, Guoqiao Chen, Xiuna Sun, F. Cheng, Huiyao Gu, Huanping Wang, Anyong Xie, Zhen Cai, Cai Lab
          </td>
          <td>2025-06-26</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA methylation is essential for transcriptional regulation and the maintenance of chromosome stability, and its precise inheritance upon DNA replication is indispensable for cellular homeostasis. The DNMT1/UHRF1 complex is critical in copying DNA methylation with accessory proteins, including CDCA7 and HELLS. The DNMT1/UHRF1 complex is also crucial for maintaining DNA methylation at imprinting control regions during preimplantation development against genome-wide DNA demethylation, an essential process for early embryos to acquire totipotency. Pathogenic variants in the genes involved in the mechanism of DNA methylation maintenance result in immunodeficiency, centromeric instability, and facial anomalies (ICF) syndrome, multilocus imprinting disturbance (MLID), autosomal dominant cerebellar ataxia-deafness and narcolepsy (ADCADN), neuropathy, hereditary sensory, type 1E (HSN1E), Kleefstra syndrome 1 (KLEFS1), and immunodeficiency 96 (IMD96). This review discusses recent progress in understanding the possible molecular pathogenesis of these diseases, with a particular focus on recent progress in understanding ICF syndrome and MLID.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0311b04391f3ff48779575d6888611f38c56b5" target='_blank'>
              Congenital diseases with defects in DNA methylation maintenance: Focusing on ICF syndrome and multilocus imprinting disturbance.
              </a>
            </td>
          <td>
            Motoko Unoki
          </td>
          <td>2025-06-12</td>
          <td>Genes & genetic systems</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Previously, we identified that mutant p53 expression in cancer cells promotes engulfment of neighbouring cancer cells to form cell-in-cell (CIC) structures. This process gave mutant p53 cells an advantage in tumour formation in mouse xenograft experiments. TP53 can be found mutated at nearly every amino acid in cancers and mutant p53 expression is associated with various GOF (Gain-of-function) processes, including cancer cell invasion, metastasis, stemness and drug resistance. In the current manuscript, we identified SH3BGRL (Src homology 3 binding glutamate rich protein like) as a mutant p53-regulated gene and investigated to what extent SH3BGRL expression and cell engulfment are responsible for mutant p53-dependent anchorage-independent growth and chemoresistance. We demonstrate that mutant p53 expression drives cell engulfment in which the mutant p53 host cell moves in the direction of the target internal cell to form CIC structures. This is therefore more reminiscent of cell engulfment rather than cell entosis, in which cells invade into host cells. Using NGS (Next Generation Sequencing), we identified novel target genes of mutant p53 and demonstrate that cell engulfment requires SH3BGRL expression. We generated mutant p53 and p53 KO cell lines that stably overexpressed SH3BGRL and determined that SH3BGRL promotes etoposide resistance in mutant p53 cells and anchorage-independent growth independent of mutant p53 expression. Through FACS sorting of pure cell engulfing (CIC) populations, we could also show that engulfing cells have an enhanced etoposide resistance. These data suggest that SH3BGRL and cell engulfment are required for certain GOFs of mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba6ff758bccb84d82f6118096ce3f5daf639f6e" target='_blank'>
              Mutant p53 induces SH3BGRL expression to promote cell engulfment
              </a>
            </td>
          <td>
            Lobsang Dolma, Mary I Patterson, Antonia Banyard, C. Hall, Steven J Bell, Wolfgang Breitwieser, Sudhakar Sahoo, John Weightman, Maria Pazos Gil, Garry Ashton, C. Behan, Nicola Fullard, Lewis D Williams, P. Muller
          </td>
          <td>2025-07-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) tumours are classified by pathology into several distinct subtypes. Gene expression profiling has revealed transcriptional heterogeneity across NEPC, but this is rarely considered in the context of variation between pathologies. Diagnosis typically relies on immunohistochemical markers (CHGA, SYP, NCAM1) and genomic alterations in RB1, PTEN and TP53. We hypothesized that NEPC pathologies have unique transcriptional features. Single-cell RNA sequencing of 18 632 tumour cells from nine patient-derived xenograft models representing five pathologies (small-cell and large-cell neuroendocrine carcinomas, focal neuroendocrine differentiation (Focal NED), low-grade neuroendocrine and amphicrine) demonstrated pathway-specific enrichment. Focal NED and amphicrine tumours exhibited cellular subpopulations enriched for KRAS, IL2-STAT5 and TNF signalling pathways, absent in small- and large-cell carcinomas, which were instead enriched for Myc and E2F pathways. Furthermore, focal NED cells exhibited minimal clonal divergence from adjacent adenocarcinoma cells, while small cell carcinoma cells were clonally distinct. These data underscore significant transcriptional variation among NEPC pathologies, highlighting focal NED's unique biological context and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9094a51a0844d2931b4655b974ca910515f91f" target='_blank'>
              Single-cell transcriptomics redefines focal neuroendocrine differentiation as a distinct prostate cancer pathology.
              </a>
            </td>
          <td>
            Rosalía Quezada Urban, S. Keerthikumar, A. Clark, Hong Wang, Belinda Phipson, Andrew Bakshi, Andrew Ryan, Heather Thorne, R. Taylor, Mitchell G. Lawrence, G. Risbridger, Roxanne Toivanen, D. L. Goode
          </td>
          <td>2025-07-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High expression of nucleotide synthetic enzyme thymidylate synthase (TYMS) is responsible for the resistance to fluorouracil (FU) treatment and worse survival in colorectal cancer (CRC). Herein, we revealed that pleckstrin-2 (PLEK2) cooperated with YTHDF2 to enhance TYMS mRNA stability in CRC via an m6A dependent manner. Silencing of PLEK2 led to the degradation of TYMS mRNA that suppressed DNA replication, which activated p53/p21 signaling and consequent inhibition of CRC cell proliferation via the cellular senescence. Additionally, PLEK2 is also required for CRC cell migration, invasion and stemness-like properties. PLEK2 inhibition is sufficient to ameliorate the progression of AOM/DSS-induced CRC. Together, our study identified PLEK2 as a key regulator for the progress of CRC via the regulation of TYMS expression, and demonstrated that PLEK2 is a novel therapeutic target for CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05782-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cdd75a16ceb80a9546051cd0900e34ba87bf7bd" target='_blank'>
              Pleckstrin-2 promotes the progression of colorectal cancer via YTHDF2-mediated TYMS mRNA stability
              </a>
            </td>
          <td>
            Qian Zhou, Yanxia Li, Xiaomei Li, Shujing Zhang, Ying Wang, Zhuoran Li, Xia Wang, Yuan Li, Jingxin Li, Chunhua Lu, Yuemao Shen, Baobing Zhao
          </td>
          <td>2025-07-19</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) represent a distinct subpopulation of tumor cells characterized by enlarged or multiple nuclei and aneuploidy. PGCCs are products of genomic instability, possessing cancer stem cell properties and exhibiting significant resistance to radiotherapy and chemotherapy. They can generate highly invasive daughter cells through asymmetric division, exhibiting epithelial-mesenchymal transition characteristics, and facilitating tumor recurrence and metastasis. In vivo, PGCCs with daughter cells in tumor tissue can migrate and infiltrate into the forefront stroma to form tumor budding, which are closely related to solid tumor recurrence, metastasis, and drug resistance. Studies have shown that inhibiting sphingolipid enzyme acid ceramidase or regulating autophagy can reduce the production of PGCCs with daughter cells. Under appropriate induction conditions, PGCCs with daughter cells can be induced to differentiate into benign tissues such as adipocytes, chondrocytes, and osteocytes, inhibiting their malignant proliferation and invasive destruction. This study reviewed the recent research developments regarding PGCCs, mainly explored the endogenous mechanisms of PGCCs formation and their malignant phenotype, as well as the process of tumor budding formation in vivo and potential therapeutic strategies targeting PGCCs. The main novelty of this study lies in exploring the translation of PGCCs basic research into the clinical pathological prognostic role of tumor budding, which can reveal the potential mechanism of PGCCs/tumor budding formation at the molecular level, providing theoretical basis for prognosis assessment, monitoring of recurrence and metastasis risks, as well as improving drug resistance and targeted therapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a3661c8ef6e9fb0fa2a5fd9bf5fffcb0f097f8" target='_blank'>
              Polyploid giant cancer cells and tumor budding: translation from basic research to clinical application
              </a>
            </td>
          <td>
            Peng Huang, Rong Wu, Zhimou Yang, Yuwei Li, Fei Fei, Yongjun Yu
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Q. Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Objective This study aims to uncover the molecular biology and immune microenvironment of gynecological mesonephric-like adenocarcinoma (MLA). Methods To determine the comprehensive characteristics of MLA, 17 patients with MLA were retrospectively enrolled in this study. Whole-exome sequencing and mRNA sequencing were performed to explore the molecular features. The biological differences between MLAs and epithelial-initiated gynecologic tumors reported in The Cancer Genome Atlas database were also analyzed. Results KRAS mutations (82.4%) were considered the driving mechanism and were co-mutated with PIK3CA (47.1%) and SPOP (23.5%), but their functions were mutually exclusive. In addition, pathways and genes associated with kidney development were upregulated in MLA patients. Compared with adjacent tissues and common gynecological tumors in The Cancer Genome Atlas, Th2 signature and resting mast cells account for the majority in MLAs, rendering an immunosuppressive TME. Particularly, the expression levels of IFNG, IFN6, and IFN1 KRAS_SPOP group, significantly lower than the rates found in KRAS_PIK3CA group. KRAS_SPOP mutant MLAs, exhibited reduced immune infiltration in their tumor microenvironment. Conclusion This is the first study to demonstrate the comprehensive molecular characteristics of MLA and detect biologically distinct subtypes of KRAS mut/SPOP mut and KRAS mut/PIK3CA mut MLAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2ab50137a98d92f1666e3c4e9b08052c52e0d10" target='_blank'>
              Immune microenvironment heterogeneity characterizes biologically distinct KRASmut/SPOPmut and KRASmut/PIK3CAmut mesonephric-like adenocarcinoma subtypes revealed by integrated whole-exome and transcriptomic profiling
              </a>
            </td>
          <td>
            Jing Zeng, Qingli Li, Kemin Li, Lu Yang, Lian Xu, Wei Wang, Kaixuan Yang, Qingbo Wei, Jin Wang, Changbin Zhu, Rutie Yin
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a61dbd5f51e45226cc5e1a9ea36770552ca20" target='_blank'>
              Fragmentation of nucleoli in senescent cancer cells is associated with increased levels of polyadenylated transcripts derived from noncoding regions of rDNA units.
              </a>
            </td>
          <td>
            Jana Sochorová, Emilie Lukášová, Eva Volfová Polanská, Kateřina Řehůřková, A. Kovařík
          </td>
          <td>2025-06-23</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Chromatin remodeling plays an integral part in endometrial homeostasis, having roles in the maintenance of cell identity, epithelial integrity, and prevention of endometrial disease. Chromodomain-helicase-DNA-binding protein 4 (CHD4) is a chromatin remodeling protein and member of the NuRD complex, which predominantly represses transcription. CHD4 is mutated in endometrial carcinoma, with most mutations leading to loss of function. CHD4 has been identified as a tumor suppressor and regulator of stemness in endometrial carcinoma cells, but little is known about the tissue-specific roles of CHD4 in the endometrial epithelia in vivo. We generated a conditional Chd4 floxed allele and combined it with BAC-Sprr2f-Cre to drive Chd4 loss in the endometrial epithelium. Consistent with previous reports, BAC-Sprr2f-Cre expression is absent in the oviducts, ovaries, and kidneys, and it shows variegated expression within the endometrial epithelium. Loss of CHD4 was confirmed by immunohistochemistry, and stained cells were quantified to determine the percentage of endometrial epithelial cells with and without CHD4. Compared to the glandular epithelium, the extent of CHD4 loss was higher in the luminal epithelium and unaffected by age. Mice with conditional knockout of Chd4 had normal endometrial histology. A 6-month breeding trial was performed to assess the functional effects of endometrial epithelial Chd4 loss on fertility. No difference in litter size, mean number of pups per litter per dam, or pup weight was found between genotypes. These findings demonstrate that Chd4 conditional loss using BAC-Sprr2f-Cre is not sufficient to alter the structure and function of the endometrial epithelium or drive tumorigenesis. As CHD4 is frequently co-mutated with other cancer driver genes such as TP53, PIK3CA, and PTEN, future mouse modeling efforts emulating patient mutational profiles might provide insight into the role of CHD4 in endometrial carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acfbf17a9021bcd1d6e7def98b028c7986feabf0" target='_blank'>
              Generation and characterization of a mouse model of conditional Chd4 knockout in the endometrial epithelium.
              </a>
            </td>
          <td>
            Shannon K Harkins, Hilary Skalski, Abigail Z. Bennett, Laura Pavliscak, Amelia R. Arendt, Lauren Wood, Genna E. Moldovan, Ronald L. Chandler
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The mechanisms of cancer are discussed by analyzing the characteristics of the functional state and biological behavior of the abnormal nuclear cells. The abnormal nuclear cells with abnormal nuclear structure and function are a kind of sick cell or functional defect cells having existed in human body for a long time. The abnormal nuclear cells are resulted from the nuclear damage caused by the radiation, viruses and various carcinogenic compounds. Some of genes in human body are expressed, some are not expressed for life. The expressional genes are functional genes, the genes never expressed for life in human body are dormant genes or sealed genes. The nuclear damages destroy cell state of differentiation, affect gene expressional regulation and change gene expressional profiling, resulting in loss of expression of the functional genes and reactivation of the sealed genes; which finally leads to cancer, aging and other chronic refractory diseases. The cancer is not resulted from the genetic mutations or chromosomal aberrations, but rather the reactivation of genes involved in proliferation due to the nuclear damage. The biological characteristics of the cancer cells, such as the shedding and metastasis, immune tolerance, uncontrolled, loss of contact inhibition function and so on, all originate from the nuclear aberrant cells. The nuclear damage can trigger the genes that drive mitosis, leading to cancer. Thus, re-sealing the several genes that trigger the proliferation may completely prevent or cure cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd09554e196fbeabd1e2fd9b39e2eabe92efdf78" target='_blank'>
              Cancers Are Derived from the Disruption of Cell Differentiation
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-06-23</td>
          <td>Cancer Research Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cell-free DNA (cfDNA) is a promising biomarker and carries information about physiological and pathological states. Nucleosome and transcription factor footprints have been extensively studied based on double-stranded cfDNA sequencing; however, protein-DNA interaction footprints in single-stranded cfDNA fragments remain undercharacterized.


METHODS
We used a single-stranded library preparation and sequencing method to investigate a new characteristic of cfDNA-strand specificity, based on the depth differences between the 2 DNA strands. The plasma cfDNA of 34 healthy individuals was analyzed to evaluate strand-specific bias on chromosomes and the mitochondrial genome. Then, the cfDNA samples were compared between 17 males and 17 females, as well as between 14 lung cancer patients and 18 noncancer controls.


RESULTS
Short cfDNA (<160 bases) had higher strand-specific bias than long fragments. CfDNA strand-specific-biased regions were more enriched in chromosomes with higher gene densities and enriched in promoters, exons, 5' untranslated region (UTR), 3' UTR, and CpG islands. Genomic regions with higher CpG densities and lower methylation levels were more likely to generate strand-specific bias. For the mitochondrial DNA (mtDNA), the cfDNA strand-specific bias was pronounced in the noncoding region, exhibiting the footprints of mtDNA replication and transcription. Significant molecular sex differences could be revealed on cfDNA at the gene level, and distinguished features of lung cancer patients were also discovered.


CONCLUSIONS
As a proof-of-concept, we demonstrated the existence of a cfDNA strand-specific characteristic, which reflects footprint signatures including transcription and methylation. This study provides insights into the biological properties of cfDNA and a potential novel role as biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fe1ad62eb2836556b498c0ff8038957069e336" target='_blank'>
              Cell-Free DNA Comprises the Strand-Specific Characteristic Associated with Transcription and Methylation.
              </a>
            </td>
          <td>
            Changlin Yang, Rijing Ou, Guodan Zeng, Wei Chen, Yeqin Wang, Xinxin Wang, Lingguo Li, Jinjin Xu, Wenwen Zhou, Haiqiang Zhang, Jianhua Yin, Xin Jin, Yan Zhang
          </td>
          <td>2025-06-16</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer and the leading cause of cancer-related death among women worldwide. Triple negative breast cancer (TNBC) represents 15-20% of all incident breast cancers and is a highly aggressive subtype. Neo(adjuvant) chemotherapy (NAC) is standard for early stage TNBC; however, for those with residual breast cancer following NAC, recurrence is common and associated with few effective therapeutic approaches. Thus, identification of novel resistance mechanisms and alternative treatment strategies remain an unmet clinical need. Circular RNAs (circRNAs) are a newly identified class of largely noncoding RNA molecules with covalently closed circular structures. Recent studies suggest that circRNAs play crucial roles in regulating tumor development, progression and chemoresistance. However, the role of circRNAs in mediating chemotherapy resistance remains unknown. Methods: As a first step towards identifying circRNAs that participate in the development of chemoresistance, and to determine if targeting such circRNAs may be an efficacious strategy, doxorubicin-resistant (Doxo-R), paclitaxel-resistant (PTX-R), and double-resistant (DP-R) cell lines were generated from MDA-MB-231 cell lines. Human circRNA microarrays were utilized to profile the expression of approximately 14,000 known circRNAs in normal breast tissue, matched TNBC patient-derived xenografts (PDX) generated prior to and following NAC, and TNBC chemosensitive and chemoresistant cell lines. Top hits were validated using RT-PCR. Bioinformatic approaches were employed to identify circRNA-miRNA-mRNA networks regulated by differentially expressed circRNA transcripts. A reverse genetics approach was also employed using CRISPR to interrogate the contributions of over 4000 specific circRNA in driving the development of resistance to doxorubicin and paclitaxel. Results: circRNA microarray profiling identified 429 and 310 transcripts differentially expressed in doxorubicin and paclitaxel resistant cells, respectively, compared to parental chemosensitive cell lines (|FC| ≥ 1.5; p value < 0.05). Given that circRNAs can act as microRNA (miRNA) sponges, thereby inhibiting miRNA activity, we computationally identified circRNA-miRNA pairs, and assessed the abundance of mRNA transcripts predicted to be targets of paired miRNAs via RNA-seq. A differentially expressed circRNA-miRNA-mRNA network was further constructed. Dysregulated canonical pathways and gene ontology (GO) enrichment analysis were determined from these networks and revealed multiple GO term included negative regulation of miRNA transcription, miRNA and RNA metabolic process, and endoplasmic reticulum-associated degradation (ERAD) pathway. The CRISPR-Cas13 screen is ongoing at this time but results will be available prior to the meeting. Conclusions: We have identified cancer specific circRNAs in TNBC cell lines, PDX models and patient tumors in the context of chemotherapy sensitive and chemotherapy resistant disease. Specific biological processes, pathways and signaling cascades have been implicated in the down-stream consequences of dysregulated circRNAs and in the potential development of resistance. Our ongoing studies seek to identify specific circRNAs sufficient to drive resistance to chemotherapy. Outcomes of these studies highlight the potential of specific circRNAs to serve as predictive/prognostic biomarkers and novel therapeutic targets for advanced forms of TNBC.
 Citation Format: Xiyin Wang, Michael J. Emch, Xiaojia Tang, Esther P.B. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, Krishna R. Kalari, John R. Hawse. The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb284df6818a5ca75827ea02126a84e48b2886e9" target='_blank'>
              Abstract P3-02-22: The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance
              </a>
            </td>
          <td>
            Xiyin Wang, M. Emch, Xiaojia Tang, E. Rodman, Liewei Wang, J. Boughey, Matthew P. Goetz, K. Kalari, John R. Hawse
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) represents the most common solid renal malignancy, accounting for 90% of kidney cancers and 3% of all malignancies. Clear cell RCC (ccRCC) is the predominant histologic subtype, comprising roughly 80% of cases. Despite a 5-year survival rate of 75%, up to 30% of patients with localized ccRCC eventually develop recurrence or distant metastases. Mitotic spindle organizing protein 2A (MZT2A), also known as FAM128A, GCP8, or MOZART2A, is located on chromosome 2q21.1 and encodes a 158-amino acid protein first described by Hutchins et al. in 2010. To date, limited studies have linked MZT2A to tumorigenesis, primarily in lung and gastric cancers. The tumor microenvironment (TME), particularly tumor-infiltrating immune cells, plays a critical role in cancer progression and prognosis. Using paired tumor and normal samples from TCGA, we assessed MZT2A expression across multiple cancer types. Collectively, these findings suggest that MZT2A is involved in immune cell infiltration and the tumor microenvironment, may promote KIRC progression, and is overexpressed in cases with poor outcomes, supporting its potential as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d43bd634f019908094dd923cc5fef3d5a13d2f5" target='_blank'>
              Integrated transcriptomic analysis identifies MZT2A as a prognostic and immune-related biomarker in KIRC.
              </a>
            </td>
          <td>
            Yunjing Zhang, Zekun Xin, Xinnuo Li, Yanmei Chen
          </td>
          <td>2025-07-15</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bladder cancer remains a major global health challenge due to its high incidence, recurrence rates, and reliance on invasive diagnostic procedures. While conventional methods like cystoscopy and urine cytology are standard, they often fall short in sensitivity, especially for early-stage disease. Epigenetic alterations, particularly DNA methylation, are now recognized as critical in bladder cancer development and progression. Non-invasive urine-based DNA methylation assays have emerged as promising diagnostic and monitoring tools by detecting tumor-derived molecular changes in exfoliated DNA. Notably, hypermethylation of tumor suppressor gene promoters such as CDKN2A, RASSF1A, and DAPK, as well as hypomethylation in oncogene-associated regions like MYC and CCND1, have shown strong associations with bladder cancer presence and severity. Recent advancements in detection technologies, including methylation-specific PCR (MSP), droplet digital PCR (ddPCR), and Next-Generation Sequencing (NGS) have significantly improved sensitivity and specificity, enabling earlier diagnosis, better risk stratification, and the development of personalized management strategies. Clinical studies also point to the potential of novel markers like TWIST1, NID2, and mDMRTA2 in enhancing diagnostic accuracy. Despite these advances, broader clinical validation and standardized testing protocols are essential for integrating methylation assays into routine clinical practice. These assays represent a transformative shift toward non-invasive, accurate, and patient-centric bladder cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e2eae715ddf68f75b437c1947a0e2e08d44b12c" target='_blank'>
              Epigenetic Insights into Bladder Cancer: The Role of Urine DNA Methylation Assays: Mini Review
              </a>
            </td>
          <td>
            Mohamed Hussein
          </td>
          <td>2025-06-17</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dysregulated RNA splicing is a well-recognized characteristic of colorectal cancer (CRC); however, its intricacies remain obscure, partly due to challenges in profiling full-length transcript variants at single-cell level. Here, we employ high-depth long-read scRNA-seq to define the full-length transcriptome of colorectal epithelial cells in 12 CRC patients, revealing extensive isoform diversities and splicing alterations. Cancer cells exhibited increased transcript complexity, with widespread 3'-UTR shortening and reduced intron retention. Distinct splicing regulation patterns were observed between intrinsic-consensus molecular subtypes (iCMS), with iCMS3 displaying even higher splicing factor activities and more pronounced 3'-UTR shortening. Furthermore, we revealed substantial shifts in isoform usage that result in alterations of protein sequences from the same gene with distinct carcinogenic effects during tumorigenesis of CRC. Allele-specific expression analysis revealed dominant mutant allele expression in key oncogenes and tumor suppressors. Moreover, mutated PPIG was linked to widespread splicing dysregulation, and functional validation experiments confirmed its critical role in modulating RNA splicing and tumor-associated processes. Our findings highlight the transcriptomic plasticity in CRC and suggest novel candidate targets for splicing-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c51fa89f4862ae8ddeae87716369c8a92a2e111" target='_blank'>
              Systematic characterization of full-length RNA isoforms in human colorectal cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Ping Lu, Yu Zhang, Yueli Cui, Y. Liao, Zhenyu Liu, Zhi-Jie Cao, Jun-E Liu, Lu Wen, Xin Zhou, Wei Fu, Fuchou Tang
          </td>
          <td>2025-07-22</td>
          <td>Protein & cell</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07bd79e38eeb67868a3c867814b6e2e9bf7277a8" target='_blank'>
              Sequential PIDD1 auto-processing is essential for ploidy control in the liver and heart
              </a>
            </td>
          <td>
            F. Eichin, V. Sladky, Matthäus A. Reiner, M. Leone, Ernesto Abila, André F. Rendeiro, Ralph Böttcher, M. Dahlhoff, Thomas Kolbe, A. Villunger
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) comprise ∼7% of pediatric cancers, with over half classified as rhabdomyosarcoma (RMS), equating to approximately 4.5 cases per 1 million children. Germline mutations in the TP53 tumor suppressor gene (MutTP53), most notably in Li-Fraumeni Syndrome (LFS), represent the strongest heritable risk factor for RMS and are associated with poorer overall survival. Despite this, a transcriptomic signature specific to LFS-associated RMS (LFS-RMS) remains undefined. To address this, we performed droplet-based single-cell RNA sequencing (scRNA-seq; 10x Genomics) on RMS tumors and matched healthy muscle tissue from Trp53 R172H/WT mice, harboring a mutation analogous to the human TP53 R175H hotspot. Tumors were collected at endpoint from male (n = 2) and female (n = 3) mice. Cell types were identified through differential gene expression (DGE) analysis and annotated via consensus across machine learning-based tools. Trajectory analysis using Monocle3 revealed lineage transitions, while Generalized Additive Models (GAMs) identified genes associated with the progression from non-malignant to cancer-associated states. We observed consistent alterations in cell subtype proportions between tumor and healthy tissue, including a significant reduction in the M1/M2 macrophage proportion. Trajectory analysis identified key transcripts driving the transition from muscle-associated to cancer-associated fibroblasts and myogenic cells. Pathways enriched in GO analysis suggest that during the transition to cancer-associated phenotypes, cells in LFS-RMS adopt gene programs that reduce susceptibility to immune cell invasion in the tumor microenvironment. Our findings provide the first single-cell map of LFS-RMS, revealing dynamic transcriptional states and biomarkers of cells states associated with tumorigenesis. These insights pave the way for improved therapeutic approaches tailored to the complex cellular heterogeneity of LFS tumors. Further, these findings may broadly enhance RMS diagnostics and precision oncology approaches.



 Ashby Kissoondoyal, Paula R. Quaglietta, Brianne Laverty, Safa Majeed Grant, David Malkin. Identifying single-cell transcriptomic signatures and changes underlying the evolution of fusion-negative rhabdomyosarcoma in Li-Fraumeni Syndrome mice [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B042.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f326946c46ad1fff7d92ca7092d9a3464822119" target='_blank'>
              Abstract B042: Identifying single-cell transcriptomic signatures and changes underlying the evolution of fusion-negative rhabdomyosarcoma in Li-Fraumeni Syndrome mice
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, Paula R. Quaglietta, B. Laverty, Safa Majeed Grant, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) is important in assisted reproductive technologies. However, its reprogramming efficiency remains low. A considerable drawback of SCNT-cloned embryos is the reduction in telomerase activity, which is crucial for DNA stability and genetic and epigenetic reprogramming. The present study aimed to examine the effects of madecassic acid (MA), a potent telomerase activator, on the developmental rate, embryonic genome activation, and implantation potential of SCNT-derived bovine embryos. The treatment of bovine signal cell-cloned zygotes with 3.0 μg/mL MA significantly increased embryo cleavage (71.5%) and blastocyst rate (28.1%) compared with that in non-treated (control) SCNT-cloned bovine embryos. In addition, MA treatment enhanced the bovine granulosa cells' telomerase activity and telomerase expression are assessed using qTRAP assay and ELISA. Of note, MA enhanced the expression of embryonic genome activation (EGA)-related genes including NFYA, SP1, DPRX, GSC, CTNNB1, DUX, and ARGFX in MA-treated cloned embryos compared to the control group. Moreover, MA-treatment of cloned embryos showed substantially less DNA damage than the control SCNT embryos. Mechanistically, MA activation of telomerase reverse transcriptase (TERT) significantly enhanced the nuclear localization of β-catenin and c-Myc and improved EGA. Reduction in the nuclear localization of this triose may be the leading cause of reduced EGA in cloned embryos. In conclusion, MA impacted the EGA reprogramming and development of cloned bovine embryos via probable activation of TERT. This telomerase activator may have the application of improving SCNT-cloned bovine embryos.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b3db694f46c9e1d3c6cd7a013e5c53b07ffe0c" target='_blank'>
              Activation of telomerase via madecassic acid enhances the developmental competence of the SCNT-derived bovine embryos
              </a>
            </td>
          <td>
            Zaheer Haider, Safeer Ullah, Tahir Muhammad, C. Perera, Muhammad Tayyab Khan, Asif Jan, Seung-Eun Lee, Seo-Hyun Lee, Sung Woo Kim, Muhammad Idrees, Il-Keun Kong
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background and Objectives: Colorectal cancer (CRC) is a major cause of cancer morbidity and mortality worldwide. Genetic and epigenetic changes, especially DNA methylation alterations, are key in CRC development. LINE-1 hypomethylation marks global DNA methylation loss and genomic instability, making it a potential early CRC biomarker. This study investigates the methylation status of LINE-1 in colorectal adenocarcinoma, precancerous lesions (tubular and serrated adenomas), and the surrounding normal mucosa, aiming to elucidate its role as an epigenetic marker in early colorectal tumorigenesis. Materials and Methods: Paired lesion and normal tissue samples from 66 patients were analyzed for LINE-1 methylation at three CpG sites using bisulfite pyrosequencing. Results: Adenocarcinomas and tubular adenomas showed significant hypomethylation, especially at loci A and B, while serrated adenomas exhibited no significant differences. Conclusions: LINE-1 hypomethylation is associated with colorectal tumorigenesis, with distinct patterns observed between tubular and serrated adenomas, indicating distinct pathways forming and progressing specific adenomas. These findings support the potential of LINE-1 methylation as an early epigenetic biomarker for CRC risk stratification and highlight the need for further research into its clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f329dd204061f5e39a657b56f9a71cd23ad452f" target='_blank'>
              Analysis of LINE-1 DNA Methylation in Colorectal Cancer, Precancerous Lesions, and Adjacent Normal Mucosa
              </a>
            </td>
          <td>
            I. Kildusiene, Ryte Rynkeviciene, A. Kačėnienė, Rima Miknaite, K. Suziedelis, G. Smailytė
          </td>
          <td>2025-07-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Genomic imprinting is required for normal development, and abnormal methylation of differentially methylated regions (iDMRs) controlling the parent of origin-dependent expression of the imprinted genes has been found in congenital disorders affecting growth, metabolism, neurobehavior, and in cancer. In most of these cases the cause of the imprinting abnormalities is unknown. Also, these studies have generally been performed on a limited number of CpGs, and a systematic investigation of iDMR methylation in the general population is lacking. Results By analysing a vast number of either in-house generated or online available whole-genome methylation array datasets of unaffected individuals, and patients with complex and rare disorders, we determined the most common iDMR methylation profiles in a large population and identified many genetic and non-genetic factors contributing to their variability in blood DNA. We found that methylation variability was not homogeneous within the iDMRs and that the CpGs closer to the ZFP57 binding sites are less susceptible to methylation changes. We demonstrated the methylation polymorphism of three iDMRs and the atypical behaviour of several others, and reported the association of 25 disease- and 47 non-disease-complex traits as well as 15 Mendelian and chromosomal disorders with iDMR methylation changes. The most significantly associated complex traits included ageing, intracytoplasmic sperm injection, African versus European ancestry, female sex, pre- and postnatal exposure to pollutants and blood cell type compositions, while the associated genetic diseases included Down syndrome and the developmental disorders with molecular defects in the DNA methyltransferases DNMT1 and DNMT3B, H3K36 methyltransferase SETD2, chromatin remodelers SRCAP and SMARCA4 and transcription factor ADNP. Conclusions These findings identify several genetic and non-genetic factors including new genes associated with genomic imprinting maintenance in humans, which may have a role in the aetiology of the diseases with imprinting abnormalities and have clear implications in molecular diagnostics. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00612-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5be47c6d920f5616fee68f2d0479c030325ef78" target='_blank'>
              Identification of genetic and non-genetic modifiers of genomic imprinting through screening of imprinted DMR methylation in humans
              </a>
            </td>
          <td>
            Francesco Cecere, R. Relator, Michael A. Levy, Ankit Verma, H. McConkey, B. H. Mele, Laura Pignata, Carlo Giaccari, Emilia D'Angelo, Subham Saha, Abu Saadat, A. Sparago, Claudia Angelini, F. Cerrato, B. Sadikovic, Andrea Riccio
          </td>
          <td>2025-07-24</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Y-chromosome gene copy number variations (CNV) are closely associated with testicular development and male fertility. ZNF280AY, originating from the transposition of the ZNF280A gene, has been identified as a multicopy gene in the male-specific region of the Y chromosome within the Bovidae family. However, the genomic structure of ZNF280AY and its impact on male fertility through CNV remain poorly characterized. Our study revealed that ZNF280AY exhibits pseudogene characteristics in sheep, as evidenced by disrupted coding sequences and the absence of mRNA expression across seven somatic tissues (heart, liver, spleen, lung, kidney, testes, muscle) at 6 months of age. The absence of expression was further found at various developmental stages (0-12 months). Comparative genomic analysis confirmed sequence homology (>88%) with bovine orthologs, while structural divergence supports its pseudogenization trajectory. Large-scale qPCR analysis of 723 rams from eight breeds revealed substantial CNV (15-122 copies, median=42), with significant negative correlations between ZNF280AY copy number and testicular morphometric parameters (left testicular length: r = -0.096, P = 0.044; left testicular width: r = -0.112, P = 0.020; right testicular length: r = -0.136, P = 0.005). These findings establish ZNF280AY CNV as a novel molecular marker for ram testicular development evaluation, providing critical insights for marker-assisted selection in sheep breeding programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910de7419b8d99d5682ceb019b19f719e3a3dd13" target='_blank'>
              ZNF280AY: A Pseudogene on the Ovine Y Chromosome and Its Copy Number Variation (CNV) Associated with Testicular Size in Hu Sheep.
              </a>
            </td>
          <td>
            Yangkai Liu, Li Wang, Jialei Chen, Tajmal Hussain Solangi, Ran Li, Mengzhen Le, X. Yue
          </td>
          <td>2025-06-14</td>
          <td>Journal of animal science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite significant successes, immunotherapy continues to face numerous challenges in practical applications. CD82 plays an important role in the tumor microenvironment (TME) across various cancers. However, the prognostic value of CD82 in immunotherapy and its role in the immune response remain unclear. The expression levels of CD82 in healthy human tissues were assessed using the human protein atlas (HPA) database tool. The association among CD82 expression levels, copy number variations (CNV), and methylation was explored using the gene set cancer analysis (GSCA) database. Timer tumor immune estimation resource (TIMER) database is used for immune checkpoint and immune infiltration. The LinkedOmincs database is used for enrichment analysis. The statistical inquiry was performed through the R program (version 4.4.1). The result has shown that CD82 mRNA levels were significantly downregulated in most tumor types. Abnormal CD82 mRNA expression was associated with CNV and methylation changes in tumor tissues. Furthermore, several immune checkpoint genes related to CD82 influenced prognosis and were linked to increased immune cell infiltration across various cancer forms, including liver hepatocellular carcinoma (LIHC). The prognosis of the patients under multiple cancer forms, particularly LIHC, was remarkably influenced by CD82, which plays a vital role in modulating immune infiltration. CD82 has the potential to serve as both a prognostic indicator and an immunological biomarker in human cancer. CD82 expression is frequently downregulated across various cancer associated with poor prognosis, influenced by CNV and DNA methylation alterations. Its abnormal expression correlates with changes in immune checkpoint gene activity and enhanced immune cell infiltration, notably in liver. These associations suggest that CD82 plays a significant role in shaping the tumor immune microenvironment and affects patient prognosis. Consequently, CD82 holds promise as both a prognostic biomarker and an immunomodulatory target in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e6b1867f8cfd33352d9c4d7c23852fa5423b5ed" target='_blank'>
              Prognostic and immunological significance of CD82 in cancers with focus on hepatocellular carcinoma: A bioinformatics approach
              </a>
            </td>
          <td>
            Sikandar Ali, Shiyu Liu, Jiali Chen, Jawad Ullah, Guoyu Mou, Xiaohua Huang
          </td>
          <td>2025-07-25</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant methylation of a particular region plays a crucial role in the occurrence and development of malignant tumors. However, the sensitive and precise determination of DNA methylation from clinical samples remains challenging. Here, we developed a sensitive and specific approach for determining methylation with multiple isothermal amplification cascade quantum dot signal output. To increase the sensitivity, an entropy-driven circuit was exploited to specifically recognize the single P16 promoter methylation site and trigger the first amplification, whose product could initiate a hybridization chain reaction for the second amplification. Then, G-quadruplex-rich DNA polymer nanowires were self-assembled, which could bind with quantum dot-labeled TMPyP to turn on the fluorescence, realizing a reliable signal output of trace target. Finally, the proposed approach demonstrated a detection limit of 358.92 aM for the P16 promotor methylation concentration ranging from 1 fM to 1 nM within 1.5 h. In summary, this study provides a sensitive and rapid approach for the determination of methylation via isothermal amplification coupled with quantum dot labeling, paving a new strategy for the early diagnosis and monitoring of cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bd9b7c73f58c431ca60c773e1b2da1761f823" target='_blank'>
              Precise Determination of CpG Methylation through Entropy-Driven Cascade Circuit.
              </a>
            </td>
          <td>
            Xiaohui Chen, Haixia Liu, Ningyu Ma, Chenyu Sun, Yican Li, H. Chen, Qianqian Wu, Zhizeng Wang, Jichun Yang, Guoxiang Liu, Yang Luo
          </td>
          <td>2025-06-06</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is commonly characterized by high-grade and aggressive features, resulting in an augmented likelihood of distant metastasis and inferior prognosis for patients. Tumor immune microenvironment (TME) has been recently considered to be tightly correlated with tumor progression and immunotherapy response. However, the actual heterogenous TME within TNBC remains more explorations. Methods The thorough analyses of different cell types within TME were conducted on the self-tested single-cell RNA sequencing dataset which contained nine TNBC treatment-naïve patients, including subclusters classification, CellChat algorithm, transcription factors (TFs) expression, pseudotime analysis and functional enrichment assay. The malignant epithelial cluster was confirmed by copy number variations analysis, and subsequently LASSO-Cox regression was carried out to establish a Malignant Cell Index (MCI) model on the basis of five crucial genes (BGN, SDC1, IMPDH2, SPINT1, and UQCRFS1), which was validated in several TNBC cohorts through Kaplan–Meier survival and immunotherapy response analyses. The public spatial transcriptome, proteome data and qRT-PCR, western blotting experiments were exploited to corroborate UQCRFS1 expression in RNA and protein levels. Additionally, functional experiments were implemented to unravel the impacts of UQCRFS1 on TNBC cells. Results The diverse subclusters of TME cells within TNBC were clarified to display distinct characteristics in cell–cell interactions, TFs expression, differentiation trajectory and functional pathways. Particularly, IL32high Treg imparted an essential effect on tumor evasion and predicted a worsened prognosis of TNBC patients. Furthermore, MCI model enabled to notify the inferior prognosis and immunotherapy resistance in TNBC. Ultimately, UQCRFS1 knockdown dampened the proliferative and migratory competence in vitro as well as tumor growth in vivo of TNBC cells. Conclusions Our study offers innovative perspectives on comprehending the heterogeneity within TME of TNBC, thereby facilitating the elucidation of TNBC biology and providing clinical recommendations for TNBC patients' prognosis, such as IL32high Treg infiltration, MCI evaluation, and UQCRFS1 expression. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00672-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde7e708bae8d3687cc62eafeff7c490a66352ee" target='_blank'>
              Multi-omics analyses of the heterogenous immune microenvironment in triple-negative breast cancer implicate UQCRFS1 potentiates tumor progression
              </a>
            </td>
          <td>
            Yuhui Tang, Aiqi Xu, Zhongbiao Xu, Jindong Xie, Wei Huang, Liulu Zhang, Yitian Chen, Lu Yang, Shasha Du, Kun Wang
          </td>
          <td>2025-06-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Müller glia (MG) are retinal resident cells with diverse functions, including reprograming and regeneration in certain species. While the mammalian retina possesses molecular mechanisms for MG dedifferentiation and neuronal differentiation, it fails to generate neural progenitors in vivo. We previously proposed that an epigenetic barrier, driven by DNA methylation, may prevent complete MG reprograming in response to damage. DNA demethylases, such as Ten-Eleven Translocation (TET) and Growth Arrest and DNA Damage-Inducible Protein 45 (GADD45) families, are induced by damage and play roles in cellular reprograming and regeneration. Tet enzymes are regulated by metabolic factors (iron, vitamins A and C) and respond to environmental stimuli. Here, we investigated DNA demethylation as a regulatory signal for MG dedifferentiation and neuronal differentiation in response to N-methyl-D-aspartate (NMDA), a neurotransmitter used to model retinal neurodegeneration. Using mouse primary cultures and antibodies against methylated (5mC, MeCP2) and unmethylated (5 hmC, H3K4Me3) DNA, we analyzed MG epigenetic changes under control, vitamin-supplemented, and NMDA-stimulated conditions. We also assessed DNA demethylase expression and key reprograming genes (Ascl1, Lin28, Nestin). Vitamin A and C increased 5hmC levels but did not upregulate Tet enzymes or reprograming genes. In contrast, NMDA increased Tet3 and reprograming gene expression. Tet3 knockdown led to a rapid 5mC increase and impaired NMDA-induced upregulation of Ascl1, Lin28, and Nestin, suggesting its critical role in MG dedifferentiation. Conversely, Tet3 overexpression induced morphological changes and early neuronal marker expression. These findings identify Tet3 as a key epigenetic regulator of MG reprograming and a potential target for retinal regeneration strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ab362578948bd169fde4a8815b59e370d39ecc" target='_blank'>
              Tet3-mediated DNA demethylation is essential for maintaining the dedifferentiation capacity of mammalian Müller glia
              </a>
            </td>
          <td>
            Erick J. Martinez-Colin, Ivonne Lezama, Lenin D. Ochoa-de la Paz, Mónica Lamas
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="An R-loop is a complex nucleic acid structure consisting of an RNA–DNA hybrid and an associated single-stranded DNA. This structure plays an important role in many biological processes, including gene regulation, DNA replication, and maintenance of genome stability. Under physiological conditions, the R-loop is in a dynamic balance of generation and decomposition, which is a complex process involving multiple mechanisms of its formation, clearance, and regulation. Emerging evidence indicates that R-loops can drive inflammation and participate in immune processes. This review summarizes immune dysregulation resulting from impaired R-loop clearance and analyzes recent advances in computational methods for R-loop regulation, aiming to identify new avenues for developing R-loop-targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bce6b2f2435a1bf918ec22e6cdf3b596fdd7f0" target='_blank'>
              Unraveling R-loops: The hidden drivers of inflammation and immune dysregulation
              </a>
            </td>
          <td>
            Fan Zhang, Dingyun Wang, Guodong Zhao, Dingmao Wang
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: The Phase 3 TROPiCS-02 study found that the efficacy of sacituzumab govitecan (SG), an antibody-drug conjugate (ADC) directed to tumor-associated calcium signal transducer 2 (Trop2), in breast cancer (BC) does not depend on Trop2 expression levels. SG is the only FDA-approved ADC for advanced HR+/HER2− BC and triple-negative BC; it targets Trop2 by delivering SN-38, a topoisomerase I inhibitor. AT-rich interactive domain–containing protein 1A (ARID1A) is the most prevalent mutation in the SWItch/Sucrose Non-Fermentable (SWI/SNF) complexes in metastatic BC. ARID1A interacts with transcription factors involved in gene transcription and DNA repair, acting as a tumor suppressor. It is essential for maintaining luminal characteristics and influences treatment response. Patients with tumors harboring ARID1A alterations experience significantly shorter progression-free survival on selective estrogen receptor degraders than those with tumors with wild-type ARID1A. We hypothesize that loss or suppression of ARID1A expression results in a high level of Trop2 expression in HR+/HER2− BC. Identifying the mechanism linking ARID1A and Trop2, as well as determining the presence or absence of ARID1A mutations, could help identify patient groups that may benefit from combination therapy with SG and other inhibitors, enhancing treatment efficacy even in those with ARID1A mutations. Methods: We used transposase-accessible chromatin sequencing (ATAC-seq) and an RNA-seq database (GSE124228) from ARID1A-knockout MCF7 cells to determine whether ARID1A regulates Trop2 gene transcription. To elucidate how SG regulates the ARID1A-Trop2 axis in HR+/HER2− BC, we used specific small-molecule inhibitors, siRNA, and an expression plasmid targeting ARID1A. Target proteins in the ARID1A-Trop2 axis were detected with immunofluorescence staining, immunohistochemistry, flow cytometry, Western blotting, quantitative real-time PCR, and the TCGA database. Results: ARID1A protein expression negatively correlated with Trop2 expression in HR+/HER2− BC cell lines. Through Ataxia-Telangiectasia Mutated (ATM), the protein kinase critical for the cellular response to DNA damage activation, SG or SN-38 reduces ARID1A protein expression in HR+/HER2− BC cells. Further, ARID1A expression is diminished in tissue samples from SG-treated MCF7 xenografts resistant to both tamoxifen and abemaciclib. Mechanistic studies showed that overexpression of ARID1A reduces Trop2 mRNA and protein expression, whereas ARID1A knockdown elevates Trop2 levels in HR+/HER2− BC cells. Immunofluorescence staining data indicated an inverse correlation between ARID1A expression and Trop2 expression. ATAC-seq data revealed increased chromatin accessibility at the Trop2 promoter in ARID1A-knockout MCF7 cells. T47D cells, which exemplify low ARID1A and high Trop2 expression, are approximately ten times more sensitive to SG than MCF7 and HCC1428 cells, which have high ARID1A and low Trop2 expression. Thus, low ARID1A expression leads to elevated Trop2 levels, resulting in increased therapeutic activity of SG. Conclusion: We discovered that SG induces Trop2 expression through SN-38–mediated reduction of ARID1A protein expression in HR+/HER2− BC cell lines. Moreover, lower ARID1A in HR+/HER2− BC cells was associated with increased sensitivity to SG. These results suggest a novel mechanism for Trop2 ADC efficacy in HR+/HER2− BC with low Trop2 expression. Our findings highlight the importance of knowing a patient’s ARID1A status to predict the efficacy of Trop2-ADCs. Further mechanistic studies will be presented at SABCS.
 Citation Format: Nanae Ogata, Nakyung Oh, Dileep R. Reddy, Lan Vo, Naoto T. Ueno, Jangsoon Lee. ARID1A as a Novel Regulator of Trop2-ADC Efficacy in Trop2-Low Hormone Receptor–Positive Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e12694360d8c196fe293c0581c8daae847947da9" target='_blank'>
              Abstract PS13-08: ARID1A as a Novel Regulator of Trop2-ADC Efficacy in Trop2-Low Hormone Receptor–Positive Breast Cancer
              </a>
            </td>
          <td>
            Nanae Ogata, Nakyung Oh, Dileep R. Reddy, Lan Vo, Naoto T Ueno, Jangsoon Lee
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/510186795452720c77e13e46654ec25972305866" target='_blank'>
              Mechanisms for licensing origins of DNA replication in eukaryotic cells.
              </a>
            </td>
          <td>
            Bruce Stillman, J. Diffley, Janet Iwasa
          </td>
          <td>2025-06-30</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Aberrant DNA methylation has been shown to be a fingerprint characteristic in human colorectal tumors. In this study, we hypothesize that investigating global DNA methylation could offer potential candidates for clinical application in CRC. The epigenome-wide association analysis was conducted in both the tumor area (N = 27) and the adjacent tumor-free (NAT) area (N = 15). We found 78,935 differentially methylated CpG sites (DMCs) (FDR < 0.05), 42,888 hypomethylated and 36,047 hypermethylation showing overall hypomethylation. Gene ontology and KEGG analysis of differentially methylated genes showed significant enrichment in developmental genes, as well as in genes involved in metabolic processes and the cell cycle, such as the TFGβ and cAMP signaling pathways. Through filtered analysis, we identified TNXB as the most epigenetically dysregulated gene, hypomethylated and downregulated in CRC (both with p < 0.001) and associated with poor overall survival. In the functional analysis, TNXB was epigenetically regulated in a dose-dependent manner, suggesting a potential role in CRC. The epigenetic dysregulation and functional role of TNXB in CRC could have clinical implications, serving as indicators of malignant potential, with adverse effects associated with disease origin and progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5735874128c08fad50b2549a2c915b47f637e3f9" target='_blank'>
              Unraveling TNXB Epigenetic Alterations Through Genome-Wide DNA Methylation Analysis and Their Implications for Colorectal Cancer
              </a>
            </td>
          <td>
            Jesús Pilo, Alejandro Rego-Calvo, L. García-Flores, I. Arranz-Salas, Ana Isabel Álvarez-Mancha, A. G. Izquierdo, A. Crujeiras, Julia Alcaide, María Ortega-Castan, H. Boughanem, M. Macías-González
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Background: Combinations of endocrine therapy and CDK4/6 inhibitors (CDK4/6i) used in metastatic breast cancer (MBC) result in heterogenous clinical benefit with subsets of patients having disease control ranging anywhere from a few months to more than 5 years. Understanding the mechanisms that account for these differences can enable new pharmacologic approaches to convert more of these cancers into the long-term control subset. Methods: To elucidate the causal factors that underlie the observed clinical variability, we have utilized multi-omic analyses of patient samples as well as experimental modeling using a diverse cohort of ER+ breast cancers. For clinicogenomic analyses, we analyzed 467 patients with ER+/ HER2- MBC treated with first-line CDK4/6i at our institution (MSK) with detailed clinical follow annotation and follow up. For genomic/transcriptomic studies, we analyzed single cell RNA sequencing data, whole genome sequencing data, and Ki67 score from the neoadjuvant CDK4/6i containing arms in the FELINE trial (NCT02712723). Patient derived xenografts, patient derived organoids, isogenic cell lines were used to define the underlying mechanisms. Live cell imaging assays were performed by FUCCI cell cycle sensor to assess cell cycle transitions. Results: Of 467 patients from MSK cohort, 129 (27.6%) had pre-CDK4/6i loss-of-function variants in TP53, corresponding to short PFS (median PFS= 8.7 months, 95% CI: 6.6–10.3; HR = 2.12, 95% CI: 1.68 to 2.68; p = 2.64e-10). This finding was validated in two large independent clinical cohorts outside MSK. In patient derived and isogenic cell lines, p53 loss had no impact on initial drug response but promoted long-term cell outgrowth and prevented emergence of senescence phenotypes. Following cells in detailed time courses revealed the propensity of p53 mutant but not wild type cells to reenter the cell cycle after initial arrest by CDK4/6 inhibition. Gene expression analysis of RNA seq data revealed a significant positive association between TP53 mutation status and DREAM complex genes, assessed by GSEA (NES= 1.51, p = 1e-4) In models, deletion of DREAM complex component, p130-Rbl2 was found to fully recapitulate the effects of p53 loss: promoting long-term outgrowth from CDK4/6i, blocking therapy induced senescence, and promoting cell cycle re-entry. To validate these results, we analyzed the multi-omic and histologic characteristics of serial samples from the FELINE trial. Compared with WT cases, baseline TP53 LOF had a higher amount of proliferating cells at the end of treatment, measured by Ki67 > 10% (4.5% of WT cases vs. 60% of mutant cases, respectively, OR = 26.8, p = 0.014), the per cell proliferation score, measured by ssGSEA (log2FC: 0.2 vs. 2.8, respectively) as well as a higher pseudobulk expression of DREAM complex related genes (GSEA: NES = 2.0, q = 3.7e-4). Finally, to identify strategies that might facilitate stable cell cycle control of p53KO tumors, we investigated which targets of p53 regulate DREAM and identified p21. Given role of p21 in regulating both CDK4/6 and CDK2 we investigated whether CDK2 inhibitors would provide additional benefit to CDK4/6i. Addition of CDK2i was able to fully restore efficacy in p53KO lines as well as a patient derived model with TP53 mutation enabling irreversible cell cycle arrest and long-term growth suppression. Conclusion: Wild type p53 facilitates DREAM complex assembly in response to CDK4/6i in ER+ breast cancer, underlying long-term disease control. p53 loss enables cells to reenter the cell cycle and facilitates the development of CDK4/6i resistance. Combined inhibition of CDK2 with CDK4/6 represents a promising therapeutic strategy for overcoming loss of p53 and enabling long-term antitumor efficacy.
 Citation Format: Rei Kudo, Anton Safonov, Enrico Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, Edaise M. da Silva, Qing Li, Marie Will, Atsushi Fushimi, Harikrishna Nakshatri, Andrew Koff, Britta Weigelt, Qamar J. Khan, Pedram Razavi, Sarat Chandarlapaty. DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c291e911332efcfe859d324b671af4bba597c2" target='_blank'>
              Abstract PS13-04: DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition
              </a>
            </td>
          <td>
            Rei Kudo, A. Safonov, E. Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, E. D. da Silva, Qing Li, Marie Will, A. Fushimi, H. Nakshatri, Andrew Koff, B. Weigelt, Qamar J Khan, P. Razavi, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Mucosal melanoma (MM) is a deadly cancer derived from mucosal melanocytes. To test the consequences of MM genetics, we develop a zebrafish model in which all melanocytes experience CCND1 expression and loss of PTEN and TP53. Surprisingly, melanoma only develops from melanocytes lining internal organs, analogous to the location of patient MM. We find that zebrafish MMs have a unique chromatin landscape from cutaneous melanomas. Internal melanocytes are labeled using a MM-specific transcriptional enhancer. Normal zebrafish internal melanocytes share a gene expression signature with MMs. Patient and zebrafish MMs show increased migratory neural crest and decreased antigen presentation gene expression, consistent with the increased metastatic behavior and decreased immunotherapy sensitivity of MM. Our work suggests that the cell state of the originating melanocyte influences the behavior of derived melanomas. Our animal model phenotypically and transcriptionally mimics patient tumors, allowing this model to be used for MM therapeutic discovery. As this is a non-MAPK driven genetically engineered model of melanoma, our work also has implications for the 15% of cutaneous melanoma patients who lack MAPK-driving mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99321e90c4b2d4aa26110c1e8b75f84421ebcd50" target='_blank'>
              Specific oncogene activation of the cell of origin in mucosal melanoma
              </a>
            </td>
          <td>
            Swathy Babu, Jiajia Chen, Chloé S. Baron, Kaiwen Sun, Emily J. Robitschek, Alicia M. McConnell, Constance Wu, Aikaterini Dedeilia, Moshe Sade-Feldman, Rodsy Modhurima, Michael P. Manos, Kevin Y. Chen, Anna M. Cox, Calvin G. Ludwig, M. Kellis, Elizabeth I Buchbinder, N. Hacohen, Jie-Ying Yang, Genevieve M Boland, Brian J. Abraham, David Liu, Leonard I. Zon, Megan L. Insco
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcdcfe442326883ec1ff1d1b7880290f079cb82" target='_blank'>
              Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy.
              </a>
            </td>
          <td>
            Jie Liu, Duc Tran, Liying Xue, B. Wiley, C. Vlasschaert, Caroline J. Watson, Hamish A J MacGregor, X. Zong, I. Chan, Indraniel Das, Md Mesbah Uddin, A. Niroula, Gabriel Griffin, Benjamin L Ebert, T. Mack, Y. Pershad, B. Sharber, Michael F. Berger, A. Zehir, R. Ptashkin, Ross L. Levine, E. Papaemmanuil, V. Joseph, Teng Gao, Y. Kemel, D. Mandelker, Konrad H Stopsack, Paul D. P. Pharoah, Semanti Mukherjee, Li Ding, Yinyin Cao, MJ Walter, Jamie R Blundell, Nilanjan Chatterjee, K. Offit, Lucy A. Godley, Daniel C. Link, Z. Stadler, Alexander G. Bick, P. Natarajan, Kelly L Bolton
          </td>
          <td>2025-07-15</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Plasmodium parasites, the causative agents of malaria, undergo crucial developments within the mosquito vector, initiated by the formation of male and female gametes. Male gametogenesis involves three rapid rounds of mitosis without nuclear or cell division, followed by a single round of DNA segregation and nuclear division during gamete budding. How the cell organizes the segregation of eight genomes from a single octoploid nucleus into eight haploid gametes is currently unknown. Here we discovered an atypical Arp2/3 complex in Plasmodium important for DNA segregation during male gametogenesis. Unlike the canonical Arp2/3 complex found in other eukaryotes, Plasmodium Arp2/3 localizes to endomitotic spindles and interacts with a kinetochore-associated protein. Disruption of Arp2/3 subunits or actin polymerization interferes with kinetochore–spindle association, causes the formation of subhaploid gametes, and blocks transmission. Our work identified an evolutionary divergent Arp2/3 complex in malaria parasites, provides insights into gametogenesis, and reveals potential targets for transmission-blocking interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0eb8cd0287d7e4de3c5f68362eedb92788d67" target='_blank'>
              An atypical Arp2/3 complex is required for Plasmodium DNA segregation and malaria transmission
              </a>
            </td>
          <td>
            Franziska Hentzschel, David Jewanski, Yvonne Sokolowski, Pratika Agarwal, Anna Kraeft, Kolja Hildenbrand, Lilian P. Dorner, Mirko Singer, Matthias Marti, Friedrich Frischknecht
          </td>
          <td>2025-06-13</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, D. Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most aggressive and lethal brain tumor in adults and poses significant challenges to patient survival. This review provides a comprehensive exploration of the molecular and genetic landscape of GBM, focusing on key oncogenic drivers, such as epidermal growth factor receptor (EGFR), platelet-derived growth factor receptor (PDGFR), and the PI3K/AKT/mTOR pathway, which are critical for tumorigenesis and progression. We delve into the role of epigenetic alterations, including DNA methylation and histone modifications, in driving therapy resistance and tumor evolution. The tumor microenvironment is known for its pivotal role in immune evasion, with tumor-associated macrophages, myeloid-derived suppressor cells, and regulatory T cells creating an immunosuppressive niche that sustains GBM growth. Emerging therapies, such as immunotherapies, oncolytic viral therapies, extracellular vesicle-based approaches, and non-coding RNA interventions, are highlighted as promising avenues to disrupt GBM pathogenesis. Advances in precision medicine and innovative technologies, including electric field therapy and locoregional treatments, are discussed for their potential to overcome the blood‒brain barrier and treatment resistance. Additionally, this review underscores the importance of metabolic reprogramming, particularly hypoxia-driven adaptations and altered lipid metabolism, in fueling GBM progression and influencing the therapeutic response. The role of glioma stem cells in tumor recurrence and resistance is also emphasized, highlighting the need for targeted therapeutic approaches. By integrating molecular targeting, immune energetics, and technological advancements, this review outlines a multidisciplinary framework for improving GBM treatment outcomes. Ultimately, the convergence of genetic, metabolic, and immune-based strategies offers transformative potential in GBM management, paving the way for increased patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1445c096bc679931586564b48bbb15167d4847f8" target='_blank'>
              Glioblastoma at the crossroads: current understanding and future therapeutic horizons
              </a>
            </td>
          <td>
            Shilpi Singh, Devanjan Dey, Debashis Barik, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Sujata Prasad, Peize Wang, Amar Singh, Gatikrushna Singh
          </td>
          <td>2025-07-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract RecD2 is a superfamily 1B helicase involved in DNA replication and repair, modulating replication restart, fork progression, and RecA recombinase activity. In this work, we have characterized the functions of Bacillus subtilis RecD2 using biochemical and single-molecule approaches. ATPγS binding and low MgCl2 concentrations enhance DNA association, with a preference for forked structures and unstructured DNA longer than 30 nucleotides. RecD2 binds to end-less single-stranded DNA stretched at 8–20 pN and translocates through ATP hydrolysis over long distances (>20 kb) with 5′–3′ polarity at high rates. RecD2 shows limited unwinding activity on fork structures, strongly dependent on protein concentration and duplex length, reflecting low processivity. However, processivity improves significantly when force is applied to the translocating strand or unwound DNA ends, enabling the unwinding of thousands of base pairs at rates up to 160 bp/s. Single-molecule assays reveal frequent strand switching on fork substrates, resulting in a non-productive cycle of unwinding and rewinding, likely mediated by the N-terminal domain. This behavior explains the low helicase activity observed in bulk assays. We propose that regulation of strand-switching activity may be relevant for RecD2’s in vivo function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d3016a9bf1dbf9c320eaeb5c544e744f1550f5d" target='_blank'>
              Bacterial RecD2 is a processive single-stranded DNA translocase with strand-switching capacity at DNA forks
              </a>
            </td>
          <td>
            Silvia Hormeno, Cristina Ramos, Javier Mendia-Garcia, Clara Aicart-Ramos, S. Ayora, F. Moreno-Herrero
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has recently become a focus of cardiovascular disease (CVD) research as it is possibly involved in disease mechanisms and may function as a biomarker. The literature suggests that eccDNAs may modify gene expression patterns and contribute to the pathogenesis of diseases such as pulmonary arterial hypertension (PAH). Many eccDNAs have been reported to be upregulated in PAH patients with promisingly high diagnostic sensitivity and specificity values. The advent of eccDNA detection from plasma samples using high‐throughput sequencing technologies has opened new avenues for non‐invasive diagnosis. In this review, we focus on the role of eccDNA in CVDs, concentrating on its prospects for use as a biomarker of disease development and stage. However, further studies should be conducted to explicate the functional aspects of eccDNAs in cardiovascular health and disease. This work may eventually lead to the development of novel treatment options and improved clinical outcomes for CVD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903ec3642b6c2ce94d9db06e20223312c1b439d8" target='_blank'>
              Extrachromosomal Circular DNA: Emerging Insights Into Cardiovascular Disease Mechanisms and Biomarker Potential
              </a>
            </td>
          <td>
            Saim Mahmood Khan, Jawairya Muhammad Hussain, Surraiya Riaz Mahmood Khan, Bushra Nadeem, Taniyat Naseem, Hammad Jawed, Aina Lashari
          </td>
          <td>2025-06-29</td>
          <td>iLABMED</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the primary cause of death worldwide. Cancer occurrence and progression are closely associated with DNA damage repair. DNA glycosylase were the first enzymes to initiate base excision repair, and the Nei-like DNA glycosylase (NEIL) family has attracted increasing attention as an important component of DNA glycosylases. Here, we introduce the role of the NEIL family in the malignant biological behaviors of cancer, including cell proliferation, chemoradiotherapy resistance, invasion and migration, apoptosis, and stemness. Mechanisms affecting the expression of the NEIL protein family range from the transcriptional level and mRNA editing to the level of post-translational modification. Additionally, we emphasize the different functions of the NEIL family in various malignancies and present useful information that supports the hypothesis that the NEIL family could be a potential target in the treatment and diagnosis of various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9582798727585833402f33b5e71abe53251dc2" target='_blank'>
              Research progress on the role of the NEIL family in cancer
              </a>
            </td>
          <td>
            Yinghan Chen, Muyun Ma, Aixue Zou, Xinjia Wang, Weiwei Dong
          </td>
          <td>2025-07-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PrimPol is a human DNA primase and DNA polymerase involved in DNA damage tolerance in both nuclei and mitochondria. PrimPol restarts stalled replication forks by synthesizing DNA primers de novo and also possesses DNA translesion activity (TLS activity). PrimPol efficiently and relatively accurately bypasses several DNA lesions including 8-oxoguanine, thymine glycol and 5-formyluracil. In this work, we showed that PrimPol possesses efficient and accurate TLS activity across 8-oxoadenine, another common DNA lesion caused by oxidative stress. The accuracy of PrimPol on DNA with 8-oxoA was significantly higher compared to DNA containing 8-oxoG. Replacement of Mg2+ ions with Mn2+ stimulated activity of PrimPol on DNA with 8-oxoA and 8-oxoG as well as undamaged A in a sequence-dependent manner by the lesion skipping (or template scrunching) mechanism. Altogether, our data support the idea that PrimPol possesses efficient TLS activity across a wide range of DNA lesions caused by oxidative stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c9f308376c56521542db39b24955edc0c9ead07" target='_blank'>
              Accurate DNA Synthesis Across 8-Oxoadenine by Human PrimPol
              </a>
            </td>
          <td>
            E. O. Boldinova, Alexander A Kruchinin, P. Kamzeeva, Andrey V. Aralov, Alena V. Makarova
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e90dcf5c828cf55b60bd84b7542d25e18df7d49a" target='_blank'>
              Single-cell omics reveal distinct gene regulatory dynamics underpinning embryonic and extraembryonic lineage functions during pig blastocyst development
              </a>
            </td>
          <td>
            Adrien Dufour, Marie-Noëlle Rossignol, Patrick Manceau, Yoann Bailly, Stéphane Ferchaud, M. Mercat, Ali G Turhan, S. Djebali, S. Foissac, Jérôme Artus, Hervé Acloque
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Mutations in PIK3CA are prevalent in advanced colorectal cancer (CRC) and are associated with advanced-stage disease and poor prognosis. However, the prognostic value of these mutations remains unclear. To better understand their role in CRC progression, we evaluated the genomic landscape of PIK3CA mutations (PIK3CAm) in advanced CRC using plasma-based ctDNA from ~17,000 samples (Guardant360 database). Mutation co-occurrence, MSI-H status, TMB scores, and exon-specific mutation frequencies were analyzed. Our study shows that the frequency of PIK3CAm (19.2%) and distribution of exons 9 and 20 are similar to those found previously in the literature. Our results demonstrate PIK3CAm having a low frequency of MSI-H co-occurrence, higher TMB scores, and increased co-occurring alterations in APC, BRAF, EGFR, ERBB2 and KRAS, which may suggest higher genomic instability. Our findings underscore the clinical significance of PIK3CAm in CRC and supports the development of tailored therapeutic strategies targeting this mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c39945611065d70241e847cc40e1982a682875" target='_blank'>
              Comprehensive analysis of PIK3CA mutations in advanced colorectal cancer using circulating tumor DNA profiling
              </a>
            </td>
          <td>
            Allante Milsap, J. Tsai, Baqir Jafry, Alex Kim, Syed M Ali Kazmi, Nilesh Verma
          </td>
          <td>2025-07-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The study of β-hemoglobinopathies and associated β-globin genes has revealed that genetic elements, such as the Locus Control Region (LCR) or the replication Initiation Region (IR) of the β-globin gene locus, are essential for the regulation of β-globin genes replication and expression. The LCR at 5' of the β-globin genes plays major role in the intricate regulation of transcription of the "β-like globin genes" expression in situ and in gene therapy protocols by viral gene transfer, ensuring globin gene expression independent from integration site and exerting a critical role in chromatin organization and boundary formation. The IR element, located at the 5' site of the HBB gene promoter, functions as the initiation point for physiological, bidirectional DNA replication, both in situ and within an episomal vector, and induces replication in positions that do not possess such capacity. It enhances plasmid replication, establishment, and transgene expression in the descendants of transfected human CD34+ cells during colony-forming cell assays. A third required genetic element is the promoter of the transgene(s). This is either the HBB gene native promoter or the CD34+ cell-functional ubiquitous promoter spleen focus-forming virus. Both promoters, in in vitro studies, can direct accurate, efficient transcription from episomal, S/MAR-based vectors. Mutations in the HBB gene native promoter as well as in LCR and IR lead to β-thalassemia. Another genetic element, the S/MAR, deriving from the 5' of the human β-interferon gene, ensures plasmid nonintegration and long-term nuclear retention in the prototype episomal vector pEPI-1 and derivative episomal vectors. Such S/MAR-based episomal vectors form the basis from which the genetic elements collectively- HBB gene promoter, LCR, and IR-represent a comprehensive model for the design of efficient episomal vectors with efficient transcription, replication, and long-term nuclear retention of vector for gene therapy applications for the β-hemoglobinopathies within the context of gene addition strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6c71f2b648b487c5f8f7dbba89ef3c715212f38" target='_blank'>
              The β-Hemoglobinopathies as a Model for the Development of Nonviral, Episomal Vectors for Gene Therapy.
              </a>
            </td>
          <td>
            A. Athanassiadou, Argyro Sgourou, Meletios Verras
          </td>
          <td>2025-06-30</td>
          <td>Human gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Interphase chromosomes are mainly shaped by loop extrusion and compartmentalisation mechanisms. However, their temporal component and cause-effect relationships remain largely unknown. In this study, we use acute degradation of WAPL, CTCF and cohesin in mouse embryonic stem cells to investigate the dynamics of loop extrusion and its relationship to compartmentalisation. Stabilisation of cohesin on chromatin by depletion of WAPL results in the formation of extended loops and promotes looping between non-convergent CTCF sites. Loss of WAPL also results in a rapid decrease in compartmentalisation, which is reversed by subsequent removal of cohesin, directly demonstrating the opposite role of extrusion on compartmentalisation. Using combined depletion of WAPL and CTCF, we identify fountains, a feature of chromosome organisation that emanates from enhancer regions and exhibits strong cohesin binding. Fountains form rapidly after mitosis and early in mammalian development. Cohesin depletion confirms that fountains are cohesin dependent, and their disruption leads to the downregulation of fountain-proximal genes, suggesting a role in gene regulation. Taken together, by exploiting the temporal precision of acute protein depletion, our study reveals fountains as an extrusion-mediated, fast-forming feature of 3D genome organisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f8f50a44096e211e83ee9e8d36589f32f7b8c64" target='_blank'>
              Extrusion fountains are restricted by WAPL-dependent cohesin release and CTCF barriers
              </a>
            </td>
          <td>
            Ning Qing Liu, Mikhail D. Magnitov, Marijne M G A Schijns, Tom van Schaik, Hans Teunissen, B. van Steensel, Elzo de Wit
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The canonical DNA damage response (cDDR) maintains genome stability, involving DNA synthesis arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously characterized a noncanonical response specific to nonblocking replication stress, i.e. low-level stress (LoL-DDR), in primary cells. Although LoL-DDR generates replication stress-induced ROS (RIR), it prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a threshold triggers the cDDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: (1) RIR are not detoxified under high-level stress, resulting in a continuous increase in the dose‒response curve of RIR production; (2) RIR are not produced by NADPH oxidases; (3) replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” replication-stress/ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05172-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1826ee141a3bb5d6a8079b9d816c3e423819f87e" target='_blank'>
              Dysregulation of the low-level replication stress response in transformed cell lines
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-06-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Despite therapeutic advances, high-risk subtypes of B-cell acute lymphoblastic leukemia (B-ALL) such as Philadelphia chromosome-like (Ph-like) and KMT2A-rearranged (KMT2A-R) remain a formidable clinical challenge. BRG1 (gene name SMARCA4), the ATPase subunit of the SWI/SNF chromatin-remodeling complex, has been extensively studied in solid tumors, where inactivating mutations are linked to aggressive disease and poor prognosis. Although BRG1 is known to be essential for early B cell development, its role in B-ALL remains poorly understood. Therefore, we investigated the therapeutic relevance of BRG1 in high-risk B-ALL. Meta-analysis of gene expression data revealed that BRG1-inactivating mutations are exceedingly rare (0.35%) in B-ALL, suggesting that intact BRG1 function may be critical for leukemogenesis. Subtype-specific analyses revealed that elevated BRG1 expression is associated with significantly shorter overall survival in children with Ph-like B-ALL, while the opposite trend was observed in KMT2A-R B-ALL. We confirmed higher BRG1 expressions in Ph-like compared to KMT2A-R B-ALL via gene expression analysis, RT-PCR, and Western blotting. The pharmacologic inhibition of BRG1 using two selective inhibitors, BRM014 and FHD-286, revealed marked sensitivity in Ph-like B-ALL cell lines, whereas KMT2A-R B-ALL was resistant. Mechanistically, we found that BRG1 inhibition results in cell cycle arrest via downregulation of cell cycle regulators (CCND3, CDK4, CDK6, E2F1, and MYC) and upregulation of the cell cycle inhibitor CDKN1B (p27). Importantly, treatment with FHD-286 significantly prolonged the survival of NSG mice engrafted with Ph-like B-ALL cells. Taken together, these findings establish BRG1 as a critical, subtype-specific dependency in Ph-like B-ALL and demonstrate that its pharmacologic inhibition effectively suppresses leukemic cell proliferation through induction of cell cycle arrest. The pronounced in vitro sensitivity and improved in vivo survival upon BRG1 inhibition provide compelling preclinical evidence for its therapeutic targeting. These results support the advancement of BRG1-directed strategies as a viable treatment approach for patients with Ph-like B-ALL, with the potential to improve outcomes in this high-risk population. Highlights Higher levels of BRG1 correlate to poor clinical outcomes in Ph-like but KMT2A-R B-ALL Inhibition of BRG1 induces cell cycle arrest in Ph-like cells in vitro and extends the survival of mice in pre-clinical in vivo studies Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c3ded805682d4eff5c7bc4d110a3fd55426d2b" target='_blank'>
              Selective Sensitivity of Ph-like B-ALL to BRG1 Inhibition Reveals a Novel Targeted Therapy Strategy
              </a>
            </td>
          <td>
            V. S. S. A. Ayyadevara, Shikha Gaur, Ashley Paik, Ria Perencsik, Monika M. Toma, Gerald Wertheim, H. Geng, Tomasz Skorski, Srividya Swaminathan, C. Hurtz
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e559b4aa95ac215d2b0dba18f35a9867916e3c3" target='_blank'>
              ADH5/ALDH2 dehydrogenases and DNA polymerase theta protect normal and malignant hematopoietic cells from formaldehyde challenge: therapeutic implications.
              </a>
            </td>
          <td>
            Jessica Atkins, Anna-Mariya Kukuyan, Monika M. Toma, M. Drzewiecka, Umeshkumar Vekariya, Adam Karami, M. Nieborowska-Skorska, R. Nejati, E. Hadzijusufovic, P. Valent, Tomasz Stokłosa, Tomasz Sliwinski, Mariusz A. Wasik, Tomasz Skorski
          </td>
          <td>2025-07-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background/Objectives: The treatment of multiple myeloma (MM) remains a challenge, as almost all patients will eventually relapse. Proteasome inhibitors are a cornerstone in the management of MM. Unfortunately, validated biomarkers predicting drug response are largely missing. Therefore, we aimed to identify genes associated with drug resistance or sensitization to proteasome inhibitors. Methods: We performed genome-wide CRISPR-Cas9 knockout (KO) screens in human KMS-28-BM myeloma cells to identify genetic determinants associated with resistance or sensitization to proteasome inhibitors. Results: We show that KO of KLF13 and PSMC4 induces drug resistance, while NUDCD2, OSER1 and HERC1 KO cause drug sensitization. Subsequently, we focused on top sensitization hit, NUDCD2, which acts as a co-chaperone of Hsp90 to regulate the LIS1/dynein complex. RNA sequencing showed downregulation of genes involved in the ERAD pathway and in ER-associated ubiquitin-dependent protein catabolic processes in both untreated and carfilzomib-treated NUDCD2 KO cells, suggesting that NUDCD2 depletion alters protein degradation. Furthermore, bortezomib-treated NUDCD2 KO cells showed a decreased expression of genes that have a function in oxidative phosphorylation and the mitochondrial membrane, such as Carnitine Palmitoyltransferase 1A (CPT1A). CPT1A catalyzes the uptake of long chain fatty acids into mitochondria. Mitochondrial lipid metabolism has recently been reported as a possible therapeutic target for MM drug sensitivity. Conclusions: These results contribute to the search for therapeutic targets that can sensitize MM patients to proteasome inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0627ff07231b746c3c64319d7f31fa666e89863" target='_blank'>
              Genome-Wide CRISPR-Cas9 Knockout Screening Identifies NUDCD2 Depletion as Sensitizer for Bortezomib, Carfilzomib and Ixazomib in Multiple Myeloma
              </a>
            </td>
          <td>
            Sophie Vlayen, T. Dierckx, Marino Caruso, Swell Sieben, Kim De Keersmaecker, Dirk Daelemans, Michel Delforge
          </td>
          <td>2025-07-16</td>
          <td>Hemato</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9147f200657432499c5c0092f9ff2f8be29f38a" target='_blank'>
              Deciphering the differential impact of CDK4 mutations on abemaciclib binding in oral squamous cell carcinoma: a precision medicine approach.
              </a>
            </td>
          <td>
            Yu Huang, Mi Zhang, Yi Ni
          </td>
          <td>2025-07-21</td>
          <td>Irish journal of medical science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Here we present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the National Institute of General Medical Sciences Human Genetic Cell Repository (NIGMS Repository). This resource includes five iPSCs and matched parental cell lines with accompanying publicly available, HiFi whole-genome sequencing data. Structural variant (SV) and single nucleotide variant (SNV) concordance between iPSC and parental lines was generally high; however, we found a notable reduction in concordance between the iPSC reprogrammed with retroviral reprogramming and its parental line consistent with previous work showing newer Sendai approaches to be more robust in preserving genomic integrity. This iPSC resource additionally includes pharmacogenomic and human leukocyte antigen (HLA) gene annotations as well as a set of user-friendly, web-based search tools to visualize and explore SVs and SNVs. This new resource is designed to offer a highly characterized set of in vitro models for research into cell-type specific functional characterization of genetic, genomic and pharmacogenomic variation. More generally, these renewable biospecimens and genomic data search tools are available to the scientific community to support high-quality and reproducible biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/664bbdaaf704bf6f26d8a71b37d0bd99ff2c3655" target='_blank'>
              New iPSC resource with long-read whole genome sequencing characterizations for enhanced in vitro modeling
              </a>
            </td>
          <td>
            Laura Scheinfeldt, Anthony Pompetti, G. Calendo, Tatyana Pozner, Christine Grandizio, Gretchen Smith, Kelly Hodges, Neda Gharani, D. Kusic, Matthew W Mitchell, N. Turan
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The study of DNA tumor viruses has revolutionized cancer biology, partly by virtue of the unique capabilities of viral oncoproteins to manipulate key proteins and pathways involved in tumorigenesis. We find a high affinity and selective binding of the polyoma small T antigen (PyST) with the transcription cofactor TAZ. By engineering a degradable version of PyST, we demonstrate that, when TAZ activity is modulated by PyST, a surprisingly small number of genes have altered expression and thus are candidate transcription targets of TAZ. Notably, knocking out TAZ, or its target genes CTGF or CYR61, abolishes the growth-promoting properties of PyST that are evident upon growth factor withdrawal. Therefore, by controlling the protein abundance of PyST and consequently TAZ activity, we find that TAZ is a transcriptional coactivator that can achieve important biological effects by acting on a limited number of gene targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a4dcc37489aa9a0942107e458f9ebbf6f6693a5" target='_blank'>
              A degradable form of polyoma small T antigen reveals the high specificity of TAZ in regulating gene expression.
              </a>
            </td>
          <td>
            Yubao Wang, C. Manokaran, Kevin Huang, Brian Schaffhausen, Thomas M Roberts
          </td>
          <td>2025-07-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Micropeptides are endogenous peptides translated from alternative open reading frames (alt-ORFs) within coding or non-coding genes. Emerging evidence suggests that some micropeptides play critical roles in both physiological and pathological processes. Multiple myeloma (MM), a haematological malignancy, remains incurable due to frequent relapses and a limited understanding of its underlying mechanisms. In this study, we sought to investigate the function and molecular mechanism of a novel micropeptide in MM pathogenesis. We identified a novel micropeptide, altH19, encoded by the lncRNA H19, which is highly expressed in patients of MM. Functional assays revealed that altH19 promotes myeloma cell proliferation and colony formation significantly. Furthermore, altH19 induces multipolar mitosis by upregulating the expression of Aurora B, Centrin 2 and phosphorylated histone H3. Flow cytometry analyses confirmed that overexpression of altH19 enhances DNA replication and accelerates the transition from early to mid-late stages of the DNA replication process. Conversely, knockout of altH19 reverses these effects. Mechanistically, altH19 directly interacts with phosphorylated CDK2 at threonine 160, thereby enhancing CDK2 T160 phosphorylation and activating the downstream E2F1 target RB phosphorylation. Notably, altH19 was able to restore phosphorylation levels of CDK2 and RB that were otherwise suppressed by the CDK2-selective inhibitor Seliciclib. In summary, we identify altH19 as a novel lncRNA-derived micropeptide with a pivotal role in myeloma progression, highlighting the therapeutic potential of targeting the altH19-CDK2-RB axis in MM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790582f1ede9cdc6910628de9fe99a0b37b482f7" target='_blank'>
              LncRNA H19-Encoded Micropeptide altH19 Promotes DNA Replication and Mitosis in Myeloma Cells by Enhancing the Phosphorylation of CDK2 at Threonine 160.
              </a>
            </td>
          <td>
            Yaxin Zhang, Wenjing Li, Xu Cao, Jiwei Mao, Xiaodan Zhou, Linlin Liu, Ruosi Yao
          </td>
          <td>2025-06-27</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Currently, there are no targeted therapies for triple-negative breast cancer (TNBC). 1 While radiation therapy plays a crucial role in breast cancer treatment, its effectiveness against TNBC is hindered by acquired resistance to radiation over time. This highlights the need for more sensitive and specific therapeutic approaches. 2 ALDH1A1 is associated with stemness and poor outcomes in TNBC and is a potential new target for treatment. ALDH1A1 contributes to chemoresistance in TNBC3, and we propose that it also promotes radiation resistance through both its antioxidant properties and regulation of cancer stemness. Previous studies have indicated that the radiation resistance observed in TNBC is linked to increased levels of ALDH1A14, suggesting it plays a protective role against radiation-induced cell death. In this study, we did a CRISPR/Cas9 knock out (KO) of ALDH1A1 in SUM 159 cells and derived single cell clones to screen for the effects of ALDH1A1 KO. Confirmation of ALDH1A1 KO was done at the genomic level by Sanger sequencing and at the protein level by western blotting analysis. The response of ALDH1A1 knockout clonal derived cells to radiation was analyzed using colony forming units (CFU), proliferation assays (Alamar blue), and Annexin V/PI staining. All the ALDH1A1 knockout clones analyzed showed enhanced radiation response with reduced proliferation, and colony formation units post-radiation, as compared to the parental cells. Furthermore, tumor initiation was measured in the parental cells and ALDH1A1 KO clones using the organoid culture system platform. Overall, ALDH1A1 KO clonal derived cells formed fewer tumoroids post-radiation, as measured by cell proliferation assays in a 3D culture system. Together these findings highlight the critical relationship between ALDH1A1 expression and radiation sensitivity, emphasizing the potential of ALDH1A1 as a target for improved treatment of TNBC.
 Citation Format: Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, Jennifer Sims Mourtada. CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/addcc6f0bdb6fd3863e3c5a752c6092ba78d179c" target='_blank'>
              Abstract P3-02-25: CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer
              </a>
            </td>
          <td>
            Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, J. Mourtada
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Early detection of epithelial ovarian cancer (EOC) is crucial for improving patient survival. Current screening methods have limitations, highlighting the need for novel biomarkers. Circulating tumor DNA (ctDNA) methylation analysis offers a promising approach. Methods This study included 10 patients with EOC and 10 patients with benign pelvic masses. We collected plasma samples from these patients and isolated ctDNA. We then conducted whole-genome methylation sequencing using the TAPS (TET-assisted pyridine borane sequencing) method, which allows for single-base resolution detection of 5-methylcytosine and 5-hydroxymethylcytosine. Bioinformatics analysis was performed to identify differentially methylated genes and regions. We further validated candidate biomarkers using bisulfite sequencing, qRT-PCR, and IHC. TCGA methylation data were analyzed for external validation. Results We identified 35 differentially methylated genes, with NBL1 and CASZ1 as potential candidates. NBL1 gene hypermethylation in EOC patients was significantly associated with reduced mRNA expression, suggesting its role as a tumor suppressor gene. CASZ1 methylation patterns were inconsistent between blood and tissue, indicating limited utility as a diagnostic biomarker. We also observed widespread hypo-methylation across the genome and hyper-methylation in specific regions of differential methylation. GO and KEGG pathway enrichment analyses revealed that the differentially methylated genes were involved in various biological processes and pathways relevant to cancer pathogenesis. There is a significant negative correlation between the methylation level and the mRNA level of the NBL1 gene, suggesting that hypermethylation of the NBL1 gene may be associated with a reduction in its expression. Furthermore, immunohistochemical analysis indicates a downregulation of NBL1 expression in ovarian cancer tissues, which contrasts with the strong positive expression observed in benign tissues. Conclusion Our study demonstrates the potential of ctDNA methylation analysis for early EOC detection. we propose that NBL1 gene hold potential as screening biomarkers for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03957-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701ff19d15b67d1d6589de020677e9a446c8f2c4" target='_blank'>
              ctDNA methylation profiling reveals NBL1 as a promising biomarker for early ovarian cancer screening
              </a>
            </td>
          <td>
            Shunxian Zhou, Huan Yin, Lei Yan, Nian Xie, Chun Fu
          </td>
          <td>2025-07-28</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Conventional wisdom holds that nuclear oncogenes and tumor suppressors initiate malignant transformation. However, mounting research suggests that mitochondrial dysfunction—rooted in the unique evolutionary history and genetic autonomy of mitochondria—may serve as a more fundamental driver of oncogenesis. This paper proposes a “mitochondria-first” hypothesis of cancer, emphasizing the pivotal role of mitochondrial DNA (mtDNA) mutations, metabolic reprogramming, and immune evasion. By examining the evolutionary conflict between host and mitochondria, evaluating high mtDNA mutation rates, and highlighting the disruptive potential of mitochondrial transfer to immune cells, we outline robust mechanisms through which mitochondria could ignite cancer development. We also discuss emerging diagnostic and therapeutic approaches that target mitochondrial integrity, offering a potential paradigm shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09759d1fd0b99bd9f5921041ad32e37eb247e60" target='_blank'>
              Are Mitochondria the True Origin of Cancer? A Hypothesis-Driven Perspective
              </a>
            </td>
          <td>
            Sergio Da Silva
          </td>
          <td>2025-07-01</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e75d3ecf904d9e7aa51e15582bd637748462a3" target='_blank'>
              Novel mutations of SRPX facilitate the stemness and malignant progression of glioma.
              </a>
            </td>
          <td>
            Siyuan Tang, Chunhui Qu, Mingyu Zhang, Peijun Zhou, Xingzhi Peng, Zhuan Zhou, Liangfang Shen, Lifang Yang
          </td>
          <td>2025-07-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers, with an incidence that has increased significantly over the last 20 years. The World Health Organization (WHO), under its cancer program, identifies CRC as the third most prevalent cancer worldwide with a high mortality rate, especially in patients under the age of 50. Despite advances in surgery, chemotherapy, radiotherapy, and molecular targeted therapy, CRC shares a low survival rate and poor prognosis due to late diagnosis. To address these challenges, research into alternative diagnostic and therapeutic strategies has increasingly focused on molecular mechanisms, including noncoding RNAs (ncRNAs). Circular RNAs (circRNAs), a subclass of endogenous ncRNAs characterized by their covalently enclosed loop structure, demonstrate greater stability than linear RNAs, making them potential candidates for clinical applications. The circRNAs possess differential expressions in cancers and function as tumor suppressive or oncogenic activities. This review discusses the recent findings on circRNAs and their potential for biomarkers and therapeutic targets in CRC. As circRNAs can serve as miRNA sponges, increase or decrease mRNA expression, and function to regulate an oncogenic or tumor suppressor pathway, there may be the possibility for an early diagnosis, prognosis, or therapeutic role of circRNAs in CRC. Highlighting the clinical implications of circRNAs, this review discusses their potential to transform current CRC management strategies and address critical gaps in timely diagnosis and effective treatment. It also emphasized the need for further clinical research to validate their utility and translate these findings into practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db80f20f575f8c248f315abb0f46ec8b7f78d57b" target='_blank'>
              A review of circular RNAs in colorectal cancer: insights into biomarker discovery and therapeutic targeting
              </a>
            </td>
          <td>
            Sibin Nambidi, Karunakaran Bhuvaneswari Shruthi, Sujatha Padmanaban Kanimozhe, Antara Banerjee, S. Kondaveeti, A. Duttaroy, S. Pathak
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Recent advances in next-generation sequencing and bioinformatics have driven growing interest in the distinct roles of intratumoral microbiota, particularly intracellular bacteria, during tumor evolution. These bacteria increase the likelihood of metastasis, play important roles in cancer progression, and impact therapy efficiency. The present review explores the sources, mechanisms of invasion into cancer cells, and potential survival strategies of intracellular bacteria in neoplasms, highlighting their critical role in cancer development. We also examine the heterogeneity and intricate interplay of intratumoral microbial communities with immune and cancer cells, emphasizing their potential roles in modulating host genetics, epigenetics, and immunity. Finally, we discuss novel approaches to targeting intracellular bacteria, particularly engineered drug delivery systems, and synthetic biology, which aim to enhance bacterial clearance, reprogram the tumor immune microenvironment, and enhance the efficacy of chemotherapy and immunotherapy. As a result, this review provides new insights to guide future investigations and support the development of microbiota-based interventions in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e7699eeb6508bcb55f9d12ce3e14753975b4d10" target='_blank'>
              Advancements in understanding tumor-resident bacteria and their application in cancer therapy
              </a>
            </td>
          <td>
            Yi-Chen Luo, Xiu-Ting Huang, Rui Wang, Yu-Jing Lin, Jia-Xin Sun, Ke-Feng Li, De-Yun Wang, Yan Yan, Yong-Kang Qiao
          </td>
          <td>2025-07-25</td>
          <td>Military Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: The integration of cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors with endocrine therapy (ET) has revolutionized the management of estrogen receptor-positive (ER+), human epidermal growth factor receptor 2-negative (HER2-) breast cancer. The ability of CDK4/6 inhibitors to induce cell cycle arrest is well documented, however, their potential to modulate the tumor immune microenvironment (TIME) has recently been appreciated. Pre-clinical studies suggest that CDK4/6 inhibitors elicit an anti-tumoral immune response through increased antigen presentation, activation of T- cells, and suppression of T regulatory cells. However, these data lack the granularity to characterize the impact of CDK4/6 inhibitors on the diverse immune cell populations including the myeloid compartment within the TIME. Here, we use single-cell RNA-sequencing (scRNA-seq) of isolated CD45+ cells to characterize the baseline immune landscape and the immunomodulatory effects of abemaciclib and ET in matched advanced ER+/HER2- breast tumor biopsies. Methods: Sixteen patients with ER+/HER2- advanced breast cancer with lesions amenable to biopsy and beginning 1st-line abemaciclib and ET were enrolled (NCT04352777) and underwent a biopsy at baseline and 4 weeks post-treatment. Biopsies were immediately dissociated into single-cell suspensions and viably cryopreserved. Prior to scRNA-seq, cryopreserved single-cell suspensions were thawed and CD45+ cells were isolated. scRNA-seq libraries were prepared using 10X Genomics 3’ Gene Expression kit and raw data were processed using Cell Ranger (10X Genomics). Data integration, clustering, and differential expression analysis were performed using Seurat. Clusters were annotated using canonical cell type markers. Responders were defined as patients on treatment for greater than 6 months without progression as assessed by RECIST 1.1. Changes in relative cell type proportion were assessed per patient, timepoint and lineage (myeloid or lymphoid). Survival analyses were performed with gene expression data from the METABRIC cohort. Results: In total, 13 matched pairs were evaluable with an average patient age 60.8 years. Of these patients, 8 were classified as responders, 3 as non-responders, and 2 were not evaluable. From the 170,798 cells analyzed, we identified T-cells, NK cells, B-cells, plasmablasts, myeloid cells, and a cluster of proliferating immune cells. Responders had an expansion of activated, GZMA+/IFNG+ CD8+ T-cells and NK cells (p < 0.05) and a concordant decrease in exhausted TIGIT+/PD-1+ T-cells. Additionally, our data reveal a complex and functionally heterogeneous myeloid subpopulation. We identified 14 diverse myeloid populations. Within the myeloid compartment, we identify a population of TREM2+ lipid-associated macrophages (LAMs), associated with a highly suppressive phenotype in multiple solid tumor types including breast, that decrease in relative proportion following treatment with abemaciclib and ET in treatment-responsive patients. Differential expression analysis was used to generate a gene signature defining this population. Lower expression of our TREM2+ LAM signature is significantly associated with improved overall survival in ER+/HER2- breast cancer patients from the METABRIC cohort. Conclusions: Using scRNA-seq analysis, we show that response to abemaciclib and ET is associated with an increase in cytotoxic lymphoid populations and a decrease in exhausted T-cells. For the first time, we show an immunosuppressive population of TREM2+ LAMs decreases following treatment with abemaciclib and ET in responsive patients. In a separate dataset, lower expression of a TREM2+ LAMs gene signature is associated with improved overall survival. Further translational work is ongoing to assess the promise of targeting TREM2+ LAMS in ER+/HER2- MBC.
 Citation Format: Sarah Sammons, Corinne Strawser, Binita Chakraborty, Daniel Michaud, Susan Dent, Jaycee Cushman, Vaibhav Jain, Sara Tolaney, Kent Weinhold, Sandra McAllister, Katherine Miller, Katelyn Steadman, Stephanie Arvai, Emily Hocke, Charles Perou, Shom Goel, Donald McDonnell, Jennifer Guerriero, Sarah Sammons. Matched pair single-cell RNA-sequencing highlights the impact of abemaciclib and endocrine therapy on different myeloid cell populations in advanced ER+/HER2- breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-16.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6f6413a6c751b271482fb22b353da3a57c32b7" target='_blank'>
              Abstract P1-02-16: Matched pair single-cell RNA-sequencing highlights the impact of abemaciclib and endocrine therapy on different myeloid cell populations in advanced ER+/HER2- breast cancer
              </a>
            </td>
          <td>
            S. Sammons, Corinne H Strawser, Binita Chakraborty, Daniel E. Michaud, Susan Dent, Jaycee Cushman, Vaibhav Jain, S. Tolaney, Kent J. Weinhold, Sandra McAllister, Katherine Miller, Katelyn Steadman, Stephanie Arvai, Emily Hocke, C. Perou, S. Goel, Donald McDonnell, Jennifer L. Guerriero
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is a prevalent malignant hematologic disease characterized by the abnormal proliferation and accumulation of immature lymphocytes in bone marrow and lymphoid tissues. In our study, Oxford Nanopore Technologies (ONT) sequencing was performed to investigate four types of methylation modifications—6 mA, CHG, CHH, and CpG—in a pair of monozygotic twins, where one twin has ALL and the other is healthy. The results showed the significant global hypomethylation of CpG sites and an increase in 6 mA, CHG, and CHH methylation in the twin diagnosed with ALL. Notably, the hypomethylation of CpG was particularly increased in the open sea, gene body, and 3'UTR regions, while 6 mA and CHG modifications exhibited high methylation levels in the gene body, TSS1500, TSS200, and 3'UTR regions. Additionally, CHH modifications showed high methylation across all genomic regions. Within the differential methylation loci (DML), we identified several genes related to tumorigenesis and progression (such as ZDHHC11, NBPF1, and TPTE). Furthermore, we systemically reviewed the literatures on leukemia and DNA methylation modifications, providing a comprehensive description of their correlation. In summary, these findings indicate that DNA methylation plays a crucial role in the onset and progression of ALL, offering valuable insights for future research into its impact on leukemia development. Graphic abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01906-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42946eeaa2c1927bc81f747dbb50fc72408e9f3e" target='_blank'>
              DNA methylation in monozygotic twins discordant for acute lymphoblastic leukemia: a case report and systematic review
              </a>
            </td>
          <td>
            Mao-ling Sun, Yang Li, Rong-Xi Man, Si-wen Wang, Rong Guo, Ying Yang, Yu-qing Pan
          </td>
          <td>2025-06-06</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Elucidating the mechanisms of radioresistance in highly radiotolerant organisms can provide valuable insights into the adaptation and evolution of organisms. However, research has been limited on many naturally occurring radioresistant organisms due to a lack of information regarding their genetic and biochemical characteristics and the difficulty of handling them experimentally. To address this, we conducted an experiment on adaptive evolution using gamma radiation as the selection pressure to generate evolved Escherichia coli with gamma radiation resistance approximately one order of magnitude greater than that of wild-type E. coli. Gene expressions in all wild-type and evolved radioresistant E. coli in the presence or absence of gamma irradiation were analyzed and compared using RNA sequencing. Under steady-state conditions, the genes involved in survival, cell recovery, DNA repair, and response following stress exposure were upregulated in evolved E. coli compared with those in wild-type E. coli. Furthermore, the evolved E. coli induced these genes more efficiently following gamma irradiation and greater DNA repair activity than that in the wild-type E. coli. Our results indicate that an increased steady-state expression of various anti-stress genes, including DNA repair-related genes, and their highly efficient induction under irradiation are responsible for the remarkable radioresistance of evolved E. coli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13cc80bdc29b64dbce3840bea6a235a4ab6a6a91" target='_blank'>
              Acquired Radioresistance Through Adaptive Evolution with Gamma Radiation as Selection Pressure: Increased Expression and Induction of Anti-Stress Genes
              </a>
            </td>
          <td>
            Takeshi Saito, Hiroaki Terato
          </td>
          <td>2025-07-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8700bbb72504289e42534db89ad8208a3e240803" target='_blank'>
              Modeling the Impact of Dedifferentiation on Colorectal Cancer Growth and Chemo-Immunotherapy Response.
              </a>
            </td>
          <td>
            Yuman Wang, Mengfan Tan, Da Zhou
          </td>
          <td>2025-07-22</td>
          <td>Bulletin of mathematical biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>