<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-12 09:41:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>44</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>90</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>124</td>
          <td>147</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>397</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>341</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres, stochastically segregate during cell division3, 4, 5–6 and persist over many generations. It has been more than 40 years since ecDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei, but the mechanism underlying this process remains unclear3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. Using Retain-seq, a genome-scale assay that we developed, we reveal thousands of human retention elements that confer generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromosome conformation capture show that retention elements physically interact with mitotic chromosomes at regions that are mitotically bookmarked by transcription factors and chromatin proteins. This activity intermolecularly recapitulates promoter–enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated. Targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, which suggests that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, and reveal the principles of episome immortality intrinsic to the human genome. A combination of genome-wide functional screening, imaging and chromatin profiling identifies a new class of highly prevalent genomic elements that help retain extrachromosomal DNA copies in dividing cells and persist across generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D.
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, Jessica Alsing, J. Luebeck, Maria E. Stefanova, Owen S Chapman, Katerina Kraft, Shu Zhang, Jun Yi Stanley Lim, Yipeng Xie, Yoon Jung Kim, Sihan Wu, Lukas Chavez, Guy Nir, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract In this Perspective article, we follow the journey of a gene that breaks free from its stringent chromosomal transmission dynamics to become an independently multiplying agent on so-called extrachromosomal circular DNA (ecDNA/eccDNA). We discuss how the release of a gene from its chromosomal anchor can affect its dosage, regulatory context, and potential evolutionary trajectory before examining the phenotypic implications for unicellular and multicellular eukaryotes that transmit genes on circular DNA. We also briefly explore the fundamental difference between circularized genes in flowering plants (angiosperms) and mammals (primarily cancer) concerning somatic and germline inheritance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) has been linked to oncogene amplification, evolution of drug resistance, and poor outcomes in various adult cancers. However, the extent to which ecDNA contributes to oncogenesis of pediatric solid tumors has heretofore remained understudied. In addition, the relative importance of intra- versus extrachromosomal amplification remains an open question. We have therefore reanalyzed whole genome sequencing data to identify ecDNA sequences in a large retrospective cohort of 3,000 pediatric solid tumors. ecDNA was most frequent in aggressive tumor types and rare in low-grade or benign histologies. Among pediatric CNS tumors, we find ecDNA most frequently in embryonal tumors with multilayered rosettes (ETMR), pediatric high-grade gliomas (pHGG) and medulloblastomas, and examples of ecDNA+ tumors belonging to ultra-rare subtypes of pineoblastoma and ependymoma. Recurrently ecDNA-amplified genomic regions included well-established oncogenes as well as genes with emerging evidence of possible oncogenic roles. Notably, multivariate Cox regression identifies additive effects of amplification and extrachromosomal amplification which contributed independently to poor patient outcomes. These results constitute a comprehensive map of the genetic and phenotypic diversity of extrachromosomal amplification in pediatric tumors, and underscore the utility of large genomic data resources to drive clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba3f4a1804d64a16c42528baab0131c5bc86b5e" target='_blank'>
              10186-CO-2 The genomic landscape of extrachromosomal DNA amplification across pediatric brain tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Daisuke Kawauchi, Lukas Chavez
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage-replication/fusion process explains complex rearrangements and segmental DNA amplification.
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db26ccc6dc7e76ac6a63d600396d97f0ca6b89f2" target='_blank'>
              Passenger mutations link cellular origin and transcriptional identity in human lung adenocarcinomas
              </a>
            </td>
          <td>
            Sukanya Panja, Padmaja Mantri, K. E. Johnson, J. Andrade-Martínez, S. Yang, Aditya Deshpande, Huasong Tian, Shaham Beg, Kentaro Ohara, Alessandro Leal, Joel Rosiene, Marlon Stoeckius, Peter Smibert, William D. Travis, J. Mosquera, P. Polak, Marcin Imieliński
          </td>
          <td>2025-11-27</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, Uri Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent mini-chromatin. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template is mediated by decreased efficacy of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS with LANA as similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA TR binding establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memories from prior activation, determines the degree of KSHV lytic replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43595752cf1e93615641d0b346a78e9c25448ab3" target='_blank'>
              KSHV TR deletion episomes uncover enhancer–promoter dynamics in gene regulation
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, J. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-12-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary This study investigates the genomic alterations of cleavage and polyadenylation factors and their clinical relevance in cancer. We show that mutations in CPA genes may not be clinically relevant. However, copy number variations, particularly amplification of CPSF1, demonstrate significant impact on patient prognosis. We show that CPSF1 amplification, although co-occurring with MYC amplification, can independently change alternative polyadenylation (APA) patterns and dysregulate cancer-associated gene signatures. These results highlight the crucial impact of CPA gene copy number variation on biological processes and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has emerged as a critical determinant of poor clinical outcomes and immune escape in tumors, but the high cost and technical complexity of current detecting techniques limit its broader investigation in cancer immunotherapy. Leveraging the combined machine learning algorithms including the least absolute shrinkage and selection operator (LASSO) regression, RandomForest (RF) and Recursive Feature Elimination (RFE), we developed a 12-gene transcriptomic score (EC_score) to predict the existence of ecDNA through RNA-seq. EC_score demonstrated reliable predictive performance in two independent cohorts (AUC > 0.70), validated by fluorescence in situ hybridization (FISH) in both cell lines and clinical samples. Next, we found that EC_score emerged as an independent adverse prognostic factor across multiple immunotherapy cohorts. Notably, high EC_score correlated with cell cycle activation and immunosuppression, characterized by reduced lymphocytes infiltration and upregulated immunosuppressive markers, including MHC molecules, co-inhibitory immune checkpoints and TGF-β signals. In general, we established and validated a 12-gene signature (EC_score) derived from RNA-seq, offering a novel computational tool for predicting the presence of extrachromosomal circular DNA and stratifying cancer immunotherapy response. Overview of the analytical framework integrating multi-omics data, machine learning, and experimental validation to establish EC_score and characterize its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97428a7d98b9f7fbbd3050ae63fe8f2b1c83fe29" target='_blank'>
              Machine learning identified extrachromosomal DNA-related 12 gene signatures to predict cancer immunotherapy response
              </a>
            </td>
          <td>
            Yan Ju, Jingwei Zhang, Jiaming Deng, Xingyuan Wu, Haotian Yang, Changyu Tao, Xiao Li
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer.
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of ovarian research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Copy number variants at genomic loci evolve at a high rate, are linked to many different diseases, and play a role in adaptive evolution in humans and other organisms. Here we show that stickleback fish from freshwater environments have rapidly and repeatedly evolved an expanded number of copies of a gene family involved in muscle development, Myosin Heavy Chain 3 Cluster C (MYH3C), compared to marine populations. Differences in copy number between marine and freshwater fish are maintained even in the presence of gene flow, suggesting that MYH3C changes represent adaptive divergence between ecotypes. Copy number expansion occurs by tandem duplication of MYH3C coding and regulatory regions on the stickleback sex chromosome. We identify a muscle regulatory enhancer within the expanded MYH3C region and show that elevated copy number is associated with developmental and tissue-specific increases in corresponding mRNA expression levels in skeletal muscle. Common MYH3C clusters include 3-, 4-, 5-, and 6-copy variants that likely evolved through a combination of microhomology-mediated break repair and non-allelic homologous recombination. Our results provide a new example of copy number changes in a wild species and identify CNVs as potential “hotspots” of repeated adaptive evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f303c1abd69915406b352bedd1f516532851ab" target='_blank'>
              Rapid and repeated evolution of myosin copy number in threespine stickleback
              </a>
            </td>
          <td>
            Alyssa M. Yoxsimer, Rhea R. Daugherty, Emily E. Hare, Y. F. Chan, Felicity C. Jones, Garrett A. Roberts Kingman, Emma G. Offenberg, Timothy R. Howes, Haili Zhang, Alex A. Pollen, S. Brady, Kathleen T. Xie, Heidi I. Chen, Craig B. Lowe, Eric H. Au, J. Grimwood, Jeremy Schmutz, Richard M. Myers, D. Schluter, D. Heins, Miguel L. Reyes, John A. Baker, Bjarni Jónsson, T. Reimchen, Michael A. Bell, David M. Kingsley
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Structural variations in cancer predisposition genes significantly contribute to the pool of pathogenic variants underlying heritable cancer susceptibility. In this report, we describe and functionally characterize a novel transposon-mediated germline pathogenic insertion identified within the coding region of the BRCA1 gene. The inserted sequence comprises the entire processed transcript of RPL18A, a ribosomal protein-coding gene, and may have been generated through a target-primed reverse transcription event. Robust molecular testing confirmed the heritability of the variant and provided evidence of its correlation with the observed phenotype. This is the first documented case of a germline cancer susceptibility variant arising through this unique mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab6faf5664bd66e2e317ebc80d8ac230de46d4a" target='_blank'>
              Processed Transcript Insertion as a Novel Germline Mutational Mechanism in BRCA1-Associated Hereditary Breast Cancer
              </a>
            </td>
          <td>
            A. Bozsik, H. Butz, V. Grolmusz, Petra Nagy, T. Pócza, Erika Tóth, E. Csernák, Attila Patócs, János Papp
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pedigree analysis remains the gold standard for rare disease diagnostics, yet whole genome sequencing studies typically omit critical regions like centromeres, telomeres, and acrocentric chromosome p-arms. Here, we present telomere-to-telomere (T2T) reference genomes for four self-identified African American individuals of admixed ancestry spanning three generations. Our parent-of-origin assigned, chromosome-level assemblies revealed precise meiotic recombination breakpoints in previously inaccessible regions, including recombination events across acrocentric and subtelomeric sequences. Centromeric regions were highly stable, with multi-megabase arrays inherited intact across three generations, while the position of kinetochore assembly sites remained consistent and predominantly associated with the p-arm proximal region. The relative lengths of telomeres on individual chromosomes were maintained across generations. Using a targeted rDNA assembly approach, we reconstructed a complete megabase-scale ribosomal DNA (rDNA) array corresponding to the paternal chromosome 14. This openly available pedigree provides a benchmark dataset for studying recombination and genetic and epigenetic variation across the complete genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56491af219e3e15ccd2642a4c21320991315613" target='_blank'>
              Complete genomes of a multi-generational pedigree to expand studies of genetic and epigenetic inheritance
              </a>
            </td>
          <td>
            Monika Cechova, T. Potapova, A. Rechtsteiner, Glenn Hickey, Rebecca Serra Mari, Mira Mastoras, Julian Menendez, Nikol Poláková, Prajna Hebbar, Fedor Ryabov, Hailey Loucks, Aljona Groot, Tomáš Pavlík, Mobin Asri, Shihua Dong, Stephanie M. Yan, Julian K. Lucas, Steven J. Solar, M. Borchers, Mark Mattingly, S. McKinney, M. Krátká, Catherine Mikhailova, Ondřej Hanák, Sohinee Tiffany Saha, Emily Xu, D. Antipov, S. Koren, Jordan M. Eizenga, B. McNulty, Joshua M.V. Gardner, T. Hillaker, I. Violich, Christopher Markovic, S. Kruglyak, Shawn Levy, Trevor Wolf, Matthew W. Mitchell, Laura Scheinfeldt, Haoyu Cheng, I. Alexandrov, R. McCoy, B. Paten, A. Phillippy, J. Zook, J. Gerton, Robert S. Fulton, N. Stitziel, Ting Wang, T. Marschall, C. Greider, K. Miga
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6ec9da57c22371a4a23a66c61abb71c70d8441" target='_blank'>
              A global view of human centromere variation and evolution
              </a>
            </td>
          <td>
            Shenghan Gao, Keisuke K. Oshima, Shu-Cheng Chuang, Mark Loftus, Annalaura Montanari, David S. Gordon, Pinghsun Hsieh, Miriam K. Konkel, M. Ventura, Glennis A. Logsdon
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Diverse haematological neoplasms are driven by tyrosine kinase (TK) fusion genes formed by recurrent or non-recurrent genomic rearrangements. The resulting chimeric proteins often present excellent targets for treatment with kinase inhibitors, and the fusion transcripts or genomic junctions can be used as specific targets for molecular monitoring. Whilst the TK genes involved are generally well characterised (e.g. ABL1, PDGFRA, FGFR1), the fusion partners are very diverse, presenting a challenge for detection and characterisation of these structural variants (SV) using current diagnostic methods. We assessed the ability of targeted nanopore sequencing using adaptive sampling to detect fusion genes in myeloid neoplasms. We sequenced genomic DNA from patients (n = 20) with a known or suspected TK gene fusion and identified rearrangements in 18 cases, including all cases with a known TK fusion, typical and atypical BCR::ABL1 rearrangements, an 843Kb deletion causing a FIP1L1::PDGFRA fusion, novel AGAP2::PDGFRB and NFIA::PDGFRB fusions, and a complex CCDC88C::PDGFRB rearrangement with multiple translocation events. The approach was fast (<72 h/sample from DNA to result), flexible with minimal hands-on laboratory time, and provided accurate, patient-specific characterisation of genomic breakpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afde2671402c0ec98e756c36e5ab2f520fb96100" target='_blank'>
              Long read nanopore DNA sequencing with adaptive sampling to identify tyrosine kinase fusion genes
              </a>
            </td>
          <td>
            Matthew Salmon, Nicole Naumann, J. Rinke, M. Meggendorfer, D. Radia, Mark Pomfret, Thomas Ernst, A. Hochhaus, A. Reiter, William J. Tapper, Helen E. White, Nicholas C. P. Cross
          </td>
          <td>2025-11-18</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Chromosomal structural changes happen when genomic stability is compromised, such as in disease or in species hybrids. In these contexts, diminished control of repetitive elements has been reported, but the reasons for this are not yet well understood. There are causal associations between repetitive elements and phenotypes such as disease progression, leading us to the hypothesis that chromosomal structure may be affected by transposable elements (TEs). In an intraspecific hybrid Drosophila melanogaster cell line (PnM), the degree of pairing among trans homologous chromosomes was affected by the presence of nearby TEs, in particular, LINE and LTR elements, such as Baggins1 or Gypsy. Chromosomal pairing was significantly lower in windows containing TEs than in windows without any TEs. Pairing was also affected by TEs in mouse, which suggests a possible general association between TEs and pairing that is highly conserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b05bf5cfcd82a8fc82e293a52e5e86b57f277d" target='_blank'>
              Haplotype-Resolved Chromatin Conformation Data Reveals Relationship Between Transposable Elements and Chromosomal Pairing
              </a>
            </td>
          <td>
            Luke K Genutis, Patrick Villanueva, Rita M. Graze, Jumana Alhaj Abed, Lauren M McIntyre, S. Nuzhdin
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Most animal genomes are partitioned into topologically associating domains (TADs), created by cohesin-mediated loop extrusion and defined by convergently oriented CCCTC-binding factor (CTCF) sites. The dynamics of loop extrusion and its regulation remain poorly characterized in vivo. Here we tracked the motion of TAD anchors in living human cells to visualize and quantify cohesin-dependent loop extrusion across multiple endogenous genomic regions. We show that TADs are dynamic structures whose anchors are brought in proximity about once per hour and for 6–19 min (~16% of the time). Moreover, TADs are continuously extruded by multiple cohesin complexes. Remarkably, despite strong differences in Hi-C patterns across chromatin regions, their dynamics is consistent with the same density, residence time and speed of cohesin. Our results suggest that TAD dynamics is primarily governed by the location and affinity of CTCF sites, enabling genome-wide predictive models of cohesin-dependent chromatin interactions. Live-cell fluorescence microscopy and polymer simulations in human HCT116 cells show similar cohesin density, residence times and extrusion speed across multiple genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715a0a11e407c2847c391eae8db4fee949a2d73b" target='_blank'>
              Uniform dynamics of cohesin-mediated loop extrusion in living human cells
              </a>
            </td>
          <td>
            Thomas Sabaté, Benoît Lelandais, Marie-Cécile Robert, Michael Szalay, J. Tinevez, Edouard Bertrand, Christophe Zimmer
          </td>
          <td>2025-11-14</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Large-scale genomic rearrangements in cancer cells can lead to the creation of fusion genes which alter the expression or function of the partner genes. Some fusion genes act as tumour drivers and the use of kinase inhibitors in patients with specific fusions have led to breakthroughs in cancer therapy. The large clinical and scientific interest in fusions has led to the development of software that use RNA sequencing data to identify fusion transcripts. Unfortunately, fusion transcript callers output many predictions which lack underlying genomic rearrangements and large datasets with validated fusions are scarce. This paper and the accompanying Fusions4U web application present a resource of validated and annotated gene fusions for 328 cell lines from the Cancer Cell Line Encyclopedia. Predicted fusion transcripts from Arriba and STAR-Fusion were analysed with our published validation pipeline that uses matched whole-genome sequencing data to identify discordantly mapped read pairs and candidate genomic breakpoints that support genuine fusion events. This resulted in 8,753 and 2,244 validated fusion transcripts for Arriba and STAR-Fusion predictions, respectively, with 1,596 fusions common to both. Additional layers of annotation include alternative splicing of fusion transcripts, kinases, microRNA host genes, genes in the COSMIC Cancer Gene Census, as well as known fusion gene pairs from the Mitelman Database of Chromosome Aberrations and Gene Fusions in Cancer and the TumorFusions dataset. We furthermore analysed information about fusion genes together with cell line data from the PRISM drug repurposing screening as an example of how this dataset can be used. This resource can be used to design experiments for functional studies and drug development, alone or in combination with publicly available information for the Cancer Cell Line Encyclopedia cell lines. The large collection of validated fusion transcripts with candidate genomic breakpoints can also be used in development and evaluation of bioinformatic tools for fusion transcript detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1751293449b5956386c8f2f0dbda4aebaafaddec" target='_blank'>
              Fusions4U: a resource of validated and annotated gene fusions in 328 cancer cell lines
              </a>
            </td>
          <td>
            Arianna Alamshahi, Helena Persson
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24028ea6e94f36acd4cba9ad220ab6b9d876ab02" target='_blank'>
              Telomeric repeat distribution and chromosomal evolution in the slipper orchid genus Paphiopedilum
              </a>
            </td>
          <td>
            T. Lan, Victor A. Albert
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tandemly repeated non-coding sequences, widely known as satellite DNAs (satDNAs), are extremely diverse and highly variable components of eukaryotic genomes. In recent years, advances in high-throughput sequencing and new bioinformatics platforms have enabled in-depth studies of all (or nearly all) tandem repeats in any genome (the satellitome), while a growing number of telomere-to-telomere assemblies facilitates their detailed mapping. Research performed on a large number of non-model plant and animal species changed significantly the “classical” view on these sequences, both in an organizational and functional sense, from ballast compacted in the form of heterochromatin to elements that are important for structuring the entire genome, as well as for its functions and evolution. The diversity of repeat families, and the complexity of their intraspecies and interspecies distribution patterns, posed new questions, urging for species-by-species comparative analyses. Here we integrate some basic features of different forms of sequences repeated in tandem and rapidly growing data evidencing extensive dispersal of satDNA sequences in euchromatin, their putative roles and evolutionary significance. Importantly, we also present and discuss various issues brought on by the use of new methodological approaches and point out potential threats to the analysis of satDNAs and satellitomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df02ce17404430620feb58c2c132e48c76b2721" target='_blank'>
              Satellite DNA Genomics: The Ongoing Story
              </a>
            </td>
          <td>
            M. Garrido-Ramos, Miroslav Plohl, Eva Šatović-Vukšić
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5adbde5a06ec4e198106c731d03948bb0cc143" target='_blank'>
              CAD-C reveals centromere pairing and near-perfect alignment of sister chromatids
              </a>
            </td>
          <td>
            Axel Delamarre, Maurane Reveil, David Pintor Pichon, Michael A. Boemo, I. Whitehouse
          </td>
          <td>2025-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Oncogene-targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance. Therapeutic approaches to treat tumours that have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms. Rather than treating resistance after it emerges, it may be possible to prevent it by inhibiting the adaptive processes that initiate resistance, but these are poorly understood. Here we report that residual cancer persister cells that survive oncogene-targeted therapy are growth arrested by drug stress-induced intrinsic type I interferon signalling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA fragmentation factor B (also known as caspase-activated DNase), which induces DNA damage, mutagenesis and stress response factor activating transcription factor 3 (ATF3). ATF3 limits activator protein 1-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DNA fragmentation factor B or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes the regrowth of residual cancer cells that survive drug treatment. Williams et al. report a growth arrest mechanism in residual cancer persister cells through targeted therapy-induced upregulation of type I interferon signalling, which is negatively regulated by apoptotic DNA endonuclease DFFB to allow tumour relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0eb23b0262f48e292a21b5682ac6f6faf0a4d24" target='_blank'>
              DNA fragmentation factor B suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F. Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X Wang, B.E. Mauch, Rhea Plawat, Ariel H. Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J. Hangauer
          </td>
          <td>2025-11-17</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecfb36caac2df07b165f21a1abfc41a4ec2bd5d" target='_blank'>
              A PIWI protein-dependent DNA N6-adenine methylation pathway in Oxytricha protects genomic sequences from deletion
              </a>
            </td>
          <td>
            Margarita T. Angelova, Yi Feng, D. Villano, Erhan Aslan, , P. Esteve, Sagnik Sen, William E. Jack, S. Pradhan, L. Landweber
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f07c976060b51790e18f58bb3a0d0f1e60e54e5" target='_blank'>
              Lineage-wide evolution of 3D genome organisation and centromeres in brown algae
              </a>
            </td>
          <td>
            Pengfei Liu, Rory J. Craig, Elena Avdievich, Fabian B. Haas, Chang Liu, Susana M. Coelho
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Diverse Igh repertoires require successful V(D)J recombination allowing B cell receptor expression and Ig secretion for humoral immune responses. Igh locus contraction has been implicated in generating spatial proximity between distal VH segments and the recombination center via cohesin mediated loop extrusion. However, it remains unclear why some distal VH segments recombine with high frequency while other more proximal VH are rarely used. Long non-coding RNAs (lncRNAs) have emerged as regulators of cellular development, differentiation and gene expression. Here we report exceptionally high expression of lncRNAs at the Igh locus and other AgR loci engaged in V(D)J recombination. A tight correlation was found between positions of multi-exonic lncRNAs, Igh enhancers and chromatin loop anchors. We propose an integrated model of factors including lncRNAs and loop extrusion in determining Igh locus topology and VH gene usage during recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c775445a515375ee21f6cc1808e2ec8c711272f" target='_blank'>
              Exuberant long noncoding RNA expression may sculpt Igh locus topology
              </a>
            </td>
          <td>
            Ellen Drake, Sarah Naiyer, Xinyan Qu, Khalid Bhat, Hammad Farooq, M. Maienschein-Cline, Jie Liang, Amy Kenter
          </td>
          <td>2025-11-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DIS3 is the main catalytic subunit of the nuclear RNA exosome, a complex playing a crucial role in RNA processing and the degradation of various noncoding RNA substrates. In mice, DIS3 is essential for genomic rearrangements during B cell development, but its role in terminal plasma cell (PC) differentiation has not been explored. Although DIS3 gene alterations are frequent in multiple myeloma (MM), a PC malignancy, their molecular impact remains poorly understood. In this study, we developed an antisense oligonucleotide strategy to knock down DIS3 expression in a well-characterized model of human PC differentiation. Reducing DIS3 expression systematically led to decreased B cell proliferation and impaired PC differentiation with lower levels of switched immunoglobulin secretion. Transcriptome analyses confirmed alterations in the proliferation and differentiation programs, alongside an accumulation of noncoding RNAs. Notably, centromere-associated noncoding RNAs were highly sensitive to DIS3 activity, and their accumulation in DIS3-deficient cells, either as transcripts or DNA-associated RNAs, correlated with the mislocalization of the centromere-specific histone variant CENP-A. We finally observed reduced physiological DNA recombination and somatic hypermutation but increased genomic instability in DIS3-deficient cells, in agreement with the higher levels of IGH translocations observed in our large cohort of DIS3-mutant MM patients. Together, these results underscore the essential role of DIS3 in regulating B cell proliferation, DNA recombination, and physiological or malignant PC differentiation in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eccad993afcb380e43e9bb3557c59e3b0bb73b7" target='_blank'>
              DIS3 licenses B cells for plasma cell differentiation in humans
              </a>
            </td>
          <td>
            Emma Miglierina, Julien Bouder, Delfina Ordanoska, Maiwenn Pineau, S. Léonard, Anaïs Schavgoulidze, Gwenaëlle Quéré, Maeva Le Goff, Maé Bouchet, Steve Genebrier, Samuel Bastos Serra Trinca, Laurent Deleurme, C. Monvoisin, Laure Derrier, Charles Marie Dumontet, L. Delpy, J. Moreaux, Jill Corre, Michel Cogné, Brice Laffleur
          </td>
          <td>2025-11-25</td>
          <td>Cellular and Molecular Immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Dipteran insects include numerous harmful species that cause significant agricultural damage. However, assembly of genomes for species in this order has been difficult due to their small body size, poor conservation of telomere and centromere structures, high levels of heterozygosity, and complex genetic backgrounds. In this study, we assemble a high-quality 596 Mb telomere-to-telomere genome for Bactrocera dorsalis, a fruit crop pest, using a strategy with a low-input HiFi CCS library from a male individual and an ONT sequence from pooled inbred individuals. The assembly includes complete structural organization information for centromeres and telomeres, providing insights into the evolution of chromosome structure in insects. Comparative genomic analysis reveals the polyphyletic origin of sex chromosomes across Diptera. Furthermore, we identify a homolog of ATPsynβ as a Y chromosome-specific gene that is highly expressed across multiple male tissues and may provide critical support for male-specific physiological activities. Additionally, we discover several tandem duplications of odorant receptor genes, including a triplet of the OR88a family, which was validated to be involved in the behavioral response to methyl eugenol. In summary, this complete reference genome provides a foundation for future genomic research in Diptera and offers genetic insights for the control of B. dorsalis. Insect genomics is entering a telomere-to-telomere (T2T) era. This study presents a 596-Mb T2T genome of an individual male Bactrocera dorsalis—a species with atypical telomeres, centromeres, and sex-chromosome architecture—advancing genetics, chromosome biology, and pest management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e252de05422a173b0813800c369c40f6c9670cc" target='_blank'>
              Telomere-to-telomere genome assembly of the Dipteran Bactrocera dorsalis from a single individual
              </a>
            </td>
          <td>
            Wei Liu, Qiang Lin, Qi Wang, Wanfei Liu, Jing Jia, Jie Zhang, Ling Yang, Yongyue Lu, Peng Cui, Guirong Wang
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA methylation is a key regulator of eukaryotic genomes, most commonly through 5-methylcytosine (5mC). In contrast, the existence and function of N6-methyladenine (6mA) in eukaryotes have been controversial, with conflicting reports resulting from methodological artifacts. Nevertheless, some unicellular lineages, including ciliates, early-branching fungi and the alga Chlamydomonas, show robust 6mA signals, raising questions about their origin and evolutionary role. Here we apply Oxford Nanopore sequencing to profile 6mA at base-pair resolution across 18 unicellular eukaryotes representing all major supergroups. We find that robust 6mA patterns occur only in species that encode the adenine methyltransferase AMT1. Notably, 6mA consistently accumulates downstream of transcriptional start sites, positioned between H3K4me3-marked nucleosomes, indicating a conserved association with transcriptional activation. Our results support the idea that the last eukaryotic common ancestor had a dual methylation system, with transcription-linked 6mA and repressive 5mC, which has been repeatedly simplified in both multicellular and unicellular lineages through the loss of the AMT1 pathway. Long-read sequencing in 18 unicellular eukaryotes reveals that 6mA is widespread across eukaryotes and is enriched at transcriptionally permissive regions, which are also marked by H3K4me3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84210062581c67c09a67215ecb3a57b62da836b" target='_blank'>
              Adenine DNA methylation associated with transcriptionally permissive chromatin is widespread across eukaryotes
              </a>
            </td>
          <td>
            Pedro Romero Charria, Cristina Navarrete, Vladimir Ovchinnikov, Lan Xu, Luke A. Sarre, Victoria Shabardina, Ewa Księżopolska, Elena Casacuberta, David Lara-Astiaso, Arnau Sebé-Pedrós, Alex de Mendoza
          </td>
          <td>2025-11-18</td>
          <td>Nature Genetics</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40038069867c0720d29003167b0971829e037042" target='_blank'>
              Nanopore whole-genome sequencing reveals conserved chromosome-specific telomere architecture across tissues and populations
              </a>
            </td>
          <td>
            Niklas L. Engel, B. Brors, L. Feuerbach, Peter J. Park
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Flap endonuclease 1 (FEN1) is a structure-specific nuclease with critical functions in DNA replication and repair. FEN1 has been proposed as an anti-cancer drug target because of its synthetic lethal interaction with homologous recombination deficiency (HRD). However, the exploration of FEN1 in this context has been hampered by the lack of suitable small molecule tools. Here, we describe MSC778, a highly potent, specific and selective small molecule inhibitor of FEN1 nuclease activity. MSC778 directly engages FEN1 in cells, enhances its retention on chromatin, and selectively kills BRCA-deficient cells. Mechanistically, MSC778 suppresses DNA replication causing S-phase accumulation, DNA damage, and ultimately, cell death. Cancer cell panel screening identified Ewing sarcoma (EWS) cells as sensitive to MSC778, which is driven by the expression of SLFN11. In addition to HRD, CRISPR screening revealed a spectrum of synthetic lethal interactions between MSC778 and DNA damage response (DDR) factors, such as PARP1, USP1, PARG, and ATR. Furthermore, we demonstrate that combined inhibition of these factors with MSC778 induces synergistic killing of cancer cells. Together these data highlight FEN1 inhibition as an attractive precision oncology strategy either as monotherapy or as a combination therapy with a broad range of current and next generation DDR-targeting agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d720ebfea477ebcfaccacc0ad325d640a78165dc" target='_blank'>
              Mechanistic insights into the monotherapy and combination potential of FEN1 inhibition in cancer therapy
              </a>
            </td>
          <td>
            E. Rajendra, Claudio A Lademann, Bethany Mason, Balca R. Mardin, Lucy Armstrong, Silvia Peripolli, Julian Kreis, Timothea Konstantinou, U. Pehl, J. Bomke, Stefanie Scharrelmann, David Perera, B. Leuthner, Maria Filipa Pinto, Owen A Davis, A. Galbiati, Ana Toste Rêgo, Claire L McWhirter, E. Elinati, Ada Sala-Hojman, Julien Lefranc, Sam E Mann, Robert A Heald, F. Zenke, Graeme C M Smith, Helen M R Robinson, L. Burgdorf
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fdb1478a9bc7792b93fd92427bd911d727f3989" target='_blank'>
              Resolving the CES1 Genomic Locus with Cas9-Directed Targeted Long-Read Sequencing for Precision Pharmacogenomics
              </a>
            </td>
          <td>
            E. Lekka, A. Ambrodji, A. Nater, A. Ballah, U. Amstutz, A. Ramette, C. R. Largiadèr
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a25c976684f63347d1fe095c70af1dbd2cc6459" target='_blank'>
              A molecular framework of chromatin extrusion in plants
              </a>
            </td>
          <td>
            Sofia Tzourtzou, Zhidan Wang, Sarah Leuchtenberg, D. Latrasse, M. Benhamed, Chang Liu
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1f59de44d8e32e88fed7310a38e534218b692e" target='_blank'>
              From Circles to Signals: Representation Learning on Ultra-Long Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jien Li, Zhenke Liu, Ziqi Zhang, Jiaqi Zhang, Ritambhara Singh
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Low-pass whole-genome sequencing (LP-WGS) of circulating tumor DNA (ctDNA) is increasingly recognized for its utility in identifying somatic copy-number aberration (CNA). In this study, we analyzed LP-WGS ctDNA data from 73 pediatric patients with neuroblastic tumor and 11 healthy controls to explore diagnostic value of ctDNA CNA burden (including the genotypings) with a customized bioinformatics workflow. We found that a high baseline ctDNA CNA burden [tumor DNA fraction (TFx) ≥0.2%] was present in 36 of 41 patients (87.80%) with neuroblastoma, six of 22 patients (27.27%) with ganglioneuroblastoma, and three of 10 patients (30%) with ganglioneuroma. High baseline ctDNA CNA burden could predict high-risk neuroblastic tumors with an area under curve (AUC) of 0.95, sensitivity of 94.12%, and specificity of 100%. Frequent chromosomal copy-number changes, including chr17q gain, chr7 gain, chr3p loss, and chr11q loss, were found in ctDNA. Gain of chr17q demonstrated the highest diagnostic value with an AUC of 0.92, indicating strong sensitivity and specificity for detecting high-risk neuroblastic tumors. The homologous recombination deficiency score in the high- and intermediate-risk groups was significantly elevated compared with those in the low-/very low–risk group. The TFx levels and segmental alterations significantly decreased in patients with neuroblastic tumor who underwent chemotherapy, from median TFx = 13.82% before treatment to 0.24% after treatment (P < 0.0001). Our findings highlight the effectiveness of LP-WGS ctDNA CNA analysis as a promising approach for diagnosis and risk stratification of pediatric neuroblastic tumors and for monitoring chemotherapy response. Particularly, ctDNA analysis is minimally invasive, rapid, and cost-effective, which could bring additional benefits in pediatric practices. Significance: Our results support the development of ctDNA CNA analysis as a robust and minimally invasive approach for early detection, molecular diagnosis, and risk stratification of peripheral neuroblastic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01256a37f8920b62f2d81f1a7bbc4542d1a51191" target='_blank'>
              Copy-Number Aberrations in Circulating Tumor DNA Enable Diagnosis and Risk Stratification of Pediatric Neuroblastic Tumors
              </a>
            </td>
          <td>
            Ting Tao, Jiabin Cai, Yinbing Tang, Dong-Fang Lu, Lifeng Zhang, Jinkai Peng, Yilong Wang, Weizhong Gu, Shouhua Zhang, Jinhu Wang
          </td>
          <td>2025-12-03</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA:RNA hybrids are unusual structures found throughout the genomes of many species, including yeast and mammals. While DNA:RNA hybrids may promote various cellular functions, persistent hybrids lead to double strand breaks, resulting in genomic instability. DNA:RNA hybrid formation and removal are therefore highly regulated, including by enzymes that either degrade or unwind RNA from the hybrid. Meiosis is the specialized cell division that creates haploid gametes for sexual reproduction. Previous work in yeast and mammals showed that elimination of DNA:RNA hybrids by RNase H facilitates meiotic recombination. This work demonstrates that the conserved Sen1 DNA/RNA helicase functions during three temporally distinct processes during yeast meiosis. First, SEN1 allows meiosis-specific genes to be expressed at the proper time to allow entry into meiosis. Second, SEN1 prevents the accumulation of hybrids during premeiotic DNA replication. Third, SEN1 promotes the repair of programmed meiotic double strand breaks that are necessary to form crossovers between homologous chromosomes to allow their proper segregation at the first meiotic division. Given the evolutionary conservation of Sen1 with its mammalian counterpart, Senataxin, studies of Sen1 function in yeast are likely to be informative about the regulation of DNA:RNA hybrids during human meiosis as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8231ab0fa8d6701759423756235a545680a4b00a" target='_blank'>
              The conserved SEN1 DNA/RNA helicase has multiple functions during yeast meiosis
              </a>
            </td>
          <td>
            Robert Gaglione, Leonidas Pierrakeas, Lihong Wan, Jonathan Caradonna, A. MacQueen, Ed Luk, N. M. Hollingsworth
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6212cc21f6c9c548c0deb4b30bef6b44b0cc40c" target='_blank'>
              Analyses of Genome-Wide Recombination in Sulfolobus islandicus
              </a>
            </td>
          <td>
            David J. Krause, Changyi Zhang, Rachel J. Whitaker
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            G. Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c06b0dfa9d58dc239114dc3e3b926c74b839c6" target='_blank'>
              Genomic analysis of PLNTY-like tumor progression into epithelioid glioblastoma: a case report.
              </a>
            </td>
          <td>
            S. Mäntylä, Anssi Nurminen, Sanna Huovinen, S. Jaatinen, T. Haapaniemi, R. Nurminen, Ismaïl Hermelo, Stefanie Volz, K. Maass, K. Pajtler, K. Nordfors, H. Haapasalo, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2026-01-08</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0bf2ebc24868e52a4bf9e9e5a2567d97f5b5a0" target='_blank'>
              Integrated proteogenomics uncovers ancestry-specific and shared molecular drivers in localized prostate cancer
              </a>
            </td>
          <td>
            Cara C. Schafer, T. Abulez, Xijun Zhang, Kun-Lin Ho, Jiji Jiang, D. Young, Jesse M. Fox, K. Conrads, B. Hood, G. Sukumar, Darryl Nousome, Praveen-Kumar Raj-Kumar, Mariano Russo, Ayesha A. Shafi, Xiaofeng A. Su, A. Dobi, Amina Ali, S. Elsamanoudi, J. Cullen, W. Figg, G. Petrovics, C. Dalgard, M. D. Wilkerson, N. Bateman, T. Conrads, I. Sesterhenn, Leigh Ellis, Craig D. Shriver, Gregory T. Chesnut, Shyh-Han Tan
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ab72daae241f6f66396542f56383a72c207669" target='_blank'>
              Rescue from pyridostatin-stabilized telomeric G-quadruplexes by DNA cleavage and microhomology-mediated end joining
              </a>
            </td>
          <td>
            Samah Matmati, Satyajeet Rao, Joachim Lingner
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1172fef101356cc11ee77ccf9a62403d7ea10d" target='_blank'>
              A recent shift in centromere size and DNA content in Podospora pseudocomata co-occurs with the loss of a fungal genome defense system
              </a>
            </td>
          <td>
            Ivar Westerberg, Mengyuan Li, Eve Mercier, S. L. Ament-Velásquez, Linnea Sandell, P. Silar, Aaron A. Vogan, P. Grognet, F. Malagnac, Hanna Johannesson
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel C. de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="High-grade gliomas are one of the most lethal cancers and the leading cause of cancer-related mortality in children. Standard treatment typically involves resection, chemotherapy and radiation, yet offer limited improvement in survival rates. Targeted therapies are emerging to show effect in solid tumors harboring tyrosine-kinase activating fusion genes. Here, we present a thorough characterization of an epithelioid glioblastoma in a six-year-old patient, with four relapses and a variety of different treatment strategies. We identified a rare TRIM24::NTRK2 fusion in the primary tumor, which enabled a targeted TRK-inhibitor treatment. Our findings show that the TRIM24::NTRK2 fusion initially had oncogenic abilities, but this became less imperative as the tumor evolved. A general drug resistance to TRK-inhibition was documented. This study addresses the complex and adaptive nature of pediatric epithelioid glioblastomas and highlights the need for continued molecular profiling from relapses and various tumor regions to enable multitarget treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e83f96185edbab3b833499e7bdcbd141d1efe42" target='_blank'>
              Comprehensive characterization and targeted treatment of a pediatric epithelioid glioblastoma with a rare TRIM24-NTRK2 fusion
              </a>
            </td>
          <td>
            S. Keane, Ida Sjöberg, L. Deland, A. Lindlöf, T. O. Bontell, T. Kling, M. Blomstrand, H. Fagman, J. Nilsson, D. Nilsson, M. Sabel, H. Carén, F. Abel
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d50bb9e4155c441955e8609994d5be654a1b395" target='_blank'>
              Lineage-specific evolution of regulatory landscapes in polyploid plant and its diploid progenitors
              </a>
            </td>
          <td>
            Xiang Li, Xuan Zhang, Ziliang Luo, Hao Zhang, J. P. Mendieta, Robert J. Schmitz
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c5e33a1532b92b0810287a27ea33b2ef13bfc31" target='_blank'>
              Targeting of human mitochondrial DNA with programmable pAgo nuclease
              </a>
            </td>
          <td>
            B. Rimskaya, E. Kropocheva, Elza Shchukina, Egor Ulashchik, Daria Gelfenbein, L. Lisitskaya, Vadim Shmanai, Svetlana Smirnikhina, A. Kulbachinskiy, I. Mazunin
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC.
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The fruit fly Drosophila melanogaster is a pivotal model organism, yet its reference genome (ISO-1 strain) retains unresolved gaps in complex regions. Here, we present a near-complete telomere-to-telomere genome assembly (Dm.nT2T) of the Canton S strain (male), generated using an integrative approach combining PacBio HiFi, Oxford Nanopore ultra-long reads, and Hi-C data. This assembly spans 161.63 Mb, closes 93.28% of gaps in the current reference genome, and improves contiguity (contig N50 of 21.93 Mb). We identify centromeric and telomeric regions, yielding a more complete genome map. Comparative analysis with the ISO-1 reference identifies 7989 structural variants, including large insertions/deletions enriched in telomeric and centromeric regions. Notably, we detect SINE transposable elements, and identify 92 genes via multi-strategy annotation. Functional validation of two genes shows that Chr3L.2449, expressed in olfactory tissues, is critical for olfactory sensitivity. Dm.nT2T provides a high-quality resource for exploring genomic complexity, strain diversity, and gene function in Drosophila. The fruit fly Drosophila melanogaster is an important model organism, but its reference genome retains unresolved gaps. Here, the authors report a near-complete telomere-to-telomere genome of the D. melanogaster S strain closing many remaining gaps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c0c53dc4babb329ba51c40b756e0ff7b694249" target='_blank'>
              Near complete assembly of Drosophila melanogaster Canton S strain genome
              </a>
            </td>
          <td>
            Yan-Nan Liu, Jian-Jun Gao, Xiao-Lin Zhuang, Dongdong Wu, Yanbo Sun
          </td>
          <td>2025-12-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progresses from pre-invasive to invasive stages, as well as from ground-glass opacities (GGOs) to solid nodules. However, the dynamic genomic and transcriptomic changes underlying LUAD progression are incompletely understood. Here, we performed whole-genome and transcriptome sequencing on 1008 LUAD samples from 954 patients who underwent surgery at Fudan University Shanghai Cancer Center, with comprehensive follow-up data. There was one atypical adenomatous hyperplasia, 42 adenocarcinomas in situ, 116 minimally invasive adenocarcinomas, and 849 invasive adenocarcinomas spanning all pathological stages. EGFR was the most frequently mutated gene in the study cohort, followed by TP53, RBM10, KRAS, and KMT2D. Mutation frequencies of tumor suppressor genes, such as TP53, RB1, MGA, KEAP1, and STK11, increased as the disease progressed to higher stages. A higher level of genomic instability was seen in LUAD compared with AAH/AIS/MIA samples, characterized by a higher tumor mutation burden, increased somatic copy number alteration burden, and increased structural variation burden. Notably, MAP2K1 E102–I103 deletion was frequently observed in pre-invasive samples, which endowed alveolar type II cells with increased growth potential and initiated tumor formation, suggesting that it is a potential driver mutation of LUAD. In summary, our study highlights key molecular changes during the stepwise progression of LUAD, provides insights into the identification of novel therapeutic targets, and helps to define the curative time window for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90cf835255eacd329046c889367761e186d358b" target='_blank'>
              Cohesin forms fountains at active enhancers in C. elegans.
              </a>
            </td>
          <td>
            Bolaji N. Lüthi, Jennifer I Semple, Anja Haemmerli, Saurabh Thapliyal, Kalyan Ghadage, Klement Stojanovski, Dario D’Asaro, Moushumi Das, Nick Gilbert, Dominique A. Glauser, Benjamin D. Towbin, Daniel Jost, Peter Meister
          </td>
          <td>2025-12-11</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0453b8fd3ee789a5730a5cb95cb52509da58c9" target='_blank'>
              Unraveling miRNA-Driven DNA Damage Response Networks in Pancreatic Adenocarcinoma: A Multi-Omics and Machine Learning Approach
              </a>
            </td>
          <td>
            Shayori Bose, Paramita Basak Upama, S. Ahamed
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy a significant fraction of a wide variety of eukaryotic genomes and can be domesticated into functional sequences harbouring a coding or regulatory potential. While studies in mammals have revealed that retrotransposons can frequently give rise to tissue-specific transcriptional enhancers our understanding of this phenomenon in other vertebrate groups is scarcer. Here, we examined TE occupancy at tissue-specific nucleosome free regions (NFRs) which are not annotated as promoters in the amphibian model organism Xenopus tropicalis. We report three distinct miniature inverted-repeat TEs (MITEs) enriched at distal liver-specific NFRs and belonging to the hAT, Harbinger and Kolobok superfamilies of DNA transposons. These MITEs show a marked depletion at NFRs specific to the bone tissue, probably reflecting a process of negative selection. In addition, we show that they are enriched for transcription factor binding sites known to be bound by key regulators of liver biology, hematopoiesis, and the immune system, and that they are more likely to be located in the vicinity of genes specifically expressed in the liver than other MITE copies that are not associated to a NFR. We also find that these MITEs are not present at orthologous positions in the genome of the related allotetraploid frog Xenopus laevis, while they globally are abundant in this species. We discuss how independent bursts of MITE amplification followed by subsequent domestication episodes might independently have given rise to liver-specific transcriptional enhancers in the Xenopus tropicalis lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24529c91db0790a41856023783a87c69aa0832af" target='_blank'>
              Liver-specific enhancers evolved from independent episodes of MITE domestication in Xenopus tropicalis
              </a>
            </td>
          <td>
            Japhet Rojas, Héctor Castillo, Marco Mundaca, Jorge Fraga, Clément Gilbert, Braulio Valdebenito-Maturana, Sylvain Marcellini
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro. Methods UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization. Results In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts. Conclusions Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="To delineate the genomic and cellular features of CCNE1 amplification across gynecologic cancer types using large-scale multi-omics data and assess its relevance for patient stratification and therapeutic targeting. We analyzed genomic and transcriptomic data from 12,845 patients with gynecological cancers curated from the cBioPortal database, including ovarian, endometrial, and cervical carcinomas. CCNE1 amplification rates, cooccurring genomic alterations, and pathway enrichment were evaluated. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic data from cervical, endometrial, and ovarian tumors were used to investigate CCNE1 expression patterns, cell type specificity, and microenvironmental interactions. CCNE1 amplification was detected in 8.40% of all patients, with the highest rates in patients with ovarian cancer (8.01%) and endometrial cancer (5.06%). Among the ovarian subtypes, carcinosarcoma showed the highest amplification frequency (12.46%), followed by high-grade serous carcinoma (9.84%). Amplified tumors are frequently co-altered by TP53, PIK3CA, and PI3K-AKT pathway genes. Single-cell analyses revealed that CCNE1 expression was largely confined to epithelial cells and was associated with proliferative and structural remodeling pathways, with limited immune engagement across tumor types. Spatial transcriptomics confirmed the localization of CCNE1 in epithelial-rich regions and its correlation with adhesion- and motility-related genes in clear cell and high-grade serous ovarian carcinomas. CCNE1 amplification is enriched in aggressive histologic subtypes of gynecological cancers and is linked to epithelial-specific expression and immune exclusion. These findings indicate that CCNE1 represents a potential molecular marker for tumor aggressiveness, warranting further investigation in the context of patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07c358e69444ecc527d46e34a80b43d34d202949" target='_blank'>
              Multiomic analysis of CCNE1 amplification associated molecular and immune features in gynecological cancers
              </a>
            </td>
          <td>
            Linlin Liu, Wenjin Zheng, Liang Wen, Zhe Zhang, Yuanguang Meng
          </td>
          <td>2025-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma.
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Brain cancers are the most common solid tumors in children and adolescents and remain the leading cause of disease-related death in these populations. Despite their clinical importance, therapeutic progress has been limited by extensive intra- and inter-tumor heterogeneity. In particular, the clonal architecture and evolutionary dynamics of pediatric brain tumors remain poorly understood. Addressing this knowledge gap is critical, as genetically distinct subclones can shape disease progression, treatment resistance, and clinical outcome. To investigate clonal evolution at the population level, we analyzed longitudinal whole-genome sequencing data from 1,742 tumors across 27 brain cancer types in 1,479 pediatric patients from the Children’s Brain Tumor Network. We inferred clonal composition and phylogenetic relationships for each patient and reconstructed the space of tumor phylogenies for the entire cohort using a regularized Gromov-Hausdorff distance. Tumor phylogenetic structure was significantly associated with mutational burden, mutational signatures, cancer classification, and patient's survival status. In longitudinal samples, survival status remained significantly associated with tumor clonal architecture after adjusting for mutational burden, tumor subtype, and anatomical location, underscoring the clinical value of this readout. In contrast, in primary tumors, mutational signatures remained significant after adjusting for mutation count, tumor subtype, anatomical location, and purity, while survival status associations were no longer significant. Mutational signature timing analysis revealed that pediatric brain tumor genetic architecture is shaped by distinct mutational processes acting at different stages of tumor evolution. Truncal somatic mutations were enriched for age-related, defective proofreading, and homologous recombination deficiency signatures, whereas subclonal mutations showed significant contributions from DNA mismatch repair, chemotherapy-related, temozolomide-related, and somatic hypermutation activity signatures. Comparisons of matched longitudinal and primary phylogenetic trees further revealed that deceased patients and those with recurrent or metastatic disease exhibited significantly greater evolutionary divergence, suggesting that primary tumors are less predictive of complete evolutionary trajectories in poor-outcome cases. Together, these findings suggest that pediatric brain tumor evolution is shaped by cancer type and various mutational processes, and that patients with adverse outcomes follow distinct evolutionary trajectories independently of subtype. Our study provides a population-level view of tumor evolution in pediatric brain cancers and highlights the value of genetic clonal dynamics for understanding tumor etiology and progression.



 Minh A. Nguyen, Pablo G. Cámara. Mapping clonal architecture and evolution in pediatric brain cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d01cdf935c0ff4316725768de123e5059b9b4ae1" target='_blank'>
              Abstract B008: Mapping clonal architecture and evolution in pediatric brain cancers
              </a>
            </td>
          <td>
            Minh A. Nguyen, Pablo G. Cámara
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Receptor tyrosine kinase (RTK) fusions are a large class of oncoproteins found in ∼5% of cancers. Key questions remain, however, about how RTK fusions transmit oncogenic signals, including how these largely cytoplasmic proteins activate downstream pathways that originate at the plasma membrane. Fusions are multimeric and can form mesoscale condensates in cancer cells, and condensation has been implicated as an essential mechanism to enable signal transmission from the cytoplasm. However, whether condensates play a causal role, or whether smaller ‘diffuse’ assemblies are sufficient to transduce signals, has been challenging to establish. Here we apply advanced microscopy, single-cell analysis, and synthetic fusions to determine the principles by which multimerization and condensation drive signaling from cytoplasmic RTK fusions. For EML4-ALK, a prominent fusion that forms condensates, we found poor correlation between condensation and signaling. By contrast, EML4-ALK activity was abundant in the diffuse phase, and the kinetics of diffuse-phase activity aligned more closely with downstream Erk signaling than did kinetics of signaling within condensates. Synthetic RTK fusions showed that cytoplasmic ALK or RET fusion dimers—and even constitutively active monomers—were sufficient to induce strong Ras-Erk signaling despite the absence of condensates, and diffuse fusions were sufficient to transform cells in vitro and in subcutaneous tumor models. A panel of various other cancer-driving RTK fusions showed that low-order multimerization was universal across fusions, whereas mesoscale condensation was rare and did not correlate with signaling. Our results suggest that low-order fusion multimerization is sufficient to drive its phosphorylation, which is necessary and sufficient to trigger downstream oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6563f441f99dac16cec7dceb7a7c5f130e146e" target='_blank'>
              Low-order assemblies drive oncogenic RTK fusion signaling without condensation
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Thomas R. Mumford, Delaney Wilde, Sofia Wissert, Yuzhi (Carol) Gao, Emily Brackhahn, R. Kriwacki, Elizabeth Rhoades, Lukasz J. Bugaj
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fde5c67c0f8ab3015db102bca2f37f2904ea7f9" target='_blank'>
              Role of Myc family proteins in transcriptional regulation of growth and oncogenic transformation in fusion-positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bishwanath Chatterjee, P. K. Raut, Hana Kim, Rachel A. Hoffman, P. Pandey, Shabir Zargar, Wenyue Sun, Benjamin Z. Stanton, Frederic G Barr
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, Gang Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin plays a critical role in regulating gene expression and genomic processes like DNA replication, repair, and genome stability. Although these processes occur at the individual-cell level, most chromatin structure data are derived from population-averaged assays, such as Hi-C, obscuring the heterogeneity of single-cell conformations. To address this limitation, we developed a polymer physics-based modeling framework, the sequential Bayesian inference framework (sBIF), that deconvolutes bulk Hi-C data to reconstruct single-cell 3D chromatin conformations. To support a broader use of sBIF, we created ChromPolymerDB, a publicly accessible, high-resolution database of single-cell chromatin structures inferred by sBIF. The database contains ∼108 reconstructed 5 kb-resolution single cell structures, spanning over 60,000 genomic loci across 50 human cell types and experimental conditions. ChromPolymerDB features an interactive web interface with tools for 3D structural analysis and multi-omics integration. Users can explore associations between chromatin conformation and gene expression, epigenetic modifications, and regulatory elements. The platform also supports comparative analyses to identify structural changes across cell types, developmental stages, or disease contexts. ChromPolymerDB offers a unique resource for researchers studying the relationship between genome architecture and gene regulation, and for advancing comparative 3D genomics. ChromPolymerDB is available online at https://chrompolymerdb.bme.uic.edu/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff3acfa75c8e6fa42bde5e8caa704c202f1fa88" target='_blank'>
              ChromPolymerDB: a high-resolution database of single-cell 3D chromatin structures for functional genomics.
              </a>
            </td>
          <td>
            Min Chen, Lin Du, Siyuan Zhao, Bowei Ye, Pourya Delafrouz, Hammad Farooq, Debaleena Chattopadhyay, G. Marai, Zhifeng Shao, Jie Liang, D. Czajkowsky, Constantinos Chronis
          </td>
          <td>2025-11-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Carcinogenesis in human urothelium is driven by a high burden of mutations caused by the antiviral “APOBEC3” (apolipoprotein B mRNA editing enzyme, catalytic subunit–like 3) cytosine deaminase enzymes; however, there is no established viral etiology. BK polyomavirus (BKPyV) is a ubiquitous childhood infection that persists in the kidney during adulthood and is frequently detected in urine. Chronic BKPyV infections of normal human urothelium induced an innate response, including apical extrusion of infected cells. Local paracrine interferon signaling induced APOBEC3 expression in both infected and juxtaposed bystander cells, leading to acquisition of hallmark APOBEC3-mediated mutational signatures that recapitulated the variation in mutational character found in patients with muscle-invasive bladder cancer. In our model for urothelial carcinogenesis, uninfected bystander cells witnessing BKPyV infection become APOBEC3 damaged, escape extrusion, and acquire hypermutable advantage. “Transmutagenesis” explains how cells proximal to infected neighbors acquire cancer-initiating mutations. This hypothesis for how urothelial cancers can develop as APOBEC3 signature rich, while remaining virus-negative, suggests that a large proportion of urothelial carcinomas may be preventable by antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc0144870a73bc87ab37368eb4e688c7f59b32" target='_blank'>
              Virus-induced APOBEC3 transmutagenesis in bladder cancer initiation
              </a>
            </td>
          <td>
            George H Hatton, Sally James, Andrew S. Mason, Richard T Gawne, Helena Vogel, Karen Hogg, Parisa Boukani, Gemma Swinscoe, Anjum Khan, Matthew Welberry Smith, Michael A Carpenter, Omar Masood, I. Martincorena, Andrew Macdonald, R. Harris, G. Starrett, Jennifer Southgate, Simon C Baker
          </td>
          <td>2025-12-03</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC), an epigenetic modification derived from the oxidation of 5-methylcytosine (5mC) by the ten-eleven translocation (TET) family of dioxygenases, plays a pivotal role in the regulation of gene expression, cellular differentiation, and developmental plasticity. Once considered an intermediate in DNA demethylation, 5hmC is now recognized as a stable and functionally significant epigenetic mark with distinct genomic distributions and significant regulatory implications. This review provides a comprehensive analysis of the biological functions of 5hmC in normal cellular processes, including its role in maintaining tissue-specific gene expression, lineage commitment, and genomic integrity. We also describe its role in cancer, the mechanistic underpinnings of its loss or redistribution in tumor cells, and how these changes contribute to oncogenic signaling pathways, epithelial-mesenchymal transition, and tumor heterogeneity. Furthermore, we explore the utility of 5hmC as a biomarker in cancer diagnostics and prognostics, supported by recent advances in sequencing technologies and cell-free DNA profiling. We also examine the intersection of 5hmC and chemotherapy, highlighting how aberrant 5hmC levels can influence drug resistance and sensitivity, and assess the therapeutic potential of targeting TET enzymes and associated pathways. By integrating insights from basic epigenetics, cancer biology, and therapeutic research, this review underscores the multifaceted role of 5hmC in human malignancies and outlines the translational opportunities for exploiting 5hmC-related mechanisms in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3164793ee65481d7edac6287a5b81d63211bc" target='_blank'>
              5-Hydroxymethylcytosine: a key epigenetic mark in cancer and chemotherapy response
              </a>
            </td>
          <td>
            S. S. Kharat, Shyam K. Sharan
          </td>
          <td>2025-11-17</td>
          <td>Epigenetics & Chromatin</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, T. F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03533d5a535f8136a5f4c27e74c671f6ec467639" target='_blank'>
              cGAS restricts PARP1-mediated microhomology-mediated end joining by suppressing poly-ADP-ribosylation.
              </a>
            </td>
          <td>
            Haibo Yang, Boya Gao, Bo Wu, Wenjing Li, Xueping Zhu, Laiyee Phoon, Jing Luo, Fu Gui, Weixing Zhao, Li Jia, Li Lan
          </td>
          <td>2025-12-17</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism.
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceca0fc9a67ea509cda7d2d2cbdefbe5c21b1e0b" target='_blank'>
              Evidence for Transcription and Horizontal Gene Transfer in Dipteran Germline-Restricted Chromosomes
              </a>
            </td>
          <td>
            Riccardo G. Kyriacou, M. Herbette, Robert B. Baird, Katy M. Monteith, Laura Ross, Christina N. Hodson
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, Allen L Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, Etienne Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 For quite some time, knowledge about mitochondria and the mitochondrial genome has been primarily limited to energy production. However, there is now increasing evidence that they have many important roles in cell function and that synergy between the nuclear and mitochondrial genomes is an essential prerequisite to developmental outcome. This review describes the mitochondrial genome and its contribution to overall cellular genomic content; and discusses mitochondrial DNA (mtDNA) inheritance. mtDNA homoplasmy and heteroplasmy are defined and distinctions between pathogenic and non-pathogenic rearrangements are drawn; how they are transmitted; and their effects on oocyte quality and developmental outcomes. This is followed by analysis of mtDNA replication and changes in mtDNA copy number during development; why they need to happen; and how they influence developmental outcomes. Changes to nuclear DNA methylation events are then discussed in the context of changes to mtDNA replication throughout development. This leads to the concept of ‘genomic balance’, which defines how cells at any stage of development require adjustments to both genomes to ensure successful cellular function and development; and how this process can be perturbed by some of the more invasive assisted reproductive technologies designed to treat infertility and mtDNA disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902806be1c6e8a4fa2af943ea8c04c26b2c9f2b7" target='_blank'>
              The nuclear and mitochondrial genomes need to tango. How is their dance synchronised during development?
              </a>
            </td>
          <td>
            J. S. St. John
          </td>
          <td>2025-12-08</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Anaplastic thyroid cancer is a fast-growing and deadly malignancy. A major reason for its extremely low survival rate is the absence of effective diagnostic, prognostic, and therapeutic tools that have significantly improved outcomes in other cancer types. A long non-coding RNA molecule investigation presents a novel and unique approach for combating anaplastic thyroid cancer and may contribute to finding the answers needed to control and treat this disease. Identifying dysregulated long non-coding RNAs may reveal previously unknown molecular mechanisms of ATC and provide potential diagnostic, prognostic, or therapeutic targets. These molecules often display regulatory capabilities that extend beyond those of proteins or other protein-coding genes. The ability of long non-coding RNAs to fine-tune the genome and control phenotypic output makes them extremely attractive avenues of study and investigation. We found that reducing the expression of a single long non-coding RNA, DUXAP10, significantly diminished multiple cancer-associated phenotypes of anaplastic thyroid cancer. DUXAP10 expression is significantly higher in some patients with anaplastic thyroid cancer, and we developed a cell model that describes the consequences of the anaplastic thyroid cancer proliferative and metastatic cascade when this RNA molecule is overexpressed, and how these cancer-promoting mechanisms can be alleviated by gene-specific targeting and lowering of this RNA molecule’s expression. Abstract Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, X. Li, J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh R. Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, G. Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="In eukaryotes, the mutation rate of the chloroplasts is lower than that of the nuclear genomes. Advances in Next-Generation Sequencing (NGS) and Third-Generation Sequencing (TGS) technologies, together with improvements in genome assembly algorithms, have substantially propelled research in chloroplast genomics. Although nearly 9,000 chloroplast genomes have been released, chloroplast population genetics for specific species remains unexplored. The chloroplast genome possesses a quadripartite structure consisting of a large single-copy (LSC) region, a small single-copy (SSC) region, and two inverted repeats (IRs). A longstanding question is why the maternally inherited chloroplast genome does not appear to suffer from the Muller’s ratchet effect. It has been hypothesized that intramolecular recombination within the chloroplast genome may counteract this effect; however, direct evidence for such recombination remains lacking. We conducted chloroplast population data analysis in hexaploid wheat and its ancestral relatives. One the basis of the pan-genome constructed and phylogeny analysis of chloroplast genomes of all samples, we calculated the chloroplast diversity of hexaploid wheat (π = 0.0001) is the lowest among the three ploidy types tested. Additionally, we found that during the formation of hexaploid wheat, only the chloroplasts from tetraploid wheat were inherited. Moreover, Aegilops tauschii contributed solely as the paternal provider of nuclear genome material. In the chloroplast genome assembly, we assembled IRa (inverted repeat A) and IRb (inverted repeat B), revealing multiple insertion/deletion sequence differences between them. Importantly, we discovered that recombination occurs between the IR regions of the chloroplast genome. Frequent recombination results in two structural configurations existing in nearly equal proportions within a single sample. This phenomenon has led to an increase in the nucleotide diversity of the chloroplast IR region within the wheat population, which was originally low among species in Poaceae. This study demonstrates the feasibility of assembling chloroplast genomes using low-depth whole-genome sequencing (WGS) and confirms that the chloroplast genome of hexaploid wheat originates from tetraploid wheat rather than Aegilops tauschii. Furthermore, we provide evidence of frequent intra-molecular recombination in the chloroplast IR regions, leading to the coexistence of two equimolar inversion isomers. Despite strong purifying selection, recombination increases genetic diversity within the IR regions, facilitating adaptation and maintaining the functional stability of essential genes. Our findings highlight the role of recombination in balancing genetic stability and flexibility in chloroplast genome evolution, offering new insights into nuclear-cytoplasmic interactions and polyploid adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeeb60c185e1a9b6552bcd942f3df2b59678f938" target='_blank'>
              Wheat chloroplast pangenome reveals frequent intramolecular recombination in the inverted repeat regions
              </a>
            </td>
          <td>
            Hua Zhang, Xuebing Qiu, Zhiliang Zhang, Jijin Zhang, You Qing, Yafei Guo, Xinyue Song, Chengzhi Liang, Yanling Sun, Yang Zhao, Changbin Yin, Jing Wang, Fei Lu, Mingsheng Chen
          </td>
          <td>2025-11-27</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Spindle cell lipomas (SCL) and pleomorphic lipomas (PL) are today considered a single tumor entity (SCLPL). Atypical SCLPL, in contrast, represents a recently recognized related but distinct entity. Here we explored the correlation between genomic features and morphological aspects of SCLPL and atypical SCLPL and the role of the RB1 gene in tumor development. Seventy‐one samples from 68 patients with SCLPL or atypical SCLPL, as well as a lipomatous tumor from a retinoblastoma patient with a germline pathogenic variant in the RB1 gene, and two pleomorphic liposarcomas were analyzed using chromosome banding, high‐resolution genomic arrays (SNP array), whole exome sequencing (WES), and/or RNA sequencing (RNA‐seq). Common for all tumors was involvement of 13q; other recurring variants were deletion of 16q, 6q, and 17p. A minimally deleted region that only contained RB1 was found on 13q. A distinction was seen between SCL on the one hand and PL and atypical SCLPL on the other; SCL had fewer copy number aberrations in general, and loss of 17p/TP53 gene or a SNV affecting TP53 was only rarely detected in SCL but seen in the vast majority of PL and atypical SCLPL tumors. Thus, at the molecular level, SCL is different from PL/atypical SCLPL. Furthermore, the finding of the same copy number changes (loss of 13q, 6q, 16q, and 17p) in some pleomorphic liposarcomas raises the possibility that a subset of SCLPL/atypical SCLPL have the potential for malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b1988ef8c66c3a7438406d89e232e1e91d6e09" target='_blank'>
              The Role of RB1 and Secondary Genomic Changes in the Development of Spindle Cell and Pleomorphic Lipomas
              </a>
            </td>
          <td>
            Maria Hellberg, V. Difilippo, Emilia Gottberg, A. Hesla, Felix Haglund de Flon, J. Nilsson, L. Magnusson, Saskia Sydow, Paul Piccinelli, Jakob Hofvander, Fredrik Mertens
          </td>
          <td>2025-12-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transcription factors (TFs) control the rate of transcription of genetic information by binding to specific DNA sequences. The time needed for a TF to find its specific target sites is a bottleneck to the genetic response mechanism. While TF target site search is a well-studied problem, the effect of genome 3D architecture on the TF target search times is poorly understood. Here, we use accurate and cell-specific 3D structural ensembles of human chromosomes to investigate how the spatial organization of binding sites on chromosomes influences the dynamics of TFs. We use Chromatin Immuno-Precipitation data to map the position of binding sites for several TFs on chromosomal structures and simulate the dynamics of individual TF within chromosomal territories. We find that the distribution of binding sites along chromosomes cooperates with the 3D folding of the chromatin fiber to induce dynamics in which TFs tend to visit sites distributed sequentially along the genome. In this way, genome 3D architecture appears to reduce the time each TF spends in the unbound state while commuting from one target site to the other. At the same time, genome 3D architecture further reduces the flux of TFs between binding sites already well separated along the genome, effectively isolating distant clusters of binding sites. We compare the TF traffic patterns generated by the 3D structures of human chromosomes with those generated by several alternative structural models characterized by increasing randomness. Finally, we study the effect of lengthwise compaction and phase separation, known architectural features of the human genome, in TFs target search. In short, our analysis demonstrates that genome architecture regulates the traffic of TFs within chromosomal territories and reduces the time each TF spends commuting between binding sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695099640d117c423978d610f4cad3ca5a3675b3" target='_blank'>
              3D genome architecture regulates the traffic of transcription factors throughout human chromosomes
              </a>
            </td>
          <td>
            Amit Das, Ryan R. Cheng, D. Potoyan, Michele Di Pierro
          </td>
          <td>2023-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The MGT1 gene encodes O⁶-methylguanine-DNA methyltransferase, a highly conserved enzyme that directly reverses alkylation damage by transferring methyl groups from guanine to a catalytic cysteine. While its nuclear function in genome maintenance is well characterized, its contribution to mitochondrial homeostasis remains elusive. Here, we investigated the Δmgt1 mutant of Saccharomyces cerevisiae. Remarkably, the mutant displayed higher colony formation on non-fermentable carbon sources, indicating altered respiratory efficiency. DAPI-based imaging revealed increased and dispersed mitochondrial DNA (mtDNA), with quantitative analysis confirming elevated mtDNA content. Despite this, Δmgt1 cells exhibited a markedly shortened chronological lifespan, suggesting a link between MGT1 deficiency and genomic instability, mitochondrial dysregulation, and accelerated cellular aging. Bioinformatic analyses highlighted interactions with key DNA repair proteins, including Msh6p, Mlh1p, Mre11p, Exo1p, and Ogg1p, suggesting integration into broader genome-stability networks. These findings reveal a previously unrecognized mitochondrial dimension of MGT1 function and raise the intriguing possibility that the plant homolog AtAGT1 may similarly modulate mitochondrial integrity, stress resilience, and lifespan in higher eukaryotes.
Plant Tissue Cult. & Biotech. 35(2): 309-320, 2025 (December)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5be28c7675a0b15b5de29b1f65b36b979febc6" target='_blank'>
              Loss of DNA Repair Methyltransferase Gene MGT1 Alters Mitochondrial Dynamics and Accelerates Aging in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Mahapara Tabassum, Nazela Ahmed Chowdhury, Tahmina Islam, Md Arafat Al Mamun, R. Sarker, R. A. Begum, K. Muid
          </td>
          <td>2026-01-04</td>
          <td>Plant Tissue Culture and Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Genome size varies widely, even among closely related species, yet much less is known about chromosome size variation. Here we use the fourth chromosome of Drosophila, also known as the “Muller F element” or “dot chromosome”, as a model to investigate chromosome-specific size expansion. The F element of most Drosophila species is small (∼1.3 Mb) and almost entirely heterochromatic, yet harbors approximately 80 protein-coding genes. Here, we study D. kikkawai, D. takahashii, D. ananassae, and D. bipectinata, whose F elements are 2- to 15-fold larger in size compared to D. melanogaster. Through manual gene curation and comparative genomic analysis, we find that their F elements have expanded primarily via accumulation of transposable elements (TEs) in introns and intergenic regions. Natural selection appears less efficient on these expanded F elements: they have smaller effective population sizes and their genes exhibit reduced usage of optimal codons, compared to D. melanogaster. We propose that F element size variation is driven by differences in F element recombination rates. The ultra-long (∼20 Mb) F elements of D. ananassae and D. bipectinata display high rates of rearrangement and sequence evolution and exhibit independent TE-driven expansions. Our results suggest that F elements of most Drosophila species likely recombine enough to prevent size expansion, while F element recombination in D. ananassae and D. bipectinata is either absent or rare enough to allow TEs and other deleterious mutations to accumulate via Muller's ratchet; thus, these chromosomes evolve more like a Y chromosome than a typical Drosophila F element.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85dd457d5df59c40e0913145a9a7cc25ff82fdf" target='_blank'>
              Recombination Suppression Drives Expansion of the Drosophila Dot Chromosome
              </a>
            </td>
          <td>
            Timothy J Stanek, Wilson Leung, Christopher D. Shaffer, Ishtar Olaveja, Annabelle Laughlin, Jaquelyn Hester, Darwin Garrido, Emily K Oh, Maria Volski, Nistha Panda, Mia Mo, Ethan Cordes, Martin Dalling, Kacie Kershaw, Malcolm Arnott, Stephen Daly, Silvia Garcia Valenzuela, Paige Thompson, Kayla L Hastert, Destiny Sabb, Kathryn Karpinski, Meher Naaz Arora, Nicholas Rius, Larissa LoBello, Sebastian Jaramillo, Omkar Sonavane, Alice Herrmann, Laura K. Reed, Sarah C. R. Elgin, C. Arrigo, Christopher E. Ellison
          </td>
          <td>2025-11-28</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B. Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Retinoblastoma is the most common intraocular malignancy of childhood, yet its genomic landscape remains incompletely defined, particularly in understudied populations. Beyond RB1 loss, the contribution of additional somatic and germline alterations to disease heterogeneity and clinical behavior is unclear.



 We performed whole-exome sequencing of 166 retinoblastoma samples from 166 patients with matched germline DNA, representing the largest cohort analyzed to date. Clinical data were available for 160 patients. Variant calling, copy number alteration (CNA) profiling, and integrative analyses were performed to characterize genetic drivers and their associations with clinical features.



 Pathogenic RB1 variants were identified in 120 patients, and MYCN amplification in six patients. Additional recurrent alterations involved BCOR, CCND3, ERBB2, and PDGFRB. Copy number gains of 6p (41.3%) and 17q (8.1%) were significantly associated with high-risk features including rubeosis, subretinal seeding, and tumor extension beyond the lamina cribrosa. Germline ERBB2 variants correlated with orbital invasion, while germline PDGFRB variants were associated with second primary cancers. Together, these findings underscore the genetic heterogeneity of retinoblastoma and reveal novel genotype–phenotype correlations.



 This study provides the most comprehensive genomic characterization of retinoblastoma to date, expands the known mutational spectrum, and identifies biomarkers with direct clinical relevance. These insights have potential to refine risk stratification, inform precision therapeutic strategies, and improve long-term outcomes for children with retinoblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca1a82a307761a6281f02514b112932e6b6ae23" target='_blank'>
              Genomic landscape of retinoblastoma: Insights into risk stratification and precision pediatric Neuro-Oncology
              </a>
            </td>
          <td>
            A. Maktabi, Yang Liu, Saleh A. Almesfer, M. Abdelhakim, Hind Aldakhil, Maxat Kulmanov, Deepak P Edward, R. Hoehndorf, M. Abedalthagafi
          </td>
          <td>2025-12-12</td>
          <td>Neuro-Oncology Pediatrics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kaposi sarcoma‐associated herpesvirus (KSHV), a gamma‐herpesvirus, is the main etiological agent of several tumors, including Kaposi's sarcoma (KS) and primary effusion lymphoma (PEL). The large, double‐stranded viral genome of KSHV is maintained as a latent episome in the nucleus of host cells, where a small subset of viral genes is expressed that facilitate evasion of immune responses and promotion of cell survival and proliferation supporting tumorigenesis. The major latency‐associated nuclear antigen (LANA) is an essential viral factor that is required for genome replication and segregation to maintain viral genomes within dividing cells. Given the essential role of LANA to maintain viral latency, efforts have focused on targeting LANA's role in replication and segregation as a mechanism to overcome latency and tumorigenesis. However, given the complexity in small molecule targeting to DNA‐binding domains, current efforts focused on drugging LANA continue to reach potency milestones for clinical trials. Here, we developed a HaloTag‐based PROTAC model as a proof‐of‐concept for targeted LANA protein degradation. Our findings highlight the power of a PROTAC‐based strategy on eliminating LANA and viral persistence in tissue culture demonstrating promise in the on‐going challenge for targeting this major viral factor in KSHV latency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfa79b63dd2067eec7b314131de284f9e1fe9922" target='_blank'>
              Targeted Protein Degradation of the Latency‐Associated Nuclear Antigen Evicts Kaposi Sarcoma‐Associated Herpesvirus Episomes From Infected Cells In Vitro
              </a>
            </td>
          <td>
            Sarah McMahon, Maha Amer, Brooke Hall, Ritu Shekhar, Rolf Renne
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements (TEs), commonly referred to as “jumping genes,” are discrete DNA fragments capable of mobilization within the genome, often creating new copies during transposition. Once considered non-functional “junk DNA,” TEs are now recognized as essential regulators of genome evolution, gene networks, and epigenetic processes. They contribute to genome innovation through gene duplication, exon shuffling, regulatory rewiring, and stress-responsive plasticity. Conversely, their uncontrolled activation can disrupt genetic integrity. Such aberrant TE activity has been associated with immune dysfunction, cancer, neurodegeneration, and reproductive disorders. 
In livestock, recent studies have identified TE-derived regulatory elements that influence growth, reproduction, immunity, and adaptation. Notably, TEs constitute approximately 45–60% of the genomes of major livestock species, including pigs, sheep, and cattle, highlighting their extensive regulatory impact. Advances in genome editing and artificial intelligence have facilitated their functional characterization. This allows a sharper focus on TE-driven regulation in animal genomes without repeatedly specifying the livestock context. 
Emerging technologies such as CRISPR–transposon systems, programmable epigenome editing, and AI-driven TE mapping offer new opportunities for precise genome engineering, functional genomics, and livestock breeding. This review consolidates current knowledge on TE classification, molecular mechanisms, evolutionary roles, disease associations, and biotechnological  applications, with emphasis on livestock genomics. It also outlines existing challenges, biosafety considerations, and future directions toward harnessing TEs for precision animal breeding and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434f9971fc3a63cead1542771f75e88db7b4209d" target='_blank'>
              Jumping Genes (Transposable Elements): A Comprehensive Review of Their Molecular Mechanisms, Evolutionary Impact and Applications in Livestock and Biomedicine
              </a>
            </td>
          <td>
            Dhruv Bhatia, Prakashji Thakor, Pankti J Naik, Brijal Barot
          </td>
          <td>2025-12-22</td>
          <td>Journal of Experimental Agriculture International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints.
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Copy number aberrations (CNAs) are critical drivers of genomic diversity in oncology, where recurrent CNAs frequently underlie tumorigenesis. However, existing public resources are limited in their somatic CNA specificity, breadth across multiple data modalities, and support for recurrent CNAs with online functional annotation and interactive visualization. Here, we present CNAScope (https://cna.fengslab.com/), a database that curates and functionally annotates over 3 954 361 CNA profiles and 3 946 319 metadata from 810 datasets, 174 464 samples, 3 018 672 single cells, and 764 232 spatial cells/spots, spanning 77 cancer subtypes from eight data sources and 55 cancer initiatives and institutions. CNAScope offers downloadable CNA annotations and interactive visualizations at bin, gene, and pathway term levels, including phylogenetic inference, clustering, dimension reduction, and focal/consensus CNA detection. Users can explore data through interactive heatmaps, phylogenetic trees, embedding plots, CN charts, and focal/consensus plots, or upload and annotate their own CNAs in real time. In all, with its large curated data volume and rich annotation capabilities, CNAScope serves as a vital resource for accelerating cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997b8f03647982aaf3c88867cb53c56e4c242e6b" target='_blank'>
              CNAScope: pan-cancer copy number aberration database with functional annotation and interactive visualization.
              </a>
            </td>
          <td>
            Xikang Feng, Jieyi Zheng, Sisi Peng, Anna Jiang, Ka Ho Ng, Chengshang Lyu, Qiangguo Jin, Lingxi Chen
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="INTRODUCTION
The transposase protein, InsAB', is the key cytoplastic determinant that catalyzes IS1 movement within a single DNA molecule or between two DNA molecules. Regulation of InsAB' synthesis, its structural characteristics, and its potent cis effect on IS1 transposition have been subjects of thorough investigation. However, limited efforts have been devoted to measuring the levels of IS1 transcripts and transposases made by native IS1 elements and examining their trans-acting function, particularly concerning their role in transposing otherwise immobile IS1 elements and other target DNA segments within the bacterial genome.


METHODS
All constructs were made on the E. coli chromosome using the lambda-Red system. IS1 transposition frequencies were determined using Bgl+ mutation assays, colony PCR and DNA sequencing. IS1 transcripts and transposases were quantitated using lacZ transcriptional and translational reporters.


RESULTS
We first confirm that a native IS1 element, IS1E, can transpose at a dramatically elevated rate in the absence of both transcriptional and post-transcriptional regulations. Using lacZ reporters targeting the transposase gene insAB', we reveal that the InsA repressor moderately reduces insAB' transcription by as much as 16.8 fold. However, the combined action of InsA and ribosomal frameshifting leads to a much stronger reduction in InsAB' production, by up to 735 fold. We show that only a small percentage (roughly 1.6%) of IS1 transcripts are successfully translated into InsAB', highlighting the primary role of post-transcriptional regulation over transcriptional repression in governing IS1 transposition. We also demonstrate, for the first time, that high levels of InsAB' exhibit a strong trans effect, being capable of efficiently transposing not only normally non-mobile IS1 elements but also a larger miniIS1 cassette within the genome, with greater amounts of InsAB' leading to higher frequencies of transposition.


CONCLUSIONS
By dissecting and quantitating the roles of transcriptional and translational controls in transposase production, our study reveals that post-transcriptional regulation via ribosomal frameshifting is the central mechanism governing IS1 intragenomic transposition. Our findings demonstrate that IS1 transposases function in trans, mediating the movement of DNA segments specifically bordered by its IRL and IRR sequences. These findings advance our comprehension of how IS elements mobilize within bacterial genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc63253e7e508e0ba8aecec525150bf22bacf59e" target='_blank'>
              Investigating the Trans Effects of IS1 Transposases on Intragenomic DNA Movements in Escherichia coli.
              </a>
            </td>
          <td>
            J. Ord, Zhongge Zhang, Janie Wu, Sofia Smith, Zhenhao Feng, Keisha Ancheta, Luis Fernando Garrido, Ge Yao, Sahil Gupta, Ethan T Lee, Milton H. Saier Jr.
          </td>
          <td>2025-11-25</td>
          <td>Microbial physiology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Low-grade diffusely infiltrative tumor, SMARCB1-mutant (LGDIT), is a rare brain tumor with pathologically low-grade features defined by the infiltrative growth of SMARCB1-deficient tumor cells. Although LGDIT generally exhibits indolent growth, some cases contain high-grade components that are histopathologically consistent with atypical teratoid/rhabdoid (ATRT). However, little is yet known about the molecular characteristics distinguishing LGDIT from ATRT. To characterize the molecular profile of LGDIT, we performed whole-genome sequencing (WGS), RNA-seq, DNA methylation array, and single-nucleus RNA-seq on four cases with LGDIT and two cases with ATRT-MYC subtype. Additionally, we analyzed publicly available ATRT cases, including 15 WGS, 43 RNA-seq, and 112 DNA methylation arrays. All four LGDIT cases exhibited homozygous deletions of the SMARCB1 gene, underscoring its fundamental role in the tumorigenesis of LGDIT. The median mutation burden was 0.96/Mb, yet higher than the 0.57/Mb in ATRT, possibly reflecting the advanced age of onset. Structural variants in LGDIT were rare except for the SMARCB1 locus, indicating a genomically stable tumor. Clustering analyses based on gene expression and DNA methylation profiles consistently clustered LGDIT alongside ATRT-MYC. However, gene set enrichment analysis revealed that LGDIT exhibits relatively reduced expression of cell cycle and protein translational activity, consistent with its lower cell proliferation. snRNA-seq demonstrated that LGDIT comprised both tumor cell populations with gene expression profiles similar to ATRT-MYC and populations exhibiting distinct expression signatures, suggesting that LGDIT and ATRT-MYC represent closely related entities within the identical neural differentiation trajectory. In conclusion, LGDIT is characterized by SMARCB1 loss, genomic stability, and low proliferation, with molecular features overlapping those of ATRT-MYC while maintaining distinctive biological properties. These findings support LGDIT as a distinct, yet related tumor entity within the spectrum of SMARCB1-deficient neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c68ef755c06bcf4c683429d0cd3cf1e4dbf4bdf" target='_blank'>
              10214-GEN-5 A multi-omics characterization of low-grade diffusely infiltrative tumor, SMARCB1-mutant
              </a>
            </td>
          <td>
            Hirokazu Sugino, Takuma Nakashima, Tsubasa Miyauchi, Joji Ishida, K. Tateishi, Atsufumi Kawamura, Genshin Mouri, Junko Hirato, Takako Yoshioka, Koichi Ichimura, Shinya Tanaka, Y. Narita, S. Nobusawa, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541ea2c99c79b880da284a2c3416b1647546a060" target='_blank'>
              A Genome-Wide Genetic Screen Identifies a Novel kDNA Replication Protein in Trypanosomes
              </a>
            </td>
          <td>
            M. Miskinyte, Clirim Jetishi, A. Kalichava, A. Ivens, Martin Waterfall, Matt Gould, Lucy Glover, David Horn, T. Ochsenreiter, A. Schnaufer
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer. Krepelova and colleagues identify an aging- and colon cancer-associated DNA methylation drift in healthy and cancerous human colon samples, echoed in the mouse intestinal epithelium. They show it expands via intestinal cell clonality and is driven by dysregulated iron metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retrovirus K (HERV-K) proviruses of the HML-2 subgroup are the most recently integrated retroviral elements within the human genome. The HERV-K Rec protein, a functional homolog of HIV Rev, is essential for the expression of viral mRNAs with retained introns. However, their diversity and functional capacities have remained largely unexplored. We identified Rec coding sequences in the human genome and selected intact Rec proteins from 28 proviral loci for functional characterization. Using a dual-color fluorescent reporter containing the HERV-K Rec-response element and a complementary enzyme-linked immunosorbent assay, we found that Rec proteins from nine proviral genomic loci promoted function at the post-transcriptional level. All but one of these proviruses are insertionally polymorphic in the human genome. In addition, several of the non-functional Rec proteins were trans-dominant negative. Detailed mutational analysis of the most potent inhibitory variant, encoded by the HERV-K provirus 12q14.1, displayed only two amino acid changes (N2H and E34del) relative to the prototypical functional Rec protein encoded by the reconstructed consensus HERV-K. Insertion of a glutamic acid at position 34 in this mutant fully restored the functional activity. Our results unveil an unexpected complexity in HERV-K (HML-2) post-transcriptional regulation, with a few Rec variants showing function, while others are trans-dominant negative. These findings significantly expand our understanding of the various Rec proteins that can be expressed in human cells and how they function at the post-transcriptional level. IMPORTANCE In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects. In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42022bba88ac63086a6e5b60150a1271cd5595a" target='_blank'>
              Characterization of HERV-K (HML-2) Rec proteins encoded in the human genome and their post-transcriptional function
              </a>
            </td>
          <td>
            Katarzyna Zurowska, Godfrey Dzhivhuho, David Grabski, D. Rekosh, M. Hammarskjold
          </td>
          <td>2025-11-18</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract Z-DNA is a left-handed DNA conformation linked to gene regulation, chromatin dynamics, and immunity. Despite its importance, genome-wide prediction of Z-DNA forming sequences (ZFS) remains limited by the absence of fast and accessible tools. Here, we present Z-DNA Hunter, a user-friendly web server for genome-scale ZFS prediction utility. The algorithm employs a pattern-based approach optimized for canonical motifs such as (GC)n and (CA)n repeats, with adjustable parameters for detection stringency. Compared with existing methods, Z-DNA Hunter achieves similar or higher accuracy while reducing runtime from hours to seconds, making large-scale analyses feasible. To demonstrate its application, we analyzed the Drosophila melanogaster genome and uncovered a pronounced enrichment of long ZFS on the X chromosome, contrasting with their near absence on the satellite repeat- and transposable element-rich Y chromosome. These findings illustrate both the scalability of Z-DNA Hunter and its potential to reveal biologically meaningful patterns of non-B-DNA. The tool provides direct visualization and export options (e.g. BedGraph for UCSC Genome Browser) and is freely available at https://bioinformatics.ibp.cz/#/analyse/zdna.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782d54fb0da840e7a91c37526570fd49fc670b36" target='_blank'>
              Z-DNA Hunter tool for straightforward detection of Z-DNA forming regions and a case study in Drosophila
              </a>
            </td>
          <td>
            Michal Petrovič, M. Bartas, Alistair N Garratt, P. Pečinka, Michaela Dobrovolná, Klára Koňaříková, O. Trenz, Václav Brázda, Jiří Šťastný
          </td>
          <td>2025-10-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Chromatin topological structure is critical for gene regulation. Hi-C based experiments have significantly advanced our understanding chromatin organization. Numerous computational tools have been developed to identify various structural levels of chromatin, ranging from compartments to loops. However, there remains a lack of specialized tools for identifying non-homologous inter-chromatin contacts (NHCCs), which play important roles in chromosome territories. In this study, we present iceDP, a tool that leverages the Density Peaks clustering algorithm to identify local high-density regions within inter-chromatin. These regions undergo two subsequent filtering steps to eliminate obvious false positives. When applied to three Hi-C datasets, iceDP accurately identified known NHCCs, including olfactory receptor genes in mature olfactory sensory neurons and Polycomb repressive complex-regulated developmental genes in mouse embryonic stem cells (mESCs). Notably, iceDP also uncovered previously unreported transcriptionally active NHCCs. Compared to diffHiC and FitHiC, iceDP exhibited superior performance with the highest positive rate. Moreover, iceDP is compatible with a wide range of chromatin conformation capture techniques, including in-situ Hi-C, Micro-C, HiChIP, and BL-HiC, demonstrating its versatility and utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028ec9a0781ece9ec91130ee7fd06c68f9b569e8" target='_blank'>
              iceDP: identifying inter-chromatin engagement via density peaks clustering algorithm
              </a>
            </td>
          <td>
            Ruhai Chen, Jiekai Chen, Lingling Shi, Jiangping He
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3e6f63cb8217d7c8dbbb34f41d546581037b401" target='_blank'>
              The haplotype-resolved and near telomere-to-telomere genome assembly for Hypsebarbus vernayi.
              </a>
            </td>
          <td>
            Zhibang Wang, Yu Gao, Chuanshuai Xie, Luohao Xu, Lingfu Kong
          </td>
          <td>2025-12-01</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Lipomas are benign tumors of adipocytic origin. The most frequent chromosome rearrangement in lipomas involves the high-mobility group AT-hook 2 gene (
 HMGA2
 ), which maps to 12q14. We investigated two lipomas showing rearrangements of chromosomal bands 8q22 and 12q14 in their karyotypes, with the aim of identifying the gene products affected by these aberrations.





 The two lipomas were selected because their karyotype showed an 8;12-rearrangement with the very same breakpoint position. The cases were investigated using RNA sequencing, reverse transcription polymerase chain reaction (RT-PCR), Sanger sequencing techniques, Fluorescence
 in situ
 hybridization (FISH), and array comparative genomic hybridization (aCGH).





 RNA sequencing and RT-PCR of the two lipomas showed the presence of the
 HMGA2::CIBAR1-DT
 chimera.





 The
 HMGA2::CIBAR1-DT
 fusion, identified here for the first time, is a recurrent transcript in lipomas.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35b7cf2f12e853b8a43f5d8374c50a16fbddd11" target='_blank'>
              Identification of the HMGA2::CIBAR1-DT fusion transcript in two lipomas with chromosomal rearrangements involving chromosomes 8 and 12
              </a>
            </td>
          <td>
            M. Brunetti, K. Andersen, I. Lobmaier, F. Micci
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb3bc3b62a546392f132deef63cdce1ec580a2a" target='_blank'>
              PSMD12 Overexpression Promotes Lung Adenocarcinoma Progression via Ubiquitin-Proteasome Pathway Dysregulation.
              </a>
            </td>
          <td>
            Yuya Ono, Hajime Otsu, Takaaki Masuda, K. Kosai, Shohei Shibuta, Kosuke Hirose, Takashi Ofuchi, Yuki Ando, Koto Kawata, Yasuo Tsuda, Yusuke Yonemura, Taro Tobo, T. Takenaka, T. Yoshizumi, Koshi Mimori
          </td>
          <td>2026-01-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d1b81a23649db090e91d0538d0fc5a13f45fdb" target='_blank'>
              STK11 mutations and deletions define a distinct subtype of cervical adenocarcinoma
              </a>
            </td>
          <td>
            E. Robinson, E. Murphy, A. Albanez, S. Tulsyan, H. Lou, Y. Xie, W. Zhou, Q. Yang, T. Joe, T. Zhang, J. Liu, W. Luo, D. Wu, H. Higson, M. Viard, M. Carrington, M. Oberley, D. R. Stewart, H. Morales, L. Mirabello, E. Alvirez, R. Orozco, E. Gharzouzi, M. Dean
          </td>
          <td>2026-01-04</td>
          <td>None</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Identifying viruses in tumor transcriptome helps to unravel the potential role of viruses in oncogenesis and tumor progression. Most of the current tools for virus identification in RNA-Seq data rely on sequence alignment, whose performance is constrained by fast mutations, large divergence, and the incompleteness of viral genomes. In this study, we develop ViTrace to detect viral sequences in human transcriptomic data by a hybrid language representation learning model, which integrates DNA contexts, position relationships and amino acid coding information. Although ViTrace is only trained on 13 species from 7 genera, it achieves 86.39% recall in 1179 absent-in-train virus strains of 935 species belonging to 167 genera across 10 phyla. Applied to single-cell RNA-seq data from esophageal and oropharyngeal squamous cell carcinomas, the model reveals tumor-, cell-, and patient-specific viral colonization patterns, uncovering both known and previously unreported viruses. Overall, ViTrace provides a scalable framework for guiding precision oncology and facilitating future discoveries of previously uncharacterized tumor-associated viruses. Hybrid language model ViTrace identifies viral sequences from human transcriptomes, revealing tumor-, cell-, and patient-specific viral colonization patterns and uncovering novel tumor-associated viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235fa47ea6e35bee83f6f73e3fb62ee13668fc7d" target='_blank'>
              ViTrace detects viral signatures in tumor transcriptomes using a hybrid language model
              </a>
            </td>
          <td>
            Feng Zhou, Yushuang He, Fan Yang, Jin Gu, Fengzhu Sun, Shunzhi Zhu, Xiaobing Huang, Ying Wang
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) is a powerful isothermal strategy for nucleic acid detection, but its linear kinetics and dependence on externally supplied primers limit its sensitivity and programmability. Here, we report an exponential RCA (E-RCA) platform that integrates primer regeneration and signal amplification into a single DNA-encoded system. The design uses a circular DNA template encoding the I-R3 self-cleaving DNAzyme sequence; during RCA, tandem I-R3 units are generated within the DNA amplicons, which then catalyze site-specific cleavage to release new primers. This self-sustained (RCA ↔ DNAzyme) amplification circuit enables robust exponential signal growth using only a circular DNA probe and a DNA polymerase without requiring external primers or protein enzymes. We elucidate the mechanism through biochemical experiments and kinetic modeling and validate the system in multiplexed intracellular microRNA imaging and quantitative dual-marker profiling of clinical breast cancer tissues. The E-RCA strategy achieved high diagnostic accuracy (AUC = 0.914; specificity = 100%; sensitivity = 81.3%), demonstrating its potential for sensitive, programmable, and autonomous molecular analysis in both biological and clinical contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779a0bfc06573a96632b1849e24d829296851cc1" target='_blank'>
              Autonomous Exponential Amplification via Rolling Circle-DNAzyme Feedback Programming.
              </a>
            </td>
          <td>
            Jinhua Shang, Mengdi Yu, Yifei Wang, Shanshan Yu, Xianzheng Zhang, Fuan Wang, Yingfu Li
          </td>
          <td>2025-11-26</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Trichomonas vaginalis is a common sexually transmitted parasite that colonizes the human urogenital tract, causing infections that range from asymptomatic to highly inflammatory. As an extracellular pathogen, adherence to host epithelial cells is an important step to colonize the human host. Hence, understanding how the process of attachment to host cells is regulated remains an important goal in parasitology research and human health. T. vaginalis-host interaction is regulated by changes in gene expression, but it is still largely unknown how these changes in transcriptional profiles are controlled, as very few transcriptional regulatory elements have been described. Our recent work highlighted the importance of epigenetics in the regulation of transcription, and a specific role for N6-methyladenine (6mA) in modulating three-dimensional chromatin structure has been suggested. Building on these findings, we analyzed here the role of 6mA and chromatin accessibility during the process of host-parasite interaction by integrating MeDIP-seq and assay for transposase-accessible chromatin sequencing data with RNA-seq in free vs host cell-attached parasites. Consistent with our previous results, we identified transcriptionally active and repressive regions flanked by 6mA modifications, observed both in the presence and absence of host cells. Importantly, we detected differentially accessible chromatin regions that influence the gene expression of key pathogenesis-related genes during T. vaginalis host cell interaction. These findings highlight the importance of chromatin architecture in regulating gene expression during parasitic infection. IMPORTANCE Trichomonas vaginalis, the most common non-viral sexually transmitted parasite, relies on adherence to host epithelial cells to establish infection. Our previous work highlighted the importance of N6-methyladenine (6mA) DNA methylation in the regulation of transcription and three-dimensional chromatin structure. Now, our study integrates RNA-seq, MeDIP-seq, and assay for transposase-accessible chromatin sequencing data to reveal how 6mA and chromatin accessibility modulate gene expression during T. vaginalis interaction with human host cells. We identified over 3,600 differentially expressed genes upon parasite contact with prostate cells, including pathogenesis-related genes. Moreover, we identified transcriptionally active and repressive regions flanked by 6mA that remain largely stable during the process of host interaction. We mapped genome-wide chromatin accessibility and uncovered differentially accessible regions upon host cell contact associated with a subset of genes involved in adhesion. These results suggest that local chromatin accessibility has a major role in modulating gene expression of key virulence genes during host interaction. Trichomonas vaginalis, the most common non-viral sexually transmitted parasite, relies on adherence to host epithelial cells to establish infection. Our previous work highlighted the importance of N6-methyladenine (6mA) DNA methylation in the regulation of transcription and three-dimensional chromatin structure. Now, our study integrates RNA-seq, MeDIP-seq, and assay for transposase-accessible chromatin sequencing data to reveal how 6mA and chromatin accessibility modulate gene expression during T. vaginalis interaction with human host cells. We identified over 3,600 differentially expressed genes upon parasite contact with prostate cells, including pathogenesis-related genes. Moreover, we identified transcriptionally active and repressive regions flanked by 6mA that remain largely stable during the process of host interaction. We mapped genome-wide chromatin accessibility and uncovered differentially accessible regions upon host cell contact associated with a subset of genes involved in adhesion. These results suggest that local chromatin accessibility has a major role in modulating gene expression of key virulence genes during host interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a18326734a8af0f8385e13e0eb222d101b7528" target='_blank'>
              Influence of DNA methylation and chromatin accessibility on regulation of gene expression during Trichomonas vaginalis-host cell interaction
              </a>
            </td>
          <td>
            Daniela Muñoz, Ayelén Lizarraga, Patricia J. Johnson, P. Strobl-Mazzulla, N. de Miguel
          </td>
          <td>2025-12-03</td>
          <td>mBio</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6bb7c74dc3226bfd915b13024e794addeb2154" target='_blank'>
              Diversity and distribution of the subtelomeric Y’ elements across Saccharomyces cerevisiae strains
              </a>
            </td>
          <td>
            Liébaut Dudragne, Juliana Silva Bernardes, Zhou Xu
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Apurinic/apyrimidinic endonucleases - APE1 and APE2 are central to genome maintenance and the cellular DNA damage response, with expanding relevance in cancer biology. APE1 is the primary endonuclease in base excision repair and functions as a redox coactivator of transcription factors. In contrast, APE2 exhibits PCNA dependent 3′–5′ exonuclease and 3′-phosphodiesterase activities, contributing to microhomology-mediated end joining, ATR-Chk1 activation, and immunoglobulin diversification. Both enzymes are often deregulated in cancer: APE1 is frequently overexpressed, drives tumor progression and chemoresistance, while APE2 is similarly upregulated in multiple malignancies. APE1 can be targeted by redox-specific or endonuclease inhibitors, with early clinical evidence of biological activity and tolerability. Although APE2-specific inhibitors remain in early development, emerging synthetic lethality data and preclinical studies highlight APE2 as a novel clinical target in breast cancer type 1/2 susceptibility (BRCA)-mutated cancers. This review discusses the structural and functional roles of APE1 and APE2, their contributions to cancer biology and therapeutics, recent advances in inhibitor development, and future strategies for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42492162323d747cbf3ed4c8e187621438e9d88" target='_blank'>
              Contrasting roles of APE1 and APE2 in genome maintenance, cancer development, and therapeutic targeting
              </a>
            </td>
          <td>
            Aman Sharma, Helen E. Grimsley, Katharine Courtemanche, Simon N. Powell
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma.
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a996b545a09908ed34cc10307fee3ac53ae5d526" target='_blank'>
              The SWI/SNF complex mediated chromatin remodeling promotes hepatitis B virus cccDNA transcription
              </a>
            </td>
          <td>
            Xiaoxue Yuan, Wenqian Geng, Jiyin Wang, Chaoyang Xiong, Yue Wu, Yang Wang, Ronghua Jin, Xi Wang
          </td>
          <td>2025-11-20</td>
          <td>Science China Life Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome.
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="Dysregulated transcription is a defining hallmark of cancer. Recently, novel chemically induced proximity approaches have enabled the rewiring of transcriptional machinery to drive expression of pro-apoptotic genes using bivalent small molecules. In this work, we demonstrate that this strategy is amenable to relocalizing DNA bound transcriptional machinery, such as fusion transcription factors that commonly drive pediatric malignancies. Targeting fusion transcription factors, such as EWSR1::FLI1 in Ewing sarcoma, with these bivalent compounds may open new therapeutic avenues. Here, we develop a small molecule, EB-TCIP, that recruits FKBP12F36V-tagged EWSR1::FLI1 to DNA sites bound by the transcriptional regulator BCL6, leading to rapid chromatin remodeling and expression of BCL6 target genes. This proof-of-concept study demonstrates that DNA binding proteins with pioneering transcription factor activity, such as EWSR1::FLI1, can be relocalized on chromatin to induce expression of repressed genes. Insights herein will guide the development of future bivalent molecules that rewire DNA binding transcriptional machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f7b6214e9ef05ce0f8a706c26c1ffab5a52705" target='_blank'>
              Rewiring the Fusion Oncoprotein EWSR1::FLI1 in Ewing Sarcoma with Bivalent Small Molecules.
              </a>
            </td>
          <td>
            Michael J Bond, Ryan P. Golden, G. Digiovanni, Briana Howard, Roman C. Sarott, Basel Karim, S. Gourisankar, G. Alexe, Kenneth Ross, Hannah M Jones, Brendan G. Dwyer, N. Gray, K. Stegmaier
          </td>
          <td>2025-11-27</td>
          <td>Journal of the American Chemical Society</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c276e2f22ad6897e1f3b914ccbc31d7d9f5eea" target='_blank'>
              Uncovering High-Resolution Organization of Genomic Loci using Experimentally Informed Polymer Model
              </a>
            </td>
          <td>
            Rahul Mittal, D. Heermann, A. Bhattacherjee
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Sequestosome 1 (p62/SQSTM1) is a multifunctional adaptor protein whose dysregulation promotes tumorigenesis through autophagy, metabolic reprogramming, and immune modulation. However, its role across cancer types and impact on the tumor immune microenvironment remain poorly defined. Here, we performed a comprehensive pan-cancer analysis to delineate the molecular and immunological landscape of p62 across human malignancies. TCGA analysis revealed rare mutations and limited prognostic impact of genomic alterations but marked transcriptomic upregulation in LIHC, LUAD, BRCA, KIRP, KICH, KIRC, and READ, correlating with poor survival, advanced stage, and higher T classification, particularly in BRCA and LUAD. Pathway analysis showed strong positive associations between p62 and metabolic adaptation and stress tolerance pathways, including oxidative phosphorylation, reactive oxygen species, and DNA repair, in most cancers. Notably, high p62 expression inversely correlated with immune cell infiltration in most epithelial cancers, such as BRCA, COAD, ESCA, HNSC, KIRC, LIHC, LUAD, LUSC, PAAD, PRAD, READ, THCA, while coinciding with elevated expression of immunosuppressive checkpoints such as PD-L1, B7-H3, EBAG9, PVR, and TGFB1, supporting a link between p62-driven metabolic remodeling and an immune-excluded tumor microenvironment. In contrast, GBM, LGG, OV, SARC, and TGCT showed positive correlations between p62 and immunoscore, with enrichment of interferon and pro-inflammatory pathways, reflecting a distinct immune-activated phenotype. Collectively, these findings identify p62 as a central regulator of tumor metabolism and immunity, suggesting that its context-dependent activity may dictate the balance between immune suppression and activation across cancers, and highlight two natural-product-derived PB1 inhibitors (ZINC70669789 and ZINC08877690) as promising candidates for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716e845c24cbd3bb41e4b234ab94e5c67dc1b5b3" target='_blank'>
              Comprehensive pan-cancer analysis of p62 reveals its contribution to shaping tumor microenvironment and anti-tumor immunity
              </a>
            </td>
          <td>
            Zahra Nayeri, Elahe Tavakol, M. Rahmati, Marco Cordani, M. Djavaheri-Mergny, Vahid Shariati, M. Moosavi
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The genus Acer (maples) is widely distributed and holds significant ecological and economic value, with over 200 species, including Acer truncatum, an important member known for its ornamental features, environmental stress tolerance, and high content of unsaturated fatty acids and nervonic acid in its seeds. Despite the genus’s importance, a high-completeness genome assembly has been lacking. In this study, we present a near-complete telomere-to-telomere (T2T) genome assembly of A. truncatum using advanced sequencing technologies, including PacBio HiFi long reads, ONT ultra-long reads, and Hi-C chromatin conformation capture. The resulting 1.2 Gb haplotype-resolved genome contains 58,569 protein-coding genes and shows a 4.8-fold improvement in contiguity over previous assemblies, with a scaffold N50 of 44 Mb. This high-quality genome provides a foundational resource for investigating the evolutionary history, functional genomics, and molecular breeding of A. truncatum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618994bd9618f76569fe586c53cfb665fec645f8" target='_blank'>
              A near-complete reassembled haplotype-resolved reference genome of Acer truncatum
              </a>
            </td>
          <td>
            Longxin Wang, Zhi-Wei Wang, Hai-Yao Ma, Zi-Meng Sun, Rengang Zhang, Xiaochun Qin, Kai-Hua Jia, Dan Liu
          </td>
          <td>2025-12-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Epstein-Barr virus nuclear antigen 1 (EBNA1) is essential for the replication and stable maintenance of the viral episome in infected cells. Here, we identify the SUMO E3 ligase PIAS1 as a key regulator of EBNA1 through site-specific SUMOylation. Our chromatin immunoprecipitation sequencing analysis revealed that PIAS1 is enriched at the viral origin of plasmid replication (oriP), where it physically associates with EBNA1 and catalyzes its SUMOylation. Using mutational analysis, we identified three lysine residues on EBNA1 (K17, K75, and K241) as major SUMOylation sites. Disruption of these sites compromises EBNA1’s ability to restrict Epstein-Barr virus (EBV) lytic replication. In addition, both PIAS1 depletion and the disruption of EBNA1 SUMOylation lead to reduced retention of EBNA1-OriP-based EBV mini-replicon, indicating the importance of EBNA1 SUMOylation in viral episome maintenance. Together, these results uncover a conserved post-translational mechanism by which PIAS1-mediated SUMOylation modulates EBNA1 function and EBV episome maintenance, suggesting a broader role for SUMOylation in viral latency, lytic replication, and persistence. IMPORTANCE Epstein-Barr virus (EBV) persists in infected cells by maintaining its episome through the viral protein EBNA1. We discovered that PIAS1 SUMOylates EBNA1 at specific sites, a process essential for EBNA1 to retain the viral episome and suppress reactivation. When SUMOylation is disrupted, the EBV-based replicon becomes less stable, and EBV is more likely to reactivate. These findings reveal a new layer of host control of EBV latency and reactivation and highlight PIAS1-mediated EBNA1 SUMOylation as a key mechanism regulating viral persistence. Epstein-Barr virus (EBV) persists in infected cells by maintaining its episome through the viral protein EBNA1. We discovered that PIAS1 SUMOylates EBNA1 at specific sites, a process essential for EBNA1 to retain the viral episome and suppress reactivation. When SUMOylation is disrupted, the EBV-based replicon becomes less stable, and EBV is more likely to reactivate. These findings reveal a new layer of host control of EBV latency and reactivation and highlight PIAS1-mediated EBNA1 SUMOylation as a key mechanism regulating viral persistence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc01f3f9413962f3c30d44d5bf1286791904fed" target='_blank'>
              EBNA1 SUMOylation by PIAS1 suppresses EBV lytic replication and enhances episome maintenance
              </a>
            </td>
          <td>
            F. G. Sugiokto, Kun Zhang, Yunash Maharjan, Renfeng Li
          </td>
          <td>2025-12-11</td>
          <td>mBio</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L.
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that dictate complex transcriptional repertoire. Herpes viruses exhibit programmed latent and lytic gene expression depending on the infected tissue and physiological state. Previously, using a systematic functional assay, we identified six enhancers within the genome of Kaposi’s sarcoma-associated herpesvirus (KSHV). In this study, we present a natural language processing model (NLP)-based tool, ENHAvir, that is trained with these six enhancers and non-enhancer control sequences from the KSHV genome. ENHAvir identifies known enhancers and predicted novel enhancer elements in human herpesviruses. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV is confirmed in an enhancer reporter assay. The terminal repeats of all the herpes viruses also serve as a strong enhancer. Comparing herpesvirus enhancers with human enhancers reveals conserved enhancer signatures and the involvement of Alu elements. Here, we present an AI tool that successfully predicts enhancers in both viral and human genomes. Artificial intelligence (AI) can detect and predict patterns that are hidden from the human eye and from conventional homology-detection tools. Enhancers are distal DNA cis-regulatory elements that regulate complex transcriptional repertoire. A natural language processing (NLP) model was trained using only six enhancer sequences from the Kaposi’s sarcoma-associated herpesvirus (KSHV) genome. This tool, termed ENHAvir, can identify known enhancers and predict novel enhancer elements in other herpesviruses, different viruses, and the human genome. The activity of the predicted enhancers in HSV-2, HCMV, HHV-6, HHV-7, and EBV was confirmed experimentally, enabling the creation of a comprehensive enhancer map of human herpesviruses. All human herpesviruses contain terminal repeats (TRs), which play important roles in cleaving the viral genome into genome-size units, genome encapsidation, and genome circularization following entry into the nucleus of a newly infected cell. This study adds another role for the TR of all human herpesviruses, a strong enhancer with the features of a “viral super enhancer”. Comparing herpesvirus enhancers with human enhancers revealed conserved enhancer signatures and the involvement of Alu elements. Here, an AI tool is presented that successfully predicts enhancers in both viral and human genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980482a475765af78491bb69bee337aba4e9cbe1" target='_blank'>
              Discovering the complete enhancer map of human herpesviruses using a natural language processing model
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Deepanway Ghosal, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-12-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Almost half the mammalian genomes consist of transposable elements (TEs) and their derivatives. The distribution density of TEs can be associated with genomic regions of chromosomal rearrangements in different mammalian species and with the genomic localization of protein-coding genes that differ in length and function. To evaluate these characteristics at the local genomic level, an analysis of the distribution of various TEs (retrotransposons and DNA transposons) was performed in three mammalian species (human, cattle, and domestic rabbit) in genes with different functions and chromosomal localizations and their flanking regions. In humans and rabbits, melanophilin (MLPH) and myostatin (MSTN) are syntenic, but not in cattle. In the latter, MLPH and the leptin receptor (LEPR) are syntenic, but not in humans and rabbits. The alpha-thalassemia gene is always located on chromosome X. The results indicate that the frequencies of different TEs are species-specific and do not depend on the length of genes, their function, or chromosomal localization. There were also species-specific differences in the ratio of “ancient” and “young” short interspersed nuclear elements (SINEs) and long interspersed nuclear elements (LINEs). There was a statistically significant positive correlation between ancient SINE + LINE and LTR-ERV (p < 0.01) and a significant negative correlation between young SINE + LINE and DNA transposons (p < 0.05). Competitive relationships between TEs are probably defined by the presence of identical regulatory motifs in different TEs, associated with the reliance of TE amplification on the host’s own regulatory systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34daa4f3c8a087345e5a546465d4ca231c98f60" target='_blank'>
              Structural heterogeneity and functional convergence of transposable elements
              </a>
            </td>
          <td>
            G. Kosovsky, Galina V. Glazko, Tatiana Glazko
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Understanding genomic function has historically relied on sequence conservation across evolutionary time. However, advances in genomics have revealed that functional innovations often arise from rapidly evolving, nonconserved elements that are frequently overlooked by conservation-based approaches. Among these, variable number tandem repeats (VNTRs) act as engines of both functional innovation and phenotypic consequence. VNTRs are repetitive genomic sequences whose copy numbers can vary significantly between individuals and species, influencing gene regulation, protein structure, and eventually, phenotypic diversity. Recent long-read assemblies and pangenomes now resolve VNTR loci accurately, enabling robust evolutionary reconstruction and functional associations. Here, we synthesize emerging insights into the functional and evolutionary impact of VNTRs in mammals. Specifically, we outline pressing questions on the mutational mechanisms driving VNTR evolution in humans, the selective forces maintaining their structural heterogeneity, and propose a theoretical framework for their persistence through evolutionary tradeoffs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/606c1d5115973998729908888d2dc238d5ceeb64" target='_blank'>
              Evolutionary Balancing of Genetic Consequence and Innovation in Mammals Through Variable Number Tandem Repeats
              </a>
            </td>
          <td>
            Petar Pajic, Omer Gokcumen
          </td>
          <td>2025-12-24</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs), constituting approximately 8% of the human genome, represent genomic remnants of ancestral retroviral infections that colonized the germline through evolutionary processes. While most HERVs remain epigenetically silenced, their reactivation through environmental stimuli or epigenetic dysregulation enables participation in oncogenesis via viral mimicry, immunomodulation, and insertional mutagenesis. Substantial evidence now implicates aberrant HERVs activity across urologic malignancies—including prostate cancer, renal cell carcinoma (RCC), bladder cancer, and testicular germ cell tumors—where cancer-type-specific mechanisms drive tumor development and progression. These encompass androgen-responsive HERV-K activation in prostate malignancies, hypoxia-inducible factor-mediated ERV immunogenicity in RCC, HERV-derived microRNA silencing of tumor suppressors in bladder cancer, and DNA hypomethylation-associated HERV expression in testicular germ cell tumors. This review synthesizes fundamental HERV biology with recent advances in their diagnostic and therapeutic applications for urologic neoplasms. Key clinical translations include ERV-based stratification models predicting immune checkpoint inhibitor response in metastatic RCC, HERV-E-targeted adoptive T cell therapies, and noncoding RNA biomarkers for early bladder cancer detection. We further discuss unresolved mechanistic paradoxes such as contradictory prognostic associations between HERV superfamily expression and PBRM1 inactivation in RCC, concluding with priorities for future research: validating HERV-derived neoantigens in immunotherapy platforms, optimizing epigenetic priming strategies to enhance viral mimicry effects, and establishing standardized HERV signatures as clinical biomarkers through multi-institutional cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38383e2f180ac1b254c2d12832458ab1cd318585" target='_blank'>
              The expression, regulation, and function of human endogenous retroviruses in genitourinary cancers
              </a>
            </td>
          <td>
            Wenjie Ma, Chencheng Ji, Abudukelimu Abudushataer, Ning Liu, Tao Xu, Kunlun Zhao, Yiguan Qian, Paerhati Tuerxun, Xiaotian Jiang, Zhongli Xiong, Min Wang, Ruipeng Jia, Zheng Xu, Yang Li, Yu-Zheng Ge
          </td>
          <td>2025-11-28</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Meningiomas are the most common primary tumors of the central nervous system and are typically treated with surgery or radiation, as targeted therapies remain limited. Despite extensive study, seventeen percent of meningiomas lack known genetic drivers. Our analysis of meningiomas without driver mutations or major chromosomal alterations identifies a subset with recurrent genomic rearrangements involving the FOS and FOSB genes. These tumors exhibit elevated FOS/FOSB protein levels and retain meningothelial lineage. Here we show that FOS/FOSB fusion-positive meningiomas represent a distinct molecular subgroup, defined by unique gene expression patterns, including activation of AP-1 target genes and signatures resembling preadipocyte-like and mast cell–associated profiles. Clinically, these tumors display low-grade behavior and DNA methylation profiles consistent with benign subtypes. Our findings identify a meningioma subgroup with distinct genetic, transcriptomic, and clinical features, expanding the molecular classification of meningiomas and opening new avenues for targeted treatment strategies. Approximately 17% of meningiomas remain genomically uncharacterized. Here, the authors analyze 105 meningiomas without known driver mutations or significant copy number alterations and identify a subgroup of meningiomas, defined by FOS/FOSB gene fusions with distinctive transcriptomic and histopathological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/848d67ce829255f08e0caafaf14bc914e2cca146" target='_blank'>
              Subgroup of meningiomas involving FOS and FOSB gene fusions
              </a>
            </td>
          <td>
            K. Yalcin, Hasan Alanya, B. Gultekin, Diego Samper Figuera, Tanyeri Barak, D. Miyagishima, Mark W. Youngblood, Mateo Gomez Hjerthen, Amos Brooks, Natalia Samuel Lopez, Joseph O’Brien, M. M. Chavez, Alper Dincer, Hao Wu, S. Omay, Katsuhito Yasuno, K. Bilguvar, A. Ercan-Sencicek, Ketu Mishra-Gorur, D. McGuone, Jennifer Moliterno, Murat Gunel, E. Erson-Omay
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020f876f8e4a48a7cc99413edfa413effb89316e" target='_blank'>
              Interphase chromosome conformation is specified by distinct folding programmes inherited through mitotic chromosomes or the cytoplasm.
              </a>
            </td>
          <td>
            A. Schooley, Sergey V. Venev, V. Aksenova, Jesse W. Lehman, E. Navarrete, A. Pai, M. Dasso, Job Dekker
          </td>
          <td>2025-12-22</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The pathogenesis of koala retrovirus (KoRV) has been explored in various contexts, yet its role in tumorigenesis remains incompletely understood. Unlike acute transforming retroviruses, KoRV lacks a viral oncogene but may contribute to oncogenesis via indirect mechanisms. However, the relationship between KoRV and telomere length, as a potential indicator of telomerase activity, has not been examined. This study investigates the effect of KoRV infection on telomere length in 47 samples from Southern Australian koalas in a novel telomere length quantification method. Telomere lengths of 30 KoRV-negative samples were compared to those of 17 KoRV-positive samples using the Absolute Human Telomere Length Quantification qPCR kit (ScienCell Research Laboratories, California, USA). The telomere length in KoRV-infected WBCs was significantly longer than the uninfected ones (t = −2.059, p-value = 0.045). In line with this, telomere length correlated positively with proviral load (r = 0.421, p-value = 0.003), further linking viral burden to telomere elongation. Furthermore, the effect of age on telomere length differed by infection status (β = −5329.7, p-value = 0.0038); KoRV-positive individuals exhibited longer telomeres at a younger age but experienced more rapid telomere attrition over time compared to KoRV-negative individuals. These results suggest KoRV promotes telomerase elongation ability and modulates age-related telomere dynamics, potentially contributing to subsequent cellular immortality and oncogenesis. These pathways may overlap with other retroviruses, where telomerase dysregulation contributes to their oncogenic potential. This study provides new insights into KoRV pathogenesis and DNA quantification methodology, which could be valuable for future research by identifying predictive markers for tumour progression and potential therapeutic targets in affected koalas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f125ee840ff1c5667ff63226a6db37a09c90835b" target='_blank'>
              Infection-Induced Telomere Length Variation: Insights into Pathogenesis of Koala Retrovirus
              </a>
            </td>
          <td>
            Hiu Ming Cheung, Sze Wing Jamie Lin, Hanh Thi Hong Nguyen, T. Stephenson, N. Speight, F. Hemmatzadeh
          </td>
          <td>2025-11-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            , Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d2ed0a420dd36b96bfce3b68f81e00b3cf387cb" target='_blank'>
              Novel genes arise from genomic deletions across the bacterial tree of life
              </a>
            </td>
          <td>
            Arya Kaul, F. Rossine, K. Břinda, M. Baym
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Complex chromosomal rearrangement (CCR) refers to a structural rearrangement involving at least two chromosomes or a minimum of three breakpoints. CCR may lead to intellectual disability, structural anomalies, infertility, and recurrent miscarriages. Chromosome karyotyping and chromosomal microarray analysis (CMA) are unable to detect complex chromosomal rearrangements. As multiple diagnostic approaches are available in clinical practice for detecting chromosomal structural abnormalities and copy number variations—each with its own advantages and limitations—selecting the appropriate testing method is crucial for effective clinical management. Optical genome mapping (OGM) is an advanced genomic technology that utilizes ultra-long single-molecule analysis to comprehensively detect chromosomal aberrations and structural variants at high resolution. Amniocentesis was performed for a 36-year-old multipara (advanced maternal age), with subsequent comprehensive fetal genetic analysis including chromosome karyotyping, CMA, and OGM. Family members underwent peripheral blood karyotyping and OGM. The fetal karyotype derived from amniotic fluid was 46,XN,?ins(18)(q21.2;p11.31p11.2). CMA demonstrated duplications of four segments and a deletion of one segment on chromosome 18. Therefore, OGM was performed on the fetal and family members to further elucidate the chromosomal structure. The fetus has derived CCRs on chromosome 18 of maternal origin. In contrast, both the mother and the second daughter, who carried the identical CCRs, were phenotypically normal. OGM is of significant importance in the diagnosis and characterization of CCRs. OGM plays a critical role in diagnosing complex chromosomal rearrangements and has proven to be invaluable in clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a05870664d5ee26da7943564ddcc7debed72e97b" target='_blank'>
              Optical genome mapping of a complex structural rearrangement family line on chromosome 18
              </a>
            </td>
          <td>
            Liyi Cai, Yuying Jiang, Na Zhang, Xinying Chen
          </td>
          <td>2025-12-01</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f8abb290a62c5b3aa53581143c941a3ce167f6" target='_blank'>
              Lenacapavir-induced capsid damage uncovers HIV-1 genomes emanating from nuclear speckles.
              </a>
            </td>
          <td>
            Thorsten G. Müller, Severina Klaus, V. Zila, Bojana Lucic, C. Penzo, Svenja L. Nopper, Gonen Golani, Maria Anders-Össwein, Vera Sonntag-Buck, A. Heuser, U. Schwarz, V. Laketa, M. Lusic, Barbara Müller, H. Kräusslich
          </td>
          <td>2025-12-01</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Epstein–Barr virus (EBV)-associated gastric carcinoma (EBVaGC) represents a distinct molecular subtype characterized by extensive DNA methylation and altered cell death signaling. This study investigated the regulation and function of gasdermin E (GSDME), a key mediator of pyroptosis, in EBVaGC. Transcriptomic analysis of The Cancer Genome Atlas (TCGA) data revealed that GSDME expression was selectively suppressed in EBV-positive gastric cancer, while other gasdermin family members were upregulated. Validation in multiple cell lines confirmed that EBV infection markedly reduced GSDME expression through promoter hypermethylation, which was reversed by treatment with the DNA methyltransferase inhibitor 5-azacytidine. EBV-positive cells exhibited enhanced caspase-3 activation and increased GSDME cleavage upon paclitaxel (PTX) exposure, leading to elevated lactate dehydrogenase (LDH) release and pyroptotic morphology. Overexpression of GSDME amplified, whereas siRNA-mediated knockdown or caspase-3 inhibition suppressed, PTX-induced pyroptosis without significantly altering overall cell viability. These findings demonstrate that EBV-induced epigenetic silencing of GSDME contributes to the modulation of chemotherapy-induced cell death, and that GSDME acts as a critical effector converting apoptosis to pyroptosis through caspase-3 activation. Collectively, our results reveal a novel link between EBV-driven DNA methylation and pyroptotic cell death, suggesting that restoration of GSDME expression may enhance therapeutic responses in EBV-associated gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a0b5107e1c2a7dee621e1b78f47e3daf342a13" target='_blank'>
              Epstein–Barr Virus Silences GSDME and Pyroptosis in Gastric Cancer
              </a>
            </td>
          <td>
            Min-Hyeok Lee, Ju Yeon Lee, Jun Yeob Kim, Yea Rim An, Suk Kyeong Lee
          </td>
          <td>2025-11-27</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971170b40f565f98905175aa9310e7d4d0007c6c" target='_blank'>
              Deep conservation of cis-regulatory elements and chromatin organization in echinoderms uncover ancestral regulatory features of animal genomes.
              </a>
            </td>
          <td>
            M. S. Magri, D. Voronov, Saoirse Foley, P. M. Martínez-García, Martin Franke, Gregory A. Cary, J. Santos-Pereira, C. Cuomo, Manuel Fernández-Moreno, Marta Portela, A. Gil-Gálvez, R. D. Acemel, Periklis Paganos, Carolyn Ku, Jovana Ranđelović, Maria Lorenza Rusciano, Panos N. Firbas, J. Gómez-Skarmeta, Veronica F Hinman, M. Arnone, Ignacio Maeso
          </td>
          <td>2026-01-07</td>
          <td>Nature ecology & evolution</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/552a330ec3ca5c7fe5f601ebf3e9706e25f9ad5d" target='_blank'>
              High PARP1 expression is associated with proliferative tumor biology in breast cancer
              </a>
            </td>
          <td>
            Farhad Ghasemi, Jun Arima, K. Takabe
          </td>
          <td>2025-11-27</td>
          <td>Breast Cancer Research and Treatment</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Medulloblastoma (MB) is the most common malignant pediatric brain tumor and exhibits high heterogeneity in genetic alterations and molecular subgroups. However, current treatment remains largely uniform, and more precise risk stratification is necessary to improve prognosis. Large-scale genomic studies have primarily focused on Western populations, while comprehensive data on Japanese patients remain limited due to low incidence. This study aimed to elucidate molecular genetic characteristics and prognostic factors in Japanese MB through integrative genomic and transcriptomic analyses. Methods Targeted DNA and RNA sequencing was performed on 200 cases from the Japanese Children’s Cancer Group (JCCG), and publicly available whole-genome and RNA sequencing data from 472 Western MB patients were analyzed for comparison. Results Known subgroup-specific driver mutations were identified in the Japanese cohort, and age-adjusted comparison with the Western cohort showed no significant differences in mutation frequencies, suggesting limited racial differences in MB pathogenesis. Deep sequencing detected subclonal mutations, including SMARCA4 in the WNT subgroup. Missense mutations in the MYC functional domain were observed in cases lacking MYC amplification; these are known drivers in hematological malignancies and may also play an oncogenic role in MB. MYC mutations were found in WNT and SHH subgroups, suggesting broader involvement beyond Group 3. Intron retention analysis revealed a non-canonical intronic ZMYM3 mutation in a Group 4 case, highlighting improved driver detection by integrating DNA and RNA data. Whole chromosomal aberrations correlated with better outcomes, supporting their role in risk classification. Conclusions This study identified novel MYC alterations and emphasized the utility of RNA-integrated analysis. Cohort expansion may uncover additional diversity and aid in precision medicine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394acb3917250c67e7f0d75d4a797dbd4bea5da9" target='_blank'>
              10102-GEN-2 Integrated Genomic and Transcriptomic Analysis Reveals Molecular Diversity and Novel Drivers in Japanese Medulloblastoma
              </a>
            </td>
          <td>
            Tsubasa Miyauchi, Takuma Nakashima, Miho Kato, Junko Hirato, Takako Yoshioka, Hajime Okita, Y. Kanemura, Yuko Watanabe, Masahiro Toda, Hiromichi Suzuki
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="PURPOSE
HPV integration (HPVint) is associated with carcinogenesis and tumor progression in HPV-associated cancers, including head and neck squamous cell carcinomas (HNSCC). While its impact on human DNA has been well characterized, its relationship with clinical outcomes remains unconfirmed.


EXPERIMENTAL DESIGN
We analyzed HPVint events from 261 HPV-associated HNSCC bulk and single-cell RNA-seq samples from five cohorts, including 62 from a new University of Michigan cohort, and DNA HPVint events from 102 HPV(+) HNSCC participants in two of the cohorts. We investigated the consequences of HPVint both with respect to human and HPV gene expression and clinical outcomes (recurrence and overall survival).


RESULTS
By leveraging this large meta-cohort of HNSCC, we first reveal an oncogenic gene network based on the recurrent HPV integration locations in the human genome and gene expression alterations, highlighting key recurrent and overexpressed genes including NR4A2, CD274, CCER1 and genes from the CAMK and KLF families. We then stratify HPVint-positive participants by risk using HPV RNA features, specifically spliced HPV-human fusion transcripts (E1* integration) and HPV gene expression ratios (HGER), showing that subsets of participants have worse clinical outcomes based on these two candidate biomarkers.


CONCLUSIONS
By focusing on RNA instead of DNA, we expand our understanding of the carcinogenic mechanisms of HPVint, in part addressing the conflicting findings of whether HPVint is associated with aggressive phenotypes and worse clinical consequences and provide potential biomarkers to advance precision oncology in HPV-associated HNSCC. Newly identified genes with recurrent integration events may serve as candidates for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/737e8ec9fc9860af7b63c3f955141de9c82f430b" target='_blank'>
              HPV integration in head and neck cancer: downstream splicing events and expression ratios linked with poor outcomes.
              </a>
            </td>
          <td>
            Shiting Li, Shaomiao Xia, Maria Lawas, Aishani Kulshreshtha, Bailey F. Garb, Sarah E. Soppe, Chamila Perera, Chen Li, Min Liu, Y. X. Lim, Tingting Qin, Joshua D. Welch, N. D’Silva, Laura S Rozek, Maureen A. Sartor
          </td>
          <td>2025-11-21</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f186703aec8952740844b8d3f7ea360a94084d2a" target='_blank'>
              The human DDX52 protein is a nucleic acid helicase and strand annealase that promotes cell migration.
              </a>
            </td>
          <td>
            Ashley J Parkes, Philipp J Springer, E. Bolt
          </td>
          <td>2026-01-09</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In Poaceae, the rho whole-genome duplication (ρ-WGD) event gave rise to three Paleo-duplicated chromosome pairs (PdCPs) that are enriched in resistance genes (nucleotide-binding site-leucine-rich repeat (NBS-LRR) genes) but show low transcriptional levels. Despite their evolutionary importance, systematic family-wide studies of PdCPs have been lacking. We analyzed 37 Poaceae genomes using karyotyping and evolutionary analyses, mapped NBS-LRR genes distributions, and examined transcriptomic and epigenomic patterns across representative species to characterize the evolution and regulation of PdCPs. Our findings show that PdCPs have persisted throughout the evolutionary history of Poaceae, exhibiting distinct patterns of rearrangements among different subfamilies. Notably, one PdCPs is consistently enriched in disease resistance genes across multiple subfamilies and displays conserved low transcriptional activity. This PdCPs also exhibits conserved CHH hypomethylation around transcription start site regions. Further analyses suggest that this PdCPs maintains a more closed chromatin state, associated with repressive histone modifications that suppress gene expression. These results imply that the low transcriptional levels of genes on this PdCPs may be regulated by multiple epigenetic mechanisms. This study advances our understanding of the formation, diversification, and epigenetic regulation of PdCPs following the ρ-WGD in Poaceae.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c523386367da789f11d93db121c4e7fb73dc60a" target='_blank'>
              Evolution of paleo-duplicated chromosome pairs with enrichment of NBS-LRR genes generated through ρ-WGD in Poaceae.
              </a>
            </td>
          <td>
            Yuhao Wang, Yiying Qi, Yixing Zhang, Baiyu Wang, Xiaoli Sun, Hongyan Ding, Jingsheng Xu, Qing Zhang, Jisen Zhang
          </td>
          <td>2026-01-02</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract RECQL4 plays an important role in maintaining the integrity of the genome and regulating DNA replication. However, the role of RECQL4 in CNS tumors remains unknown. Sequencing data were reviewed and immunohistochemistry was performed on a variety of glial and nerve sheath tumors. Functional studies were performed in glioma (U251) and malignant peripheral nerve sheath tumors (MPNSTs) (NF90-8, ST88-14) cell lines following RECQL4 knockdown and treatment with ATR-inhibitors. Across 1580 CNS tumors, RECQL4 gene variants were identified in 71 cases (4.5%), with 21 (29.6%) of probable pathogenic significance. RECQL4 expression differed significantly across glioma subgroups (P = 0.012). Low-grade gliomas (diffuse: median H-score 57.5; circumscribed: median 130) showed lower expression than high-grade gliomas (median 145, P < 0.05). Neurofibromas displayed higher RECQL4 expression (median 160) compared with MPNSTs (median 97.5, P < 0.001). Among MPNSTs, NF1-associated cases (n = 24, median 95) expressed significantly less RECQL4 than sporadic cases (n = 8, median 162.5, P < 0.001). RECQL4 knockdown in glioma and MPNST cell lines resulted in increased apoptosis and susceptibility to ATR-inhibitors. Our findings show that RECQL4 expression has divergent patterns across tumor types and that targeting RECQL4 may dampen tumor survival and enhance susceptibility to ATR inhibitor therapy in CNS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad38b8a0f643cf6b41568ada22ffe40705acaba7" target='_blank'>
              RECQL4 alterations in gliomas and nerve sheath tumors: Expression patterns and therapeutic implications
              </a>
            </td>
          <td>
            Sarra Belakhoua, Gianluca Lopez, Swati Dubey, Simran Rai, Liam Chen, Suping Chen, Aparna Pallavajjala, Ming Yuan, M. Pekmezci, Marija Stojanova, Christopher M. Heaphy, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2025-11-29</td>
          <td>Journal of Neuropathology and Experimental Neurology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Plant mitochondrial genomes exist and function in a special way, this third-kind genetic material is essential for plant growth and development. However, its complex and highly variable genome structure leads to a poor understanding compared with chloroplasts. Especially Phoebe bournei (Magnoliaceae), an indigenous species of China, showcases a profusion of metabolites and excellent wood properties with long growth cycle to maturity. With the help of HiFi sequencing, we obtained new insights of their structure and variations. We initially curated the complete reference mitochondrial genome for P. bournei, a 775 Kb two-circle merged main physical map. These unequal circles exhibited a simplification trend through one long lateral repeat region, with a whole GC content of 47.09%. Conserved 43 protein coding genes were annotated and evenly distributed in the two genome rings. COX-3-ATP8-CYTB gene synteny events were found in P. bournei and M. biondii, others in Lauraceae were randomly shuffled by recombinant fragment due to rapid reorganization. Aberrant selective pressure occurred for ccmB, Ka/Ks more than 1 in Magnoliids and around 2.5 in representative angiosperms. Mitogenome genes bore a greater burden of pressure compared to genes in those of chloroplasts. Comparisons of consistent sequences revealed significant large-scale and inclined rearrangements within both cytoplasmic and nuclear genomes. Furthermore, the identification of lateral transfers from chloroplast to mitochondrial genome highlighted the consistent capture of functional genes such as petN. Besides, transferred sequences displayed chromosomal preference in nuclear genome. Based on the first report of a simple multi-circular conformation of the P. bournei mitogenome, our results may provide a primary foundation for the evolution of Magnoliids and further application in the tree breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d28a5a96ede5a624f57f04162b08b16fcdb33c49" target='_blank'>
              The mitochondrial genome of Phoebe Bournei highlighted the structure variation driven evolution in magnoliids
              </a>
            </td>
          <td>
            Kai Zhao, Mingli Shen, Ruiyue Zheng, Yukun Peng, Jiemin Chen, Xuanyi Zhu, Xianmei Luo, Tong Tong, Jianlian Xie, Sagheer Ahmad, Yuzhen Zhou, Chaobin Xu, Baoyin Li, Quanlin Zhong
          </td>
          <td>2025-12-02</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e4000185fee005531c8551eb93fcffa867d702" target='_blank'>
              Transcription-induced mutation and gBGC at the Transcriptional Start Site impact the evolution of human protein-coding genes
              </a>
            </td>
          <td>
            Yi Qiu, Emilie Hebraud, Fanny Pouyet, Alexander F. Palazzo
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde6df8438fec9eca3014692df8a1930274045c8" target='_blank'>
              A pangenome framework uncovers the role of deletions in repeated evolution of cave-derived traits
              </a>
            </td>
          <td>
            Emma Y. Roback, X. Maggs, Edward S Ricemeyer, Adam Warlen, Rachel A. Carroll, Christine G Elsik, Alex C. Keene, Nicolas Rohner, S. McGaugh, Wes C. Warren
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11e6d7e02da2372f0b7447b006156f71a150d88" target='_blank'>
              Evo2HiC: a multimodal foundation model for integrative analysis of genome sequence and architecture
              </a>
            </td>
          <td>
            Tangqi Fang, Xiao Wang, Zhiping Xiao, Shengqi Hang, Ghulam Murtaza, Junwei Yang, Hanwen Xu, Anupama Jha, W. Noble, Sheng Wang
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966381538c6447576a47e18b17f5babd106c5c90" target='_blank'>
              Structural organization and function of telomeric chromatin.
              </a>
            </td>
          <td>
            Ruben van der Lugt, J. Jacobs
          </td>
          <td>2026-01-07</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Retroviruses are exclusive group of positive-sense RNA viruses defined by their ability to reverse transcribe their RNA genome and integrate it into the host’s chromosomal DNA. This distinctive replication strategy enables persistent infection and has profoundly shaped our understanding of molecular biology, gene regulation, and evolution. Retroviruses have contributed to landmark discoveries, including the identification of oncogenes, mechanisms of transcriptional control, and the development of gene therapy vectors. This review provides an updated overview of retroviral molecular biology, emphasizing the coordinated steps of the viral life cycle and emerging insights that are reshaping classical models. It explores virion structure, genome organization, and the interplay of cis-acting sequences and trans-acting factors that govern replication. Special focus is given to recent advances in understanding nuclear trafficking of capsids, spatial dynamics of reverse transcription and integration leading to provirus formation, RNA nuclear export, and selective genome packaging. The structural and functional roles of viral proteins, particularly Gag, are discussed in the context of assembly and maturation. By integrating foundational concepts with new discoveries, this review highlights the molecular sophistication of retroviral replication and identifies outstanding questions that guide future research, with implications extending to antiviral strategies, gene therapy, cancer biology, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2158d0cf0754098c7826e6edf394a24a8aa054" target='_blank'>
              Beyond reverse transcription: molecular mechanisms and emerging paradigms in retroviral replication
              </a>
            </td>
          <td>
            Mohammad Abdullah Jehad, L. Ali, Vineeta N Pillai, Suresha G. Prabhu, Farah Mustafa, Tahir A. Rizvi
          </td>
          <td>2025-12-26</td>
          <td>FEMS Microbiology Reviews</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, the roles of which are under debate. We found that double-strand breaks (DSBs) induce de novo formation of chromatin loops in human cells, with the loop base positioned at the DSB site. These loops form in the S and G2 phases of the cell cycle during homologous recombination repair, concomitantly with DNA end resection and radiation-sensitive protein 51 (RAD51) recruitment. RAD51 shows a broad (megabase-sized) chromatin domain reflective of the homology search. This domain is regulated by cohesin unloader and overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of the loop-extruding cohesin subunit NIPBL lowers homologous recombination in mouse embryonic stem cells, and this effect is more pronounced when the homologous recombination donor is hundreds of kilobases from the DSB. Our data indicate that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e61307e4b5e3ffa40f9315b36ee4db40b3f94b6" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam T. Rybczynski, Namrata Nilavar, Daniel Nguyen, Andrew G. Li, Violetta Karwacki-Neisius, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>2</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Proserpinaca palustris, a representative heterophyllous plant, represents an emerging model for studying morphological plasticity and adaptive evolution in amphibious environments. However, the absence of a high-quality reference genome has hindered molecular investigations into its heterophylly regulation mechanisms. Here, we present the first chromosome-level genome assembly of P. palustris. The final assembly spans 385.32 Mb, anchored onto seven pseudochromosomes (1n = 7), and encodes 22,756 predicted protein-coding genes. Benchmarking Universal Single-Copy Orthologs (BUSCO) assessment demonstrated high completeness, with scores of 95.61% for the genome assembly and 96.43% for the gene set. Comparative genomic analyses revealed that P. palustris underwent only the ancient γ whole-genome triplication (γ-WGT) event, with no subsequent whole-genome duplication (WGD). Nevertheless, the genome has experienced extensive re-diploidization, likely facilitated by structural rearrangements. Transcriptomic profiling identified three KNOTTED1-like homeobox (KNOX) genes (i.e., PPA02C1640, PPA03C1159, and PPA01C1293) that were significantly differentially expressed between submerged and terrestrial leaves in P. palustris, suggesting their potential role in regulating heterophylly. This study provides the first high-quality genomic resource for the genus Proserpinaca (Haloragaceae) and identifies key KNOX genes implicated in heterophylly regulation in P. palustris.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e74e7485e23cc3414f37553359e06ad9b96d4f3" target='_blank'>
              Chromosome-level genome of Proserpinaca palustris reveals key KNOX genes related to heterophylly formation
              </a>
            </td>
          <td>
            Yan-hong Dong, Qian Qin, Fengqin Li, Zhonghua Tu, Chengyi Tang
          </td>
          <td>2025-12-01</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g., “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, S. Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) is a DNA virus that establishes a persistent asymptomatic infection during childhood and can cause Merkel cell carcinoma (MCC) later in life. Its Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, drive viral replication and tumorigenesis by binding to and perturbing the function of host proteins. LT binds and inhibits RB1, deregulating E2F activity and host cell cycle control to permit viral replication during S phase. While the functions of LT and ST are relatively well characterized, how their expression is controlled is poorly defined. Here, we discovered that E2F1-3/DP1 dimers bind the MCPyV Non-Coding Control Region (NCCR) via a consensus E2 site close to the LT/ST transcriptional start site. Inhibiting E2F-NCCR binding, either by deleting the E2 site or treatment with a E2F small molecule inhibitor, downregulated LT/ST mRNA and protein expression in MCC cells and in 293A cells transfected with MCPyV. Our findings reveal an E2F/LT/RB1 positive feedback loop that appears to have evolved to support viral replication and is hijacked in MCC cells to promote cellular proliferation. Furthermore, we identified E2 sites in the NCCRs of PyVs closely related to MCPyV, including murine PyV, which mediate E2F/DP binding and potentiate viral early transcription. E2F/DP also bound weakly to the SV40 and BKPyV NCCRs, despite lacking an E2 site. Our findings challenge the prevailing model that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV early transcription and replication. Significance statement Polyomaviruses (PyVs) express Large and Small Tumor antigens (LT, ST), splice variants of a common viral early transcript, that drive viral replication and tumorigenesis by binding and perturbing the function of host cells. LTs bind and inhibit RB1 via conserved LxCxE motifs, deregulating E2F activity and host cell cycle control to permit viral replication. Here, we discovered that E2F1-3 bind to the Non-Coding Control Regions (NCCRs) of many, but not all, PyVs to activate LT/ST transcription, revealing an E2F/LT/RB1 positive feedback loop that appears to have evolved to promote viral replication. Our findings challenge the existing paradigm that PyV LT expression drives S phase entry and suggest, in contrast, that S phase entry stimulates PyV LT/ST transcription and replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92fb2e450770315d4079985e1a3aa56f6892c80a" target='_blank'>
              E2F1–3 activate Merkel cell polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Nick Salisbury, Supriya Patil-Amonkar, Ann Roman, Denise A. Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A correlation between karyotype diversity and species richness was first observed in mammals in 1980, and subsequently confirmed after controlling for phylogenetic signal. The correlation was attributed to submicroscopic factors, presumably operating at the level of the genome. At the same time, an unexpected association between mutation rates and substitution rates has been observed in all eukaryotes so far examined. One hypothesis to explain the latter observation proposed that neutral mutation (dS) and non-neutral (dN) substitution rates in gene codons co-vary according to genomic position, or location in the genome. Later, it was found that mutation and substitution rates in eukaryotes increase with DNA replication timing during the Synthetic phase, or S phase, of the cell cycle. In 1991, Motoo Kimura proposed a molecular theory of non-adaptive radiation (NAR). Accordingly, genetic drift plays a significant role in speciation, albeit in parallel to and in conjunction with natural selection. The following will examine the contribution of DNA replication timing and DNA repair factors to the relationship between species richness and karyotype diversity, and by extension, to the large variation in species richness across the mammalian Tree of Life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8f06330c7331a025a26ac3aac8d62db4f0ac6f" target='_blank'>
              DNA Replication Timing, Genome Stability and Non-adaptive Radiation
              </a>
            </td>
          <td>
            John Herrick
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Maintaining genome integrity in germ cells is crucial for fertility and species survival. However, the DNA repair mechanisms that sustain genome integrity in primordial germ cells (PGCs), which cope with high levels of replication stress, remain largely unknown. While the TatD family of proteins, evolutionarily conserved nucleases, has been found to play roles in various DNA-related processes, their in vivo functions in vertebrates have yet to be fully elucidated. TATDN2 has recently been implicated in resolving R-loops and participating in the replication stress response in BRCA1-deficient cancer cells. Here we found that tatdn2 exhibits conserved expression in mitotic and early meiotic germ cells across teleosts and mammals. Using medaka fish as a model, we then showed that loss of tatdn2 leads to all-phenotypically male adults and infertility due to PGC depletion during mitotic proliferation. We further demonstrated that knockout of tatdn2 increases R-loop accumulation and DNA damage, subsequently triggering apoptosis in PGCs. These findings indicate that tatdn2 plays a critical role in DNA damage repair associated with R-loop resolution in mitotic PGCs. Our study provides novel insights into the physiological function of TATDN2 and the mechanisms of genome maintenance in PGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aba7e41b7a154253da5adb67bf89677856b6ff" target='_blank'>
              Tatdn2 is required for DNA repair to safeguard genome stability in primordial germ cells
              </a>
            </td>
          <td>
            Hanqiao Shang, Puxuan Jiang, Zehui Wang, Zhaojun Shan, Yuze Chen, Ting Zhang, Yingshu Li, Qiang Tu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Lysine specific demethylase 1 (LSD1), encoded by the gene KDM1A, is overexpressed and correlates with poor patient prognosis in Ewing sarcoma. LSD1 and the pathognomonic fusion oncoprotein, EWSR1::FLI1, colocalize throughout the genome, suggesting LSD1 is a critical co-regulator driving the progression of Ewing sarcoma. However, therapeutic targeting of LSD1 by competitive and noncompetitive inhibitors has yielded mixed results. Irreversible, enzymatic inhibition seems ineffective, but reversible noncompetitive inhibition has predominant off target mechanisms, leaving open the question of LSD1 function in Ewing sarcoma. Here we take a robust approach through multiple methods of depletion in multiple EwS cell lines to define enzymatic and nonenzymatic contributions of LSD1 to transcriptional regulation. We define a core set of 22 genes that are commonly repressed by LSD1 in all cell lines, and that repression of these genes downregulates synapse functioning and e-cadherin target genes. Derepression of these genes with LSD1 loss is an early and sustained genotype in all cell lines tested. We further define distinct gene sets in each cell line that are regulated by enzymatic and nonenzymatic LSD1 activity and find repression of e-cadherin target genes to be nonenzymatically regulated. This finding supports the growing body of evidence that in addition to their canonical catalytic activity, chromatin regulatory enzymes serve essential noncanonical roles as well. Furthermore, we uncovered evidence through use of the irreversible inhibitor OG-L002 that 2D cytotoxicity and proliferation assays may be insufficient to determine Ewing sarcoma response to LSD1 inhibition. SIGNIFICANCE Here we address a long-standing question in the field surrounding LSD1 and define the distinct enzymatic and nonenzymatic functions of LSD1 in Ewing sarcoma. In doing so, we have created a robust data set using genetic and pharmacological techniques in multiple models to thoroughly characterize LSD1 function in Ewing sarcoma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa31c55d8000608b4f5b4507892563a7a74a977c" target='_blank'>
              LSD1 Performs Demethylase-Independent and Context-Specific Roles in Ewing Sarcoma
              </a>
            </td>
          <td>
            Rachel D. Dreher, C. Taslim, Ira Miller, John W. Sherman, Ariunaa Bayanjargal, Emily R. Theisen
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, Mohammad Alinoor Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="


 Despite advancements in treatment, 5-year survival for medulloblastoma (MB) remains 60-70%. Human cytomegalovirus(HCMV) has been implicated as an oncomodulator in MB, but its impact on survival has not been explored. This study evaluates HCMV expression in pediatric MB tissue and its association with molecular risk groups and survival.



 A retrospective study of pediatric MB cases (≤19 years) from 2007-2023 was conducted using the WHO 2021 classification, the Northcott brain-tumor classifier, and the SIOP-Europe/ERN PaedCan risk stratification. HCMV immediate-early (HCMV-IE) and late-antigen (HCMV-LA) expression were assessed by optimized immunohistochemistry, and whole-genome sequencing (WGS) data from 20 tumors were analyzed for viral gene expression.



 Forty-five patients (mean age 8.2 years; 22% ≤3 years) were included: WNT (18%), SHH TP53-wildtype (18%), and non-WNT/non-SHH (64%). 20% of MB belonged to low-risk, 33% to standard-risk, and 47% to high-risk (HR-MB) groups. Four tumors exhibited MYC/N amplification. HCMV-LA positivity was found in 84% of cases, with 53% showing high expression (≥25% of cells). Cox-regression identified HR-MB (HR = 4.197, p = 0.021) and high HCMV-LA (HR = 4.334, p = 0.027) as independent predictors of poor outcomes. WGS revealed UL88 as the most abundantly expressed HCMV gene(log2[TPM+1] ≈14–15), with maximal expression in Group 3 MB.



 This study provides the first evidence linking high HCMV-LA expression to adverse outcomes and high-risk molecular features in pediatric MB. UL88 emerged as the most strongly expressed HCMV gene across MB samples. These findings suggest a potential prognostic and therapeutic role for HCMV in MB, warranting validation in larger, prospective cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60feabdb3f3fd823a43f478c57c8aa48c2982d72" target='_blank'>
              High Human Cytomegalovirus antigen Expression in pediatric medulloblastoma tissue is associated with poor event-free survival
              </a>
            </td>
          <td>
            Maria F De la Cerda-Vargas, J. Schittenhelm, Martin Ebinger, J. Zipfel, R. Beschorner, Ghazaleh Tabatabai, Martin U Schuhmann
          </td>
          <td>2025-12-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte-rich malignant tumours with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte-rich tissues, and then performed genome-wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis-regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour-specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D-loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour-specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome-wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas.
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Kaposi’s sarcoma–associated herpesvirus (KSHV) causes cancers for which no vaccine exists, and current treatments are often imperfect. Recent work shows that the virus and infected cells produce circular RNAs, which are stable RNA loops that can influence gene activity and build up during active infection. Because these molecules resist degradation and may travel inside virus particles, they can help track disease or shape how the virus behaves. This review explains how viral circular RNAs are produced, how researchers detect and assay them, and what is known about their roles during quiet and active stages of infection in the context of KSHV. It also outlines experimental tools to test what these RNAs do and how to target them. By organizing current knowledge and practical methods, this review aims to guide future studies and speed progress toward better diagnostics and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c49ebc6839a316e30284da0f7440e889a77090bc" target='_blank'>
              Recent Studies on Kaposi’s Sarcoma-Associated Herpesvirus Circular RNAs
              </a>
            </td>
          <td>
            Cristian J. Pagtalunan, Isadora Zhang, Ariella Turley, Fenyong Liu
          </td>
          <td>2025-11-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The Major Histocompatibility Complex (MHC) is a gene-dense genomic region essential to adaptive immunity, exhibiting both conserved features and lineage-specific rearrangements across jawed vertebrates. Amphibians, as basal tetrapods, offer critical insights into MHC evolution; however, research has largely focused on anurans (e.g., Xenopus), while urodeles remain understudied, primarily due to their exceptionally large genomes. With recent advances in sequencing technologies, chromosome-scale assemblies for urodeles are now becoming available, opening new opportunities to explore their MHC architecture. Yet, the MHC region remains notoriously challenging to annotate due to its complexity, high polymorphism, and dynamic evolutionary history. For instance, the initial annotation of the axolotl (Ambystoma mexicanum) MHC relied heavily on synteny with mammalian genomes, which led to an overestimation of its size and misinterpretation of its structure. These inaccuracies have fueled evolutionary debates regarding the ancestral genomic structure of the MHC. Here, we present a comprehensive re-annotation of the axolotl MHC, revealing a typical organization found in tetrapods other than eutherian mammals: core MHC region with several, expressed MHC class I genes, tightly linked to their antigen processing genes, and to single loci of MHC class II genes. Contrary to the previous report, class I and class II genes are not separated by class III genes, and the overall MHC region is relatively compact (by axolotl genome standards). These findings correct earlier misconceptions and emphasize the need for annotation strategies that reflect the complex and lineage-specific nature of genomic regions rich in immune genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b74730aaf5e0adf1edf33d8597ff226cc12da49" target='_blank'>
              Compact genomic architecture of the axolotl MHC region: setting the record straight
              </a>
            </td>
          <td>
            M. Migalska, Krzysztof Fluder, K. Dudek, W. Babik
          </td>
          <td>2025-11-15</td>
          <td>Immunogenetics</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Simple Summary Transposable elements (TEs) are mobile genetic elements that are present in great numbers in the majority of eukaryotic genomes. They are major drivers of genome evolution as they can facilitate chromosome rearrangements, provide mechanisms for genomic shuffling, and contribute to genome expansion, thereby altering genome architecture. Additionally, TEs can also modify gene expression by disrupting regulatory sequences, impairing genes, and promoting the emergence of new sequences. Despite their abundance, identifying TEs is challenging and time-consuming due to their extreme diversity in DNA sequence. Many TE families are ancient, and most of their sequences have become inactive due to accumulated mutations and fragmentation. Consequently, different copies of the same TE can differ greatly from each other, which makes identifying decayed copies particularly difficult. In this work, we employ transformer architecture for TE classification. TEclass2 is an integrated classifier that rapidly predicts TE orders and superfamilies using models built on this advanced machine learning approach. The software is available through a web interface, allowing users to classify sequences into sixteen superfamilies according to the Wicker classification system, or alternatively, users can download the source code to train and build custom classification models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf277b82c7226a65ad3cdcbe7a255c99eb31947" target='_blank'>
              Transformer-Based Classification of Transposable Element Consensus Sequences with TEclass2
              </a>
            </td>
          <td>
            Lucas Bickmann, Matias Rodriguez, Xiaoyi Jiang, Wojciech Makałowski
          </td>
          <td>2025-12-29</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="   DNA oxidation is one of the main types of damage to the genetic material of living organisms. Of the many dozens of oxidative lesions, the most abundant is 8-oxoguanine (8-oxoG), a premutagenic base that leads to G→T transversions during replication. Double-stranded DNA can conduct holes through the π system of stacked nucleobases. Such electron vacancies are ultimately localized at the 5’-terminal nucleotides of polyguanine runs (G-runs), making these positions characteristic sites of 8-oxoG formation. While such properties of G-runs have been studied in vitro at the level of chemical reactivity, the extent to which they can influence mutagenesis spectra in vivo remains unclear. Here, we have analyzed the nucleotide context of G-runs in a representative set of 62 high-quality prokaryotic genomes and in the human telomere-to-telomere genome. G-runs were, on average, shorter than polyadenine runs (A- runs), and the probability of a G-run being elongated by one nucleotide is lower than in the case of A-runs. The re presentation of T in the position 5’-flanking G-runs is increased, especially in organisms with aerobic metabolism, which is consistent with the model of preferential G→T substitutions at the 5’-position with 8-oxoG as a precursor. Conversely, the frequency of G and C is increased and the frequency of T is decreased in the position 5’-flanking A- runs. A biphasic pattern of G-run expansion is observed in the human genome: the probability of sequences longer than 8–9 nucleotides being elongated by one nucleotide increases significantly. An increased representation of C in the 5’-flanking position to long G-runs was found, together with an elevated frequency of 5’-G→A substitutions in telomere repeats. This may indicate the existence of mutagenic processes whose mechanism has not yet been charac terized but may be associated with DNA polymerase errors during replication of the products of further oxidation of 8-oxoG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83075f585c67b259c09dcc5cab31e9fb64185e60" target='_blank'>
              DNA damage reflected in the evolution of G-runs in genomes
              </a>
            </td>
          <td>
            I. Grin, D. Zharkov
          </td>
          <td>2025-12-12</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In mitosis the duplicated genome is aligned and accurately segregated between daughter nuclei. CTCF is a chromatin looping protein that localizes to the centromere in mitosis with an unknown role. We previously published data showing that CTCF constitutive knockdown causes mitotic failure, but the mechanism remains unknown. To determine the role of CTCF in mitosis, we used a CRISPR CTCF auxin inducible degron cell line for rapid degradation. CTCF degradation for 3 days resulted in increased failure of mitosis and decreased circularity in post-mitotic nuclei. Upon CTCF degradation, CENP-E is still recruited to the kinetochore and there is a low incidence of polar chromosomes that occur upon CENP-E inhibition. Instead, immunofluorescence imaging of mitotic spindles reveals that CTCF degradation causes increased intercentromere distances and a wider and more disorganized metaphase plate, a disruption of key functions of the centromere. These results are similar to partial loss of cohesin, an established component of the centromere. Thus, we reveal that CTCF is a key maintenance factor of centromere function, successful mitosis, and post-mitotic nuclear shape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d9bc6bb6d8691a893743a8a807a0d273d05cd7" target='_blank'>
              CTCF maintains centromere function and mitotic fidelity.
              </a>
            </td>
          <td>
            Erin Walsh, Thomas Laskarzewski, T. Maresca, A. Stephens
          </td>
          <td>2026-01-05</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, limiting the efficiency and precision of genome editing in many clinically relevant tissues. Here, we address this barrier by using induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. CRISPR editing outcomes differ dramatically in neurons compared to genetically identical dividing cells: neurons take longer to fully resolve this damage, and upregulate non-canonical DNA repair factors in the process. Manipulating this response with chemical or genetic perturbations allows us to direct DNA repair toward desired editing outcomes in nondividing human neurons, cardiomyocytes, and primary T cells. By studying DNA repair in clinically relevant cells, we reveal unforeseen challenges and opportunities for precise therapeutic editing. Genome editing tools can precisely introduce a specified lesion into the DNA, but ultimately rely on the cell’s DNA repair machinery to determine the editing outcome. Here, authors demonstrate how neurons’ unique DNA repair pathways impact the safety, efficiency, and precision of CRISPR edits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3be0abd5edf19f4fcddf4c37efa7ec4d61c80" target='_blank'>
              Characterizing and controlling CRISPR repair outcomes in nondividing human cells
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J. Namaganda, Manasi M. Kumar, J. Hamilton, Rohit Sharma, Karena G. Chow, Luke A. Workley, Bria L. Macklin, Mengyuan Sun, Alvin Ha, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Philip H. Dierks, Rudra Bose, Julianne Jin, Barbara S. Perez, Cindy R. Sandoval Espinoza, Madeline P. Matia, Serena H. Lu, Luke M. Judge, Brian R. Shy, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2025-11-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive subtype of ovarian cancer. The combination of late-stage diagnosis and the tendency to exhibit resistance to existing treatments highlights a critical gap in effective therapeutic options. There is thus a need for novel strategies for targeting HGSOC, particularly in its advanced stages. To address this gap, we developed a comprehensive atlas profiling both the proteome and phosphoproteome levels across a panel of nine ovarian cancer cell lines from different subtypes. Subsequent differential expression analysis between the KURAMOCHI and the other cell lines, followed by phosphosite analyses, proposed the MTDH protein as a potential target. Further functional analyses of MTDH and the interacting protein SND1 with RNA silencing, as well as targeting their interaction, revealed that disrupting this interaction leads to the dysregulation of several pathways associated with cancer progression and invasion. In particular, interference with the MTDH-SND1 complex was associated with enrichment of ferroptosis-related pathways. Moreover, combining C26A6 treatment with ferroptosis inducers produced enhanced inhibitory effects in ovarian cancer cells, suggesting a possible strategy for targeting cancer cell vulnerabilities in HGSOC, which warrants further investigation beyond in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ebcb2c256b8e8b610debccb3b83d01feb81b9" target='_blank'>
              Proteomics discovery of MTDH and SND1 interaction vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Parisa Esmaeili, Ahmad Nasimian, Lucas Werner, Sergio Mosquim Junior, Magnus E Jakobsson, A. Gerdtsson, Julhash U. Kazi, Fredrik Levander
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d6b8d33286dfd47e06aa024a1a7a5fb5466454" target='_blank'>
              Deciphering APOBEC1 in Avians: Unravelling loss events and functional insights
              </a>
            </td>
          <td>
            Aswin S Soman, N. Vijay
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="p53 is the most important tumor suppressor in humans as well as the most frequently mutated gene found in human cancers, with ∼50% of all human tumors bearing p53 missense mutations that leave p53 inactive. Restoring p53 activity has been shown to lead to tumor regression even in advanced tumors in mouse models and thus is among the most attractive potential strategies for novel cancer therapy. Full-length p53 (fl-p53) consists of 393 residues and multiple domains, some of which are folded while others are disordered. Using the crystal structures of folded domains and integrative molecular modeling techniques for disordered domains, we generated the first wild-type (WT) fl-p53 tetramer model bound to DNA. When solvated, the system size was ∼500 K atoms, challenging extensive sampling. Using the Anton2 supercomputer for microsecond timescale simulations in explicit solvent and the rigorous Markov state model (MSM) framework, we elucidated the conformational landscape of wild-type p53 as well as two of the p53 hotspot cancer mutants, Y220C and G245S, in a physiological DNA-bound, full-length tetramer context. In the simulated timescale, DNA-bound fl-p53 tetramer bent DNA and formed a compact complex with interactions between the N-terminal and DNA-binding domains (DBDs) and C-terminal domains (CTDs) with DNA. The WT fl-p53 tetramer also sampled a unique quaternary DBD organization that is not accessed by the cancer mutants. The free energy landscapes indicated differential dynamics for the inner and outer p53 DBDs due to the dimer-dimer interface. The dynamics of the druggable L1/S3 pocket was also closely monitored. Ultimately, MSMs identified an underexplored loop 6 (L6) cryptic pocket and captured the effect of p53 tetramerization and cancer mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/224ca78e22a43ba20d877b50802618d9011edcb5" target='_blank'>
              Toward In Situ Dynamics of DNA-Bound Full-Length p53 Tetramer.
              </a>
            </td>
          <td>
            Özlem Demir, Emília P. Barros, Rommie E. Amaro
          </td>
          <td>2026-01-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples are the gold standard for tissue preservation in clinical and research settings. Current single-cell chromatin accessibility technologies cannot resolve cell-type-specific epigenetic profiles in FFPE tissues due to extensive DNA damage. We present scFFPE-ATAC, a high-throughput single-cell chromatin accessibility assay for FFPE samples that integrates an FFPE-adapted Tn5 transposase, ultra-high-throughput DNA barcoding (>56 million barcodes per run), T7 promoter-mediated DNA damage repair, and in vitro transcription. We benchmark scFFPE-ATAC on FFPE mouse spleen and validate its performance against fresh tissue. We apply it to human lymph node samples archived for 8–12 years and to lung cancer FFPE tissues, revealing distinct regulatory trajectories between tumor center and invasive edge. Analysis of archived follicular lymphoma and transformed diffuse large B-cell lymphoma samples identifies relapse- and transformation-associated epigenetic dynamics. scFFPE-ATAC enables retrospective, spatial, and mechanistic epigenetic studies in long-term archived specimens. The study introduces scFFPE-ATAC, a high-throughput single-cell chromatin accessibility profiling method enabling profiling from clinically archived formalin-fixed paraffin-embedded (FFPE) tissues, unlocking retrospective epigenetic insights across cancer and human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42464048e0ef78f0787ee6a93b5eb7756d4e366" target='_blank'>
              scFFPE-ATAC enables high-throughput single cell chromatin accessibility profiling in formalin-fixed paraffin-embedded samples
              </a>
            </td>
          <td>
            Ram Prakash Yadav, Pengwei Xing, Miao Zhao, P. Hollander, C. Strell, Minglu Xie, Maede Salehi, Emma Torell, Tobias Sjöblom, G. Enblad, Rose-Marie Amini, F. Swartling, I. Glimelius, P. Micke, Mats Hellström, Xingqi Chen
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) resulting from inactivation of BRCA1/2 genes promotes chromosomal instability and renders tumor cells susceptible to platinum derivatives and PARP inhibitors (PARPi). The contribution of alterations in other homologous recombination repair (HRR) genes to HRD remains understudied. This investigation aimed to analyze the spectrum of mutations in 34 HRR genes in prostate carcinomas (PCs) and study the relationship between HRR status and HRD. HRR mutations and HRD scores were examined by NGS in 1131 and 680 PCs, respectively. Pathogenic or likely pathogenic variants in HRR genes were detected in 216/1131 cases (19.1%). HRD, defined by an HRD score cut-off of ≥42, was observed more frequently in HRR-mutated than in wild-type tumors (23/120 (19.2%) vs. 29/560 (5.2%), p < 0.0001). The highest HRD scores were detected in PCs with biallelic inactivation of the BRCA2 or PALB2 genes, as well as in tumors with BRIP1 mutations. HRD was also occasionally seen in PCs with ATM, NBN, FANCM, BRCA1 and CDK12 alterations, but never in cases with CHEK2 mutations. HRD was significantly more associated with aggressive PC features than HRR mutations. The majority of CDK12-mutated tumors exhibited a distinct type of copy number variations (CNV)–a tandem duplication phenotype. Our study suggests that the selection of PC patients for PARPi treatment requires a significant revision of existing attitudes towards tumor genetic profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4a6b0e731768bb34ffda6a1da1c394d76e62" target='_blank'>
              Complex Relationships Between Homologous Recombination Deficiency (HRD) Score and Mutational Status of Homologous Recombination Repair (HRR) Genes in Prostate Carcinomas
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, A. Shestakova, E. Kuligina, N. S. Morozova, Sergey V. Popov, Pavel V. Vyazovcev, Tatyana Y. Luchkova, Aleksey S. Peremyshlenko, Timur M. Topuzov, Olga M. Gudkova, R. V. Orlova, Andrey V. Levushkin, Daniil O. Moiseev, Oksana N Shkodenko, Demyan V. Gubarev, A. V. Dzyuba, Irina Y. Povorina, Anna V. Agaeva, V. F. Kutyan, Mhitar M. Grigoryan, Alexey N. Orlov, Spartak I. Lefterov, Aleksey V. Belousov, M. N. Nechaeva, E. Vorobyeva, Irina K. Amirkhanova, Nadezhda V. Kryukova, Lyubov I. Zatulivetrova, Aryuna B. Gomboeva, Vladimir N. Krivtsov, Olga I. Shchiglik, Natalya V. Prokudina, Natalya A. Butunina, A. M. Belyaev, E. Imyanitov
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Sorex araneus, the Eurasian common shrew, has seasonal brain size plasticity (Dehnel's phenomenon) and abundant intraspecific chromosomal rearrangements. Genomic contributions to these traits, however, remain unknown. We couple a chromosome-scale genome assembly with seasonal brain transcriptomes to discover relationships between molecular changes and both traits. While positively selected genes enriched the Fanconi anemia DNA repair pathway (FANCI, FAAP100), which is likely involved in chromosomal rearrangements by preventing the accumulation of chromosomal aberrations, genes under positive selection or showing seasonal differential expression in the brain implicate neurogenesis (PCDHA6, SOX9, Notch signaling) and metabolic regulation (VEGFA, SPHK2) as key mechanisms underlying Dehnel's phenomenon. We also find that both positively selected and differentially expressed genes in the hippocampus are overrepresented near Sor. araneus evolutionary breakpoints. This relates both positive selection and differential expression to accessible chromatin configuration, suggesting that chromosomal rearrangements are integral to adaptive evolution and the regulation of brain size plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e5d59bc3d794d733f5961db0a47844813d84f1" target='_blank'>
              Genomic comparisons shed light on the adaptive basis of brain size plasticity and chromosomal instability in the Eurasian common shrew.
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Cecilia Baldoni, Laia Marín-Gual, Diana D Moreno Santillán, Marta Farré, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Ying Sims, Alan Tracey, Jonathan M. D. Wood, David A. Ray, Dominik von Elverfeldt, J. Nieland, Angelique P. Corthals, A. Ruiz-Herrera, D. Dechmann, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2026-01-09</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background CUB domain-containing protein 1 (CDCP1) is implicated in pancreatic ductal adenocarcinoma (PDAC) prognosis, but its relationship to the tumor microenvironment (TME) and oncogenic signaling remains incompletely defined. We hypothesized that CDCP1 expression is associated with hallmark cancer signaling pathways and transcriptionally inferred TME remodeling in PDAC. Methods We analyzed transcriptomic and clinical data from 214 PDAC cases (The Cancer Genome Atlas (TCGA), n = 145; GSE62452, n = 69). Patients were stratified into high CDCP1 and low CDCP1 groups based on the top tertile of expression. Immune and stromal components of the TME were quantified using the xCell algorithm. Gene Set Enrichment Analysis (GSEA) with Hallmark gene sets was used for pathway enrichment. Results High CDCP1 expression was significantly associated with reduced infiltration of CD8+ T cells, adipocytes, and fibroblasts, suggesting an immune-excluded and stromally depleted TME. It also correlated with increased homologous recombination deficiency scores, mutation burden, and single-nucleotide variants. CDCP1 expression correlated with CDKN2A mutation but was only weakly associated with KRAS, TP53, and SMAD4 alterations. GSEA showed consistent enrichment of proliferative (E2F, MYC, G2M, p53) and protumorigenic (transforming growth factor-β, hypoxia, glycolysis) pathways in high CDCP1 tumors across both datasets. Conclusion CDCP1 defines a transcriptionally distinct PDAC subtype characterized by immune evasion, stromal depletion, and genomic instability. These findings highlight CDCP1 as a potential therapeutic target and biomarker reflecting interplay between oncogenic signaling and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc74e64f603534d40445c3a46e26eba6ab33298" target='_blank'>
              High CDCP1 Expression Reflects Immune and Stromal Remodeling and Oncogenic Signaling in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Kentaro Miyake, M. Oshi, Akira Takenouchi, Kota Sahara, Jun Yamamoto, Yutaro Kikuchi, Y. Sawada, Y. Homma, R. Matsuyama, I. Endo
          </td>
          <td>2025-12-17</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf0ad87ffc8bb1692d847c310099f651a0e6259" target='_blank'>
              Genome organizer SATB1 restrains the BACH1-cMAF axis that drives autoimmunity
              </a>
            </td>
          <td>
            Dionysios-Alexandros Papamatheakis, Petros Tzerpos, Despina Tsoukatou, Eleftherios Morres, Manouela Kapsetaki, Dezso Balázs, Kazuhiro Igarashi, Charalampos G Spilianakis
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: Systemic sclerosis (SSc) is a rare, complex autoimmune disease characterized by fibrosis of the skin and internal organs. While its pathogenesis is not fully understood, chromosomal instability and telomere attrition have emerged as significant areas of investigation. Methods: This review provides a historical narrative perspective and synthesizes current findings on the role of these genomic anomalies in SSc pathogenesis. We synthesized findings from foundational and recent research articles investigating genotoxic factors, chromosomal aberrations, and telomere biology in SSc. Results: There is a strong historical basis for chromosomal instability in SSc, manifesting as micronuclei, translocations, and breaks. This instability is driven by clastogenic factors and oxidative stress. SSc-specific autoantibodies are implicated; anti-centromere antibodies correlate with aneuploidy and micronuclei, while anti-topoisomerase I may inhibit DNA repair. SSc is also characterized by significant telomere attrition, first reported in 1996 and now confirmed by additional genetic studies. This telomere loss is associated with reduced telomerase activity and the presence of autoantibodies against telomere-associated proteins, including shelterin components. Conclusions: We conclude that inflammation, telomere attrition, and chromosomal instability are linked in a self-perpetuating cycle that drives SSc pathogenesis. We propose that an initial inflammatory stimulus leads to reactive oxygen species production, causing telomere damage and attrition. Critically short telomeres trigger faulty DNA repair mechanisms, such as breakage–fusion–bridge cycles, resulting in chromosomal instability. This genomic damage, in turn, acts as a danger signal, further activating inflammatory pathways and creating a feedback loop that perpetuates fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a5b157843adfebc89c062c9bea135fbbb56036" target='_blank'>
              Chromosomal Instability and Telomere Attrition in Systemic Sclerosis: A Historical Perspective
              </a>
            </td>
          <td>
            C. M. Artlett
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10944325f830532ecfb5fb037e3e00b3a58c4a6" target='_blank'>
              Sema7a drives an immunosuppressive microenvironment of breast cancer via Kdm4a-mediated DNA replication regulation.
              </a>
            </td>
          <td>
            Xianan Bai, Shanshan Cai, Jie Jiang, Maojin Tian, Jianxin Du, Peiqing Zhao
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Mitoxantrone (MX) is regularly used to treat several cancers. Despite its long history in the clinic, recent studies continue to unveil novel protein targets. These targets may contribute to the cytotoxic effects of the drug, as well as potential non-canonical antitumor activity. A better understanding of MX’s cellular targets is required to fully comprehend the molecular consequences of treatment and to interpret MX sensitivity in homologous recombination (HR)-deficient cancer. Methods: Here, we evaluated MX activity in HR-deficient UWB1.289 (BRCA1−) ovarian cancer cells and surveyed the binding profile of MX using TMT-labeled quantitative proteomics and chemoproteomics. Results: Mass spectrometry (MS) analysis of cellular extracts from MX-treated BRCA1−UWB1.289 cells revealed unique downregulation of pathways instrumental in maintaining genomic stability, including single-strand annealing. Moreover, the BRCA1− cells exhibited a significant upregulation of proteins involved in ribosome biogenesis and RNA processing. Additional MS analyses following affinity-purification using a biotinylated-mitoxantrone probe corroborated these findings, which showed considerable targeting of proteins involved in genome maintenance and RNA processing. Conclusions: Our results suggest that an interplay of both canonical and non-canonical MX-antitumor activity overwhelms the BRCA1− UWB1.289 cells. Furthermore, this study characterizes the target landscape of MX, providing insights into off-target effects and MX action in HR-deficient cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed8f08582f0dd8776423c5be2d239307bc42082" target='_blank'>
              Surveying the Proteome-Wide Landscape of Mitoxantrone and Examining Drug Sensitivity in BRCA1-Deficient Ovarian Cancer Using Quantitative Proteomics
              </a>
            </td>
          <td>
            Savanna Wallin, Sneha Pandithar, Sarbjit Singh, Siddhartha Kumar, Amar Natarajan, G. Borgstahl, Nicholas T. Woods
          </td>
          <td>2025-11-14</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In the 40 years since the discovery of the CENP proteins, many studies have examined the role of these proteins and their interactions with other chromosomal proteins of the centromere and beyond. Together, these studies have yielded vast amounts of sequencing and proteomics data. Typically, each study has focused on a single question and the majority of each dataset remains largely unexplored. Often the interesting details of publicly deposited data are left behind, buried in archives online, while more and more new data are generated. Reanalysing these databases can represent a new paradigm for investigating diverse biological pathways in unprecedented detail. Here, we explore two publicly available pan-cancer proteomic datasets to compare proteins whose abundance correlates with CENP proteins, with a particular focus on CENP-C. Our analysis confirms an expected link between CENP-C and cohesin levels but reveals a surprising and unexpected correlation between CENP-C and proteins of the inner nuclear membrane and the NuMA protein. This guilt-by-association analysis has the potential to identify proteins that act in common pathways but never associate or colocalize and may not even be expressed at the same time in cells. As an example, we show here that it can reveal unexpected links that expand our conception of centromeric chromatin beyond chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b131b447bc4a00bf7e5247f66ecd03a10d2ff6" target='_blank'>
              Sustainable integrative cell biology: CENP-C is guilty by association
              </a>
            </td>
          <td>
            N. Y. Kochanova, I. Samejima, William C. Earnshaw
          </td>
          <td>2025-11-25</td>
          <td>Chromosome Research</td>
          <td>2</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ab7d87288d1c4f12531bc9f091ec799a59efb" target='_blank'>
              Insights into DNA repeat expansions among 900,000 biobank participants.
              </a>
            </td>
          <td>
            M. Hujoel, R. Handsaker, David Tang, N. Kamitaki, R. E. Mukamel, Simone Rubinacci, P. Palamara, S. Mccarroll, Po-Ru Loh
          </td>
          <td>2026-01-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Cellular senescence is a hallmark of cancer and induces senescence-associated inflammatory responses in the tumor microenvironment (TME). CDKN1A, an important cellular senescence marker, plays significant roles in cell proliferation, invasion, migration, and apoptosis. Previous studies have implied its drug resistance role in certain cancer types. However, its impact on immunotherapy efficacy in advanced LUAD remains unclear. Using TCGA and SU2C-MARK cohorts, we investigated CDKN1A’s biological features in advanced LUAD, analyzing pathway regulation, immune infiltration, and immunotherapy associations. Single-cell RNA sequencing (GSE148071) validated TME and cellular communication differences between CDKN1A high/low expressing tumors in advanced LUAD. CDKN1A expression was positively associated with immunosuppressive environment including extracellular matrix, cancer-associated fibroblasts (CAFs) and myeloid-derived suppressor cells (MDSCs) in advanced LUAD for both TCGA and SU2C-MARK cohorts (P < 0.05). CDKN1A showed significantly positive correlations with many senescence genes in advanced LUAD (P < 0.05), which were also positively associated with endothelial cells, epithelial cells, fibroblast cells and macrophages, but negatively associated with immune cells (P < 0.05). In multivariable cox regression, patients with high CDKN1A expression had worse OS (HR = 2.74, 95% CI = 1.31–5.73, P = 0.007) and PFS (HR = 1.78, 95% CI = 1.01–3.11, P = 0.045) than those with low CDKN1A expression when treated with immunotherapy. In contrast, the high CDKN1A expression was not associated with PFS and OS in the TCGA cohort, in which the LUAD patients received standard chemotherapy, suggesting the immunotherapy predictive role instead prognostic role of CDKN1A. Moreover, single-cell analysis revealed that CDKN1A highly expressed tumors were accompanied by an enrichment of stromal and endothelial cells within the tumor microenvironment, along with enhanced activity of SMAD3/4, the downstream transcription factors of TGFB signaling. These tumors exhibited increased cell–cell communication with stromal cells (COL1A1/COL1A2-ITGA1/ITGB1/SDC1) and endothelial cells (NAMPT-ITGAS/ITGB1/INSR). CDKN1A expression was associated with cellular senescence, immunosuppressive environment and exhibited resistance to immunotherapy in advanced LUAD, suggesting a potential combination strategy with senolytic or senomorphic therapies to overcome immunotherapeutic resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f34f805e0422258abbde7dbe983d9c6f010ed5" target='_blank'>
              CDKN1A and cellular senescence are associated with immune resistance in advanced lung adenocarcinoma
              </a>
            </td>
          <td>
            Wang Shen, Feidie Duan, Shuiting Fu, Peng Cui, Yezhen Shi, Lei Zhang, Xiao Mi, Yu Xu, Guoqiang Wang, S. Cai, Xuebing Li, Xiaojun Tang
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Cell-free DNA (cfDNA) refers to DNA fragments that circulate freely in bodily fluids, primarily blood plasma. These fragments are released from cells via apoptosis and necrosis. Discovered in 1948, cfDNA gained clinical importance when linked to vascular dysfunction, positioning it as a promising non-invasive biomarker.



 To highlight cfDNA's diagnostic and prognostic applications in oncology and other fields, and to examine key challenges and future directions.



 In healthy individuals, cfDNA originates from normal cell turnover. In contrast, in diseases like cancer, circulating tumor DNA (ctDNA)—a subset of cfDNA—originates from tumor cells and contains mutations such as point mutations, rearrangements, and copy number variations. These genetic alterations reflect the primary tumor, allowing cfDNA analysis to guide clinical decision-making.



 In cancer, cfDNA enables early detection, therapy monitoring, tumor evolution tracking, and residual disease assessment. It also plays a role in prenatal screening, transplant rejection monitoring, and detecting infectious and cardiovascular conditions. Its short half-life (15 minutes to 2.5 hours) offers dynamic, real-time monitoring. However, low cfDNA concentrations, technical limitations (e.g., background noise, sequencing errors), and high costs in low-resource settings pose significant challenges. Detection methods like digital PCR and NGS are essential but expensive. Standardization issues in protocols and ethical concerns regarding genetic privacy further complicate adoption.



 cfDNA represents a transformative tool in precision medicine. Technological advances, integration with AI, and combining cfDNA with other biomarkers could enhance sensitivity and specificity. As clinical validation progresses and protocols standardize, cfDNA testing is expected to become routine, improving early disease detection and patient outcomes.



 Mohamed Tharwat. Kamouna. Cell-Free DNA: A Paradigm Shift in Cancer Diagnostics and Precision Medicine [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: The Rise in Early-Onset Cancers—Knowledge Gaps and Research Opportunities; 2025 Dec 10-13; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(23_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aec98a64fa684695d33212f886da7270836c0f" target='_blank'>
              Abstract A023: Cell-Free DNA: A Paradigm Shift in Cancer Diagnostics and Precision Medicine
              </a>
            </td>
          <td>
            Mohamed Tharwat. Kamouna
          </td>
          <td>2025-12-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77bd7f8b5c335d0db9519995ce46928ab5913c8e" target='_blank'>
              ClinBioNGS: A Clinical Bioinformatics Pipeline for Integrated Analysis of Somatic Next-Generation Sequencing Cancer Panels.
              </a>
            </td>
          <td>
            R. Marín, Ania Alay, M. Ajenjo-Bauza, Sara Hijazo-Pechero, Carla Montironi, E. Hernandez-Illan, P. Jares, D. Azuara, M. Varela, August Vidal, J. Puig-Butillé, C. Lázaro, Víctor Moreno, David Cordero, E. Nadal, Xavier Solé
          </td>
          <td>2026-01-01</td>
          <td>JCO clinical cancer informatics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181f022b28db4f3ed7a11dbea0e2ef041b80de4b" target='_blank'>
              Discovery of the TGCB microprotein translated from the lncRNA CHASERR
              </a>
            </td>
          <td>
            Bernardo Bonilauri, Xiaochun Yang
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dd3ec60ada06fb1f0e08f80162640c7452bc0" target='_blank'>
              Hakai links m6A RNA methylation to immune regulation in colorectal cancer
              </a>
            </td>
          <td>
            Macarena Quiroga, J. Escuder-Rodríguez, Andrea Iannucci, Victoria Suarez, Ivan Monteleone, Angélica Figueroa
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d96eca4dc76fb7d7dcfd02bd48a5d396e340f16" target='_blank'>
              Role of circAGFG1 as an oncogene in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wei Zhang, Lizhe Zhu, Xiaoqian Li, Jingyi Yang, Yu Ren, Jianjun He, Jing Xu
          </td>
          <td>2026-01-01</td>
          <td>Apoptosis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4238125d7b1a4373986b810a53811f6f3ecf9f4f" target='_blank'>
              Chromosome-scale assembly and annotation of Phytophthora capsici isolate BYA5.
              </a>
            </td>
          <td>
            Yiman Wan, Fuqiang Zhu, Meng Zhang, Xiaohui Li, Chunxue Xie, Shenglan Gao, Qunqing Wang, Yuan Chen
          </td>
          <td>2026-01-05</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad136fbb22f0ec3c1dbef1bcfae99f57ba3a39b5" target='_blank'>
              Transposable element dynamics in Arthropoda genomes and their impacts on the evolution of functional genes.
              </a>
            </td>
          <td>
            Aftab Ahmad, Botong Zhou, Lei Duan, Xinyu Fu, Wenyu Zhang
          </td>
          <td>2026-01-09</td>
          <td>BMC biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor that occurs in the mucosal epithelial cells of the colon or rectum, with high mortality and morbidity. Therefore, the study of its molecular principle is more important. In recent years, more attention has been paid to the regulation of CRC by epigenetic factors, especially the regulation of non-coding RNAs (ncRNAs) and RNA modifications. ncRNAs include miRNA, circRNA, and lncRNA. Among them, miRNAs regulate MAPKs, Wnt, and TGF-β signaling pathways and are involved in the tumor process. CircRNAs, as regulators of gene expression, can participate in various biological processes. lncRNA interacts with DNA, RNA, and proteins to regulate signaling pathways in various ways, potentially acting as biomarkers in non-invasive examinations. RNA modifications, especially N6-methyladenosine (m6A), affect CRC development. m6A consists of three proteins, writers, erasers, and readers, which maintain the stability of RNA modifications. Different levels of expression of these three proteins in CRC can lead to changes in the tumor microenvironment, gene expression, and other factors, and thus play an important role in CRC. ncRNAs and RNA modifications comprise the CRC epigenetic regulatory network, providing a new direction for the diagnosis and treatment of CRC in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d5caa5bf293c7752dde20eabcef8521208dcce" target='_blank'>
              Center Molecular Mechanisms of Non-coding RNAs and RNA Modifications in Colorectal Cancer
              </a>
            </td>
          <td>
            Tongyue Yu
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Type II Topoisomerases (topo II) are critical to simplify genome topology during transcription and replication. They identify topological problems and resolve them by passing a double-stranded DNA segment through a transient break in another segment. The precise mechanisms underpinning topo IIs ability to maintain a topologically simple genome are not fully understood. Here, we investigate how binding kinetics affects the resolution of two distinct forms of topological entanglement: decatenation and torsional relaxation. First, by single-molecule measurements, we quantify how monovalent cation concentration affects the dissociation rate of topo II from DNA. Second, we discover that increasing dissociation rates accelerate decatenation while slowing down relaxation catalytic activities. Finally, by using molecular dynamics simulations, we uncover that this opposite behaviour is due to a trade-off between search of target through facilitated diffusion and processivity of the enzyme in catenated versus super-coiled DNA. Thus, our findings reveal that a modulation of topo II binding kinetics can oppositely regulate its topological simplification activity, and in turn can have a significant impact in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02756b92213fd9ac89f480ffc7b784d0e58cb003" target='_blank'>
              Binding Kinetics Oppositely Regulates type II Topoisomerase Relaxation and Decatenation Activities
              </a>
            </td>
          <td>
            Cleis Battaglia, Filippo Conforto, Yair Augusto Gutiérrez Fosado, Matt Newton, E. Cutts, Davide Michieletto, Antonio Valdés
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="An R-loop is a three-stranded nucleic acid structure that serves as a transcriptional intermediate, consisting of an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA). Small RNAs are RNA molecules shorter than 300 nucleotides that perform a wide range of essential functions within cells. Both R-loops and small RNAs are widely present in the genomes of prokaryotes and eukaryotes, where they play crucial roles in regulating gene expression, maintaining genomic stability, and facilitating DNA damage repair. Aberrant formation and accumulation of R-loops, coupled with dysregulation of small RNA pathways, can induce DNA damage and genomic instability, ultimately contributing to cellular senescence or cell death. Here, we discuss recent advances in understanding the crosstalk between R-loops and small RNAs, with a focus on their synergistic roles in maintaining genome stability and their therapeutic potential in oncology and neurodegeneration. We propose a novel model integrating R-loop dynamics with small RNA-mediated epigenetic regulation, supported by emerging clinical trial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853eaea8f06e1b31e5bea0e5a97d17746ae77587" target='_blank'>
              R-loops and small RNA regulatory interactions: mechanisms and clinical perspectives
              </a>
            </td>
          <td>
            Si-Yao Wang, Ya-Ni Liu, Si-Qiao Zhao, Wen Liu, Jin Hu, Fan Yang, Sheng Wang
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2025', '2026'],
    y: [0, 27, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>